<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/12/3839" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/12/3839.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/12/3839.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/12/3839.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/12/3839/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/12/3839.atom" />
<meta name="HW.pisa" content="ggg;7/12/3839" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data" />
<meta name="DC.Identifier" content="10.1534/g3.117.300271" />
<meta name="DC.Date" content="2017-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Xia. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Two major stumbling blocks exist in high-throughput sequencing (HTS) data analysis. The first is the sheer file size, typically in gigabytes when uncompressed, causing problems in storage, transmission, and analysis. However, these files do not need to be so large, and can be reduced without loss of information. Each HTS file, either in compressed .SRA or plain text .fastq format, contains numerous identical reads stored as separate entries. For example, among 44,603,541 forward reads in the SRR4011234.sra file (from a Bacillus subtilis transcriptomic study) deposited at NCBI’s SRA database, one read has 497,027 identical copies. Instead of storing them as separate entries, one can and should store them as a single entry with the SeqID_NumCopy format (which I dub as FASTA+ format). The second is the proper allocation of reads that map equally well to paralogous genes. I illustrate in detail a new method for such allocation. I have developed ARSDA software that implement these new approaches. A number of HTS files for model species are in the process of being processed and deposited at &lt;http://coevol.rdc.uottawa.ca&gt; to demonstrate that this approach not only saves a huge amount of storage space and transmission bandwidth, but also dramatically reduces time in downstream data analysis. Instead of matching the 497,027 identical reads separately against the B. subtilis genome, one only needs to match it once. ARSDA includes functions to take advantage of HTS data in the new sequence format for downstream data analysis such as gene expression characterization. I contrasted gene expression results between ARSDA and Cufflinks so readers can better appreciate the strength of ARSDA. ARSDA is freely available for Windows, Linux. and Macintosh computers at &lt;http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx&gt;." />
<meta name="DC.Contributor" content="Xuhua Xia" />
<meta name="article:published_time" content="2017-12-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Two major stumbling blocks exist in high-throughput sequencing (HTS) data analysis. The first is the sheer file size, typically in gigabytes when uncompressed, causing problems in storage, transmission, and analysis. However, these files do not need to be so large, and can be reduced without loss of information. Each HTS file, either in compressed .SRA or plain text .fastq format, contains numerous identical reads stored as separate entries. For example, among 44,603,541 forward reads in the SRR4011234.sra file (from a &lt;i&gt;Bacillus subtilis&lt;/i&gt; transcriptomic study) deposited at NCBI’s SRA database, one read has 497,027 identical copies. Instead of storing them as separate entries, one can and should store them as a single entry with the SeqID_NumCopy format (which I dub as FASTA+ format). The second is the proper allocation of reads that map equally well to paralogous genes. I illustrate in detail a new method for such allocation. I have developed ARSDA software that implement these new approaches. A number of HTS files for model species are in the process of being processed and deposited at http://coevol.rdc.uottawa.ca to demonstrate that this approach not only saves a huge amount of storage space and transmission bandwidth, but also dramatically reduces time in downstream data analysis. Instead of matching the 497,027 identical reads separately against the &lt;i&gt;B. subtilis&lt;/i&gt; genome, one only needs to match it once. ARSDA includes functions to take advantage of HTS data in the new sequence format for downstream data analysis such as gene expression characterization. I contrasted gene expression results between ARSDA and Cufflinks so readers can better appreciate the strength of ARSDA. ARSDA is freely available for Windows, Linux. and Macintosh computers at http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/12/01" />
<meta name="citation_mjid" content="ggg;7/12/3839" />
<meta name="citation_id" content="7/12/3839" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/12/3839" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/12/3839.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/12/3839.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/12/3839.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300271" />
<meta name="citation_pmid" content="29079682" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="3839" />
<meta name="citation_lastpage" content="3848" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xuhua Xia" />
<meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada" />
<meta name="citation_author_email" content="xxia@uottawa.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3092-7566" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. M. Abraham;citation_author=J. E. Feagin;citation_author=K. Stuart;citation_title=Characterization of cytochrome c oxidase III transcripts that are edited only in the 3' region.;citation_pages=267-272;citation_volume=55;citation_year=1988;citation_issue=2;citation_pmid=2844415;citation_doi=10.1016/0092-8674(88)90049-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=V. S. Alatortsev;citation_author=J. Cruz-Reyes;citation_author=A. G. Zhelonkina;citation_author=B. Sollner-Webb;citation_title=Trypanosoma brucei RNA editing: coupled cycles of U deletion reveal processive activity of the editing complex.;citation_volume=28;citation_year=2008;citation_pmid=18227152;citation_doi=10.1128/MCB.01886-07" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Arava;citation_author=Y. Wang;citation_author=J. D. Storey;citation_author=C. L. Liu;citation_author=P. O. Brown;citation_title=Genome-wide analysis of mRNA translation profiles in Saccharomyces cerevisiae.;citation_volume=100;citation_year=2003;citation_pmid=12660367;citation_doi=10.1073/pnas.0635171100" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. R. Awan;citation_author=A. Manfredo;citation_author=J. A. Pleiss;citation_title=Lariat sequencing in a unicellular yeast identifies regulated alternative splicing of exons that are evolutionarily conserved with humans.;citation_volume=110;citation_year=2013;citation_pmid=23861491;citation_doi=10.1073/pnas.1218353110" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=G. Benoit;citation_author=C. Lemaitre;citation_author=D. Lavenier;citation_author=E. Drezen;citation_author=T. Dayris;citation_title=Reference-free compression of high throughput sequencing data with a probabilistic de Bruijn graph.;citation_pages=288;citation_volume=16;citation_year=2015;citation_pmid=26370285;citation_doi=10.1186/s12859-015-0709-7" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Dai;citation_author=R. C. Thompson;citation_author=C. Maher;citation_author=R. Contreras-Galindo;citation_author=M. H. Kaplan;citation_title=NGSQC: cross-platform quality analysis pipeline for deep sequencing data.;citation_pages=S7;citation_volume=11;citation_year=2010;citation_pmid=20158878" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Q. Deng;citation_author=D. Ramskold;citation_author=B. Reinius;citation_author=R. Sandberg;citation_title=Single-Cell RNA-Seq Reveals Dynamic, Random Monoallelic Gene Expression in Mammalian Cells;citation_pages=193-196;citation_volume=343;citation_year=2014;citation_issue=6167;citation_pmid=24408435;citation_doi=10.1126/science.1245316" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_title=Closure of the NCBI SRA and implications for the long-term future of genomics data storage.;citation_pages=402-402;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21418618" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=J. F. Dufayard;citation_author=V. Lefort;citation_author=M. Anisimova;citation_author=W. Hordijk;citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.;citation_volume=59;citation_year=2010;citation_pmid=20525638;citation_doi=10.1093/sysbio/syq010" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. T. Ingolia;citation_author=S. Ghaemmaghami;citation_author=J. R. S. Newman;citation_author=J. S. Weissman;citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.;citation_volume=324;citation_year=2009;citation_pmid=19213877;citation_doi=10.1126/science.1168978" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. T. Ingolia;citation_author=L. F. Lareau;citation_author=J. S. Weissman;citation_title=Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes.;citation_pages=789-802;citation_volume=147;citation_year=2011;citation_issue=4;citation_pmid=22056041;citation_doi=10.1016/j.cell.2011.10.002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Janin;citation_author=O. Schulz-Trieglaff;citation_author=A. J. Cox;citation_title=BEETL-fastq: a searchable compressed archive for DNA reads.;citation_volume=30;citation_year=2014;citation_pmid=24950811;citation_doi=10.1093/bioinformatics/btu387" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=K. Katoh;citation_author=H. Toh;citation_title=Recent developments in the MAFFT multiple sequence alignment program.;citation_volume=9;citation_year=2008;citation_pmid=18372315;citation_doi=10.1093/bib/bbn013" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. Kawashima;citation_author=S. Douglass;citation_author=J. Gabunilas;citation_author=M. Pellegrini;citation_author=G. F. Chanfreau;citation_title=Widespread use of non-productive alternative splice sites in Saccharomyces cerevisiae.;citation_pages=e1004249;citation_volume=10;citation_year=2014;citation_pmid=24722551;citation_doi=10.1371/journal.pgen.1004249" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Kingsford;citation_author=R. Patro;citation_title=Reference-based compression of short-read sequences using path encoding.;citation_volume=31;citation_year=2015;citation_pmid=25649622;citation_doi=10.1093/bioinformatics/btv071" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Kodama;citation_author=M. Shumway;citation_author=R. Leinonen;citation_title=The sequence read archive: explosive growth of sequencing data.;citation_volume=40;citation_year=2012;citation_pmid=22009675;citation_doi=10.1093/nar/gkr854" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=A. I. Lamond;citation_title=RNA editing and the mysterious undercover genes of trypanosomatid mitochondria.;citation_pages=283-284;citation_volume=13;citation_year=1988;citation_issue=8;citation_pmid=3154277" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=K. D. Hansen;citation_author=J. T. Leek;citation_title=Cloud-scale RNA-sequencing differential expression analysis with Myrna.;citation_pages=R83-R83;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20701754;citation_doi=10.1186/gb-2010-11-8-r83" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Leinonen;citation_author=H. Sugawara;citation_author=M. Shumway;citation_title=The sequence read archive.;citation_volume=39;citation_year=2011;citation_pmid=21062823;citation_doi=10.1093/nar/gkq1019" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. Li;citation_author=P. Ge;citation_author=W. H. Hui;citation_author=I. Atanasov;citation_author=K. Rogers;citation_title=Structure of the core editing complex (L-complex) involved in uridine insertion/deletion RNA editing in trypanosomatid mitochondria.;citation_volume=106;citation_year=2009;citation_pmid=19590014;citation_doi=10.1073/pnas.0901754106" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Liu;citation_author=H. Guo;citation_author=M. Brudno;citation_author=Y. Wang;citation_title=deBGA: read alignment with de Bruijn graph-based seed and extension.;citation_volume=32;citation_year=2016;citation_pmid=27378303;citation_doi=10.1093/bioinformatics/btw371" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=V. L. MacKay;citation_author=X. Li;citation_author=M. R. Flory;citation_author=E. Turcott;citation_author=G. L. Law;citation_title=Gene expression analyzed by high-resolution state array analysis and quantitative proteomics: response of yeast to mating pheromone.;citation_volume=3;citation_year=2004;citation_pmid=14766929;citation_doi=10.1074/mcp.M300129-MCP200" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Nicolae;citation_author=S. Pathak;citation_author=S. Rajasekaran;citation_title=LFQC: a lossless compression algorithm for FASTQ files.;citation_volume=31;citation_year=2015;citation_pmid=26093148;citation_doi=10.1093/bioinformatics/btv384" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=I. Numanagic;citation_author=J. K. Bonfield;citation_author=F. Hach;citation_author=J. Voges;citation_author=J. Ostermann;citation_title=Comparison of high-throughput sequencing data compression tools.;citation_pages=1005-1008;citation_volume=13;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. A. Pleiss;citation_author=G. B. Whitworth;citation_author=M. Bergkessel;citation_author=C. Guthrie;citation_title=Rapid, transcript-specific changes in splicing in response to environmental stress.;citation_pages=928-937;citation_volume=27;citation_year=2007;citation_issue=6;citation_pmid=17889666;citation_doi=10.1016/j.molcel.2007.07.018" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=V. Pobre;citation_author=C. M. Arraiano;citation_title=Next generation sequencing analysis reveals that the ribonucleases RNase II, RNase R and PNPase affect bacterial motility and biofilm formation in E. coli.;citation_pages=72;citation_volume=16;citation_year=2015;citation_pmid=25757888;citation_doi=10.1186/s12864-015-1237-6" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Roberts;citation_author=C. Trapnell;citation_author=J. Donaghey;citation_author=J. L. Rinn;citation_author=L. Pachter;citation_title=Improving RNA-Seq expression estimates by correcting for fragment bias.;citation_pages=R22-R22;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21410973;citation_doi=10.1186/gb-2011-12-3-r22" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Roberts;citation_author=L. Schaeffer;citation_author=L. Pachter;citation_title=Updating RNA-Seq analyses after re-annotation.;citation_volume=29;citation_year=2013;citation_pmid=23677943;citation_doi=10.1093/bioinformatics/btt197" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. F. Rogers;citation_author=J. Thomas;citation_author=A. S. Reddy;citation_author=A. Ben-Hur;citation_title=SpliceGrapher: detecting patterns of alternative splicing from RNA-Seq data in the context of gene models and EST data.;citation_pages=R4-R4;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22293517;citation_doi=10.1186/gb-2012-13-1-r4" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=I. B. Rogozin;citation_author=D. Managadze;citation_author=S. A. Shabalina;citation_author=E. V. Koonin;citation_title=Gene family level comparative analysis of gene expression in mammals validates the ortholog conjecture.;citation_volume=6;citation_year=2014;citation_pmid=24610837;citation_doi=10.1093/gbe/evu051" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=R. M. Simpson;citation_author=A. E. Bruno;citation_author=J. E. Bard;citation_author=M. J. Buck;citation_author=L. K. Read;citation_title=High-throughput sequencing of partially edited trypanosome mRNAs reveals barriers to editing progression and evidence for alternative editing.;citation_volume=22;citation_year=2016;citation_pmid=26908922;citation_doi=10.1261/rna.055160.115" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Stepankiw;citation_author=M. Raghavan;citation_author=E. A. Fogarty;citation_author=A. Grimson;citation_author=J. A. Pleiss;citation_title=Widespread alternative and aberrant splicing revealed by lariat sequencing.;citation_volume=43;citation_year=2015;citation_pmid=26261211;citation_doi=10.1093/nar/gkv763" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=C. Trapnell;citation_author=A. Roberts;citation_author=L. Goff;citation_author=G. Pertea;citation_author=D. Kim;citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.;citation_pages=562-578;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22383036;citation_doi=10.1038/nprot.2012.016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=C. Trapnell;citation_author=D. G. Hendrickson;citation_author=M. Sauvageau;citation_author=L. Goff;citation_author=J. L. Rinn;citation_title=Differential analysis of gene regulation at transcript resolution with RNA-seq.;citation_pages=46-53;citation_volume=31;citation_year=2013;citation_pmid=23222703;citation_doi=10.1038/nbt.2450" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Vlasschaert;citation_author=X. Xia;citation_author=D. A. Gray;citation_title=Selection preserves Ubiquitin Specific Protease 4 alternative exon skipping in therian mammals.;citation_pages=20039;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=X. Xia;citation_title=DAMBE5: a comprehensive software package for data analysis in molecular biology and evolution.;citation_volume=30;citation_year=2013;citation_pmid=23564938;citation_doi=10.1093/molbev/mst064" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=X. Xia;citation_title=DAMBE6: new tools for microbial genomics, phylogenetics and molecular evolution.;citation_pages=431-437;citation_volume=108;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Xia;citation_author=V. MacKay;citation_author=X. Yao;citation_author=J. Wu;citation_author=F. Miura;citation_title=Translation initiation: a regulatory role for poly(A) tracts in front of the AUG codon in Saccharomyces cerevisiae.;citation_volume=189;citation_year=2011;citation_pmid=21840854;citation_doi=10.1534/genetics.111.132068" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Zhu;citation_author=L. Li;citation_author=Y. Zhang;citation_author=Y. Yang;citation_author=X. Yang;citation_title=CompMap: a reference-based compression program to speed up read mapping to related reference sequences.;citation_volume=31;citation_year=2015;citation_pmid=25282641;citation_doi=10.1093/bioinformatics/btu656" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=Z. Zhu;citation_author=Y. Zhang;citation_author=Z. Ji;citation_author=S. He;citation_author=X. Yang;citation_title=High-throughput DNA sequence data compression.;citation_volume=16;citation_year=2015;citation_pmid=24300111;citation_doi=10.1093/bib/bbt087" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="twitter:description" content="Two major stumbling blocks exist in high-throughput sequencing (HTS) data analysis. The first is the sheer file size, typically in gigabytes when uncompressed, causing problems in storage, transmission, and analysis. However, these files do not need to be so large, and can be reduced without loss of information. Each HTS file, either in compressed .SRA or plain text .fastq format, contains numerous identical reads stored as separate entries. For example, among 44,603,541 forward reads in the SRR4011234.sra file (from a Bacillus subtilis transcriptomic study) deposited at NCBI’s SRA database, one read has 497,027 identical copies. Instead of storing them as separate entries, one can and should store them as a single entry with the SeqID_NumCopy format (which I dub as FASTA+ format). The second is the proper allocation of reads that map equally well to paralogous genes. I illustrate in detail a new method for such allocation. I have developed ARSDA software that implement these new approaches. A number of HTS files for model species are in the process of being processed and deposited at &lt;http://coevol.rdc.uottawa.ca&gt; to demonstrate that this approach not only saves a huge amount of storage space and transmission bandwidth, but also dramatically reduces time in downstream data analysis. Instead of matching the 497,027 identical reads separately against the B. subtilis genome, one only needs to match it once. ARSDA includes functions to take advantage of HTS data in the new sequence format for downstream data analysis such as gene expression characterization. I contrasted gene expression results between ARSDA and Cufflinks so readers can better appreciate the strength of ARSDA. ARSDA is freely available for Windows, Linux. and Macintosh computers at &lt;http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx&gt;." />
<meta name="og-title" property="og:title" content="ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/12/3839" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Two major stumbling blocks exist in high-throughput sequencing (HTS) data analysis. The first is the sheer file size, typically in gigabytes when uncompressed, causing problems in storage, transmission, and analysis. However, these files do not need to be so large, and can be reduced without loss of information. Each HTS file, either in compressed .SRA or plain text .fastq format, contains numerous identical reads stored as separate entries. For example, among 44,603,541 forward reads in the SRR4011234.sra file (from a Bacillus subtilis transcriptomic study) deposited at NCBI’s SRA database, one read has 497,027 identical copies. Instead of storing them as separate entries, one can and should store them as a single entry with the SeqID_NumCopy format (which I dub as FASTA+ format). The second is the proper allocation of reads that map equally well to paralogous genes. I illustrate in detail a new method for such allocation. I have developed ARSDA software that implement these new approaches. A number of HTS files for model species are in the process of being processed and deposited at &lt;http://coevol.rdc.uottawa.ca&gt; to demonstrate that this approach not only saves a huge amount of storage space and transmission bandwidth, but also dramatically reduces time in downstream data analysis. Instead of matching the 497,027 identical reads separately against the B. subtilis genome, one only needs to match it once. ARSDA includes functions to take advantage of HTS data in the new sequence format for downstream data analysis such as gene expression characterization. I contrasted gene expression results between ARSDA and Cufflinks so readers can better appreciate the strength of ARSDA. ARSDA is freely available for Windows, Linux. and Macintosh computers at &lt;http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
    <title>ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZM3PlcTrtEUPHsyFQxm5anEGcZcTcrUUrDhQw_x7qmI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487479","apath":"\/ggg\/7\/12\/3839.atom","pisa":"ggg;7\/12\/3839","ac":{"\/ggg\/7\/12\/3839.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/12\/3839.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/12\/3839"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/12\/3839"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/12\/3839":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022h6jE7pEkNl7llrcorriG46QbfHtiay4HxHhkD80jlcPSHhvecPGzwJzwHvkjhgANZUZrMFz\\\\\\\/H2n2s+IQATwDSPNPIxtc1slPi\\\\\\\/8hxJLMRAt4eJ5h232oBciCoHW6SqIWiRK9nWymaSzxTM7PzYiCbNPA5vGkdU5Ew1yavYyTOUamaqk+rS\\\\\\\/Lgj0F4kLcVaylTEYUxSIaG1Zd7IIkNa4oEQ+TneV3vd4fxZAUb95CAJ+yc6N\\\\\\\/\\\\\\\/fwBdAcZgMRyax\\\\\\\/q1dFL8NObn10K7TVubxhP+kc9aXZTUCnh8fVYyyinnyy\\\\\\\/hEmB4n7kbZfxKnaW7nrAELQhVIijFSTTo+Q0Vt06nowRx6yE2mo0XQU30USS+10QGJdZGDI9QsvP82wattxkvOFnTee704UfJIqPzqGxbWVH61VwMKW+EnX\\\\\\\/SpPkOCx6aEu6uBqLU6F0ISSnGaBo9rlUILWDXd9Tliwe+80BQ5N9YZDNDVz1QpOscjob\\\\\\\/dOrpm71laki8KypWk69gYCzCtPRDpRYnmfvvOCxJEhN5vuTLxQUfMP3PtTAJNHkI29WYvjpVpXTZiwy\\\\\\\/iPP\\\\\\\/Byt5+Fz\\\\\\\/oRom8BF9TOi4D+byhWgPbumdUng5z4yhyxy7jIwik0E8D5bHyDHCuIsFwpiFkXzyxlhzDZneAi94Dt7kcLeetA6PneCLPwqtzvJ6uxgNEPfbS35oqsT3G+RF9q8lYb44P4sMFt8tiWFkX9MxOAnbc6c3c4ZOrunTBHsspFqs1DMo8V8jQlaYQcbkScqy+PgMFM59fl5bJeCi4ABESXbcAV2tcLYUzbItQpGO5mlgOrofbfglmwf51Tu4emSUxWSkTGIThlIchmrYdMPkdDXfL9o8vmbcPekLVa3g1RlLUFSgUV2s5xsiZELeW5ntn9xwpocW0uvQrG1oN7bFNQsXbSH6JOmqpIpZ\\\\\\\/SC2GaLiWbJpC1\\\\\\\/Wm8hucrD5enhwIJGkrlmNlQXWJuR\\\\\\\/\\\\\\\/mllBjUI0skAew9TpPdJFQL+tT84Rpt5n0LAsThNHaFeI0Bbz0EkZakF4nLypMKn5R2AGBvryLUQn9TNFfdJ2HMFbldC86fqxJNdz4+faiytORFUOcv0E75y3ThS5uGkkQrGU3e\\\\\\\/yYEjuI5fpDzkxJLQaXhjTxGJImAEGBMHetyxXONxVlbvBJ9H1OZJG+CvkK3IG6nH1W+331DpPEzcfIXJ+fTFHVKaHGdg5HABpT4YSGzWjOeGkr4Fgw\\\\\\\/LTbGA7D9VukeVrDk7wlWMKZPIlSBTEDJ138ILFoCpfJP+E9tlZcC\\\u0022,\\\u0022iv\\\u0022:\\\u0022dCUyAY6oGCer0ZBVrB5tqA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bb8e72c37a98035d470ca407f9f21cdc192d8bd20e8629b252c597e0957bf29c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224Ng8R9fgAR7IdZU1Ztlp1kwEYLRPCo56K7YFKQxQxNfWS7ArvWfxIgxBoHi3MK7GIA4Pr9lem9Flh4rUxEim\\\\\\\/RdhH+1Z9LTYKJ2uA6Q5H31yiAeUoomYVIOjcv1NuQnt99P89EQVs4dPkisw6W1Qairh6T\\\\\\\/wygpwEMD6zAXdKrUsYHjOd2IqPZIPfrK35vVTZ6aOPGet9fxDBhes7TsuWeM6ZMTQMX1zMmijsl5fSvrVc3Axst8swbsN9c15Hxho31s+A8h3lT9NTp2eJzv8akweRfD0z8aTF9b4r6CTkVbMjE34Pne1m86qoThZ6X2uVpjZKLAYTXZgOcSmfrRc6HhfryN6njrHuEe01WUwX20eXurYpdug+\\\\\\\/6E64S9cx3v5mEGp3KnRETaOOuimTRQLu0v7mmfON0zM\\\\\\\/wPthZwdKcV3i5C60CgcAj70bySgppGTkA1GaFpyX4CI7+4hKkirRGdXa5vaBm8P\\\\\\\/dO5uXVy8mr1nCQR9FPFYbZQZfYQ2PvoMPy7UYUTwSlcQBaF3r9WD6xGsm3ZHYIMfXYuxYkwyEzZu0fRyR7ira1sN32d1CLG3bjW7Wmex3P5os8S0fOIf1iR9PvwVsK3gW4vkafeLpfUIG+p3X+OjzyJekhRvkjJ4CbJt+Gu4DP4LlNtVt8qqcDskIC8Ho963v3SoqXDvy5bmQkmKtZhdxJZL+pZxLWsTDAnW6fw8OzffS9WpQZ6790BAE0zJL5JlnSHZT5jAKfjgQZ2TXFqUxCz627raz2K4yyJYsZYrGXnsp7oKSpEcAEY6O7C65cVjG7OAFXN9z\\\\\\\/DgKw53inAztFPh8ABnGGCeW2CFKEzPTrB5\\\\\\\/doKvyaSc4ePx6H9AlwGe9khLzodjQxvBXAPQykJKWEtwj3l5Pl4zep2KlEb\\\\\\\/Ykv4cr6cxlSxioIpMIxsjhnIF6MUgrSRM5B9yLmHDVO2g0G81hKfIxBT38s7q4fqohqssz9AV84UhtENR\\\\\\\/naATNeHS8sL7MhSUi8vAqZHfbD8aMvoU+VVHDmKSZl+xgK\\\\\\\/JG0yoUyt54CUhcBlvRU3BgvDy4NbFHuQ9AoXd9sEiOR6nYipqwBckvyr1bmNH\\\\\\\/GqtVkhEcsq6HTPdM0xcQGPob9p4FkrBloxuqBk9JlkcjHHBGPyBekTnMVisECHhVmkHhXf77hmK5s3B2NrVJujWJ4JlEN1gBExgpSAYTTVsfywj19PCzheYiPt7bd3CL0qb5gHV721zbhICxbUoLqFb6oQesBeLMeU329WNH\\\\\\\/\\\\\\\/sJBy+FgSXjbP0xc\\\\\\\/TyazgD7KKhpPmTOCGJcvDGz6gRdpza3h4bXO9dB8HHrBWsRVRfAwj\\\\\\\/6b9KNAUe\\\\\\\/+c9DjPYyuuYYU\\\\\\\/Ra3+455Flq6z\\\\\\\/ezyfztwQm2xM\\\\\\\/UnVNiJJobh\\\\\\\/k+Gqa1AMtbVa2WP7W\\\\\\\/vkaw9TE+cOZObF24YeR1hjBM747olwe67hzq3uZ58LTGhB16nSbzMiS8uqOHHwlmkF8zJQ\\\\\\\/8JOJEy3EZgccMDGhK7Siz43Oer9miU57c\\\\\\\/JcFPUDbKPc1vI7ekzRK+G75p8oQmBxwRDa5H7MZohfSaIN9p+dHMGhGpZ\\\\\\\/E0zvKSSzTn\\\\\\\/3iOJA8fwqW3bgVIFcSykJFMFWDi0pcuTVGz7dISVvHAeQdS8o1SZfRJjVrDt2Pk0bneoMPqDKAGS4PqBW2PDmIUZ0+ND2ZsPB5SfCyWQh6+bfJuq8ABL5qHcjRFchjF2EZhuVyYHGphQfmZDsEKX3nY96JgYnBg2WEHm1rF5xn6QlsD9c25Td4Y3Pb3esE8U8vxy0ndLmJzNC+OTbjS3Xj5CeihR9sdN4Gl58qt2UnoOCg+YHR3Eek+EE5S1eCOuCPyfSUTJirmzh93\\\\\\\/+IFhEN30qiPdsq4xo3tdXtCykEuFNrPLQC\\\\\\\/2ByTzJADrXoIV3KLxxPxXGAMOhIPW2ewhvMadFonWWLnwCvwBq8HevYujdgystm5NyamJH8J1pqotHhigLJ2HDNpDQc6qK7\\\\\\\/5MiY7I4e8EaHElurkrI5WnNdFdwThyMMhRiDklN1tJyLMqNNgSTQ5fKSQ40yX1TB47AIytNINE\\\\\\\/60NWNljyrd1U1rU\\\\\\\/ZEiMgx4\\\\\\\/lqROUQKSye16Crcg6bzEPfML\\\\\\\/NTHB3zHxi7FBJvt5dCbxz9IB04CJwupDTvPZDfhMLOu2zV2boDeYOK9vwzGLgTfwSXexB2fwAdRcnq74c+RNqhfmdf45xMIArLSBFTiQagbBcFWwH17bvfsGefEWI2GQoyaRZsyGbLBxmC1LAFSS2XAhl1vNAKMRnsbJYs6Mujv8mZchkyZ5WTRsTaRPnr4MvftyacmoJ5w\\\\\\\/Ql09yxBW4uApR62o+nFGxDtQrpSb1IGNL4YPfxG6DdneY9nNHjqYZvSPO+t6zvnlvweY4cAJoQcPw+wY1dKKJyBCXO6f4bmiYSBnfbC\\\\\\\/B+MlX7kQRN4CDg51FmIOqEjXPU5E4o0vidP0dPY8PWkkSJq3ypxTM4ROgJrK86mlWvWiwbuUxSyhJCqQHCOLdPxzlhknCRuVvBedFt8ln1DfbNRESha0GFjkblElIoAKTXrN18D6PZ+ObJ8sIcXzRNWR35Zai0iAXFEZK0WGK1BM1K8iKgTL+Au7Dwv1Tp+IHrMkW68UCV5o0udd\\\\\\\/Hjr0lgkygeVvvoGAL4qi0ItFIZK4sYgva67oNWTGwI00b6XAV1yfXGx2XWQ1sJ0BqaSPPx\\\\\\\/pbQaliw6KVvByxoviW+ZGCIqoADlXbV8R1\\\\\\\/zvsrUmBu5sCFhiqf0kTsMDNSPNQwXIrNnQVbn34bu94JrfqRkPj8Ks8ok6ECxsJEeKFSMTw6LR4Q31VHgChb+Z\\\\\\\/\\\\\\\/od4vbTmzJkSl+9VA9IAK51SJC7RvsH5Pku5warT\\\\\\\/WrA3tO1D0MvX\\\\\\\/EQ1cYy1HvHWsanBww52lobN64kScAxkCrhqbkQgVkURUf\\\\\\\/n4+5pBO1VD1PQVxuJEygY7QScmWVAccswZHxlXn18itKPbpS1AslbSKUH77G+tVUlS6we78L8QQ7jQYE2DyYJwHI4brInpfffr5+M\\\\\\\/AEwXDRmzIMcSHUeKqyLv8cmQzouOOgZXSiG8aZ+HXl\\\\\\\/4Zzpf1ezYmPNNxIAWXicztmG4wGzM7\\\\\\\/1ifJTA39U+5BIxz5dj9xE2ioCIEnsOcs0ay5BhUYpExP2Z7Rf3QqzINfYTUNo+4Q\\\\\\\/SPH0Il3oM4d1Pn71oGlLMaQG9lsWQki3c4gKHYAHWEwtSAwg+sPA6kODrGbJ4J9\\\\\\\/d8NJg015w2+vPLIbYDvHy8Oh2myLiuxPpsJYNNWiKYNdqzQIuYfNPuRujFW5j8oxAuoJ8o+D6rFDOU1o1e2xTfbRDwVw+pHF4\\\\\\\/mhRaqhUfArBqGVhKDcELBcmNWgw5EWk0PWZTPN+Q8PPDQS1Ron65AZ5+hyteUXBzH2b\\\\\\\/HOv5KoDJEbC7+TK8iUBP4utCbO7Yxnw8F0F3+TwGoIa2BkrSfzrKB3oxV3JzBi5SArPqbdzXY8+soYcwyurtOI+oWt2fwrQtY4N2dWGx9+jj7oQzsRmuO1BQ\\\\\\\/TK4vQOaDKJqBIzzmst0T53QxUv3N7rFAjRGgLUPiKnXChjPZVAkeDtHMNyYGls13RLLmCBmACgzPHdpeLdfQd1BO4+YPv\\\\\\\/Hf\\\\\\\/REw0dx3obIxfvMzhdcAmQc8xZWpYXChbQpiBJ4GT8w9ZvJv\\\\\\\/chMajFybgtvpUGEIZDeJm8NkFuDsXnZvNfFxEauR9laKL9uiHjhunbfA9Dn9hlVEUkRkpSNlOeQpoBbhOmwEZksuMlhhJGym21kUWUbH+bM+p5ahXVmBp6olc3zKk9lv2SmbHo4SfNnuC\\\\\\\/+tL99tYn006rZw08F657p1Uy\\\\\\\/ZW44WchhTUgWDD7hzuL1fDgaqZSGFYmMnJ+Le32bJLl\\\\\\\/lYePwKE+grP1Z3bY5rfN2e7tW\\\\\\\/cWjceOupEtromdg5zkTmMYu\\\\\\\/nbj9YaLFpw6UDpXXgd\\\\\\\/1U1zSQCF2kR7d3RTwSpIa9CoQ+44zodnOThWLbBFmpdJr6zL0ZdukAm06ZVxdsVRMkFFw+gQaeWcb3MvqGXmJFo2tzmW2WMlwEa4B40id1QBmb3ldAmjiIpmDsecJXsKvQNT2\\\\\\\/030CpRjxXeMY6J78MoywMqoTOVrP3MwKFuTOINSXxgaHm3tWT8KKC3NPQobh92HLPBMLe1HROiBARPnw6gXHv2tjdq0hyALGtatbl0kCyXJg29Kj3tYHJJrdvRGqsUtMdqd3Ckq+xInIkBPYyBj6nPlvaP7mwoNg4Zx6RvqHkLUMzzMajwTPuNfrsCnKihdS3XI3NMDUJk5DgQwDaTmB\\\\\\\/ILNJ+mhpIE+MjygL0y8XvJJCAegP2mzOB3xir16Gz9ZLeq\\\\\\\/pSdcgd7INAawZ1YSUbp7EdmKYrW\\\\\\\/XUZ22oYwezJSXjPbZOaOcaFCwkxPtDdqC\\\\\\\/nZVzkDaujcmkqlFgbMiZhtA1NRY\\\\\\\/PbnF+5Koa7XHr1UcSdELmBE0qMFfPIL2jdoDbINODDPwu2dY8koMjXvC9YCnHKag1pINYk5TGc2AVJp7sWdAnFa6ty2qH67LrdO+7qCCY+eK+rXVA5WEW6B6BvR5bOeBFZmosJ1knJqNybvOpo3wRb2OULG\\\\\\\/t\\\\\\\/s8QNOKmKfBb\\\\\\\/+Joql+5\\\\\\\/llCmlIud1DvYuMEbiRctWfDbJ5DsAUXo6IEQ9vtrER52eg9YmQE2UZorxe3RoVjbxPxqnr6qm+iHEEFWDMqowI2xqjUyNH5b1sXbdCcByXGLDi+5GjjaUiyuwVusuChjwvZk6I7Ux6+VRDKenS6F\\\\\\\/+IVijCQg+QcMHVoj\\\\\\\/YHsKctJv6BQFHiID4eDSKM2uICcg7N2NKUKXEw0kRJ7teGjNtLTjnDFmmTgRgfSjbRFICqbNR7Ela4nhl+iNjspBNDuFiP8N8jPG0yqWiQV6IgK0cuHGtHWdwS7aHx6ZH5ZaH72qxQMvaiaDqjo94lk\\\\\\\/rcB1FoL33sQNrMbvHAAfFo7h3yEhA78HmSKN1LOhi2yTBzZArNN9DDfvolCpjAY2Z+RE9cTM7Omr47hEGHaVXj5K68RR\\\\\\\/rdnXILzm9vbFz0lXM6CcWfzny6wzAcwZrc2Cjo+pSjIW4\\\\\\\/bkQSmMk3aJxyk5DZb59bc0mpm9JWMWGpa\\\\\\\/I\\\\\\\/\\\\\\\/O6n16HGkyfIiVwZ\\\\\\\/ceMa39u8n7xWqPtna19v1Bz5DiU1VK05k3\\\\\\\/iiBPZGqFA77alusDyR4xz6uVXDpjTZ91qNY45w3rVbubTIYgOgQbu\\\\\\\/aUQUxLLvtHiXpR8X0yyYrL0TqXldBWt8uBQ26NUK7CrXkGsiuItTxYA8oidUQDZZlPPiE7+U1z6Pxwy4oPh1LCeV6ZGSSp995BPkulyLYdLJsheac32odJECwmMwwoktM7Qa\\\\\\\/0OhsujVm3\\\\\\\/nfWJiMhkiEF8Iliio0N4kXPatVR+I9A7scXt00SPq2\\\\\\\/U8TKAIboesrUlrH+FRwbz8QlsCbAQQk\\\\\\\/1p49PtqdjVENzWTOalxf\\\\\\\/X8hwRESX8kwRq0GaOA0MrblqREb\\\\\\\/QC8QOH1S1obrTxWDPUxcdkX8UZLbVVca8Q2hpF\\\\\\\/+sESkPOn\\\\\\\/S8Pd2RkkglNV07+vvz2q0oIOc6G+qmGmEl0a6vjpajCRnsWzh6\\\\\\\/9xmLiqV\\\\\\\/Vd3CKO5PCHghhXKlFs8LD3EWfuvyEURl2\\\\\\\/neQbafie+XhoeL5yIEQ8+4vbZDs+xuBM3cWWS7BVES\\\\\\\/YLgzsLLSh4VA6ntJAHIl9wSG65w2UuZi6xdnpDGpOnFF7RfrB+PyuNcU0XOWdOR5ad0w5p7M3KlCJQR2khedY4ihlxRHrBWUNUxASLrx9YVBE09U3acfInJJuyhdsToI+sfJqJ\\\\\\\/PLy9C9VFMCYoWheuES2q9z8iqkBQkqB58UmpSQGG7bLRPY9At\\\\\\\/oMP+dfnZ3ABKV5bw\\\\\\\/AxRjOM1G7q2GfN2cvJfZFc1lv0LscvX07myu2UIT9rxK4MJbXvUEDwJC8QCM3YHo\\\\\\\/6MSOCduf+cLR7xJN8irtu2Wc\\\\\\\/RRJC107ruFYETMTNyoJgPYMUnmfJTtzLa62S2KQuZk4neIpNd9gI0Ft7Av5oaBaMD\\\\\\\/FLJ2738RT4E0s1qJn5IqS3hgSkhtdNluKxvHR926u6iEJvRAoYEOHcARBQIod634ZqeGz5hV534daNjYQeLOkG104Qk1KpH00+bHVVzNOBhgi7REOrpxWUMwOKTcmTxrddyj0e3PbmV8KFc\\\\\\\/ftROAQtvyTNmpf3XK5t\\\\\\\/uMhKe6Xi+6FAeD0q3JoSEQtHx\\\\\\\/un+5SaWjmB4NXhHL9g8MKVA1J7BtlyATP931b2ea\\\\\\\/LMZ9tPQxEQt422teTwkfaN9dh1OYIZA6Z1W8QiY6frIfT5mZ9oxC9lhLmA8LcrB3VpKsEOJgZ2xUN549OFeTLkbCeBGCaVTf78r7HFFjmhQoXJsE1PrpuuCHjnVWW4G4sevzB2EvVcCGWOcPRztNaFap8l9RkgqWEIIV5y99opNWvE7FLB98qHVlUjxeoNA5dJ1iqGXzdVBo5rz62Kt2VdCzK7R0+QxuDO76FISwwB8uZl0PmOGfno6LHIHsJakaoON50De64K8GlB1h6StQSsR8QqJZKwtQ2lS6h8JxRUM5KfrIa+ozEmIedDCrw98qS6p3HInm3upW1UvE5wvHH9z6FWbaMVGBuSsdvEGtDpHr1bnzTNTEbTFBW4aaKP6JblVa\\\\\\\/4LsGTfVLykXBYfTBthbhcKGOnsOx9JJbrriNpgh4AuPMb2F5MJ8Dqp7O\\\\\\\/OoaBpWkET6GKyTs820idmIHVaRQ\\\\\\\/tH9eY\\\\\\\/UK0ybtXTSnkxBV1OCTuG9ZxJIjZZfjma5E0H7hDpk+4m1XRxOpc3PwH2z8uCrF1cEvBVQsuS2Nwq6wT1sa1RayoJUAY7GRdSRjKnyC7ACDGeQc7nJCoJvgfS58sQ7hKFUGhMIM9vdMBHlpvfu1IXfmuI2cQ7f+N5JWtI\\\\\\\/pRlH\\\\\\\/ARoTlk9ujdszjuMIgDYBCw52c5DliM\\\\\\\/79hKUKZH3mKTolpmxXRadGNioEvo8+rqtSrKgVzIJjbXDyPGeSWbHR9+ZnJpvtGT4I2BaDbsggC67Y9Ym+7PZFGpppUOHOfRrjncDbO5Lo8anqKc0hTM3H7h8Gu6IddJQhE+sOyNpPl3OrVtA8q0Zk7HWy2+qiVarUl5Pv0Hnm3yH39f7tUtkbPKiyioRZkZSPJmEU9BlaVoFs0oGR6ibhAZ3WgnKwGOXmAkgFsdoiTn2Pmg6P0jzG7WbSihm0vwB+BoOiY+tKVRhWlrAZNF27fFw5Oj6ZUM1SuktQhrrZqjEJ4nDZg617XgiVTd0ml0Y8ZMfTCKMPAzzvgHW82qZNWPGKin8sp2fUZFwkyNS\\\\\\\/48luKgHi4b9mxqvVWtZ6otWAbg3iPJmehn05h9\\\\\\\/IjhEcP07kaX6n6c3Kv3kGjSbm8jEKK9IeXdUshBNqwxQbJN4efQJP\\\\\\\/brfnlsRs5WqTtk2VNqPfvSZUdKGvAxazNc1rWbOlxvDflG+S7DO7ow5o9XlN1mzPParNwk2dCsKXPhz3hJsrMPd8ub+OhvWETquOMO+1LQD361pBoi70U6MtsvDyRzjLY9K6FuVIu2S8pDpf+41WLi09CjThjSOYYolDOn8B3Rdz2qm4HxJ7uVs02XRVFJWe4UkTCmi6AR75+gl3XWi9GdHNmWiCVffcmozux72ahdAQhRZW48e2UEfd2maaWvpnR1364wg6xHKVKMVd7\\\\\\\/kBH65kkaDb0bPemvzP7kbTaTDrDc4e1IxOTW6pxCPfw9d8Ki\\\\\\\/wR+v4FTvl3j35PwC865Afwz58Ku2y6EiZ+sjXz2557W2xVmVIP3CGf2nf1rkeMJ+amIag\\\\\\\/UHG49V9+a3XzzBp0uW8SnGRv1krXUBj1Ck438+DeqyooWBeBWFSr3bi9\\\\\\\/QbYQjQwgF6joMNIly7e+yPBDoN5nbBxalJ8o5WBhcuUCtzFx0FTHH4MyqH7Rx\\\\\\\/Tcc4XDcjef8uMTh0v+Uk305QMiTHtKIgKNUnm51dnoBjAloXTWqx4W1Us7BuYAB7L2MiJMTL+03JB0T9SkIjjDiSy4Apk\\\\\\\/7eiaUV4iED1uUpAGc0RZYphwP4sLSwy57pV7lg7t6M9ZCl4OzVds02qskXmYmlsCl+y2pkSJwxm2gb8eo9yGGdhwXC7DoIB+CXFNFOzWNU04jpAJ2CvHbM5dKH\\\\\\\/adNCzoOg\\\\\\\/1fzkxGSCviTqN\\\\\\\/xVFmmwAujzpN1q5WirXwVLn2wtzgVA+TfL0ZAFKr5amejiWAezZYWa55InqfrPGTt9GWlRCcHeHRMkxTXWxViRaUKMobQfLO3rkBkfvXsGXnYZMuqhaDvjthRVb3k2hM+E9slXWmAPg5hj+ynHK5fDW7tfJ\\\\\\\/HrlAHSNOYuM6ZCUZ5peW2HQO5EZwnqyLhtgH35i9aJc9ue0ZjHssR3\\\\\\\/KsyZx5F3p8hPNLgAEYhIxI91QtzuUjQ8nG4Sy8wd77JLKWGhhT\\\\\\\/1KAhYqqvdoh1F6vg156PRN61\\\\\\\/pOH9CviYxlMM2txtpvm+1IfmWn3lk6P3FG2tNIFpIAZneSoFE6FVdYcsmVXMKSdsm2Hyoq9V2iLyimsRXsUi+rigFbVPGDq6Sd2iiy+3KI29g+CbeTbzk1uodaLdLg0Djqr0JSJyYyIdbK46Bb5u+ggZf7aUpZ5duJVB5i8Te8rTu3ykDaUBYczOzyKmtgnYfZn9WUeb6HGBs+Sl6sC0U9taYBg4TqlIkP32sI6+huzpHALVLrZYcKQJRM3IsYY3yX48BL+tHLERy0bqbU2lj2OU8H1zFYkgspC9qcmyX63m2nrUfGUqTlX+eDTLOPNCK9LX9U4K7\\\\\\\/tnMTR4qPewHf1UAdmT9Oyo0UDXS7mJ7jnEGkPTSX2M3XSU19WvwiKWmkPhXlct7Fi46Pd4cQey30PrXk8FbiLrTQjwSCoZ6Ng6ZfSioZFPhrq+26jc6UF5Makx5DQ0vxFJSQW9BDOT9y4\\\\\\\/6wENA1SEFvPli9rYZ2whGugiLMhgJiFWxz2Pl7P0MkzM4cqqXeB9qO3BZMyC86XdIleXvrh\\\\\\\/NrJB4cvvd5GOPYfoiaFDm5hRqXH82uF8GRbaMoOoPCbhCUXGBHtZLSUqqyw91GYnx2cTbCLV+dqW7yUFvmRsnsQrD116kdVRlpxgPHJ0a8hKPtPulgy5jTdHNHfCXW7Z+9VNfaUzbYlftB\\\\\\\/00dj+nYTijaFWa5DD1RPzEqyrdLDIEKumcYRYaD0KF1omSQ2KSW9Cj0OJ+BSEOMSexQF5QfOI1OjSU2Q3bGB3gXDeliIUy39RxfidwX\\\\\\\/nzg5j6l3vI4PtcmxKKbaPf31N\\\\\\\/1tNwVUX\\\\\\\/tRafyawsJ1J6oS3GJdRVYutlsUlG7ltRQxUl3kHxT8l9uKxDm659\\\\\\\/r2zykLwLl66C3FIxTbdpL\\\\\\\/3yQVi9dj+Jz9WJs\\\\\\\/\\\\\\\/d2EwK7oS6vcc0WRofQ7UiPP3gCtk\\\\\\\/FmsWWoeABrZ8r0wbm5cbA1FMkw6wPZiIcXGb4IzGiGeIh2Jpo0Z46PNiVJlGxllUKpIufvNlEvLd72MW3S6kJ8GWlkDTSqGHfMaVYSDNSruyV+IVg1NelsT6ZDxLgrru17q0+icXEVQo\\\\\\\/BoWgY7SCziQRTBQ1Gcss0AG2oBLj7BU89K8lJIuYCqJNlvZ9ImcLoYj3T8yM4QPTLCtxxL\\\\\\\/gklKsn2u27w6pThnu6T5\\\\\\\/gDD3h2q0Q5Nn2nNfr9YxXdhGfLpMwCqzfon2erID9m7L+XI1\\\\\\\/c+mcb+5v274PStdGL9GYOsEiHinh74IofJmeA9bhtn4EpmVlJIGR81BW+umcFmLRmtPRHLppvGgDCLwxJZvyLLP7X6zQxJE6WbcdlC2M0o9r7AaAglcRKYDY8lpN1953\\\\\\\/Ejgf\\\\\\\/f\\\\\\\/4\\\\\\\/UON9Lu96JWgKXBmFKwQNeFlwySUTBvZJZuw9hXoVF5ny5vvKUiAVjh2iTqQiy1OvH2n9w8xDNQKyBvrUhPXD5SH0yWz7NJXYZ8CzthXNlyb3GFcRvxepJSMFXgTyHLFUwP\\\\\\\/dV2zpz23RubyUglSfCekIBrngP3V1vXrWp5LHD\\\\\\\/SU3j0uON0m8Zk+p17XFNC65R2cm6ZDLt8F\\\\\\\/YstbgLTMfPkuZyrZYw44nOQtKG6WRGOC+qU4HZf6UwwSFq\\\\\\\/7cKqnONlwoKT9BrbgNciTjZMkqcyndf4LMpZZWaVH+\\\\\\\/SuNkR6yYT4CaLkj2ci+A2MGvmQVNvv6mumcrKixucNsdz+7GGWDUzHf7ZqaOgd2W\\\\\\\/QnR3TuQeYr\\\\\\\/OL0lDae7rWDc8xPeR608+ArthPfNNy79G3vyFqhcigJrozxaroKiNo1XQ\\\\\\\/Xrq6uleu1fNdpS0mwP+yYvAvjatpyOziv0Y4JSvk4ZCRv9FlrkhEtjk2QL2ZUnKa5iv6TXNIoGg\\\\\\\/Sfh3p3fQqfMTuUsmPHvzFznJ5DhOo549p5n7XKrpv2vBlBa8SPl2mXkGl4vBzKBShKtGpqJToOk+\\\\\\\/kQz+K7DUrjb91W5otYZiVr7BNjjec1Md5i3oeiczgP+mhpVa\\\\\\\/QAdD6radKNOgvuFoC1D7sE5jhEyIQKG4ghi2sFikeX910lOl8yKBGdGo44+742w4O\\\\\\\/b9be2aRQ78V04MF2bPU8FMpjaUTCIzvvjFBjQHsjpNhWdaKeCYnIwmbqUqNh+dcaig0l4vuCt7qky0vM6KpJW\\\\\\\/kbjg0DQroMkl4CuwOR8JSl+YcCKrPE9vSqCeo79RIeH+SSAH7TzjPfZ3jlaY7HouIl5DZZI4fvGf3EF4YmqhJ7H+nBJZHzMi7XhBhH35trQM1dkf4LLVoqt8q0vvqhNAGsFwS3DTKmJQ3iJEhH5ETB\\\\\\\/MB+oZj0r7+d8rOdNDiY+1Pv95jAOzSr9yftGPjS+BVU4UD\\\\\\\/+9M581k3SYvNUI5T29JDVmMqxQTBz6kG\\\\\\\/ErGW3ibtlbnJ509ysAWj12X6x69h5\\\\\\\/nVESSzdnBqaBIHXSNJzVUUECpVb7aB26LRtZK66rOcfV9kKR3FcHzJgZDH3VS1m536LB0ngpNdo7FprJrmNPsGE9OMBSF+p+Xh2efgvdG\\\\\\\/U2Rpi1nepqEf9Lfdjg5XC80CJ0VvFpCyeEQFR9GINc7+96oFRWFDkbivqqcAmlYWBk6syFmQ+kPLFnQZqYXAMuOKLOmo073hjZqcUHubPj0uG8AL2L9h64MAmrUfIM2GKZVYMrczPs3e\\\\\\\/ps+CKiQiyUs2OHEKhxwbCtjGv97wWgWSx3BvIN5Pe1vFg9Q6rcBG4Y7y1fHIa0w8PmiKTfNzYjp4bOSF0vSgy3uRKnihfEYulfPnyia7UVHgLIldclpfG\\\\\\\/SDvd\\\\\\\/iJruuzic5yCuH\\\\\\\/YdQyONVCDwrZS1N450jIvy8iTiOisZi9sAbMNwp\\\\\\\/elIClYg+F4rN406QSn7x3GppLdtLzm08ecuz30KhFuD10RBigk9zO9zUMuk\\\\\\\/zDhJMsuCyMlb9dHiPsOe9n0IQ2LzFgFmJt243ElKdSvJExEm72\\\\\\\/fQgKkYlS7NDIQp44hdSWJOvvv6FzIFqIvIPTyWL4tTngoEbgCO3IarTz50yMe+MRJi7Rjbij4US27QyV2HAmKTZ\\\\\\\/bqa0FJ4qGJzfgYVQmHWPYIjy8+cxiEcPSl\\\\\\\/hUvMiMoDKg7BFyG32RAKZQReIl3MAObwobQExwyOMkBzL4N3RWGCJeowI3QqtK22yrUsRqKDvWeQRn8YjciKMnC0sh8boWvld4RyoLGHydyG\\\\\\\/mY3ZZDb\\\\\\\/m+bWrc4I0ck\\\\\\\/l6yNXBj9HOl5OcmFVoVMnWpFahRLBfWHkibIfihP37dbovULmA\\\\\\\/aLTCBg0ks6++RWXj9lzX7ig9s96V9EM+N0M+4QloDYDLqPJoFlPgoZV21Pyn1t1X9QXWBT6Rs14WfWZ0WNPKu1x3zLwSEjLg+gauRYXHXKIPprINKfQWYFnKePN+EXYXQWVcGU1FCVP3o0KVsNramy798IeY3qmtt6pVEiGDqJ2F8vcxod2ZB+jLHBoUAMUvO87lCDMbYvZgWhK3ZN3OYqo9oX2BWz4NRSkbxkk2CWD\\\\\\\/mD9pjggNq+HdBK6ZC22iO8KbGuXN3nLmH60+A0Gleo5e4nGx8HF9xxsFenDa777LOtmbf7\\\\\\\/1qWKn7Gj72AYKb1LsnTbaHNp5XARmxzX1vyIMIVPKZYMg3EiF9NqL0mE14ydXBpBj3PoiAgTYvyhgyzygdW5MLspHZiJPGkN43Gdwtj0E3AzeH2raF3bB411mnT1KEU\\\\\\\/FLrcArJ7n9wllBkIqJhtC9LHNi3J9dONRJyyjwM7keiDBgCy3RIp4p59cZXVXOIlyX3rDDbUBQrW6D1y4akYDLEsikofyeMLowb2+YaXEgKqK4rNZlJ\\\\\\\/C3UPlq+dFSomIyEi\\\\\\\/WoIcNl9QAmT1LJUUp2D+HFe2f+FuewZSgQ3jsT4R36fFBj5EnBVh2INLuTNqxayhCrkmJhPlV5oabrpPkaj8WBlT6wKRffm6f4TmVYcPoLX1ylt6r8dHUWktKdmXzpMC+ODQ\\\\\\\/l\\\\\\\/OND8bqhX1BkqT8UZgKdOl5LJt8vkmzQu29t+6jOihT2ZV7qN180lvx9UIBlkSkDPX8uu2tzgSma3Qws1VoCzu01EJR6SLQ+GEh8afMauSD6Su79p3tVfWCy0dJLTSDKMuIjlEekUzMoZbjl6Hd9P1ykYXmz\\\\\\\/6YDXpCIUK8NnOyJprjRZ99Suln4faz6BCkGFTXbxpgiWMc\\\\\\\/yEst7RN1BnSXbwzgfvy2XzLG7ZlgNAK4NlqRByv50zbg9+eRCR6u9nyOR2V06QQOTVMvYjpXWFfd7tnuUk96NUEaBo2DxOKbNkxGBuq\\\\\\\/gJbiN1h5El6yrLN3RMEq0JKIWsK0Qcg\\\\\\\/NNKeApYoO4t4NbIhVAhdJQzUqZz+s6GkxThJ0tFDL8FgGVgshmh37shFD5s9nyzcj1\\\\\\\/BeyDU1ymjSwDVBxoKhH2JFrcOAq3MEUdmiv\\\\\\\/t\\\\\\\/fEvBk7DKE6snNuoV06L2O6VtURAcbKCJXbpMvlIXTzZXSKqkCoI45qRmiNMZ\\\\\\\/jB7qI\\\\\\\/8FZHD7Ru1eSxdfoIXoIYWxrDTf+CzIla7tAfkPy8upoHwFLjKpelRedVImhnEDiYduQgrTwlzekgBYoJ7mrXU0oOhtrRnkG1rnD23JRZrOUXu2t8xMrCPL99z8eY0oU10xpROqKGx0MiU2ZMBJiAzQfhEmXoc0yJf61pXA50gjFSTM+mQmhFWtsT4BuohM\\\\\\\/VjQU27iVXhQlyC3zfBWaXFaH8I4op59PiUvxpLSSsdTLFVIKfDTZ0FCNjPnZfoYdzcCnjdYLSF7ILcBCJVhPQoeRFfFSXoqzQfKJRb+9AqB5x8bwpPltr6HXUNPtFQLRrec\\\\\\\/m3HgGefBDavoeBAu3295fqGbtIjxkNTJRfTeLT3LwqqE2wxPFXFvJ06csu+lQTHT1qsQAht0jVAQTwEM2oLqOYA+0VIyIhlccZWOy0P0T2IYKmJuwVfqEE3qM0PB3PWxKXrKhnD9tjUaqOpPOhaLckPmjsEVWLUpwqDZPoEDXdxh0yToE82AefOxUp7687v\\\\\\\/0QQFC7TGkdYNyTaVXz08QZ4SxGLxkXTkr7N1FDMVcErwjefX9RUMgU9xm2peZ\\\\\\\/pyEFJwC6vzE7L8LxbQn7I7zDdR\\\\\\\/4hY0OQ\\\\\\\/4+U1bU9Ae+IhIZQQ8LlXP91wzXm4YjGuTI5wFS0fvY4hPs2t2O\\\\\\\/VHu1ut9+s68CeELQtRdDQU1h808kKtAXiQ\\\\\\\/LI\\\\\\\/c3XFzuLHedi5oTjkMl9UwEGB+w+qq+jiTV0JAHc+xD4Kob3SIZCIZz61b\\\\\\\/CXcj9KxlNs66TaszyR\\\\\\\/9VnZtOmz4hsFcyKuQt8DZOfy+rKi\\\\\\\/Ej+5PMh1wFEH9kcEXfSb5i8+Ep7yXglhPfx8QykUvAlSP3dL6wncOPagfyJyMJwRzoscdMQgsBH64IibabW\\\\\\\/kOyL1T06R73Ytp7H9uT0t7u0+RRNQWkOHtGeH7bxnfBUdnsR0IMohpC1wo9yGdBln913rf64GtbaiOKLtEed0JqVMXJAZPGJBQIfg3mOwJT+pI5nYRWs9EvhFDHefXRkFKob8OuL9ym2+KwBzWSZwSqvB64ryw4E7k6ippjaXT41r3LcL4Ls78rLvBZWW7MK52lKj\\\\\\\/l33R4iNMMSqJGkccZsM4IekUzeGO853lTgPXAR9i0keCUlXuVA84MrUbI35PytiIEHGNNati5zE8nfRmzIaXYVi5pOoz\\\\\\\/o\\\\\\\/ue7J3EAw9\\\\\\\/HSHrcBHI91MkTCHq3kqQ8P8hheEiBalUetulK4E1UCsjZkIaJltCitiSk4PW1CNmIwkE7YWdAjuM94bC0ExAhLtEcqYkvBONd8775xr3or7JrINwIC6TqwtFlPA99RPF9pydEDpaqjSOGaxL3nWfUU51Do8CwbCT208lRgVxPtiM+2WRb4tlUCVVhV+SABX1g0DgUlWIDBqYRa0CExTxJe0VcP1yWXcawVU+Fxeio0k30QfyRQ439X\\\\\\\/uZCXbiT15o\\\\\\\/M1DeDK1MFbF5W1dAPw6XK8X8IROce+ujJZ7ll\\\\\\\/zb7xKUNadeAHfsjBFvUexqgiZssviqBhAKdlQ6ByNx0+G8tJcXLAGi23E6l58el8v\\\\\\\/\\\\\\\/bdorVRFOxiefNJjtPVa7qysk9I\\\\\\\/BG99rSCkO3znPMiuYjVglyb2fLeipM2TZpycn9\\\\\\\/icJKLAV1BkhB6+N+jc6wZxx5bnv5K8+18Bm7M10PVheCK5vm7ensgaFqEvvhDV6COMS0OlyL30jOTKaGfTNK\\\\\\\/64k4PQfx+rxgHWf7uL7PEGKqMDO2TCIkAcQRs\\\\\\\/yw7zmRN1VTJOlEEXB6rmoFysitrbgUwMPCtpKofmeSzZMSR+XU5IRJgT80cWfPMOCYitrmdwmIZ2BXuO35\\\\\\\/zHu\\\\\\\/c4lP56Y1UZCOST65hM3tNUmDLMBDgcsSrf1fUTJmIdn2SRiFfmWY3oVnEz\\\\\\\/ZPb19FBZNLB0zaFmvzD8h+jhSNkY7S2qKNCUYscv0lvDYJuflvglFZRHiI3ohqIJtH\\\\\\\/fDckZsnB5khEtt6ibw6j+8\\\\\\\/u\\\\\\\/ES2lTbDA5QYwTLHz59YsbpylXX1A+3BRCgMml0n5lTtWQALHlVBAvlRFakPT3AqjXW2ckUyRSyEwYGtX+XImVTCYLG6OOhX+\\\\\\\/1qlEczHFLIZJvqdvE5PH68tgTm9tGIFvhyfQwWbUnmWLI7nuvuyeplbq1PIO0JKUxbH9cps\\\\\\\/e6ksWOtI8Linj7rQm5z5U\\\\\\\/kcYFfuobGIzzjGmoHeS2jHGlIHpZFRnDyeIHj4qTy\\\\\\\/HC+WFW94TRCAFuy7ykEz\\\\\\\/D1moH\\\\\\\/+Zcbzjok4TryEN01am0V2gOdLqhF4Mh23xFC6E1crcKB0HPhsj0Mq7ZmLFOvlES7HhfiDQYKNv7czZxDDRcVcaIyVsGX7Qy5WA1Ab+DvahwbTVwpn8Je0Sbt2RfsAAve9mNdjIjDFv24bpRthDLejIMwgTs8pPkhFJCYzHnyqhJ5h0\\\\\\\/m9J0XThz87F0iOOOtLK+kDoeCsG2bidAQ+Rq3burVctHGbrpOjPOx64K+1CE5x7Rmxx3HoIFwTy2jMt6lAgW\\\\\\\/hRS7AKWZ5jSyb6aAVidVzZETan3YPaiakkef\\\\\\\/fJDhcYoRqGqjJs843+25W+zx9oZZBxXy0H1IbKYrczuoSwR2PLJq2iEmd2XRUl5nQsfrmHefB+2QJ7Pd0yVJORBLfR3JLaIejzue7bOg6hm9SXXAUcRHWKplvMAenaht+P036hZnBakD0i6jiF0eGoAdtuP5Rr86O5\\\\\\\/LgkL4lGnMgFX12bPzpmAOLyGft2A5NzK1IezsBvDi1+UwpaHjNk9T2CQujDUB\\\\\\\/fz1UMWUtCz7QUPvJOGgSDzz+FyY4bLWWKPDyyvSiGd6t7s4Mcjs0+CnHNZ6ciMJ0+Wuz3WdEnEULFpH58OQC\\\\\\\/\\\\\\\/aGgYKZMXOYe5lbhkxs91h78fiTxdcIDGrhZI7wqNltpFowaBgKhgfJMs9L9585bSybCeSGiLA1qnBFRLYgipPN6TdyVXasSRRp5A7rtd+OOOpVpshVFUEjpK360X4pGgfWq7ZkD6W4BU4vvegiv0sITOYkDIuE15MZv8\\\\\\\/BfGVny90XgoE+QerJmkCcyc2uVpzU04mJiAaEpH3lGTdT3BxU+jP+cLFtv00lj3xQdboEVM9vmlnl+Ijwrr1BIpyyxJ8XwpTGFVwPHSB0XvTixhdC9J+4r5eQxkWGAZIww8YI0zseYCgjohWFttLlkBaRavxoljmw6YF8RUHTypcsh6jzFSIhDRDM0uSoRWQNLhkJF65AF7yuNB0OFZhBTD6bCRBPL4KvN4jPe13vwqL2s5QxJIwk\\\\\\\/W3ikOTr0V\\\\\\\/QkSmkd0zKffB11Pj\\\\\\\/9rQ326+ZKZGoHvsEpGVQTAsjUNxWMrAvtZDFgUTBz60NvZpbVb7iZDp8CT6vkZi1jXbMUDNnS+eb3iJACMFn9Vfz4iP3H9P2HEwoBsUZ4uKfG8OglemSXNBkmTUQAo7Z1sLIW6pRoDcm7O1q+WGt6M0R8ZgLqjE7RcSVU3vxXVTRcW5vtxxGdYaeM5TQc9QiWOKAIk79OZr0AS+mbeWbsueKbOU9E+xAxwnhfsQ1jWp+wpSkSd1fCz9dCw+bTrfmA+tOQOti5LvXify9VRmZ6jpcBBVtHhh1+F78ArcKi6GwkDBYQuiCC6ZU6EC14UQ9C6Fvp+9sjMPIK8b\\\\\\\/OFIHrhsEcf9HRrkon+wcgVvyO\\\\\\\/nmuwyNYvw0nB7Huc6kqXz3NkAumYjHQTO2nFOjU5JofiTgxPPZM8vRZX1B94s8IwLM7KGKPUChCfjvDPHK++BeUS+6QmZMue\\\\\\\/2jqFXQcotTxrU9OC\\\\\\\/KhJ+F6RpvJUUNnaKFdB2HpX+fJv+JVC1qoaie8TwUNRcSs+NxoDz1ZnYbyl3AhDbYk6DfFMvdYw+awksnAUVC1Oi8b8UnpxqhCUC1HJKcgFXM5uxGxcxIfNjWTRWFe8n6egjfwNwTu4\\\\\\\/MRJN3D5gyqWFjy7Zve7gL08aXdUbtk8w33zaBQeMJenynOng0R9bRVBU0bpgMZaXcfRDgi8BFh\\\\\\\/KtIMC23El4WKWjKcFzyY6DU8LVUxAoaoVb1CmhhIx0W6Gv\\\\\\\/eoqXCFKcafnom6KrPn6A1bPGBASojA5ixHu\\\\\\\/k\\\\\\\/\\\\\\\/Cbe+ktQvSg9olyPiwTuVByqVHEQ8NGvF8UoKGgBwTNujTacchUICq5XigG2IwkZiccADbvknmUkePOmtdV\\\\\\\/Dv0qak72FE4IOWkWoroXE4+mhmOEYC3pV7qnU\\\\\\\/F+3GABDVcpF8nsA55+EJjLWh1rKxT9poojXzF6g8mo48t+fD4gSWy3VXfDjrcl6sdt+c4uFM9WLNr69XRNXxi2A9qh8wmPA+qwHGNtp9k3Fxioli0kHjLt\\\\\\\/EjEa49GNtZ\\\\\\\/tZNd6C4z2ALAHEu3iRr1BGFaXqoMmwaLrIzdGxrO0tbLGPo00gQ5oUYMRiGTcF4mNIRqRlJkw6w9bKP+CuPSUb9uyzj3R9R9I3gx9eWR6aXl71YdRImCWR8mhiIOY8qJSuz8FCQvqdSg+apYQp2ca7lUjJRlDY616MDq1FvyI2uFvxCcP2w+eXI1ADNoVslLmElzE5sj0qwp3YYKugjwelAvfluWFJS65zzK7+gCEOrbLVRK6VSTsBr5rs38609xHoCPHT8gi5WqFTaOacRYvo\\\\\\\/LHmQRF6PpsexupW4CPvdyKN5fw69OqfuvVE3Ogh9rs5zRJckpRN7RmPvu1x4GD8iCEpBnSQ0u1qoMkY6of8K3gIMZMVa+NIGzWeE4\\\\\\\/pY6pnXwfWIISWcioii41FEALlOnK8YbryW7yn9SkfHYbGyGe9DN30L\\\\\\\/XULYwUNGNUHM\\\\\\\/dYpGiUNHyNEucfknlUicQB8fRqMTyDuO2ZSK9kj4vJkh0I8fYvRC2XlN55YmwHga1P9DCj0IRwdxvSLJpS77f7Yr+Pa0a2jPDS3h9LlAd1DfI1JZhhD+urLRSITW209R9rtH70826kLb10hb4qGnTcVFOxxy4\\\\\\\/71Qp0NCaHr2l7nQ\\\\\\\/vIRQciQ5RBDNXlaHZiTC5+SyVnvbOVhA9KXi4+CIVJPBxKwhnph9QaZ0gzjwvuwrObD5mfuwVWA4yiLpKOOgl7wkj7zNSY37kLKpIiiK6efMRmjWvKOKOGllfuJxH7hODIxBge+auBMwKQq2FSTmxUroq\\\\\\\/rK8UT3tNVfR\\\\\\\/WfrlCUEr7dT9IPwDjoOYx3CLkD5LjVv\\\\\\\/WQM21k9HGUrSCPeqNemFvOkGOM0Vqh3wjFRfdwUDlyCvhnPQPnAj4UjnaBL6shqoj6gLyuyi39kwNP7gLyaDmHpNwqNGQ1ei380UJUekYqNo9VqmbKHdubJH2qsln6mJ8K2tbdjbuBZCBf+GUJlTdwc36pcVGNFgwnkiyV4wIEky6NhbIpsEdnUwbvFGnddNAr3hXFvIs\\\\\\\/jMp1YCwqMBxIEWpnrBNZRUgswZaWcDe8kQlJlEjR0qgF\\\\\\\/0t6WMNh9YosdxHPiYzqZY85i7wyDXoEE73hlLxj5gcm4g6prVl1L58HhBrH2jAYL95IlM7GfOBEsZd+Z1T+FYD1T8AzUPzPSLr23OQtTPUAuJPKscKDnfa2DzCxqRMETNOBQVjqPFWIRoX8B6qWPb4gGnoaw7lOtRiPNVYmdeg+ezthzR8x8S5inLnxmierOzna9+Gg8ILHrlkYNbRG35PLi09NH0lNu4Nt2S43wAEXTM5Hp+n8lad8PWnpbn6ctcTcGOk+7sQOTITOBUA3MSZoN9NVDz6dA2Zbjv+r5y0QpUsypbPsMWeQdhT5wWitsVaF3ZHZqLbgcSXvj2mwnvyCB1QhdNKypwO03CoQooF3L\\\\\\\/YR\\\\\\\/VkGobgD\\\\\\\/nvUpgtzMNfVA352+2zUuTI9mAgWeYOwTs6wChgIPOltul3Zr3DjYW2bxsVjNUX+RcyR3Od33NdlDhVQxj4Wn\\\\\\\/W1s6moaEoItSD8nCqC76gJVwJv\\\\\\\/PBN4+ZxD959SZep5wv9Fo6Zm7IHHUZ06XgVA1bOlGAtVxW7mLIE5rdkF4nyop8MHGYnh415n5TkJL\\\\\\\/7FnMgF4Xjr6WFaMtnNt\\\\\\\/0tned1V\\\\\\\/pXOUp8nIxykhekmOJQKO5nij6pGeTBX2R5z3x73dDdOTU+SM\\\\\\\/o4GlAbT871Om0rE5L4lN6Bwrab17sUCAir6G37kWA0Pbw28EZIIVQYnAM0doiJr+bx28mDmMPJ7DoWVVzG4KFDnMlqzFAC\\\\\\\/1QPoT3pD3b+Co9X6YtAcwnoxQ8fIgJtyblc2loI2iyZFKixH3qoN6bGfCScHvaeYf4FH+X7BKFg\\\\\\\/XutoGVoT1qHHrLL8GzdoU6l8\\\\\\\/GgnqC4Zxgb09A9xwgnfKPCt5HKrpXGPdaSpec6biM66rQH+\\\\\\\/EB5YUmSpGfDl5OHzAw2DUEfr96tcHBWUEaq+lG+7c1fh8lmEr3VonKRpVfo5KOFGJ3KXt7mq+ABHNNGhOb05Pcp6yxHxcyqJE00ksw57FtSD3zCe0jLO2W8MCkc34s5hekQ8UIczNj26FXjBtNG\\\\\\\/m76qIIqm37mVlScYLMp8pkCBURfn4+YfT0RW2bAiPde3BGxlhLsHiS50Z9a3\\\\\\\/km60REwNmhMMpIeyOXIptNigb5itV8+hhrbQuZ3H5r3HMZFdEM4GJAJjz4SWvemCXLxnRkvKoZH8K3zMaQIn9MTya4GH6hzQzaMKcNh5ZAuSB7w8ENeFd3Z8qSDZZn05mG\\\\\\\/kltqy5VTME3rmcH3y7qwRADnz3vCkV5pD7sin13rLCxQWQgbulKKo4iLd6IA2sJ8WfCaxKbU6kqNyYZOwJp5H5a0gH38Dgq2fLwn0qD\\\\\\\/L\\\\\\\/faDNCG19j7oSDOktmiu9zIucM5mDcywwlvAd+ct4JZFl6Oc08giWTK80O3KeRdVoYN0jLrm6yeWO4GMmhjYjdmA8khzewSXcBQkgqwQ7kz9ObIp4YlsSKVV1LRn2JLS7pN3LMbHpIX0qunq6N83Qd1Ws+UGHLoHnNzKl7fbwejB7nancnde+NfSIjfe8hPO0xSr4+IkXX0s5K+hnHyXHLuKF9Ja2LHRcOGP826DexWvDRf4lmDQMI6l+RUwDO80iWNxYRZlXbfV3JwvPxakpX+QUVEtBVV2af42FJBXo3KGeOYuHq3J3nRXg\\\\\\\/NvyRWmcHm7DthnWLxaW0f2JQlOnZjdI2+tNteeCYB\\\\\\\/2TppO3qWLbccbpHsHKktzt9gaAHj3aC5DKFQMpy9\\\\\\\/MCmwGeSYM95m7HKjcCjuNSynzqNvKvDWk\\\\\\\/jt1KyOUiHKnwBqe8Cq70ppnzn1uPmDbhAxDNO0orZ2o9GtckYHtzskgybBWMiDuiyfWFoc1321gH9V5IjVdU9xE\\\\\\\/7cx9ByI6iRi0SOhDTDb7jM50oljP0yRcBBbPGk9PcVdqC1atiu0I+WEVu\\\\\\\/Eksiau6lgzKUG6tp+JGabUmePA85vnW\\\\\\\/fDKtw0HdvebBR6M+HcPWii\\\\\\\/vvMJGE+gPXizGgkeG7pbbqcozlLhoD6EcqPFK\\\\\\\/ZHuo791o+98w6lyVwUG4hO\\\\\\\/KgxnDTeM6ataqKwSY623zQsHUEEPfGfR4EP3Y3dcIddRL5PkW3B+A7HkHPnJOWfq2fzOHR7obDcC3cL9U9QbLISuJRo9NWlKqubUq6XYS3SzTAUER3O3u90W20UXRuRqlGRkYHPBh9WFKIzqBhBels35iB9dHx96lM3uZQ2ikTtxs3HuRzji8q+7HFNxW9xgAHov+V9FlniDqTF7IZPr53RpNZkXaKZ3JHNciGsfvbQqteqmT5+8eDCa6CVemu0KSl9es65IdOWucauQtq9lkUX8TV1SSilnaUd399M8vDe2uEJE\\\\\\\/CUtdNzqIJFLHEgU+d3n6aDQGX+4WnZems6r7eER6lwMREN7VbjcgOJ41id8xziOupjW7HS8ApTLxAKEpnhAZqmXcZk7HqPESJpC75ADWayji5ODaurFh5wM9gZzfT0O7RhbAhA5kJ6UhHThhgl8BTev5xg8xomXMiMUIGchWfwHGR\\\\\\\/OqPgR9DUHYi\\\\\\\/3lhRJE31uDytOzaRfhkRHxg8AzmXz3BK\\\\\\\/XgSSAvLFL8FFH6oGkTWsrmTR\\\\\\\/JZgZD3FGMIgJ2RHHVHuHsyodRn\\\\\\\/xgP77L9NgEdbJb\\\\\\\/12kvRsMgQl4dCECLGnW41ZjiBR0uqh55UKvrVIyHbvnLksRBx\\\\\\\/INIJE4M13c1+zz200sX\\\\\\\/QMBh5xAPiTS58raQGz3V\\\\\\\/+GuiIixjOg6OLftP\\\\\\\/br+vmCk2v5dAVO5BOV10LFe7jaCd5AgQWmTInNeuawdTpnEJV54ermaHIsE52YDl3F5plYcVHENIUlz20i+RG5n1pFAECPmgmxZfkmIPr4haxnmtEB5yB5DWXN6Wctz0ek+espOV2pCytm6kdzCMVaiH5kJZZgNU9eUm7nTZXNyCPloO0W6DRWWlS78L6Gu6IWirTPwDokyi2nNUEYSEuRx1tyxSsQ0SP5UrV8MPcXV\\\\\\\/wB7E9\\\\\\\/77i741JesFFB4wLffqQ3QiiynW0SRetAKmBSq6C8SFIbmaCgU2c3Idi0v1V2KPX\\\\\\\/UdWnLMmsXh7aR3SaaocPGkZz9c4jqi63LMirG5OiSbrxNArOkQHp3utWGfF6viGnH2Rfk\\\\\\\/KFJkLWLSsKMDxsdWapsHqazVucomQlXiivcSxEufSy+sYtzLQY2yMKT520b3mxG1qM1sFeyLqmgPvX\\\\\\\/3XuXq3qo2iv9hpw+79B5zg64OFNhsAVmhv+ay7+i0HBm7GSxLPjHorEgP8PCFE90dBu2LGoFeWFXCGaXOikpinT1w+GXrAa01\\\\\\\/RzY+KjViQIhPY0mVmsGEedk6Kj3BChIWWwlI8vIFjC7GT5QWPd2\\\\\\\/O2HvpziCr0gGMEqMhDAXdpap7I4NrV9omLp8VMcWXYA5h0zs0vjPom9uBJDiHQopjac\\\\\\\/BXzZAe2RQv7RKCdxxb4UG5qcB907OCj+TWUoHVCzhvUrIMDRkf4+crMKAy0TtpxaOGDYJ4n+vMdk6S2vSB2wXr2oOWW1WHyZLis8qc20J0peBOjl4+g5DBMdiiW0W0Dp86IkBVHLndyBZdqQLybW3zcoeJeJ6gmyBGo2ooBI0+IsJS1ZH7YUfMEqwstPDR7bQgeDXc\\\\\\\/K+BgYua9BCrKAIsQsD2mqK\\\\\\\/h37DAPihGVhzgYnUp5AKy7PRzazWZn4EuykSaUkZbr0zdFMilCfpdiqXnxXhJ0EoAKLwPt5704lcROGfGG+nitadBxYGyMQqBbSIombWPgG5KTruK9Hk7pKcrkemm5ug0wz5V2HpRAawAla82mudo4ttobb5qHfXX+dlQN6Q6TXB93bN8MGWnhT3pfTCL9GbOQxqUqPME2+Gsr1MpONcFpt\\\\\\\/KIeNMVzsA\\\\\\\/mJ7i+9NZ1R07ulS+UHUg+26e06r0KpmEcvc+4CwWdEhy5G8ScN+dbS6vGfaPYpxlUnR2bPkYU62X8PhY3pM6j3jW0URy7jSBDe7o0VpGQvcqpcPDxMxak5r0wpCoiiwuZq27uHlxEYVjVdTM8Q9a\\\\\\\/nqCzm0ztqlS+vMK7olYHv+e4dMClMxeqKOElPqC4+tJuJwbUJ8aX7nvNqTirXGBoS6huaWMA\\\\\\\/M8ulMooILsUJDqtB1MEdALh8AhNmX642ViQRGWGqbkcJiqAbAeAfJr\\\\\\\/sRB9oywAB19SFDagRnjCKq1S\\\\\\\/XZgjvtaXjDDKCoHjnqRkthTmMdqIt66mC+zI6ZcSoCQPjqWB6Qu02fsm+kflzKICjjR8YDzejMi2VYMA1PWgHy3sSt3YYDjfO6wvvTXPylHObVOikbZKbggA5tFQm\\\\\\\/GToVeIdHoEShKWe\\\\\\\/Rtz+VN2lOH3ok+m7FwI0cs2uCFErPxOUHUqeslUUiOQ498Ms8IgSvcSuVrYhXevatol6w5Rncm6XEpY2tHcbDUAx7D623o5ihsC96dC15b7e0Rq6opBHZTCyXZWOXvHm5VMnthQb8W6+1P\\\\\\\/aL8TopHcvMMxJNklZWIgAIQQ6ylKNHntGcTpuAWE3kWYIYk19GVGS4SoCRwXjZizw6iIdZTGfP6yH2V64JqB6vJ\\\\\\\/SqSKl0+2sUD4jSCh\\\\\\\/gKHDhYCr7vH+7mzCdglUdYH\\\\\\\/3biaY3EBU9mjCm8\\\\\\\/SCRwgqD8Y8KYtTMuuzxGWoqlz+fgdwZ8UnAj6XrAVoh3+IDhx0XDEl6bWJvpV7hdOKNs5JPiITOIYLhlIdPlHPy\\\\\\\/OgktRVh26Yuo\\\\\\\/+Go630pFeP85ePlt+Ox7epRAhrgeMROGJWeSzIoFgXaDxl3w5lLsHYkHciq\\\\\\\/pdg4bxZVGLi5n5qR9Z9ArMcyQz+88sMOd3mfCmxMe6Yz+ka+o6L13msShDkM\\\\\\\/0novgyc5SBvIkWSC1asfEv9HOaqQYbzyP+4sGUiMy+1AePKNDGM4fydSy9sgvznPObYuqaa0\\\\\\\/fDt50z39xlYrkiKCqUWo1oU4EvLOGt\\\\\\\/4+ekQfE3ZpCDW7X4XqRJ6l8yDYHUC0Re+7FjaT1cfq7zjOqKmBkfYAyxmH5jN1Nyw4RtHD6oLEmkXsHZvXaBPeKI\\\\\\\/\\\\\\\/7aN6JTwlAByGiN65QaMrxFu8w23azdmGMuYwlYqrlAeltGRpoZoWQl+Yh5Yr0Hsl9LqEhe\\\\\\\/vGTlMBFjZL9xD3BdO7lgd1iewn+VOg4oLDOHod\\\\\\\/qsjkTrIAZw06RYr+DuZBqJ4LRjKAqjkl5XzDp1oCep82qtU14yBkULCqyLq1sUiyKf82nBrtPkKL48gpxjP51oF3rx2S2bJHwp0b5nvYcPlG7KHVMxelYggBzXQ\\\\\\\/Bc3ilVj9VdyZH1E7OyuCmErdNKN9iwiUhHsBRsp4zsI9t7PCZE3XO7yfhmIyH8NFt5sBQr2Dl4FUnRMxqtxspxwbp10ITB8caRW99JTJBot8Ze0n6ma5fNuG8riE72i0UibQezYnZTCv7NHhXeLOFtnjQ1TdScNNe4kKzKrw6xxuFdPXdzETlOiBHTPkSAba6mnWZXW\\\\\\\/oX38vNvZ74+Un0wrdfUytmEzuUGqbrvjMTdGp56NO6TuTyyEW48pyOi6brdRDCEVmVMln22s8edryhoh+SfjCYtzAeXez7CvLuXnx59wfzIekGkGU4iiLh3LpQFhhsI0Ff796MSVVjdkEsTqayDxHEVHnFPepM87YUq6wOF0nRQspgvsTTUx0Jstyb+PxiIDS440s7dimN9jSl3RceLcp12u629L74op\\\\\\\/jSP3TfH\\\\\\\/UYyuem77uQOF9sbKk1IhempwRaTe+fEfwl929hpwTYUeF+Y85fDVgWdP3RNtrjfFz1VrQ1+kBQaYzJbWdT9v7nzHazRHCQ1vDPdHI\\\\\\\/QKN21aM89wyUXH6Axt0XJhUo1ZFoOVRBgCDYEZE6aEtRPeL9muOREwEkDTuTnvirpka8e1Y\\\\\\\/f60vRPi93MDFv7PUkISmm07JkEI\\\\\\\/JtCO4kXexhkw0\\\\\\\/l4wIhHWniD19V0HYegYGMbEagNlq9A9H2xvjffRZ6aAt0TrweoWwplU+CDuZJTmL\\\\\\\/So033VUQmBe1ISVlbGq7do+m7fqCA52MD463+GCDFmQ2FYnFWTqCSpr039mePu8rj9ne\\\\\\\/fzXMwy4jBX4Qo66ceG7Nfg\\\\\\\/jRWijee+FPYUdmlABY++YNfVDN1QH8KN5nuV8OBGbamEUKTRrfjUJGUKSG5745AIDC\\\\\\\/LutYypPsoyij2tuVow5093iL2uS+OiSnXCDW6nRu\\\\\\\/Efd4vleyuRtSaoiWm+vUUYxrTecuE0FAE7ViRssX6c7vJHFj\\\\\\\/5ldsx7LDdGwNJUiA9XNWuKFjMToTuvChqUkb9u2abbT+vXArrSsXsvZJ6Fd5b06wVVcixK1+2dJTvYN\\\\\\\/LpwEpIKRNXDHD0Yqe7mW5pSCKD4TtBbVG31YMCHwPEvmtp6BIf+VqDkzoCNhxKofN0ycDup01k\\\\\\\/2RecnY4BYnSZrteMxUKwlHh6nQu1Qzb2CeeVje4SJBtMBqN7dQmQi7HDBstcw6\\\\\\\/IFLg\\\\\\\/MEnFSh5sxs5trcAFyd48toDCyTVRFI2eKGatUYIVZoH+GBNYetLfvUUO5LmCyLQZ+uAEmi+Jz\\\\\\\/As6mDiGdCm+w1RASSDTCpV\\\\\\\/PE3c4lT0olNuA9hyaDOFVJMkX51Zjk\\\\\\\/5Z2YtHvZZwL3BUvGyFfYahbmhLla2skYu\\\\\\\/JA4HdWYTH6YeEx2ol0nNfkOf1b5CDOu89S+DlU51JHpyJTdD02OwAvsL\\\\\\\/PxqYDdlQcp2kFbED0gtMMs2LAXppU6CRd7yfwQsNMe8HQpIe\\\\\\\/Bens2i2ZsVdYusnr+3Lc\\\\\\\/QyKYegK6B2PWQQ+8fOmjjZ+WLkBUQ4Hi2ke5iw6b3a5O9v0qTVgBRQVaJezRrYKj39xUm2OJVtgH0uOTxc+NqOCI527fyZ+kjnryfPspqzFx2iOBmix85HTc0bxZJwpif2kPC0DBseSFswtdJKs51aCP3jl0Hy5bwHbGxddMI1G9EuqC1dO\\\\\\\/ZfPSaFK5eS1dJZBAsilMWc7w3lkVNShQaeCC1XyIPW6LkNiF37GYD4CPTrTlYJ+Li709zvp6RGjSMR7XNCvoW+Jb3yZG9m1dmcIT5a863iNrA5+ogK6b6JJ2upLKvdLuVGnAxHTp05rGyYceKZb\\\\\\\/tWt5nkt9fF80IgoTcAR8dpwU18rN3x3nc64doUzpTxq7d21U+RVe6jCVKrdNMREkFTXLWpCRaxg4NudRiWXdsMSihOZzIvcwzGROVCzqvKC3B6Vk09xXN9t9aylqqs6hkn0hQfJDrFTYc34XZng6j94a+HeBTiAjB4X+jmFxtDrDtdTcibFM1JnnAO0UIsdtDnhCbn5LI8dm8GripioLiVJBjhhAAqyQxgCN9eagSqczTOg32KNIHweCqoPO08IZEiIgm5K32yeKwob8mQx8H0B2tsVchApYC7HgrIWHWwR519VG6uzQBT609BvDy5TN+xwjdeGfPLLZzjjboSR4zS7qTP3w35KjGleFuq2W8I4arwQW+tBoF+d4NtoYob7hCCicDRMCPGuBVG+yNK5rSG2OgzjEFGv54E3qZKfZS8IkNOGqCKurfmoQ+bF5QUBk5Y9EpS4dNBF7RL2B6EGhu5Gz0oxyTTanTbCpaYQ2+DiH2o2wDuQ0d+ygJoIW9JLnMTRtIEv1xRFSeKat\\\\\\\/+xH+n8EoxuzTKYktTJUZYV\\\\\\\/iEAX\\\\\\\/vXPDtmQE7DE+zMR\\\\\\\/MKXxW9fsTsZof4smQKziNzJSlnmMa\\\\\\\/DtApW50zqicLTPdVP6\\\\\\\/Ctl5r1Z6B2mtWd\\\\\\\/kLEj6VcHqwvsZxeLArv5VWnlk1uRfprAJUMjZ3HsUSC6Hc32imqZtHbJznj0483hucJejIZFMgFzr8D8MagMJyqT4gMzR+Fz\\\\\\\/FaQP3kwcpnk7NvQOR+wHM8WfL8RIiNBGigTVU5qVpxEFHpiIoIZz80B48mcxEVds2vF5jH1ESnEQGCwDCpO9VUzC\\\\\\\/k7q2QhAZaA4eydnJ\\\\\\\/WgxtsMI1G6z5nE0cRsJtX25inLnPVNbp6svovz8ICtIGEYyQ81iNEWEffU+YgqURUEyzn7YUP1wH9Mfz\\\\\\\/2yxRqN1wHUyAb+nkxbxjBev6DoU5GmkP7gVy4MEcgBXCi\\\\\\\/8fhfh0JF\\\\\\\/5KwrCZTlU+MkxzIs\\\\\\\/7Uql2HEs9ahfCcPAyk4M04gJQr7YOb9xdqFu8EvVHs\\\\\\\/\\\\\\\/YbFqxdAgLbfmP26OZE6Jben0SGDG67L4df+clDldl3m2eFCsCfvsLLRvr79OiN4cVMBo+c7Qnr3sOLgtYazEGYv5KU6tUseKyzLOfW2JYNHrJoOf3qFMYlCIYElI+wMvdLcsEdh2aRfxpU+hb9yae\\\\\\\/xqTmZNAhaqMxaMv9gm78j\\\\\\\/fx\\\\\\\/c2salf07YN4AVokhUjp7Tqe\\\\\\\/nA\\\\\\\/VExfRagyE\\\\\\\/Qo7P5kkwPlffxQyHiCZBrTr2jYIsJR6Ts+EHOUMaV1z6Oo9+TJEmcv\\\\\\\/zWW8Bm\\\\\\\/kllKLVKrEU+\\\\\\\/n8z3UuvF\\\\\\\/rwelt1cuDw6N9Aqn2hqxrKglW+n+AEnaUNHxyKWJ9KWOL04+SvTmpnTXiJXRqd+PI7ytDna5i1nEcxgSdaMFrTh5zY7HAqxW1kTu73aglBOm4KjmymoQom35jY4jzxvoJcHT2VVn7NwJPSeZSpbnem6OBECsAdTjOLGW5wDE56VLkf6d8w\\\\\\\/6zANVYoCf1yHhMOR79HWwVKz0GyUK9PbWJPvLF9+ObHESXPGZd5OlMZFCpm19Nkpa2Bz8V7iM1bRnbsq5McLnfTTz+MQg3Bmky01Cu2mDBoa68PJIDRIzJnpX6pVWj1hcP+oSKiGnLzP3Gwg2Q6GdmPdoyD+T7wUaWu2awjowrh3DphfjwZZPfEppQGFgu8eSx9k4UCg6wTADCeIwbfC3kbQZc\\\\\\\/qrkMZQtIwDYtcV1a7LIXek4n8K1jVWXVv5mwGrFKS3aCpwYzjEvp9R8f76CcuDrEDEiMtKO4kN3p5+cTyHVbpPo8\\\\\\\/Ff04lJVIesQL1AJmVzk2m0ymf2d3DQxiCEeJ2YUVP471zsgi8PQmFT2TJKreMaxJQ1ANhy7Ug5o5MFzr4GfsRRsMLMLevSaHMQF96uyKHf5Id3dm3kleDLBfxVKQSUNxbQgzfZDjBtJcWZgwy9dU3BHtpzAJxto9RvabM5WqlPZrT2GgXsTVm5lc51KfdhIgsK6UEzwvJwaYjd26cKlhqQnaX91SFZrrJXlRHiizWgjiSTouMIYZm1VteJlPwfI5RGY+6W23VOjJk7MLp9TgodnhtlWLwGBli4RiBnJEOAy3sW9bM9J4uObNMJkcqpkw6CpUNkIcesBzYNHhpiGgoSnAKwqZ5HCd4R76326MTjv9hgEvy392DmHfBYArD1rPOOV96F+2sf46W33PkYV\\\\\\\/0i7D\\\\\\\/9JgJNQ4nhKGmc2b5WWbJ1PgsNsRISCFV8le0Nrtn5oXnqC8NOXY8aEZAvWddiz3kRTeK3QpskyglaRV\\\\\\\/TCgEkMHlR0SqHXNI0ZEOTrWfDTa00cxTj0tjzNcWqfdxGI96\\\\\\\/gdpGFWToAT3W4VQ77RAE03CSMoDMjRUswSOG5TRtSGuTn0dMuDUTkQyO+jkMVJkC\\\\\\\/ybiFVTMZ37jSmXKLpcbK2XiALSzCOMSVD27mKqyJXoSYl+Bb8cO78w1Mb9Ynzxu5WZQJpqyoM0H52XR6UOZTG6Aw0F0OjJfQvo9cHzTGlYoLZxxKqo1+G3sRMs3niee+msf3yHZk3bMoPsg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XHm6F4C+qEJ98aCZnQc6\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228b60f15c184ae0806ad37c6df58aa6784868192747ed2814f67a5af56e333977\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022g+VO\\\\\\\/q2ht7QGuuc8m7HtDx3Ds5sX6eRPzF3LEjNSBjToLwC3LRINtoQWWIKgX\\\\\\\/+XBjLfB0UMm00fvb0RIm5nzWdmWli5nDYxkwDncZeGwZqRHvFnposc\\\\\\\/XL1SR4pQdSgxSW9G1mCXiENev7m+nFOnHonlX2hc9Vi3NU4IZMABfV54wdVW6C0Dl5KBzLQjR9i8BJ7bzUzzU807j6XZ5IqCdFFlyxGJPmmSvygAf\\\\\\\/BPLdnb14sg\\\\\\\/1jWMpQDFW\\\\\\\/+YteMZ0t4624VvCKuCYvrHvV3lOMb0muxGF43JGBn7UttomAh90ub9iB5N3QI34LMl03MS1vbZ2w59KbCK7cx1kx+vi906OVK0WoQXr5pLSbzuYLEppP\\\\\\\/Fj1Am0XyVCSzXZ675bF3GG2b6nUp6RRDkNDNEXibUL2rrHkes4b6zXZ2KcPEpakv+QujLT1R6BIi4MsU0UzIsgjnGJFnzUEvRFm0SnimTHX6BnPawK9Owu8\\\\\\\/WRHJJJtvSe+4FVbaR\\\\\\\/2PMmcmTqIgEUAOD7478\\\\\\\/pU8zr59RqjS07QZq8HMRq7DZrC3ymF2L96z\\\\\\\/p7MfUK5kSKM1bI3js1Ey8yFht0Opy6LOc++O8FMXLuKtdKrX3ghjC+jbvrJJ0EBHs5+8vCCcdjmRd\\\\\\\/OxdIMVokVZWa4H0kuZcj1mlyZrYADbKHAqqG6BLni5ztj0i1vke\\\\\\\/VyXPUsiAcg1y7us4wlalmKOn+zFEG8tQgMhfkae9N4mP1G2DeP81qGbXmbRq1RaGnc6IFFFnpjRNqHkRp3fOAU6pOpI1lM2rCrjkXgzQRbUfL2bDvKHmw4gf0u4gJ8mC7sZcDMuceWsjs1LtZg+pE\\\\\\\/URA7c2orLVcam88PBcbz\\\\\\\/VXcnl3Qkvcknvje4iQTgQcQpcEQ08uaTArsooJu02JnOimtw4owdYfj4A6fB9v6AW+C1I7r0oHcSfwu6JFPgdUPZH95IO64jcVWyVkuSuMqfOwBENUGesJpJsLFZYZFdxPZt\\\\\\\/DsXAatDjiN7LT6wEBeycdJeqwfZ5Occ207aAnj3hIZzKH4G4yQqxAKqMyoD17cpMXoQIH63muZ8TFVnRxjX7Uy1nhR10PhJ6txkjsCDrJ2hUd7U8SLVmXCjl5vPN6wB9fOTM3IMGUTSyW2zFQSbdHUUoAbQHYH+QkGYt4CuU7TqTNdoTQQY3O8LUFhWQtoiEuY748rmrdP3RDR0FkWX+ZVlfxjnAW7LlBEqcX4t9JzWtiMrkCYYpqzcrZNiXtgjX2Eo37UnC2RI2Puxt5qdtn1\\\\\\\/t2C\\\\\\\/Vi3nO4oalR0bgSnYqyJRchBOvv2Of0DZHsXAC9sVmtq00yTjvYKE8dAEuqqLamp+Kik5kc2SBBShWTdteUKGPVykFBGq7RKfRdePksgZfA3K2smcLh+yPEY\\\\\\\/noma7vDY7TaVriyp1gAxqddVrUCCLpQ4C0imGeJODOLIdhG7LJfcPeY8yEiq\\\\\\\/4xgljmgz4QjI5+tUmuttIAv9eXMJ9RObvfGzjugrpj5FESEVBgpf35ZhkcubyZ2WPHDH\\\\\\\/ANBPahEIOEn9W4eQX9YlCtSSBnmR5b5TVQ9Pz4vuoZBTnlWxAFOZ\\\\\\\/oo3EN0oXg0FeHmlQ3e2uXfhuqIsjo5fZKSDtl2dWmnQg78ypA01FVjzQASgFlOHIq0y3nVCcZcNJWHDitBCQwotdh1YzaIvaul5CpNMcHAnJJVlLZO8yOGlXzGaIvNBh7SAlSOw3EZ4rjn2WI5Pi2WmOJ8cA1\\\\\\\/nEb3Eur8VEaG6PSkebtVFnE0WEZu8eeRBkXHftvFNtgvm9kDl94JWmOnj7RVt9B5F0StDYwdc7+8ukTIE03OOwb0HRmJrL6x4bXv\\\\\\\/gjYMPE596ODEhlGMTksEanD7gveOuQhsz82HOeIF0kK+4lpYGMihBSJh\\\\\\\/bZkl4gecRazMdPA5XRiWe4i0RwsI3ZSXpKHf5K0UthnIzYN9jm3lJXM7USbxoZ\\\\\\\/Bx6\\\\\\\/6K0A\\\\\\\/eMCduccvl6pP6CXPmwzJcVzJGzn99M9sM7A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ahCLTe8KK2Ua86yF\\\\\\\/h\\\\\\\/rew==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cc8b73e52379582f517553779003609579e23e07448432ddbbbef9d24ba76c81\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487479 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3839" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OUNQ3FvBlHM_8nLSo0pdzEhBP0BfQgrx0LgKHIr31D4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3839" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-500oWVot81A1lXQrJLLhfS-awL87y7teSSnF-hiQez8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3837" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/3849" title="rSalvador: An R Package for the Fluctuation Experiment" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487479" id="node487479" data-pisa="ggg;7/12/3839" data-pisa-master="ggg;g3.117.300271" data-apath="/ggg/7/12/3839.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xuhua Xia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3839-3848;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300271 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487479" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuhua Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuhua%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuhua%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xuhua Xia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xxia@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">xxia@uottawa.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/12/3839" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487479" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487479/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3839.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487479" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487479/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3839.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487479" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487479/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3839.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487479" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487479/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/12/3839.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487479" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487479/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Two major stumbling blocks exist in high-throughput sequencing (HTS) data analysis. The first is the sheer file size, typically in gigabytes when uncompressed, causing problems in storage, transmission, and analysis. However, these files do not need to be so large, and can be reduced without loss of information. Each HTS file, either in compressed .SRA or plain text .fastq format, contains numerous identical reads stored as separate entries. For example, among 44,603,541 forward reads in the SRR4011234.sra file (from a <em>Bacillus subtilis</em> transcriptomic study) deposited at NCBI’s SRA database, one read has 497,027 identical copies. Instead of storing them as separate entries, one can and should store them as a single entry with the SeqID_NumCopy format (which I dub as FASTA+ format). The second is the proper allocation of reads that map equally well to paralogous genes. I illustrate in detail a new method for such allocation. I have developed ARSDA software that implement these new approaches. A number of HTS files for model species are in the process of being processed and deposited at <a href="http://coevol.rdc.uottawa.ca">http://coevol.rdc.uottawa.ca</a> to demonstrate that this approach not only saves a huge amount of storage space and transmission bandwidth, but also dramatically reduces time in downstream data analysis. Instead of matching the 497,027 identical reads separately against the <em>B. subtilis</em> genome, one only needs to match it once. ARSDA includes functions to take advantage of HTS data in the new sequence format for downstream data analysis such as gene expression characterization. I contrasted gene expression results between ARSDA and Cufflinks so readers can better appreciate the strength of ARSDA. ARSDA is freely available for Windows, Linux. and Macintosh computers at <a href="http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx">http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transcriptomics" class="hw-term hw-article-keyword hw-article-keyword-transcriptomics" rel="nofollow">transcriptomics</a></li><li class="kwd"><a href="/keyword/novel-storage-solution" class="hw-term hw-article-keyword hw-article-keyword-novel-storage-solution" rel="nofollow">novel storage solution</a></li><li class="kwd"><a href="/keyword/quantifying-expression-paralogous-genes" class="hw-term hw-article-keyword hw-article-keyword-quantifying-expression-of-paralogous-genes" rel="nofollow">quantifying expression of paralogous genes</a></li><li class="kwd"><a href="/keyword/sequence-format" class="hw-term hw-article-keyword hw-article-keyword-sequence-format" rel="nofollow">sequence format</a></li><li class="kwd"><a href="/keyword/arsda" class="hw-term hw-article-keyword hw-article-keyword-arsda" rel="nofollow">ARSDA</a></li></ul><p id="p-3">High-throughput sequencing (HTS) is now used not only in characterizing differential gene expression, but also in many other fields, where it replaces the traditional microarray approach. Ribosomal profiling, traditionally done through microarray (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Arava <em>et al.</em> 2003</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">MacKay <em>et al.</em> 2004</a>), is now almost exclusively done with deep sequencing of ribosome-protected segments of messages (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ingolia <em>et al.</em> 2009</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2011</a>), although the results from the two approaches for ribosomal profiling are largely concordant (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Xia <em>et al.</em> 2011</a>). Similarly, EST-based (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rogers <em>et al.</em> 2012</a>) and microarray-based (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pleiss <em>et al.</em> 2007</a>) methods for detecting alternative splicing events and characterizing splicing efficiency is now replaced by HTS (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kawashima <em>et al.</em> 2014</a>), especially by lariat sequencing (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Awan <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stepankiw <em>et al.</em> 2015</a>). The availability of HTS data has dramatically accelerated the test of biological hypotheses. For example, a recent study on alternative splicing (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Vlasschaert <em>et al.</em> 2016</a>) showed that skipping of exon 7 (E<sub>7</sub>) in human and mouse <em>USP4</em> is associated with weak signals of splice sites flanking E<sub>7</sub>. The researchers predicted that, in some species where the splice site signals are strong, E<sub>7</sub> skipping would disappear. This prediction is readily tested and confirmed with existing HTS data, <em>i.e.</em>, E<sub>6</sub>-E<sub>8</sub> mRNA was found in species with weak splice signals flanking E<sub>7</sub>, and E<sub>6</sub>-E<sub>7</sub>-E<sub>8</sub> mRNA in species with strong splice signals flanking E<sub>7</sub> (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Vlasschaert <em>et al.</em> 2016</a>).</p><p id="p-4">In spite of the potential of HTS data in solving practical biological problems, severe underusage of HTS data has been reported (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">GB Editorial Team 2011</a>). One major stumbling block in using the HTS data are the large file size. Among the 6472 HTS studies on human available at NCBI/DDBJ/EBI by April 14, 2016, 196 studies each contribute &gt;1 Terabyte (TB) of nucleotide bases, with the top one contributing 86.4 TB. Few laboratories would be keen on downloading and analyzing this 86.4 TB of nucleotides, not to mention comparing this study to HTS data from other human HTS studies.</p><p id="p-5">The explosive growth of HTS data in recent years has caused serious problems in data storage, transmission, and analysis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Leinonen <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kodama <em>et al.</em> 2012</a>). Because of the high cost of maintaining such data, coupled with the fact that few researchers had been using such data, NCBI had planned the closure of the sequence read archive a few years ago (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">GB Editorial Team 2011</a>), but continued the support only after DDBJ and EBI decided to continue their effort of archiving the data. The incident highlights the prohibitive task of storing, transmitting, and analyzing HTS data, and motivated the joint effort of both industry and academics to search for data compression solutions (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Janin <em>et al.</em> 2014</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhu <em>et al.</em> 2015b</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Numanagic <em>et al.</em> 2016</a>).</p><p id="p-6">HTS data files do not need to be so huge. Take, for example, the characterized transcriptomic data for <em>Escherichia coli</em> K12 in the file SRR1536586.sra (where SRR1536586 is the SRA sequence file ID in NCBI/DDBJ/EBI). The file contains 6,503,557 sequence reads of 50 nt each, but 195,310 sequences are all identical (TGTTATCACGGGAGACACACGGCGGGTGCTAACGTCCGTCGTGAAGAGGG), all mapping exactly to sites 929–978 in <em>E. coli</em> 23S rRNA genes (the study did use the MICROBExpress Bacterial mRNA Enrichment Kit to remove the 16S and 23S rRNA, otherwise there would be many more). There are much more extreme cases. For example, one of the 12 HTS files from a transcriptomic study of <em>E. coli</em> (SRR922264.sra), contains a read with 1,606,515 identical copies among its 9,690,570 forward reads. There is no sequence information lost if all these 1,606,515 identical reads are stored by a single sequence with a sequence ID such as UniqueSeqX_1606515 (<em>i.e.</em>, SeqID_CopyNumber format which I dub FASTA+ format with file type .fasP). Such storage scheme not only results in dramatic saving in data storage and transmission, but also leads to dramatic reduction in computation time in downstream data analysis. At present, all software packages for HTS data analysis will take the 1,606,515 identical reads separately, and search them individually against the <em>E. coli</em> genome (or target gene sequences such as coding sequences). The SeqID_CopyNumber storage scheme reduces the 1,606,515 searches to a single one.</p><p id="p-7">A huge chunk of SRA data stored in NCBI/DDBJ/EBI consists of ribosome profiling data (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Ingolia <em>et al.</em> 2009</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">2011</a>), which is obtained by sequencing the mRNA segment (∼30 bases) protected by the ribosome after digesting all the unprotected mRNA segments. Mapping these ribosome-protected segments to the genome allows one to know specifically where the ribosomes are located along individual mRNAs. In general, such data are essential to understand translation initiation, elongation, and termination efficiencies. For example, a short poly(A) segment with about eight or nine consecutive A immediately upstream of the start codon in yeast (<em>Saccharomyces cerevisiae</em>) genes is significantly associated with ribosome density and occupancy (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Xia <em>et al.</em> 2011</a>), confirming the hypothesis that short poly(A) upstream of the start codon facilitates the recruitment of translation initiation factors, but long poly(A) would bind to poly(A)-binding protein and interfere with cap-dependent translation. Sequence redundancy is high in such ribosomal profiling data and the FASTA+ format can lead to dramatic saving in the disk space of data storage and time in data transmission.</p><p id="p-8">Aside from the file size problem, HST data analysis also suffers from the problem of how to allocate multiple-matched reads to paralogous genes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Trapnell <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rogozin <em>et al.</em> 2014</a>). The commonly used options of ignoring such multiple-matched reads or allocating them equally among matched paralogous genes are unsatisfactory. The software ARSDA I present here offers solutions to both the problem of file size and the problem of read allocation to paralogous genes.</p><div class="section other1" id="sec-1"><h2 class="">ARSDA</h2><p id="p-9">I developed software ARSDA (for Analyzing RNA-Seq Data, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) to alleviate the problem associated with storage, transmission and analysis of HTS data. ARSDA can take input .SRA files or .fastq files of many gigabytes, build an efficient dictionary of unique sequence reads in a FASTA/FASTQ file, keep track of their copy numbers, and output them to a FASTA+ file in the SeqID_CopyNumber format (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Both fixed-length and variable-length sequences can be used as input. In addition, I have implemented functions in ARSDA to take advantage of the new sequence format to perform gene expression, with the main objective of demonstrating how much faster downstream data analysis can be done with data in FASTA+/FASTQ+ format. Furthermore, ARSDA includes a unique feature in assigning shared reads among paralogous genes that I will describe below. ARSDA also includes sequence visualization functions for global base-calling quality, per-read quality, and site-specific read quality (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, C and D</a>), but these functions are also available elsewhere, <em>e.g.</em>, FastQC (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andrews 2017</a>) and NGSQC (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dai <em>et al.</em> 2010</a>) and consequently will not be described further (but see the attached QuickStart.PDF). ARSDA includes nine programs in the BLAST and sratoolkit from NCBI that enhance part of ARSDA functions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="User interface in ARSDA. (A) The menu system, with database creation under the &#x201C;Database&#x201D; menu, gene expression characterization under the &#x201C;Analysis&#x201D; menu, etc. (B) Converting a FASTQ/FASTA file to a FASTQ+/FASTA+ file. (C) Site-specific read quality visualization. (D) Global read quality visualization." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1753868244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;User interface in ARSDA. (A) The menu system, with database creation under the &#x201C;Database&#x201D; menu, gene expression characterization under the &#x201C;Analysis&#x201D; menu, etc. (B) Converting a FASTQ/FASTA file to a FASTQ+/FASTA+ file. (C) Site-specific read quality visualization. (D) Global read quality visualization.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3839/F1.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/12/3839/F1.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487588" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">User interface in ARSDA. (A) The menu system, with database creation under the “Database” menu, gene expression characterization under the “Analysis” menu, etc. (B) Converting a FASTQ/FASTA file to a FASTQ+/FASTA+ file. (C) Site-specific read quality visualization. (D) Global read quality visualization.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Converting HTS data to FASTA+/FASTQ+</h3><p id="p-11">The output from processing the SRR1536586.sra file (with part of the read matching output in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) highlights two points. First, many sequences in the file are identical. Second, although the transcriptomic data characterized in SRR1536586 have undergone rRNA depletion by using Ambion’s MICROBExpress Bacterial mRNA Enrichment Kit (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pobre and Arraiano 2015</a>), there are still numerous reads in the transcriptomic data that are from rRNA genes. This suggests that storing mRNA reads separately from rRNA reads can dramatically reduce file size because most researchers are not interested in the abundance of rRNAs.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487507/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487507/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Part of read-matching output from ARSDA, with 195,310 identical reads matching a segment of large subunit (LSU) rRNA, 86,308 identical reads matching another segment of LSU rRNA, and so on</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">While the conversion of FASTA/FASTQ files to FASTA+ files may take a few minutes, it needs to be done only once for data storage, and the resulting saving in storage space, internet traffic, and computation time in downstream data analysis is tremendous. The file size is 1.49 GB for the original FASTQ file derived from SRR1536586.sra, but is only 66 MB for the new FASTA+ file, both being plain text files.</p><p id="p-14">I have applied ARSDA to reduce the file size of transcriptomic data from yeast (<em>S. cerevisiae</em>), nematode (<em>Caenorhabditis elegans</em>), fruit fly (<em>Drosophila melanogaster</em>), zebrafish (<em>Danio rerio</em>), and mouse (<em>Mus musculus</em>), and deposited the resulting reduced data at coevol.rdc.uottawa.ca in the form of BLAST databases. BLAST reduces sequences further by representing tetranucleotides AAAA, AAAC, ..., TTTT by 0, 1, ..., 255 so that each tetranucleotide takes only 1 byte in storage. The sequence ID in these BLAST databases are in the form of SeqID_CopyNumber. These files reduce the computation time for quantifying gene expression from many hours to only a few minutes (&gt;2 min for my Windows 10 PC with an i7-4770 CPU at 3.4 GHz and 16 GB of RAM). This eliminates one of the key bottlenecks in HTS data analysis (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2016</a>), and would make it feasible for any laboratory to gain the power of HTS data analysis. I attach the gene expression characterized by ARSDA for the 4321 <em>E. coli</em> K12 coding sequences as Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.txt">File S1</a>. A part of it is reproduced in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487553/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487553/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Partial output of gene expression, with the gene locus_tag (together with start and end sites) as Gene ID</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">One may ask how quality scores are treated when reads with identical sequences are grouped into the form of SeqID_CopyNumber format. Let me first highlight two observations. First, different reads with the same sequence have similar quality scores. For example, sequence “TGTTATCACGGGAGACACACGGCGGGTGCTAACGTCCGTCGTGAAGAGGG” occur many times in file SRR1526586.sra. I took the first six reads with this 50-nt sequence, and computed Pearson correlation among the associated quality scores (each read is associated with a vector of 50 quality scores). The correlation coefficients are all high and positive (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The same for sequences that occur just twice. Second, quality score itself is a statistical estimate. For these reasons, when reads with the same sequence are combined into the SeqID_CopyNumber format in Fastq+ file, the quality scores for this combined sequence are simply average quality scores. For a sequence of length <em>L</em> that occurred twice in the transcriptomic data, the sequence ID will be SeqID_2, and quality scores will simply be (Q<sub>1i</sub> + Q<sub>2i</sub>)/2, where <em>i</em> = 1, 2, …, <em>L</em>, and Q1 and Q2 are quality scores from the two reads of the same sequence.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487486/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487486/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Correlation among quality scores from first six reads (Q1–Q6) of the same sequence of 50 nt (TGTTATCACGGGAGACACACGGCGGGTGCTAACGTCCGTCGTGAAGAGGG)</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">Size-reduction differs dramatically with read quality (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). For high-quality data, <em>e.g.</em>, SRR922267 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, where an overwhelming majority of bases are at the high-quality end), ARSDA can shrink file size to ∼0.05 of the original. However, for poor-quality data, <em>e.g.</em>, SRR5484239 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), ARSDA can shrink file size only to 0.64 of the original. The reason is that, with high-quality data, reads from the same segment of the transcript are identical, as one would expect, but, with low-quality data, reads from the same segment of the transcript have “mutated” during the amplification and sequencing step, and are often no longer identical. For SRR922267, the most redundant sequence has 2,341,386 identical copies out of 14,872,404 forward reads. In contrast, the most redundant sequence in SRR5484239 has only 1540 identical copies out of 10,702,871 reads. This implies that the paired-end reads, especially long ones, will likely have low size-reduction efficiency because reverse read quality is typically much worse than forward read quality. Size-reduction with the ARSDA approach works best with high-quality reads. Base-calling quality typically decreases rapidly with read length (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Trimming off the low-quality 3′ end of the reads typically leads to dramatically increased size-reduction efficiency.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contrasting read quality between two transcriptomic data files (SRR5484239.sra from M. musculus and SRR922267.sra from E. coli. It does not imply that E. coli data are always better than mouse data as there are also poor-quality E. coli data and high-quality mouse data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1753868244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Contrasting read quality between two transcriptomic data files (SRR5484239.sra from &lt;em&gt;M. musculus&lt;/em&gt; and SRR922267.sra from &lt;em&gt;E. coli&lt;/em&gt;. It does not imply that &lt;em&gt;E. coli&lt;/em&gt; data are always better than mouse data as there are also poor-quality &lt;em&gt;E. coli&lt;/em&gt; data and high-quality mouse data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3839/F2.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/12/3839/F2.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487489" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Contrasting read quality between two transcriptomic data files (SRR5484239.sra from <em>M. musculus</em> and SRR922267.sra from <em>E. coli</em>. It does not imply that <em>E. coli</em> data are always better than mouse data as there are also poor-quality <em>E. coli</em> data and high-quality mouse data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">One of the frequently used sequence-compression scheme is to use a reference genome so that each read can be represented by a starting and an ending number on the genome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benoit <em>et al.</em> 2015</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kingsford and Patro 2015</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhu <em>et al.</em> 2015a</a>). This approach has two problems. First, many reads do not map exactly to the genomic sequence because of either somatic mutations or sequencing errors, so representing a read by the starting and ending numbers leads to loss of information. Second, RNA-editing and processing can be so extensive that it becomes impossible to map a transcriptomic read to the genome (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abraham <em>et al.</em> 1988</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lamond 1988</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alatortsev <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Simpson <em>et al.</em> 2016</a>). Furthermore, there are still many scientifically interesting species that do not have a good genomic sequence available. One could add additional annotation and indexing for sequence variants resulting from RNA-editing and “mutants” resulting from amplification and sequencing to avoid information loss, but such additional steps reduces the efficiency of compression as well as increases an extra layer of complexity for downstream data analysis.</p><p id="p-20">Software tools for compressing HTS files are often benchmarked against general-purpose GZIP tools (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Numanagic <em>et al.</em> 2016</a>). Among nonreference compression tools for FASTQ files, LFQC (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nicolae <em>et al.</em> 2015</a>) was benchmarked to be the most efficient (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Numanagic <em>et al.</em> 2016</a>), partly because LFQC uses several compression programs separately for sequence IDs, sequences, and quality scores. I compared file size reduction from FASTQ+ format against compression results from GZIP and LFQC (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Because FASTQ+ files are simple text files that can be further compressed by GZIP and LFQC, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> also include compression output of GZIP+FASTQ+ and LFQC+FASTQ+. The results (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>) confirms the previous finding (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Numanagic <em>et al.</em> 2016</a>) that LFQC is much more efficient than GZIP. They also show FASTQ+ to offer a substantial further reduction of file sizes. For SRR1536586, file size reduction efficiency is comparable between LFQC and FASTQ+ (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). However, further compression with GZIP+FASTQ+ and LFQC+FASTQ+ both leads to much reduced file size than using GZIP or LFQC alone (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>), the same being true for the paired-end file SRR922270 (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). Furthermore, FASTQ+ has one additional advantage in that it dramatically reduces computation time in downstream data analysis. Take SRR1536586 for example, FASTQ+ would reduce computation time for read-matching (which is the most time-consuming part of any transcriptomic data analysis) to a fraction of roughly 0.075 (≈119,596,093/1,604,183,348).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487543/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487543/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Comparison of different compression methods from the original single-read file (SRR1536586) and paired-end read file (SRR922270) in FASTQ format</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Assigning sequence reads to paralogous genes</h3><p id="p-23">One of the most fundamental objectives of RNA-Seq analysis is to generate an index of gene expression (FPKM: matched fragment/reads per kilobases of transcript per million mapped reads) that can be directly compared among different genes and among different experiments with different total number of matched reads (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mortazavi <em>et al.</em> 2008</a>). The main difficulty in quantifying gene expression arises with sequence reads matching multiple paralogous genes (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Trapnell <em>et al.</em> 2013</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Rogozin <em>et al.</em> 2014</a>). This problem, which has plagued microarray data analysis, is now plaguing RNA-Seq analysis. Most publications of commonly used RNA-Seq analysis methods (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Langmead <em>et al.</em> 2009</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Trapnell <em>et al.</em> 2009</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Roberts <em>et al.</em> 2011</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Langmead and Salzberg 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dobin <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Deng <em>et al.</em> 2014</a>) often avoided mentioning read allocation to paralogous genes. The software tools associated with these publications share two simple options for handling matches to paralogous genes. The first is to use only uniquely matched reads, <em>i.e.</em>, reads that match to multiple genes are simply ignored. The second is to assign such reads equally among matched genes. These options are obviously unsatisfactory. Here, I describe a new approach which should substantially improve the accuracy of HTS data analysis such as gene expression characterization.</p></div><div id="sec-4" class="subsection"><h3>Allocating sequence reads to paralogous genes in a two-member gene family</h3><p id="p-24">We need a few definitions to explain the allocation. Let <em>L</em><sub>1</sub> and <em>L</em><sub>2</sub> be the sequence length of the two paralogous genes. Let <em>N<sub>U</sub></em><sub>.1</sub> and <em>N<sub>U</sub></em><sub>.2</sub> be the number of reads that can be uniquely assigned to paralogous gene 1 or 2, respectively (<em>i.e.</em>, the reads that matches one gene better than the other). Now for those reads that match the two genes equally well, the proportion of the reads contributed by paralogous gene 1 may be simply estimated as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Now, for any read that matches the two paralogous genes equally well, we will assign <em>P</em><sub>1</sub> to paralogous gene 1, and (1−<em>P</em><sub>1</sub>) to paralogous gene 2. In the extreme case when paralogous genes are all identical, then <em>N<sub>U</sub></em><sub>.1</sub> = <em>N<sub>U</sub></em><sub>.2</sub> = 0, and we will assign 1/2 of these equally matched read to genes 1 and 2. We should modify Equation (1) to make it more generally applicable as follows<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where 0.01 in the numerator and 0.02 in the denominator are pseudocounts. The treatment in Equation (2) implies that, when <em>N<sub>U</sub></em><sub>.1</sub> = <em>N<sub>U</sub></em><sub>.2</sub> = 0(<em>e.g.</em>, when two paralogous genes are perfectly identical), then a read matching equally well to these paralogous genes will be equally divided among the two paralogues.</p><p id="p-25">One problem with this treatment is its assumption of <em>L</em><sub>1</sub> = <em>L</em><sub>2</sub>. If paralogous gene 1 is much longer than the other, then <em>N<sub>U</sub></em><sub>.1</sub> is expected to be larger than <em>N<sub>U</sub></em><sub>.2</sub>, everything else being equal. One may standardize <em>N<sub>U</sub></em><sub>.1</sub> and <em>N<sub>U</sub></em><sub>.2</sub> to number of unique matches per 1000 nt, designated by <em>SN<sub>U.i</sub></em> = 1000<em>N<sub>U.i</sub></em>/<em>L</em><sub>i</sub> (where <em>i</em> = 1 or 2) and replace <em>N<sub>U.i</sub></em> in Equation (2) by <em>SN<sub>U.i</sub></em> as follows (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Mortazavi <em>et al.</em> 2008</a>):</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span></div><div id="sec-5" class="subsection"><h3>Allocating sequence reads in gene family with more than two members</h3><p id="p-26">One might, mistakenly, think that it is quite simple to extend Equation (3) for a gene family of two members to a gene family with <em>F</em> members by writing<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>This does not work. For example, if we have three paralogous genes designated A, B, and C, respectively. Suppose that the gene duplication that gave rise to B and C occurred very recently so that B and C are identical, but A and the ancestor of B and C have diverged for a long time. In this case, <em>N<sub>U.B</sub></em> = <em>N<sub>U.C</sub></em> = 0 because a read matching B will always match C equally well, but <em>N<sub>U.A</sub></em> may be &gt;0. This will result in unfair allocation of many transcripts from B and C to A according to Equation (4). I outline the approach below for dealing with gene families with more than two members.</p><p id="p-27">With three or more paralogous genes, one may benefit from a phylogenetic tree for proper allocation of sequence reads. I illustrate the simplest case with a gene family with three paralogous genes A, B, and C, idealized into three segments in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. The three genes shared one identical middle segment with 23 matched reads (that necessarily match equally well to all three paralogues). Genes B and C share an identical first segment to which 20 reads matched. Gene A has its first segment different from that of B and C, and got four matched reads. The three genes also have a diverged third segment where A matched three reads, B matched six and C matched 12. Our task is then to allocate the 23 reads shared by all three, and 20 reads shared by B and C to the three paralogues.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allocation of shared reads in a gene family with three paralogous genes A, B, and C with three idealized segments with a conserved identical middle segment, strongly homologous first segment that is identical in B and C, and a diverged third segment. Reads and the gene segment they match to are of the same color." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1753868244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Allocation of shared reads in a gene family with three paralogous genes A, B, and C with three idealized segments with a conserved identical middle segment, strongly homologous first segment that is identical in B and C, and a diverged third segment. Reads and the gene segment they match to are of the same color.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3839/F3.medium.gif" width="348" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/12/3839/F3.medium.gif" width="348" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487518" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Allocation of shared reads in a gene family with three paralogous genes A, B, and C with three idealized segments with a conserved identical middle segment, strongly homologous first segment that is identical in B and C, and a diverged third segment. Reads and the gene segment they match to are of the same color.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">One could apply maximum likelihood or least-squares method for the estimation, but ARSDA uses a simple counting approach by applying the following<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Thus, we allocate the 23 reads (that matched three genes equally) to paralogous genes A, B, and C according to <em>P<sub>A</sub></em>, <em>P<sub>B</sub></em>, and <em>P<sub>C</sub></em>, respectively. For the 20 reads that matched B and C equally well, we allocate 20*6/(6 + 12) to B and 20*12/(6 + 12) to C. This gives the estimated number of matches to each gene as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>These numbers are then normalized to give FPKM (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Mortazavi <em>et al.</em> 2008</a>). The current version of ARSDA assume that gene families with more than two members to have roughly the same sequence lengths. This is generally fine with prokaryotes but may become problematic with eukaryotes.</p><p id="p-30">In practice, one can obtain the same results without actually undertaking the extremely slow process of building trees for paralogous genes. One first goes through reads shared by two paralogous genes (<em>e.g.</em>, the 20 reads shared by genes B and C in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) and allocate the reads according to <em>P<sub>B</sub></em> = 6/(6 + 12) = 1/3 and <em>P<sub>C</sub></em> = 12/(6 + 12) = 2/3. Now genes B and C will have 12.66667 (= 6 + 20*<em>P<sub>B</sub></em>) and 25.33333 (= 12 + 20*<em>P<sub>C</sub></em>) assigned reads, <em>i.e.</em>, <em>N<sub>U.B</sub></em> = 12.66667 and <em>N<sub>U.C</sub></em> = 25.33333. Once we have done with reads shared by two paralogous genes, we go through reads shared by three paralogous genes, <em>e.g.</em>, the 23 reads shared by genes A, B, and C in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>. With <em>N<sub>U.A</sub></em> = 7, <em>N<sub>U.B</sub></em> = 12.66667, <em>N<sub>U.C</sub></em> = 25.333333, and <em>N</em> = <em>N<sub>U.A</sub></em> + <em>N<sub>U.B</sub></em> + <em>N<sub>U.C</sub></em> = 45, so we have<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>which are the same as shown in Equation (6). This progressive process continues until we have allocated reads shared by the largest number of paralogous genes. The gene expression output in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020040/-/DC1/FileS1.txt">File S1</a> was obtained in this way.</p><p id="p-31">There are alternative approaches for read allocation among paralogous genes. ARSDA also has an alternative allocation scheme based on BitScores and <em>e</em>-values. For example, when a read exhibits strong homology to <em>N</em> paralogous genes, but with different <em>e</em>-values or BitScores, I will not assign the read to the paralogous gene with the smallest <em>e</em>-value (or largest BitScore). Instead, all <em>N</em> paralogous genes will get a share of the read according to sequence similarity reflected in <em>e</em>-value and BitScore. The simplest scheme based on BitScore is to allocate such a read to paralogous gene <em>i</em> according to<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>which would give a paralogous gene with high BitScore a higher share. An alternative based on <em>e</em>-value is<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3839/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <em>E</em> is <em>e</em>-value and <em>K</em> is a scaling factor to ensure that ∑<em>p<sub>i</sub></em> = 1. Equation (10) allocates shared reads more to the paralogous gene with a small <em>e</em>-value than those with large <em>e</em>-value. In practice, Equation (9) is often close to equal allocation, whereas Equation (10) results in more biased allocation favoring the best-match paralogous genes.</p></div></div><div class="section other2" id="sec-6"><h2 class="">Contrast Between ARSDA and Cufflinks in Characterizing Gene Expression</h2><p id="p-32">The most frequently used software for gene expression is Cufflinks (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Trapnell <em>et al.</em> 2012</a>), which is why I am contrasting ARSDA against it. I will use the transcriptomic data for an <em>E. coli</em> wild type (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pobre and Arraiano 2015</a>), archived in NCBI’s SRA database as SRR1536586.sra. The Cufflinks-quantified gene expression for this file is in file GSM1465035_WT.txt.gz from NCBI Geo DataSets GSM1465035. Gene expression from ARSDA and Cufflinks in mostly concordant (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), but four points (labeled in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) stand out as outliers (although many more discordant points will be revealed by a log-log plot). Such dramatic differences demand an explanation. Take <em>rpmJ</em> for example. Either ARSDA severely overestimated, or Cufflinks severely underestimated, the gene expression (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). I originally expected the discrepancy to be due to different allocation of paralogous genes. The expectation is only partially true.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contrast in gene expression (RPKM) between ARSDA and Cufflinks output for the same transcriptomic data in file SRR1536586.sra for E. coli wild type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1753868244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Contrast in gene expression (RPKM) between ARSDA and Cufflinks output for the same transcriptomic data in file SRR1536586.sra for &lt;em&gt;E. coli&lt;/em&gt; wild type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3839/F4.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/12/3839/F4.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487576" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Contrast in gene expression (RPKM) between ARSDA and Cufflinks output for the same transcriptomic data in file SRR1536586.sra for <em>E. coli</em> wild type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">There are 6426 reads can be mapped unambiguously to <em>rpmJ</em> (which is in fact a single-copy ribosomal protein gene). Although there are <em>rpmA</em>, <em>rpmB</em>, …, <em>rpmJ</em> genes in <em>E. coli</em>, they are not paralogous. One particular read “AGTGCCGAGCCGAAGCATAAACAGCGCCAAGGCTGATTTTTTCGCATATT” occurs 2684 times in SRR1536586.sra. It matches perfectly to the 36 nt at the 3′ end of <em>rpmJ</em> and 14 nt immediately downstream. However, Cufflinks output reported a count of only 2114 reads for <em>rmpJ</em> instead of 6426 (and consequently the much reduced RPKM in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). I originally suspected that <em>rpmJ</em> may be in an operon with an immediate downstream gene so that some read overlapping <em>rpmJ</em> and the downstream gene would be divided between the two. However, the downstream gene, which is <em>rpsM</em>, is 146 nt away. It is difficult to reconcile 6426 nonambiguous read matches to Cufflinks’ 2114. Similarly, <em>rpmD</em> and <em>rplV</em> (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>) has 14,468 and 22,747 unambiguous read matches, respectively, but the corresponding counts in Cufflinks output are only 8108 and 11,801, respectively. Note that <em>rpmD</em> and <em>rplV</em> are also single-copy genes with no ambiguous read matches. <em>E. coli</em> genes <em>rpmA</em>–<em>rpmJ</em> are not paralogous, neither are <em>rplA</em>–<em>rplY</em>.</p><p id="p-35">The last outlying gene (<em>cpsA</em> in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>) does involve a paralogous gene family (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). <em>cspA</em> has 19,776 unambiguous read matches, but Cufflinks output has only 10,957, which resulted in a much lower RPKM than that from ARSDA (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>). Also puzzling are the counts involving <em>cspF</em> and <em>cspH</em>. There are 264 unambiguous read matches to <em>cspF</em> and 58 to <em>cspH</em>. There are also 55 reads that match well to both <em>cspF</em> and <em>cspH</em>, with 27 of them matching <em>cspF</em> better than <em>cspH</em>, and 28 matching <em>cspH</em> better than <em>cspF</em>. So we may assign (264 + 27) reads to <em>cspF</em> and (58 + 28) reads to <em>cspH</em>, with relative proportions of 0.7719 and 0.2281 for <em>cspF</em> and <em>cspH</em>, respectively. Twelve reads match <em>cspF</em> and <em>cspH</em> equally well (the same <em>e</em>-value and the same BitScore), so we assign them proportionally to the two genes, <em>i.e.</em>, 12*0.7719 to <em>cspF</em> and 12*0.2281 to <em>cspH</em>. The final counts for <em>cspF</em> and <em>cspH</em> are 300.2626 and 88.7374, respectively. However, Cufflinks output shows counts of 2 and 63 for <em>cspF</em> and <em>cspH</em>, respectively. The discrepancy is particularly striking given that gene expression from ARSDA and Cufflinks are mostly concordant (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>). The alternative allocation to paralogous genes as specified in Equations (9) and (10) does not help reconcile the discrepancy. I hope that these numbers will prompt authors of Cufflinks to be more explicit about how they treats counts.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationship among paralogous genes cspA to cspI in E. coli, based on coding sequences, with bootstrap values next to internal nodes. Sequences were aligned by MAFFT (Katoh and Toh 2008) with accurate L_INS-i option and a maximum of 16 iterations. Coding sequences were first translated in amino acid sequences, which are aligned with BLOSUM62 matrix. Nucleotide sequences were then aligned against aligned amino acid sequences. Phylogenetic analysis was done with PhyML (Guindon et al. 2010). All these analyses were automated in DAMBE (Xia 2013, 2017)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1753868244" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic relationship among paralogous genes &lt;em&gt;cspA&lt;/em&gt; to &lt;em&gt;cspI&lt;/em&gt; in &lt;em&gt;E. coli&lt;/em&gt;, based on coding sequences, with bootstrap values next to internal nodes. Sequences were aligned by MAFFT (Katoh and Toh 2008) with accurate L_INS-i option and a maximum of 16 iterations. Coding sequences were first translated in amino acid sequences, which are aligned with BLOSUM62 matrix. Nucleotide sequences were then aligned against aligned amino acid sequences. Phylogenetic analysis was done with PhyML (Guindon &lt;em&gt;et al.&lt;/em&gt; 2010). All these analyses were automated in DAMBE (Xia 2013, 2017).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3839/F5.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/12/3839/F5.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3839/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487492" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Phylogenetic relationship among paralogous genes <em>cspA</em> to <em>cspI</em> in <em>E. coli</em>, based on coding sequences, with bootstrap values next to internal nodes. Sequences were aligned by MAFFT (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Katoh and Toh 2008</a>) with accurate L_INS-i option and a maximum of 16 iterations. Coding sequences were first translated in amino acid sequences, which are aligned with BLOSUM62 matrix. Nucleotide sequences were then aligned against aligned amino acid sequences. Phylogenetic analysis was done with PhyML (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Guindon <em>et al.</em> 2010</a>). All these analyses were automated in DAMBE (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Xia 2013</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2017</a>).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-7" class="subsection"><h3>Software and data availability</h3><p id="p-37">ARSDA is freely available at <a href="http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx">http://dambe.bio.uottawa.ca/ARSDA/ARSDA.aspx</a>, together with a QuickStart.PDF file showing HTS file conversion from FASTA/FASTQ file to FASTA+ format, three types of HTS data quality visualization tools, and downstream characterization of gene expression. It is a Windows program, but can run on any computer with .NET framework installed (<em>e.g.</em>, Macintosh and Linux with MONO installed and activated). The BLAST databases derived from HTS reads for several model species, in which sequence IDs are in the format of SeqID_CopyNumber, are deposited at coevol.rdc.uottawa.ca. One can use these BLAST databases with ARSDA to characterize gene expression or other analysis. The QuickStart.PDF file available at the same site details the use of ARSDA, either alone or in conjunction with the free DAMBE software (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Xia 2013</a>, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">2017</a>). The source code is available as a zipped supplemental file ARSDA.Src.zip in <a href="https://github.com/xuhuaxia/ARSDA">https://github.com/xuhuaxia/ARSDA</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">Nabil Benabbou of Center for Advanced Computing of Queen’s University helped me with testing compression software tools on the center’s servers. ARSDA was demonstrated in two workshops, one organized by J. Lu of Peking University, and another by E. Pranckeviciene of University of Vilnius. I thank participants, as well as J. Silke, J. Wang, Y. Wei, and C. Vlasschaert, for their feedback. Two anonymous reviewers offered outstanding feedback leading to substantial improvement of the manuscript. This study is funded by the Discovery Grant from Natural Science and Engineering Research Council (RGPIN/261252) of Canada. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-39">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300271/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300271/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-40"><em>Communicating editor: C. Myers</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-13"><span class="received-label">Received </span>September 13, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-24"><span class="accepted-label">Accepted </span>October 24, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Xia</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.1" data-doi="10.1016/0092-8674(88)90049-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feagin</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Characterization of cytochrome <em>c</em> oxidase III transcripts that are edited only in the 3′ region.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAbraham%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D55%26rft.issue%253D2%26rft.spage%253D267%26rft.epage%253D272%26rft.atitle%253DCharacterization%2Bof%2Bcytochrome%2Bc%2Boxidase%2BIII%2Btranscripts%2Bthat%2Bare%2Bedited%2Bonly%2Bin%2Bthe%2B3%2527%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252888%252990049-9%26rft_id%253Dinfo%253Apmid%252F2844415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(88)90049-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2844415&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q697200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.2" data-doi="10.1128/MCB.01886-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alatortsev</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz-Reyes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhelonkina</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sollner-Webb</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Trypanosoma brucei</em> RNA editing: coupled cycles of U deletion reveal processive activity of the editing complex.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2437</span>–<span class="cit-lpage">2445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01886-07%26rft_id%253Dinfo%253Apmid%252F18227152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI4LzcvMjQzNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzgzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-7.12.3839.3"><div class="cit-metadata unstructured">Andrews, S., 2017 FastQC, Babraham Bioinformatics. Available at: <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc">https://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.4" data-doi="10.1073/pnas.0635171100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arava</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide analysis of mRNA translation profiles in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">3889</span>–<span class="cit-lpage">3894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0635171100%26rft_id%253Dinfo%253Apmid%252F12660367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzcvMzg4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzgzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.5" data-doi="10.1073/pnas.1218353110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Awan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pleiss</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lariat sequencing in a unicellular yeast identifies regulated alternative splicing of exons that are evolutionarily conserved with humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">12762</span>–<span class="cit-lpage">12767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1218353110%26rft_id%253Dinfo%253Apmid%252F23861491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzMxLzEyNzYyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.6" data-doi="10.1186/s12859-015-0709-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benoit</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemaitre</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavenier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drezen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dayris</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reference-free compression of high throughput sequencing data with a probabilistic de Bruijn graph.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0709-7%26rft_id%253Dinfo%253Apmid%252F26370285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0709-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26370285&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maher</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Contreras-Galindo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">NGSQC: cross-platform quality analysis pipeline for deep sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>(<span class="cit-supplement">Suppl. 4</span>): <span class="cit-fpage">S7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D11%26rft.spage%253DS7%26rft_id%253Dinfo%253Apmid%252F20158878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20158878&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.8" data-doi="10.1126/science.1245316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramskold</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinius</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandberg</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Single-cell RNA-seq reveals dynamic, random monoallelic gene expression in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDeng%26rft.auinit1%253DQ.%26rft.volume%253D343%26rft.issue%253D6167%26rft.spage%253D193%26rft.epage%253D196%26rft.atitle%253DSingle-Cell%2BRNA-Seq%2BReveals%2BDynamic%252C%2BRandom%2BMonoallelic%2BGene%2BExpression%2Bin%2BMammalian%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1245316%26rft_id%253Dinfo%253Apmid%252F24408435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDMvNjE2Ny8xOTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM4MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.9" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">GB Editorial Team</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Closure of the NCBI SRA and implications for the long-term future of genomics data storage.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D402%26rft.epage%253D402%26rft.atitle%253DClosure%2Bof%2Bthe%2BNCBI%2BSRA%2Band%2Bimplications%2Bfor%2Bthe%2Blong-term%2Bfuture%2Bof%2Bgenomics%2Bdata%2Bstorage.%26rft_id%253Dinfo%253Apmid%252F21418618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21418618&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.11" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefort</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hordijk</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.12" data-doi="10.1126/science.1168978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM4MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.13" data-doi="10.1016/j.cell.2011.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lareau</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DIngolia%26rft.auinit1%253DN.%2BT.%26rft.volume%253D147%26rft.issue%253D4%26rft.spage%253D789%26rft.epage%253D802%26rft.atitle%253DRibosome%2Bprofiling%2Bof%2Bmouse%2Bembryonic%2Bstem%2Bcells%2Breveals%2Bthe%2Bcomplexity%2Band%2Bdynamics%2Bof%2Bmammalian%2Bproteomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.10.002%26rft_id%253Dinfo%253Apmid%252F22056041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22056041&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296902300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.14" data-doi="10.1093/bioinformatics/btu387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Trieglaff</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">BEETL-fastq: a searchable compressed archive for DNA reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2796</span>–<span class="cit-lpage">2801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu387%26rft_id%253Dinfo%253Apmid%252F24950811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24950811&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343082900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.15" data-doi="10.1093/bib/bbn013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Recent developments in the MAFFT multiple sequence alignment program.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbn013%26rft_id%253Dinfo%253Apmid%252F18372315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbn013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18372315&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256756400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.16" data-doi="10.1371/journal.pgen.1004249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douglass</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabunilas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellegrini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanfreau</span>  <span class="cit-name-given-names">G. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Widespread use of non-productive alternative splice sites in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004249%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004249%26rft_id%253Dinfo%253Apmid%252F24722551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722551&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.17" data-doi="10.1093/bioinformatics/btv071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patro</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reference-based compression of short-read sequences using path encoding.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1920</span>–<span class="cit-lpage">1928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv071%26rft_id%253Dinfo%253Apmid%252F25649622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25649622&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.18" data-doi="10.1093/nar/gkr854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The sequence read archive: explosive growth of sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D54</span>–<span class="cit-lpage">D56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr854%26rft_id%253Dinfo%253Apmid%252F22009675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr854&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22009675&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamond</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">RNA editing and the mysterious undercover genes of trypanosomatid mitochondria.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DLamond%26rft.auinit1%253DA.%2BI.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D283%26rft.epage%253D284%26rft.atitle%253DRNA%2Bediting%2Band%2Bthe%2Bmysterious%2Bundercover%2Bgenes%2Bof%2Btrypanosomatid%2Bmitochondria.%26rft_id%253Dinfo%253Apmid%252F3154277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3154277&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.20" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.21" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.22" data-doi="10.1186/gb-2010-11-8-r83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leek</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cloud-scale RNA-sequencing differential expression analysis with Myrna.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR83%26rft.epage%253DR83%26rft.atitle%253DCloud-scale%2BRNA-sequencing%2Bdifferential%2Bexpression%2Banalysis%2Bwith%2BMyrna.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r83%26rft_id%253Dinfo%253Apmid%252F20701754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20701754&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.23" data-doi="10.1093/nar/gkq1019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugawara</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The sequence read archive.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D19</span>–<span class="cit-lpage">D21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1019%26rft_id%253Dinfo%253Apmid%252F21062823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21062823&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.24" data-doi="10.1073/pnas.0901754106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hui</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atanasov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Structure of the core editing complex (L-complex) involved in uridine insertion/deletion RNA editing in trypanosomatid mitochondria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">12306</span>–<span class="cit-lpage">12310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0901754106%26rft_id%253Dinfo%253Apmid%252F19590014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzMwLzEyMzA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.25" data-doi="10.1093/bioinformatics/btw371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brudno</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">deBGA: read alignment with de Bruijn graph-based seed and extension.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">3224</span>–<span class="cit-lpage">3232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw371%26rft_id%253Dinfo%253Apmid%252F27378303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27378303&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.26" data-doi="10.1074/mcp.M300129-MCP200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flory</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turcott</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene expression analyzed by high-resolution state array analysis and quantitative proteomics: response of yeast to mating pheromone.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">478</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M300129-MCP200%26rft_id%253Dinfo%253Apmid%252F14766929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjc6IjMvNS80NzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM4MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.27" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.28" data-doi="10.1093/bioinformatics/btv384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicolae</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pathak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajasekaran</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">LFQC: a lossless compression algorithm for FASTQ files.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3276</span>–<span class="cit-lpage">3281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv384%26rft_id%253Dinfo%253Apmid%252F26093148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26093148&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Numanagic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonfield</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hach</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voges</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostermann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparison of high-throughput sequencing data compression tools.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D1005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.30" data-doi="10.1016/j.molcel.2007.07.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pleiss</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitworth</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergkessel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid, transcript-specific changes in splicing in response to environmental stress.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">928</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DPleiss%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D27%26rft.issue%253D6%26rft.spage%253D928%26rft.epage%253D937%26rft.atitle%253DRapid%252C%2Btranscript-specific%2Bchanges%2Bin%2Bsplicing%2Bin%2Bresponse%2Bto%2Benvironmental%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.07.018%26rft_id%253Dinfo%253Apmid%252F17889666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.07.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17889666&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249808500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.31" data-doi="10.1186/s12864-015-1237-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pobre</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arraiano</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Next generation sequencing analysis reveals that the ribonucleases RNase II, RNase R and PNPase affect bacterial motility and biofilm formation in <em>E. coli</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1237-6%26rft_id%253Dinfo%253Apmid%252F25757888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1237-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25757888&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.32" data-doi="10.1186/gb-2011-12-3-r22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaghey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Improving RNA-Seq expression estimates by correcting for fragment bias.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRoberts%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253DR22%26rft.epage%253DR22%26rft.atitle%253DImproving%2BRNA-Seq%2Bexpression%2Bestimates%2Bby%2Bcorrecting%2Bfor%2Bfragment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-3-r22%26rft_id%253Dinfo%253Apmid%252F21410973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-3-r22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21410973&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.33" data-doi="10.1093/bioinformatics/btt197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Updating RNA-Seq analyses after re-annotation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt197%26rft_id%253Dinfo%253Apmid%252F23677943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23677943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.34" data-doi="10.1186/gb-2012-13-1-r4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben-Hur</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SpliceGrapher: detecting patterns of alternative splicing from RNA-Seq data in the context of gene models and EST data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRogers%26rft.auinit1%253DM.%2BF.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253DR4%26rft.epage%253DR4%26rft.atitle%253DSpliceGrapher%253A%2Bdetecting%2Bpatterns%2Bof%2Balternative%2Bsplicing%2Bfrom%2BRNA-Seq%2Bdata%2Bin%2Bthe%2Bcontext%2Bof%2Bgene%2Bmodels%2Band%2BEST%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-1-r4%26rft_id%253Dinfo%253Apmid%252F22293517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-1-r4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22293517&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.35" data-doi="10.1093/gbe/evu051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Managadze</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalina</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gene family level comparative analysis of gene expression in mammals validates the ortholog conjecture.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu051%26rft_id%253Dinfo%253Apmid%252F24610837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24610837&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.36" data-doi="10.1261/rna.055160.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bard</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">L. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-throughput sequencing of partially edited trypanosome mRNAs reveals barriers to editing progression and evidence for alternative editing.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.055160.115%26rft_id%253Dinfo%253Apmid%252F26908922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjIyLzUvNjc3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.37" data-doi="10.1093/nar/gkv763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stepankiw</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogarty</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pleiss</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Widespread alternative and aberrant splicing revealed by lariat sequencing.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">8488</span>–<span class="cit-lpage">8501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv763%26rft_id%253Dinfo%253Apmid%252F26261211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26261211&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.38" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19289445&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265523300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.39" data-doi="10.1038/nprot.2012.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D562%26rft.epage%253D578%26rft.atitle%253DDifferential%2Bgene%2Band%2Btranscript%2Bexpression%2Banalysis%2Bof%2BRNA-seq%2Bexperiments%2Bwith%2BTopHat%2Band%2BCufflinks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2012.016%26rft_id%253Dinfo%253Apmid%252F22383036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2012.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22383036&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.40" data-doi="10.1038/nbt.2450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrickson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Differential analysis of gene regulation at transcript resolution with RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2450%26rft_id%253Dinfo%253Apmid%252F23222703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23222703&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vlasschaert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Selection preserves Ubiquitin Specific Protease 4 alternative exon skipping in therian mammals.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">20039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D20039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.42" data-doi="10.1093/molbev/mst064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DAMBE5: a comprehensive software package for data analysis in molecular biology and evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1720</span>–<span class="cit-lpage">1728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst064%26rft_id%253Dinfo%253Apmid%252F23564938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23564938&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321056200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">DAMBE6: new tools for microbial genomics, phylogenetics and molecular evolution.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D108%26rft.spage%253D431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.44" data-doi="10.1534/genetics.111.132068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Translation initiation: a regulatory role for poly(A) tracts in front of the AUG codon in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132068%26rft_id%253Dinfo%253Apmid%252F21840854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNDY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.45" data-doi="10.1093/bioinformatics/btu656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">CompMap: a reference-based compression program to speed up read mapping to related reference sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">426</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu656%26rft_id%253Dinfo%253Apmid%252F25282641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25282641&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3839.46" data-doi="10.1093/bib/bbt087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">High-throughput DNA sequence data compression.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbt087%26rft_id%253Dinfo%253Apmid%252F24300111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbt087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24300111&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3839.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/12/3839.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/11/3837" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/3849" title="rSalvador: An R Package for the Fluctuation Experiment" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/12"><span>Volume 7 Issue 12, December 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/12.cover-source.jpg?itok=-p16LxzL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3839" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-spobdcUjgZZJz4FlT0shL1Zxa5gHnWkpwznT_YyAlSY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/12/3839" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/12/3839" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/12/3839" class="active" data-icon-position="" data-hide-link-title="0">ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487479" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rzO0qBzGFaRUTVfsyH8DP8VQmYWokkjkTR4oVrU9h_s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/3839.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/12/3839" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-A5wntBbjQNEeCadMeNWxF37NEwwOhO9vBS8mXlpTVRg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/12/3839" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/3839.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487479" id="node487479--2" data-pisa="ggg;7/12/3839" data-pisa-master="ggg;g3.117.300271" data-apath="/ggg/7/12/3839.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xuhua Xia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3839-3848;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300271 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487479--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuhua Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuhua%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuhua%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xuhua Xia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xxia@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">xxia@uottawa.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/12/3839" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/12/3839&amp;title=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/12/3839&amp;title=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/12/3839&amp;title=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/12/3839&amp;text=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/12/3839&amp;title=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/12/3839&amp;t=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/12/3839&amp;title=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/12/3839&amp;title=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487479/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487479" id="node487479--3" data-pisa="ggg;7/12/3839" data-pisa-master="ggg;g3.117.300271" data-apath="/ggg/7/12/3839.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xuhua Xia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3839-3848;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300271 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487479--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xuhua Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuhua%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXuhua%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3092-7566" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xuhua Xia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xxia@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">xxia@uottawa.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3839&amp;rft.epage=3848&amp;rft.atitle=ARSDA%3A%20A%20New%20Approach%20for%20Storing%2C%20Transmitting%20and%20Analyzing%20Transcriptomic%20Data&amp;rft.volume=7&amp;rft.issue=12&amp;rft.date=2017-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xia%2C+Xuhua"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487479/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487479/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487479/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487479/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487479/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487479/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487479/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487479/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487479/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487479/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487479/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487479/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/12/3839" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ARSDA</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Contrast Between ARSDA and Cufflinks in Characterizing Gene Expression</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/12/3839.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/12/3839.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/12/3839.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/12/3839.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
