<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/12/3849" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/12/3849.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/12/3849.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/12/3849.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/12/3849/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/12/3849.atom" />
<meta name="HW.pisa" content="ggg;7/12/3849" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="rSalvador: An R Package for the Fluctuation Experiment" />
<meta name="DC.Identifier" content="10.1534/g3.117.300120" />
<meta name="DC.Date" content="2017-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Zheng. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The past few years have seen a surge of novel applications of the Luria-Delbrück fluctuation assay protocol in bacterial research. Appropriate analysis of fluctuation assay data often requires computational methods that are unavailable in the popular web tool FALCOR. This paper introduces an R package named rSalvador to bring improvements to the field. The paper focuses on rSalvador’s capabilities to alleviate three kinds of problems found in recent investigations: (i) resorting to partial plating without properly accounting for the effects of partial plating; (ii) conducting attendant fitness assays without incorporating mutants’ relative fitness in subsequent data analysis; and (iii) comparing mutation rates using methods that are in general inapplicable to fluctuation assay data. In addition, the paper touches on rSalvador’s capabilities to estimate sample size and the difficulties related to parameter nonidentifiability." />
<meta name="DC.Contributor" content="Qi Zheng" />
<meta name="article:published_time" content="2017-12-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="rSalvador: An R Package for the Fluctuation Experiment" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The past few years have seen a surge of novel applications of the Luria-Delbrück fluctuation assay protocol in bacterial research. Appropriate analysis of fluctuation assay data often requires computational methods that are unavailable in the popular web tool FALCOR. This paper introduces an R package named rSalvador to bring improvements to the field. The paper focuses on rSalvador’s capabilities to alleviate three kinds of problems found in recent investigations: (i) resorting to partial plating without properly accounting for the effects of partial plating; (ii) conducting attendant fitness assays without incorporating mutants’ relative fitness in subsequent data analysis; and (iii) comparing mutation rates using methods that are in general inapplicable to fluctuation assay data. In addition, the paper touches on rSalvador’s capabilities to estimate sample size and the difficulties related to parameter nonidentifiability.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/12/01" />
<meta name="citation_mjid" content="ggg;7/12/3849" />
<meta name="citation_id" content="7/12/3849" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/12/3849" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/12/3849.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/12/3849.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/12/3849.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300120" />
<meta name="citation_pmid" content="29084818" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="3849" />
<meta name="citation_lastpage" content="3856" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Qi Zheng" />
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, Texas A&amp;M School of Public Health, College Station, Texas 77843" />
<meta name="citation_author_email" content="qzheng@sph.tamhsc.edu" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=W. P. Angerer;citation_title=A note on the evaluation of fluctuation experiments.;citation_pages=207-224;citation_volume=479;citation_year=2001;citation_issue=1-2;citation_pmid=11470493" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Ser. B;citation_author=P. Armitage;citation_title=The statistical theory of bacterial populations subject to mutation.;citation_pages=1-44;citation_volume=14;citation_year=1952" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=L. Boe;citation_author=T. Tolker-Nielsen;citation_author=K.-M. Eegholm;citation_author=H. Spliid;citation_author=A. Vrang;citation_title=Fluctuation analysis of mutations to nalidixic acid resistance in Escherichia coli.;citation_pages=2781-2787;citation_volume=176;citation_year=1994;citation_issue=10;citation_pmid=8188580;citation_doi=10.1128/jb.176.10.2781-2787.1994" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=J. Cairns;citation_author=L. Becks;citation_author=M. Jalavuori;citation_author=T. Hiltunen;citation_title=Sublethal streptomycin concentrations and lytic bacteriophage together promote resistance evolution.;citation_volume=372;citation_year=2017;citation_pmid=27920385;citation_doi=10.1098/rstb.2016.0040" />
<meta name="citation_reference" content="citation_journal_title=Applied microbiology;citation_journal_abbrev=Appl Microbiol;citation_author=H. L. David;citation_title=Probability distribution of drug-resistant mutants in unselected populations of Mycobacterium tuberculosis.;citation_pages=810-814;citation_volume=20;citation_year=1970;citation_issue=5;citation_pmid=4991927" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Demerec;citation_title=Production of Staphylococcus Strains Resistant to Various Concentrations of Penicillin;citation_pages=16-24;citation_volume=31;citation_year=1945;citation_issue=1;citation_pmid=16588677;citation_doi=10.1073/pnas.31.1.16" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=B. Efron;citation_author=D. V. Hinkley;citation_title=Assessing the accuracy of the maximum likelihood estimator: Observed versus expected Fisher information;citation_pages=457-483;citation_volume=65;citation_year=1978;citation_issue=3;citation_doi=10.1093/biomet/65.3.457" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=C. B. Ford;citation_author=R. R. Shah;citation_author=M. K. Meada;citation_author=S. Gagneux;citation_author=M. B. Murray;citation_title=Mycobacterium tuberculosis mutation rate estimates from different lineages predict substantial differences in the emergence of drug-resistant tuberculosis.;citation_pages=784-790;citation_volume=45;citation_year=2013;citation_pmid=23749189;citation_doi=10.1038/ng.2656" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=A. Gillet-Markowska;citation_author=G. Louvel;citation_author=G. Fischer;citation_title=bz-rates: a web-tool to estimate mutation rates from fluctuation analysis.;citation_pages=2323-2327;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_title=An Introduction to Genetic Analysis;citation_author=A. J. F. Griffiths;citation_author=J. H. Miller;citation_author=D. T. Suzuki;citation_author=R. C. Lewontin;citation_author=W. M. Gelbart;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. M. Hall;citation_author=C.-X. Ma;citation_author=P. Liang;citation_author=K. K. Singh;citation_title=Fluctuation AnaLysis CalculatOR: a web tool for the determination of mutation rate using Luria-Delbrück fluctuation analysis.;citation_volume=25;citation_year=2009;citation_pmid=19369502;citation_doi=10.1093/bioinformatics/btp253" />
<meta name="citation_reference" content="citation_journal_title=Electron. J. Stat.;citation_author=A. Hamon;citation_author=B. Ycart;citation_title=Statistics for the Luria-Delbrück distribution.;citation_pages=1251-1271;citation_volume=6;citation_year=2012;citation_doi=10.1214/12-EJS711" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=W. S. Kendal;citation_author=P. Frost;citation_title=Pitfalls and Practice of Luria-Delbruck Fluctuation Analysis: A Review;citation_pages=1060-1065;citation_volume=48;citation_year=1988;citation_issue=5;citation_pmid=3277705" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res.;citation_author=A. L. Koch;citation_title=Mutation and growth rates from Luria-Delbrück fluctuation tests.;citation_pages=129-143;citation_volume=95;citation_year=1982;citation_doi=10.1016/0027-5107(82)90252-4" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=R. Krašovec;citation_author=R. V. Belavkin;citation_author=J. A. D. Aston;citation_author=A. Channon;citation_author=E. Aston;citation_title=Mutation rate plasticity in rifampicin resistance depends on Escherichia coli cell-cell interactions.;citation_pages=3742;citation_volume=5;citation_year=2014;citation_pmid=24776982;citation_doi=10.1038/ncomms4742" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. I. Lang;citation_author=A. W. Murray;citation_title=Estimating the Per-Base-Pair Mutation Rate in the Yeast Saccharomyces cerevisiae;citation_pages=67-82;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202359;citation_doi=10.1534/genetics.107.071506" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=E. A. Lea;citation_author=C. A. Coulson;citation_title=The distribution of the numbers of mutants in bacterial populations.;citation_pages=264-285;citation_volume=49;citation_year=1949;citation_pmid=24536673;citation_doi=10.1007/BF02986080" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lieb;citation_title=FORWARD AND REVERSE MUTATION IN A HISTIDINE-REQUIRING STRAIN OF ESCHERICHIA COLI;citation_pages=460-477;citation_volume=36;citation_year=1951;citation_issue=5;citation_pmid=17247359" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. E. Luria;citation_author=M. Delbrück;citation_title=MUTATIONS OF BACTERIA FROM VIRUS SENSITIVITY TO VIRUS RESISTANCE;citation_pages=491-511;citation_volume=28;citation_year=1943;citation_issue=6;citation_pmid=17247100" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Probab.;citation_author=W. T. Ma;citation_author=G. vH. Sandri;citation_author=S. Sarkar;citation_title=Analysis of the Luria and Delbrück distribution using discrete convolution powers.;citation_pages=255-267;citation_volume=29;citation_year=1992;citation_doi=10.2307/3214564" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Probab.;citation_author=B. Mandelbrot;citation_title=A population birth-and-mutation process, I: explicit distributions for the number of mutants in an old culture of bacteria.;citation_pages=437-444;citation_volume=11;citation_year=1974;citation_doi=10.2307/3212688" />
<meta name="citation_reference" content="citation_journal_title=R J.;citation_author=A. Mazoyer;citation_author=R. Drouilhet;citation_author=S. Despréaux;citation_author=B. Ycart;citation_title=flan: an R package for inference on mutation models.;citation_pages=334-351;citation_volume=9;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Meneely;citation_title=Pick Your Poisson: An Educational Primer for Luria and Delbrucks Classic Paper;citation_pages=371-375;citation_volume=202;citation_year=2016;citation_issue=2;citation_pmid=26869481;citation_doi=10.1534/genetics.115.184564" />
<meta name="citation_reference" content="citation_journal_title=Environmental and molecular mutagenesis;citation_journal_abbrev=Environ Mol Mutagen;citation_author=A. Nádas;citation_author=E. I. Goncharova;citation_author=T. G. Rossman;citation_title=Mutations and infinity: improved statistical methods for estimating spontaneous rates.;citation_pages=90-99;citation_volume=28;citation_year=1996;citation_issue=2;citation_pmid=8844989;citation_doi=10.1002/(SICI)1098-2280(1996)28:2&lt;90::AID-EM4&gt;3.0.CO;2-I" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. B. Newcombe;citation_title=DELAYED PHENOTYPIC EXPRESSION OF SPONTANEOUS MUTATIONS IN ESCHERICHIA COLI;citation_pages=447-476;citation_volume=33;citation_year=1948;citation_issue=5;citation_pmid=17247291" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Raue;citation_author=C. Kreuta;citation_author=T. Maiwald;citation_author=J. Bachmann;citation_author=M. Schilling;citation_title=Structural and practical identifiability analysis of partially observed dynamical models by exploiting the profile likelihood.;citation_volume=25;citation_year=2009;citation_pmid=19505944;citation_doi=10.1093/bioinformatics/btp358" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=W. A. Rosche;citation_author=P. L. Foster;citation_title=Determining mutation rates in bacterial populations.;citation_pages=4-17;citation_volume=20;citation_year=2000;citation_issue=1;citation_pmid=10610800;citation_doi=10.1006/meth.1999.0901" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. D. Scanlan;citation_author=A. R. Hall;citation_author=G. Blackshields;citation_author=V.-P. Friman;citation_author=M. R. Davis;citation_title=Coevolution with bacteriophage drives genome-wide host evolution and constrains the acquisition of abiotic-beneficial mutations.;citation_volume=32;citation_year=2015;citation_pmid=25681383;citation_doi=10.1093/molbev/msv032" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. M. Stewart;citation_author=D. M. Gordon;citation_author=B. R. Levin;citation_title=Fluctuation analysis: the probability distribution of the number of mutants under different conditions.;citation_pages=175-185;citation_volume=124;citation_year=1990;citation_issue=1;citation_pmid=2307353" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Microbiology;citation_journal_abbrev=J. Clin. Microbiol.;citation_author=J. Werngren;citation_author=S. E. Hoffner;citation_title=Drug-Susceptible Mycobacterium tuberculosis Beijing Genotype Does Not Develop Mutation-Conferred Resistance to Rifampin at an Elevated Rate;citation_pages=1520-1524;citation_volume=41;citation_year=2003;citation_issue=4;citation_pmid=12682139;citation_doi=10.1128/JCM.41.4.1520-1524.2003" />
<meta name="citation_reference" content="citation_title=Mathematica, Version 11.0;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. Ycart;citation_title=Fluctuation analysis: can estimates be trusted?;citation_pages=e80958;citation_volume=8;citation_year=2013;citation_pmid=24349026;citation_doi=10.1371/journal.pone.0080958" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. Ycart;citation_author=N. Veziris;citation_title=Unbiased estimation of mutation rates under fluctuating final counts.;citation_pages=e101434;citation_volume=9;citation_year=2014;citation_pmid=24988217;citation_doi=10.1371/journal.pone.0101434" />
<meta name="citation_reference" content="citation_journal_title=Mathematical biosciences;citation_journal_abbrev=Math Biosci;citation_author=Q. Zheng;citation_title=Statistical and algorithmic methods for fluctuation analysis with SALVADOR as an implementation.;citation_pages=237-252;citation_volume=176;citation_year=2002;citation_issue=2;citation_pmid=11916511;citation_doi=10.1016/S0025-5564(02)00087-1" />
<meta name="citation_reference" content="citation_journal_title=Mathematical biosciences;citation_journal_abbrev=Math Biosci;citation_author=Q. Zheng;citation_title=New algorithms for Luria-Delbruck fluctuation analysis.;citation_pages=198-214;citation_volume=196;citation_year=2005;citation_issue=2;citation_pmid=15950991;citation_doi=10.1016/j.mbs.2005.03.011" />
<meta name="citation_reference" content="citation_journal_title=Mathematical biosciences;citation_journal_abbrev=Math Biosci;citation_author=Q. Zheng;citation_title=On Haldane's formulation of Luria and Delbruck's mutation model.;citation_pages=500-513;citation_volume=209;citation_year=2007;citation_issue=2;citation_pmid=17462675;citation_doi=10.1016/j.mbs.2007.03.003" />
<meta name="citation_reference" content="citation_journal_title=Mathematical biosciences;citation_journal_abbrev=Math Biosci;citation_author=Q. Zheng;citation_title=A note on plating efficiency in fluctuation experiments.;citation_pages=150-153;citation_volume=216;citation_year=2008;citation_issue=2;citation_pmid=18822300;citation_doi=10.1016/j.mbs.2008.09.002" />
<meta name="citation_reference" content="citation_journal_title=Statistics;citation_author=Q. Zheng;citation_title=A new discrete distribution induced by the Luria-Delbrück mutation model.;citation_pages=529-540;citation_volume=44;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=Q. Zheng;citation_title=Methods for comparing mutation rates using fluctuation assay data.;citation_pages=20-22;citation_volume=777;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=Q. Zheng;citation_title=A new practical guide to the Luria-Delbrück protocol.;citation_pages=7-13;citation_volume=781;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=Q. Zheng;citation_title=Comparing mutation rates under the Luria-Delbrück protocol.;citation_pages=351-359;citation_volume=144;citation_year=2016;citation_doi=10.1007/s10709-016-9904-3" />
<meta name="citation_reference" content="citation_journal_title=J. Theor. Biol.;citation_author=Q. Zheng;citation_title=A second look at the final number of cells in a fluctuation experiment.;citation_pages=54-63;citation_volume=401;citation_year=2016" />
<meta name="citation_reference" content="citation_author=Q. Zheng;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=Q. Zheng;citation_title=Sample size determination for the fluctuation experiment.;citation_pages=10-14;citation_volume=795;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bull. Math. Biol.;citation_author=Q. Zheng;citation_title=Toward a unique definition of the mutation rate.;citation_pages=683-692;citation_volume=79;citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="rSalvador: An R Package for the Fluctuation Experiment" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="twitter:description" content="The past few years have seen a surge of novel applications of the Luria-Delbrück fluctuation assay protocol in bacterial research. Appropriate analysis of fluctuation assay data often requires computational methods that are unavailable in the popular web tool FALCOR. This paper introduces an R package named rSalvador to bring improvements to the field. The paper focuses on rSalvador’s capabilities to alleviate three kinds of problems found in recent investigations: (i) resorting to partial plating without properly accounting for the effects of partial plating; (ii) conducting attendant fitness assays without incorporating mutants’ relative fitness in subsequent data analysis; and (iii) comparing mutation rates using methods that are in general inapplicable to fluctuation assay data. In addition, the paper touches on rSalvador’s capabilities to estimate sample size and the difficulties related to parameter nonidentifiability." />
<meta name="og-title" property="og:title" content="rSalvador: An R Package for the Fluctuation Experiment" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/12/3849" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The past few years have seen a surge of novel applications of the Luria-Delbrück fluctuation assay protocol in bacterial research. Appropriate analysis of fluctuation assay data often requires computational methods that are unavailable in the popular web tool FALCOR. This paper introduces an R package named rSalvador to bring improvements to the field. The paper focuses on rSalvador’s capabilities to alleviate three kinds of problems found in recent investigations: (i) resorting to partial plating without properly accounting for the effects of partial plating; (ii) conducting attendant fitness assays without incorporating mutants’ relative fitness in subsequent data analysis; and (iii) comparing mutation rates using methods that are in general inapplicable to fluctuation assay data. In addition, the paper touches on rSalvador’s capabilities to estimate sample size and the difficulties related to parameter nonidentifiability." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
    <title>rSalvador: An R Package for the Fluctuation Experiment | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZNGrobF2Pw5zmBtmrEeLENx2bhRS0WmFZmzOprPFWmw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487482","apath":"\/ggg\/7\/12\/3849.atom","pisa":"ggg;7\/12\/3849","ac":{"\/ggg\/7\/12\/3849.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/12\/3849.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/12\/3849"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/12\/3849"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/12\/3849":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KfrbIIW2h58a\\\\\\\/LHaxmBvSj+fFh+ukh7\\\\\\\/bS\\\\\\\/On4t7mFXIH1R\\\\\\\/bGBkC7GzV3DRyt31MbUY9qwpEmRcIdUjoOB2zApN8KBMISV6hZcKJr7ewH8TZ0yQRkEfKVN9l7fSjCAwfaMrZk3SGise3hHjW5a1LnfuorvPLRqEj5z7OyTXf5cYQDEU4DDV8F09xc+IHU7mzha8LmfvMYQJHAwMZ5VJ91ddrBd1s3o4TmXacXfXQc5WoQX42e+VTfvsQciYhGBhSTiwR2xjuSWKqimUcz5bMKe4FuO42uC46\\\\\\\/IOuhiW+yiaIkdGLWXdlaAqHvQd4VRVWDEOcBTsrYjrAkgB0CkQCwve0tVvPHfaeuAC\\\\\\\/JK1\\\\\\\/iZs680RuzKBbX2bHMHNJ\\\\\\\/wnKtUJYUAbkV+8j7j9YYpF\\\\\\\/rDAmCoA0QNKZwsU\\\\\\\/D8Ge6qr4M40RwxQfj1LwGsfmhc8JQEuZoNndDZvb+WjaOsKwZPxHHgOeknRwKWSyKFH3oHGMgG8QKnUWjkqJCrfSVjqN8Wlon+32RqOn6DtrLaIk182y8KRrzQdYYv96UBlrAbRUSI7HSsCzfsamP2OzTRcjXbbBNjlvR6fUTORaeBRPLyuk8sCT3HBKXmqzIQ0K+BtwQ\\\\\\\/e7fyM50s8Bbt1uwETv6JjCJ1gYU0fJJ6cs0G6LLtwvpXYCCFh4\\\\\\\/9AWIQAwX8hkc3Mif2ph019uJNOyikK1cjnbtA8KzzL65SJ9D4I4YxlWIEIB1oRYuyi0KByvgU6KmXLJ5ivKxr1vhqN6CZzf+AM1WmwiC25fJdONMO4pzjpMxRCoBOOk\\\\\\\/2V2PnpUiwDBzruHcI0e7OIeb0mvy5nCPjdqBztdaTqDvx55\\\\\\\/FWQNsBIKV7CQMEKPxnQ65f6wmC37n2liKxBrSpYBLOGXKP1JpqdpZnIWthh9QD\\\\\\\/JsAIFa9qYsud1lC2hePszTMx0ch36\\\\\\\/gOwFxI+SL38FdeEY\\\\\\\/AeKFPdoayxG1oPEnBDiJClO\\\\\\\/CSTCQT3EAP8QlZuk77sxYUztj9kmumONoe4iJv0s+kMNMuOTBLv2E+3Ke5rlbFY8r57ad3PJZwFGIQu1OaH62oIlqPnkpg5U2lw2CQVuOiqizIM9WZhW5lv8r5P49JL2o2+fWUV55VbmRkoVHYh7yKn4u1g2on6wZwkQ2D02ggxh2Fc9dczpEjj17W4Whz0eHdGCJqymSwfoIOw11KrbV4Eb0gX\\\\\\\/Vapo9O8vV2LHTx1rtv3j5lg6uP0tXDJrRstEdRUusoR0iboletpRIXJSdULYP6c33UZkGFcc2mJM\\\u0022,\\\u0022iv\\\u0022:\\\u0022rXpqfC+bxehzu0qQpMbetA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022abbd4a7208e6c6a5186e49bbaeeaa55a3760c9d2a7c84ddff814e08f415170dd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZeY5TKylNobKaxCACy\\\\\\\/utRGdjUyDYAUkJrdH1MJJ3KUx\\\\\\\/4\\\\\\\/MubT3Z1YFkrQEstkG5cobZ4snuxAwdu+NGufDWE4GGt5tTqIwW+rokluLF0o8iqbaZZyIjgdSbgWrsoYKQejPuQWAbQJdLPuAkZD1vzgia7gCHWIey3dFT++8AIL\\\\\\\/ZmE1MT7pu\\\\\\\/7ytjZlm+LbKHUDF0iejemQQdVngB0m2oXVKsDZWW61f7\\\\\\\/IKWGwVREbekkBxb0KxP2tfEZKVU9EBG5WUs1SC4fslRwjX0mLK00z3WPix3T9eEjWk1177ILyR76RFCRRZ4Ji8SpJcugvCfWX82xB2VSM1Zbs72ae1AttYW0dpNhPuJaAqzgNanO\\\\\\\/8PtzeMCTQC60MPWAuf5uFJo8gMxVvjLTyuaz+yn6Ar1z0Y4ZC6pgt3czq9gk7MrDp9QD2D+92aTCiZB+BHNkq\\\\\\\/D4n7173NjuXvjLxYhPjzWZPhC58kiQyiPpwc4mL2kTpHccG5ZTQMSK3MAQ98JI7SJnNr+qg7\\\\\\\/Amd+zTP8sfAPindqQ0xRWuQ087L+UjYwjUfCC1RbKJSaLUtySzSR8k7PULLnESGNSvxUzfG0US5VUjrkd4sb15qSTYSP+1upMG\\\\\\\/an9GPVdwWBaSPyQSmpePuy+9MrDYZPWNdvWanqRFBwvGdMfHpJi7HYKLo3DTCvyPIq4DfGWeH54B35AIMZh5iRE4wXZxETkiCOhF235L5SxDiHiqfr7t6o7b8xT7\\\\\\\/oCALAcd9YDxdMfsccQvtjPvzuOC8tUJU6C3LPQHHLTeweLw0TQG\\\\\\\/psCCoKtCEAGM6nK621OK5WHj0YRyomXZ5oaU1xqm9N2L7w1oRwUuWIbFX+hMN9ty08PwBJVkoW\\\\\\\/IfSIVK40BOuYQx+QSvK5mNR6qgILM4033Z7g7VGfS1Kt6Hm3L6jCSf8CMYRQJH95OxPJbNgf5nKwRkSjb2xcHxdxq5jE\\\\\\\/PaIYXLCmVUnirSDyY01oe8M7ezryCy4te3GuMu32qzySxuUJ3OCjnFtM5Z+kbrc9CWGEiPP8BY98G8L7L9q2giJz4Esfm2vupXhhBMuI\\\\\\\/HvaPEKGrCJ\\\\\\\/wnwqSQooV9zezLsHaD+2tr4sXjtbdwVmGX0LjRW2qxun6mdlsOpf4jbG2RqLfcT7Mbvxn+bq2ATPT5D2+WJE4O+6v00fgwxMDNj9TQmE9HAo\\\\\\\/s3StRO1jgLHhxprrkosRGbUsq+5LRIF2rgRkfD6fUpyqIHk4PmuiXPEh7ABPgNt++aXTABJR\\\\\\\/xiyFAn9wEfFzQ6++V0sc4yxC632r80kXbEr1Xf3wYY8S8VNbrphtWdkiiAEeee2a8KWlh1PUqnK6wu7tciOxWgE9n+qLIgfMnHfdSjKJywJnenhhPeWJO1fhaBBJN6PLuy8CPtH7GqCkJ9GQhrEOwScRgTqeUYIzH24rqxmqcJLFTuRyNj\\\\\\\/z0hhUmuDs53hDtjWHAUkjcKT7On1NXcppiSukQZY9+k1FxHFZGFUFsm\\\\\\\/\\\\\\\/olabNqThuwPDRK89p9V6oHoG7tX5a5lcJfgvrX4mD1lyCLleLPYWudnymJPvhERExwqwMhKh3cvrUcIYDSy4KJpQUZC1KuctOR4kgdoKFKuP97ObSglo1Vrj63J4BWy57r0+7T2ZNEFgtDnD7YeeeiZLSf66T4WCZczTh6pR3wgSpTExtpa038mibAy2CYERHgrw0RrscqFkU81FcX3FUHAnGHIzX7W+egGz7+vXKc0epZo6kygStE\\\\\\\/9uU0jJXJF+8GKWlYVHfi8xztYzhLI5K8gE0IvTdxxYBxSW\\\\\\\/6REf59qy1geWKJcoXXaqJulmXU5NQWVEtLa0Y7AtgLWqKoqs7tAlYwq5di0g+fR3b4i3waqQb\\\\\\\/vta1lB7\\\\\\\/qPiMWSxzd1OUjNDf8fhUrUPCZ4deIyoTvkf2eU9zMvfjlelPlvscjVELz4aaNMmQ1+sgTVrAiboNuOFe40CVguzuAaI02JFLWGUWEPlAZmiW7Y+sKFedgnHnW3tSS3g2W7IAM6W9kWg9sHoWvZVY7fAz+tQhwAdBh8ZtHvaQfbG4m8l9v3mkYU5OT+w\\\\\\\/lALtCLyNzXw4buoXuI7OYGQFi2fPxJ6kyUB38yQlb79524d1RNZ2V\\\\\\\/X3HkqtiS0LqRzrogaC4wUIoQkpeR+czabMzHCPKgIq4S4rvFQgWV4uksnkCKaZJOpcwgt4Bd4RQx3tLIxC5rxsnnhN5ONDFK\\\\\\\/lm955E8f6i05aNw2skEPnXyzy69F9uw6L\\\\\\\/6H72IjAayf2T\\\\\\\/viKV5MEAh5plY8sz3hl1Oyaw7GUrTHFKdcH99477PTnT9JJcDmViLqTUHeMk1y1pN6YmLfQLaNcjqFlzJ2jq1DxsNXcVdKFjOmJDZvvSmgGaUnE4MTXvx\\\\\\\/8B1mBRSz+arD9WWc0xCpi90NlNpyfHgMhE1jCba00UZqrZ5lmIzI7UGyZ\\\\\\\/HZOPXFN8LIKcXhA8avSV0inrNpvFOTQr2StJJuc4s+V1h\\\\\\\/v3Rr1BivtL7k2WX+e8fB7sjl5k4kae6M0IuVVxMxym7BnQNbtx\\\\\\\/Gl49ulVhMbPHYCmoUuH+QUdziQLgDlX5s\\\\\\\/3HDBAQsZ2uDX5TgEGddGVcfMIMcvDEafR+LAUwRfpPtw8QC1C54kDdVbfyDJgHB\\\\\\\/zFHT3Qgj4OfX72UZkNSHTSjQML6vdg63NSw5vrYvPYEOD59NVr9ruMkWlpbZf6QlMu0a\\\\\\\/es1o\\\\\\\/H9wxKpLtnWpiVTOVlOhmFT0KtnGKZVQGLlJsdxTKNqNFRGBqqSNfrY9pHymKXdxINIOAhQgJE5wywkpGgBSBiuCS9My8iGFqZ70zevbcHYPHBO5\\\\\\\/xuftFQ\\\\\\\/1Z0b1tVwXsenVhlpNFYnU4a+kEx\\\\\\\/kxr8al831L2lsyTQ0klO6bNDhrny3S+ZWBIL0nDMk3fHls8MaFiX5gNud13oeBV7P2W7\\\\\\\/Zqbf3Gamu3rYxXgiUicddaQredS+BJLzeXP65dUYsbafkMF22PQwOqKDu5II3zy43lCstj5dKAG02E8CWoO7\\\\\\\/hCs\\\\\\\/Ob9BSKJPo6osJVlrsEIiHULgxAFVcbHoklJXx4IxE6lBJPfFJATC+juRE2uhoObXSkGAtMARGSPyc3+Fg\\\\\\\/c7EfoLk1jCPrW0eL8RvM0z6nHvHz+A9LPlTk2L3JeE+vfN39cnoVoxAwgMez0BqXJg+cha6zOjhzAtr9GUH2ne5Ija8J\\\\\\\/ac+cIoKv7zvKFQPeLme9MuY1Obd3xngi1oPfss8c9kPFbdC8oNwvs56omIXCmj0m0TZXt5rw3llYbl2xxxpxH\\\\\\\/\\\\\\\/Gb9uHm+mGBgH+FTfdRKDBwwz3Ss0dbQ5t23mpsbPZcQPQBZ83jXEponQaRloTF1kJwbjxIoIgvbZh1OZqHBglQR7QxgZX+BeyDH+XFYyQGCPZmQ+tMwpzuC1tGNuTKL4K\\\\\\\/GPqEj0fqWcqo6UV8MllggJ6ocCJOyhfSx3e7elWkhtztqzW8CbtYL8iC7YWFbasPZda6tLfihdthgK3vXvQD8omznwSEJqjbr3eEhLXhIrK5WklCRUpcvmehpjf16EoH1ziY\\\\\\\/QpMbW6gcznbgs\\\\\\\/vkGpBmve0qy94dPHy3ojIbcLy8xFaS7cSIN+P9bvuwUMb3XAxXNV\\\\\\\/ymXxlVZppUJgQxQZjfHMQi1KbqPz2UXwEEbCcgF+rpQoai7CQF91US1U45i78R7teFMxxpL7pgiD7WGVkPe7rp2IDjH\\\\\\\/2OHuEcDLIGcNLUnvkiKkgJS9dzNQQt3\\\\\\\/9DPM1norV1Se+OmP\\\\\\\/PuaYNn5sHqXY1OSAeVYIJi7ADEE1JmNd+QKHDq5uLQx66PE+bGqi6+52\\\\\\\/C3bTJKX2lkIyJyyb780+O+iuEXALxcEqFjHn5DIQ3iEpMdyh220+m08jee5uWxqoQfYdW4mBMKdDPp43+0HTLMWKmDbVCFFEdZJaoKVMD6NsBc4hlO6SFsbre7v6Il3gnbRtALg7p228G\\\\\\\/OwMEICuEEUm1z2K2w6qJ6A8tFwEEg\\\\\\\/umxgSljnQC056GhPrrMTcG\\\\\\\/yWf00fw\\\\\\\/noM5zQNS0bpq3LwRm95AoPB3IoTYNOLUB7s0rr5V4DmMz\\\\\\\/hZKekt7RM\\\\\\\/qKmX3a5YMOr8fZ8agn\\\\\\\/aPDLBkIThJeVNtOVXNociLZNZtt98iHV+gJER4KBpdqtTTgrY6XRfdDqJGeLPfxsvIuwJ6nJSNR\\\\\\\/Zl6iVKSKHLN4Iey60xJRu80wVY0PJ292r\\\\\\\/pa6oyFyyCtthFQ6cBJQjmUUgob2zraFuNpKF3M1dnbbgMZjHAJN4s31MBE2uJRrCJQNAeLVRZsi8TN8o72rxMlxgITEfuyBZtBpSizCZKCJoQsZAC90oFssNS\\\\\\\/jmuuvD6fBEMmX7LsKq+VphrJBfvHWPW91zb2wAHN3ZZSOoLIwclVIQEnFHTXMLsrnJNAcV7SQCV491cPkoc0KEq8HIKBWad9vcw1hEeg4psNVb87Vy7h02+i0HklJMZO3oMpQYMER9l0dHtX4GLXY8lSZHkiFlW6uHiCDapkhfe24dMNt\\\\\\\/1Y5Drfvw8sMktqZ8uNyJrycKSWOCF4QEO4azB2NEmU3gjZF9ChWZhoxsgCDC+ncZ0JykRhQ4KSlL2fsbKqy2Fmjj3JvLdXVexi2XaqHH0HRKJWMkLEmXoawce8BI4WUjyCPQoQhEn7Q0V3MMeSjMt5mH5yHK5HE8I2vcMLpkHBbFRLdx208uBPMXvVbrUwpTWRZlytF2mETdMg58PrwkB0XS2ghY\\\\\\\/WN9MElCNyBL66a7M1mlHqjskSs+yUKrsf9WUwQ1R1mJbtacrvrCj35tXWhFLbjQpFyADdYSOcrJ9x8uRaFfaw\\\\\\\/4UQVYF389UTHz\\\\\\\/iMsbV\\\\\\\/avC0\\\\\\\/DVPm318WtcILXx7FvffBTNhixpA+2QWCT1pPCliN2e0k540IJpCN5B7q8z+WaNbhac7Tm4nNjChJxvJDIfGoqxEru1xSbF6FaR\\\\\\\/NgxCxx2PxLgIIvI0VeDGNkoGPq6oVCDLlyGmdI\\\\\\\/2OHOMo3vin+p4xqST5PChH8WJo0Q8bJWT2I4rG4RsAq4hdewzQCev\\\\\\\/jt9rbj8HEB1q4DIziHQgOoYeCVx4hBL2rwUnJbrEytUxZ79HM3IdByw1SKu9xtQ1uRUIxBHXTxmIfJaRQflsAq+wM7IK2Wrr27yvd6p0KLalYgXPUCr19UopaV3F2jKSKIxJlBCmPqzy2S1GcOvpRTj6qHu2RZF4YTqfB6AUnkSodePT5RE\\\\\\\/WwgopvjO2TU7phhU7aBLBE4TnZqNqhsnWpAu839AD7oB9q40L44giLjAuYd77zsquzQQ51rXODgDIjX3+hnO1y\\\\\\\/+ZRh6uf7LPYvlRtYSSsJWMX8GgbSpX+hUFy9rsKD53+MI36b8mOTR6Y8bxjGzf6NxBZfdit0DOAZcny+s+QHALbSwHrDUPE\\\\\\\/p1I2IZ13Jr0GJZA+EFVgi3gLuq26pDbL03sKkiA60xhvo\\\\\\\/aOmSg2n3Hwxnz5oujS2Ud9xy3maNCVGAoszvShwg018kJPh+adUw666ScyUUWBIJtKaNURepdxdheavEBklSK7uTu5yKvK4SVz0gjoQdNcbqKGbmrmH0xBQ2LusffB42yCOAMeHDeojKE+gmuGB4bjQPMGGydDubfargtCwb+KSTfHcmCYrW9oloRRJo8doGxQsRYsvXseQ6Hy6GdcoOS+QLKn2Oz6e106vLuyd2hDchZdISD0fz3gZL5sAbgj54jGFQjT+YU6jmU4bSnhkCF+x7llajlxdUbJ3ufI3JwxNoiJrsnYTlZMLoNWpILALcrGnVpN4bAuEjWAOz+hAFJr9It3qgXsgJzvycqkonuwa\\\\\\\/ET9S+z7j6ag+92nEGl7IqBO3hTXNpFuEBNve0x3Kam6LSRnJd1pYwXrHWz9uNMnyjgOWkz0tfgSeKz\\\\\\\/6OYfc+hXcGwr5ELsbVksPOyCStFSDd\\\\\\\/ornW82V3Q4sLNmBEdsMltwt\\\\\\\/oQjSQifQOb3j5b25CwtScyidWJO3N2FzFl+2+cBDDn1VozBHSTrFw8tsSdMO1QLCfY5b4vxbnQqg6q8jrgkT4c98SJRdI\\\\\\\/YTe9LRJz1ZgrX329BOuZnMULFP0ZegqZ60eN9bIbpsWea2Uin54Vm53g9V6aCj5lS4mFQjuNO3T3bHSllhbTzU885CgyUcDNNDEL8eovO3imM0w7kG\\\\\\\/2veQFESkkWhv6ctT0+Z+prhp4u1ld\\\\\\\/dFrwzcWfBAf6Vj5GAdVmSVAQLBro9tMHyOxUH8ddnlm64bVFYqgodEuh+VQN912X\\\\\\\/xmY8hqUh8\\\\\\\/cmnTXO0w+\\\\\\\/aaV1tiQ3zKGPj6clRILU6fSa1QlWCFpgk97vAGJVeRZdXd\\\\\\\/Gg7daQYYvy1RzPlQ0T5Arv4jN\\\\\\\/LVHLJH\\\\\\\/RWUrdanI12gnP+RYI4v72J75KOnu6bgkPZXtdCBn3Ufpq4mufmbQfz6qeaVwmMTaHaL9cZe\\\\\\\/s26dkA2EJ34b1ZmjaLADI1iY3RJmDAc0aGb35Pg54+1328Qt8FYQpoCGo6dxX+JVausQrDIM8iH+7jXm14iWPSwkjvdI2JPGhyu0AAjOaMxYycDf6GcU2AfCdeZElyJXVWflH0RO8l8iWrM8398X8uQkD\\\\\\\/tbcNhfkh47OWT8FXnw\\\\\\\/5diMKfRk\\\\\\\/RUw+tccusK5d7PtQc3TBMiwVUrSpd8TRaPNtOkrpBvR09vBcB+K0j47EoupITrcN\\\\\\\/3ULsuJ1SdzUPdo8FxFHTIlvaIqxgtkfLyHBHesqQg1PdfGQgcAWxq02vAmurihcGlaEb1Us2DpVPwfKM5824ASNvRMd1AShX5jSn1IGyZlYHOsBmyROCJSuuuOtENVKwS+znt9AjxQoJlsYae5VDQw7oHIBc9NTb+CYhPuB0LO\\\\\\\/TF1lrUr9+vGnTdDBRmVKOCTtgC5VTIsENkmeRxryCA\\\\\\\/eT8qHjittGUKIWWFq7ELN2Dqy6PhN3GxDdCEbUCc\\\\\\\/vaZN39w+LA2kfgETYdcvLJROn7\\\\\\\/\\\\\\\/wBPzqRFhLlyPrXZLpVUFzR\\\\\\\/WTkshKq\\\\\\\/L2gNZzDx9r9FbWOWYUw33NnOIuDunzJ+uQy4+l\\\\\\\/g5alRYgK1poRKTRu\\\\\\\/hqSDfnTiv2egcR865tlFB\\\\\\\/EhC7tpwN3uOPBcGj64JkOULKWnUGWMNY\\\\\\\/t8revCUJC0oIVYovkvCxBRhOSAJqOCLnzEcTx5uK0J30helw5hf9O5zpulOedXg9YcH30nxqbFW2l9tMVG1JpgNk8\\\\\\\/SCsoNH5o2dK7UWD4\\\\\\\/EQfkmq\\\\\\\/NEp204AQQU3sHn1lMgPtAZZ1tvW0K0nl+5YlYxIDw\\\\\\\/x9xiIiqPN0abL1I+lUh3U1\\\\\\\/pCz0EQNkuief9QhvuN04mBa1WV\\\\\\\/\\\\\\\/1GUXSJBZkFyyB09gU+kV2EzFgHHm0vq9rHCgCGuVpDgkwxxgFiIJ+GkmA\\\\\\\/9SKZmuKzN\\\\\\\/LTezOT7vXj5yILG7lom0L1gWw7drx9bfNeXXB8BVl5imRPCcrfQFVdRFaWY9t2EL3ZBjtppYhsg2OKDlP+0D1pYRW9o4c2HCH4lVxid4mTcifljHjsVVeaN3ivyilmv11\\\\\\\/mdX5MGGt4PX\\\\\\\/gSPFOg36b1FxwkNiUX82\\\\\\\/j+x2l6gRCa1QUQ8plRT+Vq1iGT5uVy23xyYl3shHGhDr2SnvcMPLMedM6cBz8MtoJPvBF44Tb+lh5UMlWws1fCz+HdmoQRJ9+W\\\\\\\/zYRe2PrkT0bhdLznW+gEWPmFhYl0SniM\\\\\\\/sNf8TVbjD6c+AB1Voik530\\\\\\\/ZPhb9R5Z1A\\\\\\\/qNupqw5HiB+Ge6eccRxx7wzsOhpcQDju9uVhXmCWGxnA\\\\\\\/2Hz87fvLV0+yDWzZrbOlER7ByHdXlt+BYivbBBrvReIBezAfJREuUlgiIT\\\\\\\/nJfKFn52qjkRbLWDI0eb05HaFH2apUYasgqdCfaMqEZzdEKLzZaHsYkDCA3+AtDJ5PQko7QVYae+krycTQAFFv9BEkrW9uOGwiiOVz8VUTf1u8pJ3IHiv7pfywzTCi\\\\\\\/LkbyuaUynehQRpCrV5M7YwscVt0wWWd8\\\\\\\/jKpbiuopyoNatIfNY6n32YrfOauq879rQ09bunrfbzcz4q3sPL3nke+oQXllwtQwU3H7KI39cnam5pYhcdEL5IWLNqSFiji59VmWmtIQhkFb+flP4F2nTmvgjUsBEKjKP58E4dLLgCaAPqqZdiys+1lh\\\\\\\/duM9nRp4H5ObDIQQqEOa\\\\\\\/9XRi7F88HBQD6zGolXHsi\\\\\\\/z4t5v83wMDxd9svWuzHQrDFkZO\\\\\\\/kgm8EP\\\\\\\/2+95v\\\\\\\/pqkkk13R52SSjqeJ8MNZaFaFFydg1\\\\\\\/\\\\\\\/l0nW9hzOGlfpajB2vtJzJD11IH5efHe7u9sqKR2uvSWV91ya7Sxz+foRiFqSWUy2Cvy1wa9AMIZUGEGYUD6CcpYGn\\\\\\\/s64Od8140jgV4FV0LORP8ZJo8\\\\\\\/i3fW1IZZ5eAONVbajIzBYLrJzY4Vf90assKb8PK24Uj7r+gkn+QtcQYehgUchgCpEvN5nFkXFo9y3iq9hRmPg39lh7o0e51NudjD1EzUdFgAmoeynhjZeZ1UuGVss7JPvQB71LsImZzRhjOlqCrv4lrqaDYrbbJ5bl\\\\\\\/Z4amBGDDVkbk1v85EO8pA3+a+6N7pfcLDQUl4fhkusu5tSJ6wcJHmTCq6AzYzvs8GPPb5XBokcHv1PmqXT9cQKmiuua7JDH0YXsExqDNBq2sbrPHwQEtPc6k+14yfyYDMP60+DXi1CUYnKkEfWGoAjilD25QlMbJZIildJCDcMPKLoqpfnnP3JWReGcWjfbjTl0pag9HAATrGe81FhEyN8dALhjNmskjICTXrKCsZBkZe4UiRstWRZQrlGU3yl0f9JJiYAUZX9AGzLA5PaO5hHoLObgrJomKB+H2BBiuxmUELY28InfMM5WgmeB6zDKzLoh\\\\\\\/n1bfF1EUSPKrihyfua8RjLy0HvC8j1GVodamzfl2crhhY7gIRGE+MXZ+q8Zb9Rr\\\\\\\/9HyguMfYbxPtio8ZEHKMSbPaiRW13NV3nXwX+UfazcMAnECrAjoZ3I54qAsYkREAZ68954MutLSdzfGXIya0UMMTz3pZeOjubfI72QqwJMcdXKSqAE6gQGbKMgpOkNT2Gu0nDYehwKMe0O0FkYVie4k7AuM\\\\\\\/QBsKAzsKCukBhjpZbVuMNUnIDjawBGw3szbOvuOwjrm\\\\\\\/btxIljUjV43ePsm6ZW\\\\\\\/70YkOEQnim4XZBtQlB9H0SeCAZmseHI3+EHkdWLY4\\\\\\\/0C+O9yvJizhlytzuHn\\\\\\\/A+YXw8u+uMfVaGKOjVYmij7vzefK7T74HZxg\\\\\\\/3Gl4AWDhTORZAiWrqAYrh0YfTnDmXMh+8nkQZq4NGcUrEUXPslFXZ66OqhCxu937OrT7AXGA\\\\\\\/CEHZYFLCMbzHLrquJoKXy8gRDDdVuwpZYpqOYKuLF9d+1Btr49DG4hsPzXD7mTO0dtUujMza\\\\\\\/e\\\\\\\/lUmmAtBex7hit\\\\\\\/pos37iFdPBff7YAsyeYqF5NnxUhk\\\\\\\/Eo\\\\\\\/I\\\\\\\/5rQYiyQE2RIEdq4qxol8MlbbW6MuUrFiq2hu98EMymgPM6Qh\\\\\\\/OVEuTDoc2RWlTRr76om6fV5ARg\\\\\\\/Gd3+1mHVzKqvq66KUAz5LYioXweqHT4UVXVyoBcY1Q2rj6bqweokFPSbmbdhQjMsnhJ9x61mOM7qS7bgBLkNe2VmO\\\\\\\/yh4j30V0SIYnF9tG4y73pjP0u\\\\\\\/fuXLCILjo0MnWXQHJv4IF9Fe++FFhe4o+JzFcqgPjqRSSs4j7bNZe\\\\\\\/kQ7SPSTozWNHQPLNnpY1Xl+LO0y08223bpJhXxnbzqx7\\\\\\\/X9wJS18gUahXAmH7TGvUeWB+Cpq0Y9v1S9rQcCU\\\\\\\/mvJAKuC69rUdoxO8RiImzl7Sbu0LVdagXNNhUHnwu17uYY0soYLm1ztM19FyQ5T\\\\\\\/8\\\\\\\/d7+c4jnYkpko15FS3sD8WVNi3Yf0AL\\\\\\\/SxKRaKCDNIpFw9\\\\\\\/7O\\\\\\\/nqr0OxDF1yFSY9hIIS0NdTb9C4uMNTXYUQFwMLSrdlCsTa3EuZ2FviqN75grFwZhodrkFkCwG3kdul9NXyZdIjxJG09SDcfYqaD\\\\\\\/Tyq2peqDIEzix8QWGB6hZqzCRBtDHylBjWR4EtAz7+w7U6CXZY8ogOL1F3dtatPqmXdmjsJd\\\\\\\/bE2ffER1+dhSkpOJoI0rqKXmDS0APxvUdQE63jhF\\\\\\\/gMBAIsUXWmk3ZkBMsH6kiQ9agqClmwoLX\\\\\\\/c8kVAfr+CgCCX+I1K\\\\\\\/9p4M3nLiA\\\\\\\/M8c56IChURriC1+cvuyI2\\\\\\\/zjM5cIgzhgrJxhRmduLLU8ztJUKeKF7hAZCSfRIEoB4pDtB9DHBVgCDZSjEcCOBBVK2cpEQQFA+MIHgj17aZkitpdsB7LOz6VZ8eWWI995MvsMdltcAY8EwNaKXighIcaMHCAGf3j2NIWNIieX4a\\\\\\\/gyBE8eVzwUj2OuVGN98zKOjWoZXeX6rgxurLc2X2gRkYbuQ+\\\\\\\/UUfeaAk\\\\\\\/3Ljbm+5ePruVAnicgq3tWOdGEdHgLQwYUYpB05KmX19LgsfO5OzkOzvP3STj1Uccj5cjmdxesS4d4OJXqjHr93GafsMtuj24eki32d4qC7QrdrQ4nGVCCDEH6F6+gglDc97\\\\\\\/Pv18WefQJ7qe4HOfWl0tdDxhfEWXHaZRfUhGn8rwbSAWqOQzaVuL4+dDusGG0Uc3sNCUaDoAJLpwh5i8h9EFdMFPTtCLd2btFjYkUW3xuXrTcZ1iLAt9STuFs9qioMDRdBQIg3\\\\\\\/iyNNHPJchvKebUVMuUSZ7HUUbsK+LCAbIsaICpkyFXHVMSDYfu2CL2WXTZ4Ht4gVbOtJtyPxbKjQQnolqClVnGJHmS\\\\\\\/NRpLKrRqVy7T21vA+9+T3Z7H\\\\\\\/OCmhFnHGeAdfiBxFmq4lYqU8VN5xYJb1pxK4QjDdH7ExsBniivkLonFoM9oWJnDiNHvWeq2IQUxp+MjmSWpboQO9dXQCu9ccYytP1Wpxcur3ETSrMce\\\\\\\/8WaCzkVjk1\\\\\\\/HIPn8VTwxTISsh2LRoO+BjIHPo6RQbdRLgHyBsDV1BTvJDAzM+tMA6BPREdC3ZRy+bHLx4088mrRLUraYMMmHv87642DT6txXvn9Tt8XAQYUIoTeolqdljcjZI4pUSA5g1yoY4NUxUtitR8pW1iy0ib3W0Ih4IBj8IbQvGWDKxSymGWfIk9\\\\\\\/QX\\\\\\\/EnAURfWQjim6tYAx8jJJqFDzPYJkj1fCRYXA2vuGN4GHOfp2Aei3ji0zQrW45WBH0uVHl4S62nGrtilWYp3Ai64m19zx\\\\\\\/\\\\\\\/xXDqVN\\\\\\\/1D6oDX\\\\\\\/YO4H2JUgiyC3VmqDMIOF1LWmP2\\\\\\\/GaxLZRI6pQKaeCELdKv9414PHKNY86o9JrXw1nPruvBunHBD\\\\\\\/gpNoIfCh+Pq91RPddNL4XoziCI2VEbnfwvRfUVgwjdvQe4OCGP90a+0E+p3EYOouadNSI3NH4gs35LCHi9X+OKwJEaErSKL1EzoO0wiIldzuV+Vs1NcuGDEvZ3y8LCd0E+c48IZpBGH2auqfGrBZh\\\\\\\/BVY3ORvzr7e9TjZW1lSX\\\\\\\/\\\\\\\/Yg4DVYnni1zIIpazpvepLqj8W1oRII1uk+B7m86gqmS14lTUuOieKDSkiqN1F0Jwz5meAIJ2Jl58Dt\\\\\\\/ieEFFw+9z9yiWbZh3K5cQn1uzRBrIrallVnHcMVRp8ioLPGqbZDexI0HqhfvClt1NeR65RSxPRW\\\\\\\/BpUXtSicwpAixLpEn6A0IWNRaKgxe8H2zA2SPkfXeO3rDA43Hpwemh4hRMMOiMERhT5JNQU7d3ULXg57Sm9TToH\\\\\\\/zIAf8Hp5I56w3++b\\\\\\\/Drz2iBfM0Zqup\\\\\\\/hFFBcFiGkiBvfy5z\\\\\\\/\\\\\\\/4zcJXU8bBuvgVifIxCUAcY5BWvMp5usTe1S51Y9MEYd6TCkwAMhmGRsKbkkzW4lW3mOink8qcoJjyGUHJAsZClLuBd3gmvtIjJr7UMz\\\\\\\/r\\\\\\\/KEArY70BAuKPrCSamlLtAPaPs0OY6\\\\\\\/jcqh4Y+1w9JpiIeE4qPp089F6lb6xkyNgjExEW59rg0jgRmU4PtZ5SveuqLMjFkTeNuapqoLraa4kKvZiy4Ux5rHHTkritCbfrWQcuBQ9yjDw+4GJftTKB+S6WTrNpF9imBXRJGsWoBH\\\\\\\/YgbpbP2kpzroNlFeoudgCm5CQESbnQGQW5nfeG8x0IhtixorFT9wKMtYgSIXbvS9hXW4kvhZQxLieMCXUXSSO6XmfK638agDrIpy6o+j04urEiED8u8pF3BrOtukkxaMPe+TG6In5G\\\\\\\/L56ZRaYD58oLHe5yTyOBEYgRzPI7q84u\\\\\\\/8OS7yCQ+gwlc068gYsUhkE+mevfU\\\\\\\/3p8BldpZJV7td54xT+yztBZVqkrlwu3Dzh9D60cXjhLLURhKlvWiYZ2LyPQNf4dsXahaGvMt+gEnsXS44walK0nHnjiBARVpTTjrlF4HBE\\\\\\\/Y8BDrV0KcSpk96rBgx\\\\\\\/EYTgI54M+mpgu6n9itTFvmWphQLaYgYCpJ0E74+uT\\\\\\\/z4bK0T2l3tFD5bJVLzARz1GCdgIQBGMoUFT4Advdrf\\\\\\\/9q3cz8DMKwFkc0Cn8Wpn62ZNaVUMbZ2jdr8jEYqFr5zNp5fEcIKTbyCHFHk6tkmTSIlWgoR+2PI1osVhowUueURzhTEEapngw3Dtve\\\\\\\/aqW6JxCjeuM4miSGLQHSllZrL2HqzTSGJYOk7\\\\\\\/e\\\\\\\/XWA8sdodE3gxiWpzIMVyJaDtQV9OzoOLZbGyiLJ761tKTEJmNSh+lVmFwgVC6qc0ow5l\\\\\\\/yYiWjjQtJO2sp6PKGQdgggZ9drYMsKMyFhDYtc1OGEu8hfCtgeyNl+kUqigxW9kSdAicf+6xEpHYDP+gLHhPk+17h2E8bcTjqGYhB2BpyCKzSpGLYdLVOddO25uTWmhRkea3eBYhAVy1Qal0urV8jsWOi2qkwTuvF3jlRM+LcY40sjQg6ieCAMJbrHim36mprxcwmE4tYs6odk\\\\\\\/OFzFUyMcBIRALp+cOEsIZLmVcVThvo0C2Wrkj2ngE7Uk\\\\\\\/b8Oa0gSCCs+ChTNtLODZRdllpb7H4DcpeXHLQk1d3b\\\\\\\/0isMHkSNmxjyBs7oTXiZ5e73n5E8yfaGcUJO\\\\\\\/6PORLSyFHZa2YN590rh1o6YqjMMn7AFvEEK71+9MgNlulpn1UoP5KneNZylWzw7tBE8eAekeDvjMh3rz6b3rpR1cMP9\\\\\\\/VGonIVRF3kYZGtvoD0mpHWmhCj2iuubnJ0eyAHfT1+IT7lWZCWFNX27hE8F2lwkc8JqbXFYN4OckPzndnHGMAQX\\\\\\\/SdNAgT9xOtAUija6\\\\\\\/rd3ZpMNdQIwA7wRUr\\\\\\\/a7n7JX+e505YCtCCWpZ4HJaOtzAarNbQN7R1GfkbXftq3MnQuP7vgpcfQjomivqi+Vst\\\\\\\/AgnueqHojIBB3km2EXP0kEeEbuYZl4hZ\\\\\\\/JRzdD+992VN3wrF+E3cUAbZBvs0\\\\\\\/LqUK3mnCW+\\\\\\\/YIkZoDwGK4Asi7KUo9fF0BDn568S\\\\\\\/bkPJbKi3uT+C6aIv7ZPsMYg7sCW+T3JPcljo6H+MHhUy3Y\\\\\\\/gkymfSz0Yhk1FaMgsFwaLM2Ts1ndL0wMVV+a22v0oxyE77sXzNiqZNtZmd8AiNq1YzQUqx7ZqcpTPObsNk7UxAbAqRu1KiiL9cvdDx3l6OglxM0ruCTFEuIX4nYqZk\\\\\\\/29OyQxPMUd4Fw8MpyTKyywzI3xbtfnCxh5YkrWGXbnAzCl4JL6geh5BYNrrmJnq\\\\\\\/mnwSOibfak2kXZxaFYByhr\\\\\\\/s5fhXT4Oo6MQcXP\\\\\\\/tuBVk0l3k21l92YBOk2hPNWoPWScuXrUtAt0bZsLGTW35vGX8voKr5QnrOtgW9NlW9XMVRokytta9gpxEdL93GiD4hniCPDgoBj7Z22J59RALd4K1SxGLHIHPthjwNqIgMbGGqookcEo2QyFYv6tjZySYS1Ww7vgZjkHq2JungBJ\\\\\\\/OV+uildCvSDfMO6LbiNSRSNW\\\\\\\/dF+jqWQ8clEjnsAX9y+WsupN6Czrs4MTTXnlC5Bs1mM+3U0ZHU3PLHLfXirzXw4vCKKVLwkeSkFXcjHm2lVDBJfjLDai1gExyi51w3v7V4rwJwShlJBDzh4sdR0weWeqV28vo6qB\\\\\\\/TfHFcZkfdQfnFSCPmiWOYXGJgcs4zT2M8af64GLdJbCPxc+SweZXkBUsqVwz1ELD\\\\\\\/RBrpJ0Vbv03z526EgUq\\\\\\\/Y1JtmhY8C84i0m+48yMx+UOmgY12LHKYbc\\\\\\\/hu4d7OWWcKRefYnd++itf2FFx3wUJ1G9a9oDSnosQjfK1AMtU5vBL\\\\\\\/T3mT6wi2iYhnT0t+8RPacbSzm9uVTMabdtk+eLFiqTYCSszL9GGKsSPVR4Sr6+VOzFudVOMUlgA6iUrbcMtXFWU0+v4Hhg\\\\\\\/AL5bZ8m7XgjwdvAwKj3gLAcaX\\\\\\\/UKATgJ1522ZMXumOYX4mdvQ3rqb\\\\\\\/FJpJ4C3GY+lmnX2Ql+gm2Gmu0Xjx7gK0NMU9w50LE\\\\\\\/3VoJp8TSiwl5cZl9VvZ2hisR+X4HjclATvXx0fuLGP804VR5mXSf2MqUtRx+8iq\\\\\\\/XALI+fTfLNj2K4r7SVUFwqY1UKM\\\\\\\/6jglHRJhmRVvJ013p7CzLKfzMy1ZdVNTE9BGE\\\\\\\/v2TerVMfBuJSQkdZnfebDMYYslaGzalQtQs+F9a2nGWlyTYzGS7S9kKTMqVmG5heYiL7UxzwmkeyQyu4SBI+lPuxQP293AcXSlCh+A9A\\\\\\\/FG3Mein5mn\\\\\\\/HGSocgnrpT46OUDBDgVKDCVbA0Gm7Komr5TI\\\\\\\/2QRhcdryzLY7HD8fSLHqgvF0dEszhtVV4Ext4pkCr2HDHQiZGFa9rBKotL\\\\\\\/QCuNahqyv4i2CpdGP6lzbDY\\\\\\\/oZiX2f\\\\\\\/dKHV2\\\\\\\/GDvDjRYjq2bClMTPuP7VaYEnZdVyPJuJkRIHr0Iw6tzgKlLAtSM7tsfupA47YjOpqECV2WPvtHO19ebWLHuqkeYg8oQGQV\\\\\\\/FHCjMbzzbqxDHgXDeG472BjKDDRZ42oIbHFIKZWODfVQeSo0J9d7+4X8t0vIZCrHMiXHOQ8t3ulmJ5NYpBJNN6Azu4zUCmLI37eFnQPERkF+Sv7Z8o3DbkdxnF1uoviAEpRIX8kDaCd+WpCqW87omG6idHVooBlr32IDBtN0sWaItCljJ7oLHR4fQMD8HOuTSXW2C96\\\\\\\/bhlIETOhUd477FTlFxJ7jzUdYTBSvAigYuIuRaEVo4KEBnQnQC58wJAzO1CXBrBGvCzi2DaolkxQ8ZAnxtc070+HIl0WaDoMesaWGdKSq\\\\\\\/SFrXXzwX6rMAYCGwYl+4YTWvN3JbOHrP\\\\\\\/bdzRWCDaQVPHwbnQriDLTZZAP6eaRQFUn+o9UrDZjoQyy9kQDqg7w0eb10HcHOJZunklOT\\\\\\\/SRlaMcgAtGa1OlZbQQAutFa0iYct7Rrs0Sm7Pew5mS6HGg7nEgnPUXfNX+jrirCnMj+wEC7IcQfhlzeoENWS5iIyKC\\\\\\\/CrQyjJUgZE5AUVGanTTooZNKCkEWZEHJihknESuVbWtuckfNL4FsNEAsEJHnWZnnyJ3eK1xfSS7QuTnOIujIX\\\\\\\/I9Z8K320dhf\\\\\\\/H2pbhQRb4QrDYlEMjOmWHErkXVkR4bhs8o7\\\\\\\/33RTLSADNjeNqpHBFDttYTcaIjz1hzx7rXxYMEG0Czy4IgXNRVvxziHPvYWfKvqi2iHIJuQpOaQZniXw5wj3GAH3mWP9\\\\\\\/VqYuPi9e+0FTy13hB4FUTgQ\\\\\\\/pVzn8H0nUG35Xf\\\\\\\/2\\\\\\\/vZ7UaKr+OED51rtha+UknWLaICiKgGSivY3usx8AxRIkQyEgvriZAs5jmTCHs54eOm4POnLEzWT1edc2\\\\\\\/VFJ2qeb1FNPGmgbrM3R+Y0uL0R328ph02Ro44LkIGVBt1UBdNTnBCoiuNzOvbDntgnopeAFcOZu+B0RnYGl68FNDNcCEPQozJR5Y3+Sj2rrkvrsjcVJEn+dL0tr4VgQ\\\\\\\/UmLXG+D720SulFPJgxi3Og+fi7OFVMKA8p0isOLi6UbWa2MGr0kzCu0+\\\\\\\/4G\\\\\\\/3dN+lpIOddTD7ud5jYGtVsaKByDDCX\\\\\\\/hIhTwsk6ZeUw1+i22C5A+Rt1YP2SFdLpdq+rdE2pMF4v\\\\\\\/MP2lyv\\\\\\\/a6uI4iomFf80LjaW9yKWgjwv0Dn05H8yHREeWV2R+fyMUsDhzNz99e0gH3rqK2fDZAejpLAGQLgOju+bn2NqxeoK8iKYn6SZtKYMaglMlu7FcIZlnERGZZDILFWtUdMIF1\\\\\\\/J0QzYOLSRoSADOuk00MhTtv6eYWQKZkk73QGQ5OFrowOfWQjPQTrwHeuk6pVXevV7L3e+35PGhcC4IqffqlHUU94Wne1Wcw0RjwJdPVCBguJXSyQQj9q8eBsMLiqdkcz93W0lL+TNYmZp+Nb7RIzJ+QCkLS1jBuaUwGj4kix\\\\\\\/d00+J7RIyb9TCam+pLqWLBQ1TbGMaIjsn5nqVleaeBCJQw2ekW9mQB98TAn9nx6R8ARYl9h6gpXz\\\\\\\/oxSuPJ0zab4ImKDA2TlFzHXKGD6cLP\\\\\\\/ojrX+sOyUhcXRgQkNY2k0vTvo69VjvvAUUEMyJMYGTWZOe\\\\\\\/ufIiGk11\\\\\\\/YOagkXN2HFChpHvcxPu3MAvJ4lyWXQg\\\\\\\/OpMqXkZTUSReO41L\\\\\\\/t0oZVfa5mzLxlEhwLgD1gC0CSAYRuRMgvgYETrRsym2Rz17XVD7ebO9I6HtmQjP\\\\\\\/bWouGqP7ZOWVG8WbEF33M1sQvxN9HDYA4AoCeStvMIv45ylKp9eooj9kwBE7ranjM5n8hzwaI8SHKzmQ96Ir4z6UlVoSOblHKugDgtwNoWp\\\\\\\/yo3G3WbbAkPi5kYGM5GcueJJ75w5DMaIYEuWcUClOAHph12R8bjTlY4hp3B4VCY05f+BbNhf2wKL5ELEhaG3ykyMCQf+vFAmr8SQnXzFXSbmlaY4cgVB9yF6Oc0oYQLQQaF4lolYqglrINn4h5N9w9iR9MKVeiJOxCzv1pLR+VRf8YNajTFCtYWYyr4D9n4Fhp7qMnhbosIU44cqaSyaNGAaR0sDgwIvppRG+D0dTiEufryRm2lXv1MQ2ReKDBQht6S8beW2G7kfo+kPUJyBKbSE9nLAN7\\\\\\\/O3kkLQvSSErRJp2KB2exMaPiJMSNj7HIsVPVPtaBDiI9zxenreq5A70HewTURbu3uhUE16H8ZKYLT6GSgnleENbzK6\\\\\\\/rvJ9KRfSw7LrtiV9EMjG9WWbQPtK+pOkW9eyZ1MjTu4niWFlooTb39TtK8CUdVYPHEQAA3W\\\\\\\/F9wMhrM0fzNosJpCP8ZrDqFwDoepPHMWoC5bHYbJYcZd9Z8aH6M1Fxws+sOlcb7TKdJ+34k70pv1NRxgLKrctJzvMTxKa8b0VaILjTcu3BOpR0dzLH9I+pbqzUR59ujp89kICWz1SzejCKZqSo9i+K1v9FA5wXBYnfzubl7km2Tjw0wCoQee2FgGRLx+twOW4W82Aabx20XMfrKfBJoXFlNq\\\\\\\/cGDNrQUKojM4\\\\\\\/0c2Tn1IF8vaJW4+WWd1sheqiE9HbgOnNxaoJR6eynx1NXAC4Br+4qfT+0MljD3li\\\\\\\/fq5Ni+n7kJRXhr\\\\\\\/gfYIuJ\\\\\\\/ikq18829TarGOP92WTrJplIojUrUgWCrIS71xqhQgGpbmA6bDX+Z45dxfA8sFZyvz4fNTLDXvJrAok5utHg8DiYA26cy3diSq0UEdBgSbfUyDnj4rq8wbMi+3rbMFLZx4mfnFWKFrovnnsRO4KDi3WfpDwn3xt14RoIJvKyjULcNsIPpz6xk\\\\\\\/hDy9gQIra+C2s1dKVwH+MCJPwDEiSpjfAajHWA0A1prKZKlBVzxm+SQHQni2spMKcS4dCjKao4f10T1UobdQzBjtYAA4GINcZMqBY90BttfSC\\\\\\\/q+bit3e0TAhEIXxfybrV5SNNPzHvl0WSi3DmvHq+i8Zhkzccc\\\\\\\/zFq65gtggwKQHS74f9CCbpaTN3SrUTLEurEjWYikRiTktM4eQpaW9RhuhVV3mDS2W06m8tJ8g\\\\\\\/8w9K4p7WyWRJDBfID5BGMeDOlI5GPndpbs7zEN6nvPkJVPfhnKFdbm7IpPSDxES0iFWFcZMYKxlY3nrhATGJK07n9MO\\\\\\\/S0YCoQFEIVxxHm46wh19lHrTiXMMvMVMiM36UBAFmDUDBbS+yjaeK4dhVetFJSbATW5W8HidfVtGg4rmfNNwi2jJwMKb7MwmoCR\\\\\\\/6Ui3keP6o7Gg0TtS1UT2ELyAyAR+X5cQOhK1u+4H46OOKfIurR4hqyCvnPsQwFCn3iokmZbiwb9+xKHf098pJQ8vWw3KsN0zoKv6+oEb\\\\\\\/J3lZWa4Frjv4g9sI8iPXbs0nQSKsPVpaOZy2Cm9ShqZAOslbxyICt\\\\\\\/EWCIyKzpnicpz8X8IS9Ml8gw7e1zxjH8C06cQdK40XRh19BMPJx4nSxm1OnyfNuVhQFq6Dsv\\\\\\\/LLs2aSzoJzY\\\\\\\/VUa+ixHSsaRwCqMDgoeNVFzqSbMWmcaGIQ1y6v63MDgpIUf1Fk6fKt0QrbukMZhxFRi0h9hIDHUnia+EmxRSKnQWt9fvKGLtXL5aLQN8yiPdKMPwMQGi0OY2eCY+5IlqGwSjvSfht\\\\\\\/9hzt7lB+bV5MaH9ZprGHs0bngkwUaS+sxIRWPMP0XAEwCucBUqjYPTzUsdFpAX7+aK2gSHjKZ4HsZEKIZlv9HqU3pqVhEMstknXP8rYgmUa1O8N\\\\\\\/HbX3rdgp1DFK9IAQyKjAUa0enIBj0cQKn0GbUI5\\\\\\\/68PWv1s0HXOjQTxVjDoclEMIi1qmW4JLdl27AWVr7UIgt\\\\\\\/uATi\\\\\\\/Co7oBk+i1nM2tSIVKrBbFMEQ80EQaV2SCvd2ZchcMtDw03\\\\\\\/8XOzJiR9bgRb5WSt\\\\\\\/I1cwzbpeOjc\\\\\\\/d4EjsAnyKV8amh98BN3NkxkbfeqXoNHE9lZe2qVv5BQdSLNPhuVeOJ8ZG0fvq03A2TRQd\\\\\\\/yiEBWRKGIsV4u6wh4eReFseX5BJ2R34OuC1dXcPwqnqdleUlkT2ULsIYL5BIgaUXkaRJC4AV5xE5U\\\\\\\/Nf3PzHHy5Uv9jSJjapukCD1RvSeo8zU7xFyEqhbNk\\\\\\\/YYzaHftDotPyESvf7WTtohZ6ArXH3tthuA1p4JNhAjqUejvYH\\\\\\\/Fvrd8EBW1rWI+9F339\\\\\\\/4oTzmVE1ehiarjBjloHLRMpYTUXEegquqgZv2JXH9kkFZs+EhakTDBDUyJoJukWuevNs4LVI8l+hzejAIn\\\\\\\/il0tICUGtD7bDFeq26Xf7DEv\\\\\\\/NE6n+Np\\\\\\\/6eyjamklBtxpc84yvvIKB\\\\\\\/r8JEyL1+RHa1+wNGieX+LgRkEFROoJCynwr8HG4hQffVg2Cp6oGJPlmUbx8DGm6sVbvcq86\\\\\\\/O7E48Q5AdlnA0VLheCHFGiOUXMz2lNe\\\\\\\/jnfgdg5rsOPlQFp3bVNe0GfDx6Ljuf9UsnGMqNLnqkUrtRfCbWxbfUZEnXXTlYYFOLLS0S+1lhOUapMwAADhRtWD30dQGLVya4sv0O5lQBQlJKckNjB2QoX4n3t1QMXEZQORaEtYt7keYsAt5\\\\\\\/IGx5r0FfnswYk2pwooh79Nv5SNMnCYltkO3\\\\\\\/YLKGKjzFpqQsYUWOWO5iMYnr\\\\\\\/Xfv\\\\\\\/WWMlCnYQAIDTPR63m2nTOHTuG\\\\\\\/4h9mxTTBD9HOEhzSCjJlu\\\\\\\/F6Ej7cVRUTbnBfjKFkFanWj37W9DPl0lugLHae6kF1qTU7ab8rsUqwd+mk+ESunWLBc9B+D30yudBue3tar+S7NHfoomEe+H6JYVucB\\\\\\\/XM0u1fq1otY4mvp3\\\\\\\/LUSmZTa96ugIFWV8jYxi27KiHmxum9WF11HmIiUlFjw1qsQCDuIfAWvl4462TrY+MGsHclQBiV6Z80kmVgSY58mLpBmFtDJpjsRt9xcMZZKZkRqvjglfq+26Ovz90RyU\\\\\\\/DaXMHWpjZ1fQ1I+YojliLLy\\\\\\\/yGJeR3ZVpdFi+EzbnBMvYE2ygTqq7cG7HeaqelawAHxrKxkPpxiPLIe48ZY1nthk6smZXWXRPljpk5l7diFlqLtpXQkJOfnRUqyxPjJh+NUbN3izlJ4nGynHqXe6Gkb9pHUfrwIujFm0rgIzL9SgI2yxKUC+duD6iG2H7i699e+kDX5Uvi\\\\\\\/VlzvCrQrx\\\\\\\/3nvdq2xyW9U8XixSNPppXj5UmH+CwEPSM6DkG0xY7Ofh9RNCl+QFMVpWG3QyLyTg+VlnR+rpBO7u\\\\\\\/dVYImv8txicL9rEFINYe\\\\\\\/bUVNFShYhCPj4TKRmQsaB5Fdat1xE+AiwMkMbqUKRz5Va9cAZdzUBLoTNICLJPPTO9fJChqZ+up+L0kuXnlkLkR4aOWAFnZbf91Rx5Qy9nzOJcJrzNJhcP0Bl9tyQ64wkUMyTD1VYfdamBOpF7ZmcSRqYa8IVmvvIAYtAhcyFbtZbTyX71odfgvpqe7+k9AHPRqYhtrp0d7G3wkjTc5wELF9ySkD7WNezLpy6h+I41JoON27agQLINg4mUjmdpvgnZTYuTq3Kgwn7I+nscUNYBtwy2gvDIkoTOeJQWtkHL9QCV4KBBYUHkFHtiKOZzLgONyrUrrgmuvYay2mAnEWKLIsXBa75BmTDjIRF6xd9\\\\\\\/+lw\\\\\\\/jBJET4rdeHugfGLfvWRlGNU3rLRXmdfK0B+f38Z8b7y99w0zV03l4m1EaEqSb+MlU2f46rM9H62gTxOKh8eLHqnIiKMo4AU\\\\\\\/YY3WlUfNMyl3V7vcpyeMAQDV\\\\\\\/aq1O\\\\\\\/38CvvlExc+AlUt4KNCZk8LWuMB7gzER93TNEhIEZMfhhhCzaNaRxT5AC4wDeBY1s2ujbyhm7l0LVDNsy7fG3fcMY7nDQx3pTcOdrG+pM9tqS4CuEVAIB6KoncNmUOLoTdGhnM7zVgZL9vDkOLfKvDv5jJT12RgYQQ\\\\\\\/Koh51lcExrNAZ2iWsMGK0TFERGFEScfsZtVhqqsa7FbSvHR4Ra1Vm\\\\\\\/ECtRbOnCLW+\\\\\\\/u6uvami0gjDoBJyWsbS93x3X+BuvYpzFvEvsNB22uCA+yEr9TIPBCe2I3D1pj8RgjdUNeb9U+6pPy7sc7alakKTKTWniLWA14qSGpIbzjqBwYEGebVD3VvSkcygW4lQ5Il00evEw50\\\\\\\/wweADUljGOuaNRFp60MAUT4iDvLii0kbbcYZzMCQP2eMYclWkII85salHHibwX+EzqxE1BEQhazCVPA6ZEyY09vxHMwtC0c3wbwrvGzSwHLkTEfX7XLxmiVmKUt12ILFVYBQ+Vbja4F63WMOEMzFE58cSktan1ttgD\\\\\\\/1jKPQqOZabHqc0SUpSBgr+V2WiiInOouJcI9a83yimW8ZQjsPB\\\\\\\/d\\\\\\\/gatC8ddk6ic1kXR6sC\\\\\\\/BUlH+pIU\\\\\\\/kQWx7dISzQmJ8TmqwTqTJwrRWAYarTsTc0VFQqBtshPJZELRQDB2VioQSWL8R\\\\\\\/GqKeSZ4f3zgPHpRhNvgVGpoIzGaCkLVzDpRqNdYadYiVUiHuil+55wwHfvVcVgThfmynfDmjZ9Gz9cjtRIU4uWhilYgQEZfqiBaJUISmU94e5Q\\\\\\\/Z1RZAwEVTmzNdG3gkYd4B3ZFe1zsY1Oj2LrkbDGEc8pxmUus\\\\\\\/1OOnZMRojdLRIxoOcgRP0N39VIj2lLVbPO3\\\\\\\/Y+EW6oN1aZ6x2b3YKTVY+PjpZrBiD5LAT4YJ4L8b\\\\\\\/I38w\\\\\\\/Bva8g8szYFHoU9h6g9ChKNzZntgZlj1nD5TyXI9Qh7ZJV+3yHuj2LmJc68a3nUyJiowIOQUMZn0X0UM5z6bMdqndLvp00BQ0ajxWPKt2kjRvXuSTwGORXowXe6uaihiNTGTLWFUUFwIS+ER80GIMSORtxNwLfMB2Jb2OcRml1s+1qWbzYyWOW9\\\\\\\/7T74hjx1iiF3mxSHohKsHP\\\\\\\/VodY6c1H\\\\\\\/sQyok1BivxI5n\\\\\\\/1exrF46I7KXIDeoU6tFerldl+BStZCDt50Ku8wC0VLcFoB8yfe0o22D2PPL99rd81UyelK3oB1eUJu9ORuYhMLDUuumzNosxwNHwOHuw5mmX4ropjGUd9nQjUpd3O7SFSGKFx9MTTDg7iohT51amGSMlTgr80t8hkKc9QJujpRuvxjkX4iAmnfEquINIMfohc\\\\\\\/cCXzGcDzvYnfmVAQzd2A0CLUwM6hubiHVHzyxPRcl7hbQyPDRWL03tJA5dHWw3Tzue5XNOVL7FyKmOZ7aeGn4Ux7NOYA3RqHxtwpYH82S2y5xKuWM\\\\\\\/cGrjYxdYb9jtTtiTD823EuiqlBEJzMLskzTfeicZpfACPD8Cfftq4p8x7aV3cAWoNPc6AHi8JyiMnyvwFSQHTXMSE0iNKxpqCD5MAb+HYsj5+RN0j31NvCG8juCEufjRxog9YTs4eLkwuwIfQjv+6iCP++AsXI5hzDoFYvDFr8GF5CpM2eQCjhXeEgcRRKl52Fb6poD\\\\\\\/JDXGy8GL3Nvm\\\\\\\/ootD4nHQ4XfXRCS9uiT+t2zTWA5Vr6unNe9\\\\\\\/8OO6P8j8gacPoXtnBDCuIsH\\\\\\\/4NTBUYMu6Bpz5TGU9Fme7IQJOPqQLtenJL698KcdWR+OIm8AcaqQIEtTpaE49NIomneztnxQj6fwCKqRaNeW0Unrn66xU88PTs5VIom4pr83P9Ida476jP62UKlV+ykGSAA4JTcSjbGvx2NfWSeK2Ain0XYxC1DEvF920PkEB7f11+Su16ULo+27qu5glTZTTiF7ppIcsh0UldTDDZprulG+Hy\\\\\\\/WuAQi0UhN071R9hhJA8fSiwMqK7rqiv2i9IKSu3XdvkK0WuFqaVicxjo0+7mJW061O+91+k0Y1\\\\\\\/K63KXY57JQ1thrhtUYSr0a4mo+kSkAjrkCE4rBWDy+Y08GdBXk5b1GX\\\\\\\/+WGcSNyWBF3JLNxDnSQP7CVMTt2Lxg1P4Mzx4my+bKuszvxm5g4foMgRn3luBUYyqsjJIWZJIU50YjUoF4N8QfXQPWiukpfdd+TL4T8LB5J8hEQEkwbuy59GH0D0I\\\\\\\/skaNYK1Pb0YUrNHlo5jVaB4X8BZQjDHWqHl+sZvdTAW7KIYFmxsbwaUwnQwKiaCKlhp1ZphDDeGqhvpx5eNqz3sgiWswncoVXwC3\\\\\\\/FhktEClIAJT9RC\\\\\\\/dPLnv1dIZ7B2J3+laQQsAIs4SgQCKs+KkrW7Q0cF4PidOQNlpZsATisnTdabebqdor3lMXnV\\\\\\\/BkH\\\\\\\/VjaqXWktWQ46+wYVecpnz03LO8czPJx0W7ADhwmn6kWdRGjoLTDOTVWyYMrLGyYlDY+1lLtVCt0RiBzK3NeFA+gT2tf\\\\\\\/W5pE9zJ+8nq5gD8YoRBGz5GVPVhUMRVlA+LKn7K82YxSaXdFGYzFw0geRgAVCz2YFcqoVRMKwMiK+ucMah1WsPQibGtXw5DDbaD80YVJg1n9o9p6\\\\\\\/T2\\\\\\\/6EPAKK8yEFL2fKg7kETk8TNQnBzfmZR+xsRz62BlOWycVUp+9BiMVDXwaLhptERRwaUrm8fRXbk89R9\\\\\\\/A0D3xmncWVga4rl+nvQbswbsIdMnPIiildJaoW0bavEPy1kVt5XEdKkleVwHwBXFr2PcWcd4GDbs8yaGI0ydruVNTVAtqNQ+t489ZL3mjtgBMXqq6VSwwYz29RAx83BN4fPzGX8Jq36EFTZrtZssKK7RWlG68a8\\\\\\\/ZkrC4owEMkwx5jMUvirXoLrGU73+k2RWVU2SsHwa5aet1MhN1rsW86TT94YOjETeIt3\\\\\\\/tiySrTgtdBjhxyryoAhxuxuElgECzSt6XyPOGWQ7pTEN\\\\\\\/VdHdWXM\\\\\\\/89N+vaKLbOMbu0tlWm5kXLOnqJbKpCihJ3oEojTVW6Wc0tSDZyygNDD\\\\\\\/GJpoOvdjrs6AvMu7Clq+KOFpzmH3U4fFlm8Trq2Cnhe+n\\\\\\\/wLCtUVr8toYZMs0fcybUpnY2NK+ihn7ZxH0F4KaGv64SWSjb6VqneryTL4cqXlCOQXYFW+0Mm4AluXRUKIKByrMquTx5XlDJKAU3PvfyOw3UO4gGsYeCcUR3Km44uLCmihv6F4nkDbEjcBUrdCTni3TAwbG6RxoreqG\\\\\\\/NV\\\\\\\/iszq+aEhR5\\\\\\\/eET7Z580TmEpwAOPvWTyJu3jHs19Gt2j8Nxn+jlw3KAVOs0cPllRkatEM3U6jRspF2AfFKIvGBqvggC2lZX8n5wCGcWmDmKYCrFFxlz2azawZBUa2dsjZdg4gcJlgvj5acTlbS3lAeKyV6FmJhz30qt+YczYGLfHpL5SbNem08gBZ7T9ewO984nujCWFwkgbOVTZLQvTt88EnbGfjnddHgaCFApBgeYQRqDd6i1Q4cJmSU3EfVB\\\\\\\/pTpqKodSjsNXIACLiM5d0LK743PMh0XyjkP2uwxSxGjVaEyiJr1XKF7jaUkCKqdtoQW36DeBKJvMzTaCwisVxXMUUtlhwFRI5Zt8LPCMnsc4dZiXNNvhXwBDjwwjRHsqc4gJqOicNq6hgwTnpR+tFUDclFk7Zw+ImMzqF37COgAsBqnj53bldUBReQM7dgLEv8+LzaGdUUcQ7Z4kf64P3f0WVART\\\\\\\/eaGaomsGnw3Nxy0nfUAX70syC3D\\\\\\\/daxKmz99UiPyFFrQ1yn9pVc\\\\\\\/6+O8y6TngtFWYxPiff5KH35FfbENyLCi4uvLitOhBDHN+y0Cox\\\\\\\/bSpJ5Hqd1GQkNov3O+lcFS4+ZG16m74GyjnF3vhNVkPaVWl+MceJQ3JNBV+ekpYeiE3v7gY2F6YgRyrdpXAOdT1VCOPSVHEyn55yu7GFlXTzrXNKngDfKFHwcTWMRQGg5ny6UHCoBqh74Q+5jUbGlfc9oy+oPXgul2PVE14PoSssSdbTTSLU+zhN\\\\\\\/y5u6SqZn+NHJ98EC\\\\\\\/+RhBKH6qMgl69gy6S7Y3yCtgCAk7\\\\\\\/kInCCeAaUI0iH1+tb8T\\\\\\\/cgBThR6RgqWCzaalXVou5bCYfcVi+TuU1UTFulzfahrkxCmcLUiUgYOQt+H4FjRbM3IZVIqgwbkWNgOFftHGTr2H4z0I5uOmZoPg2FE5zHyLdwVqgX1LQ9XX0yoMhRY9HmKQndMANDq9S5Q1fETRb195ryJ59sUPle2\\\\\\\/tLGE\\\\\\\/RcFGSyZ684ruQQH430k2RcilG9LVicSSfzEUtzYWikslR+IJJ\\\\\\\/EtArNzVC1BTFEOE1JDjCqGnxdgxgEr1xeZ3g7N63QfEaztcruJJJTh3oqNRXss8IEu3bEOC1U5wwUBrtpsixVQ8e9+Hp3x7aecdMtmR2oWTWm\\\\\\\/svX0XSo6QtLicdT1sjYvbcAoL2liVIH+sS0l3AGR5N6u9XqghxF6TAGWLS3TG+KAwysONd5EyaO+WFwi5gF0EIxUj2wYrXDcEXbnDCbhyylr2+18c62ZaFkEwBtTEmEDVJplMuEXhbSYjYUFJNdNL0ViaYsrqQHHn0XBOJ4eWgMOxWZ7hMYfG22F68uuZZ6IhCZ\\\\\\\/0f8EkNGoQfEzCcWjI1Erl8kOmMb4DGOC8lC1PyQp1OVWnQmZjxeXF3\\\\\\\/iA3Cwd6hlLN+9pN9OLiTnd67fxP8Ce\\\\\\\/6Dn6lz15fBg6i7e8pIjEoa+4F8UOPE4CzJcDNY8CoJ2Ut0+z6TEEP\\\\\\\/CtMbx5ikTZn1jJb0P\\\\\\\/f6WE6s01s+Pn4Jg4UAb38Do7cLRvOTNWe\\\\\\\/6SztOBbQ\\\\\\\/mgIoxoAwty6XkuOTVNlu9LO5OnjE0jsdU7V5\\\\\\\/1GM9iLym1tFvqZN+HvCjILxebJE2sljhvuTYaFhh1it4wY6PzSWvQ9EH7QJX2Nm3bn3\\\\\\\/a\\\\\\\/BMS4WRwebSz6m6zyg1P0ZP4\\\\\\\/CkePmivm4XivVCK2V4WCtTOTBP1foy\\\\\\\/3p9S56LN4gE4a5Mz3ufi69GHCaPOCjtdSS0pdym53inYI4axWLbU8mQkOFv5N2nMhNFl0o5kERsm8v6I36O+7opfMFoCKkWL0hgs8b8eQl6AGxsrMNUVElTLeL2EfvDAefoqBAZIQIKZZWtD+HuvBEkBvaZ+42zvZTaBlM6+Y66w2GOG3saM8yPWM5ApdSqLpodv0R4ZFk9CfqQrBme7CmRuqdmQI+OAF9AO2iU6QU3nCN83Pkz08dT+fWUhKVuWOKb++h8OyCLuXnHKZPPTfiax3QWNHgxAKSBOAHS9wKnmmtV46e8qJrUFisSMqyiDhklKR1Y+0u2R9dKzkl0sBEAKcPRvjQ70weRXknN88oCX6xxF4psp9rnrb+4X8QYIP8Y6zGJ+ZBJE5dJQgnzoE+mI9I896ycxmkkC4uPdKw+7DV3ItUylTVHf9f3k0Vfw2oiA7cSRUA7vxDCMg1LPkKoN1FTvVIVt+CMCRAARUNXLdGYcRZO+RfbSxxbYHDYFaNWIumTxgnbm7N+VDt02J7260\\\\\\\/wJz8pYH0ov6K8v4SXa2AuKq5DfeZmjDc9cYL\\\\\\\/noSTilq1yaM0CQEcP8deVtszJofPO6JxCOP8XOOLot4FgAuW0SkwE\\\\\\\/QtjdeOrSFzmzhFDpwGyeKQ8SlGCNrhp9GvTYuwkt1tYnwJTFZ9CyDa0Co31vim6JSoC9iSglP2aGk4iiJuVEy+kr3sdX\\\\\\\/RBqO42j3s+0EL4k1qIjf4vApRx3Ho3iLq9boAVs3dmtAGwTBDeotHlJdX60+PVfqIx4KzOLkXK8TI7vgmjKEq1zcIuSDtzMae5o70AkD4VGudkpCNED6SQ86iGUSfp7+tHiClNLTWjoiUJCIbRsKGufX8tAKtr1qgXT8rX2Z422BuMmQ7ahYfP2Z5bqBDtmBpHBD9Ru34w1yUWfFZaGJtDUWmMp8Pxhd34t\\\\\\\/c47UGVBlSMA9AjVs3fQ8SMmweUc81cqTdENRLZrNuJsPz94wGpulm2+SJXhWw2JgCq40OzzCAvCIwrMQGSIcwgyOJcHx4adCRSVdkucZKssswcdw22GsxKpCXOES2oSe77J6HXFSh72dS0ULU5YE1rYN3jtpgkNmsFQKg48tiTNm469Ow8kiJGq8+VtK2BaQo2bl\\\\\\\/f8sU\\\\\\\/Nan45CB5D8lU\\\\\\\/GFFb+wMC7kD36cBJKRkickn\\\\\\\/0v+7q2c0xJdtlmrwbMpLtqsuSYdT5qiWSkdd77x6U9ssoOG1JS4Id4OUlEjjy\\\\\\\/0Cy5CgvDxH23ptEcSNugjsPavqcZz2eqV0ReXlNvEdsDJ7MvCbmFTLm9vTfMOzY94gwq0LG9dHmeD0mQaXEGVpkUZKEZq03v9i\\\\\\\/2vnTYKklay6jjRUgmF4kXk23MwxV8qFXHM4ptOviiTIjRDhilbnZixx9rge0jnUeS+EBRpLRWLNRtejfSW5JOuzIdt72TgKMEpXS0nSgrh90KmNjllAta+Qm7KsAH+RbgdCAiuP2cZ\\\\\\\/bK5OtCDvgqTivITRUzskLoJk5LjuITXcO7trjnu4zwy\\\\\\\/zN3v75Lu3xYc08hNDefWtD08NMO66yP19eC9M1BfADzFl0vCvOzahHIsIFPoEfl4U5tJMgP9C51nsNRa1mP03\\\\\\\/zAAQR4R8tJ6pt2okA7cB0jmrr7K9c5KdaPONFRFH+f7f4DpwzkulcbkQk\\\\\\\/38HqNDXeQtvj\\\\\\\/avYmuy8x0P70a4lcMeCuir9+XiI3AtlqDzRWR\\\\\\\/02WG31db5oQ5wdmuppCCPQkCVHWqKeb2YEaTsCyPTONw+MkOFnyw2Wt53r1UzvTsRZIJiQCtWhmX7QDmkIIF9YC7KXxq0EcSb3rPp54taiGLWwXGE1MYMo+KEP5O5Ph+TCg8UVAnctG1NB599hTQPLT0YHMmWLdtYeCqjITtGSoZJuIZUafw07XQJ+MsGRMmvUUbOvZRe8HUafacv2PBvAZYgc4\\\\\\\/nWOF8QOc3OI5H4YKAcwMQygVtRtcmv7BuJadF8x6FlDz0hc+SX5op7dURZFw\\\\\\\/9KHX+6\\\\\\\/s+EhFZE7RL6zUI0tKJTMMH2owgB2Xt5qIG5Nrb+OO+Q5aXRpgxvByg\\\\\\\/v\\\\\\\/P18OkevND8z085ECqsUhhnn2UdKluPwTaQmXirpoWX7cSv\\\\\\\/Imqhai486raXH7Pzs7sbk6rcEcbp639VPdbYIbKTcgnvxAO\\\\\\\/2T2teWY5hgHijmKgdXehti16u9ailH2F7Uboq4qZWkYsuSvFy\\\\\\\/ma98KMX3veOu\\\\\\\/BV2n1x2e8fwfQWrf1zeQ4P3goG09BE6r7DrE3RqW6cag4dDXFomX+3iySRJj8bA7xKGJj7E\\\\\\\/cJf\\\\\\\/AFQcoIP6lrO7HJHBlydGONlUyJL7RHGf0JdZ3X8ENDmWaxNa+z0z3M8TiOiZvts1rNMwV3DN9IqfTdgH8Erduy88Z+CC9cSAtiHoXH3J+aqvMu0LOKjgKK3lNDdWiL9p6JV8\\\\\\\/pWcKm27ipseHRSO7D1coJIVQ+VfSTS\\\\\\\/a3G9WeYq2kfqXYeLa4I3QynIjydXQ2OLDli0oML4L600LZY4x5lGcVLJeyXpSBfm9a5uAhFfHVRe\\\\\\\/UTw\\\\\\\/zyd\\\u0022,\\\u0022iv\\\u0022:\\\u0022uLf6RZK06kek0VxnWaVX8A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a85c297a116078cfe6cab7b76eff5c390f67f40e77758b98544d9e9370b30b3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r3df2l4kYsewxG+69XcpIQX\\\\\\\/up7IftM46d7G5ib2Hjm8Lm0AfuRD98AbpyhcVYeGIexcorUAbw7uW1aqGDZKF6qUs6OLtu0V7x634g\\\\\\\/fA1nW9tL6E16kh+6ND285ChxJnGbExtQI38mgB\\\\\\\/mp2u2+I\\\\\\\/MrqAy43bcZptywFgUyTEeZ6F3daWOIRH8ow90\\\\\\\/sfiWPWp+CpdpfJfl7cJEQxGVVSGzD24Fm1q0ovf7fvtz5eiALGANWOMCBn2Mx58Qd1t5e6lqZTEppgwAC3B8tlbHn\\\\\\\/jTCaSeL5AMd1sarOg4A5TuSQfgiiSP57zc+\\\\\\\/Wa3LAEjw8+Q5BbweA5OMqcbBsvk3vDgfpzorf+rRTpbIXq9ppd4cNlbKExbp1j6Fg\\\\\\\/zVRs\\\\\\\/BZVd3GX\\\\\\\/Ws9FAYOhsEpMMizbpUXhBLplRe57umcGQ2F1221jE4+FY8y0FpKhVNhToLcsEEvr3BfwXpmE8KKh4myL9nnsewk3y8+p6ovmqYAUVfnw82OPBMgoSwzzBMOtMkz4fpW61547PMffKfko1GoH8LWGJj9uL4zbPe8grybcIv3AgRFYPuv+A+E0CCEXOhhK3F3oY9RzQHuWOHPii4LXdGBnx3LJZLG9t90UCUrpyUcP4vLoO8E+4PF1H\\\\\\\/Bb4+lFGvxUvin26IwPw1PvrzC6NDlpG9wpe3q4jTnpNjBF6wv7JmUUJLLH2Q3n68YU\\\\\\\/jPMGaikzMpbvJ3OWVIvCb0gX1vD4aTnC1D1UgQgcTcBX8n2rqCVRZJtFMFJ3TSZTY2B2gx3S3mjyYHWTDrtce6czmVxppQyvEd2MffV+SA7uS\\\\\\\/KZYldi4Ty8+DL\\\\\\\/qGmFwtkcUkilLrBRh56SKJMrhezBlKVE6J09nTBXl\\\\\\\/fZ1LpiqMjo2F06eq6b6nvQ3ZJKeIultyg4sfhngq3JkVu1LX\\\\\\\/HdrSedZ2+65\\\\\\\/gbqsuVQuxnfblb6uUIN0QFZJ7hJbJP2EyLQJQklp5YFqkAgnauPM4GgDa9H6qo0UtiBQgDxKsadEwDQwPK6YmnahIm08kFc5\\\\\\\/y8rnJTgF4638izrUpMQad6d+Sxe1naFa6gon7WEiGg3DIF2SwU4t5FO2dND\\\\\\\/qaH3wbq0hydeW1S\\\\\\\/Xsf98HjtGNHI2klaR95P7MJYfwL+47sL99P5qP8ok+RCULs9DXz9aGdlJkTvpPUqgRoj4+ANKsMhiTXy2dLyUOONOgxxABhN4t08mlql7HQilAg+h4lJKm36aOVvpckXZIGtXZrcJdNRzMMrLzmFH7RCHyx9icZpixlsjgew2HppZ5xPNKjkXUBbuAr\\\\\\\/nPCViH33xRYcCdtKpdwSW84AexbZbjFmo9rT04lbY2M4HrQWe52Vd6hz6lqkDkhaFQrfZxa3lzrRgmHz08EEyF5YClHJaWdJi+hrWSB25fRlZ81YDMOONBiHh2SvXSVZ8CEj5aZNIQxWHNIEt9\\\\\\\/sMHCqFShjzc\\\\\\\/VNglLYUISCTGgwoyvlh6bEJ56oh7BOkAwY45U7zxlxZXhiSTo0P0pmFc6eSUHqQIPRVKFc+2lrvXEmMf2gSqvP2Hb2tXgvEzipJt0jmTS9UtPiShehAiUMgbWmlnvxKdV2MGg\\\\\\\/zzzTyiq4f+o+236j99Qys2gEuRG9VzjdWe2mEX0KKzx29T53Xm7uD3tYL9es+aJ6saAR6elvLgl\\\\\\\/7dsOFBKJqfB47tsx1RRWkVsruftdSJ5BaT+p\\\\\\\/b+V8yvHSRW0jVWYCCSp9PF7e8RarapsvRHyhndg3vLKjdhG8sbo+dygxUSgQqLzL6V5q5WCvvphGbdk5yruq4zQ5aU\\\\\\\/HcNbW3Z02Te6lQARkiGJIv83TpNFlEJRSElkHhZ2Ex5VbWOLgxdmPxGf3vVaFOEmtNrExQjyqcmK8zEkHsQgChEYyyOfUD0tR35D52Ut+3r8tyC7I5X2RNulO5onpakhJeosIsDs3W2EVnM7GhdrsPaUxTTczPOaSuXygDxpufj5UdOXoSeCc2igbgUM9HtzRF9VwCpSVl4jaYbEYsLUfCp9DUxhJlEPkIEk7vv7Ac\\\\\\\/Q=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Nk39gUzAIvTIynQcjck75w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002214a82174ca1d1036e36b16fd93a159ad7264b257dd80d264437e57c31fe28422\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487482 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3849" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-T7d1oNbKlVrrGuY4PaXWiU6mHWgUYNqtbTh2Nl_N9iU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3849" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-re-9CIPAfrMZfv9RCepPfXxII0tnDa4x4AcQsQWD1FI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/3839" title="ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/3857" title="Dissecting Nucleosome Function with a Comprehensive Histone H2A and H2B Mutant Library" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487482" id="node487482" data-pisa="ggg;7/12/3849" data-pisa-master="ggg;g3.117.300120" data-apath="/ggg/7/12/3849.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">rSalvador: An R Package for the Fluctuation Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qi Zheng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3849-3856;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487482" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qi Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Epidemiology and Biostatistics, Texas A&M School of Public Health, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qi%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQi%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:qzheng@sph.tamhsc.edu" class="" data-icon-position="" data-hide-link-title="0">qzheng@sph.tamhsc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/12/3849" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487482" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3849.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487482" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/3849.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487482" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/12/3849.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487482" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The past few years have seen a surge of novel applications of the Luria-Delbrück fluctuation assay protocol in bacterial research. Appropriate analysis of fluctuation assay data often requires computational methods that are unavailable in the popular web tool FALCOR. This paper introduces an R package named rSalvador to bring improvements to the field. The paper focuses on rSalvador’s capabilities to alleviate three kinds of problems found in recent investigations: (i) resorting to partial plating without properly accounting for the effects of partial plating; (ii) conducting attendant fitness assays without incorporating mutants’ relative fitness in subsequent data analysis; and (iii) comparing mutation rates using methods that are in general inapplicable to fluctuation assay data. In addition, the paper touches on rSalvador’s capabilities to estimate sample size and the difficulties related to parameter nonidentifiability.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/luria-delbr%C3%BCck-protocol" class="hw-term hw-article-keyword hw-article-keyword-luria-delbr&#xFC;ck-protocol" rel="nofollow">Luria-Delbrück protocol</a></li><li class="kwd"><a href="/keyword/practical-nonidentifiability" class="hw-term hw-article-keyword hw-article-keyword-practical-nonidentifiability" rel="nofollow">practical nonidentifiability</a></li><li class="kwd"><a href="/keyword/likelihood-ratio-test" class="hw-term hw-article-keyword hw-article-keyword-likelihood-ratio-test" rel="nofollow">likelihood ratio test</a></li></ul><p id="p-3">Nearly 75 yr ago, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Luria and Delbrück (1943)</a> proposed an experimental protocol that today is known variously as the fluctuation test, the fluctuation experiment, and the fluctuation assay. This innovative experimental protocol is so far the preferred method for determining microbial mutation rates in the laboratory. Because computational methods for inferring mutation rates from fluctuation assay data seemed complicated to most biologists, for over half a century the fluctuation experiment had been an investigative tool only for mathematically minded biologists or biologists who could find adequate computational assistance. The advent of SALVADOR (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zheng 2002</a>), a package written in the Wolfram language (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wolfram Research, Inc. 2016</a>), induced a surge of applications of the fluctuation experiment by investigators who otherwise would not have considered the classical protocol. However, because SALVADOR runs in the proprietary Mathematica (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Wolfram Research, Inc. 2016</a>) environment, it raised a new, albeit lesser, barrier to the widespread use of the fluctuation experiment. It was <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hall <em>et al.</em> (2009)</a> who helped bench scientists break free from the shackles of proprietary software. Their web tool, FALCOR, enables investigators to perform basic analysis of fluctuation assay data in a way no more arduous than calculating the sample mean and sample SD. Numerous researchers, empowered by this convenient web tool, used the fluctuation experiment in tandem with DNA sequencing techniques in their research. As a result, an astonishing flood of novel applications of the fluctuation experiment followed. FALCOR was an effective catalyst for a marriage between the classical fluctuation experiment and modern DNA sequencing techniques.</p><p id="p-4">With FALCOR’s popularity came a disturbing irony. As noted in a recent review (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zheng 2015b</a>), FALCOR offers only methods that were reviewed in 2000 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Rosche and Foster 2000</a>). Many applications require methods that either were developed after 2000 or are to be developed. Researchers who acquainted themselves with the analysis of fluctuation assay data via FALCOR were often under the impression that all fluctuation assay data could be analyzed by a standardized approach, which <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hall <em>et al.</em> (2009)</a> advocated. Owing to this mistaken perception, investigators in a large fraction of recent studies adopted a common method to analyze their fluctuation assay data, paying little attention to important features that distinguished their experiments. A conscientious practitioner would then be baffled when <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ycart (2013)</a> sounded a note of warning, in stark contrast to the sanguine views of <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Hall <em>et al.</em> (2009)</a>. The rhetorical question “Can estimates be trusted?”, sparked by a common mathematical assumption about cell life spans, seemed to obliterate bench scientists’ hopes for a practical way to analyze fluctuation assay data on their own. Moreover, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a> voiced their complaints about the extensive numerical problems that were supposedly caused by the maximum likelihood method. Besides clarifying the disquieting confusion prompted by these issues, the present article uses rSalvador 1.7 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zheng 2017a</a>) to guide readers in the analysis of their fluctuation assay data.</p><p id="p-5">rSalvador was first released in April 2014. It was written as an R package, but the most compute-intensive parts, as in SALVADOR, were coded in the C programming language. rSalvador is not merely an R adaptation of SALVADOR, for it affords new methods developed after the last release of SALVADOR. This article describes rSalvador’s capabilities by focusing on three kinds of inadequacies commonly found in recent analyses of fluctuation assay data. The three kinds of inadequacies are: (i) resorting to partial plating without properly accounting for the effects of partial plating; (ii) conducting attendant fitness assays without accounting for mutants’ relative fitness when calculating estimates of mutation rates; and (iii) comparing mutation rates using methods that are in general inapplicable to fluctuation assay data.</p><div class="section other1" id="sec-1"><h2 class="">Relevant Assumptions</h2><p id="p-6">The fluctuation experiment is conceptually simple. Although neither mutations nor mutants are directly observable, a mutant can be made visible by allowing it to form a colony on a solid culture. As a result, analysis of fluctuation assay data revolves around inferring the number of mutations from the number of mutants in a test tube. This challenging task relies critically on mathematical models that bridge the gap between mutations and mutants. Like mathematical models for any other purposes, a mathematical model for fluctuation assay data depends on simplifying assumptions. Acquiring a sound understanding of the key assumptions not only helps experimentalists to better design experiments, but also enables them to analyze their data more confidently. Major assumptions underlying common mathematical models for fluctuation assay data include the following.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-7">A1. Cells (nonmutants and mutant alike) undergo unimpeded growth in a liquid culture (contained in a test tube). In other words, cells are in logarithmic phase (also known as exponential phase).</p></li><li id="list-item-2"><p id="p-8">A2. When a cell divides, the probability is a constant <em>p</em> that one of the two daughter cells is a mutant. The probability is 1 − <em>p</em> that both daughter cells are nonmutants. Therefore, there is no possibility of a nonmutant splitting into two daughter mutants.</p></li><li id="list-item-3"><p id="p-9">A3. Mutants grow at the same rate as nonmutants. As a consequence, the relative fitness is unity.</p></li><li id="list-item-4"><p id="p-10">A4. Back mutation is negligible, and hence is not considered.</p></li><li id="list-item-5"><p id="p-11">A5. Cell death is negligible, and hence is not considered. As <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kendal and Frost (1988)</a> rightly asserted, this assumption is not a problem when dealing with bacterial cells (in contrast to somatic cells). But there is an important exception. When the Luria-Delbrück protocol is used to study antibiotic-induced mutations, as in the study of <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cairns <em>et al.</em> (2017)</a>, cells must grow in the presence of an antibiotic. The dose administered should be sufficiently low so as not to cause considerable death of nonmutants. Otherwise, owing to death of large numbers of nonmutants, unimpeded growth of antibiotic-resistant mutants will grossly inflate the mutant frequency. This caution should not be confused with the use of a lethal dose of an antibiotic after the cells are plated, as a selective agent.</p></li><li id="list-item-6"><p id="p-12">A6. Every mutant exiting prior to plating is capable of growing into a colony after the plating process. In other words, the plating efficiency is perfect.</p></li><li id="list-item-7"><p id="p-13">A7. Cell life span obeys the negative exponential distribution.</p></li><li id="list-item-8"><p id="p-14">A8. The total numbers of nonmutants immediately before plating, denoted by <em>N<sub>t</sub></em>, are approximately the same across tubes within an experiment.</p></li><li id="list-item-9"><p id="p-15">A9. Nonmutants die immediately after plating. In other words, selection after plating is so lethal that postplating mutation is negligible. Note that postplating mutation was of interest to <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lang and Murray (2008)</a>, and to <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ford <em>et al.</em> (2013)</a>.</p></li><li id="list-item-10"><p id="p-16">A10. When a nonmutant undergoes a mutation, the resulting daughter mutant is capable of immediately manifesting the mutation’s phenotype. This assumption thus excludes the possibility of phenotypic delay, a concept that can be traced to <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Newcombe (1948)</a>.</p></li><li id="list-item-11"><p id="p-17">A11. <em>N</em><sub>0</sub>, the number of cells used to seed a test tube, is small enough to guarantee that the <em>N</em><sub>0</sub> seeding cells contain no mutants.</p></li></ul><p id="p-18">A useful mutation model yields a so-called mutant distribution, which describes the probability distribution of the number of mutants, and which usually involves a parameter customarily denoted by <em>m</em>, the expected number of mutations per tube. Once an accurate estimate <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-1.gif"/></noscript></span></span> of <em>m</em> is inferred from fluctuation assay data using an appropriate mutant distribution, the mutation rate is determined by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>For a detailed discussion of the mutation rate <em>p</em>, the reader is referred to <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zheng (2017c)</a>.</p><p id="p-19">Almost by definition, a simplifying assumption is a deviation from biological reality. One is doomed to disappointment when checking any of the above assumptions outside the context of what the fluctuation experiment aims to achieve. As the point of the fluctuation experiment is to estimate the fundamental parameter <em>m</em>, the validity of the assumptions (A1)–(A11) should be assessed by whether they allow acceptable estimates of <em>m</em> to be extracted from fluctuation assay data. Although limited research has been conducted in this regard, the assumptions seem to be less worrisome than originally thought. For example, the assumption (A7), the most contrived at first glance, was recently found to be an acceptable assumption.</p><p id="p-20">A peculiar feature of all mutant distributions is a lack of explicit analytic expressions. A mutant distribution must therefore be identified by its probability generating function (PGF). If <em>Y</em> denotes the random number of mutants, the PGF of <em>Y</em> is the function <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-3.gif"/></noscript></span></span> defined by <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-4.gif"/></noscript></span></span>. rSalvador encompasses five mutation distributions. The first mutant distribution is due to <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lea and Coulson (1949)</a>. This distribution depends on all the above 11 assumptions, and its PGF is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Here, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-6.gif"/></noscript></span></span> is strictly less than unity (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Zheng 2002</a>). But φ is assumed to be unity by convention. Strictly speaking, the Lea-Coulson mutant distribution refers to the special case where the PGF is of the form <span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nádas <em>et al.</em> (1996)</a> were the first to suspect that setting ∅ = 1 could have adverse effects on the estimation of <em>m</em>. However, in practice, ∅ &gt; 0.999 almost always holds true. Under such a condition, the effects of ∅ are negligible. rSalvador is the only software tool that permits the user to specify a value smaller than unity for ∅, mainly for educational purposes.</p><p id="p-21">The second mutant distribution resulted from an attempt to relax the assumption (A3) by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mandelbrot (1974)</a>. Because <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Koch (1982)</a> made a similar attempt to allow for differential growth rates between nonmutants and mutants, the mutant distribution is usually called the Mandelbrot-Koch distribution. The following PGF is a generalization of the PGF <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-8.gif"/></noscript></span></span></p><p id="p-22"><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-9.gif"/></noscript></span>.<span class="disp-formula-label">(4)</span></span>Here <em>B</em> denotes the usual beta function. The parameter <em>w</em> is relative fitness, that is, the ratio of the growth rate of mutants to that of nonmutants.</p><p id="p-23">The third mutant distribution can be traced to <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Armitage (1952)</a>, who was the first to attempt to relax the assumption (A6). The resulting mutant distribution was further studied by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Stewart <em>et al.</em> (1990)</a>. The PGF of that distribution is</p><p id="p-24"><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Here, ξ = <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-11.gif"/></noscript></span></span> / (1− <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-12.gif"/></noscript></span></span>). The parameter ϵ is called the plating efficiency, which denotes the portion of culture plated or the probability that a mutant forms a visible colony after the plating process.</p><p id="p-25">The fourth mutant distribution is a special case of the relatively new <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-13.gif"/></noscript></span></span> distribution (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zheng 2010</a>). The <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-14.gif"/></noscript></span></span> distribution has two parameters, <em>A</em> and <em>k</em>, and is denoted by <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-15.gif"/></noscript></span></span> The PGF of a <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-16.gif"/></noscript></span></span> variable is</p><p id="p-26"><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>The <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-18.gif"/></noscript></span></span> distribution’s usefulness lies in its ability to relax the assumption (A8).</p><p id="p-27">The last mutant distribution derives from Haldane’s mutation model (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Zheng 2005</a>). Because cell growth is synchronous in the Haldane model, cell life spans are nonrandom. This is in violation of the assumption (A7). The Haldane mutant distribution is indexed by the mutation rate <em>p</em> and the number of elapsed cell generations <em>g</em>. rSalvador includes the Haldane model mainly for educational purposes, and it relies on algorithms developed by <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zheng (2007)</a>.</p></div><div class="section other2" id="sec-2"><h2 class="">Classical Analysis</h2><p id="p-28">In most published applications of the fluctuation experiment, inference about <em>m</em> relied on the Lea-Coulson mutant distribution (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Lea and Coulson 1949</a>) defined by the PGF <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-19.gif"/></noscript></span></span> given in Equation (3). The Lea-Coulson distribution owes its popularity partly to the work by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ma <em>et al.</em> (1992)</a>. SALVADOR 1.0 (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Zheng 2002</a>) was the first publicly available software package allowing the user to calculate maximum likelihood (ML) estimates of <em>m</em> under the Lea-Coulson model. The web tool FALCOR (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Hall <em>et al.</em> 2009</a>) rendered this capability more easily accessible to bench scientists. But FALCOR does not automate the process of computing confidence intervals (C.I.s) for <em>m</em>. As a result, the user is forced to compute C.I.s outside FALCOR by following instructions given on FALCOR’s website.</p><p id="p-29">rSalvador has a unique advantage. Since SALVADOR 2.0 (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Zheng 2005</a>), the expected Fisher information has been replaced by the observed Fisher information in all algorithms for computing point and interval estimates of <em>m</em>. As a consequence, all C.I.s produced by SALVADOR (or rSalvador) are based on the likelihood ratio method. <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Efron and Hinkley (1978)</a> were among the first to observe that likelihood ratio C.I.s are preferable to the commonly used Wald-type C.I.s. <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Raue <em>et al.</em> (2009)</a> referred to likelihood ratio C.I.s as finite sample C.I.s, to underscore the superiority of likelihood ratio C.I.s over Wald-type C.I.s. rSalvador (including SALVADOR) is so far the only fluctuation assay tool that advocates likelihood ratio C.I.s. One distinctive feature of the likelihood ratio C.I. for <em>m</em> is that it is usually asymmetric about the ML estimate of <em>m</em>. As an example, consider the well-known Demerec data (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Demerec 1945</a>). The ML point estimate of <em>m</em> is</p><p id="p-30">&gt; newton.LD(demerec.data)</p><p id="p-31">[1] 10.84383</p><p id="p-32">But a 95% likelihood ratio C.I. for <em>m</em> is</p><p id="p-33">&gt; confint.LD(demerec.data)</p><p id="p-34">[1] 8.650538 13.194765</p><p id="p-35">The center of this C.I. is at 10.92, not 10.84—the ML estimate of <em>m</em>. Unlike the Wald-type C.I.s, a likelihood ratio C.I. requires iterative computing. The user can view the iterative process by setting the option show.iter = T in the above command.</p><p id="p-36">As an educational feature, rSalvador allows the user to specify an arbitrary value between 0 and 1 for the parameter φ given in Equation (2). In the Demerec experiment, <em>N</em><sub>0</sub> = 90 and <em>N<sub>t</sub></em> = 1.9 × 10<sup>8</sup> (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Zheng 2002</a>). However, if the experiment were terminated prematurely at <em>N<sub>t</sub></em> = 900, one would have φ = 1 − 90/900 = 0.9. A noticeably different estimate of <em>m</em> would result, as predicted by <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Nádas <em>et al.</em> (1996)</a>.</p><p id="p-37">&gt; newton.LD(demerec.data,phi = 0.9)</p><p id="p-38">[1] 18.92394</p><p id="p-39">The simple Lea-Coulson model relies on each of the assumptions (A1)–(A11), which may belie the model’s practical usefulness. One source of skepticism is the assumption (A7), as cell life spans do not obey the negative exponential distribution. It was this assumption that triggered the rhetorical question “Can estimates be trusted?” In a study of the Haldane model, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Zheng (2007)</a> compared the Lea-Coulson model with the Haldane model and found that the two models produced comparable estimates of mutation rates. This finding suggests that the assumption (A7) is acceptable, because the two models assume drastically different cell life distributions. A simulation study by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Ycart (2013)</a> reached essentially the same conclusion. Still, <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Ycart (2013)</a> suggested using a cell life distribution derived from a particular yeast study as a universal cell life distribution. In response to Ycart’s suggestion, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gillet-Markowska <em>et al.</em> (2015)</a> emphasize that “there is no universal cellular division time model as it depends on experimental conditions like the strain or the media.”</p></div><div class="section other3" id="sec-3"><h2 class="">Accounting for Relative Fitness</h2><p id="p-40">Classical analysis of fluctuation assay data excludes the possibility of differential growth rates between mutants and nonmutants. Recognition of the importance of allowing for this possibility can be traced to <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lieb (1951)</a>, who highlighted the <em>P</em><sub>0</sub> method of <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Luria and Delbrück (1943)</a> for its capability to accommodate differential growth rates. Using the mutant distribution defined by the PGF in Equation (4), <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Zheng (2002)</a> developed algorithms for computing ML estimates of <em>m</em> and <em>w</em>. These algorithms rely on the expected Fisher information that must be computed by truncating infinite series. Because computing infinite series could be prohibitively expensive in practice, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Zheng (2005)</a> improved these algorithms by replacing the expected Fisher information with the observed Fisher information, which not only obviated the problem of infinite series but also made possible the computation of likelihood ratio C.I.s. A comparison between Equations (32) and (46) in <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Zheng (2005)</a> clarifies the distinction between the expected and the observed Fisher information. The elegant work of <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Efron and Hinkley (1978)</a> provides helpful information on this topic.</p><p id="p-41">However, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a> drew renewed attention to the Mandelbrot-Koch model by making the following observation.</p><blockquote id="disp-quote-1" class="disp-quote"><p id="p-42">Indeed, the likelihood and its derivatives can be computed by iterative algorithms: theoretically at least, the problem could be considered solved. This is not so in practice, mainly because the multiple sums that must be computed by the optimization algorithm make it quite unstable. According to the numerous tests that we have made, the ML estimates cannot be reliably computed for samples whose maximum exceeds 1000.</p></blockquote><p id="p-43">Such a disastrous outcome is unreproducible, as it was an artifact of the computer code of Hamon and Ycart. The improved algorithms for computing ML estimates given by <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Zheng (2005)</a> had been thoroughly tested using simulated fluctuation experiments in which the maximum mutant counts far exceeded 5000. A diagnostic clue to the enigma is on p. 1258 of <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a> below an expression for the expected Fisher information, denoted by <em>I</em> (α, ρ).</p><blockquote id="disp-quote-2" class="disp-quote"><p id="p-44">Fortunately the partial sums increase, so that when computing the inverse <em>I<sup>−1</sup></em>(α, ρ) the sum of the first <em>m</em> terms yields conservative confidence intervals; yet we do not consider it satisfactory.</p></blockquote><p id="p-45">Unaware of the advantages of using the observed Fisher information, <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a> still used algorithms reliant on the expected Fisher information to compute ML estimates. This then-outdated practice led to an exaggerated sense of disaster, so Hamon and Ycart recommended the generating function (GF) method as a remedy. They envisioned future experiments in which cultures would contain enormous numbers of mutants to allow the experimentalist to harness the GF method’s potential. This would imply the abandonment of the Luria-Delbrück protocol, as large numbers of mutants must be directly counted, without the aid of plating. Such views caused confusion. For example, a simulation study of the GF method led <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gillet-Markowska <em>et al.</em> (2015)</a> to emphasize the importance of avoiding exceedingly large mutant counts.</p><blockquote id="disp-quote-3" class="disp-quote"><p id="p-46">[T]he precision on the estimation of <em>m</em> (and by consequence the estimation of <em>μ</em>) is higher for the smallest values of <em>m</em>. Therefore, users should not outgrow the cultures in order to limit the number of mutants that grow on selective plates.</p></blockquote><p id="p-47">The exploration of the GF method by <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a> offers valuable lessons, which I here illustrate using the experimental data given in Table 3 of <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Rosche and Foster (2000)</a>. Eight of the 60 mutant counts were excluded in the analysis by <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a>, but I shall use the intact data. One reason for choosing this data set is that it contains the largest mutant count ever reported—3000. At least three versions of the GF method are available: that given by <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a> via the R code attendant to their paper, that given by the web tool <em>bz-rates</em> (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Gillet-Markowska <em>et al.</em> 2015</a>), and that given by the R package flan (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mazoyer <em>et al.</em> 2017</a>). ML estimates of <em>m</em> and <em>w</em>, along with estimates produced by the three versions of the GF method, are displayed in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Judging by the contours of the log-likelihood function, the ML method outperforms all three versions of the GF method. The large mutant count did not derail the ML method, contradicting the observation of <a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012)</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3849/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contours of the log-likelihood function for the data given in Table 3 of Rosche and Foster (2000). The four colored dots represent estimates of m and w. The green dot indicates the ML estimates, the blue dot indicates the GF estimates via flan, the purple dot indicates the GF estimates via bz-rates, and the red dot indicates the GF estimates by an earlier implementation of the GF method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884815788" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Contours of the log-likelihood function for the data given in Table 3 of Rosche and Foster (2000). The four colored dots represent estimates of &lt;em&gt;m&lt;/em&gt; and &lt;em&gt;w&lt;/em&gt;. The green dot indicates the ML estimates, the blue dot indicates the GF estimates via &lt;em&gt;flan&lt;/em&gt;, the purple dot indicates the GF estimates via &lt;em&gt;bz-rates&lt;/em&gt;, and the red dot indicates the GF estimates by an earlier implementation of the GF method.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3849/F1.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/12/3849/F1.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3849/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3849/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487490" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-48" class="first-child">Contours of the log-likelihood function for the data given in Table 3 of <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Rosche and Foster (2000)</a>. The four colored dots represent estimates of <em>m</em> and <em>w</em>. The green dot indicates the ML estimates, the blue dot indicates the GF estimates via <em>flan</em>, the purple dot indicates the GF estimates via <em>bz-rates</em>, and the red dot indicates the GF estimates by an earlier implementation of the GF method.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">&gt; newton.joint.MK(cairns.foster.data)</p><p id="p-50">[1] 1.3027909 0.7281044</p><p id="p-51">Likelihood ratio C.I.s for <em>m</em> and <em>w</em> can be computed separately as follows.</p><p id="p-52">&gt; confint.profile.m(cairns.foster.data)</p><p id="p-53">[1] 0.9855115 1.6749828</p><p id="p-54">&gt; confint.profile.w(cairns.foster.data)</p><p id="p-55">[1] 0.5209636 1.0298620</p><p id="p-56">The exploration of the GF method drew attention to the issue of parameter identifiability. Only recently did investigators in systems biology and related fields recognize practical identifiability as a problem distinct from the better-known problem of structural identifiability (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Raue <em>et al.</em> 2009</a>). Practical identifiability considers the effects of the amount and quality of data. Computational difficulties encountered with the Mandelbrot-Koch mutant distribution may also be viewed as practical nonidentifiability. The inability to estimate <em>w</em> in the examples given by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Mazoyer <em>et al.</em> (2017)</a> is indicative of practical nonidentifiability. In addition, the C.I.s that extend to zero in these examples are a symptom of the same problem. No multi-parameter mutant distribution seems to be immune to practical nonidentifiability. <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Angerer (2001)</a> was perhaps the first to document computational difficulties that were due to practical nonidentifiability. In my own simulation exploration of the mutant distribution used by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Lang and Murray (2008)</a> and by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Ford <em>et al.</em> (2013)</a>, I also found practical nonidentifiability a stubborn problem.</p><p id="p-57">A motivation for studying the GF method was to improve the estimation of the fitness parameter <em>w</em>. Simulation suggests that it is uneconomical (in terms of sample size) to use the Luria-Delbrück protocol as a tool for measuring bacterial fitness. The traditional fitness assay (also known as the competition assay) is a more efficient tool, which has been increasingly conducted in tandem with fluctuation assays in recent studies (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Scanlan <em>et al.</em> 2015</a>). The Mandelbrot-Koch mutant distribution enables the investigator to incorporate information on <em>w</em> in the analysis of fluctuation assay data. For instance, if a fitness assay yields an estimated <em>w</em> of 0.73 in the above example, one obtains an estimate of <em>m</em> by</p><p id="p-58">&gt; newton.MK(cairns.foster.data,w = 0.73)</p><p id="p-59">[1] 1.30215</p><p id="p-60">In the absence of attendant fitness assays, the investigator may use joint ML estimation cautiously as a tool to obtain an estimate of <em>m</em> adjusted for the nuisance parameter <em>w</em>. In doing so, the issue of parameter identifiability should be kept in mind. While the ML approach is unsuitable for unrealistically large mutant counts, it remains the method of choice for real-world applications. An experimentalist is rarely able to count 500 mutant colonies per culture (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boe <em>et al.</em> 1994</a>), let alone 5000 mutant colonies per culture. The experimentalist resorts to partial plating if a poorly designed experiment yields unwieldy mutant counts.</p></div><div class="section other4" id="sec-4"><h2 class="">Accounting for Plating Efficiency</h2><p id="p-61">When an experimentalist plates only a portion of each culture, we refer to the actually plated proportion as the experiment’s plating efficiency, often denoted by ϵ. A culture before plating is densely populated by cells, but mutants constitute only a minute fraction of the cell population. As a result, mutants are sparsely and randomly dispersed in the culture. If 10% (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-20.gif"/></noscript></span></span>) of a culture is plated, in general, the experimentalist does not transfer exactly 10% of the mutants onto a selective culture—the actual number of mutants transferred is a random number. Although a liquid culture resides in a three-dimensional tube, the randomness induced by partial plating is more easily understood by analogy with a slice of raisin bread, as illustrated by <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/3849/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="This diagram draws an analogy between a three-dimensional liquid culture and a slice of raisin bread, to help explain the randomness induced by partial plating. The green points, each symbolizing a mutant, are randomly dispersed in the square area. The two circles indicate two possible ways of sampling (plating) an equal portion of the square area that represents a three-dimensional liquid culture." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884815788" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;This diagram draws an analogy between a three-dimensional liquid culture and a slice of raisin bread, to help explain the randomness induced by partial plating. The green points, each symbolizing a mutant, are randomly dispersed in the square area. The two circles indicate two possible ways of sampling (plating) an equal portion of the square area that represents a three-dimensional liquid culture.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/3849/F2.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/12/3849/F2.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/3849/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/3849/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487544" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-62" class="first-child">This diagram draws an analogy between a three-dimensional liquid culture and a slice of raisin bread, to help explain the randomness induced by partial plating. The green points, each symbolizing a mutant, are randomly dispersed in the square area. The two circles indicate two possible ways of sampling (plating) an equal portion of the square area that represents a three-dimensional liquid culture.</p><div class="sb-div caption-clear"></div></div></div><p id="p-63">Partial plating is common, as it is a practical measure to circumvent overwhelmingly large mutant counts. For example, except for one experiment, all experiments reported by <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Luria and Delbrück (1943)</a> had a plating efficiency smaller than unity. Their experiment 16 is perhaps the best known by biology students. In that experiment a portion of 0.08 ml was plated from each of the twenty 0.2-ml cultures. Hence, the plating efficiency was <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-21.gif"/></noscript></span></span>. The popular genetics textbook of <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Griffiths <em>et al.</em> (2000</a>, p. 481) missed the partial plating feature of that experiment. A recent tutorial (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meneely 2016</a>, p. 373) also mistakenly omitted the partial plating feature of the same experiment. Because 11 of the 20 cultures had no mutants, the tutorial argues by the <em>P</em><sub>0</sub> method that the mean number of mutations per culture is <em>m</em> ∼ 0.6.</p><p id="p-64">Why is the above claim incorrect? There is a crucial distinction between the number of zero mutant counts and the number of actual null cultures—cultures having no mutants prior to plating. Suppose a culture contains one mutant prior to plating. When 40% of that culture is plated, the mutant has a 60% chance of not being counted. Thus, a zero mutant count may result from a nonnull culture. In other words, the experimentalist is uncertain about the number of actual null cultures. To account for this additional source of uncertainty, one can apply a modified <em>P</em><sub>0</sub> method, which was proposed by <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Stewart <em>et al.</em> (1990</a>, p. 184).</p><p id="p-65">&gt; p0.LD.plating(luria.16.data,e = 0.4)</p><p id="p-66">[1] 0.9786801</p><p id="p-67">Hence, the mean number of mutations per culture is <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-22.gif"/></noscript></span></span> not 0.60. Luria and Delbrück reported that each culture contained 5.6 × 10<sup>8</sup> bacteria. Is this the number of cells in the plated portion or in the whole culture? To remove the ambiguity, note that Luria and Delbrück also reported an average mutant count of 28.4 mutants per culture. Because the mean number of the observed mutants in the plated portion was 11.35, the meaning of “per culture” is now clear. There must be 5.6 × 10<sup>8</sup> cells in the whole culture. That is, <em>N<sub>t</sub></em> = 5.6 × 10<sup>8</sup>. Therefore, the mutation rate is <em>p</em> = 0.98/<em>N<sub>t</sub></em> = 1.75 × 10<sup>−9</sup> mutations per cell division, according to Equation (1).</p><p id="p-68">rSalvador relies on the mutant distribution defined by the PGF <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-23.gif"/></noscript></span></span> in Equation (5) to compute ML estimates and likelihood ratio C.I.s for the fundamental parameter <em>m</em>. It employs the algorithms developed by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zheng (2008)</a>. For example, an ML estimate of <em>m</em> for Luria and Delbrück’s experiment 16 is</p><p id="p-69">&gt; newton.LD.plating(luria.16.data,e = 0.4)</p><p id="p-70">[1] 1.18636</p><p id="p-71">and an LR-based 95% C.I. for <em>m</em> is</p><p id="p-72">&gt; confint.LD.plating(luria.16.data,e = 0.4)</p><p id="p-73">[1] 0.5803079 2.0908012</p><p id="p-74">Many recent investigators were unappreciative toward the effects of partial plating. For example, in the reanalysis of the experiments of <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Luria and Delbrück (1943)</a>, <a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Hamon and Ycart (2012</a>, Table 1) and <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Ycart (2013</a>, Table 3) ignored the effects of partial plating. In the reanalysis of data from <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">David (1970)</a> and from <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Werngren and Hoffner (2003)</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ycart and Veziris (2014)</a> also disregarded the effects of partial plating. Moreover, unusual care may be required to determine the plating efficiency of an experiment that involves sophisticated procedures such as culture condensation. For instance, the plating efficiency in the study by <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Werngren and Hoffner (2003)</a> was 0.4, as explained in <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zheng (2016a</a>, p. 357). But at first glance the plating efficiency may seem to be 0.2, as was assumed by <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Mazoyer <em>et al.</em> (2017)</a>.</p><p id="p-75">Investigators who were aware of the effects of partial plating often relied on an adjustment due to <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Stewart <em>et al.</em> (1990)</a>, as this was the only approach recommended on the FALCOR web site. To apply the Stewart approach, the experimentalist first computes an initial estimate <em>m</em>* of <em>m</em> by assuming perfect plating efficiency.</p><p id="p-76">&gt; newton.LD(recent.expt)</p><p id="p-77">[1] 15.36162</p><p id="p-78">Because the adjustment factor is <span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-24.gif"/></noscript></span>, an estimate of <em>m</em> is <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-25.gif"/></noscript></span></span>× A = 15.36 × 21.5 = 330.24. Note that this estimate of <em>m</em> is ∼56% of the corresponding ML estimate (593.6). The reason for this inadequacy of the Stewart approach was given by <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Zheng (2015b)</a>. The above discussion is limited to cases where <em>w</em> = 1. <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Gillet-Markowska <em>et al.</em> (2015)</a> proposed applying the Stewart method also to cases where <em>w</em> ≠ 1, which is inappropriate in view of the above example. rSalvador currently is unable to deal with such cases.</p></div><div class="section other5" id="sec-5"><h2 class="">Comparing Mutation Rates</h2><p id="p-79">Rarely did a researcher perform fluctuation experiments solely to estimate mutation rates. The researcher’s task is often to compare mutation rates across strains or under distinct experimental conditions. Early investigators applied a modified <em>t</em>-test to accomplish this task. This practice can lead to misleading <em>p</em>-values (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zheng 2015a</a>). Recent investigators turned to the Mann-Whitney test (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Ford <em>et al.</em> 2013</a>), which also is inappropriate. When applying the Mann-Whitney test to fluctuation assay data, the investigator actually compares the mean numbers of mutations (<em>m</em>) between two experiments, not the two mutation rates that need comparing. To compare mutation rates, the investigator must take into consideration the final cell numbers <em>N<sub>t</sub></em>. It is a common feature that <em>N<sub>t</sub></em> values are consistent within each group (<em>e.g.</em>, a strain or a particular experimental condition), but they differ noticeably between groups.</p><p id="p-80">rSalvador provides two approaches to mutation rate comparison. The recommended approach is the likelihood ratio test (LRT) for fluctuation assay data, developed by <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Zheng (2016a)</a>. The second is purely empirical; it tests for equality of two mutation rates by checking whether two 84% C.I.s for the mutation rates overlap. If the two 84% C.I.s overlap, the test is not significant at the 5% level. Empirical evidence supporting the latter approach can be found in <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Zheng (2015a)</a>. Consider now the following two experiments from the study of <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Krašovec <em>et al.</em> (2014)</a> (data were kindly provided by R. Krašovec).</p><p id="p-81">expt1 = c(0, 2, 0, 3, 0, 15, 21, 0, 1, 0, 0,</p><p id="p-82">4, 4, 1, 3, 2, 1, 0, 0, 0, 1, 0, 9, 0)</p><p id="p-83">expt2 = c(8, 2, 4, 3, 6, 11, 2, 2, 0, 13, 6, 8)</p><p id="p-84">The investigators measured the relative fitness in both experiments and obtained <em>w</em><sub>1</sub> = 1.47 and <em>w</em><sub>2</sub> = 1.45. They also measured the two <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-26.gif"/></noscript></span></span> values: <em>N<sub>t,1</sub></em> = 2.27 × 10<sup>8</sup> and <em>N<sub>t,2</sub></em> = 5.15 × 10<sup>8</sup>. A Mann-Whitney test would yield a <em>p</em>-value of 0.01. But this <em>p</em>-value is misleading, as cells in the second population underwent about twice as many cell divisions as cells in the first population (the precise ratio is <em>R</em> = <em>N<sub>t</sub></em><sub>,2</sub> / <em>N<sub>t</sub></em><sub>,1</sub> = 2.29). The LRT approach gives a likelihood ratio statistic of 0.19, corresponding to an approximate <em>p</em>-value of 0.66.</p><p id="p-85">&gt; LRT.MK(expt1,expt2,w1 = 1.47,w2 = 1.45,R = 2.29)</p><p id="p-86">[1] 0.1924875 0.6608543</p><p id="p-87">Not surprisingly, the 84% C.I.s for the two mutation rates overlap.</p><p id="p-88">&gt; confint.MK(expt1,w = 1.47)/2.27e8</p><p id="p-89">[1] 1.798407e–09 5.244954e–09</p><p id="p-90">&gt; confint.MK(expt2,w = 1.45)/5.15e8</p><p id="p-91">[1] 2.155918e–09 6.105526e–09</p><p id="p-92">Both approaches are applicable to cases where partial plating is adopted. Consider mutant count data from the first (strains H37Rv) and third (strain E 729/94) experiments in the study of <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Werngren and Hoffner (2003)</a>. The plating efficiencies in both experiments were 0.4. The final cell density for the H37Rv strain was 2.3 × 10<sup>8</sup> cells per ml and that for the E 729/94 strain was 1.3 × 10<sup>8</sup> cells per ml. Because cultures in both experiments were of the same size (5 ml), the ratio of the two <em>N<sub>t</sub></em> values is therefore <em>R</em> = 1.3/2.3. An LRT gives a <em>p</em>-value of 0.62.</p><p id="p-93">&gt; LRT.LD.plating(wh.data[[1]], wh.data[[3]], e1 = 0.4,</p><p id="p-94">e2 = 0.4, R = 1.3/2.3)</p><p id="p-95">[1] 0.2435538 0.6216511</p><p id="p-96"><a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Mazoyer <em>et al.</em> (2017)</a> proposed a new method based on asymptotic normality of estimated mutation rates. The following simulation study compares statistical power of the three methods. Seven groups of 20-culture fluctuation experiments were simulated using the Lea-Coulson mutant distribution. Each group comprises 10,000 experiments. In the baseline group, <em>N<sub>t</sub></em> = 2 × 10<sup>8</sup> and the mutation rate is 1.0 × 10<sup>−8</sup> . In the other groups, <em>N<sub>t</sub></em> = 1 × 10<sup>8</sup> and the mutation rates are <em>k</em> × 10<sup>−8</sup> for <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-27.gif"/></noscript></span></span> and 3.0. In all groups, <em>N</em><sub>0</sub> = 50. Each experiment in the baseline group is compared with the corresponding experiments (<em>i.e.</em>, experiments having the same serial number) in all other six groups. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows statistical power of the three methods at the 5% significance level. The LRT method and the method of checking overlapping of 84% C.I.s performed almost equally well, but the asymptotic normality method was less powerful.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487503/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487503/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Statistical power (Lea-Coulson model)</span><div class="sb-div caption-clear"></div></div></div></div><div class="section other6" id="sec-6"><h2 class="">Accounting for Variability in <em>N<sub>t</sub></em></h2><p id="p-98">There was deep concern that variability in <em>N<sub>t</sub></em> among cultures could interfere with the analysis of fluctuation assay data (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Rosche and Foster 2000</a>). However, findings in a recent study (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zheng 2016b</a>) have somewhat lessened worries about that issue. In practice, the experimentalist can use a few cultures to gauge the variability in <em>N<sub>t</sub></em> by calculating the coefficient of variation (CV) for <em>N<sub>t</sub></em>. The new study found that a CV of 0.2 or smaller has a negligible effect on the estimation of mutation rates. More importantly, larger CVs for <em>N<sub>t</sub></em> were rarely encountered in practice. Therefore, if the experimentalist is facing a more critical issue, <em>e.g.</em>, partial plating, consideration of variability in <em>N<sub>t</sub></em> should give way to addressing the more critical issue.</p><p id="p-99">Several methods to account for variability in <em>N<sub>t</sub></em> were reviewed in <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Zheng (2016b)</a>. For example, FALCOR refers to a method that requires the experimentalist to measure <em>N<sub>t</sub></em> for all cultures, which often is too laborious to be practical. rSalvador offers a method based on the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-29.gif"/></noscript></span></span> distribution defined by the PGF <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-30.gif"/></noscript></span></span> in Equation (6). The <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-31.gif"/></noscript></span></span> method assumes that the expected number of mutations in a culture is proportional to the <em>N<sub>t</sub></em> of that culture. It further assumes that, conditional on a culture’s <em>N<sub>t</sub></em>, the number of mutants in that culture obeys a Lea-Coulson distribution. If the CV for <em>N<sub>t</sub></em> in an experiment is <em>C</em>, then, under additional mild assumptions, the unconditional distribution of the number of mutants is a <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-32.gif"/></noscript></span></span> distribution (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Zheng 2016b</a>). Here, <em>m</em><sub>0</sub> is regarded as the overall mean number of mutations per culture, and one can use rSalvador to make inferences about the parameter <em>m</em><sub>0</sub>. If <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-33.gif"/></noscript></span></span>is an estimate of <em>m</em><sub>0</sub>, then an estimate of the mutation rate is <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-34.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-35.gif"/></noscript></span></span> is the mean of <em>N<sub>t</sub></em> of all cultures in the experiment.</p><p id="p-100">As an example, suppose that the CV for <em>N<sub>t</sub></em> in the Demerec experiment (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Demerec 1945</a>) is 0.15. Point and interval estimates of <em>m</em><sub>0</sub> are obtained as follows.</p><p id="p-101">&gt; newton.B0(demerec.data,cv = 0.15)</p><p id="p-102">[1] 11.09696</p><p id="p-103">&gt; confint.B0(demerec.data,cv = 0.15)</p><p id="p-104">[1] 8.765365 13.665749</p></div><div class="section other7" id="sec-7"><h2 class="">Determining Sample Size</h2><p id="p-105">The question of “How many cultures are needed” must have been asked by investigators countless times, but it has received little attention in the literature. Investigators often choose a sample size on the basis of a published example in which the sample size was determined intuitively. There were no guidelines for determining sample size until recently (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zheng 2017b</a>).</p><p id="p-106">In planning an experiment, the investigator may form a preliminary idea about the magnitude of <em>m</em>. A large <em>m</em> may reduce the sample size, but it may result in cultures containing more mutants than the investigator is able to count. <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Zheng (2017b)</a> proposed the following η<sub>500</sub> index to guide experimental design:</p><p id="p-107"><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>As a rule of thumb, the experimentalist may choose η<sub>500</sub> &lt; 0.01. Thus, the experimentalist allows no &gt;1% of the cultures to have &gt;500 mutants per culture. As an illustration, assume that the mutation rate to be determined is in the neighborhood of 2 × 10<sup>−7</sup> mutations per cell division. Assume further that the mutants have a relative fitness of 0.75. If the investigator plans to allow the final number of cells per culture to reach 2 × 10<sup>7</sup>, then <em>m</em> would be ∼4.0 according to Equation (1). Therefore, the η<sub>500</sub> index is</p><p id="p-108">&gt; 1 – sum(prob.MK(m = 4,w = 0.75, n = 500))</p><p id="p-109">[1] 0.001243678</p><p id="p-110">But if the mutants have a relative fitness of 1.2, the η<sub>500</sub> index will be larger.</p><p id="p-111">&gt; 1 – sum(prob.MK(m = 4,w = 1.2, n = 500))</p><p id="p-112">[1] 0.02221585</p><p id="p-113">If the anticipated value of <em>m</em> meets the η<sub>500</sub> criterion, the experimentalist determines a required sample size by choosing an appropriate ψ score. The ψ score was introduced by <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Zheng (2017b)</a> as a convenient yardstick for judging the quality of C.I.s. The ψ score is defined by</p><p id="p-114"><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-37.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>Consider a case where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-38.gif"/></noscript></span></span> has been determined via a fitness assay. The experimentalist further believes that <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-39.gif"/></noscript></span></span> Choosing a ψ score of 0.25 would lead to an estimated sample size of 31.</p><p id="p-115">&gt; samp.size.MK(m = 4,w = 0.75,psi = 0.25)</p><p id="p-116">[1] 31</p><p id="p-117">If an anticipated value of <em>m</em> is large enough to lead to an unacceptable η<sub>500</sub> index, the experimentalist may resort to partial plating. For example, in the case where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/3849/embed/mml-math-41.gif"/></noscript></span></span> the η<sub>500</sub> index is</p><p id="p-118">&gt; 1 – sum(prob.MK(m = 50,w = 1,n = 500))</p><p id="p-119">[1] 0.1894214</p><p id="p-120">However, by plating 10% of each culture, the experimentalist reduces η<sub>500</sub> to</p><p id="p-121">&gt; 1 – sum(prob.LD.plating(m = 50,e = 0.1,n = 500))</p><p id="p-122">[1] 0.01086645</p><p id="p-123">And the corresponding sample size is</p><p id="p-124">&gt; samp.size.LD.plating(m = 50,e = 0.1)</p><p id="p-125">[1] 15</p><div id="sec-8" class="subsection"><h3>Conclusions</h3><p id="p-126">Methodological advances made in the past 74 yr have presented the experimentalist with a bewildering array of methods to analyze fluctuation assay data. By providing concrete examples, this article helps the reader make an informed decision about which method is the most appropriate for a given experiment. All discussed methods are approximate, as the underlying mathematical models inevitably depend on simplifying assumptions. Further research will no doubt lead to more flexible methods by relaxing or dropping one or other of these assumptions, <em>e.g.</em>, the assumptions (A1)–(A11). However, it is impossible to construct a comprehensive mathematical model by discarding all the 11 assumptions. Even if this herculean feat were accomplished someday, the resultant model would possess these two characteristics: it would still be an approximation to the infinitely complex biological reality, and it would be too cumbersome to be useful in practice. Researchers investigating a challenging biological problem should also play an active part in choosing an appropriate method to analyze their fluctuation assay data, treating the task as an existing, integral part of their research endeavor, not merely as an inescapable, tedious last leg of a long scientific journey.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-127">I express my heartfelt appreciation to four referees and the communicating editor, whose comments helped me reshape an earlier draft of the manuscript. I am also indebted to Texas A&amp;M High Performance Research Computing, which allowed me to spend above half a million computing hours in testing rSalvador via simulation. The open-access publishing fees for this article have been covered by the Texas A&amp;M University Open Access to Knowledge Fund (OAKFund), supported by the University Libraries and the office of the Vice President for Research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-128"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-05"><span class="received-label">Received </span>August 5, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-24"><span class="accepted-label">Accepted </span>October 24, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Zheng</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angerer</span>  <span class="cit-name-given-names">W. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A note on the evaluation of fluctuation experiments.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">479</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DAngerer%26rft.auinit1%253DW.%2BP.%26rft.volume%253D479%26rft.issue%253D1-2%26rft.spage%253D207%26rft.epage%253D224%26rft.atitle%253DA%2Bnote%2Bon%2Bthe%2Bevaluation%2Bof%2Bfluctuation%2Bexperiments.%26rft_id%253Dinfo%253Apmid%252F11470493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11470493&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Armitage</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">The statistical theory of bacterial populations subject to mutation.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Ser. B</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSer.%2BB%26rft.volume%253D14%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.3" data-doi="10.1128/jb.176.10.2781-2787.1994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boe</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tolker-Nielsen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eegholm</span>  <span class="cit-name-given-names">K.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spliid</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vrang</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Fluctuation analysis of mutations to nalidixic acid resistance in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2781</span>–<span class="cit-lpage">2787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DBoe%26rft.auinit1%253DL.%26rft.volume%253D176%26rft.issue%253D10%26rft.spage%253D2781%26rft.epage%253D2787%26rft.atitle%253DFluctuation%2Banalysis%2Bof%2Bmutations%2Bto%2Bnalidixic%2Bacid%2Bresistance%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.10.2781-2787.1994%26rft_id%253Dinfo%253Apmid%252F8188580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3Ni8xMC8yNzgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.4" data-doi="10.1098/rstb.2016.0040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becks</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jalavuori</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiltunen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Sublethal streptomycin concentrations and lytic bacteriophage together promote resistance evolution.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">372</span>: <span class="cit-fpage">20160040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0040%26rft_id%253Dinfo%253Apmid%252F27920385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNzIvMTcxMi8yMDE2MDA0MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Probability distribution of drug-resistant mutants in unselected populations of <em>Mycobacterium tuberculosis</em>.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">810</span>–<span class="cit-lpage">814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Bmicrobiology%26rft.stitle%253DAppl%2BMicrobiol%26rft.aulast%253DDavid%26rft.auinit1%253DH.%2BL.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D810%26rft.epage%253D814%26rft.atitle%253DProbability%2Bdistribution%2Bof%2Bdrug-resistant%2Bmutants%2Bin%2Bunselected%2Bpopulations%2Bof%2BMycobacterium%2Btuberculosis.%26rft_id%253Dinfo%253Apmid%252F4991927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4991927&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970H858300034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.6" data-doi="10.1073/pnas.31.1.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demerec</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1945</span> <span class="cit-article-title">Production of <em>Staphylococcus</em> strains resistant to various concentrations of penicillin.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDemerec%26rft.auinit1%253DM.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DProduction%2Bof%2BStaphylococcus%2BStrains%2BResistant%2Bto%2BVarious%2BConcentrations%2Bof%2BPenicillin%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.31.1.16%26rft_id%253Dinfo%253Apmid%252F16588677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo3OiIzMS8xLzE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.7" data-doi="10.1093/biomet/65.3.457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinkley</span>  <span class="cit-name-given-names">D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Assessing the accuracy of the maximum likelihood estimator: observed <em>vs.</em> expected Fisher information.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DEFRON%26rft.auinit1%253DB.%26rft.volume%253D65%26rft.issue%253D3%26rft.spage%253D457%26rft.epage%253D483%26rft.atitle%253DAssessing%2Bthe%2Baccuracy%2Bof%2Bthe%2Bmaximum%2Blikelihood%2Bestimator%253A%2BObserved%2Bversus%2Bexpected%2BFisher%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F65.3.457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/65.3.457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978GA54900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.8" data-doi="10.1038/ng.2656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meada</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagneux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Mycobacterium tuberculosis</em> mutation rate estimates from different lineages predict substantial differences in the emergence of drug-resistant tuberculosis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">784</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D784%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2656%26rft_id%253Dinfo%253Apmid%252F23749189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23749189&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-7.12.3849.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillet-Markowska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louvel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">bz-rates: a web-tool to estimate mutation rates from fluctuation analysis.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">5</span>: <span class="cit-fpage">2323</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D5%26rft.spage%253D2323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3849.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">A. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelbart</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">An Introduction to Genetic Analysis</span>, <span class="cit-edition">Ed. 7</span>. <span class="cit-publ-name">W.H. Freeman and Company</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.11" data-doi="10.1093/bioinformatics/btp253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fluctuation AnaLysis CalculatOR: a web tool for the determination of mutation rate using Luria-Delbrück fluctuation analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1564</span>–<span class="cit-lpage">1565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp253%26rft_id%253Dinfo%253Apmid%252F19369502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19369502&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266498300065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.12" data-doi="10.1214/12-EJS711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ycart</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Statistics for the Luria-Delbrück distribution.</span> <abbr class="cit-jnl-abbrev">Electron. J. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron.%2BJ.%2BStat.%26rft.volume%253D6%26rft.spage%253D1251%26rft_id%253Dinfo%253Adoi%252F10.1214%252F12-EJS711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/12-EJS711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kendal</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Pitfalls and practice of Luria-Delbrück fluctuation analysis: a review.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1060</span>–<span class="cit-lpage">1065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DKendal%26rft.auinit1%253DW.%2BS.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1060%26rft.epage%253D1065%26rft.atitle%253DPitfalls%2Band%2BPractice%2Bof%2BLuria-Delbruck%2BFluctuation%2BAnalysis%253A%2BA%2BReview%26rft_id%253Dinfo%253Apmid%252F3277705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjk6IjQ4LzUvMTA2MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.14" data-doi="10.1016/0027-5107(82)90252-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Mutation and growth rates from Luria-Delbrück fluctuation tests.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%26rft.volume%253D95%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252882%252990252-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(82)90252-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.15" data-doi="10.1038/ncomms4742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krašovec</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belavkin</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aston</span>  <span class="cit-name-given-names">J. A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Channon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aston</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mutation rate plasticity in rifampicin resistance depends on <em>Escherichia coli</em> cell-cell interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3742%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4742%26rft_id%253Dinfo%253Apmid%252F24776982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24776982&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.16" data-doi="10.1534/genetics.107.071506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimating the per-base-pair mutation rate in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLang%26rft.auinit1%253DG.%2BI.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D82%26rft.atitle%253DEstimating%2Bthe%2BPer-Base-Pair%2BMutation%2BRate%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071506%26rft_id%253Dinfo%253Apmid%252F18202359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc4LzEvNjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzM4NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.17" data-doi="10.1007/BF02986080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lea</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The distribution of the numbers of mutants in bacterial populations.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D49%26rft.spage%253D264%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02986080%26rft_id%253Dinfo%253Apmid%252F24536673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02986080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24536673&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1949XX17200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">Forward and reverse mutation in a histidine-requiring strain of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">460</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLieb%26rft.auinit1%253DM.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D460%26rft.epage%253D477%26rft.atitle%253DFORWARD%2BAND%2BREVERSE%2BMUTATION%2BIN%2BA%2BHISTIDINE-REQUIRING%2BSTRAIN%2BOF%2BESCHERICHIA%2BCOLI%26rft_id%253Dinfo%253Apmid%252F17247359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzNi81LzQ2MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luria</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delbrück</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">Mutations of bacteria from virus sensitivity to virus resistance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLuria%26rft.auinit1%253DS.%2BE.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D491%26rft.epage%253D511%26rft.atitle%253DMUTATIONS%2BOF%2BBACTERIA%2BFROM%2BVIRUS%2BSENSITIVITY%2BTO%2BVIRUS%2BRESISTANCE%26rft_id%253Dinfo%253Apmid%252F17247100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyOC82LzQ5MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.20" data-doi="10.2307/3214564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandri</span>  <span class="cit-name-given-names">G. vH.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Analysis of the Luria and Delbrück distribution using discrete convolution powers.</span> <abbr class="cit-jnl-abbrev">J. Appl. Probab.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BProbab.%26rft.volume%253D29%26rft.spage%253D255%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3214564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3214564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992JC82500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.21" data-doi="10.2307/3212688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mandelbrot</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">A population birth-and-mutation process, I: explicit distributions for the number of mutants in an old culture of bacteria.</span> <abbr class="cit-jnl-abbrev">J. Appl. Probab.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BProbab.%26rft.volume%253D11%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3212688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3212688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mazoyer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouilhet</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Despréaux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ycart</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">flan: an R package for inference on mutation models.</span> <abbr class="cit-jnl-abbrev">R J.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BJ.%26rft.volume%253D9%26rft.spage%253D334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.23" data-doi="10.1534/genetics.115.184564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meneely</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Pick your Poisson: an educational primer for Luria and Delbrück’s classic paper.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeneely%26rft.auinit1%253DP.%2BM.%26rft.volume%253D202%26rft.issue%253D2%26rft.spage%253D371%26rft.epage%253D375%26rft.atitle%253DPick%2BYour%2BPoisson%253A%2BAn%2BEducational%2BPrimer%2Bfor%2BLuria%2Band%2BDelbrucks%2BClassic%2BPaper%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.184564%26rft_id%253Dinfo%253Apmid%252F26869481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvMzcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.24" data-doi="10.1002/(SICI)1098-2280(1996)28:2&lt;90::AID-EM4&gt;3.0.CO;2-I"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nádas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goncharova</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossman</span>  <span class="cit-name-given-names">T. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mutations and infinity: improved statistical methods for estimating spontaneous rates.</span> <abbr class="cit-jnl-abbrev">Environ. Mol. Mutagen.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Band%2Bmolecular%2Bmutagenesis%26rft.stitle%253DEnviron%2BMol%2BMutagen%26rft.aulast%253DN%25C3%25A1das%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D90%26rft.epage%253D99%26rft.atitle%253DMutations%2Band%2Binfinity%253A%2Bimproved%2Bstatistical%2Bmethods%2Bfor%2Bestimating%2Bspontaneous%2Brates.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291098-2280%25281996%252928%253A2%253C90%253A%253AAID-EM4%253E3.0.CO%253B2-I%26rft_id%253Dinfo%253Apmid%252F8844989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1098-2280(1996)28:2&lt;90::AID-EM4&gt;3.0.CO;2-I&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8844989&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newcombe</span>  <span class="cit-name-given-names">H. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">Delayed phenotypic expression of spontaneous mutations in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNewcombe%26rft.auinit1%253DH.%2BB.%26rft.volume%253D33%26rft.issue%253D5%26rft.spage%253D447%26rft.epage%253D476%26rft.atitle%253DDELAYED%2BPHENOTYPIC%2BEXPRESSION%2BOF%2BSPONTANEOUS%2BMUTATIONS%2BIN%2BESCHERICHIA%2BCOLI%26rft_id%253Dinfo%253Apmid%252F17247291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzMy81LzQ0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.26" data-doi="10.1093/bioinformatics/btp358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raue</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreuta</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maiwald</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schilling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Structural and practical identifiability analysis of partially observed dynamical models by exploiting the profile likelihood.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1923</span>–<span class="cit-lpage">1929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp358%26rft_id%253Dinfo%253Apmid%252F19505944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505944&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.27" data-doi="10.1006/meth.1999.0901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosche</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Determining mutation rates in bacterial populations.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DRosche%26rft.auinit1%253DW.%2BA.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D4%26rft.epage%253D17%26rft.atitle%253DDetermining%2Bmutation%2Brates%2Bin%2Bbacterial%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.1999.0901%26rft_id%253Dinfo%253Apmid%252F10610800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.1999.0901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10610800&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084627600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.28" data-doi="10.1093/molbev/msv032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scanlan</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friman</span>  <span class="cit-name-given-names">V.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Coevolution with bacteriophage drives genome-wide host evolution and constrains the acquisition of abiotic-beneficial mutations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1425</span>–<span class="cit-lpage">1435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv032%26rft_id%253Dinfo%253Apmid%252F25681383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25681383&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Fluctuation analysis: the probability distribution of the number of mutants under different conditions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStewart%26rft.auinit1%253DF.%2BM.%26rft.volume%253D124%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D185%26rft.atitle%253DFluctuation%2Banalysis%253A%2Bthe%2Bprobability%2Bdistribution%2Bof%2Bthe%2Bnumber%2Bof%2Bmutants%2Bunder%2Bdifferent%2Bconditions.%26rft_id%253Dinfo%253Apmid%252F2307353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzEvMTc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi8zODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.30" data-doi="10.1128/JCM.41.4.1520-1524.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Werngren</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffner</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Drug-susceptible <em>Mycobacterium tuberculosis</em> Beijing genotype does not develop mutation-conferred resistance to rifampin at an elevated rate.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1520</span>–<span class="cit-lpage">1524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DWerngren%26rft.auinit1%253DJ.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D1520%26rft.epage%253D1524%26rft.atitle%253DDrug-Susceptible%2BMycobacterium%2Btuberculosis%2BBeijing%2BGenotype%2BDoes%2BNot%2BDevelop%2BMutation-Conferred%2BResistance%2Bto%2BRifampin%2Bat%2Ban%2BElevated%2BRate%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.41.4.1520-1524.2003%26rft_id%253Dinfo%253Apmid%252F12682139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjQxLzQvMTUyMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvMzg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.3849.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Wolfram Research, Inc.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source"><em>Mathematica</em>, <em>Version 11.0</em></span>. <span class="cit-publ-name">Wolfram Research, Inc.</span>, <span class="cit-publ-loc">Champaign, IL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.32" data-doi="10.1371/journal.pone.0080958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ycart</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fluctuation analysis: can estimates be trusted?</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e80958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De80958%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0080958%26rft_id%253Dinfo%253Apmid%252F24349026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0080958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24349026&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.33" data-doi="10.1371/journal.pone.0101434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ycart</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veziris</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unbiased estimation of mutation rates under fluctuating final counts.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e101434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101434%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101434%26rft_id%253Dinfo%253Apmid%252F24988217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24988217&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.34" data-doi="10.1016/S0025-5564(02)00087-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Statistical and algorithmic methods for fluctuation analysis with SALVADOR as an implementation.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DZheng%26rft.auinit1%253DQ.%26rft.volume%253D176%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D252%26rft.atitle%253DStatistical%2Band%2Balgorithmic%2Bmethods%2Bfor%2Bfluctuation%2Banalysis%2Bwith%2BSALVADOR%2Bas%2Ban%2Bimplementation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0025-5564%252802%252900087-1%26rft_id%253Dinfo%253Apmid%252F11916511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0025-5564(02)00087-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11916511&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175028400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.35" data-doi="10.1016/j.mbs.2005.03.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">New algorithms for Luria-Delbrück fluctuation analysis.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DZheng%26rft.auinit1%253DQ.%26rft.volume%253D196%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D214%26rft.atitle%253DNew%2Balgorithms%2Bfor%2BLuria-Delbruck%2Bfluctuation%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mbs.2005.03.011%26rft_id%253Dinfo%253Apmid%252F15950991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mbs.2005.03.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15950991&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231329200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.36" data-doi="10.1016/j.mbs.2007.03.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">On Haldane’s formulation of Luria and Delbrück’s mutation model.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">500</span>–<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DZheng%26rft.auinit1%253DQ.%26rft.volume%253D209%26rft.issue%253D2%26rft.spage%253D500%26rft.epage%253D513%26rft.atitle%253DOn%2BHaldane%2527s%2Bformulation%2Bof%2BLuria%2Band%2BDelbruck%2527s%2Bmutation%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mbs.2007.03.003%26rft_id%253Dinfo%253Apmid%252F17462675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mbs.2007.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17462675&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.37" data-doi="10.1016/j.mbs.2008.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A note on plating efficiency in fluctuation experiments.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DZheng%26rft.auinit1%253DQ.%26rft.volume%253D216%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D153%26rft.atitle%253DA%2Bnote%2Bon%2Bplating%2Befficiency%2Bin%2Bfluctuation%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mbs.2008.09.002%26rft_id%253Dinfo%253Apmid%252F18822300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mbs.2008.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18822300&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F3849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A new discrete distribution induced by the Luria-Delbrück mutation model.</span> <abbr class="cit-jnl-abbrev">Statistics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistics%26rft.volume%253D44%26rft.spage%253D529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Methods for comparing mutation rates using fluctuation assay data.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">777</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D777%26rft.spage%253D20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">A new practical guide to the Luria-Delbrück protocol.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">781</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D781%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.41" data-doi="10.1007/s10709-016-9904-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Comparing mutation rates under the Luria-Delbrück protocol.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D144%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-016-9904-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-016-9904-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">A second look at the final number of cells in a fluctuation experiment.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">401</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BTheor.%2BBiol.%26rft.volume%253D401%26rft.spage%253D54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-7.12.3849.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> rSalvador: an R tool for the Luria-Delbrück fluctuation assay. Available at: <a href="http://eeeeeric.github.io/rSalvador">http://eeeeeric.github.io/rSalvador</a>. Accessed: November 11, 2017.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Sample size determination for the fluctuation experiment.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">795</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D795%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.3849.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">c</span> <span class="cit-article-title">Toward a unique definition of the mutation rate.</span> <abbr class="cit-jnl-abbrev">Bull. Math. Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull.%2BMath.%2BBiol.%26rft.volume%253D79%26rft.spage%253D683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/12/3849.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/3839" title="ARSDA: A New Approach for Storing, Transmitting and Analyzing Transcriptomic Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/3857" title="Dissecting Nucleosome Function with a Comprehensive Histone H2A and H2B Mutant Library" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/12"><span>Volume 7 Issue 12, December 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/12.cover-source.jpg?itok=-p16LxzL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/3849" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PrZ4sJnwIn9j60PkAgg0eR4GT57sF6QC6lgudxqvJ4g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/12/3849" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/12/3849" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/12/3849" class="active" data-icon-position="" data-hide-link-title="0">rSalvador: An R Package for the Fluctuation Experiment</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487482" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7Gb4zUCDZX4DX_9Otq3FhsuQNtPU2NT93BPfrk0ZQ0w" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/3849.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/12/3849" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-gwcxYneD1R8J-vu_hXlQ6htHbH3xAP8jkNBLop-vMWQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/12/3849" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/3849.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487482" id="node487482--2" data-pisa="ggg;7/12/3849" data-pisa-master="ggg;g3.117.300120" data-apath="/ggg/7/12/3849.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">rSalvador: An R Package for the Fluctuation Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qi Zheng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3849-3856;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487482--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qi Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Epidemiology and Biostatistics, Texas A&M School of Public Health, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qi%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQi%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:qzheng@sph.tamhsc.edu" class="" data-icon-position="" data-hide-link-title="0">qzheng@sph.tamhsc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/12/3849" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/12/3849&amp;title=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/12/3849&amp;title=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/12/3849&amp;title=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/12/3849&amp;text=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/12/3849&amp;title=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/12/3849&amp;t=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/12/3849&amp;title=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/12/3849&amp;title=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487482/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487482" id="node487482--3" data-pisa="ggg;7/12/3849" data-pisa-master="ggg;g3.117.300120" data-apath="/ggg/7/12/3849.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">rSalvador: An R Package for the Fluctuation Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qi Zheng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3849-3856;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487482--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qi Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Epidemiology and Biostatistics, Texas A&M School of Public Health, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qi%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQi%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:qzheng@sph.tamhsc.edu" class="" data-icon-position="" data-hide-link-title="0">qzheng@sph.tamhsc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3849&amp;rft.epage=3856&amp;rft.atitle=rSalvador%3A%20An%20R%20Package%20for%20the%20Fluctuation%20Experiment&amp;rft.volume=7&amp;rft.issue=12&amp;rft.date=2017-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zheng%2C+Qi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487482/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487482/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487482/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487482/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487482/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487482/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487482/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487482/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487482/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487482/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487482/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487482/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/933" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/12/3849" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Relevant Assumptions</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Classical Analysis</a></li><li class="even"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Accounting for Relative Fitness</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Accounting for Plating Efficiency</a></li><li class="even"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Comparing Mutation Rates</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Accounting for Variability in <em>N<sub>t</sub></em></a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Determining Sample Size</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/12/3849.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/12/3849.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/12/3849.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
