<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/12/4009" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/12/4009.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/12/4009.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/12/4009.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/12/4009/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/12/4009.atom" />
<meta name="HW.pisa" content="ggg;7/12/4009" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle" />
<meta name="DC.Identifier" content="10.1534/g3.117.300272" />
<meta name="DC.Date" content="2017-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Wang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The long-term performance of different selection strategies was evaluated via simulation using the example of a local cattle breed, German Angler cattle. Different optimum contribution selection (OCS) approaches to maximize genetic gain were compared to a reference scenario without selection and truncation selection. The kinships and migrant contribution (MC) were estimated from genomic data. Truncation selection achieved the highest genetic gain but decreased diversity considerably at native alleles. It also caused the highest increase in MCs. Traditional OCS, which only constrains kinship, achieved almost the same genetic gain but also caused a small increase of MC and remarkably reduced the diversity of native alleles. When MC was required not to increase and the increase of kinship at native alleles was restricted, the MC levels and the diversity at native alleles were well managed, and the genetic gain was only slightly reduced. However, genetic progress was substantially lower in the scenario that aimed to recover the original genetic background. Truncation selection and traditional OCS selection both reduce the genetic originality of breeds with historical introgression. The inclusion of MC and kinship at native alleles as additional constraints in OCS showed great potential for conservation. Recovery of the original genetic background is possible but requires many generations of selection and reduces the genetic progress in performance traits. Hence, constraining MCs at their current values can be recommended to avoid further reduction of genetic originality." />
<meta name="DC.Contributor" content="Yu Wang" />
<meta name="DC.Contributor" content="Dierck Segelke" />
<meta name="DC.Contributor" content="Reiner Emmerling" />
<meta name="DC.Contributor" content="Jörn Bennewitz" />
<meta name="DC.Contributor" content="Robin Wellmann" />
<meta name="article:published_time" content="2017-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The long-term performance of different selection strategies was evaluated via simulation using the example of a local cattle breed, German Angler cattle. Different optimum contribution selection (OCS) approaches to maximize genetic gain were compared to a reference scenario without selection and truncation selection. The kinships and migrant contribution (MC) were estimated from genomic data. Truncation selection achieved the highest genetic gain but decreased diversity considerably at native alleles. It also caused the highest increase in MCs. Traditional OCS, which only constrains kinship, achieved almost the same genetic gain but also caused a small increase of MC and remarkably reduced the diversity of native alleles. When MC was required not to increase and the increase of kinship at native alleles was restricted, the MC levels and the diversity at native alleles were well managed, and the genetic gain was only slightly reduced. However, genetic progress was substantially lower in the scenario that aimed to recover the original genetic background. Truncation selection and traditional OCS selection both reduce the genetic originality of breeds with historical introgression. The inclusion of MC and kinship at native alleles as additional constraints in OCS showed great potential for conservation. Recovery of the original genetic background is possible but requires many generations of selection and reduces the genetic progress in performance traits. Hence, constraining MCs at their current values can be recommended to avoid further reduction of genetic originality.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/12/01" />
<meta name="citation_mjid" content="ggg;7/12/4009" />
<meta name="citation_id" content="7/12/4009" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/12/4009" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/12/4009.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/12/4009.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/12/4009.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300272" />
<meta name="citation_pmid" content="29089375" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="4009" />
<meta name="citation_lastpage" content="4018" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yu Wang" />
<meta name="citation_author_institution" content="Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany" />
<meta name="citation_author_email" content="yu.wang@uni-hohenheim.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3756-6246" />
<meta name="citation_author" content="Dierck Segelke" />
<meta name="citation_author_institution" content="Vereinigte Informationssysteme Tierhaltung w.V., 27283 Verden, Germany" />
<meta name="citation_author" content="Reiner Emmerling" />
<meta name="citation_author_institution" content="Institute of Animal Breeding, Bavarian State Research Centre for Agriculture, 85586 Grub, Germany" />
<meta name="citation_author" content="Jörn Bennewitz" />
<meta name="citation_author_institution" content="Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany" />
<meta name="citation_author" content="Robin Wellmann" />
<meta name="citation_author_institution" content="Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4748-8042" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=C. Amador;citation_author=M. Á. Toro;citation_author=J. Fernández;citation_title=Removing exogenous information using pedigree data.;citation_pages=1565-1573;citation_volume=12;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. Amador;citation_author=J. Fernández;citation_author=T. H. E. Meuwissen;citation_title=Advantages of using molecular coancestry in the removal of introgressed genetic material.;citation_pages=13;citation_volume=45;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Baran;citation_author=B. Pasaniuc;citation_author=S. Sankararaman;citation_author=D. G. Torgerson;citation_author=C. Gignoux;citation_title=Fast and accurate inference of local ancestry in Latino populations.;citation_volume=28;citation_year=2012;citation_pmid=22495753;citation_doi=10.1093/bioinformatics/bts144" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=J. Bennewitz;citation_author=T. H. E. Meuwissen;citation_title=Estimation of Extinction Probabilities of Five German Cattle Breeds by Population Viability Analysis;citation_pages=2949-2961;citation_volume=88;citation_year=2005;citation_issue=8;citation_pmid=16027209" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=P. Bijma;citation_title=Accuracies of estimated breeding values from ordinary genetic evaluations do not reflect the correlation between true and estimated breeding values in selected populations.;citation_pages=345-358;citation_volume=129;citation_year=2012;citation_issue=5;citation_pmid=22963356;citation_doi=10.1111/j.1439-0388.2012.00991.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. G. Bulmer;citation_title=The effect of selection on genetic variability.;citation_pages=201-211;citation_volume=105;citation_year=1971;citation_doi=10.1086/282718" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=A. Caballero;citation_author=M. Á. Toro;citation_title=Analysis of genetic diversity for the management of conserved subdivided populations.;citation_pages=289-299;citation_volume=3;citation_year=2002;citation_doi=10.1023/A:1019956205473" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=I. Curik;citation_author=M. Ferenčaković;citation_author=J. Sölkner;citation_title=Inbreeding and runs of homozygosity: a possible solution to an old problem.;citation_pages=26-34;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.05.034" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. Á. R. de Cara;citation_author=J. Fernández;citation_author=M. Á. Toro;citation_author=B. Villanueva;citation_title=Using genome-wide information to minimize the loss of diversity in conservation programmes.;citation_pages=456-464;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059579;citation_doi=10.1111/j.1439-0388.2011.00971.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Á. R. de Cara;citation_author=B. Villanueva;citation_author=M. Á. Toro;citation_author=J. Fernández;citation_title=Using genomic tools to maintain diversity and fitness in conservation programmes.;citation_pages=6091-6099;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12560" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=H. Eding;citation_author=T. H. E. Meuwissen;citation_title=Marker-based estimates of between and within population kinships for the conservation of genetic diversity.;citation_pages=141-159;citation_volume=118;citation_year=2001;citation_doi=10.1046/j.1439-0388.2001.00290.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. E. Eynard;citation_author=J. J. Windig;citation_author=S. J. Hiemstra;citation_author=M. P. L. Calus;citation_title=Whole-genome sequence data uncover loss of genetic diversity due to selection.;citation_pages=33;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=A. Frkonja;citation_author=B. Gredler;citation_author=U. Schnyder;citation_author=I. Curik;citation_author=J. Sölkner;citation_title=Prediction of breed composition in an admixed cattle population.;citation_pages=696-703;citation_volume=43;citation_year=2012;citation_issue=6;citation_pmid=23061480;citation_doi=10.1111/j.1365-2052.2012.02345.x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=F. Gómez-Romano;citation_author=B. Villanueva;citation_author=J. Sölkner;citation_author=M. Á. R. de Cara;citation_author=G. Mészáros;citation_title=The use of coancestry based on shared segments for maintaining genetic diversity.;citation_pages=357-365;citation_volume=133;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=G. Gorjanc;citation_author=P. Bijma;citation_author=J. M. Hickey;citation_title=Reliability of pedigree-based and genomic evaluations in selected populations.;citation_pages=65;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Math. Biosci.;citation_author=H.-R. Gregorius;citation_title=The concept of genetic diversity and its formal relationship to heterozygosity and genetic distance.;citation_pages=253-271;citation_volume=41;citation_year=1978;citation_doi=10.1016/0025-5564(78)90040-8" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=B. Grundy;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Dynamic selection procedures for constrained inbreeding and their consequences for pedigree development.;citation_pages=159-168;citation_volume=72;citation_year=1998;citation_doi=10.1017/S0016672398003474" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Gusev;citation_author=J. K. Lowe;citation_author=M. Stoffel;citation_author=M. J. Daly;citation_author=D. Altshuler;citation_title=Whole population, genome-wide mapping of hidden relatedness.;citation_volume=19;citation_year=2009;citation_pmid=18971310;citation_doi=10.1101/gr.081398.108" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=J. L. Harper;citation_author=D. L. Hawksworth;citation_title=Biodiversity: measurement and estimation.;citation_volume=345;citation_year=1994;citation_doi=10.1098/rstb.1994.0081" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. C. Keller;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Quantification of inbreeding due to distant ancestors and its detection using dense single nucleotide polymorphism data.;citation_volume=189;citation_year=2011;citation_pmid=21705750;citation_doi=10.1534/genetics.111.130922" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=E. J. McTavish;citation_author=D. M. Hillis;citation_title=How do SNP ascertainment schemes and population demographics affect inferences about population history?;citation_pages=266;citation_volume=16;citation_year=2015;citation_pmid=25887858;citation_doi=10.1186/s12864-015-1469-5" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Mészáros;citation_author=S. A. Boison;citation_author=A. M. Pérez O’Brien;citation_author=M. Ferenčaković;citation_author=I. Curik;citation_title=Genomic analysis for managing small and endangered populations: a case study in Tyrol Grey cattle.;citation_pages=1-12;citation_volume=6;citation_year=2015;citation_pmid=25674101;citation_doi=10.3389/fgene.2015.00001" />
<meta name="citation_reference" content="citation_journal_title=Acta Agric. Scand. A Anim. Sci.;citation_author=T. Meuwissen;citation_title=Genetic management of small populations: a review.;citation_pages=71-79;citation_volume=59;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=T. H. E. Meuwissen;citation_title=Maximizing the response of selection with a predefined rate of inbreeding.;citation_pages=2575-2583;citation_volume=75;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Nielsen;citation_title=Estimation of Population Parameters and Recombination Rates From Single Nucleotide Polymorphisms;citation_pages=931-942;citation_volume=154;citation_year=2000;citation_issue=2;citation_pmid=10655242" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=E. Peripolli;citation_author=D. P. Munari;citation_author=M. V. B. Silva;citation_author=A. L. F. Lima;citation_author=R. Irgang;citation_title=Runs of homozygosity: current knowledge and applications in livestock.;citation_pages=255-271;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_author=B. Pfaff;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=S. T. Rodríguez-Ramilo;citation_author=L. A. García-Cortés;citation_author=M. Á. R. de Cara;citation_title=Artificial selection with traditional or genomic relationships: consequences in coancestry and genetic diversity.;citation_pages=127;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Ron;citation_author=Y. Blanc;citation_author=M. Band;citation_author=E. Ezra;citation_author=J. I. Weller;citation_title=Misidentification Rate in the Israeli Dairy Cattle Population and Its Implications for Genetic Improvement;citation_pages=676-681;citation_volume=79;citation_year=1996;citation_issue=4;citation_pmid=8744233;citation_doi=10.3168/jds.S0022-0302(96)76413-5" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=P. Stam;citation_title=The distribution of the fraction of the genome identical by descent in finite random mating populations.;citation_pages=131;citation_volume=35;citation_year=1980;citation_doi=10.1017/S0016672300014002" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. Á. Toro;citation_author=B. Villanueva;citation_author=J. Fernández;citation_title=Genomics applied to management strategies in conservation programmes.;citation_pages=48-53;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.04.020" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. M. Visscher;citation_author=S. E. Medland;citation_author=M. A. R. Ferreira;citation_author=K. I. Morley;citation_author=G. Zhu;citation_title=Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.;citation_pages=e41-e41;citation_volume=2;citation_year=2006;citation_issue=3;citation_pmid=16565746;citation_doi=10.1371/journal.pgen.0020041" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Y. Wang;citation_author=J. Bennewitz;citation_author=R. Wellmann;citation_title=Novel optimum contribution selection methods accounting for conflicting objectives in breeding programs for livestock breeds with historical migration.;citation_pages=45;citation_volume=49;citation_year=2017" />
<meta name="citation_reference" content="citation_author=R. Wellmann;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=R. Wellmann;citation_author=S. Hartwig;citation_author=J. Bennewitz;citation_title=Optimum contribution selection for conserved populations with historic migration.;citation_pages=34-34;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=23153196" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Z.-Q. Weng;citation_author=M. Saatchi;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=D. J. Garrick;citation_title=Recombination locations and rates in beef cattle assessed from parent-offspring pairs.;citation_pages=34;citation_volume=46;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=J. A. Woolliams;citation_author=P. Berg;citation_author=B. S. Dagnachew;citation_author=T. H. E. Meuwissen;citation_title=Genetic contributions and their optimization.;citation_pages=89-99;citation_volume=132;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
<meta name="twitter:description" content="The long-term performance of different selection strategies was evaluated via simulation using the example of a local cattle breed, German Angler cattle. Different optimum contribution selection (OCS) approaches to maximize genetic gain were compared to a reference scenario without selection and truncation selection. The kinships and migrant contribution (MC) were estimated from genomic data. Truncation selection achieved the highest genetic gain but decreased diversity considerably at native alleles. It also caused the highest increase in MCs. Traditional OCS, which only constrains kinship, achieved almost the same genetic gain but also caused a small increase of MC and remarkably reduced the diversity of native alleles. When MC was required not to increase and the increase of kinship at native alleles was restricted, the MC levels and the diversity at native alleles were well managed, and the genetic gain was only slightly reduced. However, genetic progress was substantially lower in the scenario that aimed to recover the original genetic background. Truncation selection and traditional OCS selection both reduce the genetic originality of breeds with historical introgression. The inclusion of MC and kinship at native alleles as additional constraints in OCS showed great potential for conservation. Recovery of the original genetic background is possible but requires many generations of selection and reduces the genetic progress in performance traits. Hence, constraining MCs at their current values can be recommended to avoid further reduction of genetic originality." />
<meta name="og-title" property="og:title" content="Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/12/4009" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The long-term performance of different selection strategies was evaluated via simulation using the example of a local cattle breed, German Angler cattle. Different optimum contribution selection (OCS) approaches to maximize genetic gain were compared to a reference scenario without selection and truncation selection. The kinships and migrant contribution (MC) were estimated from genomic data. Truncation selection achieved the highest genetic gain but decreased diversity considerably at native alleles. It also caused the highest increase in MCs. Traditional OCS, which only constrains kinship, achieved almost the same genetic gain but also caused a small increase of MC and remarkably reduced the diversity of native alleles. When MC was required not to increase and the increase of kinship at native alleles was restricted, the MC levels and the diversity at native alleles were well managed, and the genetic gain was only slightly reduced. However, genetic progress was substantially lower in the scenario that aimed to recover the original genetic background. Truncation selection and traditional OCS selection both reduce the genetic originality of breeds with historical introgression. The inclusion of MC and kinship at native alleles as additional constraints in OCS showed great potential for conservation. Recovery of the original genetic background is possible but requires many generations of selection and reduces the genetic progress in performance traits. Hence, constraining MCs at their current values can be recommended to avoid further reduction of genetic originality." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12.cover-source.jpg" />
    <title>Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6eb7nwSZ-QL8LKisYI6RWLy12Aqe03xNO2Lx9OYFyLg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487468","apath":"\/ggg\/7\/12\/4009.atom","pisa":"ggg;7\/12\/4009","ac":{"\/ggg\/7\/12\/4009.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/12\/4009.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/12\/4009"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/12\/4009"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/12\/4009":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225e5dnqdFmG\\\\\\\/g3P3i+Xo5VIW+YCQLvlgIcH7mMA5Bf15e9JBJvhUoI+aPph4v2wx4GteIlJKhqSTAMw4ZP8EYtlfFdMSobL51pcfcCkIWlYSmpN4qkBrTy8Ni8CgYRFzpdD3TeFl9wgnG4kpSebd6bYyft3Fka6+pc\\\\\\\/8r6\\\\\\\/Cp66bdKByzFOMI5LFYdxQIbIZG7KT4P82rH+78Ukzl1M2k1pzV7s+9PYy+ryP8gfhk6epEG4OMGVKJz9Y+UhunlGsDyzQp814nbb02eh3hHWLox0pwy5noPvyUkrpX7ykYajjhyne4yBm7vtOtgLSJsNtvoLEe2PQ7y2M\\\\\\\/R2KW0A5wVgfxD5s3\\\\\\\/3Imc\\\\\\\/AfpD4EO0emDM0rFMr8z8HbFtRsQj25w3m4jWJaLoSSe7GnGrBWvjkJzjXO7om+wKS8idJUsUUnCRlg1DCEXodwaJaJaRMSSmxowOFq05cqmFGEE\\\\\\\/FZyeCL\\\\\\\/zEM7\\\\\\\/2RYaFO4GqWGGEqM7KkPHvS24eyskbxYDgojiS3EbxXwgZVO4Jn7VsMPctXagNbC7w1cMMYwixR+a+rqWLfLrIQv6vSrVgivcOUQ9NGkq4vfmT0IthNIyJ3gC9cStJJxHx\\\\\\\/Cp9GJuy\\\\\\\/Yqk193th5z2GDjCGQNL\\\\\\\/kN\\\\\\\/hxNqZOjwrwWi51iqeyQXS9N1G894MH9Tg4\\\\\\\/\\\\\\\/t6Mac20TFZIGd\\\\\\\/dFoH5F2ExMjIhoxD9K5GiBFyuogq2Y4DEYCVjbZOzGzwfve\\\\\\\/07dgLNR5BiltkZdVaxf1SBezHxY+Nh2Da27qKl3tftaCMSX6cDZ\\\\\\\/nNMUzJy67o0BLhVxgPvEtB95d\\\\\\\/I8tZy5GnZbHmlDNMHSr2eQoCea5x+khqGzPIMr80reOd7hBFbRt+SXGJo9pj2wx9HjuKwUUvKnsXi+oVgbECtjt2dImaVOUf+HrpLAn+DYHjRSyZyIqfmXAj51a\\\\\\\/cn9Fg9e63qS6slSwt6cBfUY9W\\\\\\\/k6Sq1ijPhkrjVlam9h\\\\\\\/0dlVGwTGFQ8+rjwdTCNmDOFBstEF13jjWH9QUfvuWfgjgAmOl61+BHS0++opNqwXoNpw+1WOACOyLyWAzqVfKq6urVkms5qN\\\\\\\/fB+CFaieEUWXV65RRZs0fmG3d6\\\\\\\/kKAMVVQwwoyk8u2ECNGDfcZXuTMihP1qLN78urNPaJJiMkgRfiPjmkNbdPSOA4CJnqbfXUJOt3oDejt2L5ijdzBuSJlWgwOcNCI\\\\\\\/9xnBKGV8\\\\\\\/iMESFKhUNAC1GTc9Vq31Gv1oVjfHK9l3Qfpgv5+02fXWHihMkbo\\\\\\\/cbD0QMDji3J\\\u0022,\\\u0022iv\\\u0022:\\\u0022dv1vldPSOTXU3FKzrvWZAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253c29e184af4602aa36fe9e4c7083c84491d8f9e7ca8dfba932e0a9bb60b0292\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PpTcMYmCc1WXjKGCxuzFPWUMSHaUyyJsCoz4NhtrivRPIRhjretRpDFl92ID+AJdr8RM6C2Khuudv8NUgRzkMCvXgnjeD\\\\\\\/Tk5uxqhRJHKfbwbqWaluzUaD2zI7jVjmsGm5PRUIwLs0RbwVIW6p4h0Ircp2ZRB38kTTwuFVAw+a9wlN3Mh69+Q3l9XhhyJt1XvFD5QgEH6Gsr\\\\\\\/atoLqCCn2LyZzcy0oDK1GF0XsQHDa8kyDLsg30GCU8+j\\\\\\\/z7eryoEikousnVIJoaltMNdS+q+2HsG4Rh025s3zjG70vwfJYLVSveGD5qOhQPu2tQj1315AmpiG0bDa8izHb6rOqp7uRpnBnqkUlYU9\\\\\\\/3pn3K1hrrlotDEkjkSZV7USp+Ed2A94FjREMA+Fs4sKVihNBe\\\\\\\/kLo8MmOsKEmNxHq+4GBnm\\\\\\\/TA1choMY30GY8Hp9zSoo2pRFuit8c1UUQwCNjewi4dQ+IZa\\\\\\\/w0F6cQtpExeRZqXNpfqLnEB30LC7sApZ8nVx+cWJP+7sQCrYYgDiQ2X9ERLgdyn9amvN3C2LECiTtOo0UKo\\\\\\\/hqn4aRQewfPACPh5VfQpFO0jJ5w25gGf\\\\\\\/6vGn5BRYakDWyS\\\\\\\/u7yFX39R+u8VwrlUL2Vq0n\\\\\\\/dAsaFAd84vUQpIO64Hj0vI252eMQqhpQOTYUS2hYpU3KzWDqiqghLR5cFY9nZHl\\\\\\\/8pFjpM10BiIyhpKe5oIPGiG0r1imALvMIapb2hgStu8MlVa6HR1R3ojEP5X2cvY9Eg58rWYhEwjh2vxtRCs30osg+MMr9ZwHl6FP8mCmazfbjhYLMwGDX2ciTQC4r8MrZKuDD47Tdk26EH0R42UpnNRWmGKW4IphsnS11a9sLREaY\\\\\\\/uNKi9da2PBrrxA+zvg0WhkFneLw1mCs2nLJYAqLnNdHp4eUWvzDuF8tluZhBFQOYy+qjiXVcbHJJixvgQjXoYux\\\\\\\/aoLQJzCh4cqF+avW3D0ADH5GV0f9uywkxBpfGPicuHJaxe4bqo0uZGP\\\\\\\/OKZQfK6KiPyjlJSY1lnoaZXguQzY3NxXzhdg1yU55te7xO+cCxgbqLOCxXGkRMNLk04y\\\\\\\/ufi7gqqnnOpEdpDdEbE4\\\\\\\/km8fySKzs9WEiSdhoP2y01rxjC1tjyz+cwtORrXhC5qqIWmvUBhRY+osmQV6mTZyrNaHzD2wU3csfu3aK1Gd+mG6f+wVr96C1+I1v3x1CBHhgwcl6PygnUyK0Y0GYuX3xVNiLKVcOE+O3QQfqou8Kr2qc0gzidrEdIRxOIJuE70R+Qp0h+kysdlI1y\\\\\\\/YymiDMoUOa3WL1SSF9OHjNm1NJd\\\\\\\/z6eG6ZK6ykMtZPJWZf8FHHBR7KB4FCQlJy6b49hQhn\\\\\\\/tqBjETxnUJEnkTDW1OEs8wm7gyRrGSGRgsf+MeXpJ\\\\\\\/WhHbvkkk39uvFfzOPkQgcMGTo4R5OL4ECmze3RC3YTKOggHj9tjzgHDe+XI7IudZWl6ixJmZEHXK+jlnb7HWsq+iw5uOWLVBw+fABPQx2Bs988vd20dkKh9rJupV9XC3pSN+MFu2GFha6C0ez6hXBjbMBprNpDOfsC2ZoQjZpO2Jt8rflfMBEbFai8ompDuaMxPLPr7\\\\\\\/2SbBWai\\\\\\\/p5rR+C11LDHDxkiDBcDhRaD+iJVZcFHQWdsh6yJsa\\\\\\\/YE1EZBxFB\\\\\\\/Q9O+\\\\\\\/1JohNVyF9O8l3\\\\\\\/gyxcks+pW89tJiQMFMoRlyeVA1SD4oxy\\\\\\\/beZcVC\\\\\\\/ZjM\\\\\\\/ZA5EQp2tMlB6xYgv3O3HlrTIyscbixALwGyb4\\\\\\\/iZqA9AnsIZhB06tGzQhz1aM85b+ugtM70gpM6XligNokGSgMy4gSgLUVNSHtkk+u+zbJH+waq7qQnEAGTDnkdPWUG8l6yknAlIdY8ACYxrfaDXsLy3wNt5Grftn+Ni4HDdAOXhEBDgFn9shtOm\\\\\\\/fKtpvgLjRYaHnbeBxA9NSq1NtLYQe6V87KQ5+PUhI89GOx\\\\\\\/nqsJNy0PAW+vhM3ZZ3eQ70MxzQLEaa+LGKRMILekqGFsWQ0iIiYFYIhf7dNr5JXjUsbOJVbddoNYD20tWbMESONPr4Z8Oky2dBbx8+D+ZkRyKPKkOwDxsFNMfHo7UtSqbRdM0nuTzEy4282zHscOqTsVnlgCcNDaOVA+rFq2NcfJaO8gvJtBEyrKe71RGgdgbD2fhlrWunp2saq049Wx+7QDWTMhM\\\\\\\/CtoTL2emg5oDJ6fwRPvnyCL4EASiU5wog0mWkvAMwuP9PL4H\\\\\\\/bpfZUREiDziHrCYtgKGdoNrKYV2lpvj0SrLsvZyRB0QLHCAMzXh8kNTypa9bxRpCBF0o68qZLxpczVmSXlikualjpVIBt7AIkMp0LznBVNXUL2iZf3N50zkKb5EchAqyEOKNcUWxsVoeLuG74VYUv4WXcEFQrpUf535QTZtDoBuTOEa6MyNHVHzlpsbAgfMzAcySk5aRRlk5mMLFR6BpcBZx5DhsDhOiIyA+g5lmDQTG0bGbN9EeBJJFgmwlWc3n0hyBBbPiEmiTgKLyoj5jCSuBQO8egu3Z6cK4vo6XkypqFbMzzprKijvLaLbtXUU+e9A568tTNspGoNRZDDTVHUiuk0DNlNTCzgmtlei7PX2OMyvJgOXEoVLxzxbazG1YntbuUH\\\\\\\/ePNKFEkaC32cCY2uEYUMI2jafMI3X9HjogZ05bVi5zW1BXCPWBz4hDhHxM\\\\\\\/kexOskoQFttP0ed2SlN66B647TGBxG1GzGkBhSyNDrS47JrqdmhYVhlAMg4YCh7z0M4cLsTD0IxW4jFwJgjv4Tb3TbnDNSXaCMcK0YG\\\\\\\/5Jz6UDgyYcgVKgSwpkfTizoWKwHH7Z5f8dcc3ezQLS5Eg64q35IQU6tNsX1EIIZrKfnTDk8HliMK\\\\\\\/3j30larnSWtiw1iEnCLZwc+AM0MteRaEz6p0XmOnOUdHKOyBj6Z1eZ9hxqQ0qwe22MKM2kYocgqkur3CPRu0jggua\\\\\\\/UJn9GXOqZNoT8tkXKPsJdH99cR4lkZjvv4R\\\\\\\/+fCyhN2S4MgL\\\\\\\/OfGoV74bjWSRdF56LF4HrV78Ffp0Snwi6sUqDLJhvHZVRAkHVS\\\\\\\/\\\\\\\/0yvCkviR3OKyYAfGUpGXAUeWzbf9uTCn\\\\\\\/lFICbWppllnLyWXvJab4QVisjoa7Y098nXsPE4CNWrTOY\\\\\\\/BOrjCGq66b675yc64pAfJX1eYEkyVHSqJJ8KANu4U2sD0kNm2itRhgwwdE3ffPEecrv6fxa88UcQDDifmUJyWvPq4FSdWeGNEQthyX6QGkHt\\\\\\\/Q0xuZYD5Im4UzpulLQy8QmnqLKhOp5kT0sIoy2MMEgdne9OU5LDJZ2qiHLpNBPMT\\\\\\\/p5PrtPU1HdBgKINAQkQwNJ2WuWCJHMDbd4P1Qu4j\\\\\\\/Lvqtwox1nptyuzB9ILfesMtFtK8q5GSNjGzK7hQER4+YaVaI+mTJQqkHJfXnToD6TtKsdPxH4seoQsxuxLyTJg9vwcwK+CarBPAsJjt8CykTmuSwqonD6DBpSoTAQKFOftCFTc6pjtyQrgZf+eS3s4U0k8jbLiO5RI36b7A0r433\\\\\\\/SSWsIQypJuMIIwFyFVYwPIYHlb8D42HtBTDrLGBbPp76jcKUFeQfqea9yyelwhEogqmkSgHc2xQPcqV6fRvnAQCyZ95TVgl040MNMqQ3Tj9PxZCsAqONxqdXZfZF6wJZ7K7eJDgAdaoHu3Hv82LpyXAbSTe6y5VttYoYg4AoBQwHha8RzrU8ie4s+hAjmgvsE6+otgj3furRsTQtVHFaS654mTrOVUsBKTaXpoNAcrjAKQU7984rBx6XmVTaiWDLHxWLnBMIoovoUKca0DuqKSFzUqUu0iY92KNaSK1Mf1OMuyI9\\\\\\\/N9pr96xXDTjHqLzhdl0ZjmyE3NHx0yMR+aZwQjgoetEx5UiBVKd9t+J9RFbTM9zH1+u7ZzguwQ0+hrGQWqVrFXbimj9Ea+1nxoj4SR9W9prokZajXf2NblFn\\\\\\\/JRaNxuS9t3vhz8jR9AHVLMCkoN16YqowoQR9xU32PfWU9k16WSAN45x+GlzDbG9dcPgeRxgslOjDVo5r5cfhTPM48gCF7O+mvEH5FEHPi8fZQcYHkk8gNbOgkqKwz88I3RpoJm6ivPZjaTKWAQe62rsxjBHiVv6Y8DsZUyzun18AV3gy3bVTT1BxSYzEap\\\\\\\/F7wVL3Ra4CPl9DLMPWmeDP\\\\\\\/hB82WRWW2SBvQFPt2gB\\\\\\\/xBwWG3PbKDxQiqyRx+uJLSJZqRvbCh\\\\\\\/m144Q1DaDH6WX+5WYNV3GqjZ694juUFG8wG\\\\\\\/q9HG4gbdsJESuGZkznAkrIlhAt6ADvqkowNfwuCJZ931VxTGiQzx8B3IZDANS\\\\\\\/9bmUFS+RlFv88KiLwbdmMxnsrSjvKh6BWFVAAc7rjwr0jpmBUULON+Gnp2raAjUoaYtttEWJFGl5IzZtKUzMogZJvH\\\\\\\/YyDCP\\\\\\\/8fLX\\\\\\\/xp9KTPzOas72QWA3LwxLH8XxBGZIoettXKEU9bjSw0gcAoUBhxaT3+e+zvCurXWJJdTlyQRofUukAXdlAnrcM78zIQr32eJMu0WwRw6m1qdMlDjWdSGCoYiFwI4BlqkjM5\\\\\\\/FtJbV0qp7cwJPLDHNsXauca6xjN\\\\\\\/UBe6VMC1Vb3iiK+FvrclwYz9uahQeBE6ikA9h\\\\\\\/Loc1v0hpxvv4ij0fRuYc8TzRo6PyugGW7lv\\\\\\\/36NTFqEVUAiozxXKfqOrcerf+zA6O6Eqgx65C9sWe\\\\\\\/KK86QJ3q3eJHMqGNV4DQsrfTPOfZjp3BHr0Kz\\\\\\\/cGcEBVQ9SWCZvc4GBAIY8PkoHB0CcoUCkKg3RmooNs6JTBEuMWUXTz2MGM7xiO1YIPyA9yHwABCfEl+x3HPdhmoje1T1cl1+eH5N2pXsBdpprfRuBxlXO+h7PgEGi8zA+lvTtiBz2tkURugPoOkExtJjpFDD6bcPRnmB2cwVxmme3BwtUFhfnz\\\\\\\/hj02F95jvYsOOmyGjCsumq8xE8DPR\\\\\\\/6vqwt25NvGNdC50fauh6k6AI2lL5TMQ513Hk1Z5SA6MUN7CQbrIDNZwZal+En0zw7Y85vvUYSQ48n7T9pxDr8aWLcs2zrsvcr\\\\\\\/bNRTmZod6Tvz\\\\\\\/u+74P+P82E0+GBVOzRZ79rggdWuZECHJHSn4O6XAs3Aamq4o3\\\\\\\/PLJFazGrycLzyCHmYLK4kso32DY3mn+V19z75V5CgfNkRnhyODuLGIR\\\\\\\/EIVWgl+FfONW2E5yRaU4I3hkhiuK06EsLBP++PvJ3RUSp5CdLaO9BWLvswW8ZRXHfSNvXR4NGihG1C\\\\\\\/+Rg7Sgy8+UlVj1FM5ycPNwIdNQOsd3DlGEq6Z3fztthCk4N2eZ4V5+uh2R4WKvN2y7IB6QZ95ldkzNKk+jPlXBlUECI2SAPp17bN1Fb\\\\\\\/j8t37rUralvFY+sSMwFB8RvggKR2UDMlydkOzWyxsy2+UDTe16Szzfv84Y+r2EhSMtM+HF8nvYvcrf2G0U0OCaURaeY4GXiDUuuvFf6p5Q\\\\\\\/enq54SiZCfKm7BXDjTZjTXHnQ2yQEzwe8jc+ebPtfIfS\\\\\\\/xsDzDM+ioxxldiHRGC4BIT6DsxeXaWp5LOCRETC1YtedqgwI048QCYUWeIlJboVlxdIJjnZ1bhvq2MVSSxcAw+cK3qVjrP1taYmCd2AP8R83EGV0BC8cYLsXmRQLLUbr8Idu3JzirmYUGLn9o\\\\\\\/9UNDR5dMmFXo6XQAvexly+8pKfvwxFvveneziwninLrBuST3a4eNTNv61foxVSZ4U9bmiNxbetFiMvmNrbYU\\\\\\\/fx7d4o89M6qIxUL9azkXN5fbHwQP8Ql\\\\\\\/HDH0qlJUkmXFYp5XgDAE90aT+dIxcG10\\\\\\\/s1WSHreePf0YQPoy\\\\\\\/8KO\\\\\\\/576oE\\\\\\\/wIcEksNmYhl138RSqz\\\\\\\/wR87Se0lSNdIhYWme46XcfJdiYWIigW4kWD2\\\\\\\/EXmjYqx7cy5ThmDHdT4nCBeGkvIvO7DFmsj8llD4TO84HvuO\\\\\\\/\\\\\\\/bRnqA3bQXdZma67m7EZHeBRW6JhUDAhhRQJuGEprhrf2\\\\\\\/\\\\\\\/zPfi\\\\\\\/kgI32SFDyVUBlbV8aGs3gywiCGre992KJK+J1Jd9wyZ8xLmuMxt3eRoZyH7zqZohvQrC4esq9eJljP7jV2EGCtuk0nJNZSodJTpE4AmRxFwXtLb21WHcE9NtspFAZl7\\\\\\\/+7gm56watVqyxy792v3D28I6+70kYZvm3Z+qHMFpoAXPD7t3jftaXqUYGAUWeSmMBtANbDMLxfLQHThyErQBau0pet7wKs3aBmRbl96Gj1DfFLtgWnZ7deaAdIY\\\\\\\/vASIyApXB2yxvATLJWuwkvDmao8rGSgzN946iqqlgNqIwXfolLBTEl2+xutPbhM9zG2uzekj4tYtwxtPpLoJHzPk8HbMglC7twHLLcfBHKZIhz0HMfiOO1fKApcy\\\\\\\/5OsdOAKQyXuUaPce+dP9QJxIoV6TtSsPU2tPUK12Hzna9mziJIiSlafalY0EDXWgFrTPK8ihRvE6j87NQ1KTmarrcU91HLn5WUgcunmaPImLPO1Nt4rcbe9DZfrCbPua7feTsURZqrcKPFHvPPaLlFA1wzCGBWl+XiXHM6t3fOJ4jS1s3kx4MNC59oXK0UZAJmHOGmOTHZhF+40Q0Dn86d9GnBDKVJcqrd7c5vqOdIUgFzQvtJ7cMccGcXTTEwN0mF4AfuQzyd+XdRe7cj85L2VYGO3ycd\\\\\\\/qnXI97GF0haaNjgTUi9le7DNfn+8Dn\\\\\\\/6jlicLmCLh7uAldor+VElRAigcW+HlfhG6Ymb6EJHYXnYRcDJhtP6DlPnJleIiaeEQ+zCLIBT3R73eshW9OV2KOsLIjf8N4nbRIABIzBdrzpPUlLcR2\\\\\\\/PANCx2IfYyV3sIMBV1wzEkPjlinBzn66cDWbuM5u4q5L70TuQriJ89TMtU8oTZM9HUQVjN9+9hfrerosEgZnc\\\\\\\/Fv9dCH6pcQN6UVL7KI\\\\\\\/xbPp0\\\\\\\/ce+44SsdEH8GYlKcBB++huSlCr\\\\\\\/WEC0lbWpJEvMu41fS1mhJa9wXgfQ1TLyBBL7oJIb0xVoIz\\\\\\\/WUfCV276\\\\\\\/NWwnnYtUv+3cllP6dZzHnR3yeVzUPctsK8CTrH4y\\\\\\\/vTPqUVP8YW4JfGnUmhnc5K0bgpQ9FTLk8TSARCg5Zto21dp9ufhlciYJV4EJiVQc7W1\\\\\\\/2p0IO8LRWUvbunwG4MmgsCF2V79Iu\\\\\\\/L1cDkIk4pmUAAEBEcp6f463S1U0LY8nO3+zHcP8kb2n8otuzNy0rpOSwby2L8qG\\\\\\\/yfYk3j2c0k9o0hQkK8ON1xIHJvovnQkbScVRw3P+1TUGohj3C12t1MAgmQ+8UMTDMUQAfJCcPtA9Msgh0T88hdfprTCHAuwasTlpy7C+E+FL7pMHNIQV+vGx+7A3E5qkK\\\\\\\/YwKfCHyoqBqmb1kKc86olckL47cYQzNhWuGXol+v6wuIbUGRXfoi6Wm9r5+xYl1+U7gAJQydFaMn68v\\\\\\\/EIrUS7pXW7OfG13DZnvj\\\\\\\/QHqVSBoYrN0eh\\\\\\\/agC\\\\\\\/708Ol0cGkHgI87+PsiS6G4xtC7qAh68D+7abRNzbXLqAryp7MF6WYHvNjYN8NzHtDLLhqnwa+FpzWh2rVIS0Xz1AoA7DL2qR0goL4Nf3LfX6N1hdt8Tbw5wI+9ta9SAB4Gxu8gmz4+j5TC4055NysR4Q\\\\\\\/ivcGVXMfiOcUsOQbRxLdYor4twUtV\\\\\\\/j8\\\\\\\/CYMSHlLECjD6ZbSHYx9SiHOwSJUQsOqfuQINbK55v5omSEFteVyL5szZ8CfDNVl8eOflUpQ2gyiPvOHUIxZIbB2qTDJYS\\\\\\\/LL8rYXaMtRMZMTZbSyPozXQAnBqWLJhXJ1n4eSYKYsEUPuTnrug0ofKyb795tUbBBbf9aHTbB1PpVvLTa04AR7p\\\\\\\/YWga7F5FZGSOdpud8Q1mio1JWEU3hwAkZ\\\\\\\/WxhAQWnI3ZUw6Adt1cN6XRy3Iw3pQYx+1jLwu3L0PuxTe+YJdCkdR+diDyD1GMpn7Ris+c5dZKi2yuiMBw4dVy6q7RvZokY5lRu33m1s+ZdoLeFMFJSbM6PrUyEBnWVns3\\\\\\\/9xkGQSm9EMCYndcmDNMBKAHl+SLktWckj9bXovjjj0xVWwA52+cBGjklfCW1AyDeVwoC3B3AaWF8kiZkweJdnZTzr17ySzcnXJJB7R4bgTn7p\\\\\\\/8gRVehazpQSoWbg9oOqPKuCW2VWgg\\\\\\\/iWlaI3f+J7EH8QVRVq+FcYzzRKOrk0\\\\\\\/BQiUfFtuLsxf7jBKYJ3W6D3Qbh0+l7AY4DTRRjIXOa7KoWXyZeTzAam3pVntbe76NrpbyBh9I5nIjX8nvYa0lYu0pTPRv5ztK2g3Cw\\\\\\\/zcj4uw7+R7yw+02NkzgEK9P+3P+oMOnaHWaZcWXHlHCeuB9yrtFcN6rOySvNFALLh0gv+wSt\\\\\\\/OYoHM0gwBxZK\\\\\\\/e5oQczvwcg4T2kslUcHKSuX\\\\\\\/3NgTdpN+NAfv6DGpP5JARqXiEMrOvcrSRkCah7TwATh7q4w+mILV2YNEhVvzGlb1mX9f49aM59meIIB4Hg3TwDsgRQswrhIlMq\\\\\\\/yhWihfJRZMXEKNe\\\\\\\/45rYCEp8z+iCyDWlZNmsD67Tot184chsdm+z\\\\\\\/P3U3Et7scINif419+pH4lPtOMZapFi2FNlv5NPqdGAIGLSDDuGqDPQSzqkq2PsEuCtZhTksD4NYl1MmA7QwiJz3C3rhpG3gWcvlshk+AmTGjXovGiSPNPKdFsGVcslv8Mkl954RHvn1gixUmJxYeSCfHfKdiGrbDcJqVw9T2qHgePoktktoQDns8KtTbTY1PjrGyzG2WEZ75B9k0EQ42zTqjsZXYvmFfUQtjTAQkWdWY0\\\\\\\/L1ik0IETDmt83laHLYVTdloE0kpWBOfG8FjIjQEehHuGBVyant7S4spiOQAUdY+LvaEBxXBJluQ3aCnarrveW6yLWlcMZowbZJpgfmmYGhDllhwo01cNNdswwEFejz6iCu\\\\\\\/7AXOCyhHTqugOLyHCvpHt+kiuJODFwYBHYVldv0h3GrV6OcxtsjZ3cwdjj1Jksqu9vFQQA+YHN3y4gGjGCCTY6kM140KbrPLwx4XJig9EbmlDhHhzEiz6oWP7biJZIwYud+jJweLMT3LpeENBxCPbv4qTEqk4545nE1nmbrZizweiVLEUHDkoRdW6a2ZWF1CZsmTTBAxn8cMf25g5Ctx2796aQBPpr4z6lkepwkwz+RRnro1V1XgDQHriu\\\\\\\/2hePEB+T46S0LO21F75MiB7gGko857mHCsoBs3hzNeMO6P2TytCFYJRVovunC0U884\\\\\\\/ZYah453hTIweTiLwLYhwAX0ZKnF+VFM0UK9UHjXUq1KvQKSLUwJZDbOUvXV957zw\\\\\\\/DcW9XH15II3Ey+uqiIWetcaz3eHwEpC64UFxD7vW3rQgdHZbPv3hsJNaWPiOJ8RupGShuvvIqzNHtG3oIL8GxZ6FpL6+X2N\\\\\\\/QRsink\\\\\\\/4nSMr64dIWdINa1gkfo3t9gG9UAuLV5rYKz9nw\\\\\\\/8t9WxfHiToddiQkLl1\\\\\\\/1qpypCcph9gebgT4MfGCC2XJHJqC0zvvYkWDW17BPdPDxdeEqQ1kXQnbUsIq9Crt+mSvbaBv10ktiAzLmfSiJkVk02nq0jXiqYEMuMP4BaI8VeEqYO0tg\\\\\\\/4i4i7Vxb6MKyArCuJzFBe1UIoql4sD2PGnghA9fEIysprJ\\\\\\\/WStm9vmz5bwwElvbtBzc9EjqZ6\\\\\\\/sppy0EyJpmQxFI7z7FognL1FoN+1ubIPvwjIv2sXDLZaHhVO7ZbiT60Xq2AFLQD0a2\\\\\\\/XYJV+WrRDxsKk4aMzXVlkTQ1KAwgeY+eTw0Sez8UhKtPIjX8AmjEwxHku6xY6gOlZjPp2gfVKiMcgkjhJkDhhd9D1g1Q55flAGaM\\\\\\\/TolwnBbGCxAoWSp28fa51WiYLwgJh8e0JtHEOah2TT6Mnr\\\\\\\/L8iH1f92++Q7azsX16y26a37rY3UeVr1JjWCvNbzYXo4hpAou9ScPa0+3G665ff0PLS52bJPVe+bJYHSKu4duFGj2FoWschoGnkAbfyDgRNJKzZH3wwLW3QqjB5gB9QYcqiDeXhqXhX9mpowm8kzJSxCI+8TgKaObdKgAHwbDWz\\\\\\\/mgyRc89Cy5ZQMyitXGts5kcPhp62yqZ1munQDlZRVU39yr46W4EKmuTo0jTZM7BWwMI5JU7qG24iH9YUvJeliEpziw7uZvpVbdqcO+Uxr9ndcSjIXrN0WN9paMWa4D7OWSrp1\\\\\\\/ia6yhc+8ClcHsxGxmcOKC+\\\\\\\/fxZdeUiOvZyNy2CGYH4ndtEiAjnFCbkRL6pivM1+AaxxaIFDJID9ZPvy+wOI+HRBT+QXP415DTtJwQTwi0U1HLd89XvVf6Kr0QkSoRV1QNOpl586CdtEWP3JUkOWW0C90LfBH5E8Ijs9hmxDfeO8mIFwG0IdEIUOGxwBRGALQxPCJ7ISwKG1FfKLl0L9ZP99nUTpkjhkB3fxS6Gfoz\\\\\\\/tMAmgr2OvMDrT363zR124oGO\\\\\\\/2ph6ph8UIDsY8DZHEyL5EAeRo3HdmPNGFXq6be84tk4Gvj5rRtZNpFlNQzg1zhTxqSR2JvinU6uDuC\\\\\\\/L0fiIxeENdidqJ0k+BGwsERzhywCDsvT2WeaPU6XTIbRo0VYbD74\\\\\\\/5kVeHtrUFv8J\\\\\\\/SI\\\\\\\/WLCx1t3+gtFHZ1dsoPYcq5ix4B8u6zBfPydsxlv7mLhr63lafGxDHtD50EPsEh6oUM\\\\\\\/xHVk7AMv9wBdHuKaBtxUHmldpfjZKUoILACx7uD45B8DoJFwDOcqWKnJnBeiBG1yu3QRzgGzaKligYiIdxjgsQ93XIkkmFiWjXQQoyT4m81WGBdTViKDlZjR4YGPoWOZ7Q0wvuhJTssabzpVhMca5ke3zMPx6FzzsrPtqFtU6vTiCSHS24xiG\\\\\\\/dNf9WmoKy17xXFq1FaaP+ZX9cS4kHer1A4TUBMYkgZ9wSC1DCXNhOxgg6jIoXVq5mFuB\\\\\\\/mqWZJml4nOy88AjOurajXVQwqMWhAHd\\\\\\\/2Do\\\\\\\/t9GsND3KhN1I14s9+DCS4EgZWNDAKzw4Q6EYNmCa0y\\\\\\\/7x1gtwDXmcNBkFa\\\\\\\/Si1ByrlLXUx1r8WZVK9oYfitx\\\\\\\/lWdox9mf\\\\\\\/SWPIm\\\\\\\/e9xGwTSqE2t1Ollwq4gZ+MX9Qf7SX4PJPVChfbUqYtvlFrfnT6n9ksv6Lu9eTKbP9Pe5nHyFv2tbHURZLSmxh4dxWqJs8DHWFfE1vpYi\\\\\\\/YI9Uw\\\\\\\/fUWYY\\\\\\\/QNxNwoAdSjR\\\\\\\/gcNmueV6lBR0nbvAT3rQDFJw8ZzAqMopMgfqt+z\\\\\\\/LEjqWaFLUIs0HlaEQyVdLPVlqKrVWMmJw\\\\\\\/vP9qDPf9KPXEMCY4cK5mY0KvCnc68LRrr\\\\\\\/JJL1q8ZkgV34uRKSsxPjCPf24VvoumDArRWXZTzmmgpWQx9041lpeDcmxqIqeVkvw2oUsWL6R8pR96AGgPll\\\\\\\/gTUf4Xmz9tTQvc0Uuffqctf+NEyQP8tEOlb\\\\\\\/0KkMv\\\\\\\/3xqkfNSOjyZIZ7f6U6CbBK6N+7XxcvwlakWBUpY6j1ZuyOU2O\\\\\\\/vyNzFo7oUlmOZluI2meoIb3n4hQR2FCRmNdOzzjtz9UdoMpb+RM0Jv6hwYiQ8nlM6i0XraL5YHEpkpRs\\\\\\\/yi8mkIOyeFMqEUWyT2xZ3JclazAbv7GoQ0rzkWaQq6TG6Zi5OQEpS5MbQiPxJB2Z61MyDvydqSJrptZfIapjt21IWQFQKEdv\\\\\\\/ab2hMoNoDJCS7ItQUfBUrGYtT9QsgQqLJ+GjyC1SG8I7xK7POWGz0cv8vb7wwej\\\\\\\/qeyN+XmEX8S7kUNSInRvBHVrftMt3BgMkYDGINgnXd0zFaQ7az1gNJAogRTIjjFsVAqdY6y4+L11q+emT+v+XPulKkZfB6tfvA8t\\\\\\\/EL8gpGY4LDQKHzypH02oNoBG49hsR1i2IbbUxlAz89ut7LFgFhyBnLmoB4X5LYH0snjbhYPjMvv4pXsLP\\\\\\\/OIo1BNgyDGrqd8VlnuN7aDVrm3HhITt1r65tnJ6xEnqJzRST+3WRkrZeUp4\\\\\\\/OhY8t7zEMltfNRd4ccyzIf9IwzBksq19icAPsk5cMXuf6qwoF3MHJeyZaSXV3TwPKR\\\\\\\/ETOqcwTuMVNoqZ+P1l84QZ76tuFRJlklf3lhP8dSlTJtcwi4MC\\\\\\\/V8NDEwgRTw88r4D7RZ3zxdXe\\\\\\\/dURTt+lBv2ReNz22YNGQNpbTu6GXfdjRxZ0ngO4aok7wj0mAqawcZao41vb\\\\\\\/p\\\\\\\/g\\\\\\\/2PCdfdjACEU8tpiyhyLcafKIJic\\\\\\\/Ue4RbIw0H5xedece2STeAe3bh+vnJAbfZMbRf+EVAICw+xiOQMFTuCmJq83sQgwSpXgNy56LeFvxh65LOlHzPiN7IMJQeqbbhAv\\\\\\\/HarcyUu+sus9QzFpmgmqcPx68yBVl+nbC3MCQqIKoOL9ah\\\\\\\/6C49x48Y3RNfCpToKEjkM7uy2K4LCZCNS2XQDIaqAXH1k4rf37qfQ2vwlHYrmTe\\\\\\\/bFHh\\\\\\\/s0SOCOC+ZP99xGw4ojpaFg9aGeZnX4RUuNJJT0v+4\\\\\\\/TL2NwnhsOzwRLWTQMBNzNFN2NNHmwknLBM+N7UAEKc08hy\\\\\\\/8pBRNQUiXE1zYNHq6AQH2IQc6GYbGkD431KCdLFMn+cZyGae8Y5U4\\\\\\\/WaOGKdy5eYdPOSjZscRNMRNy4RKbvHHrCS4r\\\\\\\/Pmx3fdehm1G0pwe5PcZdw\\\\\\\/FjW9CcHDUbn94CIuF1Ej1JfmBxH4txkuuKYdzeSDN5VWv4Yk+6uERFEgBQd6LTQRZmGPv05AslnMVam3Xskx7C2fZ8RkSVW598ZPvhWgSfZNgxoGEJuHP3KNUA2tlDMfSR\\\\\\\/gXS5Ar18jFtzOC2c1EQVezX5LcUxBmvJehfZdfWk5JIxMPfC\\\\\\\/D5OuLNCnTSnyTiNqm+ngHWHpa7uZ4PHhfhkNLoPzeQ2MH3WdWgg9c7nMHl+CkezfcZIAhodJXQHzHKCIWFUhbbB2DN5bMSMFyZ1CuAsccBiPqrrTd16vZESUguy0FeBitBy9cTX5wxJMpCofpCB9IJltUaRvt9H4zekh2+n21SUC\\\\\\\/d45TXjnCpkeIDpHWqjxju+F\\\\\\\/EtGSGVnCmSqFIGUEp2u1hQRURWt3QFpgjqQuy73Z1JeDqyuu0XtddD8i9t3mNqT9TGPXtPR6O01OhuhKrlgwUpj2faJaPIQ7+Gxtue8sVd7UDN4hoCEykHbLE4\\\\\\\/ddBdR\\\\\\\/LF8+WZXOl4CaJypEEPRD70zXXPgIxJUcl0GOqo5dp0af4HzUvWkZS\\\\\\\/ngoB+UmRLNCN7aq78ldZt6Tzi\\\\\\\/tlvi3MG8qE0jqTKUChXbn90KvZTLKeFWw8YpXhcmfolc7tY4b1mK+NVuHBrrKs2hjWQPfHKmsjWJ20ZTUVmeKxcOfuKGOUlHbd2Mwfc0Q03rYCmGc64KxBJKx7w2iq+TSBcHiv+Q5+s+xSXnj7DLSmRqNrPKnD8oTSx\\\\\\\/HqeO2tLtGGbRahPgD2oeEZh6jw7jmPGTMYhF3ygeUvRCP6SeCylo6\\\\\\\/A8oL0eGQRv2qWotmvv3salTGfL7drL8UfvTpoC\\\\\\\/f36Kj4cHOcUSialRtwGmhyxEldR2BD77U5lpxnYUC4\\\\\\\/SizQlUjfO76slCpeJ+B3GT7gePZ1\\\\\\\/fWS+5+lut3SjgH4pxu1sXvU+WEl1YlfzgZTiq2K\\\\\\\/8rwtdGHjjhch4et2KZPMYHwcZo2jG2wXcZb7Q61Wq5cMTgbLjVmCis3HNcVE1jWGEtzi1SyobZdsoVzQqmBlaYSUHeyT5iHv2CtOIgC97rxnJRe8jIsL+FbOzN5IAGch9UXfFLo97Ad3QWpN7NZkesgkzK5d+HBHIYv7uv9EYjIfT8kSzILelHE+AgZKd+NP\\\\\\\/iNr4qm7LqXOWgDrMV46WC3BpBOhElFKYfs8kmVzhJIfv4c9lRvFwz+6wAIUeOTcj9eslk9gSYZf2qDf1nnQA8mX+UOYA\\\\\\\/E16tN39F7mYc3WK3Vr4LDP03ac4PMXbLNWdk5+t6l+4vZxfRuiOV50\\\\\\\/KPiLmw1V\\\\\\\/h4uM\\\\\\\/hI6crC\\\\\\\/0w75fMBNEms2ty\\\\\\\/hFhRx+kk3dKtVXW2fUl7YGwOg+sXSW\\\\\\\/J6Eg5fhouiCZZj5Bfa\\\\\\\/vEjfB+TS3Uvc5JMkVlOj5K8Rnxr9FRkJDzyNnEiTicowTE0YwoUeMCRQ7A3WGOECjnP3utNy+3XXtwmxKvS1jaJFJIYLghUAdaj0vYGzOo16OJY4CaVlefHg57eWHlGvIxrQOzjqocfvwBCiLCarIentt6nXuPSafFmmCau3MDmhKyd1XOSAYC9ukSajhF05I+jg6tqOO8BPmEwvp+Fx9\\\\\\\/bg81tymGKtU34hxKDUd6cIHyIPgx8SOTJxfSFEMqdQxXCAjphtfb3YfIMDQSW7gRVid9l9VQA1FEztBhy0SH3FGTgCpNM0IWk9o494g8kUjpD2CGfBjIDcMOaGWQbtvTWoXSTA2NV1fH7Lp4\\\\\\\/mPkG9R4ON+9w7rI\\\\\\\/st2ZNB5\\\\\\\/1+MkCEY8qkzF\\\\\\\/5H9BfRpK11E9kf31JKW8fw8JB7jHO22Q8y8n+r\\\\\\\/4PO3+k6XFbkqETE8niat9Z\\\\\\\/ddNuQtA4j4v3jCfchftYLEaNkNa2slKYT9FXsxHMITTftWxf+erZQoCBIENbEFDNfkoNYvUQvzONVXCsjMoG97FqmVfpioMO56eL9Jm2a+EO543m2dVMjj9fMMyvUPoqqnsxPAhDHG3vy0Rkjnr017aj+h5T3czHv6wIDSUxwtGAgPObfNar0iEkYql4wgp+HYpg6bKs\\\\\\\/Qv2BvAoyWYivAWQRbrGmHA6MDhLYDp3yOgT7U7P2EItlQQtbyewp+4OAcq5oj8A+eQ0DuVcrT\\\\\\\/GHEbfGww\\\\\\\/NBV9ovACe\\\\\\\/u5w9k8f5nWR\\\\\\\/ks0p2Rjh0VnPkKbA0uTjaFWfW\\\\\\\/6\\\\\\\/MKEFzl1KpHDeF6djVaYPED5D2Xl3swfsruj0UYHs6FS2OYpSkxhQXzkTcMynixefduFizIovp0CyidK+Fl0e7U+OkGnSJbrBk\\\\\\\/6ZMoxPTMGHUgqg3Jt7DXW2gqZlKpc18GqhWDzmFmEJZBrkepemwEeTf1fcLEbQCLVJtNXp\\\\\\\/qODjrU7PPdFL+rOIr6UEPDW8YdIrP2uonvUpJegdhQuF9FgEMn6XPI+zPakzBLbhaPwDzlTESgTH8a8JzCBoyn3LD9g\\\\\\\/CvUxZhACsBhCv3XcvNPtzWnCVkyQVS3q5Z02bR\\\\\\\/+iS+BS6jaUkYpulOr7MxaG7sMK4f+wrfK9cGTvI7HSK1cRAQgDVpdnmd\\\\\\\/ZbZm8gwP1t8mswx260h8lHtW7PM8PVvJNr+EzAGA1yoOGNqvmWQH1x7bYkPAMCjy32QMD7KG1gULl7ELZTsMfrRww50C8KC8Eb0aUAdBmqFJFMLkGu\\\\\\\/HQHnW7ILykLB9rz9umefXRPJn7NQI59hy0XsmLO6YxonDoBfRTv1JhpqeEsrLBI0t\\\\\\\/EdgwQ8Lt+vDqq2LV+wKDqkzkjd1a+418x\\\\\\\/TfKP0hqOBO1hAeqOhsQuLrBZ4XlMJqb9Gz42062E1+xmxnwBLzHe2Bb4vQjalAdQa6sFPqO0XtQxjUduqGuzZqbMvw5LX9qNPitzPXF1dRdA3y0DUeRdT5lKWy2Sw6wBK9rRICr\\\\\\\/4UonB7bHlpI9VNcH86Uehm6v0A5nlgxC3fIw892wz2LS\\\\\\\/mzjEiVHThaj5T42TYGU7mxtyC6xM8A6XUUYYawJUfNgtATvSZotn3edsRXi1LCNB2+4nb908DwnyZgUCOWhdDhkS8fIyCgIradlKBJp5RZxg3MeB4baUrO\\\\\\\/3r2mh1LkVyXdamBNXRvbvaR02oCQMilHLkMCIQsmuZttFzNLJmDEeHRnclrpJ4bazfmvW33bJ1K+7891zSQoCr7mui40XfH1Wegsd7cuGa6gvSLAeJik3+weAvnqrEca0dHPuHmV7owdmYrUUUeeFxvZIuSCy0KFR9L1Fbear5Gol+UGT44rmPpUW8xTFXKhLZEugv+1oe5WuS++lJd5otcSKYiEve3yDRAU4GKmN9vMVLTlEznUw9IYSi0XlxZ0vqa3OwlFisH3fa3V6QK\\\\\\\/ijPfmJk+2G4\\\\\\\/JUr6mbOKsSWo5HfQClLRZieKn7wXaPZ9vOS74LM251gS8cOkzh6lQmrfJSH+3wsK7Q1wUr0LXflMyJ7Zlen4nkwNP4PodGe\\\\\\\/9FsGUMG4UvUVEfMr5o52bQVBS0KU7UAS2At9+k2v0CEgzVroBcce46lbLbRRh1ucI9YbYk6TkRYVt4lu7\\\\\\\/AALzawacV+YEdx35RUMC4TYL5MYHfSwpsjFpaxe6zfrC\\\\\\\/XWxv3BcoaGOFU0YrDUx++WZ+fTSqkSoKsb\\\\\\\/djapwzHMyB9++n2L+Ip4Z4WdbEzvTe3Oaxb6E+nbMMpg1WAH5DAnbA9QLOo8zgWcwVV61HtTY4uFynTa5I+T4ICbhBC1qQHTiMW+3N07f85a72Br5aSMfVyajLS+ueoCIHSKFPYjUesgibTJxRbZmzxjcZVgfe0joxx6\\\\\\\/9cqcNqdXH8LTuQ+uHBFiyk7fKtahC+aVMuLgP+gC77xTdOSQooe3q8kzOemxcXDHoET4RvrN+7XMj79vjjoT8Ke5Gfbrk3ZQ43XubfcXPQlJncw9DroiIQh+OegmePmuNJtUqTaC6PbgWQqC4aVF5WoU1NstiBwo7sVTCRbVjC+PPz+hahy+Bv8qRvFjWAdPPUi8MiGKZnh63xfdNLa9BKAewaUBcmKYsVOYxl7USfupfssXT5kMHs2\\\\\\\/vTJCP0pbw5iF9+b1dSZNxLDQitXc5cta1XxAQu1oYNdc04GesvUySPE6N+J5bl49MII1stmo6KSYdZCoX2yq3z54LystmwB982DrGq3gOJgspQ2bUESEdPnseevK6eqRwj2IyXHabak3H7tNrDStN9XoL3h5fxDV3Km2ZYWnPqxwNmtHvuXGFgGZDuz66SAgkTyQrtd14f9I+g7J6Z13Ryk8l5mQmserqmJbq6EoHXbDvgyfLC2GX4PPP+H+y7XqB6GF70veyXpjXSi4oflqkfEP\\\\\\\/Pb87NeCM\\\\\\\/9WCsy56NOUVng8qf\\\\\\\/QfgSePLn2QgzuhlArU0aMOlgqpsBt3NznlDd0DvLAh5tdvq8y17lo9TXcbSbis+TQYDOaVSLMfjDHiPUAUIZ2XEO50ko6ab0CwRdeaexz5fcHFo3PWMTuOHtDZEHWFEjAkmhm8nfQr9KCfkHGASU1SYAT5sRkS1YP0K6RY30+CUw3VYVhsgbZGkXedQcWz6xyNR6ZZc8LJS6EX21mgNS4gvkuB4fWnNVfsrC7zE305WRN7zeH9KadSjPCMqoAE7OJVuf7h8HQiID6uTdf7BVTLkwcYT7mChnVr\\\\\\\/G1CQjIHe4oqcXKlZW6wzPYFOdJI1ZBU7i7vUsIoTr5FE6z37bOc4HfdRIKS+Z3BJ3TwyTinXKA6qfPGJqWXEjE6jIyvRcBbBXvaPV2KAQe+ukyjSd7yqp29EjcgaRYZVasyoKiMlgKNLOUQAOoDCqz5\\\\\\\/XB8EwqnG5TOnaRcmkJbWxP1Y\\\\\\\/6ZQcAlVDM4ET9vO3LiAFywFqBfpF\\\\\\\/VIJs5LonN3lZC\\\\\\\/bIXBKFjtm5qCQE\\\\\\\/RET\\\\\\\/WX26geHgPH7Fveqf+\\\\\\\/4glN0grxs0hpl0tqIxC7Ua0ol8S4q3NcVFpnen+r7PcZP0AtYO40vVOd3KS0knTi7bKytvFvVxocgWpQZETTS3qB569jVfwCs0EGVb\\\\\\\/86sPeo6besFaVuNizXOkd6w1AAVMPJMTp1V\\\\\\\/HCg6f0uHxRxxAg6Zrw22p2OuKEuTWeJKEoaCZp+up\\\\\\\/h3GzPKdE4+OShb6LX1XSukf\\\\\\\/AzGPRbeTJN\\\\\\\/USUUkdFpxEMDs7FmUpLbgIspo6ewS7IiH4d8YCfo35xRHbkmCyFqJd4o8NDnsP\\\\\\\/5+DAUmoyC1PQvlDCSwgB\\\\\\\/E5Irx491I7BfljQ\\\\\\\/6XytydmeNFAdBI6ucp1P\\\\\\\/30j+iulnVx46GERPuU9TMRo1E\\\\\\\/4DX8qC1P\\\\\\\/r+f7tCWRULcqh9ad4eFQwFe9RQXoyrmVYTbTvMQwhpBddqxGBzObkgYXONkNIHd4w+t6IknNITxtJmlXlFEBUOy9NwqVGUfNT5Txn4ln9iLn21hYgE9FrxrjMcQ\\\\\\\/3VWkTPTmQiw0qJpeTy9x2HfZ9\\\\\\\/UA1s0+\\\\\\\/fY7HedSXFCSZGU7gQFXUPASYkaz1e+WSF2lQvXrShJeQwJzh5wn9bL4VytsF+TQBZqzacHXbqKTmkEMtMP9l2dEBgH4NjZ5OsUMrK8o1rf97DPPT58vRZPMVYNM5xra5ACzv5rm4xp1XvV1B31bonGOegaP8R0AwFz+h8ZChweVt6uH3rnS616cwOBCnWJ8UG014wKN4PsNFQvvs53iqsI+cDp8sOYHuZ+9vom4TGk2gfqM73sd5x0IKZtfaXH+CwjZwkzF\\\\\\\/r+J4FTwzldyCKJ4Tttu3e9AFNpy1+YhNWrMHRG7JCN2\\\\\\\/TPBP0VY2ZfwrbJ4cg45lgA7AFQIxA0No5+Ga8WcWjmERuTmb9Ifz\\\\\\\/IxhI6SWWVVw4pJX742VRFAC9Na6X0HrDctH5OBDdsGZc79\\\\\\\/LLswY7ILIZw\\\\\\\/2f7OAcjsgjF8QuJFMzViBLXotaWFE46Rpoj1DWtxNwLvtcX3jaKKtuOzOlxCLtguKfKPIUWbzSX4t2yc9M2rnVNgM7RHaoIH4YkhNmt0GYC+3q2jDNrDDDJ5Zff26RCycynqkPxhvOu6oHWAYJPFK0kBZcxbYWnNT3mX2aPBccheAsqj6xpEktA+wxU2ECIat4Pc9dYQwRZqrEBcgfyubpspMuAAVgzNTgW0n9DwO6818ZeN6PKqYc+M6KBL0ukjlDAeb7otz\\\\\\\/1A7XRuPHaB8e782GVvwGKS87LxmRC+V6yg51w0nUwHV5XRqvt8PlU2b5HBiS2fvmARU8Mj03cKdGBN141lS2PSUdyPFotDJK22CTNSqnkqHBKdXEL2lk+s+wXWUPCa2p43L4V\\\\\\\/PpVwyU\\\\\\\/RSH04H\\\\\\\/mpxRHxmHi9uR9ijEiHLiaNtEQ0xW9sP8IaoXDP44XsVHVjpO3tY5t+rNE6km0WWJGzv\\\\\\\/oI8Wl7APCh4wq0q2wsIpVJwCdZ45Cep6sJM6ewoBEwpfOf7CSakC42NUgbm53B5ldtWFbKWoZTpZ+oGbeCSg\\\\\\\/75RxtwdTYImxOJFfp7hS\\\\\\\/LC+FYNzia25fh9EIZC2Akk24bK\\\\\\\/slnbdtvNsKjwYZC7bHZmG+IvYeUK2BPQBbYzQ6hT1SArcs7Zk+Gxu+G00zFG6b7Ev588BSAkISw3UhhbQ6xJNzGBnulOoWnlKydieg3LPZTKdZxUNV4T8A1cLNvZJlpNw8V0d5VucX6mkMK1+zqUMcqujKhE9i3i\\\\\\\/Kj2N+juGlUzo\\\\\\\/F8SzmZtZh5EcEih2deyfAk626K4YbNZ1NUs55CCugg7pXpwHXfuQCAIPyKP3J5mCTps935haaIIDXY8YbFt1ANq3NEWEoAvGXg9P0P\\\\\\\/CCLrmYMNIsTZPHG5OfhgK3TQ0sVf0EGQMGfO79YiZbKTHkZLjhDsmsrmPQHwFKIYn+qnjEUXwanqzGwDqSBtNxwZCRrLoNsRLcvUr9jyaabTzT9\\\\\\\/Emhmv00+kTiZMnW9D7oGiDRGFHBvTpm9UDlhnOS7wlMk22u+RqxLQXnPTQcUtJMUU9RUIywH0tMers4FUOo4gt5wMilkmNsMaJ+erGMsM15C+2FWiz12yBExrdrXe3HoV7dkzcZ5sIJgkfQtbKUfty9qfePfE7oASL9dn1w5M0aN6i4ZDkBUCiQEk4MYb6ZJaVIh+69qGWJrq9JFW6OT1PZ1uBIbSTjGEy+F9Edv0ZaMmOWxLmmmQ\\\\\\\/aGZtD14rF4zFMFdirb3ECeRstVGPaujmJGqqtku8D+mI0lFhPHZ2E7DpFlgxtus04fiLH61k4A+lffFuTIoWdwcd7esDu61ZWXpGw1kAUjX+UTfRYigWDHJp5C2uqDDc0RQMptyyNXtr\\\\\\\/UDb3qLogC7xJd5ozOW4A9KAga3haCeep50fD1Q4hQeYualI62NRj+Gnuqby0L08qKRAMdo8UmYGi4I0fPyP+QCHwcCHp+vGR5xJF9GWmurXW6cV8wIbS+ThsiDDKyygkwMPympSVvmSmWJizXspvBpXKUqaM5HCUC3jMpnLErZyitEtyPJlmCCYhC8BEdgWWw3lfvs8iXZZiebHx3f\\\\\\\/rQt3mSj4h1ao9+lRqWBjoXR+o59geVnRs6Xa0c5BFaABzN6KLJoiqjqkUAFghxvM4JPirfZoULDu8fihWhG4kF7PB3NGVJJ57BvJ7G6RKo6hw7VtWzOIAeegbNqZzQmrkwVpiQ1oaD9u91qZw1G9IOvY2JgvdSyvGPlGw0yBYnVz8daorfs3\\\\\\\/zDFEu7Iem\\\\\\\/2ECABaN0Yuktdm\\\\\\\/z27X70SaRqvvo6CQgxueBwEKFVU1Rwj\\\\\\\/ScHkemjzz\\\\\\\/HEiakPxrtZI0eVGa3r1wdKm+KGGY2y8xMxHuL0Q7FGyrz9wKjOWuHSXLuMWV\\\\\\\/cz+0tpOeLYZqBnqTT14qVh7e5\\\\\\\/f2KwWKThzojcAOBhEyQhH3gyZd8fVBgzIxIedaZbFwQJPvc+IHIm2SccsTZ76voKS\\\\\\\/bHDuxrPMctcTohu\\\\\\\/+mx38Mq8CGsllsKzEAajPTh+8LZi96hWcFwmDY6nLAYxLRjkbEKdCp1mSA+eGpnJTA+cWVShL\\\\\\\/Mx1gjK\\\\\\\/IsM2CfG5MMxRx0pMzTTbkhFDacMPbCPK\\\\\\\/BLH+Djv8J21A2+84vqJRpEyKZHMB3FH31r7kxCOgy+bGRqga27xa50JPLlufjKbsTO61+fXdnKPWBIxyXMCzg+eD0LyJSZ3v\\\\\\\/GY5Ff+P5bbaAa\\\\\\\/IjONAm14WfbEym+6BGDnrgX3j5gHzmQ0HuqVth7+HUEnXtYzWyu\\\\\\\/CAeHxkjjUUbIVHUs49BHf8kXtbYy2dzgh4TpB1e2PhkG0hnv4g1na25dSnZdQOeEtPVO0rgDCj0AdU0pJd4Ib2H1iXfKybxCtNhNWa4mvJZTfi3YSVeLOfniXnsgbAl+kLz9lOdwz2JmvZW2qtgIzJW1vMw69ZERrbVBlVR+LjygTTT7B\\\\\\\/U4o5303C43cmMWiJZJS7vaenxH95CGVbSsQBBVc0lnMNyYUyvJJ1ZNvp4zP8niOW7XMSUNayQDgmx57LI3JViQfN3eqYi83Y7tEJY33HACDG13Pt3gqvBj0aUUmXKBgoK4GBIC4g0Dknyn1b0hlwHpY7e9L78+0gHFlh0a4hR3\\\\\\\/fw4WQ+JZHeVN4DLtLG15TDOl7Y2TLY\\\\\\\/vRPVifVNPpXnJMWV\\\\\\\/+CZ8BhrD4gFlacQ9y5tMAfLKl7RXpqAmotDpk0G\\\\\\\/SlbHBXLENzcAnzveq329VdjYJDrshbdrl3TUTXYAoMSPNnTptHUrFrAeNveBK0oJUvB3EwzltFCVlfPK8+wXPRXU1vFIP8aIYBb9R2YE7Ucftybl1EarQE3Ajr2XEQRr1p4B8FS++gknVvICtVNBIfC+L\\\\\\\/TfHY5mKG0WKUdL11k78fAxF+CVcfcSmldquGqr1IL+b16lBYFoHHjRK081zN1IS9ChnovnhAiWexhpeOju\\\\\\\/fgDJ0iWb4Sad06deha7uo94qzMh+vPwbJeJoJ4zF8SZx0cuLwnvYg5O5vSuCa56ywLPlhD85ObVeie2ZB2L2H8BE7sCjVxmU11s0pYD92CCllhkKpB0ytEfb5xVdwRzyf0ru9yYJsjolRKEZp2GpMlqABWVv9GEHnowjBGsGGoMnZv9bq+K0fpIBbjozBqcdEANvZXNpbvxp4l0JdM\\\\\\\/G9WSgaen07S7KoM\\\\\\\/L9Pn62DzXSZCSqzIFKxXAX+qWf868tMi+CYAirYbbWO87AkDImVTKLv7jKDgr2knraKOgi0UMrDJrCCE9V7aRzi4ss1F2SAGQMN6WAqnsMm+D6x3qa70P6A+ON\\\\\\\/wnjh0wtZxNqjLojil8xVc+6RDosK5zWN+E45r5a2rmhLsmBQ3NL7TNTQx+gC0YNpKkYi58UKNFZXxjWeWvosjSxqgAj83vMjhIszoe+cWgp5OdY8J70vOUmY5NUu0OFf5CXT\\\\\\\/JyPSkppAwPMxEUiJKLfCXFD7NN3AZe3btGU648xLv1hxQdZdsWHCu+TzZpqBy7DIC6vj7SBBn2xAlhvEtkrYIo7goMVwhQ9lKUVvw9fMJjzFOPkFi+iQvLcna1yzcJ+bGz2i4oguI16wE\\\\\\\/AlAPDLfwzes2bOfFFXy6TB+7UueUK7tBF0s6w6quj49uLb8jeFgDAMLCPfqhj4rk5zpGO8fxlgvUg+tppn8aIOFnagW8ki7AvHVM03ZISAG1wU3d2z\\\\\\\/zQvh1diiqCKn3NaCFRXKBUxxYl1fb7VozdWWDsyviTs6CtbT7CSwGkg7yjivTSgsSxIzPPKhuSzxe7KOO\\\\\\\/T8kQDzBUrpUUdRvvp1CtpRhn2\\\\\\\/\\\\\\\/IoCqhQf3sj00CHZM5NWO96AbmxlWcOH6oy4dhOgPvmlr6LPhGbjBSPucn7rvYTlB0CokMyQLFEUL6Jon7sNSoaicMaKzhduX21BhQ8BL5rCNHGh\\\\\\\/glv1x8M5E9DJGGHCX8hqs3ymZUzlr0Us\\\\\\\/+gakLPniiootQxMwoYO9TTK5\\\\\\\/7c+RLOD5k10XoNNVqY\\\\\\\/YPSLF7A34rmHdVcAzb4ALUItioMY1cs\\\\\\\/7FNyZmX7RhLc6coer9x8ccq6+SY5LaLxPC6j3o7oG\\\\\\\/dyplCwHXQDBTNhL3wN9WQKVjKnctLry3FgXK8nHY+p16jAYhkjiyRGHrL5tYbXkqBfc\\\\\\\/KO45Nd7cb+GCTrQ6znazlzH2ojBmu1M28aHMJxy+4wy5\\\\\\\/6WI9Tl904Mn2LuHbA1vt\\\\\\\/+SQWPJFrJ+HeFHySEF9R5Soo1jkz7FNvxxItVDw3eKEiZ3h3OiP3Kfmkm+tUK+AAoBEtBE1X6SCR\\\\\\\/xNvu4mows9ema1rJ1ZGVZMHnyvaycwhlLme1Qzt3sT\\\\\\\/SVinnPacdZzzgzO9Kd7i4aWMmdMhVQua0HkhhDYXmrF2Hert1vZwHP10J3BKhqVL42\\\\\\\/r9k0xZoZI+HVSFX80YsMwtJgAZpx1qVnnhRNJynWshky2I9M7MzxmwndLn3f3EKPS5DCAXVza\\\\\\\/V3uPWhsRRufPT2VKmN3CV2aTtujnf4kIXI3UVKj8cUmMopxEt4lqU+HL9u5Mce01k+E2HRQk1QddSyZI8KOpz8nB\\\\\\\/UThHPwY67ySwyEbPXV4bLp07jaduSKQcx0lO+MXAtOtVQW33UrKdupwamJavynfAMembZMzZRASw8f316lvRz7j4phUIeAAlqmxpnLD8M5vy35GHOOfiSKqG\\\\\\\/RY+tsHJ8M\\\\\\\/WjcNOE\\\\\\\/\\\\\\\/bxFnsoVgNErUOe4S\\\\\\\/Aa9sVXmJnTquRAd1lziftl8dfPe340CaiI43u4m5ELkdOeKRdGfXkbbIHvGW9Ne38G+hM6kf7rVYj+KNFesMZfrGm2JK5O1i8bKLH3EVT66NnvCC4o5y9ThlfqmjJ+LQ+GWSrJ36LT8zQ+JVwxtxcfF7EioKT3lZTScast2JnHrTP64tNEgQN76aUJOl2KPBKcCJbc8DztLvJ47YDz06T0g1L5SAonJJ0+lYRcqQi0+Q86oG4rneHcDsmgVxVQho9gWRGXTVljUqWLngp2daz\\\\\\\/Sf0PHsMs9miuePg4VRTiy4ws15Pfp7P\\\\\\\/bqjoRU5uq0s5HVdu6XqV3TvIIprbJv1YiE6W\\\\\\\/NdE1mXCj2p4Rwf2qZejtb76Bvgrno9NsMCJ6vIYroqDS2\\\\\\\/WSiOKSNQ960WcrAB\\\\\\\/75MPZHrZEk9HhfjnTjfHnzyu7\\\\\\\/6cNiKZrN0monNsYaJbPX27PXpiBKy+6S2LhlpWL96FltKySwE3UzwRv7US3lbljBo2VGo74G7agRtACGEa1witIjZLJYDBWTxfmCqVSJ5R3umR7ZYsj100sfviNqdijYGrY42bn1Dx46dFS93MHn1YIRY84jcQFIZqhNpoeJ50Wdnf3scjDTMhnZCieFnOo5ZVuHrd9\\\\\\\/XZ6k+ZGREpioGc0pHdXIUz8sOk3dtFc3kOUq1ZMkrQc24aPyerJ61dmODwsVFULz01WVy7Xw+BFtnWsHuUyxMpH5iq+4\\\\\\\/064yTaHbc5TXod8WJr+ibXUDvYGrIdJo98CUh+IgDtHARfz0QTQk7LkAWpVBW1ZVRK\\\\\\\/YXLnoDZ4vuye9iG7Snb8sVuwMjfRo\\\\\\\/iwzUc+REwMM6vL98nsP\\\\\\\/15VvTTodDTQwVWtpTU\\\\\\\/THkpErL6KjxPzl11VvJPK6iSVkkJnH40i3M65cS3a5TfFcgxrCQ\\\\\\\/ij2lMrCILLuxuu6+CXHIZpKsFcshAoc7ivosYtS35QbwPfj4SgzAlU3BJUcH8hddflg6wUg28zrk9ZWbmxznrkr9jrBmKreS7Yz02DnjHsHrg9eARPuouym9aBSwSAMxzsfm5K6XCcYku0uepCHRmjKmI+f4K\\\\\\\/+53oYx1WTSFvYrBizZ002WM4l0yXXRhHuKMXGjDBvYLHUc\\\\\\\/\\\\\\\/Ha\\\\\\\/o4BM5LvxTEJk2xJhNErvt\\\\\\\/9ybv1O8NAgFEkn3FjbBpcv1r7Adp5nDcvD\\\\\\\/wJt92DSmUZnnUl47V77p6kCa2RkwpsQbrFGCGqXnHiGmYbSYK10AyB20rHgbXH3giwUPKAvrGfN1\\\\\\\/HEBqYVwBOKT2BOnoesyI8uNv7wyAevwOhx1UV29NX+ggfGTzP1rnuq3q58OqtzTsMjRdppsnsf6Xq2AnhR9F0RQQ9U5\\\\\\\/YUa7SfQBSsAm8WDajG+bT4ZUSOfC\\\\\\\/dDGpWyZOgrkZ1Xw9i+pbdyFsCbfkaDg5YUSmZx\\\\\\\/t\\\\\\\/olLwhFE2qf0ajN+J7YlBLsU0U4fyaimOuHJ5LcJUC\\\\\\\/HJhayHScIn\\\\\\\/luWvJdf\\\\\\\/OMiRon4f7RiMIzPLoV8vhFcNErdgC0md5jh7UZThJssA9zlO4h4ReMgqgt1GWzKinKRrA0DPbmUGKBcVZic28YuVZ4\\\\\\\/AKdMCdAtVvNk\\\\\\\/mU6ZAwfElwyjBsq62JhTtElnt6vMSaUyMr84HSEthwBXiAOMta5ZQR1ZVci++3e9bZM\\\\\\\/s7c7xLkN+ttNUD1YEXyFfEUyAvENdXXSXSw89sOJRxu10QnRKFCeSLY07403NvxEGchTW1cyanNWwxH\\\\\\\/CiMPW+VYR9B9YiP+Oydc8bowKbBirt6cOzkHDCZbpWRizR8PNJtjytE3PUnkUEQBNRsTqqW\\\\\\\/2hM++lN8oBar7rEqdfIrnNvPTXf7jDGeBp2vTFGyIodhDAK2SqiNoyVrlmPC5xZ6q+zbpXTZRRZHbJn4ih6i1HbN5PEFslRgaYgf\\\\\\\/z4rxfyQyc5GSiYi3iqIhTuDYJ\\\\\\\/TDRGDjBYN15fuXjY6ySQiKd71cC\\\\\\\/1pBUvlCn1NcnAKA0Mqpy8C67erMcVZWZZxXlmgU2LeGvlFoeS2j21fGS2V9KT7C6pEor8\\\\\\\/Ppt2OCB0NE4obHrxud49CPPi0BaMp+mn9vlf\\\\\\\/M1mVYeJGuISjQ0dLJzgy0SwXfn22YS+MYde\\\\\\\/KR4p3AlZMaR7z0c+UVYKdpVzio2QAoCfuUq0uspHrKTDs8DHyaN34ugWmwRVRDt5NTrETy5RVqSfT7GSTDnNrAywbLgt7efFGGkth54YbbxW6YrDfBG7hg6Y7YkfF3cO+Cq\\\\\\\/v\\\\\\\/3RkTIfDlG9hZ\\\\\\\/s7ob0v0Llq8zrn1G+sOBilgPQLVX7OcjeBJ041laeZRPAZURzK2woSb0aZ\\\\\\\/rhR7hxM4o3l275Tfv8XxPjEJcAnikckLxBIuGUS+F+0L1rxOyXT0dfrQSdR2uWd+mb6NRIF0Joa8SE9ZNI7mY\\\\\\\/jmhoFXr8k3uJ3gJu8izxdmtCwr5z1vKKD480Zzkm5YrrhBlyHCfTAwFoEXP5NnzPEQZgqrMi\\\\\\\/UnT9+M2\\\\\\\/ExdAto7+xGF8tWjc5tl9dW32O6JiHCzmlwbRXxO3NKGU5IPpVOxtp8qViBnZOdTDujcqE1XIxf2DtR6YHbZSRE\\\\\\\/rtTAg1H85eOvMr1PaeahrZu2mqOftAoI\\\\\\\/3QFq+xmh6QnZqqymXIEckJ0wMQjCCvrYXaLOESKuFEsOoiV\\\\\\\/huGB3hb3UqhQumXN\\\\\\\/\\\\\\\/+a3p2h+LtNq09RkR9oA8SbVyWf0QivKbbj4qR0ssFjuaDHpByb60MkzQa7A5wG1tTc2AKl+HCoQTpLyVAtwSjznFFCa4JdlZCaclx0UCItn\\\\\\\/HqYpvcqiOQNhsOZJiLDvK6vPPIHETx13riSi95OL5wU18Wz4QLdzrjXcmyIJAPasAFWX1WFaz80WArTjNo6k2Apf8uUbDYpdTI2KlagyODKWwJPi2voJJJzOv0y6Z4zMBjrqdgAmVCpphC1sWmN8hmkdO8SbhfnvguOwtzmlpO96RV8LGzuotPdrvT00B0DwRJMZJdoZYWI8uNjmbLczGIVKMtH8w9wHNpaYonffi9MkYIWv4\\\\\\\/J+PC\\\\\\\/ES39lQ84mqDQhGhfvS9tDDSMUoAbUxkgytpDEHR+APXSXJ4cRRVTWmkvmkoHhmBQrSUUnrN3huVgAFBeYXL0E8koci2DHf8NvHwlczjAN1fnXnEbh77hFJcThV8GdCv+gErtLdWQzsR+K0quZgQ8PDPPjCWKDNRsyDEZFpJ2QwBvIXcLJkBi6E6i4mRLcktxvs3\\\\\\\/mZ3b1Jf060\\\\\\\/MdnbBIYvryqCkHnpmcxPhojBX9gTOyIQ3G8rpn0AakRXfZGX+g1TiFlylzQuGsR+cPAUSURUKZMWrAbDqnxj1GzaMlJOzYILq1rIPVtknKzMlRq8rSxJ\\\\\\\/MQDjpyLfXG\\\\\\\/OQ+BC3dVOfLlMGpa3N6tY1uZLGFZ\\\\\\\/\\\\\\\/EHq8jXodEeqa3FvKS3qXracx9sQ4vXc5eOVcdxZIQHu\\\\\\\/SzyVSO0UGAymJFNA4s4+jZiZ+TUi0bdkWzMVkYoJapRkW9Cf5dHThuvV0sDn7rJ4VpT1pVlKtQIVEZaP+x4XYwYE5LuI3eAb2NHV1fbbMiaQ1hjOGNTdWNg1fLfZ3rV+knpET2\\\\\\\/SDpnKWMUJPq1cDUx3bjbW9d5y546uY7MYubuYEAPQZ1E9WEhfC3WZar4k\\\\\\\/6c9frcjxGf3s0ZPExh2KCI9I6fCXbhN5+X5eycstQknXNk9RXO5dAKhrmzThM2q3sCXt13lOEWNnD3sCp65t3r2yprERsu0OrVpFjCGsyQ2Cc5WUjqql\\\\\\\/ilB4vnJxNxo6d\\\\\\\/gltK4UP+iRoB3vVL8O67vq3c3kxCzJpl0E9kC1P5u+jXH\\\\\\\/COEGDE7h4xGA9MaYrhGTO08SlwKElmJG6+blnyAfN9vvbsmsnhJZ0nLQ8xD\\\\\\\/IbDOYpZMUbDdqZG5Ss7qTDcix1t\\\\\\\/DsDoxRyJ5NZHMP4ruALTB1ohv+hXE+SMfjjrs+fO4cYm1b3ZkBNOsSbuYQZ8h5pM05+Ej2YgEhb0oM\\\\\\\/y1jWoGZyEX+\\\\\\\/JewoDs8IFe5BNNtDceXmtPg0zV5Vqd2wHncIIa\\\\\\\/\\\\\\\/hARGrXxLQr2fFPQPMmx7O8\\\\\\\/v96KseuH4+spJ+RfnxUUFKHXHpaHThPakgzW\\\\\\\/bB0b06wjA9o3qQ7LPl\\\\\\\/VLssifM5+jNh9pMn9njWoCHK+wIII8OurSzuywfDvPFO2pTY7G43ZhNY\\\\\\\/7c1L4W+LdAdMbcus2fbeTVyeoJ0aZuP5mNRC9+G2wLW3K0AVVpTnqbt6HSMXuVHA+aQqbwFjxbKDr05LG2xjoV6ki0PZQSpa1+2e2g6mzJr6yiUFD33dVsGDJYdxqkVAPrsTVABLiQA3RwsmohYTW6EKLhXumtZYlq2jDqPCWNJJo8VQBuR7nl3oTCyO99UTFsD9K4lWu5VTOP8Wg1t09GIGzXACcy2rfxVwT5TNjG81rOCweQ5+noYLtCZk3Sb47sln1FLK0+D2SBdy\\\\\\\/ASMLxZtHW07KB76XNwGaYLmL90K6DqN32Wb5x3crMLuM9ogMnpG3Q9C7\\\\\\\/ju0g5jx+lWJv5Tj2GGDFJ1APWI0hOZsRzR\\\\\\\/pCNVt9PAHy6CaRiVRcBKuC6oAK1jo9TT8zK957cx4zeCLSSCwH0FRCfSHvFJUlmIAgwFpOfB+aQ0vjn8a\\\\\\\/rvZV8LtY9oZKKoFjM7njdLr2U0x0tIku3ra+bpIpCbJbVPwVPnDgkoTn+WZip0OeF\\\\\\\/YvKw2wTKv0RwLKMVPY9HTGcpN3ziZVPypWY+gjewODhufwxxVUY7Eus+bzKDyu3j8LhJ\\\\\\\/CTA8utzztvix2ILO7VFSuEqRIRf3q\\\\\\\/GtLXy0\\\\\\\/M5C9PpFgXOCZgyib1gdHpLW4jD3A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zV0QbdDAM7BCwjw6UMCPVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002294e1ba0c2452a2baa83d9249fda5d66a9507965fedb6756dc57643efcf3a4555\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HxS99wN51QpvxF+4oyStqlHqTIt4uB+Osj21vjlwj+S49dSasKzovZKii5Awky+tj0BKgceWELe\\\\\\\/hYbaPTCekJ4sLCnCUv8exLJkh7RKAa2gRmILNiRwTcJqcQZXvpttTFHZSOY2ZSCHRD\\\\\\\/5Zg0RqSAmb4G72WbqqFFpOum2kp9RqtAfHTAZTbDm6s7A36QFUDhmEmiwbd4aWRr8rg1XM0lOm3fbKYDG7qxWc3AbMRBLmwjebNg3x0VAGZTPEAJ35U\\\\\\\/5tkre04JVdyhaZnSTXNonx26BvpDF2QNF0GAS7bX28ex0xce3ZF0tA4BkQcX9AlnHoXPmYLC3ZJF7btc22q\\\\\\\/HyufS0HLn5oIx7gQ0RalwPVRWjCAI5GFCHxAEWjnhvalKcc9KfmhTpK\\\\\\\/itlU2LjsybX72JtjiP3bpeuQY8Xzi1kfnAtwd3kf1i0Pfvm7DTii0NQMlVjtkbiEAwaI7wu7ph0t8xQ1r2eI\\\\\\\/WgKsfjZE6g4szdRyg9o+XaGuHrDQgkYfL+8B8fCPe7kqiFqEHiHSKtAfGDLp8\\\\\\\/fdomdRHgJ8HZw+3UnSnSHFRqJrA\\\\\\\/XEXc6VUPMr0Ig5C3jkieWQ8MdohAKwjBPJDh9QJcBsTi01IBkJUaU6l7gWmQjqEggyEsQweOfWcJ2GZQmLsM602UcIHLmHO\\\\\\\/8iuZoBcfBeGY0EZNp96fSLG1YsBloVdhefxxJCCzMrQI02aiWOhFvhR\\\\\\\/tPuuGXAGZVM0zxRp2UrdP219+sGdH2FRs0OOrmve+YvLFvOVpc\\\\\\\/cAmz1RcAi4EqI2JWfXOi1xB4HDfgokqD6QYTMybHfoHuAUd455UFkwQew9Oe1FBAF0fFgGrjHD\\\\\\\/FgAL3CQJvMBJlmXmjbua+3O+v4e54aQX\\\\\\\/4rFQkfVDRcWYfu8zARKAetq7+rh2\\\\\\\/W+XVEssufFGNE\\\\\\\/d4mVkKrv6R7Q59txMfE90XxzAxyYFDoykeSLfLgbWQDv7q12WJtgQnfl7QOwIzz19jIV7rDYpe+1GX0OwF0lqKX1Rl\\\\\\\/MSt+1\\\\\\\/n+f6utN3r\\\\\\\/jFA58kuhTFdbdSTmzirGv00r1ap8Jj1EZjEgLLfRUNKjODGCoZcj+eOJSy9BCi8Dlv4Qwwg0u4onJRnIIJZFosv4vrIeOSvKIiH\\\\\\\/JMGoMnItk3jfIQTWqRjS4xKAHKBZdHWJUfyg7EiCUkAM51mWRo\\\\\\\/CBm4BFNknXy27KmA5FKCrMhCXl7M0hMtXAkh81f3W5QPYJ90\\\\\\\/FGMgybzgfuBlNNQQARGgYJ19hcnImYs+jFwGY8Oer47Gbo4KaYznCR+RdNO+AdvRQJ7gE3cGdMbDMc3YIduXlfF1nKiOCGVpNIhNbYwLAjbyJNzJrsVOsDn+yQJ03eFg6tIJk5bAKNpt8Kpn26CdFZhu0pYAScdoLbzqMB+JnpLHqLUlfj3P2skCKc6qi36iCEuLtX9JELHPcekm3j81RKSEgBC6I7pOQfRETfgwI5WzWvn1UYiqesTChh3AKtKjE2EfNA0EC89CA3Y4rPdwq+mvTNCw+6gjdV5u\\\\\\\/vdybao+\\\\\\\/fJsKxSfohICvqdI7KiUW3Dnf\\\\\\\/RiEaYzwLcUnxSnXso\\\\\\\/JYjlD+iOhp3A8vMCzXQBfPFIDcOGSciRqtfbIjP7njI4+1ZCPYi9lcxyMT4qwHwNnBmFP19eOhw\\\\\\\/sYA\\\\\\\/nk2RKdXHFClMtftdFCe4lGTeRUqtWofXs+XtSurGy9hUEIhvJ4ugFZckdJpvjiIAKqC8ijs3z3vDwcoLNa7ncH+aKcK6gwEu50LH4xNWdtr+urzSGnVRefhfu7kUelvHWC6v7IToTkPF0ltPUI\\\\\\\/0N7ZED5OVe\\\\\\\/azd2Bbtk0RpDy2AMP8peWLGTUl6sYEcna\\\\\\\/bNVhqimSSdeBBE7GEfCMhy+BF1gki+Ew2skIgvSyZWs9SLwZiFc6RtLtmKhP5Aegz5Gd6EftGhlSeFfFVXA8Zl0xj50sEYPwaQ5QLlKv+tTGcqNZYtTbn9T0b7bVNhtXfL70hFPN19xte\\\\\\\/KMP4l0GTmXDAA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022u1ZI7vPx64oQLgZGc7mtXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d1d83a561f0dd2501d52ab2e937b8303151733bdb6455ca3aab9f4a9c68baf61\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487468 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/4009" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MlaO-JLRu4bvCBoJtc_QkKbxqXOEKsDHuQy1M-IFv6Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/4009" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-M5xajOIMjoriHesYcYaWA2QPivjbr9_xIHUOg61qUX8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/3999" title="Understanding microRNA Regulation Involved in the Metamorphosis of the Veined Rapa Whelk (Rapana venosa)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/4019" title="G3 Reviewer Index 2017" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487468" id="node487468" data-pisa="ggg;7/12/4009" data-pisa-master="ggg;g3.117.300272" data-apath="/ggg/7/12/4009.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3756-6246" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yu Wang</span>, <span class="highwire-citation-author" data-delta="1">Dierck Segelke</span>, <span class="highwire-citation-author" data-delta="2">Reiner Emmerling</span>, <span class="highwire-citation-author" data-delta="3">Jörn Bennewitz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4748-8042" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robin Wellmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4009-4018;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300272 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487468" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yu Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yu%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3756-6246" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yu Wang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yu.wang@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">yu.wang@uni-hohenheim.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dierck Segelke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Vereinigte Informationssysteme Tierhaltung w.V., 27283 Verden, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dierck%2BSegelke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segelke%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADierck%2BSegelke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Reiner Emmerling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Breeding, Bavarian State Research Centre for Agriculture, 85586 Grub, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Reiner%2BEmmerling%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmerling%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AReiner%2BEmmerling%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jörn Bennewitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BBennewitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennewitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B6rn%2BBennewitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robin Wellmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robin%2BWellmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellmann%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobin%2BWellmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4748-8042" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robin Wellmann</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/12/4009" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487468" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/4009.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487468" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/4009.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487468" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/12/4009.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487468" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/12/4009.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487468" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487468/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The long-term performance of different selection strategies was evaluated via simulation using the example of a local cattle breed, German Angler cattle. Different optimum contribution selection (OCS) approaches to maximize genetic gain were compared to a reference scenario without selection and truncation selection. The kinships and migrant contribution (MC) were estimated from genomic data. Truncation selection achieved the highest genetic gain but decreased diversity considerably at native alleles. It also caused the highest increase in MCs. Traditional OCS, which only constrains kinship, achieved almost the same genetic gain but also caused a small increase of MC and remarkably reduced the diversity of native alleles. When MC was required not to increase and the increase of kinship at native alleles was restricted, the MC levels and the diversity at native alleles were well managed, and the genetic gain was only slightly reduced. However, genetic progress was substantially lower in the scenario that aimed to recover the original genetic background. Truncation selection and traditional OCS selection both reduce the genetic originality of breeds with historical introgression. The inclusion of MC and kinship at native alleles as additional constraints in OCS showed great potential for conservation. Recovery of the original genetic background is possible but requires many generations of selection and reduces the genetic progress in performance traits. Hence, constraining MCs at their current values can be recommended to avoid further reduction of genetic originality.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/optimum-contribution-selection" class="hw-term hw-article-keyword hw-article-keyword-optimum-contribution-selection" rel="nofollow">optimum contribution selection</a></li><li class="kwd"><a href="/keyword/conservation" class="hw-term hw-article-keyword hw-article-keyword-conservation" rel="nofollow">conservation</a></li><li class="kwd"><a href="/keyword/genetic-gain" class="hw-term hw-article-keyword hw-article-keyword-genetic-gain" rel="nofollow">genetic gain</a></li><li class="kwd"><a href="/keyword/migrant-contribution" class="hw-term hw-article-keyword hw-article-keyword-migrant-contribution" rel="nofollow">migrant contribution</a></li><li class="kwd"><a href="/keyword/runs-homozygosity" class="hw-term hw-article-keyword hw-article-keyword-runs-of-homozygosity" rel="nofollow">runs of homozygosity</a></li></ul><p id="p-3">Crossbreeding can have positive and negative consequences for managed livestock populations. The introgressive hybridization of breeds with high economic value is common to improve performance. Furthermore, the gene flow between populations can counteract the loss of genetic diversity and avoid inbreeding depression. However, it is possible that persistent introgression of genetic material causes breeds to become genetically extinct. For the management of local breeds with historical introgression, three conflicts have to be addressed, <em>i.e.</em>, the conflict between increasing genetic gain while managing the inbreeding level, the conflict between maintaining genetic diversity while controlling the loss of genetic uniqueness, and the conflict between increasing genetic gain while recovering the original genetic background. The traditional approach of OCS (traditional OCS) provides a solution to solve the first problem. It aims to maximize genetic gain while controlling the rate of inbreeding by optimizing the genetic contribution of each selection candidate to the next generation (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Meuwissen 1997</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Grundy <em>et al.</em> 1998</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Woolliams <em>et al.</em> 2015</a>). However, traditional OCS cannot solve the other problems. For breeds with historical introgression, although OCS is efficient for controlling the level of kinship and maintaining genetic diversity (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eynard <em>et al.</em> 2016</a>), this diversity may be caused by a large proportion of genetic contributions from other breeds. Additionally, although OCS is efficient in increasing genetic gain, this genetic gain may be achieved by sustained introgression with high-yielding breeds. The reduction of genetic uniqueness due to high MCs and the reduction of diversity at native alleles is a risk for the conservation of the genetic background of the breed. Apart from focusing on high breeding values, reducing MCs to recover genetic originality could also be included as a breeding objective. Advanced OCS approaches could effectively maintain native allele diversity and genetic originality, while ensuring genetic improvement by including MC and kinship at native alleles (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wellmann <em>et al.</em> 2012</a>) as additional constraints, which has been shown for OCS based on pedigree information (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2017</a>).</p><p id="p-4">High-density marker panels of single-nucleotide polymorphisms (SNPs) allow us to obtain more accurate estimates of kinship than pedigrees, as it is common for pedigrees to contain errors (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ron <em>et al.</em> 1996</a>). In addition, genotype-based kinship reflects the actual relatedness between two individuals, whereas pedigree-based estimates are only expectations (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Visscher <em>et al.</em> 2006</a>). Furthermore, genotype-based kinship is able to capture the relationships due to distant common ancestors that pedigree-based estimates fail to reflect. Thus, using genotype-based kinship is more efficient than using a pedigree-based approach for the conservation of local breeds (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Toro <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mészáros <em>et al.</em> 2015</a>), especially for the removal of undesirable genetic materials (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amador <em>et al.</em> 2013</a>). Currently, there are three approaches to estimate kinships from genome-wide SNPs. The first one is the molecular kinship, <em>i.e.</em>, the proportion of SNPs that are identical by state (IBS) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eding and Meuwissen 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Caballero and Toro 2002</a>). The second one is the genomic covariance between individuals computed from gene contents (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">VanRaden 2008</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yang <em>et al.</em> 2010</a>). The third one is the segment-based kinship computed from shared haplotype segments, which are also known as runs of homozygosity (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gusev <em>et al.</em> 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de Cara <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rodríguez-Ramilo <em>et al.</em> 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gómez-Romano <em>et al.</em> 2016</a>). Both molecular kinship and genomic relationship matrices have the disadvantage of being biased due to the preselection of markers included in the SNP panel (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nielsen 2000</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McTavish and Hillis 2015</a>). Moreover, increasing genetic diversity by reducing average molecular kinship drives allele frequencies toward 0.5 and increases the frequency of rare deleterious alleles. Thus, it accumulates deleterious variants in the genome and may reduce the fitness of the population (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de Cara <em>et al.</em> 2011</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">2013</a>). The use of segment-based kinship has been shown to provide a good compromise between maintaining diversity and fitness levels in populations. The estimate based on segments reflects recent identity by descent rather than IBS (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Keller <em>et al.</em> 2011</a>). In this study, the segment-based kinship will be used in the optimization process.</p><p id="p-5">Genomic information can also be used to estimate the breed composition of an individual (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Frkonja <em>et al.</em> 2012</a>). Software packages for predicting breed composition are usually based on either hidden Markov model clustering algorithms or maximum likelihood procedures (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baran <em>et al.</em> 2012</a>). Such analysis can be carried out by Admixture (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>) or Structure (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Pritchard <em>et al.</em> 2000</a>), where individuals are assumed to be unrelated, and linkage disequilibrium (LD) is not taken into account. Another approach is to assign haplotype segments to the breeds in which they have maximum frequency, which is carried out by <em>optiSel</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wellmann 2017</a>).</p><p id="p-6">The objective of this study was to evaluate the long-term performance of different genomic OCS strategies, using the example of a local cattle breed, by simulating several subsequent generations. The scenarios were compared not only with respect to the genetic gain but also with respect to parameters measuring genetic diversity and genetic uniqueness.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><p id="p-7">The dataset consisted of genotype information for 889 individuals belonging to five cattle breeds: 268 Angler, 200 Fleckvieh, 200 Holstein-Friesian, 200 Red Holstein, and 21 Norwegian Red. The targeted breed in this study was Angler cattle, which is a dual-purpose cattle breed with an emphasis on milk production. It is mainly located in the northern part of Germany (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bennewitz and Meuwissen 2005</a>). The reference breeds, which include animals from the nontargeted breeds, were only used for the identification of native haplotype segments in Angler cattle. Two hundred Fleckvieh animals were genotyped with the Illumina BovineHD BeadChip (HD), and the remaining animals were genotyped with the Illumina BovineSNP50 BeadChip (50K) with standard quality control parameters. SNPs that were not available for all breeds were discarded. Finally, 23,448 autosomal SNPs were used for the analysis. Haplotypes were phased for all breeds jointly as part of a larger dataset, and missing genotypes were imputed using BEAGLE software (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Browning and Browning 2007</a>). To visualize the relationship between Angler cattle and the other four breeds, principal component analysis (PCA) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Price <em>et al.</em> 2006</a>) was performed on the SNP genotypes using PLINK 1.9 software (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2015</a>).</p></div><div id="sec-3" class="subsection"><h3>Simulation</h3><p id="p-8">The simulations comprised two parts. First, a base population (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-1.gif"/></noscript></span></span>) was generated from the phased genotypes of the Angler cattle. Second, this base population was managed for the following 10 nonoverlapping generations in accordance with the respective scenario.</p><p id="p-9">The base population <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-2.gif"/></noscript></span></span> consisting of 1000 simulated individuals, was generated from genotypes of 131 Angler bulls and 137 Angler cows based on a random sampling of gametes. The animals from other breeds, which were used to identifying native segments, remained the same for each generation. The selection process started from generation <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-3.gif"/></noscript></span></span> For all scenarios, the optimum genetic contribution of each selection candidate (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-4.gif"/></noscript></span></span>) to the next generation was calculated for each generation. The corresponding number of offspring of each parent was generated, and mates were allocated randomly. Offspring received haplotypes from their parents via Mendelian inheritance, allowing recombination to occur according to the length of the chromosomes; <em>i.e.</em>, one crossover occurs on average on a chromosome of size 1 M (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Weng <em>et al.</em> 2014</a>). For all generations, the population size remained 1000 (500 males and 500 females). For each scenario, five replicates were simulated and the results presented are averages over replicates.</p><p id="p-10">A total of 1500 SNPs were sampled randomly without replacement to become quantitative trait loci (QTL). The QTL effects were sampled from a γ distribution with a shape parameter of 0.4 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Meuwissen <em>et al.</em> 2001</a>) and standardized afterward. The effect of each QTL had a 50% chance of being positive or negative. The highest positive QTL effects were assigned to SNPs that were more frequent in the reference breeds than in Angler cattle. Hence, the mean breeding value in Angler cattle was lower than the mean breeding value in the reference breeds that were used for introgression.</p><p id="p-11">The simulated true breeding value (TBV) of animal <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-5.gif"/></noscript></span></span> was calculated as the sum of all QTL effects:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-6.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-7.gif"/></noscript></span></span> is the number of QTL, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-8.gif"/></noscript></span></span> is the additive effect of QTL <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-10.gif"/></noscript></span></span> is the QTL genotype of individual j at locus <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-11.gif"/></noscript></span></span> The genotypes were coded as 0, 1, or 2, as the number of copies of the alternative allele. For each individual, an estimated breeding value (EBV) for total merit with the reliability of 0.75 was simulated as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-12.gif"/></noscript></span>,</span>where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-13.gif"/></noscript></span></span> is the mean of the breeding values of the corresponding generation, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-14.gif"/></noscript></span></span> is a residual term sampled from a normal distribution with mean 0, and variance <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-15.gif"/></noscript></span></span></p></div><div id="sec-4" class="subsection"><h3>MC, kinships, and diversity parameters</h3><p id="p-12">For calculating the kinship matrices and MC, the origin of each marker had to be determined for each haplotype from the breed of interest. A haplotype was classified to be native in Angler cattle at a particular marker position if the frequency of the segment containing the marker was sufficiently low in all reference breeds. Only haplotype segments consisting of ≥ 20 consecutive markers and a minimum length of 2.5 Mb were considered. A marker was classified to be native in Angler cattle if the frequency of the segment containing the marker was &lt; 0.01 in all reference breeds. The MC of each individual was calculated as the proportion of its genome that was not classified to be native. The mathematical definitions can be found in the Appendix. For identification of the origin of markers, the R package <em>optiSel</em> (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wellmann 2017</a>) was used.</p><p id="p-13">Two SNP-based kinship parameters were considered, which are denoted as <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-16.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-17.gif"/></noscript></span></span> Kinship <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-18.gif"/></noscript></span></span> between individual <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-19.gif"/></noscript></span></span> and individual <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-20.gif"/></noscript></span></span> (element of the matrix <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-21.gif"/></noscript></span></span>) is the probability that two alleles taken from a random position from randomly chosen haplotypes of both individuals belong to a shared segment, which is in accordance with <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">de Cara <em>et al.</em> (2013)</a>. The mean kinship <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-22.gif"/></noscript></span></span> for the offspring generation is estimated as <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-23.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-24.gif"/></noscript></span></span> is the vector of optimum genetic contributions of all selection candidates. In addition, average kinships among different breeds were calculated from a segment-based kinship matrix that included individuals from all breeds.</p><p id="p-14">For breeds with historical introgression, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wellmann <em>et al.</em> (2012)</a> proposed that kinship at native alleles should be restricted to preserve local breeds. The kinship <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-25.gif"/></noscript></span></span> is the conditional probability that two alleles taken at random from the population belong to a shared segment, given that they are native. For the computation of the segment-based kinship <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-26.gif"/></noscript></span></span> and the kinship at native alleles <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-27.gif"/></noscript></span></span>, we used R package <em>optiSel</em> (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wellmann 2017</a>). The corresponding pedigree-based kinships were referred to as <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-29.gif"/></noscript></span></span> in <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> (2017)</a>. The mathematical definitions can be found in the Appendix.</p><p id="p-15">Three additional genetic parameters were calculated to evaluate the level of genetic diversity of Angler cattle, <em>i.e.</em>, the average observed heterozygosity <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-30.gif"/></noscript></span></span> the variance of the TBVs (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-31.gif"/></noscript></span></span>), and the genic variance (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-32.gif"/></noscript></span></span>). The observed heterozygosity quantifies the amount of genetic variation due to polymorphic loci, which is an important parameter of estimating genetic variation within a population (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gregorius 1978</a>). We calculated the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-33.gif"/></noscript></span></span> of each generation in each scenario with software PLINK 1.9 (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2015</a>). The genic variance was calculated as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-34.gif"/></noscript></span>,</span>where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-35.gif"/></noscript></span></span> is the number of QTL, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-36.gif"/></noscript></span></span>is the allele frequency at locus <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-37.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-38.gif"/></noscript></span></span> is the additive effect of QTL <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-39.gif"/></noscript></span></span> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>).</p></div><div id="sec-5" class="subsection"><h3>Optimization scenarios</h3><p id="p-16">Except for the reference scenario, the objective of all the other scenarios was to maximize the genetic gain of the following generation, so the objective function was <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-40.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-41.gif"/></noscript></span></span> is a vector of the EBVs of all selection candidates. Three OCS scenarios were considered and compared to two non-OCS scenarios, <em>i.e.</em>, a reference scenario without selection and a truncation selection scenario.</p><div id="sec-6" class="subsection"><h4>Reference scenario (REF):</h4><p id="p-17">In this scenario, all animals were used as parents and each selection candidate had two offspring. Thus, no selection or optimization was done. The effective population size (<span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-42.gif"/></noscript></span></span>) is 2000, thus, the increase of kinship is negligible.</p></div><div id="sec-7" class="subsection"><h4>Truncation selection (TS):</h4><p id="p-18">Maintenance of an effective population size of 100 was envisaged, as recommended in <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Meuwissen (2009)</a>. Calculated from <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-43.gif"/></noscript></span></span> (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>), 26 bulls with the highest EBVs were selected for breeding to create the following generation by truncation selection. In this scenario, all selected bulls had equal contributions to the offspring. Note that the effective size in this scenario is expected to deviate slightly from 100 because the formula does not take into account how the individuals with highest breeding values are related.</p></div><div id="sec-8" class="subsection"><h4>Traditional OCS method (OCS-I):</h4><p id="p-19">To restrict the rate of inbreeding, the upper bound of kinship <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-44.gif"/></noscript></span></span> was defined as follows. Since the targeted effective population size was <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-45.gif"/></noscript></span></span>=100, the desired rate of inbreeding, which can be calculated from <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-46.gif"/></noscript></span></span> (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>), was 0.5% per generation. The threshold for <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-47.gif"/></noscript></span></span> of generation <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-48.gif"/></noscript></span></span> was calculated as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-49.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-50.gif"/></noscript></span></span> is the average kinship of the population in generation <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-51.gif"/></noscript></span></span></p></div><div id="sec-9" class="subsection"><h4>OCS with constraint on kinship <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-52.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-53.gif"/></noscript></span></span>, and MC (OCS-II):</h4><p id="p-20">The constraint of kinship <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-54.gif"/></noscript></span></span> was the same as in the OCS-I scenario. Additionally, constraints on conditional kinship <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-55.gif"/></noscript></span></span> and MC were applied. The upper bound threshold for <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-56.gif"/></noscript></span></span> in generation <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-57.gif"/></noscript></span></span> was calculated as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-58.gif"/></noscript></span>,</span>where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-59.gif"/></noscript></span></span> is the mean kinship at native alleles of the population at generation <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-60.gif"/></noscript></span></span> Additionally, we required that for each generation, the average level for the estimated MC did not exceed the average level in the base generation <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-61.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-62.gif"/></noscript></span></span>).</p></div><div id="sec-10" class="subsection"><h4>OCS with constraint on kinship <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-63.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-64.gif"/></noscript></span></span>, and reduced level of MC (OCS-III):</h4><p id="p-21">The upper bounds of kinship <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-65.gif"/></noscript></span></span> and of kinship <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-66.gif"/></noscript></span></span> were the same as in scenario OCS-II for each generation. Additionally, in this scenario, we required that the MC level estimated from haplotypes decreased by 3% per generation.</p><p id="p-22">Several reasonable conditions were made for all scenarios. The genetic contribution of a selection candidate <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-67.gif"/></noscript></span></span> expressed as the proportion of genetic material originating from this individual in the next generation, was assumed to be nonnegative <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-68.gif"/></noscript></span></span> In diploid species, each sex group contributes half of the genes to the gene pool. Thus, the sum of genetic contribution of all selection candidates of a sex was 0.5; <em>i.e.</em>, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-69.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-70.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-71.gif"/></noscript></span></span> are vectors for indicators of a candidate’s sex. For all OCS scenarios, optimization was done only for males. All females were assumed to have equal numbers of offspring. All 500 males were used as selection candidates, which reflects a breeding program with genomic selection in which a substantial number of the bull calves are genotyped.</p><p id="p-23">The specific values for each constraint are shown in Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1/FileS3.docx">File S3</a>. Solver “cccp” (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pfaff 2014</a>), which was called from the R package <em>optiSel</em> version 0.9.1(<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Wellmann 2017</a>), was used to solve the optimization problems. Five replicates per scenario were simulated and the results presented are averages across replicates.</p></div></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-24">The data used in this study are available as supplemental files. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1/FileS1.txt">File S1</a> contains SNP ID numbers and locations. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1/FileS2.zip">File S2</a> contains simulated genotypes for each individual of the Angler cattle base generation <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-72.gif"/></noscript></span></span></p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Analysis of base generation (<span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-73.gif"/></noscript></span></span>)</h3><p id="p-25">The simulated base generation reflects the structure of the genotyped animals well. PCA plots of both populations were almost identical (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1/FigureS1.jpg">Figure S1</a>). The first and second principal components (PC1 and PC2) separated animals in the simulated base generation according to their breed (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). PC1 explained 23.81% of the total variance and distinguished Fleckvieh from the other four breeds. Angler was separated from the Holstein family by PC2, which explained 15.85% of the total variance. Overlap existed between Holstein-Friesian and Red Holstein since Red Holstein is known to be a subpopulation of Holstein-Friesian.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the first two principal components (PCs) for the dataset of the simulated base population  The analysis was based on 1621 individuals and 23,448 single-nucleotide polymorphisms. Different colors and shapes represent individuals from different breeds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1138452173" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plot of the first two principal components (PCs) for the dataset of the simulated base population &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; The analysis was based on 1621 individuals and 23,448 single-nucleotide polymorphisms. Different colors and shapes represent individuals from different breeds.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/4009/F1.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/12/4009/F1.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487584" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Plot of the first two principal components (PCs) for the dataset of the simulated base population <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F1/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F1/embed/mml-math-74.gif"/></noscript></span></span> The analysis was based on 1621 individuals and 23,448 single-nucleotide polymorphisms. Different colors and shapes represent individuals from different breeds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Relationships within and between breeds are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The smallest average kinship within a breed was found in Angler (0.048). This shows that Angler has a higher genetic diversity and lower inbreeding than the other breeds. Angler had a close relationship with Red Holstein (0.039) and Holstein-Friesian (0.037), a moderately close relationship with Norwegian Red (0.017), and a distant relationship with Fleckvieh (0.004). This is in agreement with the estimated genetic contributions that the Angler breed has from other breeds, which were 0.448 from Holstein-Friesian and Red Holstein, 0.152 from Norwegian Red, and 0.021 from Fleckvieh (data not shown).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487597/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487597/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average kinship (<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T1/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T1/embed/mml-math-75.gif"/></noscript></span></span>) among five breeds of the simulated base generation <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T1/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T1/embed/mml-math-76.gif"/></noscript></span></span> based on shared segments</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">A basic statistical analysis of the simulated TBVs of animals based on each breed group is presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Angler and Fleckvieh had relatively low average TBVs, with a mean of 0.560 and 0.367, respectively. Holstein-Friesian, Norwegian Red, and Red Holstein had relatively high average TBVs, with means of 2.160, 2.390, and 2.431, respectively (as desired). As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1/FigureS2.jpg">Figure S2</a>, there was a positive correlation between MC and TBV in the base population of Angler cattle.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487528/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487528/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Basic statistics of the simulated true breeding values of each breed group of base generation <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T2/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T2/embed/mml-math-77.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Values of each parameter obtained in five scenarios</h3><p id="p-29">The mean and SD of the parameter values at the starting stage [base generation (<span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-78.gif"/></noscript></span></span>)] and final stage [10th generation (<span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-79.gif"/></noscript></span></span>)] for all scenarios are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The mean and SD were estimated from five replicates. The values of the corresponding parameter for all generations can be found in Tables S2–S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1/FileS3.docx">File S3</a>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487546/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487546/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Basic statistics of each parameter achieved in base generation <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T3/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T3/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T3/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/T3/embed/mml-math-81.gif"/></noscript></span></span> for each selection scenario</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>EBV and MC</h3><p id="p-32">Except for the reference scenario, the average EBV level of Angler cattle increased in varying degrees from generation to generation in all scenarios, which is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> (left). The EBV level remained stable at ∼0.558 in REF. The average increase of EBV was ∼0.9 genic SD per generation in TS and OCS-I. The average EBV in <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-94.gif"/></noscript></span></span> was very similar in both scenarios TS and OCS-I. The genetic gain was lower in scenario OCS-II, which achieved an average EBV of 2.757 in <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-95.gif"/></noscript></span></span> and was considerably lower in scenario OCS-III, for which the mean EBV was 1.825 in <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-96.gif"/></noscript></span></span></p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average estimated breeding values (left) and MC (right) achieved in each generation of each selection scenario.  kinship;  kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship  kinship , and MC; OCS-III, OCS with constraint on kinship  kinship , and reduced level of MC; REF, reference scenario; TS, truncation selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1138452173" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average estimated breeding values (left) and MC (right) achieved in each generation of each selection scenario. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-97&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-98&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-99&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-100&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and MC; OCS-III, OCS with constraint on kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-101&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-102&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and reduced level of MC; REF, reference scenario; TS, truncation selection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/4009/F2.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/12/4009/F2.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487537" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Average estimated breeding values (left) and MC (right) achieved in each generation of each selection scenario. <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-97.gif"/></noscript></span></span> kinship; <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-98.gif"/></noscript></span></span> kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-99.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-100.gif"/></noscript></span></span>, and MC; OCS-III, OCS with constraint on kinship <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-101.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F2/embed/mml-math-102.gif"/></noscript></span></span>, and reduced level of MC; REF, reference scenario; TS, truncation selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">MC decreased slightly from 0.622 to 0.587 in the reference scenario because old introgressed haplotype segments were split by crossing-over into smaller pieces and could no longer be detected in <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-103.gif"/></noscript></span></span> In contrast, with the increase of EBV in scenarios TS and OCS-I, the level of MC increased to a different extent (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, right). In scenario TS, MC increased from 0.622 in <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-104.gif"/></noscript></span></span> to 0.676 in <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-105.gif"/></noscript></span></span> and became stable in later generations. Similarly, in scenario OCS-I, MC increased to 0.647 at <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-106.gif"/></noscript></span></span> and became stable afterward. For scenarios OCS-II and OCS-III, MC was set as a constraint. Thus, the average MC values obtained in each generation were approximately equal to the threshold setting in the corresponding generation with a rather small SD; that is, the estimated MC remained 0.618 in OCS-II and decreased by 3% each generation in scenario OCS-III.</p></div><div id="sec-16" class="subsection"><h3>Kinship <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-107.gif"/></noscript></span></span> and kinship <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-108.gif"/></noscript></span></span></h3><p id="p-35">Kinship <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-109.gif"/></noscript></span></span> and kinship<span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-110.gif"/></noscript></span></span> increased from generation to generation to varying extents, except for scenario REF, which can be seen in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> (left:<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-111.gif"/></noscript></span></span> right: <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-112.gif"/></noscript></span></span>). Kinship <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-113.gif"/></noscript></span></span> had a small reduction in REF from 0.048 to 0.044, which was because old segments were split into smaller pieces, so after some generations, the pieces were no longer involved in shared segments. Kinship increased the most in scenario TS, which moved from 0.048 in <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-114.gif"/></noscript></span></span> to 0.115 in <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-115.gif"/></noscript></span></span> Kinship <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-116.gif"/></noscript></span></span> was set as a constraint in the other three scenarios. For scenario OCS-I, the <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-117.gif"/></noscript></span></span> value of each generation equals the corresponding value of the constraint setting. For scenario OCS-II, in generation <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-118.gif"/></noscript></span></span> the .<span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-119.gif"/></noscript></span></span> value increased to 0.085, which is lower than the constraint setting. The smallest mean kinship (0.073) was obtained for scenario OCS-III.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average kinship  (left) and kinship at native alleles  (right) achieved in each generation of each selection scenario.  kinship;  kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship  kinship , and MC; OCS-III, OCS with constraint on kinship  kinship , and reduced level of MC; REF, reference scenario; TS, truncation selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1138452173" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-120&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (left) and kinship at native alleles &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-121&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (right) achieved in each generation of each selection scenario. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-122&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-123&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-125&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and MC; OCS-III, OCS with constraint on kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-126&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-127&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and reduced level of MC; REF, reference scenario; TS, truncation selection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/4009/F3.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/12/4009/F3.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487572" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Average kinship <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-120.gif"/></noscript></span></span> (left) and kinship at native alleles <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-121.gif"/></noscript></span></span> (right) achieved in each generation of each selection scenario. <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-122.gif"/></noscript></span></span> kinship; <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-123.gif"/></noscript></span></span> kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-124.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-125.gif"/></noscript></span></span>, and MC; OCS-III, OCS with constraint on kinship <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-126.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F3/embed/mml-math-127.gif"/></noscript></span></span>, and reduced level of MC; REF, reference scenario; TS, truncation selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Estimated kinship <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-128.gif"/></noscript></span></span> decreased from 0.061 in <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-129.gif"/></noscript></span></span> to 0.048 in <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-130.gif"/></noscript></span></span> for the reference scenario because some old introgressed segments were split into small pieces by crossing-over, so the alleles included in the segments were classified as native and contributed to the estimated diversity at native alleles. Kinship <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-131.gif"/></noscript></span></span> increased faster in scenarios TS and OCS-I than kinship<span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-132.gif"/></noscript></span></span> The value increased from 0.061 in <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-133.gif"/></noscript></span></span> to 0.157 in TS and to 0.136 in OCS-I. For scenarios OCS-II and OCS-III, <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-134.gif"/></noscript></span></span> was set as a constraint parameter. In all generations of both scenarios, the <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-135.gif"/></noscript></span></span> values were equal to the corresponding constraint setting of <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-136.gif"/></noscript></span></span> with an SD close to zero.</p></div><div id="sec-17" class="subsection"><h3>Diversity parameters <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-137.gif"/></noscript></span></span></h3><p id="p-38">The diversity parameters are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> (left: <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-138.gif"/></noscript></span></span> middle: <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-139.gif"/></noscript></span></span> and right: <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-140.gif"/></noscript></span></span>). As expected, all diversity values in <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-141.gif"/></noscript></span></span> of REF are higher than the corresponding values of all the other scenarios. The value of <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-142.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-143.gif"/></noscript></span></span> remained nearly unchanged from <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-144.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-145.gif"/></noscript></span></span> The value of <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-146.gif"/></noscript></span></span> decreased considerably from 0.197 to 0.091 from <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-147.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-148.gif"/></noscript></span></span> which is still higher than the level of all the other scenarios (see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-149.gif"/></noscript></span></span> was larger than <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-150.gif"/></noscript></span></span> in <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-151.gif"/></noscript></span></span> which was due to the effects caused by different chromosomes being correlated.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average observed heterozygosity  (left), variance of true breeding values  (middle), and genetic variance  (right) achieved in each generation of each selection scenario.  kinship;  kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship  kinship , and MC; OCS-III, OCS with constraint on kinship  kinship , and reduced level of MC; REF, reference scenario; TS, truncation selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1138452173" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average observed heterozygosity &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-152&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (left), variance of true breeding values &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-153&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (middle), and genetic variance &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-154&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (right) achieved in each generation of each selection scenario. &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-156&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-157&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-158&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and MC; OCS-III, OCS with constraint on kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-159&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; kinship &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-160&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and reduced level of MC; REF, reference scenario; TS, truncation selection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/12/4009/F4.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/12/4009/F4.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/12/4009/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487538" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Average observed heterozygosity <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-152.gif"/></noscript></span></span> (left), variance of true breeding values <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-153.gif"/></noscript></span></span> (middle), and genetic variance <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-154.gif"/></noscript></span></span> (right) achieved in each generation of each selection scenario. <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-155.gif"/></noscript></span></span> kinship; <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-156.gif"/></noscript></span></span> kinship at native alleles; MC, migrant contribution; OCS, optimum contribution selection; OCS-I, traditional OCS method; OCS-II, OCS with constraint on kinship <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-157.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-158.gif"/></noscript></span></span>, and MC; OCS-III, OCS with constraint on kinship <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-159.gif"/></noscript></span></span> kinship <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/F4/embed/mml-math-160.gif"/></noscript></span></span>, and reduced level of MC; REF, reference scenario; TS, truncation selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">For all scenarios, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-161.gif"/></noscript></span></span> is relatively stable compared to <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-162.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-163.gif"/></noscript></span></span> The greatest reduction of <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-164.gif"/></noscript></span></span> was found in scenario TS, which moved from 0.367 in <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-165.gif"/></noscript></span></span> to 0.346 in <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-166.gif"/></noscript></span></span> A similar trend but a faster reduction showed the genic variance <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-167.gif"/></noscript></span></span> In scenario TS, the average <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-168.gif"/></noscript></span></span> decreased from 0.075 in <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-169.gif"/></noscript></span></span> to 0.049 in <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-170.gif"/></noscript></span></span> A higher <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-171.gif"/></noscript></span></span> value in <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-172.gif"/></noscript></span></span> was achieved in OCS-I (0.052) and OCS-II (0.054), and the highest genic variance was maintained in OCS-III (0.063).</p><p id="p-41">In all scenarios, the level of <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-173.gif"/></noscript></span></span> decreased considerably from <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-174.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-175.gif"/></noscript></span></span> Thereafter, it decreased at a slower rate and approached the level of the genic variance around <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-176.gif"/></noscript></span></span> For the scenarios with selection, the variance of TBVs in <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-177.gif"/></noscript></span></span> was very similar to the genic variance. It was 0.044 in TS, 0.049 in OCS-I, 0.056 in OCS-II, and the highest level was maintained in scenario OCS-III (0.065). In all scenarios, the average <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-178.gif"/></noscript></span></span> level was much higher than the average genic variance <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-179.gif"/></noscript></span></span> in the first generations. This can be seen in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1/FigureS3.jpg">Figure S3</a> using the example of scenario OCS-II.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-42">In this study, we evaluated the long-term performance of five different scenarios for maximizing genetic gain in the context of conserving breeds with historic introgression using the example of Angler cattle. MCs and kinships at native alleles based on shared haplotype segments were restricted in some scenarios. A large proportion of the Angler breed’s genetic background was contributed by other breeds, especially Holstein, which is in accordance with results obtained from pedigree records (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2017</a>). Truncation selection achieved the highest genetic gain among the five scenarios with the highest degree of reduction in genetic diversity. Traditional OCS (OCS-I) achieved a slightly lower genetic gain and slightly higher genetic diversity compared to truncation selection. However, both are inappropriate for the situation of Angler cattle, as both reduced the diversity at native alleles considerably and increased the MC. Constraining MC and kinship at native alleles enabled recovery of the genetic originality but also slowed the genetic progress in performance traits compared to truncation selection and traditional OCS.</p><div id="sec-19" class="subsection"><h3>Genetic progress <em>vs.</em> genetic conservation</h3><p id="p-43">Due to the protocol for simulating QTL effects, a positive correlation between MC and EBV was observed, which is in agreement with results obtained from the pedigree information (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2017</a>). The positive correlation persisted in all generations in all five scenarios (data not shown). There was no genetic progress in REF due to the absence of selection. Truncation selection and traditional OCS achieved similar genetic gain. When MC and kinship at native alleles were constrained, the genetic gain in performance traits was reduced. Hence, to achieve maximum genetic gain, it is essential to allow for the introgression of foreign genetic material.</p><p id="p-44">Maximizing genetic gain is not the only objective of a breeding program. To recover the genetic background of the original endangered population from admixtures, two goals must be set: maintain the genetic diversity at native alleles and remove the introgressed genomic material in the long run. The average MC of the population can be treated as a parameter for measuring genetic uniqueness. Among the five scenarios, truncation selection has the least ability to maintain genetic uniqueness. Although the situation improves in traditional OCS, the estimated MC level in <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-180.gif"/></noscript></span></span> is still higher than at the starting stage and higher than in the reference scenario without selection. This is in accordance with what we obtained from OCS based on pedigrees. The reason why traditional OCS did not cause a larger increase in MC is probably that most MCs were from related Holstein cattle. Thus, increasing genetic gain by increasing MCs would also increase the average kinship in Angler cattle, so restricting average kinship implicitly restricted the MC. Similarly, in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amador <em>et al.</em> (2011)</a>, traditional OCS did not eliminate any exogenous representation but kept the value constant, irrespective of the number of generations that had elapsed before management started. Genetic originality could be maintained with the OCS-II method, while genetic gain was only marginally lower, as in traditional OCS and truncation selection.</p><p id="p-45">In OCS-III, the EBV level kept increasing throughout all generations, even though the original genetic background was gradually reconstructed and the highest diversities were maintained with this method. Compared to OCS-II, the reduced genetic progress in OCS-III is directly linked to the strictness of the constraint MC setting. Due to the conflict between achieving genetic gain and maintaining genetic uniqueness, a breeding organization should choose MC constraint settings carefully to achieve both breeding purposes.</p><p id="p-46">In truncation selection, 26 sires with the highest breeding values were selected along with 500 dams to achieve an effective population size of 100 in each generation. However, the formula from which the number of selected sires was obtained did not take into account that the individuals with the highest breeding values were related because they had high genetic contributions from closely related Holstein ancestors. Thus, the rate of inbreeding in truncation selection was higher than in the OCS scenarios. Compared to truncation selection, traditional OCS has good performance in controlling inbreeding via restricting average relatedness in the offspring.</p></div><div id="sec-20" class="subsection"><h3>Different kinship estimators</h3><p id="p-47">The predictions from <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-181.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-182.gif"/></noscript></span></span> from <em>optiSel</em> were close to the values estimated from offspring haplotypes (results not shown). However, they were slightly larger because some segments were split by crossing-over into small pieces, so they did not contribute to the kinship estimated from offspring haplotypes. This indicates that the estimate obtained from offspring haplotypes is slightly biased. The rate of inbreeding estimated from segments remained 0.5% per generation in traditional OCS, in accordance with the constraint level setting.</p><p id="p-48">Kinship <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-183.gif"/></noscript></span></span> estimates the probability that randomly chosen alleles are IBD. However, it lacks the ability to distinguish whether the alleles originated from native or migrant ancestors. In scenario OCS-I, where <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-184.gif"/></noscript></span></span> was restricted, the increasing rate of <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-185.gif"/></noscript></span></span> was higher than the increasing rate of <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-186.gif"/></noscript></span></span> This suggests that restricting only <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-187.gif"/></noscript></span></span> had the consequence that diversity at introgressed segments was maintained, which tend to have higher breeding values. But a depletion of diversity at native segments could not be avoided. Because kinship and kinship at native alleles are correlated, restricting <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-188.gif"/></noscript></span></span> implicitly restricted <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-189.gif"/></noscript></span></span> so in scenarios OCS-II and OCS-III, the mean kinship <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-190.gif"/></noscript></span></span> was lower than the corresponding constraint setting. This suggests that the constraint for <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-191.gif"/></noscript></span></span> could be skipped if <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-192.gif"/></noscript></span></span> and MC are constrained. Similar results were obtained from pedigree information from <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> (2017)</a>.</p></div><div id="sec-21" class="subsection"><h3>MCs</h3><p id="p-49">In general, it must be distinguished whether MCs predominantly originate from closely related ancestors originating from a single high-yielding breed, or if different unrelated breeds have been used for upgrading. In the Angler breed, they predominantly originated from related Holstein ancestors, so reducing MC in OCS-III was meant to reduce the amount of genetic material contributed by Holstein cattle, which had a positive effect on the genetic diversity. Thus, the mean kinship <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-193.gif"/></noscript></span></span> in OCS-III was smaller than in all other scenarios.</p></div><div id="sec-22" class="subsection"><h3>Criteria for detecting shared segments</h3><p id="p-50">It has been suggested that the marker density of the SNP chip used, the minimum length of the shared segment, the number of genotyping errors allowed, and the minimum number of SNPs allowed in a single shared segment are likely to remarkably influence kinship estimates based on shared segments (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Peripolli <em>et al.</em> 2016</a>). However, to date, there is a lack of consensus in establishing the criteria for determining these parameters, which makes it difficult to compare results from different studies. In this paper, the minimum number of markers in a segment was 20. A shorter minimum length for shared segments allows detection of more ancient inbreeding from common ancestors occurring many generations back (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Curik <em>et al.</em> 2014</a>), but it also increases the probability that segments that are identical by chance are considered to be IBD. The minimum length of a segment was 2.50 Mb because, in this case, the correlation between the contribution from the Holstein breed estimated from the pedigree and genotype was high (0.93, data not shown) and the genetic contribution from the Fleckvieh breed was low (∼0.02), in accordance with pedigree records. The average MC of the Angler population was 0.62, which is also similar to the average MC level obtained from the pedigree (<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2017</a>). If shorter segments were also to be used, then kinships of individuals would be affected more by very old common ancestors and would consequently be higher.</p></div><div id="sec-23" class="subsection"><h3>Reduction of estimates in unselected populations</h3><p id="p-51">Estimated parameter values for MC, <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-194.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-195.gif"/></noscript></span></span> decreased slightly from generation to generation in REF, even though there was no selection in this scenario. This reduction of the above three parameters was caused by recombination, which shortened the length of the haplotype segments (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stam 1980</a>) until they became too short to meet the criteria for being segments, which led to the reduction of <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-196.gif"/></noscript></span></span> Moreover, if recombination occurred near a particular marker position at an introgressed haplotype segment, then the segment containing the marker could no longer be detected in other breeds. Hence, the marker failed to meet the criteria of belonging to a foreign segment. This gave rise to the reduction of estimated MC. Moreover, since the marker was now classified as native, it contributed to the diversity at native alleles, which caused a reduction in <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-197.gif"/></noscript></span></span> Consequently, the estimated MC should be compared with the estimates obtained from the reference scenario rather than with generation <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-198.gif"/></noscript></span></span> In particular, in scenario OCS-II, in which the constraint for MC was set equal to the MC in generation <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-199.gif"/></noscript></span></span> only the estimate of MC was kept constant, whereas the true MC was effectively increased. There are two possibilities to avoid this increase. Either the constraint for MC in generation <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-200.gif"/></noscript></span></span> is set equal to the mean MC in generation <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-201.gif"/></noscript></span></span> (rather than the mean MC in generation <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-202.gif"/></noscript></span></span>), or another method could be used to estimate the origins of the haplotype segments in generation <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-203.gif"/></noscript></span></span> That is, the origin of a marker could be set equal to the origin of the marker in the parental haplotype from which it originates.</p></div><div id="sec-24" class="subsection"><h3>Genetic diversity parameters</h3><p id="p-52">Different parameters can be used to measure genetic diversity, such as the percentage of polymorphic loci, the number of alleles per locus, expected heterozygosity, etc. (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Harper and Hawksworth 1994</a>). The genetic variation within a breed is of major importance for the conservation of local breeds. In addition to <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-204.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-205.gif"/></noscript></span></span> three further parameters were considered for evaluation of the level of genetic diversity, <em>i.e.</em>, the average observed heterozygosity <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-206.gif"/></noscript></span></span> the variance of TBV (<span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-207.gif"/></noscript></span></span>), and the genic variance (<span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-208.gif"/></noscript></span></span>). Restricting kinship at native alleles and MCs not only had an impact on recovering the original genetic background, but also showed the most potential in conserving genetic diversity among all scenarios. In this study, a similar decreasing pattern of <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-209.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-210.gif"/></noscript></span></span> was observed, with a smaller extent of reduction for parameter <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-211.gif"/></noscript></span></span> This is because <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-212.gif"/></noscript></span></span> is predominantly influenced by neutral alleles (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gregorius 1978</a>).</p><p id="p-53">The additive genetic variance <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-213.gif"/></noscript></span></span> was substantially larger than the genic variance in the first generations and decreased to a large amount from generation <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-214.gif"/></noscript></span></span> to generation <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-215.gif"/></noscript></span></span> This was predominantly because the genetic effects of different chromosomes were correlated in the Angler breed. The contribution of the covariance between different chromosomes to the variance of TBV was 0.089, so in the absence of the covariance, the variance of TBV should be 0.108. The Angler cattle in generation <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-216.gif"/></noscript></span></span> had different contributions from the high-yielding Holstein cattle. For an individual with a high contribution from Holstein cattle, the breeding values of all chromosomes tended to be high, whereas for an individual with a low contribution from Holstein, the breeding values of all chromosomes tended to be low. Consequently, in the first generations, there was covariance between effects of different chromosomes, which contributed to the variance of the breeding values. Additionally, the Bulmer Effect (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bulmer 1971</a>) and the changes in LD due to selection (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bijma 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gorjanc <em>et al.</em> 2015</a>) contributed to the difference between <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-217.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-218.gif"/></noscript></span></span></p></div><div id="sec-25" class="subsection"><h3>Conclusions</h3><p id="p-54">Advanced OCS strategies enable the achievement of a balance between the different breeding goals of populations with historic introgression, which are to improve the genetic progress, recover the original genetic background, and conserve genetic diversity. Here, truncation selection and traditional OCS achieved the highest genetic gain, but both reduced the genetic originality of the breed by depleting diversity at native alleles and increasing MCs. However, maintaining genetic originality is crucial for conserving breeds with historical introgression. The inclusion of MC and kinship at native alleles as additional constraints in OCS showed great potential for conservation. Recovering the original genetic background is possible but requires many generations of selection and reduces the genetic progress. Thus, it is essential to set an appropriate constraint for MC in order to balance both breeding goals, which are to achieve genetic progress and to recover the original genetic background of local breeds.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank the editor and two anonymous reviewers for their comments and advice. This study was supported by the German Research Foundation (Deutsche Forschungsgemeinschaft). The authors declare that they have no competing interests.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-26"><h3>Native Segments</h3><p id="p-58">Each individual <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-219.gif"/></noscript></span></span> has a maternal haplotype <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-220.gif"/></noscript></span></span> and a paternal haplotype <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-221.gif"/></noscript></span></span> For haplotype <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-222.gif"/></noscript></span></span> of Angler <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-223.gif"/></noscript></span></span> the frequency of the segment at marker <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-224.gif"/></noscript></span></span> in the set of animals from reference breed <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-225.gif"/></noscript></span></span> was calculated as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-226.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-227.gif"/></noscript></span></span> is the number of individuals from breed <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-228.gif"/></noscript></span></span> set <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-229.gif"/></noscript></span></span> contains all markers belonging to segments, which are identical in haplotypes <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-230.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-231.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-232.gif"/></noscript></span></span> if marker <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-233.gif"/></noscript></span></span> belongs to such a segment.</p><p id="p-59">A marker <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-234.gif"/></noscript></span></span> from haplotype <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-235.gif"/></noscript></span></span> was classified as native (<span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-236.gif"/></noscript></span></span>) if the frequency of the segment containing marker <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-237.gif"/></noscript></span></span> is smaller than <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-238.gif"/></noscript></span></span> in all reference breeds. That is,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-239.gif"/></noscript></span></span>where the maximum was taken over all reference breeds, which were Fleckvieh, Holstein-Friesian, Red Holstein, and Norwegian Red in our study.</p><p id="p-60">The native contribution <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-240.gif"/></noscript></span></span> of individual <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-241.gif"/></noscript></span></span> is the proportion of the genome included in native segments. That is,<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-242.gif"/></noscript></span></span>Here, <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-243.gif"/></noscript></span></span> is the length of the genome region in megabases represented by marker <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-244.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-245.gif"/></noscript></span></span> is the length of the genome in megabases. The MC of individual <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-246.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-247.gif"/></noscript></span></span></p></div><div class="section" id="sec-27"><h3>Kinship <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-248.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-249.gif"/></noscript></span></span></h3><p id="p-61">The kinship <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-250.gif"/></noscript></span></span> between individuals <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-251.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-252.gif"/></noscript></span></span> (element of matrix <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-253.gif"/></noscript></span></span>) is the probability that two alleles taken at random from individuals <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-254.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-255.gif"/></noscript></span></span> belong to identical segments:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-256.gif"/></noscript></span></span>where all parameters involved are the same as previously explained.</p><p id="p-62">The kinship <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-257.gif"/></noscript></span></span> in the offspring is the conditional probability that two alleles taken at random from the offspring belong to identical segments, given that they are native. The mean kinship <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-258.gif"/></noscript></span></span> of the following generation is estimated as:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-259.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-260.gif"/></noscript></span></span> is a matrix containing the probabilities that two alleles taken at random from both individuals belong to identical segments and are native, and <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-261.gif"/></noscript></span></span> is a matrix containing the probabilities that two alleles taken at random from both individuals are native:</p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-262.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/12/4009/embed/mml-math-263.gif"/></noscript></span>.</span></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300272/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-57"><em>Communicating editor: G. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-13"><span class="received-label">Received </span>September 13, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-27"><span class="accepted-label">Accepted </span>October 27, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Wang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzcvMTIvNDAwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amador</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. Á.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Removing exogenous information using pedigree data.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1565</span>–<span class="cit-lpage">1573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D12%26rft.spage%253D1565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amador</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Advantages of using molecular coancestry in the removal of introgressed genetic material.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.4" data-doi="10.1093/bioinformatics/bts144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasaniuc</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate inference of local ancestry in Latino populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1359</span>–<span class="cit-lpage">1367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts144%26rft_id%253Dinfo%253Apmid%252F22495753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495753&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304053300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of extinction probabilities of five German cattle breeds by population viability analysis.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">2949</span>–<span class="cit-lpage">2961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DBennewitz%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D8%26rft.spage%253D2949%26rft.epage%253D2961%26rft.atitle%253DEstimation%2Bof%2BExtinction%2BProbabilities%2Bof%2BFive%2BGerman%2BCattle%2BBreeds%2Bby%2BPopulation%2BViability%2BAnalysis%26rft_id%253Dinfo%253Apmid%252F16027209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16027209&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.6" data-doi="10.1111/j.1439-0388.2012.00991.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracies of estimated breeding values from ordinary genetic evaluations do not reflect the correlation between true and estimated breeding values in selected populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DBijma%26rft.auinit1%253DP.%26rft.volume%253D129%26rft.issue%253D5%26rft.spage%253D345%26rft.epage%253D358%26rft.atitle%253DAccuracies%2Bof%2Bestimated%2Bbreeding%2Bvalues%2Bfrom%2Bordinary%2Bgenetic%2Bevaluations%2Bdo%2Bnot%2Breflect%2Bthe%2Bcorrelation%2Bbetween%2Btrue%2Band%2Bestimated%2Bbreeding%2Bvalues%2Bin%2Bselected%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2012.00991.x%26rft_id%253Dinfo%253Apmid%252F22963356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2012.00991.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22963356&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.7" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.8" data-doi="10.1086/282718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">The effect of selection on genetic variability.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D105%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971J801400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.9" data-doi="10.1023/A:1019956205473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. Á.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Analysis of genetic diversity for the management of conserved subdivided populations.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D3%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1019956205473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1019956205473&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000177718900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.10" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vattikuti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.11" data-doi="10.1016/j.livsci.2014.05.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curik</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferenčaković</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sölkner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Inbreeding and runs of homozygosity: a possible solution to an old problem.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.05.034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.05.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.12" data-doi="10.1111/j.1439-0388.2011.00971.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Cara</span>  <span class="cit-name-given-names">M. Á. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. Á.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using genome-wide information to minimize the loss of diversity in conservation programmes.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253Dde%2BCara%26rft.auinit1%253DM.%2BA.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D456%26rft.epage%253D464%26rft.atitle%253DUsing%2Bgenome-wide%2Binformation%2Bto%2Bminimize%2Bthe%2Bloss%2Bof%2Bdiversity%2Bin%2Bconservation%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2011.00971.x%26rft_id%253Dinfo%253Apmid%252F22059579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2011.00971.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059579&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.13" data-doi="10.1111/mec.12560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Cara</span>  <span class="cit-name-given-names">M. Á. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. Á.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using genomic tools to maintain diversity and fitness in conservation programmes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">6091</span>–<span class="cit-lpage">6099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D6091%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.14" data-doi="10.1046/j.1439-0388.2001.00290.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Marker-based estimates of between and within population kinships for the conservation of genetic diversity.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D118%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0388.2001.00290.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0388.2001.00290.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169434900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eynard</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiemstra</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole-genome sequence data uncover loss of genetic diversity due to selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-7.12.4009.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>. <span class="cit-publ-name">Pearson Education Limited</span>, <span class="cit-publ-loc">Harlow, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.17" data-doi="10.1111/j.1365-2052.2012.02345.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frkonja</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gredler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnyder</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curik</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sölkner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prediction of breed composition in an admixed cattle population.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DFrkonja%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D696%26rft.epage%253D703%26rft.atitle%253DPrediction%2Bof%2Bbreed%2Bcomposition%2Bin%2Ban%2Badmixed%2Bcattle%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2012.02345.x%26rft_id%253Dinfo%253Apmid%252F23061480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2012.02345.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23061480&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gómez-Romano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sölkner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Cara</span>  <span class="cit-name-given-names">M. Á. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mészáros</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The use of coancestry based on shared segments for maintaining genetic diversity.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D133%26rft.spage%253D357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reliability of pedigree-based and genomic evaluations in selected populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.20" data-doi="10.1016/0025-5564(78)90040-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregorius</span>  <span class="cit-name-given-names">H.-R.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">The concept of genetic diversity and its formal relationship to heterozygosity and genetic distance.</span> <abbr class="cit-jnl-abbrev">Math. Biosci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BBiosci.%26rft.volume%253D41%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252878%252990040-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(78)90040-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978FZ40800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.21" data-doi="10.1017/S0016672398003474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grundy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Dynamic selection procedures for constrained inbreeding and their consequences for pedigree development.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D72%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672398003474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672398003474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077703600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.22" data-doi="10.1101/gr.081398.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole population, genome-wide mapping of hidden relatedness.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081398.108%26rft_id%253Dinfo%253Apmid%252F18971310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzE4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi80MDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.23" data-doi="10.1098/rstb.1994.0081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawksworth</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Biodiversity: measurement and estimation.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1994.0081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEwOiIzNDUvMTMxMS81IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi80MDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.24" data-doi="10.1534/genetics.111.130922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantification of inbreeding due to distant ancestors and its detection using dense single nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130922%26rft_id%253Dinfo%253Apmid%252F21705750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzEvMjM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi80MDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.25" data-doi="10.1186/s12864-015-1469-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McTavish</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillis</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">How do SNP ascertainment schemes and population demographics affect inferences about population history?</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D266%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1469-5%26rft_id%253Dinfo%253Apmid%252F25887858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1469-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25887858&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.26" data-doi="10.3389/fgene.2015.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mészáros</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boison</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez O’Brien</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferenčaković</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curik</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic analysis for managing small and endangered populations: a case study in Tyrol Grey cattle.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00001%26rft_id%253Dinfo%253Apmid%252F25674101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25674101&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic management of small populations: a review.</span> <abbr class="cit-jnl-abbrev">Acta Agric. Scand. A Anim. Sci.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgric.%2BScand.%2BA%2BAnim.%2BSci.%26rft.volume%253D59%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Maximizing the response of selection with a predefined rate of inbreeding.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">2575</span>–<span class="cit-lpage">2583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D75%26rft.spage%253D2575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy83LzEyLzQwMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimation of population parameters and recombination rates from single nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D154%26rft.issue%253D2%26rft.spage%253D931%26rft.epage%253D942%26rft.atitle%253DEstimation%2Bof%2BPopulation%2BParameters%2Band%2BRecombination%2BRates%2BFrom%2BSingle%2BNucleotide%2BPolymorphisms%26rft_id%253Dinfo%253Apmid%252F10655242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzIvOTMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi80MDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peripolli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munari</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">M. V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">A. L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irgang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Runs of homozygosity: current knowledge and applications in livestock.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D48%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-other" id="cit-7.12.4009.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfaff</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> The R package cccp: design for solving cone constrained convex programs. R Finance. Available at: <a href="http://www.pfaffikus.de/files/conf/rif/rif2014.pdf">http://www.pfaffikus.de/files/conf/rif/rif2014.pdf</a>. Accessed: May 16, 2014.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.33" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.34" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNy8xMi80MDA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Ramilo</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Cortés</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Cara</span>  <span class="cit-name-given-names">M. Á. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Artificial selection with traditional or genomic relationships: consequences in coancestry and genetic diversity.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.36" data-doi="10.3168/jds.S0022-0302(96)76413-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ron</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Band</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ezra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weller</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Misidentification rate in the Israeli dairy cattle population and its implications for genetic improvement.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DRon%26rft.auinit1%253DM.%26rft.volume%253D79%26rft.issue%253D4%26rft.spage%253D676%26rft.epage%253D681%26rft.atitle%253DMisidentification%2BRate%2Bin%2Bthe%2BIsraeli%2BDairy%2BCattle%2BPopulation%2Band%2BIts%2BImplications%2Bfor%2BGenetic%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252896%252976413-5%26rft_id%253Dinfo%253Apmid%252F8744233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(96)76413-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8744233&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.37" data-doi="10.1017/S0016672300014002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The distribution of the fraction of the genome identical by descent in finite random mating populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D35%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.38" data-doi="10.1016/j.livsci.2014.04.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. Á.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics applied to management strategies in conservation programmes.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.04.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.04.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.39" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-other" id="cit-7.12.4009.40" data-doi="10.1371/journal.pgen.0020041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medland</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morley</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">2</span>: <span class="cit-fpage">e41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253De41%26rft.epage%253De41%26rft.atitle%253DAssumption-free%2Bestimation%2Bof%2Bheritability%2Bfrom%2Bgenome-wide%2Bidentity-by-descent%2Bsharing%2Bbetween%2Bfull%2Bsiblings.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020041%26rft_id%253Dinfo%253Apmid%252F16565746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16565746&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wellmann</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Novel optimum contribution selection methods accounting for conflicting objectives in breeding programs for livestock breeds with historical migration.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-other" id="cit-7.12.4009.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellmann</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> optiSel: optimum contribution selection and population genetics. R Package. version 0.9.1 <a href="https://CRAN.R-project.org/package=optiSel">https://CRAN.R-project.org/package=optiSel</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartwig</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennewitz</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimum contribution selection for conserved populations with historic migration.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DWellmann%26rft.auinit1%253DR.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DOptimum%2Bcontribution%2Bselection%2Bfor%2Bconserved%2Bpopulations%2Bwith%2Bhistoric%2Bmigration.%26rft_id%253Dinfo%253Apmid%252F23153196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23153196&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">Z.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saatchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Recombination locations and rates in beef cattle assessed from parent-offspring pairs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D34%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dagnachew</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic contributions and their optimization.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D132%26rft.spage%253D89%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.12.4009.46" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F12%2F4009.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/12/4009.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/3999" title="Understanding microRNA Regulation Involved in the Metamorphosis of the Veined Rapa Whelk (Rapana venosa)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/12/4019" title="G3 Reviewer Index 2017" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/12"><span>Volume 7 Issue 12, December 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/12.cover-source.jpg?itok=-p16LxzL" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/12/4009" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZvMUrj539_JqnHMY0ZGoL86zxxhFO6bm7JVsuaZHww8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/12/4009" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/12/4009" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/12/4009" class="active" data-icon-position="" data-hide-link-title="0">Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487468" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SdyPu_Uu9NxcIEZtiHNQt6UYIBIt4mizM1tPF6AghyM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/4009.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/12/4009" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-clHH02mUOKLlrmr4Xisc8nCKednFpDt8VYlObpnR4BE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/12/4009" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/12/4009.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487468" id="node487468--2" data-pisa="ggg;7/12/4009" data-pisa-master="ggg;g3.117.300272" data-apath="/ggg/7/12/4009.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3756-6246" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yu Wang</span>, <span class="highwire-citation-author" data-delta="1">Dierck Segelke</span>, <span class="highwire-citation-author" data-delta="2">Reiner Emmerling</span>, <span class="highwire-citation-author" data-delta="3">Jörn Bennewitz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4748-8042" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robin Wellmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4009-4018;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300272 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487468--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yu Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yu%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3756-6246" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yu Wang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yu.wang@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">yu.wang@uni-hohenheim.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dierck Segelke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Vereinigte Informationssysteme Tierhaltung w.V., 27283 Verden, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dierck%2BSegelke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segelke%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADierck%2BSegelke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Reiner Emmerling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Breeding, Bavarian State Research Centre for Agriculture, 85586 Grub, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Reiner%2BEmmerling%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmerling%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AReiner%2BEmmerling%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jörn Bennewitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BBennewitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennewitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B6rn%2BBennewitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robin Wellmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robin%2BWellmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellmann%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobin%2BWellmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4748-8042" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robin Wellmann</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/12/4009" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/12/4009&amp;title=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/12/4009&amp;title=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/12/4009&amp;title=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/12/4009&amp;text=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/12/4009&amp;title=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/12/4009&amp;t=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/12/4009&amp;title=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/12/4009&amp;title=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487468/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487468" id="node487468--3" data-pisa="ggg;7/12/4009" data-pisa-master="ggg;g3.117.300272" data-apath="/ggg/7/12/4009.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long-Term Impact of Optimum Contribution Selection Strategies on Local Livestock Breeds with Historical Introgression Using the Example of German Angler Cattle</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3756-6246" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yu Wang</span>, <span class="highwire-citation-author" data-delta="1">Dierck Segelke</span>, <span class="highwire-citation-author" data-delta="2">Reiner Emmerling</span>, <span class="highwire-citation-author" data-delta="3">Jörn Bennewitz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4748-8042" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robin Wellmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4009-4018;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300272 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487468--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yu Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yu%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3756-6246" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yu Wang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yu.wang@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">yu.wang@uni-hohenheim.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dierck Segelke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Vereinigte Informationssysteme Tierhaltung w.V., 27283 Verden, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dierck%2BSegelke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segelke%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADierck%2BSegelke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Reiner Emmerling </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Animal Breeding, Bavarian State Research Centre for Agriculture, 85586 Grub, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Reiner%2BEmmerling%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmerling%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AReiner%2BEmmerling%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jörn Bennewitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rn%2BBennewitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennewitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25B6rn%2BBennewitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robin Wellmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Animal Science, University of Hohenheim, 70593 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robin%2BWellmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellmann%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobin%2BWellmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4748-8042" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robin Wellmann</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=4009&amp;rft.epage=4018&amp;rft.atitle=Long-Term%20Impact%20of%20Optimum%20Contribution%20Selection%20Strategies%20on%20Local%20Livestock%20Breeds%20with%20Historical%20Introgression%20Using%20the%20Example%20of%20German%20Angler%20Cattle&amp;rft.volume=7&amp;rft.issue=12&amp;rft.date=2017-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Yu&amp;rft.au=Segelke%2C+Dierck&amp;rft.au=Emmerling%2C+Reiner&amp;rft.au=Bennewitz%2C+J%C3%B6rn&amp;rft.au=Wellmann%2C+Robin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487468/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487468/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487468/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487468/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487468/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487468/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487468/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487468/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487468/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487468/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487468/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487468/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/12/4009" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/12/4009.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/12/4009.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/12/4009.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/12/4009.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
