<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/7/523" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/7/523.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/7/523.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/7/523.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/7/523.atom" />
<meta name="HW.pisa" content="ggg;1/7/523" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains" />
<meta name="DC.Identifier" content="10.1534/g3.111.000885" />
<meta name="DC.Date" content="2011-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Abbey et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Phenotypic diversity can arise rapidly through loss of heterozygosity (LOH) or by the acquisition of copy number variations (CNV) spanning whole chromosomes or shorter contiguous chromosome segments. In Candida albicans, a heterozygous diploid yeast pathogen with no known meiotic cycle, homozygosis and aneuploidy alter clinical characteristics, including drug resistance. Here, we developed a high-resolution microarray that simultaneously detects ∼39,000 single nucleotide polymorphism (SNP) alleles and ∼20,000 copy number variation loci across the C. albicans genome. An important feature of the array analysis is a computational pipeline that determines SNP allele ratios based upon chromosome copy number. Using the array and analysis tools, we constructed a haplotype map (hapmap) of strain SC5314 to assign SNP alleles to specific homologs, and we used it to follow the acquisition of loss of heterozygosity (LOH) and copy number changes in a series of derived laboratory strains. This high-resolution SNP/CGH microarray and the associated hapmap facilitated the phasing of alleles in lab strains and revealed detrimental genome changes that arose frequently during molecular manipulations of laboratory strains. Furthermore, it provided a useful tool for rapid, high-resolution, and cost-effective characterization of changes in allele diversity as well as changes in chromosome copy number in new C. albicans isolates." />
<meta name="DC.Contributor" content="Darren Abbey" />
<meta name="DC.Contributor" content="Meleah Hickman" />
<meta name="DC.Contributor" content="David Gresham" />
<meta name="DC.Contributor" content="Judith Berman" />
<meta name="article:published_time" content="2011-12-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Phenotypic diversity can arise rapidly through loss of heterozygosity (LOH) or by the acquisition of copy number variations (CNV) spanning whole chromosomes or shorter contiguous chromosome segments. In &lt;i&gt;Candida albicans,&lt;/i&gt; a heterozygous diploid yeast pathogen with no known meiotic cycle, homozygosis and aneuploidy alter clinical characteristics, including drug resistance. Here, we developed a high-resolution microarray that simultaneously detects ∼39,000 single nucleotide polymorphism (SNP) alleles and ∼20,000 copy number variation loci across the &lt;i&gt;C. albicans&lt;/i&gt; genome. An important feature of the array analysis is a computational pipeline that determines SNP allele ratios based upon chromosome copy number. Using the array and analysis tools, we constructed a haplotype map (hapmap) of strain SC5314 to assign SNP alleles to specific homologs, and we used it to follow the acquisition of loss of heterozygosity (LOH) and copy number changes in a series of derived laboratory strains. This high-resolution SNP/CGH microarray and the associated hapmap facilitated the phasing of alleles in lab strains and revealed detrimental genome changes that arose frequently during molecular manipulations of laboratory strains. Furthermore, it provided a useful tool for rapid, high-resolution, and cost-effective characterization of changes in allele diversity as well as changes in chromosome copy number in new &lt;i&gt;C. albicans&lt;/i&gt; isolates.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/12/01" />
<meta name="citation_mjid" content="ggg;1/7/523" />
<meta name="citation_id" content="1/7/523" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/7/523" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/7/523.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/7/523.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/7/523.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000885" />
<meta name="citation_pmid" content="22384363" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="523" />
<meta name="citation_lastpage" content="530" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Darren Abbey" />
<meta name="citation_author_institution" content="Department of Genetics, Cell Biology and Development and" />
<meta name="citation_author" content="Meleah Hickman" />
<meta name="citation_author_institution" content="Department of Genetics, Cell Biology and Development and" />
<meta name="citation_author" content="David Gresham" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Minnesota, Minneapolis, Minnesota 55455, and" />
<meta name="citation_author_institution" content="Department of Biology, New York University, New York, New York 10003" />
<meta name="citation_author" content="Judith Berman" />
<meta name="citation_author_institution" content="Department of Genetics, Cell Biology and Development and" />
<meta name="citation_author_institution" content="Center for Genomics and Systems Biology" />
<meta name="citation_author_email" content="jberman@umn.edu" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. G. Addinall;citation_author=E. M. Holstein;citation_author=C. Lawless;citation_author=M. Yu;citation_author=K. Chapman;citation_title=Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.;citation_pages=e1001362-e1001362;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21490951;citation_doi=10.1371/journal.pgen.1001362" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=R. Alonso-Monge;citation_author=F. Navarro-Garcia;citation_author=E. Roman;citation_author=A. I. Negredo;citation_author=B. Eisman;citation_title=The Hog1 Mitogen-Activated Protein Kinase Is Essential in the Oxidative Stress Response and Chlamydospore Formation in Candidaalbicans;citation_pages=351-361;citation_volume=2;citation_year=2003;citation_issue=2;citation_pmid=12684384;citation_doi=10.1128/EC.2.2.351-361.2003" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schäffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Andersen;citation_author=Z. W. Nelson;citation_author=E. D. Hetrick;citation_author=D. E. Gottschling;citation_title=A genetic screen for increased loss of heterozygosity in Saccharomyces cerevisiae.;citation_pages=1179-1195;citation_volume=179;citation_year=2008;citation_pmid=18562670;citation_doi=10.1534/genetics.108.089250" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Research;citation_journal_abbrev=FEMS Yeast Res;citation_author=M. Arbour;citation_author=E. Epp;citation_author=H. Hogues;citation_author=A. Sellam;citation_author=C. Lacroix;citation_title=Widespread occurrence of chromosomal aneuploidy following the routine production of Candida albicans mutants.;citation_pages=1070-1077;citation_volume=9;citation_year=2009;citation_issue=7;citation_pmid=19732157;citation_doi=10.1111/j.1567-1364.2009.00563.x" />
<meta name="citation_reference" content="citation_journal_title=Agents Chemother.;citation_author=M. C. Arendrup;citation_author=G. Garcia-Effron;citation_author=W. Buzina;citation_author=K. L. Mortensen;citation_author=N. Reiter;citation_title=Breakthrough Aspergillus fumigatus and Candida albicans double infection during caspofungin treatment: laboratory characteristics and implication for susceptibility testing. Antimicrob.;citation_pages=1185-1193;citation_volume=53;citation_year=2008;citation_pmid=19104024" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiology Letters;citation_journal_abbrev=FEMS Microbiol Lett;citation_author=J. M. Bain;citation_author=C. Stubberfield;citation_author=N. A. Gow;citation_title=Ura-status-dependent adhesion of Candida albicans mutants.;citation_pages=323-328;citation_volume=204;citation_year=2001;citation_issue=2;citation_pmid=11731143;citation_doi=10.1111/j.1574-6968.2001.tb10905.x" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=P. K. Bansal;citation_author=A. Nourse;citation_author=R. Abdulle;citation_author=K. Kitagawa;citation_title=Sgt1 dimerization is required for yeast kinetochore assembly.;citation_volume=284;citation_year=2009;citation_pmid=19073600;citation_doi=10.1074/jbc.M806281200" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=K. Bouchonville;citation_author=A. Forche;citation_author=K. E. S. Tang;citation_author=A. Selmecki;citation_author=J. Berman;citation_title=Aneuploid chromosomes are highly unstable during DNA transformation of Candida albicans.;citation_volume=8;citation_year=2009;citation_pmid=19700634;citation_doi=10.1128/EC.00209-09" />
<meta name="citation_reference" content="citation_journal_title=The Journal of infection;citation_journal_abbrev=J Infect;citation_author=L. P. Brion;citation_author=S. E. Uko;citation_author=D. L. Goldman;citation_title=Risk of resistance associated with fluconazole prophylaxis: systematic review.;citation_pages=521-529;citation_volume=54;citation_year=2007;citation_issue=6;citation_pmid=17239952;citation_doi=10.1016/j.jinf.2006.11.017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Butler;citation_author=M. D. Rasmussen;citation_author=M. F. Lin;citation_author=M. A. S. Santos;citation_author=S. Sakthikumar;citation_title=Evolution of pathogenicity and sexual reproduction in eight Candida genomes.;citation_pages=657-662;citation_volume=459;citation_year=2009;citation_issue=7247;citation_pmid=19465905;citation_doi=10.1038/nature08064" />
<meta name="citation_reference" content="citation_journal_title=Mol. Microbiol.;citation_author=X. Chen;citation_author=B. B. Magee;citation_author=D. Dawson;citation_author=P. T. Magee;citation_author=C. A. Kumamoto;citation_title=Chromosome 1 trisomy compromises the virulence of Candida albicans.;citation_pages=551;citation_volume=51;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=A. Coste;citation_author=A. Selmecki;citation_author=A. Forche;citation_author=D. Diogo;citation_author=M.-E. Bougnoux;citation_title=Genotypic evolution of azole resistance mechanisms in sequential Candida albicans isolates.;citation_volume=6;citation_year=2007;citation_pmid=17693596;citation_doi=10.1128/EC.00151-07" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Dunham;citation_author=H. Badrane;citation_author=T. Ferea;citation_author=J. Adams;citation_author=P. O. Brown;citation_title=Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae.;citation_volume=99;citation_year=2002;citation_pmid=12446845;citation_doi=10.1073/pnas.242624799" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. A. Fonzi;citation_author=M. Y. Irwin;citation_title=Isogenic strain construction and gene mapping in Candida albicans.;citation_pages=717-728;citation_volume=134;citation_year=1993;citation_pmid=8349105" />
<meta name="citation_reference" content="citation_journal_title=mBio;citation_author=A. Forche;citation_author=D. Abbey;citation_author=T. Pisithkul;citation_author=M. A. Weinzierl;citation_author=T. Ringstrom;citation_title=Stress alters rates and types of loss of heterozygosity in Candida albicans.;citation_pages=e00129-11;citation_volume=2;citation_year=2011;citation_pmid=21791579;citation_doi=10.1128/mBio.00129-11" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. Forche;citation_author=K. Alby;citation_author=D. Schaefer;citation_author=A. D. Johnson;citation_author=J. Berman;citation_title=The parasexual cycle in Candida albicans provides an alternative pathway to meiosis for the formation of recombinant strains.;citation_pages=e110-e110;citation_volume=6;citation_year=2008;citation_issue=5;citation_pmid=18462019;citation_doi=10.1371/journal.pbio.0060110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Forche;citation_author=P. T. Magee;citation_author=A. Selmecki;citation_author=J. Berman;citation_author=G. May;citation_title=Evolution in Candida albicans populations during a single passage through a mouse host.;citation_pages=799-811;citation_volume=182;citation_year=2009a;citation_pmid=19414562;citation_doi=10.1534/genetics.109.103325" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Research;citation_journal_abbrev=FEMS Yeast Res;citation_author=A. Forche;citation_author=M. Steinbach;citation_author=J. Berman;citation_title=Efficient and rapid identification of Candida albicans allelic status using SNP-RFLP.;citation_pages=1061-1069;citation_volume=9;citation_year=2009b;citation_issue=7;citation_pmid=19622074;citation_doi=10.1111/j.1567-1364.2009.00542.x" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=A. Forche;citation_author=G. May;citation_author=P. T. Magee;citation_title=Demonstration of Loss of Heterozygosity by Single-Nucleotide Polymorphism Microarray Analysis and Alterations in Strain Morphology in Candida albicans Strains during Infection;citation_pages=156-165;citation_volume=4;citation_year=2005;citation_issue=1;citation_pmid=15643071;citation_doi=10.1128/EC.4.1.156-165.2005" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=A. M. Gillum;citation_author=E. Y. Tsay;citation_author=D. R. Kirsch;citation_title=Isolation of the Candida albicans gene for orotidine-5'-phosphate decarboxylase by complementation of S. cerevisiae ura3 and E. coli pyrF mutations.;citation_pages=179-182;citation_volume=198;citation_year=1984;citation_issue=1;citation_pmid=6394964;citation_doi=10.1007/BF00328721" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Gresham;citation_author=B. Curry;citation_author=A. Ward;citation_author=D. B. Gordon;citation_author=L. Brizuela;citation_title=Optimized detection of sequence variation in heterozygous genomes using DNA microarrays with isothermal-melting probes.;citation_volume=107;citation_year=2010;citation_pmid=20080586;citation_doi=10.1073/pnas.0913883107" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Gresham;citation_author=M. M. Desai;citation_author=C. M. Tucker;citation_author=H. T. Jenq;citation_author=D. A. Pai;citation_title=The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.;citation_pages=e1000303-e1000303;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19079573;citation_doi=10.1371/journal.pgen.1000303" />
<meta name="citation_reference" content="citation_title=Guide to Yeast Genetics and Molecular Biology;citation_author=C. Guthrie;citation_author=G. R. Fink;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Hadany;citation_author=T. Beker;citation_title=On the evolutionary advantage of fitness-associated recombination.;citation_pages=2167-2179;citation_volume=165;citation_year=2003;citation_pmid=14704195" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=C. Hilton;citation_author=D. Markie;citation_author=B. Corner;citation_author=E. Rikkerink;citation_author=R. Poulter;citation_title=Heat shock induces chromosome loss in the yeast Candida albicans.;citation_pages=162-168;citation_volume=200;citation_year=1985;citation_issue=1;citation_pmid=3897792;citation_doi=10.1007/BF00383330" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. R. Hughes;citation_author=C. J. Roberts;citation_author=H. Dai;citation_author=A. R. Jones;citation_author=M. R. Meyer;citation_title=Widespread aneuploidy revealed by DNA microarray expression profiling.;citation_pages=333-337;citation_volume=25;citation_year=2000;citation_issue=3;citation_pmid=10888885;citation_doi=10.1038/77116" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Jones;citation_author=N. A. Federspiel;citation_author=H. Chibana;citation_author=J. Dungan;citation_author=S. Kalman;citation_title=The diploid genome sequence of Candida albicans.;citation_volume=101;citation_year=2004;citation_pmid=15123810;citation_doi=10.1073/pnas.0401648101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. I. Lang;citation_author=A. W. Murray;citation_title=Estimating the Per-Base-Pair Mutation Rate in the Yeast Saccharomyces cerevisiae;citation_pages=67-82;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202359;citation_doi=10.1534/genetics.107.071506" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Legrand;citation_author=A. Forche;citation_author=A. Selmecki;citation_author=C. Chan;citation_author=D. T. Kirkpatrick;citation_title=Haplotype mapping of a diploid non-meiotic organism using existing and induced aneuploidies.;citation_pages=e1-e1;citation_volume=4;citation_year=2008;citation_issue=1;citation_pmid=18179283;citation_doi=10.1371/journal.pgen.0040001" />
<meta name="citation_reference" content="citation_journal_title=Antimicrobial Agents and Chemotherapy;citation_journal_abbrev=Antimicrob. Agents Chemother.;citation_author=K. A. Marr;citation_author=C. N. Lyons;citation_author=T. R. Rustad;citation_author=R. A. Bowden;citation_author=T. C. White;citation_title=Rapid, Transient Fluconazole Resistance in Candida albicans Is Associated with Increased mRNA Levels of CDR;citation_pages=2584-2589;citation_volume=42;citation_year=1998;citation_issue=10;citation_pmid=9756759" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=K. A. Marr;citation_author=T. C. White;citation_author=J. A. van Burik;citation_author=R. A. Bowden;citation_title=Development of fluconazole resistance in Candida albicans causing disseminated infection in a patient undergoing marrow transplantation.;citation_volume=25;citation_year=1997;citation_pmid=9356806;citation_doi=10.1086/515553" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=J. Morschhauser;citation_author=K. S. Barker;citation_author=T. T. Liu;citation_author=B. W. J. Bla;citation_author=R. Homayouni;citation_title=The transcription factor Mrr1p controls expression of the MDR1 efflux pump and mediates multidrug resistance in Candida albicans.;citation_pages=e164-e164;citation_volume=3;citation_year=2007;citation_issue=11;citation_pmid=17983269;citation_doi=10.1371/journal.ppat.0030164" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. L. Myers;citation_author=M. J. Dunham;citation_author=S. Y. Kung;citation_author=O. G. Troyanskaya;citation_title=Accurate detection of aneuploidies in array CGH and gene expression microarray data.;citation_volume=20;citation_year=2004;citation_pmid=15284100;citation_doi=10.1093/bioinformatics/bth440" />
<meta name="citation_reference" content="citation_journal_title=Odontology / the Society of the Nippon Dental University;citation_journal_abbrev=Odontology;citation_author=M. Niimi;citation_author=N. A. Firth;citation_author=R. D. Cannon;citation_title=Antifungal drug resistance of oral fungi.;citation_pages=15-25;citation_volume=98;citation_year=2010;citation_issue=1;citation_pmid=20155503;citation_doi=10.1007/s10266-009-0118-3" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=S. M. Noble;citation_author=A. D. Johnson;citation_title=Strains and Strategies for Large-Scale Gene Deletion Studies of the Diploid Human Fungal Pathogen Candida albicans;citation_pages=298-309;citation_volume=4;citation_year=2005;citation_issue=2;citation_pmid=15701792;citation_doi=10.1128/EC.4.2.298-309.2005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Pavelka;citation_author=G. Rancati;citation_author=J. Zhu;citation_author=W. D. Bradford;citation_author=A. Saraf;citation_title=Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.;citation_pages=321-325;citation_volume=468;citation_year=2010;citation_issue=7321;citation_pmid=20962780;citation_doi=10.1038/nature09529" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Rancati;citation_author=N. Pavelka;citation_author=B. Fleharty;citation_author=A. Noll;citation_author=R. Trimble;citation_title=Aneuploidy underlies rapid adaptive evolution of yeast cells deprived of a conserved cytokinesis motor.;citation_pages=879-893;citation_volume=135;citation_year=2008;citation_issue=5;citation_pmid=19041751;citation_doi=10.1016/j.cell.2008.09.039" />
<meta name="citation_reference" content="citation_journal_title=Drugs;citation_journal_abbrev=Drugs;citation_author=M. J. Ruping;citation_author=J. J. Vehreschild;citation_author=O. A. Cornely;citation_title=Patients at high risk of invasive fungal infections: when and how to treat.;citation_pages=1941-1962;citation_volume=68;citation_year=2008;citation_issue=14;citation_pmid=18778118;citation_doi=10.2165/00003495-200868140-00002" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=A. Selmecki;citation_author=S. Bergmann;citation_author=J. Berman;citation_title=Comparative genome hybridization reveals widespread aneuploidy in Candida albicans laboratory strains.;citation_pages=1553-1565;citation_volume=55;citation_year=2005;citation_issue=5;citation_pmid=15720560;citation_doi=10.1111/j.1365-2958.2005.04492.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Selmecki;citation_author=A. Forche;citation_author=J. Berman;citation_title=Aneuploidy and Isochromosome Formation in Drug-Resistant Candida albicans;citation_pages=367-370;citation_volume=313;citation_year=2006;citation_issue=5785;citation_pmid=16857942;citation_doi=10.1126/science.1128242" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=A. Selmecki;citation_author=M. Gerami-Nejad;citation_author=C. Paulson;citation_author=A. Forche;citation_author=J. Berman;citation_title=An isochromosome confers drug resistance in vivo by amplification of two genes, ERG11 and TAC1.;citation_pages=624-641;citation_volume=68;citation_year=2008;citation_issue=3;citation_pmid=18363649;citation_doi=10.1111/j.1365-2958.2008.06176.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. M. Selmecki;citation_author=K. Dulmage;citation_author=L. E. Cowen;citation_author=J. B. Anderson;citation_author=J. Berman;citation_title=Acquisition of aneuploidy provides increased fitness during the evolution of antifungal drug resistance.;citation_pages=e1000705-e1000705;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19876375;citation_doi=10.1371/journal.pgen.1000705" />
<meta name="citation_reference" content="citation_journal_title=International journal of antimicrobial agents;citation_journal_abbrev=Int J Antimicrob Agents;citation_author=H.-Y. Sun;citation_author=N. Singh;citation_title=Characterisation of breakthrough invasive mycoses in echinocandin recipients: an evidence-based review.;citation_pages=211-218;citation_volume=35;citation_year=2010;citation_issue=3;citation_pmid=20036518;citation_doi=10.1016/j.ijantimicag.2009.09.020" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. M. Torres;citation_author=T. Sokolsky;citation_author=C. M. Tucker;citation_author=L. Y. Chan;citation_author=M. Boselli;citation_title=Effects of Aneuploidy on Cellular Physiology and Cell Division in Haploid Yeast;citation_pages=916-924;citation_volume=317;citation_year=2007;citation_issue=5840;citation_pmid=17702937;citation_doi=10.1126/science.1142210" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. M. Torres;citation_author=B. R. Williams;citation_author=A. Amon;citation_title=Aneuploidy: cells losing their balance.;citation_pages=737-746;citation_volume=179;citation_year=2008;citation_pmid=18558649;citation_doi=10.1534/genetics.108.090878" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. Wellington;citation_author=E. Rustchenko;citation_title=5-Fluoro-orotic acid induces chromosome alterations in Candida albicans.;citation_pages=57-70;citation_volume=22;citation_year=2005;citation_issue=1;citation_pmid=15635674;citation_doi=10.1002/yea.1191" />
<meta name="citation_reference" content="citation_journal_title=Antimicrobial Agents and Chemotherapy;citation_journal_abbrev=Antimicrob. Agents Chemother.;citation_author=T. C. White;citation_author=S. Holleman;citation_author=F. Dy;citation_author=L. F. Mirels;citation_author=D. A. Stevens;citation_title=Resistance Mechanisms in Clinical Isolates of Candida albicans;citation_pages=1704-1713;citation_volume=46;citation_year=2002;citation_issue=6;citation_pmid=12019079;citation_doi=10.1128/AAC.46.6.1704-1713.2002" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. B. Wilson;citation_author=D. Davis;citation_author=A. P. Mitchell;citation_title=Rapid Hypothesis Testing with Candida albicans through Gene Disruption with Short Homology Regions;citation_pages=1868-1874;citation_volume=181;citation_year=1999;citation_issue=6;citation_pmid=10074081" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=M. Zhang;citation_author=M. Botër;citation_author=K. Li;citation_author=Y. Kadota;citation_author=B. Panaretou;citation_title=Structural and functional coupling of Hsp90- and Sgt1-centred multi-protein complexes.;citation_pages=2789-2798;citation_volume=27;citation_year=2008;citation_issue=20;citation_pmid=18818696;citation_doi=10.1038/emboj.2008.190" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7.cover.gif" />
<meta name="twitter:description" content="Phenotypic diversity can arise rapidly through loss of heterozygosity (LOH) or by the acquisition of copy number variations (CNV) spanning whole chromosomes or shorter contiguous chromosome segments. In Candida albicans, a heterozygous diploid yeast pathogen with no known meiotic cycle, homozygosis and aneuploidy alter clinical characteristics, including drug resistance. Here, we developed a high-resolution microarray that simultaneously detects ∼39,000 single nucleotide polymorphism (SNP) alleles and ∼20,000 copy number variation loci across the C. albicans genome. An important feature of the array analysis is a computational pipeline that determines SNP allele ratios based upon chromosome copy number. Using the array and analysis tools, we constructed a haplotype map (hapmap) of strain SC5314 to assign SNP alleles to specific homologs, and we used it to follow the acquisition of loss of heterozygosity (LOH) and copy number changes in a series of derived laboratory strains. This high-resolution SNP/CGH microarray and the associated hapmap facilitated the phasing of alleles in lab strains and revealed detrimental genome changes that arose frequently during molecular manipulations of laboratory strains. Furthermore, it provided a useful tool for rapid, high-resolution, and cost-effective characterization of changes in allele diversity as well as changes in chromosome copy number in new C. albicans isolates." />
<meta name="og-title" property="og:title" content="High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/7/523" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Phenotypic diversity can arise rapidly through loss of heterozygosity (LOH) or by the acquisition of copy number variations (CNV) spanning whole chromosomes or shorter contiguous chromosome segments. In Candida albicans, a heterozygous diploid yeast pathogen with no known meiotic cycle, homozygosis and aneuploidy alter clinical characteristics, including drug resistance. Here, we developed a high-resolution microarray that simultaneously detects ∼39,000 single nucleotide polymorphism (SNP) alleles and ∼20,000 copy number variation loci across the C. albicans genome. An important feature of the array analysis is a computational pipeline that determines SNP allele ratios based upon chromosome copy number. Using the array and analysis tools, we constructed a haplotype map (hapmap) of strain SC5314 to assign SNP alleles to specific homologs, and we used it to follow the acquisition of loss of heterozygosity (LOH) and copy number changes in a series of derived laboratory strains. This high-resolution SNP/CGH microarray and the associated hapmap facilitated the phasing of alleles in lab strains and revealed detrimental genome changes that arose frequently during molecular manipulations of laboratory strains. Furthermore, it provided a useful tool for rapid, high-resolution, and cost-effective characterization of changes in allele diversity as well as changes in chromosome copy number in new C. albicans isolates." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7.cover.gif" />
    <title>High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mjtD8eRgvSBSVs3ZF-H87Fwa3LK6ck41kQH4vcanjX8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459204","apath":"\/ggg\/1\/7\/523.atom","pisa":"ggg;1\/7\/523","ac":{"\/ggg\/1\/7\/523.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/7\/523.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/7\/523"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/7\/523"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/7\/523":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022viyc3B7C43VfQ8Kr28vYqrLoDiKDSL54zdA5bYKpO2IKYL0FnTgm5RfiO\\\\\\\/vJSy+f5rf6DGpHNHq3g2w6EzCDV7iJeaD8fC9LEkn5XRaRCmNXSfdF6YT6LHSVlqmKR0VxuMxf7KDT26zger546si4PmIRxiE10yRHue2KfE3UN7vHRZM0cwg0KHz\\\\\\\/AepdyYysgy1hlrTQ8JoO\\\\\\\/Ogu0jFinz2SkpGx3wx1Dggf2mPvIIlV\\\\\\\/GSYCFzGBYhlzK+hSwrdIv2Je84vMiIzEmMDSOaTzIWHeqbh6TSKLOtQcBsPFiLJ6t265wCAGrXyz\\\\\\\/NGqrCo\\\\\\\/xyD2koMVE5yr4PS0RS+U233ZZLGY\\\\\\\/i2PVbqpuE3kdWj5S8Q6uw0iqj1QJzWTnnt595C+Mz22Pw40N8fDesSa+hcHRSZdTY74XoMLXEH3Rz+Ox3hUNECrkFQbT68YDlvt2arSPHr\\\\\\\/ZrDC8uvRHw3k57E10WJoQaEWnMc3t4tLONDT8lIy1R7FWXzVbdJEPGx2XZGYFDHYwLzUoEguP0Yk0CXkRykBxWQL4iBY9oYWAp+MP24geS7b5RgDKrOqoAJAnXB\\\\\\\/J2gzAIn\\\\\\\/FOsvZrbSBdRyt9ZPoGLBKf\\\\\\\/uPCA+6NWx90GpOAfMbzgwwzGRbuYtr93BRLxgTHXREbEp9EA41AYtHlYdMFA7wJKL5ct7jzyemvRnHsYIIF4ssid0yDi1sKKnfiVx+sa7dJEXYOmqLfLm7Oi17FPKCZmWCQhurgxKfjBULVY6zwj1tFNZ8UI9mRsYb01IBdUKR3D5fDiMbIG9S\\\\\\\/IxK91IZQYSmxrZ5E7S2KSM4bLMyvl0SGJS1HY2m2DwVzt6la35Ewzn\\\\\\\/QlNfLu55QC+pPkZM0lG+3cQtIG0Ax0Ai3eIKokHoL++hFeGRQu\\\\\\\/GBH1\\\\\\\/UV04W\\\\\\\/CxgB2kYSWdC2+xJOJ+5TqfuxGUfn8WHQuosjY9hY7AQM0Vf9HSwYzgABSKbiKAey9i1xW6s6r\\\\\\\/Bn7XG6j5H\\\\\\\/YZl3y5DvMhKG3Et\\\\\\\/jICjitoWVnImTMCJC4h8cO+\\\\\\\/2X9H+FF2FncV0TkmMXskQ\\\\\\\/ImQUtmJZfmaGq1TyGj9eReNtZ5rD5p9If39A\\\\\\\/MG2Z7sOIPt1gwEI3okbWnWBSgocFhThOa41NdrxqmId+GH00DHlc2MveIY8IlJ+5B980acF1rfXRXu6DPKsWGUekgAjVy5j4ToPNjyFJQ8ipB\\\\\\\/st1dkaNnGU4MG7U7hMynvCPrKb3xRI4iL1SX6vx7bttwD\\\\\\\/f18ZtODvt3j5vOA+QP3ilja+JMHLV\\\u0022,\\\u0022iv\\\u0022:\\\u0022d33T9YOtfeOhAdVQpRMEqg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b9a91e9dac67d68254dbd9b84e4d6c48bf4b633ce2f99234e98b2d41e8756561\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DHUYYs9xEd9iqUdqwgw\\\\\\\/1SjeuIerew2abQBgYrE4fCUflbomEaBMcGBEsnhw3Le+DsLul\\\\\\\/vY4Sm6+wQlMCmpyBrXAt5suzTUftEH4KRpYaW8LGmsdFEPn21L\\\\\\\/dlBOiVN\\\\\\\/e8VMDl+\\\\\\\/LCRoZB5GpaV6XciqOh7rGDWtiZgCBiiSnGDKeZS5v8edIcrW0DWD3DzLbs0HBoeN9vsWP6fI++elubhd9DPc0ViUqSgU247dIgEo41MQZ\\\\\\\/tXFL+RHwENmUb9Ik9ScMqbrdZK63jlON0T4gYO69Tt9bOAgt\\\\\\\/G9MyGZCaV7vsb\\\\\\\/SOhna5JQtTjaxzhk2m0fjXC79oMGozHS9eXMajUxSGeizD8kJZUnC+auI75SW+EoEDIGvwe\\\\\\\/ahc4Oo9NHULLWjphf4+x37YKbcb7x9LujQwqiB8Kaug\\\\\\\/NcEmRL06zranO4MAmW0yVjMnFDITFfeEsmyJNXsCcMi3FlKlXHjI180se1qt4d\\\\\\\/OHX1mXvZxHKvdqpKhbbYMw+03+jcU4aHUBYl9MdbH8L\\\\\\\/S9dZKi\\\\\\\/X+VQ58xnO3ZVxSa58QQcB0vQj3iWteesejokz8oUlmLhDzynbOe0aB9j0+i5djp8UO8g0HXcV9akTDiE8s67jeUF1miP1a1pc49KM8LBe7AWtnsSnLQtYZxjNFDaI+LBjA6hjQYJtN5YAq8JgYrCMIMWdGtxx0GnKb9jRbuM3hRk5XVuPmEBPnYLpLB3iswGfvG\\\\\\\/JGslDSM5a0lJYDAFRMtv8b6Lrrm6kSVzjlji8AboeiiOmbRONjTMuhwbGtvRN7uYbDvyo2HnlMYPUx86SWOG8UTc\\\\\\\/zP9Q6TuQpHWc0gCfyYhdKq8IlmuAn3vBr3mQtLZVweq5nc29EREP8WZKq0DjDbSiJW92j1OJYIHRVPu8do54P6e7PlAtcmRnrXTV4AkopDPA9PjxzacrmK9X+IYyvSMSISTy\\\\\\\/TBSNntkBJ81Jzy8TH\\\\\\\/7JD3fp3o31IvcMZUe\\\\\\\/ooTrMsk\\\\\\\/JxuOBG0jEzvMkLT+3nnxQ79zv2+iPuy3fA08lWfBSeobcd0aU8AFR10KEcZ90OytnPEkJ3Qg8zcswz3+4mHLj+ihnudInSrFkwep+vfza+VnkYBHC0HbVTg+P0bwVP1eoxnTQmv9nUHzMz\\\\\\\/SzW7Ae03Fhsq7AxYx94Fy1UExyvcHHBw3nTqGYNxJLh2uhWnMLO0VnfE4nWoazAMNi63R9G8isiyFbg0wuP0kEnWoj23ySqLlSXUmbnbrKhZdZkV8wLBnLjX1oEzsU8FZvvGl3tpc+WqRibTf63Z77m5Nm7uLYFqrx4W5NOvVLZF98uEdH8jZ4xs72zse+TbCZHcRMO95i4e5uo13yAGWozieitT6NbdmJVYInpC+a6Eou\\\\\\\/Dx2siAUb+gT0jayBrY2WmcpfkKRMDHLugY+9gttvPb12pofIYycCBSDzPpMf5TTpEDntUQTMkr3xNMHdBIFYEsn1LBKYDtcQWG6tGRi6pE0u10MfElf+NtyC4G5so+PPB5UpYYkBZkQjqOKTWR33Ddgy5SoatXCbOQ42NzEcLrEI9xCZBlLIZb5NVTYUrN57k1lF5VyBcETj2BzSvRm7zzsabaHvCQVi0i+ux0miqNORp1CdLU7qBglTiqGKhYbdaJmeUhHCVEJKApbcLc\\\\\\\/DbA41oFE\\\\\\\/MujHhOqkRUCR+vOMwJahSXc6\\\\\\\/3eHyqKa\\\\\\\/fRCFWhwwSl8TLqLNNfoRienC\\\\\\\/DYHy468rO6TM4ryjYNT34vFGwzHi\\\\\\\/3crvNXoVnnoWE6lNyDsOmNALSZUf\\\\\\\/mEfHc2uwJMVW0IXeTvOQTdR\\\\\\\/487dD\\\\\\\/H9FErekw8GXZiON3zpTvMJUxj1LR+qSdj0qmBuiivMbDZezsQjPUe7ec2bcnjm11Vfppt8bwovsRpGu0HnIHg8Og9UW5aNFYhWZVPc0eHEBypTghoRWEpwwx7LuTPAd3fEXtxHsNTTWMcFsggFWPy7EbsM1e1RUZFbb6aCfhe2t5E0\\\\\\\/0+eH5Rdy177TLeQFwwxUN+AeKfaSqDx08nxcluYH9lBNit1pacaoi3iRbN3+Ze2jFrlXpIzuLXCY6oCjkOow8Sn2xywJSPiV7YeuxvxLeNC2X6xPw9E4gXmuQYWFvMdYXZuC0LmfhWws0pKvUyGqpt52JQLnwbIMKRHKAwpCS6ipKndBvq9HUFxBwDHgmZlT4qXBsrYQbCJ8qgmpLnf2aWFRNM9NaGDPkyFJ8OeB+p\\\\\\\/ANjOt8pc7BQQWM21VS3wYoud9adQuLfkvI4p\\\\\\\/Cpmrztn8PCY+VwJbRbs\\\\\\\/4g2AUEhG4aSUQh6VBVijQmKj7w0ntcmegsL2pavZv4Rv5hFNVj\\\\\\\/By8kzbkMRHoJQr0evAUoGmvM7rLMf21QmH515sHaIFhLV0DH\\\\\\\/C\\\\\\\/nFOj\\\\\\\/mGng\\\\\\\/GlpSV+OEZ+zgiDkeXHI+qV2U+AdcYhIZ88rtv4SJSPMrjR1EG8V8pqXNFFenG5upT\\\\\\\/s9Vl44sr5nPCKxyHbnrUGQrHfXDJkjyz\\\\\\\/fDWEkLCnjkroYeFiY\\\\\\\/0fruey9w7H5q6er6diTMQbMfzrcXsgEBe5UqwBR2CVYCJ4a47VK6d5nwmUUFiZEI4p7Q0M5ZrfejpZLT4W1\\\\\\\/9t2VCtekgcFKHnZDfNsQoudlbdFW2dVEsZbxS2bY\\\\\\\/kZCYwtFUzBgMqte2lkrwsO+lP+NYRL2TTRsD8wcHkKWbXuYM3hoj0+gxXSJz2bw1Ls59mpwmSA4kw4FWAzq5WBtN5utXlTb3MZwV6p3O5ElXCOJIbUeKAQcj4XoaowQLpjOLUGoDzp7IMimnuALkli4FOpSsUoi7d1A+ib3\\\\\\\/zDz5WUPYsr1zamPwdwJdqP+jAwRlrn2Fa9JdpTYgsX8YCKn+eDfkqBg5Hwx7HrcP\\\\\\\/3TiEjMI7PaEaYNn5TVXofzgrtzoDmgG4RQfnNbD0hfa6xTaOpwYELhjVA1Qt2ic8mj+koKCHYqyCd5R4QYJERpj\\\\\\\/vzj7LmMwmUd7+uNaEn0hDvzKNAbZv+LYolqCNRfhEgFmE7BvHxCZuVskPlYJ29ygx1Yq4iflVgrShCTMqHN6ThaU1UUwiPGI+rtL+Dy4pu1ehD\\\\\\\/Teb5MTpCky8PLTvfl8lO+gvBp6o+rtkXZ0yUii25lOi02RWMASStcG3k+67o8h\\\\\\\/WlYnE5bUDpn5r0lRKaTy8QgKTis\\\\\\\/pZAL1K5jCIJts4J5+pPI3Z16XcbChJ3Jg+2YAB+jW893FYfGNsxCeTLOZrx70y753lrzmmgheOICCVhtEZL9NyZ4rFzIDR8AbmdByZTJHl+za1\\\\\\\/ETlugk2djXLwrVvsHnPdXF2jmZmT9vAm5RwB+jsiJH8xg5qs2EnNfvKOXpOruDx92dGxugfjfBEcWKQdiLh0+LeTJiQlnJiynJ9F1AvhQ7LfHOCnAhCp14ljUZm08YL29szXmkNrB5Y\\\\\\\/7iE6jUngy\\\\\\\/sqYyaBg4boOZdqVHZNhMz4VM1fgJNmavZ3ukPx8JWOA84py7uYk8w7BdfPTKpRRuZ0C0BexlSHfdUz+WVtao5lSv3JwsfGFdyyBqevGWRgpqXGMUoyy13cuuYwtH41Pug4agfXSW1MB7UGiRzl6B2MTnHynGoz9F\\\\\\\/UAmoiT7Kw4Viihq\\\\\\\/Po02PHUHaULA+07ilre36gIeU9r+bm5uavJ0+vYotll98cT4nEli6gh8ntckkzZ+bMIU0z0VhIRUWfwvAnGXzU6d7Wa2vuLashiL0ArNuj1gMFQ3iKTmGNQO2LDfKmZsCbSQX8gK4LM0uovNeRbMRjCmi8ScQL9LaUsl9hdH1EFZE\\\\\\\/LbQle06L5VSKVcGBeVIIABIeMwTswfMRpb2M\\\\\\\/MDhyVOwi1QYOi2SB3V7sW\\\\\\\/0gdqOqfX9HH8IkZTvkbiZfUwFwAZZWIeE7wZpHEj5tpaZEVJgylhjOMGU9ZCXMlW19OOK4B6NwuQy1fjVmgfXaqLmm6qyRYe+i9LMh5ZcC3pqpMV8ieckHGucHCIppm5++hunraoR2d5qEnHH2DCQWO9A9OEDlmkT7tAH2Rx84x9m6rn1Bx4zH7jwO7hKs\\\\\\\/x8C3TWk4wyMzWE8Vhux9Z\\\\\\\/EGka06zlG27FuaP1CEODf08FmsH\\\\\\\/4vWMH36Jv\\\\\\\/HQawtEfwW1QeXAeiaoFVXpn5gs2pabb0r8rh97615tIc0InXiXX4hnEEURGfeHcsr\\\\\\\/RTS5DElPcqSPLgMDxNgk71Lb3lTKFOPCgcR9K3qdMGtOhQ+DRksP6k8NhkEgwC+\\\\\\\/XIjZ\\\\\\\/2HnkSIITCDicwOzEp4A+3OmmnZamp0ml8IBeaEQ29AS4pqqbUNpuL9Q7MwsBOAGeHVGHfRwjohO81LoOiAsdeW1Y3oI\\\\\\\/c4zYdafVKaWVfhnZq+i+2Xh0shQrU\\\\\\\/Xg+HkmcI+Vk0h56eO0nBqwiNEkpUyeIze4KsrvGZy3hoNrcDp+TqM6tn4WO1Vt7hTnqtqTAGmiTtRDT5G2bg+m5MTS48hkLL91l8aEUjqOvt5AAfFhMZyLlF0dySSzbktUP4AzPRkFXS4FcA4JO8\\\\\\\/ZWT656o0BtdED0NID\\\\\\\/t4tjWk6Q9BLuchsndQFpSu+0JWfuseUeAu87cb5ANGN+hqJ9g3QiWNFM4d5jRzav2KYBZkxPm4yIR55dJHyDZpOCfo8bXAU4Wh96PE9Iqpg4PjDFxg7qmpnmqExDfjCveD0pS68CNo8R361iNUl6RAyD9OHO6in5yYLoBLr1A6SoVXXfC14GVYjcfxAaP5hTyFt7N4k0Q8eUR53YIjlegDv2efxjukj+yFjox6fbGbemDK1N6r8MZVmQyEaGOiRbNBFGlle27K19XMIlA1Z1mj3965ZbrVllm4DcxFCuaIcqydNVFlvBxcliVa22+Mu6WTQEjzaGjv\\\\\\\/gfdQI\\\\\\\/shNR\\\\\\\/6iV6AFtBGTQow7pyV6ndXXa1wn8ohYRMkkrxvEiFS4H+vIFxkE\\\\\\\/WkYFECWyhnSNtvHvUOnMIDnIoxMZheYlW\\\\\\\/HcjPOFaKk1Q7qfx30S0HLALIxT4RPHSJrxTbkA53DTMrRxHxCPH8tua7cBI\\\\\\\/MEMqUxK\\\\\\\/aiUtKsO8WF8OB0Wbx6UU4jQmcBKV\\\\\\\/lMvoDlC2NriKgEGhDdRWelSsITkpCH6n5\\\\\\\/hY3kC5kyFmoMlcjYXLK\\\\\\\/\\\\\\\/bFBsgfViD6rRL8kV8UTtuNeUzcAh2k\\\\\\\/icUTpATQW4IhZx31Ochk9K2xdygekyDfOTWENibY4Cyzr6JXpWZes80YazAK0XRG04680NnvDH6qT42WX8pKYAyLnnSlYD+ULMVi5CvZr6NIcQJ+8ApBLHPJBrVbLq06TCKlGD8SarL6vzBzkpNLVHSB06xPSJh+pHXWc2jc\\\\\\\/BAxChjPn07MaCNYPfeQXS3h1kt4liwRzEa+db5nIOkWe5aVLKlKgeZuGRJ+5sQ2bRF2WlW6mCzb9sHhOyHV6oFChwSCYwdGcE92\\\\\\\/iFKkoPQ7EcfxxIcCQKWmZQdfHkcu36+Ugu+SOej6rICFYedB+Leeni6VvWpCoMhavLECrlpkK1\\\\\\\/+WjWibBaUL9zkttjp89G1TPYUiUrnxNTTa1cSozh8OJ+lEcxNEsXLArJeFIk9hln\\\\\\\/IVQdHSyFyaFrNfZI+FOPBeH0eTWPoiTnEHOccNeABxNkcfuqeON2NjfQR839a1k6\\\\\\\/k3mX0fqGFEV1YmOZCLTNv0qBlvpMfHXYbgtt4vt8Tey1QrMQ\\\\\\\/0fJhtSmN7dXhpaat9gGB2asHZldYSJ1KFIQE32yQ7VzoAcK+KIF7qwXgnb795O4jMF3Ki3QPQBc\\\\\\\/GK6DjaA2w4q269d3YkLb3Wc5aTXy\\\\\\\/gTSqk9Wd+lVvtr3+URt0Gou2fuaKHRrj\\\\\\\/7f27M5P+kVeBi5PbUv\\\\\\\/kdT82GxxKIF0gPUbXyOzSYnJiu5gLgkey\\\\\\\/XxXLcwha9e08AOW8KSAajTYkbeoVogToaoLZZ7Kmn7nnXtbKQwi4VcAyCqbGLZ98T5jbG\\\\\\\/8w6wXwR+QVPaOe7LeqfD7locd986im05aYlD1GrDNqrqe1ASg5UHTu0fHsA6Nrxw2Tm1EahgiqoWupvSOHrrtPBwLDGklrr1nqTlHmXwigLaXGadVbAyfkPhwVU77NtyfNLQAeh2UCHhCDCMDZLptSmF4oriZY\\\\\\\/gp+vXSoY3DBRWf\\\\\\\/SrDXGMyEJ1cHu9icVXA3eWytfgr87o9mFnAXM543JbR3RcufRy0XFV\\\\\\\/GEM353AGyoR3PyLdlwjIrQYCisfjQYdCBW1l94+3cNKgmVmI5381cWGlFXuWhnvoYtBXtDJ79229g4XwqCBwybM0VfTHa9szkNQu8Sj40wWbHIRSbBy2W73SHkOU74OQTUF9o022N2+mfAX+20ObWXfZQpCJ\\\\\\\/ZpyTn6H4d5SA3n+liSp5Y6FfQ\\\\\\\/pYP5xvTbuNVwc5hX1tYLsgv\\\\\\\/NkPX5YCqC9v3JseyGtpBOHKw\\\\\\\/og\\\\\\\/LMlXlbhuYgTBjD59WspwCPMXa\\\\\\\/Rb07kSMFx9YSGaM\\\\\\\/8RYXy\\\\\\\/koOuIbv01M1aMZiUoWu9iG+rU9XGSi86WsyIPhSlxQXsshvFYZSRBO73kXg\\\\\\\/++h6ANDNh3koE+TfzOFMs+MOQXEUgZtnsdrhy0Or80azFl+Dr\\\\\\\/E+b\\\\\\\/\\\\\\\/UaViy5K0yiVto++d4UUJAl\\\\\\\/qj4\\\\\\\/Gf1h9LSjwMfUK\\\\\\\/ENOQcn2coRT0HAnkfInUGt\\\\\\\/fmlwRer77dkoGDivf+el1Ceokc0xTbgw9reMVK4D3MvWq5zhW6fPJj07jlOGZ+iP3poOffiyRPqh4BB\\\\\\\/kbsf\\\\\\\/jZbu6G2wHu0YEeZcNE5YNjYumZj4OFTB4TqAESVT8H1Ls8rkfSptHLZzHpwewigsrXctxSRLFn2eyFqGL8eSJer\\\\\\\/n1IfZpC3IeBwPyKXcHJ14fL0IhVr8neNx3CzCOarpZbhSN8cdNn8sMQCLLAA+7ZbZivXowUjw0rw0IG2UEgVoR7GAiWpEpOQNmpLZiTqTlV1UNgmp65Ux83qlKSKY30qCh1zJ+GLz+3csbPhIqLWyLZNt27Zg4zsTvk+UA8dBODgoZMv6h7MrtD\\\\\\\/v+8GD17C\\\\\\\/679qNK68tb1WSIRVuZpopAKM71PRDNZkOaslT8LCkhHTS6ucWBWcaiJDglo+pvlvUM0i7weI+zspH8r\\\\\\\/jfM7Fbp3Fnhw7J\\\\\\\/EQtij0zQZJXOlvjJ1Z27aw5CHNZ84BTLkNAuBsbWo12A3z7CveOWKND0RonR59aM7AtC2kP6mZCvco8X4bKiZD2CVhwxvVVIC+sFQpG+DDw88Oma+YCotwNDjAUB8DlXPeO5av0yHOyeyJq0Dnlh3+7hCpFmP95YrlWSsNtpg4lZPPYq08WTUvCGhfwLpCZ8PFe4mIMzIEW16L7sRbMKbFBxif6h4RllWkIvUBUGRvod7tMwHMu1Ae4oNU9q+oZmPn3MHKKZM0\\\\\\\/3w8oE4xIm9+aRpJRwnpCltBl5tE\\\\\\\/ptPWq\\\\\\\/4HDvtgUfotJbFoZmxir1o4IZWQQtc9\\\\\\\/oLTlLFZ\\\\\\\/u3V4qroT0oaOSaHQT1Db6n\\\\\\\/7VRt0mjdIFik6qjfYiI\\\\\\\/TeoJr\\\\\\\/Kn+AT1perr0LfYV4YjaCRhQkd2lIT4VR0f0R6HALj249sOoKMCd4vyNtEdNFXI7XMQ+T2DPme7CzayUd0xQ6j0IMbLMPg3ploRQ7j1C3WlcFJqHPwupvVYM1uW\\\\\\\/GMTw9kLKBx5iGSFE3sojkn53TmTII+RaPvZWBCxxjO0qViDTkzrMLnKVEzlxCg3w\\\\\\\/g89N\\\\\\\/\\\\\\\/q97A2DHgY\\\\\\\/BKyotryDCapzrDN+N6R3N+f1IyGMTZHhhncx9dpGQSVJns4rXVOD1dsHU4XFeNxccHq+vNjaSdtkYjkvBW+O14qwJ450AQ6riRpVOE7HwZbPz\\\\\\\/uBkjQyAlLRkcYd4TgxirD7UvZWWz48b0RP8+zY0cwsTBEk66dxsKH0i3qhuC0WNmayxIoPA++R\\\\\\\/RDkbmDrlEd46pV2YWxrcuSk7dEVrihbzxfqL+44nKHauSEhKLJaXaD0L7FtEqsK1W2rckRIPXWwPE2TqtqpastU3a88dBSGeH3ZcismFmfwo6XsyoYFOtDLholUxC8nYhTBgNrMSgu1FANNi3lz8dw99ThERDEmSqW65s4K7sIQ8dYMI9sI4s9nYeNfDg0X2ge\\\\\\\/xA9yMXp+YNLkyyD3xBZIzZkez11H4vbAIHxWpnIJmLZ6YO37PWjZ1vwkEfICKWW9YTTi1H3McaXWD6nLyGw\\\\\\\/PwznFAK3t08q21M9uid\\\\\\\/asrduJRtvDmDTv+Hj\\\\\\\/kTXcjv9Z5VsIIczGiPMQRlyUSCvs\\\\\\\/FBrTXTTfX+acVBgsN2mb78jEsdRrkr+K+pExPY7V7fNj1+pv0T\\\\\\\/xWzQM55EyZOVf46I\\\\\\\/B1vHvklX\\\\\\\/Nib14rtXkbAYXRf2WQLBO82jzq\\\\\\\/yP+H5QATD1pB9fuVD0dKbqKqRLZt1E1kCXdTvGtrChAaDB\\\\\\\/1AfPB1gdY8Qbc+RuiWVeolDK422Fts5K3UExPAmtI4\\\\\\\/6r5FOv3tGurLQ8s4sWDIVSpCAkU2LIYwhfk0A8MOXdBY1C7GnBPY22pM\\\\\\\/sctMXsgT5B5YqDZ+mwhDAJKdCZDb4u8hU41pKdoKb\\\\\\\/sNryV2gJHBoCrw\\\\\\\/XH\\\\\\\/9z1DHaIgHzJSADD\\\\\\\/24uaUxe9LG\\\\\\\/dvCX2+gWG4bnKWDghKGw2QI\\\\\\\/9fKe0dH81Bz+6j5WAmrVzQDP9lMG\\\\\\\/b46kgtRB1CC0sTS\\\\\\\/Wq7CCUaqH0TFCXk3UyTAy8OtYEbjjud2K8CnAQIWG3PaskjFJaE5N1MfqeZVjgy03kqvY4OP5e6xud+LOI91A+HwVrGtam\\\\\\\/N9V\\\\\\\/nqCA6tTnaOLsv8KghoF8n10\\\\\\\/6sAnjvQ2j6ZuV08lR58llV1RO3tqXVskl+AeoJ2tlJRacY68nxtX0D6AgWhQ\\\\\\\/8m6Okf+EWPMIGEn6bsovJWagS97\\\\\\\/YecWsj7ZFaj6jJtzFdK6+M79A5x1CSZDfHSgGvEdM25BTWtZSNIRNoFrQyS2g87HPtzt6DjL2T6MqdsZSytJm4n\\\\\\\/3O9YCiX1luhd0EZMUg0NvdvNlmvrg8bXOp2RmG7icB59Dsb1w6+lKq1PwfF2\\\\\\\/6kQL9tVMF5MTH0EdiSCnqm0R1ruGWytNQX4wyTk6rz0HFzE6qQj+6u\\\\\\\/37K1gVxTOCVa50a1I8pcA8PkXQE6j64F0RzuQeV6tyWLH4z9ernXnURyde17vrruXFN47WqEavXzVwiU7eWW4Ph\\\\\\\/5QQmzRRTR8iboIhb6ispWL1AcaAveUMLwwiZYVWSa7IP1\\\\\\\/ZjrBLEyRy+r4YX02UUOH5tkk2AmJt4H2883YLmrkyqcU9UBKh7RZySva5Jmc7jvw50BQixr3KxnFScyvMDT0eBV7yL9hrzPie9PcShLwHdpNfz4THsrduDhrVJ3pPn+JSHWpqzk5YU9vvC1iKgURPRyJS0Tu2OgXMeed2MbC305zjav52RB1s4fXU5P+4xMgwp6tmVfcw5E9D49tENydBPL8YH\\\\\\\/doNDnRNLEen79wYsMzYQjLdbQz0RvAMD9GnRyWT0OJSORJUFsxwOChZ7LTFqSNtQaMnA0bFezr925JYzKrjrGm1rPFniFeXF7TDW78ns7FC+iiB3IxwSz\\\\\\\/BLlNS8bcSCtFs+YLXFS94DR7X+79z7eXhgq3BNpPEGpkIiCYqIKJs8lFx6O\\\\\\\/ifMuweAqh\\\\\\\/tKMqgjECWRnG3+XUFVEhZKUzl15+E2CfR1fHCHdUe5U0YZ55L+tKYDtI+eCV4JUcJqmcMaUIGBxLq3WKQ5DngFan4fJesa8Ygntj+EZuYQsy9CQiFAmOY5UiWrXgM2\\\\\\\/EkLKX5W4NLWSq8mm\\\\\\\/tGZdPNb0cLaZJKjxo5423jYTI\\\\\\\/eWf8AX\\\\\\\/mXSx6IDl1qaG8oBQIrBfZPMfnjMd+CYmnehPpK+TwKw8NSjFD2pEX+6ydRR77Acs3rhU2rrEO\\\\\\\/9qc52DM71ShXhiga4LgiLrOscRgqjzwbDsIHFfbvcmW48I4NWSqj45dK\\\\\\\/Dn0pXbSk2\\\\\\\/bMUMWA7FaNo4C2qDJSGtxGgtbGdqV\\\\\\\/lVDooHWpZWsHR8okV4vPazvAjRDC\\\\\\\/FrYw\\\\\\\/GUbVUfKkz8IETK5VcDeEfBIJyX6j3cTaVQ3MUzXsV6YmhK4YjnOZliemrUOQSGV5+hNebFdnuJFLxMh1lyYskS\\\\\\\/6QAXSEfRrWAr50Q+JQHL8IXDc5B1ftVcA8VMyF\\\\\\\/ISAMdgvOeZqNqCqp0saWdJxtAzXTSn7mp9T+UwxIV\\\\\\\/otMCUeftDAj3XZN1UNBLdOCrtyb\\\\\\\/2s7bQbuKYnOkkkuVn1DF1Sb4N93bD8r8NHIAVgXXngDCJDZgsTA\\\\\\\/s7Zj85+zsElnsCQZxu\\\\\\\/mdkc2LN6xPftzoJ3REEh1UP03I534p\\\\\\\/1PCNOEEtE6d94Zus0rGmMqe\\\\\\\/Ogyi6x3z\\\\\\\/AF3xiyDGqUB59L8yLjDohaRH2i1yzV7Frk71nWo+YEgruOJNcMMUUnawU4lfpLPb2WlfiADCP4oO4SeBfJq3FLhpIIrlKwjdLCUdgrFBFRvmfihRpTt3BYIXBiw2a+43+xYOOf9eKpbsNDjqnFECbCqnGsdPaKyZ4LZ4X6xKqQuETfjlLIUhqWI027ZYOPvwIgj6pDtHxNextvKPQkstGK6ex6pPIHlgc8vXSZJyAHw3hcpkFx2usUIfQpnOCuDgiNfgoPuNK\\\\\\\/taDuovBm04pWydjhGglUF2NIaWMPpo3MFONoUxOddpAW5Em085+yiTV\\\\\\\/iDrvrXzroyUVoEts4jDCz6w6+t3sFGruAaqa0aWmIneUGWI6gesNe0tZJ43W0SQP\\\\\\\/e3iE+SMAjrPPilnd0Zbr\\\\\\\/kWTmUR+W2zyMBLfk7On6vpUCf7vPe9SehcZ+OR27RKay36uE0O0YH\\\\\\\/ozRC42csY7x9NPpO0SY2XPUhm\\\\\\\/SG0JpKT8ETJKkkUv2RbZ284Kl7L2Pzp8Fb04WFqE\\\\\\\/l69LO\\\\\\\/MtN6hllg1z4YeoFHkMYAJPzN14Lsy2LgAw4RJzaNO4i+rumh9xUZHQyNvnAwUfqHa0rug2Tm\\\\\\\/CkMtxFLrKMqjcJecjJUG\\\\\\\/Pu8UKBkd+pr0m6BdOqT6x0U9b0\\\\\\\/+NnF9SYXyCe9rUH9PklZgzRr2QiIOJWaConFIj2A2POBHnRQnah+03ZgOzFmWZi7x7SXhbH5JXlc32Nksx+RpJ7bJ34jU1sAUC+2eyI017AWtW5YctMm9Z9v9XMsnZA8f6J0Uq66GCm5J3bEZDPAJYdaYgmr3CMt5h2ZQ\\\\\\\/D9S73iPyBk+dNhJVvTaA2gVYkJG3KsiT1ptpaxfr8JdGGqCj15EggHkcDXad+zHNd6EqIU1l3EfoiPzkT9E4awpCiHsvYbMlRZbhqgVBEZYpCFgG27v8cP5lDCWxtChYKj1GFP5rAhh7J09gXuIV1QZDoc9A7YLpz6bMxLyvYvC1\\\\\\\/Sw\\\\\\\/6V8NOz4RaHvgEEVAqu6EafvM+btdrP3Q+3GQcjTPlap2Qf0rdcwvV1AZWkvq55BbbOozyJqJ1BjL5nMkV8sbXfKk1wwJ943tiIe19h40oIjov6L5N0XjkpjQv3wkxo0cMdr38CMTOLcTT4x\\\\\\\/sKchztUJjZHTb52mLWLJEs53rY7pZn0LbU7XYnVaBUP0SGZn5+ipaadU+e1agdnnRmoBtV\\\\\\\/RIlLIalkX3cSCjO\\\\\\\/5dgXgmb7xJjBFsRdNJLE\\\\\\\/HU82MQI5jNyDcT1Gyu\\\\\\\/2rS2J6Z3zkKmrRl3BmazGLCiLrLBdkMei7QKx\\\\\\\/euX9W7Zd46hSf2nYSAViDsvoM5viqtAczCUxIJg+BxFP5uKXKhr0MIfijkyGYwlkvK\\\\\\\/fcr1hus8qgn76pLJX21NfVcCixv6T49Xth2fXPKatvXXsQhkbs9C4Tml2brs4+EvjOImlnAsHUoYqG8RgrezjtrDJBuf3px7mPr3G76DoKfFMPeJgnmTsjymjNAvSu2TZn3GinSPX0YEkruLsTbVjxf2Z6o0UDVRhiDWILrXSa6O0GuCaGpvNXZ3OygViovEe6y5PYPAbrCE\\\\\\\/JL365IfyDKFHy9FplclMm8eX+OdOD22VhTREf57byvUVTTWu1JpyKOnxEzhy\\\\\\\/X6YHPxvv41EybflUgh0CPdZg641eXiD+UKF5aDWLA0N8SZjvirB\\\\\\\/bOGfuGk9k8GY2PYIAXq1xw2mADYDyoDjG9ZnLB3rW0+3yDwpcr2zQMAJblbvdCIGKYuikfZM\\\\\\\/fHwgjJqaLsZXes2nTw4HvwM7fJd1aPousv0ns+mIHvGHYEGaZWB2fiW4Kggo\\\\\\\/laUeGKT3Clos1vzT96icwcezuXce15qiIgLvb5xCzik9RkVFt4goTy8YacUSTvhK5IFkqMeKlxstgohYdkUZLxbHFj\\\\\\\/DBtNXLU1wXNUbih74Zrv8isKKT0ZLr7MtSa4p0j0w44PqU9rmTGHKak3pud8Iz\\\\\\\/CMuTCJklwu0FtlzzpxicXS7w7rRF1Y053Dv7xWS0dztKenhB9WB+TUF1Ulj\\\\\\\/RCNvJgjmGznXHkNMx+EJn+854dWjKj\\\\\\\/ETCVn52\\\\\\\/yLln4mPxrwvunUEwpVIKadGZkdio7qBDVijdaSl+QDIYkC9EjLciLFMJvgpB88rxQqijgPKbGAzcwK\\\\\\\/W1ssI\\\\\\\/wuOnuvtXhJOIsefZTzZtjAlxA9oGxMU2NqQi5LRGOPjMZM6ppGkYRnG2m9wh6x2c\\\\\\\/LHconKXBW9zYggZrDrYspGYDi1LUvHqGXaq\\\\\\\/1K3t9bhsOweWG4s+kGI3hctZauYtYkQwh9pOc0MP86+jYWpPRTtUHj9\\\\\\\/nB2dXyrO0XxgF6wAMqRTqbsbVSWoYlDElRyMEc7f1QJ4lSNv7O6xpD9GHNOhet7kCR+nQBK0fmFwfHk17Wm5TIi3ODPgNjlPaDke7mOprIBpQ2juvnecCSk+ASdwt0tuyWv9jbSG35h1kaUKv5hHSBVNPyKz8cNDI+5fYSRsNrsN4qo13Fee2CtVEUoUH\\\\\\\/5oY1DKrz8LsHNbOxUKNwbLoaLDNH6Tuzop1JxySXhlRUkG5\\\\\\\/20PYjgjh1PbzthwjUwABCN1Cilt2Ps\\\\\\\/cLjZ2eC2Tnj36rWWfVxI6SUNZqMZE6I9GP592IkS8CUzB7yb2jaqzq33ExEWuPjRkTCBlI9qdTBbHHRSEz\\\\\\\/rO5OJnrsYBy1gJEMJV45CBg1P\\\\\\\/ScChhod8L0ZcmL+W9lhZYBkH4dZmruwxN\\\\\\\/ViiErrX7Wgp8TMC1rskurCuMo6iN84JZF7mSxSOEMiarTaz0nWakt+e4ZOk\\\\\\\/XQ25rS8ZQDX9jspqdMPQ0d43Oxgz4ec0A0+T62vRzAOp0IitwSokCRrC4oDoa2vv7FjRsmSLPGD3Re2AdwGzSExvkRgAgDIePIrpdR3PHksQjrfympwYHlw6aMTa91cUhYWEzUOZE3gQhkjp3H6d3InyVnyTizRCN9pv36iVG1PrNBweciNpDcU57sibRQnW4g40h79mS31ABCNWsoYFvV\\\\\\\/FNiTF6\\\\\\\/v8X\\\\\\\/ns4JPVWuZ7ujKcuczvMuNoHwxacgk36LC3TcqOHNUGJ8NSxoq7bc1yQJBL761pJEK2jxDSedvsk2WG14+Jqk1VVVmZsgHn23A4WRm1ErMk73mTAJ7I1j5K4GHPYADEugq8H42oEYjM0+aROv1CxaR7MwZQPqpm3WLs8wcKhNOw5KjC03Lq7mFR92mOyxkOR3bww0BOgqM9UoiTTR7Ns0i6sVCbNmm0\\\\\\\/B8aS17XFDn0geDBtDXxKRvfq4ClM6rGUfeZYZ03wVFy2eXRpfEbcYzh8iegDU0SYF3FS8kqzsmofcYmmBDpVXM1Kh\\\\\\\/M\\\\\\\/iVLtcoPoMr0InEwidxTa5EgiqXRvcZiIZz5\\\\\\\/bTBQ2AsxMQlYbV4T2+61YZXJS5KNDg5oEB1FSFL5F9v\\\\\\\/6MNfZjRdoVfyGN2HRzVGorNGwg2J8kM455kpMr1p9a8IbfMrl\\\\\\\/dza3vxLdrUsvCgTy6vi7p3TS9gygk2t8DRPRNSkFSSoqH0ABEcEJkKKqx6ujsB9DGCpPzhjePNZJ6QPjd17l+dUpt4yxkwK0AfXPM\\\\\\\/fTiFqsPK2Dw0p0f\\\\\\\/yY+dKQ3CJ2wqI9MSdrB7TJ+ZCqn+ONzEEDk3HUz1r2NhmZlIDlSMvV1LAjNILc3Am+Bnevf+D+hctS4XRjbW\\\\\\\/STbVccVt0xaFbE1iLov44bhb6FxC2vG5fDsGDRBwVOhB+8S59YvfgywX6b2WtwSzaZCimCDCjZVCihdR1r7Mx+WVXwykghGkEyQ+18EE0EkH6jIWyO8PUa0xj7U6RplLRghiH\\\\\\\/TjWqvRWn5LOiLsDcnTn2QBbP7x9Lo09figDjNz1CH3KwT2J4lxlP1wCp2ncznZeLh1iHBIHuSbAZwLF\\\\\\\/6kLzL\\\\\\\/kxKgqGGWaoGNLjSp1iE71a0Wq7byqj\\\\\\\/dSvbqFHzyAcqOCq6pLAFxFnBnolugRx53dVa8EqS5tDThZEeeRKVu5\\\\\\\/s8ExVgYM6rn2cXoPlGlZUAgQYznnuavNJ9fr4e\\\\\\\/mGDoFvSJyEEl4r8Vqoh84ichOTt9HD1rHmqe7tfBKv2wDRpbyxnqXULRIzfAoe6bi4CrPeReraW4iRZMAHdD1hum4x22CQJ0KFEOnrts2xv0Xz8lUP2Os0HSqLHNU+G3\\\\\\\/4L3lnVIhQPii\\\\\\\/2i+ScAWnyBDZuxNe6Fsksv+cgkYX\\\\\\\/Ydsvj4FJCZulmZdiCFXzsMzr0mgegAk6Syry93XNoMzEczK25nsoskb+O2deHF6iHtMNw\\\\\\\/DaWqtFXdmHW3YS1WXdZa42mtbmEj\\\\\\\/TFoev5SYsBPLx8olkbNnaiIvYQ0o\\\\\\\/G6WJihg4rgOh8w8r7IcN9BZjLS1c60cqoAO\\\\\\\/i4MPQSslogoYUMeAuDhzNJDDBjQM7JxB7k1QRpSs\\\\\\\/fb3aH52c84Bcq7hf3mYcRs9aw3rHgKr9u4ABHvo\\\\\\\/3VeFh3Tf6DDrDwWL7zDo8CaQAAhyJqaRsHdpyZ94uFBxMhw6SWS5cX8nozbXHqUQkuJA4aRfmARhge5hth7S8KeRe\\\\\\\/i+py3X3qwd1AxjuKRkaZdzrxKn9ujAZHF9\\\\\\\/eND2cJqw8Kl2rX4HERhD7H6dtE2mk\\\\\\\/3ZkQ9yoXdJHIzXy9cFhGVgfVnwgyv2XW6Bnyur1GLO63EFZGhZoiXThKE87FazTgMMfRjFzWi\\\\\\\/oRo0CQzLToeJtOOGcNE4f1dYJUiSm+l9h29GhZ3pNPKZy2N\\\\\\\/k7xe+EciW\\\\\\\/aFXbvP54MQUb4KOvRCZpU0PEz41T0SS7bc8C8s7k2bUvKI1s0J9Xzi77BUzIC\\\\\\\/\\\\\\\/fapCIwk3p9TZvYy8SuWe0BBvgJ2h\\\\\\\/Rz8YrHr7M\\\\\\\/lMMw9Mc92BJ4uBKpbtmyjZjWv\\\\\\\/X2ctYj9L3DtrEQhAMA\\\\\\\/\\\\\\\/O6uzQOvn+8nqYDo\\\\\\\/Q+ofU7UnbHfL8a7sw4YLy0vgqcp55Fuv0ji+IBcaHlQNnHHVsbWXLOBToI2GdfDhJjB4lHTTSLCe3\\\\\\\/hEQ69IRVAcTmn1rakrgySOLMYzbwjmp4MSIKQ6Kzge4NSIq\\\\\\\/uzESSvPpZ9\\\\\\\/6k57PE07170T8kp1wtueCYdI3auvA0ekM\\\\\\\/dpWLnm4p1AOo\\\\\\\/UxBhIcZuqZGKfSTujvh1zVfY9V9DWKOzIiA9h3dRS1lE+JiqGXe+g7sMYeSAvLm5JSAGpGwuL979GRaTOm4WU1ak+OgibXoVOx\\\\\\\/FD4D\\\\\\\/snL3YD+qEzWq0usMkmwDxqpkNdAAj5DUE6JB9naX7v7R9zVDRdB3jqKmQZGyDfZFFoFvwbAgC3d29QjiLb0tqRm7\\\\\\\/r+YVbpluc3Agsx\\\\\\\/TL0A62iUOEU6gTiOc9RRPeH1+uI9wF3hrDVVDnqfba\\\\\\\/CNF5fqtMVY6yO3oPd5lNDHTNVjwgydapu35h6g\\\\\\\/6NeDnXbF7IEMDypwJ2uM3Q5ZZW3ri+fi8HvDe5p3144LGSNMc5LOcqHMQmoLoYAzoWn8vgsJaKoDyRPqSQl2AXksUsT1A1mu+xNpRGvbbtm46S0ffaLfXIbIOmJB6tQ8pNCepAI8G2opGE3l\\\\\\\/cf2Td1pxNHK24WUpH4RV\\\\\\\/NqA65czfmsNvZnCwNzZeHyYd1e01FBRq0GvqpW4q\\\\\\\/E6iXMKkSMW3oO1KeeprKSE4bL51FBzPIvSdcN62uqSZmBF7wHXIx0tGL0GXlQD+n9iTRlTNJP5uCX3hTseYqD4BMmG99Y4gOjKoyij2SBq\\\\\\\/iijvk+GQLPGQ\\\\\\\/gEzKtkkeCe6oMEVMUNu3MNH3pCbnrgRoVQKLUAyBWvSHmua9W9hU7FCEmTC5eLVcQgqBe3EZ7lkqoe\\\\\\\/gLyD6+\\\\\\\/YmBYhhKEt5BcdXhpnHtqPNxqR7cgwuNPSkGM\\\\\\\/O0OuDY7kP1Ake1ok7dVRM3fxgAkd3Uyz+jeoN7gbgBhy4BgscOXhvTGEbOAz3DzzUYOne58A64Flu\\\\\\\/qvIaHle5\\\\\\\/VT71uj\\\\\\\/JViYeKOaRUan3c43WL6o8LGVHjhfeLd6v50kVDyDYQdC+8DBpSNHOvpfYF4F3AMKhXH2PexWU1uimk80Dbk2EETw9HrEofsfbVLccfZn6L0X9lYyP7eLwxfy2jQrliKenwwAQDZ0vXdc7dJVdU6X1hXmHcWsY0AXlEdLjosIL8F3lp1WtDTwgmc2qVDM780OM1U8J0hCiGJ8ff7JxtwSuG4ygJiDZOcmbhiBkooPKsXcDs\\\\\\\/QX5Df9vJGtiGLeScT8oCjBNVPUvAihosCRGYocutEVLSqDXH7biBQGN4xBoJx+mlLa0nGlnAb52NaSMPrRFn485v1kKfAHR7tY\\\\\\\/WOnR19JMWcuXCh3\\\\\\\/bIpocackGKzDDOa3ejqpZZbljCLSXgDnWGF\\\\\\\/phI1aOPUM3rVF2mwW4qWyak42LAoAs6DOvjf4gCrxnTafjpVYgVCxEdjv8KVl6c6850N7lfV9QR\\\\\\\/eANb5AqOCwNUAMQlHRhuoYXv5reylLKsN84MmJuOQ56+BVhx5C\\\\\\\/+PTyE24r8H5VQnyL4RoXsVbRcY6DvVzCtr5uk2XJe5YPK9c9Zne7750Kb0FMLjejzH\\\\\\\/CNR67pYarWH9dNmSQAZrNnQvV40IZjrMubmNOp9Ok0BN0KWlF9r\\\\\\\/5N3qjazUSZe0R0k890VBHZSMmoUET67Vd6oQtC2RUnMhBSfn6zLz877\\\\\\\/TVkCo8VVMCzx6Clw8Sl8Tc4Vk4P06HIAEcnMrU\\\\\\\/mIhPn1M2n4sNo59jqxWuMsX1PyGe1As1bqmu+MQmtSu8vCS6P8wrK\\\\\\\/opm+rtNIyrDJwhhW8wYxavUedwUkqBY4PvH0JMEcHTs\\\\\\\/bqThVpHXZL9ZzOiN8PA+WLFQmqYjl4GmFyEjROHN4EGtDG5XbzQpWP0OwA0GRYh9IJnUTFSvSV7Oyv6Jpf5A3nRZHy3LBUVy\\\\\\\/XUhGD4O522jT4J1WopsZQhYJx9bP12CtFpmnYbaITKB9EUhmo2ee7heQHvWKyDCklLCfqPD+P0Xd4tfEDIzzWoZIld+rMF5tHX\\\\\\\/Is+dNF5RbaME4i\\\\\\\/XWiogneo0PQoFum5Lp81nm1VFXnL\\\\\\\/9Kj+cl3fdFKlH6IwqdcL8R7ikK673iIwWR+0g6ZpCejauDC7NgQyOfmI7ZtA09p2Va13e7bIq3wt2qZJuVneJR\\\\\\\/FsYWW8JmwDUIwKK3+He4ssONTzYUo+Pwltee5Qr7GCvfV4oCg7ji7KNUPBGJtAFDfF\\\\\\\/81DseMnk00Mdj5HYmru1PqqoSH7q3TLKCNSWfovRom0V7RSPd3JMFyMsgfUnIzf+miM0D7oWl\\\\\\\/n2wyhLP1OUYELBFl3HD251axFdCKS3PZlTOEtBl\\\\\\\/4+FjbpdXdJC3RzcNqCPUZVNqRZ4VbjfIyTxhJoOwMTTps4Cynfknz8Cd+D1BUvKyr7RsVm1jp6XpEgoyjsMjfZxxtMmHYWPnhokz0Saz6ggas0Itufv782uVu5NkJDKcdZyXRirELtOyQ3jjP96DZpURhMoBoy9K+NhNo0nwV5VNkOhRKryp6UqPYBrZPdUHNxIEDp0k+XIWezDcviwee3WiuUWRwKuFD6MaAh6NFaki\\\\\\\/SBy7aE4O12bNtGwhSasM8tBKh5rNuv+7dAV0HbBoxytTlzXf2vp2J8nLSc0k2laDD+bcIoRklKiMvGTGAdrjCmOsSPP+JU4lyeQUw0LqEhegnfjiKvOqVTmJK490XWezBUeksNXXsKQJB8ptKXHsBJvJPl8iLPqbjDMi9SsE7mtgpA0KPrVqamGsPuzxYj+Doxn03VtmmOltARaqczZ3XOBLAZtQyb6QZi3q6yBkqpKiAr0IYNcrLRytUY9lwjIICU9Ykv7dUeJJSFvn8rq5gFLItBMJzuATp\\\\\\\/3kdNMX0MXgSv0C5mBSNZzt7aW70mPrHYqNovZUHFAFms50p54kzkYpKfK+1\\\\\\\/hXd3E0NhJ6u3OBIy8\\\\\\\/5ngnD9HUsb79JYvNpdfd2bjaHK2L32FnGIbxPWuNM06go9zME1ftFWDxLckux2m4XI3ckJkG\\\\\\\/q37iwzZpTN7dktybhQyYGn03D8tCDefxqmUP+sZf8VZY10BfTwS0ccTwSiW4Q\\\\\\\/Yyj3GqopzBes\\\\\\\/JFMXoOM1pFiM++YCmILAB5fmVK+VjgMEsT13bNFX1mDj+SrzrilWqkzF870pdE355ds6Oj0D+HBdBsqC7MlVWVQVWkmaYyZbfqD4jxAIdjTrCnT76EIi+B28GKRCxfpkAIvHeCfm\\\\\\\/d9m\\\\\\\/GJKI619Mwn7OJ\\\\\\\/XUFlVmhNS3dj3WEzWxhTaCXS\\\\\\\/XGcLijnvyFDDHyQMLc5E\\\\\\\/eWsC+pa2mBTvcIqTNGn\\\\\\\/6fdUF+oQV0QajBwpArV8WFvhfD4YUuYdiS6Ln1amWeFbX99HJL1gy6dxkJ6g0nct+KT4D5mbG62mYBLUZo252uSc2WjWTp+dW5aH8w958Dxq4irUMnLvoA4A1PfCBEDA8A9I6Ip7nhmD6MmllXZog3CNRajlSHx5wdWHyrTl\\\\\\\/5710hsmVYio08E8bOS1wzxB7j9vEVDlbu7MYlPlnZgHmFBs26PtkLHecLGEWzVykcjC6wR326xhnQv4QM95013AyOeeUghi\\\\\\\/qTallR8Db2Q6\\\\\\\/VZ+2TTH2mEeQ5GIdMXil\\\\\\\/neYnCbIr7a1ZXpADgwgc9gar13AMvtxVTjJRH3QbK0Sl7vy89llMVLQzIkuB575OM3+SvdNSl\\\\\\\/Wn2Rg6k6iD9ZNFpB+UdHYOO9Yp0QIBDxNKzxUwEyYGXxbjRMCxWdDT\\\\\\\/UTqOxqCVqwZckm5mJkqrD2fpAmUUltgLNRH8QU\\\\\\\/djqFNfrSZolObTemcFWc\\\\\\\/RtSRgQGpWR636qx8eXz2kq2dfJXFxkzS64G7RlaqnWpjtoYA4HJtA\\\\\\\/4d5FgF0Pw2uG4CouU0tNjV9InhN1jCxkD\\\\\\\/IvEPzseilG8ATc+QW4K7mPHHIAnDTN3zIINxGeqnb+u8tkwKpBiOtyDs6F\\\\\\\/gLFn8ZGsmqP8\\\\\\\/IXfaO\\\\\\\/epur8C48HBN+SOk5R5ddap4KkiNerBXJchfRE4k+0ixRkTdtmVF9K7L\\\\\\\/55rfXuZ6BFHOc3T7CSwdGqtyBBOITzh9Dv0YwPaHTbMTd9jkMmoaGCfeJMVhpewsuplis3B0shP+ZjD9yOQD34X88\\\\\\\/oMo1zE8h7X84XpoiOcQq0LlHVc+tnRk65IAJIsjSSGb3y4\\\\\\\/on8NGnEX\\\\\\\/m1RJ+Kz9llEcNkRI9ZCUqkm0I0tZtqBoyVybz8uXn3+sucbc7P8JFvKIKxmGWr0L3\\\\\\\/4OHZdWWctWtzSgzSudpnnx7tK3tG8YcK6v1TYIOcedz5qg9wDCyzzHmzCjwsfI01UzBCwmyxL6oPHoQBuau82OUh0gwe7btVRqiy4IsZlHf3foZcj4BfDkpyKs3996t7etrXCi4P4Ap\\\\\\\/uSXIwVINnx0SGPlp1KYC2Yftiqkt64xN8\\\\\\\/5yxGp13NvjiTiT2OZPxCWE5bwhJcTLjXllLqfy01ES39yT8Iz8aEpnvbGfNosexTuBCaA5Oe9+We+MqMh\\\\\\\/rQaCd+LwhGAqvG\\\\\\\/gQW+lKNS0+5OwthpMfi9wBzN+7qXmPdz9ard8r5x2WmyMGJa7XBeFXGN+eeSYlrMi0lP2jvP2TsfbHZeaXePAgi1waHyyZyRq7PebS1VcRNikpMpQAvsq\\\\\\\/FZJ4vxM+WK5VYXoHGpnEnk8E6GgIWpi00OLJd8okZ6NWMbYeqLwykZDtYND5jOz8OEXePRezQvmzUhRYLZb+UwKeGP5NqMlG6MbqA3Ua+aI7f8M1ZO8Tt\\\\\\\/zHi5hX99uSsoimp1bDW7LrDZXTvzmyclgohIxrN48atagXW9HNHyaP8FfGL+OLodNlFfcIp2fyw+wzUFxiH4I9FuYH5+1Lh+XUHjtEj2SGa6XDNExoIZTyJoQkEb8uOljClRcc3lArk1wCDjSOXF\\\\\\\/GsiWQFW3PZTLWUaOwvgvhCcD9lyTqpznGSL81GKjeQOBD55DSjrQoewDB+1gbskuQ61ONa5x\\\\\\\/RAU9UHPOS54O1bFmhWorQaxFJrDCFkweYIaWGPU+ZMma+qGenhCDhMiXKHvChcna6uQ1Zhasp4WbxGSLDMYDqhn0tgwY402JD9xQsNlBxQr95YgvjSzOltUwjMDhqmj7iazIW0cjFWZFKavxlGTT96Prp+ZdTVcaERCN8VvXfnrtqwdLwHZmpg\\\\\\\/11o5iscMGC8yiJdxqJ7yRKD2bL8qNbpCHXFOcIJtqynb\\\\\\\/9E+wWw2BMygbN37RR4OX4WH6iZVgLmK+OIFxCgBZwagT+m0SwhVRm9qUtuT5h+iIUP3yiY\\\\\\\/WUzCYlGwYY69IAjUtqyEhJZol+DlEE7XF\\\\\\\/emW2BeGYvLAhXevY0v42Oimf+V3AN5B4vi80Jl4oSfbNcotvygl7JA1JvUVUZuPeFwTryIxN4b5BzK\\\\\\\/+ZypfIuZhZ1C2rLCFgO6mVK\\\\\\\/xRrl1D+i79C38OINVFkWn+IZDwMAJttrOiUsKWTSjFfUJnLFm6CVas+uu\\\\\\\/CKiNA1TXZwA0gla0Z9gJ7c3Y8XgofaLDxglK3cLTHy4Tv9n+PHhn6SYk2rQaH5ZzQzc7OKrqKY4KnexMHgRLClgqy4hHPRpJS3WhW0MVrjhhPoz+7F7l+a1oIk8kvOCZ6UhRwXggzs9MZAIooyTSZyoNsEBWawJPXNmvcfyBIldsgmcAAAgsJvmmVW62OYask22ukdCF\\\\\\\/DZPtz6kGEbqARh709X+PVhmnlgSVlbetoZD5zIikIaYIa8yP7FEfVD+xMXiAXOROsT+ilXYcXsVa19v\\\\\\\/HpIa6M9s38fWgYjOO5hb1OD5wyp7+rUraOWbLOkiN6Iry7RlhAQ1fG6UkItpO\\\\\\\/Qs47OT+cCm+TouAV+33yIx525bqZ0Z\\\\\\\/VP08hOPAgMg2zNM1Eo0nvtXFR7USlm32zPb6YnygoCnBIXa6u4NlGKU23naevBQ862iQsC96KC6LMmxuhzusYhg41CCDLo5HTkzcgLA57FDGQZutHG2TZtCwGgtsre\\\\\\\/xfFsssFFMra7FPzF7RCNSfkYqBMe+H\\\\\\\/1T\\\\\\\/WUSwJrsi5My\\\\\\\/oxjdQ7W7GCbD0MHO9VOK+XIDbXd7pMFgW3TWYMc4w0l+WgvCiYdTutbCaLR4rcr+yRmroZUfY8Gz7vzSY91h0n7pBsABUPNx7V13FSX2Q6dyVpN\\\\\\\/B6Mqs80kCdcHC2kJsaoHzvPbPJBRntGvTopIevfAop0I+xPnU2xnXx+uW03roHNT\\\\\\\/T8j9EFlzuMdi7OzZ5OSWLT3vGkdAresw84fKeusO8Wlw689Lid0UtnTkyPpLbKrEhj+ofoGTZttSniXhyHw6LbzcKeH1JrNvRXpVy8CuxRX9J303CiutCn7UqItapd7KecZtHQ12mQAOzwMdwoUDcVKtEnMWmBYQARO0WD\\\\\\\/ElI8qz6eRDS3TWtktmA5B74V5QNKnGxLizlYKoUI5lBr5fQSxgJn\\\\\\\/PpHvsWXG0MQSh54muJma4fZGYlmCqjMISyaaKYG5VDhOeCS19zE3g1j1yV5\\\\\\\/wRRErOzUXnBCmA6p9HsAfhM4tw5kVkKk0Tox\\\\\\\/v6SK1CFTkV2ymbh0Oo4T96Q8NYUu4XI5VAUP4Bmb\\\\\\\/jLegUQos6q71eYYFuLM4POELq+gfbO2+\\\\\\\/huQTH1oYrYT1WapQ76mSEgN8L45wtDpIiUhhyrGm3hRc00lz6qMCO4rVc+LRBI6s\\\\\\\/Te3hTBgs8JOljBSVKWaIV5aLBXwDuOJTrG2h8fdbEBo+pn2Y+J0cQbbT9vLqbE1TALgZqkv2m5OLCsC9TNl7trkWEW17mZ1OxgjG0g4QfwdFd7WcWj2YI3OBhmllSUW51um+tT6iKSHARkfLrOAKVHYuCEHiPUx25TYefqCVDVvIPqvPMV19rIMNt\\\\\\\/S4lqTFd68nOVrH9RMCcdjLL1Q0kYglBJvuL21ZxIMsIvz+L+TKR\\\\\\\/3k3QY7mgbZWAqDW5TTMuwCBjCQdm4SJssUqMPnzMTOtBEblplCQx5hcQkAHtWVt4Xz4pri+uklnbapWbIXKL0ThB3SlGGMeVcPk8MlezL\\\\\\\/1kDhamEdcUrcfk+6rZUuZB11dgx4MCbH5duSFRF2svibjRHkJYuk5BYNQ0AbiBOwmVX59SbeetUbQ420qwQrf1G559Quu6ahUS8\\\\\\\/YMTEfYb4R65oljZGW19A8BLxsRERmMJM1TV+rJd2cWzKAoWjUAEkpfAhC4KNrHv2mf5WbiqvSdarvcNATeuPLCHMeqHQVp292rlimKLo0BnZ0zxfbM\\\\\\\/vRQBZ2VcyCXP24uCafjgcNiuk7XhvwjwEgjDQ2V6uJXveU1kfzq9hV+2dp5x8YEvPKQZ7arNZwXkYPTXLKrm8ds9Q+rdtBW48cVySunsfC8nyvg8zACfnQAQThXqRtZoGtw+o+\\\\\\\/JUMkyFYPcrgSDJ8S+fX3n91zBfYNO+5jXeNxWcMCdflVHmB7+Ob1sg\\\\\\\/U7llSvlxaJTP5zMAbE93L6XfjAkcFIsYIvUnpDNLE74vjPRX8svnPcOSwzeYKqUSFlI12+FcWeDUIjsOPVL2nrWQH2bcseA6m5xGbSwdA\\\\\\\/8lyY1WcrNe\\\\\\\/BpTMOIQq53W8NFRKmzvmeerjedtnO6wsoaoV\\\\\\\/nt11SgVmDIj76nLWNOx0FW+5EcY1WVdQUQIdBr6+\\\\\\\/1yUC3U9kRcLmwMR+VU7mTIbBba21ELOMzcXrg21iPin0PExd+4iylFew4EFUpHcEHddfZYF4fI13jRRNzLJKrz6zCVFoCA9PyqFA3GPg7Ohx1HB+ceAHUBV44Y0XpsnjbiOSlugFqQHBMzWRGdrEmKQZse0FnElWM\\\\\\\/KpjIceFEtQ6DmEHcso5ctykTYIUyPyNIFPnZ2DHwTpjC7qfQnUU5sAr1kA1XGGL24yRI1dKA3k7n6qD62X+bky3GUpE7dvaANPTs1S67emetRMEfPDTD2I6jRntNBTH919UKGxMMeJhTSsa\\\\\\\/xi7cFqyj1fcsjpsAULCvj0TU\\\\\\\/qIWpvS6hth9DqahgAN5KsyesFfeYnEpqNJc2tPCBw6sWzB203xw6IaEf+nC3+Wm3UqXIE2w79oe\\\\\\\/4PvAa15eaigdENImz1JWZ0vwE4+y5AFLY2zwE0LoPknBK4mM8XWc7dTERpQ+mMrVb9CZrZZGXUnGwfEVhnREjnIIbho\\\\\\\/LPyQRUb1O8i+84MP3PLwodQx+3iecUwz4QdHRoU7tkAO3jc2Cwi0MW0AauJok1HhEi6MOjGKmnwtbOm4Of99X6dIdrdO7etcIUoq6EmKCJnTVJdCzTwP10e4kurZhUY9McbdABqKwlwZNb+xlPXq61rfGAQ260PZNYaXf0qkv5T8IqBhGGjV\\\\\\\/R12NxW47K5B9HCG0sac+I1IuDrn51a2kjcNA11gReEVB2bKpaEEjGAE1QV0ypCEPn+yQ8gN3lJYO0ZIOL0GTKaXlUoAZObXCEpdFmZcbsvWD8RSUHH52zh26cDUVke1J07i9eGx7YfYEkNcGkW8TtXTS3c+gpXhNUbX\\\\\\\/3p2\\\\\\\/W63dW1pxjqDgcX7qnjigeFHPFMKG0tGtDq5yajRSA\\\\\\\/1xfgaeDI7PDtdu+kLzzxxczVNWW3cKpEXSgPU7zL4FASS9VEXxdchA0pQKJmgp69MMjvewHw\\\\\\\/g+5ujUM6hVMvLL9Z4+RSicVvGWguCd9D\\\\\\\/RUTpGetv1eN1zH+faghwWFbsnGq+IVBwV9xs\\\\\\\/8+jjw07FIWsNWDXPiYe+9VtnMbGLb3qh4FGww\\\\\\\/GK1dYn7RbVl3lJCO48DYgn2vG01LJM49eVXUOMQdcfhkap6TXjiZaHIs\\\\\\\/aEq8Nv2Soglr1RBh0jYApMn\\\\\\\/XmXN7gzZi8rgLanshBsFobb0HOpw7aBqeVBAVqHV6iLD5Qy1LfAHlCdb51+REU71rYAtVsCEJ6S88IeOgd4mYrJb8cs3zjfUXE6D9YmJm8xOvdbn+iMZjCq9mJSFxp69b3ViKPjAp2vJCQCQyeY15s1b9ZMHlKl6Omeu7g8KP2uhMtY+qHPdXQ8\\\\\\\/FXBAd1uHlKLG\\\\\\\/XOBtl4Pb\\\\\\\/lRIR6xOxXkfTvYFiDZ5PB9dw5Mf61kBt6\\\\\\\/Jo2yQ1MTcBeLI8Tr0oOwhvrcm15pfslB7G7PPG+zfaa6rowUQADHqpqeMyN4hwh8u7aucuaqIefImL1PuJmAUPt0e0LyVDw\\\\\\\/GbylmEffM5fP8VRo3foa07g\\\\\\\/teKr4HZIaOuJwx9JyDMmbLTZcoJGemU6sBHHbwk5ICSSQbGokQH1QnxtFWelJEpZK8GSMqErxzr4PrX2Rs21Byo4UVHQwejCg9gxjCrU1OgAu9wIxgK\\\\\\\/g7OFGrQBNxUIn6l+\\\\\\\/Q8BNAFwyUSRjRfVb3IUTKOsiv1+e+VTdC+uefgX1bOcVaBBzsj4BfjI0CRPxk5UZvof+sDhvu+v\\\\\\\/T6HU0qmTkMlpuFOXkWErcihJniP1b+3fFQykUJBilJZ3KXchqVAe2nZCxGeSLdrK+qcZ1iVorAdsLdmM7LXK5g9wzz0V66aWw1TwJaTZb3HB\\\\\\\/T1gx5wC\\\\\\\/3dJbkSz58+OZs2IWhCX7mVRMT6AzZQXHrraKR24+6DZRLO6rcGpVRrCl\\\\\\\/MyYJl5CRFXuFL1\\\\\\\/t1X6K7PVHRkzYs7HVGfsvm+jFagxBr01cmi7zks2oY0\\\\\\\/2QcCxjPS9OC8FCuPhz9f2fWOlEVysUc44NYgpvTDCvIWXwLwXZaMdRglWI8d2HDObJfXWGYPlgtyyL26MKOvD8GdDLbO\\\\\\\/bcC7Q4SKC6R2XP+agvFlF7iVfMmIsBDt8iXY0OYxTe3Bjw6gHG9mJZ8+uB9UjPSkA7fkl8\\\\\\\/KH\\\\\\\/bUUG4HNwAjH1ugrqZvQWzXkQV0vSWr25UR1vpahDcVLLLLgZ2KYmGI2bz8YjxXDgYJKZpN9E33GkCUNAVBpZ\\\\\\\/IO2s0kKwEn6slUHlcjZsKQi4qJYOREdVxIbARVQT2K7Ps+w5PV1cJjET8uLB8zmZgvy3FO60joA4R8IOIWNfLumfi+gz+XLvgc5gU1+VOAkrIFDdL0R8CsI+xpGE+9UqxepAbhrSPbApb3SVgsgIHe2bflW5H3+gUaaioJUm8YszTrTrHtZQpAVuVuf+\\\\\\\/zabY8yBSbyqGreN+LGSoiD1KFj6M\\\\\\\/ca8\\\\\\\/1GXT3+rExZpYoTbJ32200cEx6KOwDAa\\\\\\\/tF5HHUEuFr1VaKBKbcTG851EnOcKbqAQrsTKNZCa3OcLnb7spBo0bhy9wggE1DWv5Et4EI9aFoCiS8ajOduUUJHgDQbl0WRGdKrwziqqiNDWFCX1lua2h0e8zjANX4G\\\\\\\/1j+Q\\\\\\\/TbI9S\\\\\\\/VdI9zMvB8E01VW2\\\\\\\/+vRmoV7aUqjyQk6HXDuN3xRalj6uaqdyWVVQCfIa0jvRynHNrGRlfOXUVVrt5NvikJbx91LTWKf17ktjNCrXgT+XaRfyFGvB11nScVOq8pocZlPkuU8ZpkzvgE44BSFwqv8SmRw7r+yQcq74LZPo9QiIqzP7pG6sbCabt8IxKchTFJKoJAd0Nt\\\\\\\/DS77udme3EAPf0XqcyWEwAZCxOGVrJDYtVRwjqFqhktqpZxbHIuqShrNoA\\\\\\\/ExJT83DRPvcFxKWi2q4ey4lyEKUpQL\\\\\\\/YmZFMWw7DPOHZgkwoescFuVxJ1DArZLuCenghMBU3ek1ipVkENWbJDdQa0vjxilK0vQsw1M3ah0D1oMQTup7oTHT+BRlsgtxbp9cYacgyJFvWCTmOeh6cJO6LhewoiPnicUjpTM6caLoN65+K\\\\\\\/Dgjpsh65b\\\\\\\/37NLf2STYgl7J9tWGraXFZABZ9SiHdUq4mOFmD3IzovRjMl9vZFGd4pnb2LKc7AiVXH+kPTh29lwqfFwaKUWK\\\\\\\/+WC4SoVoe2EfstDa5ub7Su8JUvsgN+6krA8Cw5YoyQ6aTYw+FWbWweIDaEO85zR+ds1pU+KoGBOxgErkEO+QmY1O59nRqtJWWK1E00qqCZAayd5naqjTN7LxRrYF7JSyOk+9s4EFNAwcyfrY+Kza031voe4EfEOvYb50t37s+bADqPu9NO6Z8L0OQoUEGpLVI\\\\\\\/hXzEkACTxFHfO3W9HsetSK0keYZ8cTK9ZpqRWVmCoVXgOh+kSSV5HPmAFFvd8GyBgjiFgAZa+zJmzvT5E044mmbnYYUX\\\\\\\/dK4Xgub0nPyCfzTdqr6V85Ewob2AQ9MjyUpmfcxXuCSPLHMPz3b1iylTXW8RYHVrsejij4c5LY5H\\\\\\\/OIE09\\\\\\\/11quPRVCBDJc9nhrpJINHGq2aDDnHvgHdBQqrIhQT0THfyJjRBoI0p3uN7cQyE\\\\\\\/bZXNemISTPlm7Vn1q1bhMlI385unFlfAKyjTCOfhZfqac61vlGquCdsDB0j3xHXjrfZZZvX3egtwvFZ5stVZWkcA6fD4MZu5UH7WQx1pCFzVv2szDfy3rEUqWUn6BUZwLxISg8B1rMTWldN3\\\\\\\/JsfKOe6lIINnKpy9DmSViM4ma5UZa4XEAcf2+PYc5TJomLUCmCrAYkjF+TgKtwcNR9wdwudmiRIaGrLrLHi9FvKVF6Z7Gs+lCKgFcKcIL6vGEr25MZMzVa64RE8ZLkHavtMTWEfaEyJpkB\\\\\\\/APsMRnR09IhBBkXpHJHNttKzl5BvHQq7UEO9w6ei6TCtpqWK+ZSvLLqLHm9lK9J1MLsiPPvU+wpyiHZeLwW6It6ZNNUPxj82NPq5GdC1RqSL7WqpxKTcl2VmfsnzFz9cFQiAA2W1NjwcEuDal4X3s+ud4LphuCFYboIAj4YnPEsdG2IUtJKMbvzJU0Hk9DPaFtkkOtJZf4asd35V4RajcVdQk352eFPLSEpiOqQlvW5IAjAIIO+wkiFShog3s97NSY1D6h\\\\\\\/Sd44sV7QEFbvyqfsQ3SQcKJCF21ARj8FgR8qoYaq\\\\\\\/DwVnI\\\\\\\/9DGftLoSSW26rAnvfJ2DNgAnv34+KIlYBx7CUV898Nodh9U1IxcRbU6DIXLHBDRqt8xg+JQ4P1MQodUUHfyTetOB8E50JkjA7XyF5zYXcc4Jb2GbCa7fUpAoDkjIgLc909viO3PNqTlWQf4GCd\\\\\\\/fbFi5W145VqtcV4YIEdLD8uxxZZZsotRi\\\\\\\/eoqzEQFiM7LAseZlCrr2hhQCV\\\\\\\/H6xXTvOkI7rvx2ZBHJlvOWsNbaOiOdVMDBug8OzpyhkVPKkU1ZrYj8cSx9vfJG\\\\\\\/xhpJjHxC+O4un+7P3tsOE8I+AqCkZJAKmL18hCNEDalRlAWdjbbW3MA0t9WCphMopUd+fV1\\\\\\\/MSLRTMotxSuAIREKVvOYFxoh3FmtfKO02rk9Sphc7FO3klTOF37ahRvgG0GwLNFhP5tR9P+PD9BqMmZgtsNQ5Jpr1siwXCqkdF3s5itKnRHh2ucwn71Ye7MKVdRRBZ4bT0Hj4SRoIy+0FqP465uFNnX6jeRKIbRwMumk0DyxRoOZ4cHxPU5eKbJ0bwl51fmPzuhuCjVsfb6gA5t5KkZEhvih0L5VHlKhiFZXT9yqlab2HM\\\\\\\/mlDr53h5Si04Y1oJxAWWh\\\\\\\/Yjb476kHsr1zA3cm24DVpvWzGQ3mAAnV7UGfShgLhHEvt8rnMG3qTvVah4qzbsL6KQwsIfFw4f+qg3KL22rXZgZybguGxbppZx1K8F7KXslVCKGxB6FNaxwDz8wLs7Ll\\\\\\\/iCNISXCis4+gemSDFKRWrc7cYKqrRlEvg7EgmSL4LwQI3IBYkD15KdHAanJg543DyHclTMfyc6fp4sPjbRHoJMNM8\\\\\\\/Sqib70OpNWnF0WcIiWvyOctgOMWWb7AK8V4X2M5OPTYjDGGMNVf\\\\\\\/savm8vHTCqbq3Q8nWKelk346qyBKnB0H5HegQwt1DgHsR4SaG0ovGrOTdQJNwienZ5VDIH3iPo14xpx2tOkaPHroW0xGF1fHMOadv3g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YM3wnARBRxi8RD90qebsUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e558403da596b558454fc2922e6dbe72d6033fbb81ccfc98222bde09c5ae0af8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LRnSXwqe7m8yKP5lq3vCpMUndjvTCCea9VCCXuvda1bvH54Vux3gUuB+THzTCNYgq8ZgCo00rS\\\\\\\/UwcCkW0R1kyE8HnOmuTUpRQm8EahoQIV7GMkfZKGbZw4ZFR55iFbWPhSOAKRmTEFGpv+bzqLtP7BhrwGQYIFOgiMtmbxJGvkfhl1+eiTUFPWRDvyuFHbI5RD8ZlPf7gSzpZ6c2TRXYhNSjLorukwLEV1xpKDF0JfDFTe\\\\\\\/vqVxShHU5oRhh1X7RxLy+jjboU4jNNWOKZr9jWvStl4CS1dMaIftkRGAf0ozM\\\\\\\/SIYc2r\\\\\\\/Qr6\\\\\\\/rU\\\\\\\/dbBoBTQLQmLqGH2tXpJ3gkFgKPFhjtOMqNzCtk9R1HIPf9+IlCCabSvWuD6RGiebtCPPMxhLc86BU1eyCuFbYvJqHejX9hnx3sBBS21KMjq0h+1dTESsqgx0SmgllI7HvjSN4E9J+bfMYspm3kbj7WCGbgmNDDi4IM41USe8J9jD+WkULw8Qq89YFl5IuzdMOoDJ154UYyrK1yj38aMOH9GqltkPlXxFDLeLAKtWPPs1de9J70y6Jlu2IeSsbXan2yoh0V9HSQ\\\\\\\/QjzccqMiWX29aKciZ+20DMOliRme+U5FyVkO1+fYtwB\\\\\\\/qTB0gRhEkohwlOaTGOUyID1TvP8YQF0uVhHkoFVngZCq\\\\\\\/XySFc2SF3s2WfI1EvODJY0zHgQeTD4adjJO2\\\\\\\/P0bHdcUQbv7VYBsQQGYY9XLqqL4RPRioPRqk0yMbZAL0H1km4uwOYqj+dzB7\\\\\\\/OWhEqCYZAZsIwwRhnl4WCONLECaVm1zvG5yfd0jniscdvym+xBDJLnhT\\\\\\\/cXJIlqagYiAn4dfmwd3q4nz5beNnSfX38zjx2QrWYdM008K\\\\\\\/rYKcrIrMNslySvhwi94P3Z\\\\\\\/9EBN\\\\\\\/BEavuiXZ99ZhmoOnbDNHdnB3ZfP1\\\\\\\/tS7i7WAqT5b3IYRt6ZxgevyVBWsVo30ZNv\\\\\\\/bEk4iL\\\\\\\/Tt4zbR4TSWJW+28zZSr9cVJ1a92ZkOCj7D2JKVWgRzHikwwQeHeLbItIg1F8sllb+braajkb0HRDvUZmdQpK51YWkO+7Lfd\\\\\\\/cMnDvhJuG+pba43OxNyofGHdZItQmmAVmr8oTB9iEqCaS6i+80tSHHhEAw6ES5smc9ym9C5u\\\\\\\/SwsLuntOmhXaG+vMSeVm42MBt7t7ovuXeOW7WRBGOMpX1KmtS0zNg2wK6mugfMnG8JsZZbwymV1rSlMfq2quoaAeGEq0lVHKGrqML5Mc2zfhsgKzDomOFAE5jNC\\\\\\\/lH8\\\\\\\/NMoHkpcsLBZzQJRVfvPbM7Fv3odRs\\\\\\\/azqJ8Htz+QGKr+wec5uu8sYAVTeqbE66PCFIX8LqOggKNqsCXEtvB\\\\\\\/IXsQJLNGImF81MRIHefmf5ukadwgOVPnuwJ3LA+ZFqOsRRXkkT0+WHBm4zC0PMDqr0AwymOEN\\\\\\\/4VftTaKID1BmcUN\\\\\\\/kNSlT760qp\\\\\\\/SXc9Y\\\\\\\/hgQttdgfS2+CxixPkjX7N4wo7L3TnhzGzzvdv1B6djNY55yF0cEPwDRZLkf\\\\\\\/zVFi1RaNdRKo28+c1DKLDyYUedCtyuyV+R3r+JhE+bUq0TmVbVT4YGfGF4ZqccrgzueJJAqLmSiIj21oGq0ExkN07GcRfWnYWzMoC4BgrSoX+mL7IJD+N3AED5a76H3TA2lzooFr18Z8v1386XB\\\\\\\/7tW9ZHmBq9Rjm8sAT+7bGS8Vz9yVRMTgkbpm\\\\\\\/CEyQVF7qkYMvwVYtpLOKKRbbSB4Zj8HsFtFghLLI+pclc53hTEvEMAnzVPskddqwRnWqPFEpRwxF6ckUv5HBm5icPdgq8ntJ\\\\\\\/YK+p1p1HvqxSjFFze2z0O0Tlo6n1SQEW5zuqAILEzjZXzabsY2V40XsngiEx8k2aAihUd\\\\\\\/wnQh0sDQytC9zns\\\\\\\/YeO7HML3ocHViL4yc8HTsPZcu8GPs9oJGtY2hEfGGfzhMQjem2ylZXi7O9xPKm5v4AEQvties\\\\\\\/wRIYKK7Emoq8mUmzulEKi73OcDQOPXzcBU27G\\\\\\\/XoYg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zjQxSBF6INMv8ZrM4nCudg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002282fe62b742fae0b0ce9aa9c37b6ead9ac0057cd43a27f51f9919b58562f8647d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459204 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/7/523" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Rlwf_CHnKGe9CoSvWu8jWZ51MLSsHeP8dftOKSu1jcg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/7/523" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HztPJKppPR5vyY8z_lgjmzc5aYOX-xZ_D5cxKvJ7aSw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/515" title="Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/7/531" title="Transposon-Mediated Transgenesis in the Short-Lived African Killifish Nothobranchius furzeri, a Vertebrate Model for Aging" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459204" id="node459204" data-pisa="ggg;1/7/523" data-pisa-master="ggg;g3.111.000885" data-apath="/ggg/1/7/523.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used <em>Candida albicans</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Darren Abbey</span>, <span class="highwire-citation-author" data-delta="1">Meleah Hickman</span>, <span class="highwire-citation-author" data-delta="2">David Gresham</span> and <span class="highwire-citation-author" data-delta="3">Judith Berman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">523-530;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000885 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459204" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Darren Abbey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BAbbey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abbey%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BAbbey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Meleah Hickman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meleah%2BHickman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickman%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMeleah%2BHickman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology, University of Minnesota, Minneapolis, Minnesota 55455, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Judith Berman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomics and Systems Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJudith%2BBerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jberman@umn.edu" class="" data-icon-position="" data-hide-link-title="0">jberman@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/7/523" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459204" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/7/523.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459204" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/7/523.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459204" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/7/523.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459204" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/7/523.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459204" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459204/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67290 first last"><a href="/content/2/11/1473" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - November 01, 2012</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Phenotypic diversity can arise rapidly through loss of heterozygosity (LOH) or by the acquisition of copy number variations (CNV) spanning whole chromosomes or shorter contiguous chromosome segments. In <em>Candida albicans,</em> a heterozygous diploid yeast pathogen with no known meiotic cycle, homozygosis and aneuploidy alter clinical characteristics, including drug resistance. Here, we developed a high-resolution microarray that simultaneously detects ∼39,000 single nucleotide polymorphism (SNP) alleles and ∼20,000 copy number variation loci across the <em>C. albicans</em> genome. An important feature of the array analysis is a computational pipeline that determines SNP allele ratios based upon chromosome copy number. Using the array and analysis tools, we constructed a haplotype map (hapmap) of strain SC5314 to assign SNP alleles to specific homologs, and we used it to follow the acquisition of loss of heterozygosity (LOH) and copy number changes in a series of derived laboratory strains. This high-resolution SNP/CGH microarray and the associated hapmap facilitated the phasing of alleles in lab strains and revealed detrimental genome changes that arose frequently during molecular manipulations of laboratory strains. Furthermore, it provided a useful tool for rapid, high-resolution, and cost-effective characterization of changes in allele diversity as well as changes in chromosome copy number in new <em>C. albicans</em> isolates.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/comparative-genome-hybridization" class="hw-term hw-article-keyword hw-article-keyword-comparative-genome-hybridization" rel="nofollow">comparative genome hybridization</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphisms" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphisms" rel="nofollow">single nucleotide polymorphisms</a></li><li class="kwd"><a href="/keyword/loss-heterozygosity" class="hw-term hw-article-keyword hw-article-keyword-loss-of-heterozygosity" rel="nofollow">loss of heterozygosity</a></li><li class="kwd"><a href="/keyword/aneuploidy" class="hw-term hw-article-keyword hw-article-keyword-aneuploidy" rel="nofollow">aneuploidy</a></li><li class="kwd"><a href="/keyword/genome-profiling" class="hw-term hw-article-keyword hw-article-keyword-genome-profiling" rel="nofollow">genome profiling</a></li><li class="kwd"><a href="/keyword/haplotype-mapping" class="hw-term hw-article-keyword hw-article-keyword-haplotype-mapping" rel="nofollow">haplotype mapping</a></li></ul><p id="p-4">Genome changes, such as loss of heterozygosity (LOH) or copy number variations (CNV), result in rapid and dramatic large-effect changes in phenotypic diversity. To understand relationships between genotype and phenotype, one must determine the types of changes that arose within clonal subpopulations. This includes measuring gene copy number by array comparative genome hybridization (aCGH) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hughes <em>et al.</em> 2000</a>) and determining allele frequencies by using single nucleotide polymorphism–restriction fragment length polymorphism (SNP-RFLP) analysis (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Forche <em>et al.</em> 2009b</a>) or using hybridization to microarrays that can distinguish SNP alleles by their differential hybridization properties (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Forche <em>et al.</em> 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gresham <em>et al.</em> 2010</a>).</p><p id="p-5">We use the heterozygous diploid yeast, <em>C. albicans,</em> as a model system to study genome dynamics. <em>C. albicans</em> is a harmless commensal of the gastrointestinal, genitourinary tract and skin; in hosts with immune deficiencies, it can become a mucosal and/or systemic pathogen. Resistance to fluconazole, the most commonly used antifungal drug, can appear rapidly (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Marr <em>et al.</em> 1998</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Marr <em>et al.</em> 1997</a>) and poses a serious challenge to the successful treatment of candidal infections (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Arendrup <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brion <em>et al.</em> 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ruping <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sun and Singh 2010</a>). Point mutations, homozygosis of specific resistant alleles, and aneuploidies can give rise to fluconazole resistance [reviewed in <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Niimi <em>et al.</em> (2010)</a>].</p><p id="p-6">Homozygosis of alleles through LOH occurs much more frequently (∼10<sup>−6</sup> generations) than point mutations (∼10<sup>−9</sup> generations) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lang and Murray 2008</a>). LOH events arise through recombination, repair, and/or chromosome segregation mechanisms (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andersen <em>et al.</em> 2008</a>). In <em>C. albicans</em>, rates of LOH increase when cells are exposed to the stress of growth in the host (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Forche <em>et al.</em> 2009a</a>) or to physiologically relevant stresses <em>in vitro</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Forche <em>et al.</em> 2011</a>). Despite the fact that LOH causes a decrease in allelic diversity at the individual cell level, it facilitates adaptation by revealing recessive alleles that might provide a selective advantage under stress conditions. For example, homozygosis of hyperactive alleles of several genes (<em>e.g.</em> <em>ERG11</em>, <em>TAC1</em>, and <em>MRR1)</em> significantly enhances the ability of <em>C. albicans</em> to grow in the presence of fluconazole (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Coste <em>et al.</em> 2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Morschhauser <em>et al.</em> 2007</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">White <em>et al.</em> 2002</a>). Notably, adaptive gene combinations are predicted to form more frequently when recombination (and thus LOH) rates are higher under stress and lower in the absence of stress (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hadany and Beker 2003</a>).</p><p id="p-7"><em>C. albicans</em> laboratory strains are primarily derived from clinical isolate SC5314 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gillum <em>et al.</em> 1984</a>). As the first <em>C. albicans</em> isolate with a complete genome sequence (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2004</a>), it is the reference strain used to assemble genome sequences of other isolates (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Butler <em>et al.</em> 2009</a>). It includes, on average, one SNP every 270 bp distributed through ∼81% of the genome. Approximately 19% of the genome, primarily long telomere-proximal domains, is composed of long tracts of homozygosity (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Butler <em>et al.</em> 2009</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2004</a>).</p><p id="p-8">Strain construction in <em>C. albicans</em> often incurs unintended alterations in chromosome copy number. A major breakthrough in <em>C. albicans</em> genetics was the construction of strain CAI-4, which lacks both copies of the counter-selectable marker <em>URA3.</em> However, most CAI-4 isolates carry at least one aneuploid chromosome (Chr1 and/or Chr2 trisomy) (Chen <em>et al.</em> 2004; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Selmecki <em>et al.</em> 2005</a>). The RM series of strains (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alonso-Monge <em>et al.</em> 2003</a>), in which both alleles of <em>HIS1</em> were sequentially deleted, underwent a deletion of the terminal ∼40 kb of Chr5R distal to the <em>HIS1</em> locus (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Selmecki <em>et al.</em> 2005</a>). Strains that no longer carry these aneuploidies were derived from RM1000 #2, a euploid strain that is homozygous for the distal ∼40 kb of Chr5R (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Selmecki <em>et al.</em> 2005</a>).</p><p id="p-9">Molecular genetics manipulations, including transformation with DNA and strong counterselection for the excision of markers (<em>e.g.</em> <em>URA3</em>), changes levels of aneuploidy in laboratory strains (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bouchonville <em>et al.</em> 2009</a>). Furthermore, in <em>C. albicans</em>, aneuploidy is found in clinical strains: 50% of fluconazole-resistant strains carry at least one aneuploid chromosome. Furthermore, aneuploidy can provide a strong selective advantage with clinical consequences: isochromosome (5L) arose in several different clinical and <em>in vitro</em> evolution isolates and confers drug resistance through increased numbers of two genes on Chr5L (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Selmecki <em>et al.</em> 2006</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2008</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2009</a>). Thus, aneuploidy appears to be a rapid and efficient mechanism used by <em>C. albicans</em> to generate a specific selective advantage in the face of strong selective pressure.</p><p id="p-10">To follow large-effect genome changes such as LOH and aneuploidy, it is critically important to efficiently detect genomic changes at high resolution. SNP-RFLP and PCR-based methods follow LOH using a small number of SNP markers on each chromosome arm (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Arbour <em>et al.</em> 2009</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Forche <em>et al.</em> 2011</a>, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">2009b</a>). In previous work, we established a hapmap with 150-170 markers linked specific alleles to chromosomal homologs and identified some LOH events in strains that underwent the parasexual cycle or that were passaged through a mammalian host (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Forche <em>et al.</em> 2008</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">2009a</a>).</p><p id="p-11">Here, we constructed a microarray that combines ∼39,000 SNP alleles (2 SNP alleles/locus) plus ∼20,000 CGH (non-SNP) loci to simultaneously assess LOH and CNV events in <em>C. albicans</em> using commercially available microarrays. Use of this SNP/CGH array, together with strains carrying homozygous or trisomic heterozygous chromosomes, allowed us to assemble a hapmap that distinguishes alleles on each homolog of all eight <em>C. albicans</em> chromosomes. Using the SNP/CGH array, we first determined copy number as a function of chromosome position and then used that information to analyze allele ratios across the genome in commonly used <em>C. albicans</em> laboratory strains. Importantly, we identified LOH events that accumulated sequentially within this series of strains. It appears that, like aneuploidy, increasing degrees of homozygosity incur small, but detectable, fitness costs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and growth media</h3><p id="p-12"><em>C. albicans</em> strains used in this study are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/000885SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/TableS1.xlsx">Table S1</a>. We used strains previously shown to be homozygous for one or more chromosomes and/or carrying trisomies (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Forche <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Legrand <em>et al.</em> 2008</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/FigureS1.pdf">Figure S1</a>, A–H). All strains were grown in rich medium (YPAD) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Guthrie and Fink 1991</a>) at 30° with shaking.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/523/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homolog genotypes of strains used for hapmap construction. Strains derived from a parasexual cross (Forche et al. 2008) contained homozygous (&#x201C;aa&#x201D; or &#x201C;bb&#x201D;) homologs for most or all of ChrR and Chr2&#x2013;Chr7 and heterozygous trisomies (&#x201C;aab&#x201D; or &#x201C;abb&#x201D;) as indicated. Chromosomes indicated with &#x201C;aab&#x201C; or &#x201C;abb&#x201C; were trisomic; chromosomes designated &#x201C;ab&#x201D; were heterozygous along the entire length of the chromosome. Chromosomes indicated in color were informative for hapmap construction. Chr2 in strain P1 (**) was homozygous for each homolog along different sections of the chromosome. Color schemes for allelic fractions are consistent in Figures 1&#x2013;4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-117737049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Homolog genotypes of strains used for hapmap construction. Strains derived from a parasexual cross (Forche &lt;em&gt;et al.&lt;/em&gt; 2008) contained homozygous (&#x201C;aa&#x201D; or &#x201C;bb&#x201D;) homologs for most or all of ChrR and Chr2&#x2013;Chr7 and heterozygous trisomies (&#x201C;aab&#x201D; or &#x201C;abb&#x201D;) as indicated. Chromosomes indicated with &#x201C;aab&#x201C; or &#x201C;abb&#x201C; were trisomic; chromosomes designated &#x201C;ab&#x201D; were heterozygous along the entire length of the chromosome. Chromosomes indicated in color were informative for hapmap construction. Chr2 in strain P1 (**) was homozygous for each homolog along different sections of the chromosome. Color schemes for allelic fractions are consistent in Figures 1&#x2013;4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/523/F1.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/523/F1.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/523/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/523/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461042" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-13" class="first-child">Homolog genotypes of strains used for hapmap construction. Strains derived from a parasexual cross (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Forche <em>et al.</em> 2008</a>) contained homozygous (“aa” or “bb”) homologs for most or all of ChrR and Chr2–Chr7 and heterozygous trisomies (“aab” or “abb”) as indicated. Chromosomes indicated with “aab“ or “abb“ were trisomic; chromosomes designated “ab” were heterozygous along the entire length of the chromosome. Chromosomes indicated in color were informative for hapmap construction. Chr2 in strain P1 (**) was homozygous for each homolog along different sections of the chromosome. Color schemes for allelic fractions are consistent in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figures 1</a>–<a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>SNP/CGH microarray design</h3><p id="p-14">All probes were designed to be 19 to 33 nt long with melting points (Tm) near 55° (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gresham <em>et al.</em> 2010</a>). Initially, probes were designed for 43,658 SNP loci and 28,563 non-SNP loci (including regions that do not have high frequencies of SNPs). SC5314 is homozygous for much of ChrRR, Chr3R, and Chr7L as well as for distal portions of Chr1R, Chr2R, Chr3L, Chr5L, and Chr7R (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Butler <em>et al.</em> 2009</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2004</a>); thus we did not include SNP probes from these segments. Probes were eliminated if they had more than one significant alignment (e-value &lt; 0.001) with the reference genome using BLAST analysis (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1997</a>). The acceptable Tm range was reduced to optimize the number of probes to the Agilent 8×60K microarray format (41,616 SNP allele probes (2 probes per SNP locus, representing the 2 SNP alleles) plus 20,363 non-SNP (CGH) probes (1 per locus)). Quality control of the final array design resulted in a usable SNP allele probe count of 39,222. A total of 19,016 SNP loci (97% of usable SNP loci, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/TableS2.xls">Table S2</a>) were informative and were mapped to one of the two homologs. A total of 595 SNP loci (3% of the usable SNP loci, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/TableS3.xls">Table S3</a>) were not informative and remained unresolved upon analysis of 24 additional strains (data not shown). These loci likely represent sequences found more than once in the genome or short regions of sequence that differed from the published genome sequence.</p><p id="p-15">Analysis of the final microarray data was performed for the two types of probes separately. First, the CGH probe ratios (experimental relative to diploid reference SC5314) were mapped across the genome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), providing copy number data. Segmentation of CGH data into regions with a common copy number was done manually or using ChARM (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Myers <em>et al.</em> 2004</a>). Distribution of ratio data within each region approximated a normal curve, with the position of the peak indicating copy number. Ratios of ∼1.5 (3:2) or ∼0.5 (1:2) were considered trisomic (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Chr5 and Chr6) or monosomic, respectively.</p><p id="p-16">Second, SNP allele data for experimental strains were normalized to data for the heterozygous diploid reference strain SC5314. Comparison of the hybridization of SNP alleles was used to determine the allelic fractions for each SNP locus. For a disomic heterozygous locus, the theoretical allelic fraction is 0.5 (1:1 ratio), and for a homozygous locus, the theoretical allelic fraction is zero or 1 (0:2 or 2:0 ratio, respectively) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Forche <em>et al.</em> 2005</a>). In practice, the disomic homozygous SNP ratios diverged from theoretical values by a small degree (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2B</a>). Allelic fractions were called heterozygous (“ab”) or homozygous (“aa” or “bb”) if they were within one standard deviation of the relevant peak (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, A <em>vs.</em> B). These calls were then mapped as a function of genome position and used to determine the fill color behind the CGH figures. Homozygous regions were displayed as magenta (“aa”) or cyan (“bb”) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, Chr2, Chr3, Chr4, and Chr7). Disomic heterozygous regions were displayed as gray (“ab”). For heterozygous chromosomes that were determined to be trisomic by CGH analysis, the theoretical allelic fractions are 0.33 or 0.67 (1:2 or 2:1 ratios, respectively). In practice, the allelic fractions diverged from theoretical values by a small degree (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) but were found at the expected 0.33 and 0.67 between the homozygous SNP ratios. Allelic fractions were called as heterozygous (“aab” or “abb”) if they were within one standard deviation of the relevant peak (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). These calls were then mapped as a function of genome position and used to determine the fill color behind the CGH figures (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, Chr5 and Chr6). Trisomic heterozygous regions were displayed as purple (“aab”) or blue (“abb”). Chromosome segments with no known SNPs and/or no SNP probe data remain uncolored.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/523/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assignment of SNPs from allele hybridization ratios. Allelic fractions were calculated from SNP array hybridization data for each chromosome in each strain used to construct the hapmap. In each case, unassigned alleles (left panels with only gray data) were plotted with the X-axis as the allelic fraction [b/(a+b)] and the Y-axis as the number of SNP probes, scaled to maximum height of the peaks. (A) For unassigned alleles on heterozygous chromosomes, a = b and b/(a+b) = 0.5 (dark black vertical line); probes that fall between the boundaries (right panel, green lines, values determined as described in Material and Methods) are considered heterozygous (gray); probes outside these loci are considered homozygous. (B) For unassigned alleles on homozygous chromosomes, the allelic fractions were close to either 0 or 1 (left panel). For alleles on chromosomes previously designated &#x201C;aa,&#x201D; SNP pairs with fractions that fell in the &#x201C;bb&#x201D; peak were reassigned to &#x201C;aa&#x201D; (magenta, middle panel), and vice versa for &#x201C;bb&#x201D; alleles (cyan, right panel) using boundaries that were 50% of the distance between the two peaks in the unassigned data and the heterozygous peak location (green lines). (C) For unassigned alleles on trisomic heterozygous chromosomes, the allelic fractions were close to &#x223C;0.33 or &#x223C;0.67 (&#x201C;aab&#x201D; or &#x201C;abb,&#x201D; respectively, left panel). SNP pairs with fractions that fell in the &#x201C;abb&#x201D; peak were reassigned to &#x201C;aab&#x201D; (purple, middle panel), and vice-versa for &#x201C;aab&#x201D; alleles (blue, right panel). The boundary between the two regions was set at 50% of the distance between the two peaks (red lines). SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-117737049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Assignment of SNPs from allele hybridization ratios. Allelic fractions were calculated from SNP array hybridization data for each chromosome in each strain used to construct the hapmap. In each case, unassigned alleles (left panels with only gray data) were plotted with the X-axis as the allelic fraction [b/(a+b)] and the Y-axis as the number of SNP probes, scaled to maximum height of the peaks. (A) For unassigned alleles on heterozygous chromosomes, a = b and b/(a+b) = 0.5 (dark black vertical line); probes that fall between the boundaries (right panel, green lines, values determined as described in &lt;em&gt;Material and Methods&lt;/em&gt;) are considered heterozygous (gray); probes outside these loci are considered homozygous. (B) For unassigned alleles on homozygous chromosomes, the allelic fractions were close to either 0 or 1 (left panel). For alleles on chromosomes previously designated &#x201C;aa,&#x201D; SNP pairs with fractions that fell in the &#x201C;bb&#x201D; peak were reassigned to &#x201C;aa&#x201D; (magenta, middle panel), and vice versa for &#x201C;bb&#x201D; alleles (cyan, right panel) using boundaries that were 50% of the distance between the two peaks in the unassigned data and the heterozygous peak location (green lines). (C) For unassigned alleles on trisomic heterozygous chromosomes, the allelic fractions were close to &#x223C;0.33 or &#x223C;0.67 (&#x201C;aab&#x201D; or &#x201C;abb,&#x201D; respectively, left panel). SNP pairs with fractions that fell in the &#x201C;abb&#x201D; peak were reassigned to &#x201C;aab&#x201D; (purple, middle panel), and vice-versa for &#x201C;aab&#x201D; alleles (blue, right panel). The boundary between the two regions was set at 50% of the distance between the two peaks (red lines). SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/523/F2.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/523/F2.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/523/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/523/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461077" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-17" class="first-child">Assignment of SNPs from allele hybridization ratios. Allelic fractions were calculated from SNP array hybridization data for each chromosome in each strain used to construct the hapmap. In each case, unassigned alleles (left panels with only gray data) were plotted with the X-axis as the allelic fraction [b/(a+b)] and the Y-axis as the number of SNP probes, scaled to maximum height of the peaks. (A) For unassigned alleles on heterozygous chromosomes, a = b and b/(a+b) = 0.5 (dark black vertical line); probes that fall between the boundaries (right panel, green lines, values determined as described in <em>Material and Methods</em>) are considered heterozygous (gray); probes outside these loci are considered homozygous. (B) For unassigned alleles on homozygous chromosomes, the allelic fractions were close to either 0 or 1 (left panel). For alleles on chromosomes previously designated “aa,” SNP pairs with fractions that fell in the “bb” peak were reassigned to “aa” (magenta, middle panel), and vice versa for “bb” alleles (cyan, right panel) using boundaries that were 50% of the distance between the two peaks in the unassigned data and the heterozygous peak location (green lines). (C) For unassigned alleles on trisomic heterozygous chromosomes, the allelic fractions were close to ∼0.33 or ∼0.67 (“aab” or “abb,” respectively, left panel). SNP pairs with fractions that fell in the “abb” peak were reassigned to “aab” (purple, middle panel), and vice-versa for “aab” alleles (blue, right panel). The boundary between the two regions was set at 50% of the distance between the two peaks (red lines). SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/523/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visualization of SNP/CGH array data. Data for each strain analyzed by SNP/CGH array was visualized as illustrated for strain Ps8 (and in Figure S1 and Figure S2 for all other strains). Each chromosome is illustrated to scale, with its centromere position indicated by an indentation and major repeat sequence (MRS) positions indicted by a black square below the chromosome. CGH data were calculated as ratios, converted to chromosome copy numbers, displayed as black histograms along the length of the chromosome, and summarized with a large numeral to the right of the chromosome. For SNP alleles, allelic fractions for each chromosome are plotted in the left panels. SNP data were calculated and colored as described in Figure 2. Regions that were homozygous in the reference strain or were not informative were not colored. CGH, comparative genome hybridization; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-117737049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Visualization of SNP/CGH array data. Data for each strain analyzed by SNP/CGH array was visualized as illustrated for strain Ps8 (and in Figure S1 and Figure S2 for all other strains). Each chromosome is illustrated to scale, with its centromere position indicated by an indentation and major repeat sequence (MRS) positions indicted by a black square below the chromosome. CGH data were calculated as ratios, converted to chromosome copy numbers, displayed as black histograms along the length of the chromosome, and summarized with a large numeral to the right of the chromosome. For SNP alleles, allelic fractions for each chromosome are plotted in the left panels. SNP data were calculated and colored as described in Figure 2. Regions that were homozygous in the reference strain or were not informative were not colored. CGH, comparative genome hybridization; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/523/F3.medium.gif" width="389" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/523/F3.medium.gif" width="389" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/523/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/523/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461059" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-18" class="first-child">Visualization of SNP/CGH array data. Data for each strain analyzed by SNP/CGH array was visualized as illustrated for strain Ps8 (and in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/FigureS2.pdf">Figure S2</a> for all other strains). Each chromosome is illustrated to scale, with its centromere position indicated by an indentation and major repeat sequence (MRS) positions indicted by a black square below the chromosome. CGH data were calculated as ratios, converted to chromosome copy numbers, displayed as black histograms along the length of the chromosome, and summarized with a large numeral to the right of the chromosome. For SNP alleles, allelic fractions for each chromosome are plotted in the left panels. SNP data were calculated and colored as described in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>. Regions that were homozygous in the reference strain or were not informative were not colored. CGH, comparative genome hybridization; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>DNA isolation, labeling, and microarray hybridization</h3><p id="p-19">Genomic DNA was prepared from overnight cultures and labeled with Cy3 or Cy5 thymidine, using Klenow polymerase described in (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Selmecki <em>et al.</em> 2005</a>). Labeled DNA concentrate was diluted to 16 µl with deionized water. An amount of 4.5 µl of Agilent 10× blocking agent and 22.5 µl of Agilent 2× hybridization buffer were added. Each mixture of Cy3- and Cy5-labeled DNA was treated according to Agilent standard protocols with incubation for 24 hr at 65°. After incubation, arrays were washed using wash buffers and acetonitrile solutions per Agilent standard protocols.</p><p id="p-20">Microarrays were scanned as 16-bit TIFF images and analyzed using “BlueFuse for Microarrays” (BlueGnome, Cambridge). Data were normalized using Block Lowess method and then analyzed at two levels of stringency (one with no data excluded; the second, data excluded when “Quality less than 1; Confidence less than 0.4; PON Ch1 or Ch2 less than 0.6”). In-house scripts, organized into a data analysis pipeline, were used to load, process, and display the data and are available upon request.</p></div><div id="sec-5" class="subsection"><h3>Fitness assays</h3><p id="p-21">Cells were grown at 30° to saturation in liquid YPAD media, diluted 1:1000 in fresh YPAD media and grown for 18 hr at 30° in a microplate reader (Sunrise model, Tecan). OD<sub>600</sub> was measured every 15 min and used for doubling time calculations. Each experiment included a minimum of six biological replicates for each strain, and each experiment was repeated independently at least twice. Error bars reflect the standard deviation from the mean and <em>P</em> values were determined by Student <em>t</em>-test (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Construction of a SNP/CGH microarray for efficient, high-resolution detection of LOH and CNVs</h3><p id="p-22">Detection of LOH is facilitated by accurate genotyping of SNP alleles. Recently, isothermal melting temperature probes were shown to maximize discrimination of SNP alleles (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Gresham <em>et al.</em> 2010</a>), making them ideal for high-resolution analysis of LOH events. To quickly, efficiently, and simultaneously detect large numbers of LOH and CNV events in <em>C. albicans</em>, we designed and constructed a custom microarray including ∼39,000 SNP alleles (2 SNP alleles/locus) plus ∼20,000 CGH (non-SNP) loci (see <em>Materials and Methods</em> for details) using the Agilent custom array format, which enables simultaneous analysis of eight sets of 60,000 probes. In initial calibration experiments, we found that the non-SNP oligonucleotides accurately reported chromosome copy number and that the SNP allele probes accurately distinguished chromosomes that were known to be heterozygous (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>) from those known to be homozygous (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>) (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Forche <em>et al.</em> 2008</a>).</p><p id="p-23">To assign all SNP alleles on each chromosome to a specific homolog (“a” or “b”), we used lab strains that are homozygous (“aa” or “bb”) or trisomic heterozygous (“aab” or “abb”) for specific chromosomes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Forche <em>et al.</em> 2008</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Legrand <em>et al.</em> 2008</a>). A total of 97% (38,032) of the SNP alleles were assigned to a specific homolog. The remaining 3% (1190) remained unmapped, either because the SNP was not present in the strains analyzed or because the strains carried multiple SNPs within the locus.</p><p id="p-24">First, we analyzed the CGH loci to determine copy number of whole chromosomes. Disomic (∼1:1 ratio) and trisomic (∼2:1 ratio) chromosomes were readily distinguishable. We then assigned the SNP alleles to a given homolog. SNP allele identity (initially arbitrary) was reassigned to the appropriate homolog using data from disomic homozygous (“aa” and “bb”) and/or trisomic (“aab” and “abb”) chromosomes. For disomic homozygous chromosomes, allelic fractions close to 0 or 1 were generally observed, although there was slight variation with each array experiment (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>). The threshold was set at 50% of the distance between the homozygous and heterozygous values (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>, A and B, green lines). For chromosomes previously identified as homozygous “bb,” the SNP alleles were reassigned to “bb” if their ratio fell in the “aa” region, and vice versa. For Chr1, no disomic homozygous strain was available. Thus, a strain trisomic and heterozygous for Chr1 (Ss2-l, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) was analyzed with the allelic fractions shifted (0.33 or 0.67) (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2C</a>), and the threshold was set at 50% of the distance between the two peaks (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2C</a>, red lines). SNP alleles were assigned to a homolog using similar logic to that described above.</p><p id="p-25">SNP/CGH array data collected from eight parasexual progeny isolates (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) were used to assign SNP alleles (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2</a>). For each strain, data were plotted as a function of chromosome position, and regions of the chromosomes that were informative were shaded to indicate their allelic content (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/FigureS1.pdf">Figure S1</a>; see <em>Materials and Methods</em>).</p></div><div id="sec-8" class="subsection"><h3>Analysis of a series of related laboratory strains</h3><p id="p-26">Hapmaps are primarily useful for the study of strains that are closely related to one another. The degree to which <em>C. albicans</em> lab strains undergo recombination during transformation and subsequent selection steps has not been determined. Thus, we analyzed the multiply marked strains BWP17 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wilson <em>et al.</em> 1999</a>) and SN76 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Noble and Johnson 2005</a>), as well as several intermediate strains used in their construction from reference strain SC5314. All strains ultimately were constructed from CAI-4 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), which lacks both copies of the counterselectable marker <em>URA3</em>. CAF2 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fonzi and Irwin 1993</a>), an intermediate in the construction of CAI-4, is a <em>ura3∆/URA3</em> strain derived from SC5314 by transformation (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fonzi and Irwin 1993</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/523/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome changes acquired during laboratory strain construction. Summary of SNP/CGH analyses of laboratory strains derived from SC5314 with chromosome copy number illustrated as black histograms and homolog identity colored as in Figure 3. Each strain is illustrated horizontally. Strain construction steps proceed from top to bottom. RM100#13 (bottom of A) was the parent for strain lineages in B and C. Genome changes with successive transformation and counterselection steps are enlarged and/or highlighted: new aneuploidies are highlighted with green borders; new LOH events are highlighted with yellow borders. Genome changes that were maintained in subsequent strains are highlighted with black borders. Subfigures for strains with italicized names were simulated from available data and SNP/CGH analysis of additional CAI-4-derived strains. CGH, comparative genome hybridization; LOH, loss of heterozygosity; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-117737049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome changes acquired during laboratory strain construction. Summary of SNP/CGH analyses of laboratory strains derived from SC5314 with chromosome copy number illustrated as black histograms and homolog identity colored as in Figure 3. Each strain is illustrated horizontally. Strain construction steps proceed from top to bottom. RM100#13 (bottom of A) was the parent for strain lineages in B and C. Genome changes with successive transformation and counterselection steps are enlarged and/or highlighted: new aneuploidies are highlighted with green borders; new LOH events are highlighted with yellow borders. Genome changes that were maintained in subsequent strains are highlighted with black borders. Subfigures for strains with italicized names were simulated from available data and SNP/CGH analysis of additional CAI-4-derived strains. CGH, comparative genome hybridization; LOH, loss of heterozygosity; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/523/F4.medium.gif" width="440" height="360"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/523/F4.medium.gif" width="440" height="360"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/523/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/523/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461072" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-27" class="first-child">Genome changes acquired during laboratory strain construction. Summary of SNP/CGH analyses of laboratory strains derived from SC5314 with chromosome copy number illustrated as black histograms and homolog identity colored as in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>. Each strain is illustrated horizontally. Strain construction steps proceed from top to bottom. RM100#13 (bottom of A) was the parent for strain lineages in B and C. Genome changes with successive transformation and counterselection steps are enlarged and/or highlighted: new aneuploidies are highlighted with green borders; new LOH events are highlighted with yellow borders. Genome changes that were maintained in subsequent strains are highlighted with black borders. Subfigures for strains with italicized names were simulated from available data and SNP/CGH analysis of additional CAI-4-derived strains. CGH, comparative genome hybridization; LOH, loss of heterozygosity; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Our analysis of LOH and CNVs in related strains revealed the history of changes that arose during strain construction. In brief, approximately 20% of the SC5314 genome is homozygous (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>) (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Butler <em>et al.</em> 2009</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Jones <em>et al.</em> 2004</a>). In CAF2, SNP/CGH analysis detected an additional 50 kb LOH on Chr7L, which extended the preexisting region of homozygosity (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). In CAI-4, an additional region of Chr3R (64 kb) was homozygosed (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>). In both CAF2 and CAI-4, the LOH event occurred within 12–25 kb of the centromere. Both of these LOH events are found in all derivatives of CAI-4. Furthermore, as noted previously (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Selmecki <em>et al.</em> 2005</a>), some CAI-4 isolates (<em>e.g.</em> CAI-4 F2, <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>) have one trisomy (Chr2), whereas others (<em>e.g.</em> CAI-4 F3, <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>) have two (Chr1 and Chr2).</p><p id="p-29">RM10 (<em>his1∆/HIS1, ura3∆∆</em>) was derived from a CAI-4 isolate by transformation and subsequent FOA selection; it acquired an LOH event that spans most of the left arm of trisomic Chr2 (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4A</a>, 1330 kb). RM100 #13 (<em>his1∆/his1∆::URA3</em>, <em>ura3∆∆)</em> lost one copy of Chr2, retaining the Chr2R “a” and “b” alleles, and it also lost a portion (∼40 kb) of the Chr5R “a” homolog distal to <em>HIS1</em> (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4A</a>) (<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Selmecki <em>et al.</em> 2005</a>). These changes were retained in strains RM1000 #6 (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4B</a>, <em>his1∆∆, ura3∆∆</em>) and BWP17 (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4B</a>, <em>arg4∆∆</em>, <em>his1∆∆</em>, <em>ura3∆∆).</em> The Chr5R distal, hemizygous region became homozygous disomic (“b” alleles) in transformant RM1000 #2 (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4C</a>, <em>ura3∆∆</em>, <em>his1∆∆</em>) (<a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">Selmecki <em>et al.</em> 2005</a>), and this was retained in its derivative SN76 (<a id="xref-fig-4-13" class="xref-fig" href="#F4">Figure 4C</a>, <em>ura3∆∆</em>, <em>his1∆∆</em>, <em>arg4∆∆</em>). Both BWP17 and SN76 underwent two transformations and two counterselections on 5-FOA to delete both copies of <em>ARG4.</em> BWP17 acquired an additional LOH on the terminal 30 kb of Chr4R during one of these stages, whereas the SN76 genotype is indistinguishable from that of RM1000 #2. Taken together, these results indicate that LOH, like aneuploidy, often arises during strain constructions involving DNA transformation. These LOH events generate lab strains that are more homozygous than SC5314.</p></div><div id="sec-9" class="subsection"><h3>Fitness consequences of LOH and aneuploidy</h3><p id="p-30">Ultimately, accumulated genomic changes have the potential to affect the cellular fitness of an organism. Fitness in asexual microorganisms such as <em>C. albicans</em> is often determined by population doubling time, although the maximum number of doublings in a population also plays an important role (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Addinall <em>et al.</em> 2011</a>). In yeasts, aneuploidy often causes reduced fitness [reviewed in <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Torres <em>et al.</em> (2008)</a>], although specific aneuploidies can also confer fitness advantages under selective as well as nonselective conditions (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dunham <em>et al.</em> 2002</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Selmecki <em>et al.</em> 2006</a>). To distinguish between the effects of aneuploidy and LOH on fitness, we measured the doubling times for SC5314 and its derivatives (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) under standard laboratory conditions. Sets of strains with similar degrees of LOH and different ploidies, as well as sets of strains with similar ploidies and different degrees of LOH, were compared. Overall, doubling-time measurements mirrored fitness measurements that incorporate the maximum number of doublings in a population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1/FigureS3.pdf">Figure S3</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/523/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In vitro growth of lab strains with different degrees of ploidy or LOH or with different URA3 copy number. (A) Deviations from diploidy correlate with increased doubling times. Doubling times (min) were measured for strains with similar degrees of LOH and different degrees whole-genome ploidy. Blue: CAF2 (2N), CAI-4 F2 [2.16N (2N+Chr2)] and CAI-4 F3 [2.38 (2N + (Chr1 + Chr2))] Green: RM1000 #2 (2N) and RM1000 #6 (1.997N) have identical genome content but are homozygous or hemizygous, respectively, for the terminal &#x223C;40 kb of Chr5R. Red: SN76 (2N) and BWP17 (1.997N), like the RM1000 strains, are homozygous and hemizygous for the tip of Chr5R; in addition, BWP17 acquired a small (30 kb) LOH on Chr4R. *Within each group, all strains are significantly different from each other (P &lt; 0.05, Student t-test). (B) Increases in percentage of the genome that is homozygous (as indicated on X-axis) correlate with increased doubling times. SC5314 and CAF2 are not different from each other (+) but are significantly different from RM100 #13 and SN76 (*P &lt; 0.05, Student t-test). (C) URA3 status does not affect in vitro doubling times in medium containing uridine. Pairs of strains (teal and purple) with similar ploidies and similar degrees of LOH that differ by one copy of URA3: SC5314 (URA3/URA3) vs. CAF2 (URA3/ura3&#x2206;), or SN76 (ura3&#x2206;/ura3&#x2206;) vs. SN95 (URA3/ura3)) were not significant different in doubling time relative to one another (+). LOH, loss ofheterozygosity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-117737049" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;In vitro&lt;/em&gt; growth of lab strains with different degrees of ploidy or LOH or with different &lt;em&gt;URA3&lt;/em&gt; copy number. (A) Deviations from diploidy correlate with increased doubling times. Doubling times (min) were measured for strains with similar degrees of LOH and different degrees whole-genome ploidy. Blue: CAF2 (2N), CAI-4 F2 [2.16N (2N+Chr2)] and CAI-4 F3 [2.38 (2N + (Chr1 + Chr2))] Green: RM1000 #2 (2N) and RM1000 #6 (1.997N) have identical genome content but are homozygous or hemizygous, respectively, for the terminal &#x223C;40 kb of Chr5R. Red: SN76 (2N) and BWP17 (1.997N), like the RM1000 strains, are homozygous and hemizygous for the tip of Chr5R; in addition, BWP17 acquired a small (30 kb) LOH on Chr4R. *Within each group, all strains are significantly different from each other (&lt;em&gt;P&lt;/em&gt; &lt; 0.05, Student &lt;em&gt;t&lt;/em&gt;-test). (B) Increases in percentage of the genome that is homozygous (as indicated on X-axis) correlate with increased doubling times. SC5314 and CAF2 are not different from each other (+) but are significantly different from RM100 #13 and SN76 (*&lt;em&gt;P&lt;/em&gt; &lt; 0.05, Student &lt;em&gt;t&lt;/em&gt;-test). (C) &lt;em&gt;URA3&lt;/em&gt; status does not affect &lt;em&gt;in vitro&lt;/em&gt; doubling times in medium containing uridine. Pairs of strains (teal and purple) with similar ploidies and similar degrees of LOH that differ by one copy of &lt;em&gt;URA3&lt;/em&gt;: SC5314 &lt;em&gt;(URA3/URA3)&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; CAF2 &lt;em&gt;(URA3/ura3&#x2206;),&lt;/em&gt; or SN76 (&lt;em&gt;ura3&#x2206;/ura3&#x2206;)&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; SN95 &lt;em&gt;(URA3/ura3)&lt;/em&gt;) were not significant different in doubling time relative to one another (+). LOH, loss ofheterozygosity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/523/F5.medium.gif" width="181" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/523/F5.medium.gif" width="181" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/523/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/523/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461040" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-31" class="first-child"><em>In vitro</em> growth of lab strains with different degrees of ploidy or LOH or with different <em>URA3</em> copy number. (A) Deviations from diploidy correlate with increased doubling times. Doubling times (min) were measured for strains with similar degrees of LOH and different degrees whole-genome ploidy. Blue: CAF2 (2N), CAI-4 F2 [2.16N (2N+Chr2)] and CAI-4 F3 [2.38 (2N + (Chr1 + Chr2))] Green: RM1000 #2 (2N) and RM1000 #6 (1.997N) have identical genome content but are homozygous or hemizygous, respectively, for the terminal ∼40 kb of Chr5R. Red: SN76 (2N) and BWP17 (1.997N), like the RM1000 strains, are homozygous and hemizygous for the tip of Chr5R; in addition, BWP17 acquired a small (30 kb) LOH on Chr4R. *Within each group, all strains are significantly different from each other (<em>P</em> &lt; 0.05, Student <em>t</em>-test). (B) Increases in percentage of the genome that is homozygous (as indicated on X-axis) correlate with increased doubling times. SC5314 and CAF2 are not different from each other (+) but are significantly different from RM100 #13 and SN76 (*<em>P</em> &lt; 0.05, Student <em>t</em>-test). (C) <em>URA3</em> status does not affect <em>in vitro</em> doubling times in medium containing uridine. Pairs of strains (teal and purple) with similar ploidies and similar degrees of LOH that differ by one copy of <em>URA3</em>: SC5314 <em>(URA3/URA3)</em> <em>vs.</em> CAF2 <em>(URA3/ura3∆),</em> or SN76 (<em>ura3∆/ura3∆)</em> <em>vs.</em> SN95 <em>(URA3/ura3)</em>) were not significant different in doubling time relative to one another (+). LOH, loss ofheterozygosity.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Strains harboring extra copies of large chromosomes had significantly increased doubling times compared with a euploid strain with a similar degree of LOH (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>, blue). Even very small changes in ploidy (<em>e.g.</em> truncation of 40 kb from Chr5R) resulted in longer doubling times (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>, green). Importantly, these strains have essentially identical genome content: RM1000 #6 is hemizygous and RM1000 #2 is homozygous for the terminal portion of Chr5R. A similar difference in Chr5R and a similar decrease in fitness distinguished BWP17 from SN76 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5A</a>, red), although we cannot rule out contributions from the small Chr4R LOH present in BWP17 but not in SN76.</p><p id="p-33">The effects of LOH on fitness are not as well documented, because high-resolution methods to measure LOH have not been widely available. Here, we compared the fitness of strains with similar ploidies and different degrees of LOH [SC5314 and CAF2 (Chr7L); RM100 #13 and SN76 (Chr2L, Chr3R, Chr5R, and Chr7L)] under replete growth conditions that should neutralize the contributions of any auxotrophic markers. Interestingly, larger tracts of homozygosity correlated with increased doubling time (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5B</a>), but small increases in homozygosity did not. It is very likely that this is due to the specific alleles involved in the LOH tracts rather than the size of the LOH tract <em>per se</em>.</p><p id="p-34">In <em>C. albicans</em>, <em>ura3∆/ura3∆</em> strains exhibit altered morphogenesis and reduced virulence (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bain <em>et al.</em> 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2003</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Noble and Johnson 2005</a>). Yet <em>in vitro</em> fitness costs have not been determined for <em>ura3∆/ura3∆</em> isolates. Comparison of <em>in vitro</em> growth of two sets of strains that differ in <em>URA3</em> status, yet have similar ploidy (SC5314 and CAF2; SN76 and SN95), revealed no significant change in doubling time (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5C</a>). Thus, under standard lab conditions, using YPAD medium supplemented with uridine, the <em>URA3</em> genotype has no significant effect on fitness. In general, aneuploidy and LOH both reduce fitness <em>in vitro</em>. Importantly, this is likely due to multiple allele-specific effects in both cases. Furthermore, the degree to which they affect fitness likely differs under different growth conditions.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-35">Long-range chromosomal changes such as LOH and CNVs confer large-effect phenotypic variation. Here we describe microarrays that perform both high-resolution SNP and CGH analysis simultaneously. When used together with strains that are homozygous or trisomic, this technology reveals a high-resolution hapmap that allows homolog phasing of the largely heterozygous <em>C. albicans</em> genome. In addition, analysis of lab strain construction history revealed increasing levels of LOH with increasing strain manipulation. We assume that stresses involved in transformation, as well as the stress of growing on counterselection medium, increases the frequencies of LOH as it increases aneuploidy (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bouchonville <em>et al.</em> 2009</a>).</p><p id="p-36">Aneuploidy is dynamic and reversible. For example, trisomies can be lost via nondisjunction, restoring the original genotype or resulting in whole-chromosome homozygosis. Recombination between trisomic chromosomes and subsequent chromosome loss can result in large segmental LOH tracts (<em>e.g.</em> Chr2 in RM10 and its derivatives, <a id="xref-fig-4-14" class="xref-fig" href="#F4">Figure 4</a>). Shorter segmental monosomies (<em>e.g.</em> Chr5R terminus in RM100) can be restored to disomy, but the alleles remain homozygous. Thus, while aneuploidies can appear and disappear relatively quickly, they can affect the degree of homozygosity. Thus, aneuploidies can cause transient or permanent genome changes.</p><p id="p-37">If LOH events occur in aneuploid or euploid strains, the loss of allelic diversity cannot be reversed without events involving mating or mutagenesis. Accordingly, lab strains that underwent more manipulations generally have accumulated more LOH events. These LOH events and aneuploidies do not confer a selective advantage under standard laboratory conditions. In contrast, aneuploidies and LOH events found in azole-resistant clinical isolates can provide a clear selective advantage in the presence of the drug due to specific gene contributions (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Coste <em>et al.</em> 2007</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Selmecki <em>et al.</em> 2008</a>). This is reminiscent of the situation in <em>S. cerevisiae</em>, where aneuploidies are usually detrimental to growth in standard conditions (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Torres <em>et al.</em> 2007</a>) but can provide a selective advantage under some stress conditions (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Pavelka <em>et al.</em> 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rancati <em>et al.</em> 2008</a>) and in some genetic backgrounds (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hughes <em>et al.</em> 2000</a>).</p><p id="p-38">Paradoxically, despite the stable accumulation of LOH tracts with sequential exposure to stress, most clinical isolates are largely heterozygous. This implies that parasexual mating between nonsisters must occur at a high enough frequency to restore the observed heterozygosity. Alternatively, selection forces <em>in vivo</em> strongly favor the survival of strains with more heterozygosity.</p><p id="p-39">Heat shock, which is used to isolate transformants, clearly promotes aneuploidy (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Bouchonville <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hilton <em>et al.</em> 1985</a>). This may be due to depletion of HSP90 and its cofactor Sgt1, which in <em>Saccharomyces cerevisiae</em> is required for assembly and stability of centromere/kinetochore and centrosome complexes (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bansal <em>et al.</em> 2009</a>; Zhang <em>et al.</em> 2008) and may influence the incidence of aneuploidy in that organism. Elevated temperature, such as that in febrile patients (<em>e.g.</em> 39°), also results in increased frequencies of LOH (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Forche <em>et al.</em> 2011</a>), and thus, the heat shock step in DNA transformations may promote recombination that leads to LOH as well. However, it should be noted that transformation by electroporation also results in increased aneuploidy (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Bouchonville <em>et al.</em> 2009</a>).</p><p id="p-40">Strong selection pressure applied to identify rare transformants also promotes the appearance of aneuploidy [<em>e.g.</em> on 5-FOA, (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wellington and Rustchenko 2005</a>)]. LOH rates also increase under the stress of growth <em>in vivo</em> (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Forche <em>et al.</em> 2009a</a>) and in response to fluconazole or oxidative stress applied <em>in vitro</em> (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Forche <em>et al.</em> 2011</a>).</p><p id="p-41">The effects of large aneuploidies, especially extra copies of Chr1 and Chr2, which increase ploidy by ∼20%, had a more dramatic effect on doubling time and fitness than the most extensive cumulative LOH (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>, BWP17 and SN76, homozygous for almost ∼35% of the genome). However, because SC5314 is already homozygous for ∼20% of the genome, there may be less dynamic range available to detect effects of LOH on fitness. Importantly, the effect of LOH and aneuploidy on fitness is likely to vary, depending upon the specific genes and alleles involved, as well as on the growth conditions being used to measure fitness.</p><p id="p-42">In summary, we designed and demonstrated the utility of a combined SNP/CGH array for high-resolution, rapid, and accurate detection of large-effect genome changes, including both LOH and CNV events. This array, together with its analysis pipeline, is a useful and cost-effective tool for characterizing laboratory as well as clinical strains. Furthermore, this study highlights that whole-genome analysis of engineered mutants should be performed prior to using such mutants in critical experiments. This will facilitate the identification of strains with the desired levels of ploidy and heterozygosity. In addition, the hapmap alleles provided here will greatly facilitate the phasing of SNPs in commonly used laboratory strains. This is essential for optimal appreciation of allele specificity found within next-generation sequencing data. Clearly, once high-throughput sequencing and data analysis becomes affordable and accessible to all researchers, it will obviate the need for array analysis. However, until that time, this technology and the data pipelines for its analysis are important and useful tools for the <em>Candida albicans</em> research community.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank Anja Forche for helpful discussions of SNP data analysis, Amnon Koren for assistance in the initial microarray design, Richard Bennett for providing parasexual progeny strains, Jason Funt for suggesting the horizontal layout for consecutive isolates in <a id="xref-fig-4-15" class="xref-fig" href="#F4">Figure 4</a>, and Katie Matter for help with manuscript preparation. We thank Anja Forche, Norman Pavelka, and Pete Magee for helpful comments on the manuscript. Computational resources for microarray image analysis were provided by the Minnesota Supercomputing Institute. This work was supported by National Institute of Allergy and Infectious Diseases (NIAID) Grant R01 AI-0624273 (to J.B.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000885/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-44"><em>Communicating editor: R. B. Brem</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-08-12"><span class="received-label">Received </span>August 12, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-10-03"><span class="accepted-label">Accepted </span>October 3, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Abbey <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.1" data-doi="10.1371/journal.pgen.1001362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holstein</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAddinall%26rft.auinit1%253DS.%2BG.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1001362%26rft.epage%253De1001362%26rft.atitle%253DQuantitative%2Bfitness%2Banalysis%2Bshows%2Bthat%2BNMD%2Bproteins%2Band%2Bmany%2Bother%2Bprotein%2Bcomplexes%2Bsuppress%2Bor%2Benhance%2Bdistinct%2Btelomere%2Bcap%2Bdefects.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001362%26rft_id%253Dinfo%253Apmid%252F21490951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21490951&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.2" data-doi="10.1128/EC.2.2.351-361.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Monge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro-Garcia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negredo</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Hog1 mitogen-activated protein kinase is essential in the oxidative stress response and chlamydospore formation in Candida albicans.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DAlonso-Monge%26rft.auinit1%253DR.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D351%26rft.epage%253D361%26rft.atitle%253DThe%2BHog1%2BMitogen-Activated%2BProtein%2BKinase%2BIs%2BEssential%2Bin%2Bthe%2BOxidative%2BStress%2BResponse%2Band%2BChlamydospore%2BFormation%2Bin%2BCandidaalbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.2.2.351-361.2003%26rft_id%253Dinfo%253Apmid%252F12684384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIyLzIvMzUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzUyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.3" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.4" data-doi="10.1534/genetics.108.089250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hetrick</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genetic screen for increased loss of heterozygosity in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D179%26rft.spage%253D1179%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089250%26rft_id%253Dinfo%253Apmid%252F18562670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.089250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18562670&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258313400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.5" data-doi="10.1111/j.1567-1364.2009.00563.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arbour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epp</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogues</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sellam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacroix</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Widespread occurrence of chromosomal aneuploidy following the routine production of Candida albicans mutants.</span> <abbr class="cit-jnl-abbrev">FEM. Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1070</span>–<span class="cit-lpage">1077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BResearch%26rft.stitle%253DFEMS%2BYeast%2BRes%26rft.aulast%253DArbour%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D1070%26rft.epage%253D1077%26rft.atitle%253DWidespread%2Boccurrence%2Bof%2Bchromosomal%2Baneuploidy%2Bfollowing%2Bthe%2Broutine%2Bproduction%2Bof%2BCandida%2Balbicans%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00563.x%26rft_id%253Dinfo%253Apmid%252F19732157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2009.00563.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19732157&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arendrup</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Effron</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buzina</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortensen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Breakthrough <em>Aspergillus fumigatus</em> and <em>Candida albicans</em> double infection during caspofungin treatment: laboratory characteristics and implication for susceptibility testing. Antimicrob.</span> <abbr class="cit-jnl-abbrev">Agents Chemother.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgents%2BChemother.%26rft.volume%253D53%26rft.spage%253D1185%26rft_id%253Dinfo%253Apmid%252F19104024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19104024&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.7" data-doi="10.1111/j.1574-6968.2001.tb10905.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bain</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stubberfield</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gow</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Ura-status-dependent adhesion of Candida albicans mutants.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiology%2BLetters%26rft.stitle%253DFEMS%2BMicrobiol%2BLett%26rft.aulast%253DBain%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D204%26rft.issue%253D2%26rft.spage%253D323%26rft.epage%253D328%26rft.atitle%253DUra-status-dependent%2Badhesion%2Bof%2BCandida%2Balbicans%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2001.tb10905.x%26rft_id%253Dinfo%253Apmid%252F11731143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2001.tb10905.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11731143&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.8" data-doi="10.1074/jbc.M806281200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nourse</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdulle</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagawa</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sgt1 dimerization is required for yeast kinetochore assembly.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">3586</span>–<span class="cit-lpage">3592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M806281200%26rft_id%253Dinfo%253Apmid%252F19073600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODQvNi8zNTg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzUyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.9" data-doi="10.1128/EC.00209-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouchonville</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">K. E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Aneuploid chromosomes are highly unstable during DNA transformation of Candida albicans.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1554</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00209-09%26rft_id%253Dinfo%253Apmid%252F19700634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI4LzEwLzE1NTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.10" data-doi="10.1016/j.jinf.2006.11.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brion</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uko</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Risk of resistance associated with fluconazole prophylaxis: systematic review.</span> <abbr class="cit-jnl-abbrev">J. Infect.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Binfection%26rft.stitle%253DJ%2BInfect%26rft.aulast%253DBrion%26rft.auinit1%253DL.%2BP.%26rft.volume%253D54%26rft.issue%253D6%26rft.spage%253D521%26rft.epage%253D529%26rft.atitle%253DRisk%2Bof%2Bresistance%2Bassociated%2Bwith%2Bfluconazole%2Bprophylaxis%253A%2Bsystematic%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jinf.2006.11.017%26rft_id%253Dinfo%253Apmid%252F17239952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jinf.2006.11.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17239952&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247096600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.11" data-doi="10.1038/nature08064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">M. A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakthikumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of pathogenicity and sexual reproduction in eight Candida genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DButler%26rft.auinit1%253DG.%26rft.volume%253D459%26rft.issue%253D7247%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DEvolution%2Bof%2Bpathogenicity%2Band%2Bsexual%2Breproduction%2Bin%2Beight%2BCandida%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08064%26rft_id%253Dinfo%253Apmid%252F19465905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19465905&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266608600034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumamoto</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chromosome 1 trisomy compromises the virulence of Candida albicans.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol.%26rft.volume%253D51%26rft.spage%253D551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.13" data-doi="10.1128/EC.00151-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coste</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diogo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bougnoux</span>  <span class="cit-name-given-names">M.-E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genotypic evolution of azole resistance mechanisms in sequential Candida albicans isolates.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1889</span>–<span class="cit-lpage">1904</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00151-07%26rft_id%253Dinfo%253Apmid%252F17693596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI2LzEwLzE4ODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.14" data-doi="10.1073/pnas.242624799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badrane</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16144</span>–<span class="cit-lpage">16149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.242624799%26rft_id%253Dinfo%253Apmid%252F12446845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjUvMTYxNDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fonzi</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">M. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Isogenic strain construction and gene mapping in Candida albicans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D134%26rft.spage%253D717%26rft_id%253Dinfo%253Apmid%252F8349105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8349105&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LJ51300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.16" data-doi="10.1128/mBio.00129-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pisithkul</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinzierl</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringstrom</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stress alters rates and types of loss of heterozygosity in Candida albicans.</span> <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00129</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D2%26rft.spage%253De00129%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00129-11%26rft_id%253Dinfo%253Apmid%252F21791579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00129-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21791579&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.17" data-doi="10.1371/journal.pbio.0060110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alby</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The parasexual cycle in Candida albicans provides an alternative pathway to meiosis for the formation of recombinant strains.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DForche%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De110%26rft.epage%253De110%26rft.atitle%253DThe%2Bparasexual%2Bcycle%2Bin%2BCandida%2Balbicans%2Bprovides%2Ban%2Balternative%2Bpathway%2Bto%2Bmeiosis%2Bfor%2Bthe%2Bformation%2Bof%2Brecombinant%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060110%26rft_id%253Dinfo%253Apmid%252F18462019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18462019&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.18" data-doi="10.1534/genetics.109.103325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009a</span> <span class="cit-article-title">Evolution in Candida albicans populations during a single passage through a mouse host.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D799%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103325%26rft_id%253Dinfo%253Apmid%252F19414562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.103325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19414562&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271972300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.19" data-doi="10.1111/j.1567-1364.2009.00542.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinbach</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009b</span> <span class="cit-article-title">Efficient and rapid identification of Candida albicans allelic status using SNP-RFLP.</span> <abbr class="cit-jnl-abbrev">FEM. Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BResearch%26rft.stitle%253DFEMS%2BYeast%2BRes%26rft.aulast%253DForche%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D1061%26rft.epage%253D1069%26rft.atitle%253DEfficient%2Band%2Brapid%2Bidentification%2Bof%2BCandida%2Balbicans%2Ballelic%2Bstatus%2Busing%2BSNP-RFLP.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00542.x%26rft_id%253Dinfo%253Apmid%252F19622074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2009.00542.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19622074&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.20" data-doi="10.1128/EC.4.1.156-165.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">P. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Demonstration of loss of heterozygosity by single-nucleotide polymorphism microarray analysis and alterations in strain morphology in Candida albicans strains during infection.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DForche%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D165%26rft.atitle%253DDemonstration%2Bof%2BLoss%2Bof%2BHeterozygosity%2Bby%2BSingle-Nucleotide%2BPolymorphism%2BMicroarray%2BAnalysis%2Band%2BAlterations%2Bin%2BStrain%2BMorphology%2Bin%2BCandida%2Balbicans%2BStrains%2Bduring%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.1.156-165.2005%26rft_id%253Dinfo%253Apmid%252F15643071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI0LzEvMTU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzUyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.21" data-doi="10.1007/BF00328721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillum</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsay</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirsch</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Isolation of the Candida albicans gene for orotidine-5′-phosphate decarboxylase by complementation of S. cerevisiae ura3 and E. coli pyrF mutations.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DGillum%26rft.auinit1%253DA.%2BM.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D179%26rft.epage%253D182%26rft.atitle%253DIsolation%2Bof%2Bthe%2BCandida%2Balbicans%2Bgene%2Bfor%2Borotidine-5%2527-phosphate%2Bdecarboxylase%2Bby%2Bcomplementation%2Bof%2BS.%2Bcerevisiae%2Bura3%2Band%2BE.%2Bcoli%2BpyrF%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00328721%26rft_id%253Dinfo%253Apmid%252F6394964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00328721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6394964&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984AEM2900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.22" data-doi="10.1073/pnas.0913883107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brizuela</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimized detection of sequence variation in heterozygous genomes using DNA microarrays with isothermal-melting probes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1482</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0913883107%26rft_id%253Dinfo%253Apmid%252F20080586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzQvMTQ4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.23" data-doi="10.1371/journal.pgen.1000303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenq</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGresham%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000303%26rft.epage%253De1000303%26rft.atitle%253DThe%2Brepertoire%2Band%2Bdynamics%2Bof%2Bevolutionary%2Badaptations%2Bto%2Bcontrolled%2Bnutrient-limited%2Benvironments%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000303%26rft_id%253Dinfo%253Apmid%252F19079573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079573&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-1.7.523.24"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite> <span class="cit-comment">(Editors)</span>, <span class="cit-pub-date">1991</span> <span class="cit-source">Guide to Yeast Genetics and Molecular Biology</span>. <span class="cit-publ-name">Academic Press, Inc.</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadany</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beker</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">On the evolutionary advantage of fitness-associated recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2167</span>–<span class="cit-lpage">2179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D165%26rft.spage%253D2167%26rft_id%253Dinfo%253Apmid%252F14704195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14704195&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.26" data-doi="10.1007/BF00383330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilton</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markie</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rikkerink</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulter</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Heat shock induces chromosome loss in the yeast <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DHilton%26rft.auinit1%253DC.%26rft.volume%253D200%26rft.issue%253D1%26rft.spage%253D162%26rft.epage%253D168%26rft.atitle%253DHeat%2Bshock%2Binduces%2Bchromosome%2Bloss%2Bin%2Bthe%2Byeast%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00383330%26rft_id%253Dinfo%253Apmid%252F3897792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00383330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3897792&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.27" data-doi="10.1038/77116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Widespread aneuploidy revealed by DNA microarray expression profiling.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D333%26rft.epage%253D337%26rft.atitle%253DWidespread%2Baneuploidy%2Brevealed%2Bby%2BDNA%2Bmicroarray%2Bexpression%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F77116%26rft_id%253Dinfo%253Apmid%252F10888885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/77116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10888885&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087920900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.28" data-doi="10.1073/pnas.0401648101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Federspiel</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chibana</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dungan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The diploid genome sequence of Candida albicans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">7329</span>–<span class="cit-lpage">7334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401648101%26rft_id%253Dinfo%253Apmid%252F15123810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE5LzczMjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.29" data-doi="10.1534/genetics.107.071506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimating the per-base-pair mutation rate in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLang%26rft.auinit1%253DG.%2BI.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D82%26rft.atitle%253DEstimating%2Bthe%2BPer-Base-Pair%2BMutation%2BRate%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071506%26rft_id%253Dinfo%253Apmid%252F18202359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc4LzEvNjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.30" data-doi="10.1371/journal.pgen.0040001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legrand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Haplotype mapping of a diploid non-meiotic organism using existing and induced aneuploidies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLegrand%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253De1%26rft.epage%253De1%26rft.atitle%253DHaplotype%2Bmapping%2Bof%2Ba%2Bdiploid%2Bnon-meiotic%2Borganism%2Busing%2Bexisting%2Band%2Binduced%2Baneuploidies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040001%26rft_id%253Dinfo%253Apmid%252F18179283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18179283&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marr</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rustad</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowden</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Rapid, transient fluconazole resistance in Candida albicans is associated with increased mRNA levels of CDR.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2584</span>–<span class="cit-lpage">2589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2BAgents%2Band%2BChemotherapy%26rft.stitle%253DAntimicrob.%2BAgents%2BChemother.%26rft.aulast%253DMarr%26rft.auinit1%253DK.%2BA.%26rft.volume%253D42%26rft.issue%253D10%26rft.spage%253D2584%26rft.epage%253D2589%26rft.atitle%253DRapid%252C%2BTransient%2BFluconazole%2BResistance%2Bin%2BCandida%2Balbicans%2BIs%2BAssociated%2Bwith%2BIncreased%2BmRNA%2BLevels%2Bof%2BCDR%26rft_id%253Dinfo%253Apmid%252F9756759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi8xMC8yNTg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzUyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.32" data-doi="10.1086/515553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marr</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Burik</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowden</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Development of fluconazole resistance in <em>Candida albicans</em> causing disseminated infection in a patient undergoing marrow transplantation.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">908</span>–<span class="cit-lpage">910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F515553%26rft_id%253Dinfo%253Apmid%252F9356806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/515553&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9356806&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YD93500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.33" data-doi="10.1371/journal.ppat.0030164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bla</span>  <span class="cit-name-given-names">B. W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Homayouni</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The transcription factor Mrr1p controls expression of the MDR1 efflux pump and mediates multidrug resistance in Candida albicans.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DMorschhauser%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De164%26rft.epage%253De164%26rft.atitle%253DThe%2Btranscription%2Bfactor%2BMrr1p%2Bcontrols%2Bexpression%2Bof%2Bthe%2BMDR1%2Befflux%2Bpump%2Band%2Bmediates%2Bmultidrug%2Bresistance%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0030164%26rft_id%253Dinfo%253Apmid%252F17983269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0030164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17983269&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.34" data-doi="10.1093/bioinformatics/bth440"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kung</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Accurate detection of aneuploidies in array CGH and gene expression microarray data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3533</span>–<span class="cit-lpage">3543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth440%26rft_id%253Dinfo%253Apmid%252F15284100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth440&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15284100&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225786600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.35" data-doi="10.1007/s10266-009-0118-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niimi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannon</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Antifungal drug resistance of oral fungi.</span> <abbr class="cit-jnl-abbrev">Odontology</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOdontology%2B%252F%2Bthe%2BSociety%2Bof%2Bthe%2BNippon%2BDental%2BUniversity%26rft.stitle%253DOdontology%26rft.aulast%253DNiimi%26rft.auinit1%253DM.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D25%26rft.atitle%253DAntifungal%2Bdrug%2Bresistance%2Bof%2Boral%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10266-009-0118-3%26rft_id%253Dinfo%253Apmid%252F20155503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10266-009-0118-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20155503&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274471100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.36" data-doi="10.1128/EC.4.2.298-309.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Strains and strategies for large-scale gene deletion studies of the diploid human fungal pathogen Candida albicans.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">298</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DNoble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D298%26rft.epage%253D309%26rft.atitle%253DStrains%2Band%2BStrategies%2Bfor%2BLarge-Scale%2BGene%2BDeletion%2BStudies%2Bof%2Bthe%2BDiploid%2BHuman%2BFungal%2BPathogen%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.2.298-309.2005%26rft_id%253Dinfo%253Apmid%252F15701792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI0LzIvMjk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzUyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.37" data-doi="10.1038/nature09529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saraf</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Aneuploidy confers quantitative proteome changes and phenotypic variation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">468</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPavelka%26rft.auinit1%253DN.%26rft.volume%253D468%26rft.issue%253D7321%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DAneuploidy%2Bconfers%2Bquantitative%2Bproteome%2Bchanges%2Band%2Bphenotypic%2Bvariation%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09529%26rft_id%253Dinfo%253Apmid%252F20962780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20962780&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284051000054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.38" data-doi="10.1016/j.cell.2008.09.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleharty</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trimble</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Aneuploidy underlies rapid adaptive evolution of yeast cells deprived of a conserved cytokinesis motor.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRancati%26rft.auinit1%253DG.%26rft.volume%253D135%26rft.issue%253D5%26rft.spage%253D879%26rft.epage%253D893%26rft.atitle%253DAneuploidy%2Bunderlies%2Brapid%2Badaptive%2Bevolution%2Bof%2Byeast%2Bcells%2Bdeprived%2Bof%2Ba%2Bconserved%2Bcytokinesis%2Bmotor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.09.039%26rft_id%253Dinfo%253Apmid%252F19041751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.09.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19041751&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261183900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.39" data-doi="10.2165/00003495-200868140-00002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruping</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vehreschild</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornely</span>  <span class="cit-name-given-names">O. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Patients at high risk of invasive fungal infections: when and how to treat.</span> <abbr class="cit-jnl-abbrev">Drugs</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1941</span>–<span class="cit-lpage">1962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrugs%26rft.stitle%253DDrugs%26rft.aulast%253DRuping%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D68%26rft.issue%253D14%26rft.spage%253D1941%26rft.epage%253D1962%26rft.atitle%253DPatients%2Bat%2Bhigh%2Brisk%2Bof%2Binvasive%2Bfungal%2Binfections%253A%2Bwhen%2Band%2Bhow%2Bto%2Btreat.%26rft_id%253Dinfo%253Adoi%252F10.2165%252F00003495-200868140-00002%26rft_id%253Dinfo%253Apmid%252F18778118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2165/00003495-200868140-00002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18778118&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260621900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.40" data-doi="10.1111/j.1365-2958.2005.04492.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative genome hybridization reveals widespread aneuploidy in Candida albicans laboratory strains.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1553</span>–<span class="cit-lpage">1565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSelmecki%26rft.auinit1%253DA.%26rft.volume%253D55%26rft.issue%253D5%26rft.spage%253D1553%26rft.epage%253D1565%26rft.atitle%253DComparative%2Bgenome%2Bhybridization%2Breveals%2Bwidespread%2Baneuploidy%2Bin%2BCandida%2Balbicans%2Blaboratory%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2005.04492.x%26rft_id%253Dinfo%253Apmid%252F15720560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2005.04492.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15720560&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227086300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.41" data-doi="10.1126/science.1128242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Aneuploidy and isochromosome formation in drug-resistant Candida albicans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSelmecki%26rft.auinit1%253DA.%26rft.volume%253D313%26rft.issue%253D5785%26rft.spage%253D367%26rft.epage%253D370%26rft.atitle%253DAneuploidy%2Band%2BIsochromosome%2BFormation%2Bin%2BDrug-Resistant%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128242%26rft_id%253Dinfo%253Apmid%252F16857942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTMvNTc4NS8zNjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.42" data-doi="10.1111/j.1365-2958.2008.06176.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerami-Nejad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An isochromosome confers drug resistance in vivo by amplification of two genes, ERG11 and TAC1.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">624</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSelmecki%26rft.auinit1%253DA.%26rft.volume%253D68%26rft.issue%253D3%26rft.spage%253D624%26rft.epage%253D641%26rft.atitle%253DAn%2Bisochromosome%2Bconfers%2Bdrug%2Bresistance%2Bin%2Bvivo%2Bby%2Bamplification%2Bof%2Btwo%2Bgenes%252C%2BERG11%2Band%2BTAC1.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2008.06176.x%26rft_id%253Dinfo%253Apmid%252F18363649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2008.06176.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18363649&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254641600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.43" data-doi="10.1371/journal.pgen.1000705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dulmage</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowen</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Acquisition of aneuploidy provides increased fitness during the evolution of antifungal drug resistance.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSelmecki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000705%26rft.epage%253De1000705%26rft.atitle%253DAcquisition%2Bof%2Baneuploidy%2Bprovides%2Bincreased%2Bfitness%2Bduring%2Bthe%2Bevolution%2Bof%2Bantifungal%2Bdrug%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000705%26rft_id%253Dinfo%253Apmid%252F19876375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19876375&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.44" data-doi="10.1016/j.ijantimicag.2009.09.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Characterisation of breakthrough invasive mycoses in echinocandin recipients: an evidence-based review.</span> <abbr class="cit-jnl-abbrev">Int. J. Antimicrob. Agents</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bantimicrobial%2Bagents%26rft.stitle%253DInt%2BJ%2BAntimicrob%2BAgents%26rft.aulast%253DSun%26rft.auinit1%253DH.%2BY.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D211%26rft.epage%253D218%26rft.atitle%253DCharacterisation%2Bof%2Bbreakthrough%2Binvasive%2Bmycoses%2Bin%2Bechinocandin%2Brecipients%253A%2Ban%2Bevidence-based%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijantimicag.2009.09.020%26rft_id%253Dinfo%253Apmid%252F20036518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijantimicag.2009.09.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20036518&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.45" data-doi="10.1126/science.1142210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolsky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boselli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Effects of aneuploidy on cellular physiology and cell division in haploid yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTorres%26rft.auinit1%253DE.%2BM.%26rft.volume%253D317%26rft.issue%253D5840%26rft.spage%253D916%26rft.epage%253D924%26rft.atitle%253DEffects%2Bof%2BAneuploidy%2Bon%2BCellular%2BPhysiology%2Band%2BCell%2BDivision%2Bin%2BHaploid%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142210%26rft_id%253Dinfo%253Apmid%252F17702937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTg0MC85MTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.46" data-doi="10.1534/genetics.108.090878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Aneuploidy: cells losing their balance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D179%26rft.spage%253D737%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090878%26rft_id%253Dinfo%253Apmid%252F18558649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.090878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18558649&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256892800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.47" data-doi="10.1002/yea.1191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellington</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rustchenko</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">5-Fluoro-orotic acid induces chromosome alterations in Candida albicans.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWellington%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D70%26rft.atitle%253D5-Fluoro-orotic%2Bacid%2Binduces%2Bchromosome%2Balterations%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1191%26rft_id%253Dinfo%253Apmid%252F15635674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15635674&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226590700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.48" data-doi="10.1128/AAC.46.6.1704-1713.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holleman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dy</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirels</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Resistance mechanisms in clinical isolates of Candida albicans.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1704</span>–<span class="cit-lpage">1713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2BAgents%2Band%2BChemotherapy%26rft.stitle%253DAntimicrob.%2BAgents%2BChemother.%26rft.aulast%253DWhite%26rft.auinit1%253DT.%2BC.%26rft.volume%253D46%26rft.issue%253D6%26rft.spage%253D1704%26rft.epage%253D1713%26rft.atitle%253DResistance%2BMechanisms%2Bin%2BClinical%2BIsolates%2Bof%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.46.6.1704-1713.2002%26rft_id%253Dinfo%253Apmid%252F12019079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjQ2LzYvMTcwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Rapid hypothesis testing with Candida albicans through gene disruption with short homology regions.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1868</span>–<span class="cit-lpage">1874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DWilson%26rft.auinit1%253DR.%2BB.%26rft.volume%253D181%26rft.issue%253D6%26rft.spage%253D1868%26rft.epage%253D1874%26rft.atitle%253DRapid%2BHypothesis%2BTesting%2Bwith%2BCandida%2Balbicans%2Bthrough%2BGene%2BDisruption%2Bwith%2BShort%2BHomology%2BRegions%26rft_id%253Dinfo%253Apmid%252F10074081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4MS82LzE4NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-1.7.523.50" data-doi="10.1038/emboj.2008.190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botër</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadota</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panaretou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Structural and functional coupling of Hsp90- and Sgt1-centred multi-protein complexes.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2789</span>–<span class="cit-lpage">2798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DZhang%26rft.auinit1%253DM.%26rft.volume%253D27%26rft.issue%253D20%26rft.spage%253D2789%26rft.epage%253D2798%26rft.atitle%253DStructural%2Band%2Bfunctional%2Bcoupling%2Bof%2BHsp90-%2Band%2BSgt1-centred%2Bmulti-protein%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2008.190%26rft_id%253Dinfo%253Apmid%252F18818696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2008.190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18818696&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F523.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260501100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/7/523.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/6/515" title="Genome-Scale Analysis of Programmed DNA Elimination Sites in Tetrahymena thermophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/7/531" title="Transposon-Mediated Transgenesis in the Short-Lived African Killifish Nothobranchius furzeri, a Vertebrate Model for Aging" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/7"><span>Volume 1 Issue 7, December 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/7.cover.gif?itok=8vFpWSyK" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/7/523" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-87lQMovsiTxpStNDKTn6U-Ec8YGaxdBUSDphlAGfN4g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/7/523" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/7/523" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/7/523" class="active" data-icon-position="" data-hide-link-title="0">High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used Candida albicans Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459204" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Vpbhrs3RkirXDbDGX-xSQOWCZWLmksxZ7Udm5cAUrrA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/7/523.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/7/523" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9WUZRv0Z0HAxXdWa_cbpd1fANtAQyjoQLUeaAbWcCww" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/7/523" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/7/523.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459204" id="node459204--2" data-pisa="ggg;1/7/523" data-pisa-master="ggg;g3.111.000885" data-apath="/ggg/1/7/523.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used <em>Candida albicans</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Darren Abbey</span>, <span class="highwire-citation-author" data-delta="1">Meleah Hickman</span>, <span class="highwire-citation-author" data-delta="2">David Gresham</span> and <span class="highwire-citation-author" data-delta="3">Judith Berman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">523-530;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000885 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459204--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Darren Abbey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BAbbey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abbey%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BAbbey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Meleah Hickman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meleah%2BHickman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickman%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMeleah%2BHickman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology, University of Minnesota, Minneapolis, Minnesota 55455, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Judith Berman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomics and Systems Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJudith%2BBerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jberman@umn.edu" class="" data-icon-position="" data-hide-link-title="0">jberman@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/7/523" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/7/523&amp;title=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/7/523&amp;title=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/7/523&amp;title=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/7/523&amp;text=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/7/523&amp;title=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/7/523&amp;t=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/7/523&amp;title=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/7/523&amp;title=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459204/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459204" id="node459204--3" data-pisa="ggg;1/7/523" data-pisa-master="ggg;g3.111.000885" data-apath="/ggg/1/7/523.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution SNP/CGH Microarrays Reveal the Accumulation of Loss of Heterozygosity in Commonly Used <em>Candida albicans</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Darren Abbey</span>, <span class="highwire-citation-author" data-delta="1">Meleah Hickman</span>, <span class="highwire-citation-author" data-delta="2">David Gresham</span> and <span class="highwire-citation-author" data-delta="3">Judith Berman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">523-530;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000885 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459204--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Darren Abbey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BAbbey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abbey%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BAbbey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Meleah Hickman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Meleah%2BHickman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickman%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMeleah%2BHickman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Microbiology, University of Minnesota, Minneapolis, Minnesota 55455, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, New York University, New York, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Judith Berman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Cell Biology and Development and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Genomics and Systems Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judith%2BBerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berman%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJudith%2BBerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jberman@umn.edu" class="" data-icon-position="" data-hide-link-title="0">jberman@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=523&amp;rft.epage=530&amp;rft.atitle=High-Resolution%20SNP/CGH%20Microarrays%20Reveal%20the%20Accumulation%20of%20Loss%20of%20Heterozygosity%20in%20Commonly%20Used%20Candida%20albicans%20Strains&amp;rft.volume=1&amp;rft.issue=7&amp;rft.date=2011-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Abbey%2C+Darren&amp;rft.au=Hickman%2C+Meleah&amp;rft.au=Gresham%2C+David&amp;rft.au=Berman%2C+Judith"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459204/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459204/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459204/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459204/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459204/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459204/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459204/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459204/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459204/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459204/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459204/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459204/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/7/523" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/7/523.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/7/523.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/7/523.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/7/523.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
