<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/7/549" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/7/549.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/7/549.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/7/549.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/1/7/549/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/7/549.atom" />
<meta name="HW.pisa" content="ggg;1/7/549" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Analysis of Nascent Transcription in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.111.000810" />
<meta name="DC.Date" content="2011-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2011 McKinlay et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The assessment of transcriptional regulation requires a genome-wide survey of active RNA polymerases. Thus, we combined the nuclear run-on assay, which labels and captures nascent transcripts, with high-throughput DNA sequencing to examine transcriptional activity in exponentially growing Saccharomyces cerevisiae . Sequence read data from these nuclear run-on libraries revealed that transcriptional regulation in yeast occurs not only at the level of RNA polymerase recruitment to promoters but also at postrecruitment steps. Nascent synthesis signals are strongly enriched at TSS throughout the yeast genome, particularly at histone loci. Nascent transcripts reveal antisense transcription for more than 300 genes, with the read data providing support for the activity of distinct promoters driving transcription in opposite directions rather than bidirectional transcription from single promoters. By monitoring total RNA in parallel, we found that transcriptional activity accounts for 80% of the variance in transcript abundance. We computed RNA stabilities from nascent and steady-state transcripts for each gene and found that the most stable and unstable transcripts encode proteins whose functional roles are consistent with these stabilities. We also surveyed transcriptional activity after heat shock and found that most, but not all, heat shock-inducible genes increase their abundance by increasing their RNA synthesis. In summary, this study provides a genome-wide view of RNA polymerase activity in yeast, identifies regulatory steps in the synthesis of transcripts, and analyzes transcript stabilities." />
<meta name="DC.Contributor" content="Anastasia McKinlay" />
<meta name="DC.Contributor" content="Carlos L. Araya" />
<meta name="DC.Contributor" content="Stanley Fields" />
<meta name="article:published_time" content="2011-12-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Genome-Wide Analysis of Nascent Transcription in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The assessment of transcriptional regulation requires a genome-wide survey of active RNA polymerases. Thus, we combined the nuclear run-on assay, which labels and captures nascent transcripts, with high-throughput DNA sequencing to examine transcriptional activity in exponentially growing &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. Sequence read data from these nuclear run-on libraries revealed that transcriptional regulation in yeast occurs not only at the level of RNA polymerase recruitment to promoters but also at postrecruitment steps. Nascent synthesis signals are strongly enriched at TSS throughout the yeast genome, particularly at histone loci. Nascent transcripts reveal antisense transcription for more than 300 genes, with the read data providing support for the activity of distinct promoters driving transcription in opposite directions rather than bidirectional transcription from single promoters. By monitoring total RNA in parallel, we found that transcriptional activity accounts for 80% of the variance in transcript abundance. We computed RNA stabilities from nascent and steady-state transcripts for each gene and found that the most stable and unstable transcripts encode proteins whose functional roles are consistent with these stabilities. We also surveyed transcriptional activity after heat shock and found that most, but not all, heat shock-inducible genes increase their abundance by increasing their RNA synthesis. In summary, this study provides a genome-wide view of RNA polymerase activity in yeast, identifies regulatory steps in the synthesis of transcripts, and analyzes transcript stabilities.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/12/01" />
<meta name="citation_mjid" content="ggg;1/7/549" />
<meta name="citation_id" content="1/7/549" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/7/549" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/7/549.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/7/549.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/7/549.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000810" />
<meta name="citation_pmid" content="22384366" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="549" />
<meta name="citation_lastpage" content="558" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anastasia McKinlay" />
<meta name="citation_author_institution" content="Department of Genome Sciences" />
<meta name="citation_author_institution" content="Department of Medicine" />
<meta name="citation_author" content="Carlos L. Araya" />
<meta name="citation_author_institution" content="Department of Genome Sciences" />
<meta name="citation_author" content="Stanley Fields" />
<meta name="citation_author_institution" content="Department of Genome Sciences" />
<meta name="citation_author_institution" content="Department of Medicine" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_email" content="fields@u.washington.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=E. Bengal;citation_author=Y. Aloni;citation_title=A block of transcription elongation by RNA polymerase II at synthetic sites in vitro.;citation_pages=9791-9798;citation_volume=264;citation_year=1989;citation_issue=17;citation_pmid=2722878" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. B. Brachmann;citation_author=A. Davies;citation_author=G. J. Cost;citation_author=E. Caputo;citation_author=J. Li;citation_title=Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.;citation_pages=115-132;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9483801;citation_doi=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. M. Brown;citation_author=J. M. Jeltsch;citation_author=M. Roberts;citation_author=P. Chambon;citation_title=Activation of pS2 gene transcription is a primary response to estrogen in the human breast cancer cell line MCF-7.;citation_pages=6344-6348;citation_volume=81;citation_year=1984;citation_issue=20;citation_pmid=6593703;citation_doi=10.1073/pnas.81.20.6344" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=F. Carrillo Oesterreich;citation_author=S. Preibisch;citation_author=K. M. Neugebauer;citation_title=Global analysis of nascent RNA reveals transcriptional pausing in terminal exons.;citation_pages=571-581;citation_volume=40;citation_year=2010;citation_issue=4;citation_pmid=21095587;citation_doi=10.1016/j.molcel.2010.11.004" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=L. Castells-Roca;citation_author=J. GarcÃ­a-MartÃ­nez;citation_author=J. Moreno;citation_author=E. Herrero;citation_author=G. BellÃ­;citation_title=Heat shock response in yeast involves changes in both transcription rates and mRNA stabilities.;citation_pages=2;citation_volume=6;citation_year=2011;citation_doi=10.1371/journal.pone.0025328" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. S. Churchman;citation_author=J. S. Weissman;citation_title=Nascent transcript sequencing visualizes transcription at nucleotide resolution.;citation_pages=368-373;citation_volume=469;citation_year=2011;citation_issue=7330;citation_pmid=21248844;citation_doi=10.1038/nature09652" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. J. Core;citation_author=J. J. Waterfall;citation_author=J. T. Lis;citation_title=Nascent RNA sequencing reveals widespread pausing and divergent initiation at human promoters.;citation_volume=322;citation_year=2008;citation_pmid=19056941;citation_doi=10.1126/science.1162228" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=B. Dujon;citation_title=The yeast genome project: what did we learn?;citation_pages=263-270;citation_volume=12;citation_year=1996;citation_issue=7;citation_pmid=8763498;citation_doi=10.1016/0168-9525(96)10027-5" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=E. Eden;citation_author=R. Navon;citation_author=I. Steinfeld;citation_author=D. Lipson;citation_author=Z. Yakhini;citation_title=GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.;citation_pages=48-48;citation_volume=10;citation_year=2009;citation_pmid=19192299;citation_doi=10.1186/1471-2105-10-48" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. C. Friedel;citation_author=L. Dolken;citation_author=Z. Ruzsics;citation_author=U. H. Koszinowski;citation_author=R. Zimmer;citation_title=Conserved principles of mammalian transcriptional regulation revealed by RNA half-life.;citation_volume=37;citation_year=2009;citation_pmid=19561200;citation_doi=10.1093/nar/gkp542" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. J. Fuda;citation_author=M. B. Ardehali;citation_author=J. T. Lis;citation_title=Defining mechanisms that regulate RNA polymerase II transcription in vivo.;citation_pages=186-192;citation_volume=461;citation_year=2009;citation_issue=7261;citation_pmid=19741698;citation_doi=10.1038/nature08449" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. Garcia-Martinez;citation_author=A. Aranda;citation_author=J. E. Perez-Ortin;citation_title=Genomic run-on evaluates transcription rates for all yeast genes and identifies gene regulatory mechanisms.;citation_pages=303-313;citation_volume=15;citation_year=2004;citation_issue=2;citation_pmid=15260981;citation_doi=10.1016/j.molcel.2004.06.004" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. Gariglio;citation_author=M. Bellard;citation_author=P. Chambon;citation_title=Clustering of RNA polymerase B molecules in the 5' moiety of the adult {beta}-globin gene of hen erythrocytes;citation_pages=2589-2598;citation_volume=9;citation_year=1981;citation_issue=11;citation_pmid=6269056;citation_doi=10.1093/nar/9.11.2589" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=A. P. Gasch;citation_author=P. T. Spellman;citation_author=C. M. Kao;citation_author=O. Carmel-Harel;citation_author=M. B. Eisen;citation_title=Genomic Expression Programs in the Response of Yeast Cells to Environmental Changes;citation_pages=4241-4257;citation_volume=11;citation_year=2000;citation_issue=12;citation_pmid=11102521;citation_doi=10.1091/mbc.11.12.4241" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. Grigull;citation_author=S. Mnaimneh;citation_author=J. Pootoolal;citation_author=M. D. Robinson;citation_author=T. R. Hughes;citation_title=Genome-Wide Analysis of mRNA Stability Using Transcription Inhibitors and Microarrays Reveals Posttranscriptional Control of Ribosome Biogenesis Factors;citation_pages=5534-5547;citation_volume=24;citation_year=2004;citation_issue=12;citation_pmid=15169913;citation_doi=10.1128/MCB.24.12.5534-5547.2004" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. S. Hahn;citation_author=Z. Hu;citation_author=D. J. Thiele;citation_author=V. R. Iyer;citation_title=Genome-Wide Analysis of the Biology of Stress Responses through Heat Shock Transcription Factor;citation_pages=5249-5256;citation_volume=24;citation_year=2004;citation_issue=12;citation_pmid=15169889;citation_doi=10.1128/MCB.24.12.5249-5256.2004" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=D. K. Hawley;citation_author=R. G. Roeder;citation_title=Separation and partial characterization of three functional steps in transcription initiation by human RNA polymerase II;citation_pages=8163-8172;citation_volume=260;citation_year=1985;citation_issue=13;citation_pmid=2409080" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=L. M. Hereford;citation_author=M. A. Osley;citation_author=T. R. Ludwig;citation_author=C. S. McLaughlin;citation_title=Cell-cycle regulation of yeast histone mRNA.;citation_pages=367-375;citation_volume=24;citation_year=1981;citation_issue=2;citation_pmid=7016339;citation_doi=10.1016/0092-8674(81)90326-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. T. Ingolia;citation_author=S. Ghaemmaghami;citation_author=J. R. Newman;citation_author=J. S. Weissman;citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.;citation_volume=324;citation_year=2009;citation_pmid=19213877;citation_doi=10.1126/science.1168978" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. G. Keene;citation_author=A. Mueller;citation_author=R. Landick;citation_author=L. London;citation_title=Transcriptional pause, arrest and termination sites for RNA polymerase II in mammalian N- and c-myc genes.;citation_volume=27;citation_year=1999;citation_pmid=10454615;citation_doi=10.1093/nar/27.15.3173" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Lefrancois;citation_author=G. M. Euskirchen;citation_author=R. K. Auerbach;citation_author=J. Rozowsky;citation_author=T. Gibson;citation_title=Efficient yeast ChIP-Seq using multiplex short-read DNA sequencing.;citation_pages=37-37;citation_volume=10;citation_year=2009;citation_pmid=19159457;citation_doi=10.1186/1471-2164-10-37" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Z. Levin;citation_author=M. Yassour;citation_author=X. Adiconis;citation_author=C. Nusbaum;citation_author=D. A. Thompson;citation_title=Comprehensive comparative analysis of strand-specific RNA sequencing methods.;citation_pages=709-715;citation_volume=7;citation_year=2010;citation_issue=9;citation_pmid=20711195;citation_doi=10.1038/nmeth.1491" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=B. Li;citation_author=J. A. Weber;citation_author=Y. Chen;citation_author=A. L. Greenleaf;citation_author=D. S. Gilmour;citation_title=Analyses of promoter-proximal pausing by RNA polymerase II on the hsp70 heat shock gene promoter in a Drosophila nuclear extract;citation_pages=5433-5443;citation_volume=16;citation_year=1996;citation_issue=10;citation_pmid=8816456" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. K. Mathur;citation_author=L. Sistonen;citation_author=I. R. Brown;citation_author=S. P. Murphy;citation_author=K. D. Sarge;citation_title=Deficient induction of human hsp70 heat shock gene transcription in Y79 retinoblastoma cells despite activation of heat shock factor 1.;citation_pages=8695-8699;citation_volume=91;citation_year=1994;citation_issue=18;citation_pmid=8078944;citation_doi=10.1073/pnas.91.18.8695" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=U. Nagalakshmi;citation_author=Z. Wang;citation_author=K. Waern;citation_author=C. Shou;citation_author=D. Raha;citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing.;citation_volume=320;citation_year=2008;citation_pmid=18451266;citation_doi=10.1126/science.1158441" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=R. Narsai;citation_author=K. A. Howell;citation_author=A. H. Millar;citation_author=N. O'Toole;citation_author=I. Small;citation_title=Genome-wide analysis of mRNA decay rates and their determinants in Arabidopsis thaliana.;citation_volume=19;citation_year=2007;citation_pmid=18024567;citation_doi=10.1105/tpc.107.055046" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Neil;citation_author=C. Malabat;citation_author=Y. d'Aubenton-Carafa;citation_author=Z. Xu;citation_author=L. M. Steinmetz;citation_title=Widespread bidirectional promoters are the major source of cryptic transcripts in yeast.;citation_pages=1038-1042;citation_volume=457;citation_year=2009;citation_issue=7232;citation_pmid=19169244;citation_doi=10.1038/nature07747" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=T. OâBrien;citation_author=J. T. Lis;citation_title=Rapid changes in Drosophila transcription after an instantaneous heat shock.;citation_pages=3456-3463;citation_volume=13;citation_year=1993;citation_issue=6;citation_pmid=8497261;citation_doi=10.1128/MCB.13.6.3456" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. Palangat;citation_author=T. I. Meier;citation_author=R. G. Keene;citation_author=R. Landick;citation_title=Transcriptional pausing at +62 of the HIV-1 nascent RNA modulates formation of the TAR RNA structure.;citation_pages=1033-1042;citation_volume=1;citation_year=1998;citation_issue=7;citation_pmid=9651586;citation_doi=10.1016/S1097-2765(00)80103-3" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Parkhomchuk;citation_author=T. Borodina;citation_author=V. Amstislavskiy;citation_author=M. Banaru;citation_author=L. Hallen;citation_title=Transcriptome analysis by strand-specific sequencing of complementary DNA.;citation_volume=37;citation_year=2009;citation_pmid=19620212;citation_doi=10.1093/nar/gkp596" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=G. Patrone;citation_author=F. Puppo;citation_author=R. Cusano;citation_author=M. Scaranari;citation_author=I. Ceccherini;citation_title=Nuclear run-on assay using biotin labeling, magnetic bead capture and analysis by fluorescence-based RT-PCR.;citation_pages=1012-1014, 1016â1017;citation_volume=29;citation_year=2000;citation_pmid=11084863" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=V. Pelechano;citation_author=S. Jimeno-Gonzalez;citation_author=A. Rodriguez-Gil;citation_author=J. Garcia-Martinez;citation_author=J. E. Perez-Ortin;citation_title=Regulon-specific control of transcription elongation across the yeast genome.;citation_pages=e1000614-e1000614;citation_volume=5;citation_year=2009;citation_issue=8;citation_pmid=19696888;citation_doi=10.1371/journal.pgen.1000614" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. Radonjic;citation_author=J. C. Andrau;citation_author=P. Lijnzaad;citation_author=P. Kemmeren;citation_author=T. T. Kockelkorn;citation_title=Genome-wide analyses reveal RNA polymerase II located upstream of genes poised for rapid response upon S. cerevisiae stationary phase exit.;citation_pages=171-183;citation_volume=18;citation_year=2005;citation_issue=2;citation_pmid=15837421;citation_doi=10.1016/j.molcel.2005.03.010" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. B. Rasmussen;citation_author=J. T. Lis;citation_title=In vivo transcriptional pausing and cap formation on three Drosophila heat shock genes.;citation_pages=7923-7927;citation_volume=90;citation_year=1993;citation_issue=17;citation_pmid=8367444;citation_doi=10.1073/pnas.90.17.7923" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Rodriguez-Gil;citation_author=J. Garcia-Martinez;citation_author=V. Pelechano;citation_author=L. Munoz-Centeno Mde;citation_author=V. Geli;citation_title=The distribution of active RNA polymerase II along the transcribed region is gene-specific and controlled by elongation factors.;citation_volume=38;citation_year=2010;citation_pmid=20385590;citation_doi=10.1093/nar/gkq215" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. E. Rougvie;citation_author=J. T. Lis;citation_title=Postinitiation transcriptional control in Drosophila melanogaster.;citation_pages=6041-6045;citation_volume=10;citation_year=1990;citation_issue=11;citation_pmid=2172790;citation_doi=10.1128/MCB.10.11.6041" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. J. Sims;citation_author=R. Belotserkovskaya;citation_author=D. Reinberg;citation_title=Elongation by RNA polymerase II: the short and long of it;citation_pages=2437-2468;citation_volume=18;citation_year=2004;citation_issue=20;citation_pmid=15489290;citation_doi=10.1101/gad.1235904" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. G. Theodorakis;citation_author=R. I. Morimoto;citation_title=Posttranscriptional regulation of hsp70 expression in human cells: effects of heat shock, inhibition of protein synthesis, and adenovirus infection on translation and mRNA stability.;citation_pages=4357-4368;citation_volume=7;citation_year=1987;citation_issue=12;citation_pmid=3437893;citation_doi=10.1128/MCB.7.12.4357" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. J. Venters;citation_author=B. F. Pugh;citation_title=A canonical promoter organization of the transcription machinery and its regulators in the Saccharomyces genome.;citation_volume=19;citation_year=2009;citation_pmid=19124666;citation_doi=10.1101/gr.084970.108" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=Y. Wang;citation_author=C. L. Liu;citation_author=J. D. Storey;citation_author=R. J. Tibshirani;citation_author=D. Herschlag;citation_title=Precision and functional specificity in mRNA decay.;citation_volume=99;citation_year=2002;citation_pmid=11972065;citation_doi=10.1073/pnas.092538799" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Z. Xu;citation_author=W. Wei;citation_author=J. Gagneur;citation_author=F. Perocchi;citation_author=S. Clauder-Munster;citation_title=Bidirectional promoters generate pervasive transcription in yeast.;citation_pages=1033-1037;citation_volume=457;citation_year=2009;citation_issue=7232;citation_pmid=19169243;citation_doi=10.1038/nature07728" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. Yassour;citation_author=J. Pfiffner;citation_author=J. Z. Levin;citation_author=X. Adiconis;citation_author=A. Gnirke;citation_title=Strand-specific RNA sequencing reveals extensive regulated long antisense transcripts that are conserved across yeast species.;citation_pages=R87-R87;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20796282;citation_doi=10.1186/gb-2010-11-8-r87" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Analysis of Nascent Transcription in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7.cover.gif" />
<meta name="twitter:description" content="The assessment of transcriptional regulation requires a genome-wide survey of active RNA polymerases. Thus, we combined the nuclear run-on assay, which labels and captures nascent transcripts, with high-throughput DNA sequencing to examine transcriptional activity in exponentially growing Saccharomyces cerevisiae . Sequence read data from these nuclear run-on libraries revealed that transcriptional regulation in yeast occurs not only at the level of RNA polymerase recruitment to promoters but also at postrecruitment steps. Nascent synthesis signals are strongly enriched at TSS throughout the yeast genome, particularly at histone loci. Nascent transcripts reveal antisense transcription for more than 300 genes, with the read data providing support for the activity of distinct promoters driving transcription in opposite directions rather than bidirectional transcription from single promoters. By monitoring total RNA in parallel, we found that transcriptional activity accounts for 80% of the variance in transcript abundance. We computed RNA stabilities from nascent and steady-state transcripts for each gene and found that the most stable and unstable transcripts encode proteins whose functional roles are consistent with these stabilities. We also surveyed transcriptional activity after heat shock and found that most, but not all, heat shock-inducible genes increase their abundance by increasing their RNA synthesis. In summary, this study provides a genome-wide view of RNA polymerase activity in yeast, identifies regulatory steps in the synthesis of transcripts, and analyzes transcript stabilities." />
<meta name="og-title" property="og:title" content="Genome-Wide Analysis of Nascent Transcription in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/7/549" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The assessment of transcriptional regulation requires a genome-wide survey of active RNA polymerases. Thus, we combined the nuclear run-on assay, which labels and captures nascent transcripts, with high-throughput DNA sequencing to examine transcriptional activity in exponentially growing Saccharomyces cerevisiae . Sequence read data from these nuclear run-on libraries revealed that transcriptional regulation in yeast occurs not only at the level of RNA polymerase recruitment to promoters but also at postrecruitment steps. Nascent synthesis signals are strongly enriched at TSS throughout the yeast genome, particularly at histone loci. Nascent transcripts reveal antisense transcription for more than 300 genes, with the read data providing support for the activity of distinct promoters driving transcription in opposite directions rather than bidirectional transcription from single promoters. By monitoring total RNA in parallel, we found that transcriptional activity accounts for 80% of the variance in transcript abundance. We computed RNA stabilities from nascent and steady-state transcripts for each gene and found that the most stable and unstable transcripts encode proteins whose functional roles are consistent with these stabilities. We also surveyed transcriptional activity after heat shock and found that most, but not all, heat shock-inducible genes increase their abundance by increasing their RNA synthesis. In summary, this study provides a genome-wide view of RNA polymerase activity in yeast, identifies regulatory steps in the synthesis of transcripts, and analyzes transcript stabilities." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7.cover.gif" />
    <title>Genome-Wide Analysis of Nascent Transcription in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3srYFtNc4UXeTFQjFByzh4Ie7tkxE0jGVABZm9AH9fQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459209","apath":"\/ggg\/1\/7\/549.atom","pisa":"ggg;1\/7\/549","ac":{"\/ggg\/1\/7\/549.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/7\/549.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/7\/549"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/7\/549"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/7\/549":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HH4RrJefXLfPqWF15u6c7UktHK6o60fQ8sn2ILACsEUFpoqIjPWLgbYdZ9zxYluA4cWYU1Af3HJ49HQAaxk2FoYNaoLfVqQfnKEKqVx+wynlFLmnXrpIeo6a3bmvK37U44widJC8PoXYwcxP57tBVhjdajsamV7sHL5JfQFHQYDVfNLBfvcwEC3WcKO7TtOTa7R\\\\\\\/MKdaIAnJtuJBq8z718wYSi3GuLNRr7Twft7jm\\\\\\\/QgdB0VA8cUdH4VHnSptP\\\\\\\/66FmZWqozvGJLnj6lEsOQLrecUXEckCi9jUCKzLRLrjWOgFOzPA1OzhAaDOW65Pw\\\\\\\/Db8h5bxSKtFtYzuOYHyHoilYAM9S6hYvuayZuiUhQ96GQKpNQCZjbLvTirdfPwP07L0B7dGMR5O4GBPUQdjhV0x67FmuSBMyyfG8DVjYG1agNE3G8Twamm3367OYnewNmnUfvLEyJ673QXVSM2ypJr7T9OPwUOA6Le1bq0pR4oFlM\\\\\\\/+EkWBO5lbbjDAD0SSqOuc265aj\\\\\\\/B1O8ADXN01upY8OYJA6rD3Wajl5K+7AiyPe+TVipGySTtdzkQfWtJxeQWU2OkdDR1mqCg5l43\\\\\\\/DSwmGEInw3HZnYDBf\\\\\\\/e6SJkxSpHDNrPRMGHtaRlYT6udWxxKKy7UBCOaWTz09r\\\\\\\/tw\\\\\\\/hu66b9KNTf26Q3\\\\\\\/1DtV1TAEaOC72\\\\\\\/OxGgrAKJ1KmdEUDtMgQOgIMICR5qzxU6RY6LmjSSC0oXXF6z1lDOoMco97yt0rHY7U6+hpgmur3CofFwwDdJp98b6DBsM4uosbGqOPB5G6Z+JGBeradXlyYW08jhYC7zYFGPRe\\\\\\\/Fgz2St8omAhpgiRzdhuhVkweiP7Mn1dRKdzOqAG9\\\\\\\/qAOdVeAwxh9lqUS2PmhXyOg54tVITDbdB6aS4nDxLsjc\\\\\\\/R3KrLoPmcnVSK8LZ4WSB8N8Gfw\\\\\\\/8L7CprvdjdOueRq+vi2Mjd9yEzrSkU0JVSaDu4snR25NR31sKFmQESNjOq5\\\\\\\/vzHRGZZQBpPzlFV1IQOBUvKOdSJ98GM9wDnl+blvjLc8P3CKqKjeasSHBWgLNaa0ohtbwjdCEOgLCKqyNXCZ9LIYjNUgWiELR3JZVRm+11AgcrR4rM8XEIEBEK+x8c7\\\\\\\/lkG3G103EHYs2vXtlMh36LduOz5r6FbGKKh9gB5olQFUNIqresYkz6\\\\\\\/dcGEIv1YypwxpFc9xwuifaxq3D7ftrPu8hc8N78GorRTvd8Aj8qpnW4RD816nOxD1oeShkmzwGWkazYFTP3W\\\\\\\/28X+Xsl+T+vaMQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022lHa12o8qHu09eQQV9G8u1Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002285b93b13e62a618b9ad67ac572aec85dd3731dee97385c2f169257addf92d6f6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AMBW8viwMl2DHoR9N042gOlMGUTDbPodrk5Tmqk1ZHBfT3Yt5SlE2xY4kyV\\\\\\\/i28xj2iVIsmgtNyp5WhXcZPCPISoeF2jJ81wOwAKOzASWrfyr0SBBHZN\\\\\\\/C9W2Xx2B+nQGS2frUjLRaDTdRVqboh5BiG7pNkVShsq\\\\\\\/tvtzUO\\\\\\\/l4wIMXeprSVtCbGjKOh6IHxlod+peMQTZ8bIaSV88PlrYk34gIHaaX4CsKD7FPpcqlkW9aEQsczyLoearmjuJCD452yAGARwaM3qVpbdJVpGHHQgz262k4C6+TGDh+2pmHv7dima8x6rTmI+xAutUYDfdhDopdvTdMp\\\\\\\/JsD5P3H6Weho2I++OkTAPMk90MOThnitgcl\\\\\\\/zhdY9P0py56MEOkRvGN3SwzNsCQbYCUCI7q0j0\\\\\\\/nZkUW+ltd321z4dVNIOPCK3o0x4BVCQeTRbPHw5zOdEgBmNz0+OflWd4HcBujDTvIKldrKKH6oDOBiJZZK3S3iZXSHPXLsb49y6SFtjBcav3VeP3lY6LZMX8tQ5SOn5ksOLN8MytYshwSSuV6wNFMmjzotIfGhU25qJXcYPcLmepibCJTMCqhtmky4iWx1sqVJdSz4VQaGuVBBo6FZ4nlYV+X12vaPZX0NDX01t9+NHYLE7BLL9oysjFZed3suduTbKWqXfVmuuh2OIYhhpIhe+nREh9\\\\\\\/sZjH4AUp+Y5x2DTlyHkoIUpM\\\\\\\/dlfQ3o8D3HBT6o5aXMRLRgHWoQFKbf7OZV4RNabvs3S2NlzZQOOU3rrcQNypiU52Je4\\\\\\\/qQoGvTiBEAFYFt52etDH6p4H08FHj4f8ZTLgA3g0p6gFhpqgYcgeBd4hnrRj7Aztf02HnatPj\\\\\\\/ir+ASYaSZIUacxHiCedJyVmp9UVV9O+movS2sYNder4G193RsTacHPcH3aocH+00a0cX9gHI39f6FKLdOUbzK2li1umVqhfYkgE1PEtG18jnMA1zmMRV9QtqQ21qRROkPrwgiVY\\\\\\\/Yd01DY9tDSJ6I91sn4cojlVWUSiFPIdsFhYGj4xs9GUai1gZcmrVzuHos6wvgNVUPMws1IEmCDe9GG3Stu9KyBauB5ZZ98jejuFmu1\\\\\\\/HJw\\\\\\\/A79DKoLHftauEXd3EQqO1kb7\\\\\\\/gSon9fHYQ3K85V7k1jcLeZes1DJZncfgzQjpHky2mdNzXIfiPRs4iZ3T2+W\\\\\\\/grMm8HXDiC5lcxyCECg0BTdFSx\\\\\\\/Dp5wtA+eBjr2QxzHe7C+BTTQlS40o\\\\\\\/\\\\\\\/ou4ZOyXS1mCBqEXGzN6O5n2w5QoZwdnV4GIpnC6b+zL+i3CgcIPqefhoVndPllgxMHnb5PE8bejE4VnXuhOMpMbJ2+mNzGobe9\\\\\\\/Vm15zwjXDx2x9Ox4\\\\\\\/3Y2Ps9qIsZ3T55wT4HcmlhCgw6NvkdxR\\\\\\\/dlVBRexbBVk48spHlwdAIMifW9c9MmmUtRjthGgp4wjjEaXQ+LV22fJoUhZ0lWSHcHy+WDXS6URulb5cHwLpD\\\\\\\/FlkKLdiU1bbR7Mx2l28+9J6fhCjssNv+1liPj0E0A2P11rZi\\\\\\\/\\\\\\\/FkO6BpeOwnBJxmReUDoFXt+fQVrVtA2WQF+m++bZgGE5zF66+brLktY1G25S1trh1nIOLzrb5Q1PMcsf2S4phuBRZ1uTK+DtUrIWHbm3chXrVn2xkgFH6Rvv0oCSdIfz133Rx+tRPexq6ueIDrvGbKcTfIvHo1u9JgD3HhmYNNVmeHfau4xh3KHSIkEx7fWaYZ6Gml6rHp+O3n55k67YdfDZJ7q28cyt7PT8rfmYMIgnqD0\\\\\\\/ulbbP2DgVI7oWXerveFR+\\\\\\\/O1VMAKSTffFR\\\\\\\/kSUqa04Y+hXISUztvRdLCRNEajIEpd7nJjC2hr1VB26gArX6UVV4Yrmavx1G5AXAP6KSn9pme3rb7ACeHKs\\\\\\\/vQIZTaY9MWykpTiLppHjbuXWYISZJyNqECSZjWVZjfcPk0odqsIjKsNtURTiQlAe7BRWBbkwkTvYA4f75uU2SOqDvSZrvwWtU9uUR05rm5fOi9moB1L90iLkf7EYdnO8AklAAqlH5WIOB6hrXi5z3BrlDb9tgUnmm2oHMCgUmIcnzu99uy3HOIwIWi4FZJupv\\\\\\\/aYparOua2HrLdZrEB5np188QCdEIN9j\\\\\\\/ld4jX3DVXHhMgAAsY5dPNGh9wyiElKz6FvVn4D7jCElCBYRZbXh1tA851ZzyoKJu+naN37ceX4ISgW2\\\\\\\/Dc6wORYDnULxsKGmkPdDlmoZxWMgiSaXFwY9L2feCoOfFI\\\\\\\/8HQPlxNLvnC7GXZ6um8gKR7bTo5dUGlubUAFTnywiQvRgo0o0LcB1QMkfNE3mAaPTcuvPbxpclR6uOmkGwZ7mRV3i7ariyeBVnMegt+lnwx3ZGTU\\\\\\\/j\\\\\\\/sOivU+yk5VuLKrEQzWCkId0N\\\\\\\/I+QKdUSMB7iCLGfPpoKKmt2Py\\\\\\\/jcStej6VZVzqeHYa7dNQQ\\\\\\\/SZChWexhXhzt\\\\\\\/N5lPShdF6oN+JfjbR2qrNWsYBWrLoG2cN5jaFFYQRyntqX1lahAbcoSb784Do7\\\\\\\/w0NnXn7CHezJjhvpPalsqrQ1vYPPd2NKq+FUkYdtw9UPLgJqIn0rdolBoXkB9\\\\\\\/NLzkeA5uuKSKhmyNMV+lvj\\\\\\\/VVsrnmOfFSZxepg1hWtYJd+hMhSM5LzAzs8HE+rVLUJ9+MxrDvJzea3PRcKnMf5G\\\\\\\/fUwF8c28qnyIpcoELqAh0GVLwbSBwHL26Dtap7hSm9ZjhkTUXvo++5Dc4PRaPgoEKKW1X0KhL7VfveIjlVbtfxt1+TuzIlHrNz7Xvk7BRBZ9TYFYNQv5vb8dVfCEhUtE9R7idt6+7F3hvtAmXwyVS3O92qJRYFjTovQ6FThxy9xpGW9tyuZKS406isnytMGGRIh55\\\\\\\/p12iMIrqcmuUp7aV+Su2VyvTDcrRPKSr3zoRXJhzZII4TlsOuGryEuyDRSx7s\\\\\\\/EOnYB0B7SYFJYnjiu6LmNpv0reVVja2MagTACGK1ZtKEJPPbi7KorNK3arIBHClW50G9e5KBMHSO3u2gUriVhH3J\\\\\\\/7NOWt+6HLEJnqllivjC+9\\\\\\\/J4UPVFGRwpJzUgMs4MVbATWwg7Sj+DDBBXG+XtU7up2SKllhcypu1LmKqfIUMa34Gb+hSJNR82DFaamR5Nt1HXQHAdwy6clyGhy3LC6QTYLb0NGPAkto+U5P3ev1xWU9E7vxGVooonDSmutm6pKARZajmLThxBgdx5YSf7Bb81HlEkBpiRC7PZ5pyjs01Mbb1TC9psHwHUU7MXwGLqwfi1eFxuERT0\\\\\\\/UTsuHVcXWAtqT+t1\\\\\\\/CnxCC8E4w+6MmuF0kyWGL87javFLSwCIzvknUSudgKzK32yMVsSMYqQiRKA78o123z\\\\\\\/3TUBOWbeRF+miOQJORQwj2V2k7jrj\\\\\\\/p3ccgPYPUpPClT3m9SmMEe8kUqYTrDZ5wiD8w28dg9Q4EI2SaRJI+x0iP\\\\\\\/hc0u+7Pc+JX+2Id1u+vNUS4+GWCQJa5Q4S60XsvqzuylcaoTQ7Jkpl\\\\\\\/MP4f+vtNdz\\\\\\\/LLl9gH0Gu2W7MbVdcLA1qcOjgOlNBIDPIx4hjPsniKuMCtOIb+yXVLvktDRW0Sb0lMmQ8nPVmdHhi3YtdMn71s4xgNliAoK5m2pT4tO+yE08N3qgIFbdHztRSgX+ReOH95wZFMUh84+VZsVdPg0n4fH5hsLISODcB5mYl\\\\\\\/YV62HaCZRIJkrAdQ8xy3ms1xrp9QrVtO\\\\\\\/3awV7NSg6MnMDp3ExEetkEs65f3tpIqkW9emOKpn0Z3lbal1uqNlPsO9JgnW\\\\\\\/5+a6+Dfy+4I7qtzviEzGoKmTy7Dc3v4n3IkqJRnYwJaZTQ38tufT463YBquIPOepwB+0JHZGLLQMjSe8T+G9OAM7KAKz7JMHoT4GhivHjwhT8pJbmHzPVOFgTOjlc9eRhK\\\\\\\/15sJGY5N+8gClllhD+xgphLfmf+6m8RRMNvuk4n\\\\\\\/JHl3SJagdAx54M+8TVJl9dkw1Nc6gpTYkdyq2wI+WX0tBADTA\\\\\\\/RhsY4IL1emn5PQwjOzPyGf3ZWqHZWpbFqdyi4zKYKkz9zpjGq0aqQIVgSRnhZrlUrsJ\\\\\\\/qQNVviRUo5xN+Rl7jxN3JAfoZVgpZzPJv2FWEHtx\\\\\\\/lLUL2J5QSJcI1BQNv+NokTLF\\\\\\\/2JSkeinBNtbXOetPgAjru4oIWDMB3QTmRZpvDZc4yNp8B6GksCLO2Wy2LArC9aToI\\\\\\\/u2QT+gMA1BksokRrbadHlrcqgcNEB0VU+9D1ufN58q23dnPQFAtu2G8eGLTbA21gDUZW3yXCC5CcauWOB5HjfoDriYRo2XljLwM8r\\\\\\\/XX0xg9\\\\\\\/tJk0EBKUYS72BOl7OmdTPSDDdNxC97Hz1rI8kOxRa0e\\\\\\\/X1mrJ1rSWKJ0+ylcf4QQV38W70FU6O7wCiqET6iYuDKxi2dWOwWczS5qub0S82AH\\\\\\\/0Q69VanwEWFgudcLYkjickE\\\\\\\/jwQb8qo8wotAs\\\\\\\/l6dTPnSdB5gejWrPwRQe7uL04Gnq8dmJ5b+4or3tTncbanKgMsAgzdlPdb2LMhOcpiYxtrc3F2Q\\\\\\\/zKxygRkJ4qpi1tW4PGtTqDAxQcpGjX2dMbxoC3LEcCB1nHNVMWhqSjKG7\\\\\\\/rt+7CrANZ1HAhkRlZ8w4SOdlOnAA4zUQbx37cf\\\\\\\/a7fCerQMofdq\\\\\\\/G4AsBWE\\\\\\\/AfNC4sE3RLyMBCykFrKPrkKzifanjIuy3wM5dlbbjjE0hoYf5XlGr8R9YcQTt8MLLAfySGXHSwDp8GlkXmaxNSDUcld3xkefXYjVRIuInF9LOTX7guwNWC8swJqgLSZIHD9+E2254EOcTL73pi1mcH3Hk3HNzsPNQExd79g18\\\\\\\/CZR5As6dVd+m532p2mM8X\\\\\\\/bYuCO9xhLkjDDx6PtMuBUJOESBmgMIJUo8JVSh8\\\\\\\/s08q4IRARL0G3e\\\\\\\/PIY4vPZ42LbPxotw6HflRsJg5UHoodXbyd+fNiL1qcqSqXqAu70zyj1e6Op+RaRaYTa6gqDN9tSbW39Jksw4AtClus2SlxWcRkdTz2a0rms7ssWJ0yZegZEZOWxt+6xLZVb2enKrpWcOlH1ftGSm6+XSoJchTvJ+gIhbLrTIhe5ub\\\\\\\/UsQ4TYNO06OhTQ5AgHm3L4BY7JObIBv57Y6KIgg\\\\\\\/LRxcd+C\\\\\\\/kv59yUbmrMVOtDPvNyy2WJVbQZYF12ewbDqEetBQeqLKGWvGbuUc72Q1BQqQDTCeUqgJKyLEaso6fFCmIhLELd1qdZ2sSejfb8unbeslA2fR\\\\\\\/flVrfg72MfMcdBUwRj4Dy9VHuNcIIq1wy1m4Q9ENy5a4ajcNKFGeJle+vtEHFCZJNIz+P8ZQx7mG\\\\\\\/+FekA5NOBNfiPmOoqirIdxcCYPPuE1yrKIW\\\\\\\/wenn+RZ\\\\\\\/0y01gzyGFjNyJ5liihr\\\\\\\/T4\\\\\\\/BGcdPrwZC82EK1okp3PmRMCn8Gfj3UqfldjXQRFmT1\\\\\\\/G4KBa5raABch921UpbhkURrx9AJDwlzU0QEwN+CFk03xGNkosEKAMxXTMhJa5apL9r0bcaXgC+9TkyFZjZb0TMTFXBdSb\\\\\\\/e0UgBbBhJSky+tcp7CsPAHri2BXPpFv1+KYqRmyeorlUwDy9g5+RsER\\\\\\\/RIkiE+mE\\\\\\\/PXpL2Mc3jZflqdVQQOvfPg62r3LTfINfxQZctaPwGJBqBvRQhfHF6Ym9BVi\\\\\\\/md9pSZF9Xhcc8BcN3zJiNdSWPLW9vsZ8ydV7hkteznHF6EO7eMwqjeIdnGPvMD6iOKiHrpbtdMXlE5sWcqVhjkjlDJHuZjkCqhcFANqmRaYwj5LZAfUjY6RaLT88+rh3rEZ52dNWSNB8V4uYL9XNjoFZZpcwKkMJRR28KzxEG9FqTN7NK\\\\\\\/KB9p7GdNIQXHc\\\\\\\/koeIfKqS9ip81+w+ZihXbQtk\\\\\\\/6THFOnH8FRicjcz4XB\\\\\\\/CDY4sea\\\\\\\/6xRxJcO1RR2RX1dFzknwdfBy57Yd7LC2ajdX9pQS2sAgP0Jp7RVaOOrnF4VESQ957sV3TOWFiafjVN+WDkN+mNY8IUdhHF4YxDsbQsqC4EfRNfu3MBxe5mmgbos2t9v19BpKXo6TTnTfC4fJyuqhgiUEsWzNV3zFz2DiMwj3Z9DMlkulYHQPg7l+Rzphcbs73ZiTVwhsqe\\\\\\\/0Qp14veCjffvroh4C1fLcFEL6H2TXUKWyFA3s7Q9oa\\\\\\\/w2sq5bd0aDFrlJPIGa8g2jxYk7J2BOFjbmSh1mEzCDZ6oTCBPWlBkvb57RtUuCPmlTYrDJ6942rvE7wb07U3lkDXhQgg3aKBQa6aCS7HKSjtfCI8129vBeoCzLImkxOUO6vR6CH\\\\\\\/CIk3uM0r30H4bqTPlCO7IKfOUQ5jN6O4RZauJFzLa89KckbOV\\\\\\\/MjihkdN5mHrVuBF7H+zuPND+SxlzYDy3APkfsjjIEBSYFWwyKFeFaVyWGhZrmecsNf2\\\\\\\/BoDjsM7OuZRp0qt+H3849d2Dq7bxkED3tK4+CSc7sWrXO6jnmfOZGqKNJEbcR4e8Xvwzs1VO4LEU5b1JigLhtuuY+l9M0DbQXeiySjO7Zkti6d2kXIyNz3HSzSNlv4NvBGJB1MmlIIRrorvp1Fd0DGUSDoYFt3L0Jl4E62leUb3XnrKX0KIQpNyNsRwSBXvBGHojiL1lFF6JBy1YEyzxbt5jtlg9qavS10Vhav+B9+ShM9nW5sJkw5OaxvPSt1N4geUDHiGUfztJsCjftqvobDvcLhh4405ybcfsPcDeq8dxNRIsu7uHqPmr2su036cBBe\\\\\\\/4rILwWwKr7XhQ\\\\\\\/B0xcdjhzDKPKNZG\\\\\\\/\\\\\\\/YngXM0S6Q6Ri8q4XNx\\\\\\\/hmVKJFf2rsPYxFnKUYNQfxIwOeKlC4vXDtys4h63l1mOwczrBcDnos6ti5QHzNdWyHfHURyOGB4joULMAWNAldsz4rkUqYiKqdzfa7ZJgbeJLhyOOWPSD0CuDJpBA\\\\\\\/GgsS7JAy08Eogy\\\\\\\/Wc+QZzUz+9RnMYI9run9dZSps83ipOmAcsdEhQ+cjK4pSlmY+cqGlH1hbjzik8UGowKWVISu4oVOVRhdLpi6kOdRFfqOxIZy77xbj21Iqf0SlSEGPkz9JmfJ62ERS6UMmqrAq\\\\\\\/BwunJeOhCit5rdiDEYfeXR8Dkiw0IH3AlCmNMGsPXYiq8LV5GWvk\\\\\\\/ik+vHHf0tTFKqHUGOPdP8o1+\\\\\\\/CB49J6bmYZc7YDp7pgWshhew6pAoEAXpNloDd3o4hoytf4EkQ+ekr5ZqcpeOkY6T6NmKFB2G5+8YVanX+AJ6Qfj65BzcrtZ4OGt1+89pacY\\\\\\\/QCdNDeGYTqW\\\\\\\/\\\\\\\/hbQVTTrmGqBflaaoZWo4uFfLvlbATePiSrRTLpL2p70XmJr45qlrtx7bUdOZhLXlszjnQlc7Gso4Z0H+7gSKNPGJ2BBmeDp7XLfLupncCCwtPVrdzjWbH9DOPdAQvBwwsse1qEVX4SEIGvmyWgZzBgq2c7RpAAOX0oOeorQozYTtwd+GqJoJJALSQ4QOTksi6LgfZ3axX8CzsjIEMZBNdDhKpnOR1AbEGZZ1qUa\\\\\\\/Lvb0OcicoDhcxNXtNWuhGMTWrTSVSj7MPKFJRfsxb8HzwJGWsaOlSVjXOunkGI0UweQm09O7NC5Jo5vzDCBs+P15T33Bxc9fKjhuZKxEFKiDqOQM6g1c5bC90H2watR0DIyA3mipmjlWSRdlF7I9ABEpSFeKLqbEw81fr5f0N+Pl\\\\\\\/scc7YUFKFGGQ4K5btJ0LHK45oW\\\\\\\/lW1k+2Q9X0SjURJjWU1mHyu1graYnTBrGw66EoJt0QO\\\\\\\/1LMqzeIL\\\\\\\/HipP96o8gJRcOBK7zvrODbjGgxR2J4PFB+mnc\\\\\\\/ZIWAekYCo05a+9b7wXFP2ut0+PEZSeKwVnbCY\\\\\\\/RksPNdpuiPaFwTQkO9j9h+mugIUbOibY5OEtR87I09t9j7A0uITiq+SdJZUSRfxawlzc5OSljQsXTaDE2QqyoKKQw19+gVo3X3ChwXpfZg05msLvT\\\\\\\/DJ4MQ73q1low523Hu4a8znxIbNtzIXuZshrzt18K4ClKFKz70rYhFALBFX+JikUmdW613Sln76jUi8cBtfmddvlBz1qZQF26tW8ko0g6HNtLAbbpwlI0zKc3N1q1A5nOtl+LX3LsmZOQlupnT0iaDK2f2sp0MH3WOX5ZS\\\\\\\/5OVIu5LRITs+NH40y3rZEUn+En883CZUleHS0y7UQxLM9RfjBxO\\\\\\\/fSLEUL42XiGxXirFX5JD62rgB4jsvsM0qkhKpNibAV7G9Xp0AOJsa7zF4phNWyBV\\\\\\\/Y+axdJoo\\\\\\\/EsOIUO0k7OypLtJ7Fc0s6034WKqW0ljJgQ4gzKBoYQI4Hb5ThcRWtJ\\\\\\\/CeUKdnfS9spDDChrU0tvvGKakx1Nuhw9ApmjbLUksN77\\\\\\\/I1KPWkNDYEdatR\\\\\\\/vwgpcA46Z5jQKjItn8r1Zwqr\\\\\\\/TxEPHCGkyo7V267dFpkw5x8vbiOi2wMF1JzU1McXndLqHrIXCE\\\\\\\/UGrGSrwi+pQ+ddtYf7w2BNmAaHPfR62tnulj4Za\\\\\\\/SX5uUmnGHiMkJY6vC+0F0iAlylqVYobhWcxPYgzl4w5zfw+ZzmbjPJOOr3VYiC9Htu+GCwmMACF9zBD1IHqc8ZVRnBdTuXwDZVJILRqXE+4S98BhOTxOaGXps2V\\\\\\\/ZQ5LWFoDnGdRwE9c+u\\\\\\\/M7JfGyPUSKdbFff7vMhQT0zzJ1lnl6Rp5HfPdxtuJyEGnXPZyDlYHZvliZqHj5R1\\\\\\\/AzYoJjQQA5EDlZ4VtK7rj8kc6nWEBdJinhrg8WxRl7GE0PMTYczWLPLU3OV\\\\\\\/7oIaJw7MEXhjszAnblGET7SEHJBn0KgXF4g0YUBmUxLofLlqS5Ol0PyHbZPrds7iC8JBm1a86Jw6ykNusn3fb7s\\\\\\\/aMR1TOc0OqzFIn4mJGUHZRjFuFJgaLh9A03cEFMqaxXbVvf5A1\\\\\\\/VC3osBIuvliXSTM0b3Co+W0TIkG\\\\\\\/k4edawEUjg9SnowQ2rsvAzhTnltwQleBu7rL\\\\\\\/\\\\\\\/1B3HIyWgFjYHP0FCFUkfJkFlCNYPV4da92qcrdhOlLAkBZPZyqCyGubHiUfeWseVydTBj5YOpzXYYwxSHRKwhMj8C04Hv5Tda4hdCQvLht26rRRHT1MWxdu9fu6sOphlYgE\\\\\\\/1iRRycvSLE7rLDeEZk1Afl7FWQZ88x3Gmvh4AOO9RgBsvqhQ2RPY3Kaj+daRXzsWhgd2uMGLzyGUeM697d1zLS3nQVH7ymjmUgsJ\\\\\\\/QfkO496yCa7aQ7EHKr+FeXzRHkLYEiYhybD2Yj5K\\\\\\\/f01M3MBxc9q4VDYU0aiCo85BfZ38VUh9ztxvbgULnDvROGqjjrvDJcvFtQblmUwXK5bZqtyYVIbs6\\\\\\\/3xeEWDCORL25FwMyRqdanCs\\\\\\\/YikhMU0GlUlyTOHnGZhuqDEf4YvUSXeO51eU4Hh7FN8JIKsk0sl\\\\\\\/SCNdB+BLTPnF7bU27UFlVmIGFocf1CdoZKs2pZRkcpuqIR5+L2stfm73gnXRQvt3FsC2Y4wSeVpsOQcWXbBFbZwkgln00132FPrF+Ja3z+PF2+yNw8ste7D\\\\\\\/Josw61ZRGva7jhfSYvEjwFtp4u+y+z5iab\\\\\\\/yRici7o3oF4R9CaGVEGU94zhmPABbX2UAI1FPOLqECSWQfURQ8J38tTQfAMJ+Ae+byfQlJy175X2Czl7adArh11pVhIcIYV3Z5XU1iwwVwa1ZIbVWG\\\\\\\/dmx3Oe\\\\\\\/nUcwz\\\\\\\/74WHPEHaWlUrlM29OO\\\\\\\/O6m5phkVX3IsSn8j+9q\\\\\\\/G7wAmfaKLVC9hEC7r+63kyJwI+NiD8hthnyaiRN\\\\\\\/OgQ7niVG5Xsc\\\\\\\/q9O\\\\\\\/7mXKlkpHJWK5rs502t75jRjPD7iGmu5wQCFlfGfcNrqSvfULXH3Opz4t1FMxWdBxT7BBbBTz8UuhxV71p2zCCeh6BRoNnFlEWizyEUrEaykwmD2KK\\\\\\\/Te9YDZj7vgpJMUjEBHsE7VHkzly34f0oVAZW\\\\\\\/d124qXrQ6jCZ4YmAvDDpSa1xp\\\\\\\/EDczo7cTH9hr2oX1HZMMfT7UF2A2dXtVBpEP+yVXsT77RFLF\\\\\\\/5QOE+7qLIAO7ixv8ZM02u0PEl2J6oxRG6akVL+FOK2u0DSY+hkmOFYuovE9JPal\\\\\\\/BarMqwk8S3x7UV\\\\\\\/KjJWHCInfnCZRrYDkC+a\\\\\\\/4\\\\\\\/X\\\\\\\/hPC8RUGUuB0obuhgj0wFURe\\\\\\\/q7nRLUZ2aLMvgGMErZX5FmNOTc+LOg04espJdIKrrA1w30Ux1WhUdIpDIn7glOz51zW7DTPQuO9oaTKoKs7YfYfH9pDSPHQBoKlmu+JwBP0r9T3f+v0m7llkw+5WxyzXt9i8Ep+KIapy9mKYwGVQKBZETGPxzDGpdgIX0TJrdwcLiN8+MdEL6A2+WGMtcNKrERwGidiSO3ndKTKXu0VOGhitKnvzDlsgSuXZql1nYE6A5niBRsIfleDPMwqcc+5FoL1PbGieaq42fgiyEOKX9u\\\\\\\/9Aqwe83fuxzjwV3Bqi2hAhEE2BvyNayAAidGiYl0s7\\\\\\\/ncep765YYGE8dqYJ71Kl2cvXfaZQr7X0S8liiOsipp7kIutnxr6B04gPkYYohTkL6Ssq4dS12aNPJgPjielEVJwX3bVslDbMD4fuLZwARsU6RMYx8PJkckMKg9vVs9JnF2DPpUjo5U1S\\\\\\\/xPf08yfkHxJT+q+y6CjhAkQOYa\\\\\\\/Xg7Zb59f9PvxlZ1q0LPy\\\\\\\/kgPZUHH4+d7T5qCfn2PWMolE\\\\\\\/\\\\\\\/kQVSCnUDy3Y0VdDx7c7hAQaquAGeX1ZcFzz1XxRfkwMerNEoxyr2hp+YuXdijeoJsOl3nXp9jIeTWhwEPJxa3PPvaA3+5UvKn\\\\\\\/NnyFa7a4GLCQkPDESuYENTcI0Tyzo6wyCbSPat+r+nBnbTgYvuj4Uh0CEAhoKRqMmqBEGRC2uRJYmEXirHJJPSZtfmNAwFTrI1HN3Cqe6S2c4luYmeyheUzi02xJRyBkLOlBAQ2j+ww17scXQ6XBVsFpUkuIgFmjhjSdz6i4Bv\\\\\\\/BUdoGFRrhIjHkZ3to1l5QXzbOPybsip6vOtxE1UwVVClD2\\\\\\\/JCXQKpHAyr5w9d5CP45zyZIWwpAG5mib4MJMu5p9ytYhOrh9vYq\\\\\\\/XdawiC2MtcZIu0tgMAJq6rGCX5E5dZWxH4a8teFUSc97bGhQQxdMtdIu55DyvtHlwCPraocQ36KiXgZAAOqW6RQ\\\\\\\/vu97arI\\\\\\\/TpElBZAiDEtC0PqDmil9PdCiFQdcUjFS7BKi7HzTd4SAEBFU41uexPOxAXkvcYnSCTG5ltcAjcfP+AholoPq7TIDJueVc+JxMW+wB97xGTjQSNxozEFJA710KoToheOYMyaBAtLmc64lBoQrCR\\\\\\\/hxZ9GekI1d1EkinmtoID1jZXYbt2suu\\\\\\\/Adw6t3d\\\\\\\/r1M\\\\\\\/E9oSE\\\\\\\/HeqrdoBBRiz8kzn1oQoT3HjwY96fw9hp+T4Ohv1tRvRrzV6xwNUseNgmy0XvBjjUkvAESc17\\\\\\\/eWnTG0f\\\\\\\/AbKrInT3XmCmHXSEEDnkSTrKSYyCUbEuf3W+6TWmEsR0xboUK9Lky0luwDPVNi0cAf+FkedARuQF+ANzxYqyip+ZUXqPmSeEMyXHkHjTcddnMBrJVfKoLAc9GOBx\\\\\\\/9XnGj0ppsZIOsA3BztDo0RNKq04WzOfbFBNuDoY6AahxNXZ\\\\\\\/TkCYbRPYnDIOv4SAZmCCqF1WqNBz7h5fXlC5ZWHK\\\\\\\/gLn1srf\\\\\\\/HWQYNsyrEbERFRdiUfaUNScyK5mrRldegtwkvFJHodqCpw2EUCOIUi0hgUe7TPzEozVmu0lZQlwmLFYPxbbKbqi+MC81L7zXz9anmmUdDjuLsQ2Iq4T5kMYHM5blqlZxG3\\\\\\\/SHvFtSXPCCNY19GQec2+OVV8Cl7NGW5n3KZA8mqO06f6VdtRswEfpoUPTPYc\\\\\\\/FYUMQ+NH9BA4rOF676BfQ00w8KYin4FBqI3eTpqG5xGxHC2pR2SgEQHlXVy0mSyoh3rSJZrUepTmbjV8NNGnSC1neBeaH3Dlc3BXX5r7g3HbvWpIVwkjikvaTywn3+g3VGoO4HHBvSreTkr36osAJe4glISYhkpAxMB3Gn79yej98ygmM4AijRqbsrUq3pLxFK0GBipGe2oY4sV1XZUXykIWlHuNj\\\\\\\/vEOgFcHRQzkmWHYWdjxicJE9s59vGnNEHQcMObWmEgcmKhuqzXfrBqt92IhBfXx3IX047jGLsyg+V\\\\\\\/hR05IA6M2PA6M3YZp2pknsEKF5aqVTirssMqEKp3fWS5+P4UPGZ1BF\\\\\\\/LbFN+4HM1sZ9Q6mKjVymWcYqk66QMzGkl7oxohzDiH0oSSU6q6g\\\\\\\/6y628PNu0ZkxhyTwTV7U6EIgBdGKeFGjmVn2LqwLSVc8RGhJnTY+swW2fVsSHY1FxUa7ZrhZqW4spKwo6FH6CgguDhs9\\\\\\\/xPAA0MijiM9jPaC60feeWecFSfXh\\\\\\\/gpuGxiBHiJjdF+kiXGn8E2eDyOw8Yu1wcyCi1OumoLVSKAE7qMHJTDsAVmK0L34fva1Uiv3nQaxxkRfYq9RP6KfE0OWeewpiDiehyUnPIa9VyYbbNgGJJQ4H0G289ooOB6pDES7reyJLDh7cquc6qVD+GCmVGS2MMaEDxd1zjfS6kVkKloZtZ85Qmi\\\\\\\/S\\\\\\\/ovOXQlgrbVhO9dpl3xpivhoDlkAmKpt+i\\\\\\\/uijVt3CBDoJ9TlYGXqpIaY9fnkd6vua+xA4N5Ibu8YmoLtKQiKGPQtnAFct6yei85iOw80b33jpKfplUBB\\\\\\\/cCvmlU+5LhGqBeF+3laBeXsZ1DNKp1y0ius1EXFQ5xBqTvWd+nggl3JvZUPT9UgKeD4BEIz+nZpuFsiPljUfPyJYJK0Fy6eLr\\\\\\\/usmNC2Pc8jA258\\\\\\\/nV69rt4uthAHaPFdw2RxDBc7M7hhejxBQUsojR1tgJEPRKnBwTEju0Qri6+2IVXGoAFiHJc+sOB3pNXIbhru+Sevg7aje2zqGKoEoGDvoKAU2gScWrdBP3F7KHDVowbdtONlsZS4RQ7m5h6UysWaMks7bnMxCbQ15rRCvMI0ljiW5WSiR7hEzF120T2u\\\\\\\/zgZjG6Q9XJMZ13hFmBorUc3IKxIxk790IfZKOS5ZLS6e1lbW58X\\\\\\\/ReD2o4SfCXeHrc2AOS4ipcqJjWeEy44GvJiu82oKCtn\\\\\\\/DWZqx0knG\\\\\\\/wM7g39LJZZNu0skAgI8Vi3zLLhm+8U8ijyGnkYsVzVbjRPhcsIMxy5AyaEqfD9jBFIh7F\\\\\\\/rdTAK30Dj28KvfqooEACn0KbWZoG+FkkGu7wf3Dg+JoHzH5sTIMzXZ6qT5Re1V2w+Lsy7dnIJtxAxDn9GI2fXAUIYNEwzJyblu8Jd0\\\\\\\/6wk15TyxieFnpDQfgCG6G+kogATJzg0fIoErCdPASYVhE8JXT\\\\\\\/Z\\\\\\\/hNrlwy+CI1HUkDUcK38D0NNp9dp2thQ4z7FtGzZzdPqMBYhLT1tGyP7G4cXKPwJvW2jL71jvoXWkhK7swuKInwKQ6tgb79RgouOQCXT0UZ7pbCB2gvTvK08Si8EqfkO\\\\\\\/awDVYYISChnKQ+3jCLVAJrW1Dr2s2hACYkawk3DCzsS9YKki\\\\\\\/YfMuQFhEnnDfQdEU+goCZghlthHu9FNDQAGJ+QfRBaOuq4qWzbOXHGvI9AkJQz4jWnVWEcWn\\\\\\\/F2wzefb5r+F6UVH5KckwMydbQ9+UvCT9iBBN89IQ1wcJ7fmyO2Ug3rDABCCasEJyqpPWiGD1RHhZ4Of0ouaZUNGKQdHSxTtbP49BOlxpOQds\\\\\\\/PoadY4CQAcyO9DwGCoFtqL607Hf6uVDICtxl1RnM079MhIT7xZ2nlZINFbO6iZSmdeKuuSodvLSV4HKuZeB0sgpoiYMiDS4RAzVKMbF0YRO6eu8GrzcVRU1RIJq8U\\\\\\\/P+qN07uh8oPBVL\\\\\\\/G98JAxQRsKvRHueMJsG+xOVVkXZSMhMAvtQYGzF4tXVK5YlaVC3oRIWV1T5STVXJJ055oXjWdVC1vEvy2vdMF52GlJaQ9XAyprZUNXKfiqmTaCFX93cNrb+8LLrha7J7kYUPBk8pg+V70Oq6eUnBX\\\\\\\/1JIAlJhbd0g5JTpVHnqzX81+d8hInZwQBSBOJAPDIBbHv5EpCbiRueEoJ0h1K2AXzjGkvmMLnO9gtxjyjPCg31t\\\\\\\/lm9V2k8DqrsNprJxPBNbpbUgsQEH\\\\\\\/i57EKskylLz1fwQtPik+X8rHdy2EJQnn95GTMqN3APOSmERxhRBh+G03+cuW2mxCEtQHE9btgjFhBbU5yu82P\\\\\\\/VDlrYLOSY2YZeteQ3PsiYHGc70jngIqdw3j03XIKWrJ8fmzXcx7O9VjJLLWTVsCetnffYYmUqlj8\\\\\\\/RgA\\\\\\\/kFpBG6xqMRQXCrNleY1OHUtf9t7Tw56Brz3wgsMvEh6SOlMzBawpLZz2R1K+Hu8A2SZsnf\\\\\\\/PEEHiun\\\\\\\/iHojlbOYJikGh81wTWpSZNo28tYfI9Sg32g0YJghfRUr+uoeGBXbO1TEy7Uifkh8+GZAdpev4GQ7ahP2qHAVcTQBUR1Qrar1Fq1Ro6nf9ZczByJi9SspGWjR6jz43qgrntz3PjJG6nuq81pJciTzl3yauUY00YNQmAR6RIy2iVVS2cyQ5vioEIqSeAaZR4J8q+wDbMiKwge8gHLNhUJ6GkJTYBjkSfDD9yjO1q949rPXsc\\\\\\\/eeox4K2DIByoImXM2b1\\\\\\\/FC21bRmw5Le4\\\\\\\/JK2ltMlyxwOQuRT\\\\\\\/pte8LBpK668lZdoQYLOKrQIBOsHMVNSIdHgTHrmSvh3JeuSXsHvmQhORD1uELfHFLxxA3KBuWdUs0q098y5fmCQ\\\\\\\/n6oaqWdEKfs7v8uVpvIxQOF110f1KwUH+4v9QMp51tjOc+8BKnZ1kS+HSuwwfYp8KbKTwkHWmsJs6SmWdY5h8gnLpQ238\\\\\\\/f9LGzohQtxbh4Wd04I\\\\\\\/Q5UkRP1+De54sSv22QOGXxIt9EQyVn1XsWn4bq2sitlSSLMRskaV+DGZe2sjcVUa0B24uHuq8MkO5J9h8U1JcrvHcjnzahHMrh2LEdSx77CwBgG1Vs3pwBIxz5gyCEHsB43IFSTmoo7UD6otisMX354gR57EdCaaMRcYa2X140TGIdUqz5OFpS7RHTwwHJthGhiE2aexmMtTilAMwEhgTq4FHgYCkewleOWNPAGFMxrijD+dC2XPRDo03eiVzfU42mAM9JR\\\\\\\/8oVrHturKX0GB8klLJGYrRsTBw8iyyRyeK1+IlAm5up8MTC+Z\\\\\\\/N3Vy0U+84DVN159HqvoWDsMzF5L3Gcfyi5X\\\\\\\/vo1AbCqCsIppEFYgEH2Lx8P+cyYpnoepzZ142podwVDviCGl1o3CslA+P63kbS8BSkbyUAY\\\\\\\/aY5Gg5apT+cUcWwl7BJCSI3JGlFZjc+MMfVy8jiFiiuQv2U65gJT+lRGdaiEGjv9RyWT7Et5jc+6pfloCmUNJ8FrOEz9K1rZ+VXo6Ete8UtnfNNa\\\\\\\/7I\\\\\\\/tVqnIrN0HVrKqTqLas0DTgEdCTydANOHZQU81WPZBiui9OMr4LFFP\\\\\\\/DjfAOE0g03nOErP2f\\\\\\\/moEpQqW8lg0FoyrRhjIS9Z8pjU4zGnWSqlAgUXx0dMceS\\\\\\\/6mL4RATLiZc4bVaXtAj7J4AeJB4vM\\\\\\\/JKYrTVyJko+jYvo4FDC7UwYYXkc6tBC+HgJQXVUCU0IMkHffY\\\\\\\/Ub3V4v1SaxGM9hEKj03Uj6me+HMVXvBbMkq6w9reVm66vRi2T6wHjNSMIB9i1aEBLrbp39bNXXEq7RsOmIk+VOVTJf16vBOorv21EfKQZ5OlXUYxqpFwYn8vuZ4E\\\\\\\/FM21NeUnXJk6ZvHzRqwTnEiDka7uazpQrSt2NiWQBTfkxqyVkl94okR0U+anecmnWvFp8G+sG1sWKcMJnRtCCaCTTs+gb6Gi2JQ\\\\\\\/pJmPzepHAsmksEH9x4U5Xtcj2fJHRCK999R4jkFG8TethWtozX9HlgAdy7AwKgOv0liHbtMqS7xxQUD0QXWsGpWiHg0EAnBp+tUrHAGN9U6bislfT0FlHS6DFGrsnb8lUxuNm50M\\\\\\\/dWjx+s99py8wSqBlqq1aJW5TG3eyLNSdEMl4o0tHp+bb3KyEINQpnlI4bZJepJ7Hidm14mo2aGY7choa1QGyVNKFpS9pCHC\\\\\\\/jdRsy8DPf1Wv6gxzlPInWoRZm+LYNt00OklPMfiSGEp3LpGevJBFJwkT1KxDliizpm6jVU+t\\\\\\\/WjX0fyeqUq8mWQ9QKthwjxL9yD9BykOfz0Jt6Y9LuZNavcCvZQLHWR1775BgNKf8NkQax+UugAfDOj0hc7r2bnhKOem8GLj+Azxvgu0qiJ2rq\\\\\\\/TVjZyovnhVlz+yqQh+5c\\\\\\\/vCR6zy0W5n5PrJYz72vi+98L6hMSLRpz8HyTPHkDYtrgsGZLd9H74Ivmk1FDeHxNxIaC9ykN+t1ThjFgpGwHytPX4bS7geqROmjOgOzNHYdH5t12FXJx8FFGABIaBIxEJgRvelIgi0AWvIy52nirqSD5Inyp0dfwpqrJZ0jWvWfEIGhc0O7cC9xuXuJJX1xWPyUyNIj4IaTTkwWEFCpOihNEhGR0quvehEPwEflb1QR6Wf2MvDr+JOmr89RSAjG6mNPKS2Inx07MP\\\\\\\/Ifs5VlqXNrdKDFDjjdTtZgEjgFx1LHVIZO3hWv2xgBJ37+F4UAlTsfm7IUhPaz5LwPW77K7izuBpo\\\\\\\/XShYzG6wNJvuDzenViYau2mjPPqSoMQ3MsLQCxHwIi7WGcjwM9a5H6+kqd5R\\\\\\\/W1BhK8T2N6WPzZV5prE0usFAbFcogneqCdTf0Epn8flhBAcFnEm6avWNDbMMJLbIdXY0CISaTJl0cngM14nOekfrx7XIgaJavW33ehzyXcrpde\\\\\\\/AlHp+HCgIpH6bMgVQipZ3TIxJcRRbmBNtszJd7Tj\\\\\\\/AOzSViOaZCxqSCLU+70CgrMb+qy8NOsMeCYOIRodNUfBHMZuNBshxk2wUmfWb27wThPlehv68T0zQUyeUqJhBkig39bYbvo9q6HYR0Eb5UtcE\\\\\\\/8z9zI2UiUY9Vm3nQz30EiB65omgaYDL65FRLH\\\\\\\/JZzoX7hZ8Xf3Fyoh77t0\\\\\\\/tQhgvQBYj3nYbX7bHXHorBuPMwmwaJGDzZoOhhQcYV4QPxksuNCQUADrpXJOAGNEgogSmWm77CgddeXPpCDcuuuvCv1d7tiDArgvTCrPPz1rsWtYflNlSwVccV3t7fe261gG768Xb2N\\\\\\\/wmw3eRPYUBY8wamIuwzN11HQlSO\\\\\\\/VPqXGKSOhtHZgnMU2ca2+Q2JYGkrJcQT1J6S2B86XCnW6rKN2diA\\\\\\\/156JpOTYY\\\\\\\/fJN4LEs994eytkQJb9HLqh7LH7zL0x5kDtYTaMpyB3KGDphr6WT\\\\\\\/eiUeLfRjUSKkCLQkFtB\\\\\\\/o77DVeFDRpRcs2xwvXtfiIdwq73qOsXLYrwSc7c\\\\\\\/SesmBXf+crs3klpRoiFWcfPRCOH4+7\\\\\\\/rv5\\\\\\\/xhq\\\\\\\/T547JzNeHzuNAteN8H2CbhZN4QIZ5Z5TvFPZOIaORixdQF8tfO+VJ3PWl\\\\\\\/XFs1Mb0nBTPav6nlTQJY3HJ++iVb7e7AsVqMTQ4BxJ3OHhlnVBsIqURhvZobZrwY2yRbUUSYkKb0\\\\\\\/BPpVDz4CxXJnqWw3jWZvr+k\\\\\\\/\\\\\\\/+u7+XUPiRPKY67Bcw2FG7YSdE0Nm\\\\\\\/iE5niiXKmNZpVyDAHe0w1USrvYI9Gd8VC9pANAXp3v7FHWb\\\\\\\/hOnmQ+cHeQ3EXit+GnGY+2vCdIbLUBB5S\\\\\\\/zIqlxCmaDKdsmxt8Q2Q6pHLYOaar6G06bJr\\\\\\\/ZRly\\\\\\\/koUOBwosY8KvgQ2zTUtBwY9RuyfPVZ2\\\\\\\/V+glgqy+r02S6+0LwKq7FkTcazEmvMBGtH0SDp2z91xhMChPIkXw82lJU57pJYafgn9KEI3gUm8TlEQs4oUHAWXzXQ8rKOX3h4xu1eyBqdYLjVGhWhtOPT\\\\\\\/yamxCmsKNhTpMwE7UDz9kc111ut6l2pxZGTko6RR2jJZf5nNEDXZl19RVkNDfw6f+pQvdm8+lB+ydkK+dFw1kLa\\\\\\\/VMFC\\\\\\\/1ABIJ6Fz+htB3sp9CMCFKByJJxnNxivc0JjHap8+TbsadOvwcYHRIHwALfyJmgAuk657yhEuU9xmYwKyZuDdnV3WNhZ5o5PHzDid\\\\\\\/s91alYlOM+TZxpvB\\\\\\\/cq2hglNb4c7sM65q0wDKY3z7fQg+kqsTog+tdlRTjwRXnoDxLqEJwrrY7ej6Xj2nW0XCw1tZkwj2lhRkDqLFdyHi4OoS180gtZ54yQ9nQ40aw1T7UDkK51Kk\\\\\\\/DhzGMEt5YC3B8pY5TJTlXqzsX2ZRJWxAj+TuQgdIdEQuNVRp7d0nPlwEfAwhZRC5JF\\\\\\\/Lypwjm\\\\\\\/BHMaJ1Rv9kk35vgCvt4GS0CYJSKQ\\\\\\\/l6WoxmBAt3ATKJNQgMsxkbnamGTLHGldqKBiKLSwZ2NyvBNPvYYJyqQ8WxdGhc\\\\\\\/X8JfxkhDssIVBoWo9uEO4oXn5XKFTDnSXcaIpJuUmE6Q9lHYLtlv\\\\\\\/0dIJEc6vc92G0Xzduy56c7LmIUxB30dkpKtuyf60TVcEJewmXfl96w\\\\\\\/K21PTgXxjKrE9xy7Z9db9mU2f9O4NNCSaMi2QBjt4GmOuage4bUJxttwiY5V61uoy6cVFJHrpg\\\\\\\/WbGX5BSgZrek7Y5kHBIX2gXvltSQSY8r0wPq8NwBnN\\\\\\\/82xx+tpcTuBlZ6LIuibwR0V2TfUQgRjayJU1tHOhJ2UzbPjLL4H5x8NCR752QGOrfagl2YwMod55oNlxGVaCtGbmpYqGbeNNZPjMXrqUKBSYsV986NAOeGC6Hq8mN1LxhNdyPll6l8cQZLlwrBzVQrzhCu5OAs6Uf15hfKvCNVQZDGF3gHMUhwPf0Jol1F1Cm8QGKo9SqKnEYfIYMOUGoT1LrThT9SsmwO3BvTvlg4mVUbe+5hvF1Hchpuhn2j7Rgyk72xYx05uLNAl48FQqVaS6M0VuKyIdu0YjEMiWAbPwhE2dRqeF\\\\\\\/ZCVodL5OoGt2TldfQbTVpzfE0qwQ+Eo\\\\\\\/RqXG8ivENYvcDwltsJkJRc7BYU2wNGHnbKnTXPm\\\\\\\/\\\\\\\/AN1GHc5wvOHuJcCEqL9\\\\\\\/ph4cBmZGewxSiAY9WY5Y5LJmGUgwM6kxxewIDJ4usOFUh2pjpRr2P9K9bS5CMyPMptRIqM53aEUQHHwAFOCzRbYbmp87+6Ms9Yaogcbcmoc+9nIPmUXP5+7sRleWjyF5zhOmaCNK9esTv0aswWj5aHqsFndZOC07BtGEY+0MEXATmLzcJm521Ee5DBEiEH8EcCAhhXb98U0S4o+FfEWwtlCp1FTHxnQ3WfQRUp2iiI59Y8A1ckcfn7eV9LXTPLamBg2shJWZiGALuHvWUUDYAblpSdrlqYjr0K6h9xp+dUFEAsLCZ52ubH6hHBrcZztwxAib5nrFq36Bmt+iSLAUwR04OpS23LbgkkW9kJ1EdEEpI5ZDW9rpbfnNpU8M0qpWGVS9hcMGxKfoowk4oQHj3rTgdR\\\\\\\/+3AsCp3c6jlUBdTd\\\\\\\/gY2DZiLfqnVZRMq5JL7DjZscMoPkV8hankr\\\\\\\/5\\\\\\\/XVdTM2xFCDp8vpS5gESiNCWR86fMTMZY06paoswGFSJ+fNXMK0M+EMJ+rJWanUl4mxrAUPDrBl8NuBOIjS2Istth0OY29W\\\\\\\/ed+lRyw3TaOYeapMDAVh7Y08\\\\\\\/fB5fyweOfSjy7xbn+hBYdLeutCrKeok9hr69NsFurY8KYUXTz1dcMHNRfRg7HV97NLyfr3ee3RGxTOjB9Owc7wUyf6wRZVlDAWUQ22+XQrb49IO8w0gIgxRcC0EH05vBG\\\\\\\/MN6kjVgPMgTsCl6D31nIXBKLN375VgcrzlnbK+v7hx1KMvl35QK9Vn+kxqUTnWbm6lPy0foJTJ0kWk2jF9uHplGCLBNc1o\\\\\\\/JrxoDaJwiBSH+7XXihKD+u+J7YVGh236a1N3qul2JyrvqTjXg0y1Q17IOXBTbz+6GMMREXdMSk2xqDBN6PqN2Y7NMUJrZ0CPDM3Yfh4L5RSdjnSSc7U3yxx0psTTrG7bbSSxmF6vpydWi4CN1Em5XMAk1HfPk4vO32bf1mw+xAGshxXFzUH2GjpCsc4LyCsmbG39f4bhRQe4gfVLWVm+2rPdGek4CY\\\\\\\/LlhT52wjF3oxtICC8Xx9Ye3PhL7EHEw4WtmoYNEFkBNl+Uh9csFlUpCm+cvJyN2OffzVr3n16xMzcw8vks02PscDG43zzSOtfqqpwtfqI8+WEEkAPCos2c0b92yNdOgiJVKpCt86ppsz6nrrcTt3KPlnVWpjcvUeKFha26sKk1OpmOGj2DcC3aWpaQVqxAURsnEgd8\\\\\\\/GTP0HEI2gjwqJE8rq3BC1EN6Vv2EWCDyf7Ehzp557\\\\\\\/8FZIxgJ1T6GFL6mBddBAGhYdeTLuyfieKl8oYXDvwlZwomeBbWVFyidXEEd57zcaCfQG0sU00N0XSRIZSKPuej7Wx2XbHVYFnB3ppfPswLGKBI2vN3zm8dbBUI8ddtES9FH4KAEEb\\\\\\\/5QKFRuyvpZzBKG4lpzEkZ5fiAc\\\\\\\/ypjWvMndYrnVVv365ZD0OUcB+H\\\\\\\/C9b\\\\\\\/9QKNjgaTc9+BQtHmSTZJhrCmE7zBMxVlhbDMbS9VroFwU5RZOKNyxUOaOlX79KdJIdYDmHOSN6KwQtT2CLKwgCO51MvRDC\\\\\\\/2InLpY\\\\\\\/K3HfGfj9\\\\\\\/lCgA3753VwkC8JT+mcxVcCR6t7fK1CRCQPvy+XeYoB7686\\\\\\\/KieXQE1fvjnZcqVNSfPl18SdidxkUcMgYdcMc8uS\\\\\\\/YyGuX4Sl3g\\\\\\\/66BPBvYzU1rmXhb5fZMhBNZaZrS5tahMTi0FGDRToXS5B73LaG29f19MGoPulMuLUGWhGhDQTFoj\\\\\\\/VlwYGcU46g+rDEHO6bG5yr9fQoSq+sSNOl439qsSwNtqzHAUlD5gVOd0se73+Y\\\\\\\/EiTIXXQ1RZYPXefMtWoJKLi4I0N7uQBmJ9f0ctAlUvHyM9i5l8AJ5R1ISURkTh4PbBPQeSB2rGgg2UZjdbhhBiLOPmW9yKI5mEwoVFmmhRvQp0TVjsiESxMat8Q6nmA0NJlnlBpB0i77T\\\\\\\/ZWN0Np8h+Y7FGzEU5b0HvgmRQsG3P3thuoZE5uQXJAL3GoxMHSLjRxOfvXXfjFEbc6DOAtWvgmM1WmWvHQxVvLCImWZZnx1kGlvNByX9yGKvoqwAvHzripqGJuFm7yld+RbY2p8x\\\\\\\/sM46MaO4WxtvkUr+tqZwKue2XMZkq4xjijqFvwvcH\\\\\\\/jBuZ8yu6QXVnH3zYJPGUfdsXCjT5sW7Zend4JjDNO4IlwzHoT4C1FZoTQ3U1PaAeQsJxbBsdTCJbBEw36O5irv5J6Qp8PAfuDCRY9tBfIheWY23qAE63n9lfDLGS6iHdC1tZsmCEKOM0rHYOxmr2lH2LSS\\\\\\\/fOeukHKENUV4LxmpdGkLmwl18xHplwgNnzc1EG3wegFoy6b+O6oxCyL7SkfcCuUX1+1gGZLFHVxsfjT06mJmMCrkBhVddGwCHz+Xp3fGQe6tuWkdy6hu\\\\\\\/DB4nRDPIGqRQ6S1fk1tYUF69RcdP+r8MBpjE4uF\\\\\\\/EdShoev+p3Uu8SodVivW0iYqd2V5nMOhvWaPXqu3Uj+\\\\\\\/IwbY21UC3hgRDutJmGUNXwErLqYdXAj3PKvvqOkCUjnPGTMxrRM8h8i0yV44BC0gTDJzEdzvjrz95w24vuof8mh+l+OrtkjIHrb2uMDhABkL\\\\\\\/uE2h2z4F9wqIDoi4NNmEIpmKqV8yHxnK5vrlW8oxTaX9ZH5BjMq7jC53A6LVXHqHGEfeeXx\\\\\\\/NjejH2Fz1JEbidmsX6NRS7iWmIfkVIY0r0FUGKDwzGvfOtY0V9OQWPIw2pVA7xeY5v0ky8jhiwbdnDp9jBTZZeqJnq3yWFV5FFWhjMVaxUByXwJdSTVNd8GoAvBXgvOXLdswhy7GVi\\\\\\\/eaOa3CQXL39ld5rS2WwiwE54m1cJZmQaSTb+hHlORtftG8WtRlSnh+Fo9lysKufke76\\\\\\\/GqsJ+DB632Dv9EdRjjeCSWAnxBDQZrCirGynM8ckM9sz0dX\\\\\\\/C5czPb7WtIxmCvUo5MfvtXyTljXvb9Ij5XazBJo7xbRTnorRz7wNt\\\\\\\/KufkEsZHGC1uAA7D4XFjImaaM3fgy+CiV5dGGo99kuTFtzM91NIm\\\\\\\/\\\\\\\/GgVXWFsUlxYAqcGSMPReA3LzwkBceDckmFaZDLEpnFAe+IWDoEnIwhCG17yEQirEWaH2m5jg\\\\\\\/UcRacQeTe5Pm9Gb2je3znheIC3L\\\\\\\/nO9qhBOFhOS9UwxB0w4raCWceSpYq4sQVIdIjX5vZhN0uqT6B\\\\\\\/F7ejyZuCcZRlrLTpVdWiseAPYAHxCW\\\\\\\/A3G0z\\\\\\\/+nb34c+3tIc92xoMErY0H9sGEmN6BMl7i\\\\\\\/yvd8PhoKKFzC7IpqfdKT1Hfoz\\\\\\\/mR7gyWixemMNXwB1vO2dPR5+CgRSlAdhuh5Zmorw+kxF0wKJGw8JNMbKlVKfews0Ngotv7h43ne3jbbrZXIQmz\\\\\\\/XTtfzaP2h3tn36TkkSGeZ308YYOuqDrwq4SKkWzBd+h8iN\\\\\\\/Q2lDHJ0jn0HVW1LyVeGu5JSJpch\\\\\\\/zAls+XHILdy5H2l2WDG6KtXz11Y3+1xo8qLMr4cfnPZV5mB1WRD\\\\\\\/ntvK3\\\\\\\/h2VYFsx6bYfgg5y1JUC5duEZVQ1bc7Wqq3cIlV9XaMqbrWM\\\\\\\/v50fQ+1d5OusC+VQ9iUc6kg4U6cWOPrghkUCWGYCufDfLc2dGpoV686joH6CV7zwRimEghREIQhCcComsDntaYLCLBICenVdOofSRAmgIDmgoGFFTeMIeQfOgHiFhnTXD7\\\\\\\/qvPwxTFHO3CxJS0\\\\\\\/IuT5wl0SE624EIJajVsDJAvjBu2LdkyhuLDuUjFt1ASz78D1szVrfQW\\\\\\\/uCYEldRadrKUjwvj6itQxIhcatT4M9H6Z8QWZ+bUrZJuTMV\\\\\\\/iuqgar3cwj1D\\\\\\\/9fHlztawmuu+lLFdrQRmD8oIb5vfAZYNDqKpWM2CKWz4XM1sqe9J7jUVPMjpHFUUCu7\\\\\\\/UmX3PP5oJcbMz8cBIx1wakzWVy7DbpCOcMDfYh38HcPEunM5MnVznZgNoKKCU9bDadOH9NldJRErLh\\\\\\\/55jftwlxRyFLFSvh3irHsjSLmldq2Xm4K9IQmScECp3T8lRZRUs+6SwHSKZVTdOvP1z2b9Oizl1eNdQP7sCupp3C1MH5lB8Iar69spMe36Q6\\\\\\\/6FcpMjrEZEhtENV6nSGNFRwz6XAGMNuOBYvUC33UpCRRZ1FdvjqfKaVaeN7IisztT1WqWtgzEIwQg63Z9KrgDyOlKA8HMOsSjpL7JbIMj2fRMK9TKRqgTrJCFQU7rQxS2HmBj\\\\\\\/OynRyKydrych4VaEXb+LlaT8ysgdcBIeYO0VWjf4jHSd5RisiKAzbZ\\\\\\\/2YkZIAy7J\\\\\\\/6eZ1gx4YIRsG426gb+2gbI7GPjHUQJJKux0K88Ryr30VVzowirdHnsEL+lgYUqC095FQhdkRqeVsjYDbm5ptQFYbI46IMxsac5BOk5l7LtRtITEz59xdaZhE4r4YTQIZCl6AWyvKFHDWs4DkCNz41U7idIUsOpLxdLsMxOqJp1KGEqq6BMSUBysTuOEXETJrN\\\\\\\/n1WR8zk4ou0DDUm9+Z5vnj896lvNvknddjWt+KVBtDM5pPKK4GYGC8RrCjgX0doYs\\\\\\\/ssRPXVc4gytkjRp+b6eZOy7uCwDgOl7WNDbK0Mq7KmFNRFgjupIkonm2TlA7a\\\\\\\/R4BryPt0YI9+Ah4K9NnCAegxRUux9yEsHdD3ifXqBiLAzPVFhuSqmEvXjIRQGU6Lgk8+4iZrOOGepKsR9UW2yT+soeuo8KJt6Zlr6TKf4RPPcvEkart5eUUktMeL+8ZXaSk1cmAkF2mCrG2tsCsRnr+NML\\\\\\\/xGFm0qXmHrtI\\\\\\\/mT+2BAiznYPfoFeJnFtg1WZ6a5zgU8rVZhFOCFRxpvFxOxmS7loFE7ujC1OfNvmNdl+Wq\\\\\\\/WMMHkDAi2ZKKO2zQyrf2ur1v3TV\\\\\\\/eL0l839qQyqXxM+GhdwvbtWW01YIfFsgbMoGl4K0+d\\\\\\\/RdMtGzlzre0s8wPy7GAreL1CzZdgnQ\\\\\\\/HxxZpHP6wgrmUgqLchipuF87L0XEIkumW9hKsbIEVJ3DU8SNuidp0SIzYACQyUKiB3VCj0nzY2EGc36ksxAoJOd9dHDO\\\\\\\/OY8xRzeExdb+k8gkZQ\\\\\\\/\\\\\\\/kTeCYQFrsc\\\\\\\/53FBqxikXr4OAcarSVDY7BqhHuHIn\\\\\\\/mNE+KFefKXLLYOxHgNy2omK6ietnrXzeQohf12M+D6+gI0F8yzEtXrViLp8WNg9MAEXNhhcc9gCaO+OXpTHKe3LwavHZ1\\\\\\\/7uAT3YExvVXBwwzvDxEiX0rHKRsCosf3kuMkPW3F\\\\\\\/3cXPT22pk6Xxnam3oghdrrsjvsROcuVBcpCcQr+SivvflWYOqvYbUvSuNh+BNt9+\\\\\\\/VPVtjJFIdwhGsV3siKUToy1vwc5dMFN96dTvMqtwYg\\\\\\\/DrbUaFaRzU8HPGS7meTJMYtGyHhzFa\\\\\\\/Cp8q11rX6\\\\\\\/VNRBlDyC29r+elRcv2SXoiRKS0+tXwlxLHd6ckRScLQU\\\\\\\/BOKy37MHGEwG267KlR7Q6pV7xxH2J01Nf4IFK+NJMUy4L+\\\\\\\/CAYN29UiB0d2NC7mHgzWBvecNMhjya3yJENnDVpa2tXQBR6783i6JQdrbrTB5Z57VOuG2l8FMrvq45X2SXhJHg9AlLDLAYxuEg4e2hEe21vb1d8Ys8qn3LJ9SreCtcn8qqasxbRbnefM3weooJ0X34eB804q7pKhUy11npbI9O\\\\\\\/A82BkD2SaNShvt7gqdg1YbiOwLq51Wb7HlN8wLlYl8l7XxtJHB2X9HJxBjzdGenloZD0ToUMRFU75l2WssbaijDGIupL9EGMsysdBdthNrTwuh+uma0DQm9Kp2i+s5kga48qBMRobz7kdUNk55v+u04sZhidhgkMWe4TXtK423V9K+cqMI+\\\\\\\/iaO\\\\\\\/5hJI6S7hQg1kaGjUReLYjuTXLsMmR2b03Y8fs21mO\\\\\\\/BKLtOMw0paMJU3QVUGP4i1nFDhEH7BXgq+HU30+cEEv0TNa4jS6A8ELe7jVjtgmBUjtc5uRFXZyCwfscSiNmfk1ADAIzknIVF\\\\\\\/6GWzI8936xbmS4IY0iF4gwAwHYPWK7L5IUlGQ4l6yFevn5Hpyv+C14EoGlr\\\\\\\/7zd2LD6fG7R1BPTYMPW+vC2TumZg0zBtfktbRDCbmtciZc6sfJF1Se\\\\\\\/qqyoB9mMo4mxwjDIwUES4FPPvKD0yH0Q1bcyim2Obc7+ZIVr1aAgRK2vZfYRDB54S7c9lQq6B5G7hDAhnTE2rt7\\\\\\\/mrVV09Gi1ucK6TD2b6aqHCAOXC5pGEsie3E3j14CNqlAGj1B2QV9PLz2hQBXUrfMLNcP0GJGZnBi1W35GKADrt7AO6N2K54YGQLT7kjAaGx5CO9DPwAMJ05jRw8t1MZwNxoe+eDEvPHMqUqff+Irf9k4pt+FiJdOfUyePXVQLGWGfoUWHaXTrSjRAlLntgbq0oJXLw4ZolAiAGJraZ0JR24H1HKdZpDXlvRPFLnUdaEfeRGpWQuvML+FjNKkxHWAP+xqIRGw4of\\\\\\\/\\\\\\\/AnAXAlxhErsC7u\\\\\\\/HpG0SNUOq8ejCWzm7zUoIN1sYGkGP\\\\\\\/hHKXa999clN5miMHJkmlWIiFk0RHPPjCqm8TkK8+VnXirearUtJjwuMDUn\\\\\\\/KuJuDgs+E+mSp3Tvtutf\\\\\\\/k+4CLFmDZAVRGNMVU\\\\\\\/E+V0WL+TRNWS8sHeH+5H7fD\\\\\\\/t8CY4tg5SXn5EwqjlhL33tdWlhbEUhLCVFgGgPnIM\\\\\\\/PmNDaOeJbyvNMXvbRMHWjSxd3K5FNE+SKJybSzWKZQYfV3HFqAuXFVzuBFbmn058NYan\\\\\\\/qIYx4Gv6ebJUrE+MDDoM8CFjXjeD+KvdYvsxSXKV82jISUoprT1iPREzMmUAQfIlS\\\\\\\/gYTV1PA+LMfYFmSPRsozlocRGIYIa+Msz48r4RHfp5fmeVAynejFh01xJMiko4hgy7VmuzezcGRqzgFUuUpBdIjlzGBo3Cz5hBWWrNn01MLiUBdfC9vTKgK3uaNUs+Qt6EdktmjDA4JjlB7Z9qFF8zuY\\\\\\\/jf5LOjO27V9Zd\\\\\\\/R9w74h2zoS+ulTTzhhgl5GjYWOjRmGd32nZ+QxlOwAjx3fHVRIRY1iJIYrZJ2eLjXRlJyKUYCX9alPqRyUOkJeMbidBa8rYWOPo8P0LgpoKlXhfjCoFmlncPfD8iHmUnaXxuXr8\\\\\\\/McSMHkCxmXpZdGM7sHlE1aZpF5SMiUbpJZRDiUNILKR84fkmcKTm9F\\\\\\\/WE2he1+Uysi46ocjRpBgERsSYsS9UmmnCrnC5dEiB142vqdOKsMlxSj4lwunxKoRjmsB7ESfRvJYne+2pwlxr1yJBZKJ+UQ9q9TD5jYJDADO1sppuf\\\\\\\/YK1mymAeAu+oSJCz\\\\\\\/bHHbd\\\\\\\/vlohNjbeCsNBrktK+jB2zBBOLFJ+21caVkyD\\\\\\\/eLZpZZQTYsRfyi3iNcx5NEMhTSYeT5+0P\\\\\\\/BnSESh2wG1Dgjfv5YvAMqmyM2MncY\\\\\\\/tGkYg9bhp9f26zkqY0XSj26FgfMI3aDN7cGsLrk44qnke+6ROKUaieAIsHzCHck9L8vzmM9DjTnIidPQO1rEsyPtF\\\\\\\/fhuQ+y9LIW2u5JLbtBO+WEudLQVXMt+Fl3SS3XV9QynRj0JtptiGF4\\\\\\\/k\\\\\\\/D3HgNJI57skcd+ncgsDXxE1kobk6ZKOkQSw0PXYMWv\\\\\\\/ezmFKf01BqzHElpUl4kugEYXwImKJIsL4wt3Xfhlk6Mn51HwZh95Xf+qZuRRNvzpoRwbnuPn5ladF4q\\\\\\\/bSmXwW4ijK4sjqbZcKIcDd3XDyCO1fpu9i+69UPhdd0Qt0Ut\\\\\\\/8bWhrRb7Xfp6JN2\\\\\\\/9wNPSnEL9kLw4XQFnW5B8GIRF8T6rnJ6r2OpwxfRYPUHUXccJWINPPVSZUOp6AwdPg7EDtqX\\\\\\\/aEWIqPzPlhXKNxWjLS4M77r3aVf+PeNydKB79QVtalIpOaF495Y9lwzRkYGSXAi6kUShkDXcIjo0j+t0Xj+4fW8h\\\\\\\/F6saAA1lvuqYqOy5+We65BAF3AUND0kRahEggIeHhE++\\\\\\\/3a7GZAwo0uIcdZE2\\\\\\\/bxiFwBDxX83l1OEi33NPjqf+AvzltcVmRWXiO7hKdqcJ9prT645Gq7EIS5yLTv6yDy7Eo5TfApEHZSI\\\\\\\/nD5lX4uczJFRfNTba1C3mlJyNAhLoUrIBYriBTE0WrTBcwqJslMiL1Dnta2QZvMxSyg\\\\\\\/umNQogIpP+M0OP3feivhHvGdGSzSI2JwNHHGKR5h\\\\\\\/akmy9QKXxNrY3SwXZV3RNlMpYjRmEC57nXyDZQcvU0xZovfXfurTktfIOrP70\\\\\\\/Ddzh\\\\\\\/u2IrsnW5Xoy076722SxneJNMzZyKconW1q4ypN1lZkbd19CVM\\\\\\\/6x6AX7duR9KV9MGFWS3LTg\\\\\\\/W1GMCvurNJfqKzfPt27TWbrWgYYMHncEv93VqzkEsrjt2rrTvGP4mVsCDBs6wSXGF7GJetir2BfOUwxBd5oWr+nbAvejimIGkBlu7guDfpFxC0c2+HtQ6fXQfweZBD1QjS6pH7RMOn5rBcEUH+HVrfWcIHY51KCWISZ+W1hUY1Hzs2pn76t8wfDkq1Qb32DC+8MfnV5hqv9GMp6La4woFWjIrLkZGH2VR2j\\\\\\\/Yq8V7DJ0Qbtr+FZANy\\\\\\\/8fadz2XqhoyuZrggsYQOGL4Ix6FA6hqB4trjLQJ9Znl\\\\\\\/GdbQbORkFOyt\\\\\\\/tSnFiQj8dvQTfyCaYfraOtMcKFd4ZXa8QZGzIfFakCNn4WSHCIUu9TRtR+JHhK61XsxPeMCdnfouK73CLegTl+XzsuIQhX0v+fynbmQjt4qvDdGr+cEValQsHa3ciKnsqSSJfrSGjFs84jkd1rxRyHwGg4PrX2OUhw0\\\\\\\/za3yHnGsuD6Au7ZEhJhSojd\\\\\\\/eZZ45v6GoD2C6k0s\\\\\\\/JZ7HhNIu9sGC8D\\\\\\\/dSv2KNNK2TW3h5tL6ieSWI6j3KJdrNRXfOLOC9\\\\\\\/GE+IoTawW2MLF7ZkaqxdNd43MToSkLwOoVi0\\\\\\\/fNwDbWqgM3iqb3hwL4nPBOGnGooAHSFXpJ1u\\\\\\\/3qpF6HL6e38sfWQ8cOFON942jva6MIFFXRNujP0X7XyENgNAlWGGbkkOdsS\\\\\\\/L31WJS2JckX0NtPIUSBZZlHDcpoVlu+s66aOorY2IQpWGpRKncSeTipVfvsuhfumtk079yf6OnCT3Kc4Map9dT7OBkBRvXViHz+ewhRM6CpvDr\\\\\\\/g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022NtlrHQZi3C8hlxnRMXvXnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229dc7b80f208704db7075150456cb5d6f5768dae4e3cee0f4a82e683610920c8f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uN9qibSrrI+X0gj9JXEpGZohN4TchObs1amtFenqQyNumnblFHKotBd7Nsu5e0ryxzgUvpYqDh0HA6MoV8OfaSPOB8YtVCN0Ws5xWVOgpZfcJYFIYJAcs4Yu0Z0Xxwb4UC+cmjhpMA9bzFonUVYE7UeksMcq3fbfE+cTxKNZR1pbvNhq1Q1cywcttndQRe4uIs2sFxyR15qggDoEAYtXvxR0h2ayH1+yWoiFEJ7l\\\\\\\/coing\\\\\\\/8DER\\\\\\\/4lIYgK7iEELU+clDpfpZcfLF775n+pHP8bVKjen7+35bk3KFEwkHJEx63HZ5Z2K6CjmVlc3vIYhfeyyg8f656fB9s9wmpJzHvbRze4ZIPCazQRus+i+3ICV7IQPnmsjsEO3Cz7lpAJymBI8xiGURZHjcdumtNFr2mjzafngJR4z86pWWhovpqjb1gPxn8JgySNll8NDtQpv006kmZN7DSUsxUYHVzfyyKTfKFvXFfST8uTCjP6mzyLEhPKs1EN75sFDVkUBBxP9uewMp2I\\\\\\\/MYtoeDZZl\\\\\\\/cgm9zXb+n9pEopWsnFFWg\\\\\\\/IRIRl2EtDlL1gf+MOqbk3PfI+5xNa2CIiOp48HrovxWaMgwWPyCRvXHfgb4\\\\\\\/tEqnt1WkzQJapCMCPbmYS4xYdyZmEQHU6uTuUrn26KrUYbCVPJNiMBkGaG\\\\\\\/r494i0MODufqbt00d+ZzMIhPU+ctnLCiuPPZdImN6JF2wriU\\\\\\\/mGngAz1n+75NcbceJ+12JZ0uC9bBWFK4o2sSmJPC3Ze+B4IEoWXlReyMAa2pzIusHwPsRJRuz3KQxCX6aoYYso+1OWuKYDtNcknn9Xcow\\\\\\\/r4HfeCCed2iZQ98WqUb6dTw5FCOXjY27uv9PDYpev8CT1e2nMFY6Emw\\\\\\\/Lk3R3eAd5PG4l32jD4B6JpqYVjthP\\\\\\\/kf0EQZrzTLlwB31ET3G\\\\\\\/Tkqh283nYKIvxnermEvK7iFLtUAJ5Cdzk94jrCO2KHpDP+6tteY2HhuEvNLkHkR+fed8Juepq2w3IGBuzqusNnw3+\\\\\\\/DIcMxgubNlmaQdtIYtXXStYK2ELuB24b0+F4n6odlqthR2mdIE5DkOSu\\\\\\\/S5wTJRslvWRu4pQ9U4h9SWDm\\\\\\\/3DDt62m8UXPwN8L5mQaYdB0Hzroa2m5nRZAfycJuvgL2vwcfHMM\\\\\\\/\\\\\\\/XjRivAe00twxVcEc0cnSJdkcPSGARMzevzXmszldi6WQ45GZgAFj+r\\\\\\\/Nond6X6d5ozhYkmARh9vkJrIzzktSkyJx69dfoFuyXG0D9iHpshOH+sVCSfRqmAPqTfw\\\\\\\/aRq0uzaAUD0BMZsiSdveUsC0VkiekE2JsZshDTmTmhUgHYtw+J37rZUUlIqxN5Kret3YewyRGGWx2Y0bzV4IQulBbV4H3ImhUksogRxstdZ9nFjrBfVp04iPaqq5M5sLuH8S9cQlGo4HEgphbEvwMRCyPWVByJNIZrz33GoA3HPrg99LSYBcJaWgEI5tdQt3FHtofG4btWqo9I38lkOnmN4GptO5MALHCKZddjI+lj\\\\\\\/baOu5XE+RB3ZrhHrB2zBvRy\\\\\\\/OVKao11bfjNTFooFnVLIev5hJDu+HUT8PkNXx4THDn12bJ87tkocf5Dpv6fFZuHUutyaLjkuODzBY2OO7hC6PZTUnlXPr7ezZs9geCib7wO3CZhcQJUasW+Oqt8+k27mnBCDEdu3CQtzelg2iw8r1ejV9d0RhliFAbtTDFG1s3mZZ9Ne6Y5bOcrmOjoPlFhAA5uJEz4EGm598oJk89SYbW19K38HZuqIIBIjK+qfQt4P5Ymm+vXMZEJfKwhdBxZf0Bs6S0nqok9PbAM8QqhJA4Aog1vsWsmt4BeTz3794o5wms2jSZJdr3AtCu3fALfj83LsGJSAY3fjiuv5neNnQjGQQfV+qa9vBdo\\\\\\\/diTvUEz+2YEIql9GpfuJ6Q5\\\\\\\/+MDLHFstRjd3DFdlqA\\\\\\\/dhdtQ7SJMzVl5q\\\\\\\/KL3Iur70eGe0hWtUBvpZPSgTtFrqyd4AD1C7dtv5mdBn3T05oTlxg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022k4NL2yKrve+7glFwKiMu\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229f0eb61e6a85a30ddd660438e231db3c9a1f3c39b0f578a85be4290d0f70bd7b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459209 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/7/549" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qPNR1-aUcU-A4wPsN-1fxL4CHGeFECeVMlI1H1dxtmQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/7/549" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-akyAWUmVYtjJ0fy-XA2Vfdo4TPcx-0XJdwXgkutAsw0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/7/539" title="Genome Evolution in the Eremothecium Clade of the Saccharomyces Complex Revealed by Comparative Genomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/7/559" title="Unexpected Diversity of Chloroplast Noncoding RNAs as Revealed by Deep Sequencing of the Arabidopsis Transcriptome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459209" id="node459209" data-pisa="ggg;1/7/549" data-pisa-master="ggg;g3.111.000810" data-apath="/ggg/1/7/549.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Nascent Transcription in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anastasia McKinlay</span>, <span class="highwire-citation-author" data-delta="1">Carlos L. Araya</span> and <span class="highwire-citation-author" data-delta="2">Stanley Fields</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">549-558;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000810 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459209" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anastasia McKinlay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Medicine</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasia%2BMcKinlay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKinlay%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasia%2BMcKinlay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carlos L. Araya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BL.%2BAraya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Araya%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BL.%2BAraya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stanley Fields </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Medicine</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Howard Hughes Medical Institute, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stanley%2BFields%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fields%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStanley%2BFields%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fields@u.washington.edu" class="" data-icon-position="" data-hide-link-title="0">fields@u.washington.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/7/549" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459209" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/7/549.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459209" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/7/549.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459209" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/7/549.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459209" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/7/549.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459209" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6">The assessment of transcriptional regulation requires a genome-wide survey of active RNA polymerases. Thus, we combined the nuclear run-on assay, which labels and captures nascent transcripts, with high-throughput DNA sequencing to examine transcriptional activity in exponentially growing <em>Saccharomyces cerevisiae</em>. Sequence read data from these nuclear run-on libraries revealed that transcriptional regulation in yeast occurs not only at the level of RNA polymerase recruitment to promoters but also at postrecruitment steps. Nascent synthesis signals are strongly enriched at TSS throughout the yeast genome, particularly at histone loci. Nascent transcripts reveal antisense transcription for more than 300 genes, with the read data providing support for the activity of distinct promoters driving transcription in opposite directions rather than bidirectional transcription from single promoters. By monitoring total RNA in parallel, we found that transcriptional activity accounts for 80% of the variance in transcript abundance. We computed RNA stabilities from nascent and steady-state transcripts for each gene and found that the most stable and unstable transcripts encode proteins whose functional roles are consistent with these stabilities. We also surveyed transcriptional activity after heat shock and found that most, but not all, heat shock-inducible genes increase their abundance by increasing their RNA synthesis. In summary, this study provides a genome-wide view of RNA polymerase activity in yeast, identifies regulatory steps in the synthesis of transcripts, and analyzes transcript stabilities.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li><li class="kwd"><a href="/keyword/transcription" class="hw-term hw-article-keyword hw-article-keyword-transcription" rel="nofollow">transcription</a></li><li class="kwd"><a href="/keyword/nuclear-run-assay" class="hw-term hw-article-keyword hw-article-keyword-nuclear-run-on-assay" rel="nofollow">nuclear run-on assay</a></li><li class="kwd"><a href="/keyword/high-throughput-sequencing" class="hw-term hw-article-keyword hw-article-keyword-high-throughput-sequencing" rel="nofollow">high-throughput sequencing</a></li><li class="kwd"><a href="/keyword/rna-stability" class="hw-term hw-article-keyword hw-article-keyword-rna-stability" rel="nofollow">RNA stability</a></li></ul><p id="p-7">Changes in mRNA steady-state levels reflect the combined effect of changes in RNA synthesis and degradation. Thus, measurement of only steady-state transcript levels provides an incomplete view of transcriptional regulation. Proper control of transcription is achieved by regulation at several steps: (1) preinitiation complex assembly; (2) promoter escape (or early elongation); (3) productive elongation; and (4) transcription termination (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fuda <em>et al.</em> 2009</a>). Although the spatial distribution of RNA polymerase II can be used to assess transcriptional regulation, polymerase occupancy alone, as detected by chromatin immunoprecipitation, cannot discriminate between polymerases that are simply bound to promoters and those that are transcriptionally engaged. For example, yeast RNA polymerase II associates with many transcriptionally inactive promoters (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Radonjic <em>et al.</em> 2005</a>).</p><p id="p-8">The nuclear run-on (NRO) assay traditionally has been used to assess the transcription of individual genes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brown <em>et al.</em> 1984</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gariglio <em>et al.</em> 1981</a>) and remains the most reliable method of directly measuring transcriptional activity. This method consists of (1) isolation of cell nuclei; (2) isotopic labeling of nascent RNA; and (3) hybridization of labeled RNA to a slot blot, which is then exposed to film. The NRO assay has been scaled up to analyze transcription on a genome-wide basis. Combining this assay with hybridization to nylon macroarrays, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">GarcÃ­a-MartÃ­nez <em>et al.</em> (2004)</a> and <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pelechano <em>et al.</em> (2009)</a> detected distinct classes of yeast genes that change their transcription in response to stimuli. By using human cell lines, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Core <em>et al.</em> (2008)</a> coupled the NRO assay to high-throughput sequencing to provide a genome-wide view of transcriptional activity and identified classes of genes with paused RNA polymerase II near their transcription start sites (TSS).</p><p id="p-9">In this study, we coupled the NRO assay with high-throughput sequencing to analyze transcriptional control in <em>S. cerevisiae</em> and found that polymerase recruitment is the predominant form of regulation. However, we demonstrate that, as in metazoans, postrecruitment steps are important for transcriptional regulation in yeast. We used the parallel measurement of nascent transcription and of steady-state RNA to examine the relationship between transcript synthesis, abundance, and stability. We also used this technology to analyze the transcriptional response to heat shock. In summary, we present a high-resolution view of yeast transcription that discriminates among different modes of transcriptional regulation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and media</h3><p id="p-10"><em>S. cerevisiae</em> strain BY4742 was used (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brachmann <em>et al.</em> 1998</a>). BY4742 was grown in yeast extract-peptone dextrose medium (2% glucose, 2% peptone, 1% yeast extract) at 30Â° to exponential growth phase (OD<sub>600</sub> = 0.5â0.6).</p></div><div id="sec-3" class="subsection"><h3>NRO RNA library construction</h3><p id="p-11">NRO assays were performed as described (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Garcia-Martinez <em>et al.</em> 2004</a>) by the use of approximately 10<sup>7</sup> cells. <em>In vitro</em> transcription was performed with 12 ÂµL of biotin-16-UTP (10 mM; Roche) instead of 13 ÂµL of [<sup>Î±-33</sup>P]UTP. Total RNA was extracted with hot phenol, treated with Turbo DNase I that was passed through CentriSpin columns (Princeton Separations, Adelphia, NJ). An aliquot of total RNA was used for total RNA library construction (see <em>Materials and Methods</em>, Total RNA library construction). Biotinylated RNA was precipitated from total RNA with Dynabeads MyOne Streptavidin C1 (Invitrogen, Carlsbad, CA). Binding was performed in 5 mM Tris-HCl (pH 7.5), 0.5 mM ethylene diamine tetra-acetic acid and 1 M NaCl for 20 min at 42Â° and 2 hr at room temperature with rotation as described (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Patrone <em>et al.</em> 2000</a>). Beads were first washed three times with 800 ÂµL of binding buffer and then washed twice with 800 ÂµL 2x saline sodium citrate (SSC)/15% formamide buffer for 15 min, followed by a 5-min wash in 800 ÂµL 2x SSC. The beads were then washed twice more with 2x SSC with 15% formamide followed by a wash with 2x SSC and a final wash with TE buffer. RNA was eluted from beads by adding 50 ÂµL of 10 Î¼M ethylene diamine tetra-acetic acid /95% formamide and incubation at 65Â° for 5 min. Eluted RNA was precipitated, fragmented (Ambion Fragmentation kit; Ambion, Austin, TX), and used for strand-specific Illumina sequencing library preparation as described (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Parkhomchuk <em>et al.</em> 2009</a>). In brief, this protocol relies on marking the second strand with dUTP during synthesis followed by its degradation after adaptor ligation and size selection. A comparative analysis of strand-specific RNA sequencing methods (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Levin <em>et al.</em> 2010</a>) identified this method as leading by performance across tested criteria. DNA libraries were submitted for 36-bp single-end sequencing.</p><div id="sec-4" class="subsection"><h4>For 5â²-end sense strand sequencing</h4><p>PE adapters sequences:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-13">5â² P-GATCGGAAGAGCGGTTCAGCAGGAATGCCGAG</p></li><li id="list-item-2"><p id="p-14">5â² ACACTCTTTCCCTACACGACGCTCTTCCGATC*T</p></li></ul><p>PE PCR primers:</p><ul class="list-simple " id="list-2"><li id="list-item-3"><p id="p-15">5â² AATGATACGGCGACCACCGAGATCTACACTCGGCATTCCTGCTGAACCGCTCTTCCGATC*T</p></li><li id="list-item-4"><p id="p-16">5â² CAAGCAGAAGACGGCATACGAGATCGGTCTCTTTCCCTACACGACGCTCTTCCGATC*T</p></li></ul><p>PE sequencing primer:</p><ul class="list-simple " id="list-3"><li id="list-item-5"><p id="p-17">5â² TACACTCGGCATTCCTGCTGAACCGCTCTTCCGATCT</p></li></ul></div><div id="sec-5" class="subsection"><h4>For 3â²-end sense strand sequencing.</h4><p>PE adapters sequences:</p><ul class="list-simple " id="list-4"><li id="list-item-6"><p id="p-19">5â² P-GATCGGAAGAGCGGTTCAGCAGGAATGCCGAG</p></li><li id="list-item-7"><p id="p-20">5â² ACACTCTTTCCCTACACGACGCTCTTCCGATC*T</p></li></ul><p>PE PCR primers:</p><ul class="list-simple " id="list-5"><li id="list-item-8"><p id="p-21">5â²âââAATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATC*T</p></li><li id="list-item-9"><p id="p-22">5â²âââCAAGCAGAAGACGGCATACGAGATCGGTCTCGGCATTCCTGCTGAACCGCTCTTCCGATC*T</p></li></ul><p>PE sequencing primer:</p><ul class="list-simple " id="list-6"><li id="list-item-10"><p id="p-23">5â² ACACTCTTTCCCTACACGACGCTCTTCCGATCT</p></li></ul><p>* indicates a phosphorothioate bond.</p></div></div><div id="sec-6" class="subsection"><h3>Total RNA library construction</h3><p id="p-24">Total RNA was treated with Turbo DNase I, fragmented (Ambion Fragmentation kit) and used for Illumina sequencing library preparation as described (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Parkhomchuk <em>et al.</em> 2009</a>). In brief, approximately 50 ng of total RNA was used for first-strand synthesis with random hexamers. After subsequent removal of dNTPs with NucAway Spin Columns (Ambion), second-strand synthesis was performed in the presence of dUTP. Double-stranded cDNA was purified with the PureLink PCR purification kit (Invitrogen) and subjected to end repair, A-tailing, and adaptor ligation steps. DNA libraries were fractionated on a nondenaturing 1x TBE, 6% acrylamide gel (Invitrogen), and fragments between 250 and 350 bp were excised from the gel and eluted by incubation in TE at room temperature with vigorous shaking for 30 min. Uridine-containing, second-strand synthesis products were digested with Uracil-N-Glycosylase (Applied Biosystems). Libraries were amplified via 16 PCR cycles. PCR products were cleaned (Invitrogen PureLink, Agencourt AMPure XP; Beckman Coulter Genomics, Danvers, MA) and submitted for 36-bp single-end sequencing.</p></div><div id="sec-7" class="subsection"><h3>Assessment of the NRO assay performance for the enrichment of nascent RNA</h3><p id="p-25">Two different regions of <em>Arabidopsis thaliana</em> ELF3 gene were transcribed <em>in vitro</em> either in the presence of UTP or B16UTP. B16UTP- and UTP-labeled transcripts from two regions were mixed at approximately 10<sup>9</sup> copies and added to 50 Î¼g of yeast total RNA as spike-in controls. The selection experiments were performed in triplicates to obtain average Cp values. Fold of enrichment (<em>f</em>) of B16UTP-RNA over nonspecifically bound UTP-RNA on streptavidin beads was calculated on the basis of the differences in their Cp values and adjusted by the differences in Cp values observed in their corresponding inputs:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7/549/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/1/7/549/embed/mml-math-1.gif"/></noscript></span></span></p></div><div id="sec-8" class="subsection"><h3>Sequence alignment</h3><p id="p-26">We aligned quality control-filtered reads from the NRO and total RNA libraries to the UCSC sacCer2 reference genome by using <em>Maq</em> (<a href="http://maq.sourceforge.net">http://maq.sourceforge.net</a>) with default parameters and filtered to retain high confidence alignments (mapping quality â¥ 30). The quality-filtered alignments were used to compile a strand-specific measure of abundance for transcribed bases in the genome, normalized by the total number of bases acquired per experiment. Alignments from biological replicates were merged by use of the <em>mapmerge</em> function in <em>Maq</em> and processed equally. All reads were 36 bp long.</p></div><div id="sec-9" class="subsection"><h3>Definition of mappable bases in the genome</h3><p id="p-27">To define bases in the genome that can be unequivocally covered by reads, we identified all unique 36 base sequences from both strands of the reference assembly. We defined mappable bases as all positions covered by these sequences.</p></div><div id="sec-10" class="subsection"><h3>Calculation of gene activity</h3><p id="p-28">The normalized per-base read depth from NRO and total RNA libraries was mapped to a collection of previously annotated features that do not overlap in the same strand, including genes with known transcription start and end sites, as well as genes with unknown transcription starts and ends, and noncoding RNAs (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Nagalakshmi <em>et al.</em> 2008</a>). For each feature, transcriptional activity was measured as the average normalized read depth in mappable, nonintronic bases (âread densityâ). Similarly, we derived a background RNA sequencing rate for each data set as the average minimum read depth in a collection of intergenic regions. The significance of the read density for each feature was calculated against the Poisson distribution of read depth per base expected from the background. A <em>P</em>-value significance cutoff of 0.01 was applied.</p></div><div id="sec-11" class="subsection"><h3>Transcriptional AUC analysis</h3><p id="p-29">The cumulative read depth along each transcript was calculated to examine the rate of transcription along transcript models. For each position <em>i</em> from the TSS, the cumulative read depth <em>Ci</em> represents the sum of bases sequenced from the TSS to <em>i</em>. The cumulative read depth at each position (<em>Ci</em>) was normalized by the total read bases in the transcript model to obtain the fraction (<em>Fci</em>) of the total read depth in the model acquired up to each position <em>i</em>. Transcript model sizes were homogenized to allow comparisons between models of different length. Sizes were adjusted by maintaining the per-base values from the first and last 100 bp of the model and adjusting the intervening values to a 1000-bp segment. Thus, the fractional cumulative NRO read depths along the uniformly sized models represent the amount of transcriptional activity that has occurred as a function of position in the 5â² to 3â² direction of the transcript model.</p><p id="p-30">For each gene, we calculated the area under the curve (AUC) from uniformly sized models by using normalized (<em>Fci</em>) and non-normalized (<em>Ci</em>) cumulative read depth data and 10,000 simulated AUCs (non-normalized). Simulations were performed by randomly placing the observed number of transcript reads within the model and summing the contributions in cumulative read depth of each alignment to the size-adjusted transcript model. The significance of the observed AUC for each transcript model was assessed against the distribution of simulated AUCs under the null hypothesis that transcription occurs evenly throughout the model. Genes were ranked by their normalized AUC values (<em>i.e.</em> based on <em>Fci</em> curves).</p></div><div id="sec-12" class="subsection"><h3>Gene Ontology analysis</h3><p id="p-31">Gene ontology analysis for subsets of gene lists was performed with the GO TermFinder (<a href="http://yeastgenome.org/">http://yeastgenome.org/</a>) with a <em>P</em>-value significance cutoff of 0.01 and a 5% FDR threshold. Analysis of GO term enrichment near the top and bottom of genes ranked by their ratios of transcription activity and transcript abundance (supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS3.pdf">Table S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS4.pdf">Table S4</a>) were performed with GOrilla (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Eden <em>et al.</em> 2009</a>).</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-32">Raw Illumina reads are available at <a href="http://www.ncbi.nlm.nih.gov/geo">http://www.ncbi.nlm.nih.gov/geo</a> as series GSE33136.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Genome-wide assessment of transcriptional activity and transcript abundance</h3><p id="p-33">We established a genome-wide NRO assay for yeast consisting of the following steps: (1) permeabilization of yeast cells with sarcosyl to deplete endogenous nucleotides and stall transcription complexes on chromosomes; (2) reactivation of transcription to extend and label newly synthesized RNA transcripts with biotin-modified UTP; (3) isolation of biotin-modified RNA on streptavidin beads; (4) elution of enriched RNA from beads; and (5) construction of strand-specific high-throughput sequencing libraries (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). To assess the extent of nonspecific binding of RNA to streptavidin beads and to optimize conditions for the NRO steps, we performed parallel NRO reactions with biotin-UTP and, as a control, with UTP. Triplicates of both reactions were processed in parallel through the steps of binding, washing, and eluting from streptavidin beads. To measure the enrichment of biotinylated RNA, we performed quantitative PCR on the eluted RNA as well as on the input RNA. The signals from biotin-UTP-containing RNA and UTP-containing RNA were first normalized to their corresponding signal from input (see the Materials and Methods) and then used to calculate the fold enrichment of biotinylated transcripts over nonbiotinylated ones for three highly expressed yeast genes (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006482">RDN18</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003071">RPL28</a></em>). The NRO assay conditions were optimized to reduce total RNA background to 4%â11% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS1.pdf">Table S1</a>). In addition, we performed NRO reactions spiking in controls consisting of two regions of the <em>Arabidopsis thaliana</em> ELF3 RNA that had been transcribed <em>in vitro</em> either in the presence of UTP or biotin-UTP (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS2.pdf">Table S2</a>). Together these quantitative PCR experiments demonstrated that total RNA represents between 2.6% and 11% of the NRO libraries and that therefore nascent RNA constitutes between approximately 89% and 97% of the NRO samples. By using a standard curve from known amounts of <em>in vitro</em> synthesized, biotin-labeled RNA, we estimate that the starting amount of biotinylated nascent transcripts in RNA extracts before affinity selection comprises approximately 0.05%. Thus, we calculate that after selection nascent RNA has been enriched by a factor of at least 89/0.05, or approximately 1,800-fold, relative to its starting level.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/549/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A strategy for the genome-wide analysis of nascent transcription. (A) Schematic of the NRO assay. Transcription complexes are stalled on genomic DNA by depleting endogenous nucleotide triphosphates via permeabilization of yeast cells with sarcosyl. Reactivation of transcription is achieved by providing biotin-UTP and other nucleotides and switching transcription reactions from ice to 30&#xB0; for 5 min, followed by extraction of total RNA that contains both steady-state and nascent biotinylated transcripts. A fraction of the total RNA is used for total RNA library preparation. Newly synthesized biotinylated RNAs are isolated from total RNA via streptavidin pull-downs to prepare the NRO sample. Both NRO and total RNA strand-specific Illumina sequencing libraries are constructed and submitted for 36 bp single-end sequencing. (B, C) After mapping reads to the reference genome and quality-filtering alignments, the read density is computed from NRO and total RNA data in 4,285 genes with fully distinguishable transcript models (i.e. that do not overlap on the same strand). These measurements of transcript synthesis and transcript abundance for each gene are found to be robust between NRO (B) and total RNA (C) replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315449415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A strategy for the genome-wide analysis of nascent transcription. (A) Schematic of the NRO assay. Transcription complexes are stalled on genomic DNA by depleting endogenous nucleotide triphosphates via permeabilization of yeast cells with sarcosyl. Reactivation of transcription is achieved by providing biotin-UTP and other nucleotides and switching transcription reactions from ice to 30&#xB0; for 5 min, followed by extraction of total RNA that contains both steady-state and nascent biotinylated transcripts. A fraction of the total RNA is used for total RNA library preparation. Newly synthesized biotinylated RNAs are isolated from total RNA via streptavidin pull-downs to prepare the NRO sample. Both NRO and total RNA strand-specific Illumina sequencing libraries are constructed and submitted for 36 bp single-end sequencing. (B, C) After mapping reads to the reference genome and quality-filtering alignments, the read density is computed from NRO and total RNA data in 4,285 genes with fully distinguishable transcript models (&lt;em&gt;i.e.&lt;/em&gt; that do not overlap on the same strand). These measurements of transcript synthesis and transcript abundance for each gene are found to be robust between NRO (B) and total RNA (C) replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/549/F1.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/549/F1.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/549/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/549/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461064" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-34" class="first-child">A strategy for the genome-wide analysis of nascent transcription. (A) Schematic of the NRO assay. Transcription complexes are stalled on genomic DNA by depleting endogenous nucleotide triphosphates via permeabilization of yeast cells with sarcosyl. Reactivation of transcription is achieved by providing biotin-UTP and other nucleotides and switching transcription reactions from ice to 30Â° for 5 min, followed by extraction of total RNA that contains both steady-state and nascent biotinylated transcripts. A fraction of the total RNA is used for total RNA library preparation. Newly synthesized biotinylated RNAs are isolated from total RNA via streptavidin pull-downs to prepare the NRO sample. Both NRO and total RNA strand-specific Illumina sequencing libraries are constructed and submitted for 36 bp single-end sequencing. (B, C) After mapping reads to the reference genome and quality-filtering alignments, the read density is computed from NRO and total RNA data in 4,285 genes with fully distinguishable transcript models (<em>i.e.</em> that do not overlap on the same strand). These measurements of transcript synthesis and transcript abundance for each gene are found to be robust between NRO (B) and total RNA (C) replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">To examine RNA polymerase II activity and monitor transcript abundance throughout the genome, we constructed both NRO and total RNA libraries. Total RNA libraries were prepared from aliquots of the biotin-UTP-treated NRO sample by excluding the steps of nascent RNA selection on streptavidin beads (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Strand-specific Illumina sequencing libraries were built from RNA isolated from three independent cultures of log phase yeast (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Parkhomchuk <em>et al.</em> 2009</a>). We acquired 63 million and 83 million 36 base-pair reads from the NRO and total RNA libraries, respectively. After aligning reads to the genome, we filtered out low-quality alignments and rRNA sequences; because we carried out the protocols without an rRNA depletion step, rRNA sequences accounted for more than 90% of the total reads. After filtering, approximately 2.5 million uniquely mapped NRO reads and approximately 1 million total RNA reads were left (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS3.pdf">Table S3</a>). This combined approach allowed us to measure the amount of nascent transcripts as well as steady-state transcripts in parallel.</p><p id="p-36">To analyze sequencing results for both NRO and total RNA samples, we defined read depth as the number of sequencing reads acquired per base, normalized by the total bases sequenced for that sample. We summed the read depth values for each mappable, non-intronic base within a transcript model and divided this sum by the total number of these bases within that transcript to calculate read density. Both read depth and read density are presented in units of read depth per million bases acquired to obtain higher scale values.</p><p id="p-37">We focused our analysis on a set of 4,285 genes with known TSSs and termination sites (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nagalakshmi <em>et al.</em> 2008</a>) and that do not overlap on the same strand. The reproducibility of the transcriptional activity and steady-state levels as determined from read densities in replicates of NRO and total RNA libraries was robust (Pearsonâs <em>r<sup>2</sup></em> &gt; 0.97 and <em>r<sup>2</sup></em> &gt; 0.99, respectively; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B and C</a>). Therefore, we merged reads from biological replicates to obtain a single NRO data set and a single total RNA data set for subsequent analyses. Of the 4,285 nonoverlapping transcript models, 2,717 and 2,802 genes were transcriptionally active in the NRO and total RNA libraries, respectively, as measured against the background from intergenic regions (<em>P</em> &lt; 0.01; see the Materials and Methods). Our measurements of nascent transcription showed moderate correlation with previous genome-wide estimates of transcriptional activity derived from hybridization of radiolabeled nascent transcripts to nylon macroarrays (Spearmanâs Ï = 0.53; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS1.pdf">Figure S1</a>A). This correlation is comparable with that between previous measures of transcriptional activity obtained via the use of NRO approaches (Spearmanâs Ï = 0.61; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS1.pdf">Figure S1</a>B) (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Garcia-Martinez <em>et al.</em> 2004</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pelechano <em>et al.</em> 2009</a>). Excluding the rDNA cluster, we detected no large regions (&gt;3 kb) in the yeast genome with extensive synthesis of nascent RNA (<em>Z</em> &gt; 3.1; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-38">Because the NRO assay reflects the activity of RNA polymerases on transcribed genes, we compared the transcription data from the NRO sample to the genome-wide distribution of RNA polymerase II in yeast (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lefrancois <em>et al.</em> 2009</a>), and found a low but significant correlation (Pearsonâs <em>r<sup>2</sup></em> = 0.41, <em>P</em> &lt; 2.2 Ã 10<sup>â16</sup>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS3.pdf">Figure S3</a>A). The modest correlation between polymerase occupancy and transcriptional activity has been previously reported (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rodriguez-Gil <em>et al.</em> 2010</a>) and might result from differences in the dynamic range of the protocols (as suggested by <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Rodriguez-Gil <em>et al.</em> 2010)</a> or the inability of the NRO assay to precisely assign the positions of transcribing polymerases, because fragmentation of RNA was performed after its affinity selection. As expected, total RNA also correlated with polymerase occupancy, but to a lesser extent (Pearsonâs <em>r<sup>2</sup></em> = 0.37, <em>P</em> &lt; 2.2 Ã 10<sup>â16</sup>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS3.pdf">Figure S3</a>B). This analysis corroborates that RNA polymerase occupancy is a better predictor of transcriptional activity than of transcript abundance.</p><p id="p-39">Although pre-mRNA splicing occurs co-transcriptionally in yeast, we observed greater intronic read depths in the NRO libraries than in the total RNA libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS4.pdf">Figure S4</a>A), and this increase was not caused by increased transcript levels. Moreover, splicing was much less abundant in the NRO libraries than in the total RNA libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS4.pdf">Figure S4</a>B), further arguing that the NRO libraries are enriched for nascent transcripts.</p></div><div id="sec-16" class="subsection"><h3>Nascent transcription is enriched near TSS</h3><p id="p-40">Analysis of RNA polymerase II occupancy data from high-density tiling microarrays demonstrated that only highly expressed yeast genes (~16% of all genes) have an even distribution of RNA polymerase II throughout their coding region, and that a comparable fraction (~18%) have an enrichment of the polymerase near their 5â² ends (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Venters and Pugh 2009</a>). This result and recent analysis of polymerase-associated RNAs suggests that transcription in yeast, as in metazoans, can be subject to rate-limiting regulatory steps in addition to polymerase recruitment and activation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Churchman and Weissman 2011</a>). To analyze RNA polymerase II activity within genes, we examined read depth along a set of 2,530 transcript models of genes that were transcriptionally active in both the NRO and total RNA libraries. In contrast to the total RNA sample, the NRO sample showed an enrichment of reads near the 5â² ends of transcript models, with a read depth peak occurring approximately 50 bp downstream of the TSS (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), as has been observed in human and <em>Drosophila</em> cells (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Core <em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rasmussen and Lis 1993</a>). Sarkosyl treatment in the NRO assay allows RNA polymerases that are either transcribing or paused to incorporate nucleotides upon transcription reactivation (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rougvie and Lis 1990</a>) but prevents further rounds of initiation (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hawley and Roeder 1985</a>). Thus, the peak in read depth near TSSs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) likely indicates an accumulation of paused RNA polymerases at 5â² ends. Because we carried out the enrichment of nascent RNA before its chemical fragmentation, all nascent transcripts detected in the NRO assay should have originated at their TSSs. This feature of the protocol results in nested sets of transcripts initiating at their 5â² end, with progressively fewer transcripts in each set containing bases further toward the 3â² end, even for cases with relatively equal distribution of RNA polymerases along the gene. Indeed, we observe a gradual decrease in read depth beginning approximately 100 bp downstream from the TSS (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>; dashed line). The slope projects a read depth peak toward the TSS of 0.20 resulting from the 5â² end being present in all the nascent transcripts. That the promoter proximal peak rises sharply near the TSS to a height more than 1.5-fold greater than the extrapolated value provides evidence for an enrichment of nascent transcripts, and hence RNA polymerase activity, near TSSs under NRO conditions. To assure that the observed promoter-proximal peak is a feature of yeast transcription, we performed additional analyses and ruled out the effect of nonspecific RNA binding (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS5.pdf">Figure S5</a>), increased average read depth of shorter transcripts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS6.pdf">Figure S6</a>A), and sequencing biases (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS6.pdf">Figure S6</a>B) as possible contributors to the increased read depth near 5â² ends of yeast transcripts.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/549/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of RNA polymerase activity relative to TSSs and termination sites. (A) The average sense read depth in the NRO (red) and total RNA (blue) libraries along 2,530 genes that are transcriptionally active in both libraries is plotted as a function of distance from TSSs. NRO libraries display a read depth peak within 100 bp 3&#x2032; from the TSS (highlighted in gray) with a peak maximum at approximately 50 bp downstream of their TSSs. Dashed line extends the slope of sequencing reads obtained along the transcript length (between 100-500 bp) toward the TSS to extrapolate read depth signal at the 5&#x2032; ends of transcripts. (B) Distribution of RNA polymerase activity relative to poly(A) sites in the 5&#x2032;-end and 3&#x2032;-end sequenced NRO libraries. The average sense read depth in the NRO (red) and total RNA (blue) libraries in transcriptionally active genes is plotted as a function of distance from poly(A) cleavage sites. A higher read depth near transcript termination sites is found for the total RNA libraries as compared to the NRO libraries. Inset: An additional strand-specific NRO library was sequenced from the 3&#x2032; end to compare sense strand read depth information to that from 5&#x2032; end libraries. Sense read depth from 5&#x2032; end (light red) and 3&#x2032; end (dark red) nascent RNA libraries is plotted as a function of distance from poly(A) sites for convergently transcribed genes (N = 2000)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315449415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of RNA polymerase activity relative to TSSs and termination sites. (A) The average sense read depth in the NRO (red) and total RNA (blue) libraries along 2,530 genes that are transcriptionally active in both libraries is plotted as a function of distance from TSSs. NRO libraries display a read depth peak within 100 bp 3&#x2032; from the TSS (highlighted in gray) with a peak maximum at approximately 50 bp downstream of their TSSs. Dashed line extends the slope of sequencing reads obtained along the transcript length (between 100-500 bp) toward the TSS to extrapolate read depth signal at the 5&#x2032; ends of transcripts. (B) Distribution of RNA polymerase activity relative to poly(A) sites in the 5&#x2032;-end and 3&#x2032;-end sequenced NRO libraries. The average sense read depth in the NRO (red) and total RNA (blue) libraries in transcriptionally active genes is plotted as a function of distance from poly(A) cleavage sites. A higher read depth near transcript termination sites is found for the total RNA libraries as compared to the NRO libraries. Inset: An additional strand-specific NRO library was sequenced from the 3&#x2032; end to compare sense strand read depth information to that from 5&#x2032; end libraries. Sense read depth from 5&#x2032; end (light red) and 3&#x2032; end (dark red) nascent RNA libraries is plotted as a function of distance from poly(A) sites for convergently transcribed genes (&lt;em&gt;N&lt;/em&gt; = 2000).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/549/F2.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/549/F2.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/549/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/549/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461056" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-41" class="first-child">Distribution of RNA polymerase activity relative to TSSs and termination sites. (A) The average sense read depth in the NRO (red) and total RNA (blue) libraries along 2,530 genes that are transcriptionally active in both libraries is plotted as a function of distance from TSSs. NRO libraries display a read depth peak within 100 bp 3â² from the TSS (highlighted in gray) with a peak maximum at approximately 50 bp downstream of their TSSs. Dashed line extends the slope of sequencing reads obtained along the transcript length (between 100-500 bp) toward the TSS to extrapolate read depth signal at the 5â² ends of transcripts. (B) Distribution of RNA polymerase activity relative to poly(A) sites in the 5â²-end and 3â²-end sequenced NRO libraries. The average sense read depth in the NRO (red) and total RNA (blue) libraries in transcriptionally active genes is plotted as a function of distance from poly(A) cleavage sites. A higher read depth near transcript termination sites is found for the total RNA libraries as compared to the NRO libraries. Inset: An additional strand-specific NRO library was sequenced from the 3â² end to compare sense strand read depth information to that from 5â² end libraries. Sense read depth from 5â² end (light red) and 3â² end (dark red) nascent RNA libraries is plotted as a function of distance from poly(A) sites for convergently transcribed genes (<em>N</em> = 2000).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Near transcript termination sites, total RNA libraries show greater read depths than NRO libraries (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>), which is consistent with the fact that most nascent RNAs are not full length (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Carrillo Oesterreich <em>et al.</em> 2010</a>). Because our protocol acquires sense strand information from the 5â²-most 36 bases of fragments, we were not able to detect a peak of nascent transcription near poly(A) sites, as was previously shown for human and yeast gene transcription (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Carrillo Oesterreich <em>et al.</em> 2010</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Core <em>et al.</em> 2008</a>). To address this issue, we prepared a new NRO library from which we specifically sequenced the 3â² end bases of transcripts and found a small peak of transcription consistent with poly(A) cleavage (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>; inset).</p></div><div id="sec-17" class="subsection"><h3>RNA polymerase II activity is consistent with transcriptional control at postrecruitment steps</h3><p id="p-43">In principle, the capacity to measure transcriptional activity for each base in the genome should allow us to distinguish rate-limiting steps in transcription other than transcription initiation. However, because read depth in high-throughput sequencing of RNA libraries is noisy (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ingolia <em>et al.</em> 2009</a>), this goal is difficult. We sought to address this challenge by analyzing how reads accumulate along transcript models. Genes with a relatively homogenous distribution of active polymerases should display a linear increase in the cumulative fraction of reads along a transcript, whereas genes with polymerases predominantly near their 5â² ends or near their 3â² ends generate deflections from this line (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). For a gene in which most reads are acquired near the 5â² end of the transcript, the cumulative fraction curve climbs sharply to its maximum value and flattens. Thus, the AUC for the cumulative fraction of reads in genes with polymerase activity concentrated at the 5â² end is larger than for those that are not paused or that accumulate reads at the 3â² end.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/549/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of paused transcripts from nascent RNA libraries. (A) Graph depicts the expected accumulation of reads as a function of distance from TSS for transcripts enriched with polymerase at their 5&#x2032; end (green line) or 3&#x2032; end (blue line) or transcripts with homogenous RNA polymerase II activity (dashed line). (B) AUC ranking of genes. For each transcript model, the fraction of the total read depth acquired per position along the direction of transcription was computed. This fractional, cumulative read depth within transcript models was adjusted to a uniform model by redistributing values to reflect the read depth in the 5&#x2032;-most 100 bp, 3&#x2032;-most 100 bp, and the intervening sequence (marked with vertical dashed lines). Genes were ranked by the AUC of fractional, cumulative read depth in the uniform models. The fraction of the total read depth acquired for each transcript model is shown as a function of distance along the uniform model, where red represents 100%. Genes in the top 5% are highlighted within the black box and contain the histone loci. (C) IGV genome browser views (http://www.broadinstitute.org/igv) of NRO read depth in the four divergently transcribed histone loci. With the exception of HTA2, core histones exhibit strong promoter-proximal pausing in NRO libraries. Normalized read depth is shown in the plus and minus strands as positive and negative values, respectively. Read depth range is indicated between brackets. Transcript models are schematized with green arrows indicating coding sequences. Mappable bases are marked green in the track underneath IGV views." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315449415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of paused transcripts from nascent RNA libraries. (A) Graph depicts the expected accumulation of reads as a function of distance from TSS for transcripts enriched with polymerase at their 5&#x2032; end (green line) or 3&#x2032; end (blue line) or transcripts with homogenous RNA polymerase II activity (dashed line). (B) AUC ranking of genes. For each transcript model, the fraction of the total read depth acquired per position along the direction of transcription was computed. This fractional, cumulative read depth within transcript models was adjusted to a uniform model by redistributing values to reflect the read depth in the 5&#x2032;-most 100 bp, 3&#x2032;-most 100 bp, and the intervening sequence (marked with vertical dashed lines). Genes were ranked by the AUC of fractional, cumulative read depth in the uniform models. The fraction of the total read depth acquired for each transcript model is shown as a function of distance along the uniform model, where red represents 100%. Genes in the top 5% are highlighted within the black box and contain the histone loci. (C) IGV genome browser views (http://www.broadinstitute.org/igv) of NRO read depth in the four divergently transcribed histone loci. With the exception of &lt;em&gt;HTA2&lt;/em&gt;, core histones exhibit strong promoter-proximal pausing in NRO libraries. Normalized read depth is shown in the plus and minus strands as positive and negative values, respectively. Read depth range is indicated between brackets. Transcript models are schematized with green arrows indicating coding sequences. Mappable bases are marked green in the track underneath IGV views.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/549/F3.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/549/F3.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/549/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/549/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461068" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-44" class="first-child">Identification of paused transcripts from nascent RNA libraries. (A) Graph depicts the expected accumulation of reads as a function of distance from TSS for transcripts enriched with polymerase at their 5â² end (green line) or 3â² end (blue line) or transcripts with homogenous RNA polymerase II activity (dashed line). (B) AUC ranking of genes. For each transcript model, the fraction of the total read depth acquired per position along the direction of transcription was computed. This fractional, cumulative read depth within transcript models was adjusted to a uniform model by redistributing values to reflect the read depth in the 5â²-most 100 bp, 3â²-most 100 bp, and the intervening sequence (marked with vertical dashed lines). Genes were ranked by the AUC of fractional, cumulative read depth in the uniform models. The fraction of the total read depth acquired for each transcript model is shown as a function of distance along the uniform model, where red represents 100%. Genes in the top 5% are highlighted within the black box and contain the histone loci. (C) IGV genome browser views (<a href="http://www.broadinstitute.org/igv">http://www.broadinstitute.org/igv</a>) of NRO read depth in the four divergently transcribed histone loci. With the exception of <em>HTA2</em>, core histones exhibit strong promoter-proximal pausing in NRO libraries. Normalized read depth is shown in the plus and minus strands as positive and negative values, respectively. Read depth range is indicated between brackets. Transcript models are schematized with green arrows indicating coding sequences. Mappable bases are marked green in the track underneath IGV views.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">To calculate the AUC for each gene, we first calculated the cumulative fraction of reads acquired at each position. We adjusted all transcript models to a uniform length, corresponding to the number of reads obtained for the 5â²-most 100 bp, the 3â²-most 100 bp, and the intervening sequence. We ranked genes by AUC to identify those with distinct distributions of RNA polymerase II activity along the length of their transcripts, which might correlate with distinct modes of transcriptional regulation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). AUC ranking of transcripts revealed that the top 5% of genes are enriched for nucleosome components (<em>P</em> = 7.15 Ã 10<sup>â5</sup>), chromatin assembly and disassembly genes (<em>P</em> = 0.001), helicases (<em>P</em> = 0.007), and ATPases (<em>P</em> = 0.008). Histone genes drive this enrichment because seven of the eight core histone genes demonstrated strong evidence of accumulation of polymerases at their 5â² ends, which could reflect promoter-proximal pausing (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Given that transcription of histone genes is tightly limited in the cell cycle to early <em>S</em> phase, these data suggest that polymerase pausing could play a role in assuring rapid induction of histone synthesis just before the DNA synthesis (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hereford <em>et al.</em> 1981</a>), consistent with the previous observation of paused RNA polymerase II on the <em>Drosophila</em> histone H3 gene (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 1996</a>). Similarly, a Poisson-based test, as used in <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Core <em>et al.</em> 2008</a>, characterized up to 20% of yeast transcripts as paused, suggesting that their expression may be subject to postinitiation transcriptional control (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS7.pdf">Figure S7</a>). However, previous chromatin immunoprecipitation analysis of yeast did not provide evidence for widespread polymerase pausing (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Venters and Pugh 2009</a>). The evidence of polymerase pausing from the NRO results may partially reflect the protocol used because fragmentation of RNA after affinity selection can make it difficult to precisely assign the position of polymerase.</p><p id="p-46">The AUC ranking allowed further identification of potential rate-limiting steps in transcription. This analysis identified not only genes that acquired the majority of their reads near their TSSs but also genes with low and middle AUC values. Genes with the lowest AUC values exhibited an accumulation of reads near their 3â² ends, which may reflect regulation at the level of transcription termination (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS8.pdf">Figure S8</a>). The middle range of the AUC-ranked list includes transcripts in which the cumulative fraction of reads increased linearly. In addition, the intermediate range includes transcripts with increased read densities in both the promoter-proximal region and near the 3â² end of the transcript (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS8.pdf">Figure S8</a>). Such transcripts could be targets of regulation at both promoter escape and transcriptional termination. Finally, transcripts in which nascent transcription is concentrated in the middle of the transcript are also found in this range. Although some of the patterns of read depth could be attributable to stochastic variations in fragment ligation, amplification or sequencing, the observation of transcriptional regulation occurring subsequent to initiation, pausing and release from pausing is consistent with recently observed patterns of RNA polymerase II activity derived from polymerase-associated RNAs (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Churchman and Weissman 2011</a>).</p></div><div id="sec-18" class="subsection"><h3>Nascent antisense transcription is not predominantly driven by bidirectional promoters</h3><p id="p-47">Because recent genomic analyses have shown substantial antisense transcription in yeast (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Nagalakshmi <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Neil <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Xu <em>et al.</em> 2009</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yassour <em>et al.</em> 2010</a>), we asked whether we could detect the presence of antisense transcripts in the NRO and total RNA libraries. We found evidence of antisense transcription in 310 genes in the NRO libraries and in 135 genes in the total RNA libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS9.pdf">Figure S9</a>). The increased prevalence of antisense transcripts in the NRO sample likely relates to their instability and their consequent loss from the pool of total RNA.</p><p id="p-48">We asked whether bidirectional transcription from single promoters, as has been recently reported in yeast (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Neil <em>et al.</em> 2009</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Xu <em>et al.</em> 2009</a>), underlies the increased antisense transcription. We classified each promoter with respect to its upstream adjacent gene as either nondivergent (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), in which the upstream gene is encoded on the same strand; or divergent (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>), in which the upstream gene is encoded on the opposite strand. We examined sense and antisense read depth in the 5â² flanking regions of the reference promoters (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, A and B</a>, in blue) for the two configurations. For nondivergent promoters, very few antisense reads accumulated in the 200-bp 5â² to the TSSs of the reference promoters (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>), indicating that these promoters drive little antisense transcription. Similarly, for divergent promoters, very few antisense reads accumulated in the 100-bp 5â² to the TSSs (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). The increasing antisense read depth beyond 100 bp reflects the activity of distinct promoters for the upstream genes, consistent with separate preinitiation complexes at divergently transcribed genes leading to a detectable signal (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Venters and Pugh 2009</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/549/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bidirectional transcription from TSSs is not predominant in yeast. Sense (solid line) and antisense (dotted line) read depths are shown near the TSSs of reference genes (blue) with nondivergent (A) and divergent (B) promoters. Schematics indicate the orientations of transcripts upstream of reference genes with solid lines annotating sense and dotted lines antisense transcription. NRO and total RNA read depths are shown in red and blue, respectively. Read depth is classified as antisense and sense with respect to the 3&#x2032; reference gene in blue. In both types of promoters, NRO libraries display predominant sense transcription and a read depth peak approximately 50 bp downstream of their TSSs but do not show significant transcription in the antisense orientation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315449415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bidirectional transcription from TSSs is not predominant in yeast. Sense (solid line) and antisense (dotted line) read depths are shown near the TSSs of reference genes (blue) with nondivergent (A) and divergent (B) promoters. Schematics indicate the orientations of transcripts upstream of reference genes with solid lines annotating sense and dotted lines antisense transcription. NRO and total RNA read depths are shown in red and blue, respectively. Read depth is classified as antisense and sense with respect to the 3&#x2032; reference gene in blue. In both types of promoters, NRO libraries display predominant sense transcription and a read depth peak approximately 50 bp downstream of their TSSs but do not show significant transcription in the antisense orientation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/549/F4.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/549/F4.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/549/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/549/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461058" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-49" class="first-child">Bidirectional transcription from TSSs is not predominant in yeast. Sense (solid line) and antisense (dotted line) read depths are shown near the TSSs of reference genes (blue) with nondivergent (A) and divergent (B) promoters. Schematics indicate the orientations of transcripts upstream of reference genes with solid lines annotating sense and dotted lines antisense transcription. NRO and total RNA read depths are shown in red and blue, respectively. Read depth is classified as antisense and sense with respect to the 3â² reference gene in blue. In both types of promoters, NRO libraries display predominant sense transcription and a read depth peak approximately 50 bp downstream of their TSSs but do not show significant transcription in the antisense orientation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Relationship between RNA synthesis, abundance, and stability</h3><p id="p-50">Having measured both transcriptional activity and steady-state levels of RNA, we sought to use the genome-wide data to explore the relationship between RNA synthesis, abundance, and stability. For this analysis, we chose to estimate transcriptional activity and steady-state RNA by calculating the read density excluding data within 200 bp of the TSS; in this way, we avoided analyzing transcripts that may have prematurely terminated. We compared the NRO-derived transcriptional activity with total RNA-derived steady-state levels and found a significant correlation (Pearsonâs <em>r<sup>2</sup></em> = 0.80), suggesting that this measure of transcriptional activity accounts for approximately 80% of the variance in transcript abundance (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/549/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contribution of transcript synthesis to transcript abundance and inference of transcript stability. Read density in the NRO and total RNA libraries is plotted for a set of 2,414 genes with significant transcription from 201 bp downstream of TSS. Transcript synthesis and transcript abundance, measured as read density within this region of transcripts in NRO and total RNA libraries, respectively, are strongly correlated. To estimate nascent transcript stability, we calculated the ratio of the transcript synthesis to transcript abundance for each gene. Genes in the top and bottom deciles of these ratios are circled in red and blue, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315449415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Contribution of transcript synthesis to transcript abundance and inference of transcript stability. Read density in the NRO and total RNA libraries is plotted for a set of 2,414 genes with significant transcription from 201 bp downstream of TSS. Transcript synthesis and transcript abundance, measured as read density within this region of transcripts in NRO and total RNA libraries, respectively, are strongly correlated. To estimate nascent transcript stability, we calculated the ratio of the transcript synthesis to transcript abundance for each gene. Genes in the top and bottom deciles of these ratios are circled in red and blue, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/549/F5.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/549/F5.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/549/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/549/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461043" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5â</span> <p id="p-51" class="first-child">Contribution of transcript synthesis to transcript abundance and inference of transcript stability. Read density in the NRO and total RNA libraries is plotted for a set of 2,414 genes with significant transcription from 201 bp downstream of TSS. Transcript synthesis and transcript abundance, measured as read density within this region of transcripts in NRO and total RNA libraries, respectively, are strongly correlated. To estimate nascent transcript stability, we calculated the ratio of the transcript synthesis to transcript abundance for each gene. Genes in the top and bottom deciles of these ratios are circled in red and blue, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">To estimate the stability of nascent transcripts, we calculated the ratio of the transcriptional activity to the steady-state level for each gene. High ratios are consistent with short-lived transcripts, because much of their nascent RNA does not reach the steady-state pool. Gene Ontology analysis revealed that genes with the greatest ratios of transcriptional activity to steady state levels (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, in red) are enriched for transcripts involved in translational elongation (<em>P</em> = 7.0 Ã 10<sup>â6</sup>), drug transport (<em>P</em> = 5.1 Ã 10<sup>â6</sup>), signal transduction (<em>P</em> = 1.0 Ã 10<sup>â4</sup>), and protein amino acid phosphorylation (<em>P</em> = 1.5 Ã 10<sup>â4</sup>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS4.pdf">Table S4</a>). Short-lived transcripts are capable of rapid adjustments in abundance to assure quick responses to stimuli (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Friedel <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Narsai <em>et al.</em> 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2002</a>). Conversely, genes with the lowest ratios of transcriptional activity to steady state levels (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, in blue) represent more stable nascent transcripts, and are enriched for structural constituents of the ribosome (<em>P</em> = 6.4 Ã 10<sup>â8</sup>) and genes involved in nucleosome assembly (<em>P</em> = 3.6 Ã 10<sup>â4</sup>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS5.pdf">Table S5</a>). Thus, our estimates of the stability of nascent transcripts agree with the expected stabilities of transcripts based on the functions of their encoded proteins, as observed previously (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Friedel <em>et al.</em> 2009</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Narsai <em>et al.</em> 2007</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2002</a>).</p><p id="p-53">The absolute half-lives of mRNAs have traditionally been measured over a time course after the inhibition of transcription by chemical or thermal inactivation of RNA polymerase II (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2002</a>). However, mRNA half-lives estimated by different groups by the use of similar types of experiments (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2002</a>) show little correlation (Pearsonâs <em>r<sup>2</sup></em> = 0.17; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS10.pdf">Figure S10</a>A). We observed no correlation between the estimated stabilities of nascent transcripts in our study and previously measured mRNA decay rates (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2002</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/FigureS10.pdf">Figure S10</a>, B and C). These discrepancies could at least in part be attributable to differences in the stability of nascent transcripts <em>vs.</em> that of mRNAs. In addition, the temperature shift to halt transcription might induce changes in RNA folding and thus affect estimates of mRNA half-lives. Finally, technical differences in the methodologies used may be contributing to these differences. For example, microarray measurements in of relative transcript abundance are subject to errors caused by differences in hybridization efficiencies. Similarly, sequencing biases introduce errors into our measurements of both transcriptional activity and steady-state levels of mRNAs and these errors may be compounded in the calculation of ratios.</p></div><div id="sec-20" class="subsection"><h3>Changes in yeast gene transcription in response to heat shock</h3><p id="p-54">Several genome-wide studies have identified yeast genes that respond to temperature shock by elevating their transcript levels (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gasch <em>et al.</em> 2000</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hahn <em>et al.</em> 2004</a>). We sought to examine the extent to which the effect of heat shock on transcript abundance occurs via the up-regulation of transcriptional activity. Thus, we built NRO libraries (âNRO [HS]â) and total RNA libraries (âtotal RNA [HS]â) from RNA isolated from six independent cultures of log phase yeast under heat shock conditions (20 min at 39Â°; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS6.pdf">Table S6</a>).</p><p id="p-55">To estimate the effect of heat shock on RNA abundance and transcription activity, we calculated the log-transformed ratios of read density values after and prior to heat shock for total RNA (total RNA [HS]/total RNA) and for nascent RNA (NRO [HS]/NRO). For total RNA, log-space ratios greater than zero represent transcripts with increased RNA abundance upon heat shock treatment, whereas those less than zero indicate decreased RNA abundance. For nascent RNA, log-space ratios greater than zero identify genes whose transcription was up-regulated upon heat shock, whereas those less than zero indicate those down-regulated (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). This analysis reveals that 304 (9%) yeast genes responded to heat stress by increasing RNA abundance by 2-fold or more (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS7.xlsx">Table S7</a>). Approximately 65% of those genes showed an increase in nascent RNA synthesis in response to heat shock, indicating that although transcript levels are predominantly regulated at the level of transcription, changes in mRNA stability play a role as well (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Castells-Roca <em>et al.</em> 2011</a>). We found that the majority (74%) of the genes induced at least 3-fold by heat shock in the study by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gasch <em>et al.</em> (2000)</a> showed concordant increases in transcript abundance and in nascent RNA synthesis after heat shock, again highlighting the transcriptional activation that contributes to the increased RNA abundance for this class of genes. In addition, heat shock-inducible genes from the study by <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gasch <em>et al.</em> 2000</a> showed a stronger correlation between changes in transcript abundance and changes in nascent transcript synthesis (Pearsonâs <em>r<sup>2</sup></em> = 0.55) compared with all yeast genes analyzed in this study (Pearsonâs <em>r<sup>2</sup></em> = 0.24). However, only 30 of the 304 genes with 2-fold or greater increased total RNA abundance after heat stress are known targets of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003041">Hsf1</a>, a master transcriptional activator of the heat shock response, or contain <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003041">Hsf1</a> or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004640">Msn2</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001545">4</a> recognition sites in their promoters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1/TableS7.xlsx">Table S7</a>). Approximately 87% (26) of these 30 genes showed increased nascent transcript levels in response to heat shock. These findings suggest that additional factors may mediate stress-induced transcriptional activation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/7/549/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcript synthesis and abundance response to heat shock treatment. Ratios of read densities before and after heat shock are plotted for the NRO and total RNA libraries with all significantly expressed yeast genes shown in green and heat shock-responsive genes (&#x2265;3x increase in abundance upon heat shock treatment in Gasch et al. 2000 data set) shown in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315449415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transcript synthesis and abundance response to heat shock treatment. Ratios of read densities before and after heat shock are plotted for the NRO and total RNA libraries with all significantly expressed yeast genes shown in green and heat shock-responsive genes (&#x2265;3x increase in abundance upon heat shock treatment in Gasch &lt;em&gt;et al.&lt;/em&gt; 2000 data set) shown in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/7/549/F6.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/7/549/F6.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/7/549/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/7/549/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461104" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6â</span> <p id="p-56" class="first-child">Transcript synthesis and abundance response to heat shock treatment. Ratios of read densities before and after heat shock are plotted for the NRO and total RNA libraries with all significantly expressed yeast genes shown in green and heat shock-responsive genes (â¥3x increase in abundance upon heat shock treatment in <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Gasch <em>et al.</em> 2000</a> data set) shown in red.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-57">In this study, we provide a high-resolution view of genome-wide transcriptional regulation in <em>S. cerevisiae</em>. By coupling the NRO assay to high-throughput sequencing, we assayed the synthesis of nascent RNA in yeast under normal growth conditions and in response to heat stress. We measured RNA abundance and synthesis in parallel, which revealed that cellular RNA abundance in yeast under both growth conditions is predominantly controlled by the rate of RNA synthesis. In accord with previous degradation experiments, we observed distinct functional gene classes between stable and unstable transcripts (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Friedel <em>et al.</em> 2009</a>; <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Narsai <em>et al.</em> 2007</a>; <a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2002</a>).</p><p id="p-58">These data allowed us to examine the spatial distribution of RNA synthesis along transcripts, which showed that nascent transcription in yeast is enriched near TSSs, as it is in humans. The data further suggest that postinitiation regulation in yeast may contribute to the control of gene expression (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Churchman and Weissman 2011</a>). Although recruitment of RNA polymerase to promoters appears to be rate-limiting for the large majority of yeast genes, transcription of up to 20% of yeast genes may be controlled at least in part at postinitiation steps. The mechanisms that determine polymerase pausing, arrest, or termination are not yet well understood. Many factors that modulate transcriptional pausing and the rate of elongation have been identified (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sims <em>et al.</em> 2004</a>). Some of them are restricted to a subset of eukaryotes, with no homologs identified in yeast. However, in a recent study, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Rodriguez-Gil <em>et al.</em> (2010)</a> identified a subset of elongation-related factors (<em>e.g.</em> DSIF, Mediator, and the RNA polymerase II subunit <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003038">Rpb9</a>) that might influence the probability of RNA polymerase II pausing or arrest in yeast. In addition, the elongation factor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003011">TFIIS</a> (yeast <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003011">Dst1</a>) stimulates intrinsic RNA polymerase RNA cleavage activity and thus promotes elongation of arrested RNA polymerase <em>in vivo</em> (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Churchman and Weissman 2011</a>). Signals in the DNA or RNA may affect RNA elongation; for example, both a hairpin structure in the nascent RNA and a run of uridines at the 3â² end of a transcript interfere with elongation and promote transcription termination (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bengal and Aloni 1989</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Keene <em>et al.</em> 1999</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Palangat <em>et al.</em> 1998</a>).</p><p id="p-59">The authors of recent studies have identified numerous cryptic unstable transcripts in yeast (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Neil <em>et al.</em> 2009</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Xu <em>et al.</em> 2009</a>). Most of these transcripts were suggested to be byproducts of bidirectional transcription that originated from the same promoter but from distinct pre-initiation complexes (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Neil <em>et al.</em> 2009</a>). We observed little bidirectional transcription in either the NRO or total RNA samples. However, detection of cryptic unstable transcripts relied on the use of a strain defective in the nuclear exosome machinery (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Neil <em>et al.</em> 2009</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Xu <em>et al.</em> 2009</a>). Thus, it is possible that most of the antisense transcription was undetectable in the strain used in our study, which is competent for RNA turnover. For nondivergent promoters, with an average intergenic distance of approximately 342 bp (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dujon 1996</a>), a weak antisense signal arises at approximately â200 bp from the TSS, much upstream of the core promoter. Similarly, for divergent promoters, the antisense signal does not begin to accumulate until approximately â150 bp from the TSS. The distances at which antisense transcription is originated suggest that it likely results from the activity of a separate transcription pre-initiation complex assembled at a different core promoter.</p><p id="p-60">An important biological question is how cells respond and cope with rapid changes in their environment, such as exposure to elevated temperatures (heat shock). Several genome-wide studies revealed that this response involves an increase in the abundance of transcripts from genes that are collectively termed âheat shock-inducibleâ (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Gasch <em>et al.</em> 2000</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hahn <em>et al.</em> 2004</a>). Earlier studies of the <em>Drosophila</em> hsp70 gene found that posttranscriptional regulation that affects RNA stability plays a role in the observed increase of hsp70 RNA levels in response to heat shock (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Theodorakis and Morimoto 1987</a>). Other studies of heat shock-inducible genes in <em>Drosophila</em> and humans found rapid transcriptional activation after heat shock, suggesting that this activation also contributes to increased RNA levels of heat shock-inducible genes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mathur <em>et al.</em> 1994</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">OâBrien and Lis 1993</a>). However, in previous studies investigators have not assessed global changes in nascent transcription in response to heat shock. Here, we found that both transcriptional activation and RNA stabilization likely play a role in the increased abundance of RNAs after heat shock treatment. Most of the heat shock-inducible genes (~74%) showed both an increase in nascent transcription and an increase in RNA abundance. Only approximately 14% of these genes increased their RNA abundance without increasing nascent transcription, indicative of RNA stabilization as the mechanism.</p><p id="p-61">In summary, we used the NRO assay to provide a high-resolution view of transcriptional regulation in yeast. We applied this assay to examine the correlation between synthesis and abundance of transcripts, the distribution of polymerase activity along transcripts, and the genome-wide changes in yeast transcription in response to heat shock treatment. Further investigations may use similar approaches to characterize other modes of yeast transcriptional control under different growth conditions and in response to other environmental perturbations.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We thank John Stamatoyannopoulos and Steve Hahn for their comments on the manuscript, and Charlie Lee and Jay Shendure for their help with DNA sequencing. We thank Christine Queitsch for the <em>Arabidopsis thaliana</em> ELF3 construct.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000810/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Raw Illumina reads have been submitted to the GEO database at NCBI as series GSE33136.</p></li><li class="fn" id="fn-5"><p id="p-63"><em>Communicating editor: A. Gasch</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-08-03"><span class="received-label">Received </span>August 3, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-10-07"><span class="accepted-label">Accepted </span>October 7, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2011 McKinlay <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-5">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bengal</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aloni</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">A block of transcription elongation by RNA polymerase II at synthetic sites in vitro.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">9791</span>â<span class="cit-lpage">9798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBengal%26rft.auinit1%253DE.%26rft.volume%253D264%26rft.issue%253D17%26rft.spage%253D9791%26rft.epage%253D9798%26rft.atitle%253DA%2Bblock%2Bof%2Btranscription%2Belongation%2Bby%2BRNA%2Bpolymerase%2BII%2Bat%2Bsynthetic%2Bsites%2Bin%2Bvitro.%26rft_id%253Dinfo%253Apmid%252F2722878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNjQvMTcvOTc5MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.2" data-doi="10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachmann</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Designer deletion strains derived from <em>Saccharomyces cerevisiae</em> S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">115</span>â<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBrachmann%26rft.auinit1%253DC.%2BB.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D132%26rft.atitle%253DDesigner%2Bdeletion%2Bstrains%2Bderived%2Bfrom%2BSaccharomyces%2Bcerevisiae%2BS288C%253A%2Ba%2Buseful%2Bset%2Bof%2Bstrains%2Band%2Bplasmids%2Bfor%2BPCR-mediated%2Bgene%2Bdisruption%2Band%2Bother%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819980130%252914%253A2%253C115%253A%253AAID-YEA204%253E3.0.CO%253B2-2%26rft_id%253Dinfo%253Apmid%252F9483801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9483801&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071817800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.3" data-doi="10.1073/pnas.81.20.6344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeltsch</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambon</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Activation of pS2 gene transcription is a primary response to estrogen in the human breast cancer cell line MCF-7.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">6344</span>â<span class="cit-lpage">6348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBrown%26rft.auinit1%253DA.%2BM.%26rft.volume%253D81%26rft.issue%253D20%26rft.spage%253D6344%26rft.epage%253D6348%26rft.atitle%253DActivation%2Bof%2BpS2%2Bgene%2Btranscription%2Bis%2Ba%2Bprimary%2Bresponse%2Bto%2Bestrogen%2Bin%2Bthe%2Bhuman%2Bbreast%2Bcancer%2Bcell%2Bline%2BMCF-7.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.20.6344%26rft_id%253Dinfo%253Apmid%252F6593703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODEvMjAvNjM0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.4" data-doi="10.1016/j.molcel.2010.11.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carrillo Oesterreich</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preibisch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neugebauer</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Global analysis of nascent RNA reveals transcriptional pausing in terminal exons.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">571</span>â<span class="cit-lpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DCarrillo%2BOesterreich%26rft.auinit1%253DF.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D571%26rft.epage%253D581%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bnascent%2BRNA%2Breveals%2Btranscriptional%2Bpausing%2Bin%2Bterminal%2Bexons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.11.004%26rft_id%253Dinfo%253Apmid%252F21095587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21095587&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284988400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.5" data-doi="10.1371/journal.pone.0025328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castells-Roca</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">GarcÃ­a-MartÃ­nez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrero</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">BellÃ­</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Heat shock response in yeast involves changes in both transcription rates and mRNA stabilities.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D6%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.6" data-doi="10.1038/nature09652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchman</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Nascent transcript sequencing visualizes transcription at nucleotide resolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">469</span>: <span class="cit-fpage">368</span>â<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChurchman%26rft.auinit1%253DL.%2BS.%26rft.volume%253D469%26rft.issue%253D7330%26rft.spage%253D368%26rft.epage%253D373%26rft.atitle%253DNascent%2Btranscript%2Bsequencing%2Bvisualizes%2Btranscription%2Bat%2Bnucleotide%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09652%26rft_id%253Dinfo%253Apmid%252F21248844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21248844&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286385600045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.7" data-doi="10.1126/science.1162228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Core</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterfall</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lis</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Nascent RNA sequencing reveals widespread pausing and divergent initiation at human promoters.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">1845</span>â<span class="cit-lpage">1848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162228%26rft_id%253Dinfo%253Apmid%252F19056941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjIvNTkwOS8xODQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.8" data-doi="10.1016/0168-9525(96)10027-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">The yeast genome project: what did we learn?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">263</span>â<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D263%26rft.epage%253D270%26rft.atitle%253DThe%2Byeast%2Bgenome%2Bproject%253A%2Bwhat%2Bdid%2Bwe%2Blearn%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9525%252896%252910027-5%26rft_id%253Dinfo%253Apmid%252F8763498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9525(96)10027-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8763498&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UW03100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.9" data-doi="10.1186/1471-2105-10-48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinfeld</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakhini</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEden%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DGOrilla%253A%2Ba%2Btool%2Bfor%2Bdiscovery%2Band%2Bvisualization%2Bof%2Benriched%2BGO%2Bterms%2Bin%2Branked%2Bgene%2Blists.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-48%26rft_id%253Dinfo%253Apmid%252F19192299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19192299&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.10" data-doi="10.1093/nar/gkp542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedel</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolken</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruzsics</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koszinowski</span>  <span class="cit-name-given-names">U. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmer</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Conserved principles of mammalian transcriptional regulation revealed by RNA half-life.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp542%26rft_id%253Dinfo%253Apmid%252F19561200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561200&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.11" data-doi="10.1038/nature08449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuda</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ardehali</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lis</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Defining mechanisms that regulate RNA polymerase II transcription in vivo.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">186</span>â<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFuda%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D461%26rft.issue%253D7261%26rft.spage%253D186%26rft.epage%253D192%26rft.atitle%253DDefining%2Bmechanisms%2Bthat%2Bregulate%2BRNA%2Bpolymerase%2BII%2Btranscription%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08449%26rft_id%253Dinfo%253Apmid%252F19741698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19741698&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269654600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.12" data-doi="10.1016/j.molcel.2004.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Ortin</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genomic run-on evaluates transcription rates for all yeast genes and identifies gene regulatory mechanisms.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">303</span>â<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGarc%25C3%25ADa-Mart%25C3%25ADnez%26rft.auinit1%253DJ.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D303%26rft.epage%253D313%26rft.atitle%253DGenomic%2Brun-on%2Bevaluates%2Btranscription%2Brates%2Bfor%2Ball%2Byeast%2Bgenes%2Band%2Bidentifies%2Bgene%2Bregulatory%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.06.004%26rft_id%253Dinfo%253Apmid%252F15260981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15260981&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222958300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.13" data-doi="10.1093/nar/9.11.2589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gariglio</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambon</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span>â<span class="cit-article-title">Clustering of RNA polymerase B molecules in the 5â² moiety of the adult beta-globin gene of hen erythrocytes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2589</span>â<span class="cit-lpage">2598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGariglio%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D2589%26rft.epage%253D2598%26rft.atitle%253DClustering%2Bof%2BRNA%2Bpolymerase%2BB%2Bmolecules%2Bin%2Bthe%2B5%2527%2Bmoiety%2Bof%2Bthe%2Badult%2B%257Bbeta%257D-globin%2Bgene%2Bof%2Bhen%2Berythrocytes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F9.11.2589%26rft_id%253Dinfo%253Apmid%252F6269056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/9.11.2589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6269056&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LU42700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.14" data-doi="10.1091/mbc.11.12.4241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Harel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genomic expression programs in the response of yeast cells to environmental changes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">4241</span>â<span class="cit-lpage">4257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D4241%26rft.epage%253D4257%26rft.atitle%253DGenomic%2BExpression%2BPrograms%2Bin%2Bthe%2BResponse%2Bof%2BYeast%2BCells%2Bto%2BEnvironmental%2BChanges%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.12.4241%26rft_id%253Dinfo%253Apmid%252F11102521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMi80MjQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.15" data-doi="10.1128/MCB.24.12.5534-5547.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigull</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mnaimneh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pootoolal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genome-wide analysis of mRNA stability using transcription inhibitors and microarrays reveals posttranscriptional control of ribosome biogenesis factors.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5534</span>â<span class="cit-lpage">5547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGrigull%26rft.auinit1%253DJ.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D5534%26rft.epage%253D5547%26rft.atitle%253DGenome-Wide%2BAnalysis%2Bof%2BmRNA%2BStability%2BUsing%2BTranscription%2BInhibitors%2Band%2BMicroarrays%2BReveals%2BPosttranscriptional%2BControl%2Bof%2BRibosome%2BBiogenesis%2BFactors%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.12.5534-5547.2004%26rft_id%253Dinfo%253Apmid%252F15169913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMi81NTM0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.16" data-doi="10.1128/MCB.24.12.5249-5256.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiele</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genome-wide analysis of the biology of stress responses through heat shock transcription factor.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5249</span>â<span class="cit-lpage">5256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHahn%26rft.auinit1%253DJ.-S.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D5249%26rft.epage%253D5256%26rft.atitle%253DGenome-Wide%2BAnalysis%2Bof%2Bthe%2BBiology%2Bof%2BStress%2BResponses%2Bthrough%2BHeat%2BShock%2BTranscription%2BFactor%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.12.5249-5256.2004%26rft_id%253Dinfo%253Apmid%252F15169889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMi81MjQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">Separation and partial characterization of three functional steps in transcription initiation by human RNA polymerase II.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">260</span>: <span class="cit-fpage">8163</span>â<span class="cit-lpage">8172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DHawley%26rft.auinit1%253DD.%2BK.%26rft.volume%253D260%26rft.issue%253D13%26rft.spage%253D8163%26rft.epage%253D8172%26rft.atitle%253DSeparation%2Band%2Bpartial%2Bcharacterization%2Bof%2Bthree%2Bfunctional%2Bsteps%2Bin%2Btranscription%2Binitiation%2Bby%2Bhuman%2BRNA%2Bpolymerase%2BII%26rft_id%253Dinfo%253Apmid%252F2409080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNjAvMTMvODE2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.18" data-doi="10.1016/0092-8674(81)90326-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hereford</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osley</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludwig</span>  <span class="cit-name-given-names">T. R.</span>  <span class="cit-name-suffix">2nd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span>â<span class="cit-article-title">Cell-cycle regulation of yeast histone mRNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">367</span>â<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHereford%26rft.auinit1%253DL.%2BM.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D367%26rft.epage%253D375%26rft.atitle%253DCell-cycle%2Bregulation%2Bof%2Byeast%2Bhistone%2BmRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252881%252990326-3%26rft_id%253Dinfo%253Apmid%252F7016339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(81)90326-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7016339&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LR43400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.19" data-doi="10.1126/science.1168978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">218</span>â<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.20" data-doi="10.1093/nar/27.15.3173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keene</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landick</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">London</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Transcriptional pause, arrest and termination sites for RNA polymerase II in mammalian N- and c-myc genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3173</span>â<span class="cit-lpage">3182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.15.3173%26rft_id%253Dinfo%253Apmid%252F10454615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.15.3173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10454615&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081832900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.21" data-doi="10.1186/1471-2164-10-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefrancois</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Euskirchen</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auerbach</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozowsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Efficient yeast ChIP-Seq using multiplex short-read DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLefrancois%26rft.auinit1%253DP.%26rft.volume%253D10%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DEfficient%2Byeast%2BChIP-Seq%2Busing%2Bmultiplex%2Bshort-read%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-37%26rft_id%253Dinfo%253Apmid%252F19159457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19159457&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.22" data-doi="10.1038/nmeth.1491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adiconis</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Comprehensive comparative analysis of strand-specific RNA sequencing methods.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">709</span>â<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLevin%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D709%26rft.epage%253D715%26rft.atitle%253DComprehensive%2Bcomparative%2Banalysis%2Bof%2Bstrand-specific%2BRNA%2Bsequencing%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1491%26rft_id%253Dinfo%253Apmid%252F20711195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20711195&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281429200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenleaf</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Analyses of promoter-proximal pausing by RNA polymerase II on the hsp70 heat shock gene promoter in a Drosophila nuclear extract.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">5433</span>â<span class="cit-lpage">5443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D5433%26rft.epage%253D5443%26rft.atitle%253DAnalyses%2Bof%2Bpromoter-proximal%2Bpausing%2Bby%2BRNA%2Bpolymerase%2BII%2Bon%2Bthe%2Bhsp70%2Bheat%2Bshock%2Bgene%2Bpromoter%2Bin%2Ba%2BDrosophila%2Bnuclear%2Bextract%26rft_id%253Dinfo%253Apmid%252F8816456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMC81NDMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.24" data-doi="10.1073/pnas.91.18.8695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathur</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sistonen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarge</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Deficient induction of human hsp70 heat shock gene transcription in Y79 retinoblastoma cells despite activation of heat shock factor 1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">8695</span>â<span class="cit-lpage">8699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMathur%26rft.auinit1%253DS.%2BK.%26rft.volume%253D91%26rft.issue%253D18%26rft.spage%253D8695%26rft.epage%253D8699%26rft.atitle%253DDeficient%2Binduction%2Bof%2Bhuman%2Bhsp70%2Bheat%2Bshock%2Bgene%2Btranscription%2Bin%2BY79%2Bretinoblastoma%2Bcells%2Bdespite%2Bactivation%2Bof%2Bheat%2Bshock%2Bfactor%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.18.8695%26rft_id%253Dinfo%253Apmid%252F8078944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTEvMTgvODY5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.25" data-doi="10.1126/science.1158441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raha</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1344</span>â<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158441%26rft_id%253Dinfo%253Apmid%252F18451266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4MS8xMzQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.26" data-doi="10.1105/tpc.107.055046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narsai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millar</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O'Toole</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Small</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genome-wide analysis of mRNA decay rates and their determinants in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3418</span>â<span class="cit-lpage">3436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.055046%26rft_id%253Dinfo%253Apmid%252F18024567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8zNDE4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.27" data-doi="10.1038/nature07747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malabat</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">d'Aubenton-Carafa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Widespread bidirectional promoters are the major source of cryptic transcripts in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">1038</span>â<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNeil%26rft.auinit1%253DH.%26rft.volume%253D457%26rft.issue%253D7232%26rft.spage%253D1038%26rft.epage%253D1042%26rft.atitle%253DWidespread%2Bbidirectional%2Bpromoters%2Bare%2Bthe%2Bmajor%2Bsource%2Bof%2Bcryptic%2Btranscripts%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07747%26rft_id%253Dinfo%253Apmid%252F19169244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19169244&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263425400048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.28" data-doi="10.1128/MCB.13.6.3456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">OâBrien</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lis</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Rapid changes in Drosophila transcription after an instantaneous heat shock.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3456</span>â<span class="cit-lpage">3463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DO%2527Brien%26rft.auinit1%253DT%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D3456%26rft.epage%253D3463%26rft.atitle%253DRapid%2Bchanges%2Bin%2BDrosophila%2Btranscription%2Bafter%2Ban%2Binstantaneous%2Bheat%2Bshock.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.13.6.3456%26rft_id%253Dinfo%253Apmid%252F8497261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEzLzYvMzQ1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.29" data-doi="10.1016/S1097-2765(00)80103-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palangat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keene</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landick</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Transcriptional pausing at +62 of the HIV-1 nascent RNA modulates formation of the TAR RNA structure.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1033</span>â<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DPalangat%26rft.auinit1%253DM.%26rft.volume%253D1%26rft.issue%253D7%26rft.spage%253D1033%26rft.epage%253D1042%26rft.atitle%253DTranscriptional%2Bpausing%2Bat%2B%252B62%2Bof%2Bthe%2BHIV-1%2Bnascent%2BRNA%2Bmodulates%2Bformation%2Bof%2Bthe%2BTAR%2BRNA%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252980103-3%26rft_id%253Dinfo%253Apmid%252F9651586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)80103-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9651586&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074389200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.30" data-doi="10.1093/nar/gkp596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkhomchuk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amstislavskiy</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banaru</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Transcriptome analysis by strand-specific sequencing of complementary DNA.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">e123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp596%26rft_id%253Dinfo%253Apmid%252F19620212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19620212&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patrone</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puppo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cusano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scaranari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceccherini</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Nuclear run-on assay using biotin labeling, magnetic bead capture and analysis by fluorescence-based RT-PCR.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1012</span>â<span class="cit-lpage">1014, 1016â1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D29%26rft.spage%253D1012%26rft_id%253Dinfo%253Apmid%252F11084863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11084863&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165274900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.32" data-doi="10.1371/journal.pgen.1000614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelechano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jimeno-Gonzalez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Gil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Ortin</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Regulon-specific control of transcription elongation across the yeast genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPelechano%26rft.auinit1%253DV.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De1000614%26rft.epage%253De1000614%26rft.atitle%253DRegulon-specific%2Bcontrol%2Bof%2Btranscription%2Belongation%2Bacross%2Bthe%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000614%26rft_id%253Dinfo%253Apmid%252F19696888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19696888&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.33" data-doi="10.1016/j.molcel.2005.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Radonjic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrau</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lijnzaad</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemmeren</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kockelkorn</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Genome-wide analyses reveal RNA polymerase II located upstream of genes poised for rapid response upon <em>S. cerevisiae</em> stationary phase exit.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">171</span>â<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRadonjic%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D171%26rft.epage%253D183%26rft.atitle%253DGenome-wide%2Banalyses%2Breveal%2BRNA%2Bpolymerase%2BII%2Blocated%2Bupstream%2Bof%2Bgenes%2Bpoised%2Bfor%2Brapid%2Bresponse%2Bupon%2BS.%2Bcerevisiae%2Bstationary%2Bphase%2Bexit.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.03.010%26rft_id%253Dinfo%253Apmid%252F15837421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15837421&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228437700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.34" data-doi="10.1073/pnas.90.17.7923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lis</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">In vivo transcriptional pausing and cap formation on three Drosophila heat shock genes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">7923</span>â<span class="cit-lpage">7927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRasmussen%26rft.auinit1%253DE.%2BB.%26rft.volume%253D90%26rft.issue%253D17%26rft.spage%253D7923%26rft.epage%253D7927%26rft.atitle%253DIn%2Bvivo%2Btranscriptional%2Bpausing%2Band%2Bcap%2Bformation%2Bon%2Bthree%2BDrosophila%2Bheat%2Bshock%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.17.7923%26rft_id%253Dinfo%253Apmid%252F8367444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTAvMTcvNzkyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.35" data-doi="10.1093/nar/gkq215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Gil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelechano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz-Centeno Mde</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geli</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The distribution of active RNA polymerase II along the transcribed region is gene-specific and controlled by elongation factors.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">4651</span>â<span class="cit-lpage">4664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq215%26rft_id%253Dinfo%253Apmid%252F20385590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20385590&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280922400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.36" data-doi="10.1128/MCB.10.11.6041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rougvie</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lis</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Postinitiation transcriptional control in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">6041</span>â<span class="cit-lpage">6045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DRougvie%26rft.auinit1%253DA%2BE%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D6041%26rft.epage%253D6045%26rft.atitle%253DPostinitiation%2Btranscriptional%2Bcontrol%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.11.6041%26rft_id%253Dinfo%253Apmid%252F2172790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMS82MDQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.37" data-doi="10.1101/gad.1235904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">R. J.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belotserkovskaya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Elongation by RNA polymerase II: the short and long of it.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2437</span>â<span class="cit-lpage">2468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSims%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D18%26rft.issue%253D20%26rft.spage%253D2437%26rft.epage%253D2468%26rft.atitle%253DElongation%2Bby%2BRNA%2Bpolymerase%2BII%253A%2Bthe%2Bshort%2Band%2Blong%2Bof%2Bit%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1235904%26rft_id%253Dinfo%253Apmid%252F15489290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzIwLzI0MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.38" data-doi="10.1128/MCB.7.12.4357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theodorakis</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morimoto</span>  <span class="cit-name-given-names">R. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">Posttranscriptional regulation of hsp70 expression in human cells: effects of heat shock, inhibition of protein synthesis, and adenovirus infection on translation and mRNA stability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">4357</span>â<span class="cit-lpage">4368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTheodorakis%26rft.auinit1%253DN%2BG%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D4357%26rft.epage%253D4368%26rft.atitle%253DPosttranscriptional%2Bregulation%2Bof%2Bhsp70%2Bexpression%2Bin%2Bhuman%2Bcells%253A%2Beffects%2Bof%2Bheat%2Bshock%252C%2Binhibition%2Bof%2Bprotein%2Bsynthesis%252C%2Band%2Badenovirus%2Binfection%2Bon%2Btranslation%2Band%2BmRNA%2Bstability.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.7.12.4357%26rft_id%253Dinfo%253Apmid%252F3437893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjcvMTIvNDM1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNy81NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.39" data-doi="10.1101/gr.084970.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venters</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A canonical promoter organization of the transcription machinery and its regulators in the Saccharomyces genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">360</span>â<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.084970.108%26rft_id%253Dinfo%253Apmid%252F19124666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvMzYwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS83LzU0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.40" data-doi="10.1073/pnas.092538799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschlag</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Precision and functional specificity in mRNA decay.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">5860</span>â<span class="cit-lpage">5865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.092538799%26rft_id%253Dinfo%253Apmid%252F11972065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS85LzU4NjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzcvNTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.41" data-doi="10.1038/nature07728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagneur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perocchi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clauder-Munster</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Bidirectional promoters generate pervasive transcription in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">1033</span>â<span class="cit-lpage">1037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXu%26rft.auinit1%253DZ.%26rft.volume%253D457%26rft.issue%253D7232%26rft.spage%253D1033%26rft.epage%253D1037%26rft.atitle%253DBidirectional%2Bpromoters%2Bgenerate%2Bpervasive%2Btranscription%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07728%26rft_id%253Dinfo%253Apmid%252F19169243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19169243&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263425400047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.7.549.42" data-doi="10.1186/gb-2010-11-8-r87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfiffner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adiconis</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gnirke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Strand-specific RNA sequencing reveals extensive regulated long antisense transcripts that are conserved across yeast species.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DYassour%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR87%26rft.epage%253DR87%26rft.atitle%253DStrand-specific%2BRNA%2Bsequencing%2Breveals%2Bextensive%2Bregulated%2Blong%2Bantisense%2Btranscripts%2Bthat%2Bare%2Bconserved%2Bacross%2Byeast%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r87%26rft_id%253Dinfo%253Apmid%252F20796282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20796282&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F7%2F549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/7/549.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/7/539" title="Genome Evolution in the Eremothecium Clade of the Saccharomyces Complex Revealed by Comparative Genomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/7/559" title="Unexpected Diversity of Chloroplast Noncoding RNAs as Revealed by Deep Sequencing of the Arabidopsis Transcriptome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/7"><span>Volume 1 Issue 7, December 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/7.cover.gif?itok=8vFpWSyK" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/7/549" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8jACz7UMiQTE0ioNd7s1Au3_1Da-oLTDucljrjIv1kg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/7/549" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/7/549" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/7/549" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Analysis of Nascent Transcription in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459209" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Z2zshkZPr7NzU6pBJub_8VX3FGPmR_EyWKrbjxKHfrU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/7/549.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/7/549" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-77d6PwMruqSXeBHmvGSfrXC-yBsb6sA2JnTrp5QfsuQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/7/549" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/7/549.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459209" id="node459209--2" data-pisa="ggg;1/7/549" data-pisa-master="ggg;g3.111.000810" data-apath="/ggg/1/7/549.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Nascent Transcription in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anastasia McKinlay</span>, <span class="highwire-citation-author" data-delta="1">Carlos L. Araya</span> and <span class="highwire-citation-author" data-delta="2">Stanley Fields</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">549-558;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000810 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459209--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anastasia McKinlay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Medicine</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasia%2BMcKinlay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKinlay%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasia%2BMcKinlay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carlos L. Araya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BL.%2BAraya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Araya%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BL.%2BAraya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stanley Fields </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Medicine</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Howard Hughes Medical Institute, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stanley%2BFields%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fields%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStanley%2BFields%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fields@u.washington.edu" class="" data-icon-position="" data-hide-link-title="0">fields@u.washington.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/7/549" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/7/549&amp;title=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/7/549&amp;title=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/7/549&amp;title=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/7/549&amp;text=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/7/549&amp;title=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/7/549&amp;t=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/7/549&amp;title=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/7/549&amp;title=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459209/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459209" id="node459209--3" data-pisa="ggg;1/7/549" data-pisa-master="ggg;g3.111.000810" data-apath="/ggg/1/7/549.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Nascent Transcription in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anastasia McKinlay</span>, <span class="highwire-citation-author" data-delta="1">Carlos L. Araya</span> and <span class="highwire-citation-author" data-delta="2">Stanley Fields</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">549-558;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000810 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459209--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anastasia McKinlay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Medicine</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasia%2BMcKinlay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKinlay%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasia%2BMcKinlay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carlos L. Araya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BL.%2BAraya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Araya%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BL.%2BAraya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Stanley Fields </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Medicine</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Howard Hughes Medical Institute, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stanley%2BFields%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fields%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStanley%2BFields%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fields@u.washington.edu" class="" data-icon-position="" data-hide-link-title="0">fields@u.washington.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=549&amp;rft.epage=558&amp;rft.atitle=Genome-Wide%20Analysis%20of%20Nascent%20Transcription%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=1&amp;rft.issue=7&amp;rft.date=2011-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=McKinlay%2C+Anastasia&amp;rft.au=Araya%2C+Carlos+L.&amp;rft.au=Fields%2C+Stanley"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459209/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459209/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459209/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459209/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459209/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459209/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459209/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459209/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459209/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459209/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459209/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459209/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496056" id="node496056" data-pisa="ggg;g3.120.401188v1" data-pisa-master="ggg;g3.120.401188" data-apath="/ggg/early/2020/03/27/g3.120.401188.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401188" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">MYCs and PIFs Act Independently in Arabidopsis Growth Regulation</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496055" id="node496055" data-pisa="ggg;g3.120.401106v1" data-pisa-master="ggg;g3.120.401106" data-apath="/ggg/early/2020/03/27/g3.120.401106.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401106" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identifying Small Molecules That Promote Quasipalindrome-Associated Template-Switch Mutations in <em>Escherichia coli</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496053" id="node496053" data-pisa="ggg;g3.120.401160v1" data-pisa-master="ggg;g3.120.401160" data-apath="/ggg/early/2020/03/27/g3.120.401160.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401160" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/7/549" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/7/549.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/7/549.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/7/549.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/7/549.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
