<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/1/43" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/1/43.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/1/43.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/1/43.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/1/43.atom" />
<meta name="HW.pisa" content="ggg;2/1/43" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome" />
<meta name="DC.Identifier" content="10.1534/g3.111.001552" />
<meta name="DC.Date" content="2012-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Yu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic linkage maps play fundamental roles in understanding genome structure, explaining genome formation events during evolution, and discovering the genetic bases of important traits. A high-density cotton ( Gossypium spp.) genetic map was developed using representative sets of simple sequence repeat (SSR) and the first public set of single nucleotide polymorphism (SNP) markers to genotype 186 recombinant inbred lines (RILs) derived from an interspecific cross between Gossypium hirsutum L. (TM-1) and G. barbadense L. (3-79). The genetic map comprised 2072 loci (1825 SSRs and 247 SNPs) and covered 3380 centiMorgan (cM) of the cotton genome (AD) with an average marker interval of 1.63 cM. The allotetraploid cotton genome produced equivalent recombination frequencies in its two subgenomes (At and Dt). Of the 2072 loci, 1138 (54.9%) were mapped to 13 At-subgenome chromosomes, covering 1726.8 cM (51.1%), and 934 (45.1%) mapped to 13 Dt-subgenome chromosomes, covering 1653.1 cM (48.9%). The genetically smallest homeologous chromosome pair was Chr. 04 (A04) and 22 (D04), and the largest was Chr. 05 (A05) and 19 (D05). Duplicate loci between and within homeologous chromosomes were identified that facilitate investigations of chromosome translocations. The map augments evidence of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 versus segmental homeologs 14 and 17 as centromeric regions show homeologous between Chr. 02 (A02) and 17 (D02), as well as between Chr. 03 (A03) and 14 (D03). This research represents an important foundation for studies on polyploid cottons, including germplasm characterization, gene discovery, and genome sequence assembly." />
<meta name="DC.Contributor" content="John Z. Yu" />
<meta name="DC.Contributor" content="Russell J. Kohel" />
<meta name="DC.Contributor" content="David D. Fang" />
<meta name="DC.Contributor" content="Jaemin Cho" />
<meta name="DC.Contributor" content="Allen Van Deynze" />
<meta name="DC.Contributor" content="Mauricio Ulloa" />
<meta name="DC.Contributor" content="Steven M. Hoffman" />
<meta name="DC.Contributor" content="Alan E. Pepper" />
<meta name="DC.Contributor" content="David M. Stelly" />
<meta name="DC.Contributor" content="Johnie N. Jenkins" />
<meta name="DC.Contributor" content="Sukumar Saha" />
<meta name="DC.Contributor" content="Siva P. Kumpatla" />
<meta name="DC.Contributor" content="Manali R. Shah" />
<meta name="DC.Contributor" content="William V. Hugie" />
<meta name="DC.Contributor" content="Richard G. Percy" />
<meta name="DC.Contributor" content="J.  Wendel" />
<meta name="article:published_time" content="2012-01-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic linkage maps play fundamental roles in understanding genome structure, explaining genome formation events during evolution, and discovering the genetic bases of important traits. A high-density cotton (&lt;i&gt;Gossypium&lt;/i&gt; spp.) genetic map was developed using representative sets of simple sequence repeat (SSR) and the first public set of single nucleotide polymorphism (SNP) markers to genotype 186 recombinant inbred lines (RILs) derived from an interspecific cross between &lt;i&gt;Gossypium hirsutum&lt;/i&gt; L. (TM-1) and &lt;i&gt;G. barbadense&lt;/i&gt; L. (3-79). The genetic map comprised 2072 loci (1825 SSRs and 247 SNPs) and covered 3380 centiMorgan (cM) of the cotton genome (AD) with an average marker interval of 1.63 cM. The allotetraploid cotton genome produced equivalent recombination frequencies in its two subgenomes (At and Dt). Of the 2072 loci, 1138 (54.9%) were mapped to 13 At-subgenome chromosomes, covering 1726.8 cM (51.1%), and 934 (45.1%) mapped to 13 Dt-subgenome chromosomes, covering 1653.1 cM (48.9%). The genetically smallest homeologous chromosome pair was Chr. 04 (A04) and 22 (D04), and the largest was Chr. 05 (A05) and 19 (D05). Duplicate loci between and within homeologous chromosomes were identified that facilitate investigations of chromosome translocations. The map augments evidence of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 versus segmental homeologs 14 and 17 as centromeric regions show homeologous between Chr. 02 (A02) and 17 (D02), as well as between Chr. 03 (A03) and 14 (D03). This research represents an important foundation for studies on polyploid cottons, including germplasm characterization, gene discovery, and genome sequence assembly.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/01/01" />
<meta name="citation_mjid" content="ggg;2/1/43" />
<meta name="citation_id" content="2/1/43" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/1/43" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/1/43.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/1/43.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/1/43.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001552" />
<meta name="citation_pmid" content="22384381" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="43" />
<meta name="citation_lastpage" content="58" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="John Z. Yu" />
<meta name="citation_author_institution" content="USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845" />
<meta name="citation_author_email" content="john.yu@ars.usda.gov" />
<meta name="citation_author" content="Russell J. Kohel" />
<meta name="citation_author_institution" content="USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845" />
<meta name="citation_author" content="David D. Fang" />
<meta name="citation_author_institution" content="USDA–ARS, Cotton Fiber Bioscience Research Unit, Southern Regional Research Center, New Orleans, Louisiana 70124" />
<meta name="citation_author" content="Jaemin Cho" />
<meta name="citation_author_institution" content="USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845" />
<meta name="citation_author" content="Allen Van Deynze" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Mauricio Ulloa" />
<meta name="citation_author_institution" content="USDA–ARS, WICSRU, Shafter Cotton Research Station, Shafter, California 93263" />
<meta name="citation_author" content="Steven M. Hoffman" />
<meta name="citation_author_institution" content="USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845" />
<meta name="citation_author_institution" content="Department of Biology and" />
<meta name="citation_author" content="Alan E. Pepper" />
<meta name="citation_author_institution" content="Department of Biology and" />
<meta name="citation_author" content="David M. Stelly" />
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas 77843" />
<meta name="citation_author" content="Johnie N. Jenkins" />
<meta name="citation_author_institution" content="USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762" />
<meta name="citation_author" content="Sukumar Saha" />
<meta name="citation_author_institution" content="USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762" />
<meta name="citation_author" content="Siva P. Kumpatla" />
<meta name="citation_author_institution" content="Dow AgroSciences LLC, Indianapolis, Indiana 46268" />
<meta name="citation_author" content="Manali R. Shah" />
<meta name="citation_author_institution" content="Dow AgroSciences LLC, Indianapolis, Indiana 46268" />
<meta name="citation_author" content="William V. Hugie" />
<meta name="citation_author_institution" content="Monsanto Company, St. Louis, Missouri 63167" />
<meta name="citation_author" content="Richard G. Percy" />
<meta name="citation_author_institution" content="USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845" />
<meta name="citation_editor" content="J.  Wendel" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=I. Y. Abdurakhmonov;citation_author=R. J. Kohel;citation_author=J. Z. Yu;citation_author=A. E. Pepper;citation_author=A. A. Abdullaev;citation_title=Molecular diversity and association mapping of fiber quality traits in exotic G. hirsutum L. germplasm.;citation_pages=478-487;citation_volume=92;citation_year=2008;citation_issue=6;citation_pmid=18801424;citation_doi=10.1016/j.ygeno.2008.07.013" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. An;citation_author=S. Saha;citation_author=J. N. Jenkins;citation_author=D. P. Ma;citation_author=B. E. Scheffler;citation_title=Cotton (Gossypium spp.) R2R3-MYB transcription factors SNP identification, phylogenomic characterization, chromosome localization, and linkage mapping.;citation_pages=1015-1026;citation_volume=116;citation_year=2008;citation_issue=7;citation_pmid=18338155;citation_doi=10.1007/s00122-008-0732-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. O. Beasley;citation_title=Meiotic chromosome behavior in species, species hybrids, haploids and induced polyploids of Gossypium.;citation_pages=25-54;citation_volume=27;citation_year=1942;citation_pmid=17247031" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Blenda;citation_author=J. Scheffler;citation_author=B. Scheffler;citation_author=M. Palmer;citation_author=J. M. Lacape;citation_title=CMD: a Cotton Microsatellite Database resource for Gossypium genomics.;citation_pages=132-132;citation_volume=7;citation_year=2006;citation_pmid=16737546;citation_doi=10.1186/1471-2164-7-132" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Bowers;citation_author=B. A. Chapman;citation_author=J. K. Rong;citation_author=A. H. Paterson;citation_title=Unravelling angiosperm genome evolution by phylogenetic analysis of chromosomal duplication events.;citation_pages=433-438;citation_volume=422;citation_year=2003;citation_issue=6930;citation_pmid=12660784;citation_doi=10.1038/nature01521" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. S. Brown;citation_title=Identification of the chromosomes of Gossypium hirsutum L. by means of translocations;citation_pages=266-274;citation_volume=71;citation_year=1980;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=J. Cotton Sci.;citation_author=C. Brubaker;citation_author=R. Cantrell;citation_author=M. Giband;citation_author=B. Lyon;citation_author=T. Wilkins;citation_title=Letter to Journal of Cotton Science Community from the Steering Committee of the International Cotton Genome Initiative (ICGI).;citation_pages=149-151;citation_volume=4;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Burr;citation_author=F. A. Burr;citation_author=K. H. Thompson;citation_author=M. C. Albertson;citation_author=C. W. Stuber;citation_title=Gene-mapping with recombinant inbreds in maize.;citation_pages=519-526;citation_volume=118;citation_year=1988;citation_pmid=3366363" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. A. Causse;citation_author=T. M. Fulton;citation_author=Y. G. Cho;citation_author=S. N. Ahn;citation_author=J. Chunwongse;citation_title=Saturated molecular map of the rice genome based on an interspecific backcross population.;citation_pages=1251-1274;citation_volume=138;citation_year=1994;citation_pmid=7896104" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=Z. J. Chen;citation_author=B. E. Scheffler;citation_author=E. Dennis;citation_author=B. Triplett;citation_author=T. Zhang;citation_title=Toward Sequencing Cotton (Gossypium) Genomes;citation_pages=1303-1310;citation_volume=145;citation_year=2007;citation_issue=4;citation_pmid=18056866;citation_doi=10.1104/pp.107.107672" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=N. D. Dighe;citation_author=A. F. Robinson;citation_author=A. A. Bell;citation_author=M. A. Menz;citation_author=R. G. Cantrell;citation_title=Linkage Mapping of Resistance to Reniform Nematode in Cotton following Introgression from Gossypium longicalyx (Hutch. &amp; Lee);citation_pages=1151-1164;citation_volume=49;citation_year=2009;citation_issue=4;citation_doi=10.2135/cropsci2008.03.0129" />
<meta name="citation_reference" content="citation_title=Advances in Genetics;citation_title=Genetics, cytology, and evolution of Gossypium;citation_author=J. E. Endrizzi;citation_author=E. L. Turcotte;citation_author=R. J. Kohel;citation_pages=271-375;citation_year=1985" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=J. E. Frelichowski;citation_author=M. B. Palmer;citation_author=D. Main;citation_author=J. P. Tomkins;citation_author=R. G. Cantrell;citation_title=Cotton genome mapping with new microsatellites from Acala 'Maxxa' BAC-ends.;citation_pages=479-491;citation_volume=275;citation_year=2006;citation_issue=5;citation_pmid=16501995;citation_doi=10.1007/s00438-006-0106-z" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=M. W. Ganal;citation_author=T. Altmann;citation_author=M. S. Röder;citation_title=SNP identification in crop plants.;citation_pages=211-217;citation_volume=12;citation_year=2009;citation_issue=2;citation_pmid=19186095;citation_doi=10.1016/j.pbi.2008.12.009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Gao;citation_author=Z. J. Chen;citation_author=J. Z. Yu;citation_author=D. Raska;citation_author=R. J. Kohel;citation_title=Wide-cross whole-genome radiation hybrid mapping of cotton (Gossypium hirsutum L.).;citation_pages=1317-1329;citation_volume=167;citation_year=2004;citation_pmid=15280245" />
<meta name="citation_reference" content="citation_journal_title=Genet. Genomics;citation_author=W. Gao;citation_author=Z. J. Chen;citation_author=J. Z. Yu;citation_author=R. J. Kohel;citation_author=J. E. Womack;citation_title=Wide-cross whole-genome radiation hybrid mapping of the cotton (Gossypium barbadense L.) genome. Mol.;citation_pages=105-113;citation_volume=275;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Z. Guo;citation_author=C. P. Cai;citation_author=C. B. Wang;citation_author=Z. G. Han;citation_author=X. L. Song;citation_title=A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in Gossypium.;citation_pages=527-541;citation_volume=176;citation_year=2007;citation_pmid=17409069;citation_doi=10.1534/genetics.107.070375" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=Y. F. Guo;citation_author=S. Saha;citation_author=J. Z. Yu;citation_author=J. N. Jenkins;citation_author=R. J. Kohel;citation_title=BAC-derived SSR markers chromosome locations in cotton.;citation_pages=361-370;citation_volume=161;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=O. Gutiérrez;citation_author=D. Stelly;citation_author=S. Saha;citation_author=J. Jenkins;citation_author=J. McCarty;citation_title=Integrative placement and orientation of non-redundant SSR loci in cotton linkage groups by deficiency analysis.;citation_pages=693-707;citation_volume=23;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. B. Haldane;citation_author=C. H. Waddington;citation_title=INBREEDING AND LINKAGE;citation_pages=357-374;citation_volume=16;citation_year=1931;citation_issue=4;citation_pmid=17246626" />
<meta name="citation_reference" content="citation_journal_title=J. Cotton Sci.;citation_author=S. M. Hoffman;citation_author=J. Z. Yu;citation_author=D. S. Grum;citation_author=J. Xiao;citation_author=R. J. Kohel;citation_title=Identification of 700 new microsatellite loci from cotton (G. hirsutum L.).;citation_pages=208-241;citation_volume=11;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. N. Jenkins;citation_author=J. Wu;citation_author=J. C. McCarty;citation_author=S. Saha;citation_author=O. A. Gutierrez;citation_title=Genetic Effects of Thirteen Gossypium barbadense L. Chromosome Substitution Lines in Topcrosses with Upland Cotton Cultivars: I. Yield and Yield Components;citation_pages=1169-1178;citation_volume=46;citation_year=2006;citation_issue=3;citation_doi=10.2135/cropsci2005.08-0269" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. N. Jenkins;citation_author=J. Wu;citation_author=J. C. McCarty;citation_author=S. Saha;citation_author=O. A. Gutierrez;citation_title=Genetic Effects of Thirteen Gossypium barbadense L. Chromosome Substitution Lines in Topcrosses with Upland Cotton Cultivars: II. Fiber Quality Traits;citation_pages=561-570;citation_volume=47;citation_year=2007;citation_issue=2;citation_doi=10.2135/cropsci2006.06.0396" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. X. Jiang;citation_author=R. J. Wright;citation_author=K. M. El-Zik;citation_author=A. H. Paterson;citation_title=Polyploid formation created unique avenues for response to selection in Gossypium (cotton);citation_pages=4419-4424;citation_volume=95;citation_year=1998;citation_issue=8;citation_pmid=9539752;citation_doi=10.1073/pnas.95.8.4419" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=H. J. Kim;citation_author=B. A. Triplett;citation_title=Cotton Fiber Growth in Planta and in Vitro. Models for Plant Cell Elongation and Cell Wall Biogenesis;citation_pages=1361-1366;citation_volume=127;citation_year=2001;citation_issue=4;citation_pmid=11743074;citation_doi=10.1104/pp.127.4.1361" />
<meta name="citation_reference" content="citation_journal_title=L. Crop Sci.;citation_author=R. J. Kohel;citation_author=T. R. Richmond;citation_author=C. F. Lewis;citation_title=Texas Marker-1. Description of a genetic standard for Gossypium hirsutum;citation_pages=670-671;citation_volume=10;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=R. J. Kohel;citation_author=J. Z. Yu;citation_author=Y.-H. Park;citation_author=G. Lazo;citation_title=Molecular mapping and characterization of traits controlling fiber quality in cotton.;citation_pages=163-172;citation_volume=121;citation_year=2001;citation_doi=10.1023/A:1012263413418" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. D. Kosambi;citation_title=The estimation of map distances from recombination values.;citation_pages=172-175;citation_volume=12;citation_year=1944;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. M. Lacape;citation_author=T. B. Nguyen;citation_author=S. Thibivilliers;citation_author=B. Bojinov;citation_author=B. Courtois;citation_title=A combined RFLP-SSR-AFLP map of tetraploid cotton based on a Gossypium hirsutum x Gossypium barbadense backcross population.;citation_pages=612-626;citation_volume=46;citation_year=2003;citation_issue=4;citation_pmid=12897870;citation_doi=10.1139/g03&amp;ndash;050" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. Lacape;citation_author=J. Jacobs;citation_author=T. Arioli;citation_author=R. Derijcker;citation_author=N. Forestier-Chiron;citation_title=A new interspecific, Gossypium hirsutum x G. barbadense, RIL population: towards a unified consensus linkage map of tetraploid cotton.;citation_pages=281-292;citation_volume=119;citation_year=2009;citation_issue=2;citation_pmid=19387609;citation_doi=10.1007/s00122-009-1037-y" />
<meta name="citation_reference" content="citation_title=Cotton, Agronomy Monograph 24;citation_title=Cotton as a world crop;citation_author=J. A. Lee;citation_pages=1-25;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Lin;citation_author=G. Pierce;citation_author=J. Bowers;citation_author=J. Estill;citation_author=R. Compton;citation_title=A draft physical map of a D-genome cotton species (Gossypium raimondii).;citation_pages=395-395;citation_volume=11;citation_year=2010;citation_pmid=20569427;citation_doi=10.1186/1471-2164-11-395" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=S. Liu;citation_author=S. Saha;citation_author=D. Stelly;citation_author=B. Burr;citation_author=R. G. Cantrell;citation_title=Chromosomal assignment of microsatellite loci in cotton;citation_pages=326-332;citation_volume=91;citation_year=2000;citation_issue=4;citation_pmid=10912681;citation_doi=10.1093/jhered/91.4.326" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=E. Mace;citation_author=J.-F. Rami;citation_author=S. Bouchet;citation_author=P. Klein;citation_author=R. Klein;citation_title=A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers.;citation_pages=13-13;citation_volume=9;citation_year=2009;citation_pmid=19171067;citation_doi=10.1186/1471-2229-9-13" />
<meta name="citation_reference" content="citation_title=Cotton. Monograph series Agronomy No. 24;citation_title=Breeding;citation_author=G. A. Niles;citation_author=C. V. Feaster;citation_pages=201-231;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=Y. H. Park;citation_author=M. S. Alabady;citation_author=M. Ulloa;citation_author=B. Sickler;citation_author=T. A. Wilkins;citation_title=Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line cotton population.;citation_pages=428-441;citation_volume=274;citation_year=2005;citation_issue=4;citation_pmid=16187061;citation_doi=10.1007/s00438-005-0037-0" />
<meta name="citation_reference" content="citation_journal_title=Cotton Sci.;citation_author=A. H. Paterson;citation_title=Sequencing the cotton genomes—Gossypium spp.;citation_pages=3;citation_volume=20;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. E. Percival;citation_author=R. J. Kohel;citation_title=Distribution, collection, and evaluation of Gossypium.;citation_pages=225-256;citation_volume=44;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. J. Reinisch;citation_author=J. Dong;citation_author=C. L. Brubaker;citation_author=D. M. Stelly;citation_author=J. F. Wendel;citation_title=A detailed RFLP map of cotton, Gossypium hirsutum x Gossypium barbadense: chromosome organization and evolution in a disomic polyploid genome.;citation_pages=829-847;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851778" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Rong;citation_author=C. Abbey;citation_author=J. E. Bowers;citation_author=C. L. Brubaker;citation_author=C. Chang;citation_title=A 3347-Locus Genetic Recombination Map of Sequence-Tagged Sites Reveals Features of Genome Organization, Transmission and Evolution of Cotton (Gossypium);citation_pages=389-417;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020432;citation_doi=10.1534/genetics.166.1.389" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Saha;citation_author=J. N. Jenkins;citation_author=J. Wu;citation_author=J. C. McCarty;citation_author=O. A. Gutiérrez;citation_title=Effects of chromosome-specific introgression in Upland cotton on fiber and agronomic traits.;citation_pages=1927-1938;citation_volume=172;citation_year=2006;citation_pmid=16387867;citation_doi=10.1534/genetics.105.053371" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Saha;citation_author=J. Wu;citation_author=J. N. Jenkins;citation_author=J. C. McCarty;citation_author=R. Hayes;citation_title=Genetic dissection of chromosome substitution lines of cotton to discover novel Gossypium barbadense L. alleles for improvement of agronomic traits.;citation_pages=1193-1205;citation_volume=120;citation_year=2010;citation_issue=6;citation_pmid=20094702" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Saha;citation_author=J. Wu;citation_author=J. Jenkins;citation_author=J. McCarty;citation_author=R. Hayes;citation_title=Delineation of interspecific epistasis on fiber quality traits in Gossypium hirsutum by ADAA analysis of intermated G. barbadense chromosome substitution lines.;citation_pages=1351-1361;citation_volume=122;citation_year=2011;citation_issue=7;citation_pmid=21301803" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Z. W. Shappley;citation_author=J. N. Jenkins;citation_author=W. R. Meredith;citation_author=J. C. McCarty;citation_title=An RFLP linkage map of Upland cotton, Gossypium hirsutum L.;citation_pages=756-761;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050952" />
<meta name="citation_reference" content="citation_title=Interfacing cytogenetics with the cotton genome mapping effort. Proc. Beltwide Cotton Improvement Conference;citation_author=D. M. Stelly;citation_pages=1545-1550;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. M. Stelly;citation_author=S. Saha;citation_author=D. A. Raska;citation_author=J. N. Jenkins;citation_author=J. C. McCarty;citation_title=Registration of 17 Upland (Gossypium hirsutum) Cotton Germplasm Lines Disomic for Different G. barbadense Chromosome or Arm Substitutions;citation_pages=2663-2665;citation_volume=45;citation_year=2005;citation_issue=6;citation_doi=10.2135/cropsci2004.0642" />
<meta name="citation_reference" content="citation_title=Challenging the Future: Proceedings of the World Cotton Research Conference-1;citation_title=Potential for crop improvement with exotic germplasm and genetic engineering, pp. 13–327;citation_author=J. M. Stewart;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Ulloa;citation_author=W. R. Meredith;citation_author=Z. W. Shappley;citation_author=A. L. Kahler;citation_title=RFLP genetic linkage maps from four F(2.3) populations and a joinmap of Gossypium hirsutum L.;citation_pages=200-208;citation_volume=104;citation_year=2002;citation_issue=2-3;citation_pmid=12582687;citation_doi=10.1007/s001220100739" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=M. Ulloa;citation_author=S. Saha;citation_author=J. N. Jenkins;citation_author=W. R. Meredith;citation_author=J. C. McCarty;citation_title=Chromosomal assignment of RFLP linkage groups harboring important QTLs on an intraspecific cotton (Gossypium hirsutum L.) joinmap.;citation_volume=96;citation_year=2005;citation_pmid=15618303;citation_doi=10.1093/jhered/esi020" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=M. Ulloa;citation_author=C. Wang;citation_author=P. A. Roberts;citation_title=Gene action analysis by inheritance and quantitative trait loci mapping of resistance to root-knot nematodes in cotton.;citation_pages=541-550;citation_volume=129;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=M. Ulloa;citation_author=C. Wang;citation_author=R. Hutmacher;citation_author=S. Wright;citation_author=R. Davis;citation_title=Mapping Fusarium wilt race 1 resistance genes in cotton by inheritance, QTL and sequencing composition.;citation_pages=21-36;citation_volume=286;citation_year=2011;citation_issue=1;citation_pmid=21533837" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=A. Van Deynze;citation_author=K. Stoffel;citation_author=M. Lee;citation_author=T. A. Wilkins;citation_author=A. Kozik;citation_title=Sampling nucleotide diversity in cotton.;citation_pages=125-125;citation_volume=9;citation_year=2009;citation_pmid=19840401;citation_doi=10.1186/1471-2229-9-125" />
<meta name="citation_reference" content="citation_title=JoinMap 4.0: Software for the Calculation of Genetic Linkage Maps in Experimental Populations.;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Wang;citation_author=M. Ulloa;citation_author=P. A. Roberts;citation_title=Identification and mapping of microsatellite markers linked to a root-knot nematode resistance gene (rkn1) in Acala NemX cotton (Gossypium hirsutum L.).;citation_pages=770-777;citation_volume=112;citation_year=2006;citation_issue=4;citation_pmid=16362274;citation_doi=10.1007/s00122-005-0183-0" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. Wang;citation_author=X. L. Song;citation_author=Z. G. Han;citation_author=W. Z. Guo;citation_author=J. Z. Yu;citation_title=Complete assignment of the chromosomes of Gossypium hirsutum L. by translocation and fluorescence in situ hybridization mapping.;citation_pages=73-80;citation_volume=113;citation_year=2006;citation_issue=1;citation_pmid=16609860;citation_doi=10.1007/s00122-006-0273-7" />
<meta name="citation_reference" content="citation_title=Advances in Agronomy;citation_title=Polyploidy and the evolutionary history of cotton;citation_author=J. F. Wendel;citation_author=R. C. Cronn;citation_pages=139-186;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Cotton Sci.;citation_author=T. A. Wilkins;citation_title=Sequencing of a cultivated diploid cotton genome - Gossypium arboreum.;citation_pages=5;citation_volume=20;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=J. Cotton Sci.;citation_author=J. Xiao;citation_author=K. Wu;citation_author=D. D. Fang;citation_author=D. M. Stelly;citation_author=J. Yu;citation_title=New SSR markers for use in cotton (Gossypium spp.) improvement.;citation_pages=75-157;citation_volume=13;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Z. Y. Xu;citation_author=R. J. Kohel;citation_author=G. L. Song;citation_author=J. M. Cho;citation_author=J. Yu;citation_title=An integrated genetic and physical map of homoeologous chromosomes 12 and 26 in Upland cotton (G. hirsutum L.).;citation_pages=108-108;citation_volume=9;citation_year=2008;citation_pmid=18307816;citation_doi=10.1186/1471-2164-9-108" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Yin;citation_author=W. Guo;citation_author=L. Yang;citation_author=L. Liu;citation_author=T. Zhang;citation_title=Physical mapping of the Rf1 fertility-restoring gene to a 100 kb region in cotton.;citation_pages=1318-1325;citation_volume=112;citation_year=2006;citation_issue=7;citation_pmid=16544127;citation_doi=10.1007/s00122-006-0234-1" />
<meta name="citation_reference" content="citation_title=A standard panel of Gossypium genotypes established for systematic characterization of cotton microsatellite markers. Plant Breeding News Edition 148 (1.07), an electronic newsletter of applied plant breeding sponsored Food and Agriculture Organization of the United Nations;citation_author=J. Z. Yu;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Cotton Sci.;citation_author=S. X. Yu;citation_author=K. B. Wang;citation_author=F. G. Li;citation_author=R. J. Kohel;citation_author=R. G. Percy;citation_title=Sequencing of the cultivated tetraploid cotton genome - Gossypium hirsutum.;citation_pages=4;citation_volume=20;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Yu;citation_author=D. J. Yuan;citation_author=S. G. Liang;citation_author=X. M. Li;citation_author=X. Q. Wang;citation_title=Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between gossypium hirsutum and G. barbadense.;citation_pages=15-15;citation_volume=12;citation_year=2011;citation_pmid=21214949;citation_doi=10.1186/1471-2164-12-15" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Z. Zhang;citation_author=Y. L. Yuan;citation_author=J. Yu;citation_author=W. Z. Guo;citation_author=R. J. Kohel;citation_title=Molecular tagging of a major QTL for fiber strength in Upland cotton and its marker-assisted selection.;citation_pages=262-268;citation_volume=106;citation_year=2003;citation_issue=2;citation_pmid=12582851" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. P. Zhao;citation_author=Y. Si;citation_author=R. E. Hanson;citation_author=C. F. Crane;citation_author=H. J. Price;citation_title=Dispersed Repetitive DNA Has Spread to New Genomes Since Polyploid Formation in Cotton;citation_pages=479-492;citation_volume=8;citation_year=1998;citation_issue=5;citation_pmid=9582192" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
<meta name="twitter:description" content="Genetic linkage maps play fundamental roles in understanding genome structure, explaining genome formation events during evolution, and discovering the genetic bases of important traits. A high-density cotton ( Gossypium spp.) genetic map was developed using representative sets of simple sequence repeat (SSR) and the first public set of single nucleotide polymorphism (SNP) markers to genotype 186 recombinant inbred lines (RILs) derived from an interspecific cross between Gossypium hirsutum L. (TM-1) and G. barbadense L. (3-79). The genetic map comprised 2072 loci (1825 SSRs and 247 SNPs) and covered 3380 centiMorgan (cM) of the cotton genome (AD) with an average marker interval of 1.63 cM. The allotetraploid cotton genome produced equivalent recombination frequencies in its two subgenomes (At and Dt). Of the 2072 loci, 1138 (54.9%) were mapped to 13 At-subgenome chromosomes, covering 1726.8 cM (51.1%), and 934 (45.1%) mapped to 13 Dt-subgenome chromosomes, covering 1653.1 cM (48.9%). The genetically smallest homeologous chromosome pair was Chr. 04 (A04) and 22 (D04), and the largest was Chr. 05 (A05) and 19 (D05). Duplicate loci between and within homeologous chromosomes were identified that facilitate investigations of chromosome translocations. The map augments evidence of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 versus segmental homeologs 14 and 17 as centromeric regions show homeologous between Chr. 02 (A02) and 17 (D02), as well as between Chr. 03 (A03) and 14 (D03). This research represents an important foundation for studies on polyploid cottons, including germplasm characterization, gene discovery, and genome sequence assembly." />
<meta name="og-title" property="og:title" content="A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/1/43" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic linkage maps play fundamental roles in understanding genome structure, explaining genome formation events during evolution, and discovering the genetic bases of important traits. A high-density cotton ( Gossypium spp.) genetic map was developed using representative sets of simple sequence repeat (SSR) and the first public set of single nucleotide polymorphism (SNP) markers to genotype 186 recombinant inbred lines (RILs) derived from an interspecific cross between Gossypium hirsutum L. (TM-1) and G. barbadense L. (3-79). The genetic map comprised 2072 loci (1825 SSRs and 247 SNPs) and covered 3380 centiMorgan (cM) of the cotton genome (AD) with an average marker interval of 1.63 cM. The allotetraploid cotton genome produced equivalent recombination frequencies in its two subgenomes (At and Dt). Of the 2072 loci, 1138 (54.9%) were mapped to 13 At-subgenome chromosomes, covering 1726.8 cM (51.1%), and 934 (45.1%) mapped to 13 Dt-subgenome chromosomes, covering 1653.1 cM (48.9%). The genetically smallest homeologous chromosome pair was Chr. 04 (A04) and 22 (D04), and the largest was Chr. 05 (A05) and 19 (D05). Duplicate loci between and within homeologous chromosomes were identified that facilitate investigations of chromosome translocations. The map augments evidence of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 versus segmental homeologs 14 and 17 as centromeric regions show homeologous between Chr. 02 (A02) and 17 (D02), as well as between Chr. 03 (A03) and 14 (D03). This research represents an important foundation for studies on polyploid cottons, including germplasm characterization, gene discovery, and genome sequence assembly." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
    <title>A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gBTC6GuoiZaKJessYA3YoFeB87GFH2B5Mbbfkf7UoEU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459230","apath":"\/ggg\/2\/1\/43.atom","pisa":"ggg;2\/1\/43","ac":{"\/ggg\/2\/1\/43.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/1\/43.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/1\/43"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/1\/43"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/1\/43":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224l+xEwllfTjSg0CMuoLb7hwHBWQnrDYPj0787qWCKkrgB7ODZvwq7SYLQtjNyvPr5wORFKC6s9PlEVEtrvWGUsdYQmK\\\\\\\/CWcw2OS5Eaa5JnN+PXnnJyuqZMNeyHyoNU4qXTp9FWyDXPCbVENWNLutW3ksqLGYKh6T3bZc2L\\\\\\\/V5mZCM4unXwfYfXK3oplVMHF9agGUHRxyABRq2J9IlEmQQ572IzUQrlotU\\\\\\\/kcpxWx5IGcl9cZI+24T1Y8fTR2V1uwHS8lctQ90LenFjpGRR+O8t2ZSw+YBGFWM3UnCwHG3HRV6fPTZ\\\\\\\/cWWOARvw0Ut1zbfknbHlTefHODXDQDKqT3wU8wAhyI4ltNVrfvlpmbcWrIRorOGkWK60DUdpUhXgxFc6knG1j2Lcz9bgxjymOZUZLmR4bROfQ3doa3I0jXVyGWbA0+lE0C8UMW5m\\\\\\\/fd63qG9J+r5qJBzyBzSU5fx6C0K3Xn2IP3zjaeFwGDADotUEik98JEnjfcCKzUzRWypB\\\\\\\/\\\\\\\/NtL3mjnxe0Cu2FhdXpLUa86IAa7iYLqt\\\\\\\/sGXLaeGVCiLT+qPXIwWBwypWWKbvGTvtEgW5klsUZ2vuot3MaKdPs03rZwo2UXOdXcyMG9OwEJNq4H9+W+SfD76LHxnQIYpBSBp5AVDHUk6LavWDneX6rooXAvdwbK2SUyvy8JXbnxg0kGZj+niPVyeeCe52Zyvea3fSxCM\\\\\\\/6vP\\\\\\\/0QJpffrGwg+UYVhGSNZS+dxhOIWeEIdag+hK99eY2CKFWvOxs3LiI\\\\\\\/rjwNyI1H4yHKCKh97Au8QFcCTaEdjLrbdu7UfeMqm\\\\\\\/+oebzVlm+68PcBA4BuITIzZ662zdvyar9Rj7lQHP\\\\\\\/\\\\\\\/4maQy+8wuFlRa229okFJN0vuMO1jkkiEfUgxsoTlShP\\\\\\\/O7FjCWzJVd7BqRu6Z1GP95ElU6gdWkfcQ2o68lcIKi20W3HNHXEX6vA95XpMRH5VHyXmWAIeTgIKHV1\\\\\\\/NDaASlTBorxn0ByIPZ2BX2QBQ5edQZAe7dQieA3462xE1BJXPDlK9v08MvRk9yUZk9C6jKJMZiH1YgQS8LPNCWj8LGi+atcUB7ANzg4AY4Ga+\\\\\\\/fYThMkNhkKHvRv\\\\\\\/5HUkqhkKxXh6JduwoURV19Eze+HabeS0N5BiN5wP\\\\\\\/xROjLxpq4LdpYOJgJq1+x2NB5+TWLV7r+g7VDTAmseXBjwM6C3CmIBXxueD70XcLJp0WNLZYXKCgjaj6Tls\\\\\\\/XcGNZBSZ7HrUab5dVDDFMX31Yaqln8a6OOxK1kgyZvoiSLVyB2\\\u0022,\\\u0022iv\\\u0022:\\\u0022qJNf4KblcCsENRA2pWjAPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022003c4a63755d1d35b90c3664b7a4e31dcb9f90b84208d8ee47624a128897c165\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OccqvJqTu\\\\\\\/wWcDrDqtH7QLkCOWDkBXll+K2IvzHjKPLWM18YAZedHi8mK\\\\\\\/TVmRdnvYXVuOWJEDOtLGaY1eAQb+UbVjaWtVblWXtl4f3ykg0SVUb1OSPOxE8EL5A2gspRROEf5gRjRnt3h2e+ElgFKN56gOF+ku8ow97XH7SqWeaLhO9bk7rgZZX5lADpVTM9cKST2Uvh0M1eEn98JeXR6Ef9QQ6pgTb38zN+3GjiIeSs9tiBm50BiNRga+SNkTYA7UJI4N5fsveIsmpGKyQCcRGPqXKi4P7Y4D59+hF9+z+NvX2ZZQ8Bon4uxLyu0wn+sA0K4aLoZZmaHQWehMnZg2pC+mna9OQsb7SsvOT+hGDuSVdwhiOnpdnen9JuHlvRu5GTgmwWAxV7TN8HGbedeXIaMiUfb6cR1HyMWw7PEml+TWRpMtmyhmuGTWZD6e203\\\\\\\/qGZNZCpTNt1k9\\\\\\\/2G28khsNh8Fw\\\\\\\/Z8ER8ZH5AF8RjtLSm\\\\\\\/2hX1fpwI9yFmtE9unnEps7dvmLNzjW8YOWSFmlYwemusphaOPI3PDAuUoYOJXqld9tcqOIzUACpnoK0J+opNbtmquSPDeEcHWqidCX02x583nyYE760ifWwnrnTWGpsewFivVJhj0hsr2djqZVzVOE7W4Y6Q8iAXMwqYPYsRDoXJo2PSIMrw3bNPRamleJ0CquXQMFsBlZ\\\\\\\/irh+9KwayUSNWWwXjDiB4Le\\\\\\\/G0JqVD+5wyJNAVQfbUgAzYPy9gw8SaU38GwckXSuCh3qeTZ+NJ6W6eL5aLfLSVdSXxGsLbkq+aBgrI+mqoV9DS6vWpo\\\\\\\/e58k0EbLYxb2cRfoMCHlDnXlKhh7eJbX5qV5xG9WahVsRVY9Q+RG9JDz1iw1ZEgw1uIM2Nd1s7pj58h2bDT\\\\\\\/147sV+fD9BXch\\\\\\\/+rTtiVbJn\\\\\\\/0sT1pfsatGDJew6VQINu9MkEnu0aRcze82J58yCl7g8ZRnU\\\\\\\/84UaaOFVoLkPJRIK0zjJ0YDTxzRdyjEAuSu3OoBysBqbYhptVmU3yDm9MsyWeRBqL+coxRoY7nXUlzWJRMeX3mrq9VK6cupg+fFVasqS23KOtBSCgzwmdUnwbCSNWuxbOmkyf4b0c+UYzjwsdk+Ph8x4kFHd9tfEj\\\\\\\/KAYQxjlhFIZtxldDuiDmpIEAEyn6Q7ZRgunLJ+zOvduvRbMaOQBbn5jCsy5FH5qXx+lQgtHetwyESvhf8CJOWRAYkHVoOggagni7b2TC3N300BTD6hRnO+VRFAimuW7kifFzzVWPMPduK7MXTBmdo93kun\\\\\\\/zl6ly9iY7hfmTlz+pUXAtk7tAhlD5OHkq02XA2NN+Q2JlxDtfTIX+qEA0osPZN0CLik5v\\\\\\\/GXOxhS7AuuHHxSbjPMg+PTI\\\\\\\/0V95oGwVuuv8HnKSLPvOccBzZVJODFXvJwbzpuSRSJzmrWkcYU+6uFJ7+AaXrLcFG+RQvrwudCDrBZYrI\\\\\\\/u3ZN+wZHpLBmBxbaNahTLPn1sVYvSKropOwziJBAQzfaGdncabvOLHx\\\\\\\/R8wQlUtBk0ipAACzJhSy7\\\\\\\/Hexi3OasxTe2dNE\\\\\\\/fyxx3mSZSnyENt1jhFeCoPFM5fySIQGzLosm59TDEsVD6lLDTD5hGPEjf8YSfBs2B6KSTqDB8MVxCHleXoi0kSr\\\\\\\/7AyofpNtfEYNMNrMt7vfUXeyTGH5ACxsx3Ntea4RCG9OoPTbwWhe20A1KUtzKh2YyimVGhk2EN0fxlsXmqWR2ejdKoDktlkeqY6W24pk6kxu9QpYEm+2rm83D8aG5P9gnEWq7c9VjkWkv2XAhfCT1IFVGFC2xsev49ShkmASD+ikZhEQ3N18bIHUBuu7ZT5fsbnut5HyEFlqk5ERgUO3TVQSzShIDrm9IZzvVVtjP\\\\\\\/NBoMZfg4cEXCyeCFwd3jmC\\\\\\\/EObTERCOz2WZQHbEqPTCQmvco\\\\\\\/hMFhcA+GgN1Y7v98Xpl5SKpB6S182j4gZfn7aoJsBRQvrbCxNQLEXq0V4lvg1SPfB1neAonMvm1Y1p9DV56OkkTUX+csimYHC667STLoNo0lMD8fD8cYWdzjpeL4W1njof2FAIdf3g3iYr1ZTEeOShxedx8Q6+Xjyg77ayOh9m08zFBIjZjkzpc8JFSiVTD4bzsZ4QMJ6xEZpCHWXYZtNQrmHt88xkmtrynFfg8OrYVyyALjh1KOZiqSO8lSrNQOIZ4Fk2fd1UDdCNMmZOhbEF3BTL8iLO1nH8hPpn5urxzy0fOx69PBehL3LIfgFPNTGlp4TzqbUoUHhWUm\\\\\\\/sScK6zphQs2OweAFdEws4Gk6EFfK0Pl8fBNKu3d64ZbS0HI\\\\\\\/f8IY+4hZ4aHfPws9P7bjmzre+IDlDx6xFdT1cPqZPUaCKWEojAcUombKsZgRN\\\\\\\/sawKIzxrB46GDk6sBMiSDnldF3Vx9eL+2xZnWiFYHoq8X0JTMpfhwQEvHpHWxz\\\\\\\/+I1qq2Jom3OJQtCYgqvpEl5DrvJVR9P2u9h\\\\\\\/VLEvebnUUT\\\\\\\/tVzUmCPSsbUwtZXaLPbqKizOuwjxIK4MxvL1TNQddKRoVgnGwGqT5Jx2VrI568Mz8bOHK8tQhKhbc0+f2bAlgnnSwyg9fKHU38h\\\\\\\/YqUqpzk7xlDYqWUN0h2FuFDGQrMUVUBwNz2v1EHHGls71F6AVaES+7u8Urecsbkd6hmWLBp8XYv6vWdyp4N4WbcHh1vKRcdk813NLBR08zOgGMJMKus+Xt05Kjuq\\\\\\\/sSNEFKTO4Q8jsv48fAhFaM\\\\\\\/5Npy1AY1Zq1QLESpz5kw8KYHF1KXQvupjwtEJwgLYOMSb9CpSQyx3D\\\\\\\/djLl7BeIwGYq5qSjKWKZ5GGvT2mK9ikcU9gaJGXn5VU9tFyvsSpZtYCrItfqelmYQgoP7f\\\\\\\/7IT9505Wdui5sZd5o4BexWE+g3Fi3X1qtU2\\\\\\\/JqvlbPTe2h68r4NOhUbInQKFmOB+Zqb8GXmKmC7FpHWm6tZC3bZ3gJl6qNZf93\\\\\\\/PcXk3oM5M0yjxmzHewmmTuIZqpp09VUdrf8BMqTPgdg0u4GOAZfzWG\\\\\\\/NqEi4qZOZVpb3OyJTuUl6V0dorVpUpWL5LcGey70rPWu1Lv8wrlskeXYgNPQekFvHNBDk8Z7WiXQ+IBldGLxs99vdc\\\\\\\/GS29McEZUYoltjk1l8RYmWizhir\\\\\\\/UStCiBj5quzaP6tcoJZbbZyW9QBgQ\\\\\\\/G2XQmj6TgM\\\\\\\/ngiT7mX04pmH7kAcWq34UtCxpjVsm1cQkUXsjeImn\\\\\\\/p8d3XuGiLLAqMCKHp5lZ5oQY\\\\\\\/oPgZpxhcfgGStzXTqxlZEKSqUKC3sehtKz3eTbLVju6BXHVCRPWuWZ+H3HStnp0WO20k9kw6A+z66KR2OFqdgM9I92qzvLx3jH1JX3fVlWFyGLHluhqwfueocYnto36Djc9XUcZDWtyKqIFjpQMrxI8zDAnLJcJWVXVWHrNk0rSbRNylSD0u2EMnUnRMHhb1sA5ZXDUL8zUN5yuM\\\\\\\/SNb7OioondjcLHlEAqxyxjBBc6hzJ64DkJ5lRasyBQs27ef1lsofOzNCTf3UyUOlfiy8RC4I+EC4PtYjgVrRsQF06kg\\\\\\\/661DlxkISsDSGvGARv6AImCoL\\\\\\\/+nNX1wwBKetjxtFDJcASLTWhErUrsY9\\\\\\\/vxSLKP9lkbYm8wdQ6eErXUvWC4+tlSbIGGb7VkTQvP8vcfAHb0N1u4HwAAZCBmXClwpxiFrVQWNnPHUVXpfLzIqZdav6TRvuL\\\\\\\/kfDz1aX+h1i3sA4Wg1dRQEkDbQML94wg+MLlhgjqgVKy8LieO+0bZKhfHfTak8boNaFxF9iU3tYpdF5CUWsFI4+RVz2cJdOS4Tp1PdL4UK6kxdhhsu1mdM7MeMxObgfjsQofs34mGW3pYx+1\\\\\\\/KhhelcaqatR32eKKFRbLBu2oJsCXQ7Rr\\\\\\\/844uxr\\\\\\\/8BtQEvqqYoHewpMeaWPSIP9eTWl2j98ftmqFET4\\\\\\\/3Eb2h21yvoOxIcHq8E32xL7iqbIp4kn8I8dtdOAQdZJ34b87xcs5NaxuRGO\\\\\\\/BODxzzRtYhyrILLcVCR5k1140Cjm8zMWNEd3LjP2ty9x08oZQk3bsLP5ZfGaorGW3E3Fmcl5tn8QHjvtQeD4Hzu34LIzVt95g0QlGBn8LKaMAaYabhjDaWiyz8meFluTNE6YBVFLvS91J5W17srHNxoTmq4\\\\\\\/RLxTpwVIPdBZ9hxbBY92fHerzfOFsNAHsGMMoThDQ7W7fctsvNbfp55G3k\\\\\\\/+lMgfh20EUo4jfwRn5gjP5pkmdbbtvKYLIznzotlgmOfwpICem39H3iAM0vaudJ5hOdCIPBC+H1zKLjLwXKNPyGyluHN2REaNu0fSMubGgZteCPdrIqGVwxtBEBLMPbPfVwOXNFoQitgd0PjBuhQmAoL+lpf\\\\\\\/B7jAnuwHdUNkshc0paA3wmACYEV6hVh6J1F7OPY5BbUca4b1Sncvd3ptuzKleu3vSy4v\\\\\\\/riMIVLcA5D3gvMstAuQODAjfQs2rQMaygY9yupGf9qyJTPP3EsUfYQI1KMc02UNUyz7o9eOvclznTwAJi8RLkxJvNPbhKzow6eHNwJ+4wLrMfc4I4mKhfgqFC40y+5mGdHGslQvbJIJRECuadgr6r8ZrUC7i2jwZm+GN0MNKEkgKpHlpHxAv8yrNnKrnCY6C6zqL1pvjPC8fUD2o+\\\\\\\/p1u+Ic+HeOFwQ6Bqt3P1L7wygHol6LwT2KCRG0yfo4vaWTeEzH0l9189ynQZD7oB16jmhFYWrgzt4tNCr3hP2WLSP6HKh4A+63d\\\\\\\/IqL8N\\\\\\\/vBeAF4KNu\\\\\\\/kenUeMVq8XXYslLAH2xf1gVbUBD3m3RECYHijatA3A2rP0wDrdwLagGAJXQ6F\\\\\\\/XXvezGIjYbiB0WAvy2cFBIOPKod+xTBbyRRskqLTwhyzdtKvybM66lXiAXMd4sAwSHjjtNtd9nCZWcvp4zffEfm+yuIV1Xcg+u+\\\\\\\/niZq1xKYo64ER7v7Z+N1rPdZ7z1Het9LFUutSUBrUF+mc+sSCwYCGvpzO9mJ7bGGtOtNN5Ofz+H9w4+914uZYT9XarPprQy+egFPD1nIRtpLFDUMegXvpgYZV9h1nP6Ol9yPp7iXh8X3Ficq9M6TsPZAYbLVI\\\\\\\/tXsm+gqaJJSDUAjO2wr1x7GxpppW9sA00ZMP6o0ElKqiKTcVWBTc8mV59cYSOpA7Bbf2MqxQBQfsu15oyvZBxddDy74XOYI8Geth\\\\\\\/2tYD5Gmeo4NDw0WXLgr69yfREDL27kRN1UogjHeoK2xJ0jFAUYK7Y4zHY9CcMOKy1gZkB8VdBxJzCd+7OHbF1DV46fV7y00uvLHulj\\\\\\\/hhub20w\\\\\\\/FcFs+3icMNFX63ddlbopGCo9DDY0bhJ08H6ABSxdp9UmrAa557A1liNh1mMJEiFV+Tbl4eV0W4NalUZVuOjqesZymK1drfDbLEn6mC4ICB\\\\\\\/P6qMwAm4JgmrPtVSS\\\\\\\/lEfqcUbBS0UNWZ1Y3PavgWm8NiHAnxjB48Ud2WGQOxJzs4TLJgRLoI1X0K0YSDfnqKB8ri0xgXYkqc16ryKz3hCO4FNhGbrD4j3ZOU57xmBNwGy5jc4GxRzDP\\\\\\\/i\\\\\\\/Wmr7XwLTmNXRCfofWwC52uPLkpcIvI7XvU0J8GHk\\\\\\\/U8d9gLvXAnZkvm7zYquYqi2woXOEY10FQXeXtQj2xliXTc7lCl0yooEUMB9xbL8PsG3LdL3ybseNrdNGU2AM9VIQIdMOabXzGjkcXarhqajzj2j\\\\\\\/W0GR9fNq7FDwCb9qFZtHXwuzZihOEffPvBrUqwehTN6JV0TxfRNHhWkHCdNhhATE\\\\\\\/5b80jF3s7Xr4eDPWSUqhkkPM2MdmQ67I+aJfM8xf4n4VwHE7r6kXl9NL5FR1LqKHnaQP4C\\\\\\\/RX1FnNc2v+A8kHtUHp14PJt6\\\\\\\/0HA3j1IN77gW8sjPRycCV6hjSc4P1vExbDsBkA+riYc5go0QHRTCTOFBAZMuNDNf7La8N3jPaJ3dIEcB\\\\\\\/BmPVYuO39BrHn2jDx9uKKFe6HF4vxd6ItSm+p8KH7DC\\\\\\\/PM2DaPVV35RnX6XkOlWYPKQCqA05zYcC9SMhbLKehuguDottpsA6eHbCwmbYb0Ixm6xzio5CG0yZHJmF2RmHxPXmS+acKyZ35s5Wf2V80ZnfZ0y7vvYxHMjkJfRX5vEsppaHLg6oRkOnUFV8PWis4HOt\\\\\\\/I0zolmVnbotam6dQ3K4KUqefxW2vX0c8aFzTPuKkOZkJgc0LSo3MpRYlzV3+xTpBkmDMHVlP4DOS6H1+iVS+5HblZbu7d2l6tbt7MF6auSfJBpjsAEYIlNxqYdZ4dH5hPs3QTaNzzkQLSFoQJyEQ9oYshxFqNMbPRLSfdRzxq4bHVl\\\\\\\/wmHe+CxWKfn70V+nNyu9+JXv2o9qQtvemlrUuyuB2Zul4TNxaFaAb\\\\\\\/EZiEFzfa3r8is83DwFmcQ2UOxVaCZ2OQLBCQ3lHttrsPCe4Y6HcMPrn3EsChS0OsSr4ERyMA2\\\\\\\/NQ02mpWQWYHP72ruq0TANr9fLfBGfqzb01UeEajly3diC9b0d7h3+uvnVZFfFmgjEhTctwHfiJW8OTGVCwX3czMvOsExCof8x4VdkFmWetcBN+5gsRnDbqzvpES9lPmirM0UxDW1K6O0GK544sIGzwAO8jaecz\\\\\\\/SwE0NYpmq9cilc0+rUpE5KHKV31Car+\\\\\\\/vvyxnvnhYfqtMAYGcvwk4yaUMK9H6xk6dzj0W7Wp0TS60X8MVpgQO7rOaENngr+MMc8dF73\\\\\\\/UxGoMcf4YgfcW1PTc7m+byk+7abzxg79nWemyOnzoWQwR7C2SGrgwnvMov31i18RPCifPnFWF\\\\\\\/ok41t9Jme03RXix+3K6NDM2gisQMsW+LHlJkoZDBKN0\\\\\\\/oB7gX8CstZCFk\\\\\\\/06HZwIhRBfo1f4mdBdZfDkh5VqikkpO1beo6q9Nu21Z9nkOoMu0BvhMKW26LdExq4YnvBZT6YI+TjD7LLSlcj2Hf5jBsfbjAfcfrZZe0Sjddk0lAzn2JniyL+S9P0btzswF0hgicZcZkWaKRtmgK1Ak2W+NnS\\\\\\\/cxVmkqdEop0X4gF7leBM28LEKitevUyW7+UL9fYOF+KconTNH7Bk33\\\\\\\/IvXOQFm6qguTrmU\\\\\\\/LHGVBRm5Ni5HiTSF9tAp6xzp\\\\\\\/lk3vwJe5sS89dJUMA0MYbMrk6xbSCwb2bdMD\\\\\\\/WYgUdYHZRFTr4bvScK7DEKlZzUP3DHhPWaPHY7aRNsaTcDv0n+htp+OnjhccGlFE2\\\\\\\/2ndprE88IaKasMDv5VJ8eDrkbNKzquWYHQCBKDmzZt7kH0ecRetd9ZtmB2hUQejQ+Mk1GZvV2OxXAINNMmFurTPhoCaLdBPBCd8DixTY8pU3XeNIX+VbHXMj4woLiyOuHlGM\\\\\\\/8g8F7qsr7d5wGOsdutu8CXg0tYoBJmkJOTcFdLtjJdRHnK6j9OTv7JKKtT1hQRBCyj50f3jG9CcRT8hM7nFIu8rSsyYRXyYd9C3PzzsV89vJqjL7WxjMMJjfKhJXKkJGqFqi4ZKlaGHRpSRxFkc83sw086GUMt\\\\\\\/RiCj98+mb+FPkC89kbcJl2V4P8yGpKJsVpSHDY0FxTePJa6PO8BOqC7nsr00mrAsNexFByfyELPjiCDIsfdK1XbwwPH8xxP8rApXZLZ4bFVXuUtdSIdVDCdyIRUsf+DUqchbfNOqhP7tq+Nk2C4dP7+MDJaLiB96LB+Czn3BSYX5fjnq3ynwBNo9U7F2p5h9c\\\\\\\/yvie0yhIA3REDt15cpyihVSt1WS\\\\\\\/DjV49YhgNLzfs2PXsYCKuUBeuV4THVCqibyvfeMXBAG6HDG0kU23Zd6jPQbKHv9dG7XyT1pxNz1widTwszfSLOGUOFrbrOtMm80C+OQFj9UxglZtTBloYKjwu1g3OxJPfoNTF2HkS6QZbloZy8kKNEdmRGafe6nysFgPx7G8iOeCrNw1DkkYy8+zvJfOYUAUEX8RL1icCAvuZIo9n\\\\\\\/4AylE1ALTF0FpdqCAiCLGAYOfRTyKCCBQHQ3G14XsaUC840euGV0ncjtenuYg0BWBtKV2uN38+E3HdmG1LOmJb8bJsVSZEBN6UM8lfdjRCuzVbR1\\\\\\\/jCwJ1VmZ4yUz7Td4aps5jFvNKW7osxi1GUSdySF0cQaukn41uj2kwsZ6MgcIWQbrqZ1ghnK3f9Fb9nb5Xgh6TTQsBoobXoTKIB\\\\\\\/qNLPIZHPPqMzRjfgjMT4N0066E3Gsi01PPEFmbMcP57RYi\\\\\\\/161a1QMh6qtq0D+\\\\\\\/JNj5er7ZFMSl9x1faCPmdS34AkKlLT12c5E3uafdDqRHst9C270Vu1MNX\\\\\\\/cP8wc8Nj1P2W0VOWzGVKDBIoqbO2axp1a\\\\\\\/J8Xehk3Ri5j+GLWtLBpNtOWaUvn\\\\\\\/dODYmFOg3+W83motdoI7OsyZGYpEFonj6F0cEPeXsX4kX++crEKeXuIdsgzmChcVujgqEOMykidu68dHCqf0m3JRqY+uxwuE5D4WAy68OPqH8qmDVzH7STMIAoJ8ZlkfLRTid2XyOFeYIvJI5SFRFu55I3AesLJlaoRQo112JTMLBCtww8e5RLpQ5G+qqoWw9j9SBk\\\\\\\/oSb9pEBa8hST2M4uI0fO7ymRG\\\\\\\/BIuP5JijlqiLnGw\\\\\\\/iEqSX2EhDc4LPF\\\\\\\/86cgsQbOEIrs2lnQKYqdHpIaT0yXtUzEQzFIW+QYJOHb+iV+lukM7RD54SzK4rZ36029GrRWOiTsJuL01fQzY8W3C6ktOXAsesfZ3jj9LzsZ6\\\\\\\/AA0LpE5vIWRz2JEkewgf3r71a6X2c3PhJbY3HZl5U0FTV6sZr+x4tJarZYwTD8P2LwF0oHK0rV10nTCz70Z2UC3EhjjL7fsyJpm3blz+Zjp4p9MBwEbgqiJwc+IwkgaCCaOAsGLg479rQZ6qzaWSwe2e9CgaXCdrJO5cTJ1MMAXi58Mj6xeZ5zuvqo0s6ltdfxrY9u\\\\\\\/ZwAcyRyJoD8dwq93Beq60xAx2I22sUt5UnzfJevCYvCo\\\\\\\/SOAVowXrSs377BWmP2z4C3Dbih8OyAt0qN9lzPWLh9KLYerxkbtmNSZFNn8Bo2+KpOtsl\\\\\\\/2DY7ph4p4e7c4tonTlNQHenhrq4tI4\\\\\\\/Tovk04\\\\\\\/AOcUzeywLqfCsCwHYL2j2QERPjQn7D5WEVogqQ3+kBNueltvBwpyMSlVS1G5OV8HqMZFN34EdeGYIKDQyhz5PMMWaul+bMt5Pg9DZEXUR1nm0j+N+wUUIxpQx8Y+DgEgFTKjnfT\\\\\\\/aZUYbyEwCGj+KRqVsTprYJR6q0QYhL5zgz8T1\\\\\\\/41BloBaoJTtUbjVvc9kFv8iMgIoCPCGLensPVzdoTiYQwgCAkfr7MURD4SBItKzZ2Wbc\\\\\\\/3vEtJqprrEKUuAr1wTSwEpgyWYDuDWbAXxt\\\\\\\/oBGrL8OpUiXdKwg4bOH9hde9z3nZvdzHEmQoLJTXDHBOjRx1r9fRqsj0cJJdXY6flDPe\\\\\\\/ABQZWosyC53eLd\\\\\\\/r8NWnR17LaTGNbUSmmycK9zNdoHAJUUy5pdYz18fwoBSSsL\\\\\\\/ZwnRit9NgLvUduIJAlfK1mgU\\\\\\\/LO+XkAg3Q\\\\\\\/VVbOVClCMBwwoD2AMnPJg\\\\\\\/cGwE5JWIWrVWOxPVOk3wiiJ1gQfvkUasF+i5uzVxxN3r\\\\\\\/\\\\\\\/f1M9cNNS5tnDlaf4qbTaz1tJmVeWsz90ooThjlxt73vcZHrOQOvevKEOTT2UkVIKOlAx2a0rYtlkXYRsw63ezy4FwhibNnqhqcEvh1uMNH3+d9bLisf4r8bfcKsPCxbue97\\\\\\\/FpTGzZyAIubynQkCCJL6nQecw1ViEImpdV23Y4bW9f5qnkLGxeJgOEJT3d9DnOjMAosAFl87Dazmjxb2Vsz9VQPxV07Yt9B4j2DZBwEh4smT5M8JOIRX7E1PSqqBZf3R0PEXLtbkpiHxh3EoFIAa7cdSA27hwIaadGL7TC3H1EvgKguVnmHysH7OVygzvODTCUaXvxndhxeN4QsL+q9TXmo5F\\\\\\\/S+m7NSRDEHQfz5MNE6pSxLsqOLWUYFW+KoK5Hl58wB2Hfc\\\\\\\/0VmzXY7WK9tmqri6JC8vZoFmFBq8z69lJfxTrduh+JRrEa62bM4H0bqreZh79Mg3GPNyUUvmw30RQx4qAUbP43LScjPyWeC34nq7WdY\\\\\\\/N+YC98iHh+eKqJ1YxLoGwW16Ek3whcmEYJVCMSvtF2A0GWc6cvSMfnUk2k7KnNnYgcIURMi\\\\\\\/\\\\\\\/SNkqBamOLuKlGzg3\\\\\\\/a2mBXdqWsCwalPpyXOOzBjz6sKGLxnL\\\\\\\/Q+LX8ryWT\\\\\\\/ZXwTnC0r5ykLjHn2M9pvKsSv++EflBDxOEzce0kBJzM\\\\\\\/KHuCbyTHe0vYNpVItJxCYecu+kxYXglGPJH2kDwJ4uVf6lYvovfD5vdjRFNZ7DhBnYHERakWcVyBEptwyNejEOg9jF0Lm93x6i+TS\\\\\\\/C8RXBCsarfCq+Wku7k4GkxUb1+NqXopbBFObLZikAbG64T99yhNWS\\\\\\\/ByTmA9cS+HdYs3EnpfN7VoHcpL0lA8NFB0OYBpsVYRaHFecvE269EBEK+MRiWWSRuBxGEioDxq8AjYYqvlHqe2YMaEVdtJ3PrB8oU2Ycc11UPPfpI9Rgxx2wrbHoh0\\\\\\\/NhxNhqSt0zOIxSdSkqULXRSEIjY99uvQEBpap3z6H9zdMLG7N8lv\\\\\\\/nMKecTyVjEoDDx+kTOqE\\\\\\\/rO88lKiWkF0DQEkO8z8IVRlF8LdLDHK7YHe0a2kMinGXE5zxHla8a0C21XQfK1miDKa5aS\\\\\\\/80cM1tZAcSspgYHgcvwe26QKiF4v1TDI\\\\\\\/7vnm3r+P88JnmEvakebaJUPqvnm7bK3\\\\\\\/tymcoFH5Q6tv6OLxx5lr4Fbxb9xFNHi7sK0vjg2fS84Q8qkiEZBCUif\\\\\\\/wZMB5AIEXFxoE5xb2iOdzG7hAy1E2nhnmQ0a3rxeCWmy29qEuSYIfrzImUkN1G2z3H4sTLlppqWoXQpy6ukmgImkKJq5vKJbeR12pWjKLYyTSIzUGfmDgVqBxE+j+tdHiOZDs+fqhZfhmG2mVqRs+UfC9woJahU5Pof\\\\\\\/5H6zFp6xyRxD5CnisxY4wXAwJLvXfLjWjSKpRtye5Pb2A8cL8KK5ZwwCnR3iOD4T2qoN2ATtVuuk8AY\\\\\\\/pKIKDky+yaulpQc3UuQeSfV5gSHpQ3DONVEds4gm\\\\\\\/LVLpC8aE\\\\\\\/bnLl8RFFEeCpkE9D9QlWCJauxMqYTnmR+EFCA9EpD43w4rTZYs6ybGmd49iSbvIX4WhW1eurhbILF1vjFsmVZzMamWyOop+yP7Y1HcwFbt\\\\\\\/npIkf2zWVpJWBUOmnHo+i+wBw0iN6J913AQI2yjc32BQFD7tiRu8ekxCi4W2nkGVg5dXjwPikqg4ICffWPrVxIAJGFTvj6loMdzTBabpZ\\\\\\\/gWvYy0YaGnK2hJYx2gwj3QagRxGuw9gLmGyfVlALPyLXgybAg0NZFDpMnB6BX\\\\\\\/8bbjtrM4R0QV+DzSb0Q4ya8FdTVsUEHyL2KVJQ\\\\\\\/iZMOseo0b3bDnq\\\\\\\/7h28UEop0wDJp3h17gVwnCZ72zPrNHqsbHyVHFTFv3XcEyFopBJhl6v2pdRHvZMDOoyHnWtaF7cBmiLb21vTCMoqqZxkz5fd022RpgulmLBq06ZkMMBwZQT9nidLmta0IPTfOwMWNh5IDRg2sY0m6xapS0OJOTpV\\\\\\\/4JannS0+MSp7e5gR8kYZI93M+5Qtd4oEuERVSMGgj3N3L\\\\\\\/cytuTMYy3rnZXaabn+2jdqg1yREy0l1DmNGnsQlVczIK\\\\\\\/p\\\\\\\/zUNLFdwtqZ4kL8zEkbCJ\\\\\\\/c0fJfjo+BUGXguTPdSc8+yVxDUowOBCAfSyl0hW2wFgq9XKIOWK6tQvzYl7H2ruaRxV4SkrEIlHfYyi\\\\\\\/ZxE67y3qGfYspmg6dfmVB4xmr0luT5akh83mG1jqEbnNqX1LunaOUnqZWIS47gS6459xpFy+kVBZ35NI6U6y0HOUKOprS5JkjOaKpdKzgMVhytF\\\\\\\/9st02IPP5vuOdv03DIjuL+lnIyOGWGnjRl6F269wlYKLGsFUl+U81RRNuhjIDpYhAGglUgXk6msgg8Z7ZumTu0fNGV8MxZTq\\\\\\\/IgQiTRqqT8S++TEjCH9zrgbzq0QbPKHgM6cWzD5rwKlwDimg9CJpKjBmNrDDoqKlKgx7EgVlaAnpBEai1Em8luMo\\\\\\\/wJt\\\\\\\/rdz+qCVjJfAt7rhiiynDdYKCz0sPmhfA4utzChQc2W9WWMbHayaaFqFv\\\\\\\/th0seo9rajeGB8hKUrKNmTMCPFN6pyTKlDhIBJ5zUJ0+unviZAaEUluwlNL\\\\\\\/mcJbLbsxIpuIFWfGEsR\\\\\\\/8iSFbFP6f65Ah7RJAcPNFMCpyuntZ\\\\\\\/dXNL3SfMYVJFuaUMjge8XmYxEhcSu1AUB8e0tD7wTm7znQka2L7GZAvPOxZRV+zo9Xesl0BQHT+2B0ihpPw2IyCtmkY3H\\\\\\\/24Wbsq4SNz5aam5OvGr4ixTpqEg737O50odmC2xdgaEg++zLZIcFB0atnJhtII6pwh7iyl5mQC7cKK568U7PVwMRzEgMqrMjvcFVl8GUPyYLS0xLqlaYFXRTj2BWFnjsSVb5Mmn+euDqCj8OvdZU3gGqyqtkyfArstCPm2vYwX0IcZonbUu\\\\\\\/5sa0Uk\\\\\\\/I3+8eot4l4eysswJXS6nswzAGDDCqZzjnUzQx0iTBqx2\\\\\\\/lncP5S7WcxcnxAltyQn26FJBmV7EHenguCPIvEaq+rW87wbzt6IScjhrFFObQdVGRi4i2SFH5Gtz1nijvcSmSVutHi427Z2U9G7fGNfcEdA+VPFT\\\\\\\/yTQqdPp3b9y\\\\\\\/dg6cmETUpnEk6dRDKfHktprAwEf5HyFfrbJND21e7pS7Ky2yUov9Wts64R8KmgMswoHvAiTo43LTSJtF478AeKfj\\\\\\\/08\\\\\\\/zJD0iNkcNkGF8mbfuJCbjs+IW4jbOeoO0M+IMMgmOUz07FzxvI+saS1xsIaxdrtnlIzU8NETCJoXVttFzWbUpgmlVHQpJhFh\\\\\\\/0vrflQ\\\\\\\/5dB05MvH04GrzADpOCxwAX9Rd9iZvJ7sJcg6mXTHVi1zhx8HXWGxS+QMZb62LxLzbFCvWh1xfVnP5gD61pHaJl7hawLMUzstfF\\\\\\\/oP1VzQYF14T\\\\\\\/mRNZXl8flL4ji9BM7Jf3FAHxfNkr2310BniKkWsfVZQOtLdrRwJpvo9ofad\\\\\\\/KAikm+nQmJaFC02famiST+GqmHE93tyzgvtRGVNKJkV0c6agqD18qn+m4pMVvIWgJPPhjlHd9KpB4o6xXvLtjCcrK6mWx6t5e2Xl+MxAyEloQ\\\\\\\/jXXLW+PWdKvy+V1z2SJI3vjWUzW3MRAuVjqKKHwq\\\\\\\/PWLSIkDdvm1XDMibNmkZwgPwayNxynRSjzi9eHo2QJAghfgou2NdzwM1gUvzedu0HeGjPEQMinB\\\\\\\/cnGCNalIMj3rFKbDriwDKSSbV8QNmNGTjULwC+PTdUyntRKiH6u1mnTIcnXlPY6OWNid5DjT\\\\\\\/LnLMyx7ihQ0stOXxiMovvnKruQXekcUFgU2nXToPwVeEnPSFW8Vh\\\\\\\/lbPRPCmyBWxF9hG0vqDD8i4+b3AiH2k27xpMPI\\\\\\\/YHv6DIZ7KJWMc9tZ124cO7KUxgMGvfrOQDBs\\\\\\\/yaW+tBvekZ\\\\\\\/E7acyBTakbDfuywa9jT1mi8rtXz+2Fzx6zRMPf8xmMVgs6U7ITlWr8hEYtuaTS8x6zMR5x+95rV1P7Tz6S\\\\\\\/gP\\\\\\\/JKIVk6WG0D5D0XUZFVXowK9mTDgiEfH6dolgasgzDM5T8Ek2WexQUhHm0ekjFLUGEjXouee2868mqJc8bjFGmFZ1QHZ1lNSsWCPJIUESrw5n6ZaLnXCMJkinGNpyF2LxQkxhjj3HLtrmhqXfAdWBVb6EzM3N2okEQWeAhy3Yx\\\\\\\/p8r3jgEyrjTGnj3SuJCEJLsK4mQNViy36rfb1mA1F84nGJAFsMqUhtEHY7Oab7McT3Y7zmbEnPgsqt2jeY2SqiUAm9ZLSjoVlRdrGmfLP\\\\\\\/7XLB6GicxzzPM7DL\\\\\\\/FNwS3PCc9Ur33S\\\\\\\/gRaBL4auzg0uYQxRRkQMPUTjRffqPxS5wddKTyBYrNvWr3BJCESApQBurP5J46f7AMWcf7iFeA+lAVJ+oLg5XhjnZJjs+h\\\\\\\/bSiM7ggwECuVzRRIveGqMde+sR1DOnfAuo9Ei2KU17o7kg7rF4rvb5YpeyyOQjMxoUC5byfzGWRliVXnNeT2XPxhcVkKZhHxq1KrEoydU6BUltT0QDxzkchDYIUCVO\\\\\\\/T9XxUbTwSLK\\\\\\\/IO7pWmnPrpcaJ6XGtXpDARZ4Vyx\\\\\\\/5XDdxS0Lzxdk9Gr39xkJwc4I\\\\\\\/tk6zSn+quvp2QEm2AjLUhZEiegf7Dcl+AUceQx0vhqBk3JSz7aPdBvUWPfRAnC8MWWHzOX5WOVGmmlfv\\\\\\\/HlvtwGMBfE41\\\\\\\/uqNbfwQUtvCBieUDblDzeLnHINX1b+MzNRdZheVtvfta8StjHkaL9uNikALwy\\\\\\\/Civ2J\\\\\\\/MIkqn4oM8Mb0xa+tW2EJT+8D5NnBEpvWlfmQ4cQZY1gBLP2NtyllibCFrYeB1chwHo9FZNoXB4zfYvzMsWzZRTY5\\\\\\\/HS1drGccpnHpO0I8otKtvYSy+xNRos8CfS6kumletfEbjzGNFBXXa\\\\\\\/I7SMV9FEkeK+LqW615pLDEuJWC\\\\\\\/RyVpFPUMegAwLoSDj\\\\\\\/T28bV9f7x+KTtK4MDStTvBQv+WJw6wNOzu3nchzTsUbp2Kk7qpgTstusabOMRR9OLozolaOPOxQS81DiCITzkFV1kvEnis4DlPdV8S5F2rLXAyWJfpazwaqf7jbdSN1dpj20aGTGE6wmWrK51PuBBgfFiJ7yTzITv\\\\\\\/TPUSZoo5g6d4lE5Vku5ONmXGFQk4lkdUA1pjZOR300DjTK6OnuuShy2AwjGy462Ux2XyhyaPRqeS77Naymu4\\\\\\\/NEOxD2hBpPnLSLWEEKNDHZ1VZESNQ4AzY4eegLpuW4fV92wtvnwoqUK85tAIf1iWMSjgPESlFoaG6XI0uRBegEgb9vU14a7GAiQxnk\\\\\\\/ejqn5en76db\\\\\\\/vpHVsNrQHoS5deVebpnlOjVZtQUols+4d2J+sVRoYDJlb7i6cbEqCt+NvckZiOR9p0spoNXHUB1H1U2F0suWpKYuNAMBxikDKXAwuOpy8JQbq87SM2mGC7+IYu556Y\\\\\\\/wOyjn4EjZOMBxOQ8Ad2tEuWdbcNIBjE\\\\\\\/WlJ4tYw9GzfNLYqJycdc2dcdSfWICVEbTVCSYsLYPh5sB7lE\\\\\\\/1PMpzJzbPhg4vKyj5dd3Zozg+Z5XZNGmNJVkx2bph92dU9r6T3H3FIEfWOgumfguxWJI+JB42LmqmVJR\\\\\\\/w1uVbPAs0RaATyccjLKG\\\\\\\/P0ENUcf\\\\\\\/VCbH5NjIe1nYHj55infX+KIdr4WjxXj9dlTrKfFxB6YqvuMzcmxWRLw6i6uOO1p+LzCy7yYPzpuuGdsb0Ez6xoSFCzEl1oXeuBRWJvk3IYvSAxYiedqKO48CWnmfdr\\\\\\\/JoU2mInF7oZpOOa7SN0gqQr+gYBYHpzr0Ri0nXVnKQK93u9a6GFWsdVp3X5Pokm4RYT9+CHwIR7oL+ceKY3a55HCJMrjKMOFkaZ6jr6fLiDgBPomX3I1a4pP6o0I\\\\\\\/rZtkHvRmPPM6FNoypyP1wSrUHPEx79ne2yfceyQAr7cfFsQC\\\\\\\/3nGBpQavbcRXOlRCdUoi8dJkBZiAeHS\\\\\\\/S69uNJ9NotJYnkCTZYt1feknFyipNLDOvg+am8pVEyM2apqVXieoZYrBD28frsKkP6h1Rb6irPSiBuyfJv55XnGFG6Jno64NhyUWW7KQaGPnlP3q6kY\\\\\\\/y99ZKUpSSSvCh6abxM4VBiNFilbLxNtvfL2NBDLhBWAJxoJZuMmqmSF+r2GYpboUc4HjBOqCwQKw9ilL2g1RxEXzuvxphDjtLXEPRdnfRC7XmHWHyzcqQDUfKe60IiQo638m6+\\\\\\\/s+3gOz8MLMvSY2NcVXM5Ip9mM1iLE6Ex8\\\\\\\/IL28Xg6\\\\\\\/3aOZ5QVNSOShf7ybjGtsA4jfMq3K5uDVqby0sE98NaEyCQXleMhnJwZRuCdO6KPIlj+CyPLClFbwvH7RtDNSQbL958Zowb1GpSFVQa9jbPYTVFTXIF0nxPNjwROimbusop2D0c9W\\\\\\\/mozVXtOqzgNJCFUKBnGk6QrT9gDY0pcSdcFWsJIq0PSSv7N3kV7xQzMXk6OnZsODA\\\\\\\/9rKuhMBSIxgvaztzNRu7nHxBTfR1wUMqW0GygM2oYA0MyHEaTw8pjGdj6Y8u0XY8sshUtP9BlcOmxYKRfMQxUHPE7DdWKYF1PynlzX7LWCUquLAH\\\\\\\/nJ9CkmQXId8EEvYo2ELhv1lfVzkW8kQIezDE1\\\\\\\/Inan+Kht6AvGiCs9SwCFvtClZmJfXbufAXR5ZzO6m4eLEMMgs4R9FX+v2l\\\\\\\/qH4Eg+HI7ExDGvqP5GoJzr49ZQawviOTaMRfLnPLZZokrPFZoBHpZYEfMwu7RD4+xpg7dH7loz0GE7RnL0Zu7QYiWf0IHi4DoLEZv89h1IhxBmp24itJUbAT3r0subtNO5jrwpUoeox6OAjCWl1BOMLT+nZ2zbY3EO22EZS\\\\\\\/uji3jo9F+\\\\\\\/+7+FXpsqwl6wR7p6hAWDx4+X9RUf1hgY3ZB8phtIxe7lRpTByV1m1n5apeYD+2U\\\\\\\/0MboNEd2l4LregsqxIJ+dsuwsgMx9xo9t4nm1Jxjml98dWkMxchOcZgme8AGbgTqsWfywPALJ8+COx55wTmpy\\\\\\\/Ia3VJRb8kRklgqc3QJL6oVwbsHj44vX668K4hLHCVe85R6momwojzL0Zxuxm2NAqp\\\\\\\/4m0yvm11Elv1VBv8OvG0kKQbmOi9OuwZv0CzBvwnlTu6JoUkM3u0cdgMt5RgpIfluPhz+1TPhaJmP2EYRhjFp\\\\\\\/eYMwA0NSJvuJdawvJUEcIPKCSGnr\\\\\\\/ixnCGM1kXOTDu5Ym5hPJ8TqMW1\\\\\\\/J+Cfx3y\\\\\\\/Ivcccdgcy6WQbO8NvR3IRw44LE2wjhwejCH91QAAEweImOTmmf71i2ee2rLDrBEWmHxTUTQcVRM1YNhVYOaMjYBzgG7w1gKGWXVSUb+PKXKDpHmUEtz1tJnqXM9qQHQbDSVa2q08o2vNmUI2sqCIdjPjOWmSqHOqO6Q8SXEMuFdNV5D8k5bn4we+oyBUIGtGhsngwU0c\\\\\\\/Y\\\\\\\/dIcG2GaM3y44t3HrATNqH9nGzg+WqbpLxZL1VSNGEWVJObbSLxtUOANMajHFpwW51iRLYw+HxA+ROnC3Q54CtowQh4dSDICvNlF8uDQj9Y9UzMz3Kfqw+fh6WOZaABNq7Ca0FtDC\\\\\\\/aM6Pdv4PLslAUB+figStupv9hUc4YNvy+rg52dTjgyP+jfUgd1EU1lxUpd\\\\\\\/7I1P1mq7297UkK+K+87eYpTuQLRhir4im6IsME0Dq747yRrcYE+PzxcdSszqIccQw342oMbouiDqRAsF8CtgHLZVClfTc10pX4J0YzH7SPc04bb6egUN\\\\\\\/Rrg4M0VLaadLWBNvTlB+vXJ6mJ2w9Y3rlts432RxOJIOwSOVNFy8PXR296aa4JSwPXH3NdqI4pNNbmUJbl8KsjWCq5JqtfYZxtBmtQGewo\\\\\\\/3EkjvNNYNEqwFcTBHI\\\\\\\/6tFW9yP+8HQGVQmKlnuVBtzfVW7h81377Kq3U3meWV\\\\\\\/DSh86m5UUNF7yv48OUlP02APkGAcseZXZz1h13jXv7Wyr+ab5Kx7MZuIYZzPfPtxREE1iIK4ceHEaAYKEIE1uXdbWRtAE0BODelk0XQvtqWHC+tT4olTpVk8io6NG6Wn8mvkdIas8qt5E2wMQxpetyeIjn3omShDsPVvHY84nXc8NjYL2gXUlAxpWlXtY71ID5\\\\\\\/rMueiPL9Kojz8kKrtJNi22yvayRYjIlT\\\\\\\/35KnPPzu4Hu14yg9uT\\\\\\\/GLY61mhmU2nPGfAYbv4k9hYxMcByBhCJ6OZ7c2BmmrogEXGlBoYuDfUaFr\\\\\\\/L8NtrUaa8iQnRKQkyCyTKjWOU0ogA9QO5sMRgEgHjYNfRCKysrymbFd01NjrDPltF6cQE+7NgCEv5+TwjbqSstE+qzkGk8mGZmC4tH23yt7rT+DmH9S\\\\\\\/7B1pr3rQPobtNL7ApdU0WkSYocGoXwU3wbZ55xRNBrOe6GVj6WEh3jJl6plRczIsoVpSCfQEaJOTSQjfk5\\\\\\\/6i5rcOwUdfD+kaQQbBLZINmZW14XgtUpylI8FwzieClNfCDJyxC44DQTgUML3Rte4uAiMsnd85DLLX5WgYO5vOloYZrEHMMO+KnRGS+t9Uuc3xaYytzT4a7J18Sxq0OYylor4wg\\\\\\\/uudEhVyi2CD5SqRZQHHmLsl6NBj55vnwANlYFNeuE423\\\\\\\/fTyoK58FtSqCVvWKRjhU3\\\\\\\/\\\\\\\/eY8uIiuwhC\\\\\\\/HqSbw788VsZpaOYeXqoVMZo\\\\\\\/7uo9d\\\\\\\/+vZ7ptfxHeTPK4\\\\\\\/sXrRB3AOkIvEDDACU5lWoZ71qo6TK94+tsHz4G19b9uIAzAE+HnCxZajXEvdSMYMj\\\\\\\/FTAJbyQyQ9c3tKJJhAW3j7nRRltmT0Rh6S\\\\\\\/v3wKPUcwf\\\\\\\/ffHMzbMFETLoBn4NtmOnejCkcPqX1yePYnuCBaP\\\\\\\/VXXUpPeHiangipZGiys6UGqSfFxPf+WZ0eQY6S+0N139TEmzbCBgXDLZMf6hB41cjrforUHZAC9hUM2IRWKA4Ewt5b3dUgOQMsZA7YFw18e+pGzhKzJp9cd8sOSx5vFqBfJSNio6DSESyq1sWTFEtXcfvmbdzxPwSYWCRA8Sr\\\\\\\/ONV7KBGY3b+cIpcq9XZjLGqyw1gARGz54jjb2ze5xI6V3zLnLvqAaXiYKl2APoQ+1AhvL6gh39VDzRJanGUHBOJOWBH7OWSqtA\\\\\\\/OmOrQMpX5L5C99Bpy\\\\\\\/Reb6qSF7tlYcmnCktMIQ9ZDtOIFffbZjnEaT+XQ8LC+CxfX\\\\\\\/99K6\\\\\\\/rAGF6c\\\\\\\/blcYS4wEHf9VIuUkjKHibwvMVRsxYlBzwsyxjwbKGm+OYW2fnLJSEvqQmrvOgZeDC\\\\\\\/Vg1v4LyEtOMMMX2g1cw3I2+KktosGV03uGYIG0T7+EvMOMd77qZOXs9FTe\\\\\\\/Oi6HnO4tMTz7BmsOBmLHTBXgitmwd0q5H3a4npYf9sad0QRZTwcm\\\\\\\/FfG+m040WHfkudQXhqzVfl1lzwhOGuTTOQUoKmhiq56QKsbAd3jHpFbx6qJxaLLT\\\\\\\/hbu4AvTXoJca1NxEj+0eRkr7nI9v\\\\\\\/l8YQJNARYqD7EjVdhRkJ9y3weixk13m7KcFw3QlGJDazjNcL7VsGZE18Kp34UMwZjBl5GnbNnBYfpYZuR+eHK0ykah6SVquWdhSuy+LAeG5BfdJCRlPNUvzroJ\\\\\\\/QvpeXulkeOqRdRnpW2aKpyuCthLZLtVe2pdUQVomQ5hhTkFOehiOtnoN28bMfRhvqvvtH2NkhrF5WaP8TocjzGgHe4B3\\\\\\\/EHeYIl6ABWzFtU8wCj6j2KOP6aYSXVyqe2vodm9\\\\\\\/yeWmrIuv\\\\\\\/M1GObOuNHME3RqhO5CGu2KrT2tBMNhNsKELk7qYNM5rDJdVazbHX\\\\\\\/Ey+uJARpjKjy+jOdTiAShL6WSf9AfYIAZprFx54lX1YYuc09vK2M5aMYLbPNLrp9PmjtdCdBV2UmpRBiYm9AEjx8tfYztYwNyu4grrGHWa1zSt4+NtokKwmpf3uWDU6RHvjgvIRaESFLcHZm44nR4PG4uqVEQEhncPrVspuJY7hruCUSI+Q+p7hDmdYiFQqdIUoPkAdnBT+ofJUQl7VyrSjdWJesE9nASiBhJP+vEXGekSHEJroa9Nzc3wX\\\\\\\/cIjX2aQwFRDAEbbEe\\\\\\\/6uheZB1eYct70rZBFuwWJqnpJ6dBaEcFR3sNtzX9GlNp3S5wk1alXIDLK78J2BBjTMPaCLumPAyTrW8j\\\\\\\/+o4xVcf+nGgsnS78T\\\\\\\/+R2bhfEUVLddDq+wqMBfsnnP1itSs6r69baT6iNoRGTNcqkTbe4OlWEHWqf5Xfax2H0aqrllds4Y\\\\\\\/y7QEf4yh4MWw4U6MjQMzE8LAeeWb6\\\\\\\/LR7xaCNGNJQhQy8twY3goz6kcb3w6OXI+NhUvC3dr5GraTGLYVzCF9QDoFxHIcz\\\\\\\/MUTHkT+YXu7eJvFWRZKNZVZDE+DhfA7+4pycJa46nGn7zasavT4AtuX\\\\\\\/HNvlaFa+6BJuTAsWLnNkXS+FQ1BE9bBc0RaYiOj81xfcgY2Trbf1XW6fByG7QJcNqnZHIiG4QWdTmwz4slGcHDsLS6nfokcyXb6ENTmUi5ThwyvbfPGZw0G9SOekdC03ylEyQ65vv7Bnv0pRvpdCMA\\\\\\\/lCQDjsb8BzSEqSP9sNjsj2rYnla2K0T55ZwXm4hqs+fvbk0YzF8pXZOjJclqWutTNhQbzEkd9uFX58Gh77zn\\\\\\\/DfGAMwH0s91avpSVwJJxmA\\\\\\\/e9Jh7FFM5vsq5G9wLi708q3drmojRR6qJpddQy\\\\\\\/\\\\\\\/yWD51m0pFEznTEt\\\\\\\/fBIYOu9b9dzcMSf3r0Qso382iBYz\\\\\\\/gE1oSENddDTFhGgQDGfcOA38x+CRv\\\\\\\/+Pur303kS7nBM72sgDXo6Nql+Q8ebs5L3hKJG9n7pBAMIUhFA8uz7V4milaKBSxnIA3OTWUPUPzYheF9K2te+Y89\\\\\\\/dcEmLPxrObWFD9IjREu6YJkYlke1mcpnldL8u\\\\\\\/Q3bqxynhOz6BW33n5AxQnIZKMrIJsowGxHLU\\\\\\\/RzIuNjcB8p9BFilhkYbrTQoseDviIOOVR8s9ic+YkhzjBd7infUtI\\\\\\\/KNyJN\\\\\\\/BUlfeaDMVx+o0u8WEl+mFd+CMx5OHWM0rGmWYGNCMBMNe0hEcHxbnDSOZCeh1RttXX1Eig+VWJ9Z9YEvHRCgv5gfOS6P2ZX1Fy8mjR5FWtkZ7X3y2THiYl+bvROqemnArW5nDOG2qVDTjJ+LfsWz2+tEpuFzqV13RWwCpAEwj0mmiXvtsC7JeDN0c1Q3nZI1D8YtqDPhcKlnjgoNaSj7qFXElwahiU22FTD8L\\\\\\\/yN6mId\\\\\\\/Xi7TS7PJQu\\\\\\\/tlcglYbJlOWc62Lb3OVrO68\\\\\\\/Oxc6i0gKgSKe50uP1WrrSLbyGZGJC9WB\\\\\\\/u3igNk2UvFsnHX7vbHhPgidVITSBik8YeTRQuB13GeBvKfrGNI30+2SKBPS0cmwTgC5UV7Tqe8F6RcIe0vtQloqLj41mTozc3ZVTTHauHD9ioojzOE5P7TBAUekeCcYFAVNPMf3dyGAYYygf\\\\\\\/CInO8tjF75\\\\\\\/WJO8Mpz1nhC2MDryh08RWttFk7dd6layVMwHvbqr3L\\\\\\\/VDEn9IO6Agd6+U5LVzv\\\\\\\/w4lJinoVE6kvE+Xdtqw8mX0ot1PbbeseqqX59bUyIuaVaLsgjU4Dn9aUxiHP\\\\\\\/dwQoBu1yI0L+4XeLo+lMO9rJtCDUmEgqQHOCu\\\\\\\/+F3+jqTO\\\\\\\/9MMxGba4oldd5Tp45NvHct7EvdCcax8Zfe27xtaCavVQhViTG81qSmy9fDS19hpq12Hvay+G7aiZ56odNbe6IBRBXO9zd2SpOGBrYerMqRP7XebuMJFE68rFn88sR\\\\\\\/dLzfvW7LQuUvHGPrjFTZcfu9\\\\\\\/2NmWXCS22n8rVOoRGFJ1\\\\\\\/LavDSjV0gMpNGkVg5lpNowgUR1FSz2K\\\\\\\/cZcS\\\\\\\/7ed03YB1qrzocD1BcLSBeoYcBNnMC1nlNPaPok+fKSDGXvb0j7xwUsI31qlY49q2qKK6LHZx2CkotJpKcVbThh8uY78mBGfMcZxTjMdEh4vpH638SzQcAI0xENv45oIzR6F1k6KllU0pDmcxQmxEABDzKSs0gbaLHJMOlxdjAFkqBXf9ygYA5FPHadh4EZqVV4TNKZhRHOjjC9GwHxCrarIlIgBYAU+AmCtI5U+onaWOgAOzn\\\\\\\/uwKV6X6MAMhaWHkX5pvi9Zqch4maRh3CnJVqhmm4F0NKP7mjQ9AHDe+Wr51ik+7NujeVO0EE3adgQbaC0QSaVxWevVpUAr2aDBHjG\\\\\\\/\\\\\\\/rB1IbL++EOtbuVZRcJpY17LT2gCc8VLIPGE9AMGroHnzxBrlA1iMbiKTSDbTnwoqmdeDimWr5In\\\\\\\/0PGLvAAdvPHrgnxC3ICjA56MFAqDxMK7oAL44OjvZrLj9842JaDnnRjF\\\\\\\/Zx22sQNuabQ9Moy4RlU8YsurgCCteatAJ+7LAGXPNWPuN8PdMnmppGM1+uw0Gx5f\\\\\\\/M2mb7cvOZMn8Ete+WfhjxYJietlicdlKLEFaMMx6\\\\\\\/uv76i\\\\\\\/Y7GtyVSkTF+bVDXyf\\\\\\\/d95hf\\\\\\\/c4FVPzPQzzSpccX7chwEeigrkIlXsaRBaYgGCGg086ZSX2Novg9wXzDDatGzRZhL3CoaqjXdzguGLFUo8Qp4iJs+\\\\\\\/UG2CsnTknMNnD0IdfzIP54ixYM8PnLMbK85D5gFaXsdHiTq9Rmh8eNzizwIhkO3XS9eknASCacAUV\\\\\\\/lxoZ\\\\\\\/LjeiXewG9zPehiszRrAl\\\\\\\/WwiiH6Yqyja8wZuM7srVNnLmth8B\\\\\\\/1L27X8DG5ifFOgrT8Js\\\\\\\/sr8R+LLMiZnKlJ0XvVfHd\\\\\\\/dAV3bdAbIE4gJ33OwftIWDyGWgNU+0wKzKahmdhvTiZMlCSk9NWrG61DIjfXqaIH0gZwuFApkrAlH+zfzvz5nOtZWNDS+znDCoVLPIZldHVHwZgxfIjKAd76OrsOqRfErwuaFycw67XedRdPxRp9MfD3t\\\\\\\/mTHQigspeNw8mRDx4i+k7NX1B7wILMxsLYFlOV4pGHjs5teO76tY5Vv7WIyu57+q4zjPZ\\\\\\\/YK1CyoDMbOTPa64yKEjkbV4iQj4rzPDrJQxjx2+KfE39GsxyXWbYCDwxHDTTEq5+cQMN570oGQjlE0Th7eiwSvUmhElgOwZ+FeuNqgvVs\\\\\\\/x+2rxAkJnFpxmLlMQBXnm1z6SC0G3cN04Rs4Cvft2OGr2gh8cFOhMTD7Y8VHXpuMPt0IqWV83b5ndZk+nbQHKGDAzrioYTXSIFvDqbyoFcAQoGaKwd09YHmhCMdanO7v\\\\\\\/z4FI86ZlsKwu8H7D5YQBkecD8UIr6PSv6IkIDQA885jKAFfzReDQhIJwEpKzHyMvZrjpJ02+e60DjSoRC9PdbWLoJZFFWgsn19lljVHXoSDh3fpaNBtMAsITpIwl7X3OxwYzUPJ8\\\\\\\/w2FR2XQMn36a4Qn1gUlVf+g6sY0EospbqOdiZVmTS8B+Q\\\\\\\/vjquxmV3By8IhkT4Vz92Ni35RYujEu5\\\\\\\/J5IlVikoZA4b\\\\\\\/kfxYmJJURLVqf1em4ZKqlLLOGDhMDWNV9hP8xI0PRLMiSjKZ2lK9fuzalvY7FhyKih5JjZjCNAO45hTJCNPYB9S423eQLCdhNtuiteIb7SUQFazchNphT0xqGC9ixIr6L1By9eYa3O1wSQx6VLIVKkRajfoSPceSQxVt89wzs2bVBp\\\\\\\/S8LabVOKxYx9H5oRVEpmhz2jvkJCpFH8Ulf9uC+t1Inp8hBIPuiD5\\\\\\\/O72McSX93WwhiH2zAhR\\\\\\\/OZlW4fH16Xf1muxIr9YTdl0pgewozWc6czF10zx2GxnhznD\\\\\\\/FvQsOd1i61JMaOYiM65vFsEv7hONvnHKwwdpLhZDl8eait0wlTwK+YTN6KZFZNjZp9kCJrweWZm8vpLEtl2rblUoFlQe5VMPj9OFBepvLs7BQPki6agKdsy3A3aDm6mz8blABXWqtVd\\\\\\\/\\\\\\\/VBERiiPty9\\\\\\\/6tlPQmxIJJKlcKAKBZIw191OY1Q6M0j+d2z\\\\\\\/PaKVg\\\\\\\/DZYZCMWVwo2KH92dZg+GQV2q5e3vJ53BLD62zWHnWUR\\\\\\\/OFN\\\\\\\/54VPyxKw0USbJFo2LlfvwaX0PwdVTNu7BYI1l4ryTiLoekRqC7gIhtqWwX7uG5J9h68xLOBf5VwVIjKg9S5LZvU90UMuXTXUo\\\\\\\/rbTnNaTVy92f4ydzHy9SMdrmfuPsBUW1QyWMOcsVI63UUfbVd9QQoIWaB2BXjIgDpSZSr83Cp+i8hdH2LjBQ+EdgACrhmrykDlIUnnNRTxAm2LSUi2GJqnz3rb9+bRdd3jTEF9fL8mHneJKQCOBZ\\\\\\\/fhlY9\\\\\\\/VrDsbXUKVbB6h1B38PiGZqepmonug9QxCqKH2x6igpoX4z6ljPrs9wliCL5e6GVMEUzP+O2DrWNThMheBs+o5Q9BaX8wepwnt+jKukO+JmzjXgK2hrlv\\\\\\\/7zuKI9Fx2vyAuDkytx04xKfD1OsmdpmJFJtGw71+M2DUHNgchnBIFPDWKycDy\\\\\\\/dYG7oA\\\\\\\/5EecF3uddduLT4jTTa4NBOaN460h9alD8m11hjKPWV2kjrIKVPyEpQIb3ufKbnU9jSHE8\\\\\\\/RrRI1a4G4q5RPJmYJwDEdphAbBKoh3Sl3dktZpOfgeVUITRUPCTVAMV17A+Z2IPEmyFN8uEHU8YFqmIuUrlRKDFJ8HyrE7dzaCxRy8FOvsKkde3ok9r4sEpnpKsOvaNO6Z3X0g0BbDUD9YGosbtalaUhwvKgID4NGKanEtrd1qOpeOf6F\\\\\\\/vwIohIwCptGseq00f1GfZpXXDIrL95ZaopLeZgM2Kt9rp4W3VYdd8raXB0BJtBhI2RTqtPUE8DyGP2sGZ+olBn5\\\\\\\/ELQVh+Asq+kqqv+Jj76R05rCKN3uLm5SQ6rkEDHdK7k8PpsgE7UdAzRj5zjbfcXBjQ0+TIUObvaQdoT0UBlcR7kapvaL55rbGAQZY9uH\\\\\\\/AXdxxvh5Yvp2X66CAyk7B8isYo5MB8ywnAkYTC1tQ8ZuRfneidtchM3mAQ0JmUxGcrqh\\\\\\\/iOUnM0I0mTgn6fQuYjMyxF13teOTuNiBO\\\\\\\/1QRN1YsDWQ1dkEIO0FJj+Wa3sIQJCmbWBE96SRQ2OBM01M\\\\\\\/idXx8Z5z4noYfgbkhPz2MQsbHNB6wF9HxNU7sgJ2Botb6DYp0WUzvxc1Avn3O\\\\\\\/GPeDVr0sQURZK9VSh6BznE5aEOasEFUo3gPewndUzbdD9F5WJa1QXYB3CS8TjfmtPjrDhKO0bOWFKM1ethrnEQbvuS1yWMrLhL+B6hm4eSAu\\\\\\\/+1xa9XB+Cr1fdffhRWA5GQWdxvmYIOTqmnOYjYOI5ermtKiMASW4PzdpIR8Nwrnr2S9HcHnUZG6kO90chG78XImX59mEyh9GicvUIPlmrpmzNTSXCYHBKwn+3ptMJbcMsxqHPj1bmgkrnwph9o6bUj7vCaN8RvqIH0cvZyZg51mpzd8w7fZvewW\\\\\\\/zFOSiArZiEgAPIXfd869hG5UrgDqF7j1V8Uj+bGfOiL04N\\\\\\\/U++yGjGeZkFzQlh\\\\\\\/lUHjSaVuQFcKtx1iYdtgSPr+Ic1OWoOw+iE3183BzBQwFf5tDiKG72bFCFiz3cX84cE\\\\\\\/XSSSyuNho4p9gRW1DIunvCO8SLav4oAlKkMHF859d+G3zYlM\\\\\\\/8+ZcBi6\\\\\\\/k99cpDIDv3U412XuX9KW0r2B8fSP\\\\\\\/z00wTCEsybdn\\\\\\\/m0uPMIYa6HfGHHmAStjY7SsbyFpkYkXUZ2DJ6uhUXzzALUil1Ezi07\\\\\\\/nSQ82KssHdzcU5+XrO8m8lK7oFGT9JYyUFm\\\\\\\/0A3WoEju3O+3g0ePw5d8rdM3AfjgbYkdBnxy+uJOQecHebT9fdGxZA6LHWunQOOHCOMoOep8kjaDCburwASJbu5zkonnYobRb5zHV725sJbloNG7cvkgttTJKBvKkGfcaMq\\\\\\\/GloyfmhVDTJ9NLKTsibkP4TjnHmtbvyg9RHAM4\\\\\\\/d+jdYlIo\\\\\\\/5ffpbp4njI+lOiwERVX\\\\\\\/DctQ3zm+qXxdkURZtWsNCOp\\\\\\\/8oQMBU5E3f774QZwjIFYIF9A7p7y2zxEmua+P97\\\\\\\/b+pQjicAoerVPOlRZWK10WDaNNqkFw9b3gIVeBDFE6AJz8uq4xdSwF2eRzdiqmTu+txbd02NBGDkwk5GU6V6gBNAP5u21RWtPT5J07NOLC6OCZ2550opf83nZoe4vxqq2bzLah8jrpm1ZBfmf9RHEjRRjT9vjhmPXDppsmhL1yEzWbXpXEhP0DM9YKplTi\\\\\\\/z+k5ocamNMOYi91DZAe8+XTpmkZBmF9Wa7GzdsirT3RWfgaRyfUa7qJazsAw1emvDsZVW8JwWRP1P6kDjVbIYsPW+TdrUePffpR3M6kkKkWlniDLj71awC7\\\\\\\/R6db\\\\\\\/Z\\\\\\\/e09ho1mr969pBdiw0htSa82y5EuZ5fv0Cut\\\\\\\/GhZz\\\\\\\/yP\\\\\\\/lF+5loKEOtvDdMmbwtGrVefmjVJmOJOgaI889vlB4B1SyIM3QfS3wRiYNPYpQpAOgPz7y9S+JfnItTUuOzBWjV76xch46eUoXmeuTI02lIpX8t5+xhUfdEs07qaTxVTqLkzLvStmyNomm9XCBowFDvmLjbAirzt426Kk1OYlwKuLwDeJXvDFHDfsQq5WjAL7SUDNFXSMCaIwP8jhg6dDvRSJfMCsI4NTet9hpxhIbjOFfAAUHIHX+DKvfBiYjB6Ocyo6IvuGNXudm0q4MGLyZ1xu444sm+jeJNup\\\\\\\/lS7ydjBCPS\\\\\\\/5tHFJTd6\\\\\\\/w2Z2TTa8UpV4XYF8xf8OxYJ0sS9RHKW+\\\\\\\/V1P4Ze6ziZiw9zLrLotxsIpdDF50OpNyOJE0qFkbH\\\\\\\/CLeLVcNy9iZf38\\\\\\\/mzk9nJ\\\\\\\/Klap+uJmItbQWT\\\\\\\/uPu+k3Di2+4J9aT9pTe5HRs4x80tFIH2sxUoaxZu+nW2y2\\\\\\\/25szTbiHPp78sZNHmEs4sBNJbKN+VRNvJw81c3PL0W4XGpNTeO9gRIQHFxI75T1gIHBV\\\\\\\/TAomCzf+9xIS+7K\\\\\\\/tqX47Ak5pChq64+tSMX1pgCq22AVKe2U4m0m1yW7FUO8edXYKSRShDVgS9xBZ3HAZhiXqXfNc7TmCENWZcy7TUHoMyPsAxvoRV2rGprs4aAr4TZGQd+Wke7635cBmbq6ezbd07PdKTVFke0mzDJKU7TIVbmbZCGOHm\\\\\\\/TQGxCy9fpqasy4uzwHYq1TKEl7hgLSuAIJINXeMbeF22RNY5cwo6XfNWK3ZqtnKb076rkhJlhFDp714VQOjCSkLZtIG0s9wXoS3XJrkb0i1hmi90SET4YdhwctCOY8BwLZ+DPDUnuTntJ1Kzn1afT8LZBct\\\\\\\/EnDfer1JSqrHY9CF6m9AizmOpmMJSmKFn4xclndBNaflKeG3GY1L7W4nuA3\\\\\\\/fE5dQP0kWoWJiaPyVPi9DaPoM6Sx+umu7BVKylCEOd8ixgmaGooSMq\\\\\\\/81XIVM+hrvn9sd3L571scxD74XDxZywKVc\\\\\\\/wHLcrwFlyu27yR4VkwjDQa5Qn5ZtriqpCEliA69bXB\\\\\\\/wHlc4uYQhQqVjOITyq4m53o0A\\\\\\\/lonwB6J0AbHLdv\\\\\\\/e8T+fpMf7NGZ83IGXmDsTWFVI0kjZSA7kuZyzBvGCBx7W\\\\\\\/aW5FLp0eWzSjSb4dTAAkR+yg4o6d35sUxcUo9szdmEZ\\\\\\\/9VBdLbLKmQw+QPdGxnD72mLQ++AtORolRyHkRjcHiYLKvowUnObggAVYQbjJQtXYs0n+FTj56ltjhEdM74ET\\\\\\\/ay9zlxmqNtbjDC3nRt4vs5jnZzgKbJpOxE5OGaPHxPQ43ImThbu1ARX78Ql2Xm5x\\\\\\\/rU7E+3ZCrN3jLo4QRAQRHmOpz1781+vQ+6FaTX8WluqkODmbN6Iogxq6Fy0yc0mbUoj5fONe2+U\\\\\\\/9Nn\\\\\\\/i2nrJrCk5xnArVt+sQUp39A\\\\\\\/Y53h92bnHefsAYJxxoBJQ5nG4Jn2z+6eNZupkAMB3hDBADnXfdTw+zYOX5\\\\\\\/cm5AgSspZHVwPebQVfDflI9PlRPgSvJjLUFyb8uJgYNvePKihKgdsy574vV+CzNboA5u0FFTA55jnnDW9jsA3Rboy20SPsNliEkDl5y0WziaM6HG1BYR43q56qLzAZCWJBgeHuILmXhg784cF6Na+6+FG1JjOYLHUQz6mFGBjHJrcbM2zL63oKDvVq9djF2rwG7c0msPdPs+K7+DqkJtzGe+57ZmXEbSbL\\\\\\\/rabln4PImWaSU22uAg0vkGQtQiNytcnEFiAlVys7hqPdjyVbJXxtAIGCupD+UNlmq64XqnnOFq28yxhSlNFRJPJanMqQMvWMPWACDUDJlfsvWsBOsLw0+OW69kbuAlE2FuK\\\\\\\/UwG3Z0kytP+2BJaqFNrxevx62o0Oa7du+b5sqEwXrP8Sbof7ndwZoZNrE2s9EUD\\\\\\\/Ae0xVwkG6YYmztvO3+s0\\\\\\\/FIKdKSERgb2LM1FEV5gE3Kk1M5RThHo2gXs4OaI4FiUgAo9n617GNUTMs\\\\\\\/BtLwoy0Pevo22NisEMLDxheTxO+uyuVqt1g4OIVpsTQz1JZdIdCvhrQfXyAxYDuFhfP0KeA18QGzXabTGlXDA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022R1fkzjQkXg9TNfVh1gFErA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c19572dd34cd6e417f72b4e2c946d59274e985cf2292a8ef0d8e289d54c26370\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ysJkhnav5iiZH3ec8AY20g3ARfuuoCeCEobZToC7ClONdnJhVUYYqpuC7X0mYR8wgEd7mcBCUcvMYuRt1L2N9Khd+STpWjU0V+1QOki7pDmAEo8Nq4WNYt6o6F01eVUskQqHqrBpoZZQqJTPiJxcT1RALE7lAsNIT+D0NkRe0AJqardBlEZC65pFPGKgc\\\\\\\/aJL4kyLrD0g0nmF+0cNa6vcxSDTU3X2eQtya51O+pAjyz+vZH2ztuheGrYqECLQXXK0xyLU0onr02v1saITajBLQ9OxdClwy3CI27cqzmXQ0stxpCBlVO5HfEVxg4TqoFFZ8rkEfRaTF1Khz4YsOEVu7KlpO6ueVI3tMVrke\\\\\\\/Z3iY0jNUg32JB+r3QTlrhqCeU8Br3pLHUgozopAATfGLRruOvYsFbwa4KHXlv6ci+clXLLP\\\\\\\/syh6DIimqQHdcYTErzDuNsUcwYjuwuUlja7qh8UkHAiQSmyNZlsEvbjjLud3wrXWlqjEC4Q2bohUAxrtjQMWprg9SBoKv4EsQy728ntv0EggC3q7AyquHKX9wfYhmZQ9r61ITqbUAuAp9rzWtpFovO8sCOQcUbRXJ89OWUtOvdhbnq2lVuVDDxM9Zkjtcrgkc\\\\\\\/UvCtpBEk\\\\\\\/RkDOsmtnWtMSdnS\\\\\\\/BiwusPS+qwKtQJ81PPELlA87JueQ\\\\\\\/910bbGuNkcMJUjzqNnIzQIgbge7LxOOAk6rF0I\\\\\\\/ZeDkLVZZWOPQz+pENPkE\\\\\\\/tZzPa1or1ra61dPIHpyRQNfnJ2ObfCfaJaqi6fo8yWn408qvFphLrSYr6SyVVowK2DhiUg1+JdetrOz0gwd0adgvBOjlQsMEOsBK9S2gKCdwttll\\\\\\\/VpVq02Wvp9sBMANG+uabdQ8vrNFw2hyee0knrYZDz65Hg8nTDXyukMWm\\\\\\\/3DN2uu6CzsqprrVIgyzr1kmAU1ofi5Y+2J6HeswEA1VI6BF85vWJG+A5VFBJulvfyz3pfYWkHUNV5ZQhJG3tbJMy0sXyZ1rOoOFQmWkVa0D7Xjk1rYQXTGe47PwgF5w\\\\\\\/aZ8kHfBlL+zTlyEDYr4+5GfmiAVyeJC9vgOBMkeQl\\\\\\\/ZEW+qI8AHTKtZhwbdN8Yp+RGNLgMqb8JLZi\\\\\\\/\\\\\\\/+2HS4ROm32UWhSQAAJFT4jAj4bV3x6qV\\\\\\\/9SQ3u56Q7gy6kMu4u+tV0wncTkzBBuypK6G1v5tuAFJRKR6w\\\\\\\/0F+RbUmzyoJ\\\\\\\/f+GY3kQwkxfv7R2zjAj4XRXI\\\\\\\/lvDAFXqJ4+llNFMOqtokE1JOEHnIvTQSfjedRGo5chiX7CtzTideS4Ce8ydzvHHMleT2pZ2tvQ5kEHAmWRNC4tqcqWsRsuYSLifF+fs9x40VRRtehTNXREXVNhxI4EID2a2\\\\\\\/IfwFTNkkRNU7YcDXWuEULjgeEL+1oG422VIoYoaMyKA2rIwviF5BK+MYdn8QYsLjwu\\\\\\\/9ThQWVlM8cLlDJpBpi3VA18FiuR9hQAKOEGSPHfoikRBc4zs0gM6DGlUORDX0zCMa7SnMh2jTevyFLeELp9dQUeX3r6RHj0fRdDyHnsnBjlhZh6PwBmtWAwwkU3fSxlBqg1YEpEtufktMIfdllKnKm5p3nE6oKboEus9+Gon7s3BYQZ+gv3i8\\\\\\\/KEAx5pBnHhqAeidNmO48U+zo6EI8YafhwzS0i2RFMBxSNRnaorDY+GdmrCRUZhWnusaKC0i4DWAjJMjY8mCBrBxDsR\\\\\\\/asJaF6A+XqLYVjzEpQimKfeEjoMSL427099OcF5bBsHgFXJNRoqesn6udf5Rc+aO85wI+iGw5LVp+uNd33QL2TQ20RKQPeT0XrFZXkaR7J55MWMxh+\\\\\\\/IsP6ptmfJ0qzSK+rS5PmbtVBNwu0iFEWc+izze\\\\\\\/6IW6sMhYR2I9tQydiSuWXNgoWYo6M6bkVmm87yf+mwwcXFG1gobgZ78LxolDnW7E60FtMPnfa3PpqeBvJAIczWhrbn2zGyHYy1pu9R3MPsXScDjSvX8EKXi5nA\\\\\\\/Y4PhaWgguQPxsx3rYEbSwQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022r0P6otdXboTGnET2xavOOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d9457476ec2be5533faae4d975d90bdeca47ea658377c0d7a897b8a6eeffeb97\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459230 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/43" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zP6NSpdFrNC5lEzIwF_diTNPR5wsfHJtYtXSFa9Sye0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/43" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pUR33FimmTXA0g9UMyQNByjR7gFFq8ociS2u8ZYLc04" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/1/35" title="Linkage and Physical Mapping of Sex Region on LG23 of Nile Tilapia (Oreochromis niloticus)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/1/59" title="Genetic Analysis in Drosophila Reveals a Role for the Mitochondrial Protein P32 in Synaptic Transmission" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459230" id="node459230" data-pisa="ggg;2/1/43" data-pisa-master="ggg;2/1/43" data-apath="/ggg/2/1/43.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">John Z. Yu</span>, <span class="highwire-citation-author" data-delta="1">Russell J. Kohel</span>, <span class="highwire-citation-author" data-delta="2">David D. Fang</span>, <span class="highwire-citation-author" data-delta="3">Jaemin Cho</span>, <span class="highwire-citation-author" data-delta="4">Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="5">Mauricio Ulloa</span>, <span class="highwire-citation-author" data-delta="6">Steven M. Hoffman</span>, <span class="highwire-citation-author" data-delta="7">Alan E. Pepper</span>, <span class="highwire-citation-author" data-delta="8">David M. Stelly</span>, <span class="highwire-citation-author" data-delta="9">Johnie N. Jenkins</span>, <span class="highwire-citation-author" data-delta="10">Sukumar Saha</span>, <span class="highwire-citation-author" data-delta="11">Siva P. Kumpatla</span>, <span class="highwire-citation-author" data-delta="12">Manali R. Shah</span>, <span class="highwire-citation-author" data-delta="13">William V. Hugie</span>, <span class="highwire-citation-author" data-delta="14">Richard G. Percy</span> and <span class="highwire-citation-author" data-delta="15">J.  Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">43-58;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001552 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459230" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John Z. Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BZ.%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20JZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BZ.%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john.yu@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">john.yu@ars.usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Russell J. Kohel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Russell%2BJ.%2BKohel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kohel%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARussell%2BJ.%2BKohel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA–ARS, Cotton Fiber Bioscience Research Unit, Southern Regional Research Center, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jaemin Cho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaemin%2BCho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cho%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaemin%2BCho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mauricio Ulloa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA–ARS, WICSRU, Shafter Cotton Research Station, Shafter, California 93263</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BUlloa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ulloa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BUlloa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Steven M. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BM.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BM.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Alan E. Pepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BPepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pepper%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BPepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Johnie N. Jenkins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johnie%2BN.%2BJenkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jenkins%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohnie%2BN.%2BJenkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Sukumar Saha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sukumar%2BSaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASukumar%2BSaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Siva P. Kumpatla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Dow AgroSciences LLC, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Siva%2BP.%2BKumpatla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumpatla%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASiva%2BP.%2BKumpatla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Manali R. Shah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Dow AgroSciences LLC, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manali%2BR.%2BShah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shah%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManali%2BR.%2BShah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">William V. Hugie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Monsanto Company, St. Louis, Missouri 63167</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BV.%2BHugie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hugie%20WV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BV.%2BHugie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Richard G. Percy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BPercy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Percy%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BPercy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">J.  Wendel </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2B%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2B%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/1/43" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459230" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459230/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/43.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459230" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459230/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/43.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459230" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459230/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/43.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459230" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459230/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/1/43.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459230" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459230/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genetic linkage maps play fundamental roles in understanding genome structure, explaining genome formation events during evolution, and discovering the genetic bases of important traits. A high-density cotton (<em>Gossypium</em> spp.) genetic map was developed using representative sets of simple sequence repeat (SSR) and the first public set of single nucleotide polymorphism (SNP) markers to genotype 186 recombinant inbred lines (RILs) derived from an interspecific cross between <em>Gossypium hirsutum</em> L. (TM-1) and <em>G. barbadense</em> L. (3-79). The genetic map comprised 2072 loci (1825 SSRs and 247 SNPs) and covered 3380 centiMorgan (cM) of the cotton genome (AD) with an average marker interval of 1.63 cM. The allotetraploid cotton genome produced equivalent recombination frequencies in its two subgenomes (At and Dt). Of the 2072 loci, 1138 (54.9%) were mapped to 13 At-subgenome chromosomes, covering 1726.8 cM (51.1%), and 934 (45.1%) mapped to 13 Dt-subgenome chromosomes, covering 1653.1 cM (48.9%). The genetically smallest homeologous chromosome pair was Chr. 04 (A04) and 22 (D04), and the largest was Chr. 05 (A05) and 19 (D05). Duplicate loci between and within homeologous chromosomes were identified that facilitate investigations of chromosome translocations. The map augments evidence of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 versus segmental homeologs 14 and 17 as centromeric regions show homeologous between Chr. 02 (A02) and 17 (D02), as well as between Chr. 03 (A03) and 14 (D03). This research represents an important foundation for studies on polyploid cottons, including germplasm characterization, gene discovery, and genome sequence assembly.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/cotton-gossypium-spp-genomes" class="hw-term hw-article-keyword hw-article-keyword-cotton-gossypium-spp-genomes" rel="nofollow">cotton (<em xmlns:default="http://www.w3.org/1999/xhtml">Gossypium</em> spp.) genomes</a></li><li class="kwd"><a href="/keyword/genetic-linkage-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-linkage-map" rel="nofollow">genetic linkage map</a></li><li class="kwd"><a href="/keyword/simple-sequence-repeat-ssr" class="hw-term hw-article-keyword hw-article-keyword-simple-sequence-repeat-ssr" rel="nofollow">simple sequence repeat (SSR)</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism-snp" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism-snp" rel="nofollow">single nucleotide polymorphism (SNP)</a></li><li class="kwd"><a href="/keyword/recombinant-inbred-line-ril-population" class="hw-term hw-article-keyword hw-article-keyword-recombinant-inbred-line-ril-population" rel="nofollow">recombinant inbred line (RIL) population</a></li></ul><p id="p-4">Cotton belongs to the <em>Gossypium</em> genus, which consists of approximately 45 diploid and 5 allotetraploid species of global distribution (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beasley 1942</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Endrizzi <em>et al.</em> 1985</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kohel <em>et al.</em> 2001</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stewart 1994</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wendel and Cronn 2003</a>). The gametic chromosome number of all diploid species is 13, but significant differences among the genomes in meiotic affinity and relative size led to the recognition of eight genome groups: A through G and K (Beasley 1942; Endrizzi <em>et al</em>. 1985; Stewart 1994). Of the approximately 50 <em>Gossypium</em> species, four have been domesticated independently: two diploid species, <em>G. arboreum</em> L. and <em>G. herbaceum</em> L. (n = x = 13) with A<sub>1</sub> and A<sub>2</sub> genomes, and two allotetraploid species, <em>G. hirsutum</em> L. and <em>G. barbadense</em> L. (n = 2x = 26) with (AD)<sub>1</sub> and (AD)<sub>2</sub> genomes (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bowers <em>et al.</em> 2003</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lee 1984</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Percival and Kohel 1990</a>). The allotetraploid cotton species are the products of a presumed single polyploidization event between ancient A-genome and D-genome diploids that occurred approximately 1-2 million years ago (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stelly <em>et al.</em> 2005</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wendel and Cronn 2003</a>). Chromosome numbers assigned in allotetraploid cottons are based on pairing relationships in diploid x tetraploid crosses, with chromosomes 1−13 corresponding to the At subgenome and chromosomes 14−26 to the Dt subgenome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brown 1980</a>).</p><p id="p-5">Cotton species serve as a model system for polyploid plants and plant cell elongation, cell wall and cellulose biosynthesis because they are the only known plants that produce single-celled fibers (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jiang <em>et al.</em> 1998</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kim and Triplett 2001</a>). The genes that make cotton valuable function in unique ways, requiring long-term research into the development of molecular tools such as DNA markers and genome maps to translate genomic information into agronomic benefits and to other biological systems.</p><p id="p-6">Cotton researchers have explored genetic mapping with multiple types of DNA markers, including restriction fragment-length polymorphism (RFLP) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Reinisch <em>et al.</em> 1994</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shappley <em>et al.</em> 1998</a>), amplified fragment-length polymorphism (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lacape <em>et al.</em> 2003</a>), random-amplified polymorphic DNA (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kohel <em>et al.</em> 2001</a>), and simple sequence repeats (SSRs) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lacape <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Xiao <em>et al.</em> 2009</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2011</a>). Although early genetic mapping with hybridization-based markers such as RFLP opened the door to important genomic studies (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Jiang <em>et al.</em> 1998</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Shappley <em>et al.</em> 1998</a>), recent genetic mapping with polymerase chain reaction (PCR)-based markers such as SSR have facilitated portable applications among different mapping populations and research programs (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdurakhmonov <em>et al.</em> 2008</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhang <em>et al.</em> 2003</a>). As such, the cotton research community has made efforts to develop many portable markers to overcome the problem of low DNA polymorphism rates among various cultivated cotton breeding programs (<a href="http://www.cottonmarker.org/">http://www.cottonmarker.org/</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blenda <em>et al.</em> 2006</a>). To date, approximately 17,000 pairs of SSR primers have been developed from four cotton species (<em>G. arboreum</em>, <em>G. barbadense</em>, <em>G. hirsutum</em>, and <em>G. raimondii</em> Ulbrich) and a portion of this number have been surveyed for polymorphism against a 12-genotype panel of six <em>Gossypium</em> species (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blenda <em>et al.</em> 2006</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yu 2004</a>). As single nucleotide polymorphism (SNP) markers are explored in other plant species (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ganal <em>et al.</em> 2009</a>), new research has been initiated to examine nucleotide sequence diversity in <em>Gossypium</em> genomes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">An <em>et al.</em> 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Van Deynze <em>et al.</em> 2009</a>). These findings are laying the groundwork for developing large numbers of SNP markers in cotton. The growing collection of portable markers in cotton provides a cost-effective tool for genome mapping and gene discovery to understand and improve the cotton plant.</p><p id="p-7">High-resolution mapping in cotton has been conducted with segregating populations that were derived from interspecific crosses between <em>Gossypium</em> species because of limited DNA polymorphism within a cotton species. The resulting segregating populations used in major mapping projects often were either F<sub>2</sub> or BC<sub>1</sub> progeny (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2007</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lacape <em>et al.</em> 2003</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2011</a>). In addition, these maps relied heavily on a single marker type such as RFLP or SSR markers derived from limited sources. <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Rong <em>et al.</em> (2004)</a> reported the first high-density map in cotton using 57 F<sub>2</sub> plants derived from an interspecific cross between <em>G. hirsutum</em> race “palmeri” and <em>G. barbadense</em> acc. “K101.” The majority of markers used in this map were RFLP markers. This map provided one of the first insights into the allotetraploid cotton genome structure and evolution, although the RFLP markers have proven to have limited portability and utility for marker assisted breeding (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ulloa <em>et al.</em> 2005</a>).</p><p id="p-8"><a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> (2007)</a> reported the first comprehensive SSR map by using 138 BC<sub>1</sub> plants derived from an interspecific cross of <em>G. hirsutum</em> TM-1/<em>G. barbadense</em> Hai 7124//<em>G. hirsutum</em> TM-1. The majority of SSR markers in this map were derived from cotton expressed sequence tag (EST) sequences. <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lacape <em>et al.</em> (2009)</a> reported a genetic linkage map that consisted of a total of approximately 800 (amplified fragment-length polymorphism, RFLP, and SSR) marker loci via the use of 140 recombinant inbred lines (RILs); derived from an interspecific cross between <em>G. hirsutum</em> Guazuncho 2 and <em>G. barbadense</em> VH8-4602. Recently, <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> (2011)</a> used 141 BC<sub>1</sub> plants derived from an interspecific cross of <em>G. hirsutum</em> Emian 22/<em>G. barbadense</em> 3-79//<em>G. hirsutum</em> Emian 22. As with <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> (2007)</a>, this map also contained SSR markers, the majority of which were derived from ESTs. In addition, a whole-genome radiation hybrid population of 93 plants derived from an interspecific cross of <em>G. barbadense</em> 3-79/<em>G. hirsutum</em> TM-1 was also explored for mapping the cotton genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gao <em>et al.</em> 2004, 2006</a>).</p><p id="p-9">Here we report the development of a high-density cotton (<em>Gossypium</em> spp.) genetic map by using representative sets of SSR markers and the first public set of SNP markers to genotype 186 RILs derived from an interspecific cross between <em>G. hirsutum</em> TM-1 and <em>G. barbadense</em> 3-79. Both TM-1 and 3-79 are considered genetic standards for their respective species because of breeding and history of genetic/genomic research conducted by the cotton community. These two lines are highly homozygous, and extensive genetic and cytogenetic materials have been developed using them as reference parents, including mutants and hypoaneuploids (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kohel <em>et al.</em> 1970</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stelly 1993</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Stelly <em>et al.</em> 2005</a>). RILs possess several advantages over F<sub>2</sub> or BC<sub>1</sub> populations for mapping genes and quantitative trait loci (QTL), and high levels of homozygosity and recombination in the RILs enable replicate studies across different environments by different research groups. This immortal TM-1 × 3-79 RIL population is maintained at USDA-ARS, College Station, Texas, USA, and it is used by the cotton research community for genetic investigations, including QTL mapping studies. In addition, we selected SSR markers derived from different sequence sources (EST, genomic, and BAC clones). These markers were developed by 16 research groups (all 16 sources available to the public at Cotton Marker Database (<a href="http://www.cottonmarker.org/">http://www.cottonmarker.org/</a>). This combined high-density genetic map will facilitate the advancement of many basic and applied genomic studies in cotton.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and DNA extraction</h3><p id="p-10">The mapping population was an immortalized set of 186 RILs. At the time of genomic DNA extraction for this study, the average generation was F<sub>7</sub>. These lines were derived from selfing via single-seed descent original individual F<sub>2</sub> plants from a cross between <em>G. hirsutum</em> TM-1 and <em>G. barbadense</em> 3-79, two highly homozygous parents (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kohel <em>et al.</em> 1970</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Niles and Feaster 1984</a>). Factors in selecting TM-1 and 3-79 as parents in creation of a segregating population for genetic mapping are the unique high-quality fiber characteristics of extra long staple cotton 3-79 and the high productivity and modest environmental sensitivity of Upland cotton TM-1 (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kohel <em>et al.</em> 2001</a>). The parents (TM-1 and 3-79) and their 186 RIL progeny are maintained as living specimens to produce seed, fiber, and leaf tissue for this mapping effort and other genetic studies.</p><p id="p-11">Interspecific F<sub>1</sub> hypoaneuploid hybrids for specific chromosomes were used for deficiency mapping by means of loss of heterozygosity. All but one were derived previously by pollinating monosomic and monotelodisomic aneuploids quasi-isogenic to TM-1 with pollen from euploid 3-79, and recovering the respective deficiency among F<sub>1</sub> progeny. The F<sub>1</sub> aneuploid monosomic for chromosome 26 was unusual in that the deficiency arose <em>de novo</em> in 3-79 pollen, <em>i.e.</em> not via transmission from the maternal TM-1−like stock. The general procedures for mapping with cotton monosomic (2n = 51) and monotelodisomic stocks have been described previously (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Beasley 1942</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Stelly 1993</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Stelly <em>et al.</em> 2005</a>).</p><p id="p-12">Genomic DNA was extracted from fresh young leaf tissue of individual cotton plants grown in the greenhouse in accordance with the modified CTAB DNA extraction procedure as described by <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Kohel <em>et al.</em> (2001)</a>.</p></div><div id="sec-3" class="subsection"><h3>PCR primers and assays</h3><p id="p-13">The primer pairs used for PCR were developed by collaborators of the cotton research community (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Approximately 10,000 pairs of SSR primers from 16 different research projects (<a href="http://www.cottonmarker.org">http://www.cottonmarker.org</a><em>/</em>) were first analyzed to identify polymorphic markers between TM-1 and 3-79. Nine genomic DNA sources for SSR primer pairs included BNL, CIR, CM, DOW, DPL, GH, JESPR, MUSB, and TMB, and seven EST sources of SSR primer pairs included HAU, MGHES, MUCS, MUSS, NAU, STV, and UCD. While EST SSR primer pairs were developed from <em>Gossypium</em> cDNA clones that contain SSR, genomic primer pairs were developed from <em>Gossypium</em> random enriched small insert libraries except MUSB and TMB. MUSB was developed from the end sequences of the bacterial artificial chromosome (BAC) clones of <em>G. hirsutum</em> acc. Acala Maxxa (Frelichowski Jr. <em>et al.</em> 2006). TMB was developed from the BAC clones and/or physical contigs of TM-1 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2008</a>). MUSB and TMB markers facilitate an integration of genetic and physical maps of the allotetraploid cotton genome (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Xu <em>et al.</em> 2008</a>). The first public SNP set (UC) also was included in this mapping project (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Van Deynze <em>et al.</em> 2009</a>). SNP primer pairs were largely derived from <em>G. arboreum</em> EST unigenes. The actual sequence of the individual primer pairs and source clone for each SSR or SNP marker set can be found at <a href="http://www.cottonmarker.org/">http://www.cottonmarker.org/</a>.</p><p id="p-14">PCR assays for amplifying SSR markers were performed in a cocktail of 10 μL containing 20 ng of DNA, 0.25 μM forward primer, 0.25 μM reverse primer, 0.25 mM dNTPs, 2.5 mM MgCl<sub>2</sub>, and 0.65 unit DNA Taq polymerase. Thirty-five PCR cycles were used to amplify SSR products, using a primer annealing temperature of 55° or 60°. For nonlabeled SSR primers, amplified DNA products were electrophoresed in a 20-cm-long horizontal agarose gel system (Owl Separation Systems, Portsmouth, NH) with 1X TBE (45 mM tris-borate, 1 mM EDTA, pH 8) running buffer and 3.5% Hi-Resolution agarose (<em>e.g.</em> Metaphor agarose, Cambrex, East Rutherford, NJ; or SFR agarose, Amresco, Solon, OH). PCR product sizes were estimated by comparison with DNA size standard ladders (E and K Scientific, Santa Clara, CA). For fluorescently labeled primers (forward primer only with 6-FAM, HEX, or NED), amplified DNA products were separated using 36-cm or 50-cm capillary electrophoresis of automated ABI PRISM 3130xl or ABI PRISM 3730 Genetic Analyzer (Applied Biosystems/Life Technology, Foster City, CA). In a separate project, an array for Ilumina (San Diego, CA) Golden Gate assay was designed to analyze 384 SNP markers between TM-1 and 3-79 (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Van Deynze <em>et al.</em> 2009</a>). Polymorphic SNP markers based on the parental survey were used to genotype the 186 RILs.</p></div><div id="sec-4" class="subsection"><h3>Marker data acquisition and linkage map construction</h3><p id="p-15">SSR data collection was performed either manually for gel-based assays or with the GeneMapper 3.7. Among nearly 10,000 pairs of primers that were surveyed, more than 2000 primer pairs that detected the best resolution of polymorphisms between TM-1 and 3-79 were selected to genotype the 186 RILs. These primer pairs included subsets (54 MUCS, 123 MUSB, and 93 MUSS) that were previously used to genotype the same population (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; Frelichowski Jr. <em>et al.</em> 2006), and the genotyping data were incorporated into this mapping project.</p><p id="p-16">Genotyping of the RIL population for SSR and SNPs was performed as previously described (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; Frelichowski Jr. <em>et al.</em> 2006; <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Van Deynze <em>et al.</em> 2009</a>). SSR markers were generally codominant, but the calling or scoring of the tetraploid cotton alleles at a specific locus required careful examination of gel images or electrographs. Allotetraploid cottons likely had multiple copies of DNA fragments or alleles amplified with a single primer-pair. To distinguish dominant markers from codominant markers, any RIL missing one pair of the parental polymorphic fragments/alleles indicated that alleles were nonallelic or simply an existence of two dominant marker loci after all pairing attempts had failed. A missing data point of a RIL was determined if there was a lack of any signal attributable to failed PCR amplification. Duplicate marker loci were designated by adding a lower-case letter in alphabetical order after the primer name. The raw scores were first inspected for any coding error and segregation distortion before using the data as input for the JoinMap 4.0 program (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Van Ooijen 2006</a>) for mapping analysis. Using the JoinMap’s function “identify identical loci,” we identified 47 identical or cosegregating loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/001552SI.pdf">supporting information,</a> <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS2.pdf">Table S2</a>) and removed them in subsequent mapping. The Kosambi mapping function (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kosambi 1944</a>) was selected to convert a recombination frequency to a genetic distance (centiMorgan, or cM), and 40 cM was the threshold to determine linkage between two markers. Linkage groups and marker orders were determined on the basis of likelihood ratio statistic (or LOD) 10 or greater (up to LOD 15). Chromosome assignment was determined by the common markers that were located by authors in previous publications (Frelichowski Jr. <em>et al.</em> 2006; <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2007</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">2008</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lacape <em>et al.</em> 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2000</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2011</a>) and by use of the subsets of new SSR markers (GH, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) with the cotton hypoaneuploid stocks described previously. SSR loci localized to one of the chromosomes (Chr.) 1 to 13 were assigned to the A-subgenome (At), whereas loci localized to Chr. 14 to 26 were assigned to the D-subgenome (Dt).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Parental polymorphisms and genotype frequencies of the mapping population</h3><p id="p-17">Approximately 25% of the genomic SSR markers and approximately 15% of the cDNA SSR markers were polymorphic between TM-1 and 3-79. A total of 1601 pairs of polymorphic SSR primers were selected and analyzed for genotyping 186 RILs. Of the 1601 SSR primer pairs that revealed 1895 marker loci, 1344 primer pairs revealed one locus, 234 revealed two loci, and the remaining 23 revealed more than two loci. Among the 1895 SSR marker loci, 1785 were codominant; 43 were dominant loci that received alleles from TM-1, and 67 were dominant loci that received alleles from 3-79. Of these 1895 marker loci, 1825 were mapped (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The remaining 70 loci were not mapped because of highly skewed segregation (χ<sup>2</sup> &gt; 8.5) and high levels of missing data. Fifty-five of the unmapped loci were dominant loci. In addition, 247 of the 384 SNP primer pairs were polymorphic between parents and used to genotype the 186 RILs. All 247 SNP markers were codominant and revealed 247 loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS1.pdf">Table S1</a>). Of these, 207 SNP loci were mapped in unique positions, and the remaining 40 SNP loci were identical to other mapped loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS2.pdf">Table S2</a>). In summary, a total of 1848 pairs of SSR and SNP primers were used to genotype the 186 RILs, and 2142 marker loci were scored, of which 2072 marker loci revealed by 1532 pairs of SSR and 247 pairs of SNP primers were mapped (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Approximately 98% of 2072 total marker loci were mapped in unique positions, with only 47 identical or cosegregating markers including 40 SNP markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS2.pdf">Table S2</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461182/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461182/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Primer sources of cotton molecular markers (<a href="http://www.cottonmarker.org/">http://www.cottonmarker.org/</a>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">This RIL population displayed a greater-than-expected level of residual heterozygosity, <em>i.e.</em> 4.2% instead of the expected 1.6% for an F<sub>7</sub> population derived by single-seed descent. Residual heterozygosity in individual lines ranged from 0.8% to 19.9%. Among the 2032 codominant SSR and SNP loci, the average residual heterozygosity for individual markers was 4.2%, ranging from 0% to 66.7% with SSR marker STV129 demonstrating the greatest heterozygosity. Markers that detected more than 20% residual heterozygosity of the RIL population were usually difficult to map because determining linkage of these markers conflicted with more than one marker. Analysis of the genotyping data revealed a statistically significant preference of TM-1 alleles to 3-79 alleles (<em>χ<sup>2</sup></em> = 768; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Overall, the allele frequencies of TM-1 and 3-79 were 52.3% and 47.7%, respectively.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the TM-1 and 3-79 allele frequencies in the RIL mapping population (&#x3C7;2 = 768 and P &lt; 0.0001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of the TM-1 and 3-79 allele frequencies in the RIL mapping population (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; = 768 and &lt;em&gt;P&lt;/em&gt; &lt; 0.0001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F1.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/43/F1.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461166" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-20" class="first-child">Distribution of the TM-1 and 3-79 allele frequencies in the RIL mapping population (χ<sup>2</sup> = 768 and <em>P</em> &lt; 0.0001).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Genetic linkage maps of the allotetraploid cotton</h3><p id="p-21">The genetic linkage map comprises 2072 SSR and SNP loci mapped to the 26 linkage groups, corresponding to 26 chromosomes of allotetraploid cotton, for a total map distance of 3380 cM (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The average marker interval in this map is 1.63 cM. Forty-seven pairs of marker loci were found to be either identical or cosegregated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS2.pdf">Table S2</a>), and therefore only one locus from each pair is shown on the map. For example, BNL3545b is identical to or cosegregated with BNL3545a, so only BNL3545a is shown on Chr. 14 (D03).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461190/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461190/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Distribution of 2072 SSR and SNP marker loci among the 26 allotetraploid cotton chromosomes</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. et al. 2006; Guo et al. 2007, 2008; Lacape et al. 2003; Liu et al. 2000; Park et al. 2005; Yu et al. 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. &lt;em&gt;et al.&lt;/em&gt; 2006; Guo &lt;em&gt;et al.&lt;/em&gt; 2007, 2008; Lacape &lt;em&gt;et al.&lt;/em&gt; 2003; Liu &lt;em&gt;et al.&lt;/em&gt; 2000; Park &lt;em&gt;et al.&lt;/em&gt; 2005; Yu &lt;em&gt;et al.&lt;/em&gt; 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-2.medium.gif" width="201" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-2.medium.gif" width="201" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. et al. 2006; Guo et al. 2007, 2008; Lacape et al. 2003; Liu et al. 2000; Park et al. 2005; Yu et al. 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. &lt;em&gt;et al.&lt;/em&gt; 2006; Guo &lt;em&gt;et al.&lt;/em&gt; 2007, 2008; Lacape &lt;em&gt;et al.&lt;/em&gt; 2003; Liu &lt;em&gt;et al.&lt;/em&gt; 2000; Park &lt;em&gt;et al.&lt;/em&gt; 2005; Yu &lt;em&gt;et al.&lt;/em&gt; 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-3.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-3.medium.gif" width="339" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. et al. 2006; Guo et al. 2007, 2008; Lacape et al. 2003; Liu et al. 2000; Park et al. 2005; Yu et al. 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. &lt;em&gt;et al.&lt;/em&gt; 2006; Guo &lt;em&gt;et al.&lt;/em&gt; 2007, 2008; Lacape &lt;em&gt;et al.&lt;/em&gt; 2003; Liu &lt;em&gt;et al.&lt;/em&gt; 2000; Park &lt;em&gt;et al.&lt;/em&gt; 2005; Yu &lt;em&gt;et al.&lt;/em&gt; 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-4.medium.gif" width="341" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-4.medium.gif" width="341" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. et al. 2006; Guo et al. 2007, 2008; Lacape et al. 2003; Liu et al. 2000; Park et al. 2005; Yu et al. 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. &lt;em&gt;et al.&lt;/em&gt; 2006; Guo &lt;em&gt;et al.&lt;/em&gt; 2007, 2008; Lacape &lt;em&gt;et al.&lt;/em&gt; 2003; Liu &lt;em&gt;et al.&lt;/em&gt; 2000; Park &lt;em&gt;et al.&lt;/em&gt; 2005; Yu &lt;em&gt;et al.&lt;/em&gt; 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-5.medium.gif" width="333" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-5.medium.gif" width="333" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. et al. 2006; Guo et al. 2007, 2008; Lacape et al. 2003; Liu et al. 2000; Park et al. 2005; Yu et al. 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. &lt;em&gt;et al.&lt;/em&gt; 2006; Guo &lt;em&gt;et al.&lt;/em&gt; 2007, 2008; Lacape &lt;em&gt;et al.&lt;/em&gt; 2003; Liu &lt;em&gt;et al.&lt;/em&gt; 2000; Park &lt;em&gt;et al.&lt;/em&gt; 2005; Yu &lt;em&gt;et al.&lt;/em&gt; 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-6.medium.gif" width="316" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-6.medium.gif" width="316" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. et al. 2006; Guo et al. 2007, 2008; Lacape et al. 2003; Liu et al. 2000; Park et al. 2005; Yu et al. 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. &lt;em&gt;et al.&lt;/em&gt; 2006; Guo &lt;em&gt;et al.&lt;/em&gt; 2007, 2008; Lacape &lt;em&gt;et al.&lt;/em&gt; 2003; Liu &lt;em&gt;et al.&lt;/em&gt; 2000; Park &lt;em&gt;et al.&lt;/em&gt; 2005; Yu &lt;em&gt;et al.&lt;/em&gt; 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-7.medium.gif" width="290" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-7.medium.gif" width="290" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. et al. 2006; Guo et al. 2007, 2008; Lacape et al. 2003; Liu et al. 2000; Park et al. 2005; Yu et al. 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. &lt;em&gt;et al.&lt;/em&gt; 2006; Guo &lt;em&gt;et al.&lt;/em&gt; 2007, 2008; Lacape &lt;em&gt;et al.&lt;/em&gt; 2003; Liu &lt;em&gt;et al.&lt;/em&gt; 2000; Park &lt;em&gt;et al.&lt;/em&gt; 2005; Yu &lt;em&gt;et al.&lt;/em&gt; 2011) and marker loci in italic bold are assigned to cotton chromosomes in this study (Table 3). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-8.medium.gif" width="353" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-8.medium.gif" width="353" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F2/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461216" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-23" class="first-child">Genetic linkage maps of 26 allotetraploid cotton chromosomes that are presented in 13 At and Dt subgenome homeologous pairs (in parentheses). The names of DNA markers are shown on the right, and the positions of the markers are shown in Kosambi centiMorgan (cM) on the left. A line bar connects duplicate marker loci between a pair of homeologous chromosomes. Marker loci in bold are assigned to cotton chromosomes by previously published studies (Frelichowski Jr. <em>et al.</em> 2006; <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2007</a>, <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">2008</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Lacape <em>et al.</em> 2003</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2000</a>; <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; <a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2011</a>) and marker loci in italic bold are assigned to cotton chromosomes in this study (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Homeologous marker linkage relationships indicate of reciprocal rearrangement between ancestral forms of Chr. 02 and 03 and/or 14 and 17 relative to each other; they also indicate that centromeric regions are homeologous between Chr. 02 (A02) and Chr. 17 (D02), as well as between Chr. 03 (A03) and Chr. 14 (D03). Intrachromosomal duplications were noted in Chr. 5, 11, and 21, the latter two in homeologous segments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The At subgenome consisted of 1138 marker loci (927 SSR and 211 SNP), and the total genetic distance was 1726.8 cM with an average marker interval of 1.52 cM. The largest chromosome in terms of recombination frequency was Chr. 05 (A05), which spans 199.2 cM with 139 marker loci. The second largest was Chr. 11 (A11), which spans 166.5 cM with 140 loci. The shortest was Chr. 04 (A04), which spans 101.6 cM with 56 loci (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). In the At subgenome, there were 11 gaps greater than 10 cM, and the largest gap between two loci was 16.52 cM on Chr. 08 (A08).</p><p id="p-25">The Dt subgenome consisted of 934 marker loci (898 SSR and 36 SNP), and the total genetic distance was 1653.1 cM, with an average marker interval of 1.77 cM. The largest chromosome with respect to recombination frequency was Chr. 19 (D05), which spans 227.2 cM with 132 loci, and the shortest chromosome was Chr. 22 (D04), which spans 77.9 cM with 45 loci (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). There were 10 gaps greater than 10 cM, and the largest gap between two loci was 22.01 cM on Chr. 17 (D02). Although SNP marker loci were largely mapped in the At subgenome because of the A-genome origin of SNP primers (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Van Deynze <em>et al.</em> 2009</a>), the At subgenome and Dt subgenome had virtually similar numbers of SSR marker loci and total genetic distances. Furthermore, there were similar amounts of recombination between each of 13 pairs of cotton homeologous chromosomes.</p></div><div id="sec-8" class="subsection"><h3>Complete assignment of linkage groups to cotton chromosomes</h3><p id="p-26">A complete set of 26 cotton chromosomes (13 At subgenome and 13 Dt subgenome) were identified that correspond to 26 respective linkage groups (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Assignment of SSR markers and linkage groups to the cotton chromosomes was achieved in part by comparison of the common markers (bold font in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>) with the previous SSR mapping reports (Frelichowski Jr. <em>et al.</em> 2006; <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2007</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Lacape <em>et al.</em> 2003</a>; <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Park <em>et al.</em> 2005</a>; <a id="xref-ref-63-6" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2011</a>) and with the three aneuploid studies for TMB markers (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2008</a>) and BNL markers (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gutiérrez <em>et al.</em> 2009</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Liu <em>et al.</em> 2000</a>), respectively. In addition, hypoaneuploid cottons were also analyzed to identify TM-1 deficiency with 37 newly developed GH markers (bold italic in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>) from <em>G. hirsutum</em> and other SSR markers of interest in the mapping study (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hoffman <em>et al.</em> 2007</a>). Although most SSR markers generally agreed with published reports, a few incongruities, such as GH034 and GH526, between various data types were encountered when cotton hypoaneuploid stocks were used along with individual mapping populations. Additional mapping analyses in the present research confirmed or reassigned such SSR markers to the corresponding cotton chromosomes (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461178/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461178/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Assignment of 37 GH SSR markers to specific allotetraploid cotton chromosomes</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/43/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deletion analysis of cotton SSR markers. GH584 amplified (from L to R) cotton hemizygous F1 hypoaneuploids as well as homozygous TM-1 and 3-79. TM-1 allele (140 bp) was missing in both lanes (see arrow) with the H09 template, suggesting the location of GH584 locus on chromosome 09 (A09)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586735127" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deletion analysis of cotton SSR markers. GH584 amplified (from L to R) cotton hemizygous F&lt;sub&gt;1&lt;/sub&gt; hypoaneuploids as well as homozygous TM-1 and 3-79. TM-1 allele (140 bp) was missing in both lanes (see arrow) with the H09 template, suggesting the location of GH584 locus on chromosome 09 (A09).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/43/F3.medium.gif" width="440" height="138"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/43/F3.medium.gif" width="440" height="138"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/43/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/43/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461082" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-28" class="first-child">Deletion analysis of cotton SSR markers. GH584 amplified (from L to R) cotton hemizygous F<sub>1</sub> hypoaneuploids as well as homozygous TM-1 and 3-79. TM-1 allele (140 bp) was missing in both lanes (see arrow) with the H09 template, suggesting the location of GH584 locus on chromosome 09 (A09).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Genomic duplication and chromosomal translocation of allotetraploid cottons</h3><p id="p-29">Among 1601 SSR primer pairs that amplified 1895 loci in TM-1 and 3-79, 257 SSR primer pairs amplified two or more loci, resulting in a total of 551 duplicate loci. Excluding dominant loci amplified by these SSRs, there were 494 codominant loci that were duplicated, resulting in 247 pairs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS3.pdf">Table S3</a>). Most of the duplicate loci were mapped on the homeologous chromosome pairs (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). The relative orders of most duplicate loci on the homeologous chromosomes were similar (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). The duplicate loci identified by these SSR markers demonstrated the complex but linear features of the allotetraploid cotton genomes. A few duplicate loci also were present between nonhomeologous chromosomes and/or within the same subgenome, which indicated likely genome rearrangements (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS3.pdf">Table S3</a>). For example, an intrasubgenome duplication was revealed by the marker BNL1044 between Chr. 04 (A04) and Chr. 05 (A05). Distinct intrachromosome duplications were indicated by one SSR duplication in Chr. 11 (A11) and three SSRs in Chr. 21 (D11) (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>). In chromosome 11 (A11), TMB0426 revealed two loci that were mapped 8.1 cM apart. In chromosome 21 (D11), three markers (<em>i.e.</em> CM0160, JESPR211, and JESPR244) each revealed two loci. In the latter, the recombination rates remained similar (~8-9 cM) but the relative orders among duplicated loci were altered.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461158/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461158/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4 </span> <span class="caption-title">Pairs of duplicate marker loci between homeologous and nonhomeologous chromosomes in cotton</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">A postpolyploidization reciprocal translocation of chromosomes 02 (A02) and 03 (A03) was suggested by 10 pairs of duplicate loci (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Five pairs of duplicate loci were identified between chromosomes 02 (A02) and 14 (D03) and 5 pairs between chromosomes 03 (A03) and 17 (D02). The marker TMB1025 revealed duplicate loci between chromosomes 02 (A02) and 03 (A03), which inferred a possible breakpoint for the reciprocal translocation in these two At subgenome chromosomes. Additional mapping data in the vicinity of TMB1025 will be necessary to confirm this conclusion. Another translocation between At subgenome chromosomes 04 (A04) and 05 (A05), as previously suggested by <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> (2007)</a>, was observed by the marker BNL1044 loci (BNL1044a at 33.6 cM of A05) and (BNL1044c at 48.1 cM of A04) (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS3.pdf">Table S3</a>). Furthermore, the marker GH252 loci showed a translocation between non-homeologous chromosomes 05 (A05) with GH252a at 136.4 cM and 22 (D04) with GH252b at 18.3 cM.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-31">The high-density genetic linkage map created in this research is composed of 2072 SSR and SNP loci representing many individual groups of the cotton research community, and it provides a transferable platform that is essential for a broad spectrum of basic and applied studies aimed at understanding and manipulating complex cotton genomes. Among the 17 sets of SSR and SNP marker loci, BAC-derived SSRs (310 TMB <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS1.pdf">Table S1</a> and 155 MUSB) facilitate an integration of genetic and physical maps of the cotton chromosomes (Frelichowski Jr. <em>et al.</em> 2006; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Xu <em>et al.</em> 2008</a>). The markers linked to the novel genes can be used to screen cotton BAC clones or physical contigs from which the SSR markers were developed (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Yin <em>et al.</em> 2006</a>). The 357 EST-derived SSR markers mapped herein offer an opportunity to study functional genes and gene islands for fiber development and other important traits of interest. In addition, the genetic mapping of the 247 SNP markers is the first major public effort to use nucleotide sequence diversity in cotton species by mapping SNP loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS1.pdf">Table S1</a>). Localization of these SNP markers to the 26 individual cotton chromosomes and their integration with large numbers of SSR markers will facilitate other studies in cotton genomics. We believe that the high-density genetic map reported herein is a saturated one for the allotetraploid cotton, as evidenced by a separate mapping analysis (data not shown). Further increase in the map density may not significantly change the total genetic length of this map but will facilitate whole-genome physical alignment, sequencing, and mapping of genes for cotton improvement.</p><p id="p-32">Deviation from a Mendelian segregation ratio is common in intra- and interspecific crosses (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Causse <em>et al.</em> 1994</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Lacape <em>et al.</em> 2009</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Rong <em>et al.</em> 2004</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ulloa <em>et al.</em> 2002</a>; <a id="xref-ref-63-7" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2011</a>). An extremely severe distortion (99%) toward <em>G. hirsutum</em> was observed by <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Lacape <em>et al.</em> (2009)</a> when 140 RILs were used to produce a low-density map of approximately 800 loci. Only 15 of the 140 RILs exhibited 50% or more <em>G. barbadense</em> parental alleles. In this research, TM-1 was less environmentally sensitive than 3-79, as reflected by the allele transmission preference in the advancement of generations of the RIL population (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Of the 2072 mapped marker loci, 1391 (67.1%) fit an expected 1:1 segregation ratio, and 681 (32.9%) deviated significantly (χ<sup>2</sup> &gt; 3.8) from expectations among 186 RILs. The 681 segregation-distorted loci (SDL) were mapped in all 26 groups with 349 mapped in At subgenome, and 332 in Dt subgenome chromosomes. Four chromosomes, <em>i.e.</em> Chr. 15 (D01), Chr. 05 (A05), Chr. 07 (A07), and Chr. 08 (A08), had the most SDL, with 68, 60, 57, and 42 loci, respectively. However, Chr. 26 (D12) has the greatest percentage of SDL, 75%, followed by Chr. 15 (D01) with 73.9%, Chr. 07 (A07) with 68.7% and Chr. 05 (A05) with 45.1%. In most cases, the SDL were mapped at centromeric regions.</p><p id="p-33">Our mapping studies indicate that the two subgenomes of allotetraploid cottons are equivalent in recombination frequencies despite the extra repetitive DNA in the At subgenome (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zhao <em>et al.</em> 1998</a>). This result is consistent with other independent mapping studies in which the authors used different allotetraploid cotton populations (F<sub>2</sub> or BC<sub>1</sub>) where variation between At and Dt map sizes supports the ratio of our genetic distances between the two subgenomes. <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Rong <em>et al.</em> (2004)</a> mapped a total of 2584 STS loci that span 4447 cM, with the A subgenome being 9.5% larger genetically than the D subgenome. To the contrary, <a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> (2007)</a> mapped a total of 1790 SSR loci that span 3426 cM, with the D subgenome being 4.5% larger genetically than the A subgenome. <a id="xref-ref-63-8" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> (2011)</a> mapped a total of 2316 SSR loci that span 4419 cM with the A subgenome being 3.9% larger genetically than the D subgenome. In this study, the tetraploid cotton were mapped with 1106 loci (54.5%) on 13 At chromosomes at 1726.8 cM (51.1%) and 922 loci (45.5%) on 13 Dt chromosomes at 1653.1 cM (48.9%). Variation in the ratio of subgenome map distances is likely the result of differences in mapping population sizes, as well as in the numbers and sources of DNA markers.</p><p id="p-34">As evidenced in our mapping data, two reciprocal translocations (between Chr. 02 and 03 and between Chr. 04 and 05) are inferred during or after the polyploidization process of two ancestral diploid genomes (A and D). The translocation breakpoint between Chr. 02 and Chr. 03 may be at or near homeologous SSR marker TMB1025. Further investigation is needed to identify additional markers in the vicinity of TMB1025. On the basis of homeologous markers of the two chromosome pairs (A02-D02 and A03-D03), the majority of duplicate loci were mapped to individual pairs of Chr. 02 <em>vs.</em> Chr. 17 and Chr. 03 <em>vs.</em> Chr. 14. The centromeric cores of these chromosomes seem to show the homeologous relationship, either reciprocal insertional translocations or two temporally separate traditional reciprocal translocations. Thus, we propose to name Chr. 17 as D02 and Chr. 14 as D03, whereas Chr. 02 and Chr. 03 remain as A02 and A03, respectively, which is a revision to <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> (2006b)</a> and <a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> (2007)</a>. Duplication of marker loci revealed genome rearrangements within the same individual chromosomes and/or between nonhomeologous chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1/TableS3.pdf">Table S3</a>). We recognize that nomenclature revision of cotton chromosomes and linkage groups would be needed in the future, but this could be accomplished by an international committee of experts in the subject matter.</p><p id="p-35">Genetic mapping coupled with physical alignment of genomic regions into chromosomal maps will expedite the discovery of resistance (R) or pathogen-induced R genes underlying QTL involved in resistance to nematode and <em>Fusarium</em> wilt (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ulloa <em>et al.</em> 2011</a>). Chromosomes 11 (A11) and 21 (D11) are homologs that harbor important genes for cotton improvement because these chromosomes contain genes for resistance to reniform (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dighe <em>et al.</em> 2009</a>) and root-knot nematodes (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2006a</a>), race 1 of <em>Fusarium</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ulloa <em>et al.</em> 2010</a>) and other traits affecting fiber yield and quality. The high-density genetic map will facilitate and expedite the analysis of plant defense genes against nematodes and other biotrophic pathogens.</p><p id="p-36">This high-density cotton map was constructed with an immortal RIL mapping population. A high level of homozygosity in this RIL population (currently in F<sub>8</sub>-F<sub>9</sub>) was achieved with less than 5% genome-wide residual heterozygosity. The RILs are maintained as living stocks to produce seed sources for multilocation research on fiber among other traits and to extract fresh DNA samples for a broad spectrum of genomic studies. Our mapping population of 186 RILs is the largest population ever used in high-density cotton genetic mapping. The accuracy of mapping results can be improved substantially as the proportion of recombination between the two linked markers in an inbred population is about twice that of a single meiotic event F<sub>2</sub> or BC<sub>1</sub> population when linkage distances are small (&lt;12.5 cM) and increase nonlinearly to 50% for unlinked markers (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burr <em>et al.</em> 1988</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Haldane and Waddington 1931</a>). This population provides the greatest mapping power currently known in cotton to detect additional loci between closely linked markers by members of the cotton research community who are interested in SSR and SNP augmentation. The advantages of this immortal RIL population and its parental lines make it practical for high-resolution consensus mapping with additional sequence-based portable markers, enabling better understanding and exploitation of complex <em>Gossypium</em> genomes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mace <em>et al.</em> 2009</a>). This information will complement other work because of the use of the same parents in developing genetic resources, such as hypoaneuploid cytogenetic stocks, chromosome substitution lines, chromosome specific RILs, and QTL mapping populations in other research programs (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jenkins <em>et al.</em> 2006</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Saha <em>et al.</em> 2006</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">2010</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2011</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Stelly <em>et al.</em> 2005</a>).</p><p id="p-37">The International Cotton Genome Initiative (<a href="http://icgi.tamu.edu/">http://icgi.tamu.edu/</a>) has proposed to map and sequence the <em>Gossypium</em> genomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brubaker <em>et al.</em> 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2007</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Paterson 2008</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wilkins 2008</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yu <em>et al.</em> 2008</a>), but large amounts of dispersed repetitive elements and duplicate loci between and within the allotetraploid cotton chromosomes present great challenges to properly assemble a complex <em>Gossypium</em> genome. Development of additional numbers of SSR and SNP markers from the fingerprinted and sequenced BAC clones or physical contigs, such as the 310 TMB and 155 MUSB markers on the present map, would provide a unique opportunity to facilitate the mapping the gaps (5−15 cM) of genomic regions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lin <em>et al.</em> 2010</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Xu <em>et al.</em> 2008</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Yin <em>et al.</em> 2006</a>). A high-density genetic map is essential in the reconciliation with a whole-genome physical map to facilitate genome sequencing, sequence assembly, gene mapping, and the design of targeted genetic markers for better understanding and improvement of the cotton plant.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We offer our appreciation to many technical staff residing in the various laboratories who contributed in the development and mapping of SSRs and SNPs on the RIL population. Special thanks go to Ping Cui, Jianmin Dong, Nicole Steele, and Jewel Stroupe for their technical assistance in surveying parental DNA polymorphism and in genotyping the RIL mapping population, and special thanks to Jared Harris for his assistance in maintaining the immortal RIL population. We also would like to thank Cotton Incorporated, Cary, NC, for facilitating the dye-labeled SSR primers, and Dr. Roy Cantrell for his contribution and valuable discussion on this research project. Mention of trade names or commercial products in this article is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the U.S. Department of Agriculture. The U.S. Department of Agriculture is an equal opportunity provider and employer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001552/-/DC1</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-06-24"><span class="received-label">Received </span>June 24, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-11-04"><span class="accepted-label">Accepted </span>November 4, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Yu <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.1" data-doi="10.1016/j.ygeno.2008.07.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdurakhmonov</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pepper</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdullaev</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular diversity and association mapping of fiber quality traits in exotic <em>G. hirsutum</em> L. germplasm.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">478</span>–<span class="cit-lpage">487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DAbdurakhmonov%26rft.auinit1%253DI.%2BY.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D478%26rft.epage%253D487%26rft.atitle%253DMolecular%2Bdiversity%2Band%2Bassociation%2Bmapping%2Bof%2Bfiber%2Bquality%2Btraits%2Bin%2Bexotic%2BG.%2Bhirsutum%2BL.%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2008.07.013%26rft_id%253Dinfo%253Apmid%252F18801424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2008.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18801424&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261460100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.2" data-doi="10.1007/s00122-008-0732-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cotton (<em>Gossypium</em> spp.) R2R3-MYB transcription factors SNP identification, phylogenomic characterization, chromosome localization, and linkage mapping.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAn%26rft.auinit1%253DC.%26rft.volume%253D116%26rft.issue%253D7%26rft.spage%253D1015%26rft.epage%253D1026%26rft.atitle%253DCotton%2B%2528Gossypium%2Bspp.%2529%2BR2R3-MYB%2Btranscription%2Bfactors%2BSNP%2Bidentification%252C%2Bphylogenomic%2Bcharacterization%252C%2Bchromosome%2Blocalization%252C%2Band%2Blinkage%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0732-4%26rft_id%253Dinfo%253Apmid%252F18338155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0732-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18338155&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beasley</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1942</span> <span class="cit-article-title">Meiotic chromosome behavior in species, species hybrids, haploids and induced polyploids of <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D27%26rft.spage%253D25%26rft_id%253Dinfo%253Apmid%252F17247031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17247031&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000201544200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.4" data-doi="10.1186/1471-2164-7-132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blenda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">CMD: a Cotton Microsatellite Database resource for <em>Gossypium</em> genomics.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBlenda%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.spage%253D132%26rft.epage%253D132%26rft.atitle%253DCMD%253A%2Ba%2BCotton%2BMicrosatellite%2BDatabase%2Bresource%2Bfor%2BGossypium%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-132%26rft_id%253Dinfo%253Apmid%252F16737546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16737546&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.5" data-doi="10.1038/nature01521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Unravelling angiosperm genome evolution by phylogenetic analysis of chromosomal duplication events.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBowers%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D422%26rft.issue%253D6930%26rft.spage%253D433%26rft.epage%253D438%26rft.atitle%253DUnravelling%2Bangiosperm%2Bgenome%2Bevolution%2Bby%2Bphylogenetic%2Banalysis%2Bof%2Bchromosomal%2Bduplication%2Bevents.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01521%26rft_id%253Dinfo%253Apmid%252F12660784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12660784&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181801200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Identification of the chromosomes of <em>Gossypium hirsutum</em> L. by means of translocations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">266</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBROWN%26rft.auinit1%253DM.%2BS.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D266%26rft.epage%253D274%26rft.atitle%253DIdentification%2Bof%2Bthe%2Bchromosomes%2Bof%2BGossypium%2Bhirsutum%2BL.%2Bby%2Bmeans%2Bof%2Btranslocations%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6ODoiNzEvNC8yNjYiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8yLzEvNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantrell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giband</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Letter to Journal of Cotton Science Community from the Steering Committee of the International Cotton Genome Initiative (ICGI).</span> <abbr class="cit-jnl-abbrev">J. Cotton Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCotton%2BSci.%26rft.volume%253D4%26rft.spage%253D149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burr</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burr</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albertson</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Gene-mapping with recombinant inbreds in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D118%26rft.spage%253D519%26rft_id%253Dinfo%253Apmid%252F3366363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3366363&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988M440400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">Y. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chunwongse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Saturated molecular map of the rice genome based on an interspecific backcross population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D138%26rft.spage%253D1251%26rft_id%253Dinfo%253Apmid%252F7896104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7896104&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PU54800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.10" data-doi="10.1104/pp.107.107672"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Triplett</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Toward sequencing cotton (<em>Gossypium</em>) genomes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DChen%26rft.auinit1%253DZ.%2BJ.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1303%26rft.epage%253D1310%26rft.atitle%253DToward%2BSequencing%2BCotton%2B%2528Gossypium%2529%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.107672%26rft_id%253Dinfo%253Apmid%252F18056866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQ1LzQvMTMwMyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzIvMS80My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.11" data-doi="10.2135/cropsci2008.03.0129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dighe</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menz</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantrell</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Linkage mapping of <em>Gossypium longicalyx</em> resistance to reniform nematode during introgression into cotton <em>Gossypium hirsutum</em>.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DDighe%26rft.auinit1%253DN.%2BD.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1151%26rft.epage%253D1164%26rft.atitle%253DLinkage%2BMapping%2Bof%2BResistance%2Bto%2BReniform%2BNematode%2Bin%2BCotton%2Bfollowing%2BIntrogression%2Bfrom%2BGossypium%2Blongicalyx%2B%2528Hutch.%2B%2526%2BLee%2529%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.03.0129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.03.0129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.43.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turcotte</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Genetics, cytology, and evolution of <em>Gossypium</em></span>, pp. <span class="cit-fpage">271</span>–<span class="cit-lpage">375</span> <span class="cit-comment">in</span> <span class="cit-source">Advances in Genetics</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Caspari</span>  <span class="cit-name-given-names">E. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">G. S.</span></span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.1.43.13" data-doi="10.1007/s00438-006-0106-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frelichowski</span>  <span class="cit-name-given-names">J. E.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Main</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomkins</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantrell</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cotton genome mapping with new microsatellites from Acala 'Maxxa' BAC-ends.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DFrelichowski%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D275%26rft.issue%253D5%26rft.spage%253D479%26rft.epage%253D491%26rft.atitle%253DCotton%2Bgenome%2Bmapping%2Bwith%2Bnew%2Bmicrosatellites%2Bfrom%2BAcala%2B%2527Maxxa%2527%2BBAC-ends.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-006-0106-z%26rft_id%253Dinfo%253Apmid%252F16501995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-006-0106-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16501995&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237504600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.14" data-doi="10.1016/j.pbi.2008.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SNP identification in crop plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D217%26rft.atitle%253DSNP%2Bidentification%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.12.009%26rft_id%253Dinfo%253Apmid%252F19186095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19186095&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265115400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raska</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Wide-cross whole-genome radiation hybrid mapping of cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1317</span>–<span class="cit-lpage">1329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D167%26rft.spage%253D1317%26rft_id%253Dinfo%253Apmid%252F15280245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15280245&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.1.43.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Womack</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Wide-cross whole-genome radiation hybrid mapping of the cotton (<em>Gossypium barbadense</em> L.) genome. Mol.</span> <abbr class="cit-jnl-abbrev">Genet. Genomics</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BGenomics%26rft.volume%253D275%26rft.spage%253D105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.17" data-doi="10.1534/genetics.107.070375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D176%26rft.spage%253D527%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.070375%26rft_id%253Dinfo%253Apmid%252F17409069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.070375&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17409069&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246960900043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">BAC-derived SSR markers chromosome locations in cotton.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Integrative placement and orientation of non-redundant SSR loci in cotton linkage groups by deficiency analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D23%26rft.spage%253D693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waddington</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Inbreeding and linkage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHaldane%26rft.auinit1%253DJ.%2BB.%2BS.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D374%26rft.atitle%253DINBREEDING%2BAND%2BLINKAGE%26rft_id%253Dinfo%253Apmid%252F17246626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxNi80LzM1NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzIvMS80My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grum</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification of 700 new microsatellite loci from cotton (<em>G. hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">J. Cotton Sci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCotton%2BSci.%26rft.volume%253D11%26rft.spage%253D208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.22" data-doi="10.2135/cropsci2005.08-0269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic effects of thirteen <em>Gossypium barbadense</em> L. chromosome substitution lines in topcrosses with Upland cotton cultivars: I. Yield and yield components.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DJenkins%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D1169%26rft.epage%253D1178%26rft.atitle%253DGenetic%2BEffects%2Bof%2BThirteen%2BGossypium%2Bbarbadense%2BL.%2BChromosome%2BSubstitution%2BLines%2Bin%2BTopcrosses%2Bwith%2BUpland%2BCotton%2BCultivars%253A%2BI.%2BYield%2Band%2BYield%2BComponents%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.08-0269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.08-0269&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237375900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.23" data-doi="10.2135/cropsci2006.06.0396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic effects of thirteen <em>Gossypium barbadense</em> L. chromosome substitution lines in topcrosses with Upland cotton cultivars: II. Fiber quality traits</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DJenkins%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D561%26rft.epage%253D570%26rft.atitle%253DGenetic%2BEffects%2Bof%2BThirteen%2BGossypium%2Bbarbadense%2BL.%2BChromosome%2BSubstitution%2BLines%2Bin%2BTopcrosses%2Bwith%2BUpland%2BCotton%2BCultivars%253A%2BII.%2BFiber%2BQuality%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.06.0396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.06.0396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.24" data-doi="10.1073/pnas.95.8.4419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Zik</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Polyploid formation created unique avenues for response to selection in <em>Gossypium</em> (cotton).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4419</span>–<span class="cit-lpage">4424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DJiang%26rft.auinit1%253DC.-x.%26rft.volume%253D95%26rft.issue%253D8%26rft.spage%253D4419%26rft.epage%253D4424%26rft.atitle%253DPolyploid%2Bformation%2Bcreated%2Bunique%2Bavenues%2Bfor%2Bresponse%2Bto%2Bselection%2Bin%2BGossypium%2B%2528cotton%2529%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.8.4419%26rft_id%253Dinfo%253Apmid%252F9539752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NS84LzQ0MTkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8yLzEvNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.25" data-doi="10.1104/pp.127.4.1361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Triplett</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Cotton fiber growth in planta and in vitro. Models for plant cell elongation and cell wall biogenesis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1361</span>–<span class="cit-lpage">1366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DKim%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D127%26rft.issue%253D4%26rft.spage%253D1361%26rft.epage%253D1366%26rft.atitle%253DCotton%2BFiber%2BGrowth%2Bin%2BPlanta%2Band%2Bin%2BVitro.%2BModels%2Bfor%2BPlant%2BCell%2BElongation%2Band%2BCell%2BWall%2BBiogenesis%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.127.4.1361%26rft_id%253Dinfo%253Apmid%252F11743074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTI3LzQvMTM2MSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzIvMS80My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Texas Marker-1. Description of a genetic standard for <em>Gossypium hirsutum</em></span> <abbr class="cit-jnl-abbrev">L. Crop Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">670</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DL.%2BCrop%2BSci.%26rft.volume%253D10%26rft.spage%253D670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.27" data-doi="10.1023/A:1012263413418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazo</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular mapping and characterization of traits controlling fiber quality in cotton.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D121%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1012263413418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1012263413418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000171398600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.28" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.29" data-doi="10.1139/g03&amp;ndash;050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibivilliers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bojinov</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A combined RFLP-SSR-AFLP map of tetraploid cotton based on a <em>Gossypium hirsutum</em> x <em>Gossypium barbadense</em> backcross population.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D612%26rft.epage%253D626%26rft.atitle%253DA%2Bcombined%2BRFLP-SSR-AFLP%2Bmap%2Bof%2Btetraploid%2Bcotton%2Bbased%2Bon%2Ba%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%2Bbackcross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg03%2526ndash%253B050%26rft_id%253Dinfo%253Apmid%252F12897870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g03&amp;ndash;050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12897870&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.30" data-doi="10.1007/s00122-009-1037-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arioli</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derijcker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forestier-Chiron</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A new interspecific, <em>Gossypium hirsutum</em> × <em>G. barbadense</em>, RIL population: towards a unified consensus linkage map of tetraploid cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLacape%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D292%26rft.atitle%253DA%2Bnew%2Binterspecific%252C%2BGossypium%2Bhirsutum%2Bx%2BG.%2Bbarbadense%252C%2BRIL%2Bpopulation%253A%2Btowards%2Ba%2Bunified%2Bconsensus%2Blinkage%2Bmap%2Bof%2Btetraploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1037-y%26rft_id%253Dinfo%253Apmid%252F19387609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1037-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19387609&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.43.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Cotton as a world crop</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span> <span class="cit-comment">in</span> <span class="cit-source">Cotton, Agronomy Monograph 24</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. F.</span></span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.32" data-doi="10.1186/1471-2164-11-395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estill</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Compton</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A draft physical map of a D-genome cotton species (<em>Gossypium raimondii</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLin%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.spage%253D395%26rft.epage%253D395%26rft.atitle%253DA%2Bdraft%2Bphysical%2Bmap%2Bof%2Ba%2BD-genome%2Bcotton%2Bspecies%2B%2528Gossypium%2Braimondii%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-395%26rft_id%253Dinfo%253Apmid%252F20569427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20569427&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.33" data-doi="10.1093/jhered/91.4.326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burr</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantrell</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Chromosomal assignment of microsatellite loci in cotton.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DSiu%26rft.auinit1%253DL.%26rft.volume%253D91%26rft.issue%253D4%26rft.spage%253D326%26rft.epage%253D332%26rft.atitle%253DChromosomal%2Bassignment%2Bof%2Bmicrosatellite%2Bloci%2Bin%2Bcotton%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F91.4.326%26rft_id%253Dinfo%253Apmid%252F10912681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6IjkxLzQvMzI2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMi8xLzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.34" data-doi="10.1186/1471-2229-9-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DMace%26rft.auinit1%253DE.%2BS.%26rft.volume%253D9%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DA%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Bsorghum%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Band%2Bhigh-throughput%2BDiversity%2BArray%2BTechnology%2B%2528DArT%2529%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-13%26rft_id%253Dinfo%253Apmid%252F19171067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19171067&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.43.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niles</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feaster</span>  <span class="cit-name-given-names">C. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Breeding</span>, pp. <span class="cit-fpage">201</span>–<span class="cit-lpage">231</span> <span class="cit-comment">in</span> <span class="cit-source">Cotton. Monograph series Agronomy No. 24</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. F.</span></span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.36" data-doi="10.1007/s00438-005-0037-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alabady</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sickler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic mapping of new cotton fiber loci using EST-derived microsatellites in an interspecific recombinant inbred line cotton population.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">428</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DPark%26rft.auinit1%253DY.%2BH.%26rft.volume%253D274%26rft.issue%253D4%26rft.spage%253D428%26rft.epage%253D441%26rft.atitle%253DGenetic%2Bmapping%2Bof%2Bnew%2Bcotton%2Bfiber%2Bloci%2Busing%2BEST-derived%2Bmicrosatellites%2Bin%2Ban%2Binterspecific%2Brecombinant%2Binbred%2Bline%2Bcotton%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-005-0037-0%26rft_id%253Dinfo%253Apmid%252F16187061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-005-0037-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16187061&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233380500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequencing the cotton genomes—<em>Gossypium</em> spp.</span> <abbr class="cit-jnl-abbrev">Cotton Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCotton%2BSci.%26rft.volume%253D20%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Percival</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Distribution, collection, and evaluation of <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D44%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinisch</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A detailed RFLP map of cotton, <em>Gossypium hirsutum</em> X <em>Gossypium barbadense</em>—chromosome organization and evolution in a disomic polyploid genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DReinisch%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D829%26rft.epage%253D847%26rft.atitle%253DA%2Bdetailed%2BRFLP%2Bmap%2Bof%2Bcotton%252C%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%253A%2Bchromosome%2Borganization%2Band%2Bevolution%2Bin%2Ba%2Bdisomic%2Bpolyploid%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F7851778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvODI5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMi8xLzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.40" data-doi="10.1534/genetics.166.1.389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (<em>Gossypium</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRong%26rft.auinit1%253DJ.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D417%26rft.atitle%253DA%2B3347-Locus%2BGenetic%2BRecombination%2BMap%2Bof%2BSequence-Tagged%2BSites%2BReveals%2BFeatures%2Bof%2BGenome%2BOrganization%252C%2BTransmission%2Band%2BEvolution%2Bof%2BCotton%2B%2528Gossypium%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.389%26rft_id%253Dinfo%253Apmid%252F15020432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvMzg5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMi8xLzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.41" data-doi="10.1534/genetics.105.053371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Effects of chromosome-specific introgression in Upland cotton on fiber and agronomic traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1927</span>–<span class="cit-lpage">1938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D1927%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.053371%26rft_id%253Dinfo%253Apmid%252F16387867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.053371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16387867&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236668100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic dissection of chromosome substitution lines of cotton to discover novel <em>Gossypium barbadense</em> L. alleles for improvement of agronomic traits.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSaha%26rft.auinit1%253DS.%26rft.volume%253D120%26rft.issue%253D6%26rft.spage%253D1193%26rft.epage%253D1205%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bchromosome%2Bsubstitution%2Blines%2Bof%2Bcotton%2Bto%2Bdiscover%2Bnovel%2BGossypium%2Bbarbadense%2BL.%2Balleles%2Bfor%2Bimprovement%2Bof%2Bagronomic%2Btraits.%26rft_id%253Dinfo%253Apmid%252F20094702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20094702&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Delineation of interspecific epistasis on fiber quality traits in <em>Gossypium hirsutum</em> by ADAA analysis of intermated <em>G. barbadense</em> chromosome substitution lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">1351</span>–<span class="cit-lpage">1361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSaha%26rft.auinit1%253DS.%26rft.volume%253D122%26rft.issue%253D7%26rft.spage%253D1351%26rft.epage%253D1361%26rft.atitle%253DDelineation%2Bof%2Binterspecific%2Bepistasis%2Bon%2Bfiber%2Bquality%2Btraits%2Bin%2BGossypium%2Bhirsutum%2Bby%2BADAA%2Banalysis%2Bof%2Bintermated%2BG.%2Bbarbadense%2Bchromosome%2Bsubstitution%2Blines.%26rft_id%253Dinfo%253Apmid%252F21301803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21301803&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.44" data-doi="10.1007/s001220050952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shappley</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">An RFLP linkage map of Upland cotton, <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D756%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000076924400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Interfacing cytogenetics with the cotton genome mapping effort. Proc. Beltwide Cotton Improvement Conference</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1550</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.46" data-doi="10.2135/cropsci2004.0642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raska</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Registration of 17 Upland (<em>Gossypium hirsutum</em>) cotton germplasm lines disomic for different <em>G. barbadense</em> chromosome or arm substitutions.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">2663</span>–<span class="cit-lpage">2665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DStelly%26rft.auinit1%253DD.M.%26rft.volume%253D45%26rft.issue%253D6%26rft.spage%253D2663%26rft.epage%253D2665%26rft.atitle%253DRegistration%2Bof%2B17%2BUpland%2B%2528Gossypium%2Bhirsutum%2529%2BCotton%2BGermplasm%2BLines%2BDisomic%2Bfor%2BDifferent%2BG.%2Bbarbadense%2BChromosome%2Bor%2BArm%2BSubstitutions%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233147100086&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.43.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Potential for crop improvement with exotic germplasm and genetic engineering, pp. 13–327</span> <span class="cit-comment">in</span> <span class="cit-source">Challenging the Future: Proceedings of the World Cotton Research Conference-1</span>, <span class="cit-comment">edited by</span> <span class="cit-ed"><span class="cit-name-surname">Constable</span>  <span class="cit-name-given-names">G. A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Forrester</span>  <span class="cit-name-given-names">N. W.</span></span>. <span class="cit-publ-name">CSIRO</span>, <span class="cit-publ-loc">Melbourne, Brisbane, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.48" data-doi="10.1007/s001220100739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">W. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shappley</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahler</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">RFLP genetic linkage maps from four F2.3 populations and a joinmap of <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DUlloa%26rft.auinit1%253DM.%26rft.volume%253D104%26rft.issue%253D2-3%26rft.spage%253D200%26rft.epage%253D208%26rft.atitle%253DRFLP%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bfour%2BF%25282.3%2529%2Bpopulations%2Band%2Ba%2Bjoinmap%2Bof%2BGossypium%2Bhirsutum%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100739%26rft_id%253Dinfo%253Apmid%252F12582687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582687&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174376400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.49" data-doi="10.1093/jhered/esi020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Chromosomal assignment of RFLP linkage groups harboring important QTLs on an intraspecific cotton (<em>Gossypium hirsutum</em> L.) joinmap.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesi020%26rft_id%253Dinfo%253Apmid%252F15618303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk2LzIvMTMyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvMi8xLzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene action analysis by inheritance and quantitative trait loci mapping of resistance to root-knot nematodes in cotton.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D129%26rft.spage%253D541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutmacher</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping <em>Fusarium</em> wilt race 1 resistance genes in cotton by inheritance, QTL and sequencing composition.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DUlloa%26rft.auinit1%253DM.%26rft.volume%253D286%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D36%26rft.atitle%253DMapping%2BFusarium%2Bwilt%2Brace%2B1%2Bresistance%2Bgenes%2Bin%2Bcotton%2Bby%2Binheritance%252C%2BQTL%2Band%2Bsequencing%2Bcomposition.%26rft_id%253Dinfo%253Apmid%252F21533837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21533837&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.52" data-doi="10.1186/1471-2229-9-125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Deynze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sampling nucleotide diversity in cotton.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BDeynze%26rft.auinit1%253DA.%2BE.%26rft.volume%253D9%26rft.spage%253D125%26rft.epage%253D125%26rft.atitle%253DSampling%2Bnucleotide%2Bdiversity%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-125%26rft_id%253Dinfo%253Apmid%252F19840401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840401&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.43.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">JoinMap 4.0: Software for the Calculation of Genetic Linkage Maps in Experimental Populations.</span> <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.54" data-doi="10.1007/s00122-005-0183-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification and mapping of microsatellite markers linked to a root-knot nematode resistance gene (rkn1) in Acala NemX cotton (<em>Gossypium hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">770</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D112%26rft.issue%253D4%26rft.spage%253D770%26rft.epage%253D777%26rft.atitle%253DIdentification%2Band%2Bmapping%2Bof%2Bmicrosatellite%2Bmarkers%2Blinked%2Bto%2Ba%2Broot-knot%2Bnematode%2Bresistance%2Bgene%2B%2528rkn1%2529%2Bin%2BAcala%2BNemX%2Bcotton%2B%2528Gossypium%2Bhirsutum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0183-0%26rft_id%253Dinfo%253Apmid%252F16362274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0183-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16362274&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235309000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.55" data-doi="10.1007/s00122-006-0273-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Complete assignment of the chromosomes of <em>Gossypium hirsutum</em> L. by translocation and fluorescence in situ hybridization mapping.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D113%26rft.issue%253D1%26rft.spage%253D73%26rft.epage%253D80%26rft.atitle%253DComplete%2Bassignment%2Bof%2Bthe%2Bchromosomes%2Bof%2BGossypium%2Bhirsutum%2BL.%2Bby%2Btranslocation%2Band%2Bfluorescence%2Bin%2Bsitu%2Bhybridization%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0273-7%26rft_id%253Dinfo%253Apmid%252F16609860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0273-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16609860&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238345000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.43.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polyploidy and the evolutionary history of cotton</span>, pp. <span class="cit-fpage">139</span>–<span class="cit-lpage">186</span> <span class="cit-comment">in</span> <span class="cit-source">Advances in Agronomy</span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequencing of a cultivated diploid cotton genome - <em>Gossypium arboreum</em>.</span> <abbr class="cit-jnl-abbrev">Cotton Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCotton%2BSci.%26rft.volume%253D20%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">New SSR markers for use in cotton (<em>Gossypium</em> spp.) improvement.</span> <abbr class="cit-jnl-abbrev">J. Cotton Sci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCotton%2BSci.%26rft.volume%253D13%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.59" data-doi="10.1186/1471-2164-9-108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An integrated genetic and physical map of homoeologous chromosomes 12 and 26 in Upland cotton (<em>G. hirsutum</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DXu%26rft.auinit1%253DZ.%26rft.volume%253D9%26rft.spage%253D108%26rft.epage%253D108%26rft.atitle%253DAn%2Bintegrated%2Bgenetic%2Band%2Bphysical%2Bmap%2Bof%2Bhomoeologous%2Bchromosomes%2B12%2Band%2B26%2Bin%2BUpland%2Bcotton%2B%2528G.%2Bhirsutum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-108%26rft_id%253Dinfo%253Apmid%252F18307816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18307816&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.60" data-doi="10.1007/s00122-006-0234-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Physical mapping of the Rf 1 fertility-restoring gene to a 100 kb region in cotton.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1318</span>–<span class="cit-lpage">1325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DYin%26rft.auinit1%253DJ.%26rft.volume%253D112%26rft.issue%253D7%26rft.spage%253D1318%26rft.epage%253D1325%26rft.atitle%253DPhysical%2Bmapping%2Bof%2Bthe%2BRf1%2Bfertility-restoring%2Bgene%2Bto%2Ba%2B100%2Bkb%2Bregion%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0234-1%26rft_id%253Dinfo%253Apmid%252F16544127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0234-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16544127&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236585600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.43.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A standard panel of <em>Gossypium</em> genotypes established for systematic characterization of cotton microsatellite markers. Plant Breeding News Edition 148 (1.07), an electronic newsletter of applied plant breeding sponsored Food and Agriculture Organization of the United Nations</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequencing of the cultivated tetraploid cotton genome - <em>Gossypium hirsutum</em>.</span> <abbr class="cit-jnl-abbrev">Cotton Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCotton%2BSci.%26rft.volume%253D20%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.63" data-doi="10.1186/1471-2164-12-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between <em>Gossypium hirsutum</em> and <em>G. barbadense</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DYu%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D15%26rft.epage%253D15%26rft.atitle%253DGenome%2Bstructure%2Bof%2Bcotton%2Brevealed%2Bby%2Ba%2Bgenome-wide%2BSSR%2Bgenetic%2Bmap%2Bconstructed%2Bfrom%2Ba%2BBC1%2Bpopulation%2Bbetween%2Bgossypium%2Bhirsutum%2Band%2BG.%2Bbarbadense.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-15%26rft_id%253Dinfo%253Apmid%252F21214949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21214949&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohel</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Molecular tagging of a major QTL for fiber strength in Upland cotton and its marker-assisted selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">262</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D2%26rft.spage%253D262%26rft.epage%253D268%26rft.atitle%253DMolecular%2Btagging%2Bof%2Ba%2Bmajor%2BQTL%2Bfor%2Bfiber%2Bstrength%2Bin%2BUpland%2Bcotton%2Band%2Bits%2Bmarker-assisted%2Bselection.%26rft_id%253Dinfo%253Apmid%252F12582851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12582851&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F43.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180846000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.43.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Si</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Dispersed repetitive DNA has spread to new genomes since polyploid formation in cotton.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DZhao%26rft.auinit1%253DX.-p.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D479%26rft.epage%253D492%26rft.atitle%253DDispersed%2BRepetitive%2BDNA%2BHas%2BSpread%2Bto%2BNew%2BGenomes%2BSince%2BPolyploid%2BFormation%2Bin%25A0Cotton%26rft_id%253Dinfo%253Apmid%252F9582192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvNS80NzkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy8yLzEvNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/1/43.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/1/35" title="Linkage and Physical Mapping of Sex Region on LG23 of Nile Tilapia (Oreochromis niloticus)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/1/59" title="Genetic Analysis in Drosophila Reveals a Role for the Mitochondrial Protein P32 in Synaptic Transmission" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/1"><span>Volume 2 Issue 1, January 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/1.cover.gif?itok=D5RR8siB" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/43" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6id0Jm1sOB1xMSHYMdNaB78qGBHTZRSnxt7iE6EQyH4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/1/43" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/1/43" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/1/43" class="active" data-icon-position="" data-hide-link-title="0">A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459230" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5-_aOCG2hDuvtYmqAB7mIkbPlOOzVm-npbGv2EBP_PA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/1/43.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/1/43" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RXueolU3RTv4GpbUZWNaYSmiKJxDNLEDL3sn7UGu_PU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/1/43" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/1/43.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459230" id="node459230--2" data-pisa="ggg;2/1/43" data-pisa-master="ggg;2/1/43" data-apath="/ggg/2/1/43.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">John Z. Yu</span>, <span class="highwire-citation-author" data-delta="1">Russell J. Kohel</span>, <span class="highwire-citation-author" data-delta="2">David D. Fang</span>, <span class="highwire-citation-author" data-delta="3">Jaemin Cho</span>, <span class="highwire-citation-author" data-delta="4">Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="5">Mauricio Ulloa</span>, <span class="highwire-citation-author" data-delta="6">Steven M. Hoffman</span>, <span class="highwire-citation-author" data-delta="7">Alan E. Pepper</span>, <span class="highwire-citation-author" data-delta="8">David M. Stelly</span>, <span class="highwire-citation-author" data-delta="9">Johnie N. Jenkins</span>, <span class="highwire-citation-author" data-delta="10">Sukumar Saha</span>, <span class="highwire-citation-author" data-delta="11">Siva P. Kumpatla</span>, <span class="highwire-citation-author" data-delta="12">Manali R. Shah</span>, <span class="highwire-citation-author" data-delta="13">William V. Hugie</span>, <span class="highwire-citation-author" data-delta="14">Richard G. Percy</span> and <span class="highwire-citation-author" data-delta="15">J.  Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">43-58;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001552 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459230--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John Z. Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BZ.%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20JZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BZ.%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john.yu@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">john.yu@ars.usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Russell J. Kohel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Russell%2BJ.%2BKohel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kohel%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARussell%2BJ.%2BKohel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA–ARS, Cotton Fiber Bioscience Research Unit, Southern Regional Research Center, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jaemin Cho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaemin%2BCho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cho%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaemin%2BCho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mauricio Ulloa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA–ARS, WICSRU, Shafter Cotton Research Station, Shafter, California 93263</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BUlloa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ulloa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BUlloa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Steven M. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BM.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BM.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Alan E. Pepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BPepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pepper%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BPepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Johnie N. Jenkins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johnie%2BN.%2BJenkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jenkins%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohnie%2BN.%2BJenkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Sukumar Saha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sukumar%2BSaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASukumar%2BSaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Siva P. Kumpatla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Dow AgroSciences LLC, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Siva%2BP.%2BKumpatla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumpatla%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASiva%2BP.%2BKumpatla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Manali R. Shah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Dow AgroSciences LLC, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manali%2BR.%2BShah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shah%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManali%2BR.%2BShah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">William V. Hugie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Monsanto Company, St. Louis, Missouri 63167</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BV.%2BHugie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hugie%20WV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BV.%2BHugie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Richard G. Percy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BPercy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Percy%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BPercy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">J.  Wendel </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2B%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2B%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/1/43" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/1/43&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/1/43&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/1/43&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/1/43&amp;text=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/1/43&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/1/43&amp;t=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/1/43&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/1/43&amp;title=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459230/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459230" id="node459230--3" data-pisa="ggg;2/1/43" data-pisa-master="ggg;2/1/43" data-apath="/ggg/2/1/43.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Simple Sequence Repeat and Single Nucleotide Polymorphism Genetic Map of the Tetraploid Cotton Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">John Z. Yu</span>, <span class="highwire-citation-author" data-delta="1">Russell J. Kohel</span>, <span class="highwire-citation-author" data-delta="2">David D. Fang</span>, <span class="highwire-citation-author" data-delta="3">Jaemin Cho</span>, <span class="highwire-citation-author" data-delta="4">Allen Van Deynze</span>, <span class="highwire-citation-author" data-delta="5">Mauricio Ulloa</span>, <span class="highwire-citation-author" data-delta="6">Steven M. Hoffman</span>, <span class="highwire-citation-author" data-delta="7">Alan E. Pepper</span>, <span class="highwire-citation-author" data-delta="8">David M. Stelly</span>, <span class="highwire-citation-author" data-delta="9">Johnie N. Jenkins</span>, <span class="highwire-citation-author" data-delta="10">Sukumar Saha</span>, <span class="highwire-citation-author" data-delta="11">Siva P. Kumpatla</span>, <span class="highwire-citation-author" data-delta="12">Manali R. Shah</span>, <span class="highwire-citation-author" data-delta="13">William V. Hugie</span>, <span class="highwire-citation-author" data-delta="14">Richard G. Percy</span> and <span class="highwire-citation-author" data-delta="15">J.  Wendel</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">43-58;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001552 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459230--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John Z. Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BZ.%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20JZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BZ.%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john.yu@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">john.yu@ars.usda.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Russell J. Kohel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Russell%2BJ.%2BKohel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kohel%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARussell%2BJ.%2BKohel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David D. Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA–ARS, Cotton Fiber Bioscience Research Unit, Southern Regional Research Center, New Orleans, Louisiana 70124</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BD.%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20DD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BD.%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jaemin Cho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaemin%2BCho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cho%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaemin%2BCho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mauricio Ulloa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA–ARS, WICSRU, Shafter Cotton Research Station, Shafter, California 93263</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mauricio%2BUlloa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ulloa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMauricio%2BUlloa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Steven M. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BM.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BM.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Alan E. Pepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biology and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BE.%2BPepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pepper%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BE.%2BPepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">David M. Stelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Soil and Crop Sciences, Texas A&M University, College Station, Texas 77843</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BStelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stelly%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BM.%2BStelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Johnie N. Jenkins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johnie%2BN.%2BJenkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jenkins%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohnie%2BN.%2BJenkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Sukumar Saha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>USDA–ARS, Genetics and Precision Agriculture Research Unit, Starkville, Mississippi 39762</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sukumar%2BSaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saha%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASukumar%2BSaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Siva P. Kumpatla </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Dow AgroSciences LLC, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Siva%2BP.%2BKumpatla%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumpatla%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASiva%2BP.%2BKumpatla%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Manali R. Shah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Dow AgroSciences LLC, Indianapolis, Indiana 46268</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manali%2BR.%2BShah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shah%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManali%2BR.%2BShah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">William V. Hugie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Monsanto Company, St. Louis, Missouri 63167</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BV.%2BHugie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hugie%20WV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BV.%2BHugie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Richard G. Percy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>USDA–ARS, Southern Plains Agricultural Research Center, College Station, Texas 77845</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BG.%2BPercy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Percy%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BG.%2BPercy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">J.  Wendel </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2B%2BWendel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wendel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2B%2BWendel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=43&amp;rft.epage=58&amp;rft.atitle=A%20High-Density%20Simple%20Sequence%20Repeat%20and%20Single%20Nucleotide%20Polymorphism%20Genetic%20Map%20of%20the%20Tetraploid%20Cotton%20Genome&amp;rft.volume=2&amp;rft.issue=1&amp;rft.date=2012-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yu%2C+John+Z.&amp;rft.au=Kohel%2C+Russell+J.&amp;rft.au=Fang%2C+David+D.&amp;rft.au=Cho%2C+Jaemin&amp;rft.au=Van+Deynze%2C+Allen&amp;rft.au=Ulloa%2C+Mauricio&amp;rft.au=Hoffman%2C+Steven+M.&amp;rft.au=Pepper%2C+Alan+E.&amp;rft.au=Stelly%2C+David+M.&amp;rft.au=Jenkins%2C+Johnie+N.&amp;rft.au=Saha%2C+Sukumar&amp;rft.au=Kumpatla%2C+Siva+P.&amp;rft.au=Shah%2C+Manali+R.&amp;rft.au=Hugie%2C+William+V.&amp;rft.au=Percy%2C+Richard+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459230/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459230/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459230/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459230/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459230/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459230/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459230/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459230/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459230/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459230/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459230/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459230/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496056" id="node496056" data-pisa="ggg;g3.120.401188v1" data-pisa-master="ggg;g3.120.401188" data-apath="/ggg/early/2020/03/27/g3.120.401188.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401188" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">MYCs and PIFs Act Independently in Arabidopsis Growth Regulation</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496055" id="node496055" data-pisa="ggg;g3.120.401106v1" data-pisa-master="ggg;g3.120.401106" data-apath="/ggg/early/2020/03/27/g3.120.401106.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401106" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identifying Small Molecules That Promote Quasipalindrome-Associated Template-Switch Mutations in <em>Escherichia coli</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496053" id="node496053" data-pisa="ggg;g3.120.401160v1" data-pisa-master="ggg;g3.120.401160" data-apath="/ggg/early/2020/03/27/g3.120.401160.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401160" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/1/43" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/1/43.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/1/43.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/1/43.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/1/43.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
