<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/1/123" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/1/123.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/1/123.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/1/123.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/1/123/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/1/123.atom" />
<meta name="HW.pisa" content="ggg;2/1/123" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the Macaca fascicularis Major Histocompatibility Complex" />
<meta name="DC.Identifier" content="10.1534/g3.111.001248" />
<meta name="DC.Date" content="2012-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Blancher et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We describe a method to estimate the rate of recombination per generation from the genotypes of a large individual sample of an expanding population, for which the founding event is dated. The approach is illustrated with an application to estimating the major histocompatibility complex (MHC) recombination rate in the Mauritian macaque population. We genotyped 750 macaques by means of 17 microsatellites across the MHC region and reconstructed the seven most frequent haplotypes assumed to represent the founding haplotypes (Hrec(0)) as well as the 31% recombinant haplotypes (Hrec(h)) resulting from a variable number “h” of recombinations between the founding haplotypes. The relative frequencies of the various classes of haplotypes (Hrec(0) and Hrec(h)) follow a Poisson distribution. By using a maximum likelihood method, we calculated the mean of the Poisson distribution that best fits the data. By dividing this mean by the number of generations (50−100) from the date of the population founding, we deduced that rate of recombination in the MHC is approximately 0.004 to 0.008 in the Mauritian macaque population. When the founding date of the population is precisely known, our method presents a useful alternative to the coalescent method." />
<meta name="DC.Contributor" content="Antoine Blancher" />
<meta name="DC.Contributor" content="Alice Aarnink" />
<meta name="DC.Contributor" content="Nicolas Savy" />
<meta name="DC.Contributor" content="Naoyuki Takahata" />
<meta name="DC.Contributor" content="J. M. Comeron" />
<meta name="article:published_time" content="2012-01-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the Macaca fascicularis Major Histocompatibility Complex" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We describe a method to estimate the rate of recombination per generation from the genotypes of a large individual sample of an expanding population, for which the founding event is dated. The approach is illustrated with an application to estimating the major histocompatibility complex (MHC) recombination rate in the Mauritian macaque population. We genotyped 750 macaques by means of 17 microsatellites across the MHC region and reconstructed the seven most frequent haplotypes assumed to represent the founding haplotypes (H&lt;sub&gt;rec(0)&lt;/sub&gt;) as well as the 31% recombinant haplotypes (H&lt;sub&gt;rec(h)&lt;/sub&gt;) resulting from a variable number “h” of recombinations between the founding haplotypes. The relative frequencies of the various classes of haplotypes (H&lt;sub&gt;rec(0)&lt;/sub&gt; and H&lt;sub&gt;rec(h)&lt;/sub&gt;) follow a Poisson distribution. By using a maximum likelihood method, we calculated the mean of the Poisson distribution that best fits the data. By dividing this mean by the number of generations (50−100) from the date of the population founding, we deduced that rate of recombination in the MHC is approximately 0.004 to 0.008 in the Mauritian macaque population. When the founding date of the population is precisely known, our method presents a useful alternative to the coalescent method.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/01/01" />
<meta name="citation_mjid" content="ggg;2/1/123" />
<meta name="citation_id" content="2/1/123" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/1/123" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/1/123.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/1/123.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/1/123.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001248" />
<meta name="citation_pmid" content="22384389" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="123" />
<meta name="citation_lastpage" content="130" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Antoine Blancher" />
<meta name="citation_author_institution" content="Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France" />
<meta name="citation_author_email" content="blancher.a@chu-toulouse.fr" />
<meta name="citation_author" content="Alice Aarnink" />
<meta name="citation_author_institution" content="Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France" />
<meta name="citation_author" content="Nicolas Savy" />
<meta name="citation_author_institution" content="Institut de Mathématiques, Université Paul Sabatier, Toulouse 3, 31059 Toulouse cedex 9, France" />
<meta name="citation_author" content="Naoyuki Takahata" />
<meta name="citation_author_institution" content="Graduate University for Advanced Studies (Sokendai), Hayama, Japan" />
<meta name="citation_editor" content="J. M. Comeron" />
<meta name="citation_reference" content="citation_journal_abbrev=Immunogenetics;citation_author=A. Aarnink;citation_author=N. Dereuddre-Bosquet;citation_author=B. Vaslin;citation_author=R. Le Grand;citation_author=P. Winterton;citation_title=Influence of the MHC genotype on the progression of experimental SIV infection in the Mauritian cynomolgus macaque.;citation_pages=267-274;citation_volume=63;citation_year=2011;citation_issue=5;citation_pmid=21234560" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=S. Beck;citation_author=J. Trowsdale;citation_title=THE HUMAN MAJOR HISTOCOMPATIBILITY COMPLEX: Lessons from the DNA Sequence;citation_pages=117-137;citation_volume=1;citation_year=2000;citation_issue=1;citation_pmid=11701627;citation_doi=10.1146/annurev.genom.1.1.117" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Primatology;citation_journal_abbrev=American Journal of Primatology;citation_author=M. Bonhomme;citation_author=A. Blancher;citation_author=B. Crouau-Roy;citation_title=Multiplexed microsatellites for rapid identification and characterization of individuals and populations of Cercopithecidae.;citation_pages=385-391;citation_volume=67;citation_year=2005;citation_issue=3;citation_pmid=16287102;citation_doi=10.1002/ajp.20193" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. Bonhomme;citation_author=A. Blancher;citation_author=S. Cuartero;citation_author=L. Chikhi;citation_author=B. Crouau-Roy;citation_title=Origin and number of founders in an introduced insular primate: estimation from nuclear genetic data.;citation_pages=1009-1019;citation_volume=17;citation_year=2008;citation_issue=4;citation_pmid=18261045;citation_doi=10.1111/j.1365-294X.2007.03645.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Immunogenetics;citation_author=M. L. Budde;citation_author=R. W. Wiseman;citation_author=J. A. Karl;citation_author=B. Hanczaruk;citation_author=B. B. Simen;citation_title=Characterization of Mauritian cynomolgus macaque major histocompatibility complex class I haplotypes by high-resolution pyrosequencing.;citation_pages=773-780;citation_volume=62;citation_year=2010;citation_issue=11-12;citation_pmid=20882385;citation_doi=10.1007/s00251-010-0481-9" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. C. Crawford;citation_author=T. Bhangale;citation_author=N. Li;citation_author=G. Hellenthal;citation_author=M. J. Rieder;citation_title=Evidence for substantial fine-scale variation in recombination rates across the human genome.;citation_pages=700-706;citation_volume=36;citation_year=2004;citation_issue=7;citation_pmid=15184900;citation_doi=10.1038/ng1376" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Cullen;citation_author=S. P. Perfetto;citation_author=W. Klitz;citation_author=G. Nelson;citation_author=M. Carrington;citation_title=High-resolution patterns of meiotic recombination across the human major histocompatibility complex.;citation_pages=759-776;citation_volume=71;citation_year=2002;citation_issue=4;citation_pmid=12297984;citation_doi=10.1086/342973" />
<meta name="citation_reference" content="citation_title=Probability and Statistics;citation_author=D. Dacunha-Castelle;citation_author=M. Duflo;citation_volume=Vol. I;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Daza-Vamenta;citation_author=G. Glusman;citation_author=L. Rowen;citation_author=B. Guthrie;citation_author=D. E. Geraghty;citation_title=Genetic Divergence of the Rhesus Macaque Major Histocompatibility Complex;citation_pages=1501-1515;citation_volume=14;citation_year=2004;citation_issue=8;citation_pmid=15289473;citation_doi=10.1101/gr.2134504" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Primatol.;citation_author=M. Kondo;citation_author=Y. Kawamoto;citation_author=K. Nozawa;citation_author=K. Matsubayashi;citation_author=T. Watanabe;citation_title=Population genetics of crab-eating macaques (Macaca fascicularis) on the island of Mauritius.;citation_pages=167-182;citation_volume=29;citation_year=1993;citation_doi=10.1002/ajp.1350290303" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. K. Kulski;citation_author=T. Anzai;citation_author=T. Shiina;citation_author=H. Inoko;citation_title=Rhesus macaque class I duplicon structures, organization, and evolution within the alpha block of the major histocompatibility complex.;citation_volume=21;citation_year=2004;citation_pmid=15269276;citation_doi=10.1093/molbev/msh216" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Physical Anthropology;citation_journal_abbrev=American Journal of Physical Anthropology;citation_author=S. H. Lawler;citation_author=R. W. Sussman;citation_author=L. L. Taylor;citation_title=Mitochondrial DNA of the Mauritian macaques (Macaca fascicularis): an example of the founder effect.;citation_pages=133-141;citation_volume=96;citation_year=1995;citation_issue=2;citation_pmid=7755104;citation_doi=10.1002/ajpa.1330960203" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Li;citation_author=M. Stephens;citation_title=Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data.;citation_pages=2213-2233;citation_volume=165;citation_year=2003;citation_pmid=14704198" />
<meta name="citation_reference" content="citation_journal_title=J. Med. Primatol.;citation_author=E. T. Mee;citation_author=A. Badhan;citation_author=J. A. Karl;citation_author=R. W. Wiseman;citation_author=K. Cutler;citation_title=MHC haplotype frequencies in a UK breeding colony of Mauritian cynomolgus macaques mirror those found in a distinct population from the same geographic origin.;citation_pages=1-14;citation_volume=38;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Shiina;citation_author=M. Ota;citation_author=S. Shimizu;citation_author=Y. Katsuyama;citation_author=N. Hashimoto;citation_title=Rapid evolution of major histocompatibility complex class I genes in primates generates new disease alleles in humans via hitchhiking diversity.;citation_pages=1555-1570;citation_volume=173;citation_year=2006;citation_pmid=16702430;citation_doi=10.1534/genetics.106.057034" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=N. Smith;citation_author=P. Donnelly;citation_title=A new statistical method for haplotype reconstruction from population data.;citation_pages=978-989;citation_volume=68;citation_year=2001;citation_issue=4;citation_pmid=11254454;citation_doi=10.1086/319501" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=P. A. Donnelly;citation_title=A comparison of bayesian methods for haplotype reconstruction from population genotype data.;citation_pages=1162-1169;citation_volume=73;citation_year=2003;citation_issue=5;citation_pmid=14574645;citation_doi=10.1086/379378" />
<meta name="citation_reference" content="citation_journal_title=Primates;citation_author=R. W. Sussman;citation_author=I. Tattersall;citation_title=Behavior and ecology of Macaca fascicularis in Mauritius: a preliminary study.;citation_pages=192-205;citation_volume=22;citation_year=1981;citation_doi=10.1007/BF02382610" />
<meta name="citation_reference" content="citation_journal_title=Folia Primatol. (Basel);citation_author=R. W. Sussman;citation_author=I. Tattersall;citation_title=Distribution, abundance, and putative ecological strategy of Macaca fascicularis on the island of Mauritius, Southwestern Indian Ocean.;citation_pages=28-43;citation_volume=46;citation_year=1986;citation_doi=10.1159/000156234" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=The MHC Sequencing Consortium;citation_title=Complete sequence and gene map of a human major histocompatibility complex. The MHC sequencing consortium.;citation_pages=921-923;citation_volume=401;citation_year=1999;citation_issue=6756;citation_pmid=10553908;citation_doi=10.1038/44853" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=A. J. Tosi;citation_author=C. S. Coke;citation_title=Comparative phylogenetics offer new insights into the biogeographic history of Macaca fascicularis and the origin of the Mauritian macaques.;citation_pages=498-504;citation_volume=42;citation_year=2007;citation_issue=2;citation_pmid=16979912;citation_doi=10.1016/j.ympev.2006.08.002" />
<meta name="citation_reference" content="citation_title=Asymptotic Statistics;citation_author=A. W. Van der Vaart;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=A. Watanabe;citation_author=T. Shiina;citation_author=S. Shimizu;citation_author=K. Hosomichi;citation_author=K. Yanagiya;citation_title=A BAC-based contig map of the cynomolgus macaque (Macaca fascicularis) major histocompatibility complex genomic region.;citation_pages=402-412;citation_volume=89;citation_year=2007;citation_issue=3;citation_pmid=17174065;citation_doi=10.1016/j.ygeno.2006.11.002" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Wegmann;citation_author=D. E. Kessner;citation_author=K. R. Veeramah;citation_author=R. A. Mathias;citation_author=D. L. Nicolae;citation_title=Recombination rates in admixed individuals identified by ancestry-based inference.;citation_pages=847-853;citation_volume=43;citation_year=2011;citation_issue=9;citation_pmid=21775992;citation_doi=10.1038/ng.894" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=R. W. Wiseman;citation_author=J. A. Wojcechowskyj;citation_author=J. M. Greene;citation_author=A. J. Blasky;citation_author=T. Gopon;citation_title=Simian immunodeficiency virus SIVmac239 infection of major histocompatibility complex-identical cynomolgus macaques from Mauritius.;citation_volume=81;citation_year=2007;citation_pmid=17035320;citation_doi=10.1128/JVI.01841-06" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the Macaca fascicularis Major Histocompatibility Complex" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
<meta name="twitter:description" content="We describe a method to estimate the rate of recombination per generation from the genotypes of a large individual sample of an expanding population, for which the founding event is dated. The approach is illustrated with an application to estimating the major histocompatibility complex (MHC) recombination rate in the Mauritian macaque population. We genotyped 750 macaques by means of 17 microsatellites across the MHC region and reconstructed the seven most frequent haplotypes assumed to represent the founding haplotypes (Hrec(0)) as well as the 31% recombinant haplotypes (Hrec(h)) resulting from a variable number “h” of recombinations between the founding haplotypes. The relative frequencies of the various classes of haplotypes (Hrec(0) and Hrec(h)) follow a Poisson distribution. By using a maximum likelihood method, we calculated the mean of the Poisson distribution that best fits the data. By dividing this mean by the number of generations (50−100) from the date of the population founding, we deduced that rate of recombination in the MHC is approximately 0.004 to 0.008 in the Mauritian macaque population. When the founding date of the population is precisely known, our method presents a useful alternative to the coalescent method." />
<meta name="og-title" property="og:title" content="Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the Macaca fascicularis Major Histocompatibility Complex" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/1/123" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We describe a method to estimate the rate of recombination per generation from the genotypes of a large individual sample of an expanding population, for which the founding event is dated. The approach is illustrated with an application to estimating the major histocompatibility complex (MHC) recombination rate in the Mauritian macaque population. We genotyped 750 macaques by means of 17 microsatellites across the MHC region and reconstructed the seven most frequent haplotypes assumed to represent the founding haplotypes (Hrec(0)) as well as the 31% recombinant haplotypes (Hrec(h)) resulting from a variable number “h” of recombinations between the founding haplotypes. The relative frequencies of the various classes of haplotypes (Hrec(0) and Hrec(h)) follow a Poisson distribution. By using a maximum likelihood method, we calculated the mean of the Poisson distribution that best fits the data. By dividing this mean by the number of generations (50−100) from the date of the population founding, we deduced that rate of recombination in the MHC is approximately 0.004 to 0.008 in the Mauritian macaque population. When the founding date of the population is precisely known, our method presents a useful alternative to the coalescent method." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
    <title>Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the Macaca fascicularis Major Histocompatibility Complex | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_WFHbxVcndH1vkScBKzboQrrpY097w1cFTMtKIbuU3Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459218","apath":"\/ggg\/2\/1\/123.atom","pisa":"ggg;2\/1\/123","ac":{"\/ggg\/2\/1\/123.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/1\/123.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/1\/123"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/1\/123"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/1\/123":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TfZD887Dq+rA\\\\\\\/BALYsJEH5Y+QR2pEDqI1lYyYCb4Uzz6+UUO4m2gDBVdLAGVc+AnTUpM+DrsOHow0YKTXSU3JkqcJXZdTYOvuSVnJ4IxoKQEHum6kOoBHGr49NRS66wMzmOrytDJXZ382J6yPCAOnnsfow++s4BoMtL+\\\\\\\/sjA6wZB8L448nSuxAmPm7jqyUr4CBqCGwngzRc0gycaALpxcKdzrOGMJ3RgbFmhaLoGQX90+cyEW43BqdAErVvWgfqdWlhm6z+KFYMisjKvUPnc9hgqrKFNNZIw1AZlApexiOYZ\\\\\\\/EgsxK6XkgnDE9xq+GurQq74gXGndq3jklyyr7OhRbce8AnF2D9pGv8b0g+VWh1NGAGl3Jzxwf7pOlYxpFMKbpE1H3K3DeaxnJtyr\\\\\\\/vIbK9k3deXeqSdfPw9BjUy5jPz+M262jUXdbEn0qVRlskpY6rucFEFKx2ffDJU8dqHQVJwzGrQ\\\\\\\/IHidXrB19KhkwwQCaOT8yFsiJvnyk45E7yJ5qDCUDT2VHVnfATWezVpsEqa7ULMlHTc4OlHHqL9OklwJZEAhS8ar0FLKM1L7\\\\\\\/nJetNaFtSmaqct0mBMzQshF0TM8F\\\\\\\/rkacccQR2BDvC+6fr7Ev8DqI5ok6g\\\\\\\/xN9JeWgh2UxIzb6dHiDZ37WVkRq0YxkKNJvsJ+Y6+H\\\\\\\/UigGky0JSuGrPWtbTOrGzfLkLHW8+Z5cw17Oad6ZLTvbKTq8ZWAPg1+ixSYT0CvdnxR9dw0DK8sMEiAKrhyjiW6VnoWxPjzY11scK0\\\\\\\/qScqztO6K+EBcDeOtv2OGJN3AK57f6iljHE71YJznUHKnb1HBds7rdYVimytTiSN\\\\\\\/a6amKOasWMi2AriuuHtH\\\\\\\/Ly5BfY9B486Cw40uDSmH7MVZP0lcBr3PA3mim6cuOqACZyYh65stN6g9TiXCNbQ662RDqQkFk7EntGpEE4OuLc0tuQ\\\\\\\/18OMoNGocpb+tT15fXmPfU2YX6wdf1AW9i2ANV0Tjqo+zHtm7Qi\\\\\\\/gLfmrW3v\\\\\\\/7RGVPjgLCm2BDqI6WEPOsnlwc6+t\\\\\\\/Cqm4rB5tKAaOKYVsYAEGlgPuSKkDPYfK8lsrpu3QhWX6AFrgKyfw5kobw3OxwtCDvPaI1VUx20fkX1f0pTH8F107JV9Rg8nzorKezdD\\\\\\\/rJH1hJDZ4Dl4fzu+xtZ7uWOSPHR9XBnEBk7nYKlt\\\\\\\/TwoQcBHoWzzLW7p2I0xSwJeJENHSmYOaqQKEHf4Oz9wD8iA9vJBbxL3wgiCZZPxBT0p2fYHcyJgWuuy2xTGhKDSP3\\\u0022,\\\u0022iv\\\u0022:\\\u00228CL8YxWPSIww8tQuPxWT1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022308fa4f38a77990ea7df2c9ee0450f7305ac00b6b9e9a80731fc2fccecbce41d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CIV+SOjwZKUoLGLgE9tT++x+S4++tY8rHzDh904\\\\\\\/I3UQkwkeX72GZW7jE5qSjcOS\\\\\\\/59gYXmDnSFJ2iHQIm6EFlvA9v3N8AZkrmm9AYD15klSZQUzolg\\\\\\\/JZIqZ\\\\\\\/B2ntnsPAsgrVHIt+kn\\\\\\\/G1SWYWD0RtXXKxRXdSGl1UvGwrwbQUkEdwzLgTM3oTdmE22x8bM\\\\\\\/QSssnNUZ9JaStuPpTX3y4IiIghRJvcbiDZuuCsJnv49sRfF0VDDyNKJ0x0YBXphkwiL+wmjBj2WdN5maXz8Iq+V5Mszz4fifsFhnHpCFi0CZEgmQPzWAD+AuSBVGZJiUrKfXJ4vXCy4hYe9nZKFZkzhiamzSGaKpGsjDwbvpksttfIi5qqvDUyehEPDcqK5R0\\\\\\\/K5rv5x9S4pndAzmyPO\\\\\\\/pvG70kqM965yBRY9JI+IdHvr4sHRenMfS57ZkuGqNYyzliPP4hBBmrY\\\\\\\/1+cNtmOYadz30ErVD888NXGOOi+p5fsnKlKKyS1I1arkNfD9e12mnKx+VXkD+Xq6PbZbvKvaD2YZiC0DsGNeruQV8onPuW2fFd6DlQr8ANyqkyC6WZcnlSPRp9DD+zzzaJWG3qxRo9TuOfCNs9v1JwOYbi7eKz6E6HGTICSILqq\\\\\\\/PsBDYV\\\\\\\/McNj\\\\\\\/FFvWXYJhpN4YBYv1vjIVTM5BYId3bIMu88zzgzq50XBMg\\\\\\\/diGmQk9gLVsWuwCyA91evN7lB6z+Ob6fwg2Mkto3Xv\\\\\\\/ru8BSBsXf7AFdQl\\\\\\\/DWJr28xfvYBZDksTjc\\\\\\\/lTmZ79kDVXmUt7cHj9pq74EUaRZc+KDiNVP8PAxXVi3dBw9uMoeiissPQDpxwyHFI0YNjYZ\\\\\\\/KxSuIg8hm7OQeZVg87QkHje5Gho9bDzRnaB7F283OXS1MsMxtOtLF4Sf\\\\\\\/bBbqgYelVt90NvxxFx5GP6mD\\\\\\\/CTAPomfVOH7Y\\\\\\\/pQijdtT0EQeiFvy\\\\\\\/6M5WtL40+KJHejvELubRG+JSDnyug4t0YyGZzMGPdSvz5ZT3OGo42GNGm2mFOsr7bRU5iTVRY5POzmWL+9mUxm\\\\\\\/bViqzI+ayU2ZYdBU9nULvc\\\\\\\/BMUJOgE\\\\\\\/BkEofjKQSUKqNx\\\\\\\/JSax0aXxlnx25g4AngOPh1UQ0zUXS1fDca3OtbVFNHW2ELAKjzS\\\\\\\/cgQP9LMnz+FyrtvqNeMkqLaMtEAKxtdnZg\\\\\\\/HCXcB+RNLDKwg7cOM2HIv2gpnu2bo8k8y0Wggfx5pFjK14jRjdflhU5OX4gjpfXModLDVoFLAmBSRFjB0uNjLG7rjfgFVnHRdAw2xzl4K6JbcK+egh21THn+UVGe0gvNwR5rv34+tW15AWMFZZHFlvGJ7QKv2n2R3FnEAAhE755\\\\\\\/JsGIM8VWBDEcRSGFc2nbpIECET5HG2saVt+w0C22yHbrdKCyiXZMQftNbyLopGKrz\\\\\\\/\\\\\\\/dfiVrkn+okBlckCILV5FdAQKW+oMPr+hm9DSTaGaOPMYWOWwcs3lsNKMTbXHcENcAkEBStKrAvVUp2O32BJCs5OMxav+\\\\\\\/hTJeXJ6ImS\\\\\\\/sQIJUR6ZkBokBoUIglNfVBwIt6a6NbaM1TxYCtcd5tjUpmXIfezCKF41s06Q6V175V+75pCAOr\\\\\\\/AYGIWsadTBhTRm+IyreMoRNBwR3Q8Rt6qpZkJpANmmcGus8Qlwnkzh7AIMLuOO0l\\\\\\\/V54UazLbJ+E1giAOzpzbN4vLAdpzp\\\\\\\/AD1D1yGfMpc\\\\\\\/pMdeB1FROVnFnzr36rTREDS2xGiXlFxf4Anhl8wIPV4T\\\\\\\/tnVKIcGlTqsqJiaqqm8QJbQFkq1EkKYbLmoQtgZSv6EQ6IXaF+UJpIk3torPKXOvXZX9j2PfRKIgxzdPZHpEmqMrOjrVtbPqnnprpTJKgplekN\\\\\\\/o0ptHxKvtv\\\\\\\/JcJXSxjFHx9yqRshkd+2CMAI2JvDnJUgO99H0t9yx8BI1orL4iR6Ky77U6pdlouS529WQKMo03wta7uyXn7k82aa0+0uyKZtk27KwhKZ2uTailQgHpRqplVXD5ODyaaaZ+\\\\\\\/vmVczcdEwT2YnPhPsvlFPPU\\\\\\\/V7fQEnIdcOi8NqyF5yEbGGyTDaSGQPRjWysLhVXNDvQUt7b9pGhKS81zkL9AMhyx\\\\\\\/FYvl1TZVD4hsAXeEYxhXiVd+qHL4xSC\\\\\\\/LbhBtYPs7799OaeWtn\\\\\\\/P8ShH92ryufbX\\\\\\\/gM\\\\\\\/prq0DA2lyTimV1jYuu3X+8JJlupi2o7MXLk8PAkpayfA0NcquLZ80Ke3YedwsJheFLNbrvkZLoUeZamAlneAQ10kpQhBF54cbeMwAx0Kv0i9oad7WL1Gip13ukG62UbKP0oEohjrddewuUAL2ku\\\\\\\/HDjN\\\\\\\/\\\\\\\/uEhGmzf4LQSiCol67rR7rrdr8Kq1I1c+YE\\\\\\\/vFB\\\\\\\/8miJ+l7pcSBWIAxyzo+mUSyP6DPz6OwQRpsrrciOd5r8VwA7GDKV3vcudmvGrZyAmzXIirRghEHioeqwBNyS0JQAA8GW3UmuPWfjvvf3kwOXLfgTMffue2Pc7mkGnGbgvuoOO93I7wN+VtT6aA5yc8bQ264A6cGcwBLh9nkR0o1qYzEeGgVn50DkWfcoHltm5JyxF0brdN0oEpwOK9UTmE9NWvyUCf7TqLCuASxQ8tY9jhCWLaPB\\\\\\\/EMOHHwg5Wzs9LyGorulkcgk3Q6gVyuctTHId4mNYnBqEw7pV+KkkSFXqdyAVonPOnLW60oK0eWD2woQprT7Y8B\\\\\\\/seGFcb00qyM2qmDupT2F7smNrNUneUU3BnNQLlufe5tXHrQKLV4ihGJOyMGRgGONT1+p6CK\\\\\\\/2XDI6K9uzlZvxKAiArUFCgNMqQOqZ3Ve9RNHPILfZVi6DfT81NzX7VRkUl5atg7\\\\\\\/+c4B4k3JTIR+QckBOyQtqlcwvZdiW7eka+s340s658D6r4i\\\\\\\/5RhFJhLoUEGcT\\\\\\\/1ohF+jaXs6f6Iy5AIsLYvHmoNeNimeayQZZRMhRccIWu2THq55UbGnKl6Gtktdue4Sui8usf6q58++og85rl\\\\\\\/\\\\\\\/pLPmOnYXRD\\\\\\\/MPQYAl7aaikQEA5i6dKw\\\\\\\/72FBoIVg3icjd\\\\\\\/flfTC2Wn9SP8U2S1KK4kVsdZEAH6IDQZ9NcioURBcIQVigU5ME\\\\\\\/KtyVMlfp2i4ykk3MKYjKJx\\\\\\\/iMby4WznW\\\\\\\/UuNADGM2CjgQPU9CPeOAWHdBUJvo1NUZx\\\\\\\/W9y+drSxWG2fUuVZf8Oz\\\\\\\/WH7BrqwDIFCCdeP2OHSMuujQC0qVViCHYDTJnJTBhpzEJ\\\\\\\/mEYoOb4tsaHIn5JFvkioiKEulbzDgcFoGKcDK77DQDzbajBSyhilgbS9bjls3pur83gnhrnAtvmxvuag0j7BEOIRnoVTub2yjiYWzh63qHmCXUSdhbKxR5NdFfxzLZsOqybgQxfqMZMnohT0hMpCuJ1xExzwwSNxaTTCGP6ncbyW7EExBroJLwwnQTaG+c31T1+uYpbXvEgItVQ6WeA2jE+2F3Lw4OCeF8aHSxAjVlkvEUv7QYj7rkvmzWCR4Oa7a6X7aWBwX\\\\\\\/yb\\\\\\\/v0yegxeJyJ5WRJKWjVlJmUIDtrB9dr\\\\\\\/FP4hrc8fLmlWYtkFqdlDAHMg6r29kbZAwD\\\\\\\/jbBglMEduHXRqjtDvbaJgYarW3loM7KxSLMjjuehJD3hWsHhPDHNT9N56wU+X1s1z8cGkbq2rEwy1EaqoqSGA9un0GbFTlchryucoEURWHnfe335O3VIeuypii+fwnxpWqxp1YwTgQmv5Q+iGnhAmWxROIA1aIW6V\\\\\\\/eRMnMNdk4HyIefs2p0Xlf9pd72Qa1nJxrbb4XzgAVqt6QxKTdr0xDjUDTaay7KDu9kmvWShDZn+ju8+dCSzfJCIhWJC8ipEvDPDJX9SPQAPtr1dM14YV9+SrWaMHgy0+zVKRub1mdGmojlxayBJtPgSL1FMp8BYF+0t6bTyToH0Yi\\\\\\\/ApKIK1TBDnaYWKOm+7YRReTtPv6oAwX5uXEs0sPw1oyI4odEG9cCClbr6+zS3ycthh0JmA8VBSuUdXAOWavC+TG30U4GaAEWgDIYwLgsEyFl3J9vvP6Y3D+G\\\\\\\/aNCfgPdbSigoSK+v6KDPz1NUPwvr0wgUoMjGuBKQNsJl9BBCmcKe8oGrG8lMr71JyR8bbCE3lZVFMo2n1b7rv8BRBv3jr13lNK7BQBwEBk5dRAighvOxC3jGdeIxeMlnfnCwLDNyNzjGAAxOEdWdJXNbeD61k5AUybt7c6U7ZIzwKTYls60cAvIf+vYzPnuATRl\\\\\\\/+hdDkeKeP5onJ81MW\\\\\\\/M4QU6Jtzon8SpYjgDx6pOZT9joMe+Yt3CV3FCWjaIf9kVCkqgfgQWgkn\\\\\\\/avojvRbMwPkc6T9HsTdr5gg2xfivzgJlLnQXlAeEPuRJ\\\\\\\/a4oFNXqeU5r+lvXdCUgIIUGB0wDUqCaVUWd+6LJNPsmAD7bNBwfpmTlv1QOfwpVZrVQY7HfkuzRIZiudmjoIu9J2ZjUcTJUf0YGGv6IV+MdLinEE+KoUo0uEJUQY0dV8ZwrR+Aw8n\\\\\\\/W29CLk9jkcT71RsfZOJ+gIVoA51yDx9IMgVC6tr0G\\\\\\\/JhRNu9l5irmlKvSGxkkGi7Fy3Y6XUps\\\\\\\/cipEep4dyFMnoMN3bIQBoMGtD8FHoeHl9eeN7mpMyhj5TWDhrrlcfwKm3fTKe\\\\\\\/5kJpJvFFeLRgt7D+DkhaDlgIrp1z7X4I54w0KClddvR9mcW5kz+OqRI4gKPGNF8hMPRXQ2bL8gLVsR0HKmbwKw4rwRFV9ZGi34LKdCD5YLULehWP5kDBritKlOTsdxO9o7lI\\\\\\\/L6JreWBnJtZIxvpxh3L657dVEITQ2H9EdnJTEYuVQvy7xYrMkxOWMgK\\\\\\\/8L3xDhVP4KEah5eWpA7GlkJxRJEbYcDkkYzPeS6nWvKlZQqvme3q0qGEigt4CR0\\\\\\\/ZvpnFMGmHhxoajz653qjgRmpkyDN5WqQr3NnTXAtgVNPunleIANUVOF425q4q8\\\\\\\/88mF\\\\\\\/BHQFxvh0BrKhlfolPM\\\\\\\/8bTx5P\\\\\\\/iKWcoWZx99YGgUnRfVhqYlDAQcaR3trSkaI7jYFn2JkNXyGLP3LIZy2oFiQcXhDGUekz6QoJryvwunj7dCfosZxINyBcdxaLMcq+MUBzNr25kHCvjMuTVuH7L9eRdP1Z80XbDB0IZzQJndUul5Hbzy6CpXc6NnToQNdYBxdXxDr7NMA2sC\\\\\\\/n5wNsWjbsIoNY382\\\\\\\/mI4QcXSMpx89BLr9zulirn5k45BTCXZv98fTaVmYEfTJZNTl9eytQ1faK7I3XxdvSp8L1C\\\\\\\/CbPzFG89wBovM9rvP8NZ45dOgzvW3vP3cibkRRU+KUdjVj9pOrz2mYhA\\\\\\\/fOzjoztAOVAW8oDAw6bmUb9m9SUo3LPJSnmQ4BC2tI5q+MguBMFFXYhSVjyJXvLLGNAesOon2xA+6G8hr7mFphD+\\\\\\\/cxczP3aiOXYPJH4N0PsLXKFxf604QbE\\\\\\\/QrK9jklTlaoSbDaS6hYWu3oFILtqh5H7tDNdqKd7ZSLXTCOHumket9nrSujcl++K5mRUIkKpiTu0a\\\\\\\/suxnkHBSVvB5p909nguhnLBP\\\\\\\/VHln2YvoBAFUWB29TkOyxscLn3Ya8iQQI5qjMe8m6hKv1OjA1ncr6dC5OHt02l\\\\\\\/MkPN4VzcWueTIe5TiPAoML3DejhAk5T32BlwVqfF1CTuKEv9jLVFgiT3b9bkjALSfNckCQhiH8Do9cK6Rslz8L1DNTyG\\\\\\\/USggSF6BUEiVWxLCxYQ+OiIyEnfw0VvKSlA0O33MiJArH6MuuWhs0wXezSzBlrA\\\\\\\/npTXbIDrZlIP908eds7IGn\\\\\\\/58AoO2cjD4SRHWvNg8Vffy0Xv\\\\\\\/ZgTZ8LJ+Bd2amQJIRoELprlOzmhsrMlUGnnhmpq3w9CWQzERz\\\\\\\/RgOKwe+1s5M99gNmFNx4oM6L86W2a3OA5XGcANfDgwlNUfWrsD+ld5L627VxAY+PCAbc5QhQLaz+Xy2Wdml0aV6MvnHmNlwbpn8f1qy3BMocGd6txC0gVLfaSDgWcFOTXxzz9HdayXl8dqiTAEz6E4W4iHKAMFLrY7dtTi3tPxIPWcnotHYx0uIuXEkb7N\\\\\\\/7md5omSr6ZMmnAiFyo8XwXkpPwadxSZhpqatG2xzrXcvB0fX4vz1XBj2qiAB8d9xT38dXyBol1i4ZNS\\\\\\\/2bt0k066v90\\\\\\\/XYmvn5rPxzgabCsvdGKG6lj8BqlcAq3OpzzK6Xy+pmMo3vsTQygTkojKQWoeqSwuWiZja+Btch++1dddrcCcYI9157waF5m4A7xE0DouNjFW0M8pwkHxAdST95lwT70uZFVhPLBWk\\\\\\\/SweYk34dgP3DCXE1aRbSMuckf4YdBu\\\\\\\/lgQfxwplN2TQAg\\\\\\\/\\\\\\\/ds6vs5ZXC3EMUNzPwz0S+YqU6LVFS5NvW8jBG7\\\\\\\/\\\\\\\/Hhq75GSobQTiaX1ZupRpw0Lt3okXOoSwT8Cy9s1d6IBKl\\\\\\\/3UfD+rV7tJKyUHFEitC0ttImFpvxUEjrDE71pJ\\\\\\\/\\\\\\\/nvTBGDHm4eAGXzX1QyA\\\\\\\/u0tZiDGdcyHPsYagiaK95YqnPvLa27geQRHTd+T9x99XKLSqu4aG9uYDHN7zRFM2JBkcaSmFTSzn56ZYYtvFCiRTke4gyD21oq8Z6anqm\\\\\\\/JNz8Bgpu3TlcomiN8xeuMwgfk\\\\\\\/P7s3GS4eQu3yDqS0FVahR7aRBwX65biKGBhftlHUIrVPQRx0NIIgiXs+VNyvp3lUwMqvI2gO7Tb9jm+0+IiovJn2\\\\\\\/KnPRHtFxwJyz5MoiTpXvGO9crqDmbvwVCI7szryvdXGg5rzpq9pvqCoqtp66zqMxsGsAzudTRFwYEjbOY3tEn9T4wjXu00KvRjn3zfXSyTSO5+P1M3l+QCvQ184kDMwwFuIbCdbDaKbQpbk8NmLlzblaMNqry\\\\\\\/j6IGlLu5OaCjYypUVyM9ACu5rsS1rn4xf7KKPgbIrB37fmHyIh3m\\\\\\\/dav4pzzwOWcj+6rWhe+6e1+59N+dsv1ERXZbgOko96wC+aRx6Mhf+CScDLMYirjh\\\\\\\/zx90SEkLjmrpJQ099llDOFiEnB1JeFmc95BIvTda14oO73tmyVNrzGgjIZDRyVcYpIz3pOS3RZ3Hu5MiQnOeKkocrxUJcTM90KcZ\\\\\\\/WsC+xuod2shqGnQC7geEvkSkytnNKpPlYTTZxPpE7cH7cne0YpbG+9z8g0FP5KdZV2mDw+2ID\\\\\\\/Oq\\\\\\\/s0yM7zMyYn+Y1vJE\\\\\\\/aFAMUeyWUTPT6RzjygO6h4OgsnKCa5QNypUx+03eqcFrLg+OkJTY3iTWtfJfQ8JaepHhuP37r2jbfS65ySb0PvH23nLtf++992RmcrTByPXTI18L0GZyjcczaBXyDTXiksEdFEFH6Bd6K8UNWKO+TdxGrgjqki1SUjTvuMdwlxALkC7SHD\\\\\\\/SP9h4A3lCfDHykHeYq7n1eMutvK042wmZyyjx9nXrxk8u+FFkjfgdB35SW+M+EJRDk+knCdsleASZK+mf7epuAZuKZ2xVOpDYOhKdX5aQNUUzMEVa4ozaGT\\\\\\\/lpT4amWWie4KnDQlKLgBXfh5\\\\\\\/KDJ7csZjPM0aMeDRtjev+fsXsxGotaltn+uL5oUfhvJzKNTxVztbhiesWQSVSeaDy3g4nvB\\\\\\\/g9NGFL93XuzgmxNRKB3hBC\\\\\\\/PsSxdql8MBvq2bTckOgwyVxnOQ9Di\\\\\\\/CLhJ5lykLDnItC1krAJOs4b4CuKYoarxw+XOAM29rDHiJxA8ebIiLYx8scD9FGwU0Uj0sUpacNAedJCnhW0qeJQZmQjUCgDssoAVvKLFQMblCIO4pbRbPRekV6F4K\\\\\\\/wU3C3eNjM0c4bec5R\\\\\\\/IzVoTG2iKfbVfxO7GCWtN6aLf0SXBT1UmadtlFRmod3GsCs6c7MAiQn4APXDCf3lZzNnbsZs2i\\\\\\\/YTfB0p\\\\\\\/FGF7hXiZHocC6I5z12DHIoMhYe1ozskSuFH0bdI5AL9zK8FAAkeDpr\\\\\\\/0p8NgycUdBsQ71XNxJzFqCzwyIpWE8D0DP7MuLs6UdAWrXIsZYH122W2Q95QbIAbB3Hg9POmKr440Ly0k1ujHCDrTdncavNbPQ6IC7ZHWqYFZSBHvwIG8fZGtUshyZaiarL+JEqbluKE6XN8fnLOs0Gl+ykTsA81zKL6L9Qa2CP8pigrDTID8V2nQ313xrB5L80q6xOl1cDbC5lXBf493V5Ed3TKEPVXKZUrYSfMLtFYFB9YJilm7BiNoHWDAFhrkwuhj0qAMyKci1odpN33bSVcZGF7mhmq4cbV7BvatVCkYb3yFcYOP1M1JQM9ITQwvuFEDT5A8Wvzbzn42b6MYWHlOiaXDtffUU\\\\\\\/vODakYTiWlqPaAiYS6kzf+5WnuD9mRDDyShwuX+oNSbPgrAM+abp2UY734\\\\\\\/dVisdsbXZGbTt8IsovxNSj4QiDwMhc8t7kD7RzTT2hK7ZdQOsCHvz9CBaFwpnYx30tfB8pX\\\\\\\/g8M+2HTz\\\\\\\/qyRL1cE8Wuqvawmzix2ifagnKvWa1tVi0v2NgiirNe5cBmIW2bzJ4lk1LMhmpNitnV80miKv6nwhn\\\\\\\/sSrr1mar5J\\\\\\\/paO0f2\\\\\\\/4t2hbGykuwvPXj1\\\\\\\/SM\\\\\\\/Hrx7Tn\\\\\\\/WR5oxkyznNTF5M67bDDyuFiOZR44U3HRlmUx\\\\\\\/01Nitwe41TTBpdtIzLTHUYBvOe1ac2XB5deLaOG8wwiZ0GvqiHn006dHOO9SEcD8sQAttdr9xC83eha54fw3O4WR1lmkNG2XR0KV5iqmmELQvv7YmRSu3060fSnwwVqf8i0Q6y31ZhgAx2cH+zxHcib7z+08UAvHp4as3c+SchHmGbGRKUrb6saFFYHxPM\\\\\\\/DA\\\\\\\/IPrP61Q6iuMk450nddWQZThaelrq\\\\\\\/dAix9ICnGixV8yK7xL29ELUAKnjCuQELgs5p6C3ZHdFvB2WCETIbGXt\\\\\\\/UKGw\\\\\\\/xnd\\\\\\\/q5377OMwgagl8NCDSz2CCnZkDjnncpEHgizL7FhzBRikjZD0HjvI4+Ehqk\\\\\\\/gxEKj6zGTN2CI84zcRq39al4TZ7XpxJekfgF\\\\\\\/iGn\\\\\\\/+1XBzRR5I0jQXbpTZOODWk6I9hpaVZbjQtxs0Kma1veRdEDPqk9AjTm4Q+3wkSHUTIvxRBS8tDLx6rUP4j9JnUvp+8WBqf4L3ouX6jX3JkuQdtTFvnNqifd96quIxjvG18F03tz9cZ+C21pE1qbxmKflYhmNkXcqC4dsLW\\\\\\\/GdEQnrr+cHN5pVluTNyAwkLSHRAQrmYup7W7XoNdC14PC9cedES++wbNYQipUpTj79As27Xiq2lCHSI16Z+nhbybszxIE2IcVZNgwRLIrWmwNteJCb\\\\\\\/D8ihjiNDOWIOFFGw+se7rBaGWySVO8POQGiV6VkADuWkz9wPTzrYSvrLXCa8350aQzAYKT3uEd89aizuPZQWzDfXdf7XBOC+ba2UamcdrpNRm2O39RLogrg+9kNzMr\\\\\\\/ORG\\\\\\\/Er244awfuFQOBAWd1d1zhujBFaW8Fe15xV7suKRorAmNnKgQELhBTK1qfxT5PrZBCyQsSv4tkXFH1yQ8FyxHLF4RCJgAhqvtP6m6FHg0Phx3YBkIz2hB9qbTwsCgTTIkOSOnYqf2KS0xe43JHoUfcxqeDrOmnuXxD2wDP+KYf0GK04gYdQRmyfqbAg2UsdKNQjVBn7NlPj\\\\\\\/QiuQf+BPU6DqU\\\\\\\/S8jl+R\\\\\\\/szLHhe0ssf4bkKNKiRH3UixiyYuGQQDkC\\\\\\\/2uR3Y95HZqQ3IsNhC5BB+c6WGstlcEa6Yv1QoV1C+is+64KQ+T91mF71mv\\\\\\\/2LEv+J8CtSZ5Q4V830+21iCcsaHiuxg\\\\\\\/bb+XrrU9x6LiHTeHHV2\\\\\\\/uyjofpQmH80QAAbAJiHVhLWecgAYzFF9PdF2bnbddJMuf10tae3UOHch7ICuFe\\\\\\\/ny7G\\\\\\\/0yDtsL4PFZYU6Lt3Ah\\\\\\\/qJ\\\\\\\/7flmm2FdVxrevb4jSYMxE\\\\\\\/\\\\\\\/uulI5XuEeymrikQ2x6hLESOjcuBesn3vYn9ghwGIk4DCYvC5zndKmIweaQZqUuN4O7By1WgI8SjbfVP31wEKwEe5X66+3o4GZjRpJ9LSVuiAwmQv8OhEQYZ8A5n95AWoOT7vNYiLAGtBdMWPzz6Id9MVsN8e5r5Q3t0osbAtz+INlTxs0y3zxL66iU66J2MA3lbEXNS9alZ8d8CetvZKE0cGghVxb9Eo3iaBChzoNt1mNB\\\\\\\/ylm1Sbb37exUKGxKH8Nj\\\\\\\/W0UTSOJt9+rjHKI8+iLtD95vWsx72YE1\\\\\\\/g0PyqHGhTAQeyqlYH2snzOlmGgvjdEe8hENi3FsL5x5u+XnCLmWvaIR\\\\\\\/M9Tliddc6p7O6g\\\\\\\/1gJX6P7LzvdH5YvsWBqbCMHCSsT5KdGMWAEp3a9D1gEiO+81WCuo\\\\\\\/3\\\\\\\/nofEQ6JHs+hH1u6Sn0PiRUitH6GDIDa71DecgatNqcza5VXy+oaD1BXJrPXWgV2vmAezqW3WnwlgmafiI7BvcAi\\\\\\\/jY\\\\\\\/QNHAhNhsyzeFDO9MRc495vVVv2FVcUsV6Lme0s+XgkiCeASSIE6iJhTPFtWc\\\\\\\/AM3M4wV8gIwsC72lXjoOQ7nyGWU1E2OGeQ7hsCVatutI9mSeH0LqeBy1PvQlRDcZScvieUfVL13Tz67m6c1YSjck3RS3YkXis+ilLVWOSeOnCRr\\\\\\\/wfHufrO8uI8AtBXjgJ8tniu8wUKG8CtuxBtq7+1sWPmhRCKrmXLUjdPY5Ltz672HkpqDNfkvBwLB1I9N47kYTzHN9jFpC2FIKX4bVqoLdcAFoEiV9p578AVh7F+HQnfNSpH3JSahKn2ue4Sy2t+P8bKE2zFQ\\\\\\\/mdXs5\\\\\\\/+d90crmjOo3SAOu8mB5qdtBf4rBKTSLrPP3D6SXX1zP0LSDL8l4B9N3R4KXaSzhf1bnOitq5cMXs3itUvJAnI6UrIOfsey\\\\\\\/Kt6HwdFqiZwdgb3LSJXvnn8NzageEL44ZgbIxiHcgmcTMCHP6UbZbUP4hho4HBppjz6mCLvNIYx1aKVGtPKTRWXCYQt7rOlVDeWGYdOb3QALiauV5NK0nFiwR9JmAKkOkwxDsNAw2e8et2IzraNJryjSNBpxOx2PBna8epjV7c\\\\\\\/TEoj3ul9zEv2cfao43wMAcS3KsSEQW+lB3se5+waD7718lesj0MGWlQeRnoAPfJecGTC8jjtWG8q4xnh5NSy8Qx8fwJFdxPHV1bVtRQvF5ldE89bqZyxc12Zo\\\\\\\/C8J8me2VNHHY0QPhatF0Z42jV+PNqtS2us3H0FXJEw1CnRihF9YHlRJJ7wwf5FJ3a2Zi2SVjs540MC+K0elNhffOQ\\\\\\\/AsA\\\\\\\/sPmP3iglmaQL2NESZ7u+qpHXZwTfADGKMI0ZwmUJ+uDqayUFEoDvM7rd97bneBZlaRc04Y5Y75u1HADg29k8alkCjo1PcW4SFpnUKOaLon+\\\\\\\/NZ24+iVbEgPMmZXa8OFfL8V+PJc\\\\\\\/rlhB0+Uk1j\\\\\\\/Me+Uk5ok+kORoc\\\\\\\/Wtuz77cltw9uA6FAqXiE6MNHirgYukTbIFqJmuuJLl5zFplUJPdHpD9KvGZ5o91qXfLpaMob9aVwk858N4XwcCHLNlep9CbbHEf6FlXfsjsZ199lSYBeXfiOvc5+KYDeEKtZSxIw36LpGKR0phdbo9cDOGCgHqbHQ0UuRY6MOXLijKzQTvX18KdHNxHVFnuUwBPG\\\\\\\/Ny6wq8vxI6p5PoNKZREXAbRel4P2sUooG5B9NNpaCD70jKVgSTIfb4ni1lbeVPWwFDZca\\\\\\\/kHQuNqhSmVyB+AF8goAcB4QYMPniBEcO6rSnSw6E5t3raNfrE6uErF9R7WdQIfTcOi81y63CbC+bJcqLdG3Jo3EhK04LLikYkrY4eNF3jD\\\\\\\/FSLFvSQDMzsmEf3iSjRNzOl1TrfZOA3QtSm6FFjKukmwKLzHjLWSVb62GIGpF7KWL8y4wAHP1F9lUInk2SG4wxucq851m6Pzbq3M3bVGFK+uLieuZCESki0LHaU4DhqA32XGBOb1W7qd1lIqaXw1lPmo6n4TMq3koSn19KzBievIX59wSmqslsF8YXhWGGZc86Gn\\\\\\\/QuQamfkegEnScnxwNT+5WbbE3dEr6E0gdp1nU5XFhR+UIiDOK6vIFy9fXHY+Q3hJxgav1NzU59WGSZHnomsmaqN2JTwg20eksAx4PZw\\\\\\\/3Np3GkMmtMefmYNdAjaYkWnzeWUu8Zlo7FsnTyWstu65O0DA8f4f\\\\\\\/HUSckgD0Ld1VhRdFy6uKqoa917yuQM1jWEnLBU6YDyarfhU8N2K3UxDBNuTOB4TOG3wGeDEkXGW0ORN7pN6aI3hRk4USsknjv11gidcoDzy3C1GZw02Lqz8EvAvqSzp02atHV401Sn7ex0ddx2Y\\\\\\\/GTPVBDKouvuUv4PTGVVIiXY8NArOBc2qeDU7ngqBKYMTuhDQl1Pu43rZQVRq9IKQ6bPUr56zi3N8+MToekO94Q2d\\\\\\\/UenxUslpu6J7jYU8zi514+200MrKhfVCvhirVhZFKhhpTxpMSehYpPThmt2R6If0wptSpJM8wPwcWc4aKPGlcImWFVrG9WJjomsVQ8a7FKj0t2bJjHOQ2J1Qqm14KGoZdMyKzjPce5rC6uQa1d1+iHqGNeWe9RoMaUHrLi8+nT1j98GwAIYNIJOdzVNkAF6FNB6syyBiN2H9uCyg7h\\\\\\\/hphq5mCLN2lM6IxfS5dA4ayQcuG6hyhb9xAfPpL4qXyHbBscw1z0f5yCz228RISOpuOPPObEHcmqqRA5TUiDg3wzH3aASBW8ilXzeb15vKnogo0ZppyaFV+m6SORT0\\\\\\\/JK\\\\\\\/B9jsqqNac4+ZE\\\\\\\/qpJskPhEY5mupfAxDeA4Aq9mUaA34QNqrLxpOfKfaqjgJemLU5WPYx3qLjosmabwy+kgnUMcSbap2zF3jgjQXPLIjbjiofSMBSDleOvh7kQ8h3gz1JwFYPsKuQy2aDv6Jp9uK3LhMOklJJoh7HvCysbeF2TRX8FNgNAQi6b87puo5K9d5+nv+1GGn1M2Dozrx8A3p0rfRT9Qh\\\\\\\/lhG4WiOQhWH+DZvZK5mwKKyJBWPUr0FJyf3kSUjATAOhUCywFR6Kv\\\\\\\/gKj3k13l1Lc0oAjFjXqbA+BZ3YmCvOyZEV+e3FewYH2pNKS1vKwGRm5uhpZ3qJVLZ9bpLioRNbNNk4zBrxaINcgdtD2\\\\\\\/4VjdlmZzPGdbsrbbmQVtXhOAx46qmP5S+JnVK1P2SURwa2EUdEl1\\\\\\\/w\\\\\\\/dfVEvoAg+FR\\\\\\\/2L4xrFPKoUIlK2L3BVRI2aKVHXvMKtW4lVriLBtv96EUMC8W4qp5mcM86A7fDm0JFkl6PPC\\\\\\\/Acn\\\\\\\/3z2cPblziZUnnMxXAvz3xfHcfJdKb\\\\\\\/XsmIjWvRHu0X5xDqAytTcjlx1ndVZsGxZSrDk8xsmojqbT\\\\\\\/s78ZG4Ov0U5IfPrVOzWjmZuPUcSeyO3mNIZwWrEyxepArH+Zok\\\\\\\/RKR4AcFHI9RWv1gwZt7vb0hmE1zTA7xygLJ51YZNoSGi3aDBxF6AsrdIXRvK4kp\\\\\\\/QOiXkXfyJr2QRoHCNEhnr4vC8cpZ2Fz5gZMbmErTHzhR35EQapUDEoWhPdcgVTpdDFV7\\\\\\\/mllvlgZSLVik7EgLCnA0qfEUnf9gTnaRrKeXGvtlvbr8Ev3MTQCmzxjInBHbISNvdgn9Wq3iSoek+eQhUkwvryawVPHh7ESRdExJDu3osrUXJWM5dzH1tFrRJYQbqdgbR+YF5wiqh3XG\\\\\\\/Uon+P5m3DsNcJhbOmG9b0sH5SC68RAdQ7ompaMf1IaIzPmcTomLw38A0Xw0ai2Kdxv0uTB0BZhJxQ9uiv2ezqKAZ\\\\\\\/1uWACg+wicQAvc9yN\\\\\\\/5VQFnoimqkinCml8zBIrZzRqccV3fQiaDKlZF1F8KC0eOK+tMLE05Bo25Kw637eP4jzVVFzEkDYTp1GaXentUHO56cdCTl3k8adZVOlyYFATYBuEIcd2vxD0aAJS4js+dRMBkbuEL71rGyspQHFXOVefc0NbnpkuF5a957UUMhVFxXlwd4Qx\\\\\\\/\\\\\\\/Ga+BAMxztbxNJHIaF\\\\\\\/OWtwlWMiIf8T8XuyGMyALhdI3ce6x1zm14sskjAqzUGtItqtnWutIvPFEbQBOOuL3ceJ8Mn7yJCPJpUu0JlaSaaCvptXdL5SDanpM7awBI0H7bq5LobV0vVOnjmykBflVibq1vLSMyaK11uTsm77pe082AHv3ta4qMDtjIH634eHQ7ojZlz48L1LZHs+VVowcdYcLnzTr8lViJ\\\\\\\/fYBHL0JnNoJnmN9OnzmZFrdM2bptt73aNi1L7p\\\\\\\/vPgaYMtFn3V6n0h0rBeuhRnmCfGU+K8bgP0G2NwpZq3P+qVp1sXkNE9wrE8pM9G6wT0Yjzk8ffH+rsRMMy91Tu8HNFciqgLOTTo9RiLJTJoX5tqvkjAClsZzvhgxSu7E+w\\\\\\\/UQDk2Ru9R6IiST+Mi4Df53k9AklEpsOcLxlVwu7Ayn0+\\\\\\\/LVt4o33aA4OFXeQ2FBgHJVGKpOnPMawWckZqi5xHoONg64esswvHHb2dJm9mk0DQD8BnGM8lZTaPXnZfhnjyJgLhZeDazWxcay2VWhmrL4v0yPa+Yvai0kkpN8JspjUQ6desX7Oe4oCIlDxtcZ9+RyChcOBJC+v5FdN\\\\\\\/ruA0pVJAfMKxtfjQ67glV5\\\\\\\/yMqmFVN6w0r\\\\\\\/VZtVIIWl05Evp8zqza+OrS7NAUdpRrwyDzQJrF5aK38cqhCjx41fWjcfjY786DldcSIL\\\\\\\/tpB91Rz1ESn4lV1XExbhsYu0wzz8CRFudXa24XRhxXC2DKQ3omqoFmq8rzRzFls7cu7ezOl6a7E7W5wxZAKVa+77zVTJLyyozyKqqqGyRmj2p\\\\\\\/wSNos17TPah45jZ\\\\\\\/VllipIXYzDlqJhcycml6kNBbjmfDR3AY6aTY5d7J\\\\\\\/nYipnpT6Qns7WJjrhw+WHSOhljZlxWV86DaSAwktIOwmI0phjtACZhm4Z6gR1shlUJs+rN1Od7w7G6J+H6\\\\\\\/KfG+j7zUXMSo3AdzjAmefg57rDizy2FsiUt831kDubIpOXgQ58bjHeUoFtwS6qTD3O6utSqAlB2ZwsIwohNirUfEr\\\\\\\/Gl0QcZukiE9y6QNfTlcDf7Dkj+0cCANTuHhUY7rPPq3FGWBT0PZGS9D\\\\\\\/cuvLwFIM7P8Vdd0zi1JThYrvKA03Iw2tsS7NfW941uies1A\\\\\\\/FF3UnxSOtnW4WxwdABfbi\\\\\\\/nbi8Jj+GbQeDWHlWDN4aKdUqQ9f3J\\\\\\\/dDfcfg2mp32UY88kdhf6hlbUBqygHkzKkx1gUaqv1ByAD2DRhXns9CJCZZ13RMDsO81eq8LRMb1RsTBHWH\\\\\\\/SpBb4LmM1OMFo\\\\\\\/OOyOLmzL\\\\\\\/zOZTQuBjllZKJBxQGyJ8dnFQxZxcmnk0xaAJzGcQfXFP+888AjbFx2fqZ2Nmn8GrnRQd4\\\\\\\/yF+EKqC9IwZdhJC1X7DzEOlC220+eKkkkeI4hC3DD3BOdbEnhLAPdPLQYLCbzE+5sifvnuarPmaf7WFxdDuSXzjaHoNw+0kVpf4x+mwj1XUOXCWXcp0VaRyizs9kgmHLE5CgEXyiV3efd+SlHP+SXnrRkgPn4TDr9xQ1bgzo7sALTlO9T0I6AkcCGJD9tO9TIyPKvyvmIqg5oXKg8uFnm2evn\\\\\\\/OdmDp\\\\\\\/i86\\\\\\\/3lgtxQs8LjKQnszj2NB\\\\\\\/U+cgPILTTTjm+0h7EzMjHzFHfKMMOOTfX7SaV6y2C46F2xgp0J6uf1\\\\\\\/lBQABdt4OYxV0kPipijWynIEDj24EWpo\\\\\\\/mPeqs7AuFYecio2jJLXcdxcu\\\\\\\/6qj+a\\\\\\\/\\\\\\\/n5+gz2fIpGAhBS2IQBBVDVxRieQ9Rr14UFAaEdAxrqoHhousdp8Z8UHaneZKD5o4PPx6yNG9jXdaMk9ChBBb3C5G8DAHDxOQU8rh2yBxVwVJI4Q\\\\\\\/u\\\\\\\/9E0ZQ\\\\\\\/TgoJjZhPXSvzE+eBlUJl0WnRcJ8gtNArFnr8Z0UxujnngonkZB+2gKbPgMsw70TI5WVDRd2bTS1OkxiP3FFw4+DL+nYjismi2Gj8XRSnaacrGA\\\\\\\/b1bHuZf2r9qYgjmiWi6rtJcXoGwAWSIcCc8dTeveF4lagFcDlzPn7NlbdTzGQPYsmcfsVqOE+2tMgCkX7CbmCmn4rkQYWTTTrqwTP7IieaGGblFHW204niGTXdgg7rJDrrWc+qXLJ8JL6gXy7RkF1mEwgHt5\\\\\\\/SGH\\\\\\\/HaHYPfYxg\\\\\\\/uTQd67yGCuzQVUTVkaX4Afk+1JNeXGAIrRRdix2Mn8\\\\\\\/TeiYB8V129Md0r0IpgdIsySmnul4lvTRqGJeRgdQVfU5Ai0aBO+pNd1lB6sVhwRaJ3Vy4p97t9xC6wJLSkYm5A+yL2giEUxgBLd8OFQ9walIxpIRr5\\\\\\\/q\\\\\\\/1ZxZ2VnqfAfz40QfIKpW8u+7rIIo9d1UPsD47acp6SMS4TjxPX4xjfyaMdIrI2mtgFqHhYNS4dwWm9TsoX7ebmq2dQuBGAJqbxL\\\\\\\/U6MfJPkCSRSEJ7aHVJb+Nz2lAI27i3+jBRjyzvTonZhW9t40\\\\\\\/+z7a9H8eSB5axQUtXRvEVCT4c+yNi1Naac7yQlW4keLS59O6Eqhs0IGqHbuOsDGYEXZBLfGSE0qOnAQ7+oB8MCRdoGWDjqk1kl1r3SGLpP2R9UQmQABYsoN8hs62JzN6f7xq90ftOKt6OyRZZmrt+VCzmXNdiIuEeNbQidgokTVhZL0wG\\\\\\\/DOPuElg3kp60uUbsj0\\\\\\\/81TRgv136Azzqfun9g3O21i\\\\\\\/f5iQHsfCsuNjxCDgszkkpx79RAMUHANS40+JpMGVnrvABLd7IvyraLDw4R8vEBSJVIIXDaitBqlaK\\\\\\\/3CBr8VPPx0NEHSHIEAiAOO6NhiGwiuOVQEsNs5zIv8XesXMx3Lu\\\\\\\/JN\\\\\\\/wmyrPfSkHjFwXxfnjreRZvmmzN9nYeWmu15WI08b9UlE1MvHXC1ANjTdIAvXXQ2VqfZ0VlkL6ddKxhQZH5StfMfhejItZlrYlR\\\\\\\/IVp8pKRsYlcpMIZKGBQxGO6GpGV3TQ1v5y+NfRgRilCnLxPFvMnPH97ysWSUE9P6XWTyQt5kBl0r29VD+Px3GKkd0vXbBWi8OUPeinL1+hKdWQDV3T\\\\\\\/Raj93Zg\\\\\\\/DE7fEEJTsi2imS6O3cAOqhqUzLzAUM25tMQdjXJFdbyBqoAu51r68mOSSbjxb2DK6c+7j8uTDiByzkXX7Mjxur1nO8yPoRs6\\\\\\\/RKgoTYIRpAH4usogP7vEcW+qBwx2mxJXVgxOde0UhlQw5UpHu5Rzm1jVhzeLarHequF0FkjFXOJEizhCItEIcBpxmCpcSyX6+WIsSCc7cVbp7ALgrcGO3FZUiyAxTewgv\\\\\\\/q7K4GboQMn4CjleDkbRtKUXJjvJGJVt7\\\\\\\/996QGrTkPdx68M3RXOyTZ30Jjf9HE+RPqt9i6vheupi5B3JgCdLMVhw4ZvZTeZHi2AOIqi9pQqmb39zmJZpvmMvNGhZUsLj6MzaF0BR7FuENjvih7sFu+J2jzUr5B8WSGYjsw2BCAeRxFzU4kUFMHbrR4U3LFfpAReXTrOUGwONqqqdxG+KM4KP3dnlwpFMSn29zpkMbvNW\\\\\\\/zr9ASw6DicJe6UrZIWGnSaJfwQuLFsBHylb28rPUMN7deuLrarOWUMajBYRlY3V2uB4\\\\\\\/TqsPTs0kt3InQ8gVm4IL0Ps+vcbThzdjKdEudzTesb\\\\\\\/qNZtuNvJdNZCeaU1lqWCKqh4sNOgG3bOyqwiju+KsEEOQVo\\\\\\\/HvQEaMqcN8nAbNuXxN7zaDSBO0yZBx937E7OQ9zQKelgiQlPhemGzBUTt4SZKmO2jXDM8hjMVgyEW18E0w6iGCmvCYDlaZ6j5AKuEsxl317v7EDQWGRmHdILsyfWO6eY49rPTHC8Pf9V9aMBv8SqFHsN8\\\\\\\/+IlittiHrIhxt0ZAQEPSj4eSgU7NB6K4A+ThkaRlNgpSG6nfPq1AzFZgOBd\\\\\\\/vVAw368MnqG7WL59t3tNJEKLtRsrcGgXaYhFM4bH0OA3u1qWDN74QL1zmDpXbDOqNmD5QJsMGghP\\\\\\\/CF55hrI3uoFlqRCGyoEXWXmNXSBJTwr8bt\\\\\\\/bPOqk2EP7YPv9i68Y0OONR5c07WMS2ajKRAo+qAHIx7AMyn3jFXwxg4\\\\\\\/XQ3qdhwBiXkxCDS7RQD4amgZc5TomhKVmOZKFFvd8yiByVEgLV3+ccSC\\\\\\\/K1N1UqN2OKV5VVLO+yh5mrL4B3QErKUyjHXRmmN6TVhdf6U8QkRIS8CankuaMDr1Ftcup886MwyRJ9Vg8oSGE9eBttFv5sjkQlkw+N9dXvRqhoOM6lVlsxch5O92\\\\\\\/H2y9comCz+9PueZ9Hsykj5YmMQPqgDXV9BkZ0mzQi5eYemw2AaO2ZuMS1hCUjPLAu2BcTCSNUO4zQH6Vz5YlPONM6Kgiyep033+CgFdglZBX+4Yv1WvEJ0sFfHoHP9vigrVCVzGCxbvlK5crvsnKBSWhRIQZ4alxK6ElYs8keUB3hO+N3\\\\\\\/NX+ACdtFxxscGke0ByfF7hYK0wK5RPj+CS+6M64f+jSp5ocbWtCuycuBsR3TzDVUQIgwGE8C\\\\\\\/VEn4iP6HzYkrIjnyJ2UXsqIA3tAI+xPvEhaSEX+czpLp858YKOhKaus47CvUCABPPBv6zCDTfzXMUZr3OGifiNe7fDSrdf+KRc8M+m+uek6CKG0kBK9BWhb0xbWYF7vWJ1IDK8wh2GcOtCEaMXno0rtRmWfknCQezRrfVVs7bV4iCjyVpOaoXjYi4AowF9vkwFsbmIO6hxXESuSU+5h9S9667Mwt0xDjryZ8pqL8V2ryFxetqocstasZnZVI5kt6ejam4mQ0jVfL4NXS8UT2FCMzlzsyfNdjqdYKrmlowgWHKkfu\\\\\\\/RUEFTPYAeBC9SdrPEZNSct+iEGf41yuEpzCBx+Ac3PlCcsD++ed5zBS+ii4DQ\\\\\\\/izIcWcKj++qMSKEyD8HGRR3ZImy3ei3jNve9eD3jMrOpTZ0fwiURquGFCqC7tssUM4sC+\\\\\\\/OgCZZ99CvLA+ToRpg7lQ+TiGgttL9nWfs2PCMWOxmmScimX0uDS+w6k5KzVKS7AqNFgPy1epRlKPzDxm3vCA+Z4xhgN\\\\\\\/1J2qQWL4lCzYVWXuzvgWhC8n+7mCKGydj+3J+f8\\\\\\\/Vf+cm8NpnoqbONuOPlmBxrR\\\\\\\/o2DnZIqe4fY5jNxr0fDrm\\\\\\\/ozVZIey5bG2TarGGBRFpW4vjXdtqIRjCfeoKDk37TxIzRMusyumbads30mMg7S3VgwSzn\\\\\\\/uPU6abRXuBXoUpIiR0QyzQ1hRCzOJHNQPOFXQMW\\\\\\\/wdxERYE84KTt2M8d1Mii2XnAVLx0ZQc2BDci+uhFpZ39\\\\\\\/EQlxAlAN2rg20i5HZu2uKoO4kpPr5sg1kyLYpMpgp5\\\\\\\/XODcTLH\\\\\\\/WQK5JeL2SmjPXL8buRfKoKBbC6R\\\\\\\/mp3qsIlHKubaw0LK+ppQRrtei6sRSAAEa1nNNjORgt50trvlVVcrSUt15MujcOPzEjOjkAhgeEIGKsTfbP2R5rh0+hu6nEdlkvewxUN+uaCI7AZQovVUhV8mloghjWQ+Nw\\\\\\\/6qad1QKco6MGXElGhg5k2zbIUBIC3kCGb5ADYb6s0MhwsaFNSVMG4kW1LVtp161KlAn\\\\\\\/PNeMRZpNyyq7W4ZNoRxeK9yYupFhT1c934H+J+sjtLUSXiGr3X4jJGQIAtDw5fdw4RaJcVxes8uQcPRW3a0JHUnHCsFKlWS32vgQlEkWQhMPlrjXFyxb1KpgWZc6QoWb+Ary92O1Kn\\\\\\\/rPxEjCReRnuC03jTB34JwtuLh6dalQVHTeQ2FItqJrklvp9THBB3B8NXum3ArQeHKSXXV0WCg5sEwL0i715Nloi2fo9KRrI7gtNagL9wh6Kb9D69iwWsOe94KCYfruKg0gIsLFdU4sEGsoKyxScOnVZc31sYJ9+oAhvXrYhNsezCkXnTN\\\\\\\/OVUk18UBmzdAeBhwQhZpD0lMNX8owsdj2zGmG+juZqyn+2DdZz\\\\\\\/nYn5uvogi96y2nXKKJjeyiEIURcC4cM0vd68eJVAROYvstWu9QWE+198ZKv3Kqa0UI4TFl8y62htFTYHyuf\\\\\\\/gnPHBK75vsefhKQOHFvmFBAJOv3eFi+EbQsMHX+G7lUoFXey8vJKjUUyQ3arhm2sRyD8d8eAxkOwif82EnYI\\\\\\\/vrJjYADGzIml+mW45Rxan20oqbdxcLdfVkDeqM2ZIBZZdftc8tkSz5+orXmEz00TTkrDigmFUb+UzHSO18oZWFJJ4Tuw4nFM4lgSAnKXAgkTE36pVFsmA36x+iUcvH22EW4yAzs2NwF78EtdV9okYitedU+rqR4YiQo0lMKgF\\\\\\\/LWIlb97d2QmVFOoQ6e\\\\\\\/7PVwiVx5FBE+9irdb4mYdQRISOZbJAWmr9eQ0gXkB5znr3gmRin2lIlxLbffvr1oOSPe1aHOv03LvxSDgGXQTojEfivr\\\\\\\/8ObjJaVx0BWLSO7mKFalRqdQlB\\\\\\\/a32fZBuMRAmILRwmMHhmh0QucV0gH\\\\\\\/3aOasWvasDNeYkMhYQvTV+OZ7+Z0J3nmnx9CGUV1RgQmnl7sSmBQ9KKVdrnZxzRB2ZL3oOBRfEoUHUl6sqabheYHpHi8XL50Bm42n+qPW8ryqtXntSwLNO1w21xed7CAlgbeFMKu70V5LZ6cRwoPpKT7nJ5TULLH9uKuFFnkVwterLc58rtimcuwI5kLXS5EUmlh0pNH5FT3zAHWo7EJEvivX2HIKhuVnoYOouFaC2ZS\\\\\\\/mxeR0D55bMwc4XaYrdN\\\\\\\/akM1lowVhn\\\\\\\/qbZQjKivz95EiuBCbIlh7iS6X91pNO\\\\\\\/chMM\\\\\\\/C67jHIl0XRi0JAQ8qO9O24m+DQ7qFo6\\\\\\\/aCpe6KxDoTzvkS9B0A+75lGUk0YcBTBw3Xvxiu0gmQXisVy0prU1cMD5jweAeleS0OyDP0Xp05bd4TwbNveEHaVbq4xwoLl37BZ4lD8+FMGYXe88glk2QH6C18nTO5Y1DNJWkk4c+spQeuU8Yy5cyyxCiH9ZjB+KKiJP2B+uX4nQMz0bd6+f8SQTa+4xBPVU\\\\\\\/\\\\\\\/wN9rKQ6KTBhwHEPlDxfRiD2JWFWyD2GQJQTjvP1FphyQ6EcFy5qW4r\\\\\\\/dNzf2COK89MgwY1+MDAUnYETkqbXJ1Pa5NOFWYV6cZp4cyOCGJEo21Cy24ymYXbBJ56U5u10v4kHaaFOl6zvY3r1G3yhms4wPcSzxXyTc\\\\\\\/ufRYRLOXHD5XIzvV4JbW8+d+ZBNmbIWlwK4e2v410bJlrnexa\\\\\\\/CE1YV8zWLcY8H8CncgZl2iO1dsCwt8QJx47UI1T6acaavoiJvqfPkf2bmCbK55DjVap1HwkUyMojgFsOn379gt+0PmEJmsP8ebLmdSXOVqzaxXB9WauL0ZIrKM97pt61oHRS2zK6Px531+m8XCqo8QX81B9nj5OWmsa0zoFU5dl8q3g5xNHt6Qa3VYovwWwfDEMLXqs65YkKLATmsnLwjIn5NoAWWPf7CCtsNjExtP2G0g+8PBJZwPS+z3zxuvUICUmREeFDo+gfUDq0rFgO73yhg4vac2cvMZtPWLJBP7DHYN+DYalBiJNTTBQbKjQYwkBW+rrSbcwYUHiaJ1uYp5E7XHauzzdiKfcDaHgrHxmwA7d\\\\\\\/95eZLkW0otfh4IorrYm6UFRBXK+Imi6Q59ftD923uFRCSw2Es2PLzwyAtixZ2SYg9kHp3gC2axytiWmiGrdLjFoG4a\\\\\\\/dMM+FLqfZrPu3q3rai\\\\\\\/8F2og6rvkd5l3H0t959U9uOC3IpQKeDByJA0aH5RazASjfCBu5N0sr\\\\\\\/okPZwvKt6s0GHF+VLhHc8AUgMZSW40TY6MHXiq7pv5vpSl1u2zKZEWeo1T5AchdARhqd3I1lWXXC8I+BbXcbFAVk14oh7cxQypHa270YTtNEybiZOAiSRuhukwcNjDP+KDt+yQ0NhuVB\\\\\\\/AK8kmRlQN58MdF3ZlEKJOBjP99NlkHy03DfzWzq0CPw8cFTBHiE\\\\\\\/kJfoAJONujlJyrKN4LCSV5fgkokFHn0fjwxrMphRJZMCtyF1it7IfxhpwfzJclPdiTu4YR6NQ9CvjQCZWyfj+5qR2mafIJvTulWmTkb1hMKC5MNF0jqCM13XBnENHCM8oPMr9QShCRMk4E\\\\\\\/MfwKHXTvdVSJ3+afPDNyjicj4cp\\\\\\\/ARAxR4xLbC2wQeLkrkbGp67MAmCLRBtve2M2aRZGHs3ELsn0ErHw4wZ1CrvE0+0Iae8UB7RL+ReA89Iv\\\\\\\/UUoKPnrNwGuimTkATeMvHlHi0U4Sy15c5vhdRjyeA\\\\\\\/pnt3\\\\\\\/w2vGckqZkGumvPu\\\\\\\/c96KBiquL6DxkjsvkVAL1YFmrqrNCrCwNgxq7KuTMRXfT38bpmbzVjFMiwGRVDy9cHcvnfVchp2mPt2Lc2xvTyDK8btbQed3Z9HyswVvfm4oCXuZVjw8NZs+VrQAgRG72z8fucmgrvD4ccR4Y\\\\\\\/fMQA3ygGyc+dzq3WdneOXKmtPErWD2LSyjBXVEnEri2aTBvywQG6AjiNAFuDUQfjHU7+vvZBJhaXlI1+7Uth2Kyc5hZbEklN4kAzOn\\\\\\\/Qz6e3KzUG4wuob7g13QUkoFkKZCcDHpksQTr2sczBpcA5xruzOuUgpISXUO6BGsso\\\\\\\/IaC8RQm9RWJStd8gAioT9fOS9THc4hNUXqBg2Orabgb5IFKVKKlXy7fSIyTv7s\\\\\\\/nMUZLkypCFJWVIYSTV1pW7xIA1JcEkKfbtwUcNWF9lrrpcNCHJyPFCP3G+E1lNOXj9qQZNsIM4mrEv2R\\\\\\\/dLh\\\\\\\/iJqxVVF3HaLnpSvkKE+4HfStOLjUousQJzpbj6noQV7hz\\\\\\\/0gv5\\\\\\\/KasjIINM\\\\\\\/kTO+ctUBhA0xwyMlSyYojR+uJZW8JWGAAxqUQYwNwSZk+5sBRuPzxeNlWfbfTfVkxhSfQBNt5fqcHgO4X\\\\\\\/nBFLW\\\\\\\/+ydNEB1Xyo0Xga0O\\\\\\\/ykHmGGe31GUWpRXGi9Y0a5WVMHxh5PCH9Hbm0ACPyS9ru7YMtycuF4EqxhTQkF2gjJB2H\\\\\\\/x+IZpypdSEkSInHEXH\\\\\\\/tmVdLAhC3yLHBxzaafJrZZTTkkv8JWGPNlg5E63F37plgnzNWXACvlt70zrqFlArpC63TQItY5K2Ru3XIsPkElG3uOyv1A3WKEkZtXdmWtGTNSUnXjmqgQQg8EsYkEYrjjHewqCqeCvK6BVydicGokByeOExoR8SINEURnk5mg6I1zth69XyfZVQmVHsgaqIPGMXhw+Csm7ajxy7M+ZZItBVOtUc25tRtN6gM8Sp5enYzFTaH3NOFl94+ajfcCaU6bK+co6ep199kBv5+sByTs0qDscsjB13dU6CNQ\\\\\\\/7e7BFKYe74pwcqlFw0eUsQrJu1d3PkoRaooZLrv\\\\\\\/3EbIu03FuCf4L\\\\\\\/rfo4orXvoctJQW8lfnkzXp67HDMhX7pdpb1awYeXtItxJdYBeXoqTpYfpipd4apbywux6D441jpyh4sOS2TySMUQgpUgBrpzi2tuEy+QXB7XR\\\\\\\/goyZXwivF9dJ6mugXnVA5WNiqKnRKOubx4Wr1p21oYPNpjectz2qCTms0CtvQ9kn69\\\\\\\/9s\\\\\\\/mi8r11WNSpPcCP\\\\\\\/zvuSdmuf3EQ7Fp5j6J+7GkXCHka6DT+GYyCt2aR0Gzkve6oz7NefHnpY2uzPMr4RbIqNOGTxMPjUPDK\\\\\\\/Q8vrQv5kcywtMCuVr8oq2HuREJ1Hzz\\\\\\\/BMvm33M9sZTnWEDG7vpb48EV4Yv2NY4k8th4yAlzdZ7BDPHegpYOQtBTTlDMk2A2Ejn4Y+6Te9P\\\\\\\/dKvtwvTZIeRzWzHwGRdQpCoFfi+YSUQVYLj3ZJAeSAACm\\\\\\\/6Gb3\\\\\\\/PLXG5tZ+K8sNn9\\\\\\\/dMY1IwKzc7SN\\\\\\\/MMNLVSDnHPoKVcjpPjFeO8rFC55QOYT4gAl\\\\\\\/cSXT7y7yeRQRvIUO2MtRLTx\\\\\\\/+DH\\\\\\\/ClSC0Uu27egz5bNRBV\\\\\\\/8ZOzKNIq\\\\\\\/PNzjEepRSSCy8gjSsjLrfcAsISVTAJrdzIQ\\\\\\\/4QLlZnoautvE93g9ytLZReFPq1wK+y8kafUC4CmyVesaW+OoEhsL6CnE+VZMfUxOCQQzP4Wys7TIubt9aS1Iws2GXD5I+9cdzDwaNm36FLp6Gfh9WkCkj2jc3e0e89LH3RqYeDL4iK7j0Mfl1PkPvpomZelubk9uustq+IcfL6Tb7Z4Vm5tS9hvC8wHNwLDltVyDLWHrNUt0+7nUe0sXdixd63yqaqQd5C6QBCcunlmD1fgakKGsP+8KyIaQn9FQMeGZFAET+jrYH6ocH5Cfx4AA1h28SlFeVbzOeJ6gz+zCGheMbXongGtZFcFwHVOC6\\\\\\\/skONAwhNV3GrYJRjYSe9pa36h0LFnhCpQ27\\\\\\\/RqgVNSEtpUIHRUx9M\\\\\\\/WCthzRJ7uAqSBryr4Xw8CW0SJ+INvjGkCWuZj8zoNhqc8TpQR5xYYh7CIQPTAxpHlkernO42d8tTnf0aubSFyMKdWM4yHjvXPQ+Tdc2Loo+ENx\\\\\\\/SWf6+69mNEYdT8a3vsHpXg0835M5qC70GGbqW6ZWHCPkzxxmUWXUx\\\\\\\/fSQd9CFRIwAB+TXgI2QE3FLe1MmZO9KlRBEaFKqWJOROlbJsTp2m0vkHhTZgp\\\\\\\/PZoFNfgWl+XrQ3ruEK2HIk7Zs7Vfw04XxNZSWGlXPPFPS66sFu3isJsTzf90FLqjec4whRvpUixtQhHOpaGgEuffChAObhBLPqsilcx0nJYPGwRH7VNnepILc1oukgw4HwyLYaG3TIg5ihEWLzVVvnfpAf3MQCfDRELPw8NrMztrGUtQCWA2urc43J+th09MKTKfSYzP5c2EwolT8\\\\\\\/YhYmU5D0WfCtOB1\\\\\\\/DhK6dnQGSiLF9mFsKHBCZ1fzNCrTBZsV0\\\\\\\/Bl2BiJaLHT9n5h2oUDXSM7dHHGpGfO016BZbOnpCFtxUzknIhi3FtZSrQdFbQFiXt7H5+Fy3SUb4QUjyae4sBwQTGPb31dL\\\\\\\/GZ1eDIQGMZzaBEqBpNsuhdPslWr6GWqdtSGSAE5UM961AeqUA6Oc+hzoPlMeNRJjbfDL\\\\\\\/n2pB52FI0+Ub8MsO9kfEMTh9B0o9UNgTz4pKTzDbKH5m+2N6rxNAQq\\\\\\\/d6YbV5\\\\\\\/kUnsF99eA2msNsyhojfAOXlIzDpB0xSwcSB4HeR7j8v9WrLhpBuYyvfSbbcntkWxp3gGzMiQormU6G2CtxpOorLNxwKNRoz3BFTnS8iQ3SR8TIWHLjUf6g3OW3gmHDgcmg4iVAZvxbGvIOiwLq4q1RF71Fi2z2djbUJOXxlCS7tSCFqLIKJQfmGCqT5FOfBq8q7pe8w5Ubb9nvsKadaephKWW75xgMMyZKgi48wusYJ36qg6ELIvp\\\\\\\/ziHngAJ4R+OV0r00LfUb7hoI2ZtdImA5RYgFx1nx3TaNvMq7KCaaYmxtdW2GXKWCQSmrO7xMY+lC3MRBH+Pvg1CtprSt\\\\\\\/K1NUETDiUoIM8dSOjRmgiP8B1LDCREzOem2su2DznEL6IY0AJlxTWCDjUuMS41POwlzemKBjXtmhlqKyqFlm56MGblV4AaeFm1UnRLbdRZ0xxjgQYfHBvKZMo\\\\\\\/7lL\\\\\\\/XT\\\\\\\/r2hvjZgBcO1E4qUVMvOsbixg6l6rXl4upiZYZGB2zRHmJwxeanfprD9jWwPJG0rLhV01a4w+jWs545z9WZ+JSVwaIeOm26QA2NedcLfW9yGWp2oh4RLs2SrwHB8T45OBHibjnhWggKhbYfLvxvdwl4SB2Dqe7UglOSNWHS738EqgFPNj23S6m3aumpzofOiC986aJP6ZQuYUW4ke8HI5doEV5bniS9g1pg51kP\\\\\\\/efcGcV2e3k9z5FrLwZ1HVPhUlIYG0WMuhgvOygi1ydDDgtgvDNc7p7Rtg0CaH\\\\\\\/1Bcq4sWSoCyt7xvxmF7H6HYH1MKjLThRQJN9fcGpt+TnLZp0LNRwsAQCMSHLBdst+al55oSLYFgv83e\\\\\\\/HD3dRzcTg\\\\\\\/G\\\\\\\/WrKCUlwt6LuLERJpuUfxXQu+XCs8zhHl\\\\\\\/6K5FPDZSNhouD1t3zz4xmkO1DAHDHJ1jJiBWAsqSRWRQq0fp89OCByh7thMkQ11EEaE\\\\\\\/ibp+5f4RBl5cFzWVg\\\\\\\/HP56Rpj+dDhlj43RaZmkt7Bp+oh8\\\\\\\/m8NSW0sHoxIj\\\\\\\/SRC2uEQl1hkYrp7e16f9rGAFbhrwuVzZTEB9LWujdJbBxTQAr8bwH8829ZhaH9G+KPknFByiMaVhUbdEvfakZqcpwZlaH\\\\\\\/KVECduU5R63ICErb0DH7x1e+RBm7wr67c0ROfBMgPslWBD+XQ7T8eS+A8sqwYRrMYtQtrUw4CPv7iBMADCYQMvDjj+OitJP0olAXXRs\\\\\\\/FNXze4cp0Q2FxCVmqSuliCEKw3cjW9JbaJ+X6TqeUqarQIi0k8d0o7ne+HnatenVZnSCxQVeifkhLwjsR7txFTA7lwRoEf+wJASxprQiESyYMoA5eeAFi9FqcUPYgsdzdcvNg8kpKuupjXepe13e9KBmkS8\\\\\\\/6Sprt74YHIZdHEpCsi\\\\\\\/Ui49a84DBP1hV3F1rrYbr3JI7dRiFnqkb9hfXTCqowYCFvy5oBywxsLZZc1IMdPSwBLwYhB2e2YQrZWMcFEoMCRpCy6a93uNLLLhGWer\\\\\\\/dTyKuYo03\\\\\\\/B+wW7AJxa+NkOnYYwGwmIrYLI3zcOdHTSIE4obZa4RAbTg82kpbzBd2e8ZMUkXmDO5t9Hqc+zRF6\\\\\\\/QryQW5Qqeiz+NH7ORzUcvOawx5xnKXmOwhm7iMW7pgQgaR6QV7Llj7gxCw3gyBdxgaKdQlQZLzRzSS5RvdHR0ByqbhW\\\\\\\/GQY1iW97Ik+6yg+sMjFuPYqxrXxwFtdEzJ0aA+QH1eWgGDfOlhkLrxhRujlcPhpgXqMZi1fQpZwSpG7R6qaE3Sh\\\\\\\/Rft78z84W4zDhSZIMsfm+pdkI7Qfxm1lnx0xEU9K3pLGGaiyNhZKes1M5a\\\\\\\/ueWX1i4TAQ0QZw\\\\\\\/+XK9rXNKrF+iXBM3A3JNZ1qsW9TzHRTcSRMEsZE2GqWCIIuTkNK3rdIOYd6KkwLwNgDcQYQjh5XvtxiKYqJ8okWRW4M2HCrHnDWdg+LuibgfeddFWnYFjjbHWGrnobJGulp7OCqOT30LMP2RV\\\\\\\/pcU6mcChc3Igqy06uxthtOzkVUpYN3dvEYL8fEHhsJpHeMwUN83Mse6bKLd7j65RMecBsCXsjV99kKfY6Xlrhx1YU1QdOP5miCT\\\\\\\/JnxdwoJ\\\\\\\/bZuVq+3tqIMcCf2Jtn2NZ2qidV0glydaqPWCMAOaj4nRk\\\\\\\/o+kRDTANj3GJM6bNgo8ewXahmJJU7jWWYDAUJnZVl0mlVvC8WU\\\\\\\/cOKd+1wwbdha48IxXfrEOD1MRhaCsThj1nH4yFOd47Pp\\\\\\\/d0RJ86k7IbaSlkg8hpo8Hscnv9g3caibryG1RzyUvDvPdkMURu1g5TTbf0A92qPxTIFz6gywfObi0nqr8sWHhvXXHM1CJu9Ggz10YQ+BDGCgj9wSokFN\\\\\\\/s2OktpiTQFL41rEZ89zya0jumjwTGv1yvObveT2crWwVfXbcZuNUic3FHdre2jCpJlSPWmcBdV1rmojXr3QSBQmNsk6K+1xmAuVW3a+cRI05FJIXQvXMSDzHkm8Cf3O\\\\\\\/m3KB44vbck23njAlgd9+jpg+wLmL1GtRY5kWaIyGI3OMRZ87ClitR23hvzBfopB8du0nsSUeE3abErsL9aR2lORNMiFKDWHajsu84A5YIFaOWhEhJH77SnoVu3HPLaTLzATEygZ\\\\\\\/JicRgrWsa793YX3tWZHvjpo3mBZ4UzkbShtfyEE5C2R0uVBSAlMNOr9jsbJtbj\\\\\\\/dFX9QMu7zl6zNQDb34d+MFGS7nzIpOrWlmqwQ6Jnknf8AkSIQnCuYCHxmQhK1rJKB2yvtP1KDVBHd80MW1AsSnMnr6sYUczT2ThyggCGJa5hkn50GTw8A1pAZYQXzTZVoO0TKQwO8e5ulGO8TgZ0C3wOOwEXE7uXxI5MinyzJWFOSoJhAg+8OicdyCQvE2b+\\\\\\\/iVkMbbri7XgmjLc3\\\\\\\/ECcDDWwxtgIcYl7M7bR4Z\\\\\\\/dAd+OHjFv6OM3ANtBYngg0Ab0yeaUCWZRs3IIxqVetFyFj9MO6joxb1P56UGaERuv8UvxQf33jPtvlfkievEzVJw9KHoWBxExlX3bIjmB23jJkN90J0lE7ss6JrW4euWjPz0IRH\\\\\\\/UHP0ckheVRFnFK2VnQNY0IA7xSJTBDfjpwFMhLI0papJQLf+Lxs7mVT\\\\\\\/2k\\\\\\\/4dmdyLCYp+1R53nuZ6mSTej6Zw9y\\\\\\\/7QNdDkZYNQG+0MzpLD9eej5lPpr8mTwBQySOg0K8hRqiHybj2HQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00225zKIQWX2cGjqzZnT9XBXrA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ff16ffa686fc270e414a105297e4a9e7c7b6e64a78dd13fe75532dd4003a569\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xCvsnAhGCOXlXJmcNQp6QGDZ8jPOflQiTxG4tVlgSeSeTWsK4OycPdH6TgOPjjQZdwrDTAAOShcwgURdwgB6645+SF8P4obS+DzZGOOmjYzMjzkU1iEiLihqF5lwT68nMurs13CD+J+Tx9L3xijJv\\\\\\\/mmGpwAI0ldlYKd9Da\\\\\\\/+9E6J+OBTxWGhhQQW2GgPl6w7zIqr2D1cY9mBXqRh3wtA\\\\\\\/0hCLqrR4Pex3rYr\\\\\\\/5C+MjR7GAgw8lrFdkIvunAYJBRSxSCwGXg9BS1Q87Q25c9fKMZrEJPWMQJXWw\\\\\\\/b9U7aDKveaDRxMNzBdzhGKYQtxiElbJBDh\\\\\\\/Oc3axZ9CRqtPKeUf1GAe8pldiNVi\\\\\\\/hFu82ju1bYdFDkmr\\\\\\\/RjGOoQutoqcBggbGOYeEzzc1dwCcWTUksZVDpRep86I9Yg4dvCEAdNwJHNU6crU2nsERjTRi\\\\\\\/2E3EaindBapifVaORxQIKCG5OzXuCcHrlTD5nB249EkhEh095GEXoA6WRYsproyNU775nJKKepaYBpULfrB89SLkFPVFblf2mnJOcKJDGri9X9LPcQ6c2xL8vKQ\\\\\\\/bxugnU\\\\\\\/POwIS0AQbEepdJlPNzVDLBLKlWANLtJZxDzBlQvLDgIPoXYLmbNUms2J+xnUoIjfgri5zK3\\\\\\\/IPgRV+jlckEjPzfkuMq4aTAm5083uyye6Astty1qiJCu32lLlB5HnmE8X69jH8QeRHIC5OaLpVy1i+itqI1D\\\\\\\/YWBF1h7OsRAb9lhbeb5jO2QnQOMajqt\\\\\\\/us4nRnsX1ggeJYSuKjuZ2HGSzKKX9wj8D6l1vC9lEKvS1sunblaRxXc\\\\\\\/I3L06bJ6\\\\\\\/RwOOSRRCQRafpHmO4TuNrtqWNPJiDhoEhUvfjyhFeLXYMk6uEpOa0T6RKPTufxwtifU4KDdla+gAHuyeTvz6t8Spvqeh2vuf8p7vO4jn3a+ELRSIdqfOm0w8B28A5XUpelDdVVgA7He14gq7mVi2Xli++rzlOFtLXasMUhuCllsTAK1gSxUG4NC8ZzNOUNIel8mNVF00+iwjYuQHypPKxPxZGMULhPe\\\\\\\/rJn5FZvC\\\\\\\/4NSLs7EJSt3Qk\\\\\\\/k4hyawDi1zlpdeU8DXAs5WnGbESeJRHCikt5cl6sXHEfpRHpUYCaUmSaqgyd8wRqFPRoPkJ9Cvn545djs0Noba\\\\\\\/OJeLhC9\\\\\\\/BG7QyOCV9olapvygempUiGOyqsDtBcxLKClQ+pRAw8rbaNtGmMhaHdv\\\\\\\/XRCyQitsta4J6ey0dR9zUR8md2mQ1f5aa7kiAsED9S8SIeoABYm7YzCvhy6WFEyHNVLmw2KeaRhlhbzlwJsFgBRDGWhjPmLkmOeyC1UeIFfM6H1GEn7mNHOSBhzbdKzVTDN1+cMESH5B\\\\\\\/s7ZQEaTn1lEApRPNINbE9Gyg8JXWYG2rJD4qa2ykqWMlhjfCwNhtU0UyJOJ+2QeJBwloYnANobPm8g4VvSqMpn9E\\\\\\\/A4Vdj1qmAkMvo43m803hmL8buz6Q0YsAmleoswWeaIk4KCBlbx9RuKrddSg0UZucsQnHw+UxX\\\\\\\/N4HVub6JKBzzJRSLrroe8aanWbKlO1v9qyJDmLakByXSrksQNzgo4REcerjjBnIIpAfiikeIfzUNtsr3gUI4MON5Es5THExG70crtrLCdily\\\\\\\/2by33WTNcPebQy7URvLycveFWoD4PJpVU5nRCCfdm5sPLp7RuYtzMAJpxQp9lyi6XWCjEydM+TQlb\\\\\\\/L2Y44vNL3uC91IG7JBsyL4f3OXnZwge2jCoHgjC1DQ3kI\\\\\\\/aOAOIq3Wt4VxdYNbAyRalIR3RNoXQU60p0h76m5DaF\\\\\\\/ruiWEOxGs4+MD\\\\\\\/3w3RfX7B7t53SclOn\\\\\\\/YccV+RdiRny6Fb59rtIYol7mmrlNbLEYrlM7JAT9v+zD6zzJQditQGuTZP68Lvpv83GFdXB3noEkgxeoMA9DKT0DyObf4KBgBGSb0xf+fXYZm\\\\\\\/8QEYRklAQMbns48xlaDQMdFNSToa5OiiVx0K6W\\\\\\\/zaWnO0OZqd0CnznQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mUlhXGuJhUwqGkT417qbMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002227eef414fc617e4119ca8bf0d7e8539d6601a9432111824e8d74ba71bd0b650e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459218 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/123" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pZIuPmPXV1b54xyBSymBenTJZvPp-CZqFHsVlsG_9IM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/123" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rRfef36knLXTgf7lCwgdPw-N4i-ifWSMq9O3-rBcLME" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/1/115" title="Chromosome Y Regulates Survival Following Murine Coxsackievirus B3 Infection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/1/131" title="Many Saccharomyces cerevisiae Cell Wall Protein Encoding Genes Are Coregulated by Mss11, but Cellular Adhesion Phenotypes Appear Only Flo Protein Dependent" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459218" id="node459218" data-pisa="ggg;2/1/123" data-pisa-master="ggg;2/1/123" data-apath="/ggg/2/1/123.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the <em>Macaca fascicularis</em> Major Histocompatibility Complex</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Antoine Blancher</span>, <span class="highwire-citation-author" data-delta="1">Alice Aarnink</span>, <span class="highwire-citation-author" data-delta="2">Nicolas Savy</span>, <span class="highwire-citation-author" data-delta="3">Naoyuki Takahata</span> and <span class="highwire-citation-author" data-delta="4">J. M. Comeron</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">123-130;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001248 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459218" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Antoine Blancher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BBlancher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blancher%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntoine%2BBlancher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:blancher.a@chu-toulouse.fr" class="" data-icon-position="" data-hide-link-title="0">blancher.a@chu-toulouse.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alice Aarnink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alice%2BAarnink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aarnink%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlice%2BAarnink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicolas Savy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut de Mathématiques, Université Paul Sabatier, Toulouse 3, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BSavy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Savy%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BSavy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Naoyuki Takahata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Graduate University for Advanced Studies (Sokendai), Hayama, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoyuki%2BTakahata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takahata%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANaoyuki%2BTakahata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">J. M. Comeron </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BM.%2BComeron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comeron%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BM.%2BComeron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/1/123" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459218" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459218/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/123.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459218" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459218/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/123.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459218" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459218/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/123.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459218" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459218/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/1/123.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459218" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459218/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">We describe a method to estimate the rate of recombination per generation from the genotypes of a large individual sample of an expanding population, for which the founding event is dated. The approach is illustrated with an application to estimating the major histocompatibility complex (MHC) recombination rate in the Mauritian macaque population. We genotyped 750 macaques by means of 17 microsatellites across the MHC region and reconstructed the seven most frequent haplotypes assumed to represent the founding haplotypes (H<sub>rec(0)</sub>) as well as the 31% recombinant haplotypes (H<sub>rec(h)</sub>) resulting from a variable number “h” of recombinations between the founding haplotypes. The relative frequencies of the various classes of haplotypes (H<sub>rec(0)</sub> and H<sub>rec(h)</sub>) follow a Poisson distribution. By using a maximum likelihood method, we calculated the mean of the Poisson distribution that best fits the data. By dividing this mean by the number of generations (50−100) from the date of the population founding, we deduced that rate of recombination in the MHC is approximately 0.004 to 0.008 in the Mauritian macaque population. When the founding date of the population is precisely known, our method presents a useful alternative to the coalescent method.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/poisson-distribution" class="hw-term hw-article-keyword hw-article-keyword-poisson-distribution" rel="nofollow">Poisson distribution</a></li><li class="kwd"><a href="/keyword/recombination-rate" class="hw-term hw-article-keyword hw-article-keyword-recombination-rate" rel="nofollow">recombination rate</a></li><li class="kwd"><a href="/keyword/expanding-population" class="hw-term hw-article-keyword hw-article-keyword-expanding-population" rel="nofollow">expanding population</a></li><li class="kwd"><a href="/keyword/major-histocompatibility-complex" class="hw-term hw-article-keyword hw-article-keyword-major-histocompatibility-complex" rel="nofollow">major histocompatibility complex</a></li><li class="kwd"><a href="/keyword/mhc" class="hw-term hw-article-keyword hw-article-keyword-mhc" rel="nofollow">MHC</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/macaca-fascicularis" class="hw-term hw-article-keyword hw-article-keyword-macaca-fascicularis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Macaca fascicularis</em></a></li></ul><p id="p-4">The major histocompatibility complex (MHC) is one of the most polymorphic regions in human as well as in nonhuman primates and other mammal species studied. This high degree of polymorphism is particularly adapted to the study of recombination in a relatively short region of the genome. The human MHC spans approximately 3.3 Mb on the short arm of chromosome 6 and contains the most polymorphic loci (and, consequently, haplotypes) known in humans. Approximately 40% of the 128 genes in this region participate in immune responses (The MHC Sequencing Consortium, 1999). It is strongly suspected that the long-term persistence of certain MHC haplotypes is favored by their positive selection because of beneficial interactions between specific allelic products of the different loci composing these stable haplotypes during a long period (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beck and Trowsdale 2000</a>).</p><p id="p-5">The recombination events across the MHC have been extensively studied in humans, first by the studies of recombination in families and more recently by high-resolution mapping of recombination events in single sperm (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cullen <em>et al.</em> 2002</a>) and through high-resolution recombination maps inferred from high-density single nucleotide polymorphism data (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Wegmann <em>et al.</em> 2011</a>).</p><p id="p-6">In the present study, we sought to estimate the rate of recombination inside the MHC of cynomolgus monkeys (<em>Macaca fascicularis</em>) originated from the Mauritius Island. The Mauritian macaque population was founded approximately 400 years ago by the introduction of a few animals via Portuguese or Dutch sailors. The strong impact of the founding bottleneck was clearly confirmed by mitochondrial DNA analysis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lawler <em>et al.</em> 1995</a>) as well as by the study of the polymorphism of several nuclear genes. We have estimated that the most probable number of animal founders was around 12 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bonhomme <em>et al.</em> 2008</a>). These animals were most likely captured in Java and/or Sumatra (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kondo <em>et al.</em> 1993</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Tosi and Coke 2007</a>). They were released in the wild on the Mauritius Island, where they reproduced so rapidly that 100 years after their introduction, the macaques have pullulated on the island, forcing the farmers to leave because of the environmental and agricultural damage caused by the macaques (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Sussman and Tattersall, 1981</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">1986</a>). From the historical records, macaques were imported only once to the island. The Mauritian macaque population, then, is an archetype of isolated insular population. It constitutes a model of a population with a strong founding effect, followed by a rapid expansion and a population size plateau of approximately 50,000 animals corresponding to the current estimated population size. There is no record of severe episodes of bottleneck after the founding event.</p><p id="p-7">In a study of a large population sample (N = 750 unrelated animals) genotyped by means of 17 microsatellites across the MHC region, we reconstructed the MHC founding haplotypes in the Mauritian macaque population (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aarnink <em>et al.</em> 2011</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wiseman <em>et al.</em> 2007</a>). The definition of ancestral haplotypes allowed us to detect and characterize recombinant haplotypes. From the frequencies of recombinant haplotypes, we determined a new method to estimate the recombination rate inside the MHC region and compared this rate with that obtained by the coalescent method.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-8">The animals belonged to several Mauritius breeding companies (Le Tamarinier, Noveprim). Some animals (20%) were captured in the wild, whereas the others were F1 animals born in captivity from animals captured in the wild. Animals born in captivity have neither their parent in common (as attested by the records of the breeding companies) and were therefore considered unrelated. The animals bleeding were performed following the ethical rules of animal handling.</p></div><div id="sec-3" class="subsection"><h3>Genomic DNA samples</h3><p id="p-9">Genomic DNA was extracted from peripheral blood sample using either QIA amp Blood Kit (QIAGEN, Courtaboeuf, France) or a classical phenol–chloroform method.</p></div><div id="sec-4" class="subsection"><h3>MHC genotyping</h3><p id="p-10">The MHC genotype was determined with 17 microsatellites scattered across the MHC region; the primers used are described in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001248/-/DC1/001248SI.pdf">supporting information,</a> <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001248/-/DC1/TableS1.pdf">Table S1</a> (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aarnink <em>et al.</em> 2011</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bonhomme <em>et al.</em> 2005</a>, <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Wiseman <em>et al.</em> 2007</a>). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> describes the locations of the microsatellite markers on the genetic map of the cynomolgus macaque MHC. Genotypes were determined with DNA Size Standard-kit- 600 (Beckman Coulter, Villepinte, France) after denaturation and separation of the amplification products by capillary electrophoresis with a CEQ8000 analyzer and scored with the software CEQ8000 Genetic Analysis System v8.0 (Beckman Coulter, Villepinte, France).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/123/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Localization of microsatellite markers on the MHC map of cynomolgus macaque. The regions of class-IA, -IB, and -II genes are shown as boxes (telomere is at the top of the diagram). The markers are positioned by reference to the sequence data (Watanabe et al. 2007; and T. Shiina, personal communication)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-493305875" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Localization of microsatellite markers on the MHC map of cynomolgus macaque. The regions of class-IA, -IB, and -II genes are shown as boxes (telomere is at the top of the diagram). The markers are positioned by reference to the sequence data (Watanabe &lt;em&gt;et al.&lt;/em&gt; 2007; and T. Shiina, personal communication).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/123/F1.medium.gif" width="286" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/123/F1.medium.gif" width="286" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/123/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/123/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461116" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-11" class="first-child">Localization of microsatellite markers on the MHC map of cynomolgus macaque. The regions of class-IA, -IB, and -II genes are shown as boxes (telomere is at the top of the diagram). The markers are positioned by reference to the sequence data (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Watanabe <em>et al.</em> 2007</a>; and T. Shiina, personal communication<span class="sc">).</span></p><div class="sb-div caption-clear"></div></div></div><div id="sec-5" class="subsection"><h4>Haplotype reconstruction:</h4><p id="p-12">First, we deduced the most frequent haplotypes from homozygous animals. The other haplotypes were deduced by the study of heterozygous animals having one of the most frequent haplotypes. In this way, we characterized seven haplotypes, thereafter referred to as the founding haplotypes, similar to those previously described (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mee <em>et al.</em> 2009</a>, <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Wiseman <em>et al.</em> 2007</a>). Thereafter, for each animal, we reconstructed the two most probable haplotypes. This individual haplotyping was carefully checked to define the most parsimonious reconstruction (<em>i.e.</em> the one that requires the minimum number of recombination of the seven ancestral haplotypes). Because the number of alleles detected in the population for each microsatellite is frequently lower than the number of founding haplotypes, the points of recombination are not always precisely defined (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> for the interhaplotype allele sharing). The haplotyping of the 750 animals obtained by this method was confirmed by means of Phase software (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Stephens <em>et al.</em> 2001</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Stephens and Donnelly 2003</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461101/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461101/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Description of the seven MHC founding haplotypes defined in the Mauritian cynomolgus macaque population by means of 17 microsatellites</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-6" class="subsection"><h3>Statistical method</h3><div id="sec-7" class="subsection"><h4>Estimation and confidence interval of the intensity:</h4><p id="p-16">The intensity of a Poisson distribution, which equals the mean of the distribution, is denoted by the Greek letter λ and can be estimated from observed data with a maximum likelihood procedure. It is usual to consider that the maximum likelihood estimation of λ, denoted by <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-1.gif"/></noscript></span></span>, is the mean of the observed values (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Van der Vaart 1998</a>). Moreover, by taking into account the asymptotic normality of the maximum likelihood estimator and applying the Slutsky’s theorem (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Van der Vaart 1998</a>), we obtained the following expression of the confidence interval of λ for a given confidence level α:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-2.gif"/></noscript></span></span>Where <em>u</em><sub>a/2</sub> is the fractile of the standard normal distribution and <em>n</em> the number of observations.</p></div><div id="sec-8" class="subsection"><h4>Quality of fitting:</h4><p id="p-17">To evaluate the quality of fitting between observed data and the Poisson distribution of intensity λ denoted by <em>P</em>(λ), we applied an usual χ<sup>2</sup> test (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dacunha-Castelle and Duflo 1986</a>).</p><p id="p-18">Consider a sample <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-3.gif"/></noscript></span></span>.</p><p id="p-19">The hypotheses are:</p><p id="p-20"><span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-4.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-5.gif"/></noscript></span></span> comes from a <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-6.gif"/></noscript></span></span> distribution,</p><p id="p-21"><span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-7.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-8.gif"/></noscript></span></span> does not come from a <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-9.gif"/></noscript></span></span> distribution.</p><p id="p-22">Consider <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-10.gif"/></noscript></span></span> the expected value under the hypothesis <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-11.gif"/></noscript></span></span>given by<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-12.gif"/></noscript></span></span>where <em>p</em><sub>0</sub> = 0 and <em>p</em><sub>i</sub> is such that for all <em>i</em>, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-13.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-14.gif"/></noscript></span></span>are the corresponding number of observed values, <em>i.e.</em> this means the number of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-15.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-16.gif"/></noscript></span></span>. By merging the bins in this way, we ensure that <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-17.gif"/></noscript></span></span> for every <em>i</em>, a necessary condition for the validity of the χ<sup>2</sup> test.</p><p id="p-23">Denote by <em>p</em> the number of such values. The statistic of the test is given by:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-18.gif"/></noscript></span></span>and the decision is taken by the use of a <em>P</em> value calculated from a χ<sup>2</sup> (<em>p</em> − 2) distribution. The degree of freedom <em>p</em>-2 is the number of bins − 1 minus 1 because we have estimated the parameter of the Poisson distribution.</p></div></div><div id="sec-9" class="subsection"><h3>Estimation of recombination rate by the coalescent method</h3><p id="p-24">To estimate the recombination rate by the coalescent method, we used the <span class="sc">P</span>hase software (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crawford <em>et al.</em> 2004</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li and Stephens 2003</a>). We used the recombination model (-MR0 option), the general model for recombination rate variation in <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Li and Stephens (2003)</a>. As recommended, we used the -X option to obtain more accurate estimates.</p><p id="p-25">For each interval between two consecutive markers, we obtained the value of ρ = 4N<sub>e</sub>c. We calculated, for each interval, the rate of recombination per generation by dividing ρ by 4N<sub>e</sub>. As the recombination rates are lower than 3% between two consecutive markers, we summed all values of “c” to deduce the rate of recombination on the complete MHC region analyzed here.</p><p id="p-26">To estimate the N<sub>e</sub> in the Mauritian macaque population, we applied the classical logistic growth model of Verhulst (equation hereafter) and calculated the harmonic mean of the population size from the founding event to now (50−100 generations depending on the mean generation time, 4−8 years) in the Mauritian macaque population.</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-19.gif"/></noscript></span></span><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-27">Where <em>N</em><sub>t</sub> number of animals at a given generation;</p></li><li id="list-item-2"><p id="p-28"><em>N</em><sub>0</sub> number of animal founders;</p></li><li id="list-item-3"><p id="p-29"><em>K</em> the maximum number of animals;</p></li><li id="list-item-4"><p id="p-30"><em>r</em> the intrinsic growth rate; and</p></li><li id="list-item-5"><p id="p-31"><em>t</em> time in generation.</p></li></ul><p id="p-32">As for <em>K</em>, we have estimated that the maximum number of animals of a given generation was one-quarter of the total population size at a given time. Therefore, with a current total population size of approximately 48,000, we obtain a reasonable estimate of 12,000 individuals in a given generation.</p><p id="p-33">It is noteworthy that the Verhuslt model does not take into account the macaque reproduction peculiarities (preference for dominant male and the partitioning among female lineages) that tend to reduce the effective population in comparison with the real population size.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-34">We studied genotyped 750 animals randomly chosen and born from different breeders. The genotypes were characterized for 18 microsatellites across the MHC region (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001248/-/DC1/FileS1.xls">File S1</a> for the allelic data set from these experiments. One microsatellite (<em>D6S2691</em>) was found instable with frequent mutations (insertion or deletion of one repeated unit) and was therefore discarded from further analyses (data not shown). The mutations of the microsatellites retained for the haplotype analysis were infrequent (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). By the analysis of these genotypes, we reconstructed seven founding haplotypes (as described in <em>Material and Methods</em> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461084/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461084/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">List of rare microsatellite alleles (F &lt; 1%) observed in the sample of 750 Mauritian macaques</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">The frequencies of founding haplotypes, called H1, 2, 3, 4, 5, 6, and H7, are similar to those reported previously by others (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Budde <em>et al.</em> 2010</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Mee <em>et al.</em> 2009</a>, <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Wiseman <em>et al.</em> 2007</a>) and by us (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Aarnink <em>et al.</em> 2011</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/123/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MHC haplotype frequencies in the Mauritian cynomolgus macaque population. (A) The frequencies of the seven most frequent haplotypes (founding haplotypes) and of the recombinant haplotypes in the 750 animals studied here. (B) The frequencies of intact and recombinant MHC haplotypes observed in our sample. The haplotypes are grouped in categories as a function of the number of recombinations they present: intact haplotypes (Hrec0) have no recombination, and recombinant haplotypes (Hrec(h)) can be deduced from founding haplotypes by assuming a variable number (h = 1&#x2212;3) of recombinations" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-493305875" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MHC haplotype frequencies in the Mauritian cynomolgus macaque population. (A) The frequencies of the seven most frequent haplotypes (founding haplotypes) and of the recombinant haplotypes in the 750 animals studied here. (B) The frequencies of intact and recombinant MHC haplotypes observed in our sample. The haplotypes are grouped in categories as a function of the number of recombinations they present: intact haplotypes (H&lt;sub&gt;rec0&lt;/sub&gt;) have no recombination, and recombinant haplotypes (H&lt;sub&gt;rec(h)&lt;/sub&gt;) can be deduced from founding haplotypes by assuming a variable number (h = 1&#x2212;3) of recombinations&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/123/F2.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/123/F2.medium.gif" width="440" height="204"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/123/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/123/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461092" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-36" class="first-child">MHC haplotype frequencies in the Mauritian cynomolgus macaque population. (A) The frequencies of the seven most frequent haplotypes (founding haplotypes) and of the recombinant haplotypes in the 750 animals studied here. (B) The frequencies of intact and recombinant MHC haplotypes observed in our sample. The haplotypes are grouped in categories as a function of the number of recombinations they present: intact haplotypes (H<sub>rec0</sub>) have no recombination, and recombinant haplotypes (H<sub>rec(h)</sub>) can be deduced from founding haplotypes by assuming a variable number (h = 1−3) of recombinations</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">As described in <em>Material and Methods</em>, for each animal, we retained systematically the most parsimonious haplotype reconstruction (the reconstruction that requires the minimum number of recombination of ancestral haplotypes). The haplotypes can be grouped in various categories as a function of the number of recombinations presented: (1) intact haplotypes (H<sub>rec(0)</sub>) have no recombination (<em>i.e.</em> they are identical to founding haplotypes); and (2) recombinant haplotypes (H<sub>rec(h)</sub>) can be deduced from founding haplotypes by assuming a variable number (h) of recombinations. Several examples of recombinant haplotypes are given in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001248/-/DC1/FigureS1.pdf">Figure S1</a>.</p><p id="p-38">We found 187 different types of recombinant haplotypes (114, 61, and 12 types of H<sub>rec1</sub>, H<sub>rec2</sub>, and H<sub>rec3</sub>, respectively). The relative frequencies of the most frequent recombinant haplotypes were lower than the frequency of the haplotype M7, the rarest Mauritian ancestral haplotype. We considered therefore that all recombinant haplotypes observed in our animal sample arose after the founding of the Mauritian population (<em>i.e.</em> that none of the recombinant haplotypes was brought by the founding animals).</p><p id="p-39">As for the single recombinant haplotypes, we have studied the relative frequencies of the haplotypes at both sides of the recombination point. The frequency distributions for the telomeric and centromeric parts of the recombinant haplotypes are very close to the frequency distribution of the nonrecombinant haplotypes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). As for double recombinant haplotypes, the frequency distributions of the telomeric and centromeric parts do not differ from the distribution of founding haplotypes, contrary to the distribution of haplotype frequencies between the two recombination points (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/123/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haplotype frequency distribution observed at the two ends of haplotypes having a single recombination. The frequency distribution of founding haplotypes observed at the telomeric and centromeric ends of the 382 haplotypes having a single recombination is compared with the relative frequency distribution of the ancestral haplotypes in the sample (hatched bars, N = 1030). The frequency distribution at the telomeric end of recombinant haplotypes does not differ from the distribution of founding haplotypes (&#x3C7;2 P = 0.56) whereas the frequency distribution at the centromeric end of recombinant haplotypes differed slightly (although significantly, &#x3C7;2 P = 0.0003)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-493305875" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haplotype frequency distribution observed at the two ends of haplotypes having a single recombination. The frequency distribution of founding haplotypes observed at the telomeric and centromeric ends of the 382 haplotypes having a single recombination is compared with the relative frequency distribution of the ancestral haplotypes in the sample (hatched bars, N = 1030). The frequency distribution at the telomeric end of recombinant haplotypes does not differ from the distribution of founding haplotypes (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; &lt;em&gt;P&lt;/em&gt; = 0.56) whereas the frequency distribution at the centromeric end of recombinant haplotypes differed slightly (although significantly, &#x3C7;&lt;sup&gt;2&lt;/sup&gt; &lt;em&gt;P&lt;/em&gt; = 0.0003).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/123/F3.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/123/F3.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/123/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/123/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461159" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-40" class="first-child">Haplotype frequency distribution observed at the two ends of haplotypes having a single recombination. The frequency distribution of founding haplotypes observed at the telomeric and centromeric ends of the 382 haplotypes having a single recombination is compared with the relative frequency distribution of the ancestral haplotypes in the sample (hatched bars, N = 1030). The frequency distribution at the telomeric end of recombinant haplotypes does not differ from the distribution of founding haplotypes (χ<sup>2</sup> <em>P</em> = 0.56) whereas the frequency distribution at the centromeric end of recombinant haplotypes differed slightly (although significantly, χ<sup>2</sup> <em>P</em> = 0.0003).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/123/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haplotype frequencies observed in the three parts of haplotypes presenting two recombinations. The white, gray, and black bars represent, respectively, the frequencies of haplotypes in the &#x201C;telomeric,&#x201D; &#x201C;central,&#x201D; and &#x201C;centromeric&#x201D; regions of 74 recombinant haplotypes presenting two recombinations. The hatched bars correspond to the frequencies of the ancestral haplotypes (N = 1030). The frequency distributions of the telomeric and centromeric ends of double recombinant haplotypes are similar to those of intact MHC haplotypes (&#x3C7;2 P = 0.56 and P = 0.39, respectively) whereas the frequency distribution in the &#x201C;central&#x201D; region differ significantly from the distribution of the founding haplotypes (&#x3C7;2 P = 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-493305875" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haplotype frequencies observed in the three parts of haplotypes presenting two recombinations. The white, gray, and black bars represent, respectively, the frequencies of haplotypes in the &#x201C;telomeric,&#x201D; &#x201C;central,&#x201D; and &#x201C;centromeric&#x201D; regions of 74 recombinant haplotypes presenting two recombinations. The hatched bars correspond to the frequencies of the ancestral haplotypes (N = 1030). The frequency distributions of the telomeric and centromeric ends of double recombinant haplotypes are similar to those of intact MHC haplotypes (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; &lt;em&gt;P&lt;/em&gt; = 0.56 and &lt;em&gt;P&lt;/em&gt; = 0.39, respectively) whereas the frequency distribution in the &#x201C;central&#x201D; region differ significantly from the distribution of the founding haplotypes (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; &lt;em&gt;P&lt;/em&gt; = 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/123/F4.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/123/F4.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/123/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/123/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461167" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-41" class="first-child">Haplotype frequencies observed in the three parts of haplotypes presenting two recombinations. The white, gray, and black bars represent, respectively, the frequencies of haplotypes in the “telomeric,” “central,” and “centromeric” regions of 74 recombinant haplotypes presenting two recombinations. The hatched bars correspond to the frequencies of the ancestral haplotypes (N = 1030). The frequency distributions of the telomeric and centromeric ends of double recombinant haplotypes are similar to those of intact MHC haplotypes (χ<sup>2</sup> <em>P</em> = 0.56 and <em>P</em> = 0.39, respectively) whereas the frequency distribution in the “central” region differ significantly from the distribution of the founding haplotypes (χ<sup>2</sup> <em>P</em> = 0.001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Assuming that recombinations are rare events, we hypothesized that their accumulation in the Mauritian population was compatible with the following assumptions:</p><ul class="list-unord " id="list-2"><li id="list-item-6"><p id="p-43">the probability of recombination is constant from the founding event up to the current time;</p></li><li id="list-item-7"><p id="p-44">the probability of a recombination is not influenced by a previous recombination events; and</p></li><li id="list-item-8"><p id="p-45">the probability for a given haplotype to have not been transmitted to the current population does not depend on the number of recombinations experienced by this haplotype (<em>i.e.</em> the recombinations are strictly neutral).</p></li></ul><p id="p-46">Moreover, we assume that recombination identified within a given animal resulted from an event independent of those resulting in identical recombinant haplotypes observed in other animals. This assumption is not unlikely because of the large growth of the macaque population during the past 400 years from a very small founder population, which reduces the amount of shared ancestry concerning the recombinant haplotypes. Under all these assumptions, one can deduce that the numbers (or frequencies) of haplotypes presenting no or a given number of recombinations follow a Poisson distribution.</p><p id="p-47">To check this latter proposal, we compared the observed relative frequencies distribution H<sub>rec(0,1,2,3,h)</sub> to those deduced from a Poisson distribution with an intensity estimated from the observed data by a maximum likelihood procedure (see <em>Materials and Methods</em> for details). The intensity of the Poisson distribution deduced from the observed frequencies H<sub>rec(0,1,2,3,4)</sub> was 0.381, and with a risk of 5%, we can say that the intensity was comprised between 0.351 and 0.413 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Moreover, the statistical comparison by the χ<sup>2</sup> test allows us to say there is no significant difference between the distribution of the observations and the values calculated from a Poisson <em>P</em>(0.381) distribution (the calculated value is <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1/123/embed/mml-math-20.gif"/></noscript></span></span> and the associated <em>P</em>-value <em>P</em> = 0.488). Therefore, assuming that the distribution follows the optimized Poisson distribution, the intensity of this distribution represents the frequency of recombination during the period spanning the founding of the Mauritian macaque population to the current time, which can be estimated to 0.381 (± 0.031). If we assume that this period is approximately 400 years and that the mean time generation of <em>Macaca fascicularis</em> is between 4 and 8 years, the frequency of recombination per generation is between 0.0038 (± 0.00031) and 0.0076 (± 0.00062).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461165/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461165/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Recombinant haplotype numbers observed in the population sample and corresponding numbers expected from the optimized Poisson distribution</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">In fact, this frequency corresponds to the recombinants detectable by our method. The recombinants events that have occurred in homozygous animals are obviously not detected and are excluded from the deduced frequency. Because of the low number of haplotypes in the population, the frequencies of homozygous animals are not low. In the current population, this frequency is around 10%. We can reasonably estimate that the mean frequency of homozygous animals was relatively stable during the history of the Mauritian macaque population. The frequency of all recombinations in the MHC (detectable or not) can be deduced from the mean frequency of heterozygotes (0.90). The frequency of all recombination events becomes 0.0041 to 0.0082 in function of the mean time generation taken into account (4 and 8 years, respectively).</p><p id="p-52">We decided to estimate the recombination rate by one of the classic coalescent methods. We chose to use the <span class="sc">P</span>hase software, which provided the recombination parameter rho = 4N<sub>e</sub>c. To determine the rate of recombination “c” by means of the coalescent method, N<sub>e</sub> is required. We calculated the N<sub>e</sub> as the harmonic mean of the effective population size from the time of population founding up to nowadays (<em>i.e.</em> 50−100 generations for a mean generation time of 4 or 8 years) under the assumption of a classical model of logistic growth (Verhulst model). Following these calculations, the range of variation of N<sub>e</sub> estimate is very large (from 63 to 3252) and varies as a function of the demographical parameters: the mean generation time (4 or 8 years), the R<sub>0</sub> (0.1 to 0.4) and N<sub>0</sub> (10 to 90; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Considering the two extreme estimates of Ne (63 and 3252) and the recombination parameter obtained by means of the <span class="sc">P</span>hase software, the rate of recombination in the complete MHC region varies from 0.0027 (N<sub>e</sub> = 3252) to 0.1365 (N<sub>e</sub> = 63). The N<sub>e</sub> that give the recombination rates identical to the estimates obtained by means of the optimized Poisson distribution (0.0041−0.0082) are 1743 or 871, respectively.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/123/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation of Ne in function of R0 and N0 under classical logistic growth model of Verhulst. From the time of the macaque population founding up to nowadays (i.e. 50&#x2212;100 generations), the average of Ne corresponds to the harmonic mean of the effective population size which was calculated under the assumption of a classical model of logistic growth (see Materials and Methods for more details). The figure shows the average Ne as a function of R0 (0.1&#x2212;0.4) and N0 (10&#x2212;90) with a constant maximum Ne population size (K = 12,000). With a mean generation time of 4 years, the average Ne varies from 125 to 3252, as shown in the figure. Under the hypothesis of a generation time of 8 years, the Ne varies from 63 to 1880 (figure not shown)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-493305875" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation of N&lt;sub&gt;e&lt;/sub&gt; in function of R&lt;sub&gt;0&lt;/sub&gt; and N&lt;sub&gt;0&lt;/sub&gt; under classical logistic growth model of Verhulst. From the time of the macaque population founding up to nowadays (&lt;em&gt;i.e.&lt;/em&gt; 50&#x2212;100 generations), the average of N&lt;sub&gt;e&lt;/sub&gt; corresponds to the harmonic mean of the effective population size which was calculated under the assumption of a classical model of logistic growth (see &lt;em&gt;Materials and Methods&lt;/em&gt; for more details). The figure shows the average N&lt;sub&gt;e&lt;/sub&gt; as a function of R&lt;sub&gt;0&lt;/sub&gt; (0.1&#x2212;0.4) and N&lt;sub&gt;0&lt;/sub&gt; (10&#x2212;90) with a constant maximum N&lt;sub&gt;e&lt;/sub&gt; population size (K = 12,000). With a mean generation time of 4 years, the average N&lt;sub&gt;e&lt;/sub&gt; varies from 125 to 3252, as shown in the figure. Under the hypothesis of a generation time of 8 years, the Ne varies from 63 to 1880 (figure not shown).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/123/F5.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/123/F5.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/123/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/123/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461121" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-53" class="first-child">Variation of N<sub>e</sub> in function of R<sub>0</sub> and N<sub>0</sub> under classical logistic growth model of Verhulst. From the time of the macaque population founding up to nowadays (<em>i.e.</em> 50−100 generations), the average of N<sub>e</sub> corresponds to the harmonic mean of the effective population size which was calculated under the assumption of a classical model of logistic growth (see <em>Materials and Methods</em> for more details). The figure shows the average N<sub>e</sub> as a function of R<sub>0</sub> (0.1−0.4) and N<sub>0</sub> (10−90) with a constant maximum N<sub>e</sub> population size (K = 12,000). With a mean generation time of 4 years, the average N<sub>e</sub> varies from 125 to 3252, as shown in the figure. Under the hypothesis of a generation time of 8 years, the Ne varies from 63 to 1880 (figure not shown).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-54">By using the distribution of the frequencies of founding and recombinant haplotypes, we estimated the recombination rate per generation in the macaque MHC to be between 0.41% and 0.82%. The rate of recombination deduced by the Poisson distribution of recombinant haplotypes is not incompatible with the very low frequency of recombination observed in pedigree of Mauritian animals bred in captivity. Among 210 informative meioses, only one recombination (0.48%) was observed (E. Mee, personal communication).</p><p id="p-55">The distribution of haplotype frequencies at both sides of recombination point in single recombinant haplotypes is very close to the intact haplotype frequency distribution. This finding suggests that the recombinations were stochastically accumulated along generations. In the central portion of double recombinants, the frequency distribution differs significantly of that of founding haplotypes. This result suggests that the frequency of haplotypes could have varied in the past.</p><p id="p-56">The recombination rate in the macaque MHC seems to be significantly lower that the recombination frequency in the human MHC (human leukocyte antigen), estimated at approximately 3% between HLA-A and HLA-DQ (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cullen <em>et al.</em> 2002</a>). Despite this, the macaque MHC differs significantly from that of human by the presence of numerous MHC class I genes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Daza-Vamenta <em>et al.</em> 2004</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Shiina <em>et al.</em> 2006</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Watanabe <em>et al.</em> 2007</a>) resulting from multiple duplications (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kulski <em>et al.</em> 2004</a>), the rate of recombination in macaque MHC is significantly lower than in humans.</p><p id="p-57">In recent studies researchers have revealed that in humans, the rate of recombination in the MHC varies greatly from one individual to another (from 0.8% to 4%) depending on its genotype (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cullen <em>et al.</em> 2002</a>). Therefore, it is not impossible that in macaque populations other than that of Mauritius, the mean rate of recombination could differ because of the presence of haplotypes favoring greater rates of recombination. Moreover, our study did not reveal a hot spot of recombination contrary to what was reported in humans. Again this is not incompatible with the low rate of recombination observed in the Mauritian population.</p><p id="p-58">The estimate of the recombination rate deduced from the intensity of the Poisson distribution has a relatively narrow confidence interval. The precision of this estimate depends on the quality of the measure of recombinant haplotype frequencies, and on the incertitude of the mean generation time and of the time elapsed since population founding. As for the mean time of generation, we have taken a relatively large interval (between 4 and 8 years). As for the macaque population founding, most specialists, from historical records, consider that macaques were imported a few decades after the discovery of the Mauritius Island by the Portuguese between 1507 and 1513.</p><p id="p-59">Because the Portuguese were already established in Goa, they took no interest in the isolated Mauritius and did not create a permanent colony there. The island was later rediscovered by Dutch navigators. One of them, Cornelius Matelieff de Jongh, in 1606 was the first to record the presence of monkeys on Mauritius. He wrote: “the only four-footed animals are monkeys” (in <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Sussman and Tattersall, 1986</a>). At the time of this observation, one can assume that the macaques on the island were abundant enough to retain de Jongh’s attention. The establishment on Mauritius of the monkey population, then, very likely preceded the founding, in 1598, of the tiny Dutch settlement. It is highly probable that the monkeys were introduced by the Dutch or Portuguese sailors at the end of 16th century. Therefore, the time elapsed from the macaque population founding is approximately 400 years. Although various scenarios of macaque population size evolution could be evoked (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bonhomme <em>et al.</em> 2008</a>), the simplest one consists in a logistic growth of the population after the founding event characterized by a rapid increase of the population size followed by a plateau of the population size corresponding to the limits of available food on the island. In this model, we must consider that the recombinant haplotypes have accumulated randomly following a rare event law during the time elapsed after the macaques were introduced to the island.</p><p id="p-60">Under these assumptions, we estimated the effective size of the Mauritian population and used it to deduce the rate of recombination from the recombination parameter rho obtained by means of the coalescent method (Phase software). This study reveals a limitation of the coalescent method to estimate the rate of recombination in a population having experienced a rapid growth after a severe founding bottleneck. Indeed, under a logistic Verhulst model, the N<sub>e</sub> estimate (the harmonic mean of the population size) varies widely (from 63 to 3252) a great incertitude for the recombination rate estimate (from 0.0027 to 0.1365). Note that the interval of recombination rate given by the “Poisson method” is included in the range of the estimate deduced from the coalescent method.</p><p id="p-61">In the comparison between the recombination rate obtained by our method and that obtained by means of Phase, it is important to note that the rho estimate obtained by means of Phase (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Li and Stephens 2003</a>) includes “hidden” recombination events that occur when, going backward in time, two lineages recombine and then immediately coalesce with each other. These “hidden” events are by definition not taken into account in our method.</p><p id="p-62">However, the estimate of the recombination rate deduced from the intensity of the Poisson distribution does not take into account the identity by descent of recombinant haplotypes observed in more than one animal in the population sample.</p><p id="p-63">Finally, the new “Poisson method” described here gives a more accurate estimate of the rate of recombination per generation than the coalescent method which requires to estimate also the N<sub>e</sub>.</p><p id="p-64">This method can be applied in populations that have experienced a strong expansion after a severe bottleneck, provided that the time of the founding event and the average time of generation are known. This method is an alternative to the coalescent method when the founding date of the population is precisely known.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">We thank Naruya Saitou and Jaume Bertranpetit for very helpful discussions. We are pleased to thank Béatrice Atlan, Audrey Dauba, Stéphanie Despiau-Schiavinato, and Sylvie Hébrard for their excellent technical assistance.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001248/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001248/-/DC1</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-09-19"><span class="received-label">Received </span>September 19, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-11-20"><span class="accepted-label">Accepted </span>November 20, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Blancher <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aarnink</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dereuddre-Bosquet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaslin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Grand</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winterton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Influence of the MHC genotype on the progression of experimental SIV infection in the Mauritian cynomolgus macaque.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DImmunogenetics%26rft.aulast%253DAarnink%26rft.auinit1%253DA.%26rft.volume%253D63%26rft.issue%253D5%26rft.spage%253D267%26rft.epage%253D274%26rft.atitle%253DInfluence%2Bof%2Bthe%2BMHC%2Bgenotype%2Bon%2Bthe%2Bprogression%2Bof%2Bexperimental%2BSIV%2Binfection%2Bin%2Bthe%2BMauritian%2Bcynomolgus%2Bmacaque.%26rft_id%253Dinfo%253Apmid%252F21234560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21234560&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.2" data-doi="10.1146/annurev.genom.1.1.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trowsdale</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The human major histocompatibility complex: lessons from the DNA sequence.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DBeck%26rft.auinit1%253DS.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D117%26rft.epage%253D137%26rft.atitle%253DTHE%2BHUMAN%2BMAJOR%2BHISTOCOMPATIBILITY%2BCOMPLEX%253A%2BLessons%2Bfrom%2Bthe%2BDNA%2BSequence%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.1.1.117%26rft_id%253Dinfo%253Apmid%252F11701627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.1.1.117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11701627&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165768900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.3" data-doi="10.1002/ajp.20193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blancher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crouau-Roy</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Multiplexed microsatellites for rapid identification and characterization of individuals and populations of Cercopithecidae.</span> <abbr class="cit-jnl-abbrev">Am. J. Primatol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPrimatology%26rft.stitle%253DAmerican%2BJournal%2Bof%2BPrimatology%26rft.aulast%253DBonhomme%26rft.auinit1%253DM.%26rft.volume%253D67%26rft.issue%253D3%26rft.spage%253D385%26rft.epage%253D391%26rft.atitle%253DMultiplexed%2Bmicrosatellites%2Bfor%2Brapid%2Bidentification%2Band%2Bcharacterization%2Bof%2Bindividuals%2Band%2Bpopulations%2Bof%2BCercopithecidae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajp.20193%26rft_id%253Dinfo%253Apmid%252F16287102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ajp.20193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16287102&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233754700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.4" data-doi="10.1111/j.1365-294X.2007.03645.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blancher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuartero</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikhi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crouau-Roy</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Origin and number of founders in an introduced insular primate: estimation from nuclear genetic data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBonhomme%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D1009%26rft.epage%253D1019%26rft.atitle%253DOrigin%2Band%2Bnumber%2Bof%2Bfounders%2Bin%2Ban%2Bintroduced%2Binsular%2Bprimate%253A%2Bestimation%2Bfrom%2Bnuclear%2Bgenetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03645.x%26rft_id%253Dinfo%253Apmid%252F18261045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03645.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18261045&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252900200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.5" data-doi="10.1007/s00251-010-0481-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Budde</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiseman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karl</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanczaruk</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simen</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Characterization of Mauritian cynomolgus macaque major histocompatibility complex class I haplotypes by high-resolution pyrosequencing.</span> <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DImmunogenetics%26rft.aulast%253DBudde%26rft.auinit1%253DM.%2BL.%26rft.volume%253D62%26rft.issue%253D11-12%26rft.spage%253D773%26rft.epage%253D780%26rft.atitle%253DCharacterization%2Bof%2BMauritian%2Bcynomolgus%2Bmacaque%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bclass%2BI%2Bhaplotypes%2Bby%2Bhigh-resolution%2Bpyrosequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-010-0481-9%26rft_id%253Dinfo%253Apmid%252F20882385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00251-010-0481-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20882385&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.6" data-doi="10.1038/ng1376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhangale</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellenthal</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evidence for substantial fine-scale variation in recombination rates across the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCrawford%26rft.auinit1%253DD.%2BC.%26rft.volume%253D36%26rft.issue%253D7%26rft.spage%253D700%26rft.epage%253D706%26rft.atitle%253DEvidence%2Bfor%2Bsubstantial%2Bfine-scale%2Bvariation%2Bin%2Brecombination%2Brates%2Bacross%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1376%26rft_id%253Dinfo%253Apmid%252F15184900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15184900&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222354100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.7" data-doi="10.1086/342973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perfetto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klitz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrington</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">High-resolution patterns of meiotic recombination across the human major histocompatibility complex.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DCullen%26rft.auinit1%253DM.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D776%26rft.atitle%253DHigh-resolution%2Bpatterns%2Bof%2Bmeiotic%2Brecombination%2Bacross%2Bthe%2Bhuman%2Bmajor%2Bhistocompatibility%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F342973%26rft_id%253Dinfo%253Apmid%252F12297984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/342973&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12297984&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178613800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.123.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dacunha-Castelle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duflo</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-source">Probability and Statistics</span>, <span class="cit-vol">Vol. I</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.9" data-doi="10.1101/gr.2134504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daza-Vamenta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glusman</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geraghty</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic divergence of the rhesus macaque major histocompatibility complex.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1501</span>–<span class="cit-lpage">1515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDaza-Vamenta%26rft.auinit1%253DR.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D1501%26rft.epage%253D1515%26rft.atitle%253DGenetic%2BDivergence%2Bof%2Bthe%2BRhesus%2BMacaque%2BMajor%2BHistocompatibility%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2134504%26rft_id%253Dinfo%253Apmid%252F15289473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzgvMTUwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMS8xMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.10" data-doi="10.1002/ajp.1350290303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nozawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsubayashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Population genetics of crab-eating macaques (<em>Macaca fascicularis</em>) on the island of Mauritius.</span> <abbr class="cit-jnl-abbrev">Am. J. Primatol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPrimatol.%26rft.volume%253D29%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajp.1350290303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ajp.1350290303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993KP17500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.11" data-doi="10.1093/molbev/msh216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulski</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anzai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoko</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Rhesus macaque class I duplicon structures, organization, and evolution within the alpha block of the major histocompatibility complex.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2079</span>–<span class="cit-lpage">2091</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh216%26rft_id%253Dinfo%253Apmid%252F15269276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMS8yMDc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8xLzEyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.12" data-doi="10.1002/ajpa.1330960203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawler</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sussman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Mitochondrial DNA of the Mauritian macaques (<em>Macaca fascicularis</em>): an example of the founder effect.</span> <abbr class="cit-jnl-abbrev">Am. J. Phys. Anthropol.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BPhysical%2BAnthropology%26rft.stitle%253DAmerican%2BJournal%2Bof%2BPhysical%2BAnthropology%26rft.aulast%253DLawler%26rft.auinit1%253DS.%2BH.%26rft.volume%253D96%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D141%26rft.atitle%253DMitochondrial%2BDNA%2Bof%2Bthe%2BMauritian%2Bmacaques%2B%2528Macaca%2Bfascicularis%2529%253A%2Ban%2Bexample%2Bof%2Bthe%2Bfounder%2Beffect.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajpa.1330960203%26rft_id%253Dinfo%253Apmid%252F7755104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ajpa.1330960203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7755104&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QJ94400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2213</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D165%26rft.spage%253D2213%26rft_id%253Dinfo%253Apmid%252F14704198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14704198&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187805300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mee</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badhan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karl</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiseman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MHC haplotype frequencies in a UK breeding colony of Mauritian cynomolgus macaques mirror those found in a distinct population from the same geographic origin.</span> <abbr class="cit-jnl-abbrev">J. Med. Primatol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BPrimatol.%26rft.volume%253D38%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.15" data-doi="10.1534/genetics.106.057034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ota</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katsuyama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Rapid evolution of major histocompatibility complex class I genes in primates generates new disease alleles in humans via hitchhiking diversity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1555</span>–<span class="cit-lpage">1570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D173%26rft.spage%253D1555%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.057034%26rft_id%253Dinfo%253Apmid%252F16702430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.057034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16702430&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239629400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.16" data-doi="10.1086/319501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new statistical method for haplotype reconstruction from population data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D978%26rft.epage%253D989%26rft.atitle%253DA%2Bnew%2Bstatistical%2Bmethod%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F319501%26rft_id%253Dinfo%253Apmid%252F11254454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/319501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11254454&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167666000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.17" data-doi="10.1086/379378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparison of Bayesian methods for haplotype reconstruction from population genotype data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D73%26rft.issue%253D5%26rft.spage%253D1162%26rft.epage%253D1169%26rft.atitle%253DA%2Bcomparison%2Bof%2Bbayesian%2Bmethods%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F379378%26rft_id%253Dinfo%253Apmid%252F14574645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/379378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574645&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186493400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.18" data-doi="10.1007/BF02382610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sussman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tattersall</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Behavior and ecology of Macaca fascicularis in Mauritius: a preliminary study.</span> <abbr class="cit-jnl-abbrev">Primates</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPrimates%26rft.volume%253D22%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02382610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02382610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.19" data-doi="10.1159/000156234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sussman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tattersall</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Distribution, abundance, and putative ecological strategy of <em>Macaca fascicularis</em> on the island of Mauritius, Southwestern Indian Ocean.</span> <abbr class="cit-jnl-abbrev">Folia Primatol. (Basel)</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFolia%2BPrimatol.%2B%2528Basel%2529%26rft.volume%253D46%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000156234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000156234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986C876700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.20" data-doi="10.1038/44853"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The MHC Sequencing Consortium</span>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Complete sequence and gene map of a human major histocompatibility complex.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">401</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D401%26rft.issue%253D6756%26rft.spage%253D921%26rft.epage%253D923%26rft.atitle%253DComplete%2Bsequence%2Band%2Bgene%2Bmap%2Bof%2Ba%2Bhuman%2Bmajor%2Bhistocompatibility%2Bcomplex.%2BThe%2BMHC%2Bsequencing%2Bconsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F44853%26rft_id%253Dinfo%253Apmid%252F10553908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/44853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10553908&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083464700060&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.1.123.21" data-doi="10.1016/j.ympev.2006.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tosi</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coke</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparative phylogenetics offer new insights into the biogeographic history of <em>Macaca fascicularis</em> and the origin of the Mauritian macaques.</span> <abbr class="cit-jnl-abbrev">Mol. Phyl. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DTosi%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D498%26rft.epage%253D504%26rft.atitle%253DComparative%2Bphylogenetics%2Boffer%2Bnew%2Binsights%2Binto%2Bthe%2Bbiogeographic%2Bhistory%2Bof%2BMacaca%2Bfascicularis%2Band%2Bthe%2Borigin%2Bof%2Bthe%2BMauritian%2Bmacaques.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2006.08.002%26rft_id%253Dinfo%253Apmid%252F16979912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2006.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16979912&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244157400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-2.1.123.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Vaart</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Asymptotic Statistics</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.23" data-doi="10.1016/j.ygeno.2006.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosomichi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanagiya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A BAC-based contig map of the cynomolgus macaque (<em>Macaca fascicularis</em>) major histocompatibility complex genomic region.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">402</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DWatanabe%26rft.auinit1%253DA.%26rft.volume%253D89%26rft.issue%253D3%26rft.spage%253D402%26rft.epage%253D412%26rft.atitle%253DA%2BBAC-based%2Bcontig%2Bmap%2Bof%2Bthe%2Bcynomolgus%2Bmacaque%2B%2528Macaca%2Bfascicularis%2529%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bgenomic%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2006.11.002%26rft_id%253Dinfo%253Apmid%252F17174065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2006.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17174065&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.24" data-doi="10.1038/ng.894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veeramah</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathias</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolae</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination rates in admixed individuals identified by ancestry-based inference.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWegmann%26rft.auinit1%253DD.%26rft.volume%253D43%26rft.issue%253D9%26rft.spage%253D847%26rft.epage%253D853%26rft.atitle%253DRecombination%2Brates%2Bin%2Badmixed%2Bindividuals%2Bidentified%2Bby%2Bancestry-based%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.894%26rft_id%253Dinfo%253Apmid%252F21775992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.894&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775992&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F123.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.123.25" data-doi="10.1128/JVI.01841-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiseman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wojcechowskyj</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasky</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Simian immunodeficiency virus SIVmac239 infection of major histocompatibility complex-identical cynomolgus macaques from Mauritius.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01841-06%26rft_id%253Dinfo%253Apmid%252F17035320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgxLzEvMzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8xLzEyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/1/123.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/1/115" title="Chromosome Y Regulates Survival Following Murine Coxsackievirus B3 Infection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/1/131" title="Many Saccharomyces cerevisiae Cell Wall Protein Encoding Genes Are Coregulated by Mss11, but Cellular Adhesion Phenotypes Appear Only Flo Protein Dependent" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/1"><span>Volume 2 Issue 1, January 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/1.cover.gif?itok=D5RR8siB" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/123" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WawetZamWbkjVvn-icePR5jXv5Z_JJPyY0BWdj5vNLI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/1/123" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/1/123" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/1/123" class="active" data-icon-position="" data-hide-link-title="0">Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the Macaca fascicularis Major Histocompatibility Complex</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459218" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yOz-ZMEeBEPl_SusTouMTsDOKXUkNazfq0pgfz8upL0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/1/123.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/1/123" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-E57tTujlQDz_xdFNA-czA_3ZmDvl7zh5giv2K1ud2F8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/1/123" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/1/123.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459218" id="node459218--2" data-pisa="ggg;2/1/123" data-pisa-master="ggg;2/1/123" data-apath="/ggg/2/1/123.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the <em>Macaca fascicularis</em> Major Histocompatibility Complex</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Antoine Blancher</span>, <span class="highwire-citation-author" data-delta="1">Alice Aarnink</span>, <span class="highwire-citation-author" data-delta="2">Nicolas Savy</span>, <span class="highwire-citation-author" data-delta="3">Naoyuki Takahata</span> and <span class="highwire-citation-author" data-delta="4">J. M. Comeron</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">123-130;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001248 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459218--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Antoine Blancher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BBlancher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blancher%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntoine%2BBlancher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:blancher.a@chu-toulouse.fr" class="" data-icon-position="" data-hide-link-title="0">blancher.a@chu-toulouse.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alice Aarnink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alice%2BAarnink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aarnink%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlice%2BAarnink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicolas Savy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut de Mathématiques, Université Paul Sabatier, Toulouse 3, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BSavy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Savy%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BSavy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Naoyuki Takahata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Graduate University for Advanced Studies (Sokendai), Hayama, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoyuki%2BTakahata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takahata%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANaoyuki%2BTakahata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">J. M. Comeron </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BM.%2BComeron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comeron%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BM.%2BComeron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/1/123" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/1/123&amp;title=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/1/123&amp;title=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/1/123&amp;title=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/1/123&amp;text=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/1/123&amp;title=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/1/123&amp;t=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/1/123&amp;title=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/1/123&amp;title=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459218/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459218" id="node459218--3" data-pisa="ggg;2/1/123" data-pisa-master="ggg;2/1/123" data-apath="/ggg/2/1/123.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Use of Cumulative Poisson Probability Distribution as an Estimator of the Recombination Rate in an Expanding Population: Example of the <em>Macaca fascicularis</em> Major Histocompatibility Complex</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Antoine Blancher</span>, <span class="highwire-citation-author" data-delta="1">Alice Aarnink</span>, <span class="highwire-citation-author" data-delta="2">Nicolas Savy</span>, <span class="highwire-citation-author" data-delta="3">Naoyuki Takahata</span> and <span class="highwire-citation-author" data-delta="4">J. M. Comeron</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">123-130;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001248 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459218--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Antoine Blancher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antoine%2BBlancher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blancher%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntoine%2BBlancher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:blancher.a@chu-toulouse.fr" class="" data-icon-position="" data-hide-link-title="0">blancher.a@chu-toulouse.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alice Aarnink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire d'Immunogénétique Moléculaire, EA 3034, Faculté de Médecine Purpan, Université Paul Sabatier, Toulouse 3, CHU de Toulouse, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alice%2BAarnink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aarnink%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlice%2BAarnink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicolas Savy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut de Mathématiques, Université Paul Sabatier, Toulouse 3, 31059 Toulouse cedex 9, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BSavy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Savy%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BSavy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Naoyuki Takahata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Graduate University for Advanced Studies (Sokendai), Hayama, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoyuki%2BTakahata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takahata%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANaoyuki%2BTakahata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">J. M. Comeron </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BM.%2BComeron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comeron%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BM.%2BComeron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=123&amp;rft.epage=130&amp;rft.atitle=Use%20of%20Cumulative%20Poisson%20Probability%20Distribution%20as%20an%20Estimator%20of%20the%20Recombination%20Rate%20in%20an%20Expanding%20Population%3A%20Example%20of%20the%20Macaca%20fascicularis%20Major%20Histocompatibility%20Complex&amp;rft.volume=2&amp;rft.issue=1&amp;rft.date=2012-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Blancher%2C+Antoine&amp;rft.au=Aarnink%2C+Alice&amp;rft.au=Savy%2C+Nicolas&amp;rft.au=Takahata%2C+Naoyuki"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459218/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459218/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459218/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459218/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459218/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459218/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459218/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459218/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459218/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459218/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459218/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459218/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496051" id="node496051" data-pisa="ggg;g3.120.400867v1" data-pisa-master="ggg;g3.120.400867" data-apath="/ggg/early/2020/03/26/g3.120.400867.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.120.400867" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Comparative Cytology of Female Meiosis 1 Among <em>Drosophila</em> Species</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496050" id="node496050" data-pisa="ggg;g3.120.401055v1" data-pisa-master="ggg;g3.120.401055" data-apath="/ggg/early/2020/03/26/g3.120.401055.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/26/g3.120.401055" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Anchor Away - A Fast, Reliable and Reversible Technique To Inhibit Proteins in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/1/123" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/1/123.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/1/123.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/1/123.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/1/123.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
