<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/1/143" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/1/143.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/1/143.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/1/143.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/1/143.atom" />
<meta name="HW.pisa" content="ggg;2/1/143" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse" />
<meta name="DC.Identifier" content="10.1534/g3.111.001669" />
<meta name="DC.Date" content="2012-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Sun et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Forward genetic screens in Mus musculus have proved powerfully informative by revealing unsuspected mechanisms governing basic biological processes. This approach uses potent chemical mutagens, such as N -ethyl- N -nitrosourea (ENU), to randomly induce mutations in mice, which are then bred and phenotypically screened to identify lines that disrupt a specific biological process of interest. Although identifying a mutation using the rich resources of mouse genetics is straightforward, it is unfortunately neither fast nor cheap. Here we show that detecting newly induced causal variants in a forward genetic screen can be accelerated dramatically using a methodology that combines multiplex chromosome-specific exome capture, next-generation sequencing, rapid mapping, sequence annotation, and variation filtering. The key innovation of our method is multiplex capture and sequence that allows the simultaneous survey of both mutant, parental, and background strains in a single experiment. By comparing variants identified in mutant offspring with those found in dbSNP, the unmutagenized background strains, and parental lines, induced causative mutations can be distinguished immediately from preexisting variation or experimental artifact. Here we demonstrate this approach to find the causative mutations induced in four novel ENU lines identified from a recent ENU screen. In all four cases, after applying our method, we found six or fewer putative mutations (and sometimes only a single one). Determining the causative variant was then easily achieved through standard segregation approaches. We have developed this process into a community resource that will speed up individual labs’ ability to identify the genetic lesion in mutant mouse lines; all of our reagents and software tools are open source and available to the broader scientific community." />
<meta name="DC.Contributor" content="Miao Sun" />
<meta name="DC.Contributor" content="Kajari Mondal" />
<meta name="DC.Contributor" content="Viren Patel" />
<meta name="DC.Contributor" content="Vanessa L. Horner" />
<meta name="DC.Contributor" content="Alyssa B. Long" />
<meta name="DC.Contributor" content="David J. Cutler" />
<meta name="DC.Contributor" content="Tamara Caspary" />
<meta name="DC.Contributor" content="Michael E. Zwick" />
<meta name="DC.Contributor" content="B. J. Andrews" />
<meta name="article:published_time" content="2012-01-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Forward genetic screens in &lt;i&gt;Mus musculus&lt;/i&gt; have proved powerfully informative by revealing unsuspected mechanisms governing basic biological processes. This approach uses potent chemical mutagens, such as &lt;i&gt;N&lt;/i&gt;-ethyl-&lt;i&gt;N&lt;/i&gt;-nitrosourea (ENU), to randomly induce mutations in mice, which are then bred and phenotypically screened to identify lines that disrupt a specific biological process of interest. Although identifying a mutation using the rich resources of mouse genetics is straightforward, it is unfortunately neither fast nor cheap. Here we show that detecting newly induced causal variants in a forward genetic screen can be accelerated dramatically using a methodology that combines multiplex chromosome-specific exome capture, next-generation sequencing, rapid mapping, sequence annotation, and variation filtering. The key innovation of our method is multiplex capture and sequence that allows the simultaneous survey of both mutant, parental, and background strains in a single experiment. By comparing variants identified in mutant offspring with those found in dbSNP, the unmutagenized background strains, and parental lines, induced causative mutations can be distinguished immediately from preexisting variation or experimental artifact. Here we demonstrate this approach to find the causative mutations induced in four novel ENU lines identified from a recent ENU screen. In all four cases, after applying our method, we found six or fewer putative mutations (and sometimes only a single one). Determining the causative variant was then easily achieved through standard segregation approaches. We have developed this process into a community resource that will speed up individual labs’ ability to identify the genetic lesion in mutant mouse lines; all of our reagents and software tools are open source and available to the broader scientific community.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/01/01" />
<meta name="citation_mjid" content="ggg;2/1/143" />
<meta name="citation_id" content="2/1/143" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/1/143" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/1/143.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/1/143.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/1/143.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.001669" />
<meta name="citation_pmid" content="22384391" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="1" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="143" />
<meta name="citation_lastpage" content="150" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Miao Sun" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author" content="Kajari Mondal" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author" content="Viren Patel" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author" content="Vanessa L. Horner" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author" content="Alyssa B. Long" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author" content="David J. Cutler" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author" content="Tamara Caspary" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author_email" content="mzwick@emory.edu" />
<meta name="citation_author_email" content="tcaspar@emory.edu" />
<meta name="citation_author" content="Michael E. Zwick" />
<meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_author_email" content="mzwick@emory.edu" />
<meta name="citation_author_email" content="tcaspar@emory.edu" />
<meta name="citation_editor" content="B. J. Andrews" />
<meta name="citation_editor_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=A. Acevedo-Arozena;citation_author=S. Wells;citation_author=P. Potter;citation_author=M. Kelly;citation_author=R. D. Cox;citation_title=ENU mutagenesis, a way forward to understand gene function.;citation_pages=49-69;citation_volume=9;citation_year=2008;citation_pmid=18949851;citation_doi=10.1146/annurev.genom.9.081307.164224" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. J. Albert;citation_author=M. N. Molla;citation_author=D. M. Muzny;citation_author=L. Nazareth;citation_author=D. Wheeler;citation_title=Direct selection of human genomic loci by microarray hybridization.;citation_pages=903-905;citation_volume=4;citation_year=2007;citation_issue=11;citation_pmid=17934467;citation_doi=10.1038/nmeth1111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. N. Arnold;citation_author=Y. Xia;citation_author=P. Lin;citation_author=C. Ross;citation_author=M. Schwander;citation_title=Rapid identification of a disease allele in mouse through whole genome sequencing and bulk segregation analysis.;citation_pages=633-641;citation_volume=187;citation_year=2011;citation_pmid=21196518;citation_doi=10.1534/genetics.110.124586" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. N. Bainbridge;citation_author=M. Wang;citation_author=D. L. Burgess;citation_author=C. Kovar;citation_author=M. J. Rodesch;citation_title=Whole exome capture in solution with 3 Gbp of data.;citation_pages=R62-R62;citation_volume=11;citation_year=2010;citation_issue=6;citation_pmid=20565776;citation_doi=10.1186/gb-2010-11-6-r62" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Bashiardes;citation_author=R. Veile;citation_author=C. Helms;citation_author=E. R. Mardis;citation_author=A. M. Bowcock;citation_title=Direct genomic selection.;citation_pages=63-69;citation_volume=2;citation_year=2005;citation_issue=1;citation_pmid=16152676;citation_doi=10.1038/nmeth0105-63" />
<meta name="citation_reference" content="citation_journal_title=Analytical and bioanalytical chemistry;citation_journal_abbrev=Anal Bioanal Chem;citation_author=S. Bau;citation_author=N. Schracke;citation_author=M. Kränzle;citation_author=H. Wu;citation_author=P. F. Stähler;citation_title=Targeted next-generation sequencing by specific capture of multiple genomic loci using low-volume microfluidic DNA arrays.;citation_pages=171-175;citation_volume=393;citation_year=2009;citation_issue=1;citation_pmid=18958448;citation_doi=10.1007/s00216-008-2460-7" />
<meta name="citation_reference" content="citation_journal_title=Current topics in microbiology and immunology;citation_journal_abbrev=Curr Top Microbiol Immunol;citation_author=B. Beutler;citation_author=E. M. Moresco;citation_title=The forward genetic dissection of afferent innate immunity.;citation_pages=3-26;citation_volume=321;citation_year=2008;citation_pmid=18727485;citation_doi=10.1007/978-3-540-75203-5_1" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=T. Caspary;citation_author=K. V. Anderson;citation_title=Uncovering the uncharacterized and unexpected: unbiased phenotype-driven screens in the mouse.;citation_pages=2412-2423;citation_volume=235;citation_year=2006;citation_issue=9;citation_pmid=16724327;citation_doi=10.1002/dvdy.20853" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in immunology;citation_journal_abbrev=Curr Opin Immunol;citation_author=M. C. Cook;citation_author=C. G. Vinuesa;citation_author=C. C. Goodnow;citation_title=ENU-mutagenesis: insight into immune function and pathology.;citation_pages=627-633;citation_volume=18;citation_year=2006;citation_issue=5;citation_pmid=16889948;citation_doi=10.1016/j.coi.2006.07.011" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. T. Eggenschwiler;citation_author=E. Espinoza;citation_author=K. V. Anderson;citation_title=Rab23 is an essential negative regulator of the mouse Sonic hedgehog signalling pathway.;citation_pages=194-198;citation_volume=412;citation_year=2001;citation_issue=6843;citation_pmid=11449277;citation_doi=10.1038/35084089" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=H. Fairfield;citation_author=G. J. Gilbert;citation_author=M. Barter;citation_author=R. R. Corrigan;citation_author=M. Curtain;citation_title=Mutation discovery in mice by whole exome sequencing.;citation_pages=R86-R86;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21917142;citation_doi=10.1186/gb-2011-12-9-r86" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Fredriksson;citation_author=J. Banér;citation_author=F. Dahl;citation_author=A. Chu;citation_author=H. Ji;citation_title=Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector.;citation_volume=35;citation_year=2007;citation_pmid=17317684;citation_doi=10.1093/nar/gkm078" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Garcia-Garcia;citation_author=J. T. Eggenschwiler;citation_author=T. Caspary;citation_author=H. L. Alcorn;citation_author=M. R. Wyler;citation_title=Analysis of mouse embryonic patterning and morphogenesis by forward genetics.;citation_volume=102;citation_year=2005;citation_pmid=15755804;citation_doi=10.1073/pnas.0501071102" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Gnirke;citation_author=A. Melnikov;citation_author=J. Maguire;citation_author=P. Rogov;citation_author=E. M. Leproust;citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing.;citation_pages=182-189;citation_volume=27;citation_year=2009;citation_issue=2;citation_pmid=19182786;citation_doi=10.1038/nbt.1523" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. M. Hilton;citation_author=M. A. Lewis;citation_author=M. Grati;citation_author=N. Ingham;citation_author=S. Pearson;citation_title=Exome sequencing identifies a missense mutation in Isl1 associated with low penetrance otitis media in dearisch mice.;citation_pages=R90-R90;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21936904;citation_doi=10.1186/gb-2011-12-9-r90" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Hodges;citation_author=Z. Xuan;citation_author=V. Balija;citation_author=M. Kramer;citation_author=M. N. Molla;citation_title=Genome-wide in situ exon capture for selective resequencing.;citation_pages=1522-1527;citation_volume=39;citation_year=2007;citation_issue=12;citation_pmid=17982454;citation_doi=10.1038/ng.2007.42" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=V. L. Horner;citation_author=T. Caspary;citation_title=Creating a &quot;hopeful monster&quot;: mouse forward genetic screens.;citation_pages=313-336;citation_volume=770;citation_year=2011;citation_pmid=21805270" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Huangfu;citation_author=A. Liu;citation_author=A. S. Rakeman;citation_author=N. S. Murcia;citation_author=L. Niswander;citation_title=Hedgehog signalling in the mouse requires intraflagellar transport proteins.;citation_pages=83-87;citation_volume=426;citation_year=2003;citation_issue=6962;citation_pmid=14603322;citation_doi=10.1038/nature02061" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Jacoby;citation_author=J. J. Cox;citation_author=S. Gayral;citation_author=D. J. Hampshire;citation_author=M. Ayub;citation_title=INPP5E mutations cause primary cilium signaling defects, ciliary instability and ciliopathies in human and mouse.;citation_pages=1027-1031;citation_volume=41;citation_year=2009;citation_issue=9;citation_pmid=19668215;citation_doi=10.1038/ng.427" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=L. Mamanova;citation_author=A. J. Coffey;citation_author=C. E. Scott;citation_author=I. Kozarewa;citation_author=E. H. Turner;citation_title=Target-enrichment strategies for next-generation sequencing.;citation_pages=111-118;citation_volume=7;citation_year=2010;citation_issue=2;citation_pmid=20111037;citation_doi=10.1038/nmeth.1419" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Moran;citation_author=A. D. Bolton;citation_author=P. V. Tran;citation_author=A. Brown;citation_author=N. D. Dwyer;citation_title=Utilization of a whole genome SNP panel for efficient genetic mapping in the mouse.;citation_volume=16;citation_year=2006;citation_pmid=16461637;citation_doi=10.1101/gr.4563306" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. B. Ng;citation_author=E. H. Turner;citation_author=P. D. Robertson;citation_author=S. D. Flygare;citation_author=A. W. Bigham;citation_title=Targeted capture and massively parallel sequencing of 12 human exomes.;citation_pages=272-276;citation_volume=461;citation_year=2009;citation_issue=7261;citation_pmid=19684571;citation_doi=10.1038/nature08250" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. B. Ng;citation_author=K. J. Buckingham;citation_author=C. Lee;citation_author=A. W. Bigham;citation_author=H. K. Tabor;citation_title=Exome sequencing identifies the cause of a mendelian disorder.;citation_pages=30-35;citation_volume=42;citation_year=2010;citation_issue=1;citation_pmid=19915526;citation_doi=10.1038/ng.499" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. I. Nikolaev;citation_author=C. Iseli;citation_author=A. J. Sharp;citation_author=D. Robyr;citation_author=J. Rougemont;citation_title=Detection of genomic variation by selection of a 9 mb DNA region and high throughput sequencing.;citation_pages=e6659-e6659;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19684856;citation_doi=10.1371/journal.pone.0006659" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=D. T. Okou;citation_author=A. E. Locke;citation_author=K. M. Steinberg;citation_author=K. Hagen;citation_author=P. Athri;citation_title=Combining microarray-based genomic selection (MGS) with the Illumina Genome Analyzer platform to sequence diploid target regions.;citation_pages=502-513;citation_volume=73;citation_year=2009;citation_issue=Pt 5;citation_pmid=19573206;citation_doi=10.1111/j.1469-1809.2009.00530.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. T. Okou;citation_author=K. M. Steinberg;citation_author=C. Middle;citation_author=D. J. Cutler;citation_author=T. J. Albert;citation_title=Microarray-based genomic selection for high-throughput resequencing.;citation_pages=907-909;citation_volume=4;citation_year=2007;citation_issue=11;citation_pmid=17934469;citation_doi=10.1038/nmeth1109" />
<meta name="citation_reference" content="citation_journal_abbrev=Open Access Bioinformatics;citation_author=V. C. Patel;citation_author=K. Mondal;citation_author=A. C. Shetty;citation_author=V. L. Horner;citation_author=J. K. Bedoyan;citation_title=Microarray oligonucleotide probe designer (MOPeD): A web service.;citation_pages=145-155;citation_volume=2;citation_year=2010;citation_issue=2010;citation_pmid=21379402" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=G. J. Porreca;citation_author=K. Zhang;citation_author=J. B. Li;citation_author=B. Xie;citation_author=D. Austin;citation_title=Multiplex amplification of large sets of human exons.;citation_pages=931-936;citation_volume=4;citation_year=2007;citation_issue=11;citation_pmid=17934468;citation_doi=10.1038/nmeth1110" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=A. S. Rakeman;citation_author=K. V. Anderson;citation_title=Axis specification and morphogenesis in the mouse embryo require Nap1, a regulator of WAVE-mediated actin branching.;citation_volume=133;citation_year=2006;citation_pmid=16831833;citation_doi=10.1242/dev.02473" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Shendure;citation_author=H. Ji;citation_title=Next-generation DNA sequencing.;citation_pages=1135-1145;citation_volume=26;citation_year=2008;citation_issue=10;citation_pmid=18846087;citation_doi=10.1038/nbt1486" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Shendure;citation_author=R. D. Mitra;citation_author=C. Varma;citation_author=G. M. Church;citation_title=Advanced sequencing technologies: methods and goals.;citation_pages=335-344;citation_volume=5;citation_year=2004;citation_issue=5;citation_pmid=15143316;citation_doi=10.1038/nrg1325" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=A. C. Shetty;citation_author=P. Athri;citation_author=K. Mondal;citation_author=V. L. Horner;citation_author=K. M. Steinberg;citation_title=SeqAnt: a web service to rapidly identify and annotate DNA sequence variations.;citation_pages=471-471;citation_volume=11;citation_year=2010;citation_pmid=20854673;citation_doi=10.1186/1471-2105-11-471" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. W. Stottmann;citation_author=J. L. Moran;citation_author=A. Turbe-Doan;citation_author=E. Driver;citation_author=M. Kelley;citation_title=Focusing forward genetics: a tripartite ENU screen for neurodevelopmental mutations in the mouse.;citation_volume=188;citation_year=2011;citation_pmid=21515572;citation_doi=10.1534/genetics.111.126862" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. Tewhey;citation_author=M. Nakano;citation_author=X. Wang;citation_author=C. Pabón-Peña;citation_author=B. Novak;citation_title=Enrichment of sequencing targets from the human genome by solution hybridization.;citation_pages=R116-R116;citation_volume=10;citation_year=2009a;citation_issue=10;citation_pmid=19835619;citation_doi=10.1186/gb-2009-10-10-r116" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=R. Tewhey;citation_author=J. Warner;citation_author=M. Nakano;citation_author=B. Libby;citation_author=M. Medkova;citation_title=Microdroplet-based PCR enrichment for large-scale targeted sequencing.;citation_pages=1025-1031;citation_volume=27;citation_year=2009b;citation_issue=11;citation_pmid=19881494;citation_doi=10.1038/nbt.1583" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=E. H. Turner;citation_author=C. Lee;citation_author=S. Ng;citation_author=D. A. Nickerson;citation_author=J. Shendure;citation_title=Massively parallel exon capture and library-free resequencing across 16 genomes.;citation_pages=315-316;citation_volume=6;citation_year=2009;citation_issue=5;citation_pmid=19349981;citation_doi=10.1038/nmeth.f.248" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Zhang;citation_author=D. Alpert;citation_author=R. Francis;citation_author=B. Chatterjee;citation_author=Q. Yu;citation_title=Massively parallel sequencing identifies the gene Megf8 with ENU-induced mutation causing heterotaxy.;citation_volume=106;citation_year=2009;citation_pmid=19218456;citation_doi=10.1073/pnas.0813400106" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
<meta name="twitter:description" content="Forward genetic screens in Mus musculus have proved powerfully informative by revealing unsuspected mechanisms governing basic biological processes. This approach uses potent chemical mutagens, such as N -ethyl- N -nitrosourea (ENU), to randomly induce mutations in mice, which are then bred and phenotypically screened to identify lines that disrupt a specific biological process of interest. Although identifying a mutation using the rich resources of mouse genetics is straightforward, it is unfortunately neither fast nor cheap. Here we show that detecting newly induced causal variants in a forward genetic screen can be accelerated dramatically using a methodology that combines multiplex chromosome-specific exome capture, next-generation sequencing, rapid mapping, sequence annotation, and variation filtering. The key innovation of our method is multiplex capture and sequence that allows the simultaneous survey of both mutant, parental, and background strains in a single experiment. By comparing variants identified in mutant offspring with those found in dbSNP, the unmutagenized background strains, and parental lines, induced causative mutations can be distinguished immediately from preexisting variation or experimental artifact. Here we demonstrate this approach to find the causative mutations induced in four novel ENU lines identified from a recent ENU screen. In all four cases, after applying our method, we found six or fewer putative mutations (and sometimes only a single one). Determining the causative variant was then easily achieved through standard segregation approaches. We have developed this process into a community resource that will speed up individual labs’ ability to identify the genetic lesion in mutant mouse lines; all of our reagents and software tools are open source and available to the broader scientific community." />
<meta name="og-title" property="og:title" content="Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/1/143" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Forward genetic screens in Mus musculus have proved powerfully informative by revealing unsuspected mechanisms governing basic biological processes. This approach uses potent chemical mutagens, such as N -ethyl- N -nitrosourea (ENU), to randomly induce mutations in mice, which are then bred and phenotypically screened to identify lines that disrupt a specific biological process of interest. Although identifying a mutation using the rich resources of mouse genetics is straightforward, it is unfortunately neither fast nor cheap. Here we show that detecting newly induced causal variants in a forward genetic screen can be accelerated dramatically using a methodology that combines multiplex chromosome-specific exome capture, next-generation sequencing, rapid mapping, sequence annotation, and variation filtering. The key innovation of our method is multiplex capture and sequence that allows the simultaneous survey of both mutant, parental, and background strains in a single experiment. By comparing variants identified in mutant offspring with those found in dbSNP, the unmutagenized background strains, and parental lines, induced causative mutations can be distinguished immediately from preexisting variation or experimental artifact. Here we demonstrate this approach to find the causative mutations induced in four novel ENU lines identified from a recent ENU screen. In all four cases, after applying our method, we found six or fewer putative mutations (and sometimes only a single one). Determining the causative variant was then easily achieved through standard segregation approaches. We have developed this process into a community resource that will speed up individual labs’ ability to identify the genetic lesion in mutant mouse lines; all of our reagents and software tools are open source and available to the broader scientific community." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/1.cover.gif" />
    <title>Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VZjOcnBIgMu_39zDgIKNfAY72Cv9IF_vFJVs7a_dArc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459224","apath":"\/ggg\/2\/1\/143.atom","pisa":"ggg;2\/1\/143","ac":{"\/ggg\/2\/1\/143.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/1\/143.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/1\/143"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/1\/143"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/1\/143":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fWyaAS\\\\\\\/R8Z406FXGSaXvGWGWKdrYDXsSE8YWwVL4pzPlX4MJgMjGg+vYltAvskmMnyQGYteQVZEqhrLgzBfim1eORC3ADmao1dFUnX4RwREX+83CKEtihqKNullQYM+2\\\\\\\/QVvVpYjaCZ6Fcl7eOe55aESYb8uWp3qhlGGPVg4sG4zv7j893u6grzCsQQV\\\\\\\/8qvYxCBXWjiTZD3PIjkjIXts6leX3MMpuuUcOUg3bxDAwtLIEJaLY\\\\\\\/fDqGzvL8vLxJckcqbs2+LN1a+3WsLLN31u6lsZwBLv9Te+DhoMuTsKVJ2JgMz\\\\\\\/UJa1Sn7V9BimDOS4PItC6pcIqdcPgzxjtbWF9H06tzVtuKd\\\\\\\/usqYjWXsTG18fXQKJPQOVYp1mDupT5RNo1yTzni1sBk8U9z1ZV6xujyTxarUYDSUIEsqFAAGFOEllG0v5F4LOH2xkydnR0agVpBEcLpMxH3dqWQf6vFYvat1Mgza9Pem28H44iv5K6JGjD0JeLNBVmfdIeWm\\\\\\\/CxB9uHcsNAb6e6+WNsOSx2xjYdbDlxQUdZH\\\\\\\/TXA\\\\\\\/04YN7s77hw68wCIKtDVFk4i6ntoM0sOGcViWjajW5RpGG1gRFyQgCOZJKimXZ\\\\\\\/LywbMMTEoWuXsoIma+6ViVgtmbBPxCTso9TnQUoAWzdFyYqN1+uTUgcm050oSo6JjUG8yiexOGjwNJGEz6XJUOLo2PevELV1WQvSYXOF\\\\\\\/yeJlgJBKp6RU1wPjjpI1CjM2htF\\\\\\\/eS6R41WjUnJu3ATg9xQK8hsmfUWpCU3OoN9\\\\\\\/pxdwJ8zk9BsniLBmSBDhmAOlrBtamlOmZ9T6H9UkKYaxOh3oLspgzalI4pLw4HDxmBI7\\\\\\\/qWmWbTqyz2P9p63MmEXkdoSvv8KWZG0XNh363cIpZJv0vM2QFSCqBECHs8nK4Ej4OealsDD\\\\\\\/cMY+0cBFC3BPhVIsG4cYBs3zVDHB+MlV6AiKDN0ZvVZUUehw\\\\\\\/L\\\\\\\/EAu\\\\\\\/S+REYWaI6cyuXZAlYG29zumK3GjC\\\\\\\/FZQoh\\\\\\\/2+8CR1BlRB2\\\\\\\/CiYHBsl33ETxbMQY6FrVInuGB66JNW0sIpVZwzUlfuYjsWT7YxwyZL7soQAmgqZWVdnpPf24Ht+EHZy0ekVBaqPl2DV9SZckePVM4KhGuGF9TI1OoN\\\\\\\/dmrLTRcSkNi\\\\\\\/10UfPQjoSxxTTGpeUCAQQUUh336AF1iiWOUheE9oKv\\\\\\\/AOccWWw7YtvKErQpIXImX+okr+kKHP2egKOyLZC0DRuBUbdJ4qig1UgDZeVbVE7r0vuOiDVVlndUk\\\\\\\/HETPIwU8\\\u0022,\\\u0022iv\\\u0022:\\\u0022Nx8e4JeGdOS0miyPLyx31Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a150d493a6b5c226d9ecc4f85a0451d654ad6c3122e0bae42b717ab2ecff429e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r6eDMlbDVy\\\\\\\/VF3xaQn\\\\\\\/usWge5j8zMAEkCYQ6hSloc2eRRmh8Hm9oDkWVQU2fIvVPMWk0BrfBBsmZ++lGRSkERmAkZu8a15SUroDQlnBMUeIM6NLBXSR3T9Qnre7oeduHu104thnpRxFdPS7yOe6lLXPATkXovZh9RXcnqpOJkkv0bZn9C2Fn+8xlbnyAaZ1EsC6yXfy3dn4P29SmS0FPOd\\\\\\\/nyoqzRduHUduXatJkp1mU8S2CkDvDDgHJFgA89SwyG0g7SuBBvOpl4pgQ7qRMdv0CRqwWTj7hr6vBd5E6hH19h8MTc2Jvjr0M+60WCN4wDMvhISrdW+wO+U9H+dpC3oWHbYX1rAnLEfOv1OATb7HGdt603TCICU5ZGtGHRa1KwVOa\\\\\\\/xL7nsYTpPpSrJ7JbJA8cNGfbyjE18jQRgyDtIV0hAoA\\\\\\\/SxlyBx\\\\\\\/7CoGFqEb9fqhN9BQKfK5tGn39mEdkeKFoYq+U8Rlb14cCZ3i8N4yVIBkgC+tsuG8TjzbX8L35f9dNW8GKt\\\\\\\/7WMD2A0dI4JYU3sMGsw6m0HgvlZAHmjFYxARC2p9VNSsaMSW47w39V5gvGDFb+AivNiEy6UqKTUeJ2kwig1Vk2MXilRRICHwfRaKCQR03w3KyCyyInp0Fp7WzWn0YFilMUWFd4ZWpcrods2Ox3aZAehrvxhHNtRc+6U6DcUi4iHdabsAMuvFRS8YHq7YBamUArMmK4oaDwTXLwgK\\\\\\\/Bu0AV9+z43vdAqyaIcq1BXcE8nVgST2\\\\\\\/hlS3RMvIkO4w2rBgPfQzNdbrLA4p92x0+aPIOipeQ0IyADMGO7z+U70BRDUgR97v25MhLdwX3QorlRt9p8OMcBHQc6S85tjiGVm8DZYsMh7n0iiaFWSKRnc0wrZttZ87uG0JEX0JOqD7CvbM73B2PP3Mb6aZbo\\\\\\\/UD9JlNW0TeD3Sx38Jr7VTPF+axpoAGyl\\\\\\\/bxN2LC7eDOHd\\\\\\\/o0ArxzP86zLG5cOhXiwMlABDfVN\\\\\\\/sGJJio7kLIl+4zrxYhic7hIyWNU1AC5TGKNW9xGv\\\\\\\/qCNXIGSQAuXUKDqrJhIll5F9sjwE8Pb5lS9KRdNksZsCwkB4f8yK0grVqb+2jvR98jfXufJSicSj0kvgMlxSOpVgZy77xS2tBb+qCVEr8pImUNKaF9fp3\\\\\\\/DU\\\\\\\/mmHceZASljGAIZ1jTxigRAvXBxHz5TdTG\\\\\\\/qcDolsxDCzsxP85NAcR3b1Agc8dRtW+Smqrn7Clx5W9+Hw28yWXbHyACNWQ51Iktdbqa20MG9BFhiqC14aSKcnPtg10ql17dIkjpjz866rGxPvgqoqqJQDFHLR39OU0bvtuiSZ8Le9hjAs5eMkW4zjggEJHJJIcgPKy5qQbSwNa\\\\\\\/8kGSFsizajMjV5u1qrYnoLYr+FDLkwsFOOgoY+hNbwK3EcVUWbXpGoNPXHiSFh1A24WQCkzsbqS8yWI22y8hOSL7d7deCaOdhuzb4T+9yx0NOlXV6\\\\\\\/42k\\\\\\\/BKDPKv\\\\\\\/xnpVpo3zqBk2Lh3\\\\\\\/pi4dJyje0AD7mYLiiAL4Q\\\\\\\/3D\\\\\\\/EHW+OBaf3\\\\\\\/QGID4+9oVuqrKh1mSANSfruXtBF3m7+24V9MN95MtjQckeKd8M11\\\\\\\/MEkQnoRYmu7cbZzUoV2qbpRDmuqV\\\\\\\/BqoVtBP1\\\\\\\/EkgWK2lWX9NhcXBPcCz4BIsIyzI4fnCZ3bVZ22MxQLHWPgxln7+OzZ1+lpKHARYtMG+pWS+hUNWza3Fdjqx44bHJhewE7YF\\\\\\\/wJt+Y7\\\\\\\/rXFR2weKeF7TaL1DtGMPChBHT2zEh1JcuPSmlZTT4wOd5KIaJpX\\\\\\\/CYpojvteXKwx7w1sIxlUecmcf\\\\\\\/cKuedQQk8XeW8IEmuiNSH4zkLbv3FhKhtEncS4ujID+nJs+5g1tHjn7yazEVB+YsFECOWJ+eUcztjJVpc38QXSLHIIJsbIIXA4FkWrp1qkAq4dZkTRfkL+syY6zK4SRmiJ5vg9TYdAxejf3lPmfk73hoCKLqMPnF5UQJbyGzQyOp8AKR8H\\\\\\\/YRVG9BPFm6yTON5hlwoXtIg7MALqDcxURx2B\\\\\\\/oa9FvNWfoqR8XHZm54HbkDVUUFS7AUdTPFU\\\\\\\/9Fd5p2dxna9MDgmwLKOpXtWX0EIFKSMik9skWTZWGiW4xKuZDQfFHHtQC5doAqN5He7YLT1xZt\\\\\\\/9denqaE8hBp6m5UIYxSYl83f\\\\\\\/lr8TZXxwNdXz2QCg8sW\\\\\\\/U3vUcg5mnr2dJ+rl3p0DT3ZtK1Qj4h\\\\\\\/t67Vm42u8mDRjZKGmyg99bf176CMTURZaYSj75SFNADMjgPL0FflxCxgu1sEnItYFONX1p4xgFT2GG5TBRIazLPNZRDoyFQyMq2+xYQ8DgNw0b8NFp\\\\\\\/\\\\\\\/xQBTMc7yyemQiXAQdNp6eNSXy+593S8bhYWKkrdvdKqExOYOTJ2TvBW35noHwndXyGt5kDRc+ihZqe\\\\\\\/JyN\\\\\\\/V2tMgfVPZyMTwm7Jn\\\\\\\/QwnEmf0lbNyz3G+8BfEy1rLBOxQXqq5SKbpzgV6qhc\\\\\\\/49ggLJDosMH8\\\\\\\/6seZjrtACQp\\\\\\\/w3eie4eyFt1jHZ5ggHtRcgg2eq4UREm\\\\\\\/Pn2nhCzMxV+mlP+lZJ\\\\\\\/Vn8Nd4zXEkRpH81iRVDxXzxJQIFK1Bz+2xkN+YFnBfqIW5N51WhL7wEx3\\\\\\\/N9\\\\\\\/sKwXcLu2njW3H1H9gNZrFGg1CXecCqzMS1paszHGXsXAJzMKYxlqP2j6QuE\\\\\\\/hKAYtKlIktHhx\\\\\\\/gVM7J3E4zDTixvZoBrek6kJnhWnMXTwy823KVYjZjZE6pXTDgFWaZ1TlQIyTNbo5jBY55FL0Qa\\\\\\\/pUVJSeWK9MPw14YGl4K3+TS6uQx1X0SfY80o5faHbfbdDWESVS8cudmFXF+uBVkilSiH3NSc+TFIBWnsF0QwbsMD6lRD1e1OFmw1GDN1HFdhE6Q74e5vDRVdLLrMCYFMVeTXjoc0qTZlU0UijjTT2\\\\\\\/3ETLMOmlYxE0k9YmwSyhgRVmfbllIZYOtaC4RWEWzhEZfYWhv5SlCxtCw6e5p74RODaMs+Twse9lhNfkuSN\\\\\\\/zipyJTNYJNbWSWNoFAJWULURgj9Cw11zKp24FdPYDENJbxvUiN4ywTYO\\\\\\\/h1AWihnWb1XfWoLg4rTiaXhFVAbYQsdkCJKpHXTmBj0vz6zcyz2xqU7F72jbFaieo+ni95ibMmglkOdSxsuyE227kfld3cGkN3RicZfbVQgjYMH8JeGELfZQkEvKdxb2DyXtWBexUzup9U67VD7acO9kcc9BQw+SoUcyFdTUPjkQ2gMM+G1hoyQM4PFl55o5aePZstJNhD1zoj1bsWFDGp8m+Wu671pp+wWazgaZQqS9g2Y78xIu\\\\\\\/ji9hS+bS1AhqQUZfOWfaNrKcVaXtkLHGJVvYiAOayIryPfEkgXkNAGMIHx7\\\\\\\/LL43HEGHL8UIAva5B4ybtEvcyUJzp4t2sUOxqtc\\\\\\\/HQzwp5YX9end5w4+yQeAtYN9xx6OlH+TkeTkOzzOo2dkh8CvHADAWuTlr0u1odVJ02TN00T3FGej\\\\\\\/I8eQJUiR+v986Gum78do3FOToBofHAcq0AFgDj86d3HT1BH9jye\\\\\\\/6EXin6\\\\\\\/L7Wzba872cglBUv0S43ZiGx6sv7RvIaDg+rz4mcGZlV72yZt+0v+sgfc\\\\\\\/fHqiY67n+a5r2DYaY\\\\\\\/4Cc78Ns3p+4HMcMY5Wwoayg5iASZ41JBJQzOjbaBq+6bo87rKL\\\\\\\/iJ0EqOUdy03tRaEW8cbS3of95U5mTn3nfFCptKJ72zoWKJT\\\\\\\/K3mU8yRTyrAEqi7JAjTVry3DJFUTF0ZF2QubEop8V\\\\\\\/XTc6jhe5D8DEyKSMmJ2F5Qr7fho6s3\\\\\\\/Ket6hJiEsndx29bagbd002WdUHD07Y3r\\\\\\\/DPDZQ6gA20YL2+1oKqwf4w2tU3IYCGM5t82JK6ycFoCazF0NwEgA5jVFCZrORvElpEISBe3YDzXSAioC1a1LjZTQKdK+ZU\\\\\\\/UPgQ4vNVPzjcfhIhR6c\\\\\\\/kfG1Km6iI7Eh8SLRLlXDU\\\\\\\/PycMspQeiFnmVRY8WgCIuE9YQCxk9Mwo4msF5WQexFPyd\\\\\\\/3luI8CNuwEdEbsfNBxjgBjJHaV+AwFAsqTB5OJlxP6TKVxCNSkgHNDyFBPZfXLDAAgwGWA4sKk7Mq\\\\\\\/MbXX60W7r9MYBbUtJEusxjr81IXlnL30u2bJWttu7pnYujHZLTqgVI8f9r4e4Rpau2zUySZT+tqO3I7DSVRX33rTwN+ugAOvlrhU0jhTRG3sft3VPezDzd8ybI5owsWzbh+Tf3nUjUIgOx13XYyLbj\\\\\\\/3j\\\\\\\/vyaC8p1X+2ppsyf+QwYPgaWmp\\\\\\\/CAV7mHSJoERU6YcTBc3ZWuKpSkwX40ThW1rlP64x\\\\\\\/pArM0wQiodY8CxUAMc5Lg4EeWGVYq5LsH2JjG7vKGKtyYYN+Owa37EfVhwWRVJO+LVhVbcrxi39ykK08YjcD6VtKB9408bhoi1\\\\\\\/RQMEJdYK45F191\\\\\\\/sMjMLuyInVeAuQ+J6uHMpVoP\\\\\\\/zmtD2N\\\\\\\/m2TS0CGn1goQorBs81sKG2hH1vf7eEXRTjgWZ1DLq+sIuphi\\\\\\\/1Qpf+GR0P\\\\\\\/YtnB6UYTE+cC2ikIkmKXZU7NoWHexTbzEekQtt+emxTodHIvRqyZPxgJgfOPXzCTwlJt0R3S2WiFh0p7hcEFvRyv31y0AFa\\\\\\\/LZJcIkWc2tjaX939PyK1WOQPPj0+\\\\\\\/\\\\\\\/7dF9pjkk22bs8MU3lT5g+Uy6WinUMEliOvhhAJVI6aITdlr6tohpviSweSKgrbzX9RiF\\\\\\\/e0XOgoO\\\\\\\/b425CrduFrihRsUMNxahU9I7RXJdg\\\\\\\/b1qGR9qLVJS0xMSQgpCreObQYJq25LByHJ\\\\\\\/COolKvyX1X7gBpJ482qUNS+JHnvBwRAqpC+6rP\\\\\\\/vuoH9jO3BEWhmMCl68XPZUyPYJnoKc5yutPboDc+u9mBRrxraoVXeoAOjyzCK+73Pw1RXligu8fRumxpWJHyBRjhQvwBxXBy9WDD8oUaqYkhjj+e6ihzg3FlYo16AZkMnA1gAKwd8EIAGtmhO5zhP4Hxfu1maO3miIFwEpBBok45o3tmPJd7NTElk7wr85zQCpWyRsYZyQ60PtnKmsu1I\\\\\\\/ub30dpq9xFi25MZPk6uQ7MZK+ieHB1NgDFO8D60PCVQiRxVnQ7ep7MfOFLGdxWUIE+KP3zBDDNTKmigtyklcDAfkmyjK6bHzb4aMUPsqDl242WKrRucTUpiajN+Q7up23iYQJbFz8n1jLAzCShLLdCwAnxPfVuWJgPpI8931ARXB8xxdnOSeTTf2S2nn6VLD3nvMzLtUcsL+OzusZwOhsCTmL3MjY8Zj6T3sm36M9ZtwVkuoNqfimMeSYLr2ipvu1JhNRD+GWiIwpYPtdewV9skbtS8wya+Jv9KknrzIRWBFloCm2SAy339fPy9sHEEHs7OOJV2\\\\\\\/1mENtcyej4XZ8yCTr57ereVU8x6elqrtSqX5gwffHaUujJS3Xk0KHYdrMwViwSEO44svnNGz7EUYQYCGark7VNEv+YqR9Tu3UGFMNYIpT1ccRCZqzR9THHvR4T2jau8lnqMYRtlb5W7X5dfoAT7iJaq4sFC8AYtof68sNmplRrbJ74rZJIdwt4iXbk06y56iRYDqCMBqcVuNX90Jp+iUCDyGpfM4T0ySJO2wcvdz4vvc4f47s5DWxkJxLUb9UyBYmOQujJD4iTwhf10BKKzQAEqhQ1NlJSxKV8m27H0WIJPQfNUoXeGOqnC7v01xVxpowv3pMYvyKzWhDKNbncr0CRZSaXnoZwrrqFEXG\\\\\\\/twv7e8utPtDUXIMc3+qc448KcZZt0r+2upEg3DBZ6iPBFxna1VWzslRCkUP+ysuJmqiQQJhy6V\\\\\\\/d8wUCaz7ihYztEiqgYcKCFb5fuxuW0X5o\\\\\\\/yKyqaJg81+4+Kgd6KCGi8aMxxhi5NUrWbVAEQfaoNGiil\\\\\\\/34U8WJY28\\\\\\\/iuo5aYLnYio+HeNKej7nvBjYsVsb258yxnM7H+vVF9TTmyTWVWl78MaQmsnC63JsNbTrz+Nu1JRODQIdGzNZsmzW1ogbUpbHR2y2RInGGwSYTxShxpHKCnMkqs2UyDLfsQR1+jmp7kpbJ56qaKu4Ijl7H2LYYd6qO7z9n9Q\\\\\\\/t3R4bDJ0JTK4eB+\\\\\\\/s+kx67IPg0lG1FY7PxskxlCWpfWLeO\\\\\\\/A6xZqTxZOSVjZmGFxiEW5Nsvyh0mb3eRsjIEEmEBObUVZPwj9WJmZPA7AS2T8dpnrokZqISmFNLFRsFtvDIAtMPsOgHF1Q\\\\\\\/jF0GD0k8zupux6CDhCOqzGv8o4IMpOfSizhCj2p5f89hYqO2RK7IQarmAGwtaLWEbGJ1+jwnW9DyRzMITkJAF5dGO\\\\\\\/14LyZzfaRD64ytY1eAQGmJLV1fGjz\\\\\\\/GjC+BzCqe+aEQNZQX6WUPYyJ0pjIWCyHR9AtpoiHeFee2Lm6\\\\\\\/g2xCXOVRZdDq808crzac80MskUe3\\\\\\\/No9Pts6xE2C\\\\\\\/R5TnAcM0eCIYVAHV2E7Ybtcn+JmECaZaVdi8vjkzYUUpjJg7r1B\\\\\\\/NhPkZg245A1nua42sIaR+Mc4D9HyE9gziaZ9olM0LmOijy+imH\\\\\\\/fk1wXiXjCGc5uU3pscgoaQZUNStHSbsCznUngiZtaoVNVJqRwcSjS+G2L1m6sZtVXH3HBjDW6JT1ac2EkX+L\\\\\\\/hNMFvaIN2umPylIpkEtYR7tJotx6HUFa\\\\\\\/117deczw\\\\\\\/tRC2W\\\\\\\/VW9Rkeby9KHy1fQkSw1aI4dYDf83SJUUIYvtRFI6hbUmCgWWTpcJY3WLzL2uhQbIoarKyy37ek\\\\\\\/HcKDLLIRzJ4SItD4YYBJFcFlE8BTKoUphPaykrCcnaUtifYkUzr6kso+CPzgotP5gxzAhObe1sdq6tdCI2U2R4BVwKrtgUuFHw++CkFyGoGBA+K4ktEKf0KsiMWmr5bvT4TB8oK7FVcF\\\\\\\/i1+4aDrbsB+F93ctiNqUSU7muLA8hqVQiyEy\\\\\\\/4idqBJOabL7cY21XljHbca\\\\\\\/IVbxZ98GhjXi3CWRSDRdHeqK01r5RvMS\\\\\\\/2CbRig51T\\\\\\\/KIPA7x6whaJcm4IkhYNsHogtKIGYygFq\\\\\\\/YBaFnXyDKzvVtI5YBzz3pd7vUsaN2pa6JFa7QLcnt6lFm8PlA1mc+FyI2RFhipsP+YyDuVw7XaBpTB76JqddMGru3YF7+TyKKaa36HCq4Keaskt\\\\\\\/T+o0o40GG2F8A0qu8aEhWC3Kp\\\\\\\/S9v3NbenekRL2KTBozb3HBviQzzql6SF7iZUuKxMJaxyHWxphGmMJlZeJGdeiDoXLBKYmLyJWAnNLTYMs1Q+KLi5voT99F3CQ6UDdGuB+KVMMsFyKTYrk07IFFdodPWhOH8OwHqzM3TO93tlAhNARFKyOyF5aXMn6ovPFs5jeWDwwwHH\\\\\\\/GikLYqzWmlIqQ4CDnpV69DOjokt2DVNO8PqPGyBBxPIkAu1wWjZScMYDwMfEI\\\\\\\/02E620OJ\\\\\\\/iwIrAj7Tep6+PkCzd6UC\\\\\\\/VoSzxvdC7VDpeunmv8ppOQxc\\\\\\\/fcCALmpNhQ0LT9CzQP+8NAgApDwmc1m3g+gEBHraJUW9efTnXqxEiW9nNEwme0otNqipV9VKNVq7Mt0iDOfXIFuUdUzR1XHyk2wTe0KmP4SQoGzvOwtDUjN\\\\\\\/imBEHrDuM5rkj6\\\\\\\/cRLpr\\\\\\\/IpZbZ1kQVIxOf27jTatVEnku2tAvxWAzjQbcTPTFGreamGrJpKCck19SfGtFHrLmOIh1CZCsCmYb8+6DwY4FPNs75lZ5+H3N8+VasKhDbDOmeO1Y3VU6AWPr8VmSnicEJWvz3WExsMu\\\\\\\/+bzMBKsGpm4zBAEJRR1Ua4KBCosdAzZHvM4CH0EbbT4ik8etOrVe6o2aIxFyX4DwzpyekFesEpENoY+zTk9Btfl0WEie+1nGhBB1YZU+jF0IETw4NDZjmWV3+E\\\\\\\/QXQwrSWDcEt67FYNzofffJRH0jguOwbt\\\\\\\/nvUG5vMI6mUwqBG4hMdb2DpYmg5ZcMfqHub3wqwvGvTHFNYOE\\\\\\\/+Vdj1Dk8ePMB4fh3t5CmD4iVOZ+vo77QzogSo46j0s4UWJXBPpJwqg76xYz0vK1Qv1nO2yCIHaCHonQOiZkO5jLStVb4rblBl3FfaAJMF60gqbkjn4PPWvQPwdiMIipRiWsE2FuJJw+xC0vdeUvDK49wT7VB+taYI6H5GTt+ojXmtfL8m9ibpcDagFQ+Fn0kV+sLnyUdWsfr+dnZ2GCSfm1UWJn98bQFXxw4oePQ8S6goPcxxcK\\\\\\\/dwCbGS6keLHsQVdMytWq\\\\\\\/kgtMELIEMvprYoflB8WjUZAi6ss76e1MQgigge977P3cyyNHeK8uuGM8taHpaEEkezBbdn7gh5Uznuas\\\\\\\/il2BOl1kA94OYQfOniL\\\\\\\/wKS5c0k0Ckq1ui+srbcfYdMwjjtYlXLILFLN5hmatfaKuqam1LpZ326fHq1j+FRVyJa6MzdCzj2KdhNU8gf3HVuRNZQObNXByOIoKc2Zch2CtqYyqMnU3dihW3A2V3ePP9iwCHR0wdfzzZl6Y\\\\\\\/\\\\\\\/sb2L1x36HT\\\\\\\/Mxzni3+GFNtnMMuordeunNSNBXND+qGnFWx1M7OHg6kjACl+240lufP90A42j0gLZ0WyWXzG53NmMqsQiTxuFrCEIiWrmUWBtMZZjAmiELr8tRjQ5uXKUaS8NNu5k2laMSsrX3jmA8fHf+NUo+Y5YUoRLNiKbL\\\\\\\/A9NatBV5zHNPx63dpA8Qw8iY9\\\\\\\/l4gC39FReGB5\\\\\\\/3S1s1Yoj0DfkXYBMfl99pS6gNezt7rth6+cp71ve6OVAywrjBDBjINewsmiq06KeMr2wcw38deM0cEBqGBV5swkPS4LqqYA6ePGV1RakUrkql3\\\\\\\/jJ9Ku9imW44KVi1Gl9nuWd0g9wOg9OCkWjpD8ZgN\\\\\\\/hb9Ox32z1lmclqcuJFYbR5CBoqTXl0+PUfJj1\\\\\\\/qxgnTzEARMKiyVjrWn1pzHhWQ\\\\\\\/T3GU0R5Y6vxj\\\\\\\/NfgxGhbUtTkfJtKhssSkK7DSS0St+e5VFwLVTht4qskFo361ikn8vQAvOtVpUI2UyCepeYb5ilZloCKb6U\\\\\\\/Hu3PFHsG9qWlA6i4bbfkIg3\\\\\\\/6gKsEPVNGJ6\\\\\\\/qEYeWfo6RONyArC7OcXvuVoHH\\\\\\\/lF6498JiaifvAn7v2N2ukMdaZXSRD7LWwqLLu9C6lbQETwHu4VVH0LMl15MwbotMkdnuTYk82sEsoZ2\\\\\\\/a8\\\\\\\/2aOV6\\\\\\\/INkwExpNRlFyHw0LvjPOthrwZ+nCyhmzDXXwCrOGZd2hoFTTzLIaVWoc9C0zjNTE29kFu3RUnKsD\\\\\\\/Klefv5XKWfBaltR7DT+o9u2FRSIxP1aTWH57lcKfYDDHND113LMRuYwSvwSfOp1XzKeS\\\\\\\/rTKRcPpMtE7rpRcVZ\\\\\\\/ayQCWWajN2u6y9cI5FcqJZv3xW0Wv19\\\\\\\/kmD4BWhQFPr5zTukziTnze4sCSfh8mmLwjn7x\\\\\\\/BGXYIrVt7WIMOa9hl4DYm8sF+kqp0z3KhU7opAxGZszx3ue6nfkBIX4neKtlLge0zxV6irN23aCam\\\\\\\/pazgM2tCJUJ83gitOjXXiU4L\\\\\\\/lRw2L\\\\\\\/HarH\\\\\\\/NqpCOC0Y24JXbqiybV3O5fvLmQYYDf2nBZOfPPAdJxBecggILn5F7YJ6MFhbx91EE+sWW4Tr6yMqJw7bzcvNh8xiqhWjcFWNXMLqz2Hmna6eOI13e6CQUHWqrkpEW\\\\\\\/Yf21DheKqMioYHuWwXbAiNsBDQB0CEYQixAr\\\\\\\/q1IEVUxII9hpHyrfv8LtjR66TKsgQ6OQSAJi50KPRR6vFc1cNHvZn+ZhB1JX7ZnLH3GsQGvgLaRp5YhVmdWHrjn1VOwS83uNESlszpvbXRZiYve3bEr36f+YW3mp84IVeTJS5g601jVdgf27bEY3bRQGtzSN07Yq5F\\\\\\\/n40i6NyWBc5S0RtFaHNm8iegPl97YVjwhi+DnQnPfsyr\\\\\\\/JI8F6PZswRTVNNd1g8pnWDCElTkDZe9QVjc8imQiyb5kDCkI62hEWuZtKADrAVRdbZMb7u8xI8PspMan5v6JjHaLjXvBy8R8\\\\\\\/o\\\\\\\/0\\\\\\\/woqk3GcJDrbg+wf9abEcGKlC5qoLOEqAi4SEiA8aV3Gg1td2fjVWpjZigcWMHGs9gjbwARiGFYs3BiRjBCVjGGmTf4i7Y7+lKZH2R0fSQrsy+Kkn\\\\\\\/GKZO+Du7jFetcitKdmMflawiQIcZ0SmYB\\\\\\\/XJ7U7mU9LuPizOyukJjiqn7QGgw2p8oCQahWaJHaPvbHMzNka7STqghKTo31f41XirelFMvI40RcBPCshSH4Jhx5X\\\\\\\/kUmQHyHLfYXaZOnZGaO63COmKMv3YnS8cEz8oMflQKN8bsxsXPRBKwWw2r01EKToDs9Tl3O8ZdUbQRnFlGIxgI3PgSOXPOu2uI09kyr2MCF928L2J5pkmGDADGZP9IwQ+wH9n79DEKqaGNLuRIikEAV+VSVvrHgZGbEtYoC\\\\\\\/QdkkZ\\\\\\\/Ez5I9Gna47pxHOmHlz+aiUFoZ8DxjnK6e8C\\\\\\\/H6AP57DhGYozwaA1RtxI6AI7VBuX1ScO+NmPuscJe1kDI3BanLc74yEm\\\\\\\/f4tj71sGSvfbN6jzlvfYpMz8\\\\\\\/\\\\\\\/25AFzzfRxY2NtSVzwiPQBNkwfzS100Tahxs\\\\\\\/ZfOq0he0Ye7ZdLa930\\\\\\\/bBDsLmnJDWYWKnlKGLuCkKk41KPPz9BuiiCN7dHNUESjT6DmkOT\\\\\\\/OCMtkUbx7TN5z1+CRk4E4r7QfN+SsabKmXOGMdCi0vb3CqZozeAvhfOAsc0BUTz\\\\\\\/2vn5XOLlT4wYzPWssKuY+nupVcZBpcMg5FI+RU8sgYKrdrZSbK37SEksNYTd+6o1C8V66TP5DCSwDtSQpZh5qPDiDfImO8Uhmd92PL4t7I9gXfPOjH3+BoWBKiRSTdD8Gvzom4eUE5q9TlDXscyGSnAp3zDnGLAD+3t+ooNQGKkBrK69mkkKXRNHOZ1L30ou5ulUqnY\\\\\\\/zfeUPdIMLT46oJizs2WgPwLCGdkx37yWgXF5I97WbjFp1drEt6YjW2Zjg8Rgfcwp0hBULPeQiVowtAWAL\\\\\\\/udKof1C4reJvr7RnNlbmhiUdKeMl9VwG0QSVJtf\\\\\\\/fd+bfDMoqB1uIhIqtz9SWTaz+G5qzUWfAG6mssK0Q8WmWbyiBGzPtVK6aLSL2BiqTw480pwF0j2Hk5HkbL0KzGYwdGLLi65YBMqoRVgmtEzhMGUPnRycmgokg6tP1\\\\\\\/7HKYKWoGH8r5PdtCxKy208lOoV+wPItembdNnUs0udt6ykTHNuFoTE6HJtRt46agz+tTZN0XRETYgtCr8\\\\\\\/F4ibuDBLoOcF+0HKXdFrh2zvd+iem3iiTbSAAfuIv+3OtTzfTGGCqmJwtwbXSSXLJeyN3Y9KcGpqmFYys1Pl9Bdu+jC1zbxi5xj44cHZvLOg63Foi87HtquNEo\\\\\\\/+JcWV\\\\\\\/gOvg6Ua5aI3H+9etHf8hY0oyGH1014UVl4rUXlu8vWmmgSNHNepZoKXCUZIhkSGxwTnSBUEpL9i21SkyE3\\\\\\\/YZvEVl04aEBAXw4GZm5XDniRiLEDG7yVFoHZHN5h3aQi7z483IJUhz\\\\\\\/Iq3wzkJzUKYcQw6jp7wupX7JnggtYAaV0De5sDaUy+C9tgNucuMDfgikx867UB7y5uWARG6fP4rNIcpS7EJ88jyvR1nV+IpsIYYS3BFAhAdwhtse\\\\\\\/q61SGu+Bkqowj78xoLBpSfMMpT77nXBRtux8RdlufS4QI2prGRiQUDhxWzMaid9HXHMqczIWB91X1yfuwG\\\\\\\/FciTyWrLAEIWNuzLgC2DsJ9NkQWzC5ymFQFCO+17ib6yYLpkpuF\\\\\\\/jlQIO9BHpHDpweLv1oAQnk7hYbTH1bUasok1yF1NL37hD9OCdilWRJVTD+vIBH+Eq3PAxtoHHZ\\\\\\\/TVPhbizyTBwGKVr9r6SeNIt8kty\\\\\\\/rD5ssm\\\\\\\/mzpX6qm+iMIRrTKcnsyLiUrCajIwVaAcfMaIOb5ERfsjBhw0euprAvX4ALipqk6bzlIf\\\\\\\/ws1q5VTpo3StQJMxkt\\\\\\\/\\\\\\\/q\\\\\\\/0tIFC6iB4jkbMPxNdFuJhqfmOnjMXRKeHHGbXGlHIZRKMrTAVb8c4vAAESsCvWuStkKnbBQldAaiC3IwYkIPUsoEEWnIIQzJ26hJhFUR9YeyZunpAy3pqLmGiISLvnhwBmZU4QpXZshAUsD\\\\\\\/KxZ7evRk3p48fmBAqxZACUVclmALqDMght+U+xNJkrURXredHXZ\\\\\\\/69RXos1slxn9AOPlGcT4q1KU8YitUztBFiPTRB8ebq842PjOjzmqxEAIavFINmE3dFyvnnZvl4tFdppsSsV669Ztpdg1q94rpxf86PXJBHSE0GvcF1Z3pT0iSMSj5FUbbWzPRl1pblTIuHWzV8krh5gYh60rxQB6t1KkWpfJqXQ5MdVB1ofiRrrXa7pZDyJmV2zlAFiYgLhUqcn4Qm9PxS5H1WiRN2xqW5kl\\\\\\\/5ajaTFvphDLPceUnRZIsa+mfibp6eg1dUB81aVxtDOHhfq6psyh351zInsRF1SZvoBQqrE9WS\\\\\\\/bzZZH18A+fIrCePkxLEKafySTpYlT26glsf4hZo1GmgzixIZXaOPA35FhL92U3hiGsIjewhcVISZhD5K9+xoQ6BF3MxBR0Sio8J85In4POojg+4wJ4IdiSZUjy1BJHBGMCp2oluGm5xwqDJTg\\\\\\\/EQibeYEWTh+Hu4nE82kI17Uga3VByOqEoMgztYUQ2VA5Xcn4+PLvCn9IzeVjssObv0Omsf37jzFQgE4a+\\\\\\\/nahgCiU67vzjQPJ2UekYXV4htiFZBgCpXogLQWzwa8oOHT71zQDTrPEOPafHfjhFJ7rRUW7JEDdeogI3WkCykuMPxu+BAOYYpeg1Dk10V4g8ZAmcFgEgDg3xA+t2DAO3imcm7rkEClD4bHuGNnHTx237+AJrFZ99HapBonB5avMrbvlB0GQl9wdmidUK1qYCBDFSbGq7NorUdcilyL084Nxkc6GIpc+Kf8ER9SWySiU3nROnOzpFmDPnJJFVT8Y9O9KZeRi5pgSr3P3QVtgsW8fODxSMOAutgktBa4rqDgLckCDiF3BFHMSdyHS9dYPqOJiUrGcLaQoVmHg2NZLd1u0FIiiKTek1PQuhGsMjTf9qXC4e\\\\\\\/GdMQqrbynmI\\\\\\\/cr\\\\\\\/gBVojmRlsPrlLfOytDfcbn6qplJg8N2yKMc8zKxYy8BqW5UOqjU06VbIOXJOQb43XUiBRsaCyCSI1zDgVZkkCUl\\\\\\\/MIurZGRSSBZ7OXyuk0ue2Ox8UDG9SYf9clWXFTkbCcg3k9LtzYTcN0w0LtLzd766nl5OtPQopWE8mhhRkLLDE0xPIM2HjEIorA2IWZDGZHwL7MFVz5j7h3jCovr8dB28vSZtlV4Gugc9X3fUASJVYgYReMw8MPI71WpXWUBAnlGsd+G7VlZd7vG4YgPcgk+lZvXHMbMF40\\\\\\\/H\\\\\\\/onr5uRby3NHiwdG1G03EZNMg5vGceL8yiqK3v0y8Koloj+MGq0cnSeF8QlHbrgCBr8ktUVi7D1sJ\\\\\\\/Ox0oy69zxxD\\\\\\\/8i3He\\\\\\\/FgGSWDzfOTbB3\\\\\\\/AAyfWbFxSmjv7ulbpXH\\\\\\\/9yh5HG4RG3iccOgqeS9KgqCeQ+oBf+6AQ74k33OEo46bCgBqZ6iRs0qEnb6qRVsjbxhcXje68qtOLa8Yg9FQr1vPaU2u0Sj2XkW8hK0x2vnI1515uiSxePX1FDT+4b\\\\\\\/6qXDoCvUDUeewsvLmAu2BGyhDOUzVLGApmZYgq5PTCu+dNm2ABLRMvofuybMi8OEoT3kTmQn9JlV0Nn14fVT6nRkwlcj2YJ8fz4O6cWQUWqwOQxEd6IyY4XNTJbuG5xGNeqv4gMuYN8+vkFqYm6UWGTzM6W5v4+6zEavpzY08idqW9qXGtXKBZviaZkHZOJH7E3gDr5YKXtqUSBOkjlYNBBH8O3GrTPoFSFFMbmQjR4Y02x8Sbo9CC9RoQse2Gj1LZDetHkx3Ot1c841lfOkWo0n6WDSXtSzlZLrRhJuyj9OCDa8CtfqHaB5mXzHdnDZnRvazphDMzRvS6kbhRge\\\\\\\/cqqCRy\\\\\\\/Atbme3KE55CJM+1l4wumyYcdkft\\\\\\\/q0kn1FLfsUuV+yOdd13tqUujF83XbafORilZuOTp5iStn1+iYZTHW43SFhZ12TTbP0QW6haolNUqNeC6HIRkV59JkWn6bkAL\\\\\\\/F1ikbrpwEVh8CqRBUAdVBUeFsju7nHMs5Kk+5pNfKbyQtOhFshyaxGdkqfwMv\\\\\\\/atnjaSTVDOYJBsG3qmc4u3M1gV1UHi44u\\\\\\\/17TqzSRl1aFKKb0v6lF0WLgF4xsqL0IMj2XzzzC6UZgksOoYl42lHHKBUCqpI\\\\\\\/n19ntpIrWfz333GWWp0x4542TgZc3fRmLFuTR\\\\\\\/QUgaXCV5YC51\\\\\\\/Ms9Dvg1ueaMkxYKWGneZyvg2IqP4wsQxSoHBi5r6B5FQXojGZ3oWzGkoE5RAr\\\\\\\/wjxKNuOhExnDWShKzBnSkiuo5\\\\\\\/Zo7QdSGOMJBYoc0r3oZRsrvJ24r3TOI71eScLHiXBfuUUY2vGnKgSbUnZy3o9zc3fJ5Ysp959wPZjQVN48BGhC8eqtefa8UdbE7ckyXnPKJJvfPI0KnhjN6Iuebcf0TOoUoT\\\\\\\/m896NtUDRXabFreipOM9OJ3b+NRs8OswIvv2piQZX9YQS9JFSb6C6bjnsO57SzyXMz\\\\\\\/JO0Gi25SIja9SpR2RPaDBXp1CfC0t2QCWv8PEf+JRzRTZVQdO6NyRfQxsFcYiiT27E20trLuyVXvd2j6C7H4+L497hPMwiSxTcwlaOYLi6n\\\\\\\/js2H85sOFiwHzQ7uEOZ7DOissopEXQuJCewicPOgPw2Y0Ig+s0c0HHymiuB23MbnSECqtrdARyxpfK5YMQKucmWq\\\\\\\/pFtqK5TIXBM19M6dWtMiwoLmZ6q7QB4s043kV2JzfdjPjvMCoEOK\\\\\\\/nlsI1wV19pB4djIyzDYNfnUbiAGnPpDBDdOc40LQvqWEPEzdxWNCsueyuI3LI+9Vdf5b8M6tuBAVjzAtOa0RaSymNsLCijTccRwaOX\\\\\\\/8Gecpm\\\\\\\/L\\\\\\\/c1\\\\\\\/fDoJ95N0+nK\\\\\\\/sbAx3GWhG\\\\\\\/582Rk5r+PppQaK+7Uj0yNalQkKH6WRxwjXe8hjPuIzZLvdFvTH7mTRhZuDYryt7Agx5QabruphQ0+MfwyAJjzEiFwPSfQiXH2y79qHErXiKR4tNUVAjfcy8caOEK74tX+pacbrwXvLkTDBkmZKi8vBNUMpFKwT\\\\\\\/UmI6tgd0f\\\\\\\/6ou2cW5qoAYX\\\\\\\/36ZSkqNjAGJJmFIYPByoD2Kk58\\\\\\\/gnvnksQtFrZW0MSeyPCY5VGfSzz1tbCV0S4Lt1dZG3z039qsLQI9MsvNIcICmAfh5\\\\\\\/45RI7JtZu0pjD70TJ+biccVQ1lj8d3RXKMNrSNF4h+l4ObabJhfIEEErX31va\\\\\\\/0hDM\\\\\\\/VqZObr4GTesAh+ujYoQ6FWWr6wa04iyq3VC4LHdyZvBfwmS0RCd8MNBLKkV5gvJvdgF+BrOmWuuKmr2jBb\\\\\\\/lFzp6oJS1trY6hzL3mivYTa9N\\\\\\\/tCQx\\\\\\\/j9O6\\\\\\\/BPOoUdO4LMdIVGSP9i2ppYJtkpWtd6PGQm2CWuZZJKq+dWyZ2JLVkE21I2CxbgaD7RiLZLBugFAmrVTSrVTBPcJ2W5XlNX2bLfb\\\\\\\/DX5wpnsTHR6Q3yihkeyR5ps9fPK3Wujy4WlyYljpsAauv+3HQvZjRPryhXPkRkxxRUaVR\\\\\\\/zj1jgEL06uvmXBYfPVhfnjAZ7qLLQfApYMiqQtXtaPcyz+EW+ir3aDPlVn0b\\\\\\\/L3dsPK7qaEd3LZbhlFrVnO850jNPXlrrST0uPIdewoy4aCrsjYthviJWUZuEIjDlLGBu1qYsa0Sy5OVtdFCgLZJEE0LsKoLJF8TYL3W7URaPtMU2w0Uqmh+Ame+UkozqyLBPcfyXcW2PD1n7qqsi4XUiWKs90LUb2hzv\\\\\\\/AWBuaigEnShbB9ozeDYB1NKr\\\\\\\/pmvUUrQPcaVdCfRfuSUesj+EU6U5rCugX15e8pkiUPawyd4CKB07TpUKl7n2jyLV9EXocyc1VLZa0HhLrY6K\\\\\\\/LQ9cRG4AhuBQQ48BnbeDIRs80eJYXiH+yvWCBgJTocgR+HA0DnnOob0YjFif7ojx67WmW7ajsH+cDdJb+ywUAwGS1ACtCC0A3AN9N943t9YD1R55MrWGm5i9VaGRtCfpaSVp9W+0bsTEYuhd5VQRZic1hBPD1ckaogCTGhr3TeW3bwlrcBRKjRYgY13twvEt2+4YI0PC0V\\\\\\\/iJxTZu3Nd0QK5rctNYkoKTa6BTEeOBexOt8t+\\\\\\\/7cb8HkhGnycRrcu4gAX0fJzWbZC8CCIVKV3ZTqj+VTTK+6laKibKdHcYvXb7eQjNEQozzMNw+IB5XW6WEimKr8GSx3PBwYicc2GQPQjXUVwKGK13EarLaj54ZnSwzvIFjiHsUbYitQk6M5dTSYy+l1a0Dzp7MSxy2xLoUuMl1Qwm3I5GFDHLsdGPUoJG7x+6LL6Ik8kW2wSAa3EDJB5+Ky86jPEn5OE1kj8v7HLGCzuOWuz039D3TDDzsOkDUojUmk6pM4T0XjhZcRwRN6hhG4jAY0O9WMEUjY\\\\\\\/rbNUDHipO+bFINHiLHH1KNz0nPozNpBZPmumauTz2Peq1ER03j74\\\\\\\/TBreqj+kwYndMBCJrAFkPn2QKz6o8aCenh4wY6S20oDnb1xFbcIp62Wyr8MGPDC7lml72yjDNsRH64tNZ\\\\\\\/ODP\\\\\\\/bH5Ms9XJpIn9lOYL51n8npyi6Rvgxfx\\\\\\\/7DmwondvLSOp55G9W7wQ7XBIVePU0AFCUP+mAy2x+4i7T9MYPALUXtBI4R1mrjWEGup3PHX+JyyWv1FttXKFbhLgsjS1Nm8F7LWsrSJqW5+hJygk43nrHAVWlIKQ78dVquyOZXbaMf1xppkiMMNDC2h3NiAfe8Xv71CkPOiYUV3IcaJRRwi8vT5sflk+LVoKxqGb3b3VRxm16YVWp7U5CsF9NaikXRQf7yAywH5GKg9MEL6XbEiMf9sE2sJD8LTXnYetvZ452Iw5GsttGRM399weh8b7RUxPx75O0xNL+ZXO36HDfkSkPM39oI3VKVmWVVwQAYQCB4bVbPAR8ZgBwZ+n2T6IJpSDli+VU2\\\\\\\/vksvFVNu26crSC3mg8yckQQixQK5jhv8PjH+5bcoPicfN+ZISGJcui9xy0Mmek6wA4vA5mfuYHQl2ieWQKE3UJeN4i4ioIzumihRjrB3CeI8J8eTbFnc2GnRvvrs0um2kYv+T2xZR5uHKkBirzFVPqfX9SQ\\\\\\\/9I+92q9YZq0buUvlYBlMzZEujI5DBB9ohTEmc03UVnma8sSKb6b2Yq9MmU2O\\\\\\\/lYakDZNew3lejsXu1FxrEACLCR9A18PhXOpUke+NKSgl3r2pcd0qneTXD5hAFJSZ60PHOtISO7lb6+dm2Zs6fKO0+SfGy5sGeNwv6xoRWdW46EYiY+pocnjiadmoqZJ2E3OIYaMCsDez3AFM6HL0UnRmadK3cq7NwXJ7i\\\\\\\/O6d0O0UXpgcnFlfYHorzfDro+uc6ANk4ASqErCqcT2kU6z422ZzemrDQ\\\\\\\/RT\\\\\\\/nb3DD\\\\\\\/vM9Z\\\\\\\/tPWNOSBBAj+rQC+ZQ935izw78DQzxvwsJ9vkvgFtdXgfNvgEO+P+PKK7\\\\\\\/NdefStZSlW\\\\\\\/LBrdflIh9FnV8iWcQwZ9AZobBVGxF9dMB4rj4ncGWA2Jmva9lPnGwsl4+VH3ELLO1BYdyfmABeA0CdqozhN3xOq39UjOZmqNT1swAxGU5Fch7klDhBk0WsMO8GsxY\\\\\\\/DOYvoqyBz6SUFODIVGyNXo1SztxwrJqC7Ksnwvw8d\\\\\\\/C9kqa2e86cfIs8ILVGHRzj+eETt0onY48vcFyvMll\\\\\\\/LPSA0wn5MWhRGguV0RpHqMQagBRVI2BAvI9WLUJnTEAF37Vudym0u2q6hIoJOyxn2ttHEwZuKGWGK88HhS3Nzdrz4rikV7hvq6qpachxDsQA4AgVnxBUelFowK0LLmje7uQvySI3tmgGiVkjglS8jUnien7nSob620lrHyrItG+ysJZIEoHxMajgUKxOjMzrL7O8fRIzL+9KWBTJckFEiLM\\\\\\\/dmPjLRX+twq\\\\\\\/OK\\\\\\\/jkFW3LYxVk9w6QxqgErcILHq00QHbU5QnO+oWjnB4a2Yh6Fko0ph88geB310GncTS5scuyuWnQ4J0T3EHJssBIkvXpMBF2anz7pPU3ytOrIDJa76ywjC9WjFwOaFMGBA513Vl66+DxnZR2IDxxFkm4GBApGlCFCDddQyGG2wmwCmBGQECKYTbrXnB\\\\\\\/HAH7\\\\\\\/5xz7lmMbHMHMKPyosmaupZPYmqqlAkRqZfqlTd3V0REbyFCm+vQlT\\\\\\\/zytmxUI3E+HJSjtj+\\\\\\\/OF\\\\\\\/SB5+6zHg07WecnNuPW25YzCwbslYwroCg1\\\\\\\/UYWBP4IIoxsRqol8YBxAttDgZwxrGOIliJKD8G\\\\\\\/v5FKJxpCOz6fVaC24TIt+mqs+fmEyI0S5NOFpLj+bB9JutZ6BFHT1e9biWU8eNaHMsUvHSf394dwDH0c0+4JQ9TZgHvf\\\\\\\/zUQWkuRVijrOBtvNOolcCenEYnZhywe1QtW\\\\\\\/55EDrJ5EZpfRQa6pUjNbls2X063Kx9ABcCZN9pUSjxZOuaYD2RFNNw35kioY8Ilu1\\\\\\\/Nqj2UuuhGwMMQEIwZBN59tA1E+6w6H4mPDOm+CeTv6aoNAZ3M2gKN\\\\\\\/9lf+Kb86FhpYtplBcl8fC4Cf37F7\\\\\\\/gu9S6gEoA\\\\\\\/i5KJTSAbaeAHOKgw6gEGzQWagFsx\\\\\\\/8BAZbDQepHY3NcClEQnfFtaGS8uIzjhLCkZu7IeFEixST7l3jXF17+t5WlFqHd81vQX94afoDo62efjRWuMBS+r8LDkAcLzsZ2VTxJ6zES2tTqFNu2UgErzCjrIZgVK0gmjdwvao4lxtzsEPgRPNneGZlUzdiVCYd3hC8H6bLySMrryEvt5Y+6NGSGuaz9qlt8SYolvw18kMDuasaCDs4J+1WqRUxx90Uo72PtrkkUv0UycTMdK9ezeWlkVPJxtI3yR1E33lh7f+JQ9GKNxOUU5dKFachAWRXA1ZDgb9TNLyOuTYZW+Ysv4oB3FKq6CfzEm+afdrzu6gMskVcZnM36Ty6wtD1cayuYUspyrKH86iyTvuK1Yb82L8zu1z1erxR3TpeQ5TyoDCNNhMHqSfJzLddYxpiXYamW9FzyADtlbmlyQaf2aWigVAyqiaxQY1iTrPIcpqLk4afQSTgXYS2fz+Sdc4iuVkDNhalDBfjOpemwSQXJfTFHdD4zMun1wWmyvVvvTrwRSKcQJqdwGrKNsAcxmMF72jJEoezH5t18PyiA8OLrXJQaxRb6uW1ySvYWcpMHr9bdglCGc76uAI4bn5NwdFmrWmCltFuii1qU9KDDslfmYyEzDaA4Sa5fniltxrY76HcJRX2RsUpTTwswhsDRh\\\\\\\/i+J21KcHS6YYxC4DhFvSU2Q06O3Ja8aUmdmwpjxXBQUA1e4x0LWHdU06ioIouJhb4eC1SdfLoU5QhwdIvFslaaj4PDhjshM\\\\\\\/d9sX1UuEThRZfHqCAuENoZRvyDYQQJDpyEL\\\\\\\/d0lL3YMqYybbNmEh2mVWwIyoWK8yo2eFz2Fa8YgY2BbGDUSTF4iQYz7vyk1leEMwqazx3ONnRJvkagu2ww31fyadWyB2z8HkHW2CIp+mI5GMmR\\\\\\\/XDc26JZx8bMr\\\\\\\/HL\\\\\\\/cRLpwcgPbYWpb+U5KC1Or3lnlLE8y4xnx0kAe02yzCh9t4+n\\\\\\\/ZryZBzo\\\\\\\/iKZovs\\\\\\\/Qt\\\\\\\/L3V1166YEln6JtCSQu5XEIMd+sGSyf3YSn6YK4fw7D9dR65R4fQN0zh39mTpcLtjNy2lUIVfdaMLoSuY+9tHF093Uz5+bik7hpaaCcN6JBU6wSu4ejLVCcSwlsyhexONmQKUZsjH5PSX1NpGH2NbrZBXDJIthsppi7BJPiQb9ZqpAPLP2gUViPfvEX9LvT1nrKysOUNkjIxo9LY88gBG6QNZBMgr4KT0OejgSL2pqwkkxk4QATXhG6CZJsRZLha4CAKwZ\\\\\\\/5NcH6mJ8Ruo4lf6mL7zoo8\\\\\\\/iw3lvPg4n8Hw+FMeq4rsLnySXEFruszArCS2WqSN3uXEWEob+Dnn51Yu5PcOtGsw0RYrY9hLSHjEr+LV2sktYT+9FUUKvpMwA\\\\\\\/M5DKsjZbvv3CWvuuiH5CH3\\\\\\\/3DvRm5cmpQ5LL7JllQpMI6G9RtbdaAjjNPXdTcYLpx7oXk11X\\\\\\\/mzzbs6iVGdWef1FAkUVVOeKjgjmo+vkkTX++YpTSg4NrEiJ9jyJ3Sb4kT\\\\\\\/CVG+DWEUZClRyzBDwnR+A74FPyUPYcLJRa98yiJstq8CEEU1tJtQed\\\\\\\/LdjZhB7GSvvKlgB5z2CwzOfw6RjWDuWNRcRnHKE6jqLy4lbXyoxVmk3+qQa37qRAptniY3CaoLdmYMsT7WGH9WlpIhSC29LzZZ6xpR9CDtPfX3oJiy8M+5miZxKb22WnLLaoXqOgInZxObGVIPNfM0mrOI4gV0WxAXbjEyo3YUza1K9ZPLV2uTp8xX1RDNOi1CWMnKzVkHtmugal+cTEeNYSqLKzJrw+IrnW8d5ay1abUWCG9IWdZSkfbhvy0V\\\\\\\/lFNbeSNlNTE3pqHJxpV5Sx7zZvtMMaDfP+dfGU1Ykys\\\\\\\/5LhhWBtm1EFBRg3RnflmwJcUnGGUIz8abSt9FiYssSzkEoHLMpShW4gErFgLwYKwxnn\\\\\\\/aLRu2Xvog+mKebJT64AIrpb+AChTxXCt854+jR2X3ESD6F+yCOsyFh4ciYY1uP82hBXlAg6n6ur5mG2EQpqAv4T5Vfn0GGKNyhsluG5ZvUCNs08y2kV9tGUjVMMcjPrY1qWWwI9j5cGMDxIFNBwzAaXAqWM7aoay2MqTMt\\\\\\\/DVX4u01HzEFuwX6DPd3AyKUAt3HZEUc\\\\\\\/SxhuP71t6OR8+9Ooi1mcOoIs7b\\\\\\\/WDr1UTN1bg8B5Mn\\\\\\\/XWXf91hr\\\\\\\/aZmmFq5uHRK69M0Ac8Ztuo69kiOEXtYIqgvrOYjWneM2zRbXdRFJNb9Qbjmdz4LMO7QJIMt8U\\\\\\\/YK8QL9zcuVibINLbknfE16SMM1O0nTMt3TvqOqCb7YC9AL1j\\\\\\\/HvpSzpZIqeQbjl+VXy7yuSyPD1u2XdgFmUpXXXQ7G2zsxKESFtrpv2b9Szndf6hBH\\\\\\\/FsWk1Kt7BHXJhqHFvHKVYNs1Ltg1tnktH2k2sul05KpkGIHJ09CzdsWZXpShnIGuQwNBajmf5apFbEFkQ5oFXKQ3U7LJyXiYEfNHjRJfNu3mL93jg34w8JHWDJgSYK70mEMSkv4HRVD8PBsykAfYS2x9xAx57N9w2btit4XIWSAPj7vE0mqrTUkdFtoXRZx+HK7+RhRGuuMS3IgnRtOdmc+mYa9e+E3jvGQgjG7iKC8WDyjKVgMAmDbC4nZmONalfDMQlgCfnEQ\\\\\\\/9wn\\\\\\\/3x1e7cdjmBDHe+9zAgEZwUsNEgL1R4GPs6mD8hB1YaY5lVpWyfsh16E5TeziIJQmSCbLfMD1YmWjJdtLvqGkUCw80hzJkrlszhBJIUAYkfUvQBQCKdIIITxtcEf65\\\\\\\/FIev3rMmxrsNuOoUGCPSTX+2iqKjx2tZaeJ1epLuKj99Swt4IhGjRGHXZegXzlrU6WCQxbKianzf+aWvXNcvw\\\\\\\/TwnjPt0fKHv8GgAtAEIIwictANwO6vHiQYlXse0CTwgyWm0soxkGajPtKq5aV9a5dfWdVS3kM7nABYRneICSoP3Swg7yYNHEJq23LuGfN+bLcLGBEnRKu6p8r1AdnYtwgUnwFCkxlN9RbU96P0DZo5OSBugCbvV0rSSUfOojRzbIjlchyN7DkP9XHNNZYTdfmZvN\\\\\\\/5hteNgDZ1AtHG\\\\\\\/5Kswq28Beldc+jRO6V9hZXfymJzWY9RCKv06HDvKyhwblJq5jw1GFuhoaGISG9TMd6JgIvY4AHyHhAAXBVmg5eTB39OTwZU\\\\\\\/ODM6v4ym6UKxG47epcCh06UX4VqwUJe3w\\\\\\\/znTFYv2kMbGnZochzKzXFefyuEKzaZ2MG4U72MG3SdDBtpSfZTgyIwT6WH16uMaYBw8EAz1YL+jsJEwb2bixNSP9gEhTGp1deDfWPSbi5S0l+oE\\\\\\\/C0is+z0q0Rt7EhK+vRrhk46UBLlKmrsQROKmP5\\\\\\\/f5ZNLruCOfFHBKBWf7dzSicByIeHZvwcO+Wm2c9M8383vclWGxA\\\\\\\/9eoEBpa9Qp2NDN1e4UV5iQ3m\\\\\\\/Wi6hSnhV\\\\\\\/7sIvRtWvBIcN1RRvwvb8Fd36qBuOs30FJW84bMVsE7NbnrRAPRlo2gsQR3rXWTk7gCZCrgsW5ZQXzQdLxa4ETwaO9E6YVQN58Lzet69ZMdcVK9TLGYCkg+R7RYfQp1I9qsb9Bf8cmi6A7Yq7CBZJkFojRaDTa4dsbvGKySItaiXnXGIZ5PExi4ywRRUS50cU6UOeI5blJ1t\\\\\\\/QqBJ0v6AZTWtJnkWgNVAr\\\\\\\/LCgk+qmvdKVr3gP25JSyDuPg121xxx+cQzKlO6awNTnzYWwz1G7w9wrdmQVf60zUKEGryrOx8D2EhEX3HTil7G7Hb2e+OfILLBTE4QC1\\\\\\\/UHckZbsh8h68swnAQhtTIGXTbw3GyHFyAYTGdSM0YPIdfLw0+PB1tVO3d6UATZ5SGfCTXe2AKJHmGZDB01dg79lDYIxjwvSXMYrCNYZnb7KhwfnJAZnL2mVHWZ\\\\\\\/ivOs0fmnaQw0PU2maTQDvmwJR45ui04NHVN3yJUT3vjvcfNnmcwigU1gz0U6lZaqifr9MemCXY7Ampy9kqHbfU5zOrSqNtWCKF5qtv415joetOsHELfdCuNWEla4NLRv\\\\\\\/wKnXefdu7N8Px6TqAYRVHEdVz2sUTQPglN9wRfXrPnHm96+QwBHHkT3Io6XVFxe0D2VcUzlqWoDAecF03exLNpOkxs5vDix37y3JM0\\\\\\\/QWQc0igUxT++1L2rFO\\\\\\\/FaDhukjKAdgpnTQa3gcgl2sK0TzCa8y0yY6O+oz6JazVtfQctEzJ\\\\\\\/sUMNu96tGW170Gc93RfziBlsvw05TaZA5Z4OvDrm4hRhRNgkvx95SrpXS28rVIrVnPVpjVwsEF4SmM5YEFu44Z8C5xDq68uIe4QPlP9JUK7edQZIeBUegHFWCYCCKN\\\\\\\/54y8SKsy2XlH9ymMf6HzMB6x+Bdp+z1jKXf6yxIGGkVdFspMcAhbep1v7jcM90PsJ4TviDcwP+xEoCPfyp8EeDfkRPQzMZs9alhYOSz5T\\\\\\\/cyTH\\\\\\\/ciwS0A3\\\\\\\/6edfI3RB9OqBJFxaQLr7w2noGrsJVBPA1wNVRPinwaIZJzIJAWkEjxp5GnEPnMr8czzuSGvUriXOXDjr80kW2YqjpSbwtJPPzlIDivCE92d74lMIoa\\\\\\\/GeBPqluOhP0jcABksnnNvZHd7vyJmO6UCxlzqLQjpzmzmV6Ef2KgxIhPL98RblFLZQdCjHbvvjrpX8xFVLt\\\\\\\/j5ckKYzDOf2qw8kZLt2JPT\\\\\\\/xlCXReQ3ArwOMG\\\\\\\/wKjHtgODfjS4NaPWssxol5Vsfb54j4iU1hwalp32l+uj+8urzVDeo1kriXzZWuDvHB5\\\\\\\/xxck60ygDwV\\\\\\\/VH4pTJWfjPI0SFUuvoMfl+eyxG4lr0eU38F+lw+Z3yNz\\\\\\\/3D5rfoEjsIWYi1ObzmZkYzHnZRZS4E78WkYab1l4qvXoLzwwdO3bZMldZbIOChaiCoEERNUlEyepeciFMlsBWS2vqJt3d4JW8fORZ6cmEmcT37HCHVUuGXon3FYpP73eA2gGkT7X1zHiC+FPuESOOgDEugJDk4eVhLY1nOJwx3I+cIbKECcudjZ9p2f5LLKdJ2ZvQg+GXIN7J65J2h6RMAeZUTKidTlrCrIp5hupmxuedc9u\\\\\\\/P1+kzPgpmUaGytIJs1s3Ir6QeJm8S0YLZokRHTC4pdOqn4VG4+mQr+0ykgM4YepUEEF8MbhRqwvrIYZHsS3WHQhfOsLoo0k7TngESN9eoUxX1WT01+FhT4GEinh\\\\\\\/osJWrsUXdJ+Unt4qL8uCHq7xiA5IehSKT3oIx619BqSHM0bHUzhl+H+glf5e0v\\\\\\\/S2yBjQu5xgNdtkriiyQY1fe6NfLY2fTyBnpoBwKQQ8i6WnC2uLkhm3d3UwfprHNU+mhwnx3\\\\\\\/Y14\\\\\\\/cA7vKikTXrdO4II2eFbKXeIbBemT7stsqElJ2+PnLsixLyiRpOB4eTu8\\\\\\\/vQOvC1VYR4+Q\\\\\\\/9haM8G6FQlyw\\\\\\\/woytDwfti49AA9n9XYZRi2JL5yENtwksjKpCRQjEhnEgB4nGNSu8dhh8\\\\\\\/oUpiXAUor2UOeM4L1nWUxsUtYrOlpMca2dvUT+VkPok5p4wHgGXdVji7ky1LH7p2\\\\\\\/XL5fgDECsCNuhslk80NCVYdnQqwsExMCPjA78pMwBxRFWazcIrt5aRG9pnHD2+3wIvSZXTRo\\\\\\\/QO7pcbLLIwd8v1i9gS05\\\\\\\/gJ+u0rZiLk4XQZ8dM2M57SNMerga9O+iGJDcQVVrkmTRZ6LotkFBlxgUfarJ+g6SGL4MGI1NJUdi54IxKKwFFrJS+wvPm4+wpHFE9F7MlBAmQ5+c6QL6Dr+TfMqv4I3cLErW26NHAB5+crvSKIK24SgjTNdkAhouEyQqet\\\\\\\/v6e3OjYTUsipOhyg+3Y532vRNjPB6rDyqjibMTP2UeNGFgvhlFcve6iAx9iqBuH4CLhJ1uIrGFzpSrw91ntupBELrhn8vx9AbOUNH7bGdncggxCTKzsjx4yuvmZmGvKIMll\\\\\\\/GossmKoGFrrU9Q8hq3R46p+vXYO2CbIjG7LendMZ2j6rWtMqlD9U1uIIVhmfgQx+23kg1NyVioc3Dy3rETmvKYAJh4TnhEvE6Sa\\\\\\\/4IO6kGUgnEeaE+9KSSUg\\\\\\\/aZJHAEFHkUEDDcy\\\\\\\/SP6jam4bsZu51xmt5Nzixce8pbvjmJMNW01Bf6ZYYZg\\\\\\\/KaodlCjEQ1WhOsxYukKBc+vvK6+pCpTovHsBDbJR9oIDcZxTbQJZ+CzDb0IKY6TPFqyU7C9UGz0qUHY09N\\\\\\\/q7+vHJS16hBbe3TsgMvQTaFgsFiAmO96KqGpCxnHjJxy\\\\\\\/TEEJMR32bs1zOIL24R8z0i+Ju+8gJnhInMzRNFNm0x73i7Tc+lTRaEbfmxhez8zoXHhw906iVlQs\\\\\\\/maSOHYfvJ3XjbCNhU7j7irLNRC9bLlZ3FI2EkZAmlq52gumS7QDQXbPVRQ+1tt8R\\\\\\\/XMzJ0I3YtZfsv8FDG\\\\\\\/HFAC0ym9ppW29CmnkkWo8OO4j+MZO6+FPODcj27oPV\\\\\\\/QPmzuYPIaT5JePz\\\\\\\/4N0wP5GK5iaTPHvEuq\\\\\\\/YaiM+PMvn4tdaBABmYEQvcQ5SyB5f2TC+9k8APZ7XgV5SFNMtkhPiTdeRgtdOM0vIX3roWe8fZZVDvqCPhPRL6stNsliTOq7yVofkoMsFBWhu6YUWX411qhQX2BPXuCJhjNTlDMZhRF32rGeO+XZOnHvhiUPeNI4xk23JyJ\\\\\\\/uqrazXnuhfjSSC8jg9mMWEsJDulMIvsS3MU\\\\\\\/HzJ6BAR3TVDcbEERsOcZ573WSQlG2PyHiOa9RXz8CoLyVAi0pAWeeIglE1y5k+VCIaMiEx8s+NGjxQ4JP2C9zpAqSxy8rgHeWbnJOhCHpGLdVeFg\\\\\\\/7ofx7y4ZtLuWhPbH+HOjXungTlPr+C\\\\\\\/cg5SuI61X8RnecZBpF\\\\\\\/CAnppiMT9sXW5IzGS4GWYTjWJ\\\\\\\/TKywDOxMlRrjLJSfIUC1vU1bLXW0eSad1Exn5ruCRsLEczq0Km+krNw4rsNxj3c2RE2v3eByZALP3wvkMNkJ\\\\\\\/zovJfkxjYoHMScpzlVw0XFjttNxH3LhVqrULGVI\\\\\\\/3Bggb3sgcNaqSgzxwi83YwDksnwG9Aut4E1seCuGGf1zTxoIII2J0zwME1Voa3l4DZJtaFJZPVAxxYtHGFjySaBwHVqo0w39wyN9VbVXi9RoWCnJqikpNwJmNK1GJ+g7doPKv4Nl4nXUsqZL0PUs4h41SWb\\\\\\\/nd9Sg2nyVLDfPjKnYECRL1LMFpo4icr2D7wHtAJxvmsI60M+L5olZlU37FUc1dmEd87dAyN3Z0TD\\\\\\\/Il0cItlPNWG+2tqrPv\\\\\\\/\\\\\\\/t7nxxC1vd1t+LJ9flXZD1vmDENHHqKEIOHX5SEcyay5XfeccPF8Kq3kR9MYo86XUwPt2bv8n7EHPPZLc3IBDPfgU2ij7UOPgbqs\\\\\\\/HVE7Em\\\\\\\/V0zV37KeUCQmie45YjeXHS+NgKMs4rx6tAap4d5CoilBEYVNVBo9AWTJMMhghKqCol8VfQJmHae15cIbWKuCki0TOvk3I9F1HvcUYJY4Q42OPA10p+Hq7eVykQFWxbTOIZF\\\\\\\/VdjL0Sd4lVAyI63ss40DmX7Kzu4yMUbX4T\\\\\\\/pclTxaAKUh\\\\\\\/aYD+y5yJUbrOa7hrqMZlsJ\\\\\\\/n2OdTWb4nvYxICau88tkImefFwpDRIxb31l8\\\\\\\/\\\\\\\/Nqid96KrdQGvbKrUrGV7X8DV5dUXlATxCVrcxbf6gqnv50i+9QgV8\\\\\\\/6PP6PI2dosise+GvvkYBjAUyCmnjOOetU5olOvlr3gpZoBkipKPgsvWGD7ZUQHCx3jlKtEIfxVNicmI0MXMWYuARoOEtAWoOE+x5pTw7FcIZBU0tU0TCDElYXMvDS6vSzMdHb6cHoCAMGYw8fk0oc\\\\\\\/IGGfQ9lcM\\\\\\\/cZ3mnR2z4XdPoDB2kijGWJ+nDLpv7r8HV2l0LhAJQqCiYpQQr3JUdiRmZ0JuscLCNDYVzqxjZUVczT96OT2H70L96J7V\\\\\\\/HodgwTxkUYkNxNHrp1+VG0iah0NG27iQbdiEjuG6Xr2MrVfcjluU2K4eJL2rdy49MRQ5yKTmnAgM\\\\\\\/oc9VV3hwhZFmf81d97FCB1SBhYKiGula8JtDTdgEtnK9pcWFEyB6y06xgGV4PIxTGt3hrZ\\\\\\\/RntKJmmUq6GkW3V2CezFCH\\\\\\\/1Brmh4tmFPm4Oq0dRzIPEfLk9F8JWoz+JOC81dGcGofVXxPF7r2u59hAb7oYxm\\\\\\\/5\\\\\\\/yv0xHqYWq75uPRtrD+WbpLjC8ONtpyJCTULEqOwJ2FioY+oDZB47khMH8XuvuovtJCY0qA8uk6yb6VNqjNfb0rrVWGpejmYS+LJEk+lMgaDDEo9Xa8N8JBTHf56cgqGL7+cWUpM2cMmBtYtcUue3Yt3TGGNyabUDY2Oebx+3YGdonplAA51Xr46eSK8jVIZUzuTeYtRMevQwhUProEpk9ugtDUgzA3FosIifO54r283GURqqm25hEs+LEqXqcenjgiEpCOKKauRM0wiWB+lUDuAoGSdlhDKcIBgaErctm9ZBcCZrFkkSL1iXxUEGQhMELmJ\\\\\\\/USIYAgIwGhJpsn9fMs0p+dIbDdgaIVfH9zlOAuSv1wi5taLpS1UJEg+X0JrxxbebS14DLmvETMBLqrH2Z9L3dvGrZHqLcFzvsupw80dCpeEnhm9b6rHNWismEHRFMCDB28vzZ6wPjjJSDfLIzOOxW3QCazKGFcYsKp3E81jJSMNQqrHybPeTCyugWmEeRRpAuMfKcEbsU5eo+ecI41+7qWpvPfxEvCVm9fO3gpLlMKuRWHCXqEUGHLu0srbWU+8Q0neF5de+Fu1Y4EO7WvYAparvKxve64YKGYc3Vuj8gxHaqpUrizolSvSiL+OQJzC65MGQdGVantIEP40Jwq32E+ZTVCG64X3lmxWFT2pIhBSmhrcVMMINzqh6bcg496bdE83wSRtAuF5eJGteNjlu2y08wTEnqCVny6wyH8p8ztjOS2+Z5eHW+AlgIhsZwAYf7zV4gM2QH\\\\\\\/e05G1sogvJhSz5WnWtCYbTDlNRtaYaOdpCHCtDgUY4Yt\\\\\\\/BLJumAkUzoRZxc\\\\\\\/m7hQxa5pG807ociyRLq+BDXhFg7t+Frui5Q5geP5rJU6\\\\\\\/yQ8fd8xKdiGb5Q9CODRp+pWd3jUbq6ltOSuTsmc\\\\\\\/daRqfJosxL4aZVNsTrfFCFYw1t+WUQZQCaUzTVvXIalzt5OsvyVhd5Nv0uTsizjcufafv6zbMccCIYNcTAfHZQwh80iINPfKChQHz+gTGwNjGGPDdRRsETXurkOjqEoFI9TspOCUhCYQkH8Np+EXgRSmF7D6DXjPtfgtilFxiTxz7HDISQsAkIWA+0UIKfQzlW7RVXgonSlz4XyQr3m\\\\\\\/8kTcEfI2Xs4eNa3NezJGJSx47usKTAa7um0NHyIZw4vVLMTLbMsz47hQHqOUkVxO+KaOns3LBC0IcHirj7b5CrBkZFH1NBfj\\\\\\\/A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Vlx9foKKdDOt3CgMDwqlWg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220e93d98edf1b20f39e8957e05ebefbab6dcefc70b59b0a4b4044fe76f9474cf5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229gEJ1MogKm3sMT5PPP\\\\\\\/AwbVQw0+El8IjVHtf7HjgFY6OQVb38dZShYvSwIEZ9rbyyeOBtofRh4mdBP8TSw3f8A4yuE+\\\\\\\/\\\\\\\/FG6xi9ed2BHMaywTH5qdAxv0aHL\\\\\\\/b9RyLacjFYyz4J1LroQ1L9Tqf\\\\\\\/StXh78XONy0iY7H\\\\\\\/02WuTcM3Jh+KzhWLI2AUNhUvHujyg5ndMQrLaejL2f9zWHFC+kaNs1gsilNAHgRNxhOvF05AWpLOyTWjj9UM2yoFMFLqGKun4e3CeJXcpLGe9sFciz2V53aXoB6xjShen+ACG6jqg0m2PrGaWomPNruEaDzf7RC9Kz1zbheHybtYQIB4XiLKRTHbPFlZmoSji9kgTsyjM9dCUnszyCrHVXjOXeRqyn+\\\\\\\/PoGkYXqbt721Lvhrh9tw\\\\\\\/cpGOgmbww6XfK\\\\\\\/rWmVQbUc8naFsKMG2mc34ZaRZWVmUEmhB+uAA5x8of\\\\\\\/xW+UHgjdwC+8ygTKzJ7KPYroMpzddKKbGh2\\\\\\\/s7I8K3UZ1i2z4s6\\\\\\\/S6jYP4I0fE\\\\\\\/SD1SUVhfxFxBBTqC5NbrDRlwNFTyDyvPe1+72flHK7pkrpZAbvEf8a0cZHXKrPW3VjY++fjOe7WY2PWCXUsw6SewmjIn9QzXLOKNUPIXENxn3dbGA7KR0QEmezN4hn4LDxDqh9nl0H6x8jVF6ogZ8M8TfZn64kgWm9fcJvoBaA+9AGzUB9t3P7zVkmMrAaLmEZnJ+or96ezqtNUWc+w\\\\\\\/SaQN8B6ssxFVYyn3OSC1FODVY1rHvZvi3hZB2mzYuAKg4dRY2esXj\\\\\\\/sKo2cBpoN6YFY+iMRTx3STXYj3kzUI6oue0G50Mqorx+zkm5vK\\\\\\\/X\\\\\\\/wvnfWMVcwAWUkZ6Ir8Ya3esuSFsWwNMVykoFaAbGUQNZqzQ8doWxmvyLv4hDfBU\\\\\\\/toN\\\\\\\/I1357lR4r0nuyj5YxlBcqgyIN6+IAMXGeq5PJK9BBP4KV8pWihYi1oht4H2XFDX25bSbO0tryYZ9qmFPG+aGMblkl70XOGWcXwVnpzFZPho5kiElpbqMum2DpZnBc1gstwqcOYaAup\\\\\\\/TIhwr8SBEmJyx2IvRAHXqCXk1oxoaQcvDBwcQm2k+hlrcN1y2IAss84Fz+VRVwJzIWm4lRJ6OybWNdMGv8vkkFLRqZoh25URty42GPUe3vNxiGAmnVVxFRE3hCkPKtLWXAHe0Pm53W\\\\\\\/qzzXPkziLFeQerH5h8STstxMhigOFs5Y99VjO6ijXkkYQ1tiqmQhc3LZVx2B77fw381bU9R0mbYxxwBHEKx3qc5o9KVwFhvWFlzL0BbCL1XzeL9a\\\\\\\/ZwHLjrZcJyh1EvLhshAJjrvXw31jrpalIL0St748Js5cG2B\\\\\\\/epEPEUWEjC3kBX+KJZWGZrSYB6Oi1fkwkqZxZAA2Va4Zr9IxsXAYlnxB78Xyb2nEJdfCgJeTsYfiwNZAhDblRjT6IX0SH3MDBF1TxhoZJJuLcLTO55j+IrVAvFX4l5Ly57Tg0hdXeRcuRKUrcu+Pn\\\\\\\/UyrW5LixOUDDvN8pchVXhQJQkgCIipuY3B1uhHGG6UOVFnIh7FxcD\\\\\\\/aumxq+zTTT5oGS5QNhCe9VANQTTRozggeLDJzQWnxUoz6biliU9mSzR+7u5wgYGMpoQbCf6LyQ01Hyw0bbaTMIi57cBRJIdaSZJI+iIt9ZpPfF47U2rc+eJgua4DIahcMjKGLE5QHD7oiR2hBU+Z6c+gJBE4eTS+hjKZrAui3Tog17WsAZgni8LZc6mhHHlp8jCV0Ym4WthROabT1h62hC4PuZf78kJcBmwykGqzGsMVWq9sygwgj0pW3AjNx2tNAP98PI2nooZnT3HivvFcNN966KhhQA9TpaRyX\\\\\\\/RFThj8uigPTQI1y8xok+zIIQDcEn0cSrcqrhFIZ2M9ckfgEwOqug9crTW1CzcTsVKe3yx47sSWjHyhw\\\\\\\/6knng68mKphk5yIBu0sak0iv\\\\\\\/sTZ\\\\\\\/S41UGcWjbyLxp7+UCLTNTyGd9h645W1y3dPnh4eqb5C6w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022atMh3tV+C1\\\\\\\/LfCxFHgwCxg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022388bf1b916af04260a2e2051a5a980399e478ba12e1362bd2fcf4f38fbe2f7ac\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459224 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/143" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-246eq6nNzOOksVGlVaouGjWY66p9aGH4aXl_K9MOBjg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/143" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GV1sw_oqtxf7Hpjr9g-W5xn1FXcrpRGJnfujK6-3F1I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/1/131" title="Many Saccharomyces cerevisiae Cell Wall Protein Encoding Genes Are Coregulated by Mss11, but Cellular Adhesion Phenotypes Appear Only Flo Protein Dependent" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/151" title="Facilitating Discovery: The Role of Society Journals in Collaborative Science" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459224" id="node459224" data-pisa="ggg;2/1/143" data-pisa-master="ggg;2/1/143" data-apath="/ggg/2/1/143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miao Sun</span>, <span class="highwire-citation-author" data-delta="1">Kajari Mondal</span>, <span class="highwire-citation-author" data-delta="2">Viren Patel</span>, <span class="highwire-citation-author" data-delta="3">Vanessa L. Horner</span>, <span class="highwire-citation-author" data-delta="4">Alyssa B. Long</span>, <span class="highwire-citation-author" data-delta="5">David J. Cutler</span>, <span class="highwire-citation-author" data-delta="6">Tamara Caspary</span>, <span class="highwire-citation-author" data-delta="7">Michael E. Zwick</span> and <span class="highwire-citation-author" data-delta="8">B. J. Andrews</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">143-150;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001669 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459224" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miao Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miao%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiao%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kajari Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kajari%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKajari%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Viren Patel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Viren%2BPatel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patel%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AViren%2BPatel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vanessa L. Horner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BL.%2BHorner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Horner%20VL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BL.%2BHorner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alyssa B. Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alyssa%2BB.%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlyssa%2BB.%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David J. Cutler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BCutler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cutler%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BCutler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tamara Caspary </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamara%2BCaspary%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caspary%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATamara%2BCaspary%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzwick@emory.edu" class="" data-icon-position="" data-hide-link-title="0">mzwick@emory.edu</a>
<a href="mailto:tcaspar@emory.edu" class="" data-icon-position="" data-hide-link-title="0">tcaspar@emory.edu</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michael E. Zwick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BZwick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zwick%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BE.%2BZwick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzwick@emory.edu" class="" data-icon-position="" data-hide-link-title="0">mzwick@emory.edu</a>
<a href="mailto:tcaspar@emory.edu" class="" data-icon-position="" data-hide-link-title="0">tcaspar@emory.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">B. J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/1/143" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459224" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459224/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/143.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459224" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459224/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/143.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459224" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459224/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/1/143.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459224" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459224/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/1/143.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459224" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459224/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Forward genetic screens in <em>Mus musculus</em> have proved powerfully informative by revealing unsuspected mechanisms governing basic biological processes. This approach uses potent chemical mutagens, such as <em>N</em>-ethyl-<em>N</em>-nitrosourea (ENU), to randomly induce mutations in mice, which are then bred and phenotypically screened to identify lines that disrupt a specific biological process of interest. Although identifying a mutation using the rich resources of mouse genetics is straightforward, it is unfortunately neither fast nor cheap. Here we show that detecting newly induced causal variants in a forward genetic screen can be accelerated dramatically using a methodology that combines multiplex chromosome-specific exome capture, next-generation sequencing, rapid mapping, sequence annotation, and variation filtering. The key innovation of our method is multiplex capture and sequence that allows the simultaneous survey of both mutant, parental, and background strains in a single experiment. By comparing variants identified in mutant offspring with those found in dbSNP, the unmutagenized background strains, and parental lines, induced causative mutations can be distinguished immediately from preexisting variation or experimental artifact. Here we demonstrate this approach to find the causative mutations induced in four novel ENU lines identified from a recent ENU screen. In all four cases, after applying our method, we found six or fewer putative mutations (and sometimes only a single one). Determining the causative variant was then easily achieved through standard segregation approaches. We have developed this process into a community resource that will speed up individual labs’ ability to identify the genetic lesion in mutant mouse lines; all of our reagents and software tools are open source and available to the broader scientific community.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/n-ethyl-n-nitrosourea-enu-mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-n-ethyl-n-nitrosourea-enu-mutagenesis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">N</em>-ethyl-<em xmlns:default="http://www.w3.org/1999/xhtml">N</em>-nitrosourea (ENU) mutagenesis</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next generation sequencing</a></li><li class="kwd"><a href="/keyword/dna-sequencing" class="hw-term hw-article-keyword hw-article-keyword-dna-sequencing" rel="nofollow">DNA sequencing</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/targeted-enrichment" class="hw-term hw-article-keyword hw-article-keyword-targeted-enrichment" rel="nofollow">targeted enrichment</a></li></ul><p id="p-6">Forward genetic screens in the mouse have uncovered many new features of mammalian biology. The logic of these studies is straightforward. First, a mutagen is used to randomly induce mutations in mice. Then each line is phenotypically screened to identify lines that disrupt a specific biological process of interest. Finally, a putative causative mutation within a gene is found and then directly shown to be responsible for the observed phenotype. The fruit of such studies is a collection of newly identified genes whose subsequent characterization has given us remarkable insights into mammalian biology (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Caspary and Anderson 2006</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cook <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acevedo-Arozena <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beutler and Moresco 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Stottmann <em>et al.</em> 2011</a>).</p><p id="p-7">Although mapping and identifying a mutation using the rich resources of mouse genetics is straightforward, detecting mutations is neither fast nor cheap, thanks to two laborious bottlenecks. The first involves the fine chromosomal mapping required to localize a mutation to a small genomic region, and the second entails the Sanger-based sequencing of genes within that defined region to find putative mutations. Together, these bottlenecks discourage many from using forward genetics. Second-generation sequencing platforms hold out great promise for overcoming these bottlenecks, however, as they enable individual investigators to harness enormous raw sequencing power at a dramatically lower cost per sequenced base than traditional Sanger sequencing (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shendure <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Shendure and Ji 2008</a>). In the vast majority of mutations found to date, the mutagen induces the critical basepair change in a coding region or splice site, so sequencing can focus on these regions (exome sequencing). Several groups have successfully used deep sequencing of single-mutant individuals to identify causal mutations. The first of these groups sequenced a BAC they generated from mutant DNA; others have performed whole-exome sequencing, which enabled them to map the mutation while simultaneously detecting variants (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zhang <em>et al.</em> 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fairfield <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hilton <em>et al.</em> 2011</a>).</p><p id="p-8">Despite the promise of second-generation sequencing to overcome these bottlenecks, it has yet to be broadly adopted by the mouse forward genetics community. One reason may be that the most popular mutagen used in mouse forward genetics, ENU, induces point mutations, so when sequencing is used in these lines, two challenges arise: (1) distinguishing the ENU-induced variants from strain polymorphisms and (2) identifying real variants (either ENU-induced or strain-specific) from technical sequencing errors. Here we investigate how to best surmount these challenges.</p><p id="p-9">From our experience performing recessive forward genetic screens, we have made several observations with a practical impact on our strategy for overcoming these sequencing challenges. In the course of our screens (schematic in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), we look for phenotypes of interest in the generation 3 (G3) litters (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Horner and Caspary 2011</a>). We define a line once we observe the phenotype in a quarter of the embryos across three litters <em>and</em> identify at least one G2 male. The latter requires us to set up blind crosses as we cannot genotype for the unknown mutation, meaning only one out of every four crosses results in our seeing the phenotype. Practically, this means that by the time we establish a line, we have at least four litters, usually more. Because each litter averages eight embryos, of which around two individuals are affected, we can easily determine chromosomal linkage by genome scanning four affected individuals. This allows us to genotype potential carriers in the mouse room, which means immediate and dramatic reductions in mouse costs.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/143/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ENU mutagenesis screen and mapping. (A) ENU-mutagenized C57BL/6J male mice (generation 0, G0) were crossed to FVB/NJ female mice to generate G1 males, which were again crossed to FVB/NJ female mice. G2 females were backcrossed to their respective G1 fathers to produce G3 embryos, which were screened for neural phenotypes. Asterisks (*) indicate ENU-induced mutations, &#x201C;i&#x201D; next to asterisk (*i) shows the inherited ENU-induced mutation, and black and white lines depict C57BL/6J and FVB/NJ. (B) The causative mutation segregates with the C57BL/6J DNA, whose interval can be defined by linked polymorphic markers. (C) E10.5 wild-type (WT) and mutant embryos (AB5, M2, X5, and Y1) identified in ENU screen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1189216010" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ENU mutagenesis screen and mapping. (A) ENU-mutagenized C57BL/6J male mice (generation 0, G0) were crossed to FVB/NJ female mice to generate G1 males, which were again crossed to FVB/NJ female mice. G2 females were backcrossed to their respective G1 fathers to produce G3 embryos, which were screened for neural phenotypes. Asterisks (*) indicate ENU-induced mutations, &#x201C;i&#x201D; next to asterisk (*i) shows the inherited ENU-induced mutation, and black and white lines depict C57BL/6J and FVB/NJ. (B) The causative mutation segregates with the C57BL/6J DNA, whose interval can be defined by linked polymorphic markers. (C) E10.5 wild-type (WT) and mutant embryos (AB5, M2, X5, and Y1) identified in ENU screen.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/143/F1.medium.gif" width="341" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/143/F1.medium.gif" width="341" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/143/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/143/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461153" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-10" class="first-child">ENU mutagenesis screen and mapping. (A) ENU-mutagenized C57BL/6J male mice (generation 0, G0) were crossed to FVB/NJ female mice to generate G1 males, which were again crossed to FVB/NJ female mice. G2 females were backcrossed to their respective G1 fathers to produce G3 embryos, which were screened for neural phenotypes. Asterisks (*) indicate ENU-induced mutations, “i” next to asterisk (*i) shows the inherited ENU-induced mutation, and black and white lines depict C57BL/6J and FVB/NJ. (B) The causative mutation segregates with the C57BL/6J DNA, whose interval can be defined by linked polymorphic markers. (C) E10.5 wild-type (WT) and mutant embryos (AB5, M2, X5, and Y1) identified in ENU screen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">As we routinely establish chromosomal linkage of our lines, second-generation sequencing offers us at least two strategies: (1) to sequence genome-wide and ignore the data from unlinked chromosomes or (2) to sequence in the linked region more thoroughly. We chose the latter, as we could take advantage of the recent development and validation of methods of isolating target DNA from complex eukaryotic genomes that can be easily sequenced on second-generation platforms [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bashiardes <em>et al.</em> 2005</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Albert <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fredriksson <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hodges <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Okou <em>et al.</em> 2007</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Porreca <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bau <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gnirke <em>et al.</em> 2009</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ng <em>et al.</em> 2009</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nikolaev <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tewhey <em>et al.</em> 2009a</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">2009b</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Turner <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bainbridge <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Patel <em>et al.</em> 2010</a>; and review in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Mamanova <em>et al.</em> (2010)</a>]. Furthermore, for the same cost as a single sample, we pursued a multiplex approach that let us both capture and then sequence multiple samples simultaneously. Because we know the inheritance pattern of any phenotype-causing mutation in a genetic screen, we reasoned it should be possible to distinguish strain and sequencing variation by precisely defining a putative mutation as a <em>de novo</em> variant that occurred at expected ratios in homozygous and heterozygous individuals, but in neither background strain. Thus, we were able to validate a method that exploits advances in isolating target DNA and next-generation sequencing with an added comparative approach in a cost-effective manner.</p><p id="p-12">We found this strategy greatly enhanced our ability to find the most likely causal variant. Here we show that the major benefit of our multiplexed approach is to eliminate from consideration variants resulting from strain polymorphisms and technical errors. We took four novel ENU lines we had identified in a recent ENU screen, and for each line, we analyzed six DNA samples, including one from a previously identified ENU-induced line (positive control), an affected mutant embryo, each of the affected embryo’s parents, and each of the original background strains used in the screen. In all four lines examined, this approach dramatically reduced the number of candidate mutations <em>vs.</em> what we would have found had we used the data from the affected individuals only. Thus, for each of the four lines, we now have strong candidate lesions that segregate with the relevant phenotype. Notably, we found that the sequence comparison between the affected individuals and the background strains we used dramatically reduced the number of putative mutations, arguing that simplex whole-exome approaches would also benefit from this comparison.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mouse screen</h3><p id="p-13">We performed a recessive ENU mutagenesis screen as previously described (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Garcia-Garcia <em>et al.</em> 2005</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Horner and Caspary 2011</a>). Briefly, we mutagenized C57BL/6J mice and crossed them to FVB/NJ. We examined G3 embryos for abnormal development of the nervous system at embryonic day 10.5 (E10.5). Of 121 lines screened, we found 11 lines with defects in brain and spinal cord morphology. Through a genome scan using single nucleotide polymorphism (SNP) or simple sequence length polymorphism (SSLP) markers that differ between the C57BL/6J and FVB/NJ strains, we mapped four lines to defined intervals on individual chromosomes.</p></div><div id="sec-3" class="subsection"><h3>Mouse samples used in our study</h3><p id="p-14">Six samples were used for each of the microarray-based genomic selection (MGS) experiments. Samples used for each line included two genetic background strains (C57BL/6J and FVB/NJ), one mouse homozygous for the mutant allele, and its two parents heterozygous for this allele. For each chromosome, we included the relevant positive heterozygous control from one of three known strains: <em>Rab23<sup>opb2</sup></em>, <em>Nckap1<sup>khlo</sup></em>, or <em>Ift172<sup>wim</sup></em> on chr1, chr2, and chr5, respectively (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eggenschwiler <em>et al.</em> 2001</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huangfu <em>et al.</em> 2003</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rakeman and Anderson 2006</a>).</p></div><div id="sec-4" class="subsection"><h3>Design of custom microarrays for microarray-based genomic selection</h3><p id="p-15">We used a software package named MOPeD (Microarray Oligonucleotide Probe Designer, <a href="http://moped.genetics.emory.edu/">http://moped.genetics.emory.edu/</a>) to design all MGS microarrays for this project (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Patel <em>et al.</em> 2010</a>). This web-based software allows individual investigators to easily design custom genome capture arrays that have been optimized for maskless array synthesis by Roche NimbleGen. We used the existing mouse chr1, chr2, and chr5 MOPeD-designed MGS arrays for genomic selection and obtained the target sequences for the mouse chromosomes from the UCSC genome browser RefSeq Genes track (mm9 build). We targeted all the coding and noncoding exons within the region of interest. Non-overlapping fragments were padded with 100 bp on either side to ensure proper capturing of splice sites and fragment ends. For the chr1 chip, we included the exome between the MIT markers D1MIT251 and D1MIT132, where AB5 mapped, as well as all exons of the positive control <em>Rab23</em>. For the chr2 chip, we included the exome between the MIT markers D1MIT81 and D2MIT152 (candidate region for M2) and between D2MIT285 and D2MIT411(candidate region for X5), as well as all exons of the positive control gene <em>Nckap1</em>. For the chr5 chip, we included the exome between the MIT markers D5MIT361 and D5MIT20, where Y1 mapped, as well as all exons of the positive control gene <em>Ift172</em>.</p></div><div id="sec-5" class="subsection"><h3>Performing microarray-based genomic selection</h3><p id="p-16">MGS was carried out as described previously (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Okou <em>et al.</em> 2007</a>, <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">2009</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Patel <em>et al.</em> 2010</a>) with limited protocol modifications (described below). We first fragmented 5 µg of genomic DNA (mean size 300 bp, range 200–600 bp) from each of the six different mice, then repaired the ends and ligated distinct Illumina multiplex adaptors to each sample. Thus, each mouse sample had a unique multiplex adaptor tag that allowed simultaneous hybridization to a single MGS array. The sequence tag allowed us to deconvolute the sequences and, therefore, to relate the sequence to a specific mouse sample after Illumina sequencing. To ensure equal representation of multiplex adaptor-tagged DNA fragments from each of the six mice, we performed quantitative PCR with a probe complementary to the Illumina adaptors to accurately assess the quantity of tagged fragments for each sample prior to hybridization on the MGS array. We then pooled 166.6 ng from each of the six samples (total 1 µg) and hybridized this pool to the MGS array using the standard protocol. The remaining steps, which include eluting and sequencing the product, were done via our previously described standard protocol (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Okou <em>et al.</em> 2009</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Patel <em>et al.</em> 2010</a>).</p></div><div id="sec-6" class="subsection"><h3>Illumina sequencing and data analysis</h3><p id="p-17">After MGS, the enriched DNA samples were quantitated using qPCR, and then were denatured and diluted to a concentration of 8 pM. Each enriched library (having six samples with different index tags) was sequenced in a single lane of an Illumina HiSequation (100 bp, multiplexed, paired end for chr5; 100 bp, multiplexed, single end for chr1 and chr2). After sequencing, the reads were mapped against the reference sequences for respective regions using Emory Mapper (D. J. Cutler and M. E. Zwick, personal communication), and variants sites were identified. Those variants were then annotated using SeqAnt (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Shetty <em>et al.</em> 2010</a>). Next we used custom Perl scripts to process the SeqAnt output files to first identify all unique variants (Seqant.parse.pl), and then to identify and count single nucleotide variants (SNV) that were homozygous in the mutant individual and not found in dbSNP or the background strains (Seqant.count.pl), and finally to list the genotypes inferred from sequence data for the parents and mutant offspring (Seqant.genotype.pl).</p></div><div id="sec-7" class="subsection"><h3>Sanger sequencing</h3><p id="p-18">We verified the replacement variants we identified that were within the genomic interval linked to the phenotype through Sanger sequencing. For each variant, we amplified the region surrounding the variant from genomic DNA from two mutant embryos (primers in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001669/-/DC1/001669SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001669/-/DC1/TableS2.pdf">Table S2</a>).</p></div><div id="sec-8" class="subsection"><h3>Pedigree genotyping</h3><p id="p-19">For the top six candidate replacement variants, we found they either created or destroyed a restriction enzyme polymorphism that we could use to genotype directly for the variant. The enzyme used to detect the polymorphisms and their respective genes were: <em>Hae</em>III (<em>Ankrd56</em>), <em>Hae</em>III (<em>Inpp5e</em>), <em>MluC</em>I (<em>Slc2a6</em>), <em>Sfc</em>I (<em>Rqcd1</em>), <em>Mob</em>I (<em>Rbm12</em>), and <em>Mob</em>I (<em>Duspd15</em>). We genotyped all embryos from between 32 and 41 litters in each line for the variant. In the course of the screen, we found “abnormal” embryos that resembled neither the mutant nor the wild-type phenotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001669/-/DC1/TableS2.pdf">Table S2</a>). This was likely due to both spontaneous abnormal development and the high mutation load in the screen. Consistent with these ideas was our finding of no correlation between any variant and any “abnormal” phenotype in these embryos, suggesting the abnormal embryos were not the result of the variants we identified.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Forward genetic screen and mapping</h3><p id="p-20">We performed a recessive ENU mutagenesis screen to identify genes important in the development of the mouse nervous system. We induced mutations on a C57BL/6J background and crossed to an FVB/NJ background, as shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>. By examining G3 embryos, we identified four mutant lines, AB5, M2, X5, and Y1, which displayed abnormal neural morphology at E10.5 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). These phenotypes were inherited in an autosomal recessive fashion with complete penetrance. AB5 mutant embryos were posteriorly truncated and showed abnormal brain morphology and cranioedema. M2 mutant embryos exhibited midbrain and hindbrain exencephaly, a dysmorphic neural tube, and anophthalmia or microphthalmia. X5 mutant embryos showed a general developmental delay along with an open forebrain and midbrain. Y1 mutant embryos displayed whole brain exencephaly and an undulated neural tube. We genotyped 4 affected G3 embryos using the Illumina mouse low-density linkage panel for SNP mapping in order to map the location of each putative recessive mutation to an individual chromosome (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Moran <em>et al.</em> 2006</a>). We found AB5 was linked to chr1; M2 and X5 were both on chr2; and Y1 was located on chr5.</p></div><div id="sec-11" class="subsection"><h3>Targeted sequencing and mutation discovery</h3><p id="p-21">Using MOPeD to tailor existing mouse chromosome exome designs, we performed targeted enrichment of all exons and adjacent intronic sequences contained within the mapped boundaries for each of the four mutant lines (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Patel <em>et al.</em> 2010</a>). Our previous analysis of single mouse samples showed we obtained far more sequence depth than needed to identify a known mutation in a single lane of Illumina sequencing. To speed the discovery of unknown ENU-induced mutations, we reasoned that, for the same basic cost, we could simultaneously sequence six distinct samples on a single microarray during the enrichment step (multiplexing). These samples included an affected embryo, its two heterozygous parents, one unmutagenized individual from each of the two background strains used in the screen, and as a positive control, a distinct, known ENU-induced line that mapped to the same chromosome. We performed multiplex MGS followed by sequencing on an Illumina platform for each of the four mutant lines (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). After mapping sequence reads, approximately 90% of targeted regions had a sufficiently high coverage (&gt;8X) to identify variant sites for subsequent analysis. Comparable coverage was obtained for each of the multiplexed lines sequenced. The complete distribution of sequence coverage is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001669/-/DC1/FigureS1.pdf">Figure S1</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/1/143/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequencing and analysis pipeline: identifying ENU-induced mouse mutations with chromosome-specific exome sequencing. DNA samples from an affected embryo (muk/muk), the affected embryo&#x2019;s parents (+/muk), unmutagenized individuals from the two background laboratory strains (B1 and B2), and a positive control known mutation (+/mk) were multiplexed through our targeted sequencing pipeline to validate our approach and identify novel mutations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1189216010" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequencing and analysis pipeline: identifying ENU-induced mouse mutations with chromosome-specific exome sequencing. DNA samples from an affected embryo (m&lt;sub&gt;uk&lt;/sub&gt;/m&lt;sub&gt;uk&lt;/sub&gt;), the affected embryo&#x2019;s parents (+/m&lt;sub&gt;uk&lt;/sub&gt;), unmutagenized individuals from the two background laboratory strains (B1 and B2), and a positive control known mutation (+/m&lt;sub&gt;k&lt;/sub&gt;) were multiplexed through our targeted sequencing pipeline to validate our approach and identify novel mutations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/1/143/F2.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/1/143/F2.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/1/143/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/1/143/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461103" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-22" class="first-child">Sequencing and analysis pipeline: identifying ENU-induced mouse mutations with chromosome-specific exome sequencing. DNA samples from an affected embryo (m<sub>uk</sub>/m<sub>uk</sub>), the affected embryo’s parents (+/m<sub>uk</sub>), unmutagenized individuals from the two background laboratory strains (B1 and B2), and a positive control known mutation (+/m<sub>k</sub>) were multiplexed through our targeted sequencing pipeline to validate our approach and identify novel mutations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">A causative mutation that explains the phenotype we observed had to be found as homozygous within the mutant line and be newly induced by the action of the ENU mutagen. To determine whether such a variant was found within each of our multiplex lines, we first functionally annotated variants for each line using SeqAnt (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Shetty <em>et al.</em> 2010</a>). We then cataloged all homozygous variant sites by functional class for each of the four mutant lines (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, column 3). We then filtered these variants in two ways. First, because our mutation would be ENU induced, we expected it should not be a common polymorphism contained within dbSNP. Applying this filter to all the homozygous variant calls for all four mutant lines eliminated 58% (1179 of 2043) of variant sites from further consideration (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, column 4). Second, as the newly induced mutation should not be found in either of the laboratory background strains (C57BL/6J or FVB/NJ) used, we next ruled out variants found in these strains from further consideration. In total, this allowed us to eliminate an additional 40% (816 of 2043) of variants from further consideration (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, column 5). The remaining 2% (48 of 2043) of variants should thus include mutations that explain the mutant phenotypes we observed (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, column 6). Among the remaining 48 variant sites, only 15 were replacement variants, which constituted the class most likely to harbor the mutant alleles. Hence our two-stage screen eliminated 99.3% (2028 of 2043) of variant sites, allowing us to focus on the 0.7% (15 of 2043) of sites across the four lines that were most likely to be the causative alleles we sought.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461197/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461197/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Results of filtering homozygous variants sites for each mutant line sequenced</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Identification of mutant alleles</h3><p id="p-24">Mapping of the mutant lines during the course of stock maintenance reduced the size of the region harboring the mutant alleles, allowing us to eliminate an additional nine replacement variants from further consideration (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Two of the lines (AB5 and Y1) had only a single candidate mutation, whereas another two (M2 and X5) had two candidate mutations each. We confirmed all six replacement variants by Sanger sequencing (data not shown). We genotyped each variant within each line’s existing pedigree and calculated the probability that the observed correlation of the phenotype-genotype segregation pattern was due to chance (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). In all six cases, the variants segregated with the respective phenotypes.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461118/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461118/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Candidate replacement variants in four mutant lines</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461115/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461115/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Segregation data for candidate mutations</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">In the AB5 mutant line, we found a single T-to-A transversion in exon 4 of <em>Rqcd1</em>. This change is predicted to result in a leucine being converted to a stop codon. Among 33 litters examined, all 46 embryos with the mutant AB5 phenotype were shown to possess the A/A genotype, suggesting it is causal (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Providing further evidence for this conclusion is the observation that the <em>Rqcd1</em> transcript is found in an expressed sequence tag (EST) library generated from E8.0 whole-mouse embryo, such that the gene is expressed just prior to when we observed morphological embryonic anomalies.</p><p id="p-26">For line M2, we found two potential causal variants in genes that lay 0.63 Mb apart within the candidate interval: a T-to-C transition in <em>Inpp5e</em> and a C-to-A transversion in <em>Slc2a6</em>. Both mutations are nonconservative: the transition in <em>Inpp5e</em> changed an aspartic acid to a glycine, and the transversion in <em>Slc2a6</em> altered a methionine to an isoleucine. Among 41 M2 litters examined, we found that all M2 mutants genotyped were homozygous for the C/C <em>Inpp5e</em> and the A/A <em>Slc2a6</em> variants, whereas wild-type embryos that appeared wild-type were never homozygous for the newly induced alleles (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Furthermore, there are EST clones representing both genes in mouse embryonic brain libraries. Taken together, both variants remain candidate causal mutations for the observed mutant phenotype. Notably, <em>Inpp5e</em>-null mice have been characterized and share several aspects of the M2 phenotype, the most obvious being the exencephaly we see in both the M2 and <em>Inpp5e</em>-null mice (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jacoby <em>et al.</em> 2009</a>). This correlation suggests that the M2 phenotype may be caused in large part by the <em>Inpp5e</em> mutation we uncovered. The contribution, if any, of the <em>Slc2a6</em> variant to the M2 mutant phenotype awaits more careful phenotypic characterization of the M2 line, as well as either another <em>Slc2a6</em> allele or a recombination event that eliminates <em>Slc2a6</em> from the candidate interval.</p><p id="p-27">We also found two potential variants at two loci for the X5 line. Within the X5 candidate interval, we found a G-to-A transition in <em>Rbm12</em> and a T-to-A transversion in <em>Dusp15</em>. These are predicted to change amino acid 556 of <em>Rbm12</em> from threonine to isoleucine and amino acid 157 of <em>Dusp15</em> from isoleucine to phenylalanine. We identified EST clones of <em>Rbm12</em> from E8.0 embryonic libraries, but we found no evidence for embryonic <em>Dusp15</em> expression in the EST databases. Upon genotyping 32 litters from the X5 pedigree, we saw that mutant animals were always homozygous A/A at <em>Rbm12</em> (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). In contrast, we found four X5 mutant embryos that were heterozygous (T/A) for the <em>Dusp15</em> variant. Additionally, we detected two apparently wild-type embryos that were homozygous (A/A) for the <em>Dusp15</em> variant. Thus our genotype data provide greater support for the variant at <em>Rbm12</em> <em>vs.</em> the variant at the <em>Dusp15</em> (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Although our findings might be explained by incomplete penetrance of alleles at <em>Dusp15</em>, the most parsimonious explanation is that a recombination event took place, excluding <em>Dusp15</em> from the candidate interval. Taken together, we conclude that the variant in <em>Rbm12</em> is most likely to cause the X5 phenotype.</p><p id="p-28">For line Y1, we identified an A-to-G transition in the single exon gene <em>Ankrd56</em>, which is predicted to change serine 299 of the protein to proline. We analyzed 33 litters from the Y1 pedigree and found all of the phenotypic Y1 mutant embryos genotyped G/G, whereas the nonphenotypic embryos were A/A or G/A (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). We also found EST clones of <em>Ankrd56</em> in E8.0 libraries. In total, the strong genotype-phenotype correlation, the fact that the newly induced variant results in a nonconservative amino acid change, and the pattern of expression of the <em>Ankrd56</em> locus all argue that this variant is likely to underlie the Y1 phenotype.</p><p id="p-29">In all four lines, we found that the putative causative mutations were homozygous in the affected mutant embryos and called heterozygous in their parents in the initial sequencing experiment, as expected from the recessive inheritance of the mutation. The only exception was <em>Dusp15</em>, where one parent had very poor depth of coverage and was not called a heterozygote at the relevant site.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-30">Currently, the unparalleled insight into fundamental biological processes afforded by phenotype-based screens in the mouse is undermined somewhat by the challenges in identifying the affected gene. Here we accelerate the discovery of likely causative mutations in four novel ENU-induced lines using a methodology that combines multiplex chromosome-specific exome capture, next-generation sequencing, rapid mapping, sequence annotation, and variation filtering. Although additional formal genetic experiments are required to definitively demonstrate that we have identified the causative variant, the findings presented here are strong evidence that the replacement variants we found are the best candidates. Given that each line has only one or two candidate genes in need of follow up, integrating our approach more broadly into forward genetic screens in the mouse has the potential to speed up the gathering of functional information about the mammalian genome.</p><p id="p-31">Whole-exome and whole-genome sequencing with next-generation platforms have been used to successfully identify mutations in the mouse (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Zhang <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arnold <em>et al.</em> 2011</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fairfield <em>et al.</em> 2011</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hilton <em>et al.</em> 2011</a>). Here we show that by using targeted sequencing of multiple samples, we can reduce the number of candidate causative mutations to one or two per line. We believe our approach offers a number of advantages compared with simplex exome sequencing of mutant individuals. First, mapping a newly induced mutation to specific chromosomes is inexpensive and routinely conducted, regardless of the type of follow-on sequencing performed. Given the ease and low cost of linkage mapping a newly arising mutation to a chromosome, as well as the reduction in mouse costs, the seemingly most natural and efficient strategy for finding newly induced mutations would focus sequencing effort solely on those chromosomes that linkage indicates must harbor a newly induced mutation. Second, multiplexing six samples on a single MGS microarray reduces the cost of capture reagents, while providing valuable data vital to pinpointing the causative mutations. Furthermore, focusing on single chromosomes reduces sequencing costs and the bioinformatics burden associated with whole-exome/whole-genome datasets. Strikingly, we found that by including the background strains in the selection and sequencing, we eliminated all but a handful of variants sites from consideration, making the identification of the putative mutation dramatically faster. The composition of those eliminated variants presumably includes both true variant sites that are unique to a given laboratory mouse strain and false-positive variant sites that arise due to systematic mismapping and/or imperfections in genotype calling of next-generation sequencing data. In the case of a simplex whole-exome/whole-genome sequencing design, where only the affected individual is sequenced, these additional variants would have to be pursued with another technology, like Sanger sequencing, thereby slowing identification of the putative mutation. In theory, the same approach of using the identical sequencing platform for the background strains could be used for simplex whole-exome analysis. However, even as sequencing costs continue to decline, the costs of capture reagents for each additional line are fixed.</p><p id="p-32">Published approaches to identifying causative mutations routinely rely on screening out variants found in dbSNP (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Fairfield <em>et al.</em> 2011</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hilton <em>et al.</em> 2011</a>). We found that doing so eliminated 58% of the observed variants. Strikingly, an additional 40% of our detected variants appear to be real polymorphisms present in our laboratory’s parental strain or technical artifacts not currently seen in dbSNP. In contrast, our inclusion of the heterozygous parental lines only allowed us to screen out a handful of sequencing errors that appeared homozygous in the affected mutant embryo but were unobserved in the parents. Our demonstration of the power gained by sequencing the background strain using the same sequencing platform as the affected mouse is important, as it presumably needs to be done only once at a particular laboratory site. Thus, individual investigators could in effect have a personalized database of sequence variants for their background strains in addition to those found in public databases like dbSNP. Given such a resource, our design could be further modified to multiplex affected individuals from distinct lines, which would further reduce the cost per mutant line.</p><p id="p-33">At a molecular level, the clear advantage of using sequencing is evident in the M2 line, where two replacement mutations remain candidates. Using traditional methods, one can easily imagine narrowing the genomic interval to the same degree and seeing that the characterized null phenotype of <em>Inpp5e</em> mimics that of M2. Should M2 fail to complement the <em>Inpp5e</em>-null allele, we would never realize that <em>Slc2a6</em> also carries a potentially functional variant. Similarly, should neither the <em>Inpp5e</em> nor the <em>Slc2a6</em> variants underlie the M2 phenotype in formal genetic tests, we already know the two UTR variants that would be the next best candidates (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>). Thus, comprehensive targeted sequencing enables the identification of all possible mutations that will give us a more thorough understanding of alleles in the future.</p><p id="p-34">Finally, although we chose to use custom-designed microarrays as our selection platform because of their easy availability and lower cost, they do require some standard hardware to process the microarrays. Chromosome-specific liquid capture may be another viable strategy that could be pursued in the future. Our custom microarray designs, which were not empirically optimized, are freely available for use by the broader community of mouse geneticists. Using our data from this experiment will allow immediate improvement in the design of such arrays to further improve the performance of the targeted enrichment for future sequencing studies.</p><p id="p-35">The cost savings of our method are clear, albeit difficult to calculate precisely. The biggest fixed expense in performing forward genetics is the “cage costs” associated with breeding. Therefore, the initial chromosomal linkage of newly induced mutations is cost effective, because it reduces the number of crosses one needs to set up by 75%. While sequencing costs continue to drop precipitously, our method provides a means to verify results across multiple individuals, while generating sufficient sequencing coverage for each individual sample. The result is that we identify only a few real candidate variants, which inherently reduces the costs associated with following up potential variants. Taken together, our data show that we can now more efficiently pinpoint the most likely causal variants in ENU-induced lines.</p><div id="sec-14" class="subsection"><h3>Implications for mouse forward genetics</h3><p id="p-36">One valid criticism of forward genetic screens is that, in addition to identifying novel genes, one can easily invest a great deal of time discovering a new allele, or even rediscovering a previously identified allele, of an already characterized gene. For example, the mutation we identified in the M2 line likely represents a new <em>Inpp5e</em> allele. Historically, the existence of allelic series of specific loci has proved invaluable. This is especially true for structure-function studies, as well as for understanding the effects of mutations found in humans. Distributing new alleles of previously characterized genes through the regional mouse mutant resource centers could be of great value to the mouse community at large. Still, as one of the main goals of a forward genetic screen is to identify <em>novel</em> loci that govern basic biological processes, there are clear advantages to the investigator being able to choose which lines to work on knowing whether the gene has previously been characterized. Our method has the potential to change the way the forward genetics community conducts screens, as we would be able to first systematically find the molecular lesions in all lines and then integrate that information in choosing which lines to pursue. Our data argue that such an ideal is within reach.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">We are grateful to Kathryn Anderson for supplying the <em>Nckap1<sup>khlo</sup></em> and <em>Ift172<sup>wim</sup></em> embryos and to Jonathan Eggenschwiler for providing the <em>Rab23<sup>opb2</sup></em> embryos that we used as positive controls. We thank members of the Caspary and Zwick labs for comments on the manuscript, Cheryl T. Strauss for editing, and the Emory-Georgia Research Alliance Genome Center (EGC), supported in part by PHS Grant UL1 RR025008 from the Clinical and Translational Science Award program, National Institutes of Health, National Center for Research Resources, for performing the Illumina sequencing runs. This work was supported by a Hitchings-Elion Career Award from the Burroughs Wellcome Fund (T.C.), an Emory University Research Council grant (T.C.), Ruth L. Kirschstein National Research Service Award 1F32HD060368-01A2 (V.L.H.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001669/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.001669/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article have been deposited with the NCBI Sequence Read Archive (SRA) under accession no. SRA048185.1 and with the Mouse Genome Informatics (MGI) database under accession nos. MGI:5296367, MGI:5296372, MGI:5296377, and MGI:5296378.</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-09-30"><span class="received-label">Received </span>September 30, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-11-21"><span class="accepted-label">Accepted </span>November 21, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Sun <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.1" data-doi="10.1146/annurev.genom.9.081307.164224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Acevedo-Arozena</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">ENU mutagenesis, a way forward to understand gene function.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DAcevedo-Arozena%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.spage%253D49%26rft.epage%253D69%26rft.atitle%253DENU%2Bmutagenesis%252C%2Ba%2Bway%2Bforward%2Bto%2Bunderstand%2Bgene%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164224%26rft_id%253Dinfo%253Apmid%252F18949851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18949851&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259629000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.2" data-doi="10.1038/nmeth1111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molla</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muzny</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nazareth</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Direct selection of human genomic loci by microarray hybridization.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">903</span>–<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DAlbert%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D903%26rft.epage%253D905%26rft.atitle%253DDirect%2Bselection%2Bof%2Bhuman%2Bgenomic%2Bloci%2Bby%2Bmicroarray%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1111%26rft_id%253Dinfo%253Apmid%252F17934467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17934467&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.3" data-doi="10.1534/genetics.110.124586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwander</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Rapid identification of a disease allele in mouse through whole genome sequencing and bulk segregation analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D187%26rft.spage%253D633%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.124586%26rft_id%253Dinfo%253Apmid%252F21196518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.124586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21196518&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288457800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.4" data-doi="10.1186/gb-2010-11-6-r62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bainbridge</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgess</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodesch</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole exome capture in solution with 3 Gbp of data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBainbridge%26rft.auinit1%253DM.%2BN.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253DR62%26rft.epage%253DR62%26rft.atitle%253DWhole%2Bexome%2Bcapture%2Bin%2Bsolution%2Bwith%2B3%2BGbp%2Bof%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-6-r62%26rft_id%253Dinfo%253Apmid%252F20565776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-6-r62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20565776&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.5" data-doi="10.1038/nmeth0105-63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bashiardes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veile</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helms</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowcock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Direct genomic selection.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBashiardes%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D69%26rft.atitle%253DDirect%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0105-63%26rft_id%253Dinfo%253Apmid%252F16152676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0105-63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16152676&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226754000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.6" data-doi="10.1007/s00216-008-2460-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schracke</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kränzle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stähler</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Targeted next-generation sequencing by specific capture of multiple genomic loci using low-volume microfluidic DNA arrays.</span> <abbr class="cit-jnl-abbrev">Anal. Bioanal. Chem.</abbr> <span class="cit-vol">393</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Band%2Bbioanalytical%2Bchemistry%26rft.stitle%253DAnal%2BBioanal%2BChem%26rft.aulast%253DBau%26rft.auinit1%253DS.%26rft.volume%253D393%26rft.issue%253D1%26rft.spage%253D171%26rft.epage%253D175%26rft.atitle%253DTargeted%2Bnext-generation%2Bsequencing%2Bby%2Bspecific%2Bcapture%2Bof%2Bmultiple%2Bgenomic%2Bloci%2Busing%2Blow-volume%2Bmicrofluidic%2BDNA%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00216-008-2460-7%26rft_id%253Dinfo%253Apmid%252F18958448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00216-008-2460-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18958448&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261831000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.7" data-doi="10.1007/978-3-540-75203-5_1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beutler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moresco</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The forward genetic dissection of afferent innate immunity.</span> <abbr class="cit-jnl-abbrev">Curr. Top. Microbiol. Immunol.</abbr> <span class="cit-vol">321</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bmicrobiology%2Band%2Bimmunology%26rft.stitle%253DCurr%2BTop%2BMicrobiol%2BImmunol%26rft.aulast%253DBeutler%26rft.auinit1%253DB.%26rft.volume%253D321%26rft.spage%253D3%26rft.epage%253D26%26rft.atitle%253DThe%2Bforward%2Bgenetic%2Bdissection%2Bof%2Bafferent%2Binnate%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-540-75203-5_1%26rft_id%253Dinfo%253Apmid%252F18727485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-540-75203-5_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18727485&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.8" data-doi="10.1002/dvdy.20853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caspary</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Uncovering the uncharacterized and unexpected: unbiased phenotype-driven screens in the mouse.</span> <abbr class="cit-jnl-abbrev">Dev. Dyn.</abbr> <span class="cit-vol">235</span>: <span class="cit-fpage">2412</span>–<span class="cit-lpage">2423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bdynamics%2B%253A%2B%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BAnatomists%26rft.stitle%253DDev%2BDyn%26rft.aulast%253DCaspary%26rft.auinit1%253DT.%26rft.volume%253D235%26rft.issue%253D9%26rft.spage%253D2412%26rft.epage%253D2423%26rft.atitle%253DUncovering%2Bthe%2Buncharacterized%2Band%2Bunexpected%253A%2Bunbiased%2Bphenotype-driven%2Bscreens%2Bin%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvdy.20853%26rft_id%253Dinfo%253Apmid%252F16724327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvdy.20853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16724327&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240262000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.9" data-doi="10.1016/j.coi.2006.07.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinuesa</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodnow</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">ENU-mutagenesis: insight into immune function and pathology.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Immunol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bimmunology%26rft.stitle%253DCurr%2BOpin%2BImmunol%26rft.aulast%253DCook%26rft.auinit1%253DM.%2BC.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D627%26rft.epage%253D633%26rft.atitle%253DENU-mutagenesis%253A%2Binsight%2Binto%2Bimmune%2Bfunction%2Band%2Bpathology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coi.2006.07.011%26rft_id%253Dinfo%253Apmid%252F16889948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coi.2006.07.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16889948&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240816300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.10" data-doi="10.1038/35084089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eggenschwiler</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espinoza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Rab23 is an essential negative regulator of the mouse Sonic hedgehog signalling pathway.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">412</span>: <span class="cit-fpage">194</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEggenschwiler%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D412%26rft.issue%253D6843%26rft.spage%253D194%26rft.epage%253D198%26rft.atitle%253DRab23%2Bis%2Ban%2Bessential%2Bnegative%2Bregulator%2Bof%2Bthe%2Bmouse%2BSonic%2Bhedgehog%2Bsignalling%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35084089%26rft_id%253Dinfo%253Apmid%252F11449277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35084089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11449277&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.11" data-doi="10.1186/gb-2011-12-9-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fairfield</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corrigan</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtain</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutation discovery in mice by whole exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DFairfield%26rft.auinit1%253DH.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DMutation%2Bdiscovery%2Bin%2Bmice%2Bby%2Bwhole%2Bexome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-9-r86%26rft_id%253Dinfo%253Apmid%252F21917142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-9-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21917142&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.12" data-doi="10.1093/nar/gkm078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fredriksson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banér</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahl</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">e47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm078%26rft_id%253Dinfo%253Apmid%252F17317684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjM1LzcvZTQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.13" data-doi="10.1073/pnas.0501071102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Garcia</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eggenschwiler</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caspary</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alcorn</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyler</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Analysis of mouse embryonic patterning and morphogenesis by forward genetics.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">5913</span>–<span class="cit-lpage">5919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501071102%26rft_id%253Dinfo%253Apmid%252F15755804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE3LzU5MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.14" data-doi="10.1038/nbt.1523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnirke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leproust</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGnirke%26rft.auinit1%253DA.%26rft.volume%253D27%26rft.issue%253D2%26rft.spage%253D182%26rft.epage%253D189%26rft.atitle%253DSolution%2Bhybrid%2Bselection%2Bwith%2Bultra-long%2Boligonucleotides%2Bfor%2Bmassively%2Bparallel%2Btargeted%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1523%26rft_id%253Dinfo%253Apmid%252F19182786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182786&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263209000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.15" data-doi="10.1186/gb-2011-12-9-r90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilton</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grati</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingham</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exome sequencing identifies a missense mutation in Isl1 associated with low penetrance otitis media in dearisch mice.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHilton%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253DR90%26rft.epage%253DR90%26rft.atitle%253DExome%2Bsequencing%2Bidentifies%2Ba%2Bmissense%2Bmutation%2Bin%2BIsl1%2Bassociated%2Bwith%2Blow%2Bpenetrance%2Botitis%2Bmedia%2Bin%2Bdearisch%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-9-r90%26rft_id%253Dinfo%253Apmid%252F21936904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-9-r90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21936904&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.16" data-doi="10.1038/ng.2007.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balija</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molla</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide in situ exon capture for selective resequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1522</span>–<span class="cit-lpage">1527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHodges%26rft.auinit1%253DE.%26rft.volume%253D39%26rft.issue%253D12%26rft.spage%253D1522%26rft.epage%253D1527%26rft.atitle%253DGenome-wide%2Bin%2Bsitu%2Bexon%2Bcapture%2Bfor%2Bselective%2Bresequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.42%26rft_id%253Dinfo%253Apmid%252F17982454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17982454&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251267400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horner</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caspary</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Creating a “hopeful monster”: mouse forward genetic screens.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">770</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DHorner%26rft.auinit1%253DV.%2BL.%26rft.volume%253D770%26rft.spage%253D313%26rft.epage%253D336%26rft.atitle%253DCreating%2Ba%2B%2522hopeful%2Bmonster%2522%253A%2Bmouse%2Bforward%2Bgenetic%2Bscreens.%26rft_id%253Dinfo%253Apmid%252F21805270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21805270&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.18" data-doi="10.1038/nature02061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huangfu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rakeman</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murcia</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niswander</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Hedgehog signalling in the mouse requires intraflagellar transport proteins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHuangfu%26rft.auinit1%253DD.%26rft.volume%253D426%26rft.issue%253D6962%26rft.spage%253D83%26rft.epage%253D87%26rft.atitle%253DHedgehog%2Bsignalling%2Bin%2Bthe%2Bmouse%2Brequires%2Bintraflagellar%2Btransport%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02061%26rft_id%253Dinfo%253Apmid%252F14603322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14603322&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186370800046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.19" data-doi="10.1038/ng.427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacoby</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gayral</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampshire</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayub</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">INPP5E mutations cause primary cilium signaling defects, ciliary instability and ciliopathies in human and mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJacoby%26rft.auinit1%253DM.%26rft.volume%253D41%26rft.issue%253D9%26rft.spage%253D1027%26rft.epage%253D1031%26rft.atitle%253DINPP5E%2Bmutations%2Bcause%2Bprimary%2Bcilium%2Bsignaling%2Bdefects%252C%2Bciliary%2Binstability%2Band%2Bciliopathies%2Bin%2Bhuman%2Band%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.427%26rft_id%253Dinfo%253Apmid%252F19668215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668215&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.20" data-doi="10.1038/nmeth.1419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mamanova</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozarewa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Target-enrichment strategies for next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMamanova%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D118%26rft.atitle%253DTarget-enrichment%2Bstrategies%2Bfor%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1419%26rft_id%253Dinfo%253Apmid%252F20111037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20111037&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274086200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.21" data-doi="10.1101/gr.4563306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwyer</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Utilization of a whole genome SNP panel for efficient genetic mapping in the mouse.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4563306%26rft_id%253Dinfo%253Apmid%252F16461637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNDM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.22" data-doi="10.1038/nature08250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flygare</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Targeted capture and massively parallel sequencing of 12 human exomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNg%26rft.auinit1%253DS.%2BB.%26rft.volume%253D461%26rft.issue%253D7261%26rft.spage%253D272%26rft.epage%253D276%26rft.atitle%253DTargeted%2Bcapture%2Band%2Bmassively%2Bparallel%2Bsequencing%2Bof%2B12%2Bhuman%2Bexomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08250%26rft_id%253Dinfo%253Apmid%252F19684571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19684571&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269654600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.23" data-doi="10.1038/ng.499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckingham</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabor</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Exome sequencing identifies the cause of a mendelian disorder.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNg%26rft.auinit1%253DS.%2BB.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D35%26rft.atitle%253DExome%2Bsequencing%2Bidentifies%2Bthe%2Bcause%2Bof%2Ba%2Bmendelian%2Bdisorder.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.499%26rft_id%253Dinfo%253Apmid%252F19915526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19915526&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273055100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.24" data-doi="10.1371/journal.pone.0006659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nikolaev</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iseli</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robyr</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rougemont</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detection of genomic variation by selection of a 9 mb DNA region and high throughput sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNikolaev%26rft.auinit1%253DS.%2BI.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6659%26rft.epage%253De6659%26rft.atitle%253DDetection%2Bof%2Bgenomic%2Bvariation%2Bby%2Bselection%2Bof%2Ba%2B9%2Bmb%2BDNA%2Bregion%2Band%2Bhigh%2Bthroughput%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006659%26rft_id%253Dinfo%253Apmid%252F19684856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19684856&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.25" data-doi="10.1111/j.1469-1809.2009.00530.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okou</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locke</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinberg</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Athri</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Combining microarray-based genomic selection (MGS) with the Illumina Genome Analyzer platform to sequence diploid target regions.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">502</span>–<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DOkou%26rft.auinit1%253DD.%2BT.%26rft.volume%253D73%26rft.issue%253DPt%2B5%26rft.spage%253D502%26rft.epage%253D513%26rft.atitle%253DCombining%2Bmicroarray-based%2Bgenomic%2Bselection%2B%2528MGS%2529%2Bwith%2Bthe%2BIllumina%2BGenome%2BAnalyzer%2Bplatform%2Bto%2Bsequence%2Bdiploid%2Btarget%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.2009.00530.x%26rft_id%253Dinfo%253Apmid%252F19573206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.2009.00530.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19573206&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268789500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.26" data-doi="10.1038/nmeth1109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okou</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinberg</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Middle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Microarray-based genomic selection for high-throughput resequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">907</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DOkou%26rft.auinit1%253DD.%2BT.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D907%26rft.epage%253D909%26rft.atitle%253DMicroarray-based%2Bgenomic%2Bselection%2Bfor%2Bhigh-throughput%2Bresequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1109%26rft_id%253Dinfo%253Apmid%252F17934469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17934469&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shetty</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horner</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedoyan</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Microarray oligonucleotide probe designer (MOPeD): a web service.</span> <abbr class="cit-jnl-abbrev">Open Access Bioinformatics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DOpen%2BAccess%2BBioinformatics%26rft.aulast%253DPatel%26rft.auinit1%253DV.%2BC.%26rft.volume%253D2%26rft.issue%253D2010%26rft.spage%253D145%26rft.epage%253D155%26rft.atitle%253DMicroarray%2Boligonucleotide%2Bprobe%2Bdesigner%2B%2528MOPeD%2529%253A%2BA%2Bweb%2Bservice.%26rft_id%253Dinfo%253Apmid%252F21379402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21379402&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.28" data-doi="10.1038/nmeth1110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porreca</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multiplex amplification of large sets of human exons.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPorreca%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D931%26rft.epage%253D936%26rft.atitle%253DMultiplex%2Bamplification%2Bof%2Blarge%2Bsets%2Bof%2Bhuman%2Bexons.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1110%26rft_id%253Dinfo%253Apmid%252F17934468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17934468&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.29" data-doi="10.1242/dev.02473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rakeman</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Axis specification and morphogenesis in the mouse embryo require Nap1, a regulator of WAVE-mediated actin branching.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">3075</span>–<span class="cit-lpage">3083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.02473%26rft_id%253Dinfo%253Apmid%252F16831833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMzLzE2LzMwNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8yLzEvMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.30" data-doi="10.1038/nbt1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DShendure%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D1135%26rft.epage%253D1145%26rft.atitle%253DNext-generation%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1486%26rft_id%253Dinfo%253Apmid%252F18846087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846087&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259926000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.31" data-doi="10.1038/nrg1325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Advanced sequencing technologies: methods and goals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DShendure%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D335%26rft.epage%253D344%26rft.atitle%253DAdvanced%2Bsequencing%2Btechnologies%253A%2Bmethods%2Band%2Bgoals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1325%26rft_id%253Dinfo%253Apmid%252F15143316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15143316&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221139100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.32" data-doi="10.1186/1471-2105-11-471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shetty</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Athri</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horner</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinberg</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeqAnt: a web service to rapidly identify and annotate DNA sequence variations.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DShetty%26rft.auinit1%253DA.%2BC.%26rft.volume%253D11%26rft.spage%253D471%26rft.epage%253D471%26rft.atitle%253DSeqAnt%253A%2Ba%2Bweb%2Bservice%2Bto%2Brapidly%2Bidentify%2Band%2Bannotate%2BDNA%2Bsequence%2Bvariations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-471%26rft_id%253Dinfo%253Apmid%252F20854673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20854673&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.33" data-doi="10.1534/genetics.111.126862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stottmann</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turbe-Doan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Driver</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Focusing forward genetics: a tripartite ENU screen for neurodevelopmental mutations in the mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.126862%26rft_id%253Dinfo%253Apmid%252F21515572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNjE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMi8xLzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.34" data-doi="10.1186/gb-2009-10-10-r116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tewhey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pabón-Peña</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009a</span> <span class="cit-article-title">Enrichment of sequencing targets from the human genome by solution hybridization.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DTewhey%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253DR116%26rft.epage%253DR116%26rft.atitle%253DEnrichment%2Bof%2Bsequencing%2Btargets%2Bfrom%2Bthe%2Bhuman%2Bgenome%2Bby%2Bsolution%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-10-r116%26rft_id%253Dinfo%253Apmid%252F19835619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-10-r116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19835619&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.35" data-doi="10.1038/nbt.1583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tewhey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libby</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medkova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009b</span> <span class="cit-article-title">Microdroplet-based PCR enrichment for large-scale targeted sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1025</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTewhey%26rft.auinit1%253DR.%26rft.volume%253D27%26rft.issue%253D11%26rft.spage%253D1025%26rft.epage%253D1031%26rft.atitle%253DMicrodroplet-based%2BPCR%2Benrichment%2Bfor%2Blarge-scale%2Btargeted%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1583%26rft_id%253Dinfo%253Apmid%252F19881494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19881494&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271524700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.36" data-doi="10.1038/nmeth.f.248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickerson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Massively parallel exon capture and library-free resequencing across 16 genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DTurner%26rft.auinit1%253DE.%2BH.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D315%26rft.epage%253D316%26rft.atitle%253DMassively%2Bparallel%2Bexon%2Bcapture%2Band%2Blibrary-free%2Bresequencing%2Bacross%2B16%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.f.248%26rft_id%253Dinfo%253Apmid%252F19349981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.f.248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349981&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F1%2F143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265661600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.1.143.37" data-doi="10.1073/pnas.0813400106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alpert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Massively parallel sequencing identifies the gene Megf8 with ENU-induced mutation causing heterotaxy.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">3219</span>–<span class="cit-lpage">3224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0813400106%26rft_id%253Dinfo%253Apmid%252F19218456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzkvMzIxOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzIvMS8xNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/1/143.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/1/131" title="Many Saccharomyces cerevisiae Cell Wall Protein Encoding Genes Are Coregulated by Mss11, but Cellular Adhesion Phenotypes Appear Only Flo Protein Dependent" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/2/151" title="Facilitating Discovery: The Role of Society Journals in Collaborative Science" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/1"><span>Volume 2 Issue 1, January 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/1.cover.gif?itok=D5RR8siB" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/1/143" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4DQFMu1BGCsspN6MuFabmr7wyK-u3Zin5eTZH-8tH_c" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/1/143" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/1/143" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/1/143" class="active" data-icon-position="" data-hide-link-title="0">Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459224" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XmBijSM6-maOxccrrj1UoXnmdiOsxRd5RM7SwlN33gQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/1/143.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/1/143" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NW_j6taFQNMLuI_Ul0N3SHplsLYALsIwK-1Jur9g-S4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/1/143" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/1/143.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459224" id="node459224--2" data-pisa="ggg;2/1/143" data-pisa-master="ggg;2/1/143" data-apath="/ggg/2/1/143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miao Sun</span>, <span class="highwire-citation-author" data-delta="1">Kajari Mondal</span>, <span class="highwire-citation-author" data-delta="2">Viren Patel</span>, <span class="highwire-citation-author" data-delta="3">Vanessa L. Horner</span>, <span class="highwire-citation-author" data-delta="4">Alyssa B. Long</span>, <span class="highwire-citation-author" data-delta="5">David J. Cutler</span>, <span class="highwire-citation-author" data-delta="6">Tamara Caspary</span>, <span class="highwire-citation-author" data-delta="7">Michael E. Zwick</span> and <span class="highwire-citation-author" data-delta="8">B. J. Andrews</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">143-150;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001669 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459224--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miao Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miao%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiao%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kajari Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kajari%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKajari%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Viren Patel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Viren%2BPatel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patel%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AViren%2BPatel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vanessa L. Horner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BL.%2BHorner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Horner%20VL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BL.%2BHorner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alyssa B. Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alyssa%2BB.%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlyssa%2BB.%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David J. Cutler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BCutler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cutler%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BCutler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tamara Caspary </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamara%2BCaspary%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caspary%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATamara%2BCaspary%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzwick@emory.edu" class="" data-icon-position="" data-hide-link-title="0">mzwick@emory.edu</a>
<a href="mailto:tcaspar@emory.edu" class="" data-icon-position="" data-hide-link-title="0">tcaspar@emory.edu</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michael E. Zwick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BZwick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zwick%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BE.%2BZwick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzwick@emory.edu" class="" data-icon-position="" data-hide-link-title="0">mzwick@emory.edu</a>
<a href="mailto:tcaspar@emory.edu" class="" data-icon-position="" data-hide-link-title="0">tcaspar@emory.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">B. J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/1/143" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/1/143&amp;title=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/1/143&amp;title=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/1/143&amp;title=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/1/143&amp;text=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/1/143&amp;title=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/1/143&amp;t=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/1/143&amp;title=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/1/143&amp;title=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459224/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459224" id="node459224--3" data-pisa="ggg;2/1/143" data-pisa-master="ggg;2/1/143" data-apath="/ggg/2/1/143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multiplex Chromosomal Exome Sequencing Accelerates Identification of ENU-Induced Mutations in the Mouse</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Miao Sun</span>, <span class="highwire-citation-author" data-delta="1">Kajari Mondal</span>, <span class="highwire-citation-author" data-delta="2">Viren Patel</span>, <span class="highwire-citation-author" data-delta="3">Vanessa L. Horner</span>, <span class="highwire-citation-author" data-delta="4">Alyssa B. Long</span>, <span class="highwire-citation-author" data-delta="5">David J. Cutler</span>, <span class="highwire-citation-author" data-delta="6">Tamara Caspary</span>, <span class="highwire-citation-author" data-delta="7">Michael E. Zwick</span> and <span class="highwire-citation-author" data-delta="8">B. J. Andrews</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">143-150;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.001669 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459224--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miao Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miao%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiao%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kajari Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kajari%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKajari%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Viren Patel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Viren%2BPatel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patel%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AViren%2BPatel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vanessa L. Horner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BL.%2BHorner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Horner%20VL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BL.%2BHorner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alyssa B. Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alyssa%2BB.%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlyssa%2BB.%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David J. Cutler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BCutler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cutler%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BCutler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Tamara Caspary </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamara%2BCaspary%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caspary%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATamara%2BCaspary%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzwick@emory.edu" class="" data-icon-position="" data-hide-link-title="0">mzwick@emory.edu</a>
<a href="mailto:tcaspar@emory.edu" class="" data-icon-position="" data-hide-link-title="0">tcaspar@emory.edu</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Michael E. Zwick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BE.%2BZwick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zwick%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BE.%2BZwick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mzwick@emory.edu" class="" data-icon-position="" data-hide-link-title="0">mzwick@emory.edu</a>
<a href="mailto:tcaspar@emory.edu" class="" data-icon-position="" data-hide-link-title="0">tcaspar@emory.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">B. J. Andrews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia 30322</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BJ.%2BAndrews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andrews%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BJ.%2BAndrews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=143&amp;rft.epage=150&amp;rft.atitle=Multiplex%20Chromosomal%20Exome%20Sequencing%20Accelerates%20Identification%20of%20ENU-Induced%20Mutations%20in%20the%20Mouse&amp;rft.volume=2&amp;rft.issue=1&amp;rft.date=2012-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sun%2C+Miao&amp;rft.au=Mondal%2C+Kajari&amp;rft.au=Patel%2C+Viren&amp;rft.au=Horner%2C+Vanessa+L.&amp;rft.au=Long%2C+Alyssa+B.&amp;rft.au=Cutler%2C+David+J.&amp;rft.au=Caspary%2C+Tamara&amp;rft.au=Zwick%2C+Michael+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459224/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459224/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459224/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459224/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459224/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459224/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459224/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459224/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459224/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459224/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459224/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459224/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/1/143" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/1/143.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/1/143.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/1/143.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/1/143.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
