<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/4/303" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/4/303.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/4/303.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/4/303.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/4.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/4/303.atom" />
<meta name="HW.pisa" content="ggg;1/4/303" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in Neurospora crassa" />
<meta name="DC.Identifier" content="10.1534/g3.111.000307" />
<meta name="DC.Date" content="2011-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2011 McCluskey et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Classical forward genetics has been foundational to modern biology, and has been the paradigm for characterizing the role of genes in shaping phenotypes for decades. In recent years, reverse genetics has been used to identify the functions of genes, via the intentional introduction of variation and subsequent evaluation in physiological, molecular, and even population contexts. These approaches are complementary and whole genome analysis serves as a bridge between the two. We report in this article the whole genome sequencing of eighteen classical mutant strains of Neurospora crassa and the putative identification of the mutations associated with corresponding mutant phenotypes. Although some strains carry multiple unique nonsynonymous, nonsense, or frameshift mutations, the combined power of limiting the scope of the search based on genetic markers and of using a comparative analysis among the eighteen genomes provides strong support for the association between mutation and phenotype. For ten of the mutants, the mutant phenotype is recapitulated in classical or gene deletion mutants in Neurospora or other filamentous fungi. From thirteen to 137 nonsense mutations are present in each strain and indel sizes are shown to be highly skewed in gene coding sequence. Significant additional genetic variation was found in the eighteen mutant strains, and this variability defines multiple alleles of many genes. These alleles may be useful in further genetic and molecular analysis of known and yet-to-be-discovered functions and they invite new interpretations of molecular and genetic interactions in classical mutant strains." />
<meta name="DC.Contributor" content="Kevin McCluskey" />
<meta name="DC.Contributor" content="Aric E. Wiest" />
<meta name="DC.Contributor" content="Igor V. Grigoriev" />
<meta name="DC.Contributor" content="Anna Lipzen" />
<meta name="DC.Contributor" content="Joel Martin" />
<meta name="DC.Contributor" content="Wendy Schackwitz" />
<meta name="DC.Contributor" content="Scott E. Baker" />
<meta name="article:published_time" content="2011-09-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in Neurospora crassa" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Classical forward genetics has been foundational to modern biology, and has been the paradigm for characterizing the role of genes in shaping phenotypes for decades. In recent years, reverse genetics has been used to identify the functions of genes, via the intentional introduction of variation and subsequent evaluation in physiological, molecular, and even population contexts. These approaches are complementary and whole genome analysis serves as a bridge between the two. We report in this article the whole genome sequencing of eighteen classical mutant strains of &lt;i&gt;Neurospora crassa&lt;/i&gt; and the putative identification of the mutations associated with corresponding mutant phenotypes. Although some strains carry multiple unique nonsynonymous, nonsense, or frameshift mutations, the combined power of limiting the scope of the search based on genetic markers and of using a comparative analysis among the eighteen genomes provides strong support for the association between mutation and phenotype. For ten of the mutants, the mutant phenotype is recapitulated in classical or gene deletion mutants in &lt;i&gt;Neurospora&lt;/i&gt; or other filamentous fungi. From thirteen to 137 nonsense mutations are present in each strain and indel sizes are shown to be highly skewed in gene coding sequence. Significant additional genetic variation was found in the eighteen mutant strains, and this variability defines multiple alleles of many genes. These alleles may be useful in further genetic and molecular analysis of known and yet-to-be-discovered functions and they invite new interpretations of molecular and genetic interactions in classical mutant strains.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/09/01" />
<meta name="citation_mjid" content="ggg;1/4/303" />
<meta name="citation_id" content="1/4/303" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/4/303" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/4/303.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/4/303.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/4/303.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000307" />
<meta name="citation_pmid" content="22384341" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="4" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="303" />
<meta name="citation_lastpage" content="316" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kevin McCluskey" />
<meta name="citation_author_institution" content="Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110" />
<meta name="citation_author_email" content="mccluskeyk@umkc.edu" />
<meta name="citation_author_email" content="scott.baker@pnnl.gov" />
<meta name="citation_author" content="Aric E. Wiest" />
<meta name="citation_author_institution" content="Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110" />
<meta name="citation_author" content="Igor V. Grigoriev" />
<meta name="citation_author_institution" content="U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and" />
<meta name="citation_author" content="Anna Lipzen" />
<meta name="citation_author_institution" content="U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and" />
<meta name="citation_author" content="Joel Martin" />
<meta name="citation_author_institution" content="U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and" />
<meta name="citation_author" content="Wendy Schackwitz" />
<meta name="citation_author_institution" content="U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and" />
<meta name="citation_author" content="Scott E. Baker" />
<meta name="citation_author_institution" content="U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and" />
<meta name="citation_author_institution" content="Chemical and Biological Process Development Group, Pacific Northwest National Laboratory, Richland, Washington 99354" />
<meta name="citation_author_email" content="mccluskeyk@umkc.edu" />
<meta name="citation_author_email" content="scott.baker@pnnl.gov" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=K. C. Atwood;citation_author=F. Mukai;citation_title=Survival and mutation in Neurospora exposed at nuclear detonations.;citation_pages=295-314;citation_volume=88;citation_year=1954" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. F. Baer;citation_author=M. M. Miyamoto;citation_author=D. R. Denver;citation_title=Mutation rate variation in multicellular eukaryotes: causes and consequences.;citation_pages=619-631;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17637734;citation_doi=10.1038/nrg2158" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=G. W. Beadle;citation_author=E. L. Tatum;citation_title=Genetic Control of Biochemical Reactions in Neurospora;citation_pages=499-506;citation_volume=27;citation_year=1941;citation_issue=11;citation_pmid=16588492;citation_doi=10.1073/pnas.27.11.499" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=A. Beauvais;citation_author=J. M. Bruneau;citation_author=P. C. Mol;citation_author=M. J. Buitrago;citation_author=R. Legrand;citation_title=Glucan Synthase Complex of Aspergillus fumigatus;citation_pages=2273-2279;citation_volume=183;citation_year=2001;citation_issue=7;citation_pmid=11244067;citation_doi=10.1128/JB.183.7.2273-2279.2001" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=K. A. Borkovich;citation_author=L. A. Alex;citation_author=O. Yarden;citation_author=M. Freitag;citation_author=G. E. Turner;citation_title=Lessons from the Genome Sequence of Neurospora crassa: Tracing the Path from Genomic Blueprint to Multicellular Organism;citation_pages=1-108;citation_volume=68;citation_year=2004;citation_issue=1;citation_pmid=15007097;citation_doi=10.1128/MMBR.68.1.1-108.2004" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=S. M. Bowman;citation_author=A. Piwowar;citation_author=M. Al Dabbous;citation_author=J. Vierula;citation_author=S. J. Free;citation_title=Mutational Analysis of the Glycosylphosphatidylinositol (GPI) Anchor Pathway Demonstrates that GPI-Anchored Proteins Are Required for Cell Wall Biogenesis and Normal Hyphal Growth in Neurospora crassa;citation_pages=587-600;citation_volume=5;citation_year=2006;citation_issue=3;citation_pmid=16524913;citation_doi=10.1128/EC.5.3.587-600.2006" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=K. Chen;citation_author=J. W. Wallis;citation_author=M. D. McLellan;citation_author=D. E. Larson;citation_author=J. M. Kalicki;citation_title=BreakDancer: an algorithm for high-resolution mapping of genomic structural variation.;citation_pages=677-681;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668202;citation_doi=10.1038/nmeth.1363" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. V. Colot;citation_author=G. Park;citation_author=G. E. Turner;citation_author=C. Ringelberg;citation_author=C. M. Crew;citation_title=A high-throughput gene knockout procedure for Neurospora reveals functions for multiple transcription factors.;citation_volume=103;citation_year=2006;citation_pmid=16801547;citation_doi=10.1073/pnas.0601456103" />
<meta name="citation_reference" content="citation_journal_title=Annales de microbiologie;citation_journal_abbrev=Ann Microbiol (Paris);citation_author=N. Coniordos;citation_author=G. Turian;citation_title=[Studies on the conidial differentiation of &amp;quot;Neurospora crassa&amp;quot;. IV. Chemo-structural modifications of the wall in the wild type and two aconidial mutants];citation_pages=5-28;citation_volume=124;citation_year=1973;citation_issue=1;citation_pmid=4269228" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. H. Davis;citation_author=D. D. Perkins;citation_title=Timeline: Neurospora: a model of model microbes.;citation_pages=397-403;citation_volume=3;citation_year=2002;citation_issue=5;citation_pmid=11988765;citation_doi=10.1038/nrg797" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=F. J. de Serres;citation_author=B. B. Webber;citation_title=Quantitative and qualitative comparisons of spontaneous and radiation-induced specific-locus mutation in the ad-3 region of heterokaryon 12 of Neurospora crassa.;citation_pages=37-52;citation_volume=375;citation_year=1997;citation_issue=1;citation_pmid=9129678" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. G. Dieterle;citation_author=A. E. Wiest;citation_author=M. Plamann;citation_author=K. McCluskey;citation_title=Characterization of the temperature-sensitive mutations un-7 and png-1 in Neurospora crassa.;citation_pages=e10703-e10703;citation_volume=5;citation_year=2010;citation_issue=5;citation_pmid=20502699;citation_doi=10.1371/journal.pone.0010703" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. W. Drake;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_author=J. F. Crow;citation_title=Rates of spontaneous mutation.;citation_pages=1667-1686;citation_volume=148;citation_year=1998;citation_pmid=9560386" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=J. C. Dunlap;citation_author=K. A. Borkovich;citation_author=M. R. Henn;citation_author=G. E. Turner;citation_author=M. S. Sachs;citation_title=Enabling a community to dissect an organism: overview of the Neurospora functional genomics project.;citation_pages=49-96;citation_volume=57;citation_year=2007;citation_pmid=17352902;citation_doi=10.1016/S0065-2660(06)57002-6" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=I. Engh;citation_author=M. Nowrousian;citation_author=U. Kuck;citation_title=Regulation of melanin biosynthesis via the dihydroxynaphthalene pathway is dependent on sexual development in the ascomycete Sordaria macrospora.;citation_volume=275;citation_year=2007;citation_pmid=17681008;citation_doi=10.1111/j.1574-6968.2007.00867.x" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=A. Fleissner;citation_author=S. Sarkar;citation_author=D. J. Jacobson;citation_author=M. G. Roca;citation_author=N. D. Read;citation_title=The so Locus Is Required for Vegetative Cell Fusion and Postfertilization Events in Neurospora crassa;citation_pages=920-930;citation_volume=4;citation_year=2005;citation_issue=5;citation_pmid=15879526;citation_doi=10.1128/EC.4.5.920-930.2005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Freitag;citation_author=R. L. Williams;citation_author=G. O. Kothe;citation_author=E. U. Selker;citation_title=A cytosine methyltransferase homologue is essential for repeat-induced point mutation in Neurospora crassa.;citation_volume=99;citation_year=2002;citation_pmid=12072568;citation_doi=10.1073/pnas.132212899" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=K. Fujimura;citation_author=K. Tanaka;citation_author=A. Toh-e;citation_title=A dominant interfering mutation in RAS1 of Saccharomyces cerevisiae.;citation_pages=280-286;citation_volume=241;citation_year=1993;citation_issue=3-4;citation_pmid=8246881" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Galagan;citation_author=S. E. Calvo;citation_author=K. A. Borkovich;citation_author=E. U. Selker;citation_author=N. D. Read;citation_title=The genome sequence of the filamentous fungus Neurospora crassa.;citation_pages=859-868;citation_volume=422;citation_year=2003;citation_issue=6934;citation_pmid=12712197;citation_doi=10.1038/nature01554" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Garnjobst;citation_author=E. L. Tatum;citation_title=A survey of new morphological mutants in Neurospora crassa.;citation_pages=579-604;citation_volume=57;citation_year=1967;citation_pmid=5583730" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=Y. S. Ha;citation_author=S. F. Covert;citation_author=M. Momany;citation_title=FsFKS1, the 1,3-{beta}-Glucan Synthase from the Caspofungin-Resistant Fungus Fusarium solani;citation_pages=1036-1042;citation_volume=5;citation_year=2006;citation_issue=7;citation_pmid=16835448;citation_doi=10.1128/EC.00030-06" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=H. E. Hallen;citation_author=F. Trail;citation_title=The L-type calcium ion channel cch1 affects ascospore discharge and mycelial growth in the filamentous fungus Gibberella zeae (anamorph Fusarium graminearum).;citation_volume=7;citation_year=2008;citation_pmid=18083828;citation_doi=10.1128/EC.00248-07" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. Hobert;citation_title=The impact of whole genome sequencing on model system genetics: get ready for the ride.;citation_pages=317-319;citation_volume=184;citation_year=2010;citation_pmid=20103786;citation_doi=10.1534/genetics.109.112938" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. H. Horowitz;citation_author=M. Fling;citation_author=H. Macleod;citation_author=N. Sueoka;citation_title=A GENETIC STUDY OF TWO NEW STRUCTURAL FORMS OF TYROSINASE IN NEUROSPORA;citation_pages=1015-1024;citation_volume=46;citation_year=1961;citation_issue=8;citation_pmid=13715943" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=H. B. Howe;citation_author=E. W. Benson;citation_title=A perithecial color mutant of Neurospora crassa.;citation_pages=79-83;citation_volume=131;citation_year=1974;citation_issue=1;citation_pmid=4277338;citation_doi=10.1007/BF00269389" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=J. L. Kadrmas;citation_author=M. C. Beckerle;citation_title=The LIM domain: from the cytoskeleton to the nucleus.;citation_pages=920-931;citation_volume=5;citation_year=2004;citation_issue=11;citation_pmid=15520811;citation_doi=10.1038/nrm1499" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Le Crom;citation_author=W. Schackwitz;citation_author=L. Pennacchio;citation_author=J. K. Magnuson;citation_author=D. E. Culley;citation_title=Tracking the roots of cellulase hyperproduction by the fungus Trichoderma reesei using massively parallel DNA sequencing.;citation_volume=106;citation_year=2009;citation_pmid=19805272;citation_doi=10.1073/pnas.0905848106" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=S. B. Lee;citation_author=M. G. Milgroom;citation_author=J. W. Taylor;citation_title=A rapid, high yield mini-prep method for isolation of total genomic DNA from fungi.;citation_pages=23;citation_volume=1988;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Li;citation_author=J. Ruan;citation_author=R. Durbin;citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores.;citation_volume=18;citation_year=2008;citation_pmid=18714091;citation_doi=10.1101/gr.078212.108" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=Y. Li;citation_author=S. Liang;citation_author=X. Yan;citation_author=H. Wang;citation_author=D. Li;citation_title=Characterization of MoLDB1 required for vegetative growth, infection-related morphogenesis, and pathogenicity in the rice blast fungus Magnaporthe oryzae.;citation_pages=1260-1274;citation_volume=23;citation_year=2010;citation_issue=10;citation_pmid=20831406" />
<meta name="citation_reference" content="citation_journal_title=Bot. Gaz.;citation_author=C. C. Lindegren;citation_author=V. Beanfield;citation_author=R. Barber;citation_title=Increasing the fertility of Neurospora by selective inbreeding.;citation_pages=592-599;citation_volume=100;citation_year=1939" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=A. Manford;citation_author=T. Xia;citation_author=A. K. Saxena;citation_author=C. Stefan;citation_author=F. Hu;citation_title=Crystal structure of the yeast Sac1: implications for its phosphoinositide phosphatase function.;citation_pages=1489-1498;citation_volume=29;citation_year=2010;citation_issue=9;citation_pmid=20389282;citation_doi=10.1038/emboj.2010.57" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=K. McCluskey;citation_author=S. A. Walker;citation_author=R. L. Yedlin;citation_author=D. Madole;citation_author=M. Plamann;citation_title=Complementation of un-16 and the development of a selectable marker for transformation of Neurospora crassa.;citation_pages=9-11;citation_volume=54;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. W. Messer;citation_author=P. F. Arndt;citation_title=The majority of recent short DNA insertions in the human genome are tandem duplications.;citation_volume=24;citation_year=2007;citation_pmid=17322553;citation_doi=10.1093/molbev/msm035" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=O. M. Mylyk;citation_author=S. F. Threlkeld;citation_title=A genetic study of female sterility in Neurospora crassa.;citation_pages=91-102;citation_volume=24;citation_year=1974;citation_issue=1;citation_pmid=4279838" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Nakamura;citation_author=T. Egashira;citation_title=Genetically mixed perithecia in Neurospora.;citation_pages=1129-1130;citation_volume=190;citation_year=1961;citation_pmid=13727611;citation_doi=10.1038/1901129a0" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=G. Park;citation_author=S. Pan;citation_author=K. A. Borkovich;citation_title=Mitogen-activated protein kinase cascade required for regulation of development and secondary metabolism in Neurospora crassa.;citation_volume=7;citation_year=2008;citation_pmid=18849472;citation_doi=10.1128/EC.00466-07" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=E. Peiter;citation_author=M. Fischer;citation_author=K. Sidaway;citation_author=S. K. Roberts;citation_author=D. Sanders;citation_title=The Saccharomyces cerevisiae Ca2+ channel Cch1pMid1p is essential for tolerance to cold stress and iron toxicity.;citation_pages=5697-5703;citation_volume=579;citation_year=2005;citation_issue=25;citation_pmid=16223494;citation_doi=10.1016/j.febslet.2005.09.058" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. D. Perkins;citation_title=NEW MARKERS AND MULTIPLE POINT LINKAGE DATA IN NEUROSPORA;citation_pages=1185-1208;citation_volume=44;citation_year=1959;citation_issue=6;citation_pmid=17247885" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. D. Perkins;citation_author=C. Ishitani;citation_title=LINKAGE DATA FOR GROUP III MARKERS IN NEUROSPORA;citation_pages=1209-1213;citation_volume=44;citation_year=1959;citation_issue=6;citation_pmid=17247886" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Genet. Cytol.;citation_author=D. D. Perkins;citation_author=M. Glassey;citation_author=B. A. Bloom;citation_title=New data on markers and rearrangements in Neurospora.;citation_pages=187-205;citation_volume=4;citation_year=1962;citation_doi=10.1139/g62-024" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=D. D. Perkins;citation_author=A. Radford;citation_author=D. Newmeyer;citation_author=M. Bjorkman;citation_title=Chromosomal loci of Neurospora crassa.;citation_pages=426-570;citation_volume=46;citation_year=1982;citation_issue=4;citation_pmid=6219280" />
<meta name="citation_reference" content="citation_title=The Neurospora Compendium: Chromosomal Loci.;citation_author=D. D. Perkins;citation_author=A. Radford;citation_author=M. S. Sachs;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot Cell.;citation_author=K. R. Pomraning;citation_author=K. M. Smith;citation_author=M. Freitag;citation_title=Bulk segregant analysis followed by high-throughput sequencing reveals the Neurospora cell cycle gene, ndc-1, to be allelic with the gene for ornithine decarboxylase, spe-1.;citation_volume=10;citation_year=2011;citation_pmid=21515825;citation_doi=10.1128/EC.00016-11" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=M. P. Rivas;citation_author=B. G. Kearns;citation_author=Z. Xie;citation_author=S. Guo;citation_author=M. C. Sekar;citation_title=Pleiotropic Alterations in Lipid Metabolism in Yeast sac1 Mutants: Relationship to &quot;Bypass Sec14p&quot; and Inositol Auxotrophy;citation_pages=2235-2250;citation_volume=10;citation_year=1999;citation_issue=7;citation_pmid=10397762;citation_doi=10.1091/mbc.10.7.2235" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Sarin;citation_author=V. Bertrand;citation_author=H. Bigelow;citation_author=A. Boyanov;citation_author=M. Doitsidou;citation_title=Analysis of multiple ethyl methanesulfonate-mutagenized Caenorhabditis elegans strains by whole-genome sequencing.;citation_pages=417-430;citation_volume=185;citation_year=2010;citation_pmid=20439776;citation_doi=10.1534/genetics.110.116319" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=S. Seiler;citation_author=M. Plamann;citation_title=The genetic basis of cellular morphogenesis in the filamentous fungus Neurospora crassa.;citation_volume=14;citation_year=2003;citation_pmid=12960438;citation_doi=10.1091/mbc.E02-07-0433" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. K. Shiu;citation_author=D. Zickler;citation_author=N. B. Raju;citation_author=G. Ruprich-Robert;citation_author=R. L. Metzenberg;citation_title=SAD-2 is required for meiotic silencing by unpaired DNA and perinuclear localization of SAD-1 RNA-directed RNA polymerase.;citation_volume=103;citation_year=2006;citation_pmid=16461906;citation_doi=10.1073/pnas.0508896103" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Smith;citation_author=A. R. Quinlan;citation_author=H. E. Peckham;citation_author=K. Makowsky;citation_author=W. Tao;citation_title=Rapid whole-genome mutational profiling using next-generation sequencing technologies.;citation_volume=18;citation_year=2008;citation_pmid=18775913;citation_doi=10.1101/gr.077776.108" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. L. Springer;citation_author=C. Yanofsky;citation_title=A morphological and genetic analysis of conidiophore development in Neurospora crassa.;citation_pages=559-571;citation_volume=3;citation_year=1989;citation_issue=4;citation_pmid=2524423;citation_doi=10.1101/gad.3.4.559" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=S. T. Tan;citation_author=C. C. Ho;citation_title=A gene controlling the early development of protoperithecium in Neurospora crassa.;citation_pages=158-161;citation_volume=107;citation_year=1970;citation_issue=2;citation_pmid=4938066;citation_doi=10.1007/BF00333631" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. C. Turner;citation_author=D. D. Perkins;citation_title=Spore killer, a chromosomal factor in neurospora that kills meiotic products not containing it.;citation_pages=587-606;citation_volume=93;citation_year=1979;citation_pmid=17248973" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=B. C. Turner;citation_author=D. D. Perkins;citation_author=A. Fairfield;citation_title=Neurospora from natural populations: a global study.;citation_pages=67-92;citation_volume=32;citation_year=2001;citation_issue=2;citation_pmid=11352529;citation_doi=10.1006/fgbi.2001.1247" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Weijer;citation_author=N. V. Vigfusson;citation_title=Sexuality in Neurospora crassa. I. Mutations to male sterility.;citation_pages=191-204;citation_volume=19;citation_year=1972;citation_issue=3;citation_pmid=4263083" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=L. Wik;citation_author=M. Karlsson;citation_author=H. Johannesson;citation_title=The evolutionary trajectory of the mating-type (mat) genes in Neurospora relates to reproductive behavior of taxa.;citation_pages=109-109;citation_volume=8;citation_year=2008;citation_pmid=18405383;citation_doi=10.1186/1471-2148-8-109" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in Neurospora crassa" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/4.cover.gif" />
<meta name="twitter:description" content="Classical forward genetics has been foundational to modern biology, and has been the paradigm for characterizing the role of genes in shaping phenotypes for decades. In recent years, reverse genetics has been used to identify the functions of genes, via the intentional introduction of variation and subsequent evaluation in physiological, molecular, and even population contexts. These approaches are complementary and whole genome analysis serves as a bridge between the two. We report in this article the whole genome sequencing of eighteen classical mutant strains of Neurospora crassa and the putative identification of the mutations associated with corresponding mutant phenotypes. Although some strains carry multiple unique nonsynonymous, nonsense, or frameshift mutations, the combined power of limiting the scope of the search based on genetic markers and of using a comparative analysis among the eighteen genomes provides strong support for the association between mutation and phenotype. For ten of the mutants, the mutant phenotype is recapitulated in classical or gene deletion mutants in Neurospora or other filamentous fungi. From thirteen to 137 nonsense mutations are present in each strain and indel sizes are shown to be highly skewed in gene coding sequence. Significant additional genetic variation was found in the eighteen mutant strains, and this variability defines multiple alleles of many genes. These alleles may be useful in further genetic and molecular analysis of known and yet-to-be-discovered functions and they invite new interpretations of molecular and genetic interactions in classical mutant strains." />
<meta name="og-title" property="og:title" content="Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in Neurospora crassa" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/4/303" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Classical forward genetics has been foundational to modern biology, and has been the paradigm for characterizing the role of genes in shaping phenotypes for decades. In recent years, reverse genetics has been used to identify the functions of genes, via the intentional introduction of variation and subsequent evaluation in physiological, molecular, and even population contexts. These approaches are complementary and whole genome analysis serves as a bridge between the two. We report in this article the whole genome sequencing of eighteen classical mutant strains of Neurospora crassa and the putative identification of the mutations associated with corresponding mutant phenotypes. Although some strains carry multiple unique nonsynonymous, nonsense, or frameshift mutations, the combined power of limiting the scope of the search based on genetic markers and of using a comparative analysis among the eighteen genomes provides strong support for the association between mutation and phenotype. For ten of the mutants, the mutant phenotype is recapitulated in classical or gene deletion mutants in Neurospora or other filamentous fungi. From thirteen to 137 nonsense mutations are present in each strain and indel sizes are shown to be highly skewed in gene coding sequence. Significant additional genetic variation was found in the eighteen mutant strains, and this variability defines multiple alleles of many genes. These alleles may be useful in further genetic and molecular analysis of known and yet-to-be-discovered functions and they invite new interpretations of molecular and genetic interactions in classical mutant strains." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/4.cover.gif" />
    <title>Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in Neurospora crassa | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"iPkwNFDomN8EksS98QgDO67-N2lYOKGk2LsTQiIG0Jc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459180","apath":"\/ggg\/1\/4\/303.atom","pisa":"ggg;1\/4\/303","ac":{"\/ggg\/1\/4\/303.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/4\/303.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/4\/303"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/4\/303"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/4\/303":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZRtdCrpMhSxz\\\\\\\/nD2889RCQ8Rpo1GwZJ6MDr2143V4pql3rfcJj0RWsHJ1TIvLT\\\\\\\/IyLsF8EWk06xn8m9TBLVCdhtji4Gc3TsOSJWAuiWrh1d1RQ4WVCGceEhdkbrUau4iRCzL9gR5lvSUUTPUWwA84lTAprew6njY\\\\\\\/4r30gGRr+fxKCRzehPlCHLAMTMZW0EACZc4PUHyM7vVq\\\\\\\/ABkDNCoUiUZZ+T2nEJi29D3S54Cepp06v22Csai1g8Wbra1KCWyPX\\\\\\\/6RMqptY3M2m51MKzHgibSXIFIE\\\\\\\/rQn5vka2A+I8b\\\\\\\/VFdm9Hej0HyKQRr23avjG0mgfTSCI6akBGiX5vh51\\\\\\\/DQasmEcwLluQTVxMJ7hyeC8pnJjYp16Pa2gpOcJz3GRctwKl31yTckodX0BFj9YS6nvDON7yyYuasWfiRhiJ6ixWe2\\\\\\\/Z4760DphyfgOF8vvNvdNE+Bo4CDNWfuLXyt\\\\\\\/eWDyFsbtFzZ7E3OutTBw65IRCN\\\\\\\/Ju61o4xmlwcjdrYBRA\\\\\\\/m9Fz\\\\\\\/TIw3gHarjLcZNCVxJLCmOcqfVEyE\\\\\\\/A31TNoOk7iCp2p85QSlN\\\\\\\/TYOo2mhV+3PnMkInmU0spy7MBoazrrazxTz9UgXgdLnrMTXoGyIeo47LRSnmColkWUYwbk3YneciN4v4SWFfrcjzezIQGdb7\\\\\\\/R8eREhInSXx4xt4MPUpDOTW7HM2gJvWTglJZ3pzDYan2Qn6bxIWwf2ozP9\\\\\\\/XmgHFk2LZacU8htb8ycVdnBYIU\\\\\\\/Q52nO5ZNoBgYkxuJpary6Lno7EP8iyz8akylIwLgpv2z1aGHcw3TW65zblpxm9yB2i6zQdM6uQTvzHP5fTJ8m9OSUA5x6SvfLq0\\\\\\\/smFO8mkje+MJPUDz1pd6gEFrJ4FIlOqBM4LUz+gP1VjEpQJTv8y8+70sxH2eU43GQYgjCZG\\\\\\\/\\\\\\\/xmQzIC29+R1BhSxLPaPTld5H1ttlma\\\\\\\/zXVsChszwHd2wUJCEk+386Z4RzDeeqwd9AxIhUZpg\\\\\\\/jS8NgyZT4EQgr\\\\\\\/F5AIMa8qJchKsGQUueQi9hZJyNo8cqT\\\\\\\/ZB5CMWNLrXEb8JJxkAcYc2sct3fzy7Pd8KslNQBR+g85C976z4BgLkG2u0NSpz4VROjYTq5Uo1TG5VqHvSdg1t8JqoYt6N2ZT6maWp4cfCziKauXc+F986fFdjX1Fw7djTxRfjZbkijXkb0OUJL5I0qoke7BBaP0gc1q9Rt5YORehmvyth1dqV84IatW8C\\\\\\\/Gwt1f0hNHkX6yhhclJ+NrqEgi2Sb2IsydOjJn7tPt99\\\u0022,\\\u0022iv\\\u0022:\\\u0022Fyc+z8nVTA\\\\\\\/XnRvpEwResw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220d39eb4bbdb7632efa6c82421f91a30b3b1c8f296eec56d42ea86de4a8f87523\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228bHfZqgx1+ZfIyq\\\\\\\/EI2SzuyiWzjqf3mSnY3+wXxZLv0sj2ojwEvnF5GexSuHQ\\\\\\\/Q8hdcxIh5\\\\\\\/JOlO6tsL0mUTYbvv7oBJNAuOEZjzLNTPtEuX8p2Gry552oO+ga6MpmjBQ02JmvbmP9lXcM44deYAsk+EN41CA5KLod1E5sru5\\\\\\\/hziMsTUhtsVXJUvyVOahwC3qyZ9CJOxHFyJaFjMZM3lLhMYlV5JahhLWMm6NQ6jSpcExYoFV1cw0v0N0RZLM+F01YEnot5sLNe0FKqMbwySfaqEaYbhK4WSIDKfJBBov1r6F4Ecsp5J1yPY3op7csjpAxE9NdudYVanSZW5OwsX9RXrmhJdD6zZYQl7bsVNfW8+0Aa72WWbqvlV3pWLnxID88dDtoHcXiOOzOOjfumQTKRgI5m\\\\\\\/TYm477VWbUeFBh9J3J\\\\\\\/tsh9\\\\\\\/iGxmExwyGzP1ClEwg15twsDrnV0rgaZDIU880cAUenry0qPg+K5zouLZ49Ru2b7uGjD8b82XjmzqEoTMwcmBL6sFvXSMxm68ncn4MG42McP1zW9GaUKKDgB2xHVG6KUhX1T8\\\\\\\/lcPDAjkkMquXb\\\\\\\/T\\\\\\\/euowxpe6cX88pcZjpXWrg1IQLJ5x7OgWShWE9zrrznqFXKiCkwAwOBcxR54fnCmiufkDimvQynpT8hyENQ55e+19xOizZwgLXx8dnV3OCEeLo3wLRMGSdS1FSrAzHdc6+cIYZuEFZ0gyxqZEvIvs0PgXVEU7ecLaV95yp0ShkIX\\\\\\\/qmxz46+o2G6o5Jqb6oD22jjJ6YgNyKqdnJlLjqQjSxvit+\\\\\\\/feHYNRT4oWiwOz4XAdgylLepSssQQ2vTq4aetJh2deWqkTCfZGvgR9MtQN0tih99C3DYQ0RwCQO6OvllPy\\\\\\\/ngvS7mbNFnqZ4+ydtFzNolp0hmy7Oxf+2gu736rgrHI\\\\\\\/zjyWLS1pzYyWBr2yU1z2xEThiOZXtJQZFcCKyBNc5gka\\\\\\\/CnOl5XbsEZSjtsI+WDmgyfoWheUCidD7Ubkn9J31Va3EMj2Mpb\\\\\\\/48z5b4FX501D1SYhwfuiFrEu7Zzb+jV+kqzWs30UQGA427+vLWX3eU2TzJhlTs+GfsMtcN2YSHjYdRylqi9QYt\\\\\\\/EdEBz06RP6EScQzYQBbSG8xHJbQ2bXuP13Kh4gXI998qOmUSt3r4bI8yP5fKROFzqalmBMvfPtuGWaJYE8dx0knEBUDmzACxeR3KqnY7LmKwlXQrQTk3D127Qq1zGVE8J6WR4M2FOksVVVOd0gNbHoVmBhjWNM\\\\\\\/eYpfXaz8T79p3C11poVO9++bVzzE56Dilj9rQAXuaIGAUfLvf\\\\\\\/OBm8mnwszWBUDv12ktBdqyuB6e1sSG41p5+q+hPj6G8TqNO+oHRnXtH0Vld1e8DmQxeHvu5wKF9fg219adpx7ikqtgWeh79bV3jWArAcQCE+oG4lbK30fI0BAWUebpI4lQDCa2YGhVEs6nTHIqWzRGbE62Bd\\\\\\\/HI641tQem8KT1lKqEBGxFHSLA6qjqcTohdhX0tLlSzT8WOzh882oSDJ\\\\\\\/mvdoV9\\\\\\\/WwlEvqNA6RUoHKLRuGDzaKpjKPeGFy48ZazeIQ8EpzJ6LnvgKLJDJFP2YZlLrqWRvK\\\\\\\/NZaFFQ9IEh0XvT5pOz1gUB8xYlStY4db3r1MllAL9bbWwx5uBEDfbLo9JIJOB9fGRqQRDswwx66ZVImAPrFVEBdZ8sEeHA3uY\\\\\\\/8FXvJiiF+ypUYPm1JdZzM2FY9oQIgtNDhr9ptz2IsGz+OWmQwDEpfpDwe1g5qlPQyUrWnZVfxTZWoBoLcuFuzB+vao\\\\\\\/Cxn83amdpedfu8jQgAqhXJgaaXuHjqdnuscAi2y\\\\\\\/Dei8MJ8v8lnZwREb9kYxw9v2wqo0pv64Cw+It5WUjc0HZZpqxgu1\\\\\\\/B6sTxvwWvSvDSlbB\\\\\\\/wuX93EzgT0nKrvTrvGSTfNi23DcndS5Xw1yBIgilxB+fo+93g\\\\\\\/k2hr2VwQh5JTWxONbZ3V7bSJa3GqZ05DvvkNiH+zu9XQY8+A2baB6yLqppOLHNhAscNX6RTfLS3attaZ8Qx9MZXYgcjKLkiBw43g3zf9FDr9F4lhTt1\\\\\\\/hX9S5q+CcLOnpNWPxcXjuoi0xTLYk4lZD8FPLoeUT11inZwr0oAaK+TW9hpz6sVDOUepRWUSpWJq3iKgtr27s+l3wjv5WXyEQYoNjpe0+kho0xQeFF8C+K1KZEAdewlETGTskBs\\\\\\\/BeP7Kf\\\\\\\/jxOJIvKGpvEI2HFiTWHcCI+s0w2Hp8hmart4lulmKNvUpbfcCrkvaicc0yo4KjNN4sLoWvJvoOlDL5mi5Sxv+FZ0EvX+c4tG1S9HmZbC+s\\\\\\\/TCeNIB73kXLz3Bp6lBouk2dF4x1GayE\\\\\\\/pjbbMpBXhnn4CPaLyBOg3NUaJJA6k\\\\\\\/IOYAR4HcrAWlhqVeAIif7MTCov5gPFt53Iw3bzGkA2v+ib35n40Uc6fxcOtCse8jErdFN2DX+f+eo02fJG3c7cadmptXLEWQEr86WrgmOAoVDAU68ShsK5sM8DJzGFqcEHaSV2yxdmKQLXkdVQuqzQcUKtGnB+HuILJIOFeybdVEyZ5ZBP89WhPXmETKEw96sfMHF9LflpZKWddmSXCHpmvCATJaq5LzhmcS8j46ydkujRhLBN8\\\\\\\/Hg3OjsS5A0LePLq+Ur7Qg3KdbiVz5oyGS2JQg1IjE7UV82lCeb0IZCWrwqLtPFv3TZLKJDcKgzIPj29MyknpdnhRvowOXYaGnUpqPcPFFLvXREps\\\\\\\/6htPTbu8IuHKe9\\\\\\\/YUK2YJ6gcFhkr+vrQpthErhAQqoQGtkCvAZZntDZXnC1KBWwJufCU4eRZiSH6AYqf3\\\\\\\/6NRUc25YgSpmAOwL2494U++v\\\\\\\/hruDKw9UVOICVWCI6ZjX3QS7F8Qp6sRll2xO9pmG1OSsphYxXhU73yk\\\\\\\/QnX0nqwDSAyxElMDVroyM+FtpctVIDBXc4wRDa61VBhu+6NQiBgS3KuEs8JH\\\\\\\/3I4KO9aHRnklGA4zzMMDFjvLtnaJqyBBBpE5VxQqL1UWTtjHOe40cGk40VNl0uVJ\\\\\\\/xoNN1xX0MgIsvs4n\\\\\\\/w0uUh\\\\\\\/qApLF5\\\\\\\/0KKJyIffr5jHuOZVYs7SOFZDUBm01aBLWonV3vzq3Uthp1ufM5Cr\\\\\\\/ixIjBWD8zEygpt4h+mVg37uHinGHK+qM0XsVvTUAok0QO1x+N44PMOIr4FOueNrL9ZPaEfFqSXfaQ9+4u1G7MmiXdoWpCvnGhPh8WCLCjPQiNffH7IP7xlPgdsWvDuQceO2dOzioWphAthidVqvtVXjre90BiAFfm2n6TGGLdLduC3QqX74vfXofr0TkniFcAphVLW19hL7owLZl8P8NPxfLom+TBW0mrILNl\\\\\\\/mtOtnPAl11+JQHfxeuM2uDBhlNyQXP2FDzYJPPWQSlBGziN8I4eWy6TDmo3t3vvuDIVdvP8WcJO5F1UCw4bq++z37D1U+kw23Gj4FnzzqDK8eSjGgACVNO1c5rStfW9Sm2nFOcAgrSrWD54xMzJmXGePWuEgZs\\\\\\\/FAHLxJGyvsVfrDYh+ydnWpeMgVDHYPF27Rnf1wh5GOuLs7jBmWe9rqFdcU9i3PDsooDvvlscgtUV+sh3NzSDpsVV\\\\\\\/woAAYSBV7XWSz\\\\\\\/Flezcd0sB9Be2fruSW8MeGOS3I+htXw1eNs2VciXwmwNb0qwq6piLywchbPf6cpqvntRRHl5zMC76jLqJ5t\\\\\\\/4cuoXNQlyJfTLcJqNYPUMxOFz8hfPlwmWHGXbkFi83WKvxuiB1p8YbORBKtj7GkbgFROFJzYAJqHNm+jDccSYXSpxf4sOe7Fi+VhDiXwtY6mY455qOWWXqZzJaqxcmxSoracHcLeHf5hEh+59ynvVvGlVkL5tpUaya5SxsAS1DoGFpbvTfqL1ftRT28XA2ha7aAPdEP7B+3BX9Te0zOpLJWBRAJPiIjiBo4rY1QqZOWawM9S1lLIjVK97XYIGXwH918MjLa6npR3WzG95zw+yP1NUd8bk5Mx+mfHAUvpHjv7xwkyBleL5e3gfolpBMhbuV0uV75P01ShStZ\\\\\\\/lNjf8929SN2VNZK4w7kU0BHvvlPOhAFGJXhph2aIJ2LL93TpSAWuacWUUTye735JQam+j91L\\\\\\\/Kt3+AT0mSJ\\\\\\\/fefIaJr0Ah6z\\\\\\\/Ab6aEwFfaPOfX6KXCb33i5TmX7HmELe\\\\\\\/aIg1BSewlVgYPIi8t3B2BwgjI11seZBCHsNgewSddoMACCuVUgkjbxdMZVkgXDbTVn0xykcRZ1i\\\\\\\/t9P0aqdU6ibDvof\\\\\\\/jO8TD5W3Ah6Kx1fln5HEGAtI0Hm\\\\\\\/z6mOAKkiacJCVU2xsEwx1\\\\\\\/a1BEAP+EA6Yy5OWbzbX+vrHyQYZ7rtMVyKd3CvuEEnOormqmH4Wm98CrVGD84p+glI9JRyyEhb98BYpDa324\\\\\\\/MHK+scVMJM1W5s5MZ6z190gmC5kTaA06Z6qZlZG\\\\\\\/Dm1btpvwrLwRA4bJBih4+y6PWyFMHDnC5moXJYyjUuvmrhs5bbnEfx0QgJtuXi94+YYFikxCShfyPilyAA3KCf5JcEJfCZCe5ABUec3KkvO0MBJmecR46NPk4lWOMCx5Is0nDr5GDYMdGtOYOFk1f5tyM1G2caUSftOLkc66txG3VjJ2NzhjqHbgFGxI2ELe6vJ6w\\\\\\\/y96Gno3PGEct8U240EJP641EDdGSFDeDTwiLqiXw07npZLo7ZvWmrlzccVl6ZMcwi9cj0oQtlPUhFrqQmhAZP2PsMQUa74Recvc5Sw8WJYOY1jVJz0tM6ebtRk8OEPjFOQ6PPWOVhIjdtvC\\\\\\\/MG1XpEtrbHHSU3zF6dTnjSxlmmWpCiTIG481K9zPsFLmR33lAqYzwe\\\\\\\/WWOd8evp9C52t+T0WHXwPnRuYV310xoxMrDDO2cf9ONRz6T6jNxTuDVvDMzVHlYIwPTnG\\\\\\\/qL9TW4Ka3fLuB4Bfsg4P46U0QPYG7532j42\\\\\\\/IEef9eAsTtiZ0UXwxOpWASsC0gNk++Oeux1z7iJQr5K4AFH1gkE\\\\\\\/pQloin3r5d4ADOeBeJGunxCXTQ50leYTQoDhMdDMsLYv8ErpQozXe6SeFxz+3\\\\\\\/agy9gusTq9GowgdRIbi0XkGNPlONSF3Xx1OBj+Glt30L5t1oKyuKTy6KXVWz5ZIrVYA\\\\\\\/zcbvfv\\\\\\\/lmoABQkr35PrSTN7Gw\\\\\\\/W4n65NYqLNWmg9Yju4pqMu\\\\\\\/a0sPbZgcw7M8XpeJdJfvcBGOAdMZeKW0cVSO5HX2tSaTmcDkcRtciRoOr3yrjI5Bp0f7ZwVK8qOp\\\\\\\/N0OWmWpf0VYHbZ\\\\\\\/MeE1mjsMZHjF4TOiLClcBoa3E4f70iGi7oLsq\\\\\\\/mV3qvP+LgNPwwnXIsn4LvIOqHTVgc6Un1v\\\\\\\/2x7snTcWfQ3JHtQA9cVv6QjREEM6YjGgPESwPMWq227\\\\\\\/ELXdaZWclwqSvePJn9z08yQueOPEwCoeTUOi9nzbWvCZyiDS7DKtXCfex2iE\\\\\\\/1O2GfIy1JLgBWag7\\\\\\\/TOjmshGz4vAU0lARLRARUS9Z6PPjtQ2wLfPtZ834pJBDqhrSrh2EGabjFBQEIdFURf6vpb7z565MXfxT8I+JfnApqUEF\\\\\\\/vjSoXaEt63a6bRJAhShCiJfKmQHKptqP+Llc2Yqyfza+tUlfhTbOzpokq21tU7sGUCe6BRhCI1se3fkkk36S1WD2b3nNiYkHsZP0zFkTPFqXnYIA4L8QRJ4hMCaX1A8tUaA3VMmxI94C0P8qDoXvuWcNs+RH+TeEKBpZcVVZKwVScMNS\\\\\\\/\\\\\\\/UtTdU6CzaU+FsswHJB\\\\\\\/azJ6FjlTcfm0n1scAp1wo2IVlfBbqyHsQHvXisYxsGFPvh3Dd8C4NbTusmeBQ7qNW7wSE0vSETTd2gZhRaQtzvQ+Rx9tFh1O\\\\\\\/C9IgcZXd8A4PUVV+I66clD27OCqyKxm2hLPE\\\\\\\/PViZkHXoLN8rW3znHQbPh3wJK72m1DfN\\\\\\\/R84wydXM0xtYnLan7\\\\\\\/PryKAl24pH8MGpQ6qGru5KrcfYMWF5lqnTp5pbVVq0zZBHcjCWWP49c7pC\\\\\\\/IIL30MiK5BowMRiC9RWRP1rCcoorH2IeSYM2qVovKQACWfwLKLMwwspbl70C\\\\\\\/TcnjOmwqhUkb3yHiwXwVbOKVIR4m14ovemAGf0\\\\\\\/ACJCHVReGy+0Y1R0QEkZb\\\\\\\/nCmZN1UnDXm4uYFn+WFrBFUQW32k9boGvs8002bpm+TfEcLSHpp0\\\\\\\/0PlKDTfqWzNfdpdx5\\\\\\\/5KOu6HgjpyvMnKo2QgiNNXhYoQOwsSK\\\\\\\/1JNVneFVdxtdVkUa2c+Kiq57ULzdjaFMLcB72pguX9jrF\\\\\\\/QfaBhrL84jSn+JtlfKhA49bOHToTwFrgEP496x8wi5BFmfptPoP\\\\\\\/jzOqpC\\\\\\\/vf2iRaBsU3jKlKBL4g1eIVIRsA9Ti2BrnUOK8I6rugao7CgIir5p5BQUBkte+Cy5YkF1nsKLFxClfcyGwB4tHX5Ycs+uLZNcELh07nudys+0xlb28cH83c97VXMxAoWMxQnZ5D1DTuJkSHMOxGek\\\\\\\/RMO9uvi9rAx3dn9njvHUB8JpJgS4BH2Oa7Lxp+rLlS9KLBmg4WpFTZNBzVkw\\\\\\\/NOM\\\\\\\/PSllClL8O5Yqk4D9g+TjYw+50skTW2idXaiRTSiV6FcxxKzuV6NFZcQzVpSYqO9W4TP0iTEzV3upHZkQaZfTwcJZTbLNgZQhQl7NJeWAI2ZGCnNATwi+zwmDm8jKs3Azu8xVOlNJg+oL9J1FtnB6IfWWGK7W2ooM1ntmNPbjObiog9PBlW4+AOMsmG1NJOUf1FBPelOZkVP9xxFbgg2T4ohQ+Ky5BfsBGQBi8bSptvcBInKgja3J9yv1kmM0vnkDuhdOv1thYoHJAt71LL87cfN9s0\\\\\\\/oSw9omQSCbndYW06xMq\\\\\\\/Hwqx9Jd7+1JrIFN8z9s4RUIIewJpcCxqGeGf6Jfwqk9RVi\\\\\\\/bPcNh1Os16YFPY8ufguHtY382hWimlImIy9BFHpXzymwOWX4loF2+uUbbBML90r0VgFhkiexVUyv0n\\\\\\\/zxRPpZqX8pv3yrP3g7yKV\\\\\\\/4KPMcjZ5i72vf\\\\\\\/WokvXGuCsG5SMkZrCNlmXheAvDHmyisq4wBl0SkUblaLwuRmbu17dvWldWw9zvIsxnJiLkCG9aIipCnexh3A4dDY6k4p3eoFv6XZcjiBI2nYUWPUaiNyxJBcqd+BrYEYBftBym+UtaRPhFMCBOlenSxh4YY0UAFbWp9bhmFWYhyqQ2ra0Ps4GVhsrH\\\\\\\/lxvvhSD\\\\\\\/k9iFOXESQVTN1UG+ybNIz\\\\\\\/vOhPXOIwuaSS+HFEb1Wy8SOh\\\\\\\/2Fk0Zxq7l3Cw7DsEyW3voP1\\\\\\\/4Wa8rwuV1IfiY2ihu8BY42BGbsUBzfJe4e2GhzQi3XdsNbRMV4XRSqB7We\\\\\\\/8BKm4bZUziX6WrKhd5+5rgKocvyBI41NTcQK2rAYQ7E0XHB55gAg+bdq3aE2vUiG06plzaof9SY29JPIJKx3Mu5hBJP7qag4gXheAJXClSV1jcr14VdKrTluK4cywCcJld2Of0b5gu8x\\\\\\\/VqxA1ZBD1c4X9T7AwQ0uSXlxduo8Wyx0XY9xIZSeAJ5BY7nez2D1WZu+0QrPc0dlD7XkcUVOI8b28Wvk9jnyfPGgtZt25x\\\\\\\/WrgIo6h9O3Quy8XbIb2TQaooG3rtlNwfxy5vEdywBgwHO6shUByzPEA47\\\\\\\/tmv\\\\\\\/BUXQNjnah0M41GI3QYzCWxAx4FDkQaDEqwUGHQ+9lJAGUmty0Jg6pnKoG2Eumsa7\\\\\\\/mTxFhSCAkQ\\\\\\\/MwGtHa1fGo8p\\\\\\\/iKjyDfFfXlXc\\\\\\\/YzsnmCqCZlmVxvLHl6\\\\\\\/hACUerIOL2v5kESkou4ymg5PmHRbbGJeWO\\\\\\\/tDtFiBQRYCxIe5Wn4nyCMzBQUPXU83WVhxoSxuuy2Ui+9aN6Zrr3VeXMs\\\\\\\/PPshOAhRk\\\\\\\/m1jioyYj\\\\\\\/uqdDksJhAT2zKuEfNclWSvHJxFR3beYNDp5PIXI55jXAAyXqcm92ExiLw3iVCF10sGLEYc80olu8kqBKAk6JSU2ieCQ57+CKxuiJpwUcXUD1jmDApKgLQ+NEU+o7uSHlYG7NKBYgrNQqMqUzyCc1ThqeeG0Am831TJ0f\\\\\\\/sMg3xmkaZIPpzUqwp++F1fyi\\\\\\\/DNmCLpoIyLDPtd24OGYT2lfsnjV2hha3sY5EgBHieBsz8Gfs8dG2JnUD4JruzDpO2ibvsJz+Cqt0ADZWrTlrMqgbh+8jrnVvT+x3FEIYM09WuJNBC5gePogA4gW3G88XNUrnK1a\\\\\\\/UudT\\\\\\\/2GwwI5w2fnMHuDEcGhwiX2mnO21mOxD9ouHoGdbkTR4FbxSh0HfMBnds9KWrEZ5zSDuiuLpe5WqzHJeMd5Cno8Aped6McPnv+4IOhLC6H5\\\\\\\/ORFhbPha289R01C7Z4BlaicaXZVUYlnmrAaRiZjeQ40dYzBMtOA6dd\\\\\\\/6nZHhRbr0\\\\\\\/BRLbgnUVOAKpHLtGtqpxFY3ZE1CdtpYvfVFbb\\\\\\\/v8cDUozDw00FECPM7OXXDorM3t5TSXJ8RFL2ctTS\\\\\\\/RKQ4EVsfjBdoXHmvN9Jup1fins+gV0MRCjPgokPYewp+gek\\\\\\\/T4eK4N0I\\\\\\\/dmTUmW6TpBpFcdHcgFxFtmLh99KcIZV3jJ51fKgzJvrbuviwZfKn81yUzPsXeg263SInRoxJTrlrWBBKA77+aWPrxTj\\\\\\\/T8Thgkv0YNZipRypFsyhlhVjkU6BC3pUyExWS+O7IKc51SZ0E0NJhPZwevHt4RlDYkAjHUV1Dm+m6JqSDE\\\\\\\/eDjKBPzYMIV0L5BtVUAWFzk4N8PKxRuwV8ixjx1d\\\\\\\/iDvdze2eNy5azFBccaZCMTeLvveEPM2cKXFDOxl\\\\\\\/xuFrB0LqUHv2RIhHsseLwpAZj2t6bKDFt9L4jfDClOB+hM8fJpUUrgLUT9W34uVtNouAYaCPieu9PkTmx7HNp6FUQ2hUb7r+rdIg7q\\\\\\\/ipBwqtI45e01ewwzh+lmtuk4md4bVBhJDmZXdJWqcBrbdC5Ra+ImxzTkJ+axck2LesoCxHfLxAsWEQRTmD6MWwjFH9mL2b8Ei+myZB\\\\\\\/Ij2w1w1pJQJ8Zd0Zu8SFcKSKVeXUOhJ5Wk1H76ldix23yBMYYzIWOOyK5DFA9c8i2cdGBfJxXBg+539ytxUhAxlJMe0R4PowhccY39eCzuDXbxl9VQ1VyAdzXCcL4re6WvGtj7tuWwiEoSyJ60Fd4w08BdMV65G9echAZbsKkqx663hVDA6LwNt8lke\\\\\\\/cZNqCW2ag2AJpH7Hhya8xX8TqHUK+2Iobs6Hi3iufjKF6Drcz9f6uHGeQ4f99FiH\\\\\\\/zWarZGPHcWHJpbExJzi+GjUdvnf+2v6jKYUgr0FldGzPEPspvAeuVj2S7UYk48XZ4gXu3e\\\\\\\/qpZuv9EDs\\\\\\\/ph1rBFd9Hcnjr33M6NG5csMUoxjb3TmnQwCFXDBSRYIyPQw\\\\\\\/caCwpTZYoMr32GLSwf7uwsegNDcrC+XJch7jFtRCQbbuAA9YQrMwza8YllP3F86LVR6hlCf+1xx4MW2Z+DJWoyu8PGwK7dJixkkUD1mDD3uC2vaZ6L7ZHOmfomDJhh7j9GMeV1t6YVpZpOJXc\\\\\\\/9Jj0ihqdFNPKPdgBT4+N\\\\\\\/8mdToR9FDBCJ4QUxuTvAfdli3\\\\\\\/E++gQly2zQDKpl7UuodEj7qI7kqQqBoeDmuE6ds6GT0Wa45y\\\\\\\/WAIqom\\\\\\\/ETj2YdGSeQOP3WhxifwRwKJ0J5TSpy6h2ggI77L57ZgcMOFKZSrfgg3hL\\\\\\\/s\\\\\\\/MMXJtfYxU+Ily0aDEnqAwXqL0namWBy2faWLkSbsEpvoeKCn8nvh1oZBwPqDP0fze99LriEc9l+djLdxaj3UfN5l9a8ODZyXTrZNggcNrm3rGsBSwz8jq1shijLHzsAQWryZFlXh68EwVp3hgpCzxPZ4klILOTK8he8F9c7kPJ+fz5fAOW1jOrgBHUDCRuQxWUksCQOxWGZ8+QQU+w4ekWZzbbzkoxI25HQRV9Cc8Cbeib9JWBHn8WMIoKssqeOCL2ZeUlhFuTKoq0Stb4dPlxeE9RHO8\\\\\\\/gk0yexXLAVXEpa5c4xZ\\\\\\\/HH9Ewz2IMI4NWI3O25sK4rnXYRQPLOUGRDCYqVKoz4+XsPbmsPMfUvCTyPWX05CQVFiqY+HRZmGf5BdHoPJcDO4VVS8mSs3f5TdaAFX3Kpb\\\\\\\/8XRmY\\\\\\\/QcE8s+tWL6dLcTFfL20weUFVnA9OE4ic+kIPgTESYyprOy8l1AO7u4LHAlAP+EGcHrRxMC+Pt2ouveh0mKd1rF\\\\\\\/VcaSzS3Fv3dvqvW0R4orC8Nau6VAHuYV\\\\\\\/0A7VVmNQzeeibwgf8JUmSTBcGtBzwmFnlCgLA4XgTL\\\\\\\/MPTCqk8X0KNIpaB5msO\\\\\\\/MGigcMDHU6NHNR1tChhGuPT2pzzwO7ALLgcGfg2ZT\\\\\\\/0OQ4\\\\\\\/O0+SWDZLpxU1I7cv+lWkOn6CzHqbr498YWPscjbXkANaFTBkDFULePpj285\\\\\\\/uaOONXxsi2S\\\\\\\/GfqUMXeAYvrrQmAkMEzOkGe3oM0WHBKdNiEbOrZfwRrfi06NeZ6ox1tbRDhhFL214MG5711icypK2nxusdVA9OfMuclLziRFSTvXwiOEOowzA698c07M1+XCBEjsSQ+0ciAu9D+tJxJTDIAtRGMPgtYYIeb3dKtWf4BzKmnS9WbwAZ8+\\\\\\\/YMdqb9J\\\\\\\/GAVrxlG4smikc6mTuTIngPkCv7qn4i4WoN0UjYPKzcdATVHLAC\\\\\\\/2UxOkPiRaUMZg2kvV1x8NX7vyUwartfK9aqt3k5KAH1ih+gvb2XW3lYd4oX3Ws8hx9rTIp+wXD7ilA6ebGlQ0DpLM\\\\\\\/PU3jSc\\\\\\\/+78DzbKrQFYJ2vyvTWqcz2R143fg1LnA3UVn4ijleTphiJv0D4MNAcKJxgRUA50yMoG2sY2StMo0AClu2uwk3C0UY3sV51PjNJArLNpNmLhcGqzqGWhqEuvQesXC+gtnImdlYLsykxsgXfxHlEl5j9f6mHWObyGgC5zwljf1KsIPOb97x\\\\\\\/VzUdKO4Kco6exss+8tRpPEj5k9qvsgtL+DbDD+FsNQ2wZBxqSfuX22uFmWE1usiu+52z8e7bHtxM\\\\\\\/GLwJ1Nca3mQOKMMy7TzWHG1TPHBdF2O9b0FEZrQl149CFqIXHQNjK8T2SW9gT9Zwg5HywFKeArQFhqwV83gFDmGWW64v3guFTZjo82NinvUfdUTyPNLwb4A049WGWIFafoxenHmt5VO99\\\\\\\/iR9PqMCFT4kjiHbhUxrmH2kl6CNDMooY8B4V2I0XRK7GUCw76\\\\\\\/hc151N89\\\\\\\/6u+VFQxRJX26qvq2XDr36aXAQFmYEzYE1zsiZNT82+zB8UsIdq\\\\\\\/ZymzmTFaJg5F\\\\\\\/009QPLpaZI4SaeThxjgvFCL0L88n3L855UIjzV3hfggMlQ1mZs2pRd4GofQED9Lr0+wWyYdYfX5xf0Kqb+FWAlIrXXMtk8PHLX7cyUY\\\\\\\/Glsnx4rQpTEvrLK\\\\\\\/\\\\\\\/laE\\\\\\\/DIQHDmRT6ihVyOCyYCROdOuzrNYUJGFeeSMa8rBqFA45bjh+saKTJXeAIdCUHqklh4SWXJiUTp2nQQJy4sWVya7iXIk1qHQdfWditdhixZf8437C2LXVKhhvcIK6+a5yn4H74UM+O9VCcJKHSqakRefw7OfQbtV4J95EX7E0xDvZ+8nd+zIfMsBxkk9snPbe6icluaUuEB4EwyFg3V1oyBKWqc3Ed2Q4jHGsCZVNGkqmSIkSKK4CubTD3B\\\\\\\/FLLBCPrv\\\\\\\/SmpvXJOTPUAY+Q4sepjxnHSL4Wb11X61O\\\\\\\/P7jBmYResaxHixKmPYTdUMhXsli2V+jpvdvR0l0gKKjMrIrZjwfuzotxBVW3mWfPHvU9mJ3JzA2BoycC6Fe0w8hUrWywLX5G91c1ZxFBq5iOeijqJwR459Jc3OODbxZzWHBT6rPbE9EVktgFySJTMWXrr0a7T2rkq\\\\\\\/s6bREzoCKV9Kq9LXQ\\\\\\\/VjOhvcCJuzMazAcnbgXPSt1rwPw6uJ6F1Hni1VY5bwwzjYa4HSm\\\\\\\/GuX8PkpgrFa4g63EbsOt1XMxBpcI10d6DzTUwIpIf7auX4Zi02wSKHGuhH\\\\\\\/kVq5pIlYq5h918upUI4nx7akyPPk\\\\\\\/CHBLK5tKh1GuJ7OkSLKvEzmjIUyJQ7dq\\\\\\\/oye4i73GhJEqItoRq7yKEG+kd9SEnWgu+9xrOqY4csnlZR29p7bs\\\\\\\/E6um28LsUtWAWx6Q0JSGJKwTVguVx1iuVkEpEWi99nz5cORQ79V4FeNkmv0agkUj+\\\\\\\/PQQXsGCm+EcN6Wf6AnBzzpm7iPSzfPLtkZZfkZsTA9DGEFBnqMEKeF8G5VtUiWDcReOSiKaJIdoGWWjxtqOVYbVFdDrVkGCXP0pkeDCexC+6C+I\\\\\\\/OYP7TfXhH970zXBRHI\\\\\\\/f+NaY+UFpBw5lslmp2g8vVcM8HxnJYixfVLzOTj0ewnyALwpZeA8p6zIqzc8vReU\\\\\\\/x16+EWuKrpeKUpgcDErU2eSnTHwQNcwgKUZAlksB6T+zsK\\\\\\\/phoIoBu4ZG\\\\\\\/qnGZdQJYAeB24BQ14GBPfw8vTenVcAHelYsJU+e81SLaKrdWZiwndwdO\\\\\\\/y7F6Q3hschWIyvX\\\\\\\/Dodvj0CK+9YgpJ9KC9dPkfBOjHARIRGlSp2p7VtTDYK\\\\\\\/lhJSy7MiXyLQBL0g3EktoaBGhsImPMeZNQlZ1njjHkYmfNjIoY2rDYGGBhHqRncOG+\\\\\\\/DXBBCbXDbghWUnM43WaSYPAwRi\\\\\\\/XTB6c42QWEzb0AGvj4e2\\\\\\\/+TKMBLYgL1\\\\\\\/\\\\\\\/OBT13lEkIibitQSphUUMkiV7xUI96iI6DkwFJXmYXb\\\\\\\/P\\\\\\\/giUCZUoiCXlVfg1+zQSl8CsG78QYyu\\\\\\\/CVUzzDUSOWKgqTgBCqCyNoXzJmC4bjfRCLdXdIkawGohbeKm6akkjbh\\\\\\\/0yLUX5UdEd6YWFFlzilFya6ucirFdmQx7H0QDGGqdP+qJxKVfMmBI+KafzrTIvYDOfke1DGFDZI87RX9snF0Q2jGXJZyWV1QLBwTsr0aDs5srANKz6ERZAt80bbC1CYCuJUqmz970hYZd\\\\\\\/wXYBGqdNr40W66rM+fbEg3hjpUiDwaTJlPHUhCT2HhSBVy3z9nvrCLUm5R0O325S5VIAL056LSk2xuchwDtiDJcW1JZoAod\\\\\\\/wVIWGEbUKzzoqWajKnPfmjiOTHpID6df7wt8wJwdnS02l\\\\\\\/xYxClpmGnoaQLOxlLJGzyyPLPf6I\\\\\\\/6IibfqCEV94oAupZVOyessj8WuVIWIlYErhVnvcr3XVK7JYsqMIRst91GKQ0NYRMhUBkJ35lcFuLNHd2QTcwVPiHeTQ3whGFfPInoBzn\\\\\\\/c6YNPm6a4jA5TZL4LUZNZNzrjt+zzTPJXAOw2XyxZ5po7mjN8CzZ\\\\\\\/96Q8OwoA2APzWGGy9+YQDqyvJCNGteYP3GBp360zi6bfiz4k7IrmTX8CE8y7nh00icjzpaH\\\\\\\/apXdSnLfJzaaAS44cyDwTRRDh4OXVfcqIaJVU69rQSBsiv4J+Opckbt9pQw68AMEpricdv7pcvPtwH6R8w7U8w638Qxg0yekO4o+s9sKjWw9xWI5ACpVbrTfZBx3pkLkL+e1CdolEdSTwxWCl+i8PsucHQqk0lyCVctpbTYVX1Q7meRR71eHMs0w4Np2NT19YjdT+nNNv5rRqQLk1Biu\\\\\\\/vqXkDUq7CVBv1D2pxx1RTOfJFCTEl5t6rlYpQw7Ii8rxBcGP7TkUDlXMuOUB4UYLqv5HrsBIpdpaZM\\\\\\\/lltYw4WSQ75MnhfKKfQ7ltf+o+yY7MclIZF0FvzGGV\\\\\\\/4S3JEcNiuxvS+gmpyNZcmYZS8oDyey0L1f3Xu3XXCJtEeNEY7k+01b6zkYR6A1k4krT63nTR9RPrRgEYiuL6EJoX7ciKicLvDnYDcd77Xo1cpZz9dtlpIdsv87eFQyycDtn6XsrtFMdSPVDNc1M+Ur5Lo+6rqPDmeD5urVKdsF1dknrzDVyp7yAtrQJyHWju8Wr3VbAu7BmRf8cHT79jTSNTOIGX+RvrOXmhUXASKmME3dcE5P9W09KA4XGPTmbHdd34+y\\\\\\\/MzChrMWLd4Gxb6Y3WM9p5CL3BoeLQ5QgRL8YgY4JzLsrru0jE13ozTDXFJXMImFFkKLx0uxG+pDNGQf8HjsJeeuUq0yNhllsowGG2lr6OUk4BoDeaU84NbIv2ZuJfdK\\\\\\\/9c+XmhxqJ2XwK3VX4GiIowrdZ19v1RF4Kyn7UpsSRDRRrdwNfUoC0fwoCv\\\\\\\/cZz1vc8WjFBcZQpRFCElUMLOd5s08n6MzszHRNfgwrPBlq\\\\\\\/ehtyrMbkA2ti4m3rO3Y5pMHW4p+MN6Aeicha2ZSNg9BzvIVljVOhB3Bg3MhJhZRYOPpMIwxDJCxFV+EPr\\\\\\\/pExmUW0yopIjGnvgKGmX0TF0MCjogAhwL2ALYPJanaE9a2VfYBYiRORwnG4qvmN4tnX\\\\\\\/W9tjRusBLn9EXOt7URJXmgn9Dkq6oQ378NZhjS6L+nVPLeFzHuAwfi7cFmLb27qKUVVUL1kh6D5HtpWTQMz\\\\\\\/5NK0rHwBZPPIWtGtyOZs3BoSdDeGmDQMaKXZXOlMXZAoj3A1Z9kY3\\\\\\\/skJKIoBIsbuzYweKBQbAAaVzJuc7NmCsvMJgqalg2NxwdZ6yGxUOYL6isEcd4Re\\\\\\\/DeVC4pygbZzrRyC8K1vX0dmE2IGDG26iYy7BgeKUhCOq5YcDiReSunw\\\\\\\/HBYb3V5cXsS54EtfcrkU2J8i9BbERh1I7aVlQv8M4BIcUCQwxICcz3LK0T8pQboZjHwE\\\\\\\/93Y11WMS7532yC3spdT8PawgJW+vzqthZ1M1gu01G1AYT\\\\\\\/HKSpeGCPbq27cGFNasuf9YHX4tmt5qRCeF+JnE7wb8GG0UZyiJxGk6gnF5K10EQbjRYNPcTesy9okIHefdLClKI4kMNyDWt4CciVHhYA\\\\\\\/YNVQ59I7AgMRakond2QaLqLGJsYQKHuNxV4HQvkKTn6rGK3Xk\\\\\\\/zeI7raS7n05sstpH3KFFXbp+O6Tb8G6WFdLI21mtsroFE5ZVaC91NIUrFEKYqdef2H4I\\\\\\\/wKuKPbUfwYnYR4+zCPbE+Ud7WKLKrXdA0a8z5wIOWow2gBv1c04X75BXKvJYeBidbOqZxUlWd3Ork1uYtV8gcSq\\\\\\\/hZ9zcnJnuCA4a4hHmXkCzyDpkPp5iZB2cFPaFzL0x5Zfj\\\\\\\/vEgRePmMGfu1L96wSMHX140KBC82PowpOANVvY03FiZ+GD4oqJtWTPGCQsUYCJRG9+XXi3JuzaQmS383mxuSZURGp\\\\\\\/dajsYx7m3FJ9m1tI2JxMXgVNqQiNeoHsPFjiojUzo8ChfvwgNSBXwHGb804UYAyu6z6Bm\\\\\\\/BSRrreMtETP7\\\\\\\/8pGed0sXgu\\\\\\\/e1kTptMB5c0c7eA9WvyfNH+9QwPZlBM+jlBZAU0eTA9yr8XTYmrodc2sFbewvDiLgvcunAJCx5nx+1nX944vtx0MKo8K3End\\\\\\\/lmoRwmVO0pst3P868bciCUkpm3jGoIV+z43k0mx1LAF0PU2DSdmEadUlPNZbLUHnCEOaVb\\\\\\\/30Z+07nQfVQ7\\\\\\\/8kKPyEBjtHrsLU+hU0aIgE4AM7cAMuxpfKuxSwDyFuZroc\\\\\\\/IM0\\\\\\\/55\\\\\\\/ReMMA+Xp+H2rl5wUrkyPYDchN0XlMMOLxORff+j\\\\\\\/QilPh3gaa9m9j4QcBFai2pjKIJcRVxnBmH+HLeA6P2sCXQMs2VB5zDcdlU+hn3wXCaYL7EIlUsnUFNO90GniHbmj5tM+xUoj+S6+FxicdI\\\\\\\/F6af3bAawjXqv5mwRMjh4AGe+vlXCcfPASvhyUJ\\\\\\\/ImZTMh3L9FNqrFfrSF1wPHai7\\\\\\\/1DgRIP6DOS7FDY5F\\\\\\\/EbA\\\\\\\/VuwWYELMn6qR21G4zybtU3maVIFzKQUJTe7KYGuEqKJOCc2o9pn1axprmvkuacR2MUJFvWxcdbctGV7cfr09X0ouuTOCnqOHWWOMWlnWdUApDQMHR6RKmq+yfb8KwhifXWfy+wXuU2DwDyw3fr0p1xFU1N87kQleDgogksuaC7TOz9Utxs2ff44c2gFbdl5Nhol0T+iOgfrsH\\\\\\\/ABWH+N4Xt1i\\\\\\\/FCjey2G6TJhCCQq453JSsHK7AGcCHSxvLRYzZtfB5YwUeDdWCPqstVI+ASkeayl4RNYHSntCRVNNljYAXrzx1JzPO9sOUjr0D6eiL9tSQVNhyTScsRsroo4pTlLr1WktdZYJyd8VOx+5O7o4C0cxbdG59qHrxDu2ywq5KsfelyaxAIsivXkDR8uElcjVusuoYFYacSUPngK8WS0BYulOfQGhcYTFkQl9GYUK8xU+ZjXR7\\\\\\\/sOGexg04r22IG0nge\\\\\\\/w0hPJf5+9me87Y7PsOD+BzbamLzC7oY12LfuTJ5INgSQG6lIwZNrPzfsWRJb8XsrllfI3gP3BFyVYSFsB+wHRLeQJaIZS0GgMC5JreWPK2g45qlKYSt6ZBJTOFOWUClXAe7m8HOVohcWoSwN8H5jRRybUCi1h+ZxxIAT1IhPUBVY9GxGiXl1Y6YF\\\\\\\/FD\\\\\\\/E6yo+LV+0gyoHWiybHeyj0eMatvZ9+Mnq2sVyDd9f2seEMhdm8+H21QBhkjGhJe0pb+YUIgi48xLHz5w0yCc9UufCbzPa5WTIU27fBrRqhcFtLmep+L4skHWTQe7ngHwAVK8Qbn9i9cZlEPKq+vqGd8j350imctGi6lOW\\\\\\\/zGESA0RqpNPDkGQ2GMelxXJz+oHDLO4Lh1X5Vydwi7oeAzC8U0XNcnFsjRn0kqmYAo2eOJ7PfHR3eH7+M4wJEL6Vhk8vUgw6i4HCjIe1TX4D2Tb6U5m04e\\\\\\\/pHmmtr24wgsdefCkrPL7v7syWB3ZdXg4VoSRFdY3+oo3NlGmONfv8IccSKadRLWYYiDrojB+x0sF4shyTG4hObqE3buP+6ycRt\\\\\\\/C7AW7p9l6R17UWRQiNRVi3bYxu1Ju8Pw7kYXw8h\\\\\\\/w3IfMXpIiAPvogOBl7PbpAAo0dnVX96loDr4YFLa074Ey\\\\\\\/oMejiEIdw9LJZzzShEpOR3I54w+BbHpK9U2yhdTJe9UqxNPh7HqqB6ZWQ57hSKYCogKkEN\\\\\\\/F3B7rD6IqsS0446UcIMWbWnQ5bHFP8I5VtaXLOSF1CNy\\\\\\\/T5iIdoFdBd+4nG6A8esMdZ2jD9S0562pR6VvbCCzZOxK3wQ81qxZkCEOUn1VsTMEK9HVO1tSNnwDC46RAOePaN4Lt720Lyt0DTLKAaLl2RNrbyVSi302w6F4kF\\\\\\\/sX8p3R7s8HsVl1dJABHZ8stM+ZSBebBTm7x4i\\\\\\\/J4ezO4n\\\\\\\/83T\\\\\\\/kWCEd+jBjKV1C1\\\\\\\/ub2pURXV5Htf4PTNOjlXuekKU+HXwts9syNV4dqD3oNpcG+xcblbpCLNj8JYJIhi3LJe53bPPGJJVAKLiDjqsPJd4usTDVWMCkkdPjczCxFE1KmtPEOOLxrmdOhXgKCoj+HBwhBDGAH70THFXu6Ws2881yHsDRyZBpmHpP7tLleof8VgZisfL7jovMr\\\\\\\/G5GylN5w0bvF\\\\\\\/eiS+9wV+1nlXJTCVElhxoUwZ7JOcJjuc7LSmsj9t8QkWR+Vh1PL8v+JbTdcBvw+baYAwJTLgyHYVPSkD8krKN79itBkJZ4x8XPDIA9WGGzKgpzFLZuU1BE2tDbRPmxnI2LY8sZEBLDW2NQWa4tokvmIiO+nbwyAJEH01ZO\\\\\\\/uUcp3zAvb26AgV2jGU4RYGBKLGrQBZcsSF25gxUS+Sb+DXIAKu2rGqUTVdQmMy1oVWNTck0\\\\\\\/mU8m4ccUp68PcuBlMK+M+T7Q5yKppCe74RL6oPh8WMmYZ7rRb0PHiJlNZyqyUfQeJutTkiFvOOTAlC+KmCMJr4PY+fqTs+JSySnEdMXfr7YA6ZMMkuQ5vJORDie04lJ4Gs0ZcTOPxdTkLwlSZgtceVAORWkgJ1LfQVSt96nHlxhXTAQOsVGejNpUIg3Cn5g462Uz2iW1wyNhdPqlGP6\\\\\\\/VMCLPPz9sx6SAuX6aCnVLlhUBF8fgfvJe+vIt70CkB\\\\\\\/R8SZ2YtNO4dk3PkgaKVR2e9eUpAjIaMGB6UxIL\\\\\\\/uVgfcdvgtZHKiOYU+Zmv1H8Vw5Kno1n8scDguybAhl6thGOnoJqmnJL7AgI9u1s1iZejx3lO3+bLAkKgC5frOWkWiwHiAJqWxJi8wq6dyNcHmWCzNP3JJ9uCtbZikHYvkeaYn+5KHy1BSDNA1HBk7TJgAJ2rDE4kRKBz9m5A5GD0cnHLv+PI9p+hYsEmFrJFgB+orSzxH3Y0JVKe5sTJ5jBxKk0Q5YXDTXiZqGXsVj0Ad+qPXqel2kSctoqvDUPd1gNIqx5gMKnxNiHacRfCoAMaF877oVyrELeJEpeWDAbTqWt4Nx47rSrrh\\\\\\\/d0ZuiXRETYBGG\\\\\\\/VlXWfAUj\\\\\\\/kudpam\\\\\\\/y6+0Oa\\\\\\\/fVJXyLCWAJrMprK5Lscj3AGroiCAuq7GRAez\\\\\\\/TFz1SfvamKsAbZGvp5CAWeTB0m\\\\\\\/sVOp\\\\\\\/jmhjWWm8iC4lZzqh+cJUikCcOawaFq7QLqPuvyi4vpzZg45Ae3B7WiaEnNmMqO75C5b7+dE7KSYMO4YK3kxd50b4pGfnACleu6DLJgAz3tNyKx4efHXABzxAiqcIVHr3xhokKuo8YtOJhlcb7ZLTYTZh2Af3N6tMK5C0KACVaQMK93Nar03ByHMaAeTfNEqomMcymu0y9xhN+8nyAgmbTcCpWUn0PF3g0ZfKZHuqxQEX2lm9PNxRmsaNU8T45N11EW2tEIqtd3eIdb\\\\\\\/Jrd5MJGEtX6v0opcK+fFr8xwOxbbRYL\\\\\\\/TTcShAHM1+EbX3fUjtytKGlIA3NSd5fJh7X4h6v+Bf7HXURmdZ6EddbLKvm9SK5EPg1rn2hfUCIzbXppRLN1YdKsZ1h+bqivF2nxdeYeb1hRyD5rgtoUsjEKblYoKKDzDQcU8Pn1VH3o+Od\\\\\\\/SrCFPrmn+gPwEP9TAMnUbOfT8yP1RINsY+oYZH04Gp2H0msh2wELH5\\\\\\\/a50h3HYConeCnOfwLOq3+703jO5QbWPREFEtgqkTN22FCS9mkzkUMJSseOS3GDGGKqwg5pdukV3B3KEPT9QFjGdpwn5SyffTc\\\\\\\/vvG57xPVyz3tqDC0kmgCMis78NLUh9ZEzudoIOpIQtlS0iLSVoY+EEEzrrXz1w1PoH9a2NAgVuERZbK7T+4AG\\\\\\\/t5aBpuTOZeM1eKC1ZJAx34LbKRgNmJhuIFVjVCPonjP1Hc+y+RqgIo0ebcmVM9LVqk5QYlHv10d1yf+Wk40+Eox9KcCo5ard6QnknQaCw4TTPNZHhg1\\\\\\\/KUYjW5E3SpwmuiAcKZTEAIaRNnKiOvRx7Uq6q5LWd7Wrmjdd47Y0xR6dAml\\\\\\\/wwBCqoQP4o9sf5vRF7iPlxNKYXGXijiRGKUNa+HLpjFqtUWbL7KmlYKEwM9hidf7pTdZb0qMIlvF4O7M8Z50DZHvA+w+dN6IyKMeY+PKCJRu6tqelLd\\\\\\\/9xNxYB4KX\\\\\\\/zZamCVibiYjAaS4KS2adRH7lW9ie1YyvShVczfoo+yDBPDUyjDN9OHKRv0DrnDvGaO7Mqxu4S1Po2+g4aRV5t2WSmT0GO9Zau1lerd8S9OhS5Om7AVukbeZHm8KzB62SRFhPrD+W1goawCc\\\\\\\/idepdj04iJj3IASJxwDIdI9W1QQNgqpJoKe\\\\\\\/nFllNmK2zTLlJ3EyoHwnhYnClt1PEsiNqF+j40oyWD6CDpY2moRHBssf41ImWfLVLAivpjTbKC1tcALTrkiMTUjQafbn2s5ImtEix3GHRhQQKAMulSmDcetuJRcGasQ3CclaNYiwo1Q9O0Tk8bERVZ4\\\\\\\/PdCptf2G\\\\\\\/VhxZbZmDPt\\\\\\\/akw5piF1rMwYx1Nep0WWYZoHJOIY3wx16fQjJ6SpQek9QYdQY86If2twLUKOGph9siJ6jyzuPWs6R9gjS7q\\\\\\\/YBpNmZYder0svrshzu79CGs\\\\\\\/Zfag1ecS8lWWNqzffh4AE4m7YuUlE2t\\\\\\\/bx6m9cj9QL9QvOc2Nnjb\\\\\\\/ErXeNz0oT\\\\\\\/WCAre0x+kM7hqhDRqUT2ghoWsP0bU3WlZrE+EKhdYgyfb\\\\\\\/Fi9I98hXU7dyeH3fnNcZyIUPJ8ah3qsrlVPYNYRRPlXLFgg3HKhHg+pgnGT94euQFlbfIs2R9xX\\\\\\\/zZdAu8+T9nc8SrIguCEZDGz9B+4E75RmkJFbMfJUtueOlO\\\\\\\/wEGAM0dKA4Hnbo5IK2ywLroRRPt9\\\\\\\/3Ya1s7CRqH4ts4ilKqs6DTxp+VnNZWdKAZ7P0bcoUo78kTmudbxMR0ZvY\\\\\\\/+sF69E1OJMsH96UYFYowjVomOFs2jpVrEVNVGLh9X2l3kQzTsWM37Ice25wgqPu5SuIHKWjCo2H4mibJv8kqYtHWZqNi3FsA9tj0ypZKSseambpSrLo2K6Eb9dbOX6Qi0iQFtWa3rJFvF\\\\\\\/DGjHKmOv0BSGQ0NtmjkvhvUcVO\\\\\\\/aGGhSLYOq7cV5xkZiArelXhcyPkGGJPg2CIThfgv9as0+pzarcNGOxDUzesjL1OmFG5TIr8vAkfEcUi+XEMyPt+LV2U4\\\\\\\/4oV6QB9zJJQELIZKWENv+xDn9d1QGbcsFRwfv0NAP2pxIMTgchNC7789gE4rGrU3RrPcd1N8AyeLw8H4\\\\\\\/N1sBZp2o0oaqPqNiac207penggm2f2jiDe46XCugErVT8Dd+H8JZKKFWS1D6nFGATxTTB6+6xPMUKpWEvhKPgy5h8tc0Kz9hmsRvZZMQcfc81yE7F8V3TdH1T4bUSzEDx+iQz76tvkPgzWXfuq6931LoSA38wDEFeb6iyVqBRH00IlEYBRMDbBF3Kzp580yVlHaVTzmsHPF9V+PcK8ekSxPa4aI6lMQifSi0USpIElSNADOe0gWGP5ykpoSM7X+lTsRcOrlcxbR4B68I5xKGsFWv4dO\\\\\\\/0y4CIR+9JsfsAkW3xy\\\\\\\/tCKnQnFZIGB5hh4gjpNVTjvt\\\\\\\/cSyiW28dPHL\\\\\\\/CXKB9ztG7lONtC7XC1hK1pz\\\\\\\/PhzzU2xQs9C93HiILP7w4G1qcUwH44hAXTSgX6uwxpocT3JTdIoZ1jSTISkQCORZqwKWWZy5EkMdGaDjX9IieI7COUMD6ckEjTrI1Fz99PGLKODlflWoc4cEB\\\\\\\/atvwPYixMDV6k9\\\\\\\/rJw\\\\\\\/ZgQxOGJI1+8rXBCNyrP8KqmwchX0aPtlwKG2zlYSDpcYQPL3uY4pijt8T3SAP536xbBbQLsQg42olFoR999lAe5p\\\\\\\/+j2mBDYn+3eGqCu6Hp4OM1d+TzKlXuZntQmNAhQozQsAqzOSIMq3r3nqN3E58+JUMujtGZSZiueA5Xt+qvtqJrJJFRhZY3KuEE791M6jv54umIUM7IK7Cxb8npGU+j\\\\\\\/K1XhsmPnWTJa5NcFQrYz3Xiir7Y7mbUSS5O5rvg7SLSWV9P9G3KL+nVdxKB4oIXDgifq6cMB3cA2iDb399m7wpX1IUZyUyVsHP6CGFVfpOP9VzhpS0veFgHmwMDBTphrPO4GcxKzH1GlAlCzRF5EBWRTzc\\\\\\\/wh+dgIEbjGv9qMfPjGRtt+VfMumfuLTf3w8jC1rZ+DxYQQcPYau2ChjwSgy5dUuGTZ4r3FyVA9e+j8lPwls0udzykD20Gi9xj24+51rYtdUVH3GiK+Z1SOLaofezP7HoNwwByWJ9UuMXneftSFyjW5hpgEJObzyDbR80C9\\\\\\\/WQCZop8N7sVY+ellfAFCb54dJ4p6LTgaGp2rlZfg8J3FhGTQRjGNVq4jllV3VAEeqmQ+iQ2+3Wct4ydzilmEoh4gHZqLIuEX6WwijOQCvN3bGVxU9Qr5DlZkTwAcW+1ztBRRBwDqpOuT6TNeL2wisKOC8f7d+GZGioNKszwg4VEbYkbJlN3GxL1CU4K9h6r2ha85HL8kWIbz+cXLGXSXSUBsn5pQnzIJGEBGSYwKCR5sNs1KCSMdAdYK8Z0L94Z50IKpxIsAt1sVtluYD4i2LTK9heG1HJTYATklC9iTf8pw1P75Cec\\\\\\\/8FROsaW7\\\\\\\/xBefHjD73ad\\\\\\\/TU55pFBKmoi1Bm8eu2gwz7guP2le1Vw9N2y\\\\\\\/MEfSiblM+xcHQvMAJMatG87lHdFA4Xweg35i3lfiZNsOPcrB3cexbev798qVtWXMUMDm0xkLr4v8x1X11eMbHaRhtnEoIBzuJqpgiCKD8GxmDmV\\\\\\\/o6qO\\\\\\\/grRcdi6+ykyCCb+iI19z7r9Ng\\\\\\\/URTuMBqFplppzP5wu4JZJvjVTXa5DxBhjfFmjJ95OlcR9PN\\\\\\\/mXQIzN3P5i\\\\\\\/lJ\\\\\\\/mpcpr0BxLeH\\\\\\\/XcCPdSo+t5J5DLXhmaokUKpBLdcxIESr37h4Echw9hulL2ToqPmcqCqAN6JD1Fi5DD6SA0GnytfDzoe3hEO12sKoaWrrtpchnB+7b4XoVzXjMk8njYQk9lVJY3b21hfPIhuVPpUzLXUnM0MI8VVVXWCrbuvCcshP+bSL0d75ElSXDJ+T5zpVqh2hh658Y3W+n8wTkuUpufOsRItRmvAB1O8ZZB\\\\\\\/8SYibgjM2MDmiOJPG5zo4m4h7GtgsB\\\\\\\/WsQJ6f7d3sJUTSXfO0710ZiO+v030vckaZhT+OWF5PpB8HcqYm5TpH71bo0gl32BAKEg074jwlGbrxJJmPZOVO4QbfjSZoTBW4GRHUcvsMDf6+wV\\\\\\\/aUVu1lILNXtVo+n7RB\\\\\\\/wr8eJVv4OSbVBfhwzUFdcz2arUxHJPhZSOvX3hYll4\\\\\\\/DS+11OceHtXBFcfJ\\\\\\\/s2H+2ElftYN2e7PqGThB3gQmcwH0EScqpPPbtMl2NyJL16YHP54SAVedcN1U4C9nd8AnGuEok4iKccPAlVfNfMEiUVKacHJVZrT9iGegl30AuqJJHTjs3iWjwfXIzJkniDv8ambfgkZ1+deUgIf+xPakifDeQ+3RkSb5rwUoojfTahBNboKN4jhleeJ9CRZAcXEyUt3NdUlcZvMenqEUp4vxuA8PMhBoPJWYOqXaYOV24zzbkt8ZXXQ3E5FdaTJ7pSdFm1jIPmFDrNp6yG6E1xMfFabK40Qk6pCoCP8QJbaKpQpOKuQVVdkvMWFYPVEK5gbqGNk8wWV6rS++HztuXvl3dNH43rArPH2njq1MgHqvTc2ZYaNN024u1stEpiO\\\\\\\/SHtyYUw5iAonekp12SuNGEW7PVa8jJdKi\\\\\\\/tXPerpUyJlueiJ7p\\\\\\\/1o8u0g00ij3jh4cyIrAp6qNTTf6DN7voTkQKCX6Vk+YUjFXv3FqZsZJMLX8WpgJWO9fIUfVrSVbNQWICrlVhsJaiwtEVlE91Imk4SmaJ97ra8cGaEPGRUv2rVdUHvVtJGEJuiweMwZYiOHwzv9VcykInbH2S\\\\\\\/wkEj9+eMAtph\\\\\\\/fSHDLiexJjh0kaCBvOlXObJ1szyIXWEErkSOkQRqfeAaCVUR\\\\\\\/Ptw6oHzOcBc4LEucRrwhbnCYcVPiaBn2NaQa0GdwYt\\\\\\\/TTmjfIw\\\\\\\/tGs6m6UCm2GM8Emq+\\\\\\\/D9o2Dac\\\\\\\/po9hN6kX5VHSu3v2usEFHuuLsfyEv4B2cApPrIN5zz7fh1JftU46NxZXACG+htNsmM7fOwaYvgaF\\\\\\\/nRdN4\\\\\\\/kKEaBKjPIGAjkzXqPS86ynLTvYiLdtBjzZkTC5N6cbyPRIk5iA2I2mDIcAK8cH4d9lSLtkQVzn4uU6MApG9B5q6yNoiW0qp\\\\\\\/\\\\\\\/yDRls12CjvCcNyqtzdICTeesTfrSgOG2uvxMI6WSGSd1z2T0M7Mf\\\\\\\/Nqm8hc+t1ZGCXRFkqTMEhi5U8GcXbsMd2eZ5y6ZCOwbowVisdKk0D7KXs9o76H9IjaEPeJd1STLmw+kAJChIiaWhEMp4k\\\\\\\/aYSwMI92tciCEhdh1EJsKtJfQRXVlvaRQwGihDzF4KMKyTglF1tXnIrZaHmZF6ltEWAZN8VBCNxKQ2etItxJGIDt\\\\\\\/XxaEyN6BYQ+efh+LCKUgm6M+fvU5GPBLxy1yzYfYBLRiAQ9n\\\\\\\/dppAn4PHvaAy6R+MdkZkBz4y2S9bw+meNA6RRywSWQwWNfTdl3bPqpQzsdCHljXzeusDOcKQOL4YhzXPCjX7WmC4D6+CTt2mEUr3ntHw\\\\\\\/aRUViHwBZZO+Py4HqyJC244MAH6dYiV7p\\\\\\\/WyHmNeeLFXOxd0Z4rQpQz2LiMbDQ0Rwirm396M0TvfNboJ32dYFBeWoCktN7h+72dafawBrplAySfs+fMuoaRayJxdU9Yqe\\\\\\\/4kHwrRb0tbAFstIQKIcx4juyS5Q213FNygkU7Oc+AP8ZIJz+DQlHHhJwXvInbNgYurdvT0oGFMLRI\\\\\\\/RMQ8DGE\\\\\\\/j6ohZ+W\\\\\\\/nO65p5BDBzipGZ790qfZ0RaBId1Sj2aRpmntlBWwlfbo28OvpDF4BdSZKovqxTa5Y88uCXelU0f3G7epjE6D4vnV7nL1UO5hbN0wYuSHvhkylnAJA2ohL8MnSnx0PzQSagzA3ZCd9KsHRZ4b\\\\\\\/tIBoqYqYAtyPxvmrb770n7btl0+t6zVTnuqTFAPjuE997Qs8q246PAMX8fILm+kS1k686LfSUTmqXgvA3\\\\\\\/X9aszsn4hQBV0AT1QTUldex4gOtMzDqYxxDkf8qan5vOO9a+5LV+lC\\\\\\\/Z2ZRNSzYFghGMKsX7exQH\\\\\\\/pmXMD9t4axdCEWPqXotixWkA41dmp\\\\\\\/VjnLCQjl0+KchgegqQEcRG7cUa8Iz\\\\\\\/ngwAkCdkQ3O4WlX1tvC04zRZNWSKarCyAUGi1z9OxPozk\\\\\\\/6lJMrr1ruFrPtYGo6Qc85yRbA3cdRFcguEGxpT6jqQGTBb+tCl7RuSHPitOWTk73UgacSiI\\\\\\\/S9heVarLO3NOaafNJiiJ8IYASdNATl5cma1GT+ThURwOc0nhSYAkuunJ+grx10xteQGP7wfwn+9Uzlx7JuxD3Z4KdIRSo7BbihZ6pexzamHCJtoBrcuk4Z8fvtLrvoM92xXH6mR1CnWP\\\\\\\/s9bavHPsMbERBdcznPOpI0xrDQMhds75Pi1qWVCjPGOpyySD487sZWPifVEnpca6mAKJxLtE4oNIW9XoK3OLSK1p\\\\\\\/gcYUUEbe\\\\\\\/NbTLsKjNxDAc+IJk8RGofGBJarvgEMXtLhYJC5y7JgyN9BGKeIPEpYS6MB8hKd5sbt7Oxit\\\\\\\/C3eb4n\\\\\\\/sP+pkPKH41WQhxPufhx2pYYyAWlKeUwmOFNSriPL6U+v1VxmrE5YadBZ2zipghQ06nI6GJhsQbYx6PD7sGVHQBGTz03l83PoNntjxx9aUWxIGLINPbhUQBRKD705NJ8O3XhX61xmPZAEUYZ8FE5nkWTMcLdl+gpujGQrktym5jVQa0SL7TDnOJU2ALYUkT\\\\\\\/wUAU1\\\\\\\/nExkwafnW7zWbjN7T0C5MDQHLaSK5xdJyd9q9oIRh0PJgk\\\\\\\/3wcGSbcbgLJ7btNz40+1lVutNipLI55JHII+QIBcbusPEcmQyjOqOWohqER448G6cRL50WnsZRg\\\\\\\/GPrzTJSiPrSxRSLiDcz8ttzZpTxTtI3PC2p6oNQkmWAQ0F23aYNm2Y5Kfv5S1GuJBErBBQ\\\\\\\/j1qeGh\\\\\\\/dM+fG94Cy61u\\\\\\\/JTETyTJHu3k9ot7LmddTdrZ2ywuG5xeZJQlTcTVbnnTJbmJLjwwI+doAVC1SE7XyaDwT+T6zYzFEyqXZVDkrBzKUWQIExmyvET4LZK2x0Udyh5Fahi1VbxBpki91V0MJ2wwYtg\\\\\\\/XMzRsRY9hpKpkfUVK0DC734y47UnwnhNlEVcqtqSSv7in\\\\\\\/5Me5J0JHrivoAc6EynOfTkXZjSGU+UsGmrlMaETCF0Z9+zE9OFCHRvMhuWPY012f1aR3\\\\\\\/ZdJ+N10EdTOJOGNbejv6aBwHHvNo8pCuDD6XmSqTLGLNMCkPNprgmkAwkUALIDflh+shdfDYH3F0IuTWbRcWTN4Qe81a\\\\\\\/iRIqP\\\\\\\/li+MAwuBnuv0DHMDtOSw9LbFC5EABiwIe3zlDWLh4ZGIzaHB4dIOJLnMsSQvlfZe7wg6eLR6n1lKXZY95WvFC85emht2vtxflp4JbOefZin\\\\\\\/\\\\\\\/YcKbS4ZrqYFaKZLKh5cDX1mLzC6VSfifXCqYAZDRAN2EpQBw80I+GsMRcmspG7wU+hlv3pjAzrtcaxXWmcAxX4lybADxHj7eSeKUh1eOntJOl0i6I\\\\\\\/3EYSJvFyi+e1h4zuxIcXzb1P\\\\\\\/ydT9n3fghgg5HXRUmEuSsgI7wrZFzWi+81+6kXVSr+UFvub6PcKKspWHygkyhdT4aWJEHZ2G65AvIF4ks2vqbdP4Nbd28Vq6i89y9+5Lf3iWUI2W1vOX3WAd3IODZ0Xia6hk8TOm+fOslSkQxpJG9GgejNs5TPzTEuNlo0xR5b1ep8ICQ2e11IUHT8SiZpzGuL7GpccYEi\\\\\\\/vztYBUJv42vGo6UGf5XZLsvYub9A55tfcuLnjLGwwZmNnnYb0q4aau4ZAVvFB2ooqY97WKN\\\\\\\/xTRwKWH3zSzNCMAUohbqCxv5BFEwDlEmpEse7WFDElqJi1CLBbjdUqx19HmnwZGvY1HN1S+3YR11macm1UrYaSDsBWH5EcJVK83OIaVAgHOCrHsewJMwpHndt0f4+gjIJFWucyud\\\\\\\/gcTiVZZJuzSnwmnQMUjk42qJjNuKWWTwfZGI9LJYJ+uhRO+O7EZbW31v5SR9cJsuOyVgTUXF4HFdhebuWbB6R0CYZcq2lVzyatrzjWuddAFh\\\\\\\/BoL+++t+QxLKaKOICsc3o0+Hd8Ur1xYkZNWMpZQ4\\\\\\\/seoy259vd5FbfFawdrKY0Y19dPBKKDM+eoJJycmGkPYr9gHCT7BJkFfO5PG1o3sGUAP6S87SAheRejmH3pFLQ7R2h0\\\\\\\/nGZbOBC5zfTVcJEwbFKWofGa8GKi0TJL25VJaQOAgY5bKzO86wDgVNjDXzBnO\\\\\\\/6mWqMU6Z8c1pO+KfARDyCUpoXQ1A\\\\\\\/LEIAcOp+T1jH5OH\\\\\\\/iDd68ldlLMSAy6kxv7X2faN\\\\\\\/0lbbTFsnMCeQ\\\\\\\/OtaGGZ1ftUMYaxLMvonjOYhYaxWhU44R43hlkl8fj0CcTlnd7Cvt4bYzA9haY6\\\\\\\/AAeehaRKwhTFSMP5hbncE4wyknZYJCfQoiHLHaZQJWeeLuh6HeGoVPs8rvcB2Oh9eLBO2690Y1ng7sxovKUksTvVXb1tUWtZqiLsUpqS9W3Bt8dlV6lIZQU31s5+7tOx0Bj7M6gBjPYbcAtfdsUTBuwViKrSdW0CUHWqfTD7mgcJLP63Hkze0JwJaKbgS9svQXwNd8r78XipouQ9FLb3VJ0U\\\\\\\/jIcRMNKwT6BG59aKmWJSoNT7o\\\\\\\/XWfjLJxrZa6ARS5VH5kxz9O1mfz0MEcu0kQCzu4R56R3ZcvHTC1H60XRjZVdrXjvy9f3oNM2HVYrmFLRZFM1oDOEcTdXSsQoxNgDFFXI4v1wDreuwHpQZckS66+CUfYTH5LBOHvZ\\\\\\\/nOsBuqDxHBIFHFnKyIN0zYeSv9qkssCq16LFg6jh4I3IgwWWJiDzlyq4HTjJt4FUJ4ilGztpMdDUYNFPNRxt6SoxLSgWQXqnSsXHwNYobkmmxzMtdGa7\\\\\\\/0i\\\\\\\/HTg43VE9It7YhwLaUVVjTlxC3YA605vhrVuS\\\\\\\/8rnePYO\\\\\\\/o+IoBBqt1iwG9JZI+LUeNTng2wWF2\\\\\\\/f7NPoG8huBBBmKVT6DfKnCNBCh4oHyykMLz61XmtePAH8qtC4JgfyCBKH\\\\\\\/0jWG4UFiirGSNK4BP8WfenRDkEtw1Cv6HvOBza1mdH+byJv5PEZNJ0DdFTaeISgNG1UqaQXJ21vrR8rP4UPRS75+lE7km1qjbZ002ZNM6vkwPZwpWry8Snt5WEYCOW3lL66WwP2CXRHZkAAIieA2NsDg3W\\\\\\\/a6370pufFVM2CXLkUP7XbbCwZyJIi9PE1Ce+KxEaXWYmqDZOh2PWNPvBbHkwvF8mRJFIv+Hab48VVP1CBBmYF4AJJX5ivJ8SbhhJtBt5hE+vTmjnKrIJMW+VtdZQIE0QfjNq2gPKEHc1jQIkcozxPfaCYkV4nMj61cSTgzxOXFFrqy3iRP+TX\\\\\\\/iQ8Cyc55iRddlEpEJyd7p9FyKl+8HX42XgNVq2UTwpW7AtJAdZjozdfbTaexjSa4Ms90tdY2o9Sqksw4EnP6PmrK035PXE0V65yk11X9iywTPvKt1IVMgwNYGNymLM27u4gkKy3eGpXgV59scsIL+QzvVMnssN+qeJeu\\\\\\\/ZTpi26PXcg7LdAbPlGhXC4paDtsIoxwJ1kfVBQiqmVUbBqX\\\\\\\/k\\\\\\\/oIZfITyKG3bUX69voOGf6iuMgUPwUyy1AJYTOMfqaUftf1xlBJG81rgjUIy843af1vfBy9Qrkzw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022U7hv6EO51KDPQDvHByAd\\\\\\\/Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f8a4018a85e37cebdb858d7583e37d271adcfed2030c63e6bcdbcc04f5143023\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Joar4M6uxa\\\\\\\/CeswcasNtISkAdVKNUG0bT00cOQG4PgVJ8PXbhdEA3HVvKqiQMcZQi1hxawarvL0QpyZB76lObTILLIrt9k1Q8qLh8tsANqu4bcci2ylkW5glOhsG+k8BaSG+U7C2NWFabGLNwITgvgA3B\\\\\\\/xwuuUjPTI8ufr\\\\\\\/wtqxY56cnGGKRAsKKnitCBgJnqJffubuo202BrgyYC6wDIMoDvcJnvD9GjajsuIhV1zUY2f1E+EO0RhonFdF5\\\\\\\/VfKKpWej0ZhImlsvte3nyp60dX5z\\\\\\\/zciq0R7tst3PZYcIIvs7ngy589+m\\\\\\\/NNq4YziZhG+Ddq2WwMtW1i5oFpvx\\\\\\\/6rDiMR+DKkGk1Gyqi3c7hQO16MT1qh0UVqe\\\\\\\/LufpO7+1mz1aCu+KTiiEX2+t0Jc6Z1RGYLTc8m7717pK7uU8ouBggQXR\\\\\\\/iQr\\\\\\\/qZ6C6ZUEeKN0l\\\\\\\/fQQrk\\\\\\\/0BUnGii4fNBtQ9Bmd6siOMc3NW303AMsDDAcdi1TY0ujvKKIh5a+AeBgvPnzzSVUd6Qhqq5dr\\\\\\\/ascOlJoSB9FY9wrCzOweRr3KflWg123fu7dzxml7EFGgbhVQClYRIdvkms6qn7AOE95EQGafNMQpu+9wJ9L2VljN3g\\\\\\\/8UGAZAw+\\\\\\\/rp14W9WFI1utz5YRni91pIfBR+ONN41Ak8XiM5WqkFHXSQEznjg\\\\\\\/E1ElOfI0FsNra0RJa6kcogNKW1EX94rOyQ10M9ds+roBwXH\\\\\\\/t+DNPkWOubN4Ny0TWYw1pTX8ljs3UNpn+pDYNbzIS\\\\\\\/iAWZ6X+eEq7WPQE+jJZpT80CfOI3aoGnYMJIyZ9Fv32Y8DAJOnLUANiJtk74TGsbbdsw4p5fqs+euG07IPz4mStaugiAqNejLirx6ds1CAmcr9VxDshFVMs8X5RIxImw\\\\\\\/jwQDae7XmQtd8J44zsrhnSBB1VO6QWgDyOIZpdLFHliaoY9M6SMwOli\\\\\\\/mOE9PfCGHDLPK7ztEEXowGC5O3FvpHG21ohOzytMZpH0s\\\\\\\/wdeOzo7ADg2w2UybBiB65JMdSWegpnRkBRiFugvNU9e+oGjn7+1jQIEJWZu0+YNYaDnFm8qADaIfoXl5Ehbedijv4YMcCZXmzFjrMlCEyPOTgsOEN3dWIMsEFhzvbiezcp94vifqihBs799SvsaauHBl1mGYonhiQp3ozy5eD0HsYzOVyG8+5oBul\\\\\\\/\\\\\\\/MMrG2CFoNo0jQTd1SDfnminmbcaUuorbcswv1tABnxpMFXSB3mqoAnCOgBy5e0nfzeC5Pb\\\\\\\/ZvhlEjOUv2vEmYvUc6a7Y1SJeBckcil+kMfBiLn\\\\\\\/bQFUrdho4C+8\\\\\\\/ctwZ07aAciyi4EJW63n4270xWUM4hYLBj0\\\\\\\/gfobhaDM5Hm9HsF5d7EsCDATAHPd8Hl5PKsw+nKFVEvvTbY8Z4tecAAnyCSjqNCwwbM9qOHWVenfMoKQoZz5zXg2RwqLZ7SVeHmJJxiZDFNV+YXkZO48JcQhMr88tGcpR4tgKxDP2kuJ4ijV6WGUw9moiRnmEvof2fHBvGaQ+p8xZqrwZ1FhGflyOIwCMP7em8dB5e8J9+peZ2+C5l3dvpM0nt7NGp0C7dGbONGQu+UQho2QdJFMnQEhwiIoSG3NJUgRbIxDtJgm62OQR7VaOcmgoAfyaAnqziALdkA4elCZD4ljp1ltmtWDOgWxmdGH338pnFhMrWzWImNe6g0rvGnoFimJvihI+FLgPegQ81GsXjx4pRTrQ7MIENMK99BWVgwdqBI9LqD3\\\\\\\/xaTa3eBM4LVldeFyIjL2j7H89gaDD0zyJz1jEURB883srSXzXBeM0xNNarSdPrngF0Da7TJqJC8lvR51q35IIZtJQ6794tL0b+6sHLeDu46H+ha6inUhfBikUDpoEizM3pG3vQDrOrNMxNttoUKP7WYPjDQbMlALe2Jv9CKuYHxAQjNVD5vlK1NhChUJa6P8815Vjm0eH2PJttTt1B6UsuwRxGBIRaTENyIJAaS\\\\\\\/4sStrXovtzplg97\\\\\\\/wKPUIMd4yw==\\\u0022,\\\u0022iv\\\u0022:\\\u00226L4TugwBH\\\\\\\/AhMoFHnHEKpw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ea9d4829c02901e6f2aad1180b1467e98b88669ec73ab53a7db032020c2592d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459180 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/4/303" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tXDu77lsLftyQf3o2BCARp9vD1b_9HcJifv-EdGwJR8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/4/303" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JOYKgmGF6XClMPiP993whSQjoShPIXETqh2Xe_VaH_k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/4/293" title="Identification of an Active New Mutator Transposable Element in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/4/317" title="A Synthetic Human Kinase Can Control Cell Cycle Progression in Budding Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459180" id="node459180" data-pisa="ggg;1/4/303" data-pisa-master="ggg;g3.111.000307" data-apath="/ggg/1/4/303.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in <em>Neurospora crassa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kevin McCluskey</span>, <span class="highwire-citation-author" data-delta="1">Aric E. Wiest</span>, <span class="highwire-citation-author" data-delta="2">Igor V. Grigoriev</span>, <span class="highwire-citation-author" data-delta="3">Anna Lipzen</span>, <span class="highwire-citation-author" data-delta="4">Joel Martin</span>, <span class="highwire-citation-author" data-delta="5">Wendy Schackwitz</span> and <span class="highwire-citation-author" data-delta="6">Scott E. Baker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">303-316;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000307 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459180" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kevin McCluskey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BMcCluskey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCluskey%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BMcCluskey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mccluskeyk@umkc.edu" class="" data-icon-position="" data-hide-link-title="0">mccluskeyk@umkc.edu</a>
<a href="mailto:scott.baker@pnnl.gov" class="" data-icon-position="" data-hide-link-title="0">scott.baker@pnnl.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aric E. Wiest </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aric%2BE.%2BWiest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiest%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAric%2BE.%2BWiest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Igor V. Grigoriev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BV.%2BGrigoriev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grigoriev%20IV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BV.%2BGrigoriev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anna Lipzen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BLipzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipzen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BLipzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joel Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wendy Schackwitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wendy%2BSchackwitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schackwitz%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWendy%2BSchackwitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Scott E. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Chemical and Biological Process Development Group, Pacific Northwest National Laboratory, Richland, Washington 99354</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BE.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BE.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mccluskeyk@umkc.edu" class="" data-icon-position="" data-hide-link-title="0">mccluskeyk@umkc.edu</a>
<a href="mailto:scott.baker@pnnl.gov" class="" data-icon-position="" data-hide-link-title="0">scott.baker@pnnl.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/4/303" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459180" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459180/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/4/303.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459180" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459180/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/4/303.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459180" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459180/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/4/303.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459180" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459180/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/4/303.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459180" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459180/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67266 first last"><a href="/content/3/1/143" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - January 01, 2013</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Classical forward genetics has been foundational to modern biology, and has been the paradigm for characterizing the role of genes in shaping phenotypes for decades. In recent years, reverse genetics has been used to identify the functions of genes, via the intentional introduction of variation and subsequent evaluation in physiological, molecular, and even population contexts. These approaches are complementary and whole genome analysis serves as a bridge between the two. We report in this article the whole genome sequencing of eighteen classical mutant strains of <em>Neurospora crassa</em> and the putative identification of the mutations associated with corresponding mutant phenotypes. Although some strains carry multiple unique nonsynonymous, nonsense, or frameshift mutations, the combined power of limiting the scope of the search based on genetic markers and of using a comparative analysis among the eighteen genomes provides strong support for the association between mutation and phenotype. For ten of the mutants, the mutant phenotype is recapitulated in classical or gene deletion mutants in <em>Neurospora</em> or other filamentous fungi. From thirteen to 137 nonsense mutations are present in each strain and indel sizes are shown to be highly skewed in gene coding sequence. Significant additional genetic variation was found in the eighteen mutant strains, and this variability defines multiple alleles of many genes. These alleles may be useful in further genetic and molecular analysis of known and yet-to-be-discovered functions and they invite new interpretations of molecular and genetic interactions in classical mutant strains.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism" rel="nofollow">single nucleotide polymorphism</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/indel" class="hw-term hw-article-keyword hw-article-keyword-indel" rel="nofollow">indel</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/classical-mutant" class="hw-term hw-article-keyword hw-article-keyword-classical-mutant" rel="nofollow">classical mutant</a></li></ul><p id="p-5"><em>Neurospora crassa</em> has been used as a genetic model system since the early 1930s (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lindegren <em>et al.</em> 1939</a>) because it presented significant advantages for genetics research. It grows on chemically defined culture medium, has a haploid vegetative stage, can complete a full sexual generation in under a month, and produces ordered ascospores (the haploid products of meiosis), facilitating genetic analysis (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Davis and Perkins 2002</a>). <em>Neurospora</em> was quickly adopted by a number of laboratories and is famous for the demonstration of the âone-gene, one-enzyme hypothesisâ (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beadle and Tatum 1941</a>), which showed that classical mutations that could be followed in a genetic cross were associated with biochemical traits that could be characterized chemically. By the time the Fungal Genetics Stock Center (FGSC) was established in 1960 it was estimated that hundreds of genetic loci were being investigated. The classical genetic map of <em>N. crassa</em> now comprises over 1,000 phenotypic markers and an additional several hundred genetic markers such as telomeres, centromeres, the nucleolus organizer, and a variety of translocations, inversions and duplications (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Perkins <em>et al.</em> 2001</a>). Because of its many advantages and the relative ease with which one could produce biochemical mutants, <em>N. crassa</em> was used for research into every aspect of genetics and biochemistry. <em>N. crassa</em> grows as a haploid filament that makes copious asexual spores in vegetative culture, and when compatible strains are paired, it can make either vegetative hetero-dikaryons or go into the sexual cycle. To facilitate allelism tests, strains that were vegetatively compatible and displayed high fertility were generated at Yale and at Oak Ridge National Laboratory, where <em>Neurospora</em> was utilized in extensive radiation biology studies (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">De Serres and Webber 1997</a>). By 1963, much of the research community had adopted this so-called âOak Ridgeâ lineage. Therefore, many mutants in the FGSC collection have a shared lineage (e.g., <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Perkins <em>et al</em>. 1962</a>). This lineage gave rise to the current wild type strains used in both the <em>Neurospora</em> genome sequencing project (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Galagan <em>et al.</em> 2003</a>) and in the <em>Neurospora</em> functional genomics program (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Colot <em>et al.</em> 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>). Additionally, a compendium of <em>Neurospora</em> genes, first published in 1982 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Perkins <em>et al.</em> 1982</a>) and updated in 2001 (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Perkins <em>et al.</em> 2001</a>), is currently maintained as an electronic compendium online (<a href="http://bmbpcu36.leeds.ac.uk/~gen6ar/newgenelist/genes/gene_list.htm">http://bmbpcu36.leeds.ac.uk/~gen6ar/newgenelist/genes/gene_list.htm</a>).</p><p id="p-6">Despite detailed genetic, physiological and biochemical characterization, many classical mutants remain anonymous at the level of the genome sequence. However, because of the extensive genetic mapping done with <em>N. crassa</em>, most classical markers are flanked by genes that have been identified to the level of DNA sequence. This makes it possible to exclude most of the genome from consideration when looking to associate a genetic marker with the open reading frame (ORF) responsible for the mutant phenotype in any given strain. Whether the approach is chromosome walking with cosmids that are mapped onto the genome sequence, or by gene sequencing, the high resolution <em>N. crassa</em> genetic map allows one to use flanking markers to delimit the search. This approach has been used at the FGSC to identify four temperature sensitive lethal mutations by gene complementation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dieterle <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">McCluskey <em>et al.</em> 2007</a>). Advances in genome sequencing technology (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hobert 2010</a>) have made it possible to sequence the entire genome to identify individual mutations (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Le Crom <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pomraning <em>et al.</em> 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sarin <em>et al.</em> 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Smith <em>et al.</em> 2008</a>). For <em>Neurospora</em>, this is complicated by the fact that not all classical genetic mutations have been crossed into the same genetic background as that of the reference genome strain. Eight of the strains sequenced in the current report were explicitly backcrossed into the reference genome prior to deposit into the FGSC collection. Ten were reported to have âmixedâ backgrounds.</p><p id="p-7"><em>Neurospora</em> is a cosmopolitan fungus and it has been isolated from locations all over the world (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Turner <em>et al.</em> 2001</a>). While most genetic research is carried out with <em>N. crassa</em>, other species are commonly isolated from the environment and have unique characteristics. <em>Neurospora intermedia</em> is interfertile with both <em>N. tetrasperma</em> and <em>N. crassa</em>, and progeny can be recovered from crosses carried out in the laboratory. One such cross was carried out to allow a meiotic drive element, <em>Sporekiller-2</em> (<em>Sk-2</em>), rarely identified in wild isolates of <em>N. intermedia</em>, to be studied in <em>N. crassa</em> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Turner and Perkins 1979</a>).</p><p id="p-8">The FGSC collection holds and distributes a large number of morphological and developmental mutants. A subset of these for which the underlying nature of the gene defect remains unknown were chosen for this study (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In this article, we report the whole genome sequencing of seventeen classical mutant strains of <em>N. crassa</em>, and the putative identification of classical mutations in sixteen of them. An eighteenth strain that carries the <em>Sk-2</em> meiotic drive element, originally identified in the related species <em>N. intermedia</em> and subsequently introgressed into <em>N. crassa</em>, was also sequenced. Overall, these mutant strains exhibit a wide range of sequence variability that is directly proportional to their being related to the reference genome strain. Much of the variation is shared among strains while other variation is strain-specific. Some regions of the genome show enrichment for unique variation suggesting that they are hot-spots for mutation. Insertions and deletions manifest a strong size bias associated with their presence in coding <em>vs.</em> non-coding DNA. The ability to compare among multiple strains and exclude shared variants aids in the association between neutral polymorphism and phenotypically relevant mutations.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460841/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460841/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1â</span> <span class="caption-title">Strains of <em>Neurospora crassa</em> with their relevant genetic characteristics</span><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-10"><em>Neurospora</em> strains are described in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and are available from the Fungal Genetics Stock Center. Genomic DNA was purified from mycelia grown in Vogels liquid medium, using a simple phenol/chloroform extraction technique (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lee <em>et al.</em> 1988</a>). Mycelia from mutants with limited vegetative growth was produced by macerating tissue in a glass tissue pulverizer under sterile conditions. DNA was randomly sheared into small fragments of between 200 and 300 bp in size using Covaris E210 according to the manufacturer's recommendation. The overhangs created by fragmentation were converted into blunt ends using T4 DNA polymerase and DNA polymerase I Klenow fragment. Using dATP, base 'A' is added to the 3â² end of the blunt phosphorylated DNA fragment to prepare the DNA for ligation to the adaptors. Adaptors were then ligated to the DNA fragment using DNA ligase so that they would hybridize on a flowcell. Finally, using DNA Phusion polymerase, PCR was performed to selectively enrich those DNA fragments that have adapter molecules on both ends, and to amplify the amount of DNA in the library. DNA was sequenced on Illumina genome analyzer II. Reads were aligned to the reference genome sequence (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Galagan <em>et al.</em> 2003</a>) and single nucleotide polymorphisms (SNP), insertions, and deletions were characterized using maq-0.7.1 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2008</a>) and with BreakDancer (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2009</a>). Default maq parameters were used for SNP calling and for filtering as described in the maq paper, (maq.pl SNPfilter -f cns.indelse -F cns.indelpe -d 3 -q 40 -Q 60 -w 5 -N 2), with a subsequent minimum map quality filter of 30 applied after the indel proximity filters. The minimum consensus quality (phred scaled likelihood of the consensus base being incorrectly called) and depth filters minimize the likelihood that random sequencing errors lead to false positive SNP calls. A direct estimate of false SNP identification was not determined, but is expected to increase as the divergence from the reference increases. Though this group of strains is not a freely reproducing population and the variants have not been confirmed to exist at an allele frequency to fit the definition of a polymorphism, all of the single nucleotide variants are referred to as Single Nucleotide Polymorphisms (SNPs) for simplicity. To allow comparisons between strains, each SNP or indel was assigned a unique identifier that included the contig and position. For example, a SNP would have an identifier such as â3_591470_Câ indicating that it is on contig 3 at position 591,470 and that the base at that position is C. It was necessary to include the base in the SNP identifier, as multiple polymorphisms may occur at the same position. An indel would have an identifier such as â6_91954â indicating the contig and location; size information was not encoded for indels but direct comparisons among strains were directly possible because of the small number of unique indels in any region of the genome. For validation of the deletion in strain FGSC 3921, PCR was carried out with 100 ng genomic DNA using 0.5 uM primers and taq polymerase following manufacturerâs instructions. Primers for amplification of a 1.2 kb coding fragment of wild type NCU03436 were as follows: Forward- 5â²-CGATACTCGCTTCGTCTTCC-3â², Reverse- 5â²-ATCATCAAGTCCGCCACTTC-3â². Photographic microscopy of culture edge morphology was carried out using a glass microscope slide culture as previously described (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Dieterle <em>et al.</em> 2010</a>).</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Association of specific phenotypes with genes</h3><div id="sec-4" class="subsection"><h4>tangerine:</h4><p id="p-11">The morphological mutant tangerine (<em>tng</em>) causes the production of enlarged conidia that are delimited by a membrane, but no cell wall (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Springer and Yanofsky 1989</a>). Although the sequence of strain FGSC 3921 revealed neither unique nonsynonymous or nonsense mutations, nor frame breaking indels in the region flanked by markers in the 1 Mb region delimited by flanking markers <em>arg-5</em> and <em>pyr-4</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/000307.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS1.pdf">Figure S1</a>), analysis of the genome sequence of FGSC 3921 using BreakDancer (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2009</a>) revealed a unique 572 base deletion in this region that occurs in NCU03436 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). PCR amplification of this gene from FGSC 3921 confirmed that the deletion was correctly identified in this strain (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). NCU03436 is the <em>Neurospora</em> ortholog of <em>cpp-1</em> (<strong>c</strong>ell-shape-<strong>c</strong>ontrol <strong>p</strong>rotein <strong>p</strong>hosphatase) in <em>Fusarium verticillioides</em> where studies indicate an involvement in control of cell shape and fumonisin production. The phenotype for the NCU3436 knockout strain is identical to that of strains containing the classically derived mutation in <em>tng</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460834/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460834/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2â</span> <span class="caption-title">Mutations identified by whole genome sequencing</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/303/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Morphological characterization of FGSC 3921 and validation of the deletion detected by whole genome sequencing. (A) FGSC 2489 (wt). (B) FGSC 3921 (tng) showing hyperbranching, swollen hyphae and cytoplasmic bleeding at the tips. (C) FGSC 16003 (NCU03436 [stk-15] KO) showing hyperbranching and swollen hyphae. (D) Agarose gel electrophoretic analysis of PCR products from wild type or a tng genomic DNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133295909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Morphological characterization of FGSC 3921 and validation of the deletion detected by whole genome sequencing. (A) FGSC 2489 (wt). (B) FGSC 3921 (&lt;em&gt;tng&lt;/em&gt;) showing hyperbranching, swollen hyphae and cytoplasmic bleeding at the tips. (C) FGSC 16003 (NCU03436 [&lt;em&gt;stk-15&lt;/em&gt;] KO) showing hyperbranching and swollen hyphae. (D) Agarose gel electrophoretic analysis of PCR products from wild type or a &lt;em&gt;tng&lt;/em&gt; genomic DNA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/303/F1.medium.gif" width="440" height="115"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/303/F1.medium.gif" width="440" height="115"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/303/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/303/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460869" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-12" class="first-child">Morphological characterization of FGSC 3921 and validation of the deletion detected by whole genome sequencing. (A) FGSC 2489 (wt). (B) FGSC 3921 (<em>tng</em>) showing hyperbranching, swollen hyphae and cytoplasmic bleeding at the tips. (C) FGSC 16003 (NCU03436 [<em>stk-15</em>] KO) showing hyperbranching and swollen hyphae. (D) Agarose gel electrophoretic analysis of PCR products from wild type or a <em>tng</em> genomic DNA.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h4>fluffyoid:</h4><p id="p-13">Strains with a mutation in <em>fluffyoid</em> (<em>fld</em>) send up aerial hyphae that only very rarely conidiate under normal growth conditions. However, when <em>fld</em> mutants are grown under carbon limitation conditions, conidiation occurs at 25Â°C but not 34Â°C (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Springer and Yanofsky 1989</a>). Strain FGSC 7022 has over 78,000 SNPs and 8907 indels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS2.pdf">Figure S2</a>). Among 918 indels that are unique to strain FGSC 7022, one unique deletion was identified in the region near <em>arg-14</em> and to the left of <em>his-5</em> that causes a frameshift mutation in the annotated ORF NCU09739. This deletion is found in all reads covering this region in this strain and is the only frameshift inducing indel in NCU09739 among 18 strains (there is one instance of a 3 base deletion in this ORF in strain FGSC 821). There are no unique nonsynonymous or nonsense SNPs in strain 7022 left of the flanking marker <em>his-5</em> further supporting the conclusion that the variant in NCU09739 is responsible for the <em>fluffyoid</em> phenotype. NCU09739 encodes a protein predicted to be a Zn(II)2Cys6 type fungal transcription factor and is annotated as <em>all development altered-7</em> (<em>ada-7</em>) in The <em>Neurospora crassa</em> e-Compendium. Deletion of NCU09739 leads to defects in conidiation, hyphae and female (but not male) fertility(<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>).</p></div><div id="sec-6" class="subsection"><h4>compact:</h4><p id="p-14">Strains with mutant <em>com</em> make small colonies with restricted radial growth (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Perkins and Ishitani 1959</a>). Genetic mapping places the mutation associated with <em>com</em> between <em>ace-2</em> and <em>ad-4</em> on LGIII, a span of 741 kb containing only 44 ORFs on supercontig 3. Strain FGSC 106 has over 23,000 SNPs including 1,033 unique SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS3.pdf">Figure S3</a>). There is one unique variant in FGSC 106 that occurs in the region delimited by the flanking markers and it occurs in NCU06508. This variant encodes a C629T (ccg to ctg) mutation at the DNA level and a Phenylalanine to Leucine change at amino acid residue 210 out of 1123 and this variant occurred in all 76 reads covering this position. NCU06508 is annotated as a glycosylphophatidylinositol anchor phosphoethanolamine transferase-3 (<em>gpip-3</em>) that is involved in production of glycosylphosphatidylinositol (GPI) anchors between proteins and the plasma membrane. RIP mutants in this gene were found to have defects in morphology at both the colony and microscopic levels of <em>gpip-3</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bowman <em>et al.</em> 2006</a>). These defects are very similar to those described for strains containing a mutant in <em>compact</em>.</p></div><div id="sec-7" class="subsection"><h4>amycelial:</h4><p id="p-15">Multiple phenotypes associated with the mutation <em>amyc</em> are small colony size including, increased cell wall thickness, and non-hyphal, multipolar budding growth under restrictive conditions. Conidiation is considered âconditionalâ and is dependent in part on carbon source. A single unique deletion was found in strain FGSC 305 among the 166 ORFs in the 742 kb region delimited by the flanking markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS4.pdf">Figure S4</a>). The G at position 3,212,445 in gene NCU02689 is deleted resulting in a frameshift mutation and this deletion is detected in all reads covering this region in this strain. There are no other indels in this ORF among all the strains sequenced and a knockout of this gene has altered sexual morphology (FGSC #11464 Mating Type: a NCU02689.2 Heterokaryon (<em>lrg-1</em>)) (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Colot <em>et al.</em> 2006</a>). Additionally, a conditionally morphological mutant in this gene was identified and was named <em>lrg-1</em> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Seiler and Plamann 2003</a>). It was subsequently shown that LRG1 is essential for hyphal tip extension and that it plays a role in regulating (through RHO1) Î²1,3-glucan synthase activity. A mutation in <em>lrg-1</em> was shown to increase activity of Î²1,3-glucan synthase as evidenced by decreased sensitivity to caspofungin; it is possible that the increased cell wall thickness of characteristic of <em>amyc</em> grown under restrictive conditions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Coniordos and Turian 1973</a>) is due to increased Î²1,3-glucan synthase activity.</p></div><div id="sec-8" class="subsection"><h4>tiny:</h4><p id="p-16">Another morphological mutant, tiny (<em>ti</em>), shows temperature sensitivity and decreased colony size with increasing temperature. There are no nonsynonymous SNPs in strain 309 in the 730 kb region carrying the mutation tiny (<em>ti</em>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS5.pdf">Figure S5</a>) and there is only one unique indel among the 163 ORFs in this region. This 8 base deletion causes a frameshift mutation in the coding sequence of NCU10497 beginning at amino acid 405 out of 750 total amino acids. This frameshift mutation introduces fourteen nonsense codons, beginning at amino acid 445. NCU10497 encodes the oligosaccharyl transferase STT3 subunit. Stt3 is an essential gene in both <em>Saccharomyces cerevisiae</em> and <em>Schizosaccharomyces pombe</em>. Similar to alleles in <em>S. cereviseae</em>, the tiny allele is temperature sensitive. Because stt3 is an essential component of the N-linked glycosylation pathway, it is logical to hypothesize that this causes the decrease in cell wall protein in the <em>ti</em> mutant. A gene deletion mutant of NCU10497 was nonviable as a homokaryon (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>).</p></div><div id="sec-9" class="subsection"><h4>fissure:</h4><p id="p-17">Strain FGSC 1303 contains only one unique indel and no unique SNPs in the 1 Mb region between <em>pyr-1</em> and <em>ace-4</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS6.pdf">Figure S6</a>). The deletion is the only indel in the gene NCU04990 and it removes a single nucleotide in the coding sequence causing multiple nonsense codons. This deletion is found in all reads covering this region in this strain. NCU04990 is predicted to encode the ortholog of the <em>vhs/ran1</em> serine threonine kinase in <em>S. cereviseae</em>. The deletion of this gene in yeast leads to decreased resistance to hyperosmotic stress and accumulation of glycogen. An <em>N. crassa</em> knock-out mutant for this ORF has reduced asexual spore formation and abnormal sexual morphology (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>).</p></div><div id="sec-10" class="subsection"><h4>semicolonial-1:</h4><p id="p-18">FGSC 1363 has one unique insertion of 8 bases among the 409 ORFs in the 2 Mb region flanked by mating type and <em>rg-1</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS7.pdf">Figure S7</a>). This indel occurs in NCU02762, an ORF that contains domains associated with calcium channels, and which is an ortholog of <em>CCH1</em> from <em>Saccharomyces cerevisiae</em>. In <em>S. cerevisiae</em>, <em>CCH1</em> mutants have impaired ability to take up Ca<sup>2+</sup> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Peiter <em>et al.</em> 2005</a>) in response to Î± mating factor as well as increased sensitivity to some stress conditions. A deletion of the <em>cch1</em> ortholog from <em>Fusarium graminearum</em> leads to a growth phenotype reminiscent of <em>smco-1</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hallen and Trail 2008</a>). A <em>Neurospora</em> gene deletion mutant of NCU02762 was only recovered as a heterokaryon suggesting that a homokaryotic deletion would be lethal. All of the SNPs in this region are either shared, or occur in ORFs that have polymorphisms in multiple strains.</p></div><div id="sec-11" class="subsection"><h4>dot:</h4><p id="p-19">The mutation known as <em>dot</em> is in a 1.7 Mb region on LG IR that contains 415 ORFs. The only unique variant in this region in strain FGSC 1211 is the deletion of one T from NCU00896 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS8.pdf">Figure S8</a>), which is detected in all reads covering this sequence in FGSC 1211. As with other morphological mutants, a gene deletion mutant of this ORF was only recovered as a heterokaryon, suggesting that it may be essential for growth or sexual reproduction. This ORF encodes a phosphatidylinositol phosphate (PtdInsP) phosphatase and is the ortholog of <em>SAC1</em> from <em>S. cerevisiae</em>. Mutations in this gene have been implicated in growth regulation in yeast, as well as in hereditary disease in humans (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Manford <em>et al.</em> 2010</a>). In yeast, sac1p is localized to the ER and plays an important role in actin cytoskeleton organization, cell wall synthesis, Golgi function, lipid metabolism and vacuole morphology through regulation of phosphatidylinositol 4-phosphate levels (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rivas <em>et al.</em> 1999</a>).</p></div><div id="sec-12" class="subsection"><h4>doily:</h4><p id="p-20">The mutation <em>doily</em> (<em>do</em>) was deposited to the FGSC collection in 1972 and is characterized by colonial morphology that is sensitive to the carbon source in the culture medium. FGSC 2261 carries the only allele known of the mutation <em>doily</em> (<em>do</em>). While this strain has a relatively high number of polymorphisms (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS9.pdf">Figure S9</a>), most of the SNPs are not unique or occur in ORFs that have other polymorphisms in multiple strains. However, one unique nonsynonymous SNP (A-&gt;G at position 3,652,079) was found in the ORF designated NCU06871. This variant had the maximum possible quality score and changes a histidine to an arginine at position 1,246 of the predicted protein. NCU06871 encodes a glycoside transferase 48 family protein involved Î²-1,3 glucan synthesis. The ortholog of this gene in <em>Aspergillus fumigatus</em> has been shown to be essential. Similarly, strains of <em>Fusarium solani</em> in which RNAi has been used to decrease expression of its ortholog show defects in morphology (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beauvais <em>et al.</em> 2001</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ha <em>et al.</em> 2006</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460875/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460875/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3â</span> <span class="caption-title">Types of single nucleotide polymorphisms among 18 strains of <em>Neurospora crassa</em></span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460806/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460806/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4â</span> <span class="caption-title">Indel distribution among 18 strains of <em>Neurospora crassa</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h4>female fertile-1:</h4><p id="p-21"><em>female fertile-1</em> (<em>ff-1</em>) was originally isolated as a mutant displaying a defect in female fertility. It was later shown to be allelic with a spontaneous mutant (glycerol phosphate-1; <em>glp-1</em>) that was characterized by its ability to efficiently conidiate and grow on glycerol as a carbon source. This mutation lies in a 675 kb region of chromosome 2 that includes 172 ORFs and that has very few polymorphisms in strain FGSC 3831 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS10.pdf">Figure S10</a>). In this strain, NCU01543 has a unique insertion of 1 base that introduces a frameshift mutation causing multiple stop codons, the first of which is located at position 113 out of 762. The insertion was identified in all reads covering this region in strain FGSC 3831. An orthologous gene in <em>Magnaporthe grisea</em> has been characterized (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2010</a>); the gene, <em>LDB1</em>, is so named because it encodes a predicted <span class="underline">L</span>IM-<span class="underline">b</span>inding-<span class="underline">d</span>omain. LIM domains are involved in proteinâprotein interactions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kadrmas and Beckerle 2004</a>). The phenotypes associated with deletion of this gene in <em>M. grisea</em> are a lack of asexual and sexual spore production, easily-wettable colonies and an inability to develop appresoria.</p></div><div id="sec-14" class="subsection"><h4>female sterile-n:</h4><p id="p-22">Female sterile-n (<em>fs-n</em>) is on LG 1 far from <em>mat</em> (35%; 45%) and near T(4637) <em>al-1</em>, a region of 80 kb containing only 18 ORFs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS11.pdf">Figure S11</a>). This mutation was characterized in 1974 (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Mylyk and Threlkeld 1974</a>) and the original description of this mutant suggested that there could be two closely linked lesions responsible for the lack of female fertility in strains carrying this trait. The genome sequence of strain FGSC 3246 revealed a unique insertion of one A into NCU02794 at position 9,112,074 and this insertion was detected in all reads covering this region in this strain. While there are no other indels in strain FGSC 3246 in the 80 kb region delimited by the flanking markers there are a number of SNPs in this region, although most are shared among multiple strains. Although NCU02794, known variously as <em>ham-2</em> or <em>so</em> (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), has multiple nonsynonymous SNPs in other strains, further examination of the vegetative of morphology of FGSC 3246 indicates that is has a âsoftâ like phenotype. The protein encoded by NCU02794 contains a WW domain and is involved in hyphal fusion and was localized to septal plugs (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fleissner <em>et al.</em> 2005</a>). A knockout of this ORF is female sterile and has abnormal vegetative morphology (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>).</p></div><div id="sec-15" class="subsection"><h4>tyrosinaseless-1:</h4><p id="p-23">The <em>tyrosinaseless-1</em> (<em>ty-1</em>) mutant phenotype was initially described as female infertile, âvelvetâ (short aerial hyphae) and tyrosinaseless (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Horowitz <em>et al.</em> 1961</a>). Further analysis showed that tyrosinase was inducible under some conditions. Strain FGSC 322 carries over 1,400 indels in the 1.5 Mb region genetically shown to carry the mutation <em>tyrosinaseless-1</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS12.pdf">Figure S12</a>). Of these, 69 are in coding sequence but only three are unique to this strain. Of these three, NCU00240 exhibits frameshift inducing indels in other strains, suggesting that it is not responsible for the tyrosinaseless phenotype. NCU00403 and NCU00455 each exhibit unique frameshift inducing indels and both deletions are strongly supported in the sequence data. While gene deletion mutants are available for both, the phenotype for the gene deletion of NCU00403 was characterized by the <em>Neurospora</em> Program Project and exhibited ânormalâ growth and sexual development, including pigmentation of perithecia and ascospores (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>). Thus, the likely gene associated with the <em>ty-1</em> phenotype is NCU00455, which encodes the ortholog of <em>S. cerevisiae</em> Ste50, a scaffold protein that connects the mitogen-activated protein kinase (MAPK) cascade with cell cycle machinery. The MAP kinase signaling cascade regulates a significant number of processes in fungi including the regulation of tyrosinase (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Park <em>et al.</em> 2008</a>). There are three MAP kinase cascades in <em>Neurospora crassa</em> that overlap in function and are involved in many biological processes, including cell morphology, conidiation, mating, and osmotic stress response (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Borkovich <em>et al.</em> 2004</a>).</p></div><div id="sec-16" class="subsection"><h4>male barren-1:</h4><p id="p-24">The mutation known as <em>mb-1</em> is on LG VII, and the strain carrying this mutation, FGSC 3562, has 741 indels in coding sequences of which 71 are on super contig 7 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS13.pdf">Figure S13</a>). Four of the 71 indels are unique to strain FGSC 3562 and three introduce frameshift mutations. Two deletions occur in NCU02251 and both are deletions of one base (at 1,076,703 and 1,076,825). However, a homokaryotic gene deletion mutant of this strain is available suggesting that this ORF is not responsible for the phenotype in FGSC 3562. The second ORF carrying a unique frameshift mutation in strain FGSC 3562 is NCU06930, a hypothetical protein with only limited orthology among filamentous ascomycetes. This same ORF has multiple nonsynonymous SNPs in strain FGSC 821. Four additional annotated ORFs have non-unique indels with NCU11995 having thirteen distinct indels in strains 3562 and 821. Although strain 3562 has no unique nonsense codons on contig 7 it has 800 nonsynonymous SNPs on contig 7 of which 13 are unique to this strain. Of these, NCU11995 has multiple nonsynonymous SNPs. This ORF encodes a C6 zinc finger domain-containing protein. Additionally, NCU06102 has two unique nonsynonymous SNPs although a homokaryotic gene deletion mutant for this ORF is available. Other ORFs containing unique nonsynonymous SNPs are in ORFs that contain other nonsynonymous SNPs in other strains. Because of the ambiguity presented by these multiple polymorphisms, it is not possible to uniquely identify one as being responsible for the male barren phenotype in strain FGSC 3562.</p></div><div id="sec-17" class="subsection"><h4>male barren-2:</h4><p id="p-25">Although there are no indels or nonsynonymous SNPs in the region carrying the <em>male barren-2</em> trait in strain FGSC 3564 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS14.pdf">Figure S14</a>), there is one unique nonsense mutation in this region in strain FGSC 3564. The G to A substitution occurred in all reads covering this position and has a quality score of 111. This variant encodes a unique nonsense codon in NCU00652 that is the only unique variant in this ORF among all 18 strains. The nonsense SNP in strain FGSC 3564 occurs at amino acid 659 out of 1188. Only four other strains display any variants in this ORF and none of these SNPs are unique.</p></div><div id="sec-18" class="subsection"><h4>male barren-3:</h4><p id="p-26">The last of the male-barren strains in the current investigation is <em>mb-3</em>, which is on linkage group IR near <em>al-1</em> and <em>mb-2</em> (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). There is one unique nonsense mutation in NCU00658 in strain FGSC 3566 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S15</a>) and it occurs with high confidence. The nonsense mutation occurs at amino acid 219 out of 4008. A gene deletion mutant for this ORF has normal morphology, but is male barren.</p></div><div id="sec-19" class="subsection"><h4>tan spore:</h4><p id="p-27"><em>tan spore</em> (<em>ts</em>) is found in strain FGSC 821, which was deposited into the FGSC collection in 1961, and is described as having been a spontaneous mutation in strain 4a, the so called Emerson lineage (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Nakamura and Egashira 1961</a>). The phenotype for <em>ts</em> is light colored and/or immature ascoscores that do not germinate. The <em>ts</em> mutation was used in studies that showed that the majority of <em>N. crassa</em> perithecia originate from fertilization by a single male spore each. Strain FGSC 821 has the greatest nucleotide divergence relative to the reference genome (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S16</a>) of all the strains characterized in the present study. Nevertheless, no unique SNPs, and only one unique indel, occur in coding sequence in the 2 Mb region flanked by the markers <em>inl</em> and <em>cen-V</em> (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The unique mutation is a deletion of one base leading to frameshift in NCU01459 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) and this deletion is detected in all reads covering this region in strain FGSC 821. NCU01459 is a BZIP transcription factor whose deletion leads to ascospore inviability. The deletion of NCU01459 leads to immature tan ascospores and was noted in (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Colot <em>et al.</em> 2006</a>) who named the gene <em>ascospore lethal-2</em> (<em>asl-2</em>).</p></div><div id="sec-20" class="subsection"><h4>perithecial-1:</h4><p id="p-28">Analysis of FGSC 7035 genome sequence led to the identification of a unique 1bp deletion in the gene coding region of NCU03584 as well as two ORFs carrying unique nonsynonymous SNPs among the 1,033 ORFs in the region flanked by the markers <em>ilv-1</em> and <em>asn-1</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S17</a>). While the deletion in NCU03584 is the only variant in this ORF, both of the nonsynonymous SNPs occur in ORFs that have multiple nonsynonymous SNPs or frameshift causing indels in other strains. To validate the identification of NCU03584, and the identification of unique variants by whole genome sequencing, NCU03584 from FGSC 7035 was sequenced manually and the deletion was identified by this approach as well (data not shown). The deletion in NCU03584 occurs at nucleotide 114 resulting in a stop codon at nucleotide 162 in a 7271 nucleotide ORF. NCU03584 is the likely ortholog of the polyketide synthetase responsible for perithecial pigmentation in the related fungus, <em>Sordaria macrospora</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Engh <em>et al.</em> 2007</a>).</p></div></div><div id="sec-21" class="subsection"><h3>Summary of results for combined strain set</h3><p id="p-29">Eighteen genomes were sequenced to an average depth ranging from 28 to nearly 108- fold coverage (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Sixteen strains were sequenced entirely with paired-end reads producing between 27 and 90 million paired end reads resulting in an average sequence depth ranging from 29X to 108X. Two strains, FGSC 3566 and FGSC 3831 had over 20 million single-end reads in addition to over sixty-nine million paired-end reads. While strain FGSC 7035 generated over 90 million reads, many of the reads were of low quality and only 52% were mapped to the genome. Thirteen strains were sequenced with an average read length of 35 bases while five strains were sequenced with an average read length of approximately 75 bases. There were no significant differences in the ability to map these reads to the genome for either approach and each genome was compared to the over 41 million bases of the reference genome. The portion of the genome that was aligned and compared to the reference genome ranged from 99% of the reference genome size in the most conserved strain to 81% for the most diverged strain. Since multi-alleleic calls are not expected in a haploid genome, 240 coding multi-alleleic sites and 1100 non-coding multi-alleleic sites were examined to determine what might have caused these calls. Out of 240 coding sites examined, 53% were caused by indels that were too large for maq to properly align the reads, 36% were in reads that had low mapping quality scores and/or were orphan pairs, suggesting that the read could have been misaligned, 9% appeared truly homozygous, and 2% could not be explained. For non-coding sites 4% could be explained by missed indels, 93% were in reads that had low mapping quality scores and/or were orphan pairs, suggesting that the read could have been misaligned, and 3% appeared truly homozygous. Preprocessed data from this project are available via GenBank (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), or at <a href="http://genome.jgi-psf.org/Neucr1/Neucr1.home.html">http://genome.jgi-psf.org/Neucr1/Neucr1.home.html</a> and tables with the processed data are available as supplemental data.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460839/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460839/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5â</span> <span class="caption-title">Sequencing statistics</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Because most of the target mutations were flanked by genetic markers that are already identified to the level of the gene sequence, the amount of the genome that require examination for each strain ranged from a low of 74 kilobases (kb) in strain FGSC 305 to over 4 megabases (Mb) in strain FGSC 3562 (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). The average was just over 1.7 Mb, although for seven strains the amount of genome space that needed to be searched was less than 1 Mb. The number of annotated open reading frames provides another measure of the amount of genome space that must be searched to identify the mutation responsible for the relevant phenotype. For the strains in the current project this number ranged from a low of 18 ORFs in the 80 kb between flanking markers in strain FGSC 3246 to 1,433 ORFs in the 5.8 Mb to the right of <em>cen-1</em> in strain FGSC 3566 (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-32">In the 18 genomes, there are 1,184,610 identified single nucleotide variants (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Of these, 1,137,606 (96.03%) map to contigs 1-7 (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). Among these, there are 527,620 distinct positions with identified variants in all 18 strains. 244,043 of these variants occur only in one strain. 96 variants are identified as occurring in all 18 genomes, suggesting that the actual polymorphism belongs to the reference genome (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). The distribution of variants among the eighteen strains varies from a high of over 188,346 to a low of 13,274 (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), while the number of unique SNPs ranged from a high of 73,336 to a low of 704 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>). Overall, there were nearly three times more transition mutations than transversions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/TableS1">Table S1</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FileS1.zip">File S1</a>). The distribution of SNPs in the 100 kb region directly flanking each mutation was calculated, separating shared and unique SNPs, and there was no apparent association between either whether the mutation was spontaneous or induced, or which mutagen was employed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/TableS2">Table S2</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FileS1.zip">File S1</a>), suggesting that strain lineage and back-crossing are more important than the mutagen in influencing variant distribution. Remarkably, there are only 413 ORFs, out of nearly 10,000, that have no polymorphisms among the 18 strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/TableS3">Table S3</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FileS1.zip">File S1</a>). Among these are 274 conserved hypothetical proteins, hypothetical, or predicted proteins, but also several ribosomal protein genes, histone genes, a mating type gene, and the <em>frequency</em> gene that is central to the cell's circadian rhythm.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460821/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460821/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6â</span> <span class="caption-title">Distribution of SNPs among the supercontigs</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460810/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460810/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7â</span> <span class="caption-title">Frequency of individual SNPs among eighteen strains of <em>Neurospora crassa</em></span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/303/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Unique single nucleotide polymorphisms (SNPs) in 18 strains of Neurospora crassa. The number of unique SNPs (Table 8) was plotted for each strain and strains were sorted based on the number of unique SNPs. For this analysis, SNPs that occurred more than once in the dataset were not included." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133295909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Unique single nucleotide polymorphisms (SNPs) in 18 strains of &lt;em&gt;Neurospora crassa&lt;/em&gt;. The number of unique SNPs (Table 8) was plotted for each strain and strains were sorted based on the number of unique SNPs. For this analysis, SNPs that occurred more than once in the dataset were not included.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/303/F2.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/303/F2.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/303/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/303/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460848" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-33" class="first-child">Unique single nucleotide polymorphisms (SNPs) in 18 strains of <em>Neurospora crassa</em>. The number of unique SNPs (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>) was plotted for each strain and strains were sorted based on the number of unique SNPs. For this analysis, SNPs that occurred more than once in the dataset were not included.</p><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460859/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460859/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8â</span> <span class="caption-title">Number of unique SNPs in each strain</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">There are 62,952 distinct indels (compared to the reference genome) present in the 18 strains analyzed (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a> and <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table 9</a>). Of these indels, 25,894 occur in only one strain (<a id="xref-table-wrap-9-2" class="xref-table" href="#T9">Table 9</a>). Because many indels are present in multiple strains, the total number of indels mapped to the genomes is 146,762. Strains exhibit different numbers of indels (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>) ranging from a low of 1,494 in strain FGSC 1211 to a high of 24,952 in strain FGSC 1363. Thirteen strains have fewer than 10,000 indels and nine strains have fewer than 5,000 indels. Most strains have ten times more non coding indels than coding indels (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). Indels affecting the splice junction are a small fraction of all indels, ranging from 1 to 3% of coding sequence indels in each strain. Of the distinct indels, 16,682 are repeated twice and 8,818 are repeated three times (<a id="xref-table-wrap-9-3" class="xref-table" href="#T9">Table 9</a>). Relatively few distinct coding sequence indels are found more than ten times and there are more distinct indels that are present in all 18 strains than are in ten to seventeen strains combined, suggesting that the indels that are found in all 18 strains actually represent indels present in the reference genome relative to a putative consensus genome. 1,352 annotated ORFs have one indel and 495 contain two indels (<a id="xref-table-wrap-10-1" class="xref-table" href="#T10">Table 10</a>). 198 ORFs have three indels and 96 ORFs have four indels. Fewer than 100 ORFs in total have more than five indels.</p><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460832/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460832/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 9â</span> <span class="caption-title">Indels repeated among multiple strains</span><div class="sb-div caption-clear"></div></div></div><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460840/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460840/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 10â</span> <span class="caption-title">Number of indels per ORF among eighteen strains of <em>Neurospora crassa</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-35">The distribution of indel sizes was found to be biased, both with regard to size and distribution in the genome. Indels of size +4 or â4 nucleotides were more common than would be predicted (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) and are dominated indels of a few specific sequences. Though neither is overrepresented in the genome, TACC and TAGG, often tandemly repeated, comprise 30% of the tetrameric indels. Indels that are +/â 3 or are a multiple of 3 nucleotides are up to ten times more common among coding sequence than in non-coding regions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Over 24% of all indels of three nucleotides occur in protein coding sequence, whereas less than two percent of indels of two or four nucleotides occur in protein coding sequence. The size of indel that is identifiable by the maq software is dependent upon the length of the read, with insertions being harder to identify. To determine the effect of read length on the ability to identify small indels, we altered the reference of supercontig 1 at evenly spaced intervals to simulate indels, aligned un-altered data to the altered reference, and then determined if maq was able to correctly identify the simulated indels. In general insertions are harder to identify than deletions, and longer reads allow more indels to be identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS18.pdf">Figure S18</a>). BreakDancer (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2009</a>) was used to identify larger events. It is known that this program will report false positive calls, especially for genomes that are repetitive, thus validation of predicted structural variation was necessary, as for FGSC 3921, above, although <em>Neurospora</em> has relatively low levels of repeated sequence (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Galagan <em>et al.</em> 2003</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/303/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of unique indel sizes among eighteen strains of Neurospora crassa. The number of indels (Y axis) is plotted vs. the size of the indels (X axis). Insertions and deletions are pooled to give a single value for each size (Table S4 in File S1). Indels of size four are strongly overrepresented. Indels that are a multiple of three are also overrepresented. While indels may occur in multiple strains, each indel was only counted once to generate this dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133295909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of unique indel sizes among eighteen strains of &lt;em&gt;Neurospora crassa&lt;/em&gt;. The number of indels (Y axis) is plotted &lt;em&gt;vs.&lt;/em&gt; the size of the indels (X axis). Insertions and deletions are pooled to give a single value for each size (Table S4 in File S1). Indels of size four are strongly overrepresented. Indels that are a multiple of three are also overrepresented. While indels may occur in multiple strains, each indel was only counted once to generate this dataset.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/303/F3.medium.gif" width="440" height="275"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/303/F3.medium.gif" width="440" height="275"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/303/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/303/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460801" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3â</span> <p id="p-36" class="first-child">Distribution of unique indel sizes among eighteen strains of <em>Neurospora crassa</em>. The number of indels (Y axis) is plotted <em>vs.</em> the size of the indels (X axis). Insertions and deletions are pooled to give a single value for each size (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/TableS4">Table S4</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FileS1.zip">File S1</a>). Indels of size four are strongly overrepresented. Indels that are a multiple of three are also overrepresented. While indels may occur in multiple strains, each indel was only counted once to generate this dataset.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/303/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Indel size frequency in coding sequence among eighteen strains of Neurospora crassa. The total number of indels of a given size that occur in coding sequence was compared to the total number of indels of the same size and the resulting fraction was multiplied by 100 (Table S5 in File S1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133295909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Indel size frequency in coding sequence among eighteen strains of &lt;em&gt;Neurospora crassa&lt;/em&gt;. The total number of indels of a given size that occur in coding sequence was compared to the total number of indels of the same size and the resulting fraction was multiplied by 100 (Table S5 in File S1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/303/F4.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/303/F4.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/303/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/303/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460852" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4â</span> <p id="p-37" class="first-child">Indel size frequency in coding sequence among eighteen strains of <em>Neurospora crassa</em>. The total number of indels of a given size that occur in coding sequence was compared to the total number of indels of the same size and the resulting fraction was multiplied by 100 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/TableS5">Table S5</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FileS1.zip">File S1</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Finally, the distribution of all polymorphisms is not uniform and demonstrates that different regions of each chromosome are inherited as blocks, or haplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS6.pdf">Figure S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS9.pdf">Figure S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS11.pdf">Figure S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS12.pdf">Figure S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS13.pdf">Figure S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS14.pdf">Figure S14</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S15</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S16</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S17</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS19.pdf">Figure S19</a>). It is suggested that such a block occurs around the mating type locus (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Pomraning <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wik <em>et al.</em> 2008</a>), which is on supercontig 1 at approximately 1,850,000. While most strains have either uniformly low or uniformly high divergence from the reference genome (<em>e.g.</em>, strains 106 or 821 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S16</a>, respectively) in this region, strains that show abrupt changes from high divergence from the reference genome to low divergence from the reference genome in this region delimit a sub-region from position 1,840,000 on the left (Strain 7035) to position 2,280,000 on the right (in multiple strains), which has no shifts in the level of divergence from the reference genome (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This sub-region contains 116 annotated ORFs (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Colot <em>et al.</em> 2006</a>) and includes 440 kilobases of DNA sequence. Immediately adjacent to the right end of this sub-region (from 2,280,000 to 2,410,000) is another sub-region with a high number of unique SNPs (in 15 strains). This sub-region contains 1,677 unique SNPs and 2,721 shared SNPs. Of these, 4,015 SNPs are non-coding, 123 are synonymous, and only 80 are nonsynonymous. The density of unique SNPs in this region is 13 per kb, as compared to the overall unique SNP density, which ranges from 0.02 per kb (in strain FGSC 309) to 1.8 per kb (in strain 821). The <em>mat a</em> region is defined by high a region of high SNPs from 1,320,000 to 1,940,000 in strain FGSC 3246 (the only <em>mat a</em> strain, shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/303/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Putative mating type haplotype region. (A) FGSC 3246, (B) FGSC 7035, (C) FGSC 3562, (D) FGSC 3566. Distribution of SNPs and indels on the left arm of supercontig 1 in representative strains. The region from the left telomere through 2.5 million bases is shown. Total SNPs are plotted in blue. SNPs that are unique to each strain are plotted in red. Indels are plotted in green. The total number in a 10 Kb moving window is plotted on the Y axis. The X axis corresponds to the position along the contig. Strain 3246 is mat a while strains 7035, 3562, and 3566 are mat A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133295909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Putative mating type haplotype region. (A) FGSC 3246, (B) FGSC 7035, (C) FGSC 3562, (D) FGSC 3566. Distribution of SNPs and indels on the left arm of supercontig 1 in representative strains. The region from the left telomere through 2.5 million bases is shown. Total SNPs are plotted in blue. SNPs that are unique to each strain are plotted in red. Indels are plotted in green. The total number in a 10 Kb moving window is plotted on the Y axis. The X axis corresponds to the position along the contig. Strain 3246 is &lt;em&gt;mat a&lt;/em&gt; while strains 7035, 3562, and 3566 are &lt;em&gt;mat A&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/303/F5.medium.gif" width="420" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/303/F5.medium.gif" width="420" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/303/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/303/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460844" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5â</span> <p id="p-39" class="first-child">Putative mating type haplotype region. (A) FGSC 3246, (B) FGSC 7035, (C) FGSC 3562, (D) FGSC 3566. Distribution of SNPs and indels on the left arm of supercontig 1 in representative strains. The region from the left telomere through 2.5 million bases is shown. Total SNPs are plotted in blue. SNPs that are unique to each strain are plotted in red. Indels are plotted in green. The total number in a 10 Kb moving window is plotted on the Y axis. The X axis corresponds to the position along the contig. Strain 3246 is <em>mat a</em> while strains 7035, 3562, and 3566 are <em>mat A</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Nonsense mutations</h3><p id="p-40">Four hundred and five distinct nonsense mutations were detected in the current data set for a total of 884 nonsense codons. The number of nonsense mutations varies among strains from a low of 11 in strain FGSC 309 to a high of 137 in strain FGSC 1363 (<a id="xref-table-wrap-11-1" class="xref-table" href="#T11">Table 11</a>). Of these 405 different nonsense mutations, 188 occur once each. Nonsense SNPs occur in 337 different NCUs of which 175 occur in ORFs that have been deleted by the functional genomics program. While many of the ORFs carrying nonsense mutations are hypothetical or putative genes, fifteen of the genes carrying nonsense mutations have annotations in the <em>Neurospora</em> gene compendium, including genes such as <em>sad-2</em> (NCU04294; (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Shiu <em>et al.</em> 2006</a>)), or <em>so</em> (<em>ham-1</em>, NCU02794; (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fleissner <em>et al.</em> 2005</a>)). Only four strains exhibit any variants in <em>sad-2</em>, including two strains, FGSC 821 and FGSC 3921, which feature a nonsense mutation that causes a stop codon at amino acid position 208 out of 1098. This same ORF has no indels in any strains analyzed in the current work.</p><div id="T11" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460843/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460843/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 11â</span> <span class="caption-title">Number of nonsense mutations among eighteen strains of <em>Neurospora crassa</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-41">Four strains carry nonsense mutations at amino acid position 21 in NCU09544 (<em>pod-2</em>; (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Seiler and Plamann 2003</a>)). Two of these, strains, FGSC 305 and FGSC 1363, have morphological phenotypes, while a third putatively carries a mutation in a regulatory protein, and the last is a male barren mutant. While there are multiple alleles of NCU09544 present among these four strains, all alleles carry the same nonsense mutation and vary in the number SNPs and the sizes of indels present. NCU09544 has been deleted by the <em>Neurospora</em> Functional Genomics program and the resulting mutant (FGSC 12737) has no phenotype (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>) suggesting that the temperature sensitive <em>pod-2</em> allele previously isolated (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Seiler and Plamann 2003</a>) is actually a defective interfering mutation (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fujimura <em>et al.</em> 1993</a>).</p><p id="p-42">Multiple nonsense mutations in single ORFs led us to identify two genes that appear to have been RIPed (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Freitag <em>et al.</em> 2002</a>), NCU09968 and NCU09969. Neither gene is well conserved across sequenced fungi, although there does appear to be an expansion of putatively NCU09969 related genes in <em>Chaetomium globosum</em>. These two ORFs are adjacent on supercontig 6 and the co-occurrence of polymorphisms in around these two ORFs suggest that they are co-inherited. Neither have been deleted by the functional genomics program. Of the 405 nonsense mutations, 313 (77%) occur in ORFs annotated as conserved hypothetical proteins, 21 as âhypothetical proteinâ and 24 as âpredicted protein.â There are 9,734 total ORFs in the dataset, and 6,481 (48%) are conserved hypothetical proteins. Comparing the ratio of nonsense mutations among conserved hypothetical proteins to the ratio of conserved hypothetical proteins among all proteins shows that conserved hypothetical proteins are overrepresented among the ORFs carrying nonsense mutations by nearly 30%.</p></div><div id="sec-23" class="subsection"><h3>Introgressed region in strain FGSC 3114</h3><p id="p-43">The genome sequence of the strain carrying the introgressed meiotic drive element, Sk-2, has relatively low divergence from the reference genome in terms of SNPs (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>) and indels (<a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>). The variants are predominantly seen on supercontig 3, consistent with the genetic identification of the meiotic drive element (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS19.pdf">Figure S19</a>). The genome sequence data clearly shows that the introgressed region does not include the left telomere on contig 3, but does span the centromere (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The number of SNPs on supercontig 3 ranges from a low of 792 in strain FGSC 3246 to a high of 28,243 in strain 322 (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>). This variability is not evenly distributed with strains FGSC 322 and FGSC 1363 sharing most SNPs while other strains, such as FGSC 106 and FGSC 305 share SNPs only on the right end. While there are large numbers of SNPs and indels in the introgressed region, in the absence of a complete sequence for <em>N. intermedia</em>, their identity or distribution do not provide significant insight into the nature of the Sk-2 element.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/303/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of polymorphisms on supercontig 3. (A) Strain FGSC 3114 containing the introgressed Sk-2 region. (B) Strain FGSC 106 with a vertical arrow showing the location of com. (C) Strain FGSC 305. (D) Strain FGSC 1363. Total SNPs are plotted in blue. Unique SNPs are plotted in red. Indels are plotted in green. Polymorphisms were sorted by supercontig and position and the total number in a 10 Kb moving window is plotted on the Y axis. The X axis corresponds to the position along the supercontig (X 10 Kb)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1133295909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of polymorphisms on supercontig 3. (A) Strain FGSC 3114 containing the introgressed Sk-2 region. (B) Strain FGSC 106 with a vertical arrow showing the location of &lt;em&gt;com&lt;/em&gt;. (C) Strain FGSC 305. (D) Strain FGSC 1363. Total SNPs are plotted in blue. Unique SNPs are plotted in red. Indels are plotted in green. Polymorphisms were sorted by supercontig and position and the total number in a 10 Kb moving window is plotted on the Y axis. The X axis corresponds to the position along the supercontig (X 10 Kb).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/303/F6.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/303/F6.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/303/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/303/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460851" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6â</span> <p id="p-44" class="first-child">Distribution of polymorphisms on supercontig 3. (A) Strain FGSC 3114 containing the introgressed Sk-2 region. (B) Strain FGSC 106 with a vertical arrow showing the location of <em>com</em>. (C) Strain FGSC 305. (D) Strain FGSC 1363. Total SNPs are plotted in blue. Unique SNPs are plotted in red. Indels are plotted in green. Polymorphisms were sorted by supercontig and position and the total number in a 10 Kb moving window is plotted on the Y axis. The X axis corresponds to the position along the supercontig (X 10 Kb).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h3>Spontaneous mutation rate</h3><p id="p-45">The ability to sample genome sequence of strains that have been highly back-crossed provides an estimate of the upper limit of the rate of spontaneous mutations in sexual reproduction (Âµ) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baer <em>et al.</em> 2007</a>). To generate this estimate, the number of SNPs unique to each strain was calculated (<a id="xref-table-wrap-8-3" class="xref-table" href="#T8">Table 8</a>). The distribution of unique SNPs among strains follows a generally Poisson distribution (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Strains 106 and 309 were deposited into the FGSC collection in 1960 and had been backcrossed into the reference genome three times. These strains have the lowest number of unique SNPs and hence provide the best estimate for the maximum value of the background mutation rate in the organism. The total sequenced genome size in the current dataset is 41,061,603, and strain FGSC 309 has 704 unique SNPs, which translates to a mutation rate of 1.71 Ã 10<sup>â5</sup>, whereas strain FGSC 106 has 1,033 unique SNPs, which translates to a mutation rate of 2.5 Ã 10<sup>â5</sup>. These values are two orders of magnitude lower than the rate suggested in (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Drake <em>et al.</em> 1998</a>) who reported a rate of 0.003 for <em>N. crassa</em>. Adding in the unique indels does not significantly change this calculation. Strain FGSC 309 has 123 unique indels and adding these to the 704 unique SNPs gives an estimate for the maximum value of Âµ = 2 Ã 10<sup>â5</sup>. While some of the difference in value could be explained by false base calls in our data set, it is doubtful that the false call rate would account for the magnitude of this difference.</p></div></div><div class="section discussion" id="sec-25"><h2 class="">Discussion</h2><p id="p-46">Our sequencing and analysis has led to the putative identification of sixteen of the seventeen mutations in the strains sequenced, and delimited an introgressed region in the eighteenth strain. Because the sequence of each strain can be evaluated in the context of the sequence of eighteen other strains, the ability to evaluate causative sequence polymorphisms was markedly enhanced. Moreover, since each mutant strain has a defined phenotype that segregates in a limited region of the genome, any sequence polymorphisms that are outside that defined region could be excluded from the specific analysis to identify the lesion responsible for the phenotype. Within the genetically defined region, each sequence polymorphism was evaluated relative to the eighteen other sequences. If a polymorphism occurred in a strain that did not display the mutant phenotype, it was presumed that the sequence polymorphism is not responsible for the phenotype. This procedure constituted, is in one manner of speaking, an <em>in silico</em> bulk analysis. The ability to recover the same mutation by manual sequencing, as in FGSC 7035 or by PCR in FGSC 3921 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), provided technical validation of the whole genome sequencing approach.</p><p id="p-47">Additionally, these evaluations of putative mutations were strengthened by data from the <em>Neurospora</em> Functional Genomics program (<a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>), which has generated knockout strains for 7,669 genes (at the time of writing). Of these 7,699, 1,271 are only available as heterokaryons. These heterokaryons were not able to be purified by crossing to a wild type strains, suggesting that the deleted ORFs are essential, or at least essential for sexual reproduction. The availability of these mutants as viable homokaryons or as heterokaryons facilitates evaluation of whether the polymorphisms identified by whole genome sequencing would likely be responsible for the phenotypes seen in the mutants sequenced in the current project. Five mutations were associated with knock-out strains exhibiting the same phenotype and seven mutations exhibit similar phenotypes to the phenotypes of classical mutants in the same gene in <em>Neurospora</em>, or in related organisms. While complementing each mutation is outside the scope of the current study, the ability to associate an otherwise anonymous sequence variant with characterized mutations in <em>Neurospora</em> or related organisms validates the approach taken in the current project.</p><p id="p-48">The distribution of sequence divergence among these eighteen strains is highly related to the strain histories. Strains that are closely related to the reference genome strain have the lowest sequence divergence relative to the reference genome, while those that were generated in a different background and not crossed into the reference genome background have the highest sequence divergence. For example, strain FGSC 821 is in the Emerson lineage and this strain has the highest sequence divergence from the reference of all the strains. Strains FGSC 106 and FGSC 309 were both deposited into the FGSC collection in 1960, but both bore mutations that had been induced in the reference genome background, and both were backcrossed three times into the reference genome background. Both of these strains have very low sequence divergence from the reference. However, the expectation that the mutations would be found in small regions of relatively high divergence was not supported. Instead, each strain has small regions of high divergence that have the appearance of blocks of DNA that has been inherited together, or haplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS6.pdf">Figure S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS9.pdf">Figure S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS10.pdf">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS11.pdf">Figure S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS12.pdf">Figure S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS13.pdf">Figure S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS14.pdf">Figure S14</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S15</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S16</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1/FigureS17.pdf">Figure S17</a>). The characterization of a region surrounding the mating type locus provides a starting point for additional analysis of haplotype distribution among these strains.</p><p id="p-49">The distribution of indels, and especially those whose size is a multiple of three, was highly biased in coding regions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This distribution is similar to the distribution seen in the human genome (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Messer and Arndt 2007</a>). Although the present analysis does not include a characterization of the nature of indels, beyond their size and location, the abundance of tetrameric indels was unexpected.</p><p id="p-50">Nonsense mutations were identified in a surprisingly large number of exons, which, when considered in the context of the large number of numerous nonsynonymous SNPs, emphasizes the observation that most genes can be deleted with no visible phenotype (<a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">Dunlap <em>et al.</em> 2007</a>). The number of nonsense mutations and frameshift inducing insertions and deletions suggest that most of the classical mutant strains carry significant cryptic second-site mutations. Thus careful evaluation of historical functional analysis of classical mutations is warranted. Furthermore, the value of backcrossing a strain into a well characterized genetic background is strongly validated.</p><p id="p-51">Our analysis of genomic polymorphisms among eighteen strains of <em>Neurospora crassa</em> has led to identification of the nature of mutations associated with historically described phenotypes. It has also associated some of these classical mutants with other strains that have been characterized both in terms of phenotype and genotype. The ability to evaluate these polymorphisms in the context of classical genetic analysis as well as in the context of a functional genomics program has leveraged the whole genome analysis. The value of the <em>in silico</em> bulk sequence analysis is demonstrated by the facility with which polymorphisms were associated with phenotypes. The added value of the whole genome resequencing has included a characterization of SNP distribution, the identification of unexpected nonsense mutations, and has revealed the strong bias in the size of insertions and deletions relative to their location in coding <em>vs.</em> non-coding sequence. While genome resequencing is becoming more accessible, the ability to find meaningful information in this data, especially when combined with both classical and modern genetic data, is demonstrated by the present approach.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">The authors would like to thank Michael Freitag, Kristina Smith, Kyle Pomraning, N. Louise Glass, and Gerry Wyckoff for helpful comments. The FGSC is supported by U.S. National Science Foundation Grant no. 742713. The DOE Office of the Biomass Program provided support for genome analysis by S.E.B. The work conducted by the U.S. Department of Energy Joint Genome Institute is supported by the Office of Science of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Sequence data from this article have been deposited with the Genbank Data Library under the nos. SRP004312, SRP004308, SRP004304, SRP003564, SRP002767, SRP002761, SRP002757, SRP004305, SRP004307, SRP002747, SRP002750, SRP002749, SRP002748, SRP004280, SRP004330, SRP002768, SRP003536, and SRP002701.</p></li><li class="fn" id="fn-2"><p id="p-2">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1SI">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000307/-/DC1SI</a>.</p></li><li class="fn" id="fn-5"><p id="p-53"><em>Communicating editor: Brenda J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-06-02"><span class="received-label">Received </span>June 2, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-08-03"><span class="accepted-label">Accepted </span>August 3, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2011 McCluskey <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.4.303.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukai</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span>â<span class="cit-article-title">Survival and mutation in Neurospora exposed at nuclear detonations.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">295</span>â<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D88%26rft.spage%253D295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.2" data-doi="10.1038/nrg2158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baer</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyamoto</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Mutation rate variation in multicellular eukaryotes: causes and consequences.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">619</span>â<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBaer%26rft.auinit1%253DC.%2BF.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D619%26rft.epage%253D631%26rft.atitle%253DMutation%2Brate%2Bvariation%2Bin%2Bmulticellular%2Beukaryotes%253A%2Bcauses%2Band%2Bconsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2158%26rft_id%253Dinfo%253Apmid%252F17637734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637734&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248170700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.3" data-doi="10.1073/pnas.27.11.499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beadle</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatum</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1941</span>â<span class="cit-article-title">Genetic Control of Biochemical Reactions in Neurospora.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">499</span>â<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBeadle%26rft.auinit1%253DG.%2BW.%26rft.volume%253D27%26rft.issue%253D11%26rft.spage%253D499%26rft.epage%253D506%26rft.atitle%253DGenetic%2BControl%2Bof%2BBiochemical%2BReactions%2Bin%2BNeurospora%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.27.11.499%26rft_id%253Dinfo%253Apmid%252F16588492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIyNy8xMS80OTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.4" data-doi="10.1128/JB.183.7.2273-2279.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beauvais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruneau</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mol</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buitrago</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legrand</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Glucan synthase complex of Aspergillus fumigatus.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">2273</span>â<span class="cit-lpage">2279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DBeauvais%26rft.auinit1%253DA.%26rft.volume%253D183%26rft.issue%253D7%26rft.spage%253D2273%26rft.epage%253D2279%26rft.atitle%253DGlucan%2BSynthase%2BComplex%2Bof%2BAspergillus%2Bfumigatus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.183.7.2273-2279.2001%26rft_id%253Dinfo%253Apmid%252F11244067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4My83LzIyNzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.5" data-doi="10.1128/MMBR.68.1.1-108.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alex</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yarden</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Lessons from the genome sequence of Neurospora crassa: tracing the path from genomic blueprint to multicellular organism.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DBorkovich%26rft.auinit1%253DK.%2BA.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D108%26rft.atitle%253DLessons%2Bfrom%2Bthe%2BGenome%2BSequence%2Bof%2BNeurospora%2Bcrassa%253A%2BTracing%2Bthe%2BPath%2Bfrom%2BGenomic%2BBlueprint%2Bto%2BMulticellular%2BOrganism%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.68.1.1-108.2004%26rft_id%253Dinfo%253Apmid%252F15007097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo2OiI2OC8xLzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.6" data-doi="10.1128/EC.5.3.587-600.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piwowar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al Dabbous</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vierula</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Free</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Mutational analysis of the glycosylphosphatidylinositol (GPI) anchor pathway demonstrates that GPI-anchored proteins are required for cell wall biogenesis and normal hyphal growth in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">587</span>â<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DBowman%26rft.auinit1%253DS.%2BM.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D587%26rft.epage%253D600%26rft.atitle%253DMutational%2BAnalysis%2Bof%2Bthe%2BGlycosylphosphatidylinositol%2B%2528GPI%2529%2BAnchor%2BPathway%2BDemonstrates%2Bthat%2BGPI-Anchored%2BProteins%2BAre%2BRequired%2Bfor%2BCell%2BWall%2BBiogenesis%2Band%2BNormal%2BHyphal%2BGrowth%2Bin%2BNeurospora%2Bcrassa%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.5.3.587-600.2006%26rft_id%253Dinfo%253Apmid%252F16524913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI1LzMvNTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.7" data-doi="10.1038/nmeth.1363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallis</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalicki</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">BreakDancer: an algorithm for high-resolution mapping of genomic structural variation.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">677</span>â<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DChen%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D677%26rft.epage%253D681%26rft.atitle%253DBreakDancer%253A%2Ban%2Balgorithm%2Bfor%2Bhigh-resolution%2Bmapping%2Bof%2Bgenomic%2Bstructural%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1363%26rft_id%253Dinfo%253Apmid%252F19668202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668202&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.8" data-doi="10.1073/pnas.0601456103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colot</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringelberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crew</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A high-throughput gene knockout procedure for Neurospora reveals functions for multiple transcription factors.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">10352</span>â<span class="cit-lpage">10357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0601456103%26rft_id%253Dinfo%253Apmid%252F16801547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzI3LzEwMzUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coniordos</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turian</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span>â<span class="cit-article-title">Studies on the conidial differentiation of âNeurospora crassaâ. IV. Chemo-structural modifications of the wall in the wild type and two aconidial mutants.</span> <abbr class="cit-jnl-abbrev">Ann. Microbiol. (Paris)</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">5</span>â<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnales%2Bde%2Bmicrobiologie%26rft.stitle%253DAnn%2BMicrobiol%2B%2528Paris%2529%26rft.aulast%253DConiordos%26rft.auinit1%253DN.%26rft.volume%253D124%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D28%26rft.atitle%253D%255BStudies%2Bon%2Bthe%2Bconidial%2Bdifferentiation%2Bof%2B%2526quot%253BNeurospora%2Bcrassa%2526quot%253B.%2BIV.%2BChemo-structural%2Bmodifications%2Bof%2Bthe%2Bwall%2Bin%2Bthe%2Bwild%2Btype%2Band%2Btwo%2Baconidial%2Bmutants%255D%26rft_id%253Dinfo%253Apmid%252F4269228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4269228&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.10" data-doi="10.1038/nrg797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Timeline: Neurospora: a model of model microbes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">397</span>â<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavis%26rft.auinit1%253DR.%2BH.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D397%26rft.epage%253D403%26rft.atitle%253DTimeline%253A%2BNeurospora%253A%2Ba%2Bmodel%2Bof%2Bmodel%2Bmicrobes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg797%26rft_id%253Dinfo%253Apmid%252F11988765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11988765&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175350000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Serres</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webber</span>  <span class="cit-name-given-names">B. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Quantitative and qualitative comparisons of spontaneous and radiation-induced specific-locus mutation in the ad-3 region of heterokaryon 12 of Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">375</span>: <span class="cit-fpage">37</span>â<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253Dde%2BSerres%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D375%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D52%26rft.atitle%253DQuantitative%2Band%2Bqualitative%2Bcomparisons%2Bof%2Bspontaneous%2Band%2Bradiation-induced%2Bspecific-locus%2Bmutation%2Bin%2Bthe%2Bad-3%2Bregion%2Bof%2Bheterokaryon%2B12%2Bof%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Apmid%252F9129678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9129678&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.12" data-doi="10.1371/journal.pone.0010703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dieterle</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiest</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plamann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Characterization of the temperature-sensitive mutations un-7 and png-1 in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e10703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDieterle%26rft.auinit1%253DM.%2BG.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De10703%26rft.epage%253De10703%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Btemperature-sensitive%2Bmutations%2Bun-7%2Band%2Bpng-1%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010703%26rft_id%253Dinfo%253Apmid%252F20502699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20502699&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Rates of spontaneous mutation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1667</span>â<span class="cit-lpage">1686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D148%26rft.spage%253D1667%26rft_id%253Dinfo%253Apmid%252F9560386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9560386&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073187000030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.14" data-doi="10.1016/S0065-2660(06)57002-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunlap</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Enabling a community to dissect an organism: overview of the Neurospora functional genomics project.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">49</span>â<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DDunlap%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D57%26rft.spage%253D49%26rft.epage%253D96%26rft.atitle%253DEnabling%2Ba%2Bcommunity%2Bto%2Bdissect%2Ban%2Borganism%253A%2Boverview%2Bof%2Bthe%2BNeurospora%2Bfunctional%2Bgenomics%2Bproject.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252806%252957002-6%26rft_id%253Dinfo%253Apmid%252F17352902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(06)57002-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17352902&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.15" data-doi="10.1111/j.1574-6968.2007.00867.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuck</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Regulation of melanin biosynthesis via the dihydroxynaphthalene pathway is dependent on sexual development in the ascomycete Sordaria macrospora.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">62</span>â<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2007.00867.x%26rft_id%253Dinfo%253Apmid%252F17681008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2007.00867.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17681008&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.16" data-doi="10.1128/EC.4.5.920-930.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleissner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roca</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The so locus is required for vegetative cell fusion and postfertilization events in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">920</span>â<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DFleissner%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D920%26rft.epage%253D930%26rft.atitle%253DThe%2Bso%2BLocus%2BIs%2BRequired%2Bfor%2BVegetative%2BCell%2BFusion%2Band%2BPostfertilization%2BEvents%2Bin%2BNeurospora%2Bcrassa%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.5.920-930.2005%26rft_id%253Dinfo%253Apmid%252F15879526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI0LzUvOTIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.17" data-doi="10.1073/pnas.132212899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kothe</span>  <span class="cit-name-given-names">G. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">A cytosine methyltransferase homologue is essential for repeat-induced point mutation in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">8802</span>â<span class="cit-lpage">8807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.132212899%26rft_id%253Dinfo%253Apmid%252F12072568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTMvODgwMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8zMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujimura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh-e</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">A dominant interfering mutation in RAS1 of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">241</span>: <span class="cit-fpage">280</span>â<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DFujimura%26rft.auinit1%253DK.%26rft.volume%253D241%26rft.issue%253D3-4%26rft.spage%253D280%26rft.epage%253D286%26rft.atitle%253DA%2Bdominant%2Binterfering%2Bmutation%2Bin%2BRAS1%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F8246881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8246881&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.19" data-doi="10.1038/nature01554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galagan</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calvo</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">The genome sequence of the filamentous fungus Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">859</span>â<span class="cit-lpage">868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGalagan%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D422%26rft.issue%253D6934%26rft.spage%253D859%26rft.epage%253D868%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bfilamentous%2Bfungus%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01554%26rft_id%253Dinfo%253Apmid%252F12712197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12712197&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182432600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.4.303.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garnjobst</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatum</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span>â<span class="cit-article-title">A survey of new morphological mutants in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">579</span>â<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D57%26rft.spage%253D579%26rft_id%253Dinfo%253Apmid%252F5583730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=5583730&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.21" data-doi="10.1128/EC.00030-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covert</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Momany</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">FsFKS1, the 1,3-beta-glucan synthase from the caspofungin-resistant fungus Fusarium solani.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1036</span>â<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DHa%26rft.auinit1%253DY.-s.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D1036%26rft.epage%253D1042%26rft.atitle%253DFsFKS1%252C%2Bthe%2B1%252C3-%257Bbeta%257D-Glucan%2BSynthase%2Bfrom%2Bthe%2BCaspofungin-Resistant%2BFungus%2BFusarium%2Bsolani%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00030-06%26rft_id%253Dinfo%253Apmid%252F16835448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI1LzcvMTAzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8zMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.22" data-doi="10.1128/EC.00248-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallen</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trail</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The L-type calcium ion channel cch1 affects ascospore discharge and mycelial growth in the filamentous fungus Gibberella zeae (anamorph Fusarium graminearum).</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">415</span>â<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00248-07%26rft_id%253Dinfo%253Apmid%252F18083828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI3LzIvNDE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.23" data-doi="10.1534/genetics.109.112938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The impact of whole genome sequencing on model system genetics: get ready for the ride.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">317</span>â<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D184%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.112938%26rft_id%253Dinfo%253Apmid%252F20103786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.112938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20103786&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-2" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horowitz</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macleod</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sueoka</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span>â<span class="cit-article-title">A genetic study of two new structural forms of tyrosinase in Neurospora.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1015</span>â<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHorowitz%26rft.auinit1%253DN.%2BH.%26rft.volume%253D46%26rft.issue%253D8%26rft.spage%253D1015%26rft.epage%253D1024%26rft.atitle%253DA%2BGENETIC%2BSTUDY%2BOF%2BTWO%2BNEW%2BSTRUCTURAL%2BFORMS%2BOF%2BTYROSINASE%2BIN%2BNEUROSPORA%26rft_id%253Dinfo%253Apmid%252F13715943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiI0Ni84LzEwMTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.4.303.25" data-doi="10.1007/BF00269389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">H. B.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">E. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span>â<span class="cit-article-title">A perithecial color mutant of Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">79</span>â<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DHowe%26rft.auinit1%253DH.%2BB.%26rft.volume%253D131%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D83%26rft.atitle%253DA%2Bperithecial%2Bcolor%2Bmutant%2Bof%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00269389%26rft_id%253Dinfo%253Apmid%252F4277338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00269389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4277338&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T668200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.26" data-doi="10.1038/nrm1499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadrmas</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beckerle</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">The LIM domain: from the cytoskeleton to the nucleus.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">920</span>â<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DKadrmas%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253D920%26rft.epage%253D931%26rft.atitle%253DThe%2BLIM%2Bdomain%253A%2Bfrom%2Bthe%2Bcytoskeleton%2Bto%2Bthe%2Bnucleus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1499%26rft_id%253Dinfo%253Apmid%252F15520811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15520811&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224987300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.27" data-doi="10.1073/pnas.0905848106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Crom</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schackwitz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pennacchio</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnuson</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culley</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Tracking the roots of cellulase hyperproduction by the fungus Trichoderma reesei using massively parallel DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">16151</span>â<span class="cit-lpage">16156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0905848106%26rft_id%253Dinfo%253Apmid%252F19805272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2MTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milgroom</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">A rapid, high yield mini-prep method for isolation of total genomic DNA from fungi.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">1988</span>: <span class="cit-fpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D1988%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.29" data-doi="10.1101/gr.078212.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1851</span>â<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078212.108%26rft_id%253Dinfo%253Apmid%252F18714091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Characterization of MoLDB1 required for vegetative growth, infection-related morphogenesis, and pathogenicity in the rice blast fungus Magnaporthe oryzae.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1260</span>â<span class="cit-lpage">1274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D23%26rft.issue%253D10%26rft.spage%253D1260%26rft.epage%253D1274%26rft.atitle%253DCharacterization%2Bof%2BMoLDB1%2Brequired%2Bfor%2Bvegetative%2Bgrowth%252C%2Binfection-related%2Bmorphogenesis%252C%2Band%2Bpathogenicity%2Bin%2Bthe%2Brice%2Bblast%2Bfungus%2BMagnaporthe%2Boryzae.%26rft_id%253Dinfo%253Apmid%252F20831406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20831406&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindegren</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beanfield</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1939</span>â<span class="cit-article-title">Increasing the fertility of Neurospora by selective inbreeding.</span> <abbr class="cit-jnl-abbrev">Bot. Gaz.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">592</span>â<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BGaz.%26rft.volume%253D100%26rft.spage%253D592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.32" data-doi="10.1038/emboj.2010.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Crystal structure of the yeast Sac1: implications for its phosphoinositide phosphatase function.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1489</span>â<span class="cit-lpage">1498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DManford%26rft.auinit1%253DA.%26rft.volume%253D29%26rft.issue%253D9%26rft.spage%253D1489%26rft.epage%253D1498%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Byeast%2BSac1%253A%2Bimplications%2Bfor%2Bits%2Bphosphoinositide%2Bphosphatase%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2010.57%26rft_id%253Dinfo%253Apmid%252F20389282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2010.57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20389282&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277332200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yedlin</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madole</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plamann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Complementation of un-16 and the development of a selectable marker for transformation of Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">9</span>â<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D54%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.34" data-doi="10.1093/molbev/msm035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">P. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The majority of recent short DNA insertions in the human genome are tandem duplications.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1190</span>â<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm035%26rft_id%253Dinfo%253Apmid%252F17322553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17322553&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246802400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-2" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mylyk</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Threlkeld</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span>â<span class="cit-article-title">A genetic study of female sterility in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">91</span>â<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DMylyk%26rft.auinit1%253DO.%2BM.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D91%26rft.epage%253D102%26rft.atitle%253DA%2Bgenetic%2Bstudy%2Bof%2Bfemale%2Bsterility%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Apmid%252F4279838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4279838&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-2" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.36" data-doi="10.1038/1901129a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egashira</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span>â<span class="cit-article-title">Genetically mixed perithecia in Neurospora.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1129</span>â<span class="cit-lpage">1130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNakamura%26rft.auinit1%253DK.%26rft.volume%253D190%26rft.spage%253D1129%26rft.epage%253D1130%26rft.atitle%253DGenetically%2Bmixed%2Bperithecia%2Bin%2BNeurospora.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F1901129a0%26rft_id%253Dinfo%253Apmid%252F13727611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/1901129a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13727611&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.37" data-doi="10.1128/EC.00466-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Mitogen-activated protein kinase cascade required for regulation of development and secondary metabolism in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2113</span>â<span class="cit-lpage">2122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00466-07%26rft_id%253Dinfo%253Apmid%252F18849472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI3LzEyLzIxMTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.38" data-doi="10.1016/j.febslet.2005.09.058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidaway</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The Saccharomyces cerevisiae Ca2+ channel Cch1pMid1p is essential for tolerance to cold stress and iron toxicity.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">579</span>: <span class="cit-fpage">5697</span>â<span class="cit-lpage">5703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DPeiter%26rft.auinit1%253DE.%26rft.volume%253D579%26rft.issue%253D25%26rft.spage%253D5697%26rft.epage%253D5703%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BCa2%252B%2Bchannel%2BCch1pMid1p%2Bis%2Bessential%2Bfor%2Btolerance%2Bto%2Bcold%2Bstress%2Band%2Biron%2Btoxicity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2005.09.058%26rft_id%253Dinfo%253Apmid%252F16223494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2005.09.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16223494&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232824200043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.4.303.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span>â<span class="cit-article-title">New Markers and Multiple Point Linkage Data in Neurospora.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1185</span>â<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPerkins%26rft.auinit1%253DD.%2BD.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1185%26rft.epage%253D1208%26rft.atitle%253DNEW%2BMARKERS%2BAND%2BMULTIPLE%2BPOINT%2BLINKAGE%2BDATA%2BIN%2BNEUROSPORA%26rft_id%253Dinfo%253Apmid%252F17247885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiI0NC82LzExODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-2" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishitani</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span>â<span class="cit-article-title">Linkage Data for Group III Markers in Neurospora.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1209</span>â<span class="cit-lpage">1213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPerkins%26rft.auinit1%253DD.%2BD.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1209%26rft.epage%253D1213%26rft.atitle%253DLINKAGE%2BDATA%2BFOR%2BGROUP%2BIII%2BMARKERS%2BIN%2BNEUROSPORA%26rft_id%253Dinfo%253Apmid%252F17247886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo5OiI0NC82LzEyMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.41" data-doi="10.1139/g62-024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glassey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span>â<span class="cit-article-title">New data on markers and rearrangements in Neurospora.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">187</span>â<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BGenet.%2BCytol.%26rft.volume%253D4%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg62-024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g62-024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newmeyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjorkman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Chromosomal loci of Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Microbiol. Rev.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">426</span>â<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DPerkins%26rft.auinit1%253DD%2BD%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D426%26rft.epage%253D570%26rft.atitle%253DChromosomal%2Bloci%2Bof%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Apmid%252F6219280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo0OiJtbWJyIjtzOjU6InJlc2lkIjtzOjg6IjQ2LzQvNDI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-book" id="cit-1.4.303.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-source">The Neurospora Compendium: Chromosomal Loci.</span> <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.44" data-doi="10.1128/EC.00016-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pomraning</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Bulk segregant analysis followed by high-throughput sequencing reveals the Neurospora cell cycle gene, ndc-1, to be allelic with the gene for ornithine decarboxylase, spe-1.</span> <abbr class="cit-jnl-abbrev">Eukaryot Cell.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">724</span>â<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot%2BCell.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00016-11%26rft_id%253Dinfo%253Apmid%252F21515825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMC82LzcyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8zMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.45" data-doi="10.1091/mbc.10.7.2235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rivas</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearns</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekar</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Pleiotropic alterations in lipid metabolism in yeast sac1 mutants: relationship to âbypass Sec14pâ and inositol auxotrophy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2235</span>â<span class="cit-lpage">2250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DRivas%26rft.auinit1%253DM.%2BP.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D2235%26rft.epage%253D2250%26rft.atitle%253DPleiotropic%2BAlterations%2Bin%2BLipid%2BMetabolism%2Bin%2BYeast%2Bsac1%2BMutants%253A%2BRelationship%2Bto%2B%2522Bypass%2BSec14p%2522%2Band%2BInositol%2BAuxotrophy%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.10.7.2235%26rft_id%253Dinfo%253Apmid%252F10397762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEwLzcvMjIzNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8zMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.46" data-doi="10.1534/genetics.110.116319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigelow</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyanov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Analysis of multiple ethyl methanesulfonate-mutagenized Caenorhabditis elegans strains by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">417</span>â<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D185%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116319%26rft_id%253Dinfo%253Apmid%252F20439776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.116319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20439776&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281905200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.47" data-doi="10.1091/mbc.E02-07-0433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seiler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plamann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">The genetic basis of cellular morphogenesis in the filamentous fungus Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">4352</span>â<span class="cit-lpage">4364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E02-07-0433%26rft_id%253Dinfo%253Apmid%252F12960438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS80MzUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.48" data-doi="10.1073/pnas.0508896103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raju</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruprich-Robert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">SAD-2 is required for meiotic silencing by unpaired DNA and perinuclear localization of SAD-1 RNA-directed RNA polymerase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">2243</span>â<span class="cit-lpage">2248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508896103%26rft_id%253Dinfo%253Apmid%252F16461906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAzLzcvMjI0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8zMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.49" data-doi="10.1101/gr.077776.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peckham</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makowsky</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Rapid whole-genome mutational profiling using next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1638</span>â<span class="cit-lpage">1642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.077776.108%26rft_id%253Dinfo%253Apmid%252F18775913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMC8xNjM4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzMwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-2" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.50" data-doi="10.1101/gad.3.4.559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanofsky</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">A morphological and genetic analysis of conidiophore development in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">559</span>â<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSpringer%26rft.auinit1%253DM%2BL%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D559%26rft.epage%253D571%26rft.atitle%253DA%2Bmorphological%2Band%2Bgenetic%2Banalysis%2Bof%2Bconidiophore%2Bdevelopment%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.3.4.559%26rft_id%253Dinfo%253Apmid%252F2524423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMy80LzU1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8zMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.4.303.51" data-doi="10.1007/BF00333631"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span>â<span class="cit-article-title">A gene controlling the early development of protoperithecium in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">158</span>â<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DTee%26rft.auinit1%253DT.%2BS.%26rft.volume%253D107%26rft.issue%253D2%26rft.spage%253D158%26rft.epage%253D161%26rft.atitle%253DA%2Bgene%2Bcontrolling%2Bthe%2Bearly%2Bdevelopment%2Bof%2Bprotoperithecium%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00333631%26rft_id%253Dinfo%253Apmid%252F4938066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00333631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4938066&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>â<span class="cit-article-title">Spore killer, a chromosomal factor in neurospora that kills meiotic products not containing it.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">587</span>â<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D93%26rft.spage%253D587%26rft_id%253Dinfo%253Apmid%252F17248973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17248973&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979JK65000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.53" data-doi="10.1006/fgbi.2001.1247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fairfield</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Neurospora from natural populations: a global study.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">67</span>â<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DTurner%26rft.auinit1%253DB.%2BC.%26rft.volume%253D32%26rft.issue%253D2%26rft.spage%253D67%26rft.epage%253D92%26rft.atitle%253DNeurospora%2Bfrom%2Bnatural%2Bpopulations%253A%2Ba%2Bglobal%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.2001.1247%26rft_id%253Dinfo%253Apmid%252F11352529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.2001.1247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11352529&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168362800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.4.303.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weijer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vigfusson</span>  <span class="cit-name-given-names">N. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span>â<span class="cit-article-title">Sexuality in Neurospora crassa. I. Mutations to male sterility.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">191</span>â<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWeijer%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D191%26rft.epage%253D204%26rft.atitle%253DSexuality%2Bin%2BNeurospora%2Bcrassa.%2BI.%2BMutations%2Bto%2Bmale%2Bsterility.%26rft_id%253Dinfo%253Apmid%252F4263083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4263083&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">âµ</a><div class="cit ref-cit ref-journal" id="cit-1.4.303.55" data-doi="10.1186/1471-2148-8-109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wik</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannesson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The evolutionary trajectory of the mating-type (mat) genes in Neurospora relates to reproductive behavior of taxa.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DWik%26rft.auinit1%253DL.%26rft.volume%253D8%26rft.spage%253D109%26rft.epage%253D109%26rft.atitle%253DThe%2Bevolutionary%2Btrajectory%2Bof%2Bthe%2Bmating-type%2B%2528mat%2529%2Bgenes%2Bin%2BNeurospora%2Brelates%2Bto%2Breproductive%2Bbehavior%2Bof%2Btaxa.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-109%26rft_id%253Dinfo%253Apmid%252F18405383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18405383&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F303.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/4/303.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/4/293" title="Identification of an Active New Mutator Transposable Element in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/4/317" title="A Synthetic Human Kinase Can Control Cell Cycle Progression in Budding Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/4"><span>Volume 1 Issue 4, September 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/4.cover.gif?itok=yDWwhPU2" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/4/303" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yYulgnOQi9bbtfCrplbsDvJI40DotEvuyzAF0qaFBkc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/4/303" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/4/303" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/4/303" class="active" data-icon-position="" data-hide-link-title="0">Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in Neurospora crassa</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459180" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ak6aWLAXxxy0DyaswW5tNEI2VtNwcJUq1CPygBTgSdM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/4/303.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/4/303" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kCvQdMHpHi-fWKhJfR_LzcGPQsLmwCCVVgiCddMYv6w" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/4/303" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/4/303.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459180" id="node459180--2" data-pisa="ggg;1/4/303" data-pisa-master="ggg;g3.111.000307" data-apath="/ggg/1/4/303.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in <em>Neurospora crassa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kevin McCluskey</span>, <span class="highwire-citation-author" data-delta="1">Aric E. Wiest</span>, <span class="highwire-citation-author" data-delta="2">Igor V. Grigoriev</span>, <span class="highwire-citation-author" data-delta="3">Anna Lipzen</span>, <span class="highwire-citation-author" data-delta="4">Joel Martin</span>, <span class="highwire-citation-author" data-delta="5">Wendy Schackwitz</span> and <span class="highwire-citation-author" data-delta="6">Scott E. Baker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">303-316;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000307 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459180--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kevin McCluskey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BMcCluskey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCluskey%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BMcCluskey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mccluskeyk@umkc.edu" class="" data-icon-position="" data-hide-link-title="0">mccluskeyk@umkc.edu</a>
<a href="mailto:scott.baker@pnnl.gov" class="" data-icon-position="" data-hide-link-title="0">scott.baker@pnnl.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aric E. Wiest </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aric%2BE.%2BWiest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiest%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAric%2BE.%2BWiest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Igor V. Grigoriev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BV.%2BGrigoriev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grigoriev%20IV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BV.%2BGrigoriev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anna Lipzen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BLipzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipzen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BLipzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joel Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wendy Schackwitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wendy%2BSchackwitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schackwitz%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWendy%2BSchackwitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Scott E. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Chemical and Biological Process Development Group, Pacific Northwest National Laboratory, Richland, Washington 99354</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BE.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BE.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mccluskeyk@umkc.edu" class="" data-icon-position="" data-hide-link-title="0">mccluskeyk@umkc.edu</a>
<a href="mailto:scott.baker@pnnl.gov" class="" data-icon-position="" data-hide-link-title="0">scott.baker@pnnl.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/4/303" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/4/303&amp;title=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/4/303&amp;title=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/4/303&amp;title=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/4/303&amp;text=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/4/303&amp;title=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/4/303&amp;t=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/4/303&amp;title=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/4/303&amp;title=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459180/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459180" id="node459180--3" data-pisa="ggg;1/4/303" data-pisa-master="ggg;g3.111.000307" data-apath="/ggg/1/4/303.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rediscovery by Whole Genome Sequencing: Classical Mutations and Genome Polymorphisms in <em>Neurospora crassa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kevin McCluskey</span>, <span class="highwire-citation-author" data-delta="1">Aric E. Wiest</span>, <span class="highwire-citation-author" data-delta="2">Igor V. Grigoriev</span>, <span class="highwire-citation-author" data-delta="3">Anna Lipzen</span>, <span class="highwire-citation-author" data-delta="4">Joel Martin</span>, <span class="highwire-citation-author" data-delta="5">Wendy Schackwitz</span> and <span class="highwire-citation-author" data-delta="6">Scott E. Baker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">303-316;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000307 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459180--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kevin McCluskey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BMcCluskey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McCluskey%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BMcCluskey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mccluskeyk@umkc.edu" class="" data-icon-position="" data-hide-link-title="0">mccluskeyk@umkc.edu</a>
<a href="mailto:scott.baker@pnnl.gov" class="" data-icon-position="" data-hide-link-title="0">scott.baker@pnnl.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aric E. Wiest </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Fungal Genetics Stock Center, School of Biological Sciences, University of Missouri-Kansas City, Kansas City, Missouri 64110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aric%2BE.%2BWiest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiest%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAric%2BE.%2BWiest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Igor V. Grigoriev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BV.%2BGrigoriev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grigoriev%20IV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BV.%2BGrigoriev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anna Lipzen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BLipzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipzen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BLipzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joel Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Wendy Schackwitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wendy%2BSchackwitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schackwitz%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWendy%2BSchackwitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Scott E. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>U.S. Department of Energy Joint Genome Institute, Walnut Creek, California 94598, and</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Chemical and Biological Process Development Group, Pacific Northwest National Laboratory, Richland, Washington 99354</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BE.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AScott%2BE.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mccluskeyk@umkc.edu" class="" data-icon-position="" data-hide-link-title="0">mccluskeyk@umkc.edu</a>
<a href="mailto:scott.baker@pnnl.gov" class="" data-icon-position="" data-hide-link-title="0">scott.baker@pnnl.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=303&amp;rft.epage=316&amp;rft.atitle=Rediscovery%20by%20Whole%20Genome%20Sequencing%3A%20Classical%20Mutations%20and%20Genome%20Polymorphisms%20in%20Neurospora%20crassa&amp;rft.volume=1&amp;rft.issue=4&amp;rft.date=2011-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=McCluskey%2C+Kevin&amp;rft.au=Wiest%2C+Aric+E.&amp;rft.au=Grigoriev%2C+Igor+V.&amp;rft.au=Lipzen%2C+Anna&amp;rft.au=Martin%2C+Joel&amp;rft.au=Schackwitz%2C+Wendy&amp;rft.au=Baker%2C+Scott+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459180/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459180/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459180/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459180/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459180/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459180/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459180/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459180/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459180/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459180/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459180/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459180/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/4/303" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-25" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/4/303.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/4/303.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/4/303.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/4/303.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
