<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/4/263" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/4/263.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/4/263.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/4/263.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/4.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/4/263.atom" />
<meta name="HW.pisa" content="ggg;1/4/263" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach" />
<meta name="DC.Identifier" content="10.1534/g3.111.000422" />
<meta name="DC.Date" content="2011-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Ambroset et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The genetic basis of the phenotypic diversity of yeast is still poorly understood. Wine yeast strains have specific abilities to grow and ferment under stressful conditions compared with other strains, but the genetic basis underlying these traits is unknown. Understanding how sequence variation influences such phenotypes is a major challenge to address adaptation mechanisms of wine yeast. We aimed to identify the genetic basis of fermentation traits and gain insight into their relationships with variations in gene expression among yeast strains. We combined fermentation trait QTL mapping and expression profiling of fermenting cells in a segregating population from a cross between a wine yeast derivative and a laboratory strain. We report the identification of QTL for various fermentation traits (fermentation rates, nitrogen utilization, metabolites production) as well as expression QTL (eQTL). We found that many transcripts mapped to several eQTL hotspots and that two of them overlapped with QTL for fermentation traits. A QTL controlling the maximal fermentation rate and nitrogen utilization overlapping with an eQTL hotspot was dissected. We functionally demonstrated that an allele of the [ABZ1][1] gene, localized in the hotspot and involved in p -aminobenzoate biosynthesis, controls the fermentation rate through modulation of nitrogen utilization. Our data suggest that the laboratory strain harbors a defective [ABZ1][1] allele, which triggers strong metabolic and physiological alterations responsible for the generation of the eQTL hotspot. They also suggest that a number of gene expression differences result from some alleles that trigger major physiological disturbances.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316" />
<meta name="DC.Contributor" content="Chloé Ambroset" />
<meta name="DC.Contributor" content="Maud Petit" />
<meta name="DC.Contributor" content="Christian Brion" />
<meta name="DC.Contributor" content="Isabelle Sanchez" />
<meta name="DC.Contributor" content="Pierre Delobel" />
<meta name="DC.Contributor" content="Cyprien Guérin" />
<meta name="DC.Contributor" content="Hélène Chiapello" />
<meta name="DC.Contributor" content="Pierre Nicolas" />
<meta name="DC.Contributor" content="Frédéric Bigey" />
<meta name="DC.Contributor" content="Sylvie Dequin" />
<meta name="DC.Contributor" content="Bruno Blondin" />
<meta name="article:published_time" content="2011-09-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The genetic basis of the phenotypic diversity of yeast is still poorly understood. Wine yeast strains have specific abilities to grow and ferment under stressful conditions compared with other strains, but the genetic basis underlying these traits is unknown. Understanding how sequence variation influences such phenotypes is a major challenge to address adaptation mechanisms of wine yeast. We aimed to identify the genetic basis of fermentation traits and gain insight into their relationships with variations in gene expression among yeast strains. We combined fermentation trait QTL mapping and expression profiling of fermenting cells in a segregating population from a cross between a wine yeast derivative and a laboratory strain. We report the identification of QTL for various fermentation traits (fermentation rates, nitrogen utilization, metabolites production) as well as expression QTL (eQTL). We found that many transcripts mapped to several eQTL hotspots and that two of them overlapped with QTL for fermentation traits. A QTL controlling the maximal fermentation rate and nitrogen utilization overlapping with an eQTL hotspot was dissected. We functionally demonstrated that an allele of the &lt;i&gt;ABZ1&lt;/i&gt; gene, localized in the hotspot and involved in &lt;i&gt;p&lt;/i&gt;-aminobenzoate biosynthesis, controls the fermentation rate through modulation of nitrogen utilization. Our data suggest that the laboratory strain harbors a defective &lt;i&gt;ABZ1&lt;/i&gt; allele, which triggers strong metabolic and physiological alterations responsible for the generation of the eQTL hotspot. They also suggest that a number of gene expression differences result from some alleles that trigger major physiological disturbances.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/09/01" />
<meta name="citation_mjid" content="ggg;1/4/263" />
<meta name="citation_id" content="1/4/263" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/4/263" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/4/263.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/4/263.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/4/263.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000422" />
<meta name="citation_pmid" content="22384338" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="4" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="263" />
<meta name="citation_lastpage" content="281" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Chloé Ambroset" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), UMR1083, and" />
<meta name="citation_author" content="Maud Petit" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), UMR1083, and" />
<meta name="citation_author" content="Christian Brion" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), UMR1083, and" />
<meta name="citation_author" content="Isabelle Sanchez" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), UMR1083, and" />
<meta name="citation_author" content="Pierre Delobel" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), UMR1083, and" />
<meta name="citation_author" content="Cyprien Guérin" />
<meta name="citation_author_institution" content="INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France" />
<meta name="citation_author" content="Hélène Chiapello" />
<meta name="citation_author_institution" content="INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France" />
<meta name="citation_author" content="Pierre Nicolas" />
<meta name="citation_author_institution" content="INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France" />
<meta name="citation_author" content="Frédéric Bigey" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), UMR1083, and" />
<meta name="citation_author" content="Sylvie Dequin" />
<meta name="citation_author_institution" content="Institut National de la Recherche Agronomique (INRA), UMR1083, and" />
<meta name="citation_author" content="Bruno Blondin" />
<meta name="citation_author_institution" content="Montpellier SupAgro, UMR 1083 Sciences pour l'Oenologie, F-34060 Montpellier, and" />
<meta name="citation_author_email" content="blondin@supagro.inra.fr" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Ansel;citation_author=H. Bottin;citation_author=C. Rodriguez-Beltran;citation_author=C. Damon;citation_author=M. Nagarajan;citation_title=Cell-to-cell stochastic variation in gene expression is a complex genetic trait.;citation_pages=e1000049-e1000049;citation_volume=4;citation_year=2008;citation_issue=4;citation_pmid=18404214;citation_doi=10.1371/journal.pgen.1000049" />
<meta name="citation_reference" content="citation_journal_title=J. Ferment. Bioeng.;citation_author=M. Bely;citation_author=J.-M. Sablayrolles;citation_author=P. Barre;citation_title=Automatic detection of assimilable nitrogen deficiencies during alcoholic fermentation in oenological conditions.;citation_pages=246-252;citation_volume=70;citation_year=1990;citation_doi=10.1016/0922-338X(90)90057-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Ben-Ari;citation_author=D. Zenvirth;citation_author=A. Sherman;citation_author=L. David;citation_author=M. Klutstein;citation_title=Four linked genes participate in controlling sporulation efficiency in budding yeast.;citation_pages=e195-e195;citation_volume=2;citation_year=2006;citation_issue=11;citation_pmid=17112318;citation_doi=10.1371/journal.pgen.0020195" />
<meta name="citation_reference" content="citation_journal_title=Behavioural brain research;citation_journal_abbrev=Behav Brain Res;citation_author=Y. Benjamini;citation_author=D. Drai;citation_author=G. Elmer;citation_author=N. Kafkafi;citation_author=I. Golani;citation_title=Controlling the false discovery rate in behavior genetics research.;citation_pages=279-284;citation_volume=125;citation_year=2001;citation_issue=1-2;citation_pmid=11682119;citation_doi=10.1016/S0166-4328(01)00297-2" />
<meta name="citation_reference" content="citation_title=Ammonia, pp. 454–461 in Methods of Enzymatic Analysis;citation_author=H. U. Bergmeyer;citation_author=H. O. Beutler;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. R. Borneman;citation_author=A. H. Forgan;citation_author=I. S. Pretorius;citation_author=P. J. Chambers;citation_title=Comparative genome analysis of a Saccharomyces cerevisiae wine strain.;citation_volume=8;citation_year=2008;citation_pmid=18778279;citation_doi=10.1111/j.1567-1364.2008.00434.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. R. Borneman;citation_author=B. A. Desany;citation_author=D. Riches;citation_author=J. P. Affourtit;citation_author=A. H. Forgan;citation_author=I. S. Pretorius;citation_title=Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of Saccharomyces cerevisiae.;citation_pages=e1001287-e1001287;citation_volume=7;citation_year=2011;citation_issue=2;citation_pmid=21304888;citation_doi=10.1371/journal.pgen.1001287" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. J. Brauer;citation_author=C. M. Christianson;citation_author=D. A. Pai;citation_author=M. J. Dunham;citation_title=Mapping novel traits by array-assisted bulk segregant analysis in Saccharomyces cerevisiae.;citation_pages=1813-1816;citation_volume=173;citation_year=2006;citation_pmid=16624899;citation_doi=10.1534/genetics.106.057927" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. B. Brem;citation_author=G. Yvert;citation_author=R. Clinton;citation_author=L. Kruglyak;citation_title=Genetic dissection of transcriptional regulation in budding yeast.;citation_volume=296;citation_year=2002;citation_pmid=11923494;citation_doi=10.1126/science.1069516" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. B. Brem;citation_author=L. Kruglyak;citation_title=The landscape of genetic complexity across 5,700 gene expression traits in yeast.;citation_volume=102;citation_year=2005;citation_pmid=15659551;citation_doi=10.1073/pnas.0408709102" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=K. M. Brown;citation_author=C. R. Landry;citation_author=D. L. Hartl;citation_author=D. Cavalieri;citation_title=Cascading transcriptional effects of a naturally occurring frameshift mutation in Saccharomyces cerevisiae.;citation_pages=2985-2997;citation_volume=17;citation_year=2008;citation_issue=12;citation_pmid=18422925;citation_doi=10.1111/j.1365-294X.2008.03765.x" />
<meta name="citation_reference" content="citation_journal_title=J. Chromatogr. A;citation_author=M. Calull;citation_author=E. Lopez;citation_author=R. M. Marce;citation_author=J. C. Olucha;citation_author=F. Borrull;citation_title=Optimization of an ion exchange high-performance liquid chromatographic method for determination of carboxylic acids, sugars, glycerol, and ethanol in wines.;citation_pages=151-158;citation_volume=589;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Cavalieri;citation_author=J. P. Townsend;citation_author=D. L. Hartl;citation_title=Manifold anomalies in gene expression in a vineyard isolate of Saccharomyces cerevisiae revealed by DNA microarray analysis.;citation_volume=97;citation_year=2000;citation_pmid=11035792;citation_doi=10.1073/pnas.210395297" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. A. Cubillos;citation_author=E. Billi;citation_author=E. Zorgo;citation_author=L. Parts;citation_author=P. Fargier;citation_title=Assessing the complex architecture of polygenic traits in diverged yeast populations.;citation_pages=1401-1413;citation_volume=20;citation_year=2011;citation_issue=7;citation_pmid=21261765;citation_doi=10.1111/j.1365-294X.2011.05005.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. M. Deutschbauer;citation_author=R. W. Davis;citation_title=Quantitative trait loci mapped to single-nucleotide resolution in yeast.;citation_pages=1333-1340;citation_volume=37;citation_year=2005;citation_issue=12;citation_pmid=16273108;citation_doi=10.1038/ng1674" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. W. Doniger;citation_author=H. S. Kim;citation_author=D. Swain;citation_author=D. Corcuera;citation_author=M. Williams;citation_title=A catalog of neutral and deleterious polymorphism in yeast.;citation_pages=e1000183-e1000183;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18769710;citation_doi=10.1371/journal.pgen.1000183" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Dunham;citation_author=H. Badrane;citation_author=T. Ferea;citation_author=J. Adams;citation_author=P. O. Brown;citation_title=Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae.;citation_volume=99;citation_year=2002;citation_pmid=12446845;citation_doi=10.1073/pnas.242624799" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. Dunn;citation_author=R. P. Levine;citation_author=G. Sherlock;citation_title=Microarray karyotyping of commercial wine yeast strains reveals shared, as well as unique, genomic signatures.;citation_pages=53-53;citation_volume=6;citation_year=2005;citation_pmid=15833139;citation_doi=10.1186/1471-2164-6-53" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Edgar;citation_author=M. Domrachev;citation_author=A. E. Lash;citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository;citation_pages=207-210;citation_volume=30;citation_year=2002;citation_issue=1;citation_pmid=11752295;citation_doi=10.1093/nar/30.1.207" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. C. Fay;citation_author=H. L. McCullough;citation_author=P. D. Sniegowski;citation_author=M. B. Eisen;citation_title=Population genetic variation in gene expression is associated with phenotypic variation in Saccharomyces cerevisiae.;citation_pages=R26-R26;citation_volume=5;citation_year=2004;citation_issue=4;citation_pmid=15059259;citation_doi=10.1186/gb-2004-5-4-r26" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Fu;citation_author=J. J. Keurentjes;citation_author=H. Bouwmeester;citation_author=T. America;citation_author=F. W. Verstappen;citation_title=System-wide molecular evidence for phenotypic buffering in Arabidopsis.;citation_pages=166-167;citation_volume=41;citation_year=2009;citation_issue=2;citation_pmid=19169256;citation_doi=10.1038/ng.308" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=A. P. Gasch;citation_author=P. T. Spelman;citation_author=C. M. Kao;citation_author=O. Carmel-Harel;citation_author=M. B. Eisen;citation_title=Genomic Expression Programs in the Response of Yeast Cells to Environmental Changes;citation_pages=4241-4257;citation_volume=11;citation_year=2000;citation_issue=12;citation_pmid=11102521;citation_doi=10.1091/mbc.11.12.4241" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. P. Gasch;citation_title=Comparative genomics of the environmental stress response in ascomycete fungi.;citation_pages=961-976;citation_volume=24;citation_year=2007;citation_issue=11;citation_pmid=17605132;citation_doi=10.1002/yea.1512" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Gerke;citation_author=C. T. Chen;citation_author=B. A. Cohen;citation_title=Natural isolates of Saccharomyces cerevisiae display complex genetic variation in sporulation efficiency.;citation_pages=985-997;citation_volume=174;citation_year=2006;citation_pmid=16951083;citation_doi=10.1534/genetics.106.058453" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. Barrell;citation_author=H. Bussey;citation_author=R. Davis;citation_author=B. Dujon;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=5287;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. H. Hu;citation_author=M. H. Wang;citation_author=T. Tan;citation_author=J. R. Li;citation_author=H. Yang;citation_title=Genetic dissection of ethanol tolerance in the budding yeast Saccharomyces cerevisiae.;citation_pages=1479-1487;citation_volume=175;citation_year=2007;citation_pmid=17194785;citation_doi=10.1534/genetics.106.065292" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. J. Infante;citation_author=K. M. Dombek;citation_author=L. Rebordinos;citation_author=J. M. Cantoral;citation_author=E. T. Young;citation_title=Genome-Wide Amplifications Caused by Chromosomal Rearrangements Play a Major Role in the Adaptive Evolution of Natural Yeast;citation_pages=1745-1759;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704163" />
<meta name="citation_reference" content="citation_author=Instituto Nacional de la Investigación en Agronomía (INRA). GenYeasTrait Genome Browser" />
<meta name="citation_reference" content="citation_author=Institut National des Sciences Appliquées. Plateforme Biopuces" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. C. Jansen;citation_author=J. P. Nap;citation_title=Genetical genomics: the added value from segregation.;citation_pages=388-391;citation_volume=17;citation_year=2001;citation_issue=7;citation_pmid=11418218;citation_doi=10.1016/S0168-9525(01)02310-1" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biosci Bioeng;citation_author=T. Katou;citation_author=M. Namise;citation_author=H. Kitagaki;citation_author=T. Akao;citation_author=H. Shimoi;citation_title=QTL mapping of sake brewing characteristics of yeast.;citation_pages=383-393;citation_volume=107;citation_year=2009;citation_issue=4;citation_pmid=19332297;citation_doi=10.1016/j.jbiosc.2008.12.014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=N. Patterson;citation_author=M. Endrizzi;citation_author=B. Birren;citation_author=E. S. Lander;citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.;citation_pages=241-254;citation_volume=423;citation_year=2003;citation_issue=6937;citation_pmid=12748633;citation_doi=10.1038/nature01644" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. S. Kim;citation_author=J. C. Fay;citation_title=Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds.;citation_volume=104;citation_year=2007;citation_pmid=18042712;citation_doi=10.1073/pnas.0708194104" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. J. Kvitek;citation_author=J. L. Will;citation_author=A. P. Gasch;citation_title=Variations in stress sensitivity and genomic expression in diverse S. cerevisiae isolates.;citation_pages=e1000223-e1000223;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18927628;citation_doi=10.1371/journal.pgen.1000223" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Li;citation_author=J. Ruan;citation_author=R. Durbin;citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores;citation_volume=18;citation_year=2008;citation_pmid=18714091;citation_doi=10.1101/gr.078212.108" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=FEM. Yeast Res.;citation_author=P. Marullo;citation_author=M. Bely;citation_author=I. Masneuf-Pomarede;citation_author=M. Pons;citation_author=M. Aigle;citation_title=Breeding strategies for combining fermentative qualities and reducing off-flavor production in a wine yeast model.;citation_volume=6;citation_year=2006;citation_pmid=16487348;citation_doi=10.1111/j.1567-1364.2006.00034.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Nogami;citation_author=Y. Ohya;citation_author=G. Yvert;citation_title=Genetic complexity and quantitative trait loci mapping of yeast morphological traits.;citation_pages=e31-e31;citation_volume=3;citation_year=2007;citation_issue=2;citation_pmid=17319748;citation_doi=10.1371/journal.pgen.0030031" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Novo;citation_author=F. Bigey;citation_author=E. Beyne;citation_author=V. Galeote;citation_author=F. Gavory;citation_title=Eukaryote-to-eukaryote gene transfer events revealed by the genome sequence of the wine yeast Saccharomyces cerevisiae EC1118.;citation_volume=106;citation_year=2009;citation_pmid=19805302;citation_doi=10.1073/pnas.0904673106" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. E. Perez-Ortin;citation_author=A. Querol;citation_author=S. Puig;citation_author=E. Barrio;citation_title=Molecular Characterization of a Chromosomal Rearrangement Involved in the Adaptive Evolution of Yeast Strains;citation_pages=1533-1539;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368245;citation_doi=10.1101/gr.436602" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria;citation_author=R Development Core Team;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=N. Rachidi;citation_author=P. Barre;citation_author=B. Blondin;citation_title=Multiple Ty-mediated chromosomal translocations lead to karyotype changes in a wine strain of Saccharomyces cerevisiae.;citation_pages=841-850;citation_volume=261;citation_year=1999;citation_issue=4-5;citation_pmid=10394922;citation_doi=10.1007/s004380050028" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. M. Razer;citation_author=E. K. O’Shea;citation_title=Control of stochasticity in eukaryotic gene expression.;citation_volume=304;citation_year=2004;citation_pmid=15166317;citation_doi=10.1126/science.1098641" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Ronald;citation_author=J. M. Akey;citation_title=The evolution of gene expression QTL in Saccharomyces cerevisiae.;citation_pages=e678-e678;citation_volume=2;citation_year=2007;citation_issue=7;citation_pmid=17668057;citation_doi=10.1371/journal.pone.0000678" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=T. Rossignol;citation_author=L. Dulau;citation_author=A. Julien;citation_author=B. Blondin;citation_title=Genome-wide monitoring of wine yeast gene expression during alcoholic fermentation.;citation_pages=1369-1385;citation_volume=20;citation_year=2003;citation_issue=16;citation_pmid=14663829;citation_doi=10.1002/yea.1046" />
<meta name="citation_reference" content="citation_title=Analyse de l'expression du génome des levures œnologiques en fermentation alcoolique par des approches post-génomiques. Ph.D. Thesis;citation_author=T. Rossignol;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. M. Ruderfer;citation_author=S. C. Pratt;citation_author=H. S. Seidel;citation_author=L. Kruglyak;citation_title=Population genomic analysis of outcrossing and recombination in yeast.;citation_pages=1077-1081;citation_volume=38;citation_year=2006;citation_issue=9;citation_pmid=16892060;citation_doi=10.1038/ng1859" />
<meta name="citation_reference" content="citation_title=Saccharomyces Genome Database;citation_author=Saccharomyces Genome Database;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=R. H. Schiestl;citation_author=R. D. Gietz;citation_title=High efficiency transformation of intact yeast cells using single stranded nucleic acids as a carrier.;citation_pages=339-346;citation_volume=16;citation_year=1989;citation_issue=5-6;citation_pmid=2692852;citation_doi=10.1007/BF00340712" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Sinha;citation_author=B. P. Nicholson;citation_author=L. M. Steinmetz;citation_author=J. H. McCusker;citation_title=Complex genetic interactions in a quantitative trait locus.;citation_pages=e13-e13;citation_volume=2;citation_year=2006;citation_issue=2;citation_pmid=16462944;citation_doi=10.1371/journal.pgen.0020013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Sinha;citation_author=L. David;citation_author=R. C. Pascon;citation_author=S. Clauder-Munster;citation_author=S. Krishnakumar;citation_title=Sequential elimination of major-effect contributors identifies additional quantitative trait loci conditioning high-temperature growth in yeast.;citation_pages=1661-1670;citation_volume=180;citation_year=2008;citation_pmid=18780730;citation_doi=10.1534/genetics.108.092932" />
<meta name="citation_reference" content="citation_title=Bioinformatics and Computational Biology Solutions using R and Bioconductor;citation_title=Limma: linear models for microarray data;citation_author=G. K. Smyth;citation_pages=397-420;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=G. K. Smyth;citation_author=T. P. Speed;citation_title=Normalization of cDNA microarray data.;citation_pages=265-273;citation_volume=31;citation_year=2003;citation_issue=4;citation_pmid=14597310;citation_doi=10.1016/S1046-2023(03)00155-5" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. K. Smyth;citation_author=J. Michaud;citation_author=H. Scott;citation_title=The use of within-array replicate spots for assessing differential expression in microarrays experiments.;citation_volume=21;citation_year=2005;citation_pmid=15657102;citation_doi=10.1093/bioinformatics/bti270" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=E. N. Smith;citation_author=L. Kruglyak;citation_title=Gene-environment interaction in yeast gene expression.;citation_pages=e83-e83;citation_volume=6;citation_year=2008;citation_issue=4;citation_pmid=18416601;citation_doi=10.1371/journal.pbio.0060083" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. D. Stein;citation_author=C. Mungall;citation_author=S. Shu;citation_author=M. Caudy;citation_author=M. Mangone;citation_title=The Generic Genome Browser: A Building Block for a Model Organism System Database;citation_pages=1599-1610;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368253;citation_doi=10.1101/gr.403602" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. M. Steinmetz;citation_author=H. Sinha;citation_author=D. R. Richards;citation_author=J. I. Spiegelman;citation_author=P. J. Oefner;citation_title=Dissecting the architecture of a quantitative trait locus in yeast.;citation_pages=326-330;citation_volume=416;citation_year=2002;citation_issue=6878;citation_pmid=11907579;citation_doi=10.1038/416326a" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=D. Thomas;citation_author=Y. Surdin-Kerjan;citation_title=Metabolism of sulfur amino acids in Saccharomyces cerevisiae;citation_pages=503-532;citation_volume=61;citation_year=1997;citation_issue=4;citation_pmid=9409150" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=C. Varela;citation_author=F. Pizarro;citation_author=E. Agosin;citation_title=Biomass Content Governs Fermentation Rate in Nitrogen-Deficient Wine Musts;citation_pages=3392-3400;citation_volume=70;citation_year=2004;citation_issue=6;citation_pmid=15184136;citation_doi=10.1128/AEM.70.6.3392-3400.2004" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Wei;citation_author=J. H. McCusker;citation_author=R. W. Hyman;citation_author=T. Jones;citation_author=Y. Ning;citation_title=Genome sequencing and comparative analysis of Saccharomyces cerevisiae strain YJM789.;citation_volume=104;citation_year=2007;citation_pmid=17652520;citation_doi=10.1073/pnas.0701291104" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. R. Richards;citation_author=A. R. Conway;citation_author=A. L. Goldstein;citation_author=S. Kalman;citation_title=Direct Allelic Variation Scanning of the Yeast Genome;citation_pages=1194-1197;citation_volume=281;citation_year=1998;citation_issue=5380;citation_pmid=9712584;citation_doi=10.1126/science.281.5380.1194" />
<meta name="citation_reference" content="citation_journal_title=Parasitology;citation_author=E. A. Winzeler;citation_author=B. Lee;citation_author=J. H. McCusker;citation_author=R. W. Davis;citation_title=Whole genome genetic-typing in yeast using high-density oligonucleotide arrays.;citation_pages=S73-S80;citation_volume=118;citation_year=1999;citation_pmid=10466139" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. Yvert;citation_author=R. B. Brem;citation_author=J. Whittle;citation_author=J. M. Akey;citation_author=E. Foss;citation_title=Trans-acting regulatory variation in Saccharomyces cerevisiae and the role of transcription factors.;citation_pages=57-64;citation_volume=35;citation_year=2003;citation_issue=1;citation_pmid=12897782;citation_doi=10.1038/ng1222" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=A. Zuzuarregui;citation_author=L. Monteoliva;citation_author=C. Gil;citation_author=M. Del Olmo;citation_title=Transcriptomic and Proteomic Approach for Understanding the Molecular Basis of Adaptation of Saccharomyces cerevisiae to Wine Fermentation;citation_pages=836-847;citation_volume=72;citation_year=2006;citation_issue=1;citation_pmid=16391125;citation_doi=10.1128/AEM.72.1.836-847.2006" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/4.cover.gif" />
<meta name="twitter:description" content="The genetic basis of the phenotypic diversity of yeast is still poorly understood. Wine yeast strains have specific abilities to grow and ferment under stressful conditions compared with other strains, but the genetic basis underlying these traits is unknown. Understanding how sequence variation influences such phenotypes is a major challenge to address adaptation mechanisms of wine yeast. We aimed to identify the genetic basis of fermentation traits and gain insight into their relationships with variations in gene expression among yeast strains. We combined fermentation trait QTL mapping and expression profiling of fermenting cells in a segregating population from a cross between a wine yeast derivative and a laboratory strain. We report the identification of QTL for various fermentation traits (fermentation rates, nitrogen utilization, metabolites production) as well as expression QTL (eQTL). We found that many transcripts mapped to several eQTL hotspots and that two of them overlapped with QTL for fermentation traits. A QTL controlling the maximal fermentation rate and nitrogen utilization overlapping with an eQTL hotspot was dissected. We functionally demonstrated that an allele of the [ABZ1][1] gene, localized in the hotspot and involved in p -aminobenzoate biosynthesis, controls the fermentation rate through modulation of nitrogen utilization. Our data suggest that the laboratory strain harbors a defective [ABZ1][1] allele, which triggers strong metabolic and physiological alterations responsible for the generation of the eQTL hotspot. They also suggest that a number of gene expression differences result from some alleles that trigger major physiological disturbances.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316" />
<meta name="og-title" property="og:title" content="Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/4/263" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The genetic basis of the phenotypic diversity of yeast is still poorly understood. Wine yeast strains have specific abilities to grow and ferment under stressful conditions compared with other strains, but the genetic basis underlying these traits is unknown. Understanding how sequence variation influences such phenotypes is a major challenge to address adaptation mechanisms of wine yeast. We aimed to identify the genetic basis of fermentation traits and gain insight into their relationships with variations in gene expression among yeast strains. We combined fermentation trait QTL mapping and expression profiling of fermenting cells in a segregating population from a cross between a wine yeast derivative and a laboratory strain. We report the identification of QTL for various fermentation traits (fermentation rates, nitrogen utilization, metabolites production) as well as expression QTL (eQTL). We found that many transcripts mapped to several eQTL hotspots and that two of them overlapped with QTL for fermentation traits. A QTL controlling the maximal fermentation rate and nitrogen utilization overlapping with an eQTL hotspot was dissected. We functionally demonstrated that an allele of the [ABZ1][1] gene, localized in the hotspot and involved in p -aminobenzoate biosynthesis, controls the fermentation rate through modulation of nitrogen utilization. Our data suggest that the laboratory strain harbors a defective [ABZ1][1] allele, which triggers strong metabolic and physiological alterations responsible for the generation of the eQTL hotspot. They also suggest that a number of gene expression differences result from some alleles that trigger major physiological disturbances.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/4.cover.gif" />
    <title>Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EJGeWx0V-0fhG0X5X3-NAzcSQ069h0t7Qo3mafD6BVE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459175","apath":"\/ggg\/1\/4\/263.atom","pisa":"ggg;1\/4\/263","ac":{"\/ggg\/1\/4\/263.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/4\/263.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/4\/263"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/4\/263"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/4\/263":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RtJYunasS0QNvuOZstBSpgB7idN4GrdvfXq1z1asyfpTQAnEYO0HgTAAeMod1xM75QAbv0AwdZPEYoy7LlL+xD50Ey8QHFy7wAwBzZ132JfscELDlbZ02Olk4YbJKPizp+tMgSI57JchFZmvTih4D0uEJc57YhhzX6vCCt5Nvl9PhwhVyKhjofzvezSNKEtb3C+rRA5m0XOZi676HnYv\\\\\\\/rEjBEkKiXl3tfTm8Bh5SWiwfbjx3mC\\\\\\\/R+wkMx28EXO85t1ASx6edLUjHtWAhS9TAT3qyT\\\\\\\/Tgz+JE3KYEQ5\\\\\\\/pFCo4qQudIZKW2Rph7Rlf4YYKbxj9L6DDjfq+fNNc+ozndBR0hHiJgCfB7\\\\\\\/US+4tHzsnl\\\\\\\/UDSwRdDO+PSZ9pix\\\\\\\/UMp55vEWYALnHEiqPjmEV7Io9CafZ6x4YSNQH6VmDbG\\\\\\\/0Yl8Y0ygTO3K1qS0bKYuqQttLUVc9NIWoUi\\\\\\\/X7EMMztsW31N4sZuHrtYbdjDKCq6B+b+ufk743FaFz1jnH88fkUeAPgmuupHoN69nORpRjcGvQt73jojd1YfGBfB5oyqAl173XAwL1puO7qWHBRqwGjbhnTR\\\\\\\/r4UB9WG72v1kvH6K1Ld3gTVi9tFczgmfuCTa67Zub9GN1ZuwKRdv1Z5XaDM6SFONSzkOYcjZqCdLk20AEycxKR059B4SD7wEIFF1fKrBBPuG0oXD2Zwzglj7l0qdfLH1ggCBS89uWxeVPdsLERhkQkyNx9W2dKiA2uNzfWiUkD8WpiGJFtdXaYCLnVop3MGrkzNAaDtZSOtf1juwCfKLqtKGvi1FcNA\\\\\\\/6j\\\\\\\/+OaOns+D5bpZgcQZQDqGQPPxpMTO8whkTOTegwOFyr\\\\\\\/iutuvf4JObwn9+IVjpjrVxH4kT6S05E3mPIL8P\\\\\\\/b9z2wnbuF\\\\\\\/VPYwA5iXMsjPRnGXx9jJaOYms1iXyRJ0P\\\\\\\/xozRtmg\\\\\\\/fnqKMww58coNkTnHiQ+PkFVBkL1oETXgXDZZMBhgCsrFh40iwM9hnB8IDNqxp9wWmf2H\\\\\\\/cqm23mQUW3obpCg\\\\\\\/cCjdGf0ogALnlP3uhESdFcEO7Fprl2d66wSTC2D0tBgqJxTsDq2AMW+D6mKOte1k\\\\\\\/Froj5l1EEAu2np3QDDlVLBZfhPYUwM24xv3Do9XsvrUjxpvOISsHZrrfhc9Wlvb+WFX\\\\\\\/fF07eojzrZyptihNSBkAbbTciuZqx62E6Bpuf0ZuGdSAH5xGBxCedsiQbaaEMik6\\\\\\\/9PaZfi4Pnm8ciibxGMWnXs5PkXc4GQ8OuzfLlAxK6jBo29hrJQsl\\\u0022,\\\u0022iv\\\u0022:\\\u0022krqjl0+3iBaDCPl1lUsEaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225428069b66225f205475b3df794612c5fa27e24b5f6595c2ac1085315c2a5ed8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t4aCVsEZqZ2ml9uBKkNI2zM5QbBN0BsXE1j1OkPM6mjgK5rb\\\\\\\/1z0PHN\\\\\\\/xjKs7p6ZtxG191O2z2vM+88fHGU89twEDR1smsMDpyx\\\\\\\/npqU0Jm930zXv4zH6dWnPq1JeUFgvAM6z+A6y3NvckQVN1SccB8IrfoW5TMGuqlrHzSXcBwaFybjxmxRftrc1I3I\\\\\\\/28yLqA6e\\\\\\\/3F1wKRuNsO99+T+sMFQAk9p\\\\\\\/me64wGOJC+\\\\\\\/boOV7xBaG27r\\\\\\\/SANK+f5EIhQnkOOBs2sNlAcQ\\\\\\\/P5DS2pVEwRqfFnY625\\\\\\\/0yKvxKkM++jAcnQEJ9C8QkPAcsxDKXDlNJly41U+FzxlIoGbNtMkQmJiPqZdEoHFxnd+A8DN\\\\\\\/J2kH5MUX9pcGpUWz+P3AG76fDtAWnQDA8ZdtCRWLNzwKGtj1T7YQ3MybqUPAfdBEkyQtkZ0AV0M7U4ztR7gCEPhoy06VEkIl6NKPARv4SVarbpJVgK2MVZTSTbYPO0EKo32UTF3D0m9S8a+4LZdEy0PETS8J7WVk16haTzbEYfaETYPF\\\\\\\/2KEbBvNqFTAl9TnZVPDMTdBK4BY6dMRd+fUg67n6fS\\\\\\\/b\\\\\\\/ScaoJUhAXqmPXIJzTiuyxnylAcTAoJmSpKjk+rFglCPbT9V4jJea1W5tVlF\\\\\\\/XWOcewRqsgSW6RnsI9L2cT\\\\\\\/XfYlTLW1d4nqAheNHB47Vd+NMv52L6KqKQ8OO8TIk6X58UiInMjffhesiwCfjmak0+y\\\\\\\/PFyorgtskFMVeKWUy86eXebuwY3whkXMF+35o\\\\\\\/QXh6YSfTXwL7BF067+\\\\\\\/FNEgEEkLrP2np\\\\\\\/WOiNOpxtOO+z3RPdd0nXmb\\\\\\\/7qfG+hVyb9MwP0HDJPWmWhUfxnfjTrGRdCxldrkMcbcJmjk4aRgSifeE0ialYcVFH9KBRLOAjqv\\\\\\\/nCHDFB7UPLM04dtMQM8DZfEX3JzvstQfKSFfY0iE6od+ybOFqeByZAqJeAoh43XQYcMNnMiThqno6qHip6YWdN6j63TGJQflmVAa5TID7ln+wBNMeCLar7zTsTfJ5\\\\\\\/WmQ0HfG9Jt9tALYoK52UvU8m7Ym0C5LzujDyYtQOExLGAN0h0ZuECqrEdq1a9kgN7pB\\\\\\\/tWYnuq01\\\\\\\/CMQAj2dDYLPHqfIGnsBTqHMQsPlVjCKPAxMxAcvRF+UCcW\\\\\\\/SvnlqP+rNX3JHZUgDiMZ+q+IayLJK3uq\\\\\\\/gvgpxWOAqUwYXk4triSAeC\\\\\\\/h84shhQ+WItPMfluGmjmWZWXEsKerPvggL6x4iyzVY2sKjKiLcVyCrXVG5BkmMYENX+yqF0F+NnteFpwQGSgWLVkSkLpkCba48BydM5coYUJeJ7y2jmI8+GSxaxIQ9zh9+dBE2dFhdebsnJwgnqrae48rAomLKgW5eg59NdcXOUG3aBPeCPyCmy5RtIH+vpAxbe2O9eJgJPVOo6NF6i806dEeF6ZCzNLEvoWxBG8XfjAhyrV2xYOQAR6SLB8dchZw9z5liv7Pp3+l64Kcnbw3wQDpEi90jVLbZL8vk7Q3BbOtRsy2PdAh6Foxt6n\\\\\\\/IgsTQEFTVh0aoBouVAQuBy5kBADywEH2fKymYDRGrLc6jHSn3yXKr+ijXFtWseyOgwTE75u+6v9Lfy6Psnvx3BZjziUa9DIFCrpPckco6gkaRp290GHJ\\\\\\\/a4TK7eOhaiwEOocYEWYmQLVLnaQ+Smp2ofKwKNwrG71xOlh95bhSq2jTYQ94aZcplbRA2mk1uPWSt9Vq+0u8MX0mEUyj3\\\\\\\/Zsa0HDGuFlfhtlckqVK2DBZU7vGHO088hFp+M1BELLXSzIygRHEJddsGrop379wQR1iFwhwwa7hAhbozIn2fZr2gMkbiJYocShmJbO8lGcJ4ESMOIYrOGrTTnc2aQZ3haLCiQJ7N0W1bJ0QypmXd2vRrjbiJWSlIIsntdtxn4e3WI2nmmfpnzAzPprKyXU6oTIbh683XBPZZRn4ldM7LK0pAg2iBbIWGS54excbksHjVYnU\\\\\\\/Dqix1HtE0LitExkcyTkvXN8wB7RUD46M8Oqp4YhU5QcQbLRBFZLzPYPabqHvDYYkTf4gMiIjlslJ4PRUr3FjzpEcHQNLcZp01hrvvXUl+qdMj1oEmkK7lC4+Ja6rCYimsdFqFa3PBg6R8KRa\\\\\\\/InAErPKJi\\\\\\\/jnp2hKaRQPwaCwTqDlcN3xIvix2aI6qEhe5Ue+t6sFDHJnEuNBbLmqX4TZzep2PJZMidLSyNiJbP5rtrSVah2ZiTFLHoKF9kJcIa49x+saYABzwe1MNNVlhpq2ydTcWCC7kBar4ayZJUgw+GYv+GXFGF0EuysCRI3aiXR+T2LMS+MulSKsqQKrtEJoOiucTnj7319DyIzQObT8pmkyOYvfqqflundorku6mVYlHcQYrafuYgNCrhHEvdvuSXfgwWFVby\\\\\\\/2Z99E1pcPdnF3kokKuunTgc+MfG\\\\\\\/zn4vaObBT\\\\\\\/0uk412G\\\\\\\/EAQ2Q7XBE6edUfhtSvCJn4NPriJT\\\\\\\/CkP+QA2US3ZUZsA8avGZ2g+aEunVvUOfGKk0MeSIlJge1xucyHaNezQpGSzDrHnlzbWpBznrXCc+mr43Rh5iNWFP0bIvlBSg3xV0kFgWCMuF2NpmBySFICFEyYFX4fgpS8O4rrZW6tuJs6Pfoof518vKyiGIE5ceK9gbIn8q5juzwJMq3JFjgpxtNI6qEJTJCg8J9lCbtWs2T5gxT36AF9zthcBbJMnRuAsBUM6PamhTHQjRp7vk3NQt3sBIKopCOp0TCOtf1pj3LPxzBMZd48eCGG4sjeYkPHBy\\\\\\\/AfveX\\\\\\\/bAGeslNQxACuWTkn8HqR5QSvbubZdtlrMf4\\\\\\\/7z+78J0Lgqd2ZnHyEbi+3D1fqgxU3FdbZfpqUNv86\\\\\\\/Vdr8cJClvx7p99aVolpA\\\\\\\/FGFc\\\\\\\/QLiQzgdm9iE+WSe7g+\\\\\\\/4Kq1YJNbJp3ZHLv213A4d3muiAUYSgdtspuCCU4D6BM2\\\\\\\/YewlEuGAjaevuVkteXVYI7m8KjTJPP0hL+DqIUBDUQ4B\\\\\\\/tvzuiETRtobthYWJ+3OJdkk7EDQUeMoX\\\\\\\/dzg9CGQFfR+Ol8+EjE0b8JoqBMm4x5n+om\\\\\\\/kvjJv3wfRTo4bCfMhgirlOBulwRgJv1Vj8oodGSCoxOKMbOJVD6skI12wpDpUIRY38Act3H6tLjWzD6ZgQXGaeT3lKVr0C5BVFi3ekB69dV1DvqpWwgDmpifv\\\\\\\/gVthSp5ut9nNxEuXIhrk8DcWmmos+Ey8XSDBVlGTYRvL9birqDgcqna1UYg57FwfElNqGlugy+3Vk1tCCloWBM\\\\\\\/dlW+Kavykp\\\\\\\/67N4Mf4ddBtE59I02mlDFrtYO0iyZd53c5ecl3KKSkdfpLAD1s5WRYpLLKhvd2eFS1usBjz2tmqyBCwf2WDfZPFbHy6Vlp5+G4B1yNIUsPRbNz9nT4V\\\\\\\/+ERzWE5SaYNacibS2QTIHotT8EQ9sWThtxhaQfXzBTxT0r60oDt8sfcke0VdBYcWsW7+n7DNN3Z+j+1SQAJ3ewYKS0zTNu5eYYd83TUFuKlK2qmHEHj5eSRuUltoHr8TcJQV5gu4\\\\\\\/og9MbPXMVX0IAkqeRFNWpmul+07OQFXrYGiK3\\\\\\\/8swk+mqqLLyUBT7ERRy5XQRqjKYqUDX8PcAu\\\\\\\/cW3MMytb670mr2+yPgx3VtvKMr9bxN463mkM371gjB6Qu8lEBsMnBQd9nJkhLUtygHhmIDgtCf7WYO7pGiq8gpGhE97M5G1p0pCBrYL\\\\\\\/Juc3fEAh40ycj+anYvJ0+C7PXBHcM7RosshQFGPl\\\\\\\/oszIwdNhlRW1wLhl5v2e7LPvJ7K4fHt8iYK3ee5\\\\\\\/Dsc0dodxTvPW\\\\\\\/TIc9KmJBonIgIxZ+H4iTW9s1Clmjm+4zwzItOl2mTRzhGTtDP4x7NfhKgFc71Shda7g\\\\\\\/08nkdll7rgMaGwU5GIgXIhZ3QnTPTTrQ0IJPeEbUdbISpS17RLGKQgiqoNXVWfVc\\\\\\\/GDBGnAO45o64CUAc9HUpslXUhoZK+KR1ZGFXc8IVLqJ0dmTftOsmdfQ2xl8IzzpTPxEAlH3jQ8hrd6g6x6PeqoITZxd0dQ67NAO2H5W28uSTnuCR3c4JLojMjeJqq2kZ00SgYIq7ckfvGoTx67RimjyYESQT0dNcP32gNskSA16MGxA6spsjTIxXfzkE1SHwYtu0Wwqsuxqru7VObgFskO8\\\\\\\/8zF5Syoi0Fo9LZrql8NCDYj\\\\\\\/NhXXfIed6lPy2K10cZPuOWSko2z9IudvWa7RMWlWHVg2EvD9I\\\\\\\/ntkUXy4KHBq7UL1rWOrTqBvgR0wUt4L0pQ9ArWVj0Vyu2QkgReV8BjF+yOcvgO6+iGBinwf6K++0y+\\\\\\\/RE+DJrtpp09A3KQ1wN4zepU1GA\\\\\\\/ogwvY6ncb\\\\\\\/UHbGL18R0MEnXOsBXbc5J9nR9nXaIsgFwnC2FKuIOZAOMi2rk5RIUBGg56ZkQG7z9X9J2S4KkND0rlmaM5PY9YV7D8MiorerExFGCLKaVPEeuPDZ77GVUj8fnuh58p7C4kZBDWUPro199I\\\\\\\/QlzswzsQd5KywrX6qEMBMhvFf7n7WwNOjTnQ9fFyDaFKuVAffhRAU+ontg\\\\\\\/DpHDTQcjnuq5aGld1Ce2k7V2jceStFJmqe5LI7yT0CzTv7989DWaf6ySBzi95iPbCWgKft0ex9PeV3nim2vbQISAPxOPCK8i9COpJF75dR0E01PnN690BsusYduv+3aSlq5Xhk6WM\\\\\\\/ZocEWXKAayLMLazwvXoJ3+0fQIA7pLuCs6P4bvOTW+k3j+9wTU0WiYHQSYpYQHeDPAHo\\\\\\\/\\\\\\\/VfAwWV+GLGDJbDUj3jtmQimM9afDisydw3AEZ3TBqHofk8ln0nlGxFIJufYNL\\\\\\\/buH7O0NvknIE8v\\\\\\\/mjZgk7OKV11vJD9mypsiRhKpk53WoR+O25TQpERRVRTuKRfYI1U2kC8srbg7iF\\\\\\\/VaRBD4NS24nZcJNoVl\\\\\\\/zEztoTMWakw8xnm0h+qVWpXE732hgrN+H99zOFRU+AQ30O8fmUeTXna9HuaGlly8eToIRBl8ZM5y66EFtYBbkSuafI4WE8bBjmdpVcoDeNcIpmrg81jYgi4CZKHarrF57fbrCidAlMKg2fVMgsK+fjldI3rYZy4PKr4JdgWMNy+wSd6xpwe3Xa790x9QdxlhUVpH7+HiDXhLUvMjjQlB1x0k5N3hkJ\\\\\\\/+DHbZLbmxBQAYuB8qS71hKeyPh67KXapxzVJZqIIgclGWwhynl2YYKbt0L0LvVa1kLdBCLadlPXvLRtHJbsb2tzyn5HWx3\\\\\\\/wE96c+K0dBnSO9hJaiVZXo0t6hzEorjns7vKPRLkLPD+oBK5r1gZ\\\\\\\/u+AFN34iv4V6olTqaJ64u7XoGiNKvR6YkvfHXdIj8vzpuYnst1sbzMK4d6PohmBVRhzyd9jCYGYuBB4sv7DBjfOYmdHIBxNaL6J43dChcNKH5XM9CtweCgelPXmQz7pw8WjEaKtLME7to\\\\\\\/JeLMEuB5XcNVl3IlpldU\\\\\\\/iT\\\\\\\/L1gbwH1b9rdPsPFQgIuMznBykuu35laT6b1dezVZku7FzE6EH\\\\\\\/cstGK5dSUWR+O+rot+ObepNQeUItQy2i\\\\\\\/8sSiN4qA9mST0UJ72W46wA4\\\\\\\/G37Kp8IBzh2ed9U2NXzX66MIdqIYu8Fkar5QRLWfifpNVwBGLhZVLrsSUVylHpfL5\\\\\\\/ikdU\\\\\\\/2FDS0e51NSoqcokIuhq4vYgqOU72KTbwgUSm\\\\\\\/xL9Mmlw2CqNLiQlTdHfNNu8ZXVpGmS1sQfWjIIq5QCA0n1aiEeuVMpmYjcQ49gvQXa8xCUahO6CfL9GhPLk2drB2xHyPpmbLg94uEbTNQ86DIaJOI6pbZK\\\\\\\/RfG6SY7KRswqfmUpKL25Mz8gQsFpF4\\\\\\\/5xN4oHxYozZazCHz2ROyWxod8kqDsI0KdjlORbcIJ9rgrdEZ4aFjQU7D9OUhOSFJ9v1TN2qfQnwyHITcvWFTMa7TcEMFejLi\\\\\\\/dSOoeEJJh1nUyiwMTTQ9zIiPw6OErNzEDZFdB5ONbPaCXR8MTH6fJetztklOZB2ggpaWJ0GZjHiyCw8JjMv0pqKmjRoR1ukYLg22P2cGuFO3J1kxVplON9V+RNQwb502OSjULCdJiQY\\\\\\\/FSSWVe5hQigee6Bhk59aKkEagikZ6yvyCIE8Kw3vKKxm7sY5nkfjtg\\\\\\\/m2MuTIU0xK+e6WVt4a\\\\\\\/ZS+ntXPVcNpYnVcndvNsKcE8+PZkGjllGiNKXa42WQrQs4H0QOB7IQ7lXoTtYBlIrXfc7\\\\\\\/dmwy2VkgXY4MUmBWq5zqvoqCtIshGrOh5VWMl7RXbfPmL40ZEooGKf7UP5koR1fuIt6jEnjrp57w5eBQ5q02WN8ESxn4ew9WpFmBASE3617faEi8x8\\\\\\\/GZnPi2zXaTlXg37HrEiNOR+x9rBD20mu1vp2svILchGHcGowq4M\\\\\\\/0PdLc9a8g9EIiVq9FzHvLY3vAXLz8uWAPXBCkcuuGJhCY7wpSvIUGk\\\\\\\/zsYzEs9hqNzXOPrcRoSUO9+JbqTGOHnFOThgNXejqS1Cxvk7lILv3vhjEaxFAU9Q7KkNDM75EtcOK1NR\\\\\\\/cW+YXSjoMwkDZrh6209WXZocyaqXcRlIgmN190+UDfxGbQa1cmmdbE3L3OysAll0nFpvr1LEvlNuec3Plgf2q9Aoip7nWZLR1N+GOk8UXGxNUDwVr5aqEslL4ju0Qqg41dPJ8iSFanWDeDeYkAl\\\\\\\/EDBKH7+eG8R9O+XkvQ57GllQ20B23N2aKHTbDj6IJMbY\\\\\\\/9QCUg8J6IryALmp89ipJx0XWjhxIkSK+gJTL7DuvVi1ytrjcL4JWQDxqzy0zrCD\\\\\\\/G7oShUdsddKCIBD9\\\\\\\/Czea24tYTducOihyXGa0AcQ5l0eJpcLqnskR6GpJfeCzWqA1C4DpU19YXtpW\\\\\\\/wwM9XiaYWT9aXSUc+uaSiIwvILOD6RHagwp5cAcBpwh5t\\\\\\\/YStsr6fPIS+rp5GX5pPGbK9SkB0bVMmplWn9\\\\\\\/d0wDoNwR7HMnZm39q0x5p0WBX41y7H306nU1CP7NqFtOPmHkpsw5uKnmC9rDW3eHunWD5BDYuH8X980XYEz0bVHFWAxxNoGy\\\\\\\/tv6udoSb9q9pxrFH6FUCQxUfHOaiGSXKVjjshEI6DyqXS9Ul8TeyHctdlPW3cEL4OmvvrH9UG06CVlD+dm9y4nOpszmK0RieITcO+ZHEShDNgob6GfgOXjDRA6QxV00Cv+euupTvSMokcVpFAr+MEMwccSU1wsLHA4cBBAUA2HLiAqXxHGbhADNVAZT+QDZRyIP6CgTFGMJPFDn5s3m+3mEb5aZIEGzgfJQGeF2d62E0N1USgiNrptp9Adx0Bgk6x6h5+emHOgGDHCw4gA4ifKEh9tAlAk+00rpNdsYaR+w1RVFSdSLQWzvnNb6WGKfPuNdYIDSusXPkMB7Kd\\\\\\\/Bon7aLG12XwheXDoJPos5LROEMN1lr3Xp89NXNdRy\\\\\\\/rRWto1a0\\\\\\\/ioOewHY1Z9xNMqgr+6DcRb780cFXkGmYXXU7sW3hvsElzEulpBmy7ics7m5PzQEWSxfbr5UGcNp2smQhVODEAOWyyd2EAdBHqWGzuwkZTr8ghVw34OCt2uTHoGvCFjuDwMd3ZIPasIDr9FYaPhqUt5LRsMspB0L8UsCySXYv94O63zSTchd05iyVPoys4xQryk+\\\\\\\/U9pKDMcK2xPjdQeafcNeFrGb71AZI9qApiT5Zpp3xT1eq8Yc1llWgO3imHLx9oQNTaoKSgVpWmxE+j8hpURkvtM5Td7Vl7xjJXty8atHXfqB+if7qhjQS8oYLKTPrwkCxh3TSro98MdQWzaa496emScUyHYtyUFtNZdQfpIDqEd\\\\\\\/l+o\\\\\\\/GCIdipBjCnhBIMbCkojTZphtudPDuggeKrKIg\\\\\\\/AKHMaPdALGIjKYRBbqwZo2fO1ZPbT\\\\\\\/Zd1I4PcOKek0TnfmYNKXE8erd9Lls41CMRfiEFag2MRP0hpmOAyMtn++ji9PbURxAAMEce2oE1l\\\\\\\/Dk9aqg9QUqHpQtGl+FMR5Db9RyIsdw9ofN2MwPkEguB2yfZg5ldE\\\\\\\/+RNORUyesm9DzIk5Wo7idYbZclszUX0bgCwypJeRTUKM7MIgSG7cEJ1ZxT0NOooxjtEFmfrIq5DfgmzZjDAkmqmM8lyTNn9upDwg011z\\\\\\\/gzAbhoBa8guCGIM\\\\\\\/XubJPdpfI8Gh2iYepl4fgMo9MNuNIiDAlKdfrhYMcV5qe8\\\\\\\/TkJ0N6vcWkukPOvBbWPihzsaxNogl2nynHuM15bgboDsRs2CRxcx4eEAU1laz9EsY\\\\\\\/U2VEnAmZCbTGamNA7qsdvkltCtSEG4auY9WrEL9T36oqCXQuJFenCz2fDPvGVfV9sU\\\\\\\/wcmnQwL7hp506k8uAkBs7X+fDXvEx1BootLdT\\\\\\\/ehwAMjGm391QeAMIcXW2qRdTGS3gUFjAsSfKt3Rin5ReRjZTczmSs\\\\\\\/Xf9HnVE5CL9XeTzmf33htcJ56qmRVldd5GQttHK418N7TbhWfdKOJZad\\\\\\\/YP4dVDfnBvReuJmAHgNXAXBJhv2RaQzHGD+INC\\\\\\\/vBoj3C71uQGIp6O+CTlnZy35VSmnXVmw6MTwuIJRf6rwtpWUI4xhKOMDMiRil6TtgKvygsr58AszhzYJ+hqZsdulQew5WQz9Bwf935FCRbmvhCiBZHLemYitqdb2UVpCXR6l0LZ8dKcyfTysrst7DDmWE1jsXBj3rWsjmGp9S\\\\\\\/MhzZfeJaBlByCbdnJZy7uHhZ500FYtpG03CDiChBHPCsVjslb\\\\\\\/V5f1H1O13i1Prn5K69UUrnSsPOIUkvi9ejGzHg14cguQKJKer4JqKUYERG13DPU6\\\\\\\/MQFKNuUBqBl7oJDKKT2S+p3T3Ia2Efg44z4uVQ43PY9Hch8hx6mElJhfs\\\\\\\/T8KlkyCmlLaDCbPnZww1tHIOfDtw\\\\\\\/105N\\\\\\\/5xah34NoE49fym0JsIKQ3n23o0bd4vr5rPwz8OCYEbqCit7znV5Ue\\\\\\\/zTQ1lfNAojaWf\\\\\\\/j4hMluntesEFrWULAdWVO5pBpJj5tFj+qVCmOYp2EUGuGDnUlEaQvhl5M9zi3b7e2rWUqY4YtMKCC6korsu1qBqpd0xWZTBWaNczUJt6Sun7FlTXcQX4tHdfO9suacWkjSh6Ho52r69w7ulJzyQ5JXBLvuyR3Le4jbrUh+Q8Kd65mawtiJ3i\\\\\\\/RaAdxNNCNs6zGk0ow8fzlrG5hSpWrGe9FGXnj54GP9f9vWDtPyDjCDs2st6K4qyIvyirUHjCikfs7a3roJGiZeJ5bK1dXHn4G05CYVd6\\\\\\\/KHhwxPwNQm9Jtt0HgQpydw6gjcMXZ3k48NPReFehFYBHZDVhcNzzmx+gewhVbDYBZZtQKO2LoJ46KbjAntQrX712YAKtvSZ3nTHmZRET2frDxACgHu+G6RryBJ6KIlTx1WbCwkselpOrI4fW6qz\\\\\\\/VKenR+XJDyWaTJqfdYatrC\\\\\\\/LRCH4MH\\\\\\\/xslLGHfg7288UM\\\\\\\/vYhONk1RosWLRtmraTDaNjIU8a9o+4SNOa0q8qFhCVGx0Z1Eblkpq4XHGhR04RcjGOZCYpnAGMCBzGjZzHuA9DG25GrPUyk1Jzf8aw7LUWPrjoxSLTL88ZXr5cLAT+M285FXOcwsmwOa4SC8xfjkaqqNv799zOtDhr9iUiaRAZdtz543U0gudJ97oylia8iBN40tGV6TF2x6YSxgH5VoqYkTNpc5pZBnxSeiAVANEqR3eZtkwpPUxV66CKNSHrzmck+TFXCu0rmuO2ruFFS92wdZbfSTrHnM4dSwGzAUubGPJJZP7NnO2hILZqMphdesh0ybGLYg9E3OHVrbp6Dui4DbIxI2GzfazTPsvCJ6eIT8TY7Gy\\\\\\\/TeBd0MwliJ5TsqbA86wF2jJCkGehpLKVU6Jp26b11CjN+vrjob4phDiopo09RBCdaGkCaEsRn14M5WbogiIeYSx0glozMWfbJttL3GAg+4SLpyY\\\\\\\/EDJIOvZ8EvRnvlyPc77aalitgbOznEpceqoOkFxdXyBsKqNbpiScHk95Ku1bsT3g1YGLIJ9U4A\\\\\\\/2I88oFFzQGPYJGHs9wJLoq0Q\\\\\\\/mwZvbybIi9rEzwEAZsb0IfRMZE3J9PP6nQm1WpZK6VKu4YSpF5WVQRdFGBIKoK2qzHedrjNK9sdoAMiEYXeDXUwx3yPMoL5vlbQb+euCL1ZducX5i6WhnuF6yE5cRk9l2i5pGinZu54lOaSc0sbMDydIuuUriXcwhRdKapg\\\\\\\/jwOGhIdTHOjnJZiV+PsBtA3LT+fWVGIM3kPXIQipRvlxGko5912t4SJPY9\\\\\\\/TqhAc6QtwIDMEzI\\\\\\\/YC5m2Ie6+mo70aGUvj8Cwpmr4xMGN0GynX2HV46gW9cP4H0+gMJwTD0Msd9k85ThcRt+tq7QHk8TXKg2jQUzDfAN1G1oW\\\\\\\/nORfGsJh21oySWQbTmU2t8K9IGa4orBtrMQq\\\\\\\/XAntOs9KjcBeTcWGgIKDUu1lU5JpWZlIX99dUwte4\\\\\\\/WBtiRNEVB+dMMHGzEKYlFLL62gbVfRnGhZKb6QvLcpH+GGuzxMEuAI\\\\\\\/eCKFKNx0HYQG8TkdeeJKza9LKPh8pRNhXATB466FfsTwYPCm2ieozYeTb+AvmN4pfo2Of3YptLS60UM90DoYhoCytsvVoCgv870cGwYc4UuAtx5aF6NCUkQdsyM3k\\\\\\\/ad45kWJ0YILeubsBbjsWu2mT7qA6LhEdTj4xVhPSthVGfcjSAypfsWcrtCPogCwvHM\\\\\\\/1zq82EQ4hL\\\\\\\/GTx9o6RBPM0QEzYIGjUXIupMRK0xwnI2uWoRfBPre4b4jBhVvMsXjoY7jwjmHk\\\\\\\/jYKxwCDrVNHHUr1HZ1EcbcNdVnMlWOqFWmdnZ5ZehOMW3SuLdMctwGe02r4ZwpvmSunzKlG2QFchWjDLmqV86Sbz4qqT3kaUO2M2VcsAk+bAWT+tiwnISpXbcLQutuAW63F2jqPd+DOibGphGwYBH61juB7TIQUu0KANuW6adEtrzjhKfzPpTqhbSvqx\\\\\\\/+YdnJxAZg6FfoaJVSQNspIcnEB0UN+Xh\\\\\\\/YI\\\\\\\/sSks4GMdDwdFn3mDOM8e\\\\\\\/YH7RHK+1Gx3rECgwfyZXEdtZIjhLicARhNO8s0LPSUuNdfuHvELQ6caY7HibD39jqbwN+3pzZw5jAN9N0tKjd+4SKnqHzBuy5\\\\\\\/yjkJEzrbgc3CK3Ooir6lZVQGKAEDfHOYlQHwkgxUREvn8G6BTy7hyabAml+6ujuBsmY8KDorAkOEsa1XSmrqDOVeyJfzC8jyzPXhCCHVTfnu\\\\\\\/Krz\\\\\\\/Au1BqHsFhkBrC4yyGYnnzEEdYUCawqUGO1YVv\\\\\\\/xk+k22riiazLHB9rpvjIX0b5uB42AvR8XwfKXOz7ZWCVIriCNWxATxF+so8q7zDXfrMVFNhEd59BY\\\\\\\/Al3f2QdU9KioghFOmovbw3Er0KiBowDnzft0QF9C8LI+AFxFjyEcubsRDo\\\\\\\/oyZTssFRPr1sD2x+0\\\\\\\/WIdZpfYj4rZYWr8O9+qL\\\\\\\/GZdtwK0AEhJICbcy7oHRtTwzYgZIRRKD\\\\\\\/busI2VyS+vr500Q4I\\\\\\\/X6crs\\\\\\\/9ZLPYv7aGkv8SU71Xn+6mW5g5UuD2QRJmHbfsrm32wuo+CfGo5dMvueXS93Fn6kiSfjB+PG2HWVTJ4MskJY1eqRy2\\\\\\\/geUUUzRT9jn8uVxwwezoq8xs65h0gCsb8yBNGcRpDHqe1MZ6Zt68GPRtaOFmNF83hftq8oXUBqgZdtggzWdEFFhAcgeNWHF1ZWYDLJkplNLacLft9WYf88mW7d0KX1RFgqOm+mphZIiDCDpoKGoEuur1mU2VQzKxkz00y84x4EhfeItXSuDF1t9pbFqk06A11GSS8AnU58PhosLl0SJrrOqKxbw1xE\\\\\\\/7BWjtIwYwl9SnOqZkQ\\\\\\\/gtkc+l0zwlmhs0yNq2BFjaxkoCrDCA4UM0AbWF3gx8Pc9BzjxtIPweMZkANkm2jHvpk9q9NSm59tO\\\\\\\/DbOhGClZQ5l2zFXD0BT0BNeWQ42Gf0i0ZybCeamXh7XRobzEig6JIsou0dny9AM\\\\\\\/wA+Qocp6SDCn82+vyA+A\\\\\\\/TJPIymT2K6HpzvWHpzIx407mSeq4YYej1LuW7vlMfE3CLyaw2RpwyIBAZT4We4XkXuT\\\\\\\/SUGhCZlR+4njpJ4glmyiomXYELbv\\\\\\\/NBpmF2+xlfwlU2DO2mBmkyKcj7V\\\\\\\/T\\\\\\\/cMnPFTYo0orzhKbmOnfHNLsVBSM9JNqEygZgUswITLVOH+wwQStiW2Q22om7wO24DiLvImvUbq5U8IoefiT9z1zloBuumQfi28YW1uxBJM\\\\\\\/Hp3Ab+utqa+sRPoWJGOX3xrBf5DrgJSPdjP4vlC3Hs+ElxxcWUWKG0ws\\\\\\\/AuNM7rPXDKnFax0Zyx2IfrpJQsVa3QJB7ds0BziTAsMRpJ5dQjCIY+dVg9ifGsyBUHBkLB\\\\\\\/NvedlKB9qNF+ffR9fOCMnW\\\\\\\/QhnGnCm1NjcG3h1djFn448yNV58\\\\\\\/7EPoyEQyANl3xrT4WNL\\\\\\\/vE4\\\\\\\/Ze1nU9RGKPqJ5frW5fNzFLSJR8xo8Qz0HUngSZD7SE7jDghKNZb2m+ZEb6OZZROEmk8wJki4k9JLCY+8f7QP9dWNFi+mePEKbbM8QRjMqug261xcYcMHfvKN5D98IDoJFJ\\\\\\\/7lp53vD7JnJPym3kEEyp4CXtF49e8omC\\\\\\\/ulLlT0RXd6Do1T\\\\\\\/G0lYabd9sUmTGI4Uf\\\\\\\/9tPVFJN87skIJwbwgBrwo0hj5Uxpo+BQUCm5WWmAkGtHfg6omVg7g5Kp9QhpqnMwKIfWFRfsmCVVir1BfSupW9OqFtZ88QuHCoLMAHBDAUerf12a8tR86hWyd5ixYDGjrzyBnN3DeiAK1AB\\\\\\\/PJyVH+3Pd3iETNQ46hNIV6r\\\\\\\/iRnnFVicTfX6q05yxsfzMGwNjQPW60EP+Cj1Ix5\\\\\\\/f1GU52hdeSe6g1qYZ\\\\\\\/dzpANBGuetlXRMlfmu7Ig9vzSw\\\\\\\/7\\\\\\\/osyMnYzwnlNaCDIRe5bP0BIG2kjKt2efiqysQCGcREjBqV+TtrJIcat7E20tO+awix2yzn8A5uPo90tE3QwaLpA9elNAlkVUnjOUbsMvgrlTU2t0wqGc\\\\\\\/cuGcD8I5vKoBq9SlHxz6MF+K3iHbxJXhJT1Z0I3HTyP\\\\\\\/2ALPvMQopphxqMckkZVb5zzdg6sVDfeRiqOWqMcP6t0S2\\\\\\\/I1t8H3w28L5oes2\\\\\\\/M8u8uSr3YHiNts21BSjfxPxxr8YijBgWU03MJYvpjnmf8I0Cv\\\\\\\/AZJ1HYBrVPCeGiTsbFhWZLrTaYvWAu1Gji3QBQ5f5Q3S2jdPDFqSFjW2f1sKOt4iyssuJD\\\\\\\/L1Xt5V\\\\\\\/jo9jKDNBTt+79BcmUp36mMkRX8wWiCCephlyPnt4UYoy1LRvSYPIV6JHN1bYSBJSDHaaVRzsDDctCMMiuunwDCMMwlloZOeIMOpbWNG4m5397K\\\\\\\/5lePd39ueFaCSrh1Fx9DmvkdhGnZnv+ltFPyijMKHFj5DSJ644Lqn8WD9P6ClQuKZmOOVIPpGfcrF1MwexgeAlijfgOT02rxTCzouHiEv2IOem6ssqiBhxLaTYSqyhk\\\\\\\/IzksDrFuM20Fm0B9au5MqT6kpd4qKZ4XDwJ1h0MknZMiqBrluVmrg\\\\\\\/1A2nM8htv6zaaDGGZVcUVKV1+GHcT9ViHxGFhRcGc6R6aGPY39IioUQAXv8N9Bf9a6a86H2fVjmhUGzpon40uCmzm5mvvlNhylR5EbPUPuj8aO0\\\\\\\/DcqMM9oBY31rM6YNknoGK3FwaWab0ufPzUuaPHH9LsxEPTBFUDERJjmJLav35dpO6pWqVIZD7aAOcb4YbRL\\\\\\\/R9PmlIR09DRT+3FedW4Xv9eWISqFsMvNFhL5G3neK7\\\\\\\/vex0JlPe\\\\\\\/yxJ6fZ+fHwwPE9GbEJS\\\\\\\/rpTvI0osMbBV0urB90Lk3zWaMS+DEDK1TG0EXwnAosAKr5tlxdPVYGSOEBwqW9YQyRbiuw21jo\\\\\\\/Xp8AEHjlTp5ARpX7Lei97+0ZpdQSfxEAAxRha+Mnrnq+ffRxBEHxDtd1CsUtBixVLSkaKjBNUwZS6IDY6EJJjpJXjV\\\\\\\/1BAgl0kOja38Iz\\\\\\\/C489Z+fJEPJpXPYdyrXlfbrKk9yxhRmztGZi73t5Zb6PmjlGW45B9DutnsSslMljwSqIBnOCBCZKriKpLdtNK71RcbM8GNS+bg3ave7jkwsVYKz09PFosXohBMuOtYu9YWH80W4QQuB\\\\\\\/zO1GXGlOuT5eiC9jJpAiRCfRAM1qS5k037RIxSbSPq54yXJjlctp\\\\\\\/dlGdw3hEmy5q\\\\\\\/dKm90U2rgfNT4vZES2Gp6OXKTg5yByPy9Oqin7ToHZ93b91JP0d3bw48Yuhf0TxkN9ipGzMbH9XDRCjcvPIW20K9lenpczqKZ+SWEVGRf+cCTeizjRolMp\\\\\\\/B2a21TKDnomlyrwOspH4IgPo2BkqIQekZpfiz1JrrbskfBFomoTdw7fYSTpBgwEF0q9YPWc7rAW5WwI49Ahjc7sbqKMdtf2Pemjg\\\\\\\/bDJfg6RSFm3P+6oHGAlqSh8XuEJX0qNS6mm1u6C5CZg4HbZziIhtxnktBaEgRrUiU1lhezP1LwjN4zUWluJd2VE8oetXOIgytHgdmBj\\\\\\\/pi4SazibB3Z0\\\\\\\/gtlwAlJ\\\\\\\/Ll\\\\\\\/3N+xxQGDQbPN4Ws\\\\\\\/1drTgnYa9lJ0OvfVXVBxed7AncO+wzZ3b8mk3yHFPTwgjUoGJyi0rwAnm7muY78GZFvt7OEXf7Lcf22ZnXhvH6mv3EkcPpgIoXcUMQJmFYz7D\\\\\\\/HcRDYHeqYrhHbnP8T\\\\\\\/sOJCAlLS+eAFtHIFkqlmxv94dUhFTdnDxdINQvnFXrfj+wDAs+Q+vyBk2HGXCH5bxEdWC2L2L0lvohQ+AGTMrZjlmtTqkQcUQTg1QdeH4V3aEXJj1VBpsQqLvS6M2Z0WGMVgo8CjkP37rNGlcIfHNG0nmqJuE7tZyAAQz\\\\\\\/tGBWiGV2QGU3S2bWc8k76PiGb5buWy\\\\\\\/sPtCSkm83QaTYD37ZsvODfVd6uHhiVfwVp0F7XnxEfbcuYyxkyAs9Gg7uTNESSpuQl\\\\\\\/c6yJMS1OsGPjkBD0TQTX9T0aPbAgmmj8mvxyhtQ7LYn7bn4yZP6oxaAG\\\\\\\/+lmTQGzAP7wPC6dyY0Yfn97gX6ktTiedXx0VLfvbTBqoSnHyG5FRhXSznOFCzreT9jfkrQ13\\\\\\\/2ofQndWu9rXz\\\\\\\/GwGfEKGUz5SyQwQw1GAFTqyZrneAA3WIQIDfYVv0yKpk\\\\\\\/dogL7hzSoVrwcH+HuOy++cMgkBkNyIYLOTh3qc3i5tCAJ9oFdO8o6LBDlqKaKKKlt6zxpXVrjFhDXcU+HVDRahzmxzqa9gUi+cZwsaVoaTQ69FVOWBuTiEDNVuKBUHFkugTVdsDZZY1uxgofLEJEtINbTQEN+4mFhzaROwc8v1B8ouKZh\\\\\\\/VqI0rDaEPaTZgyBXe6u++N027s0Q49yw8OtXHUUId7Dj0+HpE5frpsmrjnxEHGADoAycWsLeTnfjA6Q1hzMHeJuRj3ul1L\\\\\\\/fs4jfvfSFKJHWB+v1O3DiSgac2UKfTpBwA3LLfBqORA7zJrZg3al2TxNMyAVhGnDcwyPg0QlPPx+nY0vbUHciDMI9C6FuyeTGLBD8uQ8DTkl\\\\\\\/VFtavAeHLIp0QoiNyUmBgKQM8Cah\\\\\\\/M83K7RHwvnPwRSad2eugMIXPVlhDOBjojLoVFXIT2Y1qXx1Y+F1UyZZpc4sMSpPEveBG1TSfjreQiUcQcMz3oL5gkHQjxO5OcOecAjGRDRY6nIPK8O+8qHiQXsvYc6ZIZSVPYneqauGLlL6acFuRxkWY98i8BWGcoHJ6r7W2vpNbyTk00JMLYd8XbBY9JGuR6zQ5eA90\\\\\\\/L3TEZtHI5hJm4hxgK4z6+Uo9XQWmWAq1c1q2iOXXNU6Cptaqt6sigVfCoagTUU4Z8SugJR1R3ahMJPEiu5k9heTDVbsRt6m8bhRjyIACx2Sl+ghDkDsITxt5F4SlS\\\\\\\/6uoqn3tKrpx8Na09HbvDXGKv+4fnISbOfeLH9ZMy\\\\\\\/MF3wUqNdB058p8j2o7pohtyMx8tZb6y9H1kjetlOJ\\\\\\\/vMHRoRBoRUdP6tPJLTSJW6qKm23odqGqpPxbiYyVCibZamruGUdBftqTqL1SHlKk2Gnn6uwfT0DYCiGTorzGtDgLNY9KapmoHFk8U1OqPXlaSlXOiI+ZrWKQnrj6vEC2r40ZO8vuEMFbGsUXCEY8GHLF2RJ\\\\\\\/eAPHXH0etAUioCQVWPxkc7XEec0nTxcl6lEhIo2ic6V3Awwxbe\\\\\\\/NkXY2rG\\\\\\\/g2xUaozuJ7paMW\\\\\\\/7jd1PXpzl6HiR\\\\\\\/gz7UlzV9\\\\\\\/7gEQF2arwd4KU8gs4enfFvyCwzrb1UBTcpwYd8IK141c8qVJ2UIs9B+GnnoV\\\\\\\/6fHVoB2mIBYGe8kqp0rY4\\\\\\\/8iAkEkG4yuld6KRpJOVZdymDj1EgfzfDVkJspsoYOp1kOEbx+rzOwOxK8+zd5i+ePcuDv+LMPX\\\\\\\/gbtOHOrpbtDYa1vc60BfrlK7a0kiZfoH2bPxjXwXgoHgt4SFDty0W4euUG6VHOnPhIhMSlcTYHGLYFCUoF9x5ofBCQuuYVbc5dgDTXbmxc6s\\\\\\\/FiqIhgVCby5EFJyBk17NAmq8Tbh1ACv4D75lkXUKRuNmMNyYZIyJe0sL7e0XMg9O7NBIwzJcOrl0I37adVPPFLM94uupefJiEdWDnlJdZODSJMvdEDKIh1qqTCNzZYY5CbRUQd+wQ6HLLIROofPQ1nOg6JjCtvU4d+NZnBVviEHzUccFe21FOFhbWUQ29q4RCw3019flGy9gN9kcrvqe6\\\\\\\/R2dAfO17Xfg3VFsCN8FrrzCLMlpaftmeU4waFdhIW4bFAclsEYgIDKcv3v\\\\\\\/JfLT\\\\\\\/ELd5vzWqsGa7xeaklhjmGIVi91c8Mxwx\\\\\\\/no5kod\\\\\\\/8n6\\\\\\\/Z5kpXxgWroWt1SYZSQk\\\\\\\/AGQFcpwFVWQy06z1+W3P6cMVpqK\\\\\\\/bjpLJAYPV5JQ2B+l7vAU8bLuhg1AdPyZHCOQpUJ4JXJo0PHATqPQekD\\\\\\\/DkVlmEUKg0eaZGBNB0AdK1dWgknNOUWKFQxKGLUYauRfutcdlb1Pe4VtZoBtlnX9idr\\\\\\\/zPsCPnG9h57GdhfLat\\\\\\\/c1WkH4ZrXFPPHBz2EDshGrNZLmAb7B53227ivkXy2aYui4EU8ZDhCHZeF6c6kHWfEAta5dtgrz+xaG0QHzRL9Vx+k9b7NYoxayOcxM10rorLaQDFT\\\\\\\/9bsP6GART3vJiGXoq6Gb4\\\\\\\/K98EIUYnqng4kWthL4xFWgYyzl3CRSqxapYbpQ122RoWeINKbXQll7Pk4qns73RdmK1c1hDBOTpH6b0NetYseNYAiMLLdiddIBVlayUsWDTye459sUaq661WaLU\\\\\\\/S0vCPgF0IHaColw1E+GAE4D4mllB8z5lEN7hap\\\\\\\/Uw923tJRL6JhnSsvnNEmmcL1yA8oyyXpI\\\\\\\/ZSmre8cCg+SsT59TKW63FcxyUIt5e0yd9H+yOAwWerUwefKIi97QhKfqf3XCmYVHNn6ACWeQmQBhm03TKCCmlDmXrDytzswLjoVuxrOpYIHT1ugdQYu1EcmnS1jmlLVKemNG4ivKyEX1lZB\\\\\\\/lPqYr8t9KUm256qyn7YIKvINKEFXXx68HyUaBSGmrfI9N\\\\\\\/Yn3D4vkB4VzdWGCVvyf6CYrp9ko+ixNYZAjo+d3nN1uV8EwVpAIVB+jEjtGbxjIjST3Cn85tehpCrDGO8SNM7i5tvFMCk4vO06vX1XKcJmmsYSrIbYyEhakPQZaRAr1E5zsz23AgdNOsM1qjHY39OWRJ6JUtfiS\\\\\\\/nlXPT+TBEEnddIUUPSY8ddCW4wSkTPtx\\\\\\\/ZkqadZo65q3a4lPfJtCxf6hGXqX4i7rL4903KGax1VrHE\\\\\\\/sVvwLGahiTiCRWuroHhbocAF+GyBzooZadKHBmddyAksYAZNcMI2\\\\\\\/xxStclibGM5D78VOT\\\\\\\/Hej7GEcRJsh5wzs7AENh1U84SpNot6ZV0emGM1qmflvsSaLqsBkGgbuboE7kcsnQxJHviQ7gk0KF2Pht8b7THdtzdgRogaBJe6OI1IQMU6yQfdvKf4gbxuIlmsIC896Dt9DxgCQ0s121EcKdZQYlQ2hW0gvzn7Y8OR5PRWlC5OCF3VVuhInF9hUbK0TsO\\\\\\\/7CLWcr9CmdgnRFh76EsNTQbMW0fP9Kc2u77tcF5cL3VqK5JquLriVc7k9Rmqe8V7s7vr5nd7jQ9tVnr0t4KOZo9kPlmGrr0sPcEAsues5Ed7d6yOxe8py4ByRTduZA\\\\\\\/hd8YMxc6cN4L3TKoAS0MCG2B\\\\\\\/u3ase+KUYh\\\\\\\/uF4r6VnoKoEMt5FBqTvIlZGmdaoTIElbW7ZIN5fJMjwIeq5D\\\\\\\/xAmX2jFtd9ft49jD8\\\\\\\/jtBGWd00ifIDP4+3waBzYJvoAOMbqNpJcyX3rC2dtKBVLxDWeQFKJVw0MlVjlyRhaoFBRpb7hotV0HbHDZxkT52l3dnZtf7y9M+9o1FRCEUQyAdWE1YWAI9Be\\\\\\\/gxF528z6gEdGPm8hnTr\\\\\\\/SY707uGhPYEHMYtVtbqJW2Mx25SYFL5SeLudfPIdCsasYe0KH++7XYgTbe+07DjsZ2TehOXhSBt1CArK3tz05yKbbeZovJnwirBw1PuOkU0shB9TR4OzQT9C0agRD7JvPA5ik7WvSUxCaES4WaOtwFcfxKKPYILX6DhBWJg3Avai6uSRiH5OC26QjeNkIdazb0WPsyP8q+42issBWIG5e5CVoYHVIpX+m2JaSwcE19V5OFSUqm8JXK1tzzLyEpQZXfUyu0miY4lA5E25gUTVFxUTQCRQ3U\\\\\\\/GboqBsWqw6jkxSfJgPnNRZAXMcBsZiQPzNimmEbsfJyJlJaVPLe9vA0GPeSys8LWu6tHzMCOAkU5BabtzQCJJY8EBod6Z+hIaWsOHxNRnwHUS9rLhHBouhqMhKoDnRMIsEK\\\\\\\/UYUmP\\\\\\\/ebqv8RGia1iO7ua8wA\\\\\\\/9dkGX4hXk1hZB7qQV82N0cGIqHbd+sgA7hxwuZuQIQ5lVcUegtnnXkxrlJw1QQ6rQ9h9oPzDJdVkjopMiKjKTzj3AY7JSN2F3qp8Xut+n7+wMK8lcjg0sD0PXH5nnTpwhBYM36ZoOXlGJWqdyF2\\\\\\\/ea5tn8K3pV8qLSf1avOJuLZ5patMVuBh0ElFKHwTzv9lEbuCr8RZ5I39XN\\\\\\\/IOcsW0a8jb16b9DGS42VJsW5XLQ7jJQL8SI8bOXH0RjneMk6FzrNSU76zzheYyw4oQFZgwshIplg99OspMGZH2VF77yx3i0MRhtmBCeC4O+HELOC\\\\\\\/EYcEl3FPPnlgVp10SmxXQy97fU0f4Br697W+w0tHdlVSeEgA7T\\\\\\\/rxgJYuc9aUNefGwsAlVdBwLp0C0KkNAMIFtifIvOO1DSNMRHhZfGDnRZ2tlyRILBcdpDejO5VBKR+zcTuM3x0plvJlh3RUVKMx8wa05YGizo9TTdSzulIVUtfn+kZ1Ag5LdStTS+owauvgcFius2Mr1T4sWDU8trGNALGq2e90R68WvqDhg\\\\\\\/yNgOxd0fum\\\\\\\/gnHI4tdLh7YEoHcqayDeEjZw0zbsjGxkB0z9oea6kGdvjHHJQXhbvFPqnbNEuLUfND8K4Y166tNfB6T9lua8geWqAIc9KTGyxwpBZKCQRhqtiPe3UluniSi7YRT+lMUObzm2tofaV4hIBFzgMKUYVihoeC9HDY9GfjemL4pVHRV81WnR2IKYbCTFZ53DNgMi+Jn2soCRSJts\\\\\\\/iEwJlInnXTSoON2JRRelZ7ABYTjcmexuUQZd9r0Q1ZFrOKn6GPbrSzWBXaGTN31onImyAL+okvHS8xqxOVOzfhspRSHSBqGSu8RktT0WcQp9D7GnjHnmnz2\\\\\\\/YqCbZpSNRCpFBeR3Z88fc\\\\\\\/rivurkcmAZOtzbjnz8aaZK4R50XdrM+Xds4+WKaxt6p0brCYNNz9CwT1UiUVRtAjdLIi8Wd3itMmlpMh1cvUDrj\\\\\\\/CtVhoUY12010Xg33phwylDl7E94Hipacf5cExdV6zfDk1v0oitPd3tKoffEKPQEQU8AuWN+5AeF54MPCxGLsxg9SZbiSB0nZWRiCBPc7YZIa1rf5\\\\\\\/WbdBw4ShbI9t0Hq5Ka02AsWYJ7tpHEw\\\\\\\/uQviG\\\\\\\/dEJ4\\\\\\\/Vyj6bHsMZo\\\\\\\/4Nz1h74Tg+gWqtbI2qHFl\\\\\\\/wQUAY2PflI1xYgR3HO0UiTpJvzDtwR+kD4ctnXig5wtgM\\\\\\\/ajIqchpBygGC1SPiDNdaaZuLbheAB\\\\\\\/ypFNq0IqzMFsqxh6oSGIuJzn2k6W7i0gdPyROky+RVOE2cbOyf6A0XHf9NHsmNkdloIfgSHp4J3KF3m6os6zDktT3jsZnD8l9tOCcesNyZbNYyccz9eRdehvpzrHGIzD1z0Xt3aeqNJYAfSpvyKJZ0Jsn6UfJmnGO5VQ7PXmbWct6t4zHG3a2ba+SVCxFekkCB3iC2F+rv4AFlaa9rEgeAIBSs\\\\\\\/p1DCiRqeWvFO55kClekzZejzQMSMUbPPgellE+sMBQl+6nZo\\\\\\\/05vNgZVO5jq9GPeu2As\\\\\\\/uIo\\\\\\\/5Bik6Ks2jCdXDbrSydQt4oZC3RYDOAHe3myg2NMh+wOnFGIKREsAoQitPZtuiPy\\\\\\\/k55uHAfQxaMQHbRVnBRphR7K6WaYSKgtUBRaF0tQmIgnzE\\\\\\\/3WIe5ly0CNlNIOfbNXlJW2oQM7ZcsCBclZBXXFOpEzCQ+Zl\\\\\\\/u9LByWL+\\\\\\\/Vzplej2X49\\\\\\\/H8Ix4Sfl7D1RKy40wL4sCotDYleHhhJgWs1FQIp23K+B0\\\\\\\/pQaIVv2j7NX\\\\\\\/hChENcFAp\\\\\\\/QyBeoqpQz+9oNW1P8zFp+KfKkkYp6RCq6+4ElsT8iPWKC1xHkESXjsppXVNvKvvsAvXKaeNSVOVtDl0CfmydQJeQLeT0f\\\\\\\/k1\\\\\\\/BipuRylZjs73ktsE4O1h0CzNXaVEqfZvKeIthT06MqWhYYcCM99SwtpJdXxTTA0OR55SDMtyCp+3o2mLApLSKEuOZ4dT1dIRBhztuozuEfzXC1imuy6wpu7pF+2rqL0OrRC3CKMi8fj\\\\\\\/sMc6nHr2hoA4M4z4O2AzbsxtAsEKw\\\\\\\/14rqWSkeliyNL4Eaxc+uFJ2jdSwgC5kMiNLUy5ucEBLpyiVJRg7fgvsxDQD5pt5u6XkXBRRnDtgLnjdryJZ+MBhN1lfu8qHvbxzWoSSM7HM4OJoPE9pFe1q8Nq069b6\\\\\\\/i1APDGA2e+z3vYHS4cvcXAnVMjCogVOOnZJYyW1nuM\\\\\\\/4O0KmLQ0VLypfb+xRkkcdk78JRIaYoi8yYdEwrq2ku3zdadpb65Y0XgTtg16NCe4eR3G4f92knLEAPc2oV\\\\\\\/qNPLkmMoaTu7MtOS7+9UNOzBFDib7KDR2JYMb6m8AJiH3eXW1F3jaoHXW4DOSdoMjqPrve7+Cs+ztSF9rEpfUUXipi6hsl87sMfYyQNNiIvOYevQMPIp1kENHxkFxeo3XzKGFDmTeYYLH+HcB9X\\\\\\\/025jdCmg4voKlmp548NerImLPvEzAfxgsLxKwKoOnS6Hx2uKcxSkcwFF+iwU\\\\\\\/zoGbyqBBNvSYTqWrYx3nz9FRj4qWA9rDFqvvIqm3iKFoq0n1golW\\\\\\\/0iYK\\\\\\\/hL8Z4UnzGV\\\\\\\/1rmN3l4Btzpz\\\\\\\/Q4wcALFF0Z2p4KJvPF6KqhvnSVF13Ni8fCK5ZxVsfpQL1GAzPVlYSAR5Qsbh7awFjA9qB5V04nopIvgHTxS1WyMNdYW+72VnSFN65rsOiW09eWL7kC8JnZ4zUXLq\\\\\\\/Gz++h55WHbHbEPaErOK9D5lHxf2KibFzl4xJWJ8EVmQNzwB2l3Su\\\\\\\/t9xCdjSjQ3D\\\\\\\/\\\\\\\/TjkiZnEoGSvjCXGaLqjSlCwdjfzgK+uc7rFwFDD15TN16l+s9uKv61jYGtVikR1WQEpcrgMxJbCrBLYBNpslN4GMHDGWhk7Y0\\\\\\\/skmQoSFnbBIY1GILyNivMSY36dvJ8g3FQyO61t5RsFHOrCyokLOqoKr34eNWX2wsWQrGOl7K29UU7WPcWoT4Uq2suIYgHeCX5zoa23ump5ZjRDN7gnBMmUMYVYyI1FpyqsUF2GKCwYl5VRqmtYQmgvSVHnAerzC41rL4XxeJ4LijPMJ7B4ZPJbde4SbaJJle1cpD9trGBzQMuNMf1z5ysRBMVfC0wyufn97lVJz48DJ+7sVnzjxCvkErv5p5CAPE6V7VFOGFp8yx1QaQiVJi0s6H5W+t+ayqTTntRSPxb4IbWaR2Nt8orzbKajFP9AOPsxBRT40VzSwCtDXTE6cQ3VBOYb\\\\\\\/vDKt+CeL4s1AIteuTByVM138YS7Qnf1Be0qKfPUqPp6L4av\\\\\\\/ePbLuPuMVl\\\\\\\/4MVMDNBlRCOr3DrWlAj4QbQZJA0IihzrFJ\\\\\\\/65wqf9LE2guPVTT\\\\\\\/wIM7if8y1Z5bNWBFR7AN5SLK0WdRCY0KjhydaGwN2oGdpobjbbgEw5FQr13P77LYmiXHLbJNUFh7FvvNy0MzAZ+95CjTO10SMaEjHSYcfDvjEUXFKw57+5YEQMlZs4sHaVUtq+rJilODdeGTnM2oWoummvS+ukx8hRjQDsrV9Ragg2Aw40DOzz0VLzqES\\\\\\\/R+4MIHoq9chJzhhBGlI5MGdDOqf4OGfCStZVFZtCdPdA6Fq5gbTWMvXFTZsMr4qEh1Ludv0LlnDcuXrb5sF+H4JmJklVAZgsuikeUB3ZmSakrf6Tto4F8jDsP4ur8AelU+C9ErPLE6h07jcZwgudFTexQIli6Pw\\\\\\\/pSdX\\\\\\\/I9q91Aqp1GjPH4epiwnyfp29q0RiDeItUh1A1PGGBF+JSzSunhYeEfiyYgghnzNyq7pl4QiDQdk21aYSc\\\\\\\/vVLQSJJtqMK2ODWOlRKztGRw2WO\\\\\\\/02te4btCHetKjRnBKpAmL9CfWZLcqU8v3n20xq3NYH5RzUJpWq+prXq0rXKPsTFtuqIF4skXpbN5PfSxcua8pU7450GZYVTjra2PG0E1aaPZsS0iIFn0+EJxwxNn9ZNpYWzJb0V5EOQD5yroN09twhTtxP2RqHO23mQPtxQSdAAUdcNMoy9ZPaWbGvhZWZ5zy2msuUyDkkUpeJcVPg1ODkmirq9N0JYmYc50AXficj1keblqhvYRogUDvA3zooof2HdMNs48RiqH5NSr7qKIeZTxly9zkm4OVIuUcf2jDMyMtpYKRbgsf0Y2B56DWVXEvBFohgKzOKC7M81jmdrd1SCNc8QhCIXYbZZrG\\\\\\\/Lsa7Dp4hM07itHOoPH89KXCJA0VzKftJfj5t7N0d8Xn6JPLrAx8V6yZCFwFywq\\\\\\\/\\\\\\\/gO6epNyYUq6h\\\\\\\/nMWLtg\\\\\\\/sV9K0BHJ3w01Pd5\\\\\\\/Ydgo0WwSxzsLeAhjL2NI9SktcUSTgJsa0ZbVvkSJba4yowOQo0J8GDKzBLMcu9EyUgsI0tvpZ118QUP6Gffq8+9hbG6RkQFCDcaz8jaz9ljnzy844PGcM\\\\\\\/2qAo9tLuUBqaSraOpIXYffR3b\\\\\\\/+\\\\\\\/8Q49H+GEkBgaKH5cPngQd3jGiCtxLAbBSyZ\\\\\\\/4QOiXzBjR8bHoNvoG+5raZRrnL9Fno3ckKrACEXhQ7yAprAOWeh31U+7YJZ\\\\\\\/ZizU5CKG3xH+XETYJQOsSGlT\\\\\\\/kkg59FZMHgXnEt0N7ZRuxRtWO6HVcS8j3BLWMzDFZd\\\\\\\/hr6RCGqp\\\\\\\/5VJWR3VATxCT\\\\\\\/ARdekWaWLD0OpGVlyjMhYQ9vbrBk+LNuL3qUOqiTBAn4VPY4wgaT7Am5tmT2dO0fMSjDJgGOvgRegJERAuKVQCTu4W6j5zTfAuHElM7pchX58WeF9jQxGaK9vj6EUTm6cjSRgdhpzVp\\\\\\\/+l9DbJobIAUHppB3J+9czyLQpoKRP6yaSta3vrZEay7FvQe90iqzZvwJUNyUZvQu38YhfrVzLpSkWoWwTWoK13la4TSN8RMYqn32fQCVthjXuNhKC0Wwm1L4r8OLYT4UNQmfVATU3PuT1TdPVE2Sbh0bm0pnvi2T4yScmvuIv9Ojq6iqNbNtwdIPNyGc0Z\\\\\\\/4uNX5+0oXyqzzmAmE5Dd+YePAf8Y\\\\\\\/FJnGJHHW9qvjELpc7vggzYmo+A8gUw9OxmmIpa3HdIO29qRC2WrIDpeRZhzRPxdblxSDHx8mM7HgHz5MKhjNK7XbaayqBHr3xFnq9aHXSLGlbqCoqljMWgtizKnZQ0MtFrSxhAhXTXgeLC+PsT\\\\\\\/hILwivROpce1PPfocp+HKJRJuswq\\\\\\\/9DgMQPDZUJ6G01vUE9J0N0fiMrniKVKrNH8CQbYV6iWc\\\\\\\/TDt0ShPG9lLl0\\\\\\\/0CqV1keNUvJpWWf5RZo4QkoPEj3lDvYmr4+nhvLT2QZiFDqTF\\\\\\\/ymM7jJCxd0NLUG94wu6Gf990rlGofZBVGhQZNjhbX7CskPcZO6lVnQ7EQlXagt6TZLq\\\\\\\/I48jTn9b6jRGB3IfQ4qQzITUc0Bl4lJ186H1gzHHQAH6h\\\\\\\/axggaLbX+CkcYHDMUn5Auc3AqAr8uQfy4m80IRZAdL37LTkdIyoX3U7aHFxNUiM5vmWTZdCXDbc4h9B6Qo20\\\\\\\/qwtFx7VgNuXyLe3mUOL01gqM1965ucm+mcsJCYlUzIAscuukrz4Jcfcv9gVQWOZ91gAxVbkSeKBrWaaqlU1HnSLgmTr6GaIW3kpXvtunGt\\\\\\\/ZEZ29HULzhcenAiRO+qBOZFF6tPKA6wfruaqjEaq2\\\\\\\/fscrNJMEVNfIQwS\\\\\\\/lkTy3\\\\\\\/Gtm3D31+Ap52ZCpiDv5\\\\\\\/xaX5imNQNuRnO1zjoI1o6CcmCP3MtWAH8g3KwSfwNJUFn6nqct2noo\\\\\\\/izZHsLgzbvsF\\\\\\\/SG1Z0kcf+6Zd1zHsaAcc4LZiQtGKvpzLaLDh8nlCFQujEaXFhYOcnMaLwihlb6X8KGwPihMWEO9ypDuFUq1870HLJq0aH7F5cV07qSmnwqtmWz0XktMpEMbsZ0Aqn2c5Dlutih2qqnxZ8PuLNXnA\\\\\\\/ZG6OjCyo1sUDnfrsx0rsrZrDHmLvT2fKkPNQN0IbaItv1UsG85qtdNWv3+ShcHYBnwst6deX248cbMYYHlegtybeCPilcT5le5YIWQCrOm7ArY4di8SCpgqqIqvYxVz5lHYWTE2zlcVH3XfUbuwtEjY7zGvedMVNepCOfV41j\\\\\\\/qyZYR4kK\\\\\\\/L\\\\\\\/eQJJmD6TzeahD8dSgLzAEDqJmmjfFtav9Q3nz32vZ+0q45\\\\\\\/cFXXPbyufAmxV9dK5JVnTTNEkegu8pnV2g8XqLw4tCET7t4\\\\\\\/SHg7LJIsxPWfNOzbmCtnPWEKyxZDNVT2eHeDveYLY\\\\\\\/P1GmjltyRCVchK7g9uWoqoJ03c7h1\\\\\\\/Cu0RFFitPia+HPJv3warBCDhdbMzHQ3d4FOa8RPOYOAQfEansm\\\\\\\/YwekpTMScHfdT8Hk8MdVzT0LZCoylUa\\\\\\\/OH2fH9aonnIahfi+b+OxxppFNVs1qoIKGKbSDdLuXMbyNCiH0HvJEqn9eggf03n0jvqCX1tSy02fyK1r8UdDLwQNrn07aOZkydrTrX9KyM5o\\\\\\\/Efv+TEu3WIJpcos7nNC5OSwdjOXl6J6Dsun1EWBvFpGpt2LO\\\\\\\/UjsbaESnGwnzc5R6rR5mIw7WgL9ycSFjV9Gr5ggL5KYwBvAiSpVm5Sqs3vNEmB1eTTS5oV7tQ7c+0QelbVA7GEnQyLn8XIw40ESV\\\\\\\/rVUHo0PtupIle11z+5E909NBMLLb1Zf8ALivaHhwXdngMJvtcK4P9kzVW5WnfGyHguzbo8LxB0rlaxgqJa6XqktUJ7HwVxEj5UwDNCHAYX8SnjfeK+YbOVYzSkPREmF0Nwe3GTmru8qoyyfCUXO52K5pPhz4T\\\\\\\/b078hvtYZ275hHasJ5eYVxaaf2kbB0AlhCxO3ju1FpLst0+DXkaPo7tkpuN5cAFRQjzT81UiZ4Dexd3wKo+t6zHkxawgwj6Vwjcxpn304G\\\\\\\/VXqCpJS2e5MVe2pSzqbJa69ys4k6\\\\\\\/gK42W5RmzmVrXr+GUmWFqXmAkKoTS9NZvOy4fuTbJhGV\\\\\\\/dNyG4Tjxsh4vc6F+c2j5Xnq70WnlAy2PHbqyewYh+xhV\\\\\\\/HDRkJEPZ4jpL+VTVF1uA\\\\\\\/TthA0Yre6t5K6CFx+881fKPE68dfktq4iNalw2epxEBM4vuUbFAvw9qUOFgmq2epFYR\\\\\\\/LD2H2J9UGvPGC06xgbizSbFY1+O7dUy166HwrMaN84uywH9B5urlF6ayLHTwmgiPm3dFzBGlTaUDeKejH1pvhEe98wTQacXopzG59vUFQm+r8rzzhfZ3dbrOTJzPa29H2Rq+BRpWiyfFpUHKFdWGbRU79m6zTHYSPtpvTXVB6zRzI0+gVLcLLeSf9ql8nbpgDW\\\\\\\/hMZXZV3oWQP5ExjSTH4KVp0mijiUXYIk7dl+Y5JwXJbAR\\\\\\\/fkTv2Rh4M+dsbmfN6ZjwqPiPf\\\\\\\/t71JELKPy6fwEqcMO6rFrKQtZ1xARwTCbQm4nPVN7DXQpcQS+PoBeny0P4nmK3YY8T1FZmH0Tz+zXsglkeofTU85\\\\\\\/tpo1+jEeiNMcl2xLrRg9cOxwYy\\\\\\\/9Zs91b9K05KxcDZn\\\\\\\/Tm5ZZ8jClL+Rrs8JJ8\\\\\\\/pRbL0PMmldaw1LVFcN6Q66XUP5efBtoT\\\\\\\/hLqFZAFKP21sj+zXsBCaHHCqCc5RXTCMXTyWGR3fFk6nld4B+wdfT8A1UnEcoDg7RxN\\\\\\\/pPy2Gsg3byOvAn+E6Y3eXt8wHmEfcQa5MmnxMebt4E4YACVpG3QWZJ\\\\\\\/AHrrJGQ3tmNZn40uqEbz7qOhDHrcUGjlaOSBKvFzNXBhI+PNWIZ8Q1aAHAUCpxwZRkfMBLxyeM+9blMhwLF0ID4tKsUxuzMdd1JMMlhGwWPWCM7gcTmBjBybyMrhvZMEhGkSSPPrLK0zltFv\\\\\\\/Sr85leVv1vKPotEgea7H8oXy+zEC\\\\\\\/HkkFoo9L5kefaMP5nR7VuHYCsGGE7OMOXc+GQClhKL\\\\\\\/5ApG8YI2RPlwtNkOX9cS16se8KUfJKjaoSXIhJebiCitsywS\\\\\\\/4zFIkbiGLdiOzyxLxDEWb+9\\\\\\\/sPW7ozcEBa6J1UUBzYTdx+qEiM9VB3LrKKQNYj+SQmS+w+vYKBtNP7chL0+jyvXzN\\\\\\\/yC7LedKUbFOiVxTxJwi6OMQj3+uzdThvWmPUz596l3xMr3oLNKrrMZBfJQmefn3hypcMgqTYK1Qv4PCf0JadfGP+PNmsu7HqPw70X\\\\\\\/MwvvT4+f75PTKZ4tHdVwfjbtWdRHqYKUEe35umoDo++SlP7akAKNU5cd8r2O+wnOxOhTdKCkyt1C85RNXKRkqBVMULjmq4fl508c0hL7fdMC1dN9bdqV0c2M5zMInUsqkEWTMPqdyCRwr4c6cbCP+oG8iDkjzTEhudJ\\\\\\\/cXq2ibzh9m05YFC4\\\\\\\/qgcQK0aCNGHbDV5Aqe8VdCyR3lgI5C9zolyIVv7FOnZl6ft\\\\\\\/KR6e\\\\\\\/+h2ltXPhYZKhfGY3u4mA8LkAecYsN59sj7uhDALANW7Z8Ca+5CpdT1SN+NEXltjq5WE4tmxq3pRLjLEBLh1vWX3LKDpbdZf9VOHsC8Ln9leVevsBEFkLtLNrF4LLpckBTM2uPkI3mo9CiJr8vNNgN2BDdJzV9qErCzag\\\\\\\/M88MTBI9YhAmlqwgQPOuh7G9q9LJEvu3gYF7voIIEb9VrzMs\\\\\\\/VMvCac0FkQgCGgAMU7GENa3Yat2oV2IWCRccA+NgO4ZWbE48qx+hkP7EJVDrbn8fmytpysCsc\\\\\\\/hUqg3GI4rDHqNES8YuYuqzryoQ8RkLc0\\\\\\\/Ry41JXdHYgTbhkZi4Yz7vnzO2qD8W7v3RR+ev9c58p+R9BhmJguM2OC2L4W3ttmmgKyQ+GUzhxmYwdijVgk0vzUw8www+l4SFartRxWJlj++HLH7fedS75O4+Rpm2BBnEglAYh88krp9L0fvrHQnHCqBjXzCMURKjlMhc\\\\\\\/YKwCT6CG4\\\\\\\/fpmY5yUS\\\\\\\/4AorcFqhY9AT3n78Ra5yWyTcsgbrC2FvcGmD6RRs81t72HyonF7HfEhFUEC1JqA5Km0pVhoNbDFtz3xwgMnOBWFFXeCPZw278sPIxbhjj59zCAUJ4HdNLKUp6sVBW0mpY0BUjKS269UWNkyCY3rZM7X0xC4qDO\\\\\\\/0QRYkXk4jemxNNHTCIqN3vtHUFej9EtF2YGh2ZqBUEcl8eevrMYwrfAMtNUgmL2xWRROppFj+HDn\\\\\\\/0BAwXw4g8jiQ\\\\\\\/4yTubarIwwrUmO2mSACOdXu+tBHK2pVlSkzcHDRXmneyDRxgA8MOqYobMcNEnMyvg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KP4Neiedr6\\\\\\\/wBNWiWh\\\\\\\/\\\\\\\/zQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229e9700387a636844bacdb7a308b3a3d5010e80a9afad20df3481249e3f70bad5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022V3WIdeyai75biIEoqsEJ89H0Mp48t2k3SYKfe0+HU1VN2cc7Pnz98kI4I\\\\\\\/ca+qBXwlPBnYuJ2GhusTwNy4YuJxjx9zKL5mB5KsXvazGNeaN\\\\\\\/cM2BlhIJwml02jttMXTJ92hkLrj318\\\\\\\/AwSRoX3eoUZlcq9hbCYZUROVLrwN8iyLIGyhY175mseaMxAlh5xyPv1aIMBWVoDS3CpYGZq91x\\\\\\\/5uf4Y+uC1VDunDytZt902ivx\\\\\\\/8FAsInTRr3ViT1TT48e\\\\\\\/q4BJa4So\\\\\\\/bGlHvmsjskRFAhiNYEwmFbQFNxHGYFFfQBoJ\\\\\\\/WDvKfEK6CytfQRFGCe6\\\\\\\/bSZ3i344KeOw0o+5LoDBH2danhrYSluykEcVzdHZvzcoEpOvOD0Rti0enyky6UVff1zhjbZIZxMDrSQvcatL2SO44Rh23HAGx6kw6OTGpFZFVYyZwsT\\\\\\\/PJ7qbjp4YX7DDIUEUnRWZWkSQt3etMXuEiSrbtGjqw8XIug7kf7t75a7Qg84nFKPxG9sL0GO+edZ0+hrXB1+ePJy0aoj6f15HuqpzgJfRqLnBk4+bDdlQRcnJXC7TrXVE8mKzt0k+Tu7tfevK12c8oSbowIisLARu9ZWhvT+yB41oJwbP0TciEbAN\\\\\\\/sRfLweAv8qQ0\\\\\\\/iAwcVSm2qY+yASVuC0nL19RipCZbl4xctw4\\\\\\\/aGwOGxEkeNkCfkt9\\\\\\\/bzh30bmNFrLp4GdXzw4sgCLdfzfg3EVxeXbQ5LOARcfDtsl0qaZsV3\\\\\\\/BaaGNvz\\\\\\\/6rOqBmoYPngCmMWsfL1IrKpROEhsyCnDa\\\\\\\/sUiF5aiWqUjc3GYerYPk0y1SwywvYSwpUYfQi4rY4uTW0aWjdaFCZ64VCiWW+P\\\\\\\/KwuP1lgkNrKTT0bUOSnO9K4rJZDUvglipBXkd+FSd4\\\\\\\/bTtWs9aAa94QL9cPP62UUmip21FAUreOTVkozZvwHpAnxAvB67oN+tQtivduDGUdiqbkPhWb2N9qSzKe+j4GjSEqc1lKSHU8uCO4i83oXjID7x3EofoSqjUudLBl+U+N6UitvrH2\\\\\\\/Ngo7fuZhQQSdCob7ZahN89EVZLuvLU4XF06vfb0lLN5RIejOy1mh1Hx4VRrhDHbH5e+13rZ5thy93ilpcWkS8KN5G4a+zW2+UlQn+jn1Y\\\\\\\/J1JPKMDEpeCtz\\\\\\\/5m8SybAs5IUgk5vdVc6P\\\\\\\/rZ7eph2jMT0ONZrI6gSuJauKJCPqH5Je6pavHVAEQXYQls3+Q3lCOagppMdIOE+goIs4++LoLLiUCK2sF515oMdb24nHfs+gVZyDXhqfzbvYwcYYAk4HY27Z7PhrIeTDvYzwXEUXbhVbs3FICyMxjRC6K7XYgJljjTL\\\\\\\/AZ+Zs\\\\\\\/EyYG3q4SGq+wuzA6rBxucj4RsxJq5fcHVZXtHvl+ktN+5Kb1LXxo\\\\\\\/crY7rCOb8dwAv72SUVSxITxVgv5CmH8SS97cCF3MbuJ1LBOkkOYJT4iRUUFzLqSVLxe2n3SH06GzckC+uPFb2Dkb7knkntMTWOSatXFCV2IO9Ory4+cUstaNRUmqzZYTDM5NT5ANqwi+vo8OtUPY\\\\\\\/JHtooix+xVilJf+X4eOHJZtaoP6PaBZeNA5f\\\\\\\/+1xV\\\\\\\/dA6SlPw2XBMoJN0iHtffLSFBEsmedOiQOHRmj\\\\\\\/+o96wyNFla7AI2NmE4eH3wDSERWga4rriKA\\\\\\\/5SnddBRoAej9cG2ZOS4lv0e\\\\\\\/8exE13bZZQHah2hlUE4qnfU9j0J9togD7HLFb+e4lpnHrGhuRhag6VqXaAfzBzC\\\\\\\/7vhNYGR52+cpRR5Y1zIUiLtdLBWtPg4sa7cxPnPx67iNiErf940t0BEyLxWf9RthCuub57QXoJRNSFPMSgy2oyZGfWn+YIGN2aL6Aji2E0Y12s7bHSnz6h2vnz2jIgU2xpbI4AxhHGyYPiaNSPap0snS8wuoIEe0F7II+RbAr45VwmkmesR3jhn1CI3SOQLnvCsvwgkYVelKzL930UpEaZ3x+W1EF2OcF1mc2NEKj5zfI3e2djVXrY+vi\\\\\\\/v0\\\\\\\/aBQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DP3e4AlWIhDJeWgzvwlmGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002208501469b67fbe5866b33e5629225d719170c1e4e99751f87dd3e8b6d5fd40c7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459175 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/4/263" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--RXxL_66Rxh0LR0p_mDYWERT4LWY6xxGs8P1MZr0QaM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/4/263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-B3TownyHmqo0x4oF-j5NOu7Lvp_LcS460JCIQw1Zn58" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/4/259" title="Variable Transposition of Eight Maize Activator (Ac) Elements Located on the Short Arm of Chromosome 1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/4/283" title="Identification and Precise Mapping of Resistant QTLs of Cercospora Leaf Spot Resistance in Sugar Beet (Beta vulgaris L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459175" id="node459175" data-pisa="ggg;1/4/263" data-pisa-master="ggg;g3.111.000422" data-apath="/ggg/1/4/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chloé Ambroset</span>, <span class="highwire-citation-author" data-delta="1">Maud Petit</span>, <span class="highwire-citation-author" data-delta="2">Christian Brion</span>, <span class="highwire-citation-author" data-delta="3">Isabelle Sanchez</span>, <span class="highwire-citation-author" data-delta="4">Pierre Delobel</span>, <span class="highwire-citation-author" data-delta="5">Cyprien Guérin</span>, <span class="highwire-citation-author" data-delta="6">Hélène Chiapello</span>, <span class="highwire-citation-author" data-delta="7">Pierre Nicolas</span>, <span class="highwire-citation-author" data-delta="8">Frédéric Bigey</span>, <span class="highwire-citation-author" data-delta="9">Sylvie Dequin</span> and <span class="highwire-citation-author" data-delta="10">Bruno Blondin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-281;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000422 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459175" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chloé Ambroset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chlo%C3%A9%2BAmbroset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ambroset%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChlo%25C3%25A9%2BAmbroset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maud Petit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maud%2BPetit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petit%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaud%2BPetit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christian Brion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BBrion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brion%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BBrion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Isabelle Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BSanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanchez%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BSanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Pierre Delobel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BDelobel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delobel%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APierre%2BDelobel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cyprien Guérin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyprien%2BGu%C3%A9rin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%C3%A9rin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyprien%2BGu%25C3%25A9rin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hélène Chiapello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BChiapello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiapello%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BChiapello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Pierre Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APierre%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Frédéric Bigey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fr%C3%A9d%C3%A9ric%2BBigey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bigey%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFr%25C3%25A9d%25C3%25A9ric%2BBigey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Sylvie Dequin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sylvie%2BDequin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dequin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASylvie%2BDequin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Bruno Blondin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Montpellier SupAgro, UMR 1083 Sciences pour l'Oenologie, F-34060 Montpellier, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BBlondin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blondin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BBlondin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:blondin@supagro.inra.fr" class="" data-icon-position="" data-hide-link-title="0">blondin@supagro.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/4/263" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459175" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459175/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/4/263.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459175" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459175/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/4/263.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459175" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459175/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/4/263.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459175" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459175/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/4/263.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459175" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459175/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The genetic basis of the phenotypic diversity of yeast is still poorly understood. Wine yeast strains have specific abilities to grow and ferment under stressful conditions compared with other strains, but the genetic basis underlying these traits is unknown. Understanding how sequence variation influences such phenotypes is a major challenge to address adaptation mechanisms of wine yeast. We aimed to identify the genetic basis of fermentation traits and gain insight into their relationships with variations in gene expression among yeast strains. We combined fermentation trait QTL mapping and expression profiling of fermenting cells in a segregating population from a cross between a wine yeast derivative and a laboratory strain. We report the identification of QTL for various fermentation traits (fermentation rates, nitrogen utilization, metabolites production) as well as expression QTL (eQTL). We found that many transcripts mapped to several eQTL hotspots and that two of them overlapped with QTL for fermentation traits. A QTL controlling the maximal fermentation rate and nitrogen utilization overlapping with an eQTL hotspot was dissected. We functionally demonstrated that an allele of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> gene, localized in the hotspot and involved in <em>p</em>-aminobenzoate biosynthesis, controls the fermentation rate through modulation of nitrogen utilization. Our data suggest that the laboratory strain harbors a defective <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele, which triggers strong metabolic and physiological alterations responsible for the generation of the eQTL hotspot. They also suggest that a number of gene expression differences result from some alleles that trigger major physiological disturbances.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/wine-yeast" class="hw-term hw-article-keyword hw-article-keyword-wine-yeast" rel="nofollow">wine yeast</a></li><li class="kwd"><a href="/keyword/fermentation" class="hw-term hw-article-keyword hw-article-keyword-fermentation" rel="nofollow">fermentation</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li><li class="kwd"><a href="/keyword/p-aminobenzoate" class="hw-term hw-article-keyword hw-article-keyword-p-aminobenzoate" rel="nofollow">p-aminobenzoate</a></li></ul><p id="p-3">Industrial wine yeast strains exhibit specific traits making them suitable for alcoholic fermentation. These yeasts are specifically adapted to stressful conditions and can ferment efficiently under conditions of high ethanol concentrations, low pH, nutrient starvation, while producing suitable metabolites, especially aroma compounds. Although a large phenotypic diversity has been described, the molecular basis remains largely unknown. Various molecular mechanisms may contribute to wine yeast adaptation, including gene deletions or amplifications, chromosome polyploidy (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dunham <em>et al.</em> 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dunn <em>et al.</em> 2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Infante <em>et al.</em> 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Perez-Ortin <em>et al.</em> 2002</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rachidi <em>et al.</em> 1999</a>), and nucleotide polymorphisms (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Borneman <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Goffeau <em>et al.</em> 1996</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kellis <em>et al.</em> 2003</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wei <em>et al.</em> 2007</a>). Analysis of the complete genome sequence of a commercial wine yeast also revealed that new (non-<em>Saccharomyces</em>) genes arising from horizontal gene transfer were present in the wine strains (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Novo <em>et al.</em> 2009</a>).</p><p id="p-4">Linking genetic variation with phenotypic diversity will help to increase our understanding of the adaptation of yeast to industrial stressful environments and will facilitate strain improvement through breeding and genetic engineering. Quantitative trait locus (QTL) mapping is a proven approach to map the genetic variation responsible for quantitative traits in <em>S. cerevisiae</em>. It was successfully applied to high-temperature growth (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sinha <em>et al.</em> 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sinha <em>et al.</em> 2006</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Steinmetz <em>et al.</em> 2002</a>), sporulation (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ben-Ari <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Deutschbauer and Davis 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gerke <em>et al.</em> 2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Katou <em>et al.</em> 2009</a>), cell morphology (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nogami <em>et al.</em> 2007</a>), drug sensitivity (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kim and Fay, 2007</a>), ethanol tolerance and growth (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hu <em>et al.</em> 2007</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Katou <em>et al.</em> 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Smith and Kruglyak 2008</a>), and flocculation (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brauer <em>et al.</em> 2006</a>). QTL approaches have also been used to dissect the molecular basis of several wine yeast metabolic traits such as acetic acid production, hydrogen sulphide production, and release of volatile phenol (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Marullo <em>et al.</em> 2006</a>).</p><p id="p-5">Several studies have highlighted the role of expression variations on associated phenotypes in yeast (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brown <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cavalieri <em>et al.</em> 2000</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fay <em>et al.</em> 2004</a>). Variations in expression between wine strains have been reported in various genome-wide analyses (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rossignol 2004</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zuzuarregui <em>et al.</em> 2006</a>). However, their impact on strain properties is poorly understood. Genomic approaches to variations in gene expression have been used to establish relationships between differences in transcript abundance and genetic polymorphisms via the mapping of expression QTL (eQTL) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jansen and Nap 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brem <em>et al.</em> 2002</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yvert <em>et al.</em> 2003</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brem and Kruglyak 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ronald and Akey 2007</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ansel <em>et al.</em> 2008</a>)). Combining the search of eQTL with that of QTL of industrially relevant traits may be of interest to gain insight into the relationships between expression variations and wine yeast traits. Indeed when an eQTL colocalizes with a phenotypic QTL, one can hypothesize that a common polymorphism controls transcription differences and the physiological trait.</p><p id="p-6">Here, we report an integrated approach where we searched for both phenotypic fermentation QTL and eQTL in the laboratory yeast strain S288c and a haploid derivative of the EC1118 industrial wine strain. We identified QTL for various fermentation traits, including kinetic traits, as well as eQTL for many transcripts. We show that eQTL display hotspot regions that control many transcripts and interestingly, two of them overlap with fermentation traits QTL. We functionally characterized a QTL that controlled the maximal fermentation rate and showed that a gene involved in <em>p</em>-aminobenzoate synthesis (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em>), probably defective in the strain S288C, modulates the fermentation rate by controlling nitrogen utilization. The relationships between structural alteration of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> and eQTL hotspot generation are discussed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains, growth conditions, and fermentation conditions</h3><p id="p-7">The two parental strains compared in this study are the standard S288c (<em>MAT</em>α; <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001424">SUC2</a>; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071">gal2</a></em>) strain and a haploid derivative of the industrial EC1118 (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a>) wine yeast strain, which is referred to as 59A (<em>MAT</em>a; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">ho</a>). This strain is phototrophic and has fermentation properties close to the diploid EC1118 strain (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The strains BY4742 (MATα; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3Δ</a>1; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2Δ</a>0; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2Δ</a>0; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3Δ</a>0 and BY4742∆<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> (Matα; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3Δ</a>1; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2Δ</a>0; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2Δ</a>0; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3Δ</a>0; <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">YNR033w</a>::kanMX4) were used for hemizygous constructions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/263/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fermentation profiles of industrial and laboratory strains. Fermentation rate profiles of the industrial wine yeast EC1118 (gray squares), its haploid derivative 59A (black diamonds), the laboratory strain S288C (gray line), and the hybrid Z59S (black line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452389652" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fermentation profiles of industrial and laboratory strains. Fermentation rate profiles of the industrial wine yeast EC1118 (gray squares), its haploid derivative 59A (black diamonds), the laboratory strain S288C (gray line), and the hybrid Z59S (black line).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/263/F1.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/263/F1.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/263/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/263/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460857" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-8" class="first-child">Fermentation profiles of industrial and laboratory strains. Fermentation rate profiles of the industrial wine yeast EC1118 (gray squares), its haploid derivative 59A (black diamonds), the laboratory strain S288C (gray line), and the hybrid Z59S (black line).</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Fermentation experiments and precultures were carried out at 28°C on synthetic MS300 medium, which mimics a natural must described in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bely <em>et al.</em> (1990)</a>. After preculture in 125 ml shake flasks for 24 h, fermentations were performed in 1.2 l fermenters equipped with airlocks to maintain anaerobiosis and with constant stirring.</p></div><div id="sec-3" class="subsection"><h3>Construction of strains for functional analysis</h3><p id="p-10">Hybrid strains BY4742∆<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em>/59A and BY4742/59A∆<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> were constructed for the functional characterization of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> candidate gene via a reciprocal hemizygosity analysis.</p><p id="p-11">The BY4742∆<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em>strain, which isogenic to S288c but with a copy of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> gene deleted, was obtained from the Euroscarff society. The copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> in the 59A strain was deleted and replaced with a <em>KanMX4</em> deletion module using the short-flanking homology PCR (SFH-PCR) technique (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schiestl and Gietz 1989</a>). The two custom 60-mer primers used begin with 40 nucleotides that are identical to the upstream or the downstream region of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> followed by 20 nucleotides that can amplify the <em>KanMX4</em> module, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> forward: ATGCTGTCCGATACAATTGACACAAAGCAACAACAGCAACTTCGTACGCTGCAGGTCGAC; and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> reverse: CTACATGAAAATTTGCAAGTTGCTCTCCAACTTGGTGTACGCATAGGCCACTAGTGGATCTG. After transformation using the lithium acetate transformation protocol of <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Schiestl and Gietz (1989)</a>, yeasts were grown with YEPD plus geneticin (200 µg/l) overnight at 28°C to select cells with the <em>KanMX4</em> integrated module. Clones were finally tested for integration by PCR on genomic DNA with the primers <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a> forwardctrl: TTATCGGTGCGGCAATAAAG and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a> reversectrl: TATGTGACCGTCAGGACG.</p></div><div id="sec-4" class="subsection"><h3>Measure of phenotypic parameters</h3><p id="p-12">To characterize the fermentation parameters, the fermenters were automatically weighed every 20 min and the rate of CO<sub>2</sub> production was calculated from the weight loss by a method of polynomial smoothing. The number of cells was determined with an electronic particle counter (Beckman Coulter). At the end of fermentation, samples were analyzed by high-pressure liquid chromatography (HPLC) using an HPX-87H ion-exclusion column (Bio-Rad) to assess the sugars (glucose, fructose), the organic acids (succinic acid, acetic acid, pyruvic acid), and alcohols (glycerol, ethanol) according to the procedure described by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Calull <em>et al.</em> (1992)</a>. Amino acids were separated by ion-exchange chromatography on an anionic Ultropac-8 lithium-form resin (Amersham Pharmacia Biotech) with a Chromakon 400 (Kontron) and a Biochrom 20 analyzer (Amersham Pharmacia Biotech). Amino acids were detected by reaction with nihydrin. Ammonia was measured using the enzymatic method from <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bergmeyer and Beutler (1990)</a> based on its conversion into glutamate by the glutamate dehydrogenase.</p><p id="p-13">Fermentation parameters of the parental strains were determined from five independent fermentations and from duplicated fermentations for the other strains (segregants, hemizygous strains). Metabolites were measured in five biological replicates for the parental strains and in duplicate for the segregants, except the amino acids, which were determined from a single fermentation.</p></div><div id="sec-5" class="subsection"><h3>Gene expression analysis</h3><p id="p-14">Microarray hybridization and image analysis was performed at the Biochip Platform in Toulouse (http://biopuces.genotoul.fr). The microarrays were obtained by spotting in duplicate the 6308 oligonucleotides (70-mer) from the yeast genome Oligoset (Operon) onto UltraGAP chips. Gene expression was analyzed for five independent cultures of each parent and one culture of each segregant. RNA was isolated at the midtime of fermentation when half of the sugar present in the medium was consumed (equivalent to 45 g CO<sub>2</sub>), and then labeled with the Chipshot direct labeling and cleanup kit (Promega). Each sample labeled with the Cy3 dye was compared with a common reference pool labeled with Cy5 dye and composed of a mix of equivalent concentration of the RNA of each segregant of the population (star design). Hybridizations were done twice and thus technical replicates are available for each measurement. Data analysis was performed using the R 2.9.2 software and the limma package (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">R Development Core Team 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Smyth 2005</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Smyth <em>et al.</em> 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Smyth and Speed 2003</a>). Within-array normalization was performed using the print-tip-loess method followed by a quantile method for between-slide normalization. For each measurement, the two technical replicates were averaged. Gene expression between the two parental strains (2 × 5 microarrays) was compared using <em>t</em>-tests. Differentially expressed genes were defined by filtering on adjusted <em>P</em> = 0.01 to control the false discovery rate (FDR) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini <em>et al.</em> 2001</a>). For linkage analysis in the population of segregants, transcripts with a low signal-to-noise ratio were filtered out: we required the mean red (Cy5) signal to be three times greater than the mean background signal (Cy5). The complete data set is available on the Gene Expression Omnibus database (accession number GSE26437) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Edgar <em>et al.</em> 2002</a>). An analysis of the correlation between gene expression and phenotype parameters was carried out using the Spearman correlation coefficient and the associated statistical test, followed by a correction of the multiplicity using Benjamini and Hochberg procedure. We selected genes with correlation coefficients &gt; 0.6 and adjusted <em>P</em> &lt; 0.05 to control the FDR.</p></div><div id="sec-6" class="subsection"><h3>Genotyping and linkage analysis of the population of segregants</h3><p id="p-15">Genomic DNA was isolated, fragmented, labeled, and hybridized to Affymetrix YGS98 microarrays as previously described (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Winzeler <em>et al.</em> 1999</a>). This procedure was applied three times for the parental strains S288c and 59A and one time for each of the 30 segregants. Probes with a unique match on the S288c genome were then screened to identify biallelic markers using the same statistical procedure as <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Brem <em>et al.</em> (2002)</a>. First, probes identified as low perfect-match (PM) <em>vs.</em> mismatch (MM) hybridizers to S288c genomic DNA were discarded. All further analyses were performed on normalized log(PM/MM) values. We selected probe pairs that had high hybridization differences between S288c and 59A using the Z and z statistics described in <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Brem <em>et al.</em> (2002)</a>. A second test based on a clustering algorithm was used to select probe pairs for evidence of 2:2 segregation in the segregating population. Genotypes were then inferred from cluster assignments as described in <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Brem <em>et al.</em> (2002)</a>. This resulted in the selection of 2465 probes. The probes were further filtered using the genomic sequences of the parental strains S288c and 59A. For each probe, the presence of polymorphisms in the 59A genome was confirmed. We used Illumina high-throughput sequencing to obtain SNP information for 59A. Genomic DNA of the strain 59A was sequenced using Illumina Genome Analyzer II technology (Illumina, Inc.) with paired reads of 36 bp using standard manufacturer protocols. Read sequences (deposited in the NCBI Sequence Read Archive as study SRP004704) were aligned to the S288c genome with MAQ (ver. 0.7.1) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2008</a>) using default parameters to detect SNP. This filtering step resulted in a map of 1834 markers used in the linkage analysis.</p><p id="p-16">For each phenotype and each expression trait, linkage analysis was performed using a normal model with the Haley-Knott regression method implemented in the R/qtl package (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Broman <em>et al.</em> 2003</a>). Logarithm of odds (LOD) scores were computed for each marker every 2.5 cM. An interval estimate of the location of each QTL or eQTL was obtained as the 1-LOD support interval: the region in which the LOD score is within 1 unit of the peak LOD score. Statistical significance was assessed by random permutation of the phenotypes or expression levels relative to the genotype data. For eQTL, the permutation was performed 10 times, and the average number of transcripts showing linkage at a specific LOD score was used to calculate FDR. For each phenotypic QTL, 1000 permutations were used to calculate an individual LOD score threshold and FDR.</p></div><div id="sec-7" class="subsection"><h3>The GenYeasTrait resource</h3><p id="p-17">A database named GenYeasTrait, which is dedicated to the analysis of QTL and eQTL data, was developed using the Gmod/Gbrowse software version 2.15 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stein <em>et al.</em> 2002</a>). It includes the annotation of the S288c genome from the Saccharomyces Genome Database, SNPs detected from the comparison of the 59A strain sequence with the S288c reference genome, and the fermentation and expression QTL of the project. The database can be queried by using the Gbrowse interface, which allows user friendly graphical representations of features on chromosomes and facilitates data exploration. It is available at http://genome.jouy.inra.fr/genyeastrait/.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Characterization of segregants from a cross between a derivative haploid of the wine yeast EC1118 and the laboratory strain S288C</h3><p id="p-18">To generate a population of segregants for QTL analysis, the S288C laboratory strain and the 59A strain (a haploid derivative of the EC1118 strain) were used for crossing (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The fermentation profile of the haploid derivative 59A does not however perfectly overlap with the EC1118 pattern. Such variation is consistent with the known heterozygosity of the wine yeast EC1118 genome that was shown to contain 0.2% of heterozygous SNP (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Novo <em>et al.</em> 2009</a>). The hybrid (Z59S) obtained by crossing 59A with S288C displays a fermentation profile intermediate between the two parental strains (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Fermentation rate profiles obtained in a synthetic medium (MS300) that mimics natural grape musts provided relevant criteria to characterize the fermentation capacity of the strains. As shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, the two parental strains exhibit very different fermentation profiles with a much higher maximum fermentation rate (R<sub>max</sub>) for 59A compared with S288c. Furthermore, the 59A strain also shows a higher fermentation rate at 50% of sugar utilization (R<sub>50</sub>) with a consequently shorter duration of fermentation duration (F<sub>d</sub>). All of these data are consistent with a higher fermentation capacity of the 59A wine strain derivative compared <em>vs.</em> the laboratory strain. Under such fermentation conditions, the growth phase is restricted to the period of increase of the fermentation rate until R<sub>max</sub>, and then cells are subsequently fermented in a stationary phase (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rossignol <em>et al.</em> 2003</a>).</p><p id="p-19">The zygote Z59S was used to generated a population of haploid segregants. The spores obtained had a low viability with 54% viable. The proportion of asci with four viable spores was low (only 15%), and a majority (around 50%) had three viable spores. We kept only one complete tetrad in the analyzed population, and the other segregants were selected from 20 different asci. The 30 haploid progenies where characterized under the fermentation conditions previously described. To describe the fermentation capacity, we considered the fermentation rate at three different stages of fermentation: R<sub>max</sub>, R<sub>50</sub> (fermentation rate at 50% fermentation), and R<sub>70</sub> (fermentation rate at 70% fermentation). Other fermentation traits, such as fermentation duration (F<sub>d</sub>) and cellular population (C<sub>p</sub>), were taken into account. We also considered metabolites, such as the amount of assimilated nitrogen (N<sub>ass</sub>), glycerol (gly), acetic acid (ace), succinic acid (suc) and pyruvic acid (pyr) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/TableS1.pdf">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/TableS2.pdf">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/TableS3.pdf">Table S3</a>). We observed a high variability in the segregants fermentation profiles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/000422SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS1.pdf">Figure S1</a>). Several traits values (R<sub>max</sub>, R<sub>50</sub>, gly, pyr, ace) were roughly normally distributed within the population (<em>P</em> &gt; 0.05, Shapiro-Wilk normality test), supporting a polygenic determinism of these phenotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS2.pdf">Figure S2</a>). The amount of assimilated nitrogen and, to some extent, R<sub>70</sub> and F<sub>d</sub>, displayed a bimodal distribution suggesting a possible control by a major locus. Furthermore, all of these parameters, except R<sub>50</sub>, displayed values outside the parental range. This transgressive segregation indicates that alleles with opposite effects were present in the parental strains.</p><p id="p-20">To investigate potential relationships among fermentation traits, principal component analysis (PCA) was performed (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The projection on two principal axes preserves 87% of the information and explains 68% and 19% of this variation, respectively. This analysis shows that F<sub>d</sub>, R<sub>70</sub>, and R<sub>50</sub> are negatively correlated. This is consistent with the idea that the fermentation rate during the stationary phase strongly determines the overall duration of fermentation. The second axis also distinguishes the final cell population C<sub>p</sub> and R<sub>70</sub> variables (or C<sub>p</sub> and R<sub>max</sub>) and the C<sub>p</sub> and F<sub>d</sub> variables which are not correlated. This suggests that the fermentation rate is independent from the total cell population. Finally, the fermentation rate at all stages (R<sub>m</sub>, R<sub>50</sub>, and R<sub>70</sub>) was positively correlated with the assimilated nitrogen N<sub>ass</sub>. These data indicate that cell capacity to use nitrogen has a strong impact on the fermentation rate. This is consistent with previous reports on the role of nitrogen availability in fermentation rate (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bely <em>et al.</em> 1990</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Varela <em>et al.</em> 2004</a>). The laboratory strain has a poor ability to consume nitrogen, which was associated to a weak fermentation capacity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/263/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis (PCA) of kinetic and metabolic traits. This analysis shows that all kinetic parameters are correlated. The first component shows that Nass and fermentation rates (Rmax, R50, R70) are negatively correlated with Fd. The second component shows that Cp is not correlated with the other parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452389652" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal component analysis (PCA) of kinetic and metabolic traits. This analysis shows that all kinetic parameters are correlated. The first component shows that N&lt;sub&gt;ass&lt;/sub&gt; and fermentation rates (R&lt;sub&gt;max&lt;/sub&gt;, R&lt;sub&gt;50&lt;/sub&gt;, R&lt;sub&gt;70&lt;/sub&gt;) are negatively correlated with F&lt;sub&gt;d&lt;/sub&gt;. The second component shows that C&lt;sub&gt;p&lt;/sub&gt; is not correlated with the other parameters.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/263/F2.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/263/F2.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/263/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/263/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460876" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-21" class="first-child">Principal component analysis (PCA) of kinetic and metabolic traits. This analysis shows that all kinetic parameters are correlated. The first component shows that N<sub>ass</sub> and fermentation rates (R<sub>max</sub>, R<sub>50</sub>, R<sub>70</sub>) are negatively correlated with F<sub>d</sub>. The second component shows that C<sub>p</sub> is not correlated with the other parameters.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Transcripts abundance in parental strains and in the population of segregants</h3><p id="p-22">To assess the gene expression level of the entire genome of the segregants and parental strains, we performed a transcriptome analysis. The physiological condition chosen for analyzing gene expression was the point of midfermentation (50% of fermentation process). At this stage, cells are in stationary phase and are experiencing a general stress due to nutrient starvation and alcohol accumulation (about 6% at this stage). Under these physiological conditions, the transcriptome is stable and expected to provide a relevant picture regarding the capacity to resist to such stressful conditions (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Rossignol <em>et al.</em> 2003</a>). The transcriptomes of both parents and each segregant were compared with the mean transcriptome of the population (see <em>Materials and Methods</em>). We first observed that 2184 genes were differentially expressed between the parental strains at <em>P</em> &lt; 0.01 and 262 genes up- or downregulated with a ratio higher than 2 (see <em>Materials and Methods</em>). This indicates that a large fraction of the genes in the genome are differentially expressed, with only a small subset displaying strong variations in agreement with previous reports (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Brem <em>et al.</em> 2002</a>). Among the top 100 genes that were strongly overexpressed in the parental strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/TableS3.pdf">Table S3</a>), 59A exhibited a large set of stress- and anaerobiosis-inducible cell wall genes from the <em>PAU</em> and <em>DAN</em> families. These genes are known to be highly expressed during alcoholic fermentation (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Rossignol <em>et al.</em> 2003</a>); however, given that several of them can cross-hybridize, it is not possible to identify unambiguously the set of genes really overexpressed. In the laboratory strain, there was a massive overexpression of 76 retrotransposons in agreement with a reshaping of retrotransposons in S288c and an amplification of Ty1 in the laboratory strain compared with EC1118 (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Novo <em>et al.</em> 2009</a>). Another striking feature in the laboratory strain was the strong overexpression of several genes involved in the stress response (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001181">SPS100</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000615">HSP30</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001880">HSP12</a></em>).</p></div><div id="sec-11" class="subsection"><h3>Correlations between gene expression and phenotypes</h3><p id="p-23">To address the relationships between fermentation phenotypes and gene expression, we carried out a correlation analysis (see <em>Materials and Methods</em>). The expression of a large number of genes exhibited significant correlation with kinetic parameters (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS4.pdf">Figure S4</a>). Genes with the strongest correlation with R<sub>max</sub> were involved in nitrogen metabolism (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005086">MEP2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004132">PUT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001512">MAE1</a></em>), protein synthesis and degradation (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004234">MAP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002579">SUP35</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000814">PRE1</a></em>), or Ty protein. Several of these genes were also correlated with the amount of assimilated nitrogen N<sub>ass</sub>. Expression of these genes is controlled by nitrogen catabolic repression (NCR), and they are induced at the beginning of the stationary phase in response to nitrogen depletion (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Rossignol <em>et al.</em> 2003</a>). Their correlation with R<sub>max</sub> and N<sub>ass</sub> is consistent with their control by nitrogen availability and the role of nitrogen in fermentation rate (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The genes that correlated with the fermentation rate at midfermentation (R<sub>50</sub>) and at 70% fermentation (R<sub>70</sub>) were very similar. Several genes involved in thiamine metabolism (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002238">THI3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003376">THI4</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000613">PET18</a></em>) displayed a high level of correlation as did the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">actin</a> gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em>, and a cell wall gene (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003511">SCW4</a></em>). Correlation was also seen, though to a lesser extent, for genes involved in nitrogen metabolism (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005086">MEP2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004132">PUT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004234">MAP1</a></em>). The genes correlated with fermentation duration (F<sub>d</sub>) were mostly consistent with those that correlated with R<sub>70</sub> or R<sub>50</sub>, but with the opposite relationship (these two traits are negatively correlated in PCA; see <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Peculiarly, this included various genes involved in the stress response (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001095">CUP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001880">HSP12</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000615">HSP30</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003320">CTT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000905">SSA4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002592">ATC1</a></em>) which were positively correlated with the fermentation duration F<sub>d</sub>, indicating that a high stress response was associated with a low fermentation capacity. Several members of the <em>PAU/DAN</em> family previously observed to be overexpressed in the industrial strain were correlated with nitrogen assimilation, and to some extent, with R<sub>max</sub> and R<sub>50</sub>. For the other fermentation traits, such as cell population and amount of metabolites, weak or no correlation was found with gene expression. This was probably because the stage at which gene expression was analyzed (50% of fermentation progress) was not appropriate to address these parameters, which are mainly associated with the growth phase.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460863/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460863/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Correlations between fermentation phenotypes and gene expression</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Marker map construction and identification of QTL for fermentation parameters</h3><p id="p-25">To genotype the entire population of this study, we hybridized the genomic DNA of the 30 segregants and parental strains on high-density olignonucleotide microarrays Affymetrix YGS98 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Winzeler <em>et al.</em> 1998</a>). Using the genome sequences of the two strains, we obtained a dense coverage of the genome with 1834 physical markers and an average spacing of 6.6 kbp between each marker (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS5.pdf">Figure S5</a>). The markers are expected to be reliable as we checked that each marker corresponded to a SNP in the genome sequence of the strain 59A. The coverage of the genome was rather homogeneous, with only few gaps on chromosome III and on the right arm of chromosome XIV. These regions of low coverage also have low SNP density that may explain the reduced frequency of markers (see Gbrowse at http://genome.jouy.inra.fr/genyeastrait/). The distribution of the markers in the 30 segregants was used to build a recombination map (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS6.pdf">Figure S6</a>). Only very few chromosomes did not display crossover. We used the single tetrad to estimate the number of crossovers and found a frequency of 70 crossovers per meiosis, a number close to previous reports (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Brem <em>et al.</em> 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cubillos <em>et al.</em> 2011</a>). The parental genomes markers were evenly distributed in the population, with 49.8% originating from the strain 59A and 50.2% from S288C.</p><p id="p-26">We then used these markers to map QTL for the different fermentation and metabolic parameters using an interval mapping approach (see <em>Materials and Methods</em>). We obtained a significant LOD score threshold for six different phenotypes. QTL were detected for R<sub>max</sub> on chromosome XIV, for the cell population C<sub>p</sub> on chromosome XVI, for the assimilated nitrogen N<sub>ass</sub> on chromosome XIV, and for the fermentation duration F<sub>d</sub> and R<sub>70</sub> on chromosome II (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). A QTL for metabolite levels was found only for succinate production with a clear QTL located on chromosome XIV and one additional potential QTL, with a LOD score below the threshold, on chromosome II (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). We observed that QTL for various fermentation traits overlapped (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). This concerned a QTL for the maximum fermentation rate and a QTL for the assimilated nitrogen that overlapped in a 37.5 kb region on chromosome XIV (664875 kb to 702375 kb). These results are consistent with the previous PCA results that highlighted relationships between nitrogen utilization N<sub>ass</sub> and R<sub>max</sub> (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Two other parameters, R<sub>70</sub> and F<sub>d</sub>, mapped to a common region on chromosome II, whereas R<sub>50</sub> and succinate production had a potential linkage on the same region with a LOD score just below the threshold. Careful examination of the LOD score profiles suggests that the QTL for succinate is distinct from the R<sub>70</sub> QTL (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS7.pdf">Figure S7</a>). The overlap of QTL for R<sub>70</sub>, R<sub>50</sub>, and F<sub>d</sub> is consistent with the known correlations between these parameters.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/263/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping of kinetics traits and succinate production. The concatenated chromosomes are displayed on the X-axis and LOD score values on the Y-axis. The significant LOD score thresholds are indicated by a red line. Each peak of the LOD curve corresponds to the LOD value of linkage between a marker located in the X position and the value of each trait. Further details of QTL are in Table 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452389652" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;QTL mapping of kinetics traits and succinate production. The concatenated chromosomes are displayed on the X-axis and LOD score values on the Y-axis. The significant LOD score thresholds are indicated by a red line. Each peak of the LOD curve corresponds to the LOD value of linkage between a marker located in the X position and the value of each trait. Further details of QTL are in Table 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/263/F3.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/263/F3.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/263/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/263/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460830" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-27" class="first-child">QTL mapping of kinetics traits and succinate production. The concatenated chromosomes are displayed on the X-axis and LOD score values on the Y-axis. The significant LOD score thresholds are indicated by a red line. Each peak of the LOD curve corresponds to the LOD value of linkage between a marker located in the X position and the value of each trait. Further details of QTL are in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460854/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460854/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Chromosomal location and size of the different phenotypic QTL</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Identification of expression QTL</h3><p id="p-29">In parallel, to identify loci controlling gene expression, the yeast genome was scanned for markers by gene expression linkages. We treated the expression signals from the whole data set of 4398 genes as quantitative traits without additional filtering and subjected them to linkage analysis using the set of markers. In doing so, a test statistic evaluating each marker-by-gene association was computed to detect the number of eQTL with different LOD score thresholds. For each significance threshold we then estimated a corresponding FDR using a permutation approach (see <em>Materials and Methods</em>). Depending on the LOD score threshold used, we could detect from 92 to 897 eQTL. Ninety-two eQTL were detected with a LOD score threshold of 4.5 (FDR of 0.26). A less stringent LOD score threshold of 3.5 (FDR of 0.50) was associated with 409 eQTL, and a LOD score threshold of 3.0 (FDR of 0.63) was associated with 897 eQTL. The high FDR associated with LOD scores was not surprising given the small population size of segregants used in our analysis. Indeed, all previous eQTL studies in yeast were performed using larger population sizes (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brem and Kruglyak 2005</a>).</p><p id="p-30">The main interest in this study was to use the QTL analysis to gain an initial insight into the eQTL architecture and its relationship with fermentation QTL. We therefore decided to analyze the data set using a permissive LOD score threshold of 3.5. The physical location of eQTL across the genome was plotted against the physical location of genes that were differentially deregulated and possessed an eQTL using this LOD score (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The point in the diagonal indicates <em>cis</em>-linkages, <em>i.e.</em>, meaning that the QTL linkage peak coincides with the physical location of the open reading frame for the expression trait. All of the eQTL located outside this diagonal represent <em>trans</em>-linkages with a linkage peaks at loci distant from the physical location of the open reading frames. We detected only a limited number, eight, of <em>cis</em>-associations visible on the diagonal line in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>. Among them, four ASP3 genes (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004145">ASP3-1</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004147">ASP3-2</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004148">ASP3-3</a>,</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004150">ASP3-4</a></em>) displayed a local regulation consistent with the known deletion of these genes in the industrial strain (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Novo <em>et al.</em> 2009</a>). Surprisingly, by observing within <em>trans</em>-eQTL distribution, we noticed the linkage of many transcripts to the same chromosomal region, which gave rise to vertical alignments of eQTL (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). Seven such eQTL <em>trans</em>-bands could be observed and localized on chromosomes II, IV, XI, XII, XIV, XV and XVI. This distribution of eQTL corresponds to hotspot regions, which contain loci exerting a <em>trans</em>-control on the expression of a large set of genes (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). These trans-regulatory bands were also visible at more stringent threshold levels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS8.pdf">Figure S8</a>), and this strengthens the notion of a role of these loci in the variation of expression of a large set of transcripts. The number of eQTL linked to these regions ranged from 12 on chromosome XVI to 42 on chromosome XIV (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/TableS4.pdf">Table S4</a>). The position of individual eQTL can be examined using the Gbrowse interface at http://genome.jouy.inra.fr/genyeastrait/.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/263/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic view of eQTL distribution and relationships with QTL for fermentation traits. A) Genomic view of eQTL mapping. The X-axis represents the genome location of markers, and the Y-axis represents the genome location of the regulated linked genes on concatenated chromosomes. EQTL values with LOD scores greater than 3.5 are displayed. The diagonal pattern is called &#x201C;cis-eQTL band&#x201D; and represents an association between the expression level of a gene and the genotype at the gene&#x2019;s locus. Multiple vertical bands, called &#x201C;trans-eQTL bands,&#x201D; illustrate associations between the expression of many genes and a single locus. B) Overlapping of eQTL and QTL for fermentation traits. The scale below the figure indicates the genomic position across the genome in mega base pairs (Mpb). We can observe the overlapping of the hotspot on chromosome II with Fd, R70, R50, and succinate (succ) parameters and the overlap of the hotspot located on chromosome XIV with Rmax and Nass. QTL for traits with LOD scores just below the threshold are shown in blue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452389652" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic view of eQTL distribution and relationships with QTL for fermentation traits. A) Genomic view of eQTL mapping. The X-axis represents the genome location of markers, and the Y-axis represents the genome location of the regulated linked genes on concatenated chromosomes. EQTL values with LOD scores greater than 3.5 are displayed. The diagonal pattern is called &#x201C;&lt;em&gt;cis&lt;/em&gt;-eQTL band&#x201D; and represents an association between the expression level of a gene and the genotype at the gene&#x2019;s locus. Multiple vertical bands, called &#x201C;&lt;em&gt;trans&lt;/em&gt;-eQTL bands,&#x201D; illustrate associations between the expression of many genes and a single locus. B) Overlapping of eQTL and QTL for fermentation traits. The scale below the figure indicates the genomic position across the genome in mega base pairs (Mpb). We can observe the overlapping of the hotspot on chromosome II with F&lt;sub&gt;d&lt;/sub&gt;, R&lt;sub&gt;70&lt;/sub&gt;, R&lt;sub&gt;50&lt;/sub&gt;, and succinate (succ) parameters and the overlap of the hotspot located on chromosome XIV with R&lt;sub&gt;max&lt;/sub&gt; and N&lt;sub&gt;ass&lt;/sub&gt;. QTL for traits with LOD scores just below the threshold are shown in blue.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/263/F4.medium.gif" width="311" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/263/F4.medium.gif" width="311" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/263/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/263/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460802" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-31" class="first-child">Genomic view of eQTL distribution and relationships with QTL for fermentation traits. A) Genomic view of eQTL mapping. The X-axis represents the genome location of markers, and the Y-axis represents the genome location of the regulated linked genes on concatenated chromosomes. EQTL values with LOD scores greater than 3.5 are displayed. The diagonal pattern is called “<em>cis</em>-eQTL band” and represents an association between the expression level of a gene and the genotype at the gene’s locus. Multiple vertical bands, called “<em>trans</em>-eQTL bands,” illustrate associations between the expression of many genes and a single locus. B) Overlapping of eQTL and QTL for fermentation traits. The scale below the figure indicates the genomic position across the genome in mega base pairs (Mpb). We can observe the overlapping of the hotspot on chromosome II with F<sub>d</sub>, R<sub>70</sub>, R<sub>50</sub>, and succinate (succ) parameters and the overlap of the hotspot located on chromosome XIV with R<sub>max</sub> and N<sub>ass</sub>. QTL for traits with LOD scores just below the threshold are shown in blue.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Overlap between eQTL and QTL for fermentation traits</h3><p id="p-32">To determine whether the chromosomal loci detected in the QTL/eQTL analysis have an impact both on fermentation phenotypes and the massive deregulation of gene expression we examined the possible relationships between eQTL and QTL of fermentation phenotypes. <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a> displays the positions of fermentation QTL and the number of eQTL detected across the genome to visualize the hotspots in parallel. Among the seven detected eQTL hotspot regions, two overlap with fermentation QTL. The hotspot located on chromosome II overlaps with QTL involved in the variations of F<sub>d</sub>, R<sub>70</sub>, and succinate production. The 63 kb hotspot on chromosome XIV overlaps with QTL associated with variations in R<sub>max</sub> and N<sub>ass</sub> (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>). The colocalization of the two fermentation QTL with the eQTL hotspots suggests that a common regulator affects both the expression of the set of genes and the fermentation traits. As variations in parameters such as nitrogen assimilation are expected to be associated with strong physiological changes, their coupling with massive changes in gene expression is unsurprising. In addition, some genes, whose expression was previously found to be correlated with the fermentation parameters, are linked to the same hotspot (<em>e.g.</em>, the stress-responsive genes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001880">HSP12</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000615">HSP30</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001181">SPS100</a></em> are correlated with F<sub>d</sub> and R<sub>70</sub> and have a linkage on the chromosome-II hotspot).</p></div><div id="sec-15" class="subsection"><h3>Dissection of the eQTL hotspot associated to the variation of R<sub>m</sub> and N<sub>ass</sub></h3><p id="p-33">We explored the hotspot covering the region from 635 kbp to 732 kbp on chromosome XIV to identify putative candidate genes that could be involved in the control of R<sub>max</sub> and N<sub>ass</sub>. We used the genome sequence of the 59A strain to search for genes with either nonsynonymous SNP leading to amino acid changes in the coding region or with SNPs in the regulatory region and a self-eQTL linkage in the hotspot. Genes potentially connected to nitrogen metabolism were preferentially considered. We found a relevant candidate gene, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em>, which codes for a <em>p</em>-aminobenzoate synthase. This gene harbors 13 SNPs in the coding sequence and 5 of these are nonsynonymous substitutions (see http://genome.jouy.inra.fr/genyeastrait). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> is also involved in methyl donor synthesis required for biosynthesis of methionine and nucleotides. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> gene is therefore connected to the nitrogen biosynthetic pathways and was considered to be worth for further investigations. We used a reciprocal-hemizygosity analysis described by <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Steinmetz <em>et al.</em> (2002)</a> to investigate the impact of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele on the fermentation phenotype. We constructed an isogenic pair of strains by crossing 59A with BY4742Δ<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">abz1</a> on one side and a 59A strain, in which the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> gene was deleted (59AΔ<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">abz1</a>), with BY4742 on the other side. The two diploid strains differ only in their <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> alleles. In each strain, only one allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> is functional, originating from either BY4742 or 59A. The fermentation profiles obtained with these two strains revealed that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele has a strong impact on fermentation behavior (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). R<sub>max</sub> was strongly decreased whereas F<sub>d</sub> was increased when the hemizygous strain carried the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele originating from BY4742. These results are a first confirmation of the implication of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> in the control of the fermentation rate. The lower fermentation rate with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele from the BY4742 strain suggests that this allele could be associated with a defect in <em>p</em>-aminobenzoate synthesis. Actually, the synthetic medium used (MS300) (see <em>Materials and Methods</em>) did not contain <em>p</em>-aminobenzoate. We therefore examined how the fermentation kinetics of the hemizygous strains were altered following supplementing the medium with <em>p</em>-aminobenzoate. Indeed, when <em>p</em>-aminobenzoate was added (1 mg/l) to the fermentation medium, the hemizygous strain carrying the BY4742 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele recovered a high fermentation rate (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). In contrast, supplementation with <em>p</em>-aminobenzoate had no effect on the reciprocal strain carrying the 59A allele. These data are consistent with the fermentation rate being modulated by a limitation in <em>p</em>-aminobenzoate biosynthesis due to the presence of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele of the laboratory strain. Since the QTL for N<sub>ass</sub>, overlapped with the QTL for R<sub>max</sub>, we checked whether N<sub>ass</sub> was modulated by the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele. An amino-acid analysis of the fermentation medium revealed that residual assimilable nitrogen was 30 mg/l with the strain carrying the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele from BY4742 <em>vs.</em> 11 mg/l with the hemizygous strain containing the 59A form. These results indicate that nitrogen assimilation is also modulated by the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/263/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reciprocal hemizygous analysis of ABZ1 and impact of p-aminobenzoate on the fermentation profiles. A) Fermentation rate profiles of the two reciprocal hemizygous strains carrying the ABZ1 allele from BY4742 or 59A. The hemizygous strains harbor either an active ABZ1 BY allele (strain BY4742/ABZ1-59A&#x2206;, thick gray line) or the 59A allele (strain 59A/ABZ1- BY4742&#x2206;, black line with diamonds). Profiles of the strains 59A (dark-blue line), the laboratory strain S288C (thin gray line), and the hybrid Z59S (black line) are shown. B) Impact of p-aminobenzoate on the fermentation profiles of hemizygous strains. Fermentation kinetics of two hemizygous strains in MS300 supplemented or not with 1 mg/l of p-aminobenzoic acid. Hemizygous strain carrying S288c ABZ1 allele in MS300 without (black line) or supplemented with p-aminobenzoic acid (gray kinetic line). Hemizygous strain carrying 59A ABZ1 allele in MS300, without (red kinetic line) or supplemented with p-aminobenzoic acid (pink kinetic line). The supplementation with p-aminobenzoic acid improves the fermentation capacity of the hemizygous strain carrying the S288c ABZ1 allele." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452389652" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Reciprocal hemizygous analysis of ABZ1 and impact of &lt;em&gt;p&lt;/em&gt;-aminobenzoate on the fermentation profiles. A) Fermentation rate profiles of the two reciprocal hemizygous strains carrying the &lt;em&gt;ABZ1&lt;/em&gt; allele from BY4742 or 59A. The hemizygous strains harbor either an active &lt;em&gt;ABZ1&lt;/em&gt; BY allele (strain BY4742/&lt;em&gt;ABZ1&lt;/em&gt;-59A&#x2206;, thick gray line) or the 59A allele (strain 59A/&lt;em&gt;ABZ1&lt;/em&gt;- BY4742&#x2206;, black line with diamonds). Profiles of the strains 59A (dark-blue line), the laboratory strain S288C (thin gray line), and the hybrid Z59S (black line) are shown. B) Impact of &lt;em&gt;p&lt;/em&gt;-aminobenzoate on the fermentation profiles of hemizygous strains. Fermentation kinetics of two hemizygous strains in MS300 supplemented or not with 1 mg/l of &lt;em&gt;p&lt;/em&gt;-aminobenzoic acid. Hemizygous strain carrying S288c &lt;em&gt;ABZ1&lt;/em&gt; allele in MS300 without (black line) or supplemented with &lt;em&gt;p&lt;/em&gt;-aminobenzoic acid (gray kinetic line). Hemizygous strain carrying 59A &lt;em&gt;ABZ1&lt;/em&gt; allele in MS300, without (red kinetic line) or supplemented with &lt;em&gt;p&lt;/em&gt;-aminobenzoic acid (pink kinetic line). The supplementation with &lt;em&gt;p&lt;/em&gt;-aminobenzoic acid improves the fermentation capacity of the hemizygous strain carrying the S288c &lt;em&gt;ABZ1&lt;/em&gt; allele.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/263/F5.medium.gif" width="289" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/263/F5.medium.gif" width="289" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/263/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/263/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-34" class="first-child">Reciprocal hemizygous analysis of ABZ1 and impact of <em>p</em>-aminobenzoate on the fermentation profiles. A) Fermentation rate profiles of the two reciprocal hemizygous strains carrying the <em>ABZ1</em> allele from BY4742 or 59A. The hemizygous strains harbor either an active <em>ABZ1</em> BY allele (strain BY4742/<em>ABZ1</em>-59A∆, thick gray line) or the 59A allele (strain 59A/<em>ABZ1</em>- BY4742∆, black line with diamonds). Profiles of the strains 59A (dark-blue line), the laboratory strain S288C (thin gray line), and the hybrid Z59S (black line) are shown. B) Impact of <em>p</em>-aminobenzoate on the fermentation profiles of hemizygous strains. Fermentation kinetics of two hemizygous strains in MS300 supplemented or not with 1 mg/l of <em>p</em>-aminobenzoic acid. Hemizygous strain carrying S288c <em>ABZ1</em> allele in MS300 without (black line) or supplemented with <em>p</em>-aminobenzoic acid (gray kinetic line). Hemizygous strain carrying 59A <em>ABZ1</em> allele in MS300, without (red kinetic line) or supplemented with <em>p</em>-aminobenzoic acid (pink kinetic line). The supplementation with <em>p</em>-aminobenzoic acid improves the fermentation capacity of the hemizygous strain carrying the S288c <em>ABZ1</em> allele.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-35">The genetic determinants of variation for most of the fermentation parameters in yeast are still unknown. In this study, we addressed the genetic basis of several fermentation traits and combined this with gene-expression analysis and an eQTL approach. Using a segregating population of a limited size we identified QTL for several fermentation traits. Accurate measures of kinetic parameters—specifically fermentation rates—allowed us to show that depending on the progress of fermentation, different QTL are associated with different fermentation rates; one for R<sub>max</sub>, and one for R<sub>70</sub>. This is consistent with the notion that, depending on fermentation progress, growth, or starvation, different cellular mechanisms are critical in controlling fermentation flux. On the other hand, these parameters share other components of cell metabolism, which is indicated by their high correlation and strong association with nitrogen utilization. Indeed, we found that R<sub>max</sub> and N<sub>ass</sub> shared a common QTL. This QTL was dissected, and we provided functional evidence that a gene involved in <em>p</em>-aminobenzoate synthesis plays a role in controlling R<sub>max</sub> and N<sub>ass</sub>. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele has a strong effect on the fermentation rate and on nitrogen utilization as visualized in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. We calculated that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> could explain 51% of the variance of R<sub>max</sub> and 45% of the variance of N<sub>ass</sub> in the segregants population.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/4/263/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Impact of the ABZ1 genotype on the fermentation phenotypes Rmax and Nass of the 30 segregants. A) Fermentation rate Rmax of the segregants that inherited the ABZ1 locus from S288C or 59A. B) Nitrogen assimilation (Nass) phenotype of the segregants that inherited the ABZ1 locus from S288C or 59A. The average phenotype and standard deviation are indicated for each genotype group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1452389652" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Impact of the &lt;em&gt;ABZ1&lt;/em&gt; genotype on the fermentation phenotypes R&lt;sub&gt;max&lt;/sub&gt; and N&lt;sub&gt;ass&lt;/sub&gt; of the 30 segregants. A) Fermentation rate R&lt;sub&gt;max&lt;/sub&gt; of the segregants that inherited the &lt;em&gt;ABZ1&lt;/em&gt; locus from S288C or 59A. B) Nitrogen assimilation (N&lt;sub&gt;ass&lt;/sub&gt;) phenotype of the segregants that inherited the &lt;em&gt;ABZ1&lt;/em&gt; locus from S288C or 59A. The average phenotype and standard deviation are indicated for each genotype group.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/4/263/F6.medium.gif" width="243" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/1/4/263/F6.medium.gif" width="243" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/4/263/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/4/263/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460804" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-36" class="first-child">Impact of the <em>ABZ1</em> genotype on the fermentation phenotypes R<sub>max</sub> and N<sub>ass</sub> of the 30 segregants. A) Fermentation rate R<sub>max</sub> of the segregants that inherited the <em>ABZ1</em> locus from S288C or 59A. B) Nitrogen assimilation (N<sub>ass</sub>) phenotype of the segregants that inherited the <em>ABZ1</em> locus from S288C or 59A. The average phenotype and standard deviation are indicated for each genotype group.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The association of a transcriptomic analysis of the segregants population to a classical QTL approach has provided an important value of the study. Given the small size of the population, these data did not intend to decipher the regulatory variations globally. However, they could help at elucidating fermentation QTL and offer new insights into their relationships with gene expression. An outstanding observation was the evidence of overlaps among several fermentation QTL and eQTL hotspots. Two out of the seven identified hotspots overlapped with phenotypes. The other hotspots were not associated with phenotypic QTL. However, only a very small number of phenotypes were investigated, and examination of additional phenotypes is required to address potential associations with other hotspots. Similar associations between eQTL hotspots and phenotypes have been reported in yeast (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Yvert <em>et al.</em> 2003</a>) and other organisms (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fu <em>et al.</em> 2009</a>). They are thought to originate from sequence variations that have broad pleiotropic effects. In the model plant, <em>Arabidopsis thaliana</em>, variation of complex quantitative traits could be explained by a few hotspot genomic regions that controlled various parameters (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fu <em>et al.</em> 2009</a>). In the present study, we show that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele has a strong impact on the fermentation capacity of the strain. Indeed, such physiological changes are expected to be coupled to massive changes in gene expressions, which can explain the hotspot. In this case, the hotspot results from the effects of a strong metabolic alteration and not directly from modification of a general regulator.</p><p id="p-38">Our data show that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele from S288c is unable to support an R<sub>max</sub> that is similar to the industrial form. Because this phenomenon is abolished by the addition of <em>p</em>-aminobenzoate to the fermentation medium, we can infer that a limitation in the flux of this metabolite is responsible for the lower fermentation rate. <em>p</em>-aminobenzoate is an intermediate of the tetrahydrofolate biosynthetic pathway, which leads to a family of cofactors required for one-carbon transfer reactions. These methyl-donor compounds are involved in the synthesis of methionine, serine, and purines (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Thomas and Surdin-Kerjan 1997</a>). A limiting flux in <em>p</em>-aminobenzoate can therefore trigger a limitation in the availability of one of these metabolites. Given the pivotal role of methionine in translation, this may limit the rate of protein synthesis and in turn the incorporation of nitrogen sources. This mechanism is consistent with the observed correlation between R<sub>max</sub> and N<sub>ass</sub>, as well as with the overlap of their QTL. Our results highlight the critical role of methyl-donor biosynthesis in the control of nitrogen utilization during alcoholic fermentation. Interestingly, we did not observe any significant effect of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele on the growth rate of the hemizygous strains (data not shown). This suggests that the methyl-donor pathway is more critical for sugar flux then for cell growth. The difference in nitrogen utilization triggered by the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> alleles is also clearly responsible for the differential expression of genes regulated by nitrogen sources. This explains the correlations of the genes under NCR control, such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005086">MEP2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004132">PUT1</a></em>, with R<sub>max</sub>. Most of the genes linked to the hotspot are however not involved in nitrogen metabolism but respond to various environmental changes including nitrogen depletion (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gasch <em>et al.</em> 2000</a>). This suggests that they correspond to both direct and indirect effects of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> allele. Unexpectedly, we did not observe any deregulation of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> gene itself; it was not differentially expressed in the parental strains and displayed no change in the segregants. A similar lack of differential expression was also noticed for genes downstream <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> in the biosynthetic pathway, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004902">ABZ2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005200">FOL1</a></em> (data not shown). This suggests that the genes of the pathway are not or poorly regulated by the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">Abz1</a> product, 4-amino-4-deoxychorismate, or by downstream metabolites such as <em>p</em>-aminobenzoate.</p><p id="p-39">The <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">Abz1</a> protein from the strain 59A contains five amino acids changes compared with the S288c form. Comparison with the Abz1sequence of <em>Saccharomyces cerevisiae</em> strains from various origins available at the Sanger Institute (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Liti <em>et al.</em> 2009</a>) or in Genbank shows that the same changes are found in other wine yeasts, such as VIN13 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Borneman <em>et al.</em> 2011</a>) and RM11-1a (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ruderfer <em>et al.</em> 2006</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1/FigureS9.pdf">Figure S9</a>). Three of these amino acid changes (T313R, Q650E, N777T) are found in all <em>Saccharomyces cerevisiae</em> strains (except laboratory ones) and a fourth mutation (N475D) in all except 3 of the 39 strains sequenced at the Sanger Institute. Only the L559 was found in strains from different origins [<em>e.g</em>, wild (UWOPS87.2421, UWOPS05.217) or sake (Y12)]. The four common amino acids changes were also found in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> orthologous of other <em>Saccharomyces</em> species, indicating that they correspond to an ancestral form of the gene. This situation is consistent with a divergence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> gene of S288c. This evolution has been likely associated to a partial loss of function of the <em>p</em>-aminobenzoate synthase probably due to a cultivation of the strain in rich laboratory media (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kvitek <em>et al.</em> 2008</a>). This idea is strengthened by additional information obtained from a large-scale analysis of fermentation phenotypes of <em>Saccharomyces</em> strains that included those from the Sanger list (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Liti <em>et al.</em> 2009</a>; Camarasa <em>et al.</em>; paper in preparation). It was observed that alteration of R<sub>max</sub> in the absence of PABA was associated to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">Abz1</a> amino acids that are shared by laboratory strains (Camarasa, personal communication). This is consistent with an evolution <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> toward a defective form in the laboratory strains. The laboratory allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> is probably not heavily defective since it does not lead to a true PABA auxotrophy. The strain S288c grows normally in a minimal medium without PABA with no change observed by comparison with a medium supplemented with 1 mg/l PABA) (data not shown). Such a situation may have facilitated the conservation of an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005316">ABZ1</a></em> form that triggers a phenotype only under specific conditions.</p><p id="p-40">More information based on variation of fermentation parameters and their relationships with expression will certainly be gained from the dissection of the QTL corresponding to the chromosome II hotspot that controls the fermentation rate in the late phase at R<sub>70</sub> and consistently at R<sub>50</sub>. Several genes involved in stress responses (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000615">HSP30</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001880">HSP12</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001181">SPS100</a></em>) whose level of expression correlated negatively with R<sub>70</sub> have a linkage in the hotspot. These stress genes displayed a high level of expression in the laboratory strain and in the progeny that inherited the laboratory-strain form of the region (data not shown). This indicates that the locus from S288c leads to a specific stress response and a low fermentation rate. The potential mechanisms for this observation remain unknown. Our study did not address the expression of the genes acquired by the wine strain EC1118 from a horizontal transfer (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Novo <em>et al.</em> 2009</a>) as these genes were unknown at the beginning of the study. However, we have checked for linkage to makers flanking these regions and did not find any with fermentation traits and only one weak with an eQTL, suggesting that they have no strong impact on the parameters considered here.</p><p id="p-41">The biological device used in the present study with a small population of segregants was able to detect fermentation QTL, but it had a lower power to detect eQTL. An additional difficulty for eQTL detection may originate from the specific physiological conditions under which we analyzed gene expression (<em>i.e.</em>, nongrowing cells under conditions of stress). Until now, yeast eQTL studies have been performed with growing cells. Under starvation and conditions of stress, many genes that respond to environmental changes have an expression more noisy than average (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gasch 2007</a>; Razer <em>et al.</em> 2004). However, because most critical phenomena in industrial alcoholic fermentations (reduction in carbon flux, ethanol inhibition, cell death, <em>etc</em>.) take place during this phase, an assessment of the relationships between variations in gene expression and fermentation traits is required under such conditions. The findings of our study are the first step toward understanding this process. We demonstrated the role of a QTL relevant for alcoholic fermentation performance and provided an initial basis to address the relationships among fermentation QTL and variations in gene expression.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-42">This work was supported by Agence Nationale de la Recherche grant Genyeastrait ANR-07-BLAN-0205. C. Ambroset was supported by a fellowship from the INRA Département Microbiologie et Chaîne Alimentaire (MICA) and the region Languedoc-Roussillon.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-43"><em>Communicating editor: Justin C. Fay</em></p></li><li class="fn" id="fn-4"><p id="p-44">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1SI">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000422/-/DC1SI</a>.</p></li><li class="fn" id="fn-5"><p id="p-45">Sequence data from this article have been deposited in the NCBI Sequence Read Archive as study SRP004704. Gene expression data from this article have been submitted to the Gene Expression Omnibus database under accession no. GSE26437.</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-05-24"><span class="received-label">Received </span>May 24, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-07-20"><span class="accepted-label">Accepted </span>July 20, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Ambroset <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.1" data-doi="10.1371/journal.pgen.1000049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ansel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Beltran</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Damon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cell-to-cell stochastic variation in gene expression is a complex genetic trait.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAnsel%26rft.auinit1%253DJ.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De1000049%26rft.epage%253De1000049%26rft.atitle%253DCell-to-cell%2Bstochastic%2Bvariation%2Bin%2Bgene%2Bexpression%2Bis%2Ba%2Bcomplex%2Bgenetic%2Btrait.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000049%26rft_id%253Dinfo%253Apmid%252F18404214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18404214&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.2" data-doi="10.1016/0922-338X(90)90057-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bely</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sablayrolles</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Automatic detection of assimilable nitrogen deficiencies during alcoholic fermentation in oenological conditions.</span> <abbr class="cit-jnl-abbrev">J. Ferment. Bioeng.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFerment.%2BBioeng.%26rft.volume%253D70%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0922-338X%252890%252990057-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0922-338X(90)90057-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.3" data-doi="10.1371/journal.pgen.0020195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Ari</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenvirth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klutstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Four linked genes participate in controlling sporulation efficiency in budding yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBen-Ari%26rft.auinit1%253DG.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253De195%26rft.epage%253De195%26rft.atitle%253DFour%2Blinked%2Bgenes%2Bparticipate%2Bin%2Bcontrolling%2Bsporulation%2Befficiency%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020195%26rft_id%253Dinfo%253Apmid%252F17112318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17112318&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.4" data-doi="10.1016/S0166-4328(01)00297-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drai</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elmer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kafkafi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golani</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Controlling the false discovery rate in behavior genetics research.</span> <abbr class="cit-jnl-abbrev">Behav. Brain Res.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavioural%2Bbrain%2Bresearch%26rft.stitle%253DBehav%2BBrain%2BRes%26rft.aulast%253DBenjamini%26rft.auinit1%253DY.%26rft.volume%253D125%26rft.issue%253D1-2%26rft.spage%253D279%26rft.epage%253D284%26rft.atitle%253DControlling%2Bthe%2Bfalse%2Bdiscovery%2Brate%2Bin%2Bbehavior%2Bgenetics%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0166-4328%252801%252900297-2%26rft_id%253Dinfo%253Apmid%252F11682119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0166-4328(01)00297-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11682119&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172244000034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-1.4.263.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergmeyer</span>  <span class="cit-name-given-names">H. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beutler</span>  <span class="cit-name-given-names">H. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Ammonia, pp. 454–461 in <em>Methods of Enzymatic Analysis</em></span>, <span class="cit-edition">3rd Ed</span>., <span class="cit-comment">Vol. III, edited by</span> <span class="cit-ed"><span class="cit-name-surname">Bergmeyer</span>  <span class="cit-name-given-names">H. U.</span></span>. <span class="cit-publ-name">VCH Publishers Ltd.</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.6" data-doi="10.1111/j.1567-1364.2008.00434.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative genome analysis of a Saccharomyces cerevisiae wine strain.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00434.x%26rft_id%253Dinfo%253Apmid%252F18778279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2008.00434.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18778279&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260114400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.7" data-doi="10.1371/journal.pgen.1001287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desany</span>  <span class="cit-name-given-names">B. A</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riches</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBorneman%26rft.auinit1%253DA.%2BR.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001287%26rft.epage%253De1001287%26rft.atitle%253DWhole-genome%2Bcomparison%2Breveals%2Bnovel%2Bgenetic%2Belements%2Bthat%2Bcharacterize%2Bthe%2Bgenome%2Bof%2Bindustrial%2Bstrains%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001287%26rft_id%253Dinfo%253Apmid%252F21304888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304888&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.8" data-doi="10.1534/genetics.106.057927"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christianson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping novel traits by array-assisted bulk segregant analysis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D173%26rft.spage%253D1813%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.057927%26rft_id%253Dinfo%253Apmid%252F16624899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.057927&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16624899&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239629400051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.9" data-doi="10.1126/science.1069516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clinton</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic dissection of transcriptional regulation in budding yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">752</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069516%26rft_id%253Dinfo%253Apmid%252F11923494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTYvNTU2OC83NTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.10" data-doi="10.1073/pnas.0408709102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The landscape of genetic complexity across 5,700 gene expression traits in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1572</span>–<span class="cit-lpage">1577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0408709102%26rft_id%253Dinfo%253Apmid%252F15659551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzUvMTU3MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.11" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.12" data-doi="10.1111/j.1365-294X.2008.03765.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalieri</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Cascading transcriptional effects of a naturally occurring frameshift mutation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2985</span>–<span class="cit-lpage">2997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBrown%26rft.auinit1%253DK.%2BM.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D2985%26rft.epage%253D2997%26rft.atitle%253DCascading%2Btranscriptional%2Beffects%2Bof%2Ba%2Bnaturally%2Boccurring%2Bframeshift%2Bmutation%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03765.x%26rft_id%253Dinfo%253Apmid%252F18422925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03765.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18422925&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256494700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calull</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marce</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olucha</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrull</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Optimization of an ion exchange high-performance liquid chromatographic method for determination of carboxylic acids, sugars, glycerol, and ethanol in wines.</span> <abbr class="cit-jnl-abbrev">J. Chromatogr. A</abbr> <span class="cit-vol">589</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChromatogr.%2BA%26rft.volume%253D589%26rft.spage%253D151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.14" data-doi="10.1073/pnas.210395297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavalieri</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townsend</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Manifold anomalies in gene expression in a vineyard isolate of <em>Saccharomyces cerevisiae</em> revealed by DNA microarray analysis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12369</span>–<span class="cit-lpage">12374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.210395297%26rft_id%253Dinfo%253Apmid%252F11035792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjIvMTIzNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.15" data-doi="10.1111/j.1365-294X.2011.05005.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fargier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assessing the complex architecture of complex traits in diverged yeast populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCubillos%26rft.auinit1%253DF.%2BA.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D1401%26rft.epage%253D1413%26rft.atitle%253DAssessing%2Bthe%2Bcomplex%2Barchitecture%2Bof%2Bpolygenic%2Btraits%2Bin%2Bdiverged%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05005.x%26rft_id%253Dinfo%253Apmid%252F21261765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05005.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21261765&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288705300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.16" data-doi="10.1038/ng1674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deutschbauer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait loci mapped to single-nucleotide resolution in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1333</span>–<span class="cit-lpage">1340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDeutschbauer%26rft.auinit1%253DA.%2BM.%26rft.volume%253D37%26rft.issue%253D12%26rft.spage%253D1333%26rft.epage%253D1340%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bmapped%2Bto%2Bsingle-nucleotide%2Bresolution%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1674%26rft_id%253Dinfo%253Apmid%252F16273108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16273108&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233657300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.17" data-doi="10.1371/journal.pgen.1000183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcuera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A catalog of neutral and deleterious polymorphism in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000183%26rft.epage%253De1000183%26rft.atitle%253DA%2Bcatalog%2Bof%2Bneutral%2Band%2Bdeleterious%2Bpolymorphism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000183%26rft_id%253Dinfo%253Apmid%252F18769710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18769710&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.18" data-doi="10.1073/pnas.242624799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badrane</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characteristic genome rearrangements in experimental evolution of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16144</span>–<span class="cit-lpage">16149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.242624799%26rft_id%253Dinfo%253Apmid%252F12446845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjUvMTYxNDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.19" data-doi="10.1186/1471-2164-6-53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Microarray karyotyping of commercial wine yeast strains reveals shared, as well as unique, genomic signatures.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDunn%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.spage%253D53%26rft.epage%253D53%26rft.atitle%253DMicroarray%2Bkaryotyping%2Bof%2Bcommercial%2Bwine%2Byeast%2Bstrains%2Breveals%2Bshared%252C%2Bas%2Bwell%2Bas%2Bunique%252C%2Bgenomic%2Bsignatures.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-6-53%26rft_id%253Dinfo%253Apmid%252F15833139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-6-53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15833139&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.20" data-doi="10.1093/nar/30.1.207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domrachev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lash</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D210%26rft.atitle%253DGene%2BExpression%2BOmnibus%253A%2BNCBI%2Bgene%2Bexpression%2Band%2Bhybridization%2Barray%2Bdata%2Brepository%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.1.207%26rft_id%253Dinfo%253Apmid%252F11752295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.1.207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11752295&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173077100055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.21" data-doi="10.1186/gb-2004-5-4-r26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sniegowski</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Population genetic variation in gene expression is associated with phenotypic variation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253DR26%26rft.epage%253DR26%26rft.atitle%253DPopulation%2Bgenetic%2Bvariation%2Bin%2Bgene%2Bexpression%2Bis%2Bassociated%2Bwith%2Bphenotypic%2Bvariation%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-4-r26%26rft_id%253Dinfo%253Apmid%252F15059259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-4-r26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15059259&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.22" data-doi="10.1038/ng.308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouwmeester</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">America</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstappen</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">System-wide molecular evidence for phenotypic buffering in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFu%26rft.auinit1%253DJ.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D167%26rft.atitle%253DSystem-wide%2Bmolecular%2Bevidence%2Bfor%2Bphenotypic%2Bbuffering%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.308%26rft_id%253Dinfo%253Apmid%252F19169256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19169256&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263091300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.23" data-doi="10.1091/mbc.11.12.4241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spelman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Harel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genomic expression programs in the response of yeast cells to environmental changes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">4241</span>–<span class="cit-lpage">4257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D4241%26rft.epage%253D4257%26rft.atitle%253DGenomic%2BExpression%2BPrograms%2Bin%2Bthe%2BResponse%2Bof%2BYeast%2BCells%2Bto%2BEnvironmental%2BChanges%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.12.4241%26rft_id%253Dinfo%253Apmid%252F11102521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMi80MjQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.24" data-doi="10.1002/yea.1512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparative genomics of the environmental stress response in ascomycete fungi.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D976%26rft.atitle%253DComparative%2Bgenomics%2Bof%2Bthe%2Benvironmental%2Bstress%2Bresponse%2Bin%2Bascomycete%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1512%26rft_id%253Dinfo%253Apmid%252F17605132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17605132&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251549800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.25" data-doi="10.1534/genetics.106.058453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Natural isolates of <em>Saccharomyces cerevisiae</em> display complex genetic variation in sporulation efficiency.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D174%26rft.spage%253D985%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058453%26rft_id%253Dinfo%253Apmid%252F16951083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.058453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16951083&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241798700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.26" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">546</span>, <span class="cit-comment">563–567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.27" data-doi="10.1534/genetics.106.065292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic dissection of ethanol tolerance in the budding yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1479</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D1479%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.065292%26rft_id%253Dinfo%253Apmid%252F17194785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.106.065292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194785&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245590600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Infante</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dombek</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rebordinos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantoral</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">E. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide amplifications caused by chromosomal rearrangements play a major role in the adaptive evolution of natural yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DInfante%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D1745%26rft.epage%253D1759%26rft.atitle%253DGenome-Wide%2BAmplifications%2BCaused%2Bby%2BChromosomal%2BRearrangements%2BPlay%2Ba%2BMajor%2BRole%2Bin%2Bthe%2BAdaptive%2BEvolution%2Bof%2BNatural%2BYeast%26rft_id%253Dinfo%253Apmid%252F14704163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzE3NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzQvMjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-1.4.263.29"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Instituto Nacional de la Investigación en Agronomía (INRA). GenYeasTrait Genome Browser</span>. <span class="cit-comment">Available at: <a href="http://genome.jouy.inra.fr/genyeastrait/">http://genome.jouy.inra.fr/genyeastrait/</a>. Accessed: March 01, 2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-1.4.263.30"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Institut National des Sciences Appliquées. Plateforme Biopuces</span>. <span class="cit-comment">Available at: <a href="http://biopuces.genotoul.fr">http://biopuces.genotoul.fr</a>. Accessed: January 1, 2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.31" data-doi="10.1016/S0168-9525(01)02310-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nap</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetical genomics: the added value from segregation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">388</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DJansen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D17%26rft.issue%253D7%26rft.spage%253D388%26rft.epage%253D391%26rft.atitle%253DGenetical%2Bgenomics%253A%2Bthe%2Badded%2Bvalue%2Bfrom%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252801%252902310-1%26rft_id%253Dinfo%253Apmid%252F11418218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(01)02310-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11418218&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169674900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.32" data-doi="10.1016/j.jbiosc.2008.12.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Namise</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimoi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">QTL mapping of sake brewing characteristics of yeast.</span> <abbr class="cit-jnl-abbrev">J. Biosci. Bioeng.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiosci%2BBioeng%26rft.aulast%253DKatou%26rft.auinit1%253DT.%26rft.volume%253D107%26rft.issue%253D4%26rft.spage%253D383%26rft.epage%253D393%26rft.atitle%253DQTL%2Bmapping%2Bof%2Bsake%2Bbrewing%2Bcharacteristics%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiosc.2008.12.014%26rft_id%253Dinfo%253Apmid%252F19332297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiosc.2008.12.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19332297&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.33" data-doi="10.1038/nature01644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">423</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D423%26rft.issue%253D6937%26rft.spage%253D241%26rft.epage%253D254%26rft.atitle%253DSequencing%2Band%2Bcomparison%2Bof%2Byeast%2Bspecies%2Bto%2Bidentify%2Bgenes%2Band%2Bregulatory%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01644%26rft_id%253Dinfo%253Apmid%252F12748633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12748633&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182853100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.34" data-doi="10.1073/pnas.0708194104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">19387</span>–<span class="cit-lpage">19391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0708194104%26rft_id%253Dinfo%253Apmid%252F18042712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ5LzE5Mzg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.35" data-doi="10.1371/journal.pgen.1000223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Variations in stress sensitivity and genomic expression in diverse <em>S. cerevisiae</em> isolates.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKvitek%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000223%26rft.epage%253De1000223%26rft.atitle%253DVariations%2Bin%2Bstress%2Bsensitivity%2Band%2Bgenomic%2Bexpression%2Bin%2Bdiverse%2BS.%2Bcerevisiae%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000223%26rft_id%253Dinfo%253Apmid%252F18927628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18927628&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.36" data-doi="10.1101/gr.078212.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1851</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078212.108%26rft_id%253Dinfo%253Apmid%252F18714091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.37" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.38" data-doi="10.1111/j.1567-1364.2006.00034.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marullo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bely</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masneuf-Pomarede</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pons</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aigle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Breeding strategies for combining fermentative qualities and reducing off-flavor production in a wine yeast model.</span> <abbr class="cit-jnl-abbrev">FEM. Yeast Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEM.%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2006.00034.x%26rft_id%253Dinfo%253Apmid%252F16487348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2006.00034.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16487348&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235698200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.39" data-doi="10.1371/journal.pgen.0030031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nogami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic complexity and quantitative trait loci mapping of yeast morphological traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNogami%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253De31%26rft.epage%253De31%26rft.atitle%253DGenetic%2Bcomplexity%2Band%2Bquantitative%2Btrait%2Bloci%2Bmapping%2Bof%2Byeast%2Bmorphological%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030031%26rft_id%253Dinfo%253Apmid%252F17319748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17319748&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.40" data-doi="10.1073/pnas.0904673106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigey</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyne</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galeote</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gavory</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Eukaryote-to-eukaryote gene transfer events revealed by the genome sequence of the wine yeast <em>Saccharomyces cerevisiae</em> EC1118.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">16333</span>–<span class="cit-lpage">16338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904673106%26rft_id%253Dinfo%253Apmid%252F19805302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2MzMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.41" data-doi="10.1101/gr.436602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez-Ortin</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puig</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular characterization of a chromosomal rearrangement involved in the adaptive evolution of yeast strains.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1533</span>–<span class="cit-lpage">1539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DPerez-Ortin%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1533%26rft.epage%253D1539%26rft.atitle%253DMolecular%2BCharacterization%2Bof%2Ba%2BChromosomal%2BRearrangement%2BInvolved%2Bin%2Bthe%2BAdaptive%2BEvolution%2Bof%2BYeast%2BStrains%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.436602%26rft_id%253Dinfo%253Apmid%252F12368245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNTMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-1.4.263.42"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing, Vienna, Austria</span>. <span class="cit-comment">Available at: <a href="http://www.R-project.org">www.R-project.org</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.43" data-doi="10.1007/s004380050028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rachidi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multiple Ty-mediated chromosomal translocations lead to karyotype changes in a wine strain of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">261</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DRachidi%26rft.auinit1%253DN.%26rft.volume%253D261%26rft.issue%253D4-5%26rft.spage%253D841%26rft.epage%253D850%26rft.atitle%253DMultiple%2BTy-mediated%2Bchromosomal%2Btranslocations%2Blead%2Bto%2Bkaryotype%2Bchanges%2Bin%2Ba%2Bwine%2Bstrain%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004380050028%26rft_id%253Dinfo%253Apmid%252F10394922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004380050028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10394922&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081118200028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-1.4.263.44" data-doi="10.1126/science.1098641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Razer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Shea</span>  <span class="cit-name-given-names">E. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Control of stochasticity in eukaryotic gene expression.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">1811</span>–<span class="cit-lpage">1814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1098641%26rft_id%253Dinfo%253Apmid%252F15166317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDQvNTY3OC8xODExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.45" data-doi="10.1371/journal.pone.0000678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The evolution of gene expression QTL in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRonald%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De678%26rft.epage%253De678%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bexpression%2BQTL%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000678%26rft_id%253Dinfo%253Apmid%252F17668057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17668057&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.46" data-doi="10.1002/yea.1046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dulau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julien</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide monitoring of wine yeast gene expression during alcoholic fermentation.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1369</span>–<span class="cit-lpage">1385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DRossignol%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D16%26rft.spage%253D1369%26rft.epage%253D1385%26rft.atitle%253DGenome-wide%2Bmonitoring%2Bof%2Bwine%2Byeast%2Bgene%2Bexpression%2Bduring%2Balcoholic%2Bfermentation.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1046%26rft_id%253Dinfo%253Apmid%252F14663829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14663829&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187610000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-1.4.263.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Analyse de l'expression du génome des levures œnologiques en fermentation alcoolique par des approches post-génomiques. Ph.D. Thesis</span>. <span class="cit-publ-name">University Montpellier II. Montpellier</span>, <span class="cit-publ-loc">France</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.48" data-doi="10.1038/ng1859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population genomic analysis of outcrossing and recombination in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRuderfer%26rft.auinit1%253DD.%2BM.%26rft.volume%253D38%26rft.issue%253D9%26rft.spage%253D1077%26rft.epage%253D1081%26rft.atitle%253DPopulation%2Bgenomic%2Banalysis%2Bof%2Boutcrossing%2Band%2Brecombination%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1859%26rft_id%253Dinfo%253Apmid%252F16892060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16892060&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240112100028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-1.4.263.49"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Saccharomyces Genome Database</span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Saccharomyces Genome Database</span>. <span class="cit-comment"><a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>. Accessed: February 1, 2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.50" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-1.4.263.51" data-doi="10.1007/BF00340712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiestl</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gietz</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">High efficiency transformation of intact yeast cells using single stranded nucleic acids as a carrier.</span> <span class="cit-source">Curr. Genet.</span> <span class="cit-vol">16</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DSchiestl%26rft.auinit1%253DR.%2BH.%26rft.volume%253D16%26rft.issue%253D5-6%26rft.spage%253D339%26rft.epage%253D346%26rft.atitle%253DHigh%2Befficiency%2Btransformation%2Bof%2Bintact%2Byeast%2Bcells%2Busing%2Bsingle%2Bstranded%2Bnucleic%2Bacids%2Bas%2Ba%2Bcarrier.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00340712%26rft_id%253Dinfo%253Apmid%252F2692852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00340712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2692852&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989CG90600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.52" data-doi="10.1371/journal.pgen.0020013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Complex genetic interactions in a quantitative trait locus.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSinha%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253De13%26rft.epage%253De13%26rft.atitle%253DComplex%2Bgenetic%2Binteractions%2Bin%2Ba%2Bquantitative%2Btrait%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020013%26rft_id%253Dinfo%253Apmid%252F16462944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16462944&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.53" data-doi="10.1534/genetics.108.092932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascon</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clauder-Munster</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnakumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequential elimination of major-effect contributors identifies additional quantitative trait loci conditioning high-temperature growth in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1661</span>–<span class="cit-lpage">1670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D180%26rft.spage%253D1661%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092932%26rft_id%253Dinfo%253Apmid%252F18780730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.092932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18780730&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-1.4.263.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Limma: linear models for microarray data</span>, pp. <span class="cit-fpage">397</span>–<span class="cit-lpage">420</span> <span class="cit-comment">in</span> <span class="cit-source">Bioinformatics and Computational Biology Solutions using R and Bioconductor</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">V.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Dudoit</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.55" data-doi="10.1016/S1046-2023(03)00155-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Normalization of cDNA microarray data.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DSmyth%26rft.auinit1%253DG.%2BK.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D265%26rft.epage%253D273%26rft.atitle%253DNormalization%2Bof%2BcDNA%2Bmicroarray%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252803%252900155-5%26rft_id%253Dinfo%253Apmid%252F14597310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(03)00155-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14597310&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186547800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.56" data-doi="10.1093/bioinformatics/bti270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The use of within-array replicate spots for assessing differential expression in microarrays experiments.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2067</span>–<span class="cit-lpage">2075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti270%26rft_id%253Dinfo%253Apmid%252F15657102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15657102&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228783000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.57" data-doi="10.1371/journal.pbio.0060083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene-environment interaction in yeast gene expression.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSmith%26rft.auinit1%253DE.%2BN.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De83%26rft.epage%253De83%26rft.atitle%253DGene-environment%2Binteraction%2Bin%2Byeast%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060083%26rft_id%253Dinfo%253Apmid%252F18416601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18416601&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.58" data-doi="10.1101/gr.403602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mungall</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangone</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The generic genome browser: a building block for a model organism system database.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1599</span>–<span class="cit-lpage">1610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DStein%26rft.auinit1%253DL.%2BD.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1599%26rft.epage%253D1610%26rft.atitle%253DThe%2BGeneric%2BGenome%2BBrowser%253A%2BA%2BBuilding%2BBlock%2Bfor%2Ba%2BModel%2BOrganism%2BSystem%2BDatabase%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.403602%26rft_id%253Dinfo%253Apmid%252F12368253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNTk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.59" data-doi="10.1038/416326a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegelman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oefner</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dissecting the architecture of a quantitative trait locus in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteinmetz%26rft.auinit1%253DL.%2BM.%26rft.volume%253D416%26rft.issue%253D6878%26rft.spage%253D326%26rft.epage%253D330%26rft.atitle%253DDissecting%2Bthe%2Barchitecture%2Bof%2Ba%2Bquantitative%2Btrait%2Blocus%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416326a%26rft_id%253Dinfo%253Apmid%252F11907579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416326a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11907579&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174482200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surdin-Kerjan</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Metabolism of sulfur amino acids in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DThomas%26rft.auinit1%253DD.%26rft.volume%253D61%26rft.issue%253D4%26rft.spage%253D503%26rft.epage%253D532%26rft.atitle%253DMetabolism%2Bof%2Bsulfur%2Bamino%2Bacids%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F9409150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2MS80LzUwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.61" data-doi="10.1128/AEM.70.6.3392-3400.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varela</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pizarro</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agosin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Biomass content governs fermentation rate in nitrogen-deficient wine musts.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">3392</span>–<span class="cit-lpage">3400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DVarela%26rft.auinit1%253DC.%26rft.volume%253D70%26rft.issue%253D6%26rft.spage%253D3392%26rft.epage%253D3400%26rft.atitle%253DBiomass%2BContent%2BGoverns%2BFermentation%2BRate%2Bin%2BNitrogen-Deficient%2BWine%2BMusts%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.70.6.3392-3400.2004%26rft_id%253Dinfo%253Apmid%252F15184136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjcwLzYvMzM5MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNC8yNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.62" data-doi="10.1073/pnas.0701291104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome sequencing and comparative analysis of <em>Saccharomyces cerevisiae</em> strain YJM789.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">12825</span>–<span class="cit-lpage">12830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701291104%26rft_id%253Dinfo%253Apmid%252F17652520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzMxLzEyODI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.63" data-doi="10.1126/science.281.5380.1194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conway</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Direct allelic variation scanning of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">1194</span>–<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D281%26rft.issue%253D5380%26rft.spage%253D1194%26rft.epage%253D1197%26rft.atitle%253DDirect%2BAllelic%2BVariation%2BScanning%2Bof%2Bthe%2BYeast%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.281.5380.1194%26rft_id%253Dinfo%253Apmid%252F9712584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODEvNTM4MC8xMTk0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Whole genome genetic-typing in yeast using high-density oligonucleotide arrays.</span> <abbr class="cit-jnl-abbrev">Parasitology</abbr> <span class="cit-vol">118</span> <span class="cit-comment">Suppl</span>: <span class="cit-fpage">S73</span>–<span class="cit-lpage">S80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasitology%26rft.volume%253D118%26rft.spage%253DS73%26rft_id%253Dinfo%253Apmid%252F10466139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10466139&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082170500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.65" data-doi="10.1038/ng1222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foss</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Trans-acting regulatory variation in <em>Saccharomyces cerevisiae</em> and the role of transcription factors.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYvert%26rft.auinit1%253DG.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D64%26rft.atitle%253DTrans-acting%2Bregulatory%2Bvariation%2Bin%2BSaccharomyces%2Bcerevisiae%2Band%2Bthe%2Brole%2Bof%2Btranscription%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1222%26rft_id%253Dinfo%253Apmid%252F12897782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12897782&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F4%2F263.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185018500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-1.4.263.66" data-doi="10.1128/AEM.72.1.836-847.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuzuarregui</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monteoliva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gil</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Del Olmo</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Transcriptomic and proteomic approach for understanding the molecular basis of adaptation of <em>Saccharomyces cerevisiae</em> to wine fermentation.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DZuzuarregui%26rft.auinit1%253DA.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D836%26rft.epage%253D847%26rft.atitle%253DTranscriptomic%2Band%2BProteomic%2BApproach%2Bfor%2BUnderstanding%2Bthe%2BMolecular%2BBasis%2Bof%2BAdaptation%2Bof%2BSaccharomyces%2Bcerevisiae%2Bto%2BWine%2BFermentation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.72.1.836-847.2006%26rft_id%253Dinfo%253Apmid%252F16391125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjg6IjcyLzEvODM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS80LzI2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/4/263.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/4/259" title="Variable Transposition of Eight Maize Activator (Ac) Elements Located on the Short Arm of Chromosome 1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/4/283" title="Identification and Precise Mapping of Resistant QTLs of Cercospora Leaf Spot Resistance in Sugar Beet (Beta vulgaris L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/4"><span>Volume 1 Issue 4, September 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/4.cover.gif?itok=yDWwhPU2" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/4/263" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yl8EOQ0E_jls4982a3ibeioyOT3WjJNE-aN92lY-T9E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/4/263" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/4/263" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/4/263" class="active" data-icon-position="" data-hide-link-title="0">Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459175" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TTC9lUOculvcMvkajbMGaFqAb2_9jbadbs30Pl-pNjQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/4/263.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/4/263" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-HkMDldpnnwmiwdz-xrFzhXiEYe9XidGuJyey2UCEQCY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/4/263" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/4/263.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459175" id="node459175--2" data-pisa="ggg;1/4/263" data-pisa-master="ggg;g3.111.000422" data-apath="/ggg/1/4/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chloé Ambroset</span>, <span class="highwire-citation-author" data-delta="1">Maud Petit</span>, <span class="highwire-citation-author" data-delta="2">Christian Brion</span>, <span class="highwire-citation-author" data-delta="3">Isabelle Sanchez</span>, <span class="highwire-citation-author" data-delta="4">Pierre Delobel</span>, <span class="highwire-citation-author" data-delta="5">Cyprien Guérin</span>, <span class="highwire-citation-author" data-delta="6">Hélène Chiapello</span>, <span class="highwire-citation-author" data-delta="7">Pierre Nicolas</span>, <span class="highwire-citation-author" data-delta="8">Frédéric Bigey</span>, <span class="highwire-citation-author" data-delta="9">Sylvie Dequin</span> and <span class="highwire-citation-author" data-delta="10">Bruno Blondin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-281;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000422 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459175--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chloé Ambroset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chlo%C3%A9%2BAmbroset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ambroset%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChlo%25C3%25A9%2BAmbroset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maud Petit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maud%2BPetit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petit%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaud%2BPetit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christian Brion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BBrion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brion%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BBrion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Isabelle Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BSanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanchez%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BSanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Pierre Delobel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BDelobel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delobel%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APierre%2BDelobel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cyprien Guérin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyprien%2BGu%C3%A9rin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%C3%A9rin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyprien%2BGu%25C3%25A9rin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hélène Chiapello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BChiapello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiapello%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BChiapello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Pierre Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APierre%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Frédéric Bigey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fr%C3%A9d%C3%A9ric%2BBigey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bigey%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFr%25C3%25A9d%25C3%25A9ric%2BBigey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Sylvie Dequin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sylvie%2BDequin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dequin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASylvie%2BDequin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Bruno Blondin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Montpellier SupAgro, UMR 1083 Sciences pour l'Oenologie, F-34060 Montpellier, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BBlondin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blondin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BBlondin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:blondin@supagro.inra.fr" class="" data-icon-position="" data-hide-link-title="0">blondin@supagro.inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/4/263" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/4/263&amp;title=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/4/263&amp;title=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/4/263&amp;title=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/4/263&amp;text=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/4/263&amp;title=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/4/263&amp;t=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/4/263&amp;title=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/4/263&amp;title=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459175/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459175" id="node459175--3" data-pisa="ggg;1/4/263" data-pisa-master="ggg;g3.111.000422" data-apath="/ggg/1/4/263.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Deciphering the Molecular Basis of Wine Yeast Fermentation Traits Using a Combined Genetic and Genomic Approach</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chloé Ambroset</span>, <span class="highwire-citation-author" data-delta="1">Maud Petit</span>, <span class="highwire-citation-author" data-delta="2">Christian Brion</span>, <span class="highwire-citation-author" data-delta="3">Isabelle Sanchez</span>, <span class="highwire-citation-author" data-delta="4">Pierre Delobel</span>, <span class="highwire-citation-author" data-delta="5">Cyprien Guérin</span>, <span class="highwire-citation-author" data-delta="6">Hélène Chiapello</span>, <span class="highwire-citation-author" data-delta="7">Pierre Nicolas</span>, <span class="highwire-citation-author" data-delta="8">Frédéric Bigey</span>, <span class="highwire-citation-author" data-delta="9">Sylvie Dequin</span> and <span class="highwire-citation-author" data-delta="10">Bruno Blondin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">263-281;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000422 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459175--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chloé Ambroset </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chlo%C3%A9%2BAmbroset%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ambroset%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChlo%25C3%25A9%2BAmbroset%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maud Petit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maud%2BPetit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petit%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaud%2BPetit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christian Brion </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BBrion%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brion%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BBrion%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Isabelle Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BSanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanchez%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BSanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Pierre Delobel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BDelobel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delobel%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APierre%2BDelobel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Cyprien Guérin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyprien%2BGu%C3%A9rin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gu%C3%A9rin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyprien%2BGu%25C3%25A9rin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hélène Chiapello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BChiapello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiapello%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BChiapello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Pierre Nicolas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>INRA, UR1077 Mathématique Informatique et Génome, F-78352 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BNicolas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nicolas%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APierre%2BNicolas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Frédéric Bigey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fr%C3%A9d%C3%A9ric%2BBigey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bigey%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFr%25C3%25A9d%25C3%25A9ric%2BBigey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Sylvie Dequin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut National de la Recherche Agronomique (INRA), UMR1083, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sylvie%2BDequin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dequin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASylvie%2BDequin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Bruno Blondin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Montpellier SupAgro, UMR 1083 Sciences pour l'Oenologie, F-34060 Montpellier, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BBlondin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blondin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BBlondin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:blondin@supagro.inra.fr" class="" data-icon-position="" data-hide-link-title="0">blondin@supagro.inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=263&amp;rft.epage=281&amp;rft.atitle=Deciphering%20the%20Molecular%20Basis%20of%20Wine%20Yeast%20Fermentation%20Traits%20Using%20a%20Combined%20Genetic%20and%20Genomic%20Approach&amp;rft.volume=1&amp;rft.issue=4&amp;rft.date=2011-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ambroset%2C+Chlo%C3%A9&amp;rft.au=Petit%2C+Maud&amp;rft.au=Brion%2C+Christian&amp;rft.au=Sanchez%2C+Isabelle&amp;rft.au=Delobel%2C+Pierre&amp;rft.au=Gu%C3%A9rin%2C+Cyprien&amp;rft.au=Chiapello%2C+H%C3%A9l%C3%A8ne&amp;rft.au=Nicolas%2C+Pierre&amp;rft.au=Bigey%2C+Fr%C3%A9d%C3%A9ric&amp;rft.au=Dequin%2C+Sylvie&amp;rft.au=Blondin%2C+Bruno"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459175/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459175/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459175/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459175/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459175/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459175/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459175/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459175/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459175/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459175/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459175/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459175/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/4/263" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/4/263.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/4/263.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/4/263.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/4/263.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
