<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/9/1511" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/9/1511.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/9/1511.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/9/1511.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/9/1511/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/9/1511.atom" />
<meta name="HW.pisa" content="ggg;3/9/1511" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data" />
<meta name="DC.Identifier" content="10.1534/g3.113.007096" />
<meta name="DC.Date" content="2013-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Kärkkäinen and Sillanpää. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Because of the increased availability of genome-wide sets of molecular markers along with reduced cost of genotyping large samples of individuals, genomic estimated breeding values have become an essential resource in plant and animal breeding. Bayesian methods for breeding value estimation have proven to be accurate and efficient; however, the ever-increasing data sets are placing heavy demands on the parameter estimation algorithms. Although a commendable number of fast estimation algorithms are available for Bayesian models of continuous Gaussian traits, there is a shortage for corresponding models of discrete or censored phenotypes. In this work, we consider a threshold approach of binary, ordinal, and censored Gaussian observations for Bayesian multilocus association models and Bayesian genomic best linear unbiased prediction and present a high-speed generalized expectation maximization algorithm for parameter estimation under these models. We demonstrate our method with simulated and real data. Our example analyses suggest that the use of the extra information present in an ordered categorical or censored Gaussian data set, instead of dichotomizing the data into case-control observations, increases the accuracy of genomic breeding values predicted by Bayesian multilocus association models or by Bayesian genomic best linear unbiased prediction. Furthermore, the example analyses indicate that the correct threshold model is more accurate than the directly used Gaussian model with a censored Gaussian data, while with a binary or an ordinal data the superiority of the threshold model could not be confirmed." />
<meta name="DC.Contributor" content="Hanni P. Kärkkäinen" />
<meta name="DC.Contributor" content="Mikko J. Sillanpää" />
<meta name="article:published_time" content="2013-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Because of the increased availability of genome-wide sets of molecular markers along with reduced cost of genotyping large samples of individuals, genomic estimated breeding values have become an essential resource in plant and animal breeding. Bayesian methods for breeding value estimation have proven to be accurate and efficient; however, the ever-increasing data sets are placing heavy demands on the parameter estimation algorithms. Although a commendable number of fast estimation algorithms are available for Bayesian models of continuous Gaussian traits, there is a shortage for corresponding models of discrete or censored phenotypes. In this work, we consider a threshold approach of binary, ordinal, and censored Gaussian observations for Bayesian multilocus association models and Bayesian genomic best linear unbiased prediction and present a high-speed generalized expectation maximization algorithm for parameter estimation under these models. We demonstrate our method with simulated and real data. Our example analyses suggest that the use of the extra information present in an ordered categorical or censored Gaussian data set, instead of dichotomizing the data into case-control observations, increases the accuracy of genomic breeding values predicted by Bayesian multilocus association models or by Bayesian genomic best linear unbiased prediction. Furthermore, the example analyses indicate that the correct threshold model is more accurate than the directly used Gaussian model with a censored Gaussian data, while with a binary or an ordinal data the superiority of the threshold model could not be confirmed.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/09/01" />
<meta name="citation_mjid" content="ggg;3/9/1511" />
<meta name="citation_id" content="3/9/1511" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/9/1511" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/9/1511.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/9/1511.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/9/1511.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007096" />
<meta name="citation_pmid" content="23821618" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1511" />
<meta name="citation_lastpage" content="1523" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hanni P. Kärkkäinen" />
<meta name="citation_author_institution" content="Department of Agricultural Sciences, University of Helsinki, Helsinki FIN-00014" />
<meta name="citation_author_institution" content="Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland" />
<meta name="citation_author" content="Mikko J. Sillanpää" />
<meta name="citation_author_institution" content="Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland" />
<meta name="citation_author_institution" content="Department of Biology and Biocenter Oulu, University of Oulu, Oulu FIN-90014, Finland" />
<meta name="citation_author_email" content="mjs@rolf.helsinki.fi" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=J. H. Albert;citation_author=S. Chib;citation_title=Bayesian analysis of binary and polychotomous response data.;citation_pages=669-679;citation_volume=88;citation_year=1993;citation_doi=10.2307/2290350" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=H. H. Axelsson;citation_author=W. F. Fikse;citation_author=M. Kargo;citation_author=A. C. Sørensen;citation_author=K. Johansson;citation_title=Genomic selection using indicator traits to reduce the environmental impact of milk production.;citation_pages=1-9;citation_volume=96;citation_year=2013;citation_pmid=23141821;citation_doi=10.3168/jds.2012-5409" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. W. Broman;citation_title=Mapping Quantitative Trait Loci in the Case of a Spike in the Phenotype Distribution;citation_pages=1169-1175;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663553" />
<meta name="citation_reference" content="citation_title=The Inverse Gaussian distribution: Theory, Methodology, and Applications;citation_author=R. Chhikara;citation_author=L. Folks;citation_year=1989" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=M. A. Cleveland;citation_author=J. M. Hickey;citation_author=S. Forni;citation_title=A Common Dataset for Genomic Analysis of Livestock Populations;citation_pages=429-435;citation_volume=2;citation_year=2012;citation_issue=4;citation_pmid=22540034;citation_doi=10.1534/g3.111.001453" />
<meta name="citation_reference" content="citation_title=Analysis of Binary Data;citation_author=D. Cox;citation_author=E. J. Snell;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=A. P. Dempster;citation_author=N. M. Laird;citation_author=D. B. Rubin;citation_title=Maximum likelihood from incomplete data via the EM algorithm.;citation_pages=1-38;citation_volume=39;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=Anim. Front.;citation_author=A. Eggen;citation_title=The development and application of genomic selection as a new breeding paradigm.;citation_pages=10-15;citation_volume=2;citation_year=2012;citation_doi=10.2527/af.2011-0027" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=J. Fan;citation_author=J. Lv;citation_title=Sure independence screening for ultrahigh dimensional feature space.;citation_pages=849-911;citation_volume=70;citation_year=2008;citation_doi=10.1111/j.1467-9868.2008.00674.x" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.;citation_author=M. A. T. Figueiredo;citation_title=Adaptive sparseness for supervised learning.;citation_pages=1150-1159;citation_volume=25;citation_year=2003;citation_doi=10.1109/TPAMI.2003.1227989" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=A. Gelman;citation_title=Parameterization and Bayesian modeling.;citation_pages=537-545;citation_volume=99;citation_year=2004;citation_doi=10.1198/016214504000000458" />
<meta name="citation_reference" content="citation_title=Bayesian Data Analysis;citation_author=A. Gelman;citation_author=J. B. Carlin;citation_author=H. S. Stern;citation_author=D. B. Rubin;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_title=Theory and Analysis of Threshold Characters;citation_pages=1079-1096;citation_volume=54;citation_year=1982;citation_issue=5;citation_doi=10.2527/jas1982.5451079x" />
<meta name="citation_reference" content="citation_title=Markov Chain Monte Carlo in Practice;citation_author=W. Gilks;citation_author=S. Richardson;citation_author=D. Spiegelhalter;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. González-Recio;citation_author=S. Forni;citation_title=Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.;citation_pages=7-7;citation_volume=43;citation_year=2011;citation_pmid=21329522;citation_doi=10.1186/1297-9686-43-7" />
<meta name="citation_reference" content="citation_title=BMC Proc.;citation_author=O. González-Recio;citation_author=E. L. de Maturana;citation_author=A. T. Vega;citation_author=C. D. Engelman;citation_author=K. W. Broman;citation_title=Detecting single-nucleotide polymorphism by single-nucleotide polymorphism interactions in rheumatoid arthritis using a two-step approach with machine learning and a Bayesian threshold least absolute shrinkage and selection operator (LASSO) model.;citation_pages=S63;citation_volume=3;citation_year=2009;citation_pmid=20018057;citation_doi=10.1186/1753-6561-3-s7-s63" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. A. Hackett;citation_author=J. I. Weller;citation_title=Genetic mapping of quantitative trait loci for traits with ordinal distributions.;citation_pages=1252-1263;citation_volume=51;citation_year=1995;citation_issue=4;citation_pmid=8589221;citation_doi=10.2307/2533257" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Hallander;citation_author=P. Waldmann;citation_author=C. Wang;citation_author=M. J. Sillanpää;citation_title=Bayesian inference of genetic parameters based on conditional decompositions of multivariate normal distributions.;citation_volume=185;citation_year=2010;citation_pmid=20351218;citation_doi=10.1534/genetics.110.114249" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=T. Hayashi;citation_author=H. Iwata;citation_title=EM algorithm for Bayesian estimation of genomic breeding values.;citation_pages=3-3;citation_volume=11;citation_year=2010;citation_pmid=20092655" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=F. Hoti;citation_author=M. J. Sillanpää;citation_title=Bayesian mapping of genotype x expression interactions in quantitative and qualitative traits.;citation_pages=4-18;citation_volume=97;citation_year=2006;citation_issue=1;citation_pmid=16670709;citation_doi=10.1038/sj.hdy.6800817" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Iwata;citation_author=K. Ebana;citation_author=S. Fukuoka;citation_author=J.-L. Jannink;citation_author=T. Hayashi;citation_title=Bayesian multilocus association mapping on ordinal and censored traits and its application to the analysis of genetic variation among Oryza sativa L. germplasms.;citation_pages=865-880;citation_volume=118;citation_year=2009;citation_issue=5;citation_pmid=19132337;citation_doi=10.1007/s00122-008-0945-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. P. Kärkkäinen;citation_author=M. J. Sillanpää;citation_title=Back to basics for Bayesian model building in genomic selection.;citation_volume=191;citation_year=2012;citation_pmid=22554888;citation_doi=10.1534/genetics.112.139014" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=H. P. Kärkkäinen;citation_author=M. J. Sillanpää;citation_title=Robustness of bayesian multilocus association models to cryptic relatedness.;citation_pages=510-523;citation_volume=76;citation_year=2012;citation_issue=6;citation_pmid=22971009;citation_doi=10.1111/j.1469-1809.2012.00729.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J. M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_title=BMC Proc.;citation_author=M. S. Lund;citation_author=G. Sahana;citation_author=D.-J. de Koning;citation_author=G. Su;citation_author=Ö. Carlborg;citation_title=Comparison of analyses of the QTLMAS XII common dataset. I: Genomic selection.;citation_pages=S1;citation_volume=3;citation_year=2009" />
<meta name="citation_reference" content="citation_title=Generalized Linear Models;citation_author=P. McCullagh;citation_author=J. A. Nelder;citation_year=1989" />
<meta name="citation_reference" content="citation_title=The EM Algorithm and Extensions;citation_author=G. J. McLachlan;citation_author=T. Krishnan;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. Meijering;citation_author=D. Gianola;citation_title=Linear vs. nonlinear methods of sire evaluation for categorical traits: a simulation study.;citation_pages=115-132;citation_volume=17;citation_year=1985;citation_pmid=22879189;citation_doi=10.1051/gse:19850107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. H. E. Meuwissen;citation_author=T. R. Solberg;citation_author=R. Shepherd;citation_author=J. A. Woolliams;citation_title=A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value.;citation_pages=2-2;citation_volume=41;citation_year=2009;citation_pmid=19284681;citation_doi=10.1186/1297-9686-41-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. M. Mutshinda;citation_author=M. J. Sillanpää;citation_title=Extended Bayesian LASSO for multiple quantitative trait loci mapping and unobserved phenotype prediction.;citation_volume=186;citation_year=2010;citation_pmid=20805559;citation_doi=10.1534/genetics.110.119586" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=A. Nakaya;citation_author=S. N. Isobe;citation_title=Will genomic selection be a practical method for plant breeding?;citation_volume=110;citation_year=2012;citation_pmid=22645117;citation_doi=10.1093/aob/mcs109" />
<meta name="citation_reference" content="citation_title=Learning in Graphical Models;citation_title=A view of the EM algorithm that justifies incremental, sparse, and other variants;citation_author=R. M. Neal;citation_author=G. E. Hinton;citation_pages=355-368;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The Bayesian Lasso.;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=L. Pedersen;citation_author=A. Sørensen;citation_author=M. Henryon;citation_author=S. Ansari-Mahyari;citation_author=P. Berg;citation_title=ADAM: a computer program to simulate selective breeding schemes for animals.;citation_pages=343-344;citation_volume=121;citation_year=2009;citation_doi=10.1016/j.livsci.2008.06.028" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. E. Powell;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Reconciling the analysis of IBD and IBS in complex trait studies.;citation_pages=800-805;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20877324;citation_doi=10.1038/nrg2865" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Muñz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard data set of loblolly pine (Pinus taeda L.).;citation_volume=190;citation_year=2012;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. K. Shepherd;citation_author=T. H. Meuwissen;citation_author=J. A. Woolliams;citation_title=Genomic selection and complex trait prediction using a fast EM algorithm applied to genome-wide markers.;citation_pages=529-529;citation_volume=11;citation_year=2010;citation_pmid=20969788;citation_doi=10.1186/1471-2105-11-529" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. J. Sillanpää;citation_author=F. Hoti;citation_title=Mapping Quantitative Trait Loci From a Single-Tail Sample of the Phenotype Distribution Including Survival Data;citation_pages=2361-2377;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073434;citation_doi=10.1534/genetics.107.081299" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. A. Sorensen;citation_author=S. Andersen;citation_author=D. Gianola;citation_author=I. Korsgaard;citation_title=Bayesian inference in threshold models using Gibbs sampling.;citation_pages=229-249;citation_volume=27;citation_year=1995;citation_doi=10.1051/gse:19950303" />
<meta name="citation_reference" content="citation_journal_title=Acta Agric. Scand.;citation_author=D. A. Sorensen;citation_author=D. Gianola;citation_author=I. R. Korsgaard;citation_title=Bayesian mixed-effects model analysis of a censored normal distribution with animal breeding applications.;citation_pages=222-229;citation_volume=48;citation_year=1998;citation_doi=10.1080/09064719809362502" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Sun;citation_author=J. G. Ibrahim;citation_author=F. Zou;citation_title=Genomewide multiple-loci mapping in experimental crosses by iterative adaptive penalized regression.;citation_volume=185;citation_year=2010;citation_pmid=20157003;citation_doi=10.1534/genetics.110.114280" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=X. Sun;citation_author=L. Qu;citation_author=D. J. Garrick;citation_author=J. C. M. Dekkers;citation_author=R. L. Fernando;citation_title=A Fast EM Algorithm for BayesA-Like Prediction of Genomic Breeding Values.;citation_pages=e49157-e49157;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23152868;citation_doi=10.1371/journal.pone.0049157" />
<meta name="citation_reference" content="citation_journal_title=Econometrica;citation_author=J. Tobin;citation_title=Estimation of relationships for limited dependent variables.;citation_pages=24-36;citation_volume=26;citation_year=1958;citation_doi=10.2307/1907382" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. G. Usai;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=LASSO with cross-validation for genomic selection.;citation_pages=427-436;citation_volume=91;citation_year=2009;citation_issue=6;citation_pmid=20122298;citation_doi=10.1017/S0016672309990334" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C.-L. Wang;citation_author=X.-D. Ding;citation_author=J.-Y. Wang;citation_author=J.-F. Liu;citation_author=W.-X. Fu;citation_title=Bayesian methods for estimating GEBVs of threshold traits.;citation_pages=213-219;citation_volume=110;citation_year=2013;citation_pmid=23149458;citation_doi=10.1038/hdy.2012.65" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Estimating Polygenic Effects Using Markers of the Entire Genome;citation_pages=789-801;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618414" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=S. Xu;citation_title=An expectation-maximization algorithm for the Lasso estimation of quantitative trait locus effects.;citation_pages=483-494;citation_volume=105;citation_year=2010;citation_issue=5;citation_pmid=20051978;citation_doi=10.1038/hdy.2009.180" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Yi;citation_author=S. Xu;citation_title=Bayesian LASSO for quantitative trait loci mapping.;citation_volume=179;citation_year=2008;citation_pmid=18505874;citation_doi=10.1534/genetics.107.085589" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
<meta name="twitter:description" content="Because of the increased availability of genome-wide sets of molecular markers along with reduced cost of genotyping large samples of individuals, genomic estimated breeding values have become an essential resource in plant and animal breeding. Bayesian methods for breeding value estimation have proven to be accurate and efficient; however, the ever-increasing data sets are placing heavy demands on the parameter estimation algorithms. Although a commendable number of fast estimation algorithms are available for Bayesian models of continuous Gaussian traits, there is a shortage for corresponding models of discrete or censored phenotypes. In this work, we consider a threshold approach of binary, ordinal, and censored Gaussian observations for Bayesian multilocus association models and Bayesian genomic best linear unbiased prediction and present a high-speed generalized expectation maximization algorithm for parameter estimation under these models. We demonstrate our method with simulated and real data. Our example analyses suggest that the use of the extra information present in an ordered categorical or censored Gaussian data set, instead of dichotomizing the data into case-control observations, increases the accuracy of genomic breeding values predicted by Bayesian multilocus association models or by Bayesian genomic best linear unbiased prediction. Furthermore, the example analyses indicate that the correct threshold model is more accurate than the directly used Gaussian model with a censored Gaussian data, while with a binary or an ordinal data the superiority of the threshold model could not be confirmed." />
<meta name="og-title" property="og:title" content="Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/9/1511" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Because of the increased availability of genome-wide sets of molecular markers along with reduced cost of genotyping large samples of individuals, genomic estimated breeding values have become an essential resource in plant and animal breeding. Bayesian methods for breeding value estimation have proven to be accurate and efficient; however, the ever-increasing data sets are placing heavy demands on the parameter estimation algorithms. Although a commendable number of fast estimation algorithms are available for Bayesian models of continuous Gaussian traits, there is a shortage for corresponding models of discrete or censored phenotypes. In this work, we consider a threshold approach of binary, ordinal, and censored Gaussian observations for Bayesian multilocus association models and Bayesian genomic best linear unbiased prediction and present a high-speed generalized expectation maximization algorithm for parameter estimation under these models. We demonstrate our method with simulated and real data. Our example analyses suggest that the use of the extra information present in an ordered categorical or censored Gaussian data set, instead of dichotomizing the data into case-control observations, increases the accuracy of genomic breeding values predicted by Bayesian multilocus association models or by Bayesian genomic best linear unbiased prediction. Furthermore, the example analyses indicate that the correct threshold model is more accurate than the directly used Gaussian model with a censored Gaussian data, while with a binary or an ordinal data the superiority of the threshold model could not be confirmed." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
    <title>Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KHzoBXu7CJarek6GtnIJif51wuwpxy6ixTFhPbMYdAM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459533","apath":"\/ggg\/3\/9\/1511.atom","pisa":"ggg;3\/9\/1511","ac":{"\/ggg\/3\/9\/1511.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/9\/1511.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/9\/1511"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/9\/1511"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/9\/1511":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CFz8iHUJJe9AjM2yGlF8UirDwoomDdbhRe\\\\\\\/5zrIdBwgrsw\\\\\\\/BprzaNoe2Ut7fz+1vROb03pLwc7aI\\\\\\\/VYW2Jpk+xUzp4wXwV3cmjfpLczzrpZE6+AB0rKYVidqNFjI55NiTGilioyyxL3DSCv+GVhzu\\\\\\\/ZPOhmtZQhZhRH3Y2rcr2LxF7rvpu1YvHDa5XQTtU4CwQZx4kpq1ecnuhHpskThw8C0zycrbSmpieyFDYNGZdsn1TOZhiMqaHj\\\\\\\/+LQV2fNYshHOT7N\\\\\\\/zVt2XrA+C27+LbwYQ72xyR4BX1\\\\\\\/KZgCzofw\\\\\\\/5\\\\\\\/GFC0bFkR9DdqM84sLiIevOlgO2C53T\\\\\\\/vEl3eB+Y\\\\\\\/Ayno7HzZNtZOrT\\\\\\\/ihTDMPE46\\\\\\\/45boTgStl5XaIBHrUVr7n+S5ZnD5MtuSh8Sv7ehU0Rwr1QTM7V783nolevXi5mLjYCNEghTNbDV6RNl3\\\\\\\/StAzCVOi77JjXinLFLVQlFo25OYYxgpjpi8RkT4rl1W62SHrpDdg849cTYpZun8n6LCDnNe6wCR+HQTKl4mkv2IIzQ3GPS8dud5B55tnssH1x+BSgJUiuSGeb6WG7Thgrbi4Dvr0XatK50xotqZ+EEBKLHssgKoIyJe+Ymds63wgRfD3L+SkREM4tZr9WANg78okFDiU1OYSjSMG0uGX5s7B2reBLr13iBGRB\\\\\\\/xCI3bWIrZjX3QK23X+VR++zGLAhH4Ic\\\\\\\/Qyt6doeJzcuvkjwCXhA8V+fC9b\\\\\\\/pkBaWEf4NVVRxLwGJuOHVd2zPaSHrCsDxzVxxyAVgZqkCCMldWb7yNsSVdOpZISUe6ll46SZ66oMBZB+zCYAlrUWPGtKrdW7RUAHctB\\\\\\\/Dxc6f57\\\\\\\/uRIYK6Qr\\\\\\\/kk78mE0Dcr\\\\\\\/smm9+60Vvs326X9BhnPIj2liN2RGLVrwb7ZBVOp\\\\\\\/KClLJzWGqoufCS2UYpz4T7o+837iJTZrEksTkOEdJhkQeVLotcjLxKiZ+xmXxPwRCUQM0dhTnXgheAcptN5doFMNgmDnhSP0Y1lQQyg2pSdC1wLjFbgTH4ZdQReU4NNtg8Q8TyiAtXzM6k6hMoRGNH660eUi+Yn7QuKAEjXibhess31IEaZ2AzjBhRwF2slzRraCvK0DRiVpgi2CHHBaMJhb\\\\\\\/Jtl3XfNqaKVz6KtqUBAq3s6o1JMLqz9LaP3TE2pu1xrxCMqy5Vb2UTH5\\\\\\\/t3BA\\\\\\\/dFGukpjBE4oK4JIH4cqH8pDmiHqioz9pzSuoliwNihewHZsxTWT\\\\\\\/F\\\\\\\/jr5QpuDQtLlNXagNQxFsK4Xi3KMhcfPL\\\\\\\/+X7es\\\u0022,\\\u0022iv\\\u0022:\\\u0022n+mYaIongGgTbxX2KK7g0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f93b1b5e2bd72de8a0f8dc3970d89d1967a5991480e60b1c9aa18d857cdc1187\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xoNnD3hyzMCtlklFNpfgp9lbBTgPY2t9SBeAsFUvEIGQ6FUMRsKyI2bxljumcCMWC8Jm+91mEj5beXp\\\\\\\/KXivdx5FjZFlAXs8wKH9nLzaCxnBE7Bx\\\\\\\/tywBlsaBceU5aeyvIk2\\\\\\\/HTGNcKmYUv3P\\\\\\\/M7c0Lh5yloC8XAC9GoBMWV9acqtxOLjPMFOYNd7k0i1WSvECXQmrUhGX9bcuAtw7SAYygCtEG0wfDqd49hcm+asioeghshjqQgO0jpwaB4XqadV8JQYenLAkxwExNgJmFmvfIVocZHvNn+TwWzgRJ2Gwu0f58qwfaIoywZshNeYovsNyVNDa7bsC1+Jt4dfLpdFBGbNQcac83QsFu8PAo4RAYDVwANTJ9Lq3gyzjrJriSo3kq\\\\\\\/awockVIrwV7sTYS9lVS6ydik0jBIWdj2\\\\\\\/BQfV6sJlYC1CbU3PnQROrLIiZSyKmO6joY0E5If0orZFhgRuvaEtxcaZxsS7uUS1GdkNaCBGApeLXmGtwH5tKmraorzFRgD1eAjQUpjobsvgw4EbK85abAjOZsBkeMnPgsV1+x5RjKtNA2cQkfqkPm5iJabhNa6mK6uRaXyn3RkQJ42vSQiSrT08T4xB8t+UvcaYbIyVi5NzlDQrjuzxnBcBHcyMF0WSQnRkYYCH2auxgliMl1QmhSvE1yNqdxv\\\\\\\/jpy463\\\\\\\/RnKf7lon9FyXPEPINPTUyBrzhWksqMwE+iAWbtmRtQjO4h8IzvMohs+x3nYsi8cTe8QcdDAdRB8n9HBv\\\\\\\/rx38idzgD+4IyzVXHgNToywMKQ9CaqRe00SfpRpkjjrogNGrLHIxaO+cVf20gssfqEFsVgh0F\\\\\\\/dMWU93gkHWJAfyUH78by5Rv\\\\\\\/PjgWzLud95w3mv+87ac8nm+TU2eSDrxnWn+vQ+tPmKABrYupFTo2u9t6wY7BEC\\\\\\\/MNMyzoi3YJ+CHXLr2W95pGOmBspb7xOHPY1we7KtwpLd8FP0ph4g5a1iJ5GrWgTB4VF\\\\\\\/uStFuTcw1Mfn8xNJFgnA6CKrkpgV1NnV\\\\\\\/y77yvJTO3O5UeuV1NEfrjpJYEuwxZfLYjwZxCLrErAFpX33OV8n3eHU7Ff\\\\\\\/JIafnlu72qhSowkMda2604gbpLkCi+lOC099l00rjl5fS6bCF0ITY7z8JGwzSQklbgaMWvhmW2htdbcwetbx4EFqLLkYOAqbwSvmXLWVI208CbhI6CaXUCxuCjfcVEBJhc7aemLu+w0CfulcsyEHhcRNInXtE68Cp1Sl0+\\\\\\\/MFpY+hISxqsMN50RwNGI6zia97\\\\\\\/gj\\\\\\\/g9SPisuZURkQZnMKH6zJT6VnqprGg06XNY6TARg3jkFGjZtUOdx50L762qDNWcTwPJ00l2UQZ4Lxetm+0FiQ92uBF7UECsFVIzfbYLOGJObZKRCmv86SFlUQsa1Ze\\\\\\\/krAR2Tf1Cx9cTu3w07tfCgQcmJTwQ8r5VtQtHwM2QsOKOLJMJh1ESe5TQe8L\\\\\\\/8iPkVYd6D5KkQKi0kkSF3ldUvhGA\\\\\\\/jJGtkDXOEyu7E2qnlbHVe3BqBUFnLP4ZJJBwxZsZheL7QJ3aRsxG8YrUEAZp2M59Z12LGhEZlVPllj72wonkIzHc3FtcvhssIMMlo6XkFi7f9W8trTjQ7hmMucHeiOudlTihQbCgyn1QnoOGoawF4KHZrckWA9LpPoAFL78AHBHVxgvHAXZUkMJbut\\\\\\\/j47XY+I9mBaLTAHSIllMCoIoM9YJpWYTc49SbR0xJrjXK2rmQR2xdUakUiJoR932zOxDDCuiEfhR0OFO509ekAVF0TxzU2fUJ5D271lnzpvTpJcNShz7JtznZzOvICKstGq8tJ+EtRo+3tcvxWUz5Hg54AUvgf09OyIbd5qXVdaxeMr7Hvup0KweXxtks4z6nNAIeiHKY9y+I02I89TZaz6e5q5HBLAzmRcjs8KHozHB7Fjn5CoY6SvzZvQLBH4lnfhojMzyOHS3wWsVrgE\\\\\\\/r6cMfDeY3cLygKMouMQF0O7JDx5BkKb3iHC+W1tsVIk\\\\\\\/TgWUQhB4Yl8V5p35Xyc9LdNXIy9emkbz5+d+Qc3Pvl8XeM11dvjQfrubWQLhbX9gq18Udyb7zc70xzQlrk792TTmoOksooxvU+ClP+nJXdgFRe36hM3WUkPXjqxx5tfhGvpZrBMWW4+dSRRBnzu9\\\\\\\/h8v9J5E53sW4R0aCcVqGiU04u\\\\\\\/tkyTyKWqI2n94qlW02vUXJSRl34lw90Ko49K36Vu7\\\\\\\/J4muZMJ2XKG80tm3WpPxAIQLOLIV8DDX0pupGowV4CNODDayw9fZf5kNGEwLMeeV\\\\\\\/GiRFK\\\\\\\/GDJUYllo4XpzVJ0tfUhURTkjJ3bURVZldV7n+S+S0PiM7vyHzrcqv8nHCMglnJhiEoFpG4\\\\\\\/uvXubwnvWiWmSb+M10\\\\\\\/8i2qme8HzFeaaadxmU9p0dJlM9zVtuJY1GM7ON2Uirqx\\\\\\\/v8ohcu9JhGTmt5jmxgPqknz7H3DP1Crgu5QHeT5DoLkQUKBSWBEdbRJ80prlSxd\\\\\\\/0OzG7iEMpUur7uynlkjE92doJWqPBNo3sJYpV6N3p\\\\\\\/wjfkd9cAsORrGl5emDkq\\\\\\\/BX6udb4gtZqxVbO80zGcZMGKHjB5u9mStQ4fMQ1MAohgcjdggsWbc5TPsviOR9xaSQ+OqOcQcugCda6BLVl6jlNbiMBTo4tvDbYjGcJdCY0XJi+0wO1eoNtt+K5iNQGRV9JvVVPimsTvyNJxe\\\\\\\/nauNWum7B7aNCNq0aIx5+oISNObvvIGcdnf2UpvzOF\\\\\\\/GJW2BGXLn2Xm18WObNR81yPVb3q7ZQuYh5sjPBErIwROBvul144\\\\\\\/Xpw+aLRbMyQBGi3cWm8ETYQE86VxU\\\\\\\/F71eQNXYIJ9hEux6vn4Kdce+LPANYyLe5sPavwDzkO1G4jfaQv\\\\\\\/QYkLDUhRD+nRhxy\\\\\\\/jVkuFQ5XCDgw6LgKVKNyrx\\\\\\\/Ho8XuMNfkw4II\\\\\\\/sEzCr\\\\\\\/mG0nioYeL8ZPPOa69sYswPetJlh\\\\\\\/k5g7foNOSRCcedmRciEv4ssCNudaq0ScVArhFwyDha7qOEcjJAkN0q0x2FqMUGTq9KN1LpHev6o2t9MLIbDYXPMupPKc0rsGToltWQ06kztONikXKgXNVDYqJmKpz\\\\\\\/+wLL8rIF5+g6B5RHcbl9JcskarX43d5qby6GoOvE36tk\\\\\\\/2qu7stW\\\\\\\/\\\\\\\/rFeSlaGzn\\\\\\\/jENg9UtIB2ojv2Z+quuEKR9VdD\\\\\\\/kopSuUrmguIyoOzrOSNEEu5T\\\\\\\/QhY1KrfNrChz7GAUg813C1xPoDF8IszoyZHMsAMJ2bcYM5SQSB7Fy+RU7hudEJXQBEjdhPqrqXFwMfBx4B506rqmyCEBNiOdHaT2BJIpUy61RF7b8xwjXEL7zhE+cLu\\\\\\\/OUT9ecXygPjXPKOt5mrte4wgR\\\\\\\/m2JSDvu9wDKawJilxVftO9j+S4tyOEXGS8C8X9rvlhah6BDILrRPq1xOJe2dVQrPAxDq1obiFIECq\\\\\\\/StTkox2F+lFvn3K\\\\\\\/uKn5Xa9q3cgiyeTaLHVKoXYvNxMPasZS71lrJF5lZ\\\\\\\/H+E427rEEuS1eMIgXaf0t46B3JRKJMqt7J+HfIJlaI8VuSr1+c+P26EduDnXmIvCoerFxRiz\\\\\\\/skcY7uFJ+7GnebKtdu\\\\\\\/Vi12fmRUvdV3vPD1SmLuK\\\\\\\/fUZhrqgaBTkDv7rXRXaVMPWsRpLpGsBb2PhKRnlqS3dg\\\\\\\/+V+ETM5FZBg2IF8lK6UkItWdLa+u4op+kcshnkPTJxgqf0O6XSePufGgWxikgXZrD5viK3eTwNGFjG0xO5bg33Xnz9IPPdpOAT2Fi8gDZEBo906yM6yTReXidhT5D3Jqr\\\\\\\/W4s+pGsVgWKV+FLRFXs54DvjB1w\\\\\\\/lzVlljeT6mN46qEFthh1HpyR57mi\\\\\\\/412kJb08kWBtHYilILaSWyhKSzl8SMTmv1rf8WNZg3tCyjTSrj5CUORBFXO8u1\\\\\\\/1ZXM8ihPoPi4bkSUiDjxtI\\\\\\\/Y+y+v6iukDUWHE8aICqxcVh9U\\\\\\\/jFyC\\\\\\\/6+xrUEFh4wrDXK6\\\\\\\/apbqY+LZHpuPi7Z8bwroOGhEcgo84jKTiRckZ1yvEFTT02WsMdu4Judd0bphKhvK7ghC71nGZHsmnQU8ID+DNLgSa0moIptZl4DD9WEVDWjUEV1Gpfld+t4Q0Hkk3NVYHNZ0TueVAxcPDZGWv5tWo67oWdI3jSPFmDJAEA+QAhjZDycAB1I0tERjOu5pZsa5NvpGnY5P2YRvIgTePTksRnjWZBf1vfLaetvoaM1zDj8EoAEAJvbzQIIK54GqIamNal2I1vJdyIPLqC0oHgaAOKFXY5OzEloquP5U6u1ST5aG\\\\\\\/3EYBpib9woB7nqq\\\\\\\/qyJwyS881IzGHlJcRt9Oaodbp1+235Rir8q9qOD2MfP+hFVm62l2ByLM5k48F\\\\\\\/fE\\\\\\\/q12xTuWx4prfrgd7lzgl0pL2Dr9rJc1cEkTaGrgaRY7za\\\\\\\/2kAGyRaOUOjzuMcZLfFLLzjEQGA7+wQZ4SHiaGoZlHviYm1YYdmX49O0t\\\\\\\/Z0qmrfXs230H8Qx3jBkdhywv5GzAUd4XdbWhRU\\\\\\\/JUqtc6smXMvpnHBH\\\\\\\/oSbuWXxvx9PvY1EldEUTh1eBCF2QVRX5yhJ8b56SJuNSxflKoGueFwGiOi2GcF+dLVqywOJEJd5qthbP1xADSzPGv0ZUZQ0oknyjT56mU33VF87Kb\\\\\\\/P\\\\\\\/sjzAhsfzyxMpTGtWROyEv5mzsQ64Gv6837uAb1cV2IPlBmg3xwi7KNQWnaJ0xsqgby8EigQaHAYu2xjWnzqLH1HAgLreCckEZFhvBr19xF25KYiD\\\\\\\/xzGNcHd3BU2kOB+rp+kVVw3MALnn5KnWiPyjkTtRuA2KZbQLCv1jIm1Zhp6oP4Q04EdNUwUtoUMFYJDg+jS8WkS0zmYwGHC1egCJqHXkJxAbPDaZwpRzXghJwNMh3dWzJAEwBc7qcNRXsynxh7RmlCh6\\\\\\\/Q8o0RZ\\\\\\\/fiMdUbAip\\\\\\\/ZGoeuCLaBj1\\\\\\\/wkZSObCIfcWTdZInNsFNrpBotnwH4WslZMhYGTVD1wKO1HgTRKZFQcESWPt3KwBkQsa5zHhTmTnOpdryOb0QRHCxWKTHfGK8a2D37zGEvT3z+RarWGzxxgaN9tJytHIhfuSmOIBCoB449\\\\\\\/VHd5OlMge1Y4SDfiwfRv3AHoq9DHWt7q5O2Qgk8ZTJPL96DsBCyW\\\\\\\/vWuwdTOthXya5tbMqvVTfUfkWON6tsUasmwZIrtKnz8lmqiwqANu+ZN4d\\\\\\\/6himEZ+QO\\\\\\\/hGzZm3lmnT9+UJLW7OkpLjSJnYvtPi8lglAK5GTCk+ZiZJtXD8YOfgbh2BCwv+ZJmeymQRx8fQs0R6XArRsPF30YMiQOPnBeKSvdn4wxbZfNqx+Y3LlrMuEPEK18fuairrFLpz1K5x5R36rxC+kXKODZV8rJr+z5ac50KVBo4nGF7vJIm3C2JD6XyOTEDrTnBzdnMuGGUgBNk\\\\\\\/zdMNg21FF9xQ7jY1eqW6ZHeEm9BncBwjgPMXw7AxfnnsbpgI1OPU9QIvtqiMcknE7vzEkgmRQOEPq2RDW+j\\\\\\\/jq0XzaxRQ91\\\\\\\/qPAufpiA6puJNv+l0rQBfy7UIzYV7KWbESW+yXyFka0lNIUYdbKAQEsYdBDT940aJXRXRe1Y9NdrZeMQdnhv\\\\\\\/sODfaOgZ7yqIeNaAW+vCmb38AQ8\\\\\\\/EKWN1HAH7I8LrEmEmybrgfLfyi2kIbKTffYQz8Gleptnz6bWqGYsgFFuGupWnw5vXnCbgSeYBvJpCLQKpS7MWyOooXbihyleBKYMN9o8LQ4fvSRKBVfmtdyIiDcqbeh+zLjgKLeiVsJ7dF\\\\\\\/q8obY3SV9UD1wlwlT6ZRNfhZathLYGbBYJl7coKf++beEGI4Us6AuQ0DHC5+IuwHSPdgwMtbd9sVlApdvk7rrxR\\\\\\\/b3KLRX0u0g56ijTncFo3lr4MTeFty5omvU0sOGeT4KwFQ+yUmTa5\\\\\\\/kT8uDlv3BXDbsKXLSxgV3TolrxTGJoCZSBKwJ7lcC6Ba+SizxdjPCcFHtAyyneWxV3EPi91xnHqXMvqFjRktCyoorWrrmcVtgrOz2Yxa9GfTi7s8FsXQgx3LHTyksf5xW2qLW2l6R5vhtIoaPMALWwmMr14Swut1dPArcU+pX60Tve5btUdqRFKgifM96h0gxb7Xw+Smco6wWzokhK+p+91lP2KRcJYp18q2QJpvw3shr4lR8iqcsCt9AmzP5os2lpvSMUIIhBbmfPpaX4FWS1OKNWWC0fZzyF+7lwjR4idRgw2pK8xKb4Jt9sCL5uhfb3bcfyPt3Wq\\\\\\\/A\\\\\\\/LamADnwZx4THIUdgSbhpOrTBAhSUtSzbtCYWME4AjbxpS\\\\\\\/59ntJsV1tk7I5IdPZnmMsAqHVWBBlbKOnvJq3rIC86TaxfWUeTcxv8kfhxny4MlZ9hLnwIqqhSWt3phfaT4C5JEqiSl0TrlSmVZ2UGuCroVhAaaK6pz0aINreNY04AgX8YIq6I6PXrGVbNhkl2PQsfw7Kulnmbzk0XKJNw8WtNkbydDn0XtLv\\\\\\\/C2S85VmamXyza1MKs3z0oVkcwK9Guaxs790nJO+KqtVnTMN5biMuquPY3amv6jxmvzUbgU0LOrH7SG13h5fQKiE5d7rOP8UE\\\\\\\/+XQ7+DZ4zf4uyGaTP84Umh4tsReU+lp0OWWpCBXuYYmP1YVkJgWDJJTrp7yypqC5bZyGqsnL+8+scNUaGGFngcAU5kU9MNR7fYfGBtGq9UibiYvVmkHx50XFMQe+ICgbWt9IjHs1S4yY6dAfVOdXGWiRM4Jb8biHOyVUwiXD8WmLBjp4RPmVC\\\\\\\/95wTVqE5N\\\\\\\/8WPPb\\\\\\\/qC+MhNvqAZCnKndIu3SVfblNzLxf\\\\\\\/9X80nB4EnzAfRpE0AmYIYqRgRlorJSCPl\\\\\\\/9V4mlqJpv\\\\\\\/\\\\\\\/BtnPIaIIK08vgLz2UMo5tpdZfvTFrp8d3q9m6wMyIgd\\\\\\\/Zmf4Ry01x1APf5ynJF8sAoxeAXR9x5KBBrF3VrMwmqyn1rvIlJAvae\\\\\\\/Ie1F7V86lkUdbU6Ebqd4SM59dQGGs54IGw0yBHiO7O5jVnNiVOPI6ODiIHhHs1OsjKdhLq1c6+CE2g1C4siLkmjh6T1ffHWhzB5\\\\\\\/qz6FlRsRMK0\\\\\\\/22HDI1gEC3l2z6Fze2aubDSRs9UFj9L7x1bB3RNZsxtcT8LrkNtcyiJKeqUzQXkUa+X8\\\\\\\/ZvQPe5eeaWiFeZBIW1eGonur4irBiUg9usbCKReJ5pAR6UvNWJ2EFD07TbTuqtTo2UCynNzkz3Hy8mNgwuSUntOulB0KenJptlo9q+g8gSbzVWn1RrWj4yRp\\\\\\\/m7bG5BnKou7P\\\\\\\/InAwFnPU96khh3RLUReIAs48q0lFUFpLTjLtGdzvZ\\\\\\\/jwdJ+i6WO9GPzLOUqqz\\\\\\\/d0wNOzFPhOSC4jHW+VTOsMthjK8n5ekswtrr6P2O20BNqW+VZlULGO9KeEe\\\\\\\/Hq8UK+5\\\\\\\/rqebi++b9dlHyp8DL3OVaIsiRgVuqdKaVGndgwH4hXwwJZrDCOWsPloQBHd8diaci9vlKcj1zioeC6N2qGQLMhPY\\\\\\\/Cw5LN9O3cZKN9r5QV8kkGq0Ng\\\\\\\/gtHwhWxrvDNDM3HgMAcrClweQMW7ntxE6yaLsMm8B45zs+y462AH9aaVrANXrD722naWNx\\\\\\\/1Bq4\\\\\\\/uQcEHsjD6Oezma6c0wRylaKAbqIFcEVMDUFibiWpf\\\\\\\/VssUMu9teJnbK2qGQCWvP6VS9IPXulx5jgG0owivV7fWeF6KNTUj5vPex+CZ0g1chV30qeR3siBS9FP8Z20uXLUCqRcXhsVbFuBrVztG2DSsAGhAvbXvsVeosWj80ajWPF5GGGJsglpyzCyUgOgTE93nuUQwc96NV\\\\\\\/vfOiM+v44jAt16\\\\\\\/2zWe9nPplQxyG8tpcvN0GD8Pil0KyqV2RvsWuE5bqRio0XG\\\\\\\/x71ScNgyUTwk\\\\\\\/QZTuU7XOHk+umOv1XsT7Tkz9sfVNlS9LjPzDj2+DrDgFP9urDhSDAP37eJG3kotkZbo2LFCcjpMbzoYAqjVyPK+S3VW11I3m\\\\\\\/QyJfoHwHU+wyIteAbTcCEAvMIaJa5qQCneqmvmqpbxUEn8+tLEAvFb2IdI9Ei5ORJTvjTCwXGWDEea2uu9tK6TvFowyHlGLbJFMCPA9pl0HHA2do1YmUAtmqmcyCYyrg6d5k3hODdjrNGM+nPjicjE5VmT22iDbPs+lEOxbtz1EMc6MBkp1MA7x\\\\\\\/GdHqrkH1bKk0RlFLwooaOnEQcP7otPhGVbjveSbCy0ePKr2beg3X6SgcOFdOA0BL3IEZHqJo6nu46uW8bRAs7Oa5\\\\\\\/CV1NKo8Rhi6JoO+Q3Zr0Foh+Dam6qyIHAYfE6ITVQXtT+Ik6uQTf\\\\\\\/y3D1eGI1yFKVIFa7PN9BusgSID\\\\\\\/nss3DO+Wpk8s\\\\\\\/TlFQkMHzDOvyomkxlnM9ePjAObMS\\\\\\\/JyUKPzJDGWTKR\\\\\\\/98dq3mC\\\\\\\/0YQgDlHWG7a3myymi0GnccNkzaxcwjEIfm5p6FeZS6C6D\\\\\\\/+FaQRFJLhCqpTADR9pMAKBrRkpGRLuX0v+bhRpNZ7pWqitDjuPGt4OKBCleYgLJJ4k6VgtyQAuSN2b3gXhhxFuhbfJXsF9jRrSxp4m0sAb\\\\\\\/VR6e0DrSwgLy9Qv7bNaA94JHVmvj0SZ58VNg1Mvtlhyw2hsPJiDDPipp8CGVm+vnOJTlYx2UuCZCcaYBA1obWPT9POScoEnjexVi32SnajWPCjS8kdWRYX4IdgwWfRtMS6v+ggReQbzsxbAQ+aUXtGDGfVWiPa6++kDp3JULdrNlRfde0wwQZHJO6M5+tfMrB7c5ZbdqmSc1mTFolX6qGjFp9i5U2xNB9s0mAuVRNHDkmTJitDQLuHx2OSDIUmoUjA8FMFdkhWkyffTKK4XyhqOtUyJ4bi2L6HXafeCyRxHmg+EeFF3XQMzkFyCxeIT9ODmH2MskvEtMgdKOdNnUG4vTo1EPYnTUtjA+8QQZqvj\\\\\\\/ue3RHf\\\\\\\/sLgBcxDwmSjHjYN8r8MhrkKoQkoTmDgDqsbZkrS61ty1UKI6GyrbKnwhWzgOXBeDU1HAhMysh4qXRWpXyQBzquVIVNOshhIlAEmtloyNF5dJ1pmAf9XpPaGieyLr8mYmzGmAgUdX2VmTWzYw\\\\\\\/LK0u+vSf5uymyEr0HULwpZH0+w6ufqJay3EFDLIZVpz\\\\\\\/Qg0Kx5XMchJIqW7p6AY8JbyDIhdBLxII+uxRMUZ9yn1oW6EYzXusHUGe3zzVWtVmDlGQfpIm08IjPV1\\\\\\\/i5NVxtt3Eus6A+oUouvRNYrhVrJmeORtNBZ4CtdoOw1CKVu2apZYDPx5Abuf9OUAXU4w5NGCq+Ut08HAzSTb5SeZL\\\\\\\/GuwA\\\\\\\/Phhtp5mceTrGWzRzvthgnZv53dUbMnzXmBlUPNver0\\\\\\\/9BkcmGY4kGJ+CT2KQQgcJeBffpN2VYNHOCx6RFimh1zuV0XC6SV4xjbbEnb2jwIe+Gub2KY8UvLtZGQKKSAvIZWysHRbHu\\\\\\\/NO2ItSLbwni5jq3u8OGMY7mr6OCyRicgB+qufuCNYPliFhas+qgqkfwSHTxhHggz\\\\\\\/yXU5fYbVH9dW\\\\\\\/UC241Yo5\\\\\\\/\\\\\\\/afhHhzYV3TUzUPvCDGqGTNuDB2LebcpNyCRK+0Jpr8Wqh3iGKYSsv\\\\\\\/Pb0kRw8S+C\\\\\\\/gXBopIKQ6o3mHfpxXfh8wkRJOFY+QriHEd3WkwUGnsy6HhrKWOfS+9trFOmD6yyfdO1GaHahf9OreUdOTL8LBZro222VdiZqnZEKkb4LOfpDPAYqD+\\\\\\\/uBNrU00jHLeZC4exfOzvMWPeGj78mLP0YB8CHbwJuAN2cnW2o\\\\\\\/i4GKCSU7iTbZq15i934VcqSp4mybPon5Z5ju9REKVZb1gAl9rXyL1XBbxd081j56QwbFL76FbvsiQrqHLudnb5kNcmPhhN\\\\\\\/KsVNUnxP1LwdIEu650fL3ZCLgNO4Ht4dxHTUg41acPUMdawhA\\\\\\\/soAiIiV\\\\\\\/M\\\\\\\/XAzK3KXyQGlFrOthWrvZTieVJcWOaubInHc6U89DVc1p1un4SIPFtVdYgGx3Yv8VgPSkRBgKfYlYSzxg4zM5Q17JdU2k\\\\\\\/D0+B4tt\\\\\\\/vXNVVG7AzEKhGF6gUI7q396U1\\\\\\\/+gguYBg93AdLX3qpY+mf49ssa5OkRXm3j9Sa8P147bUWJBTULRtSj1jSkDkshrSDWWNlGUf0Ni25b8eOpAfgknUXq\\\\\\\/yFtB3fuKyylYtlSXWFY8RcqVHnBi8YwMdhs0TnCpYOqxGQ3\\\\\\\/a6iV+S+D+OsIA1IAQvYuZ1ktTbiFd4nHnhhqlqYStggJoTq0M0f5UV+FiDe0Gv4DE8PjVp5Xn5Lr2QfinzTxemkRv4Mwnp8Vb8GmYGLxy3VWpjOp+ukIKg4jswcdIz\\\\\\\/ozfYaVfdZoxH+jH9qwXsbQymw6UfZQ5uVxZs91ttQKQkBfZy+l76BihmISd1zkXNGi91bVp7H+BB+9\\\\\\\/BTLiW\\\\\\\/pHDfF7OHqn6bMOfKc9OAtLEnTjmfs83Ab7tqjOve82K8vVxqvParqOvC9S0C5cmisG4xsbeRgl9yV665KpOsjlteLhxmKgWwxztscxddmSiImA+u0O5xPfDVKesnYlpIMwKvC1xKFTd1aOGrXEzHOBFQZXvbu3jZMFYq8yVCN5oFEyDRQcg+KSupty6lQh9uaUP\\\\\\\/QfxNdsXiMcJniKPHkcDj6Y8PfeLsIeu6kGQg\\\\\\\/IU4WX\\\\\\\/javq0dbWyYhCye54+BcWS3zgPUceVmE4JjJteAtrw2OY7Rsd06n3HQsjgt1V0EaQmfJddQyz3t1Cij+mJMqXet+HTrrQ3w8FMhzvAXg\\\\\\\/WyR4QSEO3UxNzzHtmMBcXL8NMIuJyJP2Bic\\\\\\\/Frr7+A6E2Rb7KY8UvARxBy9OgUEk2yJlR7KedKjn0hwd6huE+fsTYsYt28DQDLlSEBdaIwGBSDERQCvBO3I8tfcbhvrb6pKzWAeLwWF+81T+I+IgH65+c0kv6uQF\\\\\\\/UodaVlAiqOjY2hEC6XWtsM09TCzzN4h2jlXLkJyf72U5zRpk8G2s4K\\\\\\\/EJUZkDBEmQvN6LuvevV4FqFQK1DlUM81EA6f6bOcdEhbknQ8u158kn7YeACUJznSJFXjepVsjwRWS8zvvZKo739ov4GMMrvlz0jNJAhyh5RQaL7hXNbOXfF9S36qUuY4V0JLY9tso2bItZryuppx4utLwknEX\\\\\\\/1TUuTUb8n42QYRk71Xe3Dt5u6Dsrau0WGGK8tSNRIyL0gNXejTspXtkWjLOeep\\\\\\\/53sZmtIvyDhoKmuMHEUr9fDsIobkiXK5LpwvMp8ZMtL6MXJ5iXZpPTzkwjrxdiIl+HFzW1mAh+\\\\\\\/S\\\\\\\/4nYC7QK7VBlPwiwvDH+QCzOvVbNPwqYitD+mecTle6fjD9zGKaVB7C\\\\\\\/jarbUgL45vYLmUJJYam4AE6nNyWKZepJHSi9zijmUSZ8ubngV1KWClaBljS8+VC+UMznNgcEK8PjWZKxxp4yPdCFAwBaxlXFALW8q3u5ZfTXlGLq1wmuFm4ZSFcxmCKabfsFQ\\\\\\\/Ka\\\\\\\/zECtFcWtjbcpX2qdZToV9haQYoP8xgKS2z21fzj\\\\\\\/DeZxdkGI8CXZF5ctCuP\\\\\\\/c90N1Ivz0GT6QrdhE1IQxG1p+\\\\\\\/vTKBqpnvGWbR7P5vPO5jvvQ9dgcwmFflvB8bLQb6rn1cD6xqh49BtOglqLQmJmgw781kPMlmMFJIxBmGsYDFZ9xPGNX\\\\\\\/YQ3Mp1xI3UtdMhijWsjeTSYGylRtDlDYmt9D\\\\\\\/VBgLiV+r1fGLe\\\\\\\/4Ma0e5hb7zF3\\\\\\\/CVRCFYTWakWQ7safBBj\\\\\\\/S\\\\\\\/T0WPYe72hrntvbIJfzaa9dDcfPHsXh4W56BpXM9ZvP2StDCw90Zlmt8qpEZoy2G+tnglvC3\\\\\\\/A1KvofH9mY8hrOqUL719Yx1QhEa4T6HgDKUUfdNMgEBTVutR+OT2IbQu5lVVZoYUetGpHTVli3FWAIsvFaUSN4ffI+bfHdTKRkALy2LK6OlbnVicj7pn8LZw8sMbvGk6yhm4OGc763IZgudNe1Mp1FYk+9rTpMFF3p++rsfeVMwhDWoULwNkfSQBta1VV5xiIjTvsQebUFUVFUg4CXspitYYBnl4OohbYP05pP4kZMSfI9vkngdKsvdnQAS+7xsFj5f9SDJiVx4gYPqOJC4niXyRpNPVLW53R0teNIV32psf2lnp5Z3Dkd71sPB+lCtEQfTRH+VFWWy13pLZoBG+48ceP7JbkcOHC5V4O6M1F7sHxBqQ3tQEoUlnFlRtXUzc3qaPL8nMy4frwMh+XqFnd+ibVDQ+kj1Lgmm66IJNcQA+KGd2z3K0jIQeiessV+eoxT6EbMb8ysO30qu4cMjBURSYZpxZ0MIU8ovNYqqdefNaLTxBfmkkxoPem+\\\\\\\/AUsUw7XKdZuKFfw8MKbxR1m3gjNu0KRE8qASVqLQXVm8cS33Owz+x6P4f9i0\\\\\\\/xqEAuRtdzgN5lGtv5qqx2t42gtBC02\\\\\\\/ElttMhaLJIQQmu75cDX6ZDBK14u2tY7dVMZ0IAaCkpOe69ujq2WX7t0PmdAB\\\\\\\/5CkZtfjL7QeC8SbwgJ0YXv2pZy4iyaGfHuI0IUWDdgRPJNt95uDzfGbKHoXQx4xFHZv1Ws0HkS4wZJnNWHhQeO1WPm76Q1K8JzH04TMgxZfj4v67AambmXPAqKqvC8tHks5MGG0nT+N3lMjrB\\\\\\\/yTR8PcJx+2w3fL0f1Ohp9m32Rw2XhP1LHXY4jFe4R6vnpBasu6FCi1Ol0gp92Z3+5pxjV\\\\\\\/mPAgVag1cq0cSy4wyBhECuilBrl07\\\\\\\/rypgo6qhYTjWF2jF+YLJioE9zFi8FEgonKSf2RYDE26NpnUNhsiuJzmzSSx9Jt+DiklHcN1EzPs7o49YgeDDaUrfHONt0jjQ0Msc3ImKjK39U\\\\\\\/8L+fUPGeo404cIFNnPYXFy4pyJbbYdqe3+RqUlxQeu4AyXOu5wvEkGvkmkTpezdksvIhEWqDipIcPWy2\\\\\\\/z4g8nB2JfnxyL6+GAWcDTslFpueE19OyHQYHTA6rpdvexrc5\\\\\\\/bO7NveV8MGjh6kPR9bw5og1OoDR6dxxp6vGjzoODTBLKMIwE\\\\\\\/qNC\\\\\\\/Y9TM4wvsx\\\\\\\/Y3rO23dvS+vCUOLvG3YijWdrzlpgohjxrQYq9hl3E79V7PvvrEU3dm+IHPFeblWDcz9eYvfUll0ah4hmYSFg+4KVMVxs+FrbxBRvusRuYCYTJ8qIpc2Fh8NdZLLAXyYwjY+xQKM8x5HYxHIe5Gr5fo1aWbcY2yhlFfAMjA0PTOgyjT9kBvNcuhqiy\\\\\\\/OYjROhIgXh6+lcTYxDzEAekQzLUhwKW5rAqypJp4a+iFI2C0tK49l5VkFxI3DWrOTvsSSyLGneTpjTzPUkpYPE7RiIbGzSdza95L36BG779ZgtOkeisPt0gUlLbi0\\\\\\\/IpHP3IizsZSm+6tOjHNVDgiC2tNDTdueyGGXydHHPMSo83Nxs\\\\\\\/jUW7sWMRnTNL0dQZZN4\\\\\\\/BBhsmw3bpu7JHQ8Bg2FGiOcxHwEeogAUeWmi8EdO9KaCoCZa+X98dMkjnZrFVY0SAlfSLfx1gU15Ql3ECLKTTWJX8rMDnHuUmKFSTfpHIggoZFFZsUdjE\\\\\\\/x8h1shYGqYoEoQ2\\\\\\\/OHtBIbFrKQq5HDq\\\\\\\/UNY4ejE7KEdAbNvxtPmnlqOupRNI4GfL4LOKEDE783aL2E4NRsqwklKCuIwXPIE2aB3RbNBQoSUDMn7kFZl7y19jxqSAZ34oNiLu\\\\\\\/jIuDgCsN4qKIpGaq4k9CQPFOQeF9VF40jdahNoIf3u5NyCWyBnRlwUvJLx5UYAeo5FQrjgpQ6v9+k9r8MllVE+phYINgcwDZ1R+ONwt\\\\\\\/5F\\\\\\\/DqJMHlkMHdN3z9cFppBzWZhabJbAwQfSvD\\\\\\\/rPlBhhsAqhy5uFdcwUjUKBs12NDM9wUCpxMnSwrTTdtIuHj5bCQ+p2ri2P8fBTcr7MOjwjQiubhmm+NfHB228shVdVysvlSM+l75\\\\\\\/7mIOs64RGzk5mZ5KfHgHi8e32K4UWlkd2fm6vKN39EIJtKaVF52QRWsOcF59Cl25Qw8HDwRv+PEwDaDj0OAWCz08nA+YqCDP6cPAtj5jIOhggEAU4UEprmc\\\\\\\/WPDbwvLGuph+nuKoY0Wlfbf6l04BeGn10c0ckquFcJ9Wp4J2kZCR01+Wr9xyG0ltM\\\\\\\/iqf1CgguOPkcXkSFm4li7\\\\\\\/AQmUp2zKIqtqJbUbKDKmQsv4liXLhonamrAc2zRr1+axZsRgiNf8L94\\\\\\\/LFB39H+3+qsrMpGUc8A+8KorAGKuO\\\\\\\/PJRpn2eLcSj5ofrII\\\\\\\/qCM10MLr2YI2p0Yngm+thWSJ53IQv4xPGIVwz+VnK0J\\\\\\\/Wyds+CFEb8uLztJIHZBVivqdvrXQeO0BITypwK3eC8qa2L9ky8HuOvocRJ1jUO5Jce5VGPMoEuqoLKiJH5SwxiX9zBuKeijevPFAOzPQgomzKWrm1d1\\\\\\\/k8lPIxb8\\\\\\\/K47ysicxcoW9TtBzWD4\\\\\\\/XwF5ddNZ4Rix9v2YCIjZXQFBVfeCuXkWhVfhZBTu4kWhKYdSyNeF7s5DQ2b6e4lahJ2zk0ga6Iekmerfl2XJpBWse+c6ZTR7x1RtTGNK8puB5ttxY3kqPQSzd+aKRjsb4bvCwHvhkUvpGXJrhFdlNAyQcBhaH+0fbwE\\\\\\\/L3ilbC8cflYegkhdtrJjWMcwl6czFl8245a5Q4sHC4WN2rZMpc647fSOloGgrLlEa+QTmf\\\\\\\/SHOxn3fl\\\\\\\/m9YRErItleBwVmjARgQO7C9AVD5kMONHe9Ru5Q1dZk3UdMuzBNENBGe\\\\\\\/n4r0\\\\\\\/AMfIM4T9yL5392yIbYn8Grf36EJgplFHNRnMLqBJ11rziW162KZz0wvFXXm3LoKDn4ZkAWxqVkZTg5Z3M5kisDobBS0Wm95NpXzkb5DlsNNnJEdxFnZr\\\\\\\/Eg3xhmRTAJkoNqQsxv6j9twUprLMn1BT\\\\\\\/UL1PxD5QYex0IVq+OFqOmAHFH9J\\\\\\\/zv6QVw1Ee8i0UtszoAilONArQb+xZ4Y\\\\\\\/6es4KuQXRuJ2OOIuVMMog7t499DKuNi5Gd\\\\\\\/U3YtaNXO+IEeo65E1L4UwYJJvR+mBqpGUUitgde1VKmjlECxtQ+Az\\\\\\\/CKMLtKrtIfpoCeAE\\\\\\\/4DopSYapeUH\\\\\\\/CnOcJB3DBoqZOWXU+xF+o19d0iO\\\\\\\/kACuQ9\\\\\\\/88n4MZ2ooszoGUOFWtMZrLWCahmHG2E3lbyIi\\\\\\\/brWNjR+8EZMpY\\\\\\\/RfDomSCvgSqoKHpNpx9f0jRxCAT5Wg87gz5YLjdg7IlIprCpqTkvPs3DG1FX7Rku+GoAANj7TA1tnPgBXfmLfYWhVUUnDN\\\\\\\/5PvaiCWQPxXuLpF9aNXkUlm0SITdJxkQvXZKhUZoI7l6VB3LnGfchamBL5KXsg59d7kGWs0clOeZaKaw598EdQEIkeWage1YnWB8Xa9WMn00gPoFXqBEIhGCgGUYPE4Q7nVBq7gTZHQXl4mY4Z0R+dpsKWffeWO4D5AXPfNph8nSG30YEik\\\\\\\/hxfwLam7AAM8QZ+a4HiSl+wlHng0hr505TKAoyBwicnHmx\\\\\\\/mL3ttzZN7Mqr1X83cK0ZwiiFEEtnI+c8Nm2FS7bT8sYfzqyLSAO5Qk9j1iqKd0OCtRSGApeZ45iXzQe8rvJDIFrkS2H0D3cIhULjwn+ZyLgj1qPwzG6tgXPOuA+Q5N1YHNSnCxD2qCqlNsiISUrxv5FxONHPCB5lU+UGdmkHqjyhdzfxxUv8divyFRRHfosUEXTocuf193pOSj21ff5Bz76\\\\\\\/aWAq6+syvHc3Pen2mFEBGYaY1iz\\\\\\\/E9w+H54gBFKlqggLOEGpvgRhGHo3ZF7gdb9cjkqo+ob3grA2k3nDQE2IH3l5pSyrfaTtWBX+qyxfbMHNNaVIw+Mbk\\\\\\\/Gwmth+a2qTZE0akPZlHoDvKXD6\\\\\\\/IgSuo7ewLpSTnHZZaY2QKZFixXZ+yiNS6\\\\\\\/xSaGr8JuwSMWnMWuITfYE1W\\\\\\\/6f2vwkJTAT9NSYU+h+de1XleTusVyx7kkKsULHPqQrQepfNWDiSyPoituB1dVx756A673v3YyhSC5cjtbUBlIKQi3a7HlnYqjWPZj11ajQ6+Ox7o06c3uOw4SOpQLlV1QTdcFqtvEj2EOBoTkLYBtDaVKARP1C0+u2Wb7HFSzKahBpzQW7tEcm5\\\\\\\/ebsEiTdONog4vKhgt7ownggHRPuymiBW\\\\\\\/NTYuFzImOMLqww+1ab6a\\\\\\\/Rm0et1lglZY6gC6JMC4JqxBTsPB3\\\\\\\/y1ejxmMKOd2f3WhdeD7GRI6MujGBMUZHnZJwbrm3OURNJPDeqcP1ZvofRps6kAFUcLY+AiguLcO8vQDgkahqSVOGU4\\\\\\\/c+Jh+xYTsTUwu5OkgqPBvevlFgxdu6gjifAMFhrwUSvxHW4Nkb+03N0foZ4Tms4Ls6DR+H2n1OwcjsfpsCBL20Yb2FhDZLj05FBKWtjlfzFm743IkINBztfzGMyUoeDrZDoQg\\\\\\\/CoQJFZZNTyhNDASo1ho93Rf21fyTphAOjJJ7KIxSHkQXhdbVbbCc8xcpb46ZbPDjgD\\\\\\\/fBCBbp9X7MbxP5S1AUNkZNoTnXWKdYVcCZxowL4H3CZPPKUW7mi9TM0NbomVjKspA7YmiQ2+0xwFzJGfuaJKALdSt\\\\\\\/L42DH3t+nV7WU7FPOtSKgk\\\\\\\/PSeu2tu+KrGB4qtGTjzc8485x34AMjTvNyK6eRlNi5F+GaTT7X0Kb0vKuvmFBjsnDYJPx86pGs64mraBTdfiMvqwzwQ3EK4qtgqNczNqvmIBGN6t\\\\\\\/KwgqIPPtKsfy76UCLGDmUxFd5g9nrUtzH9bFtqOsMsAoZ\\\\\\\/z5ZMoISHRMy2PtmUdwyOj6Knq31lRs82J+tI0vlDQYek1lU7hjoWW6TEIRmP5Ed8iv7KotOb\\\\\\\/WBjTUffu1R2xfEBy8dYgAX+2mDiPWBb7KmpcMZXq\\\\\\\/luU9\\\\\\\/zAo\\\\\\\/F+wW0fudAq089BnqzF1S5O+A2ysxSWB8xfykiibB05Az4ut9oiug24h9Oy\\\\\\\/M9YuBiDpMDbwNG2mc0AsUZC0oAxGyeFBC3q0z70+s0tYQACfpf16ISRQupBE24X\\\\\\\/7rSuaTeiEa4uVJQu4xmGpig2GEp0vOgeCr6f6t\\\\\\\/qPacsnXNLfVhcHQihroktQkK6z6raKxgZLS8DZA5HrNI9VjDEB+BfbBsl\\\\\\\/lIGm\\\\\\\/XbY0\\\\\\\/CEToQjZvNMgOVC0eOpeT70GhQY6ILSfyWd4AHtqZXKDu31bWo1wjX68KE99P\\\\\\\/\\\\\\\/z99IpVUquFTIKy\\\\\\\/gW9mfq+Ul69jtOHRogBAbRGfwSFpjCNhXevgGhVRpVKmOKA7QEsz+myVuImu5Bi6BxzkeoiU6+9j5bYfgDpeg+0IrX4BOTM2K4po0NJlAqTsyuctTMyfxGlHjJLiLlioFELIznfCQ4Q00RZPyYCnAeUMHNqvEy\\\\\\\/AsjCI3Kfx+TdTY425fvu66JeV7FIkve89rvdGRwnVlhm+6SXk34wD0hT0\\\\\\\/jnLt\\\\\\\/CQUmq7nJr2SjkFh5+IgWMbw0\\\\\\\/YTQ6arkH\\\\\\\/nv5w6RYYBfIYAxdVjFI7hbCYuREMhggL23S+WpGLHZ1+AFEfglpHOz6axB7YYPyfCUO7HOXafaxhRdp7ChoMERb9SFkzLN35\\\\\\\/T3ZYidpu4z2jCTSAxAq6mCz0L7ywkHIrctC8p9GBHUZPrXJk9PJA9+Sh0ToV+TvOT0n3B0Tp3v\\\\\\\/PA\\\\\\\/zwGnsbKinrq\\\\\\\/l17AL1dHJeImHyF8vlBhbZOCiwHUQ0oFv6hlB+K83VGKjoIqSNEzXtAXtQt2vkWJ2rND3sKjnohuRQnjPBqYUpvMuMCXdnE49cMZqGBGxriZaXNwhu8qSAPYDpyOVKHMCfnFPCDI6CmWxWYcYwcZ243\\\\\\\/lWqZodlCJsSUqG42eK8oLRtC5+DDYEyEXOPnSFnuL0FaHeMVJLFD+TGnbIHRjet0p3tw4mxoTCHmzFLqFVfWYm+EE942zNBmcozmrGCWvaijka6LM+1fx0AQfEdVYZEvqSygqsTAnaqz9hHpg9\\\\\\\/KdvBCFI9g2G+goS18GemFa+jYRbOhGnVknurJb5eYZIWPeAvUnWM08e9RT9KjYqSaXjaPtku7kIzwGylzny8Iov\\\\\\\/ol25XtQCt6d3Tbv7LhRGJht3DUfWxZxfIknadmsPM6\\\\\\\/Zphlv8y6Rpio33iyGyp5ZRkLJQGn5tPofRcytquPTyRryCgAbMIbQptdQdYUQOR1g1bHVf\\\\\\\/5ml7kiaNqfg1tw14IvV3TvmQfKtL4trDXKdgyQ9jsAh7RtoiiD1MW104xvwgqkZeoBeI2YNhLTd4pg6idQByCJSWOOK4C9aX3Ph6ap\\\\\\\/3RoJIxuDYaVyUjyqFVw4l43faK27QvboJgYmI7sVE\\\\\\\/xTLCrj2g0GHFVOBOcrL\\\\\\\/keI\\\\\\\/9cBsVUjKeE3d\\\\\\\/KdOlnrZ1ald+tDySg5riZtfKI0gR\\\\\\\/mpBuM7s0R4bh6OMObBpsb6PxHhVyYSnSB44kh68Oi8bmFW3GVf8pEvmBj\\\\\\\/J\\\\\\\/e87BNdGOmAXZwM6jAp6NsbtYG+al3dRNAyb\\\\\\\/xuzUItVW494MBAnk5I6fEsduJwW5rD4D2iuS3uB1FQIVQDNhLtTErndeQ3icjM0cZgx50ojv+T7cdflxKZbyg7tjv5GtTjtG4VbLV9OsHXowxwTJaDiKQ1xUMHMhySMH5sPHVUDHtPrQtj4NSzIhr3EfQ5ZWhm+cIPpcspJpKGGc0QnqHGmSUyx56w\\\\\\\/JJxqcB3dd3DpCASKFdTLbWkBizh0\\\\\\\/sR0lH6wanEn+NJ1e1mLhqqo733AuRkB8qglqGO9au+kp1Jy3\\\\\\\/rsf6mnQ3ucRQafX8obwqbhncEgHL6eIAe42Xxd1xctX4I\\\\\\\/FOKAaxuS+JPamOQSmLw2vZgZuD1dZn4cJZU3wSxR3c4E90zErEAWwZkBisbMYDVE+uQNylq+9Zwj19fMTBFqUa2Q75Qvd56lOC71Tl8d\\\\\\\/fp3t1hybjPsLZRG7TqTbSY6Z0lEBDPKDqnBvlpv6b+aDfp8muAKSN0lr\\\\\\\/MOkC33zQnUP2rzl9tya522Isc2RW+Vv3fkqzIlwjaFPYlVGZOC280xPKVCKBvWzepm3I1toEL3ZyFlwa3uaH6rVOZFqUMIPe6pIRBv1LmyV9oYB0\\\\\\\/4P3wSVCKc2cZJYaNwDm6lNEa1YcxGH4zqRv9PfCtrRyAWzRVPTWRFdi2q6aLPcSZnLkpppgnW1D2f7HwgAmdi18W60gqJ\\\\\\\/P0nuwvpVC6Tn6JWHXfoUgyj19vxlz2eL69AmzUd0B7pBFmUct3wRinScVzKxqrh2a7ZUde63J9iIRtRvJJJyEx8tVfJhwKsg89VoXVnVGB87ZNNFXdNRh1RywQ2rRzZjrs5JaRSzxligkd0lyCqn3PwA6Fp0kpIyUCqvyYx7AEmsU22+A3Q8nnWdyIEVT1+5sNaeOxQHqBvEbYJ\\\\\\\/q+LbnGsRAedeJodM7bEnUuTiIFzGkeiMRMx4nu0GjrO3LS\\\\\\\/rZtQds1rdcedKHrCl3JsQZmzWwQtiBm96eHWjNZA8NlvkDxiKGW+E9wzEGXgj3yl0DGq\\\\\\\/AGR6h5mCPVfO3+rQofEHR9UDbMNrfsqB81U2qIye9He3R\\\\\\\/QLo3qg5mAsbiFRIENVRMVkMMn6bCuo6YDx\\\\\\\/bD2tpRwa86Fqrk1CFsiwM\\\\\\\/oN0gIludfOgKznW9fIyIyYVkVbbSLhhUNJHHMm4lxyqYtOjwNaChgC5FGjNLyTh7kOydIv2zj9M7Nh47U5MInqutzFQN1IIGg7xypJpi29joGQe1V2\\\\\\\/nxBFVIwhBdZSYpf4AxyGwlqqIF6W6kawpXzMarNNLSxhu4fecu7FiFnKfTfYVOmMGgJEDPwunj6A0wRq5QS3TfULbJTDqkcI6msq5snfgbkS2DoGPQf85TmPGqnDk5gnk3fFKHHJcuP+MHJ2TDFw1eYfG7PCs5XOZhQOJ+GODwxZWQNpjPLWvhIpPl+SSpseBUR\\\\\\\/AHBdtBuNvMdbdTSIMSCZIRP6sSAguymHZoYRVkfEp6Kg13hsx\\\\\\\/Gpq5pGWhEjTsboPCz2+c4Plv7QQb+nLA5MMAc\\\\\\\/tGueU767qMVtWXOEmi3fHC2BUBSCJqiIJt09TPGQI3kuopykqxrEdRq10eoKVjzHI8FdipNPp+6zVvuSRQmNxaFNFNcNKqcdVvsieCnk1YyQ5RDaUbgI5ZUkO80wggMJYLCKPy22jJD1WP\\\\\\\/Np18qh27PaQrQ1en8h3cpbhmEnFfR6qImVEVA8eFlC4OQmvvDsOj9nlwEtRN8BbO2j0\\\\\\\/3XhW6XZv2RhWhhFjmE5HoEiCmt5+71mOmtghD4tC059\\\\\\\/LbWJpro3KafZCeJAeE1MFGSyJRKgZEEduVkeIbiICpcyXQonVp56e3pyTQqHUgHupklpiIr7ewq0pWOhkCbi6zb\\\\\\\/TCBt3LgRLbtQFYkGBxGfi4ptaJpo4NaHqWe1GL4V49q\\\\\\\/c\\\\\\\/f1iWZ0\\\\\\\/dGP6aCydvewzZDRSgdLdsK1y0QZmm3OqgN\\\\\\\/GhzbHWcFa0dwnzYb7L2meAscpSauspN4\\\\\\\/iDYzqvBbpfzyZV2CLCVAQCIsVg1cQhBh82olW1mGwr9uyZNWunKl1rFMnOLe9IUSPhbaZzJmkjEHtTn69QzSJRXeLsm0aBkQv3AAcoMIs2NWPpHyld3voqB6Pr6qREYKQjHz2p23KSNl4Tv9ZBngll4csUlhyzjpO9ClHEErH02BRiL2qUZskObAxqKFXvzxPjiUA5Kxrk5+JrhboPdBQsHAfuXL7Ndqq5whWT9tCjTip5IpLjsFgK+M8Ep\\\\\\\/7+oztZDHwjRbaQpqhe8X\\\\\\\/\\\\\\\/0TyoP6VoHwa6s0QZywfVtnwVgeChogPY9G3HUWruMBD0pfkU9akkqusHHabz6KVKCE9tYwjnyOlPCJqsq\\\\\\\/WCbc2cAN9UIvHXp+Qtsc64NROMoOi7h1pvH8keeIC6jVXS+mSMLc3WkGwwKLPZGBCwwsMiswFhBwtq3NJh8KxZkqVc0eIkfXtQmD4EKjrgf8ORGZw5u1Pcmez5qkeESILjG8FvLmUQEnWwZHpJJz9ORvqF+4GcNujqg5GxPV8XYI9gOcDSk0tpC02OsKrpSiaBuU231cLMuhmldSp7rCtdLHx14NNsRFCcQsz4D\\\\\\\/I3iKQFJgVgl5lPr8pS6E+dJkNyMnEB8irent7A4uaxFd+jbpVAKjwOPd53gJDG1yI0p6ftaSO7oArBL6IPDfVJ+qn+SxnUpSMtNuvozw3dcW94LgajcNX1tZPo\\\\\\\/XBTRE5OHsRcQnpM6FXH9gEhfYwY9YZptZmAz7An+pk0InED07Jv\\\\\\\/axt9lPbg5qv4bmvzUxhxoMB+Eml2YX\\\\\\\/dnLNjZZvKnNK3SYnjRVcL04PjWf\\\\\\\/4oNM5OZPd+hFJfv5qiLZAeSJ6CiKnCLu3YIx73+sCUGdXnLRFEI7vknMiSFKiedhKCTwreQfQimJ2lyAnJEM67MYYYsufKr024fivovg4BAHv7VxL3H17c0kJ6dxr7ocym2iMvZW4UnJ+WiSsRJCeAkF7xrIoSEzi\\\\\\\/31r+T6xO+itfBoM\\\\\\\/cqooggjstqXduMqC9Z9UBrgYLr7SV1HMJWiFhI4qTxXKI+1ZGkYatGZbqU7AKG\\\\\\\/ShhaU\\\\\\\/u1YTGNCNifvSbfaXfoYPiNs+xiqzol8bQ1rNSSF922fd52xDyurQz72ZWVeCp8pAO4ypZkmGm5hjZs9B+48kUgsvG+WTqCRI1TdHmT72nCPE5T0GVqaEVmWymJ+i\\\\\\\/uX6ZgrDucm1Y7xUyoyYFmXYqxz7QVV\\\\\\\/A99i\\\\\\\/vkrBh9Sx++EXGlEWA3PrbRJebQWL0AFuFABXwFtit3etOS2pXsFqRhfIPB4NmfY1deedygfMA5Zmz09Eg82w0hoF8KkGO7Q6LpZ4MMN7T77KVaJr+0xy+nHZmhY+YBLEVVFKkmQWps6QHFy\\\\\\\/NuIzyHR+3RNjrlYyURTuHdFUjktIla2s0t9Ac3BuFXR9PeSRBJWoOPCQrXL\\\\\\\/lN\\\\\\\/UORK3aIwQMvngao9tioOrIrOnOHtd1Ejg8cmdAkTO+nbrekUZiT53kO+FEG0lnkQBjHJhXLugB\\\\\\\/Sy6IWUvqHFslmsNvpV8qi0mDzA0nctIyyz4gCzFI8CH0OBc0Mvq0im3KZSirdPX2lwD5zNMpoIpWQfJlzNO9Bp6AA\\\\\\\/c9UXHLCo8fHJgRFBNJlm93Nf54\\\\\\\/rQgJNYEmtPWHSBnjVGe5SrZa8ZxaslUXXP\\\\\\\/b1uk4RyDTVqEU22lX3eDXVX1Y9qyNTmgGhcPzTMOY13sFUMdNZc9brasoSYYS5r8b4Dkzp6L2VpouyQVerzHRqW0Wr45tBhVtIdRAUAPH1aKHw7As7YWsJOZK7yPBE9d5m4IjAWIQBqfWh8isDuEaU7CUyUhvMPPe9Dkw2asKlTiDquH0NRZVqvNqemM+wvh8yUC49uVHPG0r1ICsur4\\\\\\\/vMvI40Jf8bQEZmbXBEfzbUHI02mQPdEEI9\\\\\\\/uOyRjr04wi7BeeTFDgOiPvv\\\\\\\/rOIcay92OVXsEk5UVEi+TH4J64d0V5Lzfp+QesJM3VGVhBXZqClbsa62bG9qFcE914+VY6a9BdVbe9\\\\\\\/5nvtRSJ4J9uqxxwfwRxPda2lDRiW0rAT77U8lJBhxafC1DR2vQeRLTPHHGIjNLIQR\\\\\\\/LPmDOtPcZ9LOX2vQVvX3AMjIlbtHfn\\\\\\\/mxn+FTLKa1fdmp48MQ5WP43N0HSc22Mn81anl7tbHWGHBUBkUax38Z9RdYvyFIiq\\\\\\\/MNKIN8GHwrzGljiAa2bY+gE\\\\\\\/xYVYzfa5Jv4Ku\\\\\\\/SYpv9+C2\\\\\\\/6y0qxxcukL35av+cB0zFJ35CgBC8IKnqDrMGBiduL\\\\\\\/r4Bn21XkukEIGf72PlTneG22S\\\\\\\/ikLFGByr7su8OADp3ZqkFsVvESIfe4lccykxe7YILghSUYsQpcsYmEfIDlY483Z1xn4grXJ6gPGsfvVrpWNuBlKcpLyHWS89nlPODMtZN79VW\\\\\\\/28GnNBmOXqRDqiIpjaBFnO12LmKyDA4TBJCe\\\\\\\/i4iXiGuQPpcH6kvRn4PeG7awKVgWuQO8Ct+ZRqezoNR9CM9RVss3wSil5F1VthkOzdCcTK1RFJhZ1zKjDAqd1abfeNL+HjH1nAvzH7s76FFe3FTTv4Ff\\\\\\\/bT0wAADYxS\\\\\\\/czf8thIQv+AahFxA0lf\\\\\\\/vaiNNAlqPFy0+DFJoFbGJn4Ya0gNiXBPcC5uqBiXFBy93yIch2WSRE6e3CEWiUa0IjGW6\\\\\\\/j0n8rGuwBPYKNjXE4MbthKHPKq6zr5U3S8WRBfSOqf+6laRf4zPdScZwcvoEQZjt6+\\\\\\\/5u9HgEl5ZQkO0GvPZ+XC1JW68nsbcBYOC4LJdW9aPlWOrxYAOw8WPlDZ\\\\\\\/K24ab5+CMrSRm8X056CPFi3kEMk8ERG2eEv6I5koES3x+JpCDZmmpIzODIyoyWeQwyRtkiDQyBPCwlFM6xgW\\\\\\\/E8oeYVZl75ktR6+nuiJzDxk0g3HhZ7BdgbTV79JUddUfLYNBFCzuU42Ykv0BFzSqN1O+IvBYjeBw\\\\\\\/aA+pYaF1DuM1uicAqwOAgQbS1uioUNonHvZpjiY1JWEb7IPDLj6eq\\\\\\\/K1iRqOG5kP9LHFQWAWiS+u1OFhc5To3zMFJUvhI0qh0PURsy8X5z0TJZlXkIYLekFagnJsqERBzo+BRqtkIhxR\\\\\\\/NJpDvy+LPURapxORtRx+uURb\\\\\\\/xTFT0ZSjIxxajRflRbpYnqJAm73YlUoEtzDWrGKWFktVDLqyqFWY3iuBTKf0vpWdUh6YqXh8JdRYi21KA2EST6URqhC\\\\\\\/8mZdDWPNSj\\\\\\\/WyVPf1Mb0ld03wvPGz\\\\\\\/L4XNfb5z+LwKBt+RwYv4RVNbcd2CMYSEQyHIRpaGSscjrBNGTDnhXZ\\\\\\\/wdwqwgyHqBzxGIeiz22DOTFHzRdjD7Im16jr00FE9xvGiEABj5HaVRxmnyWOvPFZFc7i\\\\\\\/nMJ7M\\\\\\\/z7sW\\\\\\\/Suo0Li+6h5LC9Vrq9Pr55Qy2upc2Owb5tQ3NV3OH82Zjvk1jxt4upRErJAJb6IcsZyqR3H3cmbzt0nSSfsLawncP9\\\\\\\/t+Z+g1YQxF1I8I7tAqJNXrxHfyXA46yfSMWxNHQPOr+TeDQexDqGIJZVvQm1O52SNuEyF+fSpqrLAuNyC1+DNhddPzIPaZ9rFLxCoEkmx6\\\\\\\/jWk3zD10vLxNZtBPaL3GYAQLpEnTEjcKcLKJ\\\\\\\/B2h7T1TQ1jTvgupME4k6pZg5c\\\\\\\/1sQ\\\\\\\/vNbYobOQer8y1o2D+bG2IdY8CnavafIJxP16Z3oddbve\\\\\\\/FeRt4Ai50a+rfMIEeRVY0J8ChI\\\\\\\/c79AYaNLRGKomqmSpCwP3EiYEP2FnSetfg5ZcDibMQS4DkX72fuMllLYh8lMKsDCYpDi4+Gfwa8hZK6UPsE+aMpeGwwbb4UUX6vo9bIIes7\\\\\\\/EXashMHvoEBOZKHdaC8Sp1k\\\\\\\/rBvM4h7KRCXI4HaaAGfjM5+3CSPUUj5zlVX0vugEG57+AnqLXxhzILIcyVVnq6H0P8C7J2jzli9xrNzN1GAlM2+TbNODujuA7l0BEdib5PLGzH\\\\\\\/hhLjZrwRX15gG1ZfwodnOfG+0COeN8xYJwGIYgFCWmdRev5BkJE6WJibzsgajRPwtf6wve7Fnjg2HUWJ+DOMafiNCLAC3e\\\\\\\/Jumeef+HR4kU\\\\\\\/1Lh7uiJpeL07q6y0JteYa4t0\\\\\\\/sXVVwGq55gn6MvukkaKZaH5d8Irl6smwALJLsMhoQQPS8QzK63GTrfkvG6zPMVssCd7bTKYBfBQ8IZrsW\\\\\\\/k6qRVDDLh96\\\\\\\/9dIHA9DerAS7XSHSjGYohHeBajDR1yEbm6IzUFV7mWXMbULtAYMnaR1O485nm4M1\\\\\\\/KeWbKKaMK+FcwQF76yQ260EmlQmOOCXvjc1UwTbjhQNJ92\\\\\\\/dKreMC6hJe3gQULTdrpyhOCJ7kkT9yVkUOwG6R\\\\\\\/03rwEosr14fCfK8lFyzKqUmb3cpmnA9CQH5mS2LAWCYdDsysVVYbkmqJTiIY0g+zS4D+Dtme8+sM9rctgyag+k1y0hGMywJc\\\\\\\/kgZK7vHC0On7Jfp7mLpIzWUq359vuwl5ZqAIl4f\\\\\\\/CxR3UE0chDN9UfwUwKA0H2icKUhuDTNhsY7A9jFI8RwdPlbCnoqCF3+jfzTNUIBmyWRj2DqqnEEiVBTbsWq4EscXb2qx4hDnmddFEoc2wQLliJprLlbs9nbvKEhHFrBjsGi4fcJXG4YY84CmGt+SxVWk+lo0rQiQBjxKEtU63FaDv59Nk\\\\\\\/aeskrKOpZcrZ4rQBWuXULfyc8lXRd3ob3iMWueec1w77wt\\\\\\\/5RyLxTlNTmJjGlQ9r\\\\\\\/9kVyZGR\\\\\\\/nqWdBzMNQJmt87HLwKBdW1R\\\\\\\/FAKrBXOQS\\\\\\\/dChfdOLkPj7P+WN6KTINagNJDXWorlIz9+IxRV\\\\\\\/QM+jkyB2isEkA4KTcokuCGhNOPxdbi1eOMLASVgzIHB+p9nBtSZuDx7y+lSKoJAFEQ3mbrDM1dRe8veshugTbWFreOhzu7PjH2HYl4cyFL6QEvvkkUfcuJTodkvpRc4uq4xBKYlJreWtx1QJhcCkydh6+HQldn6zPN6+K25n7+rnr5J8jg4ypXa0EhhlOsQWAGXZWoN0Cvau2fJldViRTip7E5vnYGOCghBt8h6C0wLTY2Pq84XozoZ610lUOu\\\\\\\/QdsMkoZodh4IWlipaNUXYG4VsaJ9rDYhqPRcBKE7aavrVA+2uobVqz\\\\\\\/GFqUXEnVTxpTLFxi9s\\\\\\\/B2NgNSj+YZAN2Ytt98335jl6f5KDEm39fg0iU+2oTdCsdlDilXv6YqTR0oqhzZJmvg0mIRqLVTsyTrfydDY7uHXFG1I5N9bBB53bRye03Z4d2PphNeYEx+O6hOiuELDYBP8rMoFwy4+oZ7GSsomDPeUled04K+ZlIZFFU\\\\\\\/nbpd1W1k7qCRM0lls2cAvFKKEHDo1+BO64pA6SSaCABgOi9jzRdf\\\\\\\/cesGfFxq8teLs5YBaejDT6EPA6uVmPO7Brer7cegrmlE0UTYJfPNQSwseF6bxl4O4gFLcLkBTZUl9lUeTmHANUdJgDmTrlTJk8SnTXLz33IbxlFyImw3G\\\\\\\/7OD4cbppJmOW\\\\\\\/y1mmjoHVu93U9YXqr7ChEi3gr5GumLTGTS7dmPxCSqloOKRMUam4Ti7ys7k1eH2yYabpxj62nSbrzRLfhrt14u\\\\\\\/MkveU0KJUS4AmjomwGFMugbNXQichsMvOjkzdMVgIoOf\\\\\\\/0zvHzaL7SqorsyqMiUTDHLicGBmf4AJP3zi5hLFcIok7d2JvaYE\\\\\\\/ggk5s6UzX+6z4YzSdfypbumBsXArNEGX5mLnXTJOhUeTIGT4zexx87H\\\\\\\/t5pzzJlISrQd\\\\\\\/gJ+wBXgO4XsL8EeU1e\\\\\\\/v3C0cgaOx3\\\\\\\/b5SvzuOEZjZ5\\\\\\\/wKClscNpeAcMuXr4XdQFbLxlEi4KZQt3dt3\\\\\\\/UYWfALPsFuRQ5L4DcpKZpVJs7i3aahuhwylRp83x\\\\\\\/5tg9o1mRidk2HQVfvphjswMT8Pe2pZjyNjQ6qjrQNTNwB0\\\\\\\/3h\\\\\\\/IoT3xVhPiyJtr+0oE97wRJW8eCMdaoPPvoj\\\\\\\/udpsQM5nH+5gt2Er6e3uBd9A7LaSD5jipYlJGiQ00vFXpu4bN63mKOnq1Q0C4JodBJAsQRwo+V00dE+uWdW+zuQ4L+mPpm0G0bRv3ZAFTxuyyBFfb4eHxXdPYiYiSeMsFz61jWSvf7LPtbG83DYOnvEFbvlmSSxh8c05xOVpnV5+oD5Z4zLz3CDf\\\\\\\/xzsyFXY\\\\\\\/XgtPBhk26uPYRwRbtS91IQzRUNevtnHn4OJaV2Ur3s4DgNoNnkJELVtHbhziqbZ0iKXwkhKIytddcFNTFgdtRokte+9r8JXOytB3FlZKEy5In8ly3esgWY4t+OOt3XHhqWSAc9B3t0xdLkW9hL+khVX1GN+hDhFulsvZZCzg2ct7d9VL6odi19vj6J\\\\\\\/VhDaGDFuAQYOAlieArH5q2lVACbZB43tc1HNeYComIUgX3IbYXPhgIj7ibXIKeNaYb7zcl5qCObXpJ4oXQljBUN6sLVtCpO9m1CDLJ3LGQFlDzrgDMiEnouBKw34iSGJvHaBosxIm9NGc6M9AdHbui7LfjZNIqIT+hsTEMZ9qCYr5vhL+QMF949KeXwbnOXRdwUYSxmarH64HAPgnd4ubes6ClEs6ACSB6hetAYz2lBXrYzeCgl\\\\\\\/RInRPq6jhlmTIGsv1tSOEqKsSROFwCXDPdlh7aiWfZjqAQraPkdj3bUlBAjsdf2CXXu7FlFNOpM+9\\\\\\\/gXzktFxdP1OQ6i7g7yuBCm+AVUnd40ZMOsMqkhgHQCEQbCkoPepgUC8ZA9W8OFzGvcNl5IC2jBQdsUahG7H4pWOJvPCcGtswOzX3FsrSxrj+Wlr7fhZTvHSKWlkPyLBvwgWTBY23\\\\\\\/150gf97XpWQql7lpebPlsNpZYkyBdyAtSF9ABDE7YPOE7lYArYuYRBy87UYPB\\\\\\\/vH5S\\\\\\\/wp4magwWUGLaD08ZafKuK8nx9vRmPNTTrxYNI1NW\\\\\\\/CQewNASAvnbSP8CxcViSUeVc1vAHlDIKm13PsbHuojdLHq75AE5qeJwMG1Wm6vI0H0L2Wv+oLLwwWTqBTURryPrlEmqE7z47QdoWjGrqI7SFqMmUx17dpzOxUgZ5n\\\\\\\/tX4FZXwgLywb1stpbO6miz+rd8oMximL\\\\\\\/SW11ad7R2I2i3N5+PBYUyOP2x3y8EeDOCKqYZjOMx0ihYDAkyPgMgLuWvUZ\\\\\\\/Wzixs3f3vL\\\\\\\/gIzK4ekeLwA\\\\\\\/EsyqYwnf75CeDjoolpJ9bkJgwCgTrhdX2w872O7FZW7rUT2kRoB\\\\\\\/zw6NlcFo5zHkA8od8DxE2FlFXlnWOlSE01Z51fLAaFLws8cpBmPasZqxIVj5uiyIDl73V9o36FShduuXZ3SbfxL7Dj0zgx7xGWmnzWdssBePeYLaOVJUUubeW1UHlBzIUeNXaDZi7gdxGCRvwSV7Je7ardqViCxdZ0fXvCLoM6dQSs7BcVCzVF8HMQd95olLaIlOjsQs7V4KEns4OSuESoAreMBKb9ZZPeGpIKbB8bllacxTtcKIQj+U4uLrMXFY7uTONoeOjAmYALHq6iHt2y\\\\\\\/F5M5wbBTEaeYCFqfBXfIDn+nsiTFUnbegB3ZX57L0lLbp37Jb9TxIOwH9IQZoRX6hrKU36hpDOPLCRz+x\\\\\\\/Dy3pSogHU9gNm\\\\\\\/5BAAtVYYue2UdPo2Z1N0\\\\\\\/Ka2HkriL6LVNEiYbu7LWk5vvrYWZawObItKgGMr8qX0mBxSegHHT9SoJNcdEEg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ry\\\\\\\/86JwqTipbYvCLFDpNBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221bd875373d046e387dbc6ca095531bb001f505ff786dc87307c4c40a0701b6f2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225xRo3nDfqZZeHvpwps+TAcnnFOJkkCSvK9bVNqo8\\\\\\\/+kKv1coO6XlKiiQvg6CN6VBH7NCYS5ChU\\\\\\\/xhPWAcsy+ws0H9Vwf9mJFZP1XbuogfzJ4mig92qiE4bDZMcd0W5hpBKHiToHs\\\\\\\/NlbCKYwlz11\\\\\\\/lWJ7dIL51AhuF+ugoGP9WS+LhMiwbw+kIBHTcSBtuZRwdQmLRMMv3GPOaYnWU1J4CuHV4yBG\\\\\\\/2dDq+C2TLtpeS0BoMEMYqyi3CL7FaYJjmfKMjM0r\\\\\\\/BIvuVuPoLwCWYOYtKyhM7uh60eCUJg1qzp6jI66DP+c0zg3kF7I2oTJ03Y+TnARjeee4V8XTOwmD3XS5QbOhG21bMbPN9emKZ6JeLsEnDntqKSwtxgPqkDQbUv24c517s2tj5\\\\\\\/rhEL9C96mpwWG2za2LRDDsqZsKR10IphI29As9U+aIKgfq\\\\\\\/hjZJII2kF9YSLUaqO4JVPEocw3mFsAirQs13ZLEA\\\\\\\/4xgpwzJU\\\\\\\/WQxzVKEoeRLFmuwXR643JDg3zga5hWm2nb0W+XIaaLfBcJ26mOWeFY3wjDlpZa6rC4ejPpPhK1+rc9J5MLxgzNqAL\\\\\\\/o8sk6cJ5m3O6PCbNCZyxHJS14ghyLL+AopazdpE8E17\\\\\\\/a3kXIW77NG7tCoOZRUrv+4yB0wrplZp7zTje2jiyltupshIhgu0bwucxMk05FGv6Cb31zbtnr7JpHv7SuTrPN+ZqdSsODX8oKg34WHroBD1+H0bOE1Z2+4\\\\\\\/rwg1UxEEt3uwR9g7uwUHhA\\\\\\\/j\\\\\\\/9jhACD0a0Gx5YUIQJfYvLuq9MmpVep2Iy0WJDC1Y08\\\\\\\/F+LHokUun1uQQqTkFMewugnB3DQN0h1Isw\\\\\\\/V6WSVqT+uH85GmjoJQ52ozqaE9XafGgFkAKu16w0SWDJaC46buBT7FTcyTI5EFtm8dVRStF8CSzJiUtup+70SO6mVBEVzMOuUzpU1tOQcS3k\\\\\\\/25f4uVqIy0yraGxQJiA+wOdlLPa4zYtTtP\\\\\\\/r1z9Uk+oGZ9B6ruYyX46Dj+r5jSPCgL+XnOy138IoZMvwsBJ\\\\\\\/32eIre7LF\\\\\\\/5yC1lEjIq\\\\\\\/lgBd6s6bH9XMYZqcXPdnEQPc7GoP5BkBYItnym1ZkqsuVK34ZHl9hEYSjoiY55h3lRvFJUveRGT\\\\\\\/7Y+cdw0sMnmCc9v3oUfjuAd8\\\\\\\/qYScd+9k3Vh5dtTqDGSUMKjw9hRwOJ2cYJC23XVY0yTqLZdq8tPlzlRz5oDnvTdZl4XzKZ74K1bLpFsQC5+87k1gwHpa0lKUgD5SPi1dyCHQUDMrW595IUTUQv5lsbUIBnP44u6CCAVYm2\\\\\\\/hiOvA4yjdkOfVHoGApp7xzmqipJm+5GV64ItlaihcDnXKqxGITpWPxcTO\\\\\\\/kO7+GPtEIoeFdWO\\\\\\\/TW3hY14nCgJ0vkAuqr5WRbbgYa6urgprH5Yw+jyxiXDJCll465Z\\\\\\\/iL59AfCa6SqUzoOoYIZ38+0ZQS6ptrT3GgEVTxAC+QDd0wvDQC5GjYMI9AntrYtlIqzjFJvoaWMPhCr88Dka+L33NB61K0XbJc6MXYz6e6R563gBcmPGIZ47SPeyMU99DqHA6UyPve5O4NG8t4m+Z6mWD9W7D4+StFvUMI6SoKmqnMrFkyrfCXkv+Ag4CDlLCU18stFhiXmk5mTzcPumCWL5gJdUvBzAnJLImSDMEYlf162vmUEmi4iiDHixKGsMLv3V3hEpMgvJd9XCpVC3Nvtr8WdJ5RqzfWbhHZzjhHkAX8sTI3cf66cDjY7E1QdJfiiUqbsfwgJ9a8xxusTVnPGLpj570OfaP6rXM0OqYvo3benY2HuIhsSPGfVCzOofvIH62TkPzW1dU01ZSu2S3FpZMV6dGOiQiqLY7ljIE9CxYoZJIbT3yBu3GBOfU20isYzpMoJeN66TampxssPEdP+6HQuCa6108PXv3ujWER02d9RojAMXE9IB6cknG+DSh5D+co35w5hSXRgUUd6pjsYR60q96t7RARUJyDZQZC3Gm49NSCtRgSQZIQJcw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022N9JcQGcfg4pFaUtsGOwXfw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ad4353658560143abea2ef2e69dfb14dd2695830f11fa635017e4b6bda21ae6e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459533 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1511" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-59XKnLJnB9IloHsNjp38NFrxxogTMeCmnDRRx5uv3b4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1511" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UxHTxWUcqJIm-bHQNGlE6TLVc1bSCsj-sXXD-ab7YRM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/9/1493" title="The Developmental Transcriptome of the Mosquito Aedes aegypti, an Invasive Species and Major Arbovirus Vector" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/9/1525" title="Genetic Determinants of Cell Size at Birth and Their Impact on Cell Cycle Progression in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459533" id="node459533" data-pisa="ggg;3/9/1511" data-pisa-master="ggg;g3.113.007096" data-apath="/ggg/3/9/1511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hanni P. Kärkkäinen</span> and <span class="highwire-citation-author" data-delta="1">Mikko J. Sillanpää</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1511-1523;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007096 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459533" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hanni P. Kärkkäinen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agricultural Sciences, University of Helsinki, Helsinki FIN-00014</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hanni%2BP.%2BK%C3%A4rkk%C3%A4inen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4rkk%C3%A4inen%20HP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHanni%2BP.%2BK%25C3%25A4rkk%25C3%25A4inen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mikko J. Sillanpää </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Biocenter Oulu, University of Oulu, Oulu FIN-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%C3%A4%C3%A4%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%C3%A4%C3%A4%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25C3%25A4%25C3%25A4%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mjs@rolf.helsinki.fi" class="" data-icon-position="" data-hide-link-title="0">mjs@rolf.helsinki.fi</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/9/1511" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459533" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1511.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459533" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1511.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459533" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1511.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459533" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/9/1511.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459533" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459533/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Because of the increased availability of genome-wide sets of molecular markers along with reduced cost of genotyping large samples of individuals, genomic estimated breeding values have become an essential resource in plant and animal breeding. Bayesian methods for breeding value estimation have proven to be accurate and efficient; however, the ever-increasing data sets are placing heavy demands on the parameter estimation algorithms. Although a commendable number of fast estimation algorithms are available for Bayesian models of continuous Gaussian traits, there is a shortage for corresponding models of discrete or censored phenotypes. In this work, we consider a threshold approach of binary, ordinal, and censored Gaussian observations for Bayesian multilocus association models and Bayesian genomic best linear unbiased prediction and present a high-speed generalized expectation maximization algorithm for parameter estimation under these models. We demonstrate our method with simulated and real data. Our example analyses suggest that the use of the extra information present in an ordered categorical or censored Gaussian data set, instead of dichotomizing the data into case-control observations, increases the accuracy of genomic breeding values predicted by Bayesian multilocus association models or by Bayesian genomic best linear unbiased prediction. Furthermore, the example analyses indicate that the correct threshold model is more accurate than the directly used Gaussian model with a censored Gaussian data, while with a binary or an ordinal data the superiority of the threshold model could not be confirmed.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/multiocus-association-model" class="hw-term hw-article-keyword hw-article-keyword-multiocus-association-model" rel="nofollow">multiocus association model</a></li><li class="kwd"><a href="/keyword/g-blup" class="hw-term hw-article-keyword hw-article-keyword-g-blup" rel="nofollow">G-BLUP</a></li><li class="kwd"><a href="/keyword/threshold-model" class="hw-term hw-article-keyword hw-article-keyword-threshold-model" rel="nofollow">threshold model</a></li><li class="kwd"><a href="/keyword/ordinal" class="hw-term hw-article-keyword hw-article-keyword-ordinal" rel="nofollow">ordinal</a></li><li class="kwd"><a href="/keyword/binary" class="hw-term hw-article-keyword hw-article-keyword-binary" rel="nofollow">binary</a></li><li class="kwd"><a href="/keyword/censored-gaussian" class="hw-term hw-article-keyword hw-article-keyword-censored-gaussian" rel="nofollow">censored Gaussian</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li></ul><p id="p-4">Genomic estimated breeding values based on genome-wide sets of molecular markers have become an essential resource in plant and animal breeding (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Eggen 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nakaya and Isobe 2012</a>). The most commonly used approach to predict the genomic breeding values is the genomic best linear unbiased prediction (G-BLUP), a direct descendant of the pedigree-based BLUP model. G-BLUP uses the marker information in estimating realized relationships between the individuals, and then uses the marker-estimated genomic relationship matrix in a mixed model context (<em>e.g.</em>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">VanRaden 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Powell <em>et al.</em> 2010</a>). A relatively recent contender for the BLUP-type of model in the genomic selection field is to apply simultaneous estimation and variable selection or variable regularization to multilocus association models (<em>e.g.</em>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Xu 2003</a>). Contrary to G-BLUP, a multilocus association model uses the marker information directly by assigning different, possibly zero, effects to the marker genotypes. The genomic breeding value of an individual is then quantified as a sum of the marker effects. Because the number of genetic markers is usually orders of magnitude greater than the number of individuals available for the study, the selection or regularization of the predictors is the key factor of a multilocus association model. In Bayesian genomic selection models the regularization of the excess predictors is performed by shrinking the effects of the markers not linked to the phenotype toward zero by assigning a suitable shrinkage inducing prior density for the marker effects. The most widely used shrinkage inducing priors comprise the Student’s <em>t</em> and the Laplace densities, the former of which has been used by <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> (2001)</a>, by <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Xu (2003)</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yi and Banerjee (2009)</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hayashi and Iwata (2010)</a>, and <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Habier <em>et al.</em> (2011)</a>, whereas the latter has been used, <em>e.g.</em>, by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> (2009)</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> (2009)</a>, and <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sun <em>et al.</em> (2010)</a>. The models relying on the Laplace density are commonly denoted as Bayesian LASSO (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Park and Casella 2008</a>).</p><p id="p-5">In the basic form of these linear models, the response is assumed continuous with normally distributed residual variation. However, in many instances the actual phenotypic records are given as binary case-control, ordered categorical (<em>e.g.</em>, from nonaffected via different severity levels to strongly affected) or censored Gaussian records (<em>e.g.</em>, a logarithm of an event history or survival data, or spiked phenotypes as in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman 2003</a>). With a binary response, either logit or probit transformation is used to convert the binary response into the probability of the positive outcome. Both logit and probit models are developed by assuming an underlying continuous response, often called the latent variable or liability, dichotomized by setting a threshold to limit the two classes. The difference between the logit and probit models is the assumed distribution of the underlying response; the logit model assumes a logistic and the probit model a Gaussian density for the underlying variable. In the frequentist framework, the discrete response is usually modeled without considering the underlying continuous variable, leading to quite different estimation procedure for the model parameters than in linear Gaussian regression. Although under a linear Gaussian model the maximum-likelihood estimate has a closed-form solution, under the logit and probit models the partial derivatives of the likelihood function with respect to the regression coefficients either do not exist or cannot be determined analytically, and the maximum-likelihood estimate must be computed iteratively. The logistic link function leads to somewhat simpler algebraic expressions when handled as an integrated part of the linear model and is therefore often preferred by the frequentists (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McCullagh and Nelder 1989</a>).</p><p id="p-6">Contrary to majority of the frequentist models, in the Bayesian context the underlying continuous response is included into the model as a separate latent variable layer, usually assumed to follow a Gaussian density. These two factors, that the augmentation of the latent variable is now an additional layer in the hierarchical model and that the augmented variable is assumed Gaussian, permit the usage of the original linear Gaussian model as such without further modifications. Moreover, in genetics, the normal assumption is especially reasonable as the inheritance of complex traits is determined by multiple genes and environmental factors resulting the liability likely to be normally distributed.</p><p id="p-7">The ordered categorical records can be dichotomized and analyzed with a binary model, or alternatively incorporated as Gaussian observations in a general linear model (<em>e.g.</em>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meijering and Gianola 1985</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> 2013</a>). The problem with the former procedure is that it loses the information contained by the extra categories, whereas in the latter method the model is in effect not compatible with the data. Similarly to a binary phenotype, the ordinal phenotypes can be assumed to have an underlying continuous response discretized by introducing thresholds delimiting the categories (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hackett and Weller 1995</a>). Now there are several thresholds at unknown positions, but otherwise the binary model can be seen as a special case of the ordinal model. The threshold idea can be extended to censored data sets by simply using the uncensored data as such while considering the censored phenotypes as latent variables. The advantage of this approach is that it does not matter which part of the data are censored (right, left, interval, two way censoring), the latent variable is always handled similarly.</p><p id="p-8">Threshold models for ordinal and binary traits have been considered previously by several authors. A threshold model for BLUP with fixed thresholds has been covered by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola (1982)</a> and with unknown, estimated threshold positions by <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sorensen <em>et al.</em> (1995)</a>. Multilocus association models of binary and ordinal traits have been considered by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hoti and Sillanpää (2006)</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Iwata <em>et al.</em> (2009)</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">González-Recio <em>et al.</em> (2009)</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">González-Recio and Forni (2011)</a>, and <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> (2013)</a>. Furthermore, the threshold approach for censored observations has been considered by <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Broman (2003)</a>, within BLUP context by <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sorensen <em>et al.</em> (1998)</a>, and with multilocus association models by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sillanpää and Hoti (2007)</a> and <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Iwata <em>et al.</em> (2009)</a>.</p><p id="p-9">In this article we aim to enlarge on the threshold model more generally. We consider two Bayesian hierarchical models representing the alternative modeling approaches, a Bayesian version of the G-BLUP and a hierarchical Bayesian LASSO (<em>e.g.</em>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Park and Casella 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>) as a representative of a multilocus association model with variable regularization, and show that in case of a binary, ordinal, or censored Gaussian phenotype the same additional latent variable layer can be plugged into both types of the genomic selection models. In fact, the additional latent variable layer can be subsumed into legions of different linear Gaussian models; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> (2013)</a> have used it with BayesA, BayesB, and BayesC<em>π</em>, whereas in our previous work (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>) we incorporated a binary threshold-based latent layer into 13 distinct models, including a Bayesian G-BLUP, BayesA, BayesB and both hierarchical and nonhierarchical Bayesian LASSO. In this work, we exemplify the threshold method with a hierarchical Bayesian LASSO as it proved the best working model in the aforementioned previous work and, on the other hand, we did not want to pick anything lesser known, such as the extended Bayesian LASSO (introduced by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mutshinda and Sillanpää 2010</a>, used successfully, <em>e.g.</em>, in <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kärkkäinen and Sillanpää 2012b</a>) to serve as an example.</p><p id="p-10">A more immediate practical offering of this paper is the fast <em>maximum a posteriori</em> (MAP) estimation algorithm presented. Traditionally the parameter estimation for Bayesian models has been performed by finding the posterior density by Markov chain Monte Carlo (MCMC) sampling; however, because of the ever-increasing number of markers available, the focus within the genomic breeding value prediction field has been shifting to the fast MAP estimation methods (<em>e.g.</em>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2009</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Yi and Banerjee 2009</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hayashi and Iwata 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shepherd <em>et al.</em> 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Xu 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sun <em>et al.</em> 2012</a>). The MAP estimation is based on finding the maximum of the posterior density rather than the whole posterior distribution, usually by an expectation-maximization (EM) algorithm (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dempster <em>et al.</em> 1977</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McLachlan and Krishnan 1997</a>). The difference in speed between an MCMC and a MAP estimation algorithm is far from trivial: while the run time of an MCMC algorithm is typically hours at the lowest, our generalized expectation maximization (GEM) algorithms perform the example analyses in some 20 sec. With existing genome-wide data sets a fast estimation algorithm is an invaluable asset because it will significantly facilitate the frequent re-estimation of the marker effects and breeding values, the use of cross-validation and permutation-based techniques, and massive simulation studies of breeding programs. Nonetheless, in all of the Bayesian methods for threshold traits found in the literature the parameter estimation has been performed with MCMC. In this respect the methods for discrete data are a bit out of date compared to the methods for Gaussian traits.</p><p id="p-11">In our previous work (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>) we already have considered a kindred threshold approach for binary traits in MAP-estimation context. However, an ordinal data set poses an additional challenge because the model has to address the unknown thresholds as well as the latent response. Hence, although a binary phenotype can be regarded as a special case of the ordinal model considered here, a binary model is not readily expandable to several categories. Since the MAP-estimation methods are able to handle large data sets far more efficiently than MCMC methods, it is clear that an applicable MAP-algorithm is needed for all conceivable types of phenotypic observations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-12">Our hierarchical Bayesian model, depicted as a directed acyclic graph in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, consists of two separate parts, the linear Gaussian model and the threshold model. Under the linear Gaussian model the phenotype measurements are assumed to be continuous and follow a Gaussian density, while the additional threshold model handles binary, ordinal and censored Gaussian observations.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/9/1511/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hierarchical structure of the model framework. The ellipses indicate random parameters and rectangles fixed values, whereas the round-cornered rectangle representing the Gaussian phenotype may be either, depending on whether the threshold module is included in the model. Solid arrows indicate statistical dependency and dashed arrows functional relationship. The background boxes indicate the main modules of the model framework." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454556148" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Hierarchical structure of the model framework. The ellipses indicate random parameters and rectangles fixed values, whereas the round-cornered rectangle representing the Gaussian phenotype may be either, depending on whether the threshold module is included in the model. Solid arrows indicate statistical dependency and dashed arrows functional relationship. The background boxes indicate the main modules of the model framework.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/9/1511/F1.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/9/1511/F1.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/9/1511/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/9/1511/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463133" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Hierarchical structure of the model framework. The ellipses indicate random parameters and rectangles fixed values, whereas the round-cornered rectangle representing the Gaussian phenotype may be either, depending on whether the threshold module is included in the model. Solid arrows indicate statistical dependency and dashed arrows functional relationship. The background boxes indicate the main modules of the model framework.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Under the threshold model, we assume that the observed phenotype <strong>w</strong> consists of either ordered categorical or censored Gaussian observations, and that the ordered categorical variable has arisen as an underlying normally distributed continuous response <strong>y</strong> is rendered discrete with a known number of thresholds at unknown positions. The underlying Gaussian response <strong>y</strong> can be explained by genetic factors with either a multilocus association model<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>or with a G-BLUP model<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>(the linear Gaussian model module in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Both (1) and (2) are linear Gaussian models commonly used in genomic selection. In the former model (1), the matrix <strong>X</strong> denotes the genotypic records of <em>p</em> biallelic single-nucleotide polymorphisms (SNP) of <em>n</em> learning set individuals, coded with respect to the number of the rare alleles and standardized to have zero mean and unity variance, and <strong>β</strong> denotes the marker effects. In the latter model (2) the <em>N</em>-vector <strong>u</strong> denotes the additive genetic values of the <em>N</em> learning and prediction set individuals, whereas <strong>Z</strong> is a <em>n</em> × <em>N</em> design matrix connecting the genetic values of the <em>n</em> learning set individuals to the latent response. Although the additive genetic values of both learning and prediction set individuals are present in the model (2), the latter do not contribute to the likelihood. <em>β</em><sub>0</sub> denotes the population intercept in both equations. The residuals <strong>ε</strong> are considered independent and identically distributed under both models, with <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-3.gif"/></noscript></span></span>. To avoid overparametrization and ensure identifiability, the residual variance component <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-4.gif"/></noscript></span></span> is set to unity when the Gaussian response is unobservable (see <em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cox and Snell 1989</a>). When the actual Gaussian response is fully observed, the threshold module is omitted from the model (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) and the residual variance is estimated simultaneously to other model parameters.</p><p id="p-15">In this work the regularization of the excess predictors is performed by a hierarchical Bayesian LASSO (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Park and Casella 2008</a>), by specifying a Laplace prior density for the regression coefficients. The Laplace density works best and provides an easy derivation of the fully conditional posterior densities for the effect variances (even though not actual conjugacy) when expressed hierarchically as a scale mixture of normal densities (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Park and Casella 2008</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>). The hierarchical formulation of the prior densities, also known as model or parameter expansion, is a well-known method to simplify MCMC algorithms by transforming the prior into a conjugate and hence enabling Gibbs sampling, and to accelerate convergence of the sampler by adding more working parts and therefore more space for the random walk to move (see <em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gilks <em>et al.</em> 1996</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gelman <em>et al.</em> 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gelman 2004</a>). In our previous work (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>), we showed that the hierarchical formulation of the Laplace density is superior to its nonhierarchical counterpart also in EM context. The hierarchy is acquired by setting a Gaussian prior for the marker effects <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-5.gif"/></noscript></span></span> and an exponential prior to the effect variances <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-6.gif"/></noscript></span></span>. Unconditionally for the effects this leads to a Laplace density. In <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> the hierarchical formulation is observable as the fourth, latent parameters, layer. The scale parameter <em>λ</em><sup>2</sup> of the Laplace prior determines the amount of shrinkage introduced by the prior, and hence the sparseness of the model. Because the optimal amount of shrinkage depends on the heritability and the genetic architecture of the trait, the number of markers and amount of linkage disequilibrium (LD) present in the data, the selection of the hyperparameter <em>λ</em><sup>2</sup> is of central importance. Although some authors prefer to give a fixed value to <em>λ</em><sup>2</sup> (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Figueiredo 2003</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2009</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Xu 2010</a>), the most prevalent solution is to estimate it simultaneously to the model parameters (<em>e.g.</em>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yi and Xu 2008</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Shepherd <em>et al.</em> 2010</a>). To this end we give the hyperparameter <em>λ</em><sup>2</sup> a Gamma(<em>κ</em>, <em>ξ</em>) hyperprior, and tune the rate parameter <em>ξ</em> of the gamma density to a suitable data specific value (sixth layer in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). The prior densities for the population intercept <em>β</em><sub>0</sub> and the residual variance <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-7.gif"/></noscript></span></span> (when applicable, <em>i.e.</em>, under the Gaussian phenotype model) are uniform <em>p</em>(<em>β</em><sub>0</sub>) ∝ 1 and Jeffreys’ <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-8.gif"/></noscript></span></span>, respectively. As the model parameters are considered <em>a priori</em> independent, the joint posterior density of the parameter vector becomes<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/inline-graphic-1.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-9.gif"/></noscript></span></span> is a vector of the marker variances.</p><p id="p-16">Under the Bayesian G-BLUP (2) the prior density for the genetic values <strong>u</strong> is a conjugate multivariate normal <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-10.gif"/></noscript></span></span>, where the realized relationship matrix <strong>G</strong> has been estimated from the marker data. In this work the estimation has been performed with the second method described in <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">VanRaden (2008)</a>. Contrary to the classical framework, in a Bayesian approach the variance components are estimated simultaneously with the genomic breeding values (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hallander <em>et al.</em> 2010</a>; <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>). The genetic variance component <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-11.gif"/></noscript></span></span> has been given a flat Inverse-<em>χ</em><sup>2</sup>(<em>ν</em>, <em>τ</em><sup>2</sup>) prior distribution with a substantially large <em>τ</em><sup>2</sup> to ensure the flatness of the density. Similarly to the multilocus association model, the prior densities for the population intercept and the residual variance (under the Gaussian phenotype model) are <em>p</em>(<em>β</em><sub>0</sub>) ∝ 1 and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-12.gif"/></noscript></span></span>. The joint posterior density of the G-BLUP model parameters is given by</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/inline-graphic-2.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span><p id="p-17">Given the value of the continuous, normally distributed latent variable <em>y<sub>i</sub></em>, the binary or ordinal variable <em>w<sub>i</sub></em> has value <em>k</em> ∈ {1, …, <em>K</em>} with a probability<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-13.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>where <em>t<sub>k</sub></em><sub>−1</sub> and <em>t<sub>k</sub></em> are the thresholds delimiting the <em>k</em>th category. If the ordinal variable has <em>K</em> categories, there will be <em>K</em> + 1 thresholds, such that <strong>t</strong> = {(<em>t</em><sub>0</sub>, <em>t</em><sub>1</sub>, …, <em>t<sub>K</sub></em>)|<em>t</em><sub>0</sub> &lt; <em>t</em><sub>1</sub> &lt; … &lt; <em>t<sub>K</sub></em>, <em>t</em><sub>0</sub> = −∞, <em>t</em><sub>1</sub> = 0, <em>t<sub>K</sub></em> = ∞}. One of the thresholds must be fixed in order to center the underlying distribution; we adopt the common fashion to set <em>t</em><sub>1</sub> into zero (<em>e.g.</em>, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cox and Snell 1989</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Sorensen <em>et al.</em> 1995</a>). The <em>K</em> − 2 of the thresholds <strong>t</strong><sup>⋆</sup> = {(<em>t</em><sub>2</sub>, …, <em>t<sub>K</sub></em><sub>−1</sub>)|<em>t</em><sub>2</sub> &lt; … &lt; <em>t<sub>K</sub></em><sub>−1</sub>} are considered unknown, and are estimated simultaneously to the model parameters. With a binary response (<em>K</em> = 2) there obviously are no unknown threshold values.</p><p id="p-18">Although the likelihood of the observed phenotype <strong>w</strong> follows a categorical density, conditionally on the underlying response and the thresholds the observed ordinal phenotype is known with certainty and hence the likelihood P(<em>w<sub>i</sub></em> = <em>k</em>|<em>y<sub>i</sub></em>, <strong>t</strong>) gets a constant value, zero or one. Therefore, the fully conditional posterior density of the latent Gaussian variable <em>y<sub>i</sub></em>, given the value of the ordinal observation <em>w<sub>i</sub></em>, corresponds the prior density of <em>y<sub>i</sub></em> when <em>t<sub>k</sub></em><sub>−1</sub> &lt; <em>y<sub>i</sub></em> &lt; <em>t<sub>k</sub></em> and is zero otherwise. As the latent variable is an additional layer in the hierarchical model, the prior density for the underlying Gaussian response <strong>y</strong> is the likelihood of the latent variable under the linear Gaussian models (1) or (2). The likelihood of the latent Gaussian variable, given by the model equations (1) or (2) and the assumptions below them, is a multivariate normal centered at <em>β</em><sub>0</sub> + <strong>Xβ</strong> under the multilocus association model and at <em>β</em><sub>0</sub> + <strong>Zu</strong> under the G-BLUP model, respectively, the covariance being set to unity under both models. Hence, the fully conditional posterior density of <em>y<sub>i</sub></em> is a truncated normal distribution (truncated at points <em>t<sub>k</sub></em><sub>−1</sub> and <em>t<sub>k</sub></em>) with a density function (for simplicity, the ⋆ denotes the data and all other parameters)<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-14.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>where <em>ϕ</em>(⋅) and Φ(⋅) denote the standard normal density and cumulative distribution functions, respectively, while <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-15.gif"/></noscript></span></span>(<em>y<sub>i</sub></em>) is the linear predictor of the model (1) or (2).</p><p id="p-19">Following <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Sorensen <em>et al.</em> (1995)</a> the prior for the <em>K</em> − 2 unknown thresholds <strong>t</strong><sup>⋆</sup> = (<em>t</em><sub>2</sub>, …, <em>t<sub>K</sub></em><sub>−1</sub>) has been given as order statistics from an Uniform(0, <em>t<sub>max</sub></em>) distribution,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-16.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>The fully conditional posterior density for a <em>t<sub>k</sub></em> is proportional to the product of the prior and the likelihood of the observed ordinal phenotype <strong>w</strong>. Note, that the threshold values <strong>t</strong><sup>⋆</sup> appear in the prior density (7) only at the definition of the support of the distribution. As the terms not including the parameter are discarded as constants from the fully conditional posterior, the support definition is all that passes from the prior to the posterior. Therefore, the fully conditional posterior density for a <em>t<sub>k</sub></em> is given by the likelihood of the observed ordinal phenotype <strong>w</strong>, within the set of values determined by the prior density of <strong>t</strong><sup>⋆</sup>,<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-17.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>for 0 &lt; <em>t</em><sub>2</sub> &lt; … <em>t<sub>K</sub></em><sub>−1</sub> &lt; <em>t<sub>max</sub></em> and 0 otherwise. If (8) is seen as a function of <em>t<sub>k</sub></em>, it can be seen that the value of <em>t<sub>k</sub></em> must be larger that all of the <em>y<sub>i</sub></em>|<em>w<sub>i</sub></em> = <em>k</em> and smaller than all of the <em>y<sub>i</sub></em>|<em>w<sub>i</sub></em> = <em>k</em> + 1. Hence, as a function of <em>t<sub>k</sub></em>, (8) leads to the uniform density<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-18.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span>For a Gaussian phenotype with censored observations we define an additional binary variable <em>ω<sub>i</sub></em> = 1 if the <em>i</em>th observation is censored and <em>ω<sub>i</sub></em> = 0 if not. As the threshold model assumes an unity variance for the latent Gaussian response, the observed phenotype must be standardized accordingly. This is done by regarding the available observations as a sample from a truncated normal density and using the connection between the quantiles and the standard deviation of a Gaussian density (<em>e.g.</em>, 25% of the observations are ≤ <em>μ</em> − 0.67<em>σ</em>, or 15.73% are ≤ <em>μ</em> − <em>σ</em>). Now, if <em>ω<sub>i</sub></em> = 0 the standardized Gaussian phenotype is used directly, and if <em>ω<sub>i</sub></em> = 1 the underlying uncensored response is computed as previously. The latent variable parametrization of the binary phenotype is similar to a generalized linear model with the probit link function (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert and Chib 1993</a>), whereas the parametrization of the censored phenotype corresponds to a generalized linear model with the tobit link function (see <em>e.g.</em>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tobin 1958</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Sorensen <em>et al.</em> 1998</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Iwata <em>et al.</em> 2009</a>).</p><p id="p-20">The model parameters are estimated by the GEM (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Neal and Hinton 1999</a>) algorithm, which is presented in the Appendix A2. The algorithm finds a <em>maximum a posteriori</em> point estimate for each of the parameters and latent variables by repeatedly updating the parameters one at the time to their conditional expectations (see <a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>). Due to the conjugate or otherwise suitable prior densities chosen, the fully conditional posterior densities for the parameters and the latent Gaussian response are known probability density functions. This guarantees an easy derivation of the GEM-algorithm; as the expected values of the known densities are automatically available, we do not need to find the fully conditional posterior expectations by integration. In addition, if preferred it would be trivial to implement an MCMC Gibbs sampler to sample from these densities. The fully conditional posterior densities for the latent Gaussian response and for the unknown thresholds are given in the aforementioned models (6) and (9), whereas the fully conditional posterior densities for the parameters of the linear Gaussian models can be easily extracted from the joint posterior densities of the models (3) and (4). The derivations of the fully conditional posterior densities of the multilocus association model are presented in the Appendix A1.</p><p id="p-21">The fully conditional posterior densities for the multilocus association model (1) parameters are as follows. The fully conditional posterior density for a marker effect <em>β<sub>j</sub></em> is normal<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-19.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>where the residual variance <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-20.gif"/></noscript></span></span> unless the actual Gaussian phenotype is observed. The fully conditional posterior density for the inverse of a marker variance is an inverse-Gaussian (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chhikara and Folks 1989</a>)<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-21.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span>The fully conditional posterior density for the LASSO parameter <em>λ</em><sup>2</sup> is a gamma density<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-22.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span>The fully conditional posterior density for the population intercept equals a normal density<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-23.gif"/></noscript></span> <span class="disp-formula-label">(13)</span></span>where again the residual variance <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-24.gif"/></noscript></span></span> unless the actual Gaussian phenotype is observed. Finally, when estimated, the fully conditional posterior density of the residual variance is an inverse-<em>χ</em><sup>2</sup><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-25.gif"/></noscript></span> <span class="disp-formula-label">(14)</span></span>The fully conditional posterior densities for the Bayesian G-BLUP (2) parameters are the following. The fully conditional posterior density for the additive genetic values is a multivariate normal<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-26.gif"/></noscript></span> <span class="disp-formula-label">(15)</span></span>where the residual variance <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-27.gif"/></noscript></span></span> is again one if the actual Gaussian phenotype is not observed. The fully conditional posterior density for the additive genetic variance is an inverse-<em>χ</em><sup>2</sup> density<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-28.gif"/></noscript></span> <span class="disp-formula-label">(16)</span></span>where the capital <em>N</em> denotes the total number of individuals in the learning and test sets. The fully conditional posterior density for the population intercept is normal<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-29.gif"/></noscript></span> <span class="disp-formula-label">(17)</span></span>where the residual variance <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-30.gif"/></noscript></span></span> unless the actual Gaussian phenotype is observed. When estimated, the fully conditional posterior density of the residual variance is an inverse-<em>χ</em><sup>2</sup></p><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-31.gif"/></noscript></span> <span class="disp-formula-label">(18)</span></span><p id="p-22">The generalized expectation maximization algorithms presented in the Appendix A2 work by updating the parameters to the expected values of the aforementioned fully conditional posterior densities. In our method, as in nearly all Bayesian approaches, the user has to provide some, usually data specific, parameter values for the hyperprior densities at the very bottom of the model hierarchy. In the multilocus association model the hyperprior parameters for the LASSO parameter <em>λ</em><sup>2</sup> ∼ Gamma(<em>κ</em>, <em>ξ</em>) have to be given by the user, whereas in the Bayesian G-BLUP this role falls to the hyperprior parameters of the additive genetic variance component <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-32.gif"/></noscript></span></span>. The selection of the data specific hyperprior parameters is called tuning of the algorithm. The tuning is the easier to perform the fewer parameters there are to be tuned. Because the number of markers (<em>p</em>) is very large, the impact of <em>κ</em> into the fully conditional posterior expectation of the LASSO parameter <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-33.gif"/></noscript></span></span>, derived from the fully conditional posterior density in (12), is obviously negligible. As the only information the GEM algorithm uses in the update process is the fully conditional expectation, we shall simplify the tuning by setting the value of <em>κ</em> to a constant value <em>κ</em> = 1. Thereby, the rate parameter <em>ξ</em> is the only entity in the model to which the user has to provide a data specific value. Accordingly, under the Bayesian G-BLUP the degrees of freedom <em>ν</em> of the inverse-<em>χ</em><sup>2</sup> density do not have a substantial contribution to the fully conditional posterior expectation of the additional genetic variance <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-34.gif"/></noscript></span></span>, and we therefore set permanently <em>ν</em> = 2, while the scale parameter <em>τ</em><sup>2</sup> may need data specific tuning.</p></div><div class="section other1" id="sec-2"><h2 class="">Example analyses</h2><p id="p-23">In our example analyses, we have considered the predictive performance of the Bayesian multilocus association model and the Bayesian G-BLUP <em>per se</em> and with the three different latent variable layers, with two different data sets.</p><p id="p-24">The first of the data sets consists of a simulated data introduced in the XII QTL-MAS Workshop 2008 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lund <em>et al.</em> 2009</a>). The data set can be downloaded from the workshop homepage at <a href="http://www.computationalgenetics.se/QTLMAS08/QTLMAS/DATA.html">http://www.computationalgenetics.se/QTLMAS08/QTLMAS/DATA.html</a>. There are 5865 individuals from seven generations of half sib families with information on 6000 biallelic SNP loci, the loci are evenly distributed over six chromosomes of length 100 cM each (see <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lund <em>et al.</em> 2009</a> for details). Since SNPs with minor allele frequency &lt;0.05 within the learning set were discarded, the actual number of markers in the analysis is 5726. The first four generations of the data, 4665 individuals, have both marker information and a phenotypic record, and function as a learning set, whereas the generations five to seven, comprising 1200 individuals, are treated as a prediction set. There are 48 simulated quantitative trait loci (QTL) in the data set, with allele substitution effects drawn from a Gamma(0.42, 1.85) distribution (with shape and rate parametrization). The cumulative effect of the simulated QTL equals the genetic value of the individuals, while the phenotypes of the individuals have been obtained as the sum of the individuals’ genetic value and a random residual drawn from a normal distribution with mean zero and a variance set to produce heritability value 0.3 (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lund <em>et al.</em> 2009</a>). As in our previous works, we have generated 100 replicates of the data set by resampling the residuals from a normal density N(0, <em>var</em>(<em>TBV</em>)(1/<em>h</em><sup>2</sup> − 1)), where <em>var</em>(<em>TBV</em>) denotes the observed variance of the genetic values and the heritability <em>h</em><sup>2</sup> equals 0.3 (<a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">b</a>). After this each of the generated phenotype sets was scaled to have zero mean and unity variance. The advantage of using a simulated data set in the example analysis is the availability of the true genetic values of the individuals, enabling us to determine the accuracy of the estimates by a direct comparison of the simulated and estimated genetic values.</p><p id="p-25">The second data set, described in detail by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cleveland <em>et al.</em> (2012)</a>, is a real pig (<em>Sus scrofa</em>) data, provided by the Genetics Society of America to be used for benchmarking of genomic selection methods. The pig dataset consists of phenotypic records of 3184 individuals for a quantitative trait (standardized to zero mean and unity variance) with predetermined heritability 0.62, and genotypic records for 60k biallelic SNP markers (45,317 with minimum allele frequency over 0.05 actually included in the analysis). Contrary to the simulated data set, there are neither true genetic values of the individuals nor true effects of the QTL available, and hence we estimate the accuracy of the predicted genomic breeding values by dividing the correlation between the estimates and the original Gaussian phenotypic values by the square root of the predetermined heritability of the trait (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Legarra <em>et al.</em> 2008</a>). Since the data does not consist a separate validation population we compute the result statistics using cross-validation, where the 3184 individuals are randomly partitioned into 10 subsets (10-fold cross-validation) of 318 or 319 individuals. At each round 9 of the sets are treated as a learning set and the remaining one as the prediction set.</p><p id="p-26">The binary, ordinal, and censored phenotypes of the data sets were constructed as follows. We tested two binary phenotypes with success probabilities 50% and 80%, two ordinal phenotypes with four classes, and three right censored phenotypes consisting of 20%, 50%, or 80% of censored observations. The proportions of observations belonging into each class of the four-class phenotype were either even 20:30:30:20% of observations in each class, or highly unbalanced with 70% belonging to the first class and 10% in the subsequent three classes. The value of the censored observations was set to equal the largest of the noncensored values, leading to a spiked Gaussian phenotype (see <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Broman 2003</a>). The binary and the evenly distributed ordinal data sets are generated in preparation for an easy ascertainment of the extra power acquired by using the category information compared to the dichotomized phenotype. The binary phenotype with 80% success probability simply sets the first category of the ordinal phenotype as a failure and the subsequent three classes as a success, while the binary response with 50% success probability sets the first and second category as a failure and the third and fourth as a success. The same holds true for the censored data, as the threshold values are set to correspond the thresholds of the binary phenotype: the 20% and 80% censored data can be compared with the binary data with 80% success rate, as the proportions of the observations belonging to the classes is 20:80, and similarly the 50% censored data are equivalent to the 50% or 50:50 binary data. All threshold values were determined as standard normal distribution function parameters leading to the desired threshold value, <em>e.g.</em>, a threshold at 0.84, leading to 20% success probability, since Φ(0.84) = 0.8.</p><p id="p-27">The multilocus models are not able to handle an unlimited number of loci with respect to the sample size. <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hoti and Sillanpää (2006)</a> have proposed an upper limit of 10 times more loci than individuals, but it seems that in practice a smaller number of loci might be optimal (<em>e.g.</em>, real data analysis in <a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää 2012a</a>). Furthermore, to our experience, the best results are necessarily not acquired by using as many markers as the model can possible handle, but with a significantly smaller marker set (results not shown). In the QTL-MAS data, the proportion of markers to individuals is almost one-to-one, and no extra measures are needed, but with the pig data the multilocus association model becomes too oversaturated to function properly. Therefore, with the real data, in the beginning of each cross-validation round the number of SNPs is first reduced from 45,317 to 10,000 by the sure independence screening method (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fan and Lv 2008</a>). The method works by ranking the markers with respect to their marginal correlation with the phenotype within the current learning set, and selecting the 10,000 best ranking markers to the multilocus association model. The marginal correlation is computed as the Pearson’s product-moment coefficient, by using the same phenotypic records (binary, ordinal, censored or Gaussian) as is used in the actual multilocus association model, except in the case of the 80% censored Gaussian phenotype, where it proved better to dichotomize the phenotype by setting the uncensored data into zero. In <a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">Kärkkäinen and Sillanpää (2012a)</a> we performed the preselection in advance and used the same set of markers in all cross-validation rounds, whereas here the preselection is integrated into the cross-validation procedure. The correlation produced by the former approach appeared to be slightly overestimated. An advantage of the G-BLUP over the multilocus association model is that no preselection of the markers is needed. Some authors have found out that preselection of the markers might have a positive impact also to G-BLUP (see <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Resende <em>et al.</em> 2012</a>), but we did not observe such a behavior (results not shown).</p><p id="p-28">The hyperprior parameters for the example analyses are selected to produce best accuracy. The tuning is performed by testing different parameter values and choosing the one resulting the best correlation between the estimated and the true genetic values. In practice, we simply select two arbitrary values for the parameter, observe the correlation acquired under these values, and proceed to search for an optimal value to the direction pointed by the better performing one. This step could be automatized, but so far we have performed it manually. As we have used both learning and prediction sets in the parameter tuning, the obtained accuracies must be considered as best-case scenarios. Under the multilocus association model (1), we give the LASSO parameter <em>λ</em><sup>2</sup> a Gamma(1, <em>ξ</em>) hyperprior and tune the rate (<em>ξ</em>) of the gamma density into a data specific value, while under the G-BLUP model (2) the scale hyperparameter <em>τ</em><sup>2</sup> for the genetic variance is tuned. The values selected for <em>ξ</em> and <em>τ</em><sup>2</sup> and some of the proposed values, along with the corresponding accuracy estimates under the multilocus association model and the Bayesian G-BLUP, are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>, respectively.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463293/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463293/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Hyperprior selection for the Bayesian LASSO</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463297/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463297/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Prior selection for the Bayesian G-BLUP</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Comparing the relative performance of the multilocus association model and the Bayesian G-BLUP with the two data sets (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) clearly shows that the multilocus association model is superior when the trait is controlled by a moderate number of genes (QTL-MAS data), whereas the G-BLUP is a reasonable choice when the trait is either truly polygenic or there is strong linkage disequilibrium present (the pig data). With the 100 Gaussian QTL-MAS data replicates, the multilocus association model produces an average correlation 0.89 whereas the G-BLUP produces an average correlation 0.80. Consistently, with all of the binary, ordinal, and censored Gaussian QTL-MAS data sets, there is an approximately 10-point difference in favor of the multilocus association model in the average correlation. Regarding the pig data, the G-BLUP has the advantage over the multilocus association model: the average correlation in the 10 cross-validation sets is three points higher with the fully observed Gaussian phenotype and two to four points higher with most of the other phenotypes. The advantage of the G-BLUP is even more significant with the 80% success rate binary phenotype and the 80% censored phenotype, the G-BLUP being on average six points more accurate. In this case, however, the culprit is not only the multilocus association model itself but also the sure independence screening used beforehand to reduce the number of markers: if the marginal correlation was computed by using the Gaussian phenotype instead of the binary, the final average correlation would be more consistent 0.51 instead of the now observed 0.49 (data not shown).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463140/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463140/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Model accuracy</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">As the binary and the evenly distributed ordinal data sets are related to each other it is easy to ascertain the extra power acquired by using the category information compared with the dichotomized phenotype. <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> shows a significantly improved accuracy if the additional categories are taken into account: with the ordinal data the mean correlation is 6–10 points or 7–20% higher than with the 80% success rate binary data, and 3–4 points or 5–7% higher than with the 50% success rate binary data. The percentage advantage is greater in situations in which the power of the analysis is lower. As expected, the accuracy is lower with the 70:10:10:10% ordinal phenotype than with the evenly distributed ordinal phenotype, the difference being 2–4 points with both data sets under both models. The additional accuracy gained by using the correct model (<em>i.e.</em>, the threshold model) for the binary and ordinal phenotypes, instead of using the linear Gaussian model directly, was minor. The threshold model was a trifle of more accurate in some cases (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). The correlation obtained with the threshold model was one point greater in 7 cases of 16, more often with the unevenly distributed responses (binary 80% and ordinal 70:10:10:10) than with the evenly distributed (binary 50% and ordinal 20:30:30:20); with the unevenly distributed phenotypes there was five cases in which a modicum of extra accuracy was gained with the threshold model, whereas with the evenly distributed there was only two. The extra accuracy was also observed more often with the pig data than with the QTL-MAS data (five and two cases, respectively), and with the Bayesian G-BLUP than with the Bayesian LASSO (also five and two cases, respectively).</p><p id="p-34">The censored data sets consist of a continuous normally distributed phenotype with 20%, 50%, or 80% right censored observations, set to equal the maximum of the non-censored observations. The non-censored observations clearly contain extra information compared to the corresponding binary data (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The correlations acquired with the data sets with 20% censored observations were in all cases considerably, 6–11 points or 6–22%, higher than with the corresponding 20:80 binary data sets. With the 50% censored data sets the difference is 1–3 points, or 1–5%, compared to the 50:50 binary data. Even the data sets with 80% censored observations may be slightly more informative than the 20:80 binary data, the correlation being one point in favor of the censored phenotype with the QTL-MAS data under the threshold LASSO and threshold G-BLUP. The 20% censorship weakens the accuracy only slightly compared to the fully observed Gaussian phenotype: no more than one point if the threshold model is used, and 1–2 points if the Gaussian model is used directly for the censored phenotype. Contrary to the ordinal-phenotype-case, with a censored phenotype the threshold model is clearly more accurate than the Gaussian model (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). The difference between the models with the data sets with 20% censored phenotypes is tiny (one point in three cases out of four), but increases when the censoring grows stronger. With 50% censored observations the difference is 1–3 points or 1–5%, and with 80% censored observations 3–7 points, or 4–17%.</p><p id="p-35">The steps of the GEM algorithm are repeated until convergence. The algorithm is considered to be converged when the correlation between the estimated breeding values of two consecutive iterations is greater than 1−10<sup>−6</sup>. The convergence is confirmed visually by examining the behavior of parameter values during the iterations and verifying that all of the parameters have reached a constant level; this is also how the suitable value for the convergence rule has been originally ascertained. The required number of iterations is usually between 40 and 80 under the multilocus association model, and around 10 under the G-BLUP. So far we have not encountered problems in the convergence, given that appropriate hyper(prior) parameter values have been selected, and that the number of markers with respect to the sample size in the multilocus association model has not been too large. Depending on the data and the model variate, the computation time is around 15–50 sec with a 64-bit Windows 7 desktop computer with 3.50 GHz Intel(i7) CPU and 16.0 GB RAM.</p></div><div class="section discussion" id="sec-3"><h2 class="">Discussion</h2><p id="p-36">Our example analyses show that using the extra classes and the uncensored observations present in an ordered categorical and a censored Gaussian data set, instead of dichotomizing the data into case-control observations, increases the accuracy of genomic breeding values predicted by Bayesian multilocus association models or by Bayesian G-BLUP. The amount of extra information of an ordinal data depends on the number of the classes and the distribution of the observations into the classes, higher number and more even distribution corresponding to higher information content. With a mildly to moderately (20–50% of observations) censored Gaussian data, the increase of the accuracy is substantial compared to binary data, but even if 80% of the observations are censored the remaining observations seem to possess some extra information.</p><p id="p-37">Our results indicate that only a minor benefit is gained by using the correct threshold model compared with using the linear Gaussian model directly with a binary or ordinal data. These results are in concordance with the observations by <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> (2013)</a> under BayesB (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> 2001</a>) and BayesC<em>π</em> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Habier <em>et al.</em> 2011</a>), and with the early observations of <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Meijering and Gianola (1985)</a> for BLUP. However, for the same data set, under BayesA <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> (2013)</a> noted a substantial increase in the accuracy when the threshold model was used. Also, <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Meijering and Gianola (1985)</a> observed that the threshold model was more reliable than BLUP if the number of fixed effects required in the mixed model was high. Even though our results do not confirm the practical superiority of the correct threshold model over the linear Gaussian model, we urge caution when applying a Gaussian model directly for an ordinal data. Some data sets may be less well-behaving than the ones we have studied and, as proven by <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Wang <em>et al.</em> (2013)</a>, different linear models may be less robust to the incompatible data.</p><p id="p-38">The example analyses support the observation of <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Sorensen <em>et al.</em> (1998)</a> that in the case of a censored Gaussian data the threshold model behaves better than the linear Gaussian model used directly. The benefit of the correct threshold model increases as the proportion of the censored observations increase. With 20% censored observations, the threshold model is slightly more accurate, whereas with 50% censored observations the accuracy gain is substantial. On the basis of our results, a heavily censored (80%) Gaussian data should not be analyzed with a linear Gaussian model, or, if analyzed, it should be dichotomized into a binary case-control data.</p><p id="p-39">The sure independence screening (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fan and Lv 2008</a>) works very well for such a strikingly simple method. It works so well probably because all it needs to do is to let all of the important markers pass to the next step whereas, because the final variable regularization is performed by the multilocus association model, it does not matter if unimportant ones are also selected. The optimal number of SNPs selected into the multilocus association model is data specific as it depends on the number of individuals in the learning set, and probably also on the genetic architecture of the trait and the LD structure. Additionally, multilocus association models with different shrinkage or variable selection mechanisms may be able to cope with different amount of oversaturation. The number of markers selected to the multilocus association model can be tuned into an optimal value similarly to the prior parameters. The model performance seems to be reasonably robust to the number of markers: in the pig data 10,000–20,000 markers produced almost identical accuracies with all of the response types. However, even though the sure independence screening seems to be a decent method indeed, and we have contented ourselves with using it for the marker preselection for the time being, there probably is room for improvement in this respect.</p><p id="p-40">The difference in computation time between MCMC and (G)EM-algorithms is massive. Some authors have compared the speed difference between an MCMC and a MAP-estimation algorithm, for example, the fast BayesB implementation of <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> (2009)</a> took 2–5 min to converge, whereas the MCMC-based BayesB required 47 hr. Using the same QTL-MAS data we have used, <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Shepherd <em>et al.</em> (2010)</a> reported a computer time of few minutes for a MAP-estimation algorithm and 2 d for an MCMC algorithm (with a 2-GHz computer). Accordingly, with the same QTL-MAS data, the frequentist LASSO-LARS implementation of <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Usai <em>et al.</em> (2009)</a> took more than 7 hr to converge. The enormity of the time difference can be illustrated by extrapolating the computer times reported by <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Shepherd <em>et al.</em> (2010)</a> into the analysis of our 100 replicated QTL-MAS data sets: although the 100 analyses take 20 min with our GEM algorithm, with an MCMC algorithm the computation time would be stunning 600 hr, or 25 d. To get the best possible results, it seldom is sufficient to run an algorithm once, for instance, due to the tuning of the prior parameters and sensitivity analysis. The extremely short time requirement facilitates the adjusting for optimal performance, not to mention the usage of computer intensive techniques such as cross-validation and empirical threshold determination by phenotype permutation. Fast estimation is also extremely useful in simulation studies of entire breeding programs (see, <em>e.g.</em>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pedersen <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Axelsson <em>et al.</em> 2013</a>).</p><p id="p-41">The Bayesian threshold approach for binary, ordinal, and censored data enables the usage of a variety of different linear Gaussian model types—here we have demonstrated the method with Bayesian LASSO multilocus association model and Bayesian G-BLUP; however, depending on the genetic architecture and LD structure of the data, other variable selection or regularization methods than LASSO may be preferred. Whether the additional threshold layer actually increases the accuracy of the genomic breeding value estimates is questionable with a binary or an ordinal data but less so with a censored Gaussian data. To our experience it seems that especially with a heavily censored Gaussian data the threshold model should be used, but there is no harm in using it also with the binary and the ordinal phenotypes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">Supported by research grant from the Academy of Finland and Tekes funding for public research ”Individualized Connected Health.”</p></div><div class="section app" id="app-1"><h2 class="">Appendix A1</h2><div class="section" id="sec-4"><h3>Derivations of the fully conditional posterior densities</h3><p id="p-42">The joint posterior distribution of the unknown parameters, given the data, is proportional to the product of the joint prior and the likelihood. As the model parameters are considered <em>a priori</em> conditionally independent, the joint posterior density of the parameter vector under the multilocus association model (1) becomes<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/inline-graphic-3.gif"/></noscript></span><span class="disp-formula-label">(A1)</span></span>The fully conditional posteriors of the parameters can be easily determined by selecting the terms including the parameter in question from the joint posterior. (For simplicity: * = ”the data, and the parameters except the one in question”)</p><p id="p-43">As the prior density for the population intercept <em>β</em><sub>0</sub> is proportional to one, the fully conditional posterior distribution is proportional to the likelihood,<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-35.gif"/></noscript></span> <span class="disp-formula-label">(A2)</span></span>Note that we select from the joint posterior (A1) only the terms including <em>β</em><sub>0</sub>. The right hand side of (A2) is a product of <em>n</em> kernels of normal probability density functions, with a common variance <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-36.gif"/></noscript></span></span>, and means <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-37.gif"/></noscript></span></span>, <em>i</em> = 1, …, <em>n</em>. The set of Gaussian functions is closed under multiplication, <em>i.e.</em> the product of normal densities is also a normal density, with the mean and the variance of the product density given by<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-38.gif"/></noscript></span></span>respectively. Hence, since in this case variance is same for all of the factors, the product density reduces to the sum of the means of the individual distributions, divided by <em>n</em>, while the variance of the product density is given simply by the variance of the individual distributions divided by <em>n</em>, leading to the fully conditional posterior density function given in (13).</p><p id="p-44">For the fully conditional posterior distribution of the regression a coefficient <em>β<sub>j</sub></em> we select from (A1) the terms including <em>β<sub>j</sub></em>, located at the likelihood and at the prior <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-39.gif"/></noscript></span></span>, and get<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-40.gif"/></noscript></span> <span class="disp-formula-label">(A3)</span></span>Here we have a product of two types of kernels of normal densities. The latter part of (A3) comes from the prior, it has mean 0 and variance <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-41.gif"/></noscript></span></span>. The former part represents the likelihood—regarding <em>β<sub>j</sub></em> there are <em>n</em> kernels of normal distributions with means<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-42.gif"/></noscript></span></span>and variances <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-43.gif"/></noscript></span></span>, <em>i</em> = 1, …, <em>n</em>. As a product we get a normal distribution with a mean<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-44.gif"/></noscript></span></span>and variance<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-45.gif"/></noscript></span></span>which corresponds to the function in (10) when expanded by <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-46.gif"/></noscript></span></span>.</p><p id="p-45">The residual variance <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-47.gif"/></noscript></span></span> appears in the joint posterior density (A1) in the likelihood and in the prior density <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-48.gif"/></noscript></span></span>, leading to<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-49.gif"/></noscript></span> <span class="disp-formula-label">(A4)</span></span>Regarding <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-50.gif"/></noscript></span></span>, this is an unnormalized probability density function of an inverse <em>χ</em><sup>2</sup>-distribution (14), with <em>n</em> degrees of freedom and scale parameter equal to</p><span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-51.gif"/></noscript></span></span><p id="p-46">The fully conditional posterior density for the variance of the marker effects <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-52.gif"/></noscript></span></span> (11) is proportional to the product of the conditional prior for the marker effect <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-53.gif"/></noscript></span></span> and the exponential prior of the effect variance <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-54.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-55.gif"/></noscript></span></span>for all <em>j</em> = 1, …, <em>p</em>. Since exponential density is not conjugate to normal density, we need to consider the inverse of the marker variance<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-56.gif"/></noscript></span></span>where the last term is the Jacobian of the transformation <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-57.gif"/></noscript></span></span>. By rearranging the terms, we get<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-58.gif"/></noscript></span></span>and completing the numerator of the exponent into square<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-59.gif"/></noscript></span></span>As <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-60.gif"/></noscript></span></span> is canceled out from the last term inside the exponent, the term becomes constant and is left out. After that the exponent is expanded by <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-61.gif"/></noscript></span></span> and we get<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-62.gif"/></noscript></span> <span class="disp-formula-label">(A5)</span></span>This is an inverse-Gaussian probability density function (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chhikara and Folks 1989</a>) with mean <em>μ</em>′ and shape <em>λ</em>′<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-63.gif"/></noscript></span></span>the parametrization of the inverse-Gaussian density being</p><span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-64.gif"/></noscript></span></span><p id="p-47">The LASSO parameter <em>λ</em><sup>2</sup> occurs in the joint posterior (A1) in the exponential priors for the effect variances <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-65.gif"/></noscript></span></span> and in the gamma prior for the LASSO parameter.<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-66.gif"/></noscript></span> <span class="disp-formula-label">(A6)</span></span>that is an unnormalized gamma probability density function with shape <em>κ</em> + <em>p</em> and rate<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-67.gif"/></noscript></span></span>corresponding to (12).</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix A2</h2><div class="section" id="sec-5"><h3>The GEM algorithms</h3><p id="p-48">Below we present the GEM algorithms for <em>maximum a posteriori</em> parameter estimation under the threshold model combined with the Bayesian LASSO and the Bayesian G-BLUP. In both algorithms, the steps are repeated until convergence. Matlab codes of the algorithms are included as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007096/-/DC1/FileS1.zip">Supporting Information, File S1</a>.</p></div><div class="section" id="sec-6"><h3>GEM algorithm for the multilocus association model</h3><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-49">Set initial values for parameter vectors. We use zeros for <em>β</em><sub>0</sub>, <strong>β</strong>, and the latent variable <strong>y</strong>, and small positive values, namely 0.1, for the variances <strong>σ</strong><sup>2</sup> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-68.gif"/></noscript></span></span> (when estimated), and for the LASSO parameter <em>λ</em><sup>2</sup>. The initial value for the threshold vector <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-69.gif"/></noscript></span></span>.</p></li><li id="list-item-2"><p id="p-50">Update the values of the latent variable <strong>y</strong> by replacing the current values <em>y<sub>i</sub></em> with the expected values of the truncated normal distribution (6),<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-70.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-71.gif"/></noscript></span></span>, and <em>ϕ</em>(⋅) and Φ(⋅) denote the standard normal density function and distribution function, respectively. If the actual Gaussian phenotype <strong>y</strong> is fully available, this step naturally becomes obsolete.</p></li><li id="list-item-3"><p id="p-51">Update the K − 2 unknown thresholds to their conditional expectations,<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-72.gif"/></noscript></span></span>for all <em>k</em> = 2, …, K-1. This step is bypassed if i) the phenotype is Gaussian and fully observed, ii) the phenotype is binary, <em>i.e. K</em> = 2, or iii) the phenotype is left or right censored (no unknown thresholds).</p></li><li id="list-item-4"><p id="p-52">Maximize the posterior distributions of <em>β</em><sub>0</sub> and <em>β<sub>j</sub></em> (for all j) by substituting the fully conditional expectations for the current values of the parameters, one at the time.<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-73.gif"/></noscript></span></span>Note that the residual variance <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-74.gif"/></noscript></span></span> unless the Gaussian phenotype is fully observed.</p></li><li id="list-item-5"><p id="p-53">In case of a fully observed Gaussian phenotype, update the error variance <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-75.gif"/></noscript></span></span> into its conditional expectation<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-76.gif"/></noscript></span></span></p></li><li id="list-item-6"><p id="p-54">Update the effect variances <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-77.gif"/></noscript></span></span> (for all j) to their conditional expectations. The precision, or inverse of the variance parameter <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-78.gif"/></noscript></span></span>, has an inverse-Gaussian fully conditional posterior distribution leading to following fully conditional expectation for the effect variances<span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-79.gif"/></noscript></span></span></p></li><li id="list-item-7"><p id="p-55">Update the hyperparameter λ<sup>2</sup> into its conditional expectation<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-80.gif"/></noscript></span></span></p></li></ol></div><div class="section" id="sec-7"><h3>GEM algorithm for the Bayesian G-BLUP</h3><ol class="list-ord " id="list-2"><li id="list-item-8"><p id="p-56">Set initial values for parameter vectors; zeros for <em>β</em><sub>0</sub>, <strong>u</strong> and <strong>y</strong>, and 0.1 for <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-81.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-82.gif"/></noscript></span></span> (when estimated), <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-83.gif"/></noscript></span></span>.</p></li><li id="list-item-9"><p id="p-57">Update the latent variable <strong>y</strong> as previously, except that here <em>μ</em><sub>i</sub> = <em>β</em><sub>0</sub> + <em>u<sub>i</sub></em>.</p></li><li id="list-item-10"><p id="p-58">Update the thresholds as previously.</p></li><li id="list-item-11"><p id="p-59">Update the population intercept <em>β</em><sub>0</sub> into its fully conditional expectation<span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-84.gif"/></noscript></span></span></p></li><li id="list-item-12"><p id="p-60">Update the polygenic effects <strong>u</strong> by replacing the current values with the conditional expectations<span class="disp-formula" id="disp-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-85.gif"/></noscript></span></span>where <strong>G</strong><sup>−1</sup> denotes a generalized inverse of the realized relationship matrix. Note that the residual variance <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-86.gif"/></noscript></span></span> except in the fully observed Gaussian phenotype case.</p></li><li id="list-item-13"><p id="p-61">In case of a fully observed Gaussian phenotype, update the error variance <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-87.gif"/></noscript></span></span> into its fully conditional expectation<span class="disp-formula" id="disp-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-88.gif"/></noscript></span></span></p></li><li id="list-item-14"><p id="p-62">Replace the additive genetic variance component <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-89.gif"/></noscript></span></span> with its conditional expectation<span class="disp-formula" id="disp-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1511/embed/mml-math-90.gif"/></noscript></span></span>where and <em>N</em> is the total number of individuals in the learning and validation sets (<em>i.e.</em> the dimension of <strong>G</strong> is <em>N</em> × <em>N</em>), and <strong>G</strong><sup>−1</sup> is a generalized inverse of the realized relationship matrix.</p></li></ol></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007096/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007096/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-64"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-03-28"><span class="received-label">Received </span>March 28, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-06-24"><span class="accepted-label">Accepted </span>June 24, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Kärkkäinen and Sillanpää</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.1" data-doi="10.2307/2290350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chib</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Bayesian analysis of binary and polychotomous response data.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D88%26rft.spage%253D669%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LD68000034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.2" data-doi="10.3168/jds.2012-5409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fikse</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kargo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection using indicator traits to reduce the environmental impact of milk production.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D96%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2012-5409%26rft_id%253Dinfo%253Apmid%252F23141821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2012-5409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23141821&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312524300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping quantitative trait loci in the case of a spike in the phenotype distribution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D1169%26rft.epage%253D1175%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2Bin%2Bthe%2BCase%2Bof%2Ba%2BSpike%2Bin%2Bthe%2BPhenotype%2BDistribution%26rft_id%253Dinfo%253Apmid%252F12663553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2My8zLzExNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1511.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chhikara</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folks</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source"><em>The Inverse Gaussian distribution: Theory</em>, <em>Methodology</em>, <em>and Applications</em></span>. <span class="cit-publ-name">Marcel Dekker, Inc.</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-3.9.1511.5" data-doi="10.1534/g3.111.001453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A common dataset for genomic analysis of livestock populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DCleveland%26rft.auinit1%253DM.%2BA.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D429%26rft.epage%253D435%26rft.atitle%253DA%2BCommon%2BDataset%2Bfor%2BGenomic%2BAnalysis%2Bof%2BLivestock%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001453%26rft_id%253Dinfo%253Apmid%252F22540034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNC80MjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1511.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Analysis of Binary Data</span>. <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.7" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempster</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laird</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Maximum likelihood from incomplete data via the EM algorithm.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D39%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.9" data-doi="10.2527/af.2011-0027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eggen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The development and application of genomic selection as a new breeding paradigm.</span> <abbr class="cit-jnl-abbrev">Anim. Front.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BFront.%26rft.volume%253D2%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.2527%252Faf.2011-0027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/af.2011-0027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.10" data-doi="10.1111/j.1467-9868.2008.00674.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sure independence screening for ultrahigh dimensional feature space.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D70%26rft.spage%253D849%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2008.00674.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2008.00674.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.11" data-doi="10.1109/TPAMI.2003.1227989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Figueiredo</span>  <span class="cit-name-given-names">M. A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Adaptive sparseness for supervised learning.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Pattern Anal. Mach. Intell.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1150</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BPattern%2BAnal.%2BMach.%2BIntell.%26rft.volume%253D25%26rft.spage%253D1150%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTPAMI.2003.1227989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TPAMI.2003.1227989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.12" data-doi="10.1198/016214504000000458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Parameterization and Bayesian modeling.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D99%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214504000000458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214504000000458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000221572500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1511.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Bayesian Data Analysis</span>. <span class="cit-edition">Ed. 2.</span> <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.14" data-doi="10.2527/jas1982.5451079x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Theory and analysis of threshold characters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D1079%26rft.epage%253D1096%26rft.atitle%253DTheory%2Band%2BAnalysis%2Bof%2BThreshold%2BCharacters%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1982.5451079x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1982.5451079x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982NP93300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1511.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilks</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Markov Chain Monte Carlo in Practice</span>. <span class="cit-publ-name">Chapman &amp; Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.16" data-doi="10.1186/1297-9686-43-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGonzalez-Recio%26rft.auinit1%253DO.%26rft.volume%253D43%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DGenome-wide%2Bprediction%2Bof%2Bdiscrete%2Btraits%2Busing%2BBayesian%2Bregressions%2Band%2Bmachine%2Blearning.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-7%26rft_id%253Dinfo%253Apmid%252F21329522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329522&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.9.1511.17" data-doi="10.1186/1753-6561-3-s7-s63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Maturana</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelman</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detecting single-nucleotide polymorphism by single-nucleotide polymorphism interactions in rheumatoid arthritis using a two-step approach with machine learning and a Bayesian threshold least absolute shrinkage and selection operator (LASSO) model.</span> <span class="cit-source">BMC Proc.</span> <span class="cit-vol">3</span><span class="cit-supplement">(Suppl. 7)</span>: <span class="cit-fpage">S63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D3%26rft.spage%253DS63%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-3-s7-s63%26rft_id%253Dinfo%253Apmid%252F20018057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-3-s7-s63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20018057&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.18" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.19" data-doi="10.2307/2533257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weller</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic mapping of quantitative trait loci for traits with ordinal distributions.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1252</span>–<span class="cit-lpage">1263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D51%26rft.issue%253D4%26rft.spage%253D1252%26rft.epage%253D1263%26rft.atitle%253DGenetic%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bfor%2Btraits%2Bwith%2Bordinal%2Bdistributions.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533257%26rft_id%253Dinfo%253Apmid%252F8589221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8589221&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.20" data-doi="10.1534/genetics.110.114249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldmann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bayesian inference of genetic parameters based on conditional decompositions of multivariate normal distributions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114249%26rft_id%253Dinfo%253Apmid%252F20351218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">EM algorithm for Bayesian estimation of genomic breeding values.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DHayashi%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DEM%2Balgorithm%2Bfor%2BBayesian%2Bestimation%2Bof%2Bgenomic%2Bbreeding%2Bvalues.%26rft_id%253Dinfo%253Apmid%252F20092655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20092655&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.22" data-doi="10.1038/sj.hdy.6800817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoti</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bayesian mapping of genotype × expression interactions on quantitative and qualitative traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHoti%26rft.auinit1%253DF.%26rft.volume%253D97%26rft.issue%253D1%26rft.spage%253D4%26rft.epage%253D18%26rft.atitle%253DBayesian%2Bmapping%2Bof%2Bgenotype%2Bx%2Bexpression%2Binteractions%2Bin%2Bquantitative%2Band%2Bqualitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800817%26rft_id%253Dinfo%253Apmid%252F16670709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16670709&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238590200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.23" data-doi="10.1007/s00122-008-0945-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebana</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuoka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bayesian multilocus association mapping on ordinal and censored traits and its application to the analysis of genetic variation among <em>Oryza sativa L</em>. germplasms.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DIwata%26rft.auinit1%253DH.%26rft.volume%253D118%26rft.issue%253D5%26rft.spage%253D865%26rft.epage%253D880%26rft.atitle%253DBayesian%2Bmultilocus%2Bassociation%2Bmapping%2Bon%2Bordinal%2Band%2Bcensored%2Btraits%2Band%2Bits%2Bapplication%2Bto%2Bthe%2Banalysis%2Bof%2Bgenetic%2Bvariation%2Bamong%2BOryza%2Bsativa%2BL.%2Bgermplasms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0945-6%26rft_id%253Dinfo%253Apmid%252F19132337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0945-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19132337&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.24" data-doi="10.1534/genetics.112.139014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kärkkäinen</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Back to basics for Bayesian model building in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139014%26rft_id%253Dinfo%253Apmid%252F22554888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.25" data-doi="10.1111/j.1469-1809.2012.00729.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kärkkäinen</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Robustness of Bayesian multilocus association models to cryptic relatedness.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">510</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DKarkkainen%26rft.auinit1%253DH.%2BP.%26rft.volume%253D76%26rft.issue%253D6%26rft.spage%253D510%26rft.epage%253D523%26rft.atitle%253DRobustness%2Bof%2Bbayesian%2Bmultilocus%2Bassociation%2Bmodels%2Bto%2Bcryptic%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.2012.00729.x%26rft_id%253Dinfo%253Apmid%252F22971009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.2012.00729.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22971009&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.26" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granié</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.9.1511.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparison of analyses of the QTLMAS XII common dataset. I: Genomic selection.</span> <span class="cit-source">BMC Proc.</span> <span class="cit-vol">3</span>(<span class="cit-supplement">Suppl. 1</span>): <span class="cit-fpage">S1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D3%26rft.spage%253DS1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1511.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCullagh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelder</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Generalized Linear Models</span>. <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1511.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLachlan</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source">The EM Algorithm and Extensions</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.30" data-doi="10.1051/gse:19850107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meijering</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Linear <em>vs.</em> nonlinear methods of sire evaluation for categorical traits: a simulation study.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D17%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19850107%26rft_id%253Dinfo%253Apmid%252F22879189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19850107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22879189&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985ADL0600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.32" data-doi="10.1186/1297-9686-41-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A fast algorithm for BayesB type of prediction of genome-wide estimates of genetic value.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D41%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DA%2Bfast%2Balgorithm%2Bfor%2BBayesB%2Btype%2Bof%2Bprediction%2Bof%2Bgenome-wide%2Bestimates%2Bof%2Bgenetic%2Bvalue.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-2%26rft_id%253Dinfo%253Apmid%252F19284681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284681&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.33" data-doi="10.1534/genetics.110.119586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mutshinda</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Extended Bayesian LASSO for multiple quantitative trait loci mapping and unobserved phenotype prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1067</span>–<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119586%26rft_id%253Dinfo%253Apmid%252F20805559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni8zLzEwNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.34" data-doi="10.1093/aob/mcs109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakaya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Will genomic selection be a practical method for plant breeding?</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcs109%26rft_id%253Dinfo%253Apmid%252F22645117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMTAvNi8xMzAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1511.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neal</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A view of the EM algorithm that justifies incremental, sparse, and other variants</span>, pp. <span class="cit-fpage">355</span>–<span class="cit-lpage">368</span> in <span class="cit-source">Learning in Graphical Models</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">M. I.</span></span>. <span class="cit-publ-name">MIT Press</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.36" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Bayesian Lasso.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.37" data-doi="10.1016/j.livsci.2008.06.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ansari-Mahyari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">ADAM: a computer program to simulate selective breeding schemes for animals.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D121%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2008.06.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2008.06.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.38" data-doi="10.1038/nrg2865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconciling the analysis of IBD and IBS in complex trait studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D800%26rft.epage%253D805%26rft.atitle%253DReconciling%2Bthe%2Banalysis%2Bof%2BIBD%2Band%2BIBS%2Bin%2Bcomplex%2Btrait%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2865%26rft_id%253Dinfo%253Apmid%252F20877324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20877324&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.39" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard data set of loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.40" data-doi="10.1186/1471-2105-11-529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection and complex trait prediction using a fast EM algorithm applied to genomewide markers.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DShepherd%26rft.auinit1%253DR.%2BK.%26rft.volume%253D11%26rft.spage%253D529%26rft.epage%253D529%26rft.atitle%253DGenomic%2Bselection%2Band%2Bcomplex%2Btrait%2Bprediction%2Busing%2Ba%2Bfast%2BEM%2Balgorithm%2Bapplied%2Bto%2Bgenome-wide%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-529%26rft_id%253Dinfo%253Apmid%252F20969788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20969788&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.41" data-doi="10.1534/genetics.107.081299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoti</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping quantitative trait loci from a single-tail sample of the phenotype distribution including survival data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2361</span>–<span class="cit-lpage">2377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSillanpaa%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2361%26rft.epage%253D2377%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2BFrom%2Ba%2BSingle-Tail%2BSample%2Bof%2Bthe%2BPhenotype%2BDistribution%2BIncluding%2BSurvival%2BData%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081299%26rft_id%253Dinfo%253Apmid%252F18073434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.42" data-doi="10.1051/gse:19950303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsgaard</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayesian inference in threshold models using Gibbs sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D27%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19950303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19950303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RE69800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.43" data-doi="10.1080/09064719809362502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsgaard</span>  <span class="cit-name-given-names">I. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Bayesian mixed-effects model analysis of a censored normal distribution with animal breeding applications.</span> <abbr class="cit-jnl-abbrev">Acta Agric. Scand.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">222</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgric.%2BScand.%26rft.volume%253D48%26rft.spage%253D222%26rft_id%253Dinfo%253Adoi%252F10.1080%252F09064719809362502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/09064719809362502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.44" data-doi="10.1534/genetics.110.114280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibrahim</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomewide multiple-loci mapping in experimental crosses by iterative adaptive penalized regression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114280%26rft_id%253Dinfo%253Apmid%252F20157003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzEvMzQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.45" data-doi="10.1371/journal.pone.0049157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A fast EM algorithm for BayesA-like prediction of genomic breeding values.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e49157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSun%26rft.auinit1%253DX.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De49157%26rft.epage%253De49157%26rft.atitle%253DA%2BFast%2BEM%2BAlgorithm%2Bfor%2BBayesA-Like%2BPrediction%2Bof%2BGenomic%2BBreeding%2BValues.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049157%26rft_id%253Dinfo%253Apmid%252F23152868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23152868&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.46" data-doi="10.2307/1907382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobin</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Estimation of relationships for limited dependent variables.</span> <abbr class="cit-jnl-abbrev">Econometrica</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEconometrica%26rft.volume%253D26%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1907382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1907382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1958CAL6600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.47" data-doi="10.1017/S0016672309990334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usai</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">LASSO with cross-validation for genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DUsai%26rft.auinit1%253DM.%2BG.%26rft.volume%253D91%26rft.issue%253D6%26rft.spage%253D427%26rft.epage%253D436%26rft.atitle%253DLASSO%2Bwith%2Bcross-validation%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672309990334%26rft_id%253Dinfo%253Apmid%252F20122298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672309990334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20122298&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.48" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.49" data-doi="10.1038/hdy.2012.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.-X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bayesian methods for estimating GEBVs of threshold traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.65%26rft_id%253Dinfo%253Apmid%252F23149458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23149458&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating polygenic effects using markers of the entire genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D789%26rft.epage%253D801%26rft.atitle%253DEstimating%2BPolygenic%2BEffects%2BUsing%2BMarkers%2Bof%2Bthe%2BEntire%2BGenome%26rft_id%253Dinfo%253Apmid%252F12618414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.51" data-doi="10.1038/hdy.2009.180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An expectation–maximization algorithm for the Lasso estimation of quantitative trait locus effects.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D105%26rft.issue%253D5%26rft.spage%253D483%26rft.epage%253D494%26rft.atitle%253DAn%2Bexpectation-maximization%2Balgorithm%2Bfor%2Bthe%2BLasso%2Bestimation%2Bof%2Bquantitative%2Btrait%2Blocus%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.180%26rft_id%253Dinfo%253Apmid%252F20051978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051978&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283168600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.52" data-doi="10.1534/genetics.107.085589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian LASSO for quantitative trait loci mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1045</span>–<span class="cit-lpage">1055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085589%26rft_id%253Dinfo%253Apmid%252F18505874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzEwNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1511.53"><div class="cit-metadata unstructured">Yi, N., and S. Banerjee, 2009 Hierarchical generalized linear models for multiple quantitative trait locus mapping. Genetics 181: 1101–1113.</div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/9/1511.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/9/1493" title="The Developmental Transcriptome of the Mosquito Aedes aegypti, an Invasive Species and Major Arbovirus Vector" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/9/1525" title="Genetic Determinants of Cell Size at Birth and Their Impact on Cell Cycle Progression in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/9"><span>Volume 3 Issue 9, September 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/9.cover-source.jpg?itok=2NEcI52z" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1511" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JydVhP4WM94EVFsCFA0YD1IdQeismYWO326dIKnjj_I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/9/1511" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/9/1511" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/9/1511" class="active" data-icon-position="" data-hide-link-title="0">Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459533" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-e0rKA7omYwzkxLjITTgU_6ODBQBvhi1sxSLczWm52AY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/9/1511.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/9/1511" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_CS1DOsZgu0DuTwhkyz2q5_EL42OUF1iL887eOn3bB4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/9/1511" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/9/1511.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459533" id="node459533--2" data-pisa="ggg;3/9/1511" data-pisa-master="ggg;g3.113.007096" data-apath="/ggg/3/9/1511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hanni P. Kärkkäinen</span> and <span class="highwire-citation-author" data-delta="1">Mikko J. Sillanpää</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1511-1523;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007096 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459533--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hanni P. Kärkkäinen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agricultural Sciences, University of Helsinki, Helsinki FIN-00014</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hanni%2BP.%2BK%C3%A4rkk%C3%A4inen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4rkk%C3%A4inen%20HP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHanni%2BP.%2BK%25C3%25A4rkk%25C3%25A4inen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mikko J. Sillanpää </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Biocenter Oulu, University of Oulu, Oulu FIN-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%C3%A4%C3%A4%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%C3%A4%C3%A4%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25C3%25A4%25C3%25A4%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mjs@rolf.helsinki.fi" class="" data-icon-position="" data-hide-link-title="0">mjs@rolf.helsinki.fi</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/9/1511" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/9/1511&amp;title=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/9/1511&amp;title=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/9/1511&amp;title=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/9/1511&amp;text=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/9/1511&amp;title=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/9/1511&amp;t=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/9/1511&amp;title=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/9/1511&amp;title=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459533/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459533" id="node459533--3" data-pisa="ggg;3/9/1511" data-pisa-master="ggg;g3.113.007096" data-apath="/ggg/3/9/1511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast Genomic Predictions via Bayesian G-BLUP and Multilocus Models of Threshold Traits Including Censored Gaussian Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hanni P. Kärkkäinen</span> and <span class="highwire-citation-author" data-delta="1">Mikko J. Sillanpää</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1511-1523;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007096 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459533--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hanni P. Kärkkäinen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agricultural Sciences, University of Helsinki, Helsinki FIN-00014</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hanni%2BP.%2BK%C3%A4rkk%C3%A4inen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4rkk%C3%A4inen%20HP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHanni%2BP.%2BK%25C3%25A4rkk%25C3%25A4inen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mikko J. Sillanpää </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematical Sciences, University of Oulu, Oulu FIN-90014, Finland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology and Biocenter Oulu, University of Oulu, Oulu FIN-90014, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikko%2BJ.%2BSillanp%C3%A4%C3%A4%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sillanp%C3%A4%C3%A4%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMikko%2BJ.%2BSillanp%25C3%25A4%25C3%25A4%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mjs@rolf.helsinki.fi" class="" data-icon-position="" data-hide-link-title="0">mjs@rolf.helsinki.fi</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1511&amp;rft.epage=1523&amp;rft.atitle=Fast%20Genomic%20Predictions%20via%20Bayesian%20G-BLUP%20and%20Multilocus%20Models%20of%20Threshold%20Traits%20Including%20Censored%20Gaussian%20Data&amp;rft.volume=3&amp;rft.issue=9&amp;rft.date=2013-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=K%C3%A4rkk%C3%A4inen%2C+Hanni+P.&amp;rft.au=Sillanp%C3%A4%C3%A4%2C+Mikko+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459533/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459533/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459533/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459533/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459533/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459533/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459533/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459533/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459533/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459533/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459533/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459533/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/9/1511" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Example analyses</a></li><li class="even"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A1</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A2</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/9/1511.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/9/1511.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/9/1511.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/9/1511.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
