<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/9/1453" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/9/1453.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/9/1453.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/9/1453.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/9/1453.atom" />
<meta name="HW.pisa" content="ggg;3/9/1453" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.113.006429" />
<meta name="DC.Date" content="2013-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Lang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="DNA mismatch repair is a highly conserved DNA repair pathway. In humans, germline mutations in hMSH2 or hMLH1 , key components of mismatch repair, have been associated with Lynch syndrome, a leading cause of inherited cancer mortality. Current estimates of the mutation rate and the mutational spectra in mismatch repair defective cells are primarily limited to a small number of individual reporter loci. Here we use the yeast Saccharomyces cerevisiae to generate a genome-wide view of the rates, spectra, and distribution of mutation in the absence of mismatch repair. We performed mutation accumulation assays and next generation sequencing on 19 strains, including 16 [msh2][1] missense variants implicated in Lynch cancer syndrome. The mutation rate for DNA mismatch repair null strains was approximately 1 mutation per genome per generation, 225-fold greater than the wild-type rate. The mutations were distributed randomly throughout the genome, independent of replication timing. The mutation spectra included insertions/deletions at homopolymeric runs (87.7%) and at larger microsatellites (5.9%), as well as transitions (4.5%) and transversions (1.9%). Additionally, repeat regions with proximal repeats are more likely to be mutated. A bias toward deletions at homopolymers and insertions at (AT)n microsatellites suggests a different mechanism for mismatch generation at these sites. Interestingly, 5% of the single base pair substitutions might represent double-slippage events that occurred at the junction of immediately adjacent repeats, resulting in a shift in the repeat boundary. These data suggest a closer scrutiny of tumor suppressors with homopolymeric runs with proximal repeats as the potential drivers of oncogenesis in mismatch repair defective cells.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450" />
<meta name="DC.Contributor" content="Gregory I. Lang" />
<meta name="DC.Contributor" content="Lance Parsons" />
<meta name="DC.Contributor" content="Alison E. Gammie" />
<meta name="article:published_time" content="2013-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;DNA mismatch repair is a highly conserved DNA repair pathway. In humans, germline mutations in &lt;i&gt;hMSH2&lt;/i&gt; or &lt;i&gt;hMLH1&lt;/i&gt;, key components of mismatch repair, have been associated with Lynch syndrome, a leading cause of inherited cancer mortality. Current estimates of the mutation rate and the mutational spectra in mismatch repair defective cells are primarily limited to a small number of individual reporter loci. Here we use the yeast &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; to generate a genome-wide view of the rates, spectra, and distribution of mutation in the absence of mismatch repair. We performed mutation accumulation assays and next generation sequencing on 19 strains, including 16 &lt;i&gt;msh2&lt;/i&gt; missense variants implicated in Lynch cancer syndrome. The mutation rate for DNA mismatch repair null strains was approximately 1 mutation per genome per generation, 225-fold greater than the wild-type rate. The mutations were distributed randomly throughout the genome, independent of replication timing. The mutation spectra included insertions/deletions at homopolymeric runs (87.7%) and at larger microsatellites (5.9%), as well as transitions (4.5%) and transversions (1.9%). Additionally, repeat regions with proximal repeats are more likely to be mutated. A bias toward deletions at homopolymers and insertions at (AT)&lt;sub&gt;n&lt;/sub&gt; microsatellites suggests a different mechanism for mismatch generation at these sites. Interestingly, 5% of the single base pair substitutions might represent double-slippage events that occurred at the junction of immediately adjacent repeats, resulting in a shift in the repeat boundary. These data suggest a closer scrutiny of tumor suppressors with homopolymeric runs with proximal repeats as the potential drivers of oncogenesis in mismatch repair defective cells.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/09/01" />
<meta name="citation_mjid" content="ggg;3/9/1453" />
<meta name="citation_id" content="3/9/1453" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/9/1453" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/9/1453.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/9/1453.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/9/1453.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006429" />
<meta name="citation_pmid" content="23821616" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1453" />
<meta name="citation_lastpage" content="1465" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gregory I. Lang" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014" />
<meta name="citation_author" content="Lance Parsons" />
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014" />
<meta name="citation_author" content="Alison E. Gammie" />
<meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, New Jersey 08544-1014" />
<meta name="citation_author_email" content="agammie@princeton.edu" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=A. L. Abdulovic;citation_author=S. E. Hile;citation_author=T. A. Kunkel;citation_author=K. A. Eckert;citation_title=The in vitro fidelity of yeast DNA polymerase delta and polymerase epsilon holoenzymes during dinucleotide microsatellite DNA synthesis.;citation_pages=497-505;citation_volume=10;citation_year=2011;citation_pmid=21429821;citation_doi=10.1016/j.dnarep.2011.02.003" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Acharya;citation_author=T. Wilson;citation_author=S. Gradia;citation_author=M. F. Kane;citation_author=S. Guerrette;citation_title=hMSH2 forms specific mispair-binding complexes with hMSH3 and hMSH6;citation_pages=13629-13634;citation_volume=93;citation_year=1996;citation_issue=24;citation_pmid=8942985;citation_doi=10.1073/pnas.93.24.13629" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Agier;citation_author=G. Fischer;citation_title=The mutational profile of the yeast genome is shaped by replication.;citation_volume=29;citation_year=2012;citation_pmid=22114361;citation_doi=10.1093/molbev/msr280" />
<meta name="citation_reference" content="citation_journal_title=Journal of cellular physiology;citation_journal_abbrev=J Cell Physiol;citation_author=G. Aquilina;citation_author=M. Bignami;citation_title=Mismatch repair in correction of replication errors and processing of DNA damage.;citation_pages=145-154;citation_volume=187;citation_year=2001;citation_issue=2;citation_pmid=11267994;citation_doi=10.1002/jcp.1067" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Arlow;citation_author=K. Scott;citation_author=A. Wagenseller;citation_author=A. Gammie;citation_title=Proteasome inhibition rescues clinically significant unstable variants of the mismatch repair protein Msh2.;citation_volume=110;citation_year=2013;citation_pmid=23248292;citation_doi=10.1073/pnas.1215510110" />
<meta name="citation_reference" content="citation_title=Current Protocols in Molecular Biology;citation_author=F. M. Ausubel;citation_author=R. Brent;citation_author=R. E. Kingston;citation_author=D. D. Moore;citation_author=J. G. Scidman;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Baran;citation_author=A. Lapidot;citation_author=H. Manor;citation_title=Formation of DNA triplexes accounts for arrests of DNA synthesis at d(TC)n and d(GA)n tracts.;citation_pages=507-511;citation_volume=88;citation_year=1991;citation_issue=2;citation_pmid=1988950;citation_doi=10.1073/pnas.88.2.507" />
<meta name="citation_reference" content="citation_journal_title=EMBO Reports;citation_journal_abbrev=EMBO Reports;citation_author=K. Bebenek;citation_author=M. Garcia-Diaz;citation_author=M. C. Foley;citation_author=L. C. Pedersen;citation_author=T. Schlick;citation_title=Substrate-induced DNA strand misalignment during catalytic cycling by DNA polymerase lambda.;citation_pages=459-464;citation_volume=9;citation_year=2008;citation_issue=5;citation_pmid=18369368;citation_doi=10.1038/embor.2008.33" />
<meta name="citation_reference" content="citation_journal_title=Molecular biotechnology;citation_journal_abbrev=Mol Biotechnol;citation_author=A. Bhargava;citation_author=F. F. Fuentes;citation_title=Mutational dynamics of microsatellites.;citation_pages=250-266;citation_volume=44;citation_year=2010;citation_issue=3;citation_pmid=20012711;citation_doi=10.1007/s12033-009-9230-4" />
<meta name="citation_reference" content="citation_journal_title=The lancet oncology;citation_journal_abbrev=Lancet Oncol;citation_author=C. R. Boland;citation_title=Lynch syndrome: new tales from the crypt.;citation_pages=562-564;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22552010;citation_doi=10.1016/S1470-2045(12)70134-1" />
<meta name="citation_reference" content="citation_journal_title=Methods in Yeast Genetics: A Cold Spring Harbor Laboratory Course Manual;citation_author=D. Burke;citation_author=D. Dawson;citation_author=T. Stearns;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. M. Cherry;citation_author=C. Ball;citation_author=S. Weng;citation_author=G. Juvik;citation_author=R. Schmidt;citation_title=Genetic and physical maps of Saccharomyces cerevisiae.;citation_pages=67-73;citation_volume=387;citation_year=1997;citation_issue=6632 Suppl;citation_pmid=9169866;citation_doi=10.1038/387067a0" />
<meta name="citation_reference" content="citation_title=Advances in cancer research;citation_journal_abbrev=Adv Cancer Res;citation_author=C. Colas;citation_author=F. Coulet;citation_author=M. Svrcek;citation_author=A. Collura;citation_author=J. F. Flejou;citation_title=Lynch or not Lynch? Is that always a question?;citation_pages=121-166;citation_volume=Vol. 113;citation_year=2012;citation_pmid=22429854;citation_doi=10.1016/B978-0-12-394280-7.00004-X" />
<meta name="citation_reference" content="citation_journal_title=SÃ£o Paulo medical journal = Revista paulista de medicina;citation_journal_abbrev=Sao Paulo Med J;citation_author=F. C. C. da Silva;citation_author=M. D. Valentin;citation_author=F. D. Ferreira;citation_author=D. M. Carraro;citation_author=B. M. Rossi;citation_title=Mismatch repair genes in Lynch syndrome: a review.;citation_pages=46-51;citation_volume=127;citation_year=2009;citation_issue=1;citation_pmid=19466295;citation_doi=10.1590/S1516-31802009000100010" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Cancer;citation_journal_abbrev=Nat Rev Cancer;citation_author=A. de la Chapelle;citation_title=Genetic predisposition to colorectal cancer.;citation_pages=769-780;citation_volume=4;citation_year=2004;citation_issue=10;citation_pmid=15510158;citation_doi=10.1038/nrc1453" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Denver;citation_author=S. Feinberg;citation_author=S. Estes;citation_author=W. K. Thomas;citation_author=M. Lynch;citation_title=Mutation rates, spectra and hotspots in mismatch repair-deficient Caenorhabditis elegans.;citation_volume=170;citation_year=2005;citation_pmid=15716493;citation_doi=10.1534/genetics.104.038521" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. W. Drake;citation_title=A constant rate of spontaneous mutation in DNA-based microbes.;citation_pages=7160-7164;citation_volume=88;citation_year=1991;citation_issue=16;citation_pmid=1831267;citation_doi=10.1073/pnas.88.16.7160" />
<meta name="citation_reference" content="citation_journal_title=Human mutation;citation_journal_abbrev=Hum Mutat;citation_author=B. M. Dupuy;citation_author=M. Stenersen;citation_author=T. Egeland;citation_author=B. Olaisen;citation_title=Y-chromosomal microsatellite mutation rates: differences in mutation rate between and within loci.;citation_pages=117-124;citation_volume=23;citation_year=2004;citation_issue=2;citation_pmid=14722915;citation_doi=10.1002/humu.10294" />
<meta name="citation_reference" content="citation_journal_title=Molecular carcinogenesis;citation_journal_abbrev=Mol Carcinog;citation_author=K. A. Eckert;citation_author=S. E. Hile;citation_title=Every microsatellite is different: Intrinsic DNA features dictate mutagenesis of common microsatellites present in the human genome.;citation_pages=379-388;citation_volume=48;citation_year=2009;citation_issue=4;citation_pmid=19306292;citation_doi=10.1002/mc.20499" />
<meta name="citation_reference" content="citation_journal_title=Frontiers in bioscience :  a journal and virtual library;citation_journal_abbrev=Front Biosci;citation_author=C. H. Freudenreich;citation_title=Chromosome fragility: molecular mechanisms and cellular consequences.;citation_pages=4911-4924;citation_volume=12;citation_year=2007;citation_pmid=17569619;citation_doi=10.2741/2437" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Fungtammasan;citation_author=E. Walsh;citation_author=F. Chiaromonte;citation_author=K. A. Eckert;citation_author=K. D. Makova;citation_title=A genome-wide analysis of common fragile sites: What features determine chromosomal instability in the human genome?;citation_volume=22;citation_year=2012;citation_pmid=22456607;citation_doi=10.1101/gr.134395.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. E. Gammie;citation_author=N. Erdeniz;citation_author=J. Beaver;citation_author=B. Devlin;citation_author=A. Nanji;citation_title=Functional characterization of pathogenic human MSH2 missense mutations in Saccharomyces cerevisiae.;citation_volume=177;citation_year=2007;citation_pmid=17720936;citation_doi=10.1534/genetics.107.071084" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=M. Garcia-Diaz;citation_author=T. A. Kunkel;citation_title=Mechanism of a genetic glissando: structural biology of indel mutations.;citation_pages=206-214;citation_volume=31;citation_year=2006;citation_issue=4;citation_pmid=16545956;citation_doi=10.1016/j.tibs.2006.02.004" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Garcia-Diaz;citation_author=K. Bebenek;citation_author=J. M. Krahn;citation_author=L. C. Pedersen;citation_author=T. A. Kunkel;citation_title=Structural analysis of strand misalignment during DNA synthesis by a human DNA polymerase.;citation_pages=331-342;citation_volume=124;citation_year=2006;citation_issue=2;citation_pmid=16439207;citation_doi=10.1016/j.cell.2005.10.039" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Gemayel;citation_author=M. D. Vinces;citation_author=M. Legendre;citation_author=K. J. Verstrepen;citation_title=Variable tandem repeats accelerate evolution of coding and regulatory sequences.;citation_pages=445-477;citation_volume=44;citation_year=2010;citation_pmid=20809801;citation_doi=10.1146/annurev-genet-072610-155046" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=H. Geng;citation_author=M. Sakato;citation_author=V. DeRocco;citation_author=K. Yamane;citation_author=C. W. Du;citation_title=Biochemical analysis of the human mismatch repair proteins hMutS alpha MSH2(G674A)-MSH6 and MSH2–MSH6(T1219D).;citation_volume=287;citation_year=2012;citation_pmid=22277660;citation_doi=10.1074/jbc.M111.316919" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Giardine;citation_author=C. Riemer;citation_author=R. C. Hardison;citation_author=R. Burhans;citation_author=L. Elnitski;citation_title=Galaxy: a platform for interactive large-scale genome analysis.;citation_volume=15;citation_year=2005;citation_pmid=16169926;citation_doi=10.1101/gr.4086505" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Goecks;citation_author=A. Nekrutenko;citation_author=J. Taylor;citation_title=Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.;citation_pages=R86-R86;citation_volume=11;citation_year=2010;citation_issue=8;citation_pmid=20738864;citation_doi=10.1186/gb-2010-11-8-r86" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. Gragg;citation_author=B. D. Harfe;citation_author=S. Jinks-Robertson;citation_title=Base Composition of Mononucleotide Runs Affects DNA Polymerase Slippage and Removal of Frameshift Intermediates by Mismatch Repair in Saccharomyces cerevisiae;citation_pages=8756-8762;citation_volume=22;citation_year=2002;citation_issue=24;citation_pmid=12446792;citation_doi=10.1128/MCB.22.24.8756-8762.2002" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Hawk;citation_author=L. Stefanovic;citation_author=J. C. Boyer;citation_author=T. D. Petes;citation_author=R. A. Farber;citation_title=Variation in efficiency of DNA mismatch repair at different sites in the yeast genome.;citation_volume=102;citation_year=2005;citation_pmid=15932942;citation_doi=10.1073/pnas.0503415102" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. T. Henderson;citation_author=T. D. Petes;citation_title=Instability of simple sequence DNA in Saccharomyces cerevisiae.;citation_pages=2749-2757;citation_volume=12;citation_year=1992;citation_issue=6;citation_pmid=1588966;citation_doi=10.1128/MCB.12.6.2749" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of ageing and development;citation_journal_abbrev=Mech Ageing Dev;citation_author=P. Hsieh;citation_author=K. Yamane;citation_title=DNA mismatch repair: molecular mechanism, cancer, and ageing.;citation_pages=391-407;citation_volume=129;citation_year=2008;citation_issue=7-8;citation_pmid=18406444;citation_doi=10.1016/j.mad.2008.02.012" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. E. Huang;citation_author=A. G. Rio;citation_author=A. Nicolas;citation_author=R. D. Kolodner;citation_title=A genomewide screen in Saccharomyces cerevisiae for genes that suppress the accumulation of mutations.;citation_volume=100;citation_year=2003;citation_pmid=12972632;citation_doi=10.1073/pnas.2035018100" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Ito-Harashima;citation_author=P. E. Hartzog;citation_author=H. Sinha;citation_author=J. H. McCusker;citation_title=The tRNA-Tyr Gene Family of Saccharomyces cerevisiae: Agents of Phenotypic Variation and Position Effects on Mutation Frequency;citation_pages=1395-1410;citation_volume=161;citation_year=2002;citation_issue=4;citation_pmid=12196388" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y. D. Kelkar;citation_author=N. Strubczewski;citation_author=S. E. Hile;citation_author=F. Chiaromonte;citation_author=K. A. Eckert;citation_title=What is a microsatellite: a computational and experimental definition based upon repeat mutational behavior at A/T and GT/AC repeats.;citation_volume=2;citation_year=2010;citation_pmid=20668018;citation_doi=10.1093/gbe/evq046" />
<meta name="citation_reference" content="citation_journal_title=Tumori;citation_journal_abbrev=Tumori;citation_author=Y. R. Kim;citation_author=N. G. Chung;citation_author=M. R. Kang;citation_author=N. J. Yoo;citation_author=S. H. Lee;citation_title=Novel somatic frameshift mutations of genes related to cell cycle and DNA damage response in gastric and colorectal cancers with microsatellite instability.;citation_pages=1004-1009;citation_volume=96;citation_year=2010;citation_issue=6;citation_pmid=21388066" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=T. A. Kunkel;citation_author=D. A. Erie;citation_title=DNA mismatch repair.;citation_pages=681-710;citation_volume=74;citation_year=2005;citation_pmid=15952900;citation_doi=10.1146/annurev.biochem.74.082803.133243" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. A. Kunz;citation_author=K. Ramachandran;citation_author=E. J. Vonarx;citation_title=DNA Sequence Analysis of Spontaneous Mutagenesis in Saccharomyces cerevisiae;citation_pages=1491-1505;citation_volume=148;citation_year=1998;citation_issue=4;citation_pmid=9560369" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. I. Lang;citation_author=A. W. Murray;citation_title=Estimating the Per-Base-Pair Mutation Rate in the Yeast Saccharomyces cerevisiae;citation_pages=67-82;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202359;citation_doi=10.1534/genetics.107.071506" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=G. I. Lang;citation_author=A. W. Murray;citation_title=Mutation rates across budding yeast chromosome VI are correlated with replication timing.;citation_volume=3;citation_year=2011;citation_pmid=21666225;citation_doi=10.1093/gbe/evr054" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. E. LeClerc;citation_author=B. Li;citation_author=W. L. Payne;citation_author=T. A. Cebula;citation_title=High Mutation Frequencies Among Escherichia coli and Salmonella Pathogens;citation_pages=1208-1211;citation_volume=274;citation_year=1996;citation_issue=5290;citation_pmid=8895473;citation_doi=10.1126/science.274.5290.1208" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=G. Levinson;citation_author=G. A. Gutman;citation_title=Slipped-strand mispairing: a major mechanism for DNA sequence evolution;citation_pages=203-221;citation_volume=4;citation_year=1987;citation_issue=3;citation_pmid=3328815" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Y. C. Li;citation_author=A. B. Korol;citation_author=T. Fahima;citation_author=E. Nevo;citation_title=Microsatellites within genes: structure, function, and evolution.;citation_volume=21;citation_year=2004;citation_pmid=14963101;citation_doi=10.1093/molbev/msh073" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=S. Lim;citation_author=L. Notley-McRobb;citation_author=M. Lim;citation_author=D. A. Carter;citation_title=A comparison of the nature and abundance of microsatellites in 14 fungal genomes.;citation_pages=1025-1036;citation_volume=41;citation_year=2004;citation_issue=11;citation_pmid=15465391;citation_doi=10.1016/j.fgb.2004.08.004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. A. Lujan;citation_author=J. S. Williams;citation_author=Z. F. Pursell;citation_author=A. A. Abdulovic-Cui;citation_author=A. B. Clark;citation_title=Mismatch repair balances leading and lagging strand DNA replication fidelity.;citation_pages=e1003016-e1003016;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071460;citation_doi=10.1371/journal.pgen.1003016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. E. Luria;citation_author=M. Delbrück;citation_title=MUTATIONS OF BACTERIA FROM VIRUS SENSITIVITY TO VIRUS RESISTANCE;citation_pages=491-511;citation_volume=28;citation_year=1943;citation_issue=6;citation_pmid=17247100" />
<meta name="citation_reference" content="citation_journal_title=Clinical genetics;citation_journal_abbrev=Clin Genet;citation_author=H. T. Lynch;citation_author=P. M. Lynch;citation_author=S. J. Lanspa;citation_author=C. L. Snyder;citation_author=J. F. Lynch;citation_title=Review of the Lynch syndrome: history, molecular genetics, screening, differential diagnosis, and medicolegal ramifications.;citation_pages=1-18;citation_volume=76;citation_year=2009;citation_issue=1;citation_pmid=19659756" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast.;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Rep;citation_author=X. Ma;citation_author=M. V. Rogacheva;citation_author=K. T. Nishant;citation_author=S. Zanders;citation_author=C. D. Bustamante;citation_title=Mutation hot spots in yeast caused by long-range clustering of homopolymeric sequences.;citation_pages=36-42;citation_volume=1;citation_year=2012;citation_issue=1;citation_pmid=22832106;citation_doi=10.1016/j.celrep.2011.10.003" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=G. T. Marsischky;citation_author=N. Filosi;citation_author=M. F. Kane;citation_author=R. Kolodner;citation_title=Redundancy of Saccharomyces cerevisiae MSH3 and MSH6 in MSH2-dependent mismatch repair.;citation_pages=407-420;citation_volume=10;citation_year=1996;citation_issue=4;citation_pmid=8600025;citation_doi=10.1101/gad.10.4.407" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. Matic;citation_author=M. Radman;citation_author=F. Taddei;citation_author=B. Picard;citation_author=C. Doit;citation_title=Highly Variable Mutation Rates in Commensal and Pathogenic Escherichia coli;citation_pages=1833-1834;citation_volume=277;citation_year=1997;citation_issue=5333;citation_pmid=9324769;citation_doi=10.1126/science.277.5333.1833" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. J. McDonald;citation_author=W. C. Wang;citation_author=H. D. Huang;citation_author=J. Y. Leu;citation_title=Clusters of nucleotide substitutions and insertion/deletion mutations are associated with repeat sequences.;citation_pages=12;citation_volume=9;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Comprehensive molecular characterization of human colon and rectal cancer.;citation_pages=330-337;citation_volume=487;citation_year=2012;citation_issue=7407;citation_pmid=22810696;citation_doi=10.1038/nature11252" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. T. Nishant;citation_author=W. Wei;citation_author=E. Mancera;citation_author=J. L. Argueso;citation_author=A. Schlattl;citation_title=The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.;citation_pages=e1001109;citation_volume=6;citation_year=2010;citation_pmid=20838597;citation_doi=10.1371/journal.pgen.1001109" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=G. Ohnishi;citation_author=K. Endo;citation_author=A. Doi;citation_author=A. Fujita;citation_author=Y. Daigaku;citation_title=Spontaneous mutagenesis in haploid and diploid Saccharomyces cerevisiae.;citation_pages=928-933;citation_volume=325;citation_year=2004;citation_issue=3;citation_pmid=15541379;citation_doi=10.1016/j.bbrc.2004.10.120" />
<meta name="citation_reference" content="citation_journal_title=Journal of gastroenterology;citation_journal_abbrev=J Gastroenterol;citation_author=E. Oki;citation_author=Y. Hisamatsu;citation_author=K. Ando;citation_author=H. Saeki;citation_author=Y. Kakeji;citation_title=Clinical aspect and molecular mechanism of DNA aneuploidy in gastric cancers.;citation_pages=351-358;citation_volume=47;citation_year=2012;citation_issue=4;citation_pmid=22402775;citation_doi=10.1007/s00535-012-0565-4" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Oliver;citation_author=R. Canton;citation_author=P. Campo;citation_author=F. Baquero;citation_author=J. Blazquez;citation_title=High Frequency of Hypermutable Pseudomonas aeruginosa in Cystic Fibrosis Lung Infection;citation_pages=1251-1253;citation_volume=288;citation_year=2000;citation_issue=5469;citation_pmid=10818002;citation_doi=10.1126/science.288.5469.1251" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=F. Palombo;citation_author=I. Iaccarino;citation_author=E. Nakajima;citation_author=M. Ikejima;citation_author=T. Shimada;citation_title=hMutSbeta, a heterodimer of hMSH2 and hMSH3, binds to insertion/deletion loops in DNA.;citation_pages=1181-1184;citation_volume=6;citation_year=1996;citation_issue=9;citation_pmid=8805365;citation_doi=10.1016/S0960-9822(02)70685-4" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=T. Pupko;citation_author=D. Graur;citation_title=Evolution of microsatellites in the yeast Saccharomyces cerevisiae: role of length and number of repeated units.;citation_pages=313-316;citation_volume=48;citation_year=1999;citation_issue=3;citation_pmid=10093220;citation_doi=10.1007/PL00006474" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. K. Raghuraman;citation_author=E. A. Winzeler;citation_author=D. Collingwood;citation_author=S. Hunt;citation_author=L. Wodicka;citation_title=Replication Dynamics of the Yeast Genome;citation_pages=115-121;citation_volume=294;citation_year=2001;citation_issue=5540;citation_pmid=11588253;citation_doi=10.1126/science.294.5540.115" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=G. F. Richard;citation_author=A. Kerrest;citation_author=B. Dujon;citation_title=Comparative Genomics and Molecular Dynamics of DNA Repeats in Eukaryotes;citation_pages=686-727;citation_volume=72;citation_year=2008;citation_issue=4;citation_pmid=19052325;citation_doi=10.1128/MMBR.00011-08" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=O. Rose;citation_author=D. Falush;citation_title=A threshold size for microsatellite expansion;citation_pages=613-615;citation_volume=15;citation_year=1998;citation_issue=5;citation_pmid=9580993;citation_doi=10.1093/oxfordjournals.molbev.a025964" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=S. Sarkar;citation_author=W. T. Ma;citation_author=G. V. Sandri;citation_title=On fluctuation analysis: a new, simple and efficient method for computing the expected number of mutants.;citation_pages=173-179;citation_volume=85;citation_year=1992;citation_issue=2;citation_pmid=1624139;citation_doi=10.1007/BF00120324" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=S. N. Shah;citation_author=S. E. Hile;citation_author=K. A. Eckert;citation_title=Defective mismatch repair, microsatellite mutation bias, and variability in clinical cancer phenotypes.;citation_volume=70;citation_year=2010;citation_pmid=20068152;citation_doi=10.1158/0008-5472.CAN-09-3049" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. N. Shah;citation_author=P. L. Opresko;citation_author=X. Meng;citation_author=M. Y. Lee;citation_author=K. A. Eckert;citation_title=DNA structure and the Werner protein modulate human DNA polymerase delta-dependent replication dynamics within the common fragile site FRA16D.;citation_volume=38;citation_year=2010;citation_pmid=19969545;citation_doi=10.1093/nar/gkp1131" />
<meta name="citation_reference" content="citation_journal_title=SÃ£o Paulo medical journal = Revista paulista de medicina;citation_journal_abbrev=Sao Paulo Med J;citation_author=F. C. Silva;citation_author=M. D. Valentin;citation_author=O. Ferreira Fde;citation_author=D. M. Carraro;citation_author=B. M. Rossi;citation_title=Mismatch repair genes in Lynch syndrome: a review.;citation_pages=46-51;citation_volume=127;citation_year=2009;citation_issue=1;citation_pmid=19466295;citation_doi=10.1590/S1516-31802009000100010" />
<meta name="citation_reference" content="citation_journal_title=DNA repair;citation_journal_abbrev=DNA Repair (Amst);citation_author=L. Stojic;citation_author=R. Brun;citation_author=J. Jiricny;citation_title=Mismatch repair and DNA damage signalling.;citation_pages=1091-1101;citation_volume=3;citation_year=2004;citation_issue=8-9;citation_pmid=15279797;citation_doi=10.1016/j.dnarep.2004.06.006" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Subramanian;citation_author=R. K. Mishra;citation_author=L. Singh;citation_title=Genome-wide analysis of microsatellite repeats in humans: their abundance and density in specific genomic regions.;citation_pages=R13-R13;citation_volume=4;citation_year=2003;citation_issue=2;citation_pmid=12620123;citation_doi=10.1186/gb-2003-4-2-r13" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. A. Surtees;citation_author=J. L. Argueso;citation_author=E. Alani;citation_title=Mismatch repair proteins: key regulators of genetic recombination.;citation_pages=146-159;citation_volume=107;citation_year=2004;citation_issue=3-4;citation_pmid=15467360;citation_doi=10.1159/000080593" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_pages=178-192;citation_volume=14;citation_year=2012;citation_pmid=22517427" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. T. Tran;citation_author=J. D. Keen;citation_author=M. Kricker;citation_author=M. A. Resnick;citation_author=D. A. Gordenin;citation_title=Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants;citation_pages=2859-2865;citation_volume=17;citation_year=1997;citation_issue=5;citation_pmid=9111358;citation_doi=10.1128/MCB.17.5.2859" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Umar;citation_author=J. I. Risinger;citation_author=W. E. Glaab;citation_author=K. R. Tindall;citation_author=J. C. Barrett;citation_title=Functional Overlap in Mismatch Repair by Human MSH3 and MSH6;citation_pages=1637-1646;citation_volume=148;citation_year=1998;citation_issue=4;citation_pmid=9560383" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Wierdl;citation_author=M. Dominska;citation_author=T. D. Petes;citation_title=Microsatellite Instability in Yeast: Dependence on the Length of the Microsatellite;citation_pages=769-779;citation_volume=146;citation_year=1997;citation_issue=3;citation_pmid=9215886" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zanders;citation_author=X. Ma;citation_author=A. RoyChoudhury;citation_author=R. D. Hernandez;citation_author=A. Demogines;citation_title=Detection of heterozygous mutations in the genome of mismatch repair defective diploid yeast using a Bayesian approach.;citation_volume=186;citation_year=2010;citation_pmid=20660644;citation_doi=10.1534/genetics.110.120105" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=H. Zhang;citation_author=C. H. Freudenreich;citation_title=An AT-rich sequence in human common fragile site FRA16D causes fork stalling and chromosome breakage in S. cerevisiae.;citation_pages=367-379;citation_volume=27;citation_year=2007;citation_issue=3;citation_pmid=17679088;citation_doi=10.1016/j.molcel.2007.06.012" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
<meta name="twitter:description" content="DNA mismatch repair is a highly conserved DNA repair pathway. In humans, germline mutations in hMSH2 or hMLH1 , key components of mismatch repair, have been associated with Lynch syndrome, a leading cause of inherited cancer mortality. Current estimates of the mutation rate and the mutational spectra in mismatch repair defective cells are primarily limited to a small number of individual reporter loci. Here we use the yeast Saccharomyces cerevisiae to generate a genome-wide view of the rates, spectra, and distribution of mutation in the absence of mismatch repair. We performed mutation accumulation assays and next generation sequencing on 19 strains, including 16 [msh2][1] missense variants implicated in Lynch cancer syndrome. The mutation rate for DNA mismatch repair null strains was approximately 1 mutation per genome per generation, 225-fold greater than the wild-type rate. The mutations were distributed randomly throughout the genome, independent of replication timing. The mutation spectra included insertions/deletions at homopolymeric runs (87.7%) and at larger microsatellites (5.9%), as well as transitions (4.5%) and transversions (1.9%). Additionally, repeat regions with proximal repeats are more likely to be mutated. A bias toward deletions at homopolymers and insertions at (AT)n microsatellites suggests a different mechanism for mismatch generation at these sites. Interestingly, 5% of the single base pair substitutions might represent double-slippage events that occurred at the junction of immediately adjacent repeats, resulting in a shift in the repeat boundary. These data suggest a closer scrutiny of tumor suppressors with homopolymeric runs with proximal repeats as the potential drivers of oncogenesis in mismatch repair defective cells.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450" />
<meta name="og-title" property="og:title" content="Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/9/1453" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="DNA mismatch repair is a highly conserved DNA repair pathway. In humans, germline mutations in hMSH2 or hMLH1 , key components of mismatch repair, have been associated with Lynch syndrome, a leading cause of inherited cancer mortality. Current estimates of the mutation rate and the mutational spectra in mismatch repair defective cells are primarily limited to a small number of individual reporter loci. Here we use the yeast Saccharomyces cerevisiae to generate a genome-wide view of the rates, spectra, and distribution of mutation in the absence of mismatch repair. We performed mutation accumulation assays and next generation sequencing on 19 strains, including 16 [msh2][1] missense variants implicated in Lynch cancer syndrome. The mutation rate for DNA mismatch repair null strains was approximately 1 mutation per genome per generation, 225-fold greater than the wild-type rate. The mutations were distributed randomly throughout the genome, independent of replication timing. The mutation spectra included insertions/deletions at homopolymeric runs (87.7%) and at larger microsatellites (5.9%), as well as transitions (4.5%) and transversions (1.9%). Additionally, repeat regions with proximal repeats are more likely to be mutated. A bias toward deletions at homopolymers and insertions at (AT)n microsatellites suggests a different mechanism for mismatch generation at these sites. Interestingly, 5% of the single base pair substitutions might represent double-slippage events that occurred at the junction of immediately adjacent repeats, resulting in a shift in the repeat boundary. These data suggest a closer scrutiny of tumor suppressors with homopolymeric runs with proximal repeats as the potential drivers of oncogenesis in mismatch repair defective cells.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
    <title>Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8urQB20L7gD5h7yiIYJ8q2Xs_mMF_3WQhnpbbOjX1mA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459529","apath":"\/ggg\/3\/9\/1453.atom","pisa":"ggg;3\/9\/1453","ac":{"\/ggg\/3\/9\/1453.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/9\/1453.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/9\/1453"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/9\/1453"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/9\/1453.full":true,"\/content\/3\/9\/1453":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gsKae65qqcwLFwpW8R9OXLqwRgSgqOSilMQFQRTJC7rQhUAFsJbzwpUf+FpH\\\\\\\/shJb9uW3rNSGmjKzC3ai\\\\\\\/x9pxlRy\\\\\\\/sbjYKLq0t4yln1d3H+t8zlatFxASRlS5t8oIY36sKphGX3J1wOXBnOuTM3Ugh6N\\\\\\\/mvDt3TlRMsTY2qjtRcrXGQYaDvTiHP93zIGXaFOWOFwIqzC\\\\\\\/tN4HEa5ZK\\\\\\\/bxQX13ayEUZXDFp1ib\\\\\\\/uir6Dy6ktw4EokRRdcORqNGEgGwu9sgxFpuxfbvnCHYnlmEBF55quFZXBNahg1cPIZxwBTIm5jubrWYlTsSWf2y0tPv5R4m7iSGkDD67zNCLsyNwUJ+u4W\\\\\\\/vPM\\\\\\\/BSF1\\\\\\\/72pS2weP8XHocHQm2eleA4SZhrwhRYjjwmXLT6gif6wDRmPHJx07lyIWdcw5Y4aWjx\\\\\\\/xZDB9PxMw1vc73YWxB80mcJsu1zxTi2y+JY2uGKAQEV2rh6tcDRmMBw6zU6OMHcKA45SDz1yf0n7WhC2SLW3BU69zT3fwiC1QAFY\\\\\\\/P4NiRUd6km32jDPQeCoLsDqSjSy\\\\\\\/fA9p\\\\\\\/HYpEwZp+w3c6k7Or7UfKXxQ8C59m5sjhkzkqZN8GztCCqlUbdB08jJPP3A1L2JAd5PavqFthfuYl6X28soITIO15E9IFa4RJDUlVoJ\\\\\\\/i9CCO+i3CmA+ZgScvbAOvOtVAhOMEfWE5\\\\\\\/bSltty9pBzdqTNgWB4qF6cbOn\\\\\\\/QcNw1udEoum1HtEYX4QJfazGNpXLtZJ5gYKqIvkC0D10WGStoZinO+QRptBfIkGFqD+dUQ6W6MInIiFRkcRpz+P0I0dR\\\\\\\/FXis7gy0il8MYZlyimH+N3fHA5XyTaq8hKDc6yRiEJ2RVATj8W0gu+Se3+QmqdN8UAW0PKqtWHSBeAacnzyjmVa+6KiWi6fPpWMZ6Lo\\\\\\\/\\\\\\\/kT0p9Pr9HY7w5gYhgLKiaSVg6Ah2D8\\\\\\\/Iiz2KP4l8RX58p7kG\\\\\\\/QVQF\\\\\\\/qe6WlgTmUOuSaAhJawqDRCsv8U9PkD9HTlTZG8CT5zvTyB9nEw92NjbuASmSv5\\\\\\\/Ez0Z0CxslKKbqOUe1lhgnmCj8WRR9PBviNA1V36yH7fYIErVv1gfi5GUJTmFfLNvQ\\\\\\\/5brKN79zZhSt8WFZPo0gjigJHplJuW4yjyMX1N0LJAbluZveODnq6UT5VyWjkiYa3aJ2R4WR40+bCqhdyTyw8sDD8pSs6\\\\\\\/VmtfW4Usb98XF5f+kVwlOEVv3hfXEEfyl\\\\\\\/QoMa00drVZ3k2R4py0xBjL95gxri4ysbYRWJQuzFExFmTrmh\\\u0022,\\\u0022iv\\\u0022:\\\u0022sGPQt5ruKwkf\\\\\\\/LuF+hmpTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228defeb62ef3bde9a6ed7145ab24ca3e11ef689360a7f90f0aa3da0970e6a36b2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kEBlpu1jXqhb5JYndGph+yhHhIyRFeQC\\\\\\\/VqoBX8PE6xmNlRTzQVm28nJj7kgWmtca6UD5S4v\\\\\\\/nF5gkMgqhPevVrEOaUmzu3cE27eGtwDyy3XwUqfuAasnt1fcHf\\\\\\\/l3Fxx87VNeGX4Sn9npE7kbPyHZCzXyCgs\\\\\\\/K\\\\\\\/Pm\\\\\\\/Td4aNxrWCXLhMCss0wWfg0YlnQyDX9F7FYtY1ZGRSXWjJz4qNTfBPdTBA3eQGabWk83dYy0x7sfyZzWeN6RbNdTK\\\\\\\/7vP6r+aUbocirB5vNyAQ5vbtUWG1XpIGHpRyElFd+Dj6\\\\\\\/WxrX+LtyssmZczWHWCJpqrRXLbeem8mx3HCskCiFG6t+RLXAwo+c67oqSlQD50UwmAVt5H9t8XY\\\\\\\/uFUTnGd3LS0Fih+bwa4O7f+nErIxZvlJ7oWqV0S3+KLRJutiLzNfmbuPVBvGcplfUVeYORYOcXOGgDKiv5VM8Rb+lymlzAGTeS+XtoKxuhIi4+uE4Gufq5knB5Uaxr85XkyoDktkd5dVFVWsEn3ZZSInUQzdFHQCxdNMsD5J4odsffRDbE1kT1Q2Hd1kH4HsXOq4QN+tHxaIF3i1\\\\\\\/vvYynBf+vV979LkZdl9YT4JOo8qBaePiAYe9lVWSM7Y7nJhEbZZETYXZiFyivDDe\\\\\\\/O7CLsap3QNCLXOMrnI4wnFfymWixTEaDkYv1JmkNNj+YwpXzFVXX3M7\\\\\\\/dalyg3v+B\\\\\\\/6O1EJbG1QPVKGnMSwtxkNbB4Ca07946HeD1m1cGwplnSDYv834Lsu41USC9WdQshIlFB7U47dHZXqhhU0zCc6DFjijAYDYv0YivX3WKaZp2YBAb4+limbxoh0j1Dkn2Bf66FoeGTV8uqn0LjaaRmfM59lqd8MSCAnn1Mg+po5ETdpqTdQaUqEKi8cemCVKqNUm\\\\\\\/e4iHqutPovioqkaBxTYvb3EQgU4yXdYdMCgZTGeabfTy14Y+ue0UIx1OvZMtOM1tAAkPA6Pmo0+fket8NxuXkp59jrVmFx8LNnvg2OIECsSzW4frvqnibyOdM2ukO35we3NrruhnbCi60PRhq\\\\\\\/TiK7+SRgU\\\\\\\/pT0o4K2RTDvoK+eA66UhEwQanwIiL1ezfUc3u5AWDh96m3ToHMxW9dMXMOeQxUZOgEMa0OyzG+bOH7yScLEQHV71xQ5z4s+mPittMEKE0\\\\\\\/Dz1r0SzB\\\\\\\/5vtDB7vdY+d7JXZrKRY8\\\\\\\/RqFeCFTr2s2d8nO3952ZwPIdGo\\\\\\\/x2mjBQDIIvSPogCEK0B+tUnZvjiXoHbVR558nlttbYyDPxgsBp9Z8vSgzjqFBiQV1DRbcNf1Nd9FjggUX+ikM524CYzP2T8EOJkFOOlTPeN1UC3iu3k5zn0sXtbzMcfEjdhtUQAbWDd3HwepoFP1aZtQq9XdOWGX3ukg9atBexU670eUB32aRaEJsvSMFLel0Qro3rhPIe7gbfCHmC18N3dsDL6SdDvpsP1udUWZhGWlAiz36g7y89GMfpAS4gaU\\\\\\\/dzeiaH6KmJKWEQUcdbZKlLjBTdlOONsqMR4Frk1hpaDqOxHwmlFF2YJx178eeDh\\\\\\\/I\\\\\\\/WFyG8scWO9sVbu4IB\\\\\\\/uiqzMqClpCLfCl\\\\\\\/9VqbwEjoGL1xd\\\\\\\/xkFP3YsciseJCEtEcjCfaA5cMdIZfk7dj9FuyKRKljecl+Q4jUp3o+knOU5a1JiU8fMsyvtd9ShCyc4yefpud3zsHFYIPz4QMVdR1Zkb0vMs\\\\\\\/u059WeUKnIAiH+sVPOATXYmaL3viUSZI\\\\\\\/S16qkgrjT9XIonZ7v+t8wAhTkaO3U4+Pl1RCGCZMK4zc6ZUJAc8FhmdVHnslbWXt8Z3wOieHczxrBtx3FF9o6J1fVUyHsivzg\\\\\\\/EmB3o6B2ktpsuVzhKTbnGJuyRJe+EGWJ7fQfDNNVb1Zh9G6pzlT7cpJ\\\\\\\/p+3TGVIAbVi6wuCAoOlnWLqtKPblb6QqmodCLZSZbEgAdDhrZOwwZBuZlmEPB\\\\\\\/BmHYj71yrC4RgFv1A7X6DCDweRC5s04qQuHSuNn2ELM2KlN5J5l3AV+LesBhInSJrM2e2f2VTiqEJoisC\\\\\\\/I6PwxYHtX64zSzTuuxY2n86A+7UqvVYXxwHrTo8AX4eVjZnpYeiqqoBl7NdGqRFfafL80tZfRbPkKuMRuf3Hj91dG7I65MbX0RoXGVdo5NcPSOpSaaIApsrLju+qgBv7V\\\\\\\/kKzL1jzHbY4abOP8vEfLZIoDVGS8H\\\\\\\/HcRGCVZOdYk6QkLtdrgOCluK+XMY0A8j3+cCbWSailTYPWPB\\\\\\\/\\\\\\\/O0+2\\\\\\\/+C44u7tQjKz0+Rnm2JsIktwnO9pigQ8Rki87P\\\\\\\/9KrZa+gv04+gMTQW5OscSBGZVdihwHatpl+ZCEAmjvVWbCxXcynaMNhOO9M+GN8lIzHvsJ3FRumyjL8Mf7RyGuy3lejZlOtvzz6+vDa7+yOjwpan\\\\\\\/pcN6\\\\\\\/QtOy1R2jLCJMDmg8c9nE4UxNmHYGqsFNNrwhhmsd0gRJ5Ot\\\\\\\/2BFVp0AXDsg6nK3PvcuRkOq1uTUXJXp7ld3I7JP5zJunbtkdjQkhL0h\\\\\\\/H6rPpPZ\\\\\\\/AVGila5mEuL0HOLJzne5EsYQbuBsKhN9ywXeEgmqeBF29BkQVS9C7\\\\\\\/MqBS0wKvoqyH9LDlA8d9Q9R8IkLfriM4FhI\\\\\\\/dJnntEPrPlDQX6RkA+WdQatlDnCOjTWQAuryknjnJk7eU\\\\\\\/qWMgXrjRC\\\\\\\/xZ+GcnyQUeB9xTowWcN\\\\\\\/cikY753pbMnAB5HZbRfbH\\\\\\\/2Mq+4VYdavxZuyjRmuexf\\\\\\\/m72xXmWXL60ypxLrSNCUaLtVCfC6Cx6WjDzHhVFnzpR1faoSg2g3Tbf3FydP5kf8LhA51mcgV1UgmC9KmwkCPStJueo+a4eVocjKGR+MGJz2n9MTQ4ctj+U9PAU\\\\\\\/9RrLSGjwnAklrcG3EI0ahDY6Mfru9\\\\\\\/IIsDw8NsPUcVUmBUbkS6IOCohdiGnALhE89fwFgfbsBDfsEX43KfYaXP7kJFG9hldi8OZtOmTPRwUfQbTaQwR2sWl9FzAnzYK8TW2o1KjopEzEKVm1tEFVS6R4edhMiXtYaCEQDSg1q1Kn+9enFeS28R4QrkwNp9gSL6DS8MolNJ7c1D2eVilipkdZjjXZ61G4AfSsHMXiAZkLQS7yUmVVdlvBbTcQj6ZnDs8C76yLKrUTbMMHIcvKtK7MQftPcCYfixJ0Zfgv8vuMnb4m4rmCKFF9aYHlHpnsNvyAOH42pSXgC8cWiiY5ZV6Q+V1x6qvECHLrLTSPHQPbfk6\\\\\\\/8x34q1ccVqzwtvzDhOaE9IIXrN51FXppa05pc8bLtIN1oulIpU4wt0ddILLyYqoYRRgpuhQYBOgGUszsPqVhol2340spGQneMJwYBjAvpeeGZa4DZy0CjxNujmps5+SFwgnpeeXn5hKk+Yf4ejav0H4gzb2oco8j+47AR4pQHiVpOxNYxJgvunyMeSw3YOZ4cSkf8NaZoGwGqZK\\\\\\\/cyYRnnn5Byxe22ZKMmhD9\\\\\\\/a9SViDbPDYly9u0zJz1rbyY2Rig+j5oSMAkvatZ8JCevyRhKg3hw5JRJKb6or1U66U8Cn2TEBidxpWF42on+2r23cQS1c38YdbpLb07T+NvNE\\\\\\\/oTgqpqPuDpM1kzMc0hPXU\\\\\\\/L21GYfa5xVeEWg5CfsqpByB5Wy4fngmN2otKHnMlOyVax8kvtAmIOB4uLjyFUuQSMswhZd5GrYY180RKLmSFT7Me6ExRQ9E\\\\\\\/gSE6bIPY+x7S2KEZYboxA6r5hp94gPC0ikM56utulO2k\\\\\\\/rYwLtv7E\\\\\\\/+df8ZlUl5\\\\\\\/UG47bRxLff5Fg3xJIT0BoI8\\\\\\\/oWgCgLOJQCWeFVK6+XFpupGc8kkyXX7APKQGsl4TkgiS1YlVrVG1CJXXKqfxkmOntt9j8c544hXszYgGwrxIRIQmaQzPOwXlS+mvuEBHIiVF+nV+LgvWjNo+nXmhaqhaADMa45RYF1SMEFAFtF5A2h5TueOu62uCu01SWrTwNs16lCyykOXETXVTQYEZHDRZV9iONteaHvBCA6xmKlJMqG2sjYvFZhsFD2gRa7hC4G85ftUXH\\\\\\\/6XEg1R\\\\\\\/6n4bC7ViHFQstCHvoOUYKssjzTM2ETQiBtDjGtPlW9J35slfa9WJlBIGC2MrbsZaDTym2HS6yFVBcBhmuPc5bbNlgbSrcqg4WitzDWvMokUAOZzXBt5mAsRHOF40TvN9ZsKFIW48yaJBXZxdmN535svCFv\\\\\\\/9tcM+Rm922q219+zOalVYN\\\\\\\/ho4fR4Jgfonuftcc6g0keF\\\\\\\/kUUdPCmdpAIH+YmKbngHhKSQ0cptsb9wy5vO+gmvkoBmgQx2navJlEi+uZkUOO+QtOnA77CTwE1MCp9jlXjIMBPYPSE8w92niesPyYDGltTjUMd1n4yJttcGNerXAxQSj9J7vKHm+LVqJdHsKDt6Uy0m4Oo3lOa6txl0hPfgWrW5j4gYrMgYSzrmsvfKq99DbpqO7W7mDNwO4ixqmrZJsT7dIwA9\\\\\\\/SkEkWZSOW4C\\\\\\\/gdUZ6JsSg\\\\\\\/SLfMl05QX\\\\\\\/CqveR4r9CxN20zZ4Bhx+kNL7eP9LnWaxPLEmSFRyd2qb1OIBqcS8jzpyTwRdcNHfwk9ETQHYi968Wl1jMV3N\\\\\\\/ru6fKjvbw1M4sI4jYHQ1a7AQNakR4+KLQbot6+VkQfZafLlyo30z4mYV6IOsp+g106Rv6tUCZz5m4cvMzT8NJFPPTIBIcLu5KLAIZrnKUc\\\\\\\/YUdGfdYY0laK408TnW0cDsQs8HlgOd8eYZiTcvVharz58gB6cR7C0yEP9W4jwSZkQeINLcLSNnxj3EbvJXHLzd15V\\\\\\\/xeVxHZs46GkIPTjdy16ocnmXjWFSTy5v0bL+uIhlI+bcMbAsW16SAONiPxFTcNLG0rnyuyXRRRzxOrGW5ya2fGSFhNhdgkTSUdbHOcTSdF4xyaYH4RIA721TAV51GHPwyadbZztqGsgTL91htxb5MFtdkeMud2k3PDnzCW7UiZyzoAtV82ZhFf0Yjpa4VSSC1nBmvXSedBMkKaHBSuH1xRjxqK4UxFAPHTUQQmIadLENcNbNkVC8g8pNbWiGMVEEwTaD\\\\\\\/097g0a4\\\\\\\/ssnG4gU1tqpLwylpOW4B0yRKzMFCu6yg12eq5SdL811cU5rczWQrrswZRvonc1\\\\\\\/9vmNygHJumjkfxL2QCtOAKFZCgMHZ7EjUJ+910571FPzc52J2edHGAQhNLytmbrLUy0ZntRQRV6JNHkBvRzUOpPaDVXoERGLZ34ZmPw2p08Bs1xMcOw5FsIDfgUWe6uwxOQQDMjEzqNO4X5COWP883esPc88V6Mz6aP3Sloq9jSxWLFoaCDJJPtdBliPO4GHgSdyEfPpwNFPxcp4MT9DkQQk3\\\\\\\/39Kxpnuqh0X5sgt\\\\\\\/NS0MgEyVl764ZwIv1mC\\\\\\\/3kYubKUqoAd9WKOLubN7\\\\\\\/4MsUwz4vngb5pJ0iiYoOt5p5ZRjvJhDz23MGZNLkaEdIXPOcyNkcSNFfnbKpcXqxAeYtZsdPE9BZj526goexlOUVS5SWU\\\\\\\/JJK0XX\\\\\\\/irSeCrxypkGL6odlAzRQK5\\\\\\\/GFXRvebCl+cpIufeJBkze4l1bYDHZDc6cJdaO8SIBgIpfURRYQdjAAPb9LbhFCw9UzWAxbitNdn4GX06m0D31mupSiDzOWEOGkDLH2j\\\\\\\/coWKea0NtfszDJ6LzFd3n3KgntEutbchoM5aa4sB\\\\\\\/VALtt+oYDAJmqyBZ4m+IMvGW+GteJrJUG5pTKaD3Oj7Egl7Z\\\\\\\/E9q1CDwDwIxgOmqrSGqDTiWlmY3hFEl3z0tvPc0d2s+xF3hgrsTJPhmJ96VhEMh226CK0DezBoTHH0FSi4Daea9P1OIHgIpshAXPUCxUUeSfJZQUDOdyTVvBtwNfCL8ZX+opjzmDh7QyTpvRYT\\\\\\\/K+m8zTZBlfFOcmGcUZGjWxQC1kadhRzwMWosxJSmz+\\\\\\\/15zQceeUKYjRubfLkjmEH92hjkbPiMeLzW14vd9NsuyEC2WsvS3KEuDRCx+syCQVSNtBBIOkI8OWsBXoLZFZFqDvntN9yod4\\\\\\\/6fF4g7FF9jNyyfSYHTAUSjNXJyQArihyAd+\\\\\\\/UEFU2Mqvl7geqPfJor5\\\\\\\/qSQmc3V3H0\\\\\\\/9fL6smAiuWHWpGPS5K4hnnVQDcthZ6qmZPloauib5fNk3q7VJHiVNfPrryjzW5BzAkkQgXBOBM\\\\\\\/efgd\\\\\\\/nVB80aiHYcECQP0Swwm6mgPXGendyoKP6zoIhPsN+LV4+g5tY6iKtU\\\\\\\/1zI+C43MTj8MUJzRZKIc4q8Yo1FSUAnVJ4DynBFlFf12w2iuAAD3z9zRnOUxAvwtOJh6S\\\\\\\/cbhECJ9Admlt2jjwiwmAg\\\\\\\/rBLZnyQV7oHx9vT76WVYNERVpFi205U6f1xSo9GXuflck5opgGBPFJDpsFK\\\\\\\/\\\\\\\/nON3+c2D14VawVZX4GdbMjXDMTI+quQ300WM7OK94YtBVhMhJqKpWmh8h9\\\\\\\/h3t7XFJ0it\\\\\\\/a+mCKqEuhr+RM8YyvSKsahLMAmQH5O3bAKMy9zhvYkXrB0h2kc0FeQeYEMIqbfBQtB21H5ToZvUT0fJUgGHtVZReCgL6zCCyyyZJNzFB\\\\\\\/up6ur8VqpW4EbNrByiFe7Fgcf6TKp7mQgBAJyxONCnstLwQjsFI422S5PxDmk1hfZJOSqYobtd\\\\\\\/23fL6KU2S4X975z++n8H4P\\\\\\\/vKiNQTl9QURs6UJp5E9GRiyyUaG0hZN5WZtBJZIYkc9Gryv1PliTqOtjCIrn2QxOucCmKmnFvrZB44REpER6mecegfFrwTai0fG67uchbxYbBb0KUkp8NUwUtbyCUr778x\\\\\\\/6JbuudKylsdTwVTzpV6yZy+fjZMoEyGyFjGtVUzYAtQtXN6+NVLQZitbgZtqEo8cVxoU28SN1TxK50WFjxmMBEfh\\\\\\\/BaUX9Oq+lFzlzXtptM6S\\\\\\\/gTtphbcV\\\\\\\/ej4kjzqMuN3zAKuNAWFx5Uk7AsYF9zrx9+DPiuGoilDtqm76ES37TZLbx1Dv0zIA9gGJDeSKXaaxEChvDrbOO2eERi825oe6FrGasU16CEfsYszvpxzozQ0U7YzzUjgtRUB8zrYO5yh3JUJwVoY281cZge46jjuwbJp6SOuDCSfpy5Hmhv96sUB3Lw\\\\\\\/CvKc2AzA6jvtjvAU3p0RaWp5RzQ+CnwKkav2vYLDRJaxBNA0qvO+iyPR+KdQD5X1rbRMOqjnVOwGn2zx8roNq+6w8lczE9EGyQ0pKA9R9lZLMPCks4ro125sesEnvPecs25VU\\\\\\\/MoBX3quZX9Yh5LZ4dWtviOf8EVCMPBDD3iwIuY9MBwR0ERfH5hREBa0AwezeQL1mNLOhztu8Wls5maQ3CN+NLK\\\\\\\/TF0C03JKq2W20swrXS2ruwfPIyni8L4kLS+jDXMnlNAlFVrFbBLcg3JDL48Ug3DCT1qPA8fx24Irh9fR3p2tTq1AU5CjGH9QwTfQW5mBAKHxYdSA5fEIMzRlMpg97gZ1h08lD6fwR5uFb2JHi9r2KFJsaUoI2R38pWGvrmkpghldSnOeTJLQKlPw8yVSNeh4NvnZuCjv9ZxGz4ODIfOQkfIwxBf5b02r+d+JyMHDNdeoAvbFhsrJ9m\\\\\\\/m+8lQ\\\\\\\/hzivPo3P\\\\\\\/pWycTYjQUWJElVtAhi5Lps+LhIBOOa\\\\\\\/BluZK9kiuAOn8KA0YI8fHul+6eXxP7G+ih+zYhj7J6+cqTU1e\\\\\\\/Yaex9IsaWXeqK3psKsxBwZtETITWmEUazPSju0GW3\\\\\\\/BnLvf1SwtAokmm7fZdRYLX8weyc5NeItDFGPlKCTf9V0HMajPCqSjaPMXQ7Li\\\\\\\/FcR9RTIsT0lAZpG5daz0ipP1\\\\\\\/8e00SQjpXazngjCnAKPo+e3vi2iNy\\\\\\\/6vOgiMJNFz1StF421qdL901KRHANfjndGGtIcZwfWFa4xb\\\\\\\/HWDxgWEOaUjr94O+CYoxRc6+YHdhs+NPXmtfo9yssaftLXsL\\\\\\\/m9Ts+MgVoseAt6wfEptt34RVtd0tCwOF1WXiOaDli4Cl62zc0vZYtUpOfruRayqveacO3sPXZe9NExCXabSZ0ib1rnAn5jP5+A3ZpMjKXJl+W2GJGTTqw7kC0lpICGpUfIiVCvdfuBVJOxVjxAID3KvDRdz6VqzUIzPs0VIZNKy4YDZl38XdCqCyoavBOVydyqtDd+gclhYvuqnj\\\\\\\/DZdE5fJ\\\\\\\/gHfRC6qoOOk8obLqELS0csQhTDUkJ92seKA\\\\\\\/YHFk3EINc4c6zKSIViunqXT07jAcHxfpVnlgv4ngQ0j6lRXq4RQY36cEHkzhZebsYaHW7nUqEt4qofyOFvyZ1vYw8wSm2RteIV8z2Li0PWnF8u8jr9BbtSm5yMB8gbUS9GJ1be1EXlhsjZsfVrYzop03CTJJvpooNYnl4NCWc+Um5n\\\\\\\/rsSL007eIYNTBbZAeLQkAnhVwY2L2OJOwpXRr39J8oHWqqUugkC2M4yl3YQuYdbwaQcuyGQHp3oZLWJtWGEPyZl2gTJAqLOSbX19ILzhFn6Tcf2Om5g2gdvwMObqrm8RGxMzuHXzOow4ya93+zsEpPUvofWOf+yPhTEycgUxscRH\\\\\\\/P89OdiMc9AsqMM2nPRDLLOH3CFBQAQJq15i8cbXDqRTI2ggAr24i2qUwbQlRSc7GwT53BKQcsKJsnEy03QYqjr7DD5cI3XU8G1r2roQmFEoXMRwS7KxmaO+NS3NK+M1Owg7KhNkgBf7V6tCBfElNBJ8SlVxNUMmzYONd39DSqCmTWKism8xj3SFGtP\\\\\\\/v+uG0+FcypQIdmI4TwTpQEYNV2eM9736fMe5biCLrWtYS1\\\\\\\/M3scHyWjP9sDoVVCxSK2ThERZIepk4k09ob9Q8pDsv0W6GHPP1fXeIPgK6SxVRhlfZ7zlFtPyb4kGFtpqJaAeDLq8TMm076ZzWTz\\\\\\\/900Zsv82YNMr55FA9WA5bIzEfeGCIajx9JioaZMsMOk+eneae38sCRcRig3cJuqR+bAkcDCaANqieiCckLr4SFikqO\\\\\\\/r7d\\\\\\\/z9Y12q01lDSxczqKni06i+NRcol3QHVEH2RujLGgKp+Ze3ex3JnsHgEC32e8SLluj9XuxXi\\\\\\\/nFzU9S0aC8O75RtAISwMDZTcC5ApzDdEzbj3nU1MLJnv4HqbDLhP4K9qgztdCu1GcXY9b2ROygH3IWGFpmaFd+XmwVeyg3aoxQldDP\\\\\\\/gQh3f47++rkCwXU95DrdIAF\\\\\\\/A8lGzi8kzr7suGGrSMjadxr58fMO1sJcZLy\\\\\\\/CGVhBVESG\\\\\\\/bPhdmBZwa5DIT12cL6kSJ2k\\\\\\\/u0pMCbNTORl9YZ2nqj5wmalJQwAjFkfsuSshp+GQUoacP3xLm\\\\\\\/nj2NC8tT3c6+oNSkPazj13hHA611ptC5ISh4OO27qiiqpuh4E3SOwBs+uURgzjymx\\\\\\\/11Du11LRF+ctkz7UMFqmOOaoKqEDdg6crABx+sm1SdLE+X7dW8tZShc\\\\\\\/kYrzzgVUe9gvl6hUSr8uVQ1kv3wQNLA1ZOgDP3lmGx86FWIVfmQicdYfqu4ZS8DcT0U78JTWKrUDOLruz201myjfvrbqBWxiJBTfZrvT72owfvsoRyMnyETHHSRDXWJ\\\\\\\/kzj7GmISVvsfDtUY7szro6VsS0fEHQOzazHORrIFeTsDRu6ntIz5b3CiBOPchrO9aJwHqmXW\\\\\\\/uS3Ev\\\\\\\/Lgoq+Z4iAYKl9AZpWIH1GnDwfra0lO7SeRrAxVKV7aEL5d1sc6+SBnlp+Bmu86n9nV4r2E\\\\\\\/JQNQf9ehBraFQRViFEHFiudxY7u44xHkZ09Zlj3DdHoIXpMLoVC03kdjUho\\\\\\\/6\\\\\\\/DRriwzE0IbCO9+xmpwhrR13qoDshWerxg1+nf9LRXT8usSbltOfP\\\\\\\/\\\\\\\/bzqgttHeq4MfmtgoMomYQl+FRb+IUUZzvX1DOXPFOEs9L7SoiDCLGVkdzWH0NSIeiI\\\\\\\/243ecSm6l3mTZCtgKieiqKxsoI+CRYjmVOqOgo5Pfmzz3nF3Pguap6iPXfG53npVb0lf+CYrgGbZmNsIw4Qa6876uDtkuvQmCitfSDQHv64XSaqkRB7lY4pPmcdc\\\\\\\/cSWi06WZ+A0M\\\\\\\/6Z05XBmZb23h2mCxrQoCpJRx1K9mhC91Qqyx2w15CclojYM7Ndc5aqSNBkzgxdCs1i2Nk1wRxwiyBWl0HM2tlWCeC+c5ZN4FHbbYiRqbqO9STZJzQ4OcHSRSbl1lnqgP4HG9v0b8dcdMYqXGBpeU\\\\\\\/p8jpeQnUllqy8IkK7xuHEbfwGVcxsyHHPJpkQ8rlxSUN9WmuBMnVY0zOjAH4NRwl1JPZgi0SZqyQl61sBPDbKOmxnHTlZ66u88RNpRoy7DQy1IavVKm30VbQo1LJahJ6ji7sWIm72DwVF6mLnKGsYhxmuP5Ci3WGHOuOqiVH9lhC4HLui6AE6t3dz8s97eiMUEM9yx6priroE7azc3JKnFmL5f\\\\\\\/SH8ZwvRyBVU50fc2ONEL3YHrpPfJuTGiu1rXtYFe0TunBxMIH40ZNZUyefHkZmL77p7OOvbJO5ChbZDyGVZQh8IUsIxNMis\\\\\\\/uHrPrUn2GviYobmCwNNQmhdsNLhoCNxpMlKHiAk1gtv2zkMgsng7nT6w3hTNQEBr\\\\\\\/tjCdC8\\\\\\\/CW4O4E47188tQ9Z7YQbaFdys\\\\\\\/vB2RjB0i9RGxt8gUXX7oC+0W4cdtbu\\\\\\\/NCYJ43HXfRzU3IXzdlq7o+0weQWk9HVCfRd8gbSCZOf8CEwKjB1u8wLFiCCxBTvBT7S+pABWPtCrwkBeDGAS\\\\\\\/mZH6rGQ8p8N1ZTF5xPA36Jl3E+SYAXUqXbhOz5crqALBtlG\\\\\\\/Q2BUir5J9NgJfCdumF3xofDq3umIX+Ia5l2\\\\\\\/+JU+flePNJlZNg\\\\\\\/zKHBLtJQ74swL4cPlpSR1ESCvP37UVz4QBYJ4SZtms4Lz17C+eNfNDxSeZVzRamiA0Ce0HjRa2ieCyEyzWzcd6WF7fUU5tCZeDZcQyBfSW7Up2Gy\\\\\\\/t9iW51j6qkyeJyxV3csjW2oM00vXYSb3\\\\\\\/IVZer5mB2RJIpEcqvDaRd04zHgGxFY5sa3rFMcdo0EeZVw0Vz7WFkuNMnllt1ibhkOs6lUvD8AQLe9MhTRNyQsytBgU+sDYjRwsgRmeDSzdwukEkL8M\\\\\\\/0HDtHxEUiwwi5Q6pZtkdOoas\\\\\\\/+N2P4Tbxv3V36lftFkwQdV4xdiUKb9VMTVcGAA5a1O\\\\\\\/Ci+vJcaYlImkR17HXoLt2cltGbGT0Wt+iXvO2aSLdUOgkwV3VgNUHARix9W7q9dcNjfSTT9pqkEpVKIgIL00fvJPcQcAZCoGW3r0Rz2+VJ4rvUODm3p7DfIUN73a2KSgX4PX\\\\\\\/mTqcyYW7eEWInpBVC7i9dLF72ng3\\\\\\\/F0R1OIdC6RYMvzBTfam493VdeQpazhChz54RXXuTg9ZwckEYS+dws6hzm2eWxgtdydD2RUlsE7QFJQxjNObMR9Kp5P5GUhiTKcLy7uCnv07a8+MwGqHF4SHcEUFSHeblsrHaE3zxj0DaQtx15PcJ\\\\\\\/+EZuN6CdsmhsXGSVAyJbPHYd1GlCUYgp37SCL2j9VcoK2HMbzfKJQB9bs5FkIHtGDp291aoYPNnAKx3Jxr4COGdEmsdVjpDL0lb0gEnAMGmg8kxY715qKxcJUl6+xoeViLJIOXD5YKzvVJwrWjihHH0p2WzF2k9wF7vvtQ6HAcMQhRsVC0io\\\\\\\/YoNrpTEivmNHcV85fuLM2By6aIagAu6BFISxqNvj3+BYHkReMzFVKjHqWKvBlgbljajyWRI0st5SH+cmtT1ivx7dHmF40nZaL7D6aUGa5oRlfppGojXf3QCaVHxxYE3jTj6vdWagx0ZTSwBneNxh\\\\\\\/Wxse8h5kvoH6MdvTnLqBe6CWF8HGuvORkepQRo+Y3OkLKepDzZkGlJ0dc2L6ZEwuD2fF6VCW6+qOGn9rTsLxXOdXAF+xtLMLxzro6UvmTVosmrb2a441iXTHAfdoWKgkUnDRWkeT\\\\\\\/Aow6MMwHx7OvfoRplM4hkXCWojVFSf66miopOevDAjvYR6Ran\\\\\\\/sfjaS1iNcHTOqof54NrdRikb0TIoFm6FF1SzATRfd7+97sSnVTPJ60riX\\\\\\\/Ev0NkhUJ06fbRqHni2v7hziZOBfrkmXRyEoNi9dHSzmjYNfONVojxcs8IDzvmYRD08Eeq8Buh+8hQ6xN03ZTwZVmGHWCSYg1Hy5+AdqrPAD9ht8M\\\\\\\/72oCvUzTedUBVP\\\\\\\/F1KhJWe2Tb9td1yX7mmAxVXYsckM4sY5Y1TRGeGXg3eBKQi\\\\\\\/t6zia7NHYeSG+j3iHbhMD0c\\\\\\\/Dg251V3XhTnG5Y2xPSPEUymzf+hgfBq4n2GSmNWsLiml311kIPrsiOemZSa4b8oHhSUqIGZTDX2csE5mzNk36tpBkMzA0GLUxfQSg\\\\\\\/K3Ghmh44iLxm8jjrnt6wEZuT7OSfy9CTTxRjqXOV3l2wPbze9eDIskFSwe4ozD+LmTbMs1pZwKsfB11kUtwVs9z1QeW1jxBWBn7QA+4olSxAezu\\\\\\\/d1fRz3evTyD7fCsoj8Prhz97LJWyq7ucCMoh4M\\\\\\\/uqPX3SxDY4RucDQPLGFZD87P4YMdE23zmYKon4x6wNO+Q0dsQTpnih2+ECs\\\\\\\/U0uKuUc967ZHOk6DezO78Lx4InyVkX\\\\\\\/SYo8iupidEIXn5mYFHqLeA9QEobhi6wL2cKwk6+0SEsCc8m6FfHrdf3PBrYTrCKyOyx2\\\\\\\/TdGROFWMyuMYsNFhPbKoVKnTBCTpyo8lDnhddxMokeIWUgOzKdKaOmIrYrJt4FWgEDwPX8\\\\\\\/sNG4ZPjFDU5sTfPwSjAiB03iJG39wagNDr6L8wsUV5ACC8EBQpu\\\\\\\/uU8yVX8ipew3eLJmFxRjWxTsru0KelPh9au+u4cFZ61cSV1fzT7HpXIJAf0PJXoGEsrGg2fYZzHc2l7u6iXpN31KGDvSFGiZqRX3hGXI1pK1jSMO9uuYdfQgDrCeoWBjOYwHMfLZw40wR2N+OZ0OP0PqW+IpKCen\\\\\\\/ZADDFU9rpsUXZ+zPl7fPzfTgiHLxIZUxSQpMdQU7mL3qkmwwjjKNiO9O5RIA\\\\\\\/Ie3MotHp1jKy7TkbZOsTybM3Q+bjdCtP4AKQTh0ckueApldp49r9BUgSQLZBz52fnyK5q8k\\\\\\\/Jf5DIXPn6MloP2t1oqUlgO1EtQewF2i2fVGGJu80vrZJFhtRnnMSoR5sxiyTo08HAuUCrp\\\\\\\/xL2Z3kK1mJuE0ghUTm0sBEYgVqY8kwDjv6SaSQS9Hh4hyhGZp+pGK3TZn\\\\\\\/0xFGK677HR7LbKywi1lInIqbLM9T8VLPW9AzpZMshwcQKcrQy\\\\\\\/7f+Y0XAAiId61BqmBL5jDcw\\\\\\\/rvlgA\\\\\\\/HA7q7ejdpsIKNA04S0ZSH9XX8oIRlHWFNRTehDVYIGKX5iBXKwFE2jbCTacbHZrmwCVS04VG6lR2GQ6xgMDPc\\\\\\\/p49v05f6pzZPw9Z2g292U4DokNJML4qCNqSd7Rpw49l2MSt8pqTsfKXzmuwRZPK1QXMU1xcJHCRPEW+PVircFm2FVf3hRoq1rZGg6Y6j65hc6Ospu3pJiMqVdb01B5qC+wZGdPTM6TIdy6+cDh1BMIh1bJCvlrNwj5jGSN8dOLM7sxkbQmNPJ5Rs4ibhE5x\\\\\\\/3jarwNXQHc1LjV\\\\\\\/ZO5qeFPETgjmyJscvP4z0OdNf6TdjqigMzU5ySaTNhMevKAMLX7Bbvraan6ATYanyT5LU+s8kX33S6NU\\\\\\\/eEkJ1h+johU2q6dW3+f1ejP6FzLIylFlot8BO5+s0QLkMtuoQpJFeuTAGerjZ932GyZ7v9R1QQS1h0ujiVvdgnN+iHJ0xOX\\\\\\\/E0ijiukHbpFHPdRKdOpvyZtOpwDy5qieJ1z0FVtoR7oEuLLnmEuYHVJ\\\\\\\/GOiDGpr89eHCG4o4KAnwV7GFcMQsdYcsS6oqwzny7a97AVC96iXAurXUnTDCGZPOM9a4EahD4yB8ohXeL3yI86b0n\\\\\\\/LkNfvgwCYqdQxoPTb9JmSQUgULwZBPYZt53Ngw9uTUZ3aWMVGV54r0lRc9YPovJ5L7h3PC9doC6aoQXuKKvBrum1w75DuLz85FGZgXAtoiQaUQE9PolbG2HJYF1x+ufYJvbUW3t3RSplKxQaB1qdxKNXuW3aIPg9rAzwqtFbm7tivS3ndcUA8fMk2oKfTF8Q4lPwoSV7mtd6dJGTJpOul7MFtI8L5l+SdDzPnKFKCUy6MUsXYd\\\\\\\/+aVdNUJn3ZHneqizHtdbW8tPUZg4sSiwXPZL\\\\\\\/776+NYw45svjXo5Sj0F5MgFBrD136xl0vk3kSR1WizX4Y0QDMu2XoD3siLpe5dRX2WAV5xYLyRXajU9saT0BUdSZiTFX0PsW7Sdb33nNfKsM34gY0tTaufrBCRVirz6KdVn\\\\\\\/H3S7vsQGPpybvF75o\\\\\\\/ToBYampbfIscYwEAV5IY0wSZX1hwp2QwKtqEcEYgMt3+PuVlh543Y1lvyX2MRVHNU+bVHuFtWmYKyEx2VSlZPgZKk1unT5py8UOGC1xcWAzOarTgO\\\\\\\/1epL7mYkcHhqrXLqmozRmp3387xtaPQOxt3vTUBRCSqSL4ihPAMEmoEA\\\\\\\/AoU\\\\\\\/QaNUlpYEfAK++Peh3WLUy2Nx+Z+aWL\\\\\\\/4nn3ouFR4K3pphLgsG1MkItzAIlyjxrCODBefu5ymvZ+p6QDfsxZXEKL6IAcO1VuU2ZY7v7dQ2aPs\\\\\\\/p01oeBt1IJXMaE78LWZrJQomRIDJfOdXUVBnL4XhwjLr3HIqLpTn3A5BIQrLQTIiCpNqu1mjm8cdLaq31UQGQqcB3UsV5NbkuGCp7twa8ZJ7b8To279wsfLACJCyXmG2CIV54aFEDteoFiGggEPEpE\\\\\\\/YKy+dbqY1ZYQJjoR8jTd01JF7a33ipZ1yQpHUduBR+2CcELRfiO\\\\\\\/il5kf1XbE0RcsoJJnBZqgh+VESeIPUUaZvSPxTY3j2HeDEIFxoPsvis9GP\\\\\\\/zA86gsNYEok5BAUJb4gGmsRqOzDeo0JzbOwFER9eH9hCkELPQaaM8b8nCsPe+x9ugJjbhAyt+odQcqDYjVry4MFItENWLfZ1oX2\\\\\\\/woiSctGWDD8+HQXcVo5NVa++mu3Ikn3acEE2yoaSwEQt7ak4gLQG4uBmpEIHISMTgg78FICgDPKO6LoiShuRNF9iBG85hIftkKM2i0NY9YKtxbyDF6UpbIIEto\\\\\\\/QhHdwxDYgm6EW1nF7F7XQe4eb4B0T4L8IM1AGf6MPQWoWs9u3qIiSerYwMr29HarJecIKJrJRTS9vBaqcQISrAM1ZbUJM6P\\\\\\\/U7VYivCRgXB6d5kq\\\\\\\/qq6kl8hkEy6O8LT7WPShv21F\\\\\\\/Kn6CvPv8u\\\\\\\/T\\\\\\\/8IovVNhNpOSvjwIMi+HyAzoWj4GIse8oTD8qLiFuuTC66D2TrjFzTxX9fEAaaeZx4fLUYDulxR14vGdV9Ec4vabJz6Bdcz5rdFNOORaR+mPC5BOHegYy1zJV3NEHb4nBSos4HWuerFSJIQsnC7eDO4zCuEHlDk8jLzvOwOOyj9d\\\\\\\/anuggT75\\\\\\\/JjAQ2ZSnxVZDg\\\\\\\/guTStYap+\\\\\\\/dvkbGaflzYH96dtOsbcL6PnLrgl9Yi7QUsptUqjcI23iJQoFVmEu6hgNmj4261E15ldIaj1ukgBPu34nzDI+JfRTqgxUsPm9StN7SbHviTsRADxWqeGTLDNmnpM\\\\\\\/davYb1cqlJ9UUSoeuYu6QTCAVx5vdP6ksClIDaCuCAt\\\\\\\/aZyBuIhy0A8dSn3VChV28DzxRv6J7gQPrK\\\\\\\/63mkm5CS+UUPaOv3HRKBeW+RmgrIR9o5sLEaJq7nxkRT2mic6U4cFW5aL4I8F1nQk3umlUZvRCDhMfWCgk6IcR6PVUXrquP65DK16+oxvkjgieLEG9\\\\\\\/hEs5+4s+BGm9lvZso6dhmKmfEovIiJctF74Ivuo279BFG\\\\\\\/vVHniYMH2U2Bc3cY2DONTE1wJ\\\\\\\/L677RoWpi14lQInq2R1oho4TOD7kcvkFGvxhxntG8Q4TubzB8KYoi3tgQvF1DuAswR6mw0FrY8UoMsZk6Wd7PijSrfRVS7WUPRpH3q56zB04OCV0ACUNgTjrRDDwHsBEZPS58nLBfEqwM76JgDcLXHAmv3zgtEMqxyD+oiN0HaEJ05H4yQr8A9ZCjlKlstMTQEkwWgD5MWsiwRoxoU3Gqota8\\\\\\\/2kO9S7OuSh4DaUZpe7ctQXO69rCzz66MitD4CRfPI287SwFMiUlXhmBFp6QOtiW2hlo6aYT3v3hc4l1DAnKyRK+4hJuR1SgyuCjDJrqrYFbsWkTRr4V9xwx2t7LpCSruLLC8Atfc\\\\\\\/8b4R5Gg61JX\\\\\\\/nOKn+T70LIV8HxJhZbd+qrKaeZusjkpM5MExQm5O59AVWGA0A4NyMSiByA3anQv9Fb7GAI\\\\\\\/IZ9YKsr7YA3hz4RiMSoLxtKiUJGPbkAvHLR+2gyuywmQBFfGUCR7KKI58XpISO6\\\\\\\/KlANuUxy5jG+eAt21BC2d63nG0FgPesA7+gwG6pCuHn15zzNJhbYRcWDuvHAq5mrw31zjDmp9vzL3t04sXWg7JiqQNDDPUkR3AiwGX6V25aCNZgiz8RessFK40093dC8tF2zey7LZonAuDuDqBXHI9vs3rjCKNs9j9IzPn+Yr\\\\\\\/\\\\\\\/f6CheF0XAawMUMpuTz2IvGlfLWbwu9r\\\\\\\/QPYyVXQPT\\\\\\\/JSsKQoQRDX9kfxItRBKgbhNQ4QOQFLc7ObRD2TdxDtnb1+fnoLRlt9PJcKUyJaAlIF0VKX3LFgbuj8KW\\\\\\\/F6\\\\\\\/eF7KRTbIW0J5ic63rKl0SRK+Up1H+v2xL8Xblcos+yoh5iSwDTcJ7ofoxx8bqbX6o0l6QrhjCUmaN6BEYIrdUQkh+6Y0vFDSEIQh+3J7Hft93\\\\\\\/Iw4BKCtUIYJRXQXEfuOwSDPHqqD3xrSeWtNHZ1voNHq+4OOxRIU\\\\\\\/vvRXmnICi7J49NUBz\\\\\\\/osHVb5XY1J5Dx63Qjq1NnYE89dNJ\\\\\\\/NzAtEouRoi1qVNaaNsUurNMCqwfZxpNAjtt+Wn7\\\\\\\/wVjrPs2s7vAwjo6Zc5cKOv1UER9mBpYoDyzseMtgkLY\\\\\\\/FgbVIJYhNOmF0\\\\\\\/+V3wIZDEcm1E0tPPWy0XiIUxl2eExZGJLtq\\\\\\\/Y2Mlnv7usRxT98jfb349DqD9EQ+s0apu0H6eLcEKObKWfXJrLKHvkVyE0vIE7dkcir5YBOWtWMbY\\\\\\\/9J3\\\\\\\/qDHi11Ae0Wl58Z1QDGuNGbTNGsVys+8UGfl\\\\\\\/+VEDNfFnOKGfZyh95gpsetWgr+MI7bMERlTXed9ealteiK9Ej3p774Gl4V0mKTDVEHXGSuBOjWMhMQT80PnNKLM1sQc76OGxH\\\\\\\/pjcJrL1AmKGx2nfKwB3bTXQJiDNzZGNqf6+Tn2Hb1uC4fWd7AGrtZYtmklKCCPS7fIDndia6Vl9gFERWuNeT02qNXYVX\\\\\\\/P8tA9Mw+3loKYHloWTx5so5HZMvhOBz4yItVoXpmU6rHPmv509ZKzpCXM3wiP\\\\\\\/EhiW8tz7bQNc6HNe8SNpRfNcSSl90T7q5KlYuNBsQd5MYmYrJJatP\\\\\\\/WRlAHE4JJPgP6+Avpi6\\\\\\\/BcgwUmYtKAP1yf4dcc6Pi+dGpyzPBrT\\\\\\\/aE1ECM0jHcOjvixilWcX2vDPF9uZ+2pcqleB7iDxkGPx1WeYPBqtjBX7hjjnomz86n2iRpbHQk20c2624w\\\\\\\/n0g3Cyf7sDtNx\\\\\\\/KY0JOg0HlfqV3zhS3Co1TDzl9klv6zSmK6VyTOPxrSYlD8gFbKutFCU6y0HkDPI7kz1ehSVlXlv8ORq\\\\\\\/ZBX0tx0OjfgM6FUfhMBeKh957xzaGgTaa3Jdp58dOZSXFmSQVb9nzxKQeBgFkujRkOyPgEPHUsovfeHZWaeIHhO7VyVwm8Cs+gO8LG87YbxOrPIlpghbRaY+LP\\\\\\\/gKGMtdo0+07TIXnvqOokx+2PQ7BQuZn6gu6UEs4ntTuYGFVfeWY9ZbGuU8YWurTAo9xXmAefJJ2VOWo\\\\\\\/w\\\\\\\/q0djg8KOGy9e8devlRyA833\\\\\\\/KySzWEHJyqf9JI6fhVBMbutGr1Yk0xU3YaD7JpYkFkG6N4FdGJslzvFjbvaTg3BXDBri1\\\\\\\/XFGIS5l2KFJdsTo4ncyoKNhaqc99AL3MbHxPfb7NK77sPRY2bcDhW72f5FB9qMUJQzQCZVCePAX6o819MJgG1sdUpRTjXA7P4ONGQWquEQDhd8Os0NtkiPB28sUP0j9xiFYDbZYQhNkQr69ALNFjHen8JJofgPV9R4O1lb6bgIhWaaVEuex2ueFsltpQWaB0GLoLuxanHh1VY+Zxr8HdLADzvbmEp7k3di3d3\\\\\\\/cK1f0g8AdVlmtX0U+rSAACsGUyWYb5dRPJ7EWmBiIQ\\\\\\\/5BFizHcfIGjVCqPI\\\\\\\/fHprPYZk6Xt5MmULOZL4k3irUP8dLySv2jOlrulsQg9OrPW1YPBEs4DHTq73up6BC9M\\\\\\\/9kOyxJ62YEssAdVpTpZhq9004VU7JH9FYJ+BJ7l6KchauRh1fwFSrcbLz+DwBycNUkbqMPPYFwQ9QrKKr7RgCuuodAoUWRxKKKJauAB911UVq92o6oAyTgx51oYaRssuRYPxPEq2Ml0Il3VRlhrigMBeyVlq4N21126m2DLo4nvolmVEE0ylk7l8UAwB4vobm2kLb5AijlGdPRKWEsc0zPu1EnKoAfw0XiCqiYPwTkyBQnEn\\\\\\\/BXWJXLumXAEx6NfrX\\\\\\\/JjXmMZ1x0Flra52jZbaReJGsy5Fj\\\\\\\/QFkEF20+GYbKFBLRysFDuSD1HQQx3Rc6uA52kao9ZYRzYTAWwoAu4d0XNu1JVCyulIi+uaZwgU6ZZijgKlPbtVsp+QObJwQZlgdzDzhsn947itIE+xbsmdERFxCLdRlm2fBIBG12p5p21aZwKYJXajhuZLhAkRgCTfIAUCQ+hSBv2LN+k3KD54N8PFncGHrfGvgCJEfuqxn8ZxWs3ohkUNvKL+1gOJaBmjU0YUNxJFkx6gx1Ct6NDpAKeWJcn4jHNAF+g+eEutXQLiApPO5W91xHClfW\\\\\\\/GlQTxeEDSw7NGAKuMj7Y750PXGLPVwuzIA9X5GA5QRk0rVwm9Hk57ZKCbG6SivBcezzWhOzt+8Qp3TC8ii5BkCWQHYQ8qJ4sZq+ILcTQK9vwn4tsvMF6MREUA+xFUl1zg8cTm7H6txexY1D1\\\\\\\/+GNivu07nFfDlz2tUxfSMcnpeoOPYV7gZAQ8ah8vePecBbWyMFVAY+6T\\\\\\\/lySy4VkCY31dMrbnpbFrB6uT5\\\\\\\/w+WyFDjkxbsyuM5mw65H2GYgPAn3AUw\\\\\\\/A3THMnEPEJBDvxNgPQIOUL\\\\\\\/zG+dcrXQnNwx6yC\\\\\\\/zedMOshcMNMhPFCn89u7JmaHOMuECSrn1HxxoZyj65Np8lxpEU9JGvybluq\\\\\\\/xrKjVkwLTaYySFleESX7+CX0XFYwlnuerWS5Iu6lUbRd10axmm7JA882ZipsexNwSQfWVC8qFwbkWlJJXfxp9FNyp2lP9EJoST30IfICSLhXXi33n\\\\\\\/luGw3HDLiO0Mfryk6doMZBKMIvDdY+3NRju4JbKpc72GEqV6VcaF4U23RCuZX50R701hGRTh0I3UMyIjZFNbYifQXnbCrr5W9evNXSseR5dFfapqT54BukQkSZ7eTtMDms+HLEhQNo4jlW5MGq6Gs\\\\\\\/HQy\\\\\\\/LKzrWhgaP2q8J0FCGopdoXy2DqpazqlRWRyq2zs6pZJpOl+\\\\\\\/sFHUrsMQBVQonvcei3KwFgECfn54\\\\\\\/v7oVa0ZCU7X2HZtwvqFG0i0PhjZjHVKCJImXNWObZYjMWtahiiV8Q5sIW8NzwLUJdwpmXmY7iNks6HniQyupZdwc30T6Z58tT\\\\\\\/mjtsKE+FlN69nPKRNz8X0dwMzlfVvnM5Wk\\\\\\\/tIB8akvHmeiFnT29IPxg6orc\\\\\\\/JrDOe4ovX5\\\\\\\/f87r7wQG1FXFmvMArORK3gznXMeIfyaZF7kf7E58pCk8ttml\\\\\\\/5Tm4NcaQP1XjIeDdQjkd2OHlD883y7IPAj4+WJRjFN+Zr2WXKHPBgWFedSiEMrTGd52CLmpFpd6Bzm+S17pM8GJdBJVHAml2w8Mr8qhaKfgyonH+KoEJsT27kr1jtzKIjN69yugD2fqA4YKyTnkx6TCx5dRAzcAR+twaqspXbJmOh4\\\\\\\/wH3jslHsOX7zV\\\\\\\/836XjTzq\\\\\\\/KvWghn7DGMQPj5qbBqSwUQGGCQa1DZGY6XhNDJzD\\\\\\\/oms6iWgRzMr8r7ibQqcGRkPG4MovWWKKCStt8d9Q00Jw31EF\\\\\\\/IVuEweE\\\\\\\/f83SxCTVz3DtkZYQG8kHBcvZPt8vCwH4M44\\\\\\\/19kqDebLCjSzyh4C2m8dnIRGd43DF2rsDvOvN3p4ibjos8XZ20Ub81qjSkolZbTDDudEMqtrLnwwrCmKKt9AfXXxKOv8FnlkQWAQv\\\\\\\/RfbJ5H\\\\\\\/nGrnXXRE8f1dNYikQfOks+2M+w9w10usnctClHJr9kEeYFwEPMAct002KMb9hDpL9i\\\\\\\/rj16yopDCi5NVLcLXwEHQxXvTuDQYOvZ9I89r2NcEyn9FD5tpJd3E6KnjWa+Uzs6LRpKkLN9uwjigdMjwce6I58X7v7wUUKzbtgH19PHXhzujHME2QBkC4L4L7okHYSlvK2GIK4gVLswNUrKIchFHtn7PfWFz35z7pxx28YR0ihyfDEA6wgh+nmNXJVWR4IhuauqJlDc+KNXrIht1lmrxewwYcwdJQORRg7ulgPhXtFA\\\\\\\/ASJxXkq9A9ypwPmp2lhw3C+JLsWJmR+Fw68CKcDFBpt1YceARUsvAZ6fTFnsqHyYUBjIte3TbV+AShuaTsSwmNhFrEW2A3Mk0o1ef7MnCRrZKxLeqMhRuG74oDzHntZzxrZaPYsrfapnRNuzem\\\\\\\/7bC5MJc0psNLUmwhBdpXdbw78lLoeSk9PAJktY2JAWmX2FlQb3cjGzjt4UehnSI8J\\\\\\\/PUm\\\\\\\/te8oBM2tk2J4IZ\\\\\\\/ZI67+unKNdPUWJDa0IGzIgtgXqW2ARppLNUje1KzjJLZXucQKLWoKXr3dM2BD8pu8OiVGKXxj9Oz87a5Ig6zJvZ30tQyyfVszRKkWR5E36rjXp1tV5oOTt\\\\\\\/IuI9rtEFP4Bh37xcjb9Le+OzfL7zAL3+dnCC1nrnuB\\\\\\\/xZK5cxza6EHMofONIZ807B+QDBFvKixjWvNHw6ByPEuoTLcVeH9UlEn5nH9PSJ9IgmAD2hAlSU0qFVnnFfLVc7\\\\\\\/4pCJ7u1VXrFEtvLf2uAI2tOxcp\\\\\\\/GknH7emNIegwsQEgW1OVw8PqIJLumSsXZOj5qsuG2p2GHwgB+K7hacyHCbCTvfmSO4z2uDm9OLRO+SziB5RwzjCzIyc5I\\\\\\\/M6sSzuGC5iy6k\\\\\\\/IcgQ6shiSd7k3s0wXY\\\\\\\/Gp4ASKczj91bbaUQ4MkEqKVHcJGth7KMm3YtH4z53HzW3NxLSp8V9t8pZn+hRsPbfIWcqRGQeVyn7GIbT6oNKEc+IHsePM2XBObZyzIowWFBvToDA3l306A2+b1zDQiF4VEoy1j3x1QGr4NgfyPWf2sRdXS\\\\\\\/Kz2syZmYn2ZC2Yy074ADEkESeJ1E2qXNBuhYz8Fc1froWaKnlpAX0s81Y5wH3g7zFNTnIvaNJBFf19NFslIL\\\\\\\/zLxMJr4VZCsFlqA7ZnbkwsLyZgudUkhJiLjdgN+IQHFj0V605AdZt2S6NvPe4JwDb8pmV6nZe6Mo8\\\\\\\/5\\\\\\\/HsPD8rGMuZsmlbBS1syb0Emysp\\\\\\\/snbpTU8Zr+6QQBI09EpP6wGBGGjCsQgsoP5igiqsEtrjWO\\\\\\\/ghZVMXa2FKt7NoVmsYbdkZiVXSgDeZiWcMiRkDUf8WSeY3sLoiZqIXlkqXDnNQlrKnSC\\\\\\\/GWl9lrqZEy2mobP9bTFkoxzA5C28KQ1lMIc96zXKVmJcsS+LaGRWG2oGvT\\\\\\\/5JPmYOUp6GJNH5CxFQNzMrZFQ+sNxIso43lLW6sESExTO18su6e4cMWwuIVWoVrbBLN1lbcHnk+tanGxFDAlwRwVAtRCB1UcS1Nn0ilGyPXxL0k7K9Nawb1zfjMJigqk+8idILKjt5dHHF+Xcj1KvQLjFOXFHStePdgzhEYTFr9B8YCsckcyiY\\\\\\\/bPKbtNjKBsx8yTS7ep85G8ljhgnkZUyOUBg9Xd+iFV01NAIlzqU0ANn77WUjP+TXwZ7jO5C+OWue\\\\\\\/JpDaIbK5B7vocj+8IMiMBZblzibgn1BTlpSIAzKAs0Gc4VFypNGSLnlqA6LK9QrQXW\\\\\\\/0Kp6hqWr5jOX2jq0otirCPfKKOHIbhhfTcEvJ5WvH0UlgYcAIfth2JAyGRyXRivdF8TMEhIkZI1vPCzrBBZSrwvEw0B4T2m+JGFG9dPSjGCYSQlBaemSYopodi4NIMD1wyhdfx59dhufvha1pLcCCDvCvF6ByY4T\\\\\\\/DRgjn8T1R4\\\\\\\/j+rE90JuyzzvbVjVNaVWusRpu05BPEY8EvrX0GNw98ZNYHl356zJd+JOuxMsUI7RLQE6p7cxzZLgLtT4tTUXzoAGqpABzgpgyGyhHLImFMJ24pbJtESkR1gxlAAvcZnrTSJoTAgduvNX3MNWUAq41UVyYmO\\\\\\\/WAmiSBzgGdfi13TfQtrGha2Mg5yVU6WDWGvU7A4lQtv5e1BL9RfscYrPcKmTjMGIIRPaUbSaUTh2UcfGOKVh4ZP40TCxBQcCzv+cPucwRNZ8o7m2xtCL7B9ZHMmuKrnqQsdCMoZ44+MWolyIibvqSmP5qxYrvgywg6JHZFMEBh1d6\\\\\\\/Wuc2vfVkVun4RRNQQU0DyYojoFJbSyi3KgKTGYevz0jnBuL24KqUGmpWQABjdyvtz7B8LGnRUdcEn+wiPGfGsAq4WHKXwjHj2ac\\\\\\\/MWkvUOM84h2ypzedsfLDOy5Sqp7pyDimjut0y4ywe5un9FgrYI5i9RjONE38dAU1aWebFgOvG6FE9UEXXr8bEISyHPiA4KHUw8B3dwb\\\\\\\/2D\\\\\\\/s5xtCI01AR7I\\\\\\\/N56+tkz4N9aTWTsf67DMEQGRLZACLtZ4Ib60JQj2rQ9vnLSXYyUG3Vw9un01UxnySUcP3QQdZAwzhkkl+i2yO5uu4Mxkx1TQBdv96psj5fox7hR8arYgOin6No+eUZDx1oRYC50LlN8xi5l\\\\\\\/7vK8cRliiuZL5SoM1r3MJBFw5WTMPDToBcZptvQzka5iR9no6pacfo8gFlnS6R7FVlNMJcu6Reor4NEglotDnvgiyJ0qps03M\\\\\\\/WTdFjlE1t8I0y7ZkNNFL5qNq1HA0AbGKQBV\\\\\\\/\\\\\\\/GdqWtOq3m5x8SQKr0T2ZyuvHS6r8k2PVjHNa8Mmpfd9WFz9hAb7pOAFQnxxc7iH5ZOf4hp1tL2XD7jI+dGGBHTcS1F948w19gBkf+1pFOzuGLYLl2AoBygozGytV7+8L93HiKZkf7kCTmxoLAVYzoVmEWF+O7n3mduZYAk\\\\\\\/+ur2Szfeg+uS7\\\\\\\/X9paSn1iJq0pi1DjvKjbLlBL12ykW6yGSJUyKRhSJPDeGQ6Uyo4dpnnYgOxy41UZZgst0HsTXpcicZmRIzvatmctzpj1D\\\\\\\/DHSavCqjAUZJyl8FI0dPIf1TOGi\\\\\\\/CW6zTRgfAfmc8w4u4uNNqavR7NA\\\\\\\/cuoYVwozve2DUB2V+YJsmFcaXvGlqMZBk9RfpcEScgZBFMDihgQBbA0srhvZh1lf1g7CreQIE03EqxR\\\\\\\/xGoaVn4hufZFGfz0L6DIW5HlSIAiFrilTdMWJf3Ps9LcKG1W1VdMh+uT7wAKRy\\\\\\\/HwZlPZ2I3dgkyVE9CSKQwsSRddlsrrHxPlk0XjFA5wmcQx7I92Lbhk4oMCnJoAuTjRMbEuPN8aThPaY+RuF8PHBaZj9p5Qp8+f9Cw14+9YoTiNERJ6v\\\\\\\/ZNt64TxfFbMb+uFt18y6trabdOtPHC9F4rJgzfqDiRH6KQ0vGhKvWs+9mMxzolfU\\\\\\\/zBtyw0ReQIrXsU4Fs3fpQOICG1IqIn1mcbBEmf6wn6HjTN0vFueMoQt05oBonlUrRzoXa30p7z5EbTPF0bGhE3FiOJJQHcPE5r4x2JwUGF+u7O6x+lXqBzl88dRWfSh1tT7uJN1+k6nUB8TDgN0aemTPPSVfrMmHGWZf9vH6tTugVC7uYNHzHyy\\\\\\\/OtarKPrSEEUknnm6ejlIKMlNHTqQLjl0fSIu563qxq0UzXCO1cjCAyqiih7zXYGiyeMaldKISXpaLfmKfqFPsKqozLwokbPAZLJQvT7yYPrS+0R+JB09rjAtlPhfZZKXHkZadSW54SbAmETvRbd+oSrkPBfUWxERUIZo8KOKhnYQVZZOicA3Ppc5t\\\\\\\/ZRQLTn++m6HQBlTS5EqToSpdUnCueHZ9Vtc50427tYAM\\\\\\\/DiOw9tNuWazuniHH4m8KZzVlI9d2QLDnlg4wrB\\\\\\\/TJ7y9khJRCt0LuevThV1D1YOsCwUNFl2hoQqS4ufFYmlPJY37mW9ZdYWVdPtTHcsCHzZ5D8Wo9VNm5mJmlbmFdGYy+8MIF94q4eEKmWuoUDmWtqSUp\\\\\\\/L\\\\\\\/bTf58zRhz05JB9sy1BEBXga20OpqJjO7muXzn\\\\\\\/jqZkwGLFnZe0yZRhyzGraXPggSdVRqh0xGxVxndFUg4DftWBmfip1UtBsctTrxbOc7\\\\\\\/uEH5hokkZmWVJesTnufvW4E9gXehdUlD3JclS2cajckJX5S4rJbEsAzNgRn6YtVrY0zXguQ1mYdxsq3T8H+RHc+qMUvaqndXl4VLNQ875YhI1wDeMoREbqf47D46z82EP3KLCdgKaxyOXQ\\\\\\\/rlhvMTUhpEtytfdRFqljfm9J80pd8f2Buk+IntdIGd3IiCmE1DzRH82uL1IcCijFGbkfmSigBh7Q3utzyhFD7kq3xIc8slHXI7TZcwgA3SBfRQPKrMpZ+07GKhQ6iSYKxA0lMYf+cw1O0FAS0Rus4\\\\\\\/J90qQytX2mjujPrMzppeIp3ffA4E6lFu+Qn4OeIOr\\\\\\\/fhGo2fvKkph+2Pq+eLAe5sBIm4XtPeJFDVJwcT5LBqTvlR3JqBsFtyFPxheGM1DlGWM14wEPrmfSzDUilVgfVMlHIsysV8DjPZD4bFGxyR2TuFa802fDnUWX3966fFlGyTmb81yl9z\\\\\\\/5cUfxCQD+t9kzrVJu9LFej\\\\\\\/JZaVYAA9YhUpClz\\\\\\\/kwVR84W91uA0Jwj7COCSLQduOD5S4ZtC05vo5j\\\\\\\/iKjVyDwGHztXHWuV09kTKMMsb5yXYIDdYn9AOnjMsap07x+UsPwjZX4qXxh5\\\\\\\/AC0gZD6KDQ7ursWBMsEKBL7t1pqodPlXsWkl1wOD9a+1woVVngaNvbDoC2OgGKyo2+swdW\\\\\\\/4Hg5wPW82Tb8G2ELfW4jrqtL\\\\\\\/N\\\\\\\/rVz36nwgrYyiPypoQfljd3hcOkw3JBx9cD6LUtXnsDTtRCR+KpxoH12qocVIRMtzD2nHD3beOIiV+xgshnYmkBO9UXb9phJURD5wd1ZAYdq7U+fvKyIykfa+bycjthwVOq62rb8vsTFeTXRcuSEWRtqjqsBxLkNrtLM532kD5TOR3g2pJ4OoRhUz1j0cCQFVno8CWcLyvWaHnY4SFcxhKiQA2ePXIxaNnY2r1DSt1b2\\\\\\\/q65Pg0O3XYJ5SEwVyJtNsLcZi\\\\\\\/KmeiBU+T7ijDu75Q1O+5p\\\\\\\/n7FP+OIZ87xPGWBWlrlvVLsZ7LRrYutBDyXxBwETfS5VQ9hPR1WCrDQfx4UqUM87G\\\\\\\/6aGykwrN8wJj12iDe5CYPt9nn8oNut\\\\\\\/kfXLQ78BhnMFFl5YrLUvggRzscdj3vUQnsoZE+26tGm+HdvEscrUVb05z48u6\\\\\\\/MrODmeYIFOzxWbc4I+EW5mBjiAo9oroi2vA5\\\\\\\/vZHi\\\\\\\/L4HeyDDr\\\\\\\/zthb1WfW9NJApdljxy6tXBlnqNMunWKALyokddqtQ6REKunHHWqCM9FtnsNZWAzqb90IZ09QXUJQdC3MfAT8gMQKLrIhym7ZG1ztP9YzKCpH4PHSR4YJj0tSHo2\\\\\\\/fuRKCLOH+K7IRQ3NRglisV1d2rwROT\\\\\\\/fhPOzu0v7KJscl5SbR6IydeF3COtuETFMd2o66A9XmDo38SLaSxqo0Bn3V953V7oDcbe2joAjIJoaaaz2C1q9IbgW1lyhsQJMRmpKT4An5KXGHD5MJLN9S14gllv4QXy5NkjdhY0H9pIT8fBfbuIP0\\\\\\\/pasMYjv6Jz4U\\\\\\\/jw1k9W1hvswIyCaQQxL8PkMLet6W8roMVyAPulIFuF\\\\\\\/TEbQXS23dKOjYCbCn9\\\\\\\/YZ3wIQ5wdX2ELsCXjGfyptojA7v3TRcIA+aZrUdHUfq847\\\\\\\/gkxpsIgZlJ3H4gMYhU4WItynLaEhCJ1mC5cCHK4869iBLAONfph9017n5oQB3DgOHpnq2UE0HhadJ1jlrvyK0yMSRiPtKvPIOy002YI+yWujXIX71AV8\\\\\\\/\\\\\\\/ii0DUHoU2giS\\\\\\\/IQD\\\\\\\/BJ7mqmbaoq29vZkAFBJutDq0lLmkBPGQ64fJj3mPdgff4wn4D6+NMejr5l81D9M25OvnYQqY6Iq1Wf9f1sikbd5Lt6mYpm4WP3J3M1pfoSItBCCJoz2FADEtVM4JT\\\\\\\/7TZqt1NNtg5xO5GM9t+tmY5ME2LWX16zyyL35pGcM+K\\\\\\\/BFsQSbllA+IFinE6AlGRSfbvluCCDEmXaT58h2IaLvv9UjjG1RMPt4oGI\\\\\\\/5fOoghHZZWJxUm5ViuBl7CwMe6LGX0yQQOhpz+lK+3ph9dmLnx28Ta5v\\\\\\\/4GwE8+\\\\\\\/Z\\\\\\\/yf0EVwvgq6UfFD8CFnHDz6JqSk0p2OeiNcmpUrXTuW5wSPYYrcMOTtDcFl0DpDB78VjwpS50CcNWmy6nmv5nUA0KdJZ7J\\\\\\\/tCyYzyS5p+UHCtssfpcBjbHFBkaJw48gmF8f86yvVLHq2ZS4i53qwcPobJ8Gcn7ZkmSVESsyvtAOIuHbS614PkdlmnBNHD4YpZFwNCLPg+1cUPzbcWLk5XNZtDZZpJTwCsVK0fygv4kuL5BNh29jFxhrxOTIlrgbLEz8mj9jDUl3NIxRYzrAQ6MWe1U3BHhtn6GqU95ARajz5o+uoMcTGBPvlyAxE\\\\\\\/9YBV17s0PQsGDEZcjGKMYGe9edI4puGh1WHOtRMnVUYgakWMlOnLbSCsyJGG1PMHJM+h7fv2Gs\\\\\\\/olkjR2sxL9ahwDPck2se0IJ9x0a9Oi6lFX3TA+tmrwGzRwWcdWjM2FiPbFIf5W6\\\\\\\/+HA8pS3eMdROgrCGTwFGPsa188S3YFfxDermPzDJ5hCTLkxr59Y3vMjyOxlIq26J43ZIKIH9NIPvlYwj6pp7n7HnG3TsFspfmw4WzdnjAy5wEaPSsVZX56lvv0SwoUVMWGi6RqJlmf7bzaWjx3ixgmn+kCJ7CYiza4H71QsUBXtarKN\\\\\\\/LNAzddO3PoNf3YogqL1VIuQOWJ\\\\\\\/4rhQPaOI3qykNWYUyc6Fv3QL00WyJzSaZJS1gCCRCZcB9bQ9nqQ2Vc9VO0kJmiE57LbCDJfOxzFozCf+b3RsDXVdKlfZF8lCWl82X7poddWvcCKefTifIkR84W1LncBMZ7XoHDqApYZpWOGCeRaJ0hhmZ7WfQ8tpbYDpCFi7Ao8vlfth5pf9kJFYVE9FIEijKXQ\\\\\\\/rJa9ZsXkxQmAYvbMHkgibD11Ja3DvIBC8E+VQz6ofA6KawP9h2ZL8eq2VdGx4DTMkXWvsAQHCiPUwqXIeXRVv2lwWl2wtQbvPIS78OlTEjIEJo\\\\\\\/pZdlzJnVIQdmlf4DDgD5gCP+053VjpbF\\\\\\\/usNzwzHo0cP+V2RQgMO7OF2K5ndtH53hf1sU9BtT\\\\\\\/8d13Y9qEVnNWONOJoLxX9zDU84GFO4kDI\\\\\\\/vpV84Vh36POQYera+1e2hNLloUFBlhaJhNppbWKUQ48cF2HdH99teOzuPraWS1\\\\\\\/DsNV9MNuamNYFOwHvZorV+FXbUpqqcJhnvkHra3VXbGIvjzMOiKg8BBZ7J5KYfXEn5mbx64tm3BrYvoJ6g8X94OceHGC20u9CNrGXcyHlNEIBwhWbTQSRnAPB4zmUrNDRqZ97fGn+UcYHy9EjFbE5Uibmke3n1+AA\\\\\\\/5c6xA+7GQV+iGsAZ+eYg6BaFZNfC0e1C\\\\\\\/dBaliPJAs+P+ehustLW741vtRPZbtK7kaqTbhimMNYCeayCd+gixKh6PyfJzs2QXWG5RkDN6NBETI42064a3Ft6\\\\\\\/HTVAbueUGbyJk51G\\\\\\\/rUgzR\\\\\\\/jwKn99mKCxMjJC6f6SqRmXVBQ9MFAKhwyYGGXqfDnzKys9XkTqaIlnWoZC+p\\\\\\\/WArFtk0B0J2jd1y9a9BFNSGqG5mvvYYQqSKkpNGCSsryoWBh+g0yNCqfyMh4CWwe23cPmkXtb9WM1xIPirorl7NPMLgM0Kci0LENAD8rpWkAwa1J5l1vIlv\\\u0022,\\\u0022iv\\\u0022:\\\u0022i5JU6FI3oayyvsBLS5Kl0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220379f3fa1fdc931b1526456f75aaeaaa2f98f7f6fa66dde4090cf4917302a707\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O32UI4yy290hQ+b+d5lu\\\\\\\/xUaS26QoUuBmaGARPjgOESZKlVZBpGsMTa6OopaDPryRA6YIhhlf4AYudzjhWcLJTjmBr9mXrbcK8tgBEV+UvuoiBK93WZOqm4qgAzeQwvFJES0HmOgIXqbg1PsTDD1iXU26BFmEoTpokVwMuTG9b1dWP0EYsVSZZkYWSKBmvNuP2UGfi3vW5uStQWWYIdI4l2MJe49WITfUD3KlDMeFEal3Rqvh7OWhQiKEMh03LXhQcD3fc4ymotf58VT2CXIQv+b0xRtOGWvy24jNfa4U2ZX8aO3TT8NZoOqqtHkkuf+v52cYA8S37wGy3cjmFS\\\\\\\/edheBBuISutsBnN\\\\\\\/owtv+TQEkBI0x\\\\\\\/LMB49AaUGqX8eggw\\\\\\\/it1qyynJmw6yrP61VN9xzlQiLSq0UbJmZuUaq0v\\\\\\\/JNfNppxDDOthHvbup4Fifkxjle4Jmd95brwnev\\\\\\\/p81mqiBNWbkGacKYoJW5rbOBwCJXIM8AKRP1Ntprsmxq5GTXDgJrLDE01x4kXPGkSOsH8D9sYNDDE7vIXpxXxLnlP8yEjxJbtwIi6c+yMUZ8Pwjo2e6AaMdKPmU\\\\\\\/BdoGsrX2Ia\\\\\\\/baS+IOUY0nAtcb+q8dZZ5n5I4ueUun3WBCnxI3j84oGckVtLhe5RIv8uLLl1dHDEVgiZmqUD98\\\\\\\/SL4TfxqRLTBhMmyDDBnCxA26BHD7++qceRGp77HsToDS\\\\\\\/aEIwexgg81GLlM+tH7tNV\\\\\\\/wMoOOVEXXTa+o99bg9m75atf5efLuvh6YUhXmJK88mqSMu5DMcQ1HSTcTa\\\\\\\/5nbINJvv+o3bnYF52AO67mR9hjgcsyTL9Mrteg1cDYp\\\\\\\/fmnFaU4PYzWvI6GKon59FGiNPZDlFP1m\\\\\\\/nikeZB0BUpgsfKJrW2BoNb6+whDPopuK\\\\\\\/fpngeqBSzbF6jWM1zLitXgZqzfBFBnZ3DuBxMHVpEET8Ymd+Gq3cioZGgTjmcQ95BgYQ4g1UZueQXYVVU2dXAwbq72nMaWIvUDsvaGn4q71YCg0OxkiUczkw06wahhZYcj0\\\\\\\/8\\\\\\\/S8Yn6Z2an13bexCwiQmJLxVK9qBP2HnrX2t9Lo4iAMZfx1oOE127gfXxq6sZDcH43vaU9+a9tx16ae8gtEBk6913vPGOS6PhhptrjLIvdYKRSWw6bXGRoPEPuzIgFVPXCvDDcpRvE3tXun3hgI9sDl8jP+pZWPMpYBwt+xZPD\\\\\\\/C8vdEruHITZTAhuCFavSse2bm6ufIgrIshfJW0i0uL\\\\\\\/TGMT71CmLOaaVoB8QHdpQ4ZqjtRiJLvubEsYIfYYUnQCQ3H81emTHRqGm7EinTDGoyi8O3\\\\\\\/H5ZzSLYBRV+WtWXPmo4djhWea+Jag9t+axecNBgALu5RnzFHlqz22g51Pkg28\\\\\\\/E0rVtc+agjP77XvbBbTIsXmchzC3m87qj7zm+1LNsGIiJq0NFhdm1UITKxP7dZ3sAl7LHQHYENbKs98CGC7SpzEtx4XfG6EAnjgXekghzzcUVVPj0ApKsZyvezrzVppxVEcGv9YEG3Rr8ywhQHgYxnpmXU7roKRZtLfLG8qVaDmvWVSQvhMZK4DJnUg18\\\\\\\/uwVphD7KmYJ\\\\\\\/Wh\\\\\\\/XEZ1dxpE7uEhK0haQhtuBXmODeuEDt3WrRCMOtGSRJA6XUdKrbPvvWrQojRsZdArXCQIj5UNVZCZLzJ46roZedpeKX7N46ubQjna4hUbItjcYJvSu3gFVfV4a72F11MXUC21iw1G9TiPIOfV5E0AD6wo65VQhsfZMymYHTin0iy4PQu2ggoTMRvxZMv06O8LwXzvTqeaS2gmiDTICFK4ibarGx+6ueXeyP\\\\\\\/NHhcphldY9rvMAqz942mxVT9zMrslI6VUcyeyS5OmcawbyGahFBwVkC9u65uCmHCp5aqIYIvIgHjUuXIUkvq\\\\\\\/Jp\\\\\\\/puAxcuRL6EVrvKoIpefYqMfbGKrbP4JWcrMS64IaK7i2AUuZWz\\\\\\\/wBkVvpfb+\\\\\\\/2EoO4QspS\\\\\\\/8AO576sYx31lJeVldbBeqxJqPlHlrwSqswThroHKYMCg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022nJFGvB4ZS8aJgLd6L1wnXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f40869241244a30b8d42569098e12188dd040b13a114067d5b391c189ba90674\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459529 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1453" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--H8Y0YJQ-9Ui8WXfn9oRzU6OL8fOJSz_fRYej0f_HEE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1453" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oD8Yqsg592VPTpTwcthwskp0ltcCsGGvj5BsAUVrY6I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/9/1451" title="Functional Genetics in the Post-genomics Era: Building a Better Roadmap in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/9/1467" title="Sequence Diversity in Coding Regions of Candidate Genes in the Glycoalkaloid Biosynthetic Pathway of Wild Potato Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459529" id="node459529" data-pisa="ggg;3/9/1453" data-pisa-master="ggg;g3.113.006429" data-apath="/ggg/3/9/1453.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory I. Lang</span>, <span class="highwire-citation-author" data-delta="1">Lance Parsons</span> and <span class="highwire-citation-author" data-delta="2">Alison E. Gammie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1453-1465;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006429 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459529" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory I. Lang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BI.%2BLang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lang%20GI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BI.%2BLang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lance Parsons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lance%2BParsons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parsons%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALance%2BParsons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alison E. Gammie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BE.%2BGammie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gammie%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BE.%2BGammie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agammie@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">agammie@princeton.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/9/1453" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459529" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1453.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459529" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1453.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459529" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1453.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459529" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/9/1453.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459529" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459529/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">DNA mismatch repair is a highly conserved DNA repair pathway. In humans, germline mutations in <em>hMSH2</em> or <em>hMLH1</em>, key components of mismatch repair, have been associated with Lynch syndrome, a leading cause of inherited cancer mortality. Current estimates of the mutation rate and the mutational spectra in mismatch repair defective cells are primarily limited to a small number of individual reporter loci. Here we use the yeast <em>Saccharomyces cerevisiae</em> to generate a genome-wide view of the rates, spectra, and distribution of mutation in the absence of mismatch repair. We performed mutation accumulation assays and next generation sequencing on 19 strains, including 16 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> missense variants implicated in Lynch cancer syndrome. The mutation rate for DNA mismatch repair null strains was approximately 1 mutation per genome per generation, 225-fold greater than the wild-type rate. The mutations were distributed randomly throughout the genome, independent of replication timing. The mutation spectra included insertions/deletions at homopolymeric runs (87.7%) and at larger microsatellites (5.9%), as well as transitions (4.5%) and transversions (1.9%). Additionally, repeat regions with proximal repeats are more likely to be mutated. A bias toward deletions at homopolymers and insertions at (AT)<sub>n</sub> microsatellites suggests a different mechanism for mismatch generation at these sites. Interestingly, 5% of the single base pair substitutions might represent double-slippage events that occurred at the junction of immediately adjacent repeats, resulting in a shift in the repeat boundary. These data suggest a closer scrutiny of tumor suppressors with homopolymeric runs with proximal repeats as the potential drivers of oncogenesis in mismatch repair defective cells.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mismatch-repair" class="hw-term hw-article-keyword hw-article-keyword-mismatch-repair" rel="nofollow">mismatch repair</a></li><li class="kwd"><a href="/keyword/mutation-accumulation" class="hw-term hw-article-keyword hw-article-keyword-mutation-accumulation" rel="nofollow">mutation accumulation</a></li><li class="kwd"><a href="/keyword/mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate" rel="nofollow">mutation rate</a></li><li class="kwd"><a href="/keyword/homopolymeric-runs" class="hw-term hw-article-keyword hw-article-keyword-homopolymeric-runs" rel="nofollow">homopolymeric runs</a></li><li class="kwd"><a href="/keyword/microsatellites" class="hw-term hw-article-keyword hw-article-keyword-microsatellites" rel="nofollow">microsatellites</a></li></ul><p id="p-6">Mutations in DNA have far ranging consequences, from driving evolution to causing disease. DNA mismatch repair is a highly conserved process that maintains the fidelity of genomes by decreasing the mutation rate 100- to 1000-fold (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kunkel and Erie 2005</a>). Mismatch repair proteins detect helical distortions or mismatches derived from exposure to mutagens (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Stojic <em>et al.</em> 2004</a>) during inexact replication of the genome (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hsieh and Yamane 2008</a>) and upon recombination of nonidentical DNA molecules (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Surtees <em>et al.</em> 2004</a>). If the damaged or mismatched DNA is not repaired, and a new round of replication is initiated, the mutation becomes stably incorporated into the genome.</p><p id="p-7">Lynch syndrome is a prevalent hereditary cancer syndrome caused by defects in DNA mismatch repair (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lynch <em>et al.</em> 2009</a>). Individuals with Lynch syndrome are typically heterozygous for either <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">MSH2</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004777">MLH1</a></em>, core components of DNA mismatch repair (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Silva <em>et al.</em> 2009</a>). As part of the disease process, the sole wild-type copy of the mismatch repair gene becomes inactivated, and a cell then begins to accumulate mutations at an accelerated rate, often leading to tumor formation (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boland 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Colas <em>et al.</em> 2012</a>). A distinguishing feature of most mismatch repair defective tumors is the presence of microsatellite instability (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Shah <em>et al.</em> 2010a</a>). Microsatellites are composed of repetitive sequences with 1−10 nucleotides as the repeat unit (reviewed in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bhargava and Fuentes 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gemayel <em>et al.</em> 2010</a>). Microsatellite instability is a consequence of unrepaired slippage events during DNA replication of these repeat regions (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Levinson and Gutman 1987</a>) and is confirmed when length of the microsatellite loci from an individual’s tumor differs significantly from the same loci in healthy cells (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Lynch <em>et al.</em> 2009</a>). In addition to frequently displaying microsatellite instability, mismatch repair defective tumors tend to be diploid on a gross chromosomal level, as opposed to the more typical aneuploidy observed in other cancers (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Oki <em>et al.</em> 2012</a>).</p><p id="p-8">Since the discovery of the link between mismatch repair and Lynch syndrome, many germline and somatic mutations have been identified in mismatch repair genes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">de la Chapelle 2004</a>). Approximately 20% of these mutations are missense variants, resulting in a single amino acid substitution in the mismatch repair protein (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">de la Chapelle 2004</a>). Our previous characterization of these missense variants has provided insights into the molecular defects associated with Lynch syndrome cancers (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gammie <em>et al.</em> 2007</a>). In this work, we analyzed clinically significant missense variants of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">MSH2</a></em> along with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> null in yeast to characterize the genomic signature associated with Lynch syndrome.</p><p id="p-9">Our current understanding of the effects of mismatch repair deficiency on genome stability is derived mainly from analyses using reporter genes in organisms ranging from bacterial to human systems (reviewed in <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aquilina and Bignami 2001</a>). The types of reporters include those that assay single-base substitutions and/or microsatellite instability of mono-, di-, tri-, and larger nucleotide repeats (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hawk <em>et al.</em> 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Henderson and Petes 1992</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Marsischky <em>et al.</em> 1996</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Tran <em>et al.</em> 1997</a>). These reporters are typically expressed episomally or integrated into the genome at select loci. Although informative, reporter constructs do not reveal the full spectrum of possible mutations, nor do they capture mutational variability associated with genomic architecture, sequence contexts, or processes such as replication and transcription.</p><p id="p-10">The mutation accumulation assay provides an alternative to reporter assays. In a mutation accumulation assay, the population is propagated through recurrent single-cell bottlenecks, thus mitigating the effect of selection and allowing mutations (other than lethal mutations) to accumulate as if they were neutral. Sequencing the end point of a lineage reveals the number, positions, and identities of accumulated mutations. In this work, we passaged mismatch repair defective haploid yeast cells over hundreds of generations with recurrent bottlenecks and determined the mutation rates, spectra, and genome-wide distributions of mutations by using whole-genome sequencing. We find that mismatch repair deficient strains accumulate ~1 mutation per genome per generation (corresponding to a ~200- to 300-fold increase in mutation rate relative to wild type). Because the mutation accumulation assay queries many types of mutation events and contexts simultaneously, it not only produces a more accurate estimate of the per-genome per-generation mutation rate, but also allows one to determine how the mutation rate is influenced by sequence-specific features and genomic context. We find that mutations occurred randomly across the genome, with no chromosomal, gene, or replication timing biases; however, mismatch repair defective cells do display a distinctive mutational signature, with deletions at homopolymeric runs representing the primary mutational event. We find that microsatellite instability increases with repeat length and that microsatellites adjacent to other repeats are more mutable. Overall, these data provide insight into the oncogenic process and should aid in the identification of the likely drivers of tumor formation in cancers displaying microsatellite instability.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Microbial and molecular techniques</h3><p id="p-11">Microbial manipulations were conducted according to previously published procedures (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ausubel <em>et al.</em> 1994</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Burke <em>et al.</em> 2000</a>). Molecular methods were performed with the use of standard protocols (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Ausubel <em>et al.</em> 1994</a>). Plasmid DNA extractions were performed using the Qiagen procedure (QIAGEN Inc., Valencia, CA). Primers were synthesized by Integrated DNA Technologies Inc. (Coralville, IA). Restriction endonuclease digestions and polymerase chain reaction (PCR) were performed using the enzyme manufacturer recommended reaction conditions (New England Biolabs, Beverly, MA).</p></div><div id="sec-3" class="subsection"><h3>Strains and plasmids</h3><p id="p-12">XL2-Blue (Stratagene, La Jolla, CA) bacterial cells were used for plasmid propagation. The salient features of the plasmids used in this work are listed in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/006429SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS1.pdf">Table S1</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> missense mutations encoded on centromere-based plasmids were generated as described previously (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gammie <em>et al.</em> 2007</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> knockout strain AGY1079 (<em>MAT</em>α <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000854">hom3</a>-10 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005654">ade2</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>-3,112 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>-11,15</em>) and a wild-type strain from the same cross AGY1100 (<em>MAT</em><strong>a</strong> <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000854">hom3</a>-10 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005654">ade2</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>-3,112</em>) were derived from W303. The strains were confirmed to be wild type at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004022">RAD5</a></em> locus by PCR and at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em> locus by canavanine resistance assays.</p></div><div id="sec-4" class="subsection"><h3>Qualitative mismatch repair and fluctuation assays</h3><p id="p-13">Qualitative mismatch repair assays as described previously (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Gammie <em>et al.</em> 2007</a>). Canavanine resistance was selected for using plates supplemented with 60 μg/mL canavanine (Sigma-Aldrich, St. Louis, MO). Luria-Delbrück fluctuation assays, used to determine the rates of loss of function of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em> were performed as described previously (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lang and Murray 2008</a>). Mutation rates were calculated using both the Luria-Delbrück P<sub>0</sub> method (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Luria and Delbrück 1943</a>) and the MSS maximum-likelihood method (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sarkar <em>et al.</em> 1992</a>).</p></div><div id="sec-5" class="subsection"><h3>Mutation accumulation</h3><p id="p-14">The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> knockout strain was transformed with the plasmids listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS1.pdf">Table S1</a> and propagated in synthetic medium lacking histidine to select for the plasmids. A single colony from each transformation was selected to begin the mutation accumulation experiment. Strains were passaged on synthetic medium lacking histidine for ~170 generations with bottlenecks every ~21 generations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/FigureS1.pdf">Figure S1</a>). The bottlenecks were accomplished by picking a single colony and streaking for single colonies approximately every 2 d; the process was repeated eight times. Taking into account population expansion between the bottlenecks, we estimate an effective population size of approximately 10. The theory underlying the mutation accumulation assay is that all mutations other than lethal mutations accumulate as if neutral. If the population size were exactly one, this would be true; however, the population expansion between bottlenecks introduces the opportunity for selection. Given a rate of one mutation per cell division, the likelihood of losing a strongly deleterious mutation (0.1) is only 10% (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/FigureS1.pdf">Figure S1</a> in <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lynch <em>et al.</em> 2008</a>).</p></div><div id="sec-6" class="subsection"><h3>Sequencing</h3><p id="p-15">In preparation for sequencing, a single colony was selected and grown in 25 mL of yeast extract, peptone, dextrose medium supplemented with adenine (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Burke <em>et al.</em> 2000</a>) until saturation was achieved (24−40 hr). Genomic DNA preparations from yeast were as described previously (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Burke <em>et al.</em> 2000</a>) except the glass bead lysis step was accomplished with a Fastprep-24 instrument (MP Biomedicals LLC).Yeast genomic DNA was prepared for sequencing with the Illumina TruSeq DNA Sample Preparation kit with six indices for multiplexing. Whole-genome sequencing was performed at the Lewis-Sigler Institute for Integrative Genomics Core Sequencing Facility with an Illumina HiSequation 2000. Four lanes with six samples each were used. The ancestor samples were doubled to maximize coverage. Single end reads of 100 bp were performed giving from 50x to 300x coverage of each genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS2.pdf">Table S2</a>).</p></div><div id="sec-7" class="subsection"><h3>Sequencing data analysis</h3><p id="p-16">Each sequencing read was aligned to a draft yeast genome with BWA for Illumina version 1.2.2 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Li and Durbin 2009</a>) using parameters listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS3.pdf">Table S3</a>. Mutations were identified using Freebayes version 0.8.9.a, a Bayesian single-nucleotide polymorphism and short insertion/deletion (indel) caller (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Garrison and Marth 2012</a>) using parameters listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS4.pdf">Table S4</a>. The default parameters for the BWA mapping and Freebayes mutation calling programs missed almost all (93%) of the insertion/deletion mutation. Using the parameters listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS3.pdf">Table S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS4.pdf">Table S4</a> was essential for calling the insertions/deletions. BWA and Freebayes were implemented using the Galaxy user interface (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Blankenberg <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Giardine <em>et al.</em> 2005</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Goecks <em>et al.</em> 2010</a>).</p><p id="p-17">The draft W303 genome is available upon request and was generated as follows. Three ancestral W303 strains, including the wild-type (AGY1100) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> (AGY1079) ancestors described in this study as well as a wild-type W303 strain from a different cross (G. Lang collection), each with &gt;300x coverage, were used to identify common and unique polymorphisms when compared with the S288C genome as detailed previously. The common polymorphisms were applied to the S288C reference using the FastaAlternateReferenceMaker utility from the Genome Analysis Toolkit (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">McKenna <em>et al.</em> 2010</a>), generating an updated reference. The sequence reads were mapped to this new reference, and common polymorphisms were again identified and applied to the reference. This was repeated for several iterations and resulted in a final list of polymorphisms, including 9657 single-base-pair substitutions and small insertion/deletions. Larger insertion/deletions or duplications were not identified.</p><p id="p-18">We identified 14 unique polymorphisms in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>∆ ancestor not found in the other two W303 ancestors (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS5.pdf">Table S5</a>). Seven were intergenic or within an intron, the remaining were missense/nonsense or frameshift mutations in well-characterized genes that are not associated with mutator phenotypes. These findings support the conclusion that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>∆ was the only mutator allele present in the starting strain.</p><p id="p-19">The mutations in passaged lines were identified by mapping to the draft W303 genome and comparing the called mutations from the lineages with the ancestor. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">MSH2</a></em> chromosomally encoded wild-type passaged line was compared to the wild-type ancestor and the plasmid based lines were compared to their shared <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>∆ ancestor. Each unique mutation in the passaged strains was verified manually using Integrative Genomics Viewer (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Thorvaldsdottir <em>et al.</em> 2012</a>). Only fixed mutations (<em>i.e.</em>, mutations in 100% of the reads) were scored. Thus, mutations arising during the few generations required for obtaining genomic DNA for sequencing were not scored because these mutations would not be present in all of the reads. Insertions/deletions are difficult to score because of inherent problems with PCR amplifications and sequencing of repeat regions. To score as an insertion/deletion, at least three reads must have traversed the entire repeat region for both the passaged line and the ancestor.</p><p id="p-20">We identified 10 lineages with three common end-point single base substitutions and two insertion/deletion mutations not present in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>∆ ancestor. We reasoned that these common mutations were likely to represent mutations that arose during growth of the ancestral strain prior to transformation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/FigureS1.pdf">Figure S1</a>). To test this, for each of the five common mutations, using PCR we amplified and resequenced the region from the first time point of each lineage (frozen immediately after transformation). In all cases the common mutations were observed immediately after transformation, suggesting that these five mutations occurred during growth of the ancestral strain prior to the transformation of the plasmids. We, therefore, removed these mutations from subsequent analyses.</p><p id="p-21">To assess mutation rates at microsatellites, an accurate count of the repeat number was required. Microsatellites in the draft W303 genome were identified using msatfinder (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Thurston and Field 2005</a>). Bedtools IntersectBed (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Quinlan and Hall 2010</a>) was used to find the number of reads that overlap a microsatellite region as well as non-repeating regions of varying length. Using R for Statistical Computing (<a href="http://www.r-project.org/">http://www.r-project.org/</a>) regions from chromosome XII (rDNA repeats) as well as regions with a read count ≥4x median were removed before plotting. R was also used to generate box plots of the number of reads that span the regions of each length, stratified by repeating or nonrepeating.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>DNA mismatch repair defective cells accumulate approximately 1 mutation per generation, ~200- to 300-fold greater than the wild-type rate</h3><p id="p-22">Until recently (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ma <em>et al.</em> 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Nishant <em>et al.</em> 2010</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Zanders <em>et al.</em> 2010</a>), obtaining estimates of the increase in mutation rate in mismatch repair defective cells depended solely on reporter genes. In this study, we calculated the mutation rates across the entire genome by using haploid wild-type and mismatch repair defective cells in a mutation accumulation assay over ~170 generations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/FigureS1.pdf">Figure S1</a>). We tested 16 clinically significant missense variants of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> by expressing each from a centromere-based plasmid in an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>∆ strain. The wild-type control was the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>∆ strain containing the wild-type version of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">MSH2</a></em> expressed from a centromere-based plasmid (CEN WT) and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>-null control was the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>∆ strain with the empty plasmid vector. The mutation accumulation experiment also included a wild-type control in which <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">MSH2</a></em> was intact in the chromosome (genomic WT). After passaging, genomic DNA was prepared for whole-genome sequencing. The sequencing depth ranged from 50x to 300x coverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS2.pdf">Table S2</a>). The mutations in each passaged strain were compared with the relevant ancestor (genomic WT, or the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>-null ancestor). All mutations were manually verified as described in the <em>Materials and Methods</em>.</p><p id="p-23">In this analysis (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and previously (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Arlow <em>et al.</em> 2013</a>; <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Gammie <em>et al.</em> 2007</a>) we used the plasmid based controls to classify the missense variants into functional categories: null, intermediate, and wild type. In the current study, one missense mutant, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a>-P689L</em>, was classified as a pseudo-wild type based on the fluctuation assays, whereas the remaining missense strains were indistinguishable from the null allele (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). For the remainder of the paper, unless specifically indicated, we combined the mutations for the 16 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>-null-like strains for increased statistical power. Three strains harbored rearranged plasmids in which the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">MSH2</a></em> coding sequence was not intact (noted in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The rearrangement occurred early in the passaging and these variants were thus classified as true nulls for certain statistical tests.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463242/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463242/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Classification of sequenced strains</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463235/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463235/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mutation rate based on mutation accumulation over ~170 generations</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">In the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>-null strains, we identified 158 base pair substitutions and 2318 insertion/deletion mutations across the 16 lineages. The average rate of mutation for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>-null strains was 7.4 × 10<sup>−8</sup> mutations per base pair per generation (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). This rate is two orders of magnitude greater than the estimate of 3 × 10<sup>−10</sup> mutations per base pair per generation for wild-type yeast strains (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Nishant <em>et al.</em> 2010</a>); the genomic wild-type strain accumulated only a single mutation over the 170 generations, consistent with a wild-type per-base pair per-generation mutation rate of ~10<sup>−10</sup> mutations per base pair per generation. In the absence of mismatch repair, the mutation rate for single-base pair substitutions was 4.8 × 10<sup>−9</sup> mutations per base pair per generation, and for insertions or deletions at mono-, di-, and trinucleotide repeats was 7.0 × 10<sup>−8</sup> mutations per base pair per generation. Overall, this suggests a 225-fold increase over genomic wild-type in the number of mutations for mismatch repair defective cells, or ~1 mutation per genome per generation.</p></div><div id="sec-10" class="subsection"><h3>In the absence of mismatch repair, mutation accumulation occurs randomly with respect to chromosomal position</h3><p id="p-29">Previous experimental and comparative genomic analyses in yeast showed that there are mutational differences with respect to the chromosomal context (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hawk <em>et al.</em> 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ito-Harashima <em>et al.</em> 2002</a>) and replication timing (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Agier and Fischer 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lang and Murray 2011</a>). Examining the mutations across the entire genome allowed us to determine if there were any position effects that might relate to chromosomal structure or replication timing. We determined that both single base pair substitutions and insertions or deletions at repeats occurred randomly across the genome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). In keeping with this, the number of single base pair substitutions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>) and insertions/deletions (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>) per chromosome correlated with chromosome size (R<sup>2</sup> = 0.91 and 0.87, respectively).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutations in mismatch repair defective cells occur randomly across the genome. (A) Chromosomal distribution of mutations including the single base pair substitutions (open circles) and the insertions/deletion at mono-, di-, and trinucleotide microsatellites (filled circles) are shown at their chromosomal position for each of the 16 yeast chromosomes. Mutation number was plotted against chromosome size for single-base pair substitutions (B) and for insertions/deletions at microsatellites (C). Single-base substitutions in (B) represent data pooled from two independent mutation accumulation experiments. R2 values were generated in Microsoft Excel (Redmond, WA) and are indicated on the graphs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1653639577" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutations in mismatch repair defective cells occur randomly across the genome. (A) Chromosomal distribution of mutations including the single base pair substitutions (open circles) and the insertions/deletion at mono-, di-, and trinucleotide microsatellites (filled circles) are shown at their chromosomal position for each of the 16 yeast chromosomes. Mutation number was plotted against chromosome size for single-base pair substitutions (B) and for insertions/deletions at microsatellites (C). Single-base substitutions in (B) represent data pooled from two independent mutation accumulation experiments. R&lt;sup&gt;2&lt;/sup&gt; values were generated in Microsoft Excel (Redmond, WA) and are indicated on the graphs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/9/1453/F1.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/9/1453/F1.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463144" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Mutations in mismatch repair defective cells occur randomly across the genome. (A) Chromosomal distribution of mutations including the single base pair substitutions (open circles) and the insertions/deletion at mono-, di-, and trinucleotide microsatellites (filled circles) are shown at their chromosomal position for each of the 16 yeast chromosomes. Mutation number was plotted against chromosome size for single-base pair substitutions (B) and for insertions/deletions at microsatellites (C). Single-base substitutions in (B) represent data pooled from two independent mutation accumulation experiments. R<sup>2</sup> values were generated in Microsoft Excel (Redmond, WA) and are indicated on the graphs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Although the mutation positions were random at a gross chromosomal level, we wanted to determine if they were in regions that have been associated with greater mutation rates such as late replicating portions of the genome. By binning the genome by replication timing (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Raghuraman <em>et al.</em> 2001</a>) at 10-min intervals and calculating the mutation rate for each bin, we fail to find a significant correlation between replicating timing and the mutation rate (<em>P</em> = 0.31, χ<sup>2</sup>).</p><p id="p-32">Because these experiments did not depend on reporter genes, we analyzed whether there was any relationship between mutation position and coding sequences. We found that the single base pair substitutions occurred mostly in coding regions (72%). This number is in contrast to the insertions/deletion mutations that were more likely to be in noncoding regions than in coding sequences (14%), reflecting the composition of the yeast genome. Approximately 74% of the yeast genome is comprised of coding sequences (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cherry <em>et al.</em> 1997</a>) consistent with the distribution of single base pair substitutions. Additionally, only 10–20% of the microsatellite DNA, including mono-, di-, and trinucleotides, is found in eukaryotic coding sequences (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Li <em>et al.</em> 2004</a>), similarly reflecting the distribution of insertions/deletion mutations we identified. Taken together, these data suggest that any mutational bias associated with chromosome structure, gene organization, or replication timing is diminished in the absence of mismatch repair.</p></div><div id="sec-11" class="subsection"><h3>Insertion/deletion loop repair is the predominating mismatch repair role required During passaging of cells over 170 generations</h3><p id="p-33">Measuring the frequency for the entire spectrum of mutations at endogenous loci in parallel was not possible until recently. Here we report the concurrent measurement of mutation frequency of single base pair substitutions as well as insertions/deletions at mono-, di-, and trinucleotide repeats (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). For the remainder of this work, we will maintain a distinction between single nucleotide microsatellites (homopolymeric runs) and larger di-, tri-, and tetranucleotide microsatellites. We find that the mutation frequency spectrum for mismatch repair defective cells included deletions/insertions at homopolymers (87.7%) and at di- and trinucleotide microsatellites (5.9%), as well as transitions (4.5%) and transversions (1.9%).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463210/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463210/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of genome-wide mutations in mismatch defective cells</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>In the absence of mismatch repair, the mutation rate at homopolymeric runs and microsatellites increases nonlinearly with repeat length</h3><p id="p-36">Previous work showed that the mutation rate at microsatellites increased with repeat unit length (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Tran <em>et al.</em> 1997</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wierdl <em>et al.</em> 1997</a>). In this study, we compared the rates of mutation at endogenous microsatellite loci and over hundreds of generations using multiple strains in parallel. We confirmed that the number of mutations increased with repeat length (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and D</a>) at a much higher frequency than was expected from the occurrence of such repeats in the genome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, B and E</a>, note the log scale). The strong length dependence on instability is evident with each additional repeat unit resulting in a progressive fourfold and sevenfold increase in sequence instability for homopolymers and larger microsatellites, respectively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutation rate increases with microsatellite repeat length. The number of insertion/deletion mutations identified at A/T homopolymeric repeats (A), or dinucleotide microsatellites (D) are plotted according to repeat length. Shaded areas indicate that the numbers might be an underrepresentation because of the decreased ability to detect insertions or deletions at long repeats. The number of A/T homopolymers (B) or dinucleotide microsatellites (E) in the yeast genome (y-axis) is plotted according to repeat length (x-axis) on semi-log graphs. The mutation rate (mutation per repeat per generation) for homopolymers (C) or dinucleotide microsatellites (F) are plotted according to repeat unit. The exponential increase in mutation rate from 3 to 8 repeat units is plotted on semi-log graphs in enclosed panels. Formulas and R2 values were generated in Microsoft Excel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1653639577" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutation rate increases with microsatellite repeat length. The number of insertion/deletion mutations identified at A/T homopolymeric repeats (A), or dinucleotide microsatellites (D) are plotted according to repeat length. Shaded areas indicate that the numbers might be an underrepresentation because of the decreased ability to detect insertions or deletions at long repeats. The number of A/T homopolymers (B) or dinucleotide microsatellites (E) in the yeast genome (y-axis) is plotted according to repeat length (x-axis) on semi-log graphs. The mutation rate (mutation per repeat per generation) for homopolymers (C) or dinucleotide microsatellites (F) are plotted according to repeat unit. The exponential increase in mutation rate from 3 to 8 repeat units is plotted on semi-log graphs in enclosed panels. Formulas and R&lt;sup&gt;2&lt;/sup&gt; values were generated in Microsoft Excel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/9/1453/F2.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/9/1453/F2.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463193" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Mutation rate increases with microsatellite repeat length. The number of insertion/deletion mutations identified at A/T homopolymeric repeats (A), or dinucleotide microsatellites (D) are plotted according to repeat length. Shaded areas indicate that the numbers might be an underrepresentation because of the decreased ability to detect insertions or deletions at long repeats. The number of A/T homopolymers (B) or dinucleotide microsatellites (E) in the yeast genome (y-axis) is plotted according to repeat length (x-axis) on semi-log graphs. The mutation rate (mutation per repeat per generation) for homopolymers (C) or dinucleotide microsatellites (F) are plotted according to repeat unit. The exponential increase in mutation rate from 3 to 8 repeat units is plotted on semi-log graphs in enclosed panels. Formulas and R<sup>2</sup> values were generated in Microsoft Excel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The mutation rate data for homopolymers and larger microsatellites revealed a striking, overall nonlinear increase in the mutation rate with repeat length (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, C and F</a>). The mutation rates at homopolymers and dinucleotide microsatellites show an exponential increase with repeat unit until reaching a repeat unit of eight. For example, the rate of mutations per repeat per generation for (A/T)<sub>n</sub> homopolymer runs ranged from 9.7 × 10<sup>−10</sup> (repeat unit of three) to 1.3 × 10<sup>−5</sup> (repeat unit of eight). For repeat units greater than nine, the observed increase in rate changed from exponential to linear (y = 0.0001x − 0.0012; R<sup>2</sup> = 0.98). The same trends were also observed for (C/G)<sub>n</sub> homopolymers, but with slightly greater mutation rates (~7-fold greater on average, not shown). The differences in rates at the two types of homopolymers have been observed previously (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gragg <em>et al.</em> 2002</a>); however, in this study, the sample size for (C/G)<sub>n</sub> homopolymers was significantly lower (n = 38 compared with n = 2134) and therefore the apparent differences in rates may be a consequence of the number of events measured. The trend from exponential to linear at repeat units greater than nine was also observed for dinucleotide microsatellites; however the data are less accurate beyond repeat units of seven because of the lower sample size.</p><p id="p-39">The change in the rate increase from exponential to linear may have a biological explanation; however, we speculate that the rates are less accurate for longer repeats, because multiple sequencing reads must traverse the entire repeat to confidently call an insertion or deletion mutation. We performed an analysis of sequencing read counts that spanned entire repeats for all of the sequenced strains and found a significant drop with repeats greater than 13 bp regardless of the genome coverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/FigureS2.pdf">Figure S2</a>). Therefore, our ability to detect an insertion/deletion mutation in repeats greater than or equal to 14 bp in length is diminished, leading to underestimates of the true mutation rate at these positions (gray shading in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, A and D</a>).</p><p id="p-40">The larger quantity of mutations at homopolymers, relative to dinucleotide repeats, does not result from a greater rate of mutation at homopolymers. In fact, for repeat units between five and seven the rate of mutation of homopolymers is ~20-fold less than that of dinucleotides of the same repeat unit. The greater number of observed mutations in (A/T)<sub>n</sub> homopolymers simply reflects the relative abundance in the yeast genome (compare <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, B and E</a>).</p></div><div id="sec-13" class="subsection"><h3>A mutational bias toward deletions at homopolymeric runs and insertions at certain microsatellites is observed in mismatch repair defective cells</h3><p id="p-41">When assaying for insertion/deletion events, some reporter loci influence the type of mutation because of reading frame constraints, the requirement for active transcription, the proximity and orientation with respect to origins of replication, and/or unusual chromatin structure. Mutation accumulation followed by genome-wide sequencing allows for the determination of any potential insertion/deletion bias at mono-, di-, and tri- microsatellites without the use of reporter loci.</p><p id="p-42">Although the increase in mutation rate at homopolymers and dinucleotide microsatellites is similar when adjusted for repeat unit, we observed a difference in the types of mutations generated at these sites (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). We find that (A/T)<sub>n</sub> homopolymers suffer deletions at a high rate (93%, n = 2134, <em>P</em> &lt; 10<sup>−10</sup>, χ<sup>2</sup>). The (C/G)<sub>n</sub> repeats also have a bias toward deletions, but it is less pronounced (74%, n = 38, <em>P</em> = 3.5 × 10<sup>−3</sup>, χ<sup>2</sup>). The (GT/CA)<sub>n</sub> dinucleotide microsatellite instability events show a trend toward deletions (65%, n = 17, <em>P</em> = 0.23, χ<sup>2</sup>), although this finding is not statistically significant. In contrast, (AT/TA)<sub>n</sub> dinucleotide microsatellite instability shows a significant insertion bias (63%, n = 113, <em>P</em> = 6.4 × 10<sup>−3</sup>, χ<sup>2</sup>). Finally, the trinucleotide repeats show a slight tendency toward insertions (57%, n = 14); however, the number of events was not sufficient to for a statistical analysis to determine an insertion/deletion bias within each sequence type. In summary, the bias toward an insertion or deletion event is likely to be dependent on the composition of the repeat.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463196/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463196/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Insertion/deletions at homopolymeric runs and larger microsatellites</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>DNA regions with a greater density of repeats are more mutable in mismatch repair defective cells</h3><p id="p-44">Although no gross chromosomal mutational hotspots were identified, we observed that regions with a higher density of repeats were more mutable. We used motif-searching algorithms and observed that the mutated mono-, di-, or tri nucleotide repeat loci were often found in close proximity to other repeats. For example, we find that 28% of the mutated repeats are within 3 bp of the next repeat in the genome and 51% are 7 bp from the most adjacent repeat. To determine if this was statistically significant we sorted the loci according to the closest adjacent repeat and plotted the cumulative percentages of all genomic repeat loci and the mutated repeat loci (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The plot illustrates the differences between the distributions. Using a Kolmogorov-Smirnov comparison of two data sets we find that there is a statistical difference (<em>P</em> = 2.8 × 10<sup>−6</sup>), confirming that repeats are more mutable if there is a proximal repeat. This finding is in agreement with comparative genomic analyses (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">McDonald <em>et al.</em> 2011</a>) and with genome-wide sequencing of the accumulated mutations in mismatch repair defective yeast cells (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ma <em>et al.</em> 2012</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Microsatellites proximal to other repeats are more mutable. (A) The cumulative frequency plots for microsatellites sorted according to the distance to the nearest neighboring repeat for the whole genome (open circles) or for the mutated regions (closed circles) are shown. MATLAB (MathWorks, Inc.) kstest2, Kolmogorov-Smirnov comparison of two data sets, was used to determine the p value, P = 2.8 &#xD7; 10&#x2212;6. The schematic diagram provides an illustration of the relative distance between repeats for the whole genome compared with the mutated microsatellites and the nearest neighboring repeat for a particular point on the graph. (B) The table lists single base substitutions found in regions with immediately adjacent repeats, including homopolymeric runs (HPR), dinucleotide (di), trinucleotide (tri), and tetranucleotide (tetra) microsatellites. The nucleotide sequence is shown and the wild-type base that is mutated in the experimental strain is underlined. The nucleotide change is indicated as is the mutational class. The chromosome position is given for the W303 draft genome (available upon request)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1653639577" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Microsatellites proximal to other repeats are more mutable. (A) The cumulative frequency plots for microsatellites sorted according to the distance to the nearest neighboring repeat for the whole genome (open circles) or for the mutated regions (closed circles) are shown. MATLAB (MathWorks, Inc.) kstest2, Kolmogorov-Smirnov comparison of two data sets, was used to determine the p value, &lt;em&gt;P&lt;/em&gt; = 2.8 &#xD7; 10&lt;sup&gt;&#x2212;6&lt;/sup&gt;. The schematic diagram provides an illustration of the relative distance between repeats for the whole genome compared with the mutated microsatellites and the nearest neighboring repeat for a particular point on the graph. (B) The table lists single base substitutions found in regions with immediately adjacent repeats, including homopolymeric runs (HPR), dinucleotide (di), trinucleotide (tri), and tetranucleotide (tetra) microsatellites. The nucleotide sequence is shown and the wild-type base that is mutated in the experimental strain is underlined. The nucleotide change is indicated as is the mutational class. The chromosome position is given for the W303 draft genome (available upon request).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/9/1453/F3.medium.gif" width="371" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/9/1453/F3.medium.gif" width="371" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463289" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Microsatellites proximal to other repeats are more mutable. (A) The cumulative frequency plots for microsatellites sorted according to the distance to the nearest neighboring repeat for the whole genome (open circles) or for the mutated regions (closed circles) are shown. MATLAB (MathWorks, Inc.) kstest2, Kolmogorov-Smirnov comparison of two data sets, was used to determine the p value, <em>P</em> = 2.8 × 10<sup>−6</sup>. The schematic diagram provides an illustration of the relative distance between repeats for the whole genome compared with the mutated microsatellites and the nearest neighboring repeat for a particular point on the graph. (B) The table lists single base substitutions found in regions with immediately adjacent repeats, including homopolymeric runs (HPR), dinucleotide (di), trinucleotide (tri), and tetranucleotide (tetra) microsatellites. The nucleotide sequence is shown and the wild-type base that is mutated in the experimental strain is underlined. The nucleotide change is indicated as is the mutational class. The chromosome position is given for the W303 draft genome (available upon request).</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">We also used motif finding algorithms to find potential consensus site for single base pair substitutions. One of the most striking motifs represented regions with adjoining repeat sequences (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Based on the elevated mutation rates of mono-, di-, and trinucleotide microsatellites (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>) and on the increased mutability if the repeats are proximal (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, A and B</a>), we speculate that certain single base pair substitutions might, in fact, reflect double slippage events rather than DNA polymerase base substitution errors.</p></div><div id="sec-15" class="subsection"><h3>The mutation spectra of certain <em>msh2</em> alleles differ from the <em>msh2</em> null- and wild-type cells</h3><p id="p-47">As mentioned previously, we find that the mutation frequency spectrum for the combined mismatch repair defective cells included ~6% single base pair substitutions, as well as deletions/insertions ~88% at homopolymers and ~6% at di- and trinucleotide microsatellites. We tested whether any of the strains expressing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> alleles had a different mutation spectrum when compared to the null. Although the missense mutant spectra were not significantly different from the null spectrum (all <em>P</em> &gt; 0.01), five mutants had slightly altered ratios (<em>P</em> &lt; 0.05, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1/TableS6.pdf">Table S6</a>). The differences were primarily accounted for by more insertion/deletions at di- and tri nucleotide repeats.</p><p id="p-48">Mismatch repair defective cells have historically been associated with microsatellite instability, but the distinctive mutational spectrum for single base substitutions was not well established. Because the number of observed base-pair substitutions is low (163), we bolstered this data with a replicate mutation accumulation experiment through 200 generations (A. Gammie, unpublished data). Analysis of the pooled data set revealed that there is a characteristic signature for single-base pair substitutions in mismatch repair defective cells. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> shows the differences between the reported signature for wild-type (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Lynch <em>et al.</em> 2008</a> and references therein) compared with the mismatch repair defective one from our analysis. Unlike wild-type yeast cells, where transversions predominate with G:C &gt; T:A being the most common, mismatch repair defective cells accumulate more transition mutations, particularly G:C &gt; A:T substitutions.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Single-base substitution signature for mismatch repair defective cells. (A) The percentages of each class of single-base substitutions are shown for the pooled mismatch repair defective cells (msh2) and the wild-type reporter construct data (Kunz et al. 1998; Lang and Murray 2008; Ohnishi et al. 2004) compiled by Lynch et al. (i.e., WT Lynch et al.) (Lynch et al. 2008). Transitions and transversions are indicated. The sample size for each strain is given (n). (B) The single-base-pair substitution signatures for the strains completely lacking msh2 function (msh2&#x2206;), for the Lynch et al. (2008) wild-type sequencing data (WT seq Lynch et al.) and the wild-type reporter data (WT Lynch et al.) (Kunz et al. 1998; Lang and Murray 2008; Ohnishi et al. 2004) from panel (A) and for strains expressing missense variants of msh2 indicated on the graph as the amino acid substitution (e.g., P640T, proline at codon 640 in the yeast coding sequence is mutated to a threonine). Only signatures that were statistically different (P &lt; 0.01) from the msh2&#x2206; signature using the Fisher exact test (MATLAB script, Guangdi, &#xA9; 2009) are shown. All but P640L missense substitutions fall in the ATPase domain of Msh2. The sample size for each strain is given (n). Single-base substitutions in this figure represents data pooled from two independent mutation accumulation experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1653639577" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Single-base substitution signature for mismatch repair defective cells. (A) The percentages of each class of single-base substitutions are shown for the pooled mismatch repair defective cells (&lt;em&gt;msh2&lt;/em&gt;) and the wild-type reporter construct data (Kunz &lt;em&gt;et al.&lt;/em&gt; 1998; Lang and Murray 2008; Ohnishi &lt;em&gt;et al.&lt;/em&gt; 2004) compiled by Lynch &lt;em&gt;et al.&lt;/em&gt; (&lt;em&gt;i.e.&lt;/em&gt;, WT Lynch &lt;em&gt;et al.&lt;/em&gt;) (Lynch &lt;em&gt;et al.&lt;/em&gt; 2008). Transitions and transversions are indicated. The sample size for each strain is given (n). (B) The single-base-pair substitution signatures for the strains completely lacking &lt;em&gt;msh2&lt;/em&gt; function (&lt;em&gt;msh2&lt;/em&gt;&#x2206;), for the Lynch &lt;em&gt;et al.&lt;/em&gt; (2008) wild-type sequencing data (WT seq Lynch &lt;em&gt;et al&lt;/em&gt;.) and the wild-type reporter data (WT Lynch &lt;em&gt;et al.&lt;/em&gt;) (Kunz &lt;em&gt;et al.&lt;/em&gt; 1998; Lang and Murray 2008; Ohnishi &lt;em&gt;et al.&lt;/em&gt; 2004) from panel (A) and for strains expressing missense variants of &lt;em&gt;msh2&lt;/em&gt; indicated on the graph as the amino acid substitution (&lt;em&gt;e.g.&lt;/em&gt;, P640T, proline at codon 640 in the yeast coding sequence is mutated to a threonine). Only signatures that were statistically different (&lt;em&gt;P&lt;/em&gt; &lt; 0.01) from the &lt;em&gt;msh2&lt;/em&gt;&#x2206; signature using the Fisher exact test (MATLAB script, Guangdi, &#xA9; 2009) are shown. All but P640L missense substitutions fall in the ATPase domain of Msh2. The sample size for each strain is given (n). Single-base substitutions in this figure represents data pooled from two independent mutation accumulation experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/9/1453/F4.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/9/1453/F4.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/9/1453/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463314" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-49" class="first-child">Single-base substitution signature for mismatch repair defective cells. (A) The percentages of each class of single-base substitutions are shown for the pooled mismatch repair defective cells (<em>msh2</em>) and the wild-type reporter construct data (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kunz <em>et al.</em> 1998</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lang and Murray 2008</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ohnishi <em>et al.</em> 2004</a>) compiled by Lynch <em>et al.</em> (<em>i.e.</em>, WT Lynch <em>et al.</em>) (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Lynch <em>et al.</em> 2008</a>). Transitions and transversions are indicated. The sample size for each strain is given (n). (B) The single-base-pair substitution signatures for the strains completely lacking <em>msh2</em> function (<em>msh2</em>∆), for the <a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Lynch <em>et al.</em> (2008)</a> wild-type sequencing data (WT seq Lynch <em>et al</em>.) and the wild-type reporter data (WT Lynch <em>et al.</em>) (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Kunz <em>et al.</em> 1998</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Lang and Murray 2008</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Ohnishi <em>et al.</em> 2004</a>) from panel (A) and for strains expressing missense variants of <em>msh2</em> indicated on the graph as the amino acid substitution (<em>e.g.</em>, P640T, proline at codon 640 in the yeast coding sequence is mutated to a threonine). Only signatures that were statistically different (<em>P</em> &lt; 0.01) from the <em>msh2</em>∆ signature using the Fisher exact test (MATLAB script, Guangdi, © 2009) are shown. All but P640L missense substitutions fall in the ATPase domain of Msh2. The sample size for each strain is given (n). Single-base substitutions in this figure represents data pooled from two independent mutation accumulation experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">We tested whether any of the 16 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> missense variants displayed a unique spectrum of base-pair substitutions when compared to wild-type or the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em> null. As noted previously and in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, three strains suffered plasmid rearrangements early in the passaging and were subsequently treated as true nulls. The single-base pair mutation distribution from these strains were combined with the null (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a>∆</em> + vector) and the spectrum was found to be statistically different when compared to the reported values for wild-type using χ<sup>2</sup> analysis (<em>P</em> = 4.82 × 10<sup>−8</sup>) and Fisher exact tests (<em>P</em> = 0.01). Several of the missense variants showed differences (<em>P</em> ≤ 0.01) from the null set using the Fisher Exact test (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). On the basis of our previous characterization of these variants (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Gammie <em>et al.</em> 2007</a>), we observed that these particular missense alleles express detectable quantities of the defective protein with alterations that mostly affected the ATPase domain (G688D, G693R, S742F; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). We found that removal of the strains with statistical differences (<em>P</em> &lt; 0.01) from the aggregate data set did not significantly affect our calculations of mutation rates or mutational spectra.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>The mutation rate in the absence of mismatch repair</h3><p id="p-51">Mutations in mismatch repair proteins, among the strongest elevators of mutation rate (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Huang <em>et al.</em> 2003</a>), are commonly observed in long-term evolution experiments as well as in commensal and pathogenic strains (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">LeClerc <em>et al.</em> 1996</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Matic <em>et al.</em> 1997</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Oliver <em>et al.</em> 2000</a>) and are associated with Lynch syndrome, a heritable predisposition to cancer (reviewed in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">da Silva <em>et al.</em> 2009</a>). Yet, despite the importance of the mismatch repair mechanism, we have an incomplete understanding of the mutation rate and spectra associated with defects in mismatch repair. Previous calculations placed the fold-increase in mutation rate for mismatch repair defective cells between 10<sup>1</sup> and 10<sup>4</sup> (reviewed in <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Kunkel and Erie 2005</a>). The large range is attributable to the variable mutability of different sequences. For example, homopolymeric runs have been shown to have as high as a 5 × 10<sup>4</sup>-fold increase in mutation rates in mismatch repair defective yeast (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Tran <em>et al.</em> 1997</a>); whereas the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em> locus shows only a 40-fold elevation (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Marsischky <em>et al.</em> 1996</a>). Traditionally, mutation rate estimates are made at individual reporter loci. Here we report whole genome sequencing of 16 mutation accumulation lines containing mismatch repair defective alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">msh2</a></em>. By assaying the accumulation of mutations genome-wide, this method averages over differences at individual loci to provide an accurate estimate of the per-genome per-generation mutation rate in mismatch repair defective cells. We find that the average mutation rate for mismatch repair defective cells is 7.5 × 10<sup>−8</sup> mutations per base pair per generation, corresponding to approximately one mutation per genome per generation. This is consistent with a recent mutation accumulation experiment using a mismatch repair deficient, temperature-sensitive mutation in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004777">mlh1</a></em> (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Zanders <em>et al.</em> 2010</a>). Our true wild-type line, in contrast, accumulated only a single mutation over the 170 generations of growth, consistent with previous estimates of the wild-type per-base pair, per-generation mutation rate on the order of 10<sup>−10</sup>, or one mutation ever few hundred generations (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Drake 1991</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Lang and Murray 2008</a>; <a id="xref-ref-51-7" class="xref-bibr" href="#ref-51">Lynch <em>et al.</em> 2008</a>).</p></div><div id="sec-18" class="subsection"><h3>Why chromosomal and replication timing effects disappear in mismatch repair defective cells</h3><p id="p-52">Previous work has demonstrated a correlation between mutation rate and replication timing (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Agier and Fischer 2012</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lang and Murray 2011</a>). We find, however, no correlation between mutation rate and replication timing in mismatch repair deficient lines. Our data are consistent with a random distribution of mutations across the genome as would be expected if mismatch repair has an equal opportunity to correct replication errors across the genome. This is supported by the previous observation that removing mismatch repair decreases the position effects on mutation rate (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Hawk <em>et al.</em> 2005</a>).</p><p id="p-53">A previous study has implicated the action of translesion polymerases on late-replicating regions as a possible mechanism underlying the correlation between mutation rate and replication timing in mismatch repair proficient cells (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Lang and Murray 2008</a>). If mismatch repair were capable of correcting errors introduced by translesion polymerases, one would expect the absence of mismatch repair to exacerbate the correlation between replication timing and mutation rate. We do not see this, nor do we observe any mutations with the characteristic spectra of translesion polymerases. Overall the genome-wide distribution and spectra of mutations in mismatch repair deficient lines is consistent with mismatch repair correcting errors by the replicative, but not translesion polymerases.</p></div><div id="sec-19" class="subsection"><h3>The mutation rate at homopolymeric runs and microsatellite sequences increases with length in the absence of mismatch repair</h3><p id="p-54">The mismatch repair machinery is responsible for binding and repairing insertion/deletion loops that go undetected by the DNA polymerase proof-reading function (reviewed in <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hsieh and Yamane 2008</a>). Interesting, when the repeat length of microsatellites surpasses 8−10 base pairs, the insertion/deletion loop is postulated to have the capacity to be propagated to a region outside the proof-reading domain of the DNA polymerase (reviewed in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bebenek <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Garcia-Diaz and Kunkel 2006</a>). The data presented in this paper show that in the absence of mismatch repair, the mutation rate increases exponentially with repeat length for both homopolymeric runs and larger microsatellites and switches to a linear increase as the repeat unit surpasses eight. If the threshold model is correct, there is an increased need for DNA mismatch repair to capture the unrepaired insertion/deletion loops as the microsatellite increases in length. This model, in part, explains the wide range of estimates for the effect of mismatch repair on mutation rate based on individual reporter loci.</p><p id="p-55">Previously, several groups have attempted to determine in yeast whether a threshold exists, above which the repeats are unstable, and below which the mutability is indistinguishable from the background mutation (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Pupko and Graur 1999</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Rose and Falush 1998</a>). We find mutations in homopolymeric runs as small as four nucleotides and mutations in microsatellites as small as three repeat units, or six nucleotides. Our findings that small repeats are mutable in the absence of mismatch repair are consistent with data from reporter constructs using homopolymeric repeats (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Marsischky <em>et al.</em> 1996</a>; <a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Tran <em>et al.</em> 1997</a>). Taken together, the data suggest that, if a threshold exists for increased mutability of homopolymers and microsatellites in the absence of mismatch repair, it is small.</p></div><div id="sec-20" class="subsection"><h3>Model for insertion-deletion biases at microsatellites</h3><p id="p-56">Insertion/deletion mutations at microsatellites are thought to occur as a consequence of unrepaired DNA polymerase “slippage” events (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Levinson and Gutman 1987</a>). The genome-wide insertion/deletion mutation results in this work are in best agreement with previous <em>in vivo</em> reporter assays that did not bias the mutational event with reading frame constraints. These previous analyses revealed that in the absence of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">MSH2</a></em>, homopolymers (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Denver <em>et al.</em> 2005</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Gragg <em>et al.</em> 2002</a>; <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Marsischky <em>et al.</em> 1996</a>) and (GT/CA)<sub>n</sub> di-nucleotide microsatellites (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Hawk <em>et al.</em> 2005</a>) are more likely to suffer a single unit deletion. We speculate that the deletion bias is likely to be a consequence of DNA polymerase errors. Specifically, compelling crystal structure data revealed examples of DNA polymerase bound to DNA containing a single nucleotide deletion loop where the unpaired base is in the template strand (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bebenek <em>et al.</em> 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Garcia-Diaz <em>et al.</em> 2006</a>). If such events were to go unrepaired <em>in vivo</em>, the newly synthesized strand would have a single nucleotide deletion. In addition, the (GT/CA)<sub>n</sub> di-nucleotide deletion bias was observed <em>in vitro</em> with purified yeast replicative DNA polymerases using a gap filling assay (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdulovic <em>et al.</em> 2011</a>). Thus, DNA polymerase errors could account for the deletion bias at mono- and certain dinucleotide microsatellites.</p><p id="p-57">In contrast, we observed an insertion bias at (AT/TA)<sub>n</sub> di-nucleotides as well as some trinucleotide microsatellites. The bias toward insertion mutations at these sites might be attributed to the fact that most microsatellites have the capacity to form stable, complex non-B DNA structures <em>in vitro</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kelkar <em>et al.</em> 2010</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Richard <em>et al.</em> 2008</a>). In some cases the secondary structure−forming microsatellites have been shown to inhibit DNA polymerase (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baran <em>et al.</em> 1991</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Shah <em>et al.</em> 2010b</a>). Although proving that such structures form <em>in vivo</em> is difficult, microsatellites are often sites of chromosome fragility, a phenomenon typically attributed to secondary structure formation and replication fork collapse (reviewed in <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Freudenreich 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fungtammasan <em>et al.</em> 2012</a>). We hypothesize that the formation of certain structures at microsatellites may cause increased pausing or switching of the DNA polymerase, thereby increasing the likelihood of the newly synthesized strand to become misaligned with the template. To fit the data, the (AT/TA)<sub>n</sub> misalignment would have to occur with a bias toward slipping “back” one unit such that when the polymerase restarts, an extra unit will be introduced in the newly synthesized strand.</p></div><div id="sec-21" class="subsection"><h3>Model for mutability of a microsatellite proximal to another repeat</h3><p id="p-58">In this work, we demonstrate that in the absence of mismatch repair, microsatellite repeats with proximal repeats are more likely to be mutated. This finding is in keeping with recent work describing mutational hot spots among clustered homopolymeric sequences (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Ma <em>et al.</em> 2012</a>). Additionally, comparative genomics suggests that the presence of a repeat increases the mutability of the region (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">McDonald <em>et al.</em> 2011</a>). Several explanations exist for the increased mutability of repeats with proximal repeats, including the possibility of altered chromatin or transcriptional activity, or decreased replication efficiency (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Ma <em>et al.</em> 2012</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">McDonald <em>et al.</em> 2011</a>).</p><p id="p-59">As mentioned previously, microsatellite repeats have the capacity to form an array of non-B DNA structures that decrease the fidelity of the polymerase (reviewed in <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Richard <em>et al.</em> 2008</a>). Proximal repeats have the capacity to produce complex structural regions. For example, a well-documented chromosomal fragility site depends on an (AT/TA)<sub>24</sub> dinucleotide repeat as well as a proximal (A/T)<sub>19-28</sub> homopolymeric repeat for the formation of a replication fork inhibiting (AT/TA)<sub>n</sub> cruciform (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Shah <em>et al.</em> 2010b</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Zhang and Freudenreich 2007</a>). Additionally, parent-child analyses revealed that microsatellites with proximal repeats were more likely to be mutated (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dupuy <em>et al.</em> 2004</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Eckert and Hile 2009</a>). Finally, recent work demonstrated that a triplet repeat region inhibits the function of mismatch repair (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lujan <em>et al.</em> 2012</a>). Taken together, we predict that the more complex secondary structures found at proximal repeats will increase the likelihood of DNA polymerase stalling or switching. At least two subsequent fates could account for an increase of insertion/deletions. First, the template and newly synthesized strand could misalign with the bulge outside of the DNA polymerase proof-reading domain. Second, if a lower-fidelity polymerase is installed at the paused replisome, the chances of an adjacent repeat or single base pairs in the vicinity becoming mutated would increase (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">McDonald <em>et al.</em> 2011</a>). We further predict that mismatch repair function is not likely to be associated with error-prone polymerases and this could explain why some repeat regions might appear to inhibit mismatch repair.</p></div><div id="sec-22" class="subsection"><h3>The most common mutations in mismatch repair defective tumors are likely to be insertion/deletions at homopolymeric runs</h3><p id="p-60">On the basis of the mutational signature we observed in yeast we predict that ~90% of the mutational events in a mismatch repair defective tumor will be single-base insertion/deletions within homopolymers, particularly those with proximal repeats. This prediction is based on the observations that humans and yeast are remarkably similar with respect to (1) the percentage of total microsatellite DNA (~3% in humans and ~4% in yeast; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lim <em>et al.</em> 2004</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Subramanian <em>et al.</em> 2003</a>), (2) the density of microsatellites (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Richard <em>et al.</em> 2008</a>), and (3) homopolymer to larger microsatellite ratio (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Lim <em>et al.</em> 2004</a>; <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Richard <em>et al.</em> 2008</a>).</p><p id="p-61">Interestingly, the redundancy of MutSα (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">Msh2</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002504">Msh6</a>) and MutSβ (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">Msh2</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000688">Msh3</a>) in recognizing a single-nucleotide insertion/deletion loop at homopolymeric runs (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Acharya <em>et al.</em> 1996</a>; <a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Marsischky <em>et al.</em> 1996</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Palombo <em>et al.</em> 1996</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Umar <em>et al.</em> 1998</a>) ensures that the most common mismatch generated during replication is likely to be identified and repaired. In keeping with this, tumor formation rarely arises as a consequence of loss of only <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002504">Msh6</a> or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000688">Msh3</a> (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">de la Chapelle 2004</a>). It will be of interest to determine whether the entire panel of rare <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002504">MSH6</a></em> Lynch Syndrome alleles confers a dominant negative function as has been previously reported for a variant of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002504">MSH6</a></em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Geng <em>et al.</em> 2012</a>).</p><p id="p-62">Given the mismatch repair deficiency mutation spectrum, we further predict that the drivers of tumor formation are likely to be genes that contain homopolymers with proximal repeats. Homopolymers and microsatellites represent unique challenges for whole genome sequencing algorithms designed to call mutations, resulting in a lower efficiency of confidently finding insertion/deletion mutations. For this reason, the candidate gene approaches are still commonly used when trying to determine cancer drivers in mutator tumor cells (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">The Cancer Genome Network 2012</a>). Candidate cancer drivers encoding homopolymeric or larger microsatellite repeats have been extensively examined in mutator tumor cell lines; for example many potential drivers with homopolymeric runs, such as TGFBRII, are found to be frequently mutated in mismatch repair defective tumors (reviewed in <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kim <em>et al.</em> 2010</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Li <em>et al.</em> 2004</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Shah <em>et al.</em> 2010a</a>). Challenges in identifying true drivers in tumors with a high rate of mutation still remain because it is difficult to determine if an identified mutation was causative or simply a consequence of the repair defect. Additionally, finding novel tumor drivers is difficult because of the difficulty of whole genome sequencing in calling mutations at homopolymers and microsatellites. Going forward, computational approaches should allow for the detection of novel potential drivers based on the mutability of repeats with proximal repeats.</p><p id="p-63">In this study, we have shown that the combination of mutation accumulation assays and next-generation sequencing is a powerful general method for revealing the genome-wide rate, spectra, and distribution of mutations in lines harboring Lynch Syndrome associated variants of the mismatch repair protein, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005450">Msh2</a>. These data provide mechanistic insight into the mutagenic processes in the absence of mismatch repair and has potential as a tool for identifying target loci that contribute to the progression of this disease.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">We thank the following students who participated in a graduate level project-based course for which this project was designed: Thomas Bartlett, Derek Clay, Geoffrey Dann, Whitby Eagle, Hendia Edmund, Karla Frietze, John Fuesler, Daniela Garcia, Carly Lay Geronimo, Megan Gladwin, Bobak Hadidi, Allison Hall, Alexandria Hammons, Matthew Howard, Hao Huang, Joseph Koos, Vikram Kumar, Wenyang Li, Kelsi Lindblad, Kinnari Matheson, Scott Mellon, Donald Miller, Nicholas Morante, Emily Nelson, Nettie Pyne, Cesar Perez Ramirez, Gregory Shimamura, Jean Smith, Joel Tamayo, Colin Watson, Julia Wittes, and Christopher Wright. We also thank Wei Wang, Donna Storton, and Jessica Wiggins at the Lewis-Sigler Institute for Integrative Genomics Sequencing Core Facility for their assistance with high-throughput sequencing. This research was supported by Princeton University Teaching Funds and a New Jersey Commission on Cancer Research (NJCCR) Seed Grant 10-1064-CCR-E0 awarded to A.G. The Princeton Sequencing Core Facility is supported in part by the National Institute of General Medical Sciences (NIGMS) [NIH grant P50 GM071508].</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006429/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">The sequencing data are available through NCBI (SRA Study Accession Number SRP026313).</p></li><li class="fn" id="fn-11"><p id="p-65"><em>Communicating editor: J. Rine</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-04-15"><span class="received-label">Received </span>April 15, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-06-19"><span class="accepted-label">Accepted </span>June 19, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Lang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.1" data-doi="10.1016/j.dnarep.2011.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdulovic</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The in vitro fidelity of yeast DNA polymerase delta and polymerase epsilon holoenzymes during dinucleotide microsatellite DNA synthesis.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D10%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2011.02.003%26rft_id%253Dinfo%253Apmid%252F21429821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2011.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429821&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.2" data-doi="10.1073/pnas.93.24.13629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gradia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerrette</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">hMSH2 forms specific mispair-binding complexes with hMSH3 and hMSH6.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">13629</span>–<span class="cit-lpage">13634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAcharya%26rft.auinit1%253DS.%26rft.volume%253D93%26rft.issue%253D24%26rft.spage%253D13629%26rft.epage%253D13634%26rft.atitle%253DhMSH2%2Bforms%2Bspecific%2Bmispair-binding%2Bcomplexes%2Bwith%2BhMSH3%2Band%2BhMSH6%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.24.13629%26rft_id%253Dinfo%253Apmid%252F8942985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTMvMjQvMTM2MjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.3" data-doi="10.1093/molbev/msr280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mutational profile of the yeast genome is shaped by replication.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr280%26rft_id%253Dinfo%253Apmid%252F22114361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI5LzMvOTA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.4" data-doi="10.1002/jcp.1067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aquilina</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bignami</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mismatch repair in correction of replication errors and processing of DNA damage.</span> <abbr class="cit-jnl-abbrev">J. Cell. Physiol.</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcellular%2Bphysiology%26rft.stitle%253DJ%2BCell%2BPhysiol%26rft.aulast%253DAquilina%26rft.auinit1%253DG.%26rft.volume%253D187%26rft.issue%253D2%26rft.spage%253D145%26rft.epage%253D154%26rft.atitle%253DMismatch%2Brepair%2Bin%2Bcorrection%2Bof%2Breplication%2Berrors%2Band%2Bprocessing%2Bof%2BDNA%2Bdamage.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcp.1067%26rft_id%253Dinfo%253Apmid%252F11267994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcp.1067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11267994&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167834800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.5" data-doi="10.1073/pnas.1215510110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arlow</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagenseller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gammie</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Proteasome inhibition rescues clinically significant unstable variants of the mismatch repair protein Msh2.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215510110%26rft_id%253Dinfo%253Apmid%252F23248292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMS8yNDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1453.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingston</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scidman</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source">Current Protocols in Molecular Biology</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.7" data-doi="10.1073/pnas.88.2.507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapidot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manor</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Formation of DNA triplexes accounts for arrests of DNA synthesis at d(TC)n and d(GA)n tracts.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBaran%26rft.auinit1%253DN.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D507%26rft.epage%253D511%26rft.atitle%253DFormation%2Bof%2BDNA%2Btriplexes%2Baccounts%2Bfor%2Barrests%2Bof%2BDNA%2Bsynthesis%2Bat%2Bd%2528TC%2529n%2Band%2Bd%2528GA%2529n%2Btracts.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.2.507%26rft_id%253Dinfo%253Apmid%252F1988950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4OC8yLzUwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.8" data-doi="10.1038/embor.2008.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bebenek</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Diaz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlick</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Substrate-induced DNA strand misalignment during catalytic cycling by DNA polymerase lambda.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BReports%26rft.stitle%253DEMBO%2BReports%26rft.aulast%253DBebenek%26rft.auinit1%253DK.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D459%26rft.epage%253D464%26rft.atitle%253DSubstrate-induced%2BDNA%2Bstrand%2Bmisalignment%2Bduring%2Bcatalytic%2Bcycling%2Bby%2BDNA%2Bpolymerase%2Blambda.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2008.33%26rft_id%253Dinfo%253Apmid%252F18369368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/embor.2008.33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18369368&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255502300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.9" data-doi="10.1007/s12033-009-9230-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhargava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuentes</span>  <span class="cit-name-given-names">F. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutational dynamics of microsatellites.</span> <abbr class="cit-jnl-abbrev">Mol. Biotechnol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiotechnology%26rft.stitle%253DMol%2BBiotechnol%26rft.aulast%253DBhargava%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D250%26rft.epage%253D266%26rft.atitle%253DMutational%2Bdynamics%2Bof%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12033-009-9230-4%26rft_id%253Dinfo%253Apmid%252F20012711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12033-009-9230-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20012711&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274303900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1453.10"><div class="cit-metadata unstructured">Blankenberg, D., G. Von Kuster, N. Coraor, G. Ananda, R. Lazarus <em>et al.</em>, 2010 Galaxy: a web-based genome analysis tool for experimentalists. Curr Protoc Mol Biol 19: Unit 19 10 11–21.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.11" data-doi="10.1016/S1470-2045(12)70134-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boland</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Lynch syndrome: new tales from the crypt.</span> <abbr class="cit-jnl-abbrev">Lancet Oncol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Blancet%2Boncology%26rft.stitle%253DLancet%2BOncol%26rft.aulast%253DBoland%26rft.auinit1%253DC.%2BR.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D562%26rft.epage%253D564%26rft.atitle%253DLynch%2Bsyndrome%253A%2Bnew%2Btales%2Bfrom%2Bthe%2Bcrypt.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1470-2045%252812%252970134-1%26rft_id%253Dinfo%253Apmid%252F22552010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1470-2045(12)70134-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22552010&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stearns</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth cit-collab">Cold Spring Harbor Laboratory</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <abbr class="cit-jnl-abbrev">Methods in Yeast Genetics: A Cold Spring Harbor Laboratory Course Manual</abbr>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Plainview, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.13" data-doi="10.1038/387067a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juvik</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic and physical maps of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCherry%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D387%26rft.issue%253D6632%2BSuppl%26rft.spage%253D67%26rft.epage%253D73%26rft.atitle%253DGenetic%2Band%2Bphysical%2Bmaps%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F387067a0%26rft_id%253Dinfo%253Apmid%252F9169866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/387067a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9169866&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1453.14" data-doi="10.1016/B978-0-12-394280-7.00004-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svrcek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flejou</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Lynch or not Lynch? Is that always a question?</span> pp. <span class="cit-fpage">121</span>–<span class="cit-lpage">166</span> in <span class="cit-source">Advances in Cancer Research</span>, <span class="cit-vol">Vol. 113</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Tew</span>  <span class="cit-name-given-names">K. D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">P. B.</span></span>. <span class="cit-publ-name">Elsevier Academic Press Inc</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bcancer%2Bresearch%26rft.stitle%253DAdv%2BCancer%2BRes%26rft.aulast%253DColas%26rft.auinit1%253DC.%26rft.volume%253D113%26rft.spage%253D121%26rft.epage%253D166%26rft.atitle%253DLynch%2Bor%2Bnot%2BLynch%253F%2BIs%2Bthat%2Balways%2Ba%2Bquestion%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-394280-7.00004-X%26rft_id%253Dinfo%253Apmid%252F22429854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-394280-7.00004-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22429854&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303133000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.15" data-doi="10.1590/S1516-31802009000100010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Silva</span>  <span class="cit-name-given-names">F. C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valentin</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carraro</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mismatch repair genes in Lynch syndrome: a review.</span> <abbr class="cit-jnl-abbrev">Sao Paulo Med. J.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DS%25C3%25A3o%2BPaulo%2Bmedical%2Bjournal%2B%253D%2BRevista%2Bpaulista%2Bde%2Bmedicina%26rft.stitle%253DSao%2BPaulo%2BMed%2BJ%26rft.aulast%253DSilva%26rft.auinit1%253DF.%2BC.%26rft.volume%253D127%26rft.issue%253D1%26rft.spage%253D46%26rft.epage%253D51%26rft.atitle%253DMismatch%2Brepair%2Bgenes%2Bin%2BLynch%2Bsyndrome%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS1516-31802009000100010%26rft_id%253Dinfo%253Apmid%252F19466295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S1516-31802009000100010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19466295&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266149800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.16" data-doi="10.1038/nrc1453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de la Chapelle</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic predisposition to colorectal cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253Dde%2Bla%2BChapelle%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253D769%26rft.epage%253D780%26rft.atitle%253DGenetic%2Bpredisposition%2Bto%2Bcolorectal%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc1453%26rft_id%253Dinfo%253Apmid%252F15510158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc1453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15510158&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224245700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.17" data-doi="10.1534/genetics.104.038521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feinberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mutation rates, spectra and hotspots in mismatch repair-deficient <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038521%26rft_id%253Dinfo%253Apmid%252F15716493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzEvMTA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.18" data-doi="10.1073/pnas.88.16.7160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A constant rate of spontaneous mutation in DNA-based microbes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">7160</span>–<span class="cit-lpage">7164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDrake%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D88%26rft.issue%253D16%26rft.spage%253D7160%26rft.epage%253D7164%26rft.atitle%253DA%2Bconstant%2Brate%2Bof%2Bspontaneous%2Bmutation%2Bin%2BDNA-based%2Bmicrobes.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.16.7160%26rft_id%253Dinfo%253Apmid%252F1831267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMTYvNzE2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.19" data-doi="10.1002/humu.10294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dupuy</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenersen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egeland</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olaisen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Y-chromosomal microsatellite mutation rates: differences in mutation rate between and within loci.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.stitle%253DHum%2BMutat%26rft.aulast%253DDupuy%26rft.auinit1%253DB.%2BM.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D117%26rft.epage%253D124%26rft.atitle%253DY-chromosomal%2Bmicrosatellite%2Bmutation%2Brates%253A%2Bdifferences%2Bin%2Bmutation%2Brate%2Bbetween%2Band%2Bwithin%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.10294%26rft_id%253Dinfo%253Apmid%252F14722915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.10294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14722915&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188715800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.20" data-doi="10.1002/mc.20499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Every microsatellite is different: intrinsic DNA features dictate mutagenesis of common microsatellites present in the human genome.</span> <abbr class="cit-jnl-abbrev">Mol. Carcinog.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcarcinogenesis%26rft.stitle%253DMol%2BCarcinog%26rft.aulast%253DEckert%26rft.auinit1%253DK.%2BA.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D379%26rft.epage%253D388%26rft.atitle%253DEvery%2Bmicrosatellite%2Bis%2Bdifferent%253A%2BIntrinsic%2BDNA%2Bfeatures%2Bdictate%2Bmutagenesis%2Bof%2Bcommon%2Bmicrosatellites%2Bpresent%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fmc.20499%26rft_id%253Dinfo%253Apmid%252F19306292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/mc.20499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19306292&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264918500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.21" data-doi="10.2741/2437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freudenreich</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Chromosome fragility: molecular mechanisms and cellular consequences.</span> <abbr class="cit-jnl-abbrev">Front. Biosci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">4911</span>–<span class="cit-lpage">4924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bbioscience%2B%253A%2B%2Ba%2Bjournal%2Band%2Bvirtual%2Blibrary%26rft.stitle%253DFront%2BBiosci%26rft.aulast%253DFreudenreich%26rft.auinit1%253DC.%2BH.%26rft.volume%253D12%26rft.spage%253D4911%26rft.epage%253D4924%26rft.atitle%253DChromosome%2Bfragility%253A%2Bmolecular%2Bmechanisms%2Band%2Bcellular%2Bconsequences.%26rft_id%253Dinfo%253Adoi%252F10.2741%252F2437%26rft_id%253Dinfo%253Apmid%252F17569619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2741/2437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17569619&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253943900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.22" data-doi="10.1101/gr.134395.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fungtammasan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiaromonte</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makova</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A genome-wide analysis of common fragile sites: What features determine chromosomal instability in the human genome?</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">993</span>–<span class="cit-lpage">1005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134395.111%26rft_id%253Dinfo%253Apmid%252F22456607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzYvOTkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.23" data-doi="10.1534/genetics.107.071084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gammie</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erdeniz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaver</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nanji</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functional characterization of pathogenic human MSH2 missense mutations in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071084%26rft_id%253Dinfo%253Apmid%252F17720936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzIvNzA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.24" data-doi="10.1016/j.tibs.2006.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Diaz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mechanism of a genetic glissando<sup>*</sup>: structural biology of indel mutations.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DGarcia-Diaz%26rft.auinit1%253DM.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D206%26rft.epage%253D214%26rft.atitle%253DMechanism%2Bof%2Ba%2Bgenetic%2Bglissando%253A%2Bstructural%2Bbiology%2Bof%2Bindel%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2006.02.004%26rft_id%253Dinfo%253Apmid%252F16545956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2006.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16545956&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237259600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.25" data-doi="10.1016/j.cell.2005.10.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Diaz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bebenek</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krahn</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structural analysis of strand misalignment during DNA synthesis by a human DNA polymerase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGarcia-Diaz%26rft.auinit1%253DM.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D331%26rft.epage%253D342%26rft.atitle%253DStructural%2Banalysis%2Bof%2Bstrand%2Bmisalignment%2Bduring%2BDNA%2Bsynthesis%2Bby%2Ba%2Bhuman%2BDNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.10.039%26rft_id%253Dinfo%253Apmid%252F16439207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.10.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16439207&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235068500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-3.9.1453.26"><div class="cit-metadata unstructured">Garrison, E., and G. Marth, 2012  Haplotype-based variant detection from short-read sequencing. Available at: <a href="http://arxiv.org/pdf/1207.3907v2.pdf">http://arxiv.org/pdf/1207.3907v2.pdf</a>. Accessed July 9, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.27" data-doi="10.1146/annurev-genet-072610-155046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gemayel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinces</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variable tandem repeats accelerate evolution of coding and regulatory sequences.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DGemayel%26rft.auinit1%253DR.%26rft.volume%253D44%26rft.spage%253D445%26rft.epage%253D477%26rft.atitle%253DVariable%2Btandem%2Brepeats%2Baccelerate%2Bevolution%2Bof%2Bcoding%2Band%2Bregulatory%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-072610-155046%26rft_id%253Dinfo%253Apmid%252F20809801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-072610-155046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20809801&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286042600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.28" data-doi="10.1074/jbc.M111.316919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeRocco</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamane</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Biochemical analysis of the human mismatch repair proteins hMutS alpha MSH2(G674A)-MSH6 and MSH2–MSH6(T1219D).</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">9777</span>–<span class="cit-lpage">9791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.316919%26rft_id%253Dinfo%253Apmid%252F22277660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyODcvMTMvOTc3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.29" data-doi="10.1101/gr.4086505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giardine</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riemer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardison</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burhans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elnitski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Galaxy: a platform for interactive large-scale genome analysis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4086505%26rft_id%253Dinfo%253Apmid%252F16169926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.30" data-doi="10.1186/gb-2010-11-8-r86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goecks</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nekrutenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGoecks%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253DR86%26rft.epage%253DR86%26rft.atitle%253DGalaxy%253A%2Ba%2Bcomprehensive%2Bapproach%2Bfor%2Bsupporting%2Baccessible%252C%2Breproducible%252C%2Band%2Btransparent%2Bcomputational%2Bresearch%2Bin%2Bthe%2Blife%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-8-r86%26rft_id%253Dinfo%253Apmid%252F20738864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-8-r86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20738864&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.31" data-doi="10.1128/MCB.22.24.8756-8762.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gragg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harfe</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks-Robertson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Base composition of mononucleotide runs affects DNA polymerase slippage and removal of frameshift intermediates by mismatch repair in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">8756</span>–<span class="cit-lpage">8762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGragg%26rft.auinit1%253DH.%26rft.volume%253D22%26rft.issue%253D24%26rft.spage%253D8756%26rft.epage%253D8762%26rft.atitle%253DBase%2BComposition%2Bof%2BMononucleotide%2BRuns%2BAffects%2BDNA%2BPolymerase%2BSlippage%2Band%2BRemoval%2Bof%2BFrameshift%2BIntermediates%2Bby%2BMismatch%2BRepair%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.24.8756-8762.2002%26rft_id%253Dinfo%253Apmid%252F12446792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yNC84NzU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.32" data-doi="10.1073/pnas.0503415102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawk</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefanovic</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Variation in efficiency of DNA mismatch repair at different sites in the yeast genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">8639</span>–<span class="cit-lpage">8643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0503415102%26rft_id%253Dinfo%253Apmid%252F15932942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzI0Lzg2MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.33" data-doi="10.1128/MCB.12.6.2749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Instability of simple sequence DNA in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2749</span>–<span class="cit-lpage">2757</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHenderson%26rft.auinit1%253DS%2BT%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D2749%26rft.epage%253D2757%26rft.atitle%253DInstability%2Bof%2Bsimple%2Bsequence%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.12.6.2749%26rft_id%253Dinfo%253Apmid%252F1588966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEyLzYvMjc0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.34" data-doi="10.1016/j.mad.2008.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamane</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">DNA mismatch repair: molecular mechanism, cancer, and ageing.</span> <abbr class="cit-jnl-abbrev">Mech. Ageing Dev.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bageing%2Band%2Bdevelopment%26rft.stitle%253DMech%2BAgeing%2BDev%26rft.aulast%253DHsieh%26rft.auinit1%253DP.%26rft.volume%253D129%26rft.issue%253D7-8%26rft.spage%253D391%26rft.epage%253D407%26rft.atitle%253DDNA%2Bmismatch%2Brepair%253A%2Bmolecular%2Bmechanism%252C%2Bcancer%252C%2Band%2Bageing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mad.2008.02.012%26rft_id%253Dinfo%253Apmid%252F18406444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mad.2008.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18406444&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257816100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.35" data-doi="10.1073/pnas.2035018100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rio</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A genomewide screen in <em>Saccharomyces cerevisiae</em> for genes that suppress the accumulation of mutations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">11529</span>–<span class="cit-lpage">11534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2035018100%26rft_id%253Dinfo%253Apmid%252F12972632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIwLzExNTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ito-Harashima</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartzog</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The tRNA-Tyr gene family of <em>Saccharomyces cerevisiae</em>: Agents of phenotypic variation and position effects on mutation frequency.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1395</span>–<span class="cit-lpage">1410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DIto-Harashima%26rft.auinit1%253DS.%26rft.volume%253D161%26rft.issue%253D4%26rft.spage%253D1395%26rft.epage%253D1410%26rft.atitle%253DThe%2BtRNA-Tyr%2BGene%2BFamily%2Bof%2BSaccharomyces%2Bcerevisiae%253A%2BAgents%2Bof%2BPhenotypic%2BVariation%2Band%2BPosition%2BEffects%2Bon%2BMutation%2BFrequency%26rft_id%253Dinfo%253Apmid%252F12196388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MS80LzEzOTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.37" data-doi="10.1093/gbe/evq046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelkar</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strubczewski</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiaromonte</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">What is a microsatellite: a computational and experimental definition based upon repeat mutational behavior at A/T and GT/AC repeats.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq046%26rft_id%253Dinfo%253Apmid%252F20668018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjIvMC82MjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Novel somatic frameshift mutations of genes related to cell cycle and DNA damage response in gastric and colorectal cancers with microsatellite instability.</span> <abbr class="cit-jnl-abbrev">Tumori</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTumori%26rft.stitle%253DTumori%26rft.aulast%253DKim%26rft.auinit1%253DY.%2BR.%26rft.volume%253D96%26rft.issue%253D6%26rft.spage%253D1004%26rft.epage%253D1009%26rft.atitle%253DNovel%2Bsomatic%2Bframeshift%2Bmutations%2Bof%2Bgenes%2Brelated%2Bto%2Bcell%2Bcycle%2Band%2BDNA%2Bdamage%2Bresponse%2Bin%2Bgastric%2Band%2Bcolorectal%2Bcancers%2Bwith%2Bmicrosatellite%2Binstability.%26rft_id%253Dinfo%253Apmid%252F21388066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21388066&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288578600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.39" data-doi="10.1146/annurev.biochem.74.082803.133243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erie</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">DNA mismatch repair.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">710</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DKunkel%26rft.auinit1%253DT.%2BA.%26rft.volume%253D74%26rft.spage%253D681%26rft.epage%253D710%26rft.atitle%253DDNA%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.74.082803.133243%26rft_id%253Dinfo%253Apmid%252F15952900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.74.082803.133243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15952900&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231235100023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vonarx</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">DNA sequence analysis of spontaneous mutagenesis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKunz%26rft.auinit1%253DB.%2BA.%26rft.volume%253D148%26rft.issue%253D4%26rft.spage%253D1491%26rft.epage%253D1505%26rft.atitle%253DDNA%2BSequence%2BAnalysis%2Bof%2BSpontaneous%2BMutagenesis%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F9560369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OC80LzE0OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.41" data-doi="10.1534/genetics.107.071506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimating the per-base-pair mutation rate in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLang%26rft.auinit1%253DG.%2BI.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D82%26rft.atitle%253DEstimating%2Bthe%2BPer-Base-Pair%2BMutation%2BRate%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071506%26rft_id%253Dinfo%253Apmid%252F18202359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc4LzEvNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.42" data-doi="10.1093/gbe/evr054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutation rates across budding yeast chromosome VI are correlated with replication timing.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr054%26rft_id%253Dinfo%253Apmid%252F21666225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC83OTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.43" data-doi="10.1126/science.274.5290.1208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LeClerc</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cebula</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">High mutation frequencies among <em>Escherichia coli</em> and Salmonella pathogens.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">1208</span>–<span class="cit-lpage">1211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLeClerc%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D274%26rft.issue%253D5290%26rft.spage%253D1208%26rft.epage%253D1211%26rft.atitle%253DHigh%2BMutation%2BFrequencies%2BAmong%2BEscherichia%2Bcoli%2Band%2BSalmonella%2BPathogens%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5290.1208%26rft_id%253Dinfo%253Apmid%252F8895473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzQvNTI5MC8xMjA4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levinson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutman</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Slipped-strand mispairing: a major mechanism for DNA sequence evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DLevinson%26rft.auinit1%253DG%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D203%26rft.epage%253D221%26rft.atitle%253DSlipped-strand%2Bmispairing%253A%2Ba%2Bmajor%2Bmechanism%2Bfor%2BDNA%2Bsequence%2Bevolution%26rft_id%253Dinfo%253Apmid%252F3328815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjQvMy8yMDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.45" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.46" data-doi="10.1093/molbev/msh073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microsatellites within genes: structure, function, and evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">1007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh073%26rft_id%253Dinfo%253Apmid%252F14963101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.47" data-doi="10.1016/j.fgb.2004.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Notley-McRobb</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A comparison of the nature and abundance of microsatellites in 14 fungal genomes.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1025</span>–<span class="cit-lpage">1036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DLim%26rft.auinit1%253DS.%26rft.volume%253D41%26rft.issue%253D11%26rft.spage%253D1025%26rft.epage%253D1036%26rft.atitle%253DA%2Bcomparison%2Bof%2Bthe%2Bnature%2Band%2Babundance%2Bof%2Bmicrosatellites%2Bin%2B14%2Bfungal%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2004.08.004%26rft_id%253Dinfo%253Apmid%252F15465391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2004.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15465391&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226263600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.48" data-doi="10.1371/journal.pgen.1003016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lujan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pursell</span>  <span class="cit-name-given-names">Z. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdulovic-Cui</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mismatch repair balances leading and lagging strand DNA replication fidelity.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLujan%26rft.auinit1%253DS.%2BA.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1003016%26rft.epage%253De1003016%26rft.atitle%253DMismatch%2Brepair%2Bbalances%2Bleading%2Band%2Blagging%2Bstrand%2BDNA%2Breplication%2Bfidelity.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003016%26rft_id%253Dinfo%253Apmid%252F23071460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071460&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luria</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delbrück</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">Mutations of bacteria from virus sensitivity to virus resistance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLuria%26rft.auinit1%253DS.%2BE.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D491%26rft.epage%253D511%26rft.atitle%253DMUTATIONS%2BOF%2BBACTERIA%2BFROM%2BVIRUS%2BSENSITIVITY%2BTO%2BVIRUS%2BRESISTANCE%26rft_id%253Dinfo%253Apmid%252F17247100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyOC82LzQ5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanspa</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Review of the Lynch syndrome: history, molecular genetics, screening, differential diagnosis, and medicolegal ramifications.</span> <abbr class="cit-jnl-abbrev">Clin. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bgenetics%26rft.stitle%253DClin%2BGenet%26rft.aulast%253DLynch%26rft.auinit1%253DH.%2BT.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DReview%2Bof%2Bthe%2BLynch%2Bsyndrome%253A%2Bhistory%252C%2Bmolecular%2Bgenetics%252C%2Bscreening%252C%2Bdifferential%2Bdiagnosis%252C%2Band%2Bmedicolegal%2Bramifications.%26rft_id%253Dinfo%253Apmid%252F19659756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19659756&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268051600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.51" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.52" data-doi="10.1016/j.celrep.2011.10.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogacheva</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mutation hot spots in yeast caused by long-range clustering of homopolymeric sequences.</span> <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BRep%26rft.aulast%253DMa%26rft.auinit1%253DX.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D42%26rft.atitle%253DMutation%2Bhot%2Bspots%2Bin%2Byeast%2Bcaused%2Bby%2Blong-range%2Bclustering%2Bof%2Bhomopolymeric%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2011.10.003%26rft_id%253Dinfo%253Apmid%252F22832106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2011.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22832106&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309709500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.53" data-doi="10.1101/gad.10.4.407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsischky</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filosi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Redundancy of Saccharomyces cerevisiae MSH3 and MSH6 in MSH2-dependent mismatch repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMarsischky%26rft.auinit1%253DG%2BT%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D407%26rft.epage%253D420%26rft.atitle%253DRedundancy%2Bof%2BSaccharomyces%2Bcerevisiae%2BMSH3%2Band%2BMSH6%2Bin%2BMSH2-dependent%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.4.407%26rft_id%253Dinfo%253Apmid%252F8600025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTAvNC80MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.54" data-doi="10.1126/science.277.5333.1833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taddei</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doit</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Highly variable mutation rates in commensal and pathogenic <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">1833</span>–<span class="cit-lpage">1834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMatic%26rft.auinit1%253DI.%26rft.volume%253D277%26rft.issue%253D5333%26rft.spage%253D1833%26rft.epage%253D1834%26rft.atitle%253DHighly%2BVariable%2BMutation%2BRates%2Bin%2BCommensal%2Band%2BPathogenic%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.277.5333.1833%26rft_id%253Dinfo%253Apmid%252F9324769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzcvNTMzMy8xODMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leu</span>  <span class="cit-name-given-names">J. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Clusters of nucleotide substitutions and insertion/deletion mutations are associated with repeat sequences.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D9%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.56" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.57" data-doi="10.1038/nature11252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Cancer Genome Network</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comprehensive molecular characterization of human colon and rectal cancer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">487</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D487%26rft.issue%253D7407%26rft.spage%253D330%26rft.epage%253D337%26rft.atitle%253DComprehensive%2Bmolecular%2Bcharacterization%2Bof%2Bhuman%2Bcolon%2Band%2Brectal%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11252%26rft_id%253Dinfo%253Apmid%252F22810696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22810696&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306506500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.58" data-doi="10.1371/journal.pgen.1001109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001109%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001109%26rft_id%253Dinfo%253Apmid%252F20838597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838597&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.59" data-doi="10.1016/j.bbrc.2004.10.120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohnishi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endo</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daigaku</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Spontaneous mutagenesis in haploid and diploid <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">928</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DOhnishi%26rft.auinit1%253DG.%26rft.volume%253D325%26rft.issue%253D3%26rft.spage%253D928%26rft.epage%253D933%26rft.atitle%253DSpontaneous%2Bmutagenesis%2Bin%2Bhaploid%2Band%2Bdiploid%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2004.10.120%26rft_id%253Dinfo%253Apmid%252F15541379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2004.10.120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15541379&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225350300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.60" data-doi="10.1007/s00535-012-0565-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oki</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hisamatsu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ando</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saeki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakeji</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Clinical aspect and molecular mechanism of DNA aneuploidy in gastric cancers.</span> <abbr class="cit-jnl-abbrev">J. Gastroenterol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgastroenterology%26rft.stitle%253DJ%2BGastroenterol%26rft.aulast%253DOki%26rft.auinit1%253DE.%26rft.volume%253D47%26rft.issue%253D4%26rft.spage%253D351%26rft.epage%253D358%26rft.atitle%253DClinical%2Baspect%2Band%2Bmolecular%2Bmechanism%2Bof%2BDNA%2Baneuploidy%2Bin%2Bgastric%2Bcancers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00535-012-0565-4%26rft_id%253Dinfo%253Apmid%252F22402775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00535-012-0565-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22402775&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.61" data-doi="10.1126/science.288.5469.1251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canton</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baquero</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blazquez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">High frequency of hypermutable <em>Pseudomonas aeruginosa</em> in cystic fibrosis lung infection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOliver%26rft.auinit1%253DA.%26rft.volume%253D288%26rft.issue%253D5469%26rft.spage%253D1251%26rft.epage%253D1253%26rft.atitle%253DHigh%2BFrequency%2Bof%2BHypermutable%2BPseudomonas%2Baeruginosa%2Bin%2BCystic%2BFibrosis%2BLung%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5469.1251%26rft_id%253Dinfo%253Apmid%252F10818002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODgvNTQ2OS8xMjUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.62" data-doi="10.1016/S0960-9822(02)70685-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palombo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iaccarino</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakajima</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikejima</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">hMutSbeta, a heterodimer of hMSH2 and hMSH3, binds to insertion/deletion loops in DNA.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPalombo%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D1181%26rft.epage%253D1184%26rft.atitle%253DhMutSbeta%252C%2Ba%2Bheterodimer%2Bof%2BhMSH2%2Band%2BhMSH3%252C%2Bbinds%2Bto%2Binsertion%252Fdeletion%2Bloops%2Bin%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252802%252970685-4%26rft_id%253Dinfo%253Apmid%252F8805365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(02)70685-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8805365&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VH27200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.63" data-doi="10.1007/PL00006474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pupko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graur</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Evolution of microsatellites in the yeast Saccharomyces cerevisiae: role of length and number of repeated units.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DPupko%26rft.auinit1%253DT.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D313%26rft.epage%253D316%26rft.atitle%253DEvolution%2Bof%2Bmicrosatellites%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae%253A%2Brole%2Bof%2Blength%2Band%2Bnumber%2Bof%2Brepeated%2Bunits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006474%26rft_id%253Dinfo%253Apmid%252F10093220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10093220&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078808700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.64" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.65" data-doi="10.1126/science.294.5540.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wodicka</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Replication dynamics of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRaghuraman%26rft.auinit1%253DM.%2BK.%26rft.volume%253D294%26rft.issue%253D5540%26rft.spage%253D115%26rft.epage%253D121%26rft.atitle%253DReplication%2BDynamics%2Bof%2Bthe%2BYeast%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.294.5540.115%26rft_id%253Dinfo%253Apmid%252F11588253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTQvNTU0MC8xMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.66" data-doi="10.1128/MMBR.00011-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richard</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerrest</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative genomics and molecular dynamics of DNA repeats in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DRichard%26rft.auinit1%253DG.-F.%26rft.volume%253D72%26rft.issue%253D4%26rft.spage%253D686%26rft.epage%253D727%26rft.atitle%253DComparative%2BGenomics%2Band%2BMolecular%2BDynamics%2Bof%2BDNA%2BRepeats%2Bin%2BEukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00011-08%26rft_id%253Dinfo%253Apmid%252F19052325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Mi80LzY4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.67" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.68" data-doi="10.1093/oxfordjournals.molbev.a025964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A threshold size for microsatellite expansion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DRose%26rft.auinit1%253DO%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D613%26rft.epage%253D615%26rft.atitle%253DA%2Bthreshold%2Bsize%2Bfor%2Bmicrosatellite%2Bexpansion%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025964%26rft_id%253Dinfo%253Apmid%252F9580993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a025964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9580993&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073343800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.69" data-doi="10.1007/BF00120324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandri</span>  <span class="cit-name-given-names">G. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">On fluctuation analysis—a new, simple and efficient method for computing the expected number of mutants.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DSarkar%26rft.auinit1%253DS.%26rft.volume%253D85%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D179%26rft.atitle%253DOn%2Bfluctuation%2Banalysis%253A%2Ba%2Bnew%252C%2Bsimple%2Band%2Befficient%2Bmethod%2Bfor%2Bcomputing%2Bthe%2Bexpected%2Bnumber%2Bof%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00120324%26rft_id%253Dinfo%253Apmid%252F1624139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00120324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1624139&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JA10400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.70" data-doi="10.1158/0008-5472.CAN-09-3049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hile</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">Defective mismatch repair, microsatellite mutation bias, and variability in clinical cancer phenotypes.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-09-3049%26rft_id%253Dinfo%253Apmid%252F20068152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjg6IjcwLzIvNDMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.71" data-doi="10.1093/nar/gkp1131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opresko</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">DNA structure and the Werner protein modulate human DNA polymerase delta-dependent replication dynamics within the common fragile site FRA16D.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp1131%26rft_id%253Dinfo%253Apmid%252F19969545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM4LzQvMTE0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.72" data-doi="10.1590/S1516-31802009000100010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valentin</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira Fde</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carraro</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mismatch repair genes in Lynch syndrome: a review.</span> <abbr class="cit-jnl-abbrev">Sao Paulo Med. J.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DS%25C3%25A3o%2BPaulo%2Bmedical%2Bjournal%2B%253D%2BRevista%2Bpaulista%2Bde%2Bmedicina%26rft.stitle%253DSao%2BPaulo%2BMed%2BJ%26rft.aulast%253DSilva%26rft.auinit1%253DF.%2BC.%26rft.volume%253D127%26rft.issue%253D1%26rft.spage%253D46%26rft.epage%253D51%26rft.atitle%253DMismatch%2Brepair%2Bgenes%2Bin%2BLynch%2Bsyndrome%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS1516-31802009000100010%26rft_id%253Dinfo%253Apmid%252F19466295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S1516-31802009000100010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19466295&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266149800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.73" data-doi="10.1016/j.dnarep.2004.06.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stojic</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brun</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiricny</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mismatch repair and DNA damage signalling.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2Brepair%26rft.stitle%253DDNA%2BRepair%2B%2528Amst%2529%26rft.aulast%253DStojic%26rft.auinit1%253DL.%26rft.volume%253D3%26rft.issue%253D8-9%26rft.spage%253D1091%26rft.epage%253D1101%26rft.atitle%253DMismatch%2Brepair%2Band%2BDNA%2Bdamage%2Bsignalling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2004.06.006%26rft_id%253Dinfo%253Apmid%252F15279797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2004.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15279797&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.74" data-doi="10.1186/gb-2003-4-2-r13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide analysis of microsatellite repeats in humans: their abundance and density in specific genomic regions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">R13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSubramanian%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253DR13%26rft.epage%253DR13%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bmicrosatellite%2Brepeats%2Bin%2Bhumans%253A%2Btheir%2Babundance%2Band%2Bdensity%2Bin%2Bspecific%2Bgenomic%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2003-4-2-r13%26rft_id%253Dinfo%253Apmid%252F12620123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2003-4-2-r13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12620123&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.75" data-doi="10.1159/000080593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Surtees</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mismatch repair proteins: key regulators of genetic recombination.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DSurtees%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D107%26rft.issue%253D3-4%26rft.spage%253D146%26rft.epage%253D159%26rft.atitle%253DMismatch%2Brepair%2Bproteins%253A%2Bkey%2Bregulators%2Bof%2Bgenetic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000080593%26rft_id%253Dinfo%253Apmid%252F15467360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000080593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15467360&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224988400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft.volume%253D14%26rft.spage%253D178%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-web" id="cit-3.9.1453.77"><div class="cit-metadata unstructured">Thurston, M. I., and D. Field, 2005 Msatfinder: detection and characterisation of microsatellites. Available at <a href="http://www.genomics.ceh.ac.uk/msatfinder/">http://www.genomics.ceh.ac.uk/msatfinder/</a>. Accessed July 9, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.78" data-doi="10.1128/MCB.17.5.2859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kricker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordenin</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2859</span>–<span class="cit-lpage">2865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTran%26rft.auinit1%253DH.%2BT.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D2859%26rft.epage%253D2865%26rft.atitle%253DHypermutability%2Bof%2Bhomonucleotide%2Bruns%2Bin%2Bmismatch%2Brepair%2Band%2BDNA%2Bpolymerase%2Bproofreading%2Byeast%2Bmutants%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.5.2859%26rft_id%253Dinfo%253Apmid%252F9111358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzUvMjg1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNDUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Umar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risinger</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaab</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tindall</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Functional overlap in mismatch repair by human MSH3 and MSH6.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1637</span>–<span class="cit-lpage">1646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DUmar%26rft.auinit1%253DA.%26rft.volume%253D148%26rft.issue%253D4%26rft.spage%253D1637%26rft.epage%253D1646%26rft.atitle%253DFunctional%2BOverlap%2Bin%2BMismatch%2BRepair%2Bby%2BHuman%2BMSH3%2Band%2BMSH6%26rft_id%253Dinfo%253Apmid%252F9560383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OC80LzE2MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8zLzkvMTQ1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wierdl</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Microsatellite instability in yeast: dependence on the length of the microsatellite.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWierdl%26rft.auinit1%253DM.%26rft.volume%253D146%26rft.issue%253D3%26rft.spage%253D769%26rft.epage%253D779%26rft.atitle%253DMicrosatellite%2BInstability%2Bin%2BYeast%253A%2BDependence%2Bon%2Bthe%2BLength%2Bof%2Bthe%2BMicrosatellite%26rft_id%253Dinfo%253Apmid%252F9215886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzMvNzY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.81" data-doi="10.1534/genetics.110.120105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">RoyChoudhury</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demogines</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection of heterozygous mutations in the genome of mismatch repair defective diploid yeast using a Bayesian approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120105%26rft_id%253Dinfo%253Apmid%252F20660644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNDkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE0NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1453.82" data-doi="10.1016/j.molcel.2007.06.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freudenreich</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An AT-rich sequence in human common fragile site FRA16D causes fork stalling and chromosome breakage in S. cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DZhang%26rft.auinit1%253DH.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D379%26rft.atitle%253DAn%2BAT-rich%2Bsequence%2Bin%2Bhuman%2Bcommon%2Bfragile%2Bsite%2BFRA16D%2Bcauses%2Bfork%2Bstalling%2Band%2Bchromosome%2Bbreakage%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.06.012%26rft_id%253Dinfo%253Apmid%252F17679088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.06.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17679088&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1453.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248823400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/9/1453.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/9/1451" title="Functional Genetics in the Post-genomics Era: Building a Better Roadmap in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/9/1467" title="Sequence Diversity in Coding Regions of Candidate Genes in the Glycoalkaloid Biosynthetic Pathway of Wild Potato Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/9"><span>Volume 3 Issue 9, September 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/9.cover-source.jpg?itok=2NEcI52z" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1453.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BtEZ2p-RQlAZpFg9mpDZDIIJbefBE5BfITU4tOEyyPk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/9/1453" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/9/1453" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/9/1453" class="active" data-icon-position="" data-hide-link-title="0">Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459529" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-D9vKfXEzVrVyr7rscZ-8Qo5PT4JVhzfVyqLqsy_NKIE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/9/1453.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/9/1453" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-HyLfvWZBZFCYZoyRzTr-pEBmkPvA7ZJkd1C0vhkcKTU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/9/1453" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/9/1453.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459529" id="node459529--2" data-pisa="ggg;3/9/1453" data-pisa-master="ggg;g3.113.006429" data-apath="/ggg/3/9/1453.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory I. Lang</span>, <span class="highwire-citation-author" data-delta="1">Lance Parsons</span> and <span class="highwire-citation-author" data-delta="2">Alison E. Gammie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1453-1465;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006429 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459529--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory I. Lang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BI.%2BLang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lang%20GI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BI.%2BLang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lance Parsons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lance%2BParsons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parsons%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALance%2BParsons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alison E. Gammie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BE.%2BGammie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gammie%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BE.%2BGammie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agammie@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">agammie@princeton.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/9/1453" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/9/1453&amp;title=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/9/1453&amp;title=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/9/1453&amp;title=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/9/1453&amp;text=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/9/1453&amp;title=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/9/1453&amp;t=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/9/1453&amp;title=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/9/1453&amp;title=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459529/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459529" id="node459529--3" data-pisa="ggg;3/9/1453" data-pisa-master="ggg;g3.113.006429" data-apath="/ggg/3/9/1453.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mutation Rates, Spectra, and Genome-Wide Distribution of Spontaneous Mutations in Mismatch Repair Deficient Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gregory I. Lang</span>, <span class="highwire-citation-author" data-delta="1">Lance Parsons</span> and <span class="highwire-citation-author" data-delta="2">Alison E. Gammie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1453-1465;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006429 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459529--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory I. Lang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BI.%2BLang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lang%20GI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BI.%2BLang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lance Parsons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lance%2BParsons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parsons%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALance%2BParsons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alison E. Gammie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Biology, Princeton University, Princeton, New Jersey 08544-1014</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BE.%2BGammie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gammie%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BE.%2BGammie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agammie@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">agammie@princeton.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1453&amp;rft.epage=1465&amp;rft.atitle=Mutation%20Rates%2C%20Spectra%2C%20and%20Genome-Wide%20Distribution%20of%20Spontaneous%20Mutations%20in%20Mismatch%20Repair%20Deficient%20Yeast&amp;rft.volume=3&amp;rft.issue=9&amp;rft.date=2013-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lang%2C+Gregory+I.&amp;rft.au=Parsons%2C+Lance&amp;rft.au=Gammie%2C+Alison+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459529/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459529/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459529/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459529/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459529/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459529/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459529/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459529/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459529/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459529/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459529/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459529/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/9/1453" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/9/1453.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/9/1453.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/9/1453.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/9/1453.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
