<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/9/1553" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/9/1553.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/9/1553.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/9/1553.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/9/1553/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/9/1553.atom" />
<meta name="HW.pisa" content="ggg;3/9/1553" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of Pedigree Relationship from Genome Sharing" />
<meta name="DC.Identifier" content="10.1534/g3.113.007500" />
<meta name="DC.Date" content="2013-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Hill and White. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Determination of degree of relationship traditionally has been undertaken using genotypic data on individual loci, typically assumed to be independent. With dense marker data as now available, it is possible to identify the regions of the genome shared identical by descent (ibd). This information can be used to determine pedigree relationship ( R ), e.g. , cousins vs. second cousins, and also to distinguish pedigrees that have the same Wright’s relationship ( R ) such as half-sibs and uncle–nephew. We use simulation to investigate the accuracy with which pedigree relationship can be inferred from genome sharing for uniparental relatives (a common ancestor on only one side of their pedigree), specifically the number, position (whether at chromosome ends), and length of shared regions ibd on each chromosome. Moments of the distribution of the likelihood ratio (including its expectation, the Kullback-Leibler distance) for alternative relationships are estimated for model human genomes, with the ratio of the mean to the SD of the likelihood ratio providing a useful reference point. Two relationships differing in R can be readily distinguished provided at least one has high R , e.g. , approximately 98.5% correct assignment of cousins and half-cousins, but only approximately 75% for second cousins once removed and third cousins. Two relationships with the same R can be distinguished only if R is high, e.g. , half-sibs and uncle–nephew, with probability of correct assignment being approximately 5/6." />
<meta name="DC.Contributor" content="William G. Hill" />
<meta name="DC.Contributor" content="Ian M. S. White" />
<meta name="article:published_time" content="2013-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Identification of Pedigree Relationship from Genome Sharing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Determination of degree of relationship traditionally has been undertaken using genotypic data on individual loci, typically assumed to be independent. With dense marker data as now available, it is possible to identify the regions of the genome shared identical by descent (ibd). This information can be used to determine pedigree relationship (&lt;b&gt;R&lt;/b&gt;), &lt;i&gt;e.g.&lt;/i&gt;, cousins &lt;i&gt;vs.&lt;/i&gt; second cousins, and also to distinguish pedigrees that have the same Wright’s relationship (&lt;i&gt;R&lt;/i&gt;) such as half-sibs and uncle–nephew. We use simulation to investigate the accuracy with which pedigree relationship can be inferred from genome sharing for uniparental relatives (a common ancestor on only one side of their pedigree), specifically the number, position (whether at chromosome ends), and length of shared regions ibd on each chromosome. Moments of the distribution of the likelihood ratio (including its expectation, the Kullback-Leibler distance) for alternative relationships are estimated for model human genomes, with the ratio of the mean to the SD of the likelihood ratio providing a useful reference point. Two relationships differing in &lt;i&gt;R&lt;/i&gt; can be readily distinguished provided at least one has high &lt;i&gt;R&lt;/i&gt;, &lt;i&gt;e.g.&lt;/i&gt;, approximately 98.5% correct assignment of cousins and half-cousins, but only approximately 75% for second cousins once removed and third cousins. Two relationships with the same &lt;i&gt;R&lt;/i&gt; can be distinguished only if &lt;i&gt;R&lt;/i&gt; is high, &lt;i&gt;e.g.&lt;/i&gt;, half-sibs and uncle–nephew, with probability of correct assignment being approximately 5/6.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/09/01" />
<meta name="citation_mjid" content="ggg;3/9/1553" />
<meta name="citation_id" content="3/9/1553" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/9/1553" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/9/1553.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/9/1553.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/9/1553.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007500" />
<meta name="citation_pmid" content="23893739" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1553" />
<meta name="citation_lastpage" content="1571" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="William G. Hill" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom" />
<meta name="citation_author_email" content="w.g.hill@ed.ac.uk" />
<meta name="citation_author" content="Ian M. S. White" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. R. Abecasis;citation_author=S. S. Cherny;citation_author=W. O. Cookson;citation_author=L. R. Cardon;citation_title=Merlin--rapid analysis of dense genetic maps using sparse gene flow trees.;citation_pages=97-101;citation_volume=30;citation_year=2002;citation_issue=1;citation_pmid=11731797;citation_doi=10.1038/ng786" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=M. S. Blouin;citation_title=DNA-based methods for pedigree reconstruction and kinship analysis in natural populations;citation_pages=503-511;citation_volume=18;citation_year=2003;citation_doi=10.1016/S0169-5347(03)00225-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Improving the accuracy and efficiency of identity-by-descent detection in population data;citation_volume=194;citation_year=2013;citation_pmid=23535385;citation_doi=10.1534/genetics.113.150029" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=A fast, powerful method for detecting identity by descent.;citation_pages=173-182;citation_volume=88;citation_year=2011;citation_issue=2;citation_pmid=21310274;citation_doi=10.1016/j.ajhg.2011.01.010" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Identity by descent between distant relatives: detection and applications.;citation_pages=617-633;citation_volume=46;citation_year=2012;citation_pmid=22994355;citation_doi=10.1146/annurev-genet-110711-155534" />
<meta name="citation_reference" content="citation_journal_title=Wildl. Res.;citation_author=K. P. Burnham;citation_author=D. R. Anderson;citation_title=Kullback–Leibler information as a basis for strong inference in ecological studies;citation_pages=111-119;citation_volume=28;citation_year=2001;citation_doi=10.1071/WR99107" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=K. P. Donnelly;citation_title=The probability that related individuals share some section of genome identical by descent.;citation_pages=34-63;citation_volume=23;citation_year=1983;citation_issue=1;citation_pmid=6857549;citation_doi=10.1016/0040-5809(83)90004-7" />
<meta name="citation_reference" content="citation_title=The theory of inbreeding;citation_author=R. A. Fisher;citation_year=1949" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. A. Fisher;citation_title=A fuller theory of ‘Junctions’ in inbreeding;citation_pages=187-197;citation_volume=8;citation_year=1954;citation_doi=10.1038/hdy.1954.17" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. Epstein;citation_author=W. L. Duren;citation_author=M. Boenkhe;citation_title=Improved inference of relationship for pairs of individuals.;citation_pages=1219-1231;citation_volume=67;citation_year=2000;citation_issue=5;citation_pmid=11032786;citation_doi=10.1086/321195" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variation in actual relationship as a consequence of Mendelian sampling and linkage.;citation_pages=47-64;citation_volume=93;citation_year=2011;citation_issue=1;citation_pmid=21226974;citation_doi=10.1017/S0016672310000480" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. D. Huff;citation_author=D. J. Witherspoon;citation_author=T. S. Simonson;citation_author=J. C. Xing;citation_author=W. S. Wa;citation_title=Maximum-likelihood estimation of recent shared ancestry (ERSA);citation_volume=21;citation_year=2011;citation_pmid=21324875;citation_doi=10.1101/gr.115972.110" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=X. Kong;citation_author=K. Murphy;citation_author=T. Raj;citation_author=C. He;citation_author=P. S. White;citation_title=A combined linkage-physical map of the human genome.;citation_pages=1143-1148;citation_volume=75;citation_year=2004;citation_issue=6;citation_pmid=15486828;citation_doi=10.1086/426405" />
<meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.;citation_author=S. Kullback;citation_author=R. A. Leibler;citation_title=On information and sufficiency;citation_pages=79-86;citation_volume=22;citation_year=1951;citation_doi=10.1214/aoms/1177729694" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Kyriazopoulou-Panagiotopoulou;citation_author=D. K. Haghighi;citation_author=S. J. Aerni;citation_author=A. Sundquist;citation_author=S. Bercovici;citation_title=Reconstruction of genealogical relationships with applications to Phase III of HapMap;citation_pages=I333-I341;citation_volume=27;citation_year=2011;citation_pmid=21685089;citation_doi=10.1093/bioinformatics/btr243" />
<meta name="citation_reference" content="citation_title=Multivariate Analysis;citation_author=K. V. Mardia;citation_author=J. T. Kent;citation_author=J. M. Bibby;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=T. C. Marshall;citation_author=J. Slate;citation_author=L. E. B. Kruuk;citation_author=J. M. Pemberton;citation_title=Statistical confidence for likelihood-based paternity inference in natural populations.;citation_pages=639-655;citation_volume=7;citation_year=1998;citation_issue=5;citation_pmid=9633105;citation_doi=10.1046/j.1365-294x.1998.00374.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. S. McPeek;citation_author=L. Sun;citation_title=Statistical tests for detection of misspecified relationships by use of genome-screen data.;citation_pages=1076-1094;citation_volume=66;citation_year=2000;citation_issue=3;citation_pmid=10712219;citation_doi=10.1086/302800" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=J. M. Pemberton;citation_title=Wild pedigrees: the way forward;citation_pages=613-621;citation_volume=275;citation_year=2008;citation_issue=1635;citation_pmid=18211868;citation_doi=10.1098/rspb.2007.1531" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. D. O. Roberson;citation_author=J. Pevsner;citation_title=Visualization of shared genomic regions and meiotic recombination in high-density SNP data.;citation_pages=e6711-e6711;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19696932;citation_doi=10.1371/journal.pone.0006711" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. V. Rohlfs;citation_author=S. M. Fullerton;citation_author=B. S. Weir;citation_title=Familial identification: population structure and relationship distinguishability.;citation_pages=e1002469-e1002469;citation_volume=8;citation_year=2012;citation_issue=2;citation_pmid=22346758;citation_doi=10.1371/journal.pgen.1002469" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=P. Stam;citation_title=The distribution of the fraction of the genome identical by descent in finite populations;citation_pages=131-155;citation_volume=35;citation_year=1980;citation_doi=10.1017/S0016672300014002" />
<meta name="citation_reference" content="citation_journal_title=IMA J. Math. Appl. Med. Biol.;citation_author=A. Thomas;citation_author=M. H. Skolnick;citation_author=C. M. Lewis;citation_title=Genomic mismatch scanning in pedigrees;citation_volume=11;citation_year=1994;citation_pmid=8057038;citation_doi=10.1093/imammb/11.1.1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resources;citation_author=J. Wang;citation_title=COANCESTRY: A program for simulating, estimating and analysing relatedness and inbreeding coefficients;citation_pages=141-145;citation_volume=11;citation_year=2011;citation_doi=10.1111/j.1755-0998.2010.02885.x" />
<meta name="citation_reference" content="citation_title=Genetic data analysis. II;citation_author=B. S. Weir;citation_year=1996" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification of Pedigree Relationship from Genome Sharing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
<meta name="twitter:description" content="Determination of degree of relationship traditionally has been undertaken using genotypic data on individual loci, typically assumed to be independent. With dense marker data as now available, it is possible to identify the regions of the genome shared identical by descent (ibd). This information can be used to determine pedigree relationship ( R ), e.g. , cousins vs. second cousins, and also to distinguish pedigrees that have the same Wright’s relationship ( R ) such as half-sibs and uncle–nephew. We use simulation to investigate the accuracy with which pedigree relationship can be inferred from genome sharing for uniparental relatives (a common ancestor on only one side of their pedigree), specifically the number, position (whether at chromosome ends), and length of shared regions ibd on each chromosome. Moments of the distribution of the likelihood ratio (including its expectation, the Kullback-Leibler distance) for alternative relationships are estimated for model human genomes, with the ratio of the mean to the SD of the likelihood ratio providing a useful reference point. Two relationships differing in R can be readily distinguished provided at least one has high R , e.g. , approximately 98.5% correct assignment of cousins and half-cousins, but only approximately 75% for second cousins once removed and third cousins. Two relationships with the same R can be distinguished only if R is high, e.g. , half-sibs and uncle–nephew, with probability of correct assignment being approximately 5/6." />
<meta name="og-title" property="og:title" content="Identification of Pedigree Relationship from Genome Sharing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/9/1553" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Determination of degree of relationship traditionally has been undertaken using genotypic data on individual loci, typically assumed to be independent. With dense marker data as now available, it is possible to identify the regions of the genome shared identical by descent (ibd). This information can be used to determine pedigree relationship ( R ), e.g. , cousins vs. second cousins, and also to distinguish pedigrees that have the same Wright’s relationship ( R ) such as half-sibs and uncle–nephew. We use simulation to investigate the accuracy with which pedigree relationship can be inferred from genome sharing for uniparental relatives (a common ancestor on only one side of their pedigree), specifically the number, position (whether at chromosome ends), and length of shared regions ibd on each chromosome. Moments of the distribution of the likelihood ratio (including its expectation, the Kullback-Leibler distance) for alternative relationships are estimated for model human genomes, with the ratio of the mean to the SD of the likelihood ratio providing a useful reference point. Two relationships differing in R can be readily distinguished provided at least one has high R , e.g. , approximately 98.5% correct assignment of cousins and half-cousins, but only approximately 75% for second cousins once removed and third cousins. Two relationships with the same R can be distinguished only if R is high, e.g. , half-sibs and uncle–nephew, with probability of correct assignment being approximately 5/6." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9.cover-source.jpg" />
    <title>Identification of Pedigree Relationship from Genome Sharing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jk7_D-X-wtd1rrVJMU7Vba8D6AOghAjWDmLh1PA_DEw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459534","apath":"\/ggg\/3\/9\/1553.atom","pisa":"ggg;3\/9\/1553","ac":{"\/ggg\/3\/9\/1553.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/9\/1553.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/9\/1553"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/9\/1553"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/9\/1553":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qDnqvCmFRNxUr9xfNOxcBn8xQBZii\\\\\\\/wOZX0Nml8iCZ4XT4NXtvlEaASyGyxplaPNKt2wJcHpme7KWBXjioE89GPUofsPEwr2BsxxyOuw3nHg\\\\\\\/bJ22flOuKgsShNlc4HonIiQwHgYCdGdXlPvS2uh9fb2vcGPQvin5\\\\\\\/I5Zm9Colb7j6K4bfKlQCU\\\\\\\/zeaXbd2yDOQGDa6UGGqb258wBAHlf+EDB34LTY4J257a0o7VEzkgySBMQR7gIs6tXQh35FeXNM9lQDBgRhxHGz6yPWgWLPdRuiDjRm0XmJq+i+nQMzT4k9iIrhX2OTYwZsSk\\\\\\\/TdA9iAtRn+SSwH+teZp9CNhidbYVAhc9Bs+4h4Y\\\\\\\/QL1n6Wto2+FvIFN+5lxzJmJbt9xBhsvbjsLhHYlt4wSTODC8X95HmvXqLztw5ZuqWedWHy3rV1+l44xg3amMYHlraM5ZYbWIbfedQY0vR3VJ2fVU7UodWooqpjQucZ+npTJREX4zL29ircmZHVdLVaXAf0o3XKG16KLW6zMaAEswX\\\\\\\/Uisu2n1tM9o65VpguT1uef1UKRV0Ldl5XJb2JnGgUOK\\\\\\\/hVFcbE0gQFRJHPVjTS1ZPrMflC+cUUjG9AOp\\\\\\\/qKh8104SyNzTyQ\\\\\\\/bBNa7+FJ6ji7mXh8G2s3f9Gz2lJBBdGwrKn6M4SF1+HAdpxPm+298tC0sYSnY8l6YZs40lbkQ5mQAvhVva5Z0bRfZdZCMlQTqFkeDyy4tf+dmOeEREpi5UNyHXfhe3NKmwpbn7jUEWDauQhabQErXu01jemb0dLs7DbbM05bFzp95+ReZ4vTwoDYEFnVJhUajkJd1+K20\\\\\\\/WTNG9P05BHrlBA57hmrxP2PibIALSMriAxvFFcgi7SLhKJ7RKOyEhoJjcz3aBGRVT6ljwHUV0xR0HwXBUeW73bXio0+AI8\\\\\\\/GSbSIIiTcmdKPdUA\\\\\\\/vlX1yZgPXybo\\\\\\\/z6lcfRUfWP30qKMoz7SZPKkEA374ZMTx+Y8ERlenAv3gc72b3djbNyvp4RRcVfdNoQpeQ5m3cnE\\\\\\\/NOUuqujTfkKoelxmXZs\\\\\\\/QwPsoF430SViaR\\\\\\\/+1juaVOsWXJH6f7zBIIfnrspOkQEPnssERpta4becYi8oA\\\\\\\/HpMpGorD6kewJLQ3qEi79Q\\\\\\\/uDalrVc56SrTtfJqwlyXmLTweGQTXWBZ+3\\\\\\\/oHb\\\\\\\/Bca8F+2TELnHoUptWVlIkNJTUMNglOF\\\\\\\/Nz2HmeSi0\\\\\\\/knsol57GNgpm5XnVpASc2pM216YYOIiEPMb4tjVtYIO5rq9qdMS\\\\\\\/\\\\\\\/oV81Nnq9i9Z\\\u0022,\\\u0022iv\\\u0022:\\\u0022jrW6TQon992PKK4ol6KaOQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c2afd153f4390d56284140d3fa3f6c7cdd4a9677ac879996f7b2d8b023fe37ef\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T6DQ4iwh+Od3ugxuuZbpluiTa65gzyn3aZms9mcgazPfVnc4tpaGZ46Q2PxeEvt\\\\\\\/9GMHDYG0q8uvAuC31\\\\\\\/4ssSSzDUUBuRTdPT59Qp4J7seV2qKcXJgnHHQ5MbNJHdQMI7w43W9lAaYGLii6S9cJHjyPQIs8\\\\\\\/5wqmBPS6FxArAEr9V4OS8UEu9qI\\\\\\\/37+itmVtmB4eV717XkYBOMTFTpKhwCxMl0eFQXLX5idmLyAWMiPQkSYLK6JVW83\\\\\\\/9B3gTS6O2e1tHPzElXLjScdn18ef6Hped0sJbDzdOWlOWNei6t+ehrI+FMUFNu1QBRxFSnNUFrvPYZFKLPO1eU+g2z55kT9CO3KArWBYTofQ\\\\\\\/2nz9Bjwsx8QoZJnjfwMhb\\\\\\\/SSK1Guo6bjRDXf0CMdx48LmcM5ShEQm1+cPQ4zIkPYD2JRxih40U5MUjCI1VOsxLWM4DqgVzA\\\\\\\/QwWxmT9yXRE\\\\\\\/fSFu+IHkenu6ZCXwuZZSlnPc8F5DO6IN9izGU0TP\\\\\\\/T99yfOc6HUJq1Kfjd\\\\\\\/Og\\\\\\\/i1+hg3YerdtMkVzRAfAss4aNG5jlV5c3jbk4ghmy5sztfJeB3VRWdWjHkN2jesCaE6MjT0Tz5wu6fQ3nngHgJ1A4J9PAGAtLnUee34oELpuyIWXv12Ye5f73tfGIBHKpkr3Fk8bGxbh57iPEYvs755Mp\\\\\\\/GJLwrecMS7nTeS4HmjaejoiWH+Ni5zjXDLv6NhqnhEYwBMGquJ8MZdGqpQwAA+0HVMlD5c1DcDNJ+cRMdC4HC7qOSx5eZ2wdxyX1Eq25+6SW0N+uIlguSvXiRuHGdTxYWVidTC22YrEoaIYmgtXWktV8kMyyILqNny9YowvjCopOLhYkIzDOH3Qm\\\\\\\/j4WH4jfYFnA\\\\\\\/ZPdOBfB8Sea6L7y4SBMCvXIfhx7c13AQ3dKzpSaRyNCRQBsltY0Vjh2rO4hzjPeJUd\\\\\\\/wAW9DcVR91vFUytw\\\\\\\/dJ6YlMMM8DQajiU91gC4UpEX2zDoazNtTqrrdXwtc5SOTF54xc6jZ2GQh+2cSgND1wQVGKvX5s4DgU0wrNT+l54ZRh\\\\\\\/6Rb1BGy\\\\\\\/De4moUXGa5D52ICS3Nt36D3PA2LLbR2jxwQV9h41XR13Gh15lIUWzYOraDuCKCXZ\\\\\\\/9Bdsgvc8pvsWAcOqOXd03KGEUCugkFlbW66x7eSN3SonLgLpw0zUjtTPeBznLIni90ANqZFhyW\\\\\\\/1BreqtR+6kiutEVhtqNgo7PlodDg6vHzSrmttOMfC9Y9jpBG8BdHzeL1bb+6166AqlqLai9M9czcJ30cCHoUIMc+RSNpoL1KLN9C6niDETZUAEbTYOg9C5+4Kmmc9\\\\\\\/DB5hzcKirj\\\\\\\/9lHszrNPnxXTi2+AJbFCAVT6bdPWbTwWjxCsPY++3TU3seYqS8\\\\\\\/6+MnKv4061IGb30bJGCdRwqNttNTwLVmuGlaza9ssyVmibiYA+TrPoYU8AozmyEHgK1\\\\\\\/9AnlFWkhUqQnhnFYerLNupirH1GgMo+J\\\\\\\/2h8ND7jY59v1eQwKZG4hiA+7KB2jvY8tTh2QcmJ\\\\\\\/MnQ30oeANfVwczc\\\\\\\/POm9O9zqeXPx33IlHau29i2L0sld0ydpb6mDwipFBr35NY\\\\\\\/uoR0HeV2WSzqQHL9mqh1Jx8V6Q0qyWBtBpVVFnmGZ+v3TnPam+3vF9mTAXfEzsLr0KRtxFNVoAn3oD8631wkizqvXSdktu05Ieg8XQak2ieQlyNRr52Hkp6up2VF8hNPiUaa38CdIsv2nUOmHdHECXIOs8oTmL+Ve\\\\\\\/GEv+VbtNYEBOTj5u+o4K6BC5\\\\\\\/DUO1u03u6KxtieOB3q9T3kmg\\\\\\\/a1kcJp819KrdIAUXpMiPeqnXqR4+WiRxhXwti55oWnh5vskwqHey7NLPwAtmKDtVrYBiT+BjwdpB6x8xlmxnusyFAOyJx\\\\\\\/270MWhabThcDNh5Trqmy45qEf3FBTfNwr1esyPb51SdZlxMpA7RQK\\\\\\\/qy0DeNeAHyp3rj0qduN45kiLFUGS122shEYf62h1VMU2CUBJtiZbi1WhyRUuL8XvgehWTZzd3SJrYJUmH3gCoXZ3i2nroDMKDW\\\\\\\/\\\\\\\/LqK6j9zl+zmT9xhhiswqoGQDnohMlYGNxxMkZ3XIfVEpHPqTrj\\\\\\\/qZpdlAvKM0cJp4f4DPheYINQsc6myDHs2lR6wqsSMS1RFbm57vUS9\\\\\\\/CaS0VUwgN38xZqV990y9s4nCxHYoaYrUS0rf38wYG+CLTk46AVqWS8LsaqdSZC6UfQyz2nrGxGRTK+Gpn\\\\\\\/5P\\\\\\\/NQxE1iqnpCFTBHdCst+B5QlEY+0g+iNwLRdRfLaHOnUH\\\\\\\/fyjSNzet+4CHFWJOva1HMZGwH0sqjGqEs4XuhltU3eUUl3\\\\\\\/4fKMftlrDVy7SCizwSzVKIJ+g6ZnMVvLrMIBKjfamfhfMJ019IQNQ1wkezgSIs3ejXckoDRFZ02Qgkas+k2rsQZZTqO7Iqa1Ll3u96c7L+W0SgVSN\\\\\\\/ufgzCdU85Wz1GXA4a\\\\\\\/5EJG0qLd5Zye2fLEoTqcPYEnpLjLXpawlF0e9updOabgC9YSGi5C71rs9NaXmRKHFzCJIbFBobKcwd1DkPIRKyKbo6ZfP2Ep5GIyUOA7I6zdpshD4Aexytj63MHLdMYLJAUQkm5wNi7fNiI65Wwk+HQQHX+fNaiswzEr7yxFLhiH+L7U51MfKYS6eSs40sMBxRVFEZy+\\\\\\\/cuKMimtjTJFixafdTh2pTuGeIEnlFjG2yQP\\\\\\\/uCy12cO3ltMhS+gOLfmctETmAQmxhxFd8tEV9dkgch3IYgCMPqSzvNA+0zwSvx3dyYjGK5QTtcF3d7HJlmUxB2zcWMMvYbf4p9ZspFozxzofeFqMvKWPEngDHoMNmqtG8qID0M4LDWOpT8gp5Lr12Tbp\\\\\\\/sKiudrklWOqogUn8F0CtDEaX2yjj1G0rfjCk6Uhq3woHOl8\\\\\\\/x9MK86qq+Ns24SFOjw5kHtQ6DejP9mvGWWgfpnqwbSfRMfZ+vwtSi7GrVX0diw4eAIgl\\\\\\\/AdugB4HzUC\\\\\\\/9KyomGm8tHSbJ9aFAUoHmkNohyxx\\\\\\\/TC1YukPM5Aw1c0kEGNDaUqcwBmgiu2Vh49lqZmWNV7QVzjIughJxukZ+pjGsVKr0bXnG+1sidq0iBi5N19sRLwSOi8JBO4xK2IfjF89SgnvYyjti6tNhOmhdYtvaUtqX2qysi8dR+cuihF7n7A\\\\\\\/WahkQ8BxilW31+0jF4H7uoJiE4Tm4UoX5yR0cCOXYQbz2ZvuFpnsTWHtRm1KSWfazhlUIDR4S17R9X7Qf8A+o6Dx8BIQL0G3FactWAvxs4HqI93Uj9invg3X6leux4eU8iz6yhzCVNdBYHd5cG6VrM8BlmvJCTfyLW47MPJ7VRqa7n0TceHG1CALLkD39YWm7Q1FTXww3JiZBOZWt0uQfqLqhs9LLG\\\\\\\/pPA39e7LscLz3Vx\\\\\\\/Cwll4KUhukqv6gLuHMA4DAzQDDAlrPx4g+o5bGmtjmTAO24se\\\\\\\/a8qPjMJg9nRHDQJuE+iFjUmERqu+23Ta9A\\\\\\\/4nbPTVBOt\\\\\\\/HsAabGHXJAcTqN7SDWlTfuzCbOdDO2cmFaF5NMz\\\\\\\/sDStozUtD8k661hc8c6UDm6bT+97oj2AuKiwF2ZABjxfMkXARAMHM21d3rRwq21THNJ5DXC8Td13OSNCEJ+9sQ6nkclhn4dHaf0r8xiFgNgHrpMsqHvvE+M1pDuAbpG0oZjhNYOUyCGdmLbL+TqOCHwqDlurFYve95AqAZiUU3voPyJQeCGxg7ciwWnLrxEk0gQgI9vCYVOw2IjpoRZVwQEnKK7Ag45rk5WVa4Lkr7PdB8Pu29prT70SBmrwInxF8JK9CsAhTUliz5FYRi5pGzM8IUd2VKQIbaFF1Lq3Ca3djqAssAsFaC7qZl31bErz9ksSN0LgncXMxGrV18f0\\\\\\\/J4uxT7h3fhVw+RsCsH69jZYusOyWwKQkjLhEVeXUiN1fMQypiELaeHeshqLu8q\\\\\\\/PrhWzHyEcD4ax1AXZfcmoeUCJUQfqyfYKaNOxqOugukHRlOJZm+KlXO9pmpC7ogyM7e2vwUJVZR07eduB+6c6JclwScxq+I6v1AtKaClYLDtSTySWFjPcnIo07bxBHQfmHAxjh\\\\\\\/NEnMNYavQqx8820+k3tzN\\\\\\\/cTQJCDSkxhcssoyZJrdAbB5AMChL6oP163NPKO4uxkNanugnljAslr8Bjm6SP3X3h+i7956\\\\\\\/2ppmNrEExbisD\\\\\\\/TlWZ9RzdSucz4mOvwOvh8m9IDnkzgrVWrujXjPmVDoaMMBc9II3lPwfo928VP3ZbRUBo9O+\\\\\\\/9VHK03DV85gwd93LNFbPHlCIz18mWwtzlGihJfQnjMNPJEyDqTRKIoDR4G1opFPlM+rNg8kFqOCqB9gaXIeAeG+za0tyXDKjemrDAwmTjUKPwG66IbbdoLKGyDonvzbG2et0F1QzoeTEnxlU1cqW7JpFfH0RUEa4IxEyyY9XCZDy2YQJgdBH1G9Ww\\\\\\\/VPce39iC1YDmyG5ubNjLeD4Kxv5HSMLX5L+KUhOE+Un\\\\\\\/lxHhFZ+txRumMD8sQCkvlNx0NRMnHV+dEs+i2jbRtTkNTzt6zdOzC9t\\\\\\\/eWIp8PgRScnYKgI03FEcBmzJLniVuW5igZOWkIWXkuN\\\\\\\/G1pUZo5rF3ntlBu\\\\\\\/XSTz9HaDPDD3EU4e6hza7TRXL\\\\\\\/uD53w0EeDbPJjDnejb0v5rviE73aBoC6cHZwPfxyyqv93FrEdXqdIZut4C4VW7tSckC0+Lwz6gcY8hFRvkhiQt\\\\\\\/RxA0qEkEbg7u4MD2ueIg9IbhHgOJy6yY+emmUqpujIPYP892+YwjcXLCVqrN0MeZ2vV4dfIUbFpgLipY0WMxABpQiKpTQz4uOlnOcHaZMgMRD41ePLBNLdyp9h65GQy2rKF7fY0Znex5O4Ktf8zY58VebrsnSM49c\\\\\\\/Pojvixu9nZk8NRAq7FLdun4rFJZHVj8C6wJe7qsn8j4gV2zcqGZRpbRAZRWhjeQEns5Ucsu\\\\\\\/PJwGNJ5EVLguYJNgnSjctMxAt2PKHXFPXkQBMlDjWjj0hChjuKQUqOwHCePJG05x+7chUyGSUvWBNhZo1yJLU3f7oko6t7p9xrdf\\\\\\\/UrnlP2qd+oZhAcNcdKs8nQiXTKecoYpkBtP08kHD0Sy89SB8dvthiPsAUc\\\\\\\/PwXPD0X1FmwrES6Yv8qgOb2L89rERRcjbfztY8aVtJ+sHv1ObKHbiodSX0f0JgimCyJPWJr6+ITP4KMxcDIjUcrdevPTgz5h6fS0O6rVHDfOXcws\\\\\\\/o2jX3Zo6QPkGx2hJD9wPXwbpBGyWgBhyPnXUKInLi8Ai+gOTnrnJIKC\\\\\\\/MC+Ezt4P26G9ME1X35EAdVbafNhWSQqv2jkSQ7w6hTyteooo3MxnUxHaxBhy699yLYJ\\\\\\\/4nMQn6SzC1R6R1DeK++wuvbhD+hI1mMtdnp9EfE6xOzjhyIS\\\\\\\/3ezIZo9zDWmUjVuy+AdLFLP+ubnWt\\\\\\\/slYkr5GghWdRorjqJkoH6xCImoOdX8WtsDhmVET9ErN98oWGY8JBuqdmvQeUxFYCyTXZQO61ZuyzLNW9liuGIzL99icJaj03xiNLkMwCQ7uaVWwYlGtLBbkuaVQHrZ41AOKvUnUIHlNjgpn0nNtDkT593cJqz3HRxbj7Z8qeSXYvRo4rY33Y1Rd0NmA5qyiC9q5+ptc\\\\\\\/DFgE81AD21WTRuhcEpMor4mRAr4iNDFcl6SzdaWrbbS4kufQC2X43tn4STHnwaSDfOeDAdNAkWRPrap3lJkq4+v5ZMp64i2lkiMO9g2ycboHmUK4YVX+gLi1d1KuUCObomJKkJkurqo200K4onR2U9tHvRsoAO4qnxyxkt5+dip1IhNcoI98wqxVzOhr6XFX7P08eGdIKjdpCdtqRKmWry+w2yD95uodQ0iHzIvO7X9nqChNRk8+LiHca3McXg2i2OBop3rb\\\\\\\/xagKQ1CvOLaEgaQ709v0LuZDjA24Nn9WtzWi7xXrqGqSFfvK1xE0ov\\\\\\\/iS21+h3KrP7YUv9Uuwx3yQMH2DAJt1Msui4IKtrigyocXXj\\\\\\\/LyQvBuISt7BV2qG38mrtiXtazlAIQoWqvw03C1JrzQbmOoY2WKTYKTOTynlLm2csbu2iTfVmYROdgSUAEk5+uQmRTq83RTs5EkFVZnOjXo5DoRZhFiaeLlEzVeIBBY8iwXSD0YpSL8YXCsBhgvubkW6VySdmXNLx\\\\\\\/HTKpZLq8yjyNcTr2aUjB1RoemDa+OWu9Vo\\\\\\\/sZ1uufT3DkzU63PwGcODOohegZaxSeAaUswvxagJPlt+Dp2hIC80Tszo9iWe\\\\\\\/5nP7vCYxgG5T4WGzALrMBkQsZehJN4C42Cf1AOhmfVgHq9yUpfgqvDgaVpAZgZ0W2E+ZvjSWZcX4UcPVw1C0dNi5BexQnMVF+doyfDbXzdRGQXF2fscvqM6dQiXK8Nlx29HMXM7uurxnUXiJKaVqDzcRQtBJXP5ABJck5iJHeYeyC2qLqwnY4Jvh7b8G95sVLXtkFeIkGbhDc27VINTR+jXmmIBkqcDJu0eOJcpjd39gXQza2aMwLzn6feGaDeqkm+KU2JkG0EPmri8lKuqEfYNawzC+Orp\\\\\\\/bMpHOo5Uiw5dsGHTNtwa0XvlrPPS43qSxkxBvetEKPWJSc5kIZ1MsNKnXqUMQRw1RVbEZqR0o3XA33pCzjZhVu30dQJAUMdf++gItBuEhwQXGMlv6nfQqTBxn\\\\\\\/8g\\\\\\\/HxyFrnii+LuUVNhr5d7YJSlRwGAYOb9OOGrubDliHFe\\\\\\\/ocC1STDD3uqXGu186ge9Iaxd7AtPxcseBiCwSV7ZN5xo8zqBZBTFU2lEXv5tx0mNVF75JvdGXsA+tbf6Rtu4vFmw4khJ3BFy3vVwUbjf94Cbe0wQP9WLld5f3CC7pj0NFHbWtR0MNCinH7Fc8TwKZO7HFAJ0dm9iIe9ZiyRWt17iqchklT2vxOpavP+5YEpjnHrmMy42opEvNCKwXYXmjw7tGka9\\\\\\\/uMj+wcOw1Grd5K\\\\\\\/q18O6A\\\\\\\/HqIkPf0XUk79Itu9K7IiMTj3LdApgShCaNmlle4wu4fXHkRyiMVfNvkf6R\\\\\\\/cVfZb1eGhqxyvtG6weSDapWDzTjetaifI8EapJJL5qH2D6P7t0RiuI86hUDsfiW5GCeaqFc2pLLzWki1\\\\\\\/KBLstvr5huCc3CPsXCo+Q0k6kKTGoNog8HW0saQvTMkM5nE33UfT1ulboi3HQ4klaXwVrL99TRh\\\\\\\/f4f6hWKIV5TFS5Mwz+AJsaNj7GeGgI\\\\\\\/TKHHRnKMs5YTe2nLkU\\\\\\\/ErYdaBnMqF6F0rvfnbh0h1DN7MidForBFc0717MXs49f5V7sZJmoM6G9p2gMvu+iQDYIUhPvc3b1uPv7e7oxHIAerXoXCrbKHJTlCNC3H2JGJgeM8yChulNiB9NfASxZofQu0PKXZFxCol5pC2+17Er7NVejl0Mnywnib55gknu6GdP0SdMwLxWoOyfGbUPePd5smjoUl6LSvKqNgp3+0uW1MA3HdUP3UgFcvUosHCv32YqWna1n1c4aN8XTKMIKus8+GsPQML64RsUvdOPFVlFGuNrSvWiJq7JkjKLnQ7kb+2jXAQ0\\\\\\\/voeFrP46PH\\\\\\\/\\\\\\\/s2vJuaiPJTGeNKZ4CGaS6exQVFqDln8GUDTRru1kuX\\\\\\\/8wbvmLXY5dO9yOhNNJVIMF70RpilgFjFmoVwUWOEO+WCp3iUsopjOxl3uxqwpFZF3ONawqfNucYZfdEnKN9q1gFNYPiD\\\\\\\/iQp2DmxXBtNQSs+uGr6o3EZbRQNp12XkZZoQzhPHpswAFCKWrB1Josmcb2QaM0sFJOH2bsVhk3\\\\\\\/8hJYx1E1d8XdBhv1\\\\\\\/ZVdcgb\\\\\\\/ssKxUi43HZTU0uc7cANhH0nWOgggbjWqb1t\\\\\\\/2XS38cCgeAawLVa2al875Um48GvdIzhL+UppVL9d547YEqF4s\\\\\\\/WdfTEKApsl9CM06zdkn7eitopWcAqjp6Sm+8K4K9DNS9yUCA0hWjPniE9tGITRal4kwcbJNF\\\\\\\/UY8kno6KOuFvOZ3XpcqmngbC+WVAXi8xceZ19gvBpyr0w5XfrKrC4wB8NFByPSScZdcboCQzzTixFqT93GRhxBBmfk7\\\\\\\/iVVrDjONsp\\\\\\\/rwA1eGa8jkZG8ShMO0RCheugTTJg8cBVvz6psnve+7szNP6aSw85oDrfNkAA+VaKIpqAIcbEo+NSw1Cre\\\\\\\/6f6UAeTQBAV9QU6TLgIBfnY0kHskrHUL5yFe8Ix60UP6ZyLjONkbDfuhvR\\\\\\\/wRm6+eZ3tgpeMmS6H\\\\\\\/aM0whQKRdd165\\\\\\\/Av2GZZfTTfFW0B2luI0pqNJiXmAqDINjBidxYcwvwt7+gfX\\\\\\\/ulUk1Neby27Yr8MYf9AHYwOrmVpF+prOCB7EPuoOkzLAqSly3OLzHvXFK0FOG2wDk0o6siy9wIaB4\\\\\\\/kIXcnA7adn2dkNiTLcr50mIQT9qe6FL9ymzFn53\\\\\\\/QipbIjiw5Akz+PSecHIdXFTkpKIUWCnSpcgB89oKk8\\\\\\\/PaqxAA4UWRdmpDP3WgzfU80Ud1nc5ntckPunKOtDfkzTX3cRAbftiCmqCgyp+2xR+lm1hQ0Ik2Z8Ly8hUpjRlb0LkgxXMCu4MJPFIUPr4YjvQlHYVYwyP9hZJ+q5abBbVQsJZ86vAOQNYjpz5dmp7J762PDFzGLlbBsZgqUCwZUTjvE8nX9EwJSJWW9UEBGVxQAJQF0xZIzN5ZCu0N4cc4Su++ApC14e5ZL3SJ6VqqUyPWOwCAsen77uvSNT++WZRHON06T3P94Fe9Cc+Sqyd6w7vo+vehPMQFDg\\\\\\\/fLKHE9+IeTsFA4D0dYK36BFkN5ayJd\\\\\\\/v3OLpY0XDHnOThXvtTOopiXg1Gcxou0C5jh3jdtuNqEw43aXZ7cEvAtNxccpNdYrHlbZA+hTLFf+vN7\\\\\\\/URCpVj4+uWUOT2MEtZmanwLKotVRd8If4CNGqAXXiv69arYHjlwv9Xabz1BVq33WjzjZhDkG89F0qe\\\\\\\/Z+aim2zFMO56JmNmpYt425HUz4cgzjFL0rUcf5XTgDVA9l3ATwzwMRQPXHQ9Hyvn9I6y4P7T0KTg51IzPzLlz\\\\\\\/dRfoehOeXNZKFhzBzD5DUCnWDCIzHKw54NTw3lKTesAIi+cdqBQ1xXzeGwLxdWYCNPnycUXzKCqY7HkYSyOmHy\\\\\\\/2ycgkebZBPHTs73oYNjlpce\\\\\\\/XkbFht4K4QvYHK9FBJEPtLl3y48eOg1TVpa+yavV6Jcl42U31R\\\\\\\/bosh3BeQpwlNv5EcRjf0AB53H2kn9FQUtCDAWIlRh3NqlOJBOkbHXJxAGL\\\\\\\/+McNJtXqdpswgUan5iiXB7eKHXeOerem72+cUmzynwbB9Q+UHmGOF\\\\\\\/Gc62dFVdQk\\\\\\\/ry8tDNCKibCdmREwUmoveIy5e9RO\\\\\\\/bHYRTc7zwYqr9g7h7757sxnm9wBKO4hNTRjbNEc+Z9VgaIo7JN19w0rNtl2tb3960vXkk\\\\\\\/qbGy1Ul0V+T8XhVyhrGitlfIz\\\\\\\/bZopNVIjqgp8LIk3etNachbvilF2XwNqCivNIdsJc3Mkh543dgdXXNFwBqvwmRkaXyUi28ZFnQDADSTHeFe9xUk7SD7WagM8N+mnkNi6yOT3MBhxCLFs0YbCO2WroDc7V65eM7JoZVdyKaLtj7Ih2b18osZktY+RguTq\\\\\\\/Em0TKvoFjg+fufFLtQoZjN1ZYObbPh5EdpDm7naQIWfpR2iW2EnY0n5kaWrDVMOZOlq6g4vxz8LojS0TWMjtJGrZp+rMZUTd+lCYTaVwr0lcJEftTyt\\\\\\\/ZjU0JH0rC1j9PY3hRbTVHXLPMlKgsYDigjc\\\\\\\/miVb8Je0Kd\\\\\\\/2ZUmo3SDk6\\\\\\\/4AVZyIxqZCaOVdiFDhjoKSSCKXGsmqmbTkx3TKQXLULO+lcK+yCkoGqcNtXPI48h39gJZfx14ArK9imndxk+A28XN\\\\\\\/mlqDsXoukqgKYAEoTRCaIzNgImoEKUnDQMKMz6exPBPMtLt5RR0LwEIGGr+gLSUdfDEIrdAkzNxj+deNg245bYBKOvYolW1R95XAh69PbWZXTrcRF4E+dgIpWDRmirWLPpKlTpcR7hHTuh\\\\\\\/4ggOtS6ZQ+VbyzXz6qhHn22j089tCEco7bg9W1xVjqEZ9VMkdCg7sDUd8tN4M\\\\\\\/eTHJZo8NLXYy3NPgPRutwR4L9nNpr9wNnYV8hkdo2YgjcwaNuj+POi5NiJ6Fd\\\\\\\/R\\\\\\\/uSAjQBKrQaCxkDhhA66KZFU8yI9TpW9qVBXiKh6H1Ngl\\\\\\\/w71ytEZx5WbqxCD8cOn4aQx4J+UE634HcMUBKIOsi73b0eOuWCHjWab1Nl0b1no3tvOMxHrkZPExr6tqpwET2W6x7DzsVwc+yph4ABN+TSvckPcBr1ZYzcz33zN9gWGRTYYdGIOs\\\\\\\/cyDYfGXk7LUM7qfXsprqsmqmuFMaJt7lNB0KAPCI7ux7GmnTKD8\\\\\\\/dEQVE\\\\\\\/W25rkOROMwc45Lx5wARW1S1K9Re+LXkSiO4lRrcx04gdPS8ellK8iJM7gmsjf\\\\\\\/saVKk5u6b6ahRGfVDzAagFmb+5HeHwybBzu8N0NlkCchuw56CpGdnRcmDi8M6lQRq\\\\\\\/9uKhwaq5xj6\\\\\\\/XQHg1mJmuV9vxRs3bkx1p4zIEa2IuDYaJ2ibm3\\\\\\\/52tz4RO7Yy7D4sSVxXhcnKAq4iGGYmY3U3XF6C0Z2YzR39fnMCCNR+YvNu\\\\\\\/zWyAIlYafnDFKI1nUO0ItnGUSlHP9S8VYqjtjmPF155CbGD1OuwnS6ZKgzlz0mn\\\\\\\/kAObdmstIaRJpG\\\\\\\/PD8wEXrWClhdIA2FBFwaH1qi7tk4DKmeFuyXbL36OK9kovUjXu8JEeR8UwpJqRA9ApLTIHhEbTmSMo3NqjSYROMs\\\\\\\/do2oYpGnodr4KpEUKxESvE1tOn+at33+hmnb741zryOn1ujvKfP75L+tSQbJK\\\\\\\/iyAfmf14b3cVcKcxA\\\\\\\/JGlJHx3UQp0gtaT6l7nIXkf1MzINdw6sOqeGXym+OZReIIKkz4PYRQy32\\\\\\\/CrBY5tNAJL7ebUDKsWgnOSsl2kS8uUAlvxY0hLnqI+I9OYpH3D+\\\\\\\/kyhuw6xBpOPyddfScTeUWgor8dTT\\\\\\\/Ww05tiZ0CnMdticeX+hz3Z2aKnQccEFee50ytxGXMMsNzMbd949t85phjxg0kY2A\\\\\\\/vJWBef2WR\\\\\\\/zup7mTdmoMkyVwf2O3pkMA7N2CDvXbcMWWulz\\\\\\\/qajcUToCT0PzURWTs8EZHnwAGVzVLJvAef6MF2HQPZwm8hDMlFwYl6tsfdSTtEiO1AAfCd08qoORJUpw8wB6rVf9KLqKIN7oXGNhm+\\\\\\\/Uzc2cwbw0CS+fjY1GlLhPwOj9oMjSGBUrN\\\\\\\/HYhlVbC+\\\\\\\/gLdHGpc0r8HFjlsyGf62mEi8ROaeboMb0QoLpU4SfrIMIaEtaVJRdYsw9WCn7VUtXEoz8Q\\\\\\\/z4Yb1Y1sWZyFJsMBBuFvMu4YYlWUzVoDeQb7nyslSEubFJFjzCjWnsoDA1nfb42A+eFA0Ow7Nq6QFD2xgyT4vssZM6aQhdUK0BUES2JxyZrax6A7MJZrVkwvTF+lsxXQjzYv8OrYVRXb8zjUVWwlp9BeSmkcOpNDSdps\\\\\\\/cpjE3JjBijXGAqa3OHvIk6VZ0d7slPIazLNH7C08FtdJNgtljU0Y7sx51Klt5vKvnhpV2hV\\\\\\\/FdsR+PLHlesk+SBh\\\\\\\/krB1WtarqwL4TnBMgW2FlPUR+\\\\\\\/ylV\\\\\\\/HhjiZT4XppzO8Db\\\\\\\/OLjhGNENnCUOR5LqXpmjC0EMEX+ikt+Uc91m4rA0DVlKzX6MYHNT9jV5FNSyeQcTbwahMbgcyWWYyw7\\\\\\\/+UlqqLKuzk6YaYePBfKKiVgKkCvWq3uNs+Y+QKpbP9ijtHUWmfnAdlf\\\\\\\/Gx4Sj6hHltyBhL0svGNSLHGyS6I3By38nDDYvlQami3gLeR5bTd3v0ylKkIrS9fcVaAbpCLcI9MZnkTO6QD2P6GNzOiKYu+vOrhDUN6n72\\\\\\\/w26\\\\\\\/YWD7Vwa4V3ezXjffyfzEMHddQExLuIZOCLPp76jz9YFtvyg4NQnYg1\\\\\\\/JZwP0ETeaZ9uP51jRH7na2OkaUA6xdMA30kBEtRzeuEsmDI+n+qwfMru6BBlLClPkgNQYFeYFgdBjMylPQjo8iX0UDFi4vqDjsvi4vn5DG2JQYjyml5OzcGlND09pltIiHkQaanCfXb0BpeKWcgo4Dx8rtsOUKNqNLtMgM44oMfczpZzpEXaZhl97bJRzmYKBYJPi+K0B856a5moxGnQrE+kvdRYYfa664NQIZXsat7eRg9JI9T1eNSOtzy1onYRDncqcgSfl+5Abfk6c\\\\\\\/VSjYHw+0TJ0W+NXn8d13prPONuB+Qn0f60YVMjWda\\\\\\\/x\\\\\\\/hO3LK4OQ7ZAJCBasT2KJiXu8U2B7rjXtKmhvRlOz8RA5+OP0IpReEi3EUvb5S3zMFxYdO\\\\\\\/zGBImKbz0sskfZs+h8u3L2aTTZ9Z7A\\\\\\\/3ZUmPCuVI0rAXEaXbJksitO+zsAaqHYPvzjKridYqVgo\\\\\\\/9ad3Wz+MpOdy0LdhEKCcYTBcCat5Lsdzosr6c7mfNzmOlQgulm676l6L\\\\\\\/fAq4GScbRARi1h1LB1qhp1WDujwf+8+P0tHr7pFc9IbIv2bVJ\\\\\\\/vuBWpaNHHL+ol11eLXCaLH\\\\\\\/1HAS2mHqONzxeRXK4f8wjx6i2BKoggwH2S\\\\\\\/F3jMCXTkaE89OTwANzirJ4yy3byBjGAeEtc2\\\\\\\/purZqjotqsw9MskYNqKoR52P8Jl2DSZLNpVen\\\\\\\/aS9Y+Y8l5e3B5N2akCxu3qN7N5t7UKCrEfLmeakLFzETtTDREeFFhEUYG8hwJSQnHMB133AyfPTQ\\\\\\\/6p+pQ15aT51YOTRXB19JM5RQrQtMn1WhBHh1TDkepIU50+q\\\\\\\/Im368+RVNwo69n5cLYF0SWTB+T9JD\\\\\\\/lya+9JWCZZ2FWmIZkEvhODCzWPzal95A8t3sXy2jkoLvVc0lG4\\\\\\\/BcszPmcr4qenFFebEVVJsNVJGtiUSY2L0y8WaEE6zGwDM4DIzXwYV+cWrCySrmBd1+CXAmUuPG8l1lmW3e3PnHU3UZfcM8iJaSedKrfkCqXQr2R0rg8DPWkK\\\\\\\/qmI1wCDrgw6GlTwuPKdBWdaxKaCRUpGfFO8rjQtdP2nO833b5FmGbU5D5kvxKOJe9H+zUP99J9RogTMtaGbQ443KRy4LNQi3+xb9Lt\\\\\\\/kJQzlcISNscjp2ezwQc6OV+2bdpopE0uBwIVLzcBkR+v8oz4cxOi646ztyKMygrVy\\\\\\\/TgGhSbDdhXhk2o9EqeR0aDFYSrvf+4zzCWUdLP2HDI+zSd5jXkZi\\\\\\\/WIqUwlfXC7gN17s\\\\\\\/aeIgIiy5zvCoWz9dBL9X+0teLXOdY9jh9tojPEf\\\\\\\/gPAFov5vJheMyqW+Kx\\\\\\\/SEVXYNZ3R0Y+eri8\\\\\\\/LLL9DrzDvah\\\\\\\/6QpCgK9VdUdc0VydqrkYT5ehC0TwybgIJLXM350FwRlCr3JDNKSWS253XHo\\\\\\\/SdeQDor2EX8zixSi\\\\\\\/lR\\\\\\\/SGfUX2IO4c\\\\\\\/yaV8bnSo4N355bRgx6MncQXW9o2K8ek754h0cNOTOZPSoS4gHLGKX2XWW21+f5ri+Ja6XgyWYXoj4de8kMJSyY13d5j5G6r2ShemEChmtJt3P\\\\\\\/+G2Bx\\\\\\\/NTbxWJCrm2O3oMLQkha8XuO0xWt\\\\\\\/YARSf1xY4W8537hi8apVYh8tSGrAItR6khP2b6hNCaRtgFNGscQz\\\\\\\/sjErkj3WyMihQ1o5V0XNiEiW7dHP85ot15olFF0s5kRjKJaM3LtU1u0bHL\\\\\\\/R+Z89t+UrwRu3CwutWUnafR+4xFp8kIhcQgQDCdXS+ReldFAnntK6zJWFcUTUDc6JZ\\\\\\\/96NZPneeAwoX9aK8rEMz3vt9n2sjR0yO6FUorBUjZpF48MC5jkptzyb3R\\\\\\\/Knaxwd7gjADpM908IuXEwBbnxxFOLRFkZFjvUQmkn9ujIdzFnochd8+KPZOGkqS3IeHzZLwgWqqBGOhoZXp9lWRx1BJaSotLXYV3jUNqDUhZ5k2Pjdgshcdtg0vBu7PUs2XmhSc9hQge377aJtruzrLH\\\\\\\/dehMqSFURb7Rc20WptlPRwyet4f20FI\\\\\\\/81uMnBs+ousSitKbea5fOeBMknNaXOkoAtma4QzCZYXbkQR5FAobqhyq92PBu9A0jRyG0wB13r9A0FBbqQShcMuSIyUGTOTSrpuq64fA1yZLadhFxZPh0anQsb\\\\\\\/Z3sXmQR9IKElOWSju0UHy6yt0LvxEFQd9Os48cm\\\\\\\/mPdIhCGCidVrNpOecC2VY3XHWhGZDR0UnMM5kK+i5L8B2\\\\\\\/78o7hPNss8jN32KWguNt73qeBi91Y2ijWFSvzF9YvHN2DJi41ETjJYkAQpoffY\\\\\\\/24tvZxKHP3TrrpE6dB7uI4E7T3mVEHwadLIjMQXaw4WFBPFv\\\\\\\/fdVg+Rwe+lbEcGiJzM5s761An5fyJ98S\\\\\\\/rgf6DC8RupjwS\\\\\\\/i8NyHCQSdhpZnYF\\\\\\\/+8S1GtUcDJ2PmYgRlBxkQ3\\\\\\\/05fpaknQvWPOE2UlGlC5\\\\\\\/u42298PBhEW36Dpfgkg0I+vcdC6Ml9Qwm5ftKnyBJmmIw68ID6HI4aFy9tMLgCG\\\\\\\/bL42twlzwZgQvzxAAjxc0j3+ZVy80XdnNuVPFdyByUg1BYnxDHcRBt+VuCtNKu6Lj59b4CTIRSP8MfBMGR7AIr\\\\\\\/pXG6OSVl4QXlktm1IAYifBKfpfawUTPopEA1FzgVqF3qOeBF5dD876ojSKTmN3pdgj6k+gB8cQ5+enKvWhxrTkje231DjtrNxAPKltGz6twLBYMr0AMGs585R1EdlDYTGxrOfjsBZhaRIjg\\\\\\\/FfU679vnPqT85m2VcnTJ09dSmGyiAC63uT8fpVU2yGAFBeKX3cRitqsY1lUuNzL+0NT3TRJivwhCYupc\\\\\\\/3HH0Cocaj9RqbtyQcE5OnGGLLN3HcsGLSC4WbHk0PcJYQ+yFZX9+gaPp4ntWXLoeZNIXlrWorR\\\\\\\/GDNJ3sFxHGFQ72PJ15k71ge+HbeGiXWNDtejThnMrJQcsKez50nLZWH\\\\\\\/CeqS8Z\\\\\\\/F72\\\\\\\/5vIo5xXs7W4idoRMsiD8pV9mc+i73DO36BiGN5c3qjYUh6XAqROdtc45zNChQq8GbW+8rBVGiv381KYOxMz79eBWZbBaWP1jyPoXI1BSbRWRsUETLBO9sc31ujPPMZjdEN8d4D\\\\\\\/2DkT5RPk8zOiyEvbjf0hAv0LFemQywwHGimAf1t3CV\\\\\\\/CHpHHU6TRZwX1XYngxcgM6POALDUutytdvpGTrgEucKiGLJk5wzWvncEsx5fqiXwR+cLSkbyPQH4mW3pruN7LDSpYWTICsu3lP\\\\\\\/1CreyWYEQqzYisdAPd5vHs6zV8lpiF\\\\\\\/Ldjn3WNMsmuAu+pAzJbIrLqzAZl6Yco8wKN1II91TH7Ls1KWBdxlQRqZ2K2lEupy1ODBl05MlCqQz27r7+IkAUIwYrr+Sc7Nhd0jQDGZSFLKEHAVq3pxz9coP5cWGOCPPungspPXPUl7gvjSLQomALsf\\\\\\\/Vn4eXVnS7Wg5Foevxly9JMxyyiDkuaYK6zmh56uwUlbso7XMF\\\\\\\/7a61pDx7eUfc0kQwX2Oa6Xsr6sBUAVb9oIo05+0aPXgMhsd+EPctJKF6GQF4aPNqsD4sQvMok4WwQNFhl9tMbkRHV2qNKIJ2NoSpEm5aS2R783zmjoDxwkgqV39FH2VbQ5pznFLX3WZnJBC+a2fjBrsv+RSLuN3Dhbe7Bfh2uTU1WtScuMq6rvTQidBqNi2LhZhVYpt95HB5Cbek\\\\\\\/OAyoD8Y1lKMDjrtXRZxJd0ubawSNvf8nxRc\\\\\\\/1BpDBJmTBgaH60DyMLmnVGW\\\\\\\/X2ioSJ6i9+JBMFBNO0V0LSjHUfeZr2PjowVHRs5kCmSNwWpzU5IGIYEbv04dhdx7Wm\\\\\\\/HdU\\\\\\\/3kOuts+jhY88XCltr4QWkyRfDh4rbtlq\\\\\\\/un+bg4oc6qbbgFVxSWXxv965KTUc3CwTSwJDF0yj0NAk76aY6S8XoiSbyLjAo6Lcp3tQuM2IckhVHqEGOLRppbqAFZ3AIZTRP3AcOFLQcQfVHWOy5ZwI7M9qxr\\\\\\\/0heD7A+n14Ewah5iOqnJpZlAAxWGnD1UgIkcYUxAxgym1wtInAIl103RK1X9ODZqNyAuS605ILOjWdtjJb9XfNXkaRCiM0gFzwe6\\\\\\\/haCc+AM5ge\\\\\\\/oA4YWDefSSfLuynWovB9JZQlpKCkVnE6ySq4ZUodSajoHd3VGbll1IKT7hIoSixb0bYla7xQDV4M0g2BYqxR\\\\\\\/XJQM0PLQGAuWbqqajYEBlR03rLCt8hrBpqNHwLE5WZIjfr20aj5ZRJZXHwFsCuemx40dKffaJcTEJf7qng1kDuGw0WZraCmNgYEByL2VzcFRLyuUjV2xrAdtmfOl0LY3DUEs38iXWsg9w9qtrGq9sIxoxTRHcF1N+bjujHvQ4xPa+0b13JuE4rOqPmjhKIMzBugY739uQBZABIKHBRACRUb0J40TvFsBzSPP8f5iXdpWxjSzuq+AWmhWPIcXy1QyI5FMUbpULz28+wmXJx7XVjID2S0Wxjv44rB2FVv4TOaxw5BxW0wCk\\\\\\\/Ug8V9F4NFsBLpyFqIZlcbf\\\\\\\/F9wfBWteltFrG8ndXarmQWgiaH1FaeBNOr2I0Zo7icLS0mYF0zomwGUma8VLl2TGR8f09bvtmvadALKzLXci2bphXJB2Uxw9Xh0XWFDeIU3ClinicipWO\\\\\\\/4wMu+pUIfaKa87zYbiCzVw9OXfq7GMkvn87yNQo6q5dozougLv7jQ2NLyaLwWkI5Qv7uQbUqliX10US4v5R4p34K6wpG10HyqWcqBp3BlG2Lcvdt7\\\\\\\/aWhhFO6GLIVyAoh\\\\\\\/antd1Go1kxQmk2TjDaHyGF3WeUFaAV1JWqKpnALxGdqUA06To268uHU9ExFCJP6nKP9dPV7UtZ2Nzt4BIcOQtMO0yPGcU0FUuUhY1I3Olkq4viq6j6iHNAwxdUMPeCswr4Bf4UPblFKZzbXHzT2f9pz34vfGUspLpJqJw7nZ0xxhesJnIH4xXBe+I3PpsqropNz1cF\\\\\\\/2kTAv+CcjmFd2YaLD2tSe4GJ7rY+8mpheV80mpHoGD3LT9s0m5Y4wilBpXPCNE\\\\\\\/kLpywpGlH7i46qKuhpPu\\\\\\\/kQQlqCmlVBf6jRKxmCMLlozWiDAIpjvtX3rma3ALRYZIdSLXGimFO9Oswx01dqsGl3em6M8s4Eu5ch48fb9Pjs7QNmee+THe8du6q6yTfCfu3xdt9qQNudV50s08a26R4lVc1NMwbMWbTmFGd1kYmSUD5LVd3TAsOGIg+JH+3xLVvEjItKSriOzgDQXZcTflqH7TpFOeHX+aCz\\\\\\\/0YRBk3TEHcUp7gasStxnPfG+gCq2UTJT7SH7yh9bagMRaLjygXQ1Q2Id\\\\\\\/WbNNAD+hjaWhXkvshOJ6DFJLrw5CqiYEXzq7H51XUP0xycCCrMHlyQ8UqXmTdMTR6W\\\\\\\/iZSsiGl3rvd+Sq8RVdkOZx0fcEgcxIOffzMqxyEekXL47SL6ZngWHF582\\\\\\\/XRliyY6sDNozXfv+p2f8jwZ+GEDNXa2NPs0gYJmw8pCz1hoecPedChsHfM3+HVtAbwJ9YpfLNQpOWeEihElT0DLIuIBsjfCWI7tOGpYszvt2XcwfTAkxQyFAAMDWnnNbhU3HmU5xl298EOoYeGAGje4FHk8OlRl61p0W65Lg6h0FFuwwKHOsAnYOAIZMEyW5hlnP7jSwJXGkChShemNtTws+n8zwCdQ\\\\\\\/FYefxs+CozduuY+hRnQ7q+oAFjBPjEi0SCDMZQ59CTC6yqAOIH0bhpKjegma4RH8LtOfRIZYTmG8tw3tKiMMf\\\\\\\/E8X+w4vW3\\\\\\\/qmdqQ+R2BgMb4UtKMSKrJdEAime\\\\\\\/OXFp\\\\\\\/ofIKz9u12Exbm2Rn\\\\\\\/q0y8EIVLd4eYAOCLVnGn\\\\\\\/fBpelvO57fCQm67wmDGSZ8grfHyiv3\\\\\\\/stP4HQI8KLPN+KskHrbNumNLMUfgVDn3mDK4Je\\\\\\\/SoDBGgyHMe\\\\\\\/2Ml6mlSaupfNSwV\\\\\\\/EenR524G\\\\\\\/7gcBAjxrUYlQe1Uba32FrpcGv7Y+LP3Z6IZkAPOqyHkAhTvo1kY9+SWSO0CaqsU3WM\\\\\\\/YXXGxKNbatCc6JvFc5ZStA7GY2ZJFSlFdc37qtd2TQZABbCyIXpuukWOBbwfegikegtLo2jjSSwVJ0VnGep8iLeFCS3Cdg1JEgtl60rIVHtTiNqx9X8gdSu5j\\\\\\\/pOcTyjmYrMSDkIZzNVMocFjvhBLvt8boeF+bZTucIDzYRlladi80cPhO0CCBLeztlBEJOJuD2VTwEnClEENY97ZLsTfuIEeuKRY0X4jX2P8SH5OXc5wKNTTLUJv5BkIzvpoXio0CMDIa5qntZYrkwuTMRe0MWxHHb0nJ4j8cLpB3QojQfd53zPeQekbPq4WZnKakgjxpSsBGb8lLluHVJN1eN5eo8Q\\\\\\\/eYJjLPWB9w3NxXJBz3RHriTuPCSDPeM86F9f6U6YktyXWxNxZ1hVOOGpBDou4iBh3S7+Xuzz44R1rtmfmfgZyxhV7dq225pj9IsJZmwLvqmld0lOpprgmwbYqPpLM+URcNjrwhBVf6vewBiPycEq4bDH+okrmY15ANjGgqRGCV2fr3aKYnqWXHrG+\\\\\\\/KKgqQH6GfuyT7wyZzKriTNpt\\\\\\\/mfA2Gf9tJjAoYzqpg4GTMEeUS6uCTZrvLopwzQLufAZYFYWMOAAp9zgIC1PYFG9R\\\\\\\/3v7hDG3STsD0TXT+uF7unn1RhdMb\\\\\\\/tXEgjGFU\\\\\\\/JurMyZ0Ex\\\\\\\/ukrlocdb8d1FqytUVBj\\\\\\\/CDpD2JvJB9WYG9q01YPSzaJcgLzA\\\\\\\/Q9jSPFJEyF4S3Gx9ThOUlGQtd1Rvxc+9oy5flI53OvBw93fAMfUppufnZk5HV8sHk6N8Qiuw31MKjlXnk1reJ8ePKjNh5vH71hHK4KcrPgv+BpftWfFatf6vbs8c6FmZskGz+bVH9P827IQluZRLGVkzxvHwOIaAl+TCEnIG89GC\\\\\\\/kF\\\\\\\/LZU7SMj1uinKish9He\\\\\\\/QwKUts\\\\\\\/+\\\\\\\/i8aNffUuwKfy\\\\\\\/uMEz54r41jTPSTzcIEqlg0YT0zTmui5By8f00xgg61Kff36d4EDedfOgib0g2D\\\\\\\/fU2wNcNCc6bt2akDdyY+pdtw2EsC4PsDzjeojazNLKBEnxVD75W3WV4SBmZEHQNEQylql2n4ViNILu6TUZemTB0EDf+ryu16+OV4YLXHGFW6BIErVj6jVazbNIgmLkgxKVg2kGXyE7T3VRHVIo0Hyw0En+jUBQYPfeuduZhEG8Bn9UiWLSoFgDrCj63iX9nBfA1XJMvXsb+NXqE80LIddbTBscUUtGt0mrq7kKT0Wq4MOl927qFWnOYapNTDhT5arexb3weoLky+WbpdAi0tvg7gp6cgM4\\\\\\\/H3gKXoNziUWk5cSWb2NdYMP7i2ynCnkva6a8+ICxsQuH1frdDDPbrF5z7K6TPtdo22h3sAZ9P4gw6pMy3tqB4jriTx\\\\\\\/L1LnGQFSjQ9EbMNQpxPTLsJhlSnORpR9yQ64TCiLDorVU1zaTKhWAUvP2Amyl6P0sXbf\\\\\\\/wk9tIBtNpp3rjcJE4vwM+pmtlU\\\\\\\/geIQZUQM8LZbKvQH6EYacdDTLK7zMjI+56uESThZoULKDKFWm5vc9WIsfDkPHaMI0SVQdCp4Bkw1u0npIIBQW79NpLa4ksR3xXbBuR1npb7PA8+bl02WY3b80wEW9yzKU5KANueO3MEySKIxGBj+kP6+4Bzt9HHbJQ0i7KF2iomq2hjnMEdb2NZVI9sfUs8LZNNYiSHEwqvbcz0MNkJzg6PPiGp5IsD7aGUxAKwSCuGnS7mXmmgrLtbJMK2e9ogQpsFASsFIYxZX3He2t2XxF1b0tsQj1GtJHX8GA5z2OLqntx7lsFekUNB8S6YDsAQgKFrcdjBQ2\\\\\\\/odRjNgxB2Du+u6b03TSgB5\\\\\\\/jXJnipNHc97r6rxB\\\\\\\/RVEWM5nDBL2jKEQCd0tZTrCG6sQeZepqxiOY244dT7tjQzYOFL\\\\\\\/k6IAyGyDGB+gMAbukDOlJM0DmKlnni273KuhEEr7NDRhNftGDbcQweemRitLCKimcBSCZ+Jdk0j0ukP6Ul4hz\\\\\\\/m8lCJt\\\\\\\/hmw8sPno4yPd+phWolmvjZOfMGI3j2jkAS55lCS\\\\\\\/j2MUzFChaCN9lYBI1hfKjPSHloS6XF1Mz5u+aBN1AyrzdLQybhGM\\\\\\\/gEOCdcdV5OGLWY0cdiHRUj0pOZ86+GfE5xohQAIIwI+W\\\\\\\/2fztr0eX6ITltifgYkw3cVGjsWGg2mH33oBc9uowu7WfId0nrakQ7BcIvgUIm+s5d1h+9WBnw+8wSIUzW1ttoQZFmBhKFOANnHKKio0iYhcMMJmybepRSghi8SwgPWIthAlWxFNRiKpLPpfc7PCMUVWiRHKAiBoNf\\\\\\\/cpxwpryUC\\\\\\\/H6JTvgI7y9p3sxo\\\\\\\/nKYFyfpfgC7Z2qyo9pDdlXJ+YmCwGDm6H7TH9Q3O4KF2u3GyKniZ0Jkbgu38LVr4bL\\\\\\\/ewag8IhnEoFBVlKhNy9EciACPU9rCup49J5a6+bVvAlgWIm6O2+dEKzygh9FJ5GK5p2nZK6RyuvzXOhalq1Shb5JlPGZkhIheecv8QNLNUOW6zhkJ0EUEdJiTuuRcrEkemyWyJXYn1Si5k1m4vL72KF\\\\\\\/2WCnllXDDcbCe2ATW1Qc348Q\\\\\\\/8XgxrAurt0TfCXgMZRSF63mZB3gTgTqsZp9ye0OLDTCMb\\\\\\\/QW1yzfArnlyk8ffYEDc6jUdMiI+osX+mMfuAUqN\\\\\\\/EW5B75quGTpwTdPOPNqVZqCZXw8ESFylf\\\\\\\/AnauJ9P1AayoIDOTEXe+w9I72BSZ84l5O+nkQzpwZjYxADBGF+VjSbnrwqGzns6vguEqV9mTQZg5pwXp5Hj7wuUV3eKjkUp\\\\\\\/NH69DXHGpSiAeRDQVgRuiCLJStu5xWRH76PsR\\\\\\\/U976x9w3WcpXkRjPcCyFq6bC30WCJG6xoe+aih5axRmtUUJptM0bDTvvFewcYDWBT05wDdjMzXTrbfwQd95IEWLoReMXfYhGNy\\\\\\\/xOZtOjZ4cX7Qs1Dpp+AlS2txMzU5oHV3m4o4PyJXnyjaOJUcHwcTsppqjl1VEoGB58EB8W93Pc49tOW5CfAQ25\\\\\\\/c8eXsCS66mdDbMuhzAcomCiiVFAlFG2GL5Uq49ohV+rGBnedUt1Mo2omrnTGrCVgp8nm+0TXzkQCwq9Yc4RS7y5FA9Oe0nOQN5ASCojH35IhED4++X2Of5+oEcjN17GVMeI0TAZwR\\\\\\\/B+Oyaq9fe6y0KXAjDEnf+ZzBeIAwQLv6nB0OSxITW16tO7SwidaXYzLT2YY1JV6thrcUZIdmwbbIownEGtApyDb6969wmGdYGw63BveeuGUFAmj0XKS5nudJFX+dP8LwYasjNGwEKWhhaFB1JnsqVsv8PXdwWpTy2nL1vF03uXoRyuaI4PsAAdeLVV4g3eB5XWK01lMJcd0Oc7dCGRutlm371FrArPB10XNz6dCMveeAH0JHJYCsNQfTIgz3nPNEQx7nXC5F8+R4Cjz4C5oJkPBSdAIp\\\\\\\/GjuAEGIEuVs2AgkTjbT8fqovc9lXY\\\\\\\/d9lwwkIyoeUpfcuw78bl6zik4mlEeCFX49flhZWyGgDnU2f0sdOxaR0TmNj5A6\\\\\\\/cO7sUKp0npiypaM1qFC+ZuaESFisZJmvYx+wa4llTn+GWsj0cjC4+ltMVqpWx\\\\\\\/GdXaw3Qvv0Zpnpci06lAdrn1YrVHPaym2xc31O1PLwkvh7GORAoejEC6ZmA69X2FrQPn6FwSJLShnEVUoijQHia+ELa0hbP0NGlmUO8V2gy07\\\\\\\/2SOza7va8HB8QlZfhSJ5xh07JFSPgEotAyxoBkxAbBGXxhyiy3TawrAXyzgSn8Q3ec1sQL7dvhwpQbocPqbIXMOilzTU+rrcU8bsaDJxQgPnFVo+XqmPQ3dmN\\\\\\\/txl7djD7PJr\\\\\\\/YRYUE4G32Ce7t3Bh2vBFBRpcmvV2hOg8T9gHW9oZ7nax\\\\\\\/13otehbfU0jfKwWsRtHT8Wx\\\\\\\/1vWln19q4II91ZIj\\\\\\\/A\\\\\\\/uC2YFeT9VJU7q5LqmmvErjTQUFkFFO1RVOLwbsPTpg9DtA2t6\\\\\\\/0EVWOLTdv+i+nR4qx3zv2fGz3F0dz5Mf+T3u8bzwac203iYqAabNKkhllw9XXNK6oILoiGdQguSPh2Upg5LnWWeG1l32hTB\\\\\\\/kyTQ2GXXNAy\\\\\\\/nPZ\\\\\\\/3KWV9aPOWXkI8R\\\\\\\/6sVuuqKr17xwaHSife+YKB9pj3hfbaETYjDQu+GEwciA7YDwQrWTdIl07Tx1j9yN75dH0eluu7W7PfECzs1EbJjXrC3eIyCWZZtZxaq3Jg20Z75QJf2twJPj65woGBk8R43JeNkfUrVvig74txXzumH4nNH5iGHXKsKci6vF8VOH5V7L2MuIU5XpOWkhThIIs+uVuuhzzRnejwqSAnr3kxK47+WVU\\\\\\\/RV2Ds7aKch7iVXYZETLKaRDRvF3Q3gBKT9FrPOFJzVt\\\\\\\/ncqWcPu2vfAi4WgFArAiwG+LPIDyd1DFyvKzT2IGl7KUZdgVxS4txGrRBCFjuZK4V8M39wvmZ1SHA7GfVNHpDESFitfsJhCTcEBnnXcVcgzrvttXnRtjfBhVdPNYxW5D+8iVA1TZDggBor\\\\\\\/vrTLicMdtBbNyXhpAtciOjJjJeOR4OagRIsM\\\\\\\/OxvMdogrV9SgjNERo9VRYbItclM8vYMlGtv9kYjM5UnQUmUbXgk3ycBYZlYMA06oekQGVJsiv\\\\\\\/csPLy3djXmdtKlYln5wPbdFH68pOmk0g+jiIxs9V0O6qDkfcJSLr10qjOQKbYjJnAwKf7\\\\\\\/JX4TRKffQsm2N9BtIJRkcWRIejrKmo1Z\\\\\\\/vE68CCHDmyx3i79wVSWiXHwC62YlIe2Gmv6P4Z4q5ie0ttytFvlD1XrDc+4P+HZrJSUALWAu4Sxuhpe798PV147NxxLgzDTj3hyhR5UYREa0cobaVtPvUc\\\\\\\/df47\\\\\\\/0Wp+3urh0gWtn43sTiCnXS49jEtALVUZe1WW3Byk+Mp8h56iSw2ROjSZz\\\\\\\/2WzwXJVaYGm99CTsa02HYW3QMwnd9JwKUFXF9ZJON4BX2T0l43NeGHwtzMVDqlpbxMkWgrEGXLZiy+wA89YS7Kl3XK7QjSz01lOG6HbQ9LVk2ImQW6CaiU86Rhm219yfo9rQx1gBUnzcTzumctokFau7pFrmMAf4IrhOtuk1zQusHDiFfcx3L2CA1zd0\\\\\\\/6\\\\\\\/+YVvfBOZt7qTHUzrmvg2BcJQLNxZHFjhhJdyYbH\\\\\\\/BmblQQ1Dl7+dTtEW84ULwDWp\\\\\\\/luWf6dewlipBRdbscNWZGpLFcVjBuisnvfrOB4zCDGBx2bZuTIXtGU9eSFp9rPkHm84esDZd8wod2SzBhxw3sIs9+deYFzV80DJxKGKMn7ZT1DSvJuVNQmJ6ZprWA82cOPDRpJa\\\\\\\/AvaEbCeNKNdwyPDzsfHWyucb37V5ZKbuqItYiM9k0IUEFvIzOyDIlaZTIlZLp\\\\\\\/3PiSCDHKxvWwqHNFr1wdXOL+40+GXUkAPIl+ZKv0eos\\\\\\\/bPsGp2lDqrgKsusDTIekgQ32ma8NQwrMDcpBvfWAEojZssZWbh2b5epQCSLPV4aCqo6QQSGRZwKjNk7nfWuFLcHXV2O2ttfLcTvnm+TAqLGlb3CNxSt3gD2CGykU4zBcy2RVToW7sP2gaj\\\\\\\/QuIzMiGhiRXcYMh29YeztKXRzdXCKgPoTQN74X\\\\\\\/4jEe0C31DKbHAvxNV8tsuq4BrGEmHM8fiWFsDtYO0++tZi41AiaBvHJi4y1evVvk\\\\\\\/eX8qbaZBMuXkdGl0oPkRLnE38M9BH2sIst0FrNcuZY87bolMOhvPDdsyrdLET0yYzl1lLkKiym+ZMLfIWcA8arML76aLZAl3Er9eOClwxmdKpvUd\\\\\\\/\\\\\\\/K6jkQwhHX7qLKE4SndxPKwcP4zwdlqWkioSjkzH6KOL68Y7QduQdBVQkDOfbd5\\\\\\\/gREpJvVkn3JymwnpDRTKK1EtfqdLUrsmkooFqX3Z+7eelFC7HDJFYoxe4cst5PN8DWygQxzhtj6RMHpIhrWQBHTCzCsUEbQYJgrncSuuZOZ0jy1CIL+hjPabEqa+BylaZWuygqY\\\\\\\/gdN4NiuDMEyx9BlbvglZTiDlAb9MGqbnMCPWy8ohYyFcg9RnM2kMlOxc1N6eXLGHY1NabCsBNaqXCsiiI2E7NsbAnjVMs9fVmGC4W6dVycHcBNTnR4h\\\\\\\/\\\\\\\/xT+lJpjLjvvONVTWebd0cYLORggBFy8PI5VHe5eijyrpRzfbm8f2zewxdcACD3s72MhPFhr1eWs2kSHePYiJRSkZCvd33deGvdX9Kxm8bLbnWcTXy+0LUg3nl4nbbEYpcpIF75H4zbq4kGlbZ5aCQJ7lpObEElX2KI71mJXW780RE2J1doBprKPuj8jeF07mgVDS6sWvJutdSZ17cf23a5nVQw4pQ4yjD+6FM2+49p+UlvMMJTNAHUqWjUMvIRa70ukgG9ULYKvXeeDplcDcqdFZHxgrIb9AZFxq3KwiM8nO4mNLBBC0cG7pW6+Xo05qtpBE2cUWvhJmAK7lDF7FDLVOr5P82jnHJz7DrbXfCD\\\\\\\/QpAjTx0KWJolECPXTNOStoButERLQ89pHH3yMemDzO7HV8vcPQCZB1gzxN2XWBmR7r1QAr0nk9fg62BIf8rc0eLNEyC2XzeO7rwWzB9rAHeVpXCZOXpwJovRYbFCK+kQzG5Degp\\\\\\\/dz66crk11b3KjDIe2z5pFdVNeOp00d6TI2qm1PZaKMk8DinDEKQc0JNWpfJfOpB6TOuHcKh7I1D0uwAaIGwEXGQwxrxK\\\\\\\/vJ13F8ZBdtMw2h8VIdgSgkLpnTq94ZzCbxLtx5XwxOG\\\\\\\/Mkf\\\\\\\/eswsghtwmxmpRmSRc6J6ByfdnWFeYGOmR1YFF\\\\\\\/yy9WWbMWIFmYd9oKuHQxvpRlsRdy2d+cqfFOuGDrUeZQJg5rODNw+j4+\\\\\\\/zLMDh0E05wa81vCLPtOqPcC0DFdF5bCNzUfKJdiZN7pbrQ3Te9FheQter6YFM2irSt3pek2VcEzjg+pskFW7H1ztZTkfg8TRxTyk7W5gTfBZj6FaCQN7GKfNNG5cRV9qXUuDozZTzkAw3MvaO8I6f\\\\\\\/vKX6H7b96sG16pq9IZtZoQJcyaN+PbphZ8at2PexxZLSOQSOvRzcRD+lqsbRH+slSNb8qnB4r3mzn1CTrJwDkmmo57SMbcxEpw9dvZuSvXO+IPX+Guw\\\\\\\/ZXcAjSt7nTdYZca+u7tajTI5JNKmb26qbnRxrnSeOXbBJ+N0r4JOv2AkFnB0JutCoz0yn+8EtGNyc3nTmEkV+MRn3vRq+irxfnF1n\\\\\\\/4BKzsw4flklrwrlc8eGLN3b8qmFOT1\\\\\\\/H9hsGzHNFE94T989sbYRq98Pw1kOYpJeJ85yXIm13t3mcYQ7FV6t36HYT3Nhdmv5PIdiCT7zPibvpogM\\\\\\\/By+UkG2DKNyzERS9P5jeaWzdNariO0BAJYJerPfz4iq\\\\\\\/eG0Sk+L265S0zEJ37RZShnJMlfSq5QblEag3F6af+xrrh0Zdg+XFcAjr7v1TOUKkeojKdtEsPci0YJnIibrrV87Wjj0IiWfNQyfqiWVhEnEJsGhPLy+1hu7RoFEQPaaUQMxa6TcnLW\\\\\\\/+FiQvG0kCfcDIQAr\\\\\\\/cy1Kb2r7BiE\\\\\\\/0xg22b8EyNke2vozmqq9I\\\\\\\/hlRJ6sstUbKlniCqdN95gHs9kyP+EfzzOq3yPv9Dk\\\\\\\/xa1bt1lv6Pl2CRoL\\\\\\\/zj2kk8tgflUjuwb8gZGBSeEtbufiZzeI2dsKrJJ6CIAGonKoMDk5ZpoYgbRJzgy0rYZwOZqHZDU2Zqw6\\\\\\\/vLEqJiVnDZCipS8+bdYQJGEDm4lEEBdBIJdHblU0LU611Vp4RtcZ1a\\\\\\\/7jSb+EjbNdB51SKXtWZgVv11XvMYxwbhm12qfKI++9TmZucq8F9P8glU1JgIJEDmKYwdsdvRB3XOc7hqHq2kXMmY4\\\\\\\/ex4r0xK9Z5TWWK\\\\\\\/2SJkiJN6LAhREppZgONMGlthEAw0kOjH48duR30IkH5FOAKO+go2MN6z5\\\\\\\/huVXc9wWtpToFK\\\\\\\/eOfzEZoDyF1rqQszueubiOHd1SzlZi30TY+5zXBC7kreQTQFKq5dCsE5L+8a\\\\\\\/kGOk\\\\\\\/Xr1srpPis3HNXqw1hFbMqbRS+LRE8mrpuOcBuriGeljWyxWY7GGVVaOYLhYXsgpxehPKZ8HkyyTodrx5I+VFW1YTePrUpCECqyXHhjve2CibDlPMHGb\\\\\\\/u2Y+dQFplrpjMGUo0jRY9xstxhORcO4sZMnsauB26exKCqNANIrCnqUpHwyYG\\\\\\\/SgBbNJrJ0uprtZsrTZoVgg3blGbxrU1Ye6\\\\\\\/2MU9h+hJmH99cxAddYsM4mkLY3S3lA1rNoblF0emguVuv3d6OqYr3TvcDrzFiP304o\\\\\\\/XD\\\\\\\/OfEsnSUco8OmOP7jaOxXkp33ZPfI0ApGTma6p8q2v1WuHJ9UMxp7LqQPibFQ0FZpWpTzaunSsXXomRN8v2Q7iWgTQj0DaNnlexvSibgRIT0ZdDxsUd9EEAq8EhZEVyoM89z+e64pOAoFz7uaAnwJiP7xvp35lgapKD7\\\\\\\/Tqe8N9SiQNcqqHEOcW4RvamSb5VNP6NN5ap3Wl69VZSyWBsCQCPHm4Gq0AVf507\\\\\\\/ptp3TKlP1ZYv4HMdV7zmKw\\\\\\\/+z6ghewGtYViGxI0nYh9r6k\\\\\\\/Hku7fjKdL2jtCMGRIfeMHEcrRpP\\\\\\\/c+fG\\\\\\\/jBxOOc+QLy\\\\\\\/IsjJnPCbsbe9cswN\\\\\\\/JTvngUPVOSICNVgquityqeo68zfPnh59D8JXrT5jHdpmUpxVgyHMmFvzxkqJPVMTdlymias72DOOap1VlaEDqW7DDCYFlCkm2PUJ1nQMN4OvHS2clqvLqDb\\\\\\\/mXfmBTSJHpbPFW6fCf93JrWV6lABArMC7qQ3FtM\\\\\\\/r9OMtMXXhQSu8Rej0TaHO3KxaDchjAkMLYGS4KoLzi0x07ljzzlaH\\\\\\\/gv3q5Cjt2mcSuJ9ZtKhW1IAnLWkDeOXCW4TLkhiFzCghJf4kRRaFZ6AvbbVR366M4QSDC29BfSDP3pVml3q4u2bgheILjAPhLvNiDxonxMTcrfiubyc5tr49KH5MSrOdF7GXoQK0Ph7S7dceXK7G6eL5LUVAOIHZkvphMAsK9Qd61rSV9qvigKxRqFzLL0mop0H+GAo6rrtfTn77XvfXb6\\\\\\\/Y8V3YfSWpNBMhATg43eeKbRVsWyja5BWUkGosDpUNGAWxvzVN+0xKDu4DiBSnEad2I6p4d1gkY2nypc\\\\\\\/D88pQ+i9y3jsaStePTRaGPlGPaEOZMDaFisarsEHdiVFDEZX90K5nqEcljemoQBpI6n06FpYstwScObnjxo4qIFK\\\\\\\/DYD4IvVsRZKnoCrTQZqospfSYZuKCaDVSIRT9Y8xMDbz1YSF7qBwPx5LnBUZEG\\\\\\\/LidZWlUSEZVm2IxfgwDNXbxcNXGugLfEe5j+WH5UAnOcU25LaP4\\\\\\\/2XBgH4Wi8kTzPK6866BpcE9vfvEpzW2Z5w\\\\\\\/vmD8Qu1PpHdqMNgAH23xRQy8hIArKk6CBjDzXS6p2CSKEWRegEr6dT6rrgUQe3wMJLHNQ61nL5jNzgYPdzuI6ioAggvqA+jAfbCvP1Vd0TUjScgOjSkV7unuSsuUCjD8pw6+SZ\\\\\\\/FPEu6TfARBNoTdwsDSLYg3iPykkB75SpYq9p+IM\\\\\\\/GpZnCvNvj154Wr3ezG3Sz3q2waloxD5LTNg2XOKejL4yNJxu1K67nwX+9Dcx9HU5wWp5TaK3vNE92stoCv94u44pyk5+hcn\\\\\\\/qbToUMLH047DtpcolUBldnsJTJSVV6IPd+kZ6PV13fHoKSd7YyCRHeWqEis+hQlv3EvC40AZ4LiIJ4UOAxkbAFJ6bJD31bLhT66TIZtS7bMEMTnAfVkl48u2GPNUn7ERNdaxc5OYesWu8CWF+hWsXkebci3Fd79RMopHv1j\\\\\\\/L8I1CXRdgnPhc2I+Xe0LcH1oESj5nGMiUqtYJ5PzdwN1HV+PR9N07RXuzGJ1kKX2x9J3J02vjkyM9U4FU4P8w2eNi0aaXsG9dh6aryuH1eAnT4aV8\\\\\\\/FuAqroFyfSFIF5g\\\\\\\/85e2IZEleHJ0uuNeQ2yK9j53+bKk+GVnNfWLgm0Z2xhjZGeemkeb8KETYpmwZQdsmlVrrvJCFOpCRgGsfQYfysvndVBqMaj3dKiDmhuAQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YwECFASzkjicdF6UiAqJIQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f2fa0e0269cdcb66b4c98cfce0d7b78970e347879a2b4481f28361c2c6c0495\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kYO+uBYfGLu5kaSbp4sZUMMWNFvA2tpTbR9buJd7mhstocbHVB7v+oOYRkw1PY4fc\\\\\\\/Ce4S7q78ZXjPclGuSGN5yOWmK0xNfGJ\\\\\\\/diq8kO829nAHWfl2qoOUvkx1cbB\\\\\\\/2WeqzJsOVEgtaCJv7j\\\\\\\/VBHFLJEIOeCVlA8bHbINe8781wYljYp9OxNfyvwobbJPJcbYocGECiw\\\\\\\/GlvbmNkfLgnLb4RapURgbxf4GQBl7YFxlM3S6l3v2ZdF95iEuAmk+kbkuxDu6Q29alBVoHlzSxNxFplqS6o7dNuuEaRsOCKpQFxg5TnNru6SAagJgy0OY7YM9ujiaQdozGn3SHxOU+4uqL7MrEwxUkmzwinK5pEI0Uy3N25bl1x1igKAUhky4btCxtAlT+TOUVaX6lP6bDch4WqIx7Rqtm0VjVvxQ07eacq2VaewipY4SKT2zlbu4\\\\\\\/YKT4BwTEf2i6XTp7dQM0D297oae9Im9hsN9Zf53OqUZpEGC6h\\\\\\\/oyk1GKN3s+2o2VT0JxYikDhMJWhe8y5DIWV9qBmx2p\\\\\\\/MmNgMl6mmGpHpPT98wsBvgsF8HmrNPAqN7KKeR\\\\\\\/\\\\\\\/B5cIqLyf9BWDu3PJ\\\\\\\/WBf\\\\\\\/kixPS690OWwMG77HQ2RxN4Igx+sICuAtrOI6PbN5JexzxmGHdVk+zc5\\\\\\\/C+MIwU\\\\\\\/Ig5fQz5pdJzI8UmM9AVyW3XH2urf9vUq+AWU22t8wzgLk7TMiJbuMtRYxwU17KlHkbWefnEKBPShuTAgfRweYllx1FMHwkJbUuMAmYuqWxjWUIvIxSsPfzRFKJXo8jGKghuUvZ2KgeUcWaqDXI6MATaAYXjmC5762DbWJI9srQIAdBrER8b0ocWc9vSksxNwyMi7ChnQvTgz4qosBmn9zfPxYPVtNcy3hjrkV6OYyh\\\\\\\/ScbySnOGuJ+9rEpSPt1xOgHpMVA0D4pMbiUSuHdvpD3tMVu95XT67DPcv3A62Gh8P0478vsP78Krt7KorcvPB0ixkKmfGOhhGFQB0uaYDHhTIsZPUj4Vj7WSLmJVKGITpVyuqkAfWe6Dxc+4\\\\\\\/6LcLUa8241DmoPgup3PxrbZgcxMcdlqLWec2h0P3SeUQ0qE+hTLctK\\\\\\\/nIKMD6VSUaVC+\\\\\\\/v4++\\\\\\\/76HzT5Syutllxb5IPQBBJLO2Yj1Mr8Su2nTOSDnSzXAl6A4QV6EP7Wja+JTAaxj3xcKkiyQkkhINe9cS3cmWPQ3ObsG839CfZj6bx3P+oQYXBsalG61zqmFSDz2zNDa2MP4Xc0AnGDKuotiyJV1Wli4IKUNvWQzo4bTEm8DL211qe5CSyaBIN0Hf0OTNS6o9+CT49Ab2PhgEAcnHKYjiDJKrk1Mj7mVAURC+GHsOU0t2OFxy7LvYistmbFpJgm4J4JfBzZAqNw+\\\\\\\/VbW5cIfWq5S579wBXthGY6SAO52yEfs6\\\\\\\/OFoxggklcvuKkgkZld8\\\\\\\/6brTTjmvGrQt9bh4PPe5WjZJPojcrj5fN3O3w6EOxRiKItNoxCdZlR99iNzy+fhm0quXsZRintazsSU43e0G2dQNq6v0y4QANg8LhE\\\\\\\/6ljiYYXxEd7pF\\\\\\\/XF7CBiVs1Pfwo8THiewDd551uo9cIY3TN8MwC+gIlYOjdA\\\\\\\/ujB1zGtYFN0\\\\\\\/r50BiFdwKBJM8XztfwGx1R6VdDr+ZMhNKypyGHeH2HJOp91myZC2RiUNjoEkaqfAAqopm9lMOOqCs8rXS0zC0ABQuU5VMDJirBGU3ZlJGPzIgkjHvvP7mG1k36KfjlrHcMru30swg3IYSaliC0+CtoV7IE21t2AnZze3IAfpE59RyCWubVRpShM9dmVrsaBgmLV2WlM2gqBWos6ldhjsqgYYKfkn+K39lKqbqGhBcd52FD1lrqNqbi0vAvlFfi4pMJ8BmKmtnLZjtgIKeQ4Rz68oiYQmc12eMkLFYAMAWsceIGlqgmNFZ3D\\\\\\\/u6YcCslz2ynWEneSq+fcsE429E+hYNEz9Qftr8VbKTOu8kEUL89DuhiEGGgdShqJcslKsIP9PC69siA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BTyfdz2dB+ZZIr2hFWbH6Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221916e6c72e35ffa72823cab6e75f30f052f7c779de979b551473918bffc99079\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459534 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1553" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H7U4vAU7k7mv4Nz7F94l48mzVsJPwqph_6iH88ZxKNk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1553" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Xbn-dWGzwcaJAUFyphB2DNbJtVQW_xHqFDfi9JNmI_0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/9/1545" title="Mitochondrial DNA with a Large-Scale Deletion Causes Two Distinct Mitochondrial Disease Phenotypes in Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/9/1573" title="Maternal Loss of miRNAs Leads to Increased Variance in Primordial Germ Cell Numbers in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459534" id="node459534" data-pisa="ggg;3/9/1553" data-pisa-master="ggg;g3.113.007500" data-apath="/ggg/3/9/1553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Pedigree Relationship from Genome Sharing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">William G. Hill</span> and <span class="highwire-citation-author" data-delta="1">Ian M. S. White</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1553-1571;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007500 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459534" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">William G. Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BG.%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20WG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BG.%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:w.g.hill@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">w.g.hill@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ian M. S. White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BM.%2BS.%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20IM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BM.%2BS.%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/9/1553" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459534" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1553.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459534" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1553.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459534" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/9/1553.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459534" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/9/1553.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459534" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Determination of degree of relationship traditionally has been undertaken using genotypic data on individual loci, typically assumed to be independent. With dense marker data as now available, it is possible to identify the regions of the genome shared identical by descent (ibd). This information can be used to determine pedigree relationship (<strong>R</strong>), <em>e.g.</em>, cousins <em>vs.</em> second cousins, and also to distinguish pedigrees that have the same Wright’s relationship (<em>R</em>) such as half-sibs and uncle–nephew. We use simulation to investigate the accuracy with which pedigree relationship can be inferred from genome sharing for uniparental relatives (a common ancestor on only one side of their pedigree), specifically the number, position (whether at chromosome ends), and length of shared regions ibd on each chromosome. Moments of the distribution of the likelihood ratio (including its expectation, the Kullback-Leibler distance) for alternative relationships are estimated for model human genomes, with the ratio of the mean to the SD of the likelihood ratio providing a useful reference point. Two relationships differing in <em>R</em> can be readily distinguished provided at least one has high <em>R</em>, <em>e.g.</em>, approximately 98.5% correct assignment of cousins and half-cousins, but only approximately 75% for second cousins once removed and third cousins. Two relationships with the same <em>R</em> can be distinguished only if <em>R</em> is high, <em>e.g.</em>, half-sibs and uncle–nephew, with probability of correct assignment being approximately 5/6.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/relationship" class="hw-term hw-article-keyword hw-article-keyword-relationship" rel="nofollow">relationship</a></li><li class="kwd"><a href="/keyword/identity-descent" class="hw-term hw-article-keyword hw-article-keyword-identity-by-descent" rel="nofollow">identity-by-descent</a></li><li class="kwd"><a href="/keyword/genomic-identity" class="hw-term hw-article-keyword hw-article-keyword-genomic-identity" rel="nofollow">genomic identity</a></li><li class="kwd"><a href="/keyword/likelihood" class="hw-term hw-article-keyword hw-article-keyword-likelihood" rel="nofollow">likelihood</a></li></ul><p id="p-4">Relatives carry individual genes and also genomic regions identical by descent (ibd). In many situations in human, natural, or agricultural populations, it is important to identify relatives and, if possible, degree of relationship using this information. Traditionally, methods of identifying relatives have used information regarding identity in state (ibs) of individual genes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Weir 1996</a>), with increasingly dense markers enabling increasingly high precision, using methods such as CERVUS (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Marshall <em>et al.</em> 1998</a>), components of PLINK (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Purcell <em>et al.</em> 2007</a>), or COANCESTRY (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wang 2011</a>).</p><p id="p-5">Traditionally, establishing relationships does not use information regarding location in the genome, and statistical properties are often based on assuming unlinked markers. Linkage information can be incorporated, however, by using the linkage map and taking into account the Markovian nature of the ibd process underlying the genotypes of relatives at linked loci (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Epstein <em>et al.</em> 2000</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">McPeek and Sun 2000</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kyriazopoulou-Panagiotopoulou <em>et al.</em> 2011</a>) using methods such as RELPAIR (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Epstein <em>et al.</em> 2000</a>). Regions of the genome that are shared ibd can be established using identity in state (<em>e.g.</em>, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2002</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Roberson and Pevsner 2009</a>) using programs such as MERLIN (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2002</a>).</p><p id="p-6">Alternatively, distantly related individuals can be identified from multilocus sharing of even quite small regions of the genome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2011</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2012</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">2013</a>). If it is known that two individuals are related, then the allelic information adds little on regions already clearly shared ibd as determined by common sequence (except perhaps on ibs of two very-low-frequency genes). Further, the use of information on shared regions rather than just individual loci allows, at least in principle, discrimination between relationships with the same Wright’s relationship <em>R</em> but different pedigree <strong>R</strong>, <em>e.g.</em>, uncle–nephew and half-sib, both of which have <em>R</em> = 0.25. <em>R</em> used here is strictly Wright’s <em>numerator</em> relationship, which equals twice the kinship (coancestry), but it is the same as Wright’s relationship in the absence of inbreeding. <strong>R</strong> defines the pedigree (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Further, the actual proportion of the genome shared can, by chance, be higher by more distant (<em>e.g.</em>, second cousins: <em>R</em> = 1/32) than closer relatives (<em>e.g.</em>, first cousins once removed: <em>R</em> = 1/16). The proportion of overlap of the distribution of actual relationship increases as the relationship of each of the pairs becomes more distant (<em>e.g.</em>, <em>R</em> = 1/64 <em>vs.</em> 1/32) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hill and Weir 2011</a>), further increasing the problem of determining the pedigree <strong>R</strong>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463137/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463137/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Pedigree relationships (R), Wright’s coefficient of relationship (<em>R</em>), and abbreviations used</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">The pedigree relationship may be needed in a number of situations. The estate of an individual who dies intestate may by law have to be divided among his or her closest relatives. Courts would assume this to be defined by pedigree. Another situation would be in identification of individuals in forensic cases, for example, in identifying a body or a body part in a disaster zone, or in familial searching for relatives of an offender already in the database (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Rohlfs <em>et al.</em> 2012</a>). In studies of natural populations, pedigree construction is an important component in determining breeding structure and estimation of genetic parameters (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blouin 2003</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Pemberton 2008</a>).</p><p id="p-12">Detection of genomic regions for which there is biparental sharing, <em>i.e.</em>, individuals with ibd genotype at each diploid locus due to relationship through both parents (<em>e.g.</em>, full-sibs or double first cousins), is quite straightforward because there is ibs at each locus in the region. We consider here only the much more common situation of uniparental sharing, in which case <em>R</em> is half the probability relatives share one allele ibd at a locus, or half the expected proportion of uniparental genome shared.</p><p id="p-13">Therefore, a quantitative description of the number, position, and length of shared regions is all the information we can have about relationship of a pair of individuals in the absence of pedigree information, and this sets an upper limit to what we can detect. Our objective is to find what this limit is for different alternative pedigrees. Therefore, as a reference point, we work on the premise that we have precise estimates of these quantities but later consider this assumption. We also assume there are no confounding factors, such as inbreeding of the common ancestor or relationships among other ancestors of the pair. Data on gene frequencies and genotypes at individual loci then add no further information.</p><p id="p-14">We focus on identifying specified pedigree relationship from actual or realized genomic sharing, for example, whether a pair of individuals are related either as second cousins or as cousins once removed, in each case assuming there is uniparental sharing. Such comparisons can be undertaken based on a likelihood ratio, although the appropriate test or discrimination depends on the questions to be answered, such as the following. Which of two or more alternative relationships is the most probable? How sure are we? What relationships can we exclude?</p><p id="p-15">The variation in the total length can be computed (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hill and Weir 2011</a>) and there are also various methods and approximations for computing the numbers and distribution of the lengths of shared regions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fisher 1954</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Donnelly 1983</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Stam 1980</a>). Recently, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huff <em>et al.</em> (2011)</a> have proposed methods to identify whether pairs of individuals taken from the population are related more closely, <em>e.g.</em>, as second cousins, than background relationship among all population members from distant relationships in a finite closed population.</p><p id="p-16">There is no theory available that enables prediction of the numbers and distribution of shared segments exactly for arbitrary relationships. Therefore, we use simulation to generate the required probability distributions. There are, however, approximations for some of these distributions available: for example, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Huff <em>et al.</em> (2011)</a> assumed a Poisson distribution of number and exponential distribution of shared regions (<em>i.e.</em>, independence), and we also investigated their accuracy. We conclude with a discussion on inference. The primary objective was to set the theoretical framework and compute what can be achieved rather than focus on applications <em>per se</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Simulation</h3><p id="p-17">The simulation program was used previously to check theoretical results for the variance of the length of shared regions on a chromosome (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Hill and Weir 2011</a>), which in turn provided a check on the program itself. Simulations were undertaken for a single chromosome, for example, of length <em>l</em> Morgans, in each independent replicate. There was assumed to be a uniform recombination rate and no interference, <em>i.e.</em>, corresponding to a Haldane mapping function. The number of recombination events was sampled from a Poisson distribution and their positions were sampled as real valued numbers independently from the uniform distribution. All regions of ancestral chromosomes were labeled by the same integer value, <em>e.g.</em>, 1, 2, and so on. Hence, a chromosome of a descendant was defined by the position (<em>π</em>) of each of the <em>n</em> − 1 recombination events, <em>e.g.</em>, 0 = <em>π</em><sub>0</sub> &lt; <em>π</em><sub>1</sub>, <em>π</em><sub>2</sub>, …, <em>π<sub>n</sub></em><sub>−1</sub> &lt; <em>π<sub>n</sub></em> = 1, defining the <em>n</em> chromosomal regions labeled <em>h</em><sub>1</sub>, …, <em>h<sub>n</sub></em>. Then, for example, <em>n</em> = 4, <em>π</em><sub>1</sub> = 0.1256, <em>π</em><sub>2</sub> = 0.5701, <em>π</em><sub>3</sub> = 0.9012, and <em>h</em><sub>1</sub> = 1, <em>h</em><sub>2</sub> = 2¸ <em>h</em><sub>3</sub> = 1, <em>h</em><sub>4</sub> = 3 denote a chromosome for which the first region (from 0.0 to 0.1256) and third region (from 0.5701 to 0.9012) were derived from ancestor 1, the second was derived from ancestor 2, and the third was derived from ancestor 3, and thus ibd for that genomic region with these respective ancestors. This does not imply that the parent has a chromosome with exactly that haplotype, but that a gamete could be formed from it that does, <em>i.e.</em>, the shared region may span grandpaternal and grandmaternal origins. Hence, for a second descendent of the same individuals with, for example, <em>n</em> = 2 and <em>π</em><sub>1</sub> = 0.3659, <em>h</em><sub>1</sub> = 2, <em>h</em><sub>2</sub> =1, there is sharing between the two descendents in two regions, between 0.1256 and 0.3659 from ancestor 2 and between 0.5701 and 0.9012 from ancestor 1, <em>i.e.</em>, internal regions of length 0.2403 and 0.3311, respectively, with a total proportion of 0.5714.</p><p id="p-18">To obtain the results presented here, 100,000 or more independent replicates were performed. For each replicate the sharing among different kinds of relatives was computed, so for a founder full-sib family, the degree of sharing of, for example, uncle and nephew (or aunt and nephew, etc, because only autosomes were simulated), great-uncle and great-nephew, and cousins of degree up to third cousins were sampled successively. Although this induced sampling correlations, these were trivial because replicates were numerous and independent. Simulations were performed independently for chromosomes of different length and for three different founder relationships: linear descendants, full-sib–based, and half-sib–based (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-3" class="subsection"><h3>Distribution of shared segments</h3><div id="sec-4" class="subsection"><h4>Numbers of shared segments:</h4><p id="p-19">We provide examples to illustrate the kind of data available from the simulation for a map length of an “average” human chromosome of 1.632 M (based on <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kong <em>et al.</em> 2004</a>). <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the distribution of numbers of shared segments (<em>n</em><sub>s</sub>) for a range of relationships from a full-sib base and for a more limited number of half-sib–derived and lineal relationships. For this length of chromosome there is a less than 1% chance that uncle and nephew share no genome and approximately 15%, 35%, and 31% probability that they share 1, 2, and 3 regions, respectively. For half-sibs, who have the same Wright’s relationship (1/4) as uncle–nephew, the probabilities are 2%, 25%, 41%, and 24%, respectively. Of course, more distant relatives share fewer and smaller regions. For longer chromosomes (in terms of map length or expected number of recombinations) than shown in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, the expected number of shared regions increases and length of individual segments decreases.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463183/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463183/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Examples from 100,000 simulated replicates of the distribution of the numbers (<em>n</em><sub>s</sub>) of genomic segments shared by relatives on a chromosome of 1.632 M</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h4>Positions of shared segments:</h4><p id="p-21">Information also can be obtained from position of the shared regions, specifically whether they include the chromosome ends. Examples of the distribution of shared regions on the chromosome according to their position, specifically whether they include both, one, or no ends of the chromosome (<em>p</em><sub>s</sub> = 2, 1, 0, respectively), are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. A single region sharing both ends rarely occurs unless the relationship is close, and the proportion sharing at neither end increases as the relationship becomes more distant. Half-sibs are more likely to share regions including both chromosome ends than are uncle and nephew.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463139/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463139/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Examples using simulations as in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> of the distribution of numbers (<em>n</em><sub>s</sub>), positions (<em>p</em><sub>s</sub>), mean, and SD of total length of genomic regions shared by relatives on a chromosome of 1.632 M</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h4>Lengths of shared segments:</h4><p id="p-24">The expected proportion of genome shared (<em>i.e.</em>, 2<em>R</em>) is of course the same as the overall length of shared regions expressed as a proportion of the genome length, but the distribution of the lengths of the total and of individual shared segments depends on the pedigree <strong>R</strong>. Examples are also given in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> for the mean and SD of the total length actually shared, expressed as a proportion of the chromosome length <em>l</em> = 1.632 M, as a function of whether the shared regions include zero, one, or two chromosome ends. A special case is when <em>n</em><sub>s</sub> = 1 and <em>p</em><sub>s</sub> = 2, when the length is invariant because the whole chromosome is shared.</p></div></div><div id="sec-7" class="subsection"><h3>Summary of simulated statistics</h3><p id="p-25">Because the numbers (<em>n</em><sub>s</sub>) and positions (<em>p</em><sub>s</sub>) are discrete valued variables, to facilitate subsequent analysis total length shared on the chromosome also was summarized in discrete values, namely as the number of tenths of the chromosome shared (<em>t</em><sub>s</sub>): if <em>x</em> (&gt;0) is the length shared, then for <em>t</em><sub>s</sub> = 1: 0 &lt; <em>x</em> ≤ 0.1, <em>t</em><sub>s</sub> <em>=</em> 2: 0.1 &lt; <em>x</em> ≤ 0.2; …; <em>t</em><sub>s</sub> = 10: 0.9 &lt; <em>x</em> ≤ 1.0. The distribution of the length of individual shared segments conditional on the numbers, positions, and total length shared on each chromosome was not included in subsequent analyses because it contains no additional information. For example, if there are two shared segments of total length <em>x</em>, then the relative lengths <em>y</em> and <em>x</em> – <em>y</em> tell us nothing about the numbers of generations apart. Although shown by simulation, on reflection it is obvious because the distribution is uniform.</p><p id="p-26">To simulate the 22 human autosomes, map lengths were simplified into five classes, based on the data of <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kong <em>et al.</em> (2004)</a>, and previously were used for illustration (Figure 5 of <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Hill and Weir, 2011</a>), namely two chromosomes of 0.75 M, eight chromosomes of 1.25 M, six chromosomes of 1.75 M, four chromosomes of 2.1 M, and two chromosomes of 2.75 M, totaling 35.9 M. Simulation also was undertaken assuming 22 chromosomes each of 1.632 M, <em>i.e.</em>, with the same average length as in the model using five lengths. As shown later, there is little difference in predictions of discriminating ability between the five-length and one-length models, so further subdivision of chromosome lengths to more closely match those for humans for analysis would have little impact on calculations or conclusions. This does not, however, imply that individual lengths should be ignored in analyses of real data.</p><p id="p-27">As inferred from <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Hill and Weir (2011)</a>, from variances of actual relationship and also from simulations, for half-sib–based relationships the distribution of shared regions (<em>n</em><sub>S</sub>, <em>p</em><sub>S</sub>, and <em>t</em><sub>S</sub>) depends only on Wright’s numerator relationship <em>R</em>. For example, it is the same for half-cousins and half-great-uncle–great-nephew relationships (both <em>R</em> = 1/16), and for half second cousins and half-cousins twice removed (<em>R</em> = 1/64). Similarly, for full-sib–based relationships, the distribution is the same for second cousins and first cousins twice removed (<em>R</em> = 1/32), but is not the same for great-uncle–great-nephew and cousins (<em>R</em> = 1/8).</p></div><div id="sec-8" class="subsection"><h3>Likelihood ratios</h3><div id="sec-9" class="subsection"><h4>Computation:</h4><p id="p-28">Let <em>k</em> denote a specific realization {<em>n</em><sub>s</sub>, <em>p</em><sub>s</sub>, <em>t</em><sub>s</sub>} of genome sharing on a specified chromosome, and let <em>P</em><strong><sub>R</sub></strong>(<em>k</em>) denote the probability of this outcome dependent on the chromosome length and conditional on the relationship being <strong>R</strong> (<em>e.g.</em> half-sibs). If, for example, only information on <em>n</em><sub>s</sub> is used, then the realization is simply {<em>n</em><sub>s</sub>}. The contribution provided by the observation <em>k</em> to the log likelihood ratio <em>λ</em>(<strong>A : B)</strong> for relationships <strong>A</strong> and <strong>B</strong> is then log<em>P</em><strong><sub>A</sub></strong>(<em>k</em>) <strong>−</strong> log<em>P</em><strong><sub>B</sub></strong>(<em>k</em>) using the logarithm inter alia because it has better sampling properties. We use the simulation results to obtain these probabilities, computed simply as the proportion of replicates with the appropriate outcome. Thus, using only data on <em>n</em><sub>s</sub>, for example, and assuming three shared segments, then <em>λ</em>(UN : HS) ∼ ln(0.310/0.243) = 0.245. and <em>λ</em>(UN : GUGN) ∼ ln(1.91) = 0.645 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Because segregation over chromosomes is independent, the total log likelihood ratio is obtained by summing contributions to the log likelihood ratio from different chromosomes, using probabilities appropriate to the map length and realization for each chromosome. If there is previous information regarding the relationships from nongenetic data and these can be quantified, then Bayes theorem can be used straightforwardly to compute posterior probabilities of alternative relationships. Otherwise, application is context-dependent, and we discuss that subsequently.</p></div><div id="sec-10" class="subsection"><h4>Moments:</h4><p id="p-29">Although any testing is situation-specific, we can investigate the properties of the log likelihood ratio as a function of the data used and possible relationships to be compared. Thus, we consider its moments, specifically its mean and variance. If the real relationship is <strong>A</strong>, then the contribution to the mean from a single chromosome is as follows in equation 1:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1553/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1553/embed/mml-math-1.gif"/></noscript></span></span>and there is an equivalent formula for the variance. The overall mean and variance of <em>λ</em> are obtained by summing contributions over chromosomes. We also compute its skew and kurtosis.</p><p id="p-30">The mean <em>λ</em> is the (directed) Kullback-Leibler distance between the two distributions <em>P</em><sub><strong>A</strong></sub> and <em>P</em><sub><strong>B</strong></sub> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kullback and Leibler 1951</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burnham and Anderson 2001</a>). This “distance” is not symmetric, <em>i.e.</em>, in general, E<strong><sub>A</sub></strong>[<em>λ</em>(<strong>A</strong> : <strong>B</strong>)] ≠ E<strong><sub>B</sub></strong>[<em>λ</em>(<strong>B</strong> : <strong>A</strong>)]. Subsequently, we tabulate values over the correct distribution (<em>i.e.</em>, real relationship) such that they are positive.</p><p id="p-31">Because the numbers of shared segments and their positions are count data and because lengths shared were analyzed similarly as discrete variables, the numbers in each defined class <em>k</em> have a multinomial distribution with parameters estimated from the simulation results. In computing the moments of <em>λ</em>, the expected probabilities <em>P</em><strong><sub>R</sub></strong>(<em>k</em>) were assumed to have been estimated by simulation with negligible error. If the estimate from simulation of <em>P</em><strong><sub>A</sub></strong>(<em>k</em>) was not zero but that of <em>P</em><strong><sub>B</sub></strong>(<em>k</em>) was zero, then in computing the term <em>P</em><strong><sub>A</sub></strong>(<em>k</em>)[log<em>P</em><strong><sub>A</sub></strong>(<em>k</em>) <strong>–</strong> log<em>P</em><strong><sub>B</sub></strong>(<em>k</em>)<strong>],</strong> it was assumed that <em>P</em><strong><sub>B</sub></strong>(<em>k</em>) = 1/(2<em>N</em>), where <em>N</em> was the number of replicates simulated. This term becomes important only when the distributions differ greatly [in which case E(<em>λ</em>) is already large] and when expected numbers in cells become very small. To reduce errors such as this due to simulation, because data regarding numbers of segments itself included data regarding lengths, results given utilizing <em>n</em><sub>s,</sub> <em>p</em><sub>s</sub>, and <em>t</em><sub>s</sub> used all three for 1 ≤ <em>n</em><sub>s</sub> ≤ 4, but only <em>n</em><sub>s</sub> and <em>p</em><sub>s</sub> for <em>n</em><sub>s</sub> &gt; 4.</p><p id="p-32">The ability to discriminate between alternative relationships using the likelihood ratio depends on the distribution of <em>λ</em>, mainly on the relative sizes of its mean and SD, so we tabulate E(<em>λ</em>)/SD(<em>λ</em>). Because there is replication of observations across chromosomes, SDs were computed over the aggregate, and therefore might be regarded as standard errors, but we retain the SD notation. We also found that λ typically has close-to-normal form.</p></div></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Moments of log likelihood ratios</h3><div id="sec-13" class="subsection"><h4>Expectation:</h4><p id="p-33">Information available for contrasting relationships expressed as expected log likelihood ratios [E(<em>λ</em>), Kullback-Leibler distances] are provided in the upper part of <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> for a subset of relationships using the full simulated data for numbers (<em>n</em><sub>s</sub>), positions (<em>p</em><sub>s</sub>), and lengths (<em>t</em><sub>s</sub>) of shared segments. In these and subsequent tables, rows denote the real relationship and columns denote the hypothesized relationship. Values of E(<em>λ</em>) for all 19 relationships analyzed and incorporating successively more information are given in Appendix Table A1 (using <em>n</em><sub>s</sub> only), Table A2 (using <em>n</em><sub>s</sub> and <em>p</em><sub>s</sub>), and Table A3 (using <em>n</em><sub>s</sub>, <em>p<sub>s</sub></em>, and <em>t</em><sub>s</sub>, <em>i.e.</em>, as in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). In all these Tables, values were computed from simulation runs for each of the designated five map lengths (0.75, 1.25. 1.75, 2.10, and 2.75 M), each replicated 100,000 times, <em>i.e.</em>, as weighted averages over a total of 500,000 replicates.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463276/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463276/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Expected log likelihood ratio, E(<em>λ</em>) (upper), and its SD, SD(<em>λ</em>) (lower), for a subset of relationships using information on numbers (<em>n</em><sub>s</sub>), positions (<em>p</em><sub>s</sub>), and total lengths (<em>t</em><sub>s</sub>) of shared segments</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">It was seen that E(<em>λ</em>) is small when relationships are distant and of similar magnitude (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, upper part), <em>e.g.</em>, second cousins and half-cousins once removed (for both of which <em>R</em> = 1/32). Although Kullback-Leibler distances are not symmetric, the reciprocal cases here are usually close but not identical in value, so only half the pairs of assumed relationships are included in <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a> (but all are in the Appendix Tables). E(<em>λ</em>) is typically higher when the likelihood ratio is conditional on the higher relationship of the two, presumably because there is a wider distribution of numbers and lengths of segments shared among close relatives and therefore there is more information in the data.</p><p id="p-37">The increment in E(<em>λ</em>) by incorporating position and length can be substantial for comparisons involving quite closely related individuals (Appendix Table A1, Table A2, and Table A3). As they become distant, <em>e.g.</em>, half-cousins <em>vs.</em> third cousins, the absolute and proportional increase is small. First, few shared segments are at the ends of chromosomes and the coefficient of variation in length shared decreases as the number of segments shared increases.</p></div></div><div id="sec-14" class="subsection"><h3>Expectation <em>vs.</em> sampling error</h3><p id="p-38">SDs of <em>λ</em> values using all information (<em>n</em><sub>s</sub>, <em>p</em><sub>s</sub>, <em>t</em><sub>s</sub>) are given in <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> (lower) for a number of relationships. Examples of E(<em>λ</em>)/SD(<em>λ</em>) for two subsets of relationships, one including pairs of high relationships (1/16 ≤ <em>R</em> ≤ 1/4 in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>) and the other including pairs of more distant relationship (1/128 ≤ <em>R</em> ≤ 1/16 in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). Later, we discuss the interpretation of these values and show that the ratio is, at least approximately, a noncentrality parameter determining the probability of misassignment. Approximately, a value of 2.0 or more indicates a pair of relationships that can be distinguished with reasonable confidence. Full data fitting different amounts of information are given for SD(<em>λ</em>) in Appendix Table A4, Table A5, Table A6 and for E(<em>λ</em>)/SD(<em>λ</em>) in Appendix Table A7, Table A8, and Table A9. It is seen that SD(<em>λ</em>) tends to increase along with E(<em>λ</em>) as relationships become more different, <em>e.g.</em>, uncle–nephew <em>vs.</em> half-sib and <em>vs.</em> cousin (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>), and therefore E(<em>λ</em>)/SD(<em>λ</em>) diverges less rapidly than E(<em>λ</em>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463274/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463274/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Ratio of expected log likelihood ratio to its SD, E(<em>λ</em>)/SD(<em>λ</em>), using information on numbers, positions, and lengths of shared segments, as in <a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>: Sets of closely to moderately related pairs</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463192/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463192/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Ratio of expected log likelihood ratio to its SD, E(<em>λ</em>)/SD(<em>λ</em>), using information on numbers, positions, and lengths of shared segments as in <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>: Sets of more distantly related pairs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Contributions from segment position and length</h3><p id="p-41">The contributions of different components of the data to E(<em>λ</em>)/SD(<em>λ</em>) are illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for some of the relationships in <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a> and <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>. It shows the ratio fitting only numbers of shared segments and shows the increments in the ratio by fitting positions and then lengths. A high proportion [in some cases almost all the information as judged by E(<em>λ</em>)/SD(<em>λ</em>)] is contained in the number of shared segments. A little more is added by including position, but only for close relationships when chromosomes ends are likely to be shared (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). More information is obtained by incorporating length of chromosome shared, although not with a clearly defined pattern over relationships.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/9/1553/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative contribution of sources of information to expected log likelihood ratio relative to its SD. E(&#x3BB;)/SD(&#x3BB;): number of segments only (ns left), additional contribution from fitting position of segments (ps center), and additional contribution from fitting length of segments (ts, right). (Upper) Sets of closely related pairs for real relationship uncle&#x2013;nephew (UN) or cousin (C). (Lower) Sets of more distantly related pairs for cousins once removed (C1R), second cousins (2C), and second cousins once removed (2C1R)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-85653404" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cumulative contribution of sources of information to expected log likelihood ratio relative to its SD. E(&lt;em&gt;&#x3BB;&lt;/em&gt;)/SD(&lt;em&gt;&#x3BB;&lt;/em&gt;): number of segments only (&lt;em&gt;n&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt; left), additional contribution from fitting position of segments (&lt;em&gt;p&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt; center), and additional contribution from fitting length of segments (&lt;em&gt;t&lt;/em&gt;&lt;sub&gt;s&lt;/sub&gt;, right). (Upper) Sets of closely related pairs for real relationship uncle&#x2013;nephew (UN) or cousin (C). (Lower) Sets of more distantly related pairs for cousins once removed (C1R), second cousins (2C), and second cousins once removed (2C1R).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/9/1553/F1.medium.gif" width="212" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/9/1553/F1.medium.gif" width="212" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/9/1553/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/9/1553/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463277" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-42" class="first-child">Cumulative contribution of sources of information to expected log likelihood ratio relative to its SD. E(<em>λ</em>)/SD(<em>λ</em>): number of segments only (<em>n</em><sub>s</sub> left), additional contribution from fitting position of segments (<em>p</em><sub>s</sub> center), and additional contribution from fitting length of segments (<em>t</em><sub>s</sub>, right). (Upper) Sets of closely related pairs for real relationship uncle–nephew (UN) or cousin (C). (Lower) Sets of more distantly related pairs for cousins once removed (C1R), second cousins (2C), and second cousins once removed (2C1R).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Approximating likelihoods</h3><div id="sec-17" class="subsection"><h4>Equal chromosome lengths:</h4><p id="p-43">To facilitate analysis of the distribution of log likelihood ratios, we consider a computational simplification, namely assuming all chromosomes have the same length rather than ranging over five different lengths. Hence, data also were simulated using a larger number of replicates (300,000) for chromosomes of length 1.632 M, the mean of those simulated previously, and likelihood ratios computed for genomes with 22 such chromosomes. Very similar values of E(<em>λ</em>), SD(<em>λ</em>), and, consequently, E(<em>λ</em>)/SD(<em>λ</em>) as those in Appendix Tables A1 through A9 were obtained. Results in Appendix Table A10 for E(<em>λ</em>)/SD(<em>λ</em>) enable comparison directly with those in Appendix Table A9 computed using the five chromosome lengths model. In summary, of the 342 off-diagonal comparisons of E(<em>λ</em>)/SD(<em>λ</em>) for the 19 relationships, only 32 deviated by more than 2% and of these 32, E(<em>λ</em>)/SD(<em>λ</em>) exceeded 1.0 in only 11, <em>i.e.</em>, large proportional differences typically occurred when absolute differences were small.</p></div><div id="sec-18" class="subsection"><h4>Replication:</h4><p id="p-44">Because differences in moments of <em>λ</em> ascribed to different models can arise from differences in expectation and from sampling in the simulation, a further run of 300,000 replicates for chromosomes of length 1.632 M as in Appendix Table A10 was undertaken (results not shown). The differences in E(<em>λ</em>)/SD(<em>λ</em>) between the replicates were very small; of the 342 off-diagonal comparisons, only 11 differed by more than 2%, and of those E(<em>λ</em>)/SD(<em>λ</em>) exceeded 1.0 in only 5. The main results (<em>e.g.</em>, <a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>, <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>, <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a> and corresponding Appendix Tables) computed for five lengths of chromosome involved a total of 500,000 unequally weighted runs, rather than 300,000 equally weighted runs (as we performed), so we conclude that sufficient replication was used.</p></div></div><div id="sec-19" class="subsection"><h3>Higher moments and distributions of log likelihood ratios</h3><p id="p-45">To simplify calculations, and in view of these results showing a good approximation of likelihood statistics computed for a model of chromosomes of equal length as that for chromosomes of different lengths, higher moments and distribution of <em>λ</em> were computed assuming all chromosomes had length 1.632 M (from simulations as in Appendix Table A10). Coefficients of skew and kurtosis are given in Appendix Table A11 and Table A12, respectively, for a subset of relationships. In general, both coefficients are small, indicating closeness to a normal distribution. The kurtosis coefficient is generally smaller than the skew, and kurtosis tends to be seen only in the presence of skew. The largest skew generally is found when the true relationship is weak and the assumed relationship is stronger, in which case there is negative skew. Positive skew is found less often, but typically when the assumed relationship is weaker than the true relationship. The apparent near-normality is not unexpected because each sample is of size 22 and the central limit theorem applies (as it would to results simulated for samples from chromosomes of five different lengths). Examples of the distribution of the log likelihood ratio, scaled as <em>λ/</em>SD(<em>λ</em>), are given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, showing near-“normal” form as anticipated in these particular examples.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/9/1553/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of log likelihood ratio, expressed relative to its SD, &#x3BB;/SD(&#x3BB;) using information on numbers, positions, and lengths of shared segments for examples of alternative pedigree relationships, e.g., real relationship uncle&#x2013;nephew hypothesized relationship half-sib (UN-HS). C, cousin; HC, half-cousin; 2C, second cousin; 3C, third cousin; HC1R, half-cousin once removed; GUGN, great-uncle&#x2013;great-nephew; G2PO, great-grandparent&#x2013;great-grandoffspring. Model of 22 chromosomes, each of 1.632 M." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-85653404" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of log likelihood ratio, expressed relative to its SD, &lt;em&gt;&#x3BB;&lt;/em&gt;/SD(&lt;em&gt;&#x3BB;&lt;/em&gt;) using information on numbers, positions, and lengths of shared segments for examples of alternative pedigree relationships, &lt;em&gt;e.g.&lt;/em&gt;, real relationship uncle&#x2013;nephew hypothesized relationship half-sib (UN-HS). C, cousin; HC, half-cousin; 2C, second cousin; 3C, third cousin; HC1R, half-cousin once removed; GUGN, great-uncle&#x2013;great-nephew; G2PO, great-grandparent&#x2013;great-grandoffspring. Model of 22 chromosomes, each of 1.632 M.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/9/1553/F2.medium.gif" width="440" height="387"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/9/1553/F2.medium.gif" width="440" height="387"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/9/1553/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/9/1553/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463190" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-46" class="first-child">Distribution of log likelihood ratio, expressed relative to its SD, <em>λ</em>/SD(<em>λ</em>) using information on numbers, positions, and lengths of shared segments for examples of alternative pedigree relationships, <em>e.g.</em>, real relationship uncle–nephew hypothesized relationship half-sib (UN-HS). C, cousin; HC, half-cousin; 2C, second cousin; 3C, third cousin; HC1R, half-cousin once removed; GUGN, great-uncle–great-nephew; G2PO, great-grandparent–great-grandoffspring. Model of 22 chromosomes, each of 1.632 M.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Approximations to sampling distributions</h3><p id="p-47">The results we have used have been based entirely on simulation. We investigate, however, theoretical results available that could be used to obtain some more simply computed but potentially less informative tests of pedigree relationship.</p><p id="p-48">Based on work by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Thomas <em>et al.</em> (1994)</a>, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Huff <em>et al.</em> (2011)</a> give an expression for the expected number of shared segments in the genome that, for a single chromosome, becomes the following equation (equation 2):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1553/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/9/1553/embed/mml-math-2.gif"/></noscript></span></span>where <em>a</em> is the number of ancestors (1 for half-sib mating, 2 for full-sib mating), <em>d</em> is the total number of meioses separating ancestors and descendants (back to the grandparents), and <em>l</em> is the map length. For lineal descendents, numbers of shared segments are typically one-half those of half-sib descendents, and the expected number shared with the grandparent (or founder of a recurrent backcross line) is [(<em>d</em> − 1)<em>l</em> + 1](½)<sup>(</sup><em><sup>d</sup></em><sup>−1)</sup>, where <em>d</em> is the number of meioses back to the grandparent (<em>i.e.</em>, founder, hence terms in <em>d</em> − 1 because recombination to the parent is irrelevant). Thus, for example, <em>R</em> = 1/16 and <em>d</em> = 4 for full-sib–based (cousins once removed), half-sib–based (half cousins), and lineal descendents (great-great grandparent–great-great grandoffspring). The formulae do not apply to the cases of uncle–nephew, for which (surmised from simulations as in <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>) E(<em>n</em><sub>s</sub>) = (5<em>l</em> + 2)/4, or great-uncle–great-nephew, for which E(<em>n</em><sub>s</sub>) = (7<em>l</em> + 2)/8. The mean numbers of shared segments from simulation agree (within sampling error) with prediction (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463301/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463301/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Mean and variance of number of shared segments on a chromosome of 1.632 M for different relationships obtained by simulation (300,000 replicates)<sup><em><sup>a</sup></em></sup></span><div class="sb-div caption-clear"></div></div></div><p id="p-51"><a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Huff <em>et al.</em> (2011)</a> also state that given <em>d</em>, the expected length of a shared segment is 1/<em>d</em>, based on the calculations of length surrounding a specific marker (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fisher 1949</a>). They assume independence of numbers and length, implying from equation 2 that the expected total length of a chromosome shared is (<em>dl</em> + 1)(½)<sup>(</sup><em><sup>d</sup></em><sup>−1)</sup>/<em>d</em> for half-sib descendents. However, because the expected proportion of the genome shared is 2<em>R</em> = (½)<sup>(</sup><em><sup>d</sup></em><sup>−1)</sup><sub>,</sub> the mean total length shared is actually <em>l</em>(½)<sup>(</sup><em><sup>d</sup></em><sup>−1)</sup>. It is partitioned over the expected number (<em>dl</em> + 1)(½)<sup>(</sup><em><sup>d</sup></em><sup>−1)</sup> of shared segments and, therefore, taking into account the finite length of the chromosome, the expected length of an individual segment is <em>l</em>/(<em>dl</em> + 1) = 1/(<em>d</em> + 1/<em>l</em>), not 1/<em>d</em>. These equations also hold for full-sib and lineal descendants. For example, for a chromosome of length 1.632 M, the expected lengths of a shared segment are 0.383 M, 0.277 M, and 0.217 M for half-sibs, half-uncle, and half-cousins, respectively, rather than 0.5 M, 0.333 M, and 0.25 M without the correction. The proportionate difference becomes smaller for more distant relatives, <em>e.g.</em>, 0.151 M rather than 0.167 M for half second cousins. For uncle–nephew and great-uncle–great-nephew, the expected lengths of individual segments on a chromosome are, from simulation, 2/(5<em>l</em> + 2) and 2/(7<em>l</em> + 2), respectively.</p><p id="p-52"><a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Huff <em>et al.</em> (2011)</a> also made the simplifying assumption that the number of shared segments is Poisson-distributed, implying Var(<em>n</em><sub>s</sub>) = E(<em>n</em><sub>s</sub>) on individual chromosomes and the whole genome, but simulations show departures between mean and variance (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>). For a chromosome of length <em>l</em> = 1.632 M, the actual distribution is rather less dispersed than the Poisson for close relatives, but slightly more dispersed for more distant relatives. For cousins, for example, E(<em>n</em><sub>s</sub>) = 1.882, V(<em>n</em><sub>s</sub>) = 1.311, and the proportion sharing no segments is ∼10% (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>), but the Poisson expectation is ∼15%. Further, the distribution of shared segment lengths was assumed by Huff <em>et al.</em> to be independently exponentially distributed, in which case the coefficient of variation (CV) of the total length of shared segments on a chromosome would be proportional to 1/√<em>n</em><sub>s</sub>. For close relations who may share a high proportion of the chromosome, the actual distribution is substantially underdispersed compared with the Poisson and the CV of total length shared deviates from the 1/√<em>n</em><sub>s</sub> prediction. As relationships get more distant, these predictions hold better.</p></div><div id="sec-21" class="subsection"><h3>Using approximate sampling distributions to distinguish relationships</h3><p id="p-53">Because the predicted numbers (<em>n</em><sub>s</sub>) of shared segments (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Huff <em>et al.</em> 2011</a>) have the correct mean, they provide a simple route to likelihood calculations without simulations. Further, as illustrated in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, most of the information can be obtained from the numbers of shared segments without using their positions and length. As the actual distribution departs from the Poisson (<a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>), however, there would be some reduction in discriminating power in computation of likelihoods, even from number of segments shared alone. To investigate this, we computed the log likelihood ratio for alternative types of relationships using data only regarding <em>n</em><sub>S</sub> assuming it is Poisson-distributed, and we computed its mean and SD using the actual frequency distribution obtained from simulation. For simplicity, we assumed 22 chromosomes each of length 1.632 M. Examples are given in <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a> for the log likelihood ratio computed using both the Poisson and the actual distributions.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463245/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463245/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Expected log likelihood ratio, E(<em>λ</em>), and ratio to its SD, E(<em>λ</em>)/SD(<em>λ</em>), using data from numbers of shared segments (<em>n</em><sub>s</sub>) only computed from simulated data and also from the Poisson assumption, but with weights as for the simulated (actual) data</span><div class="sb-div caption-clear"></div></div></div><p id="p-56">The log likelihood ratios remain zero when the real and assumed relationships are the same. In general, E(<em>λ</em>) is smaller when the Poisson approximation is used, but the proportional reduction is inconsistent. There are cases when it is larger, which seems illogical, but there is no guarantee <em>λ</em> decreases because the test is against a false hypothesis, with the actual distribution fitting closer to the Poisson with the wrong parameters. Because the SD is also substantially affected and typically is smaller, the ratio E(<em>λ</em>)/SD(<em>λ</em>) is often larger than that computed using the correct distribution obtained by simulation, but the pattern is not consistent. In view of this, such approximations should be used with care, and in any case we have provided an exact approach (strictly, more nearly exact, from replicate simulations).</p></div><div id="sec-22" class="subsection"><h3>Extension to other species: impact of chromosome number and length</h3><p id="p-57">Results have been given for a model human genome of <em>c</em> = 22 autosomes with a total map length of <em>L</em> = 35.9 M; however, to assess how they need modifying for other species, we consider how <em>c</em> and <em>L</em> influence results. We have shown that a model of 22 chromosomes of equal average length (1.632 M) approximates that with lengths ranging from 0.75 M to 2.75 M, with most in mid range. Therefore, if chromosomes have similar mean length to those of humans and the distribution of lengths is no more dispersed, moments for different numbers of chromosomes can be predicted well by scaling as E(<em>λ</em>) ∝ <em>c</em> and E(<em>λ</em>)/SD(<em>λ</em>) ∝ √<em>c</em> because they are independent. To investigate the impact of wider variation in length we considered alternatives with total genome length 36 M, comprising 72 chromosomes each of 0.5 M or 12 chromosomes each of 3 M.</p><p id="p-58">Ability to discriminate, expressed in terms of E(λ)/SD(<em>λ</em>), is given for some examples of relationship in Appendix Table A13 using either numbers of segments alone or all sources, <em>i.e.</em>, numbers, positions, and lengths. In summary, when there are many independent chromosomes, E(<em>λ</em>)/SD(<em>λ</em>) is generally higher than when there are few, particularly when Wright’s relationship <em>R</em> differs, because probabilities of ibd at individual loci are mostly uncorrelated with many small chromosomes. Independent loci do not provide evidence to distinguish relationships such as uncle–nephew and half-sibs having the same <em>R</em>. Information is contained in the distribution of number and length of shared segments, however, and the differences in E(<em>λ</em>)/SD(<em>λ</em>) between the 12 and 72 chromosome models for the same total map length are small, although generally higher for <em>c</em> = 72 when comparing relationships with different <em>R</em>. For relationships with the same <em>R</em> there is negligible difference, <em>e.g.</em>, real relationship half-sib, assumed uncle–nephew, E(<em>λ</em>)/SD(<em>λ</em>) = 0.97 for 72 chromosomes, 0.98 for 12 (Appendix Table A13) chromosomes, and 0.98 for 22 variable-length chromosomes (<a id="xref-table-wrap-5-6" class="xref-table" href="#T5">Table 5</a>). Overall, therefore, the discriminating power clearly depends more on total amount of genome rather than on the individual chromosome lengths for the typical range of lengths in mammals.</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><div id="sec-24" class="subsection"><h3>Inference</h3><p id="p-59">Although likelihood ratios are a natural way to describe the plausibility of alternative relationships, how to draw inferences from them is less clear-cut. Let <strong>Ω</strong> denote the set of all pedigree relationships <strong>R</strong> under consideration. Because this is a finite set of discrete elements, it removes some of the difficulties in assigning prior probabilities when, typically, these are neither specified nor easy to specify. Bayes theorem can then be used to combine likelihoods and prior probabilities to produce a posterior distribution over the elements of <strong>Ω</strong>. Unless some form of ordering, or measure of distance, is introduced in <strong>Ω</strong>, it is impossible to speak of means or variances of this distribution, but it will usually have a unique mode, and the corresponding relationship <strong>R</strong> will be our “best guess” at the true relationship. A confidence set could be obtained by ordering relationships by posterior probability and dropping relationships with the smallest probabilities until a desired probability level is achieved for the remainder.</p><p id="p-60">Without prior probabilities, everything hangs on the likelihood. The likelihood function is defined on <strong>Ω</strong>, and the relationship <strong>R</strong> in <strong>Ω</strong> that produces the maximum value of the likelihood is the maximum likelihood estimate of the true relationship, corresponding to the posterior mode with a uniform prior. Without a distance measure, and with discrete relationship classes, standard asymptotic results for maximum likelihood estimates are not available. The distribution of the maximum likelihood estimate could be calculated by simulation, however, assuming any particular <strong>R</strong> to be true.</p><p id="p-61">Any particular <strong>R</strong> can be tested as a null hypothesis against the general alternative that the true relationship is not <strong>R</strong> by using a maximum likelihood ratio test (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">McPeek and Sun, 2000</a>). The set of those <strong>R</strong> in <strong>Ω</strong> for which this test is not significant at a given significance level constitutes a confidence set for the unknown relationship. <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">McPeek and Sun (2000</a>, p. 1079) point out that although the sampling distribution of log likelihood ratios for two fixed relationships is often close to a normal distribution (as we have shown previously; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, Appendix Tables A11 and A12), the sampling distribution of the maximized version tends to be skewed (the difference is between the estimate of <strong>R</strong> fixed or varying from sample to sample). Nevertheless, even in the normal case, simulation is required to obtain the mean and variance of the null distribution.</p><p id="p-62">An issue that arises with both Bayesian and likelihood approaches is the completeness or otherwise of <strong>Ω.</strong> The true relationship might be one we neglected to consider; it might be bilinear, but not so detected (<em>e.g.</em>, paternal half-sibs and maternal second cousins), or an ancestor might be inbred so the probabilities of ibd sharing of descendents differ from those assumed here. Some relationships could be excluded based, for example, on ages of the individuals concerned, <em>e.g.</em>, some lineal or avuncular relationships.</p><p id="p-63">If all that is required is to identify the “best guess” among all relationships under consideration, then we select the relationship with the largest likelihood, or the largest posterior probability. This can be regarded as a discrimination problem, with the relationships treated symmetrically. The two solutions correspond to the maximum likelihood or Bayes discriminate rules (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mardia <em>et al.</em> 1979</a>), and the performance of such rules is judged by the set of misclassification probabilities.</p><p id="p-64">Discriminating between two relationships amounts to choosing one if the log likelihood ratio <em>λ</em> &gt; 0 and choosing the other if <em>λ</em> &lt; 0. If the two relationships are <strong>A</strong> and <strong>B</strong>, and the distribution of <em>λ</em> is normal in each case, then the misclassification probabilities are Φ(−<em>m</em><sub><strong>B</strong></sub>/<em>s</em><sub><strong>B</strong></sub>) when we choose <strong>A</strong>, and Φ(−<em>m</em><sub><strong>A</strong></sub>/<em>s</em><sub><strong>A</strong></sub>) when we choose <strong>B</strong>, where <em>m</em><sub><strong>A</strong></sub> = E<sub><strong>A</strong></sub>[<em>λ</em>(<strong>A</strong> : <strong>B</strong>)], <em>i.e.</em>, the mean of <em>λ</em> when <strong>A</strong> is the true relationship, and <em>m</em><sub><strong>B</strong></sub> = E<sub><strong>B</strong></sub>[<em>λ</em>(<strong>B</strong> : <strong>A</strong>)] (= −E<sub><strong>B</strong></sub>[<em>λ</em>(<strong>A</strong> : <strong>B</strong>)]) when <strong>B</strong> is the true relationship and <em>s</em><sub><strong>A</strong></sub> and <em>s</em><sub><strong>B</strong></sub> are the corresponding SDs. Ratios of <em>m</em>/<em>s</em> for various pairs of relationship are in <a id="xref-table-wrap-5-7" class="xref-table" href="#T5">Table 5</a> and <a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>, with more in Appendix Tables A7–A9.</p><p id="p-65">As an example, let us assume X dies intestate and a search locates one living relative, indisputably a half-cousin. Subsequently, Y appears claiming to be a cousin of X (but otherwise unrelated to Y), and thus is more closely related. Given only DNA data, can the claim be substantiated or disproved? There are two competing hypotheses: for <strong>A</strong>, Y is a cousin of X; and for <strong>B</strong>, Y is a half-cousin of X. To keep this argument simple, we discount other possible relationships. Given a prior probability that X and Y are cousins, the Bayesian approach provides a posterior probability, but it is not clear what a reasonable prior probability would be in the absence of any background information for Y. With the likelihood approach, we can clearly discriminate with confidence in this situation because both misclassification probabilities are small, ∼1.5% using <a id="xref-table-wrap-5-8" class="xref-table" href="#T5">Table 5</a>, Φ(−2.22) ∼ 0.013, <em>i.e.</em>, if we decide half-cousins, and Φ(−2.14) ∼ 0.016 if we decide cousins.</p><p id="p-66">Taking as a simple criterion a difference of 2 SD in log likelihood ratio as an indicator of discriminating ability (corresponding to a misclassification probability of approximately 0.02), it is seen that although it is possible to distinguish between a distant and a close relationship with high power, it is more difficult between relationships of the same degree (<em>R</em>), increasingly so as <em>R</em> becomes smaller (<a id="xref-table-wrap-5-9" class="xref-table" href="#T5">Table 5</a> and <a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a>). There is little power to discriminate between relationships for which <em>R</em> is 1/64 or less; for example, the probability of correct assignment (based simply on sign of the log likelihood ratio) is approximately 3/4 for second cousins once removed <em>vs.</em> third cousins as E(<em>λ</em>)/SD(<em>λ</em>) ∼ 0.6. It is easier to distinguish lineal relationships, <em>e.g.</em>, great-great-great-grandparent–offspring from second cousins, than it is to distinguish second cousins from half-cousins once removed (for all of which <em>R</em> = 1/32) because the lineal recombination and transmission process differs more than that between half and full-sib descendants.</p><p id="p-67">Without use of information as shown here regarding shared genomic regions and merely considering resemblance locus by locus, relationships such as uncle–nephew and half-sib cannot be distinguished at all. It is seen that E(<em>λ</em>)/SD(<em>λ</em>) ∼ 1, whichever relationship is the real one. Hence, the likelihood ratio will be in the correct direction approximately 5/6 of the time—not certainty at a level looked for in significance tests, but not valueless. For more distant pairs with the same <em>R</em>, the probability of correct assignment will decline; for second cousins and half-cousins once removed (<em>R</em> = 1/32), the probability declines to approximately 2/3. This illustrates the limitations of making decisions about the relationship between a pair of individuals even if based on full genomic data.</p></div><div id="sec-25" class="subsection"><h3>Assumptions</h3><p id="p-68">Many assumptions have been made in this analysis. The first is that the number of shared segments is accurately recorded, and the main risk is that short segments are missed. In population studies, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Browning and Browning (2013)</a> and S.R. Browning (personal communication) report good power to detect segments of 1.5 cM and higher using dense SNP data and 1 cM or higher with sequence data. For exponentially distributed segment lengths of expected length <em>a</em> (cM), this would imply a probability of missing an individual segment of approximately 1.5/<em>a</em> (1/<em>a</em>) from SNP (sequence) data. For half-cousins, for example, the expected segment length is 21.7 cM for a chromosome of 1.632 M (see <em>Results</em> regarding approximations to sampling distributions), implying an approximately 7% chance of missing a random segment using SNPs, slightly less for closer relatives or using sequence data. Thus, there would be bias towards underestimating both Wright’s and pedigree relationship, but little in comparing relatives with the same <em>R</em>. For known relatives, however, as considered here, the probability would be expected to be much lower because the individuals are already identified as relatives and not trawled from the population. Errors in estimating segment length would be comparatively unimportant (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-69">Errors therefore will not necessarily lead to wrong assignment but to miscalculation of the likelihood ratios. As <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a> show, however, the pattern of numbers shared is unlikely to change greatly if the error rate is no more than a few percent, and the relative parameters for different relationships will remain approximately the same. A detailed analysis of consequences of errors is beyond the scope of this article, however.</p><p id="p-70">Further assumptions made when information on chromosome length is included are that a Haldane mapping function is appropriate and that map length can be accurately inferred from physical length of the chromosome. We consider the number of segments and the probability that shared segments reach chromosome ends would depend little or not at all on the mapping function. Problems might be encountered in measuring the segment length distribution, converted to map units, before using the data and methods presented here. If there are major experimental technical problems in measuring lengths or concern about the mapping functions or conversion from physical length, then that information could just be ignored with, for most pairings of <strong>R</strong>, little impact on discriminating ability (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-71">We also have taken no account of distant background relationship, assuming all genome sharing was due to recent common ancestry, whereas <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Huff <em>et al.</em> (2011)</a> did so. Such sharing will bias predictions towards higher relationship. As in the example here, an extra rather than lost shared segment on one or two chromosomes will have little effect on likelihood calculations for fairly close relationships. Proportional errors become larger as relationships become more distant, but as results such as in <a id="xref-table-wrap-6-6" class="xref-table" href="#T6">Table 6</a> show, the power to discriminate among quite distant relationships is low in any case.</p></div><div id="sec-26" class="subsection"><h3>General conclusions</h3><p id="p-72">The results presented here show what can, in theory, be achieved in determining pedigree relationships from information on genome sharing. No further information is, in principle, available from analysis at the individual locus level (except perhaps from sequencing and tracing point mutations in the pedigree). The low levels of expected likelihood ratios compared with their sampling error for pairs of quite distant relationships illustrate both how much variability in actual relationship in terms of shared genome comes from random Mendelian segregation and linkage and the consequent difficulty in assigning relationship.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-87">We thank Bruce Weir for helpful comments and discussion and reviewers for their useful criticisms. This work was supported by grants from the Leverhulme Trust to William G. Hill and from National Institutes of Health (GM 099568) to Bruce Weir, University of Washington, and by USS.</p></div><div class="section app-group" id="app-group-1"><h2>Appendices</h2><p id="p-73">Simulated data in supplementary files.</p><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463240/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463240/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A1</span> <span class="caption-title">Expectation of log likelihood ratio E(<em>λ</em>) for 19 relationships incorporating information only on numbers of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463174/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463174/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A2</span> <span class="caption-title">E(<em>λ</em>) as Table A1, but incorporating information on numbers and positions of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T11" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463160/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463160/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A3</span> <span class="caption-title">E(<em>λ</em>) as Tables A1 and A2, but incorporating information on numbers, positions, and lengths of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T12" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463244/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463244/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A4</span> <span class="caption-title">SD(<em>λ</em>) as Table A1 using information only on numbers of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T13" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463158/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463158/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A5</span> <span class="caption-title">SD(<em>λ</em>) as Table A2 using information on numbers and position of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T14" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463264/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463264/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A6</span> <span class="caption-title">SD(<em>λ</em>) as Table A3 using information on numbers, position, and length of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T15" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463256/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463256/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A7</span> <span class="caption-title">E(<em>λ</em>)/SD(<em>λ</em>) as Tables A1 and A4 using information only on numbers of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T16" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463142/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463142/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A8</span> <span class="caption-title">E(<em>λ</em>)/SD(<em>λ</em>) as Tables A2 and A5 using information on numbers and position of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T17" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463271/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463271/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A9</span> <span class="caption-title">E(<em>λ</em>)/SD(<em>λ</em>) as Tables A3 and A6 using information on numbers, position, and length of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T18" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463282/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463282/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A10</span> <span class="caption-title">E(<em>λ</em>)/SD(<em>λ</em>) evaluated using a model of 22 chromosomes, each of length 1.632 M, using information as Table A9 on numbers, position, and length of segments</span><div class="sb-div caption-clear"></div></div></div><div id="T19" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463243/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463243/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A11</span> <span class="caption-title">Skew coefficient of log likelihood ratio for a subset of relationships for the model in Table A10 of 22 chromosomes, each of 1.632 M, using information on numbers, positions, and lengths of shared segments</span><div class="sb-div caption-clear"></div></div></div><div id="T20" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463155/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463155/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A12</span> <span class="caption-title">Kurtosis coefficient of log likelihood ratio, otherwise as Table A11</span><div class="sb-div caption-clear"></div></div></div><div id="T21" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463298/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463298/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Appendix Table A13</span> <span class="caption-title">Influence on E(<em>λ</em>)/SD(<em>λ</em>) of chromosome number <em>vs.</em> length, (72 × 0.5 M or 12 × 3M) for total 36 M, using information only on number of shared segments (<em>n</em><sub>S</sub>) or using their numbers, position, and length <em>n<sub>s</sub></em>, <em>p<sub>s</sub></em>, and <em>t<sub>s</sub></em></span><div class="sb-div caption-clear"></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007500/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007500/-/DC1</a></p></li><li class="fn" id="fn-30"><p id="p-88"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-04-09"><span class="received-label">Received </span>April 9, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-07-05"><span class="accepted-label">Accepted </span>July 5, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Hill and White</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.1" data-doi="10.1038/ng786"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherny</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cookson</span>  <span class="cit-name-given-names">W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Merlin - rapid analysis of dense genetic maps using sparse gene flow trees</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D97%26rft.epage%253D101%26rft.atitle%253DMerlin--rapid%2Banalysis%2Bof%2Bdense%2Bgenetic%2Bmaps%2Busing%2Bsparse%2Bgene%2Bflow%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng786%26rft_id%253Dinfo%253Apmid%252F11731797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11731797&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173105600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.2" data-doi="10.1016/S0169-5347(03)00225-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blouin</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">DNA-based methods for pedigree reconstruction and kinship analysis in natural populations</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D18%26rft.spage%253D503%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252803%252900225-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(03)00225-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185728100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.3" data-doi="10.1534/genetics.113.150029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improving the accuracy and efficiency of identity-by-descent detection in population data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150029%26rft_id%253Dinfo%253Apmid%252F23535385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.4" data-doi="10.1016/j.ajhg.2011.01.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, powerful method for detecting identity by descent</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D182%26rft.atitle%253DA%2Bfast%252C%2Bpowerful%2Bmethod%2Bfor%2Bdetecting%2Bidentity%2Bby%2Bdescent.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.01.010%26rft_id%253Dinfo%253Apmid%252F21310274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21310274&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.5" data-doi="10.1146/annurev-genet-110711-155534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identity by descent between distant relatives: detection and applications</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D46%26rft.spage%253D617%26rft.epage%253D633%26rft.atitle%253DIdentity%2Bby%2Bdescent%2Bbetween%2Bdistant%2Brelatives%253A%2Bdetection%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-110711-155534%26rft_id%253Dinfo%253Apmid%252F22994355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-110711-155534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22994355&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311568300028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.6" data-doi="10.1071/WR99107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burnham</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Kullback–Leibler information as a basis for strong inference in ecological studies</span>. <abbr class="cit-jnl-abbrev">Wildl. Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWildl.%2BRes.%26rft.volume%253D28%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1071%252FWR99107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/WR99107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000168859000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.7" data-doi="10.1016/0040-5809(83)90004-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The probability that related individuals share some section of the genome identical by descent</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DDonnelly%26rft.auinit1%253DK.%2BP.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D34%26rft.epage%253D63%26rft.atitle%253DThe%2Bprobability%2Bthat%2Brelated%2Bindividuals%2Bshare%2Bsome%2Bsection%2Bof%2Bgenome%2Bidentical%2Bby%2Bdescent.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252883%252990004-7%26rft_id%253Dinfo%253Apmid%252F6857549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(83)90004-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6857549&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QK30500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1553.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-source">The theory of inbreeding</span>, <span class="cit-publ-name">Oliver and Boyd</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.9" data-doi="10.1038/hdy.1954.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">A fuller theory of ‘Junctions’ in inbreeding</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D8%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1954.17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1954.17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954YC17800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.10" data-doi="10.1086/321195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duren</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boenkhe</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Improved inference of relationships for pairs of individuals</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DEpstein%26rft.auinit1%253DM.%2BP.%26rft.volume%253D67%26rft.issue%253D5%26rft.spage%253D1219%26rft.epage%253D1231%26rft.atitle%253DImproved%2Binference%2Bof%2Brelationship%2Bfor%2Bpairs%2Bof%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321195%26rft_id%253Dinfo%253Apmid%252F11032786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11032786&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165091600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.11" data-doi="10.1017/S0016672310000480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in actual relationship as a consequence of Mendelian sampling and linkage</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D64%26rft.atitle%253DVariation%2Bin%2Bactual%2Brelationship%2Bas%2Ba%2Bconsequence%2Bof%2BMendelian%2Bsampling%2Band%2Blinkage.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000480%26rft_id%253Dinfo%253Apmid%252F21226974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21226974&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.12" data-doi="10.1101/gr.115972.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huff</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witherspoon</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonson</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wa</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Maximum-likelihood estimation of recent shared ancestry (ERSA)</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115972.110%26rft_id%253Dinfo%253Apmid%252F21324875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvNzY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.13" data-doi="10.1086/426405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A combined physical-linkage map of the human genome</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DKong%26rft.auinit1%253DX.%26rft.volume%253D75%26rft.issue%253D6%26rft.spage%253D1143%26rft.epage%253D1148%26rft.atitle%253DA%2Bcombined%2Blinkage-physical%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F426405%26rft_id%253Dinfo%253Apmid%252F15486828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/426405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15486828&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224866400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.14" data-doi="10.1214/aoms/1177729694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kullback</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibler</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">On information and sufficiency</span>. <abbr class="cit-jnl-abbrev">Ann. Math. Stat.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat.%26rft.volume%253D22%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177729694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177729694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.15" data-doi="10.1093/bioinformatics/btr243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kyriazopoulou-Panagiotopoulou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haghighi</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerni</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundquist</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bercovici</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reconstruction of genealogical relationships with applications to Phase III of HapMap</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">I333</span>–<span class="cit-lpage">I341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D27%26rft.spage%253DI333%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr243%26rft_id%253Dinfo%253Apmid%252F21685089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21685089&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1553.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mardia</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bibby</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-source">Multivariate Analysis</span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.17" data-doi="10.1046/j.1365-294x.1998.00374.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Statistical confidence for likelihood-based paternity inference in natural populations</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMarshall%26rft.auinit1%253DT.%2BC.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D639%26rft.epage%253D655%26rft.atitle%253DStatistical%2Bconfidence%2Bfor%2Blikelihood-based%2Bpaternity%2Binference%2Bin%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.1998.00374.x%26rft_id%253Dinfo%253Apmid%252F9633105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.1998.00374.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9633105&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073810200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.18" data-doi="10.1086/302800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Statistical tests for detection of misspecified relationships by use of genome-screen data</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">1076</span>–<span class="cit-lpage">1094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMcPeek%26rft.auinit1%253DM.%2BS.%26rft.volume%253D66%26rft.issue%253D3%26rft.spage%253D1076%26rft.epage%253D1094%26rft.atitle%253DStatistical%2Btests%2Bfor%2Bdetection%2Bof%2Bmisspecified%2Brelationships%2Bby%2Buse%2Bof%2Bgenome-screen%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302800%26rft_id%253Dinfo%253Apmid%252F10712219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10712219&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088373200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.19" data-doi="10.1098/rspb.2007.1531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Wild pedigrees: the way forward</span>. <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DPemberton%26rft.auinit1%253DJ.M%26rft.volume%253D275%26rft.issue%253D1635%26rft.spage%253D613%26rft.epage%253D621%26rft.atitle%253DWild%2Bpedigrees%253A%2Bthe%2Bway%2Bforward%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.1531%26rft_id%253Dinfo%253Apmid%252F18211868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc1LzE2MzUvNjEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMy85LzE1NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.20" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a toolset for whole-genome association and population-based linkage analysis</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.21" data-doi="10.1371/journal.pone.0006711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberson</span>  <span class="cit-name-given-names">E. D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pevsner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Visualization of shared genomic regions and meiotic recombination in high-density SNP data</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRoberson%26rft.auinit1%253DE.%2BD.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6711%26rft.epage%253De6711%26rft.atitle%253DVisualization%2Bof%2Bshared%2Bgenomic%2Bregions%2Band%2Bmeiotic%2Brecombination%2Bin%2Bhigh-density%2BSNP%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006711%26rft_id%253Dinfo%253Apmid%252F19696932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19696932&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.22" data-doi="10.1371/journal.pgen.1002469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rohlfs</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fullerton</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Familial identification: population structure and relationship distinguishability</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRohlfs%26rft.auinit1%253DR.%2BV.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253De1002469%26rft.epage%253De1002469%26rft.atitle%253DFamilial%2Bidentification%253A%2Bpopulation%2Bstructure%2Band%2Brelationship%2Bdistinguishability.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002469%26rft_id%253Dinfo%253Apmid%252F22346758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002469&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22346758&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F9%2F1553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.23" data-doi="10.1017/S0016672300014002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The distribution of the fraction of the genome identical by descent in finite populations</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D35%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.24" data-doi="10.1093/imammb/11.1.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skolnick</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genomic mismatch scanning in pedigrees</span>. <abbr class="cit-jnl-abbrev">IMA J. Math. Appl. Med. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIMA%2BJ.%2BMath.%2BAppl.%2BMed.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fimammb%252F11.1.1%26rft_id%253Dinfo%253Apmid%252F8057038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiaW1hbW1iIjtzOjU6InJlc2lkIjtzOjY6IjExLzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzMvOS8xNTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.9.1553.25" data-doi="10.1111/j.1755-0998.2010.02885.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">COANCESTRY: A program for simulating, estimating and analysing relatedness and inbreeding coefficients</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Resources</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResources%26rft.volume%253D11%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02885.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02885.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-3.9.1553.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Genetic data analysis. II</span>, <span class="cit-publ-name">Sinauer</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/9/1553.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/9/1545" title="Mitochondrial DNA with a Large-Scale Deletion Causes Two Distinct Mitochondrial Disease Phenotypes in Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/9/1573" title="Maternal Loss of miRNAs Leads to Increased Variance in Primordial Germ Cell Numbers in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/9"><span>Volume 3 Issue 9, September 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/9.cover-source.jpg?itok=2NEcI52z" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/9/1553" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Me9Zzwad_NLnfDyTLkHbV8XsbwDBEfFe-DRcA6QnS6o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/9/1553" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/9/1553" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/9/1553" class="active" data-icon-position="" data-hide-link-title="0">Identification of Pedigree Relationship from Genome Sharing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459534" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2mBa_08TOoVmxF7xsVbttlBJYxroz8atdxIn4w3NxPI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/9/1553.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/9/1553" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-eXInktq9decNL9iUDMLLi4mluSTZtJG0-2WmTUizeIc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/9/1553" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/9/1553.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459534" id="node459534--2" data-pisa="ggg;3/9/1553" data-pisa-master="ggg;g3.113.007500" data-apath="/ggg/3/9/1553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Pedigree Relationship from Genome Sharing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">William G. Hill</span> and <span class="highwire-citation-author" data-delta="1">Ian M. S. White</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1553-1571;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007500 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459534--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">William G. Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BG.%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20WG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BG.%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:w.g.hill@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">w.g.hill@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ian M. S. White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BM.%2BS.%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20IM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BM.%2BS.%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/9/1553" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/9/1553&amp;title=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/9/1553&amp;title=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/9/1553&amp;title=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/9/1553&amp;text=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/9/1553&amp;title=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/9/1553&amp;t=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/9/1553&amp;title=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/9/1553&amp;title=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459534/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459534" id="node459534--3" data-pisa="ggg;3/9/1553" data-pisa-master="ggg;g3.113.007500" data-apath="/ggg/3/9/1553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Pedigree Relationship from Genome Sharing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">William G. Hill</span> and <span class="highwire-citation-author" data-delta="1">Ian M. S. White</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1553-1571;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007500 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459534--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">William G. Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BG.%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20WG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BG.%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:w.g.hill@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">w.g.hill@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ian M. S. White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Evolutionary Biology, School of Biological Sciences, University of Edinburgh, Edinburgh, EH9 3JT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BM.%2BS.%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20IM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BM.%2BS.%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1553&amp;rft.epage=1571&amp;rft.atitle=Identification%20of%20Pedigree%20Relationship%20from%20Genome%20Sharing&amp;rft.volume=3&amp;rft.issue=9&amp;rft.date=2013-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hill%2C+William+G.&amp;rft.au=White%2C+Ian+M.+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459534/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459534/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459534/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459534/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459534/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459534/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459534/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459534/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459534/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459534/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459534/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459534/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/9/1553" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#app-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendices</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/9/1553.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/9/1553.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/9/1553.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/9/1553.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
