<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/8/1685" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/8/1685.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/8/1685.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/8/1685.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/8/1685/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/8/1685.atom" />
<meta name="HW.pisa" content="ggg;5/8/1685" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda" />
<meta name="DC.Identifier" content="10.1534/g3.115.019588" />
<meta name="DC.Date" content="2015-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Westbrook et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A consensus genetic map for Pinus taeda (loblolly pine) and Pinus elliottii (slash pine) was constructed by merging three previously published P. taeda maps with a map from a pseudo-backcross between P. elliottii and P. taeda . The consensus map positioned 3856 markers via genotyping of 1251 individuals from four pedigrees. It is the densest linkage map for a conifer to date. Average marker spacing was 0.6 cM and total map length was 2305 cM. Functional predictions of mapped genes were improved by aligning expressed sequence tags used for marker discovery to full-length P. taeda transcripts. Alignments to the P. taeda genome mapped 3305 scaffold sequences onto 12 linkage groups. The consensus genetic map was used to compare the genome-wide linkage disequilibrium in a population of distantly related P. taeda individuals (ADEPT2) used for association genetic studies and a multiple-family pedigree used for genomic selection (CCLONES). The prevalence and extent of LD was greater in CCLONES as compared to ADEPT2; however, extended LD with LGs or between LGs was rare in both populations. The average squared correlations, r2, between SNP alleles less than 1 cM apart were less than 0.05 in both populations and r2 did not decay substantially with genetic distance. The consensus map and analysis of linkage disequilibrium establish a foundation for comparative association mapping and genomic selection in P. taeda and P. elliottii ." />
<meta name="DC.Contributor" content="Jared W. Westbrook" />
<meta name="DC.Contributor" content="Vikram E. Chhatre" />
<meta name="DC.Contributor" content="Le-Shin Wu" />
<meta name="DC.Contributor" content="Srikar Chamala" />
<meta name="DC.Contributor" content="Leandro Gomide Neves" />
<meta name="DC.Contributor" content="Patricio Muñoz" />
<meta name="DC.Contributor" content="Pedro J. Martínez-García" />
<meta name="DC.Contributor" content="David B. Neale" />
<meta name="DC.Contributor" content="Matias Kirst" />
<meta name="DC.Contributor" content="Keithanne Mockaitis" />
<meta name="DC.Contributor" content="C. Dana Nelson" />
<meta name="DC.Contributor" content="Gary F. Peter" />
<meta name="DC.Contributor" content="John M. Davis" />
<meta name="DC.Contributor" content="Craig S. Echt" />
<meta name="article:published_time" content="2015-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A consensus genetic map for &lt;i&gt;Pinus taeda&lt;/i&gt; (loblolly pine) and &lt;i&gt;Pinus elliottii&lt;/i&gt; (slash pine) was constructed by merging three previously published &lt;i&gt;P. taeda&lt;/i&gt; maps with a map from a pseudo-backcross between &lt;i&gt;P. elliottii and P. taeda&lt;/i&gt;. The consensus map positioned 3856 markers via genotyping of 1251 individuals from four pedigrees. It is the densest linkage map for a conifer to date. Average marker spacing was 0.6 cM and total map length was 2305 cM. Functional predictions of mapped genes were improved by aligning expressed sequence tags used for marker discovery to full-length &lt;i&gt;P. taeda&lt;/i&gt; transcripts. Alignments to the &lt;i&gt;P. taeda&lt;/i&gt; genome mapped 3305 scaffold sequences onto 12 linkage groups. The consensus genetic map was used to compare the genome-wide linkage disequilibrium in a population of distantly related &lt;i&gt;P. taeda&lt;/i&gt; individuals (ADEPT2) used for association genetic studies and a multiple-family pedigree used for genomic selection (CCLONES). The prevalence and extent of LD was greater in CCLONES as compared to ADEPT2; however, extended LD with LGs or between LGs was rare in both populations. The average squared correlations, r&lt;sup&gt;2&lt;/sup&gt;, between SNP alleles less than 1 cM apart were less than 0.05 in both populations and r&lt;sup&gt;2&lt;/sup&gt; did not decay substantially with genetic distance. The consensus map and analysis of linkage disequilibrium establish a foundation for comparative association mapping and genomic selection in &lt;i&gt;P. taeda&lt;/i&gt; and &lt;i&gt;P. elliottii&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/08/01" />
<meta name="citation_mjid" content="ggg;5/8/1685" />
<meta name="citation_id" content="5/8/1685" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/8/1685" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/8/1685.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/8/1685.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/8/1685.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019588" />
<meta name="citation_pmid" content="26068575" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1685" />
<meta name="citation_lastpage" content="1694" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jared W. Westbrook" />
<meta name="citation_author_institution" content="School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410" />
<meta name="citation_author" content="Vikram E. Chhatre" />
<meta name="citation_author_institution" content="Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6627-2832" />
<meta name="citation_author" content="Le-Shin Wu" />
<meta name="citation_author_institution" content="National Center for Genome Analysis Support, Indiana University, Bloomington, Indiana 47408" />
<meta name="citation_author" content="Srikar Chamala" />
<meta name="citation_author_institution" content="Department of Biology, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author_institution" content="Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida 32603" />
<meta name="citation_author_institution" content="Genetics Institute, University of Florida, Gainesville, Florida 32608" />
<meta name="citation_author" content="Leandro Gomide Neves" />
<meta name="citation_author_institution" content="RAPiD Genomics LLC, Gainesville, Florida 32601" />
<meta name="citation_author" content="Patricio Muñoz" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Florida, Gainesville, Florida 32608" />
<meta name="citation_author" content="Pedro J. Martínez-García" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, Davis, California, 95616" />
<meta name="citation_author" content="David B. Neale" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, Davis, California, 95616" />
<meta name="citation_author" content="Matias Kirst" />
<meta name="citation_author_institution" content="School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410" />
<meta name="citation_author" content="Keithanne Mockaitis" />
<meta name="citation_author_institution" content="Pervasive Technology Institute, Indiana University, Bloomington, Indiana 47408" />
<meta name="citation_author" content="C. Dana Nelson" />
<meta name="citation_author_institution" content="Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574" />
<meta name="citation_author" content="Gary F. Peter" />
<meta name="citation_author_institution" content="School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410" />
<meta name="citation_author" content="John M. Davis" />
<meta name="citation_author_institution" content="School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410" />
<meta name="citation_author" content="Craig S. Echt" />
<meta name="citation_author_institution" content="Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574" />
<meta name="citation_author_email" content="cecht@fs.fed.us" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9330-4051" />
<meta name="citation_reference" content="citation_journal_title=For. Ecol. Manage.;citation_author=M. A. Al-Rabab’ah;citation_author=C. G. Williams;citation_title=Population dynamics of Pinus taeda L. based on nuclear microsatellites.;citation_pages=263-271;citation_volume=163;citation_year=2002;citation_doi=10.1016/S0378-1127(01)00584-9" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=B. S. Baltunis;citation_author=D. A. Huber;citation_author=T. L. White;citation_author=B. Goldfarb;citation_author=H. E. Stelzer;citation_title=Genetic analysis of early field growth of loblolly pine clones and seedlings from the same full-sib families.;citation_pages=195-205;citation_volume=37;citation_year=2007;citation_doi=10.1139/x06-203" />
<meta name="citation_reference" content="citation_journal_abbrev=Biol Direct;citation_author=G. M. Boratyn;citation_author=A. A. Schäffer;citation_author=R. Agarwala;citation_author=S. F. Atschul;citation_author=D. J. Lipman;citation_title=Domain enhanced lookup time accelerated BLAST.;citation_pages=12-12;citation_volume=7;citation_year=2012;citation_pmid=22510480;citation_doi=10.1186/1745-6150-7-12" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. R. Brown;citation_author=G. P. Gill;citation_author=R. J. Kuntz;citation_author=C. H. Langley;citation_author=D. B. Neale;citation_title=Nucleotide diversity and linkage disequilibrium in loblolly pine.;citation_volume=101;citation_year=2004;citation_pmid=15477602;citation_doi=10.1073/pnas.0404231101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. R. Brown;citation_author=E. E. Kadel;citation_author=D. L. Bassoni;citation_author=K. L. Kiehne;citation_author=B. Temesgen;citation_title=Anchored Reference Loci in Loblolly Pine (Pinus taeda L.) for Integrating Pine Genomics;citation_pages=799-809;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606554" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=V. E. Chhatre;citation_author=T. D. Byram;citation_author=D. B. Neale;citation_author=J. L. Wegrzyn;citation_author=K. V. Krutovsky;citation_title=Genetic structure and association mapping of adaptive and selective traits in the east Texas loblolly pine (Pinus taeda L.) breeding populations.;citation_pages=1161-1178;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0624-x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=W. P. Cumbie;citation_author=A. J. Eckert;citation_author=J. L. Wegrzyn;citation_author=R. Whetten;citation_author=D. B. Neale;citation_title=Association genetics of carbon isotope discrimination, height and foliar nitrogen in a natural population of Pinus taeda L.;citation_pages=105-114;citation_volume=107;citation_year=2011;citation_issue=2;citation_pmid=21245892;citation_doi=10.1038/hdy.2010.168" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. S. Echt;citation_author=S. Saha;citation_author=K. Krutovsky;citation_author=K. Wimalanathan;citation_author=J. Erpelding;citation_title=An annotated genetic map of loblolly pine based on microsatellite and cDNA markers.;citation_pages=17-17;citation_volume=12;citation_year=2011;citation_pmid=21269494;citation_doi=10.1186/1471-2156-12-17" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Eckert;citation_author=J. van Heerwaarden;citation_author=J. L. Wegrzyn;citation_author=C. D. Nelson;citation_author=J. Ross–Ibarra;citation_title=Patterns of population structure and environmental associations to aridity across the range of loblolly pine (Pinus taeda L., Pinaceae).;citation_volume=185;citation_year=2010;citation_pmid=20439779;citation_doi=10.1534/genetics.110.115543" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Endelman;citation_author=C. Plomion;citation_title=LPmerge: An R package for merging genetic maps by linear programming.;citation_volume=30;citation_year=2014;citation_pmid=24532720;citation_doi=10.1093/bioinformatics/btu091" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=S. A. Flint-Garcia;citation_author=J. M. Thornsberry;citation_author=E. S. Buckler;citation_title=Structure of linkage disequilibrium in plants.;citation_volume=54;citation_year=2003;citation_pmid=14502995;citation_doi=10.1146/annurev.arplant.54.031902.134907" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=B. S. Gaut;citation_author=A. D. Long;citation_title=The Lowdown on Linkage Disequilibrium;citation_pages=1502-1506;citation_volume=15;citation_year=2003;citation_issue=7;citation_pmid=12837942;citation_doi=10.1105/tpc.150730" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection using low–density panels.;citation_volume=182;citation_year=2009;citation_pmid=19299339;citation_doi=10.1534/genetics.108.100289" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. R. Henderson;citation_title=A simple method for computing the inverse of a numerator relationship matrix used in the prediction of breeding values.;citation_pages=69-83;citation_volume=32;citation_year=1976;citation_doi=10.2307/2529339" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. Krutovsky;citation_author=M. Troggio;citation_author=G. Brown;citation_author=K. Jermstad;citation_author=D. Neale;citation_title=Comparative Mapping in the Pinaceae;citation_pages=447-461;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454556;citation_doi=10.1534/genetics.104.028381" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Lamesch;citation_author=T. Z. Berardini;citation_author=D. Li;citation_author=D. Swarbrek;citation_author=C. Wilks;citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.;citation_pages=D1202-D1210;citation_volume=40;citation_year=2011;citation_pmid=22140109" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=C. Maliepaard;citation_author=J. Jansen;citation_author=J. W. Van Ooijen;citation_title=Linkage analysis in a full–sib family of an outbreeding plant species: overview and consequences for applications.;citation_pages=237-250;citation_volume=70;citation_year=1997;citation_doi=10.1017/S0016672397003005" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=B. Mangin;citation_author=A. Siberchicot;citation_author=S. Nicolas;citation_author=A. Doligez;citation_author=P. This;citation_title=Novel measures of linkage disequilibrium that correct the bias due to population structure and relatedness.;citation_pages=285-291;citation_volume=108;citation_year=2012;citation_issue=3;citation_pmid=21878986;citation_doi=10.1038/hdy.2011.73" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Marchler-Bauer;citation_author=C. Zheng;citation_author=F. Chitaz;citation_author=M. Derbyshire;citation_author=L. Geer;citation_title=CDD: conserved domains and protein three-dimensional structure.;citation_volume=41;citation_year=2013;citation_pmid=23197659;citation_doi=10.1093/nar/gks1243" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=P. Martínez-García;citation_author=K. Stevens;citation_author=J. Wegrzyn;citation_author=J. Liechty;citation_author=M. Crepeau;citation_title=COMBINATION OF MULTIPOINT MAXIMUM LIKELIHOOD (MML) AND REGRESSION MAPPING ALGORITHMS TO CONSTRUCT A HIGH-DENSITY GENETIC LINKAGE MAP FOR LOBLOLLY PINE (PINUS TAEDA L.).;citation_pages=1529-1535;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0646-4" />
<meta name="citation_reference" content="citation_journal_title=J. For.;citation_author=S. McKeand;citation_author=T. Mullin;citation_author=T. Byram;citation_author=T. White;citation_title=Deployment of genetically improved loblolly and slash pines in the South.;citation_pages=32-37;citation_volume=101;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=P. R. Muñoz;citation_author=D. A. Huber;citation_author=J. R. Butnor;citation_title=Phenotypic analysis of first–year traits in a pseudo–backcross {(slash x loblolly) x slash} and the open–pollinated families of the pure–species progenitors.;citation_pages=183-192;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-010-0324-8" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=M. W. Nachman;citation_title=Variation in recombination rate across the genome: evidence and implications.;citation_pages=657-663;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433578;citation_doi=10.1016/S0959-437X(02)00358-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Neale;citation_author=J. Wegrzyn;citation_author=K. Stevens;citation_author=A. Zimin;citation_author=D. Puiu;citation_title=DECODING THE MASSIVE GENOME OF LOBLOLLY PINE USING HAPLOID DNA AND NOVEL ASSEMBLY STRATEGIES.;citation_pages=R59;citation_volume=15;citation_year=2014;citation_pmid=24647006;citation_doi=10.1186/gb-2014-15-3-r59" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=D. B. Neale;citation_author=O. Savolainen;citation_title=Association genetics of complex traits in conifers.;citation_pages=325-330;citation_volume=9;citation_year=2004;citation_issue=7;citation_pmid=15231277;citation_doi=10.1016/j.tplants.2004.05.006" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=N. Pavy;citation_author=M.-C. Namroud;citation_author=F. Gagnon;citation_author=N. Isabel;citation_author=J. Bosquet;citation_title=The heterogeneous levels of linkage disequilibrium in white spruce genes and comparative analysis with other conifers.;citation_pages=273-284;citation_volume=108;citation_year=2012a;citation_issue=3;citation_pmid=21897435;citation_doi=10.1038/hdy.2011.72" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=N. Pavy;citation_author=B. Pelgas;citation_author=J. Laroche;citation_author=P. Rigault;citation_author=N. Isabel;citation_title=A spruce gene map infers ancient plant genome reshuffling and subsequent slow evolution in the gymnosperm lineage leading to extant conifers.;citation_pages=84-84;citation_volume=10;citation_year=2012b;citation_issue=1;citation_pmid=23102090;citation_doi=10.1186/1741-7007-10-84" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Platt;citation_author=B. J. Vilhjálmsson;citation_author=M. Nordborg;citation_title=Conditions under which genome–wide association studies will be positively misleading.;citation_volume=186;citation_year=2010;citation_pmid=20813880;citation_doi=10.1534/genetics.110.121665" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Plomion;citation_author=E. Chancerel;citation_author=J. Endelman;citation_author=J.-B. Lamy;citation_author=E. Mandrou;citation_title=Genome–wide distribution of genetic diversity and linkage disequilibrium in a mass–selected population of maritime pine.;citation_pages=171;citation_volume=15;citation_year=2014;citation_pmid=24581176;citation_doi=10.1186/1471-2164-15-171" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_author=M. Przeworski;citation_title=Linkage disequilibrium in humans: models and data.;citation_pages=1-14;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11410837;citation_doi=10.1086/321275" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Quesada;citation_author=V. Gopal;citation_author=W. P. Cumbie;citation_author=A. J. Eckert;citation_author=J. L. Wegrzyn;citation_title=Association mapping of quantitative disease resistance in a natural population of loblolly pine (Pinus taeda L.).;citation_volume=186;citation_year=2010;citation_pmid=20628037;citation_doi=10.1534/genetics.110.117549" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Raj;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=fastSTRUCTURE: variational inference of population structure in large SNP datasets.;citation_volume=197;citation_year=2014;citation_pmid=24700103;citation_doi=10.1534/genetics.114.164350" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=J. J. Acosta;citation_author=G. F. Peter;citation_author=J. M. Davis;citation_title=Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.;citation_pages=617-624;citation_volume=193;citation_year=2012a;citation_issue=3;citation_pmid=21973055;citation_doi=10.1111/j.1469-8137.2011.03895.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Munoz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard data set of loblolly pine (Pinus taeda L.).;citation_volume=190;citation_year=2012b;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_title=Genetics, Genomics and Breeding of Conifers;citation_title=Genetic mapping in conifers;citation_author=K. Ritland;citation_author=K. V. Krutovsky;citation_author=Y. Tsumura;citation_author=B. Pelgas;citation_author=N. Isabel;citation_pages=196-238;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Density Estimation for Statistics and Data Analysis;citation_author=B. W. Silverman;citation_year=1986" />
<meta name="citation_reference" content="citation_title=Forest resources of the United States: a technical document supporting the Forest Service 2010 RPA assessment. US Department of Agriculture, Forest Service;citation_author=W. B. Smith;citation_author=P. D. Miles;citation_author=C. H. Perry;citation_author=S. A. Pugh;citation_year=2007" />
<meta name="citation_reference" content="citation_title=JoinMap 4.1, Software for the Calculation of Genetic Linkage Maps;citation_author=J. W. Van Ooijen;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Wegrzyn;citation_author=J. Liechty;citation_author=K. Stevens;citation_author=L. Wu;citation_author=C. Loopstra;citation_title=Unique Features of the Loblolly Pine (Pinus taeda L.) Megagenome Revealed Through Sequence Annotation;citation_pages=891-909;citation_volume=196;citation_year=2014;citation_issue=3;citation_pmid=24653211;citation_doi=10.1534/genetics.113.159996" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Wenzl;citation_author=H. Li;citation_author=J. Carling;citation_author=M. Zhou;citation_author=H. Raman;citation_title=A high-density consensus map of barley linking DArT markers to SSR, RFLP and STS loci and agricultural traits.;citation_pages=206-206;citation_volume=7;citation_year=2006;citation_pmid=16904008;citation_doi=10.1186/1471-2164-7-206" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. W. Westbrook;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=A. R. Walker;citation_author=J. L. Wegrzyn;citation_title=Association genetics of oleoresin flow in loblolly pine: discovering genes and predicting phenotype for improved resistance to bark beetles and bioenergy potential.;citation_pages=89-100;citation_volume=199;citation_year=2013;citation_pmid=23534834;citation_doi=10.1111/nph.12240" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. W. Westbrook;citation_author=A. R. Walker;citation_author=L. G. Neves;citation_author=P. Muñoz;citation_author=M. F. R. Resende;citation_title=Discovering candidate genes that regulate resin canal number in Pinus taeda stems by integrating association genetics and QTL analysis across environments, ages, and populations.;citation_pages=627-641;citation_volume=205;citation_year=2015;citation_pmid=25266813;citation_doi=10.1111/nph.13074" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. C. J. Wientjes;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_title=The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.;citation_volume=193;citation_year=2013;citation_pmid=23267052;citation_doi=10.1534/genetics.112.146290" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Wu;citation_author=C. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Computational Biol. Bioinformatics;citation_author=Y. Wu;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Accurate construction of consensus genetic maps via integer linear programming. IEEE/ACM Tran.;citation_pages=381-394;citation_volume=8;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Yan;citation_author=T. Shah;citation_author=M. L. Warburton;citation_author=E. S. Buckler;citation_author=M. D. McMullen;citation_title=Genetic characterization and linkage disequilibrium estimation of a global maize collection using SNP markers.;citation_pages=e8451-e8451;citation_volume=4;citation_year=2009;citation_issue=12;citation_pmid=20041112;citation_doi=10.1371/journal.pone.0008451" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Zimin;citation_author=K. A. Stevens;citation_author=M. W. Crepeau;citation_author=A. Holtz-Morris;citation_author=M. Koriabine;citation_title=Sequencing and Assembly of the 22-Gb Loblolly Pine Genome;citation_pages=875-890;citation_volume=196;citation_year=2014;citation_issue=3;citation_pmid=24653210;citation_doi=10.1534/genetics.113.159715" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="twitter:description" content="A consensus genetic map for Pinus taeda (loblolly pine) and Pinus elliottii (slash pine) was constructed by merging three previously published P. taeda maps with a map from a pseudo-backcross between P. elliottii and P. taeda . The consensus map positioned 3856 markers via genotyping of 1251 individuals from four pedigrees. It is the densest linkage map for a conifer to date. Average marker spacing was 0.6 cM and total map length was 2305 cM. Functional predictions of mapped genes were improved by aligning expressed sequence tags used for marker discovery to full-length P. taeda transcripts. Alignments to the P. taeda genome mapped 3305 scaffold sequences onto 12 linkage groups. The consensus genetic map was used to compare the genome-wide linkage disequilibrium in a population of distantly related P. taeda individuals (ADEPT2) used for association genetic studies and a multiple-family pedigree used for genomic selection (CCLONES). The prevalence and extent of LD was greater in CCLONES as compared to ADEPT2; however, extended LD with LGs or between LGs was rare in both populations. The average squared correlations, r2, between SNP alleles less than 1 cM apart were less than 0.05 in both populations and r2 did not decay substantially with genetic distance. The consensus map and analysis of linkage disequilibrium establish a foundation for comparative association mapping and genomic selection in P. taeda and P. elliottii ." />
<meta name="og-title" property="og:title" content="A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/8/1685" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A consensus genetic map for Pinus taeda (loblolly pine) and Pinus elliottii (slash pine) was constructed by merging three previously published P. taeda maps with a map from a pseudo-backcross between P. elliottii and P. taeda . The consensus map positioned 3856 markers via genotyping of 1251 individuals from four pedigrees. It is the densest linkage map for a conifer to date. Average marker spacing was 0.6 cM and total map length was 2305 cM. Functional predictions of mapped genes were improved by aligning expressed sequence tags used for marker discovery to full-length P. taeda transcripts. Alignments to the P. taeda genome mapped 3305 scaffold sequences onto 12 linkage groups. The consensus genetic map was used to compare the genome-wide linkage disequilibrium in a population of distantly related P. taeda individuals (ADEPT2) used for association genetic studies and a multiple-family pedigree used for genomic selection (CCLONES). The prevalence and extent of LD was greater in CCLONES as compared to ADEPT2; however, extended LD with LGs or between LGs was rare in both populations. The average squared correlations, r2, between SNP alleles less than 1 cM apart were less than 0.05 in both populations and r2 did not decay substantially with genetic distance. The consensus map and analysis of linkage disequilibrium establish a foundation for comparative association mapping and genomic selection in P. taeda and P. elliottii ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
    <title>A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lTd5Rzy_7RcJU7JNXs3_vH-4CVi5qlMvOG_JRbUsQXQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459993","apath":"\/ggg\/5\/8\/1685.atom","pisa":"ggg;5\/8\/1685","ac":{"\/ggg\/5\/8\/1685.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/8\/1685.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1685"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1685"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/8\/1685.full":true,"\/content\/5\/8\/1685":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CVDv0Dj7URCk3bDn8M4p5nHwbiCi75oUvXriRM1hGj0ZFBo+wdRCTgR9GuEpW9mRg9CMgzBuC\\\\\\\/hBnNW1YEdlBdHER4gYq835CwQy7YwT14F+I1Z0fPTbnXA9DPJaftFk3M54xz1ge7cMfwHP\\\\\\\/GAxYHnz4ngtgTnNsPyBm85vfFUguFMn69uvs4R1T\\\\\\\/rKmBkUgAdlUFHdyy7qaq+FqWFJERNokpKOmZWDvjfD0QXNIkqttVGN2yDuYYK5vHjTd71gwwir8OjP78Fu6YuzxwsXsOgdUlaRH0kEm7NTWiA4u0vWfAjKl6TrDAp0bXfUiAyqMfQNPIuSzWxFgOmE\\\\\\\/6dEyOA3N41aWOr2bDgObIxgvwsUDK2GZzGgzWSWDTUaW7uwBB9toeTK+nkp2KDL4tybd1k69v2vXI5kX3UTCqoN8mYd0\\\\\\\/m7OphDc1zPkkOjNdiwM\\\\\\\/eaAcafjHX8U2gJXT8FnnlM32BPXghdAMW8wyS+y9nmU8i4x6KMaaHK9ohIxLfVjLBW2NYud11ATJ\\\\\\\/q2nVgVaTeLGm8OMblbbXsXUbOCrkW0GQOPCv7Lhk8\\\\\\\/A7TgDZ3ebw1o9dA\\\\\\\/Dfa9D7Jq5UWMb6\\\\\\\/xDrmCr1K32n0pNp4+ODXIkWORMgrsTEKs6Qgexs1LmZcENd+WMoYupEFQo9A1hIVXKrsSUGkaq9dltpBohKXBjbtYKtEN2oGxxBLT09ZsKFQ4EFXDie4g8Vtr67SJAnH4STcUvE3vyVf0HvMTMCVbWYxLXMdkIFTIVl\\\\\\\/tgfg0JwkMnalLuAyRk9cz0KQ+0UC\\\\\\\/CIOto\\\\\\\/X0md+LrRZ\\\\\\\/irOsnv5iBK5+oIe5fSleT5X7bjZ4BSJW8V85SyyevDfMae85hOHGRwXBAtj6D5NGjQkcUkf5H6vi5quSkQbef+bBJrs9PDgeyQnoxqFVEm68i1I6N3HcZ\\\\\\\/0c7Wxsk7PcWc5F6\\\\\\\/516V6hRTpsBejRExjLS6om4zfNXJnEbGIdm+P\\\\\\\/8rw7bFcLTqFGVPL16vHz0VP1crH8A4nHwm01McaY50Z7qQmmz6I2Rnlz1niITwVy+Ptx1pEIj9aGzSHmGaWuZb5k1O45V1REUeGDdhOIwWVrpPDccbNP9253n4PbQ1oDQeRGqX6HkJCwD\\\\\\\/Ma36EhEDyDXh28sUK1PC+EmcCQob2e2ZY6W\\\\\\\/M0WdurzQNMzvpr\\\\\\\/D7SgujH2migxq9f2g5eCsF\\\\\\\/XKZfDbTe9iMenOekLuQ9sVghsuSYBP3SM8nTrKTVobkv35WnId3\\\\\\\/T87cDIPE9rGi8YXrUAmd2p20ebbi3ZN\\\u0022,\\\u0022iv\\\u0022:\\\u0022KdOtYYuqXquvk0SSCI3MBg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e42e4e1e1efda58ddbdcbb7a3e4da5c9c2232a70c7bb56d81a8cc615b2bd723b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yja5R7cqKh4vc8\\\\\\\/rTKJPoP+aMNbhSrMRG5j3BYuiEiVV4srmzK\\\\\\\/UWGpsJrBdk5cE+o\\\\\\\/BJ9tEYjQNjli3sySJWxV\\\\\\\/z+cJUeE\\\\\\\/eykbYaEPuio4UCZxNnDwVoyAau\\\\\\\/eUggGlgyZAlGYEXqFvwn2\\\\\\\/5NRmQEJNJy797dXisxxARicyMI8rd+bEN8ddRoGAhfYsHjFXQK1hYhC4fDBtUNfv31ScqfKqB9j\\\\\\\/XIEfH55N0wmNGA6lHJXg9QNOII6rAKh\\\\\\\/m7Dwa2hNRyF30mnQ7aPFLSveNy2\\\\\\\/2vbGdIX\\\\\\\/LQPnhwgO8RACvL7W83i25d4rgkfC6MtlPolLFhWTyyAXH+8VqHZ8LzLWuVfdg7zmj4g5tj2Cf9SIpbqifiVH6ABBko9TE0k8ab88EO+tXq6kMOP489OcNTIkkPvPQbWsZ1RxZuWBxalzygXrbeqzlAGgf8yaOAxpv4UkWnaHjkUzdm4d3iSGVzQt9JO6ufCP6rBYU4fWPi8jtr3rfB0E3nBwYjfOlE7gY1ONFugE\\\\\\\/9b6FqfQLtx\\\\\\\/gQAPCQghRYfkhFMxFTTUYVDZHQeKUH\\\\\\\/YLxjd7GoErRvhmpwwCu1u4YfZuQBwnsylh8UIRXncI6goh3eKfGqc6nPywEpj\\\\\\\/Mxlwh91XiMkF+vGjLeBFpYLap2GUCkeBgDLzBMFEtLqEK6emHFzrhntJGlO3Bn3eB2647txJFbIdL0TCkPvCod43DcgpPMXW560hoJEfQH+xIG3h+r\\\\\\\/gUqDxkQ6wE\\\\\\\/uqbzgRyeBfgtK2+gIoaYvcTKgyne1hDM9YsTjDEtijssmLz8fdJ+hgHrAoqssQna8mQFSM3hf\\\\\\\/yqpjBKNxEEz5Lix+nqGZc\\\\\\\/dT+lVGVC2vXb9CqNPnr3D+KcG4ayMIc\\\\\\\/7HtJs3C56xTS4BsEA1mn+QgmDiuBoC7WXgb+R991901QIAI+tJcLhd0deWccRkSaed7MevGjUt0FoWDL2BJhkLJozvCp+FAfdK1uzWj4YcgZjEiJc3z+oU9mRjYWDQxmBGiKTWH50KN9wGTCHd\\\\\\\/KyNdV6gny371h9onSUIFABktxMROwPum2BYZxb6grItY8L4w7gcX8XdefbAt\\\\\\\/VEhNYcjzaFheTJlcRHl8mVkjMyTXVYqYSYteeeRhVBu4ce7rNFCegUt5JpQU7GYKU0tgDKEC2W0MclWT6qHwFPNO3HtvY0+ExDnztiT9K5WJwU43o0L7ZJMqhgDt3sc0YVxBLw964yGy2JLTMzHZ3oxz9mhNXdEk24yGwLvKliU5SswylNKJssOxx9Msm1umSTbTsNFZ+hU8SylkrY8VH9qPPwNyB8Y3ZbdXlYQinlIAHm8jcegJCmLCq1J594Wy2vze0TcLuq1B0GDOMwjNzBDuni1PJdzaP0EWP4C8zpUcP1i9EcrdzRW2tbCbPVQJN0GryqzA84ZgrPpfMEvLgyQfSACWBj\\\\\\\/Jwzfx4eIuAJBM57TPd214ZTUXfWgScx+l7EDn\\\\\\\/p1pOaQJkJiPPxXkKUTcESagfooQKJnQA8QDx55zNHrj2DTfVz0UwUfKZwUHMQdorn95V65HeST\\\\\\\/ka8p84jVww\\\\\\\/8yYHt5CBa8dzFe0FDLBPLYy75TXrKroW3s5fBn1T6xsDn3M1thrF5xmKq+By0hypV5ObtRT9QzOPxQbJxdJhX8BICSbyPc1vsUggrwzDiDc5h+y2VRVHBwGAWOo55KtRmMk3dIOL21tQEyQD84l2lkXMNAHzEh2fJwgq1mpTJh23+xJmE65G\\\\\\\/kUTkSyUyZJ30TYLPEY4mQsUx1YlqyAl\\\\\\\/qzUOjM8tiaIHI8Lp4AQmWRj7DP16Ccprba2VE7k+iN6jzcmc\\\\\\\/89Vs09pbUxEkjSHhM33SvXYBpfMNmtwY8bgn28qoslClAovSLaV\\\\\\\/Bj5RMf153WcpmAAxFCAquQfyWSmwz91XZj60WnXaaTAhxtysKXBdY9Y5ItLA9b99cFY3lOA5gxh2s8Uu7rhRNogkI+5UxLZHAG4DBZ2CQBlMasmDbcTM+1qmXoHyKZhZEm\\\\\\\/v\\\\\\\/Cu39xSu3qheAbH1NMOgNC+ZekLm5n+PqWgDricSRoj3LcfLiOcaj6xf3QmmprhwEzEWyn+F6Spnk773yu4e9hfc1W3EYsYsfIhfgPmx\\\\\\\/zi5hvWoA0Ew4rl+dLG0m\\\\\\\/l6+LPUJpZwgUefJowL1Bm9pENBg4DHTIVOyzhUYzX+\\\\\\\/eWmwioVRk9x5YQjyJ5yq+yfqVKVDuwzgav9xUL6acb1jkR\\\\\\\/zCPEf13cedYgkfpcdKVPZs3VxlDxJg5WHg6l05yUp69aGVm7TvWZ4Ca6tKy805evcECic+ac1qM067er9O1eQnckmhM+Zmlho90py0qSEw4bfR3fcSUbuiXAzoQnb\\\\\\\/b6y0\\\\\\\/oUoahLvyRQy3SvmPgP\\\\\\\/XtJRjEUXKxyghYS5ftEwFgYzrtf7V1ODkuZsVAF\\\\\\\/2p\\\\\\\/jbC1BdbXsj7Jr2hhqJJTQXZAhtGcSOsf+vx1JL1AlQ3J4uw5HZftw1a2MtBylEQ94priAr4kcJH1QQYzZTrr1x2vX\\\\\\\/LOavE7EQhNtyJa7LzJIGdn0M0+A09J499KDlalwAzCwizmCUvoSrNUz5IKSX9H+vis3m2psmt8BySFZhjdvnbtKQ0QXtsJghFPLEhp+TJ556knfzsGeuMaD\\\\\\\/EEszADxyeMukl2rBP1rwDeBV+a\\\\\\\/S8D9HNdQNG+FZ1VwQ9EsfEZuw7rVJxq4erELI6sDJWYsz1hJ4QiOLTgy3KHh8RoqykE3st8EUOOJjjES5VLHEpko\\\\\\\/8n1NUAtgZS1C+\\\\\\\/enQooFBLzm6YU3uGzBtsq7pjCKZM1xZuUR2Fxa5jpwGHk5E2AxyqaJtOh1mdQCFJ2HRCjlU+RTMJOv3d7Y1wHD83GkIoGGGI7lL3yHwC5K\\\\\\\/kw04T1LCTNsZEV9vdHMNaZfbM6Nt6EScyE9Njije4qw2\\\\\\\/zod95hLoiwj4zCj941RmfjA1KOAXcy4K6ZBZhna4TcRVlmuLAJhvPYrFbXQO8Objt364MwkmayiXjdalq2Zkp8xYQepVZ\\\\\\\/OvLhjd148b3J23yjTOwwWrfYRo682CHxwbFn9xwK68sGDdS91EQmR5l+DyMbpHzTvurPcSdeIWYzwgOyT\\\\\\\/BzaQybJPlAAA1N\\\\\\\/EPbgR\\\\\\\/29TKhcAtAGwWIxLlKIO1YOpNf2ZamOYAulDaFSYCAI08pvgQQyH\\\\\\\/coK0nZfTtbr7AeyQZDNOwRJj7eJJN7PBmYN6cwbpJmqcpuq\\\\\\\/B1DwrrsOfxaWQOoU2ICbyIMEqu6bEu7+vwQMclHRWvYjiOsKAMWRyxZ1QPZUBtsr0WP+l86YREB\\\\\\\/VKEpJhuxZbhg6eLSfmNXUqyFcOucj53Emr\\\\\\\/QsZeKxBrXIcxloxx3ay+RR4V\\\\\\\/VcxLTDCjrCJI2Uza6f7+HpCHWf2n5\\\\\\\/Dfiin+0RO9zn8lyz9bn\\\\\\\/nwrBT5MAHBuJG0da3\\\\\\\/SR8sMQoH4t+re4b1j1Q1RHMseYmS3XJrdGYmqpkR+jr0c4B13NrSSzWhzlMV1CUXtC6oxz\\\\\\\/aGzD\\\\\\\/NMu6TT8kh5\\\\\\\/aU\\\\\\\/FXsHUp1IUwIoLGnJErpfy9EoQACaQnlwUzD1ggkDn766l91GLcLknYtKwonK9jyTtUHAgMDVpm0rFl77x7ajgjPdJcwJKbkGgMGuN3nvb5SYuYK7lRtEY2yDding\\\\\\\/YEbXU2rj8y+dVhDJ+9YPfSgXvQMAE0HT6GZlLvYoYUamRIMV6qoLC0Rq50qBBW3TVBtsavjFXNnwwG9S2SXK+dF11Y6Q4h60h5L51Cc1PwPgDuNW8mpgvlGr3Xdz87MVkVRXpgyqa24FIeS5KleXYtRVvvH8C++g42jWmLgIFaehBXSxON5Ju7l6wqeNnbPExC7Yyej0+UXR0Yr\\\\\\\/3L2rSWGRXScR9eUibJtotqVIhsbORvIbSh+ck2HdBUmIWLUFD3OAT3oMWjVeCyNjn8YWFNtCnFw26hURiqDctThY7uxWqN\\\\\\\/ucPGWRWLiQUavU7soNacRGUXpr0+Ds1efhfwD\\\\\\\/1VO1UPh9JqfEJOgmOoRzKuNF7E6g4yXEo6HN6RgP098P+k0VCxxz82QooOEn99utvS8hAAAFMu04V7vqyNXpBGqmjthGTCENQSFz5jd\\\\\\\/aoGoMVEgN8VhruLqA6H9FHuIazza+hG1Lo8BUdb3+ksxtTmNsQ2IHwLlmt9FnwtgP6qNFQpk98ErnxA76JF\\\\\\\/fHQvsBWth75maWgRAG\\\\\\\/mVgas7BQ6YJc1tVYfbsg9OnzDLNDeFwZEx9exz3lDXtPeERC1mZgS4ygrTzwmAYYjO9d+l4TYDBwMjXLR\\\\\\\/rJL5tWZL4ZcuAQy6m5bagqbMJMBYnlbPK2XOGSJMv+CIfvnS2HGNwKo5E0pdNHjA9aaEKjIaLhXIBnFyvrWbgAutMDAJm3DcxVONcwFYLEhvBO5KwHMoDyRIPit6JgMaKdvqbsORvK7mX264op3mIv+z7XDS0RDLSo1\\\\\\\/WHrnXH9jWEyn3NtaZgBwCi2azbTrugGaKvM2um\\\\\\\/4z96zi4GWe60bUtQH8M1H33cufFRPJK9O88Bb2nPBxWx+N7WynsToLbI8vRNc0iNEj2ODm\\\\\\\/4cl6YGroRrgSK2ZBApZavjW2KYA+kzKLMT21caMXQQHS3k0YyLt60xTr2EGuPOp6psQt3gIf7579jeHxPh1YXH4J7dWBZNl8cx8Z5gSBe0Y4\\\\\\\/czzSNfeuejI+IW64sljzZcueimNfaDxXe6FYZ6234vpWJ5NmqHluzP+jY6czXK6uB5Jj64z56OGBomG06Jo5mgX8JlCAL22r8TEjmowZfvjk+K6Z7T3tRUc08xu\\\\\\\/iCpGJ+3WJf1ybWqhc2t0Tf+9ClOrGyIJ0PCZS0oAbyclTLPmjaAqfchqMYA+2\\\\\\\/40zLmifcaf6CCKhauWSONUxBSmQX5gWH0OwjhgFbxaLeBi7cwXyYSjRCV5SbHNzTbhaSK9+iwo7Ft4RuaMrF+Jw32Ll96ObHFbfP3sfX1Q1TYJiqFqUDZMA7GtPDTri2xUtdfABkseMqH0h8Tvkq4socHjhQWNbEib5ZA++3bNs\\\\\\\/lbP3HHBAWS6SHr8Qy43fgGqJr7qXlnpBBf+3G5skG8Wkh3SMLfxpOXUY4RkVn8eSDtcSvAn+p4J9DQr2ugC0IyysWz2aHPKIAhaGnzGnKodx\\\\\\\/9PA3uAkJzfviltq7sxUoHhA66v\\\\\\\/WdFZrX3BoexgnOAm3Zu3qE5PZOObtcHohmKEbl\\\\\\\/qH9c3WbNyc9N3\\\\\\\/b6N\\\\\\\/OeFWfCIelOWVkEqTTwQRuOaxb7Hpipj7my57KIJbyyqvd1mfSpnuIqGJQtywxySwBtQD0H0E5qP3Zsmoo\\\\\\\/+\\\\\\\/x6DLSOuIhaU8DAoCkrjh8HbjMVxkmErSRVy\\\\\\\/DlSKqKH2PB9eKwySLg32E3eLpC8yDXvipmQMHxG9zfB1OuDbuONCdmBcPhIQHHr3EaTdlkflIGl0XaxSub4WgLHXIvP37zDeiVt3txbfxWJe98q4Am0yM9GlxfXvPm13qXebyXJlvJ+VUhotzFmLwRm345tKTIvKLla3j14KSEWjwSL7rmNGzpVxQbSActI\\\\\\\/FwM9ojbQNtrJgasGLHO8\\\\\\\/okc1CVsWzsP0939xa1GEIwDfhTUaPs6H\\\\\\\/e9Tdr8tjT0RoewLNBTiBMkRe1uXkFKJSYlJiQcD904VqTa8hhqMXZftb9dKflgenRuY01649jYJyMRMEQN\\\\\\\/Hho6QVhqH4p2rytle4RNLWVUmO1uPUvMlvPNlgeTmYVoh4E+EnoayDjpn6Vcz6zURtyKvwyiMcMxh06EQA8CvBsG84ZNRaL\\\\\\\/jNBnPibg+RnfIsFifI97TmSL3yZZZxoAP99oKdp+0DTlUiYd26k3hn7I4QxZ5t7zW8NSL27Z\\\\\\\/QrYb9twiIF0WvWtobkLxuDG2KtXUfjZPAxRIhT8qhG2X+1ma+f7JLSYN1dnLx9lSXve1hIYGFlmFEJjnjWOBhYJ6q+CYVbu4kDcvavOOLVvHhOzAsjgWtwerb18aLR1fbuDEycR2f00zHg5znvx+8Mgr48v4jAP+H7OBNpVnhDkupDysCRH1Dyqd0HKObBXc1ul5Y429tN01HKNLHzMGagluEMfUCwpqupnmyR0ieditxbonkfv2wzW2x3\\\\\\\/8dwUpil7bmUHD237gtNB+1LGeMnXVYqz1U9u4bvkV3eZiDK+\\\\\\\/h3y9O6usjLVoHDq9XnBiyE3UcE6crVciEgr2W7wgJKDseX7oPznNLY441IvJcQat7eeqcYoEfMQdx38MXUztKRxgKWGMuh8v3B9YBxtlcWzDPpfL6dJb6IjptiKlHkmsdYl\\\\\\\/zBSUMccVHv6PmoL6zVqi5a9X+jRMAILFyhlioLQvfplZGeLJjlrM+WV4S9xgdCUBve9PyEo9zXydkem2ujjZBiWgItMiIRt+THW0LsSUbzCMHppIUbZ8Hbu+w6qeTkGUsnOM+oMcOOdMnMs9KgKTdEn93iG+Ck2W\\\\\\\/vVUarWZvWL9GCVeOuRH4VddobuylexvjnHKZUuizKjurNqxDm7tCYt\\\\\\\/Y9EOIDHIeMpjGPmLoPxzWD98MPZRHwsM05LMlJPp3Epyr\\\\\\\/WeP\\\\\\\/8HOn+7N7\\\\\\\/22+70F\\\\\\\/vYr27uKrYs2rzaxaHxLndWMvLuT2wktPyiiqbjTjOUPoDPDKkPKVtgVkkN6GMyZ\\\\\\\/ngmTe1Wel93Op99bE4lka85O4f6\\\\\\\/LZq8m\\\\\\\/4z1WWVEtFCAOBvRD3WLnv5q8feYuDoH5kJ975M9DIrNRAW3qEZP56dc0tg\\\\\\\/DX4vst++sJIyD0OTYRrRMqKHhlYzXEDIf3GW1n3mgcicY8qbSPmena1H44TbfJf\\\\\\\/LhZtAoGvGcKRF\\\\\\\/d\\\\\\\/l9NYPlOeO3m3M5ZNSMlFi\\\\\\\/K0BDd75dAq20IGlab4sq4Cx7IA372zjEFo4+ntnD3R4I17MODJPBACqDcIyKA44HWnd2suYoEiEAqmbrAS4aJrKQH5sWwK6RqpJjFODUej8ugTNzYfFI8CLKPt82vkNsOPQ+1gu8yzhWnOvMdia1pva1E0p3YPtKk3PyuC9KM4a5SYky\\\\\\\/M8pijfIggnIOBEp1zmLhdNo9pAzoMfhNDFuTqcfc85SaQP55DkPukgvtn4qpPdBS\\\\\\\/cVhDrKslOh\\\\\\\/68ijbusaio1tynlafk4T+MllQMtnOsXHQ5wROibTGOedVUahZ7opuzEVqF66gvQrVO4yBEpIqPhp7zAFiszZJrgmbXYx757tbznQP3j9QU6BZFcMf2puK7FhocbYcwenz\\\\\\\/762Tpj9F3tibHp3rHQ1N4nlHlcmpxeFd4j\\\\\\\/SncgqZBTY5w0PZtCoVu775XIqVOFUy\\\\\\\/ln\\\\\\\/8r+KdW0c3Xk4+XCEgOAyhLuR0UPj9aVxZQlP7pcLqi25yo3TGxC+le3+i9dferCBWPkieIifN975cvdvKDZAvdNORdF\\\\\\\/6130fFbNh33yk304FoEkfD\\\\\\\/brZexON4MgsTpB\\\\\\\/2BlOyJPBs\\\\\\\/zzUh97wU4tbRgOfomvllJhx\\\\\\\/LjNpcPyJDZpz+C5\\\\\\\/0th0XO81P8pDPYMKaiBEHtwAF6jldnXz6w2i5QNkNPWc1JZNldSRpIvvy0IyBa4QpA3\\\\\\\/wC+CX7KNhYuticTqRZEqoPTdVuhg\\\\\\\/cvWA2oDWtblZ8xnweySk3uwaa+v9zIhK9m3wcZIs5tPi8GLzqs2h1hr2FgNqRHVle+vZcJ7lKZOHxNrriMx7SHxIES+4ouWJOP8gEpHBo0egWk+gYB2InXPg+WUkH49Ko8sOD7qX+8wgQa4P7P7vn0Wilx5cREg7z\\\\\\\/+tWRB0tcgK0cckvL5ZM2G9akAK4Wb6Vm\\\\\\\/bpIh45eZmETGfJAUxsOzjorfQ2wCmAnYKx7R\\\\\\\/hc4xKDKEDir6pTJrRZra\\\\\\\/h0\\\\\\\/43E52jjrQNVRlyzQisy8h2tPF29v6VRwpTBgeow8x5ZSZIHgDKOfZApxx0cv+chLsglUM8Avi\\\\\\\/A4FA8NkKQh3eD9L+VIgrKpO4C4Y+9AcBggtHy3ZMCoQlAPwmh\\\\\\\/iUX4iWvgHFzVRo92+2gDAe027hSprdPFfnfwakMadzxWHPP08cjZHsRfdXrJ2HBm47GN5oXbE5ozjaX5v1\\\\\\\/cWeNBbvImSaUuvE1ivZosozmsz0w2AuECeNYRsub2uCXzL6xJGKI9CEgfxBqVsNOqRAHfSDtqIAjXV02MmZ9+JQ75IXytqgtrl3IsG+bWuzz8CBNuhDB4iaflj9Ok4R7I9aH2GkNtxpaMXcwI+no\\\\\\\/Xja3tVhL2yFNcjzep5eio4TQFjyEjJ1fgmiDE3eY+wnr7lStzLExnF3LPFTigoJD5F\\\\\\\/ppvL44Xs3pQDvNMmmnmG7lMAl7cLFwtWgfNU+PlfS4cHgmz8e1peZt1moclm\\\\\\\/7djNlSwNBZy4wIoHLyEptWEoK866Uc4p\\\\\\\/PRrsH1\\\\\\\/DCP8d3jqZ6QYTaY\\\\\\\/w0VlNjfhntUrE0AUdBag6Ord3yPGTL5TJpCvzTujjTo9b3EtgODoe+f2FcPEHNcp\\\\\\\/zT0\\\\\\\/cEQIskm7NAIPmhgWyZreYXIJk+Hj7xLyxWutFFAd3iyBtEVbFy1E9TdWKm0SHzU\\\\\\\/D5XIpy3QwwDfZLs+2+XCoKZqdiwm5VV2DDWaKRoaH5p8ltvNIqrxlg0hTveo0XAjxB8kjXEhm0PPteE9aEAItYzyw7lyZbpDd57k0Gh9eZL5o3Zmj6\\\\\\\/ni2WtMQgodo0jsEXwOPcQSUk1LHLH6uocwD6tvpBO9uUZ4yFHnswpER7MeLGN6i\\\\\\\/NVNp5dQCvPTVz1rlF\\\\\\\/0lH86DkZTsw7OEcBJLhHwhAfyihx4V3gtOlvw3CFPg2U7apWC2jxvleI1iYwzK8aR+30bJomNz9WCjU\\\\\\\/0YgQzNFuckfzKxPO1aPvkQuF6YHYYVPlFaJrzpzpgQ2lowJpBJXpHMPyZuVSu+pOcp\\\\\\\/dz\\\\\\\/EjW4PrZBPykdlDcufrJVXjESO5Wgz5utuUD2Yutxuwisb3nyPAd8+JKTwVvWU2g8xiG0lxBleG5bpyCD2XdUu1WYdWyUJa4UTTl5fadil1vcWn8vPmEG\\\\\\\/+x2KDoeyIKg8gcUokpdDVonY5E4EXgIaxa9opEvbCcUjt2nlMmnuftubJZjrCzr+8Wn4ZrlFYji7vIg6e9OQaH0TlW9Jm0QnYhZxoYm9s8dTtQKN1R2vxpM7S4vQgrqGXMOrpKDrrkCsTFt97jWUErVBpWpNhJqZpxryr2ZWXUxYi0jRwFzcgIWCVzabhMTa0T1hfZO5pPG6I7CPQQSxfr0A26wb9hvF6smYTtGQZ+JBwBRm2r4Pdj9z5QjDh7a8nYw\\\\\\\/EonF594ZuL2xOX02IU8HH7vTVm5yVXmHS+acfyTks3ygbDpXmwqpDsyS9ORHPo5d7X29PE5Vg8qBoD6zKHsVznSPSrLsjzYLYCw5ycLK8CJDbVxpM1F9ZL39oPLZoOBkALixvKssFE+ByWdMoiRGe7xFWOiaQ+HbN\\\\\\\/5Rgoquroa1TltOy4sY3SnoEtdpvzo92xUdzYj9yAlj7NjpGwdozMW47008QKiqSRKrMQLA7xNKOgvyxFU3zQLiX3CWc0YF0PbLE+DtO9hCW5UGUJibS02Cv08flNjUkqA8VEvJaDmmeb2+19+8DHlugYZYl\\\\\\\/Wq41UhOZqWx4uGH6BUWmKQxmXL88P05QxGx2RqY\\\\\\\/Tfkgy8HKaIftYP8vBnVi4AfCM9yWhBRxxxhc5RspL120RIKcB+Kjhpa\\\\\\\/+7FvKtvSiwdfaGBKJxVRVNvFXM2pY35ZoKAL4vexxbftlHftDfPKVJeFERkmt6kVWThR1n\\\\\\\/SdgD2gn13AKQVpxVqubstDlFRuFVXE2yJchy4NDDePNzq3p4NpcUPf3JUjt5gKJBWa+SsUFKGV9uq\\\\\\\/lzm2LfC7AkCsiZjlI+iJBCc9Ll1Q7tjLklXEti3mCrGmrMUPdjHClT1sa+4LU5WFWfql5Lz4cHX9twSCI0w8TB5FcJrbzfWzwg41\\\\\\\/5HPLWyS+EcagoPHsfSmit4Rspgf6RUrwKhRwiARE8gwutks4NFg+gP8vmy1GnMXsVkLlYtf0i6jnHBXI8GD\\\\\\\/oovcrx4Y7x1gFv9AbsP2NYvpEg\\\\\\\/BeoxnceaLS+F4GrPMAvNq1zvmeBwWfYMFxt4joocGJG\\\\\\\/WXHimS5MxvOL44F\\\\\\\/tuvM6tFgRzLw1Eb0GpNgcvEuVP5ldVW0bex6w1Dc0r7EEZdVemBubkRipaV8UBQv\\\\\\\/nTGZVQQAc6PYdunYyzA0mNet+9qgtbuZj1aCvWdwnBqxtmqdAm5fv9f2Nmg6vlqfY0FxJEaWAAlECYfnPmT6vepRmFOBxigS4jCdipJQg6dSEKA\\\\\\\/6bgzeOrpCLzi1Cdy\\\\\\\/qYkFSrK4\\\\\\\/35lGEHU3YmW1Pv55u6SL26nRoutnKicEVYL7mvPCctgb9tCW6v6Zc+smviI1gWgiH5mhyT5VM6fBXJqhMyQMYVKJxiAHNzcbnhwxITzQI9dgNBPqrYifgRO1qUhaITD0nIY1iTG6kNGVzTwnca3dQPZ8YccfGxrZ+RHkadz0zd+1aFynGsMtJUxsZcHX5ASFb4iDpoN2U4r0PqhvXkXVjCzPaegSjA\\\\\\\/FQJaxliHX6ACAUCfRgeXzzOaAuhLYaKXOcPgAoSh9S7ZFE5w1RHkPtWMXytpFiGFBupAdrBjkXAS+2YLlzKiqQwoaHIlXmjOI0GQ2n4YWDGy88Le1nyl4MGTnTmYVUvTNcvDXN8gLdvY+DSGtFixpo1GxEonbP2GVMXMfnuRN3LRel9LNks0RAL4sW9FwxzcG+N1dRt6mr8i1eblzZ3a7W4QUOnbugTItBn2vdopY7J43XJORi3uoE1MmNVOc94DSATtJWu3M\\\\\\\/RocVVp54\\\\\\\/9HoES2yLzVZKszyfEFEe6jPl+WV9n\\\\\\\/DINLx0Y7x7nFyuWuq9Bu60IOtgLsd539Xq94ezNlifmYUH1Jjms0qYG8d+uXkJL3fwk9QlqGQq47DXZBuiUzvTg9XlgjEjyI9dzwdrgngvxbU4u61j6abrhGUOtXqfbXKfDLF\\\\\\\/RjT2X7tLxzRsb09fr9UKP6ty5VuIKc4Ts4jsDeDiPCHqGDse9Vb8E5PDKIf0vbgo\\\\\\\/DruGAONBpacDM251tZPHrdmqBDAcMaXFf96bWQp4K0zUyQEnZFu7KK0+63I83BTbkWt476COjgk0sPvdgonXHnQSaR26aYE2hfPuQdvZvfayL3yAhp2UnEghZ8iOoxyXghH+aOvqxPqAto508m\\\\\\\/rsarce0TPB2fdo+xc5AJbDYk+q1P6KW8wU8Tk9T4LKKevtN+OkXw5x36x43PQDpfLJG+ikQPia6IAviSg62DhC1Ar09ONqZSUpstu87qqyAbgx9rcCJrahKhIe8rBwSNbem91ZwBaHdB3TENkoCO3j1omL3DFlN0zuBuQ\\\\\\\/WYEYCpLaDLhOvsuYvtoZP1\\\\\\\/7qVI\\\\\\\/hwjvtTbyEVwRNe3hNMqfzpLtS\\\\\\\/9ULuJcQ3w8YZ2xFGP0n3kEv13F2jkm0IRuLoT7UMQ69PbsFsFZzD+MbRCHcq8NGzOSu7IZecex\\\\\\\/Rg0J2whUQbs8k1V3k3de\\\\\\\/Nry7o7kiNCeqrk7EvrCDZR32pPoP1GOMZwQo2yZ\\\\\\\/QH6bXAgO99\\\\\\\/9+upy4zTWWsL5W8RwKdaQz+hDNJ8uXNYg1FsC8vxSC+q7N8QBNavnRN2lf0hdW33aGwLFtCmOouWisTaNS6LNR63S9WihUOAKOTqD7O7LnRdigdiouBt5DLsMnJ0gE9bsEq8fGfzjvBxjVdjQbcuc989DfvvKes4hIoIxIvLppsK05\\\\\\\/0vKB1bgjg5CE6p9Rc\\\\\\\/IXSLb6L624FmOlRj9kuWgJRUhkHnd6v3\\\\\\\/4I\\\\\\\/9Ncb9HerZAJeFz1h3UlRNmiMTFxCQxiLd99VzbJtaxCrWeZTn5HFn\\\\\\\/CWk\\\\\\\/Db6omr7+xcXWa9\\\\\\\/jv1ST94Rvv\\\\\\\/OmGRFSk2nRz0miJP4Ve6MaPYPBvXVpOClxk1uBGfXlKbHQSqia4KkAoV17x0ubjsTG\\\\\\\/ePhNn9wHi2858f3QuWjQXXM1V1dH+pA73+pr6buCNMKY5IybC5aN+WldK3cYTUoxOGYzMD6kcp+kpYkWfVqKI0B\\\\\\\/FLrrm3P2Wqkh2fXPWtmN6QE6MAq1nw68oqhH5eDVuDRZT2VzPe0PpNCeD+eKA7M0Fzox8tU4\\\\\\\/YgmifVs\\\\\\\/+0uWvOr+tOfRiGMoFM5AobZELu12mAfaaXKkLtKNybwZXOuY0G+N018FufesA7zDYPYUPEO4pl4X0QjFTdB2Qq5UjrXg5Pg5B1bNEW7\\\\\\\/C7Th1o\\\\\\\/f++Bb\\\\\\\/fJBZrJXOMy6y0Y4MQeifhlwbd2B\\\\\\\/U11gWI9A2jfZKU8M814twxg5nxOiLjRMfczBil6qQO3pmqwpRaKsd2PiDn9oc6OFY7qSu2MUBNvveJLoH57VtOhq8a\\\\\\\/wl1Mixm7AUl2xF2b6X2krpiwR4EBTx1op0ti9NioaqSGN1e1HTB6FnlrCVUWVjZvEcSB6q3ucmY4M7DWZrTYT2eFYkzyf7j5B5p+I20v7HDs7NnoQGNxpl8idJoR\\\\\\\/JcFmEz5ZJ8cLZ1odlwVYJJNVkFQaGDrohW0lJaHoTa4orzYpBt2Rl7vIVhB+WRyyHu+jzg4wja50BUSZgG9rc9S4sfzMx6SUpaZJ7LeNnTOUhLP1KQWL8H3ddN8XvIV78Al6NXmnTncpMGXCQGM5WSrEXf2RDXpw6lmCtAdpkV3byuXEl1HzahFWdpk98kpOQ2x5ocLM42DiKVETzDSoI1eZfU0MHJLQgFmdqKsXyut9\\\\\\\/idjXbpVCziR+ix\\\\\\\/alTAuOZNLafQsnTt9qkcv\\\\\\\/YXKvU58GPrpuCJDgZIR4ENfpddDBY3pim2atmYrlSPt3anf4oRglhONilla3wsZaIft6P1+POhSUnl0JHHjjLyclbrUls1cre5kzhHExL1w1kuw2r6moOIkjEvAq6BBTjmxUzOiCSlZ+i8B2o52WNnlS2eyjBhacywtHEo+uQ1IVttvqNIiH3dhnvDw52wQC8pi\\\\\\\/mA9NfDRn759\\\\\\\/L0i5RdrE99nujHwnmCAPt\\\\\\\/AD7cZD2xUpLL33QXFxaTnPBvg1U4Jnz9eN8mioJn1LUVts\\\\\\\/fa0X+Dg50ULRncoo1l5hao0ISDxzjjNv+62ZLb7\\\\\\\/EBQMVvx8iPb0vi07UJ33LL6DVUjy4C8h4kIoP3z7IHIQ049Lqmu38CcRKLMex7mi4HfiBsCg\\\\\\\/svIcUTZNuhkbPfXNheMKoBlJnfDOz33Lw+GcNrBeu1r1uXoYADif5U73vX8fl70OaohjW4hG3n0MeKWM9ZCqTXVHVzN1dwrDay0yBD74++d\\\\\\\/JSbakEPrfDUj9+2Uz\\\\\\\/W1glEdEwPTv0uZKHKCypHt8l0pSsZUldCVpjN4I9pA1UEYZTje8YjrgkGkMoAWA61DcbZAL5\\\\\\\/atwcUIh7RBKfpM68C5fImUSFoCMCddVZ10iy6958tDsn1yf274TKI36bmhJnEtHtWH95LnlpXnMZOe6EGs+h7rGpYXucTe3OOCtMFnBfN2Ts1ZN9V+9nhQ2K6ZG65qIXjXX+BWWgzymfumea5VKOGjd4ffohb\\\\\\\/UT9m2ihfA4oCL7UkmgoYKVhhUz2fTZfukFWemtoVCEQcexziKTsiS+x4VFa0ugvZujI\\\\\\\/PqKogRC0FP371P\\\\\\\/MJeNu5fjkPmXoEpzkOlvPLNayNccsUlh+Wf9rnO3+h7maYPP+9N9JyVfxH2Irinx9mSCDd2kOFgIND9\\\\\\\/a8ZmZGBdJDIAZ+VWy8hnpXYBa\\\\\\\/P1yzykXoJLCsvuMzmmGFcDVYIrwph7HNAtt0bgMCYzGRin3SJ63mtg26Jrw+c26VAG281PAqqVWmRzZW9YwvyNk5He1yCGoj2ffX6c8POY1dPHkC4HmRyYNFWdc0dn+kXOerZrGMW5bAkKwxNquGeZXMLfIv3cC3uM3Jwho9swxSLARLb4cyNP+FvhMlAb512Esww48CCOAWNaaCAJ5v+m2PT920btQYW98Bbrx8OyooxSQgWsENdRn1yOcESqH1vdm6D9DbJPd6tJAm8ZFLQ82gqXXhJzOeLiXM9akfA1bkAw1YCqh22KmCu\\\\\\\/3FWcmqDH0K8lehoP9Hf014he5kn0hBNf\\\\\\\/dW0xTAyUb5idLMxO9YYqSsywzPoWm9I2bu4KQvSEMcBmpYOwYhoD3NcqLWXyMCo1WQOeXYsBi3JSlsYL0M07H1kW+cWMUQQtcJ1sRcr5gUqoExvzwQju2pL7mVJLB1KayNy0\\\\\\\/rAwFsDzLFFpMJ5RXS7FJohvaO9eRG6DvNu+AOSEZcBaAkd2PKe72FZiJGCvb7R+CDrW4EzeNLYjH95j3k+u+9gD1otr08YF2GnriQK9FwrC1EPafei0pZZ66O\\\\\\\/QjN9IWIWp3iqZdHykFlZupQYl+dWqhGa9vejrrGZ\\\\\\\/95\\\\\\\/MMVFR1lf+x2OHH\\\\\\\/9dnibdp0FL6Hsjz5xZv9Rq8eV\\\\\\\/N39CmyJoljjJ\\\\\\\/kg4QWSuqUFr2Lob3nEbQj9AqHPKjvtQ4TD3nyePf4f83VlBw8B2TztC+6\\\\\\\/J3QB2X2PsR1a1H3BdczWBukoaREPj5zo6o994xmrINCLpb5PxN6kBwtT9WMvz6jyzJlqHMIAaDOQ8U9cyYT4wRWWGwepI4WlphanMpgHoCR+QaBugi+GKeReGyYN6LFEJjZl+6qbC6uYsOsMsCw77ThsKg9XuWNkqtZfuKwBpxOPJ5rYRvTBt6aABFmrOXd\\\\\\\/NqJmtqpAya8wHXiPKqBNAPbQKPwg\\\\\\\/6lZi76dmkGTMt4FBfQy9J+oWdCeWH9xCo+YUvsJDjksS1uGJmUy282k2xG7Oo\\\\\\\/2GV\\\\\\\/\\\\\\\/l+1Uv8oOaTDUg8oHmahSkkOej9N9OuQzHrljBhvSFfdl3aBAznUXsveUfw5YhJM56iCNPKkYhmcfW3kc4TvWOysg9wbqYUwpkRQAFoKWT1sYeHn6112PCx9SrIXg8Uu\\\\\\\/ufQ\\\\\\\/xBFtFwncesJzRgWHZSNpy28Zyrdq+6PLgQxrya1f10nxTeva0rI+w1iaUMb2IL0ww7ZhbcVzEXY7fLkJpNYGhbFr0a9KU99nNa86N6mFXmJ+\\\\\\\/7p7+eUsccdJfAB2pSn8Culuyt8txiQufAmdOzv6JlBJOou0Uz\\\\\\\/+l\\\\\\\/6iP3o8YjPl0V\\\\\\\/tCNcaRaEPz9GJsMv4EPiG91NcVjPQkQKJSNgcSj4WRC3zHuSJhW6Yw5MUtCs7O4r1caK4GvwRT3fNHLTW1yDUZigrwYVLWn\\\\\\\/xb+mnvrbnp13UtyXlhBKHAK4\\\\\\\/7S436+uB4pceTjVjtw2h40\\\\\\\/9lVu7wypiy5NiUQy7YLK2qF7DNoqWwIqRg7fFhTdT7fKEZftMouet527hxZVwFA2Hq4B92oUYwJNTQ\\\\\\\/OjX3m1aKmcdW02+wT8R4Zmb+ZCZ2vobkXjp7sgSSiDvaS5yMA3TmxMHzZ6b2w3E2ISB3g1mAmpXJ5RKHwWOvJPozAAmnvFkStEq8sQeUGq0ox8Mth1KAyvxdmWMINkmSI3IGYqV6QfmU\\\\\\\/jaOHXz65yJPt73gbnwVDMbVsB1vTvU8IC6+AwmsN2bc420jb8iYn35PMS9IZSnS37Jv6bgrKpGERL7CvSmK+71dz4BZGQ3xQ2l6LQh1PHH8i5TaIu734LJ5WcjI4DU9mb1Lsyxuv7aTY3XyXakJxTfkgivNU3HxLJSLq2aONFUdhrCduBP39u4OuLkY2zOpElr7ArQyVawmA0E7JORR8Kqo9KCaf5ZdElAflB9trpqv4RLz7C6psB6iapu037HEmpP6E1xNFu\\\\\\\/IbKY\\\\\\\/7k91P\\\\\\\/fXxllGmKAvBiyqQWUDWdpttW8ZhROH1iRJ5d0lBlaYvqj8ZiiQmHvrdpGMQ+eItTvvPrJjiO4wr66V0wDf77EJJTiaNYb2cEG2RZfP4BBozzbWYhuZ+lxKJJobts4S7eVyRZHl0k4G8v9CBYv8teejIxLgTWMpGJ7TLEPRFbFXqOOqrpBxP3EbFtnOs4zls0AZEPlSakGWRBXUWQpjlDUoreZwBkAKJrB9uZzSgRHAY1t6F3hV7TBRwGnMBYf8df\\\\\\\/WfC7xhMaVkMhs2+jo8FlyLC\\\\\\\/cl6yy3rlfftEVyzYG4j04inVhSv+oWSP4cnWI\\\\\\\/JFWoYjW3QLbGuIGIPU8M5fVVXD7c7TxMSG1fiqgjQjIHdUc5IjWysynl3xDw7b2Um+95xL\\\\\\\/1sMxNt1lrq5kVJW3BOPNLSuj3B+k\\\\\\\/TJ\\\\\\\/0NZVZYt4YuCIBrA8NguOjA4cILP\\\\\\\/3iD4EGtzfsqQC7nH+eauPWGHlxrPW7p9k9VFcK3ZKO\\\\\\\/cM72LIGi6HPXOMVwOpjWwm4aBGhNloc3\\\\\\\/0FTg0NQCrGPq4qVyBqVdaNAEO2GNUk0pdaz2luJm7PAc9WYdc\\\\\\\/rBKRO36WiGbSA26t\\\\\\\/8RASXdFHL8Gzc\\\\\\\/dt8NKJlyXhmhlAeJ1HayZt+vheeAMpNOyE13clmrtBSn8lyqjwsgd0BxN2\\\\\\\/lva2z\\\\\\\/wTnebnOGOTD5wC15yVHRHA8nlfD7oLnW5meQ\\\\\\\/gD70PnIJplcCwn6MtKM+dbyG9F03CFlDo09WRpeZLr8YBxUbZAbm18w8v8O+fQkveW+uK0RQWKbKjTS2IdFy+kddlIG6aP4ZkDohh7ydZwSy\\\\\\\/NgrBAWf55yNDLTTR7icm4DvyrK162FAWm5wnckiXGW0N8cg4GV67wpXk8Jvvmzvo4Qxpx3lAGUZmfDXT3ekwqNmOukkQ\\\\\\\/FD8nKVpVfZDzUbnIxV9oO+OAK3UmXNQiLUrS7Km0EPGMoqX0HJTmM3sFvMMQYAXC0T0HSPS0k6B6Fe9JxTl\\\\\\\/bDEXlwXWmjjL1buxJ9yzYKsw+f6qJjSsFvarlRpLDn1a0MxfsQWtZSiDCa8sx0e6I\\\\\\\/JL8OXrvfXyegwya+2fNwszIFdIFyewpfEEweuNQyQqeRpcolVy7Jcf3QcpL5eDVzPh4ZaihqDcXZnuLbNQHDzn6YTcsu\\\\\\\/IAjSAJF5Q6PcPLUlbv0qsWdymCnDgyUYcSlpiIzyBIwXXHKq4oXabumUfSUlC8+AxCiBdvm33QqZdAN\\\\\\\/0FMcIHk1x0\\\\\\\/74dlu9lP8AqCTKCScDDxRIecVFv8qVoG\\\\\\\/2Uh3tJ2zXk6lEcZPnyjc0O8pLzf+\\\\\\\/40zC+otIAygvZ7q04Es1gVXm\\\\\\\/ZpcGkycLhzoubh+VpBbp2pPXnpwxP0\\\\\\\/9UX7r61Ht7mxmthponbgZiD8XX5DCErGF0ZoO8e3ZKEUCWqgKrTmRHSNpVDOF1l617xHf6J5Ut1wi0dyyG8w8qV\\\\\\\/vXC8F\\\\\\\/xECBQJ8XrEAwyGvk5puN2kfrREFH28ClrdPONRE2s95Cr34iYf0yjqoALU308BlrJL2gaKL7B+LCaYtjl47d6nt9Z+zazCEk+0\\\\\\\/nQM1+HPDY0Z5X2evD7KxddGUNP1KXv4y9BizzMp1fq8l+e7yhfAeiEIFvUY9lfDXxpF292XT0bkC2vGlmPLhIvUWdlHqFiwfCtZTFCrXN6zR5Bqastd0fVTV1NPjDqtkeKShW4phKByVYy4jBIEClKsLhLMEa+mpoTyGiPeiU5TBFfrsJURwloFgQgIWO2TBEOyauu1BcymB1rUPcZVzeBhF\\\\\\\/k5xcAydJavr66NsBZ0nRCT4ZUuFbfJSGp51+lWpoQBGOP7Q0kXmHRShmKCh9o398FlidW0JLsAKYfA83NdLiuo+ZwHFXAhOOpl1GyMAjihIaxXhzXLgnwYWU\\\\\\\/AEcky3QLQrYs3qceiTJ6n4N8rlJf8ocSdg3AVr5Z0ZQw5y\\\\\\\/hK6m+V1TPWCMShlJDlM80Cj7mQYiFHIMcohFI7OlcyzL\\\\\\\/SIFz9SjskV6E+nv5R43hjkFOYmdCjxR2N3kCRQtqcVNZ\\\\\\\/webnB5enQ49Q2XRGjlQsatXGSOCiRyFrBIwPdw7FuKkyg7CCNgYNNvFEWk3vgT25UJy0OnvL+j9ggKHQWzPjXLwU8gPf7Qa9QmEFK35xjkdhgdYbOUlaxUd2i6OHQwedKvV9tCIsMuIqsiYC87GSFbKZ8FqsgAhpmyKx9gDr3VQzhm5pIwOLDOTybMw9hDUocKkOhuxv34bv7HRpya6wwgErE+dNackftPJXjhDyvXkUsbxsMmj1xQrdTtnRTJAHU4fs4kFCGr5Ur18hFQvI32wW1zr1b2cNMYusfJYtSAiZDF4\\\\\\\/+PD6VGHJpyWDFmx6UbKCKk1V7a9XdbyV2SesRvi3YVznlYK+RAqtmEOSx3rHefgmVunsE6RZvwJZ8sNtkeAE7hqzCAgSF6zDcUGjlD1PEldUJUn\\\\\\\/TLiGouFUr9dvrreIPmM6s5rSXvaDVkaPy0asn9w0E0UoHp0S\\\\\\\/+OgSVAp0wkzrjWNnLy2st5qvSnz\\\\\\\/ko8++0eKIEH+OgGHnQVaVRSBGpoVVGMtqN26YsmAxTucYKWc5ICHFP+djM\\\\\\\/nvPBbtbeRUU927XIQS8rk6UwT5uDU2mtQDtS4gJzlpPMiWNRWcRX29Rf7pCNV8mZ1k7fbZRFFf6tELtLX7gdMI9pELNVSHm2elAplJjB+kzGgs7\\\\\\\/79\\\\\\\/VjChZYGWmSB3nnSoGAFQATdqGvC9OhXx1rFgfBvuUxMkDP2YLbezF6i39zQDPnU7Hhoeqbwx2MjkHkgB4XWCnkpgtToZESq7FjdMjInYYu\\\\\\\/WR8pddRaYPSvcysTezSIcs1\\\\\\\/iBwLHB4qng4YK1uBSvnUiuQ2GXbNSE1xuKLVSR59BEbERQ3xpVwxZ8aa6V8Hor1E4Bg\\\\\\\/N8sMrJtIBbN8bL101TIXHi0lySJdA5cwiUyv6qx1zq5I9SfGe40vhqY02Nt8TdrQ5WPkIhNOwrGN0y\\\\\\\/ahvPxRd9oqAqi6iem\\\\\\\/zAegpLtyZIDaao1fQNfhCOk\\\\\\\/Su2eN4qK+GDO3TjMQfO909hh+5CKP7O0QypuVveiA\\\\\\\/XQ2EnzDH5xl\\\\\\\/aymIPGHbKz3fMHRwrwK0Y3zISgbVkI\\\\\\\/xdgv654ALQcOlWq7Cg3Ad717m0PfpEfmi\\\\\\\/M4WB5MJ+wf3o0m\\\\\\\/q1KxtGt+8pWqFtMpw9e6f6B6XWHWUEMh\\\\\\\/fezenzvy\\\\\\\/v2x3p1as7RT\\\\\\\/YuHCDjstn6ybw7AziR3tVsjqqeJd6Fg6FPxS2gq9L8mJjZolC2VygUzzj93SXulvjXEoEBruDuPQ9dvPNvpkco6UTVJmkXd2+16e+Kz6RXA1v8qb3ZkCZx6s2lguo8uhgNKvYLVrS3KQ\\\\\\\/px9JAuHSCdxDZSqKymAz96eeGxTtMQyy\\\\\\\/mM7Es2Kv4cg5etjGFQk4r2S4GqOua7hMtrulVi4C10hiLsdtuZtOWK6fx7q83HCNBngOp+D+Dzo0ERBNBxYAPsIaIYN3PW4\\\\\\\/DD3mWm94GNILsvqx2PSI91ouChIWfb6LjDOj958NsMVtXnQxgPtIascZF2wr1vs\\\\\\\/OSyFTJt3XVzLlGEE9wVgvbi3ZYzNHSfmEt1gluFR\\\\\\\/fEXv0VSBZLtuhVue3Vvf\\\\\\\/R9XyMUybaDznixqjBNjaR0H5fr\\\\\\\/++ENs+zuaj3AnL88rdATbfBogGTezrRIFAlEGE67AA1HZLjTB9EDeGnVrlBBOnwUuObq0zWufwj2am7NpubaJ1C2Se0vm4LWhNwk0gP1bqe5rkI5bBwv6bs4+MsIC1Bq6ddLYrkJmQ92eV2Q4o6aTjpOziFJV2960BhsSEGtlK1SR\\\\\\\/DL4s+CHKN5D3P7hayOAAYB+yj7ltiyNkq4mmIRcbRwl4RvXxPYaMVwRi4\\\\\\\/EHwnb31bq48+2Ap5OBCf1F9yLcnSJgOSwTE8R+XVU12D5VEhnca7Ywaiit7wJFa3PiD38YYI5PQWkUAA42T24kc+PB2+WBbaTlKf8tO\\\\\\\/ARlgLv1QLhpzRXcK+dqBAfLRfWkO2okTPKwYJJPA4C7L7gCmQHQJb5SgeGGLqmxd75F39yqYHeAdu16G1sACtUD4sRlQDkVbudV7uoz6oKF9B1+WYbgMhLbnOLJWut8kaC4lA\\\\\\\/uyrpbOJjwHkDpotqsafXGjLhNcdCIJl8SjY3p\\\\\\\/IOE\\\\\\\/KzhikLXDYyq5wTbhyscZlFHvES4fm1AJKP\\\\\\\/mbuzrHSb1ZHFG6IiJ4p0RfOlB07yROmwg0tO7b1pIZYbmrKoM0Rj42jzOEMR95L12wnItwc3YBvU5wAw9RVCH2qrVruTR0gQAuX\\\\\\\/y10wMxmVA8F6AJ2gItvxdp+\\\\\\\/QJBcKG0dbe3wc7Dz+1R4CMzyA4afUtf5U2vjaLjykxpjKiVfNZ56Q\\\\\\\/BmYr8DL+ALSS\\\\\\\/Y6oPFCEdPwXdWqt7n1c4JtzpDfHgev+PmcXpYGYVGof5MtsnCMdg3fTiFGuvpbPbq6srnriDTZUkN8BqdhJAZtHLKpJ7guhZ6IZkjVkrBXl+VqZinSXihvw+myqxp7eLjvVWQqZKFcHUmsgNYPjkKzyUc+\\\\\\\/EgoPNqa1JgIWS2AsRIV7O8sYGZ+IOW\\\\\\\/6ldCk7CUE1t\\\\\\\/mMyAGs3aC2Hy+qqJ0PI7Z+3gN0WPg0N3MyCQGx15jK\\\\\\\/wif8h3dfDADQ8EiwI293QfXOuL+biZ3fdw4uBAUxNe3HcSWQ9\\\\\\\/kRD915fQ9bjtfKcYZSPbgNvGmmk1RRrUK5\\\\\\\/VOPm+zQnAKRuwgvLAWEhGk8CZn5meX6iBLnodDdLJSa9zVpuoOr1GR3+NwpzKZJSC1rM+xe\\\\\\\/\\\\\\\/UKiPCPy0PrKKYS+Tk8q66D14IFJigjZl9GC0AECqtk5HTJ3OFFj\\\\\\\/8\\\\\\\/x4YfhSSs7IQVm4rIAQ7B1+7Uc4EiqduA48\\\\\\\/NNfw1EtkhL45AGiC+Xg3FTX8uCnzJXvxN1Yc+mmMhcZ4VHo+CJGtmYfCwKpD9r9m6LBsQ1Yh6pBUXLu\\\\\\\/GkFnZrdo8dcsSLne0WCftQyD\\\\\\\/Q5F007DkQ7f+fuD9RAqORNUcavPsj4NaB3M2H2P3riHyqoHb7cQ\\\\\\\/aVXQDu1fVHAlxetGyt+rYbJv1SFBt3uFnuAidiQ0IvU0g11mlgZAS5XzByprYubcki1L1G6KzF1oocqad4u1C1vthOvr0h0I5eaYy4MBb5T2atzGXOKkKBREZQWWLY83ZvEk7o2DXlKDe\\\\\\\/1Zwg18qAg1gTd49aEPvQLkaW8I8i3DaMeo6o3zP1Lw7\\\\\\\/ZRcNpYAPn5+lqTbjJq\\\\\\\/gEPIPtGqjXJVYniwEcIkSKF+byW7mo9jbCyGY9yBlop1ud35\\\\\\\/oW1L0eWwt+meubHntmj6XWL42u+aPmnsE\\\\\\\/M7C5GW7W0ngyhM55r2+12l5GS7+0gUqcmfXjqwOWR7\\\\\\\/sg5nv\\\\\\\/uLWGJxhZHGe9DjWcI6NmtOdhU787qej+6NpAY0Acpz6GhvxyEEslCof5Rf7ZGBoENEYytYSeTVIUc1367T8BV53xmOkeZyv1yUGdGyiavhxze0Px1kx6bgOY8fh7mvInCTvmtppAn+lzx72ycJoPnYmdXU2GX4ogf5VpKW1W6z3\\\\\\\/4ogRBFu4HlWn7jHxGdOBwzmONviAq2lrNoTMoJwaBwoVT09eNw0CE3h2Tr0Zb5W5VlByorVZxsyUfOa9+GZNHFbhzI2no1qdFUOfS\\\\\\\/QulFAn7aYWh58X9fXHAHS9i2k\\\\\\\/bVCWntayokThG6aBdjLJpYabfq8BxRXpTy3n7QSlMrxK84cYP901dRDBgC\\\\\\\/VNFpk\\\\\\\/pjMCeqR0u6S2OOEkNP40XT0TsaSwc11bsPXH65HwJ8QRLpvB4A+t8U\\\\\\\/tW\\\\\\\/WAcRE6c7xzn5ebKzl0PSvAhMP7OSk\\\\\\\/z9uz8MD0mZzGh8X\\\\\\\/wOfphZQ0qxjuU9yA0pBOMk95DuAPT9TYg3JE\\\\\\\/M0biWI5NHrMoo234lU7p5b65gyznJt7o5RtvYT98RjqyFDoKEYbbaRgmNIRpJEYSUpcP\\\\\\\/ocuEBQvH5NatM+5ar9nsmVoGoV6rcTwZePQwJXhC06d12oZkQhXuH2qStwxiwVH0kh5vW6fIXLC9pSEfEqREcAXdB8z0MPHhV+KYM9uI6gfjPUyIEvbKYCGtqUoxVaQGT\\\\\\\/KMdQ4e2DNBHmPBOPZiP\\\\\\\/o9jzlB7vpjIU5PePey5bsD8A3yUkz\\\\\\\/\\\\\\\/XutITpS9JvQbfJn5uiYFizCtaihqjIpEKpaUeg1vuIMK5DjVnPImcEe6BPv12mnkQBqyOV8d4qxUhinuZzEmz6nKTDpyRG83fy3VwDoiIULP7mNFxw4Z9BYNlWqFPfF3\\\\\\\/I0OpIxHwY+y9GERdshuxHLHuCNWPt7uwFKWmqzqg4PKOJcbe89nBC7s+sOtM8X8BlUVtm6PzGj1iQ9qLTbk3\\\\\\\/wSYFvPpqZXMoF\\\\\\\/JEZuCIJtAwzMyAZ5f1t86HPwG1cMEPJFIcb+g8zCfQ6wdmkGHYUcFzdPPh5CUor3BNgv6SHgUcg7xkX9wLwG3g3sMPyVArLDF7MH\\\\\\\/eJJshz+k2qKMZDRL7FyA+obpGiS93H+6EDu6IqbATXnODPWYzJtsmVrFKHItxuSiWdggdawV\\\\\\\/hx5GnOPQiCTGU6\\\\\\\/xTgP0bAKHtN9xZZAYrNSs84fNWiuS0oZy2j0tbt2Ldq4IPJ2VBxf0HHZFt\\\\\\\/AdWXHlhSgcB0C+kYEN0XkHpTmnyTV4oR0Mp0wtbAq5pXUPq9kF\\\\\\\/piF34QvqvVUZEJJ3uX22UUoQpLP4TLBT2m6KLAJ32yGS5cQOIlY3a2YKMHivCoNyK5gD6MLbHcgkJihcdlM2GW8GpVJFXWGlIqVOkBswUjmd72Kc97InGfGEdTaRHrkoNzmt6yuhk9LJFfq1FE0kS+wGCiqNa54crqsuZG12\\\\\\\/dVyNb4EFm3y4TVHBtJZJDFwETfXuaUEzoKfRI6dCBgeHVvyMqAMaOALKG\\\\\\\/+TvyU3Zkx7AAzthynFkxayGY9uVuTIcENx4Dcfqr3IFJE5DDFnZqwo1kuoiO8rL\\\\\\\/9tnneW0q0CcT2RMl9D7WPdngzfBUDp+VC1YWOKn3GM5+uy+NcKJO4uKfUpLDHgl6CgJX8QQbE3g131+cSP\\\\\\\/Cd1cMLwYuacTXUhy06Q\\\\\\\/0JwOniLHecgZq+zaX362gRlsubK60cN8i3wCo0wrWFxrGxcstuD1nHgYDvadncsrKFk9hMLPjk7mvwXieIL0JAI8hto0AO3KQkOCGKeA8Dn4VpSSviMCQ3gG+xgS3CEm0gaKdQHpTsigasVWihBAHcF2DTf7BGKSuhQTM+bmqNdvDGKIn2V4I4yst5eOEPMp+LUR6w1CxPcI2SnlgDuasiO69vtBhtZVGbs168Ls61fACw+4R3EG12bAOck0upom9AFs0Z4sqkNyFGtPKTiPms\\\\\\\/eA4+Q04qbUCVjhKdosKNMUUIJb6aHH08t4sknja0E7iqgVIgMJCRdZHILXeAp9sW8SXFuHs8jfx+dhWmCCdd6ZMbSIph+KrCu1TEd7AmZsE1qyKmJgn3HdclsLoQXp4gw4ytYaxC3kyWyzPlzq87aqG7RfCMAs\\\\\\\/\\\\\\\/Qkqb0XM+x\\\\\\\/2FapwVcDO9otjVuJEFM4CmcuuXow8g\\\\\\\/58RHIOH\\\\\\\/P15ttiizoR0dzlpbjzCGE0BRXxv0p0qAwZ6IVxbopAOZQ\\\\\\\/mWc7spHUzypfokWJxT0KE4EQ7UB63SRTPzn4on3fO4OprN\\\\\\\/2N7p\\\\\\\/iBhmX2MDLT67qXoX7NPfrKrVmc\\\\\\\/yrN8+vXno9kAdcVszYsm5jYnR8CcI6TLiWHUQmU0J2Yg1h1gc8DaGRiMBhpKWu\\\\\\\/TAUsQIGPTOX7wIae9OXwI5+Ei60KnubewrDp51NBEjqCOWXModuWpKMNJJsdtLk0UaTXwPaVf2sqTMzPA39g3vTuvIMtCReklIev93agmArbapij9BL36AwsY4jNnJj5ME\\\\\\\/GLVrQcnVQx3g9dWyBisVz3dkvExSOL4\\\\\\\/VIC3OarM+ZjiHSht5QL45BdJVYIz0PwTTqw4tc\\\\\\\/axOv3Rqi2HjGKbMePSBm3IAto98MHoL3ZuMQQsp\\\\\\\/25x2KcoCR81Wi\\\\\\\/KjuSB5uCgpvigCv0r3F3ojYBwIGzkjDpU\\\\\\\/LDoUtpFWkCbQfMk6QEP4v1WINghqnpMSapFRjabEdmviR+7pG5UoWqct96t5pxaJQrhb6fqfjeT\\\\\\\/tBJI0XIgYue9zymD0SVA6gyRk19pm4oepX9AIlWcBux5+RQrjCdZgPTafn6OeUYZKkQUOyfgBzcj6xQICd8MR5NzCSErNHpBvQLlVYjcShe8a7F+KEEU7TQAEzvLqC+QGNM4BVAojbcFMLxhGo5Cn0VGTfl5zEQkUrjVUa2nVrQguaXaG\\\\\\\/D31bmAU2gbSkiPSfOqTonibNa2JL7VEaBdY2C+Em4gSG0uEd99UYNKfKTOmxDANiETYy\\\\\\\/QREXy+tdYGRnKyxf4nbKQs6KLmJRoVKByg29\\\\\\\/7Ss3YvnLYeqRTMj3GR9AyuO3BiIZczYW1k8gGPKrCR6jGfawKwvLJ6kTG5n0cuSX93KYSmD1efGR+cgK1iF5zwpZUyGP9sd\\\\\\\/4PQHBmeF42xMo\\\\\\\/6oeDSdcSwrJ9Qicp9hmTn7JdKPbolHuGkNE13oXlO5CFN0VUvE2M48dGIAuHKMAy3u8xV\\\\\\\/IjdTXzWNk8U7Ku8btg4RZGW2AhDJpwr3581D3lVPGE3zqXLOD7ygCfTYVy6HfuRfzHXk\\\\\\\/QQ1frSBrD3NYRUhpXUXgGHtihmWaAJSGr6wdMzNVAWxHSj6jMGKrs8oTE2Cb6nskC6cyWyZC7UsrptKpI8enO10Ivxt5k\\\\\\\/FOcRTp5qrDUVFsFmop2yrnLgcexjG7eYYcA\\\\\\\/RDXoIiK6kIvtqlh92qF7tpdK3eNt5Djh6wF2pc2tBvVB4kxaPjB6OR1EPIzCnt8ce0t0wh\\\\\\\/aSNBeRmhJvyqk5unRuN0\\\\\\\/9SULQpJSCf3xutemLQGkvHtGoL7Jv8WE2JRQxYolo1ooNwBklBZuU9LdYIMWIi\\\\\\\/bWojy3qcb1i\\\\\\\/X\\\\\\\/b7gd804YE1JPraojIB2CKIJVLL\\\\\\\/2zfZlyvQ5j5ul9eayft8IppxWCF2ONTTwSB3Dzkqy9NzWlcMJ2rEy3C93qardMa8AQiBZ\\\\\\\/wpNZsz6cjiBhfGe6m8POJtc5ysaAOkqJ8ighuUSmMpyTzbRz0RGMooUpembhG9vHkHyuLUuTbjgrRvgctxcAzp3DcMj9H8DmkuvFDA+6CyvFLeAJEyqqrSlykeSUvkQP38a9iLuF+XCTWfkdJORV381UvnWni04a18mJitX9\\\\\\\/gJZF3lQ9jpBbLIDVszEbkoN3eKv7+BQ+uPIS1eDQ0c9wCH2uKATV2rjGC\\\\\\\/am1HMrRBO9nItmUQLkIQTk3IrjbzSjAZqNm15aOIBbTx45eFSFM8JgT2GXCSVZuOPACoTz2BpN0OG88JpUebFf+sUevpTTmWXk4G+2k8I8RJiZ+bK\\\\\\\/tQ3nio8\\\\\\\/d4lJepDYoSB+yAlgT2Kua8Hnz7\\\\\\\/mPb25\\\\\\\/sw+fvPCixMFNRPMFtAySfUT+Ecv3ZFan7iP7nUwWAo31HPJ6fDmkLnenykuZh5EUqdHjK7T\\\\\\\/rH7rwDvi9SnFcEsTsMIssTk5KYQd+OetovVUosVlbus72Dgfa6OtNxEU4vNJqJzjo+Y0IIs0Z33HLGr\\\\\\\/o+G8lCMzS2ETf1OLYlMR31CxvMnoXnGHETktgORJcYq4su\\\\\\\/kjXJ39MnNzZxxG\\\\\\\/\\\\\\\/malUL0VW79u8lNEIv4n9\\\\\\\/ddvElnYcj9ilkWetEND9eOLWr0sFoS7enjRRcp2QMzDmJKhJHqpSxRqapzoiuhPDZGQmgKXsHKMJ1uJ2RlzfJFUTLAiNJOrmlwpRE3JhiAACpZhU6TmXDE0OtnTs15Ri0x7rcpJCQqtRRY8OO7xBYU7+h2EvLCh+UCC\\\\\\\/7V9fkOQvJNBchssw1vLUKExXaWKhSvoj8ZmKOvC6r2UDha2mt9z0fLpoM7fRFNe5Bilhwoc6RxxeDRPv19iatWhx5fxEZAyhghLAjJ2AfvGgTGkt\\\\\\\/rS22jwQohTtK1uyT7LfE1GmUkT93bFAe2wqPMuPxYOF7D5zo2h+ldbQMcGckAACCnLa1FVRg6lrXjE+zqgpeOEUUEtUPCGQXTRpjw5nV+CZ74bG6GFjAoFg76SMME7TiAAWa5Q9eAh3b4soDmQDtjK9r6yImRzhEDg3P9lbMmAPpA98QhX+tDzndyjwtUp+qnV2KJeEQEv+lvER7Ul8P7J+crrnh7QGDR7Qr7tSiR3OlvwPQTlxtEWGQFubxTEbXOFz6LVKOzh1bWKodHWIJrvnXlVnb1gdqwVwKLRt7cDRouYMkkbgaIgQqwSolDA\\\\\\\/MzWWm0JyOnOE+vQ+JWFjg+Lwz8sFwvEx8Y7PRNXsWKs5H6x5YD8+x3\\\\\\\/rUQUDRy0qWUrbQ8Mv5r4fmZBG+yoeSqKFQQZvpGwJlcbEAh0Otz2gVGKkRnQB3kdZAr4tZw52EW+7bDNZGLr+YYF7OmTAMybF\\\\\\\/KazUK7cppofOE8+UX\\\\\\\/dIXnF797grd6rNUUUrWrn12sR0iFAzhj\\\\\\\/5PJugtZ2DES3kYGUwc7rZRHOs0VBUI+OkQUSmZQeYyDaJWXig2AI5EicHdKRjy0GHZfbqfD7kiio3+LQDtbh0AYylye0FZ6IJGguMaV0eR5EiZ6s98RZPWRPMw1MPVl2tNLJmxiNRTxS\\\\\\\/wq7cZxKSic09tZa4r4jQyCigTMtuQiFVJ+DbXm6ak1UV8JJv54dzrvqwkogHDHnHv3Hbpb6GxesXnZiZZAvIcZ2xmlQpAYBxOhiAoAhqyl5J0Xa5FVo7ytltzYEcG2ALhk1\\\\\\\/hozGDP4n9SCibzd11pXsaN1z0GO3vfwBVWfs1ONgl5yP73EiPTYbQMnVbEI8SM3qXGo9XvGCfPvuXVxuUY\\\\\\\/bjUUxZi9B4xBa5gkXprXcDrNxalHMKzVVSu4h63PT9T1WDq9Uz01gWGcLik1fP4ex4\\\\\\\/l44M7iDifHFjxX73CxoofTcPkA\\\\\\\/MXv\\\\\\\/Q4OCa8jPc7oV5kNLxXXysYLgFDixRb0sTemehcy+wCc1PKJG5eb5zzs\\\\\\\/MpkuVItXFz5FTfzWCTuuoKB+Env1YrH1CaIBGX6fZ42pI97dCtqV\\\\\\\/YOOW2dj7oES32QLKRn+2BgBUZlXGq3KerXDwMoEcR+T\\\\\\\/AXDDVoRuEKUXwd5ENsxBqAnxJbvPYsYHm2SkZsc6uY3qNHxOHQUp2DP8GbC5WDGA6V9A9PsG+ORgwyGWgytuWcriKFNaQGn3kfBPUDLhTOW2iuYksktheyvTrFBO2ltz8YhwMgeV9JVM53qhUovrjriQNf0XfoEEnD7c\\\\\\\/Z2w\\\\\\\/R4OAdLyV\\\\\\\/ts3RP+oDwqFza1AjSZ7c+zgYEJG7RDWBSigHBwq2J04KmH2NmJa46bpnj24RczrRhH6zyQvZ9V1QHWOUFcUgiaufu2m62SQ5tJCU8FvsWkelYxPqCL25RCxug2XftJPLuERimlL1AdkjfRz6M57GbeZJ2usIDFdKteBUea6sZk\\\\\\\/zX9g2lQeCK6ezW9CG+n9tLeHKS0Xh00yOw2dPEnxu7HIy0araBbklfgCxvYmAGYrV9smNDW4Jm\\\\\\\/1OD24HiTUJwCl3UoS7FhT6ZaqCcepeHmGHP4a1ZTBEzmCAeEjailPv39m0rLPcoeIIMGDnE\\\\\\\/ZsBYQ33+jqIXkC39pH6tJWRIGsAQedROudFDIBA4j4k980vxlgaUlLh0OxMSvrAU\\\\\\\/44trF9nmnntfr5Z9InFKAHucwqtcLzPOfyz\\\\\\\/YIUTRsJUip9lP2Fx+X7o2Bn0SPN+FmSBGwso2TyquamHxEZRPTOlq8583lKY\\\\\\\/\\\\\\\/6YkS\\\\\\\/xm+cKskIpfaWGV0suuzyuBG\\\\\\\/8mB45w7gSBZwsiA0SOqnxFiE8HAds0TKg3EF5I0CbnSPaH+Yj+ziuLvQAbdxxOVHGZahqmfs048oIxGZBfhy4RSTwVb7th\\\\\\\/KBOz9BllO+5xQ+V6IA0RYqrHnJlxUyefW69tCVfkgVU3jszMTKRBPdmfWP9RPbm5jC1QNEelWDrzO6p9VdVLq9D5fk7uNXjVcgEM9gWmgjpCsrBtqdhiQP+w0u0NSmxB99c3Rs8BmBeZS+NHGZ7rJ0ofz57NuWfu2Xtzi05bO9lOgCJAP5lHQblwt+fi+ujGIqW51L\\\\\\\/VQAmxoVDqMXP3GQuYa4ssRqwdXODgpbfH6RBgEZoWMsqO5Lvo7lrWA30IQWLCgGZkaewd28xLME2xXw\\\\\\\/E3juu3RpEDE2lnpKH9GdgK3oTFq9hLOGobsErIHSC03CUc79D4tXFJ2rcOZ5fLxd8PVvcae4KzAZWABU+4l6eOVcdMyquGXXk1veY7+yU2UJXftpF1o8+AY\\\\\\\/pj0YMQObIUvFJd8N8mqNTIQZmSfGS4Q3NDhaMDmIykpTVEgsDXdlTQniUuVwNUN6iLTzJmApuavG0WyfuDZQfduhnlksIkoxg5o9wA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022exVLn2\\\\\\\/Z2TB5YLgR4zCsRA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226b7fc05c83e98f6d74ea25b20eb17a87521d916c8920db1bbd73315c160f8dff\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UUJsaS\\\\\\\/xtHJtkGGqfWzzlosRmOzQ5D3voKxUKx0rsyljpy02fFaK0y9RpFm4efWbwc1tkaqABApUK+Qgc5JiZbtm7dSJ09JRRzWuMFtDW325Eld7abF5gnuCidJ5L+ID0lYig+9W+uuIdIoujX8rgIKfNUGDcYuiWJyMkPgLomQdiFFgaVmA4mf1BSFldzucCfd6c5o2W60Im8TPlmP5HVlPHom\\\\\\\/O5tOL0nz5TUA2ogoFB\\\\\\\/QrVYJpD1M9fmYpPesbQaMwPjbU\\\\\\\/KWN7o04opsG3VCN1mqBb8zI3JobX4n529Y5DRsJU5D33BtRtx6YVBtG4ZqXUU\\\\\\\/um80AKMOzM9LLuP0qnUxQA3yTa+Ll5\\\\\\\/VlPXjer6m59ZbTML4X5hYDICr4niDNfc4m+q5puKjbFQxsydmm14WI6PilUU97KYvvA2t0eKRAFQT9nbfgGO\\\\\\\/2yh1DPalxvP1BCZ9nqTqiMQWwZDqyBhnjjXzGpNOlGbfJtifvONndcdNVInwlZMBRWIcBcZqxUk20MZFIwhIwYsoazH1xT+FiRQ81DMUuD3dnfIzVrg6G1LBKx9iQBthna\\\\\\\/yIOZql5yZnonwayK6TuvxjKFfvFf4csdIKINRhjG2MFQZGmBxYmtTX9lPI0hwWBstA3ghMDF1I\\\\\\\/PcapJrItoJrvXuF65tf9eghTAuAd76R8gf6dMY0VrLxNSLTrO4bPegq9LoZDyPxfqc1mteeM\\\\\\\/Us2AHSoXVe9pgS9CvFJBtDXrzDDhFPHc3OQUoRPdKj\\\\\\\/N2Lkv\\\\\\\/NNrqIXNO7Slxct86tFTlYCEZyDKaWPVlJx5cTsuFznQf4Et++6kCecyNMIR3Tv9dhvZJJbQjRH7\\\\\\\/Edjz0yZLTeSwuiznBy56d3rWEUrC0rp9T88ju7u2XQl6SEq1b1AwZZdHl9BXK+qh+3bWiyexhKUntWUhI2Oefk4fBChIKpvghgSJ6\\\\\\\/ciGQuHM7k+up+IvjvuNN4AbJyK7+Sf6B3PiBDamHQyDQAhCuqG0pn7Lqb+Dpic4NTq5hAeABgHQh\\\\\\\/LJzeVhU1iTl23H\\\\\\\/wFs9LdKn4Wqg9AN4s1jWy7d3Er+cpXFud0U0g+HUl498sS10gzHgMCUCK7S14996TFiwbutHpJ6T0RmwhGrSe+dSj93UMiNy4sny4FFa8\\\\\\\/Z5ohkyZurZggBgPvU1Fh1q23Fn5fYCl69CwAqE1Nv2v4Sva1pU1uXz8ZURcjh37fear9vLMzI5PcRSasva7Um45yo9EbOaQ9001luvMu9FlZuxCHdTnVy6DG3q15Ghs1atMxQWwGuXlRYtfPLa04PTuBNTKgHKlrboj266b8CSsuRhZX3kDgVxgVDxTl1AwVYq+PgqW\\\\\\\/cjPizTFALKXzeXUth21TTN6LZSxB8wDGVioxNeuh75vr9AJuY5+yoYABoTLPsQmKiOJHExENPiSsQstdurvb9XIUDv5JaloxaaxZqdQgNDdkbRkdR28nXjDDN09CSjJiQDVBVtBv8FlaUGbzKfNBVo5WoGqPmkkuMwYmZTs\\\\\\\/MY1io4fzv+DbHmUpcr\\\\\\\/QmfLMheLfaiuLNkcswq+SRB1xA274E1hn1HD4liyEBxAgJzdnB5+4VNdacx67zUayYIleiyKMSDIWwQsUPkptmXj6\\\\\\\/szGWTOtCocFT42lP7xu3FeCz2JdTU6x2CJ6HLXVo1wfrCq46hwhlFd87\\\\\\\/9\\\\\\\/sAFWWwhALQEUabv8J1SPlj8fukNM+rSMlo9AlH3baea1eSInD\\\\\\\/mgXpjPE8foo7oLZFIND9XgmMeADgOqozdyc7b7PWijEkN\\\\\\\/vPYZK0QYMO08a7qQWBKWmsRIakZ+kMh4qhhhgsUtw2aamdQ+2k3lCSBG6C\\\\\\\/z0Ydrn09JfsLcDS3\\\\\\\/1JwxZK29SkD9nLweWo4epW84ZE2v6hptd4taUBANB3jv+wot1NY6oiNSl7GcO7573w4aELZHu9TK+luoHt+F\\\\\\\/CrPzIJCNFn8kV9+q3aIG0sc1mWBzRmhB4P900ZqXiYjICgPGikiFj2m2hiGvW2BiA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BWakSME5c0srA9oQSnU4Sw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022550f06d942379c0c236317566e569b9ed31f636dd8789fdb11b7a8e8a0ca9ffe\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459993 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1685" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YSHmTDWPbkn2OncN010jQU4kIk6ihNBBpIPG-0S81Pg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1685" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xXPbN6WjDnSUgg_7I0ysH7omQ_r9DGesw_juZ0LW6l4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1675" title="PUF-8 Functions Redundantly with GLD-1 to Promote the Meiotic Progression of Spermatocytes in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1695" title="Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459993" id="node459993" data-pisa="ggg;5/8/1685" data-pisa-master="ggg;g3.115.019588" data-apath="/ggg/5/8/1685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Consensus Genetic Map for <em>Pinus taeda</em> and <em>Pinus elliottii</em> and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of <em>Pinus taeda</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jared W. Westbrook</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vikram E. Chhatre</span>, <span class="highwire-citation-author" data-delta="2">Le-Shin Wu</span>, <span class="highwire-citation-author" data-delta="3">Srikar Chamala</span>, <span class="highwire-citation-author" data-delta="4">Leandro Gomide Neves</span>, <span class="highwire-citation-author" data-delta="5">Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="6">Pedro J. Martínez-García</span>, <span class="highwire-citation-author" data-delta="7">David B. Neale</span>, <span class="highwire-citation-author" data-delta="8">Matias Kirst</span>, <span class="highwire-citation-author" data-delta="9">Keithanne Mockaitis</span>, <span class="highwire-citation-author" data-delta="10">C. Dana Nelson</span>, <span class="highwire-citation-author" data-delta="11">Gary F. Peter</span>, <span class="highwire-citation-author" data-delta="12">John M. Davis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0001-9330-4051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Craig S. Echt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1685-1694;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019588 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459993" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jared W. Westbrook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jared%2BW.%2BWestbrook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Westbrook%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJared%2BW.%2BWestbrook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vikram E. Chhatre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vikram%2BE.%2BChhatre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chhatre%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVikram%2BE.%2BChhatre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vikram E. Chhatre</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Le-Shin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Genome Analysis Support, Indiana University, Bloomington, Indiana 47408</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Le-Shin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALe-Shin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Srikar Chamala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida 32603</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Genetics Institute, University of Florida, Gainesville, Florida 32608</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikar%2BChamala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chamala%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikar%2BChamala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Leandro Gomide Neves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>RAPiD Genomics LLC, Gainesville, Florida 32601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leandro%2BGomide%2BNeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neves%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeandro%2BGomide%2BNeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Agronomy, University of Florida, Gainesville, Florida 32608</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pedro J. Martínez-García </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Plant Sciences, University of California, Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BJ.%2BMart%C3%ADnez-Garc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADnez-Garc%C3%ADa%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BJ.%2BMart%25C3%25ADnez-Garc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David B. Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Plant Sciences, University of California, Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Keithanne Mockaitis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††</span>Pervasive Technology Institute, Indiana University, Bloomington, Indiana 47408</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keithanne%2BMockaitis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mockaitis%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeithanne%2BMockaitis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">C. Dana Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BDana%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BDana%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Gary F. Peter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BF.%2BPeter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peter%20GF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BF.%2BPeter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">John M. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Craig S. Echt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BS.%2BEcht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Echt%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BS.%2BEcht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9330-4051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Craig S. Echt</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cecht@fs.fed.us" class="" data-icon-position="" data-hide-link-title="0">cecht@fs.fed.us</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/8/1685" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459993" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459993/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1685.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459993" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459993/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1685.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459993" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459993/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1685.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459993" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459993/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/8/1685.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459993" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459993/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">A consensus genetic map for <em>Pinus taeda</em> (loblolly pine) and <em>Pinus elliottii</em> (slash pine) was constructed by merging three previously published <em>P. taeda</em> maps with a map from a pseudo-backcross between <em>P. elliottii and P. taeda</em>. The consensus map positioned 3856 markers via genotyping of 1251 individuals from four pedigrees. It is the densest linkage map for a conifer to date. Average marker spacing was 0.6 cM and total map length was 2305 cM. Functional predictions of mapped genes were improved by aligning expressed sequence tags used for marker discovery to full-length <em>P. taeda</em> transcripts. Alignments to the <em>P. taeda</em> genome mapped 3305 scaffold sequences onto 12 linkage groups. The consensus genetic map was used to compare the genome-wide linkage disequilibrium in a population of distantly related <em>P. taeda</em> individuals (ADEPT2) used for association genetic studies and a multiple-family pedigree used for genomic selection (CCLONES). The prevalence and extent of LD was greater in CCLONES as compared to ADEPT2; however, extended LD with LGs or between LGs was rare in both populations. The average squared correlations, r<sup>2</sup>, between SNP alleles less than 1 cM apart were less than 0.05 in both populations and r<sup>2</sup> did not decay substantially with genetic distance. The consensus map and analysis of linkage disequilibrium establish a foundation for comparative association mapping and genomic selection in <em>P. taeda</em> and <em>P. elliottii</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pine" class="hw-term hw-article-keyword hw-article-keyword-pine" rel="nofollow">pine</a></li><li class="kwd"><a href="/keyword/linkage-mapping" class="hw-term hw-article-keyword hw-article-keyword-linkage-mapping" rel="nofollow">linkage mapping</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li></ul><p id="p-5">More than one billion <em>Pinus taeda</em> L. (loblolly pine) seedlings are planted each year in the United States in 13 million hectares of plantations that extend from Eastern Texas to Delaware (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">McKeand <em>et al.</em> 2003</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Smith <em>et al.</em> 2007</a>). Southern pine plantations, composed primarily of <em>P. taeda</em> and <em>P. elliottii</em> (slash pine), supply 60% of the wood products in the United States and 18% worldwide (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Prestemon and Abt 2002</a>).</p><p id="p-6">A high-density consensus linkage map that is primarily based on polymorphisms within genes will be useful for all areas of genomic research in these economically important <em>Pinus</em> species. Genomic selection, which aims to predict breeding values from the summed effects of genome-wide genetic markers (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Meuwissen <em>et al.</em> 2001</a>), has the potential to accelerate the current breeding cycle of <em>P. taeda</em> from 12 to 20 years to less than 7 years (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Resende <em>et al.</em> 2012a</a>). A high-density genetic map may be used to design low-density panels of markers that reduce genotyping costs for genomic selection without sacrificing prediction accuracy (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Habier <em>et al.</em> 2009</a>). A consensus genetic map may also be used to compare the locations of marker-trait associations in independent populations. For example, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Westbrook <em>et al.</em> (2015)</a> used a consensus map to compare the locations of QTL associated with resin canal number in a pseudo-backcross between loblolly pine and slash pine to the locations of single nucleotide polymorphisms (SNPs) associated with the same trait in a complex pedigree of loblolly pine.</p><p id="p-7">Quantifying the genome-wide extent of linkage disequilibrium (LD) within genotype-phenotype discovery is relevant for association genetics and genomic selection. LD or the nonindependence of segregating alleles at different genomic loci may arise from two loci being in close proximity on a chromosome, thus reducing the probability of recombination between them (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Flint-Garcia <em>et al.</em> 2003</a>).</p><p id="p-8">Statistical power to detect an association between a marker and a trait is inversely proportional to the squared correlations (r<sup>2</sup>) between alleles at a marker locus and the causal variant (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pritchard and Przeworski 2001</a>). Thus, quantifying the extent of LD is useful for knowing marker densities required to represent nonrecombining haplotype segments within discovery populations (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yan <em>et al.</em> 2009</a>). LD among distant loci on the same chromosome or loci on different chromosomes may also occur because of subpopulation structure, kinship, inbreeding, directional selection, and epistasis (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gaut and Long 2003</a>). Estimating LD between distant loci on the same chromosome or on different chromosomes is useful for detecting the possibility of false-positive associations (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Platt <em>et al.</em> 2010</a>). Within outcrossing populations of conifers with large effective population sizes, r<sup>2</sup> decays rapidly to less than 0.1 over 500 to 1500 bases (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Neale and Savolainen 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pavy <em>et al.</em> 2012a</a>). However, within multi-family pedigrees used for genetic association and genomic selection studies, r<sup>2</sup> is expected to decay over greater distances proportional to levels of relatedness (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Flint-Garcia <em>et al.</em> 2003</a>).</p><p id="p-9">In the present study, a consensus genetic map was constructed by merging three gene-based linkage maps for <em>P. taeda</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Echt <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Martínez-García <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Neves <em>et al.</em> 2014</a>) with a linkage map from a pseudo-backcross between <em>P. elliottii</em> and <em>P. taeda</em> (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Westbrook <em>et al.</em> 2015</a>). The consensus map positioned 3856 markers via genotyping 1251 individuals from three full-sib populations and one haploid population. Improved functional annotations of mapped genes were obtained by aligning the partial length expressed sequence tags (ESTs) used for marker discovery against longer <em>P. taeda</em> transcript assemblies from RNA-seq data (NCBI BioProject PRJNA174450) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wegrzyn <em>et al.</em> 2014</a>). The consensus map was used to compare the genome-wide extent of LD in ADEPT2, a population composed of unrelated individuals that has been used for association genetic studies (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Quesada <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eckert <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cumbie <em>et al.</em> 2011</a>), and CCLONES, a complex multi-family population that has been used for genomic selection studies (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Resende <em>et al.</em>, 2012a</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">b</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Westbrook <em>et al.</em> 2013</a>, <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">2015</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Linkage maps used to construct the consensus map</h3><p id="p-10">A consensus map was constructed by merging two composite maps from the <em>P. taeda</em> QTL and BASE pedigrees (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Echt <em>et al.</em> 2011</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Martínez-García <em>et al.</em> 2013</a>) with a BC1 map from a (<em>P. taeda</em> × <em>P. elliottii</em>) × <em>P. elliottii</em> pseudo-backcross and a haploid map from the <em>P. taeda</em> clone 10-5 (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Neves <em>et al.</em> 2014</a>). The composition of marker types in the four input maps is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The first QTL-BASE map (QTL-BASE1) contained 460 markers (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/019588SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS1.txt">File S1</a>) genotyped in single F2 cohorts of the QTL and BASE pedigrees (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Echt <em>et al.</em> 2011</a>). The second QTL-BASE map (QTL-BASE2) contained 2466 markers (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS2.txt">File S2</a>) genotyped in two F2 full-sib cohorts in each of the BASE and QTL pedigrees (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Martínez-García <em>et al.</em> 2013</a>). Both QTL-BASE maps were constructed in JoinMap with regression mapping (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Van Ooijen 2011</a>). Linkage group (LG) numbers and orientation in all input maps were modified to match the historical designations of <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Echt <em>et al.</em> (2011)</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466496/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466496/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Composition of input maps used to construct consensus genetic map for <em>Pinus taeda</em> and <em>Pinus elliottii</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Reconstruction of the BC1 input linkage map</h3><p id="p-12">A composite map of the BC1 and 10-5 populations was presented in <a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Westbrook <em>et al.</em> (2015)</a>. Due to large differences in progeny sizes between these populations (490 diploid individuals in BC1 <em>vs.</em> 72 haploid megagametophytes in 10-5), we reconstructed a genetic map of the BC1 population separately from the 10-5 map prior to its integration into the current consensus map. The BC1 pseudo-backcross population originated from controlled pollination of a <em>P. taeda</em> × <em>P. elliottii</em> var. <em>elliottii</em> F1 hybrid with pollen from a second <em>P. elliotti</em> var. <em>elliottii</em> individual (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Muñoz <em>et al.</em> 2011</a>). Full-sib BC1 progeny, their parents, and the maternal <em>P. elliottii</em> grandmother were genotyped at 4861 SNPs discovered within ESTs with an Illumina Infinium assay designed for <em>P. taeda</em> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Eckert <em>et al.</em> 2010</a>). Loci that were monoallelic, missing parental genotypes, displayed significant segregation distortion at <em>P</em> &lt; 0.001, or had more than 5% genotyping error rate as inferred from parental genotypes were discarded. For loci that contained genotype errors in less than 5% of the individuals, the erroneous genotypes were recoded as missing data. For ESTs containing more than one SNP, the marker with the highest genotype information content (described below) was selected for mapping. The BC1 map containing 803 SNPs (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS3.txt">File S3</a>) was constructed in JoinMap v. 4.1 (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Van Ooijen 2011</a>) by specifying the cross-pollinated (CP) population type, a linkage group LOD score threshold of 6, and the Kosambi mapping function. Map positions from the third round of regression mapping were used.</p></div><div id="sec-4" class="subsection"><h3>Reconstruction of the 10-5 linkage map</h3><p id="p-13">The 10-5 map was originally constructed from exome resequencing of 72 haploid megagametophytes from a single tree and contained 2841 markers (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Neves <em>et al.</em> 2014</a>). Large regions of reversed marker order were observed on 5 of 12 linkage groups, when the 10-5 map was compared to a previously published genetic map of the <em>P. taeda</em> QTL population (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Eckert <em>et al.</em> 2010</a>). Reversals on these five LGs (4, 5, 6, 7, and 8) were also observed when the marker order of the 10-5 map was compared with the BC1 map (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS1.pdf">Figure S1</a>) and the QTL-BASE2 map (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS2.pdf">Figure S2</a>). By contrast, strong colinearity was observed among QTL-BASE1, QTL-BASE2, and BC1 maps (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS4.pdf">Figure S4</a>). Considering that the 10-5 map was constructed from genotyping a relatively small population (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) at a low median sequence depth of 2.7× (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Neves <em>et al.</em> 2014</a>), marker order reversals in the 10-5 map were likely attributable to genotyping or mapping errors instead of cytological rearrangements (Maleipaard <em>et al.</em> 1997). Based on this hypothesis, the 10-5 map was reconstructed in three steps that took advantage of the “fixed order” and “start order” map building functions in JoinMap v.4.1: (1) define a fixed order for a subset of reference loci obtained from two of the other mapping pedigrees; (2) define a robust start order for a subset of loci on a preliminary 10-5 map built from markers having the lowest levels of missing genotypes and suspect linkages; and (3) specify limited sets of fixed order and start order loci to aid reconstruction of the 10-5 map.</p><p id="p-14">First, a reference marker order was obtained by merging the BC1 and QTL-BASE2 maps with MergeMap (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2011</a>) to obtain a composite map that shared a greater number of markers with the 10-5 map as compared to the individual BC1 or QTL-BASE2 maps. For each LG, 6 to 19 loci were identified for use as fixed order reference loci based on their being separated by than more than 10 cM and having the highest genotype information content (described below) among neighboring loci.</p><p id="p-15">Second, a preliminary 10-5 map was constructed from loci that had less than 12.5% missing genotypes to help define a starting marker order. Two rounds, not three, of regression mapping in JoinMap v. 4.1 were used to ensure that no loci had negative recombination intervals and that all loci had goodness-of-fit chi-square values greater than 3. The following mapping parameters were specified: a haploid population type; a linkage group LOD threshold = 6; mapping LOD threshold = 1.5; recombination threshold = 0.44; jump threshold = 3.0; ripple value = 1; and the Kosambi mapping function. After an initial round of mapping, loci were excluded from subsequent maps if they had a high frequency of suspect pairwise linkages, recombination fractions &gt;0.60 and mapping LOD &gt;1.5, high nearest neighbor fit values and low locus mean genotype probabilities, or if they had prominent order conflicts with the BC1/QTL-BASE2 reference map. Iterative exclusion of loci and remapping continued until there was no further improvement in alignment to the BC1/QTL-BASE2 reference map. Start order loci were then identified based on their being separated by more than 5 cM, having the highest genotype information content among neighboring loci, and having marker orders not in conflict with the BC1/QTL-BASE2 reference map order. Between 11 and 28 start order loci were selected for each LG.</p><p id="p-16">Third, starting with genotype data for all loci reported by <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Neves <em>et al.</em> (2014)</a>, the final 10-5 map was reconstructed by specifying the fixed orders and start orders obtained from the first two steps. Iterative exclusion of loci and remapping were then performed using the parameters and protocols described in the second step above. Loci at the ends of LGs that were more than 10 cM apart from adjacent loci were also excluded. Mapping iterations continued until there was no further improvement in map quality as assessed by chi-square values for each LG or alignment to the BC1/QTL-BASE2 reference map. <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS1.pdf">Table S1</a> summarizes the number of start order loci, fixed order loci, and markers used to reconstruct the 10-5 map (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS4.txt">File S4</a>).</p></div><div id="sec-5" class="subsection"><h3>Marker selection for the consensus map</h3><p id="p-17">The majority of loci shared among the BC1, BASE-QTL2, and 10-5 maps were SNPs discovered within expressed sequence tags (ESTs). To construct a consensus map, SNP loci were merged based on EST names after omitting the nucleotide position of the SNP within the name. Nucleotide positions in the 10-5 map did not correspond to the positions in the QTL-BASE2 and BC1 maps because the SNPs were discovered in different populations and were based on different alignments. To merge the BC1 and 10-5 maps, which contained one SNP per EST, to the QTL-BASE2 consensus map, which contained one to three SNPs per EST, it was necessary to select the most informative SNP within ESTs. The QTL-BASE1 map was merged with the other maps via simple sequence repeats (SSR) and restriction fragment length polymorphisms shared with the QTL-BASE2 map.</p><p id="p-18">Marker informativeness was measured with genotype information content (GIC) calculated as the effective number of genotypic classes (<em>G</em><sub>e</sub>) times the number of individuals genotyped (<em>N</em><sub>indiv</sub>). The effective number of genotypic classes for each marker was calculated as <em>G</em><sub>e</sub> = <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-1.gif"/></noscript></span></span> where <em>p</em><sub>j</sub><sup>2</sup> is the squared proportion of the <em>j</em><sup>th</sup> genotypic class. For biparentally heterozygous (hk × hk) loci, which segregate 1hh:2hk:1kk, <em>G</em><sub>e</sub> was calculated using only the homozygous (hh and kk) genotypic classes, and GIC was calculated using the total number of genotyped progeny. The heterozygous (hk) genotype was excluded from the calculation of <em>G</em><sub>e</sub> to maintain an inverse linear proportionality of GIC with segregation distortion chi-square values. For genes genotyped at more than one SNP marker, only the marker with the maximum GIC summed across population cohorts was retained in the QTL-BASE2 map. <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS1.txt">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS2.txt">File S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS3.txt">File S3</a>, and <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS4.txt">File S4</a> contain the input maps with GIC used for marker selection.</p></div><div id="sec-6" class="subsection"><h3>Map weights and construction of the consensus map</h3><p id="p-19">Constructing a consensus map directly from recombination frequencies in JoinMap is computationally time-consuming and infeasible for multiple populations and cohorts (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wenzl <em>et al.</em> 2006</a>). Instead, the consensus map was constructed directly from the marker names and genetic distances in the input maps using two linear programming algorithms, MergeMap (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2011</a>; <a href="http://www.mergemap.org">http://www.mergemap.org</a>) and LPmerge (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Endelman and Plomion 2014</a>; <a href="http://cran.r-project.org/web/packages/LPmerge/">http://cran.r-project.org/web/packages/LPmerge/</a>). MergeMap uses directed acyclic graphs of shared markers to merge input maps and resolves marker order conflicts by deleting the minimum number of marker occurrences from the input maps. LPmerge groups markers into positional bins and seeks the bin order that minimizes the root mean squared error between the input maps and consensus map. For both approaches, the resolution of marker order conflicts was informed by weighting input maps in proportion to the average GIC of the markers contained in each map. This weighting method assigned confidence to maps in proportion to the average number of individuals and genotypic classes genotyped and in inverse proportion to the average segregation distortion of the markers. For the QTL-BASE1 and QTL-BASE2 maps, which were constructed from two and four cohorts of the BASE and QTL populations, respectively, marker GIC was summed across all cohorts prior to averaging across markers. The average GIC for the <em>l</em><sup>th</sup> input map constructed from <em>k</em> cohorts and <em>j</em> markers was calculated as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-2.gif"/></noscript></span></span>where <em>GIC<sub>j</sub></em><sub>(</sub><em><sub>k</sub></em><sub>)</sub> is the GIC of the <em>j</em><sup>th</sup> marker within the <em>k</em><sup>th</sup> cohort and <em>N<sub>j(l)</sub></em> is the number of markers within the <em>l</em><sup>th</sup> map. Map weights were scaled from 0 to 1 by dividing <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-3.gif"/></noscript></span></span> by the sum of the average GIC across maps, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1685/embed/mml-math-4.gif"/></noscript></span></span>. The centimorgan positions of markers varied between runs of MergeMap but did not vary between runs of LPmerge. To construct a consensus map in MergeMap, the average position of and standard error of marker positions were obtained from 100 replicate runs. Root mean squared errors (RMSE) in marker order between the consensus maps and the input maps and between the MergeMap and LPmerge consensus maps were calculated with the R package hydroGOF (<a href="http://cran.r-project.org/web/packages/hydroGOF/">http://cran.r-project.org/web/packages/hydroGOF/</a>). The consensus map with the lowest average RMSE with the input maps was used for further analysis.</p></div><div id="sec-7" class="subsection"><h3>Alignment of mapped genes to the <em>P. taeda</em> genome and transcriptome</h3><p id="p-20">Expressed sequence tags containing mapped markers (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS5.txt">File S5</a>) were aligned to <em>P. taeda</em> genome assembly version 1.01 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Neale <em>et al.</em> 2014</a>) using GMAP (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wu and Watanabe 2005</a>). For ESTs that aligned with more than one genomic scaffold or scaffolds that aligned with two or more ESTs on different LGs, the most precise alignment was chosen based on alignment length and DNA sequence identity. BlastN (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) 2.2.27+ was used to align ESTs to transcript sequences assembled from RNA-seq reads of the <em>P. taeda</em> reference genotype 20-1010 and other genotypes (K. Mockaitis, data available in NCBI BioProject PRJNA174450). Predicted functions of coding sequences were provided from results of Delta-BlastP (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boratyn <em>et al.</em> 2012</a>) alignments of complete transcript protein sequences to the <em>Arabidopsis thaliana</em> TAIR10 annotation protein set (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lamesch <em>et al.</em> 2011</a>) and to the NCBI Conserved Domain Database (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Marchler-Bauer <em>et al.</em> 2013</a>).</p></div><div id="sec-8" class="subsection"><h3>Spatial density of mapped genes along chromosomes</h3><p id="p-21">The density of markers along LGs was estimated with kernel density estimation in R (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">R Core Team 2012</a>). Fixed bandwidths (<em>bw</em>) for the Gaussian kernel density estimator were calculated for each LG following the work of <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Silverman (1986)</a>:</p><p id="p-22"><em>bw</em> = 0.9s/1.34<em>n</em><sup>−1/5</sup>, where s is the SD of marker positions in cM and <em>n</em> is the number of markers per linkage group. Kernel density estimates were multiplied by <em>n</em> × <em>bw</em> to obtain the number of markers per bandwidth and by <em>n</em> to obtain the number of markers per centimorgan. To test for regions where marker density significantly deviated from random expectation, the observed numbers of markers per bandwidth were compared to the 95% C.I. of a Poisson distribution with mean and variance equal to <em>n</em>/(LG length/<em>bw</em>).</p></div><div id="sec-9" class="subsection"><h3>Analysis of linkage disequilibrium in two populations of <em>P. taeda</em></h3><p id="p-23">The consensus map was used to compare LD in CCLONES (Comparing Clonal Lines ON Experimental Sites), a pedigreed population of <em>P. taeda</em> used for genomic selection studies, and ADEPT2 (Allele Discovery of Economic Pine Traits 2), a population of unrelated <em>P. taeda</em> individuals used for association genetic studies. The CCLONES population was composed of 923 progeny in 68 full-sib families generated from circular mating among 54 first- and second-generation selections from breeding programs in Florida, the Atlantic coastal plain, and the lower Gulf states (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baltunis <em>et al.</em> 2007</a>). The ADEPT2 population consisted of 427 distantly related <em>P. taeda</em> individuals sampled from throughout the species range (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Eckert <em>et al.</em> 2010</a>). Both populations were genotyped with an Illumina Infinium assay of 7216 SNPs discovered within ESTs (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Eckert <em>et al.</em> 2010</a>). Polymorphisms were detected at 3938 SNP loci within 3347 ESTs in ADEPT2 and at 4854 SNP loci within 4027 ESTs in CCLONES (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Westbrook <em>et al.</em> 2013</a>). Within ADEPT2 and CCLONES, respectively, 575 and 805 ESTs were genotyped at two or more SNP loci, enabling estimation of LD within genes.</p><p id="p-24">LD between SNPs at different positions on the consensus map was estimated in both populations before and after adjusting r<sup>2</sup> values for kinship or subpopulation structure with the R package LDcorSV (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Mangin <em>et al.</em> 2012</a>). Kinship was estimated from identity-by-descent proportions (IBD) expected from pedigree relationships (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Henderson 1976</a>). In CCLONES, IBD varied from 0 to 0.5; however, average IBD between all individuals was small (0.043) due to the fact that 80% of individuals were unrelated (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS2.pdf">Table S2</a>). In ADEPT2, IBD was zero among distantly related individuals. Subpopulation structure was inferred from SNPs with minor allele frequencies greater than 0.05 with the program fastSTRUCTURE (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Raj <em>et al.</em> 2014</a>). In CCLONES, the number of subpopulation clusters (K) tested varied from 2 to 10 using 3037 SNPs. A continuous increase in marginal likelihood with increasing K was observed, indicating that subpopulation structure was weak or nonexistent (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS5.pdf">Figure S5</a>). In ADEPT2, K = 2…7 were tested with 2910 SNPs, and K = 2 had the greatest marginal likelihood (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS5.pdf">Figure S5</a>). The K = 2 structure matrix was not invertible; therefore, the structure matrix with three subpopulations (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS6.txt">File S6</a>), the second most likely K, was utilized to account for structure in the estimation of LD in ADEPT2. Kernal regression of r<sup>2</sup> <em>vs.</em> genetic distance was performed with the ksmooth function in R. Plots of r<sup>2</sup> within genes, within LGs, and between LGs were prepared in ggplot2 for R (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wickham 2009</a>).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Comparisons of the input maps used to construct the consensus genetic map for <em>Pinus taeda</em> and <em>Pinus elliottii</em></h3><p id="p-25">Based on the hypothesis that large regions of reversed marker order on five LGs of the 10-5 map were artifacts of small population size and genotyping errors (see <em>Materials and Methods</em>), the 10-5 map was reconstructed by specifying limited sets of selected fixed order and start order loci, and then subsequently excluding poor-fitting loci through an iterative mapping procedure (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS1.pdf">Table S1</a>). In total, 1466 markers from the original 10-5 map were excluded to reconstruct a 10-5 map containing 1375 markers (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS4.txt">File S4</a>). The reconstructed 10-5 map was collinear with the QTL-BASE2 and BC1 maps (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS7.pdf">Figure S7</a>).</p><p id="p-26">The four input maps were merged with 69 to 497 markers that were shared between pairs of maps (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS3.pdf">Table S3</a>). The QTL-BASE1 map, composed of simple sequence repeat (SSR), restriction fragment length polymorphism (RFLP), and expressed sequence tag polymorphism (ESTP) markers, did not share markers with the 10-5 and BC1 maps, which were composed primarily of SNPs (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The QTL-BASE1 map was integrated into the consensus map via markers shared with the QTL-BASE2 map. The 54 markers on LGs 7 and 12 from the QTL-BASE1 (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Echt <em>et al.</em> 2011</a>) (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS1.txt">File S1</a>) were excluded from the merge to create the consensus map because QTL-BASE1 shared only two markers with QTL-BASE2 on these LGs. A total of 412 SNP markers were excluded from the QTL-BASE2 map prior to consensus merging because they occurred within expressed sequences containing SNPs at other nucleotide positions that had higher genotype information content (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-12" class="subsection"><h3>Comparisons of the consensus maps from two map merging algorithms</h3><p id="p-27">The consensus maps generated by MergeMap (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS7.txt">File S7</a>) and LPmerge (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS8.txt">File S8</a>) each contained 3856 markers and were strongly collinear with each other (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS8.pdf">Figure S8</a>). The total length of the MergeMap consensus map (2305 cM) was 1.2- to 1.7-times longer than the lengths of the individual input maps, whereas the length of the LPmerge consensus (1802 cM) was within the range of the lengths of the input maps (1378–1910 cM) (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Where there was uncertainty in marker order between the consensus maps, LPmerge binned markers into the same map positions, whereas MergeMap assigned unique positions to most markers (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS4.pdf">Table S4</a>). This nonbinning attribute of MergeMap accounts for the length expansion of its consensus map compared to the LPmerge map. The MergeMap consensus had lower average RMSE in marker order with the input maps for 10 of 12 LGs (LGs 1–10) as compared to the LPmerge consensus (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS5.pdf">Table S5</a>). Therefore, the MergeMap consensus map was used for subsequent analyses.</p></div><div id="sec-13" class="subsection"><h3>Summary of the consensus map and alignment to the <em>P. taeda</em> genome and transcriptome</h3><p id="p-28">Strong colinearity was observed between the MergeMap consensus and the four input maps (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The consensus map positioned 3353 SNPs discovered within ESTs, 175 restriction fragment length polymorphisms (RFLPs), 126 noncoding simple sequence repeats (SSRs), 114 expressed sequence tag polymorphisms (ESTPs), 71 SSRs within ESTs, and 17 presence/absence variants (PAVs) of ESTs (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS7.txt">File S7</a>). Of the 3856 markers mapped, 3639 (94%) aligned to <em>P. taeda</em> transcript assemblies with an average sequence identity of 98.8%. Predicted functions for 3082 mapped ESTs were obtained through alignment of the translated transcripts to 2385 unique <em>Arabidopsis</em> protein sequences in the TAIR10 database and to 1419 unique conserved domains in the NCBI CDD database. A total of 3762 mapped markers aligned to 3305 <em>P. taeda</em> version 1.01 genomic scaffolds (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zimin <em>et al.</em> 2014</a>) with 99.0% average nucleotide sequence identity. Of the 357 genomic scaffolds that aligned with more than one marker, 168 scaffolds aligned with markers on different LGs and 189 scaffolds aligned with markers from the same LG. Most scaffolds were too short to span multiple markers precluding unbiased estimation of physical to genetic distance.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of orders of shared markers between input maps and the MergeMap consensus genetic map of Pinus taeda and Pinus elliottii. Linkage group (LG) lengths were standardized to 100 units for comparison among maps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919829504" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparisons of orders of shared markers between input maps and the MergeMap consensus genetic map of &lt;em&gt;Pinus taeda&lt;/em&gt; and &lt;em&gt;Pinus elliottii&lt;/em&gt;. Linkage group (LG) lengths were standardized to 100 units for comparison among maps.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1685/F1.medium.gif" width="307" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/8/1685/F1.medium.gif" width="307" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466549" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">Comparisons of orders of shared markers between input maps and the MergeMap consensus genetic map of <em>Pinus taeda</em> and <em>Pinus elliottii</em>. Linkage group (LG) lengths were standardized to 100 units for comparison among maps.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Between 275 and 351 markers were positioned on individual linkage groups, and linkage group lengths varied from 147 cM to 222 cM (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Average and maximum distances between markers were 0.60 cM and 8.47 cM, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Average 95% C.I.s for marker positions, estimated over 100 MergeMap runs, varied from 0.15 cM to 1.75 cM for individual LGs and was 0.90 cM genome-wide. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> displays a segment of the consensus map (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS9.pdf">Figure S9</a>), which includes marker types, positions, 95% C.I.s of positions, and presence or absence of markers in the input maps.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466547/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466547/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of the consensus genetic map for <em>Pinus taeda and Pinus elliottii</em> by linkage group (LG)</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pinus taeda and Pinus elliottii consensus map features; centimorgan (cM) scale left of bar, locus mean consensus position along bar; GeneID right of bar; marker type (font color): SNP and PAV (black), SSR (red), ESTP and RFLP (green); the variable next to GeneID indicates whether the locus was present (x) or absent (0) on the four input maps, listed in the following order: QTL-BASE1, QTL-BASE2, BC1, 10-5; far right column denotes cM range of upper and lower bounds of the 95% C.I. for the locus position: 0&#x2013;1 cM (+) and 1&#x2013;2 cM (++). Detail shown is from LG-4. For full map graphic, see Figure S9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919829504" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Pinus taeda&lt;/em&gt; and &lt;em&gt;Pinus elliottii&lt;/em&gt; consensus map features; centimorgan (cM) scale left of bar, locus mean consensus position along bar; GeneID right of bar; marker type (font color): SNP and PAV (black), SSR (red), ESTP and RFLP (green); the variable next to GeneID indicates whether the locus was present (x) or absent (0) on the four input maps, listed in the following order: QTL-BASE1, QTL-BASE2, BC1, 10-5; far right column denotes cM range of upper and lower bounds of the 95% C.I. for the locus position: 0&#x2013;1 cM (+) and 1&#x2013;2 cM (++). Detail shown is from LG-4. For full map graphic, see Figure S9.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1685/F2.medium.gif" width="258" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/8/1685/F2.medium.gif" width="258" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466665" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child"><em>Pinus taeda</em> and <em>Pinus elliottii</em> consensus map features; centimorgan (cM) scale left of bar, locus mean consensus position along bar; GeneID right of bar; marker type (font color): SNP and PAV (black), SSR (red), ESTP and RFLP (green); the variable next to GeneID indicates whether the locus was present (x) or absent (0) on the four input maps, listed in the following order: QTL-BASE1, QTL-BASE2, BC1, 10-5; far right column denotes cM range of upper and lower bounds of the 95% C.I. for the locus position: 0–1 cM (+) and 1–2 cM (++). Detail shown is from LG-4. For full map graphic, see <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS9.pdf">Figure S9</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Density of markers in the consensus map and in two <em>P. taeda</em> genotype-phenotype discovery populations</h3><p id="p-32">Marker density (N cM<sup>−1</sup>) in the consensus map varied from 0.25 to 2.76, with a mean of 1.6 across LGs. Observed variation in the number of markers per cM in the consensus map was compared to random variation in marker density expected from a Poisson distribution (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Lower than expected marker densities were observed toward the distal ends of all 12 LGs, whereas greater than expected marker densities were observed in putative centromeric regions of LGs 2, 3, and 12. The consensus map positioned 2673 of 3938 SNPs (68%) genotyped in unrelated individuals from the <em>P. taeda</em> ADEPT2 population and 2829 of 4854 SNPs (58%) in the CCLONES multiple-family pedigree. The density of mapped markers varied from 0.10 to 1.43 in ADEPT2 (mean = 0.85) and from 0.1 to 1.54 in CCLONES (mean = 0.91) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Kernel density estimation of mapped markers in the MergeMap consensus genetic map and in two genotype-phenotype discovery populations of Pinus taeda. (A) Marker densities in the consensus map (red lines) were compared to the 95% C.I. of a Poisson distribution (gray regions) of random deviations from uniform marker densities. The densities of SNPs mapped in (B) ADEPT2 (unrelated association) or (C) CCLONES (multiple-family pedigree) populations (black lines) were compared against marker densities in the consensus map (red lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919829504" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Kernel density estimation of mapped markers in the MergeMap consensus genetic map and in two genotype-phenotype discovery populations of &lt;em&gt;Pinus taeda&lt;/em&gt;. (A) Marker densities in the consensus map (red lines) were compared to the 95% C.I. of a Poisson distribution (gray regions) of random deviations from uniform marker densities. The densities of SNPs mapped in (B) ADEPT2 (unrelated association) or (C) CCLONES (multiple-family pedigree) populations (black lines) were compared against marker densities in the consensus map (red lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1685/F3.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/8/1685/F3.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466466" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Kernel density estimation of mapped markers in the MergeMap consensus genetic map and in two genotype-phenotype discovery populations of <em>Pinus taeda</em>. (A) Marker densities in the consensus map (red lines) were compared to the 95% C.I. of a Poisson distribution (gray regions) of random deviations from uniform marker densities. The densities of SNPs mapped in (B) ADEPT2 (unrelated association) or (C) CCLONES (multiple-family pedigree) populations (black lines) were compared against marker densities in the consensus map (red lines).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Patterns of linkage disequilibrium in unrelated and pedigreed populations of <em>P. taeda</em></h3><p id="p-34">The r<sup>2</sup> values were first estimated without accounting for kinship in CCLONES or subpopulation structure in ADEPT2 to compare baseline LD between populations at different scales (<em>i.e.</em>, within genes, within LGs, and between LGs). The distributions of r<sup>2</sup> between SNPs within genes were bimodal in CCLONES and ADEPT2, with a high frequency of r<sup>2</sup> values that were approximately 0 and a lower frequency of r<sup>2</sup> values that were approximately 1 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Average r<sup>2</sup> (±1 SE) between markers separated by less than 1 cM within LGs of the consensus map was 0.027 (±0.001) in ADEPT2 and 0.048 (±0.001) in CCLONES. Percentages of SNP pairs less than 1 cM apart with r<sup>2</sup> greater than 0.1 varied from 4.2% to 12.6% in ADEPT2 and from 7.3% to 18.1% in CCLONES, depending on minimum minor allele frequency (MAF) thresholds (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Average LD between markers on the same LGs did not decay substantially with genetic distance in ADEPT2 and CCLONES (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of r2 between pairs of SNP loci within genes in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of P. taeda" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919829504" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of r&lt;sup&gt;2&lt;/sup&gt; between pairs of SNP loci within genes in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of &lt;em&gt;P. taeda&lt;/em&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1685/F4.medium.gif" width="299" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/8/1685/F4.medium.gif" width="299" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466510" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Distributions of r<sup>2</sup> between pairs of SNP loci within genes in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of <em>P. taeda</em></p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466548/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466548/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number and percentage of SNP pairs among different genes with r<sup>2</sup> &gt; 0.1 for three linkage classes</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium r2 values between loci on the same LGs as a function of genetic distance in ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of P. taeda. Displayed are values of r2 between mapped SNPs in different ESTs with minor allele frequencies greater than 0.1 and with less than 50% missing data. Red lines are kernel regressions of r2 vs. genetic distance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919829504" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage disequilibrium r&lt;sup&gt;2&lt;/sup&gt; values between loci on the same LGs as a function of genetic distance in ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of &lt;em&gt;P. taeda&lt;/em&gt;. Displayed are values of r&lt;sup&gt;2&lt;/sup&gt; between mapped SNPs in different ESTs with minor allele frequencies greater than 0.1 and with less than 50% missing data. Red lines are kernel regressions of r&lt;sup&gt;2&lt;/sup&gt; &lt;em&gt;vs.&lt;/em&gt; genetic distance.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1685/F5.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/8/1685/F5.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Linkage disequilibrium r<sup>2</sup> values between loci on the same LGs as a function of genetic distance in ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of <em>P. taeda</em>. Displayed are values of r<sup>2</sup> between mapped SNPs in different ESTs with minor allele frequencies greater than 0.1 and with less than 50% missing data. Red lines are kernel regressions of r<sup>2</sup> <em>vs.</em> genetic distance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Extended LD within LGs, defined as SNPs more than 1 cM apart with r<sup>2</sup> values greater than 0.1, was rare in ADEPT2, occurring between 0.08% and 0.22% of locus pairs (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Extended LD was more prevalent in CCLONES, occurring between 1.5% and 1.9% of SNP pairs. Furthermore, the range of genetic distances over which extended LD was observed was greater in CCLONES as compared to ADEPT2 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). LD between SNPs on different LGs was rare in both populations, but CCLONES had a greater percentage of SNP pairs on different LGs with r<sup>2</sup> &gt; 0.1 (0.6%–0.8%) as compared to ADEPT2 (&lt;0.1%) (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Pairs of mapped and unmapped SNPs with r<sup>2</sup> &gt; 0.1 in ADEPT2 and CCLONES are reported in <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS9.zip">File S9</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Extended linkage disequilibrium within linkage groups in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of P. taeda. Values of r2 vs. genetic distance are plotted for SNP pairs with r2 &gt; 0.1 that were more than 1 cM apart (A) before and (B) after accounting for population structure in ADEPT2 and kinship in CCLONES. The 95% confidence ellipses of r2 vs. genetic distance are depicted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919829504" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Extended linkage disequilibrium within linkage groups in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of &lt;em&gt;P. taeda&lt;/em&gt;. Values of r&lt;sup&gt;2&lt;/sup&gt; &lt;em&gt;vs.&lt;/em&gt; genetic distance are plotted for SNP pairs with r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.1 that were more than 1 cM apart (A) before and (B) after accounting for population structure in ADEPT2 and kinship in CCLONES. The 95% confidence ellipses of r&lt;sup&gt;2&lt;/sup&gt; &lt;em&gt;vs.&lt;/em&gt; genetic distance are depicted in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1685/F6.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/8/1685/F6.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-39" class="first-child">Extended linkage disequilibrium within linkage groups in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of <em>P. taeda</em>. Values of r<sup>2</sup> <em>vs.</em> genetic distance are plotted for SNP pairs with r<sup>2</sup> &gt; 0.1 that were more than 1 cM apart (A) before and (B) after accounting for population structure in ADEPT2 and kinship in CCLONES. The 95% confidence ellipses of r<sup>2</sup> <em>vs.</em> genetic distance are depicted in red.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>The effects of subpopulation structure and kinship on linkage disequilibrium</h3><p id="p-40">Adjusting r<sup>2</sup> for structure in ADEPT2 or kinship in CCLONES reduced number of locus pairs with r<sup>2</sup> &gt; 0.1 in both populations (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Larger reductions in the extent of LD within LGs (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>) and the percentages of SNPs in LD between LGs (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) were observed after adjusting for kinship in CCLONES as compared to adjusting for structure in ADEPT2. Some cases of extended LD within LGs may have been attributable to error in the estimation of marker positions or spurious effects due to low minor allele frequency (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Plomion <em>et al.</em> 2014</a>). Considering only pairs of SNPs where both loci were separated by more than 5 cM within a single input map, and considering that both SNPs had MAF &gt;0.1, 135 SNP pairs in CCLONES and seven SNP pairs in ADEPT2 had adjusted r<sup>2</sup> &gt; 0.1. No SNP pairs on different LGs had adjusted r<sup>2</sup> &gt; 0.1 in ADEPT2, and 558 SNP pairs had adjusted r<sup>2</sup> &gt; 0.1 in CCLONES (MAF &gt;0.1) (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). To test for possible epistatic LD between SNPs on different LGs in CCLONES, the distribution of adjusted r<sup>2</sup> values from 135 SNP pairs with strong evidence for extended LD within LGs was used to estimate a critical value of r<sup>2</sup> &gt; 0.82 that exceeded the Bonferroni significance threshold (α = 0.05/558 tests). No locus pairs on different LGs had adjusted r<sup>2</sup> that exceeded the 0.82 significance threshold in CCLONES (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium between SNPs on different linkage groups in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of P. taeda. Distributions of r2 were plotted for SNP pairs on different LGs with r2 &gt; 0.1. The distributions were compared (A) before and (B) after adjusting r2 values for subpopulation structure in ADEPT2 and kinship in CCLONES." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919829504" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage disequilibrium between SNPs on different linkage groups in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of &lt;em&gt;P. taeda&lt;/em&gt;. Distributions of r&lt;sup&gt;2&lt;/sup&gt; were plotted for SNP pairs on different LGs with r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.1. The distributions were compared (A) before and (B) after adjusting r&lt;sup&gt;2&lt;/sup&gt; values for subpopulation structure in ADEPT2 and kinship in CCLONES.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1685/F7.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/8/1685/F7.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1685/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466414" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-41" class="first-child">Linkage disequilibrium between SNPs on different linkage groups in the ADEPT2 (unrelated association) and CCLONES (multiple-family pedigree) populations of <em>P. taeda</em>. Distributions of r<sup>2</sup> were plotted for SNP pairs on different LGs with r<sup>2</sup> &gt; 0.1. The distributions were compared (A) before and (B) after adjusting r<sup>2</sup> values for subpopulation structure in ADEPT2 and kinship in CCLONES.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-42">The <em>Pinus taeda</em> and <em>Pinus elliotti</em> consensus map positioned 3555 polymorphic transcripts and 301 noncoding markers that segregated in four mapping populations composed of 1251 individuals. The consensus map positioned 1.3- to 8.4-times the number of markers of the input maps (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>) and genetically mapped approximately 7% of the 50,172 genes predicted for <em>P. taeda</em> (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Neale <em>et al.</em> 2014</a>). Alignment of markers in the consensus map to the <em>Pinus taeda</em> genome assembly version 1.01 (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Neale <em>et al.</em> 2014</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Zimin <em>et al.</em> 2014</a>) positioned 3305 scaffolds onto LGs. Improvement in the functional predictions for 3082 mapped genes was attained by aligning partial length ESTs (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FileS5.txt">File S5</a>) to longer <em>P. taeda</em> transcript assemblies.</p><p id="p-43">Colinearity among the QTL-BASE1, QTL-BASE2, and BC1 input maps, in contrast with large marker order reversals on five linkage groups in the original 10-5 map (<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Neves <em>et al.</em> 2014</a>), indicated that the order reversals in the 10-5 map were likely to be mapping errors (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS4.pdf">Figure S4</a>). This interpretation was supported by the strong synteny of homologous markers among conifer genetic maps (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brown <em>et al.</em> 2001</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Krutovsky <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pavy <em>et al.</em> 2012b</a>) and the fact that the 10-5 map was constructed from a much smaller population of 72 haploid megagametophytes (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Reconstruction of the 10-5 map by specifying subsets of fixed and start order loci and removing loci with suspect linkages produced a map that was collinear with the other input maps (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/FigureS7.pdf">Figure S7</a>) and the consensus map (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-44">Marker density in the consensus map varied from less than one marker per cM toward the distal ends of LGs to three markers per cM toward the middle of LGs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Similar patterns of marker densities have been observed in the consensus genetic maps of <em>Picea glauca</em> (white spruce) and <em>Picea mariana</em> (black spruce) and <em>Pinus pinaster</em> (maritime pine) (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Pavy <em>et al.</em> 2012b</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Plomion <em>et al.</em> 2014</a>). Regions of high marker density may occur in centromeric regions with reduced recombination rates, whereas regions of low marker density are associated with telomeres with higher rates of recombination. Power to detect associations between markers and traits is greater in regions with reduced recombination rates, but the genomic resolution to fine-map causal variants in these regions is reduced (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nachman 2002</a>).</p><div id="sec-18" class="subsection"><h3>Comparison of LD extent between discovery populations: implications for comparative QTL and association mapping across populations</h3><p id="p-45">LD was expected to be rare and to decay rapidly among distantly related individuals in ADEPT2, whereas pedigree relationships in CCLONES were expected to increase the extent of LD. Extended LD within LGs and LD between SNPs (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>) on different LGs (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>) was more prevalent in CCLONES compared to ADEPT2 (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). Greater reductions in the prevalence and extent of LD within LGs and between LGs after accounting for kinship in CCLONES <em>vs.</em> structure in ADEPT2 (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>) suggest that pedigree relationships increased LD more than population structure in these populations. The relatively small effects of structure on patterns of LD may be explained by high rates of gene flow and weak subpopulation structure across the geographic range of <em>P. taeda</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Al-Rabab’ah and Williams 2002</a>; <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Eckert <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chhatre <em>et al.</em> 2013</a>).</p><p id="p-46">LD between SNPs within the same gene (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) and between SNPs at different loci on the same LGs tended to be weak in both populations (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>), and LD did not decay substantially with genetic distance (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). After adjusting r<sup>2</sup> values for kinship and structure and after accounting for uncertainty marker position, cases of extended LD within LGs were rare. There was no evidence for epistatic LD between SNPs on different LGs in either population. Low levels of LD in CCLONES may be explained by the fact that 80% of the pairs of the individuals within the population were unrelated (<a href="http://www.g3journal.org/content/suppl/2015/06/11/g3.115.019588.DC1/TableS2.pdf">Table S2</a>).</p><p id="p-47">Low average r<sup>2</sup> between SNPs on the same LG and the lack of decay of r<sup>2</sup> with genetic distance imply that association genetic studies in these populations are underpowered to comprehensively detect causal variants at current marker densities. This result is not surprising considering that LD decays within hundreds to thousands of bases in outcrossing <em>P. taeda</em> populations (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2004</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Neale and Savolainen 2004</a>). Despite the low levels of LD observed in CCLONES, markers genotyped at low densities may be predictive for genomic selection in complex pedigrees of <em>P. taeda</em>. The 4854 SNP loci currently genotyped in CCLONES had substantial predictive abilities for traits related to growth, development, wood quality, disease resistance, and insect resistance (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Resende <em>et al.</em> 2012b</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Westbrook <em>et al.</em> 2013</a>; <a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Westbrook <em>et al.</em> 2015</a>). The predictive abilities of these markers coupled with the low levels of LD in CCLONES suggest that a large proportion of the predictive ability of low-density marker panels is derived from tracing pedigree relationships rather than being tightly linked to causal polymorphisms (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wientjes <em>et al.</em> 2013</a>).</p></div></div><div class="section conclusions" id="sec-19"><h2 class="">Conclusions</h2><p id="p-48">The consensus genetic map for <em>P. taeda</em> and <em>P. elliottii</em> presented here is the most densely populated linkage map for a conifer to date (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ritland <em>et al.</em> 2011</a>). The consensus map coupled with the genome-wide analysis of linkage disequilibrium in two discovery populations of <em>Pinus taeda</em> establishes a foundation for comparative association mapping between populations and the implementation of genomic selection in loblolly pine and slash pine.</p></div><div class="section" id="sec-20"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">J.W.W. was supported by a USDA CSREES Food and Agricultural Sciences National Needs Graduate Fellowship. V.E.C. was supported by USDA NIFA Award #2011-68002-30185 (PINEMAP) and the USDA Forest Service. L.S.W. was supported by the National Science Foundation under grant no. ABI-1062432 to Indiana University. P.M.G., D.N., and K.M. were supported in part by USDA NIFA Award #2011-67009-30030 (PineRefSeq) to University of California, Davis.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-5"><p id="p-50">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019588/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019588/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-51"><em>Communicating editor: J. D. Faris</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-12-10"><span class="received-label">Received </span>December 10, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-06-05"><span class="accepted-label">Accepted </span>June 5, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Westbrook <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.1" data-doi="10.1016/S0378-1127(01)00584-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Al-Rabab’ah</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Population dynamics of <em>Pinus taeda</em> L. based on nuclear microsatellites.</span> <abbr class="cit-jnl-abbrev">For. Ecol. Manage.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BEcol.%2BManage.%26rft.volume%253D163%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1127%252801%252900584-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1127(01)00584-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.2" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic Local Alignment Search Tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.3" data-doi="10.1139/x06-203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baltunis</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldfarb</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelzer</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic analysis of early field growth of loblolly pine clones and seedlings from the same full-sib families.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D37%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx06-203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x06-203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.4" data-doi="10.1186/1745-6150-7-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boratyn</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Domain enhanced lookup time accelerated BLAST.</span> <abbr class="cit-jnl-abbrev">Biol. Direct</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiol%2BDirect%26rft.aulast%253DBoratyn%26rft.auinit1%253DG.%2BM.%26rft.volume%253D7%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DDomain%2Benhanced%2Blookup%2Btime%2Baccelerated%2BBLAST.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-6150-7-12%26rft_id%253Dinfo%253Apmid%252F22510480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-6150-7-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22510480&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.5" data-doi="10.1073/pnas.0404231101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuntz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in loblolly pine.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15255</span>–<span class="cit-lpage">15260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404231101%26rft_id%253Dinfo%253Apmid%252F15477602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQyLzE1MjU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadel</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassoni</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiehne</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Temesgen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Anchored reference loci in loblolly pine (<em>Pinus taeda</em> L.) for integrating pine genomics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrown%26rft.auinit1%253DG.%2BR.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D799%26rft.epage%253D809%26rft.atitle%253DAnchored%2BReference%2BLoci%2Bin%2BLoblolly%2BPine%2B%2528Pinus%2Btaeda%2BL.%2529%2Bfor%2BIntegrating%2BPine%2BGenomics%26rft_id%253Dinfo%253Apmid%252F11606554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNzk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.7" data-doi="10.1007/s11295-013-0624-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chhatre</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byram</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic structure and association mapping of adaptive and selective traits in the east Texas loblolly pine (<em>Pinus taeda</em> L.) breeding populations.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1161</span>–<span class="cit-lpage">1178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D1161%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0624-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0624-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.8" data-doi="10.1038/hdy.2010.168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cumbie</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whetten</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association genetics of carbon isotope discrimination, height and foliar nitrogen in a natural population of <em>Pinus taeda</em> L.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DCumbie%26rft.auinit1%253DW.%2BP.%26rft.volume%253D107%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D114%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bcarbon%2Bisotope%2Bdiscrimination%252C%2Bheight%2Band%2Bfoliar%2Bnitrogen%2Bin%2Ba%2Bnatural%2Bpopulation%2Bof%2BPinus%2Btaeda%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.168%26rft_id%253Dinfo%253Apmid%252F21245892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21245892&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292911500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.9" data-doi="10.1186/1471-2156-12-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Echt</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimalanathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erpelding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An annotated genetic map of loblolly pine based on microsatellite and DNA markers.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DEcht%26rft.auinit1%253DC.%2BS.%26rft.volume%253D12%26rft.spage%253D17%26rft.epage%253D17%26rft.atitle%253DAn%2Bannotated%2Bgenetic%2Bmap%2Bof%2Bloblolly%2Bpine%2Bbased%2Bon%2Bmicrosatellite%2Band%2BcDNA%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-17%26rft_id%253Dinfo%253Apmid%252F21269494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21269494&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.10" data-doi="10.1534/genetics.110.115543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross–Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Patterns of population structure and environmental associations to aridity across the range of loblolly pine (<em>Pinus taeda</em> L., Pinaceae).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115543%26rft_id%253Dinfo%253Apmid%252F20439779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvOTY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.11" data-doi="10.1093/bioinformatics/btu091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LPmerge: An R package for merging genetic maps by linear programming.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu091%26rft_id%253Dinfo%253Apmid%252F24532720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTEvMTYyMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.12" data-doi="10.1146/annurev.arplant.54.031902.134907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Structure of linkage disequilibrium in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.54.031902.134907%26rft_id%253Dinfo%253Apmid%252F14502995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.54.031902.134907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14502995&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185094100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.13" data-doi="10.1105/tpc.150730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Lowdown of Linkage Disequilibrium.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1502</span>–<span class="cit-lpage">1506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DGaut%26rft.auinit1%253DB.%2BS.%26rft.volume%253D15%26rft.issue%253D7%26rft.spage%253D1502%26rft.epage%253D1506%26rft.atitle%253DThe%2BLowdown%2Bon%2BLinkage%2BDisequilibrium%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.150730%26rft_id%253Dinfo%253Apmid%252F12837942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE1LzcvMTUwMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.14" data-doi="10.1534/genetics.108.100289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection using low–density panels.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100289%26rft_id%253Dinfo%253Apmid%252F19299339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.15" data-doi="10.2307/2529339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">A simple method for computing the inverse of a numerator relationship matrix used in the prediction of breeding values.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D32%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BM07600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.16" data-doi="10.1534/genetics.104.028381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jermstad</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative mapping in the Pinaceae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKrutovsky%26rft.auinit1%253DK.%2BV.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D447%26rft.epage%253D461%26rft.atitle%253DComparative%2BMapping%2Bin%2Bthe%2BPinaceae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.028381%26rft_id%253Dinfo%253Apmid%252F15454556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvNDQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamesch</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarbrek</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1202</span>–<span class="cit-lpage">D1210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253DD1202%26rft_id%253Dinfo%253Apmid%252F22140109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22140109&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300179&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.8.1685.18" data-doi="10.1017/S0016672397003005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Linkage analysis in a full–sib family of an outbreeding plant species: overview and consequences for applications.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D70%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397003005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397003005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.19" data-doi="10.1038/hdy.2011.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siberchicot</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doligez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">This</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Novel measures of linkage disequilibrium that correct the bias due to population structure and relatedness.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMangin%26rft.auinit1%253DB.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D291%26rft.atitle%253DNovel%2Bmeasures%2Bof%2Blinkage%2Bdisequilibrium%2Bthat%2Bcorrect%2Bthe%2Bbias%2Bdue%2Bto%2Bpopulation%2Bstructure%2Band%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.73%26rft_id%253Dinfo%253Apmid%252F21878986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.73&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878986&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.20" data-doi="10.1093/nar/gks1243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchler-Bauer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitaz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CDD: conserved domains and protein three-dimensional structure.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D348</span>–<span class="cit-lpage">D352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1243%26rft_id%253Dinfo%253Apmid%252F23197659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9EMS9EMzQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.21" data-doi="10.1007/s11295-013-0646-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez-García</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Combination of multipoint maximum likelihood (MML) and regression mapping algorithms to construct a high–density genetic linkage map for loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1529</span>–<span class="cit-lpage">1535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D1529%26rft.atitle%253DCOMBINATION%2BOF%2BMULTIPOINT%2BMAXIMUM%2BLIKELIHOOD%2B%2528MML%2529%2BAND%2BREGRESSION%2BMAPPING%2BALGORITHMS%2BTO%2BCONSTRUCT%2BA%2BHIGH-DENSITY%2BGENETIC%2BLINKAGE%2BMAP%2BFOR%2BLOBLOLLY%2BPINE%2B%2528PINUS%2BTAEDA%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0646-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0646-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000327465100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKeand</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byram</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Deployment of genetically improved loblolly and slash pines in the South.</span> <abbr class="cit-jnl-abbrev">J. For.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFor.%26rft.volume%253D101%26rft.spage%253D32%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome–wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.24" data-doi="10.1007/s11295-010-0324-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butnor</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phenotypic analysis of first–year traits in a pseudo–backcross {(slash x loblolly) x slash} and the open–pollinated families of the pure–species progenitors.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0324-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0324-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.25" data-doi="10.1016/S0959-437X(02)00358-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Variation in recombination rate across the genome: Evidence and implications.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D657%26rft.epage%253D663%26rft.atitle%253DVariation%2Bin%2Brecombination%2Brate%2Bacross%2Bthe%2Bgenome%253A%2Bevidence%2Band%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900358-1%26rft_id%253Dinfo%253Apmid%252F12433578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00358-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433578&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.26" data-doi="10.1186/gb-2014-15-3-r59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Decoding the massive genome of loblolly pine using haploid DNA and novel assembly strategies.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR59%26rft.atitle%253DDECODING%2BTHE%2BMASSIVE%2BGENOME%2BOF%2BLOBLOLLY%2BPINE%2BUSING%2BHAPLOID%2BDNA%2BAND%2BNOVEL%2BASSEMBLY%2BSTRATEGIES.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r59%26rft_id%253Dinfo%253Apmid%252F24647006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24647006&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.27" data-doi="10.1016/j.tplants.2004.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Association genetics of complex traits in conifers.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DNeale%26rft.auinit1%253DD.%2BB.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D325%26rft.epage%253D330%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bcomplex%2Btraits%2Bin%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2004.05.006%26rft_id%253Dinfo%253Apmid%252F15231277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2004.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15231277&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223010600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1685.28"><div class="cit-metadata unstructured">Neves, L. G., J. M. Davis, W. B. Barbazuk, and M. Kirst, 2014 A high-density gene map of loblolly pine (<em>Pinus taeda</em> L.) based on exome sequence capture genotyping. G3 (Bethesda) 4: 29–37.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.29" data-doi="10.1038/hdy.2011.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Namroud</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosquet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012a</span> <span class="cit-article-title">The heterogeneous levels of linkage disequilibrium in white spruce genes and comparative analysis with other conifers.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DPavy%26rft.auinit1%253DN.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D273%26rft.epage%253D284%26rft.atitle%253DThe%2Bheterogeneous%2Blevels%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bwhite%2Bspruce%2Bgenes%2Band%2Bcomparative%2Banalysis%2Bwith%2Bother%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.72%26rft_id%253Dinfo%253Apmid%252F21897435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.72&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21897435&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300596100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.30" data-doi="10.1186/1741-7007-10-84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavy</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelgas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012b</span> <span class="cit-article-title">A spruce gene map infers ancient plant genome reshuffling and subsequent slow evolution in the gymnosperm lineage leading to extant conifers.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DPavy%26rft.auinit1%253DN.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D84%26rft.epage%253D84%26rft.atitle%253DA%2Bspruce%2Bgene%2Bmap%2Binfers%2Bancient%2Bplant%2Bgenome%2Breshuffling%2Band%2Bsubsequent%2Bslow%2Bevolution%2Bin%2Bthe%2Bgymnosperm%2Blineage%2Bleading%2Bto%2Bextant%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-10-84%26rft_id%253Dinfo%253Apmid%252F23102090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-10-84&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23102090&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.31" data-doi="10.1534/genetics.110.121665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Conditions under which genome–wide association studies will be positively misleading.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1045</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121665%26rft_id%253Dinfo%253Apmid%252F20813880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni8zLzEwNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.32" data-doi="10.1186/1471-2164-15-171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chancerel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamy</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandrou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome–wide distribution of genetic diversity and linkage disequilibrium in a mass–selected population of maritime pine.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-171%26rft_id%253Dinfo%253Apmid%252F24581176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24581176&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1685.33"><div class="cit-metadata unstructured">Prestemon, J. P., and R. C. Abt, 2002 TIMBR-1: Timber products supply and demand, pp. 299–325 in <em>Southern Forest Resource Assessment</em>. Southern Research Station General Technical Report SRS–53, Asheville, NC.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.34" data-doi="10.1086/321275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Linkage disequilibrium in humans: Models and data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D14%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2Bhumans%253A%2Bmodels%2Band%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321275%26rft_id%253Dinfo%253Apmid%252F11410837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11410837&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.35" data-doi="10.1534/genetics.110.117549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cumbie</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Association mapping of quantitative disease resistance in a natural population of loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.117549%26rft_id%253Dinfo%253Apmid%252F20628037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNjc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1685.36"><div class="cit-metadata unstructured">R Core Team, 2012 R: Language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.37" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">fastSTRUCTURE: variational inference of population structure in large SNP datasets.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.38" data-doi="10.1111/j.1469-8137.2011.03895.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012a</span> <span class="cit-article-title">Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BF.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DAccelerating%2Bthe%2Bdomestication%2Bof%2Btrees%2Busing%2Bgenomic%2Bselection%253A%2Baccuracy%2Bof%2Bprediction%2Bmodels%2Bacross%2Bages%2Band%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03895.x%26rft_id%253Dinfo%253Apmid%252F21973055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03895.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21973055&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298984900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.39" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012b</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard data set of loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1685.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsumura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelgas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic mapping in conifers</span>, pp. <span class="cit-fpage">196</span>–<span class="cit-lpage">238</span> in <span class="cit-source"><em>Genetics</em>, <em>Genomics and Breeding of Conifers</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kole</span>  <span class="cit-name-given-names">C.</span></span>. <span class="cit-publ-name">Science Publisher, Enfield, Massachusetts</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1685.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-source">Density Estimation for Statistics and Data Analysis</span>, <span class="cit-publ-name">Chapman and Hall</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1685.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miles</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">Forest resources of the United States: a technical document supporting the Forest Service 2010 RPA assessment. US Department of Agriculture, Forest Service</span>, <span class="cit-publ-name">Washington Office</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1685.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">JoinMap 4.1, Software for the Calculation of Genetic Linkage Maps</span>, <span class="cit-publ-name">Plant Research International</span>, <span class="cit-publ-loc">Wageningen, the Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.44" data-doi="10.1534/genetics.113.159996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loopstra</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unique features of the loblolly pine (<em>Pinus taeda</em> L.) megagenome revealed through sequence annotation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWegrzyn%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D891%26rft.epage%253D909%26rft.atitle%253DUnique%2BFeatures%2Bof%2Bthe%2BLoblolly%2BPine%2B%2528Pinus%2Btaeda%2BL.%2529%2BMegagenome%2BRevealed%2BThrough%2BSequence%2BAnnotation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159996%26rft_id%253Dinfo%253Apmid%252F24653211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.45" data-doi="10.1186/1471-2164-7-206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenzl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high–density consensus map of barley linking DArT markers to SSR, RFLP, and STS loci and agricultural traits.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DWenzl%26rft.auinit1%253DP.%26rft.volume%253D7%26rft.spage%253D206%26rft.epage%253D206%26rft.atitle%253DA%2Bhigh-density%2Bconsensus%2Bmap%2Bof%2Bbarley%2Blinking%2BDArT%2Bmarkers%2Bto%2BSSR%252C%2BRFLP%2Band%2BSTS%2Bloci%2Band%2Bagricultural%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-206%26rft_id%253Dinfo%253Apmid%252F16904008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16904008&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.46" data-doi="10.1111/nph.12240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westbrook</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Association genetics of oleoresin flow in loblolly pine: discovering genes and predicting phenotype for improved resistance to bark beetles and bioenergy potential.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D199%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12240%26rft_id%253Dinfo%253Apmid%252F23534834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23534834&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.47" data-doi="10.1111/nph.13074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westbrook</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Discovering candidate genes that regulate resin canal number in <em>Pinus taeda</em> stems by integrating association genetics and QTL analysis across environments, ages, and populations.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D205%26rft.spage%253D627%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13074%26rft_id%253Dinfo%253Apmid%252F25266813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25266813&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000346584600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-5.8.1685.48"><div class="cit-metadata unstructured">Wickham, H., 2009 ggplot2: elegant graphics for data analysis. Springer, New York. Available at: <a href="http://had.co.nz/ggplot2/book">http://had.co.nz/ggplot2/book</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.49" data-doi="10.1534/genetics.112.146290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146290%26rft_id%253Dinfo%253Apmid%252F23267052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNjIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.50" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accurate construction of consensus genetic maps via integer linear programming. IEEE/ACM Tran.</span> <abbr class="cit-jnl-abbrev">Computational Biol. Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">394</span> <span class="cit-comment">(TCBB)</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2BBiol.%2BBioinformatics%26rft.volume%253D8%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.52" data-doi="10.1371/journal.pone.0008451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warburton</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic characterization and linkage disequilibrium estimation of a global maize collection using SNP markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e8451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DYan%26rft.auinit1%253DJ.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8451%26rft.epage%253De8451%26rft.atitle%253DGenetic%2Bcharacterization%2Band%2Blinkage%2Bdisequilibrium%2Bestimation%2Bof%2Ba%2Bglobal%2Bmaize%2Bcollection%2Busing%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008451%26rft_id%253Dinfo%253Apmid%252F20041112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20041112&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1685.53" data-doi="10.1534/genetics.113.159715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crepeau</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtz-Morris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koriabine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing and assembly of the 22-Gb loblolly pine genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZimin%26rft.auinit1%253DA.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D875%26rft.epage%253D890%26rft.atitle%253DSequencing%2Band%2BAssembly%2Bof%2Bthe%2B22-Gb%2BLoblolly%2BPine%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159715%26rft_id%253Dinfo%253Apmid%252F24653210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/8/1685.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1675" title="PUF-8 Functions Redundantly with GLD-1 to Promote the Meiotic Progression of Spermatocytes in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1695" title="Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/8"><span>Volume 5 Issue 8, August 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/8.cover-source.jpg?itok=wjPUfMS9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1685.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZJGKRrxl0dO-MsNrbNtGbBC5Yj-6ADc5wLwWIVfEplk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/8/1685" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/8/1685" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/8/1685" class="active" data-icon-position="" data-hide-link-title="0">A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459993" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wgeK379KhMP7yjyv5sQzSZo3TS68Cj-uPYRc3wkG4lU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1685.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/8/1685" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hIqz9qfwwKAuIfGlSd9ZmejHxqVZ9hn-gvEjTBSvv2E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/8/1685" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1685.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459993" id="node459993--2" data-pisa="ggg;5/8/1685" data-pisa-master="ggg;g3.115.019588" data-apath="/ggg/5/8/1685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Consensus Genetic Map for <em>Pinus taeda</em> and <em>Pinus elliottii</em> and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of <em>Pinus taeda</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jared W. Westbrook</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vikram E. Chhatre</span>, <span class="highwire-citation-author" data-delta="2">Le-Shin Wu</span>, <span class="highwire-citation-author" data-delta="3">Srikar Chamala</span>, <span class="highwire-citation-author" data-delta="4">Leandro Gomide Neves</span>, <span class="highwire-citation-author" data-delta="5">Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="6">Pedro J. Martínez-García</span>, <span class="highwire-citation-author" data-delta="7">David B. Neale</span>, <span class="highwire-citation-author" data-delta="8">Matias Kirst</span>, <span class="highwire-citation-author" data-delta="9">Keithanne Mockaitis</span>, <span class="highwire-citation-author" data-delta="10">C. Dana Nelson</span>, <span class="highwire-citation-author" data-delta="11">Gary F. Peter</span>, <span class="highwire-citation-author" data-delta="12">John M. Davis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0001-9330-4051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Craig S. Echt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1685-1694;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019588 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459993--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jared W. Westbrook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jared%2BW.%2BWestbrook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Westbrook%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJared%2BW.%2BWestbrook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vikram E. Chhatre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vikram%2BE.%2BChhatre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chhatre%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVikram%2BE.%2BChhatre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vikram E. Chhatre</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Le-Shin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Genome Analysis Support, Indiana University, Bloomington, Indiana 47408</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Le-Shin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALe-Shin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Srikar Chamala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida 32603</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Genetics Institute, University of Florida, Gainesville, Florida 32608</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikar%2BChamala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chamala%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikar%2BChamala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Leandro Gomide Neves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>RAPiD Genomics LLC, Gainesville, Florida 32601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leandro%2BGomide%2BNeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neves%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeandro%2BGomide%2BNeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Agronomy, University of Florida, Gainesville, Florida 32608</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pedro J. Martínez-García </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Plant Sciences, University of California, Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BJ.%2BMart%C3%ADnez-Garc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADnez-Garc%C3%ADa%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BJ.%2BMart%25C3%25ADnez-Garc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David B. Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Plant Sciences, University of California, Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Keithanne Mockaitis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††</span>Pervasive Technology Institute, Indiana University, Bloomington, Indiana 47408</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keithanne%2BMockaitis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mockaitis%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeithanne%2BMockaitis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">C. Dana Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BDana%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BDana%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Gary F. Peter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BF.%2BPeter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peter%20GF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BF.%2BPeter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">John M. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Craig S. Echt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BS.%2BEcht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Echt%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BS.%2BEcht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9330-4051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Craig S. Echt</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cecht@fs.fed.us" class="" data-icon-position="" data-hide-link-title="0">cecht@fs.fed.us</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/8/1685" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/8/1685&amp;title=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/8/1685&amp;title=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/8/1685&amp;title=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/8/1685&amp;text=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/8/1685&amp;title=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/8/1685&amp;t=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/8/1685&amp;title=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/8/1685&amp;title=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459993/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459993" id="node459993--3" data-pisa="ggg;5/8/1685" data-pisa-master="ggg;g3.115.019588" data-apath="/ggg/5/8/1685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Consensus Genetic Map for <em>Pinus taeda</em> and <em>Pinus elliottii</em> and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of <em>Pinus taeda</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jared W. Westbrook</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vikram E. Chhatre</span>, <span class="highwire-citation-author" data-delta="2">Le-Shin Wu</span>, <span class="highwire-citation-author" data-delta="3">Srikar Chamala</span>, <span class="highwire-citation-author" data-delta="4">Leandro Gomide Neves</span>, <span class="highwire-citation-author" data-delta="5">Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="6">Pedro J. Martínez-García</span>, <span class="highwire-citation-author" data-delta="7">David B. Neale</span>, <span class="highwire-citation-author" data-delta="8">Matias Kirst</span>, <span class="highwire-citation-author" data-delta="9">Keithanne Mockaitis</span>, <span class="highwire-citation-author" data-delta="10">C. Dana Nelson</span>, <span class="highwire-citation-author" data-delta="11">Gary F. Peter</span>, <span class="highwire-citation-author" data-delta="12">John M. Davis</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0001-9330-4051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Craig S. Echt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1685-1694;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019588 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459993--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jared W. Westbrook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jared%2BW.%2BWestbrook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Westbrook%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJared%2BW.%2BWestbrook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vikram E. Chhatre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vikram%2BE.%2BChhatre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chhatre%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVikram%2BE.%2BChhatre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vikram E. Chhatre</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Le-Shin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Genome Analysis Support, Indiana University, Bloomington, Indiana 47408</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Le-Shin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALe-Shin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Srikar Chamala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of Florida, Gainesville, Florida 32611</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Pathology, Immunology and Laboratory Medicine, University of Florida, Gainesville, Florida 32603</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Genetics Institute, University of Florida, Gainesville, Florida 32608</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikar%2BChamala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chamala%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikar%2BChamala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Leandro Gomide Neves </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>RAPiD Genomics LLC, Gainesville, Florida 32601</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leandro%2BGomide%2BNeves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neves%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeandro%2BGomide%2BNeves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Agronomy, University of Florida, Gainesville, Florida 32608</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pedro J. Martínez-García </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Plant Sciences, University of California, Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BJ.%2BMart%C3%ADnez-Garc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADnez-Garc%C3%ADa%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BJ.%2BMart%25C3%25ADnez-Garc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David B. Neale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>Department of Plant Sciences, University of California, Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neale%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Keithanne Mockaitis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†††</span>Pervasive Technology Institute, Indiana University, Bloomington, Indiana 47408</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keithanne%2BMockaitis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mockaitis%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeithanne%2BMockaitis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">C. Dana Nelson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BDana%2BNelson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nelson%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BDana%2BNelson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Gary F. Peter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BF.%2BPeter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peter%20GF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BF.%2BPeter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">John M. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Forest Resources and Conservation, University of Florida, Gainesville, Florida, 32611-0410</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Craig S. Echt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Southern Research Station, USDA Forest Service, Saucier, Mississippi, 39574</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BS.%2BEcht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Echt%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BS.%2BEcht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9330-4051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Craig S. Echt</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cecht@fs.fed.us" class="" data-icon-position="" data-hide-link-title="0">cecht@fs.fed.us</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1685&amp;rft.epage=1694&amp;rft.atitle=A%20Consensus%20Genetic%20Map%20for%20Pinus%20taeda%20and%20Pinus%20elliottii%20and%20Extent%20of%20Linkage%20Disequilibrium%20in%20Two%20Genotype-Phenotype%20Discovery%20Populations%20of%20Pinus%20taeda&amp;rft.volume=5&amp;rft.issue=8&amp;rft.date=2015-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Westbrook%2C+Jared+W.&amp;rft.au=Chhatre%2C+Vikram+E.&amp;rft.au=Wu%2C+Le-Shin&amp;rft.au=Chamala%2C+Srikar&amp;rft.au=Neves%2C+Leandro+Gomide&amp;rft.au=Mu%C3%B1oz%2C+Patricio&amp;rft.au=Mart%C3%ADnez-Garc%C3%ADa%2C+Pedro+J.&amp;rft.au=Neale%2C+David+B.&amp;rft.au=Kirst%2C+Matias&amp;rft.au=Mockaitis%2C+Keithanne&amp;rft.au=Nelson%2C+C.+Dana&amp;rft.au=Peter%2C+Gary+F.&amp;rft.au=Davis%2C+John+M.&amp;rft.au=Echt%2C+Craig+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459993/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459993/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459993/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459993/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459993/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459993/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459993/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459993/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459993/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459993/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459993/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459993/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/8/1685" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/8/1685.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/8/1685.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/8/1685.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/8/1685.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
