<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/8/1603" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/8/1603.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/8/1603.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/8/1603.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/8/1603/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/8/1603.atom" />
<meta name="HW.pisa" content="ggg;5/8/1603" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction" />
<meta name="DC.Identifier" content="10.1534/g3.115.019299" />
<meta name="DC.Date" content="2015-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Technow and Totir. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Training set size is an important determinant of genomic prediction accuracy. Plant breeding programs are characterized by a high degree of structuring, particularly into populations. This hampers the establishment of large training sets for each population. Pooling populations increases training set size but ignores unique genetic characteristics of each. A possible solution is partial pooling with multilevel models, which allows estimating population-specific marker effects while still leveraging information across populations. We developed a Bayesian multilevel whole-genome regression model and compared its performance with that of the popular BayesA model applied to each population separately (no pooling) and to the joined data set (complete pooling). As an example, we analyzed a wide array of traits from the nested association mapping maize population. There we show that for small population sizes ( e.g. , &lt;50), partial pooling increased prediction accuracy over no or complete pooling for populations represented in the training set. No pooling was superior; however, when populations were large. In another example data set of interconnected biparental maize populations either partial or complete pooling was superior, depending on the trait. A simulation showed that no pooling is superior when differences in genetic effects among populations are large and partial pooling when they are intermediate. With small differences, partial and complete pooling achieved equally high accuracy. For prediction of new populations, partial and complete pooling had very similar accuracy in all cases. We conclude that partial pooling with multilevel models can maximize the potential of pooling by making optimal use of information in pooled training sets." />
<meta name="DC.Contributor" content="Frank Technow" />
<meta name="DC.Contributor" content="L. Radu Totir" />
<meta name="article:published_time" content="2015-08-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Training set size is an important determinant of genomic prediction accuracy. Plant breeding programs are characterized by a high degree of structuring, particularly into populations. This hampers the establishment of large training sets for each population. Pooling populations increases training set size but ignores unique genetic characteristics of each. A possible solution is partial pooling with multilevel models, which allows estimating population-specific marker effects while still leveraging information across populations. We developed a Bayesian multilevel whole-genome regression model and compared its performance with that of the popular BayesA model applied to each population separately (no pooling) and to the joined data set (complete pooling). As an example, we analyzed a wide array of traits from the nested association mapping maize population. There we show that for small population sizes (&lt;i&gt;e.g.&lt;/i&gt;, &amp;lt;50), partial pooling increased prediction accuracy over no or complete pooling for populations represented in the training set. No pooling was superior; however, when populations were large. In another example data set of interconnected biparental maize populations either partial or complete pooling was superior, depending on the trait. A simulation showed that no pooling is superior when differences in genetic effects among populations are large and partial pooling when they are intermediate. With small differences, partial and complete pooling achieved equally high accuracy. For prediction of new populations, partial and complete pooling had very similar accuracy in all cases. We conclude that partial pooling with multilevel models can maximize the potential of pooling by making optimal use of information in pooled training sets.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/08/01" />
<meta name="citation_mjid" content="ggg;5/8/1603" />
<meta name="citation_id" content="5/8/1603" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/8/1603" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/8/1603.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/8/1603.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/8/1603.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019299" />
<meta name="citation_pmid" content="26024866" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1603" />
<meta name="citation_lastpage" content="1612" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Frank Technow" />
<meta name="citation_author_institution" content="DuPont Pioneer, Johnston, Iowa 50131" />
<meta name="citation_author_email" content="Frank.Technow@pioneer.com" />
<meta name="citation_author" content="L. Radu Totir" />
<meta name="citation_author_institution" content="DuPont Pioneer, Johnston, Iowa 50131" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=V. Wimmer;citation_author=J. Ogutu;citation_author=C. Knaak;citation_title=Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years.;citation_pages=1375-1386;citation_volume=127;citation_year=2014;citation_pmid=24723140;citation_doi=10.1007/s00122-014-2305-z" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen.;citation_author=F. G. Asoro;citation_author=M. Newell;citation_author=M. Beavis;citation_author=M. Scott;citation_author=J.-L. Jannink;citation_title=Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.;citation_pages=132-144;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.02.0007" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Blanc;citation_author=A. Charcosset;citation_author=B. Mangin;citation_author=A. Gallais;citation_author=L. Moreau;citation_title=Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize.;citation_pages=206-224;citation_volume=113;citation_year=2006;citation_issue=2;citation_pmid=16791688;citation_doi=10.1007/s00122-006-0287-1" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. F. Brøndum;citation_author=G. Su;citation_author=M. S. Lund;citation_author=P. J. Bowman;citation_author=M. E. Goddard;citation_title=Genome position specific priors for genomic prediction.;citation_pages=543-543;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23050763;citation_doi=10.1186/1471-2164-13-543" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. Combs;citation_author=R. Bernardo;citation_title=Accuracy of genomewide selection for different traits with constant population size, heritability, and number of markers.;citation_pages=1-7;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_title=On the genomic analysis of data from structured populations.;citation_pages=163-164;citation_volume=131;citation_year=2014;citation_pmid=24838115;citation_doi=10.1111/jbg.12091" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. A. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Erbe;citation_author=B. J. Hayes;citation_author=L. K. Matukumalli;citation_author=S. Goswami;citation_author=P. J. Bowman;citation_title=Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.;citation_pages=4114-4129;citation_volume=95;citation_year=2012;citation_issue=7;citation_pmid=22720968;citation_doi=10.3168/jds.2011-5019" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_title=Chapter response to selection;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_pages=184-193;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=A. E. Gelfand;citation_author=S. K. Sahu;citation_title=Identifiability, improper priors and gibbs sampling for generalized linear models.;citation_pages=247-253;citation_volume=94;citation_year=1999;citation_doi=10.2307/2669699" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. Gelman;citation_title=Multilevel (hierarchical) modeling: what it can and cannot do.;citation_pages=432-435;citation_volume=48;citation_year=2006;citation_doi=10.1198/004017005000000661" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=A. Gelman;citation_title=Prior distributions for variance parameters in hierarchical models.;citation_pages=515-533;citation_volume=1;citation_year=2006;citation_doi=10.1214/06-BA117A" />
<meta name="citation_reference" content="citation_title=Data Analysis Using Regression and Multilevel/Hierarchical Models;citation_author=A. Gelman;citation_author=J. Hill;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. Gelman;citation_author=I. Pardoe;citation_title=Bayesian measures of explained variance and pooling in multilevel (hierarchical) models.;citation_pages=241-251;citation_volume=48;citation_year=2006;citation_doi=10.1198/004017005000000517" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Giraud;citation_author=C. Lehermeier;citation_author=E. Bauer;citation_author=M. Falque;citation_author=V. Segura;citation_title=Linkage disequilibrium with linkage analysis of multiline crosses reveals different multiallelic QTL for hybrid performance in the flint and dent heterotic groups of maize.;citation_volume=198;citation_year=2014;citation_pmid=25271305;citation_doi=10.1534/genetics.114.169367" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Guo;citation_author=D. Tucker;citation_author=J. Lu;citation_author=V. Kishore;citation_author=G. Gay;citation_title=Evaluation of genome-wide selection efficiency in maize nested association mapping populations.;citation_pages=261-275;citation_volume=124;citation_year=2012;citation_issue=2;citation_pmid=21938474;citation_doi=10.1007/s00122-011-1702-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic-BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Plant Gen;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=S. Dreisigacker;citation_author=J. Crossa;citation_author=S. Hearne;citation_author=R. Babu;citation_title=Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.;citation_pages=1476-1488;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0195" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Karoui;citation_author=M. J. Carabaño;citation_author=C. Díaz;citation_author=A. Legarra;citation_title=Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.;citation_pages=39;citation_volume=44;citation_year=2012;citation_pmid=23216664;citation_doi=10.1186/1297-9686-44-39" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krmer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=A. J. Lorenz;citation_title=Resource Allocation for Maximizing Prediction Accuracy and Genetic Gain of Genomic Selection in Plant Breeding: A Simulation Experiment;citation_pages=481-491;citation_volume=3;citation_year=2013;citation_issue=3;citation_pmid=23450123;citation_doi=10.1534/g3.112.004911" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. S. Lund;citation_author=G. Su;citation_author=L. Janss;citation_author=B. Guldbrandtsen;citation_author=R. F. Brøndum;citation_title=Invited review: genomic evaluation of cattle in a multi-breed context.;citation_pages=101-110;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.05.008" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=M. Martin;citation_author=B. S. Dhillon;citation_author=T. Miedaner;citation_author=A. E. Melchinger;citation_title=Inheritance of resistance to Gibberella ear rot and deoxynivalenol contamination in five flint maize crosses.;citation_pages=28-32;citation_volume=131;citation_year=2012;citation_doi=10.1111/j.1439-0523.2011.01908.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. S. Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_title=Concepts and Breeding of Heterosis in Crop Plants;citation_title=Overview of heterosis and heterotic groups in agronomic crops;citation_author=A. E. Melchinger;citation_author=R. K. Gumber;citation_pages=29-44;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. E. Melchinger;citation_author=H. F. Utz;citation_author=C. C. Schön;citation_title=Genetic Expectations of Quantitative Trait Loci Main and Interaction Effects Obtained With the Triple Testcross Design and Their Relevance for the Analysis of Heterosis;citation_pages=2265-2274;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430948;citation_doi=10.1534/genetics.107.084871" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. A. Mikel;citation_author=J. W. Dudley;citation_title=Evolution of North American Dent Corn from Public to Proprietary Germplasm;citation_pages=1193-1205;citation_volume=46;citation_year=2006;citation_issue=3;citation_doi=10.2135/cropsci2005.10-0371" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=K. Olson;citation_author=P. VanRaden;citation_author=M. Tooker;citation_title=Multibreed genomic evaluations using purebred Holsteins, Jerseys, and Brown Swiss.;citation_pages=5378-5383;citation_volume=95;citation_year=2012;citation_issue=9;citation_pmid=22916944;citation_doi=10.3168/jds.2011-5006" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=M. Plummer;citation_author=N. Best;citation_author=K. Cowles;citation_author=K. Vines;citation_title=CODA: convergence diagnosis and output analysis for MCMC.;citation_pages=7-11;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Riedelsheimer;citation_author=J. B. Endelman;citation_author=M. Stange;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic predictability of interconnected biparental maize populations.;citation_volume=194;citation_year=2013;citation_pmid=23535384;citation_doi=10.1534/genetics.113.150227" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Riedelsheimer;citation_author=A. E. Melchinger;citation_title=Optimizing the allocation of resources for genomic selection in one breeding cycle.;citation_pages=2835-2848;citation_volume=126;citation_year=2013;citation_pmid=23982591;citation_doi=10.1007/s00122-013-2175-9" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=T. Schulz-Streeck;citation_author=J. Ogutu;citation_author=Z. Karaman;citation_author=C. Knaak;citation_author=H. Piepho;citation_title=Genomic selection using multiple populations.;citation_pages=2453-2461;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2012.03.0160" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=F. Technow;citation_author=A. Bürger;citation_author=A. E. Melchinger;citation_title=Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups;citation_pages=197-203;citation_volume=3;citation_year=2013;citation_issue=2;citation_pmid=23390596;citation_doi=10.1534/g3.112.004630" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=F. Technow;citation_author=T. Schrag;citation_author=W. Schipprack;citation_author=A. Melchinger;citation_title=Identification of key ancestors of modern germplasm in a breeding program of maize.;citation_pages=2545-2553;citation_volume=127;citation_year=2014;citation_pmid=25208647;citation_doi=10.1007/s00122-014-2396-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Technow;citation_author=T. A. Schrag;citation_author=W. Schipprack;citation_author=E. Bauer;citation_author=H. Simianer;citation_title=Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.;citation_volume=197;citation_year=2014;citation_pmid=24850820;citation_doi=10.1534/genetics.114.165860" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. Thomas;citation_author=B. O’Hara;citation_author=U. Ligges;citation_author=S. Sturtz;citation_title=Making BUGS open.;citation_pages=12-17;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_author=P. J. Brown;citation_author=H. Hung;citation_author=Q. Sun;citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population.;citation_pages=159-162;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217756;citation_doi=10.1038/ng.746" />
<meta name="citation_reference" content="citation_journal_title=Crop J.;citation_author=X. Wang;citation_author=Y. Pang;citation_author=J. Zhang;citation_author=Q. Zhang;citation_author=Y. Tao;citation_title=Genetic background effects on QTL and QTL × environment interaction for yield and its component traits as revealed by reciprocal introgression lines in rice.;citation_pages=345-357;citation_volume=2;citation_year=2014;citation_doi=10.1016/j.cj.2014.06.004" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=K. L. Weber;citation_author=R. M. Thallman;citation_author=J. W. Keele;citation_author=W. M. Snelling;citation_author=G. L. Bennett;citation_title=Accuracy of genomic breeding values in multibreed beef cattle populations derived from deregressed breeding values and phenotypes.;citation_volume=90;citation_year=2012;citation_pmid=22767091;citation_doi=10.2527/jas.2011-4586" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.;citation_pages=1427-1436;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=X. Zhang;citation_author=P. Perez-Rodriguez;citation_author=K. Semagn;citation_author=Y. Beyene;citation_author=R. Babu;citation_title=Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.;citation_pages=291-299;citation_volume=114;citation_year=2015;citation_pmid=25407079;citation_doi=10.1038/hdy.2014.99" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="twitter:description" content="Training set size is an important determinant of genomic prediction accuracy. Plant breeding programs are characterized by a high degree of structuring, particularly into populations. This hampers the establishment of large training sets for each population. Pooling populations increases training set size but ignores unique genetic characteristics of each. A possible solution is partial pooling with multilevel models, which allows estimating population-specific marker effects while still leveraging information across populations. We developed a Bayesian multilevel whole-genome regression model and compared its performance with that of the popular BayesA model applied to each population separately (no pooling) and to the joined data set (complete pooling). As an example, we analyzed a wide array of traits from the nested association mapping maize population. There we show that for small population sizes ( e.g. , &lt;50), partial pooling increased prediction accuracy over no or complete pooling for populations represented in the training set. No pooling was superior; however, when populations were large. In another example data set of interconnected biparental maize populations either partial or complete pooling was superior, depending on the trait. A simulation showed that no pooling is superior when differences in genetic effects among populations are large and partial pooling when they are intermediate. With small differences, partial and complete pooling achieved equally high accuracy. For prediction of new populations, partial and complete pooling had very similar accuracy in all cases. We conclude that partial pooling with multilevel models can maximize the potential of pooling by making optimal use of information in pooled training sets." />
<meta name="og-title" property="og:title" content="Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/8/1603" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Training set size is an important determinant of genomic prediction accuracy. Plant breeding programs are characterized by a high degree of structuring, particularly into populations. This hampers the establishment of large training sets for each population. Pooling populations increases training set size but ignores unique genetic characteristics of each. A possible solution is partial pooling with multilevel models, which allows estimating population-specific marker effects while still leveraging information across populations. We developed a Bayesian multilevel whole-genome regression model and compared its performance with that of the popular BayesA model applied to each population separately (no pooling) and to the joined data set (complete pooling). As an example, we analyzed a wide array of traits from the nested association mapping maize population. There we show that for small population sizes ( e.g. , &lt;50), partial pooling increased prediction accuracy over no or complete pooling for populations represented in the training set. No pooling was superior; however, when populations were large. In another example data set of interconnected biparental maize populations either partial or complete pooling was superior, depending on the trait. A simulation showed that no pooling is superior when differences in genetic effects among populations are large and partial pooling when they are intermediate. With small differences, partial and complete pooling achieved equally high accuracy. For prediction of new populations, partial and complete pooling had very similar accuracy in all cases. We conclude that partial pooling with multilevel models can maximize the potential of pooling by making optimal use of information in pooled training sets." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
    <title>Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2dSI882eOP2KQoaux7IrIvoLw2bTbAxRcFyRkzgjVWk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460011","apath":"\/ggg\/5\/8\/1603.atom","pisa":"ggg;5\/8\/1603","ac":{"\/ggg\/5\/8\/1603.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/8\/1603.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1603"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1603"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/8\/1603":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222lA118gB1Hj4LjF\\\\\\\/oZjSRMqO7J30WxPEIT2kYlVpYF2wpK3Gv6DhOV2wdK2ZLoW3XIiiUHq66EUhGGs2Cje2Id5d79D\\\\\\\/9OJ8FwK5VuRd2wEoWUKyuFV0rT5qPBwj2gkGlHXhiX\\\\\\\/yTudoetJkO7n6AaQz510BySJN+Qf8uhnNQ+mxbjYx462gGCYgHN9rN9Gxh3EFpWMjfF3hA0RClI8ISLRm0OOAMzn5m5XypJpbAvNwTsnENtAohav\\\\\\\/AWizt3MXgCvguzU37plK3VIxRQ9nE5Y6EieU9rM1Kcixp+UJpsXm\\\\\\\/EZfCycRMKVXnhunJA7IClGivAlpjQEZfWswu7lHOLJN2QYvBfWpzKc20H5ixygYqMY6zX8bwTxpdGYKI7S+ERD98Gb0s+HX4LsxRjqzCqwmDEJZ0Wma9D1OeNRvo7C+2p2xkeZdQ26VdfCjFz8DgmwBRO\\\\\\\/401pf82tOW9Cj0HrD\\\\\\\/1xYfKBFX2\\\\\\\/xQOEmHtSl\\\\\\\/hog1eDWm0NDCdWsenkw0u1q2\\\\\\\/7F\\\\\\\/2t6urDV0dOBN4k3sZHl3JETfPmP0mW8DiGkEmf55e5\\\\\\\/jh8KUKYY+tabWLBHtK1DdN5\\\\\\\/LIDuFMFfmNPCeoGM3COFpCpAEeGt+FoJvuG5bJlwBw351RP5TgL9GzkMu1bkmLRoS84Fsku6xNq8IS8\\\\\\\/U3eQYwzesB0PuOGxP+Igbv00sLgAlpyLa4wlVjWLGNqKFGGfDa8lGcD6NhTeE9jq7+YGIVEtomTMwJEcxXeuqS07\\\\\\\/Ihjy9TxqEo5Ezv3qn4oOuButtOiUt5fAVUD3ZJQanR8Kh7gp+3xcRIBSd2yOf0rAJnObLbR+1hM8UrHXOxKaOl8fMlvHsQU39jWC8AQaP2mXcHutvV4lxaBMD0HyJ1dqII+4b8jPoQ+894rXWrQdyLMilCbV03Sp1t4\\\\\\\/7JJA1gYElQyW\\\\\\\/mGZUguZoqrJZhmfLaR1Rqv1P1NW+nUW8wPJSEStWX4eheNl4\\\\\\\/Xq1J+FQPVz0GiJXvRRWNpVsbqEMizRbudCjG6\\\\\\\/chcjtvwFP2vWHASeqHxrVW2hRJWe5rTqX1PFaponqyM9lc7kBgYChcwo6p\\\\\\\/DR9zViUYwRm02yaLx5yPxyRRP7iIPifQLvkhNYY0MY3gvkiebX8aMwjPZSQ0r1Xvbyt7VBvqQlHS81VlHgrpGjyaAiEtHYJMaH+xmSl9lWjbTj2EZmdgSlLaScEod9OXecgtEQP\\\\\\\/auTWF1KiZPiGgP\\\\\\\/X98jnEiemqX3beqlYNjIA8pdxhUxS1p9V3KUncKZbEQSB\\\\\\\/hev\\\u0022,\\\u0022iv\\\u0022:\\\u00224juwohmwwbfd\\\\\\\/LRZX5PjUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002275c3f0b26ec9440f5a4efa0d5ba48993d9eee50e83e9abb9224632d0173da55c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t3Y1xsx+ONYFXxKfDef8zPcOC2ce3RFv1V9jwx0cm\\\\\\\/Jsp5Ct3fMb8y7u82YaiXpNgc5vqRHBuAc7TIXcwefP2F9po1TT+8lR36ILDELN9ASghFS39rfVCltJYhAgdxHKGeWh+L1B1VDdJcUaMQrvS9GodfpbYdTJ9O+ZOqfvnN81RpNXOwxQHLNTYLsAcPS4mSEI45LEztFBDAgejJWasY8HDMGU1+ZQzkP+zDxZsjUIwPnIf\\\\\\\/8PRCqiUEeypQCaKQRot8gxjOtgDVpvgH04Gvcvweiq2ClbHZvXAIXG5jfTlLSqjnAZcRiZYv0N8VWYhZnm0I8Ox5fTJQSwtNFkzzxm4nTHsyq45o7O6Ihx4ZyuxouRHmwaZrg2epof8+LzxL4FTTQgmksjhtSBY++z7duNJHzIDCMZOe\\\\\\\/kaFoNNPa3ZweN4NCM89mPnBlR7T\\\\\\\/QGl6eP55MrPdspgB2VorLQW\\\\\\\/BgF2ZbCH2IFO4gD+1cvYehWpZNOm1QQr1f6sGO1Tfnk+7ACv1wvTyOcLbUxV0OnWBjk6lxcbKw+QEIjgebcWJCvFcmmQyRy\\\\\\\/dmWkXnl521lg926F8OnsrVyNBSqFv88\\\\\\\/YycgqZinGPb2euR9dWxRjQJSqhHEjteDTJDylBqVDl0VLanyaGhgnT59MVOgGuZG4JNpXzDBAt0yUWYtaqCE\\\\\\\/OymD7x9oRClW8lLBfDHE1eKV7j56\\\\\\\/ktNvsadUS9V0D0QFB91BdBr1l7GuVGcUxjelHzvgXL+cVt5XiNWc\\\\\\\/3vDFde51\\\\\\\/kkamDc37GGgKVpa+b5Cflm2\\\\\\\/5HRxgC0aOCJVE58OoiCnjmTHsTd1f4tp2khZrvk9fUxcUb6CB4UvVxY5zL1PRizECo5pdCWtmnaBvRBpXv+s6rKoPlMV\\\\\\\/1hRcQ5swuvsAZMSrk4aShvbBqiSgU1qUmQz8BX+SxCK7poxX63yjnVkHWD+5EHJ8WOjv7gcqG+GS6MbkgPMHC5yOBqBPyApMdqJTm2d2UM9tiuvpGc4nIilfa4kUMukMrKi1L\\\\\\\/bwdGg+ma3yN2fQoa\\\\\\\/uLvlOs7z2tHOfKdhoqBaIVOtOny9pEpQrUdlAtBM9bjUJLeeKbRFJ4Uyq4xxx1XyN8aEpAMlBQlo7T2dBs4LqK4JYxB8S5rW\\\\\\\/Jf0HabSvrE1DJ0ryZJdxU2Y7renWemayTP6LHjy4KIASpO5O9Farq3R1NliprLUqP6ts\\\\\\\/sqqVKFO2U2iSb5DvBwhMEFH1d6PXUQ8+44argsbagP4U8bRp0fp5fl6sQ3SpqytDLTBNb+eKPeTs6qbJsKxYktPVYkrWAWs3tjSTFbYZevUlBEfz7Bu7ekCaWk4wJpnGr4EKplyhJarMSZAionGDSCrWGGSs8SKjgwqPAuLgM8cvvlPHv+RYgcHMrQ\\\\\\\/xlVdtq5OH+wiNSKOfPd1r7DU8OypStz+nD2poliCKcbPKlv\\\\\\\/LP4wSWc1AosTBluAL3CobYRuUots9etg0TQEpOaWMO2VjOMXTWyspz4t6nNp1voSC742YDpdaDeBK9d56a1cj\\\\\\\/q8zdt+fMMXk1fGg+Bx1chHi13icLKNZppne755kvKn0MkFQSfIRWnRd86mytw\\\\\\\/HmsnAOwQsxaFIlwRGTSYOwVeArzFNcX6CFQVA63FbIhrhaawt3u1svRUlQ0KRQRCV5t91aibD6\\\\\\\/muUoMs+9\\\\\\\/HW3igjz4iXP9TMamd28tnsR59NJPb3Hi\\\\\\\/HZfTpKdXrojpuOKP\\\\\\\/qReqcxfk\\\\\\\/1lTYYdXMWXVkh1R2d1SmkJHn9HkwBZAHXgOh1+N87N07t19Apv1oFD\\\\\\\/78IWa8UF5D1vPDlHmGID+g3fqyNyOlKCpxF6eJC\\\\\\\/RBEnTTCs4\\\\\\\/ZP68+OrePZs1yNRqZ+Oc+GkVEhsiHHxdWG8Y124j\\\\\\\/Rq+DEbvp1Hcb2hLsQ4xGOBWUzv+U5KTA6npPE0HbsyOMPze6cKTPSMCPkdC0uEjKJPde2eIAx4KqQjvLF3fzS+Jce9WVUIU3\\\\\\\/yl3G1jLlrlgHCl9fev8F+fJSwOg9RBWNFnDE9OP6ChP5TwJoharjFbYGQWriH\\\\\\\/nTxrkDG6tQFcyA\\\\\\\/SH7m7UsZsa3uEXq4tIY1swEwfs87WcQr3N14nqWjgY6b2ur2cJbxeSmo6Y1iJycmZUa\\\\\\\/ZzTEg9JiAZf5q3UYi4XaE\\\\\\\/yfvRi+VXZrkYvEPs7HHmP1bOVEo\\\\\\\/7c6x5rJ9qrLOpCH7LVAJkReiQmbtxxHlW0tcPGVHstPqtsI2fL1QyxJsXVPUHcwVIpvhxSHckjd26HaQuYFOKzNPwzJZ7sVb12s8lV5YNAmKgUaCkPn\\\\\\\/B3IPZeaW+J9Id7vI\\\\\\\/H141pgTMFwUgBotRV3KKjXzPZaOIwEgjfK8ok9oRZPUDMVad2eBUnZT0Hb5KqG2zLcds4q+M6WahqVinwmkyQHJlvgbfV3xn1Nag7QFd8pVqP4H4xTQ1UsEubO6eePHlZ0mXXuw5xM3my1f4V\\\\\\\/Weo2NfBOx5s+AoixI3rLgEUN4XIJ6jBNDzvTTuNTtQxcdlFaZh74qVfzlW\\\\\\\/L+WxZkbCAr\\\\\\\/B27P1RTFU0gLjVYgQb\\\\\\\/v5PEW0mhldml5VDmK8VzAJtoxtnrtC\\\\\\\/V+OESPHMuTu7uPw\\\\\\\/l\\\\\\\/2lMAJt\\\\\\\/SNCm\\\\\\\/nq8I8h7clA+6LHT1lbCCNqwiHohS9nuVRcp4hlwE9itCE\\\\\\\/HdPbdj0s5cysJeSaCsdxCc9lydi9gq6Qy51mm2atSrtWdgSNqVgV\\\\\\\/qKd2b\\\\\\\/ovOZ4E2NOJB8u4HX\\\\\\\/AyX0x8pgUZgEsguSCcW547aSM6vohpAMZ8DxyBDWT7W1lXLaXCKpkA5SC599aToxYsXcFvrvrpBSxPD4dfVC4sBnicSKHlZ5\\\\\\\/nZPqlVdLiDhWFNYAYVHRZdnTPYQsOnebKI\\\\\\\/hWxzgfbBR+u4n3EUrgIuhTi6QxCZmVqGOMg3YqV1VbFb2ui+L8ZgW3FTHVNmGvppndJ1GFIQ7wVwQf6jbWQuJUkpPeFCzxW75UEFzoopi96ZNw5MAxuO3drvuLf1P0wLFHVx\\\\\\\/45TqFWbr5Ahh\\\\\\\/5ljWVIYVzxjMkH6n6ln4CgMh3rDbNbY8D5hJ0mlb8kC8fJ81bZ8UI+x+gxXyyvUQXt+vdbsb\\\\\\\/imJqXYNzxbUUC7c\\\\\\\/sZ8EaBJf3hwvClwfoWAZyWMaBUWaUxFP8TseONXSyz+Bp3b0yBqGnhcUW3Hv2+DnF09KO4WpTp2XK9H460G8qxXXlEOQTwzYxdC46AinXkkY8ibuZz1ZUwQRhJWCfwmbvWyNgoEzWAdFG4NjyFPIuWrv+t2cDNc1Akrv11eeenHWi3f5d9ygNiQNfwTlbV+Q\\\\\\\/KPeXVz2hSjJGmjenIt0egBa0YyavjCBmDzpLpw85sByQgR9izBnM2uvFRp1HE5FJ0pypQ3MHl\\\\\\\/K2T41pxRKpxqTOl+xWX8ZD3EMt7G43MP8D7dMqDtnzwQyColP8tVKgKzG+7JNyHATtefNqvf16ddkSAmZEUmj23koh671bRJFdQcAVs3bgXy4xKuGE96toPMr6BVxlv6Hsu85dBhGBNAvxlGVkBtAcD31\\\\\\\/KeL4uiV61GYMUGMxIZYeltbrpRPuiyZjzKtLWv6GLe6sm\\\\\\\/J+DxbRX0AwWAuzTTWiX\\\\\\\/3GiiOaHcXG5zhADNNJvPpqN8DdvHI4o2cDrOoCuVGnN8NlQ6I1n7rCIlUO1xkss9KK8POhpKREAPlirZkJ5avtrPI5bS96iDUk59Lc8DaOoO1DFROqxcwQTD4840kYbXurLkVn\\\\\\\/cz4iPkSbGyfwUlHN4JKTMpAmxbqy6i2M9L+xxthZN4qTeXtO7PbQzbFOwN7hVydlMJrps98uCdiaVYXr5jVwaSygoP5\\\\\\\/Lz2uIqdqFno8ihZ51otVYufCAaR\\\\\\\/oNTPGOSev892z7DBUM1ssMghvKmWZeRpIcUVzbvTRmDg2L\\\\\\\/AEoRdz6iPGMhBJauEzU7HpOOGV\\\\\\\/NZwjw0pnoQ1FstNR3ydQ1OOOq22m5y\\\\\\\/a0Cq5lcKUXqgqGX6HnMlQvaeGWPa9tFIV1BFBUi9Ut\\\\\\\/v1KCLMW6rC9UZgNYhAhbUK3s++3iSAygkTYV8j\\\\\\\/RFOq57SJXd1heXLj5H5iWQKWQOTgAsWbEj+2E8U6ILPt+moJRB\\\\\\\/vlESu3hVIGcOqsEjpp0Tnc\\\\\\\/mOtVWTz2gDoq29ATjpkPheyXanxOC7RdswXb7yor9x1y\\\\\\\/w8XrJHksYnEmGYF\\\\\\\/Q92rpNKtKN3kQt+1+IfoRV9JthPE1iOMmRBLEk2lAV9a93XdJ8mt3OI1BmCE\\\\\\\/hiKLhIJbF7ggmbq3vSO64B3wxib\\\\\\\/THUnpfZpSQfKWh+Edmy+Z\\\\\\\/IWW65Idfmnx8n60Kc9vwvhtDAUDqq44YAMTooxzrOI7uhROWzNIL0uZ3bdJ2MKpcTiV5gokDcIbF\\\\\\\/ifu6BET9HTvJn614MREbM6Mvqp5B0uZlvw37FsIfej0KeNP4D6hdQhDjHdhUsvB9e893FMrELmWifOSy5Vaf0+eQZWLs+jEzEAjJOS36BK3su8OQACzeQy\\\\\\\/rIPmuJNlO08sPPxwcNtKA5nBIP2NreEJTvxt7\\\\\\\/nivm11PGTGJnLbq82mN6bw3rbmyHAW3QsS4eoGtMOeUt42R0bbnbx+MmEFMtW1oCtxcVvI2KT4+HYucqrSbZjmKqk6QFANWOzRK8hxT23pvIA96iu7VXODcsvvYkb0E+sIruKNjLWPUe3hUwQE822E6s3yrxlHvs+Oge9\\\\\\\/Tbc+poRTvlYB5bnRGlqH3jSPXYuspeJiNG5QVPEmQWK4sXXSFzKJRvYmDumxrZTaGaccXm2rHD7SQWFguFC99Q7x63F7yBSpX4IhjaT6v63RVb\\\\\\\/frZisHaLvJBPdp8je9ms7x\\\\\\\/G7YNC\\\\\\\/sNP1cTcPFT\\\\\\\/Jp82K4Qk8aZk8gagTIzS+\\\\\\\/Vu73ttmLmsOQ\\\\\\\/Urwgkcme\\\\\\\/a1rdqxe824KrURyyHk\\\\\\\/o2C2oQpK9ESxdGqplCXBSZMB8rVbCYSRga1TZSKy1As23mVvLCXMbGCKGlqnWSFLcIKymrlBt4RP6PFqm8bJ9cisOPUhY45UD4IHuoU56CBd3\\\\\\\/MzEdVZMCRSk+9Q4UXwqth1i0nHT4WCfz9iFTJc9sd3jOn05JC96JHHaN9x\\\\\\\/vyC6pHViWUZNZnOS0\\\\\\\/IzFNJ4Y6rpYmlzgvwqA923QL+6qNTkYW+31qAh233OunAcq4qiq\\\\\\\/nq9LGF0U0HDrmGhfqLHCZmYm4umiOn+t+87W1oOU8vltv8vmxGYDwCFAm9AXsVSiKvYjwGr3ETsx4azAye+N42Pw07CpmRi5yqTwSDyYllFIz5c4euvIU3DL4jlM\\\\\\\/e\\\\\\\/4Qa8HdKy9HDy\\\\\\\/Jx4h2TwciYvJ+z\\\\\\\/4rrVdBQiCHhuWOj+wxnkGi+tSeS0R0yNULlV4mXKDIYEyi3NCReqrfGAaN2T9I0fxDTtVbjuQTrcLo3\\\\\\\/FzKxo56F7up2pZnjrmghOWA13gGdeWGSOHfZ3DYyTLlcUQwGDcCqChPh1uvlwThDyndA6mJR6OQHQsdKTKNeW7Us6LrGO5Pe\\\\\\\/KQeMNpePqVNAydNIY6u5Sk9YAqxv4uctvPOX+Cy\\\\\\\/UNQEQkDI882nL9WevhQo1MoH4tUe3byeYjwlJ983R5dikY6w5gyJF3aVGexRWR3g16rM9zBJicQ8a9cA8+DemlZW7O6aehZdA7ikCFItIEHIaimmha6B0ukYg4wmMOCOk5mYIEtMAsvgs0oo8Rlwocurv+xloIALQ7eaR\\\\\\\/yFegdPGNQsLJGnWr6WCiQEkUXRJXr6cWj4DmESlZrbrBiVlRxhPvASDQyAYQtj6ri5+Y432PPohkfWRfU+aqvKftNTex9LXZzo8u6\\\\\\\/donrJf6oo\\\\\\\/QrSRyYh4Ymq2PWXZxRlpUsBJRLkDDEry04q\\\\\\\/mvckHcmvXkD21AcZUalWIm5y6Op571ou1IhW2p80rUnBDwN6Q+lCf9xAMai0FiS8V1rKb9yV1tB6g3EI4OqW4z+3vtBVScgojbdj6lZwB88CYodWbJjDvmRZMrcqIbZV8QfkFvwMRNP2lIMwHsFURQ8R5ZQQuKvo0nvqAeAYrchwQuW4pP\\\\\\\/r+jY5ra1k4qepDL5Fvw7\\\\\\\/Ant4uz3V2dUuxWt9yhETwa7aXpkbivEO7C\\\\\\\/MGG4TQeEUyWNpVhirgn50LrPy0RJv3x+GWzLBSs+fr1\\\\\\\/a4lwaZLuc3vG+j0yaIIVzbKgFTrta\\\\\\\/0bkmUAthmhLgwJHcDfH9ibALtLCOoU\\\\\\\/d80R+plT\\\\\\\/8Cocz1lVMzjmp2iPvty2dFe5REj6UNYIryQGen3j9bNb5jwuNdmpMckEO82Hgnwu87zraDWljmX6Sapvuf7s5Zzd4NRf5n50G5dwwnBVzODY8plAnXqL4c78oJiMVzHB14s40RbrK49GvQhXAui8tuIOxCxJbc9raSS3JtSM4D0k7sF0bqMShyKgA9aXKzu3lgRtGEdopib0K+6KYccY7mpUmy0WpxQBUxMlYit4SSzbA7JrwhLuW5H26a\\\\\\\/zybNtbvRYiqE1+t3EIL9ZXd5zDHPZgII6a0MbzD0Q9IwYJgN88Oj2hcZV6w3kN9JRW9VDsigOFXzDr9qBStbzOCxugbaQBRGoJnsoGdjMk5kYt8WuKGgNDCtwgKiNUo5Z78wo4TGjwa6KLjZlafnV8Appksr5aUX8zZEskdSuRRiPB+bz4DkNOm4gK6Sc0QPmtacj37Y4mvz6skPHKN07CBXgYZ+2wcv6+wTLk9+mIMjoLh0LicCfWOX05DRnZ1N6HyDDItV3GsiGrZkP9bngYBRbQ8tJ+xzIBN\\\\\\\/uGSVfchEv\\\\\\\/4mIVsMjFIgp\\\\\\\/fXv\\\\\\\/+MU6txXUWSbkFGapwfk5\\\\\\\/aEMhPDrXU6birJ\\\\\\\/Q4TX3SaNwlNJyknp1Nnz95HDpA8r1amMGORqQJLq8hmQlIbDafJ9ohQVX9FdD6IkQZ1AlC6HT4yRiSYL13FH6xxx80fJ76X6g4dFmWrI24gyJ0hzzE3Am2DvCHJd7YOwBfU3uUwUORLAWWmQyTfPWxAtiDG6I3ndgPvt1wMPxpXHPWfYtNqfpPkmKfuUBf6K6h5T0GXKfW2OnYS+6bnn9EBkTGdkSZIBR8ppOrD8XE6IZ0kjUMJP9e\\\\\\\/sXebPjdJ5lVE7OZtMp0U+SNPwIyUG8xK\\\\\\\/sVh3X63hb63pKGJg4nqvNhDSd1VDo0xFQ0UFd\\\\\\\/CMw+RDolc9f3W424iFJysVs0pZMnKV5rN+9pVAQSw9qIGd9OFqmfcs9QWMxcHPvXYVgMWMzmFFeF8Y7KLbrT3yL3Svq0T1TBWFj4VbiLuH2Vx7Tax9f\\\\\\\/m938AE9K38MS9VbaH9QOo+Ap63Rx4UUNl0sfbeVYecqzOzLU2MN0+2dHOjtqS+a7iXoBARraMNGBQnVHN3cJ7X9qUDqD0Ve02WtDEcfIWbGjUIdc84btSwgHAWSjlrhXbzPfOIBIC4FGmu2OjMdsjUAhcZlKNvtGdrB2nB7zJKay6661fzeXkW8CKeE044FnPly1GFsT8O9Mvf8zIehQoU0yIEvfMwrbcCaVnJUDANxX4k9ubHlJ0\\\\\\\/GRN7SAsjcTKK782wJGWsqihto9M35dVKJqfiK2FbV6dJIX\\\\\\\/C\\\\\\\/02zrWl+BNDIWwIfYRXXWCl\\\\\\\/+9gmZRwraLygQH0j+s4H2XfX29ziHMKRof\\\\\\\/lw8Mx\\\\\\\/Z\\\\\\\/r\\\\\\\/28KbHbvPEPwMVZZqvPITroJn4IrkLTAukbJ+QhuJiaciJYGSRADEf7tdZ9+qrx9Sn959Tjjbq4LBkl4sv106khTGIkDQkwxJlgwzaFHHqsCRdx7+mr\\\\\\\/tFT9ZonhHWxlW\\\\\\\/\\\\\\\/278dOkrbeoJujzKiztOOaSBkUfoDfcFGJnzG7jx+\\\\\\\/lpDT0SVe5C9UDeNgkBAhEh2+L92Wb7rWM3OBJwNnoWhTPgRJKC2x\\\\\\\/2N54yEl3byo5LT\\\\\\\/tFMHTSOB5JPCy4BVzZZe6jsxC2c+bYgy9R1OTgUZ8QbUipjRLnSb+7+M8kix2Uiz4i\\\\\\\/E7faDIndXv5qhEJBLp0KjxxLQSqZqLb5FwI0XadRkZdvEhPhC7G5yCcv4nLP+xGyoX1ekfyBnPQlk\\\\\\\/s\\\\\\\/l1g5tyWuTG18WAGLd8r4eZfq+NLDETDyxuatmMvSLAfl6IVoiapzrntQ+uLwYVrz6VwOCXwmrDjavv8dWelgHwCO+2mOIRu0MK\\\\\\\/hyxM02a\\\\\\\/xUFh9cGfgsbC8TIHd5aGHIXv0XqBj9Xc\\\\\\\/yQExA6azt8FcxYH12xWwttuUszYNSVfOVZarSPaSIqwNAJP1VmOYDHH0FKTujzpgNxgerGG9s7v1XPwz4uHyGmTe8mm\\\\\\\/bQKr1Qv4nVBYmGMHAU6in5Txh4UqN5iBAvlZR9KEc8E0z4bqi82yyoxYda1drJHTveLeklGmMS\\\\\\\/8L3+uwXmlwC6jg33EBA6gHhfoLFmmMn+zAOMa+pA7arVp5VB8vuny5zndPLo6ZcbQZZbNFp4Lop4yynNgSZxkrv\\\\\\\/F8ms6C9zM+BvCG1ymtP7AXSZhmT4j+7rgFDg60uJdDF6q7IEhSG8NqDpH3lCPjafGny9kbvmpu0ZZJYy0\\\\\\\/cIs+kWHcneLABwpcXAseQshcf6Ej1QVK6Suu0sEX750tB2r8yy4foks+br8tuBslXj3+0NPNqJtTHP506pl6cFpgSh23yOQxHPmOoeF6QyXrgcEHhFbRYUk6+W3FPy3ZRTt+\\\\\\\/6i7WXQI2auPADeUrSUT0vkdphSPJpSkI8NEDCaPhElXxDmHa52GrLnCzVCBK4fRIAcBytdh\\\\\\\/yAIxtjm0LCAzq0RQ8NwHXnqoGh2y8IVJaW8J8NdYP4+ezvT9X3yB\\\\\\\/6VceVunE0giQoEClm9FTpgP4g9WQ9qIL3kLhMLRBcVhcbAQVs57jpA8OOfOrb2ma2NWWPNp4E3AMYYjIINkW0QexenwCnTau5zIELGtIJI\\\\\\\/AYhwwgKCvV2T9JqbfMx8R8zbbfHbjj61fgbcvKzXHsFv5oHpkzoyOzpr1scfc5lfXP1qoOQKa2WegJLMbxWZ1QORB6GtoJb\\\\\\\/qja6SX4MJDQqWSEoISKVXZOt7nwQ9hw\\\\\\\/wBTMJmBKnqvI+bBaeBMZpgokED5IUhXQBT\\\\\\\/K\\\\\\\/i0EtvCEK6+huLslCnYorOeOOSUUz9K+m8YpxGZ33hEaDk2oucT2q+EmXaP5Uqe7bQSv99EEeojonWnvJ0l\\\\\\\/jn0JdGTKFJHGai7\\\\\\\/tdQvkmi2rqRYwJfJWbY+gz2jIPTNii7IwMkqMCR7rHH040LUmWNhmyBcBnML+mlpWbBXTWUSek0i8yaXTJyFmAvKTJiBnkSN7Dnq3B03US4PCGmGJyD5Y7hX4TV778QfzlwMZQ8mX0UsjcH26oPQLg95tAKyADo2exhP72r4UL4I0tC4kojOPGKfKL6tHxRmFrrq8ymfxo\\\\\\\/f2iol9rQ8j6Aeq0RClusEA8+vXG9ifUAKarpcgSnJJ44UsFQVuUN7MM\\\\\\\/\\\\\\\/3yXVDx5\\\\\\\/OkVxeSMUuLLJkPAlT0LOBNCNtLStd3wlpK7WevE8KoeY+eqhm57vSUL6NeSsuM8RyFJoLNGbZ29omDWIOqkXagufO1OXnCHEk1yV+i56oleNhpNbHLtEXy47u3l3hsSVMUQCvB7wEuVscLWxIVEaZDgjG61pI33H8ielBBoyFf8Cn6SdJ8iCiKGr5de85jajNQqS7Y+3+uyAq2gKbs2UuZ0y7haQeQiK6UTW2hWH9N\\\\\\\/oqCQfEw2tYSjzBzps8nAIL8HlrZoWJUuTTcNrtUWqEHdX+\\\\\\\/+fGHSh6QDNZZfTIN0NAu1Ou4IRX87AZFpwNsiPsm7HajyRhsw+Vg9Q9fbS7sPN0Erej6FMoO2\\\\\\\/pnNzpWrt\\\\\\\/7HhiV6D9sA1\\\\\\\/+OBY8rJ6qEF4KIULw18jy69oaitvyk9KufK2hJzhX5fWeaCclRUrnqCvS1\\\\\\\/Y1Qr6FwrWGpL\\\\\\\/PnY6rTtPCpIc954Wnzmj30Cdk\\\\\\\/9bypdcUJkRcucAn5TKlA3dZcRv5sp+afuYv0MAM+jhNrPR4nqGCPcpl\\\\\\\/p852eLLpZvFThQR2VApzMrE12FMDY3\\\\\\\/6CBbmfe48UCPsDBRYnGK3162lMgOs8I4VVD0WZeChDMNEtWdQpSyNpJvG5jelLp8G3dQ4I+N18Tum9lSv6jCfZ7RYSRz4KbaU\\\\\\\/B8e9lBkYgv+9B3iFxbRilfZKu6zTFrls2e+NQ+Fal8Vi6EVNfuchnuR3abOybOoyuTdygqyTiAhWlBjYPeblIVs82kkw59VIZIpTFrioNlioJ\\\\\\\/T2\\\\\\\/UjhkKXYOZCWeVMzHyveHZFVubqpscNGV2AqBGTHPNGjpRrHJT4YDp9E3dN7fkZGiODtx3AXHO\\\\\\\/hQR5GT6ZMuqwaHb12bnUBl0JS7C8Tl5APUguX196ciYYQtizhJJ7ki2rEjESFI9VK3hMhL+gbXR+FX5xlRsnzYDj2Lt88phHqeoE+c+2nkCHasKqt1eFhystP0ytCHQ6adfJmvzyW\\\\\\\/jjNW1P\\\\\\\/IGxaDWu3m8xic7+uTrj25FKAVclWIDgHAvGkVQssV\\\\\\\/PuYMNuYVaIvQCPCDPo8kHYgNIrNXYImmqyoGfuf7KZwSkNUuF6L1WRnnl\\\\\\\/R5pg+7uB8PaIlqthmmHkq\\\\\\\/6kr5lSjVhXetpvaKPf9hW2PeBlGGnIWJWR0lAUMY3btseNCunU7bFeYpZ87kHkoBlpawyTyQJK5smN41iFyGnzGEf3RFNK5mFv65D4Pd7YcISmo1Wazp8yR+FpJgN5jzsl\\\\\\\/wILdmhtrkjAHdkFIPSKnTatVPLDp9iCSfjf8YMZ8ydp6E10xLEKIUFHvOmHRfqK30MQc5V9DfdMY37JbZibZeA7AxLjHA965XuNAnIgTBHaliQVaoa1KvJQI3mhh0p25wc6zIUNcxYuOMloocOyuLhW5fmWn6bwsg2\\\\\\\/ytmBG06Oyy2c24Wf09ghep3fAv6XLaBc8tKoXeRdtJXcDy19abdr18lKAaeMA3di39lNnzJOwfjMOmo+wVKjQ05kMKqPpfJAnG+pgGAFJm3T5+ShG\\\\\\\/Eah1k1mD81kFtrtwodQh8hPW3s5RZ+D2E6\\\\\\\/7vaBplku4p\\\\\\\/neF3efqn7\\\\\\\/NUSU8v3Dae4TkQHEtp9zLaqkTucgaWaAcMQxDKWp0hhewROy02mOMXaDpboZW0EPAXbhZoSfzuNo5V4JVvpUoYGVU8Emcorrd71KkYLCFW4Jo\\\\\\\/EWTw8JXqaM4VSz5hfLUxOPQ7cM\\\\\\\/UJriVKwVcKrKTtlYEEPysSYGXvZ\\\\\\\/Zt+1+1d4kie7i4OrD8m5hzQpPwX2+gfx8\\\\\\\/omxUPA1mcDur9FFtmhTT36ukt4lRKMfP1FcT9Skk5QGQJAeSpqJHYLg2iadaroBNYVxnvBzuU4KQe5dHjxvclKurG0jNyo\\\\\\\/RnVQiWzpuQxTEY\\\\\\\/lWvWNkq0+kxA8VqCbyFiBJCp8M29d1BRIw9\\\\\\\/jVF+grKgiPZzzVHX8HrY4ghWayqhJyVGn+smVLgXl8PkjQXSkiFmEKKNd0JeOV6Y1seomXZuau1lNsQ02dVv\\\\\\\/IrwBhASYrL8jsugmnT0G\\\\\\\/4aBIYai8zjGt0QgEv54jMZF31Qmtt6LRylvy+kscbAwo2cMat+ejNDEN91OAKAr840DJvWujePqfIKRIlegCiICPJRQvftfQ192WZ+HSbwoxfKQpBiWAajcj0bT1GHyq8aWlY1v487d9YUyrAnyAiTyCJbejd7Yte6OPd3Ce2S38qv2s3oewRGpFElV2fEjwE7fl6rHHsppzJMGid0KgwUIR4FhaAgwbvtHehCziRX4k0lJh5+9vMqKvQ5aZugYwJfUCSsSQyUqv3piMek0VKWXl8rLYeIGPsdZdy+2wQnTILk8CZTTHUPs+nkQdgB7ej1GIurDw4hQkldxT4KE7y72RpQapgNm\\\\\\\/IeFyH5iITA1WMekCZWwvI7NP5UPCCBV+6yrZnH8ChfBeyKSCU9tX3Np1+TlvYYfAH9QOaCp6GX9SVF17DjiNza3ldbeX6M+bD7JWBIo5tZTqgIwayizcHz\\\\\\\/KI8x2WbiUnVaHV2vKgPSZfEd3+B60Nnp5mFURDY2U\\\\\\\/N+9ZbZGjBlOBXdCNIK5CT8gtP\\\\\\\/Wyr5pO8mCCQt7Qw5AvA5ijgFwO29LUsMN4HvbH+TCGZ3Ucv9FSMGZ0Nv\\\\\\\/NisMviatuAYxwlTSQNgOn91bEgMhUSYx1WMzBlDLxGVh3o\\\\\\\/Le34TCn985CTweyK9Nuvg\\\\\\\/YVuVzIU7p0JmY+UWgzHpYsoah663HXBfosGd0jt7N4hxzElhBEbctg7qdKNxfBRgsU5twf14ALk7jLw56zbiA1TemQdLAk7HvW9gjnufp8BXqSSkW4gcOqd3uHs3JFEZTxuvgv3b6ObZEJRLLO\\\\\\\/5dYxi1FY1Fqlae5+BQmChmKCIKBLMBUJNwdD3xsS0sy4ESjRgVrxdvMkn8jiyW8toAofBlCpymkbrUKGaLC3uwQpp0xSrT86MIXhRrxefpUu\\\\\\\/Dma0VXu\\\\\\\/w9fsW5oExo00FMvARkmoSYS8D4YTjiw2WX\\\\\\\/Z5y908JbPKFn+2JQZY7DQH\\\\\\\/LKbI1WKSwJw498YWCFcacQxPiYSi0q4e1YLkMkq7WHElQ3noxgxmJ3yjFGUVs5\\\\\\\/Jd4HkerDBhWL+n3m4iCGfsrKsVatU1S7bW8DlotGAK99kE6qxHD6Sz7m6mkcGn5fiHw43pXvEMSwgt7sZk3alv+GoS0nUCWyMgl4WNqeSGrpRiFCUPWPawLcJuBKgKHkJwxNnJibfaLxykm+z6BK75MvTnu52y6CTnf9BVmUIHJiMQPQ2unB3prgdpaAWE6Ya2xX05ZlPbYl2F4JM3iMqsphdW5j9bVP05LDWmwv3X+SNhwY7G91Fe0mUEKZeRzbSPRrFLmk7E+x0nvycsx1Lrpklc1a3T1cyXRB1mlVBfdpxxBRjfAxosB+LKeC9PjttbM+oeGf4UX8hZDJ6gknqtfcPtDm04I62LkfxlNAP4lpyQl14BxiaYPrWa1ppoBrz6QSnSzpBwytvSwDdcY6MIDjCoxzbT4sLWNkPBIBUXatqEYMLbtEX5I5z+ygJoSFNT1BBBdIGRSJeXyTG7URgcQKoiuvjUCVcaBDV9EpRrCD7JMWmFHTzCov0Ipcnq\\\\\\\/NIoh3FcFBZ5oOqXsG5umOtQyAgVIQPXECzxLRN0XPv\\\\\\\/WFp55l8OP7hHVGrtgyabNwSyLKGJyYAmh5tcI1en6nMpzuHv1tmqJdQb43FmborcG71Px+Jzy5PEqdVeQRpXiRirNskTD3nGjMwacMx7duXdy5xMgxxPrKQ+vck3u+Ovqnp6pHY+BhDkjYVYWJ2yaP0Ba9ibqgTcRqdKh+8\\\\\\\/mViP+pMmCVDheUMmNGp8dVt3NXfct3ldzOWTHZvPSPL8ek1gIqHvvmOP0d23G5llN+HuXqp+wQBz+lDJ4EpWUlytxOpH1IbnJV0rn+yYNRd4aU2PiKcuPfNV+KtgP+2z9AruZm9YGPorPKjZUcRcQKILntFhmZ3b3Fv3z4OC17KKhe4Z3Me9HFvY2f7lnsm6wSqD\\\\\\\/paacCOrwj1fCq3\\\\\\\/rfQ65d7IT\\\\\\\/Vmhn3V4TMNtdoqWqJOWJFazhW7RQApsGOVu+GoxUL3AB7RPNt7yhq1\\\\\\\/4dEZ63orTancm1j\\\\\\\/jFqApRLZV014H3Gg87vLyETXwrckzk\\\\\\\/9ZzA9+MVOaOnYPnD8r+\\\\\\\/Y+duSoRwsg\\\\\\\/zErvzc+fwA1DBCe4D5vcYt6XLGqKNqBOmQnioolJ2hyrmVoh9YI6NcEygBgpI8R+Ioz75VEKWG5fGgWoTxaQBVf26DOc5p2GrdJNxCHs5Hazbh6GlkNAq417MGAt7q0kmFdJxix\\\\\\\/jMGfkRPS8VP3Oswr\\\\\\\/PAc2UCrew7e8Q92xS461So9Cwr\\\\\\\/f7RYKe0IL9lSuTRAa+Yda6xzRhjPH0b7TcNRdZ3OMfw5rZxHFyXGtZSeSVQmVrAioGoB6TR9FACaUZGpFe0dp7s47QaFzZUVkvfNiOSPwcu0sywozYRslMLDyA7Q4wjHUnXLuloBr\\\\\\\/w8k3UVKIv1QyjFrM8ifOO47Xy1XteeXeXk0o8SESv94kWrhvTjbEw\\\\\\\/bg6aNoQQdgfzeiknieduPmqMVKYcJX4emOBGEvbDmaYqhuFY0bHVGKoFzKQYAJQ2xcRUgyJp7TBBQvttjQRuGsGAb5IQ3XLPSkrOZQRfxTu5KVvKAScJmoVzjNuz4luiTESzy4cXot1rzjRfetQOaliWmsyMgsvOjejE7A67t1w22nZuht5qQNsjDEV9g0GdztHD+o0FULeYqNdlVmQqBWV6YeiNvwBWfkY0e3dwW4J316QzkL3FYpDe+NB9a0uufVBuqAEtO0XfRO4Q6QYY0z9q\\\\\\\/prjCGmeScsHbh99\\\\\\\/g76dQGuThQHqRft1OZdk+\\\\\\\/\\\\\\\/gKQs0sJBXoLNX5KgaBRudumCatvdbAnaZ\\\\\\\/VafDJBKzVxbFdAHnvFjJT5nfFfzjXvcteceIo5RHKMlu7SSg7a8YodftMuaMC6\\\\\\\/L8AuBPGgiCOupCf7texgNaJYOrIFCZOwhyg9H1gxXKHwKwwRlupew65QB\\\\\\\/H0FgXMp19A2C35naxPvPb1DbibnlVlipIxBW0qyftfQmxpYb85cPi8k3joQMbTkia3JGjzwiI6KaNjBJX90o9nSh6x0FuQP59AHxoc\\\\\\\/5iePjhT2I3wLg2w+rP+aEBsCCr\\\\\\\/lQ8ngySxxczcKXEqTeNmjyiQxRhfefBfOaXrGtKVtM6+zOpBkDgxTomn18kVyh0IFNNwFsi3q9KDTU12emth0b4thTx8TBE+bR0XLO1NRQChhX6OX1GGs3tkILZaAkD3ciincAMFsOHXQfVGnHHMIC55WLySwDoHy5K7dWOqcQOhA+WtkxTsuTflzbamkLFQPn9G+vVY9EVmq3Qz3DoSAXP4Ep7eJNHAy9pTce+rLGXhbQx6rjEizNMl3H+wZjVMZ\\\\\\\/gJgys30QnRhcn01GgBWPfaeE4eIni6i2+S7Jt7iSmJUyHvV4UtRgYiP5AtRfv+DEyz6ts1cYOLTQL3cpDuFED38uQ7GqyO\\\\\\\/WZhrGCjUiOjqVezLsmggcDd7+BH1k\\\\\\\/P5Gx8Lq8Zq696LQK8IbcpAqB94vXE4\\\\\\\/ICVu1E8oGpHnKZACkHR8LcC4+4UWz5VuZfCku2whcJaPUbX0AdycoJoalNiaGaGrYvYFaZonZ+jFOo5XRoNwMINON38xKcs2TcEjxQLapi6xk+6xN7dC++sw3MoaYvXcwuqOf8jQuRICw49zKkxEYs05GF8xi+Bq8H8c48yHPCmtaOWVhhKuqy3+N353xND8uGTncjGUelCE6w9fEbmEMxuDRzbDioNJMfgOHrX9kfjZyYpr13GLVOm2xyLM3l7Mbn7KO1AMKK540ZvfFEI2IZqYbYjoGvQAoH+XeImIZjlIEWgjZyqAuTTAHXBmie2Bh3abrG7ls0lKGOzJu243WyBmJ16kE4BIBwnj\\\\\\\/BZ+8Ki+Jet9nftT4v9xmlUdv2LpVRHfEAdStldsNdHvGCzi+vzlltOT4QFJ+WW\\\\\\\/FHhSjPvt2WIv5liVk065TEx6P7kKrDhb+m66T7pSXl11i\\\\\\\/k33AT9tedAHDrfV9VFCOM8RuOg62my5Jz1R\\\\\\\/8J1wp+4BMKZ069ppnZCK3udpOJMYp7RNcQiDvYVOd7NqyoLmqOMlLqq8oXB8SelUtC9HNUhIXxQq1S+ezheZ1EL2UkHPexo5WUn5bk+eKOH0Xi39OHcrA3lewj8Tn6AxrBTW6cpYYMxj5IXvjtDpg1JUAhD8SRvWc5dI8lbcGaUNQ5Gv\\\\\\\/MeTiBxljY3OMa+UEj4vHFC6FiurdWgAwsrUrtcNM8sqSgmf77ocRqNVPtwbf9u9Y\\\\\\\/JrlfR1a00u3KzplSx4IwizEIoixt2kGff7ONBGjzgZd2SAlcmdErG7lkvwpuNQfQd9UKleCDobedSklB2IPvF+zRUXHWJawCkdoNE3QzVb5FGpCtABM07QWPEMXPjCldRKXgpLpn\\\\\\\/uwBOFhipLf0mus8kBcaTykl3XKI74+5n4mXovKk7BEk+5\\\\\\\/PFsNZRpeiQkbum8rLYD\\\\\\\/Mc0nvS1F8Lw6VVnEVAyZVEfJz3OZlBq1k3qGjaUMJkDqrEahGgwfVhaQPrX\\\\\\\/lzBN92CuftYM9SeJ7M6rWzxP\\\\\\\/qpoLGZecISLPgryiBgtf4OgnwTRKWpWspGCMmgxzWV+aywkBQQPYtvZ4T\\\\\\\/zCA4gfoRVg1ZnTXPMNLyiWLfXejG9oeILOuqf6bUIgU00DjqfJ15K\\\\\\\/495f1wL8o93HIPmlkAkBC1WIUwfOAZC8EH8v+UGWpz3YQPzQqoF+HC7h24WJPV8vCCadBYscxtnAc8iNaNuBKf43eGfIQkRfczVrikZyQXaffkYuDnBsAG7Vfxcw3Xfnkn4eH8mvSqBS09fAkrxwJ8SEDDBnrPzx52COL6iSuhYs7gpeNP2JHNguIiLXVrogVMYuSfgTSgjawffzvFALJs5JAjzOJMGQnK0kkRazacvxnQht2Ink5ESzO7VEpkcRyPQmecEZzi5+Z4UIjUrUK9N9FP5wu9viLOch5ALp+p8Q7qfF4T57fGT5M8piwX\\\\\\\/1ccGcL7nMp1Cp4\\\\\\\/1QZSYb5YQaylCDUI\\\\\\\/wWtM0fOM2QdA6LlBlIRk\\\\\\\/03u3k44UJy6Hl8Y5frSL+8D4ljVO7wpI8nK8yBgK2ROIGjbeJ5gyk9ULgMQQZOSnDTZKbk8LPGmPSQ52dxt8i1\\\\\\\/Fu0Rbde3pdMev8F9U0Zpdk3sFsrw1wnekzhLPBC\\\\\\\/BQYeCakRYzDcolkSLaUw3WR34VdBWZN3uECoY0jAB+sdT3vBJ+L33WhNERAWky0L\\\\\\\/4sGYdFIbIg4e0MjgOw3pDPaf7m7S19QGR2FlWLQ8zszOp7PZNlkW3RgP+liK8qG0zPC\\\\\\\/zg501SfKYgbWGyH4xHT3jH8EJNsWBWzid5BopVqt0Yo6GaVLjxPjMF7ax\\\\\\\/IsNNi1qL8EKg4sLWoT9\\\\\\\/mCVNk9J+kB4VCVmy2OM3txhr06HwSCq2rMYNBbja8dA9\\\\\\\/kAfFskA9z96J9yB93vW00Mb0Sr7HV+bLEA53WruDdLL70ZZVSVuFYAvwRuvPhryu13Xh1mZFoPnh2gATZYlfeeNfS4lLOAK57eiMoIMChzmntsjqjqed+CQr4U84tQj\\\\\\\/D\\\\\\\/PWY0whGTfphIxqokYFDCS8SBx6J8dAtDvIvSND7r865GuQVzP\\\\\\\/8\\\\\\\/zPd22XLNu8KJw+CaMVUDoTccVLHIsT47iC2pWWigx6qIHmR9jAZw2TPKbKpPmUcQDVuej5mXXELU2MSBWx9cJ+NxjwyTcMFmc2wEYo4uxErUrObpSzFKEjFSlBNF5QbwQRzkHZuM9uj8jOCarawWtWdAyB4dDghM8SPMPhpeaayD7AFrrlrvc31U8U0sAZ5PkHVnsnlsmlZ5xLlYSencJtNgXgP+GKqzdWIDszsk0F3hm1xixk9ew7n8tVTAdcgHP32+drPuXN9zWkp+KrcwDgO4Mbc1WrXPCbukIPxQC9UMgVWphU34lufLj3TTwvWtaaUUbAy6+NGextLUJziVP8meEsj49BVJt+qu39PboFp7SfBFyyInnlSMrGNI9HQzeUjA3QW785uOEfH5HlLR64CGlr+flXkSLyMdKOsjwc9tEbBY24WvT4NSuPsnqFj0\\\\\\\/fhRc2mbPjsYQ0hZUSkfgzZkOEcLiHjw1UA3fJU9RENGrofqDBOJWcVZ2\\\\\\\/EWD19YI20UBIWIxkhBSrYZP\\\\\\\/V1ANU3VytYv0jIno5cxtuciCqnvVTcCqFfrAb\\\\\\\/rP1HXiUrCmlr1fyUb1kHFCEUyMc1cmRqBYb+65ZTnpaKphnaAdFAZp5xcJPSQ5wmMK0KtnbduDnR3Gs8zFq9NaXeD7YTbEKC2LB43bWo28njfgYF60tM6LjWFT5w5g4HKkW4hwK0s8BbY+0NL\\\\\\\/bmbYgNtDi6rdaCJdg1vBlFzNwc76yT2Yo3JcZIVGcP258Os2CFpqmzcO62huEDcfLB99XnSGRahRnRipYYtEUMXKfVQb9YVHU\\\\\\\/O8dMa264WotLJy6kWcBhUPrrdx\\\\\\\/9DcClSz\\\\\\\/p1i57Jle+INTkX4hlW\\\\\\\/n2DSvqxkhfVbOl8lsMqFS77gNP6u5QxjPt8a9hVCVSoRocxAYi52NILNpZZAufEkZnKmZZlXKTmYIlm\\\\\\\/BjXqzzswhAO6ug4BxncWvGDBSwhQfeCMYIwc7uJO\\\\\\\/x24PIs8Gp2i8ohF\\\\\\\/V1kXFqIaCGWbEfdOfwP07HaMaiXi3AdFQx5jp83MZDWGMC575+QKKiKmZOtkN60jZeQWdwgw6CNNFHIdE4062bEADNs20DPp7OXy4UbHA4uGrHOmVllV6RJfCyNHOJZMHhE6zLXRzZj2UmyxUiNFxPgRrqor26H6EY91dH7oS0EGgDnkurcGn8Z5FcmftxaXldzGKFK4XekuG3Tpxu2f4dBH0SALhc2+04+RJ2rAIbxqFAz9h1je3cDiy6SWkp4nAyMdrGgWKvtZhPfYARViLe\\\\\\\/guAWWazfQgmx2AWovwy+qzZMADoHPaxjp67OPZD0EJu\\\\\\\/UVOx12MTVXcI\\\\\\\/BRA2rQKZepketBSB1O6ixp8UKa3+J2fQL4IlZnQ5CHDSGBIZPgmzboWmAeCBCnY5nIfe33oGfNqc\\\\\\\/zLPSXouq6LSJYaa0vcjhkmvy9670HSdCtYiQfhJaonENoaUYXcVcGORHvtJm7HHC7bwGyYQYElOcS2vsPY9G64kNmS6Vd8wgDJ5jeMiKfuY35\\\\\\\/O\\\\\\\/ZlJ+g7ZUFMqA9kTTsk+xymyVY1AP4dAzHGF5AwOwjm23vy1wl3FWMWsHfOJU27ywwVA8dwwTuE+t2s5eJhh0w4Ulsqo7\\\\\\\/L66LRjkJfIN8DjOgLcedaiBN4n\\\\\\\/9rf3Du1M9OiyTjcRD0iQFJ1c0t6IxX14JBSoFY3XPQKfCO\\\\\\\/DiUq5FEAQ2TKyPZWBTXqFk06TtMarh\\\\\\\/20Uv\\\\\\\/JFZLd3R0i2N\\\\\\\/EZ\\\\\\\/kiHlyAEPDSKXXXnuj1kXdbfpY7k1NmDmq\\\\\\\/CU9crAzbuONBCOyeNJxm0o2MPFJrDQjkBEy84yt0dw4jMBNuSQQiJI+qlVoQV91q+tpRkZQQns\\\\\\\/E9zrgPGHZC8in\\\\\\\/QMcqzqCORtZ8TvzKxaBPkC+cfPVaOE3wIcWauguONyejdE5oN1YvJtsghi7Xcw\\\\\\\/v1c1ffVoLh8AYtVkyxo9XhlvYqdyZtulnMEUalDyjgcp0oAW+tC39JGfDC\\\\\\\/Wrk+gtsQT6mO8yj07GQlI\\\\\\\/D8RsO5GTEtxKLcCKrEfPy681Q1eqmkXGYh1qtbQ65FtWCK9RLzoADrGzSyZMm6jUsX77Y85yqzJC2p469yqXX437ZQMObkejzdiVdjLDDN7FQrxBHeFQATYKwQAKYulVry6f4yZQ5RbWlzP\\\\\\\/ea58L7f7V9RTsKjXnfcq5SYoTnVx8hWQZiFZfzkJmYvWOEZRmF8T9qi21icdSRWMcoNQJM0TZFPWAB5qD6WI3ZBdi9OSkl+9fssNlY8kI46XYkCnWilVHKSR+bqzT\\\\\\\/79LMys67g0MB+qeSSrQ+5cQi87CTm+QyUeipkQVglFqE3us2\\\\\\\/V7Tx0MkoDRKi315Hvl\\\\\\\/5qTY92ylCwXdIbu2oDNK2ItdwkOxBBdPR2zXelgruUveZfQlfb9oyrzMBGqEBmcg1fh4EzfZ02xEHpzY7rREFDg92ZuEmkRo+GLApM0oE3ksSd7PuPSR7Jy\\\\\\\/XGD9Qpz5PchagB9aj5mZcsQRRWFDizTeLmDadkZ5s7by5bdtkbIk3lyX2zV6eERdmGwE81jNNOZHDHmyl75JgTTHs+YkEr0WqyXJDwanRD0MGhaCnIPQ0lx7YqK63JUNQKPyrBgq4iqSD3ecZJKKHguvnH7o74L1Kd6q6vTFW2OxZ8n\\\\\\\/6uR2BrF4JLn5zusNI8rPCz5+bRZ\\\\\\\/sjFBZLnmT+3S4g9IYPt9qRbvFOwYyq8d++VaMnr1Q0rLWsYW9JTIRssgC\\\\\\\/b1DS3JChj52JTPFl4UD6Hn\\\\\\\/ry9UFHU8p84JlSKGrIkAuUeucqQdAzFwb4q8v2Y8P8ffsQVPWAKe+JZak7V+pE2JTlRdM6lN8C1LnLR8ffPOR5WGqzVvwdogmsg7jK\\\\\\\/vMdSgcVEQkrlU2VTJHiX3rFjhES4pOWyZeuwxjy4EZNojhkMWal2MoncZSFHMeYPVYP1itPzoQl9kgJrfUT709KfQEia2PQ3+IcyAkLtY4f6r1YQwurtDJ88P42sH7tOoQgjkQk7nHkJ1EpOAm24SlV9DcEx4rAy6lhw13y0FvNhMZ+g\\\\\\\/CezWlE6EA5oONQxflaK\\\\\\\/M5oxPQXbdm2wviBxSjRvbkgagr+iOmHVAvqjy86BCjVNH5aPvBQchIXFjjFHKKGxcx4OtHY\\\\\\\/dfupR93pS8fUtwAoPmam0fBD7Y2\\\\\\\/0rhoIvn1CRCkJPOiwyOWWOmr3iKpThF1KtH22xxUhE4NRSMwlJWRKQIHJ1YKdkS88s0A3cviRGU5bd3cnNEyiU6Fyysdv8IoIwWK1gbLIVPKoykRZJLTjPf3YAps39ViFO8n86LNurUjW2W+d4eCUlQSLXpXYU7TFlNlJxNwu\\\\\\\/RtDl\\\\\\\/S6Y8n55+DQ5OKimc5Qxc1aqsGdtGByT+vjXys3fbI9\\\\\\\/ftxtDQlI+LrmW+XuwOaogjyMa3kOF2cz4CIqNfcoqUgKbo\\\\\\\/JODWczdY7OLO7ypxfkjRQ98+IS5fbJ0hUmYV3tH\\\\\\\/KqKEY1O7ELkrvhaNWNFSKFeDQEiky9sDWMUgaqgkrA4PdU5EGiIJuuJsCDzMOHFtY9H4cLM4OWYgD4LWT2XXQU345U6wPmd2s368xKii7Uo0yYqNGhF5kOneOMU4HysuwYhgiSLOP78qtrrjvIfv4eu3Pv5kfSQPkSboR3NWHOnaLGS\\\\\\\/U04QM7x2T30hAcQlYEXzTO2GBgLJH9h6zO25Zy3jR0exIu1oXGNHw9JxQ3jcTmlq\\\\\\\/TFrJDiukmUBw2GtqFwlTwvar2Kv7sHtPcjOdITckPkVlXAssamerWHl3CD3pWN6uboWfJLItlKaNlhOdC7m7HTLaW76N4O7sWGWXVw0RxAwaqZmE4SlqDeE9zsbRS4Y1DwX4TuPfssFc2VgFgYGXaqCD1MHxg+g4XyCd2DU5\\\\\\\/xL0OKkBLe\\\\\\\/5ej1GWjrjFrcOBQ8QaCeAe1gBGi7luPVnYrgsoPvpopCfxC9YTVGeuBlSkyEpe\\\\\\\/vvKZwGmjousTID1yQA0o0bj15SBlJQWnNMBazKsEjkc3yhlepfo\\\\\\\/epIwYZSgPZaP4iMOj9lacXqyL9d+Dm4PiG\\\\\\\/32A3uu7WJrHbUa+ZKYnpc4aPYl8pI9ZSoVE\\\\\\\/5U2Yah5OrF434jNOyXumQIqRl4VZFOK0Ygk7y7GqkQtzk1Nh4w2Wk4pwXaKLx7gHkNTuAA9M6qlZrWQpKwVyg9CQzmy2Se5S\\\\\\\/PU44nZc7g4asVFlEhsKjFD9+tV7YYRfFehyV7Oyk62O9X0FuolP7hyUlAsgDi3tkoyZw4lewP086A2EToB+1eAS\\\\\\\/qDtgHlSOB3LNTlCqzYnM\\\\\\\/Ww8Vl\\\\\\\/fHuINUIhOvaKOARO8XdAVuN7oYWlpuDuqgFQPqWbX1IkxRjS1XG1K\\\\\\\/vfyDNvKy6k44SsmZAjLQxqCHiqCJdbqI8YGwG9\\\\\\\/0wYxPx+cIbL76Q9+50UT6P1ztxJeAkwX1ZhJ4gNV6kUTg9UpaLpPBOxxK0RPBwGi8L4U0iPGyKivwkDtvdqYnGJO9H4rzLIES+KQhKvMzN6l1xwxs3uT0RzAaIVQdWYnOVKjTpGBD3dqtJgqXmc6LTJs6aOXNx6U9FgJIGj0SUg+vyqMvwuq1a\\\\\\\/bU7pz8cheanx7NItkHnrKCt60NuEEitJ3QcQRJ\\\\\\\/JAtdyMlDWXEFGeHEXdFsJQFnoqvpxGP\\\\\\\/bwjZZ5uHUluMkNBIPEU15tokmmN2EGSz0MFHLiSSlDtHXuh0C1siseaIVeUmV6xlz8ezRdVZI8R6SpZFY8esMnm7pvJh5Mmnb70Y0nBQCSR9mw8QSikw\\\\\\\/rFbmq8OeGlfqFYaRAuhhrbaf1vj01T0gHMZ2ONvXWe074aUUzGqQpHPDGmzA+Y0bptkm6tc2rWwiVBvZ4W6tYnh31G3\\\\\\\/XYMYXUGVtXSat220Ar2x7lBL2jIlMLlHzvK7ctP8ru0ZyODLoCZIMqNVRVoCkDyPo4UG6l3b2czJZaDjNnCh4KeAiuCItfNU+RCLVil4+LiCSsp1SKWUtnm4hysxXlueV88ec2VI9w+UtX0OCt\\\\\\\/yXA8xPrRil3DkFEfNCMSxVdmC8hA1WCTgJ8qx0lJj8UHC2P8+es7k6rgHaWCGsxaDh2o68ctGURsaPy1KLrcTmfmSnbpPXI\\\\\\\/28VwPfcy3n\\\\\\\/PU6ZA3cYvN62Lmkkl+uoAeIG6\\\\\\\/08qPNJ4ez2yBftzdF91K8J\\\\\\\/LY8DhMQCk4QErS31m1VPof506CAYYM6hXVts+f8bwxTwMyWDA+sUE66kMvIceYtlC5jwXLr+RBatIS0BaCQ4SRTS5AK\\\\\\\/I9OSRnfg9qqGKmIaqEJlxrtRohh8QNUUTUhyjqwOdyKtJNtlI+FqhbRPm0+zVd9Y1O9OTrbl3AurI4Xmxw0kClGvfNCwYwiKgobXVvRsk0wofkhEF2CwpCNwaKT1t9FDiQQV5UeQYXtGiAFg0deqQsHA87ccfOt8KqmgJ0ZmBrLU40whddqYzMeFpaVuiQ3G1vOpRCGXpGnOJkCq\\\\\\\/YzyRjAiGfnK75GfzHTBTrq5NNMVybr3wBQzjj00VEDjLVpq94d+akR1tDUedNvdMDvgrsTDnxLffD641+nn9xtFrVBMsWIPduweMYNr\\\\\\\/FQugFcBnKjLTZmiodK8Z+AjOfVTsR43umW8ozSlf7aXePVAyFI2EcMhJkgzFa2VYL08iuIXq6nnxTTc2IGVJDVZ6wFOQqCghULBp7WIsxCvQh4mTyIgNuf1fPPzw9BFIBBD+gdCYq\\\\\\\/j5N2bgmpH\\\\\\\/d91QrVPfxGDz8aYmfTLDjFL0xuHV6Hqby\\\\\\\/2+hbl81cEtfLvWSm6vQeJeK1MRvL\\\\\\\/EPJbSAwmwnJmNsvwazAMjDMJwDDOUmr8b0hfDkM1S\\\\\\\/Dc6EM5wZgdkN4GAxcsHH9smzwpxKLAH7fDgLXS6u5APZXvkvBLkb6BzvcG2uYeh0s1J9R2jCykwkKoQ2eGBbIm9XPxqSLImxZnGvVjMkHqQInOkSVblUJoAOx4vO3zre9pPwS9npjsVDjfD6uNzvnDX7hlErfKrPIlhBpjwy2TmLzwXW0ry4zhoIwDp\\\\\\\/lJQsFAp5MVAXGXcnjj9tWrPwK1GW5K995cQi5UQAOI0z9jTS+VFSu7D7Dx4k0mwwoCFGjn8C27Euffb851h8VqrcpNF9A3tJcocn3PTolmzJ\\\\\\\/cP4KFYDnRkaYSLHbXGPMWFOASvYLT2n8AvHQCMnmEPwximjKoDF+UELaW6ndl60aYd3UufU7+MAqd6lMcF4bJ4JmBwpKcTe+z8yR1yqhFRMRpQFu70FRXS9ERiC9Ls2s7NPdK3kzfhPb9qAUPsp+Sezzp8xqJfE2TemVcVolgqFF4KlPX0Cl9qOvmquFgH3jQ2kjVPSVJxR1x\\\\\\\/En1PqYD9mZBrt5gsnqIXH4GRgUFnlj63qR8VMBnij9i9W3V7VFkeg2dJwHKu6ZnvV687S6iQ4aZi9+AqQaZTzwpmiuYNn37vSoOXFJPpaqMneZi5zzeyHsAvabn6payIkT+IeTM3SXvklbWuAVG9X672VYU182mE71RyUcCXz6cSII8kj\\\\\\\/sNb9T5vH77QU9yyR3frVRUEPqzPkx0x8\\\\\\\/JKQqc04lXAk2Gu1hUoLdv4y2o0icOKI0WCDwUkPdVkJ6spJokkt75XBh92qwY3aJz2inZvLsiCqBFtwsYo3c4QmRPKZNQPc7doGQKtJSO1xUyD5yssxQwnDl1rq4T\\\\\\\/MPRbrRUA5MmzPdTLpSyLMft4HraY5Ah9DExTzZGRZ+QCD8kQL6FuCTEUoLoN6Kwc3aWIZOPlQtVrcTUhR6LUwvIwDQZc0efYagmJBYCLzBbfWYiFjfBBV4K8IOBfQ0LnUJm2\\\\\\\/RdQns350sRH9Nz9ti1tn3eaAqPWQUK2ivr\\\\\\\/UgOe7v3nLr7Wu2p5In8Ph7NIZKlFwQUQi2M9MMoHWpb+xn4nH0zI1pwcOgNNPeftn65ZFEENYnYwQ4VY3hkEFTjNzRTmk+oiDpEOvVnVJW73ELOL+MJzGWcnnrk37NQBajU1uRo1fWCVkf1kqGl\\\\\\\/WsctoWTh+SpVP3d4x5\\\\\\\/pssUkxJU5ly7cswFiYnkPcv\\\\\\\/XxVuYXC9+wdZ9pZa57f1fbene7kCEfuJJtz1Cm7hSL9bT+wlI27R4p3aNqeNaIfvjLWIlLauP7hXoQFcUgX94q+9s0HDDqHj7I+1ltGHEvxzKU754mhKHmx\\\\\\\/Qf3PbqMPPbu+m5p9mOm7S9\\\\\\\/nfiH3Kjg85FUTkfx3\\\\\\\/pdRD0L0+WaKI0QM63cVNplRYQ7PvNziJ\\\\\\\/z\\\\\\\/lOiEIm3O9L7Wxl3IqbhlGA7s522M1GtjmuOIhmM2EQlpWFQLJR1AdZdKfiyiWJMrXzFYIEwN\\\\\\\/bHIqCUaZzRaXFQw8L67PqgPbXBug5hP1n3KoUEWVNgX3c5IGmU1O3Kiq1W+Uqtg+BgawIw75CEbyjzqU9Ccj9yG6GGdfw2RW7wywKGE2n2Gg9E7OfL8PBVyu\\\\\\\/gKY9TNQ1K8AX6lLbrwkXm0tzOg8R1gQ7mB2J\\\\\\\/nHg0ne872V6bJvn4mQt6HGtJV0JOC0WHWsrUBm1qDRGbzT7L+cuB+dS\\\\\\\/TBf2d4lt+GZGUcG+MQVEz8bqUMRe2akpNqhtrq5rCJWvSLkjyfIy4gDxu6flbdY0mjawIDtJi1JDvPpINNiFwDpgXO4pYGWIt4ugZVh1sy9vjclUTv\\\\\\\/YxUczgPkiPThGE2KmYVEu3VKlCSuQnk0bnTf8lRaj2zILW4SWDRJ8j2Z\\\\\\\/JddHkRRJdAUSJWQ9Gi1lial+PMqWqXAAxJTkYBgluqCm\\\\\\\/4+x7mZPX8b68K0Ri\\\\\\\/BMyKFUwW1D1jsA+3ZC+JMCLdpg4\\\\\\\/Mec17W6wA8N2j9L3owcvmoglbL4J1ksN3gKt6Xc+zn7eheTIIO+1wl5mGwRBpfLWFL2O8YCk0VlcjPaTaomgIyhsiE0jvBnQ1o2UFm6es2EQ0zyaVbp5lyn7pa4ojmzm7Jg7hBCzmA4KsFAschVNpfhRJosoA4ejCGk7V3bKH0XmfFwy8X+KL\\\\\\\/r8XJHEz1kCXveGq7meFuJAIE\\\\\\\/TWkW5NhXt3sYIMfjAVkfnsSG1zDN215qpH1zhDFT8fh6qpdHgTAQfvneAZcVfhqM8aBNkMKUkQuD6epJ6VjFubL3iQciLeB22jpb+52NZV\\\\\\\/RnzYgydNFOYA6BMnfhUXMPrVRCoXEKne+HUkCIaswn6EXoFh30ZDidrG6e+L11B5TNa2P00E0CrmbKO82OsRYLhBIymw9J7HoF5+xJMJftSVp3ehMWwhpjqXd0bm6rH9yAIEz+juTtuXPf+7JtYOH7JtgnsyKUN+VknlNWH34\\\\\\\/xRXDVaBnyrkVayn297fZN2bWiyLZ7cSfAfdvVbEBlduHE7XWSLMulfe\\\\\\\/J7Ihg2m7ERte+qVZmBI4+UaVFcKdF4FCC\\\\\\\/WEiw7XPPxXXMgERx57KRsugqd9KI0p11lBRZKMw0FkSqz01gkSEd2rWuZiS1k7g3NcI4Wv1Cu8mDPg98tSsT3n2odxFFmQKs3Rm\\\\\\\/kNa0YRI7vyYbwcPrsmd\\\\\\\/yEeV80cBuYlefJtOkCbGKoHPRd\\\\\\\/ewj1P2sFYH1UNXpjoPx7hVsk3RufYgwdg+G3nVnPVFycQPdOJOrU+hUsFR9R603B\\\\\\\/8rGKZPUxJdP+wFbht2gtqDb5hbIKi2qOEtLMSczhjRYPHJPQX5jwB9QiooZG5RS5RheiENi9L0dXCwof5rsdF8Qs7CImJ01ZZryozCiMFeFT1U5GppP1kdzoMBp4FL\\\\\\\/KN2DiOHCGnIAfoS3hwmwpo7Ap53va9NZGaXrXOOxCDe9rNdkmXMaVIW++3DEhAxh8oBOPk3FXPSepsbNXROFhGx0Lp6wFlpp35n\\\\\\\/DMhX7o1RIz\\\\\\\/wtBvENoqSX5dAfdA4CABzB0Sz\\\\\\\/Id5MzgQ+Wk8ot2eAamwCO1SdBOxyJqYy8K+bCH\\\\\\\/md5iZugDk2xfhZt\\\\\\\/LbMpz890nAuGwHgIgudmDGh4iu1K07YRYScclT7mWrMc0MwnE4s8BAsrbVYaAH1uLcEY3BiVNYpO2Nv7gw+OE9zPzPwekjViXo1pn7Bs2i8JLnyz6QqGCM8TXDY10doRHbmFAnkHI5lJib+DGU5nrvJSsGbzm6rRLym2BaTJCkYM0l6TGXnCdUl1qP5U9G5t1IAl2DJhJnAUX5V12O1X6XCA3cob2eU9Rvpor2hLyYB958VxS7BgIQDEll9dqfQ7Yo9AOgy\\\\\\\/6eBwkkRGy\\\\\\\/93+nvZN670rcQODB1uuWpza1R6UKD0h1YppL\\\\\\\/+1IehgWFaAPU5+JD6Nw9N29uom17IRK\\\\\\\/LZSHBzWbNYvJ7hwU4cFJqQv31Hn5rUOm9fe9BRJ87JOMcLELnD+olOruHjceEfTUoB7ppdlJEt9Z4ezzZuGcastKjSNtBkgsU9lHEU51RHDDjkSNLdRMPsSxGgwD+CnTsuBZ76mY\\\\\\\/xvTBCX1PupZ85dZu\\\\\\\/zW7AD80FNOI9SHiX8CNlJFOmPqNwSj6Jkcf\\\\\\\/mvjpHaDT2KONM3LYcBnTsTLJs\\\\\\\/Ej2kyJaArA+RnJqITptjoiPhYmWouZYkOR+QOH1YkxsuzDyt0mBEyvKaJFEApMLeFrTsNDyNJ571cJ3iEvJ\\\\\\\/wJjStLi\\\\\\\/chDMDXsLeLc1ht9m+yFGSlUzcGewsFmf5e3vXC04MZLchBoaD4xtHS0l+ICVtnNodlV3wdyG+P\\\\\\\/1bxZEzRAzoUVw434uQGhmS5LFyCAhEawi+LXVvnCVLtePuoOra1R\\\\\\\/OK3NGWTQvAC6HSQpjAgInctaxk2KYXZn2vYnisbK9Mj2I5M4mRJIx8e6f3jbutSxi8Cf2QvU2hSqFvDgvYeMDtaIywOFOHl2qcIHzlsSOhY8o8Qupj9ggOC4mDxSQNgC7FGgtXxv8alO\\\\\\\/RNwXPXgw0LlvOWia2gWVQPNRp0pjH3luDdtEI+lPm\\\\\\\/87xCwmHcUamETl7LLYiy01GHbQ4pQveWUE7KqMiaJHjgrkpHg7MhnMNMfNDcRo5zWHk7Li4SIiPLezP8260GHZim5zwZNrFXafGzMSgPOxOxEDUwJmW85ctH1Zw\\\\\\\/JRtflz97n9K8QFa+0gcjoWUS4FcEcHLp+vY97x5mxoTo9H6wVMxbF4JK5wBklqjGrDLjdITJTXXMJ5vmdH6FdaCf1HeazhrcPTNObTSHUgeZkCTIUVEro0LsppxxKUe\\\\\\\/XzFt2hSBR6sCDmuGoYbJxAHcvw17NHhfLowaMjNZWnUDhdFpr2WSiQg1wdUrs1YtGKcNUWN4kiYxFrw+kK8YP4p2iz5llrOgIHXkQGKi30EbpTCjoquvOW6YGgCTp\\\\\\\/MXuCnMZYHRzoXxcDI\\\\\\\/KcEyaUJSOmp3ec29kR7RluDdYEaPF9fnPMtXNLGdcmpiRSw9SP3H85G6rKwVbZzYFsVavO3aai4T\\\\\\\/5J7lFN3V+rxEgLZxB613yTNTRP4AYaHsHlf08mplvn1TOH4mQQNRhhwws0j8fKDjMPGsNJyDVu7f\\\\\\\/JOx9p\\\\\\\/4EPEpyLv8ja81UV3CJ1oLL7LcX1o3703bICKsLw8WhiDZhnpJtAQNvaLGMOHCXRcvfT72d09zKBt8ct+Udda8VhZcOjG9FKndJi7kHDE3TPlOYVpaHL8Bv4RuWPWkbxhXfBcQ9yaDyaqBJy+tbglmhv21xx3NnP5wtNcbrWLrW3onr+pXvbXmk\\\\\\\/t17Ver+xf25TTedLJ5EdZkTWIdhrCZszFVk8q0D0L7+0XqRWX0LEkSMqwNoCloZoqUC0qkmjJ23HxT4E0duhnQ4aD5wYNDJxb2snM8NEnjRljZvPY+TOtcrJDbeswujz0Vebjqr16p+D84Gu6Dwz9dpqTbRsuDFjvGjEwZ7Vq\\\\\\\/9rer2nXMpJBhP8FJb7Eyc9FR\\\\\\\/r1jBnDr0qOXgl65ImHCTLyp4sGgnukjE4XbSc8gYqT+\\\\\\\/lRFTvwMf6P7PGLGPXQlbCr9PUqwVBvZf51d72\\\\\\\/tHXoEk3yXL6tim7B0d+YwhSkrk2p4lX5zhXCavM0enuNJVoVVPZwclSQ+5vkDt8EffgZ8AqE6KXSYYfSz4p+F4jH5+T775g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pWIaNFC2q8X2PJ+rkHAlbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022117436eb8a14175b84ff5b9453d4b987fe36816f81d9f9c20fa716770fb4ac91\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YMQ23yT4KFR3EjnUjoQvrz0g7orQGCl8oS01NTG1UnI1uuziqu2FqH5jYBe0LtrEf+tGJm+GUDoH89mq3akBqBf3HKnmyD9UECD6mUt\\\\\\\/EZhsRczZThE23Zgl1g7S9jAbzOKcg0g\\\\\\\/qbIQk\\\\\\\/nKxNc8qwA6fUiuiqe\\\\\\\/Kn+RyiPtDUmFxKdEPCApf1HQzVozOWWne1soJpphV7GP9MLo9heQ8PRT41NFf661NQPvglVmAJvJzXGKVrD1Ir6jGgnXoevTWMZD29\\\\\\\/6Hn5EhuWyghWXlWtB5VZk1TiX\\\\\\\/yYFF3iB01zhVQ5c1CX6CLt1XGh4tbeXJaGipBpXdFj4Hv4kCQ0LOx+ps0ic57OfLOWhyiFQ4R\\\\\\\/J+Yc4ufX7yB8b3dcesNv0sALlvDw7TPKLX2Fr5gggc3y9gg7qcfaexfWoG+umPFut6sv8h2VKw9xDAupR+m2a2QDxj\\\\\\\/vVEzZJx0thd8ex7v9sY2+eQKO3I0Nas7rFgXIQ68d1etlUxIeUhZy1l3FZcz6T1xa\\\\\\\/9wGQzv79dgzMVUowyWQtIlozCZbgkPlP45elegHV3OCYyJb6vFexzsdGzpp0xr4nzNcI29UREPJbM3zgzcVsH9zwzSA8JSCf9DZO8+u+NYa5TSq3K1U37ppdKeqb8joWFddtGyG63esj9t6WyqarCPuOkJ07qCseZQcmgwdg3fYBzSL2vkSvQbn7SCWTUjW4DcS74ZoQ+J2Uc05+MPxioO7yXeZkYhDEwsXLZ8SEwGUhoxHn9F7qnCKTAGfPWaSA3eWPKr5dcOL0pZ9OC6Tc9LYOi\\\\\\\/slZ8FdH4mN6Xo3ZN7GUzPsrOnzcY2frKqhQ8tfXPAUxAr3NgUX5kWKgvtc0yG6tJYjX85KVCkIoqaNZyQUnJPDLKiYgmCBqpJUryYLn3LmbuuY\\\\\\\/9k2mY+4saviXskbRsKsBsskW6nbw0FGO3Ppez2FLUab\\\\\\\/0SD2\\\\\\\/5msmGqUvzjIIkq1VvBHqQ2HN20pwJeucu1\\\\\\\/tuiLVLjRzmkQErDOyUKdMk+CWFoWubRTbMOfkphK2Tt7lBOskHpy72o+EQC5JZ6ds0\\\\\\\/r1QHCqKfMsRZjkVQQKpSh2PaqppRdL1BUJD6A8P9y97oXFhyJM9FJ9+QH1scG6eawrKZhHuGarOw\\\\\\\/vjm6NmgREDoXtPHSbSAK5JH17fZysrJ\\\\\\\/lgBnsidAmfdETFpukXliIuS+Xngn59Wc4ntvlVFfVehH5LgEMR+V5cio0tt7ZgIXhM4lH97AD3IQaY2m43hkRj\\\\\\\/lhryl5W8fyPOopXS+Ur0EawzoNH8mugkzexMPjs97bN9kek+YP\\\\\\\/xCwxh8s+yD9o50rgBnfNRK\\\\\\\/iNnOeeF0rZMPgu8I4r9waoGfAoluTu4AtgyALjscqQ73eDPtSeliuRgYHl3QcIneEB8d6bUhjeJkg5F3unbUI441beYgX3nEh71xviBguZbDWt0+B3rWQHVM+wPn+D9719A0nNXG5Utzd7GDez7XPNlUoa9+AcIbWhqFiWUPeX9wY3WrH985JJ7VbiqfNMRKEk9L3K5Pz7HTcnFr9Mc6tBKnMgAgQt4PVAi5TOO7Q2N2nBaIZdaNnxHhzVM1gBAKCkB7QQalyh4mAAte9YqpxwhQldc5mEq7Ef9x5iJHPOU0W2OV3YGFuRH8dBcmrA+i98laTdHh3MaBQgpm7HtwtqZCn0hEiIrrgqIXVSCl1RKEyCzBxLyPLVh\\\\\\\/QHSrIwWUQUdOZ7TD6L7GSrYRh910W4R4MGRp\\\\\\\/+GVSfDaaXkQDhLy\\\\\\\/XvKr7cID53lmwEdF+1HuaI5et6pqlmlwMJL75Alz8lRaAQHmHBkn4d+UK+2FSexAUskXq6gCbyTUnwLN0xBtU9+WVZDZ2W5h+RMe0slAsct4IO\\\\\\\/bzZ+XJRiwUrIfMAiS5xrHp8MijPpaoFM7osPl\\\\\\\/J2jKSXaAHACc+qdhDIV8iQ1bfc4XzzFj+1+ZA\\\\\\\/0EbvB6LPNvNAyGr2nHGqeJ3YKMgnqas5ESTlM+bew21kC0hAQfHjl7VA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jismQizN1Mn724+iXLcJ2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217177350333ee24446747384e1d30fd44bc78a736e4d9f25eb4f31c7ab6826e6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460011 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1603" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8rIQyzevD5ww4cnjEdjmq4Etz36d2yX8JpuJg88FS5U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1603" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UDKcuVB1mkq8yRzmb6L9fxmx2gskZudBY1r2pF880P0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1593" title="Shared Genomic Regions Between Derivatives of a Large Segregating Population of Maize Identified Using Bulked Segregant Analysis Sequencing and Traditional Linkage Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1613" title="Nested Levels of Adaptive Divergence: The Genetic Basis of Craniofacial Divergence and Ecological Sexual Dimorphism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460011" id="node460011" data-pisa="ggg;5/8/1603" data-pisa-master="ggg;g3.115.019299" data-apath="/ggg/5/8/1603.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frank Technow</span> and <span class="highwire-citation-author" data-delta="1">L. Radu Totir</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1603-1612;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019299 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460011" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>DuPont Pioneer, Johnston, Iowa 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Frank.Technow@pioneer.com" class="" data-icon-position="" data-hide-link-title="0">Frank.Technow@pioneer.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">L. Radu Totir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>DuPont Pioneer, Johnston, Iowa 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BRadu%2BTotir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Totir%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BRadu%2BTotir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/8/1603" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460011" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460011/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1603.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460011" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460011/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1603.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460011" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460011/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1603.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460011" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460011/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/8/1603.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460011" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460011/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Training set size is an important determinant of genomic prediction accuracy. Plant breeding programs are characterized by a high degree of structuring, particularly into populations. This hampers the establishment of large training sets for each population. Pooling populations increases training set size but ignores unique genetic characteristics of each. A possible solution is partial pooling with multilevel models, which allows estimating population-specific marker effects while still leveraging information across populations. We developed a Bayesian multilevel whole-genome regression model and compared its performance with that of the popular BayesA model applied to each population separately (no pooling) and to the joined data set (complete pooling). As an example, we analyzed a wide array of traits from the nested association mapping maize population. There we show that for small population sizes (<em>e.g.</em>, &lt;50), partial pooling increased prediction accuracy over no or complete pooling for populations represented in the training set. No pooling was superior; however, when populations were large. In another example data set of interconnected biparental maize populations either partial or complete pooling was superior, depending on the trait. A simulation showed that no pooling is superior when differences in genetic effects among populations are large and partial pooling when they are intermediate. With small differences, partial and complete pooling achieved equally high accuracy. For prediction of new populations, partial and complete pooling had very similar accuracy in all cases. We conclude that partial pooling with multilevel models can maximize the potential of pooling by making optimal use of information in pooled training sets.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/data-pooling" class="hw-term hw-article-keyword hw-article-keyword-data-pooling" rel="nofollow">data pooling</a></li><li class="kwd"><a href="/keyword/bayesian-whole-genome-regression" class="hw-term hw-article-keyword hw-article-keyword-bayesian-whole-genome-regression" rel="nofollow">Bayesian whole-genome regression</a></li><li class="kwd"><a href="/keyword/multilevel-models" class="hw-term hw-article-keyword hw-article-keyword-multilevel-models" rel="nofollow">multilevel models</a></li><li class="kwd"><a href="/keyword/multiple-populations" class="hw-term hw-article-keyword hw-article-keyword-multiple-populations" rel="nofollow">multiple populations</a></li></ul><p id="p-3">Genomic selection (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>) in animal and plant breeding rests on the accurate prediction of genomic breeding values (GEBVs). An important determinant of prediction accuracy is the size of the training set (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Daetwyler <em>et al.</em> 2010</a>). In animal breeding, assembling large training sets is relatively straight forward for large dairy breeds like Holstein Friesian, where genomic selection is applied most successfully to date (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hayes <em>et al.</em> 2009b</a>). For smaller dairy cattle breeds and in particular for beef cattle breeds, however, assembling sufficiently large training sets within each breed is often not possible (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Weber <em>et al.</em> 2012</a>). The creation of multipopulation training sets by pooling several breeds is therefore of great interest and subject of current research (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Los Campos and Sorensen 2014</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lund <em>et al.</em> 2014</a>).</p><p id="p-4">A similar situation exists in plant breeding, which is characterized by a high degree of structuring (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2014</a>). This structuring results from the importance of keeping distinct heterotic groups for maximum exploitation of heterosis (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Melchinger and Gumber 1998</a>), from the predominance of distinct biparental populations (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mikel and Dudley 2006</a>), and the need for specialized breeding programs targeting specific traits or environments (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Windhausen <em>et al.</em> 2012</a>). This requires that the phenotyping and genotyping resources available to a breeding program have to be allocated to multiple populations, which prevents the creation of sufficiently large training sets for each population. Several studies therefore investigated the merit of pooled training sets combining populations (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asoro <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lorenz <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> 2014</a>) or even heterotic groups (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Technow <em>et al.</em> 2013</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> 2014</a>).</p><p id="p-5">However, pooling training sets is complicated by the unique genetic characteristics of populations, that arise because of differences in linkage disequilibrium or allele frequency, (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Weber <em>et al.</em> 2012</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Windhausen <em>et al.</em> 2012</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Technow <em>et al.</em> 2014b</a>), quantitative trait locus (QTL) by background interaction (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blanc <em>et al.</em> 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Melchinger <em>et al.</em> 2008</a>), and presence of population specific QTL alleles (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Giraud <em>et al.</em> 2014</a>). This might be the reason why using pooled training sets failed to increase prediction accuracy in some applications in plant (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Desta and Ortiz 2014</a>) and animal breeding (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lund <em>et al.</em> 2014</a>).</p><p id="p-6">Therefore, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brøndum <em>et al.</em> (2012)</a> proposed to use separate training sets for each population but to derive genome position specific priors from estimation results in the other population. In this way, unique genome properties of each population could be accounted for while still using information from other populations. A similar, but perhaps more formal, approach is <em>partial pooling</em>, facilitated by Bayesian multilevel models (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gelman and Hill 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gelman and Pardoe 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gelman 2006a</a>). In multilevel models, specific marker effects are estimated for each population. However, the prior means of these specific marker effects, which might be interpreted as overall or unspecific marker effects, are estimated from data of all populations, simultaneously with the specific marker effects. Because the specific marker effects are shrunk toward the overall effects, the former are still informed by data from the other populations to a certain degree. Partial pooling thus strikes a middle ground between <em>no pooling</em> (specific marker effects estimated from data of the specific population only) and <em>complete pooling</em> (common marker effects estimated from pooled training sets).</p><p id="p-7">Our objectives were to (i) demonstrate the use of Bayesian multilevel whole-genome regression models for genomic prediction and (ii) investigate scenarios in which partial pooling might be superior over no or complete pooling of training sets. Our investigations were based on two publicly available maize breeding data sets and supported by a simulation study.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Multilevel whole-genome regression model</h3><p id="p-8">The model fitted to the data were as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-2.gif"/></noscript></span></span> was the observed phenotypic value of the <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-3.gif"/></noscript></span></span> individual from the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-4.gif"/></noscript></span></span> population and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-5.gif"/></noscript></span></span> its linear predictor. The phenotypic data <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-6.gif"/></noscript></span></span> was centered to mean zero and scaled to unit variance. The Normal density function, which was used as likelihood function, was denoted as <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-7.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-8.gif"/></noscript></span></span> denoting the residual variance component. The common intercept was <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-9.gif"/></noscript></span></span>. Finally, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-10.gif"/></noscript></span></span> denoted the additive effect of the <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-11.gif"/></noscript></span></span> biallelic single-nucleotide polymorphism (SNP) marker in population <em>j</em>. The genotype of individual <em>i</em> from population <em>j</em> at marker <em>k</em> was represented by <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-12.gif"/></noscript></span></span>, which was the number of reference alleles, centered by twice the reference allele frequency. Which of the alleles was chosen as reference allele depended on the data set and is described below. Effects <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-13.gif"/></noscript></span></span> were only estimated when the corresponding marker was polymorphic in population <em>j</em>. Otherwise it was set to 0 and treated as a constant.</p><p id="p-9">The hierarchical prior distribution setup will be explained next. A graphical display is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. The prior of <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-14.gif"/></noscript></span></span> was<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-15.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>u<sub>k</sub></em> was the overall effect of the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-16.gif"/></noscript></span></span> marker and variance parameter <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-17.gif"/></noscript></span></span> quantified the deviations of the specific effects <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-18.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-19.gif"/></noscript></span></span>. Note that all else equal, the shrinkage toward <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-20.gif"/></noscript></span></span> is the stronger the smaller <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-21.gif"/></noscript></span></span>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphic visualization of the multilevel model (A) and the conventional BayesA model (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794459045" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Graphic visualization of the multilevel model (A) and the conventional BayesA model (B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1603/F1.medium.gif" width="440" height="118"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/8/1603/F1.medium.gif" width="440" height="118"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466425" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Graphic visualization of the multilevel model (A) and the conventional BayesA model (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Both parameters were associated with prior distributions themselves and estimated from the data. For <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-22.gif"/></noscript></span></span> this was <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-23.gif"/></noscript></span></span>. Here, the variance parameter <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-24.gif"/></noscript></span></span> controls the amount of shrinkage toward 0. It was associated with a scaled inverse χ<sup>2</sup> prior with 4.001 degree of freedom and scale parameter <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-25.gif"/></noscript></span></span>. The prior for <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-26.gif"/></noscript></span></span> thus corresponded to the well-known “BayesA” prior (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>). The prior for the intercept <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-27.gif"/></noscript></span></span> was a Normal distribution with mean 0 and a very large variance.</p><p id="p-12">For the variance parameter <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-28.gif"/></noscript></span></span>, we specified<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-29.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>which is a Normal distribution prior on <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-30.gif"/></noscript></span></span> with mean parameter <em>m</em> and SD <em>d</em>, left truncated at zero. Note that the mean of the truncated distribution <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-31.gif"/></noscript></span></span>, which is a function of <em>m</em>, <em>d</em>, and the truncation points, can be interpreted as the “typical” deviation of the specific marker effects <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-32.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-33.gif"/></noscript></span></span>. Greater values of this mean indicate larger deviations and vice versa. This parameter might therefore be used to quantify population divergence. We chose a truncated Normal as prior distribution because it is straightforward to specify and interpret its hyperparameters and to potentially include prior knowledge. It is also our experience that a truncated Normal prior can be more robust and improve convergence compared to other potential choices such as Gamma distributions.</p><p id="p-13">A Uniform distribution prior <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-34.gif"/></noscript></span></span> was used for the hyperparameters <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-35.gif"/></noscript></span></span>, <em>m</em>, and <em>d</em>. For the residual variance <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-36.gif"/></noscript></span></span> we specified a Uniform distribution prior over the interval [0, 1] on <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-37.gif"/></noscript></span></span>, which agrees with recommendations for uninformative priors on variance components (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gelman, 2006b</a>). We used Uniform distributions because they allow us to convey prior ignorance while still bounding the parameters to a sensible value range. The latter feature can improve robustness and convergence. In contrast to other choices for uninformative priors, such as improper distributions, the implications of Uniform distributions also are readily apparent to researchers less familiar with Bayesian statistics. <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gelman (2006b)</a> state further reasons for why a proper Uniform distribution might be preferable over improper uniform distributions. However, we note that our choice of prior distributions differs from those commonly used for Bayesian whole genome regression, which are chosen mainly for their conjugacy and computational efficiency. Although we see several advantages in choosing Uniform and truncated Normal distribution priors, their lack of conjugacy is a drawback and can be associated with computational performance penalties.</p><p id="p-14">Samples from the posterior distribution were drawn with Gibbs sampling, implemented in the JAGS Gibbs sampling environment (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Plummer 2003</a>). The total number of samples was 1000, drawn from a single chain with burn in of 10,000 and thinning intervals of 500. These settings ensured convergence and an effective sample size (ESS) of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-38.gif"/></noscript></span></span> for all parameters (ESS of <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-39.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-40.gif"/></noscript></span></span> were typically <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-41.gif"/></noscript></span></span>).</p><p id="p-15">The ESS was calculated with the R (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Core Team 2013</a>) package CODA (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Plummer <em>et al.</em> 2006</a>), which was also used to monitor convergence using diagnostic plots.</p></div><div id="sec-3" class="subsection"><h3>Conventional whole-genome regression model</h3><p id="p-16">We used the popular Bayesian whole-genome regression method “BayesA” (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>), with the modifications of <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Yang and Tempelman (2012)</a> pertaining to the hyperparameter <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-42.gif"/></noscript></span></span> (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> for a graphical representation). The linear model was<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-43.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>which is principally the same as in (1), with the difference that the population index <em>j</em> was dropped. For no pooling, the model was applied to each population in turn, for complete pooling to the joint data set. For <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-44.gif"/></noscript></span></span> we used an improper scaled inverse χ<sup>2</sup> prior with −1 degrees of freedom and scale equal to zero. This is equivalent to an improper uniform prior density on <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-45.gif"/></noscript></span></span> (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gelman 2006b</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Yang and Tempelman 2012</a>), which is similar to the proper Uniform density that was used for the multilevel model but exploits conjugacy.</p><p id="p-17">The BayesA Gibbs Sampler was implemented as a C routine compatible with the R statistical software environment. Again we drew a total number of 1000 samples from a single chain with burn in of 10,000 and thinning of 500.</p></div><div id="sec-4" class="subsection"><h3>Estimation, prediction, and testing procedure</h3><p id="p-18">Let Π denote the set of <em>P</em> populations represented in the training set and the set of <em>N<sub>p</sub></em> training individuals from a population in Π as <em>Λ</em><em><sub>p</sub></em>, where <em>p</em> indexes the population in Π. A graphic representation is presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Further, let those individuals from a population in Π that are not in <em>Λ<sub>p</sub></em> be denoted as <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-46.gif"/></noscript></span></span> and the set of populations not in Π as <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-47.gif"/></noscript></span></span>. Populations in <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-48.gif"/></noscript></span></span> will be referred to as “new” populations. The training set thus comprised all individuals belonging to <em>Λ<sub>p</sub></em>, for <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-49.gif"/></noscript></span></span>. The test set used for calculating prediction accuracy, comprised individuals in <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-50.gif"/></noscript></span></span> from populations in Π and all individuals from populations in <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-51.gif"/></noscript></span></span>. The phenotypic observations of test individuals were masked in the estimation procedure. The separation of populations into Π and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-52.gif"/></noscript></span></span> and of individuals within a population into <em>Λ<sub>p</sub></em> and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-53.gif"/></noscript></span></span> was done at random.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphic visualization of the testing strategy for evaluating prediction accuracy. The training set comprises &#x39B;1 and &#x39B;2 from populations P1 and P2 (set &#x3A0;). The prediction accuracy of lines from populations represented in training set () was computed from  and , the prediction accuracy of lines from populations not represented in training set from lines in P3 and P4 (set )." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794459045" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Graphic visualization of the testing strategy for evaluating prediction accuracy. The training set comprises &#x39B;&lt;sub&gt;1&lt;/sub&gt; and &#x39B;&lt;sub&gt;2&lt;/sub&gt; from populations &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; and &lt;em&gt;P&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt; (set &#x3A0;). The prediction accuracy of lines from populations represented in training set (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) was computed from &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-55&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-56&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, the prediction accuracy of lines from populations not represented in training set from lines in &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt; and &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;4&lt;/sub&gt; (set &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-57&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1603/F2.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/8/1603/F2.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466609" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Graphic visualization of the testing strategy for evaluating prediction accuracy. The training set comprises Λ<sub>1</sub> and Λ<sub>2</sub> from populations <em>P</em><sub>1</sub> and <em>P<sub>2</sub></em> (set Π). The prediction accuracy of lines from populations represented in training set (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-54.gif"/></noscript></span></span>) was computed from <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-55.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-56.gif"/></noscript></span></span>, the prediction accuracy of lines from populations not represented in training set from lines in <em>P</em><sub>3</sub> and <em>P</em><sub>4</sub> (set <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F2/embed/mml-math-57.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Within each population, prediction accuracy was computed as the correlation between GEBVs and observed phenotypic values of individuals in the testing set. The within population prediction accuracies were subsequently averaged for populations in Π and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-58.gif"/></noscript></span></span>. These average within population prediction accuracies will henceforth be denoted as <em>r</em><sub>Π</sub> and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-59.gif"/></noscript></span></span>. Thus, <em>r</em><sub>Π</sub> and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-60.gif"/></noscript></span></span> correspond to the prediction accuracy for populations represented and not represented in the training set, respectively.</p><p id="p-21">When using partial pooling, GEBVs of individuals in <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-61.gif"/></noscript></span></span> were predicted using the posterior means of the marker effects estimated for the corresponding population (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-62.gif"/></noscript></span></span>). GEBVs of individuals from populations in <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-63.gif"/></noscript></span></span> were predicted using the posterior means of the overall (unspecific) marker effects <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-64.gif"/></noscript></span></span>.</p><p id="p-22">When using complete pooling, GEBVs of all individuals in the test set were predicted from the posterior means of marker effects <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-65.gif"/></noscript></span></span> estimated from the joint data set with model (4).</p><p id="p-23">Finally, when using no pooling, GEBVs of individuals in <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-66.gif"/></noscript></span></span> were predicted using the posterior means of the marker effects <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-67.gif"/></noscript></span></span> obtained after applying model (4) to the training data from the corresponding set <em>Λ<sub>p</sub></em>. The no pooling approach does not provide a direct way of predicting GEBVs of individuals from populations in <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-68.gif"/></noscript></span></span>. Thus, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-69.gif"/></noscript></span></span> was not evaluated for the no-pooling approach.</p></div><div id="sec-5" class="subsection"><h3>Application to nested association mapping (NAM) maize populations</h3><p id="p-24">The NAM data set was obtained from <a href="http://www.panzea.org">http://www.panzea.org</a>. It comprised 4699 recombinant inbred lines from 25 biparental crosses between a genetically diverse set of maize inbred lines and line B73 as common parent (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McMullen <em>et al.</em> 2009</a>). The average population size was 188 (range 126–196). The recombinant inbred lines were genotyped with 1106 polymorphic SNP markers covering the whole genome. The non-B73 allele was defined as the reference allele. We confirmed that all SNP were biallelic and thereby that the reference allele corresponded to the same nucleotide in all 25 populations. To facilitate computations, we used a thinned set of 285 markers, chosen in such a way that there was one marker per 5-cM interval, on average. A previous study showed that a density of one marker per 10-cM interval is sufficient for genomic prediction in the NAM population (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2012</a>). We analyzed the traits days to silking (DS), ear height (EH), ear length (EL), southern leaf blight resistance (SLB), near-infrared starch measurements (NS) and upper leaf angle (ULA), which were phenotyped in multienvironment field trials. The phenotypic records used for fitting the models were averages over the single environment phenotypes. The number of environments were 10, 11, 8, 3, 7, and 9 for DS, EH, EL, SLB, NS, and ULA, respectively. The traits chosen represent the major trait categories available: yield component (EL), agronomic (EH), disease resistance (SLB), flowering (DS), quality (NS), and morphology (ULA).</p><p id="p-25">To investigate the effect of total number of lines <em>N</em>, number of populations <em>P</em>, and number of lines per population <em>N<sub>p</sub></em> in the training set on prediction accuracy and the relative performance of the pooling approaches, the following combinations of <em>P</em> and <em>N<sub>p</sub></em> were considered: <em>P</em> = 5 and <em>N<sub>p</sub></em> = 50 and 100, <em>P</em> = 10 and <em>N<sub>p</sub></em> = 25, 50, and 100, <em>P</em> = 20 and <em>N<sub>p</sub></em> = 12.5, 25, and 50. For <em>P</em> = 20 and <em>N<sub>p</sub></em> = 12.5, we sampled 19 populations with 12 individuals and one with 22, which results in an average <em>N<sub>p</sub></em> of 12.5. The <em>P</em> and <em>N<sub>p</sub></em> combinations thus gave rise to <em>N</em> of either 250, 500, or 1000. For each combination of trait, <em>P</em> and <em>N<sub>p</sub></em>, 50 estimation-testing data sets were generated by repeating the sampling of Π and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-70.gif"/></noscript></span></span> as described previously. Throughout, the three pooling approaches were applied to the same data sets. The sampling variation between different data sets thus does not enter the comparisons among pooling approaches.</p><p id="p-26">To provide measures of the consistency of differences between pooling approaches under repeated sampling, we plotted the prediction accuracies observed for one pooling approach against those observed for the other in the same 50 data sets. In addition, we conducted paired <em>t</em>-tests to assess statistical significance of pairwise differences between pooling approaches.</p><p id="p-27">We conducted an analysis of variance (ANOVA) to assess the overall significance of influencing factors and of interactions between factors on prediction accuracies. For this we fitted a linear model with the main effects of pooling approach, trait, <em>P</em> and <em>N<sub>p</sub></em> and all possible interactions among them (<em>P</em> and <em>N<sub>p</sub></em> were fitted as numerical predictor variables) and the replication as blocking factor.</p><p id="p-28">To investigate the response to increased marker density, we repeated the analyses using 575 markers, which corresponds to a marker density of 2.5 cM<sup>−1</sup>. Because of the considerably increased central processing unit (CPU) time, particularly for partial pooling (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/019299SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS10.pdf">Figure S10</a>), this was only done for a subset of the traits (EH, EL, and SLB). To facilitate computations for partial pooling, it was also necessary to reduce the length of the thinning interval from 500 to 50 and the number of stored samples from 1000 to 500. The thinning interval length and the number of stored samples for no and complete pooling were not changed, however.</p><p id="p-29">For the purpose of gauging the CPU time requirements with increasing number of markers, we ran the multilevel Gibbs sampler for partial pooling as well as the no and complete pooling BayesA algorithm with numbers of markers from 100 to 1000 in steps of 100. This was done for trait SLB with <em>P</em> = 20 and <em>N<sub>p</sub></em> = 25. Because we were only interested in measuring computation time, the Gibbs samplers were run for only 1000 iterations. The whole process was repeated 50 times for each number of markers.</p></div><div id="sec-6" class="subsection"><h3>Application to interconnected biparental (IB) maize populations</h3><p id="p-30">This data set was obtained from the supplement of <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> (2013)</a>. It comprised 635 doubled haploid (DH) lines from five biparental populations with average size of 127 (range 43–204). The populations were derived from crosses between four European flint inbred lines. For all DH lines, 16,741 SNP markers polymorphic across populations were available. We replaced missing marker genotypes with twice the frequency of the reference allele, which was the allele with the lower frequency. When analyzing the data we used a thinned set of 285 markers. Because the data set did not include a map of the markers, the markers were chosen randomly.</p><p id="p-31">The DH lines were phenotyped in multienvironment field trials for Giberella ear rot (GER) severity, a fungal disease caused by <em>Fusarium graminearum</em>, deoxynivalenol (DON) content (major mycotoxin produced by the fungus), ear length (EL), kernel rows, and kernels per row (KpR). A more detailed description of this data set can be found in <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> (2013)</a> and <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Martin <em>et al.</em> (2012)</a>.</p><p id="p-32">As described previously, populations were randomly split into <em>Λ<sub>p</sub></em> and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-71.gif"/></noscript></span></span>. However, because there were only five populations in total, we did not exclude any populations from Π. Set <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-72.gif"/></noscript></span></span> was thus empty and we did not evaluate <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-73.gif"/></noscript></span></span>.</p><p id="p-33">The sets <em>Λ<sub>p</sub></em> comprised 25%, 50%, and 75% of the lines in each population, which corresponded to an average <em>N<sub>p</sub></em> of 31, 63, and 95, respectively. For each trait and percentage value of estimation individuals, 100 estimation-testing data sets generated, each time resampling the subset of 285 markers, too.</p></div><div id="sec-7" class="subsection"><h3>Application to simulated data set</h3><p id="p-34">We conducted a simulation study to specifically investigate the performance of the pooling approaches under increasing levels of differences in QTL effects among populations. The basis for the simulation were the marker genotypes of the lines in the NAM populations. To simulate genetic values, we first randomly chose 20 marker loci as QTL, which were subsequently removed from the set of observed markers. We drew additive overall effects <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-74.gif"/></noscript></span></span> from a standard normal distribution. Then population specific QTL effects <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-75.gif"/></noscript></span></span> were sampled from <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-76.gif"/></noscript></span></span>. The variance parameter <em>t</em><sup>2</sup> was chosen such that the relative SD (rSD), <em>i.e.</em>, <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-77.gif"/></noscript></span></span>, was equal to 2, 1, 0.5, 0.25, and 0.0. The greater rSD, the less similar the population specific QTL effects are. True genetic values were obtained by summing QTL effects <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-78.gif"/></noscript></span></span> according the QTL genotypes of each individual. Finally, phenotypic values were simulated by adding a normally distributed noise variable to the true genetic values. The variance of the noise variable was chosen such that the heritability across populations was equal to 0.70. The average within family heritability necessarily increased with decreasing rSD, and was 0.53, 0.58, 0.64, 0.68, and 0.70 at rSD 2, 1, 0.5, 0.25, and 0.0, respectively.</p><p id="p-35">Set Π comprised <em>P</em> = 10 populations and sets <em>Λ<sub>p</sub></em> had size <em>N<sub>p</sub></em> = 25. For each rSD value 50 training-testing data sets were generated. The QTL positions and effects were randomly generated anew for each data set. Also in this case we used a thinned set of 285 markers. Because the true genetic values were known, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-79.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-80.gif"/></noscript></span></span> were computed as the correlation between true genetic values and GEBVs.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>NAM maize populations</h3><p id="p-36">Trends typically held across traits. The results presented and discussed therefore apply to all traits, unless otherwise mentioned. We will also present results for 285 markers first and then contrast these with those obtained with 575 markers.</p><p id="p-37">Increasing <em>N<sub>p</sub></em>, while keeping <em>N</em> constant (<em>i.e.</em>, having fewer but larger populations in the training set) generally increased <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-81.gif"/></noscript></span></span> and decreased <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-82.gif"/></noscript></span></span> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS1.pdf">Table S1</a>). However, the increase in <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-83.gif"/></noscript></span></span> was much more pronounced than the decrease in <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-84.gif"/></noscript></span></span>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average prediction accuracy in the nested association mapping population using 285 markers. The number of populations represented in the training set is P and the number of individuals per population is Np. The three leftmost columns in each subplot show the accuracy for no, partial, and complete pooling for populations represented in the training set (). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (). The red bars over each column indicate the standard errors of the averages. The traits are: days to silking (DS), ear height (EH), ear length (EL), NIR starch measurements (NS), southern leaf blight resistance (SLB), and upper leaf angle (ULA)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794459045" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Average prediction accuracy in the nested association mapping population using 285 markers. The number of populations represented in the training set is &lt;em&gt;P&lt;/em&gt; and the number of individuals per population is &lt;em&gt;N&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt;. The three leftmost columns in each subplot show the accuracy for no, partial, and complete pooling for populations represented in the training set (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-85&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-86&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The red bars over each column indicate the standard errors of the averages. The traits are: days to silking (DS), ear height (EH), ear length (EL), NIR starch measurements (NS), southern leaf blight resistance (SLB), and upper leaf angle (ULA).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1603/F3.medium.gif" width="420" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/8/1603/F3.medium.gif" width="420" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466416" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Average prediction accuracy in the nested association mapping population using 285 markers. The number of populations represented in the training set is <em>P</em> and the number of individuals per population is <em>N<sub>p</sub></em>. The three leftmost columns in each subplot show the accuracy for no, partial, and complete pooling for populations represented in the training set (<span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F3/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F3/embed/mml-math-85.gif"/></noscript></span></span>). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F3/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F3/embed/mml-math-86.gif"/></noscript></span></span>). The red bars over each column indicate the standard errors of the averages. The traits are: days to silking (DS), ear height (EH), ear length (EL), NIR starch measurements (NS), southern leaf blight resistance (SLB), and upper leaf angle (ULA).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">When increasing <em>N<sub>p</sub></em> with constant <em>P</em> or when increasing <em>P</em> with constant <em>N<sub>p</sub></em> (<em>i.e.</em>, increasing <em>N</em>), both <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-87.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-88.gif"/></noscript></span></span> increased. However, while in the first case, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-89.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-90.gif"/></noscript></span></span> increased in similar magnitudes, the increase in <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-91.gif"/></noscript></span></span> was much smaller than the increase in <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-92.gif"/></noscript></span></span> in the second case, in particular when <em>N<sub>p</sub></em> was high. Per definition, the accuracy of no pooling is not expected to change as long as <em>N<sub>p</sub></em> remains constant.</p><p id="p-40">For low <em>P</em> and high <em>N<sub>p</sub></em>, <em>e.g.</em>, <em>P</em> = 5 and <em>N<sub>p</sub></em> = 100, no pooling achieved the greatest <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-93.gif"/></noscript></span></span> and complete pooling the lowest. For high <em>P</em> and low <em>N<sub>p</sub></em>, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-94.gif"/></noscript></span></span>20 and <em>N<sub>p</sub></em> = 25, partial pooling achieved the greatest <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-95.gif"/></noscript></span></span>. Here no pooling resulted in the lowest <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-96.gif"/></noscript></span></span>. The only exception to this was trait DS, where no pooling had a <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-97.gif"/></noscript></span></span> equal or higher to partial and complete pooling also for low <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-98.gif"/></noscript></span></span>. Practically relevant differences among the pooling approaches (<em>e.g.</em>, &gt;0.01) were statistically significant (<em>p</em> &lt; 0.05, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS1.pdf">Table S1</a>) and consistent in repeated sampling (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS6.pdf">Figure S6</a>).</p><p id="p-41">Partial and complete pooling achieved practically identical prediction accuracies <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-99.gif"/></noscript></span></span> for new populations. In general, <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-100.gif"/></noscript></span></span> of a particular pooling approach was considerably lower than the corresponding <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-101.gif"/></noscript></span></span>. The differences between <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-102.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-103.gif"/></noscript></span></span> tended to be larger for high <em>N<sub>p</sub></em>.</p><p id="p-42">In the ANOVA, all main effects and interactions were found to have effects on prediction accuracy that were significantly different than zero (<em>p</em> &lt; 0.05), which was expected given the large amount of residual degrees of freedom (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS2.pdf">Table S2</a>).</p><p id="p-43">Results when using 575 markers were similar to those obtained with 285 markers (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS8.pdf">Figure S8</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS9.pdf">Figure S9</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS3.pdf">Table S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS4.pdf">Table S4</a>) and we did not observe a consistent increase in prediction accuracy. Averaged over traits and <em>P</em> and <em>N<sub>p</sub></em> combinations, the difference between 285 and 575 markers were statistically not significant (<em>p</em> &gt; 0.05) for <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-104.gif"/></noscript></span></span> of no and complete pooling and for <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-105.gif"/></noscript></span></span> of partial and complete pooling. The only significant difference was observed for <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-106.gif"/></noscript></span></span> of partial pooling, which dropped by 0.01 points when increasing the number of markers to 575. It is possible that increased Monte-Carlo error because of the reduced chain length complicated these comparisons, particularly for partial pooling.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average prediction accuracy in the nested association mapping population using 575 markers. The number of populations represented in the training set is P and the number of individuals per population is Np. The three leftmost columns in each subplot show the accuracy for no, partial, and complete pooling for populations represented in the training set (). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (). The red bars over each column indicate the standard errors of the averages. The traits are: ear height (EH), ear length (EL) and southern leaf blight resistance (SLB)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794459045" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Average prediction accuracy in the nested association mapping population using 575 markers. The number of populations represented in the training set is &lt;em&gt;P&lt;/em&gt; and the number of individuals per population is &lt;em&gt;N&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt;. The three leftmost columns in each subplot show the accuracy for no, partial, and complete pooling for populations represented in the training set (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-107&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-108&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The red bars over each column indicate the standard errors of the averages. The traits are: ear height (EH), ear length (EL) and southern leaf blight resistance (SLB).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1603/F4.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/8/1603/F4.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466504" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Average prediction accuracy in the nested association mapping population using 575 markers. The number of populations represented in the training set is <em>P</em> and the number of individuals per population is <em>N<sub>p</sub></em>. The three leftmost columns in each subplot show the accuracy for no, partial, and complete pooling for populations represented in the training set (<span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F4/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F4/embed/mml-math-107.gif"/></noscript></span></span>). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (<span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F4/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F4/embed/mml-math-108.gif"/></noscript></span></span>). The red bars over each column indicate the standard errors of the averages. The traits are: ear height (EH), ear length (EL) and southern leaf blight resistance (SLB).</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">CPU time increased linearly with the number of markers for no and complete pooling, but exponentially for partial pooling (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS10.pdf">Figure S10</a>). Thus, the relative differences between partial pooling on the one hand and no and complete pooling on the other hand increased with increasing number of markers. At 100 markers, the average CPU times for generating 1000 samples were 54, 23, and 2 sec for no, partial, and complete pooling. At 300 markers the corresponding CPU times were 144, 159, and 7 sec and at 1000 markers 503, 1627, and 23 sec.</p></div><div id="sec-10" class="subsection"><h3>IB maize populations</h3><p id="p-46">The prediction accuracy <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-109.gif"/></noscript></span></span> increased with increasing <em>N<sub>p</sub></em>, for all traits and pooling approaches (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS5.pdf">Table S5</a>). Averaged over traits, the increase was largest for no pooling, where the accuracy increased from an average of 0.35 at <em>N<sub>p</sub></em> = 31 to 0.48 at <em>N<sub>p</sub></em> = 95. The accuracies for the partial and complete pooling approaches increased from 0.39 and 0.38, respectively, at <em>N<sub>p</sub></em> = 31 to 0.48 at <em>N<sub>p</sub></em> = 95.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average prediction accuracies in interconnected biparental maize populations for populations represented in the training set (). Np denotes the average number of individuals per population in the training set, the number of populations was 5. The red bars over each column indicate the standard errors of the averages. The traits are: ear length (EL), deoxynivalenol content (DON), Giberella ear rot severity (GER), kernel rows (KR), and kernels per row (KpR)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794459045" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Average prediction accuracies in interconnected biparental maize populations for populations represented in the training set (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-110&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). &lt;em&gt;N&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt; denotes the average number of individuals per population in the training set, the number of populations was 5. The red bars over each column indicate the standard errors of the averages. The traits are: ear length (EL), deoxynivalenol content (DON), Giberella ear rot severity (GER), kernel rows (KR), and kernels per row (KpR).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1603/F5.medium.gif" width="440" height="196"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/8/1603/F5.medium.gif" width="440" height="196"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466409" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-47" class="first-child">Average prediction accuracies in interconnected biparental maize populations for populations represented in the training set (<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F5/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F5/embed/mml-math-110.gif"/></noscript></span></span>). <em>N<sub>p</sub></em> denotes the average number of individuals per population in the training set, the number of populations was 5. The red bars over each column indicate the standard errors of the averages. The traits are: ear length (EL), deoxynivalenol content (DON), Giberella ear rot severity (GER), kernel rows (KR), and kernels per row (KpR).</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">At <em>N<sub>p</sub></em> = 31, partial pooling had the greatest <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-111.gif"/></noscript></span></span>for traits EL, KpR, complete pooling for traits DON and kernel row. For GER both had the same accuracy. The no pooling approach had the lowest <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-112.gif"/></noscript></span></span>, except for EL and KpR, where it had the same accuracy as complete pooling. For the greatest <em>N<sub>p</sub></em> of 95, the accuracy differences among the pooling approaches decreased. Partial pooling still had the greatest accuracy for EL and KpR and the same as complete pooling for DON and GER. While never better than partial pooling, no pooling had higher prediction accuracy than complete pooling for EL and KpR.</p><p id="p-49">Prediction accuracy differences between the pooling approaches were not only statistically significant (<em>p</em> &lt; 0.05, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS5.pdf">Table S5</a>) but also practically relevant (<em>e.g.</em>, &gt;0.01) and consistent in repeated sampling (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS11.pdf">Figure S11</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS12.pdf">Figure S12</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS13.pdf">Figure S13</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS14.pdf">Figure S14</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS15.pdf">Figure S15</a>). All main effects and interactions were found to have effects on prediction accuracy that were significantly different from zero (<em>p</em> &lt; 0.05) in the ANOVA (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS6.pdf">Table S6</a>).</p></div><div id="sec-11" class="subsection"><h3>Simulated maize populations</h3><p id="p-50">For all pooling approaches, <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-113.gif"/></noscript></span></span> increased with decreasing rSD (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS7.pdf">Table S7</a>). The increase for no pooling, however, was comparatively small and a result of the increasing within family heritability with decreasing rSD. The relative performance of the pooling approaches also depended on rSD. For the greatest rSD value considered, no pooling had the highest <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-114.gif"/></noscript></span></span>, for the intermediate rSD value of 1.0 partial pooling (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS16.pdf">Figure S16</a>). For the lower rSD values complete and partial pooling achieved similarly high <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-115.gif"/></noscript></span></span>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average prediction accuracy in simulated maize populations. The three leftmost columns in each subplot show the accuracy for no, partial and complete pooling for populations represented in the training set (). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (). The red bars over each column indicate the standard errors of the averages. rSD is the relative SD of simulated population specific quantitative trait locus (QTL) effects. The number of populations represented in the training set was 10 and the number of individuals per population 25. The number of markers used was 285 and the number of simulated QTL 20." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-794459045" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average prediction accuracy in simulated maize populations. The three leftmost columns in each subplot show the accuracy for no, partial and complete pooling for populations represented in the training set (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-116&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-117&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The red bars over each column indicate the standard errors of the averages. rSD is the relative SD of simulated population specific quantitative trait locus (QTL) effects. The number of populations represented in the training set was 10 and the number of individuals per population 25. The number of markers used was 285 and the number of simulated QTL 20.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1603/F6.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/8/1603/F6.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1603/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466586" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-51" class="first-child">Average prediction accuracy in simulated maize populations. The three leftmost columns in each subplot show the accuracy for no, partial and complete pooling for populations represented in the training set (<span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F6/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F6/embed/mml-math-116.gif"/></noscript></span></span>). The two rightmost columns in each subplot show the accuracy for partial and complete pooling for populations not represented in the training set (<span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F6/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/F6/embed/mml-math-117.gif"/></noscript></span></span>). The red bars over each column indicate the standard errors of the averages. rSD is the relative SD of simulated population specific quantitative trait locus (QTL) effects. The number of populations represented in the training set was 10 and the number of individuals per population 25. The number of markers used was 285 and the number of simulated QTL 20.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Also <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-118.gif"/></noscript></span></span> for both partial and complete pooling increased strongly with decreasing rSD and the differences to <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-119.gif"/></noscript></span></span> decreased. Partial and complete pooling achieved almost identical <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-120.gif"/></noscript></span></span>.</p><p id="p-53">All main and interaction effects of factors on prediction accuracy were found to be significantly different from zero (<em>p</em> &lt; 0.001) in the ANOVA (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS8.pdf">Table S8</a>) and prediction accuracy differences between methods were statistically significant (<em>p</em> &lt; 0.05, <a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/TableS7.pdf">Table S7</a>), practically relevant and consistent in repeated sampling (<a href="http://www.g3journal.org/content/suppl/2015/05/29/g3.115.019299.DC1/FigureS16.pdf">Figure S16</a>).</p><p id="p-54">The mean of the truncated Normal distribution prior <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-121.gif"/></noscript></span></span> for parameter <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-122.gif"/></noscript></span></span> increased with increasing rSD. Its average values were 0.0111, 0.0153, 0.0190, 0.0269, and 0.0296 for rSD of 0.0, 0.25, 0.5, 1.0 and 2.0, respectively. The regression coefficient of the regression of the mean of <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-123.gif"/></noscript></span></span> on rSD was significantly different from zero (<em>P</em> &lt; 0.001).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Comparison of pooling approaches</h3><p id="p-55">Partial pooling allows estimation of population specific marker effects while still facilitating the “borrowing” of information across populations. It is therefore a compromise between no pooling, which models unique characteristics of each population but ignores shared information, and complete pooling, in which the opposite is the case.</p><p id="p-56">When population sizes <em>N<sub>p</sub></em> are sufficiently large, borrowing information from other populations is not required for achieving high prediction accuracy of new individuals from the same population (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-124.gif"/></noscript></span></span>). Further enlarging training sets by pooling with other populations might then even be detrimental (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Habier <em>et al.</em> 2013</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> 2013</a>). This explains why no pooling was the most accurate approach when <em>N<sub>p</sub></em> was large (<em>e.g.</em>, ≥50), particularly in the NAM population, and why it profited most from increases in <em>N<sub>p</sub></em>. Therefore, pooling of training sets is most promising if <em>N<sub>p</sub></em> is small due to budget or other constraints. We indeed observed that pooling was more accurate than no pooling when <em>N<sub>p</sub></em> was small (<em>e.g.</em>, &lt;50). The superiority of either pooling approach over no pooling also increased with increasing <em>P</em>, because information from more populations was available, which is not used in no pooling. Thus, pooling is expected to be most advantageous when <em>P</em> is relatively high and <em>N<sub>p</sub></em> low. Whether partial or complete pooling is the better approach will then also depend on the similarity of the pooled populations. The greater the similarity, the relatively better complete pooling is expected to perform, because the ability to estimate population specific marker effects becomes less important. In this situation partial pooling might even be of disadvantage, because it requires estimation of many more effects which might lead to problems associated with nonidentifiability (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gelfand and Sahu 1999</a>). The parents of the IB populations are from the same breeding program (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> 2013</a>), whereas the noncommon parents of the NAM populations were chosen to be maximally diverse and comprise temperate, tropical and specialty (sweet and popcorn) maize germplasm (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">McMullen <em>et al.</em> 2009</a>). Accommodating for unique characteristics of the populations is therefore more important in NAM than in IB, which might explain why complete pooling was always inferior to partial pooling in the former but often equal or even superior in the latter and also why no pooling never achieved the greatest prediction accuracy in IB, even for large <em>N<sub>p</sub></em>.</p><p id="p-57">The relative performance of the pooling approaches was very stable across traits in the NAM data set, with the exception of DS. For this trait the no pooling approach was generally superior, even at high <em>P</em> and low <em>N<sub>p</sub></em>. <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Buckler <em>et al.</em> (2009)</a> found evidence for an allelic series at the QTL identified for DS in the NAM population. Thus, although the positions of the QTL are conserved across populations, their effects might have differed strongly. Possible reasons are presence of multiple alleles or QTL by genetic background interaction. In this situation, pooling of data are not expected to have an advantage over no pooling. This example also shows that decisions about whether to pool data or not have to be made on a by trait basis and should incorporate prior knowledge about genetic architecture, if available.</p><p id="p-58">The dependence of the relative performance of the pooling approaches on the similarity of populations was also reinforced by the results from our simulation study. There we also observed that the mean of <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-125.gif"/></noscript></span></span>, the prior distribution of <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-126.gif"/></noscript></span></span>, which quantifies the deviations of specific marker effects <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-127.gif"/></noscript></span></span> from the overall effect <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-128.gif"/></noscript></span></span>, increased with increasing simulated differences among population specific QTL effects. This was expected, but demonstrates that the data were informative for the high-level hyperparameters. Averaged over <em>P</em> and <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-129.gif"/></noscript></span></span>, this mean was largest for DS and ULA in NAM (results not shown). This might reflect the noted differences between population specific QTL effects for DS. Trait ULA, however, did not diverge from the pattern observed for the remainder of traits and there does not seem to be any strong indication of an allelic series as in DS (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tian <em>et al.</em> 2011</a>). There was also no obvious relation between the mean of <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-130.gif"/></noscript></span></span> and performance of the pooling approaches in IB (results not shown).</p><p id="p-59">We observed clear trends for the relative performance of the different pooling approaches with regard to changes in <em>P</em> and <em>N<sub>p</sub></em>. However, the results showed that factors such as trait and degree of similarity of genetic effects among populations do have an influence on the relative performance. Therefore, deciding <em>a priori</em> which pooling approach will be superior for a particular <em>P</em> and <em>N<sub>p</sub></em> combination in a given scenario remains challenging and would require substantial prior knowledge about trait architecture and the germplasm.</p><p id="p-60">Modeling unique characteristics of populations requires that these populations are represented in the training set. Prediction of individuals from new populations in <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-131.gif"/></noscript></span></span> therefore has to rely on the overall, unspecific marker effects <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-132.gif"/></noscript></span></span>, in both partial and complete pooling. It was thus expected that both achieved very similar prediction accuracies <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-133.gif"/></noscript></span></span> for new populations.</p><p id="p-61">Our results demonstrate that partial pooling is able to model unique characteristics of populations within the training set without compromising on the ability of prediction of individuals from new populations. Thus, our results are consistent with the conclusion of <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gelman (2006a)</a>, stating that the greatest potential of partial pooling with multilevel models is in predictive applications.</p><p id="p-62">We exemplified the use of multilevel models for partial pooling in the context of multiple populations, a scenario of high relevance for plant (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> 2014</a>) and animal (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lund <em>et al.</em> 2014</a>) breeding. However, the concept is readily applicable in a wide array of scenarios. Examples are pooling data across multiple top-cross testers or environments, as is of particular relevance in plant breeding (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2014</a>). Extending the models to more than two levels is straightforward, too, for example for pooling multiple populations from multiple heterotic groups or breeding programs.</p></div><div id="sec-14" class="subsection"><h3>Computational considerations</h3><p id="p-63">For sake of brevity, we will focus on comparing the computational requirements of partial and complete pooling. The CPU time for no pooling could be substantial, and at low number of markers it was even higher than for partial pooling. However, no pooling lends itself to “embarrassingly” parallel computation, because each population is analyzed independently. While parallel execution of analyses in no pooling would not improve total CPU time, it reduces wall time approximately by a factor of <em>P</em>. The wall time would then be comparable to complete pooling.</p><p id="p-64">The CPU time for obtaining a certain number of posterior samples was considerably larger for partial pooling than for complete pooling for all numbers of markers investigated. This is partly a consequence of the much greater complexity and dimensionality of multilevel models, in which <em>P</em> + 1 effects (specific effects for all population plus the overall effects) have to be estimated for each marker, together with the corresponding variance components. In contrast to no pooling, partial pooling with multilevel models is also not easily parallelized. Most of the prior distributions of hyperparameters used in our multilevel model were not conjugate. This required computationally more demanding sampling techniques such as Metropolis-Hastings within Gibbs. Thus, the choice of prior distributions might have also contributed to the increased computation time.</p><p id="p-65">The software implementation is responsible for the differences in computation time, too. The BayesA algorithm was implemented in the compiled computer language C, while the multilevel model was implemented in JAGS. JAGS, and similar software like BUGS (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Thomas <em>et al.</em> 2006</a>), are computing environments dedicated to Gibbs sampling and allow straightforward and flexible implementation of novel and complex models. This makes them an ideal choice for research. JAGS, however, is an interpreted language which means program execution is much slower compared to compiled languages. Because the CPU time increased exponentially with increasing number of markers, partial pooling with multilevel models would currently be computationally very intensive for high numbers of markers (<em>i.e.</em>, greater than 500).</p><p id="p-66">The populations we studied were all derived from biparental crosses. Previous studies showed that for these types of populations the gain in prediction accuracy is marginal once the number of markers reaches 200–300 (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Combs and Bernardo 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em> 2014</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2015</a>). Consequently, we did not observe a significant increase in prediction accuracy for any pooling approach when increasing the number of markers from 285 to 575 (<em>i.e.</em>, increasing marker density from 5 cM<sup>−1</sup> to 2.5 cM<sup>−1</sup>) in the NAM populations.</p><p id="p-67">There are three main reasons why estimating population specific marker effects might be beneficial: (i) inconsistent marker-QTL LD among populations; (ii) QTL by background interaction (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Blanc <em>et al.</em> 2006</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Melchinger <em>et al.</em> 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wang <em>et al.</em> 2014</a>); and (iii) the presence of genuinely different QTL alleles (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Giraud <em>et al.</em> 2014</a>). Of these, only the first is sensitive to increasing marker density. Capturing the genetic variability induced by the other two phenomena requires the use of genetic models that allow marker effects to vary across populations. This can explain why no pooling (when <em>N<sub>p</sub></em> is large) or partial pooling (when <em>N<sub>p</sub></em> is small) had an advantage over complete pooling also for biparental populations, in which marker density might not be the limiting factor.</p><p id="p-68">Biparental crosses are by far the most common source of recombinant germplasm in commercial plant breeding programs. A study by <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Mikel and Dudley (2006)</a>, for example, revealed, that 88% of commercial maize inbred lines developed for the US corn belt between 1980 and 2004 were derived from either F<sub>2</sub> populations (77%) or backcrosses (11%). The parents of these crosses were in most cases elite and often related inbred lines, with some of them being parent in a disproportionately large number of crosses (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Mikel and Dudley 2006</a>). The most prominent example of these is B73, the common parent in the NAM populations. Such key ancestor lines were also identified for the European maize germplasm (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Technow <em>et al.</em> 2014a</a>). Thus, genomic prediction in sets of biparental populations is of great relevance to plant breeding, and for these types of populations, partial pooling with multilevel models is computationally feasible and can increase prediction accuracy, as we showed.</p><p id="p-69">Nonetheless, there are scenarios where marker density is expected to be a limiting and more decisive factor, for example for pooling dairy cattle breeds (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009a</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Erbe <em>et al.</em> 2012</a>) and other types of random mating populations encountered in animal breeding. Application of partial pooling with multilevel models in these cases is more challenging and would require computationally more efficient model formulations and software implementations.</p></div><div id="sec-15" class="subsection"><h3>Alternative approaches to partial pooling</h3><p id="p-70">There are alternatives to multilevel models for partial pooling. <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Brøndum <em>et al.</em> (2012)</a> leveraged information across populations by using results obtained from one population to derive genome position specific priors for the analysis of another. For example, when there were two populations A and B, then A was analyzed first and the obtained result used as prior information when analyzing B. One disadvantage of their approach is that because analyses are done sequentially, information is not shared simultaneously among populations. In the example above, information from A is used for B but not vice versa. To use information from B for A, the analyses had to be repeated in reverse order. It is also not obvious how the approach of <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Brøndum <em>et al.</em> (2012)</a> can be generalized to more than two populations or to prediction of individuals from new populations. Another potential source of concern is that the priors derived from population A are too informative to allow substantial Bayesian learning, especially when population B is small (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gelfand and Sahu 1999</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola 2013</a>).</p><p id="p-71">Our multilevel model is similar to models that simultaneously fit main and interaction marker effects (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schulz-Streeck <em>et al.</em> 2012</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">De Los Campos and Sorensen 2014</a>). The main difference to our approach is that both effects are on the same hierarchical level, such that the genetic value of an individual is modeled as the sum of main and interaction effects. Nonetheless, both formulations could be viewed as introducing a marginal prior distribution of marker effects that has zero mean and compound symmetric covariance structure (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">De Los Campos and Sorensen 2014</a>). In multilevel models, however, the overall marker effect <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-134.gif"/></noscript></span></span> acts as the prior mean toward which population specific marker effects <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-135.gif"/></noscript></span></span> are shrunk. The magnitude of the shrinkage thereby depends on <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-136.gif"/></noscript></span></span> (the strength of the prior) and on <em>N<sub>p</sub></em> (the “strength” of the data). For very large <em>N<sub>p</sub></em>, the data are expected to diminish the influence of the prior and thereby the shrinkage toward <em>u<sub>j</sub></em>.</p><p id="p-72">Phenotypic observations from different populations could be treated as different traits and pooled data sets analyzed with multi-trait models (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">De Los Campos and Sorensen 2014</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Lund <em>et al.</em> 2014</a>). This facilitates simultaneous sharing of information across populations through covariances. Multi-trait extensions of the GBLUP method (<em>i.e.</em>, genomic best linear unbiased prediction), which make a strong assumption of uniform variances and covariances of marker effects across the genome (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Lund <em>et al.</em> 2014</a>), are straight-forward to implement and their performance was evaluated in a dairy cattle breeding context by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Karoui <em>et al.</em> (2012)</a> and <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Olson <em>et al.</em> (2012)</a>. Multitrait extensions of methods like BayesA or BayesB, which would allow for varying covariance structures across markers, are principally conceivable, too (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">De Los Campos and Sorensen 2014</a>; <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Lund <em>et al.</em> 2014</a>). However, estimating marker specific, unstructured covariance matrices can be expected to be computationally challenging. This is the case particularly when the number of populations is large, because of the increased dimensions of the matrices. In addition, with multitrait models, prediction of performance of individuals from populations not represented in the training set would not be possible directly.</p></div><div id="sec-16" class="subsection"><h3>Composition of training set</h3><p id="p-73">Increasing the number of individuals from a population in the training set (<em>N<sub>p</sub></em>) always increased prediction accuracy for untested individuals from the same population (<span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-137.gif"/></noscript></span></span>), regardless if the training set was further enlarged by individuals from other populations (partial and complete pooling) or not (no pooling).</p><p id="p-74">However, because plant breeding programs have to operate under budget constraints, optimum allocation of resources is of great importance for maximizing the potential of genomic selection (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lorenz 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Riedelsheimer and Melchinger 2013</a>). With a fixed budget for phenotying that is proportional to <em>N</em>, the number of populations <em>P</em> and the number of individuals per population <em>N<sub>p</sub></em> have to be optimized under the constraint that <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-138.gif"/></noscript></span></span>. Such an optimization could be accomplished using basic theory about response to selection (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Falconer and Mackay 1996</a>) and accounting for the different prediction accuracy for populations represented and not represented in the training set (<span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-139.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-140.gif"/></noscript></span></span>, respectively), as exemplified by <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Technow <em>et al.</em> (2013)</a>. A key point hereby is that <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-141.gif"/></noscript></span></span> will increase with increasing <em>N<sub>p</sub></em> but it will apply to fewer populations because of the decrease in <em>P</em>. This is exacerbated by the decrease in <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-142.gif"/></noscript></span></span> that we observed was associated with decreasing <em>P</em>. Thus, if the total number of populations is large, as is typically the case in plant breeding programs, having very low <em>P</em> is likely to be undesirable. In the context of plant breeding this and other studies, most recently <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Lehermeier <em>et al.</em> (2014)</a>, showed that pooling data across populations can at least partly compensate for low <em>N<sub>p</sub></em> if populations are related and there is evidence for the merit of pooling very divergent germplasm too (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Technow <em>et al.</em> 2013</a>). Using pooled training sets therefore has the potential to allow for high <em>P</em> without compromising too much on <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1603/embed/mml-math-143.gif"/></noscript></span></span>. We showed that partial pooling with multilevel models can further enhance this potential by making optimal use of the information in pooled training sets.</p></div></div><div class="section" id="sec-17"></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-75">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019299/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019299/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-76"><em>Communicating editor: G. A. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-04-21"><span class="received-label">Received </span>April 21, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-05-26"><span class="accepted-label">Accepted </span>May 26, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Technow and Totir</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.1" data-doi="10.1007/s00122-014-2305-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1375</span>–<span class="cit-lpage">1386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1375%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2305-z%26rft_id%253Dinfo%253Apmid%252F24723140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2305-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24723140&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.2" data-doi="10.3835/plantgenome2011.02.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.</span> <abbr class="cit-jnl-abbrev">Plant Gen.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen.%26rft.volume%253D4%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.02.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.02.0007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.3" data-doi="10.1007/s00122-006-0287-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Connected populations for detecting quantitative trait loci and testing for epistasis: an application in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBlanc%26rft.auinit1%253DG.%26rft.volume%253D113%26rft.issue%253D2%26rft.spage%253D206%26rft.epage%253D224%26rft.atitle%253DConnected%2Bpopulations%2Bfor%2Bdetecting%2Bquantitative%2Btrait%2Bloci%2Band%2Btesting%2Bfor%2Bepistasis%253A%2Ban%2Bapplication%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0287-1%26rft_id%253Dinfo%253Apmid%252F16791688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0287-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16791688&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238503200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.4" data-doi="10.1186/1471-2164-13-543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome position specific priors for genomic prediction.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBrondum%26rft.auinit1%253DR.%2BF.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D543%26rft.epage%253D543%26rft.atitle%253DGenome%2Bposition%2Bspecific%2Bpriors%2Bfor%2Bgenomic%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-543%26rft_id%253Dinfo%253Apmid%252F23050763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23050763&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.5" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of genomewide selection for different traits with constant population size, heritability, and number of markers.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.7" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.8" data-doi="10.1111/jbg.12091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On the genomic analysis of data from structured populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D131%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12091%26rft_id%253Dinfo%253Apmid%252F24838115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24838115&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.9" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.10" data-doi="10.3168/jds.2011-5019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goswami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4114</span>–<span class="cit-lpage">4129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DErbe%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D4114%26rft.epage%253D4129%26rft.atitle%253DImproving%2Baccuracy%2Bof%2Bgenomic%2Bpredictions%2Bwithin%2Band%2Bbetween%2Bdairy%2Bcattle%2Bbreeds%2Bwith%2Bimputed%2Bhigh-density%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5019%26rft_id%253Dinfo%253Apmid%252F22720968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720968&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305363100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1603.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chapter response to selection</span>, pp. <span class="cit-fpage">184</span>–<span class="cit-lpage">193</span> in <span class="cit-source">Introduction to Quantitative Genetics</span>. <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">Addison Wesley Longman Limited</span>, <span class="cit-publ-loc">Harlow</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.12" data-doi="10.2307/2669699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelfand</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahu</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Identifiability, improper priors and gibbs sampling for generalized linear models.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D94%26rft.spage%253D247%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2669699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2669699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000079457500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.13" data-doi="10.1198/004017005000000661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">Multilevel (hierarchical) modeling: what it can and cannot do.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">432</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D48%26rft.spage%253D432%26rft_id%253Dinfo%253Adoi%252F10.1198%252F004017005000000661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/004017005000000661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000240053400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.14" data-doi="10.1214/06-BA117A"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Prior distributions for variance parameters in hierarchical models.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D1%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1214%252F06-BA117A%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/06-BA117A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000207447000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1603.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Data Analysis Using Regression and Multilevel/Hierarchical Models</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.16" data-doi="10.1198/004017005000000517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardoe</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bayesian measures of explained variance and pooling in multilevel (hierarchical) models.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D48%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1198%252F004017005000000517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/004017005000000517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237651000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.17" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.18" data-doi="10.1534/genetics.114.169367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage disequilibrium with linkage analysis of multiline crosses reveals different multiallelic QTL for hybrid performance in the flint and dent heterotic groups of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169367%26rft_id%253Dinfo%253Apmid%252F25271305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.19" data-doi="10.1007/s00122-011-1702-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishore</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genome-wide selection efficiency in maize nested association mapping populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D261%26rft.epage%253D275%26rft.atitle%253DEvaluation%2Bof%2Bgenome-wide%2Bselection%2Befficiency%2Bin%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1702-9%26rft_id%253Dinfo%253Apmid%252F21938474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1702-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21938474&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.20" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic-BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.21" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.22" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Invited review: genomic selction in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.23" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Gen</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGen%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.24" data-doi="10.2135/cropsci2013.03.0195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1476</span>–<span class="cit-lpage">1488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.25" data-doi="10.1186/1297-9686-44-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karoui</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carabaño</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Díaz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D44%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-39%26rft_id%253Dinfo%253Apmid%252F23216664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23216664&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.26" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krmer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (Zea mays L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1603.27" data-doi="10.1534/g3.112.004911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DLorenz%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D481%26rft.epage%253D491%26rft.atitle%253DResource%2BAllocation%2Bfor%2BMaximizing%2BPrediction%2BAccuracy%2Band%2BGenetic%2BGain%2Bof%2BGenomic%2BSelection%2Bin%2BPlant%2BBreeding%253A%2BA%2BSimulation%2BExperiment%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004911%26rft_id%253Dinfo%253Apmid%252F23450123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMy80ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.28" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.29" data-doi="10.1016/j.livsci.2014.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Invited review: genomic evaluation of cattle in a multi-breed context.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.05.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.30" data-doi="10.1111/j.1439-0523.2011.01908.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miedaner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inheritance of resistance to Gibberella ear rot and deoxynivalenol contamination in five flint maize crosses.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D131%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2011.01908.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2011.01908.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.31" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeda</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1603.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gumber</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Overview of heterosis and heterotic groups in agronomic crops</span>, pp. <span class="cit-fpage">29</span>–<span class="cit-lpage">44</span> in <span class="cit-source">Concepts and Breeding of Heterosis in Crop Plants</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Staub</span>  <span class="cit-name-given-names">J. E.</span></span>. <span class="cit-publ-name">CSSA</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.33" data-doi="10.1534/genetics.107.084871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic expectations of quantitative trait loci main and interaction effects obtained with the triple testcross design and their relevance for the analysis of heterosis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2265</span>–<span class="cit-lpage">2274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMelchinger%26rft.auinit1%253DA.%2BE.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2265%26rft.epage%253D2274%26rft.atitle%253DGenetic%2BExpectations%2Bof%2BQuantitative%2BTrait%2BLoci%2BMain%2Band%2BInteraction%2BEffects%2BObtained%2BWith%2Bthe%2BTriple%2BTestcross%2BDesign%2Band%2BTheir%2BRelevance%2Bfor%2Bthe%2BAnalysis%2Bof%2BHeterosis%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084871%26rft_id%253Dinfo%253Apmid%252F18430948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.35" data-doi="10.2135/cropsci2005.10-0371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mikel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolution of North American dent corn from public to proprietary germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMikel%26rft.auinit1%253DM.%2BA.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D1193%26rft.epage%253D1205%26rft.atitle%253DEvolution%2Bof%2BNorth%2BAmerican%2BDent%2BCorn%2Bfrom%2BPublic%2Bto%2BProprietary%2BGermplasm%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.10-0371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.10-0371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000237375900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.36" data-doi="10.3168/jds.2011-5006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tooker</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multibreed genomic evaluations using purebred Holsteins, Jerseys, and Brown Swiss.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">5378</span>–<span class="cit-lpage">5383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DOlson%26rft.auinit1%253DK.%2BM.%26rft.volume%253D95%26rft.issue%253D9%26rft.spage%253D5378%26rft.epage%253D5383%26rft.atitle%253DMultibreed%2Bgenomic%2Bevaluations%2Busing%2Bpurebred%2BHolsteins%252C%2BJerseys%252C%2Band%2BBrown%2BSwiss.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5006%26rft_id%253Dinfo%253Apmid%252F22916944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22916944&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1603.37"><div class="cit-metadata unstructured">Plummer, M., 2003 JAGS: a program for analysis of Bayesian graphical models using Gibbs sampling. Available at: <a href="http://www.r-project.org/nosvn/conferences/DSC-2003/Drafts/Plummer.pdf">http://www.r-project.org/nosvn/conferences/DSC-2003/Drafts/Plummer.pdf</a>. Accessed June 5, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vines</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">CODA: convergence diagnosis and output analysis for MCMC.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1603.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.40" data-doi="10.1534/genetics.113.150227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic predictability of interconnected biparental maize populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150227%26rft_id%253Dinfo%253Apmid%252F23535384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.41" data-doi="10.1007/s00122-013-2175-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Optimizing the allocation of resources for genomic selection in one breeding cycle.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2835</span>–<span class="cit-lpage">2848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2835%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2175-9%26rft_id%253Dinfo%253Apmid%252F23982591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2175-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23982591&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.42" data-doi="10.2135/cropsci2012.03.0160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karaman</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection using multiple populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">2453</span>–<span class="cit-lpage">2461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D2453%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.03.0160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.03.0160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1603.43" data-doi="10.1534/g3.112.004630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bürger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of northern corn leaf blight resistance in maize with combined or separated training sets for heterotic groups.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D197%26rft.epage%253D203%26rft.atitle%253DGenomic%2BPrediction%2Bof%2BNorthern%2BCorn%2BLeaf%2BBlight%2BResistance%2Bin%2BMaize%2Bwith%2BCombined%2Bor%2BSeparated%2BTraining%2BSets%2Bfor%2BHeterotic%2BGroups%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004630%26rft_id%253Dinfo%253Apmid%252F23390596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMi8xOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.44" data-doi="10.1007/s00122-014-2396-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification of key ancestors of modern germplasm in a breeding program of maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">2545</span>–<span class="cit-lpage">2553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D2545%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2396-6%26rft_id%253Dinfo%253Apmid%252F25208647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2396-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25208647&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.45" data-doi="10.1534/genetics.114.165860"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165860%26rft_id%253Dinfo%253Apmid%252F24850820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEzNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Hara</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ligges</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturtz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Making BUGS open.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.47" data-doi="10.1038/ng.746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTian%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D162%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bleaf%2Barchitecture%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.746%26rft_id%253Dinfo%253Apmid%252F21217756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217756&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.48" data-doi="10.1016/j.cj.2014.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic background effects on QTL and QTL × environment interaction for yield and its component traits as revealed by reciprocal introgression lines in rice.</span> <abbr class="cit-jnl-abbrev">Crop J.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BJ.%26rft.volume%253D2%26rft.spage%253D345%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cj.2014.06.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cj.2014.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.49" data-doi="10.2527/jas.2011-4586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thallman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keele</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snelling</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic breeding values in multibreed beef cattle populations derived from deregressed breeding values and phenotypes.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">4177</span>–<span class="cit-lpage">4190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2011-4586%26rft_id%253Dinfo%253Apmid%252F22767091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2011-4586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22767091&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1603.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.51" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTYwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1603.52" data-doi="10.1038/hdy.2014.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Rodriguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.99%26rft_id%253Dinfo%253Apmid%252F25407079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25407079&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1603.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/8/1603.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1593" title="Shared Genomic Regions Between Derivatives of a Large Segregating Population of Maize Identified Using Bulked Segregant Analysis Sequencing and Traditional Linkage Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1613" title="Nested Levels of Adaptive Divergence: The Genetic Basis of Craniofacial Divergence and Ecological Sexual Dimorphism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/8"><span>Volume 5 Issue 8, August 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/8.cover-source.jpg?itok=wjPUfMS9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1603" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ewta6A7UQB-FWlj953qDOFMmK4hEZ0uCxF8CvTo-sPE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/8/1603" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/8/1603" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/8/1603" class="active" data-icon-position="" data-hide-link-title="0">Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460011" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-98ixWXdA3F2-vWxNI7SZSjwnNdEvs4YjhsHXpwdDgs8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1603.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/8/1603" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qralMXNwjNUggrzQUb6HRf53TiJAEQuFUbWvq0p3cJw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/8/1603" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1603.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460011" id="node460011--2" data-pisa="ggg;5/8/1603" data-pisa-master="ggg;g3.115.019299" data-apath="/ggg/5/8/1603.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frank Technow</span> and <span class="highwire-citation-author" data-delta="1">L. Radu Totir</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1603-1612;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019299 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460011--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>DuPont Pioneer, Johnston, Iowa 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Frank.Technow@pioneer.com" class="" data-icon-position="" data-hide-link-title="0">Frank.Technow@pioneer.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">L. Radu Totir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>DuPont Pioneer, Johnston, Iowa 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BRadu%2BTotir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Totir%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BRadu%2BTotir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/8/1603" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/8/1603&amp;title=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/8/1603&amp;title=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/8/1603&amp;title=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/8/1603&amp;text=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/8/1603&amp;title=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/8/1603&amp;t=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/8/1603&amp;title=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/8/1603&amp;title=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460011/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460011" id="node460011--3" data-pisa="ggg;5/8/1603" data-pisa-master="ggg;g3.115.019299" data-apath="/ggg/5/8/1603.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Bayesian Multilevel Whole Genome Regression Models for Partial Pooling of Training Sets in Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Frank Technow</span> and <span class="highwire-citation-author" data-delta="1">L. Radu Totir</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1603-1612;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019299 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460011--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>DuPont Pioneer, Johnston, Iowa 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Frank.Technow@pioneer.com" class="" data-icon-position="" data-hide-link-title="0">Frank.Technow@pioneer.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">L. Radu Totir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>DuPont Pioneer, Johnston, Iowa 50131</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BRadu%2BTotir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Totir%20LR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BRadu%2BTotir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1603&amp;rft.epage=1612&amp;rft.atitle=Using%20Bayesian%20Multilevel%20Whole%20Genome%20Regression%20Models%20for%20Partial%20Pooling%20of%20Training%20Sets%20in%20Genomic%20Prediction&amp;rft.volume=5&amp;rft.issue=8&amp;rft.date=2015-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Technow%2C+Frank&amp;rft.au=Totir%2C+L.+Radu"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460011/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460011/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460011/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460011/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460011/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460011/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460011/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460011/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460011/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460011/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460011/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460011/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/8/1603" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/8/1603.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/8/1603.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/8/1603.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/8/1603.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
