<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/8/1695" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/8/1695.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/8/1695.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/8/1695.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/8/1695.atom" />
<meta name="HW.pisa" content="ggg;5/8/1695" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel" />
<meta name="DC.Identifier" content="10.1534/g3.115.019554" />
<meta name="DC.Date" content="2015-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Houle and Márquez. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We calculated the linkage disequilibrium between all pairs of variants in the Drosophila Genome Reference Panel with minor allele count ≥5. We used r2 ≥ 0.5 as the cutoff for a highly correlated SNP. We make available the list of all highly correlated SNPs for use in association studies. Seventy-six percent of variant SNPs are highly correlated with at least one other SNP, and the mean number of highly correlated SNPs per variant over the whole genome is 83.9. Disequilibrium between distant SNPs is also common when minor allele frequency (MAF) is low: 37% of SNPs with MAF &lt; 0.1 are highly correlated with SNPs more than 100 kb distant. Although SNPs within regions with polymorphic inversions are highly correlated with somewhat larger numbers of SNPs, and these correlated SNPs are on average farther away, the probability that a SNP in such regions is highly correlated with at least one other SNP is very similar to SNPs outside inversions. Previous karyotyping of the DGRP lines has been inconsistent, and we used LD and genotype to investigate these discrepancies. When previous studies agreed on inversion karyotype, our analysis was almost perfectly concordant with those assignments. In discordant cases, and for inversion heterozygotes, our results suggest errors in two previous analyses or discordance between genotype and karyotype. Heterozygosities of chromosome arms are, in many cases, surprisingly highly correlated, suggesting strong epsistatic selection during the inbreeding and maintenance of the DGRP lines." />
<meta name="DC.Contributor" content="David Houle" />
<meta name="DC.Contributor" content="Eladio J. Márquez" />
<meta name="article:published_time" content="2015-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We calculated the linkage disequilibrium between all pairs of variants in the &lt;i&gt;Drosophila&lt;/i&gt; Genome Reference Panel with minor allele count ≥5. We used &lt;i&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/i&gt; ≥ 0.5 as the cutoff for a highly correlated SNP. We make available the list of all highly correlated SNPs for use in association studies. Seventy-six percent of variant SNPs are highly correlated with at least one other SNP, and the mean number of highly correlated SNPs per variant over the whole genome is 83.9. Disequilibrium between distant SNPs is also common when minor allele frequency (MAF) is low: 37% of SNPs with MAF &amp;lt; 0.1 are highly correlated with SNPs more than 100 kb distant. Although SNPs within regions with polymorphic inversions are highly correlated with somewhat larger numbers of SNPs, and these correlated SNPs are on average farther away, the probability that a SNP in such regions is highly correlated with at least one other SNP is very similar to SNPs outside inversions. Previous karyotyping of the DGRP lines has been inconsistent, and we used LD and genotype to investigate these discrepancies. When previous studies agreed on inversion karyotype, our analysis was almost perfectly concordant with those assignments. In discordant cases, and for inversion heterozygotes, our results suggest errors in two previous analyses or discordance between genotype and karyotype. Heterozygosities of chromosome arms are, in many cases, surprisingly highly correlated, suggesting strong epsistatic selection during the inbreeding and maintenance of the DGRP lines.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/08/01" />
<meta name="citation_mjid" content="ggg;5/8/1695" />
<meta name="citation_id" content="5/8/1695" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/8/1695" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/8/1695.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/8/1695.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/8/1695.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019554" />
<meta name="citation_pmid" content="26068573" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1695" />
<meta name="citation_lastpage" content="1701" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="David Houle" />
<meta name="citation_author_institution" content="Department of Biological Science, Florida State University, Tallahassee, Florida 32308" />
<meta name="citation_author_email" content="dhoule@bio.fsu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8095-3156" />
<meta name="citation_author" content="Eladio J. Márquez" />
<meta name="citation_author_institution" content="Department of Biological Science, Florida State University, Tallahassee, Florida 32308" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5548-5795" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. B. Corbett-Detig;citation_author=D. L. Hartl;citation_title=Population genomics of inversion polymorphisms in Drosophila melanogaster.;citation_pages=e1003056;citation_volume=8;citation_year=2012;citation_pmid=23284285;citation_doi=10.1371/journal.pgen.1003056" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. B. Corbett-Detig;citation_author=C. Cardeno;citation_author=C. H. Langley;citation_title=Sequence-based detection and breakpoint assembly of polymorphic inversions.;citation_volume=192;citation_year=2012;citation_pmid=22673805;citation_doi=10.1534/genetics.112.141622" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. B. Corbett-Detig;citation_author=J. Zhou;citation_author=A. G. Clark;citation_author=D. L. Hartl;citation_author=J. F. Ayroles;citation_title=Genetic incompatibilities are widespread within species.;citation_pages=135-137;citation_volume=504;citation_year=2013;citation_pmid=24196712;citation_doi=10.1038/nature12678" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. M. Cridland;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_author=K. R. Thornton;citation_title=Abundance and distribution of transposable elements in two Drosophila QTL mapping resources.;citation_volume=30;citation_year=2013;citation_pmid=23883524;citation_doi=10.1093/molbev/mst129" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Duchen;citation_author=D. Živković;citation_author=S. Hutter;citation_author=W. Stephan;citation_author=S. Laurent;citation_title=Demographic inference reveals African and European admixture in the North American Drosophila melanogaster population.;citation_volume=193;citation_year=2013;citation_pmid=23150605;citation_doi=10.1534/genetics.112.145912" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Huang;citation_author=A. Massouras;citation_author=Y. Inoue;citation_author=J. Peiffer;citation_author=M. Rámia;citation_title=Natural variation in genome architecture among 205 Drosophila melanogaster genetic reference panel lines.;citation_volume=24;citation_year=2014;citation_pmid=24714809;citation_doi=10.1101/gr.171546.113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=K. Stevens;citation_author=C. Cardeno;citation_author=Y. C. G. Lee;citation_author=D. R. Schrider;citation_title=Genomic variation in natural populations of Drosophila melanogaster.;citation_volume=192;citation_year=2012;citation_pmid=22673804;citation_doi=10.1534/genetics.112.142018" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_doi=10.1101/014837" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Pool;citation_author=R. B. Corbett-Detig;citation_author=R. P. Sugino;citation_author=K. A. Stevens;citation_author=C. M. Cardeno;citation_title=Population genomics of sub-Saharan Drosophila melanogaster: African diversity and non-African admixture.;citation_pages=e1003080;citation_volume=8;citation_year=2012;citation_pmid=23284287;citation_doi=10.1371/journal.pgen.1003080" />
<meta name="citation_reference" content="citation_title=The SAS System for Windows, Release 9.3;citation_year=2011" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="twitter:description" content="We calculated the linkage disequilibrium between all pairs of variants in the Drosophila Genome Reference Panel with minor allele count ≥5. We used r2 ≥ 0.5 as the cutoff for a highly correlated SNP. We make available the list of all highly correlated SNPs for use in association studies. Seventy-six percent of variant SNPs are highly correlated with at least one other SNP, and the mean number of highly correlated SNPs per variant over the whole genome is 83.9. Disequilibrium between distant SNPs is also common when minor allele frequency (MAF) is low: 37% of SNPs with MAF &lt; 0.1 are highly correlated with SNPs more than 100 kb distant. Although SNPs within regions with polymorphic inversions are highly correlated with somewhat larger numbers of SNPs, and these correlated SNPs are on average farther away, the probability that a SNP in such regions is highly correlated with at least one other SNP is very similar to SNPs outside inversions. Previous karyotyping of the DGRP lines has been inconsistent, and we used LD and genotype to investigate these discrepancies. When previous studies agreed on inversion karyotype, our analysis was almost perfectly concordant with those assignments. In discordant cases, and for inversion heterozygotes, our results suggest errors in two previous analyses or discordance between genotype and karyotype. Heterozygosities of chromosome arms are, in many cases, surprisingly highly correlated, suggesting strong epsistatic selection during the inbreeding and maintenance of the DGRP lines." />
<meta name="og-title" property="og:title" content="Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/8/1695" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We calculated the linkage disequilibrium between all pairs of variants in the Drosophila Genome Reference Panel with minor allele count ≥5. We used r2 ≥ 0.5 as the cutoff for a highly correlated SNP. We make available the list of all highly correlated SNPs for use in association studies. Seventy-six percent of variant SNPs are highly correlated with at least one other SNP, and the mean number of highly correlated SNPs per variant over the whole genome is 83.9. Disequilibrium between distant SNPs is also common when minor allele frequency (MAF) is low: 37% of SNPs with MAF &lt; 0.1 are highly correlated with SNPs more than 100 kb distant. Although SNPs within regions with polymorphic inversions are highly correlated with somewhat larger numbers of SNPs, and these correlated SNPs are on average farther away, the probability that a SNP in such regions is highly correlated with at least one other SNP is very similar to SNPs outside inversions. Previous karyotyping of the DGRP lines has been inconsistent, and we used LD and genotype to investigate these discrepancies. When previous studies agreed on inversion karyotype, our analysis was almost perfectly concordant with those assignments. In discordant cases, and for inversion heterozygotes, our results suggest errors in two previous analyses or discordance between genotype and karyotype. Heterozygosities of chromosome arms are, in many cases, surprisingly highly correlated, suggesting strong epsistatic selection during the inbreeding and maintenance of the DGRP lines." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
    <title>Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZF8OyBjESfv5-khpK190LzuTp0wFsUrj2Aay-dliKlw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460004","apath":"\/ggg\/5\/8\/1695.atom","pisa":"ggg;5\/8\/1695","ac":{"\/ggg\/5\/8\/1695.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/8\/1695.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1695"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1695"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/8\/1695":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FNF9nqtOZqErWNdveoS9I7auEvv9S1WNnyJ980flcMzgGz9aQfvYI\\\\\\\/Lv3CaBYvEtchh8ReIdKjNLxRg2NP5Scq1ti\\\\\\\/\\\\\\\/t5wioW7fyFQJy5H+gsh+oEsGEuWd6PyK+zH\\\\\\\/JH3J4uiSfcKhYu631f4U2U6oj06xaNCHgXTHdEIIy7UgYp1KdQ7Jj3eXzSTNmmss86E7baKd6Iy38Mgsf0\\\\\\\/BeB5tP5PPcQWjbBedkLOYFw95rteqpIOQ4oLdIYZCfmL8J9cy1ypz1V7Y76bKjcS0C6uylWlOc4KGdA8xfApipT2iiMYynnJr8z\\\\\\\/TBnFXI5zj6I218oysyOkSNyL3by2H6q4blcdeMp3YrFj05pBjq+fyoBDtUldAeaq8Zmf1aqjWGTFZbFkqDONm5VuCfPuSCJSG7S0c7yZfX1tyAGNWaCxcHTDL\\\\\\\/vT8SOqkKXrewe8k7hb2Y5xCysjW+cF04tnQGh7MiqfJ2FlcWeGQeCXJYaeOBp3AzKSJDubHPn4+xJJJk6+x0uvW+LQeb3bukwa21ExKM+zVcI68Mvv7n1TDaMIk+2pWzTZ3ltm\\\\\\\/S2xerDMr23q\\\\\\\/ODqV7W0hbeKxCfNTlqrOOW6yFFVRnn9JADOVE3v\\\\\\\/UmehLz\\\\\\\/kwjkcQ+46GEGX720m4ez2tvGfDAsIKBHxBUhFhT6QKMQdS2J5MxMF2kK\\\\\\\/g\\\\\\\/YdwVkL\\\\\\\/2pxOFWjADNTMgrrqMA7YRV8WMU32o2Sd1sgpE8DWAU2N4CrNXh07xqR\\\\\\\/X8pM+dpae8fSnWlfnkdKncJonbIU\\\\\\\/pbuDP4V+N+VHmiL8zQQVQGlzqHQ3Oi5hpzHf+kV2XEo9MKl5kpeTzdK2HbhyNF8gQ9DAbWQIWAFatlAQAwMDM0MOnUm9Le+GPFr5XcKmXFRhbUDUDd64M4t9DffyzT7R\\\\\\\/aDjfLi49mCWoeti2+DQwjTMFhqqEg\\\\\\\/KW5DxRXv2ABcEreViDiNpK1dRCHJ+8VFKMM1nFIVm6Z1s11eG9jGalXt4ooY8VOumFUgt0UWZ35lt+Xpjyjma\\\\\\\/YYp6Z8vjmhcU09ZJ4a8Gi3EtvnmMzQ5VCVKaQC8eKIG3VvIrM5rkLcIex0YMowhTbp65FT2Ty2CjcHHMGhJ8GW5iMyL3Bah5N+Ojmktavrqjofn8QUq7aV+0iCUq6GcFm5RdByQ2fIjHW7lCduVmMPZwrRd68MZXBuhDpEYEshqm8JUDgFTAVDT8ht48d1+nMuvQowBUN2iidfspl3GKCi9lghctusa5Knspu5xY9cUwkHN0Urx3AqXM2ABpHGXAv7\\\u0022,\\\u0022iv\\\u0022:\\\u0022Pz3XvUNV6NipUopmeuMklg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a6799197cb3b872839be1bb57bc18cd9dd5819e6021f6781d2b141e68793d0bf\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022V3VJObIDl1hKxn\\\\\\\/XLw4zN0l8fr1p1x+\\\\\\\/tvYqKkLylB3acNJ+2mrXh260IIOwH9U45qy4CeuIqSrfis2hO7O5wU\\\\\\\/lfrIZDL0aKX8DXqm9HRt48pNXCDI72+5ZPe2mFrF50BsJYUxKiuDscxH0RreGcezZe\\\\\\\/uiGDjTAbwQ54NYL1z9J\\\\\\\/TFBo1lnHLFDFViN51rTm8FNeqWJxAT1yqivyW5T0+8hi87GtumDXoYTabm5B4buQVg7kSD0rAV9ufUL2k2KJhocaGGxPZ1Sy\\\\\\\/hE2bwfGtlYwCWnszN7hiCCmK4WfYdUFhWaak5emxXVzPL2qTDOTMxE8DKAn5EmHAYEgRCGvljLwZoY3J6g9CCIjxLUoUfpv+RQ81rY7FFkPVCv+4bKQc\\\\\\\/mw8A\\\\\\\/oP3KQWnlA3mtJQ+a9rKu6872GaTxjOG5gUxMipBXX9CcWRv34ezRbUEU7KOx51CXTKSDSXyv+4QT7nSMHzdtRWSM79vT6K7UhJo+2vSiQuEmaH7NiWvhBBuFtgZWgWKRwTnQNxcJiAoyEDp4EMz+RQpG+sagQb80Qsk+nfNDZSYximDPltf73C2UbgF4efQjPpO\\\\\\\/QcwaJlcF7GSNyMDWkFZk779t+dEjz\\\\\\\/43HthbuMPH+\\\\\\\/38rwJ2blVfNhacKCdMlnIDBkiPcQgaQpJl4g0BQoR6Zt9THQcjBpIws7rUMIE\\\\\\\/erYApUxxq5OLk3clg5LdutWCaJM7teDMUMyaRwQhnuhO+dpZg734\\\\\\\/rzDdPq+fMNWA6Tb3W3c2NLkuiT4\\\\\\\/GP7FjoOPAUPTqDMREggDc+JqkY8uQhfnvYbraOTu4yXoq4EoaxHbS4CbHhBWEDnvJWBN2SztmiiIWkjJLyA21ATrHaEXXJ+QFrkMRkXDteQFxTL6A27A++c1fZoXFG+9LvIjT5dfLaBa\\\\\\\/ZlV1TPKgAX8sgecAU5EubL9foe8XM74UAHjsyuHq3S\\\\\\\/Swljxd9HSHqne9bwroe7xsUpE2iAJB1imgZtsuaBZ4W8BJ\\\\\\\/9yY+Kd+GKpsUW6de7uGciKFolekIyQfe534tVe3BIvqZh6uiTUBovZwxoceQ6GzZIQk1Hz5xAqU\\\\\\\/YepU333noJnUgRiuLr+KIvpQVzSshZbbxKkYbmtMtGluSxmGkqi+3N9iT58k3l1o4PvpwMSUx9yNkKXMCD0qac2IPfhIe9el+orMr8BqBG8nxg+V\\\\\\\/vU5mpaLhoSXXjTfThqdFMd9rl1PzrIYzAmjTnxADofoXIFOyqc6KdGqZgFCOzs5E6wEtVXJj3+yf9ikPUb8hYe1WKLvHcoidMpXoAMKqtbwVPAZahMehRmJU3WaMZwouu0mx2R92lahqSQvt4nQTd93FQhYqVzmcy5QXg3Pdlev3LV000xC8cDMZUd0ad3ip5\\\\\\\/xW7T4nbDv6k7WWmDX17mKfgnQzC+M0RtvoUMB6afB6JtSF9kUCmfuZwsBYNqC4+Fu8gUpn1nxmREFoygKO3rks+9DEKWhyiPzkIxpUOZdhL+oP7IPsJVeBsZtFj2UNWSz82U50NoTZ8w0wohpoa2q6LzKwaV6Ted+NmD5JrUeqZ9Gp\\\\\\\/W8RJbY+BdFmzwU2S5chtBCW2obDF7MUFm6Dl3tZQDBZWHmcpbmHSMCMODq98dMM\\\\\\\/gTgaErIgmQ\\\\\\\/oNASDATLAWfiAA2O8PUO86eIQXW7dI17yZNkUlHBVxjHjnfPFahjjo8HSUnH+OQz7NsSfZE4950cLI+giea0zHlKg3vuOIUWj1qxN6jDNK0x2r0bvJkL5todSEKwPEL3gdMYH8Pl2c0ikl1++gkGfgLgLHT1pNUV87E\\\\\\\/Xw+v3wnTLSf515Lv\\\\\\\/YOmswxI\\\\\\\/CA5r1N3n1ohur0TAg3LyJyGRT71akbI7NlpdtQraK0tJO12BmrZz\\\\\\\/f6YCKaHVNaRcd2kuW3A12+\\\\\\\/gjk8qWODqvjnEbKLjZDTH\\\\\\\/lnHLQMdGFHyv8dygmQmOE2lcJWj1RfE6z2sJHgO6BFGFFwc0h\\\\\\\/eXNX9RehXJw0qrBVheHq9fmmwg6AOmC1WYFGuee\\\\\\\/b3YrzdZ0tXKJUEuB6mv7e0GOZqWRQrrSDY6N2lpScl4WVXK8K\\\\\\\/j6e1kRTNd6sg5pknugqMNofsUXhJXqZEYWvZuoJr2AdLED6+ol5qWO88hGVBXs6be2uMXsgDnsrIPijQo2JxsNVypDyHzxbCWQUYDaXEilcQBKs5qjp4eeq\\\\\\\/ND0ZMMQkXiSO384pm1a+ZF0n7hzwqNTvUe4KyReKI3KH0XD2xqDvJdpr6iH\\\\\\\/MWvN1JOnYfpLNAFa1EFttCwJB80wpCUDtSXcRPkw9P9jaQsv6GXeRSR9NX2B1UYGHHv5Spq4SFO56S\\\\\\\/jcp3XgnVsW1KLsV\\\\\\\/IXTOkxbPyA5cAJZ62XkBN1LZvox+bgPABsruPTXoRoTfjG3838K10jZHhVdIjDJ3BQPUoHcA1HgyUOc73gusZ8VpxnIf8ML6tbZVXsh1kOmIVrOYsxIC\\\\\\\/que+lm\\\\\\\/rPCJu3ek+bo+wQcsLOMFfk2M7kp9uXUYLP\\\\\\\/ruGCHdnIFkqLMIv6hxCk8nIPspGS1FdLWdWNwly+2hlxIyAI2JGHTi9fBTy8alYrcIgSD+fdWrLP0R+mJgk4Y664Tq4vBcnND0yJxvtt0Xn0\\\\\\\/RDPVAlvmkPL13qMaSQRlSmhfxdN6llGY5bg2VLOUBw\\\\\\\/QPNtROv0qj0iL5kahWKvLBLVFbDRz7aKKg0ja27ueVAxnuqKqhbVDRq8sVjEojgG8KvQ1QAtHNYsRtdYrssImnAg7\\\\\\\/UzFUo7KnGMqU6U4rB2y5njTVnxxmeT+LYh1Wnm8z883xcYQHvOtwF7wiGOcyz9GovzRJO58+9WomtboJlI0izE+GGndcO\\\\\\\/sV9MaSYbefDhXSId5BowJzmKByoX0haDA4IpqsI3otfuE4die2vQ9PXpXcupFvb7SbGvBcvYvDvY5zHPgD+xg1XQT4JiR4vZNHHBGa4zdJXbQaIEmHfV0vi6JWdkbdPuRb9vkoig6BlsiM5nkZjSxMg6fDHv\\\\\\\/mt8x1JmyOPBKRv3GufxdxF6GRHYv9w6dbx2t+ln4h6GTFOUDxxhZu6eMUSKPTFOOsAF7tmmYvl5W2\\\\\\\/U5h98pBcbSRW9jK8sTTdl436oLl80po14AhyJj\\\\\\\/cPBU4eEvrxGEzS1XFqgDTFtUaB6+WePdX8xUxiiWeH+pGxfqdKIL8itbwPiwrAenaubZ4FajFhmQGG3GmmH0KGkderc9gG63RI00yZr5EsG9LTKvjjXDpFBggJRlXmURjq18jTrnW2SjOxMs6HtfpZq1LHzrSq65nImjjJebEi79fVzEazGBR6oms4mUAhNxe\\\\\\\/rFZjboA96pEeYk\\\\\\\/CrH1tMLaX2XFZMwWvVpiIUQ41qVjRTYANUQfO1o4joMQu22QbgHFxucyw8vqx\\\\\\\/Hg8AIEM+1PVqo5fnLavBe8HTZBOMxAFE8tZ4IMVYYYb\\\\\\\/SreGNIiQ8EKhgsmDn+shTWbITcuB8+Qnb3UQnDBcXcaLqPmj0OHNUqM\\\\\\\/0Dvm3Typmiwlxqre09h8qFlMDeD+K\\\\\\\/vAZmHtYDxlfkTCshxpEZee7F38pgAGK4O2z33Fe8+6EL52fxYj18wlddVxRp9tpVblftn9yRkOQwkzty7+69w7FmPojrNlN0YRK1kJKqckmLnpE66tewFkVWT4A+gEq9Eyz5Wu\\\\\\\/j+18j6if3AyydWU5kVNlY2Y3Tt3HaUw9WCzI3zwhvXhLSTXWq1FqXcGW7cCRn6xToZ7M5EuKFFq5EAWcJU0rMOs\\\\\\\/+7wjcQkpIL67kSNtJLYg4xrGdnG0GLk3wWLlyRQuqeeJ+hjNJ1DM6V1Yf006tpzTcNnhURMEaLa1ECCxJPF6g3I6pVbugLKSedgrA\\\\\\\/foMzNG630x3e7vBhGXIu+KO9i3rJEw8tztWos3RVVmody\\\\\\\/NGrrxgEhlFYE1h4c1UBO19ps2dL0Ls2lcidqh5bZbVJUYRzVHhGvU4EWCArR93c8al+aoYofZot1yTcLrRsuiOJT5y0umbfyKGrMmBx852y00JVUHAwb4AnOIZ88yPbx4PRpLFdqhEsfe802A+X3DuKlSjVgLjJG1J0WsUX7kCpYaW5NPYhQy8NHprYESQlZKOwbgUJP5JM2GvNZH7Y2Pd0ZP8XKQpTjTUGDOZ+vJvTCKoqaQ7n3Qb1T3TKasRTymeDA5Z3pbIPCnhw7vlfj4GC\\\\\\\/SVWr49OrLVA6smuzqEzfUd246ElmiV7ijczHfm7NgHWnPWAkql6Rv9ulPR2lRjpNhWDVc8pqVFA7+qIdacp8HrUWGf18zBbevxyl4HK331NkqNO+S7tHbGel80V8NiR1qQF28UIO3yaQXECK4VXrVPhYVwz8ECUxv0coaa+D\\\\\\\/Ef5CSSfojmxKBSEM5qoi7+1OAKDJRLAU7HU3f9VT6vRfeYZD+IwiPw\\\\\\\/IKZFRF7Yq6foHFFvIJ80CmfVeOOW6ebkljmWYFlwMBI9xK+Wr+ozgEXssSE\\\\\\\/EHTjEJ3QY0Wl\\\\\\\/mCXZlKeY886793pG5hlevpaRC1Ejmybzl4Xg+jRLyeBKAJf7VRuI\\\\\\\/IzCUzfNurNPThRZERRVPNjU5+p2eEJwGZNhJxHcNf6ogrl9hbRYUAAv+1sbKAH4J4aQM5cVl7gcY446e0dnxYel0RNXnsOh6iooDlGje4V0vqJUilbM2ySXL+rK86Lj0qOwOBvcwsHvDX5g2gnC0bKcZ77EqNjFdG0nAWA0WR69Zik+KajqP4Dl7nUHDx\\\\\\\/AxuOxo7a4jSbpOwhcd+7\\\\\\\/exPNuhI6uWncXaoTb6y+ZDcohcWLnhWGBVStGbrEaBxUQC4DSLBEf\\\\\\\/3AJ3cDcreERfJn7sA+Us50RS4L3GxtBhJlZttBPO72eg7q6792l02H348d3KiBAUeMcZo7f2+HsiMZD5mTH+YHtOEuBcpVA7VSqO+O1r0FK6xM3kA3wEHMHNS0MYvHDQHnhbXm+VblkD1rboSNzFpI5NBgZYp8z19a6fTM8EhU4To95FytvZ66U8VWR6yeN4MXzjdka5A77XDKN4k7M8mIG5RybuSgZ21cd\\\\\\\/seCSQ1FWWFJqzSTTMeDZLlsQOeAJmqTess3HtaEQeaiMSGci2TP+OgbYPqjo4LIvfhsaq8kBg6Wm+8VGd71afvJUfFheeB+xeqpnzKy6Yn63THftHQpfv7nk1MX0wAGW3XSt91UNjSS+3G98q9SSzEmSwyte2\\\\\\\/WcnzwG09EjIlQApFoE2fgsWxTBTxoRsKNXhlyp+ZTqmkSZriL24yV7p6hBgkMuxOn63Z5JgTEbv7YGZ0HkBRcssyjBwb7Fap65qqBdcB1FhGpMAl5VukUTg1MK72U9zGzREmfUD7CiWJ5yiXljDB1elWP0B1GevdZpNBVWgD2i345oJbr4Pw0y3fEC06tiDh7fY2eYBLcqNCxRsDWsUtjwl0OFJq+xwTyTAfOz9qZvujyNyZuFVvx4opR3O0C5EueGq9TUNGK830uV8R6Q\\\\\\\/bNro26PIb1V+q\\\\\\\/HuObG227QAvtNIpiCcrYnE7AIjpLM3xSmVlzA5sdM\\\\\\\/pSvZAHEI2uL0gdk+zoj9+4L527IBNUEr2TNQZ\\\\\\\/ffQRSyi+LFEF6zS1boY4dIvNmB1kX3YtZLjNz+KfnpKGR1CKcRsfB9fbKB2qxJUiBiOC6LEivfOvWMoNXK+FY0wKTHrZLEl2TdgNNaWWwI4RKh3\\\\\\\/9duog4qjSJ6JjY6x9J6pb9fa5Ui3W9U7l5zG0FdLU7PScXRtwK2xK73uBdNAGaplkWRHRdVKBuvwpbcRTqqgahXr95rsoG9rB0dKZpgopA\\\\\\\/ut0n2PiI3sETAzAbdsnxepnb9zCKpUvgodMlxbMtaRN0JAXz7gL7ivSQY\\\\\\\/ibywPjC+8Yi+zuJoWcsReCsUubAkn8wvmuxh4R\\\\\\\/pcByu7V7AMc8M9zsVfVFfU\\\\\\\/0NVpy6IONAlZe3HWvuH2SY11HNKwkNO1MXiJbV3Ulk23+q1ByqRD\\\\\\\/xkor6CMGJhtDQ90WZOkxm5dQdChKIjix\\\\\\\/oIPoKINJzvsYwstavY0PoYspXihOyIR2ZJrp7WjD1BR2vuN2wjzLsvzFhSRwRsNBP\\\\\\\/Zh4\\\\\\\/FivVvAvLd5bZsipL+5I\\\\\\\/4hFuZ7DtPbVU8u5O\\\\\\\/HePEgtM2YkbkG2JiRq+PMpOx7nfDem59yqwq5dkf4xGea7uOw5yEVtS6J52aador6PhwTsawxpyk+vyrsXN84uxKrd36zyXe10FBHDx0+GdO76JO52W9x+lu8C4xnX5etbAAV2hVUA4RtDEdPeOpekQQ7OTKFtl7VFURtneLV6b1M4N4gp93bO1Azlvre3LsTSYiCbHd5eSXWo7udhYZGnw9vKHSsf2ZJMLW05xDFpzbN+oi9JX4kOw9JgkWdcF2g9CFFmiAa2QTv2mlDDjC1BBBEVLIvnuXt+K5V3J+ONeHhfpAG2JaqLUGdAgcm\\\\\\\/9xgMkJ2tiDLZjg4C82VyvTGH7Fzvdq7TM54vsD2nVasepmX74P\\\\\\\/6oOnVOAQ\\\\\\\/\\\\\\\/WvcRCnAP2ASK0vimm+gxGXnqenwupLgdNsaa6kv\\\\\\\/1xkRZI2bdLxB7O95JVjIBP\\\\\\\/wR5A9H\\\\\\\/fSykYdRhyhyz87OFnM07NrwOgihhAfl8hc7w4jeJ\\\\\\\/0B99oyChcZ+xk1W8s\\\\\\\/LSHVlUIS8WSI2\\\\\\\/Nm6BP1Rg2SR7iPx97KlL2m2gedLj86OyRLEAjvqrY47ZK170SfWkbWzU4hmeVLKPMeztzJY\\\\\\\/g+m7DdstnHrNdoUwAgqCOLWvo41x+EdFZReTSUbMqX0VPMilpXPauLYkWeBdzq2N4hDQoaDpw\\\\\\\/5ZBx2x2JFhd2Tb3T8znPTpPpeCHdfYfKuhwCs\\\\\\\/uMph6QFbGoxwgnHCI7dR1TfJdaiMxJ71xaPvQkIOsJ3IppDOadpHJUIPrbXUv4Eid3mYYxCfzQF2xTA5Rjcnp2WyKMmPhvxccUdinOSWsr7jI5eiFWz3aHxpuMfV9CJSjFpX3RpovkVJ1IPXMYYyemguhoRCDysPsT\\\\\\\/lPUejgry7Sy3SzT6HUh5n+DZAglwpqiM2E97LehKOEHhdcro1Uia8+ysXRMTbC46mfbM0wIuiCmy2JjXJS8C1BVA586HEsK\\\\\\\/AK6o3wtcBGgmbnuoIqqQLCKihkfbOr9Opw39rWyfkeEtzETEC\\\\\\\/N0UnePV1qGGR6roIYC1f\\\\\\\/df1UgTOCEuggmtgMI\\\\\\\/R\\\\\\\/pG6OeAJK8zhw7eBWMeIQWbhbQGYc0Z2XoEwouw2EFtTKEQGpuQ5T0ZhNNZo8zwL7dzIP9fqDiXIK9SM2Suya67QuNHuK1hCzCNUgE5z\\\\\\\/SY5ZtvFqfVU6I+BZcdQuPZibgeIKrcldLighRN0zQoXZtoRatNw6FFvSjzgrSj33yXqyZpVbLF6kY+8\\\\\\\/tRLlTrFBYsJz4vk8YrwpTO18cGsBegeSNet55chD+aNYqb\\\\\\\/JXnnq7kOwTlAFgdCgRWhnMf8SN1DeVs7PNT9khxjYv1a6M1jIMzdXQV\\\\\\\/qdE9MThNG\\\\\\\/Su\\\\\\\/hM5hdh9hOjWj6x9HjmpWpLaBh2biewTZQZuHW0w6KzlL16NFyvd5IXhCGArbEF943iDTZ77eiv7q3Jyvh7VPBxGPWfVbVz1QEw8svcIR+J+z82JHZ8dRuRFb7PkCaPWHdgTlssYqgZZMa+HwqXSForZiH2jP4V4cY5XMCoF2AZMcw23xmYVwG1w9K9vw1Od1lcF7Dd1\\\\\\\/zJBccZGMBBWEO2iXcqqxRuNAMBnqBogeo8v\\\\\\\/fLwATRwNU9MKP5lMVSll5l5\\\\\\\/pFUrGGstHvdFX7NfQ0glsAMQbg2b6e8ucFUIz3W0iAis\\\\\\\/NZxTC+IY5pdh3Z3BXHrqcU8fwRW2BP2vBhQ6iXlSWYvu1uLgMC4kHHLmaj+8ziFV3bpWR5b1yGmDxWqExDVBA6xl3ole1+ildWJPHVpBgWn4Xm4ZelfU4kjOSdBK84cmJXsKCllq3sY4m9KXQO07Agvx9\\\\\\\/DbioA+txttN44lZFs+rf3pR+PJSEwHsKEw92\\\\\\\/AiKhrkbrQsA\\\\\\\/vr6BA76Cz1UvmTsBFLpJhrLBUEECjYh\\\\\\\/BrLzSptwiPPIHn+F\\\\\\\/KEUl5IMhFQUktTGgUTSEBjOH6fWIAy6S1TvwmFUx04KQGO48mCCxbMM4wPr6wuSK2xpokUuRPOY5c1T6eUMxwSwRNfCbq0icD7Xj3a3Yqe+pbADLfcOTM0RVGqVA60hj2h79BWhYNMGkRf8r5TFcAtD4VIPN5p6BI7ATicKFV6zN8Z9qO7fd8TSzdYgbIh\\\\\\\/Je5Ynyh0aB23EnAgvHiB+GFvZL4OfUJmzV6zHi\\\\\\\/KBZJ7qPhIZanEL496Lfiay1Hc5LzzN1XeIVEp6LUsTn5CQwVJGAl+ksPO+J2kaNDdaX0rfSat3ttr6FypxvCc0ZP1zrYqXyE82Jwp8T1vH\\\\\\\/o6uIqEg2lO\\\\\\\/h6ki\\\\\\\/w5Us5IvT1yfuHdvr2Pr2dbMylxLg6YwAaoLrm40noLxEdNOT6KR4db4vn7LIdzqoZOJz3B5pHQsCozh7A8Xn5mnrXzHc4ydh9svVQ9VtTez7UtU+M4u6HKeJCg4QgO8\\\\\\\/aHr\\\\\\\/IecC2nVeqd9inbrSdj+TTa3p+V1wKuO5BBJce2qzqnXrVET7orYxIL6sYCcRDGdaq8TXfP5xRvpVd6giOmsquh3Pmdsai\\\\\\\/jcBrQRygpw8TqGTkxuScUPT5bSH8IZEoJuFoF63pZcA\\\\\\\/aMiJ\\\\\\\/RI1eR+DOWlmSqLfhQnagGEIbmIoPCZGzJ9Lv6Nert8B4WBnPL52zKqzN+VC4g2Gg7Ap1jIVmK0D\\\\\\\/tg8P5h2Jz70pML8HwT0vC4JrXluaFVRuqLc42FcOGq6xZGwq+8LtaOQMuWWUt6PFeBtCh08Sd9DMIW2gp2r5YkOR\\\\\\\/KUxjHqlLIPx9wkSq2Ej73u\\\\\\\/tRhebbimvEpDJeGS4JDzVTQec\\\\\\\/YqdMK2abJEkFRHOSkL7f4u767TB3UOY61leLLmt6t3cJFKnQuoPakcYWXg1Qwhe54YxGbSqUbKqZPcQb1KemM4q58KQvRF9ZBfBavX2UVODaMG0scM\\\\\\\/F8l0B4if5bRbNeFYU+aieQL3uzLY3HjtqjUSeMJuiSXzN1fTokZUySRWsRWX848dbypVSPbdQXQsodDYzX89dGIQzzEiDJ8O9JfRUPaqBXuUI+tPNP90xHOs2pjiPOyazkHpaNUmbBS9RV5M5Gz\\\\\\\/toBruakMr9nzB6hkhWY8P7Exzjuha4LnzY0cTb78neVCL+GlR7aJ3iP0sl28HjtA9l\\\\\\\/h1Gyw3l4GjJQ9pxjOWy\\\\\\\/JycsrMHjb+jWKrBD1ajP4UdqIZ6kIPWi9oBsr0GmuZol5wLsb5tgJ1pJ9+j5TAuEV82BfsqonXRNMa1NGGvKfJCkiAjYBsgQK0HTGT0EOgR5pIitFeJUJ6oVacW\\\\\\\/1+PPh29qmWPP\\\\\\\/bVss8HRtkjpxkWTK6vLAx84li4hxO1lwzFFYqxj1knrWZH4aMPCfYSStwYhSmlINkUbriKO3mJwHahvmvZ0l5mi\\\\\\\/MJT2c6c2cEkqig8pO6F8Jc8hnAzSr5++5P4K0R0R5tzZ\\\\\\\/QrRQA7kEZ+ngHB4\\\\\\\/Emx0VzfFzBtefRy3jJqgTYIbKYZqnDlyDRj3FZ\\\\\\\/VjNsA3vp+Nrpx4Xao2ZfRmwITOZDpRu8Nl1E5h1bfTyXahPM+6N\\\\\\\/HZ4bIhQTttFiUFXQBeCNyLk\\\\\\\/Iz8ViiVoOkyWBdgL9Zt0nwFskBE5heDPYwGrjCl+QMJVmKn50WcHVPIBBgylh8Mu8sY5efwpFQA7gdTujVgxA\\\\\\\/a1nQlbE4SRD0XT3JsCmgjmmcFZMcoDwSgHQAljOUsxMle8ir9IFMGXgextvvejjwXOW0iqcOYYUbM7BezeeY5tgIc0vMc45Rxnjatx+CVvHPV+5n1JjrPgUCAThTs\\\\\\\/B\\\\\\\/v\\\\\\\/bXJgTMqs253wQp7EoC86tWMWba4iP9FlZ81+9CFyHwfXUehP5uCjE4i6RK1tBBOnFQrv23zUkMfLo3yBrStve9DPmdL7PWbwjnus6FmeLqRt0AGR7VtCX9\\\\\\\/Qaf\\\\\\\/Or1rcVfU9PScATR\\\\\\\/lngjWhf6Ioy+pozEkph53P7zlJEmLWWOB5leX5LbaJ9v4sGHb7dYHFnZwrP86CCWyULysrMlV8BYfFgaxUjxcihDMgclMhoY9Yyc7agyiW\\\\\\\/\\\\\\\/co32aipn91hGT77pda9XyB1dFGjzQrCPlNcJzsZyZLCWSCaMPhejLkSAt\\\\\\\/gztdzlhO4Sv5G86edYtPwD\\\\\\\/hzpvVlTcivZi9fxtochgzpCbiGq2pbW032C8h99ZIbXFJcIMqnGXtPKQPHMWLMr2DRU58rMj3B189ty6xZQ60uE4bzrFXkWtW4dTYgRmrfJ3dqGyGpkZNgAIvmlGU0BNniVj26Fom\\\\\\\/YdMAAlUJoRkNZpLNet7JspIkd6a0fmdT3\\\\\\\/qS1PvO+RyKxgjA4AOiN2fuQt73blCi8CUq9Q2pqveGKY8xYyCtqL1XWD6hYRG+9v8xQQG8Y+j0xLqLe305L4\\\\\\\/kv0dRGDJYxRwEiUWYKkwFeoPlSWptSOQXOG2cgObzR7GkMaZxydNoTFnn8hKqR8taL5yYJE7z9QNyetM4sprHXFSZz7Fprwm2cVNThm4bgqGCV\\\\\\\/cxtxxIiet50BEWkPaaQ28qoBsoi1d9AQydeOVDSaTCB9Pvea08KbIBDZQdhDyPtqZff46WfCXC1ihu+sZWSEUJwpEF\\\\\\\/sfF\\\\\\\/yrhQfDUbpT+N35e5xk7+n2oEZ+OJBqDhvGv5SGfCLocLw\\\\\\\/X7RPBc+1r6jiOgfpuHkjnpmy8750txMrNNGtJ5hSEmWQpPb11qD9oVtqNwjMiCerA2uRvptNmwwdMIsmc9kpORycz+HjVR4JzgTVdJIogRayr52D+a4KLtQIJ4RnkK+EJ1bNrt9GUEAwPz\\\\\\\/GgvtdK\\\\\\\/yw3rMgAB\\\\\\\/n211AThxhCmwuiUGovKYW27YsMn4uPKpLLm6hE6ypwlO\\\\\\\/FHr0Hg095GaDbVQk6pAF2NZfGxrTkjmtaA+T3n26L91ekNV5TzyGfBEplp68jDxa4lN7oV7fUqEHMbmve0eekEKYOqPTUaLDPgHfm5CWejN2eZHgmNc399YvE\\\\\\\/JSWh64L+DfgctPmGIcjLnsQG9j3nILcELpIzmVhumiKPL10\\\\\\\/swPqMXwMp+uzB43HRKt2URr+SF8BmVst68ckuSQORevnG1I84J8zg885pfpaGkpgbilCqIszF5F+OjelqYZUArPipKkt449EwSrpXtipgwfYcYTH6DON3tlyle8qKfyI3VqLCrIN2ThvaBUmFbT8qovMLExxha\\\\\\\/vUN96+kjfWkawf1ficqYPzQ2XGwEr6yPEYWg78GK1GZ6WfLZDf3Bdx7EbLohUnw2FWNa8UlA7mSfSMVkcZx4srIHtkpkzlTj5uJQUEF6aTGbddwOTTjzWCkr7XshzkdE93kYNTxnlOTGWrRdUyPMfeD5VKJLGaAjb\\\\\\\/6S+5b9FVWce9NlViTlr65b7I7kPR94GCAPL9RqyDFQQHdO5fjvsrlUqqgqo71nCiVBJ9rZYxwAVf4sUPHT6sWjL8XKMM2opqqK5ESKP2PJH\\\\\\\/97EeAKGEICnBdsb5ZYN3gWp4gyBh0+rQSJCIi2MJuLJxg4Ygw43fb3NCtOS5TEiVRvyCzZoJ8xEkfMULuUo2wBHeD4Tv\\\\\\\/x\\\\\\\/Djj4kB22z+dIQy\\\\\\\/CjzK1wAwSNNynrFbiGtbzdZX9+FuEbt9FlveF3Ewg69ilEDEiO2BGN63ErQt7mFD959YGxUOYU2IWd0wg48DnGEXvbyohzi2qQde13clvxhdFousYbuNflqOpyDYJVGTE+ui20WIh1VBYwZQtIOGlap6jbwxEK4Z8s4cRcoxF0MFdmG1S6UnOhSr04dfa7TUxaHpvpkfVz\\\\\\\/7q4yplSDpvML306Cfo0+ygtqbXSQHmBXbapN5o++urtaTbIqIvLELQVfkK\\\\\\\/39kKCgwUtw+0dYtOr5ATvSvFrmd37QOUk0nKUV\\\\\\\/OSSS9BqRPi5uVafRYBInTjdegrHUSk6HNdZ0uc2zM6cdRtkmzZg8FzrURJXB8W2ASwqzGf\\\\\\\/Mh9V+UkZdoSCfs76RCn1BCsVIl9Z3HFIKCOZ4B55PDAGvFulLmJcb2KAzz7xN5Ria1Wxc\\\\\\\/eQ1JmPlr8aqbsEjOyr2ezGabDsvNZbTOccDv+\\\\\\\/n7wq7JRIPIYzx4YWp6kRFf3Ag9DdjKjQoZdeiB6Znej5IYImUZ7xrxIXH7BLwU2V8MAwJo2raVAcKfi8nPPKactIwFd3JMMo5N2EobC14ViPeBe38demMk21z19zhq0gcbT2nrKpWIvIHyd4QnDmvstRDJN\\\\\\\/9BbpJZJnj5bMF15m++apIpzZZXIMbTa7bmZrpEHhaFUWq+lWtHkcxrQ7COVsa4EfCe4Scu83WZrxf74FIylCqRuDuRjwM4vNmdb8zMkIxupPWxpynQWxnAMDEPYOqSUQ0Sry26Q1jFNC\\\\\\\/BPUz+9wNGZcW01Al3TbHyKGmEdC8RlbE+fAbJgnDx9OMrvZHxVNCXNV99AyQQ6oaOGmS4WpbZZ8kFNEkv6yMROsFIN0wO\\\\\\\/UEFd7W625zGbPZEyrAuQm8Fp3sqqHr9oIRl1kOdJqHoNvgh4CVFSz9yrqB0MO7ric8UHItWsDBPR0T1TkfDsMBGLIMTDAtQpP3bxaGZ2MUZnqI2d0maF37iOtxv9KZQfMt+\\\\\\\/O8Neh9Oul9lj3U1U36HEhen7Zq+qybkYUVpUQ4eTPVWbUnMeDho53zwHRkh3wfcJCwvp97ESht8k8X\\\\\\\/tJR9fa6ExRFKaFlgkMR\\\\\\\/EjM5ou2JDxbgGQuxeCqvKB1bRGu7dP9CTbtf0YxgTdGUkQ9cXo60gGJIkQobhj0C2GwDYpoZAghvKq9cUmlZXmiJ6vHlItIwWxjWavysjNCLke61HgbvlXs2Xo2n+6VYcP6FEBxUSVOCw\\\\\\\/JBbDJ2yDsQiPvrvLFHfW+5kPtKKI1YhsUmmrjevRF7MrtBZ\\\\\\\/n971zdbQguhwPr42oYVUx12BlbOIvurOkjyqqaIzTm3hosCn5CYMtt6wvVO4vmdOwK2HEMgSqQVbI+bnxEWBK0U8Pa\\\\\\\/bBbmpzbSp6EaUUf38dMOj7T3eHvMTpxnJkXrFG7VWGEndXc830\\\\\\\/mHdaiFQbbKvyb7Pe\\\\\\\/PBE4rNwEbLta9DAtCIxlaFULL35TlCVwgXg2kO3dc3RjoC5p2PqsCScsCB7xDYhDbYxK+ia+QkNYHqOREBKAnKImmqI6PFDCMV40s8x6t21NV2YhQdcirxmWf0u6pM10qLI6Qt\\\\\\\/fczyc6XevzPr9OnRRjyxUo0If6Cr5XxC7vUWfAMDNWYwK9uaet0Uyr5qgR8DNwXkzpO0gHTARjHyzqfN6e4jDVkYgHHVkLtXKK23jG2E7eAeqW8dJFDKR+tVDBBg3CivhTKe9JWlihQK3biNfyhMMJvXxpscvhuJc833bosqRmFz2PgnSZOXmFzKuqPMe4Z9+Ttah\\\\\\\/iRm\\\\\\\/O2QG2DVUvhu2pgItwzXc+VQUz85+sXc9E9tkO9bib4tvrq9ErszTLKkd3z7Dk1j9RmD7zAeTGONyE+VZcVpZOGw7z5Hy20jKBZGhhCNRnyogll8oDITSZgjAIKpO\\\\\\\/yaU5wORXqEZWcebB2Fwk0nNY60F1Zi03qW+\\\\\\\/7OK7QZXqKt2caKykiVEbXgbDpYOglP+kEBeiQYCEGTtbrC7rG03moLyE5815956OU3gScw2HSPy3wmnKwyv56Hsg8PncR8gmdsrkofJuCzm9bG0UraLCXxKNpiexdakVl5JAylN94TGCbCBapb2AQ7ZswSaDCCpGmuic9nF4kDj5SkapFVAXHayJuv09Qn0RbUFsUvLZT8tBHOijwvECBggEqvk4BfA2+kr0zeldcdEszss+7TvTWtirJ2mJiObvkpD+eAAeG+j\\\\\\\/9hbBoStVjUFH+5xPls8t02xiU8L3p18togMCB1orb3KSDBQMgyKvpM5EHvQNsZzQAkvUdrsF5QQQsj5S3E3i8Ew6GcSEb9+3hqsgv9KrJjYlvyv0\\\\\\\/XIAfRu3JNfsIjqAfOeNN78bKjfRMiMTx833JORjUFDcMdH8S5E4cdOGhs9FPIQq908mZ+JBZSfabi3sTZ3URdkRU7gFkzuVocTZa1JHCtYq60D29CKk\\\\\\\/x2tDX1loOaSmwwo2eAmB5DKibENLcxgKl4o6B0kWxWanXmbZ1u1CSnyG5feU82H+Pd3Z0fFNo9DISazuXZlh\\\\\\\/m0DmZRLKQshw6gw95PUX4dJhzS6qtbRcRGztR8Gfcmk1Bugwx4ZjhLB11tmi30ZY4Nmpy3a1GooAqjOTDMwpZwULfgXnRMx1xLCVApPMwPlUTgP\\\\\\\/ylacyaseSAKDu4kvIO6KGSUS+EgHInm6dkrWJvNHUePCHsbuEntmI84dqZ8YwL0de\\\\\\\/mmv4rpSDo1C9NjhodtTA11ipLPbCU2cvydQBai+fSWvqoRdqVd+V+tstmgMSGmPb0O5jY6qg3uOqJfj\\\\\\\/iTHYnNejaHEYdbJuVPf2AxHX\\\\\\\/51k8cByqhwh7rCmQxRyNSzvEfBeC0blbbgsMZq3ONwx25EeZ3cycte4ZL+e5xCxE0M9TCq1DRRO8SjBvhjkH5rWbyb4rsqwNIeYXRKDRipsWk5wBHSu\\\\\\\/Lehkm3MXpXOUpi4VDLKNNCqhmW4+kTeJEOpHRBjatU11iK7VPjX4wyUazyCrcjegymO+hxciAyO4PeQEK1vbuCXt4Ikf912Nob\\\\\\\/FKqBLAf6zWxTVUDzw4\\\\\\\/TY5SsWlaKauGrvQVFfO4OcnzMXeYLcD+0QhxWsO2LVBg9KoRnboIoS\\\\\\\/Te5MvYQX1h85AKEQ5t0BQa\\\\\\\/MG8RnNhEpgdAYi+kxt\\\\\\\/NGPOcLXA73TzfqxaN+pvczeWyGFd4pKar0tYPOc5IwTtSrsVdkkRjd+N0z3RRBGNFZV8KflifJYHnf0pHSRl\\\\\\\/4TZSoQXry6zFDyoEOluX0mxbytP6nv2mnQkRbE8x\\\\\\\/ZWLhdXrfB7\\\\\\\/QpimmjLU7EPa7kFVtWweHA9lPgs4ySD2FSFR\\\\\\\/\\\\\\\/itqwISl88eLMvne\\\\\\\/tqNbd8LXuU\\\\\\\/BHOMGIoJZWW2iOHOtf+eY4I1RBliw71zR49GYWYNqJ3ssICS15NNU5QD4hkGqYC8UyF1tAmmAUXJC9G9apBH1rgjeMmOtfEqigR+JSvqGDrFRKaq2ohJrZyVZHO1YqDSHtjZ1\\\\\\\/4mmZsehI\\\\\\\/9q2cxqvQeIrWwkjPSYsiGVmURSS\\\\\\\/\\\\\\\/9CTXs6J5ihZSDKzyv\\\\\\\/MI9wHtayZExa64nrulUfAW7+dg9HpmjMnbOeBw7UCM65S+4zrGGfNp2rYfw4xz36yKwBEfZzi+iD7hzNe3XvV92+G2Vqlv\\\\\\\/FpZAVk23cgXeRIshgt8B3kzDKoqTWSh2tXHoyhpMTuXWijrV2qrLZABxbQQ5VeDE8Xqsapa7OAEwMcNJV+RF+MDrYC2i41LpLzKR9OrP9Dg\\\\\\\/GRETQPThsprtzvPppgQMX+F2CJoIdh2oSbhaTOZ35\\\\\\\/X0jtXmG9zVlWyWYFO91EFke9b\\\\\\\/ets3kIY+jS9dDpGEPn4SItudP14MX8Bd4FvvXg0COGhamq\\\\\\\/bTm1DV6kPW7pQ4VIeAp8kZQrCJRlegUy+PLIoGe4bH7BMvmnt+fJWz8iF4VLigSRKgzVyIPq6+IsYUaxWzuuOU6MMQ3hV3XpeLAACPB\\\\\\\/ctLMWasR8HPQxKzFx2ppPJTLI5ZqrxtIk8kB6aZgqmTTjkvjrcTvLeZt3k1uJfedmTucEYeBbroqRWiDR8eYckdfdgSUWD8E9yCk\\\\\\\/Iy6Q135OcyicLidUbkmVkXOgGC8aJRuBoWpDTYY2vOVisXrxi+iI9TcfeQvVWFfbV+zAFbAnzNa47uVFiRqo0hz8zPylqBCk3L47jXkngEotbilO3yT2RET1a1Suj86T06sgL\\\\\\\/fVCZG2WRmaBsll0nHBXJecRBFDy7yc8Pmv5I9rzxDNy6Atp+VitQ8yXPf3biT+fgPjt9rgnqBO21QGcXsBoOpptK6KIA2yqNxRamhbXeGtjP9OoplRpaq4ibOs9e3BaWMnl0mD5A5sWFd13ec1kN4oEDzsL7eoqOELsFwAq1oDraCa1AiM99qlqpgUR1tbRzoQFtjuhGxjS8V6JJ364EB69I2ml\\\\\\\/Z\\\\\\\/Fhl4U48zQ4wzBSS+HRqDscWWI7U9dMzvwSdFLAQoJJoQflZXYTJVbfTreiI92EssaH7DVPEKenOOqKt4vh5CR3+3toFKdZJZB5iXTeC6h8s+YOgIc4c9LXMOD6jPOrttfEltep1NI1VhgWz8P3mzpOzeY+Sz0SzpJSW+uH5EPRxFr+NJqf5FszYKGSrD1wwkl\\\\\\\/HVjStYvA1OPp8QgRfW7nPbfIciRmDN9Be53VbekplLX\\\\\\\/9SKlP9aTflZHytEMTgwY3Sox9AGyExBpvo3Q1un2GsGN1Dy+XCRDNuO\\\\\\\/9gz9laFJfNzg9LXlJjQAHCB+zqR\\\\\\\/nRdLBK9603dQ7H5OlVv0tNhH2HqqezytgTuZyT\\\\\\\/FHmuhnO7kUKp\\\\\\\/PMGGtyGFxGTo212dPr+rG4b0S+j9PR2p7pkg5DIhYZv\\\\\\\/siFEFxEjITFUfHeuwTc7I1ltslOUhJNPuXD\\\\\\\/pIDxJI9RUv74XM6E2KVmeiR6fhNfbq1okfdbjxnitkUWZIVjVTGuxkRGw4MKH+xbxCiiw3zLtwbQzXkHNBiEJqhetuSU\\\\\\\/Co7ThwZECOG+vTxrQnyU9fdU5T4T2l\\\\\\\/Axu7dN6Qub1PaHDr1fidwyRnsQGVpx7hhqPeeSCGc58L1Ol4ztgrQolwB20WuVSQE8aYH\\\\\\\/QsG9ESf7M6d\\\\\\\/ttQfrq0tpVdF\\\\\\\/nsuJ7sI4V2OqdYtwURojCy8oZlYBTzvk0xRIsxIiQd0zyxY1S2Qktd3ZtcNq+0sKS8MbvsSdphHXZwGzhSMcsb41Q8AsDxxki8Vv9avn\\\\\\\/7pUKfIwk7fdoV6cqQ2MoSWY2t9NjDkN2oz7tOxRYIwXJYE0UrO1jHvaeEt\\\\\\\/vFsL58Q04BKQ7VkEnp4etikxeRvBgw0TXsO4ci+kinIRo+HRBYqnnmroiok9E+TukGc4kL9aIcReSSiDuGUUIcMyn5GOoCQVnZ9RcSjMER93Pltzg0bESk8wolYwY1Dtku5Ga7PP0qRB6Acb1o\\\\\\\/4NxNr93wuzF9qF8\\\\\\\/L1mT9PHwap8xTtGVgYXv92xp9WNOfBiF8RJcOzee+gAbIYaBKuK63K97+\\\\\\\/YnVGV+7AkkVf9oqmwi2zAYFkwVarrScnsXauko3WojO6WNLDebYRcVOaHcLnt2ycESAAojgSmOo06csEeROGU8z8bTEHDqljz\\\\\\\/WTvyDG+gXIX5T4FvUKGeHbDBP3S71yksb6d\\\\\\\/Qv9e0\\\\\\\/Z6tThLAPFBUrNONOUF7avgB5qbZZAWvDYLSp7Df0ywFuKI36821vT1SY+INYMRQqP17cAdp0nmvo9rVzZ2RbZzTcPh4gYGknEgzDbOdUqHl+KogHtHKxerg6ImvL7R7XJQegtjV\\\\\\\/LAVYzgsGFuqbNAp7qtH3pwOHgdSv20Q4B38J65tHp\\\\\\\/RTtaLj6w7t5QuQHfVfmxmpDh30GswU5ho6Vvg3lst8+Ny2WmZROtO6EGc\\\\\\\/JKwI10ZyoLYC5\\\\\\\/P\\\\\\\/mCYKExUdMdhVkRYRJ+evJx6dUvE1Jtegqiw5JPb0akXzHBoLxvJIJu\\\\\\\/itJHBjqHnFv6voULDERYz3ZTl7nBeOyWnzEHo9ZcOciujN2wLDukq8EXdZVklBpcYehFLb\\\\\\\/JiiLKL8p5Szyega62DUkV6+cj0O20GTdI0O3CvTEW3i2VJtWkSAbBYLS3i9+5X1ugTUBM29rFmYZFjDPAFhzLwDqODwgPBAambTuZgyDp9t9F+qHV8TsZiN6M4a9oqwexOnuNOVkhlQ7bnWl1PJk6d9CMzH+5fkhY2RXkH4dgxPzwdcYedLd\\\\\\\/Tdt8uLq2l24Lva82YfV8C0VAG\\\\\\\/VTulGkxh+6oncKASoTWEMRfp8m0Nefa+UvKwH8DIhS2IbIs5Ywf7HnJfdF1uCWD9DQ19AINACP8D5dWGqqNThJbUkfqhpWN9dlhLk7FolDfmQGoHa+VXhcmcgfehNUpfyJyePgS56iQ65Z8BOBod2hOgXWwMaxVy4zA2CQYOhY8Rs5qz31UWVclw\\\\\\\/e9C2CjfBM1iZiC\\\\\\\/Eq+D8goS2+TSehtl3egbz2DDqzshg8KcWFYjUbHubh7\\\\\\\/xMU4arQbQMe8Hta2poTu8WmOhyvmvWgyLwaFTdNRuxpqGT2OwnCo94ks\\\\\\\/pAyYgDmfaQB\\\\\\\/IJdXnIW70r\\\\\\\/Mhx9Dz0mmkXeWmwVmBvw3\\\\\\\/zH2oOMXkroPFIPuQSnpJZNf+rvwprUFWgicAeBeTjmrTDn8g1zPTj\\\\\\\/rMvju\\\\\\\/C7fkR5GvkUiSgkQqn0H6Al1ODADNTRnRspIg1Qca6P719\\\\\\\/xhHD0jbGpqq0ZudePqk6WUMAeTOG9krlwT9pnjBPfM\\\\\\\/u9sYFjwYG2+SU2FDayR4R4R0twCYRQBRWVplTghTfxcinFGmGfZ54AVHZ3BKoP1WSEeY\\\\\\\/zRxKclUf3YDzWSfN55THJzAhVh6WHXymcvA49igCoE82kDN0nmsJmwhPBEwWXjLdpe+Aa+mhPLZzE5FnPxxai4UOZKQy9qQUdgLe+CCucL6uUxr\\\\\\\/\\\\\\\/ylB8jBdytMAxhGcqbdNX5GZwZ+TI26XU0Bf2CqY5R5ngkH\\\\\\\/FVpJrktr3IKh62e0ZtfmJz9o9TY4j\\\\\\\/6dmiP1l5mwqkQMxMMJxzsT\\\\\\\/tob6YnfD6F+2uoviHHbROhBPGrqcPtiAE9V\\\\\\\/ZFrjIcuoW19AjhpEkXajNCCU64Mhfu9a3LR1LJrh3ID+Gzo1uux4yVYrHvGd45smMW7+G6Pu5J1UOB5xWBGld63U6BXdgU7GmOQ1MLP1DZh+OMeCmPt49m4JCGDCT5ge\\\\\\\/ACIlsX205i2cb2YpbUJOjWwRcFmD7fV84TUQqeBCq2x0WYRBSeSK1iqEm+FGDSBSi9sagMCU9S+hoOR0N0MZtNiQg6APrSWy4rZy0AFB3x0STmpuU\\\\\\\/9ykCdVr73j2BaCIpzmw75l3TzOL8zM8wIyqMVTjqx7MmQXgyKOMqm9+ZpeuPp7CjN+PtM\\\\\\\/8TvTg0VvC7d7I1iLZ68B3STRR8D8aO2ayxN3XUwPsvyl51H4tqutbrj5W3A7kTBrPV3ifYbgQQl3yu6FYnbiodip\\\\\\\/VuIHeR+3pzlDjMBysg0L1lB1U56TxBK7fUMjboPvWcX0ALqLbJgM4LBEHLiIKRLVVLVOkjZgcHFHGJxekSNIjukoG9JK+AnXtk3IgMfSzZ4jFoDOQBi55wUbCsuaQbno4PhkvoJV5SSGqwVFbr6EIajlwmqBGm+F3hVyYSBjBQr2JibUveOjSxckdYCeCpn+voAOpsoaAuWu1pLL2noTN\\\\\\\/uBb2g2TFTbm7RcSiZ6CrSycDjBGfJPkJfs\\\\\\\/cG8gal+JJ\\\\\\\/e90dtOt4VfHjUu0mWIYXglsd0WoGGrsVDg7cR0lOmkwcjQ2vr\\\\\\\/f7uy2yql6CYYQt3V+OC4VvbWSnoHnBVNWuTW7BUANdgkEHN\\\\\\\/GXCz+0yE0PnC85DPjGKpwvRDJzF7vaUI142kUOcpg678iCteGKmiqPYBI46gL3XDLddbYIX\\\\\\\/NuABh7CaF9FI6nFL6ec+aqGja9GYIfPrQ4f9cI2Uhb32h514sh0287X2JNNw3Fal8fik6ImszAP\\\\\\\/slu8Pg1SIXHylIwHVVSjkR960dN2bz5M9VjEwWCG7e87wLdGKk0u\\\\\\\/xW+WRW0PidcuafEiGRqrgEL4mMmDv3KkzqGJteJU2VSLyO8PAhRkzmtQW3wb3\\\\\\\/oocJxRbYgbT6ZYGTcLT3T\\\\\\\/pRVtj6mk8m8hcgCWFjOLOzCA\\\\\\\/\\\\\\\/tw7rFAfhNMEiAcwDmz9VvpMa0KQC2dDj3yK7MEIgV4wfMYoO7rRHU8A4VKZQE0tAvBfUQ1bcDsdvaNy8rHPTW3s+ZH9Pn8ROA0rnnBQnOMbjbq5Z3ZVJtiCBpo4WABDSvFhOkZIGl\\\\\\\/gkRT6bZlG9zGUkF0u2B4aE5FKu3gp4+C3b58ARehjkYvTjo+bAMBPyAx7g0o30+tntVzif8f9HEX+ULvL5nqhH+LiDT4EyVKblPzTlPI+\\\\\\\/kqGPQm\\\\\\\/fxd8ZGiGp3Q9SO9jfmGBDYCyp+6WQfexFoNc3TYAAPk65kfTItw1gbiKR04gfVmGvzFFK+\\\\\\\/Bn6conUESGh\\\\\\\/c+B9\\\\\\\/x4GLlfR9\\\\\\\/kRa9EjCStsOKbqkIx\\\\\\\/AEQVKEnGg6xyyghDU6DT9T8AAXdnvyt2FmgDvoB0\\\\\\\/x3+q4kAOzkzQSKQGloIqAmRISmD70\\\\\\\/sLqEgvwmZ2ov1p8myFAdaYlNzf3CbmpQYg+DdGSPb4Lo\\\\\\\/FgyXPOBDTvoSGkrzZDsyiPCZPwVx0GxlLJ8Mpv\\\\\\\/131d7\\\\\\\/P65zGvARAWS5YVKriun1O+V\\\\\\\/vORIP+5GqBhXMOtHqMqSnhnXEc9DURolDGOM7QbEbjmi8+gmr3\\\\\\\/itrCQRyWKVlfqKp4M6eRieSPt8Ga0+NktmVJSsXkZFuUZg9wOiKOt0tpOr+uzhrULgjOrr+DbtN+Wsa8KMk8d2YAcQNntgBvmfEQamw0x5FAsJvuBWfIwYV5MXke5\\\\\\\/SSLJMvB0a3L6KvclJO8D8pFEHgG7zT6nwQuVZUEin1tXO5fYAqYdHu4zjca3OkeXT1TyaVKZqNYs3\\\\\\\/an66dnRSWdRp4wKafh8BgcvjuewlfbTPY5PfvtsOTAM5Id1+ZNNLS0AanLZhN9YEWqu\\\\\\\/AfjyS0zeDP4ntqEtDWZVm3gB0a5DT2HhmIbYrctrrrs\\\\\\\/OeY62ZpnezxCtmwX51HAUyVEDpvs3il5sNeDByHPiNOR1JEZK6179aRhxcuNDRMDFtJny5ocKS67he6CX2IZ4qCPa1kCvMHGVtoSdsmp+DYka3aenVG6+yG8H8ppkAY7nUXV7ELLnksAdQwA9Y2EshOZVtwbaX7qPPpLhUOORufzhrNoon8v2ywOp4ZtLqtTurYn82+MsmuI7uqsYKzUhiVIQ9umA3Irh8lxaJ7kTdWkWkv0a\\\\\\\/2n82PfL9YcE8thdUswnK8K6S\\\\\\\/q2tI\\\\\\\/biTj+kdqZEGGDoRlK7SaUggy5bCGQhbzwj37IhFBrMY4bFMX4OIICRy177fF4weXjqu66OJ5S9LhyJzu\\\\\\\/J2N3B0kOs5S9ylOOqwtmt9lEZQII\\\\\\\/i1iS0zcesFpPkM4\\\\\\\/4hd4QjBS3yj7uSLPMrMlogMrEv6PfRGd3Yy8ZH8CnpraD6MeBW\\\\\\\/ifX0hMjNVtbkt4i+f8wOCwcg+MHfkSbsQqW2tDjZ4kpz\\\\\\\/fZJBRGWDxF8sBkM8NynAlkh6claWbZfNSDPp17VWAPpsZNAW+gkj8msLTupf6Cd5qdPAJySoClOQ4lpRppGYT4cMuHOCaGLd5be4tvzbLjw5nTfJOz8tohOupGhIx8lUhygiawThVhW\\\\\\\/ApktcP7ryFURBTucfHK0CX6dzWAk1jZ+1qD1HSdp0fysF9DtyMnTz30bV9mXlEKOMxbzaddtFOQdy1txB7KkCiMh0+4NbK1v8xeZx21q5Z0MfMWTNuyYPBRf4UMP8nwqH5h7+UNNPYUVQr2+k4\\\\\\\/L7a9ijZibV2OS+9mQesEN89t\\\\\\\/6O\\\\\\\/H3TXsaEVnpC+8AXTJxCJ19SNGigoEsD1AopAzLIudRM4Q2NGGvSVEacJfuuIwNgnS+GU+GYZ9gAoUDDbMcru+1qP4mO4bVek+s5uSsmOOduLIGACMPiirDHO5vDJvSjsTOFKhOadZGx\\\\\\\/avodxBdAR9vyxHao7cDJyNKSuj5vwgktfMc6coKV5IjGH\\\\\\\/rEFi7lMpBEz1t2ONFuOfldybI7WLgoZInjxomkU\\\\\\\/EkFGDngSDs+ghKSjtmKrcxkCErw1yD2Cq+\\\\\\\/HJ3SzTKxTnRJOja1fqpchwRYCmWkH7wb0KzR0vwFOcYU0uuM1DXUipQ8X+TXgKUWRfjeVDaLp7cr3A\\\\\\\/+6+N5rThEgmuRw\\\\\\\/lVsswzvXnx9JcSux6DTu9CmdCwuxVWAgUdJQQVbXw0wytClzRepfkqLygVTl0WBo007LFMjWgAx4NgTZNHTxeRVQv2k+\\\\\\\/19ukZUMy72qRltbGOdTMw2MBQEBlRdlI1Rkt+NDWRMQUmggYP1UjE9jbGTlWrvCIi73l4z0ukDDDfswPkW1LouSs+gTQeZuCtSDEDPgt5PB8v2BU3jTt4c48era+ZjSZhftWdhKbkDqoIzit9xWKDWwyZ\\\\\\\/DrJPxd1uQtte4vokTrtFjQ3y8YfCBn+mMUWqM0AagN2jGss7XTU3HJ6dqHbk1allXYXD9PrHJGE945EZ4IC5LPrl3Y3O0DHQ3pZl+rCo2RArW1qmJoLZVtmSCh3nervqnQEPGHptklxuBBevUXntBH\\\\\\\/NnTa2kXdB\\\\\\\/6JZiFTfEfzYUA\\\\\\\/u\\\\\\\/fEZ0pKChKfqX5batAThwlRSnktHETIzSIzkYxREXVffpEGooPtU4NZKVeDqBhkPSWjaaFYJpLXKH5nvqXQhAtpy3P5ABviaHIqfZyKJUn4vjIurwx86zY2ACmY8+J152g9saEpl0be20u4hhmOADEq\\\\\\\/7UaFYj3h8wtC7aG\\\\\\\/8i3erlky91SOepsO\\\\\\\/kz5S5IO1U10gQ7rh\\\\\\\/iaRtHOidvaSv9O5kJC3crEMEwpFNU2a4lsYTTZyJ+CCWr8uhSmRbjliKMSC1EJL7ADAMKPEYpQdGxgCdm4Rx2hmbqzrJwj1Efgoa4izmDRG3KLaLM9ss05KnHSdE3PTDKjZI87N8HGtSmVReKcru9eZqn8OS7OSTog7ADTO16oQbKRqGTaxXLrxIaY+a3Xs7iA47b8sxiicDUaPvYIpHg2aY5rMbCs7gECVwgKjeDtsCAuJz2LBbtkEB7Z5Idt2YI3lgdKrWgl0DakQoWF7L1ZqFXG3W5wQ9mi0Tc3XRcP7FbQH39zgIwCdnXF0aUvJyymVjCQXoHhVrwdlPaO5+64YTISN1hoWISf7rT1qwJ43FGIWMhuwRk0H5mK2W\\\\\\\/cBjtHUl26b3STZrAzqR1sYgiaTXoge+dP2rLC8GB3TsEA9ti5jJBbcPJz\\\\\\\/LAoF6NKA7ixn6K0Ps21fXFcKrcEggAocuhtRhN418Qbc\\\\\\\/d98TbDat03kkja4IUmJB29DMmYA37q2bmtqsdko3KADw3Xc45OItsM4UH+7MXEOiW2QoRkh1b58Zi7PnZA204p3EbYhbjgzwH9udans\\\\\\\/Z13o22XNQxAqAEUOGSejIEdIQb9G20Q5LjqFflr0GXlshNnyTjh7CD0QABwerwbZ6QxG0Wm6PuZXanDMnKn4Ya2p\\\\\\\/YHYqCnthI4uwN\\\\\\\/3+DGDOpWjvR4CP9AjeX6R3teDbkmsWVq9Vr92aEoSUQ80QlR2wA2gNuwkqhOaRAZTXUNvR2jnun9iZDURW0pQaYgEzbopXtYOztbVJyTce5iJ\\\\\\\/wJoe1RFmReTKDfCAqqJRxfSn1pGCcAmdlWpMjlcmYPLScm80nPCzTpD34YZOvFJRK87gN\\\\\\\/JyX2wFRzMpDZioGMKjKozY9gf2AEOm7iwUxyokIV274VVqCvwTtbUgOp6bHz8h\\\\\\\/j5hQ6De+EiAkT0LAIWx2v9gTXNPfaiW5W3zoHOXdtkFyyahXS0VIJn+I0XK6zY5jThVY+LoRAnYlDm++0TcJ2+gBn20KeuXnxfOvfebZAsw7IKfFM3aCcktEuoo8F\\\\\\\/+G6MuivqH8+wCIqAEfIb0pRdh5oEkX9hna7nSYq4Bw1fgPSwTJAO2r0bnTQ1kqcKRo\\\\\\\/J8P++zGmmEb384rIbdzmpRvNgjg3qD43m++6opNpfZZ\\\\\\\/rgmhzFW2CtXPrXnQDDF8muhl+tGTpVjcOUhlbb6\\\\\\\/VVemc03ATiASkramC840sTHksApIA0OcqOo4eGpo1T7WQ0LCilptRnbbMe5GItH0jeJ8ZAYxXgpve6WJfaN7ZpnZ7FqvIDe1D1hq0NfJEiBluHE7C4UFZMPk7VB0+634oPYVWwCU3baOYNnH+jNfMXuhdyO0Z76DbYTo0qMGk\\\\\\\/OwhRDLdp2z2XQqEeY\\\\\\\/lrJPrysdUjKyJK\\\\\\\/xbZq5l6m+G\\\\\\\/k9vnHYlRT\\\\\\\/cHiXEcagCO\\\\\\\/KPo0kbXC\\\\\\\/mEt3z6FxvDPVieTmNCTmAKryY4kPy7+7+iT4fQY6Z6w3FAZTsPN5ndnh3270K0cbfgjJ5SU1eaFFjnl+V5fSVi0fIBgSIZcwHrPr3kJbSquiiXgdNm7ixtep2TDZoROwC2X5ZkxL4nicPpxhkU69\\\\\\\/kY9xXKCI+1giCKisNTDVq8OHj48ATabLTMoTejL1zcPU9tc2YU2TFHBr4CFULHe6F4oWabeuUZFH51WrTQMd5aJgeKJQFm6laEUyA0oekwwpQbbzNLMxRawstEE9IXFfaUMsBTkEUJL2romoaVp62T+uf8KmXZLS07GP\\\\\\\/QvkTJoShjpIcK4T3H757S277nKdJ9fEHbUtGZSD+l1lG48mMH2MQp3Szbda9V94FbUt+fyMje2jIufMv6xtRaFgdHGn\\\\\\\/M3qbcsuBTVfE5zUvq+JSNsNtruJKWCdykp\\\\\\\/aLLglLB7fp7XJ421FQkEfMX2J\\\\\\\/na\\\\\\\/uJoZYsl9aiIMaLYHxcSKTBSOF+xNkIczRSFuJnx1t+OFHa1tLSVWX+iFIWfg4vwPjljhAHmJvsL66v41tVKUfseH4Gik3f+q0SsBbNHBNhUwOshfH5WMnrvaexrcupF4lBRpkSpT+V\\\\\\\/m9nKSJEfj7Avp6oEIHDxnFYzoyyLDKU1Gy1NWMNXCjcNHcdX0M\\\\\\\/7LVIqy5v+EW2Gu48vZpEXJOgZZ21NOl2GFgULeaStPfi0oOaAxJzpGTjsGgNoUayQFxVVNTGNIYhreGZ6OCNBEl57L4k1xhV5+I5m+t3V6KrCm\\\\\\\/J6f9eEUy8+vl2e4+b1u2Lul80VGvnlwXQuqq1flbufm0IrIw3YsMBGy+Bj7gzDQRCNO9TdjQ1pOnC5P52uymjplrNfRQ0Tlxg6TxZcZAnhOkHHfPukeR70UmFYr7sDrLbsyqT9tuWJEzZz6BzoyzEPdIYmeZPyfT75BDf7eGyz6wM35fV3JeaExrf4JeQl5bpgOdMPA+Gu6\\\\\\\/PSUx6bgt7i9qLChdmWH8fYgCNEUGzf8Z218loubTu0DQrodZUpHGIUuiM0+1DGMVvyjSkui\\\\\\\/+W2ix6Kdfq33oPAi0bAILH6UsYBUj3vmizQ3m6jdjAGoXy8xhFRn+qJZWxiROyfTr62S2hqBiw0UhQAs\\\\\\\/fAeMzI3FuYPCMU2CV5zVJNcZtx\\\\\\\/PEoer8VZUZveR7jHNvw1HLosWWEiCEFlxIaG3TjfMHgX2Om3rLRg04KPvvl+cUwDTKQkbGyym9naDoINnmwUmAM4NBd9504nuWIn7\\\\\\\/b2JiBacj3ATPp6sNu51AXv8KERyiCd0s1jzimkfIoOwqhUCNS8dYuYAYDnyhsuIMr14nNbYblv6rR0\\\\\\\/v7+N979HAmIGQ2d+TfNqxx4hwgF4qNiwSRyK+dNYbX8I3xXNs2p792jdSkoI0nz4qVA4r7QhntAjUhpJgnjKHNkgWEs+J1wDbzYqcAZ928JPpdFSU56zzN8UTSvkSm3yrlasF4scCvOT1jxbG+lgJMfyiliThcSaa10gXIF\\\\\\\/OpkTxx+d6SOs4QTkqqxTyOp\\\\\\\/VPyMaanacKZikQIaNyS0eajHQgEJIbX1kBfAFJsiN4O4xl0sV4ua7c2ielwQHj0NpWhU+FYFp0GtSiwWirM8cjCYlai2vxfuLfyDZlcstECFLBEJ3cHN1BXQWWK\\\\\\\/Ep65Q0kIIB3h+\\\\\\\/S6GsRBD4oXo+LskJ9kSpspZVrz9rpl8hTXV5D+AS9JzNLzxNjW3MoApX4xqRErRQFHKSAK4cYp+CtlRwWLtKNr8\\\\\\\/tY+cdkG+qcfdu\\\\\\\/euZBKty7NFLeujwJVEAbs+RyWAXc9g0eYJAkkBBM7+2\\\\\\\/nqC2O1FPPg7ULy8lKzl5lA+NllBVX4xSigKz7ZjP4woyzleMZWQlZAS6QeMhsVBzqCkuFoS\\\\\\\/CD+3qQQVLARcmY63bOB\\\\\\\/b23WH\\\\\\\/0Mn0GjnVivnbLx3ZvU9AwGuNsn2r4rSXXhQ\\\\\\\/nrsnpX05AD6OOAZ1\\\\\\\/7k3SVHhOUKr4622M965THgSrxsMRjfcVnkaFqY9DYFYPlIlzvPvrLFWK9Xz8zQeGNNVTyTp1cnZg46QOuX0271YZ8N1SR5b5ZwrMalQ557EPUVZ9tjFHQCNt7RtLJZCx9jlCTYJb77PodekMb81ZxAHns1phyYMy7bZxu9utzdqM6bwq7Y0Tv\\\\\\\/0mJr4DzpdoawQ7iaHKyfosy0X\\\\\\\/LCBtsCnLttwwd57KKEIbpvF+jz5DcTFXfizmk8Z+R5sMIit7qeZffJq2aeP6ftxGqvnEtUyz90twswv4h9vClEVx6h3n4In7CrOn0hCaBLywwhSzNX\\\\\\\/b+HaSJU2a4uSGoIxTklHOcpVwE74TGr41eWhUMofM2q4OAsSDKz4pAUgj7I2Bqnw0GtkRvUegfIFmtJqpc+t\\\\\\\/mK39ulrszqvuyGhm9ZPcZMp0fVsApRB3QdZLXw0QD3FsJsviF1UkfGDlXM8U1OTu+MxzfIWgHKFueR+2ck\\\\\\\/hV27XPVEODyWl7aWVaCuuTexmbPA0x1OwdHIdFS3n82AD6hSh0oOHJrEFL0q2CaS7cB8FOD5C1h\\\\\\\/z\\\\\\\/+ulyCLdsqYD+9\\\\\\\/qIa5PLVgR0973SUV5wu9mNuGxUB11lO3PlPJQDRgpSSXw1hyTnh5\\\\\\\/gKJaQ1QwyDFmk0k2yFkpjhg\\\\\\\/u4Q\\\\\\\/JSCl3ZYzOUk31x2dKlR2cgns+UHwo3NivxBdvcZ8P42\\\\\\\/HpFXmxc+DJMT8CSVFI6Riq8pubQmHA8I9jiPlIgihssNzWpZg8XRACeZ+FBlp9LnQmEUnPZuYGkUV9wlOB6rXAcGHO4dvcG\\\\\\\/tNJDAKx+Dz1dW3fMGo+NrEUKmJ5x4DThT8A5l5qotgHDO8TT2bk5WmNJt9OahB6Mu5kFx4kCGuD\\\\\\\/8UG+T4loau6LVwKVNaKvItNFq2Exy\\\\\\\/ismiVtzeqQ1XeHIKOKFsPaha5WggJW4gdHIaRT8+Sx8PZHNJjvi6seKSB\\\\\\\/byCwhirgxNj3RfFAqtIhLNuwP4\\\\\\\/nUdb6wcnsA9uohINmCBbeYsv6dZd4v85+N0jGKn8ug50qmBkYrHJZaf\\\\\\\/ujeYDLJtpel10cI3yfP3KmQsNq346332dl3SOqRTXN6QvGjt7Vy9mBDyDMnFTO+5j0lyDoY5MX03wFkQb\\\\\\\/U7SGdTtsOxMNZi0tEZlSplNBvGRc6y4wDNc8Wy2Zn\\\\\\\/WedmzgxiBkXeo3L+qZKVMok+yrZfGbAkRHZtmv7MchQkhcsL\\\\\\\/5uX7ROKvDAdGJd0LHp94L\\\\\\\/9EvEN9IaS82Jzvrp+I+yBclAG4KkEBZNEYJ9JU5NhqHkWIVkqqmLrMcEC86xw0fX5wqdpZ591pYKN26V4cR84inxDq6NYwjYzc0IdRIC9kFAtQxfQEumze4XNXm1orhwn5Khl+0d16YQV6eYPbCVJfY0efnc9b+csEZmvqh3kegtxwL0ymEKmLp3KsRYixDDlVauEMjiDq0wgyfXG7n5R2Jz2DDakgVCfD6dH6yV0gAGF6nx2hMydy+kBt6UqQsFg\\\\\\\/kTGTp1PYh1OolhHEbrpIT7Zu5ziyfCH6M7WWJi8+pZiSxQyvk4tVNXdVZv\\\\\\\/Bo3YgJkMcqH+0Y2QPw6ROhJSuS\\\\\\\/nHwGRJruxqp\\\\\\\/jtNdiUeC2ZOvb3W3LquOzeMrKYr1ppY\\\\\\\/I97jg4Zy1zbB+Kiv21Zd+k8s0WhQgyx6dGrhMBCBG7kTgCOq1fiW31gUb\\\\\\\/aANy+noeIM5aHf11NzSzJeE3JPJySP8JSHMrG7v0ugSltLO\\\\\\\/qhhFbdwJBooNs+FWh4yIK82tUZgxw9SN4FMaUykcKPHS9nRIpDS3mwPNvzpKy6cDv+NaHoH3n0rXHIGxtBRC1EhwG+cCKfW0cvEWrbwp6ExIgV\\\\\\\/vIOW1BFII\\\\\\\/itQegGoVDi0R\\\\\\\/fooV\\\\\\\/1N0r5yhPGMeDPM0u+6OiWmhGWwG5xU7qpXrvOf2q4Hp1puFluDJez0GrltIbkPnPTilOimfyEZZwyXl559sNShAOc+lvuG1OjkaYXPlOKVys1Uwb3a+tVBDXP7sOA3OMYM02HArVH9JkSzQaQsqMOqfBOO5RyYqQUC2WqCCaZbFaryTUs1DrfOOQbwdgVHicdWw03WsePykFjvFFrWR46PNAaAzrXAv1KWPF4X2E564WUAE8w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OIXtjjr413STykWmK7Oo6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217b1608c1073262d56d43117a90cb50b8a150e7e4802e73328e4e30a842e7179\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ev5hph7dzAKOCbQHKCAVtzM6a0l2Zo2DVyM6Rp9Uv6IZ6gZFf3up9X\\\\\\\/c4b\\\\\\\/zBmcRAL3+NOLGibAwxj8OZ6Hi8aWhvrfs5EDT33Ur10B+44+ecZ4FnL8LLFv1hckl7Hr62bmMZ7H+e7eL0eYNE\\\\\\\/io5znejz21ENa2Pf3Oc8ONiEtmrAueLQ3BY6XAdNsii34gBxm7tBR56bDbkRLxjTrCfSwFF2FBcBpwgPuX6tkoY5ACv+r\\\\\\\/kc8dYC8HyqDlYQnd9QpVF2YGGecZ4SuJ5\\\\\\\/heFdV5EWaxE1y60B1KoXKCq5fIVbMkKvHbNrKPNfrrZIqAMZi6wy\\\\\\\/4M4278lK4daNhmVU6JN+H4do99fYK4lXtZhhB8Q9snRyD1JxVUx0eS94\\\\\\\/QJYpnfEDgddz8KcHv4O5Fp2j+8Ajo1KANrhDG5ux\\\\\\\/8eRiWnuremb2m4lM5VxQzSVuykq7Z+Fq\\\\\\\/WhbpFfaIiLfl8aZjDzWUyLPWLqftNrf2g2bFLa52yCPvut0bGnxgw+qvn5URlyTomPd4SxAHzJTNEUrfBiamtcGVOw2bwoKKUMTkfj6tETgeATBUeIutOq6Z2crn3Ksyjc8NzC8zjfMlbgGgC2ED2OjfcUPhaYQESNEA8s0spmjngz30kYb7URUdHrcv4CJqEe\\\\\\\/6eldVH6JYmuG3C2tuIpdgsDuHFMLJJSkK4veNLf85f3ZNG9R3VjgsJyzoPCEClaxTDDEGdIZo+bfSYWgWFWZA+anYvJN2vxEOfgSug1lIM5p41TWFo010cb0Cm6mvso4Q2r2Aj2ORwb3F2kmnjf0ztAJV5tBBTUnMa6V7bTBQj9fAyNO0Du6+uNbfuBWSFGdR1Y1WMdZ2oqGcqXortdtzf\\\\\\\/8c1y98hU5hlXtrIxV1O2hxOd6J5xII2McMCkz4gRwhJyLpn420HXMLA\\\\\\\/+Ncl+qKS\\\\\\\/IHasBDvhkq3r7ki4kp78PKJvxaH6ronn98vy3Rb1S0jVyIO4DXpoPDX+9UjpqSQrWe5zncALh2Fql4qhGy\\\\\\\/jNu5\\\\\\\/fVhyEYNuFJOPlnr5cxFx\\\\\\\/WlGsjf9R4VhgCp+hNKgIcDd\\\\\\\/7yFk6P6F53lCJcfxvhFlfASrm\\\\\\\/oYCrgQISAV08yMpdP7eXxWDWnok37g80bGpiSrm3Un6D9FPNBDQehlspgxbr6hXOgALU2ndhpXpV9Wgk8BS4QvWJ\\\\\\\/WVLH4kGKxpX9WLH1EvOhcjybsSWd\\\\\\\/AvIPmm99ixiDW2U705LWKFjSjzKs8RU6kfb\\\\\\\/MZhfuLeWHVsAejU1Dyhu4z5yET\\\\\\\/ikJETZH69dqjftoXoBCKWOI8MlIdAS+CvrfOrl726VNwgieyEfR7\\\\\\\/KFivcCbLe+MDeuwXcAC8gs9yILpJtKF7nBf463kzqmvNCBk4oIsOL2cSOIDWtd3PcbteY91cqglBndmkjOMYuThRdDiesGXMx6EPYFtUnGO44JKS7e5CXCITVCIgBYz+EVZZNCqUn0Y67TW4kSYCWSGMMIYnwsIyqir5uL1eEgyo99Zcrqbz4ZavrkN0Kr6c7FxiWmjJDurDnBfMWgnENBL0RNvrnJ9eC8lyKCWQxKJmWge9DAk8b2cFfcCRNqbhD3QwRRQT6aLehf6IxysY+uAZbAzLc0Pnf7AECRTaQI4sbrNkf6cDmoTtnVBmgSQRi8T+7IBjI7380V\\\\\\\/7BaO7R\\\\\\\/4iEo7bzzkn\\\\\\\/uMdY9KKStLPr8YOLa1rn7yj7nrVQNFLduYurtiGBq2\\\\\\\/m654LD+j\\\\\\\/bImd2NBpArDh4dDv5grPzcgfcH9K8IOwj3WU7SGDN29VFSA5fEnvuRrfIBhxj3ciJB0oFUR6eGFXJE3ASikberv7aWCT2rt91gGffif\\\\\\\/v2K\\\\\\\/gt4KfRQj2BJqdr3OCAAzYq1LG9sAcA+sJ1hqjls4SopfJKaaOjJa9CH8hQF+2bGOcjb0sU7Hny5OqhCQHOs5hJR16ShgmVr7KaN2YzR\\\\\\\/FZWIbgDZBpDWumSiDMEMdplD\\\\\\\/R5ziylyi88Pm\\\\\\\/v5erfPMfNSZFDFTI9It\\\\\\\/D2tDw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022o0C1DuvULliWXQOaRPDUkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229efd33e904f12d6d650428be40d0245088d436d80ac5e40272363352bdb3089d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460004 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1695" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W62e6iXy3SnRgbqa2KKXl6ujnJXHKAWkV438ldW2u7E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1695" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-q7o07aBJL_XzcjTrjTHKjrFMStYMIl06kJQRblCk08A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1685" title="A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1703" title="Differential pre-mRNA Splicing Alters the Transcript Diversity of Helitrons Between the Maize Inbred Lines" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460004" id="node460004" data-pisa="ggg;5/8/1695" data-pisa-master="ggg;g3.115.019554" data-apath="/ggg/5/8/1695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium and Inversion-Typing of the <em>Drosophila melanogaster</em> Genome Reference Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8095-3156" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Houle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5548-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eladio J. Márquez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1695-1701;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019554 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460004" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David Houle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Science, Florida State University, Tallahassee, Florida 32308</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BHoule%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houle%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BHoule%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8095-3156" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Houle</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhoule@bio.fsu.edu" class="" data-icon-position="" data-hide-link-title="0">dhoule@bio.fsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eladio J. Márquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Science, Florida State University, Tallahassee, Florida 32308</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eladio%2BJ.%2BM%C3%A1rquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%A1rquez%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEladio%2BJ.%2BM%25C3%25A1rquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5548-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eladio J. Márquez</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/8/1695" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460004" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460004/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1695.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460004" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460004/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1695.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460004" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460004/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1695.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460004" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460004/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/8/1695.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460004" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460004/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">We calculated the linkage disequilibrium between all pairs of variants in the <em>Drosophila</em> Genome Reference Panel with minor allele count ≥5. We used <em>r<sup>2</sup></em> ≥ 0.5 as the cutoff for a highly correlated SNP. We make available the list of all highly correlated SNPs for use in association studies. Seventy-six percent of variant SNPs are highly correlated with at least one other SNP, and the mean number of highly correlated SNPs per variant over the whole genome is 83.9. Disequilibrium between distant SNPs is also common when minor allele frequency (MAF) is low: 37% of SNPs with MAF &lt; 0.1 are highly correlated with SNPs more than 100 kb distant. Although SNPs within regions with polymorphic inversions are highly correlated with somewhat larger numbers of SNPs, and these correlated SNPs are on average farther away, the probability that a SNP in such regions is highly correlated with at least one other SNP is very similar to SNPs outside inversions. Previous karyotyping of the DGRP lines has been inconsistent, and we used LD and genotype to investigate these discrepancies. When previous studies agreed on inversion karyotype, our analysis was almost perfectly concordant with those assignments. In discordant cases, and for inversion heterozygotes, our results suggest errors in two previous analyses or discordance between genotype and karyotype. Heterozygosities of chromosome arms are, in many cases, surprisingly highly correlated, suggesting strong epsistatic selection during the inbreeding and maintenance of the DGRP lines.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/inversion" class="hw-term hw-article-keyword hw-article-keyword-inversion" rel="nofollow">inversion</a></li></ul><p id="p-4">The <em>Drosophila</em> Genome Reference Panel (DGRP) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Mackay <em>et al.</em> 2012</a>) is a set of sequenced inbred lines derived from a single outbred population of <em>Drosophila melanogaster</em>. The DGRP has been used for a series of genome-wide association (GWA) studies on a wide variety of phenotypes . Linkage (gametic-phase) disequilibrium (LD) is a challenge to all GWA studies, because it confounds the signal from variant sites (we call these SNPs for brevity) that cause phenotypic variation with those that are genetically correlated with the causal variant but that do not have effects on the phenotype. The nature of the DGRP in many ways minimizes the presence of LD relative to vertebrates or to line-cross-derived mapping populations . The DGRP lines are drawn from a natural population in Raleigh, North Carolina, with large effective size, as shown by the relatively low level of structure within the population. <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Mackay <em>et al.</em> (2012)</a> confirmed that the average LD drops very rapidly with distance between SNPs, to an average squared correlation <em>r</em><sup>2</sup> &lt; 0.2 at just 10 base pairs on the autosomes. This result might suggest that the overall impact of LD on GWAS results will be low.</p><p id="p-5">More detailed analyses (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Pool 2015</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cridland <em>et al.</em> 2013</a>) show evidence for potentially troublesome LD within the DGRP. <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> estimated that 2.7% of line pairs have relatedness greater than 5%, and 0.05% of pairs (11 pairs) are related at greater than 50% (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cridland <em>et al.</em> 2013</a>). A total of 16 alternate chromosomal inversion karyotypes are present in the DGRP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Langley <em>et al.</em> 2012</a>). The analysis by <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> suggests that three of these are fixed in seven or more lines. These more common inversion types are substantially differentiated from the Standard karyotypes and cause LD (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Langley <em>et al.</em> 2012</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> 2014</a>). <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Huang <em>et al</em> (2014)</a> also showed that rare SNPs have a substantial likelihood of being highly correlated with SNPs that are more than 1 kb distant even outside of inversions. Finally, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Pool (2015)</a> analyzed whether small genomic regions in the DGRP lines were likely to reflect African rather than European ancestry, following the joint colonization of North America by a mixture of <em>D. melanogaster</em> from these two source populations (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Duchen <em>et al.</em> 2013</a>). He found that approximately 20% of the DGRP genomes could be assigned African ancestry. In addition, there was significant LD between these African regions among different chromosomes, suggesting possible epistatic selection favoring genotypes from the same region.</p><p id="p-6">These findings document the problem of LD. However, to interpret associations between SNPs and phenotypes in the DGRP, we need to know whether particular SNPs that are implicated are correlated with other SNPs or inversions, and where those correlated sites are. We calculated the LD between all pairs of SNPs in the 205 Freeze2 DGRP lines and provide a comprehensive list of polymorphic sites in substantial LD with inversions and with other sites throughout the genome. The cytogenetic karyotype assignments by <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> do not always agree with other PCR-based or sequence-based assignments in other works (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Langley <em>et al.</em> 2012</a>), and we use genotypic data to investigate why these assignments differ.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">We used Freeze 2 genotype calls for the DGRP lines obtained from <a href="http://ftp://ftp.hgsc.bcm.edu/DGRP/freeze2_Feb_2013/vcf_files/">ftp://ftp.hgsc.bcm.edu/DGRP/freeze2_Feb_2013/vcf_files/</a>. We used only calls with genotypic phred scores ≥20 at sites with exactly two alternative types. For LD calculations, heterozygous calls were treated as missing data. We focused our attention on LD at focal sites with minor allele counts (MAC) of five or more, although we calculated correlations of these focal sites with those where MAC was three or more. We excluded sites where the number of missing calls was &gt;85. There are 2,659,276 focal sites with MAC ≥5, and 3,159,155 sites with MAC ≥3. For focal sites, the median minor allele frequency (MAF) was 0.13, and the median number of lines scored was 195 out of 205 possible.</p><p id="p-8">We parameterized LD as the product-moment correlation <em>r</em><sup>2</sup> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hill and Robertson 1966</a>). Our algorithm for finding such pairs is based on the fact that only SNPs with similar MAF could be highly correlated. The <em>r</em><sup>2</sup> value used as a cutoff dictates the degree of similarity possible; for rare SNPs, only sites with <em>r<sup>2</sup>p</em> &lt;MAF&lt;<em>p/r<sup>2</sup></em> can be correlated to that degree with the focal SNP. For reasons discussed below, we chose to use <em>r</em><sup>2</sup> ≥ 0.5 as our cutoff for LD. In general, the relationship between MAF and the maximum correlation is nonlinear, so we calculated an empirical estimate of how similar MAF values could be and yield <em>r</em><sup>2</sup> ≥ 0.5. Virtually every site had some missing calls, and the overlap of those missing calls is different for different sites necessitating an approximate solution to this limit problem. We first binned SNPs into frequency classes to the nearest 0.01. We then fit a quadratic equation to the upper limit of MAF that could be correlated at <em>r</em><sup>2</sup> &gt; 0.5 for SNPs at the upper limit of the bin.</p><p id="p-9">To calculate LD between all pairs of SNPs within this limit, we started with the lowest frequency bin, whose rounded frequency is <em>p<sub>b</sub></em>, and calculated correlations between all SNPs in the focal bin and those SNPs with MAF below the empirically determined limit <em>p</em><sub>b</sub> + 0.005 &lt; MAF &lt; 0.01 + 1.875<em>p</em><sub>b</sub> − 1.17(<em>p</em><sub>b</sub>)<sup>2</sup>. This process was repeated for each bin with larger MAF. We retained a list of all pairs of SNPs with <em>r<sup>2</sup></em> &gt; 0.5. SAS programs to calculate the MAF limits for a given <em>r<sup>2</sup></em> cutoff and for calculating and storing the identity of SNP pairs with high <em>r<sup>2</sup></em> are included in <a href="http://www.g3journal.org/content/suppl/2015/06/10/g3.115.019554.DC1/019554SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/10/g3.115.019554.DC1/019554SI.pdf">File S1</a>. This algorithm will miss a small number of highly correlated SNPs that have missing genotype information in many lines.</p><p id="p-10">Prior to performing the above calculation, we characterized the probability that a SNP correlated with a causal SNP will result in a false positive using simulated data. We simulated phenotypic effects on a set of correlated SNP genotypes drawn from the Freeze 1 genotype calls for 165 DGRP lines (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Mackay <em>et al.</em> 2012</a>; <a href="ftp://ftp.hgsc.bcm.edu/DGRP/freeze1_July_2010/snp_calls/">ftp://ftp.hgsc.bcm.edu/DGRP/freeze1_July_2010/snp_calls/</a>). We first identified all SNP pairs correlated at <em>r</em><sup>2</sup> &gt; 0.25 using an algorithm similar to that outlined above for the Freeze 2 data. We then drew 100 random focal SNPs that were each correlated at <em>r</em><sup>2</sup> &gt; 0.25 with at least one other SNP, which we call a SNP family. Ten focal SNPs were chosen from each MAF decile. In cases in which the focal SNP was correlated at <em>r</em><sup>2</sup> &gt; 0.25 with more than 100 SNPs (31% of all focal SNPs), we retained a random sample of 100 correlated SNPs. We simulated SNP phenotypic effects that explained 1% of the total phenotypic variance in a multivariate trait. Simulated data were analyzed using MANOVA, with SNP genotype as the sole predictor variable. MANOVA <em>P</em>-values were calculated using a chi-square approximation of Wilk’s Lambda.</p><p id="p-11"><a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> reported that three rare inversion karyotypes were fixed in seven or more DGRP lines [In(2L)t, In(2R)NS, In(3R)Mo], whereas no other karyotype was fixed in more than four lines. These karyotype assignments are sometimes in disagreement with the sequence-based assignments of karyotype reported by <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl (2012)</a> and <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Langley <em>et al.</em> (2012)</a>. We checked these characterizations statistically using the following approach. We assembled genotypic data from Freeze 2 as above, but including heterozygous assignments, and then we excluded SNPs with five or more missing genotype assignments. Missing assignments in the remaining SNPs were assigned to the common allele to provide complete genotypic data. Using the results from the genome-wide LD results, we then obtained a list of the SNPs that are inside the inversion breakpoints of the three common alternative karyotypes (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Corbett-Detig <em>et al.</em> 2012</a>) and that had LD <em>r<sup>2</sup></em> &gt; 0.5 with 200 or more other SNPs more than 100 kb sites distant. These SNPs are likely to be characteristic of inversion types. This provided a sample of 28,495 SNPs on chromosome 2L, 8174 on 2R, and 8347 on 3R. We conducted separate principal components analyses (PCA) of the genotypes of a randomly chosen subset of 5000 SNPs for each inversion and used the scores on PC1 to diagnose which genotypes are characteristic of each karyotype. We also calculated the proportion of SNPs that were scored as heterozygous for chromosomal regions defined by the inversion breakpoints.</p><p id="p-12">The African ancestry of each inverted region was calculated using the results of <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Pool (2015)</a>. In that work, table S3 lists regions identified as having a high probability of African ancestry based on applying the Hidden Markov Model of ancestry in the study by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Pool <em>et al.</em> (2012)</a> to the DGRP lines. We calculated the lengths of these regions that lay between the breakpoints of the three common inversions for each DGRP line.</p><p id="p-13">Calculations were performed in SAS version 9.3 for Windows and Unix (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">SAS Institute Inc. 2011</a>).</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><p id="p-14">To characterize LD in Freeze 2 of the DGRP, we considered all pairs of sites with MAC ≥5. These sites include indel variation, but we refer to them as SNPs for brevity. The overall magnitude of LD in Freeze 2 of the DGRP for 1 million pairs of random SNPs with MAC ≥5 is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. We use <em>r</em><sup>2</sup> as our measure of disequilibrium (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Hill and Robertson 1966</a>) because this is the most appropriate indicator of the likelihood that analyses of pairs of SNPs will yield similar results. More than 99.9% of all random SNP pairs are correlated at less than <em>r<sup>2</sup></em> ≤ 0.15, and only 0.0024% have <em>r<sup>2</sup></em> ≥ 0.5. There are, however, almost 3.5×10<sup>12</sup> SNP pairs for this data set, so the number of pairs correlated at any particular level is not small.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of LD between pairs of SNPs, measured as r2. (A) Distribution of LD among 1,000,000 random SNP pairs. Note the log10 scale. (B) Distribution of r2 values among all highly correlated pairs with MAC &#x2265;5 genome-wide." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-721867175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of LD between pairs of SNPs, measured as &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;. (A) Distribution of LD among 1,000,000 random SNP pairs. Note the log&lt;sub&gt;10&lt;/sub&gt; scale. (B) Distribution of &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; values among all highly correlated pairs with MAC &#x2265;5 genome-wide.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1695/F1.medium.gif" width="314" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/8/1695/F1.medium.gif" width="314" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466616" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Distribution of LD between pairs of SNPs, measured as <em>r</em><sup>2</sup>. (A) Distribution of LD among 1,000,000 random SNP pairs. Note the log<sub>10</sub> scale. (B) Distribution of <em>r</em><sup>2</sup> values among all highly correlated pairs with MAC ≥5 genome-wide.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">To determine what level of <em>r<sup>2</sup></em> we should focus on, we performed simulations in which we simulated a phenotypic effect at a focal SNP and then assessed the probability that correlated SNPs that do not themselves cause phenotypic variation would show a significant phenotypic effect, with the results shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. We simulated effect sizes to generate modest power of approximately 0.3 at a conservative <em>P</em>-value of 10<sup>−6</sup>, likely to be typical of many studies using the DGRP lines. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> shows that as long as <em>P</em>-values are not very liberal, <em>r<sup>2</sup></em> ≥ 0.5 will include the majority of SNPs likely to generate false positives due to LD. Consequently, we structured our calculations to ensure that the majority of correlations of <em>r<sup>2</sup></em> ≥ 0.5 would be detected, as described in <em>Materials and Methods</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probability that a SNP with no phenotypic effect will have a statistically significant association with phenotype as a function of LD with a SNP that does have a phenotypic effect. &#x3B1;= threshold for significance. The focal SNP explains 1% of the phenotypic variance in a multivariate trait. The right-most symbol is power to detect an association with the focal SNP. Values on the X-axis are the midpoints of bins of r2 values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-721867175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Probability that a SNP with no phenotypic effect will have a statistically significant association with phenotype as a function of LD with a SNP that does have a phenotypic effect. &#x3B1;= threshold for significance. The focal SNP explains 1% of the phenotypic variance in a multivariate trait. The right-most symbol is power to detect an association with the focal SNP. Values on the X-axis are the midpoints of bins of &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1695/F2.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/8/1695/F2.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466531" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Probability that a SNP with no phenotypic effect will have a statistically significant association with phenotype as a function of LD with a SNP that does have a phenotypic effect. α= threshold for significance. The focal SNP explains 1% of the phenotypic variance in a multivariate trait. The right-most symbol is power to detect an association with the focal SNP. Values on the X-axis are the midpoints of bins of <em>r</em><sup>2</sup> values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We refer to pairs of SNPs correlated at <em>r<sup>2</sup></em> ≥ 0.5 as “highly correlated,” and we refer to SNPs more than 100 kb apart as “distant.” To provide a comprehensive picture of LD for these SNPs, we also calculated LD with SNPs that had a MAC as low as three. A total of 1.36×10<sup>8</sup> highly correlated pairs were detected. More importantly, 76% of all SNPs with MAC ≥5 were highly correlated with at least one other SNP. Seventeen percent of all SNPs are highly correlated with at least one distant SNP, and 7.7% are highly correlated with a SNP on another chromosome. The distribution of <em>r<sup>2</sup></em> values for highly correlated pairs where both have MAC ≥5 is shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>. Due to the predominance of rare minor alleles (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Mackay <em>et al.</em> 2012</a>), 38% of all high correlations for sites with MAC ≥5 involve sites with MAC &lt;5. We did not include these in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, but they have a strong mode of approximately <em>r</em><sup>2</sup> ≈ 0.6 due to SNPs that match at all but one site. The full lists of highly correlated SNPs (including those between sites with MAC ≥5 and sites with MAC &lt;5) by chromosome arm are available in <a href="http://www.g3journal.org/content/suppl/2015/06/10/g3.115.019554.DC1/019554SI.pdf">File S2</a>.</p><p id="p-19"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the probability that a SNP is highly correlated with at least one other SNP. SNPs were classified as inside or outside the breakpoint of the three inversions fixed in more than four DGRP lines [In(2L)t, In(2R)NS, and In(3R)Mo]. For local disequilibrium, it makes little difference whether a SNP is in an inversion or not, but SNPs in inversions are more likely to be in high LD with a SNP that is distant (in the Standard gene order). The inversion karyotypes themselves have frequencies of 0.1 or less (see below), precluding SNPs characteristic of inversions from being highly correlated with high MAF variants. More importantly, simply excluding SNPs within inversions does not appreciably reduce the likelihood that a variant will be highly correlated with at least some other SNPs, nor does it preclude those highly correlated SNPs from being distant.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probability that a focal site is correlated at r2 &gt; 0.5 with at least one other site in the genome as a function of location relative to inversions. Inversion sites lie between the breakpoints defined by Corbett-Detig et al. (2012), with the exception of the distal segment of chromosome 3R, which was treated as part of In(3R)Mo (Corbett-Detig and Hartl 2012)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-721867175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Probability that a focal site is correlated at &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5 with at least one other site in the genome as a function of location relative to inversions. Inversion sites lie between the breakpoints defined by Corbett-Detig &lt;em&gt;et al.&lt;/em&gt; (2012), with the exception of the distal segment of chromosome 3R, which was treated as part of In(3R)Mo (Corbett-Detig and Hartl 2012).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1695/F3.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/8/1695/F3.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466566" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Probability that a focal site is correlated at <em>r</em><sup>2</sup> &gt; 0.5 with at least one other site in the genome as a function of location relative to inversions. Inversion sites lie between the breakpoints defined by <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Corbett-Detig <em>et al.</em> (2012)</a>, with the exception of the distal segment of chromosome 3R, which was treated as part of In(3R)Mo (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the mean and median numbers of SNPs highly correlated with a focal SNP for all pairs of SNPs and all distant pairs. The numbers of highly correlated pairs are substantially higher at low MAF. The difference is particularly large for distant SNPs. When MAF &lt;0.1, 37% of SNPs are correlated with a distant variant; when MAF is between 0.1 and 0.2, 5% of SNPs are correlated with a distant variant. However, the mean number of highly correlated SNPs is still substantial at all allele frequencies. This is consistent with random disequilibrium due to the very large number of SNPs with low MAF (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Mackay <em>et al.</em> 2012</a>), and due to the smaller number of permutations that can lead to a low MAF. We refer to this as rarity disequilibrium. Of the 2.6 million SNPs with MAC ≥5 in this analysis, 50% have MAF &lt;0.129, and 25% have MAF &lt;0.054. <a href="http://www.g3journal.org/content/suppl/2015/06/10/g3.115.019554.DC1/FigureS1.pdf">Figure S1</a> suggests that inversions may compound the effects of linkage and rarity disequilibrium, because the mean number of highly correlated SNPs inside inversions is substantially higher when MAF is less than 0.2. This is particularly so for SNPs distant from the focal variant.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean and median number of sites correlated with variant sites at r2 &gt; 0.5 as a function of minor allele frequency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-721867175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mean and median number of sites correlated with variant sites at &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5 as a function of minor allele frequency.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1695/F4.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/8/1695/F4.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466557" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-22" class="first-child">Mean and median number of sites correlated with variant sites at <em>r</em><sup>2</sup> &gt; 0.5 as a function of minor allele frequency.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The variance in the number of correlated SNPs is high and skewed toward smaller numbers, so that the mean number of correlated SNPs is quite a bit higher than the median, as shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. Nevertheless, the median number of correlated SNPs across the genome ranges from 20 for most of the lowest MAF classes to 2 for the high MAF SNPs. The median number of SNPs greater than 10<sup>5</sup> bp away that are in high LD is 1 or more when MAF is 0.04 or less, but 0 for all higher frequencies. The median number of variants correlated at <em>r<sup>2</sup></em> &gt; 0.5 is virtually identical between SNPs inside and outside of inversions, regardless of MAF (not shown).</p><p id="p-24">The mean number of highly correlated distant sites across the genome is shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. There is a clear peak of distant LD in In(2R)NS and near the breakpoints in In(2L)t. There is also a broader, but less intense, peak of distant LD in the neighborhood of In(3R)Mo; however, for this arm, high LD extends beyond the proximal and distal breakpoints of this inversion. The high LD between the distal end of 3R and In(3R)Mo has been noted previously (<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>). Other low recombination regions near telomeres and centromeres also show high distant LD.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean number of highly correlated distant sites/SNP in 100 kb regions across the genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-721867175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean number of highly correlated distant sites/SNP in 100 kb regions across the genome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1695/F5.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/8/1695/F5.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466588" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">Mean number of highly correlated distant sites/SNP in 100 kb regions across the genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The cytogenetic analyses of <a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> found at least seven DGRP lines fixed for each of the common cosmopolitan inversions In(2L)t, In(2R)NS, and In(3R)Mo, and many more lines that were heterozygous for these karyotypes. The remaining inversion karyotypes were fixed in four or fewer lines. Two previous studies (<a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Langley <em>et al.</em> 2012</a>) also inversion-typed a subset of the DGRP lines using a PCR-based and/or a next-generation sequence-based assay. These two studies were consistent in their assignments, so we refer to them collectively as CD-L. CD-L and <a id="xref-ref-7-10" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> both scored 501 chromosome arms for homozygotes of the three common inversions In(2L)t, In(2R)NS, and In(3R)Mo. Ten of these assignments are in conflict, and 459 are in agreement. The remaining 32 were scored as heterozygotes by <a id="xref-ref-7-11" class="xref-bibr" href="#ref-7">Huang <em>et al</em>. (2014)</a> but were not examined by <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Langley <em>et al.</em> (2012)</a>. <a id="xref-ref-1-9" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl (2012)</a> state they did detect heterozygotes for inversions, but they only reported lines positively identified as inversion homozygotes.</p><p id="p-27">Given the pattern of distant LD shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, and because each of these previous studies of inversion type makes clear that inversion karyotypes are usually substantially differentiated from each other, we reasoned that SNPs within the inversion breakpoints that show high LD with a large number of distant SNPs will tend to be diagnostic for inversion type, and identify the likely source of discrepancies between previous karyotype assignments. After selecting SNPs with nearly complete genotypic data that are also in high LD with many other SNPs, we performed principal components analyses on those SNPs located within the breakpoints of each of the three common inversions.</p><p id="p-28">The full list of previous karyotype inferences, scores on the first PC for inversion-diagnostic genotypes for each chromosome, and heterozygosities of all SNPs within regions defined by inversion breakpoints are given in <a href="http://www.g3journal.org/content/suppl/2015/06/10/g3.115.019554.DC1/019554SI.pdf">File S3</a>. We plot the PC1 scores <em>vs.</em> the average heterozygosity (H) of each inversion region in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. In all but one of the 458 cases where <a id="xref-ref-7-12" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> and CD-L both reported a homozygous karyotype, scores on PC1 predict inversion type. The exception is line RAL332 for chromosome 3R, where both <a id="xref-ref-7-13" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> and CD-L infer the Standard arrangement, whereas PC1 score and the observed heterozygosity predict a Standard/In(3R)Mo heterozygote. Intermediate scores on PC1 are found in chromosome arms identified as inversion heterozygotes by <a id="xref-ref-7-14" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> with four exceptions; PC1 score for line RAL325, chromosome 2R indicates a Standard/In(2R)NS heterozygote, and <a id="xref-ref-7-15" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> reported two different inversions as heterozygotes, In(2R)Y6 and In(2R)Y7. RAL 409 is anomalous in having PC scores suggesting heterozygotes for In(2R)NS and In(3R)Mo, but fairly low heterozygosity. It was scored as a homozygote for In(2R)NS and In(3R)Mo by both <a id="xref-ref-7-16" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> and CD-L.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inferred karyotype and heterozygosity of DGRP lines for common inversions. DGRP line numbers indicated for anomalous cases. Predictions based on our principal components analyses are shown by symbol shapes: squares, inversion homozygotes; circles, Standard homozygotes; and triangles, inversion heterozygotes. Colors reference the states inferred in Huang et al. (2014) and CD-L: red, inversion homozygote in both; blue, standard homozygote in both; lavender, heterozygous in Huang et al. (2014) (CD-L in most cases did not scores heterozygotes); yellow, Standard homozygote in Huang et al. (2014) but inversion homozygote in CD-L (excluding RAL48, which was not scored by CD-L); cyan, inversion homozygote in Huang et al. (2014) but Standard homozygote in CD-L; green, predicted heterozygote for a rare inversion [line 373 for 3R was predicted as In(3R)Mo homozygote by CD-L]; white, Standard homozygote in Huang et al. (2014) but In(2R)NS homozygote by CD-L. Note that separate PC analyses were performed separately for each inverted region, so the scale of PC1 scores for each chromosome arm is different." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-721867175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inferred karyotype and heterozygosity of DGRP lines for common inversions. DGRP line numbers indicated for anomalous cases. Predictions based on our principal components analyses are shown by symbol shapes: squares, inversion homozygotes; circles, Standard homozygotes; and triangles, inversion heterozygotes. Colors reference the states inferred in Huang &lt;em&gt;et al.&lt;/em&gt; (2014) and CD-L: red, inversion homozygote in both; blue, standard homozygote in both; lavender, heterozygous in Huang &lt;em&gt;et al.&lt;/em&gt; (2014) (CD-L in most cases did not scores heterozygotes); yellow, Standard homozygote in Huang &lt;em&gt;et al.&lt;/em&gt; (2014) but inversion homozygote in CD-L (excluding RAL48, which was not scored by CD-L); cyan, inversion homozygote in Huang &lt;em&gt;et al.&lt;/em&gt; (2014) but Standard homozygote in CD-L; green, predicted heterozygote for a rare inversion [line 373 for 3R was predicted as In(3R)Mo homozygote by CD-L]; white, Standard homozygote in Huang &lt;em&gt;et al.&lt;/em&gt; (2014) but In(2R)NS homozygote by CD-L. Note that separate PC analyses were performed separately for each inverted region, so the scale of PC1 scores for each chromosome arm is different.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1695/F6.medium.gif" width="244" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/8/1695/F6.medium.gif" width="244" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1695/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466431" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-29" class="first-child">Inferred karyotype and heterozygosity of DGRP lines for common inversions. DGRP line numbers indicated for anomalous cases. Predictions based on our principal components analyses are shown by symbol shapes: squares, inversion homozygotes; circles, Standard homozygotes; and triangles, inversion heterozygotes. Colors reference the states inferred in <a id="xref-ref-7-17" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> and CD-L: red, inversion homozygote in both; blue, standard homozygote in both; lavender, heterozygous in <a id="xref-ref-7-18" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> (CD-L in most cases did not scores heterozygotes); yellow, Standard homozygote in <a id="xref-ref-7-19" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> but inversion homozygote in CD-L (excluding RAL48, which was not scored by CD-L); cyan, inversion homozygote in <a id="xref-ref-7-20" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> but Standard homozygote in CD-L; green, predicted heterozygote for a rare inversion [line 373 for 3R was predicted as In(3R)Mo homozygote by CD-L]; white, Standard homozygote in <a id="xref-ref-7-21" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> but In(2R)NS homozygote by CD-L. Note that separate PC analyses were performed separately for each inverted region, so the scale of PC1 scores for each chromosome arm is different.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">A number of inversion regions have highly heterozygous sequence data (H &gt;0.15) but no evidence of similarity to common inversion genotypes. In four cases (shown in green in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), these were identified as heterozygotes for rare inversion karyotypes by <a id="xref-ref-7-22" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a>. Line RAL303 was scored as an inversion heterozygote for both In(2L)t and In(2R)NS by <a id="xref-ref-7-23" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a>, but it does not have a genotype characteristic of either heterozygote. Fourteen lines for which neither PC1 scores nor <a id="xref-ref-7-24" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> suggest inversion heterozygosity are highly heterozygous in the region of In(3R)Mo, which may suggest the presence of balanced polymorphism not associated with an inversion.</p><p id="p-31">There are a total of six cases where CD-L and <a id="xref-ref-7-25" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> assigned different homozygous karyotypes to the same lines; in three cases PC1 scores are consistent with CD-L, whereas PC1 scores and <a id="xref-ref-7-26" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> are in agreement for the other three. <a id="xref-ref-7-27" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a> reported an additional five cases of karyotypic heterozygosity that do not have elevated sequence heterozygosity.</p><p id="p-32">Inversions In(2L)t and In(2R)NS both have African origin, whereas In(3R)Mo has a non-African origin (<a id="xref-ref-1-10" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>). <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Pool (2015)</a> inferred homozygous regions in each DGRP line that were likely to have African ancestry, and we used the total length of these in each inverted region to provide an additional check on our inversion-typing, with results shown in <a href="http://www.g3journal.org/content/suppl/2015/06/10/g3.115.019554.DC1/TableS1.pdf">Table S1</a>. Mean African ancestry for our predicted inversion types agrees well with expectations for In(2L)t and In(3R)Mo. For In(2R)NS, however, the two mismatches between our predictions and previous studies both have African ancestry more consistent with the opposite type. The highest African ancestry in the 187 consensus Standard karyotypes is 920 kb, whereas line 83 has African identity for 5840 kb. The lowest African ancestry in the 7 consensus In(2R)NS lines is 3286 kb, whereas line 69 has African identity for just 874 kb. These are both cases where the sequence-based typing of <a id="xref-ref-1-11" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl (2012)</a> is in disagreement with our predictions and karyotyping by <a id="xref-ref-7-28" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> (2014)</a>. These two lines may represent recent double recombinants that separate the In(2R)NS breakpoints from their typical genotype.</p><p id="p-33">Heterozygosities of segments of chromosome arms defined by common inversion breakpoints are correlated as shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Segments of the same arm always have correlations of 0.86 or above. Segments of different arms of the same chromosome also remain highly correlated. These results suggest that there is strong selection against recombinants and segregants in some of the DGRP lines. It is particularly striking that X-chromosome heterozygosity is significantly correlated with the heterozygosity of chromosome 3.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466542/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466542/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Pearson correlations of heterozygosities for regions of chromosome arms</span><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-3"><h2 class="">Discussion</h2><p id="p-36">Our calculations have identified variant (SNP) pairs in 205 lines in the Freeze 2 data set that have LD above <em>r</em><sup>2</sup> &gt; 0.5. This reveals both the overall patterns of LD and will facilitate analyses that attempt to disentangle which nucleotides cause phenotypic effects.</p><p id="p-37">Most SNPs in the <em>Drosophila</em> Genome Reference Project are in strong LD (LD of <em>r<sup>2</sup></em> ≥ 0.5) with at least one other variant. More strikingly, many SNPs in the full DGRP with minor allele frequency less than 0.2 are highly correlated with at least one SNP more than 100 kbp distant. Thus, although it is true that the DGRP population has low LD relative to other eukaryotes, disequilibrium is still a major element of these data, and careful consideration should be given to its impact at all stages of an association analysis.</p><p id="p-38">The LD in the DGRP seems to reflect several possible causes (<a id="xref-ref-7-29" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Pool 2015</a>). First, population-wide LD persists among closely linked SNPs because the recombination events that break down LD are insufficiently rare to counteract the processes such as past admixture, stochastic mutation, drift, and potential natural selection. The signature of these events in the DGRP is that local LD remains appreciable throughout the range of SNP frequencies and is particularly high in regions of reduced recombination. This suggests that local LD would also be found in larger samples of genotypes. The North American population of <em>D. melanogaster</em> from which the DGRP lines were sampled was founded by admixture of African and European genotypes (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Duchen <em>et al.</em> 2013</a>), and short haplotypes (median, 0.17 cM or 80 kb) of African ancestry can be identified in the DGRP lines (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Pool 2015</a>).</p><p id="p-39">Second, variants with low MAF are on average highly correlated with multiple SNPs throughout the genome. The likely cause of this is random sampling of the very large number of low MAF variants in a relatively small sample of lines. We call this rarity disequilibrium. GWAS analyses often presume that only local LD needs to be considered, but this is not true for variants with MAF less than approximately 0.2 for the DGRP. This source of disequilibrium would be sensitive to the sample size of lines used. Larger samples of genotypes expand the number of possible combinations of lines that would allow a particular minor allele count, and thus confine rarity disequilibrium to a smaller range of MAF. Conversely, smaller samples of subsets of the DGRP lines would intensify rarity disequilibrium.</p><p id="p-40">Third, the presence of inversions allows differentiation of genotypes carried by each inversion, in turn creating additional LD. The rarity of alternative karyotypes means that this source of LD intensifies the degree of LD already present due to rarity disequilibrium in the DGRP.</p><p id="p-41">Somewhat more speculatively, <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Pool (2015)</a> has shown that haplotypes inferred to be of African origin in the DGRP have significant positive LD at a genome-wide scale. This finding is difficult to relate quantitatively to our results, because <a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Pool (2015)</a> did not report a measure of LD effect size, so it is possible that the degree of LD detected in his study is below the <em>r</em><sup>2</sup> ≥ 0.5 threshold that we used. Nevertheless, the results of <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">Pool (2015)</a> do suggest that epistatic selection in the Raleigh population could be strong enough to generate LD throughout the genome. For example, if African genotypes at widely spaced genes together enabled an adaptive response to a common environmental challenge while temperate genotypes at those same loci were favored in the alternative challenge, this could leave a persistent signature of LD. For example, perhaps multi-locus African genotypes are resistant to high temperatures and multi-locus temperate genotypes are resistant to cold shock. A joint SNP-level analysis of African origin and LD could be very informative.</p><p id="p-42">Similarly, the pattern of correlations in heterozygosity among chromosomal regions we observe (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) suggests that some of the long-distance or interchromosomal disequilibrium that we have detected may reflect epistatic selection during the process of inbreeding these lines. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Corbett-Detig <em>et al.</em> (2013)</a> observed that some genotypic combinations in regions distant from each other are observed less frequently than expected in recombinant inbred lines in <em>D. melanogaster</em>, as well as other species. The correlations of heterozygosities we detected are the converse of this pattern, but are consistent with selection creating long-distance disequilibrium during the process of inbreeding.</p><p id="p-43">When the karyotypic assignments from three previous studies are in agreement (<a id="xref-ref-7-30" class="xref-bibr" href="#ref-7">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-1-12" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Langley <em>et al.</em> 2012</a>), our genotype-based assignments of inversion type are concordant, except for three chromosome arms (3R in RAL332, 2R and 3R in RAL409). RAL332 is scored as an inversion heterozygote on the basis of our analysis, and homozygous Standard is scored by <a id="xref-ref-7-31" class="xref-bibr" href="#ref-7">Huang <em>et al</em>. (2014)</a> and <a id="xref-ref-1-13" class="xref-bibr" href="#ref-1">Corbett-Detig and Hartl (2012)</a>. This could be due to the loss of the In(3R)Mo after sequencing. The remaining discrepancies between our results and the other scorings cannot be explained on this basis and are likely either errors in these previous assignments or perhaps recombination events that have separated karyotype and genotype. Line 409 has two arms previously scored as inverted that have PC scores intermediate between Standard and Inverted types, but no elevated heterozygosity. These arms, plus 2R from RAL 377, could represent recent double recombination events. Similarly, for the two lines that were both homozygous for arm 2R and assigned an In(2R)NS inversion status in conflict with the analysis of inversion breakpoints (RAL69 and 83) by Corbett-Detig and Hartl, each has African ancestry more typical of the alternative karyotype.</p><p id="p-44">Regardless of whether the discrepancies between LD-based genotyping and karyotypic assignments are caused by errors or recombination, our genotypic typing is relevant for those performing association studies, as it summarizes similarity of genotype and therefore phenotypic effects of the genotypes captured by each inversion. Inversion breakpoints could themselves cause a phenotypic effect, and these are particularly likely to have been involved in the initial spread of rare karyotypes. Coevolution of karyotype and genotype since that time are likely to have altered those initial effects. Our results enable analyses to test whether our LD-based assignments and those based on other criteria give concordant results.</p><p id="p-45">Lines with highly heterozygous regions are potentially more troublesome, because loss of heterozygosity between sequencing and phenotyping cannot be ruled out without additional analysis. This appears to be the most likely explanation for the conflicting evidence concerning the inversion type of 3R in line RAL332. Association studies should therefore test whether results are affected by inclusion of heterozygous variants and regions that are likely to have diverged in genotype from the sequence data before they are phenotyped.</p><p id="p-46">Our LD calculations will not apply precisely to most association studies based on the DGRP, because each study is likely to use a different subset of lines for phenotyping. We have also calculated the correlations for the 184 lines that we have data for in our own association study (E.J. Márquez, W. Pitchers, J. Nye, D. Houle, and I. Dworkin unpublished results). These results show that differences in the genotypes chosen can substantially change the inferred LD structure for rarer SNPs. Nevertheless, the correlations that we have calculated will be useful as the basis for analyses of multi-SNP associations. For example, after identifying a set of SNPs with significant associations, one could reanalyze those in multi-SNP analyses that include the most highly correlated SNPs to diagnose which SNPs are most likely to represent the variants that cause phenotypic differences, and which have their signal confounded with those from other SNPs. If such SNPs are all nearby, then the inference of that genomic region as causal can be strong, even if the precise nucleotide responsible remains unknown. Follow-up studies of such regions are likely to be worthwhile. In contrast, SNPs for which the addition of distant SNPs renders effects ambiguous would be poor candidates for follow-up studies.</p></div><div class="section" id="sec-4"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">This work was supported by NIH 1R01GM094424-01. We thank two anonymous reviewers for thoughtful comments. Computing support was provided by the Research Computing Center at Florida State University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-48">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019554/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019554/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-49"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-02-04"><span class="received-label">Received </span>February 4, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-06-08"><span class="accepted-label">Accepted </span>June 8, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Houle and Márquez</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.1" data-doi="10.1371/journal.pgen.1003056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of inversion polymorphisms in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003056%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003056%26rft_id%253Dinfo%253Apmid%252F23284285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284285&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.2" data-doi="10.1534/genetics.112.141622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequence-based detection and breakpoint assembly of polymorphic inversions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141622%26rft_id%253Dinfo%253Apmid%252F22673805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMTMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.3" data-doi="10.1038/nature12678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic incompatibilities are widespread within species.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">504</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D504%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12678%26rft_id%253Dinfo%253Apmid%252F24196712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24196712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327851700046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.4" data-doi="10.1093/molbev/mst129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cridland</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Abundance and distribution of transposable elements in two Drosophila QTL mapping resources.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2311</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst129%26rft_id%253Dinfo%253Apmid%252F23883524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMC8yMzExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.5" data-doi="10.1534/genetics.112.145912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duchen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Živković</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Demographic inference reveals African and European admixture in the North American Drosophila melanogaster population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145912%26rft_id%253Dinfo%253Apmid%252F23150605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzEvMjkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.6" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.7" data-doi="10.1101/gr.171546.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rámia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in genome architecture among 205 Drosophila melanogaster genetic reference panel lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171546.113%26rft_id%253Dinfo%253Apmid%252F24714809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNjk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.8" data-doi="10.1534/genetics.112.142018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic variation in natural populations of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142018%26rft_id%253Dinfo%253Apmid%252F22673804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE2OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.9" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1695.10" data-doi="10.1101/014837"><div class="cit-metadata"><cite>Pool, J.E., 2015 Natural selection shapes the mosaic ancestry of the Drosophila genetic reference panel and the D. melanogaster reference genome. bioRxiv<span class="cit-pub-id-sep cit-pub-id-doi-sep"> 10.1101/014837.</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/014837</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1101%252F014837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwMTQ4Mzd2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNjk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1695.11" data-doi="10.1371/journal.pgen.1003080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of sub-Saharan Drosophila melanogaster: African diversity and non-African admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003080%26rft_id%253Dinfo%253Apmid%252F23284287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284287&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-5.8.1695.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">SAS Institute, Inc.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source"><em>The SAS System for Windows</em>, <em>Release 9.3</em></span>, <span class="cit-publ-name">SAS Institute</span>, <span class="cit-publ-loc">Cary, NC</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/8/1695.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1685" title="A Consensus Genetic Map for Pinus taeda and Pinus elliottii and Extent of Linkage Disequilibrium in Two Genotype-Phenotype Discovery Populations of Pinus taeda" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1703" title="Differential pre-mRNA Splicing Alters the Transcript Diversity of Helitrons Between the Maize Inbred Lines" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/8"><span>Volume 5 Issue 8, August 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/8.cover-source.jpg?itok=wjPUfMS9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1695" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UMmhvqgHBdiCnoCNqaiGncASAksazQFZibrfqxzqJ-s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/8/1695" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/8/1695" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/8/1695" class="active" data-icon-position="" data-hide-link-title="0">Linkage Disequilibrium and Inversion-Typing of the Drosophila melanogaster Genome Reference Panel</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460004" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_WWhhdOTMunRN0rgZrD4_pBegGF4R4dRtkmLy6gMi-I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1695.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/8/1695" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-XjpebPVBZnyk1bWjYwudX0wPF2ZNHdmBVjdyam1Yjpg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/8/1695" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1695.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460004" id="node460004--2" data-pisa="ggg;5/8/1695" data-pisa-master="ggg;g3.115.019554" data-apath="/ggg/5/8/1695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium and Inversion-Typing of the <em>Drosophila melanogaster</em> Genome Reference Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8095-3156" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Houle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5548-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eladio J. Márquez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1695-1701;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019554 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460004--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David Houle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Science, Florida State University, Tallahassee, Florida 32308</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BHoule%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houle%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BHoule%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8095-3156" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Houle</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhoule@bio.fsu.edu" class="" data-icon-position="" data-hide-link-title="0">dhoule@bio.fsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eladio J. Márquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Science, Florida State University, Tallahassee, Florida 32308</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eladio%2BJ.%2BM%C3%A1rquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%A1rquez%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEladio%2BJ.%2BM%25C3%25A1rquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5548-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eladio J. Márquez</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/8/1695" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/8/1695&amp;title=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/8/1695&amp;title=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/8/1695&amp;title=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/8/1695&amp;text=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/8/1695&amp;title=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/8/1695&amp;t=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/8/1695&amp;title=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/8/1695&amp;title=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460004/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460004" id="node460004--3" data-pisa="ggg;5/8/1695" data-pisa-master="ggg;g3.115.019554" data-apath="/ggg/5/8/1695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium and Inversion-Typing of the <em>Drosophila melanogaster</em> Genome Reference Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8095-3156" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Houle</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5548-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eladio J. Márquez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1695-1701;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019554 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460004--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David Houle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Science, Florida State University, Tallahassee, Florida 32308</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BHoule%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houle%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BHoule%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8095-3156" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Houle</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhoule@bio.fsu.edu" class="" data-icon-position="" data-hide-link-title="0">dhoule@bio.fsu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eladio J. Márquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Science, Florida State University, Tallahassee, Florida 32308</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eladio%2BJ.%2BM%C3%A1rquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%A1rquez%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEladio%2BJ.%2BM%25C3%25A1rquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5548-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eladio J. Márquez</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1695&amp;rft.epage=1701&amp;rft.atitle=Linkage%20Disequilibrium%20and%20Inversion-Typing%20of%20the%20Drosophila%20melanogaster%20Genome%20Reference%20Panel&amp;rft.volume=5&amp;rft.issue=8&amp;rft.date=2015-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Houle%2C+David&amp;rft.au=M%C3%A1rquez%2C+Eladio+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460004/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460004/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460004/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460004/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460004/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460004/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460004/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460004/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460004/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460004/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460004/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460004/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/8/1695" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/8/1695.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/8/1695.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/8/1695.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/8/1695.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
