<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/8/1721" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/8/1721.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/8/1721.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/8/1721.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/8/1721.atom" />
<meta name="HW.pisa" content="ggg;5/8/1721" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Gene Model Annotations for Drosophila melanogaster: Impact of High-Throughput Data" />
<meta name="DC.Identifier" content="10.1534/g3.115.018929" />
<meta name="DC.Date" content="2015-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Matthews et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We report the current status of the FlyBase annotated gene set for Drosophila melanogaster and highlight improvements based on high-throughput data. The FlyBase annotated gene set consists entirely of manually annotated gene models, with the exception of some classes of small non-coding RNAs. All gene models have been reviewed using evidence from high-throughput datasets, primarily from the modENCODE project. These datasets include RNA-Seq coverage data, RNA-Seq junction data, transcription start site profiles, and translation stop-codon read-through predictions. New annotation guidelines were developed to take into account the use of the high-throughput data. We describe how this flood of new data was incorporated into thousands of new and revised annotations. FlyBase has adopted a philosophy of excluding low-confidence and low-frequency data from gene model annotations; we also do not attempt to represent all possible permutations for complex and modularly organized genes. This has allowed us to produce a high-confidence, manageable gene annotation dataset that is available at FlyBase (&lt;http://flybase.org&gt;). Interesting aspects of new annotations include new genes (coding, non-coding, and antisense), many genes with alternative transcripts with very long 3′ UTRs (up to 15–18 kb), and a stunning mismatch in the number of male-specific genes (approximately 13% of all annotated gene models) vs. female-specific genes (less than 1%). The number of identified pseudogenes and mutations in the sequenced strain also increased significantly. We discuss remaining challenges, for instance, identification of functional small polypeptides and detection of alternative translation starts." />
<meta name="DC.Contributor" content="Beverley B. Matthews" />
<meta name="DC.Contributor" content="Gilberto dos Santos" />
<meta name="DC.Contributor" content="Madeline A. Crosby" />
<meta name="DC.Contributor" content="David B. Emmert" />
<meta name="DC.Contributor" content="Susan E. St. Pierre" />
<meta name="DC.Contributor" content="L. Sian Gramates" />
<meta name="DC.Contributor" content="Pinglei Zhou" />
<meta name="DC.Contributor" content="Andrew J. Schroeder" />
<meta name="DC.Contributor" content="Kathleen Falls" />
<meta name="DC.Contributor" content="Victor Strelets" />
<meta name="DC.Contributor" content="Susan M. Russo" />
<meta name="DC.Contributor" content="William M. Gelbart" />
<meta name="DC.Contributor" content="the FlyBase Consortium" />
<meta name="article:published_time" content="2015-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Gene Model Annotations for Drosophila melanogaster: Impact of High-Throughput Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We report the current status of the FlyBase annotated gene set for &lt;i&gt;Drosophila melanogaster&lt;/i&gt; and highlight improvements based on high-throughput data. The FlyBase annotated gene set consists entirely of manually annotated gene models, with the exception of some classes of small non-coding RNAs. All gene models have been reviewed using evidence from high-throughput datasets, primarily from the modENCODE project. These datasets include RNA-Seq coverage data, RNA-Seq junction data, transcription start site profiles, and translation stop-codon read-through predictions. New annotation guidelines were developed to take into account the use of the high-throughput data. We describe how this flood of new data was incorporated into thousands of new and revised annotations. FlyBase has adopted a philosophy of excluding low-confidence and low-frequency data from gene model annotations; we also do not attempt to represent all possible permutations for complex and modularly organized genes. This has allowed us to produce a high-confidence, manageable gene annotation dataset that is available at FlyBase (http://flybase.org). Interesting aspects of new annotations include new genes (coding, non-coding, and antisense), many genes with alternative transcripts with very long 3′ UTRs (up to 15–18 kb), and a stunning mismatch in the number of male-specific genes (approximately 13% of all annotated gene models) &lt;i&gt;vs.&lt;/i&gt; female-specific genes (less than 1%). The number of identified pseudogenes and mutations in the sequenced strain also increased significantly. We discuss remaining challenges, for instance, identification of functional small polypeptides and detection of alternative translation starts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/08/01" />
<meta name="citation_mjid" content="ggg;5/8/1721" />
<meta name="citation_id" content="5/8/1721" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/8/1721" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/8/1721.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/8/1721.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/8/1721.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.018929" />
<meta name="citation_pmid" content="26109357" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1721" />
<meta name="citation_lastpage" content="1736" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Beverley B. Matthews" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author_email" content="bmatthew@morgan.harvard.edu" />
<meta name="citation_author" content="Gilberto dos Santos" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Madeline A. Crosby" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="David B. Emmert" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Susan E. St. Pierre" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="L. Sian Gramates" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Pinglei Zhou" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Andrew J. Schroeder" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Kathleen Falls" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Victor Strelets" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Susan M. Russo" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="William M. Gelbart" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="the FlyBase Consortium" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author_institution" content="Department of Genetics, University of Cambridge, Cambridge CB2 3EH, United Kingdom" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_institution" content="Department of Biology, University of New Mexico, Albuquerque, New Mexico 87131" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. T. Aminetzach;citation_author=J. M. Macpherson;citation_author=D. A. Petrov;citation_title=Pesticide Resistance via Transposition-Mediated Adaptive Gene Truncation in Drosophila;citation_pages=764-767;citation_volume=309;citation_year=2005;citation_issue=5735;citation_pmid=16051794;citation_doi=10.1126/science.1112699" />
<meta name="citation_reference" content="citation_journal_title=BMC developmental biology [electronic resource];citation_journal_abbrev=BMC Dev Biol;citation_author=A. Bachmann;citation_author=M. Draga;citation_author=F. Grawe;citation_author=E. Knust;citation_title=On the role of the MAGUK proteins encoded by Drosophila varicose during embryonic and postembryonic development.;citation_pages=55-55;citation_volume=8;citation_year=2008;citation_pmid=18485238;citation_doi=10.1186/1471-213X-8-55" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=E. S. Balakirev;citation_author=F. J. Ayala;citation_title=The beta-esterase gene cluster of drosophila melanogaster: is psiEst-6 a pseudogene, a functional gene, or both?;citation_pages=165-179;citation_volume=121;citation_year=2004;citation_issue=2;citation_pmid=15330116;citation_doi=10.1023/B:GENE.0000040391.27307.51" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Batut;citation_author=A. Dobin;citation_author=C. Plessy;citation_author=P. Carninci;citation_author=T. R. Gingeras;citation_title=High-fidelity promoter profiling reveals widespread alternative promoter usage and transposon-driven developmental gene expression.;citation_volume=23;citation_year=2013;citation_pmid=22936248;citation_doi=10.1101/gr.139618.112" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=I. Behm-Ansmant;citation_author=I. Kashima;citation_author=J. Rehwinkel;citation_author=J. Sauliere;citation_author=N. Wittkopp;citation_title=mRNA quality control: an ancient machinery recognizes and degrades mRNAs with nonsense codons.;citation_pages=2845-2853;citation_volume=581;citation_year=2007;citation_issue=15;citation_pmid=17531985;citation_doi=10.1016/j.febslet.2007.05.027" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Berezikov;citation_author=N. Robine;citation_author=A. Samsonova;citation_author=J. O. Westholm;citation_author=A. Naqvi;citation_title=Deep annotation of Drosophila melanogaster microRNAs yields insights into their processing, modification, and emergence.;citation_volume=21;citation_year=2011;citation_pmid=21177969;citation_doi=10.1101/gr.116657.110" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Brennecke;citation_author=A. A. Aravin;citation_author=A. Stark;citation_author=M. Dus;citation_author=M. Kellis;citation_title=Discrete small RNA-generating loci as master regulators of transposon activity in Drosophila.;citation_pages=1089-1103;citation_volume=128;citation_year=2007;citation_issue=6;citation_pmid=17346786;citation_doi=10.1016/j.cell.2007.01.043" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. J. Brizuela;citation_author=L. Elfring;citation_author=J. Ballard;citation_author=J. W. Tamkun;citation_author=J. A. Kennison;citation_title=Genetic analysis of the brahma gene of Drosophila melanogaster and polytene chromosome subdivisions 72AB.;citation_pages=803-813;citation_volume=137;citation_year=1994;citation_issue=3;citation_pmid=7916308" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. B. Brown;citation_author=N. Boley;citation_author=R. Eisman;citation_title=Diversity and dynamics of the Drosophila transcriptome.;citation_pages=393-399;citation_volume=512;citation_year=2014;citation_pmid=24670639;citation_doi=10.1038/nature12962" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. E. Celniker;citation_author=L. A. Dillon;citation_author=M. B. Gerstein;citation_author=K. C. Gunsalus;citation_author=S. Henikoff;citation_title=Unlocking the secrets of the genome.;citation_pages=927-930;citation_volume=459;citation_year=2009;citation_issue=7249;citation_pmid=19536255;citation_doi=10.1038/459927a" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. Chapin;citation_author=H. Hu;citation_author=S. G. Rynearson;citation_author=J. Hollien;citation_author=M. Yandell;citation_title=In vivo determination of direct targets of the nonsense-mediated decay pathway in Drosophila.;citation_volume=4;citation_year=2014;citation_pmid=24429422;citation_doi=10.1534/g3.113.009357" />
<meta name="citation_reference" content="citation_journal_title=Differentiation; research in biological diversity;citation_journal_abbrev=Differentiation;citation_author=J. Chen;citation_author=G. G. Lakshmi;citation_author=D. L. Hays;citation_author=K. M. McDowell;citation_author=E. Ma;citation_title=Spatial and temporal expression of dADAR mRNA and protein isoforms during embryogenesis in Drosophila melanogaster.;citation_pages=312-320;citation_volume=78;citation_year=2009;citation_issue=5;citation_pmid=19720447;citation_doi=10.1016/j.diff.2009.08.003" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=G. L. Chew;citation_author=A. Pauli;citation_author=J. L. Rinn;citation_author=A. Regev;citation_author=A. F. Schier;citation_title=Ribosome profiling reveals resemblance between long non-coding RNAs and 5′ leaders of coding RNAs.;citation_volume=140;citation_year=2013;citation_pmid=23698349;citation_doi=10.1242/dev.098343" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=T. B. Chou;citation_author=Z. Zachar;citation_author=P. M. Bingham;citation_title=DEVELOPMENTAL EXPRESSION OF A REGULATORY GENE IS PROGRAMMED AT THE LEVEL OF SPLICING;citation_pages=4095-4104;citation_volume=6;citation_year=1987;citation_pmid=2832151" />
<meta name="citation_reference" content="citation_journal_title=The Rule-Benders;citation_title=Gene model annotations for Drosophila melanogaster;citation_author=M. A. Crosby;citation_author=L. S. Gramates;citation_author=G. dos Santos;citation_author=B. B. Matthews;citation_author=S. E. St. Pierre;citation_pages=1737-1749;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=V. Croset;citation_author=R. Rytz;citation_author=S. F. Cummins;citation_author=A. Budd;citation_author=D. Brawand;citation_title=Ancient protostome origin of chemosensory ionotropic glutamate receptors and the evolution of insect taste and olfaction.;citation_pages=e1001064-e1001064;citation_volume=6;citation_year=2010;citation_issue=8;citation_pmid=20808886;citation_doi=10.1371/journal.pgen.1001064" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Daines;citation_author=H. Wang;citation_author=L. Wang;citation_author=Y. Li;citation_author=Y. Han;citation_title=The Drosophila melanogaster transcriptome by paired-end RNA sequencing.;citation_volume=21;citation_year=2011;citation_pmid=21177959;citation_doi=10.1101/gr.107854.110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. dos Santos;citation_author=A. J. Schroeder;citation_author=J. L. Goodman;citation_author=V. B. Strelets;citation_author=M. A. Crosby;citation_title=FlyBase: Introduction of the Drosophila melanogaster Release 6 reference genome assembly and large-scale migration of genome annotations.;citation_volume=43;citation_year=2015;citation_issue=Database issue;citation_pmid=25398896;citation_doi=10.1093/nar/gku1099" />
<meta name="citation_reference" content="citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. D. Findlay;citation_author=M. J. Maccoss;citation_author=W. J. Swanson;citation_title=Proteomic discovery of previously unannotated, rapidly evolving seminal fluid genes in Drosophila;citation_pages=886-896;citation_volume=19;citation_year=2009;citation_issue=5;citation_pmid=19411605;citation_doi=10.1101/gr.089391.108" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=M. I. Galindo;citation_author=J. I. Pueyo;citation_author=S. Fouix;citation_author=S. A. Bishop;citation_author=J. P. Couso;citation_title=Peptides encoded by short ORFs control development and define a new eukaryotic gene family.;citation_pages=e106-e106;citation_volume=5;citation_year=2007;citation_issue=5;citation_pmid=17439302;citation_doi=10.1371/journal.pbio.0050106" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. R. Graveley;citation_author=A. N. Brooks;citation_author=J. W. Carlson;citation_author=M. O. Duff;citation_author=J. M. Landolin;citation_title=The developmental transcriptome of Drosophila melanogaster.;citation_pages=473-479;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21179090;citation_doi=10.1038/nature09715" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=Y. Guo;citation_author=S. Jangi;citation_author=M. A. Welte;citation_title=Organelle-specific control of intracellular transport: distinctly targeted isoforms of the regulator Klar.;citation_volume=16;citation_year=2005;citation_pmid=15647372;citation_doi=10.1091/mbc.E04-10-0920" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Harrison;citation_author=D. Milburn;citation_author=Z. Zhang;citation_author=P. Bertone;citation_author=M. Gerstein;citation_title=Identification of pseudogenes in the Drosophila melanogaster genome;citation_pages=1033-1037;citation_volume=31;citation_year=2003;citation_issue=3;citation_pmid=12560500;citation_doi=10.1093/nar/gkg169" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Harrow;citation_author=A. Frankish;citation_author=J. M. Gonzalez;citation_author=E. Tapanari;citation_author=M. Diekhans;citation_title=GENCODE: The reference human genome annotation for The ENCODE Project;citation_pages=1760-1774;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955987;citation_doi=10.1101/gr.135350.111" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. L. Harrow;citation_author=C. A. Steward;citation_author=A. Frankish;citation_author=J. G. Gilbert;citation_author=J. M. Gonzalez;citation_title=The vertebrate genome annotation browser 10 years on.;citation_volume=42;citation_year=2014;citation_issue=Database issue;citation_pmid=24316575;citation_doi=10.1093/nar/gkt1241" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. A. Hayden;citation_author=G. Bosco;citation_title=Comparative genomic analysis of novel conserved peptide upstream open reading frames in Drosophila melanogaster and other dipteran species.;citation_pages=61-61;citation_volume=9;citation_year=2008;citation_pmid=18237443;citation_doi=10.1186/1471-2164-9-61" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. Hilgers;citation_author=M. W. Perry;citation_author=D. Hendrix;citation_author=A. Stark;citation_author=M. Levine;citation_title=Neural-specific elongation of 3′ UTRs during Drosophila development.;citation_volume=108;citation_year=2011;citation_pmid=21896737;citation_doi=10.1073/pnas.1112672108" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Hiller;citation_author=S. Findeiss;citation_author=S. Lein;citation_author=M. Marz;citation_author=C. Nickel;citation_title=Conserved introns reveal novel transcripts in Drosophila melanogaster.;citation_volume=19;citation_year=2009;citation_pmid=19458021;citation_doi=10.1101/gr.090050.108" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. A. Hoskins;citation_author=J. M. Landolin;citation_author=J. B. Brown;citation_author=J. E. Sandler;citation_author=H. Takahashi;citation_title=Genome-wide analysis of promoter architecture in Drosophila melanogaster.;citation_volume=21;citation_year=2011;citation_pmid=21177961;citation_doi=10.1101/gr.112466.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. A. Hoskins;citation_author=J. W. Carlson;citation_author=K. H. Wan;citation_author=S. Park;citation_author=I. Mendez;citation_title=The Release 6 reference sequence of the Drosophila melanogaster genome.;citation_volume=25;citation_year=2015;citation_pmid=25589440;citation_doi=10.1101/gr.185579.114" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=S. Inagaki;citation_author=K. Numata;citation_author=T. Kondo;citation_author=M. Tomita;citation_author=K. Yasuda;citation_title=Identification and expression analysis of putative mRNA-like non-coding RNA in Drosophila;citation_pages=1163-1173;citation_volume=10;citation_year=2005;citation_issue=12;citation_pmid=16324153;citation_doi=10.1111/j.1365-2443.2005.00910.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. T. Ingolia;citation_author=L. F. Lareau;citation_author=J. S. Weissman;citation_title=Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes.;citation_pages=789-802;citation_volume=147;citation_year=2011;citation_issue=4;citation_pmid=22056041;citation_doi=10.1016/j.cell.2011.10.002" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=N. T. Ingolia;citation_author=G. A. Brar;citation_author=N. Stern-Ginossar;citation_author=M. S. Harris;citation_title=Ribosome profiling reveals pervasive translation outside of annotated protein-coding genes.;citation_pages=1365-1379;citation_volume=8;citation_year=2014;citation_pmid=25159147;citation_doi=10.1016/j.celrep.2014.07.045" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. E. Jepson;citation_author=M. Shahidullah;citation_author=A. Lamaze;citation_author=D. Peterson;citation_author=H. Pan;citation_title=dyschronic, a Drosophila homolog of a deaf-blindness gene, regulates circadian output and Slowpoke channels.;citation_pages=e1002671-e1002671;citation_volume=8;citation_year=2012;citation_issue=4;citation_pmid=22532808;citation_doi=10.1371/journal.pgen.1002671" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Jungreis;citation_author=M. F. Lin;citation_author=R. Spokony;citation_author=C. S. Chan;citation_author=N. Negre;citation_title=Evidence of abundant stop codon readthrough in Drosophila and other metazoa.;citation_volume=21;citation_year=2011;citation_pmid=21994247;citation_doi=10.1101/gr.119974.110" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. H. Kim;citation_author=S. L. Cotton;citation_author=D. Manna;citation_author=M. Welte;citation_title=Novel isoforms of the transport regulator klar.;citation_pages=e55070;citation_volume=8;citation_year=2013;citation_pmid=23457459;citation_doi=10.1371/journal.pone.0055070" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=T. Kondo;citation_author=Y. Hashimoto;citation_author=K. Kato;citation_author=S. Inagaki;citation_author=S. Hayashi;citation_title=Small peptide regulators of actin-based cell morphogenesis encoded by a polycistronic mRNA.;citation_pages=660-665;citation_volume=9;citation_year=2007;citation_issue=6;citation_pmid=17486114;citation_doi=10.1038/ncb1595" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Kozomara;citation_author=S. Griffiths-Jones;citation_title=miRBase: integrating microRNA annotation and deep-sequencing data.;citation_volume=39;citation_year=2011;citation_issue=Database issue;citation_pmid=21037258;citation_doi=10.1093/nar/gkq1027" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. F. Lin;citation_author=J. W. Carlson;citation_author=M. A. Crosby;citation_author=B. B. Matthews;citation_author=C. Yu;citation_title=Revisiting the protein-coding gene catalog of Drosophila melanogaster using 12 fly genomes.;citation_volume=17;citation_year=2007;citation_pmid=17989253;citation_doi=10.1101/gr.6679507" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. F. Lin;citation_author=I. Jungreis;citation_author=M. Kellis;citation_title=PhyloCSF: a comparative genomics method to distinguish protein coding and non-coding regions.;citation_volume=27;citation_year=2011;citation_pmid=21685081;citation_doi=10.1093/bioinformatics/btr209" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=E. Ladoukakis;citation_author=V. Pereira;citation_author=E. G. Magny;citation_author=A. Eyre-Walker;citation_author=J. P. Couso;citation_title=Hundreds of putatively functional small open reading frames in Drosophila.;citation_pages=R118-R118;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=22118156;citation_doi=10.1186/gb-2011-12-11-r118" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. G. Magny;citation_author=J. I. Pueyo;citation_author=F. M. Pearl;citation_author=M. A. Cespedes;citation_author=J. E. Niven;citation_title=Conserved regulation of cardiac calcium uptake by peptides encoded in small open reading frames.;citation_volume=341;citation_year=2013;citation_pmid=23970561;citation_doi=10.1126/science.1238802" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. M. Magwire;citation_author=F. Bayer;citation_author=C. L. Webster;citation_author=C. Cao;citation_author=F. M. Jiggins;citation_title=Successive increases in the resistance of Drosophila to viral infection through a transposon insertion followed by a Duplication.;citation_pages=e1002337-e1002337;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22028673;citation_doi=10.1371/journal.pgen.1002337" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. R. Mercer;citation_author=D. Wilhelm;citation_author=M. E. Dinger;citation_author=G. Solda;citation_author=D. J. Korbie;citation_title=Expression of distinct RNAs from 3′ untranslated regions.;citation_pages=2393-2403;citation_volume=3;citation_year=2011" />
<meta name="citation_reference" content="citation_author=S. Misra;citation_author=M. A. Crosby;citation_author=C. J. Mungall;citation_author=B. B. Matthews;citation_author=K. S. Campbell;citation_year=2002" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. W. Myers;citation_author=G. G. Sutton;citation_author=A. L. Delcher;citation_author=I. M. Dew;citation_author=D. P. Fasulo;citation_title=A Whole-Genome Assembly of Drosophila;citation_pages=2196-2204;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731133;citation_doi=10.1126/science.287.5461.2196" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Nechaev;citation_author=D. C. Fargo;citation_author=G. dos Santos;citation_author=L. Liu;citation_author=Y. Gao;citation_title=Global analysis of short RNAs reveals widespread promoter-proximal stalling and arrest of Pol II in Drosophila.;citation_volume=327;citation_year=2010;citation_pmid=20007866;citation_doi=10.1126/science.1181421" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=K. Okamura;citation_author=S. Balla;citation_author=R. Martin;citation_author=N. Liu;citation_author=E. C. Lai;citation_title=Two distinct mechanisms generate endogenous siRNAs from bidirectional transcription in Drosophila melanogaster.;citation_pages=581-590;citation_volume=15;citation_year=2008;citation_issue=6;citation_pmid=18500351;citation_doi=10.1038/nsmb.1438" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Pan;citation_author=L. Zhang;citation_title=Burst of young retrogenes and independent retrogene formation in mammals.;citation_pages=e5040-e5040;citation_volume=4;citation_year=2009;citation_issue=3;citation_pmid=19325906;citation_doi=10.1371/journal.pone.0005040" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=K. Patterson;citation_author=A. B. Molofsky;citation_author=C. Robinson;citation_author=S. Acosta;citation_author=C. Cater;citation_title=The functions of klarsicht and nuclear lamin in developmentally regulated nuclear migrations of photoreceptor cells in the Drosophila eye.;citation_volume=15;citation_year=2004;citation_pmid=14617811;citation_doi=10.1091/mbc.E03-06-0374" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=A. Pauli;citation_author=E. Valen;citation_author=A. F. Schier;citation_title=Identifying (non-)coding RNAs and small peptides: Challenges and opportunities.;citation_pages=103-112;citation_volume=37;citation_year=2014;citation_pmid=25345765" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. D. Pruitt;citation_author=T. Tatusova;citation_author=D. R. Maglott;citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins;citation_pages=D501-D504;citation_volume=33;citation_year=2005;citation_issue=Database issue;citation_pmid=15608248;citation_doi=10.1093/nar/gki476" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=M. J. Piatek;citation_author=A. Werner;citation_title=Endogenous siRNAs: regulators of internal affairs.;citation_volume=42;citation_year=2014;citation_pmid=25110021;citation_doi=10.1042/BST20140068" />
<meta name="citation_reference" content="citation_journal_title=J. Insect Sci.;citation_author=H. M. Robertson;citation_title=The insect chemoreceptor superfamily in Drosophila pseudoobscura: molecular evolution of ecologically-relevant genes over 25 million years.;citation_volume=9;citation_year=2009;citation_pmid=19613461;citation_doi=10.1673/031.009.1801" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. M. Robertson;citation_author=C. G. Warr;citation_author=J. R. Carlson;citation_title=Molecular evolution of the insect chemoreceptor gene superfamily in Drosophila melanogaster.;citation_volume=100;citation_year=2003;citation_pmid=14608037;citation_doi=10.1073/pnas.2335847100" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Savard;citation_author=H. Marques-Souza;citation_author=M. Aranda;citation_author=D. Tautz;citation_title=A segmentation gene in tribolium produces a polycistronic mRNA that codes for multiple conserved peptides.;citation_pages=559-569;citation_volume=126;citation_year=2006;citation_issue=3;citation_pmid=16901788;citation_doi=10.1016/j.cell.2006.05.053" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=K. A. Senti;citation_author=J. Brennecke;citation_title=The piRNA pathway: a fly's perspective on the guardian of the genome.;citation_pages=499-509;citation_volume=26;citation_year=2010;citation_issue=12;citation_pmid=20934772;citation_doi=10.1016/j.tig.2010.08.007" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Sisu;citation_author=B. Pei;citation_author=J. Leng;citation_author=A. Frankish;citation_author=Y. Zhang;citation_title=Comparative analysis of pseudogenes across three phyla.;citation_volume=111;citation_year=2014;citation_pmid=25157146;citation_doi=10.1073/pnas.1407293111" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=P. Smibert;citation_author=P. Miura;citation_author=J. O. Westholm;citation_author=S. Shenker;citation_author=G. May;citation_title=Global patterns of tissue-specific alternative polyadenylation in Drosophila.;citation_pages=277-289;citation_volume=1;citation_year=2012;citation_pmid=22685694;citation_doi=10.1016/j.celrep.2012.01.001" />
<meta name="citation_reference" content="citation_author=M. Stapleton;citation_author=J. Carlson;citation_author=P. Brokstein;citation_author=C. Yu;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=S. Thomsen;citation_author=G. Azzam;citation_author=R. Kaschula;citation_author=L. S. Williams;citation_author=C. R. Alonso;citation_title=Developmental RNA processing of 3′UTRs in Hox mRNAs as a context-dependent mechanism modulating visibility to microRNAs.;citation_volume=137;citation_year=2010;citation_pmid=20667912;citation_doi=10.1242/dev.047324" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. L. Tupy;citation_author=A. M. Bailey;citation_author=G. Dailey;citation_author=M. Evans-Holm;citation_author=C. W. Siebel;citation_title=Identification of putative noncoding polyadenylated transcripts in Drosophila melanogaster.;citation_volume=102;citation_year=2005;citation_pmid=15809421;citation_doi=10.1073/pnas.0501422102" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Wen;citation_author=J. Mohammed;citation_author=D. Bortolamiol-Becet;citation_author=H. Tsai;citation_author=N. Robine;citation_title=Diversity of miRNAs, siRNAs, and piRNAs across 25 Drosophila cell lines.;citation_volume=24;citation_year=2014;citation_pmid=24985917;citation_doi=10.1101/gr.161554.113" />
<meta name="citation_reference" content="citation_journal_title=Insect biochemistry and molecular biology;citation_journal_abbrev=Insect Biochem Mol Biol;citation_author=M. F. Wolfner;citation_author=H. A. Harada;citation_author=M. J. Bertram;citation_author=T. J. Stelick;citation_author=K. W. Kraus;citation_title=New genes for male accessory gland proteins in Drosophila melanogaster.;citation_pages=825-834;citation_volume=27;citation_year=1997;citation_issue=10;citation_pmid=9474779;citation_doi=10.1016/S0965-1748(97)00056-8" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=Y. C. Wu;citation_author=C. H. Chen;citation_author=A. Mercer;citation_author=N. S. Sokol;citation_title=Let-7-complex microRNAs regulate the temporal identity of Drosophila mushroom body neurons via chinmo.;citation_pages=202-209;citation_volume=23;citation_year=2012;citation_issue=1;citation_pmid=22814608;citation_doi=10.1016/j.devcel.2012.05.013" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. S. Young;citation_author=A. C. Marques;citation_author=C. Tibbit;citation_author=W. Haerty;citation_author=A. R. Bassett;citation_title=Identification and properties of 1,119 candidate LincRNA loci in the Drosophila melanogaster genome.;citation_volume=4;citation_year=2012;citation_pmid=22403033;citation_doi=10.1093/gbe/evs020" />
<meta name="citation_reference" content="citation_author=L. Yu;citation_author=Y. Song;citation_author=R. P. Wharton;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Zachar;citation_author=T. B. Chou;citation_author=J. Kramer;citation_author=I. P. Mims;citation_author=P. M. Bingham;citation_title=Analysis of autoregulation at the level of pre-mRNA splicing of the suppressor-of-white-apricot gene in Drosophila.;citation_pages=139-150;citation_volume=137;citation_year=1994;citation_issue=1;citation_pmid=8056305" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Gene Model Annotations for Drosophila melanogaster: Impact of High-Throughput Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="twitter:description" content="We report the current status of the FlyBase annotated gene set for Drosophila melanogaster and highlight improvements based on high-throughput data. The FlyBase annotated gene set consists entirely of manually annotated gene models, with the exception of some classes of small non-coding RNAs. All gene models have been reviewed using evidence from high-throughput datasets, primarily from the modENCODE project. These datasets include RNA-Seq coverage data, RNA-Seq junction data, transcription start site profiles, and translation stop-codon read-through predictions. New annotation guidelines were developed to take into account the use of the high-throughput data. We describe how this flood of new data was incorporated into thousands of new and revised annotations. FlyBase has adopted a philosophy of excluding low-confidence and low-frequency data from gene model annotations; we also do not attempt to represent all possible permutations for complex and modularly organized genes. This has allowed us to produce a high-confidence, manageable gene annotation dataset that is available at FlyBase (&lt;http://flybase.org&gt;). Interesting aspects of new annotations include new genes (coding, non-coding, and antisense), many genes with alternative transcripts with very long 3′ UTRs (up to 15–18 kb), and a stunning mismatch in the number of male-specific genes (approximately 13% of all annotated gene models) vs. female-specific genes (less than 1%). The number of identified pseudogenes and mutations in the sequenced strain also increased significantly. We discuss remaining challenges, for instance, identification of functional small polypeptides and detection of alternative translation starts." />
<meta name="og-title" property="og:title" content="Gene Model Annotations for Drosophila melanogaster: Impact of High-Throughput Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/8/1721" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We report the current status of the FlyBase annotated gene set for Drosophila melanogaster and highlight improvements based on high-throughput data. The FlyBase annotated gene set consists entirely of manually annotated gene models, with the exception of some classes of small non-coding RNAs. All gene models have been reviewed using evidence from high-throughput datasets, primarily from the modENCODE project. These datasets include RNA-Seq coverage data, RNA-Seq junction data, transcription start site profiles, and translation stop-codon read-through predictions. New annotation guidelines were developed to take into account the use of the high-throughput data. We describe how this flood of new data was incorporated into thousands of new and revised annotations. FlyBase has adopted a philosophy of excluding low-confidence and low-frequency data from gene model annotations; we also do not attempt to represent all possible permutations for complex and modularly organized genes. This has allowed us to produce a high-confidence, manageable gene annotation dataset that is available at FlyBase (&lt;http://flybase.org&gt;). Interesting aspects of new annotations include new genes (coding, non-coding, and antisense), many genes with alternative transcripts with very long 3′ UTRs (up to 15–18 kb), and a stunning mismatch in the number of male-specific genes (approximately 13% of all annotated gene models) vs. female-specific genes (less than 1%). The number of identified pseudogenes and mutations in the sequenced strain also increased significantly. We discuss remaining challenges, for instance, identification of functional small polypeptides and detection of alternative translation starts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
    <title>Gene Model Annotations for Drosophila melanogaster: Impact of High-Throughput Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CGfmkvS1LehEbBfKoJEqJPD5S8S8imIgHvPbWAALYoQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460001","apath":"\/ggg\/5\/8\/1721.atom","pisa":"ggg;5\/8\/1721","ac":{"\/ggg\/5\/8\/1721.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/8\/1721.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1721"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1721"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/8\/1721":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FjHgKjDeD3\\\\\\\/VwucaQOW2UQaHEJnBRmT7wnIXcMWsyCg4+Z6ZZbP4oXQNCs0P8WlV7kfmHnotcxBBJ0MpVNZDVMxzf6Rd7B2gVVAbb8gc7nZ0cFlYg+zKWOGQSkdP6hNrJcXWI4MIw0TN9Hj25LqGF6n9rtzBilc+gqkdlR+MzBR9Q1VF8QpDmCs4YRDQ8p+LH1d\\\\\\\/QkGz0cF220HoRDFMQZDXccim83wWNT0982XdBKujkOTbxV7OYvqXn6Uq2G+Ay7csreMcwmADe7Ryrj44rQ6Jze2gtmucOODbSHcRMqju1QTbDorbqxzpfQYBFKZYyPZgXbAcKjrjmdYcmwqbLt6oB8nRlDp5IkSMRI\\\\\\\/MgSIBRz59z\\\\\\\/\\\\\\\/H\\\\\\\/\\\\\\\/\\\\\\\/57fVBS7nIONTsg8g4KBIvMK1kTwhfKsMjCPVcWsWwTmbsXLAngDWfwwZEPbLdVOcWd1XPQ0iDwg1wJzy9j06Pd+APDL+hR9W9iu5eb\\\\\\\/zB3oZ5w\\\\\\\/SsfNepq32HYIPLerqGN+h7iD0B0pfx7cVoatFQx5GVkVsUK\\\\\\\/R8jzgzVGkGsIA+6KeIpYypNFnsPOn5ZzZjgpBY32WqGVgQXQMOYcZ+AUKC629J+D1imADA59I7xIpdFGHIpezXiaF+2exZ4L8EJDs35MOSbFeLQ4ZNT8o0i7Doh6wKoXPoTykDnF+ROjkvhn0jGLGpnffrcHpBZDFmr9dWIWXK0HM9247tYfkKhxhGeWUD+9fgyx9i14GylTF3AEMIvhDA2sNpFYZWIseyjHLxhhg78gxUUprIQ0wOTtdw1SqcpA0OensIP6TR\\\\\\\/Yj7er73LXXuHrbV6618bf3JcgMvU1aGzpexqJdT0eohRBargpqBwDTl8ShpkxWnjidthZx1Ax0ELcQFDZqmQttSX\\\\\\\/AebX0J7I5m6tf0USZPl4gv92Of4xkIl8KRmLkTD1ZrOEFtwDmC7K5\\\\\\\/tvrTGmCkmpLbn\\\\\\\/F2VKo\\\\\\\/z5yxNqxM9c7fMxOcY49ItbxXL4lPhO0JOOurUd5RCfo6\\\\\\\/HIq5WJbDv3\\\\\\\/PNpyxM1ruDTRr2JN5GdHE0+PPppsq7j4d6hxHYK4n\\\\\\\/UegVfM6OksmK9RCcUIktmJCyxcD0LAV+ZNHpEKOcc8jtxv02gKrr3PUvBQObE\\\\\\\/yhCtoUxcTQ5WPYvK6ZtIkWLYEj7dCByH745+koVFJBFrJoEP4txon3LEfJ5\\\\\\\/\\\\\\\/uAmxjd5c8fozUF4ySQ7G2s6KWfDh14zcpjQXuYdwfCP52z9mMHTylDkbPkrqz9cFyVMErhaQnVc8PdJR\\\\\\\/gsBp\\\\\\\/MgXv9QQ5E\\\u0022,\\\u0022iv\\\u0022:\\\u0022E72CwWWMbeo5Z2Lykcr4pw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b0b4ad30a9065b74a7f482cb4873fc8f4845059480a7757fd3e8b4f1d43e10d4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tZE8s46tXeaXyWT5oAuprKTDGUhc8tK+17sMiJxTf1Zi1BLK8EhglACwQa\\\\\\\/AacHSp9UOHT9Diz1jV6XfK2\\\\\\\/2UXxvS3W5UkF2H3J3p4pV6ZJmBPm8EGZTE+JO39ndIFxuhYsMYtlhHHpF6F3P967hhs1F6an++cDB6MA72wRdL7H\\\\\\\/OBkynfIZiXV309fDB7FecqhsTKLFWgoGXqKCCLCuJskaTH3bAmL4KQXa+gLUBRWKPiuxlmLR0pMt8rLLeJ+C+5wC1HdLFCX1Sm0IaLCaFPfnsDpyQHvt2eyiIaYhcN2kYpTmM4kI+cdM5Chb3nQPcK0Q3fAajvGp7dFASSpEj4tolTnMwBq5hZCncDZ\\\\\\\/uzn2b0cWd\\\\\\\/EOXZKoiI1CZM0ffNqxtPYAh3xj+RZF7H\\\\\\\/kMyJAHy6jWaCJX5FSEE61XCNrxvlJjtBQ3W7AySDXoBrv3zmR8fyKO7BiTRdCMpQpetnK5fts5lmxCQhch6RZ8jrgBURhav6yVBciFeqrKgmVFJpohHM2sNv3Hq86GniilvVeJaQwkmIWbL0XHdD87NnWyk9V1zB5rPVx1rTBJUfNupJlYIQ+4kN9QuE0dhy2UTutpdYW9TBClb++jD6QhC7IaaS8lGIPwyfFFBuSbX30eVNJsUqo1jhi4FFH6VaQc6BTgHX01G15cvSL7hhZnHNiN1utY+23jnkmvVT4KTFw39Xex5NbNgYT7eTAP0bYmSwJIJGuqZ1iCc9ve0ztpMzj+a2JMTHvjbzCZozqSG3qOnrdXHVizT7qFgTMFbHOh\\\\\\\/31JsQ7crpN8xpkAT5lNyHS\\\\\\\/CAqSLoqv0zlp+KKbmZ7nkCW\\\\\\\/cCFx0kzThyHI+moYTTwEQrJaEghAlvjj4Unv5kucmsWmsAmS0gEYmp4CGhof3G1q2bU6d4sFGEEGOBnVz9UDgmUoQ7oi3DZnWzc1XErKuPED7iep47z24jIAYWyzNyAC7ZLlw4QWlbiCeyqC\\\\\\\/owMn65416nAlAw9Hoj9qEIanmVwo\\\\\\\/M7inO+iB+cdeZ\\\\\\\/0YqwjszuJcwF0w1ifeohMH\\\\\\\/tKQLkM7cBDZWuLWf5wDW6MEFqNUPVBjjoAXhxhey5\\\\\\\/zGE3eWqm5lN05dDKCrQ8EWpMaytRHA3Qru7UbX7vLrQpAvcCifMzriSbZxv8UaJLzqJkPpDEQyYFy9WOwGf+y1gzydH\\\\\\\/fcrZ1AiMc2EUJwlGfR4BM04xg2PNZ14DxWwgkSDM9dqycUiBwrTYB5ZzKiiX1Q8c2r3bPcXWvWo5ULuFAC6Zd6NbrLF4Wf8Bj0aPToaWw7lFLdxSvUHPfZxksqqD\\\\\\\/MIJ1C5CAfGbete\\\\\\\/b2VMO4EZ3tNbyr2jDGKBQibpJ9KxekEGGUWPfMxFE57m1oxM8ZpISpdQdqsM2WdO1ezl7RhXaBp+RKg7W3lLhMa+BzPb7WW7VOTn5pWlBVPEM+yq39h8K6DOJmlMj28RP+GoMGPJQKGw\\\\\\\/XIoPj+WQkQxvJbARVDfW4lJy8akqukg4YsbvJMmWNFEzan3QyxWWq8eW7TV0rOAO+PJ6Cxnt54m9DpZNWFpbXL\\\\\\\/gsEljfjGzmEFECfmo7RIciTz\\\\\\\/LXGjR\\\\\\\/8g4LcZC\\\\\\\/fZAFkwMymC8svZjOpIsQnT4z417pOeB\\\\\\\/AAzC09SmXjXE3PasKEg0NS3LKm7p1ELP8RBLkJR2TQyHoDAbimS3ysIdsrIK\\\\\\\/JQBhQ3b6gwGnrR\\\\\\\/peqhAUWqp0YEUuAfaEgdydVuEgheXQjEduwGQBvSdEvCU2BIajgnpcAKq+wO3knLtYzd095Rx5yKI1iqI3WrGNM3PNsNDSfOJ+Mp+GqsgJfRy6IcH8qN80TmFMoDTYOIdwZma3jEjFLNUtsi6pdWLbM6pM0YvCQ0X54yF7vT0UCpT\\\\\\\/X6c8Wyj9KPoWK9oUgnH5I+FuPS7zhxCnqUfLYs449rH60e3+\\\\\\\/X2061nP1p3bF7VAb5sf3wbx7j2YqLKd3AQwWoizOXvn+9kil5J2bp9l44gsxiVsgepgrWmZTRqbKZwQWqt8O\\\\\\\/iHSuCr9fzQvHBnLuXdFHmjmw01FhkjmTbDTXQ6NvEe928gzGg9LdhamRszYg6Egxi5ehOHaEnih79bwBDGDjVkRHJ5eZPwjrn7MD3ebWy2mmyrLtF3hQiBO3l+IpRtrplgdroidpRcH08sqi556sKNLbDr2+XefE0JNc8HGFI7YsznT5Y5YUtN2zo+GZlTbTwAUXff0eSxncfySYTKCgzT78C67zjaPadypgXReuE9DhPGfG1w1Dgx3bAvei1YXHTBdmZ4Jk+Kkj41iUk84HtOzaQCiHoZhJpZcLalvJKZCovFWQJtv7sdOjYMJmMFEdaGdu15Ro5W3FZDQjKAiYxUapETkOpe6ngbqc1TaJLueqsvsMaNo7FIEkWY9Llq5QwbyMTbvNwD9sG4LXAxHVhyTLvHssCb3ZPZJbumjSu336fCMUahn9Y3gLeFx3O2+zKjBmD+6rfyeYWxEq5RbR9HXK\\\\\\\/jjr\\\\\\\/HxRc7kxT2j4iTkJYfkaTVWj4omSgnPcS4GtwV5TL5y6oyulTc1BKQ5IY8Irf4Iu2EGW1Xii8nwwj1G0Cjtm0aZDfCvbHuXsjMLJAZ5kfH9xD1LVkeFccsKVf\\\\\\\/Jw9wqwYMYRVWt2S+77oOzEkLdNoCMl3pSg2pGlN1ZFCDoN2SWsjc0f2aGToEu491HeLGhM8SHpFYjVpJAjZFrkOoD6ed+lzP8mgH1OKYKeViBmX7NXA21+M3dHa5S0Z3UoAvHQifPIFm9m9+p\\\\\\\/rkLMuQ2O1tpydpoMdePcj+hKt\\\\\\\/Rc7xhzETZEIi18v68xmy5mIb4dyI6KD3UxTNwt2CCDIwn7k13Y30aqPNrIS5kTqwRtYrMsjEhfZpvE+EmKwsw3s1cRhYCncEpZvmlVTIltSwyPtZk3EgjiBdUJY4JYPFozRnxns3df3NdhehCySfLEqtXtAC4jtbeOrFuKzMFpUY\\\\\\\/quDlfHZcLQUcCcKGYfvf3Rrq8LlpCCba1JfdCr9vaxB\\\\\\\/MxdBT4RqeO4EMobOs5oTxM0+o73DIgpSt8j6JgrbXQXStu+tcwNLZLXZNG5dDraKpT8\\\\\\\/Q3LPq2XUpq9hdsZFBfn9p1lnatZExlQguLVf3DAnlrWhet5JS6V+IRI6HcPmxFHm4gIWv5ZrxgSYWBkrOEi0aK9Mhz1UQQPIOlM5bNSFszJBAhYyeVFLFkfaNzJiXI22bo6bhp7UUcwRx6mS6sQRESsuVY8HIOFyyoS16GUvI0skspSbI2fKetaPkWU4ob0thJgp61P4\\\\\\\/QRvd\\\\\\\/Na\\\\\\\/YKaI5pYgwXFVnPHfo05VVYSF\\\\\\\/7uq1uDTCGdlktpOhQUVmVmpD3HFevo9MDZtzAgJ6WPEN3JxvUObhLctwN4puLe3XeWQtN+59pQoyJsMDXgRplYgZXHOSwFeyrVTHxFU4WTvnXrQIANR8CyHLlnY4W42g6QOvIeFJwFjyZOopRGs1c+zM5zz4lJH2ZrSpkhAB5rrzFo\\\\\\\/i50Bg3o7RSxdudaELsSdVejfxxHPu9N7+eE1q0PgkVbWU3gv3q4bMVakrIclaNIVkd+j0XJis10wg4VBkVYa+RXWkJYZmpITNs6KI2UiAPulkl3JLlNrrFFREKJlGUCy5fk4ioqfR04ithhCoifkPTsz8ToppqNy6af+h1G9cWCl6i4klrVpXap8JbkuRZkAMXUCv8IG+VTDl3OtjtfBcsij6Ztcq59lGMxpUL7grNLYPBcECGnSVXKn8WtyvCfzjjFezP\\\\\\\/pff8848B5Ggf2IIUWTLYt6xSn0AiVa0C3eeDKPdIaKSslzAyvvEpdh10cPaHXiK4KABqCfgxAGaMuTXUmGpcpoE1YnxrzGSzRmwyRxMSKqdEOKYqEvl5I7AXTvncKVq0+5mz8ol35MH85efI0oGKIypwZEnud+Jeenfqnt3aZde3t4ar9Feg8fP8RBd20m0pzh2o0JlGiDYxPEuSbp+GQN\\\\\\\/HniBzxyWTwn864yoVmVTucLiDr+r9zcB6\\\\\\\/deouIFaeGomTYVJcG++Zh5GSDPCjh3KeHfmCgvYZ8POZVWvN94fnkqF8Q5oSIok3UW1YjHWlgLZcLk1iKQ5RZ9RIJLHxrcwrqs\\\\\\\/ph\\\\\\\/dTDh6t5oAMwdAyEF3+i068f7S1qkcMNDZ7Nc2nuIxPnhMYRbqqMzfzyRPGvQtzC9bymjnIbRzu89WkFm\\\\\\\/O\\\\\\\/YRrKnCBAIFIjQvfhxo5g8Dm+0el0fxxyOGGESKanadd5Hh451fzVtvWSHoQT1MVc4qX02WLn0SJs17Y\\\\\\\/9cuD+V2QKYJjChyua\\\\\\\/KfArOSIbO+ZijPEjDbDOY21VWGTOHDtS4zziOaAUIuTQDYs5QfofC6Ngzzb16Z9C4Boy\\\\\\\/yOSSqMkDFFiFHvqNlcNT0HYZC5+\\\\\\\/4xbsg378yjOzccoDt1WhOxCBnWxLEso5x8U8DtolyyV8+2SmX9LxzhtS6Xq\\\\\\\/r+hFX7vvV95jm3MXBKc1qbXcu7vkXyPqHPaodyFc2jpPmERG0F+j+o71Lr6RybwdWPMDcSq92h5FkyZodaWcyN29dawsPsO3Yn\\\\\\\/cje6vBxuPxe9KNwuOljorIjj8wj2nesH\\\\\\\/BTDqvXmnmiILR8dZQObmQ5kZjMDNJbqIRRnK+yvPPJdnTIaAzj2qvwNC297rscy7fybMCmX9NLlycqfqfO1HrHWvKtumDXuC+7glF6GYDUjs1lLC1P\\\\\\\/uzSSrwbIlQ0YkCkxLzKGkGiTCeSkFvUrK\\\\\\\/y+Jd07XLhH3y\\\\\\\/kGScAasBivzVTq\\\\\\\/EyPUb3vRvQ3V7+inWLKXFV\\\\\\\/MfJNLU7Q82L1HjSXlfrlK05A72LjvCi8CwnakjwE9Wap\\\\\\\/I2\\\\\\\/e0jqMlOOaSVsl2yBIulFFbm4kZbHH+qjTL2kJuQJlf0Rs4ZUBuiT7L9dBgGttOGNykrazzTuMqNJavr4KhXefJEht5SwI3WBLadrReBLj1amNXlvz4Fhr50V3q+fvNQ3w62XiVWPssVR7eRBbQ0VR3bB8ogc8mFMBA0A6fMI9o\\\\\\\/upkD+qVAQ6NJ1N+Zoh5e30AdjOx3oJs9f5ZbPS5voyzpcj1AozG63ubY6vE7a0T48+FlVXfhZiY+X2PTjbpUhnwobO\\\\\\\/pY7cHuiNVHxrTH7TyTuWwQljt04mWCWi0dFNZPIzrZxQsjAFzezdZuBKxmTEOt+i\\\\\\\/tcKB8JTpocJEXEwa1Pkd+pvQyFScnPXT05ahjdw60AIxVmfbdD1flidDAOKcfKvmWbtBDJ7CxUyBYyibiY+hrF2Z1wwvZI59G34URzi0osindjJFCCRU5Z6da6ZRu8TQo9x4SKco9OHhsKyFWIsDSVyvzyuJHFpHdWJXC95qcldEHk3\\\\\\\/q\\\\\\\/2N4222EyIpp11h0XIlvoCG4liTyATfVCWvRVsprpGNpPwzblKS+wkKOwfP\\\\\\\/Q7RsL8X\\\\\\\/0GjFAk9R67\\\\\\\/vdyqvF9juE8rK3e8zILdP7fLwefErMO5ypnrxilQY\\\\\\\/DVNpmxCHpLEPzEaOL0ImoxlAlAb\\\\\\\/BX8uxPMVn1FZVHz2VnJjDybHdJEANrYmfPKsH6tgpIQtar22qKAZOz4MWcc+dSHc0UT0ZNfkR7oNTtDx1eA9aQh9IA4hox1w+CMrAzGCtm1HHHPBZHSrCaAT99boQJKqtrlBeYbgs2HYvPCu2WpbWwOcsnOBPh1xQ4qG+wYRrveXxqTXdG3ZNt+\\\\\\\/DXp1UxqNYgJE\\\\\\\/fmYNda3pafbPAvKSSYSZMe7KwCHVyT5siy6J2SmHu3FAp77ODSCUIB313vX8CXZzLiR6Yxd3oowaUGdpQ89+ZGxNvcfuQB2Bcvsp38OQ+vm3AFSfh0ykcYmDIuNzKTHlLch0JHNlBJu+hiIjMYrJi78FJKdwu7uN4D1PJM3Bu961NVIcTZgTr70dnLvIXHpb5hNu4S1g46Y\\\\\\\/l5lbgKAaZUpVvDJq\\\\\\\/ioanneAbJ+gkmglTSOFJb43toFNibVu76HpDYYG3+j3AqDUmpL3qG9gkOW61\\\\\\\/D5sLS+J9+VIMnsR572JQSlNbVVI6ea8\\\\\\\/r5V29e5rdnSptHOdSsKTxCAQ8csFbSKyhXiwk5DW5lp037RBYWcSCWYzbwph7TDm0FVVWqg3hV0F75bxtu2BtDsM0ruhMjWesaMg2WGIdVOWF1O44ZpLtqmyWOsXZfp4hnGNIk0PdWehBs0LUnb8kIaPl7Za2MP4EINimyaLCLnoKtGWi+gCGuJkrhxtyAf2fJlBp+QklP2vpdjfzLqBxlWXoz2ReTaQ92ucZe\\\\\\\/zpTQ4uta7fhZchvx5GVkt+EvpErnRzZaNhrwogE74+uPPKITUGRKo+ghp9L906s7KZ\\\\\\\/dxiC8bqpngNgc4O34NvTL6otE9Cm6lP+Rx7hdBqg3KmqZzO+Tm0Zk4ApQCKlG3BA5df3n6FVZ5YjTOaYOJ9HS9b5bhA7c7uuyRY7PoANWnHxB9TarPnX1DXPui0TIXrx38iJqJ9tOGrfkeEXfXj3iVnvTE6vcxD6gB1f\\\\\\\/Q453CPX\\\\\\\/XlQBf4SFLbKq2b0fasykgiFmqbR+UMIvOiL4DrqBkjX5oe57KPn5epuuam8wIFETfXvXDfKXQylqBc+jhCWPlln1haa4GLaQxny9C4crZf3hXxpOy2VLhJk5Szy\\\\\\\/kE3hFKMKG7s8f6nUNRYNrZKBOMC2Uyv868k6+o6CFXoA8GX7B\\\\\\\/YYOANGBqICM4OjMtSXpwg+C9jQ5DP5I5bF+xEiAtSsCaSWBBD8lFI+KHH4+Qxv6ntzeXP5u+BCt40bLp3ao19vcBbENtyY6aXzgRdfjjS8v+O+ihzJZj2ciK4v6RGG4plGyXTwLb8dGcbhDwlTnEGGTIVidpYAd9ej67+R0hyRyQB3gmBcy3cYZPdaHoiIN+yNGp5GDK7iFjsyp9fgpklVP87l2izE8CFxfSSZaDmyTwLML0NEt5x7iJajkMOZhPVVPgzKThr6E\\\\\\\/tQpqwYuHuHTpuqi1h74rgVX1lCaYIVmGREk6c+ns8IJcexEbBk3FgnVDvFxPl+UplOWGJyiI\\\\\\\/FaAr343lUE5dCklBQCj7SVEa2dT1b4dvQINbuHFowOOeqhuDPzQXzV64sXS\\\\\\\/vONHK09cqgbWAsm0aKj6TKEfjR9WKbinPAPZ7cyorstE++UBnBLwJlCRSXsvCAQr2+uF09j+irLClrFHXA7zaDpMpKc5m4HvVHYkyDMCtpxe6bXFYefjeqaKV4p+4NDu4Bo5yIuIhuo82gLlG7Q\\\\\\\/68tGr9Ys9u+OR8ieV5DKdTEtfa7dWUPKYKNHwKANlYstWN4IigHSzY43nItpXWkbUCuVHE2suM6tFZwqNjCh9QdaB\\\\\\\/ReUfN6xMrfyhAJ9lajiV4NajQX\\\\\\\/VcvfJHOYhsxiQQXVp2RlPc9qhirSeXnD4tNybPGLj8lbxE88iRxUmrnc3Ju0FzMTq0+QyUtl+abyVyk7FjVLIQJRqNZJxxU\\\\\\\/JrM27YdgEiP6\\\\\\\/B6Wyvd47ZDd5SrIGsmXHSx3fGNOyuYzYaf7aCE2uSVPl146i2ygHuMclgnprEqaQoZOPvwzixYUsdsJj7rG4rWqSrx02MoY0obWwCFEETU88U1jpuFN1sJaUjHIJGJu0RyGy2TK7v4UCAFiJSLFT5kbICcBHgdfoGuD9sG0zx9Ks2aJ1qjXN7JPmNhIcj+wIT4i0B1kYdEo75umVHQdRUvYMKfev\\\\\\\/ShuMJD+uPHoTeYDZZZVWVt2ZpkwsUzPA2FG9h81ZyLXw5DhZVaP+tn4vWM+r5UBLguZX8MYNg+fbZItj\\\\\\\/ZIj7Tomd2AgXKuwV0pkNfLRJqLwyVBwBYBadJjfSreiPRVHLQPY+Bv28FXOLm1jLqTUbc\\\\\\\/cRWcGn9mE7eMkJuq47gsBfyxaeRBqxEKQ2EKkvTMDzuITdJlCZv+vxsdLMNytjyKTqWt5Lm9bBFtnkqVvt1tQhTimyo4d5NFhkc6oF5pM2YAFHNU8NucH69P4FGH03XkQz\\\\\\\/TYGB0qnYv4jidwdQUlFxhuwf24v7IrOINkiZr72YEENblIYYRggNouOiroBVnsEWVfP+w1\\\\\\\/QP52qYSkLOOUKztYMGAbBAP8TTxuK5MJDYUCcE77BecNkqkAyf0sbyA5z6nEN8YbV5sE4tEAyZ6cgOq5zbuKgoH1qGUdzQTnWDrxafzXryGWu5Gg7qkHEQid+6EX5Zskm7TOFwr\\\\\\\/Bjahz0koS1Sluhn9PS20bu10141uQMXECQVAK0qkLYE3t0XFwarx74nHk1GkLKCMY+vI+9AoZuvlBvcK9eZ2\\\\\\\/HmsdLV+EaRMwCFNnj7JFfkq6c\\\\\\\/zmsbxPZMND8f4pILn8ajidEjtAfB6TaLkkdHkI2G9r3W20S6R3iInGqBpQhuh1AAWjyyJlP\\\\\\\/TwUD\\\\\\\/andh9+PuN1SDyaWRyv1btlLacoNx46ewaEKO3YaGdUAqam6Zpw+hw7mA73LdcimovF3\\\\\\\/AahsByE5w0cxfJeyxplMHMIqZApG5aEnysSX0+7DWB\\\\\\\/naLJ9Emo3mk1Qs4gfiZ4x+GqIc4emlYLEKmSWbqjz7evr8hn6FKkR3vwhK5MEyuQ0VKeAmPVzme+B2JXSS4RkAo91AsdBslI2AxiOrXV2prSVwgnyzQ76k+IEwi\\\\\\\/2tWJMbzYZdTsm9QQPliRunRccar2EzS0Rr4GKFC\\\\\\\/G45Gd93EXbBtA+\\\\\\\/vMIolffcFJ6iAg+rt96tjYgZGRdCclYFx5tZ+gQwCABVo2TKB648M5LXBmERT1bIT2otbt9P57VCjq6qxTVCH5aJ\\\\\\\/gy6jSnzSCYd4u15DAOGAaSLdbvXndHa1brDz\\\\\\\/YEepTGbqT75+i+ZvoIl4Db4\\\\\\\/hnTLDMrz\\\\\\\/4qaxKhgrm2yXNPMCs\\\\\\\/AdfWgTuNjAECEpeflrWhmhh3NGBC\\\\\\\/efRUZ4MRAva7mkxxhP6bA4JqGiJKLmGE78z6XKD\\\\\\\/pGATItI9u+XoK7Zy+3u2MkUAEBUiE8AbPNOySFsbw9GibP1EDseZnOTGb9oxUSa4H3sS+l2j68R4tihWs34pX1NYRTQlx4\\\\\\\/x3Hzq28FCvtkZLoLPslYN6em2F+2qNvWDB8tQ8bZ6zysl1j\\\\\\\/hXdV3RdD2SK0f+t+eeC6q9kg8MRCcuOTSTJsUKbt2HpXIyGYXr1OWPquDY49maHNa42fbpxjCODfmfTGVgoZ47XxYjQWiqNOw3DGYyaD1hMXQMgkEIotX8CbLSuus1OyPhbVGlBCQG4EkStWncDxv0hz+qI4s7tNUKNyA0ntrhFpTUfVxmf1IR0Avl\\\\\\\/SrTD+ZemfpO59aYZcElMKHDX9M49IQTGpg+RLnvBMEjw8Nfu6YgvLK3sMcjl588L3MN+LeeWTyIX5yqjmyVkNLEWsLAXXctVLUJkrWZFn3nWorrTGSvlmFD8wPtEIJscUwmFXPzkAtr\\\\\\\/dq0+gA36U1YVKUl6GhM3CBdESKMgct3\\\\\\\/YJWBUBZXec0AocUEHf5Sd0azEkeE6NOQmZPpxtIKrymO79VFhv156C\\\\\\\/H9G9jdf7lVZje8njviQy2BU7YiLrkSddSoRglj2FTvYP3Fdcp0f2YxUbueXaewaLmSI\\\\\\\/WCMeqzJQGOJgTRkyU\\\\\\\/oBTabb7p\\\\\\\/T6BLF52YWoszHR1JcvVVEQLfPXHkpUluceDR1SUt+rDQzjvnzjz6hyPy5EwXIadaamAeNVYxGk9e0mxETdDTbL8skmLim9+LQSg6C\\\\\\\/fHuc12Je6GOzq82Poe\\\\\\\/txU4RF86Xt\\\\\\\/uw2Pl1Z1d41MGuNtpUtJxcCkSWYu0YYR+E\\\\\\\/FExK0\\\\\\\/VtlhWID9Nj7M0UZIW1cqXPGc0I6UaOwQNxTeFl5Kpf872KVFR243gV\\\\\\\/fMfnAEu8IKSBGucXFoS\\\\\\\/GWzNtOLaPfRL9oVSn1PMSNC\\\\\\\/gmgBk09Ehiehhb+WkkbNNGlFb9d6Gx7tpx06qMnbm7ONv5GrkwcDPaqd\\\\\\\/H1\\\\\\\/TZqk1c7H1++mSiH\\\\\\\/4EcQNa\\\\\\\/h\\\\\\\/aQvpsiqVUGUoaMoT1B2YyfIxzbDzs4KTsXsXfzDXsAkIRsOUY1YZ5LJoudCj3xF4lhbOilltWp6UyToMQ0ZsUKASqgf9PyinViUpXiv27B8HFh8ES5VbrmEl\\\\\\\/awPr42FzGCje+qeJwPpIV9PyxH\\\\\\\/3u3db7MLjpqfpDWTws6MNy72h0Jt9NHzSundEyAULvhZCtO62Et\\\\\\\/HdkvkNHE7RtkQdVWpLyLVX5srOCyi2cgZ\\\\\\\/9Xuzt+2jpVaz0U9Bq0A+M1gG1gPrYOf4hkErRzcWSWBHtg6+Q\\\\\\\/\\\\\\\/V+JDAFeYfhDwG8VWH0aOAFktSEd\\\\\\\/v5bXPvGSQBzLB2X2NNRhdWVFL249L+bzV1X6GnpAfndlxHuWVMwgoya4mUwQ6fAR6LaGVCbfpotcQ\\\\\\\/qmaR\\\\\\\/9ikpwK+vs89K2EmGBJglArCQO0RJNyi3JOKNp3rTqFkpiAQSToN74gfgeF7dhra0sdz3rwSEOVWsz4yhSe2o9JTgzxsDF8vzQ6DSi0X\\\\\\\/Y3Gf48BgE4N5WAihnZYYlFLyIV6hr6fO\\\\\\\/IOqEgZJocdvZPRE5PZDjTkKBoyV7Ec56U7zdbMbXmdF+ZwtmI7Dxhr8mFOpwsWPIpM53YCKMYTg9Kz3yQplsIP\\\\\\\/Yd9io8G9CiJT9H4n6EclcK0neF4yZA6XkcqtWomCBZlTnxLw6RL77JMgKTzu0lXRjQ0NUOqxstziVhjixt0SgRoHc38QJDqPokgtDceeYt32njtdEYQUbAqL6Udg8DR+4eGarhJ2Ei5TujGxCGb4kL9WUj\\\\\\\/2FK18A4XnKTNwAmOuU+EBokBtruBPu355lXQUG+ENlwlKvXfUYiKmDykVg84zghIS6iwNDcgfdpo0VW4pa9HnZwnwearVYwe7ef2U+QwJA7kmC4du\\\\\\\/vpLHe7L\\\\\\\/HqsUNXxRjAap8cwqIZ\\\\\\\/BLJ+zFmeLtjAWTdxInVZNLhasZtbEj4umatStmXbOn5q2ikGZy\\\\\\\/ZdW6JE6TklXGRPzR0RxKe2uwTpMbjIM+0DxXUdb767fVReKge\\\\\\\/ArN1+uqc9A1Z1sdoECUix0VfxyTNqEqHnogTrydShKb9q7OGifXXfzo+VRgF3FuOoH\\\\\\\/1Cuy0v0yJrzhCgSSJsgy2wWkjGSc4lgatz67npwnSN1SE+M50U3DD+Un4gYg4uWD8VXJFpKZeOVtqhnSDVUjjk2k5wfaYHF2dCgPCuQ3liepIoN1w0RGesS9FElpg782Dg79eBKHwzBr8RLY7wFz\\\\\\\/mpeD2dUyVrrpXrkBS0A8HN\\\\\\\/UrZ5VYA2WEqy6SP7XWhS56dKBcyNRD+YEdOqzlLxHVS6zyKKMw7d8d8kN2Zuid3MOSjbeLvZW6SB4ccPhDqhJdde3w+lBnXhY1EVhRuq0jQynDjcP+6cO0wlYdJr9vANcbYbfCxybo7pjOrB2WlbpiUEbU0GdRfAvYz2oPHKoKf2vzEZAHuZlPYH0GBmqu\\\\\\\/8Z+PZCMUUgaHk6WeHY+EKZbeEBy\\\\\\\/tz\\\\\\\/TiNRxf11wZX6\\\\\\\/3xSzaEu43KYwasBCnQgL4aosH0Ai\\\\\\\/QTkNr01GuVaCe5k8NokBd2qML77iI6NEJc8gKVYUrx6Z9buqzPfrnDM+TUqacJ3tNNraNwVgctDj7R1oT\\\\\\\/zRYiVbhEM0X+9ziWuZypVMhV8kUs4DfZeQBYo\\\\\\\/vRYpKI44MgH1HdlOntaFjx8yHG6GioZa6g1xL0M7ZHKof+MnUxxa2HSNf8pzYu6GV73fMGggd8Gn4Dr0ecJEAK9sExmF8BcCjOcqLHNM1JNhbt63HlrrhvAcc0r\\\\\\\/8f+pMyjmWkmcyN+oW2I\\\\\\\/ceeO4rYHja2UdyrKul09Ny62UL4Dbt+Xx5T2Y5TgYhIBBNUWDTxutBCJ4d4\\\\\\\/1XYMEg31BZFYO8H3toHPwIj4X0olcjnC3b\\\\\\\/n6QIVj1aqvyJsCE4z0ZBxSvirO6Tx0SKhddg3kVhhzZVXgHbBKCONnByDBrPY6r02N0l3tcH1W7ChN54gDw4cjYtcuJFVjvpmBg4XTnxD+PdCfpsLhRdGqKfgbQ23YCrEgFxJ6RQETR73kAU20ZUDF0B2pl0M0ZM4Uz7ofUL8KInsF6KCEGqcfzGHHwbsFlmAVcy\\\\\\\/pdmswNYyggzZZnbafRWzvEEM34lhjh3VwgVr2jJSLuv0i4oBKl79eyMjHgK3w0c1YToCkzQTRyNkd6X5yYcw\\\\\\\/1Wi5+\\\\\\\/TKMCyX91ABGPhrxTPwI1ORdXxAubI9dDTKPyrJXhDDdLskMGGd6rQ2Krh7U2uK7tiaUgbVjDo1CWWzY9kkLoX6y0bJLtd\\\\\\\/yDl7g8u5SZAkb8yGXd1Gg9VXHCLSX9Ag89+srzsCLGK7lJ0wjpIpghEFK36HNCOxJSL3C8L2CHkM9rlnru3tJOtpYXT79NYh\\\\\\\/aexg48hg2Ws+dO8MCsgeTsoC0cZijotWnHE44k0tBRs6833C2axoIGAATB14iqvyQ3JAygqJ9PdLgaQXM87gsPVDqhDWPAkfpuS5rfapS\\\\\\\/to9EdKIPwIXYNGSZzgYCw\\\\\\\/aST0QMIWlOf6B1nqZKlaq9J71Lsa\\\\\\\/p8JIF32kTQ\\\\\\\/zXqzEbgSOIMm3wLvijS1i4V4z3nyNU5572ZkxOrkqRaNM5NymMoazTNdXCtKSyxKUE6oCNxDOA2Ytw5T405mRrtRor3fQU\\\\\\\/P5Qj5MaBnfReVc58J9cOQnR80zH+cvveRvp7ArLcBKhmVrgL3\\\\\\\/Re0yLFtFBJy4y9E3tpVfg5J0hufO6vcsdrCbSMh\\\\\\\/wpj2J1DsWsEYTWihFo0dIDoVdGQAPY8lpd9P9yNjNQDwyKZoHdpl2Q9f5MIfc6p9NAFNkrDUEMScNf4buCi5ULcjO0RyMDcFEDC3TpfiwqQZ+Xn2ZKNa3unR6lTN7NPIyAl1hUwuw0\\\\\\\/g1yWLF3xDm2Ed16XXLAuDFxxaPR\\\\\\\/v3Rf\\\\\\\/85uWtonLczjNnbeV4pzC15b5q2FDWfPTSwrrFh\\\\\\\/iKCIV70hssUrohpjMBOZ4SITGwNEAywjyt5hF6rLACcFtfYwRcnCT7P5z\\\\\\\/WrF+ADmnb1vP+pjF3uNOWGej\\\\\\\/CkJ9aYah+ecqL4oLVoydRTyIlCOD26mPA0jb3sR9nPYq2dSYJMZBp9AJAUCKZZNpwEqZKvvC9UzV5h2f7FakWsZNw7laIPTYndDk77ojib\\\\\\\/DMoxjGaYNxo0jBztCcPrj6SWL0CZHIrT0ID2ozaLSPodpbmKYiqVRhszO4Ed\\\\\\\/hLZ\\\\\\\/bkDhpv0eSLIV1tp0CmXDHBFRDCd4dqlUiruim\\\\\\\/P2r6UuRHoK\\\\\\\/9yKhvJS\\\\\\\/TYk7TQ6NMfBeStWQ1iB6VPdAaZaqTus2DGzoFQiXEwffIiPQxl8UjRXrZdUIC7I8FalF\\\\\\\/\\\\\\\/R2KxlwTSkaYxSMe2ZzqDMFMgc\\\\\\\/NV4lQOpoXsl1jW76Qbg6xCvVgvvUBfKzec1d5XBLbJCbQoZokYsWpLlJsISZDoY47j+5NAcRCx1svQhUO1l8JHTkUy67SYlC6qzG8RLE1NQZW59w9pBuLwJJdPvm\\\\\\\/Bk8dr4VfjumsmgfIs1WyjT5doxoGrv2php81PncdJhYApKBqt\\\\\\\/sol3Frjz\\\\\\\/CRJaskxaVlEJVAjUkQWSKKaf2MQ8XnVA5mFiq51fsATZ9VOqZb5r8rLqjCMsh3rQUEFRj7MT9Uu8nt7rCaGoU3lF0ZUF+kEvDPp\\\\\\\/tetJpIQKhIfXFA3GZ4\\\\\\\/ygAsitxF7As9jhj0z\\\\\\\/hlTbVIOQOgPhsL9Hpk\\\\\\\/wzYyuHiS5wJTlLqXCs2bdqT8T6nvr1KQim6uIRI9NMJPiya8XzELoPPMSsw2F3Pj478UgKxq84qsfzO4iI9aZ+vz2IhhXh\\\\\\\/nHJ2DsRhBiT9cxk\\\\\\\/mzoh+9Nwp5NtM0J11\\\\\\\/p4bh1CKKSm9jTo3Jj4oX2bjHwLlh0c6XjXnhwMnoSTUKiIzCOAH5L2w8dvUG2KuJNYNLKUjz8zlfz+MUFyRelXnNIxcKPpyqKffwF69WvqhcUqWte8wQzoZi2mZSOxdeGPPhnFcxPxp\\\\\\\/SSIbCEC88CeO0yVpITv\\\\\\\/rX9xDknmhP07S+uLGQPYvDpoQaLqjHntPdgnHQATuIHDDOgQ+5h2wf0kbhgL4hA+sAzuo\\\\\\\/rFU5Gc29JFh+7WilZJsItfS2npbvNm5znwOiThdHX5uEpCkNjEihxmTlBs8vjgW7k4RbBiOWkOh2yyt9jfdZbmO8Hw3YmiL0uyBNXbPZVW+JW67NI0mlHsno7CNH9kQE9\\\\\\\/lfI+\\\\\\\/8TVV92wREQ5jO8l98My1Cftduf6f2lvbVXyCeDR4mJom\\\\\\\/h7qER2C65HBWlzC\\\\\\\/EjWL9fwuEX1jea9qScY4jhSPyq9QwkIq85VIiP3eVzmdeVGEQbC4s\\\\\\\/UUVVzNHT2gw9dpKfJkje7iFEspqCJ0xHRo5rF+2osXT\\\\\\\/syaza0L\\\\\\\/xlE2kRUcOmjEKQM4uX8Ann9YDTH\\\\\\\/dfmqIcNuWqsJ2y+XOnhA1GuHFi13fRypRfJBrNTtoGprbbAJ0XlF7EALoHn52ivWlvttGW0OTRSqdGA52l0dsDtSiiHn6ooUff0YnTnwZXTtPoQCaUTaF6XsXzn0AG+VgSzm03XfCsJJPbyEk9nkVOzQ22mWZsIVBPnr8mecQ5RABBVPrPDXU7JoGyfSCAaX4RkbzOVJsz86R+3vTvm9cieqxOeDtAlXFpdsYvl9YNrYtfGBncEzgxQjKULPpCedQd5AQXx2fLTlK8KMcTcmTtsWLksy69dAdtjuuOUo95lqJsEYd8QDB45X4mi5YQ0uz7OFgVuktVHCMBkRnZxRKe3iqiPRbq0XVf8JujVMGMEZTFVMWRuC2cGA13C1GhVCxr\\\\\\\/5bYNk1BIQBNT98ZpzsJQk4WcTALVQGS09DrbFXM0TAQOjhyx7Ea+k\\\\\\\/t9UF\\\\\\\/cqCm+4CWpo1ojB6v27y2D\\\\\\\/6w5NFdOg7mWbJGm5FCAXYejfks6vM5jDuh43rJoBFjY2zer8WCIl3iiBd4n+Vkql9Mxqt+93r89\\\\\\\/d1O4T96fU3+izaQF7yrslc55ztLKMh1aDexFbaeiWX1DDS5HgKK4hoL+IQ49Ix5wo3z1e44p3ULRacFqNJRE7X09FoAEPLwU0cE7X1f3V\\\\\\\/FzCeUf4GY4zLACYckDfCSqqquW9umX67osDLAK4Thv3\\\\\\\/NjnTLw9by4vPJJYOSczisVNKEcx+ZNj408ZnaM\\\\\\\/oRR1eLY4O+yBQX3MVbRcG3gTeCscTClluALKjbGN1hhF+iNUI7DO5QBoDWJK7kNjcJMIgeA7Q6AZU2rKyvEGvDnbcJOSqEzwZQdVekW0D5O2\\\\\\\/FlNpc97Vgyx4VPuYueOtNd09KztKm\\\\\\\/KYvKXIGRXj9jh4nhKoOO8q2u2AMZHCW4J63iwL+wdqrhkhspG48NIs0l\\\\\\\/IV0GXNC9Coum5DHYFYgAhrUQGFsR9l6r5tZRmnvP1ad6T7BwLVR8BUPd2MU4EXpepEzF9A5\\\\\\\/BV4+u+U70pZFstz0zOK8N4OI2ITNXIKfrSmOxuChhuLVzyGcbkWU5vsON6NsGpRXwBTDswY5v59u8MPk\\\\\\\/xQS8KtUOXChWpVFQZ+\\\\\\\/Hd1ul0E8RP8o60YyrjXuPTp7m8tgojwPySl8w+yLMdg0kYYP4fLkHgizwpLWY4G9A1tchmn7OFrOWzkLV1w8T0vu+p+\\\\\\\/+uMXVqH43mMep0sTzJgh8Ky772j+CR4yQWStiXiLuoARN3iqYvRDn5imDwi5TxO2HSU1oa9KIV8EigtaPlkGgjs7a\\\\\\\/gCYm1OjU33eiWjfjfI6f8Txc4PPcYtMRG23238c88q0tOpwMRkXs\\\\\\\/8dYDSJiEYliGsGfVaEJCB2qMHnBUumJt+\\\\\\\/5WyYenMFlb1aR6L2dOoNLfoUB33FL\\\\\\\/CzHN6qP5dmXMThZEpa6AOT1gJMBJYDpDX7L0YDe6W8LlJRSGXdNFIpNAji70x5z6v1XPnMOopbKgFCUj7hKcPpHy6KXp1lTzczRTWw0RRSem+pfnTaXYWAtIQD22N1b21fzr609ZvEXvW5vRdkCa6pgf8Ld5FmH7Ss3iRbWEHRf5YGuZZrgYqSE4T1wJME4ZwBzLlOkLI7ccoZAv4yQJof7dZRYbq\\\\\\\/81Jq2Y3ABCDaoZ3JzfflH2VY241XoXeTcSdktYrW2crx7bz11zvLtMd3uPZry4ZiVzg1L621lwAZnXZlFR5IRl2n2firbCKJsPBxEYjbtMABmZv3HjjeSOoH0S2JRSie3NnoVEb\\\\\\\/CPt6tfkt9bZDoED8Kboj7uqQmY20rXBxMbeWqgFBVM0Uq31vZw3JAXvlrVRWFu4LRDxNz1MwVgw9IdjAno3\\\\\\\/neD9c6zVU3LQjWU9YAFZQ13ThdFwnMpJpQGn\\\\\\\/LltNy5wJ0OEp\\\\\\\/31tcc2OqU+ysWi18OCyYzKz8JnlasRR0C6fPmAfCEfo7z6ZN8aTCthq52PKZAVaxi5yIBaNaHPJXEly2suQTEKb5o7fgfIL3CSO2JOkn7wCcwSrGlvd+on01DDrEHIG5GcX\\\\\\\/GBKNiMgTu2eYn2L2sZ\\\\\\\/ctiHZ79ZnHWX\\\\\\\/ygb7FUa9axkczkwuRD0xA\\\\\\\/jBOQU8jKdkjpyw29jBrk0tG0671Y+Lo5Kkx1tWTMG0YVeDUu8Y4QyjlAx7\\\\\\\/4mb9hZObS8dkZ+yJsC5HPcZ\\\\\\\/Ac0qpg6XyeguhOk5vHT0owY8jCG62PeZAu77SDPauXIX8WiE2hVBqhv7OfVlZRWkjtcBAigZhO5JEaDUcFjDcfWUxG7hAlijBa023x1HzIwMczDwpklOgNEcaHMTdwxERa6M9wtbR752GfsfyV+X6K2AgEbZ8Nsd\\\\\\\/tulbVfgFmzqi3e49\\\\\\\/mk9I+hCQiviF2udMT9temtoppsPwXAq1Nfj+Tz296ubZuELkBjQ7fMdc9BzNHU8U8v0mUov1EWbujJOPui0KIRqjdlLUoC7+ZONKF9vF48eI97o6WoJLllRkfCrVpw4DMEddB7upWIQA1QOHNSDJI8INhujB8QM2Gxoz9b9GHQjCOmFmzytZnrKTZ45SM\\\\\\\/OGLlZKKsrCe\\\\\\\/6hVC\\\\\\\/xdGPcTBaaN3Sa4rJwMXayMI11EOu0jdja\\\\\\\/ceq+g90kP5RnvVqwzybDN3JmhL6VQGnL56XmPV6zFdaqmXsfoORALvFSTvuP\\\\\\\/ARBOtdhLc0vYnJ\\\\\\\/RgGydftwlsqOAagrJe4\\\\\\\/ty0nlmJZJF4KBImpEqNO4eww+enmR5+1C5JVEGpVEkK0VBFyARdYx5\\\\\\\/dxqIoVW0Uy4gPNmsE0\\\\\\\/MLa38yamn\\\\\\\/Wa0gRGWcuheB63rrUrHatn6arF0zRaqdJ0cL\\\\\\\/KIbFwQdTgNOCLHpoN9TWoqonMUE8bJhn4uE1Wr0qF01epQCga5A6UYp8YEQKOZrWNjphphM9qOOS509Mx7o2xLDsEXnnud7gWn9m81XK5qjgBvkFUn7obrVgpqRWCYFPh7xswtRzSfQMG\\\\\\\/kkqoyd+OkwfdTjK4Ljy4bXP\\\\\\\/RxPtMM7Y86J31DgBXq9uQta5ppJYjRlanxCqOtSqdTWrSTSN359cuRM7p93H2y3\\\\\\\/EZOUBb1Gx1RTahr\\\\\\\/ysL7OParfx97oh5+niTkFK+bdbe6+resxaw36kclnL6DYiPxRpf0hos2rY1+TnNrEMdO6HaZycpf6UoKhQ8YMs2rd9B5SDI3KZmsxqKXXjHatOQNK8FkmHw6uwzhTb+p0S4CUQBK36dx0beCcubzMdu9VAj6non5miGgUMEN9l28dAaWW093S61ajdOOmYofUYNBBYj\\\\\\\/kaL3V8NXPwXBrmjeMpMiZGPH5L7MEw\\\\\\\/flvCfaETDWVBsZ\\\\\\\/eS2MwxFBPKPrcBntzpopewCLhT9ugxG8fbyTu5JL3HUKdtAx5SKKJayXjHtWLa+ENn\\\\\\\/oQDZlOkARxqxwTFd3z\\\\\\\/79yHWppWZ1N+OCTigobHydvgcgL1zBjUWqr01KKskFQzK62EZjNl8DQRMoUHAV1a8Tr9uJLCOSlRZV\\\\\\\/95+zSr\\\\\\\/a6Wfoh68dQpN+DwLJrWUch24LB9rX+4grhq7G+O5LMF4YSGZ4dnp+1lfTsvxh2ViH1tumBizkyVnPwS9BYw826Z8iFcjm\\\\\\\/8+9xfCrFhG17GvhPX5kyW1898hvOjvlS6xTnZnB1Yf4TCu0WqBhX5Bh5X4sgwKy8XYBr2unPwO3rQAGivTqtIl8Wdxl2chl2MOFCiGRV8LLHsXgM6IDdWg95hnQzqOyU7\\\\\\\/ijF7kFxYbMXJfQl1lWUCpt8oOlU0agbJEZzJpfSfF7jq90X7TvqE6byXpJEPbTnNEBqUCVT7xNAeQf91E0aqipamywDIVsP1fWHmxh9r3lEn+vYw8UlFauJzDo+nySlL4lFhRAHZnxczx2Y\\\\\\\/23jC5bL8bcVKE6jn6j0SOJY4NWGgkM\\\\\\\/mjezx9FBqAlmJsIZBtG5Zd\\\\\\\/wjFZLJB7lYvHqpJh25y4xcdke6v5tGwzrY5sxoaxOfNOIshiGq6joQQ8B1sF98sW1v+jxS+lZkFNCpAw3veHYeBFZJLSf40dcquIOaxVUJqzLfqd9bAnkV0Gp\\\\\\\/6K41jOBw+TK5rEN8\\\\\\\/XurCf\\\\\\\/gsQNv3oncGDodgGWtKVRO5oMcA\\\\\\\/tq5K32s5j8ltnnJAHpIsJDa97vEx8IjfqPI54T1pstlIF8H3kqLdNkil6hZFQgSPr+scY2YNsugXe03YbUUoQmYHacW9EwlH3taUJn9+jPmianDGXH06oSc1pgoJonnlXTThVH5eZ\\\\\\\/EGmM8RGHMFUR8vqptQwrK6LTmgzNF++MO9eJDOd\\\\\\\/OswIvw1KaPvuIWGAIj12f8c\\\\\\\/+ZEQdUi3+Ch8Js0UBHgUpXajA6dsTE9cNRwPU\\\\\\\/cjF5oU0C706UNqGREQU6uwL4\\\\\\\/gJfNRHfhOkVg6P\\\\\\\/OItezeky3AGokdPuhs\\\\\\\/f32yOWSCL11h2QwoTxAYdINyGYuFd24TZRYEK5MTdy4+PxNQL6Dj7qZilzictbMJi28Wr0vwkkoajmIpJMzG5+QRJeK2kJKVUQc54bL03Wp2w31ihywC0eUqAXkfhjr5JAM+zPm1TQ6TVoQGcvIZO0Zyf736\\\\\\\/K9IoSQ2n2KIH2KLaFCcgIuarOtMdatGoO2xdwIBM7dMInY2h5mMSuBPMhue\\\\\\\/OHgJ+b79V+Bp\\\\\\\/QI9ElAAqn2yGU8gImzjLe21po0vMnjMnHybenjoDiYpDJHgNfVJ74TvshiQuZj6w3GUqeVnTexWP2yaGYwsZjjzPhTn1JdMvOSRQRPpHqpmIrcvDgM5\\\\\\\/+MlRHdGugx1k1ItuOySz9Y4mpK8LLLpetvtyAcKT8HewSfmmpKzHnOPI+uy1WZ6TH6tGzgDYZUMLgNjLaoqJ0E46v\\\\\\\/w5ovzhoAdrj0ToJD80UT6To12DW6iKWWQOFMitBGc81rk6FkStqZ3uq+glJD8TYuz0vInr2E2cyQi9Ypy1mWCKjlu9eJSHVaU82LgKDMQA2ERFboqDn0UQ6ke5ChmJqhsoRLvuYW3mfT+f+AHRUK13vvzOb8NmxZ\\\\\\\/G\\\\\\\/BLeLUIGDrhyZbopo7qYpactq2WQRIj2hUPGL1OT2of1sOi252g144WWLL8wAf1m+\\\\\\\/zxfavywaBQNKLNhBgV1\\\\\\\/DYpSlXztGGCjh6Q3snrTZHUjxmoTjf+iIyg\\\\\\\/jLJ8StweQq12pexHZWNKgWyq18R2ibH67XGbsu8rYMNXxRSBYIYcyXzbebbZ6tPmKMWqnpFvJEiBtXEGveMyKYQTTVBJEQREHogxQf4dpDpSTIWs7D\\\\\\\/LTcj6W15E1b1mKKBxkRTCxkLvK8jKFJVAZDPdBgvuiqhZ3fb8zE3nqpaCLl\\\\\\\/ljrRp\\\\\\\/eO8DpcaOWU39G\\\\\\\/11LIm2wwU4qYXk\\\\\\\/giEgi5ydXx1rsQb16TFmZw2HPdPp7oLOxJflJX0HJCwp1mBNdCj13Mi165Hk10FCuN\\\\\\\/arFIIKUXgTITisN4TwbZCSsWTgRnVdIQReD6AnHJ8NVyL98d7KW3G81qYY8XaZqZ4\\\\\\\/myzOTnt6QCqQCLEejEkw8AKAsPchFoaziafk8MVzRuuUauUWO4++nTPnzPbFtFaxfStl\\\\\\\/dnkJsuwZnLDnZuk6QkGJFaf7IbX30kCSXK6rKPmhTdp0c7zQhlXaN5jmZ1FpjDNsyN9T7lNQ\\\\\\\/D1xY9dcXTtKrJOo2SDnN2kZ0EXy8HDQ5\\\\\\\/Mmcy1Q2teYaaNxmi1F9boK8pPwRqT2L+\\\\\\\/JkM7ui114R7DHM743963CJjx68BW1plroVj5R1v7GlJslwvcHEUik5n5fJQ9Ysl\\\\\\\/jy349wqCXDxZJ5YiZzyARKfA1sRN\\\\\\\/yrkMkKQ361DOdWrG\\\\\\\/1ralP0OoELaSN9sOv8tJ4HCJ+l1aU18zGOdm62EkYWsMXa8vL5Zvi0MKdxmaEaBqU4UjahVNqWzIwxTQlCeKI8kfctYkUu2gubAPvseoAP1zrRyV9lRS4joRhW5P99Qb7ktjRpmRy+9hVk04ykYmdvs4ocZkrIOlvWyN8fL8aVHA12cQfncbaZ3cmWZyxfbKwjrYmMF0YW8kB46unELjTf4BvKB+JBDCZJ6PAWoid3hM7gLmqiAbtMIO\\\\\\\/QHjC6Afl8\\\\\\\/ieU46gZtCLB8uXSU5+HGg1X4OxRy2OYKjXGBcN3aYdjTr4qhUB0+kzGVqioKr1lJm2OL6WzkUhXyt63RfeJLCpE4u7CfKfKCze6UFQtdeZ3yMk9MrMFZMSKd+SpvCNqVdwM0ZuFmx7fHQTaks88RLUdfcsHYjhBuk2\\\\\\\/2s25Eg9DokQBM1al5j3CYXJYozCog3hEpaO2R9ahva+a7lbusmiTM84M\\\\\\\/sRxM9YOtBRCNddHSGHV\\\\\\\/yAfUis1wgqXWo3WS2iK7BMYs71c\\\\\\\/VfIECldu8bRr7NkyTgDWwHZZR3Lm0RUfhr1pg++4CUWRgUzy3keLtdV1uF6pGyWC2R8olf6G9\\\\\\\/HQGk6shJbmIdVZq5JfGffK0+y2Tz3bCcSubRyM0jfV9tC3VSd2XKaLHc2awWMCgS2YbtcwH0+nN+\\\\\\\/fLhyThltfJVH84OIHbNj\\\\\\\/fLPKSrbDNo8rxygXsSy49WHGTo4IPTPwKEToLUElPdwOjyfAv2wpoOWLYoq3fhGFz78RQ7v3i8ri6msO9D76GWHOCpHTXGJyBVHLAUKaFt7Z87wfWWxT\\\\\\\/4WvTPhG2IBB5bycM9NRglL2e51srZuetQw8rHTnB6h8NVTngQK5t9g2n9MaMtrb4On0wY3RBVc7r1dnbUUN8ZDVaPHHmuouWKAxpZCUh5rCHLTDRLuzYb4J10s\\\\\\\/\\\\\\\/HgJODxsVmOA0RmYdXQkw+XSGY+z69haqAbxENh4T6YR5FzyLeke\\\\\\\/rvwpIQUM\\\\\\\/5BLoXJz9glayJfpbTrA4LxBA1Rxtjz8\\\\\\\/xlTtjV1fLraaFquC\\\\\\\/lsOaXS8evc\\\\\\\/KJj69CSB9rOvRPyAGVCBkabVLn6w8uEfwjooChTqR+pRnDfZDZa4KjJegx8cyFxNAfmXJv3Gtp4uArC5Aiu42p2PgjGUuMC2j7G1AvJVJPreSsv+jVBny8lA18zuQn5hGWfS\\\\\\\/Ik80v8VYhMsgGROca8En5kOhOGVuB9P5KxiUsmDbgF6odp4pAHemk05UQl3E3htXPvXnLYKKNdIJ6LaIfx2x4iWduBLI3\\\\\\\/13DQf22qSdVbGQ\\\\\\\/jSK5majoQdwZj8u82kQbyLzDUz\\\\\\\/I7M3sGvWKNCPWH3NHl4gjdTxMxtIOj5sdCS4jAvF5TN+nIjhR9dNO\\\\\\\/LWNbSHTqE5X9ORlYDVNF9JlMxc\\\\\\\/aJCbLzGPUxI83zdAzf5OydW4pY1kaBNnywX\\\\\\\/+cZ5+\\\\\\\/sGajABgAVVXnZRbBLsVMVP+gseaLg+s1I\\\\\\\/1Tr0T5s4YdPrZyy0eb8960N28cbpyIve9u9bi+TJK1OcyWBPH4t2aqDK5esuEXCRgnoSWCU7m4XisaftRX6afTMu+H7M6GD8RrCW9lq0iSm2clq\\\\\\\/NXJBT1uy8ZPKn0Dq9nz+snWHaPuN3iTHAsTKnzlr7XSQvo0Mjjjt+yGMLJeM6Ih4o6pIMqfjwUfgCurNCNfHaF3Oj5L2xg7vqY1MgNWyvzxO7mmavIs0XDoFLCvszML7Mb1IBlahxwJXKc5BptyM9o+chYI9x5tcUjUmd\\\\\\\/gWGrztxyMhqD4ypQJSFpUaeGoQDDgJCnoTVkdZ\\\\\\\/EKHmsWPdI7dYMDAU5l+ZVn7MKjIxBD309WJWDQbYJKwwqlMlzhi8H5WzN\\\\\\\/7AzNEQb6ZEzgNfCraqL\\\\\\\/0x75hCnYb13IQ+2ADRmWeUfNl4tZ9fP0ofMYWCrhQc456WOOsm7NHSfOWRNnJ1u9PLQZpOB5F6JuMi1lMO2HFOBBSa7CXqjRuIPXKnu4X0D6FGhl21dr4Y2WXb9JX9GZu3jEVMEgqHsgtvVbEJtAokP4q3Hbmr+VtfCv3xYjGEsoavFWDnWJtBgMZ7ppuPJddYvYU\\\\\\\/6gJompyDO6KZE0qX+MxnXH+t\\\\\\\/Vr71FnWlS1ZVUUOidjpxzHT7tnUze2b410ZJkr32ro+8YQfvN8rt+r7qbEsszJ+QGIp8hyTnfLGPhnS+UcUG+uBX2AOUT0vDz2LgTtAF7JbH0BzThN5HphCxS7xVGnB9P6MCdIN3m10CrQXiBV5Ib5dVLQ8DF6aonmeVdT4GYPF8l5fliXH+z0erqjlJ7ygTuwLHYQ6xsmOhHlL6XvRVJDObN0KsbkxFniUuwIHrh5U2xMeD2NERsbvT9uFADBk5Jx21oOPZhkuACD4wo\\\\\\\/TK6N88SxB9FU4ngi97bwAhTnmfxe1DA5ruVJSX2vsncUmsCjJ2eJkg1eGO56iz1zx8VXrPgjKipjroWSDfMJ8ijrUhWI372f7JYPn3hoGdyXc1AP+ls\\\\\\\/Pxq77e8iazT8dyJYYufpZNUSut1xK5IzG7nxpTTlYLq7NK5q6zi1scVlfRatJYa65iZLZ\\\\\\\/KCglpT+xIevwFgp03W6ffmctso8AjYVgBwWy\\\\\\\/Y4U+9IAbO\\\\\\\/vziHR8xmrQqYVbDT7xP3MY8va\\\\\\\/2TFW2i2JR4cNexTAdaDn+y8t8XgL4wT6z\\\\\\\/gk8uO1kWaZWmA7I8mSv98gcnEbRO\\\\\\\/zmrn6iiNTOqlawHpStzLYzfjYLvkYq+jv1mPBalQz03\\\\\\\/OwGweXqnc3d8J8lWrF8tYWpy1D9mtxLtEl6C3k9fhz4MieOeXqEtXnQAV4vsGW\\\\\\\/Fk9JMRCYyuqbhgp+uH8fbX97\\\\\\\/B1dMwrr+QgLIzw0+OqHldF9rSu8lL9ASFOft3GNxuIhrBX+IK5f9hUDofOdgLSNJ+1z2CgT2aTiQg3p7znA46psMVnao8y84XCqrF0nUpAZo0dePSaKjNgWUBAuyAI6j\\\\\\\/DcOuDAdthozwUbn2faGlYby7KGOTk7YMv0tI9VEJtKjR1TzcY96wX4OvlCPgV8fdGV8oegNKvg4MkKupTNxIICoJF2m7ptIPeFj6pdBA8z5YYh0CzsoK+49rZMP0WPAs9\\\\\\\/LbH1M22Wx2AUyKkwwPV3Z\\\\\\\/BzEQgdu8ghKWCoM7GLqLOUmq8wBTvYY\\\\\\\/tWyGDxrquyl4c0lyEEJA2IFx7b4Nb0zEQ9khMkvnwfp3bFCgHDT2hIJCSGeKqJFNLgZhk3rLMl2Y9bv5Sp0PFiPlCXvCjMSVh++gUn5bVJuy774EGh\\\\\\\/IbiHQDpJCzCF4lN1t3wsR477k4GCrNubjq2n+\\\\\\\/gDm9ZIdXF1O02dFGwjAWKPEZaStO83dW26uWrt65I6spYI39\\\\\\\/Mvw\\\\\\\/IQX4jtZQV\\\\\\\/9YdcW6zK8udkIXzKBz+sGbRCvfUmmd9hliN081IPxn6twbN7Gy9wutdDYUdIuUXqS3JyPvD70iXrK\\\\\\\/ahaGXn\\\\\\\/kHnTXYfylkxKix\\\\\\\/A0t1BchEHTEKZG9uf7q7j1GThR6E5wXZCyeY0LeaRjXJR8gnka0sr0IDwFgTTmHjz+bH8k\\\\\\\/xx5Y6AUl\\\\\\\/IKYZUSTstRlCD1s6B7FWoZYtqEbvsCvS+qcQQHFln\\\\\\\/Pz\\\\\\\/DmNhSaGcXJxS0slpbp8JlAHYfkUiMqa8rEUnRZFfd\\\\\\\/8ici4rvqHdwmXxnJFvr\\\\\\\/miKUnGFE5hezJs6\\\\\\\/YIju\\\\\\\/fuStg9q+uv4QCZ0xPGM0NvmgINPVqvchCFbENmZ\\\\\\\/2qZUlpKJ15RfiDRyDHP+4p3iB1g1l9\\\\\\\/78s48nk3ZPqPiI12weMRowZZqlhyzKR5ZxnjvEqEkAlaxzp9iIYuJ+gu7YEzUMeOOnCN1tHIy5Sj9UnxwR4w2tIuZTdm\\\\\\\/7b5GqmCEdVzz27l67VYiPVYsAljo4K7FMVe4wZvAFfR+biZOqhOBxXt1gP0NTGanve5V7I71\\\\\\\/Mmuo9NR0bhVurCP\\\\\\\/kTE\\\\\\\/DIedTsJKJwARALpquWX7Fz7E6ChHtRZiCdwepLvRUtT7738\\\\\\\/EIbomGB739cSqKB5mBk47a8SNCBfIrvVyOG\\\\\\\/ouzCee1+KTUWJ36SMxDi\\\\\\\/YPMLN8K5iNTsHsm1pu+zT7KwaBBZ3Qb8VgFcLZ3Yow1G9JRoYGim0Dz0Z97OOb3GKayZqZ6XM87PB3Psn9x9FPoDfVfRqt1z37c22+K8VEtKnz2QiV4vT5HfuKMbunLvWfX5oTohhg12spggz+JTPQFqDsvxGEKrFQVnVQrDJEN641LP9Jgwp\\\\\\\/a0vbKq\\\\\\\/LawgU\\\\\\\/QJSJpY4fO5joM75jmQ6KZZBM4A4dA3qQDShJdpEO5\\\\\\\/QUIc5jpHVSMne\\\\\\\/UIODM58brEXI8rap6Ax3QgqawMX4d1RU9c0\\\\\\\/HCpZ2AAHLd+FTMTj1DZzYpXzOBg7sC32qTS5ylBvdJDM5Qw9f1gwHOqUtJNEKCMrduko8WM3L8OB65OVRLPWOmjeTW7kyahQV40r+10OVhPuRkEExAkRlBFV5dTUq54jvaHlsVNwaOKGhdfzSiqfgZodpELz8biRTVlm+9szZL4GMZwn1cx8BKOBxcOYMy8bW5Epz0LqRaAdTiWgH\\\\\\\/9bTnpCsXcWfpNHmV\\\\\\\/ecpOyPOpQgwobL\\\\\\\/IMgIiNWGjAF1xcCB\\\\\\\/mTreLn6xVM8FWNRfxLCw8D9JlEcgOSK9eTETKNTEaG1\\\\\\\/zy1Zbv776YiAFGON6FxCu\\\\\\\/7HVyURBjGE8+fyQ6FeOxX2EMxc7Ba0MxSjA2RSXJGLZc\\\\\\\/OFO6+BIljW6h20lLAwa9mtp\\\\\\\/qbW52a6e8UgpOSpfugVi6oP5KNk5nUgx4eSktm3b7WOS1z0ogqqSImG4ZzZjRVsY4xUIfQvQeW2W1PrCHCNprC2YW5qCUIQVAeMGY44DzUB3Fh9QrV+LFe95lWNEhUzBzZnG2qqyByYj+d1mpByRzsY1QnHopQjOlemioGYJhtuDj8Fr63qgwOKKcdj3fn4U6Zr4iCQrG\\\\\\\/RCPf9cl+kAZCkTHeeBM4eh4tmTNBmwVOubPXBayDRFcoXo6hzAG80\\\\\\\/sx9Kr2SeLFvbVwR92BcZTwpS0JxyoflvP45+oOWkVq5o+N64D9ANHFxj4gkCZj\\\\\\\/ZPUWzxYkd3g\\\\\\\/BluJG8jBiNnCI1fedqNtvb3\\\\\\\/2fofHG+nuHFmdwxorEWqHWYLCcUmDH8VQowVOVkKrvS0fssxohIqIhXA9uZXaTLFXIP9\\\\\\\/plVBrcjKWYOOCOe+CDHoWWSwXQOYGI6tR2IZvbXcY0n2WmEuFFqvoOZg+YWMHglz4KvoGRKBWaN4+FBnwhLzhfS5wjzsz2WjfB0AoS\\\\\\\/UX8Ma69\\\\\\\/nQpwMiR2ulGoXIF4wdA9Dqg3CAHDEvopE9UzInoXuKxaxNCq78EFi3mrf4HBG3XtZP8xZketIj8hXXB7o+4AKPPepee6xxBew82\\\\\\\/t0KQ7cW0wmO8mSVGtUXkDXbFJlpgxHhouhXl5HrgnufkScgcPUN7GVnCGBDcumG5YP5Z26GZbkxIShxMmfrFaCqgWAKeLFC7h3uhCdgzHag0ij1CJSJItkD7ldBg76u9TDyT5VEEx2tDWA2svBOOyT4BDR9oUwVWFOHhkYYtMJxD6UB+mNBEI9Cp+fruJSL7FRVXhSz1FtgLepShcdiyctV3PTzmHIKNgpLWlgIKqbXG9CWgmoRJkJiObxnbkZlOBlCW6zlr+8iw3ccV0aLf5QxygmV2Rq2xxE2okZKT53rjxtJcDOYTt6Gr\\\\\\\/62gV1MZU4Qup097b8jJZSzt6vHdfVH2zh7U7pUkQD5BahatQGeaNbHV+CtjLLNGpvq7g6ZEaPnzo6sFhPW+Ts6nwZim3W2K4uvqpCQzHX2IKEhxL73AWlALKlE7EUuSyw84T2FbJEU4cjnbHPF9vBXwtBOGQnpS9qy9pG15BY8mn5c2YGpB0lAag0s48F78HaT16Cd9g8qpM150t3y2UdV7CVqOxcSSWwer6O6hauS+vknx\\\\\\\/Srk0JNFD76GPNjp0Lsa0iJaZkUtxhVNxvO8va9uZG2iPOEybgbTVA4R2ckbXrBjaCePSPaqQ\\\\\\\/3PBPBGKK9wHwihwfdrWtq9qznEABusVcO5bUouyU24DVUZKhLmWpElPj8ppyRtxdZgxYRyEoVwdbt40pESBPue+Ls7fNY3BYVwHB8p6aUgPaOUJJFp5GnTrONBY1NgTAisVMMfW\\\\\\\/vASrUtQyM\\\\\\\/6wApBE8DKMXMDPzXvOEcy4kqf+jTIsoXwfe1xC6rXdkm\\\\\\\/3YnQjg\\\\\\\/nQJE5LyVLHreUfYzLPlbcjzUua1vJqJ+6At4Rn99XqYCAkON2U9NY3L5b2iObTZnghElrszT3\\\\\\\/UpHWpaO+fsQz0lVD7Yc5fNkOLH+2FVrBPgWwNRoRwlJ7DdNKirKTU4nlazEAq1ga0E4St1vWVVfnTcP9xSKWZDGLaAL0AdYP4hC2l6S6mJBEN5xa07za+XgD3efCCf42IFJwlQ3+i6\\\\\\\/+14LpezDw4rIUI595hXSn9ZDsduUnQjrXUTA9vhQrLMBQjWwE7JrZ9E61yL9xQ6FuKg16p+yk7\\\\\\\/sGrx4fn+71Feyx+lu7TeaGJ4Q57MxjpPTeA7k5M+rCjtw7XNk0rNFwM\\\\\\\/0J\\\\\\\/3QrQnRWLpQZCa6uq1Jz4G0PAL5f2nql4g52A0j08sNkl6NFu6NYxnYU2qqPrtNF\\\\\\\/ELpcf6YWu9pdASzGOkNThcKDQmiEIqqVxP86Jk0N8Wi+ejeEMRsuDaaYV+uoa8Tb0E8GFdl1qI+cqHcBxxzaCMw2Qvml6J0k\\\\\\\/Msj1H+zBZ\\\\\\\/m2O4Jo5ctkHOqAhit0Cw1oi8R4wFkBZl9raEqwtxshGr7qFAH6mqSduUaMZw4Y5MJTJ3xl6WPerjAp9XgZ33miKJqCZ3FOfsGIo4osxtvbG9lxLjQv+Eiyto8AdCSRinp+5cflT4qnz7isRWrXJluviolxQFs6eDnqg+2RvZbK+vkJs099ZCFi\\\\\\\/rmgCD0p52OfECY3iSE\\\\\\\/ZDFYorehJs343TaMlQP61+0ZhbzkoizS\\\\\\\/r25fB5E\\\\\\\/nfn3adRt2EU3hDAaRIXLrama6I4t6BMSkElmZh\\\\\\\/YsqMK4nAc4CTxG7kHXQmBLia6sxE+doIxyHBtm5k0UFElwk\\\\\\\/KX3O4BBdIFn\\\\\\\/jdKyhRWCLTei+mIDDCkVBaNj2LVhqb8F4su\\\\\\\/RkOFJ84l5ioXtv0YoQhAYZyvV9toGEjkaPwMkIHYQESrQDACYXcZjfWWcLMovB0ZaH866toGMywgFe+WazUZhDrz\\\\\\\/oiCS44S4ApZovirz70rhQ0Ehn342M4s1mGqJM9+yy\\\\\\\/vsO9G8zdTmFXEbL0f5sOq67S649ns1KfgSMiet5sEf\\\\\\\/4cXvFfCdbWMgr7vKoqZW4OPYMN8nfyjaK5NHFX1sHqbd1kUU3yRMY8InA\\\\\\\/4npYy8B3oMop46deJ94RWfVRcQmfFJ4H\\\\\\\/gh0Gu+fxZmJkNB9ylZDpXiW48t0xsmqNrN73mqZIL6PrRQ9lulw0yn552W\\\\\\\/xXjvZQ6Qs6GjV9V9D5DyJ8l7ikYV1v1TyYX+HIglaDxIfBa55BF60ZOpc9paixrC5Z2prR6gchD8xYS5hHXgyGRH5oJb23QHM5ZYkDBuPPvtFK+BOHCHWdcimQGgbb8Ycu0hd7bgRfeRHp36j196br6nLW2oWupp21rjgEccPaH+9+PsD\\\\\\\/nNdrZHbBO9Tk0azWQPQURR5OZ3q5FAcEqEzBR10DDBO4DuKLTD0dgCvpTYkOOJq9cMs27rqiRsrcS6YzyhbLFaLju6gT+Ze8KFiTx53wa2O1JIa4AdaHKsgBunRRF\\\\\\\/705QamyrCfyWHA3eJolIr\\\\\\\/hTD2u9e45EEtwuED2JQsKLKeA8vNcE6Xm3jKVbzTeZ4AMsvEGV2VLtVlgdqOHQQc7pG4+vDUCM8GH+zbXNEOuGq7hqxjEN1ppg1+uozxNnj5782\\\\\\\/Pqfy4qvnJ9bxBPfEQFoj20Y0dkX5DSKZ5Q9KRsusJkChH99ofekKzxOLYsd2T3vCsTHECIjUMBfimkKPROVEiqa54pJ2F4hCMq7WTWBo8Jv9Jw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022eyD8Ki9vy14XJXlSf+S\\\\\\\/kw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cab4abfbe7403a2555a228b118dc0b77a0dc2d1c50718bda242cb06b70d22ecc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223Hq7UVyQ2ZuJTfrHznOAzykfTo2g4tTRySxneNdHQrF95U4m1kc06Xb9CfDBFMiFGYxzVING+Vt+xGw2IQwPWoU28GDzjZtQcme+g\\\\\\\/WWBaTHeQh8PfhaF8kOtGf0tTqyygKPrRMdsTusTT4X7cmGZXyCHkae7hwzjiSuqsbzCidvI9\\\\\\\/1OFrOxTTK8uF\\\\\\\/daOu4mHwrIAOW6GSSgb3tieWY7u4lE6eKrlg5nbuZTL7C1kEjWHBH+3VLUZdCEfeE4oy4Lb5hLpSDCzn1MCThzIGzmV53CTI8Aclp9mUZ9QCKBvlNy9IpBd2j1C61MqdvuVycqcnyz2kBB4hxqnKold3okrRWAB0tHKBOYGKCsPFKccpdlGM7YdPm8I6hjHtOXsryJqHU4Y5P2+VsEipCr4w5\\\\\\\/vIZNJw41w4Nv\\\\\\\/gdP6dMZyclRXn6LmqGy3RGDcLqVA3n7eElfo\\\\\\\/\\\\\\\/hnM+t9JSIktiJiY5IjzneKhQUO5QPbibMaxenY0mbDs3SWPtqFVnI1LT1\\\\\\\/6MrV8U2djnZu1VQ3vIOwYGajFh0a7VD7NZirpG36T8cTMl75JU2uSdk4oJZU7XEx4VkoSlJ9zPwrr2a6bQIfsHv4n4b4+wPIoP+stZETVofGGcKRZy+fMadaFmXduc4TrM\\\\\\\/Wc743bzY+GY6uSJH8B+x6U46uRFNm395kGm3DWidzfXKwU+UvdehqkWoJRz7\\\\\\\/qPH61QGla4ea1MQ8yf6ZmPn+zb9CKoyIn65subivkDMNMQP+6mOadgbqCFS9a4J3zLP5gafA6berHdBJgNXUsOcUwg\\\\\\\/c0o7sfGRunXfWgIyKHKMrzGW3gB1nFTYtRF68kDn1aedwt1Zb7SNXs+nIwdb+kxrZc+Wt7RA2xfYh1JY63npySiA+DoRnDY1NHzTvJVwqJIBykrvImZXU41m5pgDlgVK0kGMl4D937kkzv+AD4VhpKr6X8+dKCR0KiEe0WIiakzbwVla9wk85OHsb14p90CKrSSgyptSwj59FXcVKufJIz7sV5iXk76duEyF6OokMNNHdi91PrbCCHUQ9DKOda3OUIeVmS75VjDja7aPeZxU+VaK9aa6PfPyXkBP89ldTATJmpNefX\\\\\\\/ivqWB8jB6tRf1e3nGwbhTXjSlzgotERf5urdQs+JXoUwa7RKzSJpo6rfo0y6Rbe0DTFQ77XQqiivgblIvvKsQvF6RkX7UHucFuw0HVBOu3HiAUkEmk9sibvjtvRQltGPStNaZwx4veDXV7wlC9kdTqBynMaJ8M9QJbHTLIgzNCtFhhk78VZESVADAKnH2czCj2Kdf7CeXLZ7TgMfQpF9hKbtIObrUApkI1aQ78sDyhhBKBfUSuNczFmEr+2DkJOntudrn7ytyN0NPxTb\\\\\\\/16D56Q7rxCnWbsteoLIfq5NDMBkWC9S3KpwTd\\\\\\\/2wJqwKWeYscsROSm6doVz0UIontg0OxQJFh+nNKOcOyq6A6YnHLjC2jJ4RU+cFF80BH+3oOSI0eyN258KB2uEGw3884KfXskL0qG5beRuQhltugTC3ImePlC6PVBGc0PYxYbH2p+PEdIi5Tfkb+E\\\\\\\/+tmPYlSXy4x43M+EJoAkTXKWse4PZmk\\\\\\\/MOJIv7kD63NHE7WRop3A3wa4AYEqTN72PuF0Yvx6DY1FO\\\\\\\/ADNBL\\\\\\\/i6KchCz8nQtSOcKl+nfyKJSoL7u51IeGjfMEfFc3AYlkBT8A0OSNoJwtEmWcisa9KufND76Dqz7Wb0OQ9fMNy3WP4uGh3KGKtzc5BakIn5CYxlFhM0PxV38m1uGcVt+uJsSChCjHQKsr6VC+7X2c1glbEyVxpuakTMzCT7FicfB60U8gL0H2bGVF7MBQmz3\\\\\\\/NJ3t1q9IaCe+B8bvjUftNIKnkkcYsBLVjdZzGJ5c6SIJoIOxHXHyJVdYQBtJaGstYE+LNHfZkZPLw+xSxGoRFBI1V3LEukT2jZTK7hQqBA6hUwo5MdN5rJdMUSFfl7P3ZAfa1iaSf51TPa0S0Ij3inE8LWdEPvKwJyWqw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BGSvgm6z9J07Pm7J1kOxKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227a43bfa2298f50f6a9f644154303b5730d923da51cd433c4b7b23ce5bb52c54e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460001 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1721" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4SSuEyvPtyglvCgmma2-xs_ABYb4WtKDNpTxgOQp95I" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1721" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8vOnNct3kksGiRj4czsU1Gko-oiQuw9NZq-nZ5p0sMU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1713" title="Chromatin Signature Identifies Monoallelic Gene Expression Across Mammalian Cell Types" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1737" title="Gene Model Annotations for Drosophila melanogaster: The Rule-Benders" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460001" id="node460001" data-pisa="ggg;5/8/1721" data-pisa-master="ggg;g3.115.018929" data-apath="/ggg/5/8/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Model Annotations for <em>Drosophila melanogaster</em>: Impact of High-Throughput Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Beverley B. Matthews</span>, <span class="highwire-citation-author" data-delta="1">Gilberto dos Santos</span>, <span class="highwire-citation-author" data-delta="2">Madeline A. Crosby</span>, <span class="highwire-citation-author" data-delta="3">David B. Emmert</span>, <span class="highwire-citation-author" data-delta="4">Susan E. St. Pierre</span>, <span class="highwire-citation-author" data-delta="5">L. Sian Gramates</span>, <span class="highwire-citation-author" data-delta="6">Pinglei Zhou</span>, <span class="highwire-citation-author" data-delta="7">Andrew J. Schroeder</span>, <span class="highwire-citation-author" data-delta="8">Kathleen Falls</span>, <span class="highwire-citation-author" data-delta="9">Victor Strelets</span>, <span class="highwire-citation-author" data-delta="10">Susan M. Russo</span>, <span class="highwire-citation-author" data-delta="11">William M. Gelbart</span> and <span class="highwire-citation-author" data-delta="12"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">the FlyBase Consortium</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1736;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018929 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460001" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beverley B. Matthews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beverley%2BB.%2BMatthews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matthews%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeverley%2BB.%2BMatthews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bmatthew@morgan.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">bmatthew@morgan.harvard.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gilberto dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilberto%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilberto%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Madeline A. Crosby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madeline%2BA.%2BCrosby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crosby%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadeline%2BA.%2BCrosby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David B. Emmert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BEmmert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmert%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BEmmert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Susan E. St. Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BSt.%2BPierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Pierre%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusan%2BE.%2BSt.%2BPierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">L. Sian Gramates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BSian%2BGramates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gramates%20LS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BSian%2BGramates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pinglei Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pinglei%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APinglei%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Andrew J. Schroeder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BSchroeder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schroeder%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BSchroeder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kathleen Falls </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BFalls%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falls%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BFalls%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Victor Strelets </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BStrelets%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strelets%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictor%2BStrelets%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Susan M. Russo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BM.%2BRusso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Russo%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusan%2BM.%2BRusso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">William M. Gelbart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BM.%2BGelbart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gelbart%20WM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BM.%2BGelbart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of Cambridge, Cambridge CB2 3EH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of New Mexico, Albuquerque, New Mexico 87131</div></span></div></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/8/1721" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460001" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460001/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1721.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460001" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460001/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1721.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460001" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460001/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1721.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460001" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460001/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/8/1721.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460001" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460001/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We report the current status of the FlyBase annotated gene set for <em>Drosophila melanogaster</em> and highlight improvements based on high-throughput data. The FlyBase annotated gene set consists entirely of manually annotated gene models, with the exception of some classes of small non-coding RNAs. All gene models have been reviewed using evidence from high-throughput datasets, primarily from the modENCODE project. These datasets include RNA-Seq coverage data, RNA-Seq junction data, transcription start site profiles, and translation stop-codon read-through predictions. New annotation guidelines were developed to take into account the use of the high-throughput data. We describe how this flood of new data was incorporated into thousands of new and revised annotations. FlyBase has adopted a philosophy of excluding low-confidence and low-frequency data from gene model annotations; we also do not attempt to represent all possible permutations for complex and modularly organized genes. This has allowed us to produce a high-confidence, manageable gene annotation dataset that is available at FlyBase (<a href="http://flybase.org">http://flybase.org</a>). Interesting aspects of new annotations include new genes (coding, non-coding, and antisense), many genes with alternative transcripts with very long 3′ UTRs (up to 15–18 kb), and a stunning mismatch in the number of male-specific genes (approximately 13% of all annotated gene models) <em>vs.</em> female-specific genes (less than 1%). The number of identified pseudogenes and mutations in the sequenced strain also increased significantly. We discuss remaining challenges, for instance, identification of functional small polypeptides and detection of alternative translation starts.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li><li class="kwd"><a href="/keyword/alternative-splice" class="hw-term hw-article-keyword hw-article-keyword-alternative-splice" rel="nofollow">alternative splice</a></li><li class="kwd"><a href="/keyword/lncrna" class="hw-term hw-article-keyword hw-article-keyword-lncrna" rel="nofollow">lncRNA</a></li><li class="kwd"><a href="/keyword/transcription-start-site" class="hw-term hw-article-keyword hw-article-keyword-transcription-start-site" rel="nofollow">transcription start site</a></li><li class="kwd"><a href="/keyword/exon-junction" class="hw-term hw-article-keyword hw-article-keyword-exon-junction" rel="nofollow">exon junction</a></li></ul><p id="p-3">The <em>Drosophila melanogaster</em> genome was one of the first metazoan genomes to be sequenced and analyzed (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Myers <em>et al.</em> 2000</a>) and its manually annotated gene model set has been one of the most carefully curated and highest quality sets available (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Misra <em>et al.</em> 2002</a>). The updated FlyBase gene model annotations (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">dos Santos <em>et al.</em> 2015</a>) are exported to GenBank annually and serve as the source of the <em>D</em><em>. melanogaster</em> annotations in the NCBI Reference Sequence (RefSeq) database (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pruitt <em>et al.</em> 2005</a>). They have also played an important role in informing the gene annotations of subsequently sequenced arthropod genomes by providing a reference set for use with gene prediction algorithms that incorporate a homology-based component.</p><p id="p-4">The strategy behind the FlyBase annotation effort has evolved over time. The initial “stem-to-stern” gene model annotation pass through the entire euchromatic portion of the <em>D</em><em>. melanogaster</em> genome was completed in 2002 (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Misra <em>et al.</em> 2002</a>) and was based primarily on cDNAs, ESTs, protein homology, and gene prediction algorithms. The next incremental phase of FlyBase annotation involved updating specific gene models in response to new data including cDNAs released by the BDGP (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Stapleton <em>et al.</em> 2002</a>), community cDNAs, and gene model updates from individual publications. The next phase involved evaluating data analysis triggers. In 2007, numerous reannotation triggers were generated from a comparative genomic analysis of the genomes of 12 <em>Drosophila</em> species and phyloCSF, an algorithm designed to identify evolutionary signatures specific to protein-coding genes (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lin <em>et al.</em> 2011</a>). This analysis applied to the <em>D. melanogaster</em> genome and existing annotation set resulted in the identification of new genes and new isoforms of existing genes, corrections to existing gene models, and the removal of existing coding genes that did not meet the minimal criteria for conservation. Additionally, translation stop-codon readthrough predictions based on the conservation of protein-coding signatures were incorporated (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jungreis <em>et al.</em> 2011</a>).</p><p id="p-5">In 2010, RNA-Seq coverage data were assessed as part of our genome annotation effort for the first time using the data from the Baylor College of Medicine Human Genome Sequencing Center (BCM) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Daines <em>et al.</em> 2011</a>). In 2011, vast quantities of data on a genome-wide scale from the modENCODE (model organism Encyclopedia of DNA Elements) project (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Celniker <em>et al.</em> 2009</a>) began to be incorporated into FlyBase. Of the many diverse datasets generated by the modENCODE project (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">modENCODE Consortium 2010</a>), three in particular were directly incorporated into the FlyBase genome annotation pipeline: the RNA-Seq coverage data (both stranded and unstranded data); RNA-Seq exon junction data; and transcription start site (TSS) profiles (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Graveley <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hoskins <em>et al.</em> 2011</a>). Several other classes of modENCODE data (chromatin domains, transcription factor binding sites, insulators, and RNA-editing sites), have been incorporated into FlyBase and are shown on GBrowse, but they do not directly affect transcript models and have not been assessed relative to the gene annotations.</p><p id="p-6">Given the quantity and quality of the data, and the likelihood that the new transcriptomics data would impact most if not all current annotations, a second complete pass through the entire genome was undertaken to update all of the gene model annotations to the same standard in the context of the new data. This article reflects the annotation set as of the FB2014_06, Dmel annotation release 6.03 (R6.03).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Philosophy of annotation</h3><p id="p-7">For <em>D. melanogaster</em>, the body of data informing gene model annotations is extensive and varied, and includes both high-throughput data and detailed characterizations of individual genes. Our goal is to integrate as much of this vast array of data as possible to create a synthesized and consistent set of gene model annotations. (As used by FlyBase, a “gene model” consists of the transcripts and polypeptides produced by a gene; it does not include regulatory elements.) We have reviewed and incorporated millions of pieces of data of widely varying types and quality into our annotations. Evidence types include cDNA, RNA-Seq exon junction, RNA-Seq expression, protein similarity, TSS, and gene prediction in approximately decreasing order of confidence for creating gene models. All data types have strengths and weaknesses and all datasets are prone to artifacts specific to the dataset.</p><p id="p-8">We have taken into account the limitations of the data when creating annotations. Cutoffs and criteria need to be determined for each type of data as curators evaluate the new data types in the context of previously existing data and annotations. Lower-confidence, lower-frequency data may be valuable because it sometimes points to interesting but infrequent variations in gene structure. The difficulty often comes when low-frequency data predict genes structures that do not conform to our expectations for well-behaved genes. Common examples are retained introns and alternative exon boundaries causing frame-shifts, which generally lead to early translation termination and in some cases very short predicted polypeptides. In some cases alternative splicing leads to downstream translation initiation, some of which may actually be unrecognized non-ATG start codons. It is often difficult to determine if these represent artifacts in the data or interesting biology. We have chosen not to represent many of these questionable predicted gene structures as annotations. We require a higher standard of evidence, more than one type of corroborating evidence and/or higher frequency scores, for creating unusual annotations. As more evidence for a previously unusual gene structure (<em>e.g.</em>, long non-coding RNAs, noncanonical splices) becomes available, patterns and rules start to take shape, and annotations of these types become more prevalent.</p><p id="p-9">The approach FlyBase has taken differs in some significant aspects from that of the Havana Annotation group, which provides expert manual annotation for human and other vertebrate genomes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Harrow <em>et al.</em> 2014</a>). The Havana annotations include detailed documentation of the data that support each component of a main reference variant for each gene, as well as for each variable component for alternative transcripts of the gene. In addition, annotated transcripts corresponding to partial or aberrant cDNAs are included in the gene model, with tags that identify them as problematic. At FlyBase, from the start of the manual annotation effort, the approach has been to create and update gene models to represent the best synthesis of the available data, but not to record the individual pieces of data that contributed to the gene structure (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Misra <em>et al.</em> 2002</a>). Our philosophy has been that the evidence speaks for itself and the supporting data are easily viewed in GBrowse on the FlyBase web site (<a href="http://flybase.org/cgi-bin/gbrowse2/dmel/">http://flybase.org/cgi-bin/gbrowse2/dmel/</a>). As described above, supporting data are vetted; problematic or low-confidence data (including some atypical cDNAs) are not incorporated into the gene model. We have chosen to present a high confidence annotation set and leave the lower confidence variations until such time as there is sufficient evidence to include these alternatives. However, we do not ignore the exceptional (and often inconvenient) cases that are well supported, including polycistronic transcripts, noncanonical splices, and <em>trans</em>-spliced transcripts (see <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Crosby <em>et al.</em> 2015</a>, which is the companion to this article). We have chosen not to represent minor variations in gene structure, particularly with respect to 5′ UTR variations. For extremely complex genes capable of producing dozens, hundreds, or, in some cases, thousands of alternative transcripts, we do not represent all possible exon combinations (see <em>All possible permutations are not annotated</em>). Our goal has been to produce a biologically accurate, integrated, and maximally useful and usable gene model annotation set.</p></div><div id="sec-3" class="subsection"><h3>Annotation guidelines</h3><p id="p-10">To ensure that the gene model annotation is performed as consistently as possible, rules to help curators evaluate the existing evidence were developed during the initial genome annotation effort (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Misra <em>et al.</em> 2002</a>) and have since been updated periodically (FlyBase Gene Model Annotation Guidelines, <a href="http://flybase.org/wiki/FlyBase:Gene_Model_Annotation_Guidelines">http://flybase.org/wiki/FlyBase:Gene_Model_Annotation_Guidelines</a>). The recent influx of high-throughput evidence included novel data types that had not previously been incorporated. Learning to use these new data types for informing gene models was a gradual process, including learning to recognize possible artifacts and reaching agreement among the annotators on how to use the data consistently. In March 2012, a comprehensive revision of our annotation guidelines was completed to accommodate the wealth of high-throughput transcriptomics data and to facilitate consistent application of these updated guidelines to gene model updates and changes.</p><p id="p-11">Explanatory comments have been more consistently applied across the entire annotation set and have been adapted to reflect the incorporation of the new data types (<a href="http://www.g3journal.org/content/suppl/2015/06/24/g3.115.018929.DC1/018929SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/24/g3.115.018929.DC1/TableS1.pdf">Table S1</a>). These appear in FlyBase with the gene model on the gene report page and are used to flag unusual aspects of a gene model (unconventional translation start postulated, for example) or as a means of noting data that have not been incorporated into the model. Examples of the latter include “low-frequency RNA-Seq exon junction(s) not annotated” and “annotated transcripts do not represent all possible combinations of alternative exons and/or alternative promoters.” We also use comments at the transcript level, for example, noting if the termini of a transcript are based on RNA-Seq data or, in the case of the 3′ terminus, a polyadenylated cDNA. All gene models have been assessed since R5.45 (FB_2012_03) and have been annotated to these new standards. Our guidelines are still evolving in some areas such as the criteria for creating non-coding RNAs, assessing the protein-coding potential of new transcripts, and using non-ATG translation starts, because general knowledge in these areas is still rudimentary.</p><p id="p-12">As we discuss the various types of high-throughput data and how we used them for annotation, we indicate the associated annotation guidelines that were developed. The principle behind our annotation guidelines is to allow an annotator to create a gene model that best represents the most commonly expressed transcripts of a given gene. In some cases, gene models can be quite complex and may have very many transcripts. In those cases, we often do not annotate every transcript and, instead, we add a gene model comment to indicate that the model is more complex than represented. Data and gene models are available at FlyBase (<a href="http://flybase.org">http://flybase.org</a>).</p></div></div><div class="section results-discussion" id="sec-4"><h2 class="">Results and Discussion</h2><div id="sec-5" class="subsection"><h3>Major changes to the annotation set</h3><div id="sec-6" class="subsection"><h4>Overview:</h4><p id="p-13"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the change in total numbers of various classes of annotations between FB2010_01, R5.24 (the last release before high-throughput data input) and the final release of 2014, FB2014_06, R6.03. Within that time span a new reference genome assembly for <em>D. melanogaster</em> became available (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hoskins <em>et al.</em> 2015</a>), necessitating the migration of FlyBase gene models from R5.57 to R6.01 (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">dos Santos <em>et al.</em> 2015</a>). Selected statistics relative to each genome annotation release are provided in FlyBase as “Release Notes”; those for prior releases are available in the “Previous Release Notes” section found in “Other Archives” under the Archive tab at FlyBase.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466638/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466638/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Annotation statistics for R5.24 and R6.03 FlyBase gene model annotation sets</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">From R5.24 to R6.03, the total number of protein-coding genes increased from 13,808 to 13,918, an increase of only 110 genes (∼0.8%). This modest net change masks a higher degree of flux in protein-coding gene annotations that becomes apparent when individual annotations are tracked between the two releases (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS1.zip">File S1</a>). In this interval, 92 protein-coding genes were deleted, 447 were created, and 604 were split, merged, or reclassified to yield 359 new coding genes. Protein-coding gene annotations that persisted from R5.24 to R6.03 also experienced a large degree of flux. These shared gene models are identified by stable annotation IDs (<a href="http://www.g3journal.org/content/suppl/2015/06/24/g3.115.018929.DC1/TableS2.pdf">Table S2</a>), but the number of associated transcripts can change substantially. For the 13,112 genes that persisted between R5.24 and R6.03, 1903 transcript isoforms were deleted and 9566 new transcript isoforms were added (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS2.zip">File S2</a>); new transcripts added to existing protein-coding gene models account for most of the 39% increase in the number of protein-coding transcripts. These changes are largely due to the flood of exon junction data, which led to a significant increase in the number of alternatively spliced transcripts per gene, despite our conservative approach to creating new transcript isoforms. This effect on the predicted proteome is also reflected in the number of unique predicted polypeptides, which increased by 20% from R5.24 to R6.03 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The availability of high-throughput data, particularly the RNA-Seq coverage data and the TSS data, also made possible the annotation of UTRs in cases where sparse or no cDNA/EST support had previously precluded any such annotation. As such, the number of mRNA annotations lacking 5′ and/or 3′ UTRs is dramatically reduced (five-fold) in R6.03 compared to R5.24 (<a href="http://www.g3journal.org/content/suppl/2015/06/24/g3.115.018929.DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS2.zip">File S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Changes to the FlyBase gene model annotation set in the era of high-throughput data. The gene model annotation set of FlyBase version FB2010_01 (R5.24), the last version to predate FlyBase incorporation of high-throughput data, was compared to that of FlyBase version FB2014_06 (R6.03) to determine the degree of change over the course of 26 annotation updates. Gene model annotations common to both sets (white) were identified. Gene model annotations specific to R6.03 were then examined to identify those derived from R5.24-specific gene model annotations through gene merge/split or reclassification (yellow). The remaining R5.24-specific annotations were classified as &#x201C;withdrawn&#x201D; (red), and the R6.03-specific annotations were classified as &#x201C;new&#x201D; (blue). The number of gene models within each category of status change is shown for protein-coding genes (A). For 13,003 of the 13,112 protein-coding genes common to both R5.24 and R6.03 (excluding nine complex cases), we also examined the degree to which the number of associated transcripts changed: a measure of gene model complexity. For these genes, the numbers of associated transcripts that persisted (white), were deleted (red), or were added (blue) at some point between R5.24 and R6.03 are shown (B). Changes to the set of annotated pseudogenes (C) and non-coding RNA genes (D) are also shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2024857102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Changes to the FlyBase gene model annotation set in the era of high-throughput data. The gene model annotation set of FlyBase version FB2010_01 (R5.24), the last version to predate FlyBase incorporation of high-throughput data, was compared to that of FlyBase version FB2014_06 (R6.03) to determine the degree of change over the course of 26 annotation updates. Gene model annotations common to both sets (white) were identified. Gene model annotations specific to R6.03 were then examined to identify those derived from R5.24-specific gene model annotations through gene merge/split or reclassification (yellow). The remaining R5.24-specific annotations were classified as &#x201C;withdrawn&#x201D; (red), and the R6.03-specific annotations were classified as &#x201C;new&#x201D; (blue). The number of gene models within each category of status change is shown for protein-coding genes (A). For 13,003 of the 13,112 protein-coding genes common to both R5.24 and R6.03 (excluding nine complex cases), we also examined the degree to which the number of associated transcripts changed: a measure of gene model complexity. For these genes, the numbers of associated transcripts that persisted (white), were deleted (red), or were added (blue) at some point between R5.24 and R6.03 are shown (B). Changes to the set of annotated pseudogenes (C) and non-coding RNA genes (D) are also shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1721/F1.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/8/1721/F1.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466537" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Changes to the FlyBase gene model annotation set in the era of high-throughput data. The gene model annotation set of FlyBase version FB2010_01 (R5.24), the last version to predate FlyBase incorporation of high-throughput data, was compared to that of FlyBase version FB2014_06 (R6.03) to determine the degree of change over the course of 26 annotation updates. Gene model annotations common to both sets (white) were identified. Gene model annotations specific to R6.03 were then examined to identify those derived from R5.24-specific gene model annotations through gene merge/split or reclassification (yellow). The remaining R5.24-specific annotations were classified as “withdrawn” (red), and the R6.03-specific annotations were classified as “new” (blue). The number of gene models within each category of status change is shown for protein-coding genes (A). For 13,003 of the 13,112 protein-coding genes common to both R5.24 and R6.03 (excluding nine complex cases), we also examined the degree to which the number of associated transcripts changed: a measure of gene model complexity. For these genes, the numbers of associated transcripts that persisted (white), were deleted (red), or were added (blue) at some point between R5.24 and R6.03 are shown (B). Changes to the set of annotated pseudogenes (C) and non-coding RNA genes (D) are also shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">While long non-coding RNA (lncRNA) gene annotations were almost nonexistent in R5.24, there are more than 2400 in R6.03 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS1.zip">File S1</a>). The number of pseudogenes has tripled (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS1.zip">File S1</a>). It is clear from these broad comparisons that between R5.24 and R6.03, the annotation set underwent major revision; details of the various categories of updates are described in the following sections.</p><p id="p-17">FlyBase relies on outside expert annotations for the various small non-protein-coding classes (miRNA, rRNA, snRNA, snoRNA, and tRNA) as well as natural transposons. Of these, RNA-Seq data led to a great increase in the number of miRNA genes and a more modest increase in the number of snoRNA genes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berezikov <em>et al.</em> 2011</a>, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Graveley <em>et al.</em> 2011</a>) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS1.zip">File S1</a>). The miRNA annotations are imported from miRBase (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kozomara and Griffiths-Jones 2011</a>).</p></div><div id="sec-7" class="subsection"><h4>Merges and splits:</h4><p id="p-18">Gene model merges were the most common category of dramatic change affecting existing gene models based on the new high-throughput data. There have been 211 gene model merges since R5.24, combining 454 different R5.24 annotations into 211 new R6.03 gene models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS1.zip">File S1</a>). Most merges were based on RNA-Seq data, usually RNA-Seq junction data. Such merges often pulled in an upstream 5′ exon to define a new 5′ terminus of the gene (for example, <em><a href="http://flybase.org/reports/FBgn0261929.html">5-HT2B</a></em>) or to create an alternative transcript or transcripts [for example, <em><a href="http://flybase.org/reports/FBgn0261015.html">Pif1A</a> (<a href="http://flybase.org/reports/FBtr0332500.html">Pif1A-RG</a></em> and <em><a href="http://flybase.org/reports/FBtr0332501.html">Pif1A-RH</a>)</em>, <em><a href="http://flybase.org/reports/FBgn0264272.html">mwh</a> (<a href="http://flybase.org/reports/FBtr0331393.html">mwh-RB</a></em> and <em><a href="http://flybase.org/reports/FBtr0336737.html">mwh-RC</a>)</em>, <em><a href="http://flybase.org/reports/FBgn0262975.html">cnc</a> (<a href="http://flybase.org/reports/FBtr0306752.html">cnc-RI</a></em>, <em><a href="http://flybase.org/reports/FBtr0308226.html">cnc-RM</a></em>, and <em><a href="http://flybase.org/reports/FBtr0308227.html">cnc-RN</a>)</em>, <em><a href="http://flybase.org/reports/FBgn0262614.html">pyd</a> (<a href="http://flybase.org/reports/FBtr0310205.html">pyd-RO</a>)</em>]; adjacent or embedded genes that turned out to be alternative exons within a complex gene model [for example, alternative transcripts of <em><a href="http://flybase.org/reports/FBgn0263289.html">scrib</a> (<a href="http://flybase.org/reports/FBtr0308630.html">scrib-RN</a></em> and <em><a href="http://flybase.org/reports/FBtr0335521.html">scrib-RT</a>)</em>, <em><a href="http://flybase.org/reports/FBgn0261854.html">aPKC</a> (<a href="http://flybase.org/reports/FBtr0303435.html">aPKC-RG</a></em> and <em><a href="http://flybase.org/reports/FBtr0329887.html">aPKC-RJ</a>)</em>, <em><a href="http://flybase.org/reports/FBgn0262872.html">milt</a> (<a href="http://flybase.org/reports/FBtr0306231.html">milt-RD</a></em> and <em><a href="http://flybase.org/reports/FBtr0306233.html">milt-RF</a>)</em>, and <em><a href="http://flybase.org/reports/FBgn0263108.html">BtbVII</a> (<a href="http://flybase.org/reports/FBtr0307306.html">BtbVII-RF</a>)</em>] were also common. Because junctions spanning related tandem genes are one of the most common types of artifactual junction calls, junctions supporting gene merges can be obscured by artifactual noise. Although the BCM RNA-Seq project (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Daines <em>et al.</em> 2011</a>) included junctions that could support a merge, the modENCODE project excluded most of them from their original published dataset (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Graveley <em>et al.</em> 2011</a>). At the request of FlyBase, modENCODE provided a list of all excluded low-confidence junctions; the 1213 junctions that either correspond to a BCM junction (n = 781) or subsequently came to be represented in a gene model annotation because they were supported by other data (n = 432) are included in the FlyBase dataset <a href="http://flybase.org/reports/FBlc0000460.html">modENCODE_mRNA-Seq_EXTRA_junctions</a> and are visible on GBrowse.</p><p id="p-19">Less common than merges were gene model splits: 63 splits that produced 131 new gene models have occurred since R5.24 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS1.zip">File S1</a>). An example of a gene model split based on RNA-Seq data is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. The original 3′ coding exon of <em><a href="http://flybase.org/reports/FBgn0263076.html">Klp54D</a></em> (based on prediction data) is now annotated as a separate gene, <em><a href="http://flybase.org/reports/FBgn0263029.html">CG43324</a></em>. The RNA-Seq developmental profile indicates that this gene is expressed at additional stages and at higher levels than <em><a href="http://flybase.org/reports/FBgn0263076.html">Klp54D</a></em>. The <em><a href="http://flybase.org/reports/FBgn0263029.html">CG43324</a></em> gene model annotation is also supported by RAMPAGE transcriptional start site data (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Batut <em>et al.</em> 2013</a>). Note that the newly annotated transcripts of <em><a href="http://flybase.org/reports/FBgn0263076.html">Klp54D</a></em> are also informed by high-throughput data, with alternative 3′ exons supported by RNA-Seq junction data.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Klp54D gene model was split into two genes. A GBrowse1 view of the Klp54D gene model as it existed in R5.30 (A). The gene (blue) and transcript (orange) annotations were based primarily on gene prediction (yellow). On the basis of high-throughput data, this gene model was split in R5.36 to give Klp54D and CG43324, as shown in an updated GBrowse2 view of this same region, as it exists in R6.03 (B). Below the transcript annotations, modENCODE RNA-Seq exon junctions (blue), aligned cDNA evidence (green), and modENCODE RNA-Seq coverage data for 30 developmental stages spanning early embryogenesis to adulthood are shown from top to bottom. The RNA-Seq expression data show that CG43324 is expressed at a much higher level and in more stages than Klp54D. There is also no RNA-Seq exon junction connecting the two genes. In addition, the annotated 5&#x2032; end of CG43324 is supported by RAMPAGE TSS data (not shown). More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2024857102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;Klp54D&lt;/em&gt; gene model was split into two genes. A GBrowse1 view of the &lt;em&gt;Klp54D&lt;/em&gt; gene model as it existed in R5.30 (A). The gene (blue) and transcript (orange) annotations were based primarily on gene prediction (yellow). On the basis of high-throughput data, this gene model was split in R5.36 to give &lt;em&gt;Klp54D&lt;/em&gt; and &lt;em&gt;CG43324&lt;/em&gt;, as shown in an updated GBrowse2 view of this same region, as it exists in R6.03 (B). Below the transcript annotations, modENCODE RNA-Seq exon junctions (blue), aligned cDNA evidence (green), and modENCODE RNA-Seq coverage data for 30 developmental stages spanning early embryogenesis to adulthood are shown from top to bottom. The RNA-Seq expression data show that &lt;em&gt;CG43324&lt;/em&gt; is expressed at a much higher level and in more stages than &lt;em&gt;Klp54D&lt;/em&gt;. There is also no RNA-Seq exon junction connecting the two genes. In addition, the annotated 5&#x2032; end of &lt;em&gt;CG43324&lt;/em&gt; is supported by RAMPAGE TSS data (not shown). More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1721/F2.medium.gif" width="440" height="418"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/8/1721/F2.medium.gif" width="440" height="418"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466572" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">The <em><a href="http://flybase.org/reports/FBgn0263076.html">Klp54D</a></em> gene model was split into two genes. A GBrowse1 view of the <em><a href="http://flybase.org/reports/FBgn0263076.html">Klp54D</a></em> gene model as it existed in R5.30 (A). The gene (blue) and transcript (orange) annotations were based primarily on gene prediction (yellow). On the basis of high-throughput data, this gene model was split in R5.36 to give <em><a href="http://flybase.org/reports/FBgn0263076.html">Klp54D</a></em> and <em><a href="http://flybase.org/reports/FBgn0263029.html">CG43324</a></em>, as shown in an updated GBrowse2 view of this same region, as it exists in R6.03 (B). Below the transcript annotations, modENCODE RNA-Seq exon junctions (blue), aligned cDNA evidence (green), and modENCODE RNA-Seq coverage data for 30 developmental stages spanning early embryogenesis to adulthood are shown from top to bottom. The RNA-Seq expression data show that <em><a href="http://flybase.org/reports/FBgn0263029.html">CG43324</a></em> is expressed at a much higher level and in more stages than <em><a href="http://flybase.org/reports/FBgn0263076.html">Klp54D</a></em>. There is also no RNA-Seq exon junction connecting the two genes. In addition, the annotated 5′ end of <em><a href="http://flybase.org/reports/FBgn0263029.html">CG43324</a></em> is supported by RAMPAGE TSS data (not shown). More information on data presented in GBrowse may be found at <a href="http://flybase.org/wiki/FlyBase:GBrowse_Tracks">http://flybase.org/wiki/FlyBase:GBrowse_Tracks</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h4>Alternative transcripts:</h4><p id="p-21">Adjusted 5′ start sites, extended 3′ UTRs supported by RNA-Seq coverage data, and additional transcript isoforms supported by junction data are the most common changes to existing gene models. Previously, alternative transcripts were based primarily on cDNA/EST data and published community data. RNA-Seq exon junction data from the modENCODE project and from the BCM group provided a large source of new potential intron/exon boundaries and thus of new transcript isoforms. Of the 71,082 unique exon junctions initially reported to FlyBase, 48,551 were already represented at least once by an intron in existing gene model annotations, leaving 22,531 to be analyzed. After review of these junctions in the context of all available data, another 8841 were incorporated into new transcripts (254 previously incorporated exon junctions were withdrawn). An additional 423 low-confidence modENCODE “EXTRA” junctions are present in R6.03 gene models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS3.zip">File S3</a>).</p><p id="p-22">Of these 9264 newly incorporated junctions, 4172 were exclusively within coding sequences and thus led to the creation of new unique polypeptides, whereas 4861 were exclusively in non-coding RNA or UTR sequences and therefore did not affect coding sequences (<a href="http://www.g3journal.org/content/suppl/2015/06/24/g3.115.018929.DC1/TableS4.pdf">Table S4</a>). The majority of exon junctions overlapping UTR or non-coding sequences (69%) were exclusively within 5′ UTRs and many of these were alternative splice donors for existing 5′ exons. In addition, a substantial number of novel 5′ leading exons and, thus, new promoters were annotated on the basis of the RNA-Seq junction data.</p><p id="p-23">For a variety of reasons, 13,953 junctions (12,824 of which overlap 4581 genes) were not included in gene models. Some were clearly artifactual: they spanned tandem repeated genes or were in repeat regions within genes. Many had noncanonical splices that could not be verified (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Crosby <em>et al.</em> 2015</a>). Some junctions were rejected because they were present at low frequency in very highly expressed genes. Others were not used in annotations because they were low frequency and did not change the gene model significantly or because they created frameshifts and lacked corroborating evidence (see below). The junctions that were not included in a gene model are indicated as “Reviewed, but not incorporated into a FlyBase gene model” in the “Comments” field on the associated junction sequence feature report.</p></div><div id="sec-9" class="subsection"><h4>Transcription initiation sites:</h4><p id="p-24">The annotation of the termini of transcripts is an example of nuanced biological reality that is impractical to capture in detail. Transcription initiation and termination are complex; multiple closely spaced starts and stops often exist for transcripts that are otherwise identical. Based on our annotation guidelines, clustered initiation sites are annotated to a single discrete location and closely spaced polyadenylation sites are annotated as a single 3′ terminus. Thus, a single transcription start site (TSS) and a single polyadenylation site are annotated for the majority of gene models.</p><p id="p-25">Until the modENCODE and other genome-wide TSS data became available, the 5′ ends of the transcripts were set to the 5′ extent of the 5′-most EST or cDNA, except in rare cases when mapped transcription starts from the literature were used (these are cited in the transcript comments). Recent high-throughput data have provided alternative sources for TSSs (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hoskins <em>et al.</em> 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nechaev <em>et al.</em> 2010</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Batut <em>et al.</em> 2013</a>), allowing us to refine the annotated 5′ ends of known transcript isoforms and to identify new transcript isoforms; the number of distinct 5′ ends for transcripts has increased more than 25% from 17,576 in R5.24 to 22,082 in R6.03.</p><p id="p-26">FlyBase has systematically reviewed and incorporated into its gene models the modENCODE embryonic TSS data, which mapped TSS regions for 12,454 promoters of 8037 genes (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Hoskins <em>et al.</em> 2011</a>). These TSS regions were reported as discrete regions containing distributions of transcription starts based on the results of three different mapping approaches. The authors identified 8678 regions supported by at least two of the three types of evidence, which were denoted as “validated,” and FlyBase efforts focused on these sites. The distribution of transcription starts within these TSS regions is complex and, in some cases, the 5′-most mapped start site is actually a nonrepresentative outlier. To define a single, discrete 5′ end for annotated transcripts overlapping a TSS region, FlyBase calculated the “90% TSS point” in each TSS region. This is defined as the point at which 90% of the transcription start signal within the TSS region is encompassed (summation algorithm hits 0.9, starting from the 3′-most TSS and moving 5′). In FlyBase the “Transcription Start Sites (embryonic)” tier in GBrowse displays the full modENCODE TSS region. Between annotation versions R5.24 and R6.03, 222 new TSSs overlapping modENCODE starts were incorporated and 7349 existing start sites were updated to exactly match the 90% point (for example, <em><a href="http://flybase.org/reports/FBgn0051717.html">CG31717</a></em>) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In R6.03, the 5′ end of 14,366 transcripts (43% of all transcripts) match the 90% point of a modENCODE TSS region; in R5.24, only 8586 transcripts (39% of all transcripts) had a 5′ end matching the 90% TSS point or falling within a TSS region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS4.zip">File S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/06/24/g3.115.018929.DC1/TableS5.pdf">Table S5</a>). TSS regions defined by <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Hoskins <em>et al</em>. 2011</a> as “supported” or “5′ RACE only” were not usually supported by other data, and thus they were infrequently used for annotation of transcription start sites.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Alternative transcription start site and 3&#x2032; end for CG31717. A GBrowse2 view of CG31717, as it exists in R6.03, depicting (from top to bottom) modENCODE embryonic transcription start site evidence, FlyBase gene and transcript annotations, aligned cDNA evidence, modENCODE RNA-Seq junctions, and modENCODE stranded RNA-Seq expression profiles for CNS tissues (larval, pupal, and adult head samples) and gonadal tissues (testis, accessory gland, virgin female ovary, and mated female ovary); plus strand signal is shown above the minus strand signal for each RNA-Seq track. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2024857102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Alternative transcription start site and 3&#x2032; end for &lt;em&gt;CG31717&lt;/em&gt;. A GBrowse2 view of &lt;em&gt;CG31717&lt;/em&gt;, as it exists in R6.03, depicting (from top to bottom) modENCODE embryonic transcription start site evidence, FlyBase gene and transcript annotations, aligned cDNA evidence, modENCODE RNA-Seq junctions, and modENCODE stranded RNA-Seq expression profiles for CNS tissues (larval, pupal, and adult head samples) and gonadal tissues (testis, accessory gland, virgin female ovary, and mated female ovary); plus strand signal is shown above the minus strand signal for each RNA-Seq track. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1721/F3.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/8/1721/F3.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466569" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Alternative transcription start site and 3′ end for <em><a href="http://flybase.org/reports/FBgn0051717.html">CG31717</a></em>. A GBrowse2 view of <em><a href="http://flybase.org/reports/FBgn0051717.html">CG31717</a></em>, as it exists in R6.03, depicting (from top to bottom) modENCODE embryonic transcription start site evidence, FlyBase gene and transcript annotations, aligned cDNA evidence, modENCODE RNA-Seq junctions, and modENCODE stranded RNA-Seq expression profiles for CNS tissues (larval, pupal, and adult head samples) and gonadal tissues (testis, accessory gland, virgin female ovary, and mated female ovary); plus strand signal is shown above the minus strand signal for each RNA-Seq track. More information on data presented in GBrowse may be found at <a href="http://flybase.org/wiki/FlyBase:GBrowse_Tracks">http://flybase.org/wiki/FlyBase:GBrowse_Tracks</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">For 2281 transcripts representing 1682 distinct TSS regions, short-capped RNA data (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nechaev <em>et al.</em> 2010</a>) or RAMPAGE promoter profiling data (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Batut <em>et al.</em> 2013</a>) were used to define the TSS where modENCODE data were absent. The RAMPAGE data were particularly useful because they defined transcription starts throughout the entire <em>Drosophila</em> life cycle. For these transcripts, comments are appended that indicate the source of the TSS.</p><p id="p-29">In the absence of any experimental TSS-mapping data, the 5′ extent of the 5′-most EST or cDNA was used, or robust RNA-Seq coverage data were used to estimate the TSS. Five hundred twenty-eight gene models (572 transcripts) still lack any evidence to indicate the extent of the 5′ UTR, and thus the transcript model begins at the translation start site.</p></div><div id="sec-10" class="subsection"><h4>3′ UTRs:</h4><p id="p-30">One of the most dramatic changes to gene models supported by the RNA-Seq coverage data is the length of 3′ UTRs: a number of genes have alternative transcripts with very long 3′ UTRs, up to 18 kb (<a href="http://www.g3journal.org/content/suppl/2015/06/24/g3.115.018929.DC1/TableS6.pdf">Table S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS5.zip">File S5</a>). In addition, more than 2500 genes have more modest 3′ UTR extensions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>); these additional transcript isoforms comprise one of the major classes of new alternative transcripts resulting from the assessment of high-throughput data.</p><p id="p-31">Before the RNA-Seq coverage data were introduced, polyadenylated cDNAs were used to set the 3′ extent of transcripts for FlyBase gene models. It is still our policy to represent 3′ termini supported by polyadenylated cDNAs; a comment is appended to the corresponding transcript specifying that this is the case. Where multiple distinct 3′ ends are supported by cDNAs (unless they are within 10 bases of each other), additional isoforms are annotated. More than 12,300 genes, including lncRNA genes (see below), have at least one transcript based on a polyadenylated cDNA. In the absence of a cDNA, the 3′ end is approximated from the RNA-Seq coverage data; in the absence of both cDNA and RNA-Seq data, the 3′ terminus is set to the 3′ end of the translation termination codon.</p><p id="p-32">RNA-Seq coverage data often support 3′ UTR sequences that extend well beyond the longest cDNA. In these cases, at least one transcript is annotated to the approximate terminus of the longest UTR supported by that data and an explanatory comment is appended to the transcript. It is generally not possible to determine which specific isoforms are most likely to be transcribed with the extended 3′ UTR. Thus, a single transcript or a subset of transcripts is annotated with the extended 3′ UTR, although extended transcripts of all isoforms may exist <em>in vivo</em>. Multiple polyadenylation sites within the extended 3′ UTR may be present (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Smibert <em>et al.</em> 2012</a>), but it is not practical to include all such variants, so usually only the longest is annotated. In some cases it is difficult to determine if an extended non-coding sequence corresponds to a long 3′ UTR or an independent lncRNA gene. In fact, both may be true (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mercer <em>et al.</em> 2011</a>). A small number of such dual-entity regions have been annotated; the lncRNA is flagged with the comment “This lncRNA overlaps and may be derived from the long 3′ UTR of the upstream gene”; examples include <em><a href="http://flybase.org/reports/FBgn0266166.html">CR44872</a></em> and <em><a href="http://flybase.org/reports/FBgn0266400.html">CR45040</a></em>.</p><p id="p-33">Several groups have performed large-scale analyses that address the phenomenon of alternative transcripts with significantly longer 3′ UTRs. Both <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hilgers <em>et al.</em> (2011)</a>, looking at changes during embryogenesis, and <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Smibert <em>et al.</em> (2012)</a>, comparing different larval and adult tissues, observed that transcripts with long extended 3′ UTRs are most frequently present in neuronal tissues. Studies of specific genes subject to regulation by miRNAs, such as <em><a href="http://flybase.org/reports/FBgn0003944.html">Ubx</a></em> (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Thomsen <em>et al.</em> 2010</a>) and <em><a href="http://flybase.org/reports/FBgn0086758.html">chinmo</a></em> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2012</a>), support the existence of additional miRNA targets within their extended 3′ UTR sequences.</p></div><div id="sec-11" class="subsection"><h4>New genes (mostly non-coding RNAs):</h4><p id="p-34">Previous rounds of annotation focused primarily on protein-coding genes. A limited number of lncRNAs had been annotated based on descriptions from the literature or cDNA evidence. The new high-throughput data, primarily stranded RNA-Seq coverage and exon junction data and to a lesser extent TSS data, provided evidence for many new genes; most of these have been annotated as lncRNAs. Because the majority of regions with any significant coding potential had already been identified, it is not surprising that most new genes appear to be non-coding. Yet, some of these may encode small polypeptides that have yet to be identified or characterized. Because of the uncertainty in this area, new genes that are annotated as non-coding are generally commented as “probable lncRNA gene; may encode small polypeptide(s),” whereas some that are designated as coding are given the comment “possible non-coding RNA gene.” In R5.24, only 133 non-coding RNA genes had been annotated. From R5.24 to R6.03, 2313 new candidate non-coding genes were annotated (including those flagged as antisense, see below).</p><p id="p-35">We assessed the proposed lncRNAs described in the published literature (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tupy <em>et al.</em> 2005</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Inagaki, <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hiller <em>et al.</em> 2009</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Young, <em>et al.</em> 2012</a>) and annotated many, but not all, of the lncRNAs proposed. Unless we had independent evidence that the region is transcribed (for example, RNA-Seq coverage data), we did not annotate the predicted lncRNA gene. In some cases, we found that the putative lncRNA corresponded to an extended 3′ UTR of an existing gene; in others, we found that there was evidence of a small functional ORF. In these cases, the symbol or designation used in the publication was captured as a synonym of the corresponding coding gene.</p><p id="p-36">Two systematic genome-wide searches were performed to identify additional regions containing potential new genes or new transcribed regions. First, we analyzed a list of intergenic RNA-Seq junctions, some of which were incorporated into new gene models. More often, intergenic junctions were indicative of new 5′ extents of existing genes. In some cases, it was impossible to incorporate such a junction into a gene model due to the lack of any additional or consistent evidence. Additional evidence might include ESTs, consistent RNA-Seq coverage data, or transcription start site data. Intergenic junctions that were not included in a gene model were marked with a comment.</p><p id="p-37">A second method to identify new genes was based on analysis of intergenic RNA-Seq coverage values calculated as reads per kilobase per million reads (RPKM) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mortazavi <em>et al.</em> 2008</a>). The genome was scanned step-wise in 200-bp windows for regions with an RPKM value of 3 or more. If the RNA-Seq coverage data (with or without supporting TSS data) made it possible to determine discrete transcript ends and was present in more than one RNA-Seq data track, then a new gene was annotated (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Although many new genes were created in this way, we found that establishing absolute criteria for making a new gene was difficult, and that discerning noise from the real RNA-Seq signal was not always possible.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="New long non-coding RNA genes are supported by RNA-Seq data. A GBrowse2 view for a region containing four recently annotated lncRNA genes is shown (R6.03). CR43132 is supported by RNA-Seq junction and expression data. CR45523, CR45524, and CR45526 are supported by RNA-Seq expression data only; they were identified in a genome-wide scan for intergenic regions with RPKM values of 3 or more. The transcript polarity is determined from the stranded &#x201C;Gonads and male accessory glands&#x201D; RNA-Seq expression tracks. CR45523, CR45524, and CG45526 show expression primarily in male testis (red RNA-Seq signal), a pattern common to many of the newly annotated ncRNA genes. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2024857102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;New long non-coding RNA genes are supported by RNA-Seq data. A GBrowse2 view for a region containing four recently annotated lncRNA genes is shown (R6.03). &lt;em&gt;CR43132&lt;/em&gt; is supported by RNA-Seq junction and expression data. &lt;em&gt;CR45523&lt;/em&gt;, &lt;em&gt;CR45524&lt;/em&gt;, and &lt;em&gt;CR45526&lt;/em&gt; are supported by RNA-Seq expression data only; they were identified in a genome-wide scan for intergenic regions with RPKM values of 3 or more. The transcript polarity is determined from the stranded &#x201C;Gonads and male accessory glands&#x201D; RNA-Seq expression tracks. &lt;em&gt;CR45523&lt;/em&gt;, &lt;em&gt;CR45524&lt;/em&gt;, and &lt;em&gt;CG45526&lt;/em&gt; show expression primarily in male testis (red RNA-Seq signal), a pattern common to many of the newly annotated ncRNA genes. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1721/F4.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/8/1721/F4.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466614" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">New long non-coding RNA genes are supported by RNA-Seq data. A GBrowse2 view for a region containing four recently annotated lncRNA genes is shown (R6.03). <em><a href="http://flybase.org/reports/FBgn0262606.html">CR43132</a></em> is supported by RNA-Seq junction and expression data. <em><a href="http://flybase.org/reports/FBgn0267079.html">CR45523</a></em>, <em><a href="http://flybase.org/reports/FBgn0267080.html">CR45524</a></em>, and <em><a href="http://flybase.org/reports/FBgn0267082.html">CR45526</a></em> are supported by RNA-Seq expression data only; they were identified in a genome-wide scan for intergenic regions with RPKM values of 3 or more. The transcript polarity is determined from the stranded “Gonads and male accessory glands” RNA-Seq expression tracks. <em><a href="http://flybase.org/reports/FBgn0267079.html">CR45523</a></em>, <em><a href="http://flybase.org/reports/FBgn0267080.html">CR45524</a></em>, and <em><a href="http://flybase.org/reports/FBgn0267082.html">CG45526</a></em> show expression primarily in male testis (red RNA-Seq signal), a pattern common to many of the newly annotated ncRNA genes. See <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> for GBrowse track descriptions. More information on data presented in GBrowse may be found at <a href="http://flybase.org/wiki/FlyBase:GBrowse_Tracks">http://flybase.org/wiki/FlyBase:GBrowse_Tracks</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Short predicted ORFs (&lt;50 aa) are extremely frequent in the genome, just by chance, but it is challenging to identify which of these are translated <em>in vivo</em>. In the past, we made use of the PhyloCSF metric that aims to identify conserved protein-coding signatures (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2007</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lin <em>et al.</em> 2011</a>). The sensitivity of this algorithm falls off for very small proteins (roughly less than 35 amino acids); for example, this analysis failed to detect the ORFs of <em><a href="http://flybase.org/reports/FBgn0266491.html">SclA</a></em> and <em><a href="http://flybase.org/reports/FBgn0266492.html">SclB</a></em>, which are highly conserved polypeptides 28 and 29 amino acids in length (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Magny <em>et al.</em> 2013</a>). The PhyloCSF analysis was also quite conservative, requiring conservation across 11 <em>Drosophila</em> species, including several in the <em>Drosophil</em>a subgenus. When creating new gene annotations supported by RNA-Seq data, we assessed the longest ORF, requiring conservation only within the melanogaster subgroup (which is in the Sophophora subgenus), including conservation of the initiating methionine and absence of frameshifts or internal stops. These case-by-case informal assessments were neither rigorous nor exhaustive; the smallest polypeptide annotated using this approach was 32 amino acids. Even for transcripts that have been accurately identified as containing a short translated ORF, the picture may be incomplete; many small proteins characterized thus far are encoded on polycistronic transcripts. Without any additional high-throughput predictive tools, we have relied on the published literature to point us to specific cases of very small proteins (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wolfner <em>et al.</em> 1997</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Savard <em>et al.</em> 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kondo <em>et al.</em> 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Galindo <em>et al.</em> 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hayden and Bosco 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Findlay <em>et al.</em> 2009</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">modENCODE Consortium 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ladoukakis <em>et al.</em> 2011</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Magny <em>et al.</em> 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2014</a>).</p><p id="p-40"><a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Ladoukakis <em>et al.</em> (2011)</a> identified 401 small ORFs (&lt;100 aa) that, by their criteria, appeared likely to encode functional polypeptides. However, they limited their assessment of conservation of ORFs to a comparison of <em>D. melanogaster</em> and <em>D. pseudoobscura</em>. They also failed to distinguish between conservation at the nucleotide <em>vs.</em> the protein level. We extended the analysis to additional sequenced species within the Sophophora subgenus (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19"><em>Drosophila</em> 12 Genomes Consortium 2007</a>), but we found that this type of analysis is not informative for most ORFs less than 25–30 aa. Of the 50 ORFs we found to show conservation consistent with a protein-coding extent, many correspond to alternative or 5′ exons of larger genes or to cases of stop-codon readthrough (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jungreis <em>et al.</em> 2011</a>). Nine appear to encode small polypeptides, including four that had been previously identified. Three of the uniquely identified small protein candidates correspond to small conserved ORFs within the 5′ UTRs of larger genes. This finding underscores one of the difficulties in identifying ORFs for very small proteins: they may share transcripts that have already been annotated with protein-coding extents. One of the newly identified genes, <em><a href="http://flybase.org/reports/FBgn0263983.html">CG43732</a></em> [referred to by <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Ladoukakis <em>et al.</em> (2011)</a> as Dm_3R:78579), appears to be conserved among the Dipterans, including mosquitos, and perhaps among other insects as well (Tribolium, GenBank accession EEZ97819).</p><p id="p-41">Of the new gene models created since R5.24, 63 are annotated as coding genes for which the largest predicted polypeptide encoded is 50 amino acids or fewer. Of these, 51 encode polypeptides between 30 and 50 amino acids and 12 encode polypeptides less than 30 amino acids in length. The total number of genes encoding polypeptides of 50 amino acids or less in R6.03 is 108; 18 of these encode polypeptides of less than 30 amino acids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS6.zip">File S6</a>).</p></div><div id="sec-12" class="subsection"><h4>Antisense transcripts:</h4><p id="p-42">In the past, an occasional cDNA provided evidence for a lncRNA gene with extensive regions of antisense transcription. When RNA-Seq junction data and stranded RNA-Seq coverage data became available, it became clear that such antisense genes are not rare. In R6.03, there are 476 non-coding RNA annotations annotated with the SO term "antisense_gene" (SO:0000077) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This represents almost 20% of annotated lncRNAs. We have not yet completed a systematic search for antisense transcription based on RPKM values and expect that more antisense genes will be identified.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="New ncRNA gene CR45161 is antisense to fln. CR45161 is a newly annotated antisense gene supported by RNA-Seq expression and junction data. Although it might be mistaken for background transcription in the unstranded &#x201C;Developmental stage&#x201D; RNA-Seq expression tracks, its strong transcription on the positive strand is obvious in the stranded &#x201C;CNS and adult head&#x201D; RNA-Seq track. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2024857102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;New ncRNA gene &lt;em&gt;CR45161&lt;/em&gt; is antisense to &lt;em&gt;fln&lt;/em&gt;. &lt;em&gt;CR45161&lt;/em&gt; is a newly annotated antisense gene supported by RNA-Seq expression and junction data. Although it might be mistaken for background transcription in the unstranded &#x201C;Developmental stage&#x201D; RNA-Seq expression tracks, its strong transcription on the positive strand is obvious in the stranded &#x201C;CNS and adult head&#x201D; RNA-Seq track. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1721/F5.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/8/1721/F5.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466596" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">New ncRNA gene <em><a href="http://flybase.org/reports/FBgn0266654.html">CR45161</a></em> is antisense to <em><a href="http://flybase.org/reports/FBgn0005633.html">fln</a></em>. <em><a href="http://flybase.org/reports/FBgn0266654.html">CR45161</a></em> is a newly annotated antisense gene supported by RNA-Seq expression and junction data. Although it might be mistaken for background transcription in the unstranded “Developmental stage” RNA-Seq expression tracks, its strong transcription on the positive strand is obvious in the stranded “CNS and adult head” RNA-Seq track. See <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> for GBrowse track descriptions. More information on data presented in GBrowse may be found at <a href="http://flybase.org/wiki/FlyBase:GBrowse_Tracks">http://flybase.org/wiki/FlyBase:GBrowse_Tracks</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Only one case of overlapping coding regions on opposite strands has been annotated. Both genes (<em><a href="http://flybase.org/reports/FBgn0083984.html">CG34148</a></em> and <em><a href="http://flybase.org/reports/FBgn0053092.html">P5CDh2</a></em>) encode proteins of known functions that are well conserved outside of <em>D. melanogaster</em>. However, 323 loci (667 genes) that share overlapping CDSs on opposite strands were recently reported (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2014</a>). Their proposed new protein-coding extents on antisense transcripts were identified on the basis of conservation with other <em>Drosophila</em> species, but such conservation may simply be the result of a shadow effect due to shared nucleotide sequence with a highly conserved CDS on the opposite strand. For that reason the newly identified antisense transcripts are annotated as non-coding genes in FlyBase.</p></div><div id="sec-13" class="subsection"><h4>Sex-specific (or highly sex-biased) gene expression:</h4><p id="p-45">During the annotation process, as we examined the RNA-Seq coverage data available for developmental stages and tissues, it became clear that new male-specific genes were relatively common, but new female-specific genes were rare. Using the FlyBase RNA-Seq Search tool (based on binned RPKM values, see description at <a href="http://flybase.org/reports/FBrf0221009.html">http://flybase.org/reports/FBrf0221009.html</a>), we were able to quantitate this informal observation. The total number of male-specific genes, ∼2400 including ∼770 lncRNA genes, dramatically exceeds the total number of female-specific genes: ∼130 with very few lncRNA genes (precise numbers depend on the specified criteria; antisense genes are included in the lncRNA counts; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS7.zip">File S7</a>). Using broad criteria with no requirement for gonadal expression and inclusion of possible maternal-effect genes to tabulate the number of newly annotated (since R5.24) genes, highly male-biased expression is observed for 171 coding genes and 753 lncRNA genes; highly female-biased expression is observed for 11 coding genes and four lncRNA genes. Assessing all genes showing sex-biased expression, only four male-specific genes show no appreciable expression in testis (<em><a href="http://flybase.org/reports/FBgn0030880.html">CG6788</a></em>, <em><a href="http://flybase.org/reports/FBgn0263405.html">CR43451</a></em>, <em><a href="http://flybase.org/reports/FBgn0264570.html">CR43942</a></em>, <em><a href="http://flybase.org/reports/FBgn0266949.html">CR45400</a></em>). A number of other male-specific genes not expressed in testis appear to exist that have not been annotated due to lack of strand information, but the total is estimated to be less than 1–2% of all male-specific genes. In contrast, 19 female-specific genes (∼15%) show no appreciable expression in ovaries. These include genes expressed in the spermatheca and two genes required for pheromone biosynthesis (<em><a href="http://flybase.org/reports/FBgn0029172.html">Fad2</a></em> and <em><a href="http://flybase.org/reports/FBgn0037762.html">eloF</a></em>). Almost half of the female-specific genes also show high transcript levels in early embryos.</p><p id="p-46">By the criteria used, only five lncRNAs appear to be female-specific; three of these share some interesting characteristics. The most dramatic is that they span extremely long genomic ranges, more than 220 kb in the cases of <em><a href="http://flybase.org/reports/FBgn0052773.html">CR32773</a></em> and <em><a href="http://flybase.org/reports/FBgn0265457.html">CR44357</a></em> and 76 kb in the case of <em><a href="http://flybase.org/reports/FBgn0264384.html">CR43836</a></em>, despite having processed transcripts of modest size (less than 2.5 kb). All three are expressed at low to moderate levels in ovaries and embryos; all are located on the X chromosome and produce multiple transcripts due to variable use of alternative exons. Genes with very low levels of expression are excluded in the analysis of sex-biased expression described above; among these are three additional apparently female-specific lncRNA genes with very long genomic extents, all on the X chromosome (<em><a href="http://flybase.org/reports/FBgn0266350.html">CR44999</a></em>, <em><a href="http://flybase.org/reports/FBgn0266199.html">CR44894</a></em>, and <em><a href="http://flybase.org/reports/FBgn0264675.html">CR43960</a></em>). Non-coding genes that span long genomic extents are relatively uncommon. These six genes comprise the majority of annotated lncRNAs transcribed from genomic regions more than 72 kb in length. There are only four others: <em><a href="http://flybase.org/reports/FBgn0264857.html">iab-8</a></em>, <em><a href="http://flybase.org/reports/FBgn0266096.html">CR44833</a></em>, <em><a href="http://flybase.org/reports/FBgn0266348.html">CR44997</a></em>, and <em><a href="http://flybase.org/reports/FBgn0267704.html">flamenco</a></em>, a piRNA-producing locus.</p></div></div><div id="sec-14" class="subsection"><h3>Limitations of the data and the approach</h3><div id="sec-15" class="subsection"><h4>Not all transcript isoforms with alternative splices within UTRs are annotated:</h4><p id="p-47">After review, 13,944 RNA-Seq exon junctions were not incorporated into gene annotations. Many of these are junctions that support alternative splices within 5′ UTRs representing only minor variations of a few bases in a splice donor or acceptor. In light of our goal to produce a useful and usable annotation set, we are reluctant to create gene models with many alternative transcripts that differ by only small changes in the 5′ UTRs. Thus, for a given TSS, only a subset of such splices may be annotated. In other cases, predicted junctions in UTRs are not annotated because they are present at a low frequency relative to other junctions in the gene (based on read-counts) or have low confidence scores (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Graveley <em>et al.</em> 2011</a>). Whenever junction data were not incorporated in the gene model, comments were added to the gene record to indicate that the annotated transcripts do not represent all supported alternative splices within the 5′ UTR or that additional RNA-Seq low-frequency junctions exist that were evaluated but not annotated.</p></div><div id="sec-16" class="subsection"><h4>Protein isoforms supported only by low-frequency junction data are omitted:</h4><p id="p-48">Many junctions that were not annotated were within coding sequences. The general rule in these cases was to create a new transcript isoform if the new junction is present at a frequency of 1% or more of the highest junction frequency within the gene, unless the new junction leads to a frameshift, in which case the threshold was set to 10%. When merited, a comment indicating that low-frequency junctions were not annotated was appended.</p><p id="p-49">Our conservative criteria for annotating alternatively spliced isoforms based on RNA-Seq junction data have implications for the number of unique 5′ exons, the overall number of transcript and protein isoforms annotated, and the “permutation problem,” as described below.</p></div><div id="sec-17" class="subsection"><h4>All possible permutations are not annotated:</h4><p id="p-50">If noncontiguous data, such as RNA-Seq junction, EST, and TSS data, support alternative exons in several regions of a gene, it is usually not possible to determine which of the possible exon combinations actually exist <em>in vivo</em> as transcripts. We call this the “permutation problem.” All configurations of exons supported by full-length cDNAs are annotated (unless they involved retained introns). The number of additional transcripts created was at the discretion of the annotator. Excluding low-frequency junctions, all alternative splices within the CDS and all promoters have been represented at least once, but not necessarily all possible combinations of available exons were combined into transcripts. An example of a gene for which all possible annotations supported by junction data have not been annotated is <em><a href="http://flybase.org/reports/FBgn0000083.html">AnxB9</a></em> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Generally, our approach was to annotate using the high-frequency RNA-Seq junctions in the majority of transcripts and to represent the low-frequency junctions in at least one transcript isoform. For cases in which all combinations were not created, a comment to that effect was included in the Gene Model Comment section of the gene report.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A subset of possible AnxB9 transcript isoforms has been annotated. RNA-Seq junction and expression data predict eight alternative splice donors from three different leading 5&#x2032; exons, of which four have been used in annotations. Low-frequency junctions have not been annotated. Alternative splicing in the last intron leads to three different protein isoforms. A low-frequency junction at the 3&#x2032; end of the gene has also been excluded. Twelve different transcript isoforms are possible using the annotated junctions (32 are possible with all junctions), but only a subset of the possible combinations has been annotated. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2024857102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A subset of possible &lt;em&gt;AnxB9&lt;/em&gt; transcript isoforms has been annotated. RNA-Seq junction and expression data predict eight alternative splice donors from three different leading 5&#x2032; exons, of which four have been used in annotations. Low-frequency junctions have not been annotated. Alternative splicing in the last intron leads to three different protein isoforms. A low-frequency junction at the 3&#x2032; end of the gene has also been excluded. Twelve different transcript isoforms are possible using the annotated junctions (32 are possible with all junctions), but only a subset of the possible combinations has been annotated. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1721/F6.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/8/1721/F6.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466623" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-51" class="first-child">A subset of possible <em><a href="http://flybase.org/reports/FBgn0000083.html">AnxB9</a></em> transcript isoforms has been annotated. RNA-Seq junction and expression data predict eight alternative splice donors from three different leading 5′ exons, of which four have been used in annotations. Low-frequency junctions have not been annotated. Alternative splicing in the last intron leads to three different protein isoforms. A low-frequency junction at the 3′ end of the gene has also been excluded. Twelve different transcript isoforms are possible using the annotated junctions (32 are possible with all junctions), but only a subset of the possible combinations has been annotated. See <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> for GBrowse track descriptions. More information on data presented in GBrowse may be found at <a href="http://flybase.org/wiki/FlyBase:GBrowse_Tracks">http://flybase.org/wiki/FlyBase:GBrowse_Tracks</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">The decision not to attempt to create all 5′ exon alternatives and all exon permutations was based on both biological and practical considerations. As stated above, with available data, it is often not possible to determine which exon combinations are transcribed in the organism. In addition, it would be completely impractical to manually annotate large numbers of transcripts using currently available tools or to represent them in the GBrowse user interface. This policy led to a significant under-representation of transcripts in some instances, particularly for the ∼50 genes that have the capacity to encode more than 1000 transcript isoforms each, <em>e.g.</em>, <em><a href="http://flybase.org/reports/FBgn0033159.html">Dscam1</a></em>, <em><a href="http://flybase.org/reports/FBgn0041605.html">cpx</a></em>, <em><a href="http://flybase.org/reports/FBgn0264255.html">para</a></em>. For a more comprehensive compilation of predicted transcripts, see <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> (2014)</a>.</p></div></div><div id="sec-18" class="subsection"><h3>Challenging annotation types</h3><div id="sec-19" class="subsection"><h4>Mutations in the sequenced strain:</h4><p id="p-53">The sequenced <em>D. melanogaster</em> strain (<em><a href="http://flybase.org/reports/FBsn0000272.html">iso-1</a></em>, Bloomington stock <a href="http://flybase.org/reports/FBst0002057.html">#2057</a>) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brizuela <em>et al.</em> 1994</a>) has a set of previously known visible mutations: <em><a href="http://flybase.org/reports/FBal0018607.html">y<sup>1</sup></a></em>, <em><a href="http://flybase.org/reports/FBal0001731.html">cn<sup>1</sup></a></em>, <em><a href="http://flybase.org/reports/FBal0001342.html">bw<sup>1</sup></a></em>, <em>and</em> <em><a href="http://flybase.org/reports/FBal0015980.html">sp<sup>1</sup></a></em>. The mutational lesions corresponding to the first three markers were readily identified; the gene model annotation that corresponds to the <em><a href="http://flybase.org/reports/FBgn0003466.html">sp</a></em> (<em><a href="http://flybase.org/reports/FBgn0003466.html">speck</a></em>) locus still has yet to be determined. During the course of annotation, many more genes were found to have mutations relative to other strains of <em>D. melanogaster</em> and/or closely related <em>Drosophila</em> species. Currently, 55 genes models are annotated with the comment “Mutation in sequenced strain” or “Known mutation in sequenced strain” (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1/FileS8.zip">File S8</a>); 11 of these may be polymorphic pseudogenes (see below). Of the 55 identified genes, 10 have an inserted transposon, four have a point mutation in the initiation codon or a splice site, 15 are characterized as having nonsense mutations or a premature stop, 12 are frameshifts, 11 are deletions, and three are “complex,” having multiple lesions. Interestingly, 33 of the 55 genes display more than 50% sequence similarity to another gene in the genome, which may explain in part why so many mutations are tolerated in this strain. An additional four gene models are labeled “Possible mutation in sequenced strain”; these correspond to transposon insertions in introns or immediately adjacent to genes. The mandate of the annotation project is to present the wild-type proteome of <em>D. melanogaster</em>. To this end, the mutant polypeptide sequences in these cases have been replaced with a “wild-type” sequence in FlyBase. These are indicated with an exception and explanatory note in the GenBank protein records. The reported transcript sequences, however, correspond to the mutant genomic sequences of the sequenced strain (<em><a href="http://flybase.org/reports/FBsn0000272.html">iso-1</a></em>).</p><p id="p-54">Some of the "mutations in strain" might be more accurately described as variants (see discussion of polymorphic pseudogenes, below). A particularly interesting mutational variant present in the sequenced strain is the insertion of a Doc transposable element in the <a href="http://flybase.org/reports/FBgn0045761.html">C<em>HKov1</em></a> <em>g</em>ene (<em><a href="http://flybase.org/reports/FBal0190391.html">CHKov1<sup>Doc1420</sup></a></em>). In wild populations this variant has recently increased in frequency under directional selection (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aminetzach <em>et al.</em> 2005</a>). It has been determined that two useful phenotypes segregate with the insertion: increased resistance to organophosphate pesticides (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aminetzach <em>et al.</em> 2005</a>) and resistance to sigma viral infection (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Magwire <em>et al.</em> 2011</a>).</p></div><div id="sec-20" class="subsection"><h4>Pseudogene annotations:</h4><p id="p-55">Pseudogenes are typically defined as regions with similarity to a functional gene but containing lesions that preclude the generation of the functional product encoded by the "parental" gene, usually a protein or a structural RNA. For the identification of pseudogenes related to non-coding structural RNAs, FlyBase relies on external, published analyses. For pseudogenes of protein-coding genes, FlyBase genome annotators examine the predicted effects of lesions on the predicted polypeptide. Pseudogene classification requires evidence of neutral selection; in practice, we take as evidence of neutral selection two compromising lesions (<em>e.g.</em>, nonsense or frameshift mutations). Genes with a single disabling mutation in the sequenced strain are annotated as coding genes and marked as "mutant in strain" (see section above). In some cases, a gene may have two disabling mutations in certain strains but appear to be wild-type in others. A good example is <em><a href="http://flybase.org/reports/FBgn0000594.html">Est-P</a></em>, a gene that exhibits strain-specific disruptions to the coding sequence and has a conservation rate lower than the parental <em><a href="http://flybase.org/reports/FBgn0000592.html">Est-6</a></em> gene (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Balakirev and Ayala 2004</a>). Such genes are classified as "polymorphic pseudogenes"; they are also annotated as "mutant in strain," with a gene model comment explaining their nature. Most annotated polymorphic pseudogenes in FlyBase have come from analyses of gustatory, olfactory, and ionotropic receptors (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Robertson <em>et al.</em> 2003</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Robertson 2009</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Croset <em>et al.</em> 2010</a>).</p><p id="p-56">Using these criteria, FlyBase has annotated 301 pseudogenes: 30 are rRNA or tRNA-derived, and the remainder are derived from protein-coding genes. Pseudogenes are marked in FlyBase by the SO term "pseudogene_attribute" (SO:000042). Additionally, a gene model comment describes the parental gene and type of duplication: “Pseudogene similar to [parental gene]; proximate/transposed/retrotransposed.” Compared to R5.24, the 301 pseudogenes represent a tripling, mostly due to the identification of transcribed pseudogenes using transcriptomic data. With the recent release of a new genomic sequence assembly, we are finding additional pseudogenes that are not reflected in these numbers, especially on the newly assembled Y-chromosome. We note that automated annotation algorithms such as GRIT (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2014</a>) are not set to differentiate functional genes <em>vs.</em> pseudogenes, so manual annotation provides added value in this respect.</p><p id="p-57">It is notable that the number of pseudogenes annotated by FlyBase pales in comparison to the thousands annotated for the human genome by GENCODE (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sisu <em>et al.</em> 2014</a>). In addition, while there are thousands of processed (retrotransposed) pseudogenes annotated for the human genome, there are only six annotated in <em>D. melanogaster (</em>"processed_pseudogene," SO:0000043). In part, this reflects differences in biology. Retrogene bursts have been characterized for several mammalian lineages (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pan and Zhang 2009</a>), whereas pseudogenes are known to be relatively rare in <em>Drosophila</em>, by comparison to other eukaryotes, due to strong selection in intergenic regions and high deletion rates (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Harrison <em>et al.</em> 2003</a>). The low number of FlyBase annotated pseudogenes may also reflect a conservative annotation approach. Our goal has not been to annotate all possible gene fragments exhaustively, but rather to annotate as pseudogenes those regions that would otherwise be mistaken for functional protein-coding genes.</p></div><div id="sec-21" class="subsection"><h4>Annotation of alternative transcripts that encode significantly shorter polypeptides:</h4><p id="p-58">A surprising number of genes have evidence supporting alternative transcripts that either produce significantly shorter protein isoforms or are non-coding transcript isoforms. These typically have a retained intron, an alternative 3′ exon, an alternative splice that results in a premature stop, or the lack of a 5′ coding exon present in other transcripts, requiring a downstream start. In some cases, there is published evidence that these are functional protein-coding transcripts, including verification of the resulting short protein isoform on Western blots (for example, <em><a href="http://flybase.org/reports/FBgn0264006.html">dysc</a></em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jepson <em>et al.</em> 2012</a>; <em><a href="http://flybase.org/reports/FBgn0262527.html">nsl1</a></em>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Yu <em>et al.</em> 2010</a>; <em><a href="http://flybase.org/reports/FBgn0026086.html">Adar</a></em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2009</a>; <em><a href="http://flybase.org/reports/FBgn0250785.html">vari</a></em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bachmann <em>et al.</em> 2008</a>). These cases are flagged in FlyBase with comments that include the phrase “short protein isoform supported” in the transcript reports. Some loci appear to encode both coding and non-coding transcripts. One well-studied case is an RNA-splicing factor for which partially processed transcripts with retained introns are relatively stable; they appear to be retained in the nucleus and thus are not translated [<a href="http://flybase.org/reports/FBgn0003638.html">su(w<sup>a</sup>)</a>] (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chou <em>et al.</em> 1987</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zachar <em>et al.</em> 1994</a>). It is interesting to note that a number of other RNA-splicing factors or spliceosome components produce abundant transcripts with retained introns (for example, <em><a href="http://flybase.org/reports/FBgn0004587.html">B52</a></em>, <em><a href="http://flybase.org/reports/FBgn0039600.html">CG1646</a></em>, <em><a href="http://flybase.org/reports/FBgn0039581.html">Moca-cyp</a></em>) or alternative exons that disrupt the CDS (<em><a href="http://flybase.org/reports/FBgn0031883.html">Caper</a></em>). Another mechanistic possibility is that some cells can tolerate a relatively high level of aberrant transcripts, and some of these transcripts are simply biological noise. Such aberrant transcripts may be targets of nonsense-mediated decay (NMD). However, while the genes effecting NMD are highly conserved and are present in <em>Drosophila</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Behm-Ansmant, <em>et al.</em> 2007</a>), genes identified as direct targets of NMD in <em>Drosophila</em> were not found to have a higher than average level of premature termination codons (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chapin <em>et al.</em> 2014</a>). For all but a few dozen well-studied genes, which of these several possibilities may be true cannot be determined by assessment of the available data.</p><p id="p-59">For transcripts of this type that encode anomalously short polypeptides, we require a higher standard of support to merit annotation. Transcripts with retained introns may be annotated in cases where there are both supporting cDNA and RNA-Seq coverage data. Many low-frequency, low-confidence score RNA-Seq junctions have been found to throw the CDS out of frame and lead to premature termination codons; we typically do not annotate such junctions. Junctions present at a level of 10% or more relative to the highest junction frequency for any junction within the gene or junctions that are also supported by cDNA data have been annotated, even when they lead to early translation termination. An explanatory comment that includes “may or may not produce functional polypeptide” is included in the transcript report. Even based on these very conservative criteria, more than 700 gene models include transcripts flagged with such a comment. By contrast, the GENCODE consortium, which aims to annotate all gene features in the human genome, includes all transcripts predicted to generate truncated polypeptides. In the GENCODE 7 reference annotation set (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Harrow <em>et al.</em> 2012</a>), 44.9% of the translated isoforms (59.9% of the alternative isoforms) would lose either functional or structural domains or functional residues relative to the constitutive isoform (as defined in the APPRIS database; <a href="http://appris.bioinfo.cnio.es/">http://appris.bioinfo.cnio.es/</a>).</p></div><div id="sec-22" class="subsection"><h4>Alternative transcripts with nonoverlapping CDSs:</h4><p id="p-60">There are a number of gene models with short protein isoforms with a counterintuitive characteristic: pairs of alternative isoforms that encode nonoverlapping protein products. Both members of the pair are considered parts of one gene model because longer isoforms that are well-supported overlap both. In 22 of 30 such cases, these nonoverlapping protein isoforms are encoded by nonoverlapping transcripts (<em>i.e.</em>, with their own well supported and distinct 5′ transcription start and 3′ polyadenylation sites). Such gene models have been flagged with the comment, “Gene model includes transcripts encoding nonoverlapping portions of the full CDS.”</p><p id="p-61">Six of these cases have been identified in the literature, but in no case has the functional significance of each of the two nonoverlapping isoforms been fully characterized. The best characterized case is the <em><a href="http://flybase.org/reports/FBgn0001316.html">klar</a></em> gene (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The <em><a href="http://flybase.org/reports/FBgn0001316.html">klar</a></em> protein has been implicated in microtubule-based transport of lipid droplets (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Patterson <em>et al.</em> 2004</a>) and nuclei (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guo <em>et al.</em> 2005</a>). The existence of nonoverlapping transcripts encoding nonoverlapping protein isoforms has been characterized by cDNA analysis, RT-PCR, and Western blot (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Guo <em>et al.</em> 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kim <em>et al.</em> 2013</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The two nonoverlapping protein isoforms of klar. A GBrowse2 view of klar is shown, as it exists in R6.03, with nonoverlapping isoforms highlighted in yellow (klar-RC and -RI do not overlap klar-RD and -RH). The C-terminus of the longer, &quot;upstream&quot; isoforms (klar-RD and -RH) is sufficient for targeting proteins to lipid droplets, whereas the &quot;KASH&quot; domain present in the &quot;downstream&quot; isoforms (klar-RC and -RI) is sufficient for targeting to the nuclear envelope (Guo et al. 2005). The &quot;upstream&quot; nonoverlapping isoform is necessary for proper lipid droplet targeting in the embryo. While the KASH domain is necessary for nuclear migration in the embryo and retina, this function is associated with the &quot;full-length&quot; KASH-containing isoforms. The short KASH-containing isoform, which lacks motor interaction domains, is expressed (Western blot, immunofluorescence) and is apparently enriched in nurse cells but is not sufficient to rescue nuclear migration in the retina. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2024857102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The two nonoverlapping protein isoforms of &lt;em&gt;klar&lt;/em&gt;. A GBrowse2 view of &lt;em&gt;klar&lt;/em&gt; is shown, as it exists in R6.03, with nonoverlapping isoforms highlighted in yellow (&lt;em&gt;klar-RC&lt;/em&gt; and &lt;em&gt;-RI&lt;/em&gt; do not overlap &lt;em&gt;klar-RD&lt;/em&gt; and &lt;em&gt;-RH&lt;/em&gt;). The C-terminus of the longer, &quot;upstream&quot; isoforms (&lt;em&gt;klar-RD&lt;/em&gt; and &lt;em&gt;-RH&lt;/em&gt;) is sufficient for targeting proteins to lipid droplets, whereas the &quot;KASH&quot; domain present in the &quot;downstream&quot; isoforms (&lt;em&gt;klar-RC&lt;/em&gt; and &lt;em&gt;-RI&lt;/em&gt;) is sufficient for targeting to the nuclear envelope (Guo &lt;em&gt;et al.&lt;/em&gt; 2005). The &quot;upstream&quot; nonoverlapping isoform is necessary for proper lipid droplet targeting in the embryo. While the KASH domain is necessary for nuclear migration in the embryo and retina, this function is associated with the &quot;full-length&quot; KASH-containing isoforms. The short KASH-containing isoform, which lacks motor interaction domains, is expressed (Western blot, immunofluorescence) and is apparently enriched in nurse cells but is not sufficient to rescue nuclear migration in the retina. See Figure 2 and Figure 3 for GBrowse track descriptions. More information on data presented in GBrowse may be found at http://flybase.org/wiki/FlyBase:GBrowse_Tracks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/8/1721/F7.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/8/1721/F7.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/8/1721/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466527" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-62" class="first-child">The two nonoverlapping protein isoforms of <em><a href="http://flybase.org/reports/FBgn0001316.html">klar</a></em>. A GBrowse2 view of <em><a href="http://flybase.org/reports/FBgn0001316.html">klar</a></em> is shown, as it exists in R6.03, with nonoverlapping isoforms highlighted in yellow (<em><a href="http://flybase.org/reports/FBtr0112807.html">klar-RC</a></em> and <em><a href="http://flybase.org/reports/FBtr0343527.html">-RI</a></em> do not overlap <em><a href="http://flybase.org/reports/FBtr0112808.html">klar-RD</a></em> and <em><a href="http://flybase.org/reports/FBtr0343526.html">-RH</a></em>). The C-terminus of the longer, "upstream" isoforms (<em><a href="http://flybase.org/reports/FBtr0112808.html">klar-RD</a></em> and <em><a href="http://flybase.org/reports/FBtr0343526.html">-RH</a></em>) is sufficient for targeting proteins to lipid droplets, whereas the "KASH" domain present in the "downstream" isoforms (<em><a href="http://flybase.org/reports/FBtr0112807.html">klar-RC</a></em> and <em><a href="http://flybase.org/reports/FBtr0343527.html">-RI</a></em>) is sufficient for targeting to the nuclear envelope (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Guo <em>et al.</em> 2005</a>). The "upstream" nonoverlapping isoform is necessary for proper lipid droplet targeting in the embryo. While the KASH domain is necessary for nuclear migration in the embryo and retina, this function is associated with the "full-length" KASH-containing isoforms. The short KASH-containing isoform, which lacks motor interaction domains, is expressed (Western blot, immunofluorescence) and is apparently enriched in nurse cells but is not sufficient to rescue nuclear migration in the retina. See <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a> for GBrowse track descriptions. More information on data presented in GBrowse may be found at <a href="http://flybase.org/wiki/FlyBase:GBrowse_Tracks">http://flybase.org/wiki/FlyBase:GBrowse_Tracks</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-23" class="subsection"><h3>The future: more data and new approaches</h3><div id="sec-24" class="subsection"><h4>Additional transcriptome data:</h4><p id="p-63">Although we have been spoiled with an abundance of transcriptome data, there are some gaps that may be minimized in the future. For example, while stranded RNA-Seq data have been available for larval and adult tissues, the data originally available for the developmental profiles are unstranded. This has compromised our annotation of lncRNAs expressed only in embryonic or early larval stages, which we will be able to correct now that stranded data for these stages are in the pipeline. The reverse problem affects annotation of transcription start sites: modENCODE TSS data are heavily weighted toward embryonic RNA samples. We have made some use of RAMPAGE TSS data (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Batut <em>et al.</em> 2013</a>), which are drawn from a complete range of developmental stages, and will soon make more systematic use of that dataset. Additional RNA-Seq junction data from defined tissues may reveal tissue-specific splicing that would allow more accurate annotation of alternative transcript isoforms.</p></div><div id="sec-25" class="subsection"><h4>Small RNAs associated with distinct genomic regions:</h4><p id="p-64">Some classes of abundant small RNAs cannot be practically annotated as distinct transcripts or genes; these include piRNAs (reviewed in <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Senti and Brennecke 2010</a>) and endogenous siRNAs (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Okamura <em>et al.</em> 2008</a>; reviewed in <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Piatek and Werner 2014</a>). However, the chromosomal regions from which these two classes of small RNAs are derived can be defined (for example, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brennecke <em>et al.</em> 2007</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wen <em>et al.</em> 2014</a>; <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Brown <em>et al.</em> 2014</a>). We plan to add annotations of such regions as sequence features that can be viewed in GBrowse.</p></div><div id="sec-26" class="subsection"><h4>Confirming annotated translation starts and confronting uORFs:</h4><p id="p-65">Most of the high-throughput data we have used to inform our gene models define transcripts or transcribed regions, not translated regions. This may be about to change. Techniques being developed in vertebrate systems, notably ribosome profiling (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ingolia <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chew <em>et al.</em> 2013</a>), may soon be applied to <em>Drosophila</em>. Using a modification of ribosome profiling that leads to accumulation of ribosomes at sites of translation initiation, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Ingolia <em>et al.</em> (2011)</a> detected multiple cases of misidentified translation starts or alternative translation starts. These studies also identified a large number of out-of-frame alternative starts in 5′ UTRs, some of which may correspond to functional upstream ORFs (uORFs). Whether these results represent experimental false positives or biological noise, or are biologically relevant, remains to be determined (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chew <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ingolia <em>et al.</em> 2014</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pauli <em>et al.</em> 2014</a>).</p><p id="p-66">Our initial annotations of translation starts depended on predictions, protein homology, and cDNA data. In the absence of obvious BLAST alignment data to the contrary, our 2007 Gene Model Annotation Guidelines specified that we always set the start codon to the most upstream ATG. This has changed little in the subsequent years. The PhyloCSF exon prediction algorithm allowed us to determine whether annotated ATG sites were conserved and led to the reannotation of 164 transcripts with a downstream translation start site. When a downstream ATG site is annotated, we add a comment to the transcript explaining the annotation. A small number of genes (27) have been annotated with a noncanonical translation start (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Crosby <em>et al.</em> 2015</a>) and are flagged with comments at the gene and transcript levels. As discussed in <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Crosby <em>et al.</em> (2015)</a>, we suspect that there are many more noncanonical translation starts, and possibly numerous undetected alternative translation starts. We have made no attempt to annotate uORFs, except in cases where they show evidence of encoding conserved polypeptides (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Lin <em>et al.</em> 2007</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hayden and Bosco 2008</a>).</p></div><div id="sec-27" class="subsection"><h4>Very small polypeptides or long non-coding RNAs:</h4><p id="p-67">One of the most difficult decisions we often had to make was whether to annotate a gene supported by high-throughput transcriptome data as coding or non-coding (see section above on annotation of new genes). This question is at the intersection of two aspects of biology about which very little is known: the roles of lncRNAs and of very small proteins. High-throughput studies have utilized various algorithms to detect conservation of short ORFs. These approaches have failed to reliably detect proteins less than 30 amino acids and, of course, proteins that are not conserved will not be detected. Until translation-based or protein-based data become available for <em>Drosophila</em>, our catalog of genes encoding short proteins remains incomplete.</p><p id="p-68">In several vertebrate systems, ribosome profiling has been used to identify transcripts that appear to be translated and also the reverse, those that do not appear to be associated with the cytoplasmic translation machinery. Using mouse embryonic stem cells, <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Ingolia <em>et al.</em> (2011)</a> found that the majority of transcripts previously identified as lncRNAs contained one or more short ORFs that were bound by elongating ribosomes. When this technique was expanded to detect small translated ORFs in UTRs, many were found, especially in 5′ UTRs (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Ingolia <em>et al.</em> 2014</a>). Using ribosome profiling data from zebrafish, combined with machine-learning analysis, <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Chew <em>et al.</em> (2013)</a> also found that many previously identified lncRNAs appear to be protein-coding mRNAs. They identified an additional class, lncRNAs that have small ORFs that behave much like the uORFs found in 5′ UTRs, and postulate that ribosome association with this class of lncRNAs might be regulatory in nature.</p><p id="p-69">Most FlyBase lncRNA annotations are based simply on evidence of transcription of the region. Currently, “lncRNA” is a very general categorization that includes short single-exon genes <em>vs.</em> genes that contain very long introns and extend hundreds of kilobases (see discussion of sex-specific genes above), genes that are almost entirely antisense and others that are antisense only in part, genes that appear to be the precursors of miRNAs, snoRNAs, or piRNAs, and genes that are tightly stage-specific or tissue-specific <em>vs.</em> those that are widely expressed. Our annotations of non-coding genes will evolve as knowledge of this relatively new field develops.</p></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">We thank Stephanie Mohr and Yanhui Hu of the DRSC and our FlyBase colleagues for their helpful comments on the manuscript. FlyBase is supported by the National Human Genome Research Institute at the National Institutes of Health (U41 HG00739 to W.G., PI) and the Medical Research Council (UK) (G1000968 to N.B., PI).</p><p id="p-71"><em>Note added in proof</em>: See <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Crosby <em>et al</em>. 2015</a> (pp. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018937">1737–1749</a>) in this issue for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-72">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018929/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-73"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">2</span> The FlyBase Consortium members at the time of writing include the following: William Gelbart,* Nicholas H. Brown,<sup>†</sup> Thomas Kaufman,<sup>‡</sup> Maggie Werner-Washburne,<sup>§</sup> Richard Cripps,<sup>§</sup> Phill Baker,<sup>§</sup> Kris Broll,* Madeline Crosby,* Gilberto dos Santos,* David Emmert,* L. Sian Gramates,* Kathleen Falls,* Beverley B. Matthews,* Susan Russo,* Andrew Schroeder,* Pinglei Zhou,* Mark Zytkovicz,* Boris Adryan,<sup>†</sup> Giulia Antonazzo,<sup>†</sup> Helen Attrill,<sup>†</sup> Marta Costa,<sup>†</sup> Steven Marygold,<sup>†</sup> Peter McQuilton,<sup>†</sup> Gillian Millburn,<sup>†</sup> Laura Ponting,<sup>†</sup> Alix Rey,<sup>†</sup> Nicole Staudt,<sup>†</sup> Raymund Stefancsik,<sup>†</sup> Josh Goodman,<sup>‡</sup> Gary Grumbling,<sup>‡</sup> Victor Strelets,<sup>‡</sup> and Jim Thurmond<sup>‡</sup>. </p></li><li class="fn" id="fn-3"><p id="p-74"><em>Communicating editor: J. M. Cherry</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-04-29"><span class="received-label">Received </span>April 29, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-06-23"><span class="accepted-label">Accepted </span>June 23, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Matthews <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.1" data-doi="10.1126/science.1112699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aminetzach</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macpherson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Pesticide resistance via transposition-mediated adaptive gene truncation in Drosophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">309</span>: <span class="cit-fpage">764</span>–<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAminetzach%26rft.auinit1%253DY.%2BT.%26rft.volume%253D309%26rft.issue%253D5735%26rft.spage%253D764%26rft.epage%253D767%26rft.atitle%253DPesticide%2BResistance%2Bvia%2BTransposition-Mediated%2BAdaptive%2BGene%2BTruncation%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1112699%26rft_id%253Dinfo%253Apmid%252F16051794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDkvNTczNS83NjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.2" data-doi="10.1186/1471-213X-8-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Draga</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grawe</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knust</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">On the role of the MAGUK proteins encoded by Drosophila varicose during embryonic and postembryonic development.</span> <abbr class="cit-jnl-abbrev">BMC Dev. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bdevelopmental%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BDev%2BBiol%26rft.aulast%253DBachmann%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DOn%2Bthe%2Brole%2Bof%2Bthe%2BMAGUK%2Bproteins%2Bencoded%2Bby%2BDrosophila%2Bvaricose%2Bduring%2Bembryonic%2Band%2Bpostembryonic%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-213X-8-55%26rft_id%253Dinfo%253Apmid%252F18485238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-213X-8-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18485238&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.3" data-doi="10.1023/B:GENE.0000040391.27307.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balakirev</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The β-esterase gene cluster of Drosophila melanogaster: is ψEst-6 a pseudogene, a functional gene, or both?</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DBalakirev%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D179%26rft.atitle%253DThe%2Bbeta-esterase%2Bgene%2Bcluster%2Bof%2Bdrosophila%2Bmelanogaster%253A%2Bis%2BpsiEst-6%2Ba%2Bpseudogene%252C%2Ba%2Bfunctional%2Bgene%252C%2Bor%2Bboth%253F%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AGENE.0000040391.27307.51%26rft_id%253Dinfo%253Apmid%252F15330116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:GENE.0000040391.27307.51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15330116&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223204200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.4" data-doi="10.1101/gr.139618.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Batut</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plessy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carninci</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gingeras</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-fidelity promoter profiling reveals widespread alternative promoter usage and transposon-driven developmental gene expression.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.139618.112%26rft_id%253Dinfo%253Apmid%252F22936248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzEvMTY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.5" data-doi="10.1016/j.febslet.2007.05.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behm-Ansmant</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashima</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rehwinkel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauliere</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittkopp</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">mRNA quality control: an ancient machinery recognizes and degrades mRNAs with nonsense codons.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">581</span>: <span class="cit-fpage">2845</span>–<span class="cit-lpage">2853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DBehm-Ansmant%26rft.auinit1%253DI.%26rft.volume%253D581%26rft.issue%253D15%26rft.spage%253D2845%26rft.epage%253D2853%26rft.atitle%253DmRNA%2Bquality%2Bcontrol%253A%2Ban%2Bancient%2Bmachinery%2Brecognizes%2Band%2Bdegrades%2BmRNAs%2Bwith%2Bnonsense%2Bcodons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2007.05.027%26rft_id%253Dinfo%253Apmid%252F17531985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2007.05.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17531985&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247479900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.6" data-doi="10.1101/gr.116657.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berezikov</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robine</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samsonova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westholm</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naqvi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deep annotation of Drosophila melanogaster microRNAs yields insights into their processing, modification, and emergence.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.116657.110%26rft_id%253Dinfo%253Apmid%252F21177969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMjAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.7" data-doi="10.1016/j.cell.2007.01.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Discrete small RNA-generating loci as master regulators of transposon activity in Drosophila.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DIwase%26rft.auinit1%253DS.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D1089%26rft.epage%253D1103%26rft.atitle%253DDiscrete%2Bsmall%2BRNA-generating%2Bloci%2Bas%2Bmaster%2Bregulators%2Bof%2Btransposon%2Bactivity%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.01.043%26rft_id%253Dinfo%253Apmid%252F17346786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.01.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17346786&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245396200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brizuela</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elfring</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamkun</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennison</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic analysis of the brahma gene of Drosophila melanogaster and polytene chromosome subdivisions 72AB.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrizuela%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D137%26rft.issue%253D3%26rft.spage%253D803%26rft.epage%253D813%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2Bbrahma%2Bgene%2Bof%2BDrosophila%2Bmelanogaster%2Band%2Bpolytene%2Bchromosome%2Bsubdivisions%2B72AB.%26rft_id%253Dinfo%253Apmid%252F7916308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM3LzMvODAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.9" data-doi="10.1038/nature12962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boley</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, G. E. May, M. H. Stoiber, <em>et al</em>., <span class="cit-pub-date">2014</span> <span class="cit-article-title">Diversity and dynamics of the Drosophila transcriptome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12962%26rft_id%253Dinfo%253Apmid%252F24670639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24670639&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340840600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.10" data-doi="10.1038/459927a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunsalus</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Unlocking the secrets of the genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCelniker%26rft.auinit1%253DS.%2BE.%26rft.volume%253D459%26rft.issue%253D7249%26rft.spage%253D927%26rft.epage%253D930%26rft.atitle%253DUnlocking%2Bthe%2Bsecrets%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F459927a%26rft_id%253Dinfo%253Apmid%252F19536255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/459927a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19536255&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267063500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.11" data-doi="10.1534/g3.113.009357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rynearson</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollien</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">In vivo determination of direct targets of the nonsense-mediated decay pathway in Drosophila.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.009357%26rft_id%253Dinfo%253Apmid%252F24429422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMy80ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.12" data-doi="10.1016/j.diff.2009.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lakshmi</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDowell</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Spatial and temporal expression of dADAR mRNA and protein isoforms during embryogenesis in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Differentiation</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDifferentiation%253B%2Bresearch%2Bin%2Bbiological%2Bdiversity%26rft.stitle%253DDifferentiation%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D5%26rft.spage%253D312%26rft.epage%253D320%26rft.atitle%253DSpatial%2Band%2Btemporal%2Bexpression%2Bof%2BdADAR%2BmRNA%2Band%2Bprotein%2Bisoforms%2Bduring%2Bembryogenesis%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.diff.2009.08.003%26rft_id%253Dinfo%253Apmid%252F19720447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.diff.2009.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19720447&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.13" data-doi="10.1242/dev.098343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chew</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schier</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ribosome profiling reveals resemblance between long non-coding RNAs and 5′ leaders of coding RNAs.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">2828</span>–<span class="cit-lpage">2834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.098343%26rft_id%253Dinfo%253Apmid%252F23698349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTQwLzEzLzI4MjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zachar</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bingham</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Developmental expression of a regulatory gene is programmed at the level of splicing.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4095</span>–<span class="cit-lpage">4104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D6%26rft.spage%253D4095%26rft.atitle%253DDEVELOPMENTAL%2BEXPRESSION%2BOF%2BA%2BREGULATORY%2BGENE%2BIS%2BPROGRAMMED%2BAT%2BTHE%2BLEVEL%2BOF%2BSPLICING%26rft_id%253Dinfo%253Apmid%252F2832151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2832151&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gramates</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St. Pierre</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene model annotations for Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">The Rule-Benders</abbr>. G3 (Bethesda) <span class="cit-fpage">1737</span>–<span class="cit-lpage">1749</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.16" data-doi="10.1371/journal.pgen.1001064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croset</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rytz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cummins</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Budd</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brawand</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ancient protostome origin of chemosensory ionotropic glutamate receptors and the evolution of insect taste and olfaction.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCroset%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De1001064%26rft.epage%253De1001064%26rft.atitle%253DAncient%2Bprotostome%2Borigin%2Bof%2Bchemosensory%2Bionotropic%2Bglutamate%2Breceptors%2Band%2Bthe%2Bevolution%2Bof%2Binsect%2Btaste%2Band%2Bolfaction.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001064%26rft_id%253Dinfo%253Apmid%252F20808886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808886&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.17" data-doi="10.1101/gr.107854.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daines</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Drosophila melanogaster transcriptome by paired-end RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107854.110%26rft_id%253Dinfo%253Apmid%252F21177959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.18" data-doi="10.1093/nar/gku1099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strelets</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">FlyBase: Introduction of the Drosophila melanogaster Release 6 reference genome assembly and large-scale migration of genome annotations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D690</span>–<span class="cit-lpage">D697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1099%26rft_id%253Dinfo%253Apmid%252F25398896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My9EMS9ENjkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.19"><div class="cit-metadata"><cite>Drosophila 12 Genomes Consortium, <span class="cit-pub-date">2007</span> Evolution of genes and genomes on the Drosophila phylogeny. Nature 450: 203–218.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.20" data-doi="10.1101/gr.089391.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Findlay</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccoss</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Proteomic discovery of previously unannotated, rapidly evolving seminal fluid genes in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">886</span>–<span class="cit-lpage">896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DFindlay%26rft.auinit1%253DG.%2BD.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D886%26rft.epage%253D896%26rft.atitle%253DProteomic%2Bdiscovery%2Bof%2Bpreviously%2Bunannotated%252C%2Brapidly%2Bevolving%2Bseminal%2Bfluid%2Bgenes%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089391.108%26rft_id%253Dinfo%253Apmid%252F19411605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvODg2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.21" data-doi="10.1371/journal.pbio.0050106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pueyo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fouix</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couso</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Peptides encoded by short ORFs control development and define a new eukaryotic gene family.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGalindo%26rft.auinit1%253DM.%2BI.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De106%26rft.epage%253De106%26rft.atitle%253DPeptides%2Bencoded%2Bby%2Bshort%2BORFs%2Bcontrol%2Bdevelopment%2Band%2Bdefine%2Ba%2Bnew%2Beukaryotic%2Bgene%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050106%26rft_id%253Dinfo%253Apmid%252F17439302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17439302&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.22" data-doi="10.1038/nature09715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The developmental transcriptome of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGraveley%26rft.auinit1%253DB.%2BR.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D473%26rft.epage%253D479%26rft.atitle%253DThe%2Bdevelopmental%2Btranscriptome%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09715%26rft_id%253Dinfo%253Apmid%252F21179090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179090&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.23" data-doi="10.1091/mbc.E04-10-0920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jangi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welte</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Organelle-specific control of intracellular transport: distinctly targeted isoforms of the regulator Klar.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1406</span>–<span class="cit-lpage">1416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E04-10-0920%26rft_id%253Dinfo%253Apmid%252F15647372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzMvMTQwNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.24" data-doi="10.1093/nar/gkg169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milburn</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertone</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Identification of pseudogenes in the Drosophila melanogaster genome.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHarrison%26rft.auinit1%253DP.%2BM.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D1033%26rft.epage%253D1037%26rft.atitle%253DIdentification%2Bof%2Bpseudogenes%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg169%26rft_id%253Dinfo%253Apmid%252F12560500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMxLzMvMTAzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.25" data-doi="10.1101/gr.135350.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tapanari</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diekhans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GENCODE: The reference human genome annotation for The ENCODE Project.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1760</span>–<span class="cit-lpage">1774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHarrow%26rft.auinit1%253DJ.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1760%26rft.epage%253D1774%26rft.atitle%253DGENCODE%253A%2BThe%2Breference%2Bhuman%2Bgenome%2Bannotation%2Bfor%2BThe%2BENCODE%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.135350.111%26rft_id%253Dinfo%253Apmid%252F22955987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.26" data-doi="10.1093/nar/gkt1241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrow</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steward</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The vertebrate genome annotation browser 10 years on.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D771</span>–<span class="cit-lpage">D779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1241%26rft_id%253Dinfo%253Apmid%252F24316575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9ENzcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.27" data-doi="10.1186/1471-2164-9-61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bosco</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative genomic analysis of novel conserved peptide upstream open reading frames in Drosophila melanogaster and other dipteran species.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHayden%26rft.auinit1%253DC.%2BA.%26rft.volume%253D9%26rft.spage%253D61%26rft.epage%253D61%26rft.atitle%253DComparative%2Bgenomic%2Banalysis%2Bof%2Bnovel%2Bconserved%2Bpeptide%2Bupstream%2Bopen%2Breading%2Bframes%2Bin%2BDrosophila%2Bmelanogaster%2Band%2Bother%2Bdipteran%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-61%26rft_id%253Dinfo%253Apmid%252F18237443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18237443&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.28" data-doi="10.1073/pnas.1112672108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilgers</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrix</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Neural-specific elongation of 3′ UTRs during Drosophila development.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">15864</span>–<span class="cit-lpage">15869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1112672108%26rft_id%253Dinfo%253Apmid%252F21896737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM4LzE1ODY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.29" data-doi="10.1101/gr.090050.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hiller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Findeiss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Conserved introns reveal novel transcripts in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.090050.108%26rft_id%253Dinfo%253Apmid%252F19458021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTI4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.30" data-doi="10.1101/gr.112466.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoskins</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide analysis of promoter architecture in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.112466.110%26rft_id%253Dinfo%253Apmid%252F21177961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMTgyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.31" data-doi="10.1101/gr.185579.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoskins</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Release 6 reference sequence of the Drosophila melanogaster genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185579.114%26rft_id%253Dinfo%253Apmid%252F25589440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNDQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.32" data-doi="10.1111/j.1365-2443.2005.00910.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inagaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Numata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification and expression analysis of putative mRNA-like non-coding RNA in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1163</span>–<span class="cit-lpage">1173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DInagaki%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D1163%26rft.epage%253D1173%26rft.atitle%253DIdentification%2Band%2Bexpression%2Banalysis%2Bof%2Bputative%2BmRNA-like%2Bnon-coding%2BRNA%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2443.2005.00910.x%26rft_id%253Dinfo%253Apmid%252F16324153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2443.2005.00910.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16324153&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233626300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.33" data-doi="10.1016/j.cell.2011.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lareau</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DIngolia%26rft.auinit1%253DN.%2BT.%26rft.volume%253D147%26rft.issue%253D4%26rft.spage%253D789%26rft.epage%253D802%26rft.atitle%253DRibosome%2Bprofiling%2Bof%2Bmouse%2Bembryonic%2Bstem%2Bcells%2Breveals%2Bthe%2Bcomplexity%2Band%2Bdynamics%2Bof%2Bmammalian%2Bproteomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.10.002%26rft_id%253Dinfo%253Apmid%252F22056041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22056041&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296902300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.34" data-doi="10.1016/j.celrep.2014.07.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern-Ginossar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, G. J. Talhouarne, <em>et al</em>., <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ribosome profiling reveals pervasive translation outside of annotated protein-coding genes.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1365</span>–<span class="cit-lpage">1379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D8%26rft.spage%253D1365%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2014.07.045%26rft_id%253Dinfo%253Apmid%252F25159147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2014.07.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25159147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.35" data-doi="10.1371/journal.pgen.1002671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jepson</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahidullah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamaze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">dyschronic, a Drosophila homolog of a deaf-blindness gene, regulates circadian output and Slowpoke channels.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJepson%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1002671%26rft.epage%253De1002671%26rft.atitle%253Ddyschronic%252C%2Ba%2BDrosophila%2Bhomolog%2Bof%2Ba%2Bdeaf-blindness%2Bgene%252C%2Bregulates%2Bcircadian%2Boutput%2Band%2BSlowpoke%2Bchannels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002671%26rft_id%253Dinfo%253Apmid%252F22532808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22532808&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.36" data-doi="10.1101/gr.119974.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jungreis</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spokony</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negre</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence of abundant stop codon readthrough in Drosophila and other metazoa.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2096</span>–<span class="cit-lpage">2113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.119974.110%26rft_id%253Dinfo%253Apmid%252F21994247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8yMDk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.37" data-doi="10.1371/journal.pone.0055070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manna</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welte</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Novel isoforms of the transport regulator klar.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e55070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De55070%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0055070%26rft_id%253Dinfo%253Apmid%252F23457459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0055070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23457459&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.38" data-doi="10.1038/ncb1595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inagaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Small peptide regulators of actin-based cell morphogenesis encoded by a polycistronic mRNA.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DKondo%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D660%26rft.epage%253D665%26rft.atitle%253DSmall%2Bpeptide%2Bregulators%2Bof%2Bactin-based%2Bcell%2Bmorphogenesis%2Bencoded%2Bby%2Ba%2Bpolycistronic%2BmRNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1595%26rft_id%253Dinfo%253Apmid%252F17486114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17486114&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246922700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.39" data-doi="10.1093/nar/gkq1027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozomara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">miRBase: integrating microRNA annotation and deep-sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D152</span>–<span class="cit-lpage">D157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1027%26rft_id%253Dinfo%253Apmid%252F21037258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8xL0QxNTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.40" data-doi="10.1101/gr.6679507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Revisiting the protein-coding gene catalog of Drosophila melanogaster using 12 fly genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1823</span>–<span class="cit-lpage">1836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6679507%26rft_id%253Dinfo%253Apmid%252F17989253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xODIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.41" data-doi="10.1093/bioinformatics/btr209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jungreis</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PhyloCSF: a comparative genomics method to distinguish protein coding and non-coding regions.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">i275</span>–<span class="cit-lpage">i282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr209%26rft_id%253Dinfo%253Apmid%252F21685081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTMvaTI3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.42" data-doi="10.1186/gb-2011-12-11-r118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ladoukakis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magny</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couso</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hundreds of putatively functional small open reading frames in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLadoukakis%26rft.auinit1%253DE.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253DR118%26rft.epage%253DR118%26rft.atitle%253DHundreds%2Bof%2Bputatively%2Bfunctional%2Bsmall%2Bopen%2Breading%2Bframes%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-11-r118%26rft_id%253Dinfo%253Apmid%252F22118156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-11-r118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22118156&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.43" data-doi="10.1126/science.1238802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magny</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pueyo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearl</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cespedes</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niven</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Conserved regulation of cardiac calcium uptake by peptides encoded in small open reading frames.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">1116</span>–<span class="cit-lpage">1120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1238802%26rft_id%253Dinfo%253Apmid%252F23970561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDEvNjE1MC8xMTE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.44" data-doi="10.1371/journal.pgen.1002337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwire</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiggins</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Successive increases in the resistance of Drosophila to viral infection through a transposon insertion followed by a duplication.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMagwire%26rft.auinit1%253DM.%2BM.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002337%26rft.epage%253De1002337%26rft.atitle%253DSuccessive%2Bincreases%2Bin%2Bthe%2Bresistance%2Bof%2BDrosophila%2Bto%2Bviral%2Binfection%2Bthrough%2Ba%2Btransposon%2Binsertion%2Bfollowed%2Bby%2Ba%2BDuplication.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002337%26rft_id%253Dinfo%253Apmid%252F22028673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22028673&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinger</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korbie</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Expression of distinct RNAs from 3′ untranslated regions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2393</span>–<span class="cit-lpage">2403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D3%26rft.spage%253D2393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mungall</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> Annotation of the Drosophila melanogaster euchromatic genome: a systematic review. Genome Biol. 3: RESEARCH0083.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.47"><div class="cit-metadata"><cite>modENCODE Consortium, <span class="cit-pub-date">2010</span> Identification of functional elements and regulatory circuits by Drosophila modENCODE. Science 330: 1787–1797.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.48" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.49" data-doi="10.1126/science.287.5461.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dew</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasulo</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A whole-genome assembly of Drosophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2196</span>–<span class="cit-lpage">2204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DE.%2BW.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2196%26rft.epage%253D2204%26rft.atitle%253DA%2BWhole-Genome%2BAssembly%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2196%26rft_id%253Dinfo%253Apmid%252F10731133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.50" data-doi="10.1126/science.1181421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nechaev</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fargo</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Global analysis of short RNAs reveals widespread promoter-proximal stalling and arrest of Pol II in Drosophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1181421%26rft_id%253Dinfo%253Apmid%252F20007866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2My8zMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.51" data-doi="10.1038/nsmb.1438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">E. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Two distinct mechanisms generate endogenous siRNAs from bidirectional transcription in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DOkamura%26rft.auinit1%253DK.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D581%26rft.epage%253D590%26rft.atitle%253DTwo%2Bdistinct%2Bmechanisms%2Bgenerate%2Bendogenous%2BsiRNAs%2Bfrom%2Bbidirectional%2Btranscription%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1438%26rft_id%253Dinfo%253Apmid%252F18500351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18500351&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.52" data-doi="10.1371/journal.pone.0005040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Burst of young retrogenes and independent retrogene formation in mammals.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPan%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De5040%26rft.epage%253De5040%26rft.atitle%253DBurst%2Bof%2Byoung%2Bretrogenes%2Band%2Bindependent%2Bretrogene%2Bformation%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005040%26rft_id%253Dinfo%253Apmid%252F19325906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19325906&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.53" data-doi="10.1091/mbc.E03-06-0374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molofsky</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cater</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The functions of klarsicht and nuclear lamin in developmentally regulated nuclear migrations of photoreceptor cells in the Drosophila eye.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E03-06-0374%26rft_id%253Dinfo%253Apmid%252F14617811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE1LzIvNjAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pauli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schier</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying (non-)coding RNAs and small peptides: Challenges and opportunities.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D37%26rft.spage%253D103%26rft_id%253Dinfo%253Apmid%252F25345765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25345765&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.55" data-doi="10.1093/nar/gki476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maglott</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">NCBi Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">Database issue</span>): <span class="cit-fpage">D501</span>–<span class="cit-lpage">D504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPruitt%26rft.auinit1%253DK.%2BD.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD501%26rft.epage%253DD504%26rft.atitle%253DNCBI%2BReference%2BSequence%2B%2528RefSeq%2529%253A%2Ba%2Bcurated%2Bnon-redundant%2Bsequence%2Bdatabase%2Bof%2Bgenomes%252C%2Btranscripts%2Band%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki476%26rft_id%253Dinfo%253Apmid%252F15608248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8xL0Q1MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.56" data-doi="10.1042/BST20140068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piatek</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Endogenous siRNAs: regulators of internal affairs.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1174</span>–<span class="cit-lpage">1179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST20140068%26rft_id%253Dinfo%253Apmid%252F25110021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo5OiI0Mi80LzExNzQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.57" data-doi="10.1673/031.009.1801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The insect chemoreceptor superfamily in Drosophila pseudoobscura: molecular evolution of ecologically-relevant genes over 25 million years.</span> <abbr class="cit-jnl-abbrev">J. Insect Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInsect%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1673%252F031.009.1801%26rft_id%253Dinfo%253Apmid%252F19613461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamlzIjtzOjU6InJlc2lkIjtzOjY6IjkvMS8xOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.58" data-doi="10.1073/pnas.2335847100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warr</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Molecular evolution of the insect chemoreceptor gene superfamily in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>(<span class="cit-supplement">Supplement 2</span>): <span class="cit-fpage">14537</span>–<span class="cit-lpage">14542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2335847100%26rft_id%253Dinfo%253Apmid%252F14608037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNzoiMTAwL3N1cHBsXzIvMTQ1MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.59" data-doi="10.1016/j.cell.2006.05.053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marques-Souza</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A segmentation gene in tribolium produces a polycistronic mRNA that codes for multiple conserved peptides.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSavard%26rft.auinit1%253DJ.%26rft.volume%253D126%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D569%26rft.atitle%253DA%2Bsegmentation%2Bgene%2Bin%2Btribolium%2Bproduces%2Ba%2Bpolycistronic%2BmRNA%2Bthat%2Bcodes%2Bfor%2Bmultiple%2Bconserved%2Bpeptides.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.05.053%26rft_id%253Dinfo%253Apmid%252F16901788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.05.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16901788&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239883400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.60" data-doi="10.1016/j.tig.2010.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senti</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The piRNA pathway: a fly’s perspective on the guardian of the genome.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DSenti%26rft.auinit1%253DK.%2BA.%26rft.volume%253D26%26rft.issue%253D12%26rft.spage%253D499%26rft.epage%253D509%26rft.atitle%253DThe%2BpiRNA%2Bpathway%253A%2Ba%2Bfly%2527s%2Bperspective%2Bon%2Bthe%2Bguardian%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.08.007%26rft_id%253Dinfo%253Apmid%252F20934772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20934772&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285237400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.61" data-doi="10.1073/pnas.1407293111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sisu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative analysis of pseudogenes across three phyla.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">13361</span>–<span class="cit-lpage">13366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1407293111%26rft_id%253Dinfo%253Apmid%252F25157146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM3LzEzMzYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.62" data-doi="10.1016/j.celrep.2012.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smibert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westholm</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shenker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global patterns of tissue-specific alternative polyadenylation in Drosophila.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D1%26rft.spage%253D277%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.01.001%26rft_id%253Dinfo%253Apmid%252F22685694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22685694&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stapleton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brokstein</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, M. Champe, <em>et al</em>., <span class="cit-pub-date">2002</span> A Drosophila full-length cDNA resource. Genome Biol. 3: RESEARCH0080.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.64" data-doi="10.1242/dev.047324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azzam</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaschula</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Developmental RNA processing of 3′UTRs in Hox mRNAs as a context-dependent mechanism modulating visibility to microRNAs.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">2951</span>–<span class="cit-lpage">2960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.047324%26rft_id%253Dinfo%253Apmid%252F20667912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM3LzE3LzI5NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.65" data-doi="10.1073/pnas.0501422102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tupy</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dailey</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans-Holm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebel</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification of putative noncoding polyadenylated transcripts in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">5495</span>–<span class="cit-lpage">5500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501422102%26rft_id%253Dinfo%253Apmid%252F15809421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE1LzU0OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.66" data-doi="10.1101/gr.161554.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammed</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bortolamiol-Becet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robine</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Diversity of miRNAs, siRNAs, and piRNAs across 25 Drosophila cell lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.161554.113%26rft_id%253Dinfo%253Apmid%252F24985917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTIzNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvOC8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.67" data-doi="10.1016/S0965-1748(97)00056-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfner</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harada</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertram</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelick</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraus</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">New genes for male accessory gland proteins in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Insect Biochem. Mol. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2Bbiochemistry%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DInsect%2BBiochem%2BMol%2BBiol%26rft.aulast%253DWolfner%26rft.auinit1%253DM.%2BF.%26rft.volume%253D27%26rft.issue%253D10%26rft.spage%253D825%26rft.epage%253D834%26rft.atitle%253DNew%2Bgenes%2Bfor%2Bmale%2Baccessory%2Bgland%2Bproteins%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0965-1748%252897%252900056-8%26rft_id%253Dinfo%253Apmid%252F9474779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0965-1748(97)00056-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9474779&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071797000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.68" data-doi="10.1016/j.devcel.2012.05.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokol</span>  <span class="cit-name-given-names">N. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">let-7-complex microRNAs regulate the temporal identity of Drosophila mushroom body neurons via chinmo.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">202</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DWu%26rft.auinit1%253DY.%2BC.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D202%26rft.epage%253D209%26rft.atitle%253DLet-7-complex%2BmicroRNAs%2Bregulate%2Bthe%2Btemporal%2Bidentity%2Bof%2BDrosophila%2Bmushroom%2Bbody%2Bneurons%2Bvia%2Bchinmo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2012.05.013%26rft_id%253Dinfo%253Apmid%252F22814608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2012.05.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22814608&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306583800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.69" data-doi="10.1093/gbe/evs020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marques</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibbit</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haerty</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassett</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification and properties of 1,119 candidate LincRNA loci in the Drosophila melanogaster genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs020%26rft_id%253Dinfo%253Apmid%252F22403033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjQvNC80MjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wharton</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> E(nos)/CG4699 required for nanos function in the female germ line of Drosophila. Genesis 48: 161–170.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1721.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zachar</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mims</span>  <span class="cit-name-given-names">I. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bingham</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Analysis of autoregulation at the level of pre-mRNA splicing of the suppressor-of-white-apricot gene in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZachar%26rft.auinit1%253DZ.%26rft.volume%253D137%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D150%26rft.atitle%253DAnalysis%2Bof%2Bautoregulation%2Bat%2Bthe%2Blevel%2Bof%2Bpre-mRNA%2Bsplicing%2Bof%2Bthe%2Bsuppressor-of-white-apricot%2Bgene%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Apmid%252F8056305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM3LzEvMTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS84LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/8/1721.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1713" title="Chromatin Signature Identifies Monoallelic Gene Expression Across Mammalian Cell Types" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1737" title="Gene Model Annotations for Drosophila melanogaster: The Rule-Benders" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/8"><span>Volume 5 Issue 8, August 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/8.cover-source.jpg?itok=wjPUfMS9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1721" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FIQtRcmTKqbFNUXqYMtvvAjdESeEnYFFwjPWSNxaR8E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/8/1721" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/8/1721" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/8/1721" class="active" data-icon-position="" data-hide-link-title="0">Gene Model Annotations for Drosophila melanogaster: Impact of High-Throughput Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460001" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-bR5605tTACTjmX2f3S7fCYAmK98Gq_eCb3fjRxo2McE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1721.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/8/1721" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WHcJf0F4wtZ7D-OxD4crbsh41uEvB44EPnG_2S4lA5g" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/8/1721" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1721.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460001" id="node460001--2" data-pisa="ggg;5/8/1721" data-pisa-master="ggg;g3.115.018929" data-apath="/ggg/5/8/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Model Annotations for <em>Drosophila melanogaster</em>: Impact of High-Throughput Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Beverley B. Matthews</span>, <span class="highwire-citation-author" data-delta="1">Gilberto dos Santos</span>, <span class="highwire-citation-author" data-delta="2">Madeline A. Crosby</span>, <span class="highwire-citation-author" data-delta="3">David B. Emmert</span>, <span class="highwire-citation-author" data-delta="4">Susan E. St. Pierre</span>, <span class="highwire-citation-author" data-delta="5">L. Sian Gramates</span>, <span class="highwire-citation-author" data-delta="6">Pinglei Zhou</span>, <span class="highwire-citation-author" data-delta="7">Andrew J. Schroeder</span>, <span class="highwire-citation-author" data-delta="8">Kathleen Falls</span>, <span class="highwire-citation-author" data-delta="9">Victor Strelets</span>, <span class="highwire-citation-author" data-delta="10">Susan M. Russo</span>, <span class="highwire-citation-author" data-delta="11">William M. Gelbart</span> and <span class="highwire-citation-author" data-delta="12"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">the FlyBase Consortium</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1736;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018929 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460001--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beverley B. Matthews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beverley%2BB.%2BMatthews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matthews%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeverley%2BB.%2BMatthews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bmatthew@morgan.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">bmatthew@morgan.harvard.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gilberto dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilberto%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilberto%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Madeline A. Crosby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madeline%2BA.%2BCrosby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crosby%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadeline%2BA.%2BCrosby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David B. Emmert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BEmmert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmert%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BEmmert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Susan E. St. Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BSt.%2BPierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Pierre%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusan%2BE.%2BSt.%2BPierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">L. Sian Gramates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BSian%2BGramates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gramates%20LS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BSian%2BGramates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pinglei Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pinglei%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APinglei%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Andrew J. Schroeder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BSchroeder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schroeder%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BSchroeder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kathleen Falls </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BFalls%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falls%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BFalls%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Victor Strelets </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BStrelets%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strelets%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictor%2BStrelets%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Susan M. Russo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BM.%2BRusso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Russo%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusan%2BM.%2BRusso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">William M. Gelbart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BM.%2BGelbart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gelbart%20WM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BM.%2BGelbart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of Cambridge, Cambridge CB2 3EH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of New Mexico, Albuquerque, New Mexico 87131</div></span></div></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/8/1721" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/8/1721&amp;title=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/8/1721&amp;title=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/8/1721&amp;title=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/8/1721&amp;text=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/8/1721&amp;title=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/8/1721&amp;t=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/8/1721&amp;title=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/8/1721&amp;title=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460001/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460001" id="node460001--3" data-pisa="ggg;5/8/1721" data-pisa-master="ggg;g3.115.018929" data-apath="/ggg/5/8/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Model Annotations for <em>Drosophila melanogaster</em>: Impact of High-Throughput Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Beverley B. Matthews</span>, <span class="highwire-citation-author" data-delta="1">Gilberto dos Santos</span>, <span class="highwire-citation-author" data-delta="2">Madeline A. Crosby</span>, <span class="highwire-citation-author" data-delta="3">David B. Emmert</span>, <span class="highwire-citation-author" data-delta="4">Susan E. St. Pierre</span>, <span class="highwire-citation-author" data-delta="5">L. Sian Gramates</span>, <span class="highwire-citation-author" data-delta="6">Pinglei Zhou</span>, <span class="highwire-citation-author" data-delta="7">Andrew J. Schroeder</span>, <span class="highwire-citation-author" data-delta="8">Kathleen Falls</span>, <span class="highwire-citation-author" data-delta="9">Victor Strelets</span>, <span class="highwire-citation-author" data-delta="10">Susan M. Russo</span>, <span class="highwire-citation-author" data-delta="11">William M. Gelbart</span> and <span class="highwire-citation-author" data-delta="12"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">the FlyBase Consortium</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1736;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018929 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460001--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beverley B. Matthews </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beverley%2BB.%2BMatthews%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matthews%20BB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeverley%2BB.%2BMatthews%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bmatthew@morgan.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">bmatthew@morgan.harvard.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gilberto dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilberto%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilberto%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Madeline A. Crosby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madeline%2BA.%2BCrosby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crosby%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMadeline%2BA.%2BCrosby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David B. Emmert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BB.%2BEmmert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmert%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BB.%2BEmmert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Susan E. St. Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BE.%2BSt.%2BPierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Pierre%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusan%2BE.%2BSt.%2BPierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">L. Sian Gramates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BSian%2BGramates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gramates%20LS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BSian%2BGramates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pinglei Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pinglei%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APinglei%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Andrew J. Schroeder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BSchroeder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schroeder%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BJ.%2BSchroeder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kathleen Falls </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BFalls%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falls%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BFalls%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Victor Strelets </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BStrelets%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Strelets%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictor%2BStrelets%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Susan M. Russo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BM.%2BRusso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Russo%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusan%2BM.%2BRusso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">William M. Gelbart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BM.%2BGelbart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gelbart%20WM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BM.%2BGelbart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of Cambridge, Cambridge CB2 3EH, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, University of New Mexico, Albuquerque, New Mexico 87131</div></span></div></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1721&amp;rft.epage=1736&amp;rft.atitle=Gene%20Model%20Annotations%20for%20Drosophila%20melanogaster%3A%20Impact%20of%20High-Throughput%20Data&amp;rft.volume=5&amp;rft.issue=8&amp;rft.date=2015-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Matthews%2C+Beverley+B.&amp;rft.au=dos+Santos%2C+Gilberto&amp;rft.au=Crosby%2C+Madeline+A.&amp;rft.au=Emmert%2C+David+B.&amp;rft.au=St.+Pierre%2C+Susan+E.&amp;rft.au=Gramates%2C+L.+Sian&amp;rft.au=Zhou%2C+Pinglei&amp;rft.au=Schroeder%2C+Andrew+J.&amp;rft.au=Falls%2C+Kathleen&amp;rft.au=Strelets%2C+Victor&amp;rft.au=Russo%2C+Susan+M.&amp;rft.au=Gelbart%2C+William+M.&amp;rft.au=%2C+"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460001/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460001/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460001/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460001/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460001/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460001/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460001/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460001/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460001/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460001/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460001/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460001/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/8/1721" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/8/1721.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/8/1721.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/8/1721.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/8/1721.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
