<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/8/1575.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/8/1575.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/8/1575/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/8/1575.atom" />
<meta name="HW.pisa" content="ggg;5/8/1575" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs" />
<meta name="DC.Identifier" content="10.1534/g3.115.018119" />
<meta name="DC.Date" content="2015-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Hidalgo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection has been widely implemented in dairy cattle breeding when the aim is to improve performance of purebred animals. In pigs, however, the final product is a crossbred animal. This may affect the efficiency of methods that are currently implemented for dairy cattle. Therefore, the objective of this study was to determine the accuracy of predicted breeding values in crossbred pigs using purebred genomic and phenotypic data. A second objective was to compare the predictive ability of SNPs when training is done in either single or multiple populations for four traits: age at first insemination (AFI); total number of piglets born (TNB); litter birth weight (LBW); and litter variation (LVR). We performed marker-based and pedigree-based predictions. Within-population predictions for the four traits ranged from 0.21 to 0.72. Multi-population prediction yielded accuracies ranging from 0.18 to 0.67. Predictions across purebred populations as well as predicting genetic merit of crossbreds from their purebred parental lines for AFI performed poorly (not significantly different from zero). In contrast, accuracies of across-population predictions and accuracies of purebred to crossbred predictions for LBW and LVR ranged from 0.08 to 0.31 and 0.11 to 0.31, respectively. Accuracy for TNB was zero for across-population prediction, whereas for purebred to crossbred prediction it ranged from 0.08 to 0.22. In general, marker-based outperformed pedigree-based prediction across populations and traits. However, in some cases pedigree-based prediction performed similarly or outperformed marker-based prediction. There was predictive ability when purebred populations were used to predict crossbred genetic merit using an additive model in the populations studied. AFI was the only exception, indicating that predictive ability depends largely on the genetic correlation between PB and CB performance, which was 0.31 for AFI. Multi-population prediction was no better than within-population prediction for the purebred validation set. Accuracy of prediction was very trait-dependent." />
<meta name="DC.Contributor" content="André M. Hidalgo" />
<meta name="DC.Contributor" content="John W. M. Bastiaansen" />
<meta name="DC.Contributor" content="Marcos S. Lopes" />
<meta name="DC.Contributor" content="Barbara Harlizius" />
<meta name="DC.Contributor" content="Martien A. M. Groenen" />
<meta name="DC.Contributor" content="Dirk-Jan de Koning" />
<meta name="article:published_time" content="2015-08-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection has been widely implemented in dairy cattle breeding when the aim is to improve performance of purebred animals. In pigs, however, the final product is a crossbred animal. This may affect the efficiency of methods that are currently implemented for dairy cattle. Therefore, the objective of this study was to determine the accuracy of predicted breeding values in crossbred pigs using purebred genomic and phenotypic data. A second objective was to compare the predictive ability of SNPs when training is done in either single or multiple populations for four traits: age at first insemination (AFI); total number of piglets born (TNB); litter birth weight (LBW); and litter variation (LVR). We performed marker-based and pedigree-based predictions. Within-population predictions for the four traits ranged from 0.21 to 0.72. Multi-population prediction yielded accuracies ranging from 0.18 to 0.67. Predictions across purebred populations as well as predicting genetic merit of crossbreds from their purebred parental lines for AFI performed poorly (not significantly different from zero). In contrast, accuracies of across-population predictions and accuracies of purebred to crossbred predictions for LBW and LVR ranged from 0.08 to 0.31 and 0.11 to 0.31, respectively. Accuracy for TNB was zero for across-population prediction, whereas for purebred to crossbred prediction it ranged from 0.08 to 0.22. In general, marker-based outperformed pedigree-based prediction across populations and traits. However, in some cases pedigree-based prediction performed similarly or outperformed marker-based prediction. There was predictive ability when purebred populations were used to predict crossbred genetic merit using an additive model in the populations studied. AFI was the only exception, indicating that predictive ability depends largely on the genetic correlation between PB and CB performance, which was 0.31 for AFI. Multi-population prediction was no better than within-population prediction for the purebred validation set. Accuracy of prediction was very trait-dependent.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/08/01" />
<meta name="citation_mjid" content="ggg;5/8/1575" />
<meta name="citation_id" content="5/8/1575" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/8/1575" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/8/1575.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/8/1575.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/8/1575.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.018119" />
<meta name="citation_pmid" content="26019187" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1575" />
<meta name="citation_lastpage" content="1583" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="André M. Hidalgo" />
<meta name="citation_author_institution" content="Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH" />
<meta name="citation_author_institution" content="Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07" />
<meta name="citation_author_email" content="andre.hidalgo@wur.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5116-4628" />
<meta name="citation_author" content="John W. M. Bastiaansen" />
<meta name="citation_author_institution" content="Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH" />
<meta name="citation_author" content="Marcos S. Lopes" />
<meta name="citation_author_institution" content="Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH" />
<meta name="citation_author_institution" content="Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA" />
<meta name="citation_author" content="Barbara Harlizius" />
<meta name="citation_author_institution" content="Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA" />
<meta name="citation_author" content="Martien A. M. Groenen" />
<meta name="citation_author_institution" content="Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH" />
<meta name="citation_author" content="Dirk-Jan de Koning" />
<meta name="citation_author_institution" content="Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. M. Badke;citation_title=Accuracy of estimation of genomic breeding values in pigs using low-density genotypes and imputation. G3 (Bethesda);citation_pages=623-631;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=A. Cecchinato;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=L. Gallo;citation_author=P. Carnier;citation_title=The relevance of purebred information for predicting genetic merit of survival at birth of crossbred piglets.;citation_volume=88;citation_year=2010;citation_pmid=19897636;citation_doi=10.2527/jas.2008-1744" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=L. Chen;citation_author=F. Schenkel;citation_author=M. Vinsky;citation_author=D. H. Crews;citation_author=C. Li;citation_title=Accuracy of predicting genomic breeding values for residual feed intake in Angus and Charolais beef cattle;citation_pages=4669-4678;citation_volume=91;citation_year=2013;citation_issue=10;citation_pmid=24078618;citation_doi=10.2527/jas.2013-5715" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=O. F. Christensen;citation_author=P. Madsen;citation_author=B. Nielsen;citation_author=T. Ostersen;citation_author=G. Su;citation_title=Single-step methods for genomic evaluation in pigs.;citation_pages=1565-1571;citation_volume=6;citation_year=2012;citation_issue=10;citation_pmid=22717310;citation_doi=10.1017/S1751731112000742" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=O. F. Christensen;citation_author=P. Madsen;citation_author=B. Nielsen;citation_author=G. Su;citation_title=Genomic evaluation of both purebred and crossbred performances.;citation_pages=23;citation_volume=46;citation_year=2014;citation_pmid=24666469;citation_doi=10.1186/1297-9686-46-23" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. De Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=J. C. M. Dekkers;citation_title=Marker-assisted selection for commercial crossbred performance.;citation_volume=85;citation_year=2007;citation_pmid=17504955;citation_doi=10.2527/jas.2006-683" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=D. S. Falconer;citation_title=The problem of environment and selection.;citation_pages=293-298;citation_volume=86;citation_year=1952;citation_doi=10.1086/281736" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. Forni;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. J. Garrick;citation_title=Deregressing estimated breeding values and weighting information for genomic regression analyses.;citation_pages=55-55;citation_volume=41;citation_year=2009;citation_pmid=20043827;citation_doi=10.1186/1297-9686-41-55" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_title=Genomic selection of purebreds for crossbred performance.;citation_pages=12-12;citation_volume=41;citation_year=2009;citation_pmid=19284703;citation_doi=10.1186/1297-9686-41-12" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. S. Lund;citation_author=G. Su;citation_author=L. Janss;citation_author=B. Guldbrandtsen;citation_author=R. F. Brøndum;citation_title=Genomic evaluation of cattle in a multi-breed context.;citation_pages=101-110;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.05.008" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=E. Lutaaya;citation_author=I. Misztal;citation_author=J. W. Mabry;citation_author=T. Short;citation_author=H. H. Timm;citation_title=Genetic parameter estimates from joint evaluation of purebreds and crossbreds in swine using the crossbred model;citation_pages=3002-3007;citation_volume=79;citation_year=2001;citation_issue=12;citation_pmid=11811453" />
<meta name="citation_reference" content="citation_journal_title=Anim. Sci. Pap. Rep.;citation_author=H. Nagyné-Kiszlinger;citation_author=J. Farkas;citation_author=G. Kövér;citation_author=I. Nagy;citation_title=Selection for reproduction traits in Hungarian pig breeding in a two-way cross.;citation_pages=315-322;citation_volume=31;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. Ostersen;citation_author=. O. F. Christensen;citation_author=B. Nielsen;citation_author=G. Su;citation_author=P. Madsen;citation_title=Deregressed EBV as the response variable yield more reliable genomic predictions than traditional EBV in pure-bred pigs.;citation_pages=38-38;citation_volume=43;citation_year=2011;citation_pmid=22070746;citation_doi=10.1186/1297-9686-43-38" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=A. M. Ramos;citation_title=Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.;citation_pages=1-13;citation_volume=4;citation_year=2009;citation_pmid=19956663;citation_doi=10.1371/journal.pone.0005361" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=B. Tier;citation_author=K. Meyer;citation_title=Approximating prediction error covariances among additive genetic effects within animals in multiple-trait and random regression models.;citation_pages=77-89;citation_volume=121;citation_year=2004;citation_doi=10.1111/j.1439-0388.2003.00444.x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=A. Toosi;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=Genomic selection in admixed and crossbred populations.;citation_volume=88;citation_year=2010;citation_pmid=19749023;citation_doi=10.2527/jas.2009-1975" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=L. Tusell;citation_author=P. Pérez-Rodriguez;citation_author=S. Forni;citation_author=X.-L. Wu;citation_author=D. Gianola;citation_title=Genome-enabled methods for predicting litter size in pigs: A comparison.;citation_pages=1739-1749;citation_volume=7;citation_year=2013;citation_pmid=23880322;citation_doi=10.1017/S1751731113001389" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. Van Tassell;citation_author=G. R. Wiggans;citation_author=T. S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=R. Veroneze;citation_title=Linkage disequilibrium patterns and persistence of phase in purebred and crossbred pig (Sus Scrofa) Populations.;citation_pages=126;citation_volume=15;citation_year=2014;citation_pmid=25421851" />
<meta name="citation_reference" content="citation_year=2013" />
<meta name="citation_reference" content="citation_author=J. Zeng;citation_author=A. Toosi;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=B. Zumbach;citation_author=I. Misztal;citation_author=S. Tsuruta;citation_author=J. Holl;citation_author=W. Herring;citation_title=Genetic correlations between two strains of Durocs and crossbreds from differing production environments for slaughter traits.;citation_volume=85;citation_year=2007;citation_pmid=17178815;citation_doi=10.2527/jas.2006-499" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection has been widely implemented in dairy cattle breeding when the aim is to improve performance of purebred animals. In pigs, however, the final product is a crossbred animal. This may affect the efficiency of methods that are currently implemented for dairy cattle. Therefore, the objective of this study was to determine the accuracy of predicted breeding values in crossbred pigs using purebred genomic and phenotypic data. A second objective was to compare the predictive ability of SNPs when training is done in either single or multiple populations for four traits: age at first insemination (AFI); total number of piglets born (TNB); litter birth weight (LBW); and litter variation (LVR). We performed marker-based and pedigree-based predictions. Within-population predictions for the four traits ranged from 0.21 to 0.72. Multi-population prediction yielded accuracies ranging from 0.18 to 0.67. Predictions across purebred populations as well as predicting genetic merit of crossbreds from their purebred parental lines for AFI performed poorly (not significantly different from zero). In contrast, accuracies of across-population predictions and accuracies of purebred to crossbred predictions for LBW and LVR ranged from 0.08 to 0.31 and 0.11 to 0.31, respectively. Accuracy for TNB was zero for across-population prediction, whereas for purebred to crossbred prediction it ranged from 0.08 to 0.22. In general, marker-based outperformed pedigree-based prediction across populations and traits. However, in some cases pedigree-based prediction performed similarly or outperformed marker-based prediction. There was predictive ability when purebred populations were used to predict crossbred genetic merit using an additive model in the populations studied. AFI was the only exception, indicating that predictive ability depends largely on the genetic correlation between PB and CB performance, which was 0.31 for AFI. Multi-population prediction was no better than within-population prediction for the purebred validation set. Accuracy of prediction was very trait-dependent." />
<meta name="og-title" property="og:title" content="Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/8/1575" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection has been widely implemented in dairy cattle breeding when the aim is to improve performance of purebred animals. In pigs, however, the final product is a crossbred animal. This may affect the efficiency of methods that are currently implemented for dairy cattle. Therefore, the objective of this study was to determine the accuracy of predicted breeding values in crossbred pigs using purebred genomic and phenotypic data. A second objective was to compare the predictive ability of SNPs when training is done in either single or multiple populations for four traits: age at first insemination (AFI); total number of piglets born (TNB); litter birth weight (LBW); and litter variation (LVR). We performed marker-based and pedigree-based predictions. Within-population predictions for the four traits ranged from 0.21 to 0.72. Multi-population prediction yielded accuracies ranging from 0.18 to 0.67. Predictions across purebred populations as well as predicting genetic merit of crossbreds from their purebred parental lines for AFI performed poorly (not significantly different from zero). In contrast, accuracies of across-population predictions and accuracies of purebred to crossbred predictions for LBW and LVR ranged from 0.08 to 0.31 and 0.11 to 0.31, respectively. Accuracy for TNB was zero for across-population prediction, whereas for purebred to crossbred prediction it ranged from 0.08 to 0.22. In general, marker-based outperformed pedigree-based prediction across populations and traits. However, in some cases pedigree-based prediction performed similarly or outperformed marker-based prediction. There was predictive ability when purebred populations were used to predict crossbred genetic merit using an additive model in the populations studied. AFI was the only exception, indicating that predictive ability depends largely on the genetic correlation between PB and CB performance, which was 0.31 for AFI. Multi-population prediction was no better than within-population prediction for the purebred validation set. Accuracy of prediction was very trait-dependent." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/5/8/1575" />
    <title>Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"x7Y-zag5oqLtIDVQvjjvRy1N_oFH4a2zlxD4hZpvv1g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459992","apath":"\/ggg\/5\/8\/1575.atom","pisa":"ggg;5\/8\/1575","ac":{"\/ggg\/5\/8\/1575.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/8\/1575.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1575"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/8\/1575"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/8\/1575.abstract":true,"\/content\/5\/8\/1575":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I9sXZitw6kimSQpFWX8Njf8Muyx7vmmC3clZZr41MIUG1MBNQXNW8qnyS+0R1kw0OepaI\\\\\\\/Dsj98Z4ka3wzXmY3t6ZBTHXjp05sUiqQliWvNXVnEPnyICRAnUToJRLHrjXl\\\\\\\/51Gg8tQ+nGqPs2u+3SR\\\\\\\/RsGoBUHITMgOuwUhC0Iu0I5RVL2yCHbqOC8RzHjpdQzYyFRMk1C8zCGc2szNA0ZDCYLiKEOCfRl61AIbe+u76d0ktW9QGzzHcRNYC7N9PGpBGMcTQ0qM4RtIS3XdCzn7KzKA5+eCqCF3\\\\\\\/ehMeCC\\\\\\\/g33F88Q\\\\\\\/qjtVFAZN2sy0uKegAqYqOQeymFliGWt7vqtts5Hy6d5jxZdnYCJ2sojCFKcqytpdOBfb3nJ1MKzIk2kUMZy0XHxh2WluQvWgUSrfhjPe7D51pvtv0Fmxd\\\\\\\/N95wV1jEJ3OKYWX3me5tYo0C6oJxrKBm3Q1VVOei66w5bB+h5TnkclkbqeGRoGkYw7RXgU+1utg8rqxSkQjF0sZqDyiVoLjVNG06zeYgokJggERG46xwKh2ub+0g+pVrFmICZg5AwtQ98i5eORSIGNXNSZv+jYL1xCOsGlHiRxWs0DNoaGG7DhehGD7xhWaTmUN2LAVChGwmwIEkXG9C5TviFUfLO7B2NNBLA3Iny9EF\\\\\\\/KO4JjcIymgqyV6aDiuRIqwTzsoInW6ZKSj1iA\\\\\\\/GTzsP342ixxeF46RprX5DQR\\\\\\\/6krfdMG2O+7lNvTm4nwPNg4cf5J3eng0TqTVkD3ivNiul1YJLOgLIX7M0wlZcoIPrKAFexentAHk\\\\\\\/DtILHRStZG6sm6EP1L7v+j6Jb6l00miKd9Ob+dJyPfJdsT8K8Z\\\\\\\/AOF\\\\\\\/KfFM7\\\\\\\/M+PwPOyHm79OVZS7cMuBr5FY+XtCmwyxhsYvIGLd2PZjHl850qXSu691dKUYjlsUr2Ht3fx+MYQa\\\\\\\/4X\\\\\\\/apm+iBJXmSqHJ3qiExKlxmINTW8V07ptdFSRo\\\\\\\/cGlts6YzBaCCvmjLj6exaNommhkP1tuwEIZfNyiG8oNceD4GViAWc+WYVf0aRM8B2UuEmLAzSdFx5aNsHN1kPW6PHPd7X7YeDjLp8npDZ1rSM0sV+LPAgdVKUweijz5vfJGDC54w2D5Z8L\\\\\\\/NoVpNdzrwwdXRULSxzflS2JjaB4g4O9qn8HpwkEWwKLG0OiT8t7yh\\\\\\\/9LGzDcLsEw5d4C\\\\\\\/ZFlKH0\\\\\\\/zjX8OANeIWlhJH4OLRuXSVsj8rMxkihi3SmzpOZf0BtI3GjZD0HNn2NjBH7zBNkpYIScleaxPPMED6juO\\\u0022,\\\u0022iv\\\u0022:\\\u0022agk7zQ5wKyN0cV6Rj2JiVw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227248609a4a6803c2d3319ef08f3190e13a48f96e9433aa8e19b9f22bb8763240\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Pii9XWM3BRcI\\\\\\\/kl1Psr1mFTn3lyId3aGC9Erd8mgsyGXds3rAbR7vjfuBRK+p+h54aT4j\\\\\\\/sSPn1d7VO3AcmdEYee7SzMEEguiR\\\\\\\/4ex3KI32cRnZMuBHoWPfj14qXlnTjCcVlwtMW6rE+CFM3YMCVZggg3CfowQtCq1Fi3gAhX3urin6LR0PD+PdOULHMi1wsJST7VvwajzRSRg5Ei7G1oR7XKQEMNQ2ExhuM8UA\\\\\\\/zfovGZXH0enuWe3vKmYR8U1BaXwnRW9rZa075zP5ZNBaepgJItZaBmKQqqzzAPfVqxnmcDry1+BXhsWi892jrMVddknONhObxchbjZ+C0bvYhKA2AksurEsuDqMq4Oa7AUxSTRsN2rM19S\\\\\\\/qAonjWzYrSdviKnKARBDxWn+D3ecLr1kUobaZmEzvbpJ1fKljfgVfK0f7atnCRG5NgZ45L\\\\\\\/b11Jt05spRnAzSm0mgioUArsTbxQL8ptfuUAD3pbAD1EcP21RlhbjyBnZfO5sfWXoSCgVV0s+WMyvSc6e2qgKYfL17Tp3EVy9AjY8BWks+Vy\\\\\\\/k4alWgyoJAPb3gPzpnHKGdr796FkHqc+0BFufCnciVX43vXXJWdDOOPhnT7WirHhf\\\\\\\/ZzOjqOdHIXSYStgVy1dPZ+ibFJhKIbQfup\\\\\\\/h1z1iwVbVhxe5sMkv\\\\\\\/B7gAzyclyHR6SBhl+73ebqqiL36nzbZOKdPFLTi2+TvUkKL74ngzwZIi1kVLp8tiH6dJt6rFOm7p0x+bEwwZMizmsWnqs41qe1yd1B\\\\\\\/z8sX0QHP4EV5Y7HW\\\\\\\/qCd1mjA3qZoYRVoDYbcyJjcMvmu\\\\\\\/fS+qGBi1vzPF9ZmeSq8cbIcfiv+0wvA4\\\\\\\/anC92QNVksoXjpv3phTOwn3Vs3skqceEYrvq3k6rGZHt5XDGwc2kJSQO1xrUtgYIrjdTaTIFzGRd5INFj1Yup6ZtaJGpuDjatc5uIZSY4NACECh2W\\\\\\\/jtuwiUvc7Z5liblaQbM\\\\\\\/eWE0YEIMZOOT57cgluygc+7R4qiZNjwLV1K2DrHpbQXy57\\\\\\\/ZfzjzDwvtJXVXxDBszG2FBeB75JM0Kal4PbnSxEAgrI\\\\\\\/Ir6eQk1rZUw0dhtSl0bdIxso1ceF0dMXGMiXKSwgdBKQG9s2oGIU3z3efE9AAIecvMmskiPWUfwzg5Pgh+DCcm6IR5jrmoY8GZoEYs5H27Xm0DPpqXgOBLwFjY0ngWAAAOr10GX9PoZUIDAKAb7AKaI7nivW9ERpwSPFEUbpiW8sK0Uv4nz2X\\\\\\\/lKS7dWNi24+ob+XoSFh3107PeJ1CoBPjdyMuf9hJL0GbgxQaIVMf+0E\\\\\\\/fOF+tkXmuzUnCvdD2CNE1Jn8vB1sIYGnFuFIp3jtCP5nLjhQvEX4wLauV3Nr\\\\\\\/A600chjXdHzoY9RxuAmUz+++lrtt1dkeqLxcLSLaZy4w4r5sCw+AquyiJXde9CG5fbrksag4S3g66Ehe3pM4B0KE\\\\\\\/nLL0HRepdxyGXxRQ3VI0Im5JEuBgd5DV2TR8Q2uCJMQPsYNGSdey35KzWhkB5k5p8ndRGubtyBxTwV5ZY\\\\\\\/97qrg71ru7LrmeLd\\\\\\\/rvHDnxxgjWFuYV+cOjQ0bVSV9KdsZlTnoRHzfvuzpZVR0+JVutsT0hWI9nS1XaXVFUZo+Bjxkwtmh3ka+ZAYXYKszMfHzcdOD85tgCRV3WdnWgvvMBfDxljmK02565gOrrVJqNe0Msh+TiuzPa2oXVFKXIr0b8YndkZpnMClRVPWTP0Lot+eRVqiC0WuAV4zR8D2CgBDVOmN9VbynMYoriLHT44QMKBgn\\\\\\\/v5iljDbcUDNIBJruK19gJBigVW6NAuHumCYq9PKAlNEpd2D5lV8wtpVXUN37qKgvKrgWZHQqNOotRLOUF1nBTVeksEKnzTse+2pLsOhA4MWHpNa1U3ejV50aGwexWLw3+2cnk7T9f7eRbl4D64iMzXih0M5+aFHorvt\\\\\\\/MVz8UkUHFnC5vtZfPvtbuE8ByZ+d2NmENzzHawBBSuZ7gqnq1xVIhslF7X7CqvwqjdKMOWavE4uIWrs0mKH4i58ueR0PFp\\\\\\\/3AMS2Ri2wLSbkrX4d4t7Zr9ksnTLEtmdcNlYwD+0F2eGGlkwK2ROZWk35v1DI90LXnYQp3X69ZtHtVTXT5PojMbMw\\\\\\\/OqifR9xjvgzIMSIg8T1SjkNkG\\\\\\\/vxbr65uUadEv4FzrxJuWrhEW3FhKQbuxs9bM8UhrmWZxX\\\\\\\/roF2kwyut5yvykEHBFSxWN03NjWKRdR13\\\\\\\/ehTmpmQBsN7uDEv4bJUYUmFvRppQf1043nn00TZieOXv0EUyoaTt2qIpfXw2oX0uSc+0YLDMifFZptkJDM11CWPRUtve1evgABDdxhL6noJbdbeV1jwjAQxbINKul7EdsMtBqIvZwBDvaJuTYCjPF1YQuj4xbpASjftjw\\\\\\\/MnfizWbl8QFuj6xY777fXRwa4Nf7ikWgLVxbsfE4ws3VG3wDGfW13b61fRikdk2Kcx69vhIVStieCYBDVxtzfwmbDzXpoScOXN73Dh\\\\\\\/YlVtpCBrJwxNRO9afKTMJDxOCSavWCLWBQ+bYuNNeph7OCJD\\\\\\\/5esVpkglNIN8z4S15j6IXRAv0LOlxUPDTNpWuvXM9oJInzGauWIjdCi6vK4Asyk\\\\\\\/lesaQKToUe\\\\\\\/W61kJ8OEZNVWXKaoaAtq2y5ojcYzcQ6zwDDha8GTLvIKXi\\\\\\\/SnBs4ULraVL9J5MnP3u\\\\\\\/GwGgJQ1Fk\\\\\\\/zuUhcOHKrgWKVmlfMvGURXVEXXx0doPnkn6b7WvE7edjjw2o4IwCc5HxFttRQYyDTW7V0JjkJZ6OSxaw6\\\\\\\/WpP9FKk\\\\\\\/lGgDboD0XP6xtLT\\\\\\\/PSvvB8bqdH\\\\\\\/Mq9NCilVmmIbCctvyM5QAM7NrOLievKgxRr0OZDMMz1IgGLS0DDeH\\\\\\\/6IKGbX2H7YHBISF\\\\\\\/VywsImI6e0MbxiOQvmuj1S7sZlXCEAkINFsmqjDOefFx+O8dDf3Avlr8+MTUvDjkqRObjJ99URXtbiajKRlMfBYZ3oSFOifNUPtZLER3BrMgEKgAIBavYSai5kTdzEH1Sul1OgccOagfCGOVBvfgsjI3Yd0H7Hy4vMzm\\\\\\\/m6yxVIdwnzVzFT\\\\\\\/DCCgy13o5f5C4BOJAQ5QXjZJlTV2PAHkqwppWNFi4UUfwQQnggRjgZxka\\\\\\\/jst8GDfHzz2m7IoIOM5M23nemMf6MlCpTlqM9HQGVV1bnr+fHN9XAczteqANCPAETs5O0g9D5Kzfbj8U5wR2wTuXTDkdeB1cfUv0Vjdk11W86tUkBgaBk9dDjNLMn+FnVhIEzxD\\\\\\\/87Ahzhycgyl3d2Bdvt8NrCosZzlp54WnqpPypBmGGkwqffj1hNd89xpcrYaRVP6gVWq580vX9wnPG5Emn1LO\\\\\\\/S8PC3QYq84JhwHVZ66fItKl9VWOghv09HpLnOOeXPIrcE6DbeuH8KHwc42NS\\\\\\\/otsxynLsoekIxgbz67s0o8rSc7xq9IExzl6pZgt4\\\\\\\/fctiZB2suYWIYHToByVBFP7vSUanYdWIfOdbMoI6ThNe3fGnlYgm5XCCO50VeOg2fJ6bvNiLs39NEOgAgWMUNsGFqi05DVVGFQks75V6Hl03bb1I8qETSQHts\\\\\\\/VbacXBrNszaY1dfSxdEofJXabAHZ3lNqwoBtsH6NPLIb3RGC2njWA0\\\\\\\/GVpkRXOEE1Auv3NNIUs5uU658HE+xGlAMOEf9mXx1xMkLsconoScsHnjydV3qSCfJDLKIPnqgU2TjwJXgqlTyt+GiAInT0RT8yClT4Uw1wpn1MusAbffFDkEdinZpRYhCtXpAOpnJtqsvgbEIISlTTxScGt\\\\\\\/rxo6ehhsCxrrPdDp+unc5MMec3M41vdc90BRK5BRLl9YrlhqnK4ielTNBsx1nWSTUgH9jCRdXkCsdr\\\\\\\/K2tyxJkhK\\\\\\\/0l6qSOA3KV22nVrF9ytqPZFzXvG1D+ntUrzeKVB78ztm2kwICqa9DD9orBnSUJ0+7UWR1b2LxAB0eKVaMBl15pgKRsf0JKFkwcNRGR8uYWlftwNyJuadb1Mxa67M5CH0SnmgoNRxWohQvo9pC4dIvegUYY1MU8UUgvToC6D2DHelNYlhRJRwrb73+XTHZTQc+xXznIsxWT4Ok526IS2gbVdzKPw4bGdDwGi5ifdHRaF6j\\\\\\\/Hj\\\\\\\/QY7M6EOhVNEuONUIgamdb4Uq3d2QX\\\\\\\/jSsVgrEM1ZozFi5eha3ckDrZyN\\\\\\\/YmG7MBVRIyGfeoNHPFnB10QErdTEmLXGccvnEptOtg6lY\\\\\\\/00fa0uLL+hxFZXDVPcCsEPwAFCTRXYm5urMeu3pWiW7wM\\\\\\\/2Sqi953eVe51kHxWyoNN\\\\\\\/PEkirkWjXOWI8w0UufCWG4IIw1yw9AVMIsxpMfFB5aUtERcCN0AMjyMb25k9AydNjJtu220mc4Z9FBvpjcOj9G5nmWxD2ktN47Cd\\\\\\\/H78qgUdS4UsjJ48bLFus4MVR5+Cz276qRhMaYKd\\\\\\\/GCtmgwfTNcpA4p8anpmMrohtk9+qQpvGinDgEc2IRc7WIBXkETWRFs7CT\\\\\\\/u4hIbJyj02pmd1CsX1SrTVhs85sHjOL+yoqwK85ulkkHk98Xprkss665WKKAxamUlfdHeXd59vh\\\\\\\/4IFd4JkxIwmMx5j8klcByIupnR\\\\\\\/SSpdma2xOedPEbOeocCNL15MyNkPmbDmhSwU5MJs6UC9GM7YllJbvp60WjGyIdt0NVDPijXBTYKaFmGeUnwuEB4K3xyyQWuwWe+prgmOv1GQsigR1nNiBh7iA5JbY\\\\\\\/HblQBowZP3ihMYIzP+uEmTJQjIqUv6b2zMb+yteEKYaFEzbYdFK4HDiOpXVO04Nky1BexcOpuVn\\\\\\\/II8fvwfx5NFiUXOt2\\\\\\\/ReOqWqzND3dp7vcw7t2SlwtMSDlZlqt020+6t8VuKCgC4Q\\\\\\\/3UOjB+5Thq9oH97Q3v369qLjZras0GuOyDsXWjxCBHwzrVkESiv5fLk22YKifljC898LN2WLPURn1\\\\\\\/sGm2pheqyMw2gnVUZnZorD8ie2f2gDLKat4YsX4heFY+H5utmP\\\\\\\/V1Xag8zcI7rEnv7\\\\\\\/v40FnWEnB5kd7S1tc5eabH7l1gs0FP5RDtodQSPEq7YZ69ilM2jAjC10l+ZRt5WCBNVtv6YgicPeWUYROaBwCQqMc0o44VNZeF4YtIALB66j6j1YI3cO8b+sj3Bst9PEVsUGoGAAdrTZGJAPsX40ktTBdgf+hG54\\\\\\\/AvTXw9AYChJV2v8+NOG\\\\\\\/AKo8f3iq2HFUk03HB0PmJTvVDkh1m\\\\\\\/ni8rpC9YcvD2QYeNquMNkJTYlNbGHcV3b8wB05fgYCbKg+i333MEXuq6OxPqNPnjrjtXM6JJZkTlhXx\\\\\\\/IUdjea5\\\\\\\/Zyu5mvXvVn9chfE9z66Y3XSpum3ggZdR0lPtHJzZjnyke4StGQML360CMnMJYf4O4F4njzTIiEZ2Gp+ksB3pzqI\\\\\\\/Q02d7qzjojERMDIhJTrH7CEJ7TXNieBkAKi7hIo5+WeoAJaPFd1eCxFQSqXlPuVv7vD68FaFOt\\\\\\\/bVyT6pLsrtvOmpfaLrNUgg4DtBM7A4LnEdgFueu4al6uSurKhqVUSvwDGAW3Tc3OAPOjvtzyEsQxpuCZcUSuvfJY7+pI9b96n5igTx9eBlusbILszMQ7j37SexszdtZsXc\\\\\\\/h8TvQZJi+FYNifYX4\\\\\\\/QOCtbEbT292GD\\\\\\\/V5xsifJ8xnV3D3hm2w5yIpovzNVhHpq\\\\\\\/FNUrNSYl0+ZKrU1QCEo\\\\\\\/UQBVaB\\\\\\\/x0hBlpW5MZ9wSGc9Lu5W1rIjwt3eZdv3a5CR+8mf0crwXHtcHk2HRth+FTCCo7dBIeJ+TLbqglAKzYI9Ff3jjdfi+S0GUZYHon6S4LGmDLV6H18cLUuYMRyXhAa3+7\\\\\\\/X9Tix\\\\\\\/342V0qvEoYEGdy4ySLmiOcYIKxblWKYVx58U057IUrVlAsMbZN2hT4WpjFMPbjdkmNjMjQw7eLm6gHjCjGmvLNTz4sUtRSOEqXwmXDPHNRm8V8Kg06ZMQrAviF97KbwbfkcWQ8z2DcBpmPEEgZfAojetnf9Lvnh04uUY6PnjvBJ7HAliKXm5A7WkOvRxzSQOBzP2jZ33nDZCMtF1NixKWZNPd4fRhvJukplh3rDiTEsfkuXsB2fD5dKQgTQK7DWFmSH6OQmsxu\\\\\\\/sth5PuUBq5qsiB3lhY9Uc6wGeAfMplj4dAvM36H9mKwninsjDeWf\\\\\\\/yNLj0gJytZAtV4YofSx7F5xadQx+ulh3GOXnsOjycMvUojzhlmUZi\\\\\\\/6qtUheuOqdVFRh1n2xmAVeyJo12xVjPK9fg2ghxSG8fR4qOOOhKQvaWzsWyFBoZGTYgf9ZcL\\\\\\\/cUVkuT7iDfVYWMWWfHjIuDncERjXXbY6MKTktL8fOK2Dy7UJxnV+uR7\\\\\\\/TNq9om8OD2qOZm4wNdTpAhsUpkNpIZbxp7Z9Si9IYMwsPaIYs\\\\\\\/YwB6dXwgdTJV2kklzfBN8CkBslEpkTpdsEUi6ESBHzaCEL2KCDB9jVhLB34G+gSA9mwgh8P2DIDVvtnfXFOZd3+FQzdcXBvEq5RoIfsiP2dTe3sVjyGeqY1ruMZTWrWUhoh5HQfI3hRRMdMQ+d3LymNZvF7t40YlN9YWRrJXgEVs3qrXy01wyWxIvmwd9oIaxzHlPQS+fAgZ8AiEiwlEhZIQPjPBapCxaqN9vgLr29zahhOPn8CYBVLOfBHBxYywW+nOHk\\\\\\\/aEnsSeRDlY6HeAk\\\\\\\/s4C2f\\\\\\\/W0nmKObNiG9TwfDjDdgf\\\\\\\/nC6GnJ0+u9c\\\\\\\/6HNKocL\\\\\\\/Emo0x4CJgyuLjcv+HtOjogQOGBrkZDdULXR4xnQ5IamvLxgmkM5px2L60FEq00v5p8xcy3jC2auk10WDDQWiKw32zSpVs03ScJkJp0kOSIn6gmM0A5GHyROxRP+zk6zOb+L\\\\\\\/lj1gzxf2kVB5tG0K+Jixv3SY0aN8rqrc3fY2jNWio+pWVQCdHGMTE3FE1md6ksqzgtm6vhmUuqXwKhQIBajqKpB4jm9GzIMDPGotZrpZfAJJS2WUgokOb2Sg1tsEWUL+OVC1+\\\\\\\/3\\\\\\\/7I4SAnopqGiL13V3xCmJJ46BsFgiQ0j60PbgSr7NyST1XBSgGtAD4ZAZf\\\\\\\/Ci5a26Bt3i8HkfONZ8DfwK5FcHPOfEkOwnVJOWCIwjVFqNz5+TqfABnbDDrdbYlKhnzjjF56ZEg6tBgB0kFy9pVjUkH61YFK7zb2XGYFJ2BIYPI5bbdHZeef8eGrBWPlk7xMUsxsWGhee4\\\\\\\/oXMcW2VUyCUetUo9Z4j+RD\\\\\\\/TftUSbeVJ6DoGKulQUwVnfGQyfdyukY4EHYJNMT1ViaoyKzsGjwVe\\\\\\\/C0Kth7+Ee5vQdLTotpuD2L1etUw+UwS8OXrHKqFEArO28yOuhj8qYkaqcdKD\\\\\\\/AL87FjVc7gs3bTdtMMw1ZUByqbnBUazJyMi+jWA5i4ouSXLN+LvmvtDs2QHOgxAsoJSwR9zXdalGyt5n5PKFzskJL8u8HnRV6vseDY7VfRcZZQ9A6q4ejWP0QtchxIu0sNPVvJ8F2NUAEJR+GxLysYxBJ4LmyXzYC99RK+DNPNfyR9zG\\\\\\\/NTYnc\\\\\\\/fHyahXe19EviPerSKXd29kP6OZKuWmHYQqP0lTN\\\\\\\/racUgi4kMBM0xTG\\\\\\\/IxfRhFywBKzdBOCFK1Ap99ak2n0YZz8\\\\\\\/hA1MIBZNgkCeZbDvc\\\\\\\/tD\\\\\\\/guW94XsD9wBi02Odd8r3ZD2nkyHHygySl15etduyJZEMp7X2dIz7u1jcUZQvamYdze6ZojEkHyc\\\\\\\/kuEn0c9DQZ3rb2Z1e1jjUyQPt4NWb43gEpN8XXSMZ63Hr4+cGcnNCNN247ViE7peqWjtZLFzRPtRf42dQFbn5e+1Tc9tM\\\\\\\/ibfs791oI29kmK6c88FEe9dTTUiptkm0f8KDJdi8HvCJGh++22KwCkymryfIFXx42bSH8T5M0OFpZ4gzZduxtJ\\\\\\\/pOR15PEvvuM4MOmZYnY8OmBYUJI\\\\\\\/kr9RYu4nuXXsu\\\\\\\/6wxYYH67ljOfh8bUQPCG3BOY\\\\\\\/Fz1oRgVcOp5pWgpQZgcJSGg+9RRnhrS2nloE0uoi\\\\\\\/XjYiqfzHR5V5+eamMLsVbJ+i307sSYJaB6pX+Gzdw7wuA3+i3Kd7FMNq6li+JiQMUXkQGqXiZWaYzvlzTAtd+5qwzu1hISIuMGnwALGP3X\\\\\\\/tGycli89dVLJOhcZP8B5xIWASDcFzx\\\\\\\/xgQs+3Ur\\\\\\\/Dm3dHO+94N3gh74vcKOf0QNACkS\\\\\\\/fHWMx1BtqBtuxzJDszIe6gXJ2uiCbGO+qGPEKYYougMNrltWJo42SIgReGtrEQDTRRl0El6CkQqMpXmpYFq\\\\\\\/BFlLhfLy9fhHfuEfG2YEcIHYW31ZErkOHlGJIOhTubGPn4DfNT8GcJ37+skHMe8PI0qyUuR0N9OQ\\\\\\\/hOdCjhFQxuPY8JrAOEOGGktRb1V\\\\\\\/Wv6apUpRP6\\\\\\\/wJH2fqXUGzxUi4bVFK2duTxxVqEc1k9zAuphATfXNky+Fc8yf18iAJjxuJ+\\\\\\\/iSoLu7mzZsqAwgj2GoIfIg6ZHrgbCIcxEj1FDYrXNlnoTPXKNa8bO\\\\\\\/LH9Rkf6bU5r755ar26hqUczsp2DB\\\\\\\/RNz0GCu\\\\\\\/Vm+TVDEPrYOG0Bf62DemO0tPFjwA6scoMMzBozFBs+iFjkkpglwK+WOuCEV4e7SsDF91vTHPjt1BFm2vD6mswQ8XaAJ3ydZjiWeGcH10lNpCYILmrXppycdMboC3Fdmxsn8+FOjimGEhF9RjNZGWqyvlkexQJCuESR1LRZAccBOrFT4w7MaXKIAmOpAYydzbTKimU3qX\\\\\\\/F4PAy9NXuQ5c3M5YN\\\\\\\/SkDTz+n\\\\\\\/E4ymTFEhSA\\\\\\\/XD0Y\\\\\\\/HqS0jGs5rCMtFM1tl1bNT1l60NujEKjPBtibIG81hjoGv6kHIaK0TDLdR0oAsMpsf3UYQ8z1K3tz\\\\\\\/fxRMXBlhXJqoGMu0QcbR\\\\\\\/zZVctigfhm5rW4ZoeftPr\\\\\\\/1LSkoJJha5MFnmijE1ioelTvH+nFXW7gVDdRT1ZQVhiukNWKqvN31cu8+RXohGO9iUhuZTEMmNdHQbtw6Tvz+QvHB6db9qWFdXIQWb0S0wlsuZsC\\\\\\\/adsI8hoSwAncE\\\\\\\/a47yTL5xW5vYvGu75gIdY8rmQ8DPZlwbMk\\\\\\\/XuNYXmIcK37IHX\\\\\\\/4GEMBNl8D0oPec1c7nTwuy3icKhTdkw7KmO37OWYS++SZ9ntzD0MF+1fN0mx+FYANOMfszoDfQJIqT+Ik6kiMd6px5xvyZ6edIgYBIygZbESAoWcQWUm0031M7pRoTCwFxtoOYQp8rqwoc96qzac+GHy0\\\\\\\/07PWQOp5V1nvAjnH6yBAbpBGFHKnop7IBwjFwSsMNcyM\\\\\\\/TTYhXHc+rAPmQhcTQL0KAcZzGtm38s0IfHfzUeGXnu5NwsLEKil7w1HXf2TC6A0hTvCFM4ZXbv6sU5JYvReiCorUCuk3HXJSj3kAICZQEX\\\\\\\/yAeGlK3o15hgUO3rE2vEjQFP4yNXdh+GuLkq8bkLIGt0Oqwqgn5+QJ3yTXn9ZjBi3nZ06NpIYRlv\\\\\\\/2a+eZXN2s8WR28rEK0nf3ouIuOoMqHFtMdRW\\\\\\\/BU0eUsJMi42iKKfhyURrot2ghP1FBv2KsgtgH+3rv8\\\\\\\/oQWzI\\\\\\\/wcTFxfD9C8NHXXV20S6wEwN982yqKcy\\\\\\\/E4072L5JWEJu\\\\\\\/s2P8U3rrJsnhA7s5H\\\\\\\/F23k0fNTNUD1G7+HXNG+7kcMEZO1gVSCVQl0Hx2h6e2C\\\\\\\/vg8EKlqChrT+Gs8\\\\\\\/08WpG\\\\\\\/mYVInixVJ8qolJHO3dZJYAYwREyNGv4Z534OX5zI8n2buMuA8\\\\\\\/kA5TSNU6PqPR4YkuGGL1jf\\\\\\\/zy6lByRXjXkhQTTBmxwnBhys7pLc05KOD81Q\\\\\\\/TeifyihowHiDX3SiSVVHAyDq7BtTPdNwR0+EoLbvsjakNfAkeleBMf58C2R3tvlRMovPlMQ6ecSdAcWNgTVUeahx9xIWIFkHiRPsfAOObMFQMgtXMk7xgomEseSHPeGt+u50R0a5NaAdOkT87BtpsnIQQGC6jvI0M5yYy\\\\\\\/loB4aRucUCklywMt86ZxEeRFfAFFD15wix7vpQkzmSGu5UPe1ir0IpspkzJzYuhZr4bdo1oVLqdK1iuNBQLjDFeNeTS5NPnQMi3CwNAvkXcIN8DbMj0hmUgRSnpU9YRXJnN6ny17JcuGL2BK7MFw9FrdyKWPYrbZDfhB6qgrTlVBy59jzZTEwn1PY+bYcIduakGGYuKH\\\\\\\/JmbaG1gkPJp0\\\\\\\/IvPgDf7pD1y5xP0+nqSNwELlTsH56yuIAX2y6RF9up6CnOks4BtZVLyK2yPPj+GEi34lojl8pXxwn+9bYyigj02q5fx0nsYqsipESe4cg0uKgqmSA2uNGCNe9lj40Dx\\\\\\\/bf1tTspqbfGBELHNvrzK0IYqRm4uDxdsa+n0DgcmZ\\\\\\\/UgreTVTGXIE5i6crtoRqDRUXx3j4UDSW8FB2gd+P\\\\\\\/pqLBBXR+pQHO3VEAKgt7aHFWh7d4IHqzLrPVWFMiiQSxJ83HNwkjzwj3FAT1yXAq3TtazAyqDbtOQBS7Z2jyjkURwDqhb5QcBYJbJmPflmPXMlt\\\\\\\/9ebh3DRgDkaLFqguSpklB\\\\\\\/sKAfLilRlAVaFIrP9My0+SRsAEyyto8AijjJJtpsO1DaIiDsraFfYK1bZASSzJA08wwiFn1aJWVZWSSbNVODoiiw56GBaovUFxzo76WjwLT8XrgHjn6gkD42b2c4H1W3ppl9zW9FEbc2VRVOP02L5ggXdL\\\\\\\/7sNAtWXb3zzwucHIWVoOOzKGxsIt+Ict001gbOpD9W9M7uBd\\\\\\\/dQWyt6qlTEmtK7a72axFWehIvJ5lzEavjAjcvZ2tIX+7Eb7hwVU3KEO37Pvwn5i4Wvy1dA6pUq2HbLSSWvQrngry\\\\\\\/t7R7pBdNqNi4aQOyEr34S1NB4CDlSu+1JHWhuz5Q4ZGH8Nnm6uV3hil98C3a\\\\\\\/ilhZ9L9DhuXLwJ5GKoniv46E1QfxjuOoMGA2fRB5lIsNrA0nD5H4ZEh\\\\\\\/O\\\\\\\/rxiov8Y4qEB3HOMnZuaR3kapfnAe\\\\\\\/zgVKb8ubFbwkNxePmCI\\\\\\\/5r5uhqqVJjrk4p68y2fQr3k\\\\\\\/w7DlcFYdjerM66gzosWyv4AkhPhp1EhZYnYTKta0mPWurSvOUQ3PB+Ljz4m\\\\\\\/vVw\\\\\\\/9Vnj5NhHoK5tzkXe0bbcV2xwy9QnHjURQZevvIToociosPGZmWGwh46YRGgBDyyEp6ELRskrJAZwZ4GYN2MLl2hV8eeexCWeKui3RsWnRWiPZoYOXve6FuxKYS1d\\\\\\\/oSAdXD4+8Oa6ADtQK3ZmiB\\\\\\\/6hUdu8KOyA+paRqSNbRhcVIwHFYwbGqygjLk5fdeU66ZP6k\\\\\\\/kz\\\\\\\/OroQVDwE6fKac8iBGGLWOvpcXrxmxcmUQRYTgd9BM36\\\\\\\/cxcuuXy3vSUcoWFiG7ZuK6Ax2QPnj3wRijb435VkBR+KlTLlwkQvrccg+wvrnnESR4ILmOk1eiGTDXwzK+uR1HOUT2a8guo2tEhCHxAKO\\\\\\\/L9X6VduaHROJXyBdWgup2z4TIi9ZocR0AzxLx7eU4blUfvnORWIADx\\\\\\\/CHzck8igBwjDNDLvnJCZRa2rcKUGTe8JQcGCqLLItWeSUfVLNe1rJcbA1+Fu+uYIBpBsTn3KR9YtqunbcUV7bSAKAkxpG\\\\\\\/9986cjzeneiPHs8wWGV8hJnT7Xh3DBThtC07DrAkdBz8thHtbGVcoFOXDGDyTHEJj3e3edytiqF4J0rM1nrL9fw8yvpHYP4gkXLWx04qK3XLH5rShzTtA\\\\\\\/CLvF6j7B0SF0K04rAHnhpOS2ckykQTDNwWitOC2kpR+mA8+lY0T1xJW93sgzUst5OGI4M1lfbn5sRdBRZSf5yVjxV1TeLAsykurz\\\\\\\/Rojn5Rj9nrWTe4MDodqDnqYrxxGFrl3u4hfwmWTzbOz2d0KEylQhPBxSKC\\\\\\\/SOIbtIY+yVUy0IN+lcPRBvMoWTy\\\\\\\/+bA8W+58bNjD3wfBL9vQE0FnmUnisEIw\\\\\\\/kKAbtn2Lbs\\\\\\\/ga0jbSb+zjELucvU4EfvObJi7qWaKG0I8q2qmfGH+stuucTzVbXzuH5vYndjv7xGm8OD1hQ2O2ioCxCWCK0gO0okr4nhFk5cx7V8yzbbCLYi3JMJ3p0sg8ZdekvngolPMSWyJ9iTBe5Wk86k12GQmAwZ87iUK1CGjyMNmJYwJzbqDebQkKIfkGMGL3e9CI15bnsltPioE7oKwKWYmOZfcNWYZz\\\\\\\/sBYiESWhV8RzvqOI6brJmzBQfiI5\\\\\\\/wN\\\\\\\/lCh+lUfYDJWsLOJJB1weGuthDg\\\\\\\/lF65MOmrEq\\\\\\\/fuUs4WQe6KR\\\\\\\/snXxmZdwr246lTk53LVBQvv7QW+Qahd7Vhq5e2iADpT36++YI6S2hOuNuGFpng4FKC3Z5OWeoKUX+lYM6SOkot+dVbfQh5uEAdrlp+\\\\\\\/n8La1vU9pghCzy5UPj9PBK8WUaFXciz3KgH7c9v\\\\\\\/HQ4HGCWQure8NcY3p6RDKJnF8LUayOdYxNVYClreL8wvYA2HZTodENbBHyAv9DtRpt1BrPmIZw+ia014BAJz0iFyZskuPG\\\\\\\/mBHFSt98PHAWTm9hLBpNKm8kjdtHM\\\\\\\/q+M+0NXvEt3p5EUHzkvgMiDQ+Vj3HNI5uRzCSqepia33gfLHWZzkeERDsb3AHo\\\\\\\/9SVnuva11MYDuY5z58rSoeNUo2kepzmBu7yNHtSH9ddX1LbE\\\\\\\/lsjypPtqUy+6ucuM7Df2oUrqCxigmbIuwq9E\\\\\\\/OWI+1IghOgif0Mv1JJdiJepbxkQ0BL566VAeTq+rVSCu7fGMsJTWX6o913nEuy2hODi+faNFCCU4yeYuZ8RDhALZRxoQ0XSHSRULlTA9eTwEswKALU5oj3e2IspdPQQ5FOouTShJhRnIUISVFcRmmBnhp7AzU2GIBURdUpxvLIq4e0lvPAl57UP5FMFvuutVh3+oR04Aonrab4aok5BXITuhgQrabtAhnlQKG6t2Eyj1eLoeJ4Arqir0RbV0WPCunStlKbLhhI5V8WDZxe5WVGU87cNxCmKBgSa95L1ViolYYNfN8tTQ1AUoaeXfUWJU3W2YcK8FZ9mSTQOXDshMm2k5pcSmR7CrVhsygJuK5zK\\\\\\\/3DLF2thGGk58ar51Lm8wKLy\\\\\\\/GiJurZTlOVT\\\\\\\/m0y50pOf8HtReuYEq4AyDvnpr5WnIj88Zkdw2v0iPPYYy5xLuRvYZm57Wrow1dcf7DAHnXyCJ\\\\\\\/837rxibOG\\\\\\\/FZtaFZF8ChjlJFYkm59Z3uCCsC79DrYRvnuqxCjFGwfW0N4Y\\\\\\\/J+jNe3iBhtAmfs8BYt4AeiXmDcHxvlaOkEFFHqcRz0\\\\\\\/XUThAM7HzGn85sb2P6soe5z\\\\\\\/i53UnkLYnxfyZIFzMmtHVeT9wf3xTjMOMDSDWsKc2k8Ri6sKry\\\\\\\/Eyv2C3vrOabQhLoZqwMPJC8o+PFpYvqfBpPK1rXc8i5yJ8hN+ok4omYQsmGoD2b0JDu4YYZdqIcNJ0nJnb9MKzqV0MObcqCF6OSDKNyKXrBEScBD8MbTYv1eUkWLjanf8AtCMha8OAtNe4M2SkMkdeMriPp3hplTdgHk49\\\\\\\/D+yqH2nwEzRhl024TpeKg3QlJhBu0KlSxxNxPvvFpHyhPXKweoTERiyzqFmc5L\\\\\\\/SSZdUvcis6vLUHMuDKoYQRW7q6E691csb\\\\\\\/4GJqpCNDD\\\\\\\/44XQDs66VlxkIOGiXNex9NJ2T0MYFwUDpfu63cb4Xf37UAxNz46XGQoUGruRogNfo8KiFILdfDQOc5CSCXHCwF9i\\\\\\\/gBpqdusENRVXa4foD1JAzvgtH0dYiFW8CmXvdeJoVkFYmmchfkVUwIqaaEiMwXhG9vrDK+pR1bQyV9XswPAs47XSClXeAEdZ3+pQ8x0HpnpfMi2KZhWO1eP3LwzyscddeAS52tSCEOFW5JF8Kf9OS+JXPu3hKxMgN2DcyipbBUppY5fjDyN8+2n5t9OYB3sMR+RDdomQbMeUrDKk0IC3g3o6IoEhepKheQgJaW68BOyS0SiSWvw7riSF3Yp0B2cyWJjotB+r+ZyhJmg9TGGzSRgn34kbECSvkMXnPCeFtQlhw4lRh2sSnh0qCKPdLMnC82Ywuchs+wL+won4mdAqp9Wzc+qy26GFTMWv5xO23VOwe8z+B9IomHeILueQNbVCaungV1vu+hyh4iVq79E6oGpnc3k222pwofv+zHq4YcYBtIniHBmkeTIQKEhKvQXYF1nG6RBKKSlXMiDVFzvRd58VIiD7cFWf0\\\\\\\/hMT8o26rqHZpbDdNoQEveKW9ZZ5VAOavgbWRXOz1FRVwdZKWajYRGgGNMhaIr18uSghk5N9BPrE3+WXq96t9ylnYG2loleeq2FifNaVMMh8qoF\\\\\\\/RmC7WIYPwkicYfHugO0JBx\\\\\\\/wIvDLEkMpVZLtbxWWcjBAfddkZY65THqj24UN3SJi1K\\\\\\\/AaFu8wayNslcZjFKOe+mUFG6gIsUj7VseleApw57XnKWa7XxtmLraprUbNVOmPOae51qlPvM1EYyz\\\\\\\/qUypt7b7T4MnpA78FsC1whgVFYY\\\\\\\/ZgzKK9L123YY+O10Wc+eNEZzp6fPfcsjzMj6jLwGYykOkORoV\\\\\\\/EA4Os6o5cXv4X6vybru6CJFQST30r2EIZ2DOqR9X0jgRG9zC4ZA2bvPl9w8ot4Dh7+3rxA0+LGyjZKg1aCDR3yW6WX1YF5ejSXBV0jfeRWhGqB+Csim+1puCgLdNe0i9zDJNUDNDfLZFrOmREFYa6moK3aNEfwUr0Nrg4W1WNpZDf3XghtWAGz1HtEZWbE\\\\\\\/aYG\\\\\\\/PWSuQdTThaI+jBJp7TqH9aqoatL+Px\\\\\\\/hRlhjZsLFg3QFMCDMWVKPYMfdh7xnlRtNjVC9xL6hK05KjQ5mh7SjVfrJ0THH9B2gsTqfa64j0d5g2lo6y09EjVNA0z1SFMDplZAW5Y8BilXbvoXeFRFN8qnwcK2L5\\\\\\\/+f0fXqYhmVe\\\\\\\/wBuB67\\\\\\\/vJzS2slcqoIBzPvApxHDI2m0B616Acn2Mpr6xgZWZuPW0VdZXaj5j9t+FrCvLz6KhMcFPlk6nmxNQJ+AYAJtJ42WvgmUgaawwgIWJuiNMOy5yRcAjaLtk12noIWGzfq8A9fHkhkLyYi7HcJReE7E8SA5DlBLrQ4lfcPqBKZEWlb3na434GtYq+mIvD5SZhpHJApOdlkcfl9kKgbDY5tf4\\\\\\\/EV3ucbYntsQeRhYZzfw4D2jUclwtYYHbxSO4fiMSqsp6ZbEaDha9zjn5mjAO3uLXdhsSRFeE91TkuWoM\\\\\\\/qFQEuyFV4K6RpYzqcTbtJzu\\\\\\\/JOdYsuW6s7bAWl7oHznt5jRerUNKSYpXi+OdPEdBidLQt3192HmC\\\\\\\/YjFv3WwHkSDSkYPL03pDEXlxP5WdUz\\\\\\\/nKeII9HTXQLdvtNG7MCGtqs+yYgtE7kksexPn9R1PflJ3weyBIb6sPi+RUDkeQU\\\\\\\/gvAnTd67e7QOISELlqqjzfgT708\\\\\\\/1DNuhfoCAKtsB7I+H51H6vGB+mDfYQVJtuhQBcnZ0Qd0TxdBSQD5\\\\\\\/PFL9xEz0RAtEtGwORByJj+ca3x3jw\\\\\\\/wJit8dmchBMT7fOAjAyv1EexfPf4cuNNru7N+Ub1CfXPG+DNWaY78nHJCs+4OVxbhnP\\\\\\\/ZRh6v\\\\\\\/4y5bMHMDFoOtluEwfxCb3onCz4UWKqFX1782QMzGV7L6CDhkjwH5H+XT2wK+KvJOmHxANxEH6XSReODuRmzQKHgig11awIOe3F+l\\\\\\\/YoiSXeYMQBj\\\\\\\/ElmQDwTeFUy+6gi2Wzoh1y0p5WX5gKe+ABOirwrLNNUhsBINmEhG2VqsKN4h0VpPlpnfZNRjIrmAVPZVA6NHv7uJwdLtJuUFG+56a61Vmz66CLQrr5VaKt6QosvEoy9P+n+MYxsHk4hdX6NS2eYHyDN5gkeh09lZHB516Eu5SzGhpXcgw4mK9rdY4JmdxYbPzuiCCvxiuU0FZXqXVcHwB0JpjlrEJAfEw+k3aFTDFV5LH9MTXHQX\\\\\\\/P8r1wn9anL9Xccec0ZlhbgITx2pSCRpNttXgS8ftjW1XxskfiNAA1tP0VhEAsB9DsxvTpi77uhy1haQO01IueWqTQVnpdvhRDKGDIs8XDH\\\\\\\/8PiR8RuX+sPR0EgcNDtTuc5n0YYmTKvN8U0dFzb5A47qRMeyP5jicciq00XSDIGOw5eQma8cSiVbBCzN+tWXlGIWcw0EkMTVFKCc0pZeXsU1LJyKgU\\\\\\\/Rt+DQxIeQVeNdVxXpqUIJZEK713P03OazwwVqxcafaFSGpCtSZ3GEN7\\\\\\\/OEK7mveWBmHjaFpBFyzrOJ9fXP0onfX0yg96WijzBFQTCFJoGMirR0925P0pjTgdZppT5zx4a76b3tIvRjF2DmkSJoJdvWM5v1+lV8cuKE5ScHopj14gnLBcyIotbPY6aGl1GFMyiJiSxWlLzk4KgcMkiy67HA16PUc+K4LVeeDmqIivHafj5UT3jQeJMaA7UUTL\\\\\\\/iQldi+uchtOx9NCSbh2diB0JvdWVLCvBcfxrGhoIZkvsxjGhyS23BKbo\\\\\\\/MZoD1JWVWUXSxvk5C2nvhMLToWAbzMSoMIGNeBkyR4wGJa34mG3XEoLc45f77YRE0Ajf+uA+LwIQQatJioKTk4Y3wwdpcfs3xBKWL8PMy7TDAGMR9VtxVBUWeo32f4TKGQQcOAEG94ZIqF4JOCBSbiHhDxEXeC+HzOMR9vqIye847KRPC0EAvxs+1yAeR0IStw0cMM76crOiqezmtUo4A2iky7qpTpaYczt4rQu1egBx2z8maXV68zggIQkB4\\\\\\\/8dh57OBt2kakMKYJHENlVjZ4H+txnKnIoqVgLq0CGiEfHMbjiLWh9FmwrNV2sDrDgRoOfjpxNjQc3Lpd7n\\\\\\\/5cVUnpXWQHYWFnrTiJ7nC\\\\\\\/I1lAoX00elYZHX6YzjCmln++VGBk0GLznsIgx96tyPMKtd6XjFQGZHghD1KN1BpBYDl+uayR4ZCZ4BExFrgiiC6CsjkUIeQUKJTZqgn0a8J+0Sgl\\\\\\\/vJudXi4dSIVIlCZLfkbINUe4stiCC7De\\\\\\\/w+btnfmriqxiexaLFR7sSugZ1oS\\\\\\\/Fw9ECEl34HKPBfFXAlFpQBQ\\\\\\\/qmVzAwQD6ZVb\\\\\\\/ukrojDJkoahujnQkMecIgJOWBvemxbYfoXkcFxT98X12leUO081S8XAR0sEKrb8iB3TQusBvECAAuhtolnWO26\\\\\\\/cE5q5FgWtLe1avxQb7A0qy1KO51WmCNvdhc5ApmSPyBtHbMJjnlJXHWHsGAsS6I7jFAls35Bj77kceBcRNgzWWF2iSajokoJZ29z1JlT2twRcstxmSnI5tJVacNhTXwe+bjoXOFnOt6mUhJ97sEhDswdkoqGvAj9wpWha7CE4fW8HItQ0S6YK6CYaXNIgHdlK1lQtcMhYMNao\\\\\\\/tJkpZ+4nAzh+rOiKKAjQWX9FAtQUdjGcW6TGnUxwHS39VXpoE7SYtOz1Rf9+5\\\\\\\/L8vRaMAfu\\\\\\\/P3fdxmH6tfNgsSIKZVOIOXd7EbjZBtG1G130ftyNnygrwoFhWyuJVHDpUmqVgE+4UkdnpS0lDktR5nFK6pfUtyPXtxf\\\\\\\/yoMWcMLcn1jOY+B0QYL4cGVCp3lxZ9qCSRyPRAgtubeKa7Ytjd+g+DZVHx3ThnOBNudhk+Rz6Fp4WtVutitAeOvAf9BWIujnmWNcWE8j6aZ2exinZQJU9Ri4Ff1wTQ8AV0kf2vS7cfmKFtxikXFr\\\\\\\/W9Ze+3tQiRgGbz1V2c26txYQMuztNzOt6uVvFGZJlvUIbaYjT6EacmOf5jP3U2eOkyQc5EfNMspP42wvPWZ6AKIYp1wlz7RuLFo5KD\\\\\\\/gBQpZGRDpqzi2a+fkupdaqom5+KFimEwKfTVMvcXyyjzpyhFuFjWfvQ9\\\\\\\/qEeHXTkQPaEG5OLeSEcQ0Z39NxT\\\\\\\/o5SZijcPqJGzLhgq3G4HjNqPrAnGp+GjI1EZbHXYWMsJS4FULTtYdh576dDF5yvdpRs4LpECjxnZNPhn4mirQvr1nNfivUa+zLF4XD\\\\\\\/1m\\\\\\\/YERl67UUrAw6js20eE2KzSEWUXZr9cSYjNbMfqevIw0JqT8qTJ56ggUCBkEo8kdRxk8PnX+2nBhlM7ap04cfeWzRhG9CX7qd4gHFnv3+nlcvYjWyGZut3S1yS5PqZMP9gzLG9fpczWV2WxHKD0JdNrI9b9hE7R1x1Cn9iSzFQW4HIwliCLmbCMp+5TC0YznJdgqydmj1FEBr5L2cQZrWSgR\\\\\\\/Yen5HXCN3+ohwrs\\\\\\\/72H2shZq\\\\\\\/ipQMH\\\\\\\/3ObsCcTHSING6BryafZAsD\\\\\\\/F+TuUasAfUu2FjNIrZQveHA6emDaSt+3Pi+XmA85YF7wXfQpQCwqVk4fTQnZebJ2g17zNZUhjn\\\\\\\/B+UfcS3XshoeN5FIxXaJHjmtzrSSc2XvuIpZHDVjRH8sMpgN94AT8IRwFy5Q4nVmtlcleaM32H7WMPwyBC\\\\\\\/LrbUbrsE4ktwYYbqO8xRj7VckHV54ItjZJx0ibXn9N9mURHZEwRDDcX4KqJ4FvNTd3G+izeRV+kWx4CYDWfDnsccWdTJ74oXxPTKRDqhD8Wx4sh+QPuQQCidCZabT\\\\\\\/io3oiT74NW8YKLROlDzBW8EeFPt7QWGulkcyjCBO2Owo9XVvosqngUDyf6tXwCcMXTUI+tylOyEMfg0oBjUtVeeLZtfS\\\\\\\/Kz+OKmdv89P9P9Fuc3gOh2vvJ3zm1AzU97WycD+4L3qI\\\\\\\/l2NW08NIJgrMdMt8X+K1zzdlspdK49Rq7rWUpQQG9+NO+WJ7x4n8yhdbI7XIjrDgbjpjzw6oCRhScksJi3flp4\\\\\\\/eBmMErwSUdxQEXG5JvlJAq1NjCLQ7i9h2y2FOdq9z7dFi9wc7zx\\\\\\\/\\\\\\\/DVZLF+pX7P2NQrb8uYB4KzJzC5DJvIghVdRMyaALaYxyiDArH\\\\\\\/s4uDdQZx8MS45kdAlqvfK6+OHg4djHRLg709V5XtZ5Bsr+wq6Zs49BZIRvMKhm0AXnQfHfh+z4aSdtECox7BRhq5Zg35gyKEgtDyEqsk6uFs6kiSfRKfiKJZ\\\\\\\/7mtExx7cwUhltqW6BidRg6qKPCetI1JAc7yy9Gs09qlNpUv0uvNcU2SyVx7kE6DIrC\\\\\\\/40V644ULz30GdlHSSA6fjYf36Ym8MjaCK0p0vf6ylafqhqYoxEmOmritJVTaMbDWRLt3ELfDQ+YPxbIs0Puw6Iz7bglp83DXEKCrB2YJ8ILFS3Uw9zBZm2tXd+VBrouuz5q\\\\\\\/9dgfFHMyW0\\\\\\\/AA9ZmFJ9Qi7WNdiAb3GA3Bj9ogI+hsaKqxMR0x9lpfnTlGMPAXEINU7XwababuunXsvHeA4HeljNuyLyDXKB5p1e8i7273tEsrlLzVjgRQTD3l3Rls2WN3oHVspMbkVTGXdT6bY\\\\\\\/uJEWdLprgyZ4qprKFXK1XLyOwHtQ1UkO4Q23lameLqSE6x2yJPpfMke+U9ZejZzGuAkE22Zu2Are4dExdqxCJzlWkssfntt6ZCd0xW0WEKI32cv93cPNiq1J\\\\\\\/3JrM9e6hGQTb4YHeg4kcVB39BTGwy0wveK35lStCiqy2RE3XyTXk7oQCtRXpOv86dh3iRzEiAu8Iz3NulTnHz4jJyx\\\\\\\/1j00klpJTUCfOOHgGWLcumDAcCitZu6hWsiIZNj6VUqLhrmrOZ3nqRN0chT1QNEcPOpb+Q5XVqXkfkBH3blieWbIcQOpTSOsXQDfeLCO0pDzlA38XcgxyFHeoXUOkcZHGQOduG\\\\\\\/28LsL6E29wBFsbfeofL1KrJtG991uxvjxfnX8il0FCR5XG339TGLwIgqQaeIYzC\\\\\\\/Yee99FHX+u66JyPB8M0HekBc2ztj0erNjtSz8VjboVgch5c4\\\\\\\/DPmn8rBEv4\\\\\\\/i9zziH4IpdzGgVk33NJecFnJXnndUsyUmXFrK8MsTA2jDS4lyMK38qmkt70TyXZvBm3+3DaT59K32PlkBDOgZjZfn94a6kf2dS2QuV6VJ320QcyiZKszm4tt6t4XiIAtXT7r4tVFUP9Zh+LfmGbMkURz+jeA0Off8JjT0\\\\\\\/nMarw8BALHtGw3hoHQ\\\\\\\/IRTz1fFucm6JwERnrz7plk0sI0wmcp4p25rIWebRAwdOFs7IiePkcF+CTklvZuk8N+4Na7f7X+QhiAKLqk4Eh3UYTEn6WsFp7ewy7XsWVTylcFNUIJqFcmsKy2cNzs0W0QcnlI1o9XofCe5bxcwXF\\\\\\\/dlTjaYiIHEkh4eI9OGvFIPkOyQPV+3xz5wIpZ4GKhUDClJQpSBGamdU0P8zlgVRdErIA3V0QMa0Ml5ND02jFjhIvLnAioHbeB8WwvgJvwD9sTOBSlzybqQhrJ377HvfDQe1yzQZ7zWPIhp5pUUdfe1uafLLcsueXU1vmz+Adt4aSWRohNvBb2BOBf5KRkCc7ycvyE+VGs+X6ff0R8ofwMEE2i8s70MG+W0IAlF2f6tHeOZgX9m+ZsLPJKDb+ux3Gq3PJeJwItuijHfv\\\\\\\/GyJ1+aV+KoQnxBtRUZNlVKgj1fO9Geit3syyFNdvDC7qPuu497ypWZoo+Xsye5wMkWx5YDjgURUc71VTwgFe2AkpI0Ao1umTJuddEuh53eptynMh0STRO\\\\\\\/MSjzsqb8BoEdHIt0Gm+r+2VlTj\\\\\\\/ZDG9rQ\\\\\\\/orw2xYgA10wxjz3UiDx5dQsGOL568R0KdDYL\\\\\\\/hC\\\\\\\/nfmFgdMmni9Iy7wfjXwOM7t2JKqsTPjC8+3oWahpAOBpbADCKWvh7RJ+DrCOzmd2831B5tbzuhsUoL3k3brKltVcT5xcNBSJMhb8sGH7rfLFuiMoF+1\\\\\\\/EZyK\\\\\\\/e92TTzRNzn8HOVrxqTbqWjOpGv8YfGMwENiYzBhhcqLEX7YuAUea0Qc4e1Akgx+YbCIOrIKJot5Mo8CdI\\\\\\\/fBIN7f\\\\\\\/WtZEuV\\\\\\\/9ydeRLvNl9vs7kWA5WpI4GJGeP\\\\\\\/GQL7d7VdQKNt+9PTKhDsvwWy5byBzX1JqO9Pw0S6Sq+QY+3hFLhfcAUFPsk4qtz64qACPhPKkbdpQaYRUAH0ypJhGDog3qEZ43DiTtvcpDNjDjzqsSxnnR3jYd78ToOIbP+zXlzsrR47tImaSEaR5DaF3sZC6Jke9uxIyumLh\\\\\\\/2tk4j+7gKf84Yx3eD1nk+NCPqzlbAkO2GgVSlbgZEYAkNpepWcKV9BReE5NICNYARE+DK7HCVyiMlkH6mzWGEFpiOy\\\\\\\/4BYU6W7jkrxZGTQB\\\\\\\/zuCqcf8+bR4Q9x0E3D5dm4bAO4I7rZ+lvnKi0XaukaGg7hGgNb6caKXWyGun18Q\\\\\\\/uGa0qHNbmZv0MHe2QHXiN6MkJI\\\\\\\/kAl6JewBEQkTxH\\\\\\\/1jy83mfTAS4LGi4mFGgfnp\\\\\\\/xa\\\\\\\/GQZNY8bHBQRqMm0tfRKS98RuWVtp9z5yDLAWiSMD79Lsgu1nsblffn337SAV8LxcPLdGr+vLhjxbKhc4qMeNB6QuX6anWO69HHlGBkRQDspAVWMpLqeolaOyprjFfRODLmMtMrTr43xRBXB9oCDdwRs2cmzg8CKftwVcw9wBIxHLzb0vo0x8tzTHOGAHFbrOMq\\\\\\\/CsqHZEIpK3xuj680VyKHcnGu2Uj7UW2g2YvF08k1O38U2blsEX4YkUbt1QneyRW\\\\\\\/p8Zsr9xDjfwdOD4Fj3EAh7fLd64pK\\\\\\\/AJfTbaSzt2Ba7vvzr51Fv54AzfW9ebk+yHqLTtDYrfAKl86EDi2N9Sp+447C1OfGUChYD7SLDmwowgbbS5sxytAB\\\\\\\/RprpWn+RknxeFaOb\\\\\\\/kyJ\\\\\\\/pDtrWfXCDlHLFqLrxBoPVAwkZcppFkNM3dYK59GO4FmnrP1dZIolbgSHWFmQ1gSS0WSmHGFML53mTJ9J9\\\\\\\/dylPXi9b2oBrYNCTuygaqgZkAidhMf7EqlxvbU83Zx8CicZzysHdUaVNCLKqoWbkmSZBwhmD34n4186ss8oJkhh8+KU44bNTgktQyObr7NwLuyHA8E0A1xgl9neUYrINNmOZURetafONc6Li2uMGVCvT7PTDIO5GqiA54m6I6KP29algp0cGLeZIvbFwpBgEf1bPTKlM5KaFUsp3pZjlY5Muvio8QP5LRKmmIQDT2i1R8PmoewQOYRXakn30qrXgebXHG5BKWPIbGEsPTlodadElx50UdeS4qr9f8Kp+4OkZEzNidIWl6bG3\\\\\\\/A3vSkIZ1\\\\\\\/dlCmS+UjIm+a7MTg4c0F+T151EBK8Jewt3HU3Fb\\\\\\\/w9Rpm7OZ8z72hwLirP1wcz2TT8pH6AVfLHloaEpO1Of+ms8uMwSl+PIWLYx2JV1H93gFkDhKPGkXg4pPiBe8pMyx6lYPk89yz\\\\\\\/vnJX+GKjmTmj1VPTGN1LG8rrDiSwzBso1+aEH+l\\\\\\\/yeV9SZqvRCOm0BlRpP0tnLPbe0TOxzwF5f6TrHVhE8ohKcQiKaIQtYIg2ryWUygbgHPukJeGF+TRC5Thh8+t0p9ncdAvdCi74uWDj5UeikFhFmdMy1\\\\\\\/kMbRthcQ7MiA\\\\\\\/Mk6AuJEwFbbA4J6Lz\\\\\\\/GnkIR44E9qAgZ7dINowuSTVE+ieUyLaTg7r3zFtgt6RRBrNzcW\\\\\\\/9K0+rThoaxCEaCe\\\\\\\/4qzQ\\\\\\\/t9O5mCwjV8CbOrRZK\\\\\\\/V9jdEPq0NyAy1Ax0qv3sJiGp4Hbs38TT+XNTLU+JFks5ANzx3VZBOMQZI+3zj91zS2dFdTulq76SDpYYaJQNIVFQ2xfoU+2qcYVsPAqVXAuPgiCXkVAVx4tcUVCK4Q9ggaeiaBZYO7xeRnHTc7dcCUy9agOJ\\\\\\\/5PRVyC6VrPl\\\\\\\/x\\\\\\\/UBFpzri8lKLW2cZwaK9PlFfFq1LGY0i1atZanvNhnnMId++uK7QrxD7JYXYZb4203Qwge47xPXVx3Jajr17yxtDOkAU+5+ruibQUq3pg41J4qyJPaG5rYxC25qQYNFiETg5RELdi44IX29+IWzSywdoqTg\\\\\\\/W8UCCWjrQ2ZBcMqdYpeP\\\\\\\/rjAvNXl\\\\\\\/1h+D2j2uxWrM9NlAQZW5eNUbMi1qfIw8sDPCSw5MGBYUaiylahT0jyv6IuMNLuRNIhCmqj1GG6uctYWbhcjOUa6gtqwtsaqEn7AsyuBwBvs0jbBlCO1N7GsQRNyWC5VXrPwkl63Nrs8dz1glzKHv39DPyXUKUCPemTw6BudXLZFFyvm74nSr9yFHfxpcQRmrVazROTmtCOUvk8Fj3aKnTeOVfC81KIJrfDNfbGO9\\\\\\\/wXsquGBTPwkrGIH5uv37dYOoTi32viFn+mMEbNt2f9hG7wQEY+YOxqZlTPDv2NWNkVN1Ipcew9q9jNDgsTqt1o7BptLYk\\\\\\\/xygyhh+44sGOUWUQ2IUsLlZoqP2FmJf92LLza4bYMimSoEBe05ttyc5Sqjn4AlMlpFBjN3kQOE73wN\\\\\\\/gqlDlet80FkStgCs\\\\\\\/fKKMPI3RH2K5BU515aimcyQiNsJMCbaA59hhi\\\\\\\/dJnQforlZlKirjLFXJd15iLZbWhpZQ7scahceyea8BitvRhQ36T68i07QXAVe3KS8gY1x4FvgtNzfIFUCZmg2XEh+nJHGhzws9T6zG\\\\\\\/\\\\\\\/Rtsx17coNWMz1Ts\\\\\\\/9Qrv4FTdh1m45eJ2xv3dPRoyrncDtq3qfZseKoFLC5o41s3MMAeWAWjt6gdC3NPtIRR01a034Qf2IpNY+P\\\\\\\/cxSTzl5Jzv1rNEtUktA56f4m7xCRb08qcYgAu9BFfmKjxVl3lVmCunsMFf6\\\\\\\/qxLZ8jH5eyoBfG8fE7oI2UZOrDEwZdYv26S276JxiOrIibxxpkeaKd4xZKMfOR2XfbWyHZGEmML2\\\\\\\/yLdTXqGUwnRQmDgFp8mejzcFStajXaWVQQOnaLsrPplx9uwcvd1tJnZMMHGoSKgnQBkpg2e2d\\\\\\\/Alsn6txyAO+w76ZV8tAhcM9zdMYh1VsoAG3BDFviM3dCrzdFTOYybav21F71ooDON8m\\\\\\\/GhqYzoE3wP+IgnbLsoEeOXHYwe\\\\\\\/sPV++kiRh8o80oiYU2GIZwY0USHK\\\\\\\/y5U2VwHLM2yPfKZ0MuEafnmlBpUXFOorMQDxAnaSClalylNYCtQ2V\\\\\\\/ggxwJJ8lAwiQOe2pIHIF1fdQrsFI7GcwuamwApqAOqfIwkRbflRXzVNSO2Nh\\\\\\\/0peY6V5BsbOh\\\\\\\/T0RyrE3ncoyhnGeFRENNWgMR+15ghC+XoYEGpx8ZEsP4EcFnK9speSHCTl\\\\\\\/W\\\\\\\/D+U+I5dxrkIdBpyZ10XStXSBUSKdZM2IoHFWTRm\\\\\\\/TY0zH1xoyO8Xc1Lak4woA\\\\\\\/34gf1TfACnryuC+gOJT5q9tuTur3PnSUGJpiCsXSaXTozONKIzWRDv20sFOyraM1ZRFRM7T5ypt4cD5CbX7\\\\\\\/ee2cdc4UxL\\\\\\\/sVOfMvBBdfKzGxdOxGdPFYKkhOn1fhNtobPP0z8MzveiSEGJ8ntSmnhpQpJ2xMmD+pF9\\\\\\\/BlWsrS\\\\\\\/ZkRGeEw7KlROnJkl9zZBPK+g0QHSk\\\\\\\/PSJrrB0u9rWoP324z1wLuGdP42OhTuGYZicE6IvsF11qywAoZW73JKpqOVXiCKto5zhvW0Kc7u9eS2lD5Ltesp3XhPAMmqbEFmkM979IIzsrPdVi4yDD9\\\\\\\/0eAXz8mfex42JJm+nrknkXG0tltX+4pkn1877dkdy91qYwTITRkYOmlnIzJKFHE7pS063jPGOjVdID0RzPWP1g\\\\\\\/EYYU+6m4gdqH82\\\\\\\/8hJA7dvuupF4yKQD5nJUpcIxr5z8PsVmtJnaf0af4NMHLKQC98tYUSXcXeWBE1GSymA9hm4amltnJtq\\\\\\\/KGXimk1cjUbGbMCL0k7qFBRHeRKsStmXKilVLoppRer7IFhyCiIDoi5syt\\\\\\\/M00niVxfkRJFjI2AECM4ua5LlQW\\\\\\\/8ccOVTC+xdLCuZVR0FFAPIsMylvPIfI+dwwsrhYOjINOEebZ37BrCjjgnrv0zSOd1MZpL+YXxhgzkBfr5racC2yg+m+SjcFzEAUP6DZs4yq3AubxA0GJ0E6sBzsBydbUVzaFt6Ui8W+YtUQFLMUmrfemlGTtkXmY3L5ZANJajEhafr8Xu2fT7I9nDk9jWGEX8DzPfxiyU5RaMZORsYpskcOq0mdRnt2HOVDi49IB4pKqossme8BsbCqmckZ9gpiBOlgN+mtVLv20lD1+yUziIauFMyNtfhzXB4yzM44HucIy2ung4fudGdWJij30n05Tjt+bXgYqTugBxDwOvBdrPa8VCJevPsO0gCqUl2eTF4Gslyj\\\\\\\/VKCYvfEpD2McHudEa+Huw\\\\\\\/TGHRxNv5vYxsMoHSoKeD9e\\\\\\\/t72pf1fHj206d1lk3\\\\\\\/O06cy2SwZ6FXkb3DoPnZ9tJGp6jvmjJ3eiy8HW83VpdpZUyfUynnfmweSwAc\\\\\\\/8+PFnzqjmqu2QyA+e5+IIkLMR3vTRjwQUOPhzEL3xzavD8\\\\\\\/7+jp0Exn1PCwO7w9PpKT3JYMOh071po3Cij\\\\\\\/VnyvJHe31xIRJQxL\\\\\\\/oNk9EwuLIVkRBvoUiqM3DM1qEIcEdbzQ6nqQnwF03KsRiAbOwKiwxmN+y+nhtgO1\\\\\\\/CMv58La61PUM+nbvbCOwgx2HeC798Newc5wxloEyIlJUKjBVa2hQCXLM7wYadqM2WByBjSCuv+nX9FI3p14lyQ5W1EVyZcKyDcCmMOpHHJc1GZEt2aERyjaARxjMy1IAcfZw9my4Bu3URt9IMp\\\\\\\/h2H3n1xJWK+cUVdOXZCxDPXDvYxrYbCVyK720RiJkyx3I3qxc2mFNmoovh3SFMHFolxRYZvOI5BARPg5Cho38tQim3mRcleviwCl+bVptBQT7iwj0Jr18l4yRMarfmd9w\\\\\\\/m0RMMxQopFMwfKC3Vd7cSX0EUajo50Y9g+Rz9OHD8UHcrFCxhlyOp4iSxa\\\\\\\/FvdYQ04M6n+5fzhKIyjvhrBDQVpsZSQ437mWFhkfLIU+C8mVY9Pk\\\\\\\/Ss5rW68kimHxZfwwO2dVJoSldkbzkSZQPeuGrUJyICrult1R+Wg6lHb+BhjFDDibZbhPATGq7evE3Oh+1pEecRzMPzS3nd+m36vwHBn9taVA+3KqwFYI1uOqRtbqHnj9Wa01xhnOVzHR3u3VP30\\\\\\\/BQT08aU7u4jqKkebv3OFpvl9iILXSi66GJlXEe1YfgO1aXsD8as3iWaqaG5bYjUboDiqoJ0C3Tz93Zy1U2Y7AzdwgxsEv3aUMUbwp+0eelH+XzqcmersVMTKVEeeQW79SG364o1jxVOrViq1oA6ro8YU6N\\\\\\\/qnasj65nfZGi7eweurxFY\\\\\\\/JaiweCToxKFOv9nJlQwd40mK3cZ8KbNgSzvQCbuMmbnYs9OnAE5gvIskZNs6WEupqV5s9t+yYD1ZLIH2gwl+hv9LpYcDV2ZjErOxVDc7YWjEWiNmuUz9JDZD+WIEN8FXh+lW06\\\\\\\/qRrekfUuexTdI3uqFt3hz97axFSgN0OKW3jiN5vi\\\\\\\/t7hg3Q2kXHysldXAhNBux7cYtRLmNk+lnJ45Mr4VZbS0PUvmki4mHxlxwq23eshSM18qFUixaAD0ZStFVU+paNVS7vug02EC5SbQdlhZcto+e0zX1lfcJn079sjmjnG8Zo5f5ErLLfXxfCaSbzy2C3gksR5EtJ4vpV3L26Y\\\\\\\/UsiX64it3K5tJt+2b8aXkmwb0\\\\\\\/R8iJnfJAufyFcVUULZY\\\\\\\/fIcebWAiaHfxnXidfWsNQ\\\\\\\/TVo\\\\\\\/1jJmpwaybBHyRab3Hnlsie\\\\\\\/\\\\\\\/dJBUMCcYuCyh1pn3GBn\\\\\\\/1B95m9mcleeoCzGPC6mXC71PNZ9VfgQdteGrvJD28K1UQF8OFjZfegIa52b+6+eazjsUUhB0BmVNcUE9gxQssaDXV42i1MNHspCCFA+sgrY1QgsfTn5pcfowueChks4hv\\\\\\\/l\\\\\\\/kAPMNbRVkiL5DfgnpcTUK\\\\\\\/KcP7Klo6XuNtV9ccvAlk84qg5I5vHqE3c97z9EIN\\\\\\\/lpd7uMGzSqOZZiBc2wa0QLfcuNcLE\\\\\\\/4X81SNLAw+BqoXDfpuenrBhULkJeaQRWRuK+9yM3mXPj2AuZalhpQgyx8UxWos0zU\\\\\\\/qJvdNbIF+wFH9fpXWLK7z+zxZ0wFvpfqHJ2s8vSX4E2PwRSbEls5MZg88JIprvQ45yG5Vv281Xpzw2a\\\\\\\/GbRwRDu69HgLqCfhNb0rpho\\\\\\\/VE6N2MQKaUkbIz7g5SCLtAosxSZRCTzLV7wOZTpmaXFrsmnmaEJT\\\\\\\/s08IGhtKYkwAWm3wEMBtjufYdgr4J7lcHIsl1Ku6V9gOgLOGSOEn0H6p1sbfv0zMIircLQspzy3DNyT23RNXAexTYfE5xjqyzPtx5i8rieD46GajByM2z5dyqx53yBTRsQzkMs5NtJPzBflQQRAEiVs2CEYiPsXHN3fsSlkskiZbav5oO+quWn2UQxcHk8Ir3E0Ry05LD7eU5+J+VIRQ0+Mf3ksflmvT6Dm2T0oBSg4QZUPl35DcKu\\\\\\\/gRsElnhO6p\\\\\\\/VG79\\\\\\\/We2lgVSDrlsLmOOyPdHLXkP8+vGl1V78+V3pGlEjOPS25wamAV7qalUN\\\\\\\/jBWiR+YmZMtVeBwAYmhMlKZwwC1EaFySteJTMH8ElH9ag1ZQMsIpvsuF66reIAMdNEXCv9hR0DmqYqqkOn0C8u+NCRA7HmDkcsqKaZNblCnWVAnSVnV5fLUR00oZp2O1IUw0opGHo7vX1XaSn9NxxihsaDE4soN+ipt2UmOvkbQIr+Taf8UPjURiqorICbCGyEbm96QwY8zyOsC+JlMhFaDWXku94HRP7ac1zyQIQlxT4j9LQA+6tIw9rYSqAUyDldq1sMWpmwIsopj8hfKxl6g+yTzXLsTbCHCphk+jNQSGizlPxKnxecH38ZUFcIccXsIJHv3hH6r7oHPysjPYUwmeqfJleNMlXI6a1sJ9ELt2Un2vaUahq6VyrSvpkDpqgXeOfP0Kof0OvBF99wi1vXFMe9EaF21nyM9aHNPl6dIyj6KBuh101F4ZHwXEMBEL11TcgoKASAS4C4c4W1LcjAKkgk+ydIA++izl4oa28kQopdZpdj1Xz\\\\\\\/MvgFHHc54k+Iw+Gf9QpEr+69tUN0Jc\\\\\\\/VwGipMe87AoSVvkFfQZ+Z72CgaMQ4Wl9NqQROO2QV+nimXdOF1NjlIRyCFJTV7WkvzwLwiDonwa+yWHUASYuiXzUvq\\\\\\\/pVXjHWAJmxt\\\\\\\/Ta\\\\\\\/r7Jhpq9sAJvrwvYdFKL5We56OxibgyX19yCMa\\\\\\\/APR1+Oh18OsyTeBzEOPK6sJSZnQYhv8C8rtX5swK+Q5Gnl+c7DtF8tgeVBZ2733VJNmKhM7T+rZ730VRfluD8T3TanTtiJwSL+Y9w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ayR1TpU0uqaU7t9HEQvCKg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224944ec026487d97dcf693c5de613b5465e0563da56a1a7cd87138d46e2777c2f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zDxx\\\\\\\/i541akQBQPqeaOAtJtFakW84LTKUmgoVzSAtEUMj94hKGanesL1UKjUbuTH\\\\\\\/GObD0R7nneSjaFC5PJai89Bemr3bQYp3XFoUQgwIH6p6d0plk8mAeJqNXfWYI1BXmRKJv+2HuDsNrs6PVWe+OWk\\\\\\\/t5S3szLxamUQeEKAMZIumhUdx\\\\\\\/GxDjWQUcXeWjiv\\\\\\\/4sbyXeELwo7ODFHshCFCzIuUoEUx2rDsKEbqU2LnyzRDy8s\\\\\\\/4mau7pmh+iwZdBhli24fj2Xe0g56g9c9gbNt+S5GtzLMNaUz6gL+wDC7GsV0ydlrZcAnCswt2prB1AYYf0vLG0+gi7zHFqi6bOK0+\\\\\\\/vKcKnrpZk6av3K849DYlLL4IqG9m73+fkwHb13Twq0m3dNssv0m+j7G0dKEZLQh3JhayrEEQAuqIX11KXATsWBqbwPJxsPxkEM6E2QlhTpVHFmkXPC4TLcO9qMsnk2d74HKql57aA21+ShlO3FiPB\\\\\\\/y\\\\\\\/Anw3JBTzgBNyW7kSyAOAhT57dl0SuSj+LQdY3CT8CXI27omJ7qL\\\\\\\/4qQdsvJlUiYVuZodCnM4K14nS0\\\\\\\/ncgNJRzkzQ4ktR4XoSG4m6wT5zrGrT9qxMYQO7n1OSCQnjTIY7hy5VcLLJ\\\\\\\/W62yjgrtyV5KV+PS0HcDdBgt2F+GNjfSgIy\\\\\\\/1eWj2Q1uc0sHWHTQq1\\\\\\\/2HkZWrXqZ3auzuSphQqpge93LCWPTsI80qHgljP2s2zhM0yZOeN8iMqu+hLlHzl7SzZ5E8jsbLUKUALxCXQ1uTB3VQNhwda8iGiGEecUOTfr9AzIj3NOIgObrwrcFy4a1Tt4LaZhISwqwQp2PGpD+QX+qy5IvS6G+HC83t\\\\\\\/zbeb7kcfBnVa1lYELqFAZ\\\\\\\/6v6RLwX8Git55WKb54LKLtZ3HJj74WFb0MVZ4ZA7VFQ8Hckj\\\\\\\/9a\\\\\\\/7A8tXdUgbP2ZFAqwwXTL9eJ3rkGnR78oRPSBoYRxttSYbthI51rZ5gZ+IwoftB2XZUMaheH+ruR7b7i0dHhl0mVJ6fylFG4mDgYMVNc82rfEjCUg4bY6TD48JmQ+kHJzaEKc3\\\\\\\/9DJrkKFXXTnlL2IREf6qwERm4K0r+f7gXinbJp5lEASfm1RftuSqnv0S9V26YQgkj9chNgR6uCAVyFgANScGIsS5T5Shv8eAgQe9kqaz4SGJIOComTqm47RUbOQUhkBp3yQgfnqcc\\\\\\\/fDcMZJxbGHaaNpyvcOQjQYxEiM\\\\\\\/9a9TbmGGE4PX62DFZFPr6K0dzL9y9d5v5jS8M9Ygau0W3sxFj+avrK2hSDd\\\\\\\/YwR4j9taKtYUq\\\\\\\/fCFJ6zlHmV7C4z1Dd50uxZtSPhUFYXKFNAUWq9Co0\\\\\\\/xpcPR8EHppR\\\\\\\/Kt1J5Z\\\\\\\/yVZIYBK9vnKrgHX99YVfzJiely36J3MRlbfF4int7APY\\\\\\\/ti0uoVRinESRfS1GqNTbAMev3i2BCCm8svk3\\\\\\\/pLWUEDCJ3b+xVWsYD87AA2Wvdo\\\\\\\/dg7Dl6JwF+Z0Z\\\\\\\/pK+JoArqG\\\\\\\/ZhfJaWY+CohcKCMRpLmxxC0BXa6HtAZ2Onz21ofcWLhOqW6q1uggpioqHN0zkJVRq95SyVF3BorIUhzY8kSLUJqExxxZB87vYUVQFsb4abxWSSZQSgdp9g\\\\\\\/EUi5nKtuQkbW1cQ6LiKokBITd\\\\\\\/PS1677QVStaAxSxtwyAWnC3muNnU+GF+tA2USHzAdBTLTyK0rmoIQKq\\\\\\\/3qup55vLk1ooP\\\\\\\/6S2tMbhNoLVAvXkFJkbMEqlctyuaHaxHt8GdKrFABMKxcbYChn3Q\\\\\\\/XwKMw\\\\\\\/mkMfXlJAVKKgZVtdK5Exj+0Biil0AAyZJrkFFGnuPjQ2auA8K1TXVOmyiWOUCKWifdJ3PYgGLymBzYzzfAxhfpjva9S\\\\\\\/J\\\\\\\/MenA5fZfVWbuizX4KsA31ZnIp3v+86DjQyauGLY2jf3wjNiivpZFhQCBqJOCDEMsxMoHIMK0M4eWYl1Afk5sY2WL6WsIOJlRBZ\\\\\\\/v9UErAnZQLzxs5bWKA2ZaJETuRUT69PO\\\\\\\/nXYQ\\\\\\\/VMgQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Fmj6qxDJ6gR7kS0ZaSuqzQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f548f9060f6bdcfedd66920856d76186cd7fd248073567518f303057407b7444\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459992 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1575" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CMGbwuuQpxF5qlxAoNQDaTrgwTcjs8YFkvOK6RSoZXY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1575" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AMl2qGCGAZr2igdS0Ios9A13V4hknX1N62PZFm3FwwQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1567" title="SDN-1/Syndecan Acts in Parallel to the Transmembrane Molecule MIG-13 to Promote Anterior Neuroblast Migration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1585" title="The Wukong Terminal-Repeat Retrotransposon in Miniature (TRIM) Elements in Diverse Maize Germplasm" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459992" id="node459992" data-pisa="ggg;5/8/1575" data-pisa-master="ggg;g3.115.018119" data-apath="/ggg/5/8/1575.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5116-4628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>André M. Hidalgo</span>, <span class="highwire-citation-author" data-delta="1">John W. M. Bastiaansen</span>, <span class="highwire-citation-author" data-delta="2">Marcos S. Lopes</span>, <span class="highwire-citation-author" data-delta="3">Barbara Harlizius</span>, <span class="highwire-citation-author" data-delta="4">Martien A. M. Groenen</span> and <span class="highwire-citation-author" data-delta="5">Dirk-Jan de Koning</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1575-1583;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018119 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459992" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">André M. Hidalgo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andr%C3%A9%2BM.%2BHidalgo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hidalgo%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndr%25C3%25A9%2BM.%2BHidalgo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5116-4628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for André M. Hidalgo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andre.hidalgo@wur.nl" class="" data-icon-position="" data-hide-link-title="0">andre.hidalgo@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John W. M. Bastiaansen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BM.%2BBastiaansen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastiaansen%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BW.%2BM.%2BBastiaansen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcos S. Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BS.%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BS.%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Barbara Harlizius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BHarlizius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harlizius%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarbara%2BHarlizius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martien A. M. Groenen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martien%2BA.%2BM.%2BGroenen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Groenen%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartien%2BA.%2BM.%2BGroenen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dirk-Jan de Koning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk-Jan%2Bde%2BKoning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Koning%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADirk-Jan%2Bde%2BKoning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/8/1575" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459992" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459992/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1575.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459992" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459992/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1575.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459992" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459992/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/8/1575.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459992" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459992/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/8/1575.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459992" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459992/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genomic selection has been widely implemented in dairy cattle breeding when the aim is to improve performance of purebred animals. In pigs, however, the final product is a crossbred animal. This may affect the efficiency of methods that are currently implemented for dairy cattle. Therefore, the objective of this study was to determine the accuracy of predicted breeding values in crossbred pigs using purebred genomic and phenotypic data. A second objective was to compare the predictive ability of SNPs when training is done in either single or multiple populations for four traits: age at first insemination (AFI); total number of piglets born (TNB); litter birth weight (LBW); and litter variation (LVR). We performed marker-based and pedigree-based predictions. Within-population predictions for the four traits ranged from 0.21 to 0.72. Multi-population prediction yielded accuracies ranging from 0.18 to 0.67. Predictions across purebred populations as well as predicting genetic merit of crossbreds from their purebred parental lines for AFI performed poorly (not significantly different from zero). In contrast, accuracies of across-population predictions and accuracies of purebred to crossbred predictions for LBW and LVR ranged from 0.08 to 0.31 and 0.11 to 0.31, respectively. Accuracy for TNB was zero for across-population prediction, whereas for purebred to crossbred prediction it ranged from 0.08 to 0.22. In general, marker-based outperformed pedigree-based prediction across populations and traits. However, in some cases pedigree-based prediction performed similarly or outperformed marker-based prediction. There was predictive ability when purebred populations were used to predict crossbred genetic merit using an additive model in the populations studied. AFI was the only exception, indicating that predictive ability depends largely on the genetic correlation between PB and CB performance, which was 0.31 for AFI. Multi-population prediction was no better than within-population prediction for the purebred validation set. Accuracy of prediction was very trait-dependent.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li><li class="kwd"><a href="/keyword/across-population" class="hw-term hw-article-keyword hw-article-keyword-across-population" rel="nofollow">across-population</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/multi-population" class="hw-term hw-article-keyword hw-article-keyword-multi-population" rel="nofollow">multi-population</a></li><li class="kwd"><a href="/keyword/reproduction-traits" class="hw-term hw-article-keyword hw-article-keyword-reproduction-traits" rel="nofollow">reproduction traits</a></li><li class="kwd"><a href="/keyword/within-population" class="hw-term hw-article-keyword hw-article-keyword-within-population" rel="nofollow">within-population</a></li></ul><p id="p-4">Genomic selection has been widely implemented in dairy cattle breeding when the aim is to improve performance of purebred animals (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Berry <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">VanRaden <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> 2009b</a>). In pigs and poultry, however, the final product is a crossbred animal. This may affect the efficiency of methods that are currently implemented for dairy cattle. In pig breeding, multiple sire and dam lines are used, with a minimum of two lines (typically for crossbred sows) and often additional sire lines to produce a three-way or four-way cross finisher pig (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Merks and De Vries 2002</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lutaaya <em>et al.</em> 2001</a>).</p><p id="p-5">Selection based on genomic estimated breeding values (GEBV) for purebreds (PB) using phenotypes on crossbreds (CB) is expected to increase the response to selection observed in CB compared to the situation in which only PB phenotypes are used. This increased response is expected when the genetic correlation between the PB and CB trait is less than 1, especially when the genetic correlation is 0.7 or less (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dekkers 2007</a>). Genetic correlations between PB and CB performance vary and can be considerably less than 1 (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lutaaya <em>et al.</em> 2001</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Zumbach <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cecchinato <em>et al.</em> 2010</a>). Adding CB individuals to the training data is very expensive because, besides genotyping, it also requires additional identification and individual recording of target traits. Breeding companies are not inclined to make these investments unless there is evidence that predictions yield greater gains and higher accuracies. Simulation studies have shown that the response to selection is greater when PB animals are selected based on CB performance and that accuracy of prediction is high (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dekkers 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ibánez-Escriche <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kinghorn <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Toosi <em>et al.</em> 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Zeng <em>et al.</em> 2013</a>). There is, however, a lack of studies using real data. The number of genotyped CB is not yet large enough to test the superiority of training on CB for PB selection. A first step toward finding the optimal genomic selection scenario for pigs is to determine predictive ability (accuracy), in real data, of GEBV for CB pigs based on PB genomic and phenotypic data. This will show how CB performance responds to the current practice of selection on GEBV in PB pigs.</p><p id="p-6">Recently, accuracies of within-population genomic prediction in pigs have been reported (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cleveland <em>et al.</em> 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Forni <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Christensen <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tusell <em>et al.</em> 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Badke <em>et al.</em> 2014</a>). These studies have shown that all traits had more than zero predictive ability within population in a variety of pig breeds using different methods. It has also been shown that using genomic information generally increased the accuracy of prediction compared to using only pedigree information (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Forni <em>et al.</em> 2011</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Christensen <em>et al.</em> 2012</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Tusell <em>et al.</em> 2013</a>). Using multi-population training might be a way to increase the accuracy of prediction further. This is especially relevant to enable genomic selection for small populations when a closely related breed, or the same breed from another country, is added to the training set (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lund <em>et al.</em> 2014</a>). An unresolved question is how to obtain accurate predictions from multi-population datasets. The effectiveness of a multi-population genomic evaluation depends on many factors, <em>e.g.</em>, differences in allele frequency and consistency of linkage disequilibrium (LD) between quantitative trait loci (QTL) and single nucleotide polymorphism (SNP), which could reduce the accuracy of prediction (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wientjes <em>et al.</em> 2013</a>), whereas the larger reference population would potentially improve the accuracy.</p><p id="p-7">The objective of our study was to determine predictive ability (accuracy) in CB pigs using real PB genomic and phenotypic data. The outcome is a first step toward determining the optimal genomic selection scenario to select PB for CB performance. As in cattle, studying accuracy of prediction for multi-population datasets is important for species in which population size imposes upper limits to the training population size. Therefore, a second objective was to compare the predictive ability of SNPs when training is done in either single or multiple populations in pigs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data</h3><p id="p-8">Genotypes were available from sows with own-performance information of three pig populations born from 2005 through 2012: 1070 Dutch Landrace-based (DL) sows from 19 farms; 1389 Large White-based (LW) sows from 14 farms; and 287 individuals from an F1 cross between these two commercial lines (DL sire/LW dams) originating from three farms. The genotyped CB animals had no specific family structure and the majority of them were not offspring of the genotyped PB animals, <em>i.e.</em>, a number of generations separated PB and CB. The 287 CB animals were offspring from 76 sires and 170 dams. Four female reproduction traits were analyzed: age at first insemination (AFI); total number of piglets born (TNB); litter birth weight (LBW); and litter variation (LVR). AFI consisted of the age at the second estrus, which was the time that the first insemination was performed. TNB was the sum of all piglets born alive and stillborn. LBW was the sum of individual birth weights of all piglets born in the same litter. Finally, LVR consisted of the standard deviation (SD) of individual birth weight of the piglets from the same litter.</p><p id="p-9">The PB and CB sows that were selected for genotyping have phenotypic records from multiple parities on multiple traits and have a large genetic contribution to future descendants. All PB sows were breeding animals from nucleus farms, whereas the CB sows belonged to farms where combined crossbred and pure line selection (CCPS) is applied. There was no strong selection for first parity performance in the genotyped sows, reducing any possible bias in TNB and LBW due to culling after first parity.</p><p id="p-10">Deregressed estimated breeding values (DEBV) were used as response variable for each trait undergoing study. The estimated breeding values (EBV) were deregressed for each trait separately using the methodology proposed by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Garrick <em>et al.</em> (2009)</a>. DEBVs, instead of EBVs, were used to compute the GEBV accuracy because this removes the influence of the parents’ EBVs and rescales the EBV according to its accuracy, <em>i.e.</em>, the DEBV of the animals reflect their genetic merit. <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ostersen <em>et al.</em> (2011)</a> have shown that using DEBVs rather than EBVs for genomic prediction yields higher GEBV accuracies. The number of animals and records used to estimate the EBVs are in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The EBV of each animal was obtained from the routine genetic evaluation by Topigs Norsvin using MiXBLUP (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mulder <em>et al.</em> 2012</a>) in a multi-trait model (including all measured reproduction traits). The genetic evaluation was done across lines with phenotypes from the different populations treated as the same trait. A fixed line effect was included in the model for estimating EBVs. In multi-population prediction scenarios, this line effect was added back to the random additive genetic effect after estimating the EBVs, and subsequently, the line effect was again included in the genomic prediction model. Adding back the line effect allows the differences of the level of EBV between-population to be maintained in the data. Therefore, in the genomic prediction step, the mean differences between populations are still present, and this allows SNP effects (that differ in allele frequencies between lines) to explain these differences between lines.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466625/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466625/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of phenotypes on crossbreds and purebreds that were used to estimate the breeding values</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">The model for obtaining the EBVs for AFI included genetic line and herd-year-season as fixed effects and an additive genetic effect (animal) as random effect. For TNB, the fixed effects were genetic line, parity, interval between weaning and pregnancy (days), whether more than one insemination procedure was performed (yes or no), and herd-year-season. The random effects consisted of service sire, a permanent effect to account for the repeated observations of a single sow, and an additive genetic effect (animal). EBVs for LBW were obtained with a model that included genetic line, parity number, TNB, and herd-year-season as fixed effects and a permanent effect and an additive genetic effect (animal) as random effects. The model used for LVR was similar to the one used for LBW, except that TNB was removed. The reliabilities per animal, needed for deregression, were extracted from the genetic evaluation based on the methodology of <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tier and Meyer (2004)</a>. The heritabilities (h<sup>2</sup>) used for deregression were estimated via restricted maximum likelihood (REML) using a pedigree-based relationship matrix and were also obtained from the routine genetic evaluation. The h<sup>2</sup> of the traits were 0.30 for AFI, 0.11 for TNB, 0.38 for LBW, and 0.14 for LVR. The genomic h<sup>2</sup> of the DEBVs were estimated via REML using ASREML 3.0 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gilmour <em>et al.</em> 2009</a>).</p><p id="p-13">Sows were genotyped using the Illumina PorcineSNP60 BeadChip (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ramos <em>et al.</em> 2009</a>). SNPs with GenCall &lt;0.15, unmapped SNPs, and SNPs located on either the X or the Y chromosome, according to the Sscrofa10.2 assembly of the reference genome (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Groenen <em>et al.</em> 2012</a>), were excluded. Quality control was performed in all populations simultaneously, which involved excluding SNPs with call rate &lt;0.95, minor allele frequency &lt;0.01, and strong deviations of Hardy-Weinberg equilibrium (<em>χ<sup>2</sup></em> &gt; 600). After quality control, 42,139 SNPs remained out of the initial 64,232 SNPs. Individuals with missing genotype frequency &gt;0.05 were also removed. Missing genotypes of the remaining animals were imputed using BEAGLE 3.3.2 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Browning and Browning 2007</a>).</p></div><div id="sec-3" class="subsection"><h3>Statistical analyses</h3><p id="p-14">GEBVs were computed based on the genomic best linear unbiased prediction method (GBLUP). GBLUP uses a genomic relationship matrix (<strong>G</strong>) instead of the numerator relationship matrix (<strong>A</strong>). The <strong>G</strong> matrix contains genomic kinship indicating relatedness between animals and was used for prediction in all scenarios with the model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-1.gif"/></noscript></span></span>where <strong>y</strong> is the vector of DEBVs, <em>μ</em> is the overall mean, <strong>g</strong> is the vector of random additive genetic effects assumed to be ∼N(<strong>0</strong>, <strong>G</strong><span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-2.gif"/></noscript></span></span>), <strong>Z</strong> is a design matrix allocating <strong>g</strong> to <strong>y</strong>, and <strong>e</strong> is a residual with heterogeneous variance due to differences in reliabilities of the DEBVs (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Garrick <em>et al.</em> 2009</a>). In predictions where the training set contained more than one population, the fixed line effect present in the model for estimating EBVs was also included in the GBLUP model as a fixed effect.</p><p id="p-15">The <strong>G</strong> matrix for within-population prediction was built according to <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">VanRaden (2008)</a>, which was computed as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-3.gif"/></noscript></span></span>, where <strong>Z</strong> is a matrix of centered genotypes and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-5.gif"/></noscript></span></span> are the allelic frequencies of the <em>i<sup>th</sup></em> marker based on observed genotypes. In predictions where the training set contained more than one population, the <strong>G</strong> matrix was built according to <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> (2013)</a>, accounting for differences in allele frequencies between populations.</p><p id="p-16">We used ASREML 3.0 (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gilmour <em>et al.</em> 2009</a>) to predict the GEBVs, with the <strong>G</strong> matrix entered as a user-defined matrix. Animals assigned to the prediction set had their DEBVs removed before predicting GEBV.</p><p id="p-17">All scenarios were also analyzed using the <strong>A</strong> matrix, which contains the average additive genetic relationships of the animals based on the pedigree (PED-BLUP). The model for these analyses was similar to the GBLUP one; however, the <strong>g</strong> vector of the random additive genetic effect was assumed to be ∼N(<strong>0</strong>, <strong>A</strong><span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-6.gif"/></noscript></span></span>).</p><p id="p-18">Genetic correlations between PB and CB performance were estimated for the four traits. We used records for DL, LW, and F1 animals born from 2005 through 2012 (<a href="http://www.g3journal.org/content/suppl/2015/05/26/g3.115.018119.DC1/018119SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/26/g3.115.018119.DC1/TableS1.pdf">Table S1</a>). Genetic correlations were estimated in bivariate analyses using REML in ASREML 3.0 (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gilmour <em>et al.</em> 2009</a>). The effects of bivariate models were the same as those used to obtain the EBVs (see above); however, to estimate genetic correlations, PB performance and CB performance were treated as different traits (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Falconer 1952</a>), which in matrix notation is:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-7.gif"/></noscript></span></span>where <strong>y<sub>i</sub></strong> is the vector of observations with <em>i</em> being 1 for purebred and 2 for crossbred data, <strong>Z<sub>i</sub></strong> is the incidence matrix for <strong>g<sub>i</sub></strong>, which is a vector of random additive genetic effects. The additive genetic variance is expressed as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/8/1575/embed/mml-math-8.gif"/></noscript></span></span>where <strong>A</strong> is the numerator relationship matrix and <strong>G<sub>0</sub></strong> is a 2×2 covariance matrix with the purebred and crossbred variances in the diagonals and the covariances in the off-diagonals.</p></div><div id="sec-4" class="subsection"><h3>Scenarios and accuracy of prediction</h3><p id="p-19">Seventeen scenarios were investigated that can be divided into four groups according to composition of the training and validation data sets as follows:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-20">Scenarios 1–3: Training and validation data were subsets from the same population, DL, LW, and F1, respectively, <em>i.e.</em>, prediction was within-population. These scenarios determine how well the within-population prediction performs for the different traits.</p></li><li id="list-item-2"><p id="p-21">Scenarios 4–7: Same as scenarios 1–3 but the remaining PB population(s) was/were added to the training data, <em>i.e.</em>, prediction was multi-population. These scenarios determine whether adding data from a different PB population to the training data would increase the accuracy compared to the within-population prediction.</p></li><li id="list-item-3"><p id="p-22">Scenarios 8–11: One PB population was used for training to predict the other PB population. F1 data were not used in these scenarios, <em>i.e.</em>, prediction was across breeds. These scenarios determine how well across-population predictions would perform.</p></li><li id="list-item-4"><p id="p-23">Scenarios 12–17: PB populations were used for training and CB animals were used for validation. These scenarios determine how well CB genetic merit can be predicted from PB data alone, and whether inclusion of more than one parental PB population increases the accuracy.</p></li></ul><p id="p-24">The accuracy of prediction was estimated as the correlation between the GEBV/EBV and the DEBV of the validation set animals for GBLUP/PED-BLUP. Prediction bias was calculated by regressing the validation variables (DEBV) on the prediction variables (GEBV/EBV). Accuracies were the average of 20 random training-validation populations in scenarios 1–7, 9, 11, 13, 15, and 17. For scenarios 1–7, we randomly set aside part of the genotyped animals (N = 50) and used those in a later step to determine the accuracy of prediction. These 50 were not included in the training for those scenarios. In scenarios 9, 11, 13, 15, and 17, not all the available animals were used for training. Subsets of the training populations were sampled such that the same number of animals was used from each population per trait undergoing study. Any differences in accuracies would then be due to the different populations used, and not to differences in the number of animals. Scenarios 8, 10, 12, 14, and 16 only had one estimate of accuracy because all the animals were used in the training population to maximize prediction accuracy of animals in another population.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><p id="p-25">Estimates of genomic h<sup>2</sup> of the DEBVs across traits and populations ranged from 0.04 to 0.58 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Estimates of pedigree-based h<sup>2</sup> of the DEBVs across traits and populations ranged from 0.03 to 0.78 (<a href="http://www.g3journal.org/content/suppl/2015/05/26/g3.115.018119.DC1/TableS2.pdf">Table S2</a>). The genomic and pedigree-based heritabilities were similar in general. Genetic correlations between PB performance and CB performance for the four traits undergoing study ranged from 0.31 for AFI to 0.90 for LBW (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466667/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466667/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimated genomic heritability (h<sup>2</sup>) of the deregressed estimated breeding values across traits and populations under study</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466607/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466607/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genetic correlations between purebred and crossbred performance for the four traits undergoing study</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">Accuracies for within-population predictions for scenarios 1–3 ranged from 0.22 to 0.72 for GBLUP and from 0.21 to 0.64 for PED-BLUP across the four traits and different training sets, indicating a modest to good predictive ability (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The regression coefficient of the GEBV/EBV on the DEBV for scenarios 1–3 ranged from 1.03 to 1.70 for GBLUP and from 0.90 to 2.21 for PED-BLUP.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466517/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466517/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">GEBV accuracies from within-population prediction using GBLUP and PED-BLUP (scenarios 1–3)</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">For multi-population prediction of PB populations (scenarios 4 and 5) the accuracies ranged from 0.18 to 0.67, whereas for multi-population prediction (two PB + one CB) of the CB population (scenarios 6 and 7) the accuracies ranged from 0.17 to 0.45 for GBLUP and from 0.32 to 0.42 for PED-BLUP. When predicting PB (scenarios 4 and 5; <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>), the addition of the other PB population resulted in lower accuracies for all four traits in comparison to within-population prediction for GBLUP. When predicting CB (scenarios 6 and 7; <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>), the addition of PB populations resulted in lower accuracies for AFI and TNB but higher accuracies for LBW and LVR. The regression coefficient of the GEBV/EBV on the DEBV for scenarios 4 and 5 ranged from 0.86 to 1.18 for GBLUP, whereas for scenarios 6 and 7 it ranged from 0.80 to 3.11 for GBLUP and from 0.97 to 5.00 for PED-BLUP. Accuracies and regression coefficients of the EBV on the DEBV were not computed for PED-BLUP for scenarios 4 and 5 because the other PB population to be added is not related according to the pedigree.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466445/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466445/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">GEBV accuracies from multi-population prediction using GBLUP and PED-BLUP (scenarios 4–7)</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">GEBV accuracy of across-breed prediction, <em>i.e.</em>, predicting genetic merit of one PB from a different PB population, performed poorly for AFI and TNB (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>); accuracies were not significantly different from zero (<em>P</em> &gt; 0.05). Accuracies for LBW and LVR ranged from 0.13 to 0.26 across the different training sets for GBLUP. The regression coefficient of the GEBV on the DEBV for AFI and TNB ranged from −0.71 to 1.37, whereas for LBW and LVR it ranged from 0.70 to 1.40. Accuracies and regression coefficients of the EBV on the DEBV were not computed for PED-BLUP because the two PB populations are not related according to the pedigree.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466594/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466594/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">GEBV accuracies from across-population prediction using GBLUP (scenarios 8–11)</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Accuracy of prediction in scenarios 12–17 that predicted genetic merit of CB using PB parental populations as training data performed poorly for AFI (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>); accuracies were not significantly different from zero for both GBLUP and PED-BLUP (<em>P</em> &gt; 0.05). For the other three traits, TNB, LBW, and LVR, however, predictive ability was observed. Accuracies ranged from 0.11 to 0.31 for GBLUP and from 0.08 to 0.22 for PED-BLUP. The regression coefficient of the GEBV/EBV on the DEBV for AFI ranged from −1.14 to −0.15 for GBLUP and from 0.15 to 0.95 for PED-BLUP, whereas for TNB, LBW, and LVR it ranged from 0.48 to 3.82 for GBLUP and from 0.53 to 7.76 for PED-BLUP.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466534/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466534/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">GEBV accuracies from prediction of crossbred genetic merit from purebred training data using GBLUP and PED-BLUP (scenarios 12–17)</span><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-6"><h2 class="">Discussion</h2><p id="p-44">Accuracies of genomically predicted breeding values in CB and PB pigs were estimated for four female reproduction traits in 17 scenarios to optimize the use of genomic data for crossbred animals. We have used DEBVs as a response variable with a moderate to high mean reliability (ranging from 0.33 to 0.80) for the different traits and populations. The SD of the accuracies in scenarios in which we had replicates of training validation populations varied according to the type of prediction (within, multi-, across, or PB to CB). Within-population and multi-population predictions showed higher SDs because the relationship between training and validation in each replicate could substantially vary due to different degrees of relationships within a population. For across-population and PB to CB predictions, the relationship between training and validation populations was naturally lower; therefore, in each replicate there was less variation.</p><div id="sec-7" class="subsection"><h3>Within-population prediction</h3><p id="p-45">LBW and LVR showed generally higher accuracies than AFI and TNB. This difference between traits may occur due to the lower reliability of the DEBV for AFI and TNB, which lowers the accuracy when the number of observations is preset. Another possibility is that there are non-additive genetic effects (<em>e.g.</em>, dominance, epistasis) affecting AFI and TNB more, whereas LBW and LVR may be regulated mainly by an additive action of the genes. Therefore, the importance of non-additive effects needs to be further investigated. Even with the low number of genotyped CB pigs, all traits showed predictive ability within the CB. Therefore, a greater number of genotyped CB should increase these accuracies. In general, GBLUP outperformed PED-BLUP across populations and traits, which is mainly a result of a better estimation of relationships among individuals by the markers. Similar results have also been reported in other studies using pigs (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Forni <em>et al.</em> 2011</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Tusell <em>et al.</em> 2013</a>). The regression coefficients of the GEBV/EBV on the DEBV for both GBLUP and PED-BLUP were, in general, close to 1, indicating that the predictions were not severely biased, except for TNB, where some of the them deviated considerably from 1.</p><p id="p-46">The level of accuracy found here is concordant with those found in other studies on pigs (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cleveland <em>et al.</em> 2010</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Forni <em>et al.</em> 2011</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Christensen <em>et al.</em> 2012</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Tusell <em>et al.</em> 2013</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Badke <em>et al.</em> 2014</a>). In these studies, as well as in ours, many traits and breeds were studied and within-population prediction always had predictive ability. One of the studies (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Tusell <em>et al.</em> 2013</a>) also studied TNB for two PB populations and their F1 cross and also found that prediction within the F1 cross has greater accuracy than within-PB prediction. They argued that this might be caused by the structure and effective sample size of the populations undergoing study. Accuracies found by <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Christensen <em>et al.</em> (2012)</a> were not statistically different between single-step BLUP (SS-BLUP) and GBLUP, but both were higher than pedigree-based prediction and GBLUP was shown to be more biased. The advantage of using SS-BLUP was an increase of accuracy for non-genotyped animals. Because our aim was to predict genotyped animals, we studied accuracies of prediction using GBLUP.</p></div><div id="sec-8" class="subsection"><h3>Multi-population prediction</h3><p id="p-47">Adding data from a different PB population to the training data (scenarios 4 and 5) decreased the accuracy of prediction compared with within-population predictions (scenarios 1–3) for GBLUP. Adding data from the two PB populations to the CB training data (scenarios 6 and 7) had different results depending on the trait. LBW and LVR that had high genetic correlation between PB and CB performance had an increase in accuracy, whereas for AFI that had a low genetic correlation there was a decrease in accuracy. TNB had a high genetic correlation; however, the accuracy also decreased, which was unexpected.</p><p id="p-48">If traits are genetically very different (low genetic correlation between PB and CB), then adding more animals with the other trait to the training is not expected to increase the accuracy. When the trait is the same, however (high genetic correlation), including more animals with the other trait (PB <em>vs.</em> CB) is expected to increase the accuracy. Besides having a high genetic correlation between the traits, the additional animals also need to have some (genomic) relationship to the validation animals. In addition to a low genetic correlation between PB and CB performance, the degradation of accuracy might result from differences in non-additive effects.</p><p id="p-49">For PED-BLUP, adding the two parental PB populations in the training also had different results depending on the trait. AFI and LBW had an increase in accuracy, whereas TNB and LVR had a slight decrease in accuracy. The regression coefficient of the GEBV/EBV on the DEBV estimated to investigate bias for scenarios 4–7 was, in general, close to 1, indicating that the predictions did not suffer from a large bias, except for AFI and TNB in scenarios 6 and 7. For these traits, whenever the PB parental populations were used as training and CB used as the validation set, the regression coefficient of the GEBV/EBV on the DEBV indicated that the estimates were severely biased.</p><p id="p-50">A review regarding multi-population prediction in cattle (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lund <em>et al.</em> 2014</a>) has shown that combining populations, in general, increases the accuracy of prediction when the breeds are the same but from different countries, to a lesser degree when the breeds are closely related, and has little or no benefit when the breeds are distantly related. Another study (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hayes <em>et al.</em> 2009a</a>) has reported slightly higher accuracies when using multi-population prediction compared to within-population prediction in dairy cattle. <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> (2013)</a> used Angus and Charolais steers to determine the accuracy of prediction with GBLUP for within-population and multi-population predictions. In their study, accuracies did not always increase, suggesting that noise was being added to the predictions. The maximum increment in accuracy that they obtained was 0.05, whereas a decrement of 0.07 was also obtained, which is within the same range as the differences observed in the current study. These studies showed that adding another PB population to the training data in cattle did not necessarily increase the accuracy of prediction, similar to our current results in pigs.</p><p id="p-51"><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">De Roos <em>et al.</em> (2009)</a>, using simulated data, also showed that increasing the size of the training data by adding animals from a different population does not always increase the accuracy. An increase in accuracy higher than within-population was only found when the populations were closely related, when marker density was high, or when the size of the initial within-population training data set was small. In our case, the number of markers was reasonable and in some scenarios the size of the within-population training data set was small, but we still did not have a great increase in accuracy of prediction. This suggests that the marker density might not be sufficient to have similar LD levels between QTL and markers in the different populations that are mixed. The genetic distance between the populations was probably an important factor that limited the benefit of adding training data from other populations.</p></div><div id="sec-9" class="subsection"><h3>Across-population prediction</h3><p id="p-52">Some predictive ability was observed when predicting across populations for LBW and LVR, whereas for AFI and TNB all the accuracies were null. Increasing the size of the training population slightly improved the accuracies of prediction, on average by 0.05. Greater accuracies were found when DL predicted LW genetic merit, rather than the other way around (scenario 9 <em>vs.</em> scenario 11). The regression coefficients of the GEBV on the DEBV for scenarios 8–11 were, in general, close to 1 for LBW and LVR, indicating that the predictions did not suffer from much bias. For AFI and TNB, however, regression values greatly deviated from 1, sometimes with negative values, which we attribute to the very low accuracies we found.</p><p id="p-53">In a study by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Harris <em>et al.</em> (2008)</a>, the prediction across Holstein-Friesian and Jersey cattle breeds was also investigated. Predictions were not accurate, ranging from −0.1 to 0.3 for 25 traits. In another study, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hayes <em>et al.</em> (2009a)</a> predicted the GEBV of Jersey animals using a Holstein population as training data and vice versa, resulting in accuracies ranging from −0.06 to 0.23 for five traits. Both studies report results that were very similar to ours that ranged from −0.05 to 0.26.</p><p id="p-54">The simulation study by <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">De Roos <em>et al.</em> (2009)</a> indicated that across-population prediction was substantially less accurate than within-population or multiple-population prediction. These lower accuracies were due to differences in marker–QTL LD phase between the populations. A marker may be in LD with QTL in a given population, but it is not necessarily in LD with those QTL in the other population, resulting in poor predictions for the other population. These simulation results suggested that, for our analyses, a higher marker density would be required. However, results of <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Veroneze <em>et al.</em> (2014)</a> show that with the same 60K porcine SNP panel, the density of SNPs is high enough to obtain reasonable levels of LD. This would predict that our SNP panel should be able to capture marker effects across breeds.</p></div><div id="sec-10" class="subsection"><h3>Using purebred training data to predict crossbred genetic merit</h3><p id="p-55">Using only the PB population(s) to predict the CB genetic merit with GBLUP has some predictive ability for TNB, LBW, and LVR, whereas all the accuracies for AFI were null. Increasing the size of the training data by adding another PB population increased the accuracy for TNB and LBW, whereas for AFI and LVR it did not. However, when we increased the size of the training population by adding more animals of the same PB population, the accuracies usually increased. The accuracy of prediction for predicting CB animals based on PB animals appears to depend largely on the genetic correlation between PB and CB performance. As our results demonstrate, the greater the genetic correlation, the higher the chances of having any or more predictive ability. AFI, for which the genetic correlation between PB and CB was very poor, had zero accuracy of prediction showing that selection on PB is expected to have no effect on CB genetic merit.</p><p id="p-56">For PED-BLUP, the accuracies were, in general, lower than for GBLUP, especially for LBW and LVR. Adding the second PB population in the training slightly increased the accuracy of prediction.</p><p id="p-57">The greater accuracies found for TNB, LBW, and LVR when training with DL rather than LW population can be explained by the slightly greater relationship between DL and F1 populations than between LW and F1. This higher relationship is specific for the animals included in this study. The F1 animals that were genotyped are more closely related to the DL animals that were genotyped than to the LW animals that were genotyped.</p><p id="p-58">To test the impact of the relationship between training and validation populations on the accuracy, we split the training data into the 50% of animals that are MOST related to the validation set and the 50% that are LEAST related to the validation set (<a href="http://www.g3journal.org/content/suppl/2015/05/26/g3.115.018119.DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/26/g3.115.018119.DC1/TableS4.pdf">Table S4</a>). For AFI, TNB, and LBW, using the 50% MOST related animals resulted in greater accuracies, whereas for LVR it did not. This indicates that if CB animals have closer genomic relationships to the PB animals used as training, then higher accuracies for scenarios 12–17 could generally be expected.</p><p id="p-59">In cattle, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Harris <em>et al.</em> (2008)</a> used PB populations (Holstein-Friesian and Jersey) to predict the genomic breeding values of a cross between these two breeds. They used data from 4500 sires genotyped for approximately 44K SNPs. Their results show that using the two breeds as the training population increased the accuracy by 5–10% compared to using only one of the breeds to predict the cross. The actual level of accuracy was not reported in their study. Our results were similar to theirs for TNB, LBW, and LVR, where the genetic correlation between PB and CB performance is close to 1, but not for AFI.</p><p id="p-60">Results indicate that using a PB population to predict CB genetic merit can generate reasonable predictions. This, however, is not consistent for all traits. Although these results do not reflect the actual practice of genomic selection in pig breeding, they do provide an estimate of the accuracy of genomic prediction between CB and PB populations using real data. The results make a strong case for the genotyping and recording of CB animals, at least for a subset of traits where genetic correlations are away from 1.</p><p id="p-61">The low genetic correlation between PB and CB performance for AFI was also found in another study (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nagyné-Kiszlinger <em>et al.</em> 2013</a>). They have reported values of 0.28 and 0.39 for Hungarian Large White and Hungarian Landrace with their reciprocal cross. Possible reasons for this low genetic correlation were reported: 1) genes that affect the trait might be different among populations; 2) this trait is affected by non-additive effects or environmental factors due to different management of PB and CB animals (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Nagyné-Kiszlinger <em>et al.</em> 2013</a>). One clear environmental factor that probably reduces the genetic correlation of AFI between PB and CB is the use of batch farrowing systems in the production environment of CB sows. Suppression of estrus is used to synchronize the heat of the CB gilts, which impacts the measurement of the trait and leads to these low correlations.</p><p id="p-62">Standardized EBVs were used; therefore, a bias would possibly be introduced during deregression due to different reliabilities between breeds (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Garrick <em>et al.</em> 2009</a>). Additional sources for potential bias affecting the SNP effect estimates are the differences in the population mean of the breeds. The differences in the mean between populations were remedied by reintroducing the line effect after deregression. To test the impact of deregression on bias, we investigated all 17 scenarios for the trait AFI by analyzing phenotypes, which are not standardized, instead of DEBVs. The correlation between the accuracies obtained by the two different approaches was 0.92, with a mean regression coefficient of the GEBV on the phenotype of 0.70. This correlation shows that using the phenotypes has good agreement with the accuracies calculated using DEBVs; therefore, any bias due to the process of standardization and deregression is expected to be limited.</p><p id="p-63">The reasonable accuracy for PB predicting CB genetic merit shows that in a current typical breeding program, selection in the PB does result in a phenotypic response in CB. AFI was an exception in our study, because the genetic correlation between PB and CB performance was very low.</p><p id="p-64">Further studies to compare the accuracy of genomic selection of PB for CB performance are needed. Other genomic models including breed-specific effects of SNP alleles or dominance (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ibánez-Escriche <em>et al.</em> 2009</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Zeng <em>et al.</em> 2013</a>) were described and were found to outperform an additive model only in specific cases, <em>e.g.</em>, with high dominance levels or when the number of SNPs is small relative to the size of the training population. Using these more complex models or a multiple-trait model (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christensen <em>et al.</em> 2014</a>) with real data is needed.</p><p id="p-65">In conclusion, there was predictive ability for purebred population(s) predicting crossbred genetic merit using an additive model in the populations studied when PB and CB traits have high genetic correlation. For practical implementation, estimation of genomic breeding values of PB animals for CB performance needs to be further studied with models that take into account the crossbred nature of training data. Multi-population prediction was no better than within-population prediction for PB populations. Accuracy of prediction was shown to be very trait-dependent; hence, for the utility of data from other breeds in the application of genomic selection, each trait needs to be studied separately and no generalizations should be made. Finally, real data accuracies were lower than what simulation studies have reported.</p></div></div><div class="section" id="sec-11"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-66">A.M.H. currently benefits from a joint grant from the European Commission (within the framework of the Erasmus-Mundus joint doctorate “EGS-ABG”), Breed4Food (a public-private partnership in the domain of animal breeding and genomics), and Topigs Norsvin. J.W.M.B. was supported by PPP Breed4Food. Topigs Norsvin had a role in study design, data collection, and preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-17"><p id="p-67">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018119/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018119/-/DC1</a></p></li><li class="fn" id="fn-18"><p id="p-68"><em>Communicating editor: G. A. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-20"><span class="received-label">Received </span>March 20, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-05-21"><span class="accepted-label">Accepted </span>May 21, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Hidalgo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y. M.</span></span></li></ol><cite>, R. O. Bates, C. W. Ernst, J. Fix, and J. P Steibel, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accuracy of estimation of genomic breeding values in pigs using low-density genotypes and imputation. G3 (Bethesda)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D4%26rft.spage%253D623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1575.2"><div class="cit-metadata unstructured">Berry, D. P., F. Kearney, and B. L. Harris. 2009 Genomic selection in Ireland. Interbull. Bull. 39:29–34.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.3" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, and B. L. Browning., <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.4" data-doi="10.2527/jas.2008-1744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cecchinato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carnier</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The relevance of purebred information for predicting genetic merit of survival at birth of crossbred piglets.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2008-1744%26rft_id%253Dinfo%253Apmid%252F19897636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2008-1744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19897636&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.5" data-doi="10.2527/jas.2013-5715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crews</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of predicting genomic breeding values for residual feed intake in Angus and Charolais beef cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4669</span>–<span class="cit-lpage">4678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DChen%26rft.auinit1%253DL.%26rft.volume%253D91%26rft.issue%253D10%26rft.spage%253D4669%26rft.epage%253D4678%26rft.atitle%253DAccuracy%2Bof%2Bpredicting%2Bgenomic%2Bbreeding%2Bvalues%2Bfor%2Bresidual%2Bfeed%2Bintake%2Bin%2BAngus%2Band%2BCharolais%2Bbeef%2Bcattle%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2013-5715%26rft_id%253Dinfo%253Apmid%252F24078618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2013-5715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24078618&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.6" data-doi="10.1017/S1751731112000742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Single-step methods for genomic evaluation in pigs.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1565</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253D1565%26rft.epage%253D1571%26rft.atitle%253DSingle-step%2Bmethods%2Bfor%2Bgenomic%2Bevaluation%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731112000742%26rft_id%253Dinfo%253Apmid%252F22717310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731112000742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22717310&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308833700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.7" data-doi="10.1186/1297-9686-46-23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic evaluation of both purebred and crossbred performances.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-46-23%26rft_id%253Dinfo%253Apmid%252F24666469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-46-23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24666469&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.8.1575.8"><div class="cit-metadata unstructured">Cleveland, M. A., S. Forni, D. J. Garrick, and N. Deeb, 2010 Prediction of genomic breeding values in a commercial pig population. In Proc. 9th WCGALP, 0266.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.9" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzgvMTU3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.10" data-doi="10.2527/jas.2006-683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Marker-assisted selection for commercial crossbred performance.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">2104</span>–<span class="cit-lpage">2114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2006-683%26rft_id%253Dinfo%253Apmid%252F17504955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2006-683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17504955&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.11" data-doi="10.1086/281736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1952</span> <span class="cit-article-title">The problem of environment and selection.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D86%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1086%252F281736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/281736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1952UY00500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.12" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.13" data-doi="10.1186/1297-9686-41-55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, J. F. Taylor, and R. L. Fernando, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deregressing estimated breeding values and weighting information for genomic regression analyses.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGarrick%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D41%26rft.spage%253D55%26rft.epage%253D55%26rft.atitle%253DDeregressing%2Bestimated%2Bbreeding%2Bvalues%2Band%2Bweighting%2Binformation%2Bfor%2Bgenomic%2Bregression%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-55%26rft_id%253Dinfo%253Apmid%252F20043827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043827&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-5.8.1575.14"><div class="cit-metadata unstructured">Gilmour, A. R., B. J. Gogel, B. R. Cullis, and R. Thompson, 2009 ASReml user guide release 3.0. VSN International Ltd, Hemel Hempstead, HP1 1ES, UK. <a href="http://www.vsni.co.uk">www.vsni.co.uk</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.15"><div class="cit-metadata"><cite>Groenen, M. A. M., A. L. Archibald, H. Uenishi, C. K. Tuggle, Y. Takeuchi, M. F. Rothschild, C. Rogel-Gaillard, <em>et al</em>., <span class="cit-pub-date">2012</span> Analyses of pig genomes provide insight into porcine demography and evolution. Nature 491: 393–8.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.8.1575.16"><div class="cit-metadata unstructured">Harris, B. L., D. L. Johnson, and R. J. Spelman, 2008 Genomic selection in New Zealand and the implications for national genetic evaluation. Proceedings of Interbull Meeting, Niagara Falls 325–330.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.17" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.18" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: Progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.19" data-doi="10.1186/1297-9686-41-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Ibánez-Escriche</span></li></ol><cite>, N., R. L. Fernando, A. Toosi, and J. C. M. Dekkers, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection of purebreds for crossbred performance.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DIbanez-Escriche%26rft.auinit1%253DN.%26rft.volume%253D41%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DGenomic%2Bselection%2Bof%2Bpurebreds%2Bfor%2Bcrossbred%2Bperformance.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-12%26rft_id%253Dinfo%253Apmid%252F19284703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284703&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.8.1575.20"><div class="cit-metadata unstructured">Kinghorn, B. P., J. M. Hickey, and J. H. J. Van Der Werf, 2010 Reciprocal recurrent genomic selection for total genetic merit in crossbred individuals. Proceedings of the 9th World Congress on Genetics applied to Livestock Production, Leipzig, Germany, 0036.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.21" data-doi="10.1016/j.livsci.2014.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic evaluation of cattle in a multi-breed context.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.05.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lutaaya</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mabry</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Short</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timm</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic parameter estimates from joint evaluation of purebreds and crossbreds in swine using the crossbred model.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">3002</span>–<span class="cit-lpage">3007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DLutaaya%26rft.auinit1%253DE.%26rft.volume%253D79%26rft.issue%253D12%26rft.spage%253D3002%26rft.epage%253D3007%26rft.atitle%253DGenetic%2Bparameter%2Bestimates%2Bfrom%2Bjoint%2Bevaluation%2Bof%2Bpurebreds%2Band%2Bcrossbreds%2Bin%2Bswine%2Busing%2Bthe%2Bcrossbred%2Bmodel%26rft_id%253Dinfo%253Apmid%252F11811453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11811453&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.8.1575.23"><div class="cit-metadata unstructured">Merks, J. W. M., and A. G. de Vries, 2002 New sources of information in pig breeding. Proceedings of the 7th World Congress on Genetics applied to Livestock Production, Montpellier, France, 30, 3–10.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-5.8.1575.24"><div class="cit-metadata unstructured">Mulder, H. A., M. Lidauer, I. Strandén, E. A. Mäntysaari, M. H. Pool, and R. F. Veerkamp, 2012  MiXBLUP Manual. Animal Breeding and Genomics Centre, Wageningen UR Livestock Research, Lelystad, the Netherlands.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagyné-Kiszlinger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farkas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kövér</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Selection for reproduction traits in Hungarian pig breeding in a two-way cross.</span> <abbr class="cit-jnl-abbrev">Anim. Sci. Pap. Rep.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BSci.%2BPap.%2BRep.%26rft.volume%253D31%26rft.spage%253D315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.26" data-doi="10.1186/1297-9686-43-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O. F. Christensen</span></span></li></ol><cite>, M. Henryon<span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deregressed EBV as the response variable yield more reliable genomic predictions than traditional EBV in pure-bred pigs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DOstersen%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.spage%253D38%26rft.epage%253D38%26rft.atitle%253DDeregressed%2BEBV%2Bas%2Bthe%2Bresponse%2Bvariable%2Byield%2Bmore%2Breliable%2Bgenomic%2Bpredictions%2Bthan%2Btraditional%2BEBV%2Bin%2Bpure-bred%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-38%26rft_id%253Dinfo%253Apmid%252F22070746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22070746&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.27" data-doi="10.1371/journal.pone.0005361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, R. P. M. A. Crooijmans, N. A. Affara, A. J. Amaral, A. L. Archibald, J. E. Beever, C. Bendixen, <em>et al</em>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D4%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005361%26rft_id%253Dinfo%253Apmid%252F19956663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19956663&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.28" data-doi="10.1111/j.1439-0388.2003.00444.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Approximating prediction error covariances among additive genetic effects within animals in multiple-trait and random regression models.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D121%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2003.00444.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2003.00444.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220353500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.29" data-doi="10.2527/jas.2009-1975"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toosi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in admixed and crossbred populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-1975%26rft_id%253Dinfo%253Apmid%252F19749023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-1975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19749023&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.30" data-doi="10.1017/S1751731113001389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tusell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodriguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-enabled methods for predicting litter size in pigs: A comparison.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1739</span>–<span class="cit-lpage">1749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D7%26rft.spage%253D1739%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731113001389%26rft_id%253Dinfo%253Apmid%252F23880322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731113001389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23880322&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.31" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.32" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Reliability of genomic predictions for North American Holstein bulls.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veroneze</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, J. W. M. Bastiaansen, E. F. Knol, S. E. F. Guimarães, F. F. Silva, B. Harlizius, M. S. Lopes, and P. S. Lopes, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage disequilibrium patterns and persistence of phase in purebred and crossbred pig (Sus Scrofa) Populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D126%26rft_id%253Dinfo%253Apmid%252F25421851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25421851&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.34"><div class="cit-metadata"><cite>Wientjes, Y. C. J., R. F. Veerkamp, and M. P. L. Calus, <span class="cit-pub-date">2013</span> The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction. Genetics 193: 621–31.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toosi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, R. L. Fernando, J. C. M. Dekkers, and D. J. Garrick, <span class="cit-pub-date">2013</span> Genomic selection of purebred animals for crossbred performance in the presence of dominant gene action. Genet. Sel. Evol. 45: 11.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.8.1575.36" data-doi="10.2527/jas.2006-499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zumbach</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herring</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic correlations between two strains of Durocs and crossbreds from differing production environments for slaughter traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2006-499%26rft_id%253Dinfo%253Apmid%252F17178815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2006-499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17178815&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F8%2F1575.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/8/1575.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/8/1567" title="SDN-1/Syndecan Acts in Parallel to the Transmembrane Molecule MIG-13 to Promote Anterior Neuroblast Migration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/8/1585" title="The Wukong Terminal-Repeat Retrotransposon in Miniature (TRIM) Elements in Diverse Maize Germplasm" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/8"><span>Volume 5 Issue 8, August 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/8.cover-source.jpg?itok=wjPUfMS9" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/8/1575.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5qU2kv_BPmcQzlLP-w4DHBIKQlwLTWv0oX0Fl3UPCs8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/8/1575" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/8/1575" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/8/1575" class="active" data-icon-position="" data-hide-link-title="0">Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459992" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dO0wULCn7AGq4ej83_k_ET8nbkoKSy7P5iIjQxN6nog" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1575.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/8/1575" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ATWumWKp-1vmeHJDrxeryjDv9_Yw7ib6Ycd_xnmR9Yc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/8/1575" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/8/1575.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459992" id="node459992--2" data-pisa="ggg;5/8/1575" data-pisa-master="ggg;g3.115.018119" data-apath="/ggg/5/8/1575.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5116-4628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>André M. Hidalgo</span>, <span class="highwire-citation-author" data-delta="1">John W. M. Bastiaansen</span>, <span class="highwire-citation-author" data-delta="2">Marcos S. Lopes</span>, <span class="highwire-citation-author" data-delta="3">Barbara Harlizius</span>, <span class="highwire-citation-author" data-delta="4">Martien A. M. Groenen</span> and <span class="highwire-citation-author" data-delta="5">Dirk-Jan de Koning</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1575-1583;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018119 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459992--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">André M. Hidalgo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andr%C3%A9%2BM.%2BHidalgo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hidalgo%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndr%25C3%25A9%2BM.%2BHidalgo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5116-4628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for André M. Hidalgo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andre.hidalgo@wur.nl" class="" data-icon-position="" data-hide-link-title="0">andre.hidalgo@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John W. M. Bastiaansen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BM.%2BBastiaansen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastiaansen%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BW.%2BM.%2BBastiaansen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcos S. Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BS.%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BS.%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Barbara Harlizius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BHarlizius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harlizius%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarbara%2BHarlizius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martien A. M. Groenen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martien%2BA.%2BM.%2BGroenen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Groenen%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartien%2BA.%2BM.%2BGroenen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dirk-Jan de Koning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk-Jan%2Bde%2BKoning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Koning%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADirk-Jan%2Bde%2BKoning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/8/1575" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/8/1575&amp;title=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/8/1575&amp;title=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/8/1575&amp;title=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/8/1575&amp;text=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/8/1575&amp;title=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/8/1575&amp;t=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/8/1575&amp;title=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/8/1575&amp;title=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459992/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459992" id="node459992--3" data-pisa="ggg;5/8/1575" data-pisa-master="ggg;g3.115.018119" data-apath="/ggg/5/8/1575.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accuracy of Predicted Genomic Breeding Values in Purebred and Crossbred Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5116-4628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>André M. Hidalgo</span>, <span class="highwire-citation-author" data-delta="1">John W. M. Bastiaansen</span>, <span class="highwire-citation-author" data-delta="2">Marcos S. Lopes</span>, <span class="highwire-citation-author" data-delta="3">Barbara Harlizius</span>, <span class="highwire-citation-author" data-delta="4">Martien A. M. Groenen</span> and <span class="highwire-citation-author" data-delta="5">Dirk-Jan de Koning</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1575-1583;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018119 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459992--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">André M. Hidalgo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andr%C3%A9%2BM.%2BHidalgo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hidalgo%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndr%25C3%25A9%2BM.%2BHidalgo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5116-4628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for André M. Hidalgo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andre.hidalgo@wur.nl" class="" data-icon-position="" data-hide-link-title="0">andre.hidalgo@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John W. M. Bastiaansen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BM.%2BBastiaansen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bastiaansen%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BW.%2BM.%2BBastiaansen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcos S. Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BS.%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BS.%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Barbara Harlizius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Topigs Norsvin Research Center, Beuningen, the Netherlands 6640 AA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BHarlizius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harlizius%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarbara%2BHarlizius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martien A. M. Groenen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Wageningen University, Animal Breeding and Genomics Centre, Wageningen, the Netherlands 6700 AH</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martien%2BA.%2BM.%2BGroenen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Groenen%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartien%2BA.%2BM.%2BGroenen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dirk-Jan de Koning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Swedish University of Agricultural Sciences, Department of Animal Breeding and Genetics, Uppsala, Sweden 750 07</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk-Jan%2Bde%2BKoning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Koning%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADirk-Jan%2Bde%2BKoning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1575&amp;rft.epage=1583&amp;rft.atitle=Accuracy%20of%20Predicted%20Genomic%20Breeding%20Values%20in%20Purebred%20and%20Crossbred%20Pigs&amp;rft.volume=5&amp;rft.issue=8&amp;rft.date=2015-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hidalgo%2C+Andr%C3%A9+M.&amp;rft.au=Bastiaansen%2C+John+W.+M.&amp;rft.au=Lopes%2C+Marcos+S.&amp;rft.au=Harlizius%2C+Barbara&amp;rft.au=Groenen%2C+Martien+A.+M.&amp;rft.au=de+Koning%2C+Dirk-Jan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459992/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459992/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459992/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459992/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459992/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459992/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459992/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459992/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459992/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459992/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459992/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459992/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/8/1575" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/8/1575.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/8/1575.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/8/1575.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/8/1575.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
