<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/1/133" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/1/133.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/1/133.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/1/133.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/1/133.atom" />
<meta name="HW.pisa" content="ggg;5/1/133" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (Manihot esculenta Crantz) from 10 Populations" />
<meta name="DC.Identifier" content="10.1534/g3.114.015008" />
<meta name="DC.Date" content="2015-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 International Cassava Genetic Map Consortium (ICGMC). This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cassava ( Manihot esculenta Crantz) is a major staple crop in Africa, Asia, and South America, and its starchy roots provide nourishment for 800 million people worldwide. Although native to South America, cassava was brought to Africa 400–500 years ago and is now widely cultivated across sub-Saharan Africa, but it is subject to biotic and abiotic stresses. To assist in the rapid identification of markers for pathogen resistance and crop traits, and to accelerate breeding programs, we generated a framework map for M. esculenta Crantz from reduced representation sequencing [genotyping-by-sequencing (GBS)]. The composite 2412-cM map integrates 10 biparental maps (comprising 3480 meioses) and organizes 22,403 genetic markers on 18 chromosomes, in agreement with the observed karyotype. We used the map to anchor 71.9% of the draft genome assembly and 90.7% of the predicted protein-coding genes. The chromosome-anchored genome sequence will be useful for breeding improvement by assisting in the rapid identification of markers linked to important traits, and in providing a framework for genomic selection-enhanced breeding of this important crop." />
<meta name="DC.Contributor" content="International Cassava Genetic Map Consortium (ICGMC)" />
<meta name="article:published_time" content="2015-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (Manihot esculenta Crantz) from 10 Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cassava (&lt;i&gt;Manihot esculenta&lt;/i&gt; Crantz) is a major staple crop in Africa, Asia, and South America, and its starchy roots provide nourishment for 800 million people worldwide. Although native to South America, cassava was brought to Africa 400–500 years ago and is now widely cultivated across sub-Saharan Africa, but it is subject to biotic and abiotic stresses. To assist in the rapid identification of markers for pathogen resistance and crop traits, and to accelerate breeding programs, we generated a framework map for &lt;i&gt;M. esculenta&lt;/i&gt; Crantz from reduced representation sequencing [genotyping-by-sequencing (GBS)]. The composite 2412-cM map integrates 10 biparental maps (comprising 3480 meioses) and organizes 22,403 genetic markers on 18 chromosomes, in agreement with the observed karyotype. We used the map to anchor 71.9% of the draft genome assembly and 90.7% of the predicted protein-coding genes. The chromosome-anchored genome sequence will be useful for breeding improvement by assisting in the rapid identification of markers linked to important traits, and in providing a framework for genomic selection-enhanced breeding of this important crop.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/01/01" />
<meta name="citation_mjid" content="ggg;5/1/133" />
<meta name="citation_id" content="5/1/133" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/1/133" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/1/133.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/1/133.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/1/133.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015008" />
<meta name="citation_pmid" content="25504737" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="133" />
<meta name="citation_lastpage" content="144" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="International Cassava Genetic Map Consortium (ICGMC)" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Altshuler;citation_author=V. J. Pollara;citation_author=C. R. Cowles;citation_author=W. J. Van Etten;citation_author=J. Baldwin;citation_title=An SNP map of the human genome generated by reduced representation shotgun sequencing.;citation_pages=513-516;citation_volume=407;citation_year=2000;citation_issue=6803;citation_pmid=11029002;citation_doi=10.1038/35035083" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=F. Awoleye;citation_author=M. Duren;citation_author=J. Dolezel;citation_author=F. J. Novak;citation_title=NUCLEAR DNA CONTENT AND IN VITRO INDUCED SOMATIC POLYPLOIDIZATION IN CASSAVA (MANIHOT ESCULENTA CRANTZ) BREEDING;citation_pages=195-202;citation_volume=76;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Bohra;citation_author=R. K. Saxena;citation_author=B. N. Gnanesh;citation_author=K. Saxena;citation_author=M. Byregowda;citation_title=An intra-specific consensus genetic map of pigeonpea [Cajanus cajan (L.) Millspaugh] derived from six mapping populations.;citation_pages=1325-1338;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22772726;citation_doi=10.1007/s00122-012-1916-5" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=R. de Carvalho;citation_author=G. Guerra;citation_title=Cytogenetics of Manihot esculenta Crantz (cassava) and eight related species.;citation_pages=159-168;citation_volume=136;citation_year=2002;citation_issue=2;citation_pmid=12369103;citation_doi=10.1034/j.1601-5223.2002.1360212.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Rep.;citation_author=S. L. Dellaporta;citation_author=J. J. Wood;citation_author=J. B. Hicks;citation_title=A plant DNA minipreparation: version II.;citation_pages=19-21;citation_volume=1;citation_year=1983;citation_doi=10.1007/BF02712670" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. B. Endelman;citation_author=C. Plomion;citation_title=LPmerge: an R package for merging genetic maps by linear programming.;citation_volume=30;citation_year=2014;citation_pmid=24532720;citation_doi=10.1093/bioinformatics/btu091" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Fregene;citation_author=F. Angel;citation_author=R. Gomez;citation_author=F. Rodriguez;citation_author=P. Chavarriaga-Aguirre;citation_title=A molecular genetic map of cassava (Manihot esculenta Crantz).;citation_pages=431-441;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050580" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Gautami;citation_author=D. Fonceka;citation_author=M. K. Pandey;citation_author=M. C. Moretzsohn;citation_author=V. Sujay;citation_title=An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (Arachis hypogaea L.).;citation_pages=e41213-e41213;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22815973;citation_doi=10.1371/journal.pone.0041213" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. T. Hamblin;citation_author=I. Y. Rabbi;citation_title=The effects of restriction-enzyme choice on properties of genotyping-by-sequencing libraries: A study in cassava (Manihot esculenta).;citation_pages=1-6;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2012.12.0710" />
<meta name="citation_reference" content="citation_title=Save and Grow: Cassava;citation_author=R. Howeler;citation_author=N. Lutaladio;citation_author=G. Thomas;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Hybridization of Crop Plants;citation_title=Cassava;citation_author=K. Kawano;citation_pages=225-233;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Plant Genetic Resources;citation_author=R. S. Kawuki;citation_author=L. Herselman;citation_author=M. T. Labuschagne;citation_author=I. Nzuki;citation_author=I. Ralimanana;citation_title=Genetic diversity of cassava (Manihot esculenta Crantz) landraces and cultivars from southern, eastern and central Africa.;citation_pages=170-181;citation_volume=11;citation_year=2013;citation_doi=10.1017/S1479262113000014" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=S. Kunkeaw;citation_author=S. Tangphatsornruang;citation_author=D. R. Smith;citation_author=K. Triwitayakorn;citation_title=Genetic linkage map of cassava (Manihot esculenta Crantz) based on AFLP and SSR markers.;citation_pages=112-115;citation_volume=129;citation_year=2010;citation_doi=10.1111/j.1439-0523.2009.01623.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=N. J. Lennon;citation_author=R. E. Lintner;citation_author=S. Anderson;citation_author=P. Alvarez;citation_author=A. Barry;citation_title=A scalable, fully automated process for construction of sequence-ready barcoded libraries for 454.;citation_pages=R15-R15;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20137071;citation_doi=10.1186/gb-2010-11-2-r15" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=E. S. Mace;citation_author=J. F. Rami;citation_author=S. Bouchet;citation_author=P. E. Klein;citation_author=R. R. Klein;citation_title=A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers.;citation_pages=13-13;citation_volume=9;citation_year=2009;citation_pmid=19171067;citation_doi=10.1186/1471-2229-9-13" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Manichaikul;citation_author=J. C. Mychaleckyj;citation_author=S. S. Rich;citation_author=K. Daly;citation_author=M. Sale;citation_title=Robust relationship inference in genome-wide association studies.;citation_volume=26;citation_year=2010;citation_pmid=20926424;citation_doi=10.1093/bioinformatics/btq559" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. Okogbenin;citation_author=M. Marin;citation_author=M. Fregene;citation_title=An SSR-based molecular genetic map of cassava.;citation_pages=433-440;citation_volume=147;citation_year=2006;citation_doi=10.1007/s10681-005-9042-y" />
<meta name="citation_reference" content="citation_journal_abbrev=Trop Plant Biol;citation_author=S. Prochnik;citation_author=P. R. Marri;citation_author=B. Desany;citation_author=P. D. Rabinowicz;citation_author=C. Kodira;citation_title=The Cassava Genome: Current Progress, Future Directions.;citation_pages=88-94;citation_volume=5;citation_year=2012;citation_issue=1;citation_pmid=22523606;citation_doi=10.1007/s12042-011-9088-z" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. A. Quail;citation_author=I. Kozarewa;citation_author=F. Smith;citation_author=A. Scally;citation_author=P. J. Stephens;citation_title=A large genome center's improvements to the Illumina sequencing system.;citation_pages=1005-1010;citation_volume=5;citation_year=2008;citation_issue=12;citation_pmid=19034268;citation_doi=10.1038/nmeth.1270" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=I. Rabbi;citation_author=M. Hamblin;citation_author=M. Gedil;citation_author=P. Kulakow;citation_author=M. Ferguson;citation_title=Genetic Mapping Using Genotyping-by-Sequencing in the Clonally Propagated Cassava.;citation_pages=1384-1396;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.07.0482" />
<meta name="citation_reference" content="citation_journal_title=Virus Res.;citation_author=I. Y. Rabbi;citation_author=M. T. Hamblin;citation_author=P. L. Kumar;citation_author=M. A. Gedil;citation_author=A. S. Ikpan;citation_title=High-resolution mapping of resistance to cassava mosaic geminiviruses in cassava using genotyping-by-sequencing and its implications for breeding.;citation_pages=87-96;citation_volume=186;citation_year=2014;citation_pmid=24389096;citation_doi=10.1016/j.virusres.2013.12.028" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Y. Rabbi;citation_author=H. P. Kulembeka;citation_author=E. Masumba;citation_author=P. R. Marri;citation_author=M. Ferguson;citation_title=An EST-derived SNP and SSR genetic linkage map of cassava (Manihot esculenta Crantz).;citation_pages=329-342;citation_volume=125;citation_year=2012;citation_issue=2;citation_pmid=22419105;citation_doi=10.1007/s00122-012-1836-4" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Sraphet;citation_author=A. Boonchanawiwat;citation_author=T. Thanyasiriwat;citation_author=O. Boonseng;citation_author=S. Tabata;citation_title=SSR and EST-SSR-based genetic linkage map of cassava (Manihot esculenta Crantz).;citation_pages=1161-1170;citation_volume=122;citation_year=2011;citation_issue=6;citation_pmid=21222095;citation_doi=10.1007/s00122-010-1520-5" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. W. van Ooijen;citation_title=Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.;citation_pages=343-349;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21878144;citation_doi=10.1017/S0016672311000279" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (Manihot esculenta Crantz) from 10 Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
<meta name="twitter:description" content="Cassava ( Manihot esculenta Crantz) is a major staple crop in Africa, Asia, and South America, and its starchy roots provide nourishment for 800 million people worldwide. Although native to South America, cassava was brought to Africa 400–500 years ago and is now widely cultivated across sub-Saharan Africa, but it is subject to biotic and abiotic stresses. To assist in the rapid identification of markers for pathogen resistance and crop traits, and to accelerate breeding programs, we generated a framework map for M. esculenta Crantz from reduced representation sequencing [genotyping-by-sequencing (GBS)]. The composite 2412-cM map integrates 10 biparental maps (comprising 3480 meioses) and organizes 22,403 genetic markers on 18 chromosomes, in agreement with the observed karyotype. We used the map to anchor 71.9% of the draft genome assembly and 90.7% of the predicted protein-coding genes. The chromosome-anchored genome sequence will be useful for breeding improvement by assisting in the rapid identification of markers linked to important traits, and in providing a framework for genomic selection-enhanced breeding of this important crop." />
<meta name="og-title" property="og:title" content="High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (Manihot esculenta Crantz) from 10 Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/1/133" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cassava ( Manihot esculenta Crantz) is a major staple crop in Africa, Asia, and South America, and its starchy roots provide nourishment for 800 million people worldwide. Although native to South America, cassava was brought to Africa 400–500 years ago and is now widely cultivated across sub-Saharan Africa, but it is subject to biotic and abiotic stresses. To assist in the rapid identification of markers for pathogen resistance and crop traits, and to accelerate breeding programs, we generated a framework map for M. esculenta Crantz from reduced representation sequencing [genotyping-by-sequencing (GBS)]. The composite 2412-cM map integrates 10 biparental maps (comprising 3480 meioses) and organizes 22,403 genetic markers on 18 chromosomes, in agreement with the observed karyotype. We used the map to anchor 71.9% of the draft genome assembly and 90.7% of the predicted protein-coding genes. The chromosome-anchored genome sequence will be useful for breeding improvement by assisting in the rapid identification of markers linked to important traits, and in providing a framework for genomic selection-enhanced breeding of this important crop." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
    <title>High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (Manihot esculenta Crantz) from 10 Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_U8L2s0Z-76EjSi4ZOUAQajcbAo48DMaNa0KPL1j6ME","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459764","apath":"\/ggg\/5\/1\/133.atom","pisa":"ggg;5\/1\/133","ac":{"\/ggg\/5\/1\/133.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/1\/133.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/1\/133"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/1\/133"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/1\/133":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224DW1GNkkN6FnVGe1wDg4ptQOUJWrAr8ggnF1hrptcZpghg2r6ixJgmox2aJSlH73Vpe57LHkxhnScrf868OHpFSlhX1A3LZZzjG3xOmAZ4h4zZxeN3fK3B1c7OrCoofYW31HuUlx3ZKnFOsnEMNPxBkPZ6fuergUwElZixi2ybN8VQjwAl+Tjo7U6S893TUe9qAFhhBUpFoYl4KPpxGHoV\\\\\\\/KZdfLSH1NaXkIGcUv1VwRGUCwdhklXchFah0MLEdVqYOMWsy5w4kIzKENowI+rNVir2nVetx5cH9F\\\\\\\/itwQyPxnMWEa5D5Ms6rzwhPEH2BVWuxYhEYrxgZSO4Uwpw+hx3LEoOrOfs8HiZ72gMtWeCkEfRZw+I2ZXlAKdoOjBZlGe+PAQHrwj3NRq7GGHN5LFv0pqmnZ+VhXv+B+1GMJm+H4aULe\\\\\\\/X1Z\\\\\\\/jTJAR3pBgqB8GpH0d0KKrC1jX0zLHNh2G7kViYS0FmlrVxlzTTXCFzNOUZh27codXmz+mAfMqocwjmcazBqj\\\\\\\/SuZmVT+7ohtzcQXLU07LbyRx\\\\\\\/+Ei4KAzZ6PmtmqEhOXj\\\\\\\/flIqbcfTq6NUdNNXFzFougL87L+qF1d9kyiAZr8vpU+mC3Ofv9mRej9L8OBqystpCunbVe60Q9hIYxf6f2+jMZLS+0tP1B6Amtw5zqoL1SKN6wSW+5\\\\\\\/6QX7lFPi69ouR3gwYgOwZd9iQK8rHNnig5dGvYX6gO0aG+IpdwYdCWiH4ZMOzm0WUYnbKwGK4f9BGtSMBUuO+iyTFBs69LPVMI3NClRVuOQtJ5IyyUJhNkqIhYbwyQWH\\\\\\\/AgpR02fvEEJsZjNEcm1M29eqsdgQraFWcaYvtyuWDGU7oJ50N6vVQMVLfaAjodvKY8mbdvw+nQ0G9cFYRaEiGtnF24Dr2Iz11mGR8EqwEQBvOz5SRsyNce2Xr\\\\\\\/gA5e8W6XKdnxkNU0qFWA\\\\\\\/dpNA3\\\\\\\/aikjNXK86qELNtBvR9AJbIhi06H4\\\\\\\/n2heM8kmdsMcQk1LH5\\\\\\\/PkYmoHtoBRANtzdd79thDnXP3sJQrStzQ9Dfp6X3BXIXPUJXbNFHDtou+rsLLb0J1R29aO3eEGf2oJSBQqKr75oXikM3rcKeEr0X8zyrYcUekeuSC+3rvjwbM3Iqrte9zhkDHbRS619xHAzzoNCfr1PTwDxF9jlAzqJaEkIuLSQsAvbMB09arD0GzE5F7KbaUCYTaid09TvhUB8GPqWMqRRBhK4dKAxch1HJ7nxgV7rimPODPxyIxR9doYv9ShxCU\\\\\\\/WU608UGsMUeU1\\\u0022,\\\u0022iv\\\u0022:\\\u0022v7+5IcaELWO1DGgRRzguMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223108e8db8c7b27e99a0fb21528dbdaea1a02c833321d48ce660c8fb65d94dd1e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZxHRGUidtYdTjWc66UtcAlbWQc2+kBqGZJpBnyG1uTMUNVkW10WAyZ+D2mZC1NnRPVfrpkoAKhTSUqHi1EB6HG\\\\\\\/7wmyeUozZD4oOm3U5LmXr7W7ngQjsEnsX+Y4m4xdvfiGGrbTjm8+w2pZO08OvNQvL\\\\\\\/+LedWI0TbylMd4hshE3XGq81881cI9iVbkAYLAcJa6HyuFmgVdvsRfduLJ6o+QAh+ZYLPJGWBHh6FdrI86wg6bJq1S8keEfKZfM7BX7H+GxUZBP5UYJ\\\\\\\/wCdi1oUWmCmfXKTXM4S6c\\\\\\\/DmsZhJURYSBiydk0y7wgJX5PYlK0UBHjL9AzTI6PnaDPXJlpwx7lCM6gUH5BdLAusytiAyVR\\\\\\\/7QwzjA8e9efQ0OOdqGCNoeBrx41oKx2dwyni5M9oQwQkbwoEcUALRCeFSmodKnOCI3qauioBaXu3NaNhMxcI1TEL1VJ2zOjwJKwRGPWqrbtzHNW4RGGMQk4k9KH4f4fSXf7QzddLEntprWgHfcL+gacSYv5po\\\\\\\/\\\\\\\/5G\\\\\\\/WUwZWWVa7feNUHNjqOME7mFU4XG\\\\\\\/XY8Tb2xsiepR3QWxujCNOZHZ3yqb1I51hOz2mRt6wbSi\\\\\\\/pHakmzN\\\\\\\/5AjkJ6+UcQmAo\\\\\\\/FQimu8KlKL+lejic+7GKqekUNDAuM0Drm07eiKv2cD58mzQslemG8oDF9A6EhVdkODnTCdfkAF1dIOFiMjKGTQ8mUfeqKUpWzRwJCKd5lphMAze+CPMaQLmVMB2N8\\\\\\\/mCg6MbHDTZS\\\\\\\/T\\\\\\\/grsBfy6c0K1JSJ\\\\\\\/muaN0xrH3CnAOLwEtHJNXVwEJkOf1\\\\\\\/Bf37DyxFglPntHaQvl\\\\\\\/CbZdE\\\\\\\/J223MKASJK8pYtZkymhydch9qB0i6\\\\\\\/WI+Sq5RAV+QYsTj0vk8tEwIOJTo7gJ1Khtc8A\\\\\\\/QsdbPYMowgArz2g53Vd9hoVdVVKqT1I5h8CnQWXC4ZEMBxKVWdMks4ag0DidgVfWxSqiNS2tKuRfljxKL7rtb9kfnYax6afxoASg7oEaACiKvYUxX1nEVRwVn+TPr2uWAV2Xl8mPz9aK4xcfIjzouyccQccAlSXxVn3uRXHEQbG+AeUEgT3BD3nApeEOWunlAPbzxY5vAxjkxext8QnhrEp7JGYZvBEiBbyoHWgwtKd1ssmrcc1hWJA4GlIyOjEopGNYZoJC1kOU7gZ92dsIG80dhZY9FSrT6wfTNQPfY7btwF93MeocX7Z7mOg73XCRfDP9Dzc7kvvSX8OaVnqx6ONTwIcrn+N2nBfWv5QPqUk+TP58pvK1PEQOSO1Pi2ee8oJLwy423Ga1XgmlpTg5E9D\\\\\\\/RnCObG6Csaz1rlWJUi+cC4fzTgo6V9qpyB80OAhhzhGjNpKRXTGTgOjR9kR\\\\\\\/PN0qQgKzuybkuWa+Tuouf65u3JVYAlCgy4uRBQ3Sm7Ak453NQ0G1nY3+y2hxhHK4qAQRKzi\\\\\\\/6l\\\\\\\/jE2sTj68Emv7Rbw+tTYyBu1fiO41NijrPzuNffa\\\\\\\/ml+5At90G5sLeVA4BR1lPFdOXRNEPtHbVYpTBo2W5DoSyyIxWDfvNcz6PG5X\\\\\\\/64NDE63zx3lDV+tt7BSvt6XvMe4buCJK4ol2RNNKfMXPr534RTObIlZBFjyTru2Sg1fUQrqWbwwASQ+gtB4gy5twpqLf\\\\\\\/PgyetxAgwTmQ6BkmBtWiXrCp8UJGYvbgxHf\\\\\\\/x8RJGRUmpdxq\\\\\\\/dd5hwMbaW3g2LuF67OUWessx4kwXjShDKVVjbDjSBfpUBULV7lJxADPJHasySEYTyZyl4\\\\\\\/mPqeO9eO\\\\\\\/ejbTqi4qTyRUMKE63qcCtvne\\\\\\\/CQkVFpz1pjEg4NPxrzC4sUzOltqjGKtH8x8LG81faEOZqeJ0MNlt5+uxCJzl+Wl018xCH6t9gcFRvpyycWj+BD6AKLHW8h1P8Rnvt83UCuobM2uaZJquvit8hWzx4IfXaj1KkLnqeqKY\\\\\\\/J5ulkOHcTG4KoPRgGQxCSRs8pOJ3lWX0\\\\\\\/GpsIrRYRV0z7lMAmXy3XZLimdpFl4jFcYjgcVmVCsRqHYChL27ESHOnfSA1zz7cKg3zlSjos2k2JMjFdwSvlyvkTm6nKfa75igNjFHoe+XoNeoPrKsjyUiVlkBXO+fnSCNldbULqRptqoS8J3WYQ2Y6BOwSJ0B7h9ZdRjpQQG7qN+nbts6swst+hubLTCubkf\\\\\\\/vzcpBTp\\\\\\\/j4FVLWJXWprpmqshWhlZaqMk4F36JNRP2r9bwIvmQ1iOlVSIsK+HCYPBf+ld1iaEDTsUxlS9y8MSWztLHW24n4r0KvESZ3uRShXsFkhn2mFt3p71Z2FENk\\\\\\\/0NogL5mgaUvB++avrT0ymGFV7Pqyoopkh8DVNMYnRrWr99xHav5dbue6lHbwm8S5Ba74zkcbvKy7UF\\\\\\\/N0j6HPfWlzOo1m\\\\\\\/C\\\\\\\/Vb+kMxpVh1twWK2L5cqFBH5Rl\\\\\\\/p01WjPlnPgyUe8\\\\\\\/noOZlmo\\\\\\\/NXl1DQeBTxlp\\\\\\\/iTLtc7rpoD8+qCJ0uc6WbKwAoZADwIjUJfxMYwUYT43eQRe5rUVnakVqAbjQ7uub5vdXhdyyvgRrPX0EmA5hwSJQXU7W7CU0Oq8VEg8man1oXwdW5rZP+u\\\\\\\/9q2lZFW\\\\\\\/hTXuFIPXi7+tfb3JsLZPltNAc8hdcL\\\\\\\/nBE\\\\\\\/N4vX5j0vpz0OzepK3iZNhgW8OznDPjyXjTjNk6sXPfqYEJsK59hYqU9lhW95gthg\\\\\\\/ykXfiBMWdh9G+KIrXsz6GyAUJa7QAso+oKCxZsvfNo4y3eX6A80yU3jLkp7yTNeMCOIkz56nCAOoNftK0rMMIEx9KXmb+n3g1KayZEgwMKlVOTASW98zPaJxU325AQVz4Sjd8rPP+p8rn4emMhCaTRI74NcMwcfNYZFqQ7845YZRexj+lU\\\\\\\/ZPlUG7pAMo9LsgEfyiZL1CPSTxEX9lHL4DuWS1blhzx2j8djDYvanSPxwDQgAiqtyhcn51kPKl+YFgZVJJ9XkwaJTuiw3vPW6zbsirhpu8gm+8pMiyjzTXFjtZ3qrs13wO\\\\\\\/3pbKHukAyDzdpd+ubUI6NZvHN1\\\\\\\/5I1fknfsyTGweXauymv87ICy2WtnZgBqfECgJ9n6TyKetLl94g8SzkLP4Z4A72PvyYoCBX9E9AQiqNOCZw\\\\\\\/\\\\\\\/tRR1kViKHyDg5y4fp\\\\\\\/efwPCv2+rDS3Bj2iM\\\\\\\/jFhLPjkSgZHP8E6FReK0H4QtSQVLg62IVjdB\\\\\\\/t5A6aUG5kypywEj4RkSTUoTGuToynilQQaIsqdKKIbRFc08QunjGjDuWYf8OLaib3jCJKCELWzpuI\\\\\\\/9PufF4fZpJm2SQie7riMMfmhC4RofoZOSqH6Bc8\\\\\\\/dL9xkLTY+jD6YjQJCRQMJn9RBf4SuVg016e92NeuLotKO3r3U+Lh3P5XiQ0uLX205icqXNJOsSpIQU8TxaADdWO0gwxdoZb8bDGINwvTqeHiPsjCeXsIgxxbzd\\\\\\\/y\\\\\\\/dn5cvjvYZedBI\\\\\\\/MiJSC28LiDdolSUxuRZ4nIq\\\\\\\/YocoE6i\\\\\\\/WNxWkJiDeFK402zfW71RKlwtHtZ8d6lzkIafJlNec5c+IPajZapWjuRyIRESkT68yaVslpsoMuRlu0Vkpbof0Oreu2S5r6\\\\\\\/6KkA9RpZuv6yvgRjkouhSeCQtL1W9jS9TZnH+CnJ0PcQ5K82J1evMeMlDUUYuOUC7m85WTGLzxpK5ZvMifLaC566WDqNHUOgYmSSSQ6ZM64ajxbY6pLkuAVBZUQ5tkdFJJQVYYvFyRmOEQVBGuYwWrvOcG8+QA6ChC7t04xxGwd2HH0E5thDnc\\\\\\\/dfHm9ivfO+PcHuJoT2+HFWUjM2zIhzW7uNahZM9j\\\\\\\/ws0BwnMtj7of8E6myazKi37u74tsonJA3LR+T7n4VKhXOcWjLVk31lkFZmFjAHXN9gWyT9Z5SBqmI7Kw3BFeADfm0N4woxG\\\\\\\/EeUeahK7NFZaeXYe5gFX\\\\\\\/CQHqrYRtmCOL\\\\\\\/1P6Bwfkw0cRHIoCuLoO6BetO5LX3lpUKb7STLr9tO3UYaWDEe0BgzppofSFDSUM1BlUeB8htXDQMjVuUgJWNRRnyeUs48TSRk7SEG3RoFpl6MR0f+JBXwiGpnDcyeU4PaO7NsIYHtJEracNOaQcAvCk6WcUKf49yDgKhwPUolYpc43Htfh0h2lQT3fRzksKYKwPK8LdRY+EvOymYtSx9zmrEb7trg3M1iWx06b43ndinc59T7+1drvW3WzsaOVI91IOGTJBSmCVhc08xYfHT6ybmL5+lrvUyznBquAsTkaqhlnIH7Rf45dtg7gF0a+\\\\\\\/W3JwgN5dM0ChHn02ub1eXv9xrpCrvTbkaP0GtrKhcOkmoV3wKvAT+Ibn1psxP+9+w4lieQ1pFYAF0aB0aLqAezhTaRQwfc3h6heOnLP9qWJ253KQhzZAnTrZWIuxRfaz8RFmgElFUZoSiY5a82h7OyTyfadEfwZ4vAWwh5qzcT1ZLVhrTWAV7opb8w\\\\\\\/KzSFsWWs0rCzs570q7rF3ZX+GIfd2hryg97LzWY0FcLO\\\\\\\/gb1FNdAq4MKxQEBdG\\\\\\\/OTYly9b+vmIldOM3jgijwBej3rIwWJcY5iM2UEGAXLVB+dUFBPQI3QzPz0kwVNIseuS8x+E9WWDPLIEGGEBYhldQrUP\\\\\\\/9aM85fi+c4JqUWxG1v5eMKxcKiL1dUlPwrdTytZI8JvphX4Yf71R6+NU2fyVd5V8SLTSgH9+c4NDN348gBxwzCX\\\\\\\/fapSEOhnmGFVQt3V\\\\\\\/h33P36wanLfkq77cFMWopWupPen1VpAf\\\\\\\/nWAv0U6T007R8JKa3\\\\\\\/q+C31s2gIDaFV5XeunbqADUjL18xEdu0V57j7g+SHI+GHa9ywVX6a5pc5bxgUNkGFwl8FYi8VLu6hrHpocvxX\\\\\\\/lHXI4WCG4rP\\\\\\\/lcZ0IcLEj8CaLhbF9PLlC9wIg0g74ENquN3MelAWtXRY3n2i9pGKxWlD5iAMiw8XO+\\\\\\\/pHyePKY\\\\\\\/A48vxQ0MEeWP6H5AK210KxUlH7szFVk4jD6E\\\\\\\/3cUJXJVC6htmweoT0Z+u0X7PeM7cJNkeFJvRNdP9Bc+g3rK6yMRJcYoIMXHGttV7kdqmPWDwOGhSqbfhWS4PzXBXXylBvvPLgUX0Wkeed2O0TwhPul4Tzbtp3Ks44DOz5HS6ItnqwlYOLWT\\\\\\\/fRBQf0\\\\\\\/ZVswg99DmPsYkRKUbVGIPF4yvccRomrvmeHt8qzT8Y2oJrTuQo8mghVHRBHPMldkWNVXYSBZ6OX9U\\\\\\\/lxBAnjOuveJvvEmUfVrma41BDkadcX0FHOH5yy8LdkngbQPqAyYXW1mtI+Gdr2a4HIauB9HPFHtIqX6DuVB6FGuJi1lHmB9m+fvmknsWZNV6YbqlzTcpUOcBLM7gx8NBBtR2O6mFDoDBnkqfwrrg08uzFgaofgDnlxRj+OGYd+7ycE\\\\\\\/sGcNiueJG40+zKnOZq4gW0tGe8PUPeiThi8kLYbdGvv2RWbE\\\\\\\/FSm5k5T1wRmbz3F1\\\\\\\/HtQMigtQzs7N7pjotGz37cSV7edSX4S109vGnJq+a97EVjCf1rjAM\\\\\\\/HKJL7uUeEbJXihNjgJlc9urhnH5Ec0ako9XQjxmCkd7dBo34AwGcOvKVk8FseHSCo4xylMGgygFW2kILW+Ksv+orTxr\\\\\\\/8UkYKvkxhaujrHyerPBoJRYF7m7IELYx0bAoPGk0kgtdABfHb9DGfUD\\\\\\\/7YsvsvHYYs\\\\\\\/zh\\\\\\\/6JELm7FGv92Q8W9wbNyvU988z0dnMLU8F9+Xn6EPOZXTZqHbLTPVSHDccqUv8BSHXws+s+oE7KjGh70\\\\\\\/qbs+G8KKAm9b3faWITJFqHTYl00DsUFdgW+SVlrd9MlbJEUk+tv94hR2G2o0zKq+TvwLhxRXqWLQcFlvTpFPJvU4ok9EN5yQdsAakrb2qTTdPqqoyUPkPcs17fWKW0lpx4yiuydZSBPaYlX3FqGmhm9o+ML8MbgXxHQYxzEeJ0G2I6Qj1kSb4zlB769hUsJxH0Ls1tddRZGmiMG8b5+uWzhDM0damIhdI3PDTIvQfEy01GRPQYEYkrwECMvNpq8iZGUNzZPlpmah3dFNOPIR+ms+F7Fxrur3rnIJ+hsaitMhkUPg4ZrFd8Gs8V9gpgCreR+Vi\\\\\\\/2ft+YhL0sOb3Xe+eE3PCt18Y5Km4w+bJ7RxgNgd+KtLIVTN8FoqXtEhxYp0JgwJVPd4P3G3V+A7A4U0H6Sbl7YZt2M89fQRnfKhDGdEInNqyjzuCxW8os96RX4sZRtOAdpkoNb4Vy4hkM6cVXipIwHU9WtS1H5XKp298koEZtACwmFgiiSWnOC04YrkCyn6VZSRov96go5HqOyP6zPfuh\\\\\\\/Ztjmmy\\\\\\\/SXS1xm+COvbIAVmU50+TPc\\\\\\\/f3A0bquoFIZwbgcDKEr3ZGRJ\\\\\\\/gSVVEn9LeowcRj9Zq0I7NL\\\\\\\/1NkAvzKGlGJvoMYSoOUVzL5hMiKuYdDJ8QMD21NNXgfmhPSeS4bvyl1Cr47qfletfilZIUXjTrqn+\\\\\\\/2LjvyaZ7mUP2orFJaRrTMKb0KDKPXaXT8ziTvdAVzzQO4oXY7aEie1B\\\\\\\/\\\\\\\/0OYJ3sTwZSCGJtrqE2Blz+NNIb1NoOGDAxE7ijpPWOXTRQ7zNCVdM\\\\\\\/wkPZYQOv99k90QYKenU4W379l5\\\\\\\/tGIPznjhhiuQiCA1Rs8d11aY6pilm2psiCzvBul5sfb\\\\\\\/RH4lEQCwtJhEco575LyV9OpqQx2q51tkFR9YH+vViYU4B3EAi7ZiFfdfFZKY\\\\\\\/Rx3pWuoFKtn2K77X9WFzM1vdmPUWSGXtTaUrouryMFez0IG68Yh2LttRfCAfFml56KvTpALuxa1LeY+FloR7M48bxt17ZqrmbTkkMha3rW61EZadChT+NEAxsnY\\\\\\\/2P1arNjfG0wzoifs48BYOIWYGTlMeEzRp1J6Hw\\\\\\\/nDQsftK3Nzwuyr878v7mc2ldXij9xyO+hd6m7RWhoW0KwFsJzP1tbh2wpyo38KW\\\\\\\/OmDaqlxIxVH9VTrzhfYyR2NTQh6I8MPfyg8oP0INVrHYsDWas92l955p2T0woTEPRC74hl0R\\\\\\\/SXqpyurg8QwLH+ZCnI0odDD5Jf\\\\\\\/XrVuWoNT5GfMSfi\\\\\\\/a7CBVuA+PDFi\\\\\\\/ivk+mManbsvGNfpRgiiDbgnEvPNmMx152BFTnFfCJuBt+iuD2f0QEPL1i1kD0B8lrKKexv5i9IjJ8z8D70RoCxNASBF1qe21bRp\\\\\\\/LH4iHiiUJd+4aYVuIkKF+KTVAWsHr6UcwDc02+S\\\\\\\/kybFJlCu1ybIQr4BrekgkS9cjbhQObg0WUPrgMeSo9ibR6yRfw4mSlerBP6XMKZiFbhzx+iqYDmTWEW1sgP2uYBkWzA+PKuiZfj8Wb06+Fuh1wD+dFN9tZi2KSe5Ysn7SKvjWA3Jf0imUk7j10cGMn71L1my\\\\\\\/pIk3FAKLKCDHgOw\\\\\\\/urAcgtLGjplMlbgfUKHSFE3n5rwdIOdJEuPmjEzyB6Y88srG6nFQGmv\\\\\\\/Y0CIOEA9NxQueIBsIH5EHEn1fEE\\\\\\\/x+fePJwj4eU1U3nLBIVueClIxDtjrApp0u2D45GYyQoUeDau85nar1q7BysT2JYUh8P+y5FDiw5BzrwSw5Td4q6n2teyYyPrMRMGORBa72ek3y8E9qEWGs8Fvy0TLU7oEhhcm1HtNIrlK0EIz3FS7cKESG3t59oJ7hHnzy2\\\\\\\/jLK5Ca7nO\\\\\\\/NiToyagLctHaGk3ZTXW0\\\\\\\/w4+vAZQtAUZ35ULk5l9vFDJHnsxQnWr2+Qf4JkM5lGTRAfy\\\\\\\/lhmT+bUVH0qduIi0jkBV1VHjfYifhUyLNfb8bkJadVA7TvjuxdoPPaE02Cy6HBS\\\\\\\/8UamnvlCZ8QwvyLS\\\\\\\/O3DiPOzr556ZJbYpFosJuHg5NLkbFvFKUDs1B1LfV\\\\\\\/chgCxnoamVywGu3oLM+ULUDvJIq1I0KMfkookcK9+eju6Ny9gwqDQzkG9ixF7ZB+1DmcBpUMXt9xJh\\\\\\\/EZknaR\\\\\\\/1OijyAFjLnSb9QLKNddFia241ItVF7YQPxVF5IA17LRsbQgDZSL9So1OT9H+C43vzISPUbC2uBfsbwCfsm8ICr\\\\\\\/QeyEgb\\\\\\\/S7j+KqnAIuQpS+IWIFFLYfgVAm5wlwJQy\\\\\\\/hHkVGMouRjAzunJz1LvbLiaAwiTahmA1pF9WSjjJtX+TsLwD+HzILQPU42tdZ95nRpgGFCRQichk\\\\\\\/W\\\\\\\/6slHcnSbDgqVS6rYTr0Djjc\\\\\\\/wUPchlJRiW9qZkkdL1QL\\\\\\\/DkcpRwTBo3Ja0ooo5CwXWZ0VXhP1PTd3\\\\\\\/lgEtsjwq9NtZjm5bexuqwp70xICOPPdav5jifjVe2m00QofjUcEwQCyxNQvZLvnmFhzUBR4tE2NRaZrfDuv4zLQSjHplZtURanKbwyVtuBLlilOpu\\\\\\\/4Sq8BnIzT7eNTEBDzb6IfThvRMomo4r4FdmphEbzEzcbdLH+vYKLiD7Gws3ijZvfi+nyKDYptmVLDZIs\\\\\\\/y1GnGNo\\\\\\\/WDmd9hsZcJRUVjMMvWiUs4uh0i+t19iphNE3eIoUpu4VpgH8kL9tCoY\\\\\\\/v9dU5Sy9n3bleq5gAZ3cLF2M7Ud3YLJUTwZ9SB7ORjd1oNbdAR+\\\\\\\/7o6jA8yKuDwVuRDZm2t1O8JQPvWakvUJOTEWxI6UzeNbDs3WAiIQT9oZ\\\\\\\/FczXFVVeWDSU+OLO8hEp7fGJkNk+cIRCAkQC3J\\\\\\\/s0ut8LFWMzTsSysXbFfBy55okavxnVF4VN+M7adVa6vL+\\\\\\\/4xQLzlueExVWLcs7edNnJL72PtQPh5tt\\\\\\\/Ppk5mMeQSUGbKljvE1wp8tKEaQ0qB9HVW\\\\\\\/7QCOvKzREW9eCITwQ1YmXv1tYLe3zweFxmwCP7MjrDnEzPRp8yARN99TPCSFrAw+Ah7n5egSb+QTCcA6xWg\\\\\\\/EyrurjrKPWKD2o\\\\\\\/8uPyL8puTzVgvDt7UPqsMLco0FcHW0b3Jx7yCJ72t7JIpZwljBpcUT8kZmDy10nXNlT0JMYvRbyQX7AvgTe5jI4930rmr1eK25kr+5sPsx4sFDluwIGLBa+d4I2EwUqXOCnqJwhwXal0o\\\\\\\/4ml74jqCIn0wEdIFERBg+pJi9sYjqMxh13nluuvH9YLDq\\\\\\\/yPY0JuWlOinefsitOQRtdxerecTdvLDYh9IxJo9Y897+Hgu6x3RCGjJhJz8RFP3iJ1VwYAQ6qOOFBMP5a0DtrE4ecJLzgH4+V7eK0BxGBhcwTSaYAtSqIximNbYfb30bCGr1QIs3CG2LFHK6SPbRZVvvE1O\\\\\\\/W8GT3JRttStANw4IkxMOn6+MLDjDZO55x3kAPp2Slu4E6UctCYHu8AlTLqfEn\\\\\\\/IXnd7y6ENIWX+qCCOSOl8EplPtPjPuOiKtZ+TUm4H5+OeMMBB4ScnlCHSQOQGcpWwEnvRF5tOfGve1E6lwTxs92AlGPBYla8X69EDqSxgiU9d7tUcrMmxIgF2Q1UkPAUuVlu3sXXfYtmr3wEATwQmNiau7abUxWpxB+8l79dUflH0sTJHoKum0KME7wbqvFnhyLE72r2g8a\\\\\\\/vhk6Ips4tXqPXaJ2IMmrh1BT1QP6\\\\\\\/gG7+TvYFbXYUwT1n7XVP80oi7d72mo8KizX3hJHl8Y390JpsaH93uu6pZMt6mtmaIDGsMNCIf72Vfc8AEdfszJLIzU0jH4F8vYUOSSCL6TBY0Qr6CvPhdrjtbIursZtlhl+thBBjCVwwvna7zzRSniPuy\\\\\\\/q2d7i\\\\\\\/b9aJ36DLkjosjmHyIyRMeQMIoPNwULtAx\\\\\\\/5ZpJiejv2HZFwO+fS6b82yW3fKzR5M1YaHuWSxxDHDPdTlHLwE\\\\\\\/wvpY+aXo8Hlha67qy+YOEiKbIAKHh0RvWqqNVoae8TU968i02gDMnYDIcmUDAW3azwc9UwHbs7CCY+2\\\\\\\/vJURlfrcN2wJ0QbM2xfiYsWQYpFcgffCU0Y2ePvr2NExmZG+88Hy+fhO6Yn6GE2ldc9ebtRHOksmKvZtGG9mwRpHjKXNT4fP8XB\\\\\\\/qVsDsnmc+nHINMpMxorRyAWJQ8iIpjxLky4Rxs6GzWUCFuBgmR2Kji8umXAwGJsbL0MOJLEa6iQR76N5u\\\\\\\/mwplwHayxYXyLFPBUYXLo3ngfn+uw75g89QWb2ofUzb4kCy4\\\\\\\/\\\\\\\/2Hiozk9cBX4c3429+N4vT3adYCtPaallarIoJx\\\\\\\/DhBat\\\\\\\/QCTp1b0Fdsvgp5epBo5ps0BQCXYzAWlxTrAjWM9MuAAmJhSxRIhng3Rx4VNBkW9SKYm1Apm2Koyltzsu8mJ2TFCD+cAYb3RW2\\\\\\\/Lq6L5fQSuyLo3d9LcoSex6gP8qMWBmxL5eO0UaK3zUUTOzpPMd3SsGn4kuYHB0APWGyMjIBL++vldFsY\\\\\\\/IcjqWy6E\\\\\\\/sjSMXKUG6nSiln31mgyCZdTsSeB0EIscWyHM+J8rgm8CGbL8G2SiPNkq6BDMVSGIcx1ZCS4W4c+uebgNEJaBJfEqWTxQl2DlNLYuonlD\\\\\\\/kWTs8uLQZ59Rox0vlC6ipVjySMPlWMZgr1OC0WtaywoHhmMb241j1NhePBaWdDXUrXbpFs44I8OQ+wzhYVrzc\\\\\\\/iaWw2q3Rg6eWwKFrl1Ar7q7y0DjFP9Y2mRQSXesdiEp+AePz9zqL2TflWsBIeHF56GoNE5RZybHlbooxkp9cJfuBiIF2PejDviiNF+6Aywmh24DNKhxeT8AyuuWIy2SpXs5Wj1\\\\\\\/Aqnf005FGyS31Ri4uPExcPj66McjJ8rpQfgvqCK0yq9+WbhQrAHd7NkfMETd6nhFqRL59uXYTgKVyeUW6BrP0Xh5chl0TsafUIKBU7npD6gnXz7EzdHP\\\\\\\/xtjZcDLrVpiTvcD3gsvSahuDSEgK5w71kV7TBpP7G6WgrYODdwvmInV1rwb0+opaHPFPMwG+VrQj4qecvKjPfYlFouIF0oWEpkdFCSLSKPjWWNctu4sHDrWgDq+FqHesbedTmf9KgCoFHP8scLYhyICxnxZJOStUZHbmEmYbD+1lkuPAXgWLScmJZ1yxf97+wU1gohOXwbWKOkcafjNk9zrYg8oOCMKFW1Nuht7ej9AS7isT6EmzNdzR3i53v2987uCanuJ8e9KfJioTG5z\\\\\\\/o9luLfvFf44DxXWMBAupPPF7zABsB3sRZEPmOlioNp2sVnVFzQu0h43CV5j3VRk8DVM4G8PYLH\\\\\\\/qJmMOAWo6uXsErhSKxP2UNckQndruQdtMboZYuZkZ26R2pwXu2zi\\\\\\\/eoTiiXhpm9K7l8cTmj9DJGDXwltpu7q\\\\\\\/3eNDSVzmAVoSzyXo99F3HwWngYqR2mq9qDV8WQo+qkajNC3r1LPlgHsBa6ZzkGLen8DbolyZFzo2f5NfxpO+gDEMsYjD3hL3ocft61e6WuSj3UWdZ3sYj4Pimw0ituAWrFxtqaTb8GvuMnJFNkQzKlma9iZZScInDQ2m0QZWQeanTkizyQJs6oL8ouD0VzkWRKPp54wiAPZ1mGjHM8gYjc+E1UEfd\\\\\\\/fIX8QYW5On3jiMR2EflxkDkxMOaVK2WTOo3fb8B\\\\\\\/A\\\\\\\/wVDTjom8rMav9KxqQj8ix39n5XAY6gvcbydyX6sFaQRhdSqzsYYCAUkMmzC200zZiwOxzUKvc+o8gQgp8FFDyRFCuxAro4t5XzmxpdLvBMckHT\\\\\\\/OVJtswMICxBcXK68S2Z+6zPQ7ythlPb6H8zyhcrSsMP6frZ3UYUzK2SYjxEWYjMO4UnJs87WdzKpTzRTl5lTrCPuG+9xDn8+IHAyZMJ1wJpJvxk\\\\\\\/AcF907p4tYIaGPzThdIhw3ETp\\\\\\\/Q7SeFdMVtbWxHjI1STvb53dqO3\\\\\\\/c0B1G2EsSmAVyFOMsKtfxC8gVCQ7rOnczFCofSlKmX57fk4jfxvSEXRYcjLHVJQaViiUJKtW5OTsc7DgSNdI79qsVyXbwrQZPFuX+XerSOB5ZrKd4CBddz6EroBaYmpo\\\\\\\/iIRBC\\\\\\\/TiGG9+Q\\\\\\\/rEiOrmww4ayWCrmX2xnd1LHytAEPuoD2Xy8LZvlUN2dqnnbjkalvJNXpfhQOZgGIuc+\\\\\\\/apD1subtmO1QU0+2hUIyNQgqnDs1EZPgsEn3SpxT6Od0nSJv\\\\\\\/xKZ5UUp+rt9+1Cn5jjtBdn7OhkEaDzcD6XtREp+3XnXdG5ggbmLHdtexKqokAhTC8SiK7ENglguHnLX04KkeN\\\\\\\/ilwofkQ4A\\\\\\\/f1KFuUeTkMkenIb\\\\\\\/ZNqQTBgiX1cj704fEAwBmJtpS5GMgND6j28DM\\\\\\\/Efa3HJNzq2R6mc9os0td79DdRJdAHEFz05ObtYL5sskJ4PkIt13T0Mgf\\\\\\\/MNJOitzEDnMBYetMq6GdVA++Tn1WV+fSwKFbmVrFQg3VO5hZTChNSZhPVeHF1uVT8ukY542k3UFtReNJce7FbTSaL4+2l19XuyzyQg5FQEJFRO5DmZJX9QFAeMlZ1dkjHqe9jYTI\\\\\\\/cuj8Kw437i2mvBiE68wIVFiRYI4MyPPG3CJDER5OLZDFLcfubq32qBCIilAHoFpNjlwRWEZSywTMxrr1AXtog9hL+S4Y4icSIqS3hqDl4bXh0tivZh0EIPio592Ds1BZ1P0Ncff3UHSMI32M95twCPA6FpM\\\\\\\/Z42OgZBjQ5lgV+JapNqt36K\\\\\\\/y8xwAYBqrf7jm\\\\\\\/Wyn6+nqxKM3g7SdPUFYADrXcYd5kiwfEjwUb60ED6MueA87VdkZdHF4axD36p\\\\\\\/yGCrub2e0WD0JnqdLOiG\\\\\\\/1GDQgy+p9P\\\\\\\/k+Wyo56z6NO+SFfW3+7krXTTJdw1qnGSDnGZZHgdwCfTGrNEvW9njWzwhYPfVxUdvK2ZiIBp8HWyXYil8n2I19w8S+mST3hKTTJjzyMapL0\\\\\\\/KTCDyH0yav9Wo+NvtmodwTpx\\\\\\\/kzKDOPQ9u+KCtsRru8v1bVLdcMFoDWgiVPa0O3+zWjSRVavQJ2EJfueu6FDKeKGSzl7\\\\\\\/GvZdZUD6UGCHbJ+wLckH8vuc9ymawUhfS8EkZ7HiDEYtSpxh6IGCvXMzNSUcSEWxTh+tNVcO6BcdpID1BxHusN2tRUYRU9WOyJQkF0nxG2Yw28X+lUWdP0A1I6yRWIYuz8v41zfmunFakkoZ8kL2QhHjiM\\\\\\\/ARlxPK+\\\\\\\/AsPKUszWWce\\\\\\\/OPOHQUhDDMu3VwUOjBKGhm4h9JmfrIICpSKX51Kei+8LqfRYfKh2KnAX+g7U3lRchLVCj0jSmGBiBdlDD5MaBQe2Ni8U0SeXageB6yw\\\\\\\/Pz6FzdjbX9v12lBUoPNyGUaERGoGLPmLFU8R06Kp511ZnP\\\\\\\/HC8KxUL1lyVaYP2Itms9lCruDWf21IOWp+9UEv0o37sTK2X9klvOErNVht+5FGs5jWbFb2x36X0DzCE9O7bf94zDoQtPiRgPzxMjYtt6SUURvgUMaJxr27SBc4H0ip6SOR+MIanYyPFPaeGE1BtfU+elT0nYLIgfRCpH99WIk6RGD\\\\\\\/SwlVjcMipMIAi8rCiP\\\\\\\/va1uc5iiKhAnAapdL77vfRRaKpq+YzkDNvqlm7slwzakbKpdoqLyPCCtR4OZJZGd+ojhgnOzkbjlTQ22YgByzXSnCrBHTzIlgYmaVs6f2nzx0BbPHqUnH3dg8qghcyOlpyesXbbhn4UVWsK4u+UUXaZLKBEWByYMKi7Zt+26UUEtQL5\\\\\\\/EZIdXziR\\\\\\\/+2n5\\\\\\\/809tT313hMe1k9HNCRZ4uDfojdjWHUCm23S3\\\\\\\/gL5AmpmoDY+rKcdgX1+uXxL0o1hL0GMll6tI3Ls+dqPP2GiT5RYJKOdJ+Z4mN0WeCl0lWtvMjvJA2GKy55qlBffsh2PlfUztUo4I03xo7dC6XgVyDW9+XN7yTRe3\\\\\\\/uSF\\\\\\\/AImX1l7JypZ0v0bxw8Y2uWgi1aYbiHm02krBbIP3ftaknSPwOVMOffSPQe9\\\\\\\/y5sGI6pJ7hq0ME+1fmdMNTdOtD1WpJ6WcE5Z8+E4XEF\\\\\\\/jFPkUVzAJ7rdHOW761trkGXyahz5ctdoj+tyM+XlzK2FZQA\\\\\\\/+2HUpXrlykChGbgI3AqqCWr8HTmJWwru\\\\\\\/rv77u9lqyFDGWgljK29ewpWrYMsoiVCDJQKzM2boPUeVh5JG3f8ytisNPNKPpYrF\\\\\\\/\\\\\\\/e8skpMQkYT\\\\\\\/ifZy6H3bmuT54u6tOQlKQU6KLcL7xxLgbn8O4+pwZSXZslR1XthB8fkuix9QKu9EsK67apX3K3dNUqbltuNXeJ6vmpbaRcFfWD2foxIIYcBVTZ27H4nSfK17zc6VBdsdsHsLmGoBTcwcYwEM8AfsOOi6S9u4gOer\\\\\\\/9vIZjrqRP1r3vHTWvl34IqFM7w7nUoZR+8A0b1TCJiAcWRPt3bCJvy2fsqxffos8ZQEhNWRmaRjjBpfdwzyB4TBGlneARSRoT2PncLboZNctT7f1bsNBtOQOtjz2VvZZU7Pb9UF6N4R7B3\\\\\\\/SkI9Ezu1RPEJzxTrAxyaj+9gbfcAVLsjHSZrUfxgR8PIO4NP8qCnhKqLKWyr3OMpXJMWAfXT\\\\\\\/ci+qDtTsEpRJKqk2N\\\\\\\/vR2x9REmZWVoh8Luw7GhV+x0gG4kJFN0+jmRP5VX09\\\\\\\/nmasgNNYPgkvoSIDx38oL4re1JAGMDjyekdz+8b4y8K\\\\\\\/Q4F4nwGXdovevjPzck4EEBBqZBDiPN047yiLvGC4G1ssUPfqGHAS+rS4q08AVsk0k\\\\\\\/QTJ\\\\\\\/7Pmk3eVhoOcMEETffT6HwS+dCCQ5inIFj2u4dIJjT7yddSAEQNx8EUCwbNy\\\\\\\/jJuaFoT5gEbtpN4gTOIOoZTH8MMOa1N+3SVNLxv\\\\\\\/\\\\\\\/lucU5+NlX9WYr1NpIBpHXMhbRceoxbei+nmKRLpn3WsvB2YjOzx69zNutIRHNgtc12GAh8Pyw4v3cIJsRmVCJj93p9Of9FwsJKhgQmITGEN6Z080b7KPOasRqtv5E3QL38MJ\\\\\\\/ge7Evrny9IL29hT6HGHkBTp5hp2Yh\\\\\\\/wIQOa6+55f87G5+smlSO6ZvN+uRa12Xo61kry0OrqAk7XVb2hum4S4YCAId7+2Y8uVS0gHW5bqcBo2kNYr1E0AnwOK\\\\\\\/d9HvkfZyX5rzmN4zjvL+NJ7ZmkuZUAZ7AF75iAXuJ2ZK3IeMvJ\\\\\\\/z1O9Am0+BgIDl21i0XRBi7Z8eiVvTZS1+Xogfx0EsKf+Qhoccsg1mzhEnsmIXQATihyUPzmpD1XAopuUWdl6EU1Tlm+ptUYrydOGOuWs\\\\\\\/UBlYjce6RqyvHguz2Ssqk0\\\\\\\/XO7lXXROnsXP8jHmDeApuE1eFI7pADQPkkK72kaVdwTza+ooyi378p7JIJ3QhThDy\\\\\\\/RVaTs\\\\\\\/c22UNRyKnKAIJfAhfmMyta30estvCf+5QAdolWOL1IDA\\\\\\\/mlnthVaoYuCJhvJMk2xf0ZQDE74ApMGJmG5fWKOHwuUn1uT28Xv1dDQBaqHTTpi2RhNU7kiwT5SijzoS30xq3ugLSZf5MisCNJf+xeNIcKka2PFDKf2Rqy7lMxV2mCCg90MTBPvVo55ytMcjtb6O+WlEsbIi6kafFYnFLjmgSJUVjn0iWkAmggr0JyHXMM+05aayUHOFmHEo0af7us3TaBeKMSAioV211P8m6YlfPZdNe+LtqazJ0YVupyy8qpwGziiHbEs2Ivik0j+sGbh+2yncg1yyLbH41Llc46AB8j+NekzSHoVAoEMI7cfXBO9jPDbnX3KUAVv0\\\\\\\/+\\\\\\\/CDs7eR\\\\\\\/xm3ljkflS83e71WdrK6IKvYmyUWINqDfA60lbIjVmU9gFHkEh\\\\\\\/KID9OtGPPBsLrMIglOctvluMl86FaP6TXKbjPlt8HCA8041P\\\\\\\/XYJYS34GGRrmhHqme8dUzduCzB8YAVSy8ZLzJrE0OYxb6SvNbZ\\\\\\\/WbLDe\\\\\\\/dF+SG344R3Ofaf6kEXDECj9n+Lsz+Na3sliZZq4VJSgCjBFCfNLirhCXpgVWqdmqclHeOOsnUd+Y4QPNTD37VMHjC6kZHkK5q\\\\\\\/7Pooe12VMnXIZvOOVnVbIiD6XJxcP8B2vqnZkmy8KB0skt6pXbmV6FKb5VF8UM30JWf6kPQJ\\\\\\\/h1TIwtnA6fx730MnVBSa+jia75KFY2q1TEI2B46R4sJD3CFMISyfkp9A8ZDFdIrcAFf0qOOf+q1yOfMh0fsXxGyICUqX1plhGdOh+aNKvFjuFLDS6S\\\\\\\/+9e6Rm3Onov7CgUF77JnKPBiF\\\\\\\/VnGGaf2pGxdCTlONLIwemDvJOc8Esp2FM3fzWsvCBT2QmWZjJvgtVNGU+PCrhAYQwtJyNcQGN8ih+b3pBhIUpkaM\\\\\\\/XyPdA6c25mdJRrsB12oW19b2pyRNNxO0L0kK+e4\\\\\\\/KGU3228IdQHD7K8hM4MhyjmUfMEwWtDEY5QJNvcTk7sOpqeTa6esIjdjSFSuLEsM5uDfYSsRW1RjxqbWX4doV\\\\\\\/HBG0wjsQ8ej\\\\\\\/TydO+4bm+2C4PuZVooeSE0s+SCgz4LY4yf1\\\\\\\/Ey6Yi7EbaFvnes4851PuIGGeIpxRtylHN9MyvaDoNvI1kdPtJzcg5vDb1WqakBol\\\\\\\/NN34u9V1bHT2yKtDlXVQYPpoObbj6M1VOb2kwUTdy4RAVShHi9ba1oVK3R6u9pLafk7Ajqr3fGhi6yob4KlnItk8Y+WYGvF8ETvAi\\\\\\\/50MZJuVCvSWfW0bEn\\\\\\\/PBUUAMzeeiUW6gTNml2OvpqaP01n32bnLtmDSvlShjQqz3isRgvl4iDBqnmrZrdT2iC4cy5T+rrPKYDNKLOv5YTTfxDXJZhQvBIez+r7zUHtixjxHGUMHvmGYKqZ8+ewcCqd83KCoW4ZXULu3BpheAPjbb0Q5AGu\\\\\\\/dvpARfLEBsDziHVfKboMIB7CLDCT7IKxJOuyYHqNjuGedsEDRZvyp74lX2JOhzfDIAIht9vr\\\\\\\/nqot1rDUGCpcZTCADfzkYAItO27\\\\\\\/VhjgP7XpEukJdD78XaXR25V6ejB8N+fH9VoqTu8BPmvbdrDQQdItk6niN0pnq\\\\\\\/IjTetlEbLa0wwKddhfD5KHWAJ7sJGzuse1b8HNkUow7HOt33FnOzbfZIFBac2fHG\\\\\\\/5mBbknvorToM2SqH9dctkE2DLLNmmZkOFg3sRxNSRBjH6Zuqh9q+nEvfyUxsae\\\\\\\/1I+EP1auAvKFSA8yDgzk7UAxQhAfx8JCzlplOsfIHG4BhTIZF+LJwAn5QI0uODdYvNOaOqUKyDnLly\\\\\\\/Pi4l7UrwqP3TpBOYrxg6izR3yJrtTPJAj7W6rf8LHRR1Aq2RUafBRu+1abHf\\\\\\\/2qCoP\\\\\\\/a9z+WvD6KcY3i3mW1L3\\\\\\\/fRJjuvCgL9bQJNvMs\\\\\\\/oylSzcNTkPCSZ\\\\\\\/6d2qN5UjMbmgHI5x7Bh0rBeUiqo+lAdRsec7Uuuhw3b\\\\\\\/2Lqm\\\\\\\/tbBQlQpwDRi\\\\\\\/DnjDktwpZ76VJKYC6F5P\\\\\\\/e3e5Rw1xoepsTy64+gvlx+r3ySd06EpY5+\\\\\\\/dO9KNYshnLN5FC94wOpJfzqOFiCvLB1oz0kkuhad7I8ICH07e2mQeRcNQyJeBaqZUuRer4rVFUIuALIHleAo99C0wAp8fl6t17KpxuQmqH1zvqAda3lazNwBeeCUq\\\\\\\/pEvxQwhSpnbLtz2d8pZAkANVb3qZKQLe7t6rqIoOGhgOzDXiYChTaTftJWKCQKvXs\\\\\\\/uMAsA+kn975tSSyB1sB+hjNtM7w0tKsIVZVtp9HwKHZxGmEGA2H1mY2rS7HVO4uj+vMwywN7vTQF7kEDBnMCt3\\\\\\\/wPUnN3C4SlKg03p3l3etkTveuxKW7ZJHIE3u4Qf\\\\\\\/q2C4fmIeB93MoGYcSvfzkHcC\\\\\\\/mbw3k+9sXJNw9QywZ8vlsVeebdDQW7a7QNQ4rpT0VbMVZahKSvMZ8ahD53p49GD5Ev5PWYEwKMfMM2nOlUr+G\\\\\\\/n6feL5D8o3TDynXwAmKGNp6BpWmPle5oRNeKfC48ikXFcDEE3j92KhpcOZ73sf3Rm\\\\\\\/BC8CNnsG0xseprATgYKeSyQDosKo8Y9RPiXGvRp0vOr5TkE\\\\\\\/7X6kQ0z25i73L1zJlYyxfFPybspYyZNNNYAcHXy34XjNAjkcNhiPTDA5U\\\\\\\/yHz92GYKOi\\\\\\\/zvBCw4tgbGG+Fh9jGGPfJG5kQAcTBT63utZOTovirUwE7brHMsvc2qKHtGuCNJPOkoQClg4iVrPkJepsVMOYY+QkJeT1MEXm9EwwnWsZEqnTBOSFJyEowN0Z\\\\\\\/\\\\\\\/cWJ1YKTO23nmfYCjc\\\\\\\/Lby5ORRA8mUoafiiJfgFZdI8uhPp97rZAErEKLRv4NDPclfWpx5n8tBFmiMsJ\\\\\\\/gDPZATGR4KGv5x77BbrOCqb9Atau\\\\\\\/O35Np7LoyPIHSoitvZvnpK2XjtgJoQW+t5kFlhrInybTpEdi1E\\\\\\\/KelDg6iuHpHLm2B0mi3jZ7e6\\\\\\\/TQbCmyrHwtQASxPTMrN6zIRuX\\\\\\\/8pJW\\\\\\\/qeRrstVkL15hJSODMYDoYf9YP213GTqYpiq4wCLgvVKxrmURdV9Q\\\\\\\/PIX\\\\\\\/X+dqyK5lwoSJkH4ms7Pmv4FwUaIynRH56TIQdUIuM7mpwXBYYodMYStDD4GYLycNVtgU6CDSdzbujHS9YB6EM3TViwfB\\\\\\\/NB+2flbozHPHI7854hh0u8qCkSOO73BHskIj6PC6RqtWrgLgLQb9ytfkJXUbEiyqpSKglyP8y50BxSqkiLy6vL+lu08BF8vvg6yJaLEYQJVDiHO2epC9XqqCmrQXPL+r5Ta3uCPO3GlVhoJeyZQRORptz8Wi6AT6Q6pLbreYKD\\\\\\\/jN9lQrXJthRlsXk9HGjnXTvGCuGTaYVMBzI1JoaEVQRgY5HQmyKUxZw8A+rgORa7CP2up5EeFhbFg2Awtz8zHuZf7jjl4ak62wzkQwvIHPhHxFlk0OV9uIqPLqFxRjbSnG3oENBGhbwoPbj60EyxfN2M2XVc5lVhaL3A\\\\\\\/4uvwQwv\\\\\\\/IYhhEXwOnhkddlr1TlI5PYQ9DrFBNUCwKqlB+Vh00nElLcT2wjh7dFRGtdp\\\\\\\/m73P+2foWVWp2yOyQffgChuSKwtmn6gk+Sg6wX77vIG6iOQPRTsVQH3ErbWGbADECMyAfQ7jkSnewMr47FEkdkHV\\\\\\\/hUPw0JyrhqfvcFWanyj3I48cwk19lGXUa\\\\\\\/1AKVHs0hn2\\\\\\\/n2i3xZgae7v0X2952zaDVb1TN9M4natVwwYR41BL2sCBoOd9dBBUd5BL5P8A+AbDbr5R8ysennfkGbZqETpg\\\\\\\/IWBtVox861KWaSAYJ52S+eTKHuJCUeBNj+m9kdKqYqS2gtusHL5vASw9C\\\\\\\/R6ISEEeHuDYkFZwU6ru1v7J3A5LWe1zDChIgObwUxlnT2clxxu+sUR5N3olJDCxGZhQSP8LGbq53tj7vhCAto103eoekbjt6ZUcYaLoqcKDhFbbM6l7ILbSpD9scLxHdR\\\\\\\/6\\\\\\\/jLiUGJFA8nnDsN8KjV0u749zzY92t0pcGztoJIOgOeqNzEP4tx3QFnBOpQC8uAOinvACSXe1lN5VsAlbj+\\\\\\\/CamdSyMnBnO62+lO+HGOCZ6Mdzh7AIiIC9S43qTuTM92Z32V7nvScGPlja2yk2q7l6mv09UoLGXH2+tCCXTQJ3fvRYMw9tkjnO\\\\\\\/3JDPiapiHrvRNAVpOWyf0WyX6ZCu\\\\\\\/ycPWWn7+2iyorZK3AEuiQ7FjWstK8vpA131s459r0ho34DwQXWE7+D+cCIV4m0gjpl7gr9mp2J63Ls\\\\\\\/DXnzN8Ru5HAaYIdqEbIwClq67tMU4YbESX\\\\\\\/SpuU5GrvAvpgtg4Qx4kEC9fGxbdbnRowUtawQxhLEFmNqXIqSzWQQSGhrMoYIkiPsX3wUz\\\\\\\/AY6bnMU4+7Y65nb0pYENAdFU0zcflqR\\\\\\\/DGK5I1PE0bEEvJU9nRxoDhP7v+QXk8QwwMhTrDf4W56jmBpNpSmoBXvFzUuXz3S6Mu73YiqOZn9FvnXzTfyGedeM7WA3f2DyKkCoQ\\\\\\\/TEYb29gHQ5FCWEweMfw4wOOe4wyNUcVUwiNmGVV5KCVqOb+S7KWPMpANEzHQLJblzQuHpJ5xDbSbB9rcGoJ6J3z\\\\\\\/C7wd3bUYTEN7TYVhTUwp5ied+FFYsEsaaNil8Wx2XYGBPe5Z8AaMqpYGQwd16Wk\\\\\\\/fV3SNe51wq2Um68zURIsVr42zStWNbpaoE3d1QdZuHpCAAzQvjGPekWduqHPI8X5IY3jjdLVFlJY\\\\\\\/YCllIPnV+X0\\\\\\\/NtNKgGKMzkgR4sdyIIlxc4eEtzQ2PEqZWiWCuJkg51no10wdnEfAHxI9QP0IwoPmWTMY84RjoGiiuaZn+QBi\\\\\\\/A1uZ34K3\\\\\\\/39Rv\\\\\\\/kqYlQWezq5kPkC2AxDhmSk6VhYteavIDczQ+OObsaLKEGTPFEn3sN4FnyLx45LSr42HyFlGmArwoc+FaUaMshFK\\\\\\\/e937WkNuqyy4hHR6MPPl\\\\\\\/sZTPxnGYhPd8jj2gpVfUtmX6dToH\\\\\\\/ONZGWqgfPvR8YiQxBcsNNeEOWxCTf00KcyvjCDCtSkaHNehxGTqio5pvDxk\\\\\\\/a9Rt616x3bFb\\\\\\\/MkxFsyENw\\\\\\\/raILy+FAcSnCKUMjxizvnw+w5B3evwgxjVi+wnIyHTkkyYqBSNhg2PeWUlb7Uhba3SZ1AgAvUuPiNHc125u0Rt4O7NqsyS3twCS1dGgiv\\\\\\\/la2TuI8BeLa0HmgOv6OdHaPFor5eU8wlwe2wo1qlCr2SGfMk+jZgCPgrEYoW98GrPMs6qonnuVMDdMTOVZ2dnCY8b\\\\\\\/PlOvSoMchY1zXqNHc6SISZD4kL2GfMcmUZtkkSS4fuyzeB1vkBvPkKBu8ARUS0nSlmGCmI4YmrbZ7Phzo4zGnn3PRf22BbYFK39mLPk9xGGXrZ0iC3s8m72qHUn6JhP9Sd2ggNvkM2drDe4Liqy5frZP7V7hGg72lZJ4miU\\\\\\\/nB80uLpxuEbXby6kG7gWgXdOWmDHU2wFu6cQ4syWVbmh2aVLAHzR+w0R6WWnhi2UEWn\\\\\\\/ZJ1uzSQbkje+JHaeLFCiOAG5xJOyPMIGFij136lvJsEuhP+4I7tiUL7JMdWQljMhQ6SVslaiFpYqhpO3FVo8IN7qlAWWixFrUqK4qYGCKDL98w9iZABpsl6tqwfqWjMKK+1MeflLG65yFz\\\\\\\/ctcbK\\\\\\\/6QFLxv04ZeD\\\\\\\/wva4gqV2E\\\\\\\/qHoMAxL2mfffjvWcd0azAgDv7jsEgr8eKP\\\\\\\/K3T9qYTa\\\\\\\/6ZtM\\\\\\\/cqZ86+2r3V0yjrAtO\\\\\\\/brOltanIkF0gW+mPioebvYGd48icjZknIpZSBOos1BxJse3AEhiTCX8QlMD2qdHwJHrE+l78fbCS64OhAVRWZSNMLNNqW6mr5PaqmoPyvp0D2eOuycfmNMWmVdff2U19W92YzPF7PKJpm0XLCOcOSTREDmER2P2fJRwPU1bfzQvTROPLch4jSTxQLAakxAKsehsYQiygGNgrALuf\\\\\\\/fuJW1Ey45kN8xrFrCTaY\\\\\\\/xPu96lCdwo1+WoOcFybaiYYgn1JYwVEOdFlmRTmfZlXx58Xg7Dt5I+3YFzkvB1Hdh8QCf\\\\\\\/w9z7DemMKYGNv9BvF7cu+4VThqeZiFbvf6r7h4aZ77DXkdGxsQ5j2TUQ50r7B0eG2Kg7m6BF6whXlT5euI9TtJemNZQzK9OtZQKTce2W9zebYX8V4MU5xZC55u7jmWC\\\\\\\/5JP7DnwYQ5Mt8N+A5vMklctPSNnTTGfHOcRYsz7bQyAqoBT1nHqhN62ILVGOoQJYOyo0XdVq081Ey+ocdnmfFsuz3RiBi6ds0LxosubIumeFGGhquCtHPW6MSZavSc\\\\\\\/aUL27UMK9Vh4U71Nq3mmHgnxlmO6pncqB+QIdpfBtwZyh1vC+iGumbv0BQJzAbh8vkTEBkFuct9QCOAfxXcdwiTuBewcm1ZxxxtvwGdoHknKcGYwJHXSDXKlD7KFZAgpJCStV\\\\\\\/bCm7pfFoRH080FstB\\\\\\\/b7Rs4at0CbfqTZODIipSEEMqqk\\\\\\\/8K6DChbQy5hrNkh0uGqIu3IPIoWvZLsHApDfWIXZ9rNlagiwdc2T5XYnkoLOnPx8LydrD\\\\\\\/mAp85E7fAng2yAH\\\\\\\/cKzHY1lUFjS2XOcgrRYVasgBvvNpQpz+gsQ3fORHSoGr\\\\\\\/E1sl93JuNAdac+o9XcCkRUNLZS+6LO3Wm1e1yHBgXEPpRltlDnOjfYThGYMa0XGGW7z3\\\\\\\/FV0qcvk9lHaAkiFLMTOLZwQAnsfzPCCDWfitp\\\\\\\/88WYyAUfwcbWStz0ks5v5RGbPgJCTIhyclEf1u8ZJLQM\\\\\\\/BCQBNQ\\\\\\\/mXoQ5lILs1rCHfLKFOF8O2KF13uQinck6+OxF9sblXZrERR0GXGIKNViZSY\\\\\\\/81SbakWRfPMppm6noz7emCQb6hlpvH5EXjWnd7aIV1g3J9uPq6neuhKUs50zVaZAVZDkhJ2gZr4GW00OahM0HtHL3JhPcBDLm+waPfRRfAApg335dkknFAiyw34DdP+z7ji6vBhgSrnw9VD37yyd3hJXw3Voq8oWObOoVL5VBfQFE3tIx\\\\\\\/ZyNYosdJdgf8sRmYk5exq5m5vNJB3Lw9RalJ+xUNiPg2k8TgvhaiSpXd7DBrRr\\\\\\\/hNmAlkxsoOQ1pxRv0J9mLfGLB15EwSPgjVvTyNrbeaOEaCYLZwQ+zhJLMCWlXvSPR4S1QfPAXZv5j3zt95cckrm4Ql69Fkn5jkADoR9cxsg8+Pe7ojmxG8CuwOktuY0iHlvPkeRptZcZdipm88J6zRvSsTT50lhsZxI6IqglRjkoJRM96B2ChJjoPzbDfJyoeGqqyo53lglDAyqHoWh4nhngVGBPAOtP0HcKqANterCv\\\\\\\/HQnoy5CJTN\\\\\\\/ZQZJqzlowjagjBj5dAUIHYMxdORaaS0NoVx0tcxXAgwG8vRPp1g3ZO0jN4VWmFx7atZF+4PS+vpGLqlQojvkVwgzkxoBettNlf7ZdHWwDsuPfgHH4umZZoFFGucLL0Z5RvovORolZExBqlJ39zMG9hFR4h+rrFRtiFrSquEJ+wm8DwgQ5RVXA5bi\\\\\\\/Eng5gy1A\\\\\\\/jqx18ejULHAthI8I1qApdBRpY0rllw9jZqli5Wq1AqzA5NEaJH1zAn8sdrI2Y3IXAJXCZQUAW4IJRvc78+4dWimKpIQ0QlfOHxFPQBxs1uqj7XiZ+xOJ4I0ftS7dnJhNeSqBIR8Zf+38NF8O7LhByF11wncJvNnCIE1vVf4wL7lgsKQjD9kD5NTv5YMjuxqbenHdVmqRGdYyoa0vpI7PY+8vLUpxglNXIJFLhgHgk3M5F\\\\\\\/bchtXTNZPZQjXtEjgZxGFYn5cv20Hmoka8ZvW5rcmPJjJ1uFwuzerU5VYrF8mNqEVbVenvjHT39M6xv3kuSFIvROtH1GkOnjxixGj820zdOAMvgRCMbhK2khaspgV6P45SMHD+EOUolyFcvajcMHmNB70XsJhQhMX4aebGRhy6k6mYCFoguRSNzlOVFpFavMpSdrd1jbBkHqfVfp8aGQrZ5C+IgGdhN3gH0A4m1Di\\\\\\\/sNV9uiITx18oI7fPYtOfSrv+dYW6aNEjSPNfil3lDIuyOmv98ONY8tLRCv\\\\\\\/3vITIFv1ovjLaYoxNdF8Wrygb6F0D9fUnkvFckUihKzWQGFyc4k4LgnV7fvTq6O71ggjMUd9Bc4krFRiwq97HkF3J9jUD\\\\\\\/mlEVzJSFD47Yq877uZE1G45LNKmfQoeBGnpOYlCK7gumudUmjnKwF2HK76muX6rztPz4p4Oq5KOqXqCtKDhFrByvh122zO4U6JuEXVwF\\\\\\\/RHVz\\\\\\\/1nyw4J0j7jtPojhlQlo9ntLWt+9cpDXC7qcu22ajacjZUybsm1WxLsJZtxzfHIIq4f9LU10b0YGDiR45lGi9tCz7fpYE7CKdWOojFhS+SNyE38D3JQIgXZ\\\\\\\/95IVrfx798ZsFQNtKxOZbOVB75tQiu1g7Fuq4tBf\\\\\\\/2ovhG5D7qTRORHbG9LjSuiZPNYa7aqp71STF99dUOVMYQ0IjshH\\\\\\\/zNIpDHnxfCR6I90rL95prov2rOBTb+tOKAm9mgzWGa1q6Q0OLJ2mtWHxCfXBvsd2Zrxmkzu\\\\\\\/al4GjbNpkhkcH\\\\\\\/NPmnWfzrMRxd7A8ZRcVLs5ulm4NDvejQklAWkZj+\\\\\\\/LPalX9Wltqw6LC9kyrgBRzxwNi1RbiHAKmbq6Z4f+0HII9m3c2FYvb4QBorXMS\\\\\\\/anX2tg7OysytM\\\\\\\/kOkapEzDSmEtmpcQwbJXYnDYOweR63PdjTodRGZuc7ywDksI1Ozgbxdk355lBUqEO6xMY10sbCvqo77u5geWQfg9dpcXhWqjj1UbWlijhShh1Gty3KTyUZyNXNPQLmCMI+oGZxzQdljMWeGSx8fiH6VnzMmdZ2JSF+3uZ7\\\\\\\/SprE1FnRl1M0FAWwRNeIanfwFS\\\\\\\/qc\\\\\\\/VruAijBq3J7C6rYuShGHTC4gALvx2faoTQyDNVRYLWfyiHPkPwl+iS183cH7ScpM2yu+OVcALeM\\\\\\\/ljmkdmNSceastEIwvqUlkUdDoCpUAxX43DfH9OwsQZlOiEzEdTcO6c3RYLLD4M+wKoA9E9OdObw66y3CXTCv1Rpe76Uh\\\\\\\/TaSPjLVHl1JZXTCeLJvOWYbCT8+NQ5BUGgQMureuqWAjN59dO1dvS3tnQWQNkblYgZ0K2sFmdPBYY8hLrfZ27MUNJoo4ZROudP4zhg5t6BF6ouFAlPzAWEbmdqoSDlin38tAMyFR8h+W3xWV0q8n8BTmIsv+ykiiZsb0rGpVd6pmZ1VaRfYNiEXPvj3GXj\\\\\\\/nJP1F1fXdVfr\\\\\\\/vhlOaXKbkJTNjVmn9fIX5exKHN7zrvIR3NxM7o0HU6+c3GFetSJdCt25Cg4Vghr774UaVtPcALKUs2R8QB06qnTJm0wWtTe0s7jM8g6m+DDMsZE3YrpJYEDuW4S85MpgH80Be7MfnhCm0Z\\\\\\\/N4hRGrN8oy6NPIB4oP4TM+53PuHJvgTvilItUwNuA70KpTefrVad2Ag7Vt1fdNRzp5peOEwRSID6+8nxJ3ukJgjF1VWkN\\\\\\\/oanbrsl5r\\\\\\\/3nDdbIWRirGFt6iX5ken1jST0EThx4jGn94qd1BlnmZozIY8grrg2PYCyxQvL7D\\\\\\\/dFTSRujJ58H2FhKGH9x9rJFd0DBKjFbfMlOtg0sLtssG2oJ4S8NFTg5MFgspHHEwdGzfv2Ln7pBwvqqIJpUZ+bJWonsFWIbRaYNksjj5\\\\\\\/fkmYrsjpmYHBAmVs2qsS7nbKFJ75DeqZ7vbqRGy+CX1FmWfLkzF0jfUZJzFVFcYMGAxpH4by\\\\\\\/Z\\\\\\\/POIePhTnluk4AqBv\\\\\\\/jGFfmEKQRwS\\\\\\\/nUp41TCed103iEJOKrNhxP\\\\\\\/3q38YvSFOAk+hQOJCmoQFUdkcHA0Q9yRwiIN7HfurwZ2ocxi9TQBZfAWe5WmN6xdDbeMqWaZaisKYaX7nCs8qs8nriHitc3Qoe7aTmjmQKUnazQMWVe1IfCJJHir1yxs2zX68puTKNuUBVdJ\\\\\\\/69\\\\\\\/5OGQKLD1xAvieHHBkAfUv3tXdnZt80\\\\\\\/hVSLqcL4KgRopTW8FWn9XuSRMmZdGiZP3BYUWzuMppvo\\\\\\\/7W7gilkSJKF2dueD+2o3dZWKYbVjq4r\\\\\\\/UCXqKgBnaQJ2uqQuc+TOB6ZBC\\\\\\\/dh7qUr77PIkI0mVMFri7b1XQFiGhWGHqjaAJKpoHRqtxZeq4YEcGHWO0aa2gYM5uh3pdYn\\\\\\\/t95RSIGdpOickUGAJmqgEu1gVWbSFtWWKBwKKAYtAycPccZhWemnvcbHzQZFrFag7HJr2oZacUuOtiG5\\\\\\\/eTCWUtFrlGApPRdoKvBBIks7ZcKCux73YpuTjL7ePBtQPNdRCXwga7dN35qW\\\\\\\/QT5xJ2wQ81xAUoqwyvAcfxO3CD1USutZXCJr9AWt5Oiw27FWsmq3ixtOA3BGEIgDAuQrWShO7UrM8bjt8+lRgWPzrNfuqJB48jsTyTSE\\\\\\\/1H2ko8hdNfiUuf6VbqnWVDZeZnz4OIFukr42ILUFIQviTs\\\\\\\/w0tcfuTzwijrUuWamKHwMYPTmKA3DpqoioUCP3lZvLKkJ+nGtkZG6qLt8LUDnjRXmCMRa6MxOV3PPjrJ0HKhdkYGIsPjakWAI8X8\\\\\\\/m\\\\\\\/uEdPsNyk+tzHSfHdC58NCwuLkShiEO43fUZAQ2Gl83D2GEttOUHg1YPL5wMQSXGmhAEqJjICLbJo7nKyavuan1nWFARLXqUMdfw4K\\\\\\\/nMWjoeSOsc2E3OX\\\\\\\/SjtLYZIOAz5IYMFLefNeGauNFw61lvyXIZocUFmFObkojvpyVxfkyGCWu0vBf4E3U5WtmiiFvVzJJe2ZsIjSjcPDJhtZruDwwXnK8zwwWeQ9fF5Tb7dbDg+4ojCuS3gFNzC87eHcgrjhcYg1DyLgZrgmhc\\\\\\\/vj47tCHOObG7xUFijNFTQ4Mmp8+ceB1wxP\\\\\\\/rVHbb8t2nXg\\\\\\\/kYOWC\\\\\\\/5MpAZ1PGivE+JJiE0CrhivZASrMS8PXD\\\\\\\/DYz1rS62+ilTSo+4b5obZ3iUVEBn9aXbwd9w2oNAzxUC2fam6DR7FQ+uwnXXQN51EbslJUY2SwND9oVSDoBnP92Xp3MQ0f9CelUxq36j0UqKEJVkASKsYaxHXs6A+dJSu8IXnLTmP2uZ6J8uepIb\\\\\\\/2vG7FVoy6wXnczUM202JOP8RT8KGVQayY6bSzTvfEGkkgnr9B52b2utDa50ptLK8x88S4S2YUh7y1XnQX1UHYQAFPW6R5B1Bp+qx\\\\\\\/Gv\\\\\\\/gZ+DSvzPrK+H5f0XEOAe1+BIPW9jMFXT6sdTzxc6TCx9szGk+LxfWoyDC\\\\\\\/JLSBYZjX9IgNdx50rlDoYeHk4\\\\\\\/MXf3wbErrdA2xQfLTVvOWDFAGJG34Nhp12Xz4Z2RiWpkjYtv0mQQqneTsQ6VyRGGmP96e9Ob7Z1zh14MfGGViAeAoDbfFwSLxflLCICF7X5BimBSIB3nYUv9cLHNYowHPQZx2wrR9y4QLduZii+SCqTeEEamHzQcnnlQnLDfVRyLelvpKVOo3KZcxslyikLo+tMZcHX5H2O4wXkzEJj8fulOtdv9NLQNbq\\\\\\\/OxZb2Z4XNfeIH6khc62PoL0BC4SoaE43dS9uZdMhVF833Z+dKeIAC4rLkha+KT6Q2X07sQv2K40kNQFRoZT5XCPHRIccFyPguQhawP\\\\\\\/cKHoMIPZACt2ojoNpO1XjQjd4TW4HTZZpYnTbzibBLWVEqe\\\\\\\/E1RimmTQoKwEM5X5dQaGZQMTlFfP3wRqs9hmdgcbHCLGhJ\\\\\\\/IKNy5dQXVv2CEKZ3MAWgqGw3074HI3\\\\\\\/q5TjxPJ9CS9pLWuw0dHO+w0oLKBqyuSOMQNVddbkEg1iyrU27E1lntXTvVUVIWSvLmeIz7DtNFgTTm60Tt9S9YTIBMbTSv\\\\\\\/wmdXU\\\\\\\/aBjjvhrH82uG\\\\\\\/4G1IwJD8axernE56gTY6yspH5z0Hw6yD+9zCQWgW5UYoOYuS+DqR0Om+FLdVJnhqHqif3DNqyw0VLqbHJLzjZVKZK0wfy7yTXoqKzewGY7tsLdfAVzMLm9ySI+nd14XtB7r\\\\\\\/aWVoaMnPuc1g+QlWS8lK1DmYvMqCnKD6X8jnehmyJ67GawJRiPDFmQEGcbCidKqXRjKBc\\\\\\\/r68qHGb9LtpJgDEHtLUVrHSjNNqEc1YSXYb6rZPXShD\\\\\\\/K\\\\\\\/Wd8xmAgctFhS14ro1iQ68cQPtFOjxW7bSsNl1u5XbAFG2OGr0FKg+pa6weiRzMQuI0qjh7g89y7pGrtS1KoJkOWxKf6XQ6k8nTVretkCLgGDeQDIUb\\\\\\\/vlNMGbTZ\\\\\\\/sIWLbJ+Z0CQf6sQb1QpVISK9vR61Qa8q0tKrUnJoV8gg9bd1x\\\\\\\/FbSxlfCIZig0jOFumgnargznQLGY1wxrob7t8lYT8uQGJ7B5QGhbOtQGlUPSLWCcl+UKXBL89uAPIpUhSw9gv9bMcDcEUTFxT8A19Wku2nSMN2BtN+X9fhOSKMeLL\\\\\\\/B\\\\\\\/o3+aIeP6wi9MZzi2p8TLJIfA8WBijEvIBllngSiNhGFm3b+RJzXXZGZbvS8UhSqj15s2Rc7EWGOJlzdkwvQJxWbk6+0t5D7mhOxekSnJv7EhlH3qGJOZx8PGw2KDnsr51DDuDqKdXI0LsU6OFdUwJpG2WrooNqQfZVz7gn4xXwoCguNrJsgTvYNRS4F4hVIlb4RmDMAgTMj\\\\\\\/fGWN5YFpUDWQvGX3+\\\\\\\/7M+TQTnCsBZE6UImOy3h30oTpqQ7LIc7mcN6w4PM7WvKsnB5i2HeA16IX9zaa2dAyoOrX6uNlVUiofPKLVejW\\\\\\\/ygUojzObAlvy+3adDg2RjhOQNeHRnsWl\\\\\\\/b6eku723FKa550vHWSfz27v6HRPHdCVii9Sz3VGUFscmvoaP6TDRMXL6KmpBscg9Pt+rKp0TMI2wyTLpmxEgqWWF38bgwLWpND0sXGchDGk9OihLkfjPrm5NOQ\\\\\\\/rk\\\\\\\/ONwA1sqB\\\\\\\/yHwf2p5zdCHC7XtFXM2cIxq5Wzle8Sdm3D7CXj\\\\\\\/gZUdI60ryNJ6ZupEBBjAB0w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PSkxoZiouTV+UcWl5+WD3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f4739d5f294dce8daebfc7d77f0e9a1ccf18f71a1cca3bf78808ed23520902fa\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Lq4ixIOqv4wO+ZMIZOelFPvBCi1YARcuH5yL6czyjaQrC6jwAGy\\\\\\\/njctiKt+u6jxVvscQFlB2dNdnijCSlc9p7Tdgg\\\\\\\/9ugbdehb38sMqhHF5CM00+CqrE9Mmq3DmpuHWuZuATcI9zZjV8TEFfQDAA+Fo9ozs\\\\\\\/6q5eJsRMz7fkuBZPKceA6ECU\\\\\\\/ru4Qz1I5u2mgKP37gz+yKaNW6nLaBuEQw+ym\\\\\\\/y8ELbLs6CAXMT2WQPHiI+87iPkor9BRu5JUjknPNX7JeRzvK0RrUVYi2uGRdfEL69mXkh2yBpmarvj79ijz4HVBDhphLVytiOLnrmydd92RZh0rzj9mR+irelXimnuCvXJpC9awx2czrSHhfWxTeH0PXrHOo0BXfwvjwAZax\\\\\\\/2wfsSc4+0PkuwfJfwYWRTeiv\\\\\\\/W0yyTbaXL0I9OeCVlp57Ma\\\\\\\/6OQhvJY7U6sGYGEIXGmL0tLKd0BCdwEUDR6YslKKDDZU1z8VMKR7F6fgjsLI2ORfB5scVMA19OpYFaTtmzjlkTevCVlcLrTw4\\\\\\\/7CeUEns2KnVB3VpTWHSvbch3nwvcWg6H+5VyrYByFYxyQfI1lLekvztzsbrPmQV2zsm\\\\\\\/1bqYMUYAinWEJ5QU+y2yFVjIpDFkIZzI4h0ua4aIaebx3wRfOFavsHTZSdhQJFMnlvsA50XSQOKZAOR11+G6Ns++gTe6865MrVtXLq+kDzlhisBrFmYJwwHuKosA8hMvK5t\\\\\\\/G7zHl9GJQNe\\\\\\\/3Hb6GzrJWls8zWBugw3cQbxsR9A9l7NcSN+2HjzF3Jp+K01+QkMmFiuOMSSjugo6tQU+oqHw78Dl4ONn\\\\\\\/2+PFkDWjAIAXnSG+8EmrLWQ8Y24DgGIFESYfVISzXBW\\\\\\\/03daKqaqpAywCnnZ+RIvFvQIY2bxQF4S+s\\\\\\\/paHHpPMQS5FD0ZoUCqLzJOmsRaX56T3lOAakucSgwQn3vXra9UIiPUAm\\\\\\\/RHthISRYDwSMqOsg3NeJuSni3Mu92TKFvvavwUZsv6k6gkfkzrSaekK06o5VbAbLkc1WW\\\\\\\/Ga\\\\\\\/oR+bjBNBZVSvaPQovlYP\\\\\\\/QK\\\\\\\/I9Yyj2k+tcov8Sko3oPt8FBRnXy9bD\\\\\\\/QKr72He0NbQHpxcvkrRx0ISiB2Ao7E3IeXB8xlTYzcvu4YaBvVu1vaWJLh9BVHvgx1HKA2jUAX5RQ02CJudziP5yJ8vHJPoX09zdsCMUcS6RY2f20VMOUSYVaF7IafUZPb++EuFUgEDsuWUiCa0OlOQ+U9NG5hCjO2jOETwskDqjzV7J3pfN59FMKCMPpbDxQhIgext0O+JWHFR9dmHqo3L6iWqsqA9iB4XYl+SSKJFbGDQL4c\\\\\\\/aP5qIlgx1nF05cGuicbwmbmN83hUWr7R5\\\\\\\/6yL\\\\\\\/xAaCLiQmaxeJfwDIVKcoZBsBlRug+xEtroepPR1QCKnMlCB8zChIQ+on6IqAvFs59qwBN0B7Wkzc5zGpkytQcT9tLTp1s6ZS9XO129waihziabMSAtZRKgWzGr+sS3RFKSU68Z6\\\\\\\/KtdzQezw4m+kegcPaWqGTjXjx\\\\\\\/YfAhg1X1xjD+x3KGVrKiqgQTqI88Els7rJ87asrIey88dYLTJqv8FjJ85N6p0vl6fEVADeczROjmcM+LAdSywV7jstfQYTALbd8AplMJb4PBZHBMjY9XCeh7ZzXIOagxP5GfFn+FL9DzhV7LIQaEffry56Uu3Msgvh86rhzbDwtMspjRXwEaDUqjFp8izjJ9qoXIyTOSw5If9xxiwj+oYQystAPV\\\\\\\/GgAjrn3dvoP\\\\\\\/wJqqYthYLQPIL0BbLCG7ESV5\\\\\\\/\\\\\\\/8qya3sH9I7fRkclT7cXIdCP80coqVBjfLEaIZn9tPw6nGeH0xPMZ8lmIigBx59\\\\\\\/TNDfccvA81wFVPucMGLUr6fQR39BFRTfhyXDNgJq9zAFi1ooMNM8T3myz1Bw5dsufBweOAa\\\\\\\/SgiuEfsPdYx4jVENjaqp7WeO5Nj43jh8dgwx0o1np36Gp7c0ELInA1Ew35XTw0gDNtM588l6ouruYg==\\\u0022,\\\u0022iv\\\u0022:\\\u00223UvV6Q0U\\\\\\\/u9CXJ1CBeSUcg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ebd22c99c34648b65a44d42c2cc180aed36e656a1f9b1b777c8f593f14e0def7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459764 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/133" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-61SGnVJ5mMSqnSkLv8aHJpUtsDMgHWsj4-qDYn77g7U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/133" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TGlqkvm9BDoPVdfc8Q0kdKEXc52mh2FydREgFke-4NI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/1/123" title="Identical Substitutions in Magnesium Chelatase Paralogs Result in Chlorophyll-Deficient Soybean Mutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/1/145" title="Parallel Profiling of Fission Yeast Deletion Mutants for Proliferation and for Lifespan During Long-Term Quiescence" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459764" id="node459764" data-pisa="ggg;5/1/133" data-pisa-master="ggg;g3.114.015008" data-apath="/ggg/5/1/133.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (<em>Manihot esculenta</em> Crantz) from 10 Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">International Cassava Genetic Map Consortium (ICGMC)</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">133-144;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459764" style="display: none;"></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/1/133" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459764" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459764/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/133.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459764" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459764/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/133.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459764" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459764/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/133.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459764" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459764/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/1/133.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459764" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459764/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Cassava (<em>Manihot esculenta</em> Crantz) is a major staple crop in Africa, Asia, and South America, and its starchy roots provide nourishment for 800 million people worldwide. Although native to South America, cassava was brought to Africa 400–500 years ago and is now widely cultivated across sub-Saharan Africa, but it is subject to biotic and abiotic stresses. To assist in the rapid identification of markers for pathogen resistance and crop traits, and to accelerate breeding programs, we generated a framework map for <em>M. esculenta</em> Crantz from reduced representation sequencing [genotyping-by-sequencing (GBS)]. The composite 2412-cM map integrates 10 biparental maps (comprising 3480 meioses) and organizes 22,403 genetic markers on 18 chromosomes, in agreement with the observed karyotype. We used the map to anchor 71.9% of the draft genome assembly and 90.7% of the predicted protein-coding genes. The chromosome-anchored genome sequence will be useful for breeding improvement by assisting in the rapid identification of markers linked to important traits, and in providing a framework for genomic selection-enhanced breeding of this important crop.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/composite-genetic-map" class="hw-term hw-article-keyword hw-article-keyword-composite-genetic-map" rel="nofollow">composite genetic map</a></li><li class="kwd"><a href="/keyword/pseudomolecules" class="hw-term hw-article-keyword hw-article-keyword-pseudomolecules" rel="nofollow">pseudomolecules</a></li><li class="kwd"><a href="/keyword/f1-cross" class="hw-term hw-article-keyword hw-article-keyword-f1-cross" rel="nofollow">F1 cross</a></li></ul><p id="p-3">Cassava (<em>Manihot esculenta</em> Crantz) is cultivated as a staple in much of Africa, South America, and Asia because it is easy to grow with limited inputs (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Howeler <em>et al.</em> 2013</a>). Smallholder farmers typically grow cassava in plots of a hectare or less. Its starchy roots can be left in the ground until they are needed, making cassava an excellent food security crop. Because cassava can be clonally propagated, desirable varieties can be genetically fixed immediately and multiplied for distribution. The crop is relatively drought-tolerant, and therefore likely robust to climate change. Cassava is also grown for industrial starch production and biofuel applications, particularly in Southeast Asia (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Howeler <em>et al.</em> 2013</a>).</p><p id="p-4">Despite its advantages, cassava faces several biotic and abiotic challenges. Clonal propagation facilitates the rapid spread of bacterial and viral diseases. Furthermore, roots of most farmer-preferred varieties are nutrient-poor and deteriorate rapidly after harvest, preventing farmers from generating income from the sale of excess crop.</p><p id="p-5">The use of modern genetic and genomic techniques, such as quantitative trait locus (QTL) mapping, genomic selection, genome-wide association studies (GWAS), and genetic engineering, can accelerate the pace of disease resistance locus identification and trait improvement. However, these require a high-quality genome assembly and a dense genetic map. A draft cassava genome assembly was generated and covers 532.5 Mb (69%) of the estimated 770 Mb cassava genome (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Awoleye <em>et al.</em> 1994</a>). This assembly captures half of the genome sequence in the 487 largest scaffolds, all longer than 258 kb, and 90% of the assembly is accounted for in 2654 scaffolds all longer than 23 kb; however, these are not linked to chromosomes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Prochnik <em>et al.</em> 2012</a>). To date, a number of genetic maps have been generated for cassava using different marker systems: restriction fragment length polymorphism (RFLP), random amplified polymorphic DNA (RAPD), microsatellite, and isoenzyme (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fregene <em>et al.</em> 1997</a>); simple sequence repeat (SSR) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Okogbenin <em>et al.</em> 2006</a>); amplified fragment length polymorphism (AFLP) and SSR (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kunkeaw <em>et al.</em> 2010</a>); expressed sequence tag (EST) and EST-SSR (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sraphet <em>et al.</em> 2011</a>); SSR and EST-derived single nucleotide polymorphism (SNP) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rabbi <em>et al.</em> 2012</a>); and GBS-SNP (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rabbi <em>et al.</em> 2014a</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">b</a>). These maps have low marker resolution and/or do not resolve a complete set of linkage groups (LGs) representative of the 2n = 36 karyotype of cassava (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Carvalho and Guerra 2002</a>). Furthermore, the densest GBS-derived SNP map anchors only 313.3 Mb (58.7%) of the reference genome assembly (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Rabbi <em>et al.</em> 2014b</a>). Because there are many cassava breeding programs, working with varying accessions and trait(s) of interest, a broadly useful genetic map should include markers that segregate in diverse populations.</p><p id="p-6">A single biparental cassava cross rarely yields enough progeny to make a dense map and, in any event, would only capture markers from a small sample of haplotypes segregating in the species. We therefore merged 10 maps derived from diverse parents to produce a composite genetic map. To generate such a composite map, we obtained one S1 and nine F1 populations (14 parents total) from African cassava breeding projects. Markers were generated via GBS (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>) and a map was constructed from each of the 10 crosses with JoinMap (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Van Ooijen 2011</a>). These maps were merged with LPmerge (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Endelman and Plomion 2014</a>) to generate a 2412-cM genetic map comprising 18 LGs, in agreement with the number of chromosomes found cytogenetically (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">De Carvalho and Guerra 2002</a>). Furthermore, 71.9% of the genome assembly was anchored to the genetic map. The resulting chromosome-scale assembly will accelerate the application of a wide variety of modern tools for crop improvement.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Generation of mapping populations</h3><p id="p-7">Nine biparental (F1) crosses and one self-pollinated (S1) cross were performed (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Biparental populations NxA, KAR, NCAR, MT, NDLAR, and ARAL were generated from crossing blocks planted in four locations in Tanzania: Naliendele [10°23′ S, 40°09′ E, altitude ∼137 m above sea level (ASL)], Kibaha (6°46′ S, 38°58′ E, ∼162 m ASL), Ukiriguru (2°43′23′′ S, 33°1′39′′ E, 1229 m ASL), and Maruku (1°24′55′′ S; 31°46′48′′ E, 1340 m ASL). Populations 412×425, MP4, and MP5 were generated at the IITA crossing sites in Nigeria. Pedigrees of parents are shown in <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/015008SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/TableS2.pdf">Table S2</a>. Cassava stakes for planting were obtained from research stations or farmers’ fields. For populations developed in Tanzania (with the exception of ARAL), more than 4000 hand pollinations were performed according to <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kawano (1980)</a>. More than 10,000 seeds were generated with more than 1000 seeds per population except for population MT. After 3 months, seeds were sown in trays and raised in a screen house for 1 month before transplanting into the field at Makutupora research station in Dodoma, Tanzania. Poor germination rates resulted in a total of approximately 3500 seedlings, with further losses incurred during field establishment.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465494/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465494/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mapping populations used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">For the TMEB419 S1 cross, cassava stakes for planting were derived from the NRCRI cassava breeding experimental plots and planted at a hybridization plot at Ubiaja, Nigeria. Up to 5000 hand pollinations were performed by selfing the same variety, using it as both male and female parents but with flowers and pollen from different plants. Approximately 800 seeds were generated. Seeds were sown in trays and raised in a screen house for 1 month before transplanting into the field at NRCRI Umudike experimental field in Nigeria. Poor germination rates resulted in a total of approximately 200 seedlings, with further losses incurred during field establishment.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation</h3><p id="p-13">Genomic DNA was isolated from the F1 populations according to <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dellaporta <em>et al.</em> (1983)</a>, with some modifications to allow for processing of many samples in parallel in small volumes. DNA was extracted from population TMEB419-S1 using the DNeasy Plant Mini Kit (Qiagen) following manufacturer protocol. In all cases, young apical leaves were first freeze-dried and then ground using a GenoGrinder beadmill at 1500 strokes/min for 2 min. The NxA, KAR, NCAR, MT, NDLAR, and ARAL populations were initially screened as described by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kawuki <em>et al.</em> (2013)</a>, with approximately 12 SSR markers that were polymorphic among the parents, to detect off-types and selfs. These were removed from the population prior to GBS. Additional off-types, half-sibs, and selfs were later detected by GBS and removed from mapping populations (see below).</p></div><div id="sec-4" class="subsection"><h3>Genotyping-by-sequencing library preparation and sequencing</h3><p id="p-14">GBS library construction was performed at the University of California, Berkeley (UC Berkeley) using a protocol adapted from <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> (2011)</a>. The restriction enzyme <em>Ape</em>KI [New England Biolabs (NEB)] was used in conjunction with the barcode sequences included in the Elshire article. Differences from their protocol following protocol optimization are described below and discussed further in the <em>Results and Discussion</em> section.</p><p id="p-15">Y-shaped adapters were designed based on the Y-shaped Illumina DNA paired-end (PE) adapters (Illumina, Inc.; <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS1.pdf">Figure S1</a>). “Forward” adapter oligos had the sequence 5′ ACACTCTTTCCCTACACGACGCTCTTCCGATCTxxxx, and “reverse” oligos had the sequence 5′ CWGyyyyAGATCGGAAGAGCGGTTCAGCAGGAATGCCGAG; where xxxx represents the 4- to 8-bp barcode and yyyy represents its reverse complement. CWG is the <em>Ape</em>KI-specific overhang. Reverse oligos were phosphorylated at the 5′ end. Adapters were ordered from IDT as a “primer premix plate,” with standard desalting, in a total volume of 50 µl with each oligo at a concentration of 200 µM. Adapters were annealed at 50 µM in TE using a thermocycler: 95° 4 min, ramp −70° at 0.1°/5 sec, 25° 5 min, hold at 4°. Annealing was confirmed by running adapters on a 4% agarose gel next to single-stranded oligos of similar length, as annealed adapters run at a larger size. Annealed adapters were diluted 1:10 and then to 12 ng/µl in TE, and finally to 1.25 ng/µl in 10 mM Tris pH 8. Adapter volumes for the last two dilutions were based on quantitation with Picogreen reagent (Invitrogen) and an FLx800 microplate reader (Biotek Instruments, Inc.).</p><p id="p-16">Libraries comprised 63–96 samples. Sample DNA quantitation was performed with Picogreen. Typically, DNA samples were diluted in water to approximately 20 ng/µl and re-quantitated before library preparation. Digests were performed on a 96-well plate, and each consisted of 100 ng DNA in 20 µl 1× NEBuffer #3 (NEB) with 5 U <em>Ape</em>KI. Three microliters of the desired pre-annealed and diluted adapter was added to each well of digested DNA, followed by ligation mix containing 720 cohesive end units T4 DNA ligase and 5 µl T4 DNA Ligase Reaction Buffer (NEB), to a final ligation volume of 50 µl. Ligations were pooled such that each offspring sample contributed an equal amount of DNA. For parental DNA samples, to ensure adequate sequence coverage, a greater amount of digested/ligated DNA was added to the pool. Pools contained a total amount of 1–2 µg DNA and were purified and concentrated using the MinElute PCR Purification Kit (Qiagen).</p><p id="p-17">Size selection was performed on pooled libraries using a 2% agarose gel run at 140 V. A size fraction of 400–800 bp was excised from the gel and purified via the MinElute Gel Extraction Kit (Qiagen), melting the gel at room temperature to avoid G-C bias (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Quail <em>et al.</em> 2008</a>). Most libraries were amplified with five PCR cycles using Phusion polymerase (NEB), 460 ng of each PCR primer, and an extension time of 45 sec. After PCR amplification, libraries were cleaned using 0.7 volumes AMPure XP SPRI beads (Beckman Coulter, Inc.). Size distributions of Illumina libraries were assayed by the Vincent J. Coates Genomic Sequencing Laboratory (VCGSL) at UC Berkeley using a 2100 Bioanalyzer (Agilent Technologies, Inc.). Library concentrations were determined by the VCGSL using a Qubit (Life Technologies) and quantitative PCR. Quantitative PCR was performed with Kapa Biosystems’ Illumina Library Quantification Kits and Roche LightCycler 480, following all kit protocols. One hundred–basepair paired-end sequencing was performed by the VCGSL on HiSeq 2000 or 2500 instruments (Illumina, Inc.). Some libraries were sequenced more than once, usually because the first run was suboptimal. Sequence read totals for each population are given in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465558/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465558/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Sequence reads and variability</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Genotype calling and filtering</h3><p id="p-19">GBS data were analyzed by pipelining several widely used sequence analysis tools with custom scripts to extract markers with parental genotype combinations useful for the cross-pollinated (CP) genetic mapping strategy implemented by JoinMap (v4.1 2013, July 11 release) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Van Ooijen 2011</a>). An outline of the pipeline is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, and step-by-step command-line instructions are available at <a href="https://bitbucket.org/rokhsar-lab/gbs-analysis">https://bitbucket.org/rokhsar-lab/gbs-analysis</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/133/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Data analysis pipeline used in this study. Using a combination of publicly available and custom tools (gray text), the pipeline starts with sequence data and generates a map for each population (Pop) through a series of analyses (white text on blue). Finally, the maps are merged using LPmerge to generate a single composite map (Endelman and Plomion 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1202962507" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Data analysis pipeline used in this study. Using a combination of publicly available and custom tools (gray text), the pipeline starts with sequence data and generates a map for each population (Pop) through a series of analyses (white text on blue). Finally, the maps are merged using LPmerge to generate a single composite map (Endelman and Plomion 2014).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/133/F1.medium.gif" width="440" height="418"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/1/133/F1.medium.gif" width="440" height="418"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/133/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/133/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465486" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Data analysis pipeline used in this study. Using a combination of publicly available and custom tools (gray text), the pipeline starts with sequence data and generates a map for each population (Pop) through a series of analyses (white text on blue). Finally, the maps are merged using LPmerge to generate a single composite map (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Endelman and Plomion 2014</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">To ensure high-quality data for genetic map estimation, the raw read data were trimmed of adapter sequences with the fastq-mcf tool from the ea-utils package (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aronesty 2011</a>), demultiplexed with an allowance of one mismatch in the barcode sequence using a custom script, and then base-quality–trimmed (Q = 28) and aligned to the cassava v4.1 draft genome assembly using BWA (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li and Durbin 2009</a>). Variants and genotypes were called using the HaplotypeCaller tool from the GATK (v2.7-2) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McKenna <em>et al.</em> 2010</a>) with a minimum mapping quality threshold of 25. Because map estimation software can be sensitive to excessive missing data and genotyping error, genotypes with quality scores and read depths less than 30 and 10×, respectively, were marked as missing data and sites with more than 20% missing genotypes were discarded. To avoid spurious genotype calls within repeat regions, sites with average depth more than approximately 120 reads per individual or with log<sub>10</sub>(GATK HaplotypeScore+1) values more than 0.5 were removed. A maximum log<sub>10</sub>(GATK HaplotypeScore+1) value of 1.0 was enforced for the ARAL and 412×425 populations because they had been sequenced more deeply.</p><p id="p-22">Further filtering removed individuals with insufficient data or half-sib, off-type, or (for biparental crosses) self-pollinated individuals (see below). The chi-squared <em>P</em> value for F1 Mendelian ratios was then calculated for each variant site, and sites with <em>P</em> &lt; 0.05 were discarded. Parental genotypes were then inferred from the segregation pattern of each marker in the progeny and this information was used to impute missing parental genotypes in either parent. Markers were then grouped into nonoverlapping 50-kb bins and one marker of each segregating type (<em>i.e.</em>, lm × ll, nn × np, hk × hk, ef × eg, and ab × cd) with the most genotyped individuals was chosen to represent each bin.</p></div><div id="sec-6" class="subsection"><h3>Identification of off-types, half-sibs, and selfs in the progeny</h3><p id="p-23">Cassava farmers prefer to grow several varieties in a field at one time, and this often leads to volunteer seedlings and, hence, genetic variability within nominally clonally propagated cassava. This can create spurious genetic variation within varieties used to generate populations. We therefore checked the progeny from each biparental mapping population for the presence of off-types, half-sibs, and selfs. We used metrics of relatedness to determine whether a progeny was a full-sib F1 (or S1). First, we used the frequency of genotypes violating, or inconsistent with, the expected Mendelian patterns of segregation. We defined this as the fraction of genotypes homozygous for the minor/unshared parental allele (or contained nonparental alleles) per individual. Individuals with a rate of Mendelian inconsistency in excess of 0.005 at a minimum genotype quality of 30 were discarded (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/133/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of parentage and sibling relationships. (A) The fraction of non-Mendelian genotypes detected in each individual in a population is plotted as a function of the minimum genotype quality (GQ) threshold. Off-types can be detected by a consistently high rate of Mendelian violation (black dotted lines), while, for legitimate progeny (solid gray), the non-Mendelian genotype rate is consistently lower than that of the putative parents (solid black lines). (B) A bivariate clustering analysis was performed on each population to verify parentage and full-sibling relationships. In this plot, we show the ARAL population as an example. Pairwise kinship coefficients phi are calculated between progeny and parents (Manichaikul et al. 2010). Each putative progeny is represented as a point in two-dimensional space colored by its cluster assignment: full-sib F1 progeny are in red and S1 progeny of AR40-6 (therefore unrelated to Albert) are in green; individuals consistent with being progeny from a full sib of Albert crossed to AR40-6 are in purple; individuals consistent with being the progeny from an S1 of AR40-6 being crossed to Albert are in blue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1202962507" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of parentage and sibling relationships. (A) The fraction of non-Mendelian genotypes detected in each individual in a population is plotted as a function of the minimum genotype quality (GQ) threshold. Off-types can be detected by a consistently high rate of Mendelian violation (black dotted lines), while, for legitimate progeny (solid gray), the non-Mendelian genotype rate is consistently lower than that of the putative parents (solid black lines). (B) A bivariate clustering analysis was performed on each population to verify parentage and full-sibling relationships. In this plot, we show the ARAL population as an example. Pairwise kinship coefficients &lt;em&gt;phi&lt;/em&gt; are calculated between progeny and parents (Manichaikul &lt;em&gt;et al.&lt;/em&gt; 2010). Each putative progeny is represented as a point in two-dimensional space colored by its cluster assignment: full-sib F1 progeny are in red and S1 progeny of AR40-6 (therefore unrelated to Albert) are in green; individuals consistent with being progeny from a full sib of Albert crossed to AR40-6 are in purple; individuals consistent with being the progeny from an S1 of AR40-6 being crossed to Albert are in blue.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/133/F2.medium.gif" width="234" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/1/133/F2.medium.gif" width="234" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/133/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/133/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465524" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Analysis of parentage and sibling relationships. (A) The fraction of non-Mendelian genotypes detected in each individual in a population is plotted as a function of the minimum genotype quality (GQ) threshold. Off-types can be detected by a consistently high rate of Mendelian violation (black dotted lines), while, for legitimate progeny (solid gray), the non-Mendelian genotype rate is consistently lower than that of the putative parents (solid black lines). (B) A bivariate clustering analysis was performed on each population to verify parentage and full-sibling relationships. In this plot, we show the ARAL population as an example. Pairwise kinship coefficients <em>phi</em> are calculated between progeny and parents (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Manichaikul <em>et al.</em> 2010</a>). Each putative progeny is represented as a point in two-dimensional space colored by its cluster assignment: full-sib F1 progeny are in red and S1 progeny of AR40-6 (therefore unrelated to Albert) are in green; individuals consistent with being progeny from a full sib of Albert crossed to AR40-6 are in purple; individuals consistent with being the progeny from an S1 of AR40-6 being crossed to Albert are in blue.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Second, a bivariate clustering analysis was performed with the Mclust (v4.2) R package using as the similarity measure the kinship coefficient <em>phi</em> calculated by vcftools (v0.1.12) (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Manichaikul <em>et al.</em> 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Danecek <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fraley <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">R Core Team 2014</a>). Values of <em>phi</em> were calculated between all individuals and the putative parents, with each parent constituting an axis of potential genetic contribution (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Mclust calculates the optimal number of clusters under a Bayesian Information Criterion (BIC) regime for a given set of candidate multivariate Gaussian mixture models, the parameters for which are estimated via the Expectation-Maximization algorithm. Individuals are then classified into clusters by choosing the maximum conditional probability among the Gaussian distributions of the chosen mixture model (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Fraley <em>et al.</em> 2012</a>). Individuals belonging to a single cluster, or several overlapping clusters, near (X,Y) = (0.25,0.25) were kept for downstream analysis. We performed a univariate analysis of the S1 population data because Mclust assumes that the two variables of the bivariate normal distribution do not have covariance near unity, and performing a bivariate analysis on an S1 population would violate this assumption.</p></div><div id="sec-7" class="subsection"><h3>Calculating genetic maps for single populations</h3><p id="p-26">A genetic map was produced for each mapping population using JoinMap software to estimate linkage, map order, and distance. To make mapping calculations more tractable, only one marker was retained from a set of markers at the same genetic position and individuals with more than 50% missing data were removed. For each population, the minimum LOD threshold for grouping was determined by identifying grouping tree branches with stable marker numbers over increasing consecutive LOD values. Groups with three or more markers at the chosen minimum LOD threshold were retained for mapping. Specific LOD thresholds applied to each map are available in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Marker order and distances were determined using JoinMap’s Maximum Likelihood mapping algorithm for cross-pollinated (CP) populations. Default mapping parameters were assumed with the following modifications: spatial sampling thresholds reduced to 0.050, 0.025, 0.015, 0.010, and 0.005, and the number of Monte Carlo EM cycles increased to a value of 7. Maps were then compared with each other as a check for internal consistency (see next section) and genetically redundant markers that had been removed earlier were reincorporated into the component maps by assigning each marker to the LG and genetic position of the marker that was physically closest to it. The scaffolds in the draft assembly were used to determine physical distance. Finally, each map was converted from Haldane to Kosambi map units prior to merging.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465538/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465538/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Mapping parameters and statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Quality control of component maps</h3><p id="p-28">Maps containing inter-marker genetic distances of 50 cM or greater were remapped with increased simulated annealing chain length and stopping chains. This process was iterated until maps no longer contained such distances or JoinMap ran out of memory. If these distances could not be removed before memory failure, then individual markers near the interval with extreme values of “Nearest Neighbor Fit” (a measure reported by JoinMap to indicate the compatibility of one marker between two flanking markers) were removed by trial and error (recalculating the genetic map after each iteration) (removed markers are listed in <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FileS1.txt">File S1</a>). If this failed to fix the gap, then the LG was split manually into two sets of parent-specific markers (<em>i.e.</em>, lm × ll and nn × np). Separate linkage maps were calculated for each parent using the methods described above. Single-parent maps that broke at the location of the initial marker gap were included for merging.</p><p id="p-29">The relative correspondences and orientations of component LGs between populations were determined by examining dot plots, plotting the genetic positions of markers between pairs of maps (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). LGs were split or joined by following majority rule of the set of cognate LGs. For example, a component LG sharing markers with two LGs in each of the other component maps was split; multiple smaller LGs in one component map sharing markers with a single LG in each of the other component maps were joined. In joining, an interval was inserted equal to the average genetic distance found in the other corresponding component groups.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/133/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise comparison of single population maps. (A) An example of a pairwise comparison between maps. Every marker is plotted at a position corresponding to its genetic distance in each map (for shared markers) or along the axis (for markers unique to one map). Shared markers reveal the correspondences and relative orientations between LGs produced by JoinMap (arbitrarily numbered). Runs of shared markers appearing as approximately straight, continuous lines demonstrate marker orders consistent between the two maps; a positive or negative gradient indicates identical or opposite orientation, respectively, in the two maps. In addition, LGs to be joined or split are revealed, respectively, as multiple unusually small LGs in one map corresponding to a single LG in another map (NCAR LG 3, and KAR LGs 13 and 14) and as a single unusually large LG in one map corresponding to multiple LGs in the other (NCAR LGs 7 and 9, and KAR LG 1). (B) An example of a &#x201C;V&#x201D;-shaped dot plot pattern, typically observed in LGs, that required (and could be corrected with) JoinMap parameters that increased the sensitivity (see Materials and Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1202962507" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pairwise comparison of single population maps. (A) An example of a pairwise comparison between maps. Every marker is plotted at a position corresponding to its genetic distance in each map (for shared markers) or along the axis (for markers unique to one map). Shared markers reveal the correspondences and relative orientations between LGs produced by JoinMap (arbitrarily numbered). Runs of shared markers appearing as approximately straight, continuous lines demonstrate marker orders consistent between the two maps; a positive or negative gradient indicates identical or opposite orientation, respectively, in the two maps. In addition, LGs to be joined or split are revealed, respectively, as multiple unusually small LGs in one map corresponding to a single LG in another map (NCAR LG 3, and KAR LGs 13 and 14) and as a single unusually large LG in one map corresponding to multiple LGs in the other (NCAR LGs 7 and 9, and KAR LG 1). (B) An example of a &#x201C;V&#x201D;-shaped dot plot pattern, typically observed in LGs, that required (and could be corrected with) JoinMap parameters that increased the sensitivity (see &lt;em&gt;Materials and Methods&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/133/F3.medium.gif" width="253" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/1/133/F3.medium.gif" width="253" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/133/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/133/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465477" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Pairwise comparison of single population maps. (A) An example of a pairwise comparison between maps. Every marker is plotted at a position corresponding to its genetic distance in each map (for shared markers) or along the axis (for markers unique to one map). Shared markers reveal the correspondences and relative orientations between LGs produced by JoinMap (arbitrarily numbered). Runs of shared markers appearing as approximately straight, continuous lines demonstrate marker orders consistent between the two maps; a positive or negative gradient indicates identical or opposite orientation, respectively, in the two maps. In addition, LGs to be joined or split are revealed, respectively, as multiple unusually small LGs in one map corresponding to a single LG in another map (NCAR LG 3, and KAR LGs 13 and 14) and as a single unusually large LG in one map corresponding to multiple LGs in the other (NCAR LGs 7 and 9, and KAR LG 1). (B) An example of a “V”-shaped dot plot pattern, typically observed in LGs, that required (and could be corrected with) JoinMap parameters that increased the sensitivity (see <em>Materials and Methods</em>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Merging maps with LPmerge</h3><p id="p-31">We chose the ARAL map to be the reference for numbering and orienting LGs during merging (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/TableS1.pdf">Table S1</a>) because it was deeply sequenced, and this map had the best agreement between genetic and physical marker order. The initial numbering of LGs followed that produced by JoinMap. As described above (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), a set of corresponding LGs and orientations was constructed for each of the 18 chromosomes in cassava. Each of these sets of corresponding LGs was merged into a composite map with LPmerge v1.4 (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Endelman and Plomion 2014</a>), with maps being weighted by population size. LPmerge was run 10 times with the maximum interval parameter set to each of the values in the range 1–10. We then chose the merged map with the value of maximum interval that gave the shortest total composite map length, as described in the documentation for LPmerge. For most LGs, the value of maximum interval was 1 (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/TableS1.pdf">Table S1</a>).</p><p id="p-32">Each merged map was plotted along with all its component maps on a plot of marker number against genetic map distance. We noticed that markers at the end of some LGs (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS2.pdf">Figure S2</a>) were tens of cM distant from adjacent markers. These markers were found to be present in only one contributing map and were distorting the merged map distances calculated by LPmerge. To fix this, terminal markers from single contributing maps were removed and another round of merging with LPmerge was performed. This process was repeated until there were no singleton markers at the end of a LG.</p></div><div id="sec-10" class="subsection"><h3>Correcting scaffolding mis-assemblies</h3><p id="p-33">Mapped markers from all 10 maps were considered simultaneously. Scaffolds from the v4.1 assembly (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Prochnik <em>et al.</em> 2012</a>) containing markers that mapped to different LGs were broken as long as the markers on the different LGs were derived from at least two maps or there were at least two markers from a single map. Mean pairwise linkage disequilibrium (LD) <em>r</em><sup>2</sup> statistics (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Purcell <em>et al.</em> 2007</a>) were calculated using vcftools (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Danecek <em>et al.</em> 2011</a>) for all variants derived from putatively mis-assembled scaffolds. Scaffolds were broken at scaffolding gaps within each region flanked by markers of different LG assignments if no evidence of LD was observed. If multiple scaffolding (<em>i.e.</em>, sequence) gaps existed between flanking markers, then the gap with the fewest supporting mate-pairs used for scaffolding the v4.1 assembly was broken. If multiple, equally supported gaps existed, then the scaffold was broken at the largest gap. Minority markers from unbroken candidate mis-assembled scaffolds were removed.</p></div><div id="sec-11" class="subsection"><h3>Scaffold anchoring and orienting</h3><p id="p-34">After breaking mis-assembled scaffolds, the v4.1 genome assembly was anchored to the genetic map and joined with 1000 Ns to produce pseudomolecules. Scaffold order was determined by median genetic position and orientation by the sign of Kendall’s rank correlation coefficient (<em>tau</em>) between physical and genetic positions. Scaffolds that could not be assigned to any LG (due to ambiguity in grouping or lack of markers) were not incorporated into the pseudomolecules and retained their original identifiers from the v4.1 assembly. Scaffolds that were anchored but could not be oriented were incorporated in their original (<em>i.e.</em>, arbitrary) orientations. The LGs were then ordered by decreasing genetic size and numbered with Roman numerals to provide a canonical nomenclature (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/133/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker distribution in the composite map. The scale on the left shows the map distance in cM. Our composite map consists of 18 LGs with a marker density of 1.95 nonredundant markers per cM. Previous maps did not recapitulate 18 LGs with a clear one-to-one mapping to our map, so we adopted a Roman numeral numbering system, ordered by decreasing genetic size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1202962507" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Marker distribution in the composite map. The scale on the left shows the map distance in cM. Our composite map consists of 18 LGs with a marker density of 1.95 nonredundant markers per cM. Previous maps did not recapitulate 18 LGs with a clear one-to-one mapping to our map, so we adopted a Roman numeral numbering system, ordered by decreasing genetic size.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/133/F4.medium.gif" width="440" height="414"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/1/133/F4.medium.gif" width="440" height="414"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/133/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/133/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465559" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Marker distribution in the composite map. The scale on the left shows the map distance in cM. Our composite map consists of 18 LGs with a marker density of 1.95 nonredundant markers per cM. Previous maps did not recapitulate 18 LGs with a clear one-to-one mapping to our map, so we adopted a Roman numeral numbering system, ordered by decreasing genetic size.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results-discussion" id="sec-12"><h2 class="">Results and Discussion</h2><p id="p-36">To produce a robust and dense genetic map that can be used to follow traits segregating in diverse mapping populations, we generated 10 mapping populations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and genotyped them using a modified GBS approach (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>). We were stringent in selecting markers to both mitigate the effects of missing data inherent in a GBS approach and to ensure that variants were segregating in a Mendelian fashion. We then generated 10 independent genetic maps from these populations. Comparison between them allowed us to identify cases where the standard map estimation approach was unable to resolve consistent LGs, and reconciling these differences produced a highly concordant set of maps. After constructing maps using a reduced set of stringently selected markers, we reintroduced genetically redundant markers into the maps, and these individual component maps were then integrated into a single composite framework map with the expected 18 LGs. Finally, we used this map to organize the v4.1 draft cassava genome sequence (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Prochnik <em>et al.</em> 2012</a>) into 18 pseudomolecules, anchoring 90.7% of the predicted genes onto LGs.</p><div id="sec-13" class="subsection"><h3>Mapping populations</h3><p id="p-37">We analyzed nine F1 and one S1 mapping populations derived from 14 diverse cultivated varieties of cassava segregating for agriculturally important traits (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). One population included here was recently used for a QTL analysis for cassava mosaic disease (CMD) resistance (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Rabbi <em>et al.</em> 2014b</a>). Due to their desirable agricultural properties, some cultivars were used as parents in multiple crosses. Six populations from the Ferguson group at IITA were selected specifically for segregation of resistance to cassava brown streak disease. Several of these parents have a known wild species component in their pedigree (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/TableS2.pdf">Table S2</a>).</p></div><div id="sec-14" class="subsection"><h3>Reduced representation sequencing</h3><p id="p-38">Several groups have developed inexpensive methods for generating a reduced representation of a genome with a restriction enzyme and sequencing to obtain genotypes (genotyping-by-sequencing or GBS) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altshuler <em>et al.</em> 2000</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>). To generate mapping markers in cassava, we adapted the GBS approach of <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> (2011)</a> (<em>Materials and Methods</em>). We used Y-shaped GBS adapters to ensure that all pieces of DNA with adapter ligated to both ends could cluster and be sequenced. Because the same barcode was added to both ends of the DNA extracted from a given individual (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS1.pdf">Figure S1</a>), checking for the identity of barcodes from both ends provided additional quality control. To save time, we dispensed with the DNA and adapter drying step before digestion. To allow for robust ligation, we phosphorylated the “reverse” oligos of our adapters at the 5′ end (<em>Materials and Methods</em>, <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS1.pdf">Figure S1</a>); although this can increase the number of adapter dimers, these dimers are effectively removed by size selection as noted below. To maximize the number of DNA fragments that are ligated to adapters, we used adapters in excess. This also prevents the formation of DNA concatemers, which would confound downstream analysis of reads. To increase the accuracy of mapping reads to the genome, the probability of detecting variation adjacent to a cut site, and the amount of data, we performed 100-bp paired-end sequencing, rather than the single-end 86-bp sequencing of <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> (2011)</a>.</p><p id="p-39">During protocol optimization, we found that a gel-based size selection step led to more robust amplification of the desired size fraction. Fragments of sizes 400–800 bp cluster well on the flow cell but are unlikely to contain adapter-mers. A gel-based size selection also provides for modularity in the GBS approach: excising a narrower size range from the gel can be a simple way to sample fewer sites in the genome if fewer markers or more depth per site are needed, and/or to facilitate multiplexing more samples per lane. In the PCR, we used Phusion polymerase rather than <em>Taq</em> to minimize error, and we found that increasing the amount of PCR primer yielded more amplified library. We reduced the number of PCR cycles to five to reduce bias that can interfere with accurate variant calling. The 0.7× ratio of SPRI beads to DNA removes fragments less than 300 bp in length (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Quail <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lennon <em>et al.</em> 2010</a>), and thus effectively removes any remaining adapter-mers and excess PCR primer (neither of which are detected in appreciable amounts by Bioanalyzer on completed libraries).</p></div><div id="sec-15" class="subsection"><h3>Genotyping-by-sequencing performance in cassava</h3><p id="p-40">Our adapted GBS protocol makes use of the <em>Ape</em>KI restriction enzyme used in the protocol developed by <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> (2011)</a> and recommended by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hamblin &amp; Rabbi (2014)</a> for cassava. This enzyme and our method of library preparation allowed us to sample, per population, an estimated 85.5k restriction cut sites (with 10 reads or more) distributed throughout the genome on 5900 scaffolds (approximately 173 cut sites/Mbp) and sampling approximately 42k variable loci. Multiplexing up to 96 samples per lane of sequencing was effective for our outbred populations: we obtained an average of approximately 3.7 million reads per individual in a typical 96-plex sequencing run.</p></div><div id="sec-16" class="subsection"><h3>Genotype calling, filtering, and quality control</h3><p id="p-41">We genotyped all individuals against the currently available v4.1 draft cassava genome assembly with the GATK (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Mckenna <em>et al.</em> 2010</a>). Based on an examination of the relationship between minimum genotype quality (GQ) threshold and the rate of violation of Mendelian segregation, we set a minimum GQ of 30, which effectively filtered low-confidence genotype calls (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). In addition, we removed 44 individuals with excessive genotyping error or in which the pattern of marker segregation was inconsistent with that expected in an F1 or S1 population.</p><p id="p-42">Because botanical seed-based fecundity is low in cassava, generating large mapping populations often requires crossing multiple cloned parental genotypes over an extended period of time, increasing the likelihood of error in tracking parental or progeny genotypes. We therefore performed a relatedness analysis (<em>Materials and Methods</em>) to identify and remove individuals that were not part of the intended cross (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Across the 10 populations, we found 14 individuals that were unrelated to both parents and 68 related to only a single parent (26 selfs, 2 clones, 40 half-sib offspring), and identified 119 additional individuals with more complex relationships not suitable for mapping. Individuals identified by the relatedness and/or the Mendelian violation analyses were removed from further analysis. The number of progeny sequenced and used for map construction from each population is listed in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. In one extreme case, none of the offspring in the MT population was related to the pollen parent that was sequenced [putatively TMS4(2)1425] (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS3.pdf">Figure S3</a>); furthermore, neither these offspring nor the putative TMS4(2)1425 parent matched the TMS4(2)1425 parent of the 412×425 population. We could, however, confirm that most members of the MT population were full sibs. Given this fact, it was nevertheless possible to statistically infer reliable genotypes for the true pollen parent and estimate the combined genetic map for the population, as was performed for all other populations included in this study (see next section). Each population of verified full-sib progeny was then subjected to a significance threshold (<em>P</em> &lt; 0.05) for deviation from Mendelian genotype frequencies, from which we obtained approximately 4400 segregating sites per population (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-17" class="subsection"><h3>Calculating genetic maps</h3><p id="p-43">Genetic maps were estimated for each population using the Maximum Likelihood mapping algorithm for CP populations implemented by JoinMap (v4.1). Forty-four (22.6%) component LGs contained intervals 50 cM or larger that could not be corrected by increasing the simulated annealing parameters and required manual curation. In this curation, markers that were causing the large gaps were identified by trial-and-error and removed. A total of 179 (out of 24,403 mapped) markers were discarded (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FileS1.txt">File S1</a>). After this step, one LG still contained a large gap. For this LG, a separate map was made from the markers unique to each parent and the LGs split into two pieces at the position of the gap. Subsequently, shared markers were reincorporated based on their scaffold coordinates.</p></div><div id="sec-18" class="subsection"><h3>Resolving discrepancies between maps, refining maps, and gauging accuracy</h3><p id="p-44">Comparing the 10 component linkage maps with each other allowed us to identify and correct inconsistencies, including incorrectly split or joined LGs. We were able to verify long-range colinearity as the median number of markers shared between any two maps was 793 (quartiles Q1 = 656 and Q3 = 983), or approximately 44 per LG (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Occasionally, the dot plot pattern of a LG was “V”-shaped (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>), indicating inconsistent marker order in one of the maps. Because we could compare 10 independently constructed maps with each other, it was straightforward to identify those specific maps that differed from the consensus. In all but two cases (see below), the discrepant “V”-shaped dot plots were resolved into colinear relationships simply by extending the simulated annealing in JoinMap. Thus, comparison among the various maps allowed us to identify cases where JoinMap had not yet converged to a final marker ordering, and to ultimately reach consistent convergence relative to other maps.</p><p id="p-45">Most LGs had a one-to-one correspondence with an LG in each of the other populations. However, in a total of 11 cases, this was not true and we used a majority rule approach to decide whether the component LGs should be split (two cases) or joined (nine cases). In one additional case, the decision to break or join was ambiguous, because five populations each contained two LGs that together corresponded to single LGs in the other five populations. We resolved this ambiguity by jointly examining the dot plots for the two-LG component maps against a corresponding single-LG map. From this, we discovered that none of the breaks in the different two-LG component maps occurred at the same genetic position. It is therefore likely that the LGs should be joined in the two-LG component maps to generate single LGs. Finally, three component LGs were not included in the merging because they could not be resolved into a map with a linear relationship with the corresponding LG from the other maps, and four were discarded because, although they could be matched with LGs in other maps, they contained too few markers to be confidently oriented.</p><p id="p-46">Comparing maps with one another using dot plots provides a visual measure of internal consistency, whereas accuracy of marker order may be estimated by comparison to the physical sequence. We calculated Kendall’s rank correlation coefficient (<em>tau</em>) between physical and genetic positions on scaffolds that contained 10 or more markers and were not broken (described below). Across all maps and scaffolds examined, the median value <em>tau</em> was 0.84 (quartiles Q1 = 0.748 and Q3 = 0.908), indicating good agreement between physical and genetic positions.</p></div><div id="sec-19" class="subsection"><h3>Reincorporation of genetically redundant markers</h3><p id="p-47">To more fully represent the genetic diversity found within the mapping populations, 8418 markers that had been excluded earlier based on their genetic redundancy (<em>Materials and Methods</em>) were reincorporated into the composite map. By creating datasets with the redundant markers included before and after map integration, and then by comparing their median Kendall’s rank correlation coefficients <em>tau</em> (for 46 scaffolds with 50 or more markers, totaling 64.5 Mb of sequence), we found that including the redundant markers prior to map integration increased the marker order accuracy (<em>tau</em> = 0.765 <em>vs. tau</em> = 0.726). This improvement arises because the same genetically redundant marker is not always chosen to represent its genetic position in all component maps, and reincorporating redundant such markers increases the number of marker order constraints used by LPmerge (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Endelman and Plomion 2014</a>) when a marker order conflict is encountered during map integration.</p></div><div id="sec-20" class="subsection"><h3>An integrated framework map</h3><p id="p-48">With the exception of the three LGs noted above, the component maps were colinear with each other (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>). We next combined them into a single composite map. We merged genetic maps by finding shared markers between individual maps. Ideally, all shared markers would be colinear, but in many situations discrepancies needed to be resolved. Sometimes these discrepancies could be due to rearrangements in the genome of one of the parents, but most often they are due to inaccurate genetic distances between markers because of insufficient recombinations, the stochastic nature of recombination, and errors introduced in individual map construction by missing and/or erroneous marker data. We used LPmerge (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Endelman and Plomion 2014</a>) to generate a composite map from the 10 component maps we had generated because it merges maps without recalculating recombination frequencies, drastically reducing the computational time required. LPmerge generates a consensus map with the minimum absolute error relative to the component maps while preserving the marker order by imposing linear inequality constraints and deleting a minimum number of constraints if a marker order conflict is observed (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Endelman and Plomion 2014</a>).</p><p id="p-49">Combining maps from multiple independent crosses has the advantage of increasing the genetic diversity that is captured in the map, increasing support for marker order and position, and allowing markers from a single map to be placed relative to other markers. Composite maps have been generated from six Diversity Array Technology [DArT]-based and RFLP-based maps for sorghum (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mace <em>et al.</em> 2009</a>), six SSR-based maps for pigeonpea (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bohra <em>et al.</em> 2012</a>), and 11 SSR-based maps for groundnut (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gautami <em>et al.</em> 2012</a>), but, to our knowledge, no attempt has been made to integrate this many GBS-SNP–based genetic maps.</p><p id="p-50">After merging, we noticed that markers at the ends of LGs that belonged to a single map were placed on the merged map tens of cM from their neighbors (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS2.pdf">Figure S2</a>). To improve the merging, we removed terminal markers belonging to a single map and repeated the merging until no further singleton markers remained. This removed a total of 182 markers (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/TableS1.pdf">Table S1</a>) to generate a 2412-cM map containing 22,403 markers, averaging 134 cM and 1245 markers per LG (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FileS2.txt">File S2</a>). The SNP density (∼9 SNPs per cM) is substantially higher than that of the previous densest map (mean inter-SNP distance = 0.52 cM) (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Rabbi <em>et al.</em> 2014b</a>), although this straightforward statistic does not account for the fact that many of the markers are genetically redundant (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). For the first time, a cassava genetic map recapitulates the coherent set of 18 LGs matching the cassava chromosomes (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465505/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465505/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Linkage groups in the composite map</span><div class="sb-div caption-clear"></div></div></div><p id="p-52">We observed a steady increase (albeit with diminishing returns) in the number of markers and amount of assembled sequence that can be placed as a function of the number of mapping populations that were used (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We note, however, that even with the addition of the tenth component map, we were still adding markers and anchoring more genome sequence (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), an observation that may be of interest to scientists building genetic maps for diverse plants. Future work will merge additional maps to take advantage of this feature. However, because the majority of markers were present in only one map (12,474 out of 22,403), they may be placed inaccurately in the merged map because they are only constrained by a single component map. This will be improved by the generation of a less fragmented genome assembly.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/133/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additional maps incorporate more markers, scaffolds, and anchored bases. These plots show the effects of adding maps to the framework map. Each additional map incorporates more genetically nonredundant markers (A) into the framework map, but the number of scaffolds incorporated is saturating (B) and the number of mapped bases (C) is reaching a plateau. This is because the scaffolds being added in later maps are getting smaller and smaller and, hence, adding ever fewer bases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1202962507" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Additional maps incorporate more markers, scaffolds, and anchored bases. These plots show the effects of adding maps to the framework map. Each additional map incorporates more genetically nonredundant markers (A) into the framework map, but the number of scaffolds incorporated is saturating (B) and the number of mapped bases (C) is reaching a plateau. This is because the scaffolds being added in later maps are getting smaller and smaller and, hence, adding ever fewer bases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/133/F5.medium.gif" width="426" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/1/133/F5.medium.gif" width="426" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/133/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/133/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465591" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-53" class="first-child">Additional maps incorporate more markers, scaffolds, and anchored bases. These plots show the effects of adding maps to the framework map. Each additional map incorporates more genetically nonredundant markers (A) into the framework map, but the number of scaffolds incorporated is saturating (B) and the number of mapped bases (C) is reaching a plateau. This is because the scaffolds being added in later maps are getting smaller and smaller and, hence, adding ever fewer bases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">All maps were, to a large extent, colinear with the integrated framework map (<a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS4.pdf">Figure S4</a>); however, component LGs from NxA (<em>e.g.</em>, Chromosome II; <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS4.pdf">Figure S4</a>B) as well as 412×425 (<em>e.g.</em>, Chromosome XVII; <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FigureS4.pdf">Figure S4</a>Q) displayed notable divergence from this, likely because these maps were of lower quality. Finally, across the whole genome, we observed four gaps larger than 10 cM (terminal portion of Chromosome V; two on Chromosome III and one on Chromosome VIII). These could be recombination hotspots or regions that are identical-by-descent and thus lack polymorphisms.</p></div><div id="sec-21" class="subsection"><h3>Generation of chromosome pseudomolecules</h3><p id="p-55">One of our goals was to organize the fragmented draft genome assembly (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Prochnik <em>et al.</em> 2012</a>) into chromosome-scale sequence; therefore, it was necessary to first detect possible discrepancies between the genetic map and physical sequence. Of the 1347 scaffolds that contained multiple markers (median size = 174 kb), 41 scaffolds contained markers along their lengths that mapped to different LGs. Because all component maps used to generate the composite map were internally consistent, these discrepancies were unlikely to be due to errors in the genetic map and suggested a sequence mis-assembly in the scaffolds in question. Manual review, along with the aid of linkage disequilibrium and scaffolding information, allowed us to break likely misjoins where weak scaffolding linkages had been made, often in regions with a high density of scaffolding gaps. After breaking, we then organized the sequence assembly into pseudomolecules by anchoring the scaffolds onto LGs using their median genetic position. The resulting chromosome-scale sequences incorporate 1608 scaffolds and 382 Mb (71.9%) of the v4.1 draft genome sequence (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). This includes 462 (94.9%) of the N50 scaffolds, 1430 (53.9%) of the N90 scaffolds, and 27,825 (90.7%) of the 30,666 predicted protein-coding genes. Of the scaffolds anchored onto the assembly, we could also orient 1024 (63.7%) of them by calculating the Kendall rank correlation coefficient <em>tau</em> between physical and genetic positions. Together, these oriented scaffolds comprise 315 Mb of sequence.</p></div><div id="sec-22" class="subsection"><h3>Concluding remarks</h3><p id="p-56">Here, we present a consensus genetic map of cassava that combines 10 mapping populations. Unlike many previous maps (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fregene <em>et al.</em> 1997</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Okogbenin <em>et al.</em> 2006</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kunkeaw <em>et al.</em> 2010</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sraphet <em>et al.</em> 2011</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Rabbi <em>et al.</em> 2012</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">2014a</a>,<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">b</a>), we recovered the 18 LGs predicted by the karyotype of <em>M. esculenta</em> (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">De Carvalho and Guerra 2002</a>) and at the same time dramatically increased the number of markers to more than 22,400. The majority of genes in the current draft genome were placed on a LG and at their approximate chromosomal position, resulting in the first chromosome-scale assembly of cassava and a map that will serve as a valuable guide for future genome assembly improvements. Our approach can be readily extended to include future mapping populations, although, as seen here, care must be taken to explore discrepancies between biparental or selfed maps that may arise from mistaken parentage, missing or erroneous data, and the details of map construction. Using multiple maps at the same time allows such individual component map problems to be identified and corrected (or excluded from merger).</p><p id="p-57">Within the context of disease threat, climate change and food insecurity, improving cassava for smallholder farmers in developing countries is becoming more urgent. A number of projects and collaborations are already engaged in this task. These initiatives are rapidly incorporating modern genetic tools and techniques such as genomic selection and QTL analysis, techniques that depend on an accurate genetic map and genome assembly. The chromosome-scale genome sequence and composite map we report here will allow cassava geneticists and breeders to generate the robust cassava varieties needed both for food security and to improve the economic conditions of smallholder farmers around the world.</p></div></div><div class="section other1" id="sec-23"><h2 class="">Additional Information</h2><p id="p-58">The composite genetic map is available in <a href="http://www.g3journal.org/content/suppl/2014/12/11/g3.114.015008.DC1/FileS2.txt">File S2</a>, and at CassavaBase (<a href="http://cassavabase.org/cview/map.pl?map_id=3">http://cassavabase.org/cview/map.pl?map_id=3</a>). The pseudomolecule assembly can be downloaded from Phytozome at <a href="http://genome.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=Mesculenta">http://genome.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=Mesculenta</a> under the directory “v5.1_unreleased”. Demultiplexed sequence reads, with the barcode and <em>Ape</em>KI cutsite removed, are available in the NCBI Sequence Read Archive (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) via Study Accession Number SRP051207.</p></div><div class="section other2" id="sec-24"><h2 class="">Members of the International Cassava Genetic Map Consortium</h2><p id="p-59">Consortium members are listed alphabetically, with their affiliations and contributions. <strong>Oluwafemi A. Alaba</strong> [<span class="em-link"><span class="em-addr">O.Alaba{at}cgiar.org</span></span>; International Institute of Tropical Agriculture (IITA), PMB 5320, Ibadan, Oyo State, Nigeria]: DNA extraction, GBS library preparation. <strong>Jessen V. Bredeson</strong> [<span class="em-link"><span class="em-addr">jessenbredeson{at}berkeley.edu</span></span>; University of California, Berkeley, Department of Molecular and Cell Biology, Berkeley, CA, USA]: Design of bioinformatics pipeline, generation of GBS-SNPs, genetic maps, anchored chromosome sequence, project coordination, wrote the paper. <strong>Chiedozie N. Egesi</strong> [<span class="em-link"><span class="em-addr">cegesi{at}yahoo.com</span></span>; National Root Crops Research Institute (NRCRI), Umudike, Nigeria]: NRCRI Breeding team leadership and generation of S1 population, edited the manuscript, contributed writing. <strong>Williams Esuma</strong> [<span class="em-link"><span class="em-addr">esumawilliams{at}yahoo.co.uk</span></span>; National Crops Resources Research Institute (NaCCRI), Namulonge, Uganda]: Development of mapping populations, DNA extractions. <strong>Lydia Ezenwaka</strong> [<span class="em-link"><span class="em-addr">lydiaezenwaka{at}yahoo.com</span></span>; National Root Crops Research Institute (NRCRI), Umudike, Nigeria]: MSc Student. <strong>Morag E. Ferguson</strong> [<span class="em-link"><span class="em-addr">M.Ferguson{at}cgiar.org</span></span>; International Institute of Tropical Agriculture (IITA) Nairobi, Kenya]: Management of IITA Nairobi crosses, scientific leadership, edited the manuscript, contributed writing. <strong>Cindy M. Ha</strong> (<span class="em-link"><span class="em-addr">cindy.ha{at}ucdenver.edu</span></span>; University of California, Berkeley, Department of Molecular and Cell Biology, Berkeley, CA. *Current institution Anschutz Medical Campus, University of Colorado, Denver, CO): Generating GBS-SNPs and component genetic maps. <strong>Megan Hall</strong> (<span class="em-link"><span class="em-addr">mhall45{at}gmail.com</span></span>; University of California, Berkeley, Department of Molecular and Cell Biology, Berkeley, CA): GBS capacity building at UC Berkeley. <strong>Liezel Herselman</strong> (<span class="em-link"><span class="em-addr">HerselmanL{at}ufs.ac.za</span></span>; Department of Plant Sciences, University of the Free State, Bloemfontein, South Africa): PhD supervisor. <strong>Andrew Ikpan</strong> (<span class="em-link"><span class="em-addr">A.Ikpan{at}cgiar.org</span></span>; Headquarters IITA-Nigeria, PMB 5320, Ibadan, Oyo State, Nigeria): Generated IITA biparental mapping populations. <strong>Elly Kafiriti</strong> (<span class="em-link"><span class="em-addr">ekafiriti{at}gmail.com</span></span>; Naliendele Agricultural Research Institute, PO Box 509, Mtwara, Tanzania): Zonal Agricultural Research Director, Southern Zone. <strong>Edward Kanju</strong> (<span class="em-link"><span class="em-addr">e.kanju{at}cgiar.org</span></span>; IITA Mikocheni, PO Box 34441, Dar es Salaam, Tanzania): Selection of parents of mapping populations. <strong>Fortunus Kapinga</strong> (<span class="em-link"><span class="em-addr">fakapinga{at}yahoo.com</span></span>; Naliendele Agricultural Research Institute, Box 509, Mtwara, Tanzania): Development of mapping populations, DNA extractions. <strong>Arthur Karugu</strong> (<span class="em-link"><span class="em-addr">a.karugu{at}cgiar.org</span></span>; CGIAR/CIMMYT Nairobi, Kenya): DNA extractions. <strong>Robert Kawuki</strong> [<span class="em-link"><span class="em-addr">kawukisezi{at}yahoo.com</span></span>; National Crops Resources Research Institute (NaCCRI), Namulonge, Uganda]: Development of mapping populations, DNA extractions. <strong>Bernadetha Kimata</strong> (<span class="em-link"><span class="em-addr">bkimatha{at}yahoo.co.uk</span></span>; Naliendele Agricultural Research Institute, Box 509, Mtwara, Tanzania): DNA extraction. <strong>Paul Kimurto</strong> (<span class="em-link"><span class="em-addr">kimurtopk{at}gmail.com</span></span>; Egerton University Box 536, Njoro, Kenya): MSc supervisor. <strong>Peter Kulakow</strong> (<span class="em-link"><span class="em-addr">P.Kulakow{at}cgiar.org</span></span>; Headquarters IITA-Nigeria, PMB 5320, Ibadan, Oyo State, Nigeria): Mapping population development. <strong>Heneriko Kulembeka</strong> (<span class="em-link"><span class="em-addr">kulembeka{at}yahoo.com</span></span>; Lake Zonal Agricultural Research and Development Institute; Box 1433; Mwanza; Tanzania): Selection of parents of mapping population, development of mapping populations. <strong>Paul Kusolwa</strong> (<span class="em-link"><span class="em-addr">kusolwap{at}gmail.com</span></span>; Sokoine University of Agriculture, Box 3005, Morogoro, Tanzania): MSc supervisor. <strong>Jessica B. Lyons</strong> (<span class="em-link"><span class="em-addr">jblyons{at}berkeley.edu</span></span>; University of California, Berkeley, Department of Molecular and Cell Biology, Berkeley, CA): Optimization of GBS protocol, GBS library preparation and sequencing, project coordination, wrote the paper. <strong>Esther Masumba</strong> (<span class="em-link"><span class="em-addr">emasumba{at}yahoo.com</span></span>; Sugarcane Agricultural Research Institute, Box 30031 Kibaha, Tanzania): Development of mapping populations, DNA extractions. <strong>Albe van de Merwe</strong> (<span class="em-link"><span class="em-addr">albe.vdMerwe{at}up.ac.za</span></span>; Department of Genetics, University of Pretoria, South Africa): PhD supervisor. <strong>Geoffrey Mkamilo</strong> [<span class="em-link"><span class="em-addr">geoffreymkamilo{at}yahoo.co.uk</span></span>; Naliendele Agricultural Research Institute (ARI), Box 509, Mtwara, Tanzania]: Head of Roots and Tuber Crops program at ARI. <strong>Alexander A. Myburg</strong> (<span class="em-link"><span class="em-addr">zander.myburg{at}fabi.up.ac.za</span></span>; Department of Genetics, Forestry and Biotechnology University of Pretoria, South Africa): PhD supervisor. <strong>Ahamefule Nwaogu</strong> [<span class="em-link"><span class="em-addr">ahamefulenwaogu{at}yahoo.com</span></span>; National Root Crops Research Institute (NRCRI), Umudike, Nigeria]: Field management. <strong>Inosters Nzuki</strong> [<span class="em-link"><span class="em-addr">i.nzuki{at}cgiar.org</span></span>; Department of Genetics, University of Pretoria, South Africa 0002, Pretoria; and Biosciences Eastern and Central Africa (BecA) 30709-00100 Nairobi Kenya]: Development of mapping populations, DNA extraction. <strong>Bunmi Olasanmi</strong> (<span class="em-link"><span class="em-addr">bunminadeco{at}yahoo.com</span></span>; University of Ibadan, Ibadan, Nigeria): Generation of S1 mapping population. <strong>Emmanuel Okogbenin</strong> [<span class="em-link"><span class="em-addr">eokogbenin{at}cassavacop.org</span></span>; National Root Crops Research Institute (NRCRI), Umudike, Nigeria]: MSc supervisor. Onyeyirichi Onyegbule [<span class="em-link"><span class="em-addr">princessonyin{at}yahoo.com</span></span>; National Root Crops Research Institute (NRCRI), Umudike, Nigeria]: DNA extractions. <strong>James O. Owuoche</strong> (<span class="em-link"><span class="em-addr">owuoche{at}hotmail.com</span></span>; Egerton University Box 536, Njoro, Kenya): MSc supervisor. <strong>Anthony Pariyo</strong> [<span class="em-link"><span class="em-addr">tkakau{at}yahoo.co.uk</span></span>; National Crops Resources Research Institute (NaCCRI), Namulonge, Uganda]: Development of mapping populations. <strong>Simon E. Prochnik</strong> [<span class="em-link"><span class="em-addr">seprochnik{at}lbl.gov</span></span>; DOE Joint Genome Institute (JGI), Walnut Creek, CA]: Generation of genetic maps, generation of merged composite map, deposition of sequence data in SRA, project coordination, wrote the paper. <strong>Ismail Y. Rabbi</strong> (<span class="em-link"><span class="em-addr">I.RABBI{at}cgiar.org</span></span>; Headquarters IITA-Nigeria, PMB 5320, Ibadan, Oyo State, Nigeria): Development and management of mapping populations IITA Ibadan, edited the manuscript, contributed writing. <strong>Daniel S. Rokhsar</strong> [<span class="em-link"><span class="em-addr">dsrokhsar{at}gmail.com</span></span>; University of California, Berkeley, Department of Molecular and Cell Biology, Berkeley, CA; and Department of Energy Joint Genome Institute (JGI), Walnut Creek, CA]: Scientific and project leadership, wrote the paper. <strong>Steve Rounsley</strong> (<span class="em-link"><span class="em-addr">rounsley{at}email.arizona.edu</span></span>; University of Arizona, Tucson, AZ; and Dow Agrosciences, Indianapolis, IN): Scientific and project leadership. <strong>Kasele Salum</strong> (<span class="em-link"><span class="em-addr">kasele_salum{at}yahoo.com</span></span>; Lake Zonal Agricultural Research and Development Institute, Box 1433, Mwanza, Tanzania): DNA extraction. <strong>Kahya S. Shuaibu</strong> [<span class="em-link"><span class="em-addr">kahyass{at}yahoo.com</span></span>; National Root Crops Research Institute (NRCRI), Umudike, Nigeria]: DNA extraction, laboratory management, GBS library preparation. <strong>Caroline Sichalwe</strong> (<span class="em-link"><span class="em-addr">carosicha{at}gmail.com</span></span>; Sugarcane Agricultural Research Institute, Box 30031 Kibaha, Tanzania): DNA extraction. <strong>Mary Stephen</strong> (<span class="em-link"><span class="em-addr">marystephen49{at}yahoo.com</span></span>; Makutupora Agricultural Research Station, Tanzania): Field technician, managing field cassava seedling nursery.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">The authors thank Richard Harland and Lisa Brunet for early input into optimization of the GBS protocol; Craig Miller lab/Andrew Glazer for reagents and helpful discussion; Minyong Chung, Justin Choi, Karen Lundy, and other members of the VCGSL at UC Berkeley for advice and technical assistance with sequencing; Jeff Endelman for advice on using LPmerge; and Ed Buckler, Jean-Luc Jannink, and Martha Hamblin, for helpful discussion. J.B.L., J.V.B., C.M.H., and work at UC Berkeley were funded by Bill and Melinda Gates Foundation (BMGF) Grant OPPGD1493 to S.R., D.S.R., and the University of Arizona. Development of populations at IITA Ibadan was funded by the CGIAR Research Program on Roots, Tubers, and Bananas. Next Generation Cassava Breeding grant OPP1048542 from BMGF and the United Kingdom Department for International Development supported S.E.P. and work at NRCRI and IITA. Development of the mapping populations in Tanzania was funded by BMGF grant OPPGD1016 to IITA. The work conducted by the U.S. Department of Energy Joint Genome Institute is supported by the Office of Science of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231. This work used the VCGSL at UC Berkeley, supported by National Institutes of Health S10 Instrumentation Grants S10RR029668 and S10RR027303.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-61">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015008/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015008/-/DC1</a></p></li><li class="fn" id="fn-9"><p id="p-62"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-15"><span class="received-label">Received </span>October 15, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-12-08"><span class="accepted-label">Accepted </span>December 8, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 International Cassava Genetic Map Consortium (ICGMC)</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.1" data-doi="10.1038/35035083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollara</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Etten</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldwin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An SNP map of the human genome generated by reduced representation shotgun sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">407</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D407%26rft.issue%253D6803%26rft.spage%253D513%26rft.epage%253D516%26rft.atitle%253DAn%2BSNP%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome%2Bgenerated%2Bby%2Breduced%2Brepresentation%2Bshotgun%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35035083%26rft_id%253Dinfo%253Apmid%252F11029002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35035083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11029002&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089727400050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-5.1.133.2"><div class="cit-metadata unstructured">Aronesty, E., 2011 Command-line tools for processing biological sequencing data. <a href="http://code.google.com/p/ea-utils">http://code.google.com/p/ea-utils</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Awoleye</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duren</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolezel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Nuclear DNA content and in vitro induced somatic polyploidization cassava (<em>Manihot esculenta</em> Crantz) breeding.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D76%26rft.spage%253D195%26rft.atitle%253DNUCLEAR%2BDNA%2BCONTENT%2BAND%2BIN%2BVITRO%2BINDUCED%2BSOMATIC%2BPOLYPLOIDIZATION%2BIN%2BCASSAVA%2B%2528MANIHOT%2BESCULENTA%2BCRANTZ%2529%2BBREEDING%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.4" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.5" data-doi="10.1007/s00122-012-1916-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gnanesh</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byregowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An intra-specific consensus genetic map of pigeonpea [<em>Cajanus cajan</em> (L.) Millspaugh] derived from six mapping populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1325</span>–<span class="cit-lpage">1338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBohra%26rft.auinit1%253DA.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1325%26rft.epage%253D1338%26rft.atitle%253DAn%2Bintra-specific%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Bpigeonpea%2B%255BCajanus%2Bcajan%2B%2528L.%2529%2BMillspaugh%255D%2Bderived%2Bfrom%2Bsix%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1916-5%26rft_id%253Dinfo%253Apmid%252F22772726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1916-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22772726&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.6" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMS8xMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.7" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.8" data-doi="10.1034/j.1601-5223.2002.1360212.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Carvalho</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerra</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Cytogenetics of <em>Manihot esculenta</em> Crantz (cassava) and eight related species.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DDe%2BCarvalho%26rft.auinit1%253DR.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D168%26rft.atitle%253DCytogenetics%2Bof%2BManihot%2Besculenta%2BCrantz%2B%2528cassava%2529%2Band%2Beight%2Brelated%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1601-5223.2002.1360212.x%26rft_id%253Dinfo%253Apmid%252F12369103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1601-5223.2002.1360212.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12369103&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178131100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.9" data-doi="10.1007/BF02712670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A plant DNA minipreparation: version II.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Rep.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BRep.%26rft.volume%253D1%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02712670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02712670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.10" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.11" data-doi="10.1093/bioinformatics/btu091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LPmerge: an R package for merging genetic maps by linear programming.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu091%26rft_id%253Dinfo%253Apmid%252F24532720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTEvMTYyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMS8xMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-other" id="cit-5.1.133.12"><div class="cit-metadata unstructured">Fraley, C., A. E. Raftery, T. B. Murphy, and L. Scrucca, 2012 MCLUST Version 4 for R: Normal Mixture Modeling for Model-Based Clustering, Classification, and Density Estimation. Technical Report no. 597, Department of Statistics, University of Washington.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.13" data-doi="10.1007/s001220050580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fregene</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavarriaga-Aguirre</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A molecular genetic map of cassava (<em>Manihot esculenta</em> Crantz).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XV26800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.14" data-doi="10.1371/journal.pone.0041213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautami</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonceka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sujay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e41213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGautami%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41213%26rft.epage%253De41213%26rft.atitle%253DAn%2Binternational%2Breference%2Bconsensus%2Bgenetic%2Bmap%2Bwith%2B897%2Bmarker%2Bloci%2Bbased%2Bon%2B11%2Bmapping%2Bpopulations%2Bfor%2Btetraploid%2Bgroundnut%2B%2528Arachis%2Bhypogaea%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041213%26rft_id%253Dinfo%253Apmid%252F22815973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22815973&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.15" data-doi="10.2135/cropsci2012.12.0710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of restriction-enzyme choice on properties of genotyping-by-sequencing libraries: A study in cassava (<em>Manihot esculenta</em>).</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.12.0710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.12.0710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000336745400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-5.1.133.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howeler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutaladio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Save and Grow: Cassava</span>, <span class="cit-publ-name">Food and Agriculture Organization of the United Nations</span>, <span class="cit-publ-loc">Rome</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-5.1.133.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawano</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Cassava</span>, pp. <span class="cit-fpage">225</span>–<span class="cit-lpage">233</span> in <span class="cit-source">Hybridization of Crop Plants</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Fehr</span>  <span class="cit-name-given-names">W. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Hadley</span>  <span class="cit-name-given-names">H. H.</span></span>. <span class="cit-publ-name">American Society of Agronomy, Crop Science Society of America</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.18" data-doi="10.1017/S1479262113000014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawuki</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herselman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Labuschagne</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nzuki</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralimanana</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic diversity of cassava (<em>Manihot esculenta</em> Crantz) landraces and cultivars from southern, eastern and central Africa.</span> <abbr class="cit-jnl-abbrev">Plant Genetic Resources</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenetic%2BResources%26rft.volume%253D11%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1479262113000014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1479262113000014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.19" data-doi="10.1111/j.1439-0523.2009.01623.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunkeaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tangphatsornruang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Triwitayakorn</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic linkage map of cassava (<em>Manihot esculenta</em> Crantz) based on AFLP and SSR markers.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D129%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2009.01623.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2009.01623.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.20" data-doi="10.1186/gb-2010-11-2-r15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lennon</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lintner</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A scalable, fully automated process for construction of sequence-ready barcoded libraries for 454.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLennon%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253DR15%26rft.epage%253DR15%26rft.atitle%253DA%2Bscalable%252C%2Bfully%2Bautomated%2Bprocess%2Bfor%2Bconstruction%2Bof%2Bsequence-ready%2Bbarcoded%2Blibraries%2Bfor%2B454.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-2-r15%26rft_id%253Dinfo%253Apmid%252F20137071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-2-r15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20137071&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.21" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMS8xMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.22" data-doi="10.1186/1471-2229-9-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DMace%26rft.auinit1%253DE.%2BS.%26rft.volume%253D9%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DA%2Bconsensus%2Bgenetic%2Bmap%2Bof%2Bsorghum%2Bthat%2Bintegrates%2Bmultiple%2Bcomponent%2Bmaps%2Band%2Bhigh-throughput%2BDiversity%2BArray%2BTechnology%2B%2528DArT%2529%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-13%26rft_id%253Dinfo%253Apmid%252F19171067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19171067&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.23" data-doi="10.1093/bioinformatics/btq559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mychaleckyj</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rich</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Robust relationship inference in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2867</span>–<span class="cit-lpage">2873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq559%26rft_id%253Dinfo%253Apmid%252F20926424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjIvMjg2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMS8xMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.24" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMS8xMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.25" data-doi="10.1007/s10681-005-9042-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okogbenin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fregene</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">An SSR-based molecular genetic map of cassava.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D147%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-9042-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-9042-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000238450600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.26" data-doi="10.1007/s12042-011-9088-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marri</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desany</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowicz</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kodira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The cassava genome: Current progress, future directions.</span> <abbr class="cit-jnl-abbrev">Trop Plant Biol</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DTrop%2BPlant%2BBiol%26rft.aulast%253DProchnik%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D94%26rft.atitle%253DThe%2BCassava%2BGenome%253A%2BCurrent%2BProgress%252C%2BFuture%2BDirections.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12042-011-9088-z%26rft_id%253Dinfo%253Apmid%252F22523606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12042-011-9088-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22523606&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000208980100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.27" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.28" data-doi="10.1038/nmeth.1270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozarewa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scally</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A large genome center’s improvements to the Illumina sequencing system.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DQuail%26rft.auinit1%253DM.%2BA.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253D1005%26rft.epage%253D1010%26rft.atitle%253DA%2Blarge%2Bgenome%2Bcenter%2527s%2Bimprovements%2Bto%2Bthe%2BIllumina%2Bsequencing%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1270%26rft_id%253Dinfo%253Apmid%252F19034268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19034268&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261212700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-5.1.133.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.30" data-doi="10.2135/cropsci2013.07.0482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gedil</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Genetic Mapping Using Genotyping-by-Sequencing in the Clonally Propagated Cassava.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1384%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.07.0482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.07.0482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.31" data-doi="10.1016/j.virusres.2013.12.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gedil</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikpan</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">High-resolution mapping of resistance to cassava mosaic geminiviruses in cassava using genotyping-by-sequencing and its implications for breeding.</span> <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D186%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2013.12.028%26rft_id%253Dinfo%253Apmid%252F24389096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2013.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24389096&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.32" data-doi="10.1007/s00122-012-1836-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulembeka</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masumba</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marri</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An EST-derived SNP and SSR genetic linkage map of cassava (<em>Manihot esculenta</em> Crantz).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRabbi%26rft.auinit1%253DI.%2BY.%26rft.volume%253D125%26rft.issue%253D2%26rft.spage%253D329%26rft.epage%253D342%26rft.atitle%253DAn%2BEST-derived%2BSNP%2Band%2BSSR%2Bgenetic%2Blinkage%2Bmap%2Bof%2Bcassava%2B%2528Manihot%2Besculenta%2BCrantz%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1836-4%26rft_id%253Dinfo%253Apmid%252F22419105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1836-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22419105&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.33" data-doi="10.1007/s00122-010-1520-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sraphet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boonchanawiwat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thanyasiriwat</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boonseng</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SSR and EST-SSR-based genetic linkage map of cassava (<em>Manihot esculenta</em> Crantz).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">1161</span>–<span class="cit-lpage">1170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSraphet%26rft.auinit1%253DS.%26rft.volume%253D122%26rft.issue%253D6%26rft.spage%253D1161%26rft.epage%253D1170%26rft.atitle%253DSSR%2Band%2BEST-SSR-based%2Bgenetic%2Blinkage%2Bmap%2Bof%2Bcassava%2B%2528Manihot%2Besculenta%2BCrantz%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1520-5%26rft_id%253Dinfo%253Apmid%252F21222095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1520-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21222095&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.133.34" data-doi="10.1017/S0016672311000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVAN%2BOoijen%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DMultipoint%2Bmaximum%2Blikelihood%2Bmapping%2Bin%2Ba%2Bfull-sib%2Bfamily%2Bof%2Ban%2Boutbreeding%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672311000279%26rft_id%253Dinfo%253Apmid%252F21878144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672311000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878144&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/1/133.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/1/123" title="Identical Substitutions in Magnesium Chelatase Paralogs Result in Chlorophyll-Deficient Soybean Mutants" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/1/145" title="Parallel Profiling of Fission Yeast Deletion Mutants for Proliferation and for Lifespan During Long-Term Quiescence" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/1"><span>Volume 5 Issue 1, January 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/1.cover-source.jpg?itok=gg-yH8oX" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/133" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AxVdzmGz87zYWISm0kfXLUAHStaru_TFALVR7ae_kms" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/1/133" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/1/133" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/1/133" class="active" data-icon-position="" data-hide-link-title="0">High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (Manihot esculenta Crantz) from 10 Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459764" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-uxyoU6B7LmjInqnExG-fx2rApnBkKMF1XCsECeiiTqw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/1/133.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/1/133" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-F2Scib0K8zA20bz3027SQgcpxMDKXppqoEUTodIWGLY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/1/133" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/1/133.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459764" id="node459764--2" data-pisa="ggg;5/1/133" data-pisa-master="ggg;g3.114.015008" data-apath="/ggg/5/1/133.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (<em>Manihot esculenta</em> Crantz) from 10 Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">International Cassava Genetic Map Consortium (ICGMC)</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">133-144;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459764--2" style="display: none;"></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/1/133" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/1/133&amp;title=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/1/133&amp;title=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/1/133&amp;title=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/1/133&amp;text=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/1/133&amp;title=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/1/133&amp;t=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/1/133&amp;title=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/1/133&amp;title=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459764/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459764" id="node459764--3" data-pisa="ggg;5/1/133" data-pisa-master="ggg;g3.114.015008" data-apath="/ggg/5/1/133.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (<em>Manihot esculenta</em> Crantz) from 10 Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">International Cassava Genetic Map Consortium (ICGMC)</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">133-144;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459764--3" style="display: none;"></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=133&amp;rft.epage=144&amp;rft.atitle=High-Resolution%20Linkage%20Map%20and%20Chromosome-Scale%20Genome%20Assembly%20for%20Cassava%20%28Manihot%20esculenta%20Crantz%29%20from%2010%20Populations&amp;rft.volume=5&amp;rft.issue=1&amp;rft.date=2015-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=%2C+"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459764/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459764/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459764/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459764/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459764/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459764/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459764/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459764/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459764/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459764/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459764/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459764/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/1/133" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Additional Information</a></li><li class="odd"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Members of the International Cassava Genetic Map Consortium</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/1/133.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/1/133.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/1/133.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/1/133.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
