<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/1/9" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/1/9.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/1/9.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/1/9.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/1/9.atom" />
<meta name="HW.pisa" content="ggg;5/1/9" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="In Silico Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of Chlamydia trachomatis Strains" />
<meta name="DC.Identifier" content="10.1534/g3.114.015354" />
<meta name="DC.Date" content="2015-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Ferreira et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Microbes possess a multiplicity of virulence factors that confer them the ability to specifically infect distinct biological niches. Contrary to what is known for other bacteria, for the obligate intracellular human pathogen Chlamydia trachomatis , the knowledge of the molecular basis underlying serovars’ tissue specificity is scarce. We examined all ~900 genes to evaluate the association between individual phylogenies and cell-appetence or ecological success of C. trachomatis strains. Only ~1% of the genes presented a tree topology showing the segregation of all three disease groups (ocular, urogenital, and lymphatic) into three well-supported clades. Approximately 28% of the genes, which include the majority of the genes encoding putative type III secretion system effectors and Inc proteins, present a phylogenetic tree where only lymphogranuloma venereum strains form a clade. Similarly, an exclusive phylogenetic segregation of the most prevalent genital serovars was observed for 61 proteins. Curiously, these serovars are phylogenetically cosegregated with the lymphogranuloma venereum serovars for ~20% of the genes. Some clade-specific pseudogenes were identified (novel findings include the conserved hypothetical protein CT037 and the predicted α-hemolysin CT473), suggesting their putative expendability for the infection of particular niches. Approximately 3.5% of the genes revealed a significant overrepresentation of nonsynonymous mutations, and the majority encode proteins that directly interact with the host. Overall, this in silico scrutiny of genes whose phylogeny is congruent with clinical prevalence or tissue specificity of C. trachomatis strains may constitute an important database of putative targets for future functional studies to evaluate their biological role in chlamydial infections." />
<meta name="DC.Contributor" content="Rita Ferreira" />
<meta name="DC.Contributor" content="Minia Antelo" />
<meta name="DC.Contributor" content="Alexandra Nunes" />
<meta name="DC.Contributor" content="Vítor Borges" />
<meta name="DC.Contributor" content="Vera Damião" />
<meta name="DC.Contributor" content="Maria José Borrego" />
<meta name="DC.Contributor" content="João Paulo Gomes" />
<meta name="article:published_time" content="2015-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="In Silico Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of Chlamydia trachomatis Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Microbes possess a multiplicity of virulence factors that confer them the ability to specifically infect distinct biological niches. Contrary to what is known for other bacteria, for the obligate intracellular human pathogen &lt;i&gt;Chlamydia trachomatis&lt;/i&gt;, the knowledge of the molecular basis underlying serovars’ tissue specificity is scarce. We examined all ~900 genes to evaluate the association between individual phylogenies and cell-appetence or ecological success of &lt;i&gt;C. trachomatis&lt;/i&gt; strains. Only ~1% of the genes presented a tree topology showing the segregation of all three disease groups (ocular, urogenital, and lymphatic) into three well-supported clades. Approximately 28% of the genes, which include the majority of the genes encoding putative type III secretion system effectors and Inc proteins, present a phylogenetic tree where only lymphogranuloma venereum strains form a clade. Similarly, an exclusive phylogenetic segregation of the most prevalent genital serovars was observed for 61 proteins. Curiously, these serovars are phylogenetically cosegregated with the lymphogranuloma venereum serovars for ~20% of the genes. Some clade-specific pseudogenes were identified (novel findings include the conserved hypothetical protein CT037 and the predicted α-hemolysin CT473), suggesting their putative expendability for the infection of particular niches. Approximately 3.5% of the genes revealed a significant overrepresentation of nonsynonymous mutations, and the majority encode proteins that directly interact with the host. Overall, this &lt;i&gt;in silico&lt;/i&gt; scrutiny of genes whose phylogeny is congruent with clinical prevalence or tissue specificity of &lt;i&gt;C. trachomatis&lt;/i&gt; strains may constitute an important database of putative targets for future functional studies to evaluate their biological role in chlamydial infections.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/01/01" />
<meta name="citation_mjid" content="ggg;5/1/9" />
<meta name="citation_id" content="5/1/9" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/1/9" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/1/9.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/1/9.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/1/9.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015354" />
<meta name="citation_pmid" content="25378473" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="9" />
<meta name="citation_lastpage" content="19" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Rita Ferreira" />
<meta name="citation_author_institution" content="Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Minia Antelo" />
<meta name="citation_author_institution" content="Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Alexandra Nunes" />
<meta name="citation_author_institution" content="Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author_institution" content="Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Vítor Borges" />
<meta name="citation_author_institution" content="Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author_institution" content="Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Vera Damião" />
<meta name="citation_author_institution" content="Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="Maria José Borrego" />
<meta name="citation_author_institution" content="Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author" content="João Paulo Gomes" />
<meta name="citation_author_institution" content="Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author_institution" content="Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal" />
<meta name="citation_author_email" content="j.paulo.gomes@insa.min-saude.pt" />
<meta name="citation_reference" content="citation_journal_title=Clinical microbiology and infection :  the official publication of the European Society of Clinical Microbiology and Infectious Diseases;citation_journal_abbrev=Clin Microbiol Infect;citation_author=F. Allerberger;citation_author=M. Wagner;citation_title=Listeriosis: a resurgent foodborne infection.;citation_pages=16-23;citation_volume=16;citation_year=2010;citation_issue=1;citation_pmid=20002687;citation_doi=10.1111/j.1469-0691.2009.03109.x" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=F. Almeida;citation_author=V. Borges;citation_author=R. Ferreira;citation_author=M. J. Borrego;citation_author=J. P. Gomes;citation_title=Polymorphisms in Inc proteins and differential expression of inc genes among Chlamydia trachomatis strains correlate with invasiveness and tropism of lymphogranuloma venereum isolates.;citation_volume=194;citation_year=2012;citation_pmid=23042990;citation_doi=10.1128/JB.01428-12" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=V. Borges;citation_author=A. Nunes;citation_author=R. Ferreira;citation_author=M. J. Borrego;citation_author=J. P. Gomes;citation_title=Directional evolution of Chlamydia trachomatis towards niche-specific adaptation.;citation_volume=194;citation_year=2012;citation_pmid=22961851;citation_doi=10.1128/JB.01291-12" />
<meta name="citation_reference" content="citation_journal_title=Br. Med. Bull.;citation_author=M. J. Burton;citation_title=Trachoma: an overview.;citation_volume=84;citation_year=2007;citation_pmid=18175788;citation_doi=10.1093/bmb/ldm034" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Investigation;citation_journal_abbrev=JCI;citation_author=H. D. Caldwell;citation_author=H. Wood;citation_author=D. Crane;citation_author=R. Bailey;citation_author=R. B. Jones;citation_title=Polymorphisms in Chlamydia trachomatis tryptophan synthase genes differentiate between genital and ocular isolates;citation_pages=1757-1769;citation_volume=111;citation_year=2003;citation_issue=11;citation_pmid=12782678;citation_doi=10.1172/JCI200317993" />
<meta name="citation_reference" content="citation_journal_title=Cellular microbiology;citation_journal_abbrev=Cell Microbiol;citation_author=R. Carabeo;citation_title=Bacterial subversion of host actin dynamics at the plasma membrane.;citation_pages=1460-1469;citation_volume=13;citation_year=2011;citation_issue=10;citation_pmid=21790944;citation_doi=10.1111/j.1462-5822.2011.01651.x" />
<meta name="citation_reference" content="citation_journal_title=J. Water Health;citation_author=A. Casadevall;citation_author=L. A. Pirofski;citation_title=Virulence factos and their mechanism of action: the view from a damage-response framework.;citation_volume=7;citation_year=2009;citation_pmid=19717929;citation_doi=10.2166/wh.2009.036" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.;citation_author=L. Christerson;citation_author=R. J. Bom;citation_author=S. M. Bruisten;citation_author=R. Yass;citation_author=J. Hardick;citation_title=Chlamydia trachomatis strains show specific clustering for men who have sex with men compared to heterosexual populations in Sweden, the Netherlands, and the United States.;citation_volume=50;citation_year=2012;citation_pmid=22915612;citation_doi=10.1128/JCM.01713-12" />
<meta name="citation_reference" content="citation_journal_title=Annals of the New York Academy of Sciences;citation_journal_abbrev=Ann. N. Y. Acad. Sci.;citation_author=I. N. Clarke;citation_title=Evolution of Chlamydia trachomatis.;citation_pages=E11-E18;citation_volume=1230;citation_year=2011;citation_pmid=22239534;citation_doi=10.1111/j.1749-6632.2011.06194.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. R. Clifton;citation_author=K. A. Fields;citation_author=S. S. Grieshaber;citation_author=C. A. Dooley;citation_author=E. R. Fischer;citation_title=A chlamydial type III translocated protein is tyrosine-phosphorylated at the site of entry and associated with recruitment of actin.;citation_volume=101;citation_year=2004;citation_pmid=15199184;citation_doi=10.1073/pnas.0402829101" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Cossart;citation_title=Illuminating the landscape of host-pathogen interactions with the bacterium Listeria monocytogenes.;citation_volume=108;citation_year=2011;citation_pmid=22114192;citation_doi=10.1073/pnas.1112371108" />
<meta name="citation_reference" content="citation_journal_title=BMC Microbiol.;citation_author=M. da Cunha;citation_author=C. Milho;citation_author=F. Almeida;citation_author=S. V. Pais;citation_author=V. Borges;citation_title=Identification of type III secretion substracts of Chlamydia trachomatis using Yersinia enterocolitica as a heterologous system.;citation_pages=40;citation_volume=14;citation_year=2014;citation_pmid=24533538;citation_doi=10.1186/1471-2180-14-40" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. E. Darling;citation_author=B. Mau;citation_author=N. T. Perna;citation_title=progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement.;citation_pages=e11147-e11147;citation_volume=5;citation_year=2010;citation_issue=6;citation_pmid=20593022;citation_doi=10.1371/journal.pone.0011147" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Dehoux;citation_author=R. Flores;citation_author=C. Dauga;citation_author=G. Zhong;citation_author=A. Subtil;citation_title=Multi-genome identification and characterization of chlamydiae-specific type III secretion substrates: the Inc proteins.;citation_pages=109-109;citation_volume=12;citation_year=2011;citation_pmid=21324157;citation_doi=10.1186/1471-2164-12-109" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=I. Derré;citation_author=R. Swiss;citation_author=H. Agaisse;citation_title=The lipid transfer protein CERT interacts with the Chlamydia inclusion protein IncD and participates to ER-Chlamydia inclusion membrane contact sites.;citation_pages=e1002092-e1002092;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21731489;citation_doi=10.1371/journal.ppat.1002092" />
<meta name="citation_reference" content="citation_journal_title=J. Infect. Dis.;citation_author=R. K. Ernst;citation_author=T. Guina;citation_author=S. I. Miller;citation_title=How intracellular bacteria survive: surface modifications that promote resistance to host innate immune responses.;citation_volume=179;citation_year=1999;citation_pmid=10081503;citation_doi=10.1086/513850" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. Felsenstein;citation_title=Confidence limits on phylogenies; an approach using the bootstrap;citation_pages=783-791;citation_volume=39;citation_year=1985;citation_issue=4;citation_pmid=7288891;citation_doi=10.2307/2408678;citation_georef=1996015960" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=R. Ferreira;citation_author=V. Borges;citation_author=A. Nunes;citation_author=P. J. Nogueira;citation_author=M. J. Borrego;citation_title=Impact of Loci Nature on Estimating Recombination and Mutation Rates in Chlamydia trachomatis;citation_pages=761-768;citation_volume=2;citation_year=2012;citation_issue=7;citation_pmid=22870399;citation_doi=10.1534/g3.112.002923" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=N. Galtier;citation_author=V. Daubin;citation_title=Dealing with incongruence in phylogenomic analyses;citation_pages=4023-4029;citation_volume=363;citation_year=2008;citation_issue=1512;citation_pmid=18852109;citation_doi=10.1098/rstb.2008.0144" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=F. Garcia-del Portillo;citation_author=B. B. Finlay;citation_title=The varied lifestyles of intracellular pathogens within eukaryotic vacuolar compartments.;citation_pages=373-380;citation_volume=3;citation_year=1995;citation_issue=10;citation_pmid=8564355;citation_doi=10.1016/S0966-842X(00)88982-9" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=T. N. Giles;citation_author=D. J. Fisher;citation_author=D. E. Graham;citation_title=Independent inactivation of arginine decarboxylase genes by nonsense and missense mutations led to pseudogene formation in Chlamydia trachomatis serovar L2 and D strains.;citation_pages=166-166;citation_volume=9;citation_year=2009;citation_pmid=19607664;citation_doi=10.1186/1471-2148-9-166" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. P. Gomes;citation_author=A. Nunes;citation_author=W. J. Bruno;citation_author=M. J. Borrego;citation_author=C. Florindo;citation_title=Polymorphisms in the Nine Polymorphic Membrane Proteins of Chlamydia trachomatis across All Serovars: Evidence for Serovar Da Recombination and Correlation with Tissue Tropism;citation_pages=275-286;citation_volume=188;citation_year=2006;citation_issue=1;citation_pmid=16352844;citation_doi=10.1128/JB.188.1.275-286.2006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. P. Gomes;citation_author=W. J. Bruno;citation_author=A. Nunes;citation_author=N. Santos;citation_author=C. Florindo;citation_title=Evolution of Chlamydia trachomatis diversity occurs by widespread interstrain recombination involving hotspots.;citation_volume=17;citation_year=2007;citation_pmid=17090662;citation_doi=10.1101/gr.5674706" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. R. Harris;citation_author=I. N. Clarke;citation_author=H. M. Seth-Smith;citation_author=A. W. Solomon;citation_author=L. T. Cutcliffe;citation_title=Whole-genome analysis of diverse Chlamydia trachomatis strains identifies phylogenetic relationships masked by current clinical typing.;citation_pages=413-419;citation_volume=44;citation_year=2012;citation_issue=4;citation_pmid=22406642;citation_doi=10.1038/ng.2214" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=D. L. Hava;citation_author=A. Camilli;citation_title=Large-scale identification of serotype 4 Streptococcus pneumoniae virulence factors.;citation_pages=1389-1406;citation_volume=45;citation_year=2002;citation_issue=5;citation_pmid=12207705;citation_doi=10.1046/j.1365-2958.2002.03106.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Horn;citation_author=A. Collingro;citation_author=S. Schmitz-Esser;citation_author=C. L. Beier;citation_author=U. Purkhold;citation_title=Illuminating the evolutionary history of chlamydiae.;citation_volume=304;citation_year=2004;citation_pmid=15073324;citation_doi=10.1126/science.1096330" />
<meta name="citation_reference" content="citation_journal_title=Infect. Immun.;citation_author=B. M. Jeffrey;citation_author=R. J. Suchland;citation_author=K. L. Quinn;citation_author=J. R. Davidson;citation_author=W. E. Stamm;citation_title=Genome sequencing of recent clinical Chlamydia trachomatis strains identifies genes associated with tissue tropism and regions of apparent recombination.;citation_volume=78;citation_year=2010;citation_pmid=20308297;citation_doi=10.1128/IAI.01324-09" />
<meta name="citation_reference" content="citation_journal_abbrev=Biol Direct;citation_author=S. J. Joseph;citation_author=X. Didelot;citation_author=K. Gandhi;citation_author=D. Dean;citation_author=T. D. Read;citation_title=Interplay of recombination and selection in the genomes of Chlamydia trachomatis.;citation_pages=28-28;citation_volume=6;citation_year=2011;citation_pmid=21615910;citation_doi=10.1186/1745-6150-6-28" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. J. Joseph;citation_author=X. Didelot;citation_author=J. Rothschild;citation_author=H. J. de Vries;citation_author=S. A. Morre;citation_title=Population genomics of Chlamydia trachomatis: insights on drift, selection, recombination, and population structure.;citation_volume=29;citation_year=2012;citation_pmid=22891032;citation_doi=10.1093/molbev/mss198" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Kari;citation_author=M. M. Goheen;citation_author=L. B. Randall;citation_author=L. D. Taylor;citation_author=J. H. Carlson;citation_title=Generation of targeted Chlamydia trachomatis null mutants.;citation_volume=108;citation_year=2011;citation_pmid=21482792;citation_doi=10.1073/pnas.1102229108" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=M. Kimura;citation_title=A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences.;citation_pages=111-120;citation_volume=16;citation_year=1980;citation_issue=2;citation_pmid=7463489;citation_doi=10.1007/BF01731581" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=Y. Kumar;citation_author=J. Cocchiaro;citation_author=R. H. Valdivia;citation_title=The obligate intracellular pathogen Chlamydia trachomatis targets host lipid droplets.;citation_pages=1646-1651;citation_volume=16;citation_year=2006;citation_issue=16;citation_pmid=16920627;citation_doi=10.1016/j.cub.2006.06.060" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=K. S. Lole;citation_author=R. C. Bollinger;citation_author=R. S. Paranjape;citation_author=D. Gadkari;citation_author=S. S. Kulkarni;citation_title=Full-Length Human Immunodeficiency Virus Type 1 Genomes from Subtype C-Infected Seroconverters in India, with Evidence of Intersubtype Recombination;citation_pages=152-160;citation_volume=73;citation_year=1999;citation_issue=1;citation_pmid=9847317" />
<meta name="citation_reference" content="citation_journal_abbrev=Comp Funct Genomics;citation_author=E. I. Lutter;citation_author=C. Martens;citation_author=T. Hackstadt;citation_title=Evolution and conservation of predicted inclusion membrane proteins in chlamydiae.;citation_pages=362104-362104;citation_volume=2012;citation_year=2012;citation_pmid=22454599;citation_doi=10.1155/2012/362104" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=A. Mehlitz;citation_author=S. Banhart;citation_author=A. P. Maurer;citation_author=A. Kaushansky;citation_author=A. G. Gordus;citation_title=Tarp regulates early Chlamydia-induced host cell survival through interactions with the human adaptor protein SHC1.;citation_volume=190;citation_year=2010;citation_pmid=20624904;citation_doi=10.1083/jcb.200909095" />
<meta name="citation_reference" content="citation_journal_title=Molecular Pharmacology;citation_journal_abbrev=Mol. Pharmacol.;citation_author=M. K. Mishra;citation_author=H. C. Gerard;citation_author=J. A. Whittum-Hudson;citation_author=A. P. Hudson;citation_author=R. M. Kannan;citation_title=Dendrimer-enabled modulation of gene expression in Chlamydia trachomatis.;citation_pages=413-421;citation_volume=9;citation_year=2012;citation_issue=3;citation_pmid=22263556;citation_doi=10.1021/mp200512f" />
<meta name="citation_reference" content="citation_journal_title=Infect. Immun.;citation_author=S. Muschiol;citation_author=G. Boncompain;citation_author=F. Vromman;citation_author=P. Dehoux;citation_author=S. Normark;citation_title=Identification of a family of effectors secreted by the type III secretion system that are conserved in pathogenic Chlamydiae.;citation_volume=79;citation_year=2011;citation_pmid=21078856;citation_doi=10.1128/IAI.00825-10" />
<meta name="citation_reference" content="citation_title=Molecular Evolution and Phylogenetics;citation_author=M. Nei;citation_author=S. Kumar;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=D. E. Nelson;citation_author=D. D. Crane;citation_author=L. D. Taylor;citation_author=D. W. Dorward;citation_author=M. M. Goheen;citation_title=Inhibition of Chlamydiae by Primary Alcohols Correlates with the Strain-Specific Complement of Plasticity Zone Phospholipase D Genes;citation_pages=73-80;citation_volume=74;citation_year=2006;citation_issue=1;citation_pmid=16368959;citation_doi=10.1128/IAI.74.1.73-80.2006" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. D. Nguyen;citation_author=R. H. Valdivia;citation_title=Virulence determinants in the obligate intracellular pathogen Chlamydia trachomatis revealed by forward genetic approaches.;citation_volume=109;citation_year=2012;citation_pmid=22232666;citation_doi=10.1073/pnas.1117884109" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Nunes;citation_author=P. J. Nogueira;citation_author=M. J. Borrego;citation_author=J. P. Gomes;citation_title=Chlamydia trachomatis diversity viewed as a tissue-specific coevolutionary arms race.;citation_pages=R153-R153;citation_volume=9;citation_year=2008;citation_issue=10;citation_pmid=18947394;citation_doi=10.1186/gb-2008-9-10-r153" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Nunes;citation_author=P. J. Nogueira;citation_author=M. J. Borrego;citation_author=J. P. Gomes;citation_title=Adaptive evolution of the Chlamydia trachomatis dominant antigen reveals distinct evolutionary scenarios for B- and T-cell epitopes: worldwide survey.;citation_pages=e13171;citation_volume=5;citation_year=2010;citation_pmid=20957150;citation_doi=10.1371/journal.pone.0013171" />
<meta name="citation_reference" content="citation_journal_title=Infect. Genet. Evol.;citation_author=A. Nunes;citation_author=M. J. Borrego;citation_author=J. P. Gomes;citation_title=Genomic features beyond Chlamydia trachomatis phenotypes: what do we think we know?;citation_pages=392-400;citation_volume=16;citation_year=2013;citation_pmid=23523596;citation_doi=10.1016/j.meegid.2013.03.018" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=J. F. Peipert;citation_title=Genital Chlamydial Infections;citation_pages=2424-2430;citation_volume=349;citation_year=2003;citation_issue=25;citation_pmid=14681509;citation_doi=10.1056/NEJMcp030542" />
<meta name="citation_reference" content="citation_journal_title=Histology and histopathology;citation_journal_abbrev=Histol Histopathol;citation_author=J. Pizarro-Cerdá;citation_author=E. Moreno;citation_author=M. Desjardins;citation_author=J. P. Gorvel;citation_title=When intracellular pathogens invade the frontiers of cell biology and immunology.;citation_pages=1027-1038;citation_volume=12;citation_year=1997;citation_issue=4;citation_pmid=9302565" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Medicine;citation_journal_abbrev=Cold Spring Harb Perspect Med;citation_author=J. Pizarro-Cerdá;citation_author=A. Kühbacher;citation_author=P. Cossart;citation_title=Entry of Listeria monocytogenes in Mammalian Epithelial Cells: An Updated View;citation_pages=a010009-a010009;citation_volume=2;citation_year=2012;citation_issue=11;citation_pmid=23125201;citation_doi=10.1101/cshperspect.a010009" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=T. D. Read;citation_author=S. J. Joseph;citation_author=X. Didelot;citation_author=B. Liang;citation_author=L. Patel;citation_title=Comparative analysis of Chlamydia psittaci genomes reveals the recent emergence of a pathogenic lineage with a broad host range.;citation_pages=e00604-e00612;citation_volume=4;citation_year=2013;citation_doi=10.1128/mBio.00604-12" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. L. Robertson;citation_author=B. H. Hahn;citation_author=P. M. Sharp;citation_title=Recombination in AIDS viruses.;citation_pages=249-259;citation_volume=40;citation_year=1995;citation_issue=3;citation_pmid=7723052;citation_doi=10.1007/BF00163230" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=J. L. Rodgers;citation_author=W. A. Nicewander;citation_title=Thirteen ways to look at the correlation coefficient.;citation_pages=59-66;citation_volume=42;citation_year=1988;citation_doi=10.2307/2685263" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1099/ijs.0.01472-0" />
<meta name="citation_reference" content="citation_journal_title=AIDS research and human retroviruses;citation_journal_abbrev=AIDS Res Hum Retroviruses;citation_author=M. O. Salminen;citation_author=J. K. Carr;citation_author=D. S. Burke;citation_author=F. E. McCutchan;citation_title=Identification of breakpoints in intergenotypic recombinants of HIV type 1 by bootscanning.;citation_pages=1423-1425;citation_volume=11;citation_year=1995;citation_issue=11;citation_pmid=8573403;citation_doi=10.1089/aid.1995.11.1423" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=J. Schachter;citation_title=Chlamydial infections (first of three parts);citation_pages=428-435;citation_volume=298;citation_year=1978;citation_issue=8;citation_pmid=340951;citation_doi=10.1056/NEJM197802232980805" />
<meta name="citation_reference" content="citation_journal_title=Infect. Immun.;citation_author=L. Song;citation_author=J. H. Carlson;citation_author=W. M. Whitmire;citation_author=L. Kari;citation_author=K. Virtaneva;citation_title=Chlamydia trachomatis plasmid-encoded Pgp4 is a transcriptional regulator of virulence-associated genes.;citation_volume=81;citation_year=2013;citation_pmid=23319558;citation_doi=10.1128/IAI.01305-12" />
<meta name="citation_reference" content="citation_journal_title=FEMS Immunol. Med. Microbiol.;citation_author=R. S. Stephens;citation_author=G. Myers;citation_author=M. Eppinger;citation_author=P. M. Bavoil;citation_title=Divergence without difference. phylogenetics and taxonomy of Chlamydia resolved.;citation_pages=115-119;citation_volume=55;citation_year=2009;citation_pmid=19281563;citation_doi=10.1111/j.1574-695X.2008.00516.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: Molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=Infect. Immun.;citation_author=L. D. Taylor;citation_author=D. E. Nelson;citation_author=D. W. Dorward;citation_author=W. M. Whitmire;citation_author=H. D. Caldwell;citation_title=Biological characterization of Chlamydia trachomatis plasticity zone MACPF domain family protein CT153.;citation_volume=78;citation_year=2010;citation_pmid=20351143;citation_doi=10.1128/IAI.01455-09" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. R. Thomson;citation_author=M. T. Holden;citation_author=C. Carder;citation_author=N. Lennard;citation_author=S. J. Lockey;citation_title=Chlamydia trachomatis: genome sequence analysis of lymphogranuloma venereum isolates.;citation_volume=18;citation_year=2008;citation_pmid=18032721;citation_doi=10.1101/gr.7020108" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=Y. Wang;citation_author=S. Kahane;citation_author=L. T. Cutcliffe;citation_author=R. J. Skilton;citation_author=P. R. Lambden;citation_title=Development of a transformation system for Chlamydia trachomatis: restoration of glycogen biosynthesis by acquisition of a plasmid shuttle vector.;citation_pages=e1002258-e1002258;citation_volume=7;citation_year=2011;citation_issue=9;citation_pmid=21966270;citation_doi=10.1371/journal.ppat.1002258" />
<meta name="citation_reference" content="citation_journal_title=Postgraduate Medical Journal;citation_journal_abbrev=Postgrad. Med. J.;citation_author=J. W. Wilson;citation_author=M. J. Schurr;citation_author=C. L. LeBlanc;citation_author=R. Ramamurthy;citation_author=K. L. Buchanan;citation_title=Mechanisms of bacterial pathogenicity;citation_pages=216-224;citation_volume=78;citation_year=2002;citation_issue=918;citation_pmid=11930024;citation_doi=10.1136/pmj.78.918.216" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_journal_abbrev=Lancet;citation_author=H. R. Wright;citation_author=A. Turner;citation_author=H. R. Taylor;citation_title=Trachoma.;citation_pages=1945-1954;citation_volume=371;citation_year=2008;citation_issue=9628;citation_pmid=18539226;citation_doi=10.1016/S0140-6736(08)60836-3" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="In Silico Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of Chlamydia trachomatis Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
<meta name="twitter:description" content="Microbes possess a multiplicity of virulence factors that confer them the ability to specifically infect distinct biological niches. Contrary to what is known for other bacteria, for the obligate intracellular human pathogen Chlamydia trachomatis , the knowledge of the molecular basis underlying serovars’ tissue specificity is scarce. We examined all ~900 genes to evaluate the association between individual phylogenies and cell-appetence or ecological success of C. trachomatis strains. Only ~1% of the genes presented a tree topology showing the segregation of all three disease groups (ocular, urogenital, and lymphatic) into three well-supported clades. Approximately 28% of the genes, which include the majority of the genes encoding putative type III secretion system effectors and Inc proteins, present a phylogenetic tree where only lymphogranuloma venereum strains form a clade. Similarly, an exclusive phylogenetic segregation of the most prevalent genital serovars was observed for 61 proteins. Curiously, these serovars are phylogenetically cosegregated with the lymphogranuloma venereum serovars for ~20% of the genes. Some clade-specific pseudogenes were identified (novel findings include the conserved hypothetical protein CT037 and the predicted α-hemolysin CT473), suggesting their putative expendability for the infection of particular niches. Approximately 3.5% of the genes revealed a significant overrepresentation of nonsynonymous mutations, and the majority encode proteins that directly interact with the host. Overall, this in silico scrutiny of genes whose phylogeny is congruent with clinical prevalence or tissue specificity of C. trachomatis strains may constitute an important database of putative targets for future functional studies to evaluate their biological role in chlamydial infections." />
<meta name="og-title" property="og:title" content="In Silico Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of Chlamydia trachomatis Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/1/9" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Microbes possess a multiplicity of virulence factors that confer them the ability to specifically infect distinct biological niches. Contrary to what is known for other bacteria, for the obligate intracellular human pathogen Chlamydia trachomatis , the knowledge of the molecular basis underlying serovars’ tissue specificity is scarce. We examined all ~900 genes to evaluate the association between individual phylogenies and cell-appetence or ecological success of C. trachomatis strains. Only ~1% of the genes presented a tree topology showing the segregation of all three disease groups (ocular, urogenital, and lymphatic) into three well-supported clades. Approximately 28% of the genes, which include the majority of the genes encoding putative type III secretion system effectors and Inc proteins, present a phylogenetic tree where only lymphogranuloma venereum strains form a clade. Similarly, an exclusive phylogenetic segregation of the most prevalent genital serovars was observed for 61 proteins. Curiously, these serovars are phylogenetically cosegregated with the lymphogranuloma venereum serovars for ~20% of the genes. Some clade-specific pseudogenes were identified (novel findings include the conserved hypothetical protein CT037 and the predicted α-hemolysin CT473), suggesting their putative expendability for the infection of particular niches. Approximately 3.5% of the genes revealed a significant overrepresentation of nonsynonymous mutations, and the majority encode proteins that directly interact with the host. Overall, this in silico scrutiny of genes whose phylogeny is congruent with clinical prevalence or tissue specificity of C. trachomatis strains may constitute an important database of putative targets for future functional studies to evaluate their biological role in chlamydial infections." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
    <title>In Silico Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of Chlamydia trachomatis Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1dn5EcDAZnco-f0jZgJ_vJ_O2HWp6hmiQZL_rL8WUmI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459762","apath":"\/ggg\/5\/1\/9.atom","pisa":"ggg;5\/1\/9","ac":{"\/ggg\/5\/1\/9.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/1\/9.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/1\/9"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/1\/9"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/1\/9":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FQATbQ4RT2YV3Ja8ETVHFVIZxNwCdyQt89EwH1e+kVfyvEVRTytUj3zXN7j+NawoHKBVBGo2dvSH1J\\\\\\\/IWNc3SnXPhNOCmx0bYFntaDH4xvrsYE6UK\\\\\\\/Pd9H4XY4kwtmGiGZiMyAiQ+02aB87KjGDTNY\\\\\\\/1almOuIXNyZcph9Lrnk4x2i5YrUAUVD9idVevRhTaR+OWk1wB4GsreBQ8qU8FabtWvAaNiDvItZLaJEuSuFJ7uixBL\\\\\\\/vsw06Wzoch\\\\\\\/\\\\\\\/EdBdBQrBopVEHhLHb1Ev1YV3cCyhasC4zYeu2hTtOEx+V+QIvqSng1VFGdnmECLXGB5HPg8hHNqkbxj63sBWCtjodKS1d2lTlznszDD8HKnHCGPMpN0yx7e6dhhI93AWqJ8xDnjvRMYwUatKov2VgWRh2wPiCy8Gx4S+qqmlHPzWWwJrdURLE5qITblezWN7o6h5bePIA+\\\\\\\/T7Z3fTHDquvS2A9uVGBN35K60FGEtOBLHP6q4NxctPkzJ8rqCAYjKh+ShmuRDXyvkKJwuhhsy5sqDBBI6ZpIAagMpAtdfrVskRrMdtIyWnKfPNjdp\\\\\\\/s7XmfGhlRfSjlE9GCFJuShsf8bMaKulT4aVX\\\\\\\/zPb1FKz5FTCKtPjJUOXS0lLyuMP7mghOigB\\\\\\\/O43iU9AwnKMRnM2M1dW1X9SRpnRid8hxzvJ98ElDOfV2qJkUPsewXVcDs0UPYUFtkFwKmw\\\\\\\/nAuju\\\\\\\/tge0LdMF96TJ3BEqcpeyd4NQq0Oc+MdNZlUd6CyG\\\\\\\/SSAe5PO28VMwPSoPeLcAEIJuwyLhQcF8Yvtygid9HzTv5cijRnpYIGYawfN\\\\\\\/0suSqldOcpsz34Bon1zh72+ILQ44KnzpbLDZpTKTNIp3xjSNbZUcKlmW57QfO\\\\\\\/la9kWiXaqmqS7vFZlthDAeZ9ggbwItveuVbDbWJcEiSROdBl54VwaFmB6yYfwJUNYGCZCkftYC+7MJvOhfCrL3GS7UJj1T8HMNV6vQlBA88mjcuQZGfKsID\\\\\\\/2FTYtrKVswVs2ZUZNe\\\\\\\/Gy5WFJ3JmRybDNerxJjufCnjZYWeDZw1WQ3K7Iejo8S8xnwM7N+0m4ireySeR\\\\\\\/yMgpcbWRuQ8zpU1InUd7kkmns\\\\\\\/w11SBhHMSa1NPytFFLpOeqS1UfNp+t8gwW8HlVJLz0Zer5O3GE4tAKXVPoqZcwOIlKB9ctCf60ylAc7RDFfijXFbOC\\\\\\\/WZu5ckMP6gcBsAOsJTh0HskSbjadSbrzbRdiylCetlVvZVkqVNEwSPst1EAFqouoXpZ5tUUYsLJP5h\\\u0022,\\\u0022iv\\\u0022:\\\u00224IV9fYRdJ0YSO6ZC7Bafjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f48d172f616c25d684339e8178d6e951304251a17e05d862961debacb8dd91d0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jCU+NsXs8DCjem2I2Nv+gm2cCefdo76JiCkos+ovh\\\\\\\/ZtNDWeytO+6HEc4+hm83WpefPCZ6wKRoCFansvkkmU7zfDuVkQUYy0Kn9KsgAHUEXsbb3vznVjNDOEbzns4IJP8SnZbQ55LNTLkn2GchP0yAL\\\\\\\/OFk2Dmyt\\\\\\\/CcWWaxGoJXS1OuaXOSyh68zjFFPCoScS0J8MFrAiwaawN+eM9qU5L9FmPgWuKyRduajq\\\\\\\/wjI1FEzl2Vuwv09SvTlCxgbpxW+REravn8tFc0YDcpdMQQqw35tpxsWp8RpxHqxt+t+TM2yBAhTeHmJlfE8hX4+5lelul6T7fw8H3edVurY6S02v9WhokqA7nNwktSJ1DNMWTgbTI2m8rAp2olcbs5vNrWFDyPZlhu5OFlgmf3eUQKsRbk9UKHyEbOkLD8OkuyguYOb+s4smSkw3gMIchZmGRkYXsBVAVuFW6L81XtdgSyT6laeS4ioTmZWmICGSAWWUrcJ7e0i2KpoqUrO1QJFS5RVJR3+cHCeWitzJ2HqzBRds5JUDEZnhtCoku+u+NEdIEmpygt5dM1GOLImPfdx4jkqfneI1u5Uz6Ey\\\\\\\/zTfNBI4yG8I35qWwjD60ab46T47WuRI\\\\\\\/rjSvaTIASMwzLY\\\\\\\/QdfOrf4Mx+xlIIFl2D9dRiVI99SD43o4+dO7lp7LDRCbTAe9sbo306t0DmqTD4VnA1Ksbfax1sQVIBpFpaaou\\\\\\\/5DOewzovrgY4fbzncQcZX0jU6YSChbecD8pb6sPfBECyhbK7u80SPvEk7CYF6WDV7Bmap548Yf5aCz33CTrPPoQMG0q8hUZq2NX9qXY2W3NsjZzQ+tHxk0+ILP4tIhIM+ivzlTaoA8pKyzhh\\\\\\\/AiN2keeYIbNJeqFr0\\\\\\\/E4O2VR9XWfYL2KStbYedX1JhXEw36m+1nm5Y0Q3BPAvMiyV+TZy\\\\\\\/ZeKmhhev1NzLX0ypR+EW1g3vsjynzinBWt6Ven9\\\\\\\/4jLtP\\\\\\\/D1urSjB9WWyOXtOTkJ70fN+77s8LDEqw4m1hslG5lzfac63tmzUtnqFhng9rfBVmi+RluKgxKSf8g4QKktsfi5CZ41HUT58NQNB4cxqm6tptK1542UbsGPnrHtQJtKYC7Yh\\\\\\\/EBjqJGDM4UfTJCNOG4KWaeGSUen5\\\\\\\/9dxID38KUvzlRLY+JyIBKcGCYj8WV+yrlWgMa+GjG2K8YC5OKvWpHZ2\\\\\\\/iHTRceKJgQfJgH12aK\\\\\\\/mPqSoz5GTuAdzmHQUOiMN5qdqWiMpN0yMjLFugbjRMILtyRMUG4iAr74wMorWh4NQ1mZ5t3s5tbUFKNo+P+u4zdZqYflY2vnt5MrqlvpBVfsOU5vAtB4gv4hDo33KuQ1ZkTaZCqIXZSlqR7\\\\\\\/rK3OP3butRQrC\\\\\\\/\\\\\\\/ahaiIMAsf3hdKio8M6LlbuF7QXnvEnyvS+55AAtEwLBYBqpKAadBFV67weQdP0r02XYWpu1w5peFNTU2+RQnU6mIF4dN2EUGdJqsnYAlzX39C237jiN\\\\\\\/v0nclk73dC4P6vJz8D3Pze6zcc\\\\\\\/UrfNbelesByCBhyKXl0D9YGZwIVbl2KPY7+IqHcXe2BA731k1yb8r6U5ZKbWOmNoquslDN7aTS5SCZ7UQc\\\\\\\/HLPlz7lMU+8Nvz5IFtKdKF4rh5I0jomNdScI0QV3wS1QUMoLFo1GIs6V7451IStF+Ze5CNaJ5n361AWpgOvR2XAw+qsh6iiva3nHyUv\\\\\\\/5\\\\\\\/qgOCyMj0vhz61wF247681frUcJp+Y3vTwIUVP+G65+b+TvWb+iPY6b+lxKrshllk1ZrA63W+8uihgEHejCk7rg+sfDQZy9Iy5SowH11YZfsOgLg6LBv3eTVoHO7F0hWY7dEWuwJVYKxY+0j7mVWjZ9Vwz7mZvOSv0eMnGxbhD4Npei0DDy9x7yAoYk9V1yEc2h91aQY6aCUX\\\\\\\/woBrUFfbGQuLWTOAehB9nq1TcKI15wpJBXDQ\\\\\\\/8e61L6l6mQZrQBc1Z3AyKWqlVvwvpgPP2dWNJ9q\\\\\\\/F1HfINLt\\\\\\\/lbKgJtbuAVm\\\\\\\/9m4N73NK1W9TnK\\\\\\\/j5IjesAKuANurTN36gJovfeZBUyrUOqoREfcwB\\\\\\\/bxS628EzhqawwuXh7q5\\\\\\\/eojdSNloD67oNJ5LUBcbOpj1IJIkZccO\\\\\\\/UYqPUKVM9YbQS9u3I0FP5J6v2jFRabcF1hNd6r75+CR2jT8p5BC6kEox0zQm0wXvioEs56Oj9S5FxYYSYRfpUHzO0HtwlG2BiCfi+jCOIvfKfupQTSfvJX+sJc0Dx+nTUL9zzuQJ+gbkx4M7MvZ2xhJdYMq3hKpBIzEw4yN3p82aTSiT9ngAc0\\\\\\\/0PzSvT1sXz+TEf6U++a+p3ewF2yjDQUCtscFeVew0LBSoq2PWmsGNVBkFDOuPn1CXlyx38D3Y9Xnakdx81wquv+XIWo7ClD1YoFySvNZMsCVoNAVlpV1Qt6eaVz+5BEqH1LgbcxSAjhwkZJ43plmhDEKVXw0fhp7vhc3vVp0+lqNda6v50yAr2sRntvarB0xFURYlfCUZO6YQUysfCTV0w5UypScnx\\\\\\\/f7dQVA\\\\\\\/GwilPcl4154+ijzAxA7wNGB+OzfNVMwXp8SrslmLh3EAPz0vBa8BFUaQh28T6SeGO1OjfFLl7B\\\\\\\/DVo8Pk\\\\\\\/QdyjaPnZn3lQ9uTEjrHdyvGu3O8rgFAL8+ap2t3p9OAIyRCmh04IK4wVwlnEtprG97MwZ2KnJOSMsqBrMJkHOBlpj70mlw7fCHnGTAjYS4rMRH1E69RD+gf5C9LufIG+v+pS89CnkXj4XnU5TE1T+6BB0RJMY26o7lKlx5PKNv\\\\\\\/ShHrsSzYe45WRwZ8sgRvTEJcjsKvz6G9nUjGyqnbwKihqeDMfA9n7icczIc06OwVyU6OiQP4snHknfw1TIra6fgLz8l43yVj2KFhHvYZJmGb0PRJXo6EPjF5\\\\\\\/UDA1za9Gwc\\\\\\\/cOwe62G2aQU7kX5ZHfuXPKod+h7i6JrfjCWjrFQwfNJyLI3amvpB6sJYFGT+Q7OLxJVQnj0v08YXOYyW0D9zUc+hMkEyXcKv4HLhgAFis+fxsqT7CU+jKycuWtJG388fBG2KqT+K1fHAYVsuFfdNgRu3zFqhUSGrdpsyQfYFxlaBjjeJEV8bEYesWIuYkmS\\\\\\\/4iqSQzD8bM0Dq995SJyZLbfpZ726LP\\\\\\\/C9MAysUPHPCvB7iNyTybWcSo5qghxTfu5oOWwx0fwOODYSmnq7Wb+z2Sx6b6YZlVnUEtI6O1ZHwtsM6L08ZAaSgo+ZjkNIJfP+UO4ZD0LzjWUgdXkrfbe5EJv8Eoeg+wXs9QKxU5v9ZAGsNxZjBSXxA5BmAF1EzuSpoC0tltz9OHjnlZyKdY7homjX5bwwTCOhUvufi9BwPG5DOfcsSJ172anV9b4flxJjZjF2TFtWyakLY9pcX1UNGCHPrBL0uUOPLH+14uR75NRzUlaX3e99CU4J8Uf5NkXVpM13hKi\\\\\\\/DLhJ\\\\\\\/DEFani7LLA9Oa6f7\\\\\\\/0OD7hDWJv2nMLGuiFXEjd1E2h+9TovlxiIojYXkLWCjzdN78ioXoipvaZzZ3iJJdkiSeiiSuD78yjZ63SctwvTL+h6TLk6TJaCLucdBiRVfMCtf50LATpcDjbMsKnf++s9QKQt\\\\\\\/w06tjdssVJG\\\\\\\/oVQ\\\\\\\/+qgYY\\\\\\\/1F\\\\\\\/zIFW+PqGEOxwZ+dq68AZuj4NrM3unFIlTWm0z1HQZ5ZzCF\\\\\\\/v0NYi7JA6fa8vHnAewzZmiLP+GG50Y6ZLaF9dUY9TLM+n1MDcD9IZ+lNhpB13GhdcOIJ2R3\\\\\\\/MGWcuQiOr6ni8K0f2hkKPpfFDIALuhMIyqFRagrRlNcSUPuBHNT\\\\\\\/jW3PMhMJhdquHp0REfawo1hxa0GynGrQCOzBRy2xsKDK8OMgEOjchnNxkWz9PHMo2EtiFkYQ8cn\\\\\\\/b0r26\\\\\\\/wBPycfZmdy5i+oAhFxq3To0ZFtcic61aqBSYhqUyGjVCHa0UnbGiEU5TWvTxiZMtmDvzL6t139oEAaWD+C9YZzi\\\\\\\/4tja6lyrTxlrPj8TzJzr6AdNy2FswqfhQMzYU6iEsZ4XqKPM\\\\\\\/F3Ps9aKQohSUa0cJw9OXARsZsnuJ\\\\\\\/WMKQr5wW3HIjQK\\\\\\\/DEmedwbDitSi5QTUvVMDDEj94n2ba2d\\\\\\\/UG3fmrYsE+pLJ4OmfGpUuJ9M6UFY0qzFhCpr2lJHDpESdpWRwO7j+WUeu1+9peZYDMoaXUuSyvoOc35uL25P5qDO7SP63EGNr1p3+Yt6vhpVmhS8V2eRt0L3jN9zYhbK2SL2uV37smD18jeljHF906SmkMsDA+s0NlDKoblY1\\\\\\\/5sDOdvo3nmOmHz9Ima\\\\\\\/ArXTrryJnBg7QXadfAXtACyQ\\\\\\\/X\\\\\\\/61q6jEeuxsU4E49F5ZFgDFFgR+6r1c\\\\\\\/OtQhZzarp1z9QjF+wpYq3fUxXc2cWdMawHfqdabYurXt2j8I+L6yuPtAN3XWvHDI3zpPaJyDEYw4dQCNlz9e\\\\\\\/UZCJDYr2JBYjZu54hPGmpxUMY3MUZ90LudZ46pDX3F9kboO3r7XO1ExXkN93s1Wix5SikvZgSsJEszH6AQ0ZfZGLrn6RxOlL+pWQ4C66EfZ4eNPKGlTeGxmOSPv\\\\\\\/I9GAdYRnx2csHC5pNljXSgN8R0evj\\\\\\\/ks\\\\\\\/ea0m7B1ke39oRdR2lCEntJ2dPb\\\\\\\/UR0qQFkuJdFH85YhNBLbPEo10qifWnC4R7QMsQS74EAnN\\\\\\\/bbkCzRrHC1vPyLZpLsyAe9CQl6w+30Dl6bGaRDEreszgr+lL0lzYcxV2oxg6MkiMHoE37pIian54yZWNYevBWwT9zH8hyJeYWlS2KEzzjjmpaHZW7t36HvdFYQjan8y74yTFYBjEeVoYzDNaG\\\\\\\/Fv87lyVY9Klbf5G40LuJonR5MT4JjqMrYh+qFYao+nyFVlnTZuy0jThpjUNpyyww88YeoADBzeRIVweSXzsT+lZx\\\\\\\/5yvzbGtYvVuSkmCI+yVUtAiwubrAOYozNbEnWet6oC4MGbwdc1eRL2nnhprlsNgnTVO95a5wR1foCyXBEMfrUirhNvyeB94HDaWYGZ8lyrIwludZ6DOXnGvgI5L0CvvulpQMiaJHn9AqmkjAaPYPcCj0VXscGf0iLpF3Vmx\\\\\\\/pGhQBNVE3Dub\\\\\\\/CBRt7DJ9vcih2iFv87Ww8dySnwTMEf7cxUfMm+p2vvp7smkjzKOJpVrmPl+24ua\\\\\\\/VgYZCVcybcTxyU98n1ie\\\\\\\/Ch\\\\\\\/z1FO+7Fmv3AS1WmZKnWJcdIpz0otP28q94wXFHojgEpA4psa9lGPjkgiizhVA3T6d6Il3lUbrqlk1cHgJK4xkiynesVxQSHE7Stp4D0Zvpi8giQUBD6Jr1P+Yw7i6BClIGfnZPEgKb6iOvMNYdGR1CBuVxoOrqVYh3ZgwruqhwYhaQTd7Xj7K7rIMNukHCSou1ULX96+3b1Av3C\\\\\\\/tQ\\\\\\\/QOPLltJOcMiUrmx6Jtaetky\\\\\\\/WpWP\\\\\\\/gYGf0b2bMfZqvLjLO8KGGUJOiL0\\\\\\\/dVuyIeWC\\\\\\\/FL8Ph30j3IXHKr\\\\\\\/F+irTAIoGdG+p\\\\\\\/x5aOPawIDTFdu3BFf9qR5Mq2bIHvdmoAAWYWUMjtZY2ILu0RK1pkoZHYqwTWGeXrKBD5YnvR8HTM2PC8t4SLre+b4JHB4N3KcSLjyOkk31Tne3IzwGX7hx1tqorSWCvL329vUDn4x5Ui2wO6a4ufSWgyFd3teNqUG1H\\\\\\\/GQKZ7isOIcqliP32btPkUpbuh4EVp41tAwQqYc4azMt3+RkdpX7szB4O6RVFmSKkyxkjIVKbOIgIpBjVOoq1voGI0DhBqfiULlbmI9Yx4BDC7KFyaV6sEd5Sbn3WOzG9o\\\\\\\/54\\\\\\\/ZLJPxILXFLFOn\\\\\\\/PD+0VuE+OMHOjdJrASOV6Hnf+sg2fXuix4W1bJlW2Y2jxxRPfqPrUOOweMC+WnIpFnC4sEq09uyfCFUG1hI\\\\\\\/QfSOFGesUoFGkqDq26Fmi5vo5TYri7cmKi9CO77RkiOaSLIX0O+2n+kfdb\\\\\\\/uvwUogbC\\\\\\\/rZLEpkEL77yXqMQmEzAwTAVJZXJHJBEQY66LTWet\\\\\\\/JyuRIrNoA04CIxmDCHXX2NTBA85o4S6+SqGk+73RnK7fQxVPuslWbK\\\\\\\/qHprk1biIsZAQRPdNxQ+S1UlUzFHQbiPY6fHtvblyJhzE1rgzDoMD62Alo\\\\\\\/wk0ZhcaDfFQiwCziHFrwnLSsnk6pg+ikjufsiGp3Cu4ga3enq15YlJMwQ5bixonxLGOcvBRLo4ZPzvmMuYDtk7vbH4GlpUjv2wKgFmBJWP2s0AOzoiJU5em2YveENVc4O4Jn8+BCkiABshn6IIGPWuurXM5C0pEOVPcsblOwW\\\\\\\/jszmJ1aWdW3MZFPz1NWSTXS1DTivLuI6hifp0qfexMbMVXyg2RS4DVfaFFsY2SHV6PnkJOhx+qBxKCxT+IS5zfQ47p\\\\\\\/qRKdy0pCXvrJuUilC0npa8UHx1bZwvYO7ekfI2T5SJroT1noRbEeC3poRUdF0j4JfFwGUKPUcpXVE4YjwwhAAUA3BZPT0Txc\\\\\\\/8wld+DwzW7aNoPAsIgdGTIOYyWGimw\\\\\\\/pa5utFmc2OG9TVqsfCkaLCA102dwm5uVEbfE4+DQjSNIYdMjFJnm7YNeFlLSBQS7vhW0UR20wW6OrL+qqpKXJ9RP1ei4pDapGLJ3mKDp1H2LDPHqHL4FziN8Rh\\\\\\\/GJq06AiHBSJdiwOqhxF10d5lC\\\\\\\/iz6xynN2p\\\\\\\/HWFaFP3yPpZJn0xzzLixysVDZPvRVd4lM91b47A7gmBo7ayJq9hUT5IMx3Wn6NXunmFHkfLUo2isSUW2u1KXpU7EfHFlFrUO\\\\\\\/TWewNdbyiUDR\\\\\\\/u+WH7gczUDgvI\\\\\\\/A6BxZFj9INaTD5nkqTSbbaB4HEsuXd1Nuq8VKM+pu+OGQ072nyOM83Ph3RIxSUKdc8\\\\\\\/dq2tFeH+XJQ2o3aGIl+fqyHAe7rD+z8to0tV39NVxxtkUTWMHfN\\\\\\\/0LigAWpYr8X3cgitLfe3yk6MBCh1gfyl+5z1goM3XN\\\\\\\/kT1hCryZt+UG6SZBlrlM8+71wujNbsvTk1Ar1txgQwIG0umA1ZclkTF1u6XXqIGg+jfS5dC9KL+kgsknNq43LO3Ie9+P2SzRr8odHpc2hmnZGcfM6FNEC3AXMf93svRgIMyxYWN6oGk0rJpFp5wilf9s99X\\\\\\\/KsgUZzkDRZxyV9V107masMbRedFiy0qQsn9I7I7SYPf51hgFi+8SDwt\\\\\\\/kQzqazIUrIuhK91Pur8YRwqRmT0fKL5BohYHqqHSUrQ1uKsGQvp6qwhXqcXFsw7+PU1QWQuojgLjovtlJmsx2O0mGMq1wHfJkv7tBmZyaUM0Ic5zQIJqm0zk8SdMUcA5Uu0scNJQdZLR7c4O4MD\\\\\\\/ChD4\\\\\\\/a7aFalBMpZ6b9l0PMaNZOwQ0bt1kLOj8yZwKoInCbbYbKNArp3FqUNOXn1266mxxiO8mLOIkZ8UwnhQPB7gou3JNg1d5ydi4O6zl9icQITXyJ4rjfVCl19BHS8BmTSlGIvydC0K0SJlDFEldxu8eLFZaosaNEY9JYrvjTCinelh37JVd5dHYv\\\\\\\/tt1TaxJJRk9PeaQlEWtjR3En8+bEvCHGhyegxkP3M0KT1J\\\\\\\/ps+4cuK0VfGAxwKq1HFSbVxIEuZDbbRwdgWggsqEFoG62m40SIMTwcQEYM4zR1K7tfQtYIVClzRfu\\\\\\\/tD9LQU1r1ttrPhNxicS2CUtof4Uj7Kmqv44pJ+\\\\\\\/a3mD0HhAOLe6DgdJi\\\\\\\/9XreNfRKmA+plPoUxP3Dh2GlCXmczSakidripQSL0DYxy0wlrL+yMPiEAlgaS9lU8HINbqHI1qU1I53H6eGBvqHFiyElC+3icVk7bHLv8gTDZHYr49DKG7xsopyDhZ+iyEjHaYmnmhpNF64AZpUapXXk4cu2IBizyxqw1xyNHHqwZwWwVv8hUH1QfAIOlKD4qFfwgDAqREhfsVRF59wzZJ6o9RcmaSnG2ud+2+0ZPVrcFgzufpUJwg521qQuCD\\\\\\\/XJ9+8QkuhjPdROrhXA4gnGUCWHngnogLkeht+g0qUIfw0TxZGi2w+uLVtGJqkLC1rzGPDcneoW8GoKEHJuVy+ZvGCLJwQWB8tT8P\\\\\\\/MHDLgkvcJCPalveFwLt57hAKvjZr6K23KjDe+ykkTSvEIZQEeEMWgNxZkvx19rsGGwRBrYgwV\\\\\\\/GrCtrzs2Z4S8yz8dhCARqQOwPisB2vu65D3rBFZ+7+\\\\\\\/X8RzDTPysO+HLUzygubecyNttqjybTCKVGeZEbbu3T\\\\\\\/VfCqVUhygwTo0exF3hcadTSOpunQL2t5ZWRMdNcuHEHQmDO1dYqJDTKxLtXV6jQC6G2RLql5foDjOejRLbzFPaF4\\\\\\\/XQVrvEkL5xa5FNDYPK++hYT8EAdb9cc7zC3MPB\\\\\\\/njV+flBHpXtPet48c9kKN8lcGPw9BJpOrW1aEFLC0B1G+gy54h\\\\\\\/l05I2SPOJg8VQVGl09JApNfkMyfWey1GjJHFwQzJWnvK2iLhXYxCh7IaNPWnJi\\\\\\\/hweulL9Nw6ecOR\\\\\\\/jimlopMPgYETYmNHVr3fEHZWnA71ev6sZ5kHME+cntXTMLsGdtaDpqYvT9TlH9Yye49ui6PJjgbd5MRDzFDZ6KnHD1GT1WECsbxCdBE86iOgPhxL6SskPq3Uj12OvnkRGSBnrhjRPxsC5Rw6dmE5lKbde1MZX2RmCdL8KLJ91XUBocOk7A0KODIIwHQltDzmR6VU+PHZMvwgUtq2pCXV9hhUNZAKtP8PxhNup4SoI0K7m5B+v0yE0HVNF2nQKelsnPXKmFo1fotpbavh+\\\\\\\/dvB70A6K9IS0wN9FVVRHn7mqdslSLrlg0Y7VI4rKabQaPWMC9aA6cUrXa\\\\\\\/svq3G66ZJ60E06wtYECVn4xySIBQ57RBiysLN9yEKQNzKApQvVegz633hqiDNJ30lmhRYALotQd8LMKAYYrXziLAyTVCRO9H\\\\\\\/pmzJEM8rXLkYiSZ2x9mfp9Q2hrqzzasHLnZbWuCsBx1GbfzGAhaTXm0Xf6ZPuixB\\\\\\\/SuYhabvTJGJzsfRTggUzkdq7URzyYEYQ4Wayj4MR6N7V2IKndxLgyLjSfUUKXHIPxQYIe\\\\\\\/V+p7g0dczXr8ujrGb3JTDTG9CrZStHDlb7LRoF\\\\\\\/KVDEHDoLIbtoETviN\\\\\\\/Ht6IedIqBgL9TeE4x4X9ImuiJxUUEG59lYLuzLpSy\\\\\\\/dx4DO0B6LYVcw89bbfZN9YiICNH\\\\\\\/2Vu2wkIIYwxCULWZvYGGNVMCYsaLKjxFTKtnHds92p48dLUu7Hm7RmUx97BASlLA7JrDjp9AwbGS\\\\\\\/P6LWovTAGtqRtunPHU6+Mpl+o8UMHn789CIByND+40Zg4KZKBcJsjSJ5z6yaETLvptCy+iC8X+Rfhm7tDfjLEpJaneao3vTOkI\\\\\\\/1lvTtJ1Xss30GH7GXccb8nIa7C0eCIOPuccdiOhd892llLgSZfGHkuLmaYKOj4GyOjz0i5GEoSwknvn\\\\\\\/X7+vmxVMaw3OkATIQKmeZ0UbvW1bvMQ8bGAQChHwF76gn\\\\\\\/uMhWzD5RzHC25j9atAcyp6l1z3\\\\\\\/AL2WUBbN3WSfzajVm6eFe6CuSfaNLfMTleFlu5UvANBxEpKPNZY3aFVbbI3cxEnYV7MWkx7wYPL3KMf4yeSJHlq4IipwkK4kMtupznaltEeiJs0tOgFAYWK8b+mdv6EzIsF4j09gwG0KFIq\\\\\\\/+rq\\\\\\\/1avXUtekARnWA6Fp0zk7Kw+quTiM1D1KXKhPsRSpsfRPK5UZ6ZcJ0e+E2cW5pyefFk0uib15M+IwApRH1nf70sXV4xqJqjUhhOTS7J1CG5NDLmndcJiu9za\\\\\\\/4Oew5C262b++6AcF6hcIqrmSUbiYVOP0i12yueXmUTmgGhmOCXYnLptWkE6KzERvVA0imtbMoPQ4\\\\\\\/xXtD\\\\\\\/eUlFRFlguJTnTuIfJm33YqZUZ190c+aBoTUCwKyPbVrQP0gdp6frIEYkj20I6PhVmaR9Eo\\\\\\\/1bIhohyfUDzCQd1yonXQkr9zEvtzgBLAUuGLW\\\\\\\/ClGDvmfW8EeUgrpnVRQt4bK3W85SNH5eGmln4IoCDOLNUif7mqoPaYzwYlBv4d4XFWWxjeyUpPmKAcXaO9SdN3TpAkK+mSpPWO7b85pRvy4TVZuM+EgWOHR3KNaaXaDBPgREjRaHPKd\\\\\\\/FxFpCy1bktD34KujYyNqC6\\\\\\\/+QUF7uenvPtTn4f4O2k7kinL5Y7EZF9IdIo9MxffX00augGMmkcpEIZEiYHFOuvPm6y8fqqcebBWbn0ArNjIgLX\\\\\\\/VxrSuDnAwTIKWLawqOZILXx59ltzGsYhHC7erpPUnJ2ZPQqizfyKvOO+ohoUQAa\\\\\\\/K09my4R3yP1CLkr6KJQHlFJGx0QckpvAbah9r6fGrBk7bZirRHoTzpB0jvmNPgvmUuvMktRPj4i1zQ6ikmHmqsGVAZfPnhFXHlgWp1lvMu3AwbeflEY7ULWxOBOnsbpmL2QIFHvMvb\\\\\\\/uXaRgUZX17jUdUAJLycwd3+dviwZ4qHYxbbHhsWOWRejnUdeUw29s\\\\\\\/OnZUegvMN71bAj8d1o8YkK2z48Drk+ZIRS5LZbofWizSstLGfJPby1SNQwsnzYtEhLMihILQAAwpTeVbE2O4WXKd6Snw3FE0FDW7Pf\\\\\\\/MGMwkXMc9ZDYNzBfvBr\\\\\\\/TC3CwSvqtjLFw0G+aoRZQ0itKXa7t6r\\\\\\\/RqTHzAIr74z0MP1aufVaUdxonTBehZKqaqLzffVcAtWZorEMFKvKG3\\\\\\\/VEiYLbUCx\\\\\\\/cKapXhyg5OkFA+QxQaVRcMWOq6V4V2RUuWa10FWa9gRK60c\\\\\\\/nNHHJzaN9YBAJXR2bPAGTESeuqLmCQzMBx7jWQf6ftK0KulpvMtHOPO6vR7v\\\\\\\/exmNqybwppLkzv2U76E2d+1schwIt9Rbb8nVA9sZJNonXjonPk\\\\\\\/oJptX0CjQDK3hrvHN6jTYSlQkbdJg3WR8QAzjtBOb9M96cvsAX00IujqjG8IDt84wVk9exJFdFYk2q1hyyu3v4XAF66dzbTm+HeQ\\\\\\\/r8z93Hf9NMhlFt8KF0sC2ueddqj4bxdVDQopj5nWGPKEE+bBt6sooHjLultD0yVkRb4+sqDZwoLDV1LG9Ex3YhCl\\\\\\\/a9yvl+7DiGHRfhdDpnEXEFhkvb87tkbyn5PS92brF47elbPu1IXCyAoTVUBne9Ft82dc4vFp0hU5C+wDH42HLIRelqjbVf8lj1QNYBzCNK3sf3wgOygQOeuroq7maiFjK5UzaFYYdwhhXaklUf11AWeySDmQfv2tqZSaeh9lsVRj505HhMsIGBOtzZAiEBG7z+5lhCLptRcIUgBQ1ZP3wyK4+RW3wP7pSkywr+NmbavaD7ZNQBhiSXf5i\\\\\\\/qqKNXNzP0h7swUofFypubEMBkOwnouz15OuKj4MUHb3\\\\\\\/FMo7DMdUXNuNPlenDG8eQ1vwN4ih5j4imXZZf1Byx862eFHxUw+0eQ13\\\\\\\/6oeNM+OqVrhqLPpXK6Fpba59Hc3j02xl5zBRXAJShijk1AnsWiI1zp\\\\\\\/welCFQAsD+pIiHCAQjWqOrbVfOHoJOfOmnXTP2qQyEbT3LbAqHVIBV6765grXPXKFF7382WZMBgdz7R4UBhwtqlq6bAtull1EAyf4hkKzbSUrZ\\\\\\\/2fB68uspoliOXAp4ySkujLuqQN+8PAzZnAGuFvg08JT0saRGIXNa1+ESfu68QOnGHyJxa7AKk8FlNjw4ZAxYz2fzJ039X9\\\\\\\/9X4waKasrWOw+rKaCDlVu8Y\\\\\\\/KgrK3l69q1HBhj4bbyDrEkog6LtB6i2o0N0Ha4AQMMkfXAKJ0KsadvKbaCM\\\\\\\/DN7Lp8d9uZyXoUf7bJdFS\\\\\\\/Zqo+eQwBjUfyyuJexeMCH4Z9hAD03fe3itor60BEqE1AUnu7gaq++Otupba9RTwQXTiqNXilm8R7khGEFn3y8EiVCL0y+ica1T1K91T11yWtTu788qstHNbeZ9ext7lbyJWvEKC42XC7cuShlFJsenpqWzIMEOiWNBIqSV\\\\\\\/vO\\\\\\\/j1Ri0m4Rv6wWwThEHUnWr+1TT1JZuu0IQKa+pzaskbaeJVRTPRgX+ZMjhH0+9tgv4BWwfL9Lc\\\\\\\/Z8sdS5gSgS+Iq9sFNaJRESvg7jfZxGVZBvo1Tgqfm4salG1V\\\\\\\/iC+NEtDtwyMHENMiLDiKVszK8YhXPXUKX\\\\\\\/e4haZteREElTKJ4PEzun8RcNSfhiP4orgHZNF+Y9SR5ZDz97g38ue8pOyr55kbnTqUcSoXkKJ397getcSDCHLqw995Fk\\\\\\\/RoKUfwpi7GHZAu22w\\\\\\\/FRpW5fRLDhDl\\\\\\\/vw6IXhdTpRZk2WIXXs4Dozy5Y+WGNy2RPSqhH2M9MxwUZWM8X3uVDcvQGZWjeUfNArtSaaTft1mQQhGkDSlkAhJp4TgUqL77GIW8KG6Qg6KyNoEoSLlLE+8YmINq4U3B4aumzr7YEJOwsit1Gy0Mf8eY5ejNJMOj8cAAAIHLs+IsUuFmrUnQCGwyXtAj8\\\\\\\/CUg1pYJ6LmYi2vu3HDshaoPNN19x+lO1vxjPo5III3SRdC87EsR3mrfaWS0bjNQxUzE48gVnSFCa0uxE++nmklHZD4J721MJ+SmKGNlJ6MVMHEGovXWzMMPcn13YZwipIXpahZegWvsQXuriOHdBk0NCHQRzAuvSqQk8p14W0dX\\\\\\\/vN336K\\\\\\\/CZ9l9Jt24kG8Tp2LQwOgDURL58xpbbgY2XpluSNvsDnTrpbQXt0U\\\\\\\/P+UEpm2i3TkyMsubRBmh1fFUlBArFBMSbKaMkJ9lVhV+lDXnulQ1o0544QrhMeNWcu\\\\\\\/uNp9HrdlLoVZW+JBI+CNpuX78Ei2zIFPrT9L7s63XT51AEYh5LNvO1rwSA94T7y1FGFyyIjUMH+yspwD1ReK6\\\\\\\/zLing84CoSjl5AIc84CMSaUcHKuUEikbn19gdSdth19InLAUe\\\\\\\/Qly2\\\\\\\/VTciQ5olrZTt+dAvO1FbqCVXA3W6sx\\\\\\\/A++4SeRr5qsuo3MOTzTFVIUiMC+HBjVRVlVRrfERxq4nB51bP8soqy\\\\\\\/DgDOJSwnP\\\\\\\/kzjwhjhN\\\\\\\/disA0a8H8lSqWo7Fs+1a0+RAo0pCdR0h4pO7JyVHiHXqyv4keby0xugKbcep9vOLJo+JvlRERCwbh+D4dXLN2Oxj0eHKrmFEdC28KIS4co4lXJHhomYNbA2iANaz\\\\\\\/MdimPgzE6D+SRI6Y7a\\\\\\\/xwYHuiMyY7UFTROdsYnGZ\\\\\\\/NSOfg1AzklABdMUdcJaMBE0u1f71+v2aak2YVkeOXAa76kVlBBNi\\\\\\\/Oy6F6B9FeXfueFUXqDCKBI3rB46MvGivDEBRAZKBZGywyQlGXYSFQ3CDQ7AFjFHzmUBYjWxlzHkmyAJzmSxx\\\\\\\/Wh8nGnz5rgee1HjcNGRK+u6dqw\\\\\\\/Fzl+TYg3zDShxiz62wpOtvgKykUDwwkTd98cmu8TCkeLH1YkVPgHLnuDxaKnE2vmfTkZwpF6\\\\\\\/vfzRBnqSfQX9Ux0uygqwX+LefFHrztMQagZ1Xn5Uu0guREgcKhYnhSpqqcDKYHUbFCOkxyOyC2DkLJD+GutskBOCDapDtuGgVZSiw03aXb8nj3SwgSo6AB9dghtvvjBBKfsZv1+Vv3ofufRqYjVfhn8cStEY9rcgoZjwt+TshTNAOR2PSVxbDC\\\\\\\/YR5DOgflwUsD6MG4SRc8joMIDV+mtd86UqWkA8fh4ZX\\\\\\\/rm1CgAvyYSqgCZnIAqOPaWyEXCYjhjKjI5xOcFaa\\\\\\\/PVKkW0mpgL\\\\\\\/+N2BJKAmkSQvN+DGiOJqPvHvxKYF9+8QA\\\\\\\/hXEWWB0gv336o3ab\\\\\\\/d6yiOe5kcYtfe4+sawolxbrwlvIfTK4+8\\\\\\\/hDehs\\\\\\\/vvRENt6GXPpGsQSSGWhDck3Z01LxiWL7O6o7MiRJ8T9L2pgrsORKhtvGrh8TUng2sFkMRZX1ESZ+8t+vqrxGafezg36oUvwBcZtbiRAKXkZRbSMX5Xk0ev+RxNohTV8xMb44zKvLEmuUUvgcS8kNoZ1SVnw82S0GlVDD3h75CpOs0Ein0GFR5kvwJ6CjqKO5MljmE6n2NCn5iP94BXhwBqvEiZ63vY0Qf2fpgpx5HV2mkSV\\\\\\\/tjX7DEI9bxPgNPtsdNUWgQGcXMQs1ErJVN9LWoUKFOJNLVfRyfzEvqFXbG9agJnE2HyKsDatJZelzxwtzW6QwJK94XEs\\\\\\\/zxSpEugS8hsofQnZrV1idJndS1NSS\\\\\\\/eEQsANUFRTkDGze0QfEw+HfcwEU7sA+jUYA8OSKEBZLfwBNWz5nuqW2Nk7\\\\\\\/fiDjkD9+fXMX6TQ5B8ApyqX0bA72ijEg7joopvGihOmbSdlZNyDx6eSyJBQRJi4TNRPwXLWils+TBNjCKuRYJBvkGH2nJM3e63Rc1Vc7aQU11bRTyhMPRGJ4oDsVhwnz616sjxodhdNXf7MBElRdSVKgwHPj6p+KT0Waku0UzBJq0pfPmjm\\\\\\\/110a7+tfP1ZhrIync7FRTzCVogglStgQ9QZG7ZOCjCliBQhYII8vuaIpKL4IXeq4gGIRAMv4A7c0ELHTO814Af\\\\\\\/fHYUFS8Ef\\\\\\\/WlRCCb6MCGs4rs30Hp9qoShzemDBokPcFS2BUoLseOwYvXeXH4kaRlIBh0CKye4GSKMNQoU3hjI9+K7TuhAodSqVs3jlncbNAmgA7C8PqWk7m78LrmkWbzTvbp7VVK1RHPMTLtpv1H5J19fsYD8uOkdG6GcBpLBEZ7tD3A2aBNDSQ8mck6NnVfnwbDZePl6coih9WAc7puxJRbMEfSm+6hcIBN\\\\\\\/kgzSl4ikbIMzFqScd0JbzvFYJg\\\\\\\/ue0j5aCIpMsrhwqE59zNeCmiT6PpWRaXanC8iDelArXrn35+W6V7ai76blsC5tmoFp5HrVbjDdT0LcLDbciOAN4VexTNOWiXgdjgHeympJu55CfdLcUNZiuZsdmPgxlD05jBPLYJWLTtkBRxhVQNip767NOTz5OrEDUK5fM3KCOT3iSsSEIbK5j5JrGdzIWkQ+Z5BwtvLeg0jm2d+adBd+BduUkYiD6ABhfW7Mb8PuyiRMByV35L0nDgDZxgunr++bs4zohuCk7hYXiR8m4pmEJbT7Hzz1xiw96yJkMqHE4aNxN3aeSmLO1kk7FIWtw0vBYPR76EeGy92\\\\\\\/6nz0FxStPBKBJzXiTC27bCI5kdjNYgeddN1po7rQ46ZAhLQ5MflPG+wuHuBlB9go\\\\\\\/S3dOSEU28GDts\\\\\\\/Rh6u1MS2a06\\\\\\\/hEYt6dNPwaWF\\\\\\\/9e9xie0sVGMSX+fBEj5q0cn19vHlPBe3fh4uGuSwh26Y4V7kVjREN8fSlFfFuzD8oZRDwCkAfkEfUOF2NnocshEmC+kUxKH2wZ41Ih4Blq7VcoyDY3Tave1bv7i1itl\\\\\\\/Ta9T01xHwoyX3Uang1NyPzWn9PqNyYBPuGbvvXR3lDBNTxzR0\\\\\\\/oaemQvE1hy74YUuN3JDGgf7JbITeCCWfBFo1itJ7ComTdVSv3\\\\\\\/YQshvy0W25I24rUuUO9fdZvINc+tsDbx1hdh4FOeqKRjXCHCDSsmkEA8KuJftmfUvhOssB4MSnR9cBnExPSCYDVe611N0bp2TaJIee6JrUWv0SZb+50to616vcJQdZ19fNG6OObBo8bkZLd9wqHphTlGbxYuBCezdZ3W2AQiuD+hQ24clpnFyuvxHibEmV24knj+0onNtw9oUg6paw7h9iewEmH6234q2cyaJFn2B1dSZSGnWoCKzyP1DLwRXGg64qIXwnwaVsB+zr2IsJyQ1NJZyRDIkawaDt1u1mBi+e4\\\\\\\/6H70owf8aQSwyz2Bc22NwMb8YTVcWy3TN4uwaBma+KL2PuHeXw+2Tk6cxiUcsjTtnhtchQbQJHzoErnNB7sHAb+r5pBAsDW6KbRp6fBP1HLpofPLgWu78C0CG7QTQo4nHnlnk2C\\\\\\\/a472KxAeC7yYZtUjvZf7klFa+SOezF\\\\\\\/\\\\\\\/usPwuyjrnN2NA8nEXJ2gPKWqGsCIXLuCyY4WYXPVHZckUUBPEXvAomuRaab6IaemQ2mCtc\\\\\\\/5E2darzfBe9y3A8DJxmKvSVyRHr\\\\\\\/JB7CgZBN5oKNsq2jsA8LsoWCmeO6j1NrZ8rs38tBRL+Ag5H44KcKgg+R\\\\\\\/rJNdsHLibFKeSoi9bSuBf26Zf1b+9evZnP5NmhfUvr2DAvWKsHtrWnaRE1vbmOn3BSn3UDjQB5LMv\\\\\\\/eb87cao+LftdoLQ9KXxitsGipQj0jKKltIPkxiT8ybJWPzQIwUSbM3AsuUSp7ot8KhZP0d75Hp1M5wRH8c2PGNdT4RrWYpPKyfmLzb\\\\\\\/3yHbd1DoMsay1JzsRCGcbjnKMQGVsM\\\\\\\/tjkk3yDaRncB1bOfGoO4MxQg8Vb\\\\\\\/BbjK2jcBjAhXxa3HvCewMucQSmb4sOmaT\\\\\\\/s7eGwoCcP8ff15WSzMAmUUeWoIcahVGnnFE5Zx0YE+3ljLnWPwcoGhKHLwKVAynh8ixLG5p1RFK78k0eXZIID7R894hyEbpMwQ3p+ms0Sq5Uy4X2PdB2ce0KrfokPAOv2kxPIgB\\\\\\\/AYIcOAPN9kcM14npSePofKCusNSaBYCh3jcV6iEUThkqGqF5Aa0z6rOv2hqL\\\\\\\/MiResArhVkH71bQLmQsosJGeW6YgbNKUQTpUuiLrcsUwEo92Bbzc4xCKdqh1IQ2ggPcwEq2l5+NVQEHXv9x+NHpUcC0h7+l4gpHxFMMj2zYPB94sCQpiTDQZVcQKMVMOPIwxEzmVBcSWJs0ye4SATAXSDCtOZq4b0QGywgr30NND81uEOpD6iD525\\\\\\\/G7Z1bBOXZYgja7Roe038sVWImwYOOHrYhcka50RRSdaMgqFUOjRgI6jvE\\\\\\\/dMC4F7mmizvYPsv1JVJDdimRIaExZ3\\\\\\\/4FnLy\\\\\\\/F9GLH5vqHvK+g3REqsEYvWMUkavYsi4S7xzMHDKvn8MOJiAY6ncwQ4EUVBvvdNdhkQIC6zDuY4KJfeC8m4W+nDDFEs0n4pnn2o3tVmxS1doSzc0mtRkwX9+S8ZxbazLfsHNr7YHrQtpxFQC4dFyq+3M5USGC5BE58qoqOWkbZTFB8JClX3HFXhFjekuBaU3oG3nY3M\\\\\\\/pXi5O3d2b9NXt9\\\\\\\/zrBW5da9RPkwkS2sFa82DIVSUdU0zBQUy+LOUeCPBXlqFGh\\\\\\\/Y7dGfajYJtTDZH0+Y\\\\\\\/7euiyL7c7YvYuMQyTYFbwlQ8tgXoCgMlZ9x4zlGLSx0Kw55jGIwijuMj2\\\\\\\/rvheEXybyTSXB7mYG01NPvTeYb9GsFmq1ecXWcc\\\\\\\/Cr9LCAipm5M82b5EaaLEeBkgElr6Rlj\\\\\\\/sB3XtpMMJunBk6jJjGQLz+tkYIlrGOS0Mg6T0SHrA4WuI7S8ZrgmjwoCtuwAIzzTKDWCik0JayT1Xd7VCgpr6oKuAQBD9srLndfktB9LERbAfQwQxzSXY+hQZiCZk4FyxsVGEupVStI8XzqzOU0HD4Qhry2ek1ilaIJNfflqGuyb3A+cEbCShuJdZy7+6xCnZYzkmtiTRnGDe95M3fQ3VlumpW+Q8ALQ1jDFNfGL9tifKwtxf8mp\\\\\\\/EGMhrcaR300iM6SMzVEuCZjaCIVJfGG7dksxuZ+yF7A0kgZ6CKwEl4rFx1OxXiHLNm1CifrjYZjFiq\\\\\\\/+9+2TFtZDNS95tPbBPVKtevEoM4tADl8ilqvgLzeygQVLUa\\\\\\\/HcILRU3Csw9W6q\\\\\\\/rFxJP\\\\\\\/g\\\\\\\/TWkLpR6Mtdh50bGE1vxqcuHKLui8L08SfnUSroCHIDeRsux7AJNE3MTGB8mv9M35gORCnvGMzmLkaxOYE3dC27neJcUAzEPHTVIN8tlzpImKj\\\\\\\/kQOPSByBt\\\\\\\/U5vlXg14DkIxRMlljBAmfAtsOBCisOupX560nfAOYIt+M8Za8\\\\\\\/HTapSZc4ae11xvPdkU6heKFy5IIxphK3gqkmn2rnJ\\\\\\\/ywGXEpV4sxJ5\\\\\\\/dRoWlJ\\\\\\\/slsBxnqFcoLuOrAGq3bMGGqX8Ob7bknhPrIDgRSRSj6OtZp2P3JJNj3RDJHhsZCH7g3VVck0u2VmhAwfeO9rvpOc+eEwYHjwOWfL8fOpcr4HPFyKsmekw3Om6h1T5kUUAIm46YjM28P4Nrlo4fKFU7wLL2KtIKiyuOF1sIVUGHmJ4gmdpibQ6\\\\\\\/si2pV0ebru3jba\\\\\\\/qiill\\\\\\\/5R2Pstc95NtcAgkJI5dkcEkUtMKBQmjRCwuS6dQBRtJiY26Q9MyJMQ5bExRDP8tv7odOGUxg09\\\\\\\/KtDSGqHsWck1bRFT9DF5ehtDkhYafZxe0KZB1YYbKZ6\\\\\\\/R78Wi5SkTi3XcjPT12OATLWQ21lJGdoBs5vnPV3y6VAc5Ns6GTVQ\\\\\\\/ibgiK101PmcYgcWaVAjAHnYdgTwhGp5NVAAU\\\\\\\/Vt5vf1VuYvbtmmljZlZQ2wJxUKLmcM98x2TEDxHhcNtdIpakGWQr9c5iPesMNr7F9pm71Q64YfjqMxFpgMOLNPpG\\\\\\\/7wbnjfkXLfIjrCy3PNj1Xd1qX6bUt17tf3X7UnQPRHM2+EZ\\\\\\\/qRM\\\\\\\/MkNWDCjlU2wFuc+cACuLkSzu+PGe1xHx6td9j7\\\\\\\/rUCqhk55\\\\\\\/Ns7WDlH4W7Sw5A+MYaI7pJeObAB7c+\\\\\\\/cXFJ14ZtTftlxjxpLqYb27PF9CMjg0DGRaR8pPkp\\\\\\\/0gq6fLcl0bH1spIwAiqsK4EnMoVhixPM5uTPR8Nb6TBX4Eg4hoDBmUU7GHeBPOxdi8UiW+g7nDGL0U4B9SnT0VKYr0anvnxEnJuFyCIiWXbg9tPk9\\\\\\\/t0xnSbO1f2H\\\\\\\/ajnQ0KF+bHcb+hyYCYAexd9VYxzh6X9SRthrvXY9WNJvMi9KSFZCFPu+iqGmEQQHK1V25K2ZYtTi2IfvFdx+4WEJO8Q78Lz3A2d1zVWLuNka6o5Q02QBnmqLXveqp7MD3OUCcnfR6XBlguBXn22X7IjB1tGM2JVGLao84FqR5CEIfsV2MRqOftcty0vmDjTFymkSvVBuWpobm4swgIqYF9lLfhTfLknQm5F9OM9N9+I8J35vlWJ4wy+oFOMTXz534YHhV6gk1i35Z8LvxU20yhj5VerfodQMqFgCXWzwQxyJvXHP5DuODtzwoRdQONBa\\\\\\\/\\\\\\\/ehpV6+DmzYm06KAQq\\\\\\\/Q9EdJTb12phV\\\\\\\/wLKMAYPKGuJTuh+gDWqjmn7VZcKgpQ9gOSvR12sZeCm6tPk4P5LEiPM+HbN90O0qeWW1\\\\\\\/eMq6PS\\\\\\\/P5H4AsrUmAyz+bLhYuKyFlGO67v7CLRQW+xbcn7txShVzcwukGyQD++VSVckK6SN\\\\\\\/vSlnvhsuTLicpw2J4XBoPIAX+iRffqPW3IB4lp\\\\\\\/iXv+b+jmkosBZU00cs6FQD1mq5gj5L9c1A68Pr52a825Ff98Rccn5GHlW5UTeWQLTfjUxYykAXOVjJK2\\\\\\\/jp3kNya9DLYaU3PRRfYPQ+x6Qo0GhS9jubW2IMEIHQ0HkzvcC0rbmWd+z+JLVp2zl+FoBAuJIQvqK3T9\\\\\\\/HgBHrW\\\\\\\/Sgfxsb4ysxaxR3+F5wlR7mL7zt+7H55j\\\\\\\/91IL3McXiL5mAE\\\\\\\/V6+mt\\\\\\\/NWKC\\\\\\\/khKFcCuyU0gh8UWExkZpGAQCFk+bWJrzBTJtTuFqE\\\\\\\/LhefK11Jg6dM7AwfQSHRjiRPX34IzxxEzYj242DKJgKPyMWqMF15NxOD2kdt9ryk3FV3mTurRQuMpQFvAA9zM6S7pOYuNOLvvaRf0hLDXoh1NeJF82u7W9FzSYTrUYCHs6YRPyone8\\\\\\\/C3d10MjDpPgPRwHwvHmzu16YJnkXVESEfZtS3VMY3eHuTdLvyUvBOHE8HFfsVr8cB34NnMRRCtzXWRq8TQf+uBmZp6PAr4kwCafg8\\\\\\\/NK92\\\\\\\/E+3pENcLiXZB9TrEUNbd4McAAmVtcyVCzZ899zGW4nKMqsIehO10N49gPGGGAUDeS5YjaGwNvve6YJm2xl1\\\\\\\/xSWXb2j0Nvm17QaQSaM3FwlJzVOa9e1d4eTSVuy4KkGoavgH35iTCvh3P5wvGZSBqJM0alfAHSPWs0+9yauegq1wUche1TzjY49fPndpwYE7hPtm5A08aUqeBTNTABOSY4bVKZBGmiMKxysejeDugxJHvDahe2+ztYTrzzlxgFysWOjy5UEy\\\\\\\/WfWGCb60EBq0pUujzjHMrrbAuQtFqerySzbRdsZiS9HgodBNOgg8GWpMGQnzs\\\\\\\/leqxOgmMX07pyPzUauJQFAxSfmjVcLdFnbdMSoOqMtcQJULYJrLj33jTtwT5ruVaNa0hkQkDkHAFvwRLyZun7PRY5I\\\\\\\/lwRSrlLnLG0YrhGwtYNGt2nAcndzF4LzneLezk9mdLSloXwxCl4k9Z1lGNETxe77S23r266VYreWjSfFmV5o\\\\\\\/u1fRLyoeQhR1i8mEMHBz4u4ZO37CzOZkzeQoXjCZ62xxQZ+ZQoMyQN9kV\\\\\\\/lAha2JraCE2z+kHhJBvNCvrdMBqO+pZ+b52RryOqVhs06DZOkFQWKtgSmJO3D19ymsWo9mxgLLnnPMRzai2xQj43f9VfRklX6j0ZUBuI\\\\\\\/wYixWAKQ6\\\\\\\/PavLcune21wISFKEiZ0CdbRMsQz4NU2NtqHniCbBJNrda2\\\\\\\/1soIL\\\\\\\/xh+1sLWtY5vI22te0xTKxr3q9OtPZRjXQJcJzD2VblEBhRs7nAAjsU6\\\\\\\/fsu8+W6Qqs4elF2JZgDKhSuR18f54eczy02DMqSK4XZX14o\\\\\\\/NKE4LgijIieDGowYx67I1v\\\\\\\/EM\\\\\\\/4iA8euC6L8EnTgEr38SzTQV4uVdgOSVSOV8GQHiGqqpLR\\\\\\\/9HPcfecUo6wgnVZ8XgzjpvqwkpAdrni0z0JtND307ZwjyrvaNnJibfttiba7bipENTXJ6jC1Owbw5\\\\\\\/Y3iHCjtT1EqwaSpoZOVxAsJADXLSldca0+MsR\\\\\\\/wJTGodhWFkAdqrUnd+V9sNAZ8n1geRsf1mWgLXQTh5e3PfGKhm250whTW7Y9IlSVp6dLH+5\\\\\\\/VRIPGsVpVlRT9OCvxY3KRJAs6aqVHGRkSMNoKnTkX6\\\\\\\/XSLijSiKGu6Cj67Z6SQyJH5mvMmehVGDRc6d1WyQBKtE\\\\\\\/PqvXWhEQDL5B9SUgc0fPy13DKeAncduiuYmStGyDsbxE7esuTPSU+HM50YFZY\\\\\\\/qlIqq4oWgydygCnVvL359QOZ+nH0sq9vOnQMdXjksicBq8ICgfT7f5qrAdy75tjYCN49OR7cN9S7g8H9p+TEY0VmgPwa0ugkoAf5peV7OwC1yizPry2SXk0PjQC9xmrf+STdtOUstiXbEXfqpAJRvabD4VmBHvumw+qgxvbnmV5IaTTQNdqFDGYnPu2qveLJDK8lctoSDC9ZLkiS1z2GDpk9+fCoeVXObF48aH\\\\\\\/HLwNbcK6rQTKb4CyyHoXGQ+LRuipoOkvXj00ht0gEwOAoHdgeYLlNqZZdianWVNJBY9rYvx9\\\\\\\/jOn+z3GFmZB7W7Sm5e7JYFtUHeTd+AtTzva0NigDm5UU\\\\\\\/qpr9GET4EPgWLJnBsZjCVXCS7bKuCnosJgz+Rf5BK9cjxBlgf3AxU+YpkJyRix\\\\\\\/YTgxL5+KQnW1wrbIcsnxYqmWhXa\\\\\\\/2IR+wFTKrO1k+0Cx3\\\\\\\/NDsiIl6FtwsXjg9GVJGXx4\\\\\\\/51yWQhnFO72eoY9S1PuMasJyByXYQ2e9moS4EyR65UInEVZbUGGx7hqW0v9HpezX1lC4Ef8KkW6YK4KKXo+RRECuMwe3yWipyqosfoPaaKYXVOMMmQue0rr+1oYH27vqbz69A34wUZUquGKh4vm0mXWQQM3qkQOq0ro6PHMiWrhv9I6LIjtmtoiCER7\\\\\\\/6T+WJrc\\\\\\\/u5PgPOjkvghu7xtpF9N2KuWQ8D9Xw54\\\\\\\/ohoCAbEtTyowS2q91gAuRpht0ApbEL8ZEW\\\\\\\/cif+dEGKyx7RbJFfJ5KkmlG\\\\\\\/K2YTLT08J7DZXLR4B4Mcqa+nQkIkdQJfGsCD+XptvFQNw5Jbe6PUhb4HDlpfLv3k5KIzyeeRpTUI\\\\\\\/xwibF+6GQLnjQoC461u0j1qkrxvHP6ynHS40ZuiDT7OqUJt8IVXnzd4t3MmJkJU42j41LXPsFRPW9lUkCl1Ten2wwK3OoqdQPOb7lnpIrbOWhUsaUyut\\\\\\\/1D2\\\\\\\/J1fHqKvrWq6WmSP6Ss4hRA\\\\\\\/R4ybghpc3iy95FLvI2r1mD\\\\\\\/X2bKgf1Gfvz\\\\\\\/VOuXzV+0OOtJs7JrVFhucy2Dk2FAjgVMixXrvFZoMmbFKtGF24XnbjhfICOiV7OhuCkIAFURej5JUnyyEIbaMc5uEF2pCRFPABSluTmc5Zd+K31r0gClXLYkn3fEFuitETrSp2FSU4gs4HTqSqDM0uYuasPPUHDZCnP8eHyTWKaSgmN9+1wX10LqhF\\\\\\\/q6Cwj3BMBeGCwDfcsMaQZPiuD3y3C+S+77yk76LZ\\\\\\\/i8cKBiqjPLcMXomU2SwwFvnHQZ3SwoQ8KBVsfC0n+JnK5UhhdpQJsQSq7Cswg3DNjjfEsM3Z9JR3Hny8cSy7mX25RVYG+K+nkD5Mb6l9pnH1lDylmA7HDsFQ7mjkyD\\\\\\\/YBYymUsYEjOttbIQaaP5KwsprxAS9yuU8xzeAi4+T7ip0CbkoaqOsWgZy2MCre7BPrvEmlkU\\\\\\\/hcc4W\\\\\\\/ZNifutOzpLOIp1KUj4pkkrFc21fO1tqB0G88oU7ep0zpDK5CKBHuCd6KAsSyrTjpRj3Z8jnSQRMfwywoYJBtCjK4KtopvHfWKdvuAptR2enr+jllsEDzCIivDdeJwWPzLHyhAdiQqsXOmubSAP1pZn9UFRSd3BY9Pnt1ASbyyZtAWifaYaMtWJVLqNb0TOk0SrpQ6oNks8eNjRUXHgj0e2UUHYwqfXjczBGkiVjXp21zc4AzvA8mBKJdOhxFRbgngp6oAA7HrtIVABp1VY21kTI1kljgGrPf\\\\\\\/T9qaaRUjgh7aZ9prJin2QzsGc4hxTZKiYWzfJsv11Y8QsQPR5c85IW9amm46n4hCJrYkuSJK136C7s2WWALhkMF2SJ25z1JfR4IUnNfORd5elgzmhU56XY7xeaYmoAuKk7jTTOk4YmIfHH+DeBGCx2FR+ri4M6h+u4KJPGqwyi0BzVCTc8OXbZRXuNEtMVz1wSIe2\\\\\\\/rx8lAcclfZw\\\\\\\/2T0n5vnZbGsT1c\\\\\\\/tsXauCzwqXdlrrl3dE6Gwm2GeItPwQuMVa4lOyfQ6CpCwi0CPYumZVs4Kk5g+yl0CKadG2dDzGCj16vik1A\\\\\\\/oMvgwnqad0xUQBf5xDDnb7VS6BEMumvhkC\\\\\\\/DLCqtFOVU3jnP8\\\\\\\/7wpWMzG8ohkpKBOp8I0lwr6PV5lUfWGXC+FeUp8dQDBrne8So8ebwQ4Ut\\\\\\\/QT78EnbL3902vIIU8x9\\\\\\\/4xbA6ua0shHexUe5HPfs37wWuioUKocEKd9t9j69KkHqzJ+mlEu6RVUTKUPdaBQ9APguOhK+cFJxa\\\\\\\/BBx7o+hyK1VwU+QFe+Tk81Fz2egUMpg9eXmcka22gBS5xMcEn4hgEfodWwFkGCm0bOM6WI028PiMLgnWyl3WL3ZjxXZMYDny2k81T9mDY3LgO6IZCbY+TPyF9XGY0\\\\\\\/ppE\\\\\\\/Ah81YRaCngO8JDpfNRYxFoG5qxaGzOzqb4pReQvNYBPXQy+sLq7xZV1CaI0iEfZvLxVBxx9Iayn0rRCbRoh+IIK1gwUlSYAoh2ewqKZz6MGIm4+ICBR1uk84qOU0Km0MaXu+fT301WP30Avl9gNntDBcmEyVniLndn8i7r6ZvZz6+pV92Gji5YDZ6TzGOrNLruIxkyigkoWNzv6r3foK0gPL3ZepVaSNNEvQIDdtJkeFFEX1Izbgsx1B\\\\\\\/weUeNcbYPCfsPh4Pe722nIcvXVkF92HbiwbO0A84NpRUPSqB19roTRkmPO1aETce15Nh0VCmJJ\\\\\\\/M97PbAluqkOPJ+KjWEN7egqdgRSp5SjG1LXIAbFLXM\\\\\\\/szf9xHO79AmZg6LiY4+k17AYHC7RyZ05QAKT2DZXLl3qg13xITqcZw8QqRXnW6MjGz9NnXMN+8TtIWTXPsAyY3C6kq6Vn71rNZGMQRjA2\\\\\\\/wflH69Xz3xvOYLQPl8AGDuGgm29WnsRYk0bY9slbs9UDou7i7WMoJr4p5m7QYDh3zyk+2YoIcn3ZSCRDzyYzQ4Aas4eArtK7\\\\\\\/xBDe\\\\\\\/mZww7nwHDuZMyPFv2PNHOsHX62TRLNtTbpw85zQ7Rh0l37l3NwwkEKXFrUwYa2TJBq384CMBY8rJETxQlCJHRdlUILasNxBTY8MnHLzSVbwd4d7AUNsaGp4bFMyLAlVzC6CTIk+EQgLrp7A+RLJyn7+UMLmvuum8SnSw3GEWkwSmMaZIZD174Vk6TiWoX8WFGB9rlnxDka1MrAkdhbazt4ikNrqG\\\\\\\/wE5ObNa1ZzrmeuEU3Hn3moy+qlniC3Sklev48PzBp35vHvLxWTX6WNoxZJSTJTOBqWAQNsrRworXqElG3Qvx7KRl\\\\\\\/WwdKwQRlXdDS8OIwi1o5ZyyppFchxWGtdFVgNJ9jJr9Ew7gAr6x7NnaqL+ZyoQ\\\\\\\/4JX7pgYjP3e9UNxKOPedZGlzBJINB6dF+Gw5zqTZBFRqJIX9osw5tWAgxEmOzQ\\\\\\\/hZAa1oGG75XTaCj67dXhgUlk7vPkm3pIgw2W46JNCeDGqKmleRqzj+8hqvTlAv+VANmV7RyKdeemSIVsmqC5Me0RXp4Nq5nEkWqUhGYAztlSnewyAPsYJJCdiprOXXv\\\\\\\/KkI1iL1QyWh7uKCOtsWCxTlILSbzlIGkAROS0pkAb9Ic713d3rLX\\\\\\\/7x1MSarYhmKDDWyCn6VF\\\\\\\/aAQwjUWNzg3AbeFeDZeWvXM4oJMJuDIMsQDQhlf9RTJIGjthmDxYiOJB01UF4vMyl0yLD0se1Z7kAbE1+LrL8c4nprrej930C+BgDx4VAXrI+azFoYznlC+CzshDfKmVgh49QPTvK1WzA6B8YS5Q5CcP3AptbL5qXpsbasVmFxOZncmvXRlEhsjJ0bTBB5WNcgfb3+p5OAq\\\\\\\/ZqVikRLqKJjS\\\\\\\/aOwopvnxZpYYcPsXHHWyE+\\\\\\\/+H4YYMoHf1pTfBFjcvtXjPa9tek6Mz5CiZ4LeVxa\\\\\\\/iqf\\\\\\\/cwxBT2Ud4Yj6YL2TY0quwmP9mBASjFTK9WSGEPfS2cHdd9oqUmviR\\\\\\\/7ndAnUEXRVqj+zURhdBXPDQxFG0K7ewLKj6OJQ9IYIbWAL+TCjQxTIgJTo1fiR5AY10LgTUZ3uPbVjU9W29ndXo6dpYJsduU52omV5IYvKsvvv8FrZaSXz7QI9xO+9LfbDToKCt5vbikijz0AbLa76OLypU60++ojEoo5YBAO0fCgPIlkJq\\\\\\\/yiLgEPit00kleOt0MLmgkf0PObglVeWJ8zQSjxmAEr\\\\\\\/bZp\\\\\\\/BxOJdWSJ9HejAWJmdZEU8ehJi75G03QT+umH05hjXbjV4r7pP7GJKQOZsWr5CWx62nqOYm\\\\\\\/BadUMH+YZ1ipjCeQRTU9gzwjf6GOiRa9NzfFcyeTdHHEtVbd3KImU5kWwYPnrgNdw5TOwtIp\\\\\\\/PHDxsjHV7XPktn6u8SnGYd8Tu8cYuFWvcqyORKHgw2CQwZ6CGl7KRjcCgb0v5mXpmZaG7yxnnzHxWPM7MI0H8HS8mMvCoiD8VeNxejPoILR192pOH7OrTvSejnbpu3\\\\\\\/h57u\\\\\\\/yKM5MJnI1t0aKJXqiB7+iRpRD3PewMh0pSiAYVmH2D+plXNZr29xr0php\\\\\\\/TI68tDDTd967RbxvsRktJLC8TkGTR7x+x3DgMcaM3fgI1zCurBocSsgNpOEn0zB8llSJqa2mjGGdwR22+oag1gjiqBXXlxj82p0pBwkOlPoxVBPAeTZXBUluMX78CiO6IvUdXX5dKzhLFnInStJg1lzz+EYwoEb1FG\\\\\\\/\\\\\\\/E8FyuTVYpGu5Fc5MpXvZbt5qjMfCt9Hk5R7hxFQL29lgMGQz\\\\\\\/Q1\\\\\\\/cwhjGajfOyiANnfxGa\\\\\\\/lS\\\\\\\/xiadN\\\\\\\/RrfdGmpq0XZ7bcVZN338XpW13pRyBUjrvMU9KytZgjFDZYA6iz0oqqODwcWLkyjy\\\\\\\/R8eedO7tvkNPOmu\\\\\\\/hn22Mc1nwB7xT1Az9+xYHmCM8X0JZBIU0X6y8g64yVPHLtFqLTA2WU+sCmkwH\\\\\\\/4+sJu4eZnJ6lefSVY1vCjq4H0+i\\\\\\\/CsLhrZWlvj5ww5Fk0E1+Igz\\\\\\\/xBDM6CB25x1SD5oMdZhSzid8nGBV+1nczNSvRlBGndLEHsUUefktM+2Wmvl7vPr0g1rwiu2dLmkkHFyXaivBOJTQj9+z38hxWaTXc9JAExXcs64Jxmcd9CqBSr37LFWZ33uFgUOyuBwcbf5m4PEwwMMNcYJ2P\\\\\\\/mBy2X0xv2Jpg4DrcH9zWahsdYvJVe\\\\\\\/uqmCy4VA2IpqpN0lIs\\\\\\\/AhaHtQxXBTz8dwdjKA0u6M2JwkqV4yvyRQPfeh2yThmMG6mqKfwxB8qqb2IRBEsMhD6So2e4u19cbGMHuNa9yNfv6QReBS3ohKcT7nUyRnoT8x083zZBsDI335bSc+A98mqlaN9J8aiZ9x9ygApc4A4i4yl5CJryhKhVKbQ4HBSA6ZFRPCLCt7XxJ2U57lCKwtjXn\\\\\\\/g1\\\\\\\/cxUjkzqlilPNtg5Rsio\\\\\\\/G6besdCfYZfGSzRFKJvIEFZorAOxUbGJ0NHj+UXNrBpNTaODmO5YQOnKhfu2O22ELVJg8mXTEa+9VuRlIiNlL\\\\\\\/6ULEFQxlgRxXPy2ine5tj8z2cr39j+6OcDzpRp2VG+AR3fo7XGdmVdLnhe3QWBX0KqqcTC9ho9rw5loDRrd9lftWr0E+fJnt1uqjAvuIfQH2RxQ2P8+R\\\\\\\/ZVgeFkLzd7QX3GEA0Ywr1VoRQpWgPSbRn9gmPqDBnU5aXApnbEiGP92BORwW65POn1CMYLZXzSijC9UTsy15MBKdaj+2bJjTtCp1rwqgZ4m4ZqmY7V6akcBfSFsn9x1kagqlksu0lZrKaD4FjJV2dOvypcrTqdJ3SdKZPXZArfKCk2jieLmSva2bl1uxF1cP+hVljZlFrwcfSSbwajpTl7sq8+bgfo\\\\\\\/ZpBnU7mFFcQNDlcCys8Z+tXgB3x4x0ScmANGkSQeUlMO7ZOsIORIIz8ATYwwKx2kZCWiWZ9tRYwe3T+xiATX8Ef0KMwAUt7pdVXCU300QgpIQs52LxWdXi8NvYhh3FPIvBRHscZ2UUJMYDdpcEkeVD\\\\\\\/cNcjQnTpTs6+EsV291LUYXFrCSR1XjYGg4lWAnaOUbXCGLuWx+u3mPuZsRZqrYUvsj79WkieD1QFuM2NnzHW5fDpyD3fUavh47wLei1ydzgvw+WiRyVEOIOC51nyj5WDrNDIyemCZd3zSW5q\\\\\\\/oGUTcz+3V8xkfzHGg13fq9aYwOD1kpyG7vRgPpc9aHMQtdn6dy3tfF9hv\\\\\\\/alUl1qlbQTPu5bpsxbvo4MDg5ipFHXWBHLVPVBZNx5WT9222OzZ3lJiWwlKCUTD3oQA+Nx5zYslTyhaSleVDCbWduoVN8PhBDMG88kj3ZbrGNyGk773pBiRhrBUENZT+f0Fc8yU5goaLZN45L\\\\\\\/KyBZxYqiou1qrsNh2FQLIS7J8mOJWrD2t3f6QsZI+XAGwYLsNBxFhmKRxKi0vq6eIKZO0uH8yLmhW4OQAuDAlTp5QV7msWYSye3fR9fNT8spNL0ByqT8dteUmX0aabngwMiNf7wi5MmwL0GWQIalgjv07Vzl+BHaXc1vYxLSlIGEOsibah+gh4X6qTU2WmE\\\\\\\/o0Kb2LUy61elKdJZuvmFTJycrO9ojqi+pdiQFAZ25g1u8RPXayXj4aq4bUA\\\\\\\/Okk+1BcvsBcNGtnH3zITuzmDNr+YibR0xLGiMhCmCHWCQUsRJMDGCojMFQv9ellDOL7\\\\\\\/V0Cdc6VHSoteC04SH8O+5COJhp9hTtPxJTv8pSzJCnLIxIftNtFHKJ3vaAVrbv6jlDnbnEOPnDJ0Rc0xjmcT07Op8ITg6dl3hBNXMDXrciMAkfcURCU0Q\\\\\\\/wOdpRaBEleKmgrqJ9uOac0W0mZmepO2AIjpoRPthFndkFLyQTgzTfGl4PP\\\\\\\/IGAzPjPZg59RNKrZVFfbmjpUrZr5EzvKD85jTwY\\\\\\\/TNGavKkZgGpUKUZzzvXsrWy6xrek\\\\\\\/\\\\\\\/bdbmr+\\\\\\\/tzidwQgUeYtnOySFJdW4RmU5isVIvQo4VMO2+Ft19LQC\\\\\\\/Zr4alb38y2H3vq35tCmMNVYz8eMoWpWNdcbUCcbk5C7SoqWLDVOexvcOt0pnxGXZdoczvjU\\\\\\\/EjpMjjlEPaTMvCgU22nm8iIVXwp2Rtu5dIIJGjwAKXuSbX5oGrrH6FGwNVPF+mHBEDBRwqsOxn0DMUvpfBglATjAnhUfPLIwIElRrSPhjv6SB3nzE\\\\\\\/yqXc583Ei+xUpXm1rED4lgNwuBdRMSwjqeCh9\\\u0022,\\\u0022iv\\\u0022:\\\u0022bUAEtjgpthqBfGfEdz3nPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224c7fcf791544b3fd9d9cf02e129d531c987f8035f0a7bd37a18ebe52599b5bda\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yAQpT0HDqOvo2Eq9VFAOiqiYsS4XHCpAgYWRzqkqhqwC1zbt+R58L+jitxo2Exrx+NW3wf9UIDFIQUAIn+rBE1UKi9uIzes1wUNjcTVzVlzCWaXh3bYIzqoQwAKi6mNvkS7APkvbja+ZdpZCla3W8WGv1rvcBIUSoMowz8DiJWIuoH2UgN7rg630ARIaB\\\\\\\/k4m4yoL24gn6mAe2nBNxvAhlNwa2nYVUeqC\\\\\\\/CT7mTSON6IlcB01q\\\\\\\/brOk3igmLUtIoPZid+CyWIlq3jG9WJumuSCzTeu6QuOSbOBqDSuaLRoKXeOFL8+82dj2VuUvcxv6mdUlF4iBDK7XzXhvd7TNfkAhKySY4WqrdxwMkDDh\\\\\\\/SOEnYXMWSo43VWE4cMoxNek0PCCsETiR9wvB+KrWv15q0BxUkoH95RRFSvoOWPuz8UFqUeGZSABO5Vj2hCEkjHEpIAOiPExKxbUIL7YTRtpwayhqJ\\\\\\\/YxUP\\\\\\\/AyHNu9+z4b9Us6AO3ZNry5lsdlQxlxm+nLdIwUdwYeDzpwsNrySRRQPEuclLS9QMgguSIRTkIbQLrvAf969+pI2fhqVccMxcNtvExz64UAt2H7klu2v80TNkdNk6tYa5+fvMpxEV\\\\\\\/Mr+zIRv4RsgOfsFPX0zXpmikfvL2uU+LrhIT42SZ3TXYtOtEMpwJEkdQiWOLiWUD10BolpaFUkJiqvPq4Q\\\\\\\/hAUnP9QR1pRxz4voEn5x+1vfmpUU21a6BsFtaJmK1yJ0CUJFW7Zdwkw4d+yJxY6qSuu8v+vESTerI2XgWs9eqkId+VIRpBJi\\\\\\\/ENWN\\\\\\\/VGhfCq9gs5+2ekRorBnHMjwlwIBGcRjFahvhjk4c1NlC2ydnD8G0AgLPX9EnDNvjfjcNTclTkKNwoLWt5BvDk4dmOWX40UzUcvd+9HNJaPU1ZZp\\\\\\\/hlvKnk82fb7iYYzQH66X6oPT9+0eATrBo0gXUSweQTfl2Gwl7FC3s8guXu3A4cF\\\\\\\/SuY\\\\\\\/EsQg8WkyOr1EzefuEluqtYEzVGNfCsaCfe+G4xi6MoqjUsskWctWXxgsA55F6PU49yo9BOHiYi3xIEHWsadhaAKGfYA9TF9BTETWBEnQW\\\\\\\/o2oYNRwpS0bI0K1ExGE2aOh7l6LRTeUcLCbUuqhFFa\\\\\\\/oOMozSJJSKJDveazXxljLEBp2pCndi1SYbP0ei655UFmkAhlRTUFVLGbD4BoVIDzaSZzqdD+uANLCYQapgJDLgx\\\\\\\/iMkP5fbCf+GmlB6Qf+JncfBgAtge+rGxOJJSktILP5OKv4Yt\\\\\\\/VisUCD+VTmPdq7xK+MLQcYrWaYZ7pW3dBWXXwD+q\\\\\\\/Li6OwIC\\\\\\\/KJjWBREiXw61o0QzdsfgKxUrbZ9Xk\\\\\\\/+JKHj8joxIT1hpPWj8F8+o8UcQvVtDWp5zzV22ZO1tYXtA4H959RgsOy7Ht2csQmvoR2Pzsx7xPL3FdRrfT4OBopjdzMWXLpV1Mbqzc2IzR5y\\\\\\\/bTF6uUqSkfaXHxJKYJH6uZn1kW4fYEWx774Kns9DlqsbBwpypLldp9rFWLHLo+aBx4WwYiMC56D4COB+H9qnY7DRHZbAt5Fgl\\\\\\\/+hkJytbAt1ScbkZPKZ27nQXWgRhCnel\\\\\\\/2IIclJHSg\\\\\\\/sl1LAP9G2lkafy7cH2a+pIBf\\\\\\\/9Z4uyONMPqeMHqdBt7c4+vhmirzUsntIhbeFQ3qJLBYsxNodhnVet8E0pjpQ8kp6BBMrrVQBuboNMOjTJM7+RsD1GlXBU5iVmaPPpehTNi6kT6jQ2OBVfQMAdBC5ywyJsu+09Uu7KwqwEeVeyG8vSmPM63jyMWqu9EGGkQjA2SreaoYibEZoIMj0sqRB2hsNk8dpJ+P2\\\\\\\/OTbNfSVBU+BEHcsB5NYoc9nqgCf8FhGUCcUW66YsRjsOBQa0ypLJ9i1sKdsB14UinayvmOthay98dLO7QefSxPDyndDLeI\\\\\\\/FB5CkDyVWbKATlN8RW2fCaQYiU5zEXiNCKPp3Q\\\\\\\/3mf0AWbnZET6DuTi7hBSmWsmkieWZoYQGdsvuPVfbH9UQ6ViclUGYQzhuGQMQQtFBsv6s\\\\\\\/vYBm6U1OY4EUhedrs86HE8\\\u0022,\\\u0022iv\\\u0022:\\\u0022N73FeiEgDJ7M0WzAK6egrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e21af26095caddc80563ac6c14b7d977afb185f02b9901b4e011d93c5375011\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459762 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/9" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HE0DDunJLnfpFCl2iR913JZpng4P7bk8RxcmUMy5VnM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/9" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r8yucjrjiQE3V9ysI5wZl03zIb_CcN96L28R22gHzFg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/1/1" title="5-Hydroxymethylcytosine Is Not Present in Appreciable Quantities in Arabidopsis DNA" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/1/21" title="Relationships of RNA Polymerase II Genetic Interactors to Transcription Start Site Usage Defects and Growth in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459762" id="node459762" data-pisa="ggg;5/1/9" data-pisa-master="ggg;g3.114.015354" data-apath="/ggg/5/1/9.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>In Silico</em> Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of <em>Chlamydia trachomatis</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rita Ferreira</span>, <span class="highwire-citation-author" data-delta="1">Minia Antelo</span>, <span class="highwire-citation-author" data-delta="2">Alexandra Nunes</span>, <span class="highwire-citation-author" data-delta="3">Vítor Borges</span>, <span class="highwire-citation-author" data-delta="4">Vera Damião</span>, <span class="highwire-citation-author" data-delta="5">Maria José Borrego</span> and <span class="highwire-citation-author" data-delta="6">João Paulo Gomes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">9-19;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015354 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459762" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rita Ferreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BFerreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BFerreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Minia Antelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minia%2BAntelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Antelo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinia%2BAntelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexandra Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vítor Borges </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%C3%ADtor%2BBorges%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borges%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV%25C3%25ADtor%2BBorges%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vera Damião </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vera%2BDami%C3%A3o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dami%C3%A3o%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVera%2BDami%25C3%25A3o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Maria José Borrego </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BBorrego%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borrego%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BBorrego%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">João Paulo Gomes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BPaulo%2BGomes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gomes%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BPaulo%2BGomes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.paulo.gomes@insa.min-saude.pt" class="" data-icon-position="" data-hide-link-title="0">j.paulo.gomes@insa.min-saude.pt</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/1/9" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459762" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459762/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/9.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459762" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459762/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/9.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459762" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459762/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/9.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459762" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459762/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/1/9.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459762" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459762/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Microbes possess a multiplicity of virulence factors that confer them the ability to specifically infect distinct biological niches. Contrary to what is known for other bacteria, for the obligate intracellular human pathogen <em>Chlamydia trachomatis</em>, the knowledge of the molecular basis underlying serovars’ tissue specificity is scarce. We examined all ~900 genes to evaluate the association between individual phylogenies and cell-appetence or ecological success of <em>C. trachomatis</em> strains. Only ~1% of the genes presented a tree topology showing the segregation of all three disease groups (ocular, urogenital, and lymphatic) into three well-supported clades. Approximately 28% of the genes, which include the majority of the genes encoding putative type III secretion system effectors and Inc proteins, present a phylogenetic tree where only lymphogranuloma venereum strains form a clade. Similarly, an exclusive phylogenetic segregation of the most prevalent genital serovars was observed for 61 proteins. Curiously, these serovars are phylogenetically cosegregated with the lymphogranuloma venereum serovars for ~20% of the genes. Some clade-specific pseudogenes were identified (novel findings include the conserved hypothetical protein CT037 and the predicted α-hemolysin CT473), suggesting their putative expendability for the infection of particular niches. Approximately 3.5% of the genes revealed a significant overrepresentation of nonsynonymous mutations, and the majority encode proteins that directly interact with the host. Overall, this <em>in silico</em> scrutiny of genes whose phylogeny is congruent with clinical prevalence or tissue specificity of <em>C. trachomatis</em> strains may constitute an important database of putative targets for future functional studies to evaluate their biological role in chlamydial infections.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/chlamydia-trachomatis" class="hw-term hw-article-keyword hw-article-keyword-chlamydia-trachomatis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Chlamydia trachomatis</em></a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/clinical-prevalence" class="hw-term hw-article-keyword hw-article-keyword-clinical-prevalence" rel="nofollow">clinical prevalence</a></li><li class="kwd"><a href="/keyword/tropism" class="hw-term hw-article-keyword hw-article-keyword-tropism" rel="nofollow">tropism</a></li><li class="kwd"><a href="/keyword/loci" class="hw-term hw-article-keyword hw-article-keyword-loci" rel="nofollow">loci</a></li></ul><p id="p-4">The observation that there are pathogenic and nonpathogenic microbes has compelled investigators to search for traits underlying their phenotypic differences. This search for the so called “virulence factors” has greatly contributed to the understanding of pathogenicity and to the elucidation of the genetic mechanisms underlying microbes’ capability to infect different cell types or organs. The notion that microbial pathogenicity relies on the interaction between a pathogen and its host (or a specific tissue), and that a virulence factor is either a microbial product or a strategy capable of causing damage to a susceptible host, can be broadly applied (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casadevall and Pirofski 2009</a>). In this perspective, virulence factors may involve an endless list of products and mechanisms, such as toxins, adhesins, motility structures like flagella and pili, immune evasion determinants, capsules, biofilms, secretion systems, and signal transduction mechanisms (reviewed in <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Casadevall and Pirofski 2009</a>). Usually, microbes carry several of these virulence factors, which work together in the process of host invasion and microbe survival.</p><p id="p-5">Among pathogenic agents, several bacteria present intracellular lifestyles (obligatory or facultative). Their host-cell targets range from epithelial cells to phagocytes, like macrophages and neutrophils (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wilson <em>et al.</em> 2002</a>), which implies that these pathogens have been developing specialized strategies that allow them, for instance, to survive within or avoid the adverse environment of the macrophage phagosome (membrane-bound vacuole) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Garcia-Del Portillo and Finlay 1995</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pizarro-Cerdá <em>et al.</em> 1997</a>). Whereas some bacteria (<em>e.g.</em>, <em>Salmonella</em> spp, <em>Coxiella burnetii</em>, and <em>Cryptococcus neoformans</em>) are able to reside within the lysosomal vacuole, others (<em>e.g.</em>, <em>Chlamydia trachomatis</em> and <em>Mycobacterium</em> spp) need to “remodel” it to allow their survival, whereas others (<em>e.g.</em>, <em>Listeria monocytogenes</em> and <em>Shigella</em> spp) degrade the vacuole membrane to gain access to the host-cell cytosol, where they may complete their developmental cycle (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Pizarro-Cerdá <em>et al.</em> 1997</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ernst <em>et al.</em> 1999</a>). Moreover, some pathogenic bacteria are also able to infect different cell types or organs of a given host. For example, <em>L. monocytogenes</em> can cross the intestinal epithelium, the blood–brain, and fetoplacental barriers (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cossart 2011</a>) and may cause severe septicemia and meningoencephalitis (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allerberger and Wagner 2010</a>), whereas <em>Streptococcus pneumoniae</em> is capable of infecting the lung, the blood, and the nasopharynx (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hava and Camilli 2002</a>).</p><p id="p-6">Another example of bacteria capable of infecting different cell types is <em>C. trachomatis</em>, an obligate intracellular human pathogen that can be classified into 15 main serovars, according to the polymorphism of the gene (<em>ompA</em>) encoding the major outer membrane protein. Serovars A-C cause ocular infections that can progress to trachoma, the leading cause of preventable blindness worldwide (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burton 2007</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wright <em>et al.</em> 2008</a>), whereas serovars D-K cause ano-urogenital infections that can evolve to cervicitis, urethritis, epididymitis (men), or pelvic inflammatory disease (women), the latter of which can lead to significant long term sequelae such as infertility and ectopic pregnancy (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Peipert 2003</a>). Finally, serovars L1-L3 are responsible for an invasive disease, the lymphogranuloma venereum (LGV), through the infection of macrophages and dissemination to regional draining lymph nodes (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Schachter 1978</a>). Despite the huge phenotypic differences among <em>C. trachomatis</em> serovars regarding tissue tropism, virulence and ecological success, little is known about the molecular factors underlying serovars’ biological uniqueness. This is mostly due to the lack of suitable animal models that mirror the human chlamydial infection <em>in vivo</em> and because <em>C. trachomatis</em> has been genetically intractable until very recently (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kari <em>et al.</em> 2011</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mishra <em>et al.</em> 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nguyen and Valdivia 2012</a>). Probably the only unequivocal demonstration of the association of a virulence factor with tropism was provided by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Caldwell <em>et al.</em> (2003)</a>, who showed that an active tryptophan operon (<em>trpRBA</em>) is mandatory for any <em>C. trachomatis</em> strain to infect the genitalia. This observation also was valid for genital strains harboring an “ocular” <em>ompA</em> gene (likely inherited by recombination), excluding the serovar status as a possible tropism determinant. Nevertheless, a revision concerning the genetics beyond tropism was recently published (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nunes <em>et al.</em> 2013</a>).</p><p id="p-7">Recent phylogenetic analysis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>) using the complete genome of several <em>C. trachomatis</em> strains found: <em>i</em>) the segregation of strains by their cell-appetence, suggesting a coevolution with the infected tissue; <em>ii</em>) the separation of the LGV strains before the separation of the ocular and the epithelial-genital strains; <em>iii</em>) that the most prevalent serovars (E and F), which account for ~50% of all chlamydial genital infections among the heterosexual population (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nunes <em>et al.</em> 2010</a>), clearly segregate apart from the remainder epithelial-genital strains; and <em>iv)</em> that the ocular strains probably derived from a nonprevalent genital serovar. On the other hand, the small genome (~1 Mb) of <em>C. trachomatis</em> reveals a high degree of conservation among serovars (&gt;98%), with nearly identical pan- and core-genomes, a high coding density, and no evidence of recent horizontal gene transfer besides allelic recombination, which suggests a likely complete genetic reduction process as a result of a long-term intracellular niche adaptation process (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Horn <em>et al.</em> 2004</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Read <em>et al.</em> 2013</a>). Considering this, one may speculate that the phenotypic disparities (tissue tropism, virulence and ecological success) among strains are encoded in a small number of variable genes along the <em>C. trachomatis</em> genome. Thus, given the recent availability of dozens of <em>C. trachomatis</em> fully sequenced genomes, our main goal was to scrutinize all the ~900 genes at the phylogenetic and evolutionary level in order to better understand the relationship between strains’ genetic diversity and phenotypic disparities. In this regard, after analyzing the global trends of polymorphism, we performed a detailed analysis of each gene tree topology to assess the degree of concordance between strains’ segregation and their clinical outcome and prevalence. This approach intends to identify the genes that phylogenetically contribute for the main branches (LGV, prevalent genital, nonprevalent genital, and ocular serovars) of the species tree (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Alignments generation</h3><p id="p-8">For the polymorphism and evolutionary analyses, different alignment strategies were conducted. First, the whole-genome sequences of the 53 studied <em>C. trachomatis</em> strains were retrieved from the GenBank (<a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/015354SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/TableS1.pdf">Table S1</a>) and aligned using progressiveMauve from Mauve software, version 2.3.1 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Darling <em>et al.</em> 2010</a>). Orthologous genes were identified by Mauve and individual alignments of each one of the 896 genes (considering the total number of annotated genes on the available D/UW-3/CX sequence) were extracted from the whole-genome alignment. These alignments were subsequently uploaded into the Molecular Evolutionary Genetics Analysis software, version 5 (MEGA 5; <a href="http://www.megasoftware.net">http://www.megasoftware.net</a>) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tamura <em>et al.</em> 2011</a>) and visually inspected for further correction (whenever needed) prior to evolutionary and genetic diversity analyses. A core-alignment was also extracted by keeping regions where the 53 genomes aligned over at least 500 bp (corresponding of ~97% of the <em>C. trachomatis</em> chromosome), and aligned segments were concatenated into a single-core genome alignment to be further used in the construction of the species phylogenetic tree. This alignment was then exported and directly uploaded into MEGA 5 for whole-genome analyses purposes.</p></div><div id="sec-3" class="subsection"><h3>Exclusion criteria</h3><p id="p-9">Among all the 53 strains, variability in start codon predictions of homologous genes was removed by trimming each start site prediction to the innermost common start codon. This was not applied when an upstream codon was annotated as a consequence of a mutation in the codon correspondent to the translation initiation codon of the other sequences. We also observed that, for some other genes, there were strains that had more than one coding sequence annotated at the same region. These cases were treated as pseudogenes and the respective strains were removed from the analysis. There were also genes for which a single frameshift yielded a biased polymorphism, and for this reason they were not considered has truly polymorphic. Nevertheless, some of them (CT120, CT160, CT162, CT172, CT172.1, CT358, CT480.1, CT793, and CT852) constitute interesting cases as the frameshift occurred solely for the strains of the same disease group. Moreover, for 22 chromosomal genes, it was not possible to obtain an accurate alignment (<a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/TableS2.xlsx">Table S2</a>) mainly because of accentuated gene size differences, hampering the analyses.</p></div><div id="sec-4" class="subsection"><h3>Polymorphism and evolutionary analyses</h3><p id="p-10">Each alignment (core-genome and individual genes) was analyzed according to previously described methods (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nunes <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2012</a>). Concerning the individual alignments of all homologous genes, we first removed from each analysis the strains’ sequences that were considered as putative pseudogenes or had annotation issues (see the section <em>Exclusion criteria</em>). By using the algorithms available in MEGA 5, we determined the overall mean distances (number of differences and <em>p</em>-distance) and matrices of pairwise comparisons at both nucleotide and amino acid level, along with the respective standard error estimates (bootstrap = 1000). Then, for each gene, the number of synonymous substitutions <em>per</em> synonymous site (dS) as well as the number of nonsynonymous substitutions <em>per</em> nonsynonymous site (dN) were determined by using the Kumar model (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nei and Kumar 2000</a>) and the standard error estimates were obtained by a bootstrap procedure of 1000 replicates. dN/dS ratios were determined and the Z-test of positive selection was applied for the genes revealing dN/dS &gt; 1. The probability of rejecting the null hypothesis of strict-neutrality (dN = dS) in favor of the alternative hypothesis of positive selection (dN &gt; dS) was considered significant when <em>P</em> &lt; 0.05 (bootstrap = 1000) (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Nei and Kumar 2000</a>). We also assessed the existence of correlation between <em>p</em>-distance and dN, dS, or dN/dS by using the Pearson’s Product Moment Correlation coefficient (R), which measures the strength and direction of a linear relationship between two variables (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rodgers and Nicewander 1988</a>).</p><p id="p-11">Phylogenetic trees for both the whole-genome and individual genes sequences were inferred by using the Neighbor-Joining method (bootstrap = 1000) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Felsenstein 1985</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Saitou and Nei 1987</a>). For the nucleotide sequences, the evolutionary distances were computed using the Kimura 2-parameter method (K2P) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kimura 1980</a>), whereas for the amino acid sequences (for individual genes solely), the evolutionary distances were computed based on the number of differences (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Nei and Kumar 2000</a>). A gene was considered to segregate a specific group of strains (ocular, genital and LGV serovars) by taking into account both the tree topology and the number of differences between sequences of different taxa. Additionally, phylogenies were also inspected for the segregation of the strains from the most prevalent genital serovars.</p></div><div id="sec-5" class="subsection"><h3>Characterization of the mosaic structure of the strains from the most prevalent serovars</h3><p id="p-12">We started by comparing the genome sequences of both D(s)/2923 and D/SotonD1 with that of the F/SW5 strain (because this strain was found to be the most closely related to both – see Results section) using the <em>DNA polymorphism</em> tool of the DnaSP software, version 5 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Librado and Rozas 2009</a>), with a window size and step size of 1000 each. Chromosomal regions with high SNP density, which may indicate the occurrence of recombination events, were further analyzed by SimPlot/BootScan (<a href="http://sray.med.som.jhmi.edu/SCRoftware/simplot/">http://sray.med.som.jhmi.edu/SCRoftware/simplot/</a>) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Salminen <em>et al.</em> 1995</a>;<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lole <em>et al.</em> 1999</a>) for a precise determination of potential mosaic structures. These analyses were performed as previously described (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gomes <em>et al.</em> 2007</a>), using a sliding window size of 200 bp moved across the alignment in a step size of 30 bp for estimating pairwise genetic distances with Neighbor-Joining method (Kimura 2-parameter method; Bootstrap = 500; gaps strip off; ts/tv of 2.0). For BootScan analyses, the likelihood that the observed distribution of informative sites (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Robertson <em>et al.</em> 1995</a>) favoring specific phylogenetic groupings might occur randomly was assessed using the maximum χ<sup>2</sup> test. A <em>P</em>-value for any specified breakpoint was determined by the Fisher’s exact test (two-tailed). A Bonferroni multiple correction testing was applied to evaluate the significance of the <em>P</em>-values at 95% confidence.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Polymorphism and molecular evolution analysis</h3><p id="p-13">Overall, we were able to analyze ~97.5% (874/896) of all the <em>C. trachomatis</em> chromosomal genes. The 22 genes excluded from the analysis (see the section <em>Exclusion criteria</em>) comprise five housekeeping genes, the cytotoxin locus, genes encoding 13 hypothetical proteins, two of the phospholipase D endonuclease superfamily gene members (PLDs), and CT081 (<a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/TableS2.xlsx">Table S2</a>).</p><p id="p-14">Besides well-known polymorphic genes (CT870/<em>pmpF</em>, CT872/<em>pmpH</em>, CT681/<em>ompA</em>, CT049-CT051), the polymorphism analyses highlighted CT619 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/TableS2.xlsx">Table S2</a>) [coding for a putative type III secretion system (T3SS) secreted protein with unknown function] that, to our knowledge, had never been considered before as polymorphic.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465529/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465529/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Top five ranking of the most polymorphic <em>C. trachomatis</em> chromosomal genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">To understand the underlying evolutionary pressures that drove amino acid changes of all 874 analyzed chromosomal proteins, we evaluated their molecular evolution by determining the dN/dS values of the respective genes. We verified that 150 genes (~17%) revealed a dN/dS &gt; 1, but only 31 (3.5%) showed a significant Z-test of positive selection (<a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/TableS2.xlsx">Table S2</a>) and were thus considered as putative targets of positive selection. Twenty-three of the latter encode 11 Inclusion Membrane Proteins (Incs), 10 T3SS effectors, and two putative membrane proteins, which are proteins expectedly involved in interactions with the host. We also found three hypothetical proteins encoding genes, one PLD encoding gene, and four housekeeping genes that are likely under positive selection. We have no reasonable explanation for the latter finding, as housekeeping genes are usually highly conserved and expected to be under purifying selection.</p><p id="p-17">Furthermore, we evaluated the correlation between nucleotide polymorphism and evolutionary parameters, such as dN, dS, and dN/dS, for all 874 chromosomal genes. From the inspection of the genomic distribution of <em>p</em>-distance and dN/dS (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>) and by determining the Pearson’s product moment correlation coefficient, we observed no correlation between them (R = 0.02), besides minor coincident peaks. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a> highlights the 25 top ranked loci for both parameters. On the other hand, a strong positive linear correlation between <em>p</em>-distance and both dN (R = 0.92) and dS (R = 0.9) was found (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1D</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/9/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evaluation of the association between polymorphism and dN, dS and dN/dS. (A and B) Distribution of dN/dS and p-distance values, respectively, obtained from the analyses of all the 874 genes from the 53 strains. The horizontal axis represents the C. trachomatis chromosomal positions where genes are placed in their chromosomal order, from the CT001 to the CT875 (genes names and positions according to D/UW-3/CX strain annotation). (C) 25 genes (ordered by their relative chromosomal position) that display the greater values for both analyses, which are representative of the lack of correlation between dN/dS and polymorphism. (D) Scatter plots of p-distance vs. dN, dS, and dN/dS, on a log-scale for clarity. The Pearson&#x2019;s product moment correlation coefficient for p-distance vs. dN, dS, and dN/dS are R = 0.92, R = 0.9, and R = 0.02, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343903224" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evaluation of the association between polymorphism and dN, dS and dN/dS. (A and B) Distribution of dN/dS and &lt;em&gt;p&lt;/em&gt;-distance values, respectively, obtained from the analyses of all the 874 genes from the 53 strains. The horizontal axis represents the &lt;em&gt;C. trachomatis&lt;/em&gt; chromosomal positions where genes are placed in their chromosomal order, from the CT001 to the CT875 (genes names and positions according to D/UW-3/CX strain annotation). (C) 25 genes (ordered by their relative chromosomal position) that display the greater values for both analyses, which are representative of the lack of correlation between dN/dS and polymorphism. (D) Scatter plots of &lt;em&gt;p&lt;/em&gt;-distance &lt;em&gt;vs.&lt;/em&gt; dN, dS, and dN/dS, on a log-scale for clarity. The Pearson&#x2019;s product moment correlation coefficient for &lt;em&gt;p&lt;/em&gt;-distance &lt;em&gt;vs.&lt;/em&gt; dN, dS, and dN/dS are R = 0.92, R = 0.9, and R = 0.02, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/9/F1.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/1/9/F1.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/9/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/9/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465551" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Evaluation of the association between polymorphism and dN, dS and dN/dS. (A and B) Distribution of dN/dS and <em>p</em>-distance values, respectively, obtained from the analyses of all the 874 genes from the 53 strains. The horizontal axis represents the <em>C. trachomatis</em> chromosomal positions where genes are placed in their chromosomal order, from the CT001 to the CT875 (genes names and positions according to D/UW-3/CX strain annotation). (C) 25 genes (ordered by their relative chromosomal position) that display the greater values for both analyses, which are representative of the lack of correlation between dN/dS and polymorphism. (D) Scatter plots of <em>p</em>-distance <em>vs.</em> dN, dS, and dN/dS, on a log-scale for clarity. The Pearson’s product moment correlation coefficient for <em>p</em>-distance <em>vs.</em> dN, dS, and dN/dS are R = 0.92, R = 0.9, and R = 0.02, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Species polymorphism <em>vs.</em> number of taxa</h3><p id="p-19">Considering that the genetic diversity among same-serovar or same-disease group strains was recently pointed out to be higher than expected (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>), we wonder whether both the polymorphism and selective pressure results are impacted by the number of sequences used. Thus, besides using all 53 strains, we also selected a group of 17 strains representative of the major branches of the phylogenetic tree constructed with the whole-genome sequences (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Both groups of strains (17 <em>vs.</em> 53) encompass the same set of 13 <em>C. trachomatis</em> serovars. We then used the 100 most polymorphic genes (as they provide the vast majority of informative sites) and compared the distribution of polymorphism and dN/dS obtained from the analysis of the two groups (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The <em>P</em>-values (paired two-tailed <em>t</em>-test) calculated for the <em>p</em>-distance and the dN/dS results were 0.91 and 0.13, respectively, which indicates that these parameters do not depend on the number of same-serovar sequences that are used. Although the validity of the traditional CT681/<em>ompA</em> typing has been strongly questioned (as its tree does not segregate strains by tissue tropism properties and disease outcomes) (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>), it is worth noting that a small group of strains encompassing the majority of the <em>C. trachomatis</em> serovars represent the main genetic variability of this bacterium.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/9/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic reconstruction of C. trachomatis species. The tree was constructed using the whole genome of 53 strains encompassing the majority of the CT681/ompA serovars. The asterisks indicate the 17 strains representative of the major tree branches (in red) that were used to evaluate the relation between species polymorphism and the number of taxa (see the section Results for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343903224" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic reconstruction of &lt;em&gt;C. trachomatis&lt;/em&gt; species. The tree was constructed using the whole genome of 53 strains encompassing the majority of the CT681/&lt;em&gt;ompA&lt;/em&gt; serovars. The asterisks indicate the 17 strains representative of the major tree branches (in red) that were used to evaluate the relation between species polymorphism and the number of taxa (see the section &lt;em&gt;Results&lt;/em&gt; for details).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/9/F2.medium.gif" width="327" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/1/9/F2.medium.gif" width="327" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/9/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/9/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465577" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Phylogenetic reconstruction of <em>C. trachomatis</em> species. The tree was constructed using the whole genome of 53 strains encompassing the majority of the CT681/<em>ompA</em> serovars. The asterisks indicate the 17 strains representative of the major tree branches (in red) that were used to evaluate the relation between species polymorphism and the number of taxa (see the section <em>Results</em> for details).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/9/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differences obtained during the analyses using 53 and 17 strains. The graphs show the differences obtained between the results of the p-distance (A) and the dN/dS (B) analyses of all the 53 and the set of 17 strains (representative of the majority of the tree branches). Each black dot represents one of the 100 polymorphic genes selected for these comparisons. P-values were calculated through the paired two-tailed t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343903224" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Differences obtained during the analyses using 53 and 17 strains. The graphs show the differences obtained between the results of the &lt;em&gt;p&lt;/em&gt;-distance (A) and the dN/dS (B) analyses of all the 53 and the set of 17 strains (representative of the majority of the tree branches). Each black dot represents one of the 100 polymorphic genes selected for these comparisons. &lt;em&gt;P&lt;/em&gt;-values were calculated through the paired two-tailed &lt;em&gt;t&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/9/F3.medium.gif" width="440" height="178"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/1/9/F3.medium.gif" width="440" height="178"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/9/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/9/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465568" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Differences obtained during the analyses using 53 and 17 strains. The graphs show the differences obtained between the results of the <em>p</em>-distance (A) and the dN/dS (B) analyses of all the 53 and the set of 17 strains (representative of the majority of the tree branches). Each black dot represents one of the 100 polymorphic genes selected for these comparisons. <em>P</em>-values were calculated through the paired two-tailed <em>t</em>-test.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Gene-based phylogenetic analysis</h3><p id="p-22">To evaluate the concordance between strains’ segregation and their clinical outcome and prevalence, we first performed a detailed analysis of the recombination phenomena involving the two D strains (D(s)/2923 and D/SotonD1) that phylogenetically cluster with the most prevalent serovars (E and F) and apart from the other D strains (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jeffrey <em>et al.</em> 2010</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>), in order to define their true genomic backbone. We verified that those D strains differ from the same serovar prototype strain (D/UW-3/CX) by ~5500 nucleotides, but differ from a serovar F strain (F/SW5) by only ~300 nucleotides, with ~50% of these mutations concentrated at the CT681/<em>ompA</em> region (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). SimPlot and BootScan analyses identified the exact location of the two breakpoints underlying the recombination event (identical for both strains) (<a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/FigureS1.pdf">Figure S1</a>). One breakpoint is located at the beginning of CT680/<em>rpsB</em> (<em>P</em> = 9.28 × 10<sup>−44</sup>) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>), whereas the other is located at the beginning of CT681/<em>ompA</em> (<em>P</em> = 6.65 × 10<sup>−19</sup>) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). These results clearly indicate that both recombinant D strains have a genome backbone of a serovar F strain, whereas solely the region spanning between the two recombination breakpoints was inherited from a serovar D strain. Therefore, from now on these two D serovar strains will be included into the cluster of the most clinically prevalent serovars.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/9/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombination analyses of the D(s)/2923 and D/SontonD1 strains. (A) Number of nucleotide differences (vertical axis) that exist between the genomic sequence of D(s)/2923 or D/SotonD1 and F/SW5. This polymorphism assessment was performed by using the DnaSP software, v5, with a window size and a step size of 1000 base pairs each. The smaller graph represents an enlarged view of the detected highly polymorphic region. (B) (first crossover) and (C) (second crossover) show the genes in each analyzed region (1) and also the results of the SimPlot (2), the BootScan (3), and the phylogenetic (4) analyses. Recombination breakpoints were individually analyzed because they were better mapped when a different outgroup strain was used for each one, i.e., the L3/404-LN for the first (B) and the C/TW-3 for the second (C) breakpoint. SimPlot graphs (2) show the level of similarity between the recombinant sequences and the respective parental strains (the number of informative sites supporting this relatedness are colored according to the graph legend box), whereas the BootScan graphs (3) show the phylogenetic relatedness (% of permuted trees) between those same sequences. Both analyses were obtained with a sliding window size of 200 bp and a step size of 30 bp. The sequence of the recombinant D strains was used as query. The vertical dashed black lines indicate the location of the estimated crossovers, shown in detail in Figure S1. Seventy-one informative sites support the similarity between the recombinant strain and F/SW5, whereas 76 support its similarity with D/UW-3/CX (P = 9.28 &#xD7; 10&#x2212;44). Forty-four informative sites support the similarity between the recombinant strain and D/UW-3/CX, whereas 25 support its similarity with F/SW5 (P = 6.65 &#xD7; 10&#x2212;19). In these defined regions there are no informative sites supporting the alternative hypotheses. The phylogenetic trees (4) were constructed with the nucleotide sequences adjacent to each estimated breakpoint region (NJ method; Kimura 2-parameter method; bootstrap = 1000) and support the recombination event." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343903224" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Recombination analyses of the D(s)/2923 and D/SontonD1 strains. (A) Number of nucleotide differences (vertical axis) that exist between the genomic sequence of D(s)/2923 or D/SotonD1 and F/SW5. This polymorphism assessment was performed by using the DnaSP software, v5, with a window size and a step size of 1000 base pairs each. The smaller graph represents an enlarged view of the detected highly polymorphic region. (B) (first crossover) and (C) (second crossover) show the genes in each analyzed region (1) and also the results of the SimPlot (2), the BootScan (3), and the phylogenetic (4) analyses. Recombination breakpoints were individually analyzed because they were better mapped when a different outgroup strain was used for each one, &lt;em&gt;i.e.&lt;/em&gt;, the L3/404-LN for the first (B) and the C/TW-3 for the second (C) breakpoint. SimPlot graphs (2) show the level of similarity between the recombinant sequences and the respective parental strains (the number of informative sites supporting this relatedness are colored according to the graph legend box), whereas the BootScan graphs (3) show the phylogenetic relatedness (% of permuted trees) between those same sequences. Both analyses were obtained with a sliding window size of 200 bp and a step size of 30 bp. The sequence of the recombinant D strains was used as query. The vertical dashed black lines indicate the location of the estimated crossovers, shown in detail in Figure S1. Seventy-one informative sites support the similarity between the recombinant strain and F/SW5, whereas 76 support its similarity with D/UW-3/CX (&lt;em&gt;P&lt;/em&gt; = 9.28 &#xD7; 10&lt;sup&gt;&#x2212;44&lt;/sup&gt;). Forty-four informative sites support the similarity between the recombinant strain and D/UW-3/CX, whereas 25 support its similarity with F/SW5 (&lt;em&gt;P&lt;/em&gt; = 6.65 &#xD7; 10&lt;sup&gt;&#x2212;19&lt;/sup&gt;). In these defined regions there are no informative sites supporting the alternative hypotheses. The phylogenetic trees (4) were constructed with the nucleotide sequences adjacent to each estimated breakpoint region (NJ method; Kimura 2-parameter method; bootstrap = 1000) and support the recombination event.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/9/F4.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/1/9/F4.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/9/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/9/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465555" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-23" class="first-child">Recombination analyses of the D(s)/2923 and D/SontonD1 strains. (A) Number of nucleotide differences (vertical axis) that exist between the genomic sequence of D(s)/2923 or D/SotonD1 and F/SW5. This polymorphism assessment was performed by using the DnaSP software, v5, with a window size and a step size of 1000 base pairs each. The smaller graph represents an enlarged view of the detected highly polymorphic region. (B) (first crossover) and (C) (second crossover) show the genes in each analyzed region (1) and also the results of the SimPlot (2), the BootScan (3), and the phylogenetic (4) analyses. Recombination breakpoints were individually analyzed because they were better mapped when a different outgroup strain was used for each one, <em>i.e.</em>, the L3/404-LN for the first (B) and the C/TW-3 for the second (C) breakpoint. SimPlot graphs (2) show the level of similarity between the recombinant sequences and the respective parental strains (the number of informative sites supporting this relatedness are colored according to the graph legend box), whereas the BootScan graphs (3) show the phylogenetic relatedness (% of permuted trees) between those same sequences. Both analyses were obtained with a sliding window size of 200 bp and a step size of 30 bp. The sequence of the recombinant D strains was used as query. The vertical dashed black lines indicate the location of the estimated crossovers, shown in detail in <a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/FigureS1.pdf">Figure S1</a>. Seventy-one informative sites support the similarity between the recombinant strain and F/SW5, whereas 76 support its similarity with D/UW-3/CX (<em>P</em> = 9.28 × 10<sup>−44</sup>). Forty-four informative sites support the similarity between the recombinant strain and D/UW-3/CX, whereas 25 support its similarity with F/SW5 (<em>P</em> = 6.65 × 10<sup>−19</sup>). In these defined regions there are no informative sites supporting the alternative hypotheses. The phylogenetic trees (4) were constructed with the nucleotide sequences adjacent to each estimated breakpoint region (NJ method; Kimura 2-parameter method; bootstrap = 1000) and support the recombination event.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To identify loci that phylogenetically contribute for the main branches of the species tree (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>), we performed a detailed analysis of each gene phylogenetic tree. For clarification purposes, a gene/protein was considered to segregate a group of strains sharing a specific phenotype (ocular, prevalent genital, non-prevalent genital and LGV serovars) when the genetic differences among them are lower than the differences to any other strain. Overall, we found that 136, 14, 431, and 695 genes phylogenetically segregate the ocular, genital, prevalent genital and LGV groups, respectively (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, and <a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/TableS2.xlsx">Table S2</a>). The low number of genes segregating the group of genital serovars reflects the high heterogeneity within this group as a direct consequence of the recombination background affecting mostly these strains (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>) and the existence of distinct polymorphism signatures. An example of the latter stands for the F(s)/70 strain, which was isolated from the cervix and frequently showed a rather unusual polymorphism pattern that did not resemble any of the other 52 strains. Therefore, only 11 (1.3%) of nucleotide trees and 12 (1.4%) of protein trees were found to segregate strains by full-tropism (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), where ocular, LGV and all genital (prevalent and nonprevalent) serovar strains are segregated into three main clusters. <em>In silico</em> studies have already implicated some of these genes in the different cell-appetence of the strains, namely CT456/<em>tarp</em>, CT870/<em>pmpF</em>, CT872/<em>pmpH</em>, CT115/<em>incD</em>, CT116/<em>incE</em>, two PLD (CT156 and CT157), and one MACPF domain family protein (CT153) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gomes <em>et al.</em> 2006</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Thomson <em>et al.</em> 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Borges <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lutter <em>et al.</em> 2012</a>). The remainders include three housekeeping genes (CT106/<em>yceC</em>, CT110/<em>groEL1</em>, and CT703/<em>engA</em>), and genes encoding one T3SS effector (CT161) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">da Cunha <em>et al.</em> 2014</a>) and one putative inclusion membrane protein (Inc) (CT383) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dehoux <em>et al.</em> 2011</a>) (<a href="http://www.g3journal.org/content/suppl/2014/11/05/g3.114.015354.DC1/TableS2.xlsx">Table S2</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/9/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes that segregate strains according to their biological characteristics. The outer circle in both panels represents the genome of C. trachomatis D/UW-3/CX strain, where each bar represents a gene at its respective genomic position (light gray bars, forward strand; dark gray bars, reverse strand). (A) The tracks&#x2019; color scheme represent genes whose phylogeny segregates at least a group of strains according to their biological characteristics, i.e., each color illustrates a particular segregation (that may not be exclusive): full-tropism (purple), LGV strains (orange), strains from prevalent genital serovars (green), cosegregation of LGV and prevalent genital serovar strains (blue), genital strains (prevalent and nonprevalent serovars) (black), and ocular strains (red). (B) The tracks&#x2019; color scheme was maintained for the different groups of strains and represent genes that exclusively segregate a unique group of strains. For both panels, the outer and inner tracks of each color correspond to nucleotide and amino acid results, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343903224" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genes that segregate strains according to their biological characteristics. The outer circle in both panels represents the genome of &lt;em&gt;C. trachomatis&lt;/em&gt; D/UW-3/CX strain, where each bar represents a gene at its respective genomic position (light gray bars, forward strand; dark gray bars, reverse strand). (A) The tracks&#x2019; color scheme represent genes whose phylogeny segregates at least a group of strains according to their biological characteristics, &lt;em&gt;i.e.&lt;/em&gt;, each color illustrates a particular segregation (that may not be exclusive): full-tropism (purple), LGV strains (orange), strains from prevalent genital serovars (green), cosegregation of LGV and prevalent genital serovar strains (blue), genital strains (prevalent and nonprevalent serovars) (black), and ocular strains (red). (B) The tracks&#x2019; color scheme was maintained for the different groups of strains and represent genes that exclusively segregate a unique group of strains. For both panels, the outer and inner tracks of each color correspond to nucleotide and amino acid results, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/9/F5.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/1/9/F5.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/9/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/9/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465537" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">Genes that segregate strains according to their biological characteristics. The outer circle in both panels represents the genome of <em>C. trachomatis</em> D/UW-3/CX strain, where each bar represents a gene at its respective genomic position (light gray bars, forward strand; dark gray bars, reverse strand). (A) The tracks’ color scheme represent genes whose phylogeny segregates at least a group of strains according to their biological characteristics, <em>i.e.</em>, each color illustrates a particular segregation (that may not be exclusive): full-tropism (purple), LGV strains (orange), strains from prevalent genital serovars (green), cosegregation of LGV and prevalent genital serovar strains (blue), genital strains (prevalent and nonprevalent serovars) (black), and ocular strains (red). (B) The tracks’ color scheme was maintained for the different groups of strains and represent genes that exclusively segregate a unique group of strains. For both panels, the outer and inner tracks of each color correspond to nucleotide and amino acid results, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465546/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465546/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of genes/proteins that segregate <em>C. trachomatis</em> strains according to distinct phenotypes</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">We also detected events of exclusive phylogenetic segregation, <em>i.e.</em>, the clustering of a particular group of strains sharing the same phenotype, whereas the remainder strains (regardless of their phenotype) are mixed together. For instance, the group of strains from the most prevalent genital serovars (E, F, and recombinant D strains) are exclusively segregated by 61 proteins, which may contain molecular features that contribute for their higher ecological success. We also observed that the most prevalent and the LGV serovars share hundreds of mutations, particularly in 173 genes (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) revealing a major tree branch where these two groups co-segregate apart from the remaining strains. Concerning the LGV group, ∼28% of all chromosomal genes exclusively segregate these strains (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>), conferring this group a unique genetic make-up within the species diversity.</p><p id="p-33">Also, based on either the presence of nonsense mutations or the considerable differences in gene size, we scrutinized the genome for the existence of genes that are putative pseudogenes exclusively for a specific disease group (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). This set includes: <em>i</em>) CT058 [a putative Inc (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2012</a>)], CT105 [a T3SS effector possibly involved in the cell-appetence of the genital strains (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Borges <em>et al.</em> 2012</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">da Cunha <em>et al.</em> 2014</a>)], <em>trpRBA</em> operon (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Caldwell <em>et al.</em> 2003</a>), and CT374/<em>aaxC</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Giles <em>et al.</em> 2009</a>), which are pseudogenes for most ocular strains; <em>ii</em>) CT101 [Inc (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2012</a>)] is a pseudogene for the majority of the genital strains; <em>iii</em>) CT473 (predicted α-hemolysin) is a pseudogene for the prevalent genital serovar strains; <em>iv</em>) CT373/<em>aaxB</em> (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Giles <em>et al.</em> 2009</a>) and CT300 [putative Inc (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Dehoux <em>et al.</em> 2011</a>)] are pseudogenes for LGV strains (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2012</a>) (for CT300, this occurs only if one considers the same start codon as that annotated for ocular and genital strains); and <em>v</em>) CT037 (conserved hypothetical protein) is a pseudogene for both prevalent genital and LGV serovars strains. This scenario suggests that these genes may be expendable for the <em>C. trachomatis</em> infection of specific biological niches.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465565/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465565/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><em>C. trachomatis</em> known and putative pseudogenes for a particular disease group and genes that present differences in gene length among strains from different disease groups</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-35">Phylogenetic studies in <em>C. trachomatis</em> have been extensively performed on dozens of genes. Given the recent availability of more than 50 genomes, we sought to perform comparative genomics to examine all the ~900 <em>C. trachomatis</em> genes. We aimed to evaluate the degree of concordance between strains’ segregation and their clinical outcome and prevalence. In fact, the molecular basis underlying tissue specificity in <em>C. trachomatis</em> remains to be elucidated, although it is believed that it may rely on SNPs or small <em>indel</em> events in specific genes (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Nunes <em>et al.</em> 2013</a>), given the tremendous genome similarity (&gt; 98%) among sequenced strains. It is known that there are biases associated with phylogenetic-based inferences (“phylogenetic dependence”), such as the weight of neutral mutations in the tree topology. Nevertheless, there are well-built examples in the literature where tree topology of <em>C. trachomatis</em> genes seems to be associated with niche specificity. This is the case of Tarp (Translocated actin-recruiting protein), for which distinct functional domains that are variable in number across serovars from different disease groups likely lead to differences in the host-cell actin-driven uptake of <em>Chlamydia</em> and to differential activation of diverse signaling pathways (like the Rac/WAVE2/Arp2/3 cascade and the humoral and cellular immune signaling pathways) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clifton <em>et al.</em> 2004</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mehlitz <em>et al.</em> 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carabeo 2011</a>). Another relevant example is provided by most Incs, which may be associated with infection of mononuclear phagocytes due to the existence of specific mutational patterns leading to the phylogenetic segregation of LGV strains and to the higher expression of some Incs in these strains (<a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lutter <em>et al.</em> 2012</a>). In this regard, although our phylogenetic approach certainly carries associated biases, the identification of genes that phylogenetically contribute for the main branches (LGV, prevalent genital, non-prevalent genital and ocular serovars) of the species tree may be highly relevant for future functional studies.</p><p id="p-36">Overall, only ~1.4% (12/874) of the proteins was found to present a plain segregation of strains according to their tissue tropism (ocular conjunctiva, genital epithelium, and lymph nodes). This low number is probably due to the existence of intra- and intergenic recombination events that take place during mixed infections [believed to occur at a frequency of approximately 1% (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Clarke 2011</a>)], essentially involving the genital strains (<a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>). Although <em>C. trachomatis</em> is known to have a low population-level recombination rate based on the frequency and relative weight of recombination and mutation events (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Joseph <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ferreira <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Joseph <em>et al.</em> 2012</a>), recombination has been detected, even among different disease-causing strains, and hotspots have been identified (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gomes <em>et al.</em> 2007</a>; <a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>). The biological role of some of these proteins has already been assessed (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nelson <em>et al.</em> 2006</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Taylor <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Derré <em>et al.</em> 2011</a>), but with exception of the above cited Tarp, only a single serovar was tested, hampering any implication in tropism. On the other hand, it is possible that each of the corresponding genes is simply evolving more quickly than the genome average (quite likely due to host pressures).</p><p id="p-37">A radically different scenario is found for the lymph nodes niche, as the majority of the genes (~80%) segregate the LGV strains and 28% (245/874) segregate them in an exclusive manner (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). This corroborates the early divergence of these strains (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stephens <em>et al.</em> 2009</a>) and/or their fastest evolutionary nature. The latter hypothesis may rely on the fact that the LGV strains must be capable of establishing a wider set of molecular interactions and be subject to additional selective pressures, given their ability of infecting two distinct cell-types (epithelial and mononuclear phagocytes). It is worth noting that the majority of the genes encoding T3SS effectors and Incs (known and putative) segregate the LGV strains. One interesting example is CT144 that codes for a putative substrate of the T3SS (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">da Cunha <em>et al.</em> 2014</a>) and is likely involved in the “men who have sex with men” epidemiological sexual network (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Christerson <em>et al.</em> 2012</a>), for which most of LGV-specific polymorphisms are concentrated in ~150 bp on the first half of the gene (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Nunes <em>et al.</em> 2008</a>), highlighting this specific region as the one hypothetically involved in the interaction with the host cell. Another example comes from the well-studied T3SS effector Tarp for which the enhanced phosphorylation found in LGV strains was shown to additionally promote high affinity interactions with proteins associated with the immune signaling pathways (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Mehlitz <em>et al.</em> 2010</a>), likely explaining the capacity of these strains to cross the mucosa epithelium and to infect mononuclear phagocytes.</p><p id="p-38">We also observed that half of the <em>C. trachomatis</em> genes segregate strains of the most prevalent genital serovars, where 61 encode proteins displaying a mutational pattern that is exclusive of these strains. The majority of these genes (33/61) encode proteins that mediate basic cellular functions, like some redox reactions (CT078/<em>folD</em>, CT278/<em>nqrB</em>, CT539/<em>trxA</em>, and CT745/<em>hemG</em>), structural ribosomal proteins (CT125/<em>rplM</em>, CT506/<em>rplQ</em>, CT511, CT523/<em>rplV</em>, CT525/<em>rplB</em>, and CT787/<em>rpsN</em>) and proteins intervenient in the translation process (CT193/<em>tgt</em>, CT437/<em>fusA</em>, and CT851/<em>map</em>). However, given the high representation of these functional categories in <em>C. trachomatis</em> genome, we can hardly assume that specific metabolic functions underlie the higher clinical prevalence of strains from serovars E and F. Nevertheless, it seems clear that these serovars share a singular genomic makeup. In fact, two recombinant strains classified as serovar D that cluster in the same branch as E and F are actually F-like strains, and so, the branch of the most ecological succeeded serovars involve exclusively taxa with E or F backbone.</p><p id="p-39">Curiously, we also found that 173 genes (19.8%) cosegregated the strains from the most prevalent genital serovars and the LGV strains. Some relevant examples refer to CT651, a possible virulence factor since it is under the regulation of <em>C. trachomatis</em> plasmid (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Song <em>et al.</em> 2013</a>), and CT338 and CT619, two T3SS substrates (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Muschiol <em>et al.</em> 2011</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">da Cunha <em>et al.</em> 2014</a>). Possible explanations for the existence of hundreds of shared polymorphisms between these two groups include: <em>i</em>) incomplete lineage sorting, where several unresolved polymorphisms would have been accumulated in the common ancestral before the clades’ separation of the current species tree (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Galtier and Daubin 2008</a>); <em>ii</em>) recombination, although it cannot fully explain this scenario as the genetic exchange between these two groups has been recently demonstrated to be restricted to limited genomic regions (<a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">Harris <em>et al.</em> 2012</a>); and <em>iii</em>) short coevolutionary process between E/F and LGV strains before the separation of the latter. One may speculate that some of the shared polymorphisms could endow “invasive” properties to the most prevalent serovars strains. If that would be the case, it would mirror for instance the infection scenario of <em>L. monocytogenes</em>, which is capable of surviving within macrophages and also replicating in a variety of nonphagocytic cells (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pizarro-Cerdá <em>et al.</em> 2012</a>). Therefore, one could hypothetically identify E or F strains during recent LGV outbreaks in Europe and USA. However, full-genome sequencing was not performed for all strains identified in those outbreaks and, to our knowledge, no E and F strains were identified so far. Thus, no immediate assumption can be made concerning specific phenotypes conferred by the related mutational pattern in those 173 genes between E/F and invasive strains.</p><p id="p-40">We also identified several putative pseudogenes occurring in different strains (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The most interesting cases were the genes that were truncated only for strains of the same disease group, as it may be an indication of their expendability for the infection of a specific niche. We highlight the CT473, a lipid droplet-associated protein (Lda3) found to be translocated into the host cell cytoplasm and capture lipid droplets (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kumar <em>et al.</em> 2006</a>), which is likely being lost on the course of the evolutionary process of the strains from prevalent genital serovars, and the CT037 (conserved hypothetical protein), which is a pseudogene in both the prevalent genital and LGV serovar strains. Although we have no clues about the importance of maintaining a functional protein in the clades where these genes are not pseudogenes, it was already demonstrated for example that a functional <em>trpRBA</em> operon is mandatory for any strain to infect the genitalia (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Caldwell <em>et al.</em> 2003</a>). Also, we have previously shown that the positively selected gene CT105 (a pseudogene for ocular strains) has a significant overrepresentation of nonsynonymous mutations when comparing sequences between urogenital and LGV strains (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Borges <em>et al.</em> 2012</a>), indicating that it has been evolutionarily diverging toward niche-specific adaptation. The identification of niche-specific pseudogenes may be indicative that further genome reduction may still be ongoing in <em>C. trachomatis</em>, leading to the future disappearance of those sequences from the chromosome. We also identified genes with differences in sequence length according to strains phenotype. For instance, both CT833 (translation initiation factor) and CT852 (integral membrane component) have longer sequences for all LGV strains, making them interesting targets for future evaluation, as the gene length may have a differential impact on the protein functionality. Additional analyses are now being performed at our lab in order to better characterize this complete set of genes (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-41">The analysis of polymorphism and dN/dS revealed no correlation between the two parameters, indicating that positive selection is highly targeted on specific genes or gene regions, or acts on strains with specific cell-appetence (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Borges <em>et al.</em> 2012</a>). Although our analysis was focused on whole genes (leading to an underestimation of positive selection), it is notable that the genes with significant dN/dS &gt; 1 were mainly <em>incs</em> and T3SS effectors encoding genes, whereas the most polymorphic ones code essentially for membrane and hypothetical proteins. This seems to corroborate the assumption that proteins involved in strict pathogen-host interactions during the infection process are more prone to fix non-synonymous mutations, as previously reported in smaller scale studies (<a id="xref-ref-2-10" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2012</a>; <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Borges <em>et al.</em> 2012</a>). On the other hand, polymorphism seems to be more pronounced in genes of other functional categories and may be due to discrete genetic drift, as most of the polymorphism is given by dS.</p><p id="p-42">Finally, despite the controversial use of the traditional <em>ompA</em>-based typing method, it is worth noting that the main genetic variability within the <em>C. trachomatis</em> species is given by the different serovars, where additional strains from the same serovar contribute with few novel polymorphisms (driven either by drift or positive selection) that may impact the individual gene phylogenies (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-43">As concluding remarks, our approach allowed the identification of genes that phylogenetically segregate strains according to specific phenotypes, namely the infection of the ocular tissue, the genitalia, the lymph nodes, as well as their clinical prevalence. It will certainly constitute an important database for prioritizing the targets for functional studies that are mandatory to clarify both their biological role and their involvement in tissue tropism, virulence and ecological success.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">This work was supported by a grant, ERA-PTG/0004/2010, from Fundação para a Ciência e a Tecnologia (FCT) (to J.P.G.), in the frame of ERA-NET PathoGenoMics. A.N. is recipient of a FCT post-doctoral fellowship (SFRH/BPD/75295/2010), V.B. and R.F. are recipients of Ph.D. fellowships (SFRH/BD/68527/2010 and SFRH/BD/68532/2010, respectively) from FCT, and V.D. is a recipient of fellowship on behalf of the grant ERA-PTG/0004/2010.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-10"><p id="p-45">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015354/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015354/-/DC1</a></p></li><li class="fn" id="fn-11"><p id="p-46"><em>Communicating editor: I. M. Hall</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-08-25"><span class="received-label">Received </span>August 25, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-11-02"><span class="accepted-label">Accepted </span>November 2, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Ferreira <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.1" data-doi="10.1111/j.1469-0691.2009.03109.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allerberger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Listeriosis: a resurgent foodborne infection.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Infect.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bmicrobiology%2Band%2Binfection%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BEuropean%2BSociety%2Bof%2BClinical%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.stitle%253DClin%2BMicrobiol%2BInfect%26rft.aulast%253DAllerberger%26rft.auinit1%253DF.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D23%26rft.atitle%253DListeriosis%253A%2Ba%2Bresurgent%2Bfoodborne%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-0691.2009.03109.x%26rft_id%253Dinfo%253Apmid%252F20002687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-0691.2009.03109.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20002687&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.2" data-doi="10.1128/JB.01428-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borges</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Polymorphisms in Inc proteins and differential expression of <em>inc</em> genes among <em>Chlamydia trachomatis</em> strains correlate with invasiveness and tropism of lymphogranuloma venereum isolates.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">6574</span>–<span class="cit-lpage">6585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01428-12%26rft_id%253Dinfo%253Apmid%252F23042990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5NC8yMy82NTc0IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.3" data-doi="10.1128/JB.01291-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borges</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Directional evolution of <em>Chlamydia trachomatis</em> towards niche-specific adaptation.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">6143</span>–<span class="cit-lpage">6153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01291-12%26rft_id%253Dinfo%253Apmid%252F22961851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5NC8yMi82MTQzIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.4" data-doi="10.1093/bmb/ldm034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Trachoma: an overview.</span> <abbr class="cit-jnl-abbrev">Br. Med. Bull.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBr.%2BMed.%2BBull.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbmb%252Fldm034%26rft_id%253Dinfo%253Apmid%252F18175788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYm1idWxsIjtzOjU6InJlc2lkIjtzOjc6Ijg0LzEvOTkiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.5" data-doi="10.1172/JCI200317993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polymorphisms in <em>Chlamydia trachomatis</em> tryptophan synthase genes differentiate between genital and ocular isolates.</span> <abbr class="cit-jnl-abbrev">J. Clin. Invest.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1757</span>–<span class="cit-lpage">1769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.aulast%253DCaldwell%26rft.auinit1%253DH.%2BD.%26rft.volume%253D111%26rft.issue%253D11%26rft.spage%253D1757%26rft.epage%253D1769%26rft.atitle%253DPolymorphisms%2Bin%2BChlamydia%2Btrachomatis%2Btryptophan%2Bsynthase%2Bgenes%2Bdifferentiate%2Bbetween%2Bgenital%2Band%2Bocular%2Bisolates%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI200317993%26rft_id%253Dinfo%253Apmid%252F12782678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI200317993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12782678&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183313400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.6" data-doi="10.1111/j.1462-5822.2011.01651.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carabeo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bacterial subversion of host actin dynamics at the plasma membrane.</span> <abbr class="cit-jnl-abbrev">Cell. Microbiol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1460</span>–<span class="cit-lpage">1469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Bmicrobiology%26rft.stitle%253DCell%2BMicrobiol%26rft.aulast%253DCarabeo%26rft.auinit1%253DR.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D1460%26rft.epage%253D1469%26rft.atitle%253DBacterial%2Bsubversion%2Bof%2Bhost%2Bactin%2Bdynamics%2Bat%2Bthe%2Bplasma%2Bmembrane.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-5822.2011.01651.x%26rft_id%253Dinfo%253Apmid%252F21790944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-5822.2011.01651.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21790944&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.7" data-doi="10.2166/wh.2009.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirofski</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Virulence factos and their mechanism of action: the view from a damage-response framework.</span> <abbr class="cit-jnl-abbrev">J. Water Health</abbr> <span class="cit-vol">7</span>(<span class="cit-supplement">Suppl 1</span>): <span class="cit-fpage">S2</span>–<span class="cit-lpage">S18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BWater%2BHealth%26rft_id%253Dinfo%253Adoi%252F10.2166%252Fwh.2009.036%26rft_id%253Dinfo%253Apmid%252F19717929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoicHBpd2Fqd2giO3M6NToicmVzaWQiO3M6NzoiNy9TMS9TMiI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzUvMS85LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.8" data-doi="10.1128/JCM.01713-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christerson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bom</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruisten</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yass</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Chlamydia trachomatis</em> strains show specific clustering for men who have sex with men compared to heterosexual populations in Sweden, the Netherlands, and the United States.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">3548</span>–<span class="cit-lpage">3555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.01713-12%26rft_id%253Dinfo%253Apmid%252F22915612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI1MC8xMS8zNTQ4IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.9" data-doi="10.1111/j.1749-6632.2011.06194.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">I. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of <em>Chlamydia trachomatis</em>.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1230</span>: <span class="cit-fpage">E11</span>–<span class="cit-lpage">E18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DClarke%26rft.auinit1%253DI.%2BN.%26rft.volume%253D1230%26rft.spage%253DE11%26rft.epage%253DE18%26rft.atitle%253DEvolution%2Bof%2BChlamydia%2Btrachomatis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2011.06194.x%26rft_id%253Dinfo%253Apmid%252F22239534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2011.06194.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22239534&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294181100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.10" data-doi="10.1073/pnas.0402829101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clifton</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieshaber</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dooley</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A chlamydial type III translocated protein is tyrosine-phosphorylated at the site of entry and associated with recruitment of actin.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">10166</span>–<span class="cit-lpage">10171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0402829101%26rft_id%253Dinfo%253Apmid%252F15199184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzI3LzEwMTY2IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.11" data-doi="10.1073/pnas.1112371108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cossart</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Illuminating the landscape of host-pathogen interactions with the bacterium <em>Listeria monocytogenes</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">19484</span>–<span class="cit-lpage">19491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1112371108%26rft_id%253Dinfo%253Apmid%252F22114192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQ5LzE5NDg0IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.12" data-doi="10.1186/1471-2180-14-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Cunha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pais</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borges</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of type III secretion substracts of <em>Chlamydia trachomatis</em> using <em>Yersinia enterocolitica</em> as a heterologous system.</span> <abbr class="cit-jnl-abbrev">BMC Microbiol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiol.%26rft.volume%253D14%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2180-14-40%26rft_id%253Dinfo%253Apmid%252F24533538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2180-14-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24533538&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.13" data-doi="10.1371/journal.pone.0011147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">progressiveMauve: Multiple genome alignment with gene gain, loss, and rearrangement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDarling%26rft.auinit1%253DA.%2BE.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De11147%26rft.epage%253De11147%26rft.atitle%253DprogressiveMauve%253A%2Bmultiple%2Bgenome%2Balignment%2Bwith%2Bgene%2Bgain%252C%2Bloss%2Band%2Brearrangement.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011147%26rft_id%253Dinfo%253Apmid%252F20593022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20593022&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.14" data-doi="10.1186/1471-2164-12-109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dehoux</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flores</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dauga</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subtil</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multi-genome identification and characterization of chlamydiae-specific type III secretion substrates: the Inc proteins.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDehoux%26rft.auinit1%253DP.%26rft.volume%253D12%26rft.spage%253D109%26rft.epage%253D109%26rft.atitle%253DMulti-genome%2Bidentification%2Band%2Bcharacterization%2Bof%2Bchlamydiae-specific%2Btype%2BIII%2Bsecretion%2Bsubstrates%253A%2Bthe%2BInc%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-109%26rft_id%253Dinfo%253Apmid%252F21324157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21324157&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.15" data-doi="10.1371/journal.ppat.1002092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derré</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swiss</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agaisse</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The lipid transfer protein CERT interacts with the <em>Chlamydia</em> inclusion protein IncD and participates to ER-<em>Chlamydia</em> inclusion membrane contact sites.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DDerre%26rft.auinit1%253DI.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002092%26rft.epage%253De1002092%26rft.atitle%253DThe%2Blipid%2Btransfer%2Bprotein%2BCERT%2Binteracts%2Bwith%2Bthe%2BChlamydia%2Binclusion%2Bprotein%2BIncD%2Band%2Bparticipates%2Bto%2BER-Chlamydia%2Binclusion%2Bmembrane%2Bcontact%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002092%26rft_id%253Dinfo%253Apmid%252F21731489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21731489&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.16" data-doi="10.1086/513850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">How intracellular bacteria survive: surface modifications that promote resistance to host innate immune responses.</span> <abbr class="cit-jnl-abbrev">J. Infect. Dis.</abbr> <span class="cit-vol">179</span>(<span class="cit-supplement">Suppl 2</span>)<strong>:</strong> <span class="cit-fpage">S326</span>–<span class="cit-lpage">S330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F513850%26rft_id%253Dinfo%253Apmid%252F10081503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiamluZmRpcyI7czo1OiJyZXNpZCI7czoyMToiMTc5L1N1cHBsZW1lbnRfMi9TMzI2IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.17" data-doi="10.2307/2408678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Confidence limits on phylogenies: an approach using the bootstrap.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D783%26rft.epage%253D791%26rft.atitle%253DConfidence%2Blimits%2Bon%2Bphylogenies%253B%2Ban%2Bapproach%2Busing%2Bthe%2Bbootstrap%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408678%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1996015960&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=7288891&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985APJ8100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-5.1.9.18" data-doi="10.1534/g3.112.002923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borges</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogueira</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Impact of genes nature on estimating recombination and mutation rates in <em>Chlamydia trachomatis</em>.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DFerreira%26rft.auinit1%253DR.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D761%26rft.epage%253D768%26rft.atitle%253DImpact%2Bof%2BLoci%2BNature%2Bon%2BEstimating%2BRecombination%2Band%2BMutation%2BRates%2Bin%2BChlamydia%2Btrachomatis%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.002923%26rft_id%253Dinfo%253Apmid%252F22870399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNy83NjEiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.19" data-doi="10.1098/rstb.2008.0144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daubin</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dealing with incongruence in phylogenomic analyses.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">363</span>: <span class="cit-fpage">4023</span>–<span class="cit-lpage">4029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DGaltier%26rft.auinit1%253DN.%26rft.volume%253D363%26rft.issue%253D1512%26rft.spage%253D4023%26rft.epage%253D4029%26rft.atitle%253DDealing%2Bwith%2Bincongruence%2Bin%2Bphylogenomic%2Banalyses%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2008.0144%26rft_id%253Dinfo%253Apmid%252F18852109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjMvMTUxMi80MDIzIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.20" data-doi="10.1016/S0966-842X(00)88982-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-del Portillo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finlay</span>  <span class="cit-name-given-names">B. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The varied lifestyles of intracellular pathogens within eukaryotic vacuolar compartments.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DGarcia-del%2BPortillo%26rft.auinit1%253DF.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253D373%26rft.epage%253D380%26rft.atitle%253DThe%2Bvaried%2Blifestyles%2Bof%2Bintracellular%2Bpathogens%2Bwithin%2Beukaryotic%2Bvacuolar%2Bcompartments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0966-842X%252800%252988982-9%26rft_id%253Dinfo%253Apmid%252F8564355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0966-842X(00)88982-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8564355&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RZ15600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.21" data-doi="10.1186/1471-2148-9-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giles</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Independent inactivation of arginine decarboxylase genes by nonsense and missense mutations led to pseudogene formation in <em>Chlamydia trachomatis</em> serovar L2 and D strains.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DGiles%26rft.auinit1%253DT.%2BN.%26rft.volume%253D9%26rft.spage%253D166%26rft.epage%253D166%26rft.atitle%253DIndependent%2Binactivation%2Bof%2Barginine%2Bdecarboxylase%2Bgenes%2Bby%2Bnonsense%2Band%2Bmissense%2Bmutations%2Bled%2Bto%2Bpseudogene%2Bformation%2Bin%2BChlamydia%2Btrachomatis%2Bserovar%2BL2%2Band%2BD%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-9-166%26rft_id%253Dinfo%253Apmid%252F19607664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-9-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19607664&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.22" data-doi="10.1128/JB.188.1.275-286.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florindo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Polymorphisms in the nine polymorphic membrane proteins of <em>Chlamydia trachomatis</em> across all serovars: evidence for serovar Da recombination and correlation with tissue tropism.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGomes%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D188%26rft.issue%253D1%26rft.spage%253D275%26rft.epage%253D286%26rft.atitle%253DPolymorphisms%2Bin%2Bthe%2BNine%2BPolymorphic%2BMembrane%2BProteins%2Bof%2BChlamydia%2Btrachomatis%2Bacross%2BAll%2BSerovars%253A%2BEvidence%2Bfor%2BSerovar%2BDa%2BRecombination%2Band%2BCorrelation%2Bwith%2BTissue%2BTropism%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.188.1.275-286.2006%26rft_id%253Dinfo%253Apmid%252F16352844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMjc1IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.23" data-doi="10.1101/gr.5674706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florindo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution of <em>Chlamydia trachomatis</em> diversity occurs by widespread interstrain recombination involving hotspots.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5674706%26rft_id%253Dinfo%253Apmid%252F17090662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE3LzEvNTAiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.24" data-doi="10.1038/ng.2214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">I. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth-Smith</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcliffe</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole-genome analysis of diverse <em>Chlamydia trachomatis</em> strains identifies phylogenetic relationships masked by current clinical typing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHarris%26rft.auinit1%253DS.%2BR.%26rft.volume%253D44%26rft.issue%253D4%26rft.spage%253D413%26rft.epage%253D419%26rft.atitle%253DWhole-genome%2Banalysis%2Bof%2Bdiverse%2BChlamydia%2Btrachomatis%2Bstrains%2Bidentifies%2Bphylogenetic%2Brelationships%2Bmasked%2Bby%2Bcurrent%2Bclinical%2Btyping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2214%26rft_id%253Dinfo%253Apmid%252F22406642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22406642&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.25" data-doi="10.1046/j.1365-2958.2002.03106.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hava</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camilli</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Large-scale identification of serotype 4 <em>Streptococcus pneumoniae</em> virulence factors.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1389</span>–<span class="cit-lpage">1406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DHava%26rft.auinit1%253DD.%2BL.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D1389%26rft.epage%253D1406%26rft.atitle%253DLarge-scale%2Bidentification%2Bof%2Bserotype%2B4%2BStreptococcus%2Bpneumoniae%2Bvirulence%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2002.03106.x%26rft_id%253Dinfo%253Apmid%252F12207705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2002.03106.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12207705&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177750400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.26" data-doi="10.1126/science.1096330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz-Esser</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beier</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purkhold</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Illuminating the evolutionary history of chlamydiae.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1096330%26rft_id%253Dinfo%253Apmid%252F15073324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MS83MjgiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.27" data-doi="10.1128/IAI.01324-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffrey</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suchland</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinn</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamm</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequencing of recent clinical <em>Chlamydia trachomatis</em> strains identifies genes associated with tissue tropism and regions of apparent recombination.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">2544</span>–<span class="cit-lpage">2553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BImmun.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.01324-09%26rft_id%253Dinfo%253Apmid%252F20308297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzYvMjU0NCI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzUvMS85LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.28" data-doi="10.1186/1745-6150-6-28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gandhi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interplay of recombination and selection in the genomes of <em>Chlamydia trachomatis</em>.</span> <abbr class="cit-jnl-abbrev">Biol. Direct</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiol%2BDirect%26rft.aulast%253DJoseph%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D6%26rft.spage%253D28%26rft.epage%253D28%26rft.atitle%253DInterplay%2Bof%2Brecombination%2Band%2Bselection%2Bin%2Bthe%2Bgenomes%2Bof%2BChlamydia%2Btrachomatis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-6150-6-28%26rft_id%253Dinfo%253Apmid%252F21615910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-6150-6-28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21615910&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.29" data-doi="10.1093/molbev/mss198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothschild</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vries</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morre</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of <em>Chlamydia trachomatis</em>: insights on drift, selection, recombination, and population structure.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3933</span>–<span class="cit-lpage">3946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss198%26rft_id%253Dinfo%253Apmid%252F22891032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMi8zOTMzIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.30" data-doi="10.1073/pnas.1102229108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kari</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goheen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Randall</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Generation of targeted <em>Chlamydia trachomatis</em> null mutants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7189</span>–<span class="cit-lpage">7193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102229108%26rft_id%253Dinfo%253Apmid%252F21482792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzcxODkiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.31" data-doi="10.1007/BF01731581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">A simple method for estimating evolutionary rate of base substitutions through comparative studies of nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D120%26rft.atitle%253DA%2Bsimple%2Bmethod%2Bfor%2Bestimating%2Bevolutionary%2Brates%2Bof%2Bbase%2Bsubstitutions%2Bthrough%2Bcomparative%2Bstudies%2Bof%2Bnucleotide%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01731581%26rft_id%253Dinfo%253Apmid%252F7463489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01731581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7463489&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980KW57300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.32" data-doi="10.1016/j.cub.2006.06.060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cocchiaro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdivia</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The obligate intracellular pathogen <em>Chlamydia trachomatis</em> targets host lipid droplets.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1646</span>–<span class="cit-lpage">1651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DKumar%26rft.auinit1%253DY.%26rft.volume%253D16%26rft.issue%253D16%26rft.spage%253D1646%26rft.epage%253D1651%26rft.atitle%253DThe%2Bobligate%2Bintracellular%2Bpathogen%2BChlamydia%2Btrachomatis%2Btargets%2Bhost%2Blipid%2Bdroplets.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.06.060%26rft_id%253Dinfo%253Apmid%252F16920627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.06.060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16920627&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240155400029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.33" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQ1MSI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzUvMS85LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lole</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bollinger</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paranjape</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadkari</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulkarni</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Full-length human immunodeficiency virus type 1 genomes from subtype C-infected seroconverters in India, with evidence of intersubtype recombination.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">152</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLole%26rft.auinit1%253DK.%2BS.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D152%26rft.epage%253D160%26rft.atitle%253DFull-Length%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%25A0Genomes%2Bfrom%2BSubtype%2BC-Infected%2BSeroconverters%2Bin%2BIndia%252C%2Bwith%2BEvidence%2Bof%2BIntersubtype%2BRecombination%26rft_id%253Dinfo%253Apmid%252F9847317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjczLzEvMTUyIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.35" data-doi="10.1155/2012/362104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lutter</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackstadt</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution and conservation of predicted inclusion membrane proteins in <em>Chlamydiae</em>.</span> <abbr class="cit-jnl-abbrev">Comp. Funct. Genomics</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">362104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComp%2BFunct%2BGenomics%26rft.aulast%253DLutter%26rft.auinit1%253DE.%2BI.%26rft.volume%253D2012%26rft.spage%253D362104%26rft.epage%253D362104%26rft.atitle%253DEvolution%2Band%2Bconservation%2Bof%2Bpredicted%2Binclusion%2Bmembrane%2Bproteins%2Bin%2Bchlamydiae.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2012%252F362104%26rft_id%253Dinfo%253Apmid%252F22454599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2012/362104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22454599&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.36" data-doi="10.1083/jcb.200909095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mehlitz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banhart</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaushansky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordus</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Tarp regulates early <em>Chlamydia</em>-induced host cell survival through interactions with the human adaptor protein SHC1.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200909095%26rft_id%253Dinfo%253Apmid%252F20624904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE5MC8xLzE0MyI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzUvMS85LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.37" data-doi="10.1021/mp200512f"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerard</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittum-Hudson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kannan</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dendrimer-enabled modulation of gene expression in <em>Chlamydia trachomatis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Pharm.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPharmacology%26rft.stitle%253DMol.%2BPharmacol.%26rft.aulast%253DMishra%26rft.auinit1%253DM.%2BK.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D413%26rft.epage%253D421%26rft.atitle%253DDendrimer-enabled%2Bmodulation%2Bof%2Bgene%2Bexpression%2Bin%2BChlamydia%2Btrachomatis.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fmp200512f%26rft_id%253Dinfo%253Apmid%252F22263556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/mp200512f&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22263556&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.38" data-doi="10.1128/IAI.00825-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muschiol</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boncompain</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vromman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehoux</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normark</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of a family of effectors secreted by the type III secretion system that are conserved in pathogenic Chlamydiae.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BImmun.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.00825-10%26rft_id%253Dinfo%253Apmid%252F21078856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjg6Ijc5LzIvNTcxIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-5.1.9.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Molecular Evolution and Phylogenetics</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.40" data-doi="10.1128/IAI.74.1.73-80.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorward</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goheen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inhibition of Chlamydiae by primary alcohols correlates with the strainspecific complement of plasticity zone phospholipase D genes.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DNelson%26rft.auinit1%253DD.%2BE.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D73%26rft.epage%253D80%26rft.atitle%253DInhibition%2Bof%2BChlamydiae%2Bby%2BPrimary%2BAlcohols%2BCorrelates%2Bwith%2Bthe%2BStrain-Specific%2BComplement%2Bof%2BPlasticity%2BZone%2BPhospholipase%2BD%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.74.1.73-80.2006%26rft_id%253Dinfo%253Apmid%252F16368959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjc6Ijc0LzEvNzMiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.41" data-doi="10.1073/pnas.1117884109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdivia</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Virulence determinants in the obligate intracellular pathogen <em>Chlamydia trachomatis</em> revealed by forward genetic approaches.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1117884109%26rft_id%253Dinfo%253Apmid%252F22232666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzQvMTI2MyI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzUvMS85LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.42" data-doi="10.1186/gb-2008-9-10-r153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogueira</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Chlamydia trachomatis</em> diversity viewed as a tissue-specific coevolutionary arms race.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DNunes%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253DR153%26rft.epage%253DR153%26rft.atitle%253DChlamydia%2Btrachomatis%2Bdiversity%2Bviewed%2Bas%2Ba%2Btissue-specific%2Bcoevolutionary%2Barms%2Brace.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-10-r153%26rft_id%253Dinfo%253Apmid%252F18947394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-10-r153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18947394&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.43" data-doi="10.1371/journal.pone.0013171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogueira</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptive evolution of the <em>Chlamydia trachomatis</em> dominant antigen reveals distinct evolutionary scenarios for B- and T-cell epitopes: worldwide survey.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D5%26rft.spage%253De13171%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013171%26rft_id%253Dinfo%253Apmid%252F20957150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20957150&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.44" data-doi="10.1016/j.meegid.2013.03.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic features beyond <em>Chlamydia trachomatis</em> phenotypes: what do we think we know?</span> <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol.%26rft.volume%253D16%26rft.spage%253D392%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2013.03.018%26rft_id%253Dinfo%253Apmid%252F23523596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2013.03.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23523596&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.45" data-doi="10.1056/NEJMcp030542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peipert</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Clinical practice. Genital chlamydial infections.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">349</span>: <span class="cit-fpage">2424</span>–<span class="cit-lpage">2430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DPeipert%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D349%26rft.issue%253D25%26rft.spage%253D2424%26rft.epage%253D2430%26rft.atitle%253DGenital%2BChlamydial%2BInfections%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMcp030542%26rft_id%253Dinfo%253Apmid%252F14681509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMcp030542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14681509&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187326200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pizarro-Cerdá</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desjardins</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorvel</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">When intracellular pathogens invade the frontiers of cell biology and immunology.</span> <abbr class="cit-jnl-abbrev">Histol. Histopathol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHistology%2Band%2Bhistopathology%26rft.stitle%253DHistol%2BHistopathol%26rft.aulast%253DPizarro-Cerd%25C3%25A1%26rft.auinit1%253DJ.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D1027%26rft.epage%253D1038%26rft.atitle%253DWhen%2Bintracellular%2Bpathogens%2Binvade%2Bthe%2Bfrontiers%2Bof%2Bcell%2Bbiology%2Band%2Bimmunology.%26rft_id%253Dinfo%253Apmid%252F9302565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9302565&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XX08600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.47" data-doi="10.1101/cshperspect.a010009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pizarro-Cerdá</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kühbacher</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cossart</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Entry of <em>Listeria monocytogenes</em> in mammalian epithelial cells: an updated view.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Med.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">a010009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BMedicine%26rft.stitle%253DCold%2BSpring%2BHarb%2BPerspect%2BMed%26rft.aulast%253DPizarro-Cerda%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253Da010009%26rft.epage%253Da010009%26rft.atitle%253DEntry%2Bof%2BListeria%2Bmonocytogenes%2Bin%2BMammalian%2BEpithelial%2BCells%253A%2BAn%2BUpdated%2BView%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a010009%26rft_id%253Dinfo%253Apmid%252F23125201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNzaHBlcnNwZWN0bWVkIjtzOjU6InJlc2lkIjtzOjEyOiIyLzExL2EwMTAwMDkiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.48" data-doi="10.1128/mBio.00604-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative analysis of <em>Chlamydia psittaci</em> genomes reveals the recent emergence of a pathogenic lineage with a broad host range.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e00604</span>–<span class="cit-lpage">e00612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D4%26rft.spage%253De00604%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00604-12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00604-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.49" data-doi="10.1007/BF00163230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Recombination in AIDS viruses.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DRobertson%26rft.auinit1%253DD.%2BL.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D259%26rft.atitle%253DRecombination%2Bin%2BAIDS%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00163230%26rft_id%253Dinfo%253Apmid%252F7723052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00163230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7723052&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QN47900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.50" data-doi="10.2307/2685263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodgers</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicewander</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Thirteen ways to look at the correlation coefficient.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D42%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2685263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2685263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988M692100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.51" data-doi="10.1099/ijs.0.01472-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.01472-0%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjQvNC80MDYiO3M6NDoiYXRvbSI7czoxNToiL2dnZy81LzEvOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.52" data-doi="10.1089/aid.1995.11.1423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salminen</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCutchan</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Identification of breakpoints in intergenotypic recombinants of HIV type 1 by bootscanning.</span> <abbr class="cit-jnl-abbrev">AIDS Res. Hum. Retroviruses</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1423</span>–<span class="cit-lpage">1425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2Bresearch%2Band%2Bhuman%2Bretroviruses%26rft.stitle%253DAIDS%2BRes%2BHum%2BRetroviruses%26rft.aulast%253DSalminen%26rft.auinit1%253DM.%2BO.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1423%26rft.epage%253D1425%26rft.atitle%253DIdentification%2Bof%2Bbreakpoints%2Bin%2Bintergenotypic%2Brecombinants%2Bof%2BHIV%2Btype%2B1%2Bby%2Bbootscanning.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.1995.11.1423%26rft_id%253Dinfo%253Apmid%252F8573403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.1995.11.1423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8573403&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TG26400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.53" data-doi="10.1056/NEJM197802232980805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schachter</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Chlamydial infections (first of three parts).</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">428</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DSchachter%26rft.auinit1%253DJ.%26rft.volume%253D298%26rft.issue%253D8%26rft.spage%253D428%26rft.epage%253D435%26rft.atitle%253DChlamydial%2Binfections%2B%2528first%2Bof%2Bthree%2Bparts%2529%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJM197802232980805%26rft_id%253Dinfo%253Apmid%252F340951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJM197802232980805&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=340951&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978EM62200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.54" data-doi="10.1128/IAI.01305-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmire</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kari</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virtaneva</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Chlamydia trachomatis</em> plasmid-encoded Pgp4 is a transcriptional regulator of virulence-associated genes.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BImmun.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.01305-12%26rft_id%253Dinfo%253Apmid%252F23319558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjg6IjgxLzMvNjM2IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.55" data-doi="10.1111/j.1574-695X.2008.00516.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eppinger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bavoil</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Divergence without difference. phylogenetics and taxonomy of <em>Chlamydia</em> resolved.</span> <abbr class="cit-jnl-abbrev">FEMS Immunol. Med. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BImmunol.%2BMed.%2BMicrobiol.%26rft.volume%253D55%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-695X.2008.00516.x%26rft_id%253Dinfo%253Apmid%252F19281563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-695X.2008.00516.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19281563&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.56" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEGA5: Molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>–<span class="cit-lpage">2739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMC8yNzMxIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.57" data-doi="10.1128/IAI.01455-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorward</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmire</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Biological characterization of <em>Chlamydia trachomatis</em> plasticity zone MACPF domain family protein CT153.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">2691</span>–<span class="cit-lpage">2699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BImmun.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.01455-09%26rft_id%253Dinfo%253Apmid%252F20351143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzYvMjY5MSI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzUvMS85LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.58" data-doi="10.1101/gr.7020108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holden</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennard</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lockey</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Chlamydia trachomatis</em>: genome sequence analysis of lymphogranuloma venereum isolates.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7020108%26rft_id%253Dinfo%253Apmid%252F18032721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTYxIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvNS8xLzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.59" data-doi="10.1371/journal.ppat.1002258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahane</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcliffe</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skilton</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambden</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development of a transformation system for <em>Chlamydia trachomatis</em>: restoration of glycogen biosynthesis by acquisition of a plasmid shuttle vector.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002258</span> .10.1371/journal.ppat.1002258</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DWang%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002258%26rft.epage%253De1002258%26rft.atitle%253DDevelopment%2Bof%2Ba%2Btransformation%2Bsystem%2Bfor%2BChlamydia%2Btrachomatis%253A%2Brestoration%2Bof%2Bglycogen%2Bbiosynthesis%2Bby%2Bacquisition%2Bof%2Ba%2Bplasmid%2Bshuttle%2Bvector.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002258%26rft_id%253Dinfo%253Apmid%252F21966270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21966270&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.60" data-doi="10.1136/pmj.78.918.216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schurr</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeBlanc</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramamurthy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mechanisms of bacterial pathogenicity.</span> <abbr class="cit-jnl-abbrev">Postgrad. Med. J.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPostgraduate%2BMedical%2BJournal%26rft.stitle%253DPostgrad.%2BMed.%2BJ.%26rft.aulast%253DWilson%26rft.auinit1%253DJ%2BW%26rft.volume%253D78%26rft.issue%253D918%26rft.spage%253D216%26rft.epage%253D224%26rft.atitle%253DMechanisms%2Bof%2Bbacterial%2Bpathogenicity%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fpmj.78.918.216%26rft_id%253Dinfo%253Apmid%252F11930024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBvc3RncmFkbWVkaiI7czo1OiJyZXNpZCI7czoxMDoiNzgvOTE4LzIxNiI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzUvMS85LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.9.61" data-doi="10.1016/S0140-6736(08)60836-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Trachoma.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">371</span>: <span class="cit-fpage">1945</span>–<span class="cit-lpage">1954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DWright%26rft.auinit1%253DH.%2BR.%26rft.volume%253D371%26rft.issue%253D9628%26rft.spage%253D1945%26rft.epage%253D1954%26rft.atitle%253DTrachoma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252808%252960836-3%26rft_id%253Dinfo%253Apmid%252F18539226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(08)60836-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18539226&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F9.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256538500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/1/9.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/1/1" title="5-Hydroxymethylcytosine Is Not Present in Appreciable Quantities in Arabidopsis DNA" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/1/21" title="Relationships of RNA Polymerase II Genetic Interactors to Transcription Start Site Usage Defects and Growth in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/1"><span>Volume 5 Issue 1, January 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/1.cover-source.jpg?itok=gg-yH8oX" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/9" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vvsLXo1rDG9EZgPyxCOfnaZXHSu1SFApajM2WE6aUtA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/1/9" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/1/9" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/1/9" class="active" data-icon-position="" data-hide-link-title="0">In Silico Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of Chlamydia trachomatis Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459762" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-t7oe2FFLustvDYGM-sBKwvc7An-PwsvgW28RJaeaBJM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/1/9.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/1/9" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZQWCIgpP_-EOyz13Kx8HlajGL1Djj2YWnMWpxbFjXPE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/1/9" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/1/9.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459762" id="node459762--2" data-pisa="ggg;5/1/9" data-pisa-master="ggg;g3.114.015354" data-apath="/ggg/5/1/9.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>In Silico</em> Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of <em>Chlamydia trachomatis</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rita Ferreira</span>, <span class="highwire-citation-author" data-delta="1">Minia Antelo</span>, <span class="highwire-citation-author" data-delta="2">Alexandra Nunes</span>, <span class="highwire-citation-author" data-delta="3">Vítor Borges</span>, <span class="highwire-citation-author" data-delta="4">Vera Damião</span>, <span class="highwire-citation-author" data-delta="5">Maria José Borrego</span> and <span class="highwire-citation-author" data-delta="6">João Paulo Gomes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">9-19;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015354 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459762--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rita Ferreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BFerreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BFerreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Minia Antelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minia%2BAntelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Antelo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinia%2BAntelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexandra Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vítor Borges </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%C3%ADtor%2BBorges%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borges%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV%25C3%25ADtor%2BBorges%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vera Damião </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vera%2BDami%C3%A3o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dami%C3%A3o%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVera%2BDami%25C3%25A3o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Maria José Borrego </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BBorrego%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borrego%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BBorrego%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">João Paulo Gomes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BPaulo%2BGomes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gomes%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BPaulo%2BGomes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.paulo.gomes@insa.min-saude.pt" class="" data-icon-position="" data-hide-link-title="0">j.paulo.gomes@insa.min-saude.pt</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/1/9" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/1/9&amp;title=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/1/9&amp;title=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/1/9&amp;title=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/1/9&amp;text=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/1/9&amp;title=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/1/9&amp;t=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/1/9&amp;title=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/1/9&amp;title=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459762/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459762" id="node459762--3" data-pisa="ggg;5/1/9" data-pisa-master="ggg;g3.114.015354" data-apath="/ggg/5/1/9.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>In Silico</em> Scrutiny of Genes Revealing Phylogenetic Congruence with Clinical Prevalence or Tropism Properties of <em>Chlamydia trachomatis</em> Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Rita Ferreira</span>, <span class="highwire-citation-author" data-delta="1">Minia Antelo</span>, <span class="highwire-citation-author" data-delta="2">Alexandra Nunes</span>, <span class="highwire-citation-author" data-delta="3">Vítor Borges</span>, <span class="highwire-citation-author" data-delta="4">Vera Damião</span>, <span class="highwire-citation-author" data-delta="5">Maria José Borrego</span> and <span class="highwire-citation-author" data-delta="6">João Paulo Gomes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">9-19;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015354 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459762--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Rita Ferreira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BFerreira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BFerreira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Minia Antelo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minia%2BAntelo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Antelo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinia%2BAntelo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexandra Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vítor Borges </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%C3%ADtor%2BBorges%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borges%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV%25C3%25ADtor%2BBorges%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vera Damião </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vera%2BDami%C3%A3o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dami%C3%A3o%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVera%2BDami%25C3%25A3o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Maria José Borrego </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BBorrego%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borrego%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BBorrego%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">João Paulo Gomes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Reference Laboratory of Sexually Transmitted Bacterial Infections, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Unit, Department of Infectious Diseases, National Institute of Health, Av. Padre Cruz, 1649-016 Lisbon, Portugal</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BPaulo%2BGomes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gomes%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BPaulo%2BGomes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.paulo.gomes@insa.min-saude.pt" class="" data-icon-position="" data-hide-link-title="0">j.paulo.gomes@insa.min-saude.pt</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=9&amp;rft.epage=19&amp;rft.atitle=In%20Silico%20Scrutiny%20of%20Genes%20Revealing%20Phylogenetic%20Congruence%20with%20Clinical%20Prevalence%20or%20Tropism%20Properties%20of%20Chlamydia%20trachomatis%20Strains&amp;rft.volume=5&amp;rft.issue=1&amp;rft.date=2015-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ferreira%2C+Rita&amp;rft.au=Antelo%2C+Minia&amp;rft.au=Nunes%2C+Alexandra&amp;rft.au=Borges%2C+V%C3%ADtor&amp;rft.au=Dami%C3%A3o%2C+Vera&amp;rft.au=Borrego%2C+Maria+Jos%C3%A9&amp;rft.au=Gomes%2C+Jo%C3%A3o+Paulo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459762/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459762/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459762/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459762/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459762/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459762/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459762/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459762/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459762/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459762/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459762/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459762/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/1/9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/1/9.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/1/9.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/1/9.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/1/9.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
