<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/1/73" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/1/73.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/1/73.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/1/73.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/1/73/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/1/73.atom" />
<meta name="HW.pisa" content="ggg;5/1/73" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data" />
<meta name="DC.Identifier" content="10.1534/g3.114.015099" />
<meta name="DC.Date" content="2015-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Dana and Tuller. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gene translation modeling and prediction is a fundamental problem that has numerous biomedical implementations. In this work we present a novel, user-friendly tool/index for calculating the mean of the typical decoding rates that enables predicting translation elongation efficiency of protein coding genes for different tissue types, developmental stages, and experimental conditions. The suggested translation efficiency index is based on the analysis of the organism’s ribosome profiling data. This index could be used for example to predict changes in translation elongation efficiency of lowly expressed genes that usually have relatively low and/or biased ribosomal densities and protein levels measurements, or can be used for example for predicting translation efficiency of new genetically engineered genes. We demonstrate the usability of this index via the analysis of six organisms in different tissues and developmental stages. Distributable cross platform application and guideline are available for download at: &lt;http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html&gt;" />
<meta name="DC.Contributor" content="Alexandra Dana" />
<meta name="DC.Contributor" content="Tamir Tuller" />
<meta name="article:published_time" content="2015-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gene translation modeling and prediction is a fundamental problem that has numerous biomedical implementations. In this work we present a novel, user-friendly tool/index for calculating the mean of the typical decoding rates that enables predicting translation elongation efficiency of protein coding genes for different tissue types, developmental stages, and experimental conditions. The suggested translation efficiency index is based on the analysis of the organism’s ribosome profiling data. This index could be used for example to predict changes in translation elongation efficiency of lowly expressed genes that usually have relatively low and/or biased ribosomal densities and protein levels measurements, or can be used for example for predicting translation efficiency of new genetically engineered genes. We demonstrate the usability of this index via the analysis of six organisms in different tissues and developmental stages. Distributable cross platform application and guideline are available for download at: http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/01/01" />
<meta name="citation_mjid" content="ggg;5/1/73" />
<meta name="citation_id" content="5/1/73" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/1/73" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/1/73.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/1/73.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/1/73.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015099" />
<meta name="citation_pmid" content="25452418" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="73" />
<meta name="citation_lastpage" content="80" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alexandra Dana" />
<meta name="citation_author_institution" content="Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel" />
<meta name="citation_author" content="Tamir Tuller" />
<meta name="citation_author_institution" content="Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel" />
<meta name="citation_author_institution" content="Segol School, Tel Aviv University, Ramat Aviv 69978, Israel" />
<meta name="citation_author_email" content="tamirtul@post.tau.ac.il" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. A. Brar;citation_author=M. Yassour;citation_author=N. Friedman;citation_author=A. Regev;citation_author=N. T. Ingolia;citation_title=High-resolution view of the yeast meiotic program revealed by ribosome profiling,;citation_volume=335;citation_year=2012;citation_pmid=22194413;citation_doi=10.1126/science.1215110" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. V. Chamary;citation_author=J. L. Parmley;citation_author=L. D. Hurst;citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals.;citation_pages=98-108;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418745;citation_doi=10.1038/nrg1770" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Charlesworth;citation_title=Fundamental concepts in genetics: effective population size and patterns of molecular evolution and variation.;citation_pages=195-205;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19204717;citation_doi=10.1038/nrg2526" />
<meta name="citation_reference" content="citation_title=Mol. Cell Proteomics;citation_author=B. K. Chi;citation_author=K. Gronau;citation_author=U. Mader;citation_author=B. Hessling;citation_author=D. Becher;citation_title=S-bacillithiolation protects against hypochlorite stress in Bacillus subtilis as revealed by transcriptomics and redox proteomics.;citation_pages=M111009506;citation_volume=10;citation_year=2011;citation_doi=10.1074/mcp.M111.009506" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=D. Chu;citation_author=E. Kazana;citation_author=N. Bellanger;citation_author=T. Singh;citation_author=M. F. Tuite;citation_title=Translation elongation can control translation initiation on eukaryotic mRNAs.;citation_volume=33;citation_year=2014;citation_pmid=24357599;citation_doi=10.1002/embj.201385651" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=A. Dana;citation_author=T. Tuller;citation_title=Determinants of translation elongation speed and ribosomal profiling biases in mouse embryonic stem cells.;citation_pages=e1002755-e1002755;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23133360;citation_doi=10.1371/journal.pcbi.1002755" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Dana;citation_author=T. Tuller;citation_title=The effect of tRNA levels on decoding times of mRNA codons.;citation_volume=42;citation_year=2014;citation_pmid=25056313;citation_doi=10.1093/nar/gku646" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. dos Reis;citation_author=R. Savva;citation_author=L. Wernisch;citation_title=Solving the riddle of codon usage preferences: a test for translational selection;citation_pages=5036-5044;citation_volume=32;citation_year=2004;citation_issue=17;citation_pmid=15448185;citation_doi=10.1093/nar/gkh834" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. dos Reis;citation_author=L. Wernisch;citation_title=Estimating translational selection in eukaryotic genomes.;citation_volume=26;citation_year=2009;citation_pmid=19033257;citation_doi=10.1093/molbev/msn272" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=P. G. Engstrom;citation_author=T. Steijger;citation_author=B. Sipos;citation_author=G. R. Grant;citation_author=A. Kahles;citation_title=Systematic evaluation of spliced alignment programs for RNA-seq data.;citation_pages=1185-1191;citation_volume=10;citation_year=2013;citation_pmid=24185836;citation_doi=10.1038/nmeth.2722" />
<meta name="citation_reference" content="citation_journal_title=Computational biology and chemistry;citation_journal_abbrev=Comput Biol Chem;citation_author=A. Fluitt;citation_author=E. Pienaar;citation_author=H. Viljoen;citation_title=Ribosome kinetics and aa-tRNA competition determine rate and fidelity of peptide synthesis.;citation_pages=335-346;citation_volume=31;citation_year=2007;citation_issue=5-6;citation_pmid=17897886;citation_doi=10.1016/j.compbiolchem.2007.07.003" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=J. M. Fox;citation_author=I. Erill;citation_title=Relative codon adaptation: a generic codon bias index for prediction of gene expression.;citation_volume=17;citation_year=2010;citation_pmid=20453079;citation_doi=10.1093/dnares/dsq012" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. V. Gerashchenko;citation_author=V. N. Gladyshev;citation_title=Translation inhibitors cause abnormalities in ribosome profiling experiments.;citation_pages=e134;citation_volume=42;citation_year=2015;citation_doi=10.1093/nar/gku671" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W. K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Guo;citation_author=N. T. Ingolia;citation_author=J. S. Weissman;citation_author=D. P. Bartel;citation_title=Mammalian microRNAs predominantly act to decrease target mRNA levels.;citation_pages=835-840;citation_volume=466;citation_year=2010;citation_issue=7308;citation_pmid=20703300;citation_doi=10.1038/nature09267" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=N. T. Ingolia;citation_title=Genome-wide translational profiling by ribosome footprinting.;citation_pages=119-142;citation_volume=470;citation_year=2010;citation_pmid=20946809;citation_doi=10.1016/S0076-6879(10)70006-9" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. T. Ingolia;citation_author=S. Ghaemmaghami;citation_author=J. R. Newman;citation_author=J. S. Weissman;citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.;citation_volume=324;citation_year=2009;citation_pmid=19213877;citation_doi=10.1126/science.1168978" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. T. Ingolia;citation_author=L. F. Lareau;citation_author=J. S. Weissman;citation_title=Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes.;citation_pages=789-802;citation_volume=147;citation_year=2011;citation_issue=4;citation_pmid=22056041;citation_doi=10.1016/j.cell.2011.10.002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Microbiol.;citation_author=A. J. Kemp;citation_author=R. Betney;citation_author=L. Ciandrini;citation_author=A. C. Schwenger;citation_author=M. C. Romano;citation_title=A yeast tRNA mutant that causes pseudohyphal growth exhibits reduced rates of CAG codon translation.;citation_pages=284-300;citation_volume=87;citation_year=2013;citation_pmid=23146061;citation_doi=10.1111/mmi.12096" />
<meta name="citation_reference" content="citation_title=The Advanced Theory of Statistics. Vol. 2, Ed. 1. Inference and Relationship.;citation_author=M. Kendall;citation_author=A. Stuart;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=N. V. Kirienko;citation_author=D. S. Fay;citation_title=Transcriptome profiling of the C. elegans Rb ortholog reveals diverse developmental roles.;citation_pages=674-684;citation_volume=305;citation_year=2007;citation_issue=2;citation_pmid=17368442;citation_doi=10.1016/j.ydbio.2007.02.021" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Kudla;citation_author=A. W. Murray;citation_author=D. Tollervey;citation_author=J. B. Plotkin;citation_title=Coding-Sequence Determinants of Gene Expression in Escherichia coli;citation_pages=255-258;citation_volume=324;citation_year=2009;citation_issue=5924;citation_pmid=19359587;citation_doi=10.1126/science.1170160" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=M. V. Lee;citation_author=S. E. Topper;citation_author=S. L. Hubler;citation_author=J. Hose;citation_author=C. D. Wenger;citation_title=A dynamic model of proteome changes reveals new roles for transcript alteration in yeast.;citation_pages=514;citation_volume=7;citation_year=2011;citation_pmid=21772262;citation_doi=10.1038/msb.2011.48" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the National Academy of Sciences;citation_author=S. Lee;citation_author=B. Liu;citation_author=S. X. Huang;citation_author=B. Shen;citation_author=S. B. Qian;citation_title=Global mapping of translation initiation sites in mammalian cells at single-nucleotide resolution,;citation_volume=109;citation_year=2012;citation_pmid=22927429;citation_doi=10.1073/pnas.1207846109" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=D. P. Letzring;citation_author=K. M. Dean;citation_author=E. J. Grayhack;citation_title=Control of translation efficiency in yeast by codon-anticodon interactions.;citation_volume=16;citation_year=2010;citation_pmid=20971810;citation_doi=10.1261/rna.2411710" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. W. Li;citation_author=E. Oh;citation_author=J. S. Weissman;citation_title=The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.;citation_pages=538-541;citation_volume=484;citation_year=2012;citation_issue=7395;citation_pmid=22456704;citation_doi=10.1038/nature10965" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. Newman;citation_author=S. Ghaemmaghami;citation_author=J. Ihmels;citation_author=D. K. Breslow;citation_author=M. Noble;citation_title=Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise.;citation_pages=840-846;citation_volume=441;citation_year=2006;citation_issue=7095;citation_pmid=16699522;citation_doi=10.1038/nature04785" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Nicolas;citation_author=U. Mader;citation_author=E. Dervyn;citation_author=T. Rochat;citation_author=A. Leduc;citation_title=Condition-Dependent Transcriptome Reveals High-Level Regulatory Architecture in Bacillus subtilis;citation_pages=1103-1106;citation_volume=335;citation_year=2012;citation_issue=6072;citation_pmid=22383849;citation_doi=10.1126/science.1206848" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. Qian;citation_author=J. R. Yang;citation_author=N. M. Pearson;citation_author=C. Maclean;citation_author=J. Zhang;citation_title=Balanced codon usage optimizes eukaryotic translational efficiency.;citation_pages=e1002603-e1002603;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22479199;citation_doi=10.1371/journal.pgen.1002603" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. P. Rocha;citation_title=Codon usage bias from tRNA’s point of view: redundancy, specialization, and efficient decoding for translation optimization.;citation_volume=14;citation_year=2004;citation_pmid=15479947;citation_doi=10.1101/gr.2896904" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=R. Sabi;citation_author=T. Tuller;citation_title=Modelling the efficiency of codon-tRNA interactions based on codon usage bias.;citation_volume=21;citation_year=2014;citation_pmid=24906480;citation_doi=10.1093/dnares/dsu017" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Z. E. Sauna;citation_author=C. Kimchi-Sarfaty;citation_title=Understanding the contribution of synonymous mutations to human disease.;citation_pages=683-691;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21878961;citation_doi=10.1038/nrg3051" />
<meta name="citation_reference" content="citation_title=Linear Regression Analysis;citation_author=G. A. Seber;citation_author=A. J. Lee;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Stadler;citation_author=K. Artiles;citation_author=J. Pak;citation_author=A. Fire;citation_title=Contributions of mRNA abundance, ribosome loading, and post- or peri-translational effects to temporal repression of C. elegans heterochronic miRNA targets,;citation_volume=22;citation_year=2012;citation_pmid=22855835;citation_doi=10.1101/gr.136515.111" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. Taniguchi;citation_author=P. J. Choi;citation_author=G. W. Li;citation_author=H. Chen;citation_author=M. Babu;citation_title=Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells;citation_pages=533-538;citation_volume=329;citation_year=2010;citation_issue=5991;citation_pmid=20671182;citation_doi=10.1126/science.1188308" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Tuller;citation_author=A. Carmi;citation_author=K. Vestsigian;citation_author=S. Navon;citation_author=Y. Dorfan;citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation.;citation_pages=344-354;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403328;citation_doi=10.1016/j.cell.2010.03.031" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Tuller;citation_author=Y. Y. Waldman;citation_author=M. Kupiec;citation_author=E. Ruppin;citation_title=Translation efficiency is determined by both codon bias and folding energy.;citation_volume=107;citation_year=2010;citation_pmid=20133581;citation_doi=10.1073/pnas.0909910107" />
<meta name="citation_reference" content="citation_journal_title=To appear in Nucleic Acids Res.;citation_title=Multiple Roles of the Coding Sequence 5' End in Gene Expression Regulation.;citation_author=T. Tuller;citation_author=H. Zur;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. A. Vizcaino;citation_author=R. G. Cote;citation_author=A. Csordas;citation_author=J. A. Dianes;citation_author=A. Fabregat;citation_title=The PRoteomics IDEntifications (PRIDE) database and associated tools: status in 2013.;citation_volume=41;citation_year=2013;citation_pmid=23203882;citation_doi=10.1093/nar/gks1262" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Wang;citation_author=C. L. Liu;citation_author=J. D. Storey;citation_author=R. J. Tibshirani;citation_author=D. Herschlag;citation_title=Precision and functional specificity in mRNA decay.;citation_volume=99;citation_year=2002;citation_pmid=11972065;citation_doi=10.1073/pnas.092538799" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=F. Wright;citation_title=The 'effective number of codons' used in a gene.;citation_pages=23-29;citation_volume=87;citation_year=1990;citation_issue=1;citation_pmid=2110097;citation_doi=10.1016/0378-1119(90)90491-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
<meta name="twitter:description" content="Gene translation modeling and prediction is a fundamental problem that has numerous biomedical implementations. In this work we present a novel, user-friendly tool/index for calculating the mean of the typical decoding rates that enables predicting translation elongation efficiency of protein coding genes for different tissue types, developmental stages, and experimental conditions. The suggested translation efficiency index is based on the analysis of the organism’s ribosome profiling data. This index could be used for example to predict changes in translation elongation efficiency of lowly expressed genes that usually have relatively low and/or biased ribosomal densities and protein levels measurements, or can be used for example for predicting translation efficiency of new genetically engineered genes. We demonstrate the usability of this index via the analysis of six organisms in different tissues and developmental stages. Distributable cross platform application and guideline are available for download at: &lt;http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html&gt;" />
<meta name="og-title" property="og:title" content="Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/1/73" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gene translation modeling and prediction is a fundamental problem that has numerous biomedical implementations. In this work we present a novel, user-friendly tool/index for calculating the mean of the typical decoding rates that enables predicting translation elongation efficiency of protein coding genes for different tissue types, developmental stages, and experimental conditions. The suggested translation efficiency index is based on the analysis of the organism’s ribosome profiling data. This index could be used for example to predict changes in translation elongation efficiency of lowly expressed genes that usually have relatively low and/or biased ribosomal densities and protein levels measurements, or can be used for example for predicting translation efficiency of new genetically engineered genes. We demonstrate the usability of this index via the analysis of six organisms in different tissues and developmental stages. Distributable cross platform application and guideline are available for download at: &lt;http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html&gt;" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1.cover-source.jpg" />
    <title>Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Wjud0Z-SjcdVn6_dwoPBKwoY_zXCZtQ5T-knme4n2KM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459753","apath":"\/ggg\/5\/1\/73.atom","pisa":"ggg;5\/1\/73","ac":{"\/ggg\/5\/1\/73.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/1\/73.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/1\/73"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/1\/73"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/1\/73":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AEzlWzFaajUoudhz6nfZ4Iwmkj+HNMBVndi2mFTubTZXldG0h+iiUIlUIl4PXMJu9AjOw6HUDk5TULqV9mmksCYVRs8own4ZroQ2VNvrTuud7WuAFfBPPXW6eyfAv9QX09WNbIl1hQ0zgB3eEk2DluI4UPeUfyS1oGYa8o\\\\\\\/\\\\\\\/AYcOJi6OQp9CfSHiT3MZy23SbEHXRdAcuKp3xa3Q6sHKKKo6NXbm9rXKh+77bu9kb7Z3T4PoG\\\\\\\/bIrvyx\\\\\\\/V6YXWiLMKjnept+WUYZoQy71t4BxHwP9t37BKZnyDTkW518JveGNLBHfg825\\\\\\\/H\\\\\\\/Uy6AOHD9m57wUEg6TCgz3KMoe9azMGRGmrv2AGd3RwIbNtD7F7VI7w6r8xcSosX6EHlHVOWtkybX0VJM5Rx76kLvyalhuS7pXSG+tCIp73WgyuTS9inAvIenpnHqHKnSxv4VIDl586ibgBQOt6kMldWhinMRPlA2xGWQnSZ3KeQKFjWPzcBG+W7p3icY30\\\\\\\/mkQxjyPXTrflP9spOGWYypUPpi2mJG\\\\\\\/DD266gbgkMQRt3LWjfRY9aK0OueC19dOSt2MAjy43nyfGT5QnvrfmgADqAoEtQrTekibMEFc2UCBWO9ssJpAg17P48C3p5BKffw7vk\\\\\\\/JUsHHWbJdPTm\\\\\\\/0WXM1pt5r01I5HmrXB0kn\\\\\\\/NFhndFlESH8LuRDsGoB1DpIdkMFMvfEqZC0IiBBfBwdZGVng1EVJmG8OqwPlYajMj37qbymh\\\\\\\/B5\\\\\\\/WuZtwkM7IU7ZpBgwkm4zHK2PiIeQF6NQVs31DuBLZwPW8fjKTQgC33iJXtaQGPST4q7HNr\\\\\\\/QkLu\\\\\\\/+2so1FVlQVlKeHChylVTP7Ewwb4TryD0HsYEOrt8QWwpskRXsn+pLm6CcpmVeKXT7n7drB3hZ180zymS4deMMuFshMfWHqBx0XyuZBm6AbYt\\\\\\\/edhKTo\\\\\\\/1awwD9LRGfVTzXON6\\\\\\\/gwJ1ArpR6V\\\\\\\/+aroj2cfa8tJ+TDCUMJAQnEGIX6LEQStRNY0O6S4pf9KPz6PwTZ5EJhZYtTRTzHKhH9SMwd8SfkWwLCONyCU5wbzB1fwa729h2bdOUBmVZh5QBJnlX5ZqIHlUE+RZsZwxvdzD78maXYv68v45+\\\\\\\/Sz+mSup4LDavsjpXBNYJXHZ58K6IXnWUvDQ\\\\\\\/gEG+7hKA5itOeJDlgcdnm0In8yJW\\\\\\\/\\\\\\\/97C6Bxhc5F0POhKHOniuxwlCRLYYg4X9UumPJvN587o1MwZFy3p8iLvn1JlFIX5IqSzmiOVBzK++xBuXhYuuJsDaaGwwtHhvZD\\\u0022,\\\u0022iv\\\u0022:\\\u0022q3+MVCct5nywzB4Kdue3jw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d05556da122ecfaa8ae8806c3d212c637fa5cc2b69dd3ba8a25ac919514ef122\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AoBgPfMe76jHai\\\\\\\/P7uipDqO1XX7AuB16dYGl3LwGRx1hsUbnFhZ3TUT\\\\\\\/LOvXsVPcxO0ULdWQrwrDkAyyhcTPW1D37qyy9jozFi5l9grsotBvVnvs8rfFBPOOznqx\\\\\\\/xXggdNqe0tkLuCl1DUirJBERj00IybRKKmXhab\\\\\\\/RaZgZaIMLDbF4O6wZWa0j10scwDtMQupWn30VzGn0tK7s4GmPBzksW2ZnvmUPloGz5fh0mmT+Bv1KUMxUTkmhX4hyrw3UFYYmy95YDOPiv590c9nETaXmnVDxFIPlvQpCSqepnLFGVoCuUEAD418nWYeQAVKCLCVt1W7iUj9AUAOft7BCBfuMp9YfCnkzQnj6NDc05htSlkNRuzgRErTrBsqUStUzot1lGb8HTIWaXm+8HwWyi0olxR8VG\\\\\\\/pgoRqwZPJQyWyqgYAeTxLOHfr25SQuWUHzNvbVXrfxbsXigMKZMnfTKv6QTbhLiT717K7cEqzbFn5Qpj0dc6V1t\\\\\\\/7C3D9kuLVuqhpBEiPJBHPdZnFSUpMhpVyc4AE6\\\\\\\/\\\\\\\/v8z04WIb8iAfIqjmqnMMDyljWt1aUQpmFrEDNEudeTveON0w6gH1y05i1AfNq0lHsVlEDpk\\\\\\\/0WXTZbKlZLeiKxO7jWaL0P5X1B3xDH3\\\\\\\/8IJHV7mH2mXeCfcg1uggcYCVMmZkovWnhYBqIdpoGtMz2wit8ttnrhoVKQn1mDJ64rEvZNv9m2gGDAaCuwTW1WH7+b6TJlG5Nu3zbXFmxWzjqOPJLIB+mdhelGs1X2EXJ2go\\\\\\\/kTMwK3MbTj56C50+CIdjqydCZLhwwdlQ6EeRw4aVfq\\\\\\\/OHiHAc97wx7Iddzc4aCHEXQuHppsdde12xlaFn58pHl9YS2KC+vsNnthkpMS+o1PLRIvNgDe4oPXs39UWx24lVzmlmBixhlA2mTsH3rADbN\\\\\\\/T2Ebohqa\\\\\\\/oQ8yppztwZglmwTWyU+ruAHa9cc\\\\\\\/ktYs1skq+ASDrKNBTySHxLGU3XMusd8y1VDK7Mpn5MVez0GlN9V5xkSDLUSQUJ5otmkaNBrrMhdokErgjnzVw9pcrrbkihe+67bCLkiPar4LLxMBSwL49GPNsh1Ct\\\\\\\/cGcI+m8YV0+8gANdIUoQU2fMpFuA9CTxeCeDdNnYNoBN7YTIRs3eYqmUuLOUoszflYu6aRLNqXpnqESLshb9Kl2gLpQz6HKFNiJeUnXUlmv0LAOKEkXuXiJ00KnuUa3wO4yqcRu4vN7BhJDn6cvYbdoftT7sn1Xth5t5Mm8u4UkBCC2BUnvmfZOSOd5j7KEncLFwrGpVsvIl1rwIDVJk7MYzTw2imG9DuQrenwsp+EdNXQG\\\\\\\/xGTOYTx0vTmsbEt51LTXoWoX7699R5CQUYaiwOBG9gk6Z9pwLyhBG6JjF9537uJtDECETJiglXrpFGO3bPF6eOAQLLDbKlsHirjRK7mUqjr2EHpG2yaCP+9wqM3TPS71gzsEdd5e1FxPv3J8g6Rv6e3R7xhjNMUbpcJNbcPkjNX+HrEN2OcxAesV2DFJsHfpRPpyyYn0QNPsJ9Neil7Bdb1W9QroxoOE3AS\\\\\\\/LBsyf8\\\\\\\/IbXUWl82gbMavYpaAws3yPKjEbtoug1SIs83P4CIwffJInDWDyi7fN0F9DLVe6\\\\\\\/cGcQwOZj+Cf93AwYdE8M6maCXpJsf0yYDY6tKc\\\\\\\/G08Rx25+p9\\\\\\\/c+Z3b7LgFgFILUM4UlWAUvBfbLu9VUoYtaVxnA3Rxu3gQVZlkfXVGYHF9dgoD1Wsj7OZoVtSDqT1opYp21NwhT\\\\\\\/UTmtaA1OMtlUvhMPAUrZyaboPFnhn6CPUyf3A7I\\\\\\\/6kcCE9hLi0187fovL6hBEJR2qTPH1Y3r4U66tteFzhROcc0XKkMaDpt3m7jCkE0XbP8aelua1y2QsRcNl69E\\\\\\\/JK5vJ3+nMcIL2OBpDYuc9ERgvvgLnc6Hu90bpvMSuNoNx5eW8HKoDpbV4xYIImMW4ha5l4RCtOcp9Cgs\\\\\\\/NaaxtH1fWkHdkuQbWGB0nNEY8jiJ0psAXa0BPKkRaLzSctaB1FB6wErD209zVUknQAxaoBKaa6EYPY9yQh8FjvVNha6I7RwoJRerh5PNV41a5+Kcrqs\\\\\\\/5ECEZF0HrYmBBobafKk7++GxKBIRtsGtIuBlW5Vh70EBL0xdrX+CfSXJs+yyXwvP1LIoOONorc6Mbp420be3Thi1msj\\\\\\\/RslXlqOGxXv\\\\\\\/NROfC6Rj3Z4iotZDTxfb+DWOopR2hrD6LOVctFPGfPNaS0z6CDI8i80Y4XKXCJAPFv6DZ\\\\\\\/PEfUzpDxyd1m6Zf8DwiFnQthKSyyJly8gkw8JZ4zeVXpdtJN3BYFk32n\\\\\\\/Hgv1YNVRtEM78XZR8PLorluPcfRGemFVkyXtgoggu+vLffFEfc+dc9laN\\\\\\\/5c+xOah01sFd7ZKW3dsc\\\\\\\/vrO\\\\\\\/HKg4J\\\\\\\/mxv4czWL6KJfpwac+tPKaTpOWrAxDmxPdNdPmJr+p1jS3s\\\\\\\/JdXj5uxeWcOL\\\\\\\/+d8CKGya8IyJpPbu\\\\\\\/DiSbhiNOMC4PL+Vdqh3CvER7Tco2C+J4aLYA2u\\\\\\\/vtpbGr2IEbCr9ofzcd1T6+qlyG6Mtnharz7LZgXEce0e0AtQVBD7x9drWkEJApxT1K2mp1JbKFO9ruO79xGmaHMg0yo4gCzSfrCjOOi9mDGvX1XWQr8vn2TIqY3zFM4Zp8wuMKZD3u78\\\\\\\/INCBWHnstuU7cLx1LA41MXJ7PuBVA6CiXVmWWMfPeY7f2BbaZb4fVBP5ZMQgtW4YZ5AVrzfBdF1EBT\\\\\\\/sUki7kSOhu70LYQQ8iWbXoPlCeMs3NYo7cg\\\\\\\/RPb0JctTAYaAXSLwWvr7xSo9d+RnMoW5Y5j3jqXFadlspHJuc\\\\\\\/IWcL79nEDl8QUX3D8ZBPHw1enG54HcbK+0zn0q1geL31mOHWfpx8iNXf1H7hVAc+04YlrfDbielvHYAUio6T8\\\\\\\/Q3Wt2wDSQ0uOj7uemwZ0YFDQS26ZkJdb\\\\\\\/7EE8Ria9vNvEqjC0JbJ1EnK2YvxBBTqF6S9coV6eNYy8kBIat0gJbdrthTju5ZJnDZz5o4VNuxyDQH6+jqDc3hhmts7xEpIrUcLRo76nlspcqTv90Qf\\\\\\\/OSgqXSmJ\\\\\\\/r+rWQpC603ILyzZIRphBBc7ZSwLRGLUvwcKaxyJBwFjdzchjjz2oVW2MqFLkY6RW3oFf00Kq5qVsU9hyibaWRO6RSqTg3H3QWXIzwRWvlIZpN4UJ8aH+OEfIHAQONkf8Ab2QW356wvtRHE6heqgmU246knmvyZD+wqcEIh48ZTcSnh0+1qiKhiB5ZqJnBXP4ZJeOTtx6YakoU9l6KVftFj63mW3lQOBkbv8pfSLmgjVOaTy\\\\\\\/K6WU+SXLgBcF+1hL+VKXIBwgkf1b05lPMeqlwy5Nr\\\\\\\/nBCCSDylW18WQbLjWuJoyBuQh5XLtuWaevFA6Y0I5Uw84QkLomXKO4Rb+0ERFsD3tJ0IGecstx7TOfIe9asBu2C8Gkof82KZFZJubidZjiRuw6Gq73HyuGFxERq+1OU3Bb7pfCmxE6XDDyYCV\\\\\\\/2v+9uJTdkKz1U5OkBQK+kCTVu7ZPnYY1Z9tfZ0YsxQ70KlTx92tVNZqn0Kwv2uPFOPiy2eXj22tyFiwae2KELCeoWt5mIws3vNQW8yhYlLZ+2rw55LPabCUK6t8VPsUcvkYeno7QW8GfE6iRx8K\\\\\\\/ZfyCAAgZiYtr8jyQa0Yxt70kNwYRQPAze8QpOQ1Qtbi\\\\\\\/0Kc0FaOCVaFVAP3HYdUh4RIitxoDFPOpL6JhtOa\\\\\\\/x5R8LAHrsjW8a42vIirt0jv4DqnX3NvwFZsA8ZCRIkPrTQUV1hmcUXx4qY\\\\\\\/6korXxNbxvWHBZ\\\\\\\/2WMVeDG5kkTERKacq8K1fKLM2Px7hzSH3aETLLciTFbrXFM9QweZY+avVUhWVMROIKhQEJfAowd+5O2uPt50s0oobeu5+faQiQ4FTNdnX3MuLQDNecZEGS8PC3k\\\\\\\/Y+3koeSq3G\\\\\\\/7DDjeGh6s8bg5TPKAC5aUzir\\\\\\\/YjGVqQ9xiCqYxIRj+XnmhSDzkgFJpdBoyFaEJqhRUTO5hmgjjB36YoiRH4xSf51iUwJrC5j0N7Cub+BTYndhDPtfR3NyvrWOZmlK7Jq\\\\\\\/6qhTbsQjgd\\\\\\\/g8kRVfXZwditovC2k\\\\\\\/yC1zodKJoZoD5WE2hLUlaqX4gbpmfRFV89JHR\\\\\\\/HO4iCu1VNR2qXbuRyOpilIe7XNspqPcu4neIK7DHooy58s1rVRiES+0kdnIGkdFfnfkxET7KXSXiEmxklYOxxen1hVx7oi3tKC24spLw1nSwZ4HxNotZlca88eI4BeZW49nG5CMJHKFmhf+JXqfu9pGErszX7r0scJCL28yLs\\\\\\\/Y2VJjf5cdYc88jaERFpirm0Tuic3IzOodbIWV6IapGvgiL1Sak2Rf89jPDOsWB5WRvyC8c5+djXpU0SyIBQY0iYdlSDWHGQiFVkz2d0fvvf5xxqEXwinWHlnV5UTIgQN4yv3qywhFUuv+trLeIVEaM3U5nFA0jh8fhte+lREdGQcSQ6idPetVTXkFUSp+\\\\\\\/uCDEchsTbQlxom2GOkkL\\\\\\\/OWxP1NAULWKnVYg5g5QcK+dpRIPOlmw125vjX4o4srpLhKuEmsXMrYs+IiqbFvXzOtclnfH7bXD7mw+MUAdJAqjwQMV1WteDK4SUkLpHcd8mVUC5y0Kxn\\\\\\\/Q9obY\\\\\\\/kulXUSWwYDiavEdivSnUeLBhK1O+EdGOoqYXoaqZ52HlUHlmzGrPbQyMg6M2fEPExQIqdxv3pN05\\\\\\\/pWoDmAKW72atNDg9tOepl7rjtMaovb8ZzJIijtOrIXA4c4DrS\\\\\\\/QkiuUTYDPyDQdYFU1m+DQxFpQKzCZ20SH6xtHZSfA4kFKLAH3buecPvMIRv2iCdfkCW8UOjo4ah1XN93Dnmhc8fUHKdkAwUbfWX8ZXs8YQK20adRkWFaTJ0zYNXeFqX2d40uxcz5ez4jIV7YbNze\\\\\\\/shr6hBkIT\\\\\\\/LsdAREil2E+167nSxrl0JvZxZU\\\\\\\/\\\\\\\/algbI5VP+mzH6nCjyitNt2eBh3gLpQ4VIqduCB54IiieG31MDJTGYW+6i4mtqAsH1RDOuuaxDi1CxDcusOCZGZGJWs1p+mEC4RorphCzZM8mFgjhUOtkwJLth2P64Q7E6h3+hFtHeqL3yrxpNXPFMfd+27lJv9hIdHOeqyUPzLeh3yip93WJ21IgTAn4\\\\\\\/FXCkusafp8BG7ZNFN\\\\\\\/N5gtWhf+oC7eBLtV8+5zdgufUeUI1u0T4jeDjtcId5tWhg9WHD9dNGbLkoBR6u0GFl2Zi9UrQIz9lth\\\\\\\/rRiBIMNYHrp2ENqfJE4A3yWxfUhfb\\\\\\\/jmtRxXorHqLp18AWtEtEFLQTWtwTrt+H2XRJ+e0OpqYZhOCJVjok9NQ14P7WCpLAq4X8NMI+kKeQdXNltShm+52RHSqXYDSmfGSyU7SqKCfVdYAD2awj60AJbxM9l\\\\\\\/EXgu53oyTFG7ISbOUaVd53iqbOGjGnQAIDVj7eofuQDXhHzSqbRV7cxAWPEoXnH967xq0+n7J2CZyHQLovJicZE\\\\\\\/UfLBATwaPDdK3t0nDrFq39g\\\\\\\/JcRhxz3JlRfhIrxh9SPGtNo+rz2H5WU8erhwBPTJODptYTJ8bKyEa0iCVfpnVCUTnDXLU0ZS97v9z6z5g1r54bpGNN5EbnwkLiaTUVko4BKPkTCWS8q1GuezkLOSLeYR51k04vbg9w4rLw01Ml+zghsAO8pL47syPGzQ2pYOppHv5Uud\\\\\\\/iatd1FYwyExykbB5+vwaLUrkdBgvarD3K4+qsxfyvsR3qLi5Pa9IKRJ8+yFxrBIoVhYjkkkLjd7XNbGc+wkwndU6f2BWQxLwdvsF3yky6FzN3\\\\\\\/pvo5MqKUloevBCI4O\\\\\\\/\\\\\\\/HwFUp9aX2e10u\\\\\\\/amccE8637w0rJWGJtsbgiYst6Qxdenn9WebYX0B2jCLOw\\\\\\\/m1plWn6uSMaLTGsleHT2trsmhu6NWgod85qcSyvL7\\\\\\\/+PhMSafI9seZtqNSMZzwUV7thiTXZ+vvMdAp0abdF9DIMRR9j2H3aUFH5UohvAG3Dm8BpDA7lUNJQdqeQnUFeZPEI1HIRd2NrK94uhnZf63Kt3I5wsSl\\\\\\\/q0nxFWmjw9ltX9WTUzLtG4BVAP3K+wO\\\\\\\/3spi2YaW7urQ+Sz4FFz4GO1AqTlA\\\\\\\/Gn9pkonmrTLg7XmPS23YhAhcQ+S8G9MB5W1UjlPeGOr4t\\\\\\\/nuF274bHhxru15RcLCd0NdYzsbTJclgzKGQqw3m4ZM\\\\\\\/v7WMtzl8QdrwGZpTj1q6qc7\\\\\\\/AHL6CJiEsv5wQDyXCy2Ru\\\\\\\/J09OIL1pEWNCfe6rctaPYhhk3Jil+R1zuw03W1T0Ju08gre75yuly8xb3+4hcIiAqLo2ZdWU6bx1dOcDFV+qhovBWdhWJXLE9NTZsPedNCU5oBjPu5uakdHGNBa2LV0fbsAMVeAO3rMXfQp9EZChvCkFI6gMt\\\\\\\/J1sTzqiN3u+J3ErUjVdvR9n92dxJiV\\\\\\\/ratJULIgTrfi67XUkgDBhqQWg8ulh+SgXpagn3cxxy\\\\\\\/KKuznqc28dtlDt\\\\\\\/rAklJYs5keEbSeU6EWcO4l\\\\\\\/AnnOj4xlQzeENMVf0yjbU2s2ZORhce7TPFE5YMbMGtv3dAZoa9ArOFVJjUFB\\\\\\\/OHvDsfeHoTUbutorY0IrYByqT4prPpnILUWOFf5EmOcyXVxwoiR6XepFBFkDTC2nr8W6zFmFV3vmCwmiPZiBd9JyfT2dRQSHvU+M8Q\\\\\\\/Gl\\\\\\\/sFS4GnZEfAEudB6kj6OjC0j9lwXVkVWTQ5dEd3KCOPvRCjR0LK4Oc2YZIUyiaGtuu9Xyx2ro60zBHMbVbvZHowUFSq2M8xXWTQxw0Q9qBHFyR\\\\\\\/51w2EP+8JcBH0p0fbUIwEj4wivTzNLMW5WuwI9G\\\\\\\/bIijPn9FQxoy9xoHEn5ekD3Xq51gyuJZ5sPyrflpuIU\\\\\\\/lwPv1lQn3vcqFx0Zb1Jcc7nXfrOKfrkrabnjL6vmGxVhas2TAXZIEU0OjUF8\\\\\\\/CevQt\\\\\\\/rtEoVZGP20QBPWxIqozGTLx71LvIlwP5SSelaROPrjSPhW0OURl7CvBzG0iGJz7LFOVH6YrZgfIZnib66sn4iPRN0Muq62pjpjadTImKNK7KOyle2ENqdO6ovTp0kqbm0Byy+UV7lGz\\\\\\\/cwK7mQTCW9OB13kHz1ETTblPWYUI5fAvDoRbumufQgwQ5P14TuTNIShUkTCnvnD79qM409YnZx8fDg6c24lpvaan\\\\\\\/wk1zYocQdFvH3NxtFmXuq+KnE5l2KjuZBKqDeBGgPU6X\\\\\\\/nnmV3DiY+ZxxdS5IDZwIqsCGNr2GQ0dpDAcuqLWULD1MSW4Qei358eyl9OfA9N1ndiL+ZuLqU1S5czrG6h5DbEfv\\\\\\\/IKL1QnEpCZ+xJX7DOLTYXumxEvtHCMW0fREfl5BH5oeGy2NeUIJ9uSJvJHv5MUs2WBh7izDD1\\\\\\\/YmuKwUC9IuOz1LysAAzgtxZsj5dpmww6qu5SVteXVgSvXKReSZQFCIkRaRtOoAJlZkgS4az1FdCJAKkeMKXrs\\\\\\\/ev+zCauVhLJOlzTb5OSVPkjjoWHT6BawTzgH9sQqJiwYHZ8YP1+xLKRDm\\\\\\\/0aLdt7DN8AHVWilBPt3VOM9uQZRbMNInviFxElZFO3yEUcI1k5CVibJB7B1fcP9ME2xGiQhzHN3u4lXs3MYh5dZgqKYnE4OgLaTwqVCHs4Iqx34GJ+nAKBszXnv0CoF9u2u0vvYwbBXhkV1xmBXx4zQhO\\\\\\\/7HXVVjZ6RVIc5EatZMx\\\\\\\/ov++z\\\\\\\/E89LdsTK+55TMd+Uyuy6UKkExOtz8YpZfmFm8Rjnrbw1xojcYK2ZZd1eI+VHFjJ\\\\\\\/uqjDGMpmBlQZuyKP3XszyGhRp1HoNNYy3shDm9Uh7w6ZAUQZVCnQVSVryNrN\\\\\\\/5T0Y9uPltD1kXIUfP2uerBcKyWN2flSQ16iQs2AI7y033QBIauoafflIgdhMkrkghle0k1SOoeC+\\\\\\\/aahwWmIUlNtqiDnKKyoGeUwe\\\\\\\/C7vEBMEkG6ueAf3jv6YfUXyR5+3yDTbyxjjWupGAT1oOwYuU462SYrOhLehgBdwNulxOjWzX+ZtkmFtwuoA1IQr7UGkeMKjWUF6lKk82juRcpMYBCw8x5NFkt4YJ3l7F5i\\\\\\\/vf4Yk91EBRZX3F7b0TSDt5j\\\\\\\/i5ZqxI6iSZT1b7dgrtt3rcFg0Dh30zZH3ilTi8Y8PJjHIc0rJnGu+y5paS3\\\\\\\/oSVAfRnpGOPXSqsfDYG3cNp5ZlB9Fbp7lrOYrZxjvN+LW659xf16HiccJrOw2hJbgythooOFDDaYOZ+J3zZyLfhQV28ZK0w0lXWI0s8eV2wFwQsi9R1r0o15G1TLYJZBu6mbaGJ3LV3xvm8IaZIrrx+YxtjO4nVeueGHj5bA\\\\\\\/Rg95LRhdMKy3uuEuFYAtthA+aPZYMUml75\\\\\\\/zDjNO5\\\\\\\/BZkau1f\\\\\\\/Qm8Y6jxMNjzVwjK4Y6BhN2b+eMDqzNLapkDE9P0p3sOFS0ySbFpRzF3DydqVgPxmTTsRq\\\\\\\/DEkzirky2nuJKHlrO9dteNGka5ylLVcFwllB3pEmt\\\\\\\/dARoLuCvyIm58idgsBGS2yEggRg02BWZWTicz0TDepBUWiONsesJG9mPtj8qISXCexUeAbeizLIXBLIHdoFbOfoPlhMud6KUgYfJ7olTuXtkQZNjXylepe7LdiUmz3c23lV+Oy4zNqbvwptEYfx8n4MEMvX2ZCq3+5F\\\\\\\/hNhBUDS2ZiYWgQu+orKqo1ZaLu2QUo8fbLy79HnvjCj5JN5GEjvLGYdc0FfJUeh+dbSMexGWSR+EhY2rbdEF1I7WcBfGVFIs\\\\\\\/5qXSUedEa0ebOA8hc0uUxK42P0fRbRIdLGFB9Z806NV0QWtiXleVi+IiQz9G4P5GObm+3ZKSejuWIb2IRBA0cUEN82zS30qSggJa5w9fsqn52aE0sbk\\\\\\\/tldNt2+g7q+3ZpvZpbfShbfy8Up5c48lzFCtVND8YQDv2MWCPaOgXJFuf6JQLr7hCB\\\\\\\/9QgzX9ZxzpfTia\\\\\\\/VZLV2ib57CxIZl7gBzXxaLW4V66CRBPCLFhzgvmwQ6BdNT1H0HBLd6o9o8Lmp3uCtymbfSQd2dfxeu0bW3xkL95RT5sCBcCqJDJcxDXD8ayr4pIlnehilCqhfYub3tcEKLkv8VXJc3hQEAx8pQyyzabZEJPZEtN7XtHL7hGuMczyL1D7QjFoFR+pFgMKhykp7SpHyJ8V9j5LF8z2ae\\\\\\\/d84CH+NCB+4gh0h3SfCZc6bijOKI84mhFMkl8axeJu+7e7q4\\\\\\\/\\\\\\\/my2m0Wz2i4Fq5UKozUdDccLgAqCc3Z53+kxzVp3Ea4Z4Iq4SWPkQIQNv8XcrBw6Y6y5840+UcEyy\\\\\\\/sfDGVDB4Uq\\\\\\\/RC4KO3DtCiX1e1lp5C7WZr7W8A\\\\\\\/VvqdrLWqbF2SjMz15P0csF+myfGjT7anB\\\\\\\/Ruq6s5SiHOROy1JL3\\\\\\\/Z0j8ngZjCj3GeUNDnQrtbKf1P3\\\\\\\/AxAc7iwb\\\\\\\/zz\\\\\\\/YZfZyltVx+PdXTDhT5Quwz8DsGns3G61GsiAvLaTqtiVooyLLTtu2qasSaJZFQMiaQ57iteGql\\\\\\\/w5OHuCXgLhtfW1miXbyMFGIcID3NfiKtb1\\\\\\\/qpxFfeSq3rlqd5DEXL7pYSrGWswAtI5KFf9OVcDXDhT2IBeGoLyHe94ja3\\\\\\\/LXqzcdWBVC9aWV+KBtPBul+tDhzgYEkDavioleDulSxp4GuNYVgypfhs0VCqIyo0wN8FvkM4COpKyVSPPTCFGeN6bRA+q7FlyseoJg6\\\\\\\/OhOHMC7WN0GQ\\\\\\\/VA9AueUiKv5fm1PCUaiQTB6Et\\\\\\\/wYb7Fix2d7Y74StbbCCjkMTfNd2GUoDc1EIXsax\\\\\\\/ou8UL0tO4rQrceVL0f+6rHqmS\\\\\\\/o2cb83ETtUZA+s9CCR0n43gTGBRGXoISOmnwbcoaTSSLyGHM9Jya0\\\\\\\/nA\\\\\\\/+Oyw7hOP2Pe9hUFlSgi7P+ejmJUHS0V23LlZ0ImHA+j6qODriIfynsigvlVDgRcVjYdi3M6OSgUTiSh8wYIXs7OMhwa5drX9RJ77xQcGIE2sL8OCKI06HSqWOfz3VeT+Aq959RrrJ11hha0lL7N41KVWgFFmDwy55JLvEfqdIUtKN2zCqEZ2g9S+cgvCf9fPVbPPd5LCwbVlrFlX7qam5rL+Cirwrs0gUQrYrOQsQWt8ciUjPwOKqLrbcz56U+RVY3V5qtMJei6ljBYi+bHqUg0axOBDRvlV8oUHpzu\\\\\\\/BCwiOJfDkQvJvrgc2Dtr8HuR8H4aYXvZYm9CZBaTEhrt6kbKZRyjdzACw6KlLaZox66PIDHGcrpg0RGh6eG52\\\\\\\/oHHnB53c6J5Bpgj0lICx8v8qibfdnAluzQWzFd+SaHfhUR7wQ07fky4FYl8KU97Z6tCVrW3yGKns3ObS9HNObtm\\\\\\\/18OoTzCpa+R0EyPKDyg2S39ORcvK9uxCWNo180ULWA5Qc9TGkWcxHHkhZypAXWcQ\\\\\\\/rDTqf8tvuRxjyNmDQ45npcn78An+M6dOxLTmY+65M2fuYBHWKaAQXixp\\\\\\\/ff7TsYd3XSah6HmAWnKKVB58HRK2G3ohPOPKZmY6t3UQRfp6wWC3kk0M2n9Oe2Eo2jdkj2kec\\\\\\\/i9SdQK+8rdNK+m8jcSRpg3fMYL1BOkkbhbyfqTqa7jncBAZPBLm1IwSx23s99q9fBuXxfBw02I10UsdZJO8bfzmmkQMChj3hTaLx+l3a7sc+W4HBasEV5TXUKOzIviVetAI76aEJ8\\\\\\\/+WjXMIp7mfDb86n2T3HM9GtKg5IVpzZSLomNtxcZQIAAwQtudYbyh3THN\\\\\\\/BuFIjLM\\\\\\\/C\\\\\\\/YyThJrc\\\\\\\/kd6mlUWlwV168j8UzYjO99G+dw9WqXt+OPT+v05DP6\\\\\\\/qmMBpqsmTRsOy7kqcJgPhsT6nHba8FoOY2J2RtssHBtXY+Vj9bPKbAyOUrIZgjbp8GXc3JZYNrf0SgPJm++4HqaFOdohgFEOwFH9XP3usfmEcI+PoqieWAHdgyCpK\\\\\\\/zK+qwt+PDhlyHESIUoJm1WTc2girUuu\\\\\\\/a80R5Md+5iSZIxb6Htw944i8TjdUuhG1O4nA2DV2oqwsgM7dbssOAQk1V7DNnMOLOiSLxV3wXbNSl9nWAOTO+maoURgS3FcYuPUT5gjoJaIxPRjghpI\\\\\\\/VZ22wd\\\\\\\/Mqy3lrgVL9ccI3gR0xcazxbtX+d1nTjC0ucl3B+IWX637+MKma1jJS582SIWFaz4Db7nBU8of+1doS6MPjkslRXJDoTxeovQWUgy8OkY2qEta0bRJSfPsmJ1ifTIrSXa\\\\\\\/jNhP87ApPBKFaZR7AAVidcBSKssmg9ixCGCdxl6gnKKMck1xr8ecwMW\\\\\\\/FCvgER1UY7qCiUgcqLQgXel6Ko7MnFOf4eoWzsBpULoQ9R+idbi6L0mjjtPYhsqcsKJxA8iJcMEH+xhROax5ZFDTXoE2yfjykB+TWHh95M1cqgnan5Sm974yk2t33lcRKjh5Gm96ehi1tnJ0sssYt+Ur2C90mdNYzKSUOtYkKwrzMEe\\\\\\\/DxfQqGW2sMuGyNuUtZGrL+AgJEcjUTmPLJEHpaUMk0zg5co4k2iVq4CqBCm5qZ+ixIBzwSdtEmYu0KeUhU7xDlV9rPM9kwtLApEuqDb0RW9Z0cOImjwsuhiiZX\\\\\\\/ux4ffAil6ot3J5I4TddoDdX0ZCaprcENp\\\\\\\/KESrOZxH\\\\\\\/vHjMdEESyeeqZWs5n4h86d1EXK6ys0YKstmEMhpzFXxPVP28rPaXSOdmPoBDnSrJV+T79VMJ2TOUUcZ6aCDQKG226bWKBVsdbtrvKoomp5kL+GCbi+1KGbLrjEezTNXTRdaGytGbkQycVlv73xppkNSkmUMU\\\\\\\/tLTg\\\\\\\/8IT6h61GwXjNRjaODaAoKfvxFBjusHQGVfXYtoImoUOJX3wjk\\\\\\\/QMGhW\\\\\\\/lLP0ifI3dhkz1TaDAAgHUqLlLbex3\\\\\\\/1XPR6Z6bSfq6gdYaGfJ6ZyYRopr1Fa8gTaM+MltxiDrhgKgBDRO3sB73WnoO2eBVaPtB3KDYU2DiTBn+\\\\\\\/U4FvIXToXuPr1Z7EkBJnAKW0FRv0b0+GBGcd2AccynZfjshIe\\\\\\\/EI7lxwr8mPZBdflh2K2DYumWb1VJ8LnTlGdjiNPgd4VZ2TEF3aaWADrYJKPIzXdRn\\\\\\\/R+8XvGhZ8FxC0qwseNw6OZ4YOAPuue2MAQ46dM5U36JZEaVHrFsvfoYa\\\\\\\/zlR6K0Q1kxr8Owk1GetnPyQBZzFiNC2A88eyW5\\\\\\\/A+dsyGuqbRkWuyBdVzV0y4UMY3ExR50og5PqKCgsdhjJ\\\\\\\/vElE07fYVIWYwKnphCl3qK2fg3AmrLuPz8AgPRuwAxBi\\\\\\\/vRWuLUgaH41N6VIPf+FCIN8yUB8z4m+OazFyymEH3S1Qi4eHIPSoe6xAXoWCHFy4u6N70Zr9Bo0rJFhGYJ3OJxySMb9fDtCYka9+fWEPvYnOtDensZyf\\\\\\\/qWMzvZhlUOtVj4Ka6KYh3jhuZ+08s4Dd545MHDixODvLGwy\\\\\\\/xyZUSJLFvUodn7UeXPMbNNSzpLZ6FHnSYqKubhu579k6XsBAn5cruXn54m3eM4681sYhLRec5fml1iswEyc9RSRFpPOHfJdyYn\\\\\\\/nhFl0Gsu+U3L9Ne8uFtRk312fx2BezYTknj+I+h9clpfOCicusSJEl6u2HFI8vFfVMCuMhKBe2lfCPrx00KxVvyqWn1Ecj7E5Xx+5p+\\\\\\\/xgS25YEUvarbU3sDXKbPw19P1rWI3Q+JdHDJwZ5EY\\\\\\\/ZUw+n7VjgvR+wfPqRdnj7AADQuifAuxvz+DaqMrOG9SQvofqJyOdrVMe6XXGtAQih9ysxczKALA2uFyf8dXDW1sF5e\\\\\\\/8LCjqoJPMUQNRSkWn5nr1bT3RU54RgG40eM8WKR8LqhkEa0pCmFA36ImexVQbbyrHGz+SxklnLJaN1GjsWfqf4BTfg1D8iA1KYwR20U\\\\\\\/krS8Q6T3DjtPO4C\\\\\\\/zIwcwE0IcdixRlMlom5P\\\\\\\/95GPXcK6w+qJlZuKsz1HVhQTum09Li3DCeBiCYXzjCagZWhb+oQ1lz18k\\\\\\\/kjqG17UTKO3ZaUayyzD4F2Q8ZDuYO6\\\\\\\/\\\\\\\/JJm3MXff8Q\\\\\\\/6fgAbG5T+REEmUbMmbyzhYOUP1WoYwC3gGTRo5cAANhG6\\\\\\\/INhrHZSpGEXKHniTAJbqWWX6Pv6M5c2ED4He7X5WJ0TC2vQIUO7fJvvKgLPAo5tvXzS\\\\\\\/Q1PuDlBVNMkDfE4kcQn+vijk5Vcqwh3i5yNWKCn9pFyH5K\\\\\\\/YeXi\\\\\\\/O5LXYHbFpZSUr3lnLuLkx6O2IYf5B8\\\\\\\/889rQB9y4GKJfAohTmcTultqmqeEAiogNl4Z3EZMi+JzPJl26MVUbLoOd38s+v\\\\\\\/vO4pOXcAyUHoRH3lxt1Xl7MYDEF+QMQMo8LK8g69Q4yxSdKf3maULthxU8vXI2112ud\\\\\\\/Dx8owF8yNLbtGgfI5mMEI1gQOoIF8x1RJj5PuclYpj586CLVqg7YuYpivBCCzi2aa4NCvvzKAL58Lj8VCtlkQ9YU2zKkeqkMl0EKGj3NG1tcDvnen0xigP\\\\\\\/enniqL7\\\\\\\/7Q1wypIVDEC9CYL3ovz5tmzQJpu9hKEI8yvWU1QvWQFJDvHrTnQi5oqM\\\\\\\/d+SdTUw27cis8ooYsB1kVZXeI\\\\\\\/84Nvj9DLPHcbOZ2iBJRZFbixJMs2Bi+WPmJfm\\\\\\\/vqARk8rF7zsl7Crs1g\\\\\\\/D3e53ar0xWElxDm15CiHL6zMNtugX3GvoyDv+n1L5pvFPqwP+vHOcgjIZkjEve3G46PUK4kc5JJuBu8WnPi0S9DUIM6mwJT6942EO\\\\\\\/McmFYZD+NfOiZ5VXs0xgnwrC2EdDm7S7dW9pO36N9Dmq9Eap8Mo6Wspr2K6V0az3KOCuEw1V8j\\\\\\\/UZ\\\\\\\/N7T89c8s0mhUf3EYF3SZqi4DYiXOKJVyYEL2wqEWOnn\\\\\\\/VSuDFPLAlmbO9sXCi5kBTNVhh4Sh6ACoX2E9NkOxVdLyAJ9Alh0KfCPUSyr7DOZyxpelGUI7g8e4i3uYG1bLUDLptDhB4wrg5WDHZpbscvypZmf49OgMVZSByO7bIpJddlej4oDAkGt4m3UPenjCBQor9rky95ousRvn+Nvkao\\\\\\\/rIlKuguisgPWU7k8prTXCw5aMi76dJjwm5dQ8Yx4U+4GRbP\\\\\\\/ewyAJbMbsjloOVgYs7bd3DPSqpAqu+LoBjHwloBRwe+R0vWMwWH01kB8LXTFZ2iDTDnWfiIWKLuC0BE3MNA6Q2Cxl8k\\\\\\\/sZ29OkLq5bR8xXAz7zHNXurf0LIrL4T+7YQs189mRfPZjp6cc+LaoQpViTIBUc064LAt5itVJ9rNusOlCEehrnkT5N70anxoqn7WiKU0ZSfF5yVCrp0fEXaiPLnT1R4rkipjAW3YBURdBHoTn5IeACXaD9ymAhz\\\\\\\/reepbAh592odnM7ZyMVWi23ilSFkCVcDXpk80QZXUHyuBOtdAxkJU3beyXA67xnoV9gngSz\\\\\\\/nGbaDghPZt1wZ2h0n6CHMLyaZMBfPpgRdLMZtjaoA9KQzmo6ZxPc1DJNPt1++VNva9Q4LuqPYs2rwmYD9UMtnCN6nufrnhHqHlqVhkHpH1ba4GmWmMxjOmyfnBlXW6ZYa\\\\\\\/gY95CP5vbxFLSVuaRLt+mTGD6IGQvqhYMZ6g3otJz\\\\\\\/6gkD2kdQoUaRbvNSXOHLFEzi34zXSQ1wf5aDY7dpHTkT89Inp0rVTk6ohFWegr5XW0ZDTnMAm3a5F9aFdYSIeOYZLU5BRfI6qx6l0bnmOV7udRodkknnXlNEy2fq0g4nr8K8FvTOlBn6MjqlCA4dvLzUYh347Xp\\\\\\\/tGf10cG5P\\\\\\\/aO74IuA4G4G9xHC9vQ+p7FVnGU4Nwu69UkTV\\\\\\\/Y+A660TxdK2naiM\\\\\\\/jv54uPEhE8cRYSy9Gclqezhm81XKWdctR+Zrbk\\\\\\\/VHSJRdXCQWDwZCwNOU8zw6fwOCcuQDVYjkp8ZS+K1mWPLYBjAd8RkHcr67+Bgt5OVH1\\\\\\\/fI8RvJ0GAh3ESbJ2UKG9qAtYrWBTn1Y37KwMLP5f5W06ZPI0MpLooBCV\\\\\\\/La+qegpnMIw3Bt6Rbljth\\\\\\\/sRpLGXUzPMzOLrnJ\\\\\\\/KNzCwGYlrad+ulX7gJYmpjHKXRDajnLPJqBk4nmrWmSjqr0KfEIPSjwql4H0mCJMthG820Erm+YkjWryE9wfQJ4ShZJ8s7MqN+mMbBdcaOqFK1nGodpmRhMIPYM4\\\\\\\/tDcVgfTrGNor4Cj8Lb3acplVGnCnIqXmkNjjACeHWfCjcLXazw0A1vtEJYwpAdXb5E3iePOaDzKwCP7r8IdJ3GBZOo\\\\\\\/woEpH9LLCdutvZ\\\\\\\/kU68QX6mX42CNrev7a0o9tshjNFFMMG+z9U6ufkxbjfgBIZxL9yL5u6egOh8eQPsXKetrhcjBZPdhBWEE2TdzbJDQYwALRELHeBl5PQrNQcfQ8rIKjLU6AAv781aU23d4sgH8Wy8SvsUqrps1HfAhS3LJQEDrWN8P+cY5HEHxnQZWYnszFyCALgd\\\\\\\/RuyWhD3uhK8WtQdxTk50j2wFzy45+EXFBrGYCfTv4KZ64Ki12J7QcgX6NaHgQauIubJeS3LzIAW+B0zbvgkIOdfnSWFp32E5umXe5dWPwh0Ikr3mnRJcTR0+ReGny84IiHo6cdK5oNhUvkbGDkWxyDyKBeBAzC3H\\\\\\\/2e9sI8qo2CdrUOeQ0E3fXv0OYatEpVMnD5eck28XI7xrRhnKyqXFfZxchchzV93tJbXCsFnejYCXJod2\\\\\\\/WsXoGQm2vh36haCs8JgNDaoHzBy\\\\\\\/yAuMrpU6URxcM5M9b5W9xvOhSSO3cw85qJzDFO2TTQY9h3CvNfnC\\\\\\\/F7wkE5ScLxH7\\\\\\\/TJ1\\\\\\\/663WknNxy+iJrFIdD5Uhd6KmbBeWjXh5QxIsbQ0kFD8kLiKKMnfi7gpbixDAN0JxSHfcpMbtmChDJV9a39dYqIIQhCxrR23g7\\\\\\\/M9o+qFX6QUZ8zvbQnNUjNvaa3\\\\\\\/aa4VYB3dgB8wfvNI4cNut7hwy7qPYRpHppfU7eQcxK6+mJI\\\\\\\/T+0iNY6TE2W\\\\\\\/hc9mH0s4Z9rj49K\\\\\\\/kE+61zY56CtDEj\\\\\\\/yZ5Fe2lvIeF9pt3WIHHM2d5dzXdlAx4uUhMdEVpwi27EsyboTZQPOJtoTgKRqdEZdxZaaY5igxJFxyf7NwAqxKamFT\\\\\\\/VGHnYIJxjVbIcy1rT5fa2i6\\\\\\\/+OJbSGKrg8VahBC9ARVcvy1BX5l76AXYB6MFbB33ehl1DKiyEyMqgWEpCVCoXY9P4LFNG72VLTGYq+qddPpJRGtkNgF+FkSP1XmFQEDt3DGfb9N0zdc193UnrbokNgNaiWkZqIIjXALKcqTs5e+9qAgEyhxd\\\\\\\/X9Zjnib0XS2i+2gLQ2EMpxgRtMV1PTrcqHDj+9QZFZzv0TA2K6Mx2jf2SNngg3kjvRbKdaoqDtWPcgFS8HOH47EkuQN2643YBgVDYEDUv0cA6X69HUgtz85msmTklq6w5kLU8HZKg3coMLuaaDzuEgDi6Nuq6mkLqCNvfRC1uIxkF2KxIXj9y4rj2sgyoCk2vQH560sinILMIfsUOsnrQPZaBQFVA4Q4r1y16NpiT8XG938yB5uo34L8wi9HGpklpoo37uFxS79KYJmFjMQb8t1xQEtLG5P6\\\\\\\/77QZyKmF7e2rsqbZi0O70LdmW2Ls0Cwmn0mk\\\\\\\/vLc9WIQGRN+l5KP3fDFjb5EXaWvo9mlzD83g+aZmJzvlMn9G6HtSNu5wyLy+GJe05M1KXCNe\\\\\\\/Cx\\\\\\\/aUBnyoJ+DGLDH2qcBBdQYSRgEqBUCwWSEw9Yf3TjwrszZOX\\\\\\\/l1uQFd6lbf+bT7t8e1RPdMTWNKx2wlaxu579PTS2+TZTxtmcSNobkKU0k8XK42DEL1ZLbjfo51w5N+Qy6ZcKoF3isoNN30nUXqqKnGUaMKQUJQ7zzmhImCnwZqPdUqSP4wRsA5FjdeXMdAvsU17hhEyBSLfJ2nxSQd9px1H9E\\\\\\\/XSliLYYSowPDN8ovKv\\\\\\\/bBRx\\\\\\\/vqT\\\\\\\/sa\\\\\\\/O6wF9PPujOcAlAMGySttciozIFvsBGK9bNf0bi+BVWTvjjI5W65S5v8C27If0R6adBpJeYurIEcjh5Nvwpy7BussGg\\\\\\\/IZvo44bnOz0BEgjXVC2GEC3dU9OHgLiScvAu1oYNkch+jYKvgPNNK1l\\\\\\\/oMq66M5vJviC5EwDLjqjP9Anr9W43TRlLWXk662KB0FrT+8wV81cJt+F57r5O7G2ZgYdLPgx\\\\\\\/HiZmWu183u7LC6kivDdZ40FVo2daQ7kzanvGo4CW9F0mzA4LhVFluixkXVbI6fkiDXmQG6exAGauutjWhZO8VFRhEPrURHTQqE3mixPMWoUZcGtQvnVJP9sbOfEmoR\\\\\\\/QdTuvuM8K35yEscq8DaeIbdb8q\\\\\\\/JLdI78KoIKWaES7zCl22EO7oG1sJxf4TryB1fCTwloCbXDzSH8n0+IetdVANVthRI3m\\\\\\\/cs9P8jJoOl2xmD2qq1ElhUPAy9Uo3vguLbjbQ++nAUBtnDGiMO4qoIOwlNr\\\\\\\/NIAbDcpLhYlX+clCchqmijO86Um9n+6n71vkWRIuNFV\\\\\\\/EEACxjtzQ8atHP\\\\\\\/N0XAVTkY\\\\\\\/KJF87mrWsHlykFTaZt7weIVOsrKqVq6HLH8+PWTbRMBMQOe+ICChCGHU+HcNnA5ns8geCVBPwOHLtZ9L72p9RKLDkzlX22ayjRrztbXi1bWm09umDDZlzmKl8+nxRh5tI8VIUFVKCyvpX7DMJIopTPRjQES+uhnAlWamiOO0VCJs\\\\\\\/HepJSX6ukWEK4FsgOW+LReho0l7oG2\\\\\\\/jngnAyOx4UsqLXkRLTBdt48xslLsticnUD\\\\\\\/Lrprp891iwu1KCq71mJGrR\\\\\\\/hKmMWJblIwWN47lrJ9r1Pf57ht052ZiI4OoAWxn+cvsnNUStU+e4m3aig20QxmKkmovzaxV\\\\\\\/nCEVbtJ1ZAk7wmng9hXvyAOpsAHFGIkkjiR0DNsLb8eYWGtaUI2nvpq3VXi49K2v3UKXTTl5RMTmesIIot2od6Vt2voxGrdsqdUa7YUGuZdNomAekn0Ai\\\\\\\/aXYJ6KmLX+2je7B99ow+uDA4o6BZjpUuMbgw83ls85kqIQyZWPOFWSqKxoY6XaG7yAinJw9zdqqQf6BSh3Uy\\\\\\\/A1gmfvgX7okh4MhPofq1+CK\\\\\\\/Mx4i9NJEgg8kSQbEYWgw8FrG77dhE+8wl7ao7GAQ00+JGw70MZVwQ6Htie7s\\\\\\\/EO3FA+viTnvs8L54xaEwPlan3jexex\\\\\\\/O6NVoeoXQM+rjrvvTUWkzRXetbhUcK8ShqWTWYK6sMphKlgjQE5p5\\\\\\\/CBm7WzuJoXh3vlVVoR72FKZ2eEJ\\\\\\\/vgcTWFVzIVPSwd0hRlDFFtKCJC8NQtC13wXwNhG23MmTKwlo5nA+H9lupRhNF9jZFVjA9gDiqlIh2lR1+p7EeVOVKpnPQ0PqX6j5oSuWd8nYrqyjwCkynUbw93N1NUQu3AqtbMNf\\\\\\\/PBq8OOyU\\\\\\\/gwjuOxUO836ew395uyI2tB8Rwp726t8oZnDJARKZwdB486hHfsWOkT0eY2bLK\\\\\\\/oXR\\\\\\\/1ceq8w9gaYOj7Q\\\\\\\/uVZpFzZ7S20d0FC2X\\\\\\\/5rCAx4brsRU31b10WaUTPJ15XGuWREF4MEP+TPOmdvXEDFJx2Ak4+8ZuOeeqyY2xIhmXFEbwHyB5qiF9fDjIaRkYJGKFy\\\\\\\/JQjugPWDJD+wLlQsOTO\\\\\\\/DYnT6fCqF8Ckc6Pjb2SlNw93vhtxl109abYTsf02SuNRrwTwjSphqxdqKu2Hx2\\\\\\\/h3TkGTJbObAWJeB36J5wss2kNj7NtIXsWD9kuFQl6xbvLJHrYWfQze3RD\\\\\\\/2RWZX6gxW7gU942p9eDDgjUBmQ3LEbav31qy\\\\\\\/2gxu\\\\\\\/GbTquLAGIIDrTR3pfKHbIePCaSxYnqaK+V\\\\\\\/7fs9tVU3ymoTuM6hDP82v79twXWGeSDG5rEndROY5z0SeAdBJfAoQ8EjV\\\\\\\/I2UHYj0hl+IlXASRBgvJv17JVkwhQZZpHcekEAhL5CDasGHlG6PFxd6o2jcXJY5ZXHmvKvSp3vywOM1IL6am3BeouVLm1hedFeaEW5KHfFjCLRjRTCFHudrTH+E5qvjoa8X7HPmVO++DlKrKrtXC9+IXbu+YbMRUw2pU0iLlSI1Hl1YjVT0oVYk\\\\\\\/0J7pOD0GcO3AugEt+BXM3LEinjtmT9fMoaQqvcNE38SBte07ipilypec2tM9VYSmjuP3ntPKOzW+paOaTTrkBMGLScdHl5cM0wwgTdV0+MlU76J5Oc9GVi8Cy7X+q0jz5INm1SueNwaUDcOHsDLortYxEAlgXc952kvsDc4vtH3N53LFX\\\\\\\/wZOEhqNOTZVKnllZ8EeWVMCt92kiHgJUWeTiqGDTsAVEIOeDZG9X5LT90cK6cjGtzumKqEJsHmk1Ket31G+V6\\\\\\\/OgdzZx2wkiZlDUYm8xG0KIsk8ZhpGskACn3kthdr0uMbRzxIgB8WJp85IEFGiBJKuNBc8BUJyNjHROShsPs618lynTKI5X1bCn2lz7wPABUkkuSHzWhJYU3S2IvAHudrEAf0Xv0b8V\\\\\\\/AFZxI2Lh8ddWZKrRQ0yY0TNlDrljxfqPtMbkdLyaLSbr+teHG8obO+nAUk4b\\\\\\\/DDi69Dhc1XRwjhnDjTkxlXrhJ8ENKCyXVA4\\\\\\\/MMpRaOvGsPJm\\\\\\\/Wocq86J5USXdzZ0L2GZVlOv2KvkQ2E06Ys94+cE4ZfVpYJbQIjy7aK7ru9ZlIfiPtkG0IRG6O\\\\\\\/uZ593XQclxbf\\\\\\\/kpe\\\\\\\/esal3H1HzRthQL3cJhlzpRj\\\\\\\/wKNK1EV4eLUVXyG39ncUJmyHQzuS97zxoLtsAu7T+f7EtX4QN3jqjddRw+GHHD0eADiblIBGT6gsC4xKR366\\\\\\\/uCqFFDucL\\\\\\\/pIEmNzh+FgxTjZMIjdTyf3ez9o+48DHb+lfRtiwNRvRy8lC\\\\\\\/JLwH9Ei9kduV7XTh1k7EPLovtEISYKki9\\\\\\\/0DrEjRIfb\\\\\\\/b34pdIu6RAk92xGnPXO8w38oXSvIk\\\\\\\/zhepUkNMaCkTWkPh1cC7CejDchTtQ0GI6jNmuJ0LA8nCaJjr6pVQ0eUm2zsfPqTLaG1d+alhITWlRLrHQO4+nr2pm87BbPNy9O82H6oKFjS\\\\\\\/cXifONxcjeYqHqx6DgezWKP9\\\\\\\/KVipHtq3B6m7BhkE6mwIBrd8Tm2d\\\\\\\/w00ir9cacXALlZEfGDtKCQK6++twaHSvU0prRh+T21NcZxvOV4iWf\\\\\\\/oi50EtzM\\\\\\\/85lXc8oh9yEQvsI9BXpLh5RXu\\\\\\\/J4c3QDoXopL\\\\\\\/i4+Ur5ByQ1Or26os0dpiWLNvAmzWv\\\\\\\/+hbszTim6oGk+3jAOMtlZlZLM6I5kbhBMI3EF86kXO\\\\\\\/klChWUe3olKhnv5QygETTOaLwsGj2SqUxHE4Ko3JTIowIdZ1QlLyHVSrzOT5hnhLxzvcQ6dnXWnohWFjXl5C7oW\\\\\\\/zgVA\\\\\\\/kypVytGwE7R1gvegcfSGAv0g8VEet4XZsVT45\\\\\\\/Kaih8PNNvEipbHlYZtWa1tNE0HMEKCx10BPKS+VHxXR2InBjb7qfqMdNiZwzuu7TA1KdqDRohx26V4\\\\\\\/qAPAVgBSfKlWRczBKbipw2u+Xi5Nk\\\\\\\/B\\\\\\\/Oqhjdi6U9P38jmk9NI1m+Q7h2Ecw7VbhMCmPJuHfIFOwaIBrq5ql3JHF49Z6qf\\\\\\\/pElHaNjDrmetTZkU7a2So+PMlV71WQYWW\\\\\\\/A+y8n9V7jrec39sAq0nAg4t+44pGHHM0xoGIZzfi8xVp68oIu36YscIARL\\\\\\\/MKMxCE7bykYWJNNq213nGVEVsLLQ04nmboZElWGqVaOslAcmPEh+egr\\\\\\\/ALoPUT0mtuzM\\\\\\\/o9b7mATN3gwZKh8ecdM9T8rrQBiOU\\\\\\\/gdLl5g6nRRM1iJJsQunjNGeKwTKnq5PHBDPWaQNIxk7dxxBYgl1v6T0dDbrCd0kKe\\\\\\\/hlUS1mGm8Q3YrlwMQAPW4BrVCd6flbmj0GNYcmWz09L\\\\\\\/6KYmUNJPGmH5o8UMYI+tv0igEiV8gJ2ft9r0DbJp4KGJI2wlilxGFnzTg+tZm\\\\\\\/aQvk8RWgck5rsMEq263ysCcXVoax9Ndrs6CegYCWe7F2XSepkBAvtEZnR47VQYUhp+TCy1CxfhfTNau7b1QLiRp5hhXU0mKN4hvbk8\\\\\\\/7sPIFp71rY0mFue8IjSn79nCT8qNNNMmtDKVQSVm4uwxLYGvVyQVic+LKgl\\\\\\\/JiUZ4S5tKJ08Iab9giJNGR+3wwmp40zp1eLgGtbJ+twB+dhKxMPZmdN0\\\\\\\/N194UT5HSs8iaQfLlT3C2Lvz4GlhH3c0L75RoXyKHOD9tWBylnf0HKYBrKDLlt8GRxlza2cWe+c2P9povFn6KWbUzLip6xf5ixqIFPXxvR6VA4DGgnUjvRBqOEn6JmtBmC3SN6nRy+CdM7e4ELP\\\\\\\/HbJagPI55YONokHQCJdVIAvppPzv7gN7il0cFS4OAXEU2bNlskv8EV\\\\\\\/CeayZRoBrnUhvj4WFHcPIATqjjdhf\\\\\\\/7WGS6x35XCHBRj3D5bn+4I4x1YZ2oA+3WcuvhAMs3V7zoRfiVqjCslvudJERQydomk0LKOUdC3DwehtfrB0vmlbrY\\\\\\\/GyaueFKYsj4L31unM2Z13QxI40+tt9sVLVnMiOrpR6ny2QA7dXAbzqAh6L611W7Ug4YVCQbLQogIWVVmcB+8\\\\\\\/YzsR1wCC8LVn6F1WjMoTMiHn8LCiuGAcZDpTepDXEEhCpseISSqQeCAw6sRkia+qN4MQBdobAnaTWBGUrtUxX+mMD2SoFVJLv+HPRypFefDr1uJwCpwnL0nhSyhpOwW9XcQxGGmoK7HLIM\\\\\\\/SXjGR8VkxwFYtrsUo\\\\\\\/12MGSViXI8WGQ0Lrk\\\\\\\/hH73+qA08vrPwd609wrNCEsMYfv4FNhMoVhq6+h+\\\\\\\/c5ZFBN9BFD0Cv32KNzB179ER8+a5AaXlwvsIR5P96+1pYGipeFSfqOdhIjjNz6XngXeWFjWgFL2ZQEyqMj\\\\\\\/Sz\\\\\\\/xu5q5KY9lQHHtAoz7Upu+Wne6aVM6JRWXjSwiPJ+9NruFSr3C2FD4SPvoH1MFc5JT75lHdMHPkhUDbC2sT3jwhRPTT3yBszn2Z52jRnXiHw0qRb7dIXOTyCpT5C\\\\\\\/34pdnH3vpCQCowhm9RTOcuwgGvbLCRThngpdrwt\\\\\\\/acP3WZHlhpCmUHkgsTqNIlcBZ8dlI5FI+tujlBtg+t1UVcpCaPOhGbmPY+D9E+ihHozGcOgB2Ycjkg7AYbtL9rTVVGRYipPTbLky8rgg6vBPabyFHlb5UDNfkMo5Z5brcub2Z\\\\\\\/g7O15R8VV5kqeCTLXPcfnhQTkNftZmJa45hL\\\\\\\/GwWW4WfqGS7mszyjt1OZNiZpfeONPvlzO0gtk3uUDiP7wKlSxGHbGaWJGCNbaplbE6Nj9M1RSqTiAW2MjaPDDLQ3wSBC3JmJSl7gL5RcOAMhnpw70r5JQzSDeRu62FAoquZo\\\\\\\/BFmr8EIrl2fGH+Xk2ibufeHjaCLU2kPaqKNchatuSZfoiege5w0TxtrAFuUa7kKN0\\\\\\\/a7rWZY5xOvNh7sTQKW53PEhj7AFc93bzFr+6zxmRofie1f80VROMA6dsDiAVBBpf9a+ovWEpshw\\\\\\\/9mcVKytHGiQ7m8mhJWRoGhToOUiUD0KgpV3WRqBEgz0ovHcLg\\\\\\\/NImi6FXOdwEvAztIcrpafiy7tihMj8nfBdBYvXgNbYoC7MkV0SpSiaZu1286xMyrZXL1gzpJVybCHaHE\\\\\\\/jifX0Ms+UWmh0Zi6Bx2OSQcno\\\\\\\/mM6owsffl\\\\\\\/978OzuwJ\\\\\\\/SXDGgdsfCFmaZntjk2JbVTH\\\\\\\/zbYn06R+v4Ku4tfPd5vE6pmC9ZCRvjsJyRAtlnZfIiS5SKt9npVHgBdVoaWCZ2JXQa1HItpOmH2zmSg1LPfG\\\\\\\/SrfHd3ArY6xSLoacBtYmhJKDsJ6KPPW+C9HHj+RolHDRwW7DL\\\\\\\/8qynnIRTRtjMOxnrLnyspMMRzmnyRGV+0tOjbHu1Eg3XXLMaBAIUt18R\\\\\\\/dT5jpnY9MJ6Hvz+4NnqbTxDRmSigic1bE\\\\\\\/qZ1UxYTVz5vf+4lBpLnJazSahb93v385pb\\\\\\\/W0gngmw9lZya001YdCIODw8XOd9mMGDA\\\\\\\/mm9t47SBAUT+lV7MwB5VvEJzrSjMBF5AaZLSVv9lFOvmvcOjQOeSZiNdCsRfiX8NRVhxWWMYUmn\\\\\\\/opCT3uXLkbNG0vQH2tby7fseujcIG4L+3cg01qdBn5Ig0HdMxiLc9XMCMRHYmbfPgRoNX\\\\\\\/07dj797989ImEUX0NFeCr8IrgAfNLPoGdMGC32QH4hxcSJs7hcpzq88ydfp2AGvkxyZQ9G8SFpJ81UJN9usPwEVtg\\\\\\\/54jgZSsuke8PByt8cHiTLnJFAWOleVD5xNI3Ek4ps6411pT\\\\\\\/WFt8L\\\\\\\/8g2NTu6BsDpT9kqF+9BkJA71r6At0dBKYrezajcpLaUq8z1d0KVtDAt5637bLZzd97JCZ55Mz9SlIDAZ7wZIMlXTbjciMzFSev\\\\\\\/kJ\\\\\\\/wioHJkXQ5A7OnT+b9sDAYK5OyHb8lm\\\\\\\/SkD43o2ji6zIMXiIQRi5fifOzIrP+hCMpN4FQbeje4cJsqQJCEQ8KwYRnB8wTBGQZBkONDycSsq2FjRTpdHcWyss+t9CPhk2Z6bRg7Q9rUWQiscaY1o8RVvMP2drkBDTKmGfGExicFY05wG4A1iv2N5e3aSETWvR1JWEFIK083zKLMGnaARZP4KAlRBeQvvS0kBa\\\\\\\/q4R\\\\\\\/9Jg+p2U3Zg0vfW\\\\\\\/blMDu5pROKOe6HxQDzp7uH3VTdpek72Q+v1PObMFuqd9akDnaTYgtrs0ULoC4tqTxWkIkVvF6mi9A80rto7eL35AbeIswIxiRlKrvXaL2O\\\\\\\/bnlQQ\\\\\\\/akWc+cEIQPWjeNh7S3nG1WZTK18Jd\\\\\\\/BJYtRM6gnIwZuhuQVScqI5AphK+7Y87n3ei7HpgcpEzgezLDcKQpDqLeJO67uZHMKit7HByPQxIaPWa+t5Wdiz\\\\\\\/jVFLkgiAPRiIaxb5VUD84myYXltmaD7TeVKG\\\\\\\/sIRyExt+rp4nYmeTrIr7tjYqrQlL8S\\\\\\\/g8jRXz6zxFUvU0I8OQrnI5hWhFyU83voivUYoCiunnmZT3HRnoWjKvCLBVCRHBiORsET8mhdfvLqdHpi+XG3FXC63BPHCUdrTN1zeSHooeTe59WRyccwqgf0\\\\\\\/BmGwyR7BwaL5ETCC6TQ4XSUBY8KawSpN0o6P5IQ8jDE5v8ehs9N6jYMApMq84yIBdl8IL3matnwh\\\\\\\/JAra74F93szk2+fohF6XItXphmjEjXjzWSrnkOSswfhKauKBsvXGFwq9+Y71oRTvwyrVeipF9XxriMFggZF1zmM8tSZ2gzMznLMtraky0IQPdci68enpPhbT2WOwXXMDibU0QRse9Ido9WlgjdN+soRBk9bWmXdVhdtXt4eTfrkftilTD1FT\\\\\\\/w4aOwTe5FDIoTOnEuZFa2CfV0ittOkVSyF6P35hqLHX3qTlVh78W+PFowXyg9y5YXdmE3L6udpPPZrEnM+3Y1GimprP60XFA5s2hzg\\\\\\\/UwuY\\\\\\\/tBfAsjjAGiACcq7nM3iHicwixKoIgqBvgYtq0nctJQpqfk0jqZcYxncmPXyotle4ocoLW+SKpMGriUCg9mvfIfXtT0qJBnb\\\\\\\/vW5ykcKLQ9c1+4eCv+Y2hLxOyD0TA0NK\\\\\\\/Wm3CQvaaQFwNOoi\\\\\\\/O6MV3euJGvxnFkjZFJwaSAncpC58CbPRZrgvNH6RYypGdCrd\\\\\\\/2fSHBcXbm+NR97hL1PAC+R3ZU5nAlzRIY4tfncETBT+cSqeyKigtrwwWTDfEngLVligS4rWosMHmIZMa\\\\\\\/YEaw4FM5o0fLpkR3THAQ3xIsFVi+6r5\\\\\\\/8Y\\\\\\\/M7RxsrayWrnoYdjdTu2LTtZ4o4FbsFXTB0oUmI0H3Z0HzF5M8VZwU38+YLxYNcvPk8PR\\\\\\\/Jbeb8jYQyA4VW3rxGS0k903PYpzeNohIFp4FEWxGNchzmHvrBo3ukm4Hh\\\\\\\/M\\\\\\\/hsLcWGM3j6U4wHsIbs3LxBECkxK6Ev8UGUvP61eKATq7JviIqhyi41o53YKaa4yBzXRXrIYit7JuZVLaMzMPZ8ZwS+ylpzdX+ru6bndqB4GfMZpocmAnO73ae41JVrG61QIwiAjMK7b+6d4pgMlTzAbnr\\\\\\\/4ezSqIhLFzOoWRPlAbSh\\\\\\\/j9MWqLgpIB\\\\\\\/EAGNPBfgAD4k0hSjp5+itmJNDmMVPJF\\\\\\\/2tMwIA02CKKfKdVjRYcByhkXtZ+07CTRl14Ay4RRDq1fuiBiZxeNRlFwwV5jTfer2f7WCEQ1QSl16BlBzIhWj3PbMYsSSC12iOBZSHr+OnpKQCdkVFkei5\\\\\\\/IST8fiubpqetj8NLjibW2zh\\\\\\\/cETOSzZ7rccSMrvHhr1ggv0kq2KeN0Cx1o23OXeBFVwBj+QtO7zr+cua6ic3Il4JHf6laxj+MQa\\\\\\\/WENDTsU5\\\\\\\/4rneDWH9m2INPoz8erjh5XT0wQIpZx0Czh\\\\\\\/OpQtwJfcezI3Aeri64yI0Tc6kleJJjHIT+6xDtDOJYg00ignCLm5hryzLEu3xxNbkTvcp2vVmG5tM3Gi2r\\\\\\\/pHA1NMsFrxLV9rFANOWbz4XqhoJRqvKODgeCzo7Q4LWK1H15jDxSYNQekMuUYN8OP8nVl3hH5k40uOITJqn4eQh1PuG8kECMWEDH8DTVUDv0x1yeiyoIrQNlQeW+XxRKcIxBrvVyycYLMetY2qvV2C7i+jjJzGZBsL0ePExSYmnCz+YuHazYeXgBO1xPJt5w3aENalicbNvqFbc7tWfSEV+P3QeSHbBUaoXKgihjQElo81oO4ZDw2dHHF3IA\\\\\\\/LuN\\\\\\\/Xe7kgAjrfvyHksw2buOm8h7OC3X4p+4wl3R+Hysk5SJ+niutZDMzAoacdgelW0Nncr53q+9uFH0Xj7H0Opc1L1KXF9DiybFSgAK9Dgm8SX66k\\\\\\\/RDMG4EBvt6YA3LZkjzTexHDleyi24UNPhDHVyiJXtp4WKa7z5SK2blJK+G6Jq60kVQBzsSxi4Wyj7GwIZbRHS476K0mNzqeL3DsHY7Y1t9TaTR2+LE\\\\\\\/DhF+zMBP8909OQekL0sTBnjLeXOibSfdMW7i65QrxqbchrOhSyCCLX2FYqmBK1eddTg9iX3RYCdpsxmtmhXqymsummI09jHAxKZELOEQzOwIw0Oi6KwNDPl8G4lC4LlEyGpznS\\\\\\\/UbbR\\\\\\\/KC0bzpglFnfIKtF\\\\\\\/TMrZhvPxsc5\\\\\\\/70mWih1b64dRJ7JzMy87yOr+zF5KYHoAsj3FNoNrRVwiTeB99wn54hRO2jSQm7+wRNqqPTDbE+RyGDgWcPjyf04i4tOSBB3ESAW\\\\\\\/SfUAbFWDnNGmis4NWtO7I9RcmBQBufk8c+EJNtCrBPpyVcovdW1rVYTBfnIagWF1vb2IoTBWaTIAVIoH7PrrL3HCVAmrT5+v3D9\\\\\\\/P67Iaqbfjf2gysSptxx86oXBgB5MHcqQtpJA3HLOlteTnbDyJI2luEwcIYfS3K+TT+3ZTIrTew0Nf07XByltMojJz67mt86gUwvO+2ac1LRWpeGsLYqubMvnazo\\\\\\\/08dZJlwFjuG0BRAgkwrrj2W0Idtedjq\\\\\\\/4eSf9EwNhcm4rrW+dpZmRDrovohpZyxij\\\\\\\/KR\\\\\\\/2BP5Bi1SH2iebgg8v1klwKoX+EanvKMrIiEhzuRZR\\\\\\\/mTfGWiwb7Y7daQr8GfhViYs4GUb\\\\\\\/q2Hyv9c0OxG2oSOSVW3Am2IK1uj1Ux1rJA6Qlx3\\\\\\\/wLuQtLQc892qWdqGSvsNqVqX2zT8cpSA1JAT8+0pO4ysGQ9i7ukaYGtNi2BFey1hCkHBcrsMd7HMi2t1Rpo843ugDMP3ENSfoNDHe5CtoLaH2L7nBMVBBs+FQz7piVP5y\\\\\\\/yAgqPYky1G9b7h8JG2wz\\\\\\\/fo1Z5\\\\\\\/lIBFpkKhN74qcDEbsQ6qHA6Id+7jyajGkDVloCSGzOqoBBGSN7qzAkos64PDQFy+1F26qf9sGGUCw0jdAsZ0YXJHLWbOs7Kk1T2Y79W8wS+khBSTs\\\\\\\/9zVrEPmalw6yesTnUPJpwwPvnpIZSg3Me2C6Hwt0gh2Kh7AkWan9hjxdDq9QXyltHJwNQ2CEd63fh\\\\\\\/a4Co1oDlQ\\\\\\\/nfLOherbFDtzYYfj\\\\\\\/+oyRNgMbaCGQxUjR8stMyrsN9aphUqY0VOD1yJKMn66we6fYBNBwOGUPaphRL8jJEdiOszycguCD9S8CV9UT5AxYXB+Yzjecgt57gWV7pKcjEb9\\\\\\\/bSS5hAnTgaeRsymE788c8FIBLj33LDgpYU0grRmJvWLXcLdVhNXuRKXOuDTqkuhWsowoMIEPHiISxhziiujbqJ10wK9K0fE8RyYTDg5F+6Sw04HPvlQGeONcg\\\\\\\/FZnoRnBkmhSVT8R1jO+yOqbVb06aowI67ykdZ\\\\\\\/X\\\\\\\/niSah13U9lXW8LbPZknfNWy1BA+K73Cs17cJtP3MMt64wX2DpPfTKZwADJmRovoXXBUTcKRkWRaDe15KIvq0oEDRd8s2zGL7kXqhGaXwt1+are2Hv7lLVaN72DujrjAp+MxJWm81F4sHM6\\\\\\\/05lhcMAZ13JUu7enEkfjucUspVNNqs3flT0SuG82dO24gfYk21yyFuJ4M67CXVNIAxb0II1IBTUqkAC\\\\\\\/hSPDzd5GqIDOmXSZuY31tT9HZ5gLtW5jw86ac\\\\\\\/EciBqjXhsaWMkFF\\\\\\\/vbC1BKEvBCBDqQB7YfUi\\\\\\\/vgu4hbdi\\\\\\\/HvZQbE6flyie4syfHruJBeLFWAj0WvWYbMGBCO6hNiSX5WfvrNuKEPkAFpUbyvt4brWC5Z945gAL6McHOnYWXyujZQJIh3yumEmdVq1jvcyDy10ZYnxcQFn5+dm5smZsJruqdX7rBFS8xeyAkIK8evdQ9OBLSAc6szGJrq4agESzh7JLQHpKqXJzqzuDzXW12nins3ntNZk77ujXh3KxpV69vca7lg2jwls3xjAPbCZUhLEiGmVENbL\\\\\\\/Vmj2Ao7Undpot2FYHKjQlshGJSUo\\\\\\\/DzaP0u0j7sY9pKXQqZWxjN1QeIx2IxBMVpyjJS0JC+dWBZQgFGasqiQDfOURASFe99XuCSPyh+CxLzYeBEmSV8oU1xq\\\\\\\/gfd42LuSpohTQmLzKEBw7fuZA8YvKSs98mQdM87MzjiqRfgApYEbcGMcz89vCMiwYjUqVpegf0UNOEE0dFzcB2bqTeGW2f+eesER7yGOGB1OYwSvjoMs3iCRKfG8IrRhfW5p2tuMEk3xKTSu+fm6CkMTdsoVH\\\\\\\/MnZxIi3bU4MxoCesA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PmxPYNyVvRVOlVdr\\\\\\\/RhIrg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022166634873ee072f345d0e6ebc27b6a0c92bab157b2c2d6d05e5b48dd9b01f5ca\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nKPXYUgCwQ39oqNnJaGsURhW6YdK9pkwS1q5lOEL1rbqK3UoxIFeznGAO17iYyOdghtYUMWbUyM2c674hHDfC\\\\\\\/4JXOj9PuOdz7mp4DtQ7i9dbtd\\\\\\\/Xh\\\\\\\/qWUUUvErd65bRTLZ7lKSS3oathTWixhCuZ6RvH07Ttu77l4+5vgsWcly6VWrI8P8wlaid\\\\\\\/3SWJXrRWF\\\\\\\/fUjx89Pk4jFthTmFwp6NP91tlo0ughtpjadKLVkXtQ\\\\\\\/MZOyfHPC6zVeBk9qfdtCoo+ESeM7rrIgSzqO+D8qzDcY07admV6eDBG+UUdlj2jlOBj9yjmhk\\\\\\\/FTaHl13WXUXCo35cBu3mQ4HLw9GmPQ9SfLMbEZhNFbiN\\\\\\\/61+Xr+wjy2aJe4CkHJbagzV1WewVfg7bjNzgo9jm3K1ZN4zFrsOK7CTGKCrkvewQ4dSc85dqKrD99VvliTrpEUXVFo9L+jtuz9FfBdVOEKo4hr3UP9vy2IRPBw\\\\\\\/dhNnyUxaQG956rkZvc+G+HpitXTH56kXmCIB6c2Zeo9rhQwXM+sbzXcUlPipXc3uBifIwjwI\\\\\\\/XYGmBTNcGa9y25F74uDUyCi+CttC\\\\\\\/wBbGDcAGKWm8fH1xJh1OPZ8PCwfJpkegSrbg+UXfIeKZSYFEEuzJ93VFrhlCqMJa8jmKo+S8mQO1kMWEEyf+Q9AGNxVD+hIduRI9RAcOYBYJPFwCVUQvgyUHgkobDKG3omkljC2x0rCChe3NXo+CyMrEhhR0JoSzG7mmWT+UN9VX\\\\\\\/9VJrT3omiQGSkx2HjgQdKO\\\\\\\/3g24xadhiQcOp5iZO5O0gxZyyFsAaa3uOeApxOJpOy2ruUCpqthwW48d1HGjAjCAWDIenhvzdg8avSDY7xkuDdzNvy7DPlScK4DQ1FyReFcMuQRfUSQTsCSKM+SrDaj+UkHnko6Y+CeeDQM7TQNijRbI1w73Cjm1Q1LTw+lNNSexD0\\\\\\\/Ldt20nKR4ZMhGsjsvLFCH1Ehp+0j+ukHrgTvO0eTIxHIQaAZRTDPDYoq4pCPEXgjEwEsE7Twx4d2+HVRUQhqU26Zl5NdXsATBfItlPxP5HtEbkAn8l3ALCj7zZj96FDBQUQDKemX6HvrLatyfky7oL4g6rvNzR7bjmQCx0naxLAnVv30s2gdU51pPzDy\\\\\\\/qqClvYTOmvFsB9N2zZcJ1WVL3JVctFOFAqOfVBgzkxrbFeRjLHcdaRNUtU4hGgI4sgrSq3m4vhubep72PUXeJrWyNTfWa8Imov+zYvH40Xb\\\\\\\/6EF5d6U2tWiJSVE7fUiV\\\\\\\/dXXUtsiYy3hrIXte7gGC8IOZvOCrEXO2XWPtcQdjiH\\\\\\\/+4jMtuDEL1pmyw\\\\\\\/OAkuhMoZXB4i46u3qldGjDNp3L7DitOUPVg5ONDwisNdj4+A4VXD6XJuYw4C7jb7LfPTg4\\\\\\\/NMTvwdUjdAhgIIwYTeAeHmH5TZqmNqMs4Ip8iU7X71ItKOJPL1w5Q0fP4WJ8QhI1fKyzqzLKw+cauTxq9hFqZJWy0iSILEmDwNRZRhCWTInMTb420Mrt7zEPYaLkAJooDkgz90LUSX+2jOmpoWLdnidgQ+DnkplY8sIsgEq5jxlMT7kV13EKGZv4DGx7jxYb\\\\\\\/b0dUqpGYJwCJIe8uDBNIL7w4YbyrqmTzfTWUOB9LinqSbMbjn\\\\\\\/faJOTaOA\\\\\\\/CiFi\\\\\\\/P9vpBnGLwT\\\\\\\/FDaVntJ68DTtV3e45jDMZ0qXOr4NZhern\\\\\\\/ZOwW6mrzS2FySXwz2cJMUijp72kvbuAVP0wTfE50BWknUXIaXQ8TT5Lytn082P1lbeztyRD4Rz\\\\\\\/bIGT52AjryYT\\\\\\\/aKWSTf6\\\\\\\/l5cHoPV2++j\\\\\\\/TR4KxKxmothAj5cgoQ+LxSkRh4aets+ZjmxDTX0y4H5BI7EcG9mGclSXxeCV\\\\\\\/tPaAX3iZor42jwGkmo8Xy8Km6rWLXS9NQo5DPeDtXYVLXE99prsVcJt0W8g6rk7fLhGhC5chLNl8VWYTXRucEY2ojlxPpTiOS+Nf6hgFSyW4zz\\\\\\\/TGqCFnyvmJRtwYIKHqYQ7Tyg==\\\u0022,\\\u0022iv\\\u0022:\\\u00226d1oPNE\\\\\\\/4dYtq8+oeIMJ8Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002244ce4b8127e3c8ed244b7386aa79c091f5d5ee0885ccd99e6508bd4e0930ad34\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459753 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/73" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HXGoWi19SYFDA6MyO2gTCT6dW9382E3MCuBC0DuLci4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/73" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rOnDYW0oqUoomH5BOGMe3wUzEj5OWjdm6u14z8Q0B6s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/1/61" title="Functional Variants in DPYSL2 Sequence Increase Risk of Schizophrenia and Suggest a Link to mTOR Signaling" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/1/81" title="Cooperation of DLC1 and CDK6 Affects Breast Cancer Clinical Outcome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459753" id="node459753" data-pisa="ggg;5/1/73" data-pisa-master="ggg;g3.114.015099" data-apath="/ggg/5/1/73.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexandra Dana</span> and <span class="highwire-citation-author" data-delta="1">Tamir Tuller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">73-80;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015099 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459753" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexandra Dana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BDana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dana%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BDana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tamir Tuller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Segol School, Tel Aviv University, Ramat Aviv 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamir%2BTuller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tuller%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATamir%2BTuller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tamirtul@post.tau.ac.il" class="" data-icon-position="" data-hide-link-title="0">tamirtul@post.tau.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/1/73" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459753" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/73.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459753" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/1/73.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459753" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/1/73.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459753" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459753/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Gene translation modeling and prediction is a fundamental problem that has numerous biomedical implementations. In this work we present a novel, user-friendly tool/index for calculating the mean of the typical decoding rates that enables predicting translation elongation efficiency of protein coding genes for different tissue types, developmental stages, and experimental conditions. The suggested translation efficiency index is based on the analysis of the organism’s ribosome profiling data. This index could be used for example to predict changes in translation elongation efficiency of lowly expressed genes that usually have relatively low and/or biased ribosomal densities and protein levels measurements, or can be used for example for predicting translation efficiency of new genetically engineered genes. We demonstrate the usability of this index via the analysis of six organisms in different tissues and developmental stages. Distributable cross platform application and guideline are available for download at: <a href="http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html">http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html</a></p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/codon-decoding-rate" class="hw-term hw-article-keyword hw-article-keyword-codon-decoding-rate" rel="nofollow">codon decoding rate</a></li><li class="kwd"><a href="/keyword/codon-usage-bias" class="hw-term hw-article-keyword hw-article-keyword-codon-usage-bias" rel="nofollow">codon usage bias</a></li><li class="kwd"><a href="/keyword/gene-translation" class="hw-term hw-article-keyword hw-article-keyword-gene-translation" rel="nofollow">gene translation</a></li><li class="kwd"><a href="/keyword/ribosome-profiling" class="hw-term hw-article-keyword hw-article-keyword-ribosome-profiling" rel="nofollow">ribosome profiling</a></li><li class="kwd"><a href="/keyword/transcript-evolution" class="hw-term hw-article-keyword hw-article-keyword-transcript-evolution" rel="nofollow">transcript evolution</a></li></ul><p id="p-3">Gene translation is a fundamental intracellular process. Thus, the ability to predict gene translation elongation efficiency (<em>i.e.</em>, a gene’s translation rate) is a central challenge related to all biomedical disciplines.</p><p id="p-4">Currently, there are no direct measures of genes translation efficiency that do not include “components” of other gene expression stages, such as transcription and/or posttranslational regulatory steps (<em>e.g.</em>, mRNA degradation, protein degradation, and protein synthesis rate). For example, conventional nondirect proxies of translation efficiency include messenger RNA (mRNA) levels, protein abundance, or the normalization of the two aforementioned factors. One drawback of these proxies is the fact that they are not available for the majority of organisms in various experimental conditions and tissues. In addition, these proxies are not highly reliable for lowly expressed genes, nor they can predict the translation efficiency of new engineered genes expressed in the same cell conditions.</p><p id="p-5">Recently, ribosome profiling was suggested for measuring some aspects of the translation process at nucleotide resolution (Ribo-seq) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ingolia <em>et al.</em> 2009</a>). This method can potentially estimate the relative time ribosomes spend on the organismal mRNA molecules, at nucleotide resolution. Thus, ribosome profiles reflect <em>in vivo</em> the translation process of specific tissues and developmental stages or conditions. As a result, it was suggested to estimate the general translation efficiency of genes by calculating their mean average footprint read counts (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Ingolia <em>et al.</em> 2009</a>).</p><p id="p-6">However, resulting ribosome profiles are reliable only for highly expressed genes, thus restricting the ability of the method to accurately measure translation efficiency of the remaining of the genes or to predict translation efficiency of newly engineered genes in similar cellular conditions. For example, as can be seen in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, for <em>S. cerevisiae</em>, <em>H. sapiens</em>, and <em>M. musculus</em> only 13.9–23.7% of the genes include more than 50% positions with nonzero mapped read counts; similarly, only 8.5–11.8% of their genes include mean footprint count (FC; per nucleotide) larger than 2.</p><p id="p-7">Additional conventional approach/indexes for estimating translation efficiency are based on various measures of codon distribution/bias within the opening reading frame (ORF) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sharp and Li 1987</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wright 1990</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">dos Reis <em>et al.</em> 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fox and Erill 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sabi and Tuller 2014</a>). These indexes were found to be correlative with the protein abundance in the cell for <em>S. cerevisiae</em>, <em>E. coli</em>, and <em>C. elegans</em> (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">dos Reis <em>et al.</em> 2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tuller <em>et al.</em> 2010b</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sabi and Tuller 2014</a>). However, these indexes are not condition nor tissue specific and may not be directly related to translation but to other steps of gene expression and gene evolution (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sharp and Li 1987</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Plotkin and Kudla 2011</a>).</p><p id="p-8">In contrast to the previous suggested indexes, the mean of the typical decoding rates (MTDR) index (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dana and Tuller 2014</a>) is based on the estimation of the typical codon decoding times from Ribo-seq data, thus potentially capturing aspects of translation elongation in specific tissues, developmental stages, and/or conditions. Specifically, the MTDR index calculates the geometrical mean of the estimated typical nominal translation rates of a gene’s codons after filtering biases and phenomena such as ribosomal traffic jams and translational pauses (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Dana and Tuller 2014</a>) (see also <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and the section <em>Materials and Methods</em>), reflecting the mean typical translation elongation rate of a gene. Thus, this index could be used to predict all genes’ translation efficiency, including newly engineered genes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/73/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram showing the mean of the typical decoding rates (MTDR) calculation process. Ribo-seq data are created per selected organism (condition and tissue specific). Resulting mRNA fragments are then mapped to transcript sequences resulting for each gene its ribosomal profile. Genes with mean footprint counts (FC) lower than one are filtered. To enable comparison between footprint counts of different genes with different mRNA levels and initiation rates, FC of each gene are normalized, resulting in normalized footprint count (NFC) profiles. Then for each codon type a NFC distribution is created. By fitting the NFC distribution to an exponentially modified Gaussian distribution type, the typical decoding rate of each codon is estimated. Finally, for each one of the input target open reading frames (ORFs), the mean typical decoding rates of its codons (MTDR index) is calculated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1060988701" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diagram showing the mean of the typical decoding rates (MTDR) calculation process. Ribo-seq data are created per selected organism (condition and tissue specific). Resulting mRNA fragments are then mapped to transcript sequences resulting for each gene its ribosomal profile. Genes with mean footprint counts (FC) lower than one are filtered. To enable comparison between footprint counts of different genes with different mRNA levels and initiation rates, FC of each gene are normalized, resulting in normalized footprint count (NFC) profiles. Then for each codon type a NFC distribution is created. By fitting the NFC distribution to an exponentially modified Gaussian distribution type, the typical decoding rate of each codon is estimated. Finally, for each one of the input target open reading frames (ORFs), the mean typical decoding rates of its codons (MTDR index) is calculated.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/73/F1.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/1/73/F1.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/73/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/73/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465564" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Diagram showing the mean of the typical decoding rates (MTDR) calculation process. Ribo-seq data are created per selected organism (condition and tissue specific). Resulting mRNA fragments are then mapped to transcript sequences resulting for each gene its ribosomal profile. Genes with mean footprint counts (FC) lower than one are filtered. To enable comparison between footprint counts of different genes with different mRNA levels and initiation rates, FC of each gene are normalized, resulting in normalized footprint count (NFC) profiles. Then for each codon type a NFC distribution is created. By fitting the NFC distribution to an exponentially modified Gaussian distribution type, the typical decoding rate of each codon is estimated. Finally, for each one of the input target open reading frames (ORFs), the mean typical decoding rates of its codons (MTDR index) is calculated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">In this study we estimate the typical codons translation time of additional organisms in different conditions and tissues (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) from ribosomal profiling data (see the section <em>Materials and Methods</em>). We also demonstrate the advantages of the MTDR index in predicting various gene expression measurements and compare it with previously suggested translation efficiency indexes (which are based on codon distributions). Finally, we provide a cross-platform tool for calculating the MTDR index of all ORFs in these organisms and conditions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465548/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465548/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Analyzed organisms and tissues/conditions</span><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Calculating the normalized footprint count (NFC) distribution</h3><p id="p-11">As seen in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, the majority of genes’ ribosome profiles have less than 50% of codons mapped with read counts. Therefore, to avoid analyzing unreliable ribosome profiles that could biases estimations, only genes with a median FC greater than one were included in the analysis (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Dana and Tuller 2014</a>). In addition, previous studies indicated an increase of FC at the beginning of the ORF (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ingolia 2010</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ingolia <em>et al.</em> 2011</a>) and for some organisms at the end of ORF (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>); therefore, the first and last 20 codons were excluded from the analysis. Moreover, to prevent analysis of unreliable reads, codons with FC values less than one were excluded from the analysis (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/73/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Footprint count (FC) statistics over all genes of six organisms. (A) Histogram of average read counts. (B) Histogram of percentage of positions with a positive number of mapped read counts. As can be seen in all analyzed organisms, most of the genes have very low read counts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1060988701" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Footprint count (FC) statistics over all genes of six organisms. (A) Histogram of average read counts. (B) Histogram of percentage of positions with a positive number of mapped read counts. As can be seen in all analyzed organisms, most of the genes have very low read counts.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/73/F2.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/1/73/F2.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/73/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/73/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465573" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-12" class="first-child">Footprint count (FC) statistics over all genes of six organisms. (A) Histogram of average read counts. (B) Histogram of percentage of positions with a positive number of mapped read counts. As can be seen in all analyzed organisms, most of the genes have very low read counts.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">To enable comparison of footprint counts of a codon type from genes with different mRNA levels and initiation rates, FC of each codon were first normalized by the average FC of each gene (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Qian <em>et al.</em> 2012</a>; <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Dana and Tuller 2014</a>), resulting in NFC. This normalization enables measuring the relative time a ribosome spends translating each codon in a specific gene relative to other codons in it, while considering the total number of codons in the gene. Then, for each codon type a vector consisting of NFC values originating from all analyzed genes was generated, creating the “NFC distribution” of a codon.</p></div><div id="sec-3" class="subsection"><h3>Estimating the codons’ typical decoding time</h3><p id="p-14">Based on the characteristics of the NFC distributions, we suggest that their topology could result from a superposition of two distributions/components (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Dana and Tuller 2014</a>): the first one describes the “typical” decoding time of the ribosomes, which was modeled by a normal distribution characterized by its mean μ and variance <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-1.gif"/></noscript></span></span>.</p><p id="p-15">The second component describes relatively rare translational pauses and ribosomal interactions such as traffic jams due to the codons’ different translation efficiency and was modeled by a random variable with an exponentially distribution, characterized by one parameter λ.</p><p id="p-16">The summation of two independent normal and exponential random variables corresponding to the distributions mentioned above results in an exponentially modified Gaussian distribution. The parameters <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-2.gif"/></noscript></span></span> were estimated by fitting the measured NFC distributions to the exponentially modified Gaussian distribution, under the log-likelihood criterion. The μ parameter is referred as the typical decoding time of a codon. For more details, see <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Dana and Tuller (2014)</a>. Then, the MTDR index of a gene was defined as the geometrical mean of its codons translation rates.</p></div><div id="sec-4" class="subsection"><h3>Protein abundance and mRNA levels measurements</h3><p id="p-17"><em>E. coli</em> protein abundance measurements were downloaded from the PRIDE database (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vizcaino <em>et al.</em> 2013</a>), and mRNA level measurements were taken from another source (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Taniguchi <em>et al.</em> 2010</a>). <em>C. elegans</em> mRNA levels were downloaded from <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kirienko and Fay (2007)</a>. <em>B</em>. <em>subtilis</em> protein abundance measures were used as published by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chi <em>et al.</em> (2011)</a>, and mRNA levels were downloaded from another source (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Nicolas <em>et al.</em> 2012</a>). <em>S. cerevisiae</em> protein abundance measures were averaged from four quantitative large-scale measurements: two large scale measurements in two conditions (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Newman <em>et al.</em> 2006</a>), and a large-scale protein abundance measurement from two sources (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lee <em>et al.</em> 2011</a>). mRNA levels were determined by averaging large scale measurements of mRNA levels (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2002</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Ingolia <em>et al.</em> 2009</a>).</p></div><div id="sec-5" class="subsection"><h3>Estimating the contribution of the MTDR index and other translation efficiency indexes to predicting protein abundance</h3><p id="p-18">To estimate whether the MTDR index improves the prediction protein levels relative to other translation efficiency indexes, we have modeled the relationship between protein abundance and the different translation efficiency indexes MTDR, transfer RNA (tRNA) adaptation index [tAI (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">dos Reis and Wernisch 2009</a>)], and codon adaptation index [CAI (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Sharp and Li 1987</a>)], using a linear regressor (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Seber and Lee 2012</a>).<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-3.gif"/></noscript></span></span>where the coefficients <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-4.gif"/></noscript></span></span> were estimated to minimize the mean square error difference between the prediction vector <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-5.gif"/></noscript></span></span> and the real protein abundance measurements <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-6.gif"/></noscript></span></span>.</p><p id="p-19">For each one of the coefficients <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-7.gif"/></noscript></span></span> confidence intervals (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kendall and Stuart 1979</a>) were calculated to determine the reliability of the estimates (at 95% confidence interval). A coefficient’s confidence interval that does not contain zeroes implies that it significantly contributes to the regression, <em>i.e.</em>, the coefficient is not zero (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kendall and Stuart 1979</a>).</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>The estimated codons decoding times correlate with measures of codon usage bias</h3><p id="p-20">We started our analysis by comparing the typical decoding times estimated from the ribosome profiling data (µ) to other estimations of codon translation efficiency estimated using additional methods which are based on codon distributions. These include: 1) <strong>d</strong>ecoding <strong>t</strong>ime based on the <strong>c</strong>odon <strong>a</strong>daptation <strong>i</strong>ndex (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-8.gif"/></noscript></span></span>), which calculates the codon bias in highly expressed genes (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Sharp and Li 1987</a>) and 2) <strong>d</strong>ecoding <strong>t</strong>ime based on the <strong>t</strong>RNA <strong>a</strong>daptation <strong>i</strong>ndex (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-9.gif"/></noscript></span></span>) (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">dos Reis <em>et al.</em> 2004</a>), which takes into consideration the number of tRNA copies in the genome recognizing each codon and additional codon/anticodon interactions. As seen in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, the correlation between µ values and the other codon decoding time estimators is significant in all analyzed organisms (0.42 &lt; <em>P</em> &lt; 0.83; <em>P</em> &lt; 0.00065).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465531/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465531/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Spearman correlation between µ and various methods for estimating the typical codon decoding time</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">In addition, we also compared the <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-12.gif"/></noscript></span></span> values to decoding times measured by using various experimental methods. For example, the amino acid insertion time per codon in <em>E. coli</em> was estimated using known biochemistry factors (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fluitt <em>et al.</em> 2007</a>). Spearman correlation between insertion time and µ was 0.43 (<em>P</em> = 0.00051), whereas the correlation between insertion time and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-13.gif"/></noscript></span></span> was lower: 0.35/0.31 (<em>P</em> = 0.0062/0.017), supporting the conjecture that µ values are better direct estimators of the amino acid insertion time. In another study (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> 2014</a>) the authors found that for <em>S. cerevisiae</em> the GAG codon is more slowly translated than the GAA codon. This result is also supported by the µ estimations (GAG: 0.24 <em>vs.</em> GAA: 0.21). The same trend was also observed for the <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-14.gif"/></noscript></span></span> values (GAG: 0.016 <em>vs.</em> GAA: 0.006) and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/1/73/embed/mml-math-15.gif"/></noscript></span></span> values (GAG: 2.44 <em>vs.</em> GAA: 1.1). In another study (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kemp <em>et al.</em> 2013</a>) it was shown that in <em>S. cerevisiae</em>, replacing codons the codons “CAA” by the codons “CAG,” which are decoded by a rare tRNA, near the 5′ end of an ORF reduces luciferase expression by 60%. Indeed the estimated µ decoding time of the CAG codon was found to be greater than of the CAA codon (0.25 <em>vs.</em> 0.15). An additional work (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Letzring <em>et al.</em> 2010</a>) studied the effect of synonymous codon repeats of luciferase expression in <em>S. cerevisiae</em>; to validate our µ estimations, we calculated Spearman correlation between translation efficiency changes with respect to the wild type and the estimated µ values and found them to be significantly correlative (r = −0.33, <em>P</em> = 0.013).</p></div><div id="sec-8" class="subsection"><h3>The MTDR index predicts protein levels in standard conditions with comparable quality as other codon bias measures</h3><p id="p-22">The usability of the suggested MTDR index was validated for four different organisms that have abundant large scale protein and mRNA levels measurements (<em>E. coli</em>, <em>B. subtilis</em>, <em>C. elegans</em>, and <em>S. cerevisiae)</em>. Spearman correlation between the MTDR index to various proxies of translation efficiency such as protein abundance resulted in significant correlations (0.32&lt; r &lt; 0.44; <em>P</em> &lt; 3.8*10<sup>−87</sup>; see also <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Similarly, a significant partial Spearman correlation between protein abundance and MTDR index given mRNA levels (partial correlation) was observed (0.23 &lt; r &lt; 0.5; <em>P</em> &lt; 4.2*10<sup>−86</sup>; see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/73/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein abundance (PA) vs. mean of the typical decoding rates (MTDR) and Spearman correlation between MTDR index and protein abundance for various organisms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1060988701" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Protein abundance (PA) &lt;em&gt;vs.&lt;/em&gt; mean of the typical decoding rates (MTDR) and Spearman correlation between MTDR index and protein abundance for various organisms.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/73/F3.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/1/73/F3.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/73/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/73/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465557" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Protein abundance (PA) <em>vs.</em> mean of the typical decoding rates (MTDR) and Spearman correlation between MTDR index and protein abundance for various organisms.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465497/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465497/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Spearman correlation between protein abundance (also when controlling for mRNA levels) and various translation efficiency indexes</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">Measures based on codon usage bias resulted in similar correlations with protein levels in standard conditions (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>); these results can be explained among others, by the fact that codon bias usage measures various (direct and indirect) aspects of gene expression (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chamary <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hershberg and Petrov 2008</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Plotkin and Kudla 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sauna and Kimchi-Sarfaty 2011</a>) (Tuller and Zur, unpublished data), by the fact that ribosome profiling data are probably “noisier” and more biased than genomic sequence data (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dana and Tuller 2012</a>; Gerashchenko and Gladyshev 2014), and by the fact that in this study we consider protein levels in “standard”/exponential conditions which are probably reflected more properly by codon bias usage.</p><p id="p-26">For some of the analyzed organisms, a decrease in correlation between the different translation efficiency indexes and protein abundance was observed when controlling for mRNA levels. As genes with greater mRNA levels potentially consume a greater percentage of the ribosomes in the cell, they are expected to undergo stronger selection forgreater translation elongation speed (and/or other aspects of translation efficiency) to reduce ribosome utilization (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tuller <em>et al.</em> 2010a</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Plotkin and Kudla 2011</a>). Therefore we expect to see a positive correlation between measures of translation efficiency and mRNA levels (and not only with protein abundance); as a result, the partial correlation between protein levels and translation efficiency indexes decreases when controlling for mRNA levels.</p><p id="p-27">Because the correlation between the MTDR index and protein abundance was found to be similar to the correlation between protein abundance and the other two translation efficiency indexes, we wanted to assess whether the MTDR index provides <em>additional</em> information with respect to the other two. To this end, we calculated a linear regressor based on tAI, CAI, and MTDR for predicting protein abundance levels (see the section <em>Materials and Methods</em>). Then, we checked the 95% confidence intervals of each one of the translation efficiency indexes and found that they do not include the value 0 for the variable MTDR (see the section <em>Materials and Methods</em>), demonstrating that the MTDR index contributes additional information to protein abundance prediction given the other two indexes.</p><p id="p-28">Finally, it should be noted that in endogenous genes it is impossible to prove the direction of causality between the various translation efficiency indexes and protein abundance based on correlation; a correlation may suggest that 1) codons with greater µ values or codons that are recognized by more abundant tRNA molecule tend to improve translation rate and thus increase protein levels (<em>e.g.</em>, <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Letzring <em>et al.</em> 2010</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Tuller <em>et al.</em> 2010b</a>); 2) genes with greater expression levels are selected to have codons with a greater MTDR/tAI index due to reasons not directly related, increasing the number of proteins per mRNA [for example, global ribosomal allocation (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kudla <em>et al.</em> 2009</a>)].</p></div><div id="sec-9" class="subsection"><h3>Using the MTDR index for predicting ribosomal load</h3><p id="p-29">To demonstrate that the MTDR index can predict ribosomal densities of each one of the analyzed organisms we used 50% of the highly expressed genes to compute the typical codon decoding times. Next, we computed the MTDR index of the genes in the rest 50% of the highly expressed genes (that were not used for computing the typical decoding times). Finally, we computed the correlation between the MTDR index and the actual mean read count of these genes. As can be seen in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, in almost all analyzed organisms/conditions the correlations are positive and significant (the top correlation is 0.8, <em>P</em> = 1.4*10<sup>−131</sup>). This result demonstrates that the MTDR index could be used as a good predictor of the ribosome load (number of ribosomes per mRNA * number of mRNA molecules), probably since highly translated genes tend to undergo selection for greater codon elongation rate, for example, to improve ribosomal allocation and translation cost (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kudla <em>et al.</em> 2009</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Tuller <em>et al.</em> 2010a</a>). For comparison, we also calculated the correlation between the tAI and CAI indexes and ribosomal load (see <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). These indexes resulted in similar correlations for the different “typical”/exponential stage conditions, however, produced a lower correlation in atypical conditions such as the different <em>S. cerevisiae</em> meiosis stages and for some <em>M. musculus</em> tissue types.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/73/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ribosomal load, as estimated by the mean footprint counts per gene vs. mean of the typical decoding rates (MTDR) index for various organisms and Spearman correlation between these factors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1060988701" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ribosomal load, as estimated by the mean footprint counts per gene &lt;em&gt;vs.&lt;/em&gt; mean of the typical decoding rates (MTDR) index for various organisms and Spearman correlation between these factors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/73/F4.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/1/73/F4.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/73/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/73/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465561" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Ribosomal load, as estimated by the mean footprint counts per gene <em>vs.</em> mean of the typical decoding rates (MTDR) index for various organisms and Spearman correlation between these factors.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465571/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465571/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Correlation between MTDR index and ribosomal load for the different tissues/conditions of the analyzed organisms</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">It should be mentioned that a decrease in this correlation was observed for non-exponential stages in <em>S. cerevisiae</em> and in greater eukaryotes such as <em>H. sapiens</em> and <em>M musculus</em>. This could result from: 1) a greater level of noise (see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) and biases caused by additional and/or more complicated biological mechanisms (<em>e.g.</em>, due substantial alternative splicing in mammals the mapping of reads to exons is less trivial) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Engstrom <em>et al.</em> 2013</a>); 2) the fact that mammals have smaller effective population size and thus lower selection pressure related to some translation aspects (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Charlesworth 2009</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">dos Reis and Wernisch 2009</a>); and 3) since growth rate is strongly related to fitness in unicellular organisms but not in mammals, there is lower effect on ribosomal allocation on the organisms fitness in mammals than in the rest of the analyzed organism (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rocha 2004</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">dos Reis and Wernisch 2009</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/73/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Footprint count (FC) statistics for S. cerevisiae different meiosis stages. (A) Histogram of average read counts. (B) Histogram of percentage of positions with a positive number of mapped read counts. As can be seen in all of the organisms, most of the genes have very low read counts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1060988701" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Footprint count (FC) statistics for &lt;em&gt;S. cerevisiae&lt;/em&gt; different meiosis stages. (A) Histogram of average read counts. (B) Histogram of percentage of positions with a positive number of mapped read counts. As can be seen in all of the organisms, most of the genes have very low read counts.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/73/F5.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/1/73/F5.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/73/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/73/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465547" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Footprint count (FC) statistics for <em>S. cerevisiae</em> different meiosis stages. (A) Histogram of average read counts. (B) Histogram of percentage of positions with a positive number of mapped read counts. As can be seen in all of the organisms, most of the genes have very low read counts.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Assessing the ability of the MTDR index to predict translation efficiency in different experimental conditions</h3><p id="p-34">Codon bias indexes (<em>e.g.</em>, CAI, tAI) are based solely on static information encoded in the genome or general chemical properties; thus, they cannot differentiate among different experimental conditions. One of the major advantages of the MTDR index is that it is condition specific. To demonstrate this advantage, we estimated the typical decoding times of <em>S. cerevisiae</em> in starvation conditions (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Ingolia <em>et al.</em> 2009</a>). Translation efficiency (defined as protein abundance normalized by mRNA levels) was calculated for rich (yeast extract peptone dextrose) and minimal (synthetic defined) media [protein abundance and mRNA levels measured per cell were taken from a previous study (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Newman <em>et al.</em> 2006</a>)].</p><p id="p-35">We found that the MTDR values of the genes with top/bottom 30% translation efficiency ratios are significantly different (<em>t</em>-test: <em>P</em> = 0.0081; Wilcoxon test: <em>P</em> = 0.021): genes with a greater change in their translation efficiency have a greater change in their estimated MTDR index (in the same direction). This result demonstrates the ability of the suggested index to estimate gene expression in different conditions (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/1/73/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The ration between mean of the typical decoding rates (MTDR) index calculated in starvation (synthetic defined [SD]) and rich medium (yeast extract peptone dextrose [YEPD]) for genes with bottom/up 30% PA/mRNA ratio in SD/YEPD conditions (green/red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1060988701" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The ration between mean of the typical decoding rates (MTDR) index calculated in starvation (synthetic defined [SD]) and rich medium (yeast extract peptone dextrose [YEPD]) for genes with bottom/up 30% PA/mRNA ratio in SD/YEPD conditions (green/red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/1/73/F6.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/1/73/F6.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/1/73/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/1/73/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465570" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-36" class="first-child">The ration between mean of the typical decoding rates (MTDR) index calculated in starvation (synthetic defined [SD]) and rich medium (yeast extract peptone dextrose [YEPD]) for genes with bottom/up 30% PA/mRNA ratio in SD/YEPD conditions (green/red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">In addition, we also found that Spearman correlation between the MTDR ratio and ribosomal load ratio for <em>different tissue/developmental conditions</em> in <em>C. elegans</em>, <em>S. cerevisiae</em>, and <em>M. musculus</em> (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>) was significantly positive, (0.17 &lt; r &lt; 0.72; <em>P</em> &lt; 0.0003), indicating that the MTDR index could predict changes in ribosomal load, contrary to the static tAI and CAI indexes.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465504/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465504/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Spearman correlation between MTDR index ratios and ribosomal load ratio for different tissues/developmental stages in <em>C. elegans</em>, <em>S. cerevisiae</em> (partial list) and <em>M. musculus</em></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section other1" id="sec-11"><h2 class="">Application</h2><p id="p-38">The MTDR application enables calculating the translation efficiency of various genes according to their ORF sequence. The input of the application includes the selection of one of the organisms and its tissue/condition and a file containing the ORFs of the requested genes (in FASTA format or text format where each ORF is defined in a separate line). ORFs could also be straightforwardly inserted in a textbox. The application returns an output file which includes for each of the ORFs their MTDR index. The available organisms and conditions/tissues are depicted in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. The distributable cross platform application and user manual are available for download at:</p><p id="p-39"><a href="http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html">http://www.cs.tau.ac.il/~tamirtul/MTDR/MTDR_Install.html</a></p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This research is partially supported by research grants from the German-Israeli Foundation (GIF) and Israel Cancer Research Fund (ICRF).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-41"><em>Communicating editor: A. M. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-21"><span class="received-label">Received </span>October 21, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-11-13"><span class="accepted-label">Accepted </span>November 13, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Dana and Tuller</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.1.73.1" data-doi="10.1126/science.1215110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution view of the yeast meiotic program revealed by ribosome profiling,</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>, <span class="cit-fpage">552</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1215110%26rft_id%253Dinfo%253Apmid%252F22194413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2OC81NTIiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy81LzEvNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.2" data-doi="10.1038/nrg1770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChamary%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D98%26rft.epage%253D108%26rft.atitle%253DHearing%2Bsilence%253A%2Bnon-neutral%2Bevolution%2Bat%2Bsynonymous%2Bsites%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1770%26rft_id%253Dinfo%253Apmid%252F16418745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418745&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.3" data-doi="10.1038/nrg2526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fundamental concepts in genetics: effective population size and patterns of molecular evolution and variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DFundamental%2Bconcepts%2Bin%2Bgenetics%253A%2Beffective%2Bpopulation%2Bsize%2Band%2Bpatterns%2Bof%2Bmolecular%2Bevolution%2Band%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2526%26rft_id%253Dinfo%253Apmid%252F19204717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19204717&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263457000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-5.1.73.4" data-doi="10.1074/mcp.M111.009506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chi</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gronau</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mader</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hessling</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">S-bacillithiolation protects against hypochlorite stress in Bacillus subtilis as revealed by transcriptomics and redox proteomics.</span> <span class="cit-source">Mol. Cell Proteomics</span> <span class="cit-vol">10</span>: <span class="cit-fpage">M111 009506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%2BProteomics%26rft.volume%253D10%26rft.spage%253DM111009506%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M111.009506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1074/mcp.M111.009506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.5" data-doi="10.1002/embj.201385651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazana</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellanger</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuite</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Translation elongation can control translation initiation on eukaryotic mRNAs.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fembj.201385651%26rft_id%253Dinfo%253Apmid%252F24357599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo3OiIzMy8xLzIxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.6" data-doi="10.1371/journal.pcbi.1002755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dana</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Determinants of translation elongation speed and ribosomal profiling biases in mouse embryonic stem cells.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DDana%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1002755%26rft.epage%253De1002755%26rft.atitle%253DDeterminants%2Bof%2Btranslation%2Belongation%2Bspeed%2Band%2Bribosomal%2Bprofiling%2Bbiases%2Bin%2Bmouse%2Bembryonic%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002755%26rft_id%253Dinfo%253Apmid%252F23133360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23133360&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.7" data-doi="10.1093/nar/gku646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dana</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effect of tRNA levels on decoding times of mRNA codons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">9171</span>–<span class="cit-lpage">9181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku646%26rft_id%253Dinfo%253Apmid%252F25056313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi8xNC85MTcxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.8" data-doi="10.1093/nar/gkh834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Reis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savva</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernisch</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Solving the riddle of codon usage preferences: a test for translational selection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">5036</span>–<span class="cit-lpage">5044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DReis%26rft.auinit1%253DM.%2Bd.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D5036%26rft.epage%253D5044%26rft.atitle%253DSolving%2Bthe%2Briddle%2Bof%2Bcodon%2Busage%2Bpreferences%253A%2Ba%2Btest%2Bfor%2Btranslational%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh834%26rft_id%253Dinfo%253Apmid%252F15448185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNy81MDM2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.9" data-doi="10.1093/molbev/msn272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Reis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernisch</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating translational selection in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn272%26rft_id%253Dinfo%253Apmid%252F19033257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI2LzIvNDUxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.10" data-doi="10.1038/nmeth.2722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engstrom</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steijger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sipos</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahles</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Systematic evaluation of spliced alignment programs for RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1185%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2722%26rft_id%253Dinfo%253Apmid%252F24185836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24185836&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327698100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.11" data-doi="10.1016/j.compbiolchem.2007.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fluitt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pienaar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viljoen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Ribosome kinetics and aa-tRNA competition determine rate and fidelity of peptide synthesis.</span> <abbr class="cit-jnl-abbrev">Comput. Biol. Chem.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2Bbiology%2Band%2Bchemistry%26rft.stitle%253DComput%2BBiol%2BChem%26rft.aulast%253DFluitt%26rft.auinit1%253DA.%26rft.volume%253D31%26rft.issue%253D5-6%26rft.spage%253D335%26rft.epage%253D346%26rft.atitle%253DRibosome%2Bkinetics%2Band%2Baa-tRNA%2Bcompetition%2Bdetermine%2Brate%2Band%2Bfidelity%2Bof%2Bpeptide%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compbiolchem.2007.07.003%26rft_id%253Dinfo%253Apmid%252F17897886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compbiolchem.2007.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17897886&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250912400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.12" data-doi="10.1093/dnares/dsq012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erill</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Relative codon adaptation: a generic codon bias index for prediction of gene expression.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsq012%26rft_id%253Dinfo%253Apmid%252F20453079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjE3LzMvMTg1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.1.73.13" data-doi="10.1093/nar/gku671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerashchenko</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gladyshev</span>  <span class="cit-name-given-names">V. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Translation inhibitors cause abnormalities in ribosome profiling experiments.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">e134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D42%26rft.spage%253De134%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.14" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.1.73.15" data-doi="10.1038/nature09267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mammalian microRNAs predominantly act to decrease target mRNA levels,</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">466</span>, <span class="cit-fpage">835</span>–<span class="cit-lpage">840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGuo%26rft.auinit1%253DH.%26rft.volume%253D466%26rft.issue%253D7308%26rft.spage%253D835%26rft.epage%253D840%26rft.atitle%253DMammalian%2BmicroRNAs%2Bpredominantly%2Bact%2Bto%2Bdecrease%2Btarget%2BmRNA%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09267%26rft_id%253Dinfo%253Apmid%252F20703300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20703300&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280766100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.16" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.17" data-doi="10.1016/S0076-6879(10)70006-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide translational profiling by ribosome footprinting.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DIngolia%26rft.auinit1%253DN.%2BT.%26rft.volume%253D470%26rft.spage%253D119%26rft.epage%253D142%26rft.atitle%253DGenome-wide%2Btranslational%2Bprofiling%2Bby%2Bribosome%2Bfootprinting.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252810%252970006-9%26rft_id%253Dinfo%253Apmid%252F20946809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(10)70006-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20946809&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275827900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.18" data-doi="10.1126/science.1168978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy81LzEvNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-2" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.19" data-doi="10.1016/j.cell.2011.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lareau</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DIngolia%26rft.auinit1%253DN.%2BT.%26rft.volume%253D147%26rft.issue%253D4%26rft.spage%253D789%26rft.epage%253D802%26rft.atitle%253DRibosome%2Bprofiling%2Bof%2Bmouse%2Bembryonic%2Bstem%2Bcells%2Breveals%2Bthe%2Bcomplexity%2Band%2Bdynamics%2Bof%2Bmammalian%2Bproteomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.10.002%26rft_id%253Dinfo%253Apmid%252F22056041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22056041&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296902300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.20" data-doi="10.1111/mmi.12096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betney</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciandrini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwenger</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A yeast tRNA mutant that causes pseudohyphal growth exhibits reduced rates of CAG codon translation.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol.%26rft.volume%253D87%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.12096%26rft_id%253Dinfo%253Apmid%252F23146061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.12096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23146061&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-5.1.73.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kendall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-source">The Advanced Theory of Statistics. Vol. 2, Ed. 1. Inference and Relationship.</span> <span class="cit-publ-name">Griffin</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.22" data-doi="10.1016/j.ydbio.2007.02.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirienko</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transcriptome profiling of the <em>C. elegans</em> Rb ortholog reveals diverse developmental roles.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">674</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DKirienko%26rft.auinit1%253DN.%2BV.%26rft.volume%253D305%26rft.issue%253D2%26rft.spage%253D674%26rft.epage%253D684%26rft.atitle%253DTranscriptome%2Bprofiling%2Bof%2Bthe%2BC.%2Belegans%2BRb%2Bortholog%2Breveals%2Bdiverse%2Bdevelopmental%2Broles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2007.02.021%26rft_id%253Dinfo%253Apmid%252F17368442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2007.02.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17368442&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246461000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.23" data-doi="10.1126/science.1170160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D324%26rft.issue%253D5924%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DCoding-Sequence%2BDeterminants%2Bof%2BGene%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170160%26rft_id%253Dinfo%253Apmid%252F19359587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yNTUiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy81LzEvNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.24" data-doi="10.1038/msb.2011.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topper</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubler</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hose</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dynamic model of proteome changes reveals new roles for transcript alteration in yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D7%26rft.spage%253D514%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.48%26rft_id%253Dinfo%253Apmid%252F21772262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2011.48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21772262&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.1.73.25" data-doi="10.1073/pnas.1207846109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global mapping of translation initiation sites in mammalian cells at single-nucleotide resolution,</span> <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">E2424</span>–<span class="cit-lpage">E2432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1207846109%26rft_id%253Dinfo%253Apmid%252F22927429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM3L0UyNDI0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.26" data-doi="10.1261/rna.2411710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Letzring</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grayhack</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Control of translation efficiency in yeast by codon-anticodon interactions.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2516</span>–<span class="cit-lpage">2528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2411710%26rft_id%253Dinfo%253Apmid%252F20971810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMi8yNTE2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-3" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.27" data-doi="10.1038/nature10965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DG.%2BW.%26rft.volume%253D484%26rft.issue%253D7395%26rft.spage%253D538%26rft.epage%253D541%26rft.atitle%253DThe%2Banti-Shine-Dalgarno%2Bsequence%2Bdrives%2Btranslational%2Bpausing%2Band%2Bcodon%2Bchoice%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10965%26rft_id%253Dinfo%253Apmid%252F22456704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22456704&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303200400056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.28" data-doi="10.1038/nature04785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ihmels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breslow</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">441</span>: <span class="cit-fpage">840</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNewman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D441%26rft.issue%253D7095%26rft.spage%253D840%26rft.epage%253D846%26rft.atitle%253DSingle-cell%2Bproteomic%2Banalysis%2Bof%2BS.%2Bcerevisiae%2Breveals%2Bthe%2Barchitecture%2Bof%2Bbiological%2Bnoise.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04785%26rft_id%253Dinfo%253Apmid%252F16699522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16699522&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238254100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.29" data-doi="10.1126/science.1206848"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mader</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dervyn</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rochat</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leduc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Condition-dependent transcriptome reveals high-level regulatory architecture in <em>Bacillus subtilis</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">1103</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNicolas%26rft.auinit1%253DP.%26rft.volume%253D335%26rft.issue%253D6072%26rft.spage%253D1103%26rft.epage%253D1106%26rft.atitle%253DCondition-Dependent%2BTranscriptome%2BReveals%2BHigh-Level%2BRegulatory%2BArchitecture%2Bin%2BBacillus%2Bsubtilis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1206848%26rft_id%253Dinfo%253Apmid%252F22383849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzUvNjA3Mi8xMTAzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.30" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.31" data-doi="10.1371/journal.pgen.1002603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maclean</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Balanced codon usage optimizes eukaryotic translational efficiency.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DQian%26rft.auinit1%253DW.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002603%26rft.epage%253De1002603%26rft.atitle%253DBalanced%2Bcodon%2Busage%2Boptimizes%2Beukaryotic%2Btranslational%2Befficiency.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002603%26rft_id%253Dinfo%253Apmid%252F22479199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22479199&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.32" data-doi="10.1101/gr.2896904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage bias from tRNA’s point of view: redundancy, specialization, and efficient decoding for translation optimization.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2279</span>–<span class="cit-lpage">2286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2896904%26rft_id%253Dinfo%253Apmid%252F15479947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS8yMjc5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.33" data-doi="10.1093/dnares/dsu017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Modelling the efficiency of codon-tRNA interactions based on codon usage bias.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu017%26rft_id%253Dinfo%253Apmid%252F24906480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvNTExIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.34" data-doi="10.1038/nrg3051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">Z. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Understanding the contribution of synonymous mutations to human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSauna%26rft.auinit1%253DZ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D683%26rft.epage%253D691%26rft.atitle%253DUnderstanding%2Bthe%2Bcontribution%2Bof%2Bsynonymous%2Bmutations%2Bto%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3051%26rft_id%253Dinfo%253Apmid%252F21878961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878961&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-5.1.73.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seber</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Linear Regression Analysis</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.36" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTI4MSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzUvMS83My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.1.73.37" data-doi="10.1101/gr.136515.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Contributions of mRNA abundance, ribosome loading, and post- or peri-translational effects to temporal repression of C. elegans heterochronic miRNA targets,</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">2418</span>–<span class="cit-lpage">2426</span>. doi: 2410.1101/gr.136515.136111. Epub 132012 Aug 136511.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.136515.111%26rft_id%253Dinfo%253Apmid%252F22855835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8yNDE4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNS8xLzczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.38" data-doi="10.1126/science.1188308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taniguchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantifying <em>E. coli</em> proteome and transcriptome with single-molecule sensitivity in single cells.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTaniguchi%26rft.auinit1%253DY.%26rft.volume%253D329%26rft.issue%253D5991%26rft.spage%253D533%26rft.epage%253D538%26rft.atitle%253DQuantifying%2BE.%2Bcoli%2BProteome%2Band%2BTranscriptome%2Bwith%2BSingle-Molecule%2BSensitivity%2Bin%2BSingle%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188308%26rft_id%253Dinfo%253Apmid%252F20671182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5MS81MzMiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy81LzEvNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.39" data-doi="10.1016/j.cell.2010.03.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vestsigian</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorfan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTuller%26rft.auinit1%253DT.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D344%26rft.epage%253D354%26rft.atitle%253DAn%2Bevolutionarily%2Bconserved%2Bmechanism%2Bfor%2Bcontrolling%2Bthe%2Befficiency%2Bof%2Bprotein%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.031%26rft_id%253Dinfo%253Apmid%252F20403328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403328&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.40" data-doi="10.1073/pnas.0909910107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldman</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruppin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">Translation efficiency is determined by both codon bias and folding energy.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">3645</span>–<span class="cit-lpage">3650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909910107%26rft_id%253Dinfo%253Apmid%252F20133581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzgvMzY0NSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzUvMS83My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.1.73.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zur</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiple Roles of the Coding Sequence 5' End in Gene Expression Regulation.</span> <abbr class="cit-jnl-abbrev">To appear in Nucleic Acids Res.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.42" data-doi="10.1093/nar/gks1262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vizcaino</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cote</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Csordas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dianes</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabregat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The PRoteomics IDEntifications (PRIDE) database and associated tools: status in 2013.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D1063</span>–<span class="cit-lpage">D1069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1262%26rft_id%253Dinfo%253Apmid%252F23203882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MS9EMS9EMTA2MyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzUvMS83My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.43" data-doi="10.1073/pnas.092538799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschlag</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Precision and functional specificity in mRNA decay.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">5860</span>–<span class="cit-lpage">5865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.092538799%26rft_id%253Dinfo%253Apmid%252F11972065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS85LzU4NjAiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy81LzEvNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.1.73.44" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The ’effective number of codons’ used in a gene.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F1%2F73.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/1/73.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/1/61" title="Functional Variants in DPYSL2 Sequence Increase Risk of Schizophrenia and Suggest a Link to mTOR Signaling" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/1/81" title="Cooperation of DLC1 and CDK6 Affects Breast Cancer Clinical Outcome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/1"><span>Volume 5 Issue 1, January 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/1.cover-source.jpg?itok=gg-yH8oX" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/1/73" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-L7n9Ea70Ysb03gYgr8Svhn_o4Lld9duveKzxYPd84PQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/1/73" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/1/73" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/1/73" class="active" data-icon-position="" data-hide-link-title="0">Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459753" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-E9TbqmirzX3HcCZx7PSnivs4N75xw7ZJf6HTCXaf3yA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/1/73.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/1/73" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-SFizn6hHZgQzj3n7cjFgfcrYgN7IOKxqJPqxPgL5QUg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/1/73" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/1/73.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459753" id="node459753--2" data-pisa="ggg;5/1/73" data-pisa-master="ggg;g3.114.015099" data-apath="/ggg/5/1/73.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexandra Dana</span> and <span class="highwire-citation-author" data-delta="1">Tamir Tuller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">73-80;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015099 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459753--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexandra Dana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BDana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dana%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BDana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tamir Tuller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Segol School, Tel Aviv University, Ramat Aviv 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamir%2BTuller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tuller%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATamir%2BTuller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tamirtul@post.tau.ac.il" class="" data-icon-position="" data-hide-link-title="0">tamirtul@post.tau.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/1/73" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/1/73&amp;title=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/1/73&amp;title=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/1/73&amp;title=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/1/73&amp;text=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/1/73&amp;title=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/1/73&amp;t=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/1/73&amp;title=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/1/73&amp;title=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459753/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459753" id="node459753--3" data-pisa="ggg;5/1/73" data-pisa-master="ggg;g3.114.015099" data-apath="/ggg/5/1/73.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mean of the Typical Decoding Rates: A New Translation Efficiency Index Based on the Analysis of Ribosome Profiling Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexandra Dana</span> and <span class="highwire-citation-author" data-delta="1">Tamir Tuller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">73-80;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015099 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459753--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexandra Dana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandra%2BDana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dana%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandra%2BDana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tamir Tuller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biomedical Engineering, Faculty of Engineering, Tel Aviv University, Ramat Aviv 69978, Israel</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Segol School, Tel Aviv University, Ramat Aviv 69978, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamir%2BTuller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tuller%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATamir%2BTuller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tamirtul@post.tau.ac.il" class="" data-icon-position="" data-hide-link-title="0">tamirtul@post.tau.ac.il</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=73&amp;rft.epage=80&amp;rft.atitle=Mean%20of%20the%20Typical%20Decoding%20Rates%3A%20A%20New%20Translation%20Efficiency%20Index%20Based%20on%20the%20Analysis%20of%20Ribosome%20Profiling%20Data&amp;rft.volume=5&amp;rft.issue=1&amp;rft.date=2015-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dana%2C+Alexandra&amp;rft.au=Tuller%2C+Tamir"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459753/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459753/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459753/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459753/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459753/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459753/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459753/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459753/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459753/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459753/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459753/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459753/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/1/73" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Application</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/1/73.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/1/73.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/1/73.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
