<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/7/1323" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/7/1323.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/7/1323.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/7/1323.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/7/1323.atom" />
<meta name="HW.pisa" content="ggg;5/7/1323" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus Zymoseptoria Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements" />
<meta name="DC.Identifier" content="10.1534/g3.115.017731" />
<meta name="DC.Date" content="2015-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Grandaubert et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The fungal pathogen Zymoseptoria tritici (synonym Mycosphaerella graminicola ) is a prominent pathogen of wheat. The reference genome of the isolate IPO323 is one of the best-assembled eukaryotic genomes and encodes more than 10,000 predicted genes. However, a large proportion of the previously annotated gene models are incomplete, with either no start or no stop codons. The availability of RNA-seq data allows better predictions of gene structure. We here used two different RNA-seq datasets, de novo transcriptome assemblies, homology-based comparisons, and trained ab initio gene callers to generate a new gene annotation of Z. tritici IPO323. The annotation pipeline was also applied to re-sequenced genomes of three closely related species of Z. tritici : Z. pseudotritici , Z. ardabiliae , and Z. brevis . Comparative analyses of the predicted gene models using the four Zymoseptoria species revealed sets of species-specific orphan genes enriched with putative pathogenicity-related genes encoding small secreted proteins that may play essential roles in virulence and host specificity. De novo repeat identification allowed us to show that few families of transposable elements are shared between Zymoseptoria species while we observe many species-specific invasions and expansions. The annotation data presented here provide a high-quality resource for future studies of Z. tritici and its sister species and provide detailed insight into gene and genome evolution of fungal plant pathogens." />
<meta name="DC.Contributor" content="Jonathan Grandaubert" />
<meta name="DC.Contributor" content="Amitava Bhattacharyya" />
<meta name="DC.Contributor" content="Eva H. Stukenbrock" />
<meta name="article:published_time" content="2015-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus Zymoseptoria Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The fungal pathogen &lt;i&gt;Zymoseptoria tritici&lt;/i&gt; (synonym &lt;i&gt;Mycosphaerella graminicola&lt;/i&gt;) is a prominent pathogen of wheat. The reference genome of the isolate IPO323 is one of the best-assembled eukaryotic genomes and encodes more than 10,000 predicted genes. However, a large proportion of the previously annotated gene models are incomplete, with either no start or no stop codons. The availability of RNA-seq data allows better predictions of gene structure. We here used two different RNA-seq datasets, &lt;i&gt;de novo&lt;/i&gt; transcriptome assemblies, homology-based comparisons, and trained &lt;i&gt;ab initio&lt;/i&gt; gene callers to generate a new gene annotation of &lt;i&gt;Z. tritici&lt;/i&gt; IPO323. The annotation pipeline was also applied to re-sequenced genomes of three closely related species of &lt;i&gt;Z. tritici&lt;/i&gt;: &lt;i&gt;Z. pseudotritici&lt;/i&gt;, &lt;i&gt;Z. ardabiliae&lt;/i&gt;, and &lt;i&gt;Z. brevis&lt;/i&gt;. Comparative analyses of the predicted gene models using the four &lt;i&gt;Zymoseptoria&lt;/i&gt; species revealed sets of species-specific orphan genes enriched with putative pathogenicity-related genes encoding small secreted proteins that may play essential roles in virulence and host specificity. &lt;i&gt;De novo&lt;/i&gt; repeat identification allowed us to show that few families of transposable elements are shared between &lt;i&gt;Zymoseptoria&lt;/i&gt; species while we observe many species-specific invasions and expansions. The annotation data presented here provide a high-quality resource for future studies of &lt;i&gt;Z. tritici&lt;/i&gt; and its sister species and provide detailed insight into gene and genome evolution of fungal plant pathogens.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/07/01" />
<meta name="citation_mjid" content="ggg;5/7/1323" />
<meta name="citation_id" content="5/7/1323" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/7/1323" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/7/1323.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/7/1323.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/7/1323.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017731" />
<meta name="citation_pmid" content="25917918" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1323" />
<meta name="citation_lastpage" content="1333" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jonathan Grandaubert" />
<meta name="citation_author_institution" content="Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany" />
<meta name="citation_author" content="Amitava Bhattacharyya" />
<meta name="citation_author_institution" content="Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany" />
<meta name="citation_author" content="Eva H. Stukenbrock" />
<meta name="citation_author_institution" content="Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany" />
<meta name="citation_author_email" content="estukenbrock@bot.uni-kiel.de" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. Banke;citation_author=B. A. McDonald;citation_title=Migration patterns among global populations of the pathogenic fungus Mycosphaerella graminicola.;citation_pages=1881-1896;citation_volume=14;citation_year=2005;citation_issue=7;citation_pmid=15910313;citation_doi=10.1111/j.1365-294X.2005.02536.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=E. Birney;citation_author=M. Clamp;citation_author=R. Durbin;citation_title=GeneWise and Genomewise;citation_pages=988-995;citation_volume=14;citation_year=2004;citation_issue=5;citation_pmid=15123596;citation_doi=10.1101/gr.1865504" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. C. Couch;citation_author=I. Fudal;citation_author=M. H. Lebrun;citation_author=D. Tharreau;citation_author=B. Valent;citation_title=Origins of host-specific populations of the blast pathogen Magnaporthe oryzae in crop domestication with subsequent expansion of pandemic clones on rice and weeds of rice.;citation_volume=170;citation_year=2005;citation_pmid=15802503;citation_doi=10.1534/genetics.105.041780" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. Croll;citation_author=M. Zala;citation_author=B. A. McDonald;citation_title=Breakage-fusion-bridge cycles and large insertions contribute to the rapid evolution of accessory chromosomes in a fungal pathogen.;citation_pages=e1003567;citation_volume=9;citation_year=2013;citation_pmid=23785303;citation_doi=10.1371/journal.pgen.1003567" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=R. de Jonge;citation_author=M. D. Bolton;citation_author=B. P. Thomma;citation_title=How filamentous pathogens co-opt plants: the ins and outs of fungal effectors.;citation_pages=400-406;citation_volume=14;citation_year=2011;citation_issue=4;citation_pmid=21454120;citation_doi=10.1016/j.pbi.2011.03.005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. de Jonge;citation_author=H. P. van Esse;citation_author=K. Maruthachalam;citation_author=M. D. Bolton;citation_author=P. Santhanam;citation_title=Tomato immune receptor Ve1 recognizes effector of multiple fungal pathogens uncovered by genome and RNA sequencing.;citation_volume=109;citation_year=2012;citation_pmid=22416119;citation_doi=10.1073/pnas.1119623109" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. de Jonge;citation_author=M. D. Bolton;citation_author=A. Kombrink;citation_author=G. C. van den Berg;citation_author=K. A. Yadeta;citation_title=Extensive chromosomal reshuffling drives evolution of virulence in an asexual pathogen.;citation_volume=23;citation_year=2013;citation_pmid=23685541;citation_doi=10.1101/gr.152660.112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Dhillon;citation_author=J. R. Cavaletto;citation_author=K. V. Wood;citation_author=S. B. Goodwin;citation_title=Accidental amplification and inactivation of a methyltransferase gene eliminates cytosine methylation in Mycosphaerella graminicola.;citation_volume=186;citation_year=2010;citation_pmid=20610411;citation_doi=10.1534/genetics.110.117408" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. Dhillon;citation_author=N. Gill;citation_author=R. C. Hamelin;citation_author=S. B. Goodwin;citation_title=The landscape of transposable elements in the finished genome of the fungal wheat pathogen Mycosphaerella graminicola.;citation_pages=1132;citation_volume=15;citation_year=2014;citation_pmid=25519841;citation_doi=10.1186/1471-2164-15-1132" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=O. Emanuelsson;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=Locating proteins in the cell using TargetP, SignalP and related tools.;citation_pages=953-971;citation_volume=2;citation_year=2007;citation_issue=4;citation_pmid=17446895;citation_doi=10.1038/nprot.2007.131" />
<meta name="citation_reference" content="citation_journal_abbrev=Annu Rev Biophys;citation_author=D. Fass;citation_title=Disulfide bonding in protein biophysics.;citation_pages=63-79;citation_volume=41;citation_year=2012;citation_pmid=22224600;citation_doi=10.1146/annurev-biophys-050511-102321" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=T. Flutre;citation_author=E. Duprat;citation_author=C. Feuillet;citation_author=H. Quesneville;citation_title=Considering transposable element diversification in de novo annotation approaches.;citation_pages=e16526-e16526;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21304975;citation_doi=10.1371/journal.pone.0016526" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. E. Galagan;citation_author=E. U. Selker;citation_title=RIP: the evolutionary cost of genome defense.;citation_pages=417-423;citation_volume=20;citation_year=2004;citation_issue=9;citation_pmid=15313550;citation_doi=10.1016/j.tig.2004.07.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Giardine;citation_author=C. Riemer;citation_author=R. C. Hardison;citation_author=R. Burhans;citation_author=L. Elnitski;citation_title=A. Galaxy: a platform for interactive large-scale genome analysis.;citation_volume=15;citation_year=2005;citation_pmid=16169926;citation_doi=10.1101/gr.4086505" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. B. Goodwin;citation_author=S. B. M’barek;citation_author=B. Dhillon;citation_author=A. H. Wittenberg;citation_author=C. F. Crane;citation_title=Finished genome of the fungal wheat pathogen Mycosphaerella graminicola reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.;citation_pages=e1002070-e1002070;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21695235;citation_doi=10.1371/journal.pgen.1002070" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Grandaubert;citation_author=R. G. Lowe;citation_author=J. L. Soyer;citation_author=C. L. Schoch;citation_author=A. P. Van de Wouw;citation_title=Transposable element-assisted evolution and adaptation to host plant within the Leptosphaeria maculans-Leptosphaeria biglobosa species complex of fungal pathogens.;citation_pages=891;citation_volume=15;citation_year=2014;citation_pmid=25306241;citation_doi=10.1186/1471-2164-15-891" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Haas;citation_author=A. L. Delcher;citation_author=S. M. Mount;citation_author=J. R. Wortman;citation_author=R. K. Smith;citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;citation_pages=5654-5666;citation_volume=31;citation_year=2003;citation_issue=19;citation_pmid=14500829;citation_doi=10.1093/nar/gkg770" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. J. Haas;citation_author=S. L. Salzberg;citation_author=W. Zhu;citation_author=M. Pertea;citation_author=J. E. Allen;citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments.;citation_pages=R7-R7;citation_volume=9;citation_year=2008;citation_issue=1;citation_pmid=18190707;citation_doi=10.1186/gb-2008-9-1-r7" />
<meta name="citation_reference" content="citation_journal_abbrev=Mycology;citation_author=B. J. Haas;citation_author=Q. Zeng;citation_author=M. D. Pearson;citation_author=C. A. Cuomo;citation_author=J. R. Wortman;citation_title=Approaches to Fungal Genome Annotation.;citation_pages=118-141;citation_volume=2;citation_year=2011;citation_issue=3;citation_pmid=22059117;citation_doi=10.1080/21501203.2011.606851" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. K. Hane;citation_author=R. P. Oliver;citation_title=RIPCAL: a tool for alignment-based analysis of repeat-induced point mutations in fungal genomic sequences.;citation_pages=478-478;citation_volume=9;citation_year=2008;citation_pmid=19014496;citation_doi=10.1186/1471-2105-9-478" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Hunter;citation_author=P. Jones;citation_author=A. Mitchell;citation_author=R. Apweiler;citation_author=T. K. Attwood;citation_title=InterPro in 2011: new developments in the family and domain prediction database.;citation_volume=40;citation_year=2012;citation_pmid=22096229;citation_doi=10.1093/nar/gkr948" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. Jurka;citation_author=V. V. Kapitonov;citation_author=A. Pavlicek;citation_author=P. Klonowski;citation_author=O. Kohany;citation_title=Repbase Update, a database of eukaryotic repetitive elements.;citation_pages=462-467;citation_volume=110;citation_year=2005;citation_issue=1-4;citation_pmid=16093699;citation_doi=10.1159/000084979" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=2013 MAFFT multiple sequence alignment software version 7: improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. Kellner;citation_author=A. Bhattacharyya;citation_author=S. Poppe;citation_author=T. Y. Hsu;citation_author=R. B. Brem;citation_title=Expression profiling of the wheat pathogen Zymoseptoria tritici reveals genomic patterns of transcription and host-specific regulatory programs.;citation_volume=6;citation_year=2014;citation_pmid=24920004;citation_doi=10.1093/gbe/evu101" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. Krogh;citation_author=B. Larsson;citation_author=G. von Heijne;citation_author=E. L. Sonnhammer;citation_title=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes.;citation_pages=567-580;citation_volume=305;citation_year=2001;citation_issue=3;citation_pmid=11152613;citation_doi=10.1006/jmbi.2000.4315" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. V. Lukashin;citation_author=M. Borodovsky;citation_title=GeneMark.hmm: new solutions for gene finding.;citation_volume=26;citation_year=1998;citation_pmid=9461475;citation_doi=10.1093/nar/26.4.1107" />
<meta name="citation_reference" content="citation_journal_title=GigaScience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Marchler-Bauer;citation_author=S. Lu;citation_author=J. B. Anderson;citation_author=F. Chitsaz;citation_author=M. K. Derbyshire;citation_title=CDD: a Conserved Domain Database for the functional annotation of proteins.;citation_volume=39;citation_year=2011;citation_pmid=21109532;citation_doi=10.1093/nar/gkq1189" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=V. Miele;citation_author=S. Penel;citation_author=L. Duret;citation_title=Ultra-fast sequence clustering from similarity networks with SiLiX.;citation_pages=116-116;citation_volume=12;citation_year=2011;citation_pmid=21513511;citation_doi=10.1186/1471-2105-12-116" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=A. Morais do Amaral;citation_author=J. Antoniw;citation_author=J. J. Rudd;citation_author=K. E. Hammond-Kosack;citation_title=Defining the predicted protein secretome of the fungal wheat leaf pathogen Mycosphaerella graminicola.;citation_pages=e49904;citation_volume=7;citation_year=2012;citation_pmid=23236356;citation_doi=10.1371/journal.pone.0049904" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=A. Muszewska;citation_author=K. Steczkiewicz;citation_author=K. Ginalski;citation_title=DIRS and Ngaro Retrotransposons in Fungi.;citation_pages=e76319;citation_volume=8;citation_year=2013;citation_pmid=24086727" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=R. A. Ohm;citation_author=N. Feau;citation_author=B. Henrissat;citation_author=C. L. Schoch;citation_author=B. A. Horwitz;citation_title=Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi.;citation_pages=e1003037;citation_volume=8;citation_year=2012;citation_pmid=23236275;citation_doi=10.1371/journal.ppat.1003037" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Palma-Guerrero;citation_author=C. R. Hall;citation_author=D. Kowbel;citation_author=J. Welch;citation_author=J. W. Taylor;citation_title=Genome wide association identifies novel loci involved in fungal communication.;citation_pages=e1003669;citation_volume=9;citation_year=2013;citation_pmid=23935534;citation_doi=10.1371/journal.pgen.1003669" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=E. J. Pritham;citation_author=T. Putliwala;citation_author=C. Feschotte;citation_title=Mavericks, a novel class of giant transposable elements widespread in eukaryotes and related to DNA viruses.;citation_pages=3-17;citation_volume=390;citation_year=2007;citation_issue=1-2;citation_pmid=17034960;citation_doi=10.1016/j.gene.2006.08.008" />
<meta name="citation_reference" content="citation_journal_abbrev=Persoonia;citation_author=W. Quaedvlieg;citation_author=G. H. Kema;citation_author=J. Z. Groenewald;citation_author=G. J. Verkley;citation_author=S. Seifbarghi;citation_title=Zymoseptoria gen. nov.: a new genus to accommodate Septoria-like species occurring on graminicolous hosts.;citation_pages=57-69;citation_volume=26;citation_year=2011;citation_pmid=22025804;citation_doi=10.3767/003158511X571841" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. Quevillon;citation_author=V. Silventoinen;citation_author=S. Pillai;citation_author=N. Harte;citation_author=N. Mulder;citation_title=InterProScan: protein domains identifier;citation_pages=W116-W120;citation_volume=33;citation_year=2005;citation_issue=suppl_2;citation_pmid=15980438;citation_doi=10.1093/nar/gki442" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=T. Rouxel;citation_author=J. Grandaubert;citation_author=J. K. Hane;citation_author=C. Hoede;citation_author=A. P. van de Wouw;citation_title=Effector diversification within compartments of the Leptosphaeria maculans genome affected by Repeat-Induced Point mutations.;citation_pages=202-202;citation_volume=2;citation_year=2011;citation_pmid=21326234;citation_doi=10.1038/ncomms1189" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. A. Salamov;citation_author=V. V. Solovyev;citation_title=Ab initio Gene Finding in Drosophila Genomic DNA;citation_pages=516-522;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779491;citation_doi=10.1101/gr.10.4.516" />
<meta name="citation_reference" content="citation_title=Molecular cloning: a laboratory manual;citation_author=J. Sambrook;citation_author=D. W. Russell;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. L. Soyer;citation_author=M. El Ghalid;citation_author=N. Glaser;citation_author=B. Ollivier;citation_author=J. Linglin;citation_title=Epigenetic control of effector gene expression in the plant pathogenic fungus Leptosphaeria maculans.;citation_pages=e1004227;citation_volume=10;citation_year=2014;citation_pmid=24603691;citation_doi=10.1371/journal.pgen.1004227" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. D. Spanu;citation_author=J. C. Abbott;citation_author=J. Amselem;citation_author=T. A. Burgis;citation_author=D. M. Soanes;citation_title=Genome Expansion and Gene Loss in Powdery Mildew Fungi Reveal Tradeoffs in Extreme Parasitism;citation_pages=1543-1546;citation_volume=330;citation_year=2010;citation_issue=6010;citation_pmid=21148392;citation_doi=10.1126/science.1194573" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=M. Stanke;citation_author=S. Waack;citation_title=Gene prediction with a hidden Markov model and a new intron  submodel;citation_pages=ii215-225;citation_volume=19;citation_year=2003;citation_issue=90002;citation_pmid=14534192;citation_doi=10.1093/bioinformatics/btg1080" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=I. Stergiopoulos;citation_author=P. J. de Wit;citation_title=Fungal effector proteins.;citation_pages=233-263;citation_volume=47;citation_year=2009;citation_pmid=19400631;citation_doi=10.1146/annurev.phyto.112408.132637" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. H. Stukenbrock;citation_author=S. Banke;citation_author=M. Javan-Nikkhah;citation_author=B. A. McDonald;citation_title=Origin and domestication of the fungal wheat pathogen Mycosphaerella graminicola via sympatric speciation.;citation_volume=24;citation_year=2007;citation_pmid=17095534;citation_doi=10.1093/molbev/msl169" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. H. Stukenbrock;citation_author=F. G. Jørgensen;citation_author=M. Zala;citation_author=T. T. Hansen;citation_author=B. A. McDonald;citation_title=Whole-genome and chromosome evolution associated with host adaptation and speciation of the wheat pathogen Mycosphaerella graminicola.;citation_pages=e1001189-e1001189;citation_volume=6;citation_year=2010;citation_issue=12;citation_pmid=21203495;citation_doi=10.1371/journal.pgen.1001189" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. H. Stukenbrock;citation_author=T. Bataillon;citation_author=J. Y. Dutheil;citation_author=T. T. Hansen;citation_author=R. Li;citation_title=The making of a new pathogen: insights from comparative population genomics of the domesticated wheat pathogen Mycosphaerella graminicola and its wild sister species.;citation_volume=21;citation_year=2011;citation_pmid=21994252;citation_doi=10.1101/gr.118851.110" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_author=E. H. Stukenbrock;citation_author=W. Quaedvlieg;citation_author=M. Javan-Nikhah;citation_author=M. Zala;citation_author=P. W. Crous;citation_title=Zymoseptoria ardabiliae and Z. pseudotritici, two progenitor species of the septoria tritici leaf blotch fungus Z. tritici (synonym: Mycosphaerella graminicola).;citation_volume=104;citation_year=2012;citation_pmid=22675045;citation_doi=10.3852/11-374" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. E. Suzek;citation_author=H. Huang;citation_author=P. McGarvey;citation_author=R. Mazumder;citation_author=C. H. Wu;citation_title=UniRef: comprehensive and non-redundant UniProt reference clusters.;citation_volume=23;citation_year=2007;citation_pmid=17379688;citation_doi=10.1093/bioinformatics/btm098" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Ter-Hovhannisyan;citation_author=A. Lomsadze;citation_author=Y. O. Chernoff;citation_author=M. Borodovsky;citation_title=Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.;citation_volume=18;citation_year=2008;citation_pmid=18757608;citation_doi=10.1101/gr.081612.108" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Wicker;citation_author=F. Sabot;citation_author=A. Hua-Van;citation_author=J. L. Bennetzen;citation_author=P. Capy;citation_title=A unified classification system for eukaryotic transposable elements.;citation_pages=973-982;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=17984973;citation_doi=10.1038/nrg2165" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. H. Wittenberg;citation_author=T. A. van der Lee;citation_author=S. Ben M’barek;citation_author=S. B. Ware;citation_author=S. B. Goodwin;citation_title=Meiosis drives extraordinary genome plasticity in the haploid fungal plant pathogen Mycosphaerella graminicola.;citation_pages=e5863-e5863;citation_volume=4;citation_year=2009;citation_issue=6;citation_pmid=19516898;citation_doi=10.1371/journal.pone.0005863" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus Zymoseptoria Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="twitter:description" content="The fungal pathogen Zymoseptoria tritici (synonym Mycosphaerella graminicola ) is a prominent pathogen of wheat. The reference genome of the isolate IPO323 is one of the best-assembled eukaryotic genomes and encodes more than 10,000 predicted genes. However, a large proportion of the previously annotated gene models are incomplete, with either no start or no stop codons. The availability of RNA-seq data allows better predictions of gene structure. We here used two different RNA-seq datasets, de novo transcriptome assemblies, homology-based comparisons, and trained ab initio gene callers to generate a new gene annotation of Z. tritici IPO323. The annotation pipeline was also applied to re-sequenced genomes of three closely related species of Z. tritici : Z. pseudotritici , Z. ardabiliae , and Z. brevis . Comparative analyses of the predicted gene models using the four Zymoseptoria species revealed sets of species-specific orphan genes enriched with putative pathogenicity-related genes encoding small secreted proteins that may play essential roles in virulence and host specificity. De novo repeat identification allowed us to show that few families of transposable elements are shared between Zymoseptoria species while we observe many species-specific invasions and expansions. The annotation data presented here provide a high-quality resource for future studies of Z. tritici and its sister species and provide detailed insight into gene and genome evolution of fungal plant pathogens." />
<meta name="og-title" property="og:title" content="RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus Zymoseptoria Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/7/1323" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The fungal pathogen Zymoseptoria tritici (synonym Mycosphaerella graminicola ) is a prominent pathogen of wheat. The reference genome of the isolate IPO323 is one of the best-assembled eukaryotic genomes and encodes more than 10,000 predicted genes. However, a large proportion of the previously annotated gene models are incomplete, with either no start or no stop codons. The availability of RNA-seq data allows better predictions of gene structure. We here used two different RNA-seq datasets, de novo transcriptome assemblies, homology-based comparisons, and trained ab initio gene callers to generate a new gene annotation of Z. tritici IPO323. The annotation pipeline was also applied to re-sequenced genomes of three closely related species of Z. tritici : Z. pseudotritici , Z. ardabiliae , and Z. brevis . Comparative analyses of the predicted gene models using the four Zymoseptoria species revealed sets of species-specific orphan genes enriched with putative pathogenicity-related genes encoding small secreted proteins that may play essential roles in virulence and host specificity. De novo repeat identification allowed us to show that few families of transposable elements are shared between Zymoseptoria species while we observe many species-specific invasions and expansions. The annotation data presented here provide a high-quality resource for future studies of Z. tritici and its sister species and provide detailed insight into gene and genome evolution of fungal plant pathogens." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
    <title>RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus Zymoseptoria Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"acbR0_Rqddsq_amg0VnQ_AWrWQW0qAJTYGRN7IjukvM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459981","apath":"\/ggg\/5\/7\/1323.atom","pisa":"ggg;5\/7\/1323","ac":{"\/ggg\/5\/7\/1323.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/7\/1323.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1323"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1323"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/7\/1323":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xv24KbbpSgpCXrJPA7N4o1kOzXjVcBf+p4jch5x+cyYbGtfLyFRuM+B4xILHf8IZI65T2tXnU3D0n2Hthmdliq\\\\\\\/BNrklv\\\\\\\/CZTVpZyalhQfgpiy7OqwIpYyImZPXXvkXeR8AxQ0HOHpoZkW\\\\\\\/uQG0J4M+skPgyG8Hen+5i9OMMqqedHG3fBPVhiUdrxCgQ5i3b8QhoIK6i6bkVoMtLnklkAoMXovurBE4sasQqJoD1wziqch66GKnDGMsGEKFiXw1zioFes6gBhb9LaZZQvVDuwoyDiU+H1li+lB0e2Mq14\\\\\\\/MqfZ7dG8ch7job4WpHnGp3+ak86piapgVTxwJCVrWvd3VXRWzL3He\\\\\\\/ed4D1WPu4+eOR9E8HLQaqaOFDSJh4LlxtYMY2WBQw16tg\\\\\\\/NmkMD\\\\\\\/zyyY2CyYi\\\\\\\/QWsNOjhjvK3ZMU2Rj\\\\\\\/IHaNP03ZJ668ge+88f1noJJYylo8qA9YwLCPNz8VRtlYtAj0855ZaV1bJVeubx2Tnm8uBxeCESg+kL+XgWTy5hBm4a8iJwZeI3DGinVgibxWwWDT3QTSNwlfyPKSFVeDivxAun8dDWsItgTckNUcX7V5YUHgtsRaj5nSD276wuu0iYhf8kFaytMU1ZbvYDWp0xWBraVmNBOAHGWV7X9ywiL1UwmLaw8YPi+d0A4TUlONFBbrQAKFS4WJy4MdyTDo4RNwqLaWsDvkNkFYNrckgMxQcpVXN4\\\\\\\/4Q\\\\\\\/J6kGzvD6OCgqjyzOpB6arIZOw3Wdc5elVvRzM6USmm\\\\\\\/BKi2k7VJafRl8dNRXwjxLvX9QaDGm3fXabZG5HM6+PTMag5X9wemAPMx60ELw1NxcO4YRZPdOUkiODDTThZBAHWgbayFS1zVLWiXiW81cjax8eBeYogKgMQEInkEpZE+Y1Lz4uDywEPHUm71lnQwiMnLPU1ut\\\\\\\/MaGUMFCPLpaZLG+zUjZ5Ke5U+r\\\\\\\/ZbSqXAYye3IBUfgYz2e63PfqKmo\\\\\\\/Xxn1nXf7xTEjk9hoiThWNu5uKL1oNtPzCETMf9481EhIWgQt1rMucTKLoR35blPuIyF0K1bB\\\\\\\/0T5mEygHLJtJwPEv\\\\\\\/gPRNU5beDwwxX4nDO6BQltOzsKPR7FvWiMNJLrLqdCbL47cax21FPnwhLazAHXbJklxKRQypTCr\\\\\\\/QpPuFRkADIATdv5ocU45KGJU3wfoREBwy+qia+io3lIBaA3wbikIQxsm9iFKTZTF25T6JzvMzyy+rVclZrARiSHev87l6KGImwqfTGKDj9J0cL05kKdD0zo9kWMqqDni+1i2bOlL\\\u0022,\\\u0022iv\\\u0022:\\\u0022B1CUvh1qdIqWyZxyCwOnKA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022839862acf89ccb081451f80d7e1b729828089ecc810ea624d5e5fe859a40043f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iSgWXnSdmHX6WozrAw4TO29NeJKmzKSWF63kt4MlaJbkLOY6I96sSTFkJ2yc9psjcK2mw7CMqx6JeLUNwOotruOb\\\\\\\/2A3cHASN+jLAKU2N14tXEXZeOegOUmmp0c2ucEVqYHJFUISCp\\\\\\\/xtJXZ\\\\\\\/stjeTtII+Qz8CWkf3Ie+ZpsPxfgRXnnSC0WDYoTEPKhNpPcG\\\\\\\/zwVcDW2DTNy1hRkGeSrKYx\\\\\\\/PzfkevAV\\\\\\\/il2qDFat10JvDBVkfV1ddKXZD7u9L3ftD37vY6y6pDvugHxUNz+qoXTw8+8CQJPrctQXsMjS6xZLEtQhZbxfu83kstb4xqT7AHjD9+d9Sc8rB3e9uu9D5lin3RkqBARysqY8HrzMtujtQ8GoCBR52VkQ8gLfUCJx8CDNfEZU\\\\\\\/y0aWBPLvq6OUz2kTARzSsAcaz7q19fPgrqRfljMxx4S5InrzpxY6nHPQXUvcldNitmnzJ\\\\\\\/yu9N9oNIipslLaZipj3DkCn2wbnGmjXwweWuJx\\\\\\\/5BParIiMgzVIT8yKdrgOCvS192AsSEQt5bnCbQxytBB1V6+6CgUV3dZufn+PwynhwtYDZ4bv+kfRbwK7M3vJacP8XJEsn2VXg9XR0IbNNDXmxoieDLuRbeiJwl6CefotjmjgKNfP76FsXgHoubMBQ6PNfdUmQX0qQPhXq\\\\\\\/qJL6F\\\\\\\/dNlt+BU\\\\\\\/xslq7DbyXzuaCD1ykTPKCXIJq0nq\\\\\\\/uI6WkC5OglDFM1S7bDCJp8O0PGK1XqMtnO0dErsH\\\\\\\/3CBloyL1RJI7P7c\\\\\\\/ZAJT6bsAw2jNoTEO1ghL+CAtGGbTvL6W7M2rjIuHchITw79vdDTo8nRAkGHSTC\\\\\\\/s8cFcT4tJMhAvjEnryaC\\\\\\\/sJwGtH3v66KTz6Qcfpx3jtmLD6E1VTL4dFf+7VIyCZt0NCUt5FdQG9lexlNj8sQm9BpWpudvpU8986eqaVw0GNw\\\\\\\/pEhEoRlrcb3yBryFEeeblasyytJxApPl015gUaGOx8eyeiuOYlPG46hvoXUCQXomnAuccm1FDM0TYPdhto7aa1py+WRt93bHZrRscg5BFKLwuA+sarBtx8CSq9EYD6ksxjQaYDPGRLCvProwHYcKs+3pwA8TqiMv6uycVB1kAaxAcVJ9BzXk\\\\\\\/MZ44loGhdVkwO5582wMwHB3xhMLNmRa5mGtasGZ5aLuDwYKb2RFNuDfAKO2PCWOonY7PhMRCtfiR+4bUak74+6eW\\\\\\\/VO7fwVuoYs3EX6HkCvTBG8SEtXbHA3NWHorvi\\\\\\\/mSVv2KecqRT9U1RnO1xSMiaqeFI4mV0eoWhAzzwC5KwnJRM4JjIM\\\\\\\/cXaS+wLx9Mg1saEhDn6IxzMfjEMcnLknDYEFECNOj5ev3G5n5PjWpuVfwmnxND98xlHfsBx0rEZBIR7PEn8h2p6uikOUsbStOnWUhe3yiZ49XYnD\\\\\\\/+qwLCG7eXB2qaLPBPrm7uZidKOUi0usui8KKmiuQeDBrhvupQoHyG+qerLDztYzscRSHMy1VAnpMbaAJQlRaefibALoq0GDmxbw7h56thtzAjfX19McjSqmiUkRdSAJArvu49dh0HXfNBCkJdM0jhapdUlMsBd0YPKUWwO2YxrzoVtU65rocPOg7PjxBdoOBfQjbDSN+OuA\\\\\\\/6iCMIaSHtVuErSoEOuExvLHoHxA9kg7+epp9iZfWo0ffc7eh7GWimXSI7\\\\\\\/aSTk8iUcvrTnOwwxKYohGmVbJcUfCP5Q\\\\\\\/y18mc1ucsig8S7xAnDLA343BSL\\\\\\\/+gKzjUNrfn2fn0eE0hrgrHTZ7ujWrMocarB5TagJoVCoGDv5Y7VUSaNs6gtElAw03ne2LZkMIPGKuwQzpy9h1+c0nKJFB7a3lxYd0EKGkEowcVwg6N5pRM6zEIaq16UYgqk+K+32xRFKAqIV3JWUVh2h1qGe6FL9OCankziLM0OUKDe+FjYn6S0ZVa1vBmCPywL9H2rntzDfdpop7fLFVNWE9N9qRRU8kXtK9rN4QueUUQse9WAOrj\\\\\\\/jB5\\\\\\\/6BRYg9vn2HLJakeWSu8e2AI4eLYKf1ofeP2hGt32sYDHGbHjllcfpWA4Cq9hGxXk37+ztXfWIKgUrpr3Wnx7Ev0J\\\\\\\/FaSLKYxWCPbdhtZIG6PTyn2rhYl\\\\\\\/VNi3DKoDf6MitpUngD3oiYHbprYGqLk++FS\\\\\\\/mJ\\\\\\\/WvR0ftzaRe+58fuG8Do\\\\\\\/QFl2KPiUNDKOZexfiHTrksvOHDWEq48+xbHiTthbcHDItPTA9fYQBfYjBbnmsDcBtTsI05wwXW0\\\\\\\/Q4GTt5NopZ699j6LoGo77n1J8U9an7a3XltSP8vcOMd1GpLHAG+8rD4EnSmW42p4+HZ7SR5QXZX+yNZFyBnOhTt3l41feDfna14iwGbx6Sw6AF0q8Ld7HUTLDdgz78JxiM5VTtwiWBltpd\\\\\\\/rkZAGOOWIwalONBJwRl+WuduRqNp\\\\\\\/L1U1M6RXc3QMRIIWg4GjO\\\\\\\/iyRq4aYQgIt4hYNQ2vKaDy9o\\\\\\\/lnYTN6ax8UTUlcXIn1Q+8W\\\\\\\/\\\\\\\/pGOpjM0LcctwYtCrpoZgCZLxA4KVKpEigvr26uebhMLUeikmVR3MLIypC9n33MRvCI7xCEolxvvQID0w51LS9lX6I8xyrgy78FDkfwiwyaq3+G6P0aIFuPqTOmrjfwGiQjVxRfQY+p1kb6GwJnn9UPG6I7dJrikCpgsJhBvG8Mav6Iax3cPa4kgDT3Ix2\\\\\\\/Qcu2iQfq4bpwtqtmXEG3ccmoSS70Jj1XD4LEy\\\\\\\/4y8I6oXoggZGLh2GO8RQgHpb18vT+66BDg4AshCiMJAZ0PctHJsegpE+b3BqWlhjDC\\\\\\\/z0OpPFVkUa6gMV\\\\\\\/I47iaThysnh4tZ3nMWCnbEypi7dbWrBVlyvqaKlrt211iTEYWK\\\\\\\/PMAsqxnmwVRixNV8+GXlZaz0NuAEYYhy6zJDwCN0VduN6yEhyDK8zSMgVtWGM+hoyLTVE\\\\\\\/PhT1sgWBxdP9UP7r4xq8NlbG0wK9YFuCPHSSHOf4XFAS1CHu1N95So\\\\\\\/7VbPdDImufWjGoB0WN6L0lpJPAyFPsGFbfuPDQT+LJP8YLSILf9Nc6mopF4rBtqWuFK3TyeHn\\\\\\\/ordMdM2y27AYx88Lj3BMyEDv38WAe2PGchgrgPyYOt9qKOj7AlNY3qWmDmX6EzyUID0HX3Kvp2sFLUhx6cvBOEC6bQYyWTCowPbdgv+fDbLxbqrTFmf74QAi0bC46A+tYvs1E6fBtSsO5jS1OBf2u2b6QGMCUadwVgWbUgKMMfBHYM9ebiWuGAyARav0cEDIjOOpvkY8UvyNU8NS5pyrrqSTuoXj4\\\\\\\/rrs9uypkQKgtDdY+JuGQLyU1WL+yLor06NxEzUeK1CWybRZVfvT7wR55vYAQ9Fsr6viQ2k3DgokEW74zeIcGovUwzxlqANzP6Fvey6oGhyS06EeUVWR5fCyIeCKIhdULyuPTYPiFrxyQuYR0PVNl3WR6Ghh0JoHKrmK5luw8l6fVlSbfof1xlI\\\\\\\/hFSMRo1IyNbkh0Yr7D4erKAJDENoZAfVdK6cTsfhu3Wzv5AoNFixbYc6dMJMzO07\\\\\\\/neVmmmxLk31MRDgkpMDR5tTqMloSUc0t6wPKi3kIr0keB\\\\\\\/Hej\\\\\\\/G45GrLM8jJeslA0JdRW+cluWedh5\\\\\\\/3xxOJA3+Pbzi+upRDtQyxf7IPs8IfWf\\\\\\\/21+Arsnrf9b53yZvQ05mhx+IAXERJbOv6nnFdluqS8brsAk703ZOaGmnmQuoDZqinfdettMPQu2ElgIkuYLQMs8cGnhZ625AWwaV5riyHdZkLfNZ07xUAzy0Q\\\\\\\/CvUJVxOXc40ZRhnPx2f\\\\\\\/cRbx25qOpQyGcKq73+ROGhwII5rdbsaUVwdbpj91SpoPlNe50HR+neNbfmXCdulGcpByGFsq6QLXPVOeYoppvtijdq3+5cpDDH\\\\\\\/p0Szag4G2wdZNl18wsunyJY7Q7D5nzlBzmtTklzfd+VJkXfiHTT3fekMjatCdFdpn3Uocl7q\\\\\\\/VYfOBSn9Xjc+5oSkdVbD3+6XC+0WLIKSoGIEhh2yBKCHiqfmn0D8Q3UpY6u7ebGzwqZVLGZowBYCYB2HgbLq2OLpsIXnS+bXpVkUKmKCDppMszFiO0MIbumoInVE\\\\\\\/RdUuYAEBHi3RKzy4ALS3\\\\\\\/6kNJh+LLHx0j7BfvOkOt71HVB27kqcq94YA9LAS1QXgmn\\\\\\\/sbFLY2A+Cun10QzF+I6hexvjswZW\\\\\\\/ucUzEZ\\\\\\\/ciBEhdNw2XyVN+q0yQoGIcr1L4kOAOKvoZaqvs8Ne\\\\\\\/Y7g7diDjslQNjsOflY0JyUXxq0ZITR4Lw4a7rqu7pQKjzKd3HOWJQ6WZSPG40Vn3hhJaBP846l25wWv59oSrU1TZaXIhxAkB\\\\\\\/1JqhOtNjvtH3WA2Cp1Yc1KYrSzwvwnTCt1z9YA5RgRYFfM6ziZkghBE3H63zepouBwe4hph0PEnK1V6FPuKutN9bp9p\\\\\\\/LuJmey+GBUz6bz0qA3tC29boXu+lf8IKt7CRcnT6JT3QMi3mnVwfEO1wocet\\\\\\\/iNxJ6HCg3FuF0peHa1ce2b2Ye0KDh6gyrQ2kgbFOWa4GLZvwO+deSr3++aoto1CdrNl6fqR63UGOsmRNHLz\\\\\\\/ECLymA9GOOxbP3O9NqtZZJyzNEDxW8u1bjTGJxgR2oXW+qePawNCbdaCl3fKrwiWWMKbuMFCfIR4hqUEHzHVpYdBOECSmQEnQxw0oQ\\\\\\\/unexU0v8raC+SL+9TZGQIZZuDf5V33+qXDEKg3OXhEOJe\\\\\\\/d2xbMXB36IjEwpkwRPiXZZ7\\\\\\\/tr2C73bjcp9JINvqE40KqgxddXXjj2jA8ZnFYAw3SGnWuvXTzgJy11L7comx1V1a6HBSx0uGAxpT28OJTzNoSpKU0tU50n\\\\\\\/HxzAA7w0rtRptkV2Fn2ej0\\\\\\\/rmGbWSqUE8a2Hw20yBd\\\\\\\/HPhlsuG04maNO9msjdGksZo2QH6EOJ4oMqqyB9SKEFlZbZWY4qNhjpqXQbady+zc3kMRwTBNWc3Q+B6\\\\\\\/QkppWaAv7G7RNt\\\\\\\/JhMCZXZCr9sQ4VursS6vKasDIAodh\\\\\\\/kdEARdUmhtSAaJgV\\\\\\\/yOPEGRJ1yzuP\\\\\\\/uAgux79KKs1MtrI5yHYzf46RgsR5FN6\\\\\\\/A3CvsDL+Yt+aBCJ96oY4rwt\\\\\\\/0ZrCA7Z7br13hOlSSBDI9YoN3upvFTb+rsM8CWuXU39imLbGm1iLJsdpoohfkrfkKxkzlG3SgQztvvcJbXCZw4V\\\\\\\/5DlLLQAAPp45q7\\\\\\\/j4rbvD59NVVo1axWsDQCJC3xWcY0ww7Q8a6GDeGLuXxcsZLXDyZjWuUGrakFP2MaZf5xSM6arrmlAGhpz5e8pl3uLG500E2FmoPlvmNhefUqZ9qzh9MScEC+XopBaGUrnOpDfN2QVw755Yu3IEwf8j8TR4xYFmbkjUSXWv82YWM7mTHoe\\\\\\\/jfpU9ZICOATu9XwX7yK\\\\\\\/lLibmE9obXJhSWYdkHeDEJANetA\\\\\\\/5IZUowA3D7iUS5KjsmeBSKgWHDJjn1oNlfvhz3GdkZurZKUmPr5ZMJL4KCxzYdFMhBZxBmDE\\\\\\\/CFsQ4p4c2AtI4TFRS8nYtqoc5y9kcIERM0elvwuWs3DpttsClyUSB5ELFGvS9vO2sGdVx4i0bJQigqFtWNC8EDsnWAWHdtjejdiPRdoCoH+7N0pQOsa7ze3TGtTvpQvTzje2Ptz0i9eoe\\\\\\\/W6x4WHYG03s7DgXjct1+ETrZ8HoXAhpcyaEEn8T7JcPIZwj97kBMKGWkWK45ohLOV0AI5vaLanioXa80RSeaJq9kEolVwTIZiw296HFJuRqzYCY70dN+Pyn85y6S4k8TwgEtGFefnczk6iZFaFeMJfdkdh8iNsW00pg5ZQi0UcG6FhzZ1ewFUrhuOb7YSoOaSzG82LbMrLfN9AOB7DwBE7AznNsRmTaKlATamEstJRPY0sCpMvqtiq8Zo0iFLfRpMPKRQKvblYCl2APdYUdMfSanBe0w2pwvii4u3yTp59PyZLbVEiwuEARaEcYaEaJGzhw7H+ADQrDeOv6YBmMpei6nwvcv8S27SBs16sglTttWo0Lw9v46apr9xeS2r+lj5tsaTYojb7MZhYORGxpjAQkzSJqT7zowRBwfR+iMo2EdIWKBik4H2b\\\\\\\/edN9z+xVZNhpODcj\\\\\\\/I7rn3G97hTHme1s8iCSutLu83EBML635W4rWxqaa+bGO7l0cbDFyd6Qzs7MOIG0HAkEU+MdiNTUjuyiKrcaCztCEORJ1axkl2zVEIP2U0b3\\\\\\\/Dx3YU05Ib7RRUCV0blgwulGxgMtStscyTYFOs5C7dGG8dm\\\\\\\/k0infWq02rLhnI0uOIdWR0NKDQa+qAKTPkgOeGyTO4xUktYL8R68HBHOIafjQM5YkHDia36KGJz3sEd7AvOOZQT6NBbsr+sJTyuPTM2cMUhd1QFk+NbDoOZ1DnSDdv67rKchpvfPqjs\\\\\\\/c58cl0xM3byl3sD6pPBEYB5ceuFcElexcfeehEGbNRmWwPGzOLozAlSPlSghWYEfe1O\\\\\\\/nZ251ImTNG9pKWhrczsrpWs\\\\\\\/6zAU1SbVi0i7\\\\\\\/Of7c5qXPpaeJ3UmBjJS4JO72BtVIIFpvpWZtNZGjOp9jJnulypiN9xIkvQS2FRpiiDtz2zV7H1MwP4q9DyjaIU\\\\\\\/wCVN9+LLRE184lTUfyKC2woLR9R7\\\\\\\/\\\\\\\/1TQqPE4a7KS\\\\\\\/t9lyG3CWrLShX+utD9KvFl8nGAA7SO7YfFKwmqweMGM2xjSuWBL5KjdAOtdenaZhEev9K85cQLSGgROXBDqffXBU1WLTpgjILMlCxI84IBp8IYDUlR2Lx+Rg1My7TsYl9\\\\\\\/RDWko0jHU8DIx+8hot+2qQMeIN9t5w0pFSYDHo7pVrwVO+Ry3vNQP5nMP\\\\\\\/KS1aGt5u0kOHs4GHpk\\\\\\\/YcHai6MpnjnYXcasZzx6EBQ5JLQLAv4w+Ir6eRjcIKKAcjn6qNub7xtj1OigfFZ6A1vNXSbQeb7L3gsFsDothlcM\\\\\\\/\\\\\\\/+l+LvbHROrDtj\\\\\\\/P7aOXq8BWiD0IRc07a3oxySLoEZibVMQycI\\\\\\\/PwXa5Hgvb+X16cElmKPeGvhGMZw2zQRNAqSH2XFBKk+dD6bwquow2NabtPXi4qtbxiDp0rEET+Fr2kYlOVKLxQi2DPen9QBvOaCfDUuDsh3ra3SNaz4CACnL4RMofesG1wY9K6hQ\\\\\\\/L5WkmtL\\\\\\\/2xzQ62zYxtXl7Zu1rRs250uIPqS\\\\\\\/bi3pMEwABcWJ74tz2q1hv2FhmYxyKTevmbAcmZs4yKri8ngDrQB5gfYnYcIPLKwBvhuNLnnF9GwATJcy5rq\\\\\\\/sbiE9rsQloC1UAmOODdpKbEkkDglpOIShLC+PiiUjiTNiQmNx2KWkcvvbJVFRzYGSOuLtvx0lZkcEhpF\\\\\\\/RafAjC8WFPmZ2HwpbzeHZFelD9Cw8gfRmN846I7EvQE5TgUsBCA\\\\\\\/S6esyzsoS6MrRbKgJgevLUfMk4E5Z7fXux3OVK50CbP3\\\\\\\/zK8wtr6HgUcemqeARSusjaXfxRx3J7uOPs\\\\\\\/oTlJiXzWh+N1D3ntaES0iNnKDBJXPf877IGKBMyaBSZeDoVvy5\\\\\\\/mr2OsqnVtW4IzUV42xIXLPfLsJ8xbJaiEgbRGbqmRAbuQVOrlv+oe0+K54bDTQ\\\\\\\/yBnKGUg3IlQoGGjbn\\\\\\\/JBf7r3x2jyBB5gyT3VB2wRLBEOMGfQ7A\\\\\\\/7ZcH3nY9hc3McyBTXX1c4mT7dT7fVd3ERe7lEooMEQlXULwV2n7bsEgFyFRR8EI+2whDZS3StkdNpgiLojrNOCudzoJBd9gsWKQstmjXWFnqJquRa2D8j6XNi\\\\\\\/dJP+Yko654zxsdtzVFyTX6ebUQrhZDHR9oaKfgq8f+sUeuEBa7Wt30Y9kVsJOmuuLTy7m3RrNm463BBPcWucAH2btL2MomFvk48ko1u6n8WkWF7ReJ8aCO+uCrCm8saYb4BsgCbfk1usuogaSZ3JNX38arfXkbnlJrD+HU0jB+2iZb4W57NXAn+9UJDsGPjAO+wpM7Q2O8i+r3KoCnR0PdKZIK+rvAVzZlrlA0ty8JjRvJtm73qUdqX0XiPx8pBa95q0MPaOkxhhvcvdk4evwxC2IpCulB7bGjAejP3iEGc\\\\\\\/216RLVpRLVtWNVyHFfxDm3bWzH34aUpsJ+r7SGg0NjZpIbC1tyGJnPpm9XSLwLsitksgV0Ekd+2dgb6uSQfeyyfj2DZajnhN58jVRGBEr85n2Yc4nMVczrEF95WBlS+Gvx78aHa5px3hZdeIGrrhXvYdMk2DOBuj202Sr8epekG8lTl6LlyalurCA+FQz\\\\\\\/TaSI8OiymlorOI2\\\\\\\/tNdxU5Z\\\\\\\/24O81BKRJ75KYjPgGyaemKODCvLY0rnvLThl\\\\\\\/8pCY2cNPA\\\\\\\/JJ\\\\\\\/VV6DpHK4YLGZLItVh9WjUxmL6wUQLj1yA5OhyDkiwm09UEYYIyWx3HtEG2g6MVWbE\\\\\\\/dIFzbu2JR7W5sUzsqO19awosuDZxpsCMCdZECJxbCC4Yox+otdi7wk\\\\\\\/FzvPYp4gaJN\\\\\\\/S4dANGpQo3j7WNeLjPpR\\\\\\\/V66QRTv\\\\\\\/HPWpfGv\\\\\\\/REBQxTNuBD3YzMAWMO0VU7xbe+gxm7crb\\\\\\\/ZWRiyGI2FpVwUqrWhGL9jA4cCPTrjMu9Xqor8rq25653wdJjK\\\\\\\/5x6P8LI35u9OFe\\\\\\\/k4aSg3768swnlsQAYg3AcDysPeb+puyZhSlpZdssCnvb1m3QK1FaU9gpQ0CX4CUaPLsOrG3XHQOEnrjxlg6CZGNMReEDOMhxID6yDm6QdwVJfgpluC+RhU8evBcIfJ6vP2fYEBVm3hbZ8WrTRNytI1pSKjvbwjkm4jZT4TKlL1X4T97909D7Iy3LsfKi62ri1Ut50TFDfh0BWduYn1wZtKoOry7J+4xnP+PSdCGJ1SRGtJO97htvApQYpPb2Xr6fIKy1fFbKxYs5V4v7jgkYzxDhysg+KmrQp1F9him8qjlw33eMS2xOpFjozIOrnlvp5KAwUzuCxAP6ikffW4sXHgDY285uzvqeg3oMlqvfqbgcSPrTOWy6vV8lWgUaY9NxRHxisr+u1Cc+GNvXwSjns7ARslm2\\\\\\\/VZmm9G7vG1PlpI1gmPCH79jT\\\\\\\/StbfeGLBveiJcx0acFWle6VYgsyqr9phFW0IR+HVqVAtD0O6MJZedwDmEAvmfb2bmGxtebboqgIRXjVQr51Djgi2DVmhMgKrNG+Nsx6fAAPjXGN8dtpvknNE56N02H6oVZNp1J\\\\\\\/NjFBgWBfGjKCW38A62OU6g3ZRnAto+0r79Fk+4zlKO4tFAeowapmyXo9uiFnl+dEOmatkhpMW4amMZoCDWX1w4mkLtrLjw8n2z\\\\\\\/o4UyZPXhGn+rVeyEIUs9klWOb9IHY6DSdMDsBD\\\\\\\/oD4W6FTAqE8bIxGyPKJa2w+I04a7T7E5DMPwNnR+yHUOreiWh5o1ztfMhp4as7fLFWL8KaBIHw9NvnrDvCYfOOPiFvKZD8cOjR3iIAGS+\\\\\\\/XHNvLLxdJ8y+BFXTULKO1l0d5nqxy64VmjgMVsUDorvt6GXjv5iE0H3cW+ANkZ7EMz5+n59xbvmOqVEoGw+RXNS8osujHm8W4cZl3uWkQhuaTnev+OOs4F\\\\\\\/helcAwRmfrUsOKF3sJ89q9w6sYrCocNqzgY61LNDtkMzhsjUxc2FUtIFZuNnYB+a7yPxD1Xhadc3mNtR3C+bqbV3VzC1ljUsXBz6\\\\\\\/ZaEY04CVJl7N2Ie2DK2Hqao+lAX\\\\\\\/9A489zSFvHYZDp0wtBxpbnrtkEHVBCKN3E9UjTM0bv0RMksYKtAmOqP3C2kRSYvCeK4iMGyXDrlwsp7Db7ub4wyuZpz362JzoeH3F3DxLuNN4zQ7eq1EveeLTEDoB0fz\\\\\\\/PG105\\\\\\\/rhAT5CljGxZwtlRkUtmeVDI6iOJ9My+atm+NX4dj6V3j6n0RAsm3YJsEtxUtIcc6P\\\\\\\/ygUJdgvA+LlMSHLqkXDXQB5E3DrzjJkFgSQ0KiZ4cl7Wri0O3GVMoHHDJtB3kJgaApvqVhzRHVWk9sF6Gn67tGrgPtC+\\\\\\\/waayyQXYIbRMQCm9G40qXZiatggxakc5gHziOesfs5rgPqWRfssg0WE04z3YrY5gEMWe0DCgxLMWAXRNWPcVROn8q1LNrMJsJYPX8VW1Zbs6h3y\\\\\\\/1zJu0y1qmp7g+cgP0UhXCjWSGVM9AzqxF+PWPCGotiCS1DXNUKFrL0buzQ\\\\\\\/tt8N5BTwDtFArtOdV1vPl0UYOCbU5W+XzOhaKZB+qAnuVaKK2ughq4GyY+7kAYVkM\\\\\\\/g0u8AJClbMIo512ucPrOFQ76mn8pQJ5osTlXEIh\\\\\\\/64IzkBFznbaJ5c6rDjm588MxftQHn9Z1sKL6LJJmK\\\\\\\/vlp98cJyyXMeGrcQUNqv6LSW3Cryt6jVcNkGZoZMgF3SsknGBZcD6NvbfRG3PKqk5b4Ev+do8f9zIrD4I4fbze0mn8WCTUKRQ03gF2aqH9J\\\\\\\/CeYIaBlj\\\\\\\/3YqMQHbNPzURbZJt7cY6XHh\\\\\\\/tfgp+fHK5VpGqjMGPfGLbJ36mmSBIch8Dly+rAXb+LnSpPGbMynF2p7Ys3WfTuaadNK1cPdkaRpryTdiiJr7+pKoI7hKOoLqTWz2O8Dpm5XvDuJDhlxM4wjKWwBS5F9JjcyW5X+WdvnTZeGM0KCLqogdeQ5FTECAmeqE0DcQ5NTa89hTNI8IeqrKAquI5\\\\\\\/hMCF2G0Azfa\\\\\\\/EMhI9rid0zzTCe2FW8EHPsrORbf3ENQrnpjqEmNRNWrPZk6\\\\\\\/phUl5jWcPKynSxP57UnSzyYm7XcM5oS\\\\\\\/oifVfHpspToE3RG7SIAU08vmrX3z8lWW1\\\\\\\/0dInPZmlJDFdrkJjf6wHKQ+EkjWnpCMm7qez70LxEgDjbcnL56tjuXtO75gwmY90dcJDU68xD9i5K8pz37hDaZoaYc5DgvDQNT6nc5ltD5OXVt3zlt24SC+uOxpZ7Kg1+tT\\\\\\\/7EloiFFSahMmRzH1gvEA8\\\\\\\/jJyYzynyO3F1HccSVu1eKeB\\\\\\\/sP0CNLWXH7oo\\\\\\\/blGWZ2LVnVuWBSlPBd27f+R5aHFDlOpCyUBb1ahur9vFj000oG\\\\\\\/5R+HSg0euRzl6lXM+wqhoJrnNfd07DbvUz4KJMKR4TQp3xZwWE2HXXlzF8xvyhhIZKcs\\\\\\\/0JDpRUSOxwIohGbxc6sgbaRMevGSO6G11WMQ05HZB7Ta+ivGDrszf81rfcw+UfEJFmE4nYrsTZyIY1RxtBJJlt00FLPWzBxEdMK2kUPKXD12\\\\\\\/Ua1zeCOdTfrsrtEZ+0TXFBd635caqgsEoci4ODWZgMhVYw3TphAR0YtFc7lrS+3vDWgh92YtpA\\\\\\\/+aoxbv9GERGD2xCV7v4U0Y+q83QMBl\\\\\\\/pXWF2nzfid8BA+6wJgXY8a63kstwgIvvQ6kHcE5ZSa7Z0biDqWmZ5x5qz7Pon+3BVFmY4BbbVGsg3Kr+synZBd9nC6aMiVuTLNYTJy1mZCoMJQ73MCL0WcH6i52TUxmQAGEtyLQ5YPQXlkHXcJ6IAXNymdEUMx3TTnqUL3QdElNkZSozYMWTc5hmdS5xfPT2ZUe2PtJ+4E\\\\\\\/WDJRwImIm4u37l+h53QyNPLccRFPvJGRjfA6UjTOy4gRyaN13xpSVMSJJCSvfNaTUwj5KzWS1OumT+FaJ4bjqB3kqqELD3ABewyxanK1j0\\\\\\\/QpzEu8y9VgMyVrlLKgLXrV8nRCYW2hjzG6RWwGUyjRebonbPVY0e5K131YdD6EzfhIKVhK30tIKt56vNjWZT2FM58Ajgf4KJC7sDmf\\\\\\\/p7HrTibOltVnEFLu10ohOmcRVawNRNOVn+RC16\\\\\\\/c7Prq0xeP4F+0fgrnLlYRphG\\\\\\\/Blx7MvZQK7X04DrlxiQ\\\\\\\/l0xns7z++YMN1f6w84D5giA+aE4oCep4jQBjRhPSVAAhMtOy6qhCnvS2zeP\\\\\\\/llMljJgP5eN05+t5Ea51H\\\\\\\/4hFjj6BvmQZT5OMpIJgvk5SdSxsxssJypD3RnYMQJJrlbW1I+tRSJejYNCLe4ySJKh4M+pIsuA\\\\\\\/AAWW06LxtJs5+R+595xbNm0IRyYg\\\\\\\/tDIvSrR8JwLLvwo7BoSOz\\\\\\\/hdEzTyK53wkahF\\\\\\\/G1zqthMhmC3nBA1wnUTMcSXiu+QaRpN+iU2bdn4Wh8NlJ0klQe+0AEr4IhNewLv7\\\\\\\/DIPzz2DxLhY6LKC3NEgwXQ0lWk+ac4VAwazZ0i\\\\\\\/4iXIjHsaJ8QtGcvqixT8TQwLMWIS1OKfoFix8mTYeCljJyIOkCz9VOr78uEcq8eDINhLGDKIgRaUfWSOull+x0Q3IQdqefV\\\\\\\/JZyxaksplu7j16VECTSdJrRuIFBclt2DYFkvDIE2WUJtbQxmPeRC9afNZm38BXsQsBiWB7f2u\\\\\\\/5qvL6pG3V7AS\\\\\\\/+Gcs6zP3pM2zi+fwVdtIdMz6t8jYLFp0V8W53QtLkBj5M2bpUVscVKtCjEk9mNAhWMYRZoi2B2KVOFFFKjune71PSkmUwStwweuaQ4\\\\\\\/EvXrVs4fljr1SyoGh3805OnZ6edI9laC9GmSYovTfv2qA6s4aQZFG0OIRgAGuGl7Iqu2cafQi2TFFpJnhb40lECiuWZEg+LBISr84Gz+BhNOSsqvcSIrkGaeos4HSj9tuBsjSv5L+SqrLVzJTQdsc4qdY88J751iottOA87PCawemMN8NXn24EsAjE7tXtLsLJmC7\\\\\\\/VcJFb9eXzTQQZDFuhIHhcoDa2+ugJn5lVjgcaH\\\\\\\/Uh0LfrweydcAEjaYGq+\\\\\\\/Ka2vd2K3OFWwVLRKUV9TbTOBNhvEaf5PmDgo\\\\\\\/3hNaGTib39te4BYPOyxpNcGT\\\\\\\/7uZBJAidDWpokCTA+0RRMkjGkSpGEF8YToABX+cQI5fUeTdlyOCP0uYZ7sKjzv+x78GY8zCL+nVzMPiCOEWfh6UrCKNoRoG8so6CH67zcIFVMLX3JZk8MYN7srJFd9mE+fcOpN3xD2iRvGqN+XnMNnDOEjBoSOILiGM1YvGV+aGdmiYCIusOjBc\\\\\\\/if2y1WythB9zk+V\\\\\\\/BDIyDDbEZRmG6BG\\\\\\\/CjxxhOmWftZ02Zed7ZWdLCqUPkreUdvGK7MLi5fc3Np2eXCDhSra5YozNVBaEWJ3IJ917JLfCIjq\\\\\\\/X1VR1MVX9mbcwZGD8WOM9UX3v+7SalZyWK3zGZyTbhJSDODu3YT\\\\\\\/PsfrJTDOkw121U5CJ5Q2tKN3i2tJCAGV2Q+wJinpi62ZIWYUZ\\\\\\\/JzYJfIKLPWRJzZVBjMrrxtuKtMjRTNTOHgvSNLd572O8eGcXBVAr0BpVAyCpA4N+p3wIaUXKhjR+vx7ipU7fljd6s5iPsNcjjOtoWMGkioI7t5G0dKi1qKSJGTzm9wROS1MzV7AWHF+qly4xkT72BJQUWNnb42NdDe0VqEdIhGK9jXHIMIgOjWQz9mY6OEKcknWhlxGIeGN2b8iZStQM0JWlG0u4STywFHHZEckZsFhg6q2JRtqdjyInKOQISzMea1o0cCGe79YORiT6djJl13AcR84bhUcbP8yY0on0yrOm26XOO3E4EnwSCSQNVF51weJA\\\\\\\/IvWySrTvRV2tshfAL2SGTWSoTkglWdCFqtMznOa1Vw\\\\\\\/A9LR2pMpBw6lnqS0W0laJdKy7h9DZzCXmqnD8EcflvM2HxmChPsjrzNNW3AXhpiOitaoPIa4j\\\\\\\/uDDq3FTJFpDfNOXzeJmSYkzUsmAuxh6YYiNPsyyHyLMzfU6oOnm+JCmkcyb\\\\\\\/AbSj\\\\\\\/uunDHkIX44mekhxVM2+zmQTsBZE8QZkntIqXOr81h6h9G+YyDaWEVmy+WC4rG\\\\\\\/Go48D\\\\\\\/ClQzfiR\\\\\\\/ADJTR\\\\\\\/jwulYAHDwFjLd66332nW7phQIz5vlOvZfLNK+LnB77FiZHBxZnhzP4Q0WrO7MNQR+pD1Wjjz8A936fkzJpFf\\\\\\\/8crGq0VGozvTNItwb8g400GAm9DO6J0moBnRUzKkj7S5mDIQ8OpjdKQbJI50CSTc5Rb4WKQcF6+0DyhsVpMKlu5582j6s1QBVYopS\\\\\\\/irMC8H6ZujaaqvW142yb4X0zkn3xYV5tKLJq41kobcLyWmHcaDoyL395rk1Oc3mvrSXWrC8o72HCz2wCCUOZ7azpceCqqSxbFQHUx1Imgd7C6BdYP3dR9vROmkAmU0F39JdNKPuxzz3zn4ABtXcFiSbmy8aeQsaxeKTCSMhsN6Jkqqpoyqsac4JIKwPQCzOsolfJEX1+9JLHdnpId0IUYi\\\\\\\/VqKetKMT6JeIohlin+lIAGK5rPSFxUNgh6vQY\\\\\\\/sgIBkHT04xGUqUjCoEY6rurW3uLRvXxSrssA0mKqqZz56470LgKWbIxxlZ7kLV9\\\\\\\/FHHlliM017npSXGN5q63zSsDGhOmDG3pkjCUatvLcC0pkiVYLKa92xcNjCLe99BGu+9eFhA1JxZFONxnnlu0Anjb0XfGtKODbf4VWlYIIvkPenz4IZ9XYDhcwnphdtyq5TiCpMc2cmNtB7yeaxe4KFdF+zKy0IqnayaUtIAbmXbSWN5yBngJ0LelrRm+ebcT89UeeGqW0MgUbkYSk7lRirDHEIUrXmZrqssV9xEV8JTEQnjFweWR6oHveIiua+NTOOanhc5Qnr+oRTmoULPOAp\\\\\\\/j7XBoqYo0FeS9\\\\\\\/zt49QJM9u9cPgTrKM5rMjHUCnBh1fx1585q70E5UPg773Z3gvMrkiKFX56tghXS3MKOxurpHgpWHlLsD\\\\\\\/S9C+DBgFHqW00ePdfpPeTcwc60rqfWY5TM+c6Ud2Dsjq39UWZBA0uWqE1ue5+L3EV0eLGbJtq19AknVwHrhYaF3wPJfsBIrD\\\\\\\/LJMGRkePF9os5oTcdY9TdgdEpDd6h5u61pqd6NUJKDQak52z40SJBSYIGobSYi5XWMMQUSJLCDG3Lfhmyx\\\\\\\/MS7QPciSgqaohoqfhrCMvaGNoVWep+SwOu41n8vz66AEyHSaWAUD8mZOFrhqDpAziTX\\\\\\\/C\\\\\\\/ZIrS9nEKw9Nw0oM8ixavwLOK4l\\\\\\\/Ipxe7\\\\\\\/efaNUOSZ8el59h3HlQT+Tu5oiMWB7G6to33Z2f8AJ1j9BStpenyLwEIhCCvJIz67D5iLF2v9Gq6LVL196+DNygsq1TLbSB9YgUQYVSaXmaUzFixb4QMqk\\\\\\\/ruUxmE29YL8LD5XBfOPbeIeN4LWrJKi541DxvWCk+Dn5E0Ie0Z1CPkNjLAahgctCl9b\\\\\\\/WRMPjVhvFLa1x8m5BlxMAf5afs8C\\\\\\\/u+wncvT29bCVGbh\\\\\\\/nL4X7e1Hi9Mi6UPfsk\\\\\\\/7h9pp7heYnA9ueTkLg2Idn28sYbzGt3pJgAhaiOi\\\\\\\/LMXWU15zKWm0Wk13\\\\\\\/di1yE4TDRKpxmpBTngSzNDIfaOyWiPcilsdYAu4owO1rw5Y4frbzQ4JoBR4Z6EjzqdM4TzcNhnYCjRpD6XMaoQRoYVY6A4SzFj0aYbMuRiEY3HLgXO0w94GvJwJFMPZE5FLVd3sKayeRzFycd7pbNjESKBiYT\\\\\\\/BbBEEDH+QzitqJB37WIvlTynP3Fmjnru9mdpq83NWRKy+bAaZxLKNSTm\\\\\\\/fFN0nX1VzPNtSKv0QrzP0Q3G2w6LaQuwXrotrWctcQUpjk8lxxSAwdqqNjGAjUJbevsah5xi+XT1ioPZPtR0tHRzhTXki41+Gl2ndeJMmq34zk1WFZXQL4vgPulbMmiZPdQgRNAL3b3WRylDwmVkrCbXPJbsnxgPMSzMqU0Wk2ylDnuVnYM27LO6xN0E9KsLL\\\\\\\/aVu8indoU57vqNSJEb\\\\\\\/3LpO9jEWnPiN3ChO+WB9DFvKUOGUZy7I+CJ+sW08Gl8\\\\\\\/NY9h7K\\\\\\\/QWLqrgY4L4rZf1Aof4JWuz0H9yg5CnEx+ZTyk1qrBBno7g4\\\\\\\/s7I7IeGgXhjzBtIzcoH8pO9d1tGaDVYoFHtQYbKhM5QJgEbvI26mlwyXQDVxd7BsGKbGSFNn\\\\\\\/NcW9elfCnymOkf5YAFbB9KkCgJ29GP3ttAjj8S89aYwSLB2KRTKujFjDqQGHrJlqfH8MmfbpBCyrF\\\\\\\/MJoDXqDk9NF3Un6e42w3fP3gXLSQI9JgI10D3dbULwC64nWGlAsKAP9dB8fryilI+8URDstIxl0XXF3rNH1ZwALBuGgiTB3gbLprhrumikgoWy2MutMpCCwb3k8vOpI6fT31trCSInIWfJFdqKd7wkYpG8191aTXLros95yYJ770DJt8pSwuH7A7WGFP2XYERedE53u5bVLuFWwoTwE\\\\\\\/OaviIFhhfJL778Rl0JbhaUZi5OpDw+KXuXpc4CYcuyt2Hh5Jk9QQR+BCtI28isynDHPUNHaytR0YOKoa2CJZPGIoAtu\\\\\\\/LP95xxwLpAdJFlo8prNeLmrl8mX8eKIbetVuMw\\\\\\\/+IdErKJl7HuwEXMkiezpImi+0rDkFUiekoxO5zVkyRuqgh\\\\\\\/BeFJKmU9vRPakbE0JfJBGaLPPTB8fRO1VNF8GOUuYXvLKnXG9zVXDINFCyLJY0NUZ+woGe3KJ6\\\\\\\/VymuBRAqqMCE3M1yb6ZtQPk89oBuhkzbfUQBRp1b0nViWSJNl5+VIEuWoKrzlpbpOih11aZ2TM0yp\\\\\\\/DFxjtrKJcchTOZbPBamlypnak+IlwwgS3tUSITC9L4s9sNHTr5fcrf3szKj2+VQGZaNhQXr4Uaz7HmoqQAjIs596O0bi3V7xX8jwrQsu1buMoWzTj+cF5eXybAC82OM2VzIPsbEknUGiHjqIxrFy2N6PjF4EMGX3Z+mPW0kgSrhGhetG0RxRE6MzL4luljxAwXghfB6GZDaZAByLJyFUDdCiPQQ3gJja1ladDITl5QPuBsV40\\\\\\\/VGqq2VKVD6E33BZC\\\\\\\/zoP9BsPTPlfi3ZcYS3WPcSxkMY2FpE5LY0e97oayDL5pG2fAj+3wEPC1OKzW8BNAFm+Y73VuiPYZuXFxL1TdMvJZuk1yUf\\\\\\\/lLUhw3vIBm2xq7Cvnach3rty8japd6yA+ZP\\\\\\\/ABCUfoZkW5YH3VETbsmoyXc+4YfdeTJCn7vCbv5GM9bh8KAy8JEIRTPPWJtsBPlUe+DbntVXFVIupmo3\\\\\\\/T\\\\\\\/3U1VxfUcla51\\\\\\\/InLEba9qX\\\\\\\/Lt340ew0hWQr2TzdAWu5jTOMtUbqnp1ueCDa2QeG9eOgKpJoPh\\\\\\\/U68FNDIUsv1plMwVtpersIuTOAv\\\\\\\/NDcFP8E7dxjmagel+2PUXEq2og4e2B7kLMi2UbKxoRAPpJ5hFQFZ20Jq7CdD39Jf6tGtqUAMBT9n9QKossaba3\\\\\\\/GM7fZ2ZG1WTUjYQJxQ0sXr1HKXXBopKTytKIaC0\\\\\\\/eM0wcKEbHZV4GlU5\\\\\\\/4yh8xV\\\\\\\/DFYv6bbgnrYoasqhnCEYehEJ3FzJMM\\\\\\\/RsY49oKR7V0hB35t+dEj1PFr5mG8BGt5Z42YRuxkvGbAaxJalLvmzq9ksgSyVsw0IcANYCcyu+E9dtM6K2M6hl7hIPoWTPnEyUd\\\\\\\/M8CoD0Iy4D1XkslfCwaFhG0Q5gpxVmIzWsBMYxJ\\\\\\\/\\\\\\\/qZMwt7NQY\\\\\\\/7HrzU+am7vUcDmSXr9eib0cuGhJBn7pop96J5PSLPZcBpXjLcJsLTG8NpR55ukgyzfuw6800xOv7wMuOscnHycWk\\\\\\\/t+iO\\\\\\\/XetGyVyjKC8qop1vhwEt73SihW5w8R8shpIvp3tbz7uArX8QUx8ogkcWA9U8U\\\\\\\/\\\\\\\/rwWR0yH6qASYrCj6bUEIvxSzPVqSSn3EFbudiPM0M5D9dVIPknECVcxi11OAAoa0xO75C49sgWQOcmzBwP+0X1Spv9LZlW2kCKbGcRfBJTlxghEM4DGUBSgEkF5s8YVcPmrjkA+pfmk08rad0O6oBOIpCjlaT5OTK770ayTv\\\\\\\/15o9wz4QD3q5xpWiANHivAhoR0RF39N2jJIfmaLR6uJ2crCYdMhcJUalYASmTy1o698vSAyOncxLCTO7WPvTQ95iFqCcCkYk3hywiUBnbs1uaLG4hXpMFdSmhqcF9VnIt+zPyyVW4eskxRBEmu+lVv783GNdB1PhimnuEdGzkenFT2kNwysFaXGsltQQ37AWhMHgCQZFqKagjC4MLjoXorw1l7Pz9L2CJxOM1c8v8VtpdtF\\\\\\\/Rhg2I5YlzCBSKt3fUs77KXdzYS56knMDvE8CH78nBlcSs2hyURl3lsGZAl1fult8UhqYTGrvDBsy534XeNugIjpOo29ej0eB\\\\\\\/QoTnIsxT0\\\\\\\/MBCN1IhCEWTvOV2pTE\\\\\\\/bg8CrKH8tKQJSiJDIG2TLxf1jMoozeRsJzXeqScb7J0xiOnkQNOwes5rHBWrv87nKHnbxU0Veb0ME\\\\\\\/kKIMu8hMB7IBBQxCL8CkvNqONr4\\\\\\\/QSyLcI5cl9vAO7p8FaLOPL6muy3vOJ33NHXtScH9VNclpHIM40NIU0bnfPMUBlclhNbGvQRdT3eouSRUR32fFfP2ap9+POClsnc3jIEXCUS5c1CqgKt0m3CtDOYc4DAX74lHb\\\\\\\/DkzkpCT3bb1Z9gkpN9Ve5zGWQF70Fp+M\\\\\\\/ZBm26f1r3\\\\\\\/Sf6D7n+CjGDDaYGsaidLiZGWZEK9BZmI7St6sq1iBYn\\\\\\\/g7HaMlrb7+mpBR1gCyBM76ZbmHO\\\\\\\/rOcRfD\\\\\\\/mawZnhtIUp65TFV+bn1FpJxxWKebOcQFoov9x0ca0Qu+zJrBXc1y6fRXHtY+oyDaxZK82Qp4YNXeINrJH8RoRTb18PhjtnEqDmIg05npnAdol7gy6QFa0jVOj5oFcpdVVi2ywmxlxQs4egNGFkLHULIeJ8R9aqbYgw3YJ+NwG\\\\\\\/j1UC2dX+GMUBXe\\\\\\\/qzfdUZY7+b4WEiJ0w0tETtNf3k1GNA9EMSA8dmIy99THNcBL7f10gbWEuDICNzESn5LtsSh52vLhFGQQsqymak3NKTvxzE4aoFFzOgX+L9dPVoOET+1bXVPJZHKhmvjJ\\\\\\\/49U4Sz2ne5ygLAVimLVmjvFwk7KQgyY0cUgk8+Sd3dIOkgOXbUDcYVNdUyui1+OQrx+QajtuT1SIRpRwy7Nj1MNVY6D4cbL5GqzZ8Li7FOxyDVntBHAp5xFZTkkmEQStfSm8Czlo8+RFTZtenvucgWNHeYE4CJMYIucaMSE9E1Ijm9qBhx2BqwG3G2v15Hrg4u55kI4km+E2jYkQvMfvESC2KeVjjeQdaSzbP2UiAvimOSyTAkIDl3qPRKFx6J9xaW9j+HHIY+p3mZCxuFvqkDuUFaXXuZUN6J3vZnyyNxY+NTF2YKZiYrqFKZu\\\\\\\/fsPMTS368u2xykJrSHBv4mCEr9CEJEA3u7mpG8fzZ08tWeYrcLTXtzcgJnUGVLyU5LBuIhCUbACq54MlFpK4a6IAOsuK5XZeVKBZ0dLz9ktYOzMJz3Ff0uJajz4UItZMDXsLLPMK8SpxkvMwNXX1+J3QbLIEPVjGxSOhmHbbDRWTau23M+2q6KYCxsC+NELc2ThOCdk5QMxux5st\\\\\\\/BlP0uwFGZNr\\\\\\\/uQ0QkodBneA9nglWpSHMl2qD\\\\\\\/0jDN5BNZjdJNHOLsdq1w5HnG57tXNzhSbRemkR1k1kx42F62fYn3AXNbMdUrxmjSxpcIhxgAGzlIbq0+4q9Aq+qqRcGGF+vmYjECD5E5gY8T6TxM6eaFHCsrLUXwUMrxDsFI8EyrxzTtIRYaA6w4iDOjTWUfj9mduNM+1ucBDyW6H\\\\\\\/rtx9ZBJkyd4Ej+lOFj5VLPspD7lnG6ZKSuBJ3+yOITZpkcurjmUqGzWKsD7j7Oxe7FyPk\\\\\\\/D7\\\\\\\/CTHa7mOwO3tEnaUZ+QJW2Rpvy0RwHbp9FCKQS3kwtfE\\\\\\\/zq67HvXZshF+TF9LOkSSNWIFwls9nzVYCPX736\\\\\\\/o2w+eBfTrQ6BxQ6Z\\\\\\\/Eyqfms+iEs2JvPxKtA9brOuBBJelUlesw38mIGemuetFlT0DUFfwPjP2dKFaq6rvKVktG0sZe+8xSkhb1PsEASJUo1bh4powt+\\\\\\\/t+\\\\\\\/ZGq3sApFIlWuXWUWCdBtuosHvEjaiXDGQdsz0uWL4KmYDovwQMYwEarv8zyRbLn72DkIeNjg8MPqMOrHcyBmEaZqnWrugyRaNts6Yk61GxnzR668ihExeU7qnfN9+4NDoI3Zt3yEafaVvljXDn4OBAv7eYhqAJ3tppXJY5XcVtODPmROvVCzVJwm9sF0trLQAxm+VTksDagdI6Zo8OoIihyOSEOisoOfsJF0sAp5uyI0oyUCnlNIzy\\\\\\\/\\\\\\\/+4TYezpEN05KAV41UCqV7ivznC3skHxpm3bKCfw4w9NHIprhqShxqdKCuDsrEpJ11HDqCpjdMABHmVbyZVS5dSJ3SA\\\\\\\/Ia6f5jOGGJKK3IvKT+EudR19HKkBdxGHj2UWSVz94Iz0axLgoxGg6D4K1ZrwC5gSgwTTfmefSagZzpoXc6XlI4LaEd2lF+KdFnmVjl\\\\\\\/Jahs4I5CRfb8k+TPsH\\\\\\\/lPJNJwN1iAOA7yx4MHgn\\\\\\\/4Cyc2zsZt7DusQ52h9RgwmgapjLQ1xA5HXtAFR9zmcU8\\\\\\\/VvAHzBOecspIfrbSbbHWwx1S1W2122YuF+hFCZL7QAe3GTRd8BibFOS72+NgRoX1I634HlrX3ScMqfJ0y730QTBq6aCfEhBSU6M3LtXXTjNMdwpbHDsWGdDOImqRy6XQ7uHROUexvSP8k7QPMZE9ExYF3fLYK6ecyeMfHeFS\\\\\\\/FslqD0SnytoQAHtXswuXB3QLPrORI65CptG13CRyCOXjnYGW\\\\\\\/mujNGAU\\\\\\\/NLTqlam1T3Tr0CLZ44gMcmt+Rdw6palBwHUzdt2yHyTHf8HJNLAd1r3tpAHzLZAyRwKXJChtG1zcG8wEdZA7M\\\\\\\/g+lZeqj4QAmqHcBl1xI11iDS\\\\\\\/X44yGzDtRcvKi7XrrI9uP5FWlM930rirRYATKZI98p86\\\\\\\/erdRymc0tqB0\\\\\\\/vxwsU\\\\\\\/W+XXESh\\\\\\\/5cb\\\\\\\/MUdDf0SVzrhGYsoA0ad\\\\\\\/gX+tm92oRbjZvaZozZ6D2sAmsu+jE60rUHQVNiGK1y1Y43Or6Nk4A5VzYzMsWgNqHndtvoRPgL\\\\\\\/dtZaoP8769fwfkLt3qjPZaMm3J60kTjkW4iNWIaTgLjFSrf2Mv\\\\\\\/yhOhGX12w0BY9HCThlyLqkLOaHpDboeqZXXOsFOAbfFe9o30\\\\\\\/CD\\\\\\\/NAAd9hjvDIg2bcZNugm+8qOPMdCUHsHjReZ1YuFh7tYgsONQT05m4e1OOjn7uB5bgXghhv+121htPLdMM4a6VQYyKLf6chaMTO3HUN6fLDV5YHEeRdJMFQV0NNL\\\\\\\/UIhOAEM3myNreVULcyiZAIO\\\\\\\/aRL+taZhHfXuAGsvCCQtD2ETkFdBbh0QZIlWnr7chdPK+CRdjlY5rjLH2wlD54zlqCWmpj0VMnU7bJLyaN0nNleVyFzvOYvOGUHkZwD4PaDs0Atg4yZ7EEOSkufwjqGc9YZZbkd1QiEJu\\\\\\\/2\\\\\\\/gOtdrJjZvzpgp1KiLsWTGcXKxjb413gmqokSUhzJB2rfwZNHld8N4A+HddBj\\\\\\\/uMWnrBsATfyrDyoPr9uJ+LxHRlQImdnS8+RZtK5oT4vpy7ZnCsdJNhR1hAz1T2+vZ1HGAZiLC145KhthdL5y7ZHrREygFr3x1dvqPup8QA1oppIeKKj3VxLH3Pp9Rbg07A8bVL+mNsM2OcxCwuBAaLpxchS56z3bZcX4W0pAuyBPAwoLeOhdqWtNcBLj0mrPNtiW34BhFY+HFwglo4PiWDVFdnL5K7vZXPnvOWCkdvqRp5wgsX4b9nXKDK2pu0d71mKuuUhpkkosYDX7pjvBfySq6KzWQSeme\\\\\\\/4rbbUkP6z3XokNYTfcaQn3nRPV3p4tk4ZODfs0KEI9HZVWYQkUBJtgZ1t0FRatXnqHQjVu7YFmSgz0WadR08\\\\\\\/uhDiTVhZolHH9iX0yPwxN3i1Gtp6L7VH5tcTHYj71L4P9ta+X3xu0NAbpEo0oPPQd6MGygofIFOYGj9XZixUIfxfjv8lpTAZ+J8j21+hEKoDY6bUdSk4FwB2DMyaiIB0KrG3xaVUaOt4LWmS4clKT9vCvz5rSY8heSfNAfk5utfHQnNpW0jrXGrkMTcIGij\\\\\\\/djUyV+j0Bq893aZ0ogtneW9mOrqCEwxf0pVaacH50D203wXx1rwePXO610iunmvHVfeqs76vNszpcUAjSzfM4vhPloq0slVxrOdZqynvgraOzo+R0Zvy1P1zo6oeGUH74vAOScgLXHQLcV5PXhSoIe3bUviYXTxqU4a4UzpwB\\\\\\\/W8EpugLOmKPf6aoh5YdW1FCwqEnNjOVyj5+PuT\\\\\\\/A2CfiBIt5kaNu0ii2UZ4eIMX8ig2\\\\\\\/+xEe7kU1FfJysIBQ3QviELHZyZY+zW6e+DXjlWyVMLtCEJ+9uoIrGcAeK7BGKmUN1\\\\\\\/6YYI0A4TH9ElUZIBhm69t6Bqtc\\\\\\\/dW3nj0qmL4ApMZjUNfcRPLkdA8ewkIbWFiD4UlHDyf+y9m\\\\\\\/gAJ00C2oBsEyowmFo\\\\\\\/ncCAY\\\\\\\/\\\\\\\/iFfzM1DY\\\\\\\/bTVu4qNCuy\\\\\\\/l2izAQeOvYmFLllm\\\\\\\/pfl3tgI6SPEzU2jhHY8WVVemm4opN5tV+cNK7G+BzSbaM6Dua1bF56ll8pkfFYkkAZ45PZmN+ixuE6xHJoxHJ9037A\\\\\\\/xXlJqkoGUVGBz9upo2i1q06zYEMa6mfhAlREw9XPV+zHazqBgmwZDozkIOXrtvLO7GIzoTR\\\\\\\/eqTAKWOUWW6fXeYOd7bQG3XXQ353Q9OWq3mHtf4sj6UUR93My6h+lvFFUI548GZVfN\\\\\\\/Oz2qg4aBsM7g+xPDnfCkJjsRqsJRJQ2vWu\\\\\\\/C9XJN++0hy6PRz7pJV2UVvcyI1D+It2rGkkabr0uG+O\\\\\\\/8xHBAbHPhviiAuKCmdxjaQUGeHls2ZWqsOBtmyVx7vf\\\\\\\/fPHkm7CADSKjX6n960\\\\\\\/m\\\\\\\/+eyITLnk1kVpPIccAoAAU6nuqIk1NQx+n6gMKsPD\\\\\\\/6JZ1TxTt4u02OsAcgnfj9GUcLpvDWFX1yT1GKyMtGg1oC439yDbC\\\\\\\/mMmXqV\\\\\\\/BjnJIZaXP71kffoy\\\\\\\/Xov5UsQJnIyII2HWlYIM9sTUn8ZKbND6ZkJ4H\\\\\\\/F32m2w4UT6DyjLTDFZPN7PCeD8zdsqNMVhFeN8Kk8wXIaP6y\\\\\\\/+7pJxN2IH4sWbEP4bgKQ2XjnakxPhx0iD7wGVKs308nulQE37FbeEnK5xpIQz9PLYjd\\\\\\\/keh1h5+Q70gjv16kxksFtfsldkAYL+3fhdEUrPwb6GuL0rbazpM8poogaAnNu2JWAG0QsUQL6XeKlGPm+sELtnH92QD4ezrz5GcanLtGR4olgZR7PV8oaQmtO5JkRg7yTHGHdgfs4R\\\\\\\/iwM3uuLJohEUrHIONnIZZ2bEnZrCLMZFGxXsvmh4+bhlI8P15Q2Ysfk5OWGyT0OeTXBOlJ7KRkWGMHVsy7JrVfERT3TmFLwJQiFL5ChFiH17WYJeD+Rwc79XPlbQpWsNAasCsTKHxYFrDeyPLBF5oAfI+MLDPj0IcZ3w7bavwfgw8FEU2C7lUl2TWlZpu9qoIfD9YER0QyUWbEukQzN3zRXkCly8vbafXLvceb6o8KWETgb94jazk6q9XlGzlf5EuvNpVsQFz0cplptKP0XCprVBoohfGqXs+gHlXMJvO2our8KmtwGGYWzmkgzAegtDpk8POFrCeq6VJBlbLk14ra4QI7iXSJHiTJQkBGHh3Oyf873vMcrESiiDbQcSqx6sodMYNuzDE6Ir\\\\\\\/EZLq5egfVuLrawjHdRd1GNdN6rNwUSXsPv1nYMtgzlC\\\\\\\/id+9MSQFGStBdlobeOwMJn5UZosQqk246QKfpClGiVQdUvSBgX2yNyAlTM3PzrjfF8qEWfiahO\\\\\\\/BU4E7VIWFR0L0A7g+E4TwsO6xhHPrDD2rs84CSc+gpfTuH5K+aSeG9p2HETsBy5szuwIQXtoEZkpqyUEB0xomHTsqBOdLRzw84cq8\\\\\\\/DxRmy267Wblaptfk1vO8Xzb3ViWmv69I43ZW9n1hdx\\\\\\\/Kei9AgEQK0jypTpEnYNseHbWtN2Odjort76SlpjGNfODQnDGfql+1sfgjL94PCjNa5PYEzHcMRRfRgjUlBBe4Wp7d1TT8Ciq\\\\\\\/ieJDcIdPPs5AWym1Jy71ShD7IHnCEHmtkK618uuTByOR8wmuyGsptpo2sYwH0XpDpswLXX+oCRUIzo1MJdrcLg013suqhXUvazwKN5u1vy0wkiiQ4uPH2fKnEeatUZqoE\\\\\\\/uDOe8Ls8QVAXx72SgvGuh+KQ5q2rLYXUS+gEYXY2tGsOxEFPkABup5G5VCAWoAal45jvUCkgrpdE+3E98xrnsi\\\\\\\/kqlVm6w0PkZgbo9Aqhr+m\\\\\\\/ZAN4oXTn5czCaNQwrnYhNJY9JC4IGMl2l9Ja8R+pWxuR\\\\\\\/7G\\\\\\\/MGbQQst4Nhmy4XbEFcP4yscG+EgYwJytHYF4nXImOsMFl7czw9AX1nKPYVs9MGL20Wn+O4Vw9\\\\\\\/KBRxzxldJL7TI4MQvlwJ61bUnb8YLEAKezJP7DbX6rEZY1iXFatGc+SCyAxImD5A0qrtGcOOmNf0cpjsflZj3nNZeJ4s3mi9v\\\\\\\/JKiZUPji5m+hM\\\\\\\/NlShQMpla8WnidHSLfDTElaM6Vwd8XOGcKsrb9Ii6SzNn5OReF4YygkIrn3dbpIJfY8UKRkJR0r8gzL4yqfTZBl0lc\\\\\\\/CaLJ+2bGjzIZqVPj\\\\\\\/Aa409KPfbP0pn3vWF+SZJ\\\\\\\/CqbDjzRyu1JXZwd9jRUUYtZJt6sEE8UCOwYj4IyRlFQMZdpadkRp11XQ3ayqr30TTSyZczWM9wEk4ZFhJTEwzXGinMzk59L\\\\\\\/2NIxphueGnfGTZX86qSyKeaogCtnB6R5KRryeiD+A3vo9d7cJl3JVbz0Xn0bO5oeohXNoi8vm3WQ\\\\\\\/gLYSwrDaVKmVFD+uB+LpRtT+vAJ+Qt3IdMULeZ2YgXP2lF8uwqs1phvU16RWohRcDFArQaunHk0rB1WtIOlflc7BprFCOjcZOozsCf+47lHHJdUgHFhdS+GlROAzcIH2Y1Cpkg67V2c7cwQdBt3hbut+mTcRJLOAXpvm0MCwJP2nqM+\\\\\\\/dc\\\\\\\/AZLI+lwZotMGVr1+8+fS0IzlwL9S+f\\\\\\\/zWFRGJq3ZHyIMrP86k7kAD1abWNJHoRcz\\\\\\\/YRnFko8Ote7C+ylbYVmiM5MBpJxJw7hPUtGKVAu8yctq1InGAbRDr05nmo+3jqKSuyKrsbPjUkEO8MTzn70zolmlHrUq8RZYHUgZhMpSmmN0gVvtZJ\\\\\\\/MvxkLNXbUYyCQCS9duHqx0wJ3p9PEyZF\\\\\\\/BW8LpvDTr4xYQWiq6u+nhHivNloW4IhZRbz1eNjnUD+wDvZ\\\\\\\/FirU\\\\\\\/DE73N7MFyCpSOzSah+RnsNKy9Y\\\\\\\/Xat8WM3rG72ifT5DWTm870TEXrGxQjYtvUxp9aGmmKqQtS3GbI9Ry0wlx81QAxzjneEa6tW09p4nrEKUWFPZKMhA+Vw7edFdDkzG9n9cnBA0d3e+BavTLFUghGicvT3tXi7JU0n+GD6DeqW6QjVEP0Z2cK\\\\\\\/YDV1gqUIIDJkkLsjl1fnhPVSfgeMcdcHzN7JJ6oXkklLrIawpBf5I3nxHFKPD1fUGHDSAM4jTCAn7UgV\\\\\\\/1vh90Jo01Pk91Pal4CB0hOK9N3DVw+dm7nHNFIueBTZvV7FCVktOFrkkPLmiJHS3oEW8WrivKYBcpmVH7pJj5dohvGl8vCTvsRQUq4ST2CuNzCRvQga+XywSk8c1bzeTGi5o3d5ltuP71uMVncQfqMTVgPaNqJNomMBlrhdj\\\\\\\/Z3vzXluxBp\\\\\\\/HhpqheciMaasiNrejH6pqYPdktchwT9SMe\\\\\\\/+MCU3utlZESjt1hUpq\\\\\\\/G+qVPlPX0D0iydaQMhuG9aTxfy4qUrjGiTb2Sff3\\\\\\\/9eA+\\\\\\\/Bzy53pmnlHtipM3OmBGRz1TTMbuadFj8986ov0btgCHW7ZpPb34cUvpcEM2kIAyVLVyLziSp2asFw80diXKqo+Ls1kCcSQYLhQd9BLD9D9PXZUtmO8QjGWfHGa3bbPgMpgW5RvdT5uO7I8Nl9uvYomn3xfs1uOi88fM91kHxi2S\\\\\\\/q8lfpw90BYHFc7HLQPULoIdML9tBqGHPsHOygjS12uC0tRf0JECTbdyUmYAZ4SC759Yd12qUT0VASWA7YPjGzilGxiku0zUBJALpP79a\\\\\\\/cb4ht5rF11+bDDne4QCvybNb8UzudcauwUXSEmm7G2ubLkqYzv4XCnXzZ8ENL8z+\\\\\\\/cBjH\\\\\\\/\\\\\\\/X6OWWK0xjeMj5ZtQ+rYAaKskEnJFME25DFYWBriLykBk\\\\\\\/PraQEDJguUAisfGXBLHbUv5XW3kBpPijOe+qNz38aPvI2l1y4SlckZTckoEPOgjl1FKbDZX3RQSyslz0UjYqiPBUdhpSFwpY5Iptx5hRtbyWeiVwn5aU2KyJ8ll5RdT6yLgDFqbC2Pc7JFEEhY4ZJU0G1jZc0dLR\\\\\\\/1du+t8nyt4FMrJcjIiFjZ6gv5Ud7kwkombkeU9n5w+\\\\\\\/OSv\\\\\\\/8INy2UWZYNAaC\\\\\\\/X5r2za0KghZiRBq8GLoSkpfDVhOo6w6HiPJw2yOCYK9A3RwLPMJei03F1Lo3X50bw9cjFk\\\\\\\/WQgbXgp9wWBuDkbVh23qcs3kKIYJw81hV\\\\\\\/EoZd73FAJVMAHYsIop+8IJE\\\\\\\/\\\\\\\/lkV4ILXbUMPdlwO+wtDtMrPLdgKdkY\\\\\\\/PU6aBXCrOyPWft9\\\\\\\/r4vjCorilB5Dgu9Y0rqDrI4uOngf3KBQ0eyY8LUGm3wVVIoAI4M2YthAMiBRbrtnq5dt4lXKCCPSk+JMYtCrElCBIGRavP5qt1U8CpCC4EVvRWypuYBk\\\\\\\/hqoWzve7ns6Ngp9kJcmSCqd1nGr\\\\\\\/utxyN4yGxj8gcCuBPX7JVwey4GZdKjatA7j+T+KmyhG61x8SW\\\\\\\/VNPgh5JQWvVwxMvKZxlZ\\\\\\\/Mu7NlREgleKJa5AQdHDi9GeWTAYCRoFn\\\\\\\/eDSjsCMbRp1HrMftR+1QsTfuVDntg\\\\\\\/fIUHnAjID8Dqe80cwkYAeMmigkoDawz3yd\\\\\\\/uPtQDEhIh7i3iV\\\\\\\/9OkV4rD\\\\\\\/+8RZ\\\\\\\/vvH7SvjJ3UTiK4uOY0tICpFSquTD15jjokVd\\\\\\\/SRpfgnRTd71og0HRfjqXMvFlp0I2fPmTJpJM6d49Sp5JRsLLh7ygOyu4y+o7aNJkJ15Kqn3ypsWBFOnmm9n+bc2ym1z9e4NWKrt9LK5bFmI+1nRhkljDX3adorevCrx4ZO+xxyjK3UVZK46rvMqq9kvt5l8SqENbfN2\\\\\\\/D9gX72AKTNpSf1PxNTF\\\\\\\/6\\\\\\\/cSpD\\\\\\\/qN4kqofBWN5A1kyTsIVvyTMe\\\\\\\/9snDrp\\\\\\\/IXfZsEVHadRlsO6N6+7SzAqPJzENDJc76Ih399AQ5ZGuKUJaTk9VE8vMD1cLj5yLbld2wlwQiYflq+pPIIla3sstpJ9j9q6Xmwdl97EmiQV7Jil1OgSimz4kpFvtVTxbDQo9q9IQW\\\\\\\/+sFisfgODZDezf20PKEnf8bQ1KbqcPOSv+NIqNsoLvA05thEe+DotfA2BAMrxwGnTotV1zdH6unWkM\\\\\\\/WoMk0wkxxv5uy7lHbrYIAarsrCEDu5hgNKwJJ7+UtqWGiUhZuFZrB0e\\\\\\\/HPZNYuKQr8L8fIvrqDuKfSQH2DQS2cADtMxIy0aqv+8B\\\\\\\/ubVgteZy6FwZtzHsjWMN6RQAkC4ZB55O2\\\\\\\/UbiEe4\\\\\\\/lwfrNdXllKEWRiQhh2ZwnXxZy0B0JTI\\\\\\\/5qB5+DfvLZBUinRSlsKOHzUI6zNunYtzp8Opx5JsQK4y9RxetpB1v8vSHgKg5q4lhFO4FLlO7K5fLZhHT2NwGU1noN9aVJAMDYuqOjIpFBiXZ8H2LyTSqQILgYlj6FUwdiqyclOv9SPl4wyZcTeqZLpUklLa2AYCjGv+QsIFAnwS1Z4nL\\\\\\\/pfy52iLN7Mau1FWJ8P\\\\\\\/+M7WWsNMAfRutE2PF61vQi2auw\\\\\\\/b9Z4HxbQ8bRJ64FQIgphg15sUbZM\\\\\\\/Um2oRss6wghVSp5WGoasrD\\\\\\\/QeCdpQ6ZJL6gqCRdANseAh32JOQ2Tk3CS0WYtx6lpQMgeZmlaM5V7IGaBiaK+hDAC4e+EVNPQqeX\\\\\\\/EMyLaxZYXbstR80+JnseMSqrG6vHCsY2FLhJz+IPVk7iitwwbbpv4JhEAT19aRccKGaLAdCuKB+ybBC17nPNa6\\\\\\\/FFtEH67fUTNC7LTeznYuoAnFrqpX4miegyAiEpzj7+KgxnHuEKxCK09SYbxOdwHDrx5nxscfl\\\\\\\/h79YTwkoQgJZ9GUlbG0niq6Ntr4+xNxgYwA9I3R6k+3M+RXSOhp96998cQZ7y7K2GcQCJFs+zCbvKsi+WiUPFS7e0Vva1iKpFnL9so5\\\\\\\/TuQg87iyKnORFQ4lYqRonODjyQyLTGQFEBY1ObSjTP+RXKqe\\\\\\\/JSJxfsAqz3IZI7TQ3xrfPA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WXJ1vYW0oL72+N8J6o98iA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222b0aa1fc755a2e31a6d8b7ac4d8446439900890915ad5ce459208cf6a3e64e7c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J9ZtuwqhoXpgHBp8b8wDBdpOSkcrL1l4rks4V\\\\\\\/XpUO0XwH9X3Hbkfaa6DxNBb1v4NZJyz\\\\\\\/HdKP6qAr5QirhzymPEk8IP5C8rmMGvrwlxK95rY\\\\\\\/6wrPmMAIPLUX5XJHNRhiilR1EF7AZaaV6DipexLJZhV\\\\\\\/KxLGkt4RsI4zY8uyM9Dfh51c23CVwnvsxBu12mnGQi6tmH6jV8WN58ZZDcGGVqG8qc4R6aQYBHDirgDoVJF5BUb87kdTSuL3u\\\\\\\/B0AYi2xMgsIIht1M9itGk8ZaPQftu88CV0Ls339bqr+IFXLuhTyLOzjElTDcyUB1L+0IUr+At\\\\\\\/hL1uW\\\\\\\/WyMYpDhdgWqxLm1T1vdBfX4T0MSLJrnzifIIW+uJk1yqUcwEDhwa58e5GYIlhMxW6sDfq5JivFS2I6DkTiXnylezFj7W4oEN2TP7+xXLZxEsIRSLMoUnuNpMh+ypRAfsfOfRxSbq430Vdf42lXaVJ8fFGpqA2BGuFOUhPPkjzlgr3cUOIE6u+IZqzyRbOds1B1c5R2fiMg+1r25a7uKz8oRdhrLt5uqBeDX+dXo8RoxCNCnp\\\\\\\/eLHyX9XmnyfWaAHVHzhMrTTAmEuUqz15b09CNlc4zNskX9fP85Ye00EVPs4lvT6HmRno7kvUUuVKHdLFs8qK1M315JuLySaFnrNSahpYpjuRXhPIzAxkOwMg56m0wJNnIB5Qe5UATzpP9DwoYWx3Di8nClZqWu8eRMlou2yNyA9vrInU0wiUNaPQtKo7T2\\\\\\\/3aON1EXkp\\\\\\\/Zgq9zboBQtr9KkpCpcrZYP+4orFiAao8\\\\\\\/2eZd71V1Cb0zEdXcLmB8qNvKL7oMNWAeytm5cnrWuiCl5c1J2CNq46pZVpzYc9rhsicVj1yHCeef9952+umeB2hVZWg3F9tmCQhvajGF292dLbh7yKcjurCL9vyEr1mHXC3BlOKsVoqE+46DS6q8A1ZoxwKFgnIhgdMtnhExUNhVDvIc6hy+HIVP+k7AYv509RqGmKVj+1b\\\\\\\/9iU\\\\\\\/\\\\\\\/24DeoHfD8t0WilK43Pn+PT+mSNZ284N8IAX\\\\\\\/wNIzRPhDOmcicZkfCRaTmaII619ACGfWj1PUu2xfjCvyMDFm5lP5fvalKIs9q15k2TZ5V308Gtl6BR9NNitTAUYzW\\\\\\\/avAupCoxvQPAXmrktqmiOH\\\\\\\/Q77i0IIxJoTpxWmoCLf2N9N4JW3EI55U4Wa6T0+JuJxn\\\\\\\/ZSsTl91RZP15ctlWwCCa9id947gL7DYYzB4K+b5Cge\\\\\\\/D4Q7\\\\\\\/VFWiTQWZiuRQ1J21A0zX4QontcwRxIglAyvKcvDBCETfSPoUIyyYyT9BhxiX3fuZovRTxFmR6R8ul34Cl4YZj3R0I8ApOJMvLT3DXQOypIzQ0ZoeYgW7DPtVXRb327EKxw4190MMXhT5ZQMFogtjDQGEvnxi3rXgFcYz2lBQSqcl7IDuepXat5WpmYkVjPgQuF9M+\\\\\\\/VMvKe+KCM4VcJ8ZRpxNmZ+1vE+cqM7PZkruX1J36k5WPtcrKqZJn3KDgH8eOZJF4Orqpw\\\\\\\/6ILq616KKd\\\\\\\/LuHL0xhTWUk62XmbyQm8ssV1tV+80imtMC8EPc6AsfxPqbfqW\\\\\\\/V5YUS+Ze9H3Nis\\\\\\\/W3IsVYx+IPpBdJQKu6N+1EP2LmlvQZFsvF8H406Ha8ViskP8OWzkm9I+jFx02ld0I1Q69JQ4oY9LEjxTYOQHo1ZYJsdYxYm9LfpRMerVdBXKfZsA9djB4WtUrNp7kT4u58jlYT4MaDM02xFtURIqXSWNUT+LmvxW+TMNbfAzvXas5\\\\\\\/hFVUfgBXUUQoksHGKpsdrDU2ORrOdyUigjoukbM0pVZdTxmbY81fUlc2BD34XmqB6DH2FECVSbWKvIAGGLDr0CYnfCXD8A4twguNvETIBYL7MEmLxiaAtsJZoZEAzrSug9jitGvRw5ZaIjQeRbpuv1tbNnC3mIB5K3\\\\\\\/k9AJzGxoXETk2GF58MGaRMDuZcOH66pvsHaYufgsfhOAsG7ywUK95GnH5+5Ta4waBbQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cBp0bom8+DcfQR2PXW1kRg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c72ddf07f8c08175bf9b93e7b6f77ce51e071c4c0bd7681e0979cf70dd2b4cc2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459981 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1323" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-a8DGiDaE7U0SXB5nTkE42v5UZ3hs9qP_IWSZ96xC8yA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1323" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IZyijLz0WgoZOV4OtjebmdOY3D43cMQb4xjG2FKA--w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1317" title="Multiple Conserved Heteroplasmic Sites in tRNA Genes in the Mitochondrial Genomes of Terrestrial Isopods (Oniscidea)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1335" title="Transcriptional Response to Acute Thermal Exposure in Juvenile Chinook Salmon Determined by RNAseq" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459981" id="node459981" data-pisa="ggg;5/7/1323" data-pisa-master="ggg;g3.115.017731" data-apath="/ggg/5/7/1323.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus <em>Zymoseptoria</em> Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jonathan Grandaubert</span>, <span class="highwire-citation-author" data-delta="1">Amitava Bhattacharyya</span> and <span class="highwire-citation-author" data-delta="2">Eva H. Stukenbrock</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1323-1333;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017731 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459981" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan Grandaubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BGrandaubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grandaubert%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BGrandaubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amitava Bhattacharyya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amitava%2BBhattacharyya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhattacharyya%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmitava%2BBhattacharyya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eva H. Stukenbrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eva%2BH.%2BStukenbrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stukenbrock%20EH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEva%2BH.%2BStukenbrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:estukenbrock@bot.uni-kiel.de" class="" data-icon-position="" data-hide-link-title="0">estukenbrock@bot.uni-kiel.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/7/1323" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459981" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459981/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1323.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459981" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459981/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1323.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459981" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459981/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1323.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459981" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459981/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/7/1323.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459981" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459981/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The fungal pathogen <em>Zymoseptoria tritici</em> (synonym <em>Mycosphaerella graminicola</em>) is a prominent pathogen of wheat. The reference genome of the isolate IPO323 is one of the best-assembled eukaryotic genomes and encodes more than 10,000 predicted genes. However, a large proportion of the previously annotated gene models are incomplete, with either no start or no stop codons. The availability of RNA-seq data allows better predictions of gene structure. We here used two different RNA-seq datasets, <em>de novo</em> transcriptome assemblies, homology-based comparisons, and trained <em>ab initio</em> gene callers to generate a new gene annotation of <em>Z. tritici</em> IPO323. The annotation pipeline was also applied to re-sequenced genomes of three closely related species of <em>Z. tritici</em>: <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>. Comparative analyses of the predicted gene models using the four <em>Zymoseptoria</em> species revealed sets of species-specific orphan genes enriched with putative pathogenicity-related genes encoding small secreted proteins that may play essential roles in virulence and host specificity. <em>De novo</em> repeat identification allowed us to show that few families of transposable elements are shared between <em>Zymoseptoria</em> species while we observe many species-specific invasions and expansions. The annotation data presented here provide a high-quality resource for future studies of <em>Z. tritici</em> and its sister species and provide detailed insight into gene and genome evolution of fungal plant pathogens.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-annotation" class="hw-term hw-article-keyword hw-article-keyword-gene-annotation" rel="nofollow">gene annotation</a></li><li class="kwd"><a href="/keyword/transposable-elements" class="hw-term hw-article-keyword hw-article-keyword-transposable-elements" rel="nofollow">transposable elements</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/species-specific-genes" class="hw-term hw-article-keyword hw-article-keyword-species-specific-genes" rel="nofollow">species-specific genes</a></li></ul><p id="p-4">The genus <em>Zymoseptoria</em> (class <em>Dothideomycetes</em>, order <em>Capnodiales</em>) comprises a group of plant pathogens that infect a variety of <em>Poaceae</em> species. The four pathogen species Z. <em>tritici</em>, <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> originated in the Middle East and have diverged recently (within the past ∼22,000 years) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stukenbrock <em>et al.</em> 2007</a>). Speciation in the genus <em>Zymoseptoria</em> has been associated with specialization and adaptation to different hosts in different environments (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Quaedvlieg <em>et al.</em> 2011</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Stukenbrock <em>et al.</em> 2011</a>). Most notably, <em>Z. tritici</em> co-evolved and spread with domesticated <em>Triticum</em> species and is today a prominent pathogen of wheat worldwide (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Banke and McDonald 2005</a>). The poorly described <em>Z. brevis</em> species (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Quaedvlieg <em>et al.</em> 2011</a>) was isolated in Iran like <em>Z. pseudotritici</em> and <em>Z. ardabiliae</em>, but from another grass host, <em>Phalaris minor</em>. Phylogenetic analyses based on information from six DNA loci place <em>Z. brevis</em> next to <em>Z. pseudotritici</em>, suggesting that this species also shares recent common ancestry with the wheat pathogen (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Stukenbrock <em>et al.</em> 2012</a>). The close relatedness of the four <em>Zymoseptoria</em> species makes this a unique model system to study host specialization and speciation of fungal plant pathogens.</p><p id="p-5">The genome of <em>Z. tritici</em> (isolate IPO323) was sequenced and annotated by the Joint Genome Institute (JGI) and provides one of the best-assembled eukaryotic genomes with 21 complete chromosomes sequenced from telomere to telomere (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2011</a>). Of the 21 chromosomes, eight are considered accessory chromosomes because they can be lost during meiosis without any apparent effect on fitness in sexual progenies (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wittenberg <em>et al.</em> 2009</a>). The functional relevance of these accessory chromosomes is so far not known. They contain a higher proportion of repetitive DNA, encode fewer genes (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2011</a>), and show a dramatically lower level of transcription compared to the core chromosomes (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kellner <em>et al.</em> 2014</a>). The previously generated annotation predicted more than 10,000 genes in the genome of IPO323. The pipeline applied for the annotation used several gene predictors including putative full-length genes from EST cluster consensus sequences (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2011</a>). Nevertheless, this gene annotation comprises many incomplete gene models (14%) without start and/or stop codons.</p><p id="p-6">Different aspects of genome evolution in <em>Z. tritici</em> have been studied using comparative analyses of different <em>Zymoseptoria</em> genomes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Stukenbrock <em>et al.</em> 2010</a>, <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">2011</a>). A comparative genome study of the reference isolate IPO323 and an isolate of <em>Z. pseudotritici</em> allowed a detailed analysis of genome divergence in coding and noncoding sequences and a comparison of chromosome content and synteny in the two species (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Stukenbrock <em>et al.</em> 2010</a>). Multiple genomes were analyzed in a comparative population genomic analysis including <em>Z. tritici</em>, <em>Z. pseudotritici</em>, and <em>Z. ardabiliae</em> to further assess signatures of positive selection and the speciation history of the three species (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Stukenbrock <em>et al.</em> 2011</a>). Both studies relied on the previously generated annotation of the <em>Z. tritici</em> reference isolate IPO323, and thereby only described evolution of gene content shared between species.</p><p id="p-7">Repeats have also been identified in the reference genome o<em>f Z. triti</em>ci (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dhillon <em>et al.</em> 2010</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ohm <em>et al.</em> 2012</a>), but they were only exhaustively classified into transposable element (TE) families recently (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dhillon <em>et al.</em> 2014</a>). In fungal genomes, as in many eukaryotic genomes, TEs can constitute a large fraction of the genome, <em>e.g.</em>, 40% of the <em>Pseudocercospora fijiensis</em> genome (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Ohm <em>et al.</em> 2012</a>) and 64% of the <em>Blumeria graminis</em> genome (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Spanu <em>et al.</em> 2010</a>). TEs play an important role in the evolution of genomes, and in fungi they are shown to be responsible for large genomic rearrangements (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Croll <em>et al.</em> 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de Jonge <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grandaubert <em>et al.</em> 2014</a>) and for the creation of highly dynamic regions that provide accelerated diversification and chromatin-based regulation for genes within or near these regions (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rouxel <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Soyer <em>et al.</em> 2014</a>). An in-depth characterization of the complete TE repertoire of the four known <em>Zymoseptoria</em> species would allow detailed insights into the impact of repetitive DNA on the evolution of core as well as accessory chromosomes in these pathogens.</p><p id="p-8">We set out to conduct a thorough annotation of protein-encoding genes and repetitive sequences in all four <em>Zymoseptoria</em> species. Our goal was to improve the existing annotation of <em>Z. tritici</em> and to generate a complete set of gene models and repeat families for the four <em>Zymoseptoria</em> species for comparative genome analyses. All predicted gene models were <em>in silico</em> functionally annotated. We particularly focused on the presence and distribution of small proteins with signal peptides because these could be effector proteins produced to overcome host immune defenses (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">de Jonge <em>et al.</em> 2011</a>). To characterize a set of core <em>Zymoseptoria</em> genes and to identify species-specific orphan genes, we conducted a detailed analyses of gene homologies. Furthermore, we addressed the distribution of TEs in the <em>Zymoseptoria</em> genomes with a particular focus on TE content of core and accessory chromosomes of <em>Z. tritici</em> and the distribution of TE families between the four <em>Zymoseptoria</em> species. The dataset presented here provides not only a new high-quality annotation of <em>Z. tritici</em> but also a detailed characterization of gene families among Z. <em>tritici</em>, <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genome data</h3><p id="p-9">For the gene annotation, the genomes of <em>Z. tritici</em> (isolate IPO323) (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2011</a>), <em>Z. pseudotritici</em> (isolate ST04IR_2.2.1), and <em>Z. ardabiliae</em> (isolate ST04IR_1.1.1) (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Stukenbrock <em>et al.</em> 2011</a>) were used. For the annotation of <em>Z. brevis</em>, the genome of the isolate Zb18110 was sequenced. Genomic DNA was isolated using a standard phenol-chloroform protocol (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sambrook and Russell 2001</a>), and sequencing of 100-bp paired-end reads was performed using a HiSeq2000 Illumina platform (AROS Applied Biotechnology, Denmark). A <em>de novo</em> assembly of the <em>Z. brevis</em> Illumina reads was generated using the CLC Genomics Workbench version 5 (CLC, Aarhus, Denmark) with standard settings for paired-end read assembly. This assembly is available under the NCBI BioProject (PRJNA273516) on Genbank. The assembled genomes of <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em> (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Stukenbrock <em>et al.</em> 2011</a>), and <em>Z. brevis</em> used for the gene annotation contained a very low number of repetitive DNA. Therefore, new assemblies better representing the repeat content of these species genomes were constructed using Illumina sequences obtained from the isolates <em>Z. pseudotritici</em> ST04IR_5.5, <em>Z. ardabiliae</em> ST11IR_6.1.1, and <em>Z. brevis</em> Zb163. Assemblies of these new Illumina genomes were generated using SOAPdenovo2 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Luo <em>et al.</em> 2012</a>) with optimized k-mer values allowing the inclusion of repeats in the assemblies. These three assemblies are available under the NCBI BioProject (PRJNA274679) on Genbank.</p></div><div id="sec-3" class="subsection"><h3>RNA-seq data</h3><p id="p-10">Two previously published RNA-seq datasets were used for <em>de novo</em> transcript assembly of <em>Z. tritici</em> IPO323 including one dataset obtained from RNA extracted from infected <em>Triticum aestivum</em> (cultivar Obelisk) seedlings and one dataset obtained from axenically grown fungal cells (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kellner <em>et al.</em> 2014</a>). For <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>, RNA-seq data were obtained from axenically grown cultures. Total RNA was extracted from fungal cells grown in YMS (4 g yeast extract, 4 g malt extract, 4 g sucrose, 20 g bacto agar, 1 liter H<sub>2</sub>O) agar in a shaking incubator at 200 rpm at 18° using the TRIZOL reagent (Invitrogen, Darmstadt, Germany) following the protocol of the manufacturer. Illumina RNA-seq libraries for two axenic culture replicates per species were prepared from an input of 10 µg total purified polyA RNA (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Palma-Guerrero <em>et al.</em> 2013</a>). Libraries were quantified by fluorometry, immobilized, and processed onto a flow cell with a cBot (Illumina), followed by sequencing-by-synthesis with TruSeq v3 chemistry on a HiSeq2000 at the Max Planck Genome Center (Cologne, Germany). RNA-seq data for <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> are respectively available in the NCBI BioProjects (PRJNA277173, PRJNA277174, PRJNA277175). For data processing, the sequence read quality was first evaluated using FASTQC (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Subsequently, reads were filtered using tools from the Galaxy server, including grooming, trimming, filtering, and masking steps (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Giardine <em>et al.</em> 2005</a>). Reads with an overall quality score less than 20 were removed. For the remaining reads, all nucleotides with a quality score less than 20 were masked with Ns. For <em>Z. tritici</em>, reads from the host (<em>T. aestivum</em>) transcriptome were initially filtered out using fastq_screen v0.4.1 (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen">www.bioinformatics.babraham.ac.uk/projects/fastq_screen</a>) as described by <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Kellner <em>et al.</em> (2014)</a>.</p></div><div id="sec-4" class="subsection"><h3>Gene annotation</h3><p id="p-11">Protein-coding genes were identified using the Fungal Genome Annotation pipeline described by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Haas <em>et al.</em> (2011)</a>. The individual steps of the pipeline are described below.</p><div id="sec-5" class="subsection"><h4>Transcript reconstruction:</h4><p id="p-12">Transcript reconstruction using Trinity (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grabherr <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Haas <em>et al.</em> 2013</a>) was carried using <em>de novo</em> and genome-guided methods. By the <em>de novo</em> method, RNA-seq reads were first assembled into unique sequences of transcripts (contigs) using the Inchworm module within Trinity. Contigs were clustered by the Chrysalis module, and corresponding De Bruijn graphs that represent the possible different isoforms were constructed. In the final step, De Bruijn graphs were processed by the Butterfly module to produce full-length transcripts. In the genome-guided method, RNA-seq reads were first aligned to the genome using GMAP (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wu and Watanabe 2005</a>). Based on these aligned-read clusters, the Chrysalis and Butterfly modules were consecutively executed to produce the final reconstructed transcripts. Transcripts generated by these two methods were combined using the “Program to Assemble Spliced Alignments” (PASA) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Haas <em>et al.</em> 2003</a>) pipeline to build a complete set of unique transcripts corresponding to gene models.</p></div><div id="sec-6" class="subsection"><h4>Training:</h4><p id="p-13">First, GeneMark-ES (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ter-Hovhannisyan <em>et al.</em> 2008</a>) was used for <em>ab initio</em> predictions, because its self-training algorithm allowed the identification of high-quality gene models. Next, evidence from homology searches using tBLASTn (e-value cut-off of 1e-10) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>) against a nonredundant protein database (UniRef90) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Suzek <em>et al.</em> 2007</a>) and from reconstructed transcripts were used to filter the <em>ab initio</em> predicted gene models. Only the complete gene models predicted by GeneMark-ES with a support from the homology-based comparison (100% of coverage for each exon) and with the exact same exon–intron boundaries as in the reconstructed transcripts were selected for the training and testing of <em>ab initio</em> gene predictors. The 2693 selected gene models were divided into two sets: one for the training (training set) containing 1611 sequences (60%) and one for assessing prediction accuracy (test set) containing 1082 sequences (40%). The evaluation of the training process was performed using Augustus (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stanke and Waack 2003</a>). For all species, better performance was obtained using datasets of <em>Z. tritici</em>. We used the <em>Z. tritici</em> training set to train the <em>ab initio</em> gene callers for all the species.</p></div><div id="sec-7" class="subsection"><h4>Gene prediction and annotation:</h4><p id="p-14">The trained GeneMark.hmm (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lukashin and Borodovsky 1998</a>) and Augustus programs were used to predict gene models. For <em>Z. tritici</em>, Fgenesh (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Salamov and Solovyev 2000</a>) gene models obtained from the first annotation by JGI (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2011</a>) were also included. Gene models obtained from the trained gene predictors were evaluated and combined using the EVidenceModeler (EVM) software (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Haas <em>et al.</em> 2008</a>) to create a weighted consensus of the gene structures. Weights of 3, 5, and 7 were used for <em>ab initio</em> predictions, homology-based predictions, and transcript evidence, respectively. Homology-based gene models obtained from GeneWise (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Birney <em>et al.</em> 2004</a>) were only used for <em>Z. tritici</em> as reliable gene models; they could not be obtained in the much more fragmented genomes of <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>. Based on transcript alignments, the PASA pipeline was used to correct the consensus gene model structure and to resolve conflicts between different isoforms.</p></div></div><div id="sec-8" class="subsection"><h3>Gene comparison</h3><p id="p-15">Based on the four predicted proteomes of the <em>Zymoseptoria</em> species, sequence comparisons were performed with BLASTp (e-value cut-off of 1e-5). Obtained pairwise protein alignments were processed using the software SiLiX to build families of homologous proteins (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Miele <em>et al.</em> 2011</a>). Proteins were clustered together into families if they shared at least 55% of sequence identity over at least 60% of sequence coverage. The building of these families was divided in two steps. First, only complete protein sequences were used to create families. Second, using a semi-bipartite graph, partial protein sequences were added to the existing families.</p></div><div id="sec-9" class="subsection"><h3>Secretome prediction</h3><p id="p-16">Plant pathogens secrete proteins to interfere with host immune defenses. Genes encoding secreted proteins are therefore of particular interest due to their potential role in infection and host–pathogen interaction. We screened predicted proteins for the presence of signal peptides and categorized proteins as secreted under the following conditions: (i) if a signal peptide was predicted by both Neural-Network and HMM methods by the software SignalP 3.0 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Emanuelsson <em>et al.</em> 2007</a>); (ii) if zero or one transmembrane domain was present in the protein (the domain having at least 30% overlap with the signal peptide) using the software TMHMM 2.0 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Krogh <em>et al.</em> 2001</a>); and (iii) if the protein was predicted to be targeted to the secretory pathway by the software TargetP 1.1 (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Emanuelsson <em>et al.</em> 2007</a>). Secreted proteins with a size of 300 amino acids or less were considered as small secreted proteins (SSPs).</p></div><div id="sec-10" class="subsection"><h3>Functional annotation</h3><p id="p-17">Automated functional annotations of the predicted proteins of <em>Z. tritici</em>, <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> were performed as previously described by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Grandaubert <em>et al.</em> (2014)</a>. Using a combination of BLAST and InterProScan (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Quevillon <em>et al.</em> 2005</a>), predicted proteins of each of the four species were categorized into three classes. The first class included proteins with no significant BLAST hit and with no known protein domain identified by InterProScan. These proteins were classified as “predicted proteins,” <em>i.e.</em>, predictions with no functional support. This class contained an excess of species-specific genes. The second class, termed “hypothetical proteins,” included proteins with at least one domain identified by InterProScan in the InterPro or Pfam databases (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hunter <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Coggill <em>et al.</em> 2008</a>) or that fulfilled the BLAST result criteria defined by <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Grandaubert <em>et al.</em> (2014)</a> and for which the description indicated “hypothetical protein” in more than 90% of the BLAST hits. Globally, this class contained predictions with poor functional evidence likely corresponding to conserved proteins among several organisms but with no defined functions. The third class included predictions that fulfilled the BLAST result conditions with at least one domain from the InterPro or Pfam database corroborating by the consensus BLAST description. This class was termed the “similar to <em>function</em>” and included well-conserved proteins with defined functions in many organisms.</p></div><div id="sec-11" class="subsection"><h3>Repeats and transposable elements identification</h3><p id="p-18">Repetitive DNA was identified and annotated in SOAP assembled genomes of the four isolates described above to create a repertoire of repeats specific to the <em>Zymoseptoria</em> genus using the REPET pipeline (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Flutre <em>et al.</em> 2011</a>). To obtain more complete elements, repeat families in each species were clustered using Blastclust from the NCBI-BLAST package (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>), aligned using Mafft (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Katoh and Standley 2013</a>), and new consensuses were then created. These steps were iterated with decreasing values of identity percentage (from 100% to 75%) and coverage (from 100% to 30%) until there was no more clustering of the sequences. Next, the sequences were classified (TEclassifier.py script from REPET) using tBLASTx and BLASTx against the Repbase Update database (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jurka <em>et al.</em> 2005</a>) and by the identification of structural features such as long terminal repeats (LTRs) or terminal inverted repeats (TIRs). The sequences were additionally translated into the six reading frames to perform a protein domain search on the conserved domain database (CDD) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Marchler-Bauer <em>et al.</em> 2011</a>) using RPS-BLAST. Transposable element (TE) families of each strain were classified and named according to the nomenclature defined by <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wicker <em>et al.</em> (2007)</a>. The 497 repeat families identified in this study are available in FASTA format in <a href="http://www.g3journal.org/content/suppl/2015/04/27/g3.115.017731.DC1/017731SI.pdf">Supporting Information</a>, File 1.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Improved gene annotation of <em>Z. tritici</em></h3><p id="p-19">RNA-seq data and detailed homology searches allowed us to generate an improved annotation of gene models in the IPO323 <em>Z. tritici</em> reference genome. Read data obtained from <em>in planta</em> and <em>in vitro</em> experiments were filtered (<a href="http://www.g3journal.org/content/suppl/2015/04/27/g3.115.017731.DC1/TableS1.pdf">Table S1</a>) and used for transcriptome assembly. Assemblies were based on two different but complementary approaches: (i) <em>de novo</em>, where the reads were assembled without a reference sequence and (ii) genome-guided, where the IPO323 reference genome was used as a template for transcript assemblies. In total, 68,653 transcripts were generated by the <em>de novo</em> method and 73,127 transcripts by the genome-guided method (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These transcripts were used to create a final database containing 13,847 putative gene models next used for the generation of a training set for <em>ab initio</em> predictors and for the correction of predicted gene models. We generated a final training set of 1611 gene models that was used to train tw<em>o ab initio</em> gene predictors: Augustus (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Stanke and Waack 2003</a>) and GeneMark-hmm (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lukashin and Borodovsky 1998</a>). The outputs of these software were combined with 10,893 predicted gene models of another <em>ab initio</em> predictor, Fgenesh (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Salamov and Solovyev 2000</a>), obtained from the JGI annotation. We thereby obtained a total of 36,423 different gene models combining models from Augustus, GeneMark-hmm, and Fgenesh. These gene models were processed by EVidence Modeler (EVM) (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Haas <em>et al.</em> 2008</a>) to identify a consensus model for each gene. Finally, corrections were applied to the gene models using PASA (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Haas <em>et al.</em> 2003</a>) and the previously reconstructed RNA-seq-based transcripts.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466513/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466513/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Annotation features for the four members of the <em>Zymoseptoria</em> species complex</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">The final gene annotation of IPO323 consists of 11,839 gene models (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Distributions of the number of gene models along the chromosomes of the annotation presented here and the previous JGI annotation are shown in <a href="http://www.g3journal.org/content/suppl/2015/04/27/g3.115.017731.DC1/FigureS1.pdf">Figure S1</a>. In our new annotation, only 44 gene models were incomplete (<em>i.e.</em>, without start and/or stop codon) compared to 1555 incomplete genes in the previous annotation (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). To identify shared and unique gene models, we used a BLAST search (e-value cut-off of 1e-10). We found 4707 identical gene models between the two annotations. Furthermore, we found 442 gene models uniquely predicted by the JGI pipeline and 1200 models uniquely predicted by the pipeline used here. Gene models of our annotation have an average length of 1621 bp and exhibit longer exons (mean exon length 575 <em>vs.</em> 505 bp) and encode longer proteins (488 <em>vs.</em> 437 amino acids) when compared to the annotation generated by the JGI pipeline (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). A BLAST comparison at the nucleotide level of our final gene models against the reconstructed transcripts showed that 10,048 out of the 11,839 predicted genes have support based on the RNA sequencing. In the previous annotation, 9423 predicted genes had support compared to the transcripts reconstructed in this study, which underlines the efficiency of this new annotation to predict biologically relevant gene models.</p><p id="p-23"><em>In silico</em> functional annotation based on protein signatures and homologies allowed us to assign a function to 43% of the 11,839 predicted proteins. A total of 2714 sequences (22.9%) had no homologies within the NCBI nonredundant protein database and did not include any known protein domain. These represent either species-specific genes of <em>Z. tritici</em> or incorrectly predicted genes. Given the fact that 55% of these have RNA-seq support, we considered that the majority of these novel genes must be correctly predicted. Characterization of the <em>Z. tritici</em> secretome yielded comparable results to the previously reported <em>Z. tritici</em> secretome (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Morais Do Amaral <em>et al.</em> 2012</a>), representing 874 secreted proteins including 441 small, secreted proteins (SSPs), <em>i.e.</em>, with a size inferior to 300 amino acids (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). SSPs of <em>Z. tritici</em> have, on average, 2.8-times more cysteine residues compared to the whole proteome (data not shown supporting an extracellular role) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fass 2012</a>).</p></div><div id="sec-14" class="subsection"><h3>Gene annotation using re-sequenced genomes and RNA-seq data from <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em></h3><p id="p-24">RNA-sequencing data obtained from axenic cultures of <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> were filtered (<a href="http://www.g3journal.org/content/suppl/2015/04/27/g3.115.017731.DC1/TableS1.pdf">Table S1</a>) and assembled. The resulting transcriptomes include 35,132, 36,546, and 33,201 transcripts clustered into 12,027, 12,719, and 11,542 putative gene models for <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>, respectively (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). The training set established for <em>Z. tritici</em> was also for the annotation of the three other <em>Zymoseptoria</em> species. For each species, EVM created a weighted consensus of the gene models, which after PASA corrections resulted in 11,044, 10,787, and 10,557 gene models for <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>, respectively (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). The total number of predicted gene models in the re-sequenced <em>Zymoseptoria</em> species was similar; however, due to the fragmented genome assemblies of <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>, more partial gene models were predicted compared to models in <em>Z. tritici</em> (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). Overall, the gene characteristics of these four species are very similar in accordance with the close phylogenetic relationships. Compared to <em>Z. tritici</em>, the functional annotation of the predicted gene models in <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> showed a larger proportion of sequences with an attributed function and a smaller proportion of sequences with no functional evidences. The four annotations are available from <a href="http://fungi.ensembl.org/Zymoseptoria_tritici/Info/Index">http://fungi.ensembl.org/Zymoseptoria_tritici/Info/Index</a> (<em>Z. tritici</em>), <a href="http://genome.jgi.doe.gov/Zymps1/Zymps1.home.html">http://genome.jgi.doe.gov/Zymps1/Zymps1.home.html</a> (<em>Z. pseudotritici</em>), <a href="http://genome.jgi.doe.gov/Zymar1/Zymar1.home.html">http://genome.jgi.doe.gov/Zymar1/Zymar1.home.html</a> (<em>Z.ardabiliae</em>), and <a href="http://www.uniprot.org/taxonomy/1047168">http://www.uniprot.org/taxonomy/1047168</a> (<em>Z. brevis</em>).</p></div><div id="sec-15" class="subsection"><h3>Gene distribution in <em>Zymoseptoria</em> species</h3><p id="p-25">Homology relationships among the four <em>Zymoseptoria</em> species including orthology (interspecies) and paralogy (intraspecies) were analyzed by a comparison of the predicted proteomes using the software package SiLiX (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Miele <em>et al.</em> 2011</a>). Of the total 44,227 proteins predicted in the four <em>Zymoseptoria</em> species, 39,177 (88.6%) were clustered into 10,612 families. These families belong to three categories. First, there are 10,361 families of orthologous sequences present in at least two species. The different distributions of these families across the <em>Zymoseptoria</em> species are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Second, there are 236 families of orthologous sequences that also included species-specific paralogous sequences, <em>i.e.</em>, independent duplication of a conserved gene occurred in one or more species. Third, there are 15 families of paralogous sequences, <em>i.e.</em>, duplication of species-specific gene<strong>s</strong>. The remaining 5050 sequences that could not be classified into a gene family were considered as proteins encoded by orphan genes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1323/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagram showing the distribution of predicted models in Z. tritici: Z. pseudotritici, Z. ardabiliae, and Z. brevis. The categorizations of core Zymoseptoria genes, orphan genes, and genes shared by three or two species were performed using a detailed characterization of gene orthology." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1361518766" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Venn diagram showing the distribution of predicted models in &lt;em&gt;Z. tritici&lt;/em&gt;: &lt;em&gt;Z. pseudotritici&lt;/em&gt;, &lt;em&gt;Z. ardabiliae&lt;/em&gt;, and &lt;em&gt;Z. brevis&lt;/em&gt;. The categorizations of core &lt;em&gt;Zymoseptoria&lt;/em&gt; genes, orphan genes, and genes shared by three or two species were performed using a detailed characterization of gene orthology.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1323/F1.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/7/1323/F1.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1323/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1323/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466477" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Venn diagram showing the distribution of predicted models in <em>Z. tritici</em>: <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>. The categorizations of core <em>Zymoseptoria</em> genes, orphan genes, and genes shared by three or two species were performed using a detailed characterization of gene orthology.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-16" class="subsection"><h4>Orphan genes:</h4><p id="p-27">The genome of <em>Z. tritici</em> encodes 1798 orphan genes (15.1% of all the predicted genes), of which 1221 are localized on core chromosomes and 577 on accessory chromosomes. However, accessory chromosomes are significantly enriched in orphan genes because they constitute 79.3% of the genes on these chromosomes compared to 10.9% on core chromosomes (χ<sup>2</sup> test, <em>P</em> &lt; 2.2e-1<strong>6</strong>). A large majority of the orphan genes (95.5%) have no <em>in silico</em> attributed function. In general, we could not link species-specific genes to any biological or pathogenicity-related function; however, this uniqueness supports the species-specific nature of the genes that have no homology with known proteins from organisms outside the <em>Zymoseptoria</em> species complex. Among the orphan sequences that could be assigned a function, we found predicted functions relating to molecule transport (drugs, proteins, amino acids), primary and secondary metabolism (oxydoreductase activity, nonribosomal peptide synthase), and organic compounds degradation (chitinase, glycoside hydrolase). Small secreted proteins (SSPs) of plant pathogens particularly play an important role as putative effectors during the host infection (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stergiopoulos and De Wit 2009</a>), and we found in <em>Z. tritici</em> that SSP-encoding genes were significantly enriched in the orphan gene set compared to the whole proteome: 8% <em>vs.</em> 3.7% (χ<sup>2</sup> test, <em>P</em> &lt; 2.2e-16). Orphan genes representing 9–11% of the predicted genes in <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> were likewise significantly enriched in SSP-encoding genes and genes that could not be assigned a function.</p></div><div id="sec-17" class="subsection"><h4>Paralogous gene<strong>s:</strong></h4><p id="p-28">Families of paralogous genes were only found in <em>Z. tritici</em> (eight families) and <em>Z. ardabiliae</em> (seven families) and included 23 and 14 genes, respectively. In <em>Z. ardabiliae</em>, 57% of these duplicated sequences are SSP-encoding genes, whereas there were no SSP paralogs in <em>Z. tritici</em>. In <em>Z. ardabiliae</em>, only one family could be associated with a gene function (kinase), whereas in <em>Z. tritici</em> several families contained sequences with known protein domains of transposable elements.</p></div><div id="sec-18" class="subsection"><h4>Orthologous genes:</h4><p id="p-29">Families of orthologous genes within the four <em>Zymoseptoria</em> species were divided into three categories. The first category comprised the core set of <em>Zymoseptoria</em> genes and included 7786 families containing one gene per species (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). A large proportion of these genes (59.5%) could be associated with a predicted function, whereas 8.3% of them had no homology to genes in other organisms and thereby potentially represented a set of <em>Zymoseptoria</em>-specific genes. The second category comprised genes present in three out of the four species and included 1294 families. Genes in this second category appeared to be less conserved because only 33% of them could be associated with function information. Finally, the third category included 1281 families comprising genes only found in two of the four species. Compared to the two other categories, genes with a function represented a smaller fraction (18%).</p></div><div id="sec-19" class="subsection"><h4>Orthologous genes with paralogy:</h4><p id="p-30">A particular category of families was also identified where orthologous sequences were conserved in two or more species with independent duplications in one or more species. This category contained 236 families with 1552 sequences. Most of the families (87%) comprised genes present in the four species. The largest family included 15 genes all encoding an alcohol dehydrogenase with four copies found in <em>Z. tritici</em>, <em>Z. pseudotritici</em>, and <em>Z. brevis</em> and three copies in <em>Z. ardabiliae</em>.</p></div></div><div id="sec-20" class="subsection"><h3>Transposable element annotation</h3><div id="sec-21" class="subsection"><h4>TEs in Z. tritici:</h4><p id="p-31">Of the 39.7-Mb genome assembly of <em>Z. tritici</em> IPO323, 7.4 Mb (18.6%) <strong>(</strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a><strong>)</strong> were found to be repetitive DNA, in agreement with the recently reported repeat content of IPO323 (&gt;17%) in <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Dhillon <em>et al.</em> (2014)</a>. Repeats were classified into TE families based on features such as terminal repeats, protein domains, and homology with known TE available in the Repbase Update database (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jurka <em>et al.</em> 2005</a>). The fact that the assembled genome of <em>Z. tritici</em> consists of complete chromosomes allowed a high-quality repeat annotation with only few uncategorized families. TEs of class I (retrotransposons) are the most abundant in the IPO323 genome and represent 13.1% of the total genome size (5.2 Mb), whereas TEs of class II (DNA transposons) represent 4.2% (1.6 Mb) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The remaining repetitive sequences represent 0.5 Mb and could not be associated with classified TEs. From the 111 consensus sequences obtained by our procedure (see <em>Materials and Methods</em>), we identified 101 families of TEs. The majority of these families belong to the class II (62 families), mainly represented by TIRs (44 families), MITEs (11 families), and Helitron (4 families). Although present in a lower number (32 families), class I elements represent 71% of the repetitive fraction of the genome. The most abundant group of class I elements is the LTR elements from the Gypsy superfamily (14 families), followed by those of the Copia superfamily (12 families) and LINEs (7 families) (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Interestingly, we also identified one family of a complete tyrosine-recombinase retrotransposon that belongs to the Ngaro superfamily and that otherwise has been described to be absent in <em>Ascomycota</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Muszewska <em>et al.</em> 2013</a>). Distinguishing core and accessory chromosomes of <em>Z. tritici</em>, we find that repetitive DNA represents 16.6% of the core chromosomes and 33.6% of the accessory chromosomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). However, the relative distribution of the major TE families on core and accessory chromosomes does not differ significantly (Wilcoxon test, <em>P</em> = 0.3692) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466384/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466384/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Transposable element content in genomes of the <em>Zymoseptoria</em> species complex</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1323/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transposable elements (TEs) characteristics along the 21 chromosomes of Zymoseptoria tritici. (A) Proportions of TEs in the chromosomes of Z. tritici. The red dashed lines show the mean values of TEs on core (1&#x2013;13) and accessory chromosomes (14&#x2013;21). (B) The distribution of five major TE classes across core and accessory chromosomes in Z. tritici show a more or less similar distribution and frequency on the two types of chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1361518766" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transposable elements (TEs) characteristics along the 21 chromosomes of &lt;em&gt;Zymoseptoria tritici&lt;/em&gt;. (A) Proportions of TEs in the chromosomes of &lt;em&gt;Z. tritici&lt;/em&gt;. The red dashed lines show the mean values of TEs on core (1&#x2013;13) and accessory chromosomes (14&#x2013;21). (B) The distribution of five major TE classes across core and accessory chromosomes in &lt;em&gt;Z. tritici&lt;/em&gt; show a more or less similar distribution and frequency on the two types of chromosomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1323/F2.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/7/1323/F2.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1323/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1323/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466503" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Transposable elements (TEs) characteristics along the 21 chromosomes of <em>Zymoseptoria tritici</em>. (A) Proportions of TEs in the chromosomes of <em>Z. tritici</em>. The red dashed lines show the mean values of TEs on core (1–13) and accessory chromosomes (14–21). (B) The distribution of five major TE classes across core and accessory chromosomes in <em>Z. tritici</em> show a more or less similar distribution and frequency on the two types of chromosomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">As already described by <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> (2011)</a>, repetitive sequences are highly affected by the fungal-specific repeat inactivation mechanism known as repeat-induced point mutations (RIP) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Galagan and Selker 2004</a>). Multiple alignments of several family copies analyzed by RIPCAL (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hane and Oliver 2008</a>) showed an excess of CpA→TpA and TpG→TpA mutations. As hypothesized by <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> (2011)</a>, these mutations should prevent TEs from expressing their own machinery. However, it appears that 98 predicted gene models overlap TE sequences with at least 50% of their length, including 75 genes models that are completely embedded in repetitive DNA. Of the 98 corresponding proteins, 33 harbor domains associated with TEs such as reverse-transcriptase, transposase, or integrase (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). These domains belong to 22 distinct TE families, including 12 TIR, eight LTR, one LINE, and one uncategorized element. Comparison of these gene models with the reconstructed transcripts showed that 48% have RNA-seq support; this suggests that TE families (mainly class II DNA transposons) are still active in the <em>Z. tritici</em> genome despite the RIP machinery.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466428/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466428/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Gene models with protein domains associated with transposable elements</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>TEs in Z. pseudotritici, Z. ardabiliae, and Z. brevis:</h4><p id="p-34">To improve the repeat annotation of <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>, we generated improved assemblies of Illumina-sequenced genomes of the isolates: ST04IR55 of <em>Z. pseudotritici</em>, ST11IR611 of <em>Z. ardabiliae</em>, and Zb163 of <em>Z. brevis</em>. The assembly sizes were 38.8 Mb, 37.3 Mb, and 39.8 Mb, respectively, compared to the 39.7 Mb genome assembly of <em>Z. tritici</em>.</p><p id="p-35">The three genomes reflect different extents of repeat invasion: repetitive DNA comprises 7.9% of the <em>Z. ardabiliae</em> genome, 15.5% of the <em>Z. pseudotritici</em> genome, and 23.9% of the <em>Z. brevis</em> genome, including a total of 117, 105, and 164 repeat families identified in the genomes, respectively (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). As a result of the fragmented assemblies, the number of uncategorized repeat families in the <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> genomes represent more than 50% of all the families and between 25% and 44% of the repetitive fraction. In contrast, the number of uncategorized families in the <em>Z. tritici</em> genome represents only 9% of the families and 7% of the repetitive fraction. Nevertheless, it was possible to classify some TE families of <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> and to record general patterns of TE distribution among the three genomes. As in the <em>Z. tritici</em> genome, class I TEs that are mainly represented by LTR-Gypsy elements are most abundant and account for 51–55% of the total repetitive fraction (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). LINE elements are hardly represented in the <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em> assemblies; however, this may be an artifact of the fragmented assemblies. In <em>Z. pseudotritici</em> and <em>Z. brevis</em>, class II TEs are mainly composed of TIRs that represent approximately 20% of the repetitive fraction similar to class II TEs in <em>Z. tritici</em>. Helitron elements were also identified in <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>; however, they accounted for only 1–6% of the repetitive fraction compared to 6.4% in <em>Z. tritici</em>. Maverick elements (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pritham <em>et al.</em> 2007</a><strong>)</strong> were only identified in <em>Z. pseudotritici</em> and were the third most represented TE family in the genome (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). The unique presence and distribution of Maverick elements in <em>Z. pseudotritici</em> could result from a recent invasion of the genome followed by massive waves of transposition of these elements.</p></div><div id="sec-23" class="subsection"><h4>The complete repertoire of repeats in Zymoseptoria spp:</h4><p id="p-36">In total, we found 497 repeat families in the four <em>Zymoseptoria</em> species, including 120 families of class I TEs (24.1%), 135 families of class II TEs (27.2%), and 242 uncategorized families (48.7%). We could improve the TE annotation using the complete <em>Zymoseptoria</em> TE library generated here. We found 19.9% of repeats (+1.3%) in the genome of <em>Z. tritici</em>, 17.6% (+2.2%) in the genome of <em>Z. pseudotritici</em>, 12.7% (+4.8%) in the genome of <em>Z. ardabiliae</em>, and 25.2% (+1.3%) in the genome of <em>Z. brevis</em>.</p><p id="p-37">Addressing TE evolution in the <em>Zymoseptoria</em> genomes, we found that some families identified independently in each species were derived from a common ancestral TE family. An analysis of the 497 families was performed using Blastclust (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>) with a coverage cut-off of 50% and an identity percentage cut-off of 80%; this resulted in the clustering of 76 families into 34 larger groups (<a href="http://www.g3journal.org/content/suppl/2015/04/27/g3.115.017731.DC1/TableS2.pdf">Table S2</a>). Fourteen of these groups were composed of TIR elements, eight groups of LTR-Gypsy elements, six groups of uncategorized elements, three groups of LTR-Copia elements, two groups of Helitrons, and one group of LINE elements. Only one of these groups included TE families present in each of the four species; this suggests that this element, an LTR-Gypsy element, was present in the genome of the common ancestor of the four species (<a href="http://www.g3journal.org/content/suppl/2015/04/27/g3.115.017731.DC1/TableS2.pdf">Table S2</a>). Interestingly, this element may have expanded prior to the divergence of species because we observed a comparable size expansion in the four species comprising between 120 and 180 kb in each genome. Given the phylogenetic tree of the four <em>Zymoseptoria</em> species, as shown in <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Stukenbrock <em>et al.</em> (2012)</a>, the other TE groups were distributed in either two or three species with eight different ancestral relationships (<a href="http://www.g3journal.org/content/suppl/2015/04/27/g3.115.017731.DC1/TableS2.pdf">Table S2</a>). This analysis mainly showed that these species have undergone events of invasion and expansion of distinct TEs after their divergence.</p></div></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-38">“Omics” analyses such as comparative and evolutionary genomics, transcriptomics, epigenomics, or proteomics mainly rely on the accuracy of predicted gene models. The <em>Zymoseptoria</em> species complex provides a unique model system for the application of “omics” data to study the underlying genetics of pathogenicity and host specificity of plant pathogens (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Stukenbrock <em>et al.</em> 2010</a>, <a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">2011</a>, <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">2012</a>). However, updated and comparable annotations of gene and TE content in species of <em>Zymoseptoria</em> have so far been missing. Here, we present a new gene and repeat annotation of the prominent fungal wheat pathogen <em>Z. tritici</em> and its three close relatives: <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>; therefore, we provide a valuable resource for future functional, experimental, and evolutionary studies.</p><p id="p-39">We conducted the re-annotation of the IPO323 genome using a pipeline that identifies the most probable gene models based on information from transcript assemblies, homology searches, and <em>ab initio</em> gene predictors (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Haas <em>et al.</em> 2011</a>). This allowed us to identify 11,839 gene models, including 1200 models that were not predicted previously by the JGI. The support of RNA-seq data for most of these genes underlines the potential of our pipeline for identifying novel genes.</p><p id="p-40">We applied the same annotation pipeline to re-sequenced genomes of the three close relatives of <em>Z. tritici</em>, <em>Z. pseudotritici</em>, <em>Z. ardabiliae</em>, and <em>Z. brevis</em>, and could call comparable numbers of genes in these species (<em>∼</em>10,500 models). Of these, 7786 represent core <em>Zymoseptoria</em> genes shared among the four species. Our exhaustive search for gene orthology also identified genes shared by two or three species and suggested recent gains or losses of genes among the <em>Zymoseptoria</em> species. Notably, we also identified 1798 orphan genes in <em>Z. tritici</em> enriched in putative pathogenicity-related factors. Species-specific orphan genes in <em>Z. tritici</em> and in the three other species may include key determinants of virulence in wheat and other grass hosts. Further functional studies will allow this hypothesis to be tested.</p><p id="p-41">Recently, a new identification and classification of repeats and TEs in <em>Z. tritici</em> was conducted (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Dhillon <em>et al.</em> 2014</a>). In the present study we found very similar patterns in terms of repeat and TE distribution, number of families, and evidence of TE activity. However, we could further improve the assignment of repeat families to known TE families; 39.8% of the 93 repeat families identified by Dhillon <em>et al.</em> were not categorized into known TE classes. We were able to improve the TE classification, leaving only 9% of the 111 repeat families unclassified. The TE annotation pipeline used here contains a number of supplementary steps of clustering of the REPET outputs and allowed us to identify more complete elements. These were all manually checked to avoid the generation of false or chimera elements.</p><p id="p-42">The gene and TE annotation of the four <em>Zymoseptoria</em> species genomes coupled with preliminary comparative analysis allowed us to highlight interesting features relevant to the biological life traits of these organisms. Establishment of homologous sequence families across the four species showed that a large part of genes are shared between two or more species and that the major difference of gene content relies on orphan genes. These orphan genes, in all four species, are significantly enriched in putative pathogenicity-related genes and potentially play a role in the determination of host specificity. Similar presence–absence patterns of pathogenicity-related genes were shown to be strong determinants of host range in plant pathogens, <em>e.g.</em>, between lineages of the rice blast pathogen <em>Magnaporthe oryzae</em>, the wilt pathogen <em>Verticillium dahliae</em>, and lineages of the <em>Leptosphaeria maculans–Leptosphaeria biglobosa</em> species complex (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Couch <em>et al.</em> 2005</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de Jonge <em>et al.</em> 2012</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Grandaubert <em>et al.</em> 2014</a>). Similarly, comparison of TE families showed a majority of species-specific sequences; this underlies the fact that each species genome has been widely invaded after the species divergence. We speculate that TEs may have played a role in host specialization by the acquisition or modification of pathogenicity-related traits. Also, large TE-rich regions could have played a role in speciation through the suppression of homologous recombination in chromosomal regions with inversions or translocations as proposed for the <em>Leptosphaeria maculans–Leptosphaeria biglobosa</em> species complex (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Grandaubert <em>et al.</em> 2014</a>). In <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Dhillon <em>et al.</em> (2014)</a>, the authors stated that several genes with putative pathogenicity-related functions were found to be associated with TEs. Preliminary analyses of genes and TEs tend to show that orphan genes are located significantly closer to TEs than other nonorphan genes (J. Grandaubert, unpublished data). TEs thus may play a role in the origin and evolution of these genes.</p><p id="p-43">The annotation data presented here will greatly improve our understanding of gene evolution, adaptation, and the role of TEs in pathogen evolution. The predicted gene models will furthermore greatly support future studies of gene expression in <em>Z. tritici</em> and its related species and provide a strong basis to understand the underlying genetics and molecular biology of host–pathogen interactions in <em>Zymoseptoria</em> spp.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">This work was supported by intramural funding from the Max Planck Society (to E.H.S.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-45">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017731/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017731/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-46"><em>Communicating editor: J. D. Faris</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-02"><span class="received-label">Received </span>March 2, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-22"><span class="accepted-label">Accepted </span>April 22, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Grandaubert <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.2" data-doi="10.1111/j.1365-294X.2005.02536.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Migration patterns among global populations of the pathogenic fungus <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1881</span>–<span class="cit-lpage">1896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBanke%26rft.auinit1%253DS.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1881%26rft.epage%253D1896%26rft.atitle%253DMigration%2Bpatterns%2Bamong%2Bglobal%2Bpopulations%2Bof%2Bthe%2Bpathogenic%2Bfungus%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02536.x%26rft_id%253Dinfo%253Apmid%252F15910313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02536.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15910313&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229190300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.3" data-doi="10.1101/gr.1865504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clamp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">GeneWise and Genomewise.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBirney%26rft.auinit1%253DE.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D988%26rft.epage%253D995%26rft.atitle%253DGeneWise%2Band%2BGenomewise%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1865504%26rft_id%253Dinfo%253Apmid%252F15123596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzUvOTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1323.4"><div class="cit-metadata unstructured">Coggill, P., R. D. Finn, and A. Bateman, 2008 Identifying protein domains with the Pfam database. Curr. Protoc. Bioinformatics Chapter 2:Unit 2.5.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.5" data-doi="10.1534/genetics.105.041780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Couch</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fudal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebrun</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tharreau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valent</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Origins of host-specific populations of the blast pathogen <em>Magnaporthe oryzae</em> in crop domestication with subsequent expansion of pandemic clones on rice and weeds of rice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.041780%26rft_id%253Dinfo%253Apmid%252F15802503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzIvNjEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.6" data-doi="10.1371/journal.pgen.1003567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Breakage-fusion-bridge cycles and large insertions contribute to the rapid evolution of accessory chromosomes in a fungal pathogen.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003567%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003567%26rft_id%253Dinfo%253Apmid%252F23785303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23785303&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.7" data-doi="10.1016/j.pbi.2011.03.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Jonge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomma</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">How filamentous pathogens co-opt plants: the ins and outs of fungal effectors.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">400</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253Dde%2BJonge%26rft.auinit1%253DR.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D400%26rft.epage%253D406%26rft.atitle%253DHow%2Bfilamentous%2Bpathogens%2Bco-opt%2Bplants%253A%2Bthe%2Bins%2Band%2Bouts%2Bof%2Bfungal%2Beffectors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2011.03.005%26rft_id%253Dinfo%253Apmid%252F21454120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2011.03.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21454120&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.8" data-doi="10.1073/pnas.1119623109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Jonge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Esse</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruthachalam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santhanam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tomato immune receptor Ve1 recognizes effector of multiple fungal pathogens uncovered by genome and RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">5110</span>–<span class="cit-lpage">5115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1119623109%26rft_id%253Dinfo%253Apmid%252F22416119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzEzLzUxMTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTMyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.9" data-doi="10.1101/gr.152660.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Jonge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kombrink</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Berg</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yadeta</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Extensive chromosomal reshuffling drives evolution of virulence in an asexual pathogen.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1271</span>–<span class="cit-lpage">1282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.152660.112%26rft_id%253Dinfo%253Apmid%252F23685541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzgvMTI3MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.10" data-doi="10.1534/genetics.110.117408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavaletto</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accidental amplification and inactivation of a methyltransferase gene eliminates cytosine methylation in <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.117408%26rft_id%253Dinfo%253Apmid%252F20610411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg2LzEvNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTMyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.11" data-doi="10.1186/1471-2164-15-1132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamelin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The landscape of transposable elements in the finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1132%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1132%26rft_id%253Dinfo%253Apmid%252F25519841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25519841&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.12" data-doi="10.1038/nprot.2007.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emanuelsson</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Locating proteins in the cell using TargetP, SignalP, and related tools.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DEmanuelsson%26rft.auinit1%253DO.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D953%26rft.epage%253D971%26rft.atitle%253DLocating%2Bproteins%2Bin%2Bthe%2Bcell%2Busing%2BTargetP%252C%2BSignalP%2Band%2Brelated%2Btools.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.131%26rft_id%253Dinfo%253Apmid%252F17446895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17446895&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253138600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.13" data-doi="10.1146/annurev-biophys-050511-102321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fass</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Disulfide bonding in protein biophysics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biophys.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnnu%2BRev%2BBiophys%26rft.aulast%253DFass%26rft.auinit1%253DD.%26rft.volume%253D41%26rft.spage%253D63%26rft.epage%253D79%26rft.atitle%253DDisulfide%2Bbonding%2Bin%2Bprotein%2Bbiophysics.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biophys-050511-102321%26rft_id%253Dinfo%253Apmid%252F22224600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biophys-050511-102321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22224600&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.14" data-doi="10.1371/journal.pone.0016526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flutre</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duprat</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuillet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quesneville</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Considering transposable element diversification in de novo annotation approaches.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e16526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFlutre%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De16526%26rft.epage%253De16526%26rft.atitle%253DConsidering%2Btransposable%2Belement%2Bdiversification%2Bin%2Bde%2Bnovo%2Bannotation%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016526%26rft_id%253Dinfo%253Apmid%252F21304975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304975&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.15" data-doi="10.1016/j.tig.2004.07.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galagan</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">RIP: the evolutionary cost of genome defense.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DGalagan%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D20%26rft.issue%253D9%26rft.spage%253D417%26rft.epage%253D423%26rft.atitle%253DRIP%253A%2Bthe%2Bevolutionary%2Bcost%2Bof%2Bgenome%2Bdefense.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.07.007%26rft_id%253Dinfo%253Apmid%252F15313550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15313550&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223726600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.16" data-doi="10.1101/gr.4086505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giardine</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riemer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardison</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burhans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elnitski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A. Galaxy: a platform for interactive large-scale genome analysis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4086505%26rft_id%253Dinfo%253Apmid%252F16169926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.17" data-doi="10.1371/journal.pgen.1002070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">M’barek</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGoodwin%26rft.auinit1%253DS.%2BB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002070%26rft.epage%253De1002070%26rft.atitle%253DFinished%2Bgenome%2Bof%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola%2Breveals%2Bdispensome%2Bstructure%252C%2Bchromosome%2Bplasticity%252C%2Band%2Bstealth%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002070%26rft_id%253Dinfo%253Apmid%252F21695235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21695235&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.18" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.19" data-doi="10.1186/1471-2164-15-891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grandaubert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soyer</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Wouw</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transposable element-assisted evolution and adaptation to host plant within the <em>Leptosphaeria maculans-Leptosphaeria biglobosa</em> species complex of fungal pathogens.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-891%26rft_id%253Dinfo%253Apmid%252F25306241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25306241&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.20" data-doi="10.1093/nar/gkg770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. K.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">5654</span>–<span class="cit-lpage">5666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D31%26rft.issue%253D19%26rft.spage%253D5654%26rft.epage%253D5666%26rft.atitle%253DImproving%2Bthe%2BArabidopsis%2Bgenome%2Bannotation%2Busing%2Bmaximal%2Btranscript%2Balignment%2Bassemblies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg770%26rft_id%253Dinfo%253Apmid%252F14500829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xOS81NjU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.21" data-doi="10.1186/gb-2008-9-1-r7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253DR7%26rft.epage%253DR7%26rft.atitle%253DAutomated%2Beukaryotic%2Bgene%2Bstructure%2Bannotation%2Busing%2BEVidenceModeler%2Band%2Bthe%2BProgram%2Bto%2BAssemble%2BSpliced%2BAlignments.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-1-r7%26rft_id%253Dinfo%253Apmid%252F18190707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-1-r7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18190707&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.22" data-doi="10.1080/21501203.2011.606851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Approaches to Fungal Genome Annotation.</span> <abbr class="cit-jnl-abbrev">Mycology</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMycology%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D118%26rft.epage%253D141%26rft.atitle%253DApproaches%2Bto%2BFungal%2BGenome%2BAnnotation.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F21501203.2011.606851%26rft_id%253Dinfo%253Apmid%252F22059117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/21501203.2011.606851&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22059117&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.23" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.24" data-doi="10.1186/1471-2105-9-478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RIPCAL: a tool for alignment-based analysis of repeat-induced point mutations in fungal genomic sequences.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHane%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D9%26rft.spage%253D478%26rft.epage%253D478%26rft.atitle%253DRIPCAL%253A%2Ba%2Btool%2Bfor%2Balignment-based%2Banalysis%2Bof%2Brepeat-induced%2Bpoint%2Bmutations%2Bin%2Bfungal%2Bgenomic%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-478%26rft_id%253Dinfo%253Apmid%252F19014496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19014496&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.25" data-doi="10.1093/nar/gkr948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attwood</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">InterPro in 2011: new developments in the family and domain prediction database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D306</span>–<span class="cit-lpage">D312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr948%26rft_id%253Dinfo%253Apmid%252F22096229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9EMzA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.26" data-doi="10.1159/000084979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapitonov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlicek</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klonowski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohany</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Repbase Update, a database of eukaryotic repetitive elements.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D110%26rft.issue%253D1-4%26rft.spage%253D462%26rft.epage%253D467%26rft.atitle%253DRepbase%2BUpdate%252C%2Ba%2Bdatabase%2Bof%2Beukaryotic%2Brepetitive%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000084979%26rft_id%253Dinfo%253Apmid%252F16093699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000084979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093699&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231064600047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.27" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">2013 MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvNzcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.28" data-doi="10.1093/gbe/evu101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharyya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poppe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Expression profiling of the wheat pathogen <em>Zymoseptoria tritici</em> reveals genomic patterns of transcription and host-specific regulatory programs.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1353</span>–<span class="cit-lpage">1365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu101%26rft_id%253Dinfo%253Apmid%252F24920004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjYvNi8xMzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.29" data-doi="10.1006/jmbi.2000.4315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKrogh%26rft.auinit1%253DA.%26rft.volume%253D305%26rft.issue%253D3%26rft.spage%253D567%26rft.epage%253D580%26rft.atitle%253DPredicting%2Btransmembrane%2Bprotein%2Btopology%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%253A%2Bapplication%2Bto%2Bcomplete%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4315%26rft_id%253Dinfo%253Apmid%252F11152613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11152613&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167760800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.30" data-doi="10.1093/nar/26.4.1107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lukashin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">GeneMark.hmm: new solutions for gene finding.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.4.1107%26rft_id%253Dinfo%253Apmid%252F9461475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI2LzQvMTEwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.31" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.</span> <abbr class="cit-jnl-abbrev">GigaScience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.32" data-doi="10.1093/nar/gkq1189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchler-Bauer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitsaz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">CDD: a Conserved Domain Database for the functional annotation of proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D225</span>–<span class="cit-lpage">D229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1189%26rft_id%253Dinfo%253Apmid%252F21109532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8xL0QyMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTMyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.33" data-doi="10.1186/1471-2105-12-116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miele</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ultra-fast sequence clustering from similarity networks with SiLiX.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMiele%26rft.auinit1%253DV.%26rft.volume%253D12%26rft.spage%253D116%26rft.epage%253D116%26rft.atitle%253DUltra-fast%2Bsequence%2Bclustering%2Bfrom%2Bsimilarity%2Bnetworks%2Bwith%2BSiLiX.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-116%26rft_id%253Dinfo%253Apmid%252F21513511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21513511&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.34" data-doi="10.1371/journal.pone.0049904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morais do Amaral</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoniw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudd</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammond-Kosack</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Defining the predicted protein secretome of the fungal wheat leaf pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e49904</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De49904%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049904%26rft_id%253Dinfo%253Apmid%252F23236356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23236356&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muszewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steczkiewicz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginalski</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DIRS and Ngaro Retrotransposons in Fungi.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e76319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De76319%26rft_id%253Dinfo%253Apmid%252F24086727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24086727&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.36" data-doi="10.1371/journal.ppat.1003037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohm</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feau</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horwitz</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D8%26rft.spage%253De1003037%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003037%26rft_id%253Dinfo%253Apmid%252F23236275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23236275&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.37" data-doi="10.1371/journal.pgen.1003669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palma-Guerrero</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowbel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome wide association identifies novel loci involved in fungal communication.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003669%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003669%26rft_id%253Dinfo%253Apmid%252F23935534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23935534&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.38" data-doi="10.1016/j.gene.2006.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritham</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Putliwala</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feschotte</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mavericks, a novel class of giant transposable elements widespread in eukaryotes and related to DNA viruses.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">390</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DPritham%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D390%26rft.issue%253D1-2%26rft.spage%253D3%26rft.epage%253D17%26rft.atitle%253DMavericks%252C%2Ba%2Bnovel%2Bclass%2Bof%2Bgiant%2Btransposable%2Belements%2Bwidespread%2Bin%2Beukaryotes%2Band%2Brelated%2Bto%2BDNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2006.08.008%26rft_id%253Dinfo%253Apmid%252F17034960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2006.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17034960&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.39" data-doi="10.3767/003158511X571841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quaedvlieg</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kema</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groenewald</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verkley</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seifbarghi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Zymoseptoria</em> gen. nov.: a new genus to accommodate Septoria-like species occurring on graminicolous hosts.</span> <abbr class="cit-jnl-abbrev">Persoonia</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPersoonia%26rft.aulast%253DQuaedvlieg%26rft.auinit1%253DW.%26rft.volume%253D26%26rft.spage%253D57%26rft.epage%253D69%26rft.atitle%253DZymoseptoria%2Bgen.%2Bnov.%253A%2Ba%2Bnew%2Bgenus%2Bto%2Baccommodate%2BSeptoria-like%2Bspecies%2Boccurring%2Bon%2Bgraminicolous%2Bhosts.%26rft_id%253Dinfo%253Adoi%252F10.3767%252F003158511X571841%26rft_id%253Dinfo%253Apmid%252F22025804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3767/003158511X571841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22025804&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292593800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.40" data-doi="10.1093/nar/gki442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quevillon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silventoinen</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harte</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">InterProScan: protein domains identifier.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">W116</span>–<span class="cit-lpage">W120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DQuevillon%26rft.auinit1%253DE.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW116%26rft.epage%253DW120%26rft.atitle%253DInterProScan%253A%2Bprotein%2Bdomains%2Bidentifier%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki442%26rft_id%253Dinfo%253Apmid%252F15980438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8yL1cxMTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTMyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.41" data-doi="10.1038/ncomms1189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouxel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandaubert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoede</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Wouw</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effector diversification within compartments of the <em>Leptosphaeria maculans</em> genome affected by Repeat-Induced Point mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DRouxel%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.spage%253D202%26rft.epage%253D202%26rft.atitle%253DEffector%2Bdiversification%2Bwithin%2Bcompartments%2Bof%2Bthe%2BLeptosphaeria%2Bmaculans%2Bgenome%2Baffected%2Bby%2BRepeat-Induced%2BPoint%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1189%26rft_id%253Dinfo%253Apmid%252F21326234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21326234&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.42" data-doi="10.1101/gr.10.4.516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solovyev</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Ab initio gene finding in Drosophila genomic DNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSalamov%26rft.auinit1%253DA.%2BA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D516%26rft.epage%253D522%26rft.atitle%253DAb%2Binitio%2BGene%2BFinding%2Bin%2BDrosophila%2BGenomic%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.516%26rft_id%253Dinfo%253Apmid%252F10779491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1323.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source">Molecular cloning: a laboratory manual</span>, <span class="cit-edition">Ed. 3rd</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.44" data-doi="10.1371/journal.pgen.1004227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soyer</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El Ghalid</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaser</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ollivier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linglin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epigenetic control of effector gene expression in the plant pathogenic fungus Leptosphaeria maculans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004227%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004227%26rft_id%253Dinfo%253Apmid%252F24603691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24603691&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.45" data-doi="10.1126/science.1194573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spanu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amselem</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgis</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soanes</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome expansion and gene loss in powdery mildew fungi reveal tradeoffs in extreme parasitism.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1543</span>–<span class="cit-lpage">1546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSpanu%26rft.auinit1%253DP.%2BD.%26rft.volume%253D330%26rft.issue%253D6010%26rft.spage%253D1543%26rft.epage%253D1546%26rft.atitle%253DGenome%2BExpansion%2Band%2BGene%2BLoss%2Bin%2BPowdery%2BMildew%2BFungi%2BReveal%2BTradeoffs%2Bin%2BExtreme%2BParasitism%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194573%26rft_id%253Dinfo%253Apmid%252F21148392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMC8xNTQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.46" data-doi="10.1093/bioinformatics/btg1080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Gene prediction with a hidden Markov model and a new intron submodel.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">ii215</span>–<span class="cit-lpage">ii225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D90002%26rft.spage%253Dii215%26rft.epage%253D225%26rft.atitle%253DGene%2Bprediction%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%2Band%2Ba%2Bnew%2Bintron%2B%2Bsubmodel%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg1080%26rft_id%253Dinfo%253Apmid%252F14534192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxNjoiMTkvc3VwcGxfMi9paTIxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.47" data-doi="10.1146/annurev.phyto.112408.132637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stergiopoulos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fungal effector proteins.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253Dde%2BWit%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D47%26rft.spage%253D233%26rft.epage%253D263%26rft.atitle%253DFungal%2Beffector%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.112408.132637%26rft_id%253Dinfo%253Apmid%252F19400631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.112408.132637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19400631&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270591800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.48" data-doi="10.1093/molbev/msl169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Javan-Nikkhah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Origin and domestication of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> via sympatric speciation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl169%26rft_id%253Dinfo%253Apmid%252F17095534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMzk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.49" data-doi="10.1371/journal.pgen.1001189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jørgensen</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome and chromosome evolution associated with host adaptation and speciation of the wheat pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DStukenbrock%26rft.auinit1%253DE.%2BH.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De1001189%26rft.epage%253De1001189%26rft.atitle%253DWhole-genome%2Band%2Bchromosome%2Bevolution%2Bassociated%2Bwith%2Bhost%2Badaptation%2Band%2Bspeciation%2Bof%2Bthe%2Bwheat%2Bpathogen%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001189%26rft_id%253Dinfo%253Apmid%252F21203495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21203495&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.50" data-doi="10.1101/gr.118851.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutheil</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The making of a new pathogen: insights from comparative population genomics of the domesticated wheat pathogen <em>Mycosphaerella graminicola</em> and its wild sister species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2157</span>–<span class="cit-lpage">2166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118851.110%26rft_id%253Dinfo%253Apmid%252F21994252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8yMTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.51" data-doi="10.3852/11-374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quaedvlieg</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Javan-Nikhah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crous</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Zymoseptoria ardabiliae</em> and <em>Z. pseudotritici</em>, two progenitor species of the septoria tritici leaf blotch fungus <em>Z. tritici</em> (synonym: <em>Mycosphaerella graminicola</em>).</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft_id%253Dinfo%253Adoi%252F10.3852%252F11-374%26rft_id%253Dinfo%253Apmid%252F22675045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibXljb2xvZ2lhIjtzOjU6InJlc2lkIjtzOjEwOiIxMDQvNi8xMzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.52" data-doi="10.1093/bioinformatics/btm098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzek</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGarvey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazumder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">UniRef: comprehensive and non-redundant UniProt reference clusters.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1282</span>–<span class="cit-lpage">1288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm098%26rft_id%253Dinfo%253Apmid%252F17379688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTAvMTI4MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.53" data-doi="10.1101/gr.081612.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ter-Hovhannisyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernoff</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1979</span>–<span class="cit-lpage">1990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081612.108%26rft_id%253Dinfo%253Apmid%252F18757608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8xOTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.54" data-doi="10.1038/nrg2165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua-Van</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A unified classification system for eukaryotic transposable elements.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMathew%26rft.auinit1%253DC.%2BG.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D973%26rft.epage%253D982%26rft.atitle%253DA%2Bunified%2Bclassification%2Bsystem%2Bfor%2Beukaryotic%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165%26rft_id%253Dinfo%253Apmid%252F17984973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17984973&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.55" data-doi="10.1371/journal.pone.0005863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Lee</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben M’barek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Meiosis drives extraordinary genome plasticity in the haploid fungal plant pathogen <em>Mycosphaerella graminicola</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5863</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWittenberg%26rft.auinit1%253DA.%2BH.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De5863%26rft.epage%253De5863%26rft.atitle%253DMeiosis%2Bdrives%2Bextraordinary%2Bgenome%2Bplasticity%2Bin%2Bthe%2Bhaploid%2Bfungal%2Bplant%2Bpathogen%2BMycosphaerella%2Bgraminicola.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005863%26rft_id%253Dinfo%253Apmid%252F19516898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19516898&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1323.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1323.56" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTMyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/7/1323.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1317" title="Multiple Conserved Heteroplasmic Sites in tRNA Genes in the Mitochondrial Genomes of Terrestrial Isopods (Oniscidea)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1335" title="Transcriptional Response to Acute Thermal Exposure in Juvenile Chinook Salmon Determined by RNAseq" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/7"><span>Volume 5 Issue 7, July 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/7.cover-source.jpg?itok=IZbYpE05" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1323" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Mf7EGWP0vDnHE0B158bY_AYufT5ge59xumcOb2rI_l4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/7/1323" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/7/1323" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/7/1323" class="active" data-icon-position="" data-hide-link-title="0">RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus Zymoseptoria Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459981" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LVeExeh6cMtYWNUAEa4cB-Tx1KWcVP_Zgx9L7K7T36o" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1323.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/7/1323" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VZifInAHN5tJocJ5ImfiM6hk_nRXLxtSqudp1EV69Ls" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/7/1323" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1323.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459981" id="node459981--2" data-pisa="ggg;5/7/1323" data-pisa-master="ggg;g3.115.017731" data-apath="/ggg/5/7/1323.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus <em>Zymoseptoria</em> Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jonathan Grandaubert</span>, <span class="highwire-citation-author" data-delta="1">Amitava Bhattacharyya</span> and <span class="highwire-citation-author" data-delta="2">Eva H. Stukenbrock</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1323-1333;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017731 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459981--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan Grandaubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BGrandaubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grandaubert%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BGrandaubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amitava Bhattacharyya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amitava%2BBhattacharyya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhattacharyya%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmitava%2BBhattacharyya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eva H. Stukenbrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eva%2BH.%2BStukenbrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stukenbrock%20EH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEva%2BH.%2BStukenbrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:estukenbrock@bot.uni-kiel.de" class="" data-icon-position="" data-hide-link-title="0">estukenbrock@bot.uni-kiel.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/7/1323" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/7/1323&amp;title=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/7/1323&amp;title=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/7/1323&amp;title=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/7/1323&amp;text=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/7/1323&amp;title=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/7/1323&amp;t=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/7/1323&amp;title=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/7/1323&amp;title=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459981/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459981" id="node459981--3" data-pisa="ggg;5/7/1323" data-pisa-master="ggg;g3.115.017731" data-apath="/ggg/5/7/1323.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">RNA-seq-Based Gene Annotation and Comparative Genomics of Four Fungal Grass Pathogens in the Genus <em>Zymoseptoria</em> Identify Novel Orphan Genes and Species-Specific Invasions of Transposable Elements</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jonathan Grandaubert</span>, <span class="highwire-citation-author" data-delta="1">Amitava Bhattacharyya</span> and <span class="highwire-citation-author" data-delta="2">Eva H. Stukenbrock</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1323-1333;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017731 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459981--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan Grandaubert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BGrandaubert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grandaubert%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BGrandaubert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amitava Bhattacharyya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amitava%2BBhattacharyya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhattacharyya%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmitava%2BBhattacharyya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eva H. Stukenbrock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Max Planck Institute for Evolutionary Biology, August-Thienemann-Str. 2, 24306 Plön, Germany and Christian-Albrechts University of Kiel, Am Botanischen Garten 9-11, 24118 Kiel, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eva%2BH.%2BStukenbrock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stukenbrock%20EH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEva%2BH.%2BStukenbrock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:estukenbrock@bot.uni-kiel.de" class="" data-icon-position="" data-hide-link-title="0">estukenbrock@bot.uni-kiel.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1323&amp;rft.epage=1333&amp;rft.atitle=RNA-seq-Based%20Gene%20Annotation%20and%20Comparative%20Genomics%20of%20Four%20Fungal%20Grass%20Pathogens%20in%20the%20Genus%20Zymoseptoria%20Identify%20Novel%20Orphan%20Genes%20and%20Species-Specific%20Invasions%20of%20Transposable%20Elements&amp;rft.volume=5&amp;rft.issue=7&amp;rft.date=2015-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Grandaubert%2C+Jonathan&amp;rft.au=Bhattacharyya%2C+Amitava&amp;rft.au=Stukenbrock%2C+Eva+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459981/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459981/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459981/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459981/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459981/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459981/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459981/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459981/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459981/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459981/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459981/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459981/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/7/1323" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/7/1323.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/7/1323.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/7/1323.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/7/1323.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
