<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/7/1379" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/7/1379.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/7/1379.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/7/1379.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/7/1379.atom" />
<meta name="HW.pisa" content="ggg;5/7/1379" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.115.018721" />
<meta name="DC.Date" content="2015-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Polleys and Bertuch. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Saccharomyces cerevisiae [Iml1][1] complex inhibits TORC1 signaling and SEACAT antagonizes the [Iml1][1] complex. Conditions in which SEACAT functions to inhibit [Iml1][1] and, hence, TORC1 signaling, remain largely unknown. The SEACAT member [Sea3][2] was linked previously to telomere maintenance and DNA repair via genome-wide genetic and physical interaction studies. Therefore, we questioned whether [Sea3][2] functioned through TORC1 to influence these pathways. Deletion of [SEA3][2] delayed the emergence of telomerase-independent survivors that use break-induced replication (BIR) to maintain their telomeres. Similarly, [sea3][2]∆ mutants exhibited a delay in colony formation in a BIR assay strain after double-strand break (DSB) induction as well as on the DNA-damaging agent bleomycin. Deletion of [IML1][1] rescued the impaired growth of [sea3][2]∆ mutants after DNA damage, consistent with [Sea3][2] functioning as a regulator of TORC1 signaling. The delay was not attributable to slowed DSB repair or termination of the DNA damage checkpoint but to tryptophan auxotrophy. High levels of tryptophan in yeast peptone dextrose media did not rescue the delay in colony formation, suggesting a defect in tryptophan import, although levels of the high-affinity tryptophan permease [Tat2][3] were not perturbed in the [sea3][2] Δ mutant. Addition of quinolinic acid, an intermediate of the de novo NAD+ biosynthetic pathway, however, rescued the delay in colony formation in the [sea3][2] Δ mutant. Together, these findings highlight the importance of enforcement of TORC1 signaling and suggest that internal tryptophan levels influence growth recovery post DNA damage through the role of tryptophan in NAD+ synthesis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380" />
<meta name="DC.Contributor" content="Erica J. Polleys" />
<meta name="DC.Contributor" content="Alison A. Bertuch" />
<meta name="article:published_time" content="2015-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;Iml1 complex inhibits TORC1 signaling and SEACAT antagonizes the Iml1 complex. Conditions in which SEACAT functions to inhibit Iml1 and, hence, TORC1 signaling, remain largely unknown. The SEACAT member Sea3 was linked previously to telomere maintenance and DNA repair via genome-wide genetic and physical interaction studies. Therefore, we questioned whether Sea3 functioned through TORC1 to influence these pathways. Deletion of &lt;i&gt;SEA3&lt;/i&gt; delayed the emergence of telomerase-independent survivors that use break-induced replication (BIR) to maintain their telomeres. Similarly, &lt;i&gt;sea3∆&lt;/i&gt; mutants exhibited a delay in colony formation in a BIR assay strain after double-strand break (DSB) induction as well as on the DNA-damaging agent bleomycin. Deletion of &lt;i&gt;IML1&lt;/i&gt; rescued the impaired growth of &lt;i&gt;sea3∆&lt;/i&gt; mutants after DNA damage, consistent with Sea3 functioning as a regulator of TORC1 signaling. The delay was not attributable to slowed DSB repair or termination of the DNA damage checkpoint but to tryptophan auxotrophy. High levels of tryptophan in yeast peptone dextrose media did not rescue the delay in colony formation, suggesting a defect in tryptophan import, although levels of the high-affinity tryptophan permease Tat2 were not perturbed in the &lt;i&gt;sea3&lt;/i&gt;Δ mutant. Addition of quinolinic acid, an intermediate of the &lt;i&gt;de novo&lt;/i&gt; NAD+ biosynthetic pathway, however, rescued the delay in colony formation in the &lt;i&gt;sea3&lt;/i&gt;Δ mutant. Together, these findings highlight the importance of enforcement of TORC1 signaling and suggest that internal tryptophan levels influence growth recovery post DNA damage through the role of tryptophan in NAD+ synthesis.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/07/01" />
<meta name="citation_mjid" content="ggg;5/7/1379" />
<meta name="citation_id" content="5/7/1379" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/7/1379" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/7/1379.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/7/1379.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/7/1379.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.018721" />
<meta name="citation_pmid" content="25943524" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1379" />
<meta name="citation_lastpage" content="1389" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Erica J. Polleys" />
<meta name="citation_author_institution" content="Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6547-3624" />
<meta name="citation_author" content="Alison A. Bertuch" />
<meta name="citation_author_institution" content="Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author_institution" content="Department of Pediatrics and Department of Molecular &amp; Human Genetics, Baylor College of Medicine, Houston, Texas 77030" />
<meta name="citation_author_email" content="abertuch@bcm.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1864-8502" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=F. Abe;citation_author=H. Iida;citation_title=Pressure-Induced Differential Regulation of the Two Tryptophan Permeases Tat1 and Tat2 by Ubiquitin Ligase Rsp5 and Its Binding Proteins, Bul1 and Bul2;citation_pages=7566-7584;citation_volume=23;citation_year=2003;citation_issue=21;citation_pmid=14560004;citation_doi=10.1128/MCB.23.21.7566-7584.2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. G. Addinall;citation_author=M. Downey;citation_author=M. Yu;citation_author=M. K. Zubko;citation_author=J. Dewar;citation_title=A genomewide suppressor and enhancer analysis of cdc13–1 reveals varied cellular processes influencing telomere capping in Saccharomyces cerevisiae.;citation_volume=180;citation_year=2008;citation_pmid=18845848;citation_doi=10.1534/genetics.108.092577" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. G. Addinall;citation_author=E. M. Holstein;citation_author=C. Lawless;citation_author=M. Yu;citation_author=K. Chapman;citation_title=Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.;citation_pages=e1001362-e1001362;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21490951;citation_doi=10.1371/journal.pgen.1001362" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. H. Askree;citation_author=T. Yehuda;citation_author=S. Smolikov;citation_author=R. Gurevich;citation_author=J. Hawk;citation_title=A genome-wide screen for Saccharomyces cerevisiae deletion mutants that affect telomere length.;citation_volume=101;citation_year=2004;citation_pmid=15161972;citation_doi=10.1073/pnas.0401263101" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. Bar-Peled;citation_author=L. Chantranupong;citation_author=A. D. Cherniack;citation_author=W. W. Chen;citation_author=K. A. Ottina;citation_title=A Tumor Suppressor Complex with GAP Activity for the Rag GTPases That Signal Amino Acid Sufficiency to mTORC1;citation_pages=1100-1106;citation_volume=340;citation_year=2013;citation_issue=6136;citation_pmid=23723238;citation_doi=10.1126/science.1232044" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=T. Beck;citation_author=A. Schmidt;citation_author=M. N. Hall;citation_title=Starvation induces vacuolar targeting and degradation of the tryptophan permease in yeast.;citation_volume=146;citation_year=1999;citation_pmid=10491387;citation_doi=10.1083/jcb.146.6.1227" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. A. Bertuch;citation_author=V. Lundblad;citation_title=EXO1 Contributes to Telomere Maintenance in Both Telomerase-Proficient and Telomerase-Deficient Saccharomyces cerevisiae;citation_pages=1651-1659;citation_volume=166;citation_year=2004;citation_issue=4;citation_pmid=15126387;citation_doi=10.1534/genetics.166.4.1651" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. J. Boulton;citation_author=S. P. Jackson;citation_title=Identification of a Saccharomyces cerevisiae Ku80 homologue: roles in DNA double strand break rejoining and in telomeric maintenance.;citation_volume=24;citation_year=1996;citation_pmid=8972848;citation_doi=10.1093/nar/24.23.4639" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=P. Briza;citation_author=E. Bogengruber;citation_author=A. Thur;citation_author=M. Rutzler;citation_author=M. Munsterkotter;citation_title=Systematic analysis of sporulation phenotypes in 624 non-lethal homozygous deletion strains of Saccharomyces cerevisiae.;citation_pages=403-422;citation_volume=19;citation_year=2002;citation_issue=5;citation_pmid=11921089;citation_doi=10.1002/yea.843" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=H.Y. Chang;citation_author=C. Lawless;citation_author=S.G. Addinall;citation_author=S. Oexle;citation_author=M. Taschuk;citation_title=Genome-Wide Analysis to Identify Pathways Affecting Telomere-Initiated Senescence in Budding Yeast;citation_pages=197-208;citation_volume=1;citation_year=2011;citation_issue=3;citation_pmid=22384331;citation_doi=10.1534/g3.111.000216" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=I. Chiolo;citation_author=W. Carotenuto;citation_author=G. Maffioletti;citation_author=J. H. Petrini;citation_author=M. Foiani;citation_title=Srs2 and Sgs1 DNA Helicases Associate with Mre11 in Different Subcomplexes following Checkpoint Activation and CDK1-Mediated Srs2 Phosphorylation;citation_pages=5738-5751;citation_volume=25;citation_year=2005;citation_issue=13;citation_pmid=15964827;citation_doi=10.1128/MCB.25.13.5738-5751.2005" />
<meta name="citation_reference" content="citation_journal_title=Ann. Neurol.;citation_author=A. M. D’Gama;citation_author=Y. Geng;citation_author=J. A. Couto;citation_author=B. Martin;citation_author=E. A. Boyle;citation_title=Mammalian target of rapamycin pathway mutations cause hemimegalencephaly and focal cortical dysplasia.;citation_pages=720-725;citation_volume=77;citation_year=2015;citation_pmid=25599672;citation_doi=10.1002/ana.24357" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=L. M. Dibbens;citation_author=B. de Vries;citation_author=S. Donatello;citation_author=S. E. Heron;citation_author=B. L. Hodgson;citation_title=Mutations in DEPDC5 cause familial focal epilepsy with variable foci.;citation_pages=546-551;citation_volume=45;citation_year=2013;citation_pmid=23542697;citation_doi=10.1038/ng.2599" />
<meta name="citation_reference" content="citation_title=Mol Cell Proteomics;citation_author=S. Dokudovskaya;citation_author=F. Waharte;citation_author=A. Schlessinger;citation_author=U. Pieper;citation_author=D.P. Devos;citation_title=A conserved coatomer-related complex containing Sec13 and Seh1 dynamically associates with the vacuole in Saccharomyces cerevisiae.;citation_volume=10;citation_year=2011;citation_pmid=21454883;citation_doi=10.1074/mcp.M110.006478" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. Dotiwala;citation_author=V. V. Eapen;citation_author=J. C. Harrison;citation_author=A. Arbel-Eden;citation_author=V. Ranade;citation_title=DNA damage checkpoint triggers autophagy to regulate the initiation of anaphase.;citation_volume=110;citation_year=2013;citation_pmid=23169651;citation_doi=10.1073/pnas.1218065109" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cancer Res.;citation_author=M. Dyavaiah;citation_author=J. P. Rooney;citation_author=S. V. Chittur;citation_author=Q. Lin;citation_author=T. J. Begley;citation_title=Autophagy-dependent regulation of the DNA damage response protein ribonucleotide reductase 1.;citation_volume=9;citation_year=2011;citation_pmid=21343333;citation_doi=10.1158/1541-7786.MCR-10-0473" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Foiani;citation_author=F. Marini;citation_author=D. Gamba;citation_author=G. Lucchini;citation_author=P. Plevani;citation_title=The B subunit of the DNA polymerase alpha-primase complex in Saccharomyces cerevisiae executes an essential function at the initial stage of DNA replication.;citation_pages=923-933;citation_volume=14;citation_year=1994;citation_issue=2;citation_pmid=8289832;citation_doi=10.1128/MCB.14.2.923" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=J. M. Garrett;citation_title=The control of morphogenesis in Saccharomyces cerevisiae by Elm1 kinase is responsive to RAS/cAMP pathway activity and tryptophan availability.;citation_pages=809-820;citation_volume=26;citation_year=1997;citation_issue=4;citation_pmid=9427410;citation_doi=10.1046/j.1365-2958.1997.6231990.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Greenall;citation_author=G. Lei;citation_author=D. C. Swan;citation_author=K. James;citation_author=L. Wang;citation_title=A genome wide analysis of the response to uncapped telomeres in budding yeast reveals a novel role for the NAD+ biosynthetic gene BNA2 in chromosome end protection.;citation_pages=R146-R146;citation_volume=9;citation_year=2008;citation_issue=10;citation_pmid=18828915;citation_doi=10.1186/gb-2008-9-10-r146" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=G. Guillemain;citation_author=E. Ma;citation_author=S. Mauger;citation_author=S. Miron;citation_author=R. Thai;citation_title=Mechanisms of checkpoint kinase Rad53 inactivation after a double-strand break in Saccharomyces cerevisiae.;citation_volume=27;citation_year=2007;citation_pmid=17325030;citation_doi=10.1128/MCB.00863-06" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Ho;citation_author=A. Gruhler;citation_author=A. Heilbut;citation_author=G. D. Bader;citation_author=L. Moore;citation_title=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry.;citation_pages=180-183;citation_volume=415;citation_year=2002;citation_issue=6868;citation_pmid=11805837;citation_doi=10.1038/415180a" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Ishida;citation_author=F. Picard;citation_author=G. Rudolf;citation_author=E. Noe;citation_author=G. Achaz;citation_title=Mutations of DEPDC5 cause autosomal dominant focal epilepsies.;citation_pages=552-555;citation_volume=45;citation_year=2013;citation_pmid=23542701;citation_doi=10.1038/ng.2601" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=S. Jarolim;citation_author=A. Ayer;citation_author=B. Pillay;citation_author=A.C. Gee;citation_author=A. Phrakaysone;citation_title=Saccharomyces cerevisiae genes involved in survival of heat shock.;citation_volume=3;citation_year=2013;citation_pmid=24142923;citation_doi=10.1534/g3.113.007971" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=M. Kato;citation_author=S. J. Lin;citation_title=Regulation of NAD+ metabolism, signaling and compartmentalization in the yeast Saccharomyces cerevisiae.;citation_pages=49-58;citation_volume=23;citation_year=2014;citation_pmid=25096760;citation_doi=10.1016/j.dnarep.2014.07.009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Le;citation_author=J. K. Moore;citation_author=J. E. Haber;citation_author=C. W. Greider;citation_title=RAD50 and RAD51 Define Two Pathways That Collaborate to Maintain Telomeres in the Absence of Telomerase;citation_pages=143-152;citation_volume=152;citation_year=1999;citation_issue=1;citation_pmid=10224249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. S. Lendvay;citation_author=D. K. Morris;citation_author=J. Sah;citation_author=B. Balasubramanian;citation_author=V. Lundblad;citation_title=Senescence Mutants of Saccharomyces cerevisiae With a Defect in Telomere Replication Identify Three Additional EST Genes;citation_pages=1399-1412;citation_volume=144;citation_year=1996;citation_issue=4;citation_pmid=8978029" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=C. Leroy;citation_author=S. E. Lee;citation_author=M. B. Vaze;citation_author=F. Ochsenbein;citation_author=R. Guerois;citation_title=PP2C phosphatases Ptc2 and Ptc3 are required for DNA checkpoint inactivation after a double-strand break.;citation_pages=827-835;citation_volume=11;citation_year=2003;citation_issue=3;citation_pmid=12667463;citation_doi=10.1016/S1097-2765(03)00058-3" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Loewith;citation_author=E. Jacinto;citation_author=S. Wullschleger;citation_author=A. Lorberg;citation_author=J. L. Crespo;citation_title=Two TOR complexes, only one of which is rapamycin sensitive, have distinct roles in cell growth control.;citation_pages=457-468;citation_volume=10;citation_year=2002;citation_issue=3;citation_pmid=12408816;citation_doi=10.1016/S1097-2765(02)00636-6" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=V. Lundblad;citation_author=J. W. Szostak;citation_title=A mutant with a defect in telomere elongation leads to senescence in yeast.;citation_pages=633-643;citation_volume=57;citation_year=1989;citation_issue=4;citation_pmid=2655926;citation_doi=10.1016/0092-8674(89)90132-3" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=V. Lundblad;citation_author=E. H. Blackburn;citation_title=An alternative pathway for yeast telomere maintenance rescues est1- senescence.;citation_pages=347-360;citation_volume=73;citation_year=1993;citation_issue=2;citation_pmid=8477448;citation_doi=10.1016/0092-8674(93)90234-H" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. Lydeard;citation_author=S. Jain;citation_author=M. Yamaguchi;citation_author=J. E. Haber;citation_title=Break-induced replication and telomerase-independent telomere maintenance require Pol32.;citation_pages=820-823;citation_volume=448;citation_year=2007;citation_issue=7155;citation_pmid=17671506;citation_doi=10.1038/nature06047" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. R. Lydeard;citation_author=Z. Lipkin-Moore;citation_author=S. Jain;citation_author=V. V. Eapen;citation_author=J. E. Haber;citation_title=Sgs1 and exo1 redundantly inhibit break-induced replication and de novo telomere addition at broken chromosome ends.;citation_pages=e1000973-e1000973;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20523895;citation_doi=10.1371/journal.pgen.1000973" />
<meta name="citation_reference" content="citation_journal_title=Clin. Genet.;citation_author=C. Martin;citation_author=C. Meloche;citation_author=M. F. Rioux;citation_author=D. K. Nguyen;citation_author=L. Carmant;citation_title=A recurrent mutation in DEPDC5 predisposes to focal epilepsies in the French-Canadian population.;citation_pages=570-574;citation_volume=86;citation_year=2014;citation_pmid=24283814;citation_doi=10.1111/cge.12311" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. Mizushima;citation_author=T. Noda;citation_author=T. Yoshimori;citation_author=Y. Tanaka;citation_author=T. Ishii;citation_title=A protein conjugation system essential for autophagy.;citation_pages=395-398;citation_volume=395;citation_year=1998;citation_issue=6700;citation_pmid=9759731;citation_doi=10.1038/26506" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=K. Ohashi;citation_author=S. Kawai;citation_author=K. Murata;citation_title=Secretion of quinolinic acid, an intermediate in the kynurenine pathway, for utilization in NAD+ biosynthesis in the yeast Saccharomyces cerevisiae.;citation_volume=12;citation_year=2013;citation_pmid=23457190;citation_doi=10.1128/EC.00339-12" />
<meta name="citation_reference" content="citation_journal_title=Science Signaling;citation_journal_abbrev=Sci Signal;citation_author=N. Panchaud;citation_author=M. P. Peli-Gulli;citation_author=C. De Virgilio;citation_title=Amino Acid Deprivation Inhibits TORC1 Through a GTPase-Activating Protein Complex for the Rag Family GTPase Gtr1;citation_pages=ra42-ra42;citation_volume=6;citation_year=2013;citation_issue=277;citation_pmid=23716719;citation_doi=10.1126/scisignal.2004112" />
<meta name="citation_reference" content="citation_journal_title=Cell Cycle;citation_author=N. Panchaud;citation_author=M. P. Peli-Gulli;citation_author=C. De Virgilio;citation_title=SEACing the GAP that nEGOCiates TORC1 activation: evolutionary conservation of Rag GTPase regulation.;citation_pages=2948-2952;citation_volume=12;citation_year=2013;citation_pmid=23974112;citation_doi=10.4161/cc.26000" />
<meta name="citation_reference" content="citation_journal_title=Neurology;citation_author=F. Picard;citation_author=P. Makrythanasis;citation_author=V. Navarro;citation_author=S. Ishida;citation_author=J. de Bellescize;citation_title=DEPDC5 mutations in families presenting as autosomal dominant nocturnal frontal lobe epilepsy.;citation_pages=2101-2106;citation_volume=82;citation_year=2014;citation_pmid=24814846;citation_doi=10.1212/WNL.0000000000000488" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Rizki;citation_author=V. Lundblad;citation_title=Defects in mismatch repair promote telomerase-independent proliferation.;citation_pages=713-716;citation_volume=411;citation_year=2001;citation_issue=6838;citation_pmid=11395777;citation_doi=10.1038/35079641" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Robert;citation_author=F. Vanoli;citation_author=I. Chiolo;citation_author=G. Shubassi;citation_author=K. A. Bernstein;citation_title=HDACs link the DNA damage response, processing of double-strand breaks and autophagy.;citation_pages=74-79;citation_volume=471;citation_year=2011;citation_issue=7336;citation_pmid=21368826;citation_doi=10.1038/nature09803" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=A. Schmidt;citation_author=T. Beck;citation_author=A. Koller;citation_author=J. Kunz;citation_author=M. N. Hall;citation_title=The TOR nutrient signalling pathway phosphorylates NPR1 and inhibits turnover of the tryptophan permease;citation_pages=6924-6931;citation_volume=17;citation_year=1998;citation_issue=23;citation_pmid=9843498;citation_doi=10.1093/emboj/17.23.6924" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=C. Shen;citation_author=C. S. Lancaster;citation_author=B. Shi;citation_author=H. Guo;citation_author=P. Thimmaiah;citation_title=TOR signaling is a determinant of cell survival in response to DNA damage.;citation_volume=27;citation_year=2007;citation_pmid=17698581;citation_doi=10.1128/MCB.00290-07" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. S. Singer;citation_author=D. E. Gottschling;citation_title=TLC1: template RNA component of Saccharomyces cerevisiae telomerase;citation_pages=404-409;citation_volume=266;citation_year=1994;citation_issue=5184;citation_pmid=7545955;citation_doi=10.1126/science.7545955" />
<meta name="citation_reference" content="citation_journal_title=J. Biochem.;citation_author=T. Takahara;citation_author=T. Maeda;citation_title=Evolutionarily conserved regulation of TOR signalling.;citation_volume=154;citation_year=2013;citation_pmid=23698095;citation_doi=10.1093/jb/mvt047" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. C. Teng;citation_author=V. A. Zakian;citation_title=Telomere-Telomere Recombination Is an Efficient Bypass Pathway for Telomere Maintenance in Saccharomyces cerevisiae;citation_pages=8083-8093;citation_volume=19;citation_year=1999;citation_issue=12;citation_pmid=10567534;citation_doi=10.1128/MCB.19.12.8083" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=L. Ungar;citation_author=Y. Harari;citation_author=A. Toren;citation_author=M. Kupiec;citation_title=Tor complex 1 controls telomere length by affecting the level of Ku.;citation_pages=2115-2120;citation_volume=21;citation_year=2011;citation_issue=24;citation_pmid=22169538;citation_doi=10.1016/j.cub.2011.11.024" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. B. Vaze;citation_author=A. Pellicioli;citation_author=S. E. Lee;citation_author=G. Ira;citation_author=G. Liberi;citation_title=Recovery from checkpoint-mediated arrest after repair of a double-strand break requires Srs2 helicase.;citation_pages=373-385;citation_volume=10;citation_year=2002;citation_issue=2;citation_pmid=12191482;citation_doi=10.1016/S1097-2765(02)00593-2" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=R. Zoncu;citation_author=A. Efeyan;citation_author=D. M. Sabatini;citation_title=mTOR: from growth signal integration to cancer, diabetes and ageing.;citation_pages=21-35;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21157483;citation_doi=10.1038/nrm3025" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="twitter:description" content="The Saccharomyces cerevisiae [Iml1][1] complex inhibits TORC1 signaling and SEACAT antagonizes the [Iml1][1] complex. Conditions in which SEACAT functions to inhibit [Iml1][1] and, hence, TORC1 signaling, remain largely unknown. The SEACAT member [Sea3][2] was linked previously to telomere maintenance and DNA repair via genome-wide genetic and physical interaction studies. Therefore, we questioned whether [Sea3][2] functioned through TORC1 to influence these pathways. Deletion of [SEA3][2] delayed the emergence of telomerase-independent survivors that use break-induced replication (BIR) to maintain their telomeres. Similarly, [sea3][2]∆ mutants exhibited a delay in colony formation in a BIR assay strain after double-strand break (DSB) induction as well as on the DNA-damaging agent bleomycin. Deletion of [IML1][1] rescued the impaired growth of [sea3][2]∆ mutants after DNA damage, consistent with [Sea3][2] functioning as a regulator of TORC1 signaling. The delay was not attributable to slowed DSB repair or termination of the DNA damage checkpoint but to tryptophan auxotrophy. High levels of tryptophan in yeast peptone dextrose media did not rescue the delay in colony formation, suggesting a defect in tryptophan import, although levels of the high-affinity tryptophan permease [Tat2][3] were not perturbed in the [sea3][2] Δ mutant. Addition of quinolinic acid, an intermediate of the de novo NAD+ biosynthetic pathway, however, rescued the delay in colony formation in the [sea3][2] Δ mutant. Together, these findings highlight the importance of enforcement of TORC1 signaling and suggest that internal tryptophan levels influence growth recovery post DNA damage through the role of tryptophan in NAD+ synthesis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380" />
<meta name="og-title" property="og:title" content="Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/7/1379" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Saccharomyces cerevisiae [Iml1][1] complex inhibits TORC1 signaling and SEACAT antagonizes the [Iml1][1] complex. Conditions in which SEACAT functions to inhibit [Iml1][1] and, hence, TORC1 signaling, remain largely unknown. The SEACAT member [Sea3][2] was linked previously to telomere maintenance and DNA repair via genome-wide genetic and physical interaction studies. Therefore, we questioned whether [Sea3][2] functioned through TORC1 to influence these pathways. Deletion of [SEA3][2] delayed the emergence of telomerase-independent survivors that use break-induced replication (BIR) to maintain their telomeres. Similarly, [sea3][2]∆ mutants exhibited a delay in colony formation in a BIR assay strain after double-strand break (DSB) induction as well as on the DNA-damaging agent bleomycin. Deletion of [IML1][1] rescued the impaired growth of [sea3][2]∆ mutants after DNA damage, consistent with [Sea3][2] functioning as a regulator of TORC1 signaling. The delay was not attributable to slowed DSB repair or termination of the DNA damage checkpoint but to tryptophan auxotrophy. High levels of tryptophan in yeast peptone dextrose media did not rescue the delay in colony formation, suggesting a defect in tryptophan import, although levels of the high-affinity tryptophan permease [Tat2][3] were not perturbed in the [sea3][2] Δ mutant. Addition of quinolinic acid, an intermediate of the de novo NAD+ biosynthetic pathway, however, rescued the delay in colony formation in the [sea3][2] Δ mutant. Together, these findings highlight the importance of enforcement of TORC1 signaling and suggest that internal tryptophan levels influence growth recovery post DNA damage through the role of tryptophan in NAD+ synthesis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
    <title>Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XU0arvYn3bMiSudY2p41sV65sKTlK632mlb-flr9pTQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459983","apath":"\/ggg\/5\/7\/1379.atom","pisa":"ggg;5\/7\/1379","ac":{"\/ggg\/5\/7\/1379.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/7\/1379.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1379"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1379"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/7\/1379?cited-by=yes\u0026legid=ggg;5\/7\/1379":true,"\/content\/5\/7\/1379":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228pNDNHsFd46t8vW+t6XoUSl7i2CEWWrSHVCzKXxBAk4V36G4NeXBhGepDuw9DtGV3OQ7LJ8Z6yioGbHS8I\\\\\\\/eWfPLGglbVkXuglmwBRhoxGi4nm\\\\\\\/x3tqOi1iMk+QOoFdpmvRXVtVxdZKR10k+6wEzt8qGj\\\\\\\/YwahBgrmitrM9G9ZlhjRqxo09j4UacKaI6bxwW+PGPGZcKiHY0yVaFBZqUTKdYUGeiFLbzYG\\\\\\\/hZCbpx+fFXM9KzNYLJrUdX88V\\\\\\\/aHZvNpAlWT+1nDOTGfVdKI4TbZt2hP0rhHicj+wnYSDIqYUqQrf0TmqKp+69tN7xHIjRm4DtxvOvMxcKwUvTz5jleGnPpJpjLP8LzZmLhjw4InWSEaMLrN43ZlsxYEoG7atc1a+8SB4qiFBGBf+T1\\\\\\\/wxJCo8OaljHkDh2HrgHNq73qyRDGXgrcCjxisshyhtGwyfy0mkdB19lOQETjIk57tJzYFA\\\\\\\/SXbPhGrzUy9AZpauHGhaF1ISu6ULJhS3H5P7fZNMtSmuZELsjbS1INnCgiWgbiEM1x6+foZTlDrkNKVEPF7T42IfAztu354ne+q17vZi2EW4R0qr8JH2qJxyqamtPi0eQjZGlwOWfzaZtpj16iGbffytoKB9TLpT2I+qHitqf39PRNSFt072yTWOPt4mJiyuIyGvdnvNIH0dS6HgYzzLCjtqbpc\\\\\\\/6G0Pd5amWoypXy2AEZcBwY5\\\\\\\/tWUeoE2MSV22kDjVkBrXA0i9FCXKUgIOUhmKo2wAr5l267Vf8HNusTsKO1iwhRozcwPrMyPTzm4TrN025V4qztNOJJ+VXNYwgZHgET4xme+DR6FZF\\\\\\\/GQuM\\\\\\\/x8YPXF\\\\\\\/EAQKqVQy3dxdXy3eFDia0200fRfGLtNupBgLLbFF9hk56rrHhqfM8njQMkn\\\\\\\/+vIWXtrIK7LzVWgYGMk4BE2aeAAOih4okwdH2vJIqeCRH+dheiRmfkyYmEexfgAiZfllCuhxG8YKjqQGmaWeKFfuSBzEadDtUk927Po2Te4G15frts87XnOhoz08zSp2eYZc49XDB\\\\\\\/HoYFcMyq8plU2kQzMA5kbft4hSGmhVppVGd9IjY8DsHWUPIlxximUA0glGjMXjhqub1fTphKUHUkbjuwtuZgCldRBRpSJFOZmszjh2q0T67\\\\\\\/PhEVWXyFwn3q3219cD3BLQl4YMu8dKiWy6Afv4Ga5hYmldyAwRZRkgJicy5lOsvDNtSYJ2OM9nxO7WlyN777HdFaD3izlradrfDYVWKZVuUUVNAhxv+s16h\\\\\\\/XIxo\\\\\\\/mKmej\\\u0022,\\\u0022iv\\\u0022:\\\u0022Iqn7Y56mVRR1gp0ISnG3yA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a94a15c13399a51fcaf1d4659409594d8066ce7139b7da209e71138c58cc4ae8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yW6lcbGX4R6ru1F1ubITUF7HvMFTMbxr2pdocShGBkbir6kFNMD3VRsDha3s2FeuBCX0cQyW6Vm1FNRoI1p8GN68Hv\\\\\\\/c\\\\\\\/tSyYNxfD4nd0tsICbGCVFmU25uBBCH7n9oYfWbPn+LQIUZMVRBQhqHolhweJytqIUitStgt3D1EwFc\\\\\\\/KF6wK4zHn6gwMSHcCSEA\\\\\\\/1ihfSXrY7q7gaCGLh\\\\\\\/xIUfJT8npT+PMLaU\\\\\\\/0pJVkdusmJO+iO8Rzd+UcSoGFJy1rzykxw3YSeuEPCrrkZulhQQsGTbEqJkVpLQGwlHizngBYSmU6w6koHL54o1sOjU21phdkF5O5pShy5LH+MyGvm7H+\\\\\\\/yuMhztOVNFiAUedVfPj7ZJrFGJNLWGdZBbyALcJH3+24kUVGruYQCYrTUOLkyY5J9hZyLnJv3LlOOnotuD8VgAv\\\\\\\/PZiozI1OADnP60w1U5OEXDLN4ZAAkNVlWd+bku3\\\\\\\/e\\\\\\\/2uBy7wpTsqmWxRoz3jyL8BNSvgW3PnzMvQTa9RWW4cgnYddlQ13mEfv2gvxMKWh3OFEKHCTD4emEUA9XqGU6iBNdOzPBMFfTrDLtPBsKUQI6i05F5T4kV8xq4+tpKNSA7jUeY+SiwpeyagJ9GK65DvOWtM9jh9cPdyjQIo7Fy1tONc3HHR7tsMdU4hpWHbM0CGPvPMPEv9HeJt+fs6wxRs\\\\\\\/kv8iGh9B5e8UZNFfw63\\\\\\\/4XIyjVHrSYFigBHUsxob2Am6BoKPMnoualzAfDNv4KWIrdi12DYlvnbZfvaW9BvKFfK2DUirlSeYlofjrZ+1nTYivbrHkd1JvAfrDO22dq5gez47WKJR9ZQmM6YQCTSjCeWXeLRkqFVOfTrd8qv7tFykETWdwR26XZfAzmX9YDi4YEkkJCcFzF01gKI73MtC\\\\\\\/+PqXUroYP1\\\\\\\/oXAD3ybNAFiNKaabTIM14x4IviONaYqxrCuvLB7Dm7zXB6NQGX5pwc\\\\\\\/dIaBVzJdsm\\\\\\\/bxLE9Rx3xlTyHHDHCAYhG5DrcHtZK\\\\\\\/KKJCRDfgO6zDiDGk3Nj6t1pAN5My505uCbgGDxhxuVydZ7er8HDWyphcJJlmIClEUO\\\\\\\/oPWigCRW1yYQQKaeeugQ53heBxn1LP92sg0glDrsq996Gg8skFMo1QvdltHqv\\\\\\\/UOhT6M4YzJl4buxqYnqanztOjaSOfAw+IT8hX7t7bagAuGERTQrUENFt3lIfUF7nYzs4NMAbPZmzOpcqsxBzGcVe3y\\\\\\\/7ty3uCj9T0YJDk0Xi2+HA1+R5xgjr5wb+YxZdP0GLO3ogNcCIdy9y+1NP4fdBnzH2rdDjtpZaltwVWLzMy\\\\\\\/ttNQrRJfQLLz6Ln0qCQtcCsfVxrFwbVLqlErCt4+et\\\\\\\/4on+p+GIDgQPfjZpSv+6e73xEftZJ\\\\\\\/y32kg7P4ZqhqvyH5sWZQjNT7s\\\\\\\/5\\\\\\\/7Wh9WuK7Tu5TinF5IWr0uZQ9VWjKBTbdsw+xTmcDToMIHokROoaLN\\\\\\\/I+ZFaLqhBWfYJsxPZdXet+XzFwvftScrdF4SHKojD2wKU21ik26PwpvQBg3Mm9g8ToqtjEiG+B5HInrkYHJyULbFEEfPa+9t5Wrjw+Ql2AY8kd5tqfRj25+gatgqGlUlB+l7LReroJTc8W1nzqsaNaRamgh4GMWG1wdejjGvhxYAX3uHXJ01p\\\\\\\/jNsO4wrsR5iPBsMjVsAEXAxuqwsvV5PH8M59FYTKEkzz8ruBCtT7\\\\\\\/WHCpROto+TkW3wJcZjOOq+CK0pqkQERNbH9oy4aY942I9afeZ1TO+n5R\\\\\\\/wixG4q7zqWADUqVertpDAdnDU14T0xNoPj2wEYwxy42UNftyyMRp+Adp\\\\\\\/uyix7dFhdezJtFqfCQO2Bw7KSV\\\\\\\/d\\\\\\\/tgYIb4RfCLqFeDTOj+OhYo2ZWNNnN9udYR3gacb5jbO7R57pb+xJsuq3sPnxm3YMHHQdWu92LMPLJRv1wodKftHYE8xPFUG3FvxTJSlF5dEtAGZhvrEwFUVKa6fMnll9WC0OOd2AHksgebXu81KVlJWTsA9hcRtwwX9zTZdZikLIqvuhsf7WZeU8bbuXMQuWm10cAYoBRSHzvVSIRZz565\\\\\\\/kGkk2TvO19sQ9c25fSdX2hPkbc2zvV5HpHL9Y\\\\\\\/BueZ6vtjF8kv916bvBfCjFfLbioDP2eKfUQ5xts1PVm7\\\\\\\/f+VqzkqQFrNN3gQUU5fpc168qLNZrXjTMk02Ed4Vgnom9Lve+o\\\\\\\/Q0hNCmSM7z9Fs9U9rvUmb0xXs5HY5Ts7I4AMiZfQzNbVqciEgkIJ96JYJLXyt5GUH0B1okvvhAp0TYE86gLdBo875BysfiZ9p8TpUtYH811GYC1x2WQ+bs4zNQn7Ef5QWbfMWuhqG1VqGqRY91zu9hBo54iwHCihMinaYLTTFfZp5zlOjYUQlLdE48JaGC7BVTKON+I7LzuTeZtefiUoRl3wxNhaJzWhV7jEKCmwc+v\\\\\\\/btzeIsol3t5GfwHg2Sim7iJTmNvAxreJn3RcrItGyL2jahvA4EyVk9vc7V+lOTOB\\\\\\\/7sVzUOzUXOTDzot7e9bnfNLj4P9eZ3tE1CLgfMOmoKGfBfHnXx8n3rgbtrhainC8zMN0whlYxQcBmJsIGmoN1zzPw5BwgQKQn8wZw5qzo+3oHyoLZyOwYb5BaQ3tFIyIk8Ylinmv8+7lak\\\\\\\/04FIGKIInEskPX3Xx6urXphxf5+2JCYws\\\\\\\/8\\\\\\\/66GeWc5MUEumruWraGpiAqQzQX\\\\\\\/USFkP3Bd79eszhslqAb4x8n4c5tBlVR3QFlFW0efQcLP+OEgrsZwHQlenHgGVTmAfsVyy+1pPjH\\\\\\\/9qVDpg+IVYZpk7ywfEorp+YbCBUllLjZLVS2fbr0vyUYxgi1h6fPFKM9S8MSPVo86g7u1bcFPSDl\\\\\\\/IBtTvTVJkRsz0ZB1beWVhLn5oTnTupX7W1QHXpPMowuqWZ8Alm0lodnKK5xjReEisIk2XpaoaMTuMSpgXSNMToar3dqW6H+fMibhDxW+X4iEO0y5fTVTI6ZumuckNnnp3dHb3GHbPNr+a4E2EJblc7b8j0CR5WZGpL78kg3sYLdOBVq+aqsuZfscJLFogD9ovtl5JjA7wzL1KD3wmEQ2CS2iyxI6XXKqCXvxPcCo4KEvqxnXvrOwdrscw2ZONKVtZYOvon736syxWJJIUFvDC3piP3EBdkbEfENxi0QMnh1RFjgU7k6dzFUiH6MaARx+pGXmf+MwrFwtwjwlssiHudiZZZihtSOKe2neMGhmlJ1F6SpyNr5W9C6NcvVIQjDeNevkKtUbPsfHRAh0A8sDKFCLwCYyVOUWwHRYfyobXNTIoK1FasqusN6hXMYcleGDQ0Pqu5mrJu2m3h4PXJv4sGyB33YbVrDVO7pBYPQ6Bs\\\\\\\/52RFfh09MLZPhJyt2SP41Rv3VbIs7oAqlUvf+itMNBcuGniQLEODREY2JNl+9\\\\\\\/huQi6FiOroYhqcL6lbN6ntjqVUyT+af8neZMP6r27i3EZNkI3PO3wAQhAJrTfrmTgdDNsg+i8\\\\\\\/jR6jon8FDhO9FsyMZuMsNtSS4wAKh4lB8TfxdanDWMrG8+NW3TE5e2oHJa35BvRuSTRlZxCSKgk2Oh4qYyu8p4672NJTp+4vbvkhnU3+k2luap79VnfB\\\\\\\/6xbRX3A6RNin86lscSuAcjWdEIpdWDoQUBZ7qVBq4NPcsNDmOo\\\\\\\/dADFaHoqCAx854E6qTuZgQXfSFmRGfKeRItcWb05thY6pi9CAOoPxnqMCe+P3\\\\\\\/ZU0R\\\\\\\/XexhfSPefBKolnNp0mktbIpR0iJMyi+KZ5P0rMUZRq9gpVy9lKjEgQMVuUrCYYx+Za\\\\\\\/kym+jEhGLeTPMoqopCOjynoPrQ0hCbNktD6aWPvwGBqgwKIX8c4y98flCVCj\\\\\\\/ZXMTajHgZOaF+sDaBngIC5yBTQ\\\\\\\/fb77n1wBORtHx5KxCxb9qPOKW8LFZba3pW7SHGHH90Lgo86My\\\\\\\/OFQj27ljwOW0C6xw4FSEiLB65xplzw4hNm8PSaxYELu4NZeITjgaR9R9DS37Y1z+BsR7QVopNSz6LqRpwuvdGwYeLAo88jW1UOvX2m\\\\\\\/gBBK+aZwOCvCsUtC6OCEPEwfmJBLYUkDns2u91sX\\\\\\\/5Y8E4HqXcn2faEMmJGUORvwYN5fRw56rVMjv5S7jXpjelUg0amgffxzZFm9vdGnM6prjurjZIPuBWTlXmgtVQ5cPaHwWp8WlbxYtFV+V6P70B6K+vRYgaFUw3xo0eUR9KAhy9Z\\\\\\\/l0cAb2+MgxNgzjkCG+Zz6FgiAfnHh6BzSVySnrO7B7syPnJmUou1k3hA17X0VcT61WK+XMzPnkLlvJNknABDQNkx1poCl1jTTRwMJduHrGgzRSX8DmvmGjV5RVo0\\\\\\\/scP5lfKcBWd\\\\\\\/SMt\\\\\\\/MeFovt+QznPJeuhUkjtuEzCiF631Y2uxUsBcXV+dz9mcUJRhmjVzax7SRWrr9+7zZm0CGriIz5EMcpe+pYTU\\\\\\\/R25HX1E606A21LLb1BKzIzG2JEPbJIVk70y7Brg7vPdyZoy3afmWQ4fsjO\\\\\\\/LDvY9+AWp2CGSgiAiTBXbRWI763FDV\\\\\\\/xWCAqoSNLwBKhMmwkPxSsSeFwXv98Ge3eBSTep2jw\\\\\\\/FuF7bOHjXeDsHowe0lQGMulI7BnAThjGKSPbw6FcG7VYMX\\\\\\\/JKjITnJxnlkwXXOJ3d\\\\\\\/oDI8+tti\\\\\\\/pu6G2gWUhIOccYLu8N54\\\\\\\/JV1IFKHofDyL+ibo\\\\\\\/JuxFyHXAjTvZQNw2ypfsGlEk7lTUtCFEhJM7NHOAnoCjqTmhWfo13+4SBgUhRQJfx4LT\\\\\\\/EdT762DV1JTIIBGOy5gHuEHIbljKHCKlikAb6tZHUcdd2qLdp2vYQkvtjFbq0WrTMR6ozVeBS9HZQWLIa1UwxLQUZ4cc43VuNM27REAzjz4OPvGg7qUiKnH+Bapgmt9HwbKZF41mzUuxlSNRgTZgX7GcSlcZ\\\\\\\/VgGL2DGjp2TYvPdgYpHDpvZHynB20HLXpkAA+c3gP9wjcVWXBldtUqUuBvEJGHtauh3Jg2wykHUuu2\\\\\\\/ZLK\\\\\\\/iRfPvSgkZw\\\\\\\/cRG91h7v9amsqyWhmdmL+ydJwSs3I7ClZEj8zzKsUb8QFFfr9ZpoUBTQsraLth0Qls1p1PxZ6HKs\\\\\\\/cYJYNhSvJ6T5ebU4dFxQYkZnTXM550upkTxyVweTncutell7vEx1q1P8bE8GyV+02soo49ICy3s1LaUDd6NCgy3KGuCL0ovfC8lq0iqarKdNj9A\\\\\\\/MtXbiQF7VCAxRXXYfsj3ryWK2sM+o2pJXoi09j9EUGy0otbOgRhIOSZbzq5tMP+R4ki1TcbqWAeMZe029gxC\\\\\\\/I8upqY1ASGclidAyVvEeayVePcbEMjhNqOyEYCmdg9kRVXsC6ERU77sJVv4xvtGji0uSwE3nF+mWfvbuaK58eHi3SW5zPLeGp9OgZQIq1FEDVVJkIWe9s+YzZWkvjqvpTOWbhrWthEhfzRM+lY84OHmahdM\\\\\\\/LK1FOcCi7rLglk\\\\\\\/RxMtA\\\\\\\/N\\\\\\\/Pvq6zuVmIrM9WmzDjyz+8yj+Pb1RYo+sH5FdtMi7e06UMtkWLqqXM1sv\\\\\\\/xP2i\\\\\\\/NLNAQwKeDPVJj\\\\\\\/VYq9zqFYITisKFRx3wuzOveUTa2Ij7jmRLEboYw8YNAnQ\\\\\\\/KFL92MNffm2JjmkrKWmhWvFlxUui9hSx96okgPy8UAA0LTvyz3EPS4fM45KUqZhG3ejRajRqXQxOvGegDZBgYlKClE22JBHabGXJRAKYPHlmnyd1v08gHqTpuKFdR5PVlFJ1W9vr5EeuEmmYwXfuo8JT1P\\\\\\\/qG2kVhyq2EdZHM\\\\\\\/ngF9SIpcZEHkdQvtodL83IpqcaaspOMUdQHN9sd7ULuyS2M4FgObwDdxMa4Tu+WrsEUxPBi6KE+tZOyh8vnpQTcITMF8LfnSNeQ+ULRA7o\\\\\\\/ZYYgCCyL6BK\\\\\\\/YLbixrLKwvXi4r\\\\\\\/FGlO90C2R5xbqOz\\\\\\\/nO73z4ZOrD8GxFwqya6UB49\\\\\\\/Ri8feLRAWSCO3dVbmzXDfWTRiujngJpVmOuUHbKnqsYaHLW4qBr\\\\\\\/LvpSwzkN7NsWHFpL5bYFDmF7WE8Fq\\\\\\\/SDkcbe21DeCGZ7Jic8cFKYibYLK4kbNDfAuN2Y1Lfm9sp0XERZfGX5CIxUgLkzxMD5RLIex4mzS4lRP5jsCKnJ+6TACUMfaS+NNB2Rxa0U\\\\\\\/9i4CFy+oE8DwL01vs\\\\\\\/9x1uFj98damDTXgKGzWHdpnZTEHzsJR3q4MWrAn7m54Y8UEzSHoVDWCyCoMVOEBZzYmpC2J0nQC2nerpVrBAg55zh6gyGp0WeRlSW2Svk32poymFErY52Xgf7MfiaGR3+JvmMVE3OelVwRG20AyXkBnnLuVr0fP\\\\\\\/Hg6pK4Ia3Gqe6Im9ctdEjVhx3GCY+JWrofmbEFIBUv5Nmf7Il7n6qLnhVIeRAinkLbynBU88YpUKa6MdKdg33pq1lztika4WtXIJ6QpxGoCjOg2LjlMDZBg5zKBRNh9W\\\\\\\/iu1\\\\\\\/yaCUiCye2RSOgjczH7EuwSb\\\\\\\/g\\\\\\\/4\\\\\\\/onCsnLrGM3I2FUJ5W4YYA4e7+SUPPJkqKLy3Qav3IsIdVG1kQAdrgXGry2fcC4aGhtxku0FrDW35tWskwSI0iFlWO1xVtvTpApRJELc+u5F60tLfGvoxDK4NM6GUTSAoxfU\\\\\\\/TttvydDEJUO7lGnJk9lwmoUVwnVEVFZLtR8rVkx+3pu5TkD8VWPfEhBC4WSEIST3D+U3zlXDa3EUETHMJIAdO+PLlT806bUPF1pNVTFxrDSnWFGNg\\\\\\\/iwTOHaDm6MH6xgzV\\\\\\\/LGI7ujeGC8K9+\\\\\\\/5F\\\\\\\/svBcdb0KdUhiLee\\\\\\\/+xksXmFkAS6U0JBgo87uSIazDUqybFxo05TizUoq9RD5s43Vbae8W0L2rVw\\\\\\\/Hqv5W2YKmW0pmLNQJsnDqWIuLzmM7BBOOcGSRogtpHw0d80QHKQC58kRybwa3MfqhO1ttCRJc3+1B\\\\\\\/FQhoFuY\\\\\\\/SDBSvcXQ+fNntqP9\\\\\\\/XxjNdRXl6GrLKE8NMXQ8v1gdQ8f55jE4GtwsLdupZEmT4iP6FGi4dK2Hp\\\\\\\/6VHGbf0fC9BF5C\\\\\\\/dh8056DRVs4\\\\\\\/H26sb8xxo88zFyX9Qe5pD9FTbPPNl0gzjtoMVlXedO2OhxblaH6fYZEKxz+hXcFdP+XqPQ3nmFkgYipmThq7IQ8bpNleI2sSvKSEBDXLkZuz0KgXu2SzEIfx18vJ2J9Sy8QcBjF03xW4d\\\\\\\/h9qMCMv5Lm9R2TKLJ7wFJcdj3\\\\\\\/FKWULYzOVN8j+MwU72YU3Qt64etoOpqHd1qBZZ+Bba+GY53Efmg2OUXhJSo0wRPBUPi+7BoQCsV7CXbROBvtBbl6hIBrfmVVDFRud+FA8sdqzotrT4\\\\\\\/Si46+fNGRp55Hfn4DQkyFy43ZphxB4tGkx2vS4I2FdalEKb\\\\\\\/L1PBo85UqMe8kjc+DGOyqnheh2nNlRJ7Dh2tgBCMmBDV7\\\\\\\/\\\\\\\/EL6VMO2TWk4J2Cv8D9zgBcX+zqpH0focKM9j\\\\\\\/aBaMK\\\\\\\/8ZdGQsBocJLZLSD54ep3hC0TEDcH89NbFQmyOgJxj+OoGdzZ3C13E1VdlTk3T9E4\\\\\\\/dVPmcGGbRBKXXCFq9HUSkQNk14scFDqJDJFmNWPOCVGQrxojxjoJ84kQe2iHH06j04TTym7AazG6w3Xs6GQXd0LEuh2ZNlQdG+C9qXHld9MBLwWH6oqDo\\\\\\\/MrJ24VOMgo6U2VTU1CMcnJa6n71w5A+d8cJ1IUorYjhhHRbAKkn6gSdh4Z\\\\\\\/BQ7SeUgMOgsREifXErvmwy3ElQbyPdcJUMeso4FC6h6M+SFKIDx2rM6BxCeOj+UsZyTtNem5WUW4\\\\\\\/rvKR3JGT7FWUljLCXpzaaViZhUSzslZ\\\\\\\/a0nEFpQvS0YiZwdqkFGe0DZSpNcVh6ayiIqnnG1Zy49vt5r4R8GBiTplChCdbtCOxOjsMELOsOWmD+bH+PGrHJnegWU3OD+JNdBmIKSpmPLE5EwPMcBsKj5FXqU2eJPfTCmm0Mk9BpR4dpMkD+I0vLvXVNfvib7UaUw3rVyJs67xFQAsndrF1MH1BCuXTJhMd3lgtffDhYrln24b1OdcfwUzsy+HhcoDJOLUq05h8nMlRi15FIBXGJyDFtiAR7V5WzfNJzRprayrXb4MXYsV7buigq9LuYTW03QOwR3VKPYqiQ6PhTQw52A7iFacrOmuOPTICMQwVZqUtdDfMCszTH3dylsIxfhA7lSUmJfYztsuic5CTeutTxASFG99cSn91\\\\\\\/+udQySar1arUr4js0AWQc99viaEwKnf0b\\\\\\\/KNdARn2RmgHOlFMcg3ktdPzqYGmjMOAGxjuQeIEy36J0d9w+WKhl\\\\\\\/UPZIJcBAXGFZcurwBZtkXh0ukEkmVTDuDE0XLcQhm4qp2QNuqEuEvjK1Hffrd1rnsp4nWIQMBqn1BXWfcOOn9Njai5vSXZvvX4ngDnzyg8impYogijUiGCXhSDebUeKkHlHpGcTDv90fLlFcOeXuwDAajoaDpYJ5cxyO8X83+omBth5ZtROUQ9SENHkX9HlVpx1D2r1L9KxSlwohpu6EkWRFoCU5WpoJgF\\\\\\\/ZX9XejYUfYcR+\\\\\\\/vfn3ThsI5NBwsRDbxdkz262McmifZQC8wLKmxoGJT82nmYPATOxTL5n7bxV1KvxLofCCEip887wKGSP018OEi\\\\\\\/Q57Hy5bekruUBtZkves6a1vG2a+Sg9UaeeRiHSRQ8\\\\\\\/+Z9L\\\\\\\/o25\\\\\\\/LxRL4nD+OErk5kVIhFYZ6lL6YBqpBhidwB0R5WK+4Dk5FS1DGzsqLm9BNaDddRFlFw3nSe4p53C+eBy5N8g2yP8aNaWHFwfBw4C6czmcJQFXZVNaAk7907IMwx\\\\\\\/hmDqgeBUhFFG9lk3nMpY0gwqAJYUl0CM6h4k4sUGAmlUyBVCxD8TIyQIalydYvXeLte3yz+Xd9pIcPSKafOHLvbj2fD9NScukMvByFrZnnx9FzvDJQH1kE+7s4Y+Hz5+XnP2nDZ2x7beAOILvH\\\\\\\/ieaH4fYJR4eIiLsAK7lhCQd8g6wIk\\\\\\\/ol+\\\\\\\/UoJV4ThOPyh8m+djaj\\\\\\\/6WTJzxeX+Sy79rkml0Ct3\\\\\\\/O95jd0aU2H97tflVpkNfS6vzLxip6mIDkG0ykxiK1bpuqvQg4+\\\\\\\/9KtYl\\\\\\\/dPgZnGE8tn5vuD6+5aqG\\\\\\\/lZUy256H3anfeHcxZawu+EIxkAnQ+5d7K2vPZIYwo6bWYtpOztWMbQOz0yQpcwrS15msYqLBO5McGjZm6gyyvwsRRCdkaNY1akX2LBRk4wPquQs8yeHrsIr5i1c88ZlP2ezh6C6lbRz1rlz0\\\\\\\/bvHWtHCztbf7R0K3Sspj3OtPnKfFzkDlUvxkRRoNI70a9ZoivMvNvqHX\\\\\\\/iWBzZ1XysECMkR2OtOgUNtGyW\\\\\\\/0AWKmCAgC9OnyWOkrYFndi1OMBUlz6ObnWsUKQS+litkMqw3tanhoQArnECngF754zGvKtbvQcFO4G9+oLkgE\\\\\\\/Eb+ekgaE3XL8LF\\\\\\\/+YOoEyXlq+KLS8\\\\\\\/DIfQFN3Tgc9tttcyla42ZhCkSGuWxOFLQeLxa\\\\\\\/aPq\\\\\\\/sqRyIe0+soA28CtNRAEeVO7TSTLdsHQ3xZc+VtivSjq4APoAX3FrW24B1GwhzSNr1mri4YpOyQz6WthYzYEXVBJmzMci4Xm9wAWabJqvHyrlEcUu37i3Pq\\\\\\\/DqpTwHHpBTcP7Lfp9rzNywj7jB+Wz8gRBb6xC\\\\\\\/BsOG+tuyb2hxJ+h5NJiXbJPTwcpja94+qBTeYQwJ4zhz6dyx9ENuiHeKnIRXtecEAPibjWIBQHmKs\\\\\\\/+K8XisQAKtAQm521wTzH+zDSF+er\\\\\\\/77FPgfFilduoQRsFCHjip849SxbDCPu3EjdfcoXbgsBgnOOmecDnWKZqeMHVvZvU2dK4\\\\\\\/Xr0aFsfX6VqgxquObDHFsE\\\\\\\/\\\\\\\/OB6Et6ce0j3HR3ro01lINEcPv1ftpO1ZXY\\\\\\\/02n\\\\\\\/yXNCc4CK68mMwpbInElVKqgOAY9ooh2w09b3orl+XtVRNv1jrZJvZhTIqJRtkuC329qghYHQGPjcAfKyGf6BefMEMc3mvT1WUsi+6ZgmOBoIVd8i+gdJ3wNUPh+stu6sCw\\\\\\\/Sbw0wzrFJBM19leexDohDMXRqnnO3Lr5VJrSAe\\\\\\\/MUM8gI5+daw+QrCfsT3y3pYBcCKc0mg\\\\\\\/xLup1yjG4YNoMlOFolmtI20ywtrzyN+aiIHI27IZhYEGXw1cmRKKsu6bGKOmqN7KEHmG2vJw3hA44EwIl8m8WxnyYJU8T4C\\\\\\\/r0YoGcxe3e46BIw6p5A\\\\\\\/l9cu8BI3J4SzvUpVj3ZSBxVw5k0sdR8z\\\\\\\/vWw+xYnca3Po08Gh2Y38n30yborm0lLX02dBsFUpPoL9Yp8bGcoWqCALJyDzVC77HWsyuoNyNS+6AFOgFJkdlI0GC5FYsgXmow90xp+3pajYZIXI0HWjFzm2gfDd8qmMt7mARZ5LHib40uSvlQ+U7aHmsS+l3LX1EOU8AZJYoNZexe5+j8F06TmCxvh8Baxy5+RTztnwLd3KpkBC8vdnlvtMjMZjLPi1SiZM02F\\\\\\\/3xVSMCT2wNkiUKK5\\\\\\\/lNVzCW8fj2o9ZgHzg0CNlGGX3ODNlo+yk6gvJujBWSC4NhnNGsqxgzYWef8w+WfG7qLTLMe92OlkuYtKiR1NHlranQpQXkMz7Aw1LvXjsbHS08aQiTMGEcJhlJV0RBCMnPqgxmrS5aYifPDqhqkufqoic4stFY4C9BygHh0dR6qj4qPzjwXQNTPYcTG8aR+E1EBPMwmQh2JGfx0wNrWDuV3VkqELB2yVEHQ8JOp5Kc9fwrpRnFuv8MHp8D6b1FRL1YkWvctIcpjrd2yhgkmUBpbbgWo7ejA0QZH5cj\\\\\\\/Xe\\\\\\\/qpH42uSLMmmDVRMAgLws9lTXlRpxsNFRDyYaLBfP+fWsWXVL\\\\\\\/9tW8tBqaX8LDdD\\\\\\\/IIYRY\\\\\\\/lrNICrtefqSryohHQHwArt78yty4xF5sYNGoH5HarXmtvZjDsVNAnazmWO57cyMb1dNFKpJukV5\\\\\\\/WflM6RmB+b8zuWAxF4oRTE7QRzxAzEGRFN2qf4nZ92\\\\\\\/8uV1829Kkbeo\\\\\\\/r4te2bymY4kYMCqQk3GkfPFmMYg0hs1GX+PZn9BNeixAGejJsdkUSEtvmbRI4rA0bkv6xMqC0iAvfsxFhCP47pxMgBNhLP2G9X4DcP8MVpefRzi89MChCl2x4jIxdXzu\\\\\\\/3KFHYJKGUt2OWWUQ9PwebzuXSg\\\\\\\/yesAhAK6L+lGD\\\\\\\/5TkETdw5M4DEU9pVrKUr6saqKdttTfDawo76OYJD78btXGO96YqcZ5upwcPDXGWx7iPIW61\\\\\\\/3c6W\\\\\\\/F40cS7Lq92s1e1yyywNb1ZB5ZnLDdTmatxBTcaj\\\\\\\/I695w\\\\\\\/begU\\\\\\\/PybvJjYerTJRRq55FByUMedTwFnkbI3dcyLMz9CyL1Yiv7x\\\\\\\/lpXmIcQHejr6Es2UHybEGquVi7wSp1mZrjJVzaQmHXGSBmQvBfbyrpuXG9S4mJhy97\\\\\\\/iCy3519YhPr0OcuQBn7GQfeemXzP9GyirxKa094n\\\\\\\/\\\\\\\/G9TaxqTH5IBn4TmhMH4fg51vlC87+PmTK6EVh7utFnTMj7tsjmw2D0WkQAzRpwpQS99gPCe+8VnJHHP0l7j\\\\\\\/wllBU8EeLxhTIOb8u+MO5yp\\\\\\\/l\\\\\\\/RPhjhjEqNPINvF1u0USyHGgCskgD998qsP6vzHLeCMJ419rgdROFMvnAMu7tCk4FDS+Ih8qNGvW50BW\\\\\\\/qr5\\\\\\\/wI1KJQAg7IfuU7+PteJOCEFNJ\\\\\\\/UGO6ab3+VLRl3ew9UiAOddJLQfOKe9+Gb9q0Xay363qyhiY\\\\\\\/6sfO+o+dLTPitfwJMu742m+A6jYy4hcqLKRmV3gEUGH7KUCyDVSQQj6YI59zfyBDeFsyMEQjRdFZagn80vJ\\\\\\\/lYNTL0dW7iEmpv7IaXsfy7TbhTK88haCIiIIcIAsqGaKg3FtmIQ5+O\\\\\\\/Qf8\\\\\\\/cJ1JqKTsXaR1Kyqc9fQx36rnEmuDxDnAdU1KOyOHIxArEVHCIiW5nt18YvXNs55fwfA9Llf0YNLjtve8tQ3KjQAkTCHDbZsi9kpv7VIEA2BHgCZIpql03+jgiwOnfnC0ZSZDwIgLnGrXSZ7jpiZVEFZ06lxNDsW77i\\\\\\\/TFOjMVmwrB5gwuqDM3pp9rM4fpCetZeGxwczOgVgeHH3oNi2DBMDxi\\\\\\\/QjhWF5afVS\\\\\\\/Ulo2Yj7\\\\\\\/qPMo3vO0S6Myk5F\\\\\\\/u6P3yNOL7G6kW5w6Bua5tsjWppHJX0KfTM5hMAT9auSf7A55tSlhtzdaGKk0K59gk54ZLaeJ4BGdPnewlnwLtqbSWsXgK1U3kq0VOHr2BDs+k6TRAJm7exhfLkfH3XkJqomP+i4E3umPXPZhOvj+rQFmYqbr2ua0nA2Ko97koDyi4fGRgpvk1nO0CDBH3vP6ApiWueMhrtlYN9\\\\\\\/kHyPghcUB2jCY8V4Atp8rK+yxN\\\\\\\/+O3cpf6fWQ13P9RSBgJQuxfCHNk\\\\\\\/eoGBOTQaB3\\\\\\\/FU7TKd9FZlkqNpumZnOVqNWOECQP89tjPIRM+71+nGlVKwG+vVuz4mMwWzgI9dS09k1cd6ZQnuH1F4bm62ozkG7RsQAnr6KAcrpuri7JrFotcFVP\\\\\\\/K483whW+OqAqMSP2wPnHmocYm1a+q2vsgGbrdAses4u0kb8EeYOIyz5vTzYBJomdAAYVObX1+dwisxRNl+2mTUAxPvuA+sseFPgtAt6FJ+5nG9ovNHbjgW28I+oWrQa+ehEinNSWeMZcM42aejb1BcjxaFXmGZ8OXVpiFNKeSm0WtDkQRusiQUY56yVHPVC5jzHzaxVNWsAAiCOpEtY4MVsQAoRN0FwaA4D\\\\\\\/MWZqMnF8P63M9WeQ\\\\\\\/S8AvjIzwpsl6SZ9N8qLz9lLO05bJBIb3EE\\\\\\\/c51yi0h1RBl2nMRiW1vW5tb3IyS6o7wiYGkQOHXVIFK0PLkCoo7mPbVXjgKO+8ifCMhm1eiapGFDuAzuRB3\\\\\\\/AcGTkcnhvkz8sXJCWEXrueRwuv4e3KE1nBA8bjw4rc6RPMNIVqXyYQj3eUWTb6SRzxcpYTJtULZ8HULEnM6zDwtruzM9swQKq1jOD5LNwSwguBGEzpy4g8uGMDJCsYXp+5zCfW53f6AlFCdJlEIC+BmUV07AHtrxDWJ4UtukSLacKpeIkGNn0\\\\\\\/UZeX6Cd\\\\\\\/ay0uDfayiXrVjRk\\\\\\\/bt90RMtL9UTKa5t9vcLEyFDMzKXx55NvrVPs6dpK9ztzpDyKFV4+4C8FAcmUZP\\\\\\\/htNMEN\\\\\\\/CkY\\\\\\\/zyBj4t576U6NXo5GGtBZ0Bq8shAWIYubG3bv+J7hKQoJ0h4ASfNXp228HMqfRWcXZRQ6Gq9Sg\\\\\\\/kFdIxjZqcZrNJg61AFOGZ\\\\\\\/68iWMrOYTgag8d2WOVrDux6\\\\\\\/hwdthdNBAc81YM6KerKVmpglqtM5ROjZAwtoc+i3iF7eSvoPIqp9VcxNzN9LyRyPe20fA4PzTxPcvw3UO7BcA5DR5hr\\\\\\\/2+V1HufLqW5wtOhx6x4NBUKAzWHiraeEW3AQPzWDv3xJn6mQe8bTpCYYPPuh3hOZ9NkZxgOUCwmsDFI+uSBsm41IparI+aurBk54Kzfpgc\\\\\\\/MA3UvVx3YApM+rFY71UodF5vOc10kACirTtaKYajNP+dwH1AQeKMcS8fROvloNZGUrSZFeq1MtpeGTM9hq6ydjEwH6y9sYxEduazQLvqBVDXC4J2Vmzg3SAUapvd2hJsWug03IW1IhUb8odYzUrcRRPLqwfvA9OvTLpA6+3DYO1Jh0UpxhB0iIuDerpRBSgSVKEL8P+qc9juaYwVrFBBqxA2yj45w5K4IKyhgCJqaLo7oPxTmRpef+1ub7EQCTKlNcAK\\\\\\\/LQGq9rp\\\\\\\/9yt0S8qaXyfNxQp\\\\\\\/X+pjvYMBKCabtcFelJuefuZ2MJ\\\\\\\/NYNKMhJ4wGarC\\\\\\\/PzD5mOcIWmspFb2EBbWEaqLKivgbWUkbnG7SKD8M+Dehn9iwY5Lh725U7E8UliWVzFnb8mENHsJfEzEXHlF1T\\\\\\\/6hluF15KdNb8en0trIl0O8Qg6h9X\\\\\\\/cpVMeiXSZkS6c27FkB0pdqCEz1DR4kpnM+hK8sER7LfMHnrdiorbRCn\\\\\\\/kKm4Dn9mwA+rVFivj1StGqRWFev2iHopHhwAZrMFnMLfFVSeTiGu8tInz+lPcrtzGh6nGCaopPnQlXqL7MnlJ\\\\\\\/lDpSmdUCVAGmX5Gc7ehESkle\\\\\\\/WsijyUTXvDk9GHfRWpJrb4vS3BVFGV1\\\\\\\/lxkV9M7jhclHLsmXEhEPVjLj2p9GZNwwYv3xSu7eWgjvnJ2rt2WeV2Ow2Q41rvfPpYFXgz797gfk6rFiXQHGqfQF2ghYe5RFhyhHixe9k41CYLtp5FWP2WpDDTH21c+dK1Bua5desYM1oTcFXgi7Y9FWwE2mV4CSk9d3FqxSjnYT5Paz2wuBzIU6I4BgHNnLwqTORGSY7jlzJioRng3CyzL9oHl5ZWSyHK+jQbBIufZL\\\\\\\/g3ZdM+Lvty0XB\\\\\\\/+q1x8xsqwVOuVMU6JnM6p6cjnLVNt61ssj1ypSAc1+2gj\\\\\\\/1Ygbz6cmUa4qQ\\\\\\\/ia80T1hdeOSdbzny\\\\\\\/vaGtQMBDhxIRRcR2Jp38t4Y5qOrBzHSrgZfqNnArE8vjqIKqZ8A\\\\\\\/DB4yyZd9xKlvYVwwghwDjwDJ0+LEMijHeNpldw0tAzYQXV5GN62QEQSgduZ1VDrr\\\\\\\/IWusq5JdrP7bmfPgtwEfYVHSv0JP6jDUw79eF21fc9ZSaT1S3XMzD61onhDXeUiBLG\\\\\\\/rMeVRypFN7NzJJsDhrB8uVJIyms\\\\\\\/Lb2WKrAWfmG1xzZop43W6zpf8hp\\\\\\\/oPFgfr0y6eZ21OJ4fTk8WJLO\\\\\\\/zqLAIuA51sGFZ28l6xUSJOOtMOXxLoMHnrWfKAYvSLYotMoPEwHZRN1TCdrsKyQSClq8iByMP4Cw1fgfUeeGuwHPLAP\\\\\\\/WTNcXZ908\\\\\\\/q3o1hccdcFl1Z82mL79Fulbmyxqky3Xvj9nFB23FOW1XpCP1EmJU7mY57iq8hBOC0Y2KH8Z1RBkJczKUfEv5SlUwocDsgZQ4UKTlWQ2T8zwMfprE50rIRfN+nxb64SeuY4j2\\\\\\\/sk2Z3KhadTZh8k0hepdBIz5n8Mw+9AYYnyJCYgtVeW6ma\\\\\\\/icz5gJqjinveL1+WUzwTa7VCdUyaOQaTzbPsa\\\\\\\/\\\\\\\/qHzOht5XdQXY3FdTfFmCo2FjMRfLoQO5QlTk6f5lCr1gPVIYdHyniBYLWAEZlFMvu9z4CAv2ZViFIqer5br6s0HGnVi3klIaW1tnZLv3fHV4jjPsupUjwI7Jz46KClGvMHMy1GZV0\\\\\\\/vaPCt9mYEZCzejjKPMjZnCbkpmZIHqdA+GOWJ887\\\\\\\/ksHmtlnis07YwW97Tiy8d7Y\\\\\\\/veKQf+tUjsSqWyARbop+8jHWV3Iqk4IBZVHN8JVvaFWFXNygF5OQRQ+SaO9HcGA97CwXigfkoFrKbY30+BW2205wlJfedMAJpMs5RjBcqdMSYH+NJXM9mPjMPPslAjgJ+Eb7Agy1tXc04brGT9NPslEeLpk2kjzX50NE+uui+VCKNZ6EqSNU04TrEGhONUKPLvicr7nE12jXbHRkyNlHKw7pfyJhxgXxLvdIJU+mFnmSN\\\\\\\/hjXMGgvs1Pit8OoLxNwwOx\\\\\\\/w+4w5aODFxmvJclnZBqrFGYU4ctc\\\\\\\/+VNgk3l+BlAipbejDJgaRq6kX+39rbg4X15lK6JQz1SdvogvWsibCx8BrP60+7hklIU+Gb9vQYw1oXDCUUfZPqQIyJLtot5Pm\\\\\\\/iuogO7Rpvw4FtzqTCxKQcyx+k5\\\\\\\/ccaonJAv4V9YnFDkoyKB5YUDF+EmXSiWt8K1NQ6AQssZ\\\\\\\/fBs1drFEWNtfm2Ec0OupYxdGWXjYAcq3gJWGCn8JuhLJNd3OAMlOG5bT+TGMogjkpUGpBGWae192ezM0RIvjryhF4iclrLhSryLlbAQ4vssVp7SPo5L5\\\\\\\/Dckt6OsGjW06kAKy9BUhHE01cIO0E4JaMr9a1zVRAE37f+GBul4fZQ42sH+XSlbJx7pfNMuCDHpHMHmtD83SyN0u65ym9ZfqgysyFYckKfNBBanT5pjS9ao+8EtHCVRusAuv0pRcL0CEy6XeafV0mP7BHrbhvVGXZm5gZqKvoO5J898dRAypOC87RvTHV4HD1TSPz8OWPyt3\\\\\\\/mPju6lKjQiF+2ynVgWoANaKKy2VfZI369p1gSzlw\\\\\\\/e4723OlNQnEoIUHLAjfgdT+oYBLfLPRkpZyqHtiwdVdlw8UvadkuggwKz2LEh9cDrQYfveBMss6pwXg3wUSIQCuZSkFYGwVbBBQIickWHCwhZY0Gs6RcYJPiD1Q1bsjhF2Y\\\\\\\/Mmy7lwIcFk1EykAZQf0yJ6jT8w71\\\\\\\/DfwCZV0tscdn4CnHQaRgQRJf70hCdI1tRRA6Op1HuOsuyqImz8mwivdiDpukdP5aYRiUyZOE0Zah1EUsTzxWtzRFcHG0AkLV+mA9r9uu5iwXdsURIQ09mm3gnoIrQoC8SDnHTA+hpI\\\\\\\/iJhNZce1C4mCcDKR\\\\\\\/Fcwa\\\\\\\/ON9Kxhz+2IC81vnTou3JMBEH3ZLQbhWo11Dsf5tv7SGx2QXyjJepu74kUAYB6VSCNkqnFtY75hhkbilIJYjzmqVEkwZRRHLag6+yaoX\\\\\\\/+\\\\\\\/XRjUtmQ+8n0cL9\\\\\\\/N6sW2iFltjRhNzOVc1rlNS+vxA3i8CeDBs3p9JE3qTB5e1NEhxFpwVU6AXgiBd2J1w9ww\\\\\\\/jAcT+mJykvrNTx+QHR6QZ9sa\\\\\\\/EB4UfhHBdq6OlVEvRJZHdyaiASJeEhX13e7mm4+FqdFKyjZragqQC+6ExBGnDP7bR1rVJ4qlfwL+kDjyHZ2IAJPlvesSgB9W51HO8wnRKSkvVhx5qRQH\\\\\\\/vQTpC9dpnchtls+\\\\\\\/9kClt7kuMjsvYA2gXEjH\\\\\\\/kZUYQtjzBiY8MHE+mccuAk7GSBwZIDD2SKwbNAAr5MzTR\\\\\\\/yKbEhBt5DcMmPscLoKEl95esILgQYRw8b\\\\\\\/7kxg7KTL5nVi7uc98VbG2EOWlTHnOexVHyh5cynKHkyFTTBM1Pfuz5iRosSL4mNh6fkhOlo4We+Nk\\\\\\\/P7X3\\\\\\\/8KVzWiv3N7PiBT\\\\\\\/SiT7iWnYpf5C0\\\\\\\/0bEdfZBCeeKQiqiIo2IR4wv4\\\\\\\/DtTbYz6oDUSlrLXGghhu0YwQdMHRhyinzlYOMLpQtKOV\\\\\\\/QppPtBTP1K0DsAyCl0K+h4gdHFgwfoZpsR4gJw9KiKSKyVaCcJDMqWS+YaAHVNiMN9OT0WGFiLmSlYZ2hYYy5gzCvLk3LXaGJ1IoEu39zZ9kA88LCng02fU70+ZIfJHJ\\\\\\\/Dcp4zYzhlXSV9pwI9OBpuaJ6WOxK\\\\\\\/8\\\\\\\/d2gShUPabMemKkuDVj6DtJv7OaOthzvT\\\\\\\/9o+eUCNAMflwvlgYmmzGzsl\\\\\\\/T8YdD5jxR\\\\\\\/pUXjeyhZnOJ1CxZZhuFD3Ycvt46hopz6zuWXIUQLdA+TtjivCwsPcw\\\\\\\/jV9Az\\\\\\\/JYa1lbEtjGXjp+8J3980q6WS3uSpUcfrYLVEFXakKvPdpEY\\\\\\\/lXieqc8UIerB+lLau+FybKxM9TMGzMyi\\\\\\\/FGdp2LmG6ZyVBVX\\\\\\\/1XNXn8lsB0E87xOddd+4oC2R9q7Jlye80jm3KHp85\\\\\\\/Q7JYfyveHKH+V51VC9ykQ50DbeuS6amk3m4wvrbWXbZh4TR0PuWZt+w04QE769ZkwKItm6\\\\\\\/2tASAqr+HLDo4OO4a8IDOeVpQZEIMuEIk\\\\\\\/CI5XSf3OaMxBUig9nR0VO1+S7EdDNceZytDsTIDSNVuVH3hdW+ka2DWQqSKNn71vKhi2FhR8bGU+0\\\\\\\/BcGp9phnBxpMhZ3kvjlPhzOJXOY6M3UUKnaX\\\\\\\/p62kRPjIvIkAuLcNFuGJzdVgmQl\\\\\\\/rNa8iU+uk31I5OZ4RYM2CRYX6i5aKb1MTMu0epsuO\\\\\\\/GdKO6VhOa3xOLuRTp3PEUemCYS8Imoe3u11ilVU07zMPmWCUnhTx2kQrnisYpE7q6UOVSSX+9eAOVYFZ21wtmpSLl5AqcEuWzDmQ1FA5noN9LNXP7rlhMX9B27yessFD8eXdeGeVRpfZGchpaeNQeTlx3YDHCSEKtrV6ZbMC7mqDlEK2AA+LFIvGs\\\\\\\/SG\\\\\\\/pPdP568S2Tqavu2Z0ihe+hRzOAqir2v8a3oXtd\\\\\\\/BQf2JwpUdtZy9rFSXregPTajIof+EtbHBhRe49Q2hNJQ3brC8Bk6SOwphsnM04XHnbTEyJvbOeAfa2gJLICq6hEvNAQG6A8sr9mMGjGoenAVpzK8plTnXdvy\\\\\\\/JTKmdiRZCvn+m63NZPSQoi2dMEbuxuE3bxKiEFbk64SJUDaUjSDLIoF2dO1rm5IzCpCmFU4PH+WesE\\\\\\\/78QQfShm40+ZTpzNy4vZWnugcLvi4knpjWuLGWlvJdk1Qeu2goa4neFM1WxgTjgRRB8wgQStWsdJ9iWP4n7wdeBLh5\\\\\\\/3IiHn\\\\\\\/ATfUhM0sXUfIMnFInVj\\\\\\\/3NHM8TQ1RF73CLUjmj0qym0uYC7eoDoAB4CFDwNFt8YP8tH8hkMNKFnQ1vL21MhqFRhduS7NapY5tNOmtjkoWs+w8gPWtIp\\\\\\\/n7DTgczyq4WCYl+aqKcjFz7rjyGSQvAb5Z3X8+nhMHb6tdDFZyqUS7tKHhMkNo8s4HNz9OM58iwdURL+mNoNmOztxFuz2pmy19HGnTFHB7tRw1gYsv\\\\\\\/DBveX7WOzACtzED1kyVnvNag33kAGY4\\\\\\\/4E5Pzngdd6Gzg4jzLpEHqZoW08Fct63EBCwMa9XB68Rn6HgALuY4BlGQHgEEl8tC8d2GYXfK50Ngn6l5zInrJwDGNdFKfpHg2DMF7u2ichL1Yb2aeRWGObCrFVUKgmpR6azy1FP1ZSaNBeon7Yu9jE4gtJT+WsANnEk9FuHtXZy0E40wvUS2GcuvLuM8RTr5iy+tIPNnWpkYlysNwJ4jNCSaJizSgkQJ3fc6JMmHZEcWT4utq96Ppf66FQ7eXG\\\\\\\/iok9rXeL5kw6N5mOwEloQZhCfbyg2BwNcje9ZOVyWUW26JQ3LGFtRhDAjxyM8e2xrzw4246rLM2w13DutodXIO0Z4nBFlRr9kGN3gwzFlpCI\\\\\\\/TAvTesXDqhpe5jNphroeHCO+Csdb6WhisphXGsBdA6xjLAJMED14ExWkuJSi6vqjZJy1BIqNz3Tzdn\\\\\\\/EE7JN4foJFQU4sL7F4oqRU3jGIqf0vqhzVe2znmo6XKxt6wsa\\\\\\\/ipG8Gx+CuZfsqwPtXg3xfmHpI2gS6Vw3IDYTQGYegaXrurl8749v9jtu2JLoi9h0bl1BCi145yOnmTb2bJIEJL+fb+5FEjtRjT7T+FFPHePJ0jpnde5NWbj5Ai50HWk2qzhO1t3NBGgwMeumFyqqTCJVcclNt5XUJb2Yj8yJ7ELv57aPL1jJIhXDph9E5EE9zGtfNWeQZEqq2vrAa90rp0akDQaDtwLb\\\\\\\/EGPigV8zZRryGQqm6DCVbUhWIJPs0CpKlFE5T5WaO6H1dL8BhQMjsS2sRWGxtoAnKaOkI3DYmLy+26NFqpIlYX\\\\\\\/CanXnzQmBBXE8Q8UC+JwLaw6B6AMDiyorpCnOt53QYQmnxldlFTRgalaUcowITlTU7oUyYwXrjqGN0w9sGptu0OR+\\\\\\\/KWwEqntQqVaNRKM6Jn8Y5B9+y4uVEUtGd0lfXuVI6nMfmZeXmNNszct1FRThmf+KfGkqdt9KvSjkfspHdlwt18imVZaDoH3rL\\\\\\\/RnJvT0h5DZXchXTaRGWSzEWkZFbih00LF0s6FvDZEd7eTT7UhElzTgIJtwqCOQ5IBummB\\\\\\\/VX5X+Dc\\\\\\\/XrZI1SDqBzsWGlMd4XDUwrluKvNrwhz+D5+9KkHDzcXoQt0nYl4nkQ20iPkqmif4\\\\\\\/QXNfAWLCcA1+WxOgJCh3WMKGlcrSToXXnf8\\\\\\\/lOWlSOHkP+heHaTcqAjuw0jTkmKXgdvtdrrNAA7qHZIAHiOoYN0ltV1ZMZdcBpq2UUijLxl5xy3Xw2mjZ0ZmcsTJWV9RQ9Fht+WjG7i9QlM+GVeSbXh+Pt6f1I6PIpiMKY5E7JcrY0b3JgprlFbMSmIOSgSNNuwb20YUw8vrEF7UjtV6pVVARMcTzrkoAz1uJiGv5xEKMeK9ehLqoQ2YweFMVj+R4FNKwG2XzO9sWLyq99ZJ2pi3PTafDu8HpZGhLpTNqNIcthKbak51S+1uwoqDPDtd9FOKisHsAAtdnhMdyxHxqO8k+o43Bx3DEDxpqV22GBKhG1orwZZH7LAaV4jq80LExg56f6G7441+ru1AV\\\\\\\/fmHbXKkJ0nQGGOFiYJuhwjxSw\\\\\\\/0HNVCOzMPDupYYcgIjHxS6ApSSrJB4lN97bfSciVBsHjydir9jUmUuM8F7KlWegoNdiacseqSrJbMGqylIikIIZERTJSbFSlQz8U0zQ5biJZzp\\\\\\\/VLKCDXJFMPuizsRZudz0HO74cI7eu3LRUlmnbyTRYu6l1\\\\\\\/\\\\\\\/hQ2MrAXabDQvj4ZmIiYVYd28Rk\\\\\\\/\\\\\\\/gs85UmTRAi0JJvLKTYbF9815LY26Xx+KL0+HAX5oYcrkZl+P4he2joK1xwABlrXLpHEfnOnRR7RyGG6uGm9rVTHtMtBlXpjeJHxderbkZ\\\\\\\/UGslUtVuOf5cJy5HZvxVgAzx9AvRJxInZJ+e5yiAZ\\\\\\\/cLMxFTMDQaoBNGqaM74GJnSTb8ZJV19oquSGMy\\\\\\\/2GUrTG1T9aERzeyOS1KFI9llPPS4TimlA8BUVAcYjtnbPHvLUsm\\\\\\\/Koml5Y7Fy\\\\\\\/nJ7R5u3Mjttbyndexg03rCLCozXYTAariYWVOyt8Hdl4iVKjw7I4ujKsuKWC+9f1FjAnj+rHhVNuz+ilP0nRjJrpYXWT79CEBC6xfPJ6gTv4uBGMePQGWyL0j9T3uMCadKlbn4L4NE3IQt2dINrh0vHk7s8aRuwSp7LDcuCMixgo+9gDR5cwlCiWNHWdWhEaJMGVpp+BB9U1VMXGKKaFtIXTP+d+s3FPFUMcaS3devz169aY1Fce5XgXj7KpmgODC\\\\\\\/homR595AgLv07GJUnyH9ZmFCbOvqRRYt+w1rqI88wm6\\\\\\\/t5SXcLdsZ75wJ7KOrSfky88niGg+EUALL8HhpSgfncKVbZrb07HnfHEWfzmjSMqY\\\\\\\/WePHWHWa3BTbJhEoDmwoYZ1\\\\\\\/vNffW\\\\\\\/RQPDY5H4wgqs+GGGUi3xJ\\\\\\\/h1kbKQa0PfDMbKqCEKmoA1XQkXhSpRldMNTj8dyDBui23W0LwUZspdYe5aMdaM2OI79ZD197Dl45hmlOPZ6+a+\\\\\\\/5mrYrZWqPeiB+e5owZfiw2ef6DYiJxe8AXSMM9uwKuqNplSlKrIg9xnU2J+w9R4lJ0oLycTpLaSd7DilQ5jcxODafLmW4p2PLtZS5PfoknTMvwQNux+R6vwRehYie5wLsZgAB1+5pEfjJtGTP5Sxgo3M\\\\\\\/vePZWbLTkvFe3Agy5B09l6gbBcbuQrTM30h9n1KJb8rZIBcjaCP2b6NGiEvhlwthQlHu2QOaLO08V3zpzVp\\\\\\\/gTWlhAAo3DpYkiZwL6gaEJ7\\\\\\\/sELcbXBGZY\\\\\\\/7sZ0\\\\\\\/yuaXdoveUAJ4XH9QjfKuKcfSAT+p3tom6AqmmUuLwqB7A\\\\\\\/8ranpJuwzkWiu7hyRgw0qling49VQ\\\\\\\/KfTMyMSIMgol+gl42GM8\\\\\\\/aeNoc9m8vmAxG9CQ3DJqNjh5QscRYev2iwx5b8hI6PGbAxUUXmj78eVP\\\\\\\/yEGhWov5M\\\\\\\/uOakcP5jfI8HnSxSFbysvLptqpLJoktN8u0mXitNrguoQCkfWzGW4T2Opi3wAcxk4n\\\\\\\/0hw7v62SSSD285AKU4nK9\\\\\\\/0AY\\\\\\\/c0ysx+QA6YDGbcyG01fzsHcbouWxi5I2e6g2SvVftA\\\\\\\/Yp77Sjvv0MJnQch7ifGaQJOjRyybKKGv2NxZ9sG91KHbSRDN5YZYzFA2giN5VgboyRK3ewxM9992Z5C4FeIiB19qHRQZdxXLUDAZ9ZRdq0pxFd0SB7UAUcomRDin6RD5vbx0azwbiSWS7d7hS9BIt5QB7Y75vHzXsoCYa5fC5LapBnjQENCtRFVbodqicWeQ+\\\\\\\/PuazMulyHnYpAf46mOEqKIhXT2KW+RAbb8vw+LxYad0656tFEHhu5MjYoH3yd5Mlssl9uD9bH5JP2Z6xdhDFNQpCLQe0LSv7oGR5CFqcH1pEVSeLV+qb7lfFv9kZB7FLGyEjJDqpQNY\\\\\\\/5ClELU0IJWSS9bRLzv282zgDPLdUD44flm+kItAJ1lI89zcpQWTQMmvPnYLSIDI3ZIC\\\\\\\/YjaAXbC2jPxdKFaKBGiSL30wr458\\\\\\\/Mh2AUnaY+7BXJ9ZALV0ZKshzTdAx\\\\\\\/4\\\\\\\/hhhAbhDnwHppLljyvh+DuloAipdqoMOUiKAXLQXioRlWn6HlrahcroYOJyb4U3q8N4ulEdW1pk03+54wn5CKx+bx3W1lYlX8f37v9lMNWprEYSKKPXs3GfPJl1Lpi4ZJd14m0Y4WvlD2ozJN3hcHSztc9xoSlW1DSp1Pb34AKsY4xJfyh6OLMJQKdtz6dFqlcPjHCfJACR1fQCAUNLUBdnMNth\\\\\\\/uzm6dVWsBp3VYKaMkeE5crXcrIXhnHuQ70ok1ARoPb69XEXOTNyuDhwNvbMjNvkn\\\\\\\/x+h0UIibRpJQ10KOarSl9WcBbUOBF3oz2e+RXPD8bO\\\\\\\/YSLH8QGAWKM9Ey+UKvoIWbTZROBYkq+z6fl3O5ubnus5KgrdwJXQBbceVI\\\\\\\/n62orsDlZgmoxodQciTSSo7IpoN9Lu1EUNgA9\\\\\\\/0mKn9fI5uDNq93s10+0sFyzpuwbRSLDADYtcfclk65XrM3Pmeop+9COKjCyl+mXyeQ2f0ofU0lYMtOqqPzHNEbLq1GytOdtGMiFVZFaRoM+M++6W2FhEC6QXY\\\\\\\/u1gKxMFwinrRGYrs1eWsIDMKSO5EByHhmN3yoJ\\\\\\\/rmiNnkc\\\\\\\/wRhOvDC3EssDvwC5ZfqWnqIAqk0asf58GIal8xj8ZqFUg+OyzgDohgnG5VftJy8mhgphqcVjBk2ViSGHEWY6Eb5LvzTkSp3hpZZSyXooUR6bIIfZrmHYtH6iXyzFw6vjbHT7V7K6X3938urZxfWlw98\\\\\\\/4xdLS6VMshw42IHoRs3gdSnL6bVQPjNz4K0CPigW6wgO0TB3\\\\\\\/lXrVsz+NQCkFCWNBXLMCbcvGVlVXHQQpqYZnRmD\\\\\\\/Rm4FodNJjgH8HiuvqzoATNAZGx7SqUZ\\\\\\\/CNie7\\\\\\\/aQYJe0EhDCvDk4\\\\\\\/cFAEHj9EW0Li9dsMujMakOd0botJ3CQDvRIUTDjaUP8xJEJtUSTq8NHIG\\\\\\\/RUA4RQH5OXmyh\\\\\\\/huSIzq8kziecx4y7uqzcrtFTInRrYceG73RldWzunLW5J+qiRMXLI9GTgb6iNVwKhAb5zuQ0RZRejcZPaxI9\\\\\\\/SIGfrFSdD+Qm\\\\\\\/iintPUqm7L1MMWrZ6ffyAbAs+3w7woRHj2wLeYkrv16Vf1+Xvk+wa88sO5jX0LzubhI18y9cU3ttKNXKweW3UUuMk7x44fR\\\\\\\/kPak8UA1MxG\\\\\\\/98roNhXMkP59\\\\\\\/ct46sRfBNpL7V2Iyxb3DMoRsDKPpyZ5tk4HhmlNJOorMJwut7+KGsQwbQk3QeO\\\\\\\/QK9bHo5XXbcTImJ2cSpOVWqIw2kfJBKQXN8CECYJINPnBFUexSGHW4b\\\\\\\/6Z0xDqfmZ8F3zSGW07tzzusyyu2LfFF\\\\\\\/DAhmjd2iybCI2VHOdGt8XCBOcXmHTZ5O4eZBdy0yIFOSQrxb2FR8ny5KF\\\\\\\/9VoToSN22VPLafJ3QX2yTLsLvfogAEpJ3ud\\\\\\\/9hmnyqJUFz4vlk6x6WF4V7\\\\\\\/2KlDLiPRIXBkWlxmcvnKH+oBlN9KHEAmzkvYn3pOrATp\\\\\\\/C1lstEUWvOJrbc9qF4cMC+BZ1FiXU2qNO9JknYtdzCMaW\\\\\\\/wujR+TzoZPoZVWWuI8A5JxYJd1Hvd14dB3i3sMLer4SMZA2kerosIhZJO6uFQXhdKtH4dT81LzMcxbJ4g5t1JJqDrBZ0U6siX0EN7PW49Z0THgwCWQRiPJit9CvaipQmhqs08l1JE\\\\\\\/iKplCX8DvGEYMD2s6e8d3rBArat3U4pcUoD+qTnYV7wEvwiEYld8zFbmy0OGewMJHwsc5LnL\\\\\\\/0rZIXA4PinGxqFSpIz+yRWSVSsudrRUBpcuePRMQ4QkpWY7BrtlgOoSuy+O6lYK8l00vAWgHz+p3cHhWHpedo0ZRCLasAuRAn6oDOarVti7AbsckF9cuqkfvOI+dt7dqKt5MrE2\\\\\\\/rDSxtqGsG71b\\\\\\\/AS4eLOmo0+HkIb6uTLu1Tjzq88wjngPc536LXVj6A0eDpksH0oLvpRu1mJTqe9EE1uSeYSSgRjyeIsmYrWWnSv5PHRWKLLQHB6wQArovWzn3nc8+jjZ4bHkAvfP8oWijBRvAcMTYNkxrl4ODRsKMCJyCv7MaYIlqf3FisWgMUCs2b8x0\\\\\\\/+nW69uGAlRZjGrEeUUcItaLT35MRZpLEYuL96zQhhwffcVmZxa+O1s15qDs5\\\\\\\/4OtygCiiBJ\\\\\\\/0VUeP3OuqDZ5k+Fm3AHYnOWmi4lzY0kqWf\\\\\\\/bAyhb\\\\\\\/BFvkbFAQhgTnnRjfhIkFvSP8Dc+hFyNgZXrtBdRXPaZkm083AaTol6PO\\\\\\\/Bjmq0UPqWwLmey+ecQVefMrnG\\\\\\\/DMfYR0Nu2iMUJRWzr0uRUOERUWXI1vZeXBkvZp\\\\\\\/zxDubUCV8RKbupNyZM25mNjRNF0dG6YARjHKqhF98RZjNdjJdyn0n8ujj30Vf0Rq1u9vVY4gHsAHyOuDZQ8HfvOuB0d6QDVlepMd2JwBIQnmzLeFQz6at5ZIwXR3Hz0y0LG6C\\\\\\\/dBqeFKoUApkmdVq+2w4ua7eJna6RCtLdPHZh6dWIjtObiwPjRkd0vybrYhJWGoXyNwQ0pkUwo9+3sLUTvl6LEG3fACYJuBa01rsVMubp30ByZRUGHE6MAR7svZAmLx0WYMC\\\\\\\/tLFobQCHTOw427Qp\\\\\\\/HQzCWOseziR0azUJF2gB9M\\\\\\\/a4nBVtGFMdRs0dtIdHTYRLJ+pyZ2NugSeqFg96r7Oto0hlzF+HCCFyKnXIarT86ItyZ1vr0e3DdvDz4UK8bFo7rjKz+F3XGEVmWW4FL36QeLuZw\\\\\\\/jD3Ddzp2arVBQ8ZnMaM1Uk+1zrVv68iGiX2qzw2mFrDqghDsUR+MCZR9gz2W5Fbaq8McYN\\\\\\\/6hhKWC7bJ5jDb1C0rni4GFaS\\\\\\\/8c9HCK+cMHc33Juf1rGH2+ZdoTJIyQ9lnyNw7eB6LmGkR9QY7AzqKpeB1iGe8u4HV\\\\\\\/m5cZYHTy+OuT7eMUsZFh3EpyzI4Q8XSBpmcR9Rq2zF3DG76leiIQDAcdWYxlC1f\\\\\\\/jacK4rYfZ1Lc91jiCSbOZWgZYeori6uPqnq5QEke+MYqXSjBC7M5kJvrrZhxuTEdidSNYLijDeokTtW97w9yRu+laLtB2Ursc7lvREVQE43v427lBC9ieZT7xnlcuzy4Ru\\\\\\\/o3Q\\\\\\\/Qfy0WYtNXiHf4FGSV2kS0mLjuiyHBUx3dtdAR1KDypogVY17t8sMi+w+rBvYLmeezqWqaaG9vOrRVFDDPu65qUCAUzJ524xgIrBFvi3WWpaxJT\\\\\\\/3qLHo1AlovPKWClcRZgcSnMiXYRgkewvA5gXdojgfNWOkVN814LGu9F7GiYkqRElzc+yrUrnQwRaUXPjHxKfk2S5wVWyQ+TfFbGfLojgTOSTS7c+bQT8pRDmGyXLk7H9JuY0TF8dhyBKSl3122l8y0SXrJsR2b+hzq+JIoqBckx30\\\\\\\/BH2CDu\\\\\\\/LumUG\\\\\\\/NUQoRgcZ7cyn5hqZJi19UI00H7m\\\\\\\/UPlt7s56MmSt9F0YX+nCIus+gyIr7vM8U4QD+t\\\\\\\/DIkBJsZAowLnNbwPEi5DPTXMJx5oSZRVWXJHCHopmGl9QWGoFvfT13\\\\\\\/HETCEFaYN5eJVEV4oTZG2avd4dhQdvNRauwVUOQVOQnYMEChg4yMq3u\\\\\\\/WB0tiwfGgBHaWaWJIrxlDy7KLzP5JjsyuvVuu3sP7gqD4NOCHgZqCxHQe4BWQ1\\\\\\\/pBY50kuaWTNz\\\\\\\/pJ2+djjKxwgVudGI0BeF7ldssAKBv493p8pa7S1VpxbMgY6PPiz0f7SUuTVZuZyl4ac9fyCalQjQqGSQX\\\\\\\/Kvrdue6nodFXXj8MXQwfasbHeNn40PrhKI\\\\\\\/VhP\\\\\\\/LnSHELqTJyY2HAxhYMSJA6s3KAQvhJj8JnWCq0s+oPpkItng8ViYzvJBQuTSlBbzvF3fr9eWOYuWIUUq5cG7Knpzkr5EXbO1Kdglucg7AeX4VIGf3Zs5FcKvYMCm8fj034kwpfZbNbJk\\\\\\\/9RLQF\\\\\\\/CRASLYv0ealNzw3SySlDGJMILZhiU1\\\\\\\/nJvK1TCSdnc+8VX1+CkEKbbbDr1umysR4DySsj8KgfefChPjGs5vWKWTEsZzZZQK9ZtNleYtnFs4YojOgBvu7jHk9\\\\\\\/hAbEoiZ+4WtLwEHDViS8GDRRzhvKX1AyM6VABc+iqpjL\\\\\\\/gBiXhPKXTp1dGMjN0p9xr+OarreTYSlKTxEvQwtFvX1pmjh8IC0MwLmB0vKV1t4Rot8MXAujgjPFNSdOlf3f9UZu4Iqpw+iIRIwzx5YsDc\\\\\\\/q67Pj80r2IRy6+buvWyJKdMNfed37UukdbbdNatsfvSCYTH5S0Z\\\\\\\/BNI9yor8ZED9j4RmqkIqBPkab91Ulai\\\\\\\/Xyu41uwAja30QQPxCwA0ivLSTwJjSMNRbTjFP88iyiRaMxBBEsow8NdrsBsZsViflXkHGzsxLk+jL76mYjM3MXj2rUGLWbwyzdsY1BPC8dTpmK0VOXAfVki\\\\\\\/XvKfOAfOB0aOglQq5xUx7WlG9A9GWzYJ8Ni9K1CZIsz3TMmmdOSpBL\\\\\\\/4YJcE2c6qJ6xPEjjAxCEaA\\\\\\\/zkBxoau4DiJ3XxIRzcUhmkz9sU1DjsaaI66fjhi\\\\\\\/\\\\\\\/r9DQ7dECKqpz3ro7w6roez+h5NNf3TS0t7lBGZmu9SStPNNngC6Ft6N018NOLYjJKekSJinHlTlmXleQp\\\\\\\/WjiG0bhAQg8PHsGOJ+YL5I5D3gNySyEFvBG2ecK7zPjGFAh9euFjU14oO5l3RBx0wPiICs9W5IJYrRXvKAqTvzJ\\\\\\\/dB3399tIagyKSlGjk4PWbOjDj8+uhUihdp14JYWiVhfLU+YwLNnXx\\\\\\\/UUvBLr3La1brE3GqDeFw5oVOVD\\\\\\\/gWjcEvrR+9GCMfbdL0uqxUGlAoVTbcrZPoVbv\\\\\\\/VI2HiV827pro+9nmYXOXJgxPfZDWQEXR5syIY4fOe5iKsiZFmV10ISZbv9JvPg9znqz6evWum3BjctdPCSqaQ0UNDhyI9LImuWYpd1F8WOsc2o+jp1b3qMRVM0blklEJN729J1TSipQPc\\\\\\\/5C6Hsp5CI1QfiIZ7ErGNWl1RCawFk3lSz0yO35LhbKXrtY2iJQC9A\\\\\\\/yQ4Oqy7s+iDi5qEtIAOIyiFG4Ml+nNKy3RXZmHbnue+ryv17psTC3sB1cWG+7pDncQNSkshgLrOVS9g3kFGEslSng9HGnTjA45O7AgS9HxBOdI6sfQQaA8nAzft01A02DNBswt9Ebi25pS6DxGal4tt4CNWzicp25OPUSwQukhlXGz8Oi79IrC++q0f+bf8ilMwHan54PWIcWOUnedIZbLAuiMc3rLP5HFhZ3g3yiCo7j87QzVvwiTgZ\\\\\\\/4mwglFPrUd8KN\\\\\\\/CjBE5+m1x27GnK1S6NWcXXvaO\\\\\\\/S70n7qHAkWUPUuHk+u+zjdbq+YdFa94vNcNUU2fz19vtnQ83dk67Wk5f2BheyxJ83n5qCs2JHMO6G2gp3Bw4JUUSvKD3LniGZZOeeZiyegwTwylLqMdNcxaL\\\\\\\/CH5v1QVJZm8fh+BcK5gA\\\\\\\/DVEA+3+XfptASLCi9jCGiqKayx4Pmn1rUFrcxe6u+VVaxzv34fDODLji5WYdu9VQKXs3g8xWAk\\\\\\\/xXk1sUYKWpRT15N9mpVSMPypDGgJwlNhvYYEWxiNT\\\\\\\/RVciaEeU2QAXmCJeAfGnX+7N1GM\\\\\\\/Khua1a4ys0SDZoHcexij1hjHfySDAMoC\\\\\\\/3ESDHQtVeWOoWOCgiGy4+eFnA2gWvv3U8MDU2Khjsukr3Xp\\\\\\\/EzKXczzoP5Bs74bLsD1r7sW11YT8QtB1AEHgt0pG1QyQWsksk5BiMpn7E6ecd\\\\\\\/sVELXIKjPRcX8Q2U5crgXMS5rjqOACEVHq3LOGS0xcvkVvzCmcZRt8lXsUXr8TIz3K+oYA4fXFCbUQYiPQxlnKTMR5BQgv\\\\\\\/dpu8l2PHGeoLmDrn6+58JuhIfseDH6ziG7a6AHnbq9cRTxD+X6SFjSBQJEvO\\\\\\\/SmRgVbdj9CL9sEchcnb3vvTQNQvikp6KDY5w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022R4ZaqHHOpy2riYGKPj4+xA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd82878f0fe96192ae5af528e9427ccd4c145bea0c6e528007d20dce3dbc444c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NSpjHfoMN7X4TarMZzOq99BYfcB6B9lPGzCo+EW81kvwCTnnWCpH0i4Rcf9B8qvNbs2uSi5uYxYU97hlKc4gGMnEpaA3rJfl7F67iCyY9hCpLtPnpzfCtPqg9t9KP7sSV63cmN5PQA8qmUhQjpZWStEPPD2S8RWk8iG06c34N3UOlRXm0HIkHOrCg7EOJarmV2vYSq4NStpK8aTjbb72UNKF45NfZKgJhqlzhvGSpY+UiyxiG8xqO6CVwTzuDYbfjrpRp8yDETbaFAstGAvgHdcXhHLBTMqAONcvqGdG6NJtVl4EtgE5gEMnNyghJqKFZIjJ5qN0XC7+fGvXutCmOUaNJAswjl7tV6sjEbbJZ7M4prJd9nF1HSjnI+mb3CE7LvilaYuMGIWZ1uTCYVnVZSj1OebfnFE\\\\\\\/cOkSUEkwjSbicXsfUaPWv3PNN9LVNuL+fpFrSNXtueg4AdidLw\\\\\\\/ykyKbv8JuedjhAEEpRn+VvIYrqIg0ePkS4BsAoyleajRhiqQLG+zVSaFuWVlnbApbrnKMAH8e58kG1kgspadIkcNwegjSc5sLKXJJRbT4vHyUWwSscgDMqrhERF0nSqPx8h7F7QHcbqC1UWvKlGrlPhgnV+AFMQDWSkh74eLLgonboSe\\\\\\\/iVM4TghmURyui1ksqUu\\\\\\\/KGzNnhKmgeYllIDgNLfEMOekrB3TlcoRxqe0N318L\\\\\\\/yy21H1Zy8TIDm1rVrRk9i2cMzSIkPI9\\\\\\\/4fw1\\\\\\\/eTbHMnUNQJ\\\\\\\/8Gq2uX84eE25CYowv0Ny8s+tPCEiGRP4yBEHY8ZUDAkKaAhyhA3KLze5nwU752yMqMpO+5hIiwcK\\\\\\\/DPmDHJSAqiGh8xnSpugg9ekwrGuVl4L1FddQQu\\\\\\\/7ym6FbvyNeS3hICpb3l7A+xStY1fwNaKaZupWkEmAlIGT2FUKxIhb0s72UsjhWvs7YvBwYlaX2EhPrRxRkiP7MOW1Dm5AH\\\\\\\/lnmuQ877g\\\\\\\/2HVwblpQXm9HVHKrv0KfApL3tcXbANtC5GbGPZC19kuzpUgmwwkEG56ZkNRsPIe5wYFMYsiiWbsmsdxoTZ8tG6cDXLrpPgYjS4r60WGmfd4EA4HOhfgGVm0kIW1BfoDn1revOMc71iKPN1asOySHmNT9\\\\\\\/h6Uj5UiehFYQpFlDZwMQoPTscF8UsyofmXLB71p3lXWZ6LrDYfDsRDhbfsgOY9KudpnPDh7dCDRdI2JPhtvMVkuwAjG84ZWm0bJaSm7irUTvLa\\\\\\\/pG8b30BoZ2Uj\\\\\\\/r8ALsGF0gk6O3ERVT1+trc8AQL93WZluWGHJloDRe2\\\\\\\/VM1WfYdpHbhFIP\\\\\\\/eU8oEneTzMMj0wZh+evbpAi1jhsFSq2wSdv\\\\\\\/wHX46P6KqFWURyR6kfzV8o0ttfw\\\\\\\/6aLiqbpfATWCQ+basTJvXu\\\\\\\/QCBYpXiKIdbkhB11LV9HkU30zcXfyLIAeXcfKohE0f8DQrJzLx5vogPXTa5nvswFRnPG26rTw9fWJ9IpVrUgSVeavo18xy+ilhMKAvEfqZJ51B41TnJbChVQD8HreZfHNeezCiQoIsiESJci8LUU62KEKE2R429wSPlivN43Dh1fQwdx8r3mROf0GKkn9TJUj2IfhWZmZ8+I5Egu4iGsdwBV2yjy5hEbME+PT3+Dish8e\\\\\\\/Zdyi6noTV2EbMbQQD+yvH2b+v5Gj+Rftg6BV845xsZTXDydNOCT6wfr4lL9JKhkcWdPBMge0kNiRGkKStZXH9SWIe91sK5kzmeGOYkwrMsZvAOfU7Suvt4jXbrRnWBtQ9J2GckgEfntCGyIXNLs6tS1nafy29PCOHwqPdf\\\\\\\/VCJCaTFlWsVsyGnARctuZ0Ufl0lOLX51rRnd5OuPczjZ0lDKZ4zKCLsJ0ld2Yhywx9APo+bYe8lOKQ3ZVQR+G5XW2p2upnli2t5YJbuxv5WoNCdrNZFP2uBW18fn3MdR4yp58Z6p8nMDs7PfxKILZdczcmjPQaFi6WAkIBLdHTv6d5Lj+2\\\\\\\/JOTWah2oeHtD447uZY9vXcb6A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FuYymdwaOIK\\\\\\\/5fB3hzOuHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022627e8fa37b43db23d0068861fdbb7930c2d5410c367380c73f828ab7de9dabd2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459983 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1379" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zFJZjU89ipyR4aCzTz8Sl0RzazjdRJlzUOSglPSqacE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1379" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jnNP_3tGn0UStAcEtQ_gWoR8VNlmhkrQFBB8pWcXFSg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1371" title="DPY-17 and MUA-3 Interact for Connective Tissue-Like Tissue Integrity in Caenorhabditis elegans: A Model for Marfan Syndrome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1391" title="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459983" id="node459983" data-pisa="ggg;5/7/1379" data-pisa-master="ggg;g3.115.018721" data-apath="/ggg/5/7/1379.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6547-3624" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erica J. Polleys</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1864-8502" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison A. Bertuch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1379-1389;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018721 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459983" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erica J. Polleys </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erica%2BJ.%2BPolleys%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polleys%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErica%2BJ.%2BPolleys%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6547-3624" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erica J. Polleys</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alison A. Bertuch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics and Department of Molecular & Human Genetics, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BA.%2BBertuch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertuch%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BA.%2BBertuch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1864-8502" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison A. Bertuch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:abertuch@bcm.edu" class="" data-icon-position="" data-hide-link-title="0">abertuch@bcm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/7/1379" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459983" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459983/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1379.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459983" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459983/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1379.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459983" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459983/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1379.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459983" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459983/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/7/1379.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459983" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459983/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The <em>Saccharomyces cerevisiae</em> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex inhibits TORC1 signaling and SEACAT antagonizes the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex. Conditions in which SEACAT functions to inhibit <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> and, hence, TORC1 signaling, remain largely unknown. The SEACAT member <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> was linked previously to telomere maintenance and DNA repair via genome-wide genetic and physical interaction studies. Therefore, we questioned whether <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> functioned through TORC1 to influence these pathways. Deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> delayed the emergence of telomerase-independent survivors that use break-induced replication (BIR) to maintain their telomeres. Similarly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutants exhibited a delay in colony formation in a BIR assay strain after double-strand break (DSB) induction as well as on the DNA-damaging agent bleomycin. Deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">IML1</a></em> rescued the impaired growth of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutants after DNA damage, consistent with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> functioning as a regulator of TORC1 signaling. The delay was not attributable to slowed DSB repair or termination of the DNA damage checkpoint but to tryptophan auxotrophy. High levels of tryptophan in yeast peptone dextrose media did not rescue the delay in colony formation, suggesting a defect in tryptophan import, although levels of the high-affinity tryptophan permease <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> were not perturbed in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant. Addition of quinolinic acid, an intermediate of the <em>de novo</em> NAD+ biosynthetic pathway, however, rescued the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant. Together, these findings highlight the importance of enforcement of TORC1 signaling and suggest that internal tryptophan levels influence growth recovery post DNA damage through the role of tryptophan in NAD+ synthesis.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/sea3" class="hw-term hw-article-keyword hw-article-keyword-sea3" rel="nofollow">Sea3</a></li><li class="kwd"><a href="/keyword/iml1-complex" class="hw-term hw-article-keyword hw-article-keyword-iml1-complex" rel="nofollow">Iml1 complex</a></li><li class="kwd"><a href="/keyword/torc1" class="hw-term hw-article-keyword hw-article-keyword-torc1" rel="nofollow">TORC1</a></li><li class="kwd"><a href="/keyword/tryptophan" class="hw-term hw-article-keyword hw-article-keyword-tryptophan" rel="nofollow">tryptophan</a></li><li class="kwd"><a href="/keyword/dna-damage" class="hw-term hw-article-keyword hw-article-keyword-dna-damage" rel="nofollow">DNA damage</a></li></ul><p id="p-3"><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> is a member of the vacuolar SEA complex, a dynamic complex of four proteins [<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Sea1</a>), <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a>–4], which associates with the nucleoporins <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003068">Seh1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004198">Sec13</a> as well as the TORC1 regulators <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000788">Npr2</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001015">Npr3</a> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dokudovskaya <em>et al.</em> 2011</a>). Initial studies in budding yeast implicated the SEA complex in the response to nitrogen starvation, amino acid biogenesis, and intracellular trafficking (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Dokudovskaya <em>et al.</em> 2011</a>), processes shared with the Tor complex TORC1 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Takahara and Maeda 2013</a>). Given this and its interaction with TORC1 regulators, it seemed likely that the SEA complex functioned in TORC1 signaling. Consistent with this, human homologs of the SEA complex components were identified as upstream regulators of mTORC1, distributing into two regulatory complexes, GATOR1 and GATOR2 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bar-Peled <em>et al.</em> 2013</a>). The GATOR1 complex contains the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> homolog, DEPDC5, as well as Nprl2 and Nprl3, whereas the GATOR2 complex contains the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">Sea4</a> homologs, WDR24, WDR59, and Mios, respectively, as well as the homologs to yeast <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003068">Seh1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004198">Sec13</a>. This initial study revealed that GATOR2 functions as a positive regulator of mTORC1 by negatively regulating GATOR1, which is an inhibitor of mTORC1 signaling.</p><p id="p-4">Further analysis of the SEA complex in <em>S. cerevisiae</em> revealed two epistatic groups, the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex (<em>a.k.a</em>. SEACIT) and SEACAT, which segregate and function like GATOR1 and GATOR2 in humans; SEACAT (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">Sea4</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003068">Seh1</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004198">Sec13</a>) negatively regulates the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex/SEACIT (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000788">Npr2</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001015">Npr3</a>), which negatively regulates TORC1 signaling (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Panchaud <em>et al.</em> 2013a</a>,<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">b</a>). Importantly, although there are two Tor complexes in <em>S. cerevisiae</em>, TORC1 and TORC2, each of which regulate a distinct set of cellular functions, (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Loewith <em>et al.</em> 2002</a>), the SEA complex has only been established as a regulator of TORC1. Analysis of TORC1 signaling under conditions of amino acid deprivation revealed that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">Sea4</a> act redundantly to attenuate the inhibitory properties of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Panchaud <em>et al.</em> 2013a</a>). Whether there are conditions under which an individual member of the SEACAT complex functions alone to regulate TORC1 signaling remains unknown.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sea3 impacts senescence progression and the formation of survivors in the absence of telomerase. (A) Proposed pathway indicating that SEACAT, which contains Sea2, Sea3, and Sea4, acts as a negative regulator of Iml1 complex/SEACIT, which contains Iml1, Npr2, and Npr3. The Iml1 complex/SEACIT functions as a negative regulator of TORC1 signaling, which regulates a variety of downstream targets. (B) Southern blot of XhoI-digested genomic DNA isolated from wild-type and haploid SEA complex gene deletion mutants were probed with a telomere repeat specific probe. Approximately two thirds of wild-type telomeres, which contain Y&#xB4; elements in the subtelomeric region, give rise to a 1.2-kb terminal restriction fragment upon XhoI digestion. The restriction fragments of greater length derive, in part, from individual telomeres that lack a Y&#xB4; element. (C) Liquid culture senescence progression assays of tlc1&#x2206; and sea3&#x2206; tlc1&#x2206; haploids obtained from sporulation and microdissection of a sea3&#x394;/SEA3 tlc1&#x394;/TLC1 diploid. Blue lines indicate growth curves of individual tlc1&#x394; spores (n = 7), and red lines indicate growth curves of individual sea3&#x394; tlc1&#x394; spores (n = 5). (D) As in (B) except genomic DNA was isolated daily from samples in the liquid senescence progression assay (C). Arrows indicate the emergence of Type II survivors, which acquired a more heterogeneous banding pattern." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-76344638" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sea3 impacts senescence progression and the formation of survivors in the absence of telomerase. (A) Proposed pathway indicating that SEACAT, which contains Sea2, Sea3, and Sea4, acts as a negative regulator of Iml1 complex/SEACIT, which contains Iml1, Npr2, and Npr3. The Iml1 complex/SEACIT functions as a negative regulator of TORC1 signaling, which regulates a variety of downstream targets. (B) Southern blot of &lt;em&gt;Xho&lt;/em&gt;I-digested genomic DNA isolated from wild-type and haploid &lt;em&gt;SEA&lt;/em&gt; complex gene deletion mutants were probed with a telomere repeat specific probe. Approximately two thirds of wild-type telomeres, which contain Y&#xB4; elements in the subtelomeric region, give rise to a 1.2-kb terminal restriction fragment upon &lt;em&gt;Xho&lt;/em&gt;I digestion. The restriction fragments of greater length derive, in part, from individual telomeres that lack a Y&#xB4; element. (C) Liquid culture senescence progression assays of &lt;em&gt;tlc1&#x2206;&lt;/em&gt; and &lt;em&gt;sea3&#x2206; tlc1&#x2206;&lt;/em&gt; haploids obtained from sporulation and microdissection of a &lt;em&gt;sea3&lt;/em&gt;&#x394;/&lt;em&gt;SEA3 tlc1&lt;/em&gt;&#x394;/&lt;em&gt;TLC1&lt;/em&gt; diploid. Blue lines indicate growth curves of individual &lt;em&gt;tlc1&lt;/em&gt;&#x394; spores (n = 7), and red lines indicate growth curves of individual &lt;em&gt;sea3&lt;/em&gt;&#x394; &lt;em&gt;tlc1&lt;/em&gt;&#x394; spores (n = 5). (D) As in (B) except genomic DNA was isolated daily from samples in the liquid senescence progression assay (C). Arrows indicate the emergence of Type II survivors, which acquired a more heterogeneous banding pattern.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1379/F1.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/7/1379/F1.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466344" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Sea3 impacts senescence progression and the formation of survivors in the absence of telomerase. (A) Proposed pathway indicating that SEACAT, which contains Sea2, Sea3, and Sea4, acts as a negative regulator of Iml1 complex/SEACIT, which contains Iml1, Npr2, and Npr3. The Iml1 complex/SEACIT functions as a negative regulator of TORC1 signaling, which regulates a variety of downstream targets. (B) Southern blot of <em>Xho</em>I-digested genomic DNA isolated from wild-type and haploid <em>SEA</em> complex gene deletion mutants were probed with a telomere repeat specific probe. Approximately two thirds of wild-type telomeres, which contain Y´ elements in the subtelomeric region, give rise to a 1.2-kb terminal restriction fragment upon <em>Xho</em>I digestion. The restriction fragments of greater length derive, in part, from individual telomeres that lack a Y´ element. (C) Liquid culture senescence progression assays of <em>tlc1∆</em> and <em>sea3∆ tlc1∆</em> haploids obtained from sporulation and microdissection of a <em>sea3</em>Δ/<em>SEA3 tlc1</em>Δ/<em>TLC1</em> diploid. Blue lines indicate growth curves of individual <em>tlc1</em>Δ spores (n = 7), and red lines indicate growth curves of individual <em>sea3</em>Δ <em>tlc1</em>Δ spores (n = 5). (D) As in (B) except genomic DNA was isolated daily from samples in the liquid senescence progression assay (C). Arrows indicate the emergence of Type II survivors, which acquired a more heterogeneous banding pattern.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">Inhibition of TORC1 signaling induces a variety of cellular changes indicative of a starvation response, including a reduction in protein synthesis, enlargement of the vacuole, activation and repression of gene transcription, and induction of autophagy (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zoncu <em>et al.</em> 2011</a>). Amino acid biosynthesis and sorting of amino acid permeases are also impacted when TORC1 is inhibited in response to starvation. For example, various high affinity amino acid permeases are relocalized in the cell, changing the import of certain classes of amino acids. Well-established examples of this are the effects on the high-affinity tryptophan permease, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a>, and the general amino acid permease, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1</a> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beck <em>et al.</em> 1999</a>). In rich media, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> is stable and imports tryptophan. Upon the inhibition of TORC1 signaling that results from nitrogen deprivation, the phosphatase <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004630">Tap42</a> dephosphorylates <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">Npr1</a>, a serine/threonine kinase, rendering <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">Npr1</a> active. Activated <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005127">Npr1</a> then mediates the degradation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> and localization of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1</a> to the plasma membrane (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schmidt <em>et al.</em> 1998</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Beck <em>et al.</em> 1999</a>). Consequently, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1</a> becomes responsible for the import of amino acids, including tryptophan. Notably, several members of the SEA complex have genetic interactions with factors that regulate <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1</a> localization, such as <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004951">Lst8</a>, a component of the TOR signaling pathway, and share fitness profiles across numerous chemical and environmental stress conditions with genes involved in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1</a> sorting (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Dokudovskaya <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hillenmeyer <em>et al.</em> 2008</a>).</p><p id="p-7">In addition to the aforementioned downstream targets, TORC1 signaling in budding yeast also influences telomere length maintenance and the repair of DNA double-stranded breaks (DSBs) via nonhomologous end joining. This is achieved, in part through the control of the levels of Ku (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ungar <em>et al.</em> 2011</a>), a heterodimeric protein that has functions at both telomeric ends and DNA ends created by DSBs (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boulton and Jackson 1996</a>). Interestingly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> was identified in a screen for gene deletions that altered the growth of a strain bearing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em>, a temperature-sensitive allele of the telomeric binding protein <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">Cdc13</a> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2008</a>). Genes that when deleted resulted in a synthetically sick phenotype at the permissive temperature when combined with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> mutation were given the maintenance of telomeric capping (MTC) designation, whereas genes that when deleted synthetically rescued the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> lethality at the nonpermissive temperature were given the rescue of telomeric capping (RTC) designation. Members of both MTC and RTC designations were varied in their known or putative biological functions. In this initial study, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> was placed in the MTC group and, as a previously anonymous gene, given the name <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">MTC5</a></em>. A subsequent study, however, reported <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">mtc5</a>∆</em> (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em>) synthetically rescued the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> lethality at the nonpermissive temperature, thus exhibiting an RTC phenotype (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Addinall <em>et al.</em> 2011</a>). Given this discrepancy and the results that follow, we have used the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> rather than <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">MTC5</a></em> gene designation in this report.</p><p id="p-8">TORC1 also contributes to the DNA damage response (DDR) secondary to DSBs. Treatment of cells with rapamycin, which directly inhibits the TORC1 complex, and then with the DNA-damaging agent methyl methanesulfonate results in suppression of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a>-dependent up-regulation of ribonucleotide reductase (<em>RNR</em>) genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000872">RNR1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>, and decreased survival (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shen <em>et al.</em> 2007</a>). In addition, strains bearing <em>RNR</em> gene deletions have decreased survival when treated with both rapamycin and methyl methanesulfonate, leading to the hypothesis that survival of cells incurring DNA damage requires TORC1 enforcement of nucleotide pools sufficient for DNA synthesis postdamage. Interestingly, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> has been found in genome-wide screens to physically interact with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004712">Yku80</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003628">Srs2</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003709">Rfa3</a>, all of which are DNA repair proteins (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ho <em>et al.</em> 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chiolo <em>et al.</em> 2005</a>).</p><p id="p-9">Given the role of TORC1 signaling in telomere maintenance and the DDR, and the potential connection of the upstream TORC1 regulator <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> with telomere and DNA repair proteins, we sought to determine whether <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> played a regulatory role in these processes. We found that deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> did not impact telomere length in telomerase-positive strains but did alter the progression of senescence in the absence of telomerase and delayed the emergence of telomerase-independent survivors that maintain their telomeres via break-induced replication (BIR). Similarly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants had a delay in colony formation after DSB induction in a strain that assays for BIR and upon exposure to the DNA-damaging agent bleomycin. We found <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> to function, as predicted, upstream of TORC1 through <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> after DSB induction. The studies unveil a novel type of recovery defect, which is not in the termination of the DDR, but rather in intracellular tryptophan. They implicate tight regulation of TORC1 signaling via <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> in the ability of the cell to achieve sufficient intracellular tryptophan to allow timely recovery after DNA damage.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and plasmids</h3><p id="p-10">The strains and plasmids used in this paper are described in <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/018721SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/TableS1.pdf">Table S1</a>. Deletion and epitope-tagged strains were generated by one-step gene replacement or integration, respectively, with the noted selectable marker. All incubations were performed at 28°.</p></div><div id="sec-3" class="subsection"><h3>Telomere analysis</h3><p id="p-11">Telomere length analysis and determination of telomerase-independent survivors were performed as described previously (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lendvay <em>et al.</em> 1996</a>).</p></div><div id="sec-4" class="subsection"><h3>Senescence progression assays</h3><p id="p-12">Serial liquid culture senescence progression assays were carried out as described previously (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Le <em>et al.</em> 1999</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rizki and Lundblad 2001</a>). To summarize, spore colonies from freshly dissected tetrads were inoculated in their entirety into YPD media and grown at 28° for 46 hr. Cell counts were determined via hemocytometer. Samples were diluted back into fresh media to a concentration of 1 × 10<sup>5</sup> cells/mL and grown for 22 hr at which time cells were counted. The cultures were again diluted back into fresh media to a concentration of 1 × 10<sup>5</sup> cells/mL. Cell counts and dilution into fresh media were repeated every 22 hr. Several isolates of each genotype were examined.</p></div><div id="sec-5" class="subsection"><h3>Growth experiments</h3><p id="p-13">To assess growth in the BIR assay strain, fivefold serial dilutions of exponentially growing liquid cultures in YPLactate were plated on YPD or YPGal. To assay strains containing plasmids, the cultures were pre-grown in -Leu, -Trp, or -Ura minimal media, as appropriate, to maintain selection of the plasmid. To assess growth under conditions of stress induction, cultures were pregrown in either YPD for the YPH strain or YPLactate for the BIR assay strain and plated for single colonies on YPD medium containing 2−4 µg/mL bleomycin, 0.25% glucose, 0.5 M NaCl, or 3 mM hydrogen peroxide or at 37°. To assess growth under conditions of added tryptophan, fivefold serial dilutions of liquid cultures grown in YPLactate were plated on YPD, YPD + 100 μM tryptophan, YPGal, and YPGal + 100 μM tryptophan. To assess growth with added quinolinic acid, fivefold serial dilutions of liquid cultures grown in YPLactate were plated on YPD, YPD + 2 μM or 4 μM quinolinic acid, YPGal, and YPGal + 2 μM or 4 μM quinolinic acid. All phenotypes were recorded 4 days postplating unless otherwise specified.</p></div><div id="sec-6" class="subsection"><h3>BIR assays</h3><p id="p-14">BIR plating and determination of percent viability were performed as described previously (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lydeard <em>et al.</em> 2007</a>). Sensitivity to canavanine and/or hygromycin determined whether repair occurred via BIR or another pathway. All individual colonies on the YPGal plate were picked and inoculated into 96-well dishes. The cell suspensions were then pinned onto YPD, canavanine and hygromycin plates.</p></div><div id="sec-7" class="subsection"><h3>HO induction and repair kinetics</h3><p id="p-15">Strains were pregrown in YPLactate. Galactose induction, sample collection and processing, and DNA analysis were performed as described previously (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lydeard <em>et al.</em> 2010</a>).</p></div><div id="sec-8" class="subsection"><h3>Protein analysis under galactose induction</h3><p id="p-16">Strains were pregrown in YPLactate to approximately 0.5 × 10<sup>7</sup> cells. Initial aliquots were taken and then galactose was added to each culture to a final concentration of 2%. Aliquots were taken at the indicated time points. Samples were spun at 3000 rpm and cell pellets washed twice with water before being frozen at −80°. The cell pellets were thawed and normalized to cell count before lysis. For analysis of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> phosphorylation, protein lysates were prepared by trichloroacetic acid method as previously described (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Foiani <em>et al.</em> 1994</a>). For analysis of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> protein levels, samples were prepared as previously described (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abe and Iida 2003</a>). Before western analysis of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a>, 50 µg of whole-cell extract was denatured in 5% SDS and 5% β-mercaptoethanol at 37° for 10 m. Western blots were probed with α-Flag (F3165; Sigma-Aldrich), α-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> (provided by M. Foiani), and α-PGK (ab113687; Abcam) antibodies.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Sea3 impacts senescence progression and the formation of survivors in the absence of telomerase</h3><p id="p-17">We initially were interested in examining the impact of the SEA complex genes on telomere maintenance as both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">sea2</a>∆</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> deletions were identified as modifiers of growth of a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> strain and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">sea2</a>∆</em> mutant was found to have short telomeres when examined in a genome-wide screen (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2008</a>, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Askree <em>et al.</em> 2004</a>). We found <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">sea2</a>∆</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">sea4</a>∆</em> haploid deletion strains had telomere lengths comparable with wild-type, whereas the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">iml1</a>∆</em> strain had slightly longer telomeres (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Thus, in this directed analysis in the YPH274 strain background, deletion of the SEA complex genes had little, if any impact, on telomere length.</p><p id="p-18">Whereas wild-type strains, which constitutively express telomerase, maintain stable telomere length with propagation, strains deficient in telomerase, such as those lacking the telomerase regulatory subunits, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">Est1</a> or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006432">Est3</a>, the telomerase catalytic subunit, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004310">Est2</a>, or the telomerase RNA subunit, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">Tlc1</a>, experience progressive telomere shortening, which leads to eventual cellular senescence (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lendvay <em>et al.</em> 1996</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lundblad and Szostak 1989</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Singer and Gottschling 1994</a>). Interestingly, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> allele was reported to synthetically interact with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a>∆</em> deletion in a genome-wide screen for genes that affected telomere-driven senescence progression and recovery (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2011</a>). Therefore, we examined directly the effect of a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutation on senescence progression in the absence of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">TLC1</a></em>. To determine this, we investigated telomere length and growth potential of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a>∆</em> strains compared with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a>∆</em> strains derived from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>Δ/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a>∆/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">TLC1</a></em> (telomerase-proficient) diploids. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>Δ/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a>∆/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">TLC1</a></em> diploids were sporulated and dissected, and individual meiotic segregants were inoculated into liquid culture. After 22 hr of growth, the cell concentration was determined and the cultures diluted back daily in a standard liquid culture senescence progression assay (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bertuch and Lundblad 2004</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Le <em>et al.</em> 1999</a>). Although the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> single-mutant strains exhibited a brief and slight decline in growth potential on early days of the experiment, they were otherwise largely indistinguishable from wild-type with respect to growth over the course of the experiment (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS1.pdf">Figure S1</a>A), consistent with previous reports (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Dokudovskaya <em>et al.</em> 2011</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Panchaud <em>et al.</em> 2013a</a>). Additionally, the single <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants, like wild-type, did not undergo progressive telomere shortening but maintained a stable telomere length over time (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS1.pdf">Figure S1</a>B). We did find, however, that, with continued propagation, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a></em>Δ mutants had a more prolonged duration of restricted growth compared with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a></em>Δ mutants alone, with a continued decline in cell numbers beyond day 9 and 10, until growth recovery on day 11 or 12 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). This finding was somewhat different than the genome-wide study, in which <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">mtc5</a>∆) <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a></em>Δ mutants were found to have <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">rad52</a>Δ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004223">est1</a>Δ</em> mutant characteristics, with accelerated entry into senescence and lack of growth recovery (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2011</a>). The differences in the growth patterns may have been attributable to differences in the media [liquid (this study) <em>vs.</em> solid (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2011</a>)], media components, or strain background [YPH274 (this study) <em>vs.</em>, BY4741 (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2011</a>)].</p><p id="p-19">Two types of telomerase-independent survivors have been described, both of which rely on BIR to elongate telomeres in the absence of telomerase (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lundblad and Blackburn 1993</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lydeard <em>et al.</em> 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Teng and Zakian 1999</a>). Type I survivors amplify subtelomeric Y´ elements and have short terminal telomeric repeat tracts, whereas Type II survivors amplify telomeric sequence from the very end of another telomere, resulting in highly heterogeneous lengths of terminal telomeric repeat tracts (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lundblad and Blackburn 1993</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Teng and Zakian 1999</a>). Type II survivors predominant in serial liquid culture assays because of a growth advantage over Type I survivors (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Teng and Zakian 1999</a>). We found that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a></em>Δ mutants still formed Type II survivors in liquid culture, but the survivors took approximately 1−2 d longer to be visualized than for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006657">tlc1</a>∆</em> mutants (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>), indicating a possible delay in the execution of BIR or in growth post survivor formation. Thus, although the absence of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> did not impact normal telomere length maintenance, it did impact growth and recovery of telomerase-independent survivors.</p></div><div id="sec-11" class="subsection"><h3>Absence of Sea3 slows colony formation after DSB induction</h3><p id="p-20">As the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutation slowed the appearance of BIR-dependent survivors in the absence of telomerase, we wanted to determine whether <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> specifically impacted BIR. To assess this, we deleted <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> in a strain created previously to analyze BIR at an induced DSB (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lydeard <em>et al.</em> 2007</a>). In this strain, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em>, which confers sensitivity to canavanine and is present on the nonessential telomere-proximal end of the recipient chromosome (Chr V), is truncated by the insertion of an <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a> endonuclease site flanked by a hygromycin resistance (<em>HPH</em>) reporter (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). In addition, a 5′ truncation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em> is inserted on the donor chromosome (Chr XI), resulting in 1157 bp of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em> homology between the donor and recipient chromosomes. The <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a> endonuclease is under the control of a galactose inducible promoter. Upon exposure to galactose, the DSB generated at the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a> site on Chr V is repaired via BIR, resulting in a full-length <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">CAN1</a></em> gene and loss of the hygromycin reporter. When we deleted <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> in this strain, we noted a consistent 2-d delay in the formation of quantifiable colonies on galactose compared with wild-type, whereas the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutants formed colonies comparably with wild-type on glucose (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS2.pdf">Figure S2</a>, uninduced). The overall viability on galactose, however, was equivalent to wild-type (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). Importantly, the effect on galactose was attributable to loss of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> gene product as it was complemented by expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> on a CEN plasmid (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS3.pdf">Figure S3</a>A).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of Sea3 impacts colony formation in the break-induced replication (BIR) assay strain and on bleomycin. (A) BIR assay strain (Lydeard et al. 2010). An HO cut site (HO), marked with HPH, is integrated into the CAN1 gene (represented as CA) on chromosome V, deleting the 3&#x2032; portion of CAN1. The CAN1 donor (represented as AN1), which shares 1157 bp of homology to the CAN1 gene, is integrated into chromosome XI. Sites marked with A indicate AvaI sites used for monitoring BIR repair in Figure 3B. (B) Platings for single colonies of wild-type and sea3&#x394; mutants in the BIR assay strain on YPD and YPGal. (C) Percent viability of wild-type and sea3&#x394; mutants as a ratio of number of colonies on YPGal divided by the dilution factor and then divided by the number of colonies on YPD. Values represent average of two independent trials and error bars indicate standard error of the mean (SEM). (D) Platings for single colonies of wild-type and sea3&#x394; mutants in the YPH274 genetic background on YPD and YPD + 3.5 &#x3BC;g/mL bleomycin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-76344638" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Loss of Sea3 impacts colony formation in the break-induced replication (BIR) assay strain and on bleomycin. (A) BIR assay strain (Lydeard &lt;em&gt;et al.&lt;/em&gt; 2010). An HO cut site (HO), marked with &lt;em&gt;HPH&lt;/em&gt;, is integrated into the &lt;em&gt;CAN1&lt;/em&gt; gene (represented as CA) on chromosome V, deleting the 3&#x2032; portion of &lt;em&gt;CAN1&lt;/em&gt;. The &lt;em&gt;CAN1&lt;/em&gt; donor (represented as AN1), which shares 1157 bp of homology to the &lt;em&gt;CAN1&lt;/em&gt; gene, is integrated into chromosome XI. Sites marked with A indicate &lt;em&gt;Ava&lt;/em&gt;I sites used for monitoring BIR repair in Figure 3B. (B) Platings for single colonies of wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; mutants in the BIR assay strain on YPD and YPGal. (C) Percent viability of wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; mutants as a ratio of number of colonies on YPGal divided by the dilution factor and then divided by the number of colonies on YPD. Values represent average of two independent trials and error bars indicate standard error of the mean (SEM). (D) Platings for single colonies of wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; mutants in the YPH274 genetic background on YPD and YPD + 3.5 &#x3BC;g/mL bleomycin.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1379/F2.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/7/1379/F2.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466483" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Loss of Sea3 impacts colony formation in the break-induced replication (BIR) assay strain and on bleomycin. (A) BIR assay strain (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lydeard <em>et al.</em> 2010</a>). An HO cut site (HO), marked with <em>HPH</em>, is integrated into the <em>CAN1</em> gene (represented as CA) on chromosome V, deleting the 3′ portion of <em>CAN1</em>. The <em>CAN1</em> donor (represented as AN1), which shares 1157 bp of homology to the <em>CAN1</em> gene, is integrated into chromosome XI. Sites marked with A indicate <em>Ava</em>I sites used for monitoring BIR repair in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3B</a>. (B) Platings for single colonies of wild-type and <em>sea3</em>Δ mutants in the BIR assay strain on YPD and YPGal. (C) Percent viability of wild-type and <em>sea3</em>Δ mutants as a ratio of number of colonies on YPGal divided by the dilution factor and then divided by the number of colonies on YPD. Values represent average of two independent trials and error bars indicate standard error of the mean (SEM). (D) Platings for single colonies of wild-type and <em>sea3</em>Δ mutants in the YPH274 genetic background on YPD and YPD + 3.5 μg/mL bleomycin.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">To determine whether the growth delay was caused by a general sensitivity to galactose (rather than the DSB induced by galactose), we took <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants in the BIR strain background that had been plated previously on galactose, had undergone BIR repair of the DSB and, therefore, would not sustain another <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a>-induced DSB when replated on galactose, and compared their growth on galactose to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants that had not been previously exposed to galactose and, therefore, would undergo DSB induction when plated on galactose. If the growth delay of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants on galactose were simply due to a general sensitivity to galactose, then the presence or absence of a cleavable <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a> cut site would not matter and both types of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants would be equally sensitive to galactose. We found, however, that when the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants that were originally plated on galactose were re-plated on galactose, quantifiable colonies appeared sooner (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS2.pdf">Figure S2</a>; see also <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS8.pdf">Figure S8</a>A), suggesting that the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant was not simply due to a sensitivity to galactose.</p><p id="p-23">To further explore the growth delay in the absence of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> after DSB induction, we examined the growth of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants in the YPH274 genetic background on the DSB-inducing agent bleomycin. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant strain was sensitive bleomycin and, again, formed colonies more slowly than wild-type (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). Growth of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant strain on galactose was comparable with the wild-type (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS3.pdf">Figure S3</a>B), thus, clearly demonstrating a growth defect attributable to DSBs and independent of galactose. Additionally, like the delay in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> BIR assay strain background when plated on galactose, the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant strain on bleomycin was complemented by expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> on a CEN plasmid (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS3.pdf">Figure S3</a>B). Therefore, absence <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> resulted in delayed growth upon DNA damage in two different genetic backgrounds—either via an inducible DSB in the BIR assay strain or by the DNA-damaging agent bleomycin in the YPH274 background.</p></div><div id="sec-12" class="subsection"><h3><em>sea3</em>Δ mutants are not classical recovery mutants</h3><p id="p-24">The delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant BIR assay strain suggested that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em><strong><em>∆</em></strong> mutants may have a delay in recovery post-DNA damage. Recovery is typically defined as resumption of mitosis after repair is completed and the checkpoint is turned off; recovery mutants demonstrate sustained activation of the DDR despite repair of the DSB (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vaze <em>et al.</em> 2002</a>). A variety of proteins have been implicated in recovery and most are associated with the DNA damage checkpoint or repair (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guillemain <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Leroy <em>et al.</em> 2003</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Vaze <em>et al.</em> 2002</a>).</p><p id="p-25">We, therefore, first determined whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant BIR assay strain was able to repair the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a>-induced DSB via BIR. Like wild-type, it repaired via BIR a majority of the time, as tracked by the sensitivity of the colonies to canavanine and hygromycin (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). Additionally, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant repaired the DSB as rapidly as wild-type with repair products appearing in the interval between 8 and 10 hr after DSB induction (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Next we determined whether the mutant had a delay in terminating the DNA damage checkpoint postrepair, which is most frequently monitored by examining the pattern of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> hyperphosphorylation after DSB induction. We found, however, that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant did not have prolonged <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> hyper-phosphorylation as compared to wild-type (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). Taken together, as the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant was able to repair the DSBs and extinguish the DNA damage checkpoint as proficiently as the wild-type, the delay in colony formation post DSB induction was distinct from a classical recovery defect.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loss of Sea3 does not delay break-induced replication (BIR) repair or extinction of the DNA damage checkpoint. (A) Percent repair types observed in wild-type and sea3&#x394; BIR assay strain mutants from (Figure 2B), determined by plating on media containing canavanine or hygromycin. Values represent average of two independent trials. (B) AvaI-digested genomic DNA collected from wild-type and sea3&#x394; BIR assay strain mutants at designated hours postgalactose induction blotted and probed with the CAN1 gene. (C) Rad53 western blot analysis of whole-cell extracts of an equivalent number of wild-type and sea3&#x394; mutant cells were prepared via the trichloroacetic acid method at designated hours postgalactose induction. Image is representative of two independent experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-76344638" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Loss of Sea3 does not delay break-induced replication (BIR) repair or extinction of the DNA damage checkpoint. (A) Percent repair types observed in wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; BIR assay strain mutants from (Figure 2B), determined by plating on media containing canavanine or hygromycin. Values represent average of two independent trials. (B) &lt;em&gt;Ava&lt;/em&gt;I-digested genomic DNA collected from wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; BIR assay strain mutants at designated hours postgalactose induction blotted and probed with the &lt;em&gt;CAN1&lt;/em&gt; gene. (C) Rad53 western blot analysis of whole-cell extracts of an equivalent number of wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; mutant cells were prepared via the trichloroacetic acid method at designated hours postgalactose induction. Image is representative of two independent experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1379/F3.medium.gif" width="326" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/7/1379/F3.medium.gif" width="326" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466441" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Loss of Sea3 does not delay break-induced replication (BIR) repair or extinction of the DNA damage checkpoint. (A) Percent repair types observed in wild-type and <em>sea3</em>Δ BIR assay strain mutants from (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>), determined by plating on media containing canavanine or hygromycin. Values represent average of two independent trials. (B) <em>Ava</em>I-digested genomic DNA collected from wild-type and <em>sea3</em>Δ BIR assay strain mutants at designated hours postgalactose induction blotted and probed with the <em>CAN1</em> gene. (C) Rad53 western blot analysis of whole-cell extracts of an equivalent number of wild-type and <em>sea3</em>Δ mutant cells were prepared via the trichloroacetic acid method at designated hours postgalactose induction. Image is representative of two independent experiments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Sea3 functions through the TORC1 pathway in response to DNA damage</h3><p id="p-27">Given <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a>′s role as a negative regulator of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex (SEACIT), which in turn negatively regulates TORC1 (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Panchaud <em>et al.</em> 2013a</a>,<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">b</a>), we reasoned that the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> BIR assay strain might be the result of hyperrepression of TORC1 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). If so, then deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">IML1</a></em> would rescue the delay. This is precisely what we observed (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Similarly, we found that deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">IML1</a></em> rescued the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant growth delay on bleomycin in the YPH274 strain background (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS4.pdf">Figure S4</a>). Thus, these data are consistent with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> functioning through TORC1 and this regulation of TORC1 impacting colony formation post-DNA repair in the BIR assay strain.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sea3 functions through TORC1, but not autophagy, in response to DNA damage. (A) Proposed effects on TORC1 if Sea3 functions as a negative regulator of the Iml1 complex/SEACIT. (B) Fivefold serial dilutions of wild-type and iml1&#x394;, sea3&#x394; iml1&#x394;, and sea3&#x394; BIR assay strain mutants plated on YPD and YPGal. (C) Fivefold serial dilutions of wild-type and atg5&#x394;, sea3&#x394; atg5&#x394;, and sea3&#x394; BIR assay strain mutants plated on YPD and YPGal." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-76344638" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sea3 functions through TORC1, but not autophagy, in response to DNA damage. (A) Proposed effects on TORC1 if Sea3 functions as a negative regulator of the Iml1 complex/SEACIT. (B) Fivefold serial dilutions of wild-type and &lt;em&gt;iml1&lt;/em&gt;&#x394;, &lt;em&gt;sea3&lt;/em&gt;&#x394; &lt;em&gt;iml1&lt;/em&gt;&#x394;, and &lt;em&gt;sea3&lt;/em&gt;&#x394; BIR assay strain mutants plated on YPD and YPGal. (C) Fivefold serial dilutions of wild-type and &lt;em&gt;atg5&lt;/em&gt;&#x394;, &lt;em&gt;sea3&lt;/em&gt;&#x394; &lt;em&gt;atg5&lt;/em&gt;&#x394;, and &lt;em&gt;sea3&lt;/em&gt;&#x394; BIR assay strain mutants plated on YPD and YPGal.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1379/F4.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/7/1379/F4.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466360" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Sea3 functions through TORC1, but not autophagy, in response to DNA damage. (A) Proposed effects on TORC1 if Sea3 functions as a negative regulator of the Iml1 complex/SEACIT. (B) Fivefold serial dilutions of wild-type and <em>iml1</em>Δ, <em>sea3</em>Δ <em>iml1</em>Δ, and <em>sea3</em>Δ BIR assay strain mutants plated on YPD and YPGal. (C) Fivefold serial dilutions of wild-type and <em>atg5</em>Δ, <em>sea3</em>Δ <em>atg5</em>Δ, and <em>sea3</em>Δ BIR assay strain mutants plated on YPD and YPGal.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">To determine whether <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> and, perhaps, TORC1 signaling were required for growth in response to other stress conditions, we plated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutants in both the YPH274 and BIR assay strain backgrounds on YPD containing a low concentration of glucose (0.25% compared to 2% in standard YPD), high salt (0.5 M NaCl added to standard YPD), or hydrogen peroxide (3 mM H<sub>2</sub>O<sub>2</sub> added to standard YPD), and at high temperature (37°). We found the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant in the YPH274 strain background had a growth delay on medium containing high salt, an effect that was mediated through TORC1 signaling as it was rescued by deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">IML1</a></em> (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS5.pdf">Figure S5</a>A). However, a growth delay was not observed on high salt with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant in the BIR assay strain background (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS5.pdf">Figure S5</a>B), suggesting the phenotype was influenced by strain specific factors and not solely the absence of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a>. In both strain backgrounds, we found that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> was not required for growth in response to any of the other stresses tested (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS5.pdf">Figure S5</a>). Taken together, although there are some strain specific differences, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants experience a growth defect under conditions that induce DSBs and under conditions of high salt in a TORC1-dependent manner.</p><p id="p-30">We next looked for possible targets downstream of TORC1 that might be responsible for the delay in colony formation phenotype. Likely candidates were factors mediating autophagy, which is negatively regulated by TORC1 signaling, and previously identified to be a pathway downstream of the yeast SEA complex (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Dokudovskaya <em>et al.</em> 2011</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Takahara and Maeda 2013</a>). If <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> functioned to promote TORC1 repression of autophagy and, thereby, regulate growth post-DNA repair, then a block in autophagy would rescue the delay observed in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant. However, we found that deletion of <em>ATG5</em>, which encodes a core autophagy factor (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mizushima <em>et al.</em> 1998</a>), had no impact on the delay in the BIR assay strain (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). Thus, the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant was not due to aberrant up-regulation of autophagy but rather due to misregulation of another downstream TORC1 target.</p></div><div id="sec-14" class="subsection"><h3>The growth delay in <em>sea3</em>Δ mutants is rescued by the presence of wild-type <em>TRP1</em></h3><p id="p-31">Both the SEA complex and TORC1 signaling have been linked to amino acid biosynthesis and internal trafficking of amino acid permeases. In the case of the SEA complex, the link has been inferred from genome-wide pairwise fitness screen data (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Beck <em>et al.</em> 1999</a>; <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Dokudovskaya <em>et al.</em> 2011</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Schmidt <em>et al.</em> 1998</a>). Therefore, it was possible that the delay in colony formation after DNA damage was attributable to a change in amino acid requirements. The parental BIR assay strain is auxotrophic for the amino acids tryptophan and leucine as well as the nucleobase uracil due to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em> mutations, respectively. Therefore, we asked whether addition of wild-type copies of these genes impacted the growth of wild-type and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant strains. Introduction of a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> CEN plasmid into the parental wild-type BIR assay strain did not result in a growth delay upon plating on galactose (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS6.pdf">Figure S6</a>A), whereas deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> with a <em>kanMX</em> cassette did (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS6.pdf">Figure S6</a>B), indicating that the growth delay of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> mutant BIR strain on galactose was not due to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> expression. Surprisingly, however, introduction of a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> CEN plasmid rescued the delay of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). The rescue was specific to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em>, as introduction of a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> CEN plasmid did not similarly rescue the delay (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS3.pdf">Figure S3</a>A). Likewise, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> rescued the growth delay observed when the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em>::<em>HPH</em> mutant in the YPH274 (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a></em>-) background was plated on bleomycin, although to a partial extent (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS7.pdf%E2%80%99">Figure S7</a>). These results suggested that the delay in colony formation observed in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutants was due to perturbations in internal levels of tryptophan.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of TRP1, but not exogenous tryptophan, rescues the delay in colony formation in sea3&#x394; mutants. (A) Fivefold serial dilutions of wild-type and sea3&#x394; mutants in the break-induced replication (BIR) assay strain with and without the addition of the TRP1 plasmid pRS414 on YPD and YPGal. The wild-type-in and sea3&#x2206;::LEU2-in strains have been previously experienced HO-induction with galactose and undergone a repair event, rendering them unable to undergo HO-mediated DSB induction upon replating on galactose. (B) Fivefold serial dilutions of the wild-type and sea3&#x394; BIR assay strain mutants plated on YPD, YPGal, YPD + 100 &#x3BC;M tryptophan (Trp) and YPGal + 100 &#x3BC;M Trp. (C) Western blots showing Tat2-3XFlag levels post-galactose induction at the indicated time points. Whole-cell extracts were prepared and blotted with &#x3B1;-Flag. Images are representative of three independent experiments. (D) Fivefold serial dilutions of the wild-type and sea3&#x394; mutants in the BIR assay strain were plated on YPD, YPGal and YPD and YPGal with either 2 or 4 &#x3BC;M quinolinic acid (QA) added. Plates were imaged on day 3, day 4, and day 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-76344638" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Expression of &lt;em&gt;TRP1&lt;/em&gt;, but not exogenous tryptophan, rescues the delay in colony formation in &lt;em&gt;sea3&#x394;&lt;/em&gt; mutants. (A) Fivefold serial dilutions of wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; mutants in the break-induced replication (BIR) assay strain with and without the addition of the &lt;em&gt;TRP1&lt;/em&gt; plasmid pRS414 on YPD and YPGal. The wild-type-&lt;em&gt;in&lt;/em&gt; and &lt;em&gt;sea3&#x2206;&lt;/em&gt;::&lt;em&gt;LEU2-in&lt;/em&gt; strains have been previously experienced HO-induction with galactose and undergone a repair event, rendering them unable to undergo HO-mediated DSB induction upon replating on galactose. (B) Fivefold serial dilutions of the wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; BIR assay strain mutants plated on YPD, YPGal, YPD + 100 &#x3BC;M tryptophan (Trp) and YPGal + 100 &#x3BC;M Trp. (C) Western blots showing Tat2-3XFlag levels post-galactose induction at the indicated time points. Whole-cell extracts were prepared and blotted with &#x3B1;-Flag. Images are representative of three independent experiments. (D) Fivefold serial dilutions of the wild-type and &lt;em&gt;sea3&lt;/em&gt;&#x394; mutants in the BIR assay strain were plated on YPD, YPGal and YPD and YPGal with either 2 or 4 &#x3BC;M quinolinic acid (QA) added. Plates were imaged on day 3, day 4, and day 5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1379/F5.medium.gif" width="378" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/7/1379/F5.medium.gif" width="378" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1379/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466505" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Expression of <em>TRP1</em>, but not exogenous tryptophan, rescues the delay in colony formation in <em>sea3Δ</em> mutants. (A) Fivefold serial dilutions of wild-type and <em>sea3</em>Δ mutants in the break-induced replication (BIR) assay strain with and without the addition of the <em>TRP1</em> plasmid pRS414 on YPD and YPGal. The wild-type-<em>in</em> and <em>sea3∆</em>::<em>LEU2-in</em> strains have been previously experienced HO-induction with galactose and undergone a repair event, rendering them unable to undergo HO-mediated DSB induction upon replating on galactose. (B) Fivefold serial dilutions of the wild-type and <em>sea3</em>Δ BIR assay strain mutants plated on YPD, YPGal, YPD + 100 μM tryptophan (Trp) and YPGal + 100 μM Trp. (C) Western blots showing Tat2-3XFlag levels post-galactose induction at the indicated time points. Whole-cell extracts were prepared and blotted with α-Flag. Images are representative of three independent experiments. (D) Fivefold serial dilutions of the wild-type and <em>sea3</em>Δ mutants in the BIR assay strain were plated on YPD, YPGal and YPD and YPGal with either 2 or 4 μM quinolinic acid (QA) added. Plates were imaged on day 3, day 4, and day 5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Therefore, we examined whether tryptophan availability was contributing to the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant phenotype. Even in rich media, the level of tryptophan is low (measured 26 µg/mL in <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jarolim <em>et al.</em> 2013</a>), so we reasoned that if tryptophan availability were the source of the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant, addition of an excessive amount of tryptophan to the media might rescue the delay. Although we found that both wild-type and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant strains grew slightly more robustly on YPGal supplemented with an additional 100 μM tryptophan compared with YPGal, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant still displayed a growth delay relative to the wild-type (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). Combined with the rescue of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> expression, this suggested that after DNA damage, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants are either unable to import tryptophan into the cell sufficiently or that tryptophan consumption was altered.</p><p id="p-34">To address this further, we examined whether the levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> changed in the absence of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> upon galactose induction in the BIR assay strain. We found that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> levels were comparable throughout the time course in wild-type and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant, with levels declining between 32 and 48 hr (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS9.pdf">Figure S9</a>). Additionally, deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">TAT2</a></em> in the BIR assay strain did not result in a delay in colony formation post-DSB induction (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS10.pdf">Figure S10</a>), suggesting that although the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant appeared to be attributable to insufficient internal tryptophan levels, loss of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> protein was not responsible.</p><p id="p-35">Of note, we also found that the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> BIR assay strain was only observed on rich media as, when the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant was plated on synthetic defined media containing galactose and tryptophan, among the added amino acids (<em>e.g.</em>, -Ura Gal or -Leu Gal minimal media; tryptophan concentration 40 µg/mL), it grew like the wild-type (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS8.pdf">Figure S8</a>). This indicated that, under those nutrient conditions, the cells were sufficient in tryptophan uptake.</p><p id="p-36">One possibility for the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant might relate to the availability of the coenzyme nicotinamide adenosine dinucleotide (NAD+), which is synthesized either via a salvage pathway or <em>de novo</em> from tryptophan via the kynurenine pathway. Notably, NAD+ is used in a wide range of cellular pathways, including those involved in DNA repair and the DDR (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kato and Lin 2014</a>). We hypothesized that the slow growth phenotype of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant post-DNA damage could be due to limiting NAD+ levels resulting from decreased internal tryptophan. Therefore, supplying an intermediate downstream from tryptophan in the <em>de novo</em> synthesis pathway might rescue the delay. Quinolinic acid is one such intermediate and, importantly, can be imported into cells via the high affinity nicotinic acid permease and, thereby, used to increase internal levels of NAD+ (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ohashi <em>et al.</em> 2013</a>). When we plated wild-type and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants in the BIR strain background on media containing galactose and varying amounts of quinolinic acid, we found the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants grew markedly better than without quinolinic acid (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D</a>). Therefore, we conclude tryptophan auxotrophy is synthetic with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutation under DNA damage conditions partially due to a decline in the levels of the coenzyme NAD+.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-37">In this study, we have shown that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> is critical for the recovery of growth post-DNA damage caused by DSBs and in the absence of telomerase. We also have shown that the growth delay observed in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutants after DNA damage is downstream of TORC1 signaling. Though a link between TORC1 signaling and the adaptation to unrepaired DSBs has been reported, this has been shown to be via TORC1 inhibition of autophagy (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dotiwala <em>et al.</em> 2013</a>). Similarly, inhibition of autophagy by TORC1 signaling has been implicated in DNA damage sensitivity (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dyavaiah <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Robert <em>et al.</em> 2011</a>). This study provides evidence that TORC1 signaling has a role in recovery post-DNA damage independent of autophagy, a role related to the availability of tryptophan, and, consequently, the <em>de novo</em> pathway of NAD+ synthesis, and enforced by <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a>.</p><p id="p-38"><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> was placed previously in the SEACAT epistatic group because of its functional redundancy with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">Sea4</a> in response to nutrient conditions (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Panchaud <em>et al.</em> 2013b</a>). Therefore, although, it was previously established that inhibition of TORC1 signaling results in shortened telomeres via alteration of the levels of Ku (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Ungar <em>et al.</em> 2011</a>), the normal telomere length we observed in haploids bearing <em>a <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">SEA2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">SEA4</a></em> gene deletion (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>) may have been due to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">Sea4</a> functioning redundantly to inhibit the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex and support TORC1 signaling as they do in response to nutrients (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Panchaud <em>et al.</em> 2013b</a>).</p><p id="p-39">We also, however, identified a role for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> that could not be compensated for by the continued presence of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000200">Sea4</a>, which is in the recovery following DNA damage, including at telomeres (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, and <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS3.pdf">Figure S3</a>). Additional evidence supports the possibility that members of the SEACAT epistasis group have disparate functions. For example, although <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> are structurally similar and both genetically interact with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2008</a>, <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">2011</a>), <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">Sea3</a> has an RWD domain, structurally similar to an E2 ubiquitin conjugating enzyme, which is not present in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">Sea2</a> (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Dokudovskaya <em>et al.</em> 2011</a>). Additionally, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005498">sea2</a></em>Δ homozygous mutants have a sporulation defect not observed with deletion of other SEA complex member genes [data not shown and (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Briza <em>et al.</em> 2002</a>)]. Together, these results suggest that individual components of the SEACAT epistasis group may function in different circumstances to regulate TORC1 signaling.</p><p id="p-40">The finding that the delay in colony formation in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutants after DNA damage was not caused by defects in the ability or kinetics of repair nor prolonged activation of the DNA damage checkpoint (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>) was surprising because no studies have shown a growth defect post-DNA repair without accompanying <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> hyperphosphorylation. Instead, we found the delay was dependent on tryptophan auxotrophy (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A</a>), uncovering the importance of tryptophan after DNA damage. Consistent with this finding, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a> protein levels increased in both wild-type and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant postbreak induction when <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> hyperphosphorylation was high but declined more slowly, suggesting that, even as the DNA damage checkpoint was alleviated, the demand for tryptophan import remained elevated (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>, <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5C</a>, and <a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS9.pdf">Figure S9</a>).</p><p id="p-41">The tryptophan auxotrophy-dependent delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> BIR assay strain on rich, but not synthetic, media (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS8.pdf">Figure S8</a>) is reminiscent of the growth phenotype of yeast with mutations in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001531">ELM1</a></em>, which encodes a serine/threonine kinase involved in cell growth and division (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Garrett 1997</a>). In previous studies, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001531">elm1</a></em> mutant strains were found to have a different set of phenotypes depending on whether they were auxotrophic or prototrophic for tryptophan. In addition, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001531">elm1</a></em> mutant phenotype was observed on rich media but not on synthetic defined media containing tryptophan. Examination of the activity of several amino acid permeases revealed <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1</a> activity was inappropriately decreased in cells deficient in both <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001531">Elm1</a> and internal tryptophan. Thus, rather than <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">Tat2</a>, misregulation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">Gap1</a>, a known target of TORC1 signaling, may be responsible for the delay in colony formation upon DNA damage in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a>∆</em> mutant. Indeed, a delay in colony formation upon DNA damage was not observed in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005380">tat2</a>∆</em> strain (<a href="http://www.g3journal.org/content/suppl/2015/05/04/g3.115.018721.DC1/FigureS10.pdf">Figure S10</a>), indicating an alternate pathway for tryptophan import upon DNA damage.</p><p id="p-42">In addition to being an essential substrate for protein synthesis, tryptophan is required for the <em>de novo</em> synthesis of NAD+. We found the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant was partially suppressed by the addition of quinolinic acid (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5D</a>), an intermediate downstream from tryptophan in the NAD+ <em>de novo</em> synthesis pathway, suggesting an increased demand for NAD+ postDNA damage. NAD+ has a variety of roles in both cellular growth and DNA transactions; however, the specific activities driving such an increased demand remain unknown. An increase demand for NAD+ synthesis has been reported in the context of DNA damage signaling emanating from the telomere in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> mutant at the semipermissive temperature, reflected by marked up-regulation of expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003839">BNA2</a></em>, which, like tryptophan, is required for <em>de novo</em> NAD+ synthesis. Notably, this was independent of the NAD+-dependent deacetylase <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002200">Sir2</a>, which has known functions at the telomere (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Greenall <em>et al.</em> 2008</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> also was identified in a genome-wide screen as a suppressor of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002379">cdc13</a>-1</em> phenotype at the restrictive temperature (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Addinall <em>et al.</em> 2011</a>). As the addition of quinolinic acid only partially suppressed the delay in colony formation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">sea3</a></em>Δ mutant, additional NAD+-independent pathways could be perturbed post-DNA damage. For example, an increased demand for tryptophan might also be due to an increase in protein synthesis during the DDR. Further studies are needed to determine the NAD+-dependent and -independent pathways underlying the increased demand for tryptophan post-DNA damage.</p><p id="p-43">In summary, these findings reveal a new class of DNA recovery mutants, distinct from those defective in terminating the DDR, that impact the status of tryptophan, an important determinant of growth post-DNA damage. Several of the commonly used yeast strains, including S288c, W303, SEY6210, YPH274, and their derivatives, are auxotrophic for tryptophan as the result of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a></em> mutations. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> is also a commonly used selectable and counterselectable marker. Thus, this work raises the possibility that the DNA damage sensitivity phenotype of some previously identified genes may be a synthetic phenotype with a background <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a></em> mutation. Conversely, mutations rescuing a DNA damage phenotype could be caused by the use of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> as the selectable marker. Addressing whether a DNA damage phenotype manifests if a strain is auxotrophic for tryptophan or other nutrients may provide important insight into the potential pathway it affects in response to DNA damage.</p><p id="p-44">The SEACAT and the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">Iml1</a> complex are functionally conserved in humans. To date, somatic mutations in the human <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">IML1</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003899">SEA1</a></em> homolog, <em>DEPDC5</em>, have been found mutated in cancers (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bar-Peled <em>et al.</em> 2013</a>) and germline mutations have been found in familial autosomal-dominant epilepsy and focal malformations of cortical development (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dibbens <em>et al.</em> 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ishida <em>et al.</em> 2013</a>; Martin <em>et al.</em> 2013; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Picard <em>et al.</em> 2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">D’Gama <em>et al.</em> 2015</a>). It is anticipated that homologs of other members of the SEA complex, including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002535">SEA3</a></em>, will also prove to be clinically relevant.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank James Haber (Brandeis University) for use of the BIR assay strain JRL346, Marco Foiani (University of Milan) for providing the α-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> antibody, and Greg Ira (Baylor College of Medicine) for thoughtful input and use of reagents for this project. We are grateful to Bertuch lab members, especially to Christopher Williams for technical assistance and Nimrat Chatterjee and Ann Sukumar for thoughtful comments and editing of the manuscript. The project described was supported by the National Institutes of Health, National Institute of General Medical Sciences [T32GM008231 supporting E.J.P. and GM077509 to A.A.B.]. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of General Medical Sciences or the National Institutes of Health.</p><p id="p-46">Conceived and designed the experiments: E.J.P. and A.A.B. Performed the experiments: E.J.P. Analyzed data: E.J.P. and A.A.B. Contributed reagents/materials/analysis tools: A.A.B. Wrote the paper: E.J.P. and A.A.B.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-47">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018721/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018721/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-48"><em>Communicating editor: C. S. Hoffman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-08-11"><span class="received-label">Received </span>August 11, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-23"><span class="accepted-label">Accepted </span>April 23, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Polleys and Bertuch</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.1" data-doi="10.1128/MCB.23.21.7566-7584.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iida</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Pressure-induced differential regulation of the two tryptophan permeases Tat1 and Tat2 by ubiquitin ligase Rsp5 and its binding proteins, Bul1 and Bul2.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">7566</span>–<span class="cit-lpage">7584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAbe%26rft.auinit1%253DF.%26rft.volume%253D23%26rft.issue%253D21%26rft.spage%253D7566%26rft.epage%253D7584%26rft.atitle%253DPressure-Induced%2BDifferential%2BRegulation%2Bof%2Bthe%2BTwo%2BTryptophan%2BPermeases%2BTat1%2Band%2BTat2%2Bby%2BUbiquitin%2BLigase%2BRsp5%2Band%2BIts%2BBinding%2BProteins%252C%2BBul1%2Band%2BBul2%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.21.7566-7584.2003%26rft_id%253Dinfo%253Apmid%252F14560004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yMS83NTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.2" data-doi="10.1534/genetics.108.092577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Downey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zubko</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genomewide suppressor and enhancer analysis of cdc13–1 reveals varied cellular processes influencing telomere capping in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">2251</span>–<span class="cit-lpage">2266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092577%26rft_id%253Dinfo%253Apmid%252F18845848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIyNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.3" data-doi="10.1371/journal.pgen.1001362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holstein</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAddinall%26rft.auinit1%253DS.%2BG.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1001362%26rft.epage%253De1001362%26rft.atitle%253DQuantitative%2Bfitness%2Banalysis%2Bshows%2Bthat%2BNMD%2Bproteins%2Band%2Bmany%2Bother%2Bprotein%2Bcomplexes%2Bsuppress%2Bor%2Benhance%2Bdistinct%2Btelomere%2Bcap%2Bdefects.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001362%26rft_id%253Dinfo%253Apmid%252F21490951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21490951&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.4" data-doi="10.1073/pnas.0401263101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askree</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yehuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolikov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genome-wide screen for <em>Saccharomyces cerevisiae</em> deletion mutants that affect telomere length.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">8658</span>–<span class="cit-lpage">8663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401263101%26rft_id%253Dinfo%253Apmid%252F15161972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzIzLzg2NTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.5" data-doi="10.1126/science.1232044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bar-Peled</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chantranupong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherniack</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ottina</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A tumor suppressor complex with GAP activity for the Rag GTPases that signal amino acid sufficiency to mTORC1.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">340</span>: <span class="cit-fpage">1100</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBar-Peled%26rft.auinit1%253DL.%26rft.volume%253D340%26rft.issue%253D6136%26rft.spage%253D1100%26rft.epage%253D1106%26rft.atitle%253DA%2BTumor%2BSuppressor%2BComplex%2Bwith%2BGAP%2BActivity%2Bfor%2Bthe%2BRag%2BGTPases%2BThat%2BSignal%2BAmino%2BAcid%2BSufficiency%2Bto%2BmTORC1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1232044%26rft_id%253Dinfo%253Apmid%252F23723238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDAvNjEzNi8xMTAwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.6" data-doi="10.1083/jcb.146.6.1227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Starvation induces vacuolar targeting and degradation of the tryptophan permease in yeast.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.146.6.1227%26rft_id%253Dinfo%253Apmid%252F10491387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNDYvNi8xMjI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.7" data-doi="10.1534/genetics.166.4.1651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertuch</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">EXO1 contributes to telomere maintenance in both telomerase-proficient and telomerase-deficient <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBertuch%26rft.auinit1%253DA.%2BA.%26rft.volume%253D166%26rft.issue%253D4%26rft.spage%253D1651%26rft.epage%253D1659%26rft.atitle%253DEXO1%2BContributes%2Bto%2BTelomere%2BMaintenance%2Bin%2BBoth%2BTelomerase-Proficient%2Band%2BTelomerase-Deficient%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.4.1651%26rft_id%253Dinfo%253Apmid%252F15126387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni80LzE2NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.8" data-doi="10.1093/nar/24.23.4639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boulton</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Identification of a <em>Saccharomyces cerevisiae</em> Ku80 homologue: roles in DNA double strand break rejoining and in telomeric maintenance.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">4639</span>–<span class="cit-lpage">4648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.23.4639%26rft_id%253Dinfo%253Apmid%252F8972848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8yMy80NjM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.9" data-doi="10.1002/yea.843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Briza</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogengruber</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutzler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munsterkotter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Systematic analysis of sporulation phenotypes in 624 non-lethal homozygous deletion strains of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBriza%26rft.auinit1%253DP.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D403%26rft.epage%253D422%26rft.atitle%253DSystematic%2Banalysis%2Bof%2Bsporulation%2Bphenotypes%2Bin%2B624%2Bnon-lethal%2Bhomozygous%2Bdeletion%2Bstrains%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.843%26rft_id%253Dinfo%253Apmid%252F11921089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.843&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11921089&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174648000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-5.7.1379.10" data-doi="10.1534/g3.111.000216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H.Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S.G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oexle</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taschuk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide analysis to identify pathways affecting telomere-initiated senescence in budding yeast.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">1</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DChang%26rft.auinit1%253DH.-Y.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D197%26rft.epage%253D208%26rft.atitle%253DGenome-Wide%2BAnalysis%2Bto%2BIdentify%2BPathways%2BAffecting%2BTelomere-Initiated%2BSenescence%2Bin%2BBudding%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000216%26rft_id%253Dinfo%253Apmid%252F22384331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.11" data-doi="10.1128/MCB.25.13.5738-5751.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiolo</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carotenuto</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maffioletti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrini</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foiani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Srs2 and Sgs1 DNA helicases associate with Mre11 in different subcomplexes following checkpoint activation and CDK1-mediated Srs2 phosphorylation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">5738</span>–<span class="cit-lpage">5751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DChiolo%26rft.auinit1%253DI.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D5738%26rft.epage%253D5751%26rft.atitle%253DSrs2%2Band%2BSgs1%2BDNA%2BHelicases%2BAssociate%2Bwith%2BMre11%2Bin%2BDifferent%2BSubcomplexes%2Bfollowing%2BCheckpoint%2BActivation%2Band%2BCDK1-Mediated%2BSrs2%2BPhosphorylation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.13.5738-5751.2005%26rft_id%253Dinfo%253Apmid%252F15964827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMy81NzM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.12" data-doi="10.1002/ana.24357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Gama</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couto</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mammalian target of rapamycin pathway mutations cause hemimegalencephaly and focal cortical dysplasia.</span> <abbr class="cit-jnl-abbrev">Ann. Neurol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BNeurol.%26rft.volume%253D77%26rft.spage%253D720%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fana.24357%26rft_id%253Dinfo%253Apmid%252F25599672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ana.24357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25599672&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.13" data-doi="10.1038/ng.2599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dibbens</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vries</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donatello</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heron</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodgson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutations in DEPDC5 cause familial focal epilepsy with variable foci.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">546</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D546%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2599%26rft_id%253Dinfo%253Apmid%252F23542697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2599&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23542697&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-other" id="cit-5.7.1379.14" data-doi="10.1074/mcp.M110.006478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dokudovskaya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waharte</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlessinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pieper</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">D.P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A conserved coatomer-related complex containing Sec13 and Seh1 dynamically associates with the vacuole in <em>Saccharomyces cerevisiae</em>.</span> <span class="cit-source">Mol Cell Proteomics</span> <span class="cit-vol">10</span>: <span class="cit-fpage">M110.006478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M110.006478%26rft_id%253Dinfo%253Apmid%252F21454883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjE2OiIxMC82L00xMTAuMDA2NDc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.15" data-doi="10.1073/pnas.1218065109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dotiwala</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eapen</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arbel-Eden</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranade</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNA damage checkpoint triggers autophagy to regulate the initiation of anaphase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E41</span>–<span class="cit-lpage">E49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1218065109%26rft_id%253Dinfo%253Apmid%252F23169651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMS9FNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.16" data-doi="10.1158/1541-7786.MCR-10-0473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dyavaiah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chittur</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begley</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Autophagy-dependent regulation of the DNA damage response protein ribonucleotide reductase 1.</span> <abbr class="cit-jnl-abbrev">Mol. Cancer Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F1541-7786.MCR-10-0473%26rft_id%253Dinfo%253Apmid%252F21343333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibW9sY2FucmVzIjtzOjU6InJlc2lkIjtzOjc6IjkvNC80NjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.17" data-doi="10.1128/MCB.14.2.923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foiani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plevani</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The B subunit of the DNA polymerase alpha-primase complex in <em>Saccharomyces cerevisiae</em> executes an essential function at the initial stage of DNA replication.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFoiani%26rft.auinit1%253DM%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D923%26rft.epage%253D933%26rft.atitle%253DThe%2BB%2Bsubunit%2Bof%2Bthe%2BDNA%2Bpolymerase%2Balpha-primase%2Bcomplex%2Bin%2BSaccharomyces%2Bcerevisiae%2Bexecutes%2Ban%2Bessential%2Bfunction%2Bat%2Bthe%2Binitial%2Bstage%2Bof%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.2.923%26rft_id%253Dinfo%253Apmid%252F8289832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvOTIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.18" data-doi="10.1046/j.1365-2958.1997.6231990.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrett</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The control of morphogenesis in Saccharomyces cerevisiae by Elm1 kinase is responsive to RAS/cAMP pathway activity and tryptophan availability.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DGarrett%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D809%26rft.epage%253D820%26rft.atitle%253DThe%2Bcontrol%2Bof%2Bmorphogenesis%2Bin%2BSaccharomyces%2Bcerevisiae%2Bby%2BElm1%2Bkinase%2Bis%2Bresponsive%2Bto%2BRAS%252FcAMP%2Bpathway%2Bactivity%2Band%2Btryptophan%2Bavailability.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1997.6231990.x%26rft_id%253Dinfo%253Apmid%252F9427410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1997.6231990.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9427410&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.19" data-doi="10.1186/gb-2008-9-10-r146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swan</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome wide analysis of the response to uncapped telomeres in budding yeast reveals a novel role for the NAD+ biosynthetic gene BNA2 in chromosome end protection.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGreenall%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253DR146%26rft.epage%253DR146%26rft.atitle%253DA%2Bgenome%2Bwide%2Banalysis%2Bof%2Bthe%2Bresponse%2Bto%2Buncapped%2Btelomeres%2Bin%2Bbudding%2Byeast%2Breveals%2Ba%2Bnovel%2Brole%2Bfor%2Bthe%2BNAD%252B%2Bbiosynthetic%2Bgene%2BBNA2%2Bin%2Bchromosome%2Bend%2Bprotection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-10-r146%26rft_id%253Dinfo%253Apmid%252F18828915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-10-r146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18828915&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.20" data-doi="10.1128/MCB.00863-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guillemain</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mechanisms of checkpoint kinase Rad53 inactivation after a double-strand break in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3378</span>–<span class="cit-lpage">3389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00863-06%26rft_id%253Dinfo%253Apmid%252F17325030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI3LzkvMzM3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.21" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.22" data-doi="10.1038/415180a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruhler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heilbut</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHo%26rft.auinit1%253DY.%26rft.volume%253D415%26rft.issue%253D6868%26rft.spage%253D180%26rft.epage%253D183%26rft.atitle%253DSystematic%2Bidentification%2Bof%2Bprotein%2Bcomplexes%2Bin%2BSaccharomyces%2Bcerevisiae%2Bby%2Bmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F415180a%26rft_id%253Dinfo%253Apmid%252F11805837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/415180a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11805837&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173159300044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.23" data-doi="10.1038/ng.2601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishida</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudolf</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noe</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achaz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutations of DEPDC5 cause autosomal dominant focal epilepsies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2601%26rft_id%253Dinfo%253Apmid%252F23542701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23542701&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-5.7.1379.24" data-doi="10.1534/g3.113.007971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarolim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillay</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gee</span>  <span class="cit-name-given-names">A.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phrakaysone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> genes involved in survival of heat shock.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">2321</span>–<span class="cit-lpage">2333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007971%26rft_id%253Dinfo%253Apmid%252F24142923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTIvMjMyMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.25" data-doi="10.1016/j.dnarep.2014.07.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Regulation of NAD+ metabolism, signaling and compartmentalization in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D23%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2014.07.009%26rft_id%253Dinfo%253Apmid%252F25096760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2014.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25096760&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greider</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">RAD50 and RAD51 define two pathways that collaborate to maintain telomeres in the absence of telomerase.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLe%26rft.auinit1%253DS.%26rft.volume%253D152%26rft.issue%253D1%26rft.spage%253D143%26rft.epage%253D152%26rft.atitle%253DRAD50%2Band%2BRAD51%2BDefine%2BTwo%2BPathways%2BThat%2BCollaborate%2Bto%2BMaintain%2BTelomeres%2Bin%2Bthe%2BAbsence%2Bof%2BTelomerase%26rft_id%253Dinfo%253Apmid%252F10224249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUyLzEvMTQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lendvay</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sah</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Senescence mutants of <em>Saccharomyces cerevisiae</em> with a defect in telomere replication identify three additional EST genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1399</span>–<span class="cit-lpage">1412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLendvay%26rft.auinit1%253DT.%2BS.%26rft.volume%253D144%26rft.issue%253D4%26rft.spage%253D1399%26rft.epage%253D1412%26rft.atitle%253DSenescence%2BMutants%2Bof%2BSaccharomyces%2Bcerevisiae%2BWith%2Ba%2BDefect%2Bin%2BTelomere%2BReplication%2BIdentify%2BThree%2BAdditional%2BEST%2BGenes%26rft_id%253Dinfo%253Apmid%252F8978029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC80LzEzOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.28" data-doi="10.1016/S1097-2765(03)00058-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaze</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochsenbein</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerois</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">PP2C phosphatases Ptc2 and Ptc3 are required for DNA checkpoint inactivation after a double-strand break.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLeroy%26rft.auinit1%253DC.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D827%26rft.epage%253D835%26rft.atitle%253DPP2C%2Bphosphatases%2BPtc2%2Band%2BPtc3%2Bare%2Brequired%2Bfor%2BDNA%2Bcheckpoint%2Binactivation%2Bafter%2Ba%2Bdouble-strand%2Bbreak.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900058-3%26rft_id%253Dinfo%253Apmid%252F12667463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00058-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12667463&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182049700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.29" data-doi="10.1016/S1097-2765(02)00636-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loewith</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacinto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crespo</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Two TOR complexes, only one of which is rapamycin sensitive, have distinct roles in cell growth control.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLoewith%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D457%26rft.epage%253D468%26rft.atitle%253DTwo%2BTOR%2Bcomplexes%252C%2Bonly%2Bone%2Bof%2Bwhich%2Bis%2Brapamycin%2Bsensitive%252C%2Bhave%2Bdistinct%2Broles%2Bin%2Bcell%2Bgrowth%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900636-6%26rft_id%253Dinfo%253Apmid%252F12408816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00636-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12408816&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178330900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.30" data-doi="10.1016/0092-8674(89)90132-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A mutant with a defect in telomere elongation leads to senescence in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLundblad%26rft.auinit1%253DV.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D633%26rft.epage%253D643%26rft.atitle%253DA%2Bmutant%2Bwith%2Ba%2Bdefect%2Bin%2Btelomere%2Belongation%2Bleads%2Bto%2Bsenescence%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990132-3%26rft_id%253Dinfo%253Apmid%252F2655926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90132-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2655926&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U743600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.31" data-doi="10.1016/0092-8674(93)90234-H"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackburn</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">An alternative pathway for yeast telomere maintenance rescues est1- senescence.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLundblad%26rft.auinit1%253DV.%26rft.volume%253D73%26rft.issue%253D2%26rft.spage%253D347%26rft.epage%253D360%26rft.atitle%253DAn%2Balternative%2Bpathway%2Bfor%2Byeast%2Btelomere%2Bmaintenance%2Brescues%2Best1-%2Bsenescence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990234-H%26rft_id%253Dinfo%253Apmid%252F8477448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90234-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8477448&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KY50500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.32" data-doi="10.1038/nature06047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lydeard</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Break-induced replication and telomerase-independent telomere maintenance require Pol32.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">448</span>: <span class="cit-fpage">820</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLydeard%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D448%26rft.issue%253D7155%26rft.spage%253D820%26rft.epage%253D823%26rft.atitle%253DBreak-induced%2Breplication%2Band%2Btelomerase-independent%2Btelomere%2Bmaintenance%2Brequire%2BPol32.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06047%26rft_id%253Dinfo%253Apmid%252F17671506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17671506&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248754200043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.33" data-doi="10.1371/journal.pgen.1000973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lydeard</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipkin-Moore</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eapen</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sgs1 and exo1 redundantly inhibit break-induced replication and de novo telomere addition at broken chromosome ends.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLydeard%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000973%26rft.epage%253De1000973%26rft.atitle%253DSgs1%2Band%2Bexo1%2Bredundantly%2Binhibit%2Bbreak-induced%2Breplication%2Band%2Bde%2Bnovo%2Btelomere%2Baddition%2Bat%2Bbroken%2Bchromosome%2Bends.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000973%26rft_id%253Dinfo%253Apmid%252F20523895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000973&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20523895&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.7.1379.34" data-doi="10.1111/cge.12311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meloche</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rioux</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmant</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A recurrent mutation in DEPDC5 predisposes to focal epilepsies in the French-Canadian population.</span> <abbr class="cit-jnl-abbrev">Clin. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">570</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BGenet.%26rft.volume%253D86%26rft.spage%253D570%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fcge.12311%26rft_id%253Dinfo%253Apmid%252F24283814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/cge.12311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24283814&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.35" data-doi="10.1038/26506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mizushima</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A protein conjugation system essential for autophagy.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">395</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMizushima%26rft.auinit1%253DN.%26rft.volume%253D395%26rft.issue%253D6700%26rft.spage%253D395%26rft.epage%253D398%26rft.atitle%253DA%2Bprotein%2Bconjugation%2Bsystem%2Bessential%2Bfor%2Bautophagy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F26506%26rft_id%253Dinfo%253Apmid%252F9759731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/26506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9759731&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076083800057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.36" data-doi="10.1128/EC.00339-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murata</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Secretion of quinolinic acid, an intermediate in the kynurenine pathway, for utilization in NAD+ biosynthesis in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00339-12%26rft_id%253Dinfo%253Apmid%252F23457190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMi81LzY0OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.37" data-doi="10.1126/scisignal.2004112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panchaud</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peli-Gulli</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Virgilio</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Amino acid deprivation inhibits TORC1 through a GTPase-activating protein complex for the Rag family GTPase Gtr1.</span> <abbr class="cit-jnl-abbrev">Sci. Signal.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">ra42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DPanchaud%26rft.auinit1%253DN.%26rft.volume%253D6%26rft.issue%253D277%26rft.spage%253Dra42%26rft.epage%253Dra42%26rft.atitle%253DAmino%2BAcid%2BDeprivation%2BInhibits%2BTORC1%2BThrough%2Ba%2BGTPase-Activating%2BProtein%2BComplex%2Bfor%2Bthe%2BRag%2BFamily%2BGTPase%2BGtr1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscisignal.2004112%26rft_id%253Dinfo%253Apmid%252F23716719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6MTA6IjYvMjc3L3JhNDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.38" data-doi="10.4161/cc.26000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panchaud</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peli-Gulli</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Virgilio</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">SEACing the GAP that nEGOCiates TORC1 activation: evolutionary conservation of Rag GTPase regulation.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2948</span>–<span class="cit-lpage">2952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D12%26rft.spage%253D2948%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.26000%26rft_id%253Dinfo%253Apmid%252F23974112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.26000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23974112&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330528200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.39" data-doi="10.1212/WNL.0000000000000488"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makrythanasis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishida</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bellescize</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">DEPDC5 mutations in families presenting as autosomal dominant nocturnal frontal lobe epilepsy.</span> <abbr class="cit-jnl-abbrev">Neurology</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">2101</span>–<span class="cit-lpage">2106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurology%26rft.volume%253D82%26rft.spage%253D2101%26rft_id%253Dinfo%253Adoi%252F10.1212%252FWNL.0000000000000488%26rft_id%253Dinfo%253Apmid%252F24814846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1212/WNL.0000000000000488&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24814846&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.40" data-doi="10.1038/35079641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rizki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundblad</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Defects in mismatch repair promote telomerase-independent proliferation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRizki%26rft.auinit1%253DA.%26rft.volume%253D411%26rft.issue%253D6838%26rft.spage%253D713%26rft.epage%253D716%26rft.atitle%253DDefects%2Bin%2Bmismatch%2Brepair%2Bpromote%2Btelomerase-independent%2Bproliferation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35079641%26rft_id%253Dinfo%253Apmid%252F11395777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35079641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11395777&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.41" data-doi="10.1038/nature09803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanoli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiolo</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shubassi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">HDACs link the DNA damage response, processing of double-strand breaks and autophagy.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRobert%26rft.auinit1%253DT.%26rft.volume%253D471%26rft.issue%253D7336%26rft.spage%253D74%26rft.epage%253D79%26rft.atitle%253DHDACs%2Blink%2Bthe%2BDNA%2Bdamage%2Bresponse%252C%2Bprocessing%2Bof%2Bdouble-strand%2Bbreaks%2Band%2Bautophagy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09803%26rft_id%253Dinfo%253Apmid%252F21368826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21368826&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287924100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.42" data-doi="10.1093/emboj/17.23.6924"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The TOR nutrient signalling pathway phosphorylates NPR1 and inhibits turnover of the tryptophan permease.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">6924</span>–<span class="cit-lpage">6931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DSchmidt%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D6924%26rft.epage%253D6931%26rft.atitle%253DThe%2BTOR%2Bnutrient%2Bsignalling%2Bpathway%2Bphosphorylates%2BNPR1%2Band%2Binhibits%2Bturnover%2Bof%2Bthe%2Btryptophan%2Bpermease%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.23.6924%26rft_id%253Dinfo%253Apmid%252F9843498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTcvMjMvNjkyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.43" data-doi="10.1128/MCB.00290-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thimmaiah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TOR signaling is a determinant of cell survival in response to DNA damage.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">7007</span>–<span class="cit-lpage">7017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00290-07%26rft_id%253Dinfo%253Apmid%252F17698581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8yMC83MDA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.44" data-doi="10.1126/science.7545955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">TLC1: template RNA component of <em>Saccharomyces cerevisiae</em> telomerase.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSinger%26rft.auinit1%253DM.%26rft.volume%253D266%26rft.issue%253D5184%26rft.spage%253D404%26rft.epage%253D409%26rft.atitle%253DTLC1%253A%2Btemplate%2BRNA%2Bcomponent%2Bof%2BSaccharomyces%2Bcerevisiae%2Btelomerase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7545955%26rft_id%253Dinfo%253Apmid%252F7545955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjYvNTE4NC80MDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.45" data-doi="10.1093/jb/mvt047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionarily conserved regulation of TOR signalling.</span> <abbr class="cit-jnl-abbrev">J. Biochem.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiochem.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjb%252Fmvt047%26rft_id%253Dinfo%253Apmid%252F23698095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamJpb2NoZW0iO3M6NToicmVzaWQiO3M6NzoiMTU0LzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.46" data-doi="10.1128/MCB.19.12.8083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakian</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Telomere-telomere recombination is an efficient bypass pathway for telomere maintenance in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">8083</span>–<span class="cit-lpage">8093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTeng%26rft.auinit1%253DS.-C.%26rft.volume%253D19%26rft.issue%253D12%26rft.spage%253D8083%26rft.epage%253D8093%26rft.atitle%253DTelomere-Telomere%2BRecombination%2BIs%2Ban%2BEfficient%2BBypass%2BPathway%2Bfor%2BTelomere%2BMaintenance%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.12.8083%26rft_id%253Dinfo%253Apmid%252F10567534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi84MDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.47" data-doi="10.1016/j.cub.2011.11.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ungar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harari</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toren</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Tor complex 1 controls telomere length by affecting the level of Ku.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2115</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DUngar%26rft.auinit1%253DL.%26rft.volume%253D21%26rft.issue%253D24%26rft.spage%253D2115%26rft.epage%253D2120%26rft.atitle%253DTor%2Bcomplex%2B1%2Bcontrols%2Btelomere%2Blength%2Bby%2Baffecting%2Bthe%2Blevel%2Bof%2BKu.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.11.024%26rft_id%253Dinfo%253Apmid%252F22169538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.11.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22169538&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.48" data-doi="10.1016/S1097-2765(02)00593-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaze</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellicioli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ira</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liberi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recovery from checkpoint-mediated arrest after repair of a double-strand break requires Srs2 helicase.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DVaze%26rft.auinit1%253DM.%2BB.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D373%26rft.epage%253D385%26rft.atitle%253DRecovery%2Bfrom%2Bcheckpoint-mediated%2Barrest%2Bafter%2Brepair%2Bof%2Ba%2Bdouble-strand%2Bbreak%2Brequires%2BSrs2%2Bhelicase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900593-2%26rft_id%253Dinfo%253Apmid%252F12191482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00593-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12191482&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177584700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1379.49" data-doi="10.1038/nrm3025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zoncu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Efeyan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatini</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">mTOR: from growth signal integration to cancer, diabetes and ageing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DZoncu%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D35%26rft.atitle%253DmTOR%253A%2Bfrom%2Bgrowth%2Bsignal%2Bintegration%2Bto%2Bcancer%252C%2Bdiabetes%2Band%2Bageing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3025%26rft_id%253Dinfo%253Apmid%252F21157483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21157483&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1379.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285544000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/7/1379.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1371" title="DPY-17 and MUA-3 Interact for Connective Tissue-Like Tissue Integrity in Caenorhabditis elegans: A Model for Marfan Syndrome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1391" title="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/7"><span>Volume 5 Issue 7, July 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/7.cover-source.jpg?itok=IZbYpE05" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1379?cited-by=yes&amp;legid=ggg;5/7/1379" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nrcx_z9L7KdO_GtgCtGfdZZvjKGaRGVqahoHm8868aw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/7/1379" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/7/1379" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/7/1379" class="active" data-icon-position="" data-hide-link-title="0">Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459983" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-oN2veaNqJIUIHXQEDurKUmh_sbm-aUn5YkO5GRLvUF4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1379.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/7/1379" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xjz4NH51Em0O7-Gy43ftv9M1vCbj2bBXlfIMDeMQ8N4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/7/1379" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1379.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459983" id="node459983--2" data-pisa="ggg;5/7/1379" data-pisa-master="ggg;g3.115.018721" data-apath="/ggg/5/7/1379.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6547-3624" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erica J. Polleys</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1864-8502" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison A. Bertuch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1379-1389;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018721 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459983--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erica J. Polleys </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erica%2BJ.%2BPolleys%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polleys%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErica%2BJ.%2BPolleys%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6547-3624" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erica J. Polleys</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alison A. Bertuch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics and Department of Molecular & Human Genetics, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BA.%2BBertuch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertuch%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BA.%2BBertuch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1864-8502" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison A. Bertuch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:abertuch@bcm.edu" class="" data-icon-position="" data-hide-link-title="0">abertuch@bcm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/7/1379" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/7/1379&amp;title=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/7/1379&amp;title=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/7/1379&amp;title=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/7/1379&amp;text=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/7/1379&amp;title=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/7/1379&amp;t=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/7/1379&amp;title=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/7/1379&amp;title=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459983/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459983" id="node459983--3" data-pisa="ggg;5/7/1379" data-pisa-master="ggg;g3.115.018721" data-apath="/ggg/5/7/1379.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6547-3624" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Erica J. Polleys</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1864-8502" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison A. Bertuch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1379-1389;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018721 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459983--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Erica J. Polleys </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erica%2BJ.%2BPolleys%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Polleys%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AErica%2BJ.%2BPolleys%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6547-3624" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Erica J. Polleys</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alison A. Bertuch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Integrative Molecular and Biomedical Sciences Graduate Program, Baylor College of Medicine, Houston, Texas 77030</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pediatrics and Department of Molecular & Human Genetics, Baylor College of Medicine, Houston, Texas 77030</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BA.%2BBertuch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertuch%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BA.%2BBertuch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1864-8502" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison A. Bertuch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:abertuch@bcm.edu" class="" data-icon-position="" data-hide-link-title="0">abertuch@bcm.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1379&amp;rft.epage=1389&amp;rft.atitle=Tryptophan-Dependent%20Control%20of%20Colony%20Formation%20After%20DNA%20Damage%20via%20Sea3-Regulated%20TORC1%20Signaling%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=5&amp;rft.issue=7&amp;rft.date=2015-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Polleys%2C+Erica+J.&amp;rft.au=Bertuch%2C+Alison+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459983/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459983/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459983/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459983/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459983/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459983/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459983/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459983/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459983/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459983/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459983/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459983/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/7/1379" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/7/1379.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/7/1379.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/7/1379.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/7/1379.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
