<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/7/1463" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/7/1463.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/7/1463.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/7/1463.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/7/1463.atom" />
<meta name="HW.pisa" content="ggg;5/7/1463" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.115.017905" />
<meta name="DC.Date" content="2015-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Glazer et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.115.026880" />
<meta name="DC.Description" content="Marine populations of the threespine stickleback ( Gasterosteus aculeatus ) have repeatedly colonized and rapidly adapted to freshwater habitats, providing a powerful system to map the genetic architecture of evolved traits. Here, we developed and applied a binned genotyping-by-sequencing (GBS) method to build dense genome-wide linkage maps of sticklebacks using two large marine by freshwater F2 crosses of more than 350 fish each. The resulting linkage maps significantly improve the genome assembly by anchoring 78 new scaffolds to chromosomes, reorienting 40 scaffolds, and rearranging scaffolds in 4 locations. In the revised genome assembly, 94.6% of the assembly was anchored to a chromosome. To assess linkage map quality, we mapped quantitative trait loci (QTL) controlling lateral plate number, which mapped as expected to a 200-kb genomic region containing Ectodysplasin , as well as a chromosome 7 QTL overlapping a previously identified modifier QTL. Finally, we mapped eight QTL controlling convergently evolved reductions in gill raker length in the two crosses, which revealed that this classic adaptive trait has a surprisingly modular and nonparallel genetic basis." />
<meta name="DC.Contributor" content="Andrew M. Glazer" />
<meta name="DC.Contributor" content="Emily E. Killingbeck" />
<meta name="DC.Contributor" content="Therese Mitros" />
<meta name="DC.Contributor" content="Daniel S. Rokhsar" />
<meta name="DC.Contributor" content="Craig T. Miller" />
<meta name="article:published_time" content="2015-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Marine populations of the threespine stickleback (&lt;i&gt;Gasterosteus aculeatus&lt;/i&gt;) have repeatedly colonized and rapidly adapted to freshwater habitats, providing a powerful system to map the genetic architecture of evolved traits. Here, we developed and applied a binned genotyping-by-sequencing (GBS) method to build dense genome-wide linkage maps of sticklebacks using two large marine by freshwater F2 crosses of more than 350 fish each. The resulting linkage maps significantly improve the genome assembly by anchoring 78 new scaffolds to chromosomes, reorienting 40 scaffolds, and rearranging scaffolds in 4 locations. In the revised genome assembly, 94.6% of the assembly was anchored to a chromosome. To assess linkage map quality, we mapped quantitative trait loci (QTL) controlling lateral plate number, which mapped as expected to a 200-kb genomic region containing &lt;i&gt;Ectodysplasin&lt;/i&gt;, as well as a chromosome 7 QTL overlapping a previously identified modifier QTL. Finally, we mapped eight QTL controlling convergently evolved reductions in gill raker length in the two crosses, which revealed that this classic adaptive trait has a surprisingly modular and nonparallel genetic basis.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/07/01" />
<meta name="citation_mjid" content="ggg;5/7/1463" />
<meta name="citation_id" content="5/7/1463" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/7/1463" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/7/1463.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/7/1463.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/7/1463.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017905" />
<meta name="citation_pmid" content="26044731" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1463" />
<meta name="citation_lastpage" content="1472" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrew M. Glazer" />
<meta name="citation_author_institution" content="Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720" />
<meta name="citation_author" content="Emily E. Killingbeck" />
<meta name="citation_author_institution" content="Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720" />
<meta name="citation_author" content="Therese Mitros" />
<meta name="citation_author_institution" content="Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720" />
<meta name="citation_author" content="Daniel S. Rokhsar" />
<meta name="citation_author_institution" content="Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720" />
<meta name="citation_author" content="Craig T. Miller" />
<meta name="citation_author_institution" content="Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720" />
<meta name="citation_author_email" content="ctmiller@berkeley.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Andolfatto;citation_author=D. Davison;citation_author=D. Erezyilmaz;citation_author=T. T. Hu;citation_author=J. Mast;citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping.;citation_volume=21;citation_year=2011;citation_pmid=21233398;citation_doi=10.1101/gr.115402.110" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. E. Arnegard;citation_author=M. D. McGee;citation_author=B. Matthews;citation_author=K. B. Marchinko;citation_author=G. L. Conte;citation_title=Genetics of ecological divergence during speciation.;citation_pages=307-311;citation_volume=511;citation_year=2014;citation_pmid=24909991;citation_doi=10.1038/nature13301" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_title=The Evolutionary Biology of the Threespine Stickleback;citation_author=M. A. Bell;citation_author=S. A. Foster;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. Berner;citation_author=D. Moser;citation_author=M. Roesti;citation_author=H. Buescher;citation_author=W. Salzburger;citation_title=Genetic architecture of skeletal evolution in European lake and stream stickleback.;citation_pages=1792-1805;citation_volume=68;citation_year=2014;citation_pmid=24571250;citation_doi=10.1111/evo.12390" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_author=B.M. Carlson;citation_author=S.W. Onusko;citation_author=J.B. Gross;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=Y. F. Chan;citation_author=M. E. Marks;citation_author=F. C. Jones;citation_author=G. Villarreal;citation_author=M. D. Shapiro;citation_title=Adaptive evolution of pelvic reduction in sticklebacks by recurrent deletion of a Pitx1 enhancer.;citation_volume=327;citation_year=2010;citation_pmid=20007865;citation_doi=10.1126/science.1182213" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Z. L. Chen;citation_author=B. B. Wang;citation_author=X. M. Dong;citation_author=H. Liu;citation_author=L. H. Ren;citation_title=An ultra-high density bin-map for rapid QTL mapping for tassel and ear architecture in a large F-2 maize population.;citation_pages=433;citation_volume=15;citation_year=2014;citation_pmid=24898122;citation_doi=10.1186/1471-2164-15-433" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=P. F. Colosimo;citation_author=C. L. Peichel;citation_author=K. Nereng;citation_author=B. K. Blackman;citation_author=M. D. Shapiro;citation_title=The genetic architecture of parallel armor plate reduction in threespine sticklebacks.;citation_pages=635-641;citation_volume=2;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. F. Colosimo;citation_author=K. E. Hosemann;citation_author=S. Balabhadra;citation_author=G. Villarreal;citation_author=M. Dickson;citation_title=Widespread Parallel Evolution in Sticklebacks by Repeated Fixation of Ectodysplasin Alleles;citation_pages=1928-1933;citation_volume=307;citation_year=2005;citation_issue=5717;citation_pmid=15790847;citation_doi=10.1126/science.1107239" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. A. Cresko;citation_author=A. Amores;citation_author=C. Wilson;citation_author=J. Murphy;citation_author=M. Currey;citation_title=Parallel genetic basis for repeated evolution of armor loss in Alaskan threespine stickleback populations.;citation_volume=101;citation_year=2004;citation_pmid=15069186;citation_doi=10.1073/pnas.0308479101" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. Day;citation_author=J. Pritchard;citation_author=D. Schluter;citation_title=A comparison of two sticklebacks.;citation_pages=1723-1734;citation_volume=48;citation_year=1994;citation_doi=10.2307/2410260" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Evodevo;citation_author=A. M. Glazer;citation_author=P. A. Cleves;citation_author=P. A. Erickson;citation_author=A. Y. Lam;citation_author=C. T. Miller;citation_title=Parallel developmental genetic features underlie stickleback gill raker evolution.;citation_pages=19;citation_volume=5;citation_year=2014;citation_pmid=24851181;citation_doi=10.1186/2041-9139-5-19" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. R. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. C. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=D. W. Hagen;citation_title=Inheritance of numbers of lateral plates and gill rakers in Gasterosteus aculeatus.;citation_pages=303-312;citation_volume=30;citation_year=1973;citation_doi=10.1038/hdy.1973.40" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. W. Hagen;citation_author=L. G. Gilbertson;citation_title=Geographic variation and environmental selection in Gasterosteus aculeatus L. in the Pacific Northwest, America.;citation_pages=32-51;citation_volume=26;citation_year=1972;citation_doi=10.2307/2406981" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. P. Harris;citation_author=N. Rohner;citation_author=H. Schwarz;citation_author=S. Perathoner;citation_author=P. Konstantinidis;citation_title=Zebrafish eda and edar mutants reveal conserved and ancestral roles of ectodysplasin signaling in vertebrates.;citation_pages=e1000206-e1000206;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18833299;citation_doi=10.1371/journal.pgen.1000206" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=T. Hatfield;citation_title=Genetic divergence in adaptive characters between sympatric species of stickleback.;citation_pages=1009-1029;citation_volume=149;citation_year=1997;citation_pmid=18811261;citation_doi=10.1086/286036" />
<meta name="citation_reference" content="citation_title=Nature;citation_journal_abbrev=Nature;citation_author=International Cassava Genetic Map Consortium (ICGMC);citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=5;citation_year=2015;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=573-61;citation_volume=484;citation_year=2012;citation_doi=10.1007/s10682-010-9411-4" />
<meta name="citation_reference" content="citation_journal_title=Evol. Ecol.;citation_author=K. K. Kahilainen;citation_author=A. Siwertsson;citation_author=K. O. Gjelland;citation_author=R. Knudsen;citation_author=T. Bohn;citation_title=The role of gill raker number variability in adaptive radiation of coregonid fish.;citation_pages=573-588;citation_volume=25;citation_year=2011;citation_doi=10.1007/s10682-010-9411-4" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_title=Genetic control of the leaf angle and leaf orientation value as revealed by ultra-high density maps in three connected maize populations.;citation_author=C. H. Li;citation_author=Y. X. Li;citation_author=Y. S. Shi;citation_author=Y. C. Song;citation_author=D. F. Zhang;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=M. T. Limborg;citation_author=R. K. Waples;citation_author=J. E. Seeb;citation_author=L. W. Seeb;citation_title=Temporally isolated lineages of pink salmon reveal unique signatures of selection on distinct pools of standing genetic variation.;citation_volume=105;citation_year=2014;citation_pmid=25149250;citation_doi=10.1093/jhered/esu060" />
<meta name="citation_reference" content="citation_author=J. Liu;citation_author=T. Shikano;citation_author=T. Leinonen;citation_author=J. M. Cano;citation_author=M. H. Li;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. D. McPhail;citation_title=cis-Regulatory changes in Kit ligand expression and parallel evolution of pigmentation in sticklebacks and humans.;citation_pages=1179-1189;citation_volume=70;citation_year=1992;citation_issue=6;citation_pmid=18083106;citation_doi=10.1016/j.cell.2007.10.055" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=C. T. Miller;citation_author=S. Beleza;citation_author=A. A. Pollen;citation_author=D. Schluter;citation_author=R. A. Kittles;citation_title=cis-Regulatory changes in Kit ligand expression and parallel evolution of pigmentation in sticklebacks and humans.;citation_volume=131;citation_year=2007;citation_pmid=24652999;citation_doi=10.1534/genetics.114.162420" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. T. Miller;citation_author=A. M. Glazer;citation_author=B. R. Summers;citation_author=B. K. Blackman;citation_author=A. R. Norman;citation_title=Modular skeletal evolution in sticklebacks Is controlled by additive and clustered Quantitative Trait Loci.;citation_pages=2841-420;citation_volume=197;citation_year=2014;citation_doi=10.1111/mec.12350" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_journal_abbrev=elife;citation_author=S. R. Narum;citation_author=C. A. Buerkle;citation_author=J. W. Davey;citation_author=M. R. Miller;citation_author=P. A. Hohenlohe;citation_title=A recurrent regulatory change underlying altered expression and Wnt response of the stickleback armor plates gene EDA;citation_pages=e05290-e05290;citation_volume=22;citation_year=2013;citation_issue=0;citation_pmid=25629660" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. M. O’Brown;citation_author=B. R. Summers;citation_author=F. C. Jones;citation_author=S. D. Brady;citation_author=D. M. Kingsley;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=4;citation_year=2015;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=1224;citation_volume=7;citation_year=2012;citation_doi=10.2307/2409768" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. E. Reimchen;citation_title=Injuries on stickleback from attacks by a toothed predator (Oncorhynchus) and implications for the evolution of lateral plates.;citation_pages=3014-1230;citation_volume=46;citation_year=1992;citation_doi=10.1111/mec.12322" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. Roesti;citation_author=D. Moser;citation_author=D. Berner;citation_title=RAD in the realm of next-generation sequencing technologies.;citation_pages=3499-3502;citation_volume=22;citation_year=2013;citation_issue=17;citation_pmid=21991593" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=H. C. Rowe;citation_author=S. Renaut;citation_author=A. Guggisberg;citation_title=RAD in the realm of next-generation sequencing technologies.;citation_pages=3499-3502;citation_volume=20;citation_year=2011;citation_issue=17;citation_pmid=21991593" />
<meta name="citation_reference" content="citation_title=The Ecology of Adaptive Radiation;citation_author=D. Schluter;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. Schluter;citation_author=J. D. McPhail;citation_title=NIH Image to ImageJ: 25 years of image analysis.;citation_pages=671-675;citation_volume=140;citation_year=1992;citation_issue=7;citation_pmid=22930834;citation_doi=10.1038/nmeth.2089" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. A. Schneider;citation_author=W. S. Rasband;citation_author=K. W. Eliceiri;citation_title=Is Genetic Evolution Predictable?;citation_pages=746-751;citation_volume=9;citation_year=2012;citation_issue=5915;citation_pmid=19197055;citation_doi=10.1126/science.1158997" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. L. Stern;citation_author=V. Orgogozo;citation_title=Is genetic evolution predictable?;citation_pages=5304-751;citation_volume=323;citation_year=2009;citation_doi=10.1111/mec.12939" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=A. Theis;citation_author=F. Ronco;citation_author=A. Indermaur;citation_author=W. Salzburger;citation_author=B. Egger;citation_title=Karyotype differentiation between two stickleback species (Gasterosteidae).;citation_pages=150-159;citation_volume=23;citation_year=2014;citation_issue=2;citation_pmid=21921583;citation_doi=10.1159/000331232" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. R. Urton;citation_author=S. R. McCann;citation_author=C. L. Peichel;citation_title=The road to modularity.;citation_pages=921-931;citation_volume=135;citation_year=2011;citation_issue=12;citation_pmid=18007649;citation_doi=10.1038/nrg2267" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. P. Wagner;citation_author=M. Pavlicev;citation_author=J. M. Cheverud;citation_title=The road to modularity.;citation_pages=921-931;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=18007649;citation_doi=10.1038/nrg2267" />
<meta name="citation_reference" content="citation_author=A. R. Wark;citation_author=M. G. Mills;citation_author=L. H. Dang;citation_author=Y. F. Chan;citation_author=F. C. Jones;citation_year=2012" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="twitter:description" content="Marine populations of the threespine stickleback ( Gasterosteus aculeatus ) have repeatedly colonized and rapidly adapted to freshwater habitats, providing a powerful system to map the genetic architecture of evolved traits. Here, we developed and applied a binned genotyping-by-sequencing (GBS) method to build dense genome-wide linkage maps of sticklebacks using two large marine by freshwater F2 crosses of more than 350 fish each. The resulting linkage maps significantly improve the genome assembly by anchoring 78 new scaffolds to chromosomes, reorienting 40 scaffolds, and rearranging scaffolds in 4 locations. In the revised genome assembly, 94.6% of the assembly was anchored to a chromosome. To assess linkage map quality, we mapped quantitative trait loci (QTL) controlling lateral plate number, which mapped as expected to a 200-kb genomic region containing Ectodysplasin , as well as a chromosome 7 QTL overlapping a previously identified modifier QTL. Finally, we mapped eight QTL controlling convergently evolved reductions in gill raker length in the two crosses, which revealed that this classic adaptive trait has a surprisingly modular and nonparallel genetic basis." />
<meta name="og-title" property="og:title" content="Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/7/1463" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Marine populations of the threespine stickleback ( Gasterosteus aculeatus ) have repeatedly colonized and rapidly adapted to freshwater habitats, providing a powerful system to map the genetic architecture of evolved traits. Here, we developed and applied a binned genotyping-by-sequencing (GBS) method to build dense genome-wide linkage maps of sticklebacks using two large marine by freshwater F2 crosses of more than 350 fish each. The resulting linkage maps significantly improve the genome assembly by anchoring 78 new scaffolds to chromosomes, reorienting 40 scaffolds, and rearranging scaffolds in 4 locations. In the revised genome assembly, 94.6% of the assembly was anchored to a chromosome. To assess linkage map quality, we mapped quantitative trait loci (QTL) controlling lateral plate number, which mapped as expected to a 200-kb genomic region containing Ectodysplasin , as well as a chromosome 7 QTL overlapping a previously identified modifier QTL. Finally, we mapped eight QTL controlling convergently evolved reductions in gill raker length in the two crosses, which revealed that this classic adaptive trait has a surprisingly modular and nonparallel genetic basis." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
    <title>Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"7mLd-fyU23BBWR_J3Axf5elYC5fx0T-8ik2U3VtX9Pw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459974","apath":"\/ggg\/5\/7\/1463.atom","pisa":"ggg;5\/7\/1463","ac":{"\/ggg\/5\/7\/1463.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/7\/1463.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1463"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1463"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/7\/1463":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PxoPYlaHCdP\\\\\\\/ORI4gOIN3P3YpzZgUZw1HIW5Ku5PTOxNpdhqERNQsCC9dceCgktReI0Yd8QSa+rzCUSbvJ\\\\\\\/Xil4CfDyTqPmlMHO+vwaXvNirbkZzObnjmejwVpCL\\\\\\\/vY1MBgsazefolmhvCt6JN7RpV24XRyp4lAX+SxazUEd3ieCpwNcsRYnvb3HdTvA6+yKeDaBIRNBpOgmaHJVR1zfTB+uuh28Z6comzLmUUiBrNXfluUIQAFqN0bGSLy9W3aCNBGdRFBJ9PrW3eN8x0fGoFKk1\\\\\\\/jkPx0b\\\\\\\/Mgzxk83ewlD6KxqLcxgzED2gmdTRSbq\\\\\\\/t03NWTigdB8SLyYPL+65xGt7tPK\\\\\\\/OAoBC1Kaup0QkUOQaDV0pJYWhDNRil9Fvh5BtILur4XadFMAzFcnbqK+e67WfecSQ6A4cu6aQkMiBVJRR9iIGKNDP9i0ssYaQo7R3K6Os0q2wjtDm\\\\\\\/pW\\\\\\\/cvlRbfqLAZbusY7KPsUPjQn8k1bg3EfApK1PJe3K0dT9QQihDMbnv6+\\\\\\\/VCxDv5iSZb91LnEetdV2\\\\\\\/89lTs9Oi8L6b2HLkNlJRPfSe\\\\\\\/+\\\\\\\/PWSaSdZTDV\\\\\\\/cPRmGzbq\\\\\\\/MnxebFEuzjkztLt9qOjHB3z4brlBJqNXtmFyQZ4xDSJjgLW+WiodBnb6pWqR6L0ozqdTHhijoO6GxwSzC4yTCs++sjEHY1+\\\\\\\/yK3HvL7ryMKOwH6U5VBYuVgxL3T0CzA1edofkth2zHvL\\\\\\\/lj1UrjhGRhm4m2w3h923D5jHmeieB+ainbvHFJoKPDIZh6mW4DPtwtA9rpmbTsUS3FPgDLenQUUq9GfBWdGgv9HUnEIkKt1oMmo3UW3Mzu7hjyYPPPtXyvnd58GXQljOYTmxVoJaD64cAf2yVJfpFE5RDE2HpaPbkUFZ2ivO+xv0pCpUjVnI76l6Z6rwhs7D12F0tupBmTk+g8B7cFeaEsuaVDDqio3UlnNtqq2xrHhNwTR4QbDKyqfSLxTQHAySiP3e1qEr5y1cHw+2ybWimNwxwWVfL9xnGDnly4rGwrktsD6lR7CkUxSWHxvL+qF1aVCyvuUrp5rcOr118h\\\\\\\/i51Uh0kli\\\\\\\/dopf9KxKUwTYc2EK506\\\\\\\/uUR0uxYcMqICRn9zVzYwX5n0yfQOtuFWWApb1hTPeVg870g94R6gw6nEm5DXPFQPVBGGpskuXnNcWS6BQAEzQGxhdJVeXvvY\\\\\\\/ernxTC0h5swpHvHG6490SBdJFytddttTdL1mZyx7NRQ4aCWRRsdehV7mi7OcqoITOhvT1jIMcPzfDDPeeT4\\\u0022,\\\u0022iv\\\u0022:\\\u0022KLD4ioJ\\\\\\\/ifzEhLDu2VMySQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002230395df7a7d617bb831e4570778024318376cf35198cd61f7135cfcc4d37e47e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rebW\\\\\\\/l9PSwnCPLMqZoeij+JM9\\\\\\\/3nonj3qEsvXLlV4jCWRrWJz++5v6bxm0G8q0GNcP7XQ5GiVwp729JAVWXQYPITBIPYrUSgM4gZyK4C2hEg7Q36fk+B6dFtlP7bX1v\\\\\\\/YpSJjvAne0oAwsL3jbaFnNWvZNEOS2NJqOGi8zG8W0+DrLX89+IGMBQjfXcXMKZBtraF98f1Y9dTU3lNlvm81oYN5USR3BRxd5Q0ki7fzhRCwM3cQqAS0I2ncSgAVkDxJ8+GJIRm3R+fjMYqkas8EyZvff7v9swTQ2Lz7fm3FmNQJ7KrLsU8KbExAQ41wyrFQoZaODtUXcvyGhXM3sfG+6rUkn+AwOehPOKVwxlHSa1thF3KQyjbdh6VjYqR3umIa3caqWf54YBy3HtYznaEtjsWAi729oi46WErIrGHQD2KMpK9EEQJT+LY9APHj+Q44R4mBy4UO\\\\\\\/iFgoqLsZOFlJNj3QqVHiVx1OiJ4eEIvBQQCeuaNnYjhLSZDrNm0i0Nov9FBFztDOr4MPxw0GlWT96BqcDDD9o05kijNv7luoA5NT17BkUcY5AB6qvziZhwjKDiCWdLJM+dqjHBpQ3aM5iGp1Tjt2YfcWs29\\\\\\\/ZZu2BuRHifeVReF9MEMJhGmfd\\\\\\\/EcuIZ9xtIKcjL9tqkOZdGZXlGUscroIKBQyuL\\\\\\\/m1HSTeCBQ25VeeCOj7EQNb\\\\\\\/dFvaRMSZjSQYmAC46QGT6PhP0bQCKhJe+y9FrhAtvJLiFwnp4kd9ejtLmSMhQzgLeoKZOhmvFarfoq0YqbXWrurH0fbOsmC0EJxX5lKiScn5GTH76tZ+60TXfROBgsxO9iT22IHyLYRINXdc0V9Bn3dXuCg2gdUbpz0aYLl6XTJ+CCe9ZTn34FTZUHTjaSSAXIce1ni54TVkLYrz9wP\\\\\\\/KRoZbGaYlUeYilTg23dDUzWrmrNL\\\\\\\/f4ckU1x5At1czgkadcBzHD0FwUYRPyQAw+elRaClpLNmrD1\\\\\\\/3PeyKGkoRjYHaRAuz5R+hXWluo\\\\\\\/2tJK6eeUl5qgjVvXKIF46fQkXkoBF7q\\\\\\\/gErYSHfqEk16IoOrq3ptECnDYHWZ3w\\\\\\\/XNUAVNrsij679OwCIEordO+554HkehN+CiwRErYTbGlSs6woA9nXjTFWJyBgizeX3+yvMK7Qxf0I5lRlkz78cnmEqzzEuAzal3\\\\\\\/iLXOhyEaQXwQr5Fu9ggIKG7f8yflM87GYIjukcIUWxDRX5im5xVq83CCzFpTmfr5ah1nnFXwdj\\\\\\\/P3zv1KqXJY8xnTJNlR6FKqE0MBmwERBECNaKz9wYB9lsZ890rGlUavuklFi5k8cFv0OZ1XE9pFhvIIi0IHRAFJiENFQw0sL8SeBQRpaVz0Y8ZGcu7rB33B9TRq4ru1TBXEUq9OeICwmc8WYePAzO0c\\\\\\\/8LEAyItycoIJvVduLbHqoKXwN\\\\\\\/\\\\\\\/Bjy7EiYHkoNsUXAIZ1kKkLQ+R2aLL9I46yURW1RtgRTMV0BJw9zw5cXWCzwjAItr7QFJ5DC5gPc6p9MgKOQdxrN7crw4O4ztEe8bm9QnIe7SZEM+aoO9veDydhESEDhpSpqpGbHM5Ake3uQEpMSiBwAKCMyxtQYzqC7GoF39An6X7dLwEVweFGsVZAJj9j0dCGW3mSZKiG05y0BZ7NvDyxj23HwS6xZJy6qRg+HDsvEP2mjENnaBsE62Mhm5nEAlCpEi0WGt6kCPOjSy6M6prxLWe1Sv55PPFvjpOxp8cUP8UlfZjs3iiBukOQ9EAOF4rZ+LoXVuhqqe8ioYTLeIORg4jtnw37dGiqto1vMrEFpKQk7kyXAoWfXE4e6af+IVMJS6ZxFMMW2DkKch\\\\\\\/eCSDr8YPFKIGeSN2PgTzV3zM4+25JICiPHzQ8hVSaflvCMN5jrJillU1zmaCHrqW7mV2kKn8QOJLAflws1dSip2+o95alAP8f4mpoNWqg8vHZEGLyGXjtNGD3gbZUlqlEAQ5xyj6mJXyg4qERj5A6V6h6LYUExFPbf0kaqdM5SQvXSqbRN\\\\\\\/UftJIp0qRBjjvkny+IDQPMEONBESH9y+eCmDvbzGerkDLPchP9jJNZwfUPyzd78ypOfI0fGgKM\\\\\\\/NSrITgAhAZCflbfLtkK6Hkz3zPbv5IJE9OqMYCp404mkYMKJeF3jbVV5c1m+WfWkLt32IQGAnIuCjguMI5z7gwXXS4UCFyiSoq4h0iXG8Gq3fr99ryuwZwPs5QfjuHH3k2IQfxSGu7xD9kmoZgAuMMyvaux5iLPnxrPYG6p8fpep1FmpVdLN6Rk5jPAVdvsmFGOrbraiNz5lvPszgWqJAfQfixketQRjj55oN8rsQRH+infyM9GPfY1u0BSckC+f5hE1c1CNxkByvGg2R0W\\\\\\\/bcKm5d3JcY13su0QNikotkyZCEoQvmFNLZ5Hu3JXiFa0bDKBLByJh3M3P5jHAWytrGmnQWsGXeHzxPkj7snDY1zVLqN4yqeEKeCd1ui79MurG2cvxCGCdNbxSPgqagjACiQF8ifqubsfn6\\\\\\\/hJGEoL6b3Nu3A3+rmulvDlwuxEV7LvkrYK7+R8zka\\\\\\\/sM7H+NhPx36eDIpLIcbi5Vau2cMBMNYi10DU9v0rSRsBjMxM7R\\\\\\\/4s2BIyjfDriRzcfqQO4Q5TwJarftT4topsf308kahu4ufligyUQw01wQlsonyznNo0Yk3WfNC9Xh3RJ5oXwiNWhSanLYlLPXkFxkYRLr+Ga4t7ZmAwJzGCYlesLVSCNA8m7AXBM9Dn\\\\\\\/yeHdkj1ep32yAaBL9L7xiE3PosmeSmapUQIJQGRDbYamrgEhKC4bWmhWHZwViL4dL0kk1jrSBMaaeynoY93\\\\\\\/QSmeIBUm3Y2iSVcImFfpZoSWQgi\\\\\\\/CziIRypi3Vv7heJ7V4nWa\\\\\\\/cjtl9zU91jOfmBGoUI5wWzq2HqtRyd0thCFMmBpF0+1bPH6uLgatEeO+TqqcU46UNH7VOT9OTLlGMGfGP4a5qTvwudSYxKfsHqGDixZ5PysxxZZXgrVuz7pzR\\\\\\\/NT8x1AMhNxReR7PKmrBinffQBYJQt3MDmIoDn84lwa4pOnlFKsKgpSOtcBw5kDQd0bD3kl2EC9J2tFn5KMhtyo7txOfUYI5+K3Mo2jacjW5PZMREYdZPrTiLYagvV\\\\\\\/6KolQ48J07eAoNlfwLTR+o8gwC7cCSKK4vHLxCyiiuCaY7JkOLVh9L+z7pHoIWj1+0CzARLbbLP37u3y2RZGQ6OHnxr26d9xEFSpcQRJJvlpH76TG5kcoFNF7nt8zgPv1HmmB+5JN0Hrl1IVf1nUnHFZ6a8u8hR0rk3cA8DVnrvexfQ1CZA5hV4QvfYx64wjeS\\\\\\\/FN3qKINWm5ucST9mLXKqhzfIEOWc9b0sjLy4gWbo7HjJks2RMkvKktB0Qmr4TtMd4m4apd5rqN++boB9+o0ZGzcKn\\\\\\\/yIJgpUmOGPz7hrw7KfdvS0ucwBFOmKMo534N42Nrz4IRSPHbkz3iCvE9dXttnhUhS1TAHxAeJmZh5yJhcRDUUnQdxsmpHqn0soACAGJ0wgNNIkwImmH3NNvgbe+MjKxgwWZx6yLXkmB5IF94dqoHaSInW52cwAV2RbVsKBsYyF9si02CfdIVnLW5WFjQ2fAPEEFecaeuPwbMCG\\\\\\\/tNHa98C50jIT4oWVzbdRq0zt\\\\\\\/tSiFKwiripvOTMphjw5bUbs4PjX2Kzid+aZElCSlC95Tqa8gbHH1aJ1Cntr92q3zW+\\\\\\\/1hhKs4WAUpJpooLbUngk+4QOAEhPtuPsEMdV97BaTEbb2V8Et6rfc\\\\\\\/jeK4mdmJsGc\\\\\\\/qH40iPlALpIqJ59DGGI71jU1zw\\\\\\\/wRrgoOil6NtvhvSNfR4NUXOjBCbZRfBrfmMzWsFxMRWkF+lHczm\\\\\\\/9T5nZeR+xf7NpVRilL7D0PafdiKZw+wDKpMpREs4zmid85JfRJRsNVjhiLkTwRjz740aoKsNULM\\\\\\\/rMh2t9jTaC6FC3ez4bzUDFgGRXHr8aEI5Pc+mc7770WXrB04kC6bj3ls\\\\\\\/iRMmoHvTrAfTzVZR7a5FbuaTGfq2jnVhS0KKRGcZ0REpTSYkkXiLDsZOPvrJmF7LfGpBZlAOru29LawGTxkLI0LVL+EzIotZ\\\\\\\/Grt7llwBOoMq6vjzhDsT+Dcq8FM9PTTrzRcWNLE+2FYk2EBudIAq96r3yGgGF\\\\\\\/h4w644e62X\\\\\\\/4EY6abKcPDgearDFNS2ktEa2Vs1F74lqgQybdrPtbgTgj9K7Nub47gkqZptONOLr+xrS4mRIW5eZ0r05onAiwrGMwtLyEPqEHDjzXwZcOAouyavrHJc9y3TTtQBLc2ufNJXOQFRh7I\\\\\\\/KmcCnfojKzW6D\\\\\\\/kVhtyXucIWSuBZWDSl6v\\\\\\\/yKZEjiA1ZkOu3Pf+5cF9v9Pn8jiuj6e0EHZgdEwmmR+gXJCYCvnDMB+SwxJMyKNOk9PYVtNA0qSvsqpHEQFlcGHPdIBaHpomuy0ArGXK5RZ7+lhzhFQ77iuiyJfyuxlZUtO0ODjyJHREOOKSLprSORahpLb6VBTGZH1Q2egqWuxlV6qhrdjHF1ZBwYP1uAg4OwbK4CLE0nGDNmPlAS4iOVEsO9\\\\\\\/NpPlJxuxGOC8dhGHVboTqz05ekQH64BBNuQoDqmr9ycQYUJ5nQUEtZdPYaaQl67MGbGpjFQhxdaYkzvdXcl2XLhEHfQS58SEN8u44DKcnMorMApztoq6EwWVDv4gt6tzJXeMw4AT6yov4XYARBHet\\\\\\\/iEbXoSR5ErRE0yQhkyHX70DOzkI9bis7oXnj0GHpOeog6hKgPZqJ3CjU3bp6VWjrVom0tirPfwjvBvLIfKRwe2VtdOQhwKhDqJykbnVdSx0V20pZbYbAdUsmlExm8rR3I9MdvT4lvMzRYaVPmlynxbktGhYrYF\\\\\\\/v2ULzVeNxIHu4moBTXv0yf07WWkHSfS0bY\\\\\\\/iqGuFhgD6qRuG3XWuvgO0mFEythQwAfnNsPj0TrZjjFXiztT58jK3cQdV5xp4cIsWzPr7cxcncLQI8D9pNfsvgC2D+sjspEvdUkk6cCKuvLdlOlIuwLzbjVTstKGyhoKoHLRpoJXlLCbYnz5PknkC\\\\\\\/QWFXaPlf0BbRCzhexIwveDf6Uoy101OeAAAmjT\\\\\\\/BhdI4XXISKPwLZzIDPlvM\\\\\\\/87OtDeoRNLe43ehuiC4UuhYMss9GfIe9ZseJBPqYONJjqgJn8s8DZMVhbkTVO\\\\\\\/ynoJDzbgTvV6X9akgOYDNe3B7SypB2ofmjWN9YcN\\\\\\\/zN8T8jD9R7cJZ+bMo71a6Eh27+R4BcD92y\\\\\\\/cchW+vVgJIyt4m7s14UtYglv+EGrb+3k63QFx6VJX0qAorH8Su+GcpihOtJ19Xd\\\\\\\/ZBXAfNc4yx1Cozy+N3+NO5pjlxVNHSLZp7m16ENi9BpCuJWSNjqax\\\\\\\/VBkI84TPO3fk6xqnJVcmYILx+8GDmETyO29+72TUSD\\\\\\\/jIO\\\\\\\/IjQRjQjUm5hfzi6JU+EpGtDRJQZEfAYGI+n0iahlXm0qKjd8GRDI+i8bo+\\\\\\\/9sggwgML4JJ\\\\\\\/X+JBlJ9XpKt7fUXhO2+OHl5Gc8s5wHoBTHgW5DJu3Xmug8HPWi4q\\\\\\\/e+HcWk8\\\\\\\/PxfYqzgzt4cvZjxMRbD\\\\\\\/yqYAGQMyM0eNU8v+Q\\\\\\\/eS\\\\\\\/um0gyMH5P+\\\\\\\/jSxK2hEDlAn4qMJ3Tw6fBHGNa\\\\\\\/YiHkVz32U4amwn5w7O0qrfF73EI\\\\\\\/mmv9eSEWzrVM2Nd3IgEQCEAo\\\\\\\/EF10EfJ2nAD+1Ehk28BTchzoCItJdtIZhJ2MIUFuPLkpZrHwr\\\\\\\/+DtNZpAYhQQUGTLBk\\\\\\\/2o4xBWkujX6Rt9ophWUha4PzUo8c3LvZ4AxQYqp4M6uP2yIolL2AyExdtiefpbBY1Bl\\\\\\\/h1XH0umrK9R9YCmFBShGnlofDU1I0zGixUgbz9khf0B7khYOZ5wvnBTHA5r8abwEEGP6QUIz5ezMgbZFRVOvDTy\\\\\\\/uxUrmYeEt+dsbwofETJPNdXayzmZwOE8luVEkD6wcPeb6bbgiAwHqgO5Rufwww9Tyksy62X1ue+7VpyoarN8d1s5IXpGFmpIaLCJJz2T+zjp8stRpIMtWVroC3L5YXwU7QxupxasW6SIF9jyy7k4wuUDHbnsv6g2eJbf87gGlNiFPFtgg2\\\\\\\/tmeN3PzXHAED7XLzKyqnUndup\\\\\\\/A0\\\\\\\/0eSw4p1cG2P2u84ljz6DVMHDYm0lDvLJC9\\\\\\\/eNuiIZGHFzoRYisqtPxpHQCdPLJFNsCC6Plk2+7o4+vLpeH\\\\\\\/u3v8VGbidTYlToPceRLegTFAenMancN7nprV+xpQ7wqyKtAQoglgR\\\\\\\/UjI9avbaR69Lvod03WyLcJAM+GYM1DMKMpqnAvN+EGoLwljwMOjsWUmmHj\\\\\\\/0yYIEir6MxaIYHuoReUHSceqIQOTLO2\\\\\\\/zWXZ7bNfYuBOvRCQ4SiNY945moLXczPgGpELZ7rC9rDK0KtHYu2kBmb6EFot4XclVggzgGzh14yznECTs2Cf\\\\\\\/qsSWXPyVTwqlR6l6Mn9A1kUTHFssyjdbob7fGe3hDQDJMhUCmizhExlkN309lJvWI5SiCIqx8C9osXpnIF4eK+EBicFvrOyLbhpWwaiRAb6DYuM4d0So1qsX4E3aeRb\\\\\\\/0HAmaSQbxq\\\\\\\/AlMztencubYUwiCP0M8Use\\\\\\\/DL+xCM0MBHA\\\\\\\/Fnk5D+66LwUDK\\\\\\\/8zSTQBCA8+W3gjX3zweCR8KoMw04dRlN2Pudx9\\\\\\\/D1SFK1HQnXhNdLZbJNdGHzjxyyfzTqz1dzTuCUE63hrhFKs0CJC4Z2aoS2mY0GsQ6DQfhklxPcjBA6nANd7ZaKJNc\\\\\\\/r5GnYOM3KH82+raCp\\\\\\\/PFTt3QKyerUOutuVG4WVIoTsrd+hxDVXOnwbX6qc1sBXk\\\\\\\/bG0BglLjmjIVtTQS9QAJMUaZfj+YDIf6r8fwnJxOc33kOexaerFLibqS4fgQu8Ikpw7inMgl2a2DbLPpt\\\\\\\/z2y6CL9Js3k\\\\\\\/aVoV5fTAX1b5OJuInMLgm4ZvUkObqX0W4qzEBXQzU0RpMUX6LYrvcJzTCqAnbxgE+D8Eg8\\\\\\\/XQFj2qDUVh1sPh0jFq53N2H\\\\\\\/PFnmFFEOVMfjo3s5NeN1FBq0oJ1YrDLITAecPyOWGOJjh\\\\\\\/\\\\\\\/fmhpS3OxCbXxlQwWbnVk0Qiw+hM6o0ItFsytkz1n0cBc2NppvZ3beIzhKpSxgvcApY\\\\\\\/uixW3lgY8CQSdfXn7PgdoJ8ECtdqBCUq7GLK7EXtgwoACwYk02eiD15ZzXbH8ELpKZ52WOwL873LJh04FgTgyFb5Dh9YSXMJZytAGGd9PwR2DTeMO1WBWxmik3+o3gMkoRb88s8U5OEjDxS+VzPxbDF5WDLwgfaiODRc53lFv78Lohk9c8lsmc0nmBFc7YRtK2p33KTsj9IjGXKTqTLl9\\\\\\\/3+ML94g7WtSrhD+V2Oc+mT3s2ta+fsWw3kWYcarDJnMECFzFZhUJntIepeEzATFRLHmOwF\\\\\\\/kK8ntgxx6a2k2goB+jWDkZE58gPDHNMKOkKRBs5g+zyxQ4PjhdwjAEcJwCjyMSUY\\\\\\\/Y5REjovhJypeT5FlTEeC5zQ0Hj1GOhygh\\\\\\\/VzATRvuQjuVgNTeL6EMyYvqPu6LL5WkikLFPeQMjvur5JjvmmTd09Ez1nWQvYh39yqL4xFFnicTvjqgKCzRHoKktVFLhFK6XgrVmAk0w4gcRspESyKBVzMVHG4HjyIGc3RfylS3v+\\\\\\\/qvJHM172B3yAcUzeotvYqxQqwRAA6SaDUaPEwTFMESTpq\\\\\\\/pRyLIv2+z3Jgf9Lo2ipLwMfNd7RxEmrjwSuZJDc+zj6kLK7xHKntO6AgGCHkafcy5UQ8MLrsDQlwh9rQRi3h+nFpI3U0DxxL5IcFdeJWSp+Rk2wIuCPB5JfY+Tcuk+uvEPzetOsImkVeEhh27O+5S8eeNVg+CQhY57X788xuld4pzKpTrlzfFfrb55got7ZC7vqbzWrm6+ez+5PsA+mIoQVlRAaIVXZnZTivTQRn+R5m8l\\\\\\\/D5KNf4MNTr9RhI6ZiSWVD9uJZBB21dbvdoK4NKO3phZ7QYlQ+bkjxrWp\\\\\\\/NkOQ6wOb31nK0Mv73imRR+ZVdYyOJqkpbdhgeUT4dNFb7fxqiXYrT9Bv8LyyzhpFojDHgsPOBzU\\\\\\\/esfySdlpQn3981fKaAbgcmqPi2\\\\\\\/m7BatpIYMeMpUyvu8mINFT1gDa+XEGZK1fFAwLKpKk5LTu0zhK7YxUGGqeSUWH+KrZmluSWrStHQ9Z+25BtLzZqWBKNxlLEBkmbnXjF9Vc9sMAEN9rLEU9uka+L3FqOO3xmTZusnJzb3eYJ0y32ziVR3w\\\\\\\/lzJoQIvRut6GO6X8QtzFb4TYGQscnELRwD9XwlstX5LHCiniQm366APSleoaGJeTbgGaO21XJ1XCUVrd0gwtLw5M4V6uA4s+FkJC+EiiqC+aKjhVRYyJn8qyDIGC9xEE\\\\\\\/CaklgJaHj+m\\\\\\\/6zl+OCUmSILe3wZwLeHEDYv2zn8z00xM\\\\\\\/ZwX9Z5pqaDxXl3RXrHyx98nQreR3ZNG\\\\\\\/jgjgmNtFktBOo2R3G+6mZ1wiVgQyNvtk\\\\\\\/xe1L8isVY\\\\\\\/325LAkt1oJFxBizQs+oD+SkssMXW3bOHK9iSXUze\\\\\\\/s+ApZ0PxLSyG6BDQbj\\\\\\\/EJ0\\\\\\\/AAabpqgZQ6zD+n4LKikq8xvfLVLSdGqLtmfFpPz0I44xiv43UnRzfzBLFze3Gp0CsrAj7HU7VKGDt5jS+Hm7oMw6BIli9Q+zvFV4G4WsbRd7v6MKPwdmv8dd13QOIex1gvkW1FJvaVMpV3JPUkhSqX4Ar90c1Fy3ur6K1ZwZdNYjQWxXTOKFuwqXGea1tiFSxh1Vqt5krahHm+O6DET1k6\\\\\\\/gkzgYxA+pKrMwskqqBCBKmd2hMwxg1Y0UI3m5Yu04Kbq+aVPFk4TaPO43uymLkdoiMlwT4od4i3wzNETEmGi4nUAU+e1GL+tdr7Ivo1OL3rp7U8TFOT3bXZTw8zPg+jUfBRsxOK\\\\\\\/B7ao2otjo7PXq3pWQPdHLphtES983BapB9BrfYDWIsruy6FO0leOSPL+HrPPL\\\\\\\/uY\\\\\\\/52GNYJQPlzjCuQBISxJtTaSLbvDHwzfhIuPwnQRv7gpNlnPGj0\\\\\\\/tuG5lgv+edXFRcYz2p7SbglI3S6E0eubzhYS5YqKXmEvbIhx5lUeD2zfUg3LroInyMxrNCoX9I4CY6VP10W0vO\\\\\\\/qbluKRqoh2H09jWxjpiP0GHZBn0Tml4XOePPEW5MdI+ePLgFsnKeSrGszauwfk7T9tADc1vyyCr2XjQpWfAAysOP6sJnCXLSVNhr5qOiRNExJlB0KxvfGAshmui0FD0elavoC2sNc1znY7nKs\\\\\\\/wHnODPqOTNbR7N6wPLm4sAIEQZSurUNw7vLWyYygsEVyOCGhzYiYwA\\\\\\\/WkwMD3gWA4WXfiC8+xL8suoyKXaJK3z6AAHABEG\\\\\\\/CJ47t4\\\\\\\/24qc9GoRduEQvPWCTvDatLMzCzSPqm2Zyil4G5\\\\\\\/fEYu8kZcWGItoLZq98RjLoDmGecbSUy0esM8\\\\\\\/NJ0+F5lqVIDc4PJIDSQbTHKnw6cmq6UXwfpjiumK2Pq01AUF39MEKH3\\\\\\\/\\\\\\\/0Ep6\\\\\\\/qJhrRf\\\\\\\/bJzc6j8PcsqS\\\\\\\/bIYkUiIQIXmhZrmjqLJ\\\\\\\/Bbnyrfkj2gFCDDTzc+38zZzHITwdCcwuW9zKyvbBK9bLzc\\\\\\\/gzowoXLRQ\\\\\\\/SGY+BIglil\\\\\\\/lx9iSzdWyWTbnvaVMHAAKZTfXf5vyQ7szSXAtFQfqT3wUjml0enbmdbGqTsgp4zmb7CoW51yrbJrZvmiluGEt\\\\\\\/ZspCDsUMnMstCWPh1rHMdrSOOpfHGCSJRj4dGVHnTnqjnHVsWakTJQ733q8iJqA7UPnf3VtfbAgHztj6YKYK0j4KhAVYj19GPMp95fOkk0EcA30oJy\\\\\\\/VuAxuCUdU+ElaMZ6\\\\\\\/cbFXa17vgL+eT7ShRgzl0mwV7+Z17trYQmVvgoxIzl23Ax6q6laTjx3NLOjBAY2R+xc9COooWPp02Y8P8zxJyyr689iiPMRJdkVjlRKvvmeB4tenAEhuperT+B3\\\\\\\/jzR\\\\\\\/bTCBabmcLCBReGE9IoP\\\\\\\/R8sACs0lVqC5B3U4vnV2rHaaWEA0RNGW+WjJ55a2VSDD0jJaMCI+IKOGnODlNMp29sLx\\\\\\\/TUBPH+k2h8JIRgbp2iO0KEBbcDv2M71BYTSCnURVYf7VxarGO4fIsWp2WpeSau2m6X+N6yAXPs4VWEQgnEWOwSAO\\\\\\\/DRV8vBBdTmIv48G1ILadkgbQ9UV3k6RgcZLT6c4mV8TMXFpvaG6973KkYQ+p3o1wX09KqTCHSIvAk00AbRoWW\\\\\\\/fAWyML2rr9SN30maxL5HrIWdVEd3VhpffLxpGou9CgFHV4YDnvMDiRGFSdooM9l0eGVP3PFuCQY0Ez1j13eHQ8tISSjIQ+W3Rg172OYnA25UVnkpy6hiB35D2ZEhfBv9OdCe60ndr9bKJHyUAh9N8KCzJKRLSbUCCmsncCHMwU0nYlsofts9cP916IrotcOPDHBt5agy3\\\\\\\/Gn9FD49Q\\\\\\\/iz6ZKfvw2r0RgpI\\\\\\\/kmvIm\\\\\\\/WcHP5sQujeNsNmRQwMhNMPVWtr527QyETXt2zsZbbMnQGKjiPvS19R0xIiaT404iueY98fE+tB3krlO1xZO2h1Cttq39R\\\\\\\/EVnbuJ6cmmzSHBcG6wP5VFANilI859kFkRvqYa5ybijT9RjEJeyjoPcXxT0gi3VVkU2LDgLaHiScuMvTcpl1HFkdcgB6g9yJtN92HHMTIuPcFD+5vozkYTzloMe7iRmMCZydO2Rjave7ecy0cef\\\\\\\/qsIiIPR0HYofqifrW85iuLQXwcD2mCyD2hbIqj1XMnhBWpUV4u64AbiWTKRiDCdGt+MgdmREwORMe2jPiusPF363HmCudc3wnOLAQf6QITwCDyVhvyEE0kKzIUt6PfkvECU0tAqHwtrbgOnd5\\\\\\\/yRF7oeuYTG1ksggJM9ZNl9+XWUirpuvcmqhS6wQHwhG+oPW\\\\\\\/8jcvP4mbh+0ICnv1+1hO+uznU5kSSWC0clzM83lPnp8mNlGWa0httLUCVqk\\\\\\\/00BaQ0Esff7vu5H\\\\\\\/CKSPg9y5GNC0rZBq31pmVjlRanISaTYfSkup30qUw\\\\\\\/\\\\\\\/Hx362G70XNeAhmBt75+6f7YgIePr4Mwv5tYH5+5a21gopvSGXcRDR0HjnRq9Qw\\\\\\\/u4PSNMm3yBh+NyqGpu8adTngNZH4tSus97VcKRToHc5t5IACRJzMK7Jkf0d4i3rbl0Bk05yuTraLRuvzhLLUaXfBwLNDzEc0GR0AIYZ9apvZbJPvOgYjrdyeO8jceBSE3fL9KKj3DGBdmAE06earifKbjMJMdfFj43mLt7HtTSZfYx1XscnCN1cFpM1HXk+EYXcpT\\\\\\\/C4LAZrXmielJoml4bjodi+UC+FBGBjtVSWJCXPv+cm1PBO0DSfHnbk0wGn+6nwJdK2WCXjuC4XZSSGpQHYATH60mrHC5QzC5vzlK8KFcDbfqP41ABP3jLaYwuaBJ1w7f54nErwIssB4UkPiJ6nmGPaSQPM8VL9EG3DY8ILD0zKLlA8krv0G7tjol0SX6OF\\\\\\\/W3wRMgypsssBwewHEFCmjCQ\\\\\\\/zGaEWlpmZ\\\\\\\/vEZLlEVwOUDCYGHnQ8Lkyg4DEiatXS+b0vSBT2Aa2g3K7zaDwFqA6aHT+Qcxs29QYGcgjuFjNWjFuUCZXW58DREvUItZZafqiH9QrZpQAfOVwIND7ShXIupsdCayrMxkqWwO\\\\\\\/lP7ADpF4zoCecHnvProjZmrXUQ0etWJlYgaYZRMxrYUDasaJXYMFsSFiZeoy1VLWriyiugkONOfsIoFTz+yg6lnb3sgsWW87D70HLze83OO7+H5Ol7HgzVV6Kv7Ji4aXjARRxoB5Yfnvea5pUa\\\\\\\/9B+tiMBTGKq9rnzhLs5ch4REWzNP\\\\\\\/FlTKC5vWvH4Yj0o7L1IZ5xjAP4eIbNpLsLYwHNjFiIZvjw5amyhSxpJp0OqLNgTqgMC4G\\\\\\\/fR9XP3ITeucPRnITBk5ohVBmv+KCOrT2AD99pjoIPsImzJZMjoRJgZBukP0Xu0q+G9fuf6NE\\\\\\\/Rn09GQ\\\\\\\/LO7MOaN7ti9P71+29zT3mH7sERm95rriZJI9DxUg6Y\\\\\\\/4Rngkhf8EinUikHEO0g1bRr59eXe7IedBnnrB5ZDO4TNvTLW8QP37KvH4yHHjxKQdJxAT35vphyNf87s6yRiQ2MJlM5yvzRY+dGvzZ1D0uxDWj7inmqA+9a+vUwI4LerjyA8GR8X4rUcAJshJvpRKFXw+ZtMFesMM6D0ngvBzLYhkX0GDTFt2iA5oguX0M4VFFWBxkdTJvHHUuAXrIh5XxA4WfNYsLDq6vcD3n5mmCQFWR+MFSkLN\\\\\\\/lpw+R+rWIwH1CcT\\\\\\\/zlnI3YA+eHJ0+2+zHHJwaERF5PqguGGBUHdqX2GrUjIlXnekflFlpo6XV8BZtSkBdYh\\\\\\\/yved1qcdLPF2q89qHzyF4vXgABIm8AJT8+pvVTMBaSe0uxd6RrPl6vU0+UwK2oDHyfZktft\\\\\\\/YsUFYN4kUm2xeTSy37wuQuG+gUlfzZSHDyeDZ0QwPs24bGXqvzUIRwSZCGsfL087WfFNn6YHeRWL7PNA4NxLrlai24pehQrZI+MQcVrcpB3EFQVgGhZqnF6iOORK2TXylbAEPUvNHD7Buh5XByJ4A9ohSLLaXN7QXUub6NLCQVUWqzBdSffM0Uf5vPsThktwtE3tMSwtejqqBPhi787NGWbsf+GqPFN6q\\\\\\\/XVCGgZW6XOOTBuT8QhFq2VgVe7J426kHjYkoaKjLXnNnGiiRBgjrVXRRSgwp4BMWNnGdoDKQmmbJ4GMQsmXqxM9ex7TTKnx4gmT8++L5N4eODWL34qu97RmKYgcpcs0p\\\\\\\/p65nQWR8BR2fsxQGreJWzxjYoPt5sFuMHmciTbVwBfWeD8GKhHphckI2WTg2lQ4shlVn\\\\\\\/9R3NDOalPTjb+2qQmd1AeOLQUt8yxi09ZdG6QRaWIwZnTC8VEpCuPpYEwLcnyC7SMMRU8GImZAcUQEq1xnpo5vRPpCm97oR86WMPjRsjYU9oeUVaKie9AM2HdvNUq8NuQcsi8JO2YYj7JB03M8Qp9HHKPnoxgmo9Y9+BTSp55YqVCHLhQeI2Rb8lYbeJcwO6mUIc74vfQifbmH0AmwLq9TF0mZKUEt1Cic1fXI1vkwnc7oHhMLqf31JrQIfR\\\\\\\/Mx8TMDO1HV0Y6bwg0DxR3zxiKme2jAU0dB\\\\\\\/OVk+RILmd1BlXEffLII39Jmt6Ovn5q4GCrDhQe588n9BQM3YqU7LMY0U8tcNIfW0705loR\\\\\\\/9K6FGXmzAAjbLB5B\\\\\\\/7ruYcOjHonROtD\\\\\\\/flRLfWl11raAfYAa50i5dAdUkcSqr9IG1A3okTUlYnyFuvUB7PvRdGzv4mBMmCx4aAY3Og9z1W0j+pVZASFf8EGTI1WdTFliG\\\\\\\/BlwDcV8Ot\\\\\\\/\\\\\\\/ySCPgir9JSTVLNaaGTDDJ8zgjp42kBnSjfl8UVgvucgiiXfTfJEHW4SgYFNMQXJG9wO55QWZiTHcqGfOFcjLTgM036m4VUPsTFHtMntimNXzsI6gzgm5c302NEdB8NU6d5NE7igBC2AK1lrsy\\\\\\\/wwgxyUEK31kaC1jVLqkNMvwagLnMx+5CUKppnvRe0dSYLx9u35n9WH3ZQbciGJSf0qaKty7Rqld94tiRQxI\\\\\\\/uta39K7eoI3VhY+pYsvjqwij34sM8jwred7YwAMu8HVbIffEH+5PizbwANhs9O9VWlAA4jjOeDwAUnQsS7BFyXgjxwboQleVux7gOj8XzS5tRJKvkmrbEET8M3aKi5\\\\\\\/CDyUEv+VC7nzkM5uqdfF8fwLCK96q9RUpatS1WoSfOSJMd7HPJe1Qf0kj1a+In4mSWQHBO0r50KoA2b52f8ipqTCYKYClWcCf5Y+SMixQ7ORL+\\\\\\\/WhLKCBa7DLkLcBdgRE3M+FyTovPPUuOA0EPHrxF+DzGQA6f3SDkOItseNukpCWOH7DAPw4xbAmc7uPChy6NkpmxISGvpuA5CEL\\\\\\\/bBPylb2GL0p8S3Yu\\\\\\\/frjszXdNFnpPg3E\\\\\\\/ed\\\\\\\/Br2E1R0eHs6sRLXkzcQ7Uqjn4nOl077uLkxtEwrWr3+aIgc\\\\\\\/qEaRHCdSy\\\\\\\/nFVxJNYkFuc8SaoQYts\\\\\\\/DMqCdUjDjWOiFxA2QIQ9yp\\\\\\\/e+G09KCkVWX7wbMbTP51Wa72waQxLNA78SG9OubAaqhGQ5RZV7nVw4ehyhAQyp31IcXH+VQCD+Zq0egZRVBCho+2W+aQ1IFXS2q1BySJDywHIzEUs4PyfefnM4JmRJ8kN+pGcGv381hb0Tb\\\\\\\/FoO63opVcB23K1plt2n8STqHwtAxQ\\\\\\\/8kGkMLc5nh4ZHZKFjSTIdmLgCm0cIziRO5ch1+TSP96qAbKR3uU2rQHFeajpkphXr+ohKk7M+6P\\\\\\\/T3Rt5z6tYkRl8InYLWZHhpxhA1aQ0yiz3UWUaqAPo8tLKjz97ISUrwoRWQqJm49W5Zzy3Jbf98Ms3eEr+JQ9AvvC9oWW44LdySnl\\\\\\\/6sunmNuUa7TNleaOFbdRYidVAin0NgYyiSLHFZi0fWStQJJLfJpaGnSmRFDdeUJ1oAQ+WOVvClpo5h01nqyw2opCmKfx7UlLRF9s+KvWn5ZqjDS2NeBAxT3alohmoarJ\\\\\\\/wtKTLcdovL4\\\\\\\/Ud5+6ydWY14GuQHkPsifO9uOn2r4dzFxAPrAjZHpCxzE2m+XH6NlVnVXv+lLb7pOhsn7pbP7uIG9A6xZrJmT4iMLKNp69JLGlHbbl+6gu4o3TQvLZx1HJ2czN1LtuAInJ0d1ZeS4YDwlhklM6larilmvTdTjGLr3CgZ2BDJAgTHSAvuYx7eT0+D4FKfcMze1fPq7jONaQ6OsPeJ5cE0NqhyI8\\\\\\\/mfnC5okPnFU6LY1OpFNdt7YaimiIMRYKZN0Ad2U6OI05nULAncgRnJVryY5auq0N5nNtpMlXG9AtQAV4ExdDhRVYEpftzwHj9M04iVzDcHursaaiFCcj6otljgB9HjhzLQXv9d8PXl14yaH+ONpNco3JcYvcZM2tEHMHm6UVb3nby9lMJw3AeeHH\\\\\\\/4a6lY2EVxSFjvxvnM2xSNHDHhCGerkfQBcX1gkyQQ4\\\\\\\/+k68L2vZxKdg0Gv0j\\\\\\\/enPV2nVA105FIgsB4z0LJ78bZrvQWEtPp9m+RE5m5l7Vkqnqc4TetxMkPmC4pFi9pb3TnQJro3Pg1saW7oYQWebvIZqlACpALYnSA8IPeiaPsGIgLP1ZZg6Dkjd\\\\\\\/py2yeFnqfZhKJS\\\\\\\/wMwUuRmBcF7KZyjROm98FRRf\\\\\\\/XgPpR8KO3HakjkymHfGIlvDgkAXZpewsqCTWHRlqICh56a0w9Fquob1q9KsrzbVMhk6V0DavtcoieGJlprL1YODc3anfeczq7GeQhtRbfe7eyZW2206oLkHwRKD3PoC5lyxqNBEnbRTBNmw3xF\\\\\\\/10IdwVWZMalQK5IcR4cw6oiRD+41HMA44\\\\\\\/bjK61vXSOGwiqXAsdq1LNpt\\\\\\\/CFIklkrS68BdLvwn6dMiUURDMvQqKhk6XyEg3sTOBUBxjukf0iysp2fbyvjpT7jcbg0VgHY4t+ZXmPT84TSxtzITnnF2mlHl+gDKuPpS5FN5L1GWbujZQBLbT2AtSUDNonrdDzzVgssUMNXnhl9R20te03WtYSMp\\\\\\\/tyAbRd6q7r2HqHyLUILH9t8lo5iQszPiWaHUwDQtAy5\\\\\\\/0wFkYeFg6H9XNEOZzDXBYrr4ghPS1aTgPUbwBw\\\\\\\/sfV8xWbKYF1NAgWHUOi7FF7fs8WsycbSq6W4f8dcYbsGFYa0tfgUvNZcZFtHUhKWbGPao2z+C7REkf62SoqCeVRpnEgO109ZW03szuwEgWex5bffbJcHv4S7IbyfY2NCswFoP8CXF2VAWn\\\\\\\/Ra71utfmAHcpejyUvyQC1k6sBDu8C+NUpWXZ7nlTwa9yHq1F7qvk1OSMX0ItPhgR9hOEErZP3D5n2iESm0MT1oN05vcVS4x+z1drxWV9cv+eWGF99fHiFa91+BIJFX1u6Ge3J9qBIkIvXEbxudzt6UcXZL4M6eFKaWIsczNsBwRZRtew3Etq\\\\\\\/etxG\\\\\\\/H5jTL+Pb9664QKwzSO0GSzryaKwMvi4bB3\\\\\\\/uyZCPHHPPbKZnexQF7oYwnwN+OCnzUkSFTiko7BIBILPhke7CIpi9\\\\\\\/gzO3VZ3TEdbIdDSi5nUlMcqLoJ5iwi1Mq3NYvUvjfvnIcxYtRSlvTJ5dNi\\\\\\\/Fp9\\\\\\\/f0cf2evX1nyc1s+ytNhxhUP4+Zc4gkSvUUY\\\\\\\/5LjGx2UvyceSnFb4H5+BOltD8aN4aeiee7t2Rw0wjep5c\\\\\\\/2YcPHadj+GUlG6ggia62WcSH+mSoO19RN8Dn1noISYbgbTU\\\\\\\/BWodccf53uRurJ\\\\\\\/QBsQoLRNM4IZpdbyuzxKYjo6mbMdlTQP8mQJnEX85wiQjn08bZO61Cx\\\\\\\/RSjloWWXMXrtFaPMm5Xx4ycLAnGfG3qcQoxU8QP+5VqCnXNSz5KmzHAYCd573e6W4+PK\\\\\\\/DlX763UMQUd4ClbXQNJKoP9DAehjTYFvpQJUkgwP2+NUByGnK3EVUAKT5M3iJN3Z3W+dEUar8aitU4NPXEaJnmiJBODC1tCShBQW8Ia88KeBJKyszDP\\\\\\\/Keyj9sfl+35syrgZEUcuQzKnzQSSBtlu6yZtKRv4ao3\\\\\\\/bwd59mkTitWvCPhWN3Fblo2Kto3\\\\\\\/Z8wXi2C7sGgo9n6PM7yXVDngQLjIDubs6MEtSW0\\\\\\\/IXBdC0trM28YSgyYgwoOepWEPEM0Aw5BzSbZfDHXmFlBUO6VCpVZZptEj1RbATCGrcRRccukoZHnBLBQ1sX497S\\\\\\\/9iLsVFxKNuToRcRwKU6Ak4WJHUS3AgoSLuwYPq2xCYN9uoGitjtM\\\\\\\/lisu\\\\\\\/5WOX8PIjZCPmK9Ua6fiMiz55hGW2K3bw+WD\\\\\\\/BFSKOYQwbxtNYQSpHtGmJ+W6qpNnCc4d5rwJ0qgmhKfXc2+OiI+8lr3PRNvWX40EzctXGtuzL2lSUQqEcIo5qtAkhaF\\\\\\\/BUVl7oxm3C6dW6+hCTIzg8vNo9KK60cKz\\\\\\\/1xt7QEMiLEvbWh+Lr09EP+PmJq7Pa0ThK31lg+dHyupGgxJkI5B0KCwF0pbs19xD4k5iBbQbBHnvMx88UmKOoqm1VmbZcy+a2EvR6o8LetDLKmTYvLJDjIVtSssl5lQLh9vjTbpXKp8MsEE0yIWCD3x6qo+rlzS7w9MszUhgEOCKxDRYZOXbYGKrgQvOcD8Lvp9\\\\\\\/viG6XwovF43YSMu1YwtulRx7a7\\\\\\\/jx+mpiuR8yO3oTcnwM51q6HoJ7Kz7SYGsokezQA4g+9gwoEAXaKrB2GFtSMswM4FrjFda+3320W\\\\\\\/lfCAZZkVNEZox4vNI2BJ2Si9iKIyOCvVjPVLhno0BBGuBSqEe+m46sMYld3Z8Qd5jk+\\\\\\\/mTnpPB75dS2hKx7nPnDvzPxrx1WFmbumuUE9p\\\\\\\/UB0zYHT5mLDz4MloBXU0JheBUqdVWZHYhpSXAqv7RDU2r8VZxmUWTKAgO0VMJfAXoON+9GKSiuWpfoqbkIedqtUNyyoEkAD11XpVFGkngjFXNJYbmhGN3DVLeuoRp4frnb7VNxtd5j\\\\\\\/seZnRdnq7siNuMlPHVqeh5WQN3T8SDDm6ebb8Gh72w0sftNYnzTsfQecnUz+eXVDSnkoDqPthmUogdvkGed7e38Sh9kMoZMJu1770Pm7qDtewtweSOY3JWacdVOdJtdOB0KaUNPinfGOJznjB87+g62bpnHpud+1ly\\\\\\\/pqlFnrde07O0UcpV2fepQLfvKAZKmPaoC99izKpjQrC78t2BX2WOnxdJN8XimffIv+GFH0\\\\\\\/5MSRiG9VG55VF1gfCqwNm3RjVm2zkSG0138Wcfcs9IOx3EvHhmNYAKVpnvFj5Yvl7JuTYu9enmH7xJhwXLIudK7pfmWQD4I7kZI6\\\\\\\/eO1nMapMsmrdGsxWWl4Tj7fD7gT6\\\\\\\/EHpVdzulog7mQqeLh7jOeXdYu8jxWdDRstCTvt3uJTuoGKSD30DgFLG2YUYpMOYtNoXQlTYL2njpxVa86J2eMxEAX4zkKsCYW\\\\\\\/UwunWifAD4auHJLzGK0OHfnRzxnQD64Vav8mAl2FKj4I1rEqlZxW9z2E96Ekepo5SKriadIPxBv2AUBWRTo\\\\\\\/yZoL2bzQo1rghy6XytqoTn0VtWCLO21FsDSDRyhzPuFcGjJAyju1JOZBV6H8KwIpMhYW9czzVW1Ds4z+5bQK6pPsvVsCX79M+\\\\\\\/vU5ljJaIQ3AZI5PE1xRcyI60COoZspUEbX1PNg47GuGc7bYDTUjcc8S4ZobVZuVxlwhYgg4P0YP2Jj4dgsFu86zc01abTZtFOVItFC14LfBVfJIa3LJ3x4naVBpCamVGORZzq+4minIgaXhIJu+31iPg+fmHo3kBbkGYOFXYkLO97Ci+2mdRy0t1TMBhcO6RD94Yas9FJzawdAyrsbocnxoJGQJvJK8FoMH78DSC\\\\\\\/u9QzePXcmCAv3FrvKnQI7XFJ7L8OXXYwZZwBWq0lJm7UnbJQ63PwD+nXKF9Pf3CTCPkNkL1yWGvd7AQ59RRiQ32ASonoxAA5fCnLAedy2xefbHRDu\\\\\\\/2Y2WjZ+E6xK5RkaXoz1fu9A4ke00yz\\\\\\\/oz86SWocHW9xh5T3ooyNRTVAQNQYFE+yZESUdW8QuXqH+Nnj7QwQExX3dOxZ1ug6JJTOUQK\\\\\\\/xRix4OMhsI0yiM8+GjUD\\\\\\\/SAgyIUT5WhB0iKU7WU+yQBkD0MOuqgEiHAeJcwcXqpUoaAZIJWCqzwUTxamxUx+xCHjAqlOa6QaZMojAVtv7DlxTmS8D24t3XOp4kioHDEhBy54RlzC4NXONmRk4vcSy6t6y87RKXSJ+UFxJhR1Ci2OnaMSXgqnhqQQTPKy0ClazL3PC+F9M4+0zm0E0OnA3y1Ov7q7aKSzV\\\\\\\/pZJeFJzJCtTPrKbeagslmODoViMMf48qTbAKfmvBPLQyLj785JXAuII3YJ\\\\\\\/dlbgGxPTQIPFOou\\\\\\\/XP879hgTi2BpCDmQrqUGX27DJOhYgKC8DLho37ebC454e+xE21GIyNU9upogontHizBX8MFjbhUiUHAMmf7XnqQMGiwxjT9f696Z3oSmVXmb39X+emKYadQpPMkCCMjvlQ+Zaj7WviT2CRiUOQzssnMb6CO+Lp+Tj0Qfhbb+0Ty9TYjZAU+lLukA6oOeGvPfakw3+yqrsDSekaWewDiZb24mijdx3ciLZLt+qUWMi9hNxhuVmJQzrzSgxsEt3YnBwk0EZyUKL9QCqFXF96EBJRnRjFXo80q5RyC8pZTYZpp7XO26m00SMcnvz3spXguLrsBd0ReR5efV7G6nA2ecqblvL3bZ7oQkfCNmA3g+gvokf4gAX9mBjO+K1i9ggkd+g3bhzvlIDxQuIgCX0RYQ+2P+\\\\\\\/RIajKV\\\\\\\/cNkwUy3nGI++bXElKezyGXBe6AZYe0Gc1fXH6KCVNvAMcEIFJQTXtZMrHyAb3PWY5AWfRD54O7DU0KUIojVInnrikfNOzSwWp7NAPp6kydXPgfxVrQYdaUJQWq6YrU6tJkG6g\\\\\\\/0OLz0i0tIpDTbi\\\\\\\/wK9IXKiA+Jv5ZfkGtlvDOoUTnE12hhOoKoP5wqVH72GJmZLiNv7I\\\\\\\/EVqk5sMr2G0QZ4XU3uVLq7HLdi70av0\\\\\\\/zz2BiQc526RvGhCVd9N8dvDnPnSjMV0SeT45LOKxe0t93ttdhK0czcGNyPdUGFO1wDlyr0y8tnvaEe3ibIYMzlFDKGoZ7szQGkHv0XMaqVzbBR+4cDGeRurlcHkvIlcu4hDcuPPzwPgaE+KcE8YlR4x6TPumr+B26WpNq2ARQCp+gfeTBgk5b0hthEr+6pcmIaLEjYe3fyFKEG7xI4\\\\\\\/5zFXUize86trH\\\\\\\/+SoTpRVlwlPgrBXoUqm7NQwsSpgw8HDdB\\\\\\\/ZuL2pfagA2I\\\\\\\/L\\\\\\\/dDC1gLhAErsMzBGXdYoiem2afaGrszCg2Kq3PxyZCWs8Q0F9Ao7X\\\\\\\/cCSGmJs2FwsPMw7BJQGvdB92hXdXpgpBc8bMo1iKuWEKM2qwx5Fpx6BhE+25e4Nsae7ui6ScOVmUGLg4BZhlJN0B0MVph5tE27pTxpyEMJa95MxI2SmN0ZGbhryrI2JcJvmUV0h72Ye1wAVlZ\\\\\\\/vavQcHeSbOBmQ0x0t3KVGWwjIiHonqaVVhN69YSOu6ZvYY8tHtiXJ+izkHbi7nnwt5vMZ5upV0ABcsR3M78XxT\\\\\\\/jeqI4dpIGQNXuPyo3Z0Fu8s\\\\\\\/Mz8A67hQ9C0bxT68Ti1nPDp7l3wus5pMX+t8OP4CoBtCA3xIujL+ghleHJzds1jbkcIB4FZSvssu\\\\\\\/qZeDqnfnuruM79DgxbWEwEZMlhv4VEzPrMuLhWJ1U5wZpTLHoQBvWTQVHs8oEOOE+mkL1NudJEIhJdtqvhp2G6c4Qmu\\\\\\\/3jvjoJatMiKMigii6iHpLht5AiUW4XgZv2ub\\\\\\\/AGcyRYpgtqKkdh2hNCL0nY2x+2wZw0zG26AXIumBWBegKHKfCXkvzVj4lCVlXCLOcgaW6wgqGtkMul3iThRJ28Z7JQRIc4j0Jl92eECKLrwdwL7BodU1cHinSO+g40ZP7bhjiAhDZUMtC+KAg5P38UPulsiACiwNkRS1k50FYjE\\\\\\\/e780PF1DVD2qwj6t2shMsBYYoD0uU86I0ZgupRePNftL5MujT7Fq7ODGqf5sWF\\\\\\\/8FMhdqjhoJCWgjlwWnX0nM31PC+EWzlQ1fGg4EivcJGBaCfIsIjJgf0JBteNJBmVDRFFDle97cGfSvxWZzXsgbKkSUq1PaXz1Hav29d0Q32jsD5wBx06yerjPJmwfFtxPuNNMIRIMcAZMuWt3CL+3rARrUOHztQiNEx1MWu8rvJFBcrT625BX9Ce+MCSgs+x0YMmCAZcC6+qNk0nFRKEsxfl8haIgMjtoKiVjdWdIsabwqNo47K6VEuUWFVAR0IVmYZKiIBGpQRnLTc35gJFoFF11PqGGj0Qz8MSyBF0C1g1gnpas6jtkCKPuAjuorrBnzCrH+ZqRk5eyCQTg8unswW8uqP6zGXh2yUpPO7gF2h75ZV4jhLFinf7zEsYcUIFijabiot2vKs1yeVbYZqdtXNcsuxmLryboPrNegAtiCS2hEKGorEN1jhGlxQ4ssv8o4Tv8kVZbiYLlcK0wjBykeVwIh68d9LaQa10OBNl1GZGM+IKXr6ljWgfQZ\\\\\\\/TTO+UEXWUpGgMyTmxF\\\\\\\/69vK4Azh2vVBDMndAK8f5orUweM5a7jPLFw8BBCYU\\\\\\\/Z+g45SglpIBESrrgnssI4S6073QzUGIUfS\\\\\\\/OejDnaEaiyRVB2WISkT+wmayRHkSQWuaZKaVFtefyaUMesA4Bko6W2mYhpVKXfMx9Ybqetz2w4ooMCjZgS+jwMlWUquAjNmi\\\\\\\/dQ6jc6cGwDsKtfKUaeSHjrhx0SNcip9bkCLAK7G7wgNK2WDggBRsC3abK\\\\\\\/Du08ieQ7OI3zbTkVlzUb3mBEfLMRTOA\\\\\\\/oa5Pzh0m69GbXKRqlzaBe4tvd\\\\\\\/IQy3N7+S7zSmGfPVOAytAxXVvGV3o\\\\\\\/R+jS\\\\\\\/fra1h4aIuCw4R5sUVoo3w3Z8x5PxKKTENvB+AAL1WXz1rhdWT7LfzAftlEW6NMlF65MTTyjrR1a+T4PuVG10DsNEwi3AmJrecCH7jch59RoM7YduJ+nazSZ5pqEIx61Fu7f5\\\\\\\/FD8+8FciBDSB+NoCHRGsOxbuYpSvHyRYMn2xE5ILYi\\\\\\\/LBm7VsDCVobxAmHpIHtpIhtiSn+fHGb5W9tnV54Rh7126c5sTX+pHkHq5dnWokOGIdA\\\\\\\/\\\\\\\/ZxrAMmUAfMAsamR5EZCPcVocObfrqOYVz5U83ZpfuU7o9B6MNVF0OF1mLhqpnzR+qBCQI33rs4CHWrKg21lb3gfHvlNyuWlHmmyJp+llWLNvHbMChZYd7H\\\\\\\/XGESJnenii\\\\\\\/4etam0ft9VFfSYVnP8CqPXZnXsdAX0iKpfuWj5IIJqVGhzcoJ\\\\\\\/o185pRDplvWywKLCEsrRzapGabpmE8349svleG9LPx30k3DK70XlXeYJdsuRN92bCYzIyY9qC7nUE9IE9JPYcu\\\\\\\/mhDq\\\\\\\/al0ogx2i1z\\\\\\\/erCVNf43853uxeQkfK+d0R3ObLFWRiswOcdZ0wRDtLSAvtoIVjnx65Dj19gVQbqiVdlhZG0Phg8ayo1kFgMH9ny7YYsuVTjhlSxErIvkpUuOK4z4\\\\\\\/5E3ZQkgd7tb4qafV6UNHyLxFABh4Xy0L86WKQpBm6JAFiAeAoqxB7iq\\\\\\\/8IordUmOAbV5Bsn8+deX9JYeokdOPKesnBRO9fIHEHtwiiYcVX3zktaBqO74F9+nbCkHlGpZwaRPF7w9yG3gYq4GdgtbpwLJu5XkNqrEIJ4ZoZG6K1vRJkSjqcgASvLa4iSpVGiBFjZFnmk9u1DKn430JCzuLDB0Ok7KMzRQu4TX3es8zLydrDNj2IQXviKD2lKrDc\\\\\\\/yf9Y53qN1hc\\\\\\\/D8kb3QSVIOzFXcyH+Aj6n1CQ8CDM0Wuk47TJvCEUXjmUdCbLomceN\\\\\\\/dGhU+jqSajL7cJ4X5PH4hCGSUuEp+0lzY4c1dhupaUltuBpcuEbJXZMtT0fhXmZLn\\\\\\\/H+jG5Z9f727FFg5cUo6hqhriC6djxdIcptPhhpCPpz3wMy9NB0Q1xIbb7cYDAXVBaZQeqWbLCiqQXHTwK2gcageTFZ7qNMBRLPuWDSqb71u2KFhD7HhfGNnPQiAxpJ5x4wpwuzslNv7jlXHN4DbUsMbdAatfB48Z+cM2N3B1uEhGfjc91j8gb5qhHRNxIL2ZaSKzjSw\\\\\\\/Vvd9E+JDvPT+YxVvbeFQI4wD\\\\\\\/mUfkMEQgr4iR6Uc4FhROed+3LrDykKO9WRVmcQx47KjOEQXbKeeNhIDKr\\\\\\\/2xEe7g7WzpQSrxP04c2aAg\\\\\\\/0RGx4fBrl2stJ\\\\\\\/823+boOhEejw+8VCpkHqVo57RlWC5Qz761OJOWTKQB5E3bdiNW4\\\\\\\/k4GzAqU4BrE\\\\\\\/uQ8Vmy8vUKt3+POjzBBZvPO8U10U28SErb2dgUz3sVbn8KL06I5Aq0+yhZHHIivcqbVI3JFDMnXsjJ44nvBSYkINnO2dSKTuHP25xE71f5kwtg0J5cPLJOLzjRk9wMNLNjiAMvkbuZOAVMYo5M2694V3dYG9LVeeDBHjEMSu4T52RXC3QtcXNf6B6vfCp\\\\\\\/TeUPGLDQetN39QqnrNDxdvxGnDkROxleoTVKwuk4uK1Otd0k+9efH2iYKr5RDlc+2JKC2BsOpXFjc+GLPXs4VEr38l9NmUkpYrodUuIaGGhfKdl8CHW+eT0GxHu0ZI9y0k4qqmxJqTPuPDbX6yPMzqtXOXdAw+N0daoTzFnz5JrmFNvuJ2Jj9TYQD0bNDEcnNgSxDFd9McNvA9koL69yr38SbrOsOLGoAIIMOJ76N8lgTPHN+d7vzaxrJLq3QFfRrpEynRryyrdppd393XWuvYMcXTC77t9HNg6GjpJMSpt7kBA+U1GmnZkgyctFtzDJxcotHuQsqea2VMSQjE2F13bISfh9rVM2mgmGTV3obvqHHzecAzIYD1\\\\\\\/4nSOi7VYXrUdQyW40StkXLZzo8Ws2aCJkaojDg1xYyptF36fSOZfX5Lup8pE3sUxQ9KvhNrpN+LGluvgUXoKhsMo0FZBQnby+d62MBPv6f0jCwDupj09MgrwaaYrTQVl2eejfLbn81MwMgmaeGHwAnocnWgqVc+6HvY8kFsLzRXFEeHHFoTFrEoRlCR6xTkJJQWkLs8jZUykbkbELXQ8lS1i7g0Dj\\\\\\\/yBV\\\\\\\/4JDA0mdweZQakNlavAMmWZe7+gcQFTuudYGWp6kNAUPXFVLjgPeGB0VhwD+w4Qp9h3SZTxTWxSBRo2\\\\\\\/HaoKPAS0M4oTnagJyd+ylwKovob+GLu7wBRNRBjO\\\\\\\/Ag2Ti+WeK5aEiidUHCMDmgq8IzbpO8M+UZIucVABBymnSORWQhcDa2LHJNInniHBvLJChP\\\\\\\/EtviGsWLE+Y96Q2SMn4r0ncvDDZY4krE\\\\\\\/cDqAJPmf5EdoXHqYMnNk44DoAGv00Z1ACjv0RgK7nEVPxJzd6fnQHZdASF95s17luruog\\\\\\\/eFtg2m5zmv8MGrttzTG0zq7hi6XGqJi8BIAeyNjPfKv6nuKX0ymskd7Y\\\\\\\/ROaVZ3WuhXgQiGy5nzXdXS5hLbwqpvieE4YeRa7cVO8tAi5w0ookJ0aFHPy8kbeB6Dv2ULt+1HMqhveAkR2KONK9TCXczFIRz27D5azjQjfBfLAW2\\\\\\\/4A7WP3OjoaQZHeOE48Kk4cC8vjSqvhLduKKmF+5L9bY2U2G8WWy6yeVc2YwD7ObPbq\\\\\\\/tnlbSKqAcTIFgPSjAvCOt0Cn83cBbpURMbBvH4L4OfLbrKT0hkyF2FSy9pJoRMd+MGcGG+5DSSF8QH5ToMZ1oA2fZTyA52ZJruOVgm+Iwlj9HKmddLFP2Tk+HtSzMrFAimgHAipKIRNkEAwaSsL4s91Jn\\\\\\\/hb1Ep3ISr\\\\\\\/toD4hKkHugJZuJ5SzvO6j+qDhrzVcq6fKGWey2N02O5nTxPHAPaTHI3\\\\\\\/JnwHBv5NX7TzNIVA6R9sAvEdMx5D1sZu9839cMX6ZAkhiHA3fl7ZDNwQ0ieMPBRBWqGao4kcQTesDz3fPocXQBxcPSQ2fj3m8FzzY6bQkKCnGQi5Wz\\\\\\\/zFZLlsodmo59WynUbowZ4DXRwVZl9S1xdi6rI56sZaCqvoW4MCoaJSxaERkeROhS4FRvFVYwZ+C1u5vwYkpR8uTmDKACfRxjTJNMkRhgu4qt3bCIb0kA+bHGVb6qCuUV\\\\\\\/hRDzLGtAaaZ09+q4Y43XkdMnu0AESm6A\\\\\\\/HnxbURuM+ZLRzSJfzq8yo8UJrzJwRli\\\\\\\/sfxr9IcFrqKios1tISUL9sgNfVB9rzf8sFipWlhpjZkyU3NsyblxLq+7COcU60purjeTa7+pCz7ReB8YU6vYJy60q6Y9yBJN\\\\\\\/TI9MesbhEl9hYxVxiulgLcvEe1Thm5JaBpXbIdsalLqrSTdrVy2sxVDkkyfLCZEypsZKzk\\\\\\\/8b9JotG0CV4+l3WwelDu0IU5D7hIqrLnLSy2Gycdp7DiHFC+mCntd4OBOH9DUMIYRA9RSTzAp3erf+SllNOteAJ4SlGzeTRS9gtC+Pn93NI\\\\\\\/oE1xngYWk\\\\\\\/pBhLs1w7pXjGg9Bp5upNsEQMzZJ8ti77\\\\\\\/7lInJi2EOKq1QcIKe7ptLsTfUwM7dDgKmwFV6QclAhZxMQ0PTOOUQ6RSjtWflmZkeIFT5ZgJOq2cJPUVPnNRFnfs6Yho7KzsSXoKgAxbzL8sD8eZY\\\\\\\/KE0pd2gRJMFSCsHi4mDxjTXgHrI0sqW2ZihZZDNFIsixMjCVi4+7wNnQ3IT\\\\\\\/euEYiTh5lRRvRB6+TUmgG3keDB0uAfxoD65\\\\\\\/nbYggid2CZBm0XLZm9D9AiKAbvwe3uAzMxdy69gwgUiG4dGbPu+lsCPJ1cRgY+whA1\\\\\\\/iyi3S5A\\\\\\\/ByrT7h\\\\\\\/6i1Ol7eUVGOxUh9dW7y+93j3PWAquYEKxJ9xlWkhV31PLHcIKHzvUJow\\\\\\\/qHljxPIc4Ej88SmPOEDa5BskecnBogF\\\\\\\/VdVD0mLXT\\\\\\\/pyXe7tehfMhdDeiJezmM65sYBx80zc5VZqtUwlltDUaAoeYrXEETqpjckgZzEV3hkKh4yVc6dQTl6SFuR6SDw3ZtUcYEi3GHu4ZH67IWvMgHrv8wIcNw+bUxanQ8RvKDMwOa4riYwW2zLuWO4INmgw7gga+i3fwFdbKlGuw4mtx9u9cjx525ZX8Az7aSWxFn+3vhAVI3uB855Mj+7IWIS8MB5hXi5YdNtefqrMKWD0Um7LFoaR3zHhuyuRsjg7hE7XnvRlYHTVcG9zm9\\\\\\\/WKNygQExeUULHtTwJvcqC9D16\\\\\\\/V6nGZAdEIQo0s4vu9TbbaU8wna3AmYOq3E1zu5IERIJXqac46JBor8CEstbu7r34+\\\\\\\/AkkSDnHM6lOkBZDFEiwPFPZgHCee\\\\\\\/04jS570ZqAb7zsrPezQUyiH9cBc\\\\\\\/ybaVFvtDJj0DcC27ZIFK3zVVLOvyliK8XhPle8NNTqPjoTBfAp9TPQjRWNG6nPr1DLxLmCcWxFyUDY+uYeIEyZBKghtPPLqeZ7wgEXz\\\\\\\/rnHjKiD4a50auWuFfTxmlIOPh0HwnzRGZYNjc3s08P6+R6QFu+Hs\\\\\\\/kygPRxMgc1pTCqUHSg+Am2ahmRD8ThpOm6amzRGwv0fzPsUW+VgwpfH\\\\\\\/TTWMlRCzTAVAd1Y+eXQuqIYaUJaRR2G89X7128hCI77nETk5OuBIXmpNbC2I2BB4utbO4y7ak\\\\\\\/90b59fo3NgJjwCdPol5TNw3ZQaNdY3R6Tzd7B5TY9hcCUMrCPaoYXi5MUe4c5\\\\\\\/U4I0zmHRCeE0fz4y\\\\\\\/tRRgYJkteQm0i648alJaV4KOOx\\\\\\\/ZCHGl+GD6hWdqMLgk0gBtxw5eN4y8fPAJEXSkDbAe4+jggYI+rOYtH+n3xBRKb4aaXV50+e1N3i2Bv+iFRd2S21C8HFob7yHBcCEzJIKwHYgBKXYe7MpxcFEBMyu\\\\\\\/Ek8SkylaD7Cljfc7qApCaGppzFbIBtM7XiX+WOaIoyk8GpXfEX66A9a0gvZH\\\\\\\/bmi014t+C5\\\\\\\/FA\\\\\\\/1qVQQ3AgBTXPQCXCrx17+Ri5q18t6z8oka2+T7kIGwFcKt+pvFP9fMNpHZm78PKP\\\\\\\/7VXObuMqhNcj+FBYCuZlAeIvSLeyb7m1Ef9PTTimMNxE6Ehtkk6\\\\\\\/6iQwgEJ5wxeSFYHWxgyx5U\\\\\\\/zt6LuucvN+vOPJ1JtOn8TbI5LJqvOPPkPNYXRscnmmvBRqaJzDjo9SepnSZtzKydr3+02pcw0JDoaQ4bbGRRrYege4sxcjh3\\\\\\\/TZcufGVJKzfrH65tPijRC9sZqzZ\\\\\\\/gZS3LGIErXjTGJgrzKEAjfXr0WEqwzSspRe1SNxbPm\\\\\\\/0e0ULEDzJyn15rZep0zs+YEtY0WutY+ilnPdV3IR5ZbUwe461a1Z4IHJ1NJnrvMtx5V3pVPayXHRlHL1eTiQbJwGMhhNzBo2ctoHQUt0OgaDqlub1NXL7ZGRm\\\\\\\/EhYapwcO1n9GfRFSIksA8pDuDqlSsqLZqunKWtDnNrj588+sPjZCrrFgLba44cBzGOa2VTvdpNvlNNojAZ6LFaH1mmxYvLRRowVYUa6nM4yibESD7L60ME7VxtVEDfio6xGbjBJetOcs60JDA2oYM+rnv3iQBFoz9ISwnRsJC3+8Bm91UXf6BNFMszLzkwyAN0Wui4y9BfkPTAG+ESt5kL3coeAIBJ9scakfsh1KGcvf5fAD\\\\\\\/jA0S6Ph1mcQCBXWlrUztDewhaTCRGhdwZaGPBO\\\\\\\/uctm58WeZcxcRuwZk6csXfkfiEFx09pVdIeg7cf4LLRrtjYNMpmss11r9hSKTjZB4y2GSwastdkW7hNWqIiQSATDt6P3a3WPtMBbEPuQPRbk+Lffegots1JgD7y5xACOWxOm5Ujo4cn512gdq+B4Ny3E9J7F7wMQzRAfWP0rEc1Nf9GdU26Mnwxe3Y2c9F7PxBMjkaTm+n\\\\\\\/CSnqbuZxVrWIxtrlD4X9bfnb\\\\\\\/VO3OGr2IL8GZSsvsyZO2Lz53uQA9xXPk\\\\\\\/lm\\\\\\\/O+UgTKkyjXrrQ2v0N\\\\\\\/VuZfOb3dU+4\\\\\\\/HgZzDGFALOd3gtd9J1b90iqyc2aOCOXxxiL8Kq+qLzuEhqs0nomVeuBuBEER\\\\\\\/o9uFqWsJfTCu8RSMPNTMbdk5YH\\\\\\\/LHhH8Da135kmn+t4Amj3PZw\\\\\\\/fgDRHxuOpEBSTkbQGh7zwj46VoPFjB1\\\\\\\/Qu3cxINVhM7NI4rwJ9xC974v4HFUUHx9tg1ReJydjq4t7K9QGrIpxdbyxpemLnoJGghvSy6YUKddVIUztDTaTGhNtCxI0r5DnPCMWGZ9eZWha\\\\\\\/2AcXc29VBBS7rqSBZGvIdt1rxEZsoid3cA5BjqJhsQL6U1Te1hRmUP02xwts\\\\\\\/V9T0cciQfXqlMkoupwIurXObFYja\\\\\\\/Hpxt8+CuHv1LrYFUSAu8\\\\\\\/BNauBXDzdhtmOcYhRVAwtsjjkAyTOQAVDK+tCZFCZ2gcPiICCRY0XklrAsOGQ9SnUAUIxRfeUs++Vlc5m6JUKsxrGtdG2ZQfHsaUSiwj4GcUQDVgxZrm9pncu8fOVkXOlMYRBTU\\\\\\\/HSNM\\\\\\\/1XLTaZ\\\\\\\/cNttDZJ60zWSWe+VThId34AMFXAHOtqaRmof526kTDA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GhD4AG0YJgaxc2vqHso7hw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222a87fba3259e210334f5ae48a69d859fecf47fad9a9ae01e64367ee093dae63c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229CjBej3p7dwo4aItjMVTw7Mp9JODh0jG\\\\\\\/pCFYoOhXjukMMUfH1i2r1vwkJwc32UUEREDqp+M09xsOfsqjNb+9yoM5V1Cmyv0GdhZNTr8QhumAeeC4k+hsGbMGARzia7uTwd7mGlG9igJgtgKD3TktDFc0mvdsCbXXAvqdnElup+Avk6VH028DCwQCZoaK+xFwGv37kUmvhWh5laIQt43Tym3uzwzTMvaK+rtusrGmU4LqumGHtWFWJMa0IfQJM3kUhs3HaBDhHV+Ruyy5xLePTWkH\\\\\\\/WeAM3Ue0OizVEn2\\\\\\\/PiPftJbsxzqrzieub5P3u0YmfQ1ugVl9RqZWQ+Mg8pb0IKPS4A83NuSC+5j6ycSQaowyFKwgmmlNgkLHvue1GuRCflD0XarZWgfjXllFWEbK5FcS4BNaJi4sv8h0uajtiFYf1nQiNhTF1dDHsqClkiD90f7jDIO0G9e1O69lck\\\\\\\/dnug23YeefvqCO\\\\\\\/rhi9NoGcvB8y2rdP8OLfg6aIEXj06X0flStSryKVxPCC2VTZHK5zzddk6ofQYIKGKMwbLXVZyd7nA3byVlllHNPoOEffFw91n+4S39lncgJdZWZSDz2oNgM+TMFXryPDjPRD8L1VMIaZVBPw8ei\\\\\\\/13ZIU7zrg2lb3M0VWX3fK63rjrs5M7gfGXXEU15ugObfWTT\\\\\\\/Pu2SyPX0PiVRe\\\\\\\/\\\\\\\/XfIq03uLOUP3I5\\\\\\\/JFKhMPcmtJPBEmjPFU6SlWau9L8OKWDJ3cEnPHTHNk5T0977xplDWuMIoDlScXNO9YM2EtbHLSZ4GU4877kCPcZ7r6HM01QlelnK85iXag8X5KRExtrUE8ARQVBWan5EfnSREMhFVWpurkIzoOX2r8SYVGqaCNXAfaLAkM\\\\\\\/4zE0oPeSOGEbAgHOKGf34E4Ub9vHyH4xu7g2twUaSrVmpD8ydMuAQtzniB9nop8aHsvV41zU\\\\\\\/OPhJc+d8Tmri9kfek0Ul6WrQac7EVKzCPia5WXfGbCtssaNnAFQJiCYqxQrudpGhvC4d6nV3qpfsFQ1+yXiKfTlEKw7EA+qWPaTXYDFzVEkxkmQgzHWaRdlEp3FgxJf1hT6ICyWyImPbsZw3rrqoX6ccCuCjKb7NdDqoxYwRLZIKbUFjnxFTszAZ7MhkYf+e\\\\\\\/vJeK0qw5EtEdggVLwb\\\\\\\/4eC00Y0B+0WMrjGTRYgPGm\\\\\\\/TnGXcTkJUlaIiB35NxDOt7+DAyfxKgHftnfG4FfhqzDqedAJWjCt7T37NR2aPSEI2OfU0qznRuWfMe6SXOxynvh4MNnHpuy6\\\\\\\/88eEC96\\\\\\\/i1DlnoSmncCUfwbdXayVpZRedAuzY5N6i2Fic1cyTc+ZC\\\\\\\/ps3Wq21peBXEfwzfpQiDJMJvMUMg3BUkOYEsLEHC82zhRSxJ1oyIg1w2CqSc7rGfAdO1od2ruUY5qq0OdqjRNNnU+ckdTnCWLySoRnb9tW3rRM1AwTf\\\\\\\/66AFsZO5c4iQxaG9p4dRREvTLdu2LkCz2Fl9LpW8RpzwMKJFk6d5ITwjfcyTLA3BnKVmcjhZSJpTpZnLe1DkyjpBRVtAUGwXkKw4O39eE\\\\\\\/7L4gMH52DTx5zXMkq6wQyA3HYIPq7h3tqt\\\\\\\/IW3rBbLs47XyL0Xrw8\\\\\\\/iTELNYmSxXN3LVcJXQp6+fjgu3fCipc0gvwM7qtlw2heinHUJuHalWHgsisSodjbUyVBNAUabrTRiaiF5\\\\\\\/gbCuIeYTrjogfIvzoSiiZ0iQtGEv5x6ZHygS\\\\\\\/9qtwIyTciscv46WBmo\\\\\\\/cQyTR3Z6V8lD9w6owWOQB0U8WU8W3VBcLfpStzi7ljPLVYr+vOx+fJ02WLFo9veFk5bfybISOhiwcAfgyr2Y3AM6XHmZcYIDxgLbEYVRqx1o\\\\\\\/nGbvbEdg2VLeZvDSRV9YjBuJ9u12olfanfiOMHLEL6bDF+UhGkBHMCM0mN6pbxw2WaLpVw+a\\\\\\\/c7YCllQr88EY0dqoT4hmcGHNWvSZlgDf3bb7UHKVIEwJb4ctW6nf0PJHv+\\\\\\\/w+ds538fmIkoQVQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pxSBwm01butrnkJmPaDMvg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ea68e64567971313044c43563657f6d433153659a6607b653cb5241f729a2d0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459974 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1463" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3ryCgonu5WAw-17hh2hNgDCiqdRrQR0amr_Xx8hnAmk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1463" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WQ555OM0VwE9Lpszx29r0bsDaDn9clOrC69dVWUA9Yk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1453" title="Intrinsic Toxicity of Unchecked Heterochromatin Spread Is Suppressed by Redundant Chromatin Boundary Functions in Schizosacchromyces pombe" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1473" title="No Significant Increase in the Δ4- and Δ7-Dafachronic Acid Concentration in the Long-Lived glp-1 Mutant, nor in the Mutants Defective in Dauer Formation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459974" id="node459974" data-pisa="ggg;5/7/1463" data-pisa-master="ggg;g3.115.017905" data-apath="/ggg/5/7/1463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrew M. Glazer</span>, <span class="highwire-citation-author" data-delta="1">Emily E. Killingbeck</span>, <span class="highwire-citation-author" data-delta="2">Therese Mitros</span>, <span class="highwire-citation-author" data-delta="3">Daniel S. Rokhsar</span> and <span class="highwire-citation-author" data-delta="4">Craig T. Miller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1463-1472;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017905 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459974" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew M. Glazer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BM.%2BGlazer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glazer%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BM.%2BGlazer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emily E. Killingbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BE.%2BKillingbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Killingbeck%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BE.%2BKillingbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Therese Mitros </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Therese%2BMitros%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitros%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATherese%2BMitros%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel S. Rokhsar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BS.%2BRokhsar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokhsar%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BS.%2BRokhsar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Craig T. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BT.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BT.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ctmiller@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">ctmiller@berkeley.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/7/1463" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459974" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459974/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1463.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459974" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459974/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1463.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459974" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459974/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1463.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459974" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459974/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/7/1463.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459974" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459974/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67954 first last"><a href="/content/6/3/769" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - March 01, 2016</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Marine populations of the threespine stickleback (<em>Gasterosteus aculeatus</em>) have repeatedly colonized and rapidly adapted to freshwater habitats, providing a powerful system to map the genetic architecture of evolved traits. Here, we developed and applied a binned genotyping-by-sequencing (GBS) method to build dense genome-wide linkage maps of sticklebacks using two large marine by freshwater F2 crosses of more than 350 fish each. The resulting linkage maps significantly improve the genome assembly by anchoring 78 new scaffolds to chromosomes, reorienting 40 scaffolds, and rearranging scaffolds in 4 locations. In the revised genome assembly, 94.6% of the assembly was anchored to a chromosome. To assess linkage map quality, we mapped quantitative trait loci (QTL) controlling lateral plate number, which mapped as expected to a 200-kb genomic region containing <em>Ectodysplasin</em>, as well as a chromosome 7 QTL overlapping a previously identified modifier QTL. Finally, we mapped eight QTL controlling convergently evolved reductions in gill raker length in the two crosses, which revealed that this classic adaptive trait has a surprisingly modular and nonparallel genetic basis.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/quantitative-trait-locus" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus" rel="nofollow">quantitative trait locus</a></li><li class="kwd"><a href="/keyword/convergent-evolution" class="hw-term hw-article-keyword hw-article-keyword-convergent-evolution" rel="nofollow">convergent evolution</a></li><li class="kwd"><a href="/keyword/stickleback" class="hw-term hw-article-keyword hw-article-keyword-stickleback" rel="nofollow">stickleback</a></li></ul><p id="p-3">Understanding the genetic basis of adaptation remains a major unsolved goal in biology. For example, when the same phenotype evolves in independent lineages (convergent evolution), is the genetic basis predictable (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stern and Orgogozo 2009</a>)? Do evolved loci typically affect phenotypes in a global or in a modular, anatomically specific manner (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wagner <em>et al.</em> 2007</a>)? In systems where differently adapted natural populations are interfertile, quantitative trait locus (QTL) mapping provides an entry point to study the genetic architecture of evolved traits.</p><p id="p-4">The threespine stickleback (<em>Gasterosteus aculeatus</em>) has undergone widespread adaptive radiation in which marine fish independently colonized and adapted to countless freshwater habitats (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bell and Foster 1994</a>). Marine and freshwater populations typically differ in many skeletal phenotypes, including a reduction of the number of lateral plates, which are used for defense against predation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Reimchen 1992</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hagen 1973</a>) and a reduction in the length of gill rakers, which comprise a set of bones used for prey retention during feeding (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schluter 2000</a>). Lateral plate reduction in many freshwater populations is largely controlled by a large-effect QTL on chromosome 4 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berner <em>et al.</em> 2014</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cresko <em>et al.</em> 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wark <em>et al.</em> 2012</a>), which has been shown to be a regulatory allele of <em>Ectodysplasin</em> (<em>Eda</em>) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Colosimo <em>et al.</em> 2005</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">O’Brown <em>et al.</em> 2015</a>). Several smaller-effect modifier QTL also contribute to plate number reduction (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Wark <em>et al.</em> 2012</a>). Reductions in gill raker length and number are important trophic adaptations in freshwater sticklebacks and other postglacial fish, and have convergently evolved multiple times (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schluter 2000</a>). Typically, fish that eat small plankton evolve more and longer gill rakers and fish that eat larger prey evolve fewer and shorter gill rakers (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arnegard <em>et al.</em> 2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kahilainen <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schluter and McPhail 1992</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Theis <em>et al.</em> 2014</a>). Our previous genetic studies found gill raker number to be a highly polygenic trait, controlled by more than 15 QTL (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Glazer <em>et al.</em> 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2014</a>). Although stickleback gill raker length differences can arise due to phenotypic plasticity, there is a large heritable component (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Day <em>et al.</em> 1994</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hatfield 1997</a>). The genetic basis of evolved freshwater reductions in stickleback gill raker length is poorly understood, but two QTL were identified in a cross between European lake and stream populations (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Berner <em>et al.</em> 2014</a>).</p><p id="p-5">The stickleback genome has been sequenced and scaffolds were anchored in the stickleback reference assembly (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jones <em>et al.</em> 2012</a>) using a linkage map made from an F2 cross of 92 fish. The assembly consists of 113 anchored scaffolds on 21 chromosomes, as well as 1822 unanchored scaffolds (13.2% of the assembly). Subsequent work inverted the orientations of 13 anchored scaffolds and anchored 18 additional scaffolds (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Roesti <em>et al.</em> 2013</a>). Three large chromosomal inversions are typically present between marine and freshwater sticklebacks (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Jones <em>et al.</em> 2012</a>), but the extent of other differences between stickleback populations in genomic structure and genome-wide recombination patterns is largely unknown.</p><p id="p-6">Here, we used genotyping-by-sequencing (GBS) to sample approximately 100,000 SNPs to low coverage (approximately 1.5× per sample) for more than 350 sticklebacks in each of two marine × freshwater F2 crosses. We binned these low-coverage SNPs into more than 1000 high-coverage (approximately 150×) markers. Using these markers, we constructed high-density genome-wide linkage maps, which we used to anchor, reorient, and rearrange scaffolds and to examine genome-wide recombination patterns. We also used these maps to map the genetic basis of two ecologically important phenotypes. First, as a positive control, we mapped QTL controlling lateral plate number. Second, we mapped gill raker length QTL to test two hypotheses: that convergent evolution in two independently derived freshwater populations involves similar genetic architectures, and that gill raker length is genetically controlled by modular QTL affecting the lengths of subsets of gill rakers.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Stickleback crosses</h3><p id="p-7">Two marine × freshwater F1 crosses were previously described (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Glazer <em>et al.</em> 2014</a>). A wild-caught male marine fish from the Little Campbell River (British Columbia, Canada; LITC) was crossed to a wild-caught female freshwater fish from Fishtrap Creek (Washington state; FTC) to generate the FTC cross. A male freshwater fish from Bear Paw Lake (Alaska; BEPA; lab-reared offspring of wild-caught parents) was crossed to a wild-caught marine female LITC fish to produce the BEPA cross. F1s were intercrossed to generate 360 and 363 F2 fish from the FTC and BEPA crosses, respectively. Fish with low genotype coverage (n = 2 from each cross) were removed from the analysis. See Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/017905SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a> for additional information on raising the crosses.</p></div><div id="sec-3" class="subsection"><h3>Animal statement</h3><p id="p-8">Wild anadromous marine fish were collected from the Little Campbell River in British Columbia under a fish collection permit from the British Columbia Ministry of Environment (permit #SU08-44549). Wild freshwater fish were collected from Fishtrap Creek in Washington under a fish scientific collection permit from the Washington Department of Fish and Wildlife (permit #08-284). All animal work was approved by the Institutional Animal Care and Use Committees of the University of California-Berkeley or Stanford University (protocol number R330 and 13834).</p></div><div id="sec-4" class="subsection"><h3>Preparation of GBS libraries</h3><p id="p-9">DNA was isolated by phenol-chloroform extraction or with a DNeasy 96 Blood and Tissue Kit (Qiagen). Genomic DNA concentration was assessed with a NanoDrop 1000 spectrophotometer (Thermo Scientific) and by Quant-iT PicoGreen Assay (Invitrogen). Unless otherwise noted, GBS Illumina sequencing libraries were constructed as previously described (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>); 50 ng/sample of genomic DNA was used. Individuals were sequenced in seven libraries (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For libraries 1–3, 48 barcode plus common adapters were used (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>). Libraries 4–6 used 96 ApeKI Y-shaped adapters with internal barcodes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">ICGMC 2015</a>), and library 7 used these 96 Y-shaped adapters and 4 different PCR primers with different index barcodes (384 total samples) (adapted from <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Peterson <em>et al.</em> 2012</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS1.pdf">Figure S1</a>). For library 7, all volumes in the digestion and ligation reactions were successfully halved relative to the <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> (2011)</a> protocol to reduce reagent costs. ApeKI digestion, ligation, and sample clean-up were performed as described (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>). PCR amplification of sequencing libraries was performed in 50 ul reactions with 25 ul Taq 2× Master Mix (NEB), 50–450 ng of primer, and 2 ul of each library at 98° for 30 sec, 10–22 cycles at 98° for 10 sec, 65° for 30 sec, 72° for 30 sec, 5 min at 72°, and held at 4°. Primer concentration and cycle number were varied to amplify enough product for sequencing. PCR products were purified and size-selected with AMPure XP beads (Agencourt) with a bead:sample ratio of 0.7. Libraries were analyzed on an Agilent 2100 Bioanalyzer High-Sensitivity Chip for quality control and sequenced with 100 bp paired-end sequencing on an Illumina HiSeq 2000 sequencer.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466494/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466494/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of GBS libraries</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Processing reads from GBS libraries</h3><p id="p-14">The two grandparents of the FTC cross and the two grandparents of the BEPA cross were resequenced to approximately 60× and 6× coverage, respectively. In each cross, sites where one grandparent was homozygous for one allele and the other grandparent was homozygous for a second allele were identified (“homozygous SNP positions”; see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>). GBS reads from F2s were sorted by barcode with a custom Perl script. Reads were mapped to the stickleback reference genome with BWA using default settings (<a href="http://www.bio-bwa.sourceforge.net">www.bio-bwa.sourceforge.net</a>), allowing up to a 4% difference between reads and the reference genome. We devised a method to identify high-quality, segregating SNPs. For each homozygous SNP position, F2 GBS reads overlapping the SNP were considered, and the number of reads supporting marine and freshwater alleles for each homozygous SNP position was determined with SAMtools (<a href="http://www.samtools.sourceforge.net">www.samtools.sourceforge.net</a>). Genomic positions identified as not having a homozygous difference in the grandparent resequencing were not examined in the F2s. For each homozygous SNP position, a weighted average of these values was calculated across all F2s, normalized by the total number of mapped reads for each F2. We multiplied the marine and freshwater weighted averages by 10<sup>6</sup> to calculate reads per million mapped (RPMM). Properly segregating SNPs should have an approximately 1:1 ratio of marine:freshwater alleles, as was observed for most SNPs (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1B</a>). However, we observed some genomic regions that had a skewed allele ratio in the F2s, possibly due to meiotic drive and/or the lethality of particular genotypic classes. For example, a region of chromosome 2 centered at marker 0_32 in the FTC cross had a freshwater allele frequency of 0.61 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS1.pdf">File S1</a>). Therefore, a wider range of allele ratios was allowed for individual SNPs (a marine/freshwater RPMM ratio between 4:1 and 1:4). To include SNPs with true segregation bias, skewed markers adjacent to other similarly skewed markers were included, but skewed markers surrounded by nonskewed markers were removed (see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>). Additionally, SNPs were filtered for those with an average marine plus freshwater RPMM between 0.2 and 3.0 to have a set of SNPs with similar coverage levels. A separate set of sieving parameters was used to determine sex chromosome genotypes (see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping-by-sequencing (GBS) approach. (A) Flowchart of GBS. For each cross, the two grandparents were resequenced to determine homozygous SNP differences, which were filtered for high coverage levels and expected allele ratios in F2s (see Materials and Methods). (B) Sieve for high-coverage, segregating SNPs. For each SNP, the mean number of mapped reads supporting the marine and freshwater alleles, normalized for the number of millions of reads mapped per sample, is displayed. Data are shown for the FTC &#xD7; LITC cross. Sieve is shown with red quadrilateral: freshwater allele frequency between 0.2 and 0.8 and total coverage between 0.2 and 3. (C) Diagram of binning approach. Low-coverage sequencing generated read pileup at a large number of SNPs. For each F2, SNPs were binned by counting the total number of marine and freshwater reads within the bin and determining a genotype from the pooled counts. Sample 2 illustrates a case in which a recombination breakpoint is near the boundary between two bins and Bin 1 containing the breakpoint is considered to have the FF genotype. Alternatively, bins containing recombination breakpoints also frequently were called with uncertain MF/FF or MM/MF genotypes (Figure S2). (D) Calling sex from sex chromosome (chromosome 19) coverage. Females (XX) have approximately equal sex chromosome and autosome coverage levels, whereas males (XY) have approximately half the coverage level on the sex chromosome compared to the autosomes. Data are shown for the FTC cross. Inset shows zoom-in of low-coverage samples showing that female and male fish can still be distinguished." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2002686072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotyping-by-sequencing (GBS) approach. (A) Flowchart of GBS. For each cross, the two grandparents were resequenced to determine homozygous SNP differences, which were filtered for high coverage levels and expected allele ratios in F2s (see &lt;em&gt;Materials and Methods&lt;/em&gt;). (B) Sieve for high-coverage, segregating SNPs. For each SNP, the mean number of mapped reads supporting the marine and freshwater alleles, normalized for the number of millions of reads mapped per sample, is displayed. Data are shown for the FTC &#xD7; LITC cross. Sieve is shown with red quadrilateral: freshwater allele frequency between 0.2 and 0.8 and total coverage between 0.2 and 3. (C) Diagram of binning approach. Low-coverage sequencing generated read pileup at a large number of SNPs. For each F2, SNPs were binned by counting the total number of marine and freshwater reads within the bin and determining a genotype from the pooled counts. Sample 2 illustrates a case in which a recombination breakpoint is near the boundary between two bins and Bin 1 containing the breakpoint is considered to have the FF genotype. Alternatively, bins containing recombination breakpoints also frequently were called with uncertain MF/FF or MM/MF genotypes (Figure S2). (D) Calling sex from sex chromosome (chromosome 19) coverage. Females (XX) have approximately equal sex chromosome and autosome coverage levels, whereas males (XY) have approximately half the coverage level on the sex chromosome compared to the autosomes. Data are shown for the FTC cross. Inset shows zoom-in of low-coverage samples showing that female and male fish can still be distinguished.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1463/F1.medium.gif" width="440" height="375"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/7/1463/F1.medium.gif" width="440" height="375"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466376" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Genotyping-by-sequencing (GBS) approach. (A) Flowchart of GBS. For each cross, the two grandparents were resequenced to determine homozygous SNP differences, which were filtered for high coverage levels and expected allele ratios in F2s (see <em>Materials and Methods</em>). (B) Sieve for high-coverage, segregating SNPs. For each SNP, the mean number of mapped reads supporting the marine and freshwater alleles, normalized for the number of millions of reads mapped per sample, is displayed. Data are shown for the FTC × LITC cross. Sieve is shown with red quadrilateral: freshwater allele frequency between 0.2 and 0.8 and total coverage between 0.2 and 3. (C) Diagram of binning approach. Low-coverage sequencing generated read pileup at a large number of SNPs. For each F2, SNPs were binned by counting the total number of marine and freshwater reads within the bin and determining a genotype from the pooled counts. Sample 2 illustrates a case in which a recombination breakpoint is near the boundary between two bins and Bin 1 containing the breakpoint is considered to have the FF genotype. Alternatively, bins containing recombination breakpoints also frequently were called with uncertain MF/FF or MM/MF genotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS2.pdf">Figure S2</a>). (D) Calling sex from sex chromosome (chromosome 19) coverage. Females (XX) have approximately equal sex chromosome and autosome coverage levels, whereas males (XY) have approximately half the coverage level on the sex chromosome compared to the autosomes. Data are shown for the FTC cross. Inset shows zoom-in of low-coverage samples showing that female and male fish can still be distinguished.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">These filtered SNPs were further grouped into bins of at most 500 kb. Bin size was scaled to divide each scaffold into evenly sized bins. Scaffolds smaller than 100 kb were binned into one bin, scaffolds between 100 kb and 1 Mb were binned into two bins of equal size, scaffolds between 1 Mb and 1.5 Mb were divided into three bins, scaffolds between 1.5 and 2 Mb were divided into four bins, and so on. For each SNP within a bin, marine and freshwater read counts were summed and genotypes were called (see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>). Fish that had missing genotypes for more than 50% of markers were removed from the analysis (n = 2 in each cross). Nine additional samples with high rates of missing genotypes in libraries 1–6 were resequenced successfully in library 7. Markers that had missing data for at least 20% of fish were removed from the analysis (n = 59 and 39 in the FTC and BEPA crosses, respectively). Markers were also removed that had aberrant allelic ratios (n = 25 and 60 in the FTC and BEPA crosses, respectively; see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>). Genetic linkage maps were created with JoinMap 4.0 (Kyazma) using regression mapping with default settings. Further information on creating a consensus scaffold map, a second method for anchoring scaffolds, calling sex of the F2s, and fine mapping of recombinant breakpoints are presented in Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>.</p><p id="p-17">GBS and grandparental sequence reads are available in the Sequence Read Archive (accession number SRP057885). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS5.pdf">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS6.pdf">File S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS7.pdf">File S7</a> and a script to convert from original to revised genome coordinates are available from the Dryad Digital Repository (<a href="http://dx.doi.org/10.5061/dryad.q018v">http://dx.doi.org/10.5061/dryad.q018v</a>).</p></div><div id="sec-6" class="subsection"><h3>Phenotyping</h3><p id="p-18">Lateral plate number and gill raker length were measured from Alizarin-stained fish (see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>). The average of plate counts on the left and right sides was used for QTL mapping. To phenotype gill raker length, branchial skeletons were dissected out of fish and mounted flat on bridged coverslips as described (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2014</a>). Measurements were obtained by acquiring digital images of left side row 1 ventral gill rakers on a Leica M165 microscope and tracing a line segment from the gill raker base to tip in imageJ (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schneider <em>et al.</em> 2012</a>). Three gill raker lengths were measured on the first ceratobranchial: lateral (second gill raker from end near ventral/dorsal joint), middle (middle of ceratobranchial), and medial (second gill raker from end near midline).</p></div><div id="sec-7" class="subsection"><h3>QTL mapping</h3><p id="p-19">For QTL mapping, plate number and gill raker length phenotypes were tested for an association with standard length and sex by linear regression in R (<a href="http://www.r-project.org">www.r-project.org</a>) and corrected for size, sex, and/or log-transformed, when appropriate (see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>). QTL mapping was performed in R/qtl (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broman and Sen 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Broman <em>et al.</em> 2003</a>). Initial QTL mapping was performed with <em>scanone</em> with Haley-Knott regression. Trait-specific genome-wide significance thresholds with α of 0.05 were calculated with 1000 permutations. In cases where multiple significant QTL affected a phenotype, multiple QTL mapping was performed with <em>stepwiseqtl</em>, QTL peak markers, and LOD plots calculated with <em>refineqtl</em>, and peak LOD scores and percent variance explained values calculated with <em>fitqtl</em>, adjusting for the effect of other QTL underlying the phenotype when appropriate.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Two genome-wide linkage maps</h3><p id="p-20">To build genome-wide linkage maps in two marine × freshwater stickleback F2 crosses, we used a binned GBS approach (modified from <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>; outlined in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Sticklebacks from two independently derived freshwater populations [Fishtrap Creek (FTC) and Bear Paw Lake (BEPA)] were crossed to fish from a single marine population (LITC). These two F2 crosses are hereafter called the FTC and BEPA crosses. F2 fish (n = 358 and 361 in the FTC and BEPA crosses, respectively) were sequenced with GBS, multiplexing up to 384 samples in a single Illumina lane (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/TableS1.xlsx">Table S2</a>). SNPs were phased using grandparent resequencing and filtered for those that had proper allele ratios and coverage levels, resulting in 131,091 and 87,419 high-quality segregating SNPs in the FTC and BEPA crosses (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, A and B</a>). To generate high-quality genotypes, we binned together multiple low-coverage SNPs into high-coverage binned markers (referred to as markers in this study) using bins of equal size of, at most, 500 kb (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Linkage maps were made with 1001 and 978 markers in the FTC and BEPA crosses (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS2.pdf">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS3.pdf">File S3</a>). These maps had missing genotype rates of 1.9% (FTC) and 1.8% (BEPA). The sex of each F2 fish was determined from sequencing coverage levels of the sex chromosome (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>).</p></div><div id="sec-10" class="subsection"><h3>Improvements to stickleback genome assembly</h3><p id="p-21">The stickleback reference genome assembly (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Jones <em>et al.</em> 2012</a>) contains 113 anchored and 1822 unanchored scaffolds, which comprise 86.8% and 13.2% of the genome assembly, respectively. In both linkage maps, all previously anchored scaffolds mapped to their originally assigned chromosome. In addition, in all cases where a scaffold mapped to a chromosome in both crosses, the scaffold mapped to the same chromosome. Combining the two linkage maps, we generated a consensus scaffold map containing 186 scaffolds, which differed from the genome assembly in 153 places. These differences consisted of 78 previously unanchored scaffolds that were newly anchored in the genome (comprising 36.1 Mb), 40 inversions of previously anchored scaffolds (113.3 Mb), and 4 rearrangements of previously anchored scaffolds (12.8 Mb) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS4.pdf">File S4</a>). In the consensus scaffold map, the 186 total scaffolds comprised 94.6% of the total assembly sequence and included the largest 124 scaffolds. Based on the linkage map positions of markers within each scaffold, an orientation was determined for 166 of the 186 scaffolds (436.2 Mb). The anchored scaffolds in the consensus scaffold map contained 9% more bases and Ensembl-predicted genes (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Jones <em>et al.</em> 2012</a>) than in the original assembly (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The revised genome assembly sequence and adjusted positions of Ensembl-predicted gene locations are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS5.pdf">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS6.pdf">File S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS7.pdf">File S7</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A revised map of stickleback scaffold order and orientation. Consensus scaffold map from the two crosses. Chromosomes are numbered on the left, and scaffolds are numbered, with previously unanchored scaffolds colored red, previously anchored scaffolds whose orientation has been flipped colored gray, and scaffolds that have switched positions colored blue. See File S4 for coordinates of scaffold locations. Figure style adapted from Roesti et al. 2013." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2002686072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A revised map of stickleback scaffold order and orientation. Consensus scaffold map from the two crosses. Chromosomes are numbered on the left, and scaffolds are numbered, with previously unanchored scaffolds colored red, previously anchored scaffolds whose orientation has been flipped colored gray, and scaffolds that have switched positions colored blue. See File S4 for coordinates of scaffold locations. Figure style adapted from Roesti &lt;em&gt;et al.&lt;/em&gt; 2013.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1463/F2.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/7/1463/F2.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466357" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">A revised map of stickleback scaffold order and orientation. Consensus scaffold map from the two crosses. Chromosomes are numbered on the left, and scaffolds are numbered, with previously unanchored scaffolds colored red, previously anchored scaffolds whose orientation has been flipped colored gray, and scaffolds that have switched positions colored blue. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS4.pdf">File S4</a> for coordinates of scaffold locations. Figure style adapted from <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Roesti <em>et al.</em> 2013</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466476/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466476/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of improved genome assembly</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">We developed a second, more sensitive method to map unanchored scaffolds by examining the correlation of read counts for every pair of markers (see Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FileS8.pdf">File S8</a>). With this read correlation method, 538 scaffolds (96.9% of total assembly sequence), including 352 scaffolds not mapped by the first method, mapped to within approximately 5 cM of a marker in the consensus scaffold map (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/TableS1.xlsx">Table S3</a>). Compared to the linkage map-based assembly, an additional 10.8 Mb and 490 Ensembl-predicted genes were linked to a chromosome.</p><p id="p-25">To determine whether large-scale genomic rearrangements or patterns of recombination rates differ between freshwater populations, we examined the genome-wide patterns of recombination. The two crosses did not indicate any large-scale differences in genomic structure and had strikingly similar patterns of recombination across the genome, with similar regions of high and low recombination (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The BEPA cross had an elevated overall recombination rate relative to the FTC cross, with a total map size of 1570 cM and 1963 cM in the FTC and BEPA crosses, respectively. This difference was due to an elevated rate of recombination throughout the genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS5.pdf">Figure S5</a>). Consistent with a previous study (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Roesti <em>et al.</em> 2013</a>), most chromosomes appeared to have suppressed recombination in the middle, with ends of chromosomes having higher rates of recombination (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). The pattern of recombination within each chromosome correlated partially with previously described chromosome morphologies (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Urton <em>et al.</em> 2011</a>). For example, as predicted, recombination rates were high on both ends of metacentric chromosome 7 and recombination occurred mostly on one end of telocentric chromosome 15. However, some chromosomes did not match predictions (<em>e.g.</em>, recombination occurred mostly on one end of metacentric chromosomes 14 and 21). As expected, in both crosses recombination was completely suppressed in three previously described (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Jones <em>et al.</em> 2012</a>) marine/freshwater inversions on chromosomes 1, 11, and 21 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS6.pdf">Figure S6</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Similar genome-wide recombination patterns in both crosses. Plots of genetic vs. physical position for each chromosome. Plots are scaled to have constant width and height for each chromosome. Markers from the FTC and BEPA crosses are plotted in blue and red, respectively. Physical position is according to the revised scaffold map, and tick marks along the x-axis indicate scaffold boundaries. Most chromosomes have highly similar regions of high and low recombination rates between the two crosses. The positions of the three marine/freshwater inversions on chromosomes 1, 11, and 21 reported in Jones et al. (2012) are indicated with gray rectangles. For a closer zoom-in of these inversions see Figure S6, which shows that no recombination events were detected within the three inversions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2002686072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Similar genome-wide recombination patterns in both crosses. Plots of genetic &lt;em&gt;vs.&lt;/em&gt; physical position for each chromosome. Plots are scaled to have constant width and height for each chromosome. Markers from the FTC and BEPA crosses are plotted in blue and red, respectively. Physical position is according to the revised scaffold map, and tick marks along the x-axis indicate scaffold boundaries. Most chromosomes have highly similar regions of high and low recombination rates between the two crosses. The positions of the three marine/freshwater inversions on chromosomes 1, 11, and 21 reported in Jones &lt;em&gt;et al.&lt;/em&gt; (2012) are indicated with gray rectangles. For a closer zoom-in of these inversions see Figure S6, which shows that no recombination events were detected within the three inversions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1463/F3.medium.gif" width="376" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/7/1463/F3.medium.gif" width="376" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466426" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Similar genome-wide recombination patterns in both crosses. Plots of genetic <em>vs.</em> physical position for each chromosome. Plots are scaled to have constant width and height for each chromosome. Markers from the FTC and BEPA crosses are plotted in blue and red, respectively. Physical position is according to the revised scaffold map, and tick marks along the x-axis indicate scaffold boundaries. Most chromosomes have highly similar regions of high and low recombination rates between the two crosses. The positions of the three marine/freshwater inversions on chromosomes 1, 11, and 21 reported in <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Jones <em>et al.</em> (2012)</a> are indicated with gray rectangles. For a closer zoom-in of these inversions see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS6.pdf">Figure S6</a>, which shows that no recombination events were detected within the three inversions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>QTL mapping of lateral plate reduction</h3><p id="p-27">Like most freshwater populations, freshwater FTC and BEPA fish have evolved reduced lateral plates (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cresko <em>et al.</em> 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hagen and Gilbertson 1972</a>). Plate reduction is typically controlled by a large-effect QTL on chromosome 4 in both Pacific Northwest and Alaskan freshwater populations, including BEPA (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cresko <em>et al.</em> 2004</a>). This QTL has been shown to be a regulatory haplotype of the <em>Ectodysplasin</em> (<em>Eda</em>) gene (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Colosimo <em>et al.</em> 2005</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">O’Brown <em>et al.</em> 2015</a>). Lab-reared FTC and BEPA fish were both low-plated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS7.pdf">Figure S7A</a>). As a positive control to validate the GBS linkage maps, we mapped lateral plate number QTL in both crosses. As expected, in both crosses a near-Mendelian QTL on chromosome 4 (percent variance explained of 97.8% and 95.7% in the FTC and BEPA crosses, respectively) controlled lateral plate number (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>; <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The chromosome 4 QTL was largely recessive in each cross; all marine homozygotes and heterozygotes had more than 15 plates and all freshwater homozygotes had fewer than 15 plates (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, C and D</a>). In contrast with <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Berner <em>et al.</em> (2014)</a>, we did not find a double QTL peak on chromosome 4 in either cross (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, A and B</a>), consistent with a single underlying genetic locus.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lateral plate reduction is controlled by a near-Mendelian locus containing Eda. QTL mapping of plate number in the FTC (A) and BEPA (B) crosses. A main large-effect QTL was found in both crosses on chromosome 4 with a single peak at Ectodysplasin (Eda). cM, centimorgan. Boxplots showing association between Eda genotype and lateral plate number in the FTC (C) and BEPA (D) crosses. M, marine; F, freshwater. The QTL is recessive, with all MF and MM fish having more than 15 plates and all FF fish having fewer than 15 plates. (E) Fine-mapping the chromosome 4 lateral plate QTL with MF/FF recombinants in both crosses. Genotype at a 199.8-kb interval perfectly correlates with whether plate number is low (15 plates). This interval contains the coding regions of 17 Ensembl-predicted genes, including Eda and recently identified intergenic regulatory mutations of Eda (O&#x2019;Brown et al. 2015). ENS24256 and ENS24272 refer to ENSGACT00000024256 and ENSGACT00000024272, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2002686072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Lateral plate reduction is controlled by a near-Mendelian locus containing &lt;em&gt;Eda&lt;/em&gt;. QTL mapping of plate number in the FTC (A) and BEPA (B) crosses. A main large-effect QTL was found in both crosses on chromosome 4 with a single peak at &lt;em&gt;Ectodysplasin (Eda)&lt;/em&gt;. cM, centimorgan. Boxplots showing association between &lt;em&gt;Eda&lt;/em&gt; genotype and lateral plate number in the FTC (C) and BEPA (D) crosses. M, marine; F, freshwater. The QTL is recessive, with all MF and MM fish having more than 15 plates and all FF fish having fewer than 15 plates. (E) Fine-mapping the chromosome 4 lateral plate QTL with MF/FF recombinants in both crosses. Genotype at a 199.8-kb interval perfectly correlates with whether plate number is low (&lt;15 plates per side) or high (&gt;15 plates). This interval contains the coding regions of 17 Ensembl-predicted genes, including &lt;em&gt;Eda&lt;/em&gt; and recently identified intergenic regulatory mutations of &lt;em&gt;Eda&lt;/em&gt; (O&#x2019;Brown &lt;em&gt;et al.&lt;/em&gt; 2015). ENS24256 and ENS24272 refer to ENSGACT00000024256 and ENSGACT00000024272, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1463/F4.medium.gif" width="300" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/7/1463/F4.medium.gif" width="300" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466470" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Lateral plate reduction is controlled by a near-Mendelian locus containing <em>Eda</em>. QTL mapping of plate number in the FTC (A) and BEPA (B) crosses. A main large-effect QTL was found in both crosses on chromosome 4 with a single peak at <em>Ectodysplasin (Eda)</em>. cM, centimorgan. Boxplots showing association between <em>Eda</em> genotype and lateral plate number in the FTC (C) and BEPA (D) crosses. M, marine; F, freshwater. The QTL is recessive, with all MF and MM fish having more than 15 plates and all FF fish having fewer than 15 plates. (E) Fine-mapping the chromosome 4 lateral plate QTL with MF/FF recombinants in both crosses. Genotype at a 199.8-kb interval perfectly correlates with whether plate number is low (&lt;15 plates per side) or high (&gt;15 plates). This interval contains the coding regions of 17 Ensembl-predicted genes, including <em>Eda</em> and recently identified intergenic regulatory mutations of <em>Eda</em> (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">O’Brown <em>et al.</em> 2015</a>). ENS24256 and ENS24272 refer to ENSGACT00000024256 and ENSGACT00000024272, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466362/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466362/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">QTL identified in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">To test the resolution of the linkage maps, we used a Hidden Markov Model on the raw allele counts for each SNP to fine-map recombination breakpoints (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS6.pdf">Figure S6A</a>). This method enabled us to fine-map recombinant breakpoints to a median resolution of 89 kb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS6.pdf">Figure S6B</a>). With the fine-mapped recombination breakpoints, we identified 10 heterozygous/homozygous freshwater recombinant animals in the two crosses that recombined within a 1-Mb interval surrounding <em>Eda</em>. These recombinant animals defined a 199.8-kb interval that perfectly correlates with plate number in both crosses (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, C–E</a>). This interval contains 17 Ensembl-predicted genes, including <em>Eda</em>, as well as an intergenic SNP recently shown to affect a lateral plate enhancer (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">O’Brown <em>et al.</em> 2015</a>). No additional QTL were detected upon conditioning on <em>Eda</em> genotype in a single model (data not shown). Because <em>Eda</em> heterozygotes had the most variance of any genotypic class (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4, C and D</a>), we mapped plate number in <em>Eda</em> heterozygotes (as in <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>) and detected one modifier QTL on chromosome 7 in the FTC cross but no significant modifier QTL in the BEPA cross (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS7.pdf">Figure S7B</a>, <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-12" class="subsection"><h3>QTL mapping of gill raker length</h3><p id="p-31">We previously discovered a strikingly high degree of modularity of skeletal evolution in sticklebacks, consistently observed across a variety of axial and craniofacial skeletal traits (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2014</a>). To test the hypothesis that gill raker length is also under modular genetic control, we examined gill rakers at three positions, located at lateral, middle, and medial points of the anterior-most ceratobranchial bone (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). We observed a modular reduction of gill raker length in lab-reared FTC and BEPA fish, with strongest length reductions in the lateral and middle domains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1/FigureS8.pdf">Figure S8</a>). QTL controlling gill raker length were detected on chromosomes 1, 4, 10, 11, and 20 in the FTC cross and chromosomes 16, 19, and 20 in the BEPA cross (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5, B and C</a>; <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). The peak marker on chromosome 4 in the FTC cross was 17_8, a bin containing <em>Eda</em>. While one QTL (chromosome 16 in the BEPA cross) had effects on lateral, middle, and medial gill raker lengths, most QTL were surprisingly modular, with significant effects on only one gill raker length (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Most (six of eight) of the gill raker length QTL were concordant with the direction of evolutionary change (freshwater allele yielding shorter gill rakers), consistent with gill raker length being under strong natural selection. QTL on chromosome 20 controlling lateral gill raker length were detected in both crosses, but had nonoverlapping 1.5 LOD intervals. Overall, no QTL with overlapping 1.5 LOD intervals were observed in both crosses. Thus, unlike lateral plate reduction, the convergent evolution of gill raker length reduction has occurred via distinct genetic bases in these two freshwater populations.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic mapping of gill raker length reduction. (A) Lengths of three ventral row 1 gill rakers were measured at lateral, middle, and medial positions. cb1, ceratobranchial 1. Scale bar = 500 um. Manhattan plots of QTL mapping of gill raker length in the FTC cross (B) and the BEPA cross (C). Three gill raker lengths were mapped: lateral (blue), middle (black), and medial (red). LOD is shown as a function of genetic position. The significance thresholds (&#x3B1; = 0.05) are shown with a dotted line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2002686072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic mapping of gill raker length reduction. (A) Lengths of three ventral row 1 gill rakers were measured at lateral, middle, and medial positions. cb1, ceratobranchial 1. Scale bar = 500 um. Manhattan plots of QTL mapping of gill raker length in the FTC cross (B) and the BEPA cross (C). Three gill raker lengths were mapped: lateral (blue), middle (black), and medial (red). LOD is shown as a function of genetic position. The significance thresholds (&#x3B1; = 0.05) are shown with a dotted line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1463/F5.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/7/1463/F5.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1463/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466485" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Genetic mapping of gill raker length reduction. (A) Lengths of three ventral row 1 gill rakers were measured at lateral, middle, and medial positions. cb1, ceratobranchial 1. Scale bar = 500 um. Manhattan plots of QTL mapping of gill raker length in the FTC cross (B) and the BEPA cross (C). Three gill raker lengths were mapped: lateral (blue), middle (black), and medial (red). LOD is shown as a function of genetic position. The significance thresholds (α = 0.05) are shown with a dotted line.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>High-quality linkage maps from binned GBS</h3><p id="p-33">The power of next-generation sequencing has revolutionized high-throughput genotyping, beginning in 2008 with the RAD-seq approach (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>). RAD-seq was rapidly applied to a variety of model and nonmodel organisms (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rowe <em>et al.</em> 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Narum <em>et al.</em> 2013</a>). An extension of RAD-seq, the simpler and cheaper GBS method was published in 2011 (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>). In fish, RAD-seq or GBS has been used to build genome-wide linkage maps in stickleback (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Roesti <em>et al.</em> 2013</a>; this study), salmon (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gonen <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Limborg <em>et al.</em> 2014</a>), and Mexican tetra (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carlson <em>et al.</em> 2015</a>), in each case successfully building a map with approximately a marker per centimorgan. Differences in these linkage maps appear largely attributable to the details of the cross design (<em>e.g.</em>, number of F2 fish genotyped), sequencing depth (<em>e.g.</em>, number of lanes sequenced and whether single or paired end reads were sequenced), and/or genome assembly used to align reads (<em>e.g.</em>, size of genome). For example, our threespine stickleback maps presented here are larger in total genetic distance than those in the work by <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Roesti <em>et al.</em> (2013)</a>; however, we analyzed more than twice as many F2 fish, sequenced paired end reads <em>vs.</em> single end reads, and generated more total sequences than in this previous study. In contrast, our stickleback maps are smaller than the total genetic length of a recently published Mexican tetra map, likely in part due to Mexican tetra having twice as large a genome as stickleback. One methodological difference between our maps and these other fish linkage maps is that we used a binned approach, binning SNPs to generate genetic markers, similar to that successfully used in corn (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2015</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2014</a>).</p><p id="p-34">This study utilized a binned GBS approach to generate high-quality genotypes. First, a large number of SNPs (approximately 100,000 per cross) were sequenced to a low level of sequencing coverage (approximately 1.5× per sample). Then, multiple SNPs were binned together to form approximately 1000 high-coverage (approximately 150×) markers. This approach contrasts with other reduced representation approaches (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Peterson <em>et al.</em> 2012</a>; but see <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andolfatto <em>et al.</em> 2011</a>) that target a smaller number of SNPs (typically 1000–5000) at high coverage (&gt;20×). The tradeoff of coverage <em>vs.</em> SNP number can easily be controlled through the choice of restriction enzymes as well as the degree of size selection of the library (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Peterson <em>et al.</em> 2012</a>). This study used ApeKI, which cuts a 5-bp restriction site that occurs frequently in the genome, and no library size selection to target a large number of SNPs. Individual SNPs can be biased toward one allele, have mapping or genotyping errors, and have variable coverage levels. Thus, binning a large number of low-coverage SNPs together resulted in a robust and reliable set of genotypes. The binning approach in this study also enabled the use of the same bins of markers for direct comparison between crosses. Several quality-control steps appeared to be crucial to generate high-quality linkage maps, including dropping SNPs and markers that deviated from expected allele ratios, dropping low-coverage samples and markers, and using a separate computational pipeline to generate sex chromosome genotypes.</p><p id="p-35">It is unlikely that the order and orientation of scaffolds presented in this study are completely correct or universal to all sticklebacks. However, there are several reasons to believe that most of the revised scaffold orders and orientations in this study are correct and typical stickleback features, rather than individual polymorphisms for genomic rearrangements. First, the majority of the scaffold orders and orientations were supported by multiple markers from both crosses, which derived from freshwater fish from two independently derived populations, including one fish from the same population as the original reference genome (BEPA). Second, the 31 changes to the stickleback genome assembly identified in a cross of (geographically distant) European sticklebacks by <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Roesti <em>et al.</em> 2013</a> were all detected by this study. Third, two of the scaffold changes identified in this study (on chromosomes 19 and 20) are consistent with previous cytogenetic evidence (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Urton <em>et al.</em> 2011</a>). The expanded set of assembly changes identified in this study (78 newly anchored scaffolds, 40 reoriented scaffolds, and 4 scaffold rearrangements) should further aid efforts to understand the evolutionary dynamics of the stickleback genome and discover the genes underlying adaptive phenotypes in sticklebacks.</p><p id="p-36">We observed little difference in the two maps whose SNPs were phased by sequencing the cross grandparents to high (60×, FTC) <em>vs.</em> low (6×, BEPA) coverage. We suspect that sequencing grandparents with GBS, as opposed to full genome sequencing, could further reduce costs. Several cost-saving factors, including halving reagent volumes during library creation and barcoding 384 samples together, did not result in a significant decrease in genotyping quality. In library 7, we used 96 barcoded adapters and 4 index primers to multiplex 384 samples in a single lane of Illumina sequencing. The high average marker coverage (approximately 50×) and high genotyping success rate with 384 barcoded samples (&lt;3% genotyping fail rate, see <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) suggest that more samples could be multiplexed together. For instance, 48 barcoded adapters and 16 index primers could allow barcoding of 768 samples while requiring fewer unique primers.</p></div><div id="sec-15" class="subsection"><h3>Genetic mapping of lateral plate reduction</h3><p id="p-37">QTL mapping of armor and trophic traits demonstrated the power of dense GBS-generated genome-wide linkage maps to detect QTL of large and small effect, as well as tested several hypotheses about the genetic basis of these adaptive skeletal changes. As expected from previous genetic studies of lateral plate number (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Cresko <em>et al.</em> 2004</a>), a large-effect QTL on chromosome 4 controlled plate number in both crosses. In contrast to a previous study that reported multiple QTL peaks on chromosome 4 for lateral plate number, perhaps suggesting multiple underlying chromosome 4 loci (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Berner <em>et al.</em> 2014</a>), in this study the chromosome 4 QTL in each cross had a clear single peak at <em>Eda</em>. A 17-gene minimal genomic interval included <em>Eda</em> and a recently identified intergenic lateral plate enhancer with a polymorphic SNP that affects enhancer activity (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">O’Brown <em>et al.</em> 2015</a>). Previous mapping of plate modifier QTL in <em>Eda</em> heterozygotes identified three plate number modifier QTL in a similarly sized cross with the Paxton benthic (PAXB) population (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>). With an identical mapping approach, we detected fewer modifier QTL (1 and 0 in the FTC and BEPA crosses, respectively). This difference in genetic architecture might be due to differences in the extent of plate reduction in the freshwater populations used in the studies [mean of 0.3 plates in PAXB (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McPhail 1992</a>) <em>vs.</em> 4.9 in FTC and 4.4 in BEPA]. Intriguingly, the chromosome 7 modifier QTL detected in the FTC cross overlaps one of the three previously detected plate modifier QTL (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>). This chromosome 7 modifier QTL might be reused along with <em>Eda</em> in multiple freshwater populations to reduce plate number.</p></div><div id="sec-16" class="subsection"><h3>Genetic mapping of gill raker length reduction</h3><p id="p-38">We also identified eight new QTL controlling the classic adaptive trait of gill raker length (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Schluter 2000</a>). Motivated by our previous finding of pervasive modularity in the evolution of serially homologous axial and craniofacial skeletal elements (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2014</a>), we hypothesized that gill raker length might also be genetically controlled in a modular fashion. Gill raker lengths measured at different mediolateral locations had surprisingly different genetic architectures, indicating complex modularity of this trait. Therefore, gill raker lengths at different positions might not be directly comparable in ecological studies. This modularity might reflect differences in retaining different types of prey with gill rakers of different lengths along the mediolateral axis. Developmental timing might contribute to this genetic modularity, as gill rakers form during embryonic development in a wave from lateral to medial (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Glazer <em>et al.</em> 2014</a>). Intriguingly, the chromosome 4 gill raker length QTL in the FTC cross has a peak marker bin that contains <em>Eda</em>. The EDA pathway, in addition to its role in plate development, is intimately involved in gill raker development in zebrafish and sticklebacks (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Glazer <em>et al.</em> 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Harris <em>et al.</em> 2008</a>). However, in the BEPA cross, a chromosome 4 plate number QTL, but not a gill raker length QTL, was detected. Therefore, if <em>Eda</em> is contributing to the FTC gill raker length QTL, there is likely different linked regulatory variation of <em>Eda</em> in FTC compared to BEPA.</p><p id="p-39">We previously identified an enrichment of skeletal QTL on chromosomes 4, 20, and 21 (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2014</a>), and our findings here add gill raker length as yet another skeletal trait controlled by two of these three trait clusters, as gill raker length mapped to chromosome 20 in both crosses, and chromosome 4 in the FTC cross. Linked chromosome 4 and 20 alleles promoting reduction of gill raker length (this study) and gill raker number (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Glazer <em>et al.</em> 2014</a>; <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2014</a>) might promote co-evolution of these phenotypes in freshwater environments. In sticklebacks, a predictable, shared genetic basis has been found to underlie the convergent evolution of several evolved phenotypes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chan <em>et al.</em> 2010</a>; <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Colosimo <em>et al.</em> 2004</a>, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">2005</a>; <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Glazer <em>et al.</em> 2014</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Miller <em>et al.</em> 2007</a>). In contrast to these studies, we detected no overlapping gill raker length QTL in the two crosses. In addition, none of the gill raker length QTL in this study overlap two previously reported QTL from a European lake × stream cross (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Berner <em>et al.</em> 2014</a>). Thus, unlike several other stickleback phenotypes, different loci appear to underlie the convergent evolution of gill raker length in different populations.</p></div></div><div class="section conclusions" id="sec-17"><h2 class="">Conclusions</h2><p id="p-40">This work used a binned GBS approach to build dense linkage maps of sticklebacks, which were used for genome assembly improvement and QTL mapping of two ecologically important traits. The revised genome assembly provides a more accurate understanding of the structure of the stickleback genome, which should aid efforts to map genes controlling stickleback phenotypes and understand genomic dynamics during stickleback evolution. The genetic mapping of distinct QTL controlling gill raker length in two crosses illustrates that, in contrast to several prominent cases in sticklebacks (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chan <em>et al.</em> 2010</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Colosimo <em>et al.</em> 2005</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Miller <em>et al.</em> 2007</a>), a nonparallel genetic basis is sometimes used in cases of repeated phenotypic evolution.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We thank David Kingsley for his input and support during the beginning of this study, Mike Bell for Bear Paw Lake fish, Alex Pollen for assistance collecting Fishtrap Creek and Little Campbell River fish, the Semiahmoo First Nation for permission to collect sticklebacks from the Little Campbell River, Patrick Lee and Anthony Lee for expert assistance in crossing and raising fish, Angela Lam, Priscilla Erickson, and Nihar Patel for DNA isolation and phenotyping assistance, Blair Gainous and Allison Quan for performing pilot experiments, and Jessica Lyons, Megan Hall, and Chris Martin for helpful discussions. This work was supported in part by the NIH (R01-DE021475 to C.T.M. and a Predoctoral Training Grant 5T32GM007127 to A.M.G.) and the NSF Graduate Research Fellowship Program (A.M.G.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-7"><p id="p-42">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017905/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-43"><em>Communicating editor: W. S. Davidson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-11"><span class="received-label">Received </span>March 11, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-05-06"><span class="accepted-label">Accepted </span>May 6, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Glazer <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.1" data-doi="10.1101/gr.115402.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erezyilmaz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mast</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115402.110%26rft_id%253Dinfo%253Apmid%252F21233398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.2" data-doi="10.1038/nature13301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arnegard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGee</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchinko</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of ecological divergence during speciation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">511</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D511%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13301%26rft_id%253Dinfo%253Apmid%252F24909991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24909991&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.3" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1463.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source">The Evolutionary Biology of the Threespine Stickleback</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.5" data-doi="10.1111/evo.12390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzburger</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic architecture of skeletal evolution in European lake and stream stickleback.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D1792%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12390%26rft_id%253Dinfo%253Apmid%252F24571250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571250&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1463.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">A Guide to QTL Mapping with R/qtl</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dordrecht</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.7" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xNDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">B.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onusko</span>  <span class="cit-name-given-names">S.W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J.B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> A high-density linkage map for Astyanax mexicanus using genotyping-by-sequencing technology. G3 (Bethesda) 5: 241–251.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.9" data-doi="10.1126/science.1182213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villarreal</span>  <span class="cit-name-given-names">G.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptive evolution of pelvic reduction in sticklebacks by recurrent deletion of a <em>Pitx1</em> enhancer.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1182213%26rft_id%253Dinfo%253Apmid%252F20007865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2My8zMDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.10" data-doi="10.1186/1471-2164-15-433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An ultra-high density bin-map for rapid QTL mapping for tassel and ear architecture in a large F-2 maize population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-433%26rft_id%253Dinfo%253Apmid%252F24898122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24898122&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colosimo</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peichel</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nereng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The genetic architecture of parallel armor plate reduction in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D2%26rft.spage%253D635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000221599500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.12" data-doi="10.1126/science.1107239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colosimo</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosemann</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balabhadra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villarreal</span>  <span class="cit-name-given-names">G.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Widespread parallel evolution in sticklebacks by repeated fixation of Ectodysplasin alleles.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span>: <span class="cit-fpage">1928</span>–<span class="cit-lpage">1933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColosimo%26rft.auinit1%253DP.%2BF.%26rft.volume%253D307%26rft.issue%253D5717%26rft.spage%253D1928%26rft.epage%253D1933%26rft.atitle%253DWidespread%2BParallel%2BEvolution%2Bin%2BSticklebacks%2Bby%2BRepeated%2BFixation%2Bof%2BEctodysplasin%2BAlleles%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1107239%26rft_id%253Dinfo%253Apmid%252F15790847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDcvNTcxNy8xOTI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.13" data-doi="10.1073/pnas.0308479101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Parallel genetic basis for repeated evolution of armor loss in Alaskan threespine stickleback populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">6050</span>–<span class="cit-lpage">6055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0308479101%26rft_id%253Dinfo%253Apmid%252F15069186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE2LzYwNTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.14" data-doi="10.2307/2410260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A comparison of two sticklebacks.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1723</span>–<span class="cit-lpage">1734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D48%26rft.spage%253D1723%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994QQ65900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.15" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.16" data-doi="10.1186/2041-9139-5-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glazer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleves</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parallel developmental genetic features underlie stickleback gill raker evolution.</span> <abbr class="cit-jnl-abbrev">Evodevo</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvodevo%26rft.volume%253D5%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2041-9139-5-19%26rft_id%253Dinfo%253Apmid%252F24851181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2041-9139-5-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24851181&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.17" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.18" data-doi="10.1038/hdy.1973.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hagen</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Inheritance of numbers of lateral plates and gill rakers in <em>Gasterosteus aculeatus</em>.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D30%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1973.40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1973.40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.19" data-doi="10.2307/2406981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hagen</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbertson</span>  <span class="cit-name-given-names">L. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Geographic variation and environmental selection in <em>Gasterosteus aculeatus</em> L. in the Pacific Northwest, America.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D26%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2406981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2406981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M135400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.20" data-doi="10.1371/journal.pgen.1000206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perathoner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konstantinidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Zebrafish <em>eda</em> and <em>edar</em> mutants reveal conserved and ancestral roles of ectodysplasin signaling in vertebrates.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHarris%26rft.auinit1%253DM.%2BP.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000206%26rft.epage%253De1000206%26rft.atitle%253DZebrafish%2Beda%2Band%2Bedar%2Bmutants%2Breveal%2Bconserved%2Band%2Bancestral%2Broles%2Bof%2Bectodysplasin%2Bsignaling%2Bin%2Bvertebrates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000206%26rft_id%253Dinfo%253Apmid%252F18833299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18833299&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.21" data-doi="10.1086/286036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic divergence in adaptive characters between sympatric species of stickleback.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D149%26rft.spage%253D1009%26rft_id%253Dinfo%253Adoi%252F10.1086%252F286036%26rft_id%253Dinfo%253Apmid%252F18811261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/286036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18811261&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WZ01800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1463.22" data-doi="10.1038/nature10944"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">International Cassava Genetic Map Consortium (ICGMC)</span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-Resolution Linkage Map and Chromosome-Scale Genome Assembly for Cassava (<em>Manihot esculenta Crantz</em>) from 10 Populations.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">5</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.23" data-doi="10.1007/s10682-010-9411-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauceli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D25%26rft.spage%253D573%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10682-010-9411-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10682-010-9411-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.24" data-doi="10.1007/s10682-010-9411-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kahilainen</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siwertsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjelland</span>  <span class="cit-name-given-names">K. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohn</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The role of gill raker number variability in adaptive radiation of coregonid fish.</span> <abbr class="cit-jnl-abbrev">Evol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D573%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10682-010-9411-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10682-010-9411-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic control of the leaf angle and leaf orientation value as revealed by ultra-high density maps in three connected maize populations.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">10</span>: e0121624.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.26" data-doi="10.1093/jhered/esu060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Temporally isolated lineages of pink salmon reveal unique signatures of selection on distinct pools of standing genetic variation.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesu060%26rft_id%253Dinfo%253Apmid%252F25149250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjEwOiIxMDUvUzEvNzQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cano</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> Identification of major and minor QTL for ecologically important morphological traits in three-spined sticklebacks (<em>Gasterosteus aculeatus</em>). G3 (Bethesda) 4: 595–604.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.28" data-doi="10.1016/j.cell.2007.10.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPhail</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Ecology and evolution of sympatric sticklebacks (<em>Gasterosteus</em>)—evidence for a species-pair in Paxton Lake, Texada Island, British Columbia.</span> <abbr class="cit-jnl-abbrev">Can. J. Zool.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMiller%26rft.auinit1%253DC.%2BT.%26rft.volume%253D131%26rft.issue%253D6%26rft.spage%253D1179%26rft.epage%253D1189%26rft.atitle%253Dcis-Regulatory%2Bchanges%2Bin%2BKit%2Bligand%2Bexpression%2Band%2Bparallel%2Bevolution%2Bof%2Bpigmentation%2Bin%2Bsticklebacks%2Band%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.055%26rft_id%253Dinfo%253Apmid%252F18083106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18083106&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252217100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.29" data-doi="10.1534/genetics.114.162420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beleza</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollen</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kittles</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">cis-Regulatory changes in <em>Kit ligand</em> expression and parallel evolution of pigmentation in sticklebacks and humans.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.162420%26rft_id%253Dinfo%253Apmid%252F24652999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvNDA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.30" data-doi="10.1111/mec.12350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glazer</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Modular skeletal evolution in sticklebacks Is controlled by additive and clustered Quantitative Trait Loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D22%26rft.spage%253D2841%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buerkle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotyping-by-sequencing in ecological and conservation genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2841</span>–<span class="cit-lpage">2847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D4%26rft.issue%253D0%26rft.spage%253De05290%26rft.epage%253De05290%26rft.atitle%253DA%2Brecurrent%2Bregulatory%2Bchange%2Bunderlying%2Baltered%2Bexpression%2Band%2BWnt%2Bresponse%2Bof%2Bthe%2Bstickleback%2Barmor%2Bplates%2Bgene%2BEDA%26rft_id%253Dinfo%253Apmid%252F25629660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25629660&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.32" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Brown</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A recurrent regulatory change underlying altered expression and Wnt response of the stickleback armor plates gene EDA.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e05290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.33" data-doi="10.2307/2409768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for <em>de novo</em> SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D46%26rft.spage%253D1224%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992JK16900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.34" data-doi="10.1111/mec.12322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reimchen</span>  <span class="cit-name-given-names">T. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Injuries on stickleback from attacks by a toothed predator (Oncorhynchus) and implications for the evolution of lateral plates.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1224</span>–<span class="cit-lpage">1230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D22%26rft.spage%253D3014%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recombination in the threespine stickleback genome–patterns and consequences.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3014</span>–<span class="cit-lpage">3027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRowe%26rft.auinit1%253DH.%2BC.%26rft.volume%253D20%26rft.issue%253D17%26rft.spage%253D3499%26rft.epage%253D3502%26rft.atitle%253DRAD%2Bin%2Bthe%2Brealm%2Bof%2Bnext-generation%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Apmid%252F21991593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21991593&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294224400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guggisberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RAD in the realm of next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3499</span>–<span class="cit-lpage">3502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRowe%26rft.auinit1%253DH.%2BC.%26rft.volume%253D20%26rft.issue%253D17%26rft.spage%253D3499%26rft.epage%253D3502%26rft.atitle%253DRAD%2Bin%2Bthe%2Brealm%2Bof%2Bnext-generation%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Apmid%252F21991593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21991593&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294224400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1463.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">The Ecology of Adaptive Radiation</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.38" data-doi="10.1038/nmeth.2089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPhail</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Ecological character displacement and speciation in sticklebacks.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneider%26rft.auinit1%253DC.%2BA.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D671%26rft.epage%253D675%26rft.atitle%253DNIH%2BImage%2Bto%2BImageJ%253A%2B25%2Byears%2Bof%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2089%26rft_id%253Dinfo%253Apmid%252F22930834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22930834&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305942200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.39" data-doi="10.1126/science.1158997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasband</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eliceiri</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NIH Image to ImageJ: 25 years of image analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStern%26rft.auinit1%253DD.%2BL.%26rft.volume%253D323%26rft.issue%253D5915%26rft.spage%253D746%26rft.epage%253D751%26rft.atitle%253DIs%2BGenetic%2BEvolution%2BPredictable%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158997%26rft_id%253Dinfo%253Apmid%252F19197055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxNS83NDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.40" data-doi="10.1111/mec.12939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orgogozo</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Is genetic evolution predictable?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">746</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D23%26rft.spage%253D5304%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000344374000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.41" data-doi="10.1159/000331232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Theis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronco</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indermaur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzburger</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egger</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Adaptive divergence between lake and stream populations of an East African cichlid fish.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5304</span>–<span class="cit-lpage">5322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DUrton%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D135%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D159%26rft.atitle%253DKaryotype%2Bdifferentiation%2Bbetween%2Btwo%2Bstickleback%2Bspecies%2B%2528Gasterosteidae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000331232%26rft_id%253Dinfo%253Apmid%252F21921583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000331232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921583&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.42" data-doi="10.1038/nrg2267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urton</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCann</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peichel</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Karyotype differentiation between two stickleback species (Gasterosteidae).</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWagner%26rft.auinit1%253DG.%2BP.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D921%26rft.epage%253D931%26rft.atitle%253DThe%2Broad%2Bto%2Bmodularity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2267%26rft_id%253Dinfo%253Apmid%252F18007649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18007649&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251551100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.43" data-doi="10.1038/nrg2267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlicev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The road to modularity.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWagner%26rft.auinit1%253DG.%2BP.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D921%26rft.epage%253D931%26rft.atitle%253DThe%2Broad%2Bto%2Bmodularity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2267%26rft_id%253Dinfo%253Apmid%252F18007649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18007649&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251551100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1463.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wark</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> Genetic architecture of variation in the lateral line sensory system of threespine sticklebacks. G3 (Bethesda) 2: 1047–1056.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/7/1463.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1453" title="Intrinsic Toxicity of Unchecked Heterochromatin Spread Is Suppressed by Redundant Chromatin Boundary Functions in Schizosacchromyces pombe" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1473" title="No Significant Increase in the Δ4- and Δ7-Dafachronic Acid Concentration in the Long-Lived glp-1 Mutant, nor in the Mutants Defective in Dauer Formation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/7"><span>Volume 5 Issue 7, July 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/7.cover-source.jpg?itok=IZbYpE05" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1463" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qvD8eQ-IvUKFN1PKvcRRFh6CB5Nvvipsipozjyox_No" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/7/1463" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/7/1463" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/7/1463" class="active" data-icon-position="" data-hide-link-title="0">Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459974" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-eNWv7qqJnWuToMCqhun43pzNuCs90RK_WcEX0AJDG2I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1463.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/7/1463" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-de_nAprwOhiE66e-jI-h5-0R6KZJ_qKrkt-t-FlgubE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/7/1463" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1463.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459974" id="node459974--2" data-pisa="ggg;5/7/1463" data-pisa-master="ggg;g3.115.017905" data-apath="/ggg/5/7/1463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrew M. Glazer</span>, <span class="highwire-citation-author" data-delta="1">Emily E. Killingbeck</span>, <span class="highwire-citation-author" data-delta="2">Therese Mitros</span>, <span class="highwire-citation-author" data-delta="3">Daniel S. Rokhsar</span> and <span class="highwire-citation-author" data-delta="4">Craig T. Miller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1463-1472;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017905 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459974--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew M. Glazer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BM.%2BGlazer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glazer%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BM.%2BGlazer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emily E. Killingbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BE.%2BKillingbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Killingbeck%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BE.%2BKillingbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Therese Mitros </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Therese%2BMitros%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitros%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATherese%2BMitros%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel S. Rokhsar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BS.%2BRokhsar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokhsar%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BS.%2BRokhsar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Craig T. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BT.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BT.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ctmiller@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">ctmiller@berkeley.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/7/1463" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/7/1463&amp;title=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/7/1463&amp;title=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/7/1463&amp;title=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/7/1463&amp;text=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/7/1463&amp;title=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/7/1463&amp;t=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/7/1463&amp;title=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/7/1463&amp;title=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459974/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459974" id="node459974--3" data-pisa="ggg;5/7/1463" data-pisa-master="ggg;g3.115.017905" data-apath="/ggg/5/7/1463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Assembly Improvement and Mapping Convergently Evolved Skeletal Traits in Sticklebacks with Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrew M. Glazer</span>, <span class="highwire-citation-author" data-delta="1">Emily E. Killingbeck</span>, <span class="highwire-citation-author" data-delta="2">Therese Mitros</span>, <span class="highwire-citation-author" data-delta="3">Daniel S. Rokhsar</span> and <span class="highwire-citation-author" data-delta="4">Craig T. Miller</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1463-1472;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017905 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459974--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew M. Glazer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BM.%2BGlazer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Glazer%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BM.%2BGlazer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Emily E. Killingbeck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BE.%2BKillingbeck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Killingbeck%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BE.%2BKillingbeck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Therese Mitros </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Therese%2BMitros%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitros%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATherese%2BMitros%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel S. Rokhsar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BS.%2BRokhsar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokhsar%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BS.%2BRokhsar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Craig T. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Molecular and Cell Biology Department, University of California-Berkeley, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BT.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BT.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ctmiller@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">ctmiller@berkeley.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1463&amp;rft.epage=1472&amp;rft.atitle=Genome%20Assembly%20Improvement%20and%20Mapping%20Convergently%20Evolved%20Skeletal%20Traits%20in%20Sticklebacks%20with%20Genotyping-by-Sequencing&amp;rft.volume=5&amp;rft.issue=7&amp;rft.date=2015-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Glazer%2C+Andrew+M.&amp;rft.au=Killingbeck%2C+Emily+E.&amp;rft.au=Mitros%2C+Therese&amp;rft.au=Rokhsar%2C+Daniel+S.&amp;rft.au=Miller%2C+Craig+T."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459974/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459974/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459974/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459974/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459974/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459974/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459974/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459974/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459974/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459974/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459974/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459974/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/7/1463" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/7/1463.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/7/1463.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/7/1463.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/7/1463.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
