<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/7/1391" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/7/1391.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/7/1391.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/7/1391.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/7/1391.atom" />
<meta name="HW.pisa" content="ggg;5/7/1391" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" />
<meta name="DC.Identifier" content="10.1534/g3.115.017772" />
<meta name="DC.Date" content="2015-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Randhawa et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Understanding the evolution and molecular architecture of complex traits is important in domestic animals. Due to phenotypic selection, genomic regions develop unique patterns of genetic diversity called signatures of selection, which are challenging to detect, especially for complex polygenic traits. In this study, we applied the composite selection signals (CSS) method to investigate evidence of positive selection in a complex polygenic trait by examining stature in phenotypically diverse cattle comprising 47 European and 8 African Bos taurus breeds, utilizing a panel of 38,033 SNPs genotyped on 1106 animals. CSS were computed for phenotypic contrasts between multibreed cohorts of cattle by classifying the breeds according to their documented wither height to detect the candidate regions under selection. Using the CSS method, clusters of signatures of selection were detected at 26 regions (9 in European and 17 in African cohorts) on 13 bovine autosomes. Using comparative mapping information on human height, 30 candidate genes mapped at 12 selection regions (on 8 autosomes) could be linked to bovine stature diversity. Of these 12 candidate gene regions, three contained known genes ( i.e. , NCAPG-LCORL , FBP2-PTCH1 , and PLAG1-CHCHD7 ) related to bovine stature, and nine were not previously described in cattle (five in European and four in African cohorts). Overall, this study demonstrates the utility of CSS coupled with strategies of combining multibreed datasets in the identification and discovery of genomic regions underlying complex traits. Characterization of multiple signatures of selection and their underlying candidate genes will elucidate the polygenic nature of stature across cattle breeds." />
<meta name="DC.Contributor" content="Imtiaz A. S. Randhawa" />
<meta name="DC.Contributor" content="Mehar S. Khatkar" />
<meta name="DC.Contributor" content="Peter C. Thomson" />
<meta name="DC.Contributor" content="Herman W. Raadsma" />
<meta name="article:published_time" content="2015-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Understanding the evolution and molecular architecture of complex traits is important in domestic animals. Due to phenotypic selection, genomic regions develop unique patterns of genetic diversity called signatures of selection, which are challenging to detect, especially for complex polygenic traits. In this study, we applied the composite selection signals (CSS) method to investigate evidence of positive selection in a complex polygenic trait by examining stature in phenotypically diverse cattle comprising 47 European and 8 African &lt;i&gt;Bos taurus&lt;/i&gt; breeds, utilizing a panel of 38,033 SNPs genotyped on 1106 animals. CSS were computed for phenotypic contrasts between multibreed cohorts of cattle by classifying the breeds according to their documented wither height to detect the candidate regions under selection. Using the CSS method, clusters of signatures of selection were detected at 26 regions (9 in European and 17 in African cohorts) on 13 bovine autosomes. Using comparative mapping information on human height, 30 candidate genes mapped at 12 selection regions (on 8 autosomes) could be linked to bovine stature diversity. Of these 12 candidate gene regions, three contained known genes (&lt;i&gt;i.e.&lt;/i&gt;, &lt;i&gt;NCAPG-LCORL&lt;/i&gt;, &lt;i&gt;FBP2-PTCH1&lt;/i&gt;, and &lt;i&gt;PLAG1-CHCHD7&lt;/i&gt;) related to bovine stature, and nine were not previously described in cattle (five in European and four in African cohorts). Overall, this study demonstrates the utility of CSS coupled with strategies of combining multibreed datasets in the identification and discovery of genomic regions underlying complex traits. Characterization of multiple signatures of selection and their underlying candidate genes will elucidate the polygenic nature of stature across cattle breeds.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/07/01" />
<meta name="citation_mjid" content="ggg;5/7/1391" />
<meta name="citation_id" content="5/7/1391" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/7/1391" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/7/1391.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/7/1391.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/7/1391.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017772" />
<meta name="citation_pmid" content="25931611" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1391" />
<meta name="citation_lastpage" content="1401" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Imtiaz A. S. Randhawa" />
<meta name="citation_author_institution" content="Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia" />
<meta name="citation_author_email" content="imtiaz.randhawa@sydney.edu.au" />
<meta name="citation_author" content="Mehar S. Khatkar" />
<meta name="citation_author_institution" content="Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia" />
<meta name="citation_author" content="Peter C. Thomson" />
<meta name="citation_author_institution" content="Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia" />
<meta name="citation_author" content="Herman W. Raadsma" />
<meta name="citation_author_institution" content="Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia" />
<meta name="citation_reference" content="citation_journal_title=Evol. Anthropol.;citation_author=P. Ajmone-Marsan;citation_author=J. F. Garcia;citation_author=J. A. Lenstra;citation_title=On the origin of cattle: How aurochs became cattle and colonized the world;citation_pages=148-157;citation_volume=19;citation_year=2010;citation_doi=10.1002/evan.20267" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. M. Akey;citation_author=A. L. Ruhe;citation_author=D. T. Akey;citation_author=A. K. Wong;citation_author=C. F. Connelly;citation_title=Tracking footprints of artificial selection in the dog genome.;citation_volume=107;citation_year=2010;citation_pmid=20080661;citation_doi=10.1073/pnas.0909918107" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=W. Barendse;citation_author=B. E. Harrison;citation_author=R. J. Bunch;citation_author=M. B. Thomas;citation_author=L. B. Turner;citation_title=Genome wide signatures of positive selection: the comparison of independent samples and the identification of regions associated to traits.;citation_pages=178-178;citation_volume=10;citation_year=2009;citation_pmid=19393047;citation_doi=10.1186/1471-2164-10-178" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Bolormaa;citation_author=J. E. Pryce;citation_author=A. Reverter;citation_author=Y. Zhang;citation_author=W. Barendse;citation_title=A multi-trait, meta-analysis for detecting pleiotropic polymorphisms for stature, fatness and reproduction in beef cattle.;citation_pages=e1004198;citation_volume=10;citation_year=2014;citation_pmid=24675618;citation_doi=10.1371/journal.pgen.1004198" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=J. L. Crisci;citation_author=Y.-P. Poh;citation_author=A. Bean;citation_author=A. Simkin;citation_author=J. D. Jensen;citation_title=Recent progress in polymorphism-based population genetic inference.;citation_volume=103;citation_year=2012;citation_pmid=22246406;citation_doi=10.1093/jhered/esr128" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Decker;citation_author=D. Vasco;citation_author=S. McKay;citation_author=M. McClure;citation_author=M. Rolf;citation_title=A novel analytical method, birth date selection mapping, detects response of the Angus (Bos taurus) genome to selection on complex traits.;citation_pages=606-606;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23140540;citation_doi=10.1186/1471-2164-13-606" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. E. Decker;citation_author=J. C. Pires;citation_author=G. C. Conant;citation_author=S. D. McKay;citation_author=M. P. Heaton;citation_title=Resolving the evolution of extant and extinct ruminants with high-throughput phylogenomics.;citation_volume=106;citation_year=2009;citation_pmid=19846765;citation_doi=10.1073/pnas.0904691106" />
<meta name="citation_reference" content="citation_journal_title=Bone;citation_author=F. Y. Deng;citation_author=S. S. Dong;citation_author=X. H. Xu;citation_author=Y. J. Liu;citation_author=Y. Z. Liu;citation_title=Genome-wide association study identified UQCC locus for spine bone size in humans.;citation_pages=129-133;citation_volume=53;citation_year=2013;citation_pmid=23207799;citation_doi=10.1016/j.bone.2012.11.028" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=T. Druet;citation_author=L. Pérez-Pardal;citation_author=C. Charlier;citation_author=M. Gautier;citation_title=Identification of large selective sweeps associated with major genes in cattle.;citation_pages=758-762;citation_volume=44;citation_year=2013;citation_pmid=23859468" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=Z. Edea;citation_author=H. Dadi;citation_author=S. W. Kim;citation_author=J. H. Park;citation_author=G. H. Shin;citation_title=Linkage disequilibrium and genomic scan to detect selective loci in cattle populations adapted to different ecological conditions in Ethiopia.;citation_year=2014;citation_doi=10.1111/jbg.12083" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Flori;citation_author=S. Fritz;citation_author=F. Jaffrézic;citation_author=M. Boussaha;citation_author=I. Gut;citation_title=The genome response to artificial selection: a case study in dairy cattle.;citation_pages=e6595-e6595;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19672461;citation_doi=10.1371/journal.pone.0006595" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=M. R. S. Fortes;citation_author=K. Kemper;citation_author=S. Sasazaki;citation_author=A. Reverter;citation_author=J. E. Pryce;citation_title=Evidence for pleiotropism and recent selection in the PLAG1 region in Australian Beef cattle.;citation_pages=636-647;citation_volume=44;citation_year=2013;citation_pmid=23909810;citation_doi=10.1111/age.12075" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. Gautier;citation_author=L. Flori;citation_author=A. Riebler;citation_author=F. Jaffrezic;citation_author=D. Laloe;citation_title=A whole genome Bayesian scan for adaptive genetic divergence in West African cattle.;citation_pages=550-550;citation_volume=10;citation_year=2009;citation_pmid=19930592;citation_doi=10.1186/1471-2164-10-550" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=M. Gautier;citation_author=D. Laloë;citation_author=K. Moazami-Goudarzi;citation_title=Insights into the genetic history of French cattle from dense SNP data on 47 Worldwide breeds.;citation_pages=e13038;citation_volume=5;citation_year=2010;citation_pmid=20927341;citation_doi=10.1371/journal.pone.0013038" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. Gautier;citation_author=M. Naves;citation_title=Footprints of selection in the ancestral admixture of a New World Creole cattle breed.;citation_pages=3128-3143;citation_volume=20;citation_year=2011;citation_issue=15;citation_pmid=21689193;citation_doi=10.1111/j.1365-294X.2011.05163.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. A. Gibbs;citation_author=J. F. Taylor;citation_author=C. P. Van Tassell;citation_author=W. Barendse;citation_author=K. A. Eversole;citation_title=Genome-Wide Survey of SNP Variation Uncovers the Genetic Structure of Cattle Breeds;citation_pages=528-532;citation_volume=324;citation_year=2009;citation_issue=5926;citation_pmid=19390050;citation_doi=10.1126/science.1167936" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. R. Grossman;citation_author=I. Shylakhter;citation_author=E. K. Karlsson;citation_author=E. H. Byrne;citation_author=S. Morales;citation_title=A composite of multiple signals distinguishes causal variants in regions of positive selection.;citation_volume=327;citation_year=2010;citation_pmid=20056855;citation_doi=10.1126/science.1183863" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Gudbjartsson;citation_author=G. B. Walters;citation_author=G. Thorleifsson;citation_author=H. Stefansson;citation_author=B. V. Halldorsson;citation_title=Many sequence variants affecting diversity of adult human height.;citation_pages=609-615;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18391951;citation_doi=10.1038/ng.122" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Anthropology;citation_journal_abbrev=Annu. Rev. Anthropol.;citation_author=A. M. Hancock;citation_author=A. Di Rienzo;citation_title=Detecting the Genetic Signature of Natural Selection in Human Populations: Models, Methods, and Data.;citation_pages=197-217;citation_volume=37;citation_year=2008;citation_pmid=20622977;citation_doi=10.1146/annurev.anthro.37.081407.085141" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=B. J. Hayes;citation_author=S. Lien;citation_author=H. Nilsen;citation_author=H. G. Olsen;citation_author=P. Berg;citation_title=The origin of selection signatures on bovine chromosome 6.;citation_pages=105-111;citation_volume=39;citation_year=2008;citation_issue=2;citation_pmid=18241301;citation_doi=10.1111/j.1365-2052.2007.01683.x" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=B. J. Hayes;citation_author=A. J. Chamberlain;citation_author=S. Maceachern;citation_author=K. Savin;citation_author=H. McPartlan;citation_title=A genome map of divergent artificial selection between Bos taurus dairy cattle and Bos taurus beef cattle.;citation_pages=176-184;citation_volume=40;citation_year=2009;citation_issue=2;citation_pmid=19067671;citation_doi=10.1111/j.1365-2052.2008.01815.x" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Plant Sciences;citation_journal_abbrev=International Journal of Plant Sciences;citation_author=P. A. Hohenlohe;citation_author=P. C. Phillips;citation_author=W. A. Cresko;citation_title=USING POPULATION GENOMICS TO DETECT SELECTION IN NATURAL POPULATIONS: KEY CONCEPTS AND METHODOLOGICAL CONSIDERATIONS.;citation_pages=1059-1071;citation_volume=171;citation_year=2010;citation_issue=9;citation_pmid=21218185;citation_doi=10.1086/656306" />
<meta name="citation_reference" content="citation_journal_title=Anim. Sci. J.;citation_author=H. Hoshiba;citation_author=K. Setoguchi;citation_author=T. Watanabe;citation_author=A. Kinoshita;citation_author=K. Mizoshita;citation_title=Comparison of the effects explained by variations in the bovine PLAG1 and NCAPG genes on daily body weight gain, linear skeletal measurements and carcass traits in Japanese Black steers from a progeny testing program.;citation_pages=529-534;citation_volume=84;citation_year=2013;citation_pmid=23607392" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=S. J. Kang;citation_author=C. W. K. Chiang;citation_author=C. D. Palmer;citation_author=B. O. Tayo;citation_author=G. Lettre;citation_title=Genome-wide association of anthropometric traits in African- and African-derived populations.;citation_volume=19;citation_year=2010;citation_pmid=20400458;citation_doi=10.1093/hmg/ddq154" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. Karim;citation_author=H. Takeda;citation_author=L. Lin;citation_author=T. Druet;citation_author=J. A. C. Arias;citation_title=Variants modulating the expression of a chromosome domain encompassing PLAG1 influence bovine stature.;citation_pages=405-413;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21516082;citation_doi=10.1038/ng.814" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=K. E. Kemper;citation_author=M. E. Goddard;citation_title=Understanding and predicting complex traits: knowledge from cattle.;citation_volume=21;citation_year=2012;citation_pmid=22899652;citation_doi=10.1093/hmg/dds332" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. Kemper;citation_author=S. Saxton;citation_author=S. Bolormaa;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Selection for complex traits leaves little or no classic signatures of selection.;citation_pages=246;citation_volume=15;citation_year=2014;citation_pmid=24678841;citation_doi=10.1186/1471-2164-15-246" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=J. W. Kijas;citation_author=J. A. Lenstra;citation_author=B. Hayes;citation_author=S. Boitard;citation_author=L. R. Porto Neto;citation_title=Genome-wide analysis of the world's sheep breeds reveals high levels of historic mixture and strong recent selection.;citation_pages=e1001258-e1001258;citation_volume=10;citation_year=2012;citation_issue=2;citation_pmid=22346734;citation_doi=10.1371/journal.pbio.1001258" />
<meta name="citation_reference" content="citation_journal_title=Journal of human genetics;citation_journal_abbrev=J Hum Genet;citation_author=J.-J. Kim;citation_author=H.-I. Lee;citation_author=T. Park;citation_author=K. Kim;citation_author=J.-E. Lee;citation_title=Identification of 15 loci influencing height in a Korean population.;citation_pages=27-31;citation_volume=55;citation_year=2010;citation_issue=1;citation_pmid=19893584;citation_doi=10.1038/jhg.2009.116" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Lango Allen;citation_author=K. Estrada;citation_author=G. Lettre;citation_author=S. I. Berndt;citation_author=M. N. Weedon;citation_title=Hundreds of variants clustered in genomic loci and biological pathways affect human height.;citation_pages=832-838;citation_volume=467;citation_year=2010;citation_issue=7317;citation_pmid=20881960;citation_doi=10.1038/nature09410" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. B. Lanktree;citation_author=Y. Guo;citation_author=M. Murtaza;citation_author=J. T. Glessner;citation_author=S. D. Bailey;citation_title=Meta-analysis of Dense Genecentric Association Studies Reveals Common and Uncommon Variants Associated with Height.;citation_pages=6-18;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21194676;citation_doi=10.1016/j.ajhg.2010.11.007" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=J. A. Lenstra;citation_author=L. F. Groeneveld;citation_author=H. Eding;citation_author=J. Kantanen;citation_author=J. L. Williams;citation_title=Molecular tools and analytical approaches for the characterization of farm animal genetic diversity.;citation_pages=483-502;citation_volume=43;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=G. Lettre;citation_title=Recent progress in the study of the genetics of height.;citation_pages=465-472;citation_volume=129;citation_year=2011;citation_issue=5;citation_pmid=21340692;citation_doi=10.1007/s00439-011-0969-x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. Lettre;citation_author=A. U. Jackson;citation_author=C. Gieger;citation_author=F. R. Schumacher;citation_author=S. I. Berndt;citation_title=Identification of ten loci associated with height highlights new biological pathways in human growth.;citation_pages=584-591;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18391950;citation_doi=10.1038/ng.125" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=M.-X. Li;citation_author=X.-M. Sun;citation_author=L.-Z. Zhang;citation_author=J. Wang;citation_author=Y.-Z. Huang;citation_title=A novel c.-274C&gt;G polymorphism in bovine SIRT1 gene contributes to diminished promoter activity and is associated with increased body size.;citation_pages=584-587;citation_volume=44;citation_year=2013;citation_pmid=23647079" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=A. Lindholm-Perry;citation_author=A. Sexten;citation_author=L. Kuehn;citation_author=T. Smith;citation_author=D. King;citation_title=Association, effects and validation of polymorphisms within the NCAPG - LCORL locus located on BTA6 with feed intake, gain, meat and carcass traits in beef cattle.;citation_pages=103-103;citation_volume=12;citation_year=2011;citation_pmid=22168586" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Rep.;citation_author=G. Mancini;citation_author=M. Gargani;citation_author=G. Chillemi;citation_author=E. Nicolazzi;citation_author=P. A. Marsan;citation_title=Signatures of selection in five Italian cattle breeds detected by a 54K SNP panel.;citation_pages=957-965;citation_volume=41;citation_year=2014;citation_pmid=24442315;citation_doi=10.1007/s11033-013-2940-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. K. Matukumalli;citation_author=C. T. Lawley;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=M. F. Allan;citation_title=Development and characterization of a high density SNP genotyping assay for cattle.;citation_pages=e5350-e5350;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19390634;citation_doi=10.1371/journal.pone.0005350" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. P. Mullen;citation_author=D. P. Berry;citation_author=D. J. Howard;citation_author=M. G. Diskin;citation_author=C. O. Lynch;citation_title=Associations between novel single nucleotide polymorphisms in the Bos taurus growth hormone gene and performance traits in Holstein-Friesian dairy cattle.;citation_pages=5959-5969;citation_volume=93;citation_year=2010;citation_issue=12;citation_pmid=21094770;citation_doi=10.3168/jds.2010-3385" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=S. Nishimura;citation_author=T. Watanabe;citation_author=K. Mizoshita;citation_author=K. Tatsuda;citation_author=T. Fujita;citation_title=Genome-wide association study identified three major QTL for carcass weight including the PLAG1-CHCHD7 QTN for stature in Japanese Black cattle.;citation_pages=40-40;citation_volume=13;citation_year=2012;citation_pmid=22607022" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=Y. Okada;citation_author=Y. Kamatani;citation_author=A. Takahashi;citation_author=K. Matsuda;citation_author=N. Hosono;citation_title=A genome-wide association study in 19 633 Japanese subjects identified LHX3-QSOX2 and IGF1 as adult height loci.;citation_volume=19;citation_year=2010;citation_pmid=20189936;citation_doi=10.1093/hmg/ddq091" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=T. K. Oleksyk;citation_author=M. W. Smith;citation_author=S. J. O’Brien;citation_title=Genome-wide scans for footprints of natural selection;citation_pages=185-205;citation_volume=365;citation_year=2010;citation_issue=1537;citation_pmid=20008396;citation_doi=10.1098/rstb.2009.0219" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=O. Perez;citation_author=A. Brien;citation_author=Y. Utsunomiya;citation_author=G. Meszaros;citation_author=D. Bickhart;citation_title=Assessing signatures of selection through variation in linkage disequilibrium between taurine and indicine cattle.;citation_pages=19;citation_volume=46;citation_year=2014;citation_pmid=24592996;citation_doi=10.1186/1297-9686-46-19" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. L. Petersen;citation_author=J. R. Mickelson;citation_author=A. K. Rendahl;citation_author=S. J. Valberg;citation_author=L. S. Andersson;citation_title=Genome-wide analysis reveals selection for important traits in domestic horse breeds.;citation_pages=e1003211;citation_volume=9;citation_year=2013;citation_pmid=23349635;citation_doi=10.1371/journal.pgen.1003211" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Pickrell;citation_author=G. Coop;citation_author=J. Novembre;citation_author=S. Kudaravalli;citation_author=J. Li;citation_title=Signals of recent positive selection in a worldwide sample of human populations.;citation_volume=19;citation_year=2009;citation_pmid=19307593;citation_doi=10.1101/gr.087577.108" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=E. Pintus;citation_author=S. Sorbolini;citation_author=A. Albera;citation_author=G. Gaspa;citation_author=C. Dimauro;citation_title=Use of locally weighted scatterplot smoothing (LOWESS) regression to study selection signatures in Piedmontese and Italian Brown cattle breeds.;citation_pages=1-11;citation_volume=45;citation_year=2013;citation_pmid=23889699" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Porto-Neto;citation_author=T. Sonstegard;citation_author=G. Liu;citation_author=D. Bickhart;citation_author=M. Da Silva;citation_title=Genomic divergence of zebu and taurine cattle identified through high-density SNP genotyping.;citation_pages=876;citation_volume=14;citation_year=2013;citation_pmid=24330634;citation_doi=10.1186/1471-2164-14-876" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. E. Pryce;citation_author=B. J. Hayes;citation_author=S. Bolormaa;citation_author=M. E. Goddard;citation_title=Polymorphic regions affecting human height also control stature in cattle.;citation_volume=187;citation_year=2011;citation_pmid=21212230;citation_doi=10.1534/genetics.110.123943" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Qanbari;citation_author=D. Gianola;citation_author=B. Hayes;citation_author=F. Schenkel;citation_author=S. Miller;citation_title=Application of site and haplotype-frequency based approaches for detecting selection signatures in cattle.;citation_pages=318-318;citation_volume=12;citation_year=2011;citation_pmid=21679429;citation_doi=10.1186/1471-2164-12-318" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Qanbari;citation_author=H. Pausch;citation_author=S. Jansen;citation_author=M. Somel;citation_author=T. M. Strom;citation_title=Classic selective sweeps revealed by massive sequencing in cattle.;citation_pages=e1004148;citation_volume=10;citation_year=2014;citation_pmid=24586189;citation_doi=10.1371/journal.pgen.1004148" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Ramey;citation_author=J. Decker;citation_author=S. McKay;citation_author=M. Rolf;citation_author=R. Schnabel;citation_title=Detection of selective sweeps in cattle using genome-wide SNP data.;citation_pages=382;citation_volume=14;citation_year=2013;citation_pmid=23758707;citation_doi=10.1186/1471-2164-14-382" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=I. A. S. Randhawa;citation_author=M. S. Khatkar;citation_author=P. C. Thomson;citation_author=H. W. Raadsma;citation_title=Composite selection signals can localize the trait specific genomic regions in multi-breed populations of cattle and sheep.;citation_pages=34;citation_volume=15;citation_year=2014;citation_pmid=24636660" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Rothammer;citation_author=D. Seichter;citation_author=M. Forster;citation_author=I. Medugorac;citation_title=A genome-wide scan for signatures of differential artificial selection in ten cattle breeds.;citation_pages=908;citation_volume=14;citation_year=2013;citation_pmid=24359457;citation_doi=10.1186/1471-2164-14-908" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=P. Varilly;citation_author=B. Fry;citation_author=J. Lohmueller;citation_author=E. Hostetter;citation_title=Genome-wide detection and characterization of positive selection in human populations.;citation_pages=913-918;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943131;citation_doi=10.1038/nature06250" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Sanna;citation_author=A. U. Jackson;citation_author=R. Nagaraja;citation_author=C. J. Willer;citation_author=W.-M. Chen;citation_title=Common variants in the GDF5-UQCC region are associated with variation in human height.;citation_pages=198-203;citation_volume=40;citation_year=2008;citation_issue=2;citation_pmid=18193045;citation_doi=10.1038/ng.74" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=D. Seichter;citation_author=I. Russ;citation_author=S. Rothammer;citation_author=J. Eder;citation_author=M. Förster;citation_title=SNP-based association mapping of the polled gene in divergent cattle breeds.;citation_pages=595-598;citation_volume=43;citation_year=2012;citation_issue=5;citation_pmid=22497248;citation_doi=10.1111/j.1365-2052.2011.02302.x" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=H. Shiura;citation_author=K. Nakamura;citation_author=T. Hikichi;citation_author=T. Hino;citation_author=K. Oda;citation_title=Paternal deletion of Meg1/Grb10 DMR causes maternalization of the Meg1/Grb10 cluster in mouse proximal chromosome 11 leading to severe pre- and postnatal growth retardation.;citation_volume=18;citation_year=2009;citation_pmid=19174477;citation_doi=10.1093/hmg/ddp049" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Shriner;citation_author=A. Adeyemo;citation_author=N. P. Gerry;citation_author=A. Herbert;citation_author=G. Chen;citation_title=Transferability and fine-mapping of genome-wide associated loci for adult height across human populations.;citation_pages=e8398-e8398;citation_volume=4;citation_year=2009;citation_issue=12;citation_pmid=20027299;citation_doi=10.1371/journal.pone.0008398" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Signer-Hasler;citation_author=C. Flury;citation_author=B. Haase;citation_author=D. Burger;citation_author=H. Simianer;citation_title=A genome-wide association study reveals loci influencing height and other conformation traits in horses.;citation_pages=e37282-e37282;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22615965;citation_doi=10.1371/journal.pone.0037282" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Soranzo;citation_author=F. Rivadeneira;citation_author=U. Chinappen-Horsley;citation_author=I. Malkina;citation_author=J. B. Richards;citation_title=Meta-analysis of genome-wide scans for human adult stature identifies novel Loci and associations with measures of skeletal frame size.;citation_pages=e1000445-e1000445;citation_volume=5;citation_year=2009;citation_issue=4;citation_pmid=19343178;citation_doi=10.1371/journal.pgen.1000445" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=U. Sovio;citation_author=A. J. Bennett;citation_author=I. Y. Millwood;citation_author=J. Molitor;citation_author=P. F. O’Reilly;citation_title=Genetic determinants of height growth assessed longitudinally from infancy to adulthood in the northern Finland birth cohort 1966.;citation_pages=e1000409-e1000409;citation_volume=5;citation_year=2009;citation_issue=3;citation_pmid=19266077;citation_doi=10.1371/journal.pgen.1000409" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Stella;citation_author=P. Ajmone-Marsan;citation_author=B. Lazzari;citation_author=P. Boettcher;citation_title=Identification of selection signatures in cattle breeds selected for dairy production.;citation_volume=185;citation_year=2010;citation_pmid=20479146;citation_doi=10.1534/genetics.110.116111" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=J. Tetens;citation_author=P. Widmann;citation_author=C. Kühn;citation_author=G. Thaller;citation_title=A genome-wide association study indicates LCORL/NCAPG as a candidate locus for withers height in German Warmblood horses.;citation_pages=467-471;citation_volume=44;citation_year=2013;citation_pmid=23418885;citation_doi=10.1111/age.12031" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=Y. T. Utsunomiya;citation_author=A. M. Pérez O’Brien;citation_author=T. S. Sonstegard;citation_author=C. P. Van Tassell;citation_author=A. S. do Carmo;citation_title=Detecting loci under recent positive selection in dairy and beef cattle by combining different genome-wide scan methods.;citation_pages=e64280;citation_volume=8;citation_year=2013;citation_pmid=23696874;citation_doi=10.1371/journal.pone.0064280" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Cattle gain stature.;citation_pages=397-398;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21522175" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. N. Weedon;citation_author=H. Lango;citation_author=C. M. Lindgren;citation_author=C. Wallace;citation_author=D. M. Evans;citation_title=Genome-wide association analysis identifies 20 loci that influence adult height.;citation_pages=575-583;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18391952;citation_doi=10.1038/ng.121" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Zhang;citation_author=S.-Z. Wang;citation_author=Z.-P. Wang;citation_author=Y. Da;citation_author=N. Wang;citation_title=A genome-wide scan of selective sweeps in two broiler chicken lines divergently selected for abdominal fat content.;citation_pages=704;citation_volume=13;citation_year=2012;citation_pmid=23241142;citation_doi=10.1186/1471-2164-13-704" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="twitter:description" content="Understanding the evolution and molecular architecture of complex traits is important in domestic animals. Due to phenotypic selection, genomic regions develop unique patterns of genetic diversity called signatures of selection, which are challenging to detect, especially for complex polygenic traits. In this study, we applied the composite selection signals (CSS) method to investigate evidence of positive selection in a complex polygenic trait by examining stature in phenotypically diverse cattle comprising 47 European and 8 African Bos taurus breeds, utilizing a panel of 38,033 SNPs genotyped on 1106 animals. CSS were computed for phenotypic contrasts between multibreed cohorts of cattle by classifying the breeds according to their documented wither height to detect the candidate regions under selection. Using the CSS method, clusters of signatures of selection were detected at 26 regions (9 in European and 17 in African cohorts) on 13 bovine autosomes. Using comparative mapping information on human height, 30 candidate genes mapped at 12 selection regions (on 8 autosomes) could be linked to bovine stature diversity. Of these 12 candidate gene regions, three contained known genes ( i.e. , NCAPG-LCORL , FBP2-PTCH1 , and PLAG1-CHCHD7 ) related to bovine stature, and nine were not previously described in cattle (five in European and four in African cohorts). Overall, this study demonstrates the utility of CSS coupled with strategies of combining multibreed datasets in the identification and discovery of genomic regions underlying complex traits. Characterization of multiple signatures of selection and their underlying candidate genes will elucidate the polygenic nature of stature across cattle breeds." />
<meta name="og-title" property="og:title" content="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/7/1391" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Understanding the evolution and molecular architecture of complex traits is important in domestic animals. Due to phenotypic selection, genomic regions develop unique patterns of genetic diversity called signatures of selection, which are challenging to detect, especially for complex polygenic traits. In this study, we applied the composite selection signals (CSS) method to investigate evidence of positive selection in a complex polygenic trait by examining stature in phenotypically diverse cattle comprising 47 European and 8 African Bos taurus breeds, utilizing a panel of 38,033 SNPs genotyped on 1106 animals. CSS were computed for phenotypic contrasts between multibreed cohorts of cattle by classifying the breeds according to their documented wither height to detect the candidate regions under selection. Using the CSS method, clusters of signatures of selection were detected at 26 regions (9 in European and 17 in African cohorts) on 13 bovine autosomes. Using comparative mapping information on human height, 30 candidate genes mapped at 12 selection regions (on 8 autosomes) could be linked to bovine stature diversity. Of these 12 candidate gene regions, three contained known genes ( i.e. , NCAPG-LCORL , FBP2-PTCH1 , and PLAG1-CHCHD7 ) related to bovine stature, and nine were not previously described in cattle (five in European and four in African cohorts). Overall, this study demonstrates the utility of CSS coupled with strategies of combining multibreed datasets in the identification and discovery of genomic regions underlying complex traits. Characterization of multiple signatures of selection and their underlying candidate genes will elucidate the polygenic nature of stature across cattle breeds." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
    <title>Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-1PIdHzEuBWM9Vo93o9f54msCmULAeIzHB6G3MNofcU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459978","apath":"\/ggg\/5\/7\/1391.atom","pisa":"ggg;5\/7\/1391","ac":{"\/ggg\/5\/7\/1391.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/7\/1391.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1391"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1391"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/7\/1391":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+INLeTPw8G3AxYFbod3hRDpK49Y6MRvXet5PfXpOdUzexc9yEPc3BIy45eZMG73ChKkNhdr+mVmIBwbg3M5viPZeQyywtRtUkAZesA0XscfgUkR7TH8o6myCPIQUPgSzIPPx1uG+4fE7p4Vn4OPTyBLkMHf19BPJeI7JiwlvhX5XAf5fDtM4eehQYkJT\\\\\\\/VrnWKSmIJ\\\\\\\/KRwkpdrKWNYj+v595\\\\\\\/nozte4Y6jR6VLFJLIb0cmTQv6yIvB67wRXkJrHlb8ZM1fzbnzfZ\\\\\\\/2L3BirHhkwkYLmZU6kTquPbi4unwnKnWkpHn\\\\\\\/BvoLAzww8tiUokYl1GMHl1XvR3ytvq1tUx9cySXihUhXY3QY6QxCvW9JVYd0AklHIPAn3v2xeCQXcu07tz+ZIfnv5NtPoUe5MHuM1OIJvoexLjTXOA6359eCE\\\\\\\/v10tKdZg03jw3R\\\\\\\/ITWFSdJfiuM7RY2FzOyrPGV72CY\\\\\\\/rTlkhaKLhBfyeSdLFZvS1EzuhGJDNQ+u5Ypu\\\\\\\/qT01e3haW5DuIL\\\\\\\/Pkrwr77IpirJxgmE6KRlr37x2QfNqK6FqG1IrHTkE97dOKb+GEC4qDuEfDKkMAT+8TrQE0HihxEUVGYUuoZTR3aYm1nk\\\\\\\/zzUw+daAsQynBAB9zrLjvNi1u7265mXcPC6g1vrd6oYwg+KPNY1n2OM5s2L7s95E\\\\\\\/e8wVutHo68CheHrGWwcS8f7upQMKpoQmR2XrKoKYY2id4tYMtcATWTnYZjrvo4X0xfVnrPiNrUPnjbBqkv7J30d5fGSlCmJICSEJPMY197xxJp0GphG0vAJk+Xl8TZLL1ghqZIQmHM09St4z1lua\\\\\\\/7nVWwCVcYEzTQceH6rpmYpOkdO+\\\\\\\/1eQ+pc3N6kK7WGpGN+EfVR5QWA2x4XVTwmpSUn3TT1rDKdwVVtloM9D1vp3V\\\\\\\/+Ny61UldGhKyy89y9J1oIGPrCs0Wh62sw5LUtDcZ2nknPal9reElB+IgSBo1BV6ZwwRud1\\\\\\\/ePBeTc5tHbaGznM5zRqhtdr49lx1SWKSEs33wX825KpQWZp650dFyuknQmgDUWbyeCQZ\\\\\\\/Apt8dVST3OMKFN6JwdRUNEjh4Z3d7sNHOjcwxI3nTr8wOw73kgjIP+icKISYTTuG7hAQnqYX7GiXI0V1KSPOAZKTIbPXMFyiCATStIhP+dOHXoelOJSVP\\\\\\\/jPdwsWs8scrPcm3NxvFxyCS236L5VHFt15a3TFkzX\\\\\\\/uiMOPj9a0AjsZXqXlYS7cYzYFLbkI9oSxrzUTKIkpBdbtkJyEs6eH9F9ZwoBD\\\u0022,\\\u0022iv\\\u0022:\\\u0022E\\\\\\\/Khwt8Jr8xooFyDXo3Z\\\\\\\/Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c5f895633d2dee39c578649176f9099c22317760baea2dd7802113adf1414959\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nLlg72U1NVy\\\\\\\/ITINNZ6tcg1xKLY0iwHOF1j9\\\\\\\/qn4PHi\\\\\\\/UyR13GIN4UY9e4Q5OcGXc2rtmhhx8IB3s1NW0yRuXTKePUSpPIiH1xUewvBgyv1uxlgOw2jObNvoUomMz1ovRbtcy2pOdaNlaXi4Efx1QFUDNEmeElcUSqY2t8um+j+26qFFGOH0Q4b\\\\\\\/vQWPo\\\\\\\/ciOFiqaJTlsIh9L+S2ziC5RDtzOkZx3KjvQtEwM\\\\\\\/N0xPGTTiwkbhCWxPyya9+2RiVajZLXfH7yd+YpJ5MJFsVSlvwErMqb3Wq2sqVhewewFxiz5YEhLmTuZ9uBviXIpeqDOR7Qbj0f6D71RUvs8dznC5u7DwwPOOrCbWbusyfcs01rzRoIvLXOPet\\\\\\\/Oz86kGhhUGlN8xMC5X9Il8IXZOkeqNnDmCku6JMJJJxL+2L9KUqYRrrTIqGsMrMrQJ8fgLZ5RcQPcgp\\\\\\\/nL3\\\\\\\/RAORmpY2R+EHRzyOygVBfVGNjJgB31EHrVzbMb740pBPUJnz206kuHjJN2oiFyJisHtWE6izEXbQgdiwHEZfo9bQG2QM+ExavazkMS5fdtTyZ2UZ5jL+I6QjhjzhgaagYd0OGYv0Zdh\\\\\\\/U9ENLJluYXoEQ3ciP5zpcirJgOZ9YXuTOgmwwO51y7yHxapzfRatVJFBJAqsyQf\\\\\\\/wbTAJa7Fu+mfJBRqEyznNEqQ9i8L6OB0OAG8AJ69VNqbB9fXh4K3TlSHEltAbK\\\\\\\/Hs7i1MFVuJwgvL745WP6SUTyn3lMjzEs9SaaCBmfCcMYbhUpmFpPs53w8qJGMG9Ffg+CWctIXE7bgKpkSnK9ysILmztyG3rjR6Wn450QWiDXlpsFFZ9GTt1xmQGfoikWjOWyJCk9p9nHuA5L8m4Pi76Towe0Ji\\\\\\\/m1KUeQwip4oieqdEWjVgj8Hc+y+JfEuhSk2Q0kPLy55UAxqRSuKNQ9HyM4xxS64Bnf56phOxoEaYPsA8sDbskjcFxZWCPyMtW\\\\\\\/E9OD24heRcMfp47IrmsRoNT4QPSkQm29FdFqGGThnb0ekYtRUyUp70CRyH4ngYyqLyAC8\\\\\\\/DRjd9Pw9lZtx9lqaJXTo7wKzenWXv5qfMK4Cya4Ddg8pNbt+4ZCtOoSd9Iynr9ahJ9\\\\\\\/yyGM2zOnhJ14\\\\\\\/33lIars0hUiT8JnIBKUCJ0jmLBvqGKpx0ObGIHY5nVk6HJ9MwluuvlerGKxe7Tdzvy6tId5Sk5Mdcfe5AARfk3utieUFDL6d3Txt0t9MA9uk+vIngFkHobK8ZLxTtT5hlW+T2OQmCqY9W6oiVmEUrbovRfEScmQZ0s3gOALkLdN\\\\\\\/rUqAJ\\\\\\\/\\\\\\\/yekGi8rQzZo3\\\\\\\/uQVvknsqo5ySx6L6UGJFKxG6kInXptJDU0yVtxnVoNreZT1UFIR5YRhbrsoQcyLanUbcZnYMxG2ZvsvmScqO9Tow9DQlCo+O5bxzkopmw+LLxG35vRSyBKkH8Fd5kkx1cjvV0q86ZObW5FTuFfsYttzJnpkO7nW6Zg+b8Z7\\\\\\\/YaCH2AOswjQRWAi\\\\\\\/oYNzuXeFoQMlDpl0vB65GZrrECAD6nOk80WxHhg6wYeeiHN9bxWGsUt0J7Mi\\\\\\\/Y9B3Mo3f5yj1Nn3vfGkJQeoffIly0TKDk9mThmoqt13HChPmBSBVEevs4hRXqtEiRJHcm1XxDsV2Pv\\\\\\\/D3JlgkFVcp9QhxNIYnKm7+9FSOG8cFtuHLt08vNga+OT3kBa+krZ\\\\\\\/DlljHaXJHQhLi6ixth1lC8ap884h1Fksb4zZH2NJdZHaeD7qfSQq317An1v99zTE2C5vH1GZDBmbUFCKpU4cJarTjEs7FtFKeuniOT25oGXTqOSGqQXLasI1ceOEuG6QG38SpWoFRUca9TrQiH51AM0KiaVapXVH2iA0cP0oCKeK1KirXkG71V\\\\\\\/I43xceA3VEWNFSbbKx6RT5D9Y8KZZXHACb0QBmOb\\\\\\\/gtCIoXjbkTmGHNf0HNnHiXPPHm+pZIi36xI8KyvLwVjDbqEgnhwtp+00B5kStgk3P68WfNjf1H6qbFn6ptEl5Hgh0zRl5xo29u3t1P7cKPfCwozzKl2QlSyEh4E8EAkzemNK24Ru\\\\\\\/GSBOxulaHNhrBYx8XuKvZysi6o5DryolYCKzN4DwEz6g9hGLHrqT6+zHa+UU4GzPcSFnZypYHHmR4rVQDJQAzIWmwjxZNRY0Eto0wohN1Uew9Gq5FJEWoZvM3\\\\\\\/TGxvyA8X4HaSYK\\\\\\\/ibMCzAsWp8c8Hz0OU9GygBcmYQojAKTHGcihGb1tEFRhA07uVZ0M\\\\\\\/qT\\\\\\\/D5VnWzJ7goMz0H9I0PvdCN4np+CJAwXuHDNbq3u\\\\\\\/JwTzooD9k862sFqDWsmkASalvb2OZk2oLi4QwDozh2PBHcDkqUt0mhLYctAjE4BbeYkTeXwclJgL7giOpjjnLu2dpqBmJxx1bxPOML3+Gb5bi66RtLkWLvQXzYqoF1g5LBoebW\\\\\\\/35a8SOkg5TSOMluXk\\\\\\\/7qDxIT9XQHRvxXEgigWmpfl6i6lr4PpdPBLEZJ4YtkFCPLn5AS0m7mvFYe3s36Ga4DqHY4RNqDVI\\\\\\\/p1eNHjUnFupQ+1OqBcnLwUDXgdiSrLudtqjBe9cmrl9YKVTikH21Ab62tLqqrqs02i1dalQR0GiiZIAwo3hUVVkcHt21Fi7CWl1plcLc+xLVBawDqQEpUD72LY\\\\\\\/rABY54L4kQw31zyUirSsKieJ0JnK5EY4x\\\\\\\/FVvUEhppYGCjEhPvJVdTWZ8z8aLfwfb8dIye56ICKpR3L2l+hPNOH0\\\\\\\/YzaNhutQX8vFuyjmBfsaP+aCgIj7csh6G10BZ+ZVLqZqcjOJRjYNpFICpmwFrfb8wUtoOQ0nCPEebCr8oJilcd9cukmsmONLcKUJSE88D3MOg9yqLlYirbONVuZ744vxGrDg8BxQdyeY5qU7wOaIrSZd+on4asgIt3HJ3FT9LPSti0u8M\\\\\\\/F0339o0TrW7+I3uBgVUSSR457jODWc3MBPLKJqz9HIv6V0cXp8IRGvE7vE4qKLv+HS3EZZT1BJKriDksluwbTJM\\\\\\\/TfvPs70FsAPer+1JkYrkI9nM6Mbcn66OWQfYCBDQM3cDjkBxi9VjoyAY2EzcWnJyZMKN\\\\\\\/A+exKauzXszfkCWkqeDzAxPMcjtvKeIwzjv9j8O\\\\\\\/VTFXuQdBGlLI6ycVbQjP4k\\\\\\\/\\\\\\\/4krSt4UxbvGEi4He0q\\\\\\\/EpGOjA8yGF3hZptguvYM0YcqDpQnQ3NgOjMUjmXoNGAmjAx8iMbL+1M3Zqe6EvDYj867Cp917gGTmyv+ICvpuOzjSUsUVUlaemZBkBk+0Tn9r\\\\\\\/npjbYSOfAqKFuBCaxczxJoCuL\\\\\\\/fOTXmPKgckPNQ1XC3bqq9kR2cvb8vsXawXWKA6hcTF1WPz9oSSvklTMviL8iHpecJ5vtMmKNhYpSLLLCTGp0YRm60PPlPAKQ1s2howcFWgVK\\\\\\\/sD54Yc4Q\\\\\\\/l1usl6ciQJN3W7GyZUKMRibf3DblU+zyjQRcVdEHVQ3CFjagfpW2\\\\\\\/U4Y9pw+cxAOUfw+HJ1GWaJIGOzLxb+y4W53H29pX1g9\\\\\\\/bfRyuV6PCbtl3GfpB8xuQ3EeBdH9xh5ykL063UCOd5bY\\\\\\\/kmpaC+4P1l12hJNSDfVKZjaUOInHyFWMybenNDWFyHNUjhMccLMdj0+GstLuA\\\\\\\/CqlogMs8IcSG9DuHmPYTQj7eoYR9W\\\\\\\/3+YPEJyhQrkJquLU+JpwbkxJeHyqrKNrtHa1bhFNjpXcmmx4mY1AMeIX37Zh2wEb3W7sO8M9gn6dqFHLZjprzSQYlTMt+bi2sCsqYJybcULfpg4pJ0JzOXfQJr9qjBC3rPuRb\\\\\\\/K5KNQgzN+BtPrNvRg9UjItVaeaN0k64Qgqbi0f5a8SBLBTKQhA+4LBqYfOzG0Z6oScl+xOdGTvq+ryTgztXV8rgiTRTmdN+2KpPEB7bgC6yLBqh2YE5bF0Wk9fflmBmudnpMJUF6BXHufRu7qU8Lv0CuFLs\\\\\\\/9Xa0bX24JmVV4n6vpqcyjPeTFnsXKhe66PgT42BYAq6IKLaWMBUnFMjLvCtPq+OHyy7s0B2P6Lfq6m37r40YG1rU4Xzap15RDo\\\\\\\/ZX1HMjfJ775MlwC3M\\\\\\\/Zuf02E1FUSswejwpgM4kJmLvLLVA4\\\\\\\/pry02VIB74gWVfG7XJC9Dtksrtb78hXhnjVE8PFHaZ0enjoeUpf+mMpypSahnXz8zeN6pvovMnibTxklKOmG5A2kuzU7v6zzjjy7R2ofCq6\\\\\\\/a1ajuD4PhzHT\\\\\\\/cvrZ5\\\\\\\/3Tj9pHYMXGIfyQ4YuHjUNdL7aajK3ovgOeJMQGUgDln\\\\\\\/ry8eJePzuS0jVZkc\\\\\\\/dO1rY6IvkQRyeTAkOeLvZIiNmwECAeXiS53ud6ig\\\\\\\/IgAXc\\\\\\\/v76OkcQrdiEk38KHExu3HqSXyT9u8JceoFy7RTJGE2bHrb5QDnbMr7UYUObBNzylTgdHkEZv4k6ldUwklMaTumWwINiZlkECsk3w8cSg+isFu7ZedLywYeNJR7kZxFeE1jfm6pWWFx5hyXIjHT2pw\\\\\\\/5rS8Ouwwf28EtkMgK\\\\\\\/2WC0gsi0cOck6grJ5D5DNn+ZQi62xLWkxWSWi96Y24AzqtqeuKNZggJgJSe\\\\\\\/n7yumbNVZXwwOP6heCESNvrzONWxtvAOkNqV7WO8c5U6F86FB3sp791aSswD6kdD8C9Og+lkbH+XG6n9kT7PAj5Id77Vi\\\\\\\/E62lk5CizD3TXXOSM9ngW+9dvd2Gp9xKzVMjl0mQVnyx4ZGy3YK77S3lI0uCWVMA50hjnVFvP3\\\\\\\/lS\\\\\\\/Tvyq\\\\\\\/qoHsFz1a1slhhb+50YoyPlFPYMXN8tZUxPkWuwUGloH5IWWsd\\\\\\\/IDxpxymdiQDKDFumUIslCjP5PIsIivWplAZ4xcJYKH32DO06Ue9XMveQsUX801KD2TN1vo5x37gmGbnK9HGQj5\\\\\\\/lW5rJ91EY26e1lz+1fTAl2qX49RQLWEr0k2Jg2CCKT4k8hf+fPlh+SflZY6fr8rJuaqz\\\\\\\/vJ0207a\\\\\\\/GNaPXEPGC9SxGoAxwam3s3Ys06EGd\\\\\\\/d6gbz5yEqhhdTMA\\\\\\\/5bVugu9QKKFQnaSzT+kltYiXwQFY5APF\\\\\\\/CQNPucao+1JS1K00rNCasR9pap3\\\\\\\/nGzfUux5wyoD4uURgZQBHxD6Ryu82mRO8EfDZJayUYO78BLmdkKlnrBJ\\\\\\\/UEJ+MwvURlFq3b4FRQUAg9hR2aW7NIlPevSQavXCH2c3ZLtwx4XwAfYgKhqpyeEF6ec\\\\\\\/0Pqs2YF\\\\\\\/7zg3Yaz2Z5bmrynmi1NsLu\\\\\\\/lF5pFzF4JOvtKWcV1Rt0ZPphuYuDGHyYe5UPO0NwrU9IwqqSj5li1+K9xB4mvacmAHUwsw+DMkYI6ZtZsep3ITggaoSP\\\\\\\/2dWB5YCPPNNB6fxopPnPkf\\\\\\\/FnEaTcaFrYvfWQPxHrhL1NB43+DB8\\\\\\\/nVqcXqGouxuYSPund7c\\\\\\\/ePI5TRa13iy7IXz3hOXMnpT4nRfr5T7aPr\\\\\\\/\\\\\\\/KIkp8wQseNKTMrdXiPwBkU06nnEXSQsoVP+qraUUdaysnfMtKM6xweulAG2a2WQ9E\\\\\\\/6Vri9BFNkyzj5+0Py\\\\\\\/\\\\\\\/pvJfyAJusUk6zmbMt\\\\\\\/N5KXb18P3RTAfRHzXK5zwSl3EId3uNS+efK33VaC380rgdn1eV70ERIjLNN2kiuQT+ubpnbKnVsLMo5Sz5CYy6jf+tBwoVzasuHQEo6p1\\\\\\\/gd+MhZjrCgK7ykZeXBpRMZYaQq4PtSXVEw+GCFgv0RWnn3TUCFYf2SkBTMai5xf0hAdLZGHVE789OPZ\\\\\\\/a6\\\\\\\/14UTnhGuhAl4uLezsfusWQCnvqm5D3URfD56SKm5kYvDE2PbMXeqgb+3gY3C0hWnPokbIHgk7DK6AsL3wlZ56\\\\\\\/OBj5VjrGElUqFEjF23dEHelaeHnVLO2vjaIwIh38sV4ZVYgakW+T35sjw5YN1nZNQ8DN9uf3SARutUeiBT9zcQww+6pMR1E8lS4mREkKrAKROTLZoaNLS2cTYAkyj72QnLDHZ7tFiuAGHu1qLg0wOoGFvswzK95JS\\\\\\\/Ldx2IjILpzb2LGbuyoy3sOFfC+Csf2vwR55rlXtZstSNQtxNqryN\\\\\\\/zodu3GiHncjGuziPi2xRb\\\\\\\/qSkdQi0P+1HH\\\\\\\/oJoZFPJGXUaPAP6Osu2x+ZbR9AfByCtgAMqWfh3ad0mkeuJWFFXm8o6wikiEsTWFDJrq5DDOMhT+zSBon0vA14B8KksvSNbXGpv3xfA90jnsIr7rbPzueerXZNASIDc1msSxaAb97IQP691OJ4FTXln+Ckwcn1cPKlTOeuq0zm3N757yEoolLqKAziWLhchoTHBgxhahkmrDBff2LLh\\\\\\\/5TL39RyVecLezrTYrUXLsqXg\\\\\\\/BfJDTFXGOemobQdri1yh7kiJDiRK5n2jcZLQPRS1xcui6htTe1WwaM8vQVZO7IGM8kPV+IeCbHMRYIlfO7SyzUHjKTLn\\\\\\\/ULqcl3M1o3l0ULMtQuY2YMOfZwsEtYrpptu2klBCLvGW0CDrqZ1CypH2HMnLXAy7mZlA5Nd\\\\\\\/BjgWyR647K1l5rsJY1vtvJzUvnCwTDXhT01N6n91DD+i+Pez4GFLekxIXaPub\\\\\\\/EwKPoziushJQ+CMYxBYhg2L\\\\\\\/D3Y1atvmgnP+2gMQp5cDMMHy0CqXmpoM+o0dOKV4\\\\\\\/DcW\\\\\\\/nEx40Z2L9j86CIB5zKgbBAbTcxh87mUBbuev8FhnCbnp5M+Fz2HScxe\\\\\\\/McK\\\\\\\/Kt86uHh52xntvrs\\\\\\\/CG+4L07Gvso4y\\\\\\\/T0dOO7Y3EsPOaEBMKVV81iwjTN+8goPGCELGgcIQ2l8YQ+f\\\\\\\/7JYDht63eyuYiK7IKCciLIU+DVgQs+DV+oFAS9JnEjU8uwKY+ogblYjuVKrwVBezkrFrSslRS9L5XF4yYBfWH6pKMc6J5RELkkhuIxcpOWsxY\\\\\\\/oyZbmC+Zo6kqtMQrgIIslRxyoNMt8WFVWVIOoFhFyB9BNGIrReKdJuYDDeiSH58PnpexR4j5SNfC7\\\\\\\/1ySMOYCT5CHt41RSKNQhM3VJTSPixvYlnPbHQhKiZZZyI0felGAe+RFqDDNDjcAgz2oHTJYmkVCyofhc8lrNdap5siyl9kDi55iPLG1vjPdAuWNBnBO+TbkkZMvw3eJxy3zR4wDEn2LXrxHK5ZhpFjm\\\\\\\/yb9Cd9Z97GsMYV5OpBElqAQwDBu4HmC3bFnOgfk4mG1s3RQw03V0exyluNV3LdK8hzEvTLU0IrtKNqsJb4Qg7VibY61UbkNqKsmyIRYyNZj06otbaS0UR9LtOfA2eznKGdqxGB\\\\\\\/bIXi1YYs154+TicRDkMJl3cUTFV6I7nET9FYywzCTYHccVlArr36GQPOXYITTK+ay+FpNn7Cj2EV1t4sn58AXELHxieLqkfmGMKrWp0lCl\\\\\\\/Bt1Eww7srkmncSuFdOzbcuMTlObs2o6EYaz45m4dIVV24N4e+d5IY1CFBd2bmMg+0rHYjSUWcuSPKdRln+Fa9Q0x9mQ3VGSC08BVRkyfFMuqPH1HozRHD5I5o7k2gmltcX\\\\\\\/x\\\\\\\/19fPRBjYbxux8X4KVpgjyE2I16ME5kXdI4JS7Cp9WzMIgHzlQbwZUTpR5URFTPt1aH2gCUgNZxrZCF8vnDB1MV0L0hX\\\\\\\/72hNW4\\\\\\\/THodvSAtTaGMFDFUAeDpG8eeH6SMpaBXtXdmWsrMABcr0BTGY7AdnT0u4QGXrtUJ7tHSJDaSOzpNgoth4PCcLR4a3QkzMVJcC8wH59v8yk23QHT\\\\\\\/BHcaF+LywJVhDgSyffecHe\\\\\\\/0Z71tKk5KETDB4amczgEnrU4DkQd8LZCjU+dnDfMG\\\\\\\/WbBsiENHfZStig5awrP86jpAc7XV2gT2bF8Ot7ECxZaPPFcRFp7jTSYd0hSxuLJI+o5dnYrsHHfyRH6g2Bis48ofqFoljM0zI6Aif7Ye98vxH3MYeRCBbxpyB41PI20i26xJaW2twKbWOq+SBYTvPv1zwRciE+kw2V7NzV\\\\\\\/2A4Fclo6bRk3QwP72KGCTMdoDhBAGm45vLS4i+Ow9UHiot1RDaF3KReotvegUpiwytzpo\\\\\\\/+PVNINDqwANl4Fx+V3fdg1pOIHbbMBtTlTp5+X7fCgY42GBeCFVmaJDy1dv1mvqPnPZp2EskT\\\\\\\/ojCaKcxXr3k\\\\\\\/BxvSaF2s9VhOa8FG0p\\\\\\\/VSl0mMMGMUOjs\\\\\\\/ArefN3EyOlz\\\\\\\/v4xj8ugaximnTmQhg4TNRlqN4+zbnduyKn3ASI17lqV7vSDKsOPUBWJoCGHn\\\\\\\/RIEy+q8t07nb0rWMC2zCiX8SMls8\\\\\\\/q8AmYTw273y6n77ueseE0SLlpUuezHIfHviU6wE6\\\\\\\/guHYpMted841Vf800LJMK1KHDKFWcs8bcBFU8sZND66VodVq8iw1XDDp1xRCRbv1sBP1rIJt5+CPbo+n39qz3AAGv0wpe3Di3I6aKiDE3uSYdyB1TLbYG3GEBtJ5O2se8pKkTop48mTebiwHUq1Y2h+366TN+97Zb1nW9Ov\\\\\\\/VlQeNgiYeNUhee\\\\\\\/w3jnP6KYC7m\\\\\\\/IFv4KknwYZ4Rin94z9kKI1rIz7W58Kwe0W\\\\\\\/04Kq65abxHc3yG1MjDf0fNUOWzgELlVqTA\\\\\\\/mfiFUtzfGfOLU\\\\\\\/qPPjUZumzl+E+\\\\\\\/wfB\\\\\\\/3vLBriFNvA\\\\\\\/fD8vyJDV93JJeTe\\\\\\\/Vdapc5BaRCMgUi\\\\\\\/EHShDsBbhvHaPFHh8585v3kG5tX7paDC+ca34o9XsEFFa2ChfdMN4bUNw9acxrQ\\\\\\\/lMaVN9tDtNfj4txFLPZoi9fpkc023U8zP+fY42RsLRoQ8LX5kzLNMTsEJ1AL07kk8\\\\\\\/nUDsz92nejfjleUYb8795RBIyM1fLom5ODX9+ZssrpAsZ8Mpg2iJsll1b4m6XvS2qL1eowxKsfdvh3lgUPh0VHhtx4GiavCJE1Mjrdp+Nq1YsPAZBdqS9315k9QQEN+jnxtnWa4vQcd7uTNLn5a4Nv8irugdTYwB0jd1G7o5DWnerc59rAZZ8kwzgmSVF\\\\\\\/JiW0PvMvVVEqabLzyv4Op+E2sSTesvbHBH\\\\\\\/Gxm\\\\\\\/yndnRH0iWT2gmdljCH3MGDHcQWmCz74FOzBEZnZlLCXI8FN7JUp0Sx5GCL+FOisN+LqVDkDk01Rf9oX3O1\\\\\\\/B+7O1TwzpEvfuQbhjZ9j\\\\\\\/mYeo9JUbh2EchEpnYRNBzPz6qF01hN6j682yjKdijxShWXvNSydIl0f6OukYreEfkvZbf8Ce6SNOU2NwNV5S52ncbitLcTtO3uxKOIiwwJeCqA8SRlzf+KWE7yejF1ZHCy4OYmY5yoaoOgA9XihKpxDWOD2O75D0nzHcSBVaD+jF5FLV3NsrbhpWtb9UckSTD+lu34qR94pgwZS\\\\\\\/WgxCwqN1Y+xvD5deq7x4y7hsMUEbheyCYcccnFu8J3arW\\\\\\\/3z22dRtPkhT+Qohgg\\\\\\\/sqIHCoRDP1TVXYsnrqUIAmJLLNAyGPxngZlkU2giCT1Pc0xWf8lp0T57t9bKbsz4uzihvHWCwpWhahpXcDgM99NLmwy0Q5A9e0RVSu6R+hCkLqZAcTnV0UcHYGJlAAw4dpZFZ+2yk9c04cJ+notBdMgR2xZlFYgVQ1ngAwiiws\\\\\\\/Y5HsFKmGuqZtvpUBDIAwjlpX+BgnVHuVUlo\\\\\\\/05eEGpv5k7JgfjzzizqYaSNL7NjaEZ+263Kf2BotIGLecwVKSW7gW\\\\\\\/u10QJmbuZa\\\\\\\/cRU2Lj0Snlg7LSRQwYQsU++cwuI\\\\\\\/nbO\\\\\\\/iNGlU2rCOqv69cAbsKx6UYKR9ZNO1J6tmu6qA47YvtraeJHj9tOSsDZQNS2dq9AvOjPy9Pg74QAzEYQfw\\\\\\\/bJfglHB3PW9SAUJ+lYFH5xftn3WdZuPliBv42OeNgU\\\\\\\/SddJ2GPyPmZtB9NyhglctwBpuTOJcBb4Z\\\\\\\/hOGQ0MqQ6TAWPoJKliw\\\\\\\/BoPJ2JHH2ib5tHoR3IICLwpGZ+DjS9OmZd8oZ1Fh1pOeRl4AdAa+KgHH27mI0ULrp2zELTCh4OaoOhQiLRgKSPkLUD1pi9V0RFLI2raZIcVssIUzfW0dqkg\\\\\\\/fvv2WzXyczNWqIhLShf4\\\\\\\/g8pYXaAJdO7rnrbcHKRPh76zWAAYvIbCWlfQr2sOz1cCiiWqdkkGOfS0VMGornyk78XLcZgvS4GxfiZIUn+a8kBsaZh2yyJWo1cp8fGRUDwnA92TFGGoddtfplpdPUbHdQqMpBHLEN0IP9rF21hvPoo636H16O+LY7HWTeHTLZGdT1j5aShUl0Qhpf3jOx4U5Zg4eKoVxdUjLP0WyIfCGdruiF4doIqWinq5ZJHXDXm4e+vPEW+qVRYSZBACr2YgeaxXDNHn\\\\\\\/GpKtybjHFfe5ii5MRxTkyzB7atqdFlOY9wFkjj3C6nQuMFMnemRDAXpZZxSX+26Ro6SkmjfcTWB\\\\\\\/pdI82zt8JaPnnEGCVp8+Qpap+vradwh3jciZIKDiTzrBsKZC02FELkJZWxZWSDwbDU7f5YFLtfGzzrmmASUT\\\\\\\/3WNvAqaTmCtCeSvkx6ISRKz8dlR0HVzP1TtyLf5tp5RqD3lbn2zmrs7udifCYKygDQPeLcojogwKeRnZTmZ77SUzfmNsYGzjOJK4wNbw3mf58CpDdj2EX+Njuw0ejloy\\\\\\\/ufQCsw0wz1nRuheCHEbiUGvj8QlKNpSDzPv4U+X\\\\\\\/Rwk4hpd+vcv4jyzyl6xMBu01XB9enuNTHeyZCnxZF5Rs0I7xQqyY2U25A\\\\\\\/ujfkmyU1pThCHRInsbvJOWIL0sKnoQ+lptDIaGzdDMkTlLi9Q2MLMYhfx6Et4T6KJKTuKTLmpwOjNzSJUeQu8IiIAjd0zhb+uKQHQc9YUIhVwMXgflc\\\\\\\/2dst36neoupVPIiY6tyIjED6v0GQSVgcP6+B8iTNJz7qnj3crmjZlCsor1VzzjR0Vj7+FuMaX6+lJIioionqOjFR1\\\\\\\/iVI01ioTXfC1L4cNwA\\\\\\\/9MMpqqsEZgAe+7sklE4Q07TOv2H4zyP55chGs36Buq2O6fAM+g80onLGBNj1\\\\\\\/mus0eOf3+fgjYiP5r0kUON\\\\\\\/OdnOIDGE99Kw8M41e6uTRBM06KoUI5Tp2OBibdfIC5D0BwbEelM0rLwCr6GUldVZXd8T1pj0\\\\\\\/FRf3AwXnrnjq0xUrFnLKV5Fha\\\\\\\/1alkBvlpOum0VqH33asr\\\\\\\/iEj7tAF6wtiVeaHyuG2PkHy2mI+WdJAnPTikdzIZGYKHr51vWZwDsII3g6XlSz2njyTJ6l85pYiUAZ34XXM0vsDJuFSb5g5AQlannfnRsC54qKvPM\\\\\\\/TiQwdYauVY5O9mmjNkUmZuWrtRxBCci8+E4hdxfnl++vMVw4OOMMEeDCoQFBDlyLFD3kvM9KmOaPYmFkne5XSHwiDYRQmnTOeg88YmQYoeqKBBpyi82DQY+KK+j7P3Frhets13lheCJQwo33pWKBKHMs53\\\\\\\/35suiVlDzDvQ0Apefc6AsgbA9WXqCdRNobi3nOCEMfQiLmllswLKqCu8h22FdcycOYyq8OgbK\\\\\\\/Orahy4kReBKupasMN81mbzINtnuKmYDbXzMZZVn\\\\\\\/jowcIGbM6LsN8PA3XMax+yUymXzLyp5uE3aHB+dTOpKByTRbUiT69QPsty9J5vOIIPPRPuWF+kDt1YGN1RbXet37akvUB3fCNZITR+6MysxwoTiHdDamevZXX8eCQYwW6WO8Iije\\\\\\\/qSmPUhVFzS8HV3AcY56eXQJrKuZnsLki4KGgYTONkURiSnXOsW6J61UuPWONhDS9miVHm3wfnoThsqLLMKpHFUuG5E3+dItCXaen1\\\\\\\/Qu6PE7WA7L9uR\\\\\\\/cSnR0wRhFXhDOL2HfbenLjiEVi7OKK1XXg1l7xAgHgYfpjEHgUrrBpdNMnpLqI6CiLAbA90xb63N2M9AzDIq89MUUI0OopTBc2pwBzXVIvpTgARJXFedtHe3HwjyqLngopAR68PF3D4bcrSbOdj7r5Y4Wa9PgzcR6qyMHaOlEMkSSdAGi4ucisec3lQZmQ1DWjQ6F2h3HBEdI1vkPrywhrZS7Vv+PN1AfjRNQrJrDCEW68zMaLzlBdsC9\\\\\\\/HbaCEL1ibO31MGvQNMwn5YC64XXJYv+T0FZMu0Gc\\\\\\\/vw7g6v+5AwTduLHrfnlNwl6gtmw06eoeJgqeAQG9fWGi0+qbKdFNZ661oZgoBU12wieTrTaf0FGosZtSghc0uBynSQiBmLodoilm33\\\\\\\/YM4AzHkCT8vR1r5UYdH7w8lNBqaPDDKW2nm1oIq+WqB59HN4nlngbi0VNXsdWaVpT1Z2JbMS26dJDD1TJI2CfTGdrSQqdwwa0fVPmL+r8Zta+GN96O4n3GGrpnqq80G5DSMhVS0hn2+224aiInbbXlHoZY+FNEYMmC2IQBJi\\\\\\\/AL2QYtJCSqIlymPLGh9ug93cmlH9UVaoKchKZ0VOHY4AWyvgNXtUeNQ7JYdtDkn0WT2frJGy10jQdO16pI+PmTGhOuuDWFmkyav0sGJC3nJFJgyF00KGQLsRtlzyI3ZhIR4pZ6RLvn7GBkZvkrkh+5fXhvi1fmAhaMxr5HDKcl8zf+ViYPGlfJTOOaBcihSGSctNVa+glRlXd3JlJ2qrNLDamNguJa0Jwrp6ynbQ1VDAvuUs0mVdZi7hCMYukFqiAGXu0dDimB8M4pJXGq1d\\\\\\\/tS4KQDcKzwlLmDrX3hiK\\\\\\\/29D1Nnbl0rEsNJrageTALRvbNZhZ5NTNbYYbNInFZL9FZSXjAalvXRV1nIrBVKQeI+l98mp6RvAV9SUjXK85DOH4o7uZTyixReGes\\\\\\\/\\\\\\\/WoCt8u\\\\\\\/jGPV5b5crFeDvQALx2nVzcEzE6jOOPEE6+xhdXQl1IXpibB25agGM94\\\\\\\/INzVz1IOGEKwLVuTWCow02BElcyQYSLSqVoQ8bdezXMSeMRPZIf+SGy34fIyPtQ56S5pPRKZtGJM3o9W9\\\\\\\/rmI1kFisxIDKtXFoFLV8YycBJO5Son4P1KN7yDHDysBIPJk4kIJCd7VORR07diP048Qlp+U\\\\\\\/KmyFoQF4CDey8wEPjqWT37sDuyBssPeuJggahoMSeWm4Cv\\\\\\\/\\\\\\\/yzZVmpOM\\\\\\\/Uf\\\\\\\/IEa3JZJxgl4E5BraOtLwGPp1MmVrXrSqQ5VyQP79cmYF9vMbdcdsqzDx\\\\\\\/U\\\\\\\/ApPQvhx5mr83c9Tiw70wqAFsgD0YBOo4+g825Ok7ptNmy5Ngh85ynZaQV44HzYdRaa8NM8c1Jr6hYU6tjNWEkKHiSw7VMVSnh1yrTVpq46LG6tspPLvOIjVXoricofKxDnlWTgumkSLapJaFRWxDrMljBzEah4\\\\\\\/Ra4HSKbZp86UE8kGWOPvzrYFt2Zxm\\\\\\\/a4BgYVbjP\\\\\\\/SNJ58y6MYdhRZU9AzDKTRQRUZbH8inHmLVIOKSBPSliEk2Gs9Y6hgrUXNFidHyoJzx90\\\\\\\/t+eOf+Oc3\\\\\\\/HTE8U1l1hzoqJVMHNkRtkWfTv4uailrYhENtw6dOzT2\\\\\\\/E0SsC1CW8uJaZR99OkVMYUsruSWYlSdpfv1HGmhqfEqRuADs6k9hsK9BAfHp9vIXHtPuEOhvwbqGug0UEB6p0Yax9pXuIY5zEduX2BcV7RHk0866WFbHGturSBRklsNktQ1dckAiI5pt0cWBtqmQ6bqlZVl8r8YjMW3E\\\\\\\/UWoxu4sA\\\\\\\/d2hJp8oCAvAvGa8yUeL3z1UT2AIr1gGH5TzhrvWrm0xHxxuP6eiSr12No3Dk2KRU\\\\\\\/du6EbhAaw7I9Cu9IfwoAGaJZGmJtXrSSP+PzwFQcvSQj0welcVz9HVZmdy7kwXr6LLvoS+nEPqG\\\\\\\/yoMT6QMA8ODVNVlHUlJxrDm4kGWfEqKA2T\\\\\\\/r4G0os8g1HX+gc6uDISkDIwXJdFgrlhuajJDFw2YuDtkhZOnYkDVI3RKeQj+ZnX5\\\\\\\/hW0I3934wcr0waMMeqX8vrmwlegyxvjZJ42r+WU\\\\\\\/5fMV1ea2bkUyriQVQY7HOy3NamXHK\\\\\\\/kqq+AqMwb2XGfTo+i1sO0oPj7\\\\\\\/xHbHGyBVh+GZekdoDOPifZ9QNz6IC8sRlgrP2PiqPqtc2BHclknKl1VY5ZOkZcwYiD+xc7kpTyvOxfStz8aZKRB5FbqKZWeLLuxkt1e7g2QL9MP04HshvUaVFRWYZ5UM+ElXnIjsMCHj+HqW7CpnXeqI8XohLpdHczg7mMDKPrmdsqY+WeM05lmATomuGzcHm1mq3PdDhCmnujg1FM1TnNSzefdw7mOPFyYgsrKtBx9L+YOON5EjXz+\\\\\\\/mbIRv9pLXD9ikQecm3iZs3gZlHGKeDMs8tXZ2ucPhkyfEz3A02uLrVdrjloGHIOFnrfPteWguZJnn\\\\\\\/0x9H8z30RUSanMhCiMlxmpeI58zLgqaOoqE8FCIk+holqG6swTt\\\\\\\/75U3+EgVe3WVE0pwLd7zGUVZK+A9p9pmUrYBXuXCZFMX+ThXLk3q0w+ZK63vFWTP6mUu0No3A6fSa3tsnp0CvV30f\\\\\\\/BMo5muz4XBbYi6gSxsO4W6SJN9ZStgMjDIpcwsuO79qc5Ps6TZhdca7xQi7R\\\\\\\/IYd2NxhguZAsTS4vuZb4CHP04tw3E0GKFDyE1wrS6pC9kg0opP22mEQX01ozoNkl44ufqBw6D23MXCzTmkD3jVBOlQSmiYFaXzoDSNkjnMVIrjGVSSmWT8kyHso\\\\\\\/r\\\\\\\/42EKl41m7jhDYHGV+q1uMfIomNMvOy6ijAbLJnBr+moGrmq1f8AHzLWHbw0Vexdkb2K1whGizwHaaa+Xg3wKqMIX3R+BMCPikWhkTsRBf5pxIVkEZY9GXnvezm4kfjOUF1LXZicVGCRkrbaic+DK2z7pZyk3Q8RMH+6+UOqJxwfPIjOoHqtk+EmxGQ8P3gB0TgznB+cFnV1HbTyG2BA5GWhZWVSck\\\\\\\/qsOZf6m+DTWoFhjSHr9Ux3WjHF+F3YQA1b7qH2mjRPD5rAiqPcGrc72tGiBjXTBK3eSOG4k8rsNR1\\\\\\\/Vi1Mtcx9rtxuraNsVnFSwTmAk3DjbMbc84FHTQDQO4p\\\\\\\/LEmEQJ+PzyWxZaEb7xdw3tbCsHCrYEf2rxEWNReeo2ZPatQ0oIMJyiWpqmGZDKvrh2Ymxm74Bi4FZ1Jrb5HvnwFACGObJHenVce5epq5kreWeTQCqOeqSlptoG8KLDOR997Mx6S7+shvicbcsMHgUqaedtRypCP\\\\\\\/TRGPMxEZ\\\\\\\/PO4NQNmQUpNSgxZ1LJU0ts7GpWtWlGJcIPHyWpG8ZUHXdUejftLoVmK7ybs8Ys1p3XomyV9DKLNrpByd2ghhj6J8wK5xCjrUAmzZroP1q5frBYW\\\\\\\/FrlZ0U5y0kaBfcnLlIS+Ijq06+XQh1rezSMBgwLfXvUnwzY37JwUWmvbki7mPQiZL6\\\\\\\/hOfdqpNTxJ6\\\\\\\/bYEsgAnXofa+8yjIn+wXIYOyLW\\\\\\\/FE61aulmhsA0yLkkGl8OIkMiVwmvH9o2dNzXi5zPXWEcSjQXDlGFfd+huKiluQCjpIha+VKGEkMhLcbCcjOfq3mGAQjS9Gllbf\\\\\\\/\\\\\\\/1c6ikvwR0kWygYH\\\\\\\/yd54uSbKpePWckC4f7RIguCpniAwWZ90wu9WSaMml6pHXRqummKpIoTy+8CXb0ZZPnm4FL6lgeYWKiGN4YmC\\\\\\\/JQU7rJKvtRd7eW\\\\\\\/cPrrSx9kKEwIuuNz3sXQb49sRidk6TfF8zNbWMvwNoHkQ9l9dpvb4xXmI1J6g2+ljuQ9gJjCmGwlcqz4NeDLJMACf33hU7lPby2U1Fjbt4YCeRic4u0D99jUig1fkeCk502+XZuMERqlzvmdj37vulA\\\\\\\/UKXm0nYovtkqEDWQV17Yl1bvtgOJT6qjGE3yb1\\\\\\\/cEcOW1qvHhB3qg1aMHaxN8RwYVjjIR6j2\\\\\\\/1PbJp94U3xMHo58JvU0Xt+rxYXaH9H\\\\\\\/8ktezh\\\\\\\/hS7YYuUYQBwhdM9Ox4pzcnOnQ6l2aEXPqaPxt27Gu3thVR+GMMfnz5N8wAaGJSJQrM5HKx1uz2BgQgtSAGQeKY+Ay46Mbca2pIpV+Ubo7tVHpMPPo3ywR+qYJ0GyD0DULsVcUdknsw9IjycYFe86WO3PB24cCtxRZ+HW3u7mDh031oA9M94RA3JeiFBb5a2DVimqxRHAQqjeq89S9kKKM8PMdzsHwCQg5MJj4E\\\\\\\/YFWY86jObk6VEFiO36lJUyHPXTqooUybQ0QAbkisWktShOEhEGm+myODen6lJxc9Sw6On0Gfvix6yffitp1+IQgvJdxiJdmeAV+5krxh7PTfdqyzfzdpDf\\\\\\\/ecAWKFJT+vte1VFlKS+6JQta2lG19zovqcFUDE0KQkRVA\\\\\\\/se2UTlv6oegJ7hye5HWFkLoNrBqLY\\\\\\\/mOqYs6vjNUPVMptCUz8AhMLzPe4LJc31WKVI\\\\\\\/UUFFts+rLtEXyza7cW89xsKjZtwRO2nKWAP3vONBFQnibydKDPuXxu3vxL26ws1eY\\\\\\\/AI7A3K4SUQbT5r6iaAjzlgq3CPKv9yXjToeXAeO2Uha8mGS+9zPj90BZlx6YYS2qLU4yoFe1x5gXO9ODDfGNHFDgrewtAB2XqUUzx3p\\\\\\\/1jSsGXgCEoNtPqWmeUCdyA8X14eYgO0ZRSXazmUrHNMAlDghhGzmfCZ1LGH3sAhEFDHssNgJvJrklAytwZQg6hKz2cyHVCLPtwX4LlFcTKBKenfsgL+dS17q7U5qsJ1pDxfmDN\\\\\\\/j2mV2KUHPLQAKyoxWnlpo4FOHoTuWH5mRmZ3Dakfwm3\\\\\\\/h171v532UXNcPNoE0JGv5wkpDu3KXfvHGhsA6DpKwLIOqG8h\\\\\\\/WqI6TZaugWpATFkC0bnS\\\\\\\/m9pC\\\\\\\/3UDG8G6HeZpLf9hYLWA2uMpZ3x4ufeB07Cd5jEFiZ8OXJO1nkraie9eUyPmeFkEYf7hkKIeU4bFjQDLN\\\\\\\/Rz9TuKB7kT3+0rs8yri21AdvdeozRm1nwn+cxa+ISaMvMM8m9cI8yajygZtJATpR9bc3QNV8i9DaG0cb6TAbMGrk3+SBuPD9iKa2RId3CAKHTS1gJ8xXpluaPc6PRCaq\\\\\\\/Sq8t4Nkx8iM8BVTNFQBzO2YXsVeNkQVBUkgp+NQondty6o7Ycf8eNmF5oSKzWJ0gB4o83G\\\\\\\/m70F550tCBoun9a\\\\\\\/pe9ccf\\\\\\\/AJAa1MDw4I71UfZsAR2T8ItdZ\\\\\\\/9lDgspmbMcGcn\\\\\\\/JHQlaR81PhCrmaTKvAbSGZVU\\\\\\\/6TqkRvyCBYBvU\\\\\\\/TCVy\\\\\\\/TzyagmC666xKeXBdG4R2TXoX7JuLR2d4IqHThNbkUpgAlRzPKgsyhSUkxdKeZ+UN8B3DtCLuMKNPvqZQhAlFLMxk2mwgS2IKYFKbgkwAgoCvMMpMGOOOapCfuOM0fa6UNa4c+fu8mjH2FOXA2u8fFPND5VG2U8V1zopO1VH4hjVY9HxX1HmIfA1YR0qb1OHBII2u1i+gAmnUs+qIPUDsyZOXOgbUDCBBysLitNyNfSviZvesaUalm2R7jvKLfd2V\\\\\\\/HNTtMT7Xmx2tSf5TFEvhWspxYeLfr1fJsME2IFIwhnVxVr2zulpnbye0IY+Rv2xvEeXjvtDofcuEnwCiNpakXMIyUVL1GKgbe3vFFQJu9agy0oDqmVMq2srpauPCPjSDzUYYAHtjy8Y2tPjoXE0kGHtHHWfz9hpySpL28j10dqEhR4otgzvja9pKzepEAFBPYHK7WidOtwrCNa3xqfKF4uVLhDH\\\\\\\/mh9AgUctN5D8dOucnC6qNvO9fGuQof77U423XP0MPgQPOPHcbFyT5ABLMJYNKog\\\\\\\/E4WfrRxLiU2I49sNxO+3sAK\\\\\\\/9ltwoZu22Jg3kxzH+fux2Om5wArZmXzrbfRiTRPz6u4fEmAm63enO5+k30DUFRyDLLIXg+HNHG8mxaIa5xyReeTJVd6O5v\\\\\\\/H0lbaRUiafpq5qiIynQsXGxbg4+Zxm10TRT4rD8hHlHtahWMvjP07cW2h\\\\\\\/6waGB2GWSbb\\\\\\\/f77G2lplfaO4FHEna3YYzl91uEbC8MahAz8ZxgOPF26hpb+lfRzzdxeqexpThThFxy6zZoyZG14I0tKpk1GOOH5xG\\\\\\\/xDhs\\\\\\\/WeJGHJmUce3SSXVhkjQSbMORDV1J9ucYuWne4k4Y3C3wDtJGCnLwr8yDGgb0ow6wXbjnjjn5thN6qv\\\\\\\/UNfST77BECDqSZXkiBS0F4hhbJNXSJfUdw0mfnd+9iMJ6plVLF\\\\\\\/MeWp\\\\\\\/0cAQ+k9PhS57P9Nbpd9wq4vMRc3KC1i80kt+YOAp\\\\\\\/s\\\\\\\/bWxxkW9WdsvSfFUQa9tyPQM3hWTQVS6+HeqDriK8ngMcmJSgmXeShDgPAHjgRAgA1Vkd\\\\\\\/ONObrQ+aWeaZWjRGQCIaYKPNO2iWhXNcOFdcoGHFmdTG4fVh7eRCV+gPdPtErxUDrK0oVNAm6DqV75W1Yd\\\\\\\/K7XcP9AbA7+Ibmb40S2f1sgR1yqUt+pESzG+P9QbfBpTU1qmaGpPF1+F7OWoOW1RYkvoHa3a8xsK9Ch9ormrU1ufP9jhjoCR1QcTaeJw\\\\\\\/HmJruAhgMGP6wF2kPyi6ViTG\\\\\\\/I79oEgs7+M8GxeMCf2NER\\\\\\\/jO3W3hKDgCSvKR4uCpn\\\\\\\/xFNp1JfE7p6WVH9zHQXalbAi5Vr+qB\\\\\\\/Lc89TPNuDZQ2DtBvjniF3PINeJpapZ0w2LNH2c2dFDDrfIjVSNyEG+HiRMDDFamFpGk+cKcu9rDJXqMT5wqr+E2NtlaVc+wpZyswMme+ZAEQ8Q6CF1cfCn4V5PTKFXsbjhBM1iw79EGynFGfa5stYOTT3bxJtRLFdbK1zY\\\\\\\/COswW82zmP1KOIu2RpuQPooiMe0fP9zuEh5j3jGc0YUtEjLpcwv99Q1UaZvX6NEF3Q7Nm+GyhNE9\\\\\\\/CJuLu3nSXABYoN7OH\\\\\\\/Uky+I7TFiT8nz5814Pwa53nsyD3k7suvnWhVj7YvDereXNQP3p1XyYQxAyyA8x0AbEnSSQNn4K1yRXeZ\\\\\\\/yw4rC1wRbl75utqFvER2htFHEdYhltew66VbRX9Y\\\\\\\/8eaLY0FvHcEL0luKXQBZGic+Dj58tCWIKSXgU2IiS8XXtTQKtiy4bcmqa6tLiXvtThPK9NYZfTITndSzz53wo9FkksHQF2fwqI0am+g3go4sld8CkgD4Iml77z+s9nnU70h6X\\\\\\\/UGe3Pux0Dug9Z8MCYhvykr2mImZU5tXurgHV\\\\\\\/FwBUJL7llkG0GbFWqqy18FoPI1ZokUyi+aTy\\\\\\\/rK1uumkrSPvRXV9tL5RZnOS9jSs4LEIM9IaYkryOhIvBtTwCCJ0CgOfyuKAV1L8IVBi8AUBz3npFn8K2Ipe6LCZD7FrUc8hc2jX8z5xHjJtg3bXKK4l0g8DQ\\\\\\\/cQirrB0ZEC\\\\\\\/lkqJ62XoSTeXTsUkC\\\\\\\/7SgBRHPdQEx9jEQmRMwgomyOqyTPQ5bylKjGMkReWxtN6QGR2GICkM2HN8TP7bdBhApE4R+NHrPl2rJo8YI6KCvP6TXcvfdT\\\\\\\/y0Tze2G5hKKVSbWGTuzZZrv8wExgufOghLGXu7kqcXLO\\\\\\\/xo+katyjMcVANS9Pnx735+DTx\\\\\\\/3g0BYNOWTGbnhkoWbjBTpZoWw1+u4yDzdETEiAZsKil\\\\\\\/5qMbHKKa+wrl98XouOJ6nooiH99B+lZ4q\\\\\\\/UDUo0IK2\\\\\\\/9hDBKF+v7yEFroomoXTG\\\\\\\/8L1jjMJy4+6+604hSX08JAecbsM48jy3Vn4mNVVvcPbeS+gGlYTuCtvJnxed2xyixb3\\\\\\\/gVXN3GYawvHwH2QeL9b76XWjoDfl7a8Xh34a1Y+39PKrd3TmReRc9oc9pnuKcQDUXj3spULvNVgnTyTyz2uhm580YnYcCvApSGTWGh9\\\\\\\/L9LgdAPmVL0tyB1gkCnckfAr4Ynsy1Hx4jkUUn6Qaf8dX4NoaP6\\\\\\\/aJfdAQNVqapLBS323R1Jo7GLgG3SktbfwU2V7ksDwSoNZl2wXk2f6yJDlRIH3z5rdVTYDiIq+dSNtMx6\\\\\\\/rJopyc\\\\\\\/+e1F4Qj+xL8O\\\\\\\/b3XvNnXtwofuRyRX1lOl1KdXYbWIcDfr4AImeemVGQt8dImg3nbAiuv+stuZ+NjSMlA3zVFAosENVqJQn+KXdulNlIpbD3MOZ0rKFzXPoKRgHdYZlkmSOCmGijFtwibdkb44tOXkCdKYZK5oXhj0lsxgqGV+0GMVii6HAiKqt+RoQb5Ny3cIo6oPu2Bg1jBrS7ORrBKAWDiav6UAv9dl+daVHfPfGrYcnbafNiNBoOlj6y6\\\\\\\/GTzhYUH3U0bulaOxlnolcXk\\\\\\\/KI55zKvGXaAyvpNegLJZ\\\\\\\/nuP5UP8ykrHCbCHZWuLtUzK4fwu5iDtMoWJ8Xjqn4CK00U3hExzWUPh0x\\\\\\\/OKTDl39j9ZRRv6t9DZ6AETsIGRmGoGecw7EfUABa4t4sMSpVTqK2aol1qZXKEjFafmqZDbxxupfxG30Tz4DYHRS38f\\\\\\\/nCdEx03wis6rcsMikQfExtvjIkHRsklKbp6U3\\\\\\\/yt\\\\\\\/GzUzFRhgs9qfo\\\\\\\/wJE8L6v7DmEKXoD2\\\\\\\/99TVuB7BzevsB2QzPjT7vmKcIzgi\\\\\\\/3HdKXuLS3Rdqwi7oeeJeknDb5Na9uYPEb6o+fmZl5t4q6u4zal9p7M+BjXs225KiO7BZYb0b3t1pM71dp9RbOeqXWsD9aI17dyt3Ppq36D4GdJCGPzPlxtQU\\\\\\\/F6j51KYKQnDAEHwYfYPa3aB\\\\\\\/w90tdSe\\\\\\\/KTKnl2F8d8+cOv2lOPqiQYQobTUuLfpEs43qZxiYvj3mL6fnKZTSqMuwpxeyTtiQMQb9O2C\\\\\\\/W1cGjlPcJhdRVaeS91B3aFHsXPTL5M5rYtRiFVS\\\\\\\/utScD2C\\\\\\\/NC9yvz2+Q1aaEtTfnQtg0\\\\\\\/fylW\\\\\\\/0UeIChg1Zb7LNOB6pn4RtA7eQvdJA32\\\\\\\/6r4ivohgqUJ7PzsOSqtd\\\\\\\/vF5hiuxoQL0I2A7Rj3VWjSmbb1JtNZcw9bOC3E7L3zGlulCAg8KMtSo4YAQp5lNtjCdITfIOkjnNV9pXqZSGTDDn4Tl14r3hWDnXeLpy6rCifa4Lz3bBB1kzTZVIOT79QURaXSq94fOM8erYGqqsEKEak+qbLYBTMkOVQ\\\\\\\/XxSTYBcp+SmLHKcsNsDdy7mMi6J79KBDvY5iTYQJ6rzwvtktLBtf0D7mEVIJTA3M0oDVHhxTBp\\\\\\\/ge5Hz+0IIf6GATJizM3GqD\\\\\\\/u+kBEMAGwLtfyW4SYe8WupPojKSEq\\\\\\\/+nQj8ej\\\\\\\/ik5UW+DRMDHZntJfqSV+JSX5NpjAi6Q2lYNl8B+EcjFa4urijyjJwxQnse2m70PV+X2ZPbeCB\\\\\\\/tiN1H7mk7Jy3olrmnGR3ioeumotBcZv9VOp3NwHvTZpkL951jXt9qfC\\\\\\\/D7ZscoaEPLpNlo0cMcot\\\\\\\/d8jqH06Bbh3LShB\\\\\\\/pUaj5UYT54VMTF55rtGz8STJpDCTtkQ\\\\\\\/tNshJjg1RSQ2D9\\\\\\\/HOLvB6rYLS8aYydR8+18hhz1YVLEYeAcNnuhvSLgw4dtBctqmHr0liM6EnDTqJuusX5kLgXd\\\\\\\/WaKG1wb0ahowG4c5Yoc6jtShm5gPK3iyGieRx97ea\\\\\\\/4CHkYBqmyz5\\\\\\\/TPlLirAhMMeNEqWxcZOSd9VE8xDabkd\\\\\\\/ivnLMVu10IB8\\\\\\\/P0IRuS2lywmM5rezTpSyz9aziQ0jPshUa8xV1dxsGiwn\\\\\\\/pVLmTNV0hGBx2Z2pb0avmbpLAsGZsjf9dzZF3lo07SSJ6q8nU+pm\\\\\\\/NjBPO7Y9BYWmSwpzf2d+zb7U6ZY5jNbMWaRq2oKcJO3z1rUGW\\\\\\\/Lyv3822VIAwM+jO5zo7xsn56uhS7B+ybioWZNn6eLrrI1vpUbO3sacF+0agOo7B+EY+subJ\\\\\\\/p25XrWD94TFBD1i5+YcAuIBnUBNQ99JnuzgzI1R8SWTvEk6IO1G8eWsfuwRGpzEMVmmGyg1yo\\\\\\\/CJY3xmplXVm9h121eBoMfLtEIenawxPf2kIE9rwF6zmA26ZsSMw8O3dtYfnko17ZnQAR2GOv+ZmS67Adeog2bJuqlfWapjOM+PwgC9y2ObiveNdtcedEGMysERgy9HyBdJNB76e5x5QZHSWDSDYuF9cfRi1pwJafAPP\\\\\\\/8dAjSkn1fIiMBk1djbPRyfqElBheyPawxjXx8zK7jpm96LImMSiTRH89ex0+zbqZ1f+0VXpG1LPQ2x5GUNCkIg5uWY3A+\\\\\\\/qa+k1f\\\\\\\/7FLBhKHa6jVTJUUbXdviDkFABghaWhm4XJmw\\\\\\\/4Fw9t4tcVJxouuirL5mcu7nQrSzP65EWtAdeTTVUTFrLrBoOiFyjTu9GqQ0SHDrb+wxUbT+DlttZT3Iz7xyEacqFuvifAxePdJ80lzh8digoWsffZN9WDxeHSf\\\\\\\/yjcEqYM0hT828XxBWodvoOOmKljX+OqAiQkvBeAgYPekZcoesmo+EP3MSXZNzl1FZ0CMhOl1H8Lk3+fXkrqvK5TDN\\\\\\\/bqfBHGPLEZ6fInz\\\\\\\/1uAa6GgK1qFQ37V9P75DnqUZ5atvyfVOrQP5HQWZJdKDvZUPFPer1l\\\\\\\/yeT+Kck6AKzFcGnJifYwCM1md8Lz4EDJb18gfwHg\\\\\\\/uz5sAmbf8H\\\\\\\/0O5jQ70ygcI2pdCrTZ4ggeKe0SBJT3Q92dQGauLbcBrvJ9JbjcKRPq8I6vhLJ9mlbqQzTA4qWbAWq9sYilhXtwlAiM1SE6HCmjfPy3cnzFTq\\\\\\\/GjnXGQjydzGgA00UOEr9kuPQInKkiBAmbv9rOnXGqjsgrIepPq0syvTn1xuftyg3abigxuaWyKjL6WQUcDtVS3+w5Pr04LK5JrdXOjb23UUwVHl9BxVb2wcjIKpsZVGzI\\\\\\\/elDWjanbtgoQNSJ+ts+Zd6I\\\\\\\/UticyyX+uhCn6njffqia+AKSVwYGjPapacI4y\\\\\\\/5LfifTUhphTwWEabWy2zsdqD0KPhtOCEzBEu5z3kdSfj+EA6aCr8Zih6WKtHuVqOJeHqXdkbtOhEtDnoeYOiwSXavgE\\\\\\\/WA+WKNR6NKXCenbXt8JMAqDeYRiN4fCT7+9n3QhGHV19v+N3EZGOvbjetsU0k\\\\\\\/D0KGNsAK3kwSACXQ\\\\\\\/DuM8qrFSHECuU3McCLhzmwjmWR+htkjlG9Y23gEhnppOvLxfZqkncqczZ1nFlGxe4Gnq1kbE6NS8jH7GYRuVhe0ExgjtZSKt7\\\\\\\/y9yk262DtmZPHfJCNS9GipnHbnd+HM2J3IKEOwWtJl3tHYovwLLPzwY90qHg2hKskNAdee8kPyVjfheN9C9fpTjOnr2ZfMFepVg8lPpn0GbSmPjfcNgDzRMnPlY1u2HVmUGMSCw\\\\\\\/1xrhumIqga1lGpAc5T1+LbkKf5AAX54aCD3N6sholmfe1jw93i6xVLv0M9e5gS6gzWHmHUI17B8fONG3C4U9j4cZECdQ+wW2QNY\\\\\\\/cpN7TidsoExVbZeTFs8yRUv9+5wn1gbFBE+CqELj2Eqn\\\\\\\/qYC1CivqIy5L5Gw3X3PZdKb54aEwB01TCqM8kkOiLVsETK0A0lExmHHgrLMVqYuMnG6s3nvANbnFs+Ko7YoAfSfs0heZAQfd92GwsyfFfQdaI31ZojiXV7MY8Hc6USlGdEJ+4n99Pk2zVJKK5PplafJeBgR6iHfv+zDB5CF8zNvs14xw\\\\\\\/YM2ZWJzjhrZhRhf0YWL9fLo5Hu2e1HvIs5pNj8wXJSFo1iCN\\\\\\\/z3JSaVSKfabOYTcXC6MnvuXA83ZjTy9iNQz5l0fAtr0pJcR5J5VBOTZKPbCtfBY6rkrzhSC4D6utsapvCHOwOkopJMUP6QgYX0LEKuWK0wOPjM2s8v\\\\\\\/MbehDsqEAXdgRPYITA1WcxH28Fe6PpzuuYDMIQ1BtEAVRcBn\\\\\\\/ZbGSzMJKa7zCoPnzVtOx8ha1JW+ABMYeb9xYOwhF4Lm\\\\\\\/ZW55\\\\\\\/2fT6zUt7T3Q7uMfkI\\\\\\\/L348yAcJGD133VZwWO2hlffSH\\\\\\\/LGvbCnA\\\\\\\/GzOcYGZ4D5v+YzUxYGEarsmYHmWPSm7Imy7y+Li+iH5n4Lk+Y3IlYiMW9VZ\\\\\\\/Y9+B2p3h0ve32a8t+COVE6vbObvo3dq6PflxktXDlS+bYj9wSKvqjYlCI5\\\\\\\/4cNqBweQ02GhyE4Jb39WJJdtMRInU7UKoRIex1iAulMcV4Ex5n\\\\\\\/Pqc6s7VJbN0zWEW6NWSSLKpPcF1G4Vw0FidzbO7f9Yyko\\\\\\\/jkx5DS9K7mOjfPxmIXaUIPNhj9ZlchKw5HK4zhTm+HoEDVlXK3xILFQiGPn83HfQlYQaZRMunYZVBK+3tKxhoSPaIoedjF1NBKwSGYkiY2QaV\\\\\\\/RJrIer\\\\\\\/qcMcVBxHB9y53XYTKC0YGeMgwlqrY0XWoP7REkpseIKq17o4rk\\\\\\\/KVLl+VY6Wk8Ftmac1crvdO6HgrtQ79zurriN9oxYCfUXe+oU7v+Urx4SrC0izs4iK7DxMOS2hdfFl35v4gbGoO5\\\\\\\/oLTs90RIx8mPWbcYxqGB9b11JVfO09Kz3jSI818CylQzP1b2yl0LkeOj0YWMXcdWqaWqdO9bfV3NkB8BR1ku8yi4Zg8Es6S+umq+Ont7puTGywvkiJzvxNdWe55otmN5h8bMUiR\\\\\\\/LjsbY383DycgLH47ShcpGAjN6yBUHF1P\\\\\\\/WPm2LqdqDvDHCMTBa51giDKqKko0YX3gNDq\\\\\\\/lkvlYgyEx7DTelEeKOmEjVNbFd398cHhg2YJgtsnGlw7jgwUv3ukoSjNrhjrVYKbf974jRUUn2NO1N5g1RHWmDBPhNXLYvYPq\\\\\\\/MoVbCrbK8r2r2+hfIdoCYoEN+B5KdYT0eQsZkbQLy7X5i8itb9Jc7qJpofdDY4FVeVMlPEfVgqwrlzOMrc43S6Xp7PwMiFFRTYey2Riaf01Tv8cUSuE6XJFBdmJrrEQd0u2x7JUnMtXJruE9CHCy6V7qwuk9ar8Soy2SwwRr7qChPknZ22CPpwRZJUR9SRz+m36OVpixujeHqCke4i4iJ167Bb9nuAdCDVRZeYVLRH+cjqrd\\\\\\\/YBY7m4laTX3RyK4MAjAty7S70QsM7I8NZ5mnEXPk01ISNApAd99D6NMUg18RSOVTyHExZ2XMmVdwFnMGIPh\\\\\\\/N\\\\\\\/KMV\\\\\\\/Hew+RI9PQvKi6VWy6kudbQ0dGjE1gT\\\\\\\/fuT5kB7Gd1SlDvhKOTzwco\\\\\\\/yMsfEK53hBsNG+HrFCViNyKlFSYGGFXkOkPkKdfR2Fkd4b\\\\\\\/DErtC\\\\\\\/+NvXv6rzTi3GFHG6mCP\\\\\\\/2elQ54Qp1Ubdi1DMW4kSfAwp18BUf9XoT\\\\\\\/\\\\\\\/TIvtOMIy3mO1oWIa25TLZw\\\\\\\/sEpGTDaD9bqONBU1v9ad06V71sdShlCAn2mOj3cjUoust2ZyaWyzt5K4aLkxZmwN4jcThcYQw60YEkfATV77J6w0+mFLcirHnHOiK+eH4EJjC6c\\\\\\\/rCNPg5skOE4eS0oFlcallR5tEg3gsz6XjY02SA\\\\\\\/ajDimUVnxyOdoyp+cme8i42+0tf1iZlGscqe3pURKRTxokC5yvx6bY8uJ\\\\\\\/OHNJvXgYgRp\\\\\\\/171Xw5VOkZtmLEi8nP35FwK3e9yxDr0FLV64rJKO8MMDFUz2T+TXEJlsF4PyMcD+MBMm0X0mifYF9jW6tWeiNXSHHQf4oHFXp\\\\\\\/UMdgZys6f2TS2cleT\\\\\\\/MIEHvaylp7SgvEO78k35FN6JCwLkCvWeIgeXmuHiH6X\\\\\\\/OoiH05znWGeX0j\\\\\\\/68avCpXvVM+7JGnKMSJavPd8TiThp\\\\\\\/hgsqjXI6xtazK6vSkOtXyjWrbjV3Gx5kXDZc+SdkGx1AFHMOSpLYtblTgOKXFqqjEEFdkdr+ve2Bvqh6LkoVNLCkog+KXbuwW+athjm2PcOcWOCWmOJ7+wjH\\\\\\\/PiVlNyfzNKd225R9jiMdRomBgEAo90+T4uxL6mmcu4Vhwpa\\\\\\\/okmbMTlSuT7P1h3kkHh0PSveKE1Dy5mfdS45DT5MjeeLLS0vfA70vpHSsClquw+uFsa7rhH9UwAdSZTvnAKd5OWsUSoamhHUd1PstoqEE00Vdt1SJh2aHz2C+Fq5jw0V3jHbSvuQl39LKMdYKD6O4O8LSPwHmGEOnPbeQmKRtZ14Pw02Dx5TKlIg037W+99laKrdSFyi6K8QdW+pQooSJFmuzZFesYNWXGRzn1FbgS5CdqkwKV25Y66egMTMYCRof\\\\\\\/3jPOpUIQKnmCDBqoq2RlmzmJdjG6kT4CifaygNqGs5rhoXHqbUU+UZj\\\\\\\/h9JmXyYl2JpKzlgwmIP4IUaIJP0D88j6T\\\\\\\/dcOe3I6PLs0F5G6qklOP23q4eYMbXabuBE1bV8iWeAH577jkvwE86BdJnZKlC6wB9I2i9P9mIP613id6dmo2CM\\\\\\\/YyKmBUY+cgBIi73Ms0j4HHM0InaYy+tGkAMXOS\\\\\\\/K9kEsqLsGS\\\\\\\/Mi6X0rKbEdzp1dAUgV+rG2QX+OJu\\\\\\\/FWWN4AjI\\\\\\\/32N0FHvkIFzMngFzAl9F9onk6zZ3O+dqLRQUaKOuWH59ugGMX02N4rLIQArV3zzD\\\\\\\/MMiZAZatFVGJgg+PfmNdQc8zdnhyf0WpeDPVDyVQZncsZB6g2+upKZiLLb2w7MY+eSq2bVrTXLfn6+kXt8QX9CTW8NEckfOKQ80RzNWSgZLebcnjDsuPHXn5FM+2L11XPfVgPe4f+SWRt1lsuiq9AYrk3AMnQd2panIV9yGp\\\\\\\/1MHMy27ZKh6PFyDkK0g6Lr4A7bPws5PzxRNFR7aKYvQD+WSDJ+CIWCLpbvK+nIq78Mx10y60JKP72MGUkJOpAlcxGRU1SJcj5sIB6m7KWtM4eKR9Bcv6vXb3eE0wvPf7dfy8NuER1vVEXcINeQTGL0el0SwZB35nBJxt0MFw\\\\\\\/l\\\\\\\/GHgnUUvAlzBArR9THqLRg9WIKmzSt5F9sFlrupd31tYvbWLslMCRDapSwYfdTCl79COui17hlrR4y4+3\\\\\\\/VOaaAh7TIWl\\\\\\\/Ivx9twTSBrw6CYQwAwDOEZlctWx9dBOXyx6Rw3MnauzDQ+peq9IluFfuHF2fUlM4xmIaAmzY0B7sfNwvXngyaviDvvRqhhqPD2qQ1hfTenqrZwolLEkkz4W5jrbCU2VelTKrTXViOr7QC\\\\\\\/rJ7wSH9UCBnPGcOoj1DSdySUS9UT9YHQAxwKIkGirGiNBdAOvsjg2GWaQJhJ9jZsCp+QRcZMhetEWOiKnkWhTEOTx9O982Sqw03unr9P1z6UuxgmXmsulJau60Av6LUaUQXzuzwcHEjU+7G5n2nN2YwyQTyZxhubKxPy4sDMRxoUsCC6SflUwRP3GvosA1fcn8eWy3F7dmg54cFPcD17Ln2\\\\\\\/ZEJsQb3ZzAQtN2ozKcx7EXF2MNFmwo3boccPdU+jR9wwx8bA5kHfzVLiVLiiUpi5jq7+4Jbqofn1Q9Gf6cqdUiwoaAa5EEqUgoHQii2Fle7uXNQiZFl5cfZxEZG3bzQ7pEgdXeDIZHOum8oX7v6E2JyqMtU47rNHKdL+LStZE50r1kARXLsxEqGhmFAbOPcT43KtUAdDYetqyNVlq1ru2NuRrLn1fWNrj24RrOLvA2J8L9e9cD7q5vbRTkIpMePIiiH+tB+3z5q4JyDYHYQdMTSVD5zV\\\\\\\/brjcBJ0AeCZvH8t+lw\\\\\\\/vNDJ86k8B8OaGu1zP\\\\\\\/0jqTwlXuIe+zK\\\\\\\/PSfQZ7XBkYUFX9OlQPLHAd85naBTAbsRIKlyx9KvG4cLl4ZVg1WuRQzn++Wuf8g3NuRKPK6a7yB7ZOlMRLrcSC0pp0eYdgvpnWCHqtj4bcQ+1alcMm3AdVqHJldDH1X+obUetqqxnV\\\\\\\/8PrIetpJ1jzb3afo1rVt2Ix1PPB0+FPbiVdAN1QV4\\\\\\\/jdGabgwCWzu9CAQO5Jy3AlaFz5I5jQ5CxKB11SqgXO96YUkTwb2bR4Ww+RZyiT8J22WX3crgb5ldGrhhlY47gqieKm8zqVX1\\\\\\\/haP7hxWOUQzzuvKnVBzJd1mlWNNkBMmSrqw9F8G2YjTpZWMokoKUZ8KMK4pyo+fmVfXynMCHg\\\\\\\/4FYmXMCzBBW3bQxtgoLGuNPR9MdfrI0ckOsUFxTraVyQfyVuBRAOASvxjNhwUVkmGJkQdk++oKnX9tJWVfxEPXfJ0J+IxNl69BBIqNM4S9kTLqM4Yaf6iMqkNuSNHEmSZk5x1wJi84qy2tqyKlN8FlOXOf6CrE8Ss902gGNwiVFSLEJ5bjE9UgZhAfjvXCg6yIzpASngNviGBPfbcu8IcK4a9YOlwj2gDFumKdalMX8G6IcBvc9v7v6UGcKpgGTMJDwdcfDAjh7gwRQDxO6BPbxfKc4YuRC+B9wfm6vbuAeO07qW+nvzxlx\\\\\\\/nHgk9o2nFS7TTchEdbbaa6jc4yPt7lFlmkx4oKHuH5TBMDKOUiFTUKxKAf6Wkxf+xH64M9S+bJY1RnxZ0VGUMOj1siyHA0yYhSPr9SuNoRiO1G4LwDpSIIlK3hPhBx8RIKgA\\\\\\\/4dPNXgAA8BMIt8v13Oj45Xc4epBtCJb8WA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Pfrqsdvx8FEvwWEf5Cd\\\\\\\/0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022096e1376363ec70374a1eafaf51cf6b1c9567729cf0a3c9cd05b3b4a071e3200\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KUE8qMIQKDeiYAAGPo36\\\\\\\/yoaW+OJKujefD4zg8FBnVuD2BtWpiVkZg8czFgPX616MpNq2Vw8\\\\\\\/gdyV6TmGDqj03IK0q6WKqKLFhwR9dNqBWnqbx+yK42kGZHhdzTxVTOoVNuPwaU5DF7isrnL2RBv2HMXxxL0QNb4zxAGpiCgsOp3gTSpGDoIKTcjkVMe\\\\\\\/C6gK8u22ckMKSqsfqhjQVjoEHSML7exVptmTaGtUnE\\\\\\\/0dX7r6d4b\\\\\\\/Ta0Ps\\\\\\\/wxZrasCLuZA5sPTQnASlt4A2SKaSjcgwY8zRLwNe89PQbs\\\\\\\/NA8AUfGMKuypIcXKZK50gS2Kl1Oxl9WeR9oVHF4Yid78nqN4om2V7xqDoJrKpZ6f3K9xybifgyFyhbTYsq03GSV51df7DpXA0vB8e71UGWudqaN1qCv9pG0iVtUr+lsxxhbSNlSMkFlvv3LBkgBe3QPgfbivJsy\\\\\\\/Sav01lCnijqSGcPGEp0as5sA6SLa4cUqqEeb6bl5vYgqGN8o3zJTLXSQqthVMyA1+KbEnrAgjhoNmv4R0rGDvCmc3NO+s+4UMSmVGJix5SOVX+KOVC+m3J\\\\\\\/frul2YZ2mf9Nc7vicwwypkiaM1XWgPRytZp2I6CC81ly7QW4XiZ6txcNaxuDkz5v+83S2HwA4NPH\\\\\\\/VCIzzL\\\\\\\/3RpGMrm9V7chRWC9XoX1lOM\\\\\\\/udWeMBH8tCbb+zlSGyV3WD52f0o2vDChCu7oY30CjRtKKCrKQRuSdRntMmtOvlt2\\\\\\\/Jhi+ZoX8PCOGaHrwbLmgk3zLOLXMtTNBuB1YJNhT4iwQQvd6DGFU+rlrxxcyJIau4ft85E+I6omgZkBCbBbxhy0ADKWiBXDsbhzLNUIR3ZBBdbtSyvLJrr6PCgqIZVMoQEAJsFIkGTsEB4kD4rsNn0qYGE69krsmRFpba0JCygjmtgwRDqaxcF0eqfK3m3Mj7ZODkFyjVpZdPIhAhlE5b5+t+JvXryll9UVtXhWV7n8FTdjrLe1M03Kl4\\\\\\\/Vmh6Yi+u3tKtFSVlP9x0UTSqeQGcgB+Lc\\\\\\\/EqGScBaErETt2CovyjGGrCN+HLe0TPuHzr6NJZEDNxX3JL1fVITmoZ38NSA5NEGHelMD26XhqhkzQvmNT4sxpuzCELAs8CP8jXIqfpNNLK\\\\\\\/xZTGMmirnLEltwSDgHVeo1xfzLBuKOZPPd6sZiU2rYsTfI7INCXTnxzv5SfX4Z07ZVQBe3HV53mVcH6i\\\\\\\/HEYw29ynI\\\\\\\/5gP+fj4qvbD4kO40ZnXJkMk7e+PNUNv7zN5hzGyotozLg\\\\\\\/nV9BQsJ65WCdM8WJEti3GKSMDM4N8IrlJ3wz+TMKxlW\\\\\\\/bVnn+to20x8gxm5TzqjELfRClqawB+X2\\\\\\\/YNKOCOM87\\\\\\\/azaUk9\\\\\\\/xDPkB0iOnxZH+nSDSfDtbmgq2Xi11Ogsjoil964+FSMZlrpcm11c7z2b1yJ3c09uSUfFY19qQSv0NmVykWj4rVjqVC1HGbE3n77fZ6i1gOD1DWoYu1c8yM8QyS1b97C00WSFDXEsPojbawrkYOpCtabQKbq+QwxvrcNGDLhbMHKc8q0n8CMn3UjddZPwU4uLd8KXbgObQIR1y+h11kpltsHo7xpO21ORZW0Fioa5G77pQqkenlg77f5MxBsHxZbiONwY4NhaMe6HF32oAt\\\\\\\/YLIS+Z+83\\\\\\\/FFZNrPcoK2uTPxOcRMftbbIvWPjapf9npjHxPIGSdGoI7Nopi0cUOiQQJViwg+UcDQ4wg3D8M0rSwg1yZJ9AimUefOBfPP2dLlRXyK6TbQztnwQh6NStS3LojfLzsMU1igJDZmxTYDQOf\\\\\\\/NI7rtRHfvfXyWceOJ\\\\\\\/H98bugGbCzCulL6+bECIWtx3FNJD2sQ4bksnmpVl5E1MWWH0nS0qBm5abjSs7gY9snu1olqe03DxcoPh+hhD6jSMblrLmSyrB5+0NOelAS\\\\\\\/MxopWDr27Emw5bYHyC+mecKO8Jm3GaeiU0cQcDX9WoNRlXrXTqp54brf1nGtJVG7c7WLRo9LQYxy+dqFEVNjw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022oglhR6rwPTyZuxNurkEBYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cd0b3b7c695686a10ff913fe0ffc8be0fc982bfb6b5c4d6880a742f3f00c6d66\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459978 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1391" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Uoc3KeN9ThhYziu3BVX2ZBO502CS8avgwItNiYLD93U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1391" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NVXsCZMcWhrNrc2J282cZT3q0gns4vrEKkSai0Ij5ok" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1379" title="Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1403" title="Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459978" id="node459978" data-pisa="ggg;5/7/1391" data-pisa-master="ggg;g3.115.017772" data-apath="/ggg/5/7/1391.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African <em>Bos taurus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Imtiaz A. S. Randhawa</span>, <span class="highwire-citation-author" data-delta="1">Mehar S. Khatkar</span>, <span class="highwire-citation-author" data-delta="2">Peter C. Thomson</span> and <span class="highwire-citation-author" data-delta="3">Herman W. Raadsma</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1391-1401;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017772 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459978" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Imtiaz A. S. Randhawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Imtiaz%2BA.%2BS.%2BRandhawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Randhawa%20IA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AImtiaz%2BA.%2BS.%2BRandhawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:imtiaz.randhawa@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">imtiaz.randhawa@sydney.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mehar S. Khatkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehar%2BS.%2BKhatkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khatkar%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMehar%2BS.%2BKhatkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter C. Thomson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BC.%2BThomson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomson%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BC.%2BThomson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Herman W. Raadsma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herman%2BW.%2BRaadsma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raadsma%20HW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerman%2BW.%2BRaadsma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/7/1391" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459978" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1391.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459978" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1391.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459978" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1391.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459978" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/7/1391.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459978" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Understanding the evolution and molecular architecture of complex traits is important in domestic animals. Due to phenotypic selection, genomic regions develop unique patterns of genetic diversity called signatures of selection, which are challenging to detect, especially for complex polygenic traits. In this study, we applied the composite selection signals (CSS) method to investigate evidence of positive selection in a complex polygenic trait by examining stature in phenotypically diverse cattle comprising 47 European and 8 African <em>Bos taurus</em> breeds, utilizing a panel of 38,033 SNPs genotyped on 1106 animals. CSS were computed for phenotypic contrasts between multibreed cohorts of cattle by classifying the breeds according to their documented wither height to detect the candidate regions under selection. Using the CSS method, clusters of signatures of selection were detected at 26 regions (9 in European and 17 in African cohorts) on 13 bovine autosomes. Using comparative mapping information on human height, 30 candidate genes mapped at 12 selection regions (on 8 autosomes) could be linked to bovine stature diversity. Of these 12 candidate gene regions, three contained known genes (<em>i.e.</em>, <em>NCAPG-LCORL</em>, <em>FBP2-PTCH1</em>, and <em>PLAG1-CHCHD7</em>) related to bovine stature, and nine were not previously described in cattle (five in European and four in African cohorts). Overall, this study demonstrates the utility of CSS coupled with strategies of combining multibreed datasets in the identification and discovery of genomic regions underlying complex traits. Characterization of multiple signatures of selection and their underlying candidate genes will elucidate the polygenic nature of stature across cattle breeds.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/body-size" class="hw-term hw-article-keyword hw-article-keyword-body-size" rel="nofollow">body size</a></li><li class="kwd"><a href="/keyword/outbred-populations" class="hw-term hw-article-keyword hw-article-keyword-outbred-populations" rel="nofollow">outbred populations</a></li><li class="kwd"><a href="/keyword/polygenic-traits" class="hw-term hw-article-keyword hw-article-keyword-polygenic-traits" rel="nofollow">polygenic traits</a></li><li class="kwd"><a href="/keyword/signatures-selection" class="hw-term hw-article-keyword hw-article-keyword-signatures-of-selection" rel="nofollow">signatures of selection</a></li><li class="kwd"><a href="/keyword/stature-genes" class="hw-term hw-article-keyword hw-article-keyword-stature-genes" rel="nofollow">stature genes</a></li></ul><p id="p-3">Genomes that have experienced selective pressure due to their functional role in phenotypic diversity contain unique patterns of genetic diversity called signatures of selection (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Oleksyk <em>et al.</em> 2010</a>). Recent progress in molecular genetics has provided powerful tools to generate and analyze genetic polymorphism data (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lenstra <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hancock and Di Rienzo 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crisci <em>et al.</em> 2012</a>). Consequently, genome-wide scans to detect signatures of selection have been conducted in several species to understand the role of molecular variants in shaping their genetic diversity (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gibbs <em>et al.</em> 2009</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ramey <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kijas <em>et al.</em> 2012</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2012</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Petersen <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Qanbari <em>et al.</em> 2014</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Utsunomiya <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akey <em>et al.</em> 2010</a>).</p><p id="p-4">Footprints of sustained selection in livestock are generally diverse and complex. It has been a great challenge to differentiate between the neutrally (random drift) and rapidly (selection) evolving traits (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hohenlohe <em>et al.</em> 2010</a>). In general, signatures of selection are identified from the patterns of DNA variation without recorded phenotypic information on populations undergoing investigation. Nonetheless, the signatures of selection detected between populations are often indicative of segregating regions of functional mutations underlying divergently selected quantitative traits (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hayes <em>et al.</em> 2008</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2009</a>). In complex genomes such as mammalian species, the functional mutations and their systematic patterns of polymorphism exist in multiple locations. Identification of the trait-specific genomic regions requires comprehensive phenotypic and genomic data of large populations to detect causal effects of selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Decker <em>et al.</em> 2012</a>). Despite using known mapping approaches, it has often been difficult to localize historical selection events for complex traits (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>). A number of tests are available to detect candidate regions under selection, which often provide differing results from the same genomic dataset (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Qanbari <em>et al.</em> 2011</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Crisci <em>et al.</em> 2012</a>). Combining several tests with complementary characteristics can improve the ability to localize the candidate regions under selection (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Utsunomiya <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grossman <em>et al.</em> 2010</a>).</p><p id="p-5">Recently, we developed a composite selection signal (CSS) test (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> 2014</a>), a weighted index of signatures of selection from multiple estimates, to identify genomic regions under selection in domestic species. CSS is a nonparametric procedure that uses fractional ranks of constituent tests and it does not depend on their distributional assumptions. Hence, as compared to the previous approaches of combining multiple selection tests that have limitations to incorporate only the statistics with <em>p</em>-values (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Utsunomiya <em>et al.</em> 2013</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Grossman <em>et al.</em> 2010</a>), CSS has an appealing feature to combine the evidence of historical selection from any selection test. We demonstrated that CSS is a robust method to identify loci for across-population genetic diversity of genomic regions, linking them to target phenotypes of monogenic traits under selective pressure (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> 2014</a>). CSS can be extended to complex traits. It is expected that co-selection on multiple genomic loci underling a complex trait can be captured in the form of CSS by combining the signals detected from the patterns of population differentiation, derived allele frequency, and the extended haplotype homozygosity.</p><p id="p-6">The quantitative trait loci (QTL) and signatures of selection mapped at the candidate regions suggested the role of underlying genes in diversity of stature across various breeds (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> 2013</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Ramey <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Karim <em>et al.</em> 2011</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nishimura <em>et al.</em> 2012</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>).</p><p id="p-7">Diversity in bovine stature can potentially be used as a model to demonstrate the application of new approaches, such as CSS, for the detection of selection for complex traits in cattle, given that the trait can be easily measured and data are available in a wide range of worldwide breeds. Notably, stature is a trait strongly influenced by domestication (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Karim <em>et al.</em> 2011</a>) and has a known history of selective pressure over time, both negative and positive (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ajmone-Marsan <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fortes <em>et al.</em> 2013</a>). Stature (or adult height) is a complex trait under polygenic control with high heritability in many mammalian species including cattle (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kemper and Goddard 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lettre 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>). Genetic architecture of height has been extensively investigated to identify genes with major effects across the genome of multiple species, although the leading model has been the human genome (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lettre <em>et al.</em> 2008</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sanna <em>et al.</em> 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Shiura <em>et al.</em> 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tetens <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Deng <em>et al.</em> 2013</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>). To date, only a few genes responsible for stature, including body size traits, have been reported in cattle from genome-wide association studies (GWAS) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Fortes <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hoshiba <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolormaa <em>et al.</em> 2014</a>).</p><p id="p-8">Two major candidate gene regions, <em>NCAPG-LCORL</em> and <em>PLAG1-CHCHD7</em>, on chromosome 6 and chromosome 14, respectively, have been linked to stature in European breeds of cattle. The <em>NCAPG</em> and <em>LCORL</em> genes are strong candidates for growth and height-related traits in multiple species of mammals, including cattle (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gudbjartsson <em>et al.</em> 2008</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Weedon <em>et al.</em> 2008</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Soranzo <em>et al.</em> 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lango Allen <em>et al.</em> 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sovio <em>et al.</em> 2009</a>). This candidate gene region has frequently shown strong signatures of selection in multiple breeds of cattle (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barendse <em>et al.</em> 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Flori <em>et al.</em> 2009</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gibbs <em>et al.</em> 2009</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mancini <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pintus <em>et al.</em> 2013</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Porto-Neto <em>et al.</em> 2013</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rothammer <em>et al.</em> 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Stella <em>et al.</em> 2010</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Qanbari <em>et al.</em> 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Perez <em>et al.</em> 2014</a>). Several polymorphic variants at this locus have been identified in GWAS and QTL studies for wither height in horse (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Tetens <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Signer-Hasler <em>et al.</em> 2012</a>), and growth, skeletal, carcass, and stature traits in cattle (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lindholm-Perry <em>et al.</em> 2011</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hoshiba <em>et al.</em> 2013</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Nishimura <em>et al.</em> 2012</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bolormaa <em>et al.</em> 2014</a>). The <em>PLAG1-CHCHD7</em> locus harbors nine candidate genes and has been associated with height in humans (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gudbjartsson <em>et al.</em> 2008</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Soranzo <em>et al.</em> 2009</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lettre <em>et al.</em> 2008</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lango Allen <em>et al.</em> 2010</a>) and stature in cattle (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Nishimura <em>et al.</em> 2012</a>; <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Karim <em>et al.</em> 2011</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Fortes <em>et al.</em> 2013</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bolormaa <em>et al.</em> 2014</a>). This locus was also identified as a candidate region based on selective sweeps in several cattle breeds that have been under strong selection for body size (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> 2013</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Ramey <em>et al.</em> 2013</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>). However, known genes account only for a small proportion of the phenotypic variation in bovine stature (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kemper and Goddard 2012</a>). Comparative mapping provides additional evidence that some orthologous genes of human and cattle affect stature in both species (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>). Therefore, the comparative mapping of genes identified in human GWAS can be used as supporting evidence to elucidate additional candidate gene regions in cattle (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Visscher and Goddard 2011</a>).</p><p id="p-9">Selective pressure on phenotypically similar populations may have acted on the same causal variants in analogous genes (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kijas <em>et al.</em> 2012</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Stella <em>et al.</em> 2010</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pickrell <em>et al.</em> 2009</a>). Therefore, a multibreed panel by combining phenotypically alike breeds, and comparing such multibreed groups for contrasting phenotypes can maximize the detectable signatures of selection for bovine stature (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Barendse <em>et al.</em> 2009</a>; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Stella <em>et al.</em> 2010</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Kemper and Goddard 2012</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Visscher and Goddard 2011</a>).</p><p id="p-10">This study applies the CSS method to identify candidate genomic regions harboring genes responsible for bovine stature, a polygenic complex trait. Cohorts of contrasting phenotypes were investigated in multibreed panel comparisons using morphological information on wither height of 55 worldwide breeds of European and African <em>Bos taurus</em>. Finally, we compiled orthologous genes from several GWAS on human height and mapped them on the bovine genome assembly (UMD3.1) to explicate the significant genomic regions in cattle defined by CSS.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotypic data</h3><p id="p-11">In total, 1106 animals of 55 breeds were genotyped with an Illumina BovineSNP50 chip assay (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/017772SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS1.pdf">Table S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS2.pdf">Table S2</a>). Information regarding DNA samples and SNP genotypes were previously reported (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Decker <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gautier <em>et al.</em> 2009</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2010</a>). The duplicate samples across multiple datasets of cattle were identified and removed based on an IBS matrix computed using PLINK (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Purcell <em>et al.</em> 2007</a>). The IBS matrix estimates the genetic relationship based on genome-wide SNPs similarity. A quality control was implemented to retain SNPs that can be mapped on UMD3.1 bovine assembly and have minor allele frequency (MAF) ≥0.01 (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS3.pdf">Table S3</a>). Ancestral and derived allelic phases of these SNPs were determined based on information on several closely related out-group species, including bison, buffalo, and yak (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Decker <em>et al.</em> 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Matukumalli <em>et al.</em> 2009</a>). Imputation of missing genotypes and haplotype phasing were performed with BEAGLE 3.3 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Browning and Browning 2009</a>). Analyses of signatures of selection for European and African cohorts were performed using polymorphic and phased SNPs within each cattle type.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data</h3><p id="p-12">Bovine stature is generally defined as height at the withers; however, this information was not directly available on the genotyped animals of the selected breeds. Therefore, in this study, for a broader perspective of phenotypic characterization, wither heights of 47 European <em>Bos taurus</em> breeds (listed in <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS1.pdf">Table S1</a>) and 8 African <em>Bos taurus</em> breeds (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS2.pdf">Table S2</a>) reported by 52 countries were acquired from the online database (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">DAD-IS 2014</a>) of the Food and Agriculture Organization (FAO). FAO data for the bovine stature are available as the country-wise breed averages for male and female height at withers. Availability of breed averages for wither height ranged from a single country (<em>i.e.</em>, country of breed origin or country of DNA sampling) to many countries (<em>n</em> = 37). Breed-wise as well as overall estimates of median, upper quartile, and lower quartile of whither height were computed to categorize breeds into phenotypically contrasting cohorts (see below). For this purpose, first, country-wise stature data for each breed were prepared from the averages of male and female withers heights (cm). Second, if a breed was represented by a single country, then the country’s average stature for the breed was used to represent the breed; otherwise, the median stature from multiple countries (based on the FAO data available as country-wise average) was used.</p></div><div id="sec-4" class="subsection"><h3>Multibreed cohorts of contrasting stature</h3><p id="p-13">The contrasting cohorts of multiple breeds, within European (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS1.pdf">Table S1</a>) and African (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS2.pdf">Table S2</a>) breed types, were defined for analyses following the expectation that identification of plausible signatures of selection for stature would require a relatively stringent separation of breeds and sufficient sample sizes in each cohort. Therefore, a stringent strategy was adopted to construct two cohorts of contrasting phenotypes, <em>i.e.</em>, the large and small stature cohorts (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS1.pdf">Figure S1</a>). The individual breeds that have their median stature above the overall upper quartile and below the overall lower quartile were categorized into the large and the small cohorts, respectively. Moreover, the large cohort’s breeds (represented by several countries) are further restricted to have their individual lower quartiles above the overall median. Similarly, the small cohort’s breeds (represented by several countries) have their individual upper quartiles below the overall median. Separate analyses to identify signatures of selection were performed to compare the large and small cohorts of European and African breeds.</p><p id="p-14">Previously, we have shown that cohort composition had minimal effect on the computation of composite selection tests when the breeds with small sample size (<em>n</em> &lt; 10 and <em>n</em> &lt; 20) were excluded (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> 2014</a>). Further, in this study, we also assessed the effect of cohort composition comprising European breeds of cattle, specifically whether a few breeds with large sample size (<em>n</em> &gt; 20) can disproportionately affect the analyses. To this effect we performed additional analyses by limiting the sample size (<em>n</em> ≤ 10 and <em>n</em> ≤ 20) per breed. These results were compared to those from the original cohorts, where we included all available samples from each breed and the constituent and composite selection tests were computed by analyzing the European large and small cohorts against each other.</p></div><div id="sec-5" class="subsection"><h3>Composite selection signal</h3><p id="p-15">The detailed methods of computing composite selection signal (CSS) are provided by <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> (2014)</a>. Briefly, the CSS statistics were computed at each locus by combining three popular constituent selection methods, namely, change in allele diversity by <em>F</em><sub>ST</sub> (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Weir and Cockerham 1984</a>) across-population extended haplotype homozygosity (XP-EHH) test (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sabeti <em>et al.</em> 2007</a>), and change in derived allele frequencies (ΔDAF) test (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Grossman <em>et al.</em> 2010</a>). <em>F</em><sub>ST</sub> statistics were computed as the differentiation index between the large and small cohorts. XP-EHH and ΔDAF statistics were computed for the large and small cohorts such that, by analyzing the large (as selected) cohort against the small (as nonselected) cohort and vice versa. The composite statistics (CSS) were computed as follows:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-16">For each constituent method, test statistics were ranked (1, ..., <em>n</em>) genome-wide on <em>n</em> SNPs.</p></li><li id="list-item-2"><p id="p-17">Ranks were converted to fractional ranks (<em>r</em>′) (between 0 and 1) by 1/(<em>n</em> + 1) through <em>n</em>/(<em>n</em> + 1).</p></li><li id="list-item-3"><p id="p-18">Fractional ranks were converted to <em>z</em>-values as <em>z</em> = Φ<sup>-1</sup>(<em>r</em>′) where Φ<sup>-1</sup>(⋅) is the inverse normal cumulative distribution function (CDF).</p></li><li id="list-item-4"><p id="p-19">Mean <em>z</em> scores were calculated by averaging <em>z</em>-values across all constituent tests at each SNP position and <em>p</em>-values were directly obtained from the distribution of means from a normal <em>N</em>(0, <em>m<sup>−</sup></em><sup>1</sup>) distribution where <em>m</em> is the number of constituent test statistics.</p></li><li id="list-item-5"><p id="p-20">Logarithmic (−log<sub>10</sub> of <em>p</em>-values) of these mean <em>z</em>-values were declared as CSS and were plotted against the genomic positions to identify the significant selection signals.</p></li><li id="list-item-6"><p id="p-21">To reduce spurious signals, the empirical CSS scores were smoothed by averaging values within 0.5 Mb sliding windows on both sides at each SNP. The top 0.1% of smoothed CSS scores was used to declare a significant SNP relative to the rest of the genome.</p></li></ol></div><div id="sec-6" class="subsection"><h3>Genomic regions and genes under selection</h3><p id="p-22">Following a similar approach of <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gibbs <em>et al.</em> (2009)</a> and <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kijas <em>et al.</em> (2012)</a>, significant genomic regions were defined as those that harbor at least one significant SNP (top 0.1%) flanked by a set of five or more adjoining SNPs in the top 1%. The genomic positions of the significant regions were defined by the first and last of the top 1% SNP in the cluster. In addition, multiple clusters localized in close proximity (∼1 Mb) were considered as a single significant genomic region.</p><p id="p-23">The genes underlying the significant regions and surrounding locations (∼1 Mb of cluster boundaries) were listed. These putative genes were searched for evidence of association or signatures of selection for stature within other mammalian species. For this purpose, the records of orthologous genes for the major GWAS reports on human height (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Gudbjartsson <em>et al.</em> 2008</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lango Allen <em>et al.</em> 2010</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lettre <em>et al.</em> 2008</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Weedon <em>et al.</em> 2008</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Soranzo <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kim <em>et al.</em> 2010</a>) were mapped on UMD3.1 bovine genome assembly by matching the gene names] (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS4.pdf">Table S4</a>).</p><p id="p-24">In total, 268 orthologous genes were successfully mapped on the bovine autosomes (BTA) and were located in 134 genomic locations, termed as “candidate gene regions” here and defined by the consecutive orthologous genes located on a chromosome within 1 Mb distance from each other. The number of genes per region ranged from one to nine, with an average size of 355 kb. Hence, the 134 regions harboring genes for human stature covered 47.50 Mb (1.9%) of the bovine genome. In addition, after adding 1 Mb surrounding of each candidate gene region, these regions cover 7% of the bovine genome. Hence, finding a candidate gene carrying CSS region out of those 134 regions as expected by chance is equal to 7%.</p><p id="p-25">To compare whether the proportion of identified regions linked to the bovine stature in various analyses was by chance, we performed a permutation test whereby the animals were randomly assigned to two cohorts under a no-selection model. Sample sizes of the permuted cohorts were equal to the samples sizes of real cohorts and samples from individual breeds within cohorts were not weighted. Five sets of independently permuted cohorts, separately for European and African breeds, were analyzed by CSS and the results were compared against the empirical cohorts.</p><p id="p-26">Finally, signatures of selection identified by CSS in candidate gene regions were further classified as previously reported in cattle or novel to the species. Signatures of selection outside of the candidate gene regions were considered as regions of interest that may be putatively related to stature.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Genotypic data after quality control</h3><p id="p-27">Genotypic data passing quality control consisted of 38,033 SNPs that were filtered after combining the available data on various cattle breeds from <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Decker <em>et al.</em> (2009)</a> and <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gautier <em>et al.</em> (2009</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">2010)</a> (see <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS3.pdf">Table S3</a> to access raw datasets). The retained SNPs had MAF ≥0.01, known ancestral and derived alleles, and mapped on 29 autosomes covering 2,512.08 Mb of UMD3.1 bovine genome assembly. The mean (median) distance between consecutive SNPs on the bovine autosomes was 65.52 kb (64.54 kb), ranging from 2.22 kb to 1.44 Mb (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS3.pdf">Table S3</a>). Finally, data subsets of 37,218 and 33,024 informative SNPs were analyzed to detect signatures of selection within European and African cattle, respectively.</p></div><div id="sec-9" class="subsection"><h3>Classification of breeds into phenotypically contrasting cohorts</h3><p id="p-28">From the FAO data (DAD-IS 2014), the overall lower quartile (Q1), median, and upper quartile (Q3) for the combined values for wither height of 47 European breeds were 134 cm, 140 cm, and 143 cm, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). In comparison, the median wither height for the European large (11 breeds) and European small (12 breeds) stature cohorts were 146.8 cm and 129.5 cm, respectively (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS1.pdf">Table S1</a>, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). The minimum median (145 cm of Charolais and Montbeliarde breeds) of the large cohort and the maximum median (133.5 cm of Finnish Ayrshire and Belted Galloway breeds) of the small cohort values were separated by 11.5 cm.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the wither height (cm) of the 47 European (A) and eight African (B) Bos taurus breeds. Each colored dot represents the median of a breed&#x2019;s stature using FAO data from the multiple countries and the vertical error bars range between the upper and lower quartiles. Several breeds are represented from a single country; therefore, they do not have error bars. The horizontal dashed red, green, and blue lines, respectively, represent the overall upper quartile, median, and lower quartile of all breeds&#x2019; data. The red, green, and blue dots represent cattle breeds categorized in the large, medium, and small stature cohorts, respectively (see Figure S1). Boxplots show distribution of breed median data for stature within each cohort." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-220645324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of the wither height (cm) of the 47 European (A) and eight African (B) &lt;em&gt;Bos taurus&lt;/em&gt; breeds. Each colored dot represents the median of a breed&#x2019;s stature using FAO data from the multiple countries and the vertical error bars range between the upper and lower quartiles. Several breeds are represented from a single country; therefore, they do not have error bars. The horizontal dashed red, green, and blue lines, respectively, represent the overall upper quartile, median, and lower quartile of all breeds&#x2019; data. The red, green, and blue dots represent cattle breeds categorized in the large, medium, and small stature cohorts, respectively (see Figure S1). Boxplots show distribution of breed median data for stature within each cohort.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1391/F1.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/7/1391/F1.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">Distribution of the wither height (cm) of the 47 European (A) and eight African (B) <em>Bos taurus</em> breeds. Each colored dot represents the median of a breed’s stature using FAO data from the multiple countries and the vertical error bars range between the upper and lower quartiles. Several breeds are represented from a single country; therefore, they do not have error bars. The horizontal dashed red, green, and blue lines, respectively, represent the overall upper quartile, median, and lower quartile of all breeds’ data. The red, green, and blue dots represent cattle breeds categorized in the large, medium, and small stature cohorts, respectively (see <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS1.pdf">Figure S1</a>). Boxplots show distribution of breed median data for stature within each cohort.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Similarly using the FAO data for the eight African breeds, overall Q1, median, and Q3 for wither heights calculated were 100 cm, 106 cm, and 115 cm, respectively (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The median wither heights for the African large (two breeds) and African small (three breeds) stature cohorts were 138.5 cm and 97.5 cm, respectively (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS2.pdf">Table S2</a>). The separation of closest medians at the tails of the two African cohorts was 31.7 cm.</p><p id="p-31">Overall, the distribution of breed-wise data (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) shows that implementing the stringent grouping strategy resulted in a clear separation (contrast) of wither height (stature) between the large and the small cohorts of the European and African categories.</p></div><div id="sec-10" class="subsection"><h3>Significant CSS</h3><p id="p-32">In the analysis of the European and African cohorts, 38 and 33 SNPs represented the top 0.1%, respectively, based on ranks of smoothed CSS, and these were used to map the significant genomic regions. The number of top 0.1% SNPs within each significant region ranged from 1 to 20 (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS2.pdf">Figure S2</a>). The number of significant regions ranged from four to nine (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The number of top SNPs per region and region size was highest in the European cohorts and lowest in the African cohorts (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS2.pdf">Figure S2</a>). Median (mean) size of the significant regions was 920 kb (1.20 Mb), ranging from 410 kb to 3.46 Mb. Moreover, merging two neighboring regions of European small cattle cohort on BTA-3 increased the maximum region length to 6.34 Mb.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466460/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466460/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of significant signatures of selection and the candidate regions containing stature-associated genes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>CSS in European cattle cohorts</h3><p id="p-36">Genome-wide distribution of the smoothed CSS (−log<sub>10</sub><em>p</em>) for the large and small cohorts of European cattle identified nine genomic regions carrying clusters of significant SNPs <strong>(</strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). A complete list of putative regions identified in the two cohorts of European <em>Bos taurus</em> is provided in <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS5.pdf">Table S5</a>. In the large cohort, the selected regions were identified on BTA-3, BTA-6, BTA-14, and BTA-19. In the small cohort, the selected regions were located on BTA-3, BTA-5, BTA-7, BTA-10, and BTA-16.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots of genome-wide (smoothed) composite selection signals (CSS) in the large and small cohorts of (A) European and (B) African breeds. Each cohort was compared against the other as a reference population, as shown in each Manhattan plot for positive and negative CSS scores above or below the dashed lines. Dashed lines (red for positive CSS and blue for negative CSS) indicate the cut-off at the top 0.1% of the genome-wide smoothed CSS distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-220645324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Manhattan plots of genome-wide (smoothed) composite selection signals (CSS) in the large and small cohorts of (A) European and (B) African breeds. Each cohort was compared against the other as a reference population, as shown in each Manhattan plot for positive and negative CSS scores above or below the dashed lines. Dashed lines (red for positive CSS and blue for negative CSS) indicate the cut-off at the top 0.1% of the genome-wide smoothed CSS distribution.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1391/F2.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/7/1391/F2.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466383" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Manhattan plots of genome-wide (smoothed) composite selection signals (CSS) in the large and small cohorts of (A) European and (B) African breeds. Each cohort was compared against the other as a reference population, as shown in each Manhattan plot for positive and negative CSS scores above or below the dashed lines. Dashed lines (red for positive CSS and blue for negative CSS) indicate the cut-off at the top 0.1% of the genome-wide smoothed CSS distribution.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38"><a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS4.pdf">Figure S4</a> shows a genome-wide comparison for the distribution of significant CSS peaks between the original results (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>) and three additional analyses—performed to evaluate composition of European cohorts given the variable sample sizes of cattle breeds. Overall, high concordance was observed between the significant peaks in cohort-wise analyses using all available individuals (original), with a maximum of 20 and 10 individuals per breed. For comparison, the significant regions were considered valid if found within the top 1% significant threshold. Notably, the significant regions detected in original analyses (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS5.pdf">Table S5</a>) were found above the top 1%, with the only exception being a region on BTA-19 in the European large cohort being not detected with a maximum of 10 individuals per breed. Overall, the impact of varying the number of animals per breed had negligible impact on the CSS profile. Hence, we propose that the maximum number of animals should be used when possible.</p></div><div id="sec-12" class="subsection"><h3>CSS in African cattle cohorts</h3><p id="p-39">The Manhattan plot for African cohorts shows smoothed CSS results from the intercohort comparisons (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). We observed that the genome-wide CSS peaks were less pronounced in African cohorts compared with European breeds. A complete list of 17 regions identified in the two cohorts of African <em>Bos taurus</em> is provided in <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS6.pdf">Table S6</a>. Overall, nine regions (on BTA-3, BTA-7, BTA-10, BTA-19, and BTA-22) were identified carrying selection signatures in the African large cohort and eight regions (on BTA-4, BTA-6, BTA-7, BTA-8, BTA-10, BTA-11, BTA-15, and BTA-19) were identified in the African small cohort.</p></div><div id="sec-13" class="subsection"><h3>Genomic regions and candidate genes for stature</h3><p id="p-40"><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> shows a summary of the significant regions in each cohort with comparison to respective reference populations within the European and African categories. It also shows the number of regions harboring candidate genes associated with stature, which were found in human GWAS (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS4.pdf">Table S4</a>). Overall, out of 9 and 17 significant genomic regions within the European and African cohorts, 7 (77.8%) and 5 (29.4%) were co-located with candidate gene regions that harbor stature-associated genes. The remaining 2 and 12 significant genomic regions in European and African cohorts, respectively, were identified outside the known candidate gene regions. Overall, 46% (<em>n</em> = 12 out of 26) of all significant CSS were co-located with candidate gene regions, this proportion is much higher as compared to the 7% expected by chance alone as described above (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-41"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows a summary comparison of empirical and permuted cohort results. Overall, 8.3% of the significant regions based on permuted CSS in five sets (<em>i.e.</em>, 20 permuted cohorts) contained stature-associated loci (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) that was in agreement with the 7% expected by chance (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS3.pdf">Figure S3</a>). On average, only 6.7% and 9.8% of significant regions of permuted CSS from European and African cohorts had stature-associated genes as compared to 77.8% and 29.4% of empirical regions, respectively.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of empirical and permuted CSS scores for localization of significant regions harboring stature-associated genes. Horizontal lines, labeled with cohorts (European large = 100%, European small = 60%, African small = 37.5%, and African large = 22.2%), indicate the percentage of significant genomic regions from each cohort that localized the stature associated genes. Vertical bars show the distribution of significant regions (%) harboring stature-associated genes in five sets of permutation cohorts that mimicked the empirical cohorts in matching colors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-220645324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of empirical and permuted CSS scores for localization of significant regions harboring stature-associated genes. Horizontal lines, labeled with cohorts (European large = 100%, European small = 60%, African small = 37.5%, and African large = 22.2%), indicate the percentage of significant genomic regions from each cohort that localized the stature associated genes. Vertical bars show the distribution of significant regions (%) harboring stature-associated genes in five sets of permutation cohorts that mimicked the empirical cohorts in matching colors.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1391/F3.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/7/1391/F3.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466449" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Comparison of empirical and permuted CSS scores for localization of significant regions harboring stature-associated genes. Horizontal lines, labeled with cohorts (European large = 100%, European small = 60%, African small = 37.5%, and African large = 22.2%), indicate the percentage of significant genomic regions from each cohort that localized the stature associated genes. Vertical bars show the distribution of significant regions (%) harboring stature-associated genes in five sets of permutation cohorts that mimicked the empirical cohorts in matching colors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The Circos image (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) represents all 26 significant genomic regions, names of important candidate genes for bovine stature (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), and the locations of all the orthologous genes from human GWAS (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS4.pdf">Table S4</a>). <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> shows the list of those 12 genomic regions that harbor 30 candidate genes, which have previously been associated with stature in human GWAS and other species. The results from the major contrasting cohorts, especially large and small cohorts of European <em>Bos taurus</em>, supported the strategy of finding candidate gene regions by grouping phenotypically alike breeds. All the significant signatures of selection (four out of four) identified in the European large cohort contained known candidate genes in multiple species (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) within region 1 (gene: <em>DUSP23</em>), region 6 (<em>NCAPG</em>, <em>LCORL</em>), region 10 (<em>TGS1</em>, <em>LYN</em>, <em>RPS20</em>, <em>MOS</em>, <em>PLAG1</em>, <em>CHCHD7</em>, <em>SDR16C5</em>, <em>RDHE2</em>, <em>PENK</em>), and region 11 (<em>POLR2A</em>). However, in cattle, only regions 6 and 10 have previously been investigated in relation to stature. In the European small cohort, three out of the four significant genomic regions, <em>i.e.</em>, region 3 (<em>PKN2</em>), region 5 (<em>ATP5G2</em>, <em>ATF7</em>), and region 7 (<em>CAMLG</em>, <em>DDX46</em>, <em>TXNDC15</em>, <em>CATSPER3</em>, <em>PITX1</em>) harbor candidate genes that were previously reported to be associated with human height.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Circos image showing localization of genome-wide significant regions and candidate genes across all cohorts. The inner circle shows a bovine chromosome (29 autosomes) ideogram. The first outer circle, labeled as Stature Loci, shows the location of 134 loci listed in Table S4, and each red dot represents a stature-associated candidate gene (clustered red dots, within a locus, located on top of each other represent multiple neighboring candidate genes implicated in the same or different GWAS studies). Two circles, in the middle, labeled as EUROPEAN COHORTS and AFRICAN COHORTS show cohort-wise genomic regions identified by top 0.1% CSS (legends in the center for cohort-wise colored bars). The outer track shows genomic locations of 30 candidate genes within the 12 significant signatures of selection linked to bovine stature. Gene colors indicate whether they were identified in European (green) or African (purple) cattle types." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-220645324" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Circos image showing localization of genome-wide significant regions and candidate genes across all cohorts. The inner circle shows a bovine chromosome (29 autosomes) ideogram. The first outer circle, labeled as &lt;em&gt;Stature Loci&lt;/em&gt;, shows the location of 134 loci listed in Table S4, and each red dot represents a stature-associated candidate gene (clustered red dots, within a locus, located on top of each other represent multiple neighboring candidate genes implicated in the same or different GWAS studies). Two circles, in the middle, labeled as &lt;em&gt;EUROPEAN COHORTS&lt;/em&gt; and &lt;em&gt;AFRICAN COHORTS&lt;/em&gt; show cohort-wise genomic regions identified by top 0.1% CSS (legends in the center for cohort-wise colored bars). The outer track shows genomic locations of 30 candidate genes within the 12 significant signatures of selection linked to bovine stature. Gene colors indicate whether they were identified in European (green) or African (purple) cattle types.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1391/F4.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/7/1391/F4.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1391/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466365" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Circos image showing localization of genome-wide significant regions and candidate genes across all cohorts. The inner circle shows a bovine chromosome (29 autosomes) ideogram. The first outer circle, labeled as <em>Stature Loci</em>, shows the location of 134 loci listed in <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS4.pdf">Table S4</a>, and each red dot represents a stature-associated candidate gene (clustered red dots, within a locus, located on top of each other represent multiple neighboring candidate genes implicated in the same or different GWAS studies). Two circles, in the middle, labeled as <em>EUROPEAN COHORTS</em> and <em>AFRICAN COHORTS</em> show cohort-wise genomic regions identified by top 0.1% CSS (legends in the center for cohort-wise colored bars). The outer track shows genomic locations of 30 candidate genes within the 12 significant signatures of selection linked to bovine stature. Gene colors indicate whether they were identified in European (green) or African (purple) cattle types.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466528/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466528/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Significant selection regions that harbor candidate genes for stature identified by composite selection signals in all analyses of European and African cohorts</span><div class="sb-div caption-clear"></div></div></div><p id="p-46">In the African cohorts, several significant signatures of selection regions did not contain known stature-associated candidate genes. Consequently, the percentage of selection regions with known candidate genes was lower, <em>i.e.</em>, 29.4%, which is almost one-third of that of European cohorts (77.8%). Despite that, this percentage in the African cohorts was almost three-times more than that of the likelihood of occurrence by chance. Nevertheless, we identified five candidate gene regions (2, 4, 8, 9, and 12) in African cohorts that contain important genes for stature, as reported by multiple human GWAS (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-47">Finally, <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/FigureS5.pdf">Figure S5</a> compares the distribution of CSS and its component tests (<em>F</em><sub>ST</sub>, ΔDAF, and XP-EHH) at the 12 candidate regions harboring stature-associated genes. These comparisons show that the CSS scores are not being derived from one particular component selection test. The results also suggest that the three component tests provide complementary evidence to each other at the candidate regions. However, it is evident that many of these candidate regions would not have been detected with any of the component selection tests, particularly regions 1, 2, 5, 8, 9, and 12. These results provide overwhelming support for the CSS strategy, suggesting that it is a robust and efficient method for combining complementary evidence to detect the historical signatures of selection for complex traits, which otherwise cannot be detected by available tests of selection.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><div id="sec-15" class="subsection"><h3>CSS for complex traits</h3><p id="p-48">The CSS approach demonstrates that the test is generally applicable for the identification of trait-specific genomic regions for complex traits by comparing phenotypically contrasting populations. The CSS test combines multiple pieces of evidence from the rank distribution of different selection tests in a weighted index of signatures of selection. The power of the test and its constituents (<em>F</em><sub>ST</sub>, ΔDAF, and XP-EHH) depends on the phenotypic and genetic divergence between the candidate and reference populations, as we demonstrated with simple binary monogenic (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> 2014</a>) and complex polygenic traits (this study).</p><p id="p-49">Use of multibreed cohorts has increased the discovery of trait-specific regions because of shared linkage disequilibrium between the causal mutations and neighboring SNPs (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Kemper and Goddard 2012</a>) arising from long-term historical selection. Contrasting patterns of genomic variation at the putative regions under selection across groups increases the likelihood of capturing the signatures of selection linked to stature. In our approach, signals of selection are amplified at candidate regions across breeds within groups, whereas background noise (false-positive signals) is reduced in the rest of the genome. Such noise is usually expected from the demographic history of breed formation and random genetic drift (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Seichter <em>et al.</em> 2012</a>). Overall, application of the CSS method combined with grouping breeds according to their wither height identified candidate regions in the bovine genome for this complex trait.</p><p id="p-50">Detection of signatures of selection is valuable in the discovery of potential genomic regions of functional mutations affecting quantitative traits (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hayes <em>et al.</em> 2008</a>). Recent investigations suggest that, in the absence of classic selective sweeps, signatures of selection for complex traits are less likely to be detected by using individual selection tests on single breed data (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>). Our approach combines multiple selection tests and grouping of phenotypically alike breeds, and has been found to be highly efficient at detecting signatures of selection and identifying candidate gene regions for complex traits. This approach makes use of existing resources under long-term historical selection and provides a relatively inexpensive entry for more detailed follow-up studies in the genetic architecture of complex traits.</p></div><div id="sec-16" class="subsection"><h3>Detection of signatures of selection for bovine stature</h3><p id="p-51">In this study, we used bovine wither height (stature) measures from an online resource, namely the FAO database (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">DAD-IS 2014</a>), recorded as averages for both males and females across multiple countries. Height and body size have generally been considered important traits in cattle during domestication. During early stages of domestication, selection favored animals with lower stature compared to wild ancestors (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Karim <em>et al.</em> 2011</a>). However, during the recent past, strong selection for increased stature has been applied in selected breeds of modern cattle (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ajmone-Marsan <em>et al.</em> 2010</a>).</p><p id="p-52">We found 26 bovine genomic regions in European and African <em>Bos taurus</em> carrying distinctive signatures of selection. Of these, 12 (46%) contained 30 stature-associated candidate genes derived from comparative mapping. We showed that when cohorts of randomly permuted animals were tested, a much lower proportion (8.3%) of the significant regions contained candidate genes, which is consistent with random expectations (∼7%). This suggests that our approach was powerful in capturing bovine genomic regions related to stature. Several of these should be considered novel because they were not previously reported in cattle.</p><p id="p-53">It is noteworthy why different genomic regions are detected in the large and small cohorts within European and African cattle. The cohorts within each cattle type were compared against each other to compute the constituent (<em>F</em><sub>ST</sub>, ΔDAF, and XP-EHH) selection tests of CSS. Two selection tests (ΔDAF and XP-EHH) provide directional signatures of selection. This implicates that different gene variants are regulating stature in large and small breeds of cattle (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Visscher and Goddard 2011</a>).</p><p id="p-54">Several traits, such as birth weight, growth rate, adult weight, and stature, are found to be correlated with each other and have pleiotropic effects. Some of the genes in the validation regions (see below) have been previously found to be associated with multiple correlated traits. The FAO data on stature have been used to objectively classify cohorts of cattle, and the data were not adjusted for other traits given the nonavailability of FAO records for the correlated traits. Multibreed classification is expected to minimize the breed sampling effects; however, some of the candidate regions may not be directly reflecting selection of stature but could be confounded by some of the other correlated traits.</p><p id="p-55">The significant CSS outside the candidate gene regions (<a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS5.pdf">Table S5</a> and <a href="http://www.g3journal.org/content/suppl/2015/04/30/g3.115.017772.DC1/TableS6.pdf">Table S6</a>) may indicate putative genes associated with height or targets of selection for traits other than stature (<a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Utsunomiya <em>et al.</em> 2013</a>; <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> 2013</a>; <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Ramey <em>et al.</em> 2013</a>; <a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Stella <em>et al.</em> 2010</a>; <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Perez <em>et al.</em> 2014</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Porto-Neto <em>et al.</em> 2013</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rothammer <em>et al.</em> 2013</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Gautier <em>et al.</em> 2009</a>). Moreover, several additional significant CSS regions in African cattle have not been reported previously, given the limited reports on selective sweep analyses in the African breeds.</p></div><div id="sec-17" class="subsection"><h3>Validation of known stature-associated genomic regions in cattle</h3><p id="p-56">This study validated two important candidate gene regions (regions 6 and 10) linked to stature in European breeds of cattle. Region 6 harboring <em>NCAPG</em> and <em>LCORL</em> genes located on BTA-6 has frequently shown strong signatures of selection in multiple breeds of cattle (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Barendse <em>et al.</em> 2009</a>; <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> 2013</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Flori <em>et al.</em> 2009</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Gibbs <em>et al.</em> 2009</a>; <a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Mancini <em>et al.</em> 2014</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Pintus <em>et al.</em> 2013</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Porto-Neto <em>et al.</em> 2013</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Rothammer <em>et al.</em> 2013</a>; <a id="xref-ref-65-5" class="xref-bibr" href="#ref-65">Stella <em>et al.</em> 2010</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Qanbari <em>et al.</em> 2014</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Perez <em>et al.</em> 2014</a>). Region 10 harboring <em>PLAG1</em> and <em>CHCHD7</em> genes on BTA-14 has been associated with height in humans (<a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Gudbjartsson <em>et al.</em> 2008</a>; <a id="xref-ref-63-7" class="xref-bibr" href="#ref-63">Soranzo <em>et al.</em> 2009</a>; <a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Lettre <em>et al.</em> 2008</a>; <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Lango Allen <em>et al.</em> 2010</a>) and stature in cattle (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Nishimura <em>et al.</em> 2012</a>; <a id="xref-ref-50-9" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>; <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Karim <em>et al.</em> 2011</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Fortes <em>et al.</em> 2013</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bolormaa <em>et al.</em> 2014</a>). This locus was also identified as a candidate region based on selective sweeps in several cattle breeds that have been under strong selection for body size (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> 2013</a>; <a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Ramey <em>et al.</em> 2013</a>; <a id="xref-ref-29-8" class="xref-bibr" href="#ref-29">Kemper <em>et al.</em> 2014</a>).</p></div><div id="sec-18" class="subsection"><h3>Genomic regions associated with stature in European <em>Bos taurus</em></h3><p id="p-57">Based on strong recent selection for stature in European <em>Bos taurus</em>, our comparison of the large cohort with the small cohort successfully localized height-related genes at all significant regions (regions 1, 6, 10, and 11). Regions 1 and 11 can be considered novel and may contain functional variants contributing to the genetic control of bovine stature, because they harbor <em>DUSP23</em> (BTA-3: 10.02-10.03 Mb) and <em>POLR2A</em> (BTA-19: 27.78-27.80) as candidate genes, respectively (<a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>).</p><p id="p-58">There can be some advantages of limiting body size in smaller breeds, for instance, in relation to ease of calving or resource-limited production systems. As noted above, early domestication favored selective breeding to reduce body size for ease of management (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Karim <em>et al.</em> 2011</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Ajmone-Marsan <em>et al.</em> 2010</a>). With very few exceptions (<em>e.g.</em>, Dexter), recent history of European <em>Bos taurus</em> breeds do not document selective breeding for smaller size. Interestingly, 60% of all significant regions can be considered novel in the small cohort and harbor stature-associated genes, namely <em>PKN2</em> at region 3 (<a id="xref-ref-33-11" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>), <em>ATP5G2</em> and <em>ATF7</em> at region 5 (<a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Okada <em>et al.</em> 2010</a>), and <em>CAMLG</em>, <em>DDX46</em>, <em>TXNDC15</em>, <em>CATSPER3</em>, and <em>PITX1</em> at region 7 (<a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Gudbjartsson <em>et al.</em> 2008</a>). The functional role of these candidate genes in relation to stature is not well-documented.</p></div><div id="sec-19" class="subsection"><h3>Genomic regions associated with stature in African <em>Bos taurus</em></h3><p id="p-59">Five candidate regions (2, 4, 8, 9, 12) in the African cohorts harbor nine candidate genes associated with human height identified by GWAS (<a id="xref-ref-20-10" class="xref-bibr" href="#ref-20">Gudbjartsson <em>et al.</em> 2008</a>; <a id="xref-ref-63-8" class="xref-bibr" href="#ref-63">Soranzo <em>et al.</em> 2009</a>; <a id="xref-ref-43-10" class="xref-bibr" href="#ref-43">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-33-12" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Kim <em>et al.</em> 2010</a>; <a id="xref-ref-69-7" class="xref-bibr" href="#ref-69">Weedon <em>et al.</em> 2008</a>). Previously, <em>FBP2</em> and <em>PTCH1</em> genes underlying region 8 (BTA-8) have been associated with stature in European dairy and beef breeds, respectively (<a id="xref-ref-69-8" class="xref-bibr" href="#ref-69">Weedon <em>et al.</em> 2008</a>; <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Kim <em>et al.</em> 2010</a>; <a id="xref-ref-33-13" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>; <a id="xref-ref-50-10" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>). In region 12 (BTA-19), the growth hormone genes (<em>GH1</em>, <em>GH2</em>) are strong candidates for dairy production traits (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Rothammer <em>et al.</em> 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mullen <em>et al.</em> 2010</a>). Moreover, three genes (<em>WDR68</em>, <em>MAP3K3</em>, and <em>LYK5</em>) neighboring growth hormone genes have also been associated with human height in multiple GWAS reports (<a id="xref-ref-20-11" class="xref-bibr" href="#ref-20">Gudbjartsson <em>et al.</em> 2008</a>; <a id="xref-ref-63-9" class="xref-bibr" href="#ref-63">Soranzo <em>et al.</em> 2009</a>; <a id="xref-ref-33-14" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>). The remaining three candidate regions, <em>i.e.</em>, regions 2, 4, and 9, harbor the stature-associated <em>SPAG17</em>, <em>SCMH1</em>, and <em>CYP19A1</em> genes, respectively (<a id="xref-ref-69-9" class="xref-bibr" href="#ref-69">Weedon <em>et al.</em> 2008</a>; <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Kim <em>et al.</em> 2010</a>; <a id="xref-ref-43-11" class="xref-bibr" href="#ref-43">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-33-15" class="xref-bibr" href="#ref-33">Lanktree <em>et al.</em> 2011</a>).</p><p id="p-60">Overall, four of the five candidate regions identified in the African cohorts are novel for their association with bovine stature. This suggests that the strategy of combining CSS and multibreed panels showed utility in the African cohorts also, although the overall proportion of candidate gene regions detected was lower compared to European cohorts. High levels of <em>Bos indicus</em> admixture, smaller sample size, and ascertainment bias of the Illumina BovineSNP50 chip in the African <em>Bos taurus</em> breeds may have impacted the power to detect regions under selection (<a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> 2014</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Decker <em>et al.</em> 2009</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gautier <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gautier and Naves 2011</a>).</p></div><div id="sec-20" class="subsection"><h3>Comparison of significant CSS regions between European and African <em>Bos taurus</em></h3><p id="p-61">No candidate gene regions were found in common across the European and African cattle types. This may be due to the long-term differences in natural and production-oriented selective pressures in these diverse genetic archetypes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edea <em>et al.</em> 2014</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Gautier <em>et al.</em> 2009</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gautier and Naves 2011</a>; <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Gibbs <em>et al.</em> 2009</a>; <a id="xref-ref-55-7" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> 2014</a>). Recent attempts to find the transferability of height-associated loci across human populations have shown comparable results (low consistency) for finding the same genetic variants when discoveries made in non-African ethnicities were compared in populations of African ancestry (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kang <em>et al.</em> 2010</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Shriner <em>et al.</em> 2009</a>). Our results indicate that there may be a persistent trend of dissimilarities for African genealogies across species, and some environmental factors would have shaped a comparatively different genetic architecture. We suggest that investigation of several other African breeds with sufficient sample sizes can further elaborate the nature of different candidate gene regions.</p><p id="p-62">A region on BTA-13 harboring <em>UQCC</em> and <em>GDF5</em> genes, which we putatively linked to bovine stature (<a id="xref-ref-55-8" class="xref-bibr" href="#ref-55">Randhawa <em>et al.</em> 2014</a>), was also not detected in this study. The absence of BTA-13 genes (<em>UQCC</em> and <em>GDF5</em>) as the possible candidate for bovine stature in this study are due to the sensitivity of CSS (and its constituent tests of selection) to the sample composition because the cohort composition between the two papers is quite different regarding inclusion and exclusion of several breeds. Moreover, another gene, <em>ASIP</em> (related to coat color), was also located at the BTA-13 peak, and <em>ASIP</em> may also be another plausible candidate of the signatures of selection detected in the previous study.</p><p id="p-63">In comparison to previous findings in cattle, we note that some of the known genes and regions related to bovine stature were not detected here, mainly because they were identified in <em>Bos indicus</em> and composite breeds (<a id="xref-ref-50-11" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>). Moreover, genetic architecture of stature is not considered fully analogous in cattle and human; our results suggest that many common genes might be participating in the physiological control of stature in the two species (<a id="xref-ref-50-12" class="xref-bibr" href="#ref-50">Pryce <em>et al.</em> 2011</a>).</p><p id="p-64">Characterization of biological function of candidate genes could help our understanding of mechanisms underlying stature diversity. Future research should focus on regional genomic sequencing (involving multiple genes) rather than single gene-centric approaches. In addition, gene networks and functional pathway analyses on subsets of genes identified by genome-wide association or selection scans can help uncover the biological processes in complex traits.</p></div></div><div class="section conclusions" id="sec-21"><h2 class="">Conclusion</h2><p id="p-65">This study demonstrates an approach to detect signatures of selection for complex traits. We successfully used CSS in the analysis of the genetic architecture of a complex polygenic trait using multibreed panels. Our results demonstrated the localization of genomic regions harboring candidate genes of known effect related to body development, skeletal growth, and stature in cattle. We identified 26 regions with strong signatures of selection for stature, of which 12 regions contained known genes linked to height or stature and 9 could be considered novel in European and African <em>Bos taurus</em>. These results should be of interest for future investigations to characterize causal variants to explain their functional role in the diversity of bovine stature. Implementation of new tools, such as CSS, can resolve the selected genomic regions underlying the complex traits in livestock species. We suggest that the application of genome-wide selection scans and GWAS using larger resource populations with phenotypic information can further improve the fine-mapping of causal mutations controlling complex traits.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-5"><p id="p-66">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017772/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017772/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-67"><em>Communicating editor: D. L. Bannasch</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-08-02"><span class="received-label">Received </span>August 2, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-28"><span class="accepted-label">Accepted </span>April 28, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Randhawa <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.1" data-doi="10.1002/evan.20267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ajmone-Marsan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenstra</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the origin of cattle: How aurochs became cattle and colonized the world</span>. <abbr class="cit-jnl-abbrev">Evol. Anthropol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAnthropol.%26rft.volume%253D19%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevan.20267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evan.20267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.2" data-doi="10.1073/pnas.0909918107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruhe</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Tracking footprints of artificial selection in the dog genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1160</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909918107%26rft_id%253Dinfo%253Apmid%252F20080661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzMvMTE2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.3" data-doi="10.1186/1471-2164-10-178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barendse</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bunch</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome wide signatures of positive selection: The comparison of independent samples and the identification of regions associated to traits.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBarendse%26rft.auinit1%253DW.%26rft.volume%253D10%26rft.spage%253D178%26rft.epage%253D178%26rft.atitle%253DGenome%2Bwide%2Bsignatures%2Bof%2Bpositive%2Bselection%253A%2Bthe%2Bcomparison%2Bof%2Bindependent%2Bsamples%2Band%2Bthe%2Bidentification%2Bof%2Bregions%2Bassociated%2Bto%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-178%26rft_id%253Dinfo%253Apmid%252F19393047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19393047&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.4" data-doi="10.1371/journal.pgen.1004198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolormaa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reverter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barendse</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A multi-trait, meta-analysis for detecting pleiotropic polymorphisms for stature, fatness and reproduction in beef cattle.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004198%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004198%26rft_id%253Dinfo%253Apmid%252F24675618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24675618&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.5" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.6" data-doi="10.1093/jhered/esr128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crisci</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simkin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recent progress in polymorphism-based population genetic inference.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesr128%26rft_id%253Dinfo%253Apmid%252F22246406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwMy8yLzI4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-5.7.1391.7"><div class="cit-metadata"><cite>DAD-IS, 2014 Domestic Animal Diversity Information System (DAD-IS), Food and Agriculture Organization of the United Nations. Available at: <a href="http://www.fao.org/dad-is/">http://www.fao.org/dad-is/</a>. <span class="cit-comment">Accessed: February 1, 2014.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.8" data-doi="10.1186/1471-2164-13-606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vasco</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClure</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolf</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A novel analytical method, Birth Date Selection Mapping, detects response of the Angus (Bos taurus) genome to selection on complex traits.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDecker%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D606%26rft.epage%253D606%26rft.atitle%253DA%2Bnovel%2Banalytical%2Bmethod%252C%2Bbirth%2Bdate%2Bselection%2Bmapping%252C%2Bdetects%2Bresponse%2Bof%2Bthe%2BAngus%2B%2528Bos%2Btaurus%2529%2Bgenome%2Bto%2Bselection%2Bon%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-606%26rft_id%253Dinfo%253Apmid%252F23140540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23140540&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.9" data-doi="10.1073/pnas.0904691106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conant</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heaton</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Resolving the evolution of extant and extinct ruminants with high-throughput phylogenomics.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">18644</span>–<span class="cit-lpage">18649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904691106%26rft_id%253Dinfo%253Apmid%252F19846765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQ0LzE4NjQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.10" data-doi="10.1016/j.bone.2012.11.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association study identified <em>UQCC</em> locus for spine bone size in humans.</span> <abbr class="cit-jnl-abbrev">Bone</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBone%26rft.volume%253D53%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bone.2012.11.028%26rft_id%253Dinfo%253Apmid%252F23207799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bone.2012.11.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23207799&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Pardal</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of large selective sweeps associated with major genes in cattle.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D44%26rft.spage%253D758%26rft_id%253Dinfo%253Apmid%252F23859468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23859468&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.12" data-doi="10.1111/jbg.12083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edea</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dadi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage disequilibrium and genomic scan to detect selective loci in cattle populations adapted to different ecological conditions in Ethiopia.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.13" data-doi="10.1371/journal.pone.0006595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flori</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaffrézic</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boussaha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gut</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genome response to artificial selection: A case study in dairy cattle.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFlori%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De6595%26rft.epage%253De6595%26rft.atitle%253DThe%2Bgenome%2Bresponse%2Bto%2Bartificial%2Bselection%253A%2Ba%2Bcase%2Bstudy%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006595%26rft_id%253Dinfo%253Apmid%252F19672461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19672461&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.14" data-doi="10.1111/age.12075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortes</span>  <span class="cit-name-given-names">M. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasazaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reverter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evidence for pleiotropism and recent selection in the <em>PLAG1</em> region in Australian Beef cattle.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">636</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D44%26rft.spage%253D636%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12075%26rft_id%253Dinfo%253Apmid%252F23909810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23909810&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.15" data-doi="10.1186/1471-2164-10-550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flori</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riebler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaffrezic</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A whole genome Bayesian scan for adaptive genetic divergence in West African cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGautier%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.spage%253D550%26rft.epage%253D550%26rft.atitle%253DA%2Bwhole%2Bgenome%2BBayesian%2Bscan%2Bfor%2Badaptive%2Bgenetic%2Bdivergence%2Bin%2BWest%2BAfrican%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-550%26rft_id%253Dinfo%253Apmid%252F19930592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930592&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.16" data-doi="10.1371/journal.pone.0013038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moazami-Goudarzi</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Insights into the genetic history of French cattle from dense SNP data on 47 Worldwide breeds.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D5%26rft.spage%253De13038%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013038%26rft_id%253Dinfo%253Apmid%252F20927341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20927341&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.17" data-doi="10.1111/j.1365-294X.2011.05163.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naves</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Footprints of selection in the ancestral admixture of a New World Creole cattle breed.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3128</span>–<span class="cit-lpage">3143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGautier%26rft.auinit1%253DM.%26rft.volume%253D20%26rft.issue%253D15%26rft.spage%253D3128%26rft.epage%253D3143%26rft.atitle%253DFootprints%2Bof%2Bselection%2Bin%2Bthe%2Bancestral%2Badmixture%2Bof%2Ba%2BNew%2BWorld%2BCreole%2Bcattle%2Bbreed.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05163.x%26rft_id%253Dinfo%253Apmid%252F21689193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05163.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21689193&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292963100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.18" data-doi="10.1126/science.1167936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barendse</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eversole</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BBovine%2BHapMap%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D324%26rft.issue%253D5926%26rft.spage%253D528%26rft.epage%253D532%26rft.atitle%253DGenome-Wide%2BSurvey%2Bof%2BSNP%2BVariation%2BUncovers%2Bthe%2BGenetic%2BStructure%2Bof%2BCattle%2BBreeds%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1167936%26rft_id%253Dinfo%253Apmid%252F19390050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNi81MjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.19" data-doi="10.1126/science.1183863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shylakhter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morales</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A composite of multiple signals distinguishes causal variants in regions of positive selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183863%26rft_id%253Dinfo%253Apmid%252F20056855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.20" data-doi="10.1038/ng.122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefansson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halldorsson</span>  <span class="cit-name-given-names">B. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Many sequence variants affecting diversity of adult human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGudbjartsson%26rft.auinit1%253DD.%2BF.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D609%26rft.epage%253D615%26rft.atitle%253DMany%2Bsequence%2Bvariants%2Baffecting%2Bdiversity%2Bof%2Badult%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.122%26rft_id%253Dinfo%253Apmid%252F18391951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391951&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.21" data-doi="10.1146/annurev.anthro.37.081407.085141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Detecting the genetic signature of natural selection in human populations: Models, methods, and data.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anthropol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BAnthropology%26rft.stitle%253DAnnu.%2BRev.%2BAnthropol.%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D37%26rft.spage%253D197%26rft.epage%253D217%26rft.atitle%253DDetecting%2Bthe%2BGenetic%2BSignature%2Bof%2BNatural%2BSelection%2Bin%2BHuman%2BPopulations%253A%2BModels%252C%2BMethods%252C%2Band%2BData.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.anthro.37.081407.085141%26rft_id%253Dinfo%253Apmid%252F20622977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.anthro.37.081407.085141&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20622977&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.22" data-doi="10.1111/j.1365-2052.2007.01683.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nilsen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The origin of selection signatures on bovine chromosome 6.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D111%26rft.atitle%253DThe%2Borigin%2Bof%2Bselection%2Bsignatures%2Bon%2Bbovine%2Bchromosome%2B6.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2007.01683.x%26rft_id%253Dinfo%253Apmid%252F18241301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2007.01683.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18241301&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.23" data-doi="10.1111/j.1365-2052.2008.01815.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maceachern</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPartlan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genome map of divergent artificial selection between <em>Bos taurus</em> dairy cattle and <em>Bos taurus</em> beef cattle.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D176%26rft.epage%253D184%26rft.atitle%253DA%2Bgenome%2Bmap%2Bof%2Bdivergent%2Bartificial%2Bselection%2Bbetween%2BBos%2Btaurus%2Bdairy%2Bcattle%2Band%2BBos%2Btaurus%2Bbeef%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2008.01815.x%26rft_id%253Dinfo%253Apmid%252F19067671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2008.01815.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19067671&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.24" data-doi="10.1086/656306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Using population genomics to detect selection in natural populations: Key concepts and methodological considerations.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Sci.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1059</span>–<span class="cit-lpage">1071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BPlant%2BSciences%26rft.stitle%253DInternational%2BJournal%2Bof%2BPlant%2BSciences%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D171%26rft.issue%253D9%26rft.spage%253D1059%26rft.epage%253D1071%26rft.atitle%253DUSING%2BPOPULATION%2BGENOMICS%2BTO%2BDETECT%2BSELECTION%2BIN%2BNATURAL%2BPOPULATIONS%253A%2BKEY%2BCONCEPTS%2BAND%2BMETHODOLOGICAL%2BCONSIDERATIONS.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F656306%26rft_id%253Dinfo%253Apmid%252F21218185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/656306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21218185&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284000800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoshiba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Setoguchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinoshita</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizoshita</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of the effects explained by variations in the bovine <em>PLAG1</em> and <em>NCAPG</em> genes on daily body weight gain, linear skeletal measurements and carcass traits in Japanese Black steers from a progeny testing program.</span> <abbr class="cit-jnl-abbrev">Anim. Sci. J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BSci.%2BJ.%26rft.volume%253D84%26rft.spage%253D529%26rft_id%253Dinfo%253Apmid%252F23607392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23607392&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.26" data-doi="10.1093/hmg/ddq154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">C. W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayo</span>  <span class="cit-name-given-names">B. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association of anthropometric traits in African- and African-derived populations.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddq154%26rft_id%253Dinfo%253Apmid%252F20400458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMy8yNzI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.27" data-doi="10.1038/ng.814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karim</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">J. A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variants modulating the expression of a chromosome domain encompassing <em>PLAG1</em> influence bovine stature.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKarim%26rft.auinit1%253DL.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D405%26rft.epage%253D413%26rft.atitle%253DVariants%2Bmodulating%2Bthe%2Bexpression%2Bof%2Ba%2Bchromosome%2Bdomain%2Bencompassing%2BPLAG1%2Binfluence%2Bbovine%2Bstature.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.814%26rft_id%253Dinfo%253Apmid%252F21516082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.814&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21516082&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.28" data-doi="10.1093/hmg/dds332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Understanding and predicting complex traits: knowledge from cattle.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">R45</span>–<span class="cit-lpage">R51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fdds332%26rft_id%253Dinfo%253Apmid%252F22899652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjk6IjIxL1IxL1I0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.29" data-doi="10.1186/1471-2164-15-246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolormaa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Selection for complex traits leaves little or no classic signatures of selection.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-246%26rft_id%253Dinfo%253Apmid%252F24678841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24678841&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.30" data-doi="10.1371/journal.pbio.1001258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenstra</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porto Neto</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide analysis of the World’s sheep breeds reveals high levels of historic mixture and strong recent selection.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DKijas%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253De1001258%26rft.epage%253De1001258%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bthe%2Bworld%2527s%2Bsheep%2Bbreeds%2Breveals%2Bhigh%2Blevels%2Bof%2Bhistoric%2Bmixture%2Band%2Bstrong%2Brecent%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001258%26rft_id%253Dinfo%253Apmid%252F22346734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22346734&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.31" data-doi="10.1038/jhg.2009.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.-E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of 15 loci influencing height in a Korean population.</span> <abbr class="cit-jnl-abbrev">J. Hum. Genet.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DJ%2BHum%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D31%26rft.atitle%253DIdentification%2Bof%2B15%2Bloci%2Binfluencing%2Bheight%2Bin%2Ba%2BKorean%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fjhg.2009.116%26rft_id%253Dinfo%253Apmid%252F19893584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/jhg.2009.116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19893584&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.32" data-doi="10.1038/nature09410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lango Allen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estrada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berndt</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hundreds of variants clustered in genomic loci and biological pathways affect human height.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">838</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLango%2BAllen%26rft.auinit1%253DH.%26rft.volume%253D467%26rft.issue%253D7317%26rft.spage%253D832%26rft.epage%253D838%26rft.atitle%253DHundreds%2Bof%2Bvariants%2Bclustered%2Bin%2Bgenomic%2Bloci%2Band%2Bbiological%2Bpathways%2Baffect%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09410%26rft_id%253Dinfo%253Apmid%252F20881960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20881960&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282898700065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.33" data-doi="10.1016/j.ajhg.2010.11.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lanktree</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murtaza</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glessner</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Meta-analysis of dense genecentric association studies reveals common and uncommon variants associated with height.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLanktree%26rft.auinit1%253DM.%2BB.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D6%26rft.epage%253D18%26rft.atitle%253DMeta-analysis%2Bof%2BDense%2BGenecentric%2BAssociation%2BStudies%2BReveals%2BCommon%2Band%2BUncommon%2BVariants%2BAssociated%2Bwith%2BHeight.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.007%26rft_id%253Dinfo%253Apmid%252F21194676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21194676&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286501500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenstra</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groeneveld</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kantanen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular tools and analytical approaches for the characterization of farm animal genetic diversity.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D43%26rft.spage%253D483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.35" data-doi="10.1007/s00439-011-0969-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recent progress in the study of the genetics of height.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">465</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DLettre%26rft.auinit1%253DG.%26rft.volume%253D129%26rft.issue%253D5%26rft.spage%253D465%26rft.epage%253D472%26rft.atitle%253DRecent%2Bprogress%2Bin%2Bthe%2Bstudy%2Bof%2Bthe%2Bgenetics%2Bof%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-011-0969-x%26rft_id%253Dinfo%253Apmid%252F21340692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-011-0969-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21340692&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.36" data-doi="10.1038/ng.125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gieger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schumacher</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berndt</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of ten loci associated with height highlights new biological pathways in human growth.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLettre%26rft.auinit1%253DG.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D584%26rft.epage%253D591%26rft.atitle%253DIdentification%2Bof%2Bten%2Bloci%2Bassociated%2Bwith%2Bheight%2Bhighlights%2Bnew%2Bbiological%2Bpathways%2Bin%2Bhuman%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.125%26rft_id%253Dinfo%253Apmid%252F18391950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391950&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.-Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A novel c.-274C&gt;G polymorphism in bovine <em>SIRT1</em> gene contributes to diminished promoter activity and is associated with increased body size.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D44%26rft.spage%253D584%26rft_id%253Dinfo%253Apmid%252F23647079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23647079&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindholm-Perry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sexten</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association, effects and validation of polymorphisms within the <em>NCAPG</em> - <em>LCORL</em> locus located on BTA6 with feed intake, gain, meat and carcass traits in beef cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DLindholm-Perry%26rft.auinit1%253DA.%2BK.%26rft.volume%253D12%26rft.spage%253D103%26rft.epage%253D103%26rft.atitle%253DAssociation%252C%2Beffects%2Band%2Bvalidation%2Bof%2Bpolymorphisms%2Bwithin%2Bthe%2BNCAPG%2B-%2BLCORL%2Blocus%2Blocated%2Bon%2BBTA6%2Bwith%2Bfeed%2Bintake%252C%2Bgain%252C%2Bmeat%2Band%2Bcarcass%2Btraits%2Bin%2Bbeef%2Bcattle.%26rft_id%253Dinfo%253Apmid%252F22168586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22168586&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.39" data-doi="10.1007/s11033-013-2940-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gargani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chillemi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolazzi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Signatures of selection in five Italian cattle breeds detected by a 54K SNP panel.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">957</span>–<span class="cit-lpage">965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BRep.%26rft.volume%253D41%26rft.spage%253D957%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-013-2940-5%26rft_id%253Dinfo%253Apmid%252F24442315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-013-2940-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442315&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.40" data-doi="10.1371/journal.pone.0005350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and characterization of a high density SNP genotyping assay for cattle.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMatukumalli%26rft.auinit1%253DL.%2BK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5350%26rft.epage%253De5350%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bfor%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005350%26rft_id%253Dinfo%253Apmid%252F19390634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19390634&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.41" data-doi="10.3168/jds.2010-3385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diskin</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">C. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Associations between novel single nucleotide polymorphisms in the Bos taurus growth hormone gene and performance traits in Holstein-Friesian dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5959</span>–<span class="cit-lpage">5969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMullen%26rft.auinit1%253DM.%2BP.%26rft.volume%253D93%26rft.issue%253D12%26rft.spage%253D5959%26rft.epage%253D5969%26rft.atitle%253DAssociations%2Bbetween%2Bnovel%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bin%2Bthe%2BBos%2Btaurus%2Bgrowth%2Bhormone%2Bgene%2Band%2Bperformance%2Btraits%2Bin%2BHolstein-Friesian%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3385%26rft_id%253Dinfo%253Apmid%252F21094770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21094770&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284404400046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizoshita</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatsuda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association study identified three major QTL for carcass weight including the <em>PLAG1</em>-<em>CHCHD7</em> QTN for stature in Japanese Black cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DNishimura%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.spage%253D40%26rft.epage%253D40%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bidentified%2Bthree%2Bmajor%2BQTL%2Bfor%2Bcarcass%2Bweight%2Bincluding%2Bthe%2BPLAG1-CHCHD7%2BQTN%2Bfor%2Bstature%2Bin%2BJapanese%2BBlack%2Bcattle.%26rft_id%253Dinfo%253Apmid%252F22607022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22607022&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.43" data-doi="10.1093/hmg/ddq091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosono</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A genome-wide association study in 19 633 Japanese subjects identified <em>LHX3</em>-<em>QSOX2</em> and <em>IGF1</em> as adult height loci.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2303</span>–<span class="cit-lpage">2312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddq091%26rft_id%253Dinfo%253Apmid%252F20189936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8yMzAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.44" data-doi="10.1098/rstb.2009.0219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oleksyk</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide scans for footprints of natural selection.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DOleksyk%26rft.auinit1%253DT.%2BK.%26rft.volume%253D365%26rft.issue%253D1537%26rft.spage%253D185%26rft.epage%253D205%26rft.atitle%253DGenome-wide%2Bscans%2Bfor%2Bfootprints%2Bof%2Bnatural%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0219%26rft_id%253Dinfo%253Apmid%252F20008396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNjUvMTUzNy8xODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.45" data-doi="10.1186/1297-9686-46-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brien</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utsunomiya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meszaros</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickhart</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessing signatures of selection through variation in linkage disequilibrium between taurine and indicine cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-46-19%26rft_id%253Dinfo%253Apmid%252F24592996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-46-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24592996&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.46" data-doi="10.1371/journal.pgen.1003211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mickelson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rendahl</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valberg</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide analysis reveals selection for important traits in domestic horse breeds.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003211%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003211%26rft_id%253Dinfo%253Apmid%252F23349635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349635&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.47" data-doi="10.1101/gr.087577.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Signals of recent positive selection in a worldwide sample of human populations.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.087577.108%26rft_id%253Dinfo%253Apmid%252F19307593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvODI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pintus</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorbolini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaspa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimauro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Use of locally weighted scatterplot smoothing (LOWESS) regression to study selection signatures in Piedmontese and Italian Brown cattle breeds.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D45%26rft.spage%253D1%26rft_id%253Dinfo%253Apmid%252F23889699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23889699&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.49" data-doi="10.1186/1471-2164-14-876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porto-Neto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickhart</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Da Silva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic divergence of zebu and taurine cattle identified through high-density SNP genotyping.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D876%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-876%26rft_id%253Dinfo%253Apmid%252F24330634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24330634&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.50" data-doi="10.1534/genetics.110.123943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolormaa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Polymorphic regions affecting human height also control stature in cattle.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">981</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123943%26rft_id%253Dinfo%253Apmid%252F21212230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzMvOTgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzEzOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.51" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.52" data-doi="10.1186/1471-2164-12-318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qanbari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Application of site and haplotype-frequency based approaches for detecting selection signatures in cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQanbari%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D318%26rft.epage%253D318%26rft.atitle%253DApplication%2Bof%2Bsite%2Band%2Bhaplotype-frequency%2Bbased%2Bapproaches%2Bfor%2Bdetecting%2Bselection%2Bsignatures%2Bin%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-318%26rft_id%253Dinfo%253Apmid%252F21679429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21679429&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.53" data-doi="10.1371/journal.pgen.1004148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qanbari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pausch</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strom</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Classic selective sweeps revealed by massive sequencing in cattle.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004148%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004148%26rft_id%253Dinfo%253Apmid%252F24586189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586189&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.54" data-doi="10.1186/1471-2164-14-382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolf</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detection of selective sweeps in cattle using genome-wide SNP data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D382%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-382%26rft_id%253Dinfo%253Apmid%252F23758707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23758707&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Randhawa</span>  <span class="cit-name-given-names">I. A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Composite selection signals can localize the trait specific genomic regions in multi-breed populations of cattle and sheep.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D34%26rft_id%253Dinfo%253Apmid%252F24636660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24636660&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.56" data-doi="10.1186/1471-2164-14-908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothammer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seichter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medugorac</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genome-wide scan for signatures of differential artificial selection in ten cattle breeds.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D908%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-908%26rft_id%253Dinfo%253Apmid%252F24359457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24359457&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.57" data-doi="10.1038/nature06250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varilly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hostetter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide detection and characterization of positive selection in human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D913%26rft.epage%253D918%26rft.atitle%253DGenome-wide%2Bdetection%2Band%2Bcharacterization%2Bof%2Bpositive%2Bselection%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06250%26rft_id%253Dinfo%253Apmid%252F17943131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943131&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.58" data-doi="10.1038/ng.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">A. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaraja</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Common variants in the <em>GDF5-UQCC</em> region are associated with variation in human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSanna%26rft.auinit1%253DS.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D203%26rft.atitle%253DCommon%2Bvariants%2Bin%2Bthe%2BGDF5-UQCC%2Bregion%2Bare%2Bassociated%2Bwith%2Bvariation%2Bin%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.74%26rft_id%253Dinfo%253Apmid%252F18193045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.74&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18193045&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252732900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.59" data-doi="10.1111/j.1365-2052.2011.02302.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seichter</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russ</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothammer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eder</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Förster</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP-based association mapping of the polled gene in divergent cattle breeds.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DSeichter%26rft.auinit1%253DD.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D595%26rft.epage%253D598%26rft.atitle%253DSNP-based%2Bassociation%2Bmapping%2Bof%2Bthe%2Bpolled%2Bgene%2Bin%2Bdivergent%2Bcattle%2Bbreeds.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2011.02302.x%26rft_id%253Dinfo%253Apmid%252F22497248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2011.02302.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22497248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.60" data-doi="10.1093/hmg/ddp049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiura</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hikichi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hino</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Paternal deletion of <em>Meg1</em>/<em>Grb10</em> DMR causes maternalization of the <em>Meg1</em>/<em>Grb10</em> cluster in mouse proximal chromosome 11 leading to severe pre- and postnatal growth retardation.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1424</span>–<span class="cit-lpage">1438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp049%26rft_id%253Dinfo%253Apmid%252F19174477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjk6IjE4LzgvMTQyNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xMzkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.61" data-doi="10.1371/journal.pone.0008398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adeyemo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerry</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transferability and fine-mapping of genome-wide associated loci for adult height across human populations.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e8398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DShriner%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8398%26rft.epage%253De8398%26rft.atitle%253DTransferability%2Band%2Bfine-mapping%2Bof%2Bgenome-wide%2Bassociated%2Bloci%2Bfor%2Badult%2Bheight%2Bacross%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008398%26rft_id%253Dinfo%253Apmid%252F20027299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20027299&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.62" data-doi="10.1371/journal.pone.0037282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Signer-Hasler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flury</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A genome-wide association study reveals loci influencing height and other conformation traits in horses.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSigner-Hasler%26rft.auinit1%253DH.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37282%26rft.epage%253De37282%26rft.atitle%253DA%2Bgenome-wide%2Bassociation%2Bstudy%2Breveals%2Bloci%2Binfluencing%2Bheight%2Band%2Bother%2Bconformation%2Btraits%2Bin%2Bhorses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037282%26rft_id%253Dinfo%253Apmid%252F22615965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22615965&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.63" data-doi="10.1371/journal.pgen.1000445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soranzo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivadeneira</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinappen-Horsley</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malkina</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Meta-analysis of genome-wide scans for human adult stature identifies novel loci and associations with measures of skeletal frame size.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSoranzo%26rft.auinit1%253DN.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253De1000445%26rft.epage%253De1000445%26rft.atitle%253DMeta-analysis%2Bof%2Bgenome-wide%2Bscans%2Bfor%2Bhuman%2Badult%2Bstature%2Bidentifies%2Bnovel%2BLoci%2Band%2Bassociations%2Bwith%2Bmeasures%2Bof%2Bskeletal%2Bframe%2Bsize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000445%26rft_id%253Dinfo%253Apmid%252F19343178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19343178&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.64" data-doi="10.1371/journal.pgen.1000409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sovio</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millwood</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molitor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic determinants of height growth assessed longitudinally from infancy to adulthood in the northern Finland birth cohort 1966.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSovio%26rft.auinit1%253DU.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De1000409%26rft.epage%253De1000409%26rft.atitle%253DGenetic%2Bdeterminants%2Bof%2Bheight%2Bgrowth%2Bassessed%2Blongitudinally%2Bfrom%2Binfancy%2Bto%2Badulthood%2Bin%2Bthe%2Bnorthern%2BFinland%2Bbirth%2Bcohort%2B1966.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000409%26rft_id%253Dinfo%253Apmid%252F19266077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19266077&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.65" data-doi="10.1534/genetics.110.116111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ajmone-Marsan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazzari</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boettcher</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of selection signatures in cattle breeds selected for dairy production.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116111%26rft_id%253Dinfo%253Apmid%252F20479146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzE0NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTM5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.66" data-doi="10.1111/age.12031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Widmann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kühn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genome-wide association study indicates <em>LCORL</em>/<em>NCAPG</em> as a candidate locus for withers height in German Warmblood horses.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D44%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12031%26rft_id%253Dinfo%253Apmid%252F23418885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23418885&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.67" data-doi="10.1371/journal.pone.0064280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utsunomiya</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez O’Brien</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">do Carmo</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detecting loci under recent positive selection in dairy and beef cattle by combining different genome-wide scan methods.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e64280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De64280%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0064280%26rft_id%253Dinfo%253Apmid%252F23696874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0064280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23696874&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cattle gain stature.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D397%26rft.epage%253D398%26rft.atitle%253DCattle%2Bgain%2Bstature.%26rft_id%253Dinfo%253Apmid%252F21522175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21522175&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.69" data-doi="10.1038/ng.121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lango</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindgren</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide association analysis identifies 20 loci that influence adult height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWeedon%26rft.auinit1%253DM.%2BN.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D575%26rft.epage%253D583%26rft.atitle%253DGenome-wide%2Bassociation%2Banalysis%2Bidentifies%2B20%2Bloci%2Bthat%2Binfluence%2Badult%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.121%26rft_id%253Dinfo%253Apmid%252F18391952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391952&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.70" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1391.71" data-doi="10.1186/1471-2164-13-704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Da</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A genome-wide scan of selective sweeps in two broiler chicken lines divergently selected for abdominal fat content.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D13%26rft.spage%253D704%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-704%26rft_id%253Dinfo%253Apmid%252F23241142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-704&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23241142&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1391.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/7/1391.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1379" title="Tryptophan-Dependent Control of Colony Formation After DNA Damage via Sea3-Regulated TORC1 Signaling in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1403" title="Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/7"><span>Volume 5 Issue 7, July 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/7.cover-source.jpg?itok=IZbYpE05" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1391" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ut-KHhLVz57zxZMMKZGIb8OmZyUV28asWwcGEDZkrOw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/7/1391" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/7/1391" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/7/1391" class="active" data-icon-position="" data-hide-link-title="0">Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459978" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lDXuPAFBfaWFvXFmZJh4dTVkaA4FmKHimZMgJfUEICE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1391.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/7/1391" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-I-jlsUXD0A6JsapRPaLB0yEu9GSCVk8hpq7DdjKHtlQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/7/1391" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1391.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459978" id="node459978--2" data-pisa="ggg;5/7/1391" data-pisa-master="ggg;g3.115.017772" data-apath="/ggg/5/7/1391.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African <em>Bos taurus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Imtiaz A. S. Randhawa</span>, <span class="highwire-citation-author" data-delta="1">Mehar S. Khatkar</span>, <span class="highwire-citation-author" data-delta="2">Peter C. Thomson</span> and <span class="highwire-citation-author" data-delta="3">Herman W. Raadsma</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1391-1401;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017772 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459978--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Imtiaz A. S. Randhawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Imtiaz%2BA.%2BS.%2BRandhawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Randhawa%20IA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AImtiaz%2BA.%2BS.%2BRandhawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:imtiaz.randhawa@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">imtiaz.randhawa@sydney.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mehar S. Khatkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehar%2BS.%2BKhatkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khatkar%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMehar%2BS.%2BKhatkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter C. Thomson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BC.%2BThomson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomson%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BC.%2BThomson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Herman W. Raadsma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herman%2BW.%2BRaadsma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raadsma%20HW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerman%2BW.%2BRaadsma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/7/1391" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/7/1391&amp;title=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/7/1391&amp;title=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/7/1391&amp;title=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/7/1391&amp;text=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/7/1391&amp;title=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/7/1391&amp;t=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/7/1391&amp;title=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/7/1391&amp;title=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459978/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459978" id="node459978--3" data-pisa="ggg;5/7/1391" data-pisa-master="ggg;g3.115.017772" data-apath="/ggg/5/7/1391.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African <em>Bos taurus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Imtiaz A. S. Randhawa</span>, <span class="highwire-citation-author" data-delta="1">Mehar S. Khatkar</span>, <span class="highwire-citation-author" data-delta="2">Peter C. Thomson</span> and <span class="highwire-citation-author" data-delta="3">Herman W. Raadsma</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1391-1401;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017772 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459978--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Imtiaz A. S. Randhawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Imtiaz%2BA.%2BS.%2BRandhawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Randhawa%20IA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AImtiaz%2BA.%2BS.%2BRandhawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:imtiaz.randhawa@sydney.edu.au" class="" data-icon-position="" data-hide-link-title="0">imtiaz.randhawa@sydney.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mehar S. Khatkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mehar%2BS.%2BKhatkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khatkar%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMehar%2BS.%2BKhatkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter C. Thomson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BC.%2BThomson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomson%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BC.%2BThomson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Herman W. Raadsma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Reprogen–Animal Bioscience Group, Faculty of Veterinary Science, The University of Sydney, Camden, 2570, New South Wales, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Herman%2BW.%2BRaadsma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raadsma%20HW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHerman%2BW.%2BRaadsma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1391&amp;rft.epage=1401&amp;rft.atitle=Composite%20Selection%20Signals%20for%20Complex%20Traits%20Exemplified%20Through%20Bovine%20Stature%20Using%20Multibreed%20Cohorts%20of%20European%20and%20African%20Bos%20taurus&amp;rft.volume=5&amp;rft.issue=7&amp;rft.date=2015-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Randhawa%2C+Imtiaz+A.+S.&amp;rft.au=Khatkar%2C+Mehar+S.&amp;rft.au=Thomson%2C+Peter+C.&amp;rft.au=Raadsma%2C+Herman+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459978/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459978/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459978/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459978/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459978/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459978/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459978/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459978/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459978/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459978/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459978/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459978/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/7/1391" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/7/1391.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/7/1391.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/7/1391.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/7/1391.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
